
<html lang="en"     class="pb-page"  data-request-id="54145251-84dd-4f21-a06e-03fcc9ca35c8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-11;article:article:10.1021/acs.jmedchem.1c00398;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Synthesis and Pharmacological Evaluation of Tetrahydro-γ-carboline Derivatives as Potent Anti-inflammatory Agents Targeting Cyclic GMP–AMP Synthase" /></meta><meta name="dc.Creator" content="Jing  Tan" /></meta><meta name="dc.Creator" content="Bing  Wu" /></meta><meta name="dc.Creator" content="Tingting  Chen" /></meta><meta name="dc.Creator" content="Chen  Fan" /></meta><meta name="dc.Creator" content="Jiannan  Zhao" /></meta><meta name="dc.Creator" content="Chaodong  Xiong" /></meta><meta name="dc.Creator" content="Chunlan  Feng" /></meta><meta name="dc.Creator" content="Ruoxuan  Xiao" /></meta><meta name="dc.Creator" content="Chunyong  Ding" /></meta><meta name="dc.Creator" content="Wei  Tang" /></meta><meta name="dc.Creator" content="Ao  Zhang" /></meta><meta name="dc.Description" content="The activation of cyclic GMP–AMP synthase (cGAS) by double-stranded DNA is implicated in the pathogenesis of many hyperinflammatory and autoimmune diseases, and the cGAS-targeting small molecule ha..." /></meta><meta name="Description" content="The activation of cyclic GMP–AMP synthase (cGAS) by double-stranded DNA is implicated in the pathogenesis of many hyperinflammatory and autoimmune diseases, and the cGAS-targeting small molecule ha..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 28, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00398" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00398" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00398" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00398" /></link>
        
    
    

<title>Synthesis and Pharmacological Evaluation of Tetrahydro-γ-carboline Derivatives as Potent Anti-inflammatory Agents Targeting Cyclic GMP–AMP Synthase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00398" /></meta><meta property="og:title" content="Synthesis and Pharmacological Evaluation of Tetrahydro-γ-carboline Derivatives as Potent Anti-inflammatory Agents Targeting Cyclic GMP–AMP Synthase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0024.jpeg" /></meta><meta property="og:description" content="The activation of cyclic GMP–AMP synthase (cGAS) by double-stranded DNA is implicated in the pathogenesis of many hyperinflammatory and autoimmune diseases, and the cGAS-targeting small molecule has emerged as a novel therapeutic strategy for treating these diseases. However, the currently reported cGAS inhibitors are far beyond maturity, barely demonstrating in vivo efficacy. Inspired by the structural novelty of compound 5 (G140), we conducted a structural optimization on both its side chain and the central tricyclic core, leading to several subseries of compounds, including those unexpectedly cyclized complex ones. Compound 25 bearing an N-glycylglycinoyl side chain was identified as the most potent one with cellular IC50 values of 1.38 and 11.4 μM for h- and m-cGAS, respectively. Mechanistic studies confirmed its direct targeting of cGAS. Further, compound 25 showed superior in vivo anti-inflammatory effects in the lipopolysaccharide-induced mouse model. The encouraging result of compound 25 provides solid evidence for further pursuit of cGAS-targeting inhibitors as a new anti-inflammatory treatment." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00398"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00398">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00398&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00398&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00398&amp;href=/doi/10.1021/acs.jmedchem.1c00398" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 7667-7690</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00370" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00406" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Synthesis and Pharmacological Evaluation of Tetrahydro-γ-carboline Derivatives as Potent Anti-inflammatory Agents Targeting Cyclic GMP–AMP Synthase</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jing Tan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">Pharm-X Center, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Tan">Jing Tan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bing Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bing Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bing++Wu">Bing Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tingting Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tingting Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharm-X Center, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tingting++Chen">Tingting Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chen Fan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chen Fan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chen++Fan">Chen Fan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiannan Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiannan Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">Pharm-X Center, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiannan++Zhao">Jiannan Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chaodong Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chaodong Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">Pharm-X Center, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chaodong++Xiong">Chaodong Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chunlan Feng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunlan Feng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunlan++Feng">Chunlan Feng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ruoxuan Xiao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ruoxuan Xiao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharm-X Center, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ruoxuan++Xiao">Ruoxuan Xiao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chunyong Ding</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunyong Ding</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharm-X Center, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div><div class="loa-info-affiliations-info">Shanghai Key Laboratory for Molecular Engineering of Chiral Drugs, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#86e5eef3e8e2efe8e1c6f5ecf2f3a8e3e2f3a8e5e8"><span class="__cf_email__" data-cfemail="3d5e5548535954535a7d4e57494813585948135e53">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunyong++Ding">Chunyong Ding</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Tang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Tang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a3d7c2cdc4d4c6cae3d0cacece8dc2c08dc0cd"><span class="__cf_email__" data-cfemail="0b7f6a656c7c6e624b78626666256a68256865">[email protected]</span></a>. Phone: +86-21-50806820.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Tang">Wei Tang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Ao Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ao Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Institute of Materia Medica (SIMM), Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">Pharm-X Center, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">Shanghai Key Laboratory for Molecular Engineering of Chiral Drugs, School of Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#4928267f707870332128272e093a233d3c672c2d3c672a27"><span class="__cf_email__" data-cfemail="fb9a94cdc2cac281939a959cbb88918f8ed59e9f8ed59895">[email protected]</span></a>. Phone: +86-21-34205128.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ao++Zhang">Ao Zhang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7205-9202" title="Orcid link">https://orcid.org/0000-0001-7205-9202</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00398&amp;href=/doi/10.1021%2Facs.jmedchem.1c00398" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 7667–7690</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 27, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 March 2021</li><li><span class="item_label"><b>Published</b> online</span>28 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00398" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00398</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7667%26pageCount%3D24%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJing%2BTan%252C%2BBing%2BWu%252C%2BTingting%2BChen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D11%26contentID%3Dacs.jmedchem.1c00398%26title%3DSynthesis%2Band%2BPharmacological%2BEvaluation%2Bof%2BTetrahydro-%25CE%25B3-carboline%2BDerivatives%2Bas%2BPotent%2BAnti-inflammatory%2BAgents%2BTargeting%2BCyclic%2BGMP%25E2%2580%2593AMP%2BSynthase%26numPages%3D24%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7690%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00398"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">906</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00398" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Synthesis and Pharmacological Evaluation of Tetrahydro-γ-carboline Derivatives as Potent Anti-inflammatory Agents Targeting Cyclic GMP–AMP Synthase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Bing&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Tingting&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Chen&quot;,&quot;last_name&quot;:&quot;Fan&quot;},{&quot;first_name&quot;:&quot;Jiannan&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Chaodong&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Chunlan&quot;,&quot;last_name&quot;:&quot;Feng&quot;},{&quot;first_name&quot;:&quot;Ruoxuan&quot;,&quot;last_name&quot;:&quot;Xiao&quot;},{&quot;first_name&quot;:&quot;Chunyong&quot;,&quot;last_name&quot;:&quot;Ding&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Tang&quot;},{&quot;first_name&quot;:&quot;Ao&quot;,&quot;last_name&quot;:&quot;Zhang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;7667-7690&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00398&quot;},&quot;abstract&quot;:&quot;The activation of cyclic GMP–AMP synthase (cGAS) by double-stranded DNA is implicated in the pathogenesis of many hyperinflammatory and autoimmune diseases, and the cGAS-targeting small molecule has emerged as a novel therapeutic strategy for treating these diseases. However, the currently reported cGAS inhibitors are far beyond maturity, barely demonstrating in vivo efficacy. Inspired by the structural novelty of compound 5 (G140), we conducted a structural optimization on both its side chain and the central tricyclic core, leading to several subseries of compounds, including those unexpectedly cyclized complex ones. Compound 25 bearing an N-glycylglycinoyl side chain was identified as the most potent one with cellular IC50 values of 1.38 and 11.4 μM for h- and m-cGAS, respectively. Mechanistic studies confirmed its direct targeting of cGAS. Further, compound 25 showed superior in vivo anti-inflammatory effects in the lipopolysaccharide-induced mouse model. The encouraging result of compound 25 provides soli&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00398&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00398" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00398&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00398" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00398&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00398" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00398&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00398&amp;href=/doi/10.1021/acs.jmedchem.1c00398" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00398" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00398" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00398%26sid%3Dliteratum%253Aachs%26pmid%3D34044539%26genre%3Darticle%26aulast%3DTan%26date%3D2021%26atitle%3DSynthesis%2Band%2BPharmacological%2BEvaluation%2Bof%2BTetrahydro-%25CE%25B3-carboline%2BDerivatives%2Bas%2BPotent%2BAnti-inflammatory%2BAgents%2BTargeting%2BCyclic%2BGMP%25E2%2580%2593AMP%2BSynthase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D11%26spage%3D7667%26epage%3D7690%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (7)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290694" title="Bioluminescent probes">Bioluminescent probes</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291297" title="Cell signaling">Cell signaling</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/jmcmar.2021.64.issue-11/20210610/jmcmar.2021.64.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0024.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The activation of cyclic GMP–AMP synthase (cGAS) by double-stranded DNA is implicated in the pathogenesis of many hyperinflammatory and autoimmune diseases, and the cGAS-targeting small molecule has emerged as a novel therapeutic strategy for treating these diseases. However, the currently reported cGAS inhibitors are far beyond maturity, barely demonstrating in vivo efficacy. Inspired by the structural novelty of compound <b>5</b> (G140), we conducted a structural optimization on both its side chain and the central tricyclic core, leading to several subseries of compounds, including those unexpectedly cyclized complex ones. Compound <b>25</b> bearing an <i>N</i>-glycylglycinoyl side chain was identified as the most potent one with cellular IC<sub>50</sub> values of 1.38 and 11.4 μM for h- and m-cGAS, respectively. Mechanistic studies confirmed its direct targeting of cGAS. Further, compound <b>25</b> showed superior in vivo anti-inflammatory effects in the lipopolysaccharide-induced mouse model. The encouraging result of compound <b>25</b> provides solid evidence for further pursuit of cGAS-targeting inhibitors as a new anti-inflammatory treatment.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63222" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63222" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cyclic GMP–AMP synthase (cGAS) is an important component of the innate immune system functioning as a key first-line host defense against viral and bacterial infections and cellular damage. It can detect and bind double-stranded DNA (dsDNA) in the cytoplasm that originates from invading pathogens and mislocalized or misprocessed self-dsDNA.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The binding of cGAS with dsDNA induces a conformational change, allowing cGAS to catalytically synthesize the second messenger—cyclic dinucleotide 2′,3′-cGAMP (cGAMP)—from ATP and GTP through two steps.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> cGAMP is an endogenous ligand that potently binds to the endoplasmic reticulum membrane-bound adapter protein—stimulator of interferon genes (STINGs).<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a> Subsequently, a downstream signal transduction cascade occurs, leading to phosphorylation, dimerization, and nuclear translocation of the interferon regulatory factor 3 (IRF3).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The activation of IRF3 and nuclear factor kappa-B (NF-κB) induces the secretion of type-I interferons and many other pro-inflammatory cytokines, ultimately activating the innate immune response and T-cell priming to eliminate pathogenic infection and tissue damage.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a></div><div class="NLM_p">The cGAS–STING pathway plays a critical role in maintaining host innate immune homeostasis. The continuous activation of cGAS by high immunogenic DNA can cause various hyper-inflammatory and autoimmune diseases,<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> such as the Aicardi-Goutières syndrome (AGS), a lupus-like autoimmune disorder characterized by the overproduction of interferons due to cGAS overactivation.<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15−17)</a> In addition, the dysfunction of the cGAS pathway was reported to induce systemic inflammatory diseases including cytokine release syndrome and cytokine storm.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a> All these results undoubtedly demonstrate the detrimental effects of cGAS overactivation in the maintenance of innate immunity. Therefore, specific inhibitors targeting cGAS overactivation have been proposed as a promising therapeutic strategy for treating human inflammatory and autoimmune diseases where the dysfunction of the cGAS signaling pathway is involved.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20−22)</a></div><div class="NLM_p">Considerable efforts have been devoted to the identification and characterization of small-molecule inhibitors of cGAS.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> For example, the prescribed anti-malarial drugs,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> hydroxychloroquine and quinacrine, were found to indirectly impair the cGAS activity by intercalating the minor groove of DNA at the cGAS/DNA interaction interface, and the antiprotozoal drug, suramin,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> was found as a nucleic mimic to block the binding of cGAS with dsDNA. However, these cGAS inhibitors suffer from off-target effects due to their nonspecific interactions with dsDNA rather than directly binding cGAS to specifically inhibit their enzymatic activity.<a onclick="showRef(event, 'ref21 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref23 ref24">(21,23,24)</a> To identify direct cGAS inhibitors targeting the catalytic pocket, Ascano and co-workers conducted a high-throughput screening and validated a benzoimidazole compound, Ru.521 (compound <b>3</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), as the first ATP/GTP competitive inhibitor of cGAS.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This compound potently inhibits murine cGAS (m-cGAS, IC<sub>50</sub> = 0.11 μM) but displays weak activity against recombinant human cGAS (h-cGAS, IC<sub>50</sub> = 2.94 μM).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> More efforts have also been reported by many other groups (compounds <b>4–10</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) and delivered several classes of specific cGAS inhibitors targeting the active site;<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31">(27−31)</a> however, most of them exhibit a significant discrepancy in potency against h- or m-cGAS and with nearly no or weak cellular activities. Notably, compounds <b>4–6</b> bearing a common 2,3,4,5-tetrahydro-1<i>H</i>-pyrido[4,3-<i>b</i>]indole (tetrahydro-γ-carboline, THγC) scaffold reported by Tuschl and co-workers recently<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> displayed high but varying potency for h-cGAS and m-cGAS. It was found that the C6,C7-dichloro substitution pattern is critical for high potency against h-cGAS, whereas the C9-substituent has an important impact on m-cGAS. Compounds <b>4–6</b> all have high potency for h-cGAS with IC<sub>50</sub> values ranging between 10 and 27 nM; however, only compound <b>5</b> bearing a C9-(1-methyl-1<i>H</i>-pyrazol-3-yl) shows modest potency against m-cGAS (0.44 μM), and nearly no activity is observed for compounds <b>4</b> and <b>6</b> (5.15 and 25 μM, respectively).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In addition, all these compounds only exhibit modest inhibitory potency in human monocytic THP-1 cells expressing cGAS mRNA with IC<sub>50</sub> values of 1–3 μM, which is over 100-fold less potent compared to their high enzymatic potency, likely in part due to the pharmacokinetic property (PK)-related membrane barriers.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative cGAS inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">From the results mentioned above, the development of potent cGAS inhibitors remains in the infancy and high cellular potent inhibitors are not available yet, not to say the emergent need of a drug-like compound suitable for clinical study. As part of our ongoing efforts to develop small-molecule modulators targeting the cGAS–STING pathway as a potential therapeutic treatment,<a onclick="showRef(event, 'ref22 ref32'); return false;" href="javascript:void(0);" class="ref ref22 ref32">(22,32)</a> we initiate a medicinal chemistry optimization campaign based on the THγC scaffold as in compounds <b>4–6</b> through several strategies with the aim to establish an extensive structure–activity relationship that may lead to more potent compounds for cGAS both in the biochemical and cellular assays for further proof-of-concept evaluation. Accordingly, compound <b>25</b> bearing an <i>N</i>-glycylglycinoyl side chain was identified not only possessing a markedly enhanced cellular potency against cGAS but also showing much improved aqueous solubility. More importantly, this new inhibitor demonstrated superior in vivo suppressing effects on the production of inflammatory cytokines in a lipopolysaccharide (LPS)-induced mouse model. Herein, we report our compound’s design and pharmacological evaluation.</div><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Structure-Based Drug Design</h3><div class="NLM_p">Our drug design is based on tetrahydro-1<i>H</i>-pyrido[4,3-<i>b</i>]indoles <b>4–6</b> due to their novel structural chemotype but with limited structure–activity relationship reported. The binding sites of compounds <b>4</b> and <b>6</b> were confirmed by X-ray crystallography (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJU">6MJU</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJW">6MJW</a>), with structures of inhibitors bound to apo h-cGAS<sup>CD</sup> (h-cGAS catalytic domain).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> From the co-crystal structure of compound <b>6</b> in complex with h-cGAS (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJW">6MJW</a>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), the tricyclic THγC core sits in a sandwiched mode in the ATP/GTP binding pocket of h-cGAS. The carbonyl moiety within the hydroxyacetyl side chain forms key H-binding interactions with Ser434 leaving the terminal hydroxyl group free toward solvent interactions. The central pyrolyl nitrogen atom forms a few H bonds with several amino acid residues through a molecule of water. The nitrogen atom of C9-pyridine forms another key H bonding with Tyr248. To be safe, we first initiated an approach (approach 1, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) to replace or extend the terminal hydroxyl of the hydroxyacetyl side chain aiming to maintain the high enzymatic potency against cGAS and improve the overall PK properties to enhance the cellular potency, leading to compound series I and II. Meanwhile, because the C9 substituent is critical for m-cGAS potency and there is ample space around its binding domain, we decided to replace the C9-pyridinyl with other groups, such as indoles and anilines and walking the new substituent to the C12 position (approach 2, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) to generate new patentable compounds, with structures distinct from compounds <b>4–6</b>. This strategy led to compound series III and IV bearing a different 2,3,4,12-tetrahydro-1<i>H</i>-pyrido[4,3-<i>b</i>]indole skeleton. Interestingly, during the synthesis, we found that further cyclized complex structures V and VI were produced from compounds IV depending on the electronic nature of anilines used. Herein, we report the synthesis and pharmacological characterization of these new compounds.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Co-crystal of h-cGAS with compound <b>6</b> and our drug design (approaches 1 and 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">As depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, we first synthesized compounds bearing different side chains to elaborate the original hydroxyacetyl fragment in compound <b>5</b>. The synthesis is commenced from the halogenated THγC <b>11</b>, which was prepared according to a literature procedure.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Condensation of <b>11</b> with different <i>N</i>-Boc-protected amino acids or other carboxylic acids afforded <b>12a–g</b> in 45–66% yields. The hexafluoro intermediate <b>12h</b> was prepared in 39% yield by treating <b>11</b> with triphosgene and hexafluoroisopropanol in the presence of <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The C9-(1-methylpyrazol-3-yl) substituent was installed by the Suzuki cross-coupling of intermediates <b>12a–h</b> with pyrazol-3-ylborane under palladium catalysis to give THγCs <b>13a–d</b>, <b>14</b>, and <b>17–19</b> in 31–84% yields. The removal of the <i>N</i>-Boc group of <b>13a–d</b> under trifluoroacetic acid (TFA) led to target compounds <b>15–16</b> and <b>20–21</b> in 42–63% yields.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>14–21</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) <i>N</i>-Boc-amino acids or carboxylic acids, EDCI/HOBt or HATU, DIPEA, and DMF, rt, 45–66%; (ii) triphosgene, hexafluoroisopropanol, DIPEA, and DCM, rt, 39%; (iii) 1-methyl-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyrazole, Pd(dppf)Cl<sub>2</sub>, KOAc, and dioxane, 100 °C, 31–84%; and (iv) CF<sub>3</sub>COOH and DCM, rt, 42–63%.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, the Borch reductive amination of <b>20</b> under the standard conditions afforded the <i>N</i>,<i>N</i>-dimethyl derivative <b>22</b>. The condensation of <b>20</b> with <i>N</i>-Boc amino acids or carboxylic acids provided compounds <b>23a–f</b>, <b>24</b>, <b>28</b>, and <b>33–37</b> in 23–63% yields. Subsequent <i>N</i>-Boc deprotection with TFA afforded the target THγCs <b>25–27</b> and <b>29–32</b> in 42–63% yields.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>22</b> and <b>24–37</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) HCHO, NaBH<sub>3</sub>CN, HOAc, and MeOH, 0 °C, 60%; (ii) <i>N</i>-Boc amino acids or carboxylic acids, EDCI/HOBt or HATU, DIPEA, and DMF, rt, 23–63%; and (iii) CF<sub>3</sub>COOH and DCM, rt, 42–63%.</p></p></figure><div class="NLM_p">As outlined in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, Fisher indole synthesis by using commercially available 4-piperidinone hydrochloride and 2,3-dichlorophenylhydrazine hydrochloride (<b>38</b>) as starting materials in the presence of con. H<sub>2</sub>SO<sub>4</sub> gave rise to THγC <b>39</b> in 64% yield, which was then condensed with 2-acetoxyacetic acid to afford intermediate <b>40</b> in 67% yield. The treatment of <b>40</b> with <i>t</i>-BuOCl and <i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethylethylenediamine (TMEDA) afforded 3-chloroindolenine <b>41</b>, which was followed by a nucleophilic substitution reaction with various substituted anilines to provide a series of C12-aminoindolenines <b>42a–d</b>, <b>43–46</b>, and <b>51</b> in 53–73% overall yields (two steps). The removal of the <i>O</i>-acetyl group in <b>42a–d</b> and <b>51</b> with LiOH afforded the target THγCs <b>47–50</b> and <b>52</b> in 30–66% yields. The chemical structure of this series of compounds was confirmed by X-ray crystallographic analysis of compound <b>47</b>. To our surprise, the treatment of indolenines <b>45</b> and <b>43</b> bearing a formyl moiety on the C12-arylamino group with LiOH followed by HCl solution gave rise to further cyclized indolines <b>53</b> and <b>54</b> bearing a fused polycyclic system in 63–64% yields. The unique structure of <b>53</b> was determined by X-ray crystallographic analysis. The exact reaction mechanism is not yet clear and needs further exploration; however, we tentatively proposed an explanation involving the intramolecular aldol reaction followed by the acid-mediated pinacol-like rearrangement (Scheme S1 in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_001.pdf" class="ext-link">Supporting Information</a>). Treating <b>41</b> with LiOH followed by 1 N HCl provided 3-chloroindoline <b>55</b> in 24% overall yield. It is of note that the treatment of the electron-rich 2,5-dimethoxyaniline with 3-chloroindolenines <b>41</b> generated bisindoline <b>56</b> in 83% yield, which is likely formed through nucleophilic substitution followed by intramolecular cyclization.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Subsequent O-deprotection with LiOH furnished the final compound <b>57</b> in 75% yield. Luckily, the chemical structure of <b>57</b> was confirmed by X-ray crystallographic analysis.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>43–55</b> and <b>57</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) piperidin-4-one hydrochloride, H<sub>2</sub>SO<sub>4</sub>, dioxane, 115 °C, 64%; (ii) acetoxyacetic acid, EDCI, HOBt, DIPEA, DMF, rt, 67%; (iii) <i>t</i>-BuOCl, TMEDA, DCM, rt; (iv) substituted anilines, H<sub>2</sub>SO<sub>4</sub>, rt, 53–73%; (v) LiOH·H<sub>2</sub>O, THF, H<sub>2</sub>O, rt, 30–75%; (vi) LiOH·H<sub>2</sub>O, THF, H<sub>2</sub>O, rt; then 1 N HCl, rt; 24–64%; (vii) 2,5-dimethoxyaniline, H<sub>2</sub>SO<sub>4</sub>, rt, 83%.</p></p></figure><div class="NLM_p">As described in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, treatment of <b>41</b> with various substituted indoles afforded a series of C12-indolyl-substituted THγCs <b>58a–d</b> and <b>59</b> in 58–80% yields under acidic conditions. Subsequent deprotection of the <i>O</i>-acetyl group with LiOH afforded THγCs <b>60–64</b> in 34–67% yields. The chemical structure of this series was further confirmed by X-ray crystallographic analysis of <b>63</b>. The reduction of the C═N double bond in compound <b>59</b> with NaBH<sub>3</sub>CN gave rise to indoline <b>65</b> in 69% yield, respectively, which was further structurally secured by X-ray crystallographic analysis. The removal of the <i>O</i>-acetyl group in <b>65</b> furnished the target compound <b>66</b> in 61% yield.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>59–66</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) <i>t</i>-BuOCl, TMEDA, and DCM, rt; (ii) substituted indoles and H<sub>2</sub>SO<sub>4</sub>, rt, 58–80%; (iii) LiOH·H<sub>2</sub>O, THF, and H<sub>2</sub>O, rt, 34–67%; and (iv) NaBH<sub>3</sub>CN, HOAc, and MeOH, 0 °C, 69%.</p></p></figure></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07782" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07782" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Cell-Based Luciferase Assay</h3><div class="NLM_p">Considering the significant discrepancy between cellular and biochemical potencies of the reference cGAS inhibitors <b>4–6</b>,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> and to quickly identify cellular potent compounds for further studies, we directly conducted a cell-based Lucia luciferase assay to screen all the synthesized compounds for their cellular efficacy in suppressing both the dsDNA-dependent activation of cGAS and the subsequent upregulation of type-I interferon. In this assay, transfection of dsDNA into human THP1-Dual cells or mouse RAW-Lucia ISG cells leads to the upregulation of type-I interferons through the activation of IRF3 in a cGAS-dependent manner. An interferon-stimulated response element coupled to a luciferase gene (IRF-Lucia) is stably expressed in THP1-Dual cells or RAW-Lucia ISG cells, which allows utilizing a luciferase signal as the readout of cGAS activity. Compounds <b>3</b> and <b>5</b> were employed as the positive controls. Meanwhile, the cytotoxicity of these compounds was characterized as CC<sub>50</sub> values in the CCK-8 assay. All the results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<named-content content-type="anchor" rid="tbl2" type="simple"></named-content><named-content content-type="anchor" rid="tbl3" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibitory Activity of Compounds against cGAS Activation by dsDNA<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0015.gif" alt="" id="GRAPHIC-d7e1021-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0016.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The CC<sub>50</sub> and IC<sub>50</sub> values were determined by the CCK-8 assay and the cell-based Lucia luciferase assay, respectively, with GraphPad Prism 8.0 software.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Data were obtained from at least three independent tests and expressed as mean ± standard error of the mean (SEM). Dashed lines mean data not tested.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitory Activity of Compounds against cGAS Activation by dsDNA<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0017.gif" alt="" id="GRAPHIC-d7e1047-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0018.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The CC<sub>50</sub> and IC<sub>50</sub> values were determined by the CCK-8 assay and the cell-based Lucia luciferase assay, respectively, with GraphPad Prism 8.0 software.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Data were obtained from at least three independent tests and expressed as mean ± SEM. Dashed lines mean data not tested.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Inhibitory Activity of Compounds against cGAS Activation by dsDNA<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0019.gif" alt="" id="GRAPHIC-d7e1073-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0020.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">The CC<sub>50</sub> and IC<sub>50</sub> values were determined by the CCK-8 assay and the cell-based Lucia luciferase assay, respectively, with GraphPad Prism 8.0 software.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Data were obtained from at least three independent tests and expressed as mean ± SEM. Dashde lines mean data not tested.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Inhibitory Activity of Compounds against cGAS Activation by dsDNA<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0021.gif" alt="" id="GRAPHIC-d7e1099-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0022.gif" alt="" id="gr17" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">The CC<sub>50</sub> and IC<sub>50</sub> values were determined by the CCK-8 assay and the cell-based Lucia luciferase assay, respectively, with GraphPad Prism 8.0 software.</p></div></div><div></div></div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, we first evaluated the effects of variations of N-side chains connecting to the piperidinyl portion of the THγC scaffold on the cGAS inhibitory activity. It was found that the substitution of the terminal hydroxyl group with either an N- or O-heterocycle (<b>14–16</b>) showed negligible potency against h-cGAS in the human THP1-Dual cells with IC<sub>50</sub> values greater than 20 μM. Compounds <b>17–18</b> obtained by replacing the terminal hydroxymethyl with a cyclobutane bearing different substitutions also abolished the cellular potency. Compound <b>19</b> with the hydroxyacetyl side chain replaced by hexafluoroisopropyl ester displayed negligible potency against h-cGAS (19.2 μM), but modest potency against m-cGAS was observed with an IC<sub>50</sub> value of 3.23 μM. Simply replacing the terminal hydroxyl with an amino group led to compound <b>20</b> possessing high potency for h-cGAS with an IC<sub>50</sub> value of 1.87 μM, which is more potent than the reference compound <b>5</b> (2.69 μM). Both compounds <b>20</b> and <b>5</b> showed much reduced m-cGAS activity with IC<sub>50</sub> values of 10.9 and 20 μM, respectively. Further masking the terminal amino group in compound <b>20</b> with one or two methyl yielded compounds <b>21</b> and <b>22</b>, showing less potent IC<sub>50</sub> values of 6.3 and 20 μM, respectively. Analysis of the potency between compounds <b>20–22</b> indicated that the H-bonding ability of the terminal group (as hydroxyl in <b>5</b> or amino in <b>20</b>) is critical for the cGAS interaction in the cell. All these compounds showed no pounced cytotoxicity in both human THP1-Dual and mouse RAW-Lucia ISG cells. Compounds with poor potency against h-cGAS were not tested further for their m-cGAS activity. Collectively, from the elaboration of the hydroxyacetyl side chain, compound <b>20</b> bearing a glycinoyl side chain turns out as the optimal one and shows IC<sub>50</sub> values of 1.87 and 10.9 μM, respectively, which are slightly more potent than the reference compound <b>5</b> with corresponding IC<sub>50</sub> values of 2.69 and 20 μM, respectively. Therefore, this compound was selected for further modification.</div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the acetylation of the terminal amino group within the glycinoyl side chain of <b>20</b> leads to compound <b>24</b> completely losing potency against h-cGAS. However, attaching an additional glycine moiety to the terminal amino moiety yielded compound <b>25</b> showing improved potency with an IC<sub>50</sub> of 1.38 μM, whereas again the potency against m-cGAS is much lower (11.4 μM) yet still comparable to that of <b>20</b> (10.9 μM). Further substitution of the second glycine component by a hydroxyl (<b>26</b>), <i>gem</i>-dimethyl (<b>27</b>), cyclopropyl (<b>29</b>), or pyrazolmethyl (<b>30</b>) led to much reduced potency against h-cGAS with IC<sub>50</sub> values of 8.2–13.5 μM. In addition, we also introduced heterocycle carbonyl as the <i>N</i>-substituent of the terminal amino of compound <b>20</b>, leading to compounds <b>31</b> and <b>32</b>, both retaining modest potency for h-cGAS with IC<sub>50</sub> values of 6.4 and 7.86 μM, respectively. Amidization of the terminal amino group in <b>20</b> with shikimic acid afforded compound <b>33</b> totally inactive for h-cGAS.</div><div class="NLM_p">Amides <b>34–35</b> and <b>37</b> represent a subseries of analogues bearing a terminal heteroaromatic acyl as the N-substituent of compound <b>20</b>. Again, these compounds retained modest potency against h-cGAS with IC<sub>50</sub> values of 7.17–10 μM. Compound <b>36</b> bearing a 2,2-difluoro-2-(<i>p</i>-tolyloxy)acetyl substituent showed much reduced potency (15.5 μM). Notably, compounds <b>34–37</b> exhibited slightly improved potency against m-cGAS with IC<sub>50</sub> values less than 10 μM, compared to that of compound <b>20</b> (10.88 μM).</div><div class="NLM_p">Next, we evaluated derivatives with the C9-heteroaryl group of THγC <b>5</b> walking to the C12 position and substituted by anilines. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, the substituent on the aniline ring plays a significant impact on h-cGAS potency. Compounds <b>43–46</b> obtained from anilines bearing a formyl substituent generally showed higher potency (1.14–9.59 μM) than compounds <b>47–50</b> (6.48–50 μM) derived from anilines bearing no formyl substituent. Compound <b>43</b> with 4-chloro-2-formyl phenylamino moiety showed the highest potency for h-cGAS with an IC<sub>50</sub> value of 1.14 μM without significant cytotoxicity (CC<sub>50</sub> = 12.5 μM), but the potency against m-cGAS was much reduced (16.3 μM). The migration of <i>para</i>-chloro to the meta-position (<b>44</b>) or replacement of <i>para</i>-chloro with <i>para</i>-trifluoromethyl (<b>46</b>) slightly decreased the potency against h-cGAS with IC<sub>50</sub> values of 2.87 and 2.72 μM, respectively. However, the removal of the <i>para</i>-chloro group (<b>45</b>) dramatically weakened the potency with an IC<sub>50</sub> value up to 9.8 μM. Deletion of the formyl group in <b>46</b> leading to compound <b>47</b> further impaired the potency for h-cGAS (6.48 μM). Nevertheless, compounds <b>51</b> and <b>52</b> with double <i>meta</i>-trifluoromethyl substituents on the arylamino component exhibited similarly high potency against h-cGAS with an identical IC<sub>50</sub> value of 3.0 μM, which is comparable to that of <b>46</b>.</div><div class="NLM_p">Compounds <b>59–64</b> represent another subseries of compounds bearing a C12-indolyl group on the central tricyclic skeleton (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). All these compounds exhibited modest potency against h-cGAS with IC<sub>50</sub> values ranging between 6.3 and 13 μM. The nonsubstituted indolyl analogues <b>56</b> and <b>60</b> are among the most potent with a similar potency of 6 μM, whereas the rest of the compounds in this series bearing substituents either electron-donating or -withdrawing gave similarly reduced potency.</div><div class="NLM_p">To evaluate the effects of the indolenine portion in the central tricyclic skeleton, compounds <b>59–60</b> were reduced to the corresponding indolines <b>65–66</b>. Interestingly, indolines <b>65–66</b> showed slightly improved potency against both h- and m-cGAS, with IC<sub>50</sub> values between 4.07 and 8.75 μM, and no cytotoxicity were observed in both cells, indicating the NH moiety in the central tricyclic core plays a role for cGAS interaction.</div><div class="NLM_p">Compounds <b>53–54</b> and <b>57</b> bearing complex structures due to further cyclization were evaluated as well. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, though a bearing unique structural framework, none of them displayed significant inhibitory activity against cGAS with IC<sub>50</sub> values greater than 20 μM, indicating the importance of the THγC skeleton for maintaining aromatic stacking interaction with cGAS.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Inhibitory Activity of Compounds against cGAS Activation by dsDNA<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0023.gif" alt="" id="gr18" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">CC<sub>50</sub> and IC<sub>50</sub> values were determined by the CCK-8 assay and the cell-based Lucia luciferase assay, respectively, with GraphPad Prism 8.0 software.</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Specificity of Selected Compounds for Inhibiting cGAS in THP1-Dual Cells</h3><div class="NLM_p">Compounds <b>20</b>, <b>25</b>, <b>43</b>, <b>46–47</b>, <b>52</b>, <b>60</b>, and <b>65–66</b> were selected for determining their specificity for cGAS interaction, due to their moderate to good cellular potency. We first pre-treated THP1-Dual cells with cGAMP to initiate STING-dependent activation of the cGAS–STING pathway and tested the effects of these new synthesized inhibitors on the IRF Lucia expression. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, compounds <b>20</b>, <b>25</b>, and <b>66</b> at 10 μM concentration have no significant suppression on the luciferase secretion of IRF induced by STING agonist cGAMP, suggesting that these compounds are not inhibitors of STING or its downstream signaling molecules, thus indirectly indicating that these compounds may interact with cGAS, the upstream molecule of STING. While compounds <b>43</b>, <b>46</b>, <b>47</b>, <b>52</b>, <b>60</b>, and <b>65</b> showed a somewhat inhibition on the luciferase secretion induced by STING activation at the same concentration, suggesting their off-target effects.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Fold change in the IRF Lucia expression of representative compounds in the cGAS signaling pathway of THP1-Dual cells by cGAMP stimulation. THP1-Dual cells were pre-incubated at 10 μM concentration of compounds for 1 h, then cells were transfected with cGAMP (cGAMP only initiates cGAS downstream STING-dependent signaling activation) for 24 h. Luciferase luminescence of the cell supernatant was measured. The fold change of luciferase activity for each compound-treated sample was calculated, namely, the fold change of Lucia expression = RLU sample/RLU positive control, where RLU indicates a raw luciferase unit. Data are given as mean ± SEM. Statistical analysis is performed by one-way analysis of variance (ANOVA). *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to cGAMP. <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, and <sup>###</sup><i>P</i> < 0.01 compared to blank. ns, no significance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Compounds <b>25</b> and <b>66</b> Dose Dependently Suppressed the dsDNA-Induced Activation of cGAS–STING Signaling</h3><div class="NLM_p">Among the three compounds (<b>20</b>, <b>25</b>, and <b>66</b>) likely interacting with cGAS, compounds <b>25</b> and <b>66</b> were selected for further target validation because they represent two different chemotypes. Because cGAS is the sensor of cytoplasmic dsDNA, its inhibitor should have inhibitory effects on the activation of the cGAS–STING pathway triggered by dsDNA, which is characterized by the significantly upregulated phosphorylation of the downstream STING/TBK1/IRF3 signaling. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, western blot analysis confirms that the newly designed compounds <b>25</b> and <b>66</b> remarkably inhibit STING phosphorylation at Ser366 in a dose-dependent manner. Besides, both compounds effectively suppressed the phosphorylation of downstream TBK1 and IRF3 (Ser386), confirming compounds <b>25</b> and <b>66</b> as potent cGAS inhibitors although they exhibited discriminative inhibitory activities against cGAS. These results were in agreement with those in the previous test as listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, in which compound <b>66</b> has a nearly 3-fold less potent inhibitory activity against h-cGAS than compound <b>25</b>.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compounds <b>25</b> and <b>66</b> inhibited the activation of the cGAS downstream signaling pathway. THP1-Dual cells were pre-treated with or without different concentrations of compounds for 1 h. Then, cells were transfected with dsDNA for 2 h. The expression levels of p-TBK1, TBK1, p-IRF3, IRF3, p-STING, and STING were determined by western blot analysis. The band intensity was quantified by ImageJ. Relative protein expression was displayed by the ratio of phosphorylated protein to total protein. Data are given as mean ± SEM. Statistical analysis is performed by one-way ANOVA. *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to dsDNA. <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, and <sup>###</sup><i>P</i> < 0.01 compared to blank. ns, no significance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> On-Target Engagement of Compound <b>25</b> in THP1-Dual Cells by Cellular Thermal Shift Assay</h3><div class="NLM_p">To further determine the direct effects of compounds on cGAS, the more potent compound <b>25</b> was selected for measuring its direct targeting on cGAS. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, a cellular thermal shift assay (CESTA)<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> was performed in THP1-Dual cells. Compared with the dimethyl sulfoxide (DMSO) control, treatment of cells with compound <b>25</b> stabilized the h-cGAS protein at its denaturation temperatures ranging from 53 to 56 °C, which demonstrated the direct interaction of compound <b>25</b> with h-cGAS in the intact cells.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. CESTA of compound <b>25</b> in THP1-Dual cells. The cells were incubated in the presence or absence of compound <b>25</b> (25 μM) for 1 h, then were heated at the indicated temperature (ranging from 37 to 63.1 °C) and cooled at room temperature. The cells were freeze–thawed and the cell lysate was centrifuged. The expression of cGAS and GAPDH in the supernatant was detected by the western blot assay, which implied the thermostability of cGAS following heat treatment. Also, the relative band intensity (%) was calculated, namely, relative band intensity (%) = band intensity of cGAS protein/band intensity of the GAPDH protein in every group at different temperatures, where the band intensity was quantified by ImageJ. Data are given as mean ± SEM. Statistical analysis is performed by Student’s <i>t</i>-test. *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to the DMSO group. ns, no significance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Effects of Compound <b>25</b> on the mRNA Expression of Downstream ISGs and Inflammatory Genes in dsDNA-Treated THP1-Dual Cells</h3><div class="NLM_p">The THP1-Dual cells bearing the IRF-luciferase reporter were used to determine the efficacy of compound <b>25</b> on the suppression of cGAS signaling. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, the exposure of dsDNA dramatically activated the luciferase signal, which is does dependently decreased by treating <b>25</b> at the concentrations ranging from 0.5 to 50 μM. Because the activation of the cGAS–STING signaling triggered by dsDNA is accompanied by a dramatic induction of downstream interferon-stimulated genes (ISGs) such as type I interferons and pro-inflammatory cytokine genes, accordingly, as shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B,C, we perform a quantitative reverse transcription-polymerase chain reaction (qRT-PCR) assay and found that the treatment of compound <b>25</b> dose dependently blocked the dsDNA-induced mRNA expression of <i>Il-6</i> and <i>Ifn-β</i>.</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Effect of <b>25</b> on dsDNA-induced luciferase activity, <i>Il-6</i> and <i>Ifn-β</i> gene expression in THP1-Dual cells. Data are given as mean ± SEM. Statistical analysis is performed by one-way ANOVA. *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to dsDNA. <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, and <sup>###</sup><i>P</i> < 0.01 compared to blank. ns, no significance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Compound <b>25</b> Showed Improved Aqueous Solubility</h3><div class="NLM_p">To investigate the suitability of compound <b>25</b> for an in vivo study, we converted it to the hydrochloride salt and tested its aqueous solubility. The reference compound <b>5</b> and the precursor compound <b>20</b> (as HCl salt as well) were tested as comparison. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, the aqueous solubility of compounds <b>20</b> and <b>25</b> in the HCl salt forms were determined to be 3.78 and 27.65 mg/mL, respectively. In contrast, the solubility of compound <b>5</b> was very low (around 0.0010 mg/mL). These results suggested that the introduction of an <i>N</i>-glycinoyl (as in <b>20</b>) or <i>N</i>-glycylglycinoyl (as in <b>25</b>) side chain is beneficial for improving the aqueous solubility of the THγC compounds.</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Aqueous solubility of compound <b>5</b> and the HCl salts of compounds <b>20</b> and <b>25</b>. The values are mean ± SEM of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> In Vivo Anti-inflammatory Activity of Compound <b>25</b></h3><div class="NLM_p">The accumulated evidence suggest that the cGAS–STING pathway is involved in LPS-induced acute lung injury (ALI) by modulating NLRP3 inflammasome and pyroptosis.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> LPS treatment upregulates the level of mitochondrial DNA in cytosol, which further activates and enhances the expression of the cGAS–STING signaling axis. Therefore, the inactivation of this signaling pathway can significantly alleviate LPS-induced ALI of mice.</div><div class="NLM_p">To confirm whether the in vitro cGAS inhibitory effects of compound <b>25</b> can convert to in vivo anti-inflammatory activity, we evaluated the effect of <b>25</b> on the release of pro-inflammatory cytokines in a LPS-induced inflammation model. The reported cGAS inhibitors <b>3</b> and <b>5</b> were used as comparison. BALB/c mice were intraperitoneally (i.p.) injected with compound <b>25</b> (30 mg/kg) 1 h before i.p. injection of a dose of 5 mg/kg of LPS. The serum levels of pro-inflammatory cytokines (TNF-α, IL-6, and IL-12) were determined by ELISA 1.5 h after LPS treatment, and the results are shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. For the vehicle group, LPS administration exhibited serious inflammation, accompanied by elevated serum levels of TNF-α, IL-6, and IL-12. Pre-treatment with the cGAS inhibitor <b>25</b> significantly inhibited the LPS-induced production of TNF-α, IL-6, and IL-12, which is 2.6-, 2.5-, and 7.6-fold lower, respectively. In general, the more potent cGAS inhibitor induced higher inhibition on the cytokine release, and compound <b>25</b> exhibited superior inhibitory effects in these models. These data clearly demonstrate the promising therapeutic potential of cGAS inhibitor <b>25</b> against inflammation diseases.</div><figure id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inflammatory cytokine concentrations in the serum of the LPS-induced mice model. BALB/c mice were intraperitoneally injected with compounds (30 mg/kg) and vehicle (10% DMSO + 90% corn oil) 1 h before an intraperitoneal injection with a dose of 5 mg/kg of LPS. The serum was collected 1.5 h after LPS application. The concentration of cytokines in the serum was detected by ELISA. Data are given as mean ± SEM. Statistical analysis is performed by one-way ANOVA. *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to vehicle. <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, and <sup>###</sup><i>P</i> < 0.01 compared to normal. ns, no significance. Normal, vehicle, and compound <b>3</b> groups: <i>n</i> = 6 mice per group, compounds <b>5</b> and <b>25</b> groups: <i>n</i> = 7 mice per group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Molecular Docking of Compounds <b>25</b> and <b>66</b> with h-cGAS</h3><div class="NLM_p">To rationale the potent in vitro and in vivo activities of the new cGAS inhibitor <b>25</b>, molecular docking with h-cGAS was performed in the ATP and GTP binding pockets of apo h-cGAS<sup>CD</sup> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJW">6MJW</a>) by using the induced fit docking module of Maestro Schrödinger software.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> The slightly less potent but structurally distinct compound <b>66</b> was docked as well for comparison. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>, both <b>25</b> and <b>66</b> fully occupy the catalytic binding pocket of h-cGAS. Consistent with the reported binding mode of compound <b>6</b> with h-cGAS (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> the tricyclic pyridoindole scaffold of <b>25</b> is centered between the guanidinium group of Arg376 and the aromatic ring of Tyr436, while its terminal <i>N</i>-glycinoyl side chain forms hydrogen bonds and salt bridges with Asp227 and Asp319 at a hydrophilic binding site. As a contrast, the pyridoindole core of <b>66</b> is partially stacked in the binding domain through the H bonding of the indolyl group with Asp227, which may explain its less potent activity. The <i>H</i>-bonding network formed between the hydroxyacetyl side chain and the residues of Hie217, Ser435, and Lys439 provides additional assistance in stabilizing this interaction of compound <b>66</b>.</div><figure id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Docking studies for compounds <b>25</b> and <b>66</b> within the ATP and GTP binding pockets of apo h-cGAS<sup>CD</sup> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJW">6MJW</a>). (A) Chemical structures of compounds <b>25</b> and <b>66</b>. (B) Superposition of the structures of compound <b>25</b> colored in cyan and of <b>66</b> in deep pink. (C) Docking mode of compound <b>25</b> with key intermolecular contacts. (D) Docking mode of compound <b>66</b> with key interactions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74635" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74635" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, the persistent activation of cGAS by dsDNA in the cytoplasm is closely implicated in the pathogens of many hyper-inflammatory and autoimmune diseases, and small molecules targeting cGAS have been proposed as a novel and promising therapeutic strategy for treating these diseases. However, current reported cGAS inhibitors are far beyond maturity with no or weak cellular potency. Inspired by the structural novelty of compound <b>5</b> (G140), we conducted a structural optimization campaign first by replacing or extending the terminal 2-hydroxyacetyl side chain, followed by substituting the C9-pyridinyl moiety with other groups, and walking the new substituent to the C12 position. Several subseries of compounds, including those unexpectedly cyclized ones with polycyclic complex structures, were obtained and evaluated for their inhibition against cGAS. Generally, these compounds possess moderate to good activity against h-cGAS but are much less potent for <i>m</i>-cGAS. Compound <b>25</b> bearing an <i>N</i>-glycylglycinoyl side chain was identified as the most potent with cellular IC<sub>50</sub> values of 1.38 μM for h-cGAS and 11.4 μM for m-cGAS, respectively, which are nearly 2-fold more potent than the reference compound <b>5</b> (2.69 and 20.0 μM, respectively). Mechanism studies confirmed that compound <b>25</b> directly interacts with cGAS and dramatically suppresses the dsDNA-induced phosphorylation of the downstream STING/TBK1/IRF3 signaling and the mRNA expression of the downstream ISGs. Further, compound <b>25</b> in the form of HCl salt has improved aqueous solubility and shows superior in vivo anti-inflammatory effects in the LPS-induced mouse model. Although further structural optimization is needed to achieve higher potency, the validation of compound <b>25</b> in both the cell and mouse modes provides solid confidence for the continuous pursuit of cGAS-targeting inhibitors to treat inflammatory diseases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84455" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84455" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Chemical Reagents and a General Method</h3><div class="NLM_p last">All anhydrous solvents, starting materials, and reagents were purchased from commercial sources and used without any further purification unless otherwise specified. All the reactions were performed under a N<sub>2</sub> atmosphere in dry glassware with magnetic stirring. Column chromatography was performed using 300–400 mesh silica gel, and analytical thin-layer chromatography (TLC) was carried out employing silica gel plates. Unless otherwise noted, nuclear magnetic resonance (NMR) spectra were recorded at ambient temperature on a Varian-MERCURY Plus-400 spectrometer, Bruker AVANCE III 500 spectrometer, Bruker Avance III 600 spectrometer, or Bruker Avance NEO 700 spectrometer. Mass spectra were obtained on a Finnigan LTQ in electrospray ionization (ESI) mode, while high-resolution mass spectra were recorded on an Agilent G6520 Q-TOF mass spectrometer. The purity of the final compounds was detected using analytical high-performance liquid chromatography (HPLC; Agilent Technologies 1260 Series) with UV detection at 254 nm. HPLC analysis conditions are as follows: XDB-C18 (3.5 μm, 4.6 mm × 150 mm); flow rate 1.00 mL/min; and 30–90% MeOH in water or 30–90% MeCN and 0.1% diethylamine (DEA) in water. All of the final biologically evaluated compounds have confirmed purity greater than 95%. The presence of two sets of peaks for some protons and carbons in the <sup>1</sup>H and <sup>13</sup>C NMR spectra is due to the rotation of the C–N bond within the side chain of the compound, which was demonstrated by variable-temperature NMR experiments at three different temperatures (see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_001.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Synthesis of Compounds <b>14–22</b></h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>tert</i>-Butyl (2-(9-Bromo-6,7-dichloro-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)carbamate (<b>12c</b>)</h4><div class="NLM_p last">To a mixture of intermediate <b>11</b><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (1 g, 3.14 mmol, 1 equiv) and Boc-glycine (0.445 g, 3.77 mmol, 1.2 equiv) in dry dimethylformamide (DMF) (10 mL) was added <i>N</i>-(3-dimethylaminopropyl)-<i>N</i>′-ethylcarbodiimide hydrochloride (EDCI) (0.899 g, 4.70 mmol, 1.5 equiv), 1-hydroxybenzotriazole (HOBt) (0.633 g, 4.69 mmol, 1.5 equiv), and DIPEA (1.21 g, 9.38 mmol, 3 equiv). The mixture was stirred at room temperature overnight, diluted with H<sub>2</sub>O, and then extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a residue, which was purified by column chromatography (SiO<sub>2</sub>, DCM/7 N ammonia in MeOH = 500/1 to 200/1) to give title compound <b>12c</b> as a white solid (0.67 g, 1.41 mmol). Yield: 44.9%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.86 (d, <i>J</i> = 14.1 Hz, 1H), 7.42 (d, <i>J</i> = 3.8 Hz, 1H), 6.82 (d, <i>J</i> = 20.3 Hz, 1H), 4.88 (d, <i>J</i> = 10.4 Hz, 2H), 3.92 (s, 2H), 3.79 (d, <i>J</i> = 34.4 Hz, 2H), 2.90–2.88 (m, 1H), 2.75–2.71 (m, 1H), 1.42–1.35 (m, 9H).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>tert</i>-Butyl (2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)carbamate (<b>13c</b>)</h4><div class="NLM_p last">To intermediate <b>12c</b> (0.67 g, 1.41 mmol, 1 equiv) and 1-methyl-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl) pyrazole (0.59 g, 2.82 mmol, 2 equiv) in dioxane (20 mL) were added KOAc (0.41 g, 4.23 mmol, 3 equiv) and Pd(dppf)Cl<sub>2</sub> (0.207 g, 0.282 mmol, 0.2 equiv). The mixture was stirred at 100 °C for 6 h and concentrated under reduced pressure to give a residue. The residue was purified by column chromatography (SiO<sub>2</sub>, DCM/7 N ammonia in MeOH = 500/1 to 200/1) to give title compound <b>13c</b> as a yellow solid (0.45 g, 1.41 mmol). Yield: 66.9%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.37 (d, <i>J</i> = 5.2 Hz, 1H), 7.42 (dd, <i>J</i> = 12.7, 2.2 Hz, 1H), 7.28 (d, <i>J</i> = 9.5 Hz, 1H), 6.44 (dd, <i>J</i> = 14.1, 2.2 Hz, 1H), 5.61–5.52 (m, 1H), 4.67 (d, <i>J</i> = 29.0 Hz, 2H), 4.05 (s, 2H), 3.98 (d, <i>J</i> = 3.2 Hz, 3H), 3.96 (s, 1H), 3.70 (t, <i>J</i> = 5.8 Hz, 1H), 2.89 (t, <i>J</i> = 5.7 Hz, 2H), 1.44 (d, <i>J</i> = 1.7 Hz, 9H).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2-Amino-1-(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)ethan-1-one (<b>20</b>)</h4><div class="NLM_p">Compound <b>13c</b> (110 mg, 0.29 mmol, 1 equiv) was dissolved in DCM (2 mL) and then TFA (0.5 mL) was added. The resulting mixture was stirred at room temperature until the starting material was consumed completely. The organic phase was washed with saturated NaHCO<sub>3</sub> and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by column chromatography (SiO<sub>2</sub>, DCM/7 N ammonia in MeOH = 200/1 to 50/1) to give target compound <b>20</b> as a white solid (77.3 mg, 0.20 mmol). Yield: 70.6%. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.57 (d, <i>J</i> = 29.2 Hz, 1H), 7.80 (dd, <i>J</i> = 4.0, 2.2 Hz, 1H), 7.25 (d, <i>J</i> = 27.8 Hz, 1H), 6.59 (dd, <i>J</i> = 35.0, 2.3 Hz, 1H), 4.59 (d, <i>J</i> = 7.1 Hz, 2H), 3.95 (d, <i>J</i> = 22.6 Hz, 3H), 3.83 (t, <i>J</i> = 5.9 Hz, 1H), 3.68 (t, <i>J</i> = 5.8 Hz, 1H), 3.42 (s, 1H), 3.33 (d, <i>J</i> = 6.3 Hz, 3H), 2.89 (t, <i>J</i> = 5.8 Hz, 1H), 2.82 (t, <i>J</i> = 5.9 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.84, 171.53, 148.82, 148.70, 136.23, 135.74, 134.32, 134.14, 131.80, 131.66, 125.88, 125.74, 123.13, 122.58, 122.53, 122.41, 120.38, 120.27, 112.92, 112.82, 108.27, 107.62, 105.03, 105.00, 43.23, 43.05, 42.72, 41.51, 40.66, 38.66, 38.61, 38.38, 23.96, 23.06. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 378.1. HRMS (ESI) calcd for C<sub>17</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup>, 378.0883; found, 378.0893. HPLC purity: 99.1%.</div><div class="NLM_p last">Compounds <b>14–18</b> and <b>21</b> were synthesized according to a similar protocol as that used for <b>20</b>.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-morpholinoethan-1-one (<b>14</b>)</h4><div class="NLM_p last">Brown solid; yield: 30.6%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.68 (d, <i>J</i> = 10.3 Hz, 1H), 7.18 (d, 1H), 6.47 (d, <i>J</i> = 15.3 Hz, 1H), 4.64 (s, 1H), 4.52 (s, 1H), 3.99 (d, <i>J</i> = 6.5 Hz, 3H), 3.95–3.89 (m, 2H), 3.73–3.64 (m, 2H), 3.51–3.41 (m, 2H), 3.27 (s, 1H), 3.14 (s, 1H), 3.05–2.99 (m, 1H), 2.89 (t, <i>J</i> = 5.6 Hz, 1H), 2.49 (s, 2H), 2.32 (s, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 170.64, 170.56, 151.64, 151.42, 136.78, 136.27, 136.06, 136.04, 132.94, 126.70, 126.52, 125.11, 124.96, 124.90, 124.74, 122.51, 115.42, 109.48, 109.00, 106.90, 106.75, 67.85, 67.65, 62.56, 62.38, 54.67, 54.44, 46.11, 44.08, 42.83, 40.67, 39.17, 39.01, 25.38, 24.18. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 448.2. HRMS (ESI) calcd for C<sub>21</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 448.1302; found, 448.1310. HPLC purity: 99.4%.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>R</i>)-6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-2-prolyl-2,3,4,5-tetrahydro-1<i>H</i>-pyrido[4,3-<i>b</i>]indole (<b>15</b>)</h4><div class="NLM_p last">White solid; yield: 29.1%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.69 (d, <i>J</i> = 15.3 Hz, 1H), 7.19 (d, 1H), 6.48 (d, <i>J</i> = 16.8 Hz, 1H), 4.71–4.47 (m, 2H), 4.06 (m, 0.6H), 4.00 (d, 3H), 3.98–3.80 (m, 2.4H), 3.35 (s, 1H), 3.17–3.09 (m, 1H), 2.98 (m, 1H), 2.91 (m, 1H), 2.79 (m, 1H), 2.33–1.99 (m, 1H), 1.89–1.49 (m, 3H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 174.13, 173.94, 151.58, 151.38, 136.94, 136.05, 135.92, 133.01, 132.93, 126.71, 126.56, 125.01, 124.94, 124.44, 122.58, 122.48, 115.40, 115.33, 108.91, 108.61, 106.83, 106.77, 59.52, 59.17, 48.01, 45.55, 43.25, 43.20, 41.02, 39.18, 39.02, 31.85, 31.61, 27.21, 27.04, 25.22, 24.13. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 418.2. HRMS (ESI) calcd for C<sub>20</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup>, 418.1196; found, 418.1201. HPLC purity: 95.9%.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (3-Aminooxetan-3-yl)(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)methanone (<b>16</b>)</h4><div class="NLM_p last">White solid; yield: 42.1%.<sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.55 (d, <i>J</i> = 5.9 Hz, 1H), 7.78 (d, 1H), 7.26–7.19 (m, 1H), 6.54 (s, 1H), 4.87 (d, <i>J</i> = 5.9 Hz, 1H), 4.71 (d, <i>J</i> = 5.9 Hz, 1H), 4.55 (s, 1H), 4.44–4.40 (m, 1H), 4.38 (d, <i>J</i> = 6.0 Hz, 1H), 4.20 (d, <i>J</i> = 6.1 Hz, 1H), 3.94 (d, 3H), 3.80 (t, <i>J</i> = 5.5 Hz, 1H), 3.53 (t, <i>J</i> = 5.1 Hz, 1H), 2.89 (s, 1H), 2.82 (s, 1H), 2.66 (s, 1H), 2.45 (s, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.96, 170.89, 148.69, 136.20, 135.69, 134.20, 131.69, 131.64, 125.88, 125.82, 123.17, 122.73, 122.47, 122.40, 120.30, 112.91, 112.82, 108.02, 107.75, 105.15, 105.06, 81.49, 81.24, 59.72, 59.67, 45.03, 42.13, 41.60, 38.83, 38.78, 38.64, 23.83, 23.04. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 420.2. HRMS (ESI) calcd for C<sub>19</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 420.0989; found, 420.0989. HPLC purity: 99.0%.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)(3-hydroxycyclobutyl)methanone (<b>17</b>)</h4><div class="NLM_p last">Light brown solid; yield: 55.3%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.69 (dd, <i>J</i> = 2.2 Hz, 1H), 7.18 (d, <i>J</i> = 6.3 Hz, 1H), 6.48 (dd, <i>J</i> = 2.3 Hz, 1H), 4.52 (s, 1H), 4.44 (s, 1H), 4.19–4.06 (m, 1H), 4.00 (d, <i>J</i> = 10.8 Hz, 3H), 3.91 (t, <i>J</i> = 5.9 Hz, 1H), 3.79 (t, <i>J</i> = 5.8 Hz, 1H), 2.92 (d, <i>J</i> = 6.2 Hz, 1H), 2.87 (t, <i>J</i> = 5.9 Hz, 1H), 2.72 (t, <i>J</i> = 8.9 Hz, 1H), 2.58–2.44 (m, 1H), 2.41–2.34 (m, 1H), 2.13–1.99 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 174.99, 174.78, 151.52, 151.28, 136.79, 135.90, 132.85, 132.77, 130.73, 130.65, 126.61, 126.36, 124.98, 124.81, 124.77, 124.52, 122.49, 122.36, 115.30, 115.19, 109.05, 108.76, 106.83, 106.65, 63.33, 63.12, 45.45, 43.33, 42.80, 40.47, 39.00, 38.89, 37.37, 37.16, 29.56, 28.94, 27.99, 26.78, 25.27, 24.02. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 419.2. HRMS (ESI) calcd for C<sub>20</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 419.1036; found, 419.1045. HPLC purity: 97.0%.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)(3-(difluoromethoxy)cyclobutyl)methanone (<b>18</b>)</h4><div class="NLM_p last">Off-white solid; yield: 52.7%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, 1H), 7.43 (d, <i>J</i> = 33.1 Hz, 1H), 7.27 (d, 1H), 6.44 (d, <i>J</i> = 24.0 Hz, 1H), 6.14 (t, <i>J</i> = 74.2 Hz, 1H), 4.66 (s, 1H), 4.64–4.51 (m, 1H), 4.50 (s, 1H), 4.03–3.97 (m, 3H), 3.96–3.67 (m, 2H), 2.87 (s, 2H), 2.80–2.71 (m, 1H), 2.63–2.47 (m, 2H), 2.47–2.43 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.08, 171.59, 150.41, 149.84, 134.68, 134.62, 132.67, 131.08, 130.97, 126.31, 125.50, 124.84, 123.57, 122.89, 122.61, 122.43, 115.66 (t, <i>J</i><sub><i>CF</i></sub> = 262.1 Hz), 114.38, 114.01, 109.95, 109.00, 105.95, 105.63, 63.99, 63.94, 63.89, 63.87, 63.82, 63.77, 44.62, 42.17, 42.13, 39.30, 39.27, 39.22, 34.24, 34.12, 29.85, 29.35, 24.79, 23.53. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 469.3. HRMS (ESI) calcd for C<sub>21</sub>H<sub>21</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 469.1004; found, 469.1017. HPLC purity: 98.2%.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 1-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-(methylamino)ethan-1-one (<b>21</b>)</h4><div class="NLM_p last">Light tawny solid; yield: 44.9%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.67 (dd, <i>J</i> = 9.9 Hz, 1H), 7.19 (d, 1H), 6.48 (dd, <i>J</i> = 19.0 Hz, 1H), 4.57 (t, <i>J</i> = 1.6 Hz, 1H), 4.54 (t, <i>J</i> = 1.5 Hz, 1H), 4.00 (d, 3H), 3.94 (t, <i>J</i> = 5.9 Hz, 1H), 3.77 (t, <i>J</i> = 5.8 Hz, 1H), 3.74 (s, 1H), 3.61 (s, 1H), 2.98 (t, <i>J</i> = 5.9 Hz, 1H), 2.90 (t, <i>J</i> = 5.9 Hz, 1H), 2.50 (d, 3H). <sup>13</sup>C NMR (151 MHz, CD<sub>3</sub>OD): δ 168.92, 168.28, 151.60, 151.40, 136.84, 136.14, 136.03, 135.90, 133.09, 132.97, 126.70, 126.55, 125.08, 125.02, 124.96, 124.31, 122.66, 122.54, 115.40, 115.36, 108.75, 108.23, 106.75, 106.71, 51.82, 51.51, 44.96, 42.90, 42.84, 40.47, 39.18, 39.01, 35.04, 34.91, 24.92, 24.11. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 392.3. HRMS (ESI) calcd for C<sub>18</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup>, 392.1039; found, 392.1043. HPLC purity: 95.0%.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 1,1,1,3,3,3-Hexafluoropropan-2-yl 9-Bromo-6,7-dichloro-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indole-2-carboxylate (<b>12h</b>)</h4><div class="NLM_p last">A round-bottomed flask was charged with 1,1,1,3,3,3-hexafluoropropan-2-ol (168 mg, 1.00 mmol, 1 equiv), triphosgene (89.1 mg, 0.33 mmol, 0.33 equiv), DCM (10 mL), and DIPEA (387 mg, 3.00 mmol, 3 equiv). The mixture was stirred at room temperature for 1 h, and then intermediate <b>11</b> (212 mg, 0.67 mmol, 0.67 equiv) was added. The resulting mixture was stirred at room temperature for 2 h and diluted with DCM (20 mL). The mixture was washed with H<sub>2</sub>O, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by column chromatography (SiO<sub>2</sub>, petroleum ether/EtOAc = 100/1 to 10/1) to give title compound <b>12h</b> (134 mg, 0.26 mmol). Yield: 39.3%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.24 (s, 1H), 7.35 (d, <i>J</i> = 3.4 Hz, 1H), 5.81 (dt, <i>J</i> = 8.6, 6.0 Hz, 1H), 5.06 (d, <i>J</i> = 1.7 Hz, 2H), 3.92 (td, <i>J</i> = 5.9, 3.4 Hz, 2H), 3.01–2.85 (m, 2H).</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 1,1,1,3,3,3-Hexafluoropropan-2-yl 6,7-Dichloro-9-(1-methyl-1H-pyrazol-3-yl)-1,3,4,5-tetrahydro-2H-pyrido[4,3-<i>b</i>]indole-2-carboxylate (<b>19</b>)</h4><div class="NLM_p last">This compound was prepared from <b>12h</b> following a similar procedure as that for the preparation of compound <b>13c</b> as a white solid. Yield: 84.5%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.30 (d, <i>J</i> = 7.6 Hz, 1H), 7.42 (d, 1H), 7.28 (d, <i>J</i> = 3.4 Hz, 1H), 6.43 (s, 1H), 5.86–5.73 (m, 1H), 4.70 (d, <i>J</i> = 16.7 Hz, 2H), 4.03–3.93 (m, 3H), 3.89 (q, <i>J</i> = 5.2 Hz, 2H), 2.98–2.90 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 152.43, 151.66, 149.80, 134.74, 134.66, 133.23, 133.10, 131.18, 130.91, 125.93, 125.79, 125.04, 123.10, 123.07, 122.50, 122.01 (br s), 119.77 (br s), 114.25, 109.02, 108.40, 105.70, 105.57, 68.23 (m), 44.87, 43.78, 41.87, 41.54, 39.27, 38.82, 24.03, 23.55. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 515.2. HRMS (ESI) calcd for C<sub>19</sub>H<sub>15</sub>Cl<sub>2</sub>F<sub>6</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 515.0471; found, 515.0480. HPLC purity: 99.7%.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 1-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-(dimethylamino)ethan-1-one (<b>22</b>)</h4><div class="NLM_p last">To a solution of compound <b>20</b> (50 mg, 0.13 mmol, 1 equiv) in methanol (1.5 mL) were added formaldehyde solution (37% in H<sub>2</sub>O, 12 mg, 0.13 mmol, 1 equiv), HOAc (8 mg, 0.13 mmol, 1 equiv), and NaBH<sub>3</sub>CN (13 mg, 0.20 mmol, 1.5 equiv). The mixture was stirred at room temperature overnight, quenched with saturated NaHCO<sub>3</sub>, and then extracted with EtOAc. The organic phase was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by column chromatography (DCM/7 N ammonia in MeOH = 300/1 to 100/1) to give the target compound <b>22</b> as a pale-yellow solid (31.6 mg, 0.078 mmol). Yield: 60.0%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.68 (dd, <i>J</i> = 11.8 Hz, 1H), 7.18 (d, 1H), 6.47 (dd, <i>J</i> = 12.1 Hz, 1H), 4.58 (s, 1H), 4.52 (s, 1H), 4.00 (d, 3H), 3.92 (t, <i>J</i> = 6.0 Hz, 1H), 3.88 (t, <i>J</i> = 5.7 Hz, 1H), 3.29 (s, 1H), 3.15 (s, 1H), 3.00–2.95 (m, 1H), 2.88 (d, <i>J</i> = 6.1 Hz, 1H), 2.29 (s, 3H), 2.22 (s, 3H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 170.51, 170.38, 151.25, 151.09, 136.49, 135.92, 135.72, 135.68, 132.59, 126.36, 126.21, 124.79, 124.60, 124.55, 124.39, 122.18, 122.17, 115.07, 114.98, 108.66, 108.65, 106.56, 106.43, 62.35, 62.19, 45.45, 45.40, 43.46, 42.42, 40.03, 38.79, 38.67, 24.85, 23.85. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 406.2. HRMS (ESI) calcd for C<sub>19</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup>, 406.1196; found, 406.1197. HPLC purity: 96.7%.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Synthetic Procedures of Compounds <b>24–37</b></h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>tert</i>-Butyl (2-((2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)amino)-2-oxoethyl)carbamate (<b>23a</b>)</h4><div class="NLM_p last">To the mixture of compound <b>20</b> (1 g, 2.65 mmol, 1 equiv) and Boc-glycine (0.56 g, 3.18 mmol, 1.2 equiv) in dry DMF (10 mL) were added 2-(7-aza-1<i>H</i>-benzotriazole-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate (HATU) (2.02 g, 5.31 mmol, 2 equiv) and DIPEA (0.68 g, 5.31 mmol, 2 equiv). The mixture was stirred at room temperature overnight, diluted with H<sub>2</sub>O, and then extracted with EtOAc. The combined organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a residue. The residue was purified by column chromatography (SiO<sub>2</sub>, DCM/7 N ammonia in MeOH = 500/1 to 200/1) to give title compound <b>23a</b> as a white solid (0.87 g, 1.62 mmol). Yield: 61.2%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.66 (d, <i>J</i> = 8.7 Hz, 1H), 7.18 (d, <i>J</i> = 15.8 Hz, 1H), 6.47 (d, <i>J</i> = 18.7 Hz, 1H), 4.56 (d, <i>J</i> = 11.2 Hz, 2H), 4.14 (d, <i>J</i> = 34.7 Hz, 2H), 4.01 (d, <i>J</i> = 24.6 Hz, 3H), 3.93 (t, <i>J</i> = 5.9 Hz, 1H), 3.80 (d, <i>J</i> = 4.3 Hz, 1H), 3.75 (s, 2H), 2.98 (t, <i>J</i> = 6.0 Hz, 1H), 2.89 (t, <i>J</i> = 5.1 Hz, 1H), 1.44 (d, <i>J</i> = 5.9 Hz, 9H).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-Amino-<i>N</i>-(2-(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)acetamide (<b>25</b>)</h4><div class="NLM_p">To a solution of compound <b>23a</b> (110 mg, 0.21 mmol, 1 equiv) dissolved in DCM (2 mL) was added TFA (0.5 mL). The mixture was stirred at room temperature until the starting material was consumed completely. The organic phase was washed with saturated NaHCO<sub>3</sub> and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by column chromatography (SiO<sub>2</sub>, DCM/7 N ammonia in MeOH = 200/1 to 30/1) to give target compound <b>25</b> as a white solid (61.0 mg, 0.14 mmol). Yield: 66.9%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.66 (dd, <i>J</i> = 9.4, 1H), 7.18 (d, 1H), 6.47 (dd, <i>J</i> = 20.9 Hz, 1H), 4.59 (s, 1H), 4.55 (s, 1H), 4.21 (s, 1H), 4.13 (s, 1H), 4.00 (d, 3H), 3.92 (t, <i>J</i> = 5.8 Hz, 1H), 3.80 (t, <i>J</i> = 5.8 Hz, 1H), 3.44 (d, 1H), 3.35 (s, 1H), 2.98 (t, <i>J</i> = 5.9 Hz, 1H), 2.89 (t, <i>J</i> = 6.0 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 173.50, 173.49, 169.29, 169.06, 151.50, 151.38, 136.75, 136.11, 135.99, 135.96, 133.01, 132.86, 126.67, 126.58, 125.01, 124.97, 124.92, 124.32, 122.57, 122.46, 115.32, 108.85, 108.23, 106.74, 106.65, 44.94, 44.17, 44.11, 43.09, 42.93, 42.29, 41.96, 40.49, 39.26, 39.00, 24.92, 24.06. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 435.2. HRMS (ESI) calcd for C<sub>19</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 435.1098; found, 435.1095. HPLC purity: 100.0%.</div><div class="NLM_p last">Compounds <b>24</b> and <b>26</b>–<b>37</b> were synthesized according to a similar protocol as that for <b>25</b>.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-b]indol-2-yl)-2-oxoethyl)acetamide (<b>24</b>)</h4><div class="NLM_p last">Yellow solid; yield: 23.4%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.41 (d, 1H), 7.43 (dd, <i>J</i> = 11.6 Hz, 1H), 7.29 (d, <i>J</i> = 9.4 Hz, 1H), 6.69–6.62 (m, 1H), 6.45 (dd, <i>J</i> = 16.2 Hz, 1H), 4.72 (d, <i>J</i> = 4.4 Hz, 2H), 4.14 (d, <i>J</i> = 3.9 Hz, 2H), 4.10 (s, 2.4H), 3.99 (s, 0.6H), 4.00–3.97 (t, 1.4H), 3.72 (t, <i>J</i> = 5.8 Hz, 0.6H), 2.91 (t, <i>J</i> = 5.9 Hz, 2H), 2.04 (d, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 170.40, 170.28, 167.57, 166.69, 150.02, 149.89, 134.81, 134.71, 133.93, 132.61, 131.37, 131.03, 126.11, 125.83, 125.01, 123.28, 122.62, 122.53, 122.45, 114.19, 114.14, 109.32, 108.27, 105.58, 105.44, 43.84, 42.30, 42.13, 41.73, 41.49, 39.48, 39.25, 39.21, 24.32, 23.44, 23.18. MS (ESI, [M + Na]<sup>+</sup>) <i>m</i>/<i>z</i>: 442.2. HRMS (ESI) calcd for C<sub>19</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 418.0843; found, 418.0836. HPLC purity: 95.3%.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>S</i>)-2-Amino-<i>N</i>-(2-(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-3-hydroxypropanamide (<b>26</b>)</h4><div class="NLM_p last">Off-white solid; yield: 30.3%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.66 (dd, <i>J</i> = 9.7 Hz, 1H), 7.18 (d, 1H), 6.46 (dd, <i>J</i> = 21.3 Hz, 1H), 4.56 (d, <i>J</i> = 13.8 Hz, 2H), 4.28–4.18 (m, 1H), 4.17–4.07 (m, 1H), 4.00 (d, 3H), 3.92 (t, <i>J</i> = 5.9 Hz, 1H), 3.80 (t, <i>J</i> = 5.8 Hz, 1H), 3.74 (d, <i>J</i> = 5.2 Hz, 2H), 3.63–3.56 (m, 1H), 2.99 (t, <i>J</i> = 5.9 Hz, 1H), 2.89 (t, <i>J</i> = 5.8 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 174.37, 174.34, 169.27, 169.05, 151.50, 151.38, 136.75, 136.11, 135.99, 135.97, 133.02, 132.88, 126.67, 126.59, 125.02, 124.98, 124.93, 124.34, 122.58, 122.46, 115.34, 108.83, 108.20, 106.75, 106.66, 64.63, 57.48, 44.93, 43.11, 42.94, 42.42, 42.12, 40.54, 39.28, 39.01, 24.92, 24.07. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 465.2. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 465.2. HRMS (ESI) calcd for C<sub>20</sub>H<sub>23</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 465.1203; found, 465.1217. HPLC purity: 95.0%.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-Amino-<i>N</i>-(2-(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-2-methylpropanamide (<b>27</b>)</h4><div class="NLM_p last">White solid; yield: 60.7%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.67 (dd, <i>J</i> = 10.4 Hz, 1H), 7.19 (d, 1H), 6.48 (dd, <i>J</i> = 20.6 Hz, 1H), 4.59 (s, 1H), 4.56 (s, 1H), 4.16 (s, 1H), 4.07 (s, 1H), 4.01 (d, 3H), 3.94 (t, <i>J</i> = 5.9 Hz, 1H), 3.82 (t, <i>J</i> = 5.8 Hz, 1H), 2.99 (t, <i>J</i> = 5.8 Hz, 1H), 2.90 (t, <i>J</i> = 5.9 Hz, 1H), 1.32 (d, 6H). <sup>13</sup>C NMR (151 MHz, CD<sub>3</sub>OD): δ 180.46, 180.37, 169.41, 169.14, 151.51, 151.40, 136.79, 136.14, 136.01, 133.05, 132.91, 126.70, 126.62, 125.03, 124.94, 124.39, 122.59, 122.49, 115.35, 115.34, 108.88, 108.26, 106.77, 106.66, 55.75, 55.73, 44.89, 43.05, 42.91, 42.52, 42.15, 40.51, 39.29, 39.01, 28.37, 28.31, 24.94, 24.09. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 463.3. HRMS (ESI) calcd for C<sub>21</sub>H<sub>25</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 463.1411; found, 463.1409. HPLC purity: 96.1%.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>tert</i>-Butyl (1-((2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)amino)-2-methyl-1-oxopropan-2-yl)carbamate (<b>28</b>)</h4><div class="NLM_p last">Yellow solid; yield: 42.2%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.65 (d, 1H), 7.41 (dd, <i>J</i> = 14.3 Hz, 1H), 7.26 (d, 1H), 6.43 (dd, <i>J</i> = 16.6 Hz, 1H), 5.04 (s, 1H), 4.68 (d, <i>J</i> = 7.7 Hz, 2H), 4.12 (d, 2H), 4.03 (d, <i>J</i> = 46.2 Hz, 3H), 3.95 (d, <i>J</i> = 5.9 Hz, 1H), 3.68 (t, <i>J</i> = 5.8 Hz, 1H), 2.88 (t, <i>J</i> = 5.9 Hz, 2H), 1.86 (s, 1H), 1.50 (d, 6H), 1.41 (d, 9H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 175.03, 174.95, 167.60, 166.76, 154.66, 153.67, 150.08, 150.03, 134.88, 134.80, 134.28, 133.06, 131.41, 131.13, 127.43, 126.08, 125.87, 124.95, 124.92, 123.42, 122.73, 122.55, 122.39, 114.27, 114.23, 109.21, 108.22, 105.68, 105.54, 56.81, 43.88, 42.35, 42.05, 41.57, 39.64, 39.34, 39.31, 28.57 (2C), 25.91 (3C), 24.36, 23.50. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 563.2. HRMS (ESI) calcd for C<sub>26</sub>H<sub>33</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 563.1935; found, 563.1924. HPLC purity: 95.5%.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 1-Amino-<i>N</i>-(2-(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)cyclopropane-1-carboxamide (<b>29</b>)</h4><div class="NLM_p last">Yellow solid; yield: 56.9%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.49 (d, 1H), 8.45 (s, 1H), 7.42 (dd, <i>J</i> = 9.8 Hz, 1H), 7.29 (s, 1H), 6.44 (dd, <i>J</i> = 13.9 Hz, 1H), 4.70 (d, <i>J</i> = 10.1 Hz, 2H), 4.17–4.13 (m, 2H), 4.07 (s, 2H), 4.02–3.97 (m, 2.4H), 3.73 (t, <i>J</i> = 5.8 Hz, 0.6H), 2.91 (t, <i>J</i> = 5.8 Hz, 2H), 1.44–1.39 (m, 2H), 0.88–0.82 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 175.66, 175.55, 167.93, 167.17, 150.00, 149.90, 134.78, 134.68, 134.18, 132.77, 131.30, 131.00, 126.12, 125.80, 124.93, 124.90, 123.39, 122.65, 122.51, 122.38, 114.17, 114.10, 109.42, 108.36, 105.61, 105.44, 43.89, 42.22, 42.13, 41.88, 41.58, 39.53, 39.27, 39.21, 35.78, 35.75, 24.37, 23.47, 19.59, 19.50. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 461.3. HRMS (ESI) calcd for C<sub>21</sub>H<sub>23</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 461.1254; found, 461.1249. HPLC purity: 95.4%.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 2-Amino-<i>N</i>-(2-(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-3-(1<i>H</i>-pyrazol-3-yl)propanamide (<b>30</b>)</h4><div class="NLM_p last">Yellow solid; yield: 53.4%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.70–7.64 (m, 2H), 7.19 (d, <i>J</i> = 16.1 Hz, 1H), 6.96 (s, 1H), 6.48 (dd, <i>J</i> = 20.3 Hz, 1H), 4.58 (d, <i>J</i> = 10.6 Hz, 2H), 4.22 (s, 1H), 4.12 (s, 1H), 4.00 (d, 3H), 3.94 (t, <i>J</i> = 6.0 Hz, 1H), 3.90 (t, <i>J</i> = 6.4 Hz, 1H), 3.82 (t, <i>J</i> = 5.9 Hz, 1H), 3.11 (m, 1H), 3.02–2.94 (m, 2H), 2.92–2.88 (m, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 173.76, 173.73, 169.26, 169.07, 151.51, 151.39, 136.78, 136.74, 136.72, 136.12, 136.00, 133.42, 133.06, 132.91, 126.67, 126.58, 125.02, 124.98, 124.93, 124.34, 122.59, 122.47, 119.15, 119.13, 119.04, 119.02, 115.35, 115.33, 108.84, 108.23, 106.75, 106.67, 55.33, 44.97, 43.12, 42.98, 42.38, 42.09, 40.57, 39.28, 39.02, 31.66, 31.59, 24.94, 24.08. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 515.3. HRMS (ESI) calcd for C<sub>23</sub>H<sub>25</sub>Cl<sub>2</sub>N<sub>8</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 515.1472; found 515.1474. HPLC purity: 95.0%.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-(2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-4,4-difluoropyrrolidine-2-carboxamide (<b>31</b>)</h4><div class="NLM_p last">Yellow solid; yield: 59.6%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.64 (dd, <i>J</i> = 9.5 Hz, 1H), 7.16 (d, 1H), 6.45 (dd, <i>J</i> = 20.3 Hz, 1H), 4.55 (d, <i>J</i> = 11.9 Hz, 2H), 4.17 (d, 1H), 4.08 (s, 1H), 3.99 (d, 3H), 3.95–3.89 (m, 2H), 3.78 (t, <i>J</i> = 5.8 Hz, 1H), 3.24–3.07 (m, 2H), 2.96 (t, <i>J</i> = 5.8 Hz, 1H), 2.87 (t, <i>J</i> = 5.9 Hz, 1H), 2.61–2.48 (m, 1H), 2.39–2.26 (m, 1H). <sup>13</sup>C NMR (151 MHz, CD<sub>3</sub>OD): δ 175.65, 175.54, 169.10, 168.81, 151.49, 151.37, 136.73, 136.12, 135.99, 135.97, 133.83, 133.03, 132.91, 132.19, 132.17, 130.53, 126.69, 126.60, 125.02, 124.99, 124.93, 124.33, 122.58, 122.47, 115.33, 115.31, 108.81, 108.19, 106.75, 106.64, 59.79 (m), 54.18 (t, <i>J</i><sub><i>CF</i></sub> = 28.7 Hz), 44.86, 43.06, 42.86, 42.25, 41.89, 40.47, 39.72 (td, <i>J</i><sub><i>CF</i></sub> = 25.7 Hz), 24.89, 24.06. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 511.3. HRMS (ESI) calcd for C<sub>22</sub>H<sub>23</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 511.1222; found, 511.1221. HPLC purity: 95.5%.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-(2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-2-(piperazin-1-yl)acetamide (<b>32</b>)</h4><div class="NLM_p last">Light brown solid; yield: 43.3%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.67 (dd, 1H), 7.19 (d, <i>J</i> = 17.4 Hz, 1H), 6.49 (dd, <i>J</i> = 19.4 Hz, 1H), 4.60 (s, 1H), 4.56 (s, 1H), 4.22 (s, 1H), 4.13 (s, 1H), 4.01 (d, 3H), 3.95 (t, <i>J</i> = 5.9 Hz, 1H), 3.82 (t, <i>J</i> = 5.8 Hz, 1H), 3.04 (d, <i>J</i> = 4.2 Hz, 2H), 3.00 (t, <i>J</i> = 5.8 Hz, 1H), 2.92–2.88 (m, 5H), 2.58–2.51 (m, 4H). <sup>13</sup>C NMR (151 MHz, CD<sub>3</sub>OD): δ 173.09, 172.95, 169.21, 168.89, 151.51, 151.41, 136.77, 136.15, 136.02, 133.02, 126.70, 126.64, 125.05, 125.02, 124.96, 124.37, 122.62, 122.50, 115.35, 108.87, 108.27, 106.78, 106.74, 106.67, 106.63, 62.57 (2C), 54.55, 54.49, 46.28, 46.22, 44.91, 43.07, 42.87, 42.12, 41.70, 40.53, 39.28, 39.02, 24.92, 24.09. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 504.4. HRMS (ESI) calcd for C<sub>23</sub>H<sub>28</sub>Cl<sub>2</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 504.1676; found, 504.1682. HPLC purity: 95.0%.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-(2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-3,4,5-trihydroxycyclohex-1-ene-1-carboxamide (<b>33</b>)</h4><div class="NLM_p last">Yellow solid; yield: 23.6%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.66 (dd, <i>J</i> = 12.8, 2.2 Hz, 1H), 7.18 (d, 1H), 6.47 (dd, <i>J</i> = 21.5 Hz, 2H), 4.60 (s, 1H), 4.55 (s, 1H), 4.34 (t, <i>J</i> = 4.1 Hz, 1H), 4.21 (s, 1H), 4.12 (s, 1H), 4.06–3.94 (m, 5H), 3.93 (t, <i>J</i> = 4.5 Hz, 1H), 3.83 (t, <i>J</i> = 5.7 Hz, 1H), 3.64 (dd, <i>J</i> = 7.8 Hz, 1H), 2.98 (t, <i>J</i> = 5.7 Hz, 1H), 2.89 (t, <i>J</i> = 5.9 Hz, 1H), 2.78–2.72 (m, 1H), 2.19 (dd, <i>J</i> = 17.8 Hz, 1H). <sup>13</sup>C NMR (151 MHz, CD<sub>3</sub>OD): δ 170.22, 170.08, 169.41, 169.19, 151.50, 151.38, 136.80, 136.11, 136.01, 135.99, 133.64, 133.58, 133.49, 133.47, 133.06, 132.91, 126.69, 126.60, 125.01, 124.92, 124.39, 122.58, 122.47, 115.34, 115.32, 108.90, 108.27, 106.75, 106.68, 73.09, 68.25, 67.37, 44.96, 43.10, 42.98, 42.61, 42.28, 40.55, 39.28, 39.01, 32.08, 32.07, 30.77, 30.47, 24.96, 24.09. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 534.3. HRMS (ESI) calcd for C<sub>24</sub>H<sub>26</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 534.1306; found, 534.1306. HPLC purity: 95.0%.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-(2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-2-(furan-2-yl)acetamide (<b>34</b>)</h4><div class="NLM_p last">Tawny solid; yield: 63.3%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.39 (s, 1H), 7.46–7.36 (m, 2H), 7.29 (d, 1H), 6.87 (d, <i>J</i> = 18.5 Hz, 1H), 6.44 (d, <i>J</i> = 19.0 Hz, 1H), 6.35 (d, <i>J</i> = 9.9 Hz, 1H), 6.25 (d, <i>J</i> = 8.5 Hz, 1H), 4.69 (s, 2H), 4.13 (d, <i>J</i> = 3.9 Hz, 2H), 4.09 (d, 3H), 3.96 (d, 1.5H), 3.71 (t, <i>J</i> = 5.4 Hz, 0.5H), 3.66 (s, 2H), 2.89 (t, <i>J</i> = 5.8 Hz, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 168.83, 168.70, 167.26, 166.33, 150.00, 149.91, 148.54, 148.44, 142.69, 134.79, 134.71, 133.92, 132.63, 131.34, 131.08, 126.07, 125.80, 124.99, 123.20, 122.60, 122.51, 122.42, 114.18, 114.09, 110.93, 109.34, 108.77, 108.72, 108.22, 105.52, 105.43, 43.80, 42.33, 42.14, 41.79, 41.48, 39.51, 39.25, 39.21, 36.26, 36.24, 24.32, 23.39. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 486.3. HRMS (ESI) calcd for C<sub>23</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 486.1094; found, 486.1093. HPLC purity: 98.5%.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-(2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-2-(5-methylisoxazol-3-yl)acetamide (<b>35</b>)</h4><div class="NLM_p last">Yellow solid; yield: 54.2%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.42 (d, 1H), 7.43 (dd, <i>J</i> = 10.8 Hz, 1H), 7.28 (d, <i>J</i> = 9.9 Hz, 1H), 6.99–6.90 (m, 1H), 6.45 (dd, <i>J</i> = 17.8 Hz, 1H), 6.08 (d, 1H), 4.71 (d, 2H), 4.15 (d, 2H), 4.08 (s, 2H), 3.99–3.96 (m, 2H), 3.80–3.69 (m, 3H), 2.93–2.88 (m, 2H), 2.27 (d, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 167.00, 166.64, 166.50, 166.18, 165.73, 165.64, 160.33, 150.02, 149.87, 134.79, 134.70, 133.94, 132.59, 131.34, 131.08, 126.08, 125.76, 125.01, 123.22, 122.59, 122.49, 122.42, 114.20, 114.11, 109.24, 108.14, 105.54, 105.46, 104.33, 104.31, 43.85, 42.37, 42.23, 41.87, 41.53, 39.50, 39.30, 39.26, 34.77, 34.76, 24.30, 23.39, 11.59. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 501.3.MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 501.3. HRMS (ESI) calcd for C<sub>23</sub>H<sub>23</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 501.1203; found, 501.1193. HPLC purity: 95.0%.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-(2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-2,2-difluoro-2-(<i>p</i>-tolyloxy)acetamide (<b>36</b>)</h4><div class="NLM_p last">Yellow solid; yield: 53.4%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.27 (d, 1H), 7.76 (d, <i>J</i> = 14.1 Hz, 1H), 7.49–7.42 (m, 1H), 7.31 (d, <i>J</i> = 11.2 Hz, 1H), 7.18–7.06 (m, 4H), 6.47 (dd, <i>J</i> = 22.0 Hz, 1H), 4.77 (s, 2H), 4.25 (d, <i>J</i> = 4.1 Hz, 2H), 4.13 (s, 2.4H), 4.02 (t, <i>J</i> = 6.0 Hz, 1.6H), 4.00 (s, 0.6H), 3.77 (t, <i>J</i> = 5.8 Hz, 0.4H), 3.00–2.92 (m, 2H), 2.34 (d, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 166.17, 165.34, 159.85, 159.55, 150.07, 149.86, 147.19, 136.28, 134.85, 133.73, 132.26, 131.41, 131.07, 130.19, 125.87, 125.19 (t, <i>J</i><sub><i>CF</i></sub> = 243.2 Hz), 122.78, 122.57, 122.45, 121.82, 114.69, 114.27, 109.33, 108.21, 105.60, 105.52, 43.93, 42.51, 42.09, 41.57, 39.58, 39.39, 39.29, 24.38, 23.43, 20.96. MS (ESI, [M + Na]<sup>+</sup>) <i>m</i>/<i>z</i>: 584.2. HRMS (ESI) calcd for C<sub>26</sub>H<sub>22</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>5</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 560.1073; found, 560.1057. HPLC purity: 99.4%.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-(2-(6,7-Dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)-3-(5-(trifluoromethyl)-1,2,4-oxadiazol-3-yl)benzamide (<b>37</b>)</h4><div class="NLM_p last">Dark-yellow solid; yield: 34.4%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.77 (d, 1H), 8.50 (d, <i>J</i> = 9.7 Hz, 1H), 8.18 (t, <i>J</i> = 7.3 Hz, 1H), 7.98 (d, <i>J</i> = 7.5 Hz, 1H), 7.57–7.51 (m, 2H), 7.41 (dd, <i>J</i> = 25.7 Hz, 1H), 7.22 (d, 1H), 6.42 (dd, <i>J</i> = 26.9 Hz, 1H), 4.72 (d, <i>J</i> = 15.3 Hz, 2H), 4.33 (dd, <i>J</i> = 10.9, 4.0 Hz, 2H), 4.12 (s, 2H), 3.97 (t, <i>J</i> = 5.7 Hz, 1.4H), 3.95 (s, 1H), 3.70 (t, <i>J</i> = 5.8 Hz, 0.6H), 2.87 (t, <i>J</i> = 5.8 Hz, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 168.63, 168.59, 167.43, 166.64, 166.18, 166.11 (q, <i>J</i><sub><i>CF</i></sub> = 43.8 Hz), 150.04, 149.86, 135.17, 134.92, 134.78, 134.66, 134.04, 132.82, 131.36, 131.05, 130.92, 130.84, 130.72, 130.64, 129.63, 126.43, 126.35, 125.97, 125.71, 125.54, 125.49, 124.90, 124.85, 123.22, 122.49, 122.27, 116.04 (q, <i>J</i><sub><i>CF</i></sub> = 274.8 Hz), 114.16, 114.08, 109.04, 108.07, 105.54, 105.46, 43.94, 42.52, 42.43, 42.14, 41.59, 39.52, 39.37, 39.22, 24.26, 23.38. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 618.3. HRMS (ESI) calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 618.1030; found, 618.1033. HPLC purity: 96.4%.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Synthetic Procedures of Compounds <b>43–55</b> and <b>57</b></h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 6,7-Dichloro-2,3,4,5-tetrahydro-1<i>H</i>-pyrido[4,3-<i>b</i>]indole (<b>39</b>)</h4><div class="NLM_p last">To a stirred solution of compound <b>38</b> (5 g, 23.42 mmol, 1 equiv) and piperidin-4-one hydrochloride (4.78 g, 35.13 mmol, 1.5 equiv) in dioxane (100 mL) was added conc. H<sub>2</sub>SO<sub>4</sub> (46 g, 468.4 mmol, 20 equiv). The mixture was stirred at 115 °C overnight and then cooled and concentrated under reduced pressure. The residue was adjusted to pH 8 with NaOH (3 N) and filtered. The filter cake was washed with MTBE (50 mL) and dried under an infrared-ray oven to obtain title compound <b>39</b> as a gray solid (3.61 g, 15.03 mmol). Yield: 64.2%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.07 (s, 1H), 7.23 (d, <i>J</i> = 8.4 Hz, 1H), 7.14 (d, <i>J</i> = 8.2 Hz, 1H), 4.03 (t, <i>J</i> = 1.7 Hz, 2H), 3.71 (s, 1H), 3.23 (t, <i>J</i> = 5.8 Hz, 2H), 2.83–2.77 (m, 2H).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 2-(6,7-Dichloro-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl Acetate (<b>40</b>)</h4><div class="NLM_p last">To a mixture of <b>39</b> (2 g, 8.33 mmol, 1 equiv) and 2-acetoxyacetic acid (1.04 g, 10.0 mmol, 1.2 equiv) in DMF (20 mL) were added EDCI (2.4 g, 12.5 mmol, 1.5 equiv), HOBt (1.7 g, 12.5 mmol, 1.5 equiv), and DIPEA (3.23 g, 25.0 mmol, 1.63 mL, 3 equiv). The mixture was stirred at room temperature for 4 h, then diluted with H<sub>2</sub>O, and extracted with EtOAc. The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a residue. The residue was purified by column chromatography (SiO<sub>2</sub>, DCM/7 N ammonia in MeOH = 500/1 to 200/1) to give title compound <b>40</b> as a white solid (1.91 g, 5.62 mmol). Yield: 67.5%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.19 (d, <i>J</i> = 25.2 Hz, 1H), 7.24–7.11 (m, 2H), 4.83 (s, 2H), 4.76 (s, 1H), 4.55 (s, 1H), 4.00 (t, <i>J</i> = 5.8 Hz, 1H), 3.74 (t, <i>J</i> = 5.7 Hz, 1H), 2.95 (t, <i>J</i> = 4.7 Hz, 1H), 2.87 (t, <i>J</i> = 5.8 Hz, 1H), 2.18 (d, <i>J</i> = 2.4 Hz, 3H).</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 2-(6,7-Dichloro-9<i>b</i>-((4-chloro-2-formylphenyl)amino)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl Acetate (<b>43</b>)</h4><div class="NLM_p">To a solution of intermediate <b>40</b> (68 mg, 0.2 mmol, 1 equiv) in dry DCM (3 mL) was added <i>tert</i>-butyl hypochlorite (24 mg, 0.22 mmol, 1.1 equiv). TMEDA (26 mg, 0.22 mmol, 1.1 equiv) was added after 0.5 min. The solution was stirred at room temperature for 2 min. 2-Amino-5-chlorobenzaldehyde (46 mg, 0.30 mmol, 1.5 equiv) and 1 N H<sub>2</sub>SO<sub>4</sub> solutions (0.20 mL, 0.2 mmol, 1 equiv) were added subsequently. The mixture was stirred for 1 h, then diluted with DCM (10 mL), and washed with H<sub>2</sub>O. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and then concentrated to dryness. The resulting residue was purified by column chromatography (DCM/MeOH = 500/1 to 300/1) to give desired product <b>43</b> (64 mg, 0.13 mmol) as a yellow solid. Yield: 65.3%. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.79 (s, 1H), 9.18 (d, <i>J</i> = 49.1 Hz, 1H), 7.46 (d, <i>J</i> = 17.4 Hz, 1H), 7.41 (d, <i>J</i> = 7.9 Hz, 1H), 7.25 (d, <i>J</i> = 8.0 Hz, 1H), 7.01 (d, <i>J</i> = 8.9 Hz, 1H), 5.71–4.72 (m, 4H), 4.32 (d, <i>J</i> = 150.0 Hz, 1H), 3.23–2.63 (m, 3H), 2.21 (s, 3H), 2.20–1.99 (m, 1H), 1.77 (s, 1H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 194.07, 192.92, 187.10, 186.71, 170.68, 170.62, 166.78, 166.20, 151.74, 151.41, 146.15, 137.23, 136.55, 135.97, 135.65, 135.48, 134.59, 128.64, 126.56, 123.00, 122.44, 121.36, 121.07, 120.81, 120.58, 112.95, 112.85, 72.60, 71.37, 61.28, 57.18, 54.66, 46.64, 45.01, 30.54, 29.98, 20.75. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 492.1. HRMS (ESI) calcd for C<sub>22</sub>H<sub>17</sub>Cl<sub>3</sub>N<sub>3</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 492.0290; found, 492.0279. HPLC purity: 95.3%.</div><div class="NLM_p last">Compounds <b>44–46</b> and <b>51</b> were synthesized according to a similar protocol as that for <b>43</b>.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2-(6,7-Dichloro-9<i>b</i>-((3-chloro-2-formylphenyl)amino)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl Acetate (<b>44</b>)</h4><div class="NLM_p last">Yellow solid; yield: 63.3%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.50 (s, 1H), 9.91 (d, <i>J</i> = 41.0 Hz, 1H), 7.40 (d, <i>J</i> = 7.9 Hz, 1H), 7.23 (d, <i>J</i> = 8.0 Hz, 1H), 7.00–6.90 (m, 1H), 6.68 (d, <i>J</i> = 7.0 Hz, 1H), 5.63–5.22 (m, 2H), 4.86 (s, 1H), 4.46–4.18 (m, 1H), 3.42–2.93 (m, 2H), 2.73 (s, 2H), 2.22 (s, 3H), 2.04 (d, <i>J</i> = 13.6 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 192.96, 186.87, 170.67, 166.62, 151.47, 149.81, 140.85, 136.59, 134.54, 128.63, 126.61, 121.34, 119.27, 115.58, 110.56, 72.57, 61.33, 54.78, 46.66, 30.51, 29.85, 20.78. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 492.0. HRMS (ESI) calcd for C<sub>22</sub>H<sub>17</sub>Cl<sub>3</sub>N<sub>3</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 492.0290; found, 492.0277. HPLC purity: 98.3%.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 2-(6,7-Dichloro-9<i>b</i>-((2-formylphenyl)amino)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl Acetate (<b>45</b>)</h4><div class="NLM_p last">White solid; yield: 75.2%. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.85 (s, 1H), 9.24 (d, <i>J</i> = 55.8 Hz, 1H), 7.49 (s, 1H), 7.40 (d, <i>J</i> = 7.9 Hz, 1H), 7.27 (t, <i>J</i> = 9.0 Hz, 1H), 7.09 (d, <i>J</i> = 29.7 Hz, 1H), 6.76 (d, <i>J</i> = 13.1 Hz, 1H), 5.76–5.40 (m, 1H), 5.35 (d, <i>J</i> = 9.3 Hz, 1H), 5.31–5.03 (m, 1H), 4.88 (q, <i>J</i> = 14.5, 14.0 Hz, 1H), 4.61–4.08 (m, 1H), 3.23–2.63 (m, 3H), 2.21 (d, <i>J</i> = 14.0 Hz, 3H), 2.19–2.00 (m, 1H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 195.11, 194.08, 187.64, 187.18, 170.65, 170.51, 166.60, 166.19, 151.71, 151.36, 147.62, 137.64, 136.86, 136.61, 136.06, 134.59, 134.21, 128.40, 126.27, 121.35, 121.14, 119.99, 119.81, 118.10, 117.61, 111.08, 72.45, 71.34, 61.29, 57.14, 54.61, 46.50, 44.91, 30.39, 29.92, 20.69. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 458.2. HRMS (ESI) calcd for C<sub>22</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 458.0680; found, 458.0672. HPLC purity: 97.8%.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-(6,7-Dichloro-9<i>b</i>-((2-formyl-4-(trifluoromethyl)phenyl)amino)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl Acetate (<b>46</b>)</h4><div class="NLM_p last">Yellow solid; yield: 53.1%. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.89 (s, 1H), 9.52 (d, <i>J</i> = 43.3 Hz, 1H), 7.75 (s, 1H), 7.42 (d, <i>J</i> = 7.9 Hz, 1H), 7.29 (d, <i>J</i> = 8.6 Hz, 1H), 7.25 (d, <i>J</i> = 8.6 Hz, 1H), 5.79–4.67 (m, 4H), 4.47–4.21 (m, 1H), 3.23–2.66 (m, 3H), 2.21 (s, 3H), 2.20–1.99 (m, 1H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 193.21, 186.06, 170.66, 166.85, 151.51, 149.68, 136.76, 134.85, 133.84, 132.58, 128.79, 126.77, 124.77, 121.33, 119.21 (br s), 111.53, 72.51, 61.24, 54.65, 46.71, 30.59, 29.86, 20.76. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 526.1. HRMS (ESI) calcd for C<sub>23</sub>H<sub>17</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>3</sub>O<sub>4</sub> [M – H]<sup>−</sup>, 526.0554; found, 526.0541. HPLC purity: 96.2%.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 2-(9<i>b</i>-((3,5-Bis(trifluoromethyl)phenyl)amino)-6,7-dichloro-1,3,4,4<i>a</i>,5,9<i>b</i>-hexahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl Acetate (<b>51</b>)</h4><div class="NLM_p last">Pale-yellow solid; yield: 61.3%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.68 (d, <i>J</i> = 5.9 Hz, 1H), 7.34 (d, <i>J</i> = 9.2 Hz, 2H), 6.60 (dd, <i>J</i> = 8.5, 4.6 Hz, 1H), 6.22 (dd, <i>J</i> = 11.3, 8.5 Hz, 1H), 6.01 (d, <i>J</i> = 106.7 Hz, 1H), 4.83 (dd, <i>J</i> = 14.6, 6.5 Hz, 1H), 4.60 (dd, <i>J</i> = 35.0, 14.6 Hz, 1H), 4.07–3.98 (m, 1H), 3.94–3.87 (m, 1H), 3.87 (s, 1H), 3.81 (t, <i>J</i> = 9.4 Hz, 1H), 2.56 (m, <i>J</i> = 63.2, 12.5, 9.5 Hz, 1H), 2.40–2.21 (m, 1H), 2.19 (d, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 171.00, 170.93, 170.15, 169.16, 165.80, 165.62, 154.73, 154.65, 151.96, 151.89, 139.38, 139.20, 133.16 (qd, <i>J</i><sub><i>CF</i></sub> = 33.2 Hz, 2C), 124.42, 124.41, 123.23 (q, <i>J</i><sub><i>CF</i></sub> = 273.3 Hz, 2C), 120.11, 120.09, 119.53, 119.51, 117.64 (m), 116.35, 116.27, 115.80, 115.78, 73.60, 71.20, 61.90, 61.66, 57.94, 57.62, 45.58, 44.54, 39.31, 37.94, 20.83, 20.79. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 566.1. HRMS (ESI) calcd for C<sub>23</sub>H<sub>16</sub>Cl<sub>2</sub>F<sub>6</sub>N<sub>3</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 566.0478; found, 566.0471. HPLC purity: 95.3%.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-(9<i>b</i>-((3,5-Bis(trifluoromethyl)phenyl)amino)-6,7-dichloro-1,3,4,4<i>a</i>,5,9<i>b</i>-hexahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>52</b>)</h4><div class="NLM_p">To a solution of compound <b>51</b> (100 mg, 0.17 mmol, 1 equiv) in THF (2 mL) and H<sub>2</sub>O (1 mL) was added LiOH·H<sub>2</sub>O (22 mg, 0.53 mmol, 3 equiv). The mixture was stirred at room temperature for 2 h, then diluted with H<sub>2</sub>O (5 mL), and extracted with EtOAc. The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a residue. The resulting residue was purified by column chromatography (DCM/EtOAc = 20/1 to 3/1) to give the desired product <b>52</b> (39 mg, 0.074 mmol) as a yellow solid. Yield: 43.3%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.69 (s, 1H), 7.33 (d, <i>J</i> = 3.0 Hz, 2H), 7.27 (s, 1H), 6.63 (dd, <i>J</i> = 8.5, 4.5 Hz, 1H), 6.23 (t, <i>J</i> = 8.2 Hz, 1H), 5.52 (s, 1H), 4.21 (d, <i>J</i> = 3.6 Hz, 1H), 4.19–4.06 (m, 1.5H), 3.97 (q, <i>J</i> = 12.4 Hz, 1H), 3.89–3.60 (m, 2.5H), 2.59 (ddt, <i>J</i> = 46.7, 12.6, 9.3 Hz, 1H), 2.41–2.22 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.93, 170.67, 170.21, 169.10, 154.53, 154.32, 151.60, 151.40, 139.52, 139.45, 133.28 (qd, <i>J</i><sub><i>CF</i></sub> = 34.0 Hz, 2C), 124.49, 124.46, 123.21 (q, <i>J</i><sub><i>CF</i></sub> = 273.3 Hz, 2C), 120.74, 120.66, 119.54 (br s, 2C), 117.90, 117.87, 116.38, 116.32, 116.03, 116.00, 77.41, 73.27, 71.24, 60.95, 60.72, 57.84, 56.80, 45.40, 43.67, 38.91, 37.67. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 524.0. HRMS (ESI) calcd for C<sub>21</sub>H<sub>14</sub>Cl<sub>2</sub>F<sub>6</sub>N<sub>3</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 524.0373; found, 524.0361. HPLC purity: 99.2%.</div><div class="NLM_p last">Compounds <b>47</b>–<b>50</b> and <b>57</b> were synthesized according to a similar protocol as that for <b>52</b>.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1-(6,7-Dichloro-9<i>b</i>-((4-(trifluoromethyl)phenyl)amino)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>47</b>)</h4><div class="NLM_p last">White solid; yield 75.1%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.41 (d, <i>J</i> = 7.9 Hz, 1H), 7.26 (d, <i>J</i> = 3.7 Hz, 1H), 7.22 (d, <i>J</i> = 9.2 Hz, 2H), 5.98 (d, <i>J</i> = 8.4 Hz, 2H), 5.43 (dd, <i>J</i> = 13.6, 2.5 Hz, 1H), 5.34 (s, 1H), 4.45–4.34 (m, 2H), 4.00 (dd, <i>J</i> = 11.9, 5.3 Hz, 1H), 3.43 (t, <i>J</i> = 4.5 Hz, 1H), 3.18–3.09 (m, 2H), 2.77 (dt, <i>J</i> = 13.1, 6.4 Hz, 1H), 2.16 (d, <i>J</i> = 13.7 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 185.43, 173.53, 151.14, 146.85, 136.38, 134.34, 128.40, 126.70 (q, <i>J</i><sub><i>CF</i></sub> = 3.8 Hz, 2C), 126.32, 124.21 (q, <i>J</i><sub><i>CF</i></sub> = 272.1 Hz), 120.90, 120.89 (q, <i>J</i><sub><i>CF</i></sub> = 33.8 Hz), 112.09, 73.41, 60.05, 55.23, 45.66, 30.67. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 456.3. HRMS (ESI) calcd for C<sub>20</sub>H<sub>15</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>3</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 456.0499; found, 456.0490. HPLC purity: 98.0%. X-ray crystallography CCDC number: 2055187.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Ethyl 4-((6,7-Dichloro-2-(2-hydroxyacetyl)-1,2,3,4-tetrahydro-9b<i>H</i>-pyrido[4,3-<i>b</i>]indol-9<i>b</i>-yl)amino)benzoate (<b>48</b>)</h4><div class="NLM_p last">White solid; yield: 30.0%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.68 (d, <i>J</i> = 8.7 Hz, 2H), 7.39 (d, <i>J</i> = 7.9 Hz, 1H), 7.24 (d, <i>J</i> = 7.8 Hz, 1H), 5.98–5.91 (m, 2H), 5.47–5.34 (m, 2H), 4.46–4.35 (m, 2H), 4.25 (q, <i>J</i> = 7.1 Hz, 2H), 4.00 (dd, <i>J</i> = 13.4, 6.2 Hz, 1H), 3.44 (s, 1H), 3.18–3.08 (m, 2H), 2.79 (td, <i>J</i> = 13.0, 6.0 Hz, 1H), 2.16 (d, <i>J</i> = 13.7 Hz, 1H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 185.80, 173.79, 166.34, 151.45, 148.36, 136.83, 134.55, 131.65 (2C), 128.62, 126.54, 121.25, 121.18, 118.41, 112.11 (2C), 73.71, 60.53, 60.36, 55.50, 45.95, 30.92, 14.46. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 462.1. HRMS (ESI) calcd for C<sub>22</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 462.0982; found, 462.0990. HPLC purity: 95.3%.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 1-(6,7-Dichloro-9<i>b</i>-((3,5-difluorophenyl)amino)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>49</b>)</h4><div class="NLM_p last">White solid; yield: 35.2%. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.44 (d, <i>J</i> = 7.8 Hz, 1H), 7.28 (d, <i>J</i> = 4.8 Hz, 1H), 6.12 (tt, <i>J</i> = 8.9, 2.1 Hz, 1H), 5.54–5.47 (m, 2H), 5.40 (dd, <i>J</i> = 13.7, 2.6 Hz, 1H), 5.28 (s, 1H), 4.42 (qd, <i>J</i> = 15.5, 4.5 Hz, 2H), 4.06–3.98 (m, 1H), 3.43 (t, <i>J</i> = 4.6 Hz, 1H), 3.21–3.10 (m, 2H), 2.81 (td, <i>J</i> = 12.8, 12.1, 6.2 Hz, 1H), 2.15 (d, <i>J</i> = 13.7 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 185.36, 173.84, 164.12 (d, <i>J</i><sub><i>CF</i></sub> = 246.9 Hz), 164.00 (d, <i>J</i><sub><i>CF</i></sub> = 246.9 Hz), 151.34, 146.71 (t, <i>J</i><sub><i>CF</i></sub> = 12.6 Hz), 136.57, 134.73, 128.73, 126.69, 121.12, 96.20 (d, <i>J</i><sub><i>CF</i></sub> = 15.1 Hz), 96.00 (d, <i>J</i><sub><i>CF</i></sub> = 15.1 Hz), 94.73 (t, <i>J</i><sub><i>CF</i></sub> = 26.5 Hz), 73.77, 60.36, 55.51, 45.96, 30.97. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 426.1. HRMS (ESI) calcd for C<sub>19</sub>H<sub>16</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 426.0582; found, 426.0591. HPLC purity: 95.3%.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 1-(6,7-Dichloro-9<i>b</i>-((2-methoxyphenyl)amino)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>50</b>)</h4><div class="NLM_p last">White solid; yield: 51.2%. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.38 (d, <i>J</i> = 7.9 Hz, 1H), 7.28 (d, <i>J</i> = 7.9 Hz, 1H), 6.71 (dd, <i>J</i> = 8.0, 1.3 Hz, 1H), 6.62 (td, <i>J</i> = 7.7, 1.4 Hz, 1H), 6.48 (td, <i>J</i> = 7.7, 1.3 Hz, 1H), 5.49–5.43 (m, 2H), 5.39 (dd, <i>J</i> = 7.9, 1.5 Hz, 1H), 4.36 (d, <i>J</i> = 4.1 Hz, 2H), 3.94 (dd, <i>J</i> = 13.2, 6.1 Hz, 1H), 3.85 (s, 3H), 3.58 (t, <i>J</i> = 4.4 Hz, 1H), 3.10 (td, <i>J</i> = 13.1, 3.5 Hz, 1H), 3.03 (dd, <i>J</i> = 12.6, 2.1 Hz, 1H), 2.70 (td, <i>J</i> = 12.3, 6.3 Hz, 1H), 2.15 (d, <i>J</i> = 13.6 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 187.29, 173.13, 151.44, 147.14, 138.34, 134.54, 134.04, 128.33, 126.12, 121.44 (2C), 119.02, 110.47, 109.83, 73.89, 60.24, 55.80, 55.30, 45.65, 30.82. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 420.2. HRMS (ESI) calcd for C<sub>20</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 420.0876; found, 420.0882. HPLC purity: 95.3%.</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 1-(6,7-Dichloro-1,4-dimethoxy-5<i>H</i>,10<i>H</i>-4<i>b</i>,9<i>b</i>-(ethanoiminomethano)indolo[3,2-<i>b</i>]indol-12-yl)-2-hydroxyethan-1-one (<b>57</b>)</h4><div class="NLM_p last">White solid; yield: 75.0%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 6.97 (dd, <i>J</i> = 31.3, 8.0 Hz, 1H), 6.87–6.76 (m, 1H), 6.60 (d, <i>J</i> = 8.4 Hz, 1H), 6.24 (dd, <i>J</i> = 11.6, 8.4 Hz, 1H), 4.95 (s, 1H), 4.31–3.96 (m, 4H), 3.88 (d, <i>J</i> = 14.1 Hz, 1H), 3.78 (d, <i>J</i> = 37.9 Hz, 6H), 3.68–3.54 (m, 1H), 3.54–3.49 (m, 1H), 3.44–3.40 (m, 0.5H), 3.21–3.15 (m, 0.5H), 2.59–2.36 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.08, 171.07, 150.47, 149.40, 149.07, 140.66, 140.43, 139.80, 139.23, 133.58, 133.18, 130.21, 129.60, 121.78, 120.89, 120.84, 120.46, 118.80, 117.75, 114.30, 114.02, 111.66, 111.54, 102.09, 101.41, 74.86, 74.49, 73.63, 73.10, 60.01, 59.94, 55.82, 55.54, 55.49, 46.83, 44.51, 39.05, 38.32, 30.27, 29.67. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 450.2. HRMS (ESI) calcd for C<sub>21</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 450.0982; Found, 450.0994. HPLC purity: 96.4%. X-ray crystallography CCDC number: 2055186.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Synthesis of Compounds <b>53–55</b></h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 1-(12,13-Dichloro-3,3<i>a</i>,4,9-tetrahydro-14<i>H</i>-4,9<i>a</i>-epoxybenzo[6,7]pyrrolo [3′,4′:3,4]Azepino[3,2-<i>b</i>]indol-2(1<i>H</i>)-yl)-2-hydroxyethan-1-one (<b>53</b>)</h4><div class="NLM_p last">To a solution of compound <b>45</b> (100 mg, 0.22 mmol, 1 equiv) in THF (2 mL) and H<sub>2</sub>O (1 mL) was added LiOH·H<sub>2</sub>O (27 mg, 0.66 mmol, 3 equiv). The mixture was stirred at room temperature for 2 h, then diluted with H<sub>2</sub>O (5 mL), adjusted to pH 8 with HCl (1 N), and then extracted with EtOAc. The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a residue. The residue was purified by column chromatography (DCM/EtOAc = 20/1 to 3/1) to give desired product <b>53</b> (59 mg, 0.14 mmol) as a yellow solid. Yield: 64.3%. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.26 (t, J = 7.1 Hz, 1H), 7.13 (t, <i>J</i> = 7.7 Hz, 1H), 6.95 (d, <i>J</i> = 7.9 Hz, 1H), 6.91 (dd, <i>J</i> = 22.4, 7.3 Hz, 1H), 6.80 (dt, <i>J</i> = 29.8, 7.4 Hz, 1H), 6.60 (dd, <i>J</i> = 41.5, 7.9 Hz, 1H), 5.11 (dd, <i>J</i> = 32.9, 7.0 Hz, 1H), 4.87 (d, <i>J</i> = 91.6 Hz, 1H), 4.78–4.59 (m, 2H), 4.07–3.55 (m, 2H), 3.50–3.39 (m, 1H), 3.38–3.27 (m, 1H), 3.26–2.94 (m, 2H), 2.93–2.87 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 169.64, 168.56, 148.48, 148.19, 140.49, 139.47, 135.61, 135.57, 129.89, 129.47, 126.96, 126.00, 122.45, 122.18, 122.12, 122.07, 121.61, 121.23, 120.49, 120.26, 120.06, 119.14, 115.38, 115.07, 113.89, 113.64, 102.77, 102.46, 87.35, 86.46, 81.10, 80.47, 60.00, 59.33, 59.15, 59.10, 51.85, 51.64, 44.78, 44.69. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 418.1. HRMS (ESI) calcd for C<sub>20</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 418.0720; found, 418.0722. HPLC purity: 99.5%. X-ray crystallography CCDC number: 2055188.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 2-Hydroxy-1-(6,12,13-trichloro-3,3<i>a</i>,4,9-tetrahydro-14<i>H</i>-4,9<i>a</i>-epoxybenzo[6,7] pyrrolo[3′,4′:3,4]azepino[3,2-<i>b</i>]indol-2(1<i>H</i>)-yl)ethan-1-one (<b>54</b>)</h4><div class="NLM_p last">Compound <b>54</b> was prepared from <b>43</b> following a similar protocol as that for <b>53</b> as a pale-yellow solid. Yield: 63.4%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.32 (t, <i>J</i> = 8.4 Hz, 1H), 7.06–7.01 (m, 1H), 7.01–6.94 (m, 1H), 6.86 (d, <i>J</i> = 8.0 Hz, 1H), 6.56 (dd, <i>J</i> = 11.3, 8.8 Hz, 1H), 5.07 (t, <i>J</i> = 7.6 Hz, 1H), 4.86–4.41 (m, 3H), 3.95 (dd, <i>J</i> = 30.3, 12.7 Hz, 1H), 3.87–3.72 (m, 1H), 3.72–3.33 (m, 2H), 3.29–2.99 (m, 3H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 171.09, 170.49, 150.34, 150.21, 141.66, 141.13, 135.83, 129.85, 129.74, 127.42, 126.75, 125.00, 124.61, 124.02, 123.97, 123.93, 123.74, 123.71, 123.25, 119.31, 119.25, 116.68, 116.16, 113.13, 113.01, 103.92, 103.86, 88.34, 86.88, 81.40, 81.36, 61.82, 61.33, 60.92, 60.58, 52.04, 51.84, 46.09, 45.52. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 450.0. HRMS (ESI) calcd for C<sub>20</sub>H<sub>15</sub>Cl<sub>3</sub>N<sub>3</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 450.0184; found, 450.0178. HPLC purity: 99.7%.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 2-Hydroxy-1-(6,7,9<i>b</i>-trichloro-1,3,5,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)ethan-1-one (<b>55</b>)</h4><div class="NLM_p last">To a solution of intermediate <b>40</b> (68 mg, 0.2 mmol, 1 equiv) in dry DCM (3 mL) was added <i>tert</i>-butyl hypochlorite (24 mg, 0.22 mmol, 1.1 equiv). TMEDA (26 mg, 0.22 mmol, 1.1 equiv) was added after 0.5 min. The solution was stirred at room temperature for 2 min. A similar work-up procedure as that described for <b>53</b> was used to afford desired product <b>55</b>. White solid; yield: 24.5%. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.33 (s, 1H), 7.46 (d, <i>J</i> = 8.4 Hz, 1H), 7.23 (d, <i>J</i> = 8.4 Hz, 1H), 6.48 (s, 1H), 4.55 (dd, <i>J</i> = 13.4, 6.4 Hz, 1H), 4.47 (d, <i>J</i> = 13.6 Hz, 1H), 4.35–4.31 (m, 1H), 3.27 (td, <i>J</i> = 12.6, 4.8 Hz, 1H), 3.16–3.08 (m, 1H), 2.79 (dd, <i>J</i> = 16.1, 4.8 Hz, 1H), 1.34–1.29 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 169.33, 134.08, 133.62, 125.59, 123.90, 122.18, 117.25, 114.45, 109.94, 85.04, 68.02, 36.17, 22.35. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 331.0. HRMS (ESI) calcd for C<sub>13</sub>H<sub>10</sub>Cl<sub>3</sub>N<sub>2</sub>O<sub>2</sub> [M – H]<sup>−</sup>, 330.9813; found, 330.9805. HPLC purity: 98.2%.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Synthetic Procedures of Compounds <b>59–64</b></h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 2-(6,7-Dichloro-9<i>b</i>-(1<i>H</i>-indol-3-yl)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl Acetate (<b>59</b>)</h4><div class="NLM_p last">To a solution of intermediate <b>40</b> (68 mg, 0.2 mmol, 1 equiv) in dry DCM (3 mL) was added <i>tert</i>-butyl hypochlorite (24 mg, 0.22 mmol, 1.1 equiv). TMEDA (26 mg, 0.22 mmol, 1.1 equiv) was then added afterward. The mixture was stirred at room temperature for 2 min and then added with indole (35 mg, 0.30 mmol, 1.5 equiv) and 1 N H<sub>2</sub>SO<sub>4</sub> solution (0.20 mL, 0.2 mmol, 1 equiv). The mixture was stirred for 5 min, then diluted with DCM (10 mL), and washed with H<sub>2</sub>O. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and then concentrated. The resulting residue was purified by column chromatography (DCM/EtOAc = 50/1 to 10/1) to give desired product <b>59</b> (59 mg, 0.13 mmol) as a white solid. Yield: 65.4%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.39 (s, 1H), 7.69 (d, <i>J</i> = 2.6 Hz, 1H), 7.32 (d, <i>J</i> = 8.2 Hz, 1H), 7.24 (s, 1H), 7.10 (t, <i>J</i> = 7.6 Hz, 1H), 6.98 (d, <i>J</i> = 7.9 Hz, 1H), 6.86 (t, <i>J</i> = 7.5 Hz, 1H), 6.55 (d, <i>J</i> = 8.1 Hz, 1H), 5.79 (dd, <i>J</i> = 13.3, 2.3 Hz, 1H), 4.80 (d, <i>J</i> = 14.5 Hz, 1H), 4.70 (d, <i>J</i> = 14.5 Hz, 1H), 4.02 (dd, <i>J</i> = 13.0, 6.2 Hz, 1H), 3.22 (td, <i>J</i> = 12.7, 3.4 Hz, 1H), 3.07 (dd, <i>J</i> = 12.8, 2.1 Hz, 1H), 2.72 (td, <i>J</i> = 12.5, 6.2 Hz, 1H), 2.28 (d, <i>J</i> = 13.4 Hz, 1H), 2.14 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 188.10, 170.58, 165.94, 152.95, 142.37, 136.76, 133.05, 127.96, 125.11, 124.84, 123.76, 122.69, 121.93, 120.33, 118.18, 111.48, 109.14, 61.73, 61.24, 51.53, 46.18, 30.67, 20.69. MS (ESI, [M – H]<sup>−</sup>) <i>m</i>/<i>z</i>: 454.1. HRMS (ESI) calcd for C<sub>23</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub> [M – H]<sup>−</sup>, 454.0731; found, 454.0717. HPLC purity: 98.8%.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 1-(6,7-Dichloro-9<i>b</i>-(1<i>H</i>-indol-3-yl)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>60</b>)</h4><div class="NLM_p">This compound was prepared from <b>59</b> following a similar protocol as that for <b>52</b>. White solid; yield: 46.2%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.18 (s, 1H), 7.72 (s, 1H), 7.33 (d, <i>J</i> = 8.2 Hz, 1H), 7.28 (s, 1H), 7.12 (t, <i>J</i> = 7.6 Hz, 1H), 7.00 (d, <i>J</i> = 7.8 Hz, 1H), 6.88 (t, <i>J</i> = 7.6 Hz, 1H), 6.58 (d, <i>J</i> = 8.2 Hz, 1H), 5.81 (d, <i>J</i> = 13.4 Hz, 1H), 4.28–4.15 (m, 2H), 3.91–3.82 (m, 1H), 3.54 (t, <i>J</i> = 4.3 Hz, 1H), 3.18 (d, <i>J</i> = 13.0 Hz, 1H), 3.09 (d, <i>J</i> = 13.2 Hz, 1H), 2.68 (td, <i>J</i> = 12.5, 6.5 Hz, 1H), 2.37 (d, <i>J</i> = 13.4 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 187.81, 171.13, 152.88, 142.20, 136.69, 133.11, 127.98, 125.17, 124.76, 123.20, 122.88, 121.84, 120.48, 118.20, 111.43, 109.41, 61.12, 60.34, 51.79, 45.16, 30.56. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 414.1. HRMS (ESI) calcd for C<sub>21</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 414.0771; found, 414.0783. HPLC purity: 99.6%. X-ray crystallography CCDC number: 2055189.</div><div class="NLM_p last">Compounds <b>61</b>–<b>64</b> were synthesized according to a similar protocol as that for <b>60</b>.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 1-(6,7-Dichloro-9<i>b</i>-(6-methoxy-1<i>H</i>-indol-3-yl)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>61</b>)</h4><div class="NLM_p last">Off-white solid; yield: 67.4%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.10 (s, 1H), 7.58 (d, <i>J</i> = 2.6 Hz, 1H), 7.28 (s, 1H), 7.00 (d, <i>J</i> = 7.8 Hz, 1H), 6.80 (d, <i>J</i> = 2.2 Hz, 1H), 6.54 (d, <i>J</i> = 8.9 Hz, 1H), 6.44 (d, <i>J</i> = 8.9 Hz, 1H), 5.78 (d, <i>J</i> = 13.4 Hz, 1H), 4.21 (qd, <i>J</i> = 15.2, 4.1 Hz, 2H), 3.86 (dd, <i>J</i> = 13.1, 5.5 Hz, 1H), 3.76 (s, 3H), 3.56 (t, <i>J</i> = 4.3 Hz, 1H), 3.15 (td, <i>J</i> = 12.4, 2.8 Hz, 1H), 3.07 (d, <i>J</i> = 13.6 Hz, 1H), 2.68 (td, <i>J</i> = 12.5, 6.2 Hz, 1H), 2.35 (d, <i>J</i> = 13.4 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 187.87, 171.06, 156.93, 152.80, 142.27, 137.59, 133.05, 127.96, 125.09, 121.95, 121.79, 118.97, 118.75, 110.25, 109.32, 94.91, 61.07, 60.32, 55.66, 51.66, 45.12, 30.50. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 444.1. HRMS (ESI) calcd for C<sub>22</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 444.0876; found, 444.0875. HPLC purity: 99.5%.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 1-(6,7-Dichloro-9<i>b</i>-(6-chloro-1<i>H</i>-indol-3-yl)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>62</b>)</h4><div class="NLM_p last">White solid; yield: 61.2%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.38 (s, 1H), 7.76 (d, <i>J</i> = 2.7 Hz, 1H), 7.28 (d, <i>J</i> = 8.0 Hz, 1H), 7.24 (d, <i>J</i> = 8.6 Hz, 1H), 7.06 (d, <i>J</i> = 8.2 Hz, 1H), 6.97 (d, <i>J</i> = 8.0 Hz, 1H), 6.55 (s, 1H), 5.78 (d, <i>J</i> = 13.2 Hz, 1H), 4.21 (q, <i>J</i> = 15.2 Hz, 2H), 3.86 (dd, <i>J</i> = 13.3, 6.1 Hz, 1H), 3.54 (s, 1H), 3.20–3.08 (m, 2H), 2.71–2.59 (m, 1H), 2.33 (d, <i>J</i> = 13.3 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 187.30, 171.19, 152.75, 141.77, 135.12, 133.41, 128.16, 126.27, 125.84, 125.44, 124.70, 123.43, 121.70, 117.54, 112.57, 109.14, 60.95, 60.37, 51.90, 45.21, 30.61. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 448.0. HRMS (ESI) calcd for C<sub>21</sub>H<sub>17</sub>Cl<sub>3</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 448.0381; found, 448.0382. HPLC purity: 99.6%.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 1-(6,7-Dichloro-9<i>b</i>-(6-fluoro-1<i>H</i>-indol-3-yl)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>63</b>)</h4><div class="NLM_p last">White solid; yield: 50.1%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.27 (s, 1H), 7.69 (d, <i>J</i> = 2.4 Hz, 1H), 7.28 (d, <i>J</i> = 7.9 Hz, 1H), 7.04–6.95 (m, 2H), 6.68–6.59 (m, 1H), 6.46 (dd, <i>J</i> = 8.8, 5.1 Hz, 1H), 5.79 (dd, <i>J</i> = 13.4, 2.3 Hz, 1H), 4.28–4.17 (m, 2H), 3.88 (dd, <i>J</i> = 13.0, 6.2 Hz, 1H), 3.55 (s, 1H), 3.16 (td, <i>J</i> = 12.6, 3.5 Hz, 1H), 3.08 (dd, <i>J</i> = 12.6, 3.2 Hz, 1H), 2.65 (td, <i>J</i> = 12.5, 6.2 Hz, 1H), 2.36 (d, <i>J</i> = 13.3 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 187.65, 171.19, 160.25 (d, <i>J</i><sub><i>CF</i></sub> = 239.4 Hz), 152.84, 142.07, 136.83 (d, <i>J</i><sub><i>CF</i></sub> = 12.6 Hz), 133.31, 128.13, 125.30, 123.61 (d, <i>J</i><sub><i>CF</i></sub> = 2.5 Hz), 121.82, 121.32, 119.09 (d, <i>J</i><sub><i>CF</i></sub> = 2.5 Hz), 109.60, 109.35 (d, <i>J</i><sub><i>CF</i></sub> = 25.2 Hz), 97.88 (d, <i>J</i><sub><i>CF</i></sub> = 25.2 Hz), 61.05, 60.40, 51.74, 45.21, 30.58. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 432.1. HRMS (ESI) calcd for C<sub>21</sub>H<sub>17</sub>Cl<sub>2</sub>FN<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 432.0676; found, 432.0678. HPLC purity: 98.9%.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 1-(6,7-Dichloro-9<i>b</i>-(4,6-difluoro-1<i>H</i>-indol-3-yl)-1,3,4,9<i>b</i>-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>64</b>)</h4><div class="NLM_p last">White solid; yield: 34.2%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.75 (s, 1H), 7.31 (d, <i>J</i> = 8.0 Hz, 1H), 7.07 (s, 1H), 6.88 (dd, <i>J</i> = 9.2, 2.1 Hz, 1H), 6.38 (t, <i>J</i> = 10.2 Hz, 1H), 5.80 (d, <i>J</i> = 11.1 Hz, 1H), 4.88 (s, 2H), 4.29 (d, <i>J</i> = 8.3 Hz, 2H), 4.24–4.18 (m, 1H), 3.15–3.08 (m, 1H), 2.95 (d, <i>J</i> = 12.6 Hz, 1H), 2.79 (s, 1H), 2.29 (d, <i>J</i> = 13.4 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 190.87, 172.86, 160.43 (dd, <i>J</i><sub><i>CF</i></sub> = 238.1, 12.6 Hz), 156.42 (dd, <i>J</i><sub><i>CF</i></sub> = 248.2, 15.1 Hz), 154.01, 144.81, 140.71 (t, <i>J</i><sub><i>CF</i></sub> = 15.1 Hz), 133.47, 128.58, 126.51, 125.15, 123.08, 111.68, 109.06, 95.75 (t, <i>J</i><sub><i>CF</i></sub> = 27.7 Hz), 94.92 (dd, <i>J</i><sub><i>CF</i></sub> = 26.5, 5.04 Hz), 62.75, 61.69, 53.59, 47.25, 31.40. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 450.2. HRMS (ESI) calcd for C<sub>21</sub>H<sub>16</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 450.0582; found, 450.0579. HPLC purity: 99.5%.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Synthetic Procedures of Compounds <b>65</b> and <b>66</b></h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 2-(6,7-Dichloro-9<i>b</i>-(1<i>H</i>-indol-3-yl)-1,3,4,4<i>a</i>,5,9<i>b</i>-hexahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl Acetate (<b>65</b>)</h4><div class="NLM_p last">To a solution of compound <b>59</b> (113 mg, 0.26 mmol, 1 equiv) in dry MeOH (2 mL) were added NaBH<sub>3</sub>CN (47 mg, 0.75 mmol, 3 equiv) and a catalytic amount of HOAc at 0 °C using an ice bath. The mixture was kept at room temperature for 2 h, then quenched with H<sub>2</sub>O, and extracted with DCM. The combined organic phase was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by prep-HPLC (neutral condition) to give the desired product <b>65</b> (78 mg, 0.17 mmol) as a white solid. Yield: 68.7%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.32 (s, 1H), 7.51 (d, <i>J</i> = 2.6 Hz, 1H), 7.37 (d, <i>J</i> = 8.2 Hz, 1H), 7.13 (t, <i>J</i> = 7.6 Hz, 1H), 7.05 (d, <i>J</i> = 8.1 Hz, 1H), 6.92 (t, <i>J</i> = 7.6 Hz, 1H), 6.73 (d, <i>J</i> = 7.8 Hz, 1H), 6.53 (d, <i>J</i> = 7.9 Hz, 1H), 4.85 (d, <i>J</i> = 13.4 Hz, 1H), 4.80–4.70 (m, 2H), 4.62 (s, 1H), 4.35 (s, 1H), 3.73–3.62 (m, 1H), 3.50 (d, <i>J</i> = 13.2 Hz, 1H), 3.07 (d, <i>J</i> = 13.9 Hz, 1H), 2.19 (s, 3H), 1.89 (dd, <i>J</i> = 9.6, 4.0 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.78, 165.22, 148.03, 137.19, 134.24, 131.59, 125.71, 124.09, 122.96, 122.01, 120.95, 120.65, 119.29, 115.29, 114.06, 111.68, 63.73, 61.72, 49.31, 46.94, 39.78, 26.39, 20.83. MS (ESI, [M + Na]<sup>+</sup>) <i>m</i>/<i>z</i>: 480.1. HRMS (ESI) calcd for C<sub>23</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>NaO<sub>3</sub> [M + H]<sup>+</sup>, 480.0852; found, 480.0847. HPLC purity: 95.1%. X-ray crystallography CCDC number: 2055190.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 1-(6,7-Dichloro-9<i>b</i>-(1<i>H</i>-indol-3-yl)-1,3,4,4<i>a</i>,5,9<i>b</i>-hexahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-hydroxyethan-1-one (<b>66</b>)</h4><div class="NLM_p last">Compound <b>66</b> were prepared from <b>65</b> following a similar protocol as that for <b>52</b>. White solid. Yield: 41.2%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.19 (s, 1H), 7.52 (s, 1H), 7.39 (d, <i>J</i> = 8.1 Hz, 1H), 7.15 (t, <i>J</i> = 7.6 Hz, 1H), 7.07 (d, <i>J</i> = 8.1 Hz, 1H), 6.93 (t, <i>J</i> = 7.5 Hz, 1H), 6.78–6.74 (m, 1H), 6.56 (d, <i>J</i> = 7.9 Hz, 1H), 4.84 (d, <i>J</i> = 13.9 Hz, 1H), 4.65 (d, <i>J</i> = 3.3 Hz, 1H), 4.19 (s, 2H), 4.16–3.43 (m, 1H), 3.66–3.57 (m, 1H), 3.36 (d, <i>J</i> = 12.0 Hz, 1H), 3.18 (d, <i>J</i> = 13.8 Hz, 1H), 1.99–1.76 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.09, 147.84, 137.09, 133.94, 131.56, 125.50, 123.54, 122.76, 122.05, 120.85, 120.54, 119.28, 115.32, 114.01, 111.57, 63.56, 59.94, 49.16, 47.25, 38.58, 26.07. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 416.0. HRMS (ESI) calcd for C<sub>21</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 416.0927; found, 416.0924. HPLC purity: 95.7%.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> General Procedure for the Synthesis of HCl Salts of <b>20</b> and <b>25</b></h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 2-Amino-1-(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)ethan-1-one Hydrochloride (HCl Salt of <b>20</b>)</h4><div class="NLM_p last">To a solution of target compound <b>20</b> (100 mg, 0.26 mmol, 1 equiv) in MeOH (3 mL), 1 mL of 0.5 N HCl in MeOH was added dropwise. The mixture was stirred at room temperature for 5 min and concentrated. After repeating the above operation twice, a pale-yellow solid was formed. The solid was dissolved in water and stirred very thoroughly, then lyophilized to obtain the HCl salt of <b>20</b> as a pale-yellow solid. Yield: 100%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.06 (dd, <i>J</i> = 45.7, 2.4 Hz, 1H), 7.30 (s, 1H), 6.77 (dd, <i>J</i> = 19.5, 2.3 Hz, 1H), 4.52 (d, <i>J</i> = 18.8 Hz, 2H), 4.16 (d, <i>J</i> = 4.4 Hz, 3H), 4.09 (s, 1H), 4.00 (d, <i>J</i> = 5.9 Hz, 1H), 3.98 (s, 1H), 3.82 (t, <i>J</i> = 5.8 Hz, 1H), 3.00 (dt, <i>J</i> = 42.4, 5.8 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 166.28, 166.05, 149.72, 148.98, 137.79, 137.29, 136.39, 136.20, 136.11, 135.58, 125.27, 124.77, 124.31, 123.12, 123.07, 122.69, 121.15, 117.49, 116.89, 111.38, 108.58, 108.00, 107.76, 107.49, 44.62, 42.90, 42.38, 41.41, 41.26, 40.55, 39.20, 38.92, 24.83, 24.06. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 378.1. HRMS (ESI) calcd for C<sub>17</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>5</sub>O [M + H]<sup>+</sup>, 378.0883; found, 378.0885. HPLC purity: 95.7%. For <b>20</b><sup>.</sup>HCl·2H<sub>2</sub>O (C<sub>17</sub>H<sub>17</sub>Cl<sub>2</sub>N<sub>5</sub>O·HCl·2H<sub>2</sub>O), Calcd: C (45.30%), H (4.93%), N (15.54%). Found: C (45.50%), H (4.63%), N (15.73%).</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 2-Amino-<i>N</i>-(2-(6,7-dichloro-9-(1-methyl-1<i>H</i>-pyrazol-3-yl)-1,3,4,5-tetrahydro-2<i>H</i>-pyrido[4,3-<i>b</i>]indol-2-yl)-2-oxoethyl)acetamide Hydrochloride (HCl Salt of <b>25</b>)</h4><div class="NLM_p last">The HCl salt of <b>25</b> was prepared following a similar procedure as above as a pale-yellow solid. Yield: 100%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.94 (d, <i>J</i> = 40.4 Hz, 1H), 7.25 (s, 1H), 6.67 (d, <i>J</i> = 14.2 Hz, 1H), 4.53 (d, <i>J</i> = 17.4 Hz, 2H), 4.22 (d, <i>J</i> = 46.7 Hz, 2H), 4.11 (s, 3H), 3.96 (t, <i>J</i> = 5.8 Hz, 1H), 3.88–3.83 (m, 1H), 3.76 (s, 2H), 3.04 (t, <i>J</i> = 6.0 Hz, 1H), 2.92 (t, <i>J</i> = 5.9 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 169.13, 168.99, 167.77, 167.62, 150.44, 149.76, 137.39, 136.95, 136.11, 136.01, 135.24, 134.52, 125.07, 124.84, 124.34, 124.25, 122.85, 122.82, 116.74, 116.21, 108.30, 107.98, 107.92, 107.46, 44.76, 42.98, 42.61, 42.43, 42.15, 41.56, 40.54, 39.26, 38.98, 24.93, 24.07. MS (ESI, [M + H]<sup>+</sup>) <i>m</i>/<i>z</i>: 435.2. HRMS (ESI) calcd for C<sub>19</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 435.1098; found, 435.1098. HPLC purity: 98.4%. For <b>25</b><sup>.</sup>2HCl·3H<sub>2</sub>O (C<sub>19</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>2</sub>·2HCl·3H<sub>2</sub>O), Calcd: C (40.58%), H (5.03%), N (14.95%). Found: C (40.92%), H (5.17%), N (14.91%).</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Single-Crystal Structure of Compounds</h3><div class="NLM_p last">X-ray diffractions of all single crystals were carried out at 150.0, 170.0, or 299.0 K on a Bruker D8 Venture diffractometer. Using Olex2, the structure was solved with the ShelXT structure solution program using Intrinsic Phasing and refined with the ShelXL refinement package using Least Squares minimization. Additional details are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_003.cif" class="ext-link">Supporting Information</a> as CIF files.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Reagents and Antibodies</h3><div class="NLM_p last">Compounds were dissolved in dimethyl sulfoxide (DMSO, Sigma-Aldrich) at a concentration of 50 mmol/L and then diluted with the RPMI 1640 medium (Hyclone, South Logan, UT, USA) containing 10% fetal bovine serum (FBS, Hyclone). The Cell Counting Kit-8 (CCK-8) was purchased from Dojindo (Kumamoto, Japan). Herring testes DNA (HT-DNA) and LPS were purchased from Sigma. cGAMP, Normocin, Zeocin, blasticidin, and QUANTI-Luc luciferase reagent were purchased from InvivoGen. Protease/phosphatase inhibitor cocktail (#5872), primary antibodies against TBK1 (#3504), phosphor-TBK1 (Ser172, #5438), IRF3 (#4302), phosphor-IRF3 (Ser386, #37829), STING (#13647), and phosphor-STING (Ser366, #50907) were purchased from Cell Signaling Technology (CST, Boston, MA, USA). GAPDH was purchased from Abgent (Abgent, San Diego, CA, USA) and the anti-rabbit IgG horseradish peroxidase (HRP)-linked secondary antibody was purchased from Bio-Rad (Bio-Rad, Hercules, CA, USA).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Cell culture</h3><div class="NLM_p last">The THP1-Dual cells and RAW-Lucia ISG cells were purchased from InvivoGen. The THP1-Dual cells were cultured and maintained in RPMI 1640 (Gibco) supplemented with 10% FBS, 100 U/mL penicillin, 50 μg/mL streptomycin, and 100 μg/mL Normocin. To maintain luciferase expression, 100 μg/mL of Zeocin and 10 μg/mL of blasticidin were added to the growth medium every other passage. RAW-Lucia ISG cells were grown in DMEM (Gibco) supplemented with 10% FBS, 100 U/mL penicillin, 50 μg/mL streptomycin, 100 μg/mL Normocin, and 200 μg/mL Zeocin. The cells were maintained at 37 °C in a 5% CO<sub>2</sub> incubator.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Cytotoxicity Assay</h3><div class="NLM_p last">The cytotoxicity of compounds was evaluated by the Cell Counting Kit-8 (CCK-8). Briefly, the cells (5 × 10<sup>4</sup> cells/well) were seeded into 96-well plates in triplicate with 200 μL of RPMI 1640 or DMEM for 24 h in the presence or absence of the indicated concentrations of compounds. Subsequently, a total of 20 μL of CCK-8 was added to each well. After 1 h of incubation, the plates were measured at 450 nm (570 nm calibration) using a microplate reader (Molecular Devices, Sunnyvale, CA, USA) and the cell viability was calculated. The cytotoxicity of compounds was expressed as CC<sub>50</sub>, using the log (inhibitor) versus normalized response nonlinear fit (GraphPad Prism 8.0).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Cell-Based Lucia luciferase Assay</h3><div class="NLM_p last">THP1-Dual cells were pre-incubated in 96-well plates (5 × 10<sup>4</sup> cells/well) over an indicated concentration range of the inhibitor for 1 h. DMSO was added as the negative control. Cells were transfected with 2 μg/mL of HT-DNA or cGAMP in complex with Lipofectamine 2000 (Invitrogen) for 24 h. RAW-Lucia ISG cells were pre-incubated in 96-well plates (1 × 10<sup>5</sup> cells/well) over an indicated concentration range of inhibitor for 1 h. DMSO was added as the negative control. Cells were transfected with 1 μg/mL of HT-DNA in the complex with Lipofectamine 2000 (Invitrogen) for 24 h. Luciferase luminescence was measured for each sample using QUANTI-Luc luciferase reagent (InvivoGen), following the manufacturer’s protocol. Relative luciferase activity for each compound-treated sample was calculated using a Lipofectamine 2000-treated sample as the negative control and a Lipofectamine 2000:dsDNA or Lipofectamine 2000:cGAMP complex-treated sample without the compound as the positive control, namely, relative luciferase activity = (RLU sample – RLU negative control)/(RLU positive control – RLU negative control), where RLU indicates the raw luciferase unit.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Cellular Thermal Shift Assay</h3><div class="NLM_p last">For CETSA, THP1-Dual cells were seeded in a T-25 flask (5 × 10<sup>6</sup> cells/mL) and exposed to the compound at the indicated concentration (25 μM) for 1 h at 37 °C in a CO<sub>2</sub> incubator. Control cells were incubated with an equal volume of DMSO. Following incubation, the cells were washed twice with cold PBS and re-suspended in cold PBS supplemented with a protease/phosphatase inhibitor cocktail. The cells were aliquoted into PCR strip tubes, heated at the indicated temperature in a thermal cycler for 3 min, and then cooled for another 3 min at room temperature. The cells were freeze–thawed twice using liquid nitrogen and a thermal cycler or heating block set at 25 °C in order to ensure a uniform temperature between tubes. The tubes were vortexed briefly after each thawing. The resulting cell lysates were kept on (4 °C) ice after the last thawing step. Briefly, the tubes were vortexed and the cell lysate-containing tubes were centrifuged at 14000 g for 20 min at 4 °C to pellet cell debris together with precipitated and aggregated proteins. The supernatant was carefully transferred to a new tube for further western blot assay.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> mRNA Expression Analysis of THP1-Dual Cells</h3><div class="NLM_p">The cellular activation of cGAS enzyme leads to <i>Ifn-β</i> and <i>Il-6</i> mRNA expression in THP1-Dual cells and was quantified using qRT-PCR. Total RNAs was isolated from 5 × 10<sup>5</sup> THP1-Dual cells per well of a 12-well plate, which were pre-incubated with an inhibitor for 1 h. Cells were transfected using 100 μL of Opti-MEM transfection solution comprising 2 μg of HT-DNA complexed with 2 μL of Lipofectamine 2000. Cells were harvested 4 h post-transfection, and RNA was extracted using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) and quantified using a Nano-Drop 2000 spectrophotometer (Thermo Scientific, Bremen, Germany). Reverse transcription was carried out with a Trans-Script First-Strand cDNA Synthesis SuperMix Kit (Yishen, Shanghai, China) according to the manufacturer’s instructions. Subsequently, the product from reverse transcription was amplified with a SYBR Green (Yishen, Shanghai, China) using Applied Biosystems (Thermo Fisher Scientific, Waltham, USA). Primer sequences were listed as in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Primer Sequences</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top"><th class="colsep0 rowsep0" align="center">gene</th><th class="colsep0 rowsep0" align="center">(sequence 5′–3′) forward</th><th class="rowsep0" align="center">reverse</th></tr></thead><tbody><tr valign="top"><td class="colsep0 rowsep0" align="left"><i>Gapdh</i></td><td class="colsep0 rowsep0" align="left">GGAGCGAGATCCCTCCAAAAT</td><td class="rowsep0" align="left">GGCTGTTGTCATACTTCTCATGG</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><i>Il-6</i></td><td class="colsep0 rowsep0" align="left">ACTCACCTCTTCAGAACGAATTG</td><td class="rowsep0" align="left">CCATCTTTGGAAGGTTCAGGTTG</td></tr><tr valign="top"><td class="colsep0" align="left"><i>Ifn-β</i></td><td class="colsep0" align="left">AGGACAGGATGAACTTTGAC</td><td align="left">TGATAGACATTAGCCAGGAG</td></tr></tbody></table></div></div><div class="NLM_p last">All reactions were performed in triplicate, and analysis of the relative gene expression level was normalized to glyceraldehyde-3-phosphate dehydrogenase (<i>Gapdh</i>) using the 2<sup>–ΔΔ<i>Ct</i></sup> method. Melting curves were routinely performed to determine the specificity of the PCR.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Western Blot Assay</h3><div class="NLM_p last">THP1-Dual cells (2.5 × 10<sup>5</sup> cells/well) were seeded into 24-well plates and pre-treated with or without different concentrations of compounds for 1 h. Then, cells were transfected using 50 μL of an Opti-MEM transfection solution comprising 2 μg of HT-DNA complexed with 2 μL of Lipofectamine 2000. Cells samples were lysed in 1 × SDS lysis buffer (Beyotime, Shanghai, China) 2 h post-transfection and uniformed by the Pierce BCA protein assay kit (Thermo Fisher Scientific, Pittsburgh, PA, USA). Equal protein amounts were loaded to 10% SDS-PAGE and transferred to a nitrocellulose membrane (Bio-Rad, Hercules, CA, USA), which was blocked with Super-Block T20 blocking buffer (Thermo Fisher Scientific, Pittsburgh, PA, USA) and then incubated overnight at 4 °C with primary antibodies. The bands were incubated with HRP-conjugated anti-rabbit IgG (Bio-Rad, Richmond, CA, USA) and further visualized using a Super-Signal West Femto Maximum Sensitivity Substrate kit (Thermo Fisher Scientific, Pittsburgh, PA, USA) under a ChemiDoc MP Imaging System (Bio-Rad, Richmond, CA, USA).</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Animal Experiments: LPS-Induced Inflammation Model</h3><div class="NLM_p">Inbred 8-week-old female BALB/c mice were purchased from Shanghai Lingchang Biotechnology Co., Ltd. (certificate no. 2013-0018, Shanghai, China). All mice were housed under specific pathogen-free conditions and raised in a 12 h light/dark cycle with humidity (60–80%) and temperature (22 ± 1 °C). All mice were fed standard laboratory chow and water ad libitum and allowed to acclimatize in our facility for 1 week before any experiments were started. All experiments were carried out according to the National Institutes of Health Guides for the Care and Use of Laboratory Animals and were approved by the Bioethics Committee of the Shanghai Institute of <i>Materia Medica</i>, Chinese Academy of Sciences.</div><div class="NLM_p last">Female BALB/c mice were intraperitoneally injected with compounds and vehicle (10% DMSO+90% corn oil) 1 h before an intraperitoneal injection with a dose of 5 mg/kg of LPS. The serum was collected 1.5 h after LPS application.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> ELISA Analysis</h3><div class="NLM_p last">Cytokines in the serum of each mouse from all groups were determined by using mouse TNF-α, IL-6, and IL-12 ELISA kits according to the manufacturer’s instructions. All ELISA quantification kits were purchased from BD Pharmingen (San Diego, CA, USA).</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> Statistical Analysis</h3><div class="NLM_p last">The data are presented as mean ± SEM. Statistical differences were determined using one-way ANOVA with Dunnet’s multiple comparison test with no significant variance inhomogeneity (<i>F</i> achieved <i>P</i> < 0.05) using GraphPad software 8.0. <i>P</i> values less than 0.05 were considered significant.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i98"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00398" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62248" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62248" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00398?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00398</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Proposed mechanism for the formation of compounds <b>53</b> and <b>54</b>, variable-temperature NMR spectra of compound <b>20</b>, <sup>1</sup>H and <sup>13</sup>C spectra of all new compounds, and X-ray crystal structure data for compounds <b>47</b>, <b>53</b>, <b>57</b>, <b>63</b>, and <b>65</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>47</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_003.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>53</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_004.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>57</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_005.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>63</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_006.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>65</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_007.cif" class="ext-link">CIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_001.pdf">jm1c00398_si_001.pdf (7.2 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_002.csv">jm1c00398_si_002.csv (6.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_003.cif">jm1c00398_si_003.cif (1.58 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_004.cif">jm1c00398_si_004.cif (556.71 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_005.cif">jm1c00398_si_005.cif (525.78 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_006.cif">jm1c00398_si_006.cif (653.52 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_007.cif">jm1c00398_si_007.cif (306.83 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00398" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunyong Ding</span> - <span class="hlFld-Affiliation affiliation">Pharm-X
Center, School of Pharmacy, Shanghai Jiao
Tong University, Shanghai 200240, China</span>; 
    <span class="hlFld-Affiliation affiliation">Shanghai
Key Laboratory for Molecular Engineering of Chiral Drugs, School of
Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d0b3b8a5beb4b9beb790a3baa4a5feb5b4a5feb3be"><span class="__cf_email__" data-cfemail="dab9b2afb4beb3b4bd9aa9b0aeaff4bfbeaff4b9b4">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Tang</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute of Materia Medica (SIMM), Chinese
Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e99d88878e9e8c80a99a808484c7888ac78a87"><span class="__cf_email__" data-cfemail="d0a4b1beb7a7b5b990a3b9bdbdfeb1b3feb3be">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ao Zhang</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute of Materia Medica (SIMM), Chinese
Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">Pharm-X
Center, School of Pharmacy, Shanghai Jiao
Tong University, Shanghai 200240, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">Shanghai
Key Laboratory for Molecular Engineering of Chiral Drugs, School of
Pharmacy, Shanghai Jiao Tong University, Shanghai 200240, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7205-9202" title="Orcid link">https://orcid.org/0000-0001-7205-9202</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5c3d336a656d6526343d323b1c2f36282972393829723f32"><span class="__cf_email__" data-cfemail="24454b121d151d5e4c454a4364574e50510a4140510a474a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Tan</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute of Materia Medica (SIMM), Chinese
Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">Pharm-X
Center, School of Pharmacy, Shanghai Jiao
Tong University, Shanghai 200240, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bing Wu</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute of Materia Medica (SIMM), Chinese
Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tingting Chen</span> - <span class="hlFld-Affiliation affiliation">Pharm-X
Center, School of Pharmacy, Shanghai Jiao
Tong University, Shanghai 200240, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chen Fan</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute of Materia Medica (SIMM), Chinese
Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiannan Zhao</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute of Materia Medica (SIMM), Chinese
Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">Pharm-X
Center, School of Pharmacy, Shanghai Jiao
Tong University, Shanghai 200240, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chaodong Xiong</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute of Materia Medica (SIMM), Chinese
Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">Pharm-X
Center, School of Pharmacy, Shanghai Jiao
Tong University, Shanghai 200240, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunlan Feng</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Institute of Materia Medica (SIMM), Chinese
Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruoxuan Xiao</span> - <span class="hlFld-Affiliation affiliation">Pharm-X
Center, School of Pharmacy, Shanghai Jiao
Tong University, Shanghai 200240, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.T. and B.W. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i100">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86126" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86126" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by grants from the Natural Science Foundation of China (grant no. 81773565) and the Science and Technology Commission of Shanghai Municipality (18431907100). Start-up grants to the Research Laboratory of Medicinal Chemical Biology & Frontiers on Drug Discovery (AF1700037 and WF220217002) and to the Drug Target Identification Platform (WH101117001) from Shanghai Jiao Tong University are also appreciated.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AGS</td><td class="NLM_def"><p class="first last">Aicardi-Goutieres syndrome</p></td></tr><tr><td class="NLM_term">ALI</td><td class="NLM_def"><p class="first last">acute lung injury</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">cGAMP</td><td class="NLM_def"><p class="first last">cyclic GMP–AMP</p></td></tr><tr><td class="NLM_term">cGAS</td><td class="NLM_def"><p class="first last">cyclic GMP–AMP synthase</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DEA</td><td class="NLM_def"><p class="first last">diethylamine</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">dsDNA</td><td class="NLM_def"><p class="first last">double-stranded DNA</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-dimethylaminopropyl)-<i>N</i>′-ethylcarbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">GTP</td><td class="NLM_def"><p class="first last">guanosine triphosphate</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">2-(7-aza-1<i>H</i>-benzotriazole-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">h-cGAS<sup>CD</sup></td><td class="NLM_def"><p class="first last">h-cGAS catalytic domain</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">IFN</td><td class="NLM_def"><p class="first last">interferon</p></td></tr><tr><td class="NLM_term">IL</td><td class="NLM_def"><p class="first last">interleukin</p></td></tr><tr><td class="NLM_term">IRF3</td><td class="NLM_def"><p class="first last">interferon regulatory factor 3</p></td></tr><tr><td class="NLM_term">ISGs</td><td class="NLM_def"><p class="first last">interferon-stimulated genes</p></td></tr><tr><td class="NLM_term">LPS</td><td class="NLM_def"><p class="first last">lipopolysaccharide</p></td></tr><tr><td class="NLM_term">NF-κB</td><td class="NLM_def"><p class="first last">nuclear factor kappa-B</p></td></tr><tr><td class="NLM_term">NLRP3</td><td class="NLM_def"><p class="first last">NOD-like receptor protein 3</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic properties</p></td></tr><tr><td class="NLM_term">RT-qPCR</td><td class="NLM_def"><p class="first last">reverse transcription and quantitative polymerase chain reaction</p></td></tr><tr><td class="NLM_term">STING</td><td class="NLM_def"><p class="first last">stimulator of interferon gene</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOCl</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyl hypochlorite</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">THγCs</td><td class="NLM_def"><p class="first last">tetrahydro-γ-carbolines</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">TMEDA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethylethylenediamine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32653" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32653" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 39 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brubaker, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonham, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanoni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagan, J. C.</span></span> <span> </span><span class="NLM_article-title">Innate immune pattern recognition: a cell biological perspective</span>. <i>Annu. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1146/annurev-immunol-032414-112240</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1146%2Fannurev-immunol-032414-112240" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=25581309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVSisL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=257-290&author=S.+W.+Brubakerauthor=K.+S.+Bonhamauthor=I.+Zanoniauthor=J.+C.+Kagan&title=Innate+immune+pattern+recognition%3A+a+cell+biological+perspective&doi=10.1146%2Fannurev-immunol-032414-112240"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Innate Immune Pattern Recognition: A Cell Biological Perspective</span></div><div class="casAuthors">Brubaker, Sky W.; Bonham, Kevin S.; Zanoni, Ivan; Kagan, Jonathan C.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">257-290</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Receptors of the innate immune system detect conserved determinants of microbial and viral origin.  Activation of these receptors initiates signaling events that culminate in an effective immune response.  Recently, the view that innate immune signaling events rely on and operate within a complex cellular infrastructure has become an important framework for understanding the regulation of innate immunity.  Compartmentalization within this infrastructure provides the cell with the ability to assign spatial information to microbial detection and regulate immune responses.  Several cell biol. processes play a role in the regulation of innate signaling responses; at the same time, innate signaling can engage cellular processes as a form of defense or to promote immunol. memory.  In this review, we highlight these aspects of cell biol. in pattern-recognition receptor signaling by focusing on signals that originate from the cell surface, from endosomal compartments, and from within the cytosol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Q0f75-M4yLVg90H21EOLACvtfcHk0liydICPfPsJJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVSisL3O&md5=ad51df752f62a080d97890221dfd2a68</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev-immunol-032414-112240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-immunol-032414-112240%26sid%3Dliteratum%253Aachs%26aulast%3DBrubaker%26aufirst%3DS.%2BW.%26aulast%3DBonham%26aufirst%3DK.%2BS.%26aulast%3DZanoni%26aufirst%3DI.%26aulast%3DKagan%26aufirst%3DJ.%2BC.%26atitle%3DInnate%2520immune%2520pattern%2520recognition%253A%2520a%2520cell%2520biological%2520perspective%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2015%26volume%3D33%26spage%3D257%26epage%3D290%26doi%3D10.1146%2Fannurev-immunol-032414-112240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornung, V.</span></span> <span> </span><span class="NLM_article-title">Recognition of endogenous nucleic acids by the innate immune system</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2016.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.immuni.2016.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=27096317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVehs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=739-754&author=A.+Roersauthor=B.+Hillerauthor=V.+Hornung&title=Recognition+of+endogenous+nucleic+acids+by+the+innate+immune+system&doi=10.1016%2Fj.immuni.2016.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of Endogenous Nucleic Acids by the Innate Immune System</span></div><div class="casAuthors">Roers, Axel; Hiller, Bjoern; Hornung, Veit</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">739-754</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Recognition of DNA and RNA by endosomal and cytosolic sensors constitutes a central element in the detection of microbial invaders by the innate immune system.  However, the capacity of these sensors to discriminate between microbial and endogenous nucleic acids is limited.  Over the past few years, evidence has accumulated to suggest that endogenous DNA or RNA species can engage nucleic-acid-sensing pattern-recognition receptors that can trigger or sustain detrimental pathol.  Here, we review principles of how the activation of innate sensors by host nucleic acids is prevented in the steady state and discuss four important determinants of whether a nucleic-acid-driven innate response is mounted.  These include structural features of the ligand being sensed, the subcellular location and quantity of pathogen-derived or endogenous nucleic acids, and the regulation of sensor-activation thresholds.  Furthermore, we emphasize disease mechanisms initiated by failure to discriminate self from non-self in nucleic acid detection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTfTR4bXug1rVg90H21EOLACvtfcHk0liSyMEZVeA2Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVehs7o%253D&md5=624aab1b9e40165898e710b62d786fe3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2016.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2016.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DRoers%26aufirst%3DA.%26aulast%3DHiller%26aufirst%3DB.%26aulast%3DHornung%26aufirst%3DV.%26atitle%3DRecognition%2520of%2520endogenous%2520nucleic%2520acids%2520by%2520the%2520innate%2520immune%2520system%26jtitle%3DImmunity%26date%3D2016%26volume%3D44%26spage%3D739%26epage%3D754%26doi%3D10.1016%2Fj.immuni.2016.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">Cyclic GMP-AMP Is an Endogenous Second Messenger in Innate Immune Signaling by Cytosolic DNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>339</i></span>,  <span class="NLM_fpage">826</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1126/science.1229963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1126%2Fscience.1229963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=23258412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1ygtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=826-830&author=J.+Wuauthor=L.+Sunauthor=X.+Chenauthor=F.+Duauthor=H.+Shiauthor=C.+Chenauthor=Z.+J.+Chen&title=Cyclic+GMP-AMP+Is+an+Endogenous+Second+Messenger+in+Innate+Immune+Signaling+by+Cytosolic+DNA&doi=10.1126%2Fscience.1229963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic GMP-AMP Is an Endogenous Second Messenger in Innate Immune Signaling by Cytosolic DNA</span></div><div class="casAuthors">Wu, Jiaxi; Sun, Lijun; Chen, Xiang; Du, Fenghe; Shi, Heping; Chen, Chuo; Chen, Zhijian J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">339</span>
        (<span class="NLM_cas:issue">6121</span>),
    <span class="NLM_cas:pages">826-830</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cytosolic DNA induces type I interferons and other cytokines that are important for antimicrobial defense but can also result in autoimmunity.  This DNA signaling pathway requires the adaptor protein STING and the transcription factor IRF3, but the mechanism of DNA sensing is unclear.  We found that mammalian cytosolic exts. synthesized cyclic guanosine monophosphate-adenosine monophosphate (cyclic GMP-AMP, or cGAMP) in vitro from ATP and guanosine triphosphate in the presence of DNA but not RNA.  DNA transfection or DNA virus infection of mammalian cells also triggered cGAMP prodn.  CGAMP bound to STING, leading to the activation of IRF3 and induction of interferon-β.  Thus, cGAMP functions as an endogenous second messenger in metazoans and triggers interferon prodn. in response to cytosolic DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0CH1aDO6v5bVg90H21EOLACvtfcHk0liSyMEZVeA2Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1ygtbs%253D&md5=2d59a6c9c71eb7188e3e9fd01b2c7e84</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1229963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1229963%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DCyclic%2520GMP-AMP%2520Is%2520an%2520Endogenous%2520Second%2520Messenger%2520in%2520Innate%2520Immune%2520Signaling%2520by%2520Cytosolic%2520DNA%26jtitle%3DScience%26date%3D2013%26volume%3D339%26spage%3D826%26epage%3D830%26doi%3D10.1126%2Fscience.1229963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">The cytosolic DNA sensor cGAS forms an oligomeric complex with DNA and undergoes switch-like conformational changes in the activation loop</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.celrep.2014.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=24462292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Gms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=421-430&author=X.+Zhangauthor=J.+Wuauthor=F.+Duauthor=H.+Xuauthor=L.+Sunauthor=Z.+Chenauthor=C.+A.+Brautigamauthor=X.+Zhangauthor=Z.+J.+Chen&title=The+cytosolic+DNA+sensor+cGAS+forms+an+oligomeric+complex+with+DNA+and+undergoes+switch-like+conformational+changes+in+the+activation+loop&doi=10.1016%2Fj.celrep.2014.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The Cytosolic DNA Sensor cGAS Forms an Oligomeric Complex with DNA and Undergoes Switch-like Conformational Changes in the Activation Loop</span></div><div class="casAuthors">Zhang, Xu; Wu, Jiaxi; Du, Fenghe; Xu, Hui; Sun, Lijun; Chen, Zhe; Brautigam, Chad A.; Zhang, Xuewu; Chen, Zhijian J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">421-430</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The presence of DNA in the cytoplasm is a danger signal that triggers immune and inflammatory responses.  Cytosolic DNA binds to and activates cyclic GMP-AMP (cGAMP) synthase (cGAS), which produces the second messenger cGAMP. cGAMP binds to the adaptor protein STING and activates a signaling cascade that leads to the prodn. of type I interferons and other cytokines.  Here, we report the crystal structures of human cGAS in its apo form, representing its autoinhibited conformation as well as in its cGAMP- and sulfate-bound forms.  These structures reveal switch-like conformational changes of an activation loop that result in the rearrangement of the catalytic site.  The structure of DNA-bound cGAS reveals a complex composed of dimeric cGAS bound to two mols. of DNA.  Functional analyses of cGAS mutants demonstrate that both the protein-protein interface and the two DNA binding surfaces are crit. for cGAS activation.  These results provide insights into the mechanism of DNA sensing by cGAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6puAKW2zx4rVg90H21EOLACvtfcHk0liSyMEZVeA2Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Gms78%253D&md5=6b59678311798f13d749468246444dc3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DThe%2520cytosolic%2520DNA%2520sensor%2520cGAS%2520forms%2520an%2520oligomeric%2520complex%2520with%2520DNA%2520and%2520undergoes%2520switch-like%2520conformational%2520changes%2520in%2520the%2520activation%2520loop%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D6%26spage%3D421%26epage%3D430%26doi%3D10.1016%2Fj.celrep.2014.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barber, G. N.</span></span> <span> </span><span class="NLM_article-title">STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>455</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1038/nature07317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnature07317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=18724357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtF2hsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=455&publication_year=2008&pages=674-678&author=H.+Ishikawaauthor=G.+N.+Barber&title=STING+is+an+endoplasmic+reticulum+adaptor+that+facilitates+innate+immune+signalling&doi=10.1038%2Fnature07317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling</span></div><div class="casAuthors">Ishikawa, Hiroki; Barber, Glen N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">455</span>
        (<span class="NLM_cas:issue">7213</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The cellular innate immune system is essential for recognizing pathogen infection and for establishing effective host defense.  But crit. mol. determinants responsible for facilitating an appropriate immune response-following infection with DNA and RNA viruses, for example-remain to be identified.  Here the authors report the identification, following expression cloning, of a mol. (STING; stimulator of interferon genes) that appears essential for effective innate immune signaling processes.  It comprises five putative transmembrane regions, predominantly resides in the endoplasmic reticulum and is able to activate both NF-κB and IRF3 transcription pathways to induce expression of type I interferon (IFN-α and IFN-β) and exert a potent anti-viral state following expression.  In contrast, loss of STING rendered murine embryonic fibroblasts extremely susceptible to neg.-stranded virus infection, including vesicular stomatitis virus.  Further, STING ablation abrogated the ability of intracellular B-form DNA, as well as members of the herpesvirus family, to induce IFN-β, but did not significantly affect the Toll-like receptor (TLR) pathway.  Yeast two-hybrid and co-immunopptn. studies indicated that STING interacts with RIG-I and with SSR2 (also known as TRAPβ), which is a member of the translocon-assocd. protein (TRAP) complex required for protein translocation across the endoplasmic reticulum membrane following translation.  Ablation by RNA interference of both TRAPβ and translocon adaptor SEC61β was subsequently found to inhibit STING's ability to stimulate expression of IFN-β.  Thus, as well as identifying a regulator of innate immune signaling, these results imply a potential role for the translocon in innate signaling pathways activated by select viruses as well as intracellular DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM8AkjsFJ2qbVg90H21EOLACvtfcHk0liSyMEZVeA2Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtF2hsbzL&md5=67bd86a114023c8d6c68489ab7e33009</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature07317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07317%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DH.%26aulast%3DBarber%26aufirst%3DG.%2BN.%26atitle%3DSTING%2520is%2520an%2520endoplasmic%2520reticulum%2520adaptor%2520that%2520facilitates%2520innate%2520immune%2520signalling%26jtitle%3DNature%26date%3D2008%26volume%3D455%26spage%3D674%26epage%3D678%26doi%3D10.1038%2Fnature07317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barber, G. N.</span></span> <span> </span><span class="NLM_article-title">STING regulates intracellular DNA-mediated, type I interferon-dependent innate immunity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span>,  <span class="NLM_fpage">788</span>– <span class="NLM_lpage">792</span>, <span class="refDoi"> DOI: 10.1038/nature08476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnature08476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=19776740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFGqtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=788-792&author=H.+Ishikawaauthor=Z.+Maauthor=G.+N.+Barber&title=STING+regulates+intracellular+DNA-mediated%2C+type+I+interferon-dependent+innate+immunity&doi=10.1038%2Fnature08476"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">STING regulates intracellular DNA-mediated, type I interferon-dependent innate immunity</span></div><div class="casAuthors">Ishikawa, Hiroki; Ma, Zhe; Barber, Glen N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7265</span>),
    <span class="NLM_cas:pages">788-792</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The innate immune system is crit. for the early detection of invading pathogens and for initiating cellular host defense countermeasures, which include the prodn. of type I interferon (IFN).  However, little is known about how the innate immune system is galvanized to respond to DNA-based microbes.  Here the authors show that STING (stimulator of interferon genes) is crit. for the induction of IFN by non-CpG intracellular DNA species produced by various DNA pathogens after infection.  Murine embryonic fibroblasts, as well as antigen presenting cells such as macrophages and dendritic cells (exposed to intracellular B-form DNA, the DNA virus herpes simplex virus 1 (HSV-1) or bacteria Listeria monocytogenes), were found to require STING to initiate effective IFN prodn.  Accordingly, Sting-knockout mice were susceptible to lethal infection after exposure to HSV-1.  The importance of STING in facilitating DNA-mediated innate immune responses was further evident because cytotoxic T-cell responses induced by plasmid DNA vaccination were reduced in Sting-deficient animals.  In the presence of intracellular DNA, STING relocalized with TANK-binding kinase 1 (TBK1) from the endoplasmic reticulum to perinuclear vesicles contg. the exocyst component Sec5 (also known as EXOC2).  Collectively, the authors' studies indicate that STING is essential for host defense against DNA pathogens such as HSV-1 and facilitates the adjuvant activity of DNA-based vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozocwPRxdjlrVg90H21EOLACvtfcHk0lhw7bmyQhcXag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGqtbvI&md5=441c2a523b68fa98dfe5202364ec928b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature08476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08476%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DBarber%26aufirst%3DG.%2BN.%26atitle%3DSTING%2520regulates%2520intracellular%2520DNA-mediated%252C%2520type%2520I%2520interferon-dependent%2520innate%2520immunity%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D788%26epage%3D792%26doi%3D10.1038%2Fnature08476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ablasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldeck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deimling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witte, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röhl, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfner, K.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornung, V.</span></span> <span> </span><span class="NLM_article-title">cGAS produces a 2′-5′-linked cyclic dinucleotide second messenger that activates STING</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>498</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1038/nature12306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnature12306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=23722158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpslGhsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=498&publication_year=2013&pages=380-384&author=A.+Ablasserauthor=M.+Goldeckauthor=T.+Cavlarauthor=T.+Deimlingauthor=G.+Witteauthor=I.+R%C3%B6hlauthor=K.-P.+Hopfnerauthor=J.+Ludwigauthor=V.+Hornung&title=cGAS+produces+a+2%E2%80%B2-5%E2%80%B2-linked+cyclic+dinucleotide+second+messenger+that+activates+STING&doi=10.1038%2Fnature12306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">cGAS produces a 2'-5'-linked cyclic dinucleotide second messenger that activates STING</span></div><div class="casAuthors">Ablasser, Andrea; Goldeck, Marion; Cavlar, Taner; Deimling, Tobias; Witte, Gregor; Roehl, Ingo; Hopfner, Karl-Peter; Ludwig, Janos; Hornung, Veit</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">498</span>
        (<span class="NLM_cas:issue">7454</span>),
    <span class="NLM_cas:pages">380-384</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Detection of cytoplasmic DNA represents one of the most fundamental mechanisms of the innate immune system to sense the presence of microbial pathogens.  Moreover, erroneous detection of endogenous DNA by the same sensing mechanisms has an important pathophysiol. role in certain sterile inflammatory conditions.  The endoplasmic reticulum-resident protein STING is critically required for the initiation of type I interferon signalling upon detection of cytosolic DNA of both exogenous and endogenous origin.  Next to its pivotal role in DNA sensing, STING also serves as a direct receptor for the detection of cyclic dinucleotides, which function as second messenger mols. in bacteria.  DNA recognition, however, is triggered in an indirect fashion that depends on a recently characterized cytoplasmic nucleotidyl transferase, termed cGAMP synthase (cGAS), which upon interaction with DNA synthesizes a dinucleotide mol. that in turn binds to and activates STING.  We here show in vivo and in vitro that the cGAS-catalyzed reaction product is distinct from previously characterized cyclic dinucleotides.  Using a combinatorial approach based on mass spectrometry, enzymic digestion, NMR anal. and chem. synthesis, we demonstrate that cGAS produces a cyclic GMP-AMP dinucleotide, which comprises a 2'-5' and a 3'-5' phosphodiester linkage >Gp(2'-5')Ap(3'-5')>.  We found that the presence of this 2'-5' linkage was required to exert potent activation of human STING.  Moreover, we show that cGAS first catalyzes the synthesis of a linear 2'-5'-linked dinucleotide, which is then subject to cGAS-dependent cyclization in a second step through a 3'-5' phosphodiester linkage.  This 13-membered ring structure defines a novel class of second messenger mols., extending the family of 2'-5'-linked antiviral biomols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfAZc114JC5bVg90H21EOLACvtfcHk0lhw7bmyQhcXag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpslGhsrk%253D&md5=78743675ef6c36bb32d7eb80a2030c46</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature12306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12306%26sid%3Dliteratum%253Aachs%26aulast%3DAblasser%26aufirst%3DA.%26aulast%3DGoldeck%26aufirst%3DM.%26aulast%3DCavlar%26aufirst%3DT.%26aulast%3DDeimling%26aufirst%3DT.%26aulast%3DWitte%26aufirst%3DG.%26aulast%3DR%25C3%25B6hl%26aufirst%3DI.%26aulast%3DHopfner%26aufirst%3DK.-P.%26aulast%3DLudwig%26aufirst%3DJ.%26aulast%3DHornung%26aufirst%3DV.%26atitle%3DcGAS%2520produces%2520a%25202%25E2%2580%25B2-5%25E2%2580%25B2-linked%2520cyclic%2520dinucleotide%2520second%2520messenger%2520that%2520activates%2520STING%26jtitle%3DNature%26date%3D2013%26volume%3D498%26spage%3D380%26epage%3D384%26doi%3D10.1038%2Fnature12306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, H.-B.</span></span> <span> </span><span class="NLM_article-title">The adaptor protein MITA links virus-sensing receptors to IRF3 transcription factor activation</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2008.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.immuni.2008.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=18818105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlWntb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=538-550&author=B.+Zhongauthor=Y.+Yangauthor=S.+Liauthor=Y.-Y.+Wangauthor=Y.+Liauthor=F.+Diaoauthor=C.+Leiauthor=X.+Heauthor=L.+Zhangauthor=P.+Tienauthor=H.-B.+Shu&title=The+adaptor+protein+MITA+links+virus-sensing+receptors+to+IRF3+transcription+factor+activation&doi=10.1016%2Fj.immuni.2008.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The adaptor protein MITA links virus-sensing receptors to IRF3 transcription factor activation</span></div><div class="casAuthors">Zhong, Bo; Yang, Yan; Li, Shu; Wang, Yan-Yi; Li, Ying; Diao, Feici; Lei, Caoqi; He, Xiao; Zhang, Lu; Tien, Po; Shu, Hong-Bing</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">538-550</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Viral infection triggers activation of transcription factors such as NF-κB and IRF3, which collaborate to induce type I interferons (IFNs) and elicit innate antiviral response.  Here, the authors identified MITA as a crit. mediator of virus-triggered type I IFN signaling by expression cloning.  Overexpression of MITA activated IRF3, whereas knockdown of MITA inhibited virus-triggered activation of IRF3, expression of type I IFNs, and cellular antiviral response.  MITA was found to localize to the outer membrane of mitochondria and to be assocd. with VISA, a mitochondrial protein that acts as an adaptor in virus-triggered signaling.  MITA also interacted with IRF3 and recruited the kinase TBK1 to the VISA-assocd. complex.  MITA was phosphorylated by TBK1, which is required for MITA-mediated activation of IRF3.  The authors' results suggest that MITA is a crit. mediator of virus-triggered IRF3 activation and IFN expression and further demonstrate the importance of certain mitochondrial proteins in innate antiviral immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI7inp8jAxA7Vg90H21EOLACvtfcHk0lhw7bmyQhcXag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlWntb7P&md5=b4197f541f33cc09897cd5516d05eab3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2008.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2008.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.-Y.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDiao%26aufirst%3DF.%26aulast%3DLei%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DTien%26aufirst%3DP.%26aulast%3DShu%26aufirst%3DH.-B.%26atitle%3DThe%2520adaptor%2520protein%2520MITA%2520links%2520virus-sensing%2520receptors%2520to%2520IRF3%2520transcription%2520factor%2520activation%26jtitle%3DImmunity%26date%3D2008%26volume%3D29%26spage%3D538%26epage%3D550%26doi%3D10.1016%2Fj.immuni.2008.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balka, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calleja, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Silva, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moghaddas, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tailler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicks, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miner, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kile, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Nardo, D.</span></span> <span> </span><span class="NLM_article-title">TBK1 and IKKε Act Redundantly to Mediate STING-Induced NF-κB Responses in Myeloid Cells</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">107492</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2020.03.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.celrep.2020.03.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32268090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmvFSqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=107492&author=K.+R.+Balkaauthor=C.+Louisauthor=T.+L.+Saundersauthor=A.+M.+Smithauthor=D.+J.+Callejaauthor=D.+B.+D%E2%80%99Silvaauthor=F.+Moghaddasauthor=M.+Taillerauthor=K.+E.+Lawlorauthor=Y.+Zhanauthor=C.+J.+Burnsauthor=I.+P.+Wicksauthor=J.+J.+Minerauthor=B.+T.+Kileauthor=S.+L.+Mastersauthor=D.+De+Nardo&title=TBK1+and+IKK%CE%B5+Act+Redundantly+to+Mediate+STING-Induced+NF-%CE%BAB+Responses+in+Myeloid+Cells&doi=10.1016%2Fj.celrep.2020.03.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">TBK1 and IKKε Act Redundantly to Mediate STING-Induced NF-κB Responses in Myeloid Cells</span></div><div class="casAuthors">Balka, Katherine R.; Louis, Cynthia; Saunders, Tahnee L.; Smith, Amber M.; Calleja, Dale J.; D'Silva, Damian B.; Moghaddas, Fiona; Tailler, Maximilien; Lawlor, Kate E.; Zhan, Yifan; Burns, Christopher J.; Wicks, Ian P.; Miner, Jonathan J.; Kile, Benjamin T.; Masters, Seth L.; De Nardo, Dominic</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">107492</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Stimulator of Interferon Genes (STING) is a crit. component of host innate immune defense but can contribute to chronic autoimmune or autoinflammatory disease.  Once activated, the cyclic guanosine monophosphate (GMP)-adenosine monophosphate (AMP) (cGAMP) synthase (cGAS)-STING pathway induces both type I interferon (IFN) expression and nuclear factor-κB (NF-κB)-mediated cytokine prodn.  Currently, these two signaling arms are thought to be mediated by a single upstream kinase, TANK-binding kinase 1 (TBK1).  Here, using genetic and pharmacol. approaches, we show that TBK1 alone is dispensable for STING-induced NF-κB responses in human and mouse immune cells, as well as in vivo.  We further demonstrate that TBK1 acts redundantly with IκB kinase ε (IKKε) to drive NF-κB upon STING activation.  Interestingly, we show that activation of IFN regulatory factor 3 (IRF3) is highly dependent on TBK1 kinase activity, whereas NF-κB is significantly less sensitive to TBK1/IKKε kinase inhibition.  Our work redefines signaling events downstream of cGAS-STING.  Our findings further suggest that cGAS-STING will need to be targeted directly to effectively ameliorate the inflammation underpinning disorders assocd. with STING hyperactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor_LKR-rBD37Vg90H21EOLACvtfcHk0ljYSX5HYcx4BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmvFSqsbg%253D&md5=4483ba5d3616ac553c81a9fc6870bd75</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2020.03.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2020.03.056%26sid%3Dliteratum%253Aachs%26aulast%3DBalka%26aufirst%3DK.%2BR.%26aulast%3DLouis%26aufirst%3DC.%26aulast%3DSaunders%26aufirst%3DT.%2BL.%26aulast%3DSmith%26aufirst%3DA.%2BM.%26aulast%3DCalleja%26aufirst%3DD.%2BJ.%26aulast%3DD%25E2%2580%2599Silva%26aufirst%3DD.%2BB.%26aulast%3DMoghaddas%26aufirst%3DF.%26aulast%3DTailler%26aufirst%3DM.%26aulast%3DLawlor%26aufirst%3DK.%2BE.%26aulast%3DZhan%26aufirst%3DY.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26aulast%3DWicks%26aufirst%3DI.%2BP.%26aulast%3DMiner%26aufirst%3DJ.%2BJ.%26aulast%3DKile%26aufirst%3DB.%2BT.%26aulast%3DMasters%26aufirst%3DS.%2BL.%26aulast%3DDe%2BNardo%26aufirst%3DD.%26atitle%3DTBK1%2520and%2520IKK%25CE%25B5%2520Act%2520Redundantly%2520to%2520Mediate%2520STING-Induced%2520NF-%25CE%25BAB%2520Responses%2520in%2520Myeloid%2520Cells%26jtitle%3DCell%2520Rep.%26date%3D2020%26volume%3D31%26spage%3D107492%26doi%3D10.1016%2Fj.celrep.2020.03.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">Cyclic GMP-AMP synthase is a cytosolic DNA sensor that activates the type I interferon pathway</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>339</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">791</span>, <span class="refDoi"> DOI: 10.1126/science.1232458</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1126%2Fscience.1232458" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=23258413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1ygurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=786-791&author=L.+Sunauthor=J.+Wuauthor=F.+Duauthor=X.+Chenauthor=Z.+J.+Chen&title=Cyclic+GMP-AMP+synthase+is+a+cytosolic+DNA+sensor+that+activates+the+type+I+interferon+pathway&doi=10.1126%2Fscience.1232458"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic GMP-AMP Synthase Is a Cytosolic DNA Sensor That Activates the Type I Interferon Pathway</span></div><div class="casAuthors">Sun, Lijun; Wu, Jiaxi; Du, Fenghe; Chen, Xiang; Chen, Zhijian J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">339</span>
        (<span class="NLM_cas:issue">6121</span>),
    <span class="NLM_cas:pages">786-791</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The presence of DNA in the cytoplasm of mammalian cells is a danger signal that triggers host immune responses such as the prodn. of type I interferons.  Cytosolic DNA induces interferons through the prodn. of cyclic guanosine monophosphate-adenosine monophosphate (cyclic GMP-AMP, or cGAMP), which binds to and activates the adaptor protein STING.  Through biochem. fractionation and quant. mass spectrometry, we identified a cGAMP synthase (cGAS), which belongs to the nucleotidyltransferase family.  Overexpression of cGAS activated the transcription factor IRF3 and induced interferon-β in a STING-dependent manner.  Knockdown of cGAS inhibited IRF3 activation and interferon-β induction by DNA transfection or DNA virus infection.  CGAS bound to DNA in the cytoplasm and catalyzed cGAMP synthesis.  These results indicate that cGAS is a cytosolic DNA sensor that induces interferons by producing the second messenger cGAMP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxxUcksTNZRbVg90H21EOLACvtfcHk0ljYSX5HYcx4BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1ygurw%253D&md5=fe9a64a9c0441eb00852900be8d683bf</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1232458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1232458%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DCyclic%2520GMP-AMP%2520synthase%2520is%2520a%2520cytosolic%2520DNA%2520sensor%2520that%2520activates%2520the%2520type%2520I%2520interferon%2520pathway%26jtitle%3DScience%26date%3D2013%26volume%3D339%26spage%3D786%26epage%3D791%26doi%3D10.1126%2Fscience.1232458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, R. P.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohler, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlauckas, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadreyev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgibbons, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchison, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libby, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahrendorf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">IRF3 and type I interferons fuel a fatal response to myocardial infarction</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1481</span>– <span class="NLM_lpage">1487</span>, <span class="refDoi"> DOI: 10.1038/nm.4428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnm.4428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=29106401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslKmurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1481-1487&author=K.+R.+Kingauthor=A.+D.+Aguirreauthor=Y.-X.+Yeauthor=Y.+Sunauthor=J.+D.+Rohauthor=R.+P.+Ngauthor=R.+H.+Kohlerauthor=S.+P.+Arlauckasauthor=Y.+Iwamotoauthor=A.+Savolauthor=R.+I.+Sadreyevauthor=M.+Kellyauthor=T.+P.+Fitzgibbonsauthor=K.+A.+Fitzgeraldauthor=T.+Mitchisonauthor=P.+Libbyauthor=M.+Nahrendorfauthor=R.+Weissleder&title=IRF3+and+type+I+interferons+fuel+a+fatal+response+to+myocardial+infarction&doi=10.1038%2Fnm.4428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">IRF3 and type I interferons fuel a fatal response to myocardial infarction</span></div><div class="casAuthors">King, Kevin R.; Aguirre, Aaron D.; Ye, Yu-Xiang; Sun, Yuan; Roh, Jason D.; Ng, Richard P., Jr.; Kohler, Rainer H.; Arlauckas, Sean P.; Iwamoto, Yoshiko; Savol, Andrej; Sadreyev, Ruslan I.; Kelly, Mark; Fitzgibbons, Timothy P.; Fitzgerald, Katherine A.; Mitchison, Timothy; Libby, Peter; Nahrendorf, Matthias; Weissleder, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1481-1487</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Interferon regulatory factor 3 (IRF3) and type I interferons (IFNs) protect against infections and cancer, but excessive IRF3 activation and type I IFN prodn. cause autoinflammatory conditions such as Aicardi-Goutieacures syndrome and STING-assocd. vasculopathy of infancy (SAVI).  Myocardial infarction (MI) elicits inflammation, but the dominant mol. drivers of MI-assocd. inflammation remain unclear.  Here we show that ischemic cell death and uptake of cell debris by macrophages in the heart fuel a fatal response to MI by activating IRF3 and type I IFN prodn.  In mice, single-cell RNA-seq anal. of 4,215 leukocytes isolated from infarcted and non-infarcted hearts showed that MI provokes activation of an IRF3-interferon axis in a distinct population of interferon-inducible cells (IFNICs) that were classified as cardiac macrophages.  Mice genetically deficient in cyclic GMP-AMP synthase (cGAS), its adaptor STING, IRF3, or the type I IFN receptor IFNAR exhibited impaired interferon-stimulated gene (ISG) expression and, in the case of mice deficient in IRF3 or IFNAR, improved survival after MI as compared to controls.  Interruption of IRF3-dependent signaling resulted in decreased cardiac expression of inflammatory cytokines and chemokines and decreased inflammatory cell infiltration of the heart, as well as in attenuated ventricular dilation and improved cardiac function.  Similarly, treatment of mice with an IFNAR-neutralizing antibody after MI ablated the interferon response and improved left ventricular dysfunction and survival.  These results identify IRF3 and the type I IFN response as a potential therapeutic target for post-MI cardioprotection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwGZt2egB0fLVg90H21EOLACvtfcHk0li4jJhBItvJ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslKmurrI&md5=dbadb78b95ef1c1c8164c5fdb96af317</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnm.4428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4428%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DK.%2BR.%26aulast%3DAguirre%26aufirst%3DA.%2BD.%26aulast%3DYe%26aufirst%3DY.-X.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DRoh%26aufirst%3DJ.%2BD.%26aulast%3DNg%26aufirst%3DR.%2BP.%26aulast%3DKohler%26aufirst%3DR.%2BH.%26aulast%3DArlauckas%26aufirst%3DS.%2BP.%26aulast%3DIwamoto%26aufirst%3DY.%26aulast%3DSavol%26aufirst%3DA.%26aulast%3DSadreyev%26aufirst%3DR.%2BI.%26aulast%3DKelly%26aufirst%3DM.%26aulast%3DFitzgibbons%26aufirst%3DT.%2BP.%26aulast%3DFitzgerald%26aufirst%3DK.%2BA.%26aulast%3DMitchison%26aufirst%3DT.%26aulast%3DLibby%26aufirst%3DP.%26aulast%3DNahrendorf%26aufirst%3DM.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DIRF3%2520and%2520type%2520I%2520interferons%2520fuel%2520a%2520fatal%2520response%2520to%2520myocardial%2520infarction%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1481%26epage%3D1487%26doi%3D10.1038%2Fnm.4428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Motwani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesiridis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, K. A.</span></span> <span> </span><span class="NLM_article-title">DNA sensing by the cGAS-STING pathway in health and disease</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1038/s41576-019-0151-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41576-019-0151-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=31358977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVGqtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=657-674&author=M.+Motwaniauthor=S.+Pesiridisauthor=K.+A.+Fitzgerald&title=DNA+sensing+by+the+cGAS-STING+pathway+in+health+and+disease&doi=10.1038%2Fs41576-019-0151-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DNA sensing by the cGAS-STING pathway in health and disease</span></div><div class="casAuthors">Motwani, Mona; Pesiridis, Scott; Fitzgerald, Katherine A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">657-674</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The detection of pathogens through nucleic acid sensors is a defining principle of innate immunity.  RNA-sensing and DNA-sensing receptors sample subcellular compartments for foreign nucleic acids and, upon recognition, trigger immune signalling pathways for host defense.  Over the past decade, our understanding of how the recognition of nucleic acids is coupled to immune gene expression has advanced considerably, particularly for the DNA-sensing receptor cyclic GMP-AMP synthase (cGAS) and its downstream signalling effector stimulator of interferon genes (STING), as well as the mol. components and regulation of this pathway.  Moreover, the ability of self-DNA to engage cGAS has emerged as an important mechanism fuelling the development of inflammation and implicating the cGAS-STING pathway in human inflammatory diseases and cancer.  This detailed mechanistic and biol. understanding is paving the way for the development and clin. application of pharmacol. agonists and antagonists in the treatment of chronic inflammation and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA8x0aezpQcLVg90H21EOLACvtfcHk0li4jJhBItvJ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVGqtbvM&md5=f13ade7f14b63b453192b5d05f7c9f0b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41576-019-0151-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41576-019-0151-1%26sid%3Dliteratum%253Aachs%26aulast%3DMotwani%26aufirst%3DM.%26aulast%3DPesiridis%26aufirst%3DS.%26aulast%3DFitzgerald%26aufirst%3DK.%2BA.%26atitle%3DDNA%2520sensing%2520by%2520the%2520cGAS-STING%2520pathway%2520in%2520health%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2019%26volume%3D20%26spage%3D657%26epage%3D674%26doi%3D10.1038%2Fs41576-019-0151-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wight-Carter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">Activation of cyclic GMP-AMP synthase by self-DNA causes autoimmune diseases</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E5699</span>– <span class="NLM_lpage">E5705</span>, <span class="refDoi"> DOI: 10.1073/pnas.1516465112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1073%2Fpnas.1516465112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=26371324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E5699-E5705&author=D.+Gaoauthor=T.+Liauthor=X.-D.+Liauthor=X.+Chenauthor=Q.-Z.+Liauthor=M.+Wight-Carterauthor=Z.+J.+Chen&title=Activation+of+cyclic+GMP-AMP+synthase+by+self-DNA+causes+autoimmune+diseases&doi=10.1073%2Fpnas.1516465112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of cyclic GMP-AMP synthase by self-DNA causes autoimmune diseases</span></div><div class="casAuthors">Gao, Daxing; Li, Tuo; Li, Xiao-Dong; Chen, Xiang; Li, Quan-Zhen; Wight-Carter, Mary; Chen, Zhijian J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">E5699-E5705</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">TREX1 is an exonuclease that digests DNA in the cytoplasm.  Loss-of-function mutations of TREX1 are linked to Aicardi-Goutieres Syndrome (AGS) and systemic lupus erythematosus (SLE) in humans.  Trex1-/- mice exhibit autoimmune and inflammatory phenotypes that are assocd. with elevated expression of interferon (IFN)-induced genes (ISGs).  CGMP-AMP (cGAMP) synthase (cGAS) is a cytosolic DNA sensor that activates the IFN pathway.  Upon binding to DNA, cGAS is activated to catalyze the synthesis of cGAMP, which functions as a second messenger that binds and activates the adaptor protein STING to induce IFNs and other cytokines.  Here the authors show that genetic ablation of cGas in Trex1-/- mice eliminated all detectable pathol. and mol. phenotypes, including ISG induction, autoantibody prodn., aberrant T-cell activation, and lethality.  Even deletion of just one allele of cGas largely rescued the phenotypes of Trex1-/- mice.  Similarly, deletion of cGas in mice lacking DNaseII, a lysosomal enzyme that digests DNA, rescued the lethal autoimmune phenotypes of the DNaseII-/- mice.  Through quant. mass spectrometry, the authors found that cGAMP accumulated in mouse tissues deficient in Trex1 or DNaseII and that this accumulation was dependent on cGAS.  These results demonstrate that cGAS activation causes the autoimmune diseases in Trex1-/- and DNaseII-/- mice and suggest that inhibition of cGAS may lead to prevention and treatment of some human autoimmune diseases caused by self-DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoJqBQwGiNVLVg90H21EOLACvtfcHk0lj-2yQia_Pm6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntLnN&md5=1d4056b35c389e3d7fb7a07f85af4cd7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1516465112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1516465112%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.-D.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.-Z.%26aulast%3DWight-Carter%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DActivation%2520of%2520cyclic%2520GMP-AMP%2520synthase%2520by%2520self-DNA%2520causes%2520autoimmune%2520diseases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3DE5699%26epage%3DE5705%26doi%3D10.1073%2Fpnas.1516465112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crow, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manel, N.</span></span> <span> </span><span class="NLM_article-title">Aicardi-Goutières syndrome and the type I interferonopathies</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nri3850</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnri3850" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=26052098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=429-440&author=Y.+J.+Crowauthor=N.+Manel&title=Aicardi-Gouti%C3%A8res+syndrome+and+the+type+I+interferonopathies&doi=10.1038%2Fnri3850"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Aicardi-Goutieres syndrome and the type I interferonopathies</span></div><div class="casAuthors">Crow, Yanick J.; Manel, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">429-440</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dissection of the genetic basis of Aicardi-Goutieres syndrome has highlighted a fundamental link between nucleic acid metab., innate immune sensors and type I interferon induction.  This had led to the concept of the human interferonopathies as a broader set of Mendelian disorders in which a constitutive upregulation of type I interferon activity directly relates to disease pathol.  Here, we discuss the mol. and cellular basis of the interferonopathies, their categorization, future treatment strategies and the insights they provide into normal physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy0LI6TmTndrVg90H21EOLACvtfcHk0lj-2yQia_Pm6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zJ&md5=5e1414daa0c2b4d55dabe2d205b65b12</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnri3850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri3850%26sid%3Dliteratum%253Aachs%26aulast%3DCrow%26aufirst%3DY.%2BJ.%26aulast%3DManel%26aufirst%3DN.%26atitle%3DAicardi-Gouti%25C3%25A8res%2520syndrome%2520and%2520the%2520type%2520I%2520interferonopathies%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2015%26volume%3D15%26spage%3D429%26epage%3D440%26doi%3D10.1038%2Fnri3850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grieves, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fye, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grayson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrino, F. W.</span></span> <span> </span><span class="NLM_article-title">Exonuclease TREX1 degrades double-stranded DNA to prevent spontaneous lupus-like inflammatory disease</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">5117</span>– <span class="NLM_lpage">5122</span>, <span class="refDoi"> DOI: 10.1073/pnas.1423804112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1073%2Fpnas.1423804112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=25848017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVGrsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=5117-5122&author=J.+L.+Grievesauthor=J.+M.+Fyeauthor=S.+Harveyauthor=J.+M.+Graysonauthor=T.+Hollisauthor=F.+W.+Perrino&title=Exonuclease+TREX1+degrades+double-stranded+DNA+to+prevent+spontaneous+lupus-like+inflammatory+disease&doi=10.1073%2Fpnas.1423804112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exonuclease TREX1 degrades double-stranded DNA to prevent spontaneous lupus-like inflammatory disease</span></div><div class="casAuthors">Grieves, Jessica L.; Fye, Jason M.; Harvey, Scott; Grayson, Jason M.; Hollis, Thomas; Perrino, Fred W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5117-5122</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The TREX1 gene encodes a potent DNA exonuclease, and mutations in TREX1 cause a spectrum of lupus-like autoimmune diseases.  Most lupus patients develop autoantibodies to double-stranded DNA (dsDNA), but the source of DNA antigen is unknown.  The TREX1 D18N mutation causes a monogenic, cutaneous form of lupus called familial chilblain lupus, and the TREX1 D18N enzyme exhibits dysfunctional dsDNA-degrading activity, providing a link between dsDNA degrdn. and nucleic acid-mediated autoimmune disease.  We detd. the structure of the TREX1 D18N protein in complex with dsDNA, revealing how this exonuclease uses a novel DNA-unwinding mechanism to sep. the polynucleotide strands for single-stranded DNA (ssDNA) loading into the active site.  The TREX1 D18N dsDNA interactions coupled with catalytic deficiency explain how this mutant nuclease prevents dsDNA degrdn.  We tested the effects of TREX1 D18N in vivo by replacing the TREX1 WT gene in mice with the TREX1 D18N allele.  The TREX1 D18N mice exhibit systemic inflammation, lymphoid hyperplasia, vasculitis, and kidney disease.  The obsd. lupus-like inflammatory disease is assocd. with immune activation, prodn. of autoantibodies to dsDNA, and deposition of immune complexes in the kidney.  Thus, dysfunctional dsDNA degrdn. by TREX1 D18N induces disease in mice that recapitulates many characteristics of human lupus.  Failure to clear DNA has long been linked to lupus in humans, and these data point to dsDNA as a key substrate for TREX1 and a major antigen source in mice with dysfunctional TREX1 enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjNgPb6OeTz7Vg90H21EOLACvtfcHk0lj-2yQia_Pm6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVGrsL4%253D&md5=fe487e2c7fc4a4ad0421da0351510c2d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1423804112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1423804112%26sid%3Dliteratum%253Aachs%26aulast%3DGrieves%26aufirst%3DJ.%2BL.%26aulast%3DFye%26aufirst%3DJ.%2BM.%26aulast%3DHarvey%26aufirst%3DS.%26aulast%3DGrayson%26aufirst%3DJ.%2BM.%26aulast%3DHollis%26aufirst%3DT.%26aulast%3DPerrino%26aufirst%3DF.%2BW.%26atitle%3DExonuclease%2520TREX1%2520degrades%2520double-stranded%2520DNA%2520to%2520prevent%2520spontaneous%2520lupus-like%2520inflammatory%2520disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3D5117%26epage%3D5122%26doi%3D10.1073%2Fpnas.1423804112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gray, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treuting, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stetson, D. B.</span></span> <span> </span><span class="NLM_article-title">Cutting Edge: cGAS Is Required for Lethal Autoimmune Disease in the Trex1-Deficient Mouse Model of Aicardi-Goutières Syndrome</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">1939</span>– <span class="NLM_lpage">1943</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1500969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.4049%2Fjimmunol.1500969" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=26223655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOju7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2015&pages=1939-1943&author=E.+E.+Grayauthor=P.+M.+Treutingauthor=J.+J.+Woodwardauthor=D.+B.+Stetson&title=Cutting+Edge%3A+cGAS+Is+Required+for+Lethal+Autoimmune+Disease+in+the+Trex1-Deficient+Mouse+Model+of+Aicardi-Gouti%C3%A8res+Syndrome&doi=10.4049%2Fjimmunol.1500969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Cutting Edge: cGAS Is Required for Lethal Autoimmune Disease in the Trex1-Deficient Mouse Model of Aicardi-Goutieres Syndrome</span></div><div class="casAuthors">Gray, Elizabeth E.; Treuting, Piper M.; Woodward, Joshua J.; Stetson, Daniel B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1939-1943</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Detection of intracellular DNA triggers activation of the stimulator of IFN genes-dependent IFN-stimulatory DNA (ISD) pathway, which is essential for antiviral immune responses.  However, chronic activation of this pathway is implicated in autoimmunity.  Mutations in TREX1, a 3' repair exonuclease that degrades cytosolic DNA, cause Aicardi-Goutieres syndrome and chilblain lupus.  Trex1-/- mice develop lethal, IFN-driven autoimmune disease that is dependent on activation of the ISD pathway, but the DNA sensors that detect the endogenous DNA that accumulates in Trex1-/- mice have not been defined.  Multiple DNA sensors have been proposed to activate the ISD pathway, including cyclic GMP-AMP synthase (cGAS).  In this study, we show that Trex1-/- mice lacking cGAS are completely protected from lethality, exhibit dramatically reduced tissue inflammation, and fail to develop autoantibodies.  These findings implicate cGAS as a key driver of autoimmune disease and suggest that cGAS inhibitors may be useful therapeutics for Aicardi-Goutieres syndrome and related autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS0F91-JISBbVg90H21EOLACvtfcHk0liOMRD788heSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOju7nN&md5=c455112d5b482de300222cb406b48086</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1500969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1500969%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DE.%2BE.%26aulast%3DTreuting%26aufirst%3DP.%2BM.%26aulast%3DWoodward%26aufirst%3DJ.%2BJ.%26aulast%3DStetson%26aufirst%3DD.%2BB.%26atitle%3DCutting%2520Edge%253A%2520cGAS%2520Is%2520Required%2520for%2520Lethal%2520Autoimmune%2520Disease%2520in%2520the%2520Trex1-Deficient%2520Mouse%2520Model%2520of%2520Aicardi-Gouti%25C3%25A8res%2520Syndrome%26jtitle%3DJ.%2520Immunol.%26date%3D2015%26volume%3D195%26spage%3D1939%26epage%3D1943%26doi%3D10.4049%2Fjimmunol.1500969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rice, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodero, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crow, Y. J.</span></span> <span> </span><span class="NLM_article-title">Human disease phenotypes associated with mutations in TREX1</span>. <i>J. Clin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">243</span>, <span class="refDoi"> DOI: 10.1007/s10875-015-0147-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1007%2Fs10875-015-0147-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=25731743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1yhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=235-243&author=G.+I.+Riceauthor=M.+P.+Roderoauthor=Y.+J.+Crow&title=Human+disease+phenotypes+associated+with+mutations+in+TREX1&doi=10.1007%2Fs10875-015-0147-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Human Disease Phenotypes Associated With Mutations in TREX1</span></div><div class="casAuthors">Rice, Gillian I.; Rodero, Mathieu P.; Crow, Yanick J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">235-243</span>CODEN:
                <span class="NLM_cas:coden">JCIMDO</span>;
        ISSN:<span class="NLM_cas:issn">0271-9142</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Considering that it is a single exon gene encoding a 314 amino acid protein, the genotype-phenotype landscape of TREX1 is remarkably complex.  Here we briefly describe the human diseases so-far assocd. with mutations in TREX1, which include Aicardi-Goutie´res syndrome, familial chilblain lupus, systemic lupus erythematosus and retinal vasculopathy with cerebral leukodystrophy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuIZPKg1vcJ7Vg90H21EOLACvtfcHk0liOMRD788heSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1yhu7w%253D&md5=436ccc68f00d6c7517e9c497b5639848</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10875-015-0147-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10875-015-0147-3%26sid%3Dliteratum%253Aachs%26aulast%3DRice%26aufirst%3DG.%2BI.%26aulast%3DRodero%26aufirst%3DM.%2BP.%26aulast%3DCrow%26aufirst%3DY.%2BJ.%26atitle%3DHuman%2520disease%2520phenotypes%2520associated%2520with%2520mutations%2520in%2520TREX1%26jtitle%3DJ.%2520Clin.%2520Immunol.%26date%3D2015%26volume%3D35%26spage%3D235%26epage%3D243%26doi%3D10.1007%2Fs10875-015-0147-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ablasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">cGAS in action: expanding roles in immunity and inflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>363</i></span>, <span class="NLM_elocation-id">eaat8657</span> <span class="refDoi"> DOI: 10.1126/science.aat8657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1126%2Fscience.aat8657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=30846571" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=2019&author=A.+Ablasserauthor=Z.+J.+Chen&title=cGAS+in+action%3A+expanding+roles+in+immunity+and+inflammation&doi=10.1126%2Fscience.aat8657"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscience.aat8657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aat8657%26sid%3Dliteratum%253Aachs%26aulast%3DAblasser%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DcGAS%2520in%2520action%253A%2520expanding%2520roles%2520in%2520immunity%2520and%2520inflammation%26jtitle%3DScience%26date%3D2019%26volume%3D363%26doi%3D10.1126%2Fscience.aat8657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, J.</span></span> <span> </span><span class="NLM_article-title">Research advances in how the cGAS-STING pathway controls the cellular inflammatory response</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">615</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2020.00615</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.3389%2Ffimmu.2020.00615" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32411126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVWntr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=615&author=D.+Wanauthor=W.+Jiangauthor=J.+Hao&title=Research+advances+in+how+the+cGAS-STING+pathway+controls+the+cellular+inflammatory+response&doi=10.3389%2Ffimmu.2020.00615"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Research advances in how the cGAS-STING pathway controls the cellular inflammatory response</span></div><div class="casAuthors">Wan, Dongshan; Jiang, Wei; Hao, Junwei</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">615</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review: Double-stranded DNA (dsDNA) sensor cyclic-GMP-AMP synthase (cGAS) along with the downstream stimulator of interferon genes (STING) acting as essential immunesurveillance mediators have become hot topics of research.  The intrinsic function of the cGAS-STING pathway facilitates type-I interferon (IFN) inflammatory signaling responses and other cellular processes such as autophagy, cell survival, senescence. cGASSTING pathway interplays with other innate immune pathways, by which it participates in regulating infection, inflammatory disease, and cancer.  The therapeutic approaches targeting this pathway show promise for future translation into clin. applications.  Here, we present a review of the important previous works and recent advances regarding the cGAS-STING pathway, and provide a comprehensive understanding of the modulatory pattern of the cGAS-STING pathway under multifarious pathol. states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoakdOHwmfKw7Vg90H21EOLACvtfcHk0liOMRD788heSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVWntr%252FJ&md5=24d9e8dbfa4781c56444ffbb77b782f6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2020.00615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2020.00615%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DHao%26aufirst%3DJ.%26atitle%3DResearch%2520advances%2520in%2520how%2520the%2520cGAS-STING%2520pathway%2520controls%2520the%2520cellular%2520inflammatory%2520response%26jtitle%3DFront.%2520Immunol.%26date%3D2020%26volume%3D11%26spage%3D615%26doi%3D10.3389%2Ffimmu.2020.00615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of cyclic GMP-AMP synthase ameliorates sepsis-induced cardiac dysfunction in mice</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>260</i></span>,  <span class="NLM_fpage">118315</span>, <span class="refDoi"> DOI: 10.1016/j.lfs.2020.118315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.lfs.2020.118315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32835697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslGqtrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=260&publication_year=2020&pages=118315&author=Q.+Xuauthor=H.+Xiongauthor=W.+Zhuauthor=Y.+Liuauthor=Y.+Du&title=Small+molecule+inhibition+of+cyclic+GMP-AMP+synthase+ameliorates+sepsis-induced+cardiac+dysfunction+in+mice&doi=10.1016%2Fj.lfs.2020.118315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of cyclic GMP-AMP synthase ameliorates sepsis-induced cardiac dysfunction in mice</span></div><div class="casAuthors">Xu, Qingqin; Xiong, Haolan; Zhu, Wenxiu; Liu, Yiping; Du, Yun</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">260</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">118315</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cardiac dysfunction is the main cause of multi-organ failure following sepsis within crit. care units.  The present study aimed to investigate the effects of the small mol. inhibition of cyclic GMP-AMP synthase (cGAS), RU.521, on cardiac function in mice with sepsis.  Sepsis was induced in mice via i.p. lipopolysaccharide (LPS) injection (10 mg/kg, i.p.).  Mice subsequently received 5 mg/kg RU.521 within 10 min form LPS injection.  The cardiac function, inflammatory factor and oxidative stress of mice were examd. for 24 h following LPS injection.RU.521 was indicated to significantly increase the cardiac function of mice with sepsis.  In addn., the inflammatory responses, oxidative stress and apoptosis in hearts of sepsis mice were markedly mitigated by RU.521.  Moreover, inhibition of Sirt3 inhibited the protective effects of RU.521 on mice with sepsis.  The current study indicated that RU.521 alleviated the inflammatory response and alleviated the damage induced by oxidative stress, leading to cardiac protection via increased Sirt3 expression in the hearts of mice with sepsis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrokNqwpfzO0bVg90H21EOLACvtfcHk0lgWrby3VGP76g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslGqtrjJ&md5=8c323b596e895377eee8c15ba1375a18</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.lfs.2020.118315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lfs.2020.118315%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520cyclic%2520GMP-AMP%2520synthase%2520ameliorates%2520sepsis-induced%2520cardiac%2520dysfunction%2520in%2520mice%26jtitle%3DLife%2520Sci.%26date%3D2020%26volume%3D260%26spage%3D118315%26doi%3D10.1016%2Fj.lfs.2020.118315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hertzog, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehwinkel, J.</span></span> <span> </span><span class="NLM_article-title">Regulation and inhibition of the DNA sensor cGAS</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>, <span class="NLM_elocation-id">e51345</span> <span class="refDoi"> DOI: 10.15252/embr.202051345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.15252%2Fembr.202051345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=33155371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXit1KhsrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&author=J.+Hertzogauthor=J.+Rehwinkel&title=Regulation+and+inhibition+of+the+DNA+sensor+cGAS&doi=10.15252%2Fembr.202051345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation and inhibition of the DNA sensor cGAS</span></div><div class="casAuthors">Hertzog, Jonny; Rehwinkel, Jan</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e51345</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cell-autonomous sensing of nucleic acids is essential for host defense against invading pathogens by inducing antiviral and inflammatory cytokines. cGAS has emerged in recent years as a non-redundant DNA sensor important for detection of many viruses and bacteria.  Upon binding to DNA, cGAS synthesizes the cyclic dinucleotide 2'3'-cGAMP that binds to the adaptor protein STING and thereby triggers IRF3- and NFκB-dependent transcription.  In addn. to infection, the pathophysiol. of an ever-increasing no. of sterile inflammatory conditions in humans involves the recognition of DNA through cGAS.  Consequently, the cGAS/STING signalling axis has emerged as an attractive target for pharmacol. modulation.  However, the development of cGAS and STING inhibitors has just begun and a need for specific and effective compds. persists.  In this review, we focus on cGAS and explore how its activation by immunostimulatory DNA is regulated by cellular mechanisms, viral immune modulators and small mols.  We further use our knowledge of cGAS modulation by cells and viruses to conceptualise potential new ways of pharmacol. cGAS targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOCYhBccK5i7Vg90H21EOLACvtfcHk0lgWrby3VGP76g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXit1KhsrbP&md5=519e9236e497e12fc623cd1312ba7fee</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.15252%2Fembr.202051345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembr.202051345%26sid%3Dliteratum%253Aachs%26aulast%3DHertzog%26aufirst%3DJ.%26aulast%3DRehwinkel%26aufirst%3DJ.%26atitle%3DRegulation%2520and%2520inhibition%2520of%2520the%2520DNA%2520sensor%2520cGAS%26jtitle%3DEMBO%2520Rep.%26date%3D2020%26volume%3D21%26doi%3D10.15252%2Fembr.202051345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span> <span> </span><span class="NLM_article-title">Small molecules targeting the innate immune cGAS–STING–TBK1 signaling pathway</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2272</span>– <span class="NLM_lpage">2298</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.apsb.2020.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=33354501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3MXht1Ciur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=2272-2298&author=C.+Dingauthor=Z.+Songauthor=A.+Shenauthor=T.+Chenauthor=A.+Zhang&title=Small+molecules+targeting+the+innate+immune+cGAS%E2%80%93STING%E2%80%93TBK1+signaling+pathway&doi=10.1016%2Fj.apsb.2020.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules targeting the innate immune cGAS-STING-TBK1 signaling pathway</span></div><div class="casAuthors">Ding, Chunyong; Song, Zilan; Shen, Ancheng; Chen, Tingting; Zhang, Ao</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmaceutica Sinica B</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2272-2298</span>CODEN:
                <span class="NLM_cas:coden">APSBCW</span>;
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Multiple cancer immunotherapies including chimeric antigen receptor T cell and immune checkpoint inhibitors have been successfully developed to treat various cancers by motivating the adaptive anti-tumor immunity.  Particularly, the checkpoint blockade approach has achieved great clinic success as evidenced by several U.S.  Food and drug administration-approved antiprogram death receptor 1/ligand 1 or anti-cytotoxic T lymphocyte assocd. protein 4 antibodies.  However, the majority of cancers have low clin. response rates to these ICIs due to poor tumor immunogenicity.  Indeed, the cyclic guanosine monophosphate-adenosine monophosphate synthase-stimulator of interferon genes-TANK-binding kinase 1 (cGAS-STING-TBK1) axis is now appreciated as the major signaling pathway in innate immune response across different species.  Aberrant signaling of this pathway has been closely linked to multiple diseases, including auto-inflammation, virus infection and cancers.  In this perspective, we provide an updated review on the latest progress on the development of small mol. modulators targeting the cGAS-STING-TBK1 signaling pathway and their preclin. and clin. use as a new immune stimulatory therapy.  Meanwhile, highlights on the clin. candidates, limitations and challenges, as well as future directions in this field are also discussed.  Further, small mol. inhibitors targeting this signaling axis and their potential therapeutic use for various indications are discussed as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZQi2de6gBbVg90H21EOLACvtfcHk0lgWrby3VGP76g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXht1Ciur8%253D&md5=768d27bbb1096ec9f94133ef965a49d1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DA.%26atitle%3DSmall%2520molecules%2520targeting%2520the%2520innate%2520immune%2520cGAS%25E2%2580%2593STING%25E2%2580%2593TBK1%2520signaling%2520pathway%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26volume%3D10%26spage%3D2272%26epage%3D2298%26doi%3D10.1016%2Fj.apsb.2020.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkon, K. B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Cyclic GMP-AMP Synthase Using a Novel Antimalarial Drug Derivative in Trex1 -Deficient Mice</span>. <i>Arthritis Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">1807</span>– <span class="NLM_lpage">1819</span>, <span class="refDoi"> DOI: 10.1002/art.40559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1002%2Fart.40559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=29781188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVartbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=1807-1819&author=J.+Anauthor=J.+J.+Woodwardauthor=W.+Laiauthor=M.+Minieauthor=X.+Sunauthor=L.+Tanakaauthor=J.+M.+Snyderauthor=T.+Sasakiauthor=K.+B.+Elkon&title=Inhibition+of+Cyclic+GMP-AMP+Synthase+Using+a+Novel+Antimalarial+Drug+Derivative+in+Trex1+-Deficient+Mice&doi=10.1002%2Fart.40559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Cyclic GMP-AMP Synthase Using a Novel Antimalarial Drug Derivative in Trex1-Deficient Mice</span></div><div class="casAuthors">An, Jie; Woodward, Joshua J.; Lai, Weinan; Minie, Mark; Sun, Xizhang; Tanaka, Lena; Snyder, Jessica M.; Sasaki, Tomikazu; Elkon, Keith B.</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1807-1819</span>CODEN:
                <span class="NLM_cas:coden">ARRHBO</span>;
        ISSN:<span class="NLM_cas:issn">2326-5191</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Type I interferon (IFN) is strongly implicated in the pathogenesis of systemic lupus erythematosus (SLE) as well as rare monogenic interferonopathies such as Aicardi-Goutieres syndrome (AGS), a disease attributed to mutations in the DNA exonuclease TREX1.  The DNA-activated type I IFN pathway cyclic GMP-AMP (cGAMP) synthase (cGAS) is linked to subsets of AGS and lupus.  This study was undertaken to identify inhibitors of the DNA-cGAS interaction, and to test the lead candidate drug, X6, in a mouse model of AGS.  Trex1-/- mice were treated orally from birth with either X6 or hydroxychloroquine (HCQ) for 8 wk.  Expression of IFN-stimulated genes (ISGs) was quantified by quant. polymerase chain reaction.  Multiple reaction monitoring by ultra-performance liq. chromatog. coupled with tandem mass spectrometry was used to quantify the prodn. of cGAMP and X6 drug concns. in the serum and heart tissue of Trex1-/- mice.  On the basis of the efficacy-to-toxicity ratio established in vitro, drug X6 was selected as the lead candidate for treatment of Trex1-/- mice.  X6 was significantly more effective than HCQ in attenuating ISG expression in mouse spleens (P < 0.01 for Isg15 and Isg20) and hearts (P < 0.05 for Isg15,Mx1, and Ifnb, and P < 0.01 for Cxcl10), and in reducing the prodn. of cGAMP in mouse heart tissue (P < 0.05), thus demonstrating target engagement by the X6 compd.  Of note, X6 was also more effective than HCQ in reducing ISG expression in vitro (P < 0.05 for IFI27 and MX1, and P < 0.01 for IFI44L and PKR) in human peripheral blood mononuclear cells from patients with SLE.  This study demonstrates that X6 is superior to HCQ for the treatment of an exptl. autoimmune myocarditis mediated in vivo by the cGAS/stimulator of IFN genes (cGAS/STING) pathway.  The findings suggest that drug X6 could be developed as a novel treatment for AGS and/or lupus to inhibit activation of the cGAS/STING pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0sJrbIuCqErVg90H21EOLACvtfcHk0liMAwt1aEVgoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVartbjL&md5=d6183689e4227339f6a41ef1f9bbaae2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fart.40559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.40559%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DWoodward%26aufirst%3DJ.%2BJ.%26aulast%3DLai%26aufirst%3DW.%26aulast%3DMinie%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DTanaka%26aufirst%3DL.%26aulast%3DSnyder%26aufirst%3DJ.%2BM.%26aulast%3DSasaki%26aufirst%3DT.%26aulast%3DElkon%26aufirst%3DK.%2BB.%26atitle%3DInhibition%2520of%2520Cyclic%2520GMP-AMP%2520Synthase%2520Using%2520a%2520Novel%2520Antimalarial%2520Drug%2520Derivative%2520in%2520Trex1%2520-Deficient%2520Mice%26jtitle%3DArthritis%2520Rheumatol.%26date%3D2018%26volume%3D70%26spage%3D1807%26epage%3D1819%26doi%3D10.1002%2Fart.40559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sooreshjani, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opoku-Temeng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintim, H. O.</span></span> <span> </span><span class="NLM_article-title">Suramin potently inhibits cGAMP synthase, cGAS, in THP1 cells to modulate IFN-β levels</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1301</span>– <span class="NLM_lpage">1317</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.4155%2Ffmc-2017-0322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=29558821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVyqsrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=1301-1317&author=M.+Wangauthor=M.+A.+Sooreshjaniauthor=C.+Mikekauthor=C.+Opoku-Temengauthor=H.+O.+Sintim&title=Suramin+potently+inhibits+cGAMP+synthase%2C+cGAS%2C+in+THP1+cells+to+modulate+IFN-%CE%B2+levels&doi=10.4155%2Ffmc-2017-0322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Suramin potently inhibits cGAMP synthase, cGAS, in THP1 cells to modulate IFN-β levels</span></div><div class="casAuthors">Wang, Modi; Sooreshjani, Moloud A.; Mikek, Clinton; Opoku-Temeng, Clement; Sintim, Herman O.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1301-1317</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Aim: Persistent activation of STING pathway is the basis for several autoimmune diseases.  STING is activated by cGAMP, which is produced by cGAS in the presence of DNA.  Results/methodol.: HPLC-based medium throughput screening for inhibitors of cGAS identified suramin as a potent inhibitor.  Unlike other reported cGAS inhibitors, which bind to the ATP/GTP binding site, suramin displaced the bound DNA from cGAS.  Addn. of suramin to THP1 cells reduced the levels of IFN-β mRNA and protein.  Suramin did not inhibit lipopolysaccharide- or Pam3CSK4-induced IL-6 mRNA expression.  Conclusion: Suramin inhibits STING pathway via the inhibition of cGAS enzymic activity.  Suramin or analogs thereof that displace DNA from cGAS could be used as anti-inflammatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf9aUiYhp5vLVg90H21EOLACvtfcHk0liMAwt1aEVgoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVyqsrnJ&md5=d2ffc1db4e6041db7ac765bdb192f9be</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0322%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DSooreshjani%26aufirst%3DM.%2BA.%26aulast%3DMikek%26aufirst%3DC.%26aulast%3DOpoku-Temeng%26aufirst%3DC.%26aulast%3DSintim%26aufirst%3DH.%2BO.%26atitle%3DSuramin%2520potently%2520inhibits%2520cGAMP%2520synthase%252C%2520cGAS%252C%2520in%2520THP1%2520cells%2520to%2520modulate%2520IFN-%25CE%25B2%2520levels%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D1301%26epage%3D1317%26doi%3D10.4155%2Ffmc-2017-0322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adura, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lama, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aida, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothamel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogakos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reasoner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aso, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuschl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascano, M.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of cGAS reduces interferon expression in primary macrophages from autoimmune mice</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">750</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01770-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41467-017-00833-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=28963528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FjvVKjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=750&author=J.+Vincentauthor=C.+Aduraauthor=P.+Gaoauthor=A.+Luzauthor=L.+Lamaauthor=Y.+Asanoauthor=R.+Okamotoauthor=T.+Imaedaauthor=J.+Aidaauthor=K.+Rothamelauthor=T.+Gogakosauthor=J.+Steinbergauthor=S.+Reasonerauthor=K.+Asoauthor=T.+Tuschlauthor=D.+J.+Patelauthor=J.+F.+Glickmanauthor=M.+Ascano&title=Small+molecule+inhibition+of+cGAS+reduces+interferon+expression+in+primary+macrophages+from+autoimmune+mice&doi=10.1038%2Fs41467-017-01770-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of cGAS reduces interferon expression in primary macrophages from autoimmune mice</span></div><div class="casAuthors">Vincent Jessica; Rothamel Katherine; Reasoner Seth; Ascano Manuel; Adura Carolina; Luz Antonio; Lama Lodoe; Gogakos Tasos; Steinberg Joshua; Tuschl Thomas; Glickman J Fraser; Gao Pu; Patel Dinshaw J; Gao Pu; Luz Antonio; Lama Lodoe; Gogakos Tasos; Steinberg Joshua; Tuschl Thomas; Asano Yasutomi; Okamoto Rei; Imaeda Toshihiro; Aida Jumpei; Aso Kazuyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">750</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclic GMP-AMP synthase is essential for innate immunity against infection and cellular damage, serving as a sensor of DNA from pathogens or mislocalized self-DNA.  Upon binding double-stranded DNA, cyclic GMP-AMP synthase synthesizes a cyclic dinucleotide that initiates an inflammatory cellular response.  Mouse studies that recapitulate causative mutations in the autoimmune disease Aicardi-Goutieres syndrome demonstrate that ablating the cyclic GMP-AMP synthase gene abolishes the deleterious phenotype.  Here, we report the discovery of a class of cyclic GMP-AMP synthase inhibitors identified by a high-throughput screen.  These compounds possess defined structure-activity relationships and we present crystal structures of cyclic GMP-AMP synthase, double-stranded DNA, and inhibitors within the enzymatic active site.  We find that a chemically improved member, RU.521, is active and selective in cellular assays of cyclic GMP-AMP synthase-mediated signaling and reduces constitutive expression of interferon in macrophages from a mouse model of Aicardi-Goutieres syndrome.  RU.521 will be useful toward understanding the biological roles of cyclic GMP-AMP synthase and can serve as a molecular scaffold for development of future autoimmune therapies.Upon DNA binding cyclic GMP-AMP synthase (cGAS) produces a cyclic dinucleotide, which leads to the upregulation of inflammatory genes.  Here the authors develop small molecule cGAS inhibitors, functionally characterize them and present the inhibitor and DNA bound cGAS crystal structures, which will facilitate drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKzRlR5ipxKZagfuraLLzSfW6udTcc2ebHLKj5naTpz7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FjvVKjsw%253D%253D&md5=778f64f98841cea504e15155c5b83db2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00833-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00833-9%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DJ.%26aulast%3DAdura%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DLuz%26aufirst%3DA.%26aulast%3DLama%26aufirst%3DL.%26aulast%3DAsano%26aufirst%3DY.%26aulast%3DOkamoto%26aufirst%3DR.%26aulast%3DImaeda%26aufirst%3DT.%26aulast%3DAida%26aufirst%3DJ.%26aulast%3DRothamel%26aufirst%3DK.%26aulast%3DGogakos%26aufirst%3DT.%26aulast%3DSteinberg%26aufirst%3DJ.%26aulast%3DReasoner%26aufirst%3DS.%26aulast%3DAso%26aufirst%3DK.%26aulast%3DTuschl%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DGlickman%26aufirst%3DJ.%2BF.%26aulast%3DAscano%26aufirst%3DM.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520cGAS%2520reduces%2520interferon%2520expression%2520in%2520primary%2520macrophages%2520from%2520autoimmune%2520mice%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D750%26doi%3D10.1038%2Fs41467-017-01770-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascano, M.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of human cGAS reduces total cGAMP output and cytokine expression in cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">7604</span>, <span class="refDoi"> DOI: 10.1038/s41598-020-64348-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41598-020-64348-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32371942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovFGqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=7604&author=C.+Wiserauthor=B.+Kimauthor=J.+Vincentauthor=M.+Ascano&title=Small+molecule+inhibition+of+human+cGAS+reduces+total+cGAMP+output+and+cytokine+expression+in+cells&doi=10.1038%2Fs41598-020-64348-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of human cGAS reduces total cGAMP output and cytokine expression in cells</span></div><div class="casAuthors">Wiser, Caroline; Kim, Byungil; Vincent, Jessica; Ascano, Manuel</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7604</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The cGAS-STING pathway is a major mechanism that mammalian cells utilize to detect cytoplasmic dsDNA from incoming viruses, bacteria, or self.  CYCLIC GMP-AMP SYNTHASE (cGAS) is the sensor protein that directly binds dsDNAs. cGAS synthesizes cyclic GMP-AMP (cGAMP), which binds to the adaptor STIMULATOR OF INTERFERON GENES (STING), activating an INTERFERON REGULATORY FACTOR 3 (IRF3)-mediated immune response.  Constitutive activation can result in interferonopathies such as Aicardi-Goutieres Syndrome (AGS) or other lupus-like autoimmune disorders.  While inhibitors targeting mouse or human cGAS have been reported, the identification of a small mol. that targets both homologs of cGAS has been challenging.  Here, we show that RU.521 is capable of potently and selectively inhibiting mouse and human cGAS in cell lines and human primary cells.  This inhibitory activity requires the presence of cGAS, but it cannot suppress an immune response in cells activated by RNA, Toll-like receptor ligands, cGAMP, or recombinant interferon.  Importantly, when RU.521 is applied to cells, the prodn. of dsDNA-induced intracellular cGAMP is suppressed in a dose-dependent manner.  Our work validates the use of RU.521 for probing DNA-induced innate immune responses and underscores its potential as an ideal scaffold towards pre-clin. development, given its potency against human and mouse cGAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbVStZ5qmGtLVg90H21EOLACvtfcHk0lhbYyjRh7gMYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovFGqtrw%253D&md5=3e9827f13073229732f715fa9ada912b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41598-020-64348-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-020-64348-y%26sid%3Dliteratum%253Aachs%26aulast%3DWiser%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DVincent%26aufirst%3DJ.%26aulast%3DAscano%26aufirst%3DM.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520human%2520cGAS%2520reduces%2520total%2520cGAMP%2520output%2520and%2520cytokine%2520expression%2520in%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2020%26volume%3D10%26spage%3D7604%26doi%3D10.1038%2Fs41598-020-64348-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lama, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adura, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamei, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuryavyi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogakos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinberg, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos-Espiritu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashizume, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aida, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinke, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuschl, T.</span></span> <span> </span><span class="NLM_article-title">Development of human cGAS-specific small-molecule inhibitors for repression of dsDNA-triggered interferon expression</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2261</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-08620-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41467-019-08620-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=31113940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A280%3ADC%252BB3M7ot1Onsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=2261&author=L.+Lamaauthor=C.+Aduraauthor=W.+Xieauthor=D.+Tomitaauthor=T.+Kameiauthor=V.+Kuryavyiauthor=T.+Gogakosauthor=J.+I.+Steinbergauthor=M.+Millerauthor=L.+Ramos-Espirituauthor=Y.+Asanoauthor=S.+Hashizumeauthor=J.+Aidaauthor=T.+Imaedaauthor=R.+Okamotoauthor=A.+J.+Jenningsauthor=M.+Michinoauthor=T.+Kuroitaauthor=A.+Stamfordauthor=P.+Gaoauthor=P.+Meinkeauthor=J.+F.+Glickmanauthor=D.+J.+Patelauthor=T.+Tuschl&title=Development+of+human+cGAS-specific+small-molecule+inhibitors+for+repression+of+dsDNA-triggered+interferon+expression&doi=10.1038%2Fs41467-019-08620-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Development of human cGAS-specific small-molecule inhibitors for repression of dsDNA-triggered interferon expression</span></div><div class="casAuthors">Lama Lodoe; Gogakos Tasos; Steinberg Joshua I; Tuschl Thomas; Adura Carolina; Ramos-Espiritu Lavoisier; Glickman J Fraser; Xie Wei; Kuryavyi Vitaly; Gao Pu; Patel Dinshaw J; Tomita Daisuke; Kamei Taku; Miller Michael; Asano Yasutomi; Hashizume Shogo; Aida Jumpei; Imaeda Toshihiro; Okamoto Rei; Jennings Andy J; Michino Mayako; Kuroita Takanobu; Stamford Andrew; Meinke Peter; Gao Pu</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2261</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclic GMP-AMP synthase (cGAS) is the primary sensor for aberrant intracellular dsDNA producing the cyclic dinucleotide cGAMP, a second messenger initiating cytokine production in subsets of myeloid lineage cell types.  Therefore, inhibition of the enzyme cGAS may act anti-inflammatory.  Here we report the discovery of human-cGAS-specific small-molecule inhibitors by high-throughput screening and the targeted medicinal chemistry optimization for two molecular scaffolds.  Lead compounds from one scaffold co-crystallize with human cGAS and occupy the ATP- and GTP-binding active site.  The specificity and potency of these drug candidates is further documented in human myeloid cells including primary macrophages.  These novel cGAS inhibitors with cell-based activity will serve as probes into cGAS-dependent innate immune pathways and warrant future pharmacological studies for treatment of cGAS-dependent inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT436YtiSqWi9TiMd2aNQVUfW6udTcc2eZoI3OZErVKQLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M7ot1Onsw%253D%253D&md5=040007dd8aa93e051ee32fad05510622</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-08620-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-08620-4%26sid%3Dliteratum%253Aachs%26aulast%3DLama%26aufirst%3DL.%26aulast%3DAdura%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DKamei%26aufirst%3DT.%26aulast%3DKuryavyi%26aufirst%3DV.%26aulast%3DGogakos%26aufirst%3DT.%26aulast%3DSteinberg%26aufirst%3DJ.%2BI.%26aulast%3DMiller%26aufirst%3DM.%26aulast%3DRamos-Espiritu%26aufirst%3DL.%26aulast%3DAsano%26aufirst%3DY.%26aulast%3DHashizume%26aufirst%3DS.%26aulast%3DAida%26aufirst%3DJ.%26aulast%3DImaeda%26aufirst%3DT.%26aulast%3DOkamoto%26aufirst%3DR.%26aulast%3DJennings%26aufirst%3DA.%2BJ.%26aulast%3DMichino%26aufirst%3DM.%26aulast%3DKuroita%26aufirst%3DT.%26aulast%3DStamford%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DMeinke%26aufirst%3DP.%26aulast%3DGlickman%26aufirst%3DJ.%2BF.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DTuschl%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520human%2520cGAS-specific%2520small-molecule%2520inhibitors%2520for%2520repression%2520of%2520dsDNA-triggered%2520interferon%2520expression%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D2261%26doi%3D10.1038%2Fs41467-019-08620-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brault, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumlao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aivazian, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dower, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawrylik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golenbock, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovering, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niljanskul, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nistler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyminski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefker, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K.</span></span> <span> </span><span class="NLM_article-title">Discovery of PF-06928215 as a high affinity inhibitor of cGAS enabled by a novel fluorescence polarization assay</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0184843</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0184843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1371%2Fjournal.pone.0184843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=28934246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVyltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=J.+Hallauthor=A.+Braultauthor=F.+Vincentauthor=S.+Wengauthor=H.+Wangauthor=D.+Dumlaoauthor=A.+Aulabaughauthor=D.+Aivazianauthor=D.+Castroauthor=M.+Chenauthor=J.+Culpauthor=K.+Dowerauthor=J.+Gardnerauthor=S.+Hawrylikauthor=D.+Golenbockauthor=D.+Hepworthauthor=M.+Hornauthor=L.+Jonesauthor=P.+Jonesauthor=E.+Latzauthor=J.+Liauthor=L.-L.+Linauthor=W.+Linauthor=D.+Linauthor=F.+Loveringauthor=N.+Niljanskulauthor=R.+Nistlerauthor=B.+Pierceauthor=O.+Plotnikovaauthor=D.+Schmittauthor=S.+Shankerauthor=J.+Smithauthor=W.+Snyderauthor=T.+Subashiauthor=J.+Trujilloauthor=E.+Tyminskiauthor=G.+Wangauthor=J.+Wongauthor=B.+Lefkerauthor=L.+Dakinauthor=K.+Leach&title=Discovery+of+PF-06928215+as+a+high+affinity+inhibitor+of+cGAS+enabled+by+a+novel+fluorescence+polarization+assay&doi=10.1371%2Fjournal.pone.0184843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of PF-06928215 as a high affinity inhibitor of cGAS enabled by a novel fluorescence polarization assay</span></div><div class="casAuthors">Hall, Justin; Brault, Amy; Vincent, Fabien; Weng, Shawn; Wang, Hong; Dumlao, Darren; Aulabaugh, Ann; Aivazian, Dikran; Castro, Dana; Chen, Ming; Culp, Jeffrey; Dower, Ken; Gardner, Joseph; Hawrylik, Steven; Golenbock, Douglas; Hepworth, David; Horn, Mark; Jones, Lyn; Jones, Peter; Latz, Eicke; Li, Jing; Lin, Lih-Ling; Lin, Wen; Lin, David; Lovering, Frank; Niljanskul, Nootaree; Nistler, Ryan; Pierce, Betsy; Plotnikova, Olga; Schmitt, Daniel; Shanker, Suman; Smith, James; Snyder, William; Subashi, Timothy; Trujillo, John; Tyminski, Edyta; Wang, Guoxing; Wong, Jimson; Lefker, Bruce; Dakin, Leslie; Leach, Karen</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e0184843/1-e0184843/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Cyclic GMP-AMP synthase (cGAS) initiates the innate immune system in response to cytosolic dsDNA.  After binding and activation from dsDNA, cGAS uses ATP and GTP to synthesize 2', 3' -cGAMP (cGAMP), a cyclic dinucleotide second messenger with mixed 2'-5' and 3'-5' phosphodiester bonds.  Inappropriate stimulation of cGAS has been implicated in autoimmune disease such as systemic lupus erythematosus, thus inhibition of cGAS may be of therapeutic benefit in some diseases; however, the size and polarity of the cGAS active site makes it a challenging target for the development of conventional substrate-competitive inhibitors.  We report here the development of a high affinity (KD = 200 nM) inhibitor from a low affinity fragment hit with supporting biochem. and structural data showing these mols. bind to the cGAS active site.  We also report a new high throughput cGAS fluorescence polarization (FP)-based assay to enable the rapid identification and optimization of cGAS inhibitors.  This FP assay uses Cy5-labeled cGAMP in combination with a novel high affinity monoclonal antibody that specifically recognizes cGAMP with no cross reactivity to cAMP, cGMP, ATP, or GTP.  Given its role in the innate immune response, cGAS is a promising therapeutic target for autoinflammatory disease.  Our results demonstrate its druggability, provide a high affinity tool compd., and establish a high throughput assay for the identification of next generation cGAS inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvbHF0YrJ_lbVg90H21EOLACvtfcHk0lgid9tLbx_87g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVyltLc%253D&md5=3a528a9f63e573c8956583ac6977873a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0184843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0184843%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DJ.%26aulast%3DBrault%26aufirst%3DA.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DWeng%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DDumlao%26aufirst%3DD.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DAivazian%26aufirst%3DD.%26aulast%3DCastro%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DCulp%26aufirst%3DJ.%26aulast%3DDower%26aufirst%3DK.%26aulast%3DGardner%26aufirst%3DJ.%26aulast%3DHawrylik%26aufirst%3DS.%26aulast%3DGolenbock%26aufirst%3DD.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DLatz%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DL.-L.%26aulast%3DLin%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DNiljanskul%26aufirst%3DN.%26aulast%3DNistler%26aufirst%3DR.%26aulast%3DPierce%26aufirst%3DB.%26aulast%3DPlotnikova%26aufirst%3DO.%26aulast%3DSchmitt%26aufirst%3DD.%26aulast%3DShanker%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DW.%26aulast%3DSubashi%26aufirst%3DT.%26aulast%3DTrujillo%26aufirst%3DJ.%26aulast%3DTyminski%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DWong%26aufirst%3DJ.%26aulast%3DLefker%26aufirst%3DB.%26aulast%3DDakin%26aufirst%3DL.%26aulast%3DLeach%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520PF-06928215%2520as%2520a%2520high%2520affinity%2520inhibitor%2520of%2520cGAS%2520enabled%2520by%2520a%2520novel%2520fluorescence%2520polarization%2520assay%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0184843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">In Silico Screening-Based Discovery of Novel Inhibitors of Human Cyclic GMP-AMP Synthase: A Cross-Validation Study of Molecular Docking and Experimental Testing</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3265</span>– <span class="NLM_lpage">3276</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00171</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00171" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVWisr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=3265-3276&author=W.+Zhaoauthor=M.+Xiongauthor=X.+Yuanauthor=M.+Liauthor=H.+Sunauthor=Y.+Xu&title=In+Silico+Screening-Based+Discovery+of+Novel+Inhibitors+of+Human+Cyclic+GMP-AMP+Synthase%3A+A+Cross-Validation+Study+of+Molecular+Docking+and+Experimental+Testing&doi=10.1021%2Facs.jcim.0c00171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Screening-Based Discovery of Novel Inhibitors of Human Cyclic GMP-AMP Synthase: A Cross-Validation Study of Molecular Docking and Experimental Testing</span></div><div class="casAuthors">Zhao, Wenfeng; Xiong, Muya; Yuan, Xiaojing; Li, Minjun; Sun, Hongbin; Xu, Yechun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3265-3276</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic GMP-AMP synthase (cGAS) has been recently uncovered to be a promising therapeutic target for immune-assocd. diseases.  Until now, only a few inhibitors have been identified through high-throughput screening campaigns.  Here, we reported the discovery of novel inhibitors for the catalytic domain of human cGAS (h-cGASCD) by virtual screening for the first time.  To generate a reliable docking mode, we first obtained a high-resoln. crystal structure of h-cGASCD in complex with PF-06928215, a known inhibitor of h-cGAS, followed by mol. dynamics simulations on this complex structure.  Four fragment hits were identified by the virtual screening together with a thermal shift assay.  The crystal structures of these four compds. in complex with h-cGASCD were subsequently detd., and the binding modes of the compds. were similar to those predicted by mol. docking, supporting the reliability of the docking model.  In addn., an enzyme activity assay identified compd. 18 (IC50 = 29.88 ± 3.20μM) from the compds. predicted by the virtual screening.  A similarity search of compd. 18 followed by a second virtual screening led to the discovery of compds. S2 (IC50 = 13.1 ± 0.09μM) and S3 (IC50 = 4.9 ± 0.26μM) as h-cGAS inhibitors with improved potency.  Therefore, the present study not only provides the validated hit compds. for further development of h-cGAS inhibitors but also demonstrates a cross-validation study of virtual screening, in vitro exptl. assays, and crystal structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzDBOsuhjDALVg90H21EOLACvtfcHk0lgy2onXp1Tdgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVWisr3P&md5=8222d7ccfff534af59f489344d2fc441</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00171%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DXiong%26aufirst%3DM.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DIn%2520Silico%2520Screening-Based%2520Discovery%2520of%2520Novel%2520Inhibitors%2520of%2520Human%2520Cyclic%2520GMP-AMP%2520Synthase%253A%2520A%2520Cross-Validation%2520Study%2520of%2520Molecular%2520Docking%2520and%2520Experimental%2520Testing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D3265%26epage%3D3276%26doi%3D10.1021%2Facs.jcim.0c00171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padilla-Salinas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span> <span> </span><span class="NLM_article-title">Discovery of small-molecule cyclic GMP-AMP synthase inhibitors</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">1579</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.9b02666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.9b02666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlygt7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2020&pages=1579-1600&author=R.+Padilla-Salinasauthor=L.+Sunauthor=R.+Andersonauthor=X.+Yangauthor=S.+Zhangauthor=Z.+J.+Chenauthor=H.+Yin&title=Discovery+of+small-molecule+cyclic+GMP-AMP+synthase+inhibitors&doi=10.1021%2Facs.joc.9b02666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small-Molecule Cyclic GMP-AMP Synthase Inhibitors</span></div><div class="casAuthors">Padilla-Salinas, Rosaura; Sun, Lijun; Anderson, Rachel; Yang, Xikang; Zhang, Shuting; Chen, Zhijian J.; Yin, Hang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1579-1600</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic GMP-AMP (cGAS), a cytosolic DNA sensor, plays an important role in the type-I-interferon response.  DNA from either invading microbes or self-origin triggers the enzymic activity of cGAS.  Aberrant activation of cGAS is assocd. with various autoimmune disorders.  Only one selective probe exists for inhibiting cGAS in cells while others are limited by their poor cellular activity or specificity, which underscores the urgency for discovering new cGAS inhibitors.  Here, we describe the development of new small mol. human cGAS (hcGAS) inhibitors (80 compds. synthesized) with high binding affinity in vitro and cellular activity.  Our studies show CU-32 and CU-76, selectively inhibit the DNA pathway in human cells but had no effect on RIG-I-MAVS or Toll-like Receptor pathways.  CU-32 and CU-76 represent a new class of hcGAS inhibitors with activity in cells and provide a new chem. scaffold for designing probes to study cGAS function and development of autoimmune therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0wk7oOFIc47Vg90H21EOLACvtfcHk0lgy2onXp1Tdgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlygt7%252FP&md5=964f1954297ba2625a670dae7712d569</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.9b02666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.9b02666%26sid%3Dliteratum%253Aachs%26aulast%3DPadilla-Salinas%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DAnderson%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26aulast%3DYin%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520small-molecule%2520cyclic%2520GMP-AMP%2520synthase%2520inhibitors%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2020%26volume%3D85%26spage%3D1579%26epage%3D1600%26doi%3D10.1021%2Facs.joc.9b02666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huffman, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plata, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lairson, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houk, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenvi, R. A.</span></span> <span> </span><span class="NLM_article-title">Electronic complementarity permits hindered butenolide heterodimerization and discovery of novel cGAS/STING pathway antagonists</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41557-019-0413-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41557-019-0413-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32042135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivFKqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=310-317&author=B.+J.+Huffmanauthor=S.+Chenauthor=J.+L.+Schwarzauthor=R.+E.+Plataauthor=E.+N.+Chinauthor=L.+L.+Lairsonauthor=K.+N.+Houkauthor=R.+A.+Shenvi&title=Electronic+complementarity+permits+hindered+butenolide+heterodimerization+and+discovery+of+novel+cGAS%2FSTING+pathway+antagonists&doi=10.1038%2Fs41557-019-0413-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Electronic complementarity permits hindered butenolide heterodimerization and discovery of novel cGAS/STING pathway antagonists</span></div><div class="casAuthors">Huffman, Benjamin J.; Chen, Shuming; Schwarz, J. Luca; Plata, R. Erik; Chin, Emily N.; Lairson, Luke L.; Houk, K. N.; Shenvi, Ryan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">310-317</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Sp3-hybridized attached-rings are common motifs in secondary metabolites and represent tetrahedral equiv. to the biaryl substructures that overpopulate synthetic libraries.  Few methods are available that can link fully substituted carbon atoms of two rings with stereocontrol.  Here we have developed a stereoselective, heteroselective butenolide coupling that exhibits an unusually fast rate of C-C bond formation driven by exquisite complementarity of the reacting π systems.  Heterodimerization generates a compd. collection with topol. complexity and diverse principal moments of inertia.  The special status of the sp3-sp3 attached-ring motif is demonstrated in a high-throughput screen for inhibitors of the cyclic GMP-AMP synthase/stimulator of interferon genes pathway, which recruited these butenolide heterodimers from a field of 250,000 compds.  The driving forces underlying this general attached-ring coupling identify a novel paradigm for the accession of wider natural product chem. space, accelerating the discovery of selective lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUn5ppJV6fn7Vg90H21EOLACvtfcHk0ljJI7Pb_0TUTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivFKqu78%253D&md5=636d5531e45e89542e7fb349e040a709</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fs41557-019-0413-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-019-0413-8%26sid%3Dliteratum%253Aachs%26aulast%3DHuffman%26aufirst%3DB.%2BJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DSchwarz%26aufirst%3DJ.%2BL.%26aulast%3DPlata%26aufirst%3DR.%2BE.%26aulast%3DChin%26aufirst%3DE.%2BN.%26aulast%3DLairson%26aufirst%3DL.%2BL.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26aulast%3DShenvi%26aufirst%3DR.%2BA.%26atitle%3DElectronic%2520complementarity%2520permits%2520hindered%2520butenolide%2520heterodimerization%2520and%2520discovery%2520of%2520novel%2520cGAS%252FSTING%2520pathway%2520antagonists%26jtitle%3DNat.%2520Chem.%26date%3D2020%26volume%3D12%26spage%3D310%26epage%3D317%26doi%3D10.1038%2Fs41557-019-0413-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship study of amidobenzimidazole analogues leading to potent and systemically administrable stimulator of interferon gene (STING) agonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">1649</span>– <span class="NLM_lpage">1669</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01900</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01900" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVKjs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=1649-1669&author=Z.+Songauthor=X.+Wangauthor=Y.+Zhangauthor=W.+Guauthor=A.+Shenauthor=C.+Dingauthor=H.+Liauthor=R.+Xiaoauthor=M.+Gengauthor=Z.+Xieauthor=A.+Zhang&title=Structure-activity+relationship+study+of+amidobenzimidazole+analogues+leading+to+potent+and+systemically+administrable+stimulator+of+interferon+gene+%28STING%29+agonists&doi=10.1021%2Facs.jmedchem.0c01900"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship Study of Amidobenzimidazole Analogues Leading to Potent and Systemically Administrable Stimulator of Interferon Gene (STING) Agonists</span></div><div class="casAuthors">Song, Zilan; Wang, Xiyuan; Zhang, Yan; Gu, Wangting; Shen, Ancheng; Ding, Chunyong; Li, Han; Xiao, Ruoxuan; Geng, Meiyu; Xie, Zuoquan; Zhang, Ao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1649-1669</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activation of the stimulator of interferon gene (STING) has emerged as an exciting immuno-oncol. therapeutic strategy; however, the first-generation STING agonists, cyclic dinucleotide (CDN) analogs, have suffered from many disadvantages and failed in clin. trials.  Therefore, non-CDN small-mol. STING agonists are urgently needed.  In view of the unique structure of the high potency of dimeric amidobenzimidazole STING agonist 5, a structural elaboration was conducted by modifying several structural hotspots of this scaffold.  Triazole 40 was identified as a new potent STING activator, possessing EC50 values of 0.24 and 39.51 μM for h- and m-STING, resp.  This compd. has a slightly better pharmacokinetic profile and is >20-fold more aqueously sol. than 5.  It activated the STING signaling dramatically by directly binding and stabilizing all h-STING isoforms and m-STING.  In vivo, intermittent administration of 40 was found to have significant antitumor efficacy with good tolerance in two mouse tumor models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ogCDnt2s27Vg90H21EOLACvtfcHk0ljJI7Pb_0TUTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhsVKjs7o%253D&md5=bf2c2ac40a81a1424c26719c09348e46</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01900%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DDing%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DR.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DA.%26atitle%3DStructure-activity%2520relationship%2520study%2520of%2520amidobenzimidazole%2520analogues%2520leading%2520to%2520potent%2520and%2520systemically%2520administrable%2520stimulator%2520of%2520interferon%2520gene%2520%2528STING%2529%2520agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D1649%26epage%3D1669%26doi%3D10.1021%2Facs.jmedchem.0c01900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cisar, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clapper, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blankman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezekowitz, R. A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grice, C. A.</span></span> <span> </span><span class="NLM_article-title">Identification of ABX-1431, a selective inhibitor of monoacylglycerol lipase and clinical candidate for treatment of neurological disorders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9062</span>– <span class="NLM_lpage">9084</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00951</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00951" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVegt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9062-9084&author=J.+S.+Cisarauthor=O.+D.+Weberauthor=J.+R.+Clapperauthor=J.+L.+Blankmanauthor=C.+L.+Henryauthor=G.+M.+Simonauthor=J.+P.+Alexanderauthor=T.+K.+Jonesauthor=R.+A.+B.+Ezekowitzauthor=G.+P.+O%E2%80%99Neillauthor=C.+A.+Grice&title=Identification+of+ABX-1431%2C+a+selective+inhibitor+of+monoacylglycerol+lipase+and+clinical+candidate+for+treatment+of+neurological+disorders&doi=10.1021%2Facs.jmedchem.8b00951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of ABX-1431, a Selective Inhibitor of Monoacylglycerol Lipase and Clinical Candidate for Treatment of Neurological Disorders</span></div><div class="casAuthors">Cisar, Justin S.; Weber, Olivia D.; Clapper, Jason R.; Blankman, Jacqueline L.; Henry, Cassandra L.; Simon, Gabriel M.; Alexander, Jessica P.; Jones, Todd K.; Ezekowitz, R. Alan B.; O'Neill, Gary P.; Grice, Cheryl A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9062-9084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The serine hydrolase monoacylglycerol lipase (MGLL) converts the endogenous cannabinoid receptor agonist 2-arachidonoylglycerol (2-AG) and other monoacylglycerols into fatty acids and glycerol.  Genetic or pharmacol. inactivation of MGLL leads to elevation in 2-AG in the central nervous system and corresponding redns. in arachidonic acid and eicosanoids, producing antinociceptive, anxiolytic, and antineuroinflammatory effects without inducing the full spectrum of psychoactive effects of direct cannabinoid receptor agonists.  Here, we report the optimization of hexafluoroisopropyl carbamate-based irreversible inhibitors of MGLL, culminating in a highly potent, selective, and orally available, CNS-penetrant MGLL inhibitor, 28 (ABX-1431).  Activity-based protein profiling expts. verify the exquisite selectivity of 28 for MGLL vs. other members of the serine hydrolase class.  In vivo, 28 inhibits MGLL activity in rodent brain (ED50 = 0.5-1.4 mg/kg), increases brain 2-AG concns., and suppresses pain behavior in the rat formalin pain model.  ABX-1431 (28) is currently under evaluation in human clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiwfzPpgIMJ7Vg90H21EOLACvtfcHk0liuVzJLr9Mm1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVegt7fK&md5=30105d700be0454af72f9668421390a5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00951%26sid%3Dliteratum%253Aachs%26aulast%3DCisar%26aufirst%3DJ.%2BS.%26aulast%3DWeber%26aufirst%3DO.%2BD.%26aulast%3DClapper%26aufirst%3DJ.%2BR.%26aulast%3DBlankman%26aufirst%3DJ.%2BL.%26aulast%3DHenry%26aufirst%3DC.%2BL.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DAlexander%26aufirst%3DJ.%2BP.%26aulast%3DJones%26aufirst%3DT.%2BK.%26aulast%3DEzekowitz%26aufirst%3DR.%2BA.%2BB.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DG.%2BP.%26aulast%3DGrice%26aufirst%3DC.%2BA.%26atitle%3DIdentification%2520of%2520ABX-1431%252C%2520a%2520selective%2520inhibitor%2520of%2520monoacylglycerol%2520lipase%2520and%2520clinical%2520candidate%2520for%2520treatment%2520of%2520neurological%2520disorders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9062%26epage%3D9084%26doi%3D10.1021%2Facs.jmedchem.8b00951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">Construction of bisindolines via oxidative coupling cyclization</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b04037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b04037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlGitrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2020&pages=116-119&author=R.+Cuiauthor=J.+Yeauthor=J.+Liauthor=W.+Moauthor=Y.+Gaoauthor=H.+Chen&title=Construction+of+bisindolines+via+oxidative+coupling+cyclization&doi=10.1021%2Facs.orglett.9b04037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Construction of Bisindolines via Oxidative Coupling Cyclization</span></div><div class="casAuthors">Cui, Ranran; Ye, Jinxiang; Li, Jing; Mo, Wenhui; Gao, Yu; Chen, Haijun</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">116-119</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein, a general approach for the efficient construction of three-dimensional bisindolines via oxidative coupling cyclization in an intermol. manner is reported.  This reaction is featured by its operational simplicity, metal-free conditions, lack of protecting group, and high selectivity.  Notably, a wide range of anilines are suitable in this intermol. cyclization, furnishing corresponding bisindolines in up to 98% yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXACZwo7fzx7Vg90H21EOLACvtfcHk0liuVzJLr9Mm1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlGitrnL&md5=41f7a7c7af356d09e986fb6004acce0a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b04037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b04037%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DR.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMo%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DConstruction%2520of%2520bisindolines%2520via%2520oxidative%2520coupling%2520cyclization%26jtitle%3DOrg.%2520Lett.%26date%3D2020%26volume%3D22%26spage%3D116%26epage%3D119%26doi%3D10.1021%2Facs.orglett.9b04037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundb%C3%A4ckauthor=P.+Nordlundauthor=D.+M.+Molina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0liuVzJLr9Mm1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0ljjHhsE69BUfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ning, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenyang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, G.</span></span> <span> </span><span class="NLM_article-title">Cytosolic DNA-STING-NLRP3 axis is involved in murine acute lung injury induced by lipopolysaccharide</span>. <i>Clin. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e228</span> <span class="refDoi"> DOI: 10.1002/ctm2.228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1002%2Fctm2.228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=33252860" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&author=L.+Ningauthor=W.+Weiauthor=J.+Wenyangauthor=X.+Ruiauthor=G.+Qing&title=Cytosolic+DNA-STING-NLRP3+axis+is+involved+in+murine+acute+lung+injury+induced+by+lipopolysaccharide&doi=10.1002%2Fctm2.228"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fctm2.228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fctm2.228%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DWenyang%26aufirst%3DJ.%26aulast%3DRui%26aufirst%3DX.%26aulast%3DQing%26aufirst%3DG.%26atitle%3DCytosolic%2520DNA-STING-NLRP3%2520axis%2520is%2520involved%2520in%2520murine%2520acute%2520lung%2520injury%2520induced%2520by%2520lipopolysaccharide%26jtitle%3DClin.%2520Transl.%2520Med.%26date%3D2020%26volume%3D10%26doi%3D10.1002%2Fctm2.228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljjHhsE69BUfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhfaeSn6ieuVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0024.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative cGAS inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Co-crystal of h-cGAS with compound <b>6</b> and our drug design (approaches 1 and 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>14–21</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) <i>N</i>-Boc-amino acids or carboxylic acids, EDCI/HOBt or HATU, DIPEA, and DMF, rt, 45–66%; (ii) triphosgene, hexafluoroisopropanol, DIPEA, and DCM, rt, 39%; (iii) 1-methyl-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyrazole, Pd(dppf)Cl<sub>2</sub>, KOAc, and dioxane, 100 °C, 31–84%; and (iv) CF<sub>3</sub>COOH and DCM, rt, 42–63%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>22</b> and <b>24–37</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) HCHO, NaBH<sub>3</sub>CN, HOAc, and MeOH, 0 °C, 60%; (ii) <i>N</i>-Boc amino acids or carboxylic acids, EDCI/HOBt or HATU, DIPEA, and DMF, rt, 23–63%; and (iii) CF<sub>3</sub>COOH and DCM, rt, 42–63%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>43–55</b> and <b>57</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) piperidin-4-one hydrochloride, H<sub>2</sub>SO<sub>4</sub>, dioxane, 115 °C, 64%; (ii) acetoxyacetic acid, EDCI, HOBt, DIPEA, DMF, rt, 67%; (iii) <i>t</i>-BuOCl, TMEDA, DCM, rt; (iv) substituted anilines, H<sub>2</sub>SO<sub>4</sub>, rt, 53–73%; (v) LiOH·H<sub>2</sub>O, THF, H<sub>2</sub>O, rt, 30–75%; (vi) LiOH·H<sub>2</sub>O, THF, H<sub>2</sub>O, rt; then 1 N HCl, rt; 24–64%; (vii) 2,5-dimethoxyaniline, H<sub>2</sub>SO<sub>4</sub>, rt, 83%.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>59–66</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) <i>t</i>-BuOCl, TMEDA, and DCM, rt; (ii) substituted indoles and H<sub>2</sub>SO<sub>4</sub>, rt, 58–80%; (iii) LiOH·H<sub>2</sub>O, THF, and H<sub>2</sub>O, rt, 34–67%; and (iv) NaBH<sub>3</sub>CN, HOAc, and MeOH, 0 °C, 69%.</p></p></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Fold change in the IRF Lucia expression of representative compounds in the cGAS signaling pathway of THP1-Dual cells by cGAMP stimulation. THP1-Dual cells were pre-incubated at 10 μM concentration of compounds for 1 h, then cells were transfected with cGAMP (cGAMP only initiates cGAS downstream STING-dependent signaling activation) for 24 h. Luciferase luminescence of the cell supernatant was measured. The fold change of luciferase activity for each compound-treated sample was calculated, namely, the fold change of Lucia expression = RLU sample/RLU positive control, where RLU indicates a raw luciferase unit. Data are given as mean ± SEM. Statistical analysis is performed by one-way analysis of variance (ANOVA). *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to cGAMP. <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, and <sup>###</sup><i>P</i> < 0.01 compared to blank. ns, no significance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compounds <b>25</b> and <b>66</b> inhibited the activation of the cGAS downstream signaling pathway. THP1-Dual cells were pre-treated with or without different concentrations of compounds for 1 h. Then, cells were transfected with dsDNA for 2 h. The expression levels of p-TBK1, TBK1, p-IRF3, IRF3, p-STING, and STING were determined by western blot analysis. The band intensity was quantified by ImageJ. Relative protein expression was displayed by the ratio of phosphorylated protein to total protein. Data are given as mean ± SEM. Statistical analysis is performed by one-way ANOVA. *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to dsDNA. <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, and <sup>###</sup><i>P</i> < 0.01 compared to blank. ns, no significance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. CESTA of compound <b>25</b> in THP1-Dual cells. The cells were incubated in the presence or absence of compound <b>25</b> (25 μM) for 1 h, then were heated at the indicated temperature (ranging from 37 to 63.1 °C) and cooled at room temperature. The cells were freeze–thawed and the cell lysate was centrifuged. The expression of cGAS and GAPDH in the supernatant was detected by the western blot assay, which implied the thermostability of cGAS following heat treatment. Also, the relative band intensity (%) was calculated, namely, relative band intensity (%) = band intensity of cGAS protein/band intensity of the GAPDH protein in every group at different temperatures, where the band intensity was quantified by ImageJ. Data are given as mean ± SEM. Statistical analysis is performed by Student’s <i>t</i>-test. *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to the DMSO group. ns, no significance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Effect of <b>25</b> on dsDNA-induced luciferase activity, <i>Il-6</i> and <i>Ifn-β</i> gene expression in THP1-Dual cells. Data are given as mean ± SEM. Statistical analysis is performed by one-way ANOVA. *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to dsDNA. <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, and <sup>###</sup><i>P</i> < 0.01 compared to blank. ns, no significance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Aqueous solubility of compound <b>5</b> and the HCl salts of compounds <b>20</b> and <b>25</b>. The values are mean ± SEM of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inflammatory cytokine concentrations in the serum of the LPS-induced mice model. BALB/c mice were intraperitoneally injected with compounds (30 mg/kg) and vehicle (10% DMSO + 90% corn oil) 1 h before an intraperitoneal injection with a dose of 5 mg/kg of LPS. The serum was collected 1.5 h after LPS application. The concentration of cytokines in the serum was detected by ELISA. Data are given as mean ± SEM. Statistical analysis is performed by one-way ANOVA. *<i>P</i> < 0.05, **<i>P</i> < 0.01, and ***<i>P</i> < 0.001 compared to vehicle. <sup>#</sup><i>P</i> < 0.05, <sup>##</sup><i>P</i> < 0.01, and <sup>###</sup><i>P</i> < 0.01 compared to normal. ns, no significance. Normal, vehicle, and compound <b>3</b> groups: <i>n</i> = 6 mice per group, compounds <b>5</b> and <b>25</b> groups: <i>n</i> = 7 mice per group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/medium/jm1c00398_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Docking studies for compounds <b>25</b> and <b>66</b> within the ATP and GTP binding pockets of apo h-cGAS<sup>CD</sup> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJW">6MJW</a>). (A) Chemical structures of compounds <b>25</b> and <b>66</b>. (B) Superposition of the structures of compound <b>25</b> colored in cyan and of <b>66</b> in deep pink. (C) Docking mode of compound <b>25</b> with key intermolecular contacts. (D) Docking mode of compound <b>66</b> with key interactions.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00398/20210602/images/large/jm1c00398_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00398&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 39 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brubaker, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonham, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanoni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagan, J. C.</span></span> <span> </span><span class="NLM_article-title">Innate immune pattern recognition: a cell biological perspective</span>. <i>Annu. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1146/annurev-immunol-032414-112240</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1146%2Fannurev-immunol-032414-112240" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=25581309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVSisL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=257-290&author=S.+W.+Brubakerauthor=K.+S.+Bonhamauthor=I.+Zanoniauthor=J.+C.+Kagan&title=Innate+immune+pattern+recognition%3A+a+cell+biological+perspective&doi=10.1146%2Fannurev-immunol-032414-112240"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Innate Immune Pattern Recognition: A Cell Biological Perspective</span></div><div class="casAuthors">Brubaker, Sky W.; Bonham, Kevin S.; Zanoni, Ivan; Kagan, Jonathan C.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">257-290</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Receptors of the innate immune system detect conserved determinants of microbial and viral origin.  Activation of these receptors initiates signaling events that culminate in an effective immune response.  Recently, the view that innate immune signaling events rely on and operate within a complex cellular infrastructure has become an important framework for understanding the regulation of innate immunity.  Compartmentalization within this infrastructure provides the cell with the ability to assign spatial information to microbial detection and regulate immune responses.  Several cell biol. processes play a role in the regulation of innate signaling responses; at the same time, innate signaling can engage cellular processes as a form of defense or to promote immunol. memory.  In this review, we highlight these aspects of cell biol. in pattern-recognition receptor signaling by focusing on signals that originate from the cell surface, from endosomal compartments, and from within the cytosol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6Q0f75-M4yLVg90H21EOLACvtfcHk0lioHh2e6hDf6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVSisL3O&md5=ad51df752f62a080d97890221dfd2a68</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev-immunol-032414-112240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-immunol-032414-112240%26sid%3Dliteratum%253Aachs%26aulast%3DBrubaker%26aufirst%3DS.%2BW.%26aulast%3DBonham%26aufirst%3DK.%2BS.%26aulast%3DZanoni%26aufirst%3DI.%26aulast%3DKagan%26aufirst%3DJ.%2BC.%26atitle%3DInnate%2520immune%2520pattern%2520recognition%253A%2520a%2520cell%2520biological%2520perspective%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2015%26volume%3D33%26spage%3D257%26epage%3D290%26doi%3D10.1146%2Fannurev-immunol-032414-112240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornung, V.</span></span> <span> </span><span class="NLM_article-title">Recognition of endogenous nucleic acids by the innate immune system</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2016.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.immuni.2016.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=27096317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVehs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=739-754&author=A.+Roersauthor=B.+Hillerauthor=V.+Hornung&title=Recognition+of+endogenous+nucleic+acids+by+the+innate+immune+system&doi=10.1016%2Fj.immuni.2016.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of Endogenous Nucleic Acids by the Innate Immune System</span></div><div class="casAuthors">Roers, Axel; Hiller, Bjoern; Hornung, Veit</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">739-754</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Recognition of DNA and RNA by endosomal and cytosolic sensors constitutes a central element in the detection of microbial invaders by the innate immune system.  However, the capacity of these sensors to discriminate between microbial and endogenous nucleic acids is limited.  Over the past few years, evidence has accumulated to suggest that endogenous DNA or RNA species can engage nucleic-acid-sensing pattern-recognition receptors that can trigger or sustain detrimental pathol.  Here, we review principles of how the activation of innate sensors by host nucleic acids is prevented in the steady state and discuss four important determinants of whether a nucleic-acid-driven innate response is mounted.  These include structural features of the ligand being sensed, the subcellular location and quantity of pathogen-derived or endogenous nucleic acids, and the regulation of sensor-activation thresholds.  Furthermore, we emphasize disease mechanisms initiated by failure to discriminate self from non-self in nucleic acid detection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTfTR4bXug1rVg90H21EOLACvtfcHk0lizoYhu85vKyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVehs7o%253D&md5=624aab1b9e40165898e710b62d786fe3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2016.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2016.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DRoers%26aufirst%3DA.%26aulast%3DHiller%26aufirst%3DB.%26aulast%3DHornung%26aufirst%3DV.%26atitle%3DRecognition%2520of%2520endogenous%2520nucleic%2520acids%2520by%2520the%2520innate%2520immune%2520system%26jtitle%3DImmunity%26date%3D2016%26volume%3D44%26spage%3D739%26epage%3D754%26doi%3D10.1016%2Fj.immuni.2016.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">Cyclic GMP-AMP Is an Endogenous Second Messenger in Innate Immune Signaling by Cytosolic DNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>339</i></span>,  <span class="NLM_fpage">826</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1126/science.1229963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1126%2Fscience.1229963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=23258412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1ygtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=826-830&author=J.+Wuauthor=L.+Sunauthor=X.+Chenauthor=F.+Duauthor=H.+Shiauthor=C.+Chenauthor=Z.+J.+Chen&title=Cyclic+GMP-AMP+Is+an+Endogenous+Second+Messenger+in+Innate+Immune+Signaling+by+Cytosolic+DNA&doi=10.1126%2Fscience.1229963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic GMP-AMP Is an Endogenous Second Messenger in Innate Immune Signaling by Cytosolic DNA</span></div><div class="casAuthors">Wu, Jiaxi; Sun, Lijun; Chen, Xiang; Du, Fenghe; Shi, Heping; Chen, Chuo; Chen, Zhijian J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">339</span>
        (<span class="NLM_cas:issue">6121</span>),
    <span class="NLM_cas:pages">826-830</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Cytosolic DNA induces type I interferons and other cytokines that are important for antimicrobial defense but can also result in autoimmunity.  This DNA signaling pathway requires the adaptor protein STING and the transcription factor IRF3, but the mechanism of DNA sensing is unclear.  We found that mammalian cytosolic exts. synthesized cyclic guanosine monophosphate-adenosine monophosphate (cyclic GMP-AMP, or cGAMP) in vitro from ATP and guanosine triphosphate in the presence of DNA but not RNA.  DNA transfection or DNA virus infection of mammalian cells also triggered cGAMP prodn.  CGAMP bound to STING, leading to the activation of IRF3 and induction of interferon-β.  Thus, cGAMP functions as an endogenous second messenger in metazoans and triggers interferon prodn. in response to cytosolic DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0CH1aDO6v5bVg90H21EOLACvtfcHk0lizoYhu85vKyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1ygtbs%253D&md5=2d59a6c9c71eb7188e3e9fd01b2c7e84</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1229963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1229963%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DCyclic%2520GMP-AMP%2520Is%2520an%2520Endogenous%2520Second%2520Messenger%2520in%2520Innate%2520Immune%2520Signaling%2520by%2520Cytosolic%2520DNA%26jtitle%3DScience%26date%3D2013%26volume%3D339%26spage%3D826%26epage%3D830%26doi%3D10.1126%2Fscience.1229963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">The cytosolic DNA sensor cGAS forms an oligomeric complex with DNA and undergoes switch-like conformational changes in the activation loop</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">421</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.celrep.2014.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=24462292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Gms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=421-430&author=X.+Zhangauthor=J.+Wuauthor=F.+Duauthor=H.+Xuauthor=L.+Sunauthor=Z.+Chenauthor=C.+A.+Brautigamauthor=X.+Zhangauthor=Z.+J.+Chen&title=The+cytosolic+DNA+sensor+cGAS+forms+an+oligomeric+complex+with+DNA+and+undergoes+switch-like+conformational+changes+in+the+activation+loop&doi=10.1016%2Fj.celrep.2014.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The Cytosolic DNA Sensor cGAS Forms an Oligomeric Complex with DNA and Undergoes Switch-like Conformational Changes in the Activation Loop</span></div><div class="casAuthors">Zhang, Xu; Wu, Jiaxi; Du, Fenghe; Xu, Hui; Sun, Lijun; Chen, Zhe; Brautigam, Chad A.; Zhang, Xuewu; Chen, Zhijian J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">421-430</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The presence of DNA in the cytoplasm is a danger signal that triggers immune and inflammatory responses.  Cytosolic DNA binds to and activates cyclic GMP-AMP (cGAMP) synthase (cGAS), which produces the second messenger cGAMP. cGAMP binds to the adaptor protein STING and activates a signaling cascade that leads to the prodn. of type I interferons and other cytokines.  Here, we report the crystal structures of human cGAS in its apo form, representing its autoinhibited conformation as well as in its cGAMP- and sulfate-bound forms.  These structures reveal switch-like conformational changes of an activation loop that result in the rearrangement of the catalytic site.  The structure of DNA-bound cGAS reveals a complex composed of dimeric cGAS bound to two mols. of DNA.  Functional analyses of cGAS mutants demonstrate that both the protein-protein interface and the two DNA binding surfaces are crit. for cGAS activation.  These results provide insights into the mechanism of DNA sensing by cGAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6puAKW2zx4rVg90H21EOLACvtfcHk0liBDYmYyJY6rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Gms78%253D&md5=6b59678311798f13d749468246444dc3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DThe%2520cytosolic%2520DNA%2520sensor%2520cGAS%2520forms%2520an%2520oligomeric%2520complex%2520with%2520DNA%2520and%2520undergoes%2520switch-like%2520conformational%2520changes%2520in%2520the%2520activation%2520loop%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D6%26spage%3D421%26epage%3D430%26doi%3D10.1016%2Fj.celrep.2014.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barber, G. N.</span></span> <span> </span><span class="NLM_article-title">STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>455</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1038/nature07317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnature07317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=18724357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtF2hsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=455&publication_year=2008&pages=674-678&author=H.+Ishikawaauthor=G.+N.+Barber&title=STING+is+an+endoplasmic+reticulum+adaptor+that+facilitates+innate+immune+signalling&doi=10.1038%2Fnature07317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">STING is an endoplasmic reticulum adaptor that facilitates innate immune signalling</span></div><div class="casAuthors">Ishikawa, Hiroki; Barber, Glen N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">455</span>
        (<span class="NLM_cas:issue">7213</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The cellular innate immune system is essential for recognizing pathogen infection and for establishing effective host defense.  But crit. mol. determinants responsible for facilitating an appropriate immune response-following infection with DNA and RNA viruses, for example-remain to be identified.  Here the authors report the identification, following expression cloning, of a mol. (STING; stimulator of interferon genes) that appears essential for effective innate immune signaling processes.  It comprises five putative transmembrane regions, predominantly resides in the endoplasmic reticulum and is able to activate both NF-κB and IRF3 transcription pathways to induce expression of type I interferon (IFN-α and IFN-β) and exert a potent anti-viral state following expression.  In contrast, loss of STING rendered murine embryonic fibroblasts extremely susceptible to neg.-stranded virus infection, including vesicular stomatitis virus.  Further, STING ablation abrogated the ability of intracellular B-form DNA, as well as members of the herpesvirus family, to induce IFN-β, but did not significantly affect the Toll-like receptor (TLR) pathway.  Yeast two-hybrid and co-immunopptn. studies indicated that STING interacts with RIG-I and with SSR2 (also known as TRAPβ), which is a member of the translocon-assocd. protein (TRAP) complex required for protein translocation across the endoplasmic reticulum membrane following translation.  Ablation by RNA interference of both TRAPβ and translocon adaptor SEC61β was subsequently found to inhibit STING's ability to stimulate expression of IFN-β.  Thus, as well as identifying a regulator of innate immune signaling, these results imply a potential role for the translocon in innate signaling pathways activated by select viruses as well as intracellular DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM8AkjsFJ2qbVg90H21EOLACvtfcHk0liBDYmYyJY6rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtF2hsbzL&md5=67bd86a114023c8d6c68489ab7e33009</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature07317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07317%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DH.%26aulast%3DBarber%26aufirst%3DG.%2BN.%26atitle%3DSTING%2520is%2520an%2520endoplasmic%2520reticulum%2520adaptor%2520that%2520facilitates%2520innate%2520immune%2520signalling%26jtitle%3DNature%26date%3D2008%26volume%3D455%26spage%3D674%26epage%3D678%26doi%3D10.1038%2Fnature07317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barber, G. N.</span></span> <span> </span><span class="NLM_article-title">STING regulates intracellular DNA-mediated, type I interferon-dependent innate immunity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span>,  <span class="NLM_fpage">788</span>– <span class="NLM_lpage">792</span>, <span class="refDoi"> DOI: 10.1038/nature08476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnature08476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=19776740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFGqtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=788-792&author=H.+Ishikawaauthor=Z.+Maauthor=G.+N.+Barber&title=STING+regulates+intracellular+DNA-mediated%2C+type+I+interferon-dependent+innate+immunity&doi=10.1038%2Fnature08476"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">STING regulates intracellular DNA-mediated, type I interferon-dependent innate immunity</span></div><div class="casAuthors">Ishikawa, Hiroki; Ma, Zhe; Barber, Glen N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7265</span>),
    <span class="NLM_cas:pages">788-792</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The innate immune system is crit. for the early detection of invading pathogens and for initiating cellular host defense countermeasures, which include the prodn. of type I interferon (IFN).  However, little is known about how the innate immune system is galvanized to respond to DNA-based microbes.  Here the authors show that STING (stimulator of interferon genes) is crit. for the induction of IFN by non-CpG intracellular DNA species produced by various DNA pathogens after infection.  Murine embryonic fibroblasts, as well as antigen presenting cells such as macrophages and dendritic cells (exposed to intracellular B-form DNA, the DNA virus herpes simplex virus 1 (HSV-1) or bacteria Listeria monocytogenes), were found to require STING to initiate effective IFN prodn.  Accordingly, Sting-knockout mice were susceptible to lethal infection after exposure to HSV-1.  The importance of STING in facilitating DNA-mediated innate immune responses was further evident because cytotoxic T-cell responses induced by plasmid DNA vaccination were reduced in Sting-deficient animals.  In the presence of intracellular DNA, STING relocalized with TANK-binding kinase 1 (TBK1) from the endoplasmic reticulum to perinuclear vesicles contg. the exocyst component Sec5 (also known as EXOC2).  Collectively, the authors' studies indicate that STING is essential for host defense against DNA pathogens such as HSV-1 and facilitates the adjuvant activity of DNA-based vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozocwPRxdjlrVg90H21EOLACvtfcHk0lhFfOVOrvkHQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGqtbvI&md5=441c2a523b68fa98dfe5202364ec928b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature08476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08476%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DBarber%26aufirst%3DG.%2BN.%26atitle%3DSTING%2520regulates%2520intracellular%2520DNA-mediated%252C%2520type%2520I%2520interferon-dependent%2520innate%2520immunity%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D788%26epage%3D792%26doi%3D10.1038%2Fnature08476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ablasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldeck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deimling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witte, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röhl, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfner, K.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornung, V.</span></span> <span> </span><span class="NLM_article-title">cGAS produces a 2′-5′-linked cyclic dinucleotide second messenger that activates STING</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>498</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1038/nature12306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnature12306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=23722158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpslGhsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=498&publication_year=2013&pages=380-384&author=A.+Ablasserauthor=M.+Goldeckauthor=T.+Cavlarauthor=T.+Deimlingauthor=G.+Witteauthor=I.+R%C3%B6hlauthor=K.-P.+Hopfnerauthor=J.+Ludwigauthor=V.+Hornung&title=cGAS+produces+a+2%E2%80%B2-5%E2%80%B2-linked+cyclic+dinucleotide+second+messenger+that+activates+STING&doi=10.1038%2Fnature12306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">cGAS produces a 2'-5'-linked cyclic dinucleotide second messenger that activates STING</span></div><div class="casAuthors">Ablasser, Andrea; Goldeck, Marion; Cavlar, Taner; Deimling, Tobias; Witte, Gregor; Roehl, Ingo; Hopfner, Karl-Peter; Ludwig, Janos; Hornung, Veit</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">498</span>
        (<span class="NLM_cas:issue">7454</span>),
    <span class="NLM_cas:pages">380-384</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Detection of cytoplasmic DNA represents one of the most fundamental mechanisms of the innate immune system to sense the presence of microbial pathogens.  Moreover, erroneous detection of endogenous DNA by the same sensing mechanisms has an important pathophysiol. role in certain sterile inflammatory conditions.  The endoplasmic reticulum-resident protein STING is critically required for the initiation of type I interferon signalling upon detection of cytosolic DNA of both exogenous and endogenous origin.  Next to its pivotal role in DNA sensing, STING also serves as a direct receptor for the detection of cyclic dinucleotides, which function as second messenger mols. in bacteria.  DNA recognition, however, is triggered in an indirect fashion that depends on a recently characterized cytoplasmic nucleotidyl transferase, termed cGAMP synthase (cGAS), which upon interaction with DNA synthesizes a dinucleotide mol. that in turn binds to and activates STING.  We here show in vivo and in vitro that the cGAS-catalyzed reaction product is distinct from previously characterized cyclic dinucleotides.  Using a combinatorial approach based on mass spectrometry, enzymic digestion, NMR anal. and chem. synthesis, we demonstrate that cGAS produces a cyclic GMP-AMP dinucleotide, which comprises a 2'-5' and a 3'-5' phosphodiester linkage >Gp(2'-5')Ap(3'-5')>.  We found that the presence of this 2'-5' linkage was required to exert potent activation of human STING.  Moreover, we show that cGAS first catalyzes the synthesis of a linear 2'-5'-linked dinucleotide, which is then subject to cGAS-dependent cyclization in a second step through a 3'-5' phosphodiester linkage.  This 13-membered ring structure defines a novel class of second messenger mols., extending the family of 2'-5'-linked antiviral biomols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfAZc114JC5bVg90H21EOLACvtfcHk0lhFfOVOrvkHQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpslGhsrk%253D&md5=78743675ef6c36bb32d7eb80a2030c46</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature12306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12306%26sid%3Dliteratum%253Aachs%26aulast%3DAblasser%26aufirst%3DA.%26aulast%3DGoldeck%26aufirst%3DM.%26aulast%3DCavlar%26aufirst%3DT.%26aulast%3DDeimling%26aufirst%3DT.%26aulast%3DWitte%26aufirst%3DG.%26aulast%3DR%25C3%25B6hl%26aufirst%3DI.%26aulast%3DHopfner%26aufirst%3DK.-P.%26aulast%3DLudwig%26aufirst%3DJ.%26aulast%3DHornung%26aufirst%3DV.%26atitle%3DcGAS%2520produces%2520a%25202%25E2%2580%25B2-5%25E2%2580%25B2-linked%2520cyclic%2520dinucleotide%2520second%2520messenger%2520that%2520activates%2520STING%26jtitle%3DNature%26date%3D2013%26volume%3D498%26spage%3D380%26epage%3D384%26doi%3D10.1038%2Fnature12306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, H.-B.</span></span> <span> </span><span class="NLM_article-title">The adaptor protein MITA links virus-sensing receptors to IRF3 transcription factor activation</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2008.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.immuni.2008.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=18818105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlWntb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=538-550&author=B.+Zhongauthor=Y.+Yangauthor=S.+Liauthor=Y.-Y.+Wangauthor=Y.+Liauthor=F.+Diaoauthor=C.+Leiauthor=X.+Heauthor=L.+Zhangauthor=P.+Tienauthor=H.-B.+Shu&title=The+adaptor+protein+MITA+links+virus-sensing+receptors+to+IRF3+transcription+factor+activation&doi=10.1016%2Fj.immuni.2008.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The adaptor protein MITA links virus-sensing receptors to IRF3 transcription factor activation</span></div><div class="casAuthors">Zhong, Bo; Yang, Yan; Li, Shu; Wang, Yan-Yi; Li, Ying; Diao, Feici; Lei, Caoqi; He, Xiao; Zhang, Lu; Tien, Po; Shu, Hong-Bing</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">538-550</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Viral infection triggers activation of transcription factors such as NF-κB and IRF3, which collaborate to induce type I interferons (IFNs) and elicit innate antiviral response.  Here, the authors identified MITA as a crit. mediator of virus-triggered type I IFN signaling by expression cloning.  Overexpression of MITA activated IRF3, whereas knockdown of MITA inhibited virus-triggered activation of IRF3, expression of type I IFNs, and cellular antiviral response.  MITA was found to localize to the outer membrane of mitochondria and to be assocd. with VISA, a mitochondrial protein that acts as an adaptor in virus-triggered signaling.  MITA also interacted with IRF3 and recruited the kinase TBK1 to the VISA-assocd. complex.  MITA was phosphorylated by TBK1, which is required for MITA-mediated activation of IRF3.  The authors' results suggest that MITA is a crit. mediator of virus-triggered IRF3 activation and IFN expression and further demonstrate the importance of certain mitochondrial proteins in innate antiviral immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI7inp8jAxA7Vg90H21EOLACvtfcHk0lhnjDSNnKa6jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlWntb7P&md5=b4197f541f33cc09897cd5516d05eab3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2008.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2008.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.-Y.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDiao%26aufirst%3DF.%26aulast%3DLei%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DTien%26aufirst%3DP.%26aulast%3DShu%26aufirst%3DH.-B.%26atitle%3DThe%2520adaptor%2520protein%2520MITA%2520links%2520virus-sensing%2520receptors%2520to%2520IRF3%2520transcription%2520factor%2520activation%26jtitle%3DImmunity%26date%3D2008%26volume%3D29%26spage%3D538%26epage%3D550%26doi%3D10.1016%2Fj.immuni.2008.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balka, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calleja, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Silva, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moghaddas, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tailler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawlor, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicks, I. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miner, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kile, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Nardo, D.</span></span> <span> </span><span class="NLM_article-title">TBK1 and IKKε Act Redundantly to Mediate STING-Induced NF-κB Responses in Myeloid Cells</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">107492</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2020.03.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.celrep.2020.03.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32268090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmvFSqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=107492&author=K.+R.+Balkaauthor=C.+Louisauthor=T.+L.+Saundersauthor=A.+M.+Smithauthor=D.+J.+Callejaauthor=D.+B.+D%E2%80%99Silvaauthor=F.+Moghaddasauthor=M.+Taillerauthor=K.+E.+Lawlorauthor=Y.+Zhanauthor=C.+J.+Burnsauthor=I.+P.+Wicksauthor=J.+J.+Minerauthor=B.+T.+Kileauthor=S.+L.+Mastersauthor=D.+De+Nardo&title=TBK1+and+IKK%CE%B5+Act+Redundantly+to+Mediate+STING-Induced+NF-%CE%BAB+Responses+in+Myeloid+Cells&doi=10.1016%2Fj.celrep.2020.03.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">TBK1 and IKKε Act Redundantly to Mediate STING-Induced NF-κB Responses in Myeloid Cells</span></div><div class="casAuthors">Balka, Katherine R.; Louis, Cynthia; Saunders, Tahnee L.; Smith, Amber M.; Calleja, Dale J.; D'Silva, Damian B.; Moghaddas, Fiona; Tailler, Maximilien; Lawlor, Kate E.; Zhan, Yifan; Burns, Christopher J.; Wicks, Ian P.; Miner, Jonathan J.; Kile, Benjamin T.; Masters, Seth L.; De Nardo, Dominic</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">107492</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Stimulator of Interferon Genes (STING) is a crit. component of host innate immune defense but can contribute to chronic autoimmune or autoinflammatory disease.  Once activated, the cyclic guanosine monophosphate (GMP)-adenosine monophosphate (AMP) (cGAMP) synthase (cGAS)-STING pathway induces both type I interferon (IFN) expression and nuclear factor-κB (NF-κB)-mediated cytokine prodn.  Currently, these two signaling arms are thought to be mediated by a single upstream kinase, TANK-binding kinase 1 (TBK1).  Here, using genetic and pharmacol. approaches, we show that TBK1 alone is dispensable for STING-induced NF-κB responses in human and mouse immune cells, as well as in vivo.  We further demonstrate that TBK1 acts redundantly with IκB kinase ε (IKKε) to drive NF-κB upon STING activation.  Interestingly, we show that activation of IFN regulatory factor 3 (IRF3) is highly dependent on TBK1 kinase activity, whereas NF-κB is significantly less sensitive to TBK1/IKKε kinase inhibition.  Our work redefines signaling events downstream of cGAS-STING.  Our findings further suggest that cGAS-STING will need to be targeted directly to effectively ameliorate the inflammation underpinning disorders assocd. with STING hyperactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor_LKR-rBD37Vg90H21EOLACvtfcHk0lhnjDSNnKa6jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmvFSqsbg%253D&md5=4483ba5d3616ac553c81a9fc6870bd75</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2020.03.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2020.03.056%26sid%3Dliteratum%253Aachs%26aulast%3DBalka%26aufirst%3DK.%2BR.%26aulast%3DLouis%26aufirst%3DC.%26aulast%3DSaunders%26aufirst%3DT.%2BL.%26aulast%3DSmith%26aufirst%3DA.%2BM.%26aulast%3DCalleja%26aufirst%3DD.%2BJ.%26aulast%3DD%25E2%2580%2599Silva%26aufirst%3DD.%2BB.%26aulast%3DMoghaddas%26aufirst%3DF.%26aulast%3DTailler%26aufirst%3DM.%26aulast%3DLawlor%26aufirst%3DK.%2BE.%26aulast%3DZhan%26aufirst%3DY.%26aulast%3DBurns%26aufirst%3DC.%2BJ.%26aulast%3DWicks%26aufirst%3DI.%2BP.%26aulast%3DMiner%26aufirst%3DJ.%2BJ.%26aulast%3DKile%26aufirst%3DB.%2BT.%26aulast%3DMasters%26aufirst%3DS.%2BL.%26aulast%3DDe%2BNardo%26aufirst%3DD.%26atitle%3DTBK1%2520and%2520IKK%25CE%25B5%2520Act%2520Redundantly%2520to%2520Mediate%2520STING-Induced%2520NF-%25CE%25BAB%2520Responses%2520in%2520Myeloid%2520Cells%26jtitle%3DCell%2520Rep.%26date%3D2020%26volume%3D31%26spage%3D107492%26doi%3D10.1016%2Fj.celrep.2020.03.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">Cyclic GMP-AMP synthase is a cytosolic DNA sensor that activates the type I interferon pathway</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>339</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">791</span>, <span class="refDoi"> DOI: 10.1126/science.1232458</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1126%2Fscience.1232458" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=23258413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1ygurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=786-791&author=L.+Sunauthor=J.+Wuauthor=F.+Duauthor=X.+Chenauthor=Z.+J.+Chen&title=Cyclic+GMP-AMP+synthase+is+a+cytosolic+DNA+sensor+that+activates+the+type+I+interferon+pathway&doi=10.1126%2Fscience.1232458"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic GMP-AMP Synthase Is a Cytosolic DNA Sensor That Activates the Type I Interferon Pathway</span></div><div class="casAuthors">Sun, Lijun; Wu, Jiaxi; Du, Fenghe; Chen, Xiang; Chen, Zhijian J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">339</span>
        (<span class="NLM_cas:issue">6121</span>),
    <span class="NLM_cas:pages">786-791</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The presence of DNA in the cytoplasm of mammalian cells is a danger signal that triggers host immune responses such as the prodn. of type I interferons.  Cytosolic DNA induces interferons through the prodn. of cyclic guanosine monophosphate-adenosine monophosphate (cyclic GMP-AMP, or cGAMP), which binds to and activates the adaptor protein STING.  Through biochem. fractionation and quant. mass spectrometry, we identified a cGAMP synthase (cGAS), which belongs to the nucleotidyltransferase family.  Overexpression of cGAS activated the transcription factor IRF3 and induced interferon-β in a STING-dependent manner.  Knockdown of cGAS inhibited IRF3 activation and interferon-β induction by DNA transfection or DNA virus infection.  CGAS bound to DNA in the cytoplasm and catalyzed cGAMP synthesis.  These results indicate that cGAS is a cytosolic DNA sensor that induces interferons by producing the second messenger cGAMP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxxUcksTNZRbVg90H21EOLACvtfcHk0liZKdVJx4vcXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1ygurw%253D&md5=fe9a64a9c0441eb00852900be8d683bf</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1232458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1232458%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DCyclic%2520GMP-AMP%2520synthase%2520is%2520a%2520cytosolic%2520DNA%2520sensor%2520that%2520activates%2520the%2520type%2520I%2520interferon%2520pathway%26jtitle%3DScience%26date%3D2013%26volume%3D339%26spage%3D786%26epage%3D791%26doi%3D10.1126%2Fscience.1232458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, R. P.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohler, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlauckas, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadreyev, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgibbons, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchison, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libby, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahrendorf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span> <span> </span><span class="NLM_article-title">IRF3 and type I interferons fuel a fatal response to myocardial infarction</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1481</span>– <span class="NLM_lpage">1487</span>, <span class="refDoi"> DOI: 10.1038/nm.4428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnm.4428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=29106401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslKmurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1481-1487&author=K.+R.+Kingauthor=A.+D.+Aguirreauthor=Y.-X.+Yeauthor=Y.+Sunauthor=J.+D.+Rohauthor=R.+P.+Ngauthor=R.+H.+Kohlerauthor=S.+P.+Arlauckasauthor=Y.+Iwamotoauthor=A.+Savolauthor=R.+I.+Sadreyevauthor=M.+Kellyauthor=T.+P.+Fitzgibbonsauthor=K.+A.+Fitzgeraldauthor=T.+Mitchisonauthor=P.+Libbyauthor=M.+Nahrendorfauthor=R.+Weissleder&title=IRF3+and+type+I+interferons+fuel+a+fatal+response+to+myocardial+infarction&doi=10.1038%2Fnm.4428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">IRF3 and type I interferons fuel a fatal response to myocardial infarction</span></div><div class="casAuthors">King, Kevin R.; Aguirre, Aaron D.; Ye, Yu-Xiang; Sun, Yuan; Roh, Jason D.; Ng, Richard P., Jr.; Kohler, Rainer H.; Arlauckas, Sean P.; Iwamoto, Yoshiko; Savol, Andrej; Sadreyev, Ruslan I.; Kelly, Mark; Fitzgibbons, Timothy P.; Fitzgerald, Katherine A.; Mitchison, Timothy; Libby, Peter; Nahrendorf, Matthias; Weissleder, Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1481-1487</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Interferon regulatory factor 3 (IRF3) and type I interferons (IFNs) protect against infections and cancer, but excessive IRF3 activation and type I IFN prodn. cause autoinflammatory conditions such as Aicardi-Goutieacures syndrome and STING-assocd. vasculopathy of infancy (SAVI).  Myocardial infarction (MI) elicits inflammation, but the dominant mol. drivers of MI-assocd. inflammation remain unclear.  Here we show that ischemic cell death and uptake of cell debris by macrophages in the heart fuel a fatal response to MI by activating IRF3 and type I IFN prodn.  In mice, single-cell RNA-seq anal. of 4,215 leukocytes isolated from infarcted and non-infarcted hearts showed that MI provokes activation of an IRF3-interferon axis in a distinct population of interferon-inducible cells (IFNICs) that were classified as cardiac macrophages.  Mice genetically deficient in cyclic GMP-AMP synthase (cGAS), its adaptor STING, IRF3, or the type I IFN receptor IFNAR exhibited impaired interferon-stimulated gene (ISG) expression and, in the case of mice deficient in IRF3 or IFNAR, improved survival after MI as compared to controls.  Interruption of IRF3-dependent signaling resulted in decreased cardiac expression of inflammatory cytokines and chemokines and decreased inflammatory cell infiltration of the heart, as well as in attenuated ventricular dilation and improved cardiac function.  Similarly, treatment of mice with an IFNAR-neutralizing antibody after MI ablated the interferon response and improved left ventricular dysfunction and survival.  These results identify IRF3 and the type I IFN response as a potential therapeutic target for post-MI cardioprotection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwGZt2egB0fLVg90H21EOLACvtfcHk0liZKdVJx4vcXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslKmurrI&md5=dbadb78b95ef1c1c8164c5fdb96af317</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnm.4428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4428%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DK.%2BR.%26aulast%3DAguirre%26aufirst%3DA.%2BD.%26aulast%3DYe%26aufirst%3DY.-X.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DRoh%26aufirst%3DJ.%2BD.%26aulast%3DNg%26aufirst%3DR.%2BP.%26aulast%3DKohler%26aufirst%3DR.%2BH.%26aulast%3DArlauckas%26aufirst%3DS.%2BP.%26aulast%3DIwamoto%26aufirst%3DY.%26aulast%3DSavol%26aufirst%3DA.%26aulast%3DSadreyev%26aufirst%3DR.%2BI.%26aulast%3DKelly%26aufirst%3DM.%26aulast%3DFitzgibbons%26aufirst%3DT.%2BP.%26aulast%3DFitzgerald%26aufirst%3DK.%2BA.%26aulast%3DMitchison%26aufirst%3DT.%26aulast%3DLibby%26aufirst%3DP.%26aulast%3DNahrendorf%26aufirst%3DM.%26aulast%3DWeissleder%26aufirst%3DR.%26atitle%3DIRF3%2520and%2520type%2520I%2520interferons%2520fuel%2520a%2520fatal%2520response%2520to%2520myocardial%2520infarction%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1481%26epage%3D1487%26doi%3D10.1038%2Fnm.4428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Motwani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesiridis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, K. A.</span></span> <span> </span><span class="NLM_article-title">DNA sensing by the cGAS-STING pathway in health and disease</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1038/s41576-019-0151-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41576-019-0151-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=31358977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVGqtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=657-674&author=M.+Motwaniauthor=S.+Pesiridisauthor=K.+A.+Fitzgerald&title=DNA+sensing+by+the+cGAS-STING+pathway+in+health+and+disease&doi=10.1038%2Fs41576-019-0151-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DNA sensing by the cGAS-STING pathway in health and disease</span></div><div class="casAuthors">Motwani, Mona; Pesiridis, Scott; Fitzgerald, Katherine A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">657-674</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The detection of pathogens through nucleic acid sensors is a defining principle of innate immunity.  RNA-sensing and DNA-sensing receptors sample subcellular compartments for foreign nucleic acids and, upon recognition, trigger immune signalling pathways for host defense.  Over the past decade, our understanding of how the recognition of nucleic acids is coupled to immune gene expression has advanced considerably, particularly for the DNA-sensing receptor cyclic GMP-AMP synthase (cGAS) and its downstream signalling effector stimulator of interferon genes (STING), as well as the mol. components and regulation of this pathway.  Moreover, the ability of self-DNA to engage cGAS has emerged as an important mechanism fuelling the development of inflammation and implicating the cGAS-STING pathway in human inflammatory diseases and cancer.  This detailed mechanistic and biol. understanding is paving the way for the development and clin. application of pharmacol. agonists and antagonists in the treatment of chronic inflammation and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA8x0aezpQcLVg90H21EOLACvtfcHk0lg9cKZxlnbJKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVGqtbvM&md5=f13ade7f14b63b453192b5d05f7c9f0b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41576-019-0151-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41576-019-0151-1%26sid%3Dliteratum%253Aachs%26aulast%3DMotwani%26aufirst%3DM.%26aulast%3DPesiridis%26aufirst%3DS.%26aulast%3DFitzgerald%26aufirst%3DK.%2BA.%26atitle%3DDNA%2520sensing%2520by%2520the%2520cGAS-STING%2520pathway%2520in%2520health%2520and%2520disease%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2019%26volume%3D20%26spage%3D657%26epage%3D674%26doi%3D10.1038%2Fs41576-019-0151-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wight-Carter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">Activation of cyclic GMP-AMP synthase by self-DNA causes autoimmune diseases</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E5699</span>– <span class="NLM_lpage">E5705</span>, <span class="refDoi"> DOI: 10.1073/pnas.1516465112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1073%2Fpnas.1516465112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=26371324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E5699-E5705&author=D.+Gaoauthor=T.+Liauthor=X.-D.+Liauthor=X.+Chenauthor=Q.-Z.+Liauthor=M.+Wight-Carterauthor=Z.+J.+Chen&title=Activation+of+cyclic+GMP-AMP+synthase+by+self-DNA+causes+autoimmune+diseases&doi=10.1073%2Fpnas.1516465112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of cyclic GMP-AMP synthase by self-DNA causes autoimmune diseases</span></div><div class="casAuthors">Gao, Daxing; Li, Tuo; Li, Xiao-Dong; Chen, Xiang; Li, Quan-Zhen; Wight-Carter, Mary; Chen, Zhijian J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">E5699-E5705</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">TREX1 is an exonuclease that digests DNA in the cytoplasm.  Loss-of-function mutations of TREX1 are linked to Aicardi-Goutieres Syndrome (AGS) and systemic lupus erythematosus (SLE) in humans.  Trex1-/- mice exhibit autoimmune and inflammatory phenotypes that are assocd. with elevated expression of interferon (IFN)-induced genes (ISGs).  CGMP-AMP (cGAMP) synthase (cGAS) is a cytosolic DNA sensor that activates the IFN pathway.  Upon binding to DNA, cGAS is activated to catalyze the synthesis of cGAMP, which functions as a second messenger that binds and activates the adaptor protein STING to induce IFNs and other cytokines.  Here the authors show that genetic ablation of cGas in Trex1-/- mice eliminated all detectable pathol. and mol. phenotypes, including ISG induction, autoantibody prodn., aberrant T-cell activation, and lethality.  Even deletion of just one allele of cGas largely rescued the phenotypes of Trex1-/- mice.  Similarly, deletion of cGas in mice lacking DNaseII, a lysosomal enzyme that digests DNA, rescued the lethal autoimmune phenotypes of the DNaseII-/- mice.  Through quant. mass spectrometry, the authors found that cGAMP accumulated in mouse tissues deficient in Trex1 or DNaseII and that this accumulation was dependent on cGAS.  These results demonstrate that cGAS activation causes the autoimmune diseases in Trex1-/- and DNaseII-/- mice and suggest that inhibition of cGAS may lead to prevention and treatment of some human autoimmune diseases caused by self-DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoJqBQwGiNVLVg90H21EOLACvtfcHk0lg9cKZxlnbJKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntLnN&md5=1d4056b35c389e3d7fb7a07f85af4cd7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1516465112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1516465112%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.-D.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.-Z.%26aulast%3DWight-Carter%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DActivation%2520of%2520cyclic%2520GMP-AMP%2520synthase%2520by%2520self-DNA%2520causes%2520autoimmune%2520diseases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3DE5699%26epage%3DE5705%26doi%3D10.1073%2Fpnas.1516465112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crow, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manel, N.</span></span> <span> </span><span class="NLM_article-title">Aicardi-Goutières syndrome and the type I interferonopathies</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nri3850</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnri3850" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=26052098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=429-440&author=Y.+J.+Crowauthor=N.+Manel&title=Aicardi-Gouti%C3%A8res+syndrome+and+the+type+I+interferonopathies&doi=10.1038%2Fnri3850"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Aicardi-Goutieres syndrome and the type I interferonopathies</span></div><div class="casAuthors">Crow, Yanick J.; Manel, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">429-440</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dissection of the genetic basis of Aicardi-Goutieres syndrome has highlighted a fundamental link between nucleic acid metab., innate immune sensors and type I interferon induction.  This had led to the concept of the human interferonopathies as a broader set of Mendelian disorders in which a constitutive upregulation of type I interferon activity directly relates to disease pathol.  Here, we discuss the mol. and cellular basis of the interferonopathies, their categorization, future treatment strategies and the insights they provide into normal physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy0LI6TmTndrVg90H21EOLACvtfcHk0liUmVhHlR_6TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zJ&md5=5e1414daa0c2b4d55dabe2d205b65b12</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnri3850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri3850%26sid%3Dliteratum%253Aachs%26aulast%3DCrow%26aufirst%3DY.%2BJ.%26aulast%3DManel%26aufirst%3DN.%26atitle%3DAicardi-Gouti%25C3%25A8res%2520syndrome%2520and%2520the%2520type%2520I%2520interferonopathies%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2015%26volume%3D15%26spage%3D429%26epage%3D440%26doi%3D10.1038%2Fnri3850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grieves, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fye, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grayson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrino, F. W.</span></span> <span> </span><span class="NLM_article-title">Exonuclease TREX1 degrades double-stranded DNA to prevent spontaneous lupus-like inflammatory disease</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">5117</span>– <span class="NLM_lpage">5122</span>, <span class="refDoi"> DOI: 10.1073/pnas.1423804112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1073%2Fpnas.1423804112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=25848017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVGrsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=5117-5122&author=J.+L.+Grievesauthor=J.+M.+Fyeauthor=S.+Harveyauthor=J.+M.+Graysonauthor=T.+Hollisauthor=F.+W.+Perrino&title=Exonuclease+TREX1+degrades+double-stranded+DNA+to+prevent+spontaneous+lupus-like+inflammatory+disease&doi=10.1073%2Fpnas.1423804112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exonuclease TREX1 degrades double-stranded DNA to prevent spontaneous lupus-like inflammatory disease</span></div><div class="casAuthors">Grieves, Jessica L.; Fye, Jason M.; Harvey, Scott; Grayson, Jason M.; Hollis, Thomas; Perrino, Fred W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5117-5122</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The TREX1 gene encodes a potent DNA exonuclease, and mutations in TREX1 cause a spectrum of lupus-like autoimmune diseases.  Most lupus patients develop autoantibodies to double-stranded DNA (dsDNA), but the source of DNA antigen is unknown.  The TREX1 D18N mutation causes a monogenic, cutaneous form of lupus called familial chilblain lupus, and the TREX1 D18N enzyme exhibits dysfunctional dsDNA-degrading activity, providing a link between dsDNA degrdn. and nucleic acid-mediated autoimmune disease.  We detd. the structure of the TREX1 D18N protein in complex with dsDNA, revealing how this exonuclease uses a novel DNA-unwinding mechanism to sep. the polynucleotide strands for single-stranded DNA (ssDNA) loading into the active site.  The TREX1 D18N dsDNA interactions coupled with catalytic deficiency explain how this mutant nuclease prevents dsDNA degrdn.  We tested the effects of TREX1 D18N in vivo by replacing the TREX1 WT gene in mice with the TREX1 D18N allele.  The TREX1 D18N mice exhibit systemic inflammation, lymphoid hyperplasia, vasculitis, and kidney disease.  The obsd. lupus-like inflammatory disease is assocd. with immune activation, prodn. of autoantibodies to dsDNA, and deposition of immune complexes in the kidney.  Thus, dysfunctional dsDNA degrdn. by TREX1 D18N induces disease in mice that recapitulates many characteristics of human lupus.  Failure to clear DNA has long been linked to lupus in humans, and these data point to dsDNA as a key substrate for TREX1 and a major antigen source in mice with dysfunctional TREX1 enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjNgPb6OeTz7Vg90H21EOLACvtfcHk0liUmVhHlR_6TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVGrsL4%253D&md5=fe487e2c7fc4a4ad0421da0351510c2d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1423804112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1423804112%26sid%3Dliteratum%253Aachs%26aulast%3DGrieves%26aufirst%3DJ.%2BL.%26aulast%3DFye%26aufirst%3DJ.%2BM.%26aulast%3DHarvey%26aufirst%3DS.%26aulast%3DGrayson%26aufirst%3DJ.%2BM.%26aulast%3DHollis%26aufirst%3DT.%26aulast%3DPerrino%26aufirst%3DF.%2BW.%26atitle%3DExonuclease%2520TREX1%2520degrades%2520double-stranded%2520DNA%2520to%2520prevent%2520spontaneous%2520lupus-like%2520inflammatory%2520disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3D5117%26epage%3D5122%26doi%3D10.1073%2Fpnas.1423804112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gray, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treuting, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stetson, D. B.</span></span> <span> </span><span class="NLM_article-title">Cutting Edge: cGAS Is Required for Lethal Autoimmune Disease in the Trex1-Deficient Mouse Model of Aicardi-Goutières Syndrome</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">1939</span>– <span class="NLM_lpage">1943</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1500969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.4049%2Fjimmunol.1500969" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=26223655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOju7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2015&pages=1939-1943&author=E.+E.+Grayauthor=P.+M.+Treutingauthor=J.+J.+Woodwardauthor=D.+B.+Stetson&title=Cutting+Edge%3A+cGAS+Is+Required+for+Lethal+Autoimmune+Disease+in+the+Trex1-Deficient+Mouse+Model+of+Aicardi-Gouti%C3%A8res+Syndrome&doi=10.4049%2Fjimmunol.1500969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Cutting Edge: cGAS Is Required for Lethal Autoimmune Disease in the Trex1-Deficient Mouse Model of Aicardi-Goutieres Syndrome</span></div><div class="casAuthors">Gray, Elizabeth E.; Treuting, Piper M.; Woodward, Joshua J.; Stetson, Daniel B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1939-1943</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Detection of intracellular DNA triggers activation of the stimulator of IFN genes-dependent IFN-stimulatory DNA (ISD) pathway, which is essential for antiviral immune responses.  However, chronic activation of this pathway is implicated in autoimmunity.  Mutations in TREX1, a 3' repair exonuclease that degrades cytosolic DNA, cause Aicardi-Goutieres syndrome and chilblain lupus.  Trex1-/- mice develop lethal, IFN-driven autoimmune disease that is dependent on activation of the ISD pathway, but the DNA sensors that detect the endogenous DNA that accumulates in Trex1-/- mice have not been defined.  Multiple DNA sensors have been proposed to activate the ISD pathway, including cyclic GMP-AMP synthase (cGAS).  In this study, we show that Trex1-/- mice lacking cGAS are completely protected from lethality, exhibit dramatically reduced tissue inflammation, and fail to develop autoantibodies.  These findings implicate cGAS as a key driver of autoimmune disease and suggest that cGAS inhibitors may be useful therapeutics for Aicardi-Goutieres syndrome and related autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS0F91-JISBbVg90H21EOLACvtfcHk0ljlJ8WnL9y1LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOju7nN&md5=c455112d5b482de300222cb406b48086</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1500969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1500969%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DE.%2BE.%26aulast%3DTreuting%26aufirst%3DP.%2BM.%26aulast%3DWoodward%26aufirst%3DJ.%2BJ.%26aulast%3DStetson%26aufirst%3DD.%2BB.%26atitle%3DCutting%2520Edge%253A%2520cGAS%2520Is%2520Required%2520for%2520Lethal%2520Autoimmune%2520Disease%2520in%2520the%2520Trex1-Deficient%2520Mouse%2520Model%2520of%2520Aicardi-Gouti%25C3%25A8res%2520Syndrome%26jtitle%3DJ.%2520Immunol.%26date%3D2015%26volume%3D195%26spage%3D1939%26epage%3D1943%26doi%3D10.4049%2Fjimmunol.1500969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rice, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodero, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crow, Y. J.</span></span> <span> </span><span class="NLM_article-title">Human disease phenotypes associated with mutations in TREX1</span>. <i>J. Clin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">243</span>, <span class="refDoi"> DOI: 10.1007/s10875-015-0147-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1007%2Fs10875-015-0147-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=25731743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1yhu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=235-243&author=G.+I.+Riceauthor=M.+P.+Roderoauthor=Y.+J.+Crow&title=Human+disease+phenotypes+associated+with+mutations+in+TREX1&doi=10.1007%2Fs10875-015-0147-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Human Disease Phenotypes Associated With Mutations in TREX1</span></div><div class="casAuthors">Rice, Gillian I.; Rodero, Mathieu P.; Crow, Yanick J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">235-243</span>CODEN:
                <span class="NLM_cas:coden">JCIMDO</span>;
        ISSN:<span class="NLM_cas:issn">0271-9142</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Considering that it is a single exon gene encoding a 314 amino acid protein, the genotype-phenotype landscape of TREX1 is remarkably complex.  Here we briefly describe the human diseases so-far assocd. with mutations in TREX1, which include Aicardi-Goutie´res syndrome, familial chilblain lupus, systemic lupus erythematosus and retinal vasculopathy with cerebral leukodystrophy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuIZPKg1vcJ7Vg90H21EOLACvtfcHk0ljlJ8WnL9y1LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1yhu7w%253D&md5=436ccc68f00d6c7517e9c497b5639848</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs10875-015-0147-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10875-015-0147-3%26sid%3Dliteratum%253Aachs%26aulast%3DRice%26aufirst%3DG.%2BI.%26aulast%3DRodero%26aufirst%3DM.%2BP.%26aulast%3DCrow%26aufirst%3DY.%2BJ.%26atitle%3DHuman%2520disease%2520phenotypes%2520associated%2520with%2520mutations%2520in%2520TREX1%26jtitle%3DJ.%2520Clin.%2520Immunol.%26date%3D2015%26volume%3D35%26spage%3D235%26epage%3D243%26doi%3D10.1007%2Fs10875-015-0147-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ablasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span> <span> </span><span class="NLM_article-title">cGAS in action: expanding roles in immunity and inflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>363</i></span>, <span class="NLM_elocation-id">eaat8657</span> <span class="refDoi"> DOI: 10.1126/science.aat8657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1126%2Fscience.aat8657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=30846571" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=2019&author=A.+Ablasserauthor=Z.+J.+Chen&title=cGAS+in+action%3A+expanding+roles+in+immunity+and+inflammation&doi=10.1126%2Fscience.aat8657"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscience.aat8657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aat8657%26sid%3Dliteratum%253Aachs%26aulast%3DAblasser%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26atitle%3DcGAS%2520in%2520action%253A%2520expanding%2520roles%2520in%2520immunity%2520and%2520inflammation%26jtitle%3DScience%26date%3D2019%26volume%3D363%26doi%3D10.1126%2Fscience.aat8657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, J.</span></span> <span> </span><span class="NLM_article-title">Research advances in how the cGAS-STING pathway controls the cellular inflammatory response</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">615</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2020.00615</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.3389%2Ffimmu.2020.00615" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32411126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVWntr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=615&author=D.+Wanauthor=W.+Jiangauthor=J.+Hao&title=Research+advances+in+how+the+cGAS-STING+pathway+controls+the+cellular+inflammatory+response&doi=10.3389%2Ffimmu.2020.00615"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Research advances in how the cGAS-STING pathway controls the cellular inflammatory response</span></div><div class="casAuthors">Wan, Dongshan; Jiang, Wei; Hao, Junwei</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">615</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review: Double-stranded DNA (dsDNA) sensor cyclic-GMP-AMP synthase (cGAS) along with the downstream stimulator of interferon genes (STING) acting as essential immunesurveillance mediators have become hot topics of research.  The intrinsic function of the cGAS-STING pathway facilitates type-I interferon (IFN) inflammatory signaling responses and other cellular processes such as autophagy, cell survival, senescence. cGASSTING pathway interplays with other innate immune pathways, by which it participates in regulating infection, inflammatory disease, and cancer.  The therapeutic approaches targeting this pathway show promise for future translation into clin. applications.  Here, we present a review of the important previous works and recent advances regarding the cGAS-STING pathway, and provide a comprehensive understanding of the modulatory pattern of the cGAS-STING pathway under multifarious pathol. states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoakdOHwmfKw7Vg90H21EOLACvtfcHk0lhhbPwj8Dc89g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVWntr%252FJ&md5=24d9e8dbfa4781c56444ffbb77b782f6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2020.00615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2020.00615%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DHao%26aufirst%3DJ.%26atitle%3DResearch%2520advances%2520in%2520how%2520the%2520cGAS-STING%2520pathway%2520controls%2520the%2520cellular%2520inflammatory%2520response%26jtitle%3DFront.%2520Immunol.%26date%3D2020%26volume%3D11%26spage%3D615%26doi%3D10.3389%2Ffimmu.2020.00615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of cyclic GMP-AMP synthase ameliorates sepsis-induced cardiac dysfunction in mice</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>260</i></span>,  <span class="NLM_fpage">118315</span>, <span class="refDoi"> DOI: 10.1016/j.lfs.2020.118315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.lfs.2020.118315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32835697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslGqtrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=260&publication_year=2020&pages=118315&author=Q.+Xuauthor=H.+Xiongauthor=W.+Zhuauthor=Y.+Liuauthor=Y.+Du&title=Small+molecule+inhibition+of+cyclic+GMP-AMP+synthase+ameliorates+sepsis-induced+cardiac+dysfunction+in+mice&doi=10.1016%2Fj.lfs.2020.118315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of cyclic GMP-AMP synthase ameliorates sepsis-induced cardiac dysfunction in mice</span></div><div class="casAuthors">Xu, Qingqin; Xiong, Haolan; Zhu, Wenxiu; Liu, Yiping; Du, Yun</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">260</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">118315</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cardiac dysfunction is the main cause of multi-organ failure following sepsis within crit. care units.  The present study aimed to investigate the effects of the small mol. inhibition of cyclic GMP-AMP synthase (cGAS), RU.521, on cardiac function in mice with sepsis.  Sepsis was induced in mice via i.p. lipopolysaccharide (LPS) injection (10 mg/kg, i.p.).  Mice subsequently received 5 mg/kg RU.521 within 10 min form LPS injection.  The cardiac function, inflammatory factor and oxidative stress of mice were examd. for 24 h following LPS injection.RU.521 was indicated to significantly increase the cardiac function of mice with sepsis.  In addn., the inflammatory responses, oxidative stress and apoptosis in hearts of sepsis mice were markedly mitigated by RU.521.  Moreover, inhibition of Sirt3 inhibited the protective effects of RU.521 on mice with sepsis.  The current study indicated that RU.521 alleviated the inflammatory response and alleviated the damage induced by oxidative stress, leading to cardiac protection via increased Sirt3 expression in the hearts of mice with sepsis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrokNqwpfzO0bVg90H21EOLACvtfcHk0lhhbPwj8Dc89g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslGqtrjJ&md5=8c323b596e895377eee8c15ba1375a18</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.lfs.2020.118315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lfs.2020.118315%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520cyclic%2520GMP-AMP%2520synthase%2520ameliorates%2520sepsis-induced%2520cardiac%2520dysfunction%2520in%2520mice%26jtitle%3DLife%2520Sci.%26date%3D2020%26volume%3D260%26spage%3D118315%26doi%3D10.1016%2Fj.lfs.2020.118315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hertzog, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehwinkel, J.</span></span> <span> </span><span class="NLM_article-title">Regulation and inhibition of the DNA sensor cGAS</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>, <span class="NLM_elocation-id">e51345</span> <span class="refDoi"> DOI: 10.15252/embr.202051345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.15252%2Fembr.202051345" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=33155371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXit1KhsrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&author=J.+Hertzogauthor=J.+Rehwinkel&title=Regulation+and+inhibition+of+the+DNA+sensor+cGAS&doi=10.15252%2Fembr.202051345"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation and inhibition of the DNA sensor cGAS</span></div><div class="casAuthors">Hertzog, Jonny; Rehwinkel, Jan</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e51345</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cell-autonomous sensing of nucleic acids is essential for host defense against invading pathogens by inducing antiviral and inflammatory cytokines. cGAS has emerged in recent years as a non-redundant DNA sensor important for detection of many viruses and bacteria.  Upon binding to DNA, cGAS synthesizes the cyclic dinucleotide 2'3'-cGAMP that binds to the adaptor protein STING and thereby triggers IRF3- and NFκB-dependent transcription.  In addn. to infection, the pathophysiol. of an ever-increasing no. of sterile inflammatory conditions in humans involves the recognition of DNA through cGAS.  Consequently, the cGAS/STING signalling axis has emerged as an attractive target for pharmacol. modulation.  However, the development of cGAS and STING inhibitors has just begun and a need for specific and effective compds. persists.  In this review, we focus on cGAS and explore how its activation by immunostimulatory DNA is regulated by cellular mechanisms, viral immune modulators and small mols.  We further use our knowledge of cGAS modulation by cells and viruses to conceptualise potential new ways of pharmacol. cGAS targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOCYhBccK5i7Vg90H21EOLACvtfcHk0lhhbPwj8Dc89g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXit1KhsrbP&md5=519e9236e497e12fc623cd1312ba7fee</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.15252%2Fembr.202051345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembr.202051345%26sid%3Dliteratum%253Aachs%26aulast%3DHertzog%26aufirst%3DJ.%26aulast%3DRehwinkel%26aufirst%3DJ.%26atitle%3DRegulation%2520and%2520inhibition%2520of%2520the%2520DNA%2520sensor%2520cGAS%26jtitle%3DEMBO%2520Rep.%26date%3D2020%26volume%3D21%26doi%3D10.15252%2Fembr.202051345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span> <span> </span><span class="NLM_article-title">Small molecules targeting the innate immune cGAS–STING–TBK1 signaling pathway</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2272</span>– <span class="NLM_lpage">2298</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1016%2Fj.apsb.2020.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=33354501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3MXht1Ciur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=2272-2298&author=C.+Dingauthor=Z.+Songauthor=A.+Shenauthor=T.+Chenauthor=A.+Zhang&title=Small+molecules+targeting+the+innate+immune+cGAS%E2%80%93STING%E2%80%93TBK1+signaling+pathway&doi=10.1016%2Fj.apsb.2020.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules targeting the innate immune cGAS-STING-TBK1 signaling pathway</span></div><div class="casAuthors">Ding, Chunyong; Song, Zilan; Shen, Ancheng; Chen, Tingting; Zhang, Ao</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmaceutica Sinica B</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2272-2298</span>CODEN:
                <span class="NLM_cas:coden">APSBCW</span>;
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Multiple cancer immunotherapies including chimeric antigen receptor T cell and immune checkpoint inhibitors have been successfully developed to treat various cancers by motivating the adaptive anti-tumor immunity.  Particularly, the checkpoint blockade approach has achieved great clinic success as evidenced by several U.S.  Food and drug administration-approved antiprogram death receptor 1/ligand 1 or anti-cytotoxic T lymphocyte assocd. protein 4 antibodies.  However, the majority of cancers have low clin. response rates to these ICIs due to poor tumor immunogenicity.  Indeed, the cyclic guanosine monophosphate-adenosine monophosphate synthase-stimulator of interferon genes-TANK-binding kinase 1 (cGAS-STING-TBK1) axis is now appreciated as the major signaling pathway in innate immune response across different species.  Aberrant signaling of this pathway has been closely linked to multiple diseases, including auto-inflammation, virus infection and cancers.  In this perspective, we provide an updated review on the latest progress on the development of small mol. modulators targeting the cGAS-STING-TBK1 signaling pathway and their preclin. and clin. use as a new immune stimulatory therapy.  Meanwhile, highlights on the clin. candidates, limitations and challenges, as well as future directions in this field are also discussed.  Further, small mol. inhibitors targeting this signaling axis and their potential therapeutic use for various indications are discussed as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZQi2de6gBbVg90H21EOLACvtfcHk0lhGPCbOPjYjww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXht1Ciur8%253D&md5=768d27bbb1096ec9f94133ef965a49d1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DA.%26atitle%3DSmall%2520molecules%2520targeting%2520the%2520innate%2520immune%2520cGAS%25E2%2580%2593STING%25E2%2580%2593TBK1%2520signaling%2520pathway%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26volume%3D10%26spage%3D2272%26epage%3D2298%26doi%3D10.1016%2Fj.apsb.2020.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkon, K. B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Cyclic GMP-AMP Synthase Using a Novel Antimalarial Drug Derivative in Trex1 -Deficient Mice</span>. <i>Arthritis Rheumatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">1807</span>– <span class="NLM_lpage">1819</span>, <span class="refDoi"> DOI: 10.1002/art.40559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1002%2Fart.40559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=29781188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVartbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=1807-1819&author=J.+Anauthor=J.+J.+Woodwardauthor=W.+Laiauthor=M.+Minieauthor=X.+Sunauthor=L.+Tanakaauthor=J.+M.+Snyderauthor=T.+Sasakiauthor=K.+B.+Elkon&title=Inhibition+of+Cyclic+GMP-AMP+Synthase+Using+a+Novel+Antimalarial+Drug+Derivative+in+Trex1+-Deficient+Mice&doi=10.1002%2Fart.40559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Cyclic GMP-AMP Synthase Using a Novel Antimalarial Drug Derivative in Trex1-Deficient Mice</span></div><div class="casAuthors">An, Jie; Woodward, Joshua J.; Lai, Weinan; Minie, Mark; Sun, Xizhang; Tanaka, Lena; Snyder, Jessica M.; Sasaki, Tomikazu; Elkon, Keith B.</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1807-1819</span>CODEN:
                <span class="NLM_cas:coden">ARRHBO</span>;
        ISSN:<span class="NLM_cas:issn">2326-5191</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Type I interferon (IFN) is strongly implicated in the pathogenesis of systemic lupus erythematosus (SLE) as well as rare monogenic interferonopathies such as Aicardi-Goutieres syndrome (AGS), a disease attributed to mutations in the DNA exonuclease TREX1.  The DNA-activated type I IFN pathway cyclic GMP-AMP (cGAMP) synthase (cGAS) is linked to subsets of AGS and lupus.  This study was undertaken to identify inhibitors of the DNA-cGAS interaction, and to test the lead candidate drug, X6, in a mouse model of AGS.  Trex1-/- mice were treated orally from birth with either X6 or hydroxychloroquine (HCQ) for 8 wk.  Expression of IFN-stimulated genes (ISGs) was quantified by quant. polymerase chain reaction.  Multiple reaction monitoring by ultra-performance liq. chromatog. coupled with tandem mass spectrometry was used to quantify the prodn. of cGAMP and X6 drug concns. in the serum and heart tissue of Trex1-/- mice.  On the basis of the efficacy-to-toxicity ratio established in vitro, drug X6 was selected as the lead candidate for treatment of Trex1-/- mice.  X6 was significantly more effective than HCQ in attenuating ISG expression in mouse spleens (P < 0.01 for Isg15 and Isg20) and hearts (P < 0.05 for Isg15,Mx1, and Ifnb, and P < 0.01 for Cxcl10), and in reducing the prodn. of cGAMP in mouse heart tissue (P < 0.05), thus demonstrating target engagement by the X6 compd.  Of note, X6 was also more effective than HCQ in reducing ISG expression in vitro (P < 0.05 for IFI27 and MX1, and P < 0.01 for IFI44L and PKR) in human peripheral blood mononuclear cells from patients with SLE.  This study demonstrates that X6 is superior to HCQ for the treatment of an exptl. autoimmune myocarditis mediated in vivo by the cGAS/stimulator of IFN genes (cGAS/STING) pathway.  The findings suggest that drug X6 could be developed as a novel treatment for AGS and/or lupus to inhibit activation of the cGAS/STING pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0sJrbIuCqErVg90H21EOLACvtfcHk0lhGPCbOPjYjww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVartbjL&md5=d6183689e4227339f6a41ef1f9bbaae2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fart.40559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.40559%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DWoodward%26aufirst%3DJ.%2BJ.%26aulast%3DLai%26aufirst%3DW.%26aulast%3DMinie%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DTanaka%26aufirst%3DL.%26aulast%3DSnyder%26aufirst%3DJ.%2BM.%26aulast%3DSasaki%26aufirst%3DT.%26aulast%3DElkon%26aufirst%3DK.%2BB.%26atitle%3DInhibition%2520of%2520Cyclic%2520GMP-AMP%2520Synthase%2520Using%2520a%2520Novel%2520Antimalarial%2520Drug%2520Derivative%2520in%2520Trex1%2520-Deficient%2520Mice%26jtitle%3DArthritis%2520Rheumatol.%26date%3D2018%26volume%3D70%26spage%3D1807%26epage%3D1819%26doi%3D10.1002%2Fart.40559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sooreshjani, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opoku-Temeng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintim, H. O.</span></span> <span> </span><span class="NLM_article-title">Suramin potently inhibits cGAMP synthase, cGAS, in THP1 cells to modulate IFN-β levels</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1301</span>– <span class="NLM_lpage">1317</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.4155%2Ffmc-2017-0322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=29558821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVyqsrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=1301-1317&author=M.+Wangauthor=M.+A.+Sooreshjaniauthor=C.+Mikekauthor=C.+Opoku-Temengauthor=H.+O.+Sintim&title=Suramin+potently+inhibits+cGAMP+synthase%2C+cGAS%2C+in+THP1+cells+to+modulate+IFN-%CE%B2+levels&doi=10.4155%2Ffmc-2017-0322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Suramin potently inhibits cGAMP synthase, cGAS, in THP1 cells to modulate IFN-β levels</span></div><div class="casAuthors">Wang, Modi; Sooreshjani, Moloud A.; Mikek, Clinton; Opoku-Temeng, Clement; Sintim, Herman O.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1301-1317</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Aim: Persistent activation of STING pathway is the basis for several autoimmune diseases.  STING is activated by cGAMP, which is produced by cGAS in the presence of DNA.  Results/methodol.: HPLC-based medium throughput screening for inhibitors of cGAS identified suramin as a potent inhibitor.  Unlike other reported cGAS inhibitors, which bind to the ATP/GTP binding site, suramin displaced the bound DNA from cGAS.  Addn. of suramin to THP1 cells reduced the levels of IFN-β mRNA and protein.  Suramin did not inhibit lipopolysaccharide- or Pam3CSK4-induced IL-6 mRNA expression.  Conclusion: Suramin inhibits STING pathway via the inhibition of cGAS enzymic activity.  Suramin or analogs thereof that displace DNA from cGAS could be used as anti-inflammatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf9aUiYhp5vLVg90H21EOLACvtfcHk0likNuGEd6wxAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVyqsrnJ&md5=d2ffc1db4e6041db7ac765bdb192f9be</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0322%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DSooreshjani%26aufirst%3DM.%2BA.%26aulast%3DMikek%26aufirst%3DC.%26aulast%3DOpoku-Temeng%26aufirst%3DC.%26aulast%3DSintim%26aufirst%3DH.%2BO.%26atitle%3DSuramin%2520potently%2520inhibits%2520cGAMP%2520synthase%252C%2520cGAS%252C%2520in%2520THP1%2520cells%2520to%2520modulate%2520IFN-%25CE%25B2%2520levels%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D1301%26epage%3D1317%26doi%3D10.4155%2Ffmc-2017-0322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adura, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lama, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aida, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothamel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogakos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reasoner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aso, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuschl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascano, M.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of cGAS reduces interferon expression in primary macrophages from autoimmune mice</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">750</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01770-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41467-017-00833-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=28963528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FjvVKjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=750&author=J.+Vincentauthor=C.+Aduraauthor=P.+Gaoauthor=A.+Luzauthor=L.+Lamaauthor=Y.+Asanoauthor=R.+Okamotoauthor=T.+Imaedaauthor=J.+Aidaauthor=K.+Rothamelauthor=T.+Gogakosauthor=J.+Steinbergauthor=S.+Reasonerauthor=K.+Asoauthor=T.+Tuschlauthor=D.+J.+Patelauthor=J.+F.+Glickmanauthor=M.+Ascano&title=Small+molecule+inhibition+of+cGAS+reduces+interferon+expression+in+primary+macrophages+from+autoimmune+mice&doi=10.1038%2Fs41467-017-01770-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of cGAS reduces interferon expression in primary macrophages from autoimmune mice</span></div><div class="casAuthors">Vincent Jessica; Rothamel Katherine; Reasoner Seth; Ascano Manuel; Adura Carolina; Luz Antonio; Lama Lodoe; Gogakos Tasos; Steinberg Joshua; Tuschl Thomas; Glickman J Fraser; Gao Pu; Patel Dinshaw J; Gao Pu; Luz Antonio; Lama Lodoe; Gogakos Tasos; Steinberg Joshua; Tuschl Thomas; Asano Yasutomi; Okamoto Rei; Imaeda Toshihiro; Aida Jumpei; Aso Kazuyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">750</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclic GMP-AMP synthase is essential for innate immunity against infection and cellular damage, serving as a sensor of DNA from pathogens or mislocalized self-DNA.  Upon binding double-stranded DNA, cyclic GMP-AMP synthase synthesizes a cyclic dinucleotide that initiates an inflammatory cellular response.  Mouse studies that recapitulate causative mutations in the autoimmune disease Aicardi-Goutieres syndrome demonstrate that ablating the cyclic GMP-AMP synthase gene abolishes the deleterious phenotype.  Here, we report the discovery of a class of cyclic GMP-AMP synthase inhibitors identified by a high-throughput screen.  These compounds possess defined structure-activity relationships and we present crystal structures of cyclic GMP-AMP synthase, double-stranded DNA, and inhibitors within the enzymatic active site.  We find that a chemically improved member, RU.521, is active and selective in cellular assays of cyclic GMP-AMP synthase-mediated signaling and reduces constitutive expression of interferon in macrophages from a mouse model of Aicardi-Goutieres syndrome.  RU.521 will be useful toward understanding the biological roles of cyclic GMP-AMP synthase and can serve as a molecular scaffold for development of future autoimmune therapies.Upon DNA binding cyclic GMP-AMP synthase (cGAS) produces a cyclic dinucleotide, which leads to the upregulation of inflammatory genes.  Here the authors develop small molecule cGAS inhibitors, functionally characterize them and present the inhibitor and DNA bound cGAS crystal structures, which will facilitate drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKzRlR5ipxKZagfuraLLzSfW6udTcc2ea50y3j6S90Orntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FjvVKjsw%253D%253D&md5=778f64f98841cea504e15155c5b83db2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00833-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00833-9%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DJ.%26aulast%3DAdura%26aufirst%3DC.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DLuz%26aufirst%3DA.%26aulast%3DLama%26aufirst%3DL.%26aulast%3DAsano%26aufirst%3DY.%26aulast%3DOkamoto%26aufirst%3DR.%26aulast%3DImaeda%26aufirst%3DT.%26aulast%3DAida%26aufirst%3DJ.%26aulast%3DRothamel%26aufirst%3DK.%26aulast%3DGogakos%26aufirst%3DT.%26aulast%3DSteinberg%26aufirst%3DJ.%26aulast%3DReasoner%26aufirst%3DS.%26aulast%3DAso%26aufirst%3DK.%26aulast%3DTuschl%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DGlickman%26aufirst%3DJ.%2BF.%26aulast%3DAscano%26aufirst%3DM.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520cGAS%2520reduces%2520interferon%2520expression%2520in%2520primary%2520macrophages%2520from%2520autoimmune%2520mice%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D750%26doi%3D10.1038%2Fs41467-017-01770-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascano, M.</span></span> <span> </span><span class="NLM_article-title">Small molecule inhibition of human cGAS reduces total cGAMP output and cytokine expression in cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">7604</span>, <span class="refDoi"> DOI: 10.1038/s41598-020-64348-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41598-020-64348-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32371942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovFGqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=7604&author=C.+Wiserauthor=B.+Kimauthor=J.+Vincentauthor=M.+Ascano&title=Small+molecule+inhibition+of+human+cGAS+reduces+total+cGAMP+output+and+cytokine+expression+in+cells&doi=10.1038%2Fs41598-020-64348-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibition of human cGAS reduces total cGAMP output and cytokine expression in cells</span></div><div class="casAuthors">Wiser, Caroline; Kim, Byungil; Vincent, Jessica; Ascano, Manuel</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7604</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The cGAS-STING pathway is a major mechanism that mammalian cells utilize to detect cytoplasmic dsDNA from incoming viruses, bacteria, or self.  CYCLIC GMP-AMP SYNTHASE (cGAS) is the sensor protein that directly binds dsDNAs. cGAS synthesizes cyclic GMP-AMP (cGAMP), which binds to the adaptor STIMULATOR OF INTERFERON GENES (STING), activating an INTERFERON REGULATORY FACTOR 3 (IRF3)-mediated immune response.  Constitutive activation can result in interferonopathies such as Aicardi-Goutieres Syndrome (AGS) or other lupus-like autoimmune disorders.  While inhibitors targeting mouse or human cGAS have been reported, the identification of a small mol. that targets both homologs of cGAS has been challenging.  Here, we show that RU.521 is capable of potently and selectively inhibiting mouse and human cGAS in cell lines and human primary cells.  This inhibitory activity requires the presence of cGAS, but it cannot suppress an immune response in cells activated by RNA, Toll-like receptor ligands, cGAMP, or recombinant interferon.  Importantly, when RU.521 is applied to cells, the prodn. of dsDNA-induced intracellular cGAMP is suppressed in a dose-dependent manner.  Our work validates the use of RU.521 for probing DNA-induced innate immune responses and underscores its potential as an ideal scaffold towards pre-clin. development, given its potency against human and mouse cGAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbVStZ5qmGtLVg90H21EOLACvtfcHk0likNuGEd6wxAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovFGqtrw%253D&md5=3e9827f13073229732f715fa9ada912b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41598-020-64348-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-020-64348-y%26sid%3Dliteratum%253Aachs%26aulast%3DWiser%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DVincent%26aufirst%3DJ.%26aulast%3DAscano%26aufirst%3DM.%26atitle%3DSmall%2520molecule%2520inhibition%2520of%2520human%2520cGAS%2520reduces%2520total%2520cGAMP%2520output%2520and%2520cytokine%2520expression%2520in%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2020%26volume%3D10%26spage%3D7604%26doi%3D10.1038%2Fs41598-020-64348-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lama, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adura, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamei, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuryavyi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gogakos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinberg, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos-Espiritu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashizume, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aida, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinke, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuschl, T.</span></span> <span> </span><span class="NLM_article-title">Development of human cGAS-specific small-molecule inhibitors for repression of dsDNA-triggered interferon expression</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2261</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-08620-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41467-019-08620-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=31113940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A280%3ADC%252BB3M7ot1Onsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=2261&author=L.+Lamaauthor=C.+Aduraauthor=W.+Xieauthor=D.+Tomitaauthor=T.+Kameiauthor=V.+Kuryavyiauthor=T.+Gogakosauthor=J.+I.+Steinbergauthor=M.+Millerauthor=L.+Ramos-Espirituauthor=Y.+Asanoauthor=S.+Hashizumeauthor=J.+Aidaauthor=T.+Imaedaauthor=R.+Okamotoauthor=A.+J.+Jenningsauthor=M.+Michinoauthor=T.+Kuroitaauthor=A.+Stamfordauthor=P.+Gaoauthor=P.+Meinkeauthor=J.+F.+Glickmanauthor=D.+J.+Patelauthor=T.+Tuschl&title=Development+of+human+cGAS-specific+small-molecule+inhibitors+for+repression+of+dsDNA-triggered+interferon+expression&doi=10.1038%2Fs41467-019-08620-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Development of human cGAS-specific small-molecule inhibitors for repression of dsDNA-triggered interferon expression</span></div><div class="casAuthors">Lama Lodoe; Gogakos Tasos; Steinberg Joshua I; Tuschl Thomas; Adura Carolina; Ramos-Espiritu Lavoisier; Glickman J Fraser; Xie Wei; Kuryavyi Vitaly; Gao Pu; Patel Dinshaw J; Tomita Daisuke; Kamei Taku; Miller Michael; Asano Yasutomi; Hashizume Shogo; Aida Jumpei; Imaeda Toshihiro; Okamoto Rei; Jennings Andy J; Michino Mayako; Kuroita Takanobu; Stamford Andrew; Meinke Peter; Gao Pu</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2261</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclic GMP-AMP synthase (cGAS) is the primary sensor for aberrant intracellular dsDNA producing the cyclic dinucleotide cGAMP, a second messenger initiating cytokine production in subsets of myeloid lineage cell types.  Therefore, inhibition of the enzyme cGAS may act anti-inflammatory.  Here we report the discovery of human-cGAS-specific small-molecule inhibitors by high-throughput screening and the targeted medicinal chemistry optimization for two molecular scaffolds.  Lead compounds from one scaffold co-crystallize with human cGAS and occupy the ATP- and GTP-binding active site.  The specificity and potency of these drug candidates is further documented in human myeloid cells including primary macrophages.  These novel cGAS inhibitors with cell-based activity will serve as probes into cGAS-dependent innate immune pathways and warrant future pharmacological studies for treatment of cGAS-dependent inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT436YtiSqWi9TiMd2aNQVUfW6udTcc2eambva-2X7uBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M7ot1Onsw%253D%253D&md5=040007dd8aa93e051ee32fad05510622</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-08620-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-08620-4%26sid%3Dliteratum%253Aachs%26aulast%3DLama%26aufirst%3DL.%26aulast%3DAdura%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DKamei%26aufirst%3DT.%26aulast%3DKuryavyi%26aufirst%3DV.%26aulast%3DGogakos%26aufirst%3DT.%26aulast%3DSteinberg%26aufirst%3DJ.%2BI.%26aulast%3DMiller%26aufirst%3DM.%26aulast%3DRamos-Espiritu%26aufirst%3DL.%26aulast%3DAsano%26aufirst%3DY.%26aulast%3DHashizume%26aufirst%3DS.%26aulast%3DAida%26aufirst%3DJ.%26aulast%3DImaeda%26aufirst%3DT.%26aulast%3DOkamoto%26aufirst%3DR.%26aulast%3DJennings%26aufirst%3DA.%2BJ.%26aulast%3DMichino%26aufirst%3DM.%26aulast%3DKuroita%26aufirst%3DT.%26aulast%3DStamford%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DP.%26aulast%3DMeinke%26aufirst%3DP.%26aulast%3DGlickman%26aufirst%3DJ.%2BF.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DTuschl%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520human%2520cGAS-specific%2520small-molecule%2520inhibitors%2520for%2520repression%2520of%2520dsDNA-triggered%2520interferon%2520expression%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D2261%26doi%3D10.1038%2Fs41467-019-08620-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brault, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumlao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aivazian, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dower, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawrylik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golenbock, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovering, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niljanskul, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nistler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyminski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefker, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, K.</span></span> <span> </span><span class="NLM_article-title">Discovery of PF-06928215 as a high affinity inhibitor of cGAS enabled by a novel fluorescence polarization assay</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0184843</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0184843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1371%2Fjournal.pone.0184843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=28934246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVyltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=J.+Hallauthor=A.+Braultauthor=F.+Vincentauthor=S.+Wengauthor=H.+Wangauthor=D.+Dumlaoauthor=A.+Aulabaughauthor=D.+Aivazianauthor=D.+Castroauthor=M.+Chenauthor=J.+Culpauthor=K.+Dowerauthor=J.+Gardnerauthor=S.+Hawrylikauthor=D.+Golenbockauthor=D.+Hepworthauthor=M.+Hornauthor=L.+Jonesauthor=P.+Jonesauthor=E.+Latzauthor=J.+Liauthor=L.-L.+Linauthor=W.+Linauthor=D.+Linauthor=F.+Loveringauthor=N.+Niljanskulauthor=R.+Nistlerauthor=B.+Pierceauthor=O.+Plotnikovaauthor=D.+Schmittauthor=S.+Shankerauthor=J.+Smithauthor=W.+Snyderauthor=T.+Subashiauthor=J.+Trujilloauthor=E.+Tyminskiauthor=G.+Wangauthor=J.+Wongauthor=B.+Lefkerauthor=L.+Dakinauthor=K.+Leach&title=Discovery+of+PF-06928215+as+a+high+affinity+inhibitor+of+cGAS+enabled+by+a+novel+fluorescence+polarization+assay&doi=10.1371%2Fjournal.pone.0184843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of PF-06928215 as a high affinity inhibitor of cGAS enabled by a novel fluorescence polarization assay</span></div><div class="casAuthors">Hall, Justin; Brault, Amy; Vincent, Fabien; Weng, Shawn; Wang, Hong; Dumlao, Darren; Aulabaugh, Ann; Aivazian, Dikran; Castro, Dana; Chen, Ming; Culp, Jeffrey; Dower, Ken; Gardner, Joseph; Hawrylik, Steven; Golenbock, Douglas; Hepworth, David; Horn, Mark; Jones, Lyn; Jones, Peter; Latz, Eicke; Li, Jing; Lin, Lih-Ling; Lin, Wen; Lin, David; Lovering, Frank; Niljanskul, Nootaree; Nistler, Ryan; Pierce, Betsy; Plotnikova, Olga; Schmitt, Daniel; Shanker, Suman; Smith, James; Snyder, William; Subashi, Timothy; Trujillo, John; Tyminski, Edyta; Wang, Guoxing; Wong, Jimson; Lefker, Bruce; Dakin, Leslie; Leach, Karen</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e0184843/1-e0184843/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Cyclic GMP-AMP synthase (cGAS) initiates the innate immune system in response to cytosolic dsDNA.  After binding and activation from dsDNA, cGAS uses ATP and GTP to synthesize 2', 3' -cGAMP (cGAMP), a cyclic dinucleotide second messenger with mixed 2'-5' and 3'-5' phosphodiester bonds.  Inappropriate stimulation of cGAS has been implicated in autoimmune disease such as systemic lupus erythematosus, thus inhibition of cGAS may be of therapeutic benefit in some diseases; however, the size and polarity of the cGAS active site makes it a challenging target for the development of conventional substrate-competitive inhibitors.  We report here the development of a high affinity (KD = 200 nM) inhibitor from a low affinity fragment hit with supporting biochem. and structural data showing these mols. bind to the cGAS active site.  We also report a new high throughput cGAS fluorescence polarization (FP)-based assay to enable the rapid identification and optimization of cGAS inhibitors.  This FP assay uses Cy5-labeled cGAMP in combination with a novel high affinity monoclonal antibody that specifically recognizes cGAMP with no cross reactivity to cAMP, cGMP, ATP, or GTP.  Given its role in the innate immune response, cGAS is a promising therapeutic target for autoinflammatory disease.  Our results demonstrate its druggability, provide a high affinity tool compd., and establish a high throughput assay for the identification of next generation cGAS inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvbHF0YrJ_lbVg90H21EOLACvtfcHk0ljOuIPvnwSf4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVyltLc%253D&md5=3a528a9f63e573c8956583ac6977873a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0184843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0184843%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DJ.%26aulast%3DBrault%26aufirst%3DA.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DWeng%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DDumlao%26aufirst%3DD.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DAivazian%26aufirst%3DD.%26aulast%3DCastro%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DCulp%26aufirst%3DJ.%26aulast%3DDower%26aufirst%3DK.%26aulast%3DGardner%26aufirst%3DJ.%26aulast%3DHawrylik%26aufirst%3DS.%26aulast%3DGolenbock%26aufirst%3DD.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DL.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DLatz%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DL.-L.%26aulast%3DLin%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DNiljanskul%26aufirst%3DN.%26aulast%3DNistler%26aufirst%3DR.%26aulast%3DPierce%26aufirst%3DB.%26aulast%3DPlotnikova%26aufirst%3DO.%26aulast%3DSchmitt%26aufirst%3DD.%26aulast%3DShanker%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DW.%26aulast%3DSubashi%26aufirst%3DT.%26aulast%3DTrujillo%26aufirst%3DJ.%26aulast%3DTyminski%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DWong%26aufirst%3DJ.%26aulast%3DLefker%26aufirst%3DB.%26aulast%3DDakin%26aufirst%3DL.%26aulast%3DLeach%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520PF-06928215%2520as%2520a%2520high%2520affinity%2520inhibitor%2520of%2520cGAS%2520enabled%2520by%2520a%2520novel%2520fluorescence%2520polarization%2520assay%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0184843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">In Silico Screening-Based Discovery of Novel Inhibitors of Human Cyclic GMP-AMP Synthase: A Cross-Validation Study of Molecular Docking and Experimental Testing</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3265</span>– <span class="NLM_lpage">3276</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00171</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00171" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVWisr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=3265-3276&author=W.+Zhaoauthor=M.+Xiongauthor=X.+Yuanauthor=M.+Liauthor=H.+Sunauthor=Y.+Xu&title=In+Silico+Screening-Based+Discovery+of+Novel+Inhibitors+of+Human+Cyclic+GMP-AMP+Synthase%3A+A+Cross-Validation+Study+of+Molecular+Docking+and+Experimental+Testing&doi=10.1021%2Facs.jcim.0c00171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Screening-Based Discovery of Novel Inhibitors of Human Cyclic GMP-AMP Synthase: A Cross-Validation Study of Molecular Docking and Experimental Testing</span></div><div class="casAuthors">Zhao, Wenfeng; Xiong, Muya; Yuan, Xiaojing; Li, Minjun; Sun, Hongbin; Xu, Yechun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3265-3276</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic GMP-AMP synthase (cGAS) has been recently uncovered to be a promising therapeutic target for immune-assocd. diseases.  Until now, only a few inhibitors have been identified through high-throughput screening campaigns.  Here, we reported the discovery of novel inhibitors for the catalytic domain of human cGAS (h-cGASCD) by virtual screening for the first time.  To generate a reliable docking mode, we first obtained a high-resoln. crystal structure of h-cGASCD in complex with PF-06928215, a known inhibitor of h-cGAS, followed by mol. dynamics simulations on this complex structure.  Four fragment hits were identified by the virtual screening together with a thermal shift assay.  The crystal structures of these four compds. in complex with h-cGASCD were subsequently detd., and the binding modes of the compds. were similar to those predicted by mol. docking, supporting the reliability of the docking model.  In addn., an enzyme activity assay identified compd. 18 (IC50 = 29.88 ± 3.20μM) from the compds. predicted by the virtual screening.  A similarity search of compd. 18 followed by a second virtual screening led to the discovery of compds. S2 (IC50 = 13.1 ± 0.09μM) and S3 (IC50 = 4.9 ± 0.26μM) as h-cGAS inhibitors with improved potency.  Therefore, the present study not only provides the validated hit compds. for further development of h-cGAS inhibitors but also demonstrates a cross-validation study of virtual screening, in vitro exptl. assays, and crystal structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzDBOsuhjDALVg90H21EOLACvtfcHk0lgI79byIunkxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVWisr3P&md5=8222d7ccfff534af59f489344d2fc441</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00171%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DXiong%26aufirst%3DM.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DIn%2520Silico%2520Screening-Based%2520Discovery%2520of%2520Novel%2520Inhibitors%2520of%2520Human%2520Cyclic%2520GMP-AMP%2520Synthase%253A%2520A%2520Cross-Validation%2520Study%2520of%2520Molecular%2520Docking%2520and%2520Experimental%2520Testing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D3265%26epage%3D3276%26doi%3D10.1021%2Facs.jcim.0c00171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padilla-Salinas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span> <span> </span><span class="NLM_article-title">Discovery of small-molecule cyclic GMP-AMP synthase inhibitors</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">1579</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.9b02666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.9b02666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlygt7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2020&pages=1579-1600&author=R.+Padilla-Salinasauthor=L.+Sunauthor=R.+Andersonauthor=X.+Yangauthor=S.+Zhangauthor=Z.+J.+Chenauthor=H.+Yin&title=Discovery+of+small-molecule+cyclic+GMP-AMP+synthase+inhibitors&doi=10.1021%2Facs.joc.9b02666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small-Molecule Cyclic GMP-AMP Synthase Inhibitors</span></div><div class="casAuthors">Padilla-Salinas, Rosaura; Sun, Lijun; Anderson, Rachel; Yang, Xikang; Zhang, Shuting; Chen, Zhijian J.; Yin, Hang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1579-1600</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic GMP-AMP (cGAS), a cytosolic DNA sensor, plays an important role in the type-I-interferon response.  DNA from either invading microbes or self-origin triggers the enzymic activity of cGAS.  Aberrant activation of cGAS is assocd. with various autoimmune disorders.  Only one selective probe exists for inhibiting cGAS in cells while others are limited by their poor cellular activity or specificity, which underscores the urgency for discovering new cGAS inhibitors.  Here, we describe the development of new small mol. human cGAS (hcGAS) inhibitors (80 compds. synthesized) with high binding affinity in vitro and cellular activity.  Our studies show CU-32 and CU-76, selectively inhibit the DNA pathway in human cells but had no effect on RIG-I-MAVS or Toll-like Receptor pathways.  CU-32 and CU-76 represent a new class of hcGAS inhibitors with activity in cells and provide a new chem. scaffold for designing probes to study cGAS function and development of autoimmune therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0wk7oOFIc47Vg90H21EOLACvtfcHk0lgI79byIunkxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlygt7%252FP&md5=964f1954297ba2625a670dae7712d569</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.9b02666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.9b02666%26sid%3Dliteratum%253Aachs%26aulast%3DPadilla-Salinas%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DAnderson%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%2BJ.%26aulast%3DYin%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520small-molecule%2520cyclic%2520GMP-AMP%2520synthase%2520inhibitors%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2020%26volume%3D85%26spage%3D1579%26epage%3D1600%26doi%3D10.1021%2Facs.joc.9b02666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huffman, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plata, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lairson, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houk, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenvi, R. A.</span></span> <span> </span><span class="NLM_article-title">Electronic complementarity permits hindered butenolide heterodimerization and discovery of novel cGAS/STING pathway antagonists</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/s41557-019-0413-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fs41557-019-0413-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=32042135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivFKqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=310-317&author=B.+J.+Huffmanauthor=S.+Chenauthor=J.+L.+Schwarzauthor=R.+E.+Plataauthor=E.+N.+Chinauthor=L.+L.+Lairsonauthor=K.+N.+Houkauthor=R.+A.+Shenvi&title=Electronic+complementarity+permits+hindered+butenolide+heterodimerization+and+discovery+of+novel+cGAS%2FSTING+pathway+antagonists&doi=10.1038%2Fs41557-019-0413-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Electronic complementarity permits hindered butenolide heterodimerization and discovery of novel cGAS/STING pathway antagonists</span></div><div class="casAuthors">Huffman, Benjamin J.; Chen, Shuming; Schwarz, J. Luca; Plata, R. Erik; Chin, Emily N.; Lairson, Luke L.; Houk, K. N.; Shenvi, Ryan A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">310-317</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Sp3-hybridized attached-rings are common motifs in secondary metabolites and represent tetrahedral equiv. to the biaryl substructures that overpopulate synthetic libraries.  Few methods are available that can link fully substituted carbon atoms of two rings with stereocontrol.  Here we have developed a stereoselective, heteroselective butenolide coupling that exhibits an unusually fast rate of C-C bond formation driven by exquisite complementarity of the reacting π systems.  Heterodimerization generates a compd. collection with topol. complexity and diverse principal moments of inertia.  The special status of the sp3-sp3 attached-ring motif is demonstrated in a high-throughput screen for inhibitors of the cyclic GMP-AMP synthase/stimulator of interferon genes pathway, which recruited these butenolide heterodimers from a field of 250,000 compds.  The driving forces underlying this general attached-ring coupling identify a novel paradigm for the accession of wider natural product chem. space, accelerating the discovery of selective lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUn5ppJV6fn7Vg90H21EOLACvtfcHk0li-OHWxUFHmkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivFKqu78%253D&md5=636d5531e45e89542e7fb349e040a709</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fs41557-019-0413-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-019-0413-8%26sid%3Dliteratum%253Aachs%26aulast%3DHuffman%26aufirst%3DB.%2BJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DSchwarz%26aufirst%3DJ.%2BL.%26aulast%3DPlata%26aufirst%3DR.%2BE.%26aulast%3DChin%26aufirst%3DE.%2BN.%26aulast%3DLairson%26aufirst%3DL.%2BL.%26aulast%3DHouk%26aufirst%3DK.%2BN.%26aulast%3DShenvi%26aufirst%3DR.%2BA.%26atitle%3DElectronic%2520complementarity%2520permits%2520hindered%2520butenolide%2520heterodimerization%2520and%2520discovery%2520of%2520novel%2520cGAS%252FSTING%2520pathway%2520antagonists%26jtitle%3DNat.%2520Chem.%26date%3D2020%26volume%3D12%26spage%3D310%26epage%3D317%26doi%3D10.1038%2Fs41557-019-0413-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship study of amidobenzimidazole analogues leading to potent and systemically administrable stimulator of interferon gene (STING) agonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">1649</span>– <span class="NLM_lpage">1669</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01900</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01900" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVKjs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=1649-1669&author=Z.+Songauthor=X.+Wangauthor=Y.+Zhangauthor=W.+Guauthor=A.+Shenauthor=C.+Dingauthor=H.+Liauthor=R.+Xiaoauthor=M.+Gengauthor=Z.+Xieauthor=A.+Zhang&title=Structure-activity+relationship+study+of+amidobenzimidazole+analogues+leading+to+potent+and+systemically+administrable+stimulator+of+interferon+gene+%28STING%29+agonists&doi=10.1021%2Facs.jmedchem.0c01900"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship Study of Amidobenzimidazole Analogues Leading to Potent and Systemically Administrable Stimulator of Interferon Gene (STING) Agonists</span></div><div class="casAuthors">Song, Zilan; Wang, Xiyuan; Zhang, Yan; Gu, Wangting; Shen, Ancheng; Ding, Chunyong; Li, Han; Xiao, Ruoxuan; Geng, Meiyu; Xie, Zuoquan; Zhang, Ao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1649-1669</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Activation of the stimulator of interferon gene (STING) has emerged as an exciting immuno-oncol. therapeutic strategy; however, the first-generation STING agonists, cyclic dinucleotide (CDN) analogs, have suffered from many disadvantages and failed in clin. trials.  Therefore, non-CDN small-mol. STING agonists are urgently needed.  In view of the unique structure of the high potency of dimeric amidobenzimidazole STING agonist 5, a structural elaboration was conducted by modifying several structural hotspots of this scaffold.  Triazole 40 was identified as a new potent STING activator, possessing EC50 values of 0.24 and 39.51 μM for h- and m-STING, resp.  This compd. has a slightly better pharmacokinetic profile and is >20-fold more aqueously sol. than 5.  It activated the STING signaling dramatically by directly binding and stabilizing all h-STING isoforms and m-STING.  In vivo, intermittent administration of 40 was found to have significant antitumor efficacy with good tolerance in two mouse tumor models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ogCDnt2s27Vg90H21EOLACvtfcHk0li-OHWxUFHmkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhsVKjs7o%253D&md5=bf2c2ac40a81a1424c26719c09348e46</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01900%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DDing%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DR.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DA.%26atitle%3DStructure-activity%2520relationship%2520study%2520of%2520amidobenzimidazole%2520analogues%2520leading%2520to%2520potent%2520and%2520systemically%2520administrable%2520stimulator%2520of%2520interferon%2520gene%2520%2528STING%2529%2520agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D1649%26epage%3D1669%26doi%3D10.1021%2Facs.jmedchem.0c01900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cisar, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clapper, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blankman, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezekowitz, R. A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grice, C. A.</span></span> <span> </span><span class="NLM_article-title">Identification of ABX-1431, a selective inhibitor of monoacylglycerol lipase and clinical candidate for treatment of neurological disorders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9062</span>– <span class="NLM_lpage">9084</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00951</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00951" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVegt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9062-9084&author=J.+S.+Cisarauthor=O.+D.+Weberauthor=J.+R.+Clapperauthor=J.+L.+Blankmanauthor=C.+L.+Henryauthor=G.+M.+Simonauthor=J.+P.+Alexanderauthor=T.+K.+Jonesauthor=R.+A.+B.+Ezekowitzauthor=G.+P.+O%E2%80%99Neillauthor=C.+A.+Grice&title=Identification+of+ABX-1431%2C+a+selective+inhibitor+of+monoacylglycerol+lipase+and+clinical+candidate+for+treatment+of+neurological+disorders&doi=10.1021%2Facs.jmedchem.8b00951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of ABX-1431, a Selective Inhibitor of Monoacylglycerol Lipase and Clinical Candidate for Treatment of Neurological Disorders</span></div><div class="casAuthors">Cisar, Justin S.; Weber, Olivia D.; Clapper, Jason R.; Blankman, Jacqueline L.; Henry, Cassandra L.; Simon, Gabriel M.; Alexander, Jessica P.; Jones, Todd K.; Ezekowitz, R. Alan B.; O'Neill, Gary P.; Grice, Cheryl A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9062-9084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The serine hydrolase monoacylglycerol lipase (MGLL) converts the endogenous cannabinoid receptor agonist 2-arachidonoylglycerol (2-AG) and other monoacylglycerols into fatty acids and glycerol.  Genetic or pharmacol. inactivation of MGLL leads to elevation in 2-AG in the central nervous system and corresponding redns. in arachidonic acid and eicosanoids, producing antinociceptive, anxiolytic, and antineuroinflammatory effects without inducing the full spectrum of psychoactive effects of direct cannabinoid receptor agonists.  Here, we report the optimization of hexafluoroisopropyl carbamate-based irreversible inhibitors of MGLL, culminating in a highly potent, selective, and orally available, CNS-penetrant MGLL inhibitor, 28 (ABX-1431).  Activity-based protein profiling expts. verify the exquisite selectivity of 28 for MGLL vs. other members of the serine hydrolase class.  In vivo, 28 inhibits MGLL activity in rodent brain (ED50 = 0.5-1.4 mg/kg), increases brain 2-AG concns., and suppresses pain behavior in the rat formalin pain model.  ABX-1431 (28) is currently under evaluation in human clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiwfzPpgIMJ7Vg90H21EOLACvtfcHk0lg49fFuVwqd9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVegt7fK&md5=30105d700be0454af72f9668421390a5</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00951%26sid%3Dliteratum%253Aachs%26aulast%3DCisar%26aufirst%3DJ.%2BS.%26aulast%3DWeber%26aufirst%3DO.%2BD.%26aulast%3DClapper%26aufirst%3DJ.%2BR.%26aulast%3DBlankman%26aufirst%3DJ.%2BL.%26aulast%3DHenry%26aufirst%3DC.%2BL.%26aulast%3DSimon%26aufirst%3DG.%2BM.%26aulast%3DAlexander%26aufirst%3DJ.%2BP.%26aulast%3DJones%26aufirst%3DT.%2BK.%26aulast%3DEzekowitz%26aufirst%3DR.%2BA.%2BB.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DG.%2BP.%26aulast%3DGrice%26aufirst%3DC.%2BA.%26atitle%3DIdentification%2520of%2520ABX-1431%252C%2520a%2520selective%2520inhibitor%2520of%2520monoacylglycerol%2520lipase%2520and%2520clinical%2520candidate%2520for%2520treatment%2520of%2520neurological%2520disorders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9062%26epage%3D9084%26doi%3D10.1021%2Facs.jmedchem.8b00951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">Construction of bisindolines via oxidative coupling cyclization</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.9b04037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.9b04037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlGitrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2020&pages=116-119&author=R.+Cuiauthor=J.+Yeauthor=J.+Liauthor=W.+Moauthor=Y.+Gaoauthor=H.+Chen&title=Construction+of+bisindolines+via+oxidative+coupling+cyclization&doi=10.1021%2Facs.orglett.9b04037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Construction of Bisindolines via Oxidative Coupling Cyclization</span></div><div class="casAuthors">Cui, Ranran; Ye, Jinxiang; Li, Jing; Mo, Wenhui; Gao, Yu; Chen, Haijun</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">116-119</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein, a general approach for the efficient construction of three-dimensional bisindolines via oxidative coupling cyclization in an intermol. manner is reported.  This reaction is featured by its operational simplicity, metal-free conditions, lack of protecting group, and high selectivity.  Notably, a wide range of anilines are suitable in this intermol. cyclization, furnishing corresponding bisindolines in up to 98% yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXACZwo7fzx7Vg90H21EOLACvtfcHk0lg49fFuVwqd9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlGitrnL&md5=41f7a7c7af356d09e986fb6004acce0a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.9b04037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.9b04037%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DR.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMo%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DConstruction%2520of%2520bisindolines%2520via%2520oxidative%2520coupling%2520cyclization%26jtitle%3DOrg.%2520Lett.%26date%3D2020%26volume%3D22%26spage%3D116%26epage%3D119%26doi%3D10.1021%2Facs.orglett.9b04037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundb%C3%A4ckauthor=P.+Nordlundauthor=D.+M.+Molina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0licEP6y9wclwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0licEP6y9wclwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ning, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenyang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, G.</span></span> <span> </span><span class="NLM_article-title">Cytosolic DNA-STING-NLRP3 axis is involved in murine acute lung injury induced by lipopolysaccharide</span>. <i>Clin. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e228</span> <span class="refDoi"> DOI: 10.1002/ctm2.228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=10.1002%2Fctm2.228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=33252860" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&author=L.+Ningauthor=W.+Weiauthor=J.+Wenyangauthor=X.+Ruiauthor=G.+Qing&title=Cytosolic+DNA-STING-NLRP3+axis+is+involved+in+murine+acute+lung+injury+induced+by+lipopolysaccharide&doi=10.1002%2Fctm2.228"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fctm2.228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fctm2.228%26sid%3Dliteratum%253Aachs%26aulast%3DNing%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DWenyang%26aufirst%3DJ.%26aulast%3DRui%26aufirst%3DX.%26aulast%3DQing%26aufirst%3DG.%26atitle%3DCytosolic%2520DNA-STING-NLRP3%2520axis%2520is%2520involved%2520in%2520murine%2520acute%2520lung%2520injury%2520induced%2520by%2520lipopolysaccharide%26jtitle%3DClin.%2520Transl.%2520Med.%26date%3D2020%26volume%3D10%26doi%3D10.1002%2Fctm2.228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lgV4xVh71fpTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lgV4xVh71fpTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJU','PDB','6MJU'); return false;">PDB: 6MJU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6MJW','PDB','6MJW'); return false;">PDB: 6MJW</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i98"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00398">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_33501"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00398?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00398</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Proposed mechanism for the formation of compounds <b>53</b> and <b>54</b>, variable-temperature NMR spectra of compound <b>20</b>, <sup>1</sup>H and <sup>13</sup>C spectra of all new compounds, and X-ray crystal structure data for compounds <b>47</b>, <b>53</b>, <b>57</b>, <b>63</b>, and <b>65</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_002.csv">CSV</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>47</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_003.cif">CIF</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>53</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_004.cif">CIF</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>57</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_005.cif">CIF</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>63</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_006.cif">CIF</a>)</p></li><li><p class="inline">Crystallographic data of compound <b>65</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_007.cif">CIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_001.pdf">jm1c00398_si_001.pdf (7.2 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_002.csv">jm1c00398_si_002.csv (6.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_003.cif">jm1c00398_si_003.cif (1.58 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_004.cif">jm1c00398_si_004.cif (556.71 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_005.cif">jm1c00398_si_005.cif (525.78 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_006.cif">jm1c00398_si_006.cif (653.52 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00398/suppl_file/jm1c00398_si_007.cif">jm1c00398_si_007.cif (306.83 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00398&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-11%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00398%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00398" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799be47e8f23285","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
