
<html lang="en"     class="pb-page"  data-request-id="db517cd6-fe23-40e8-a79c-f529c1120028"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2017.60.issue-14;article:article:10.1021/acs.jmedchem.7b00561;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Development of an Aryloxazole Class of Hepatitis C Virus Inhibitors Targeting the Entry Stage of the Viral Replication Cycle" /></meta><meta name="dc.Creator" content="Shanshan  He" /></meta><meta name="dc.Creator" content="Kelin  Li" /></meta><meta name="dc.Creator" content="Billy  Lin" /></meta><meta name="dc.Creator" content="Zongyi  Hu" /></meta><meta name="dc.Creator" content="Jingbo  Xiao" /></meta><meta name="dc.Creator" content="Xin  Hu" /></meta><meta name="dc.Creator" content="Amy Q.  Wang" /></meta><meta name="dc.Creator" content="Xin  Xu" /></meta><meta name="dc.Creator" content="Marc  Ferrer" /></meta><meta name="dc.Creator" content="Noel  Southall" /></meta><meta name="dc.Creator" content="Wei  Zheng" /></meta><meta name="dc.Creator" content="Jeffrey  Aubé" /></meta><meta name="dc.Creator" content="Frank J.  Schoenen" /></meta><meta name="dc.Creator" content="Juan J.  Marugan" /></meta><meta name="dc.Creator" content="T. Jake  Liang" /></meta><meta name="dc.Creator" content="Kevin J.  Frankowski" /></meta><meta name="dc.Description" content="Reliance on hepatitis C virus (HCV) replicon systems and protein-based screening assays has led to treatments that target HCV viral replication proteins. The model does not encompass other viral re..." /></meta><meta name="Description" content="Reliance on hepatitis C virus (HCV) replicon systems and protein-based screening assays has led to treatments that target HCV viral replication proteins. The model does not encompass other viral re..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 13, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00561" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00561" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00561" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00561" /></link>
        
    
    

<title>Development of an Aryloxazole Class of Hepatitis C Virus Inhibitors Targeting the Entry Stage of the Viral Replication Cycle | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00561" /></meta><meta property="og:title" content="Development of an Aryloxazole Class of Hepatitis C Virus Inhibitors Targeting the Entry Stage of the Viral Replication Cycle" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0001.jpeg" /></meta><meta property="og:description" content="Reliance on hepatitis C virus (HCV) replicon systems and protein-based screening assays has led to treatments that target HCV viral replication proteins. The model does not encompass other viral replication cycle steps such as entry, processing, assembly and secretion, or viral host factors. We previously applied a phenotypic high-throughput screening platform based on an infectious HCV system and discovered an aryloxazole-based anti-HCV hit. Structure–activity relationship studies revealed several compounds exhibiting EC50 values below 100 nM. Lead compounds showed inhibition of the HCV pseudoparticle entry, suggesting a different mode of action from existing HCV drugs. Hit 7a and lead 7ii both showed synergistic effects in combination with existing HCV drugs. In vivo pharmacokinetics studies of 7ii showed high liver distribution and long half-life without obvious hepatotoxicity. The lead compounds are promising as preclinical candidates for the treatment of HCV infection and as molecular probes to study HCV pathogenesis." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00561"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00561">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00561&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00561&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00561&amp;href=/doi/10.1021/acs.jmedchem.7b00561" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 6364-6383</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00679" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00693" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Development of an Aryloxazole Class of Hepatitis C Virus Inhibitors Targeting the Entry Stage of the Viral Replication Cycle</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shanshan++He">Shanshan He</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kelin++Li">Kelin Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Billy++Lin">Billy Lin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zongyi++Hu">Zongyi Hu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jingbo++Xiao">Jingbo Xiao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Hu">Xin Hu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Amy+Q.++Wang">Amy Q. Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Xu">Xin Xu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marc++Ferrer">Marc Ferrer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Noel++Southall">Noel Southall</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Zheng">Wei Zheng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey++Aub%C3%A9">Jeffrey Aubé</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-1049-5767" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Frank+J.++Schoenen">Frank J. Schoenen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Juan+J.++Marugan">Juan J. Marugan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=T.+Jake++Liang">T. Jake Liang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kevin+J.++Frankowski">Kevin J. Frankowski</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-7173-6352" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Liver Diseases Branch, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, 10 Center Drive, Bethesda, Maryland 20892-1800, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">University of Kansas Specialized Chemistry Center, University of Kansas, Lawrence, Kansas 66047, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Division of Preclinical Innovation, National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#88e2e9e3ede4c8eaecefb9b8a6e6e1ecece3a6e6e1e0a6efe7fe"><span class="__cf_email__" data-cfemail="8ce6ede7e9e0cceee8ebbdbca2e2e5e8e8e7a2e2e5e4a2ebe3fa">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#a8c3cddec1c6cee8ddc6cb86cdccdd"><span class="__cf_email__" data-cfemail="5e353b283730381e2b303d703b3a2b">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00561&amp;href=/doi/10.1021%2Facs.jmedchem.7b00561" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 6364–6383</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 21, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 May 2017</li><li><span class="item_label"><b>Published</b> online</span>13 July 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 July 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00561" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00561</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6364%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DShanshan%2BHe%252C%2BKelin%2BLi%252C%2BBilly%2BLin%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D14%26contentID%3Dacs.jmedchem.7b00561%26title%3DDevelopment%2Bof%2Ban%2BAryloxazole%2BClass%2Bof%2BHepatitis%2BC%2BVirus%2BInhibitors%2BTargeting%2Bthe%2BEntry%2BStage%2Bof%2Bthe%2BViral%2BReplication%2BCycle%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6383%26publicationDate%3DJuly%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00561"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1392</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00561" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Development of an Aryloxazole Class of Hepatitis C Virus Inhibitors Targeting the Entry Stage of the Viral Replication Cycle&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Shanshan&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Kelin&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Billy&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Zongyi&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Jingbo&quot;,&quot;last_name&quot;:&quot;Xiao&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Amy&quot;,&quot;last_name&quot;:&quot;Q. Wang&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Marc&quot;,&quot;last_name&quot;:&quot;Ferrer&quot;},{&quot;first_name&quot;:&quot;Noel&quot;,&quot;last_name&quot;:&quot;Southall&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;Aubé&quot;},{&quot;first_name&quot;:&quot;Frank&quot;,&quot;last_name&quot;:&quot;J. Schoenen&quot;},{&quot;first_name&quot;:&quot;Juan&quot;,&quot;last_name&quot;:&quot;J. Marugan&quot;},{&quot;first_name&quot;:&quot;T.&quot;,&quot;last_name&quot;:&quot;Jake Liang&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;J. Frankowski&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;13&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;6364-6383&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00561&quot;},&quot;abstract&quot;:&quot;Reliance on hepatitis C virus (HCV) replicon systems and protein-based screening assays has led to treatments that target HCV viral replication proteins. The model does not encompass other viral replication cycle steps such as entry, processing, assembly and secretion, or viral host factors. We previously applied a phenotypic high-throughput screening platform based on an infectious HCV system and discovered an aryloxazole-based anti-HCV hit. Structure–activity relationship studies revealed several compounds exhibiting EC50 values below 100 nM. Lead compounds showed inhibition of the HCV pseudoparticle entry, suggesting a different mode of action from existing HCV drugs. Hit 7a and lead 7ii both showed synergistic effects in combination with existing HCV drugs. In vivo pharmacokinetics studies of 7ii showed high liver distribution and long half-life without obvious hepatotoxicity. The lead compounds are promising as preclinical candidates for the treatment of HCV infection and as molecular probes to study HCV&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00561&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00561" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00561&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00561" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00561&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00561" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00561&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00561&amp;href=/doi/10.1021/acs.jmedchem.7b00561" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00561" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00561" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00561%26sid%3Dliteratum%253Aachs%26pmid%3D28636348%26genre%3Darticle%26aulast%3DHe%26date%3D2017%26atitle%3DDevelopment%2Bof%2Ban%2BAryloxazole%2BClass%2Bof%2BHepatitis%2BC%2BVirus%2BInhibitors%2BTargeting%2Bthe%2BEntry%2BStage%2Bof%2Bthe%2BViral%2BReplication%2BCycle%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D14%26spage%3D6364%26epage%3D6383%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291430" title="Infectious diseases">Infectious diseases</a>,</li><li><a href="/action/doSearch?ConceptID=290659" title="DNA replication">DNA replication</a>,</li><li><a href="/action/doSearch?ConceptID=290677" title="Toxicological synergy">Toxicological synergy</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290808" title="Toxicity">Toxicity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/jmcmar.2017.60.issue-14/20170727/jmcmar.2017.60.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Reliance on hepatitis C virus (HCV) replicon systems and protein-based screening assays has led to treatments that target HCV viral replication proteins. The model does not encompass other viral replication cycle steps such as entry, processing, assembly and secretion, or viral host factors. We previously applied a phenotypic high-throughput screening platform based on an infectious HCV system and discovered an aryloxazole-based anti-HCV hit. Structure–activity relationship studies revealed several compounds exhibiting EC<sub>50</sub> values below 100 nM. Lead compounds showed inhibition of the HCV pseudoparticle entry, suggesting a different mode of action from existing HCV drugs. Hit <b>7a</b> and lead <b>7ii</b> both showed synergistic effects in combination with existing HCV drugs. In vivo pharmacokinetics studies of <b>7ii</b> showed high liver distribution and long half-life without obvious hepatotoxicity. The lead compounds are promising as preclinical candidates for the treatment of HCV infection and as molecular probes to study HCV pathogenesis.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Hepatitis C virus (HCV) leads to chronic infection in 80% of patients, and disease progression can eventually cause liver cancer or cirrhosis following decades of asymptomatic infection.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> HCV is the major underlying cause for liver transplants and is responsible for significant healthcare costs.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The prevalence of HCV has been estimated at around 200 million people worldwide<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and, to date, no effective vaccine has been developed.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The historical treatment regimen for HCV involved combination therapy of ribavirin with pegylated interferon, a poorly tolerated course of treatment with only moderate success in achieving a six-month post-treatment sustained virological response (SVR), an undetectable viral RNA level six months after treatment cessation.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The recent approval of several small-molecule direct-acting antivirals (DAAs) has dramatically improved the standard of care for HCV.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> These drugs target the viral proteins (NS3/4A protease, NS5B polymerase, and NS5A) involved in the replication stage of HCV infection. Although these treatments offer renewed hope toward curing HCV infection, the price of the medicines is prohibitively expensive for many high-risk populations such as intravenous drug users, prisoners, and those in the developing world.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Furthermore, these agents can lead to rapid development of viral resistance if the virus is not fully eradicated during the treatment regimen.</div><div class="NLM_p last">The hegemony of replication inhibitors as treatments for HCV infection is a direct consequence of the available methods to screen for inhibitors of the virus. Current HCV inhibitors have mostly been discovered through either the replicon assay, which measures the replication of isolated viral RNA, or protein-based assays using HCV proteins involved in viral replication (e.g., NS3, NS5A, or NS5B). On the other hand, a cell-based infectious HCV assay platform can cover the complete spectrum of potentially druggable targets in all stages of the HCV replication cycle and allows for the development of inhibitors acting on different phases of the viral life cycle less prone to mutation. More importantly, targeting several key processes in the viral replication cycle may not only increase antiviral efficacy but also reduce the capacity of the virus to develop resistance to the compound. We developed such an assay platform based on an HCV infectious cell culture system.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The assay was adapted for the high-throughput screening of a 350 000-member compound collection, affording 149 validated hit compounds.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Herein, we describe the optimization, preliminary mode of action (MOA) studies, and detailed biopharmaceutical and pharmacokinetic (PK) properties for an aryloxazole class of compounds discovered during the screening campaign and demonstrate that the class is promising for further pharmaceutical development.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82330" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82330" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Chemistry</h3><div class="NLM_p">The resynthesized aryloxazole HTS hit compound and most SAR analogues were constructed according to the general route in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The aryloxazole cores <b>4</b> were assembled using the method of Goto et al. from commercial benzaldehydes <b>1</b> and butane-2,3-dione mono-oxime <b>2</b> via the <i>N</i>-oxide intermediates <b>3</b>.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Derivatization of <b>4</b> with isonipecotic acid provided the carboxylic acids <b>5</b>, which were coupled with the diamine fragments <b>6</b> using diisopropylcarbodiimide and 1-hydroxybenzotriazole (HOBt) hydrate to afford the final analogues <b>7</b>. The coupling reactions were conducted either at room temperature in DCM or at 100 °C using microwave irradiation in MeCN, depending on when during the SAR campaign the analogues were synthesized. Detailed synthetic protocols for individual final analogues are provided in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. The final compounds were purified by either flash chromatography or reverse-phase, preparative-scale, mass-directed high performance liquid chromatography (HPLC), and their purity was assessed by analytical-scale HPLC under analogous conditions. The thiazole core <b>10</b> was synthesized via condensation of 3-chlorobuta-2-one and thioamide <b>8</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Chlorination with <i>N</i>-chlorosuccinimide followed by derivatization with isonipecotic acid and diamine fragment <b>6a</b> as described above afforded the final analogue <b>13</b>. The desmethyl oxazole core was constructed via condensation and monodehalogenation of 1,3-dichloroacetone with benzamide <b>14</b> to directly afford the choromethyloxazole scaffold <b>16</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Derivatization with isonipecotic acid and diamine fragments <b>6</b> as described above afforded the final analogues <b>18</b>. The 3-piperidinylpropylamines <b>6</b> that were not commercially available were readily synthesized by a short two-step sequence (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Briefly, addition of acrylonitrile to the appropriately substituted piperidine followed by Raney nickel-mediated nitrile reduction under a hydrogen atmosphere afforded the requisite diamine fragments <b>6</b>. Full synthetic protocols for individual diamine fragments are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_001.pdf" class="ext-link">Supporting Information</a>.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Route to Aryloxazole Analogues <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) HCl<sub>(g)</sub> (4 M in dioxane), AcOH, 0 °C to rt; (b) POCl<sub>3</sub>, DCE, reflux; (c) KOH, EtOH, isonipecotic acid, rt; (d) HOBt hydrate, DMAP, diisopropylcarbodiimide, DCM, rt or HOBt hydrate, diisopropylcarbodiimide, MeCN, μW irradiation, 100 °C, 10 min.</p></p></figure><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route to Thiazole Analogue <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <i>i</i>PrOH, μW irradiation, 120 °C, 1 h, 65% yield; (b) NCS, MeCN, 60 °C, 2 h, 82% yield; (c) KOH, EtOH, isonipecotic acid, rt, 78% yield; (d) HOBt hydrate, diisopropylcarbodiimide, MeCN, μW irradiation, 100 °C, 10 min, 45% yield.</p></p></figure><figure id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0008.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Representative Synthetic Route to Desmethyloxazole Analogues; Synthesis of <b>18a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) toluene, 140 °C, 5 h, 57% yield; (b) KOH, EtOH, isonipecotic acid, rt, 93% yield; (d) HOBt hydrate, DMAP, diisopropylcarbodiimide, DCM, rt, 60% yield.</p></p></figure><figure id="sch4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0009.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. General Synthetic Route to Diamine Fragments <b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) acrylonitrile, formamide, water; (b) Raney nickel, H<sub>2(g)</sub>, 200 psi, MeOH, NH<sub>3(MeOH)</sub>.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Activity Relationship (SAR) Studies</h3><div class="NLM_p">Shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, the structural elements that were investigated during the SAR study are highlighted in the HTS hit molecule (PubChem CID3244725). The majority of the structural analogues focused on varying either aryl-ring substitution or substitution on, or replacement of, the piperidine moiety (the molecular fragments on the left and right ends of the compound hit, respectively). All analogues were screened for inhibition of HCV infection in the cell-based HCV-Luc assay and counterscreened for cytotoxicity in Huh 7.5.1 cells (ATPlite assay). In addition, most analogues were profiled in three assays to assess in vitro biopharmaceutical properties: rat microsomal stability, cell permeability, and aqueous kinetic solubility. <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> summarize the results of these SAR investigations. The stereochemistry on the dimethyl-substituted piperidine ring of the HTS hit (PubChem CID3244725) was undefined in the registered structure. Thus, we began our SAR investigation by synthesizing both the <i>cis</i> and <i>trans</i> stereoisomers of the hit compound, <b>7a</b> and <b>7b</b>, respectively. The <i>trans</i> stereoisomer <b>7b</b> was synthesized and screened as a racemic mixture. Both isomers were equipotent in the HCV-Luc assay and showed favorable aqueous solubility; however, <b>7a</b> possessed slightly lower cytotoxicity and appeared to have better stability in the rat liver microsomes. Replacing the 3,5-dimethyl substitution with bulkier 3,5-diethyl (<b>7c</b>) or 3,3,5,5-tetramethyl substitution (<b>7d</b>) also afforded potent analogues, though with slightly greater cytotoxicity. Moreover, analogues exploring substitution on the piperidine moiety revealed potency to be dependent on both the size and location of substituents on the heterocycle ring. Sufficiently bulky substitution at the 4-position afforded potent analogues (<b>7e</b> and <b>7f</b>), while a single methyl group substituent at this position (<b>7g</b>) afforded less potent analogues. Consistent with this trend, the unsubstituted piperidine <b>7h</b> was even less potent. While the <i>cis</i>-2,6-dimethyl substituted analogue <b>7i</b> did not improve the potency, the 3-substituted analogues (racemic compounds <b>7j</b>–<b>7l</b>) possessed slightly improved potency compared to that of the 3,5-disubstituted compound <b>7a</b>. The <i>cis</i>-disubstituted compounds (e.g., <b>7a</b>), however, do not possess any asymmetric centers (<i>meso</i> compounds) and were therefore screened as single stereoisomers rather than enantiomeric mixtures. The constrained analogue <b>7m</b> did not provide any improvement in potency or cytotoxicity and added unnecessary structural complexity. In all the above cases, increased steric bulk on the piperidine also correlated with decreased microsomal stability. Decreasing or increasing the tether length by one −CH<sub>2</sub> group in the diamine fragment (<b>7n</b> and <b>7o</b>, respectively) did not improve the potency, and we retained the three-carbon linker for the remaining SAR investigation. Ring-opening of the piperidine to the dialkyl amine <b>7p</b> afforded a slightly less potent analogue, possibly suggesting that the piperidine constraint might not be required. However, limited exploration of more structurally drastic piperidine replacements was detrimental to potency (<b>7q</b>–<b>7s</b>).</div><figure id="fig1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Summary of structural modifications explored in optimizing the aryloxazole hit and structure of the selected lead analogue <b>18a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Effect of Varying the Diamine Fragment on Potency, Cytotoxicity, and in Vitro Biopharmaceutical Properties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0010.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Average of three separate assays (unless otherwise noted) ± standard deviation.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">ND, not determined.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Effect of Varying the Aryl Substitution on Potency, Cytotoxicity, and in Vitro Biopharmaceutical Properties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0011.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0012.gif" alt="" id="fx2a" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">Average of three separate assays (unless otherwise noted) ± standard deviation.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">ND, not determined.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Effect of Core Scaffold Modifications on Potency, Cytotoxicity, and in Vitro Biopharmaceutical Properties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0013.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">Average of three separate assays (unless otherwise noted) ± standard deviation.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">ND, not determined.</p></div></div><div></div></div><div class="NLM_p">The relationship between activity and aryl ring substitution is summarized in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. Changing the position of the 2-methyl substituent on the aryl ring to either the 3- or 4-position (<b>7t</b> and <b>7u</b>, respectively) had no significant effect on potency; however, the 4-methyl analogue <b>7u</b> had increased cytotoxicity. 2-Ethyl substitution (<b>7v</b>) or various arrangements of dimethyl substituted aryl analogues (<b>7w</b>–<b>7aa</b>) all possessed a satisfactory potency of less than 100 nM. Both <b>7v</b> and the 3,5-dimethyl-substituted analogue <b>7w</b> possessed reduced microsomal stability compared to those of other analogues. Further increasing the substitution to tri- and penta-methyl substituted aryl analogues (<b>7bb</b> and <b>7cc</b>, respectively) afforded compounds less potent than those of the mono- or dimethyl substituted analogues. The activities for the monohalogen substituted analogues (<b>7dd</b>–<b>7gg</b>) were on par with the activities for the most potent 2-methyl aryl analogues. The dihalo analogues (<b>7hh</b>–<b>7 mm</b>) were generally less potent; the notable exception being the 3,4-dichloro analogue <b>7ii</b>, which possessed a potency of 13 nM, ultimately the most potent analogue synthesized in this study. Other than <b>7ii</b> and the 3,5-difluoro-substituted analogue <b>7jj</b>, dihalo substitution reduced the microsomal stability. While either monohalogen or methyl substitution was found to afford analogues with attractive potency (<100 nM), the combination of both monohalogen and methyl substitution in a single analogue (<b>7nn</b>–<b>7ss</b>) afforded analogues of only modest potency (124–370 nM) and microsomal stability lower than that of either substitution in isolation. Methoxy substitution (<b>7tt</b>–<b>7zz</b>) was uniformly detrimental to the potency. The naphthyl analogues (<b>7aaa</b> and <b>7bbb</b>) both possess potency below 100 nM; however, the anthracene analogue (<b>7ccc</b>) was significantly less potent (357 nM).</div><div class="NLM_p">We also explored limited structural alterations to the scaffold core, as summarized in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>. The thiazole analogue <b>13</b> displayed potency similar to that of the hit and, with no clear advantage, further thiazole analogues were not explored. The desmethyl oxazole analogues <b>18a</b> and <b>18b</b> were found to be among the most potent analogues, suggesting that the methyl group on the oxazole may not be essential for activity. The <i>cis</i>-dimethylpiperidine analogue <b>18a</b> also possessed a selectivity index (SI = CC<sub>50</sub>/EC<sub>50</sub>) higher than that of either <b>7a</b> or <b>7ii</b> (SI = 97, 244, and 341 for <b>7a</b>, <b>7ii</b>, and <b>18a</b>, respectively).</div><div class="NLM_p last">During this SAR study, we established a clear relationship between substitution on the terminal piperidine moiety and on-target activity; highly potent analogues all contained apparent steric bulk at the 3- and/or 4-position of the piperidine. While a single methyl group at the 3-position gave highly potent analogues, the 4-position required more steric bulk, <i>gem</i> dimethyl or isopropyl substitution to provide equipotent analogues. The aryl moiety distal to the piperidine of the chemotype also allowed some flexibility in substitution. A single substituent (e.g., methyl or chloro) at the <i>ortho</i> position provided highly potent analogues. Disubstituted analogues either at the 3,5- or 3,4-positions also afforded highly potent compounds with these analogues also possessing improved stability in the microsomal stability assay. The limited exploration of changes to the oxazole core indicates that modest changes in this portion of the molecule are tolerated; however, more extensive investigation is needed to establish a more detailed analysis. In summary, we prepared a number of highly potent lead compounds with low cytotoxicity and promising in vitro biopharmaceutical properties.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> In Vitro Profiling of Anti-HCV Activity and Selectivity</h3><div class="NLM_p">During the SAR studies, a number of promising compounds were identified, and three analogues were chosen for further biological evaluation and characterization. <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> shows the representative titration curves for the anti-HCV activity and cytotoxicity of selected leads. Selectivity indices between 103 and 341 were achieved for these compounds, showing a significant therapeutic index. The analogue used for a particular profiling study switched over the course of the project from <b>7nn</b> to <b>7ii</b> once the latter compound emerged as the more potent and selective analogue. The intent of the profiling results is to illustrate the potential of this compound class for further development or mechanistic investigation.</div><figure id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Titration curves for anti-HCV activity (EC<sub>50</sub>, triangle in blue) and cytotoxicity (CC<sub>50</sub>, circle in red) of selected leads. Cell-culture adapted HCV (HCVcc) harboring a luciferase reporter gene (HCV-Luc, genotype 2a) was used to infect Huh7.5.1 cells in the presence of increasing concentrations of test compound. Viral infection and replication were measured by luciferase signal 48 h after treatment. Cytotoxicity was evaluated in parallel with the ATP-based cell viability assay (ATPlite). The results are mean from three replicates ± SEM. The EC<sub>50</sub> and CC<sub>50</sub> values were calculated with GraphPad Prism 5.0 software using nonlinear regression. Curves and values for each compound were from single representative experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To determine whether this class of compounds possessed a distinct antiviral mode of action from those of the known HCV inhibitors, the activities of the hit compound <b>7a</b> and the selected lead compounds were measured at 10 μM in a series of assays focused on discrete phases of the viral replication cycle (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). The HCV-Luc antiviral activity and cytotoxicity were confirmed at the same concentration for comparison. In the HCV single-cycle infection assay, single-round infectious HCV defective particles (HCVsc, genotype 2a) can infect and replicate but do not assemble new virions; thus, this assay detects compounds with inhibitory activity to HCV replication cycle events prior to assembly. The activities of <b>7a</b> and the selected lead compounds <b>7ii</b>, <b>7nn</b>, and <b>18a</b> in the HCVsc infection assay were equivalent to the values obtained in the phenotypic assay, suggesting an early stage target of the viral replication cycle such as HCV entry and/or RNA replication. Therefore, HCV subgenomic replicon assays evaluate whether compounds target viral RNA replication. All compounds showed less than 40% inhibitory effect on HCV replication in GT 1b replicon cell line except for <b>7ii</b>, possibly due to cytotoxicity, as suggested in the ATPlite assay result. This would indicate that replication-stage targets are not responsible for the HCV inhibition. HCV pseuodoparticles (HCVpp and GT 1a and 1b) utilize defective retroviral particles that display HCV envelope glycoproteins. HCVpp are a well-established surrogate system that mimic the entry stage of cell-culture adapted HCV (HCVcc).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> We thus utilized the HCVpp assay to assess the effect of compound treatment on viral entry. Viral pseudoparticles from vesicular stomatitis virus (VSVpp) were used to test the effect of compounds on an unrelated virus. The hit <b>7a</b> showed potent inhibitory activity (<30% RLU of control) in HCVpp GT 1a assay and moderate inhibition (∼50% RLU) in VSVpp assay. The lead compounds also exhibited potent to moderate inhibition in HCVpp GT 1a and 1b assays. The combined profile for <b>7a</b> and the lead compounds suggest that the chemotype is targeting an early stage of the viral replication cycle, likely by inhibiting HCV entry based on the HCVpp activity.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Activity of Selected Leads in a Series of HCV Lifecycle Assays<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="8" align="center" char=".">% RLU<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">HCVpp<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char="±"> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center" char=".">HCV-Luc</th><th class="colsep0 rowsep0" align="center" char=".">ATPlite</th><th class="colsep0 rowsep0" align="center" char=".">HCVsc<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">HCV replicon GT 1b<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char=".">GT 1a</th><th class="colsep0 rowsep0" align="center" char=".">GT 1b</th><th class="colsep0 rowsep0" align="center" char="±">MLVpp<a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center" char=".">VSVpp<a class="ref internalNav" href="#t4fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="char" char=".">2.16 ± 0.508</td><td class="colsep0 rowsep0" align="char" char=".">90.0 ± 3.0</td><td class="colsep0 rowsep0" align="char" char=".">1.63 ± 1.57</td><td class="colsep0 rowsep0" align="char" char=".">75.2 ± 4.9</td><td class="colsep0 rowsep0" align="char" char=".">21.9 ± 14.7</td><td class="colsep0 rowsep0" align="char" char=".">16.8 ± 6.5</td><td class="colsep0 rowsep0" align="char" char="±">48.7 ± 3.85</td><td class="colsep0 rowsep0" align="char" char=".">61.5 ± 5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7ii</b></td><td class="colsep0 rowsep0" align="char" char=".">0.27 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">4.88 ± 0.19</td><td class="colsep0 rowsep0" align="char" char=".">3.92 ± 1.51</td><td class="colsep0 rowsep0" align="char" char=".">9.1 ± 3.0</td><td class="colsep0 rowsep0" align="char" char=".">14.9 ± 11.6</td><td class="colsep0 rowsep0" align="char" char=".">25.3 ± 6.4</td><td class="colsep0 rowsep0" align="char" char="±">57.7 ± 2.4</td><td class="colsep0 rowsep0" align="char" char=".">28.0 ± 3.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7nn</b></td><td class="colsep0 rowsep0" align="char" char=".">0.72 ± 0.32</td><td class="colsep0 rowsep0" align="char" char=".">90.4 ± 1.5</td><td class="colsep0 rowsep0" align="char" char=".">3.29 ± 2.06</td><td class="colsep0 rowsep0" align="char" char=".">83.3 ± 7.0</td><td class="colsep0 rowsep0" align="char" char=".">76.8 ± 7.7</td><td class="colsep0 rowsep0" align="char" char=".">16.6 ± 10.5</td><td class="colsep0 rowsep0" align="char" char="±">102 ± 5</td><td class="colsep0 rowsep0" align="char" char=".">77.7 ± 12.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18a</b></td><td class="colsep0 rowsep0" align="char" char=".">0.59 ± 0.39</td><td class="colsep0 rowsep0" align="char" char=".">74.1 ± 3.0</td><td class="colsep0 rowsep0" align="char" char=".">4.14 ± 1.94</td><td class="colsep0 rowsep0" align="char" char=".">65.2 ± 3.2</td><td class="colsep0 rowsep0" align="char" char=".">35.1 ± 5.8</td><td class="colsep0 rowsep0" align="char" char=".">31.7 ± 8.5</td><td class="colsep0 rowsep0" align="char" char="±">111 ± 6</td><td class="colsep0 rowsep0" align="char" char=".">78.0 ± 4.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Activity of compound (10 μM) in a series of assays to assess efficacy in different stages of the viral life cycle. Results are presented as percentage response compared to the untreated control.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">Relative luminescence units.</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">HCV single-cycle infection assay.</p></div><div class="footnote" id="t4fn4"><sup>d</sup><p class="last">HCV subgenomic replicon assay.</p></div><div class="footnote" id="t4fn5"><sup>e</sup><p class="last">HCV pseudoparticle assay.</p></div><div class="footnote" id="t4fn6"><sup>f</sup><p class="last">Murine leukemia virus pseudoparticle assay.</p></div><div class="footnote" id="t4fn7"><sup>g</sup><p class="last">Vesicular stomatitis virus pseudoparticle assay.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> In Vitro Toxicity Profiling against a Panel of 50 CNS-Relevant Targets</h3><div class="NLM_p">Compound <b>7nn</b> was evaluated in a panel of 50 CNS-relevant receptors and neurotransmitter transporters by the Psychoactive Drug Screening Program (PDSP) at the University of North Carolina, Chapel Hill (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). The results show that <b>7nn</b> has submicromolar affinity for only 4 targets across the 50-component panel tested. The highest affinity was for the serotoninergic receptor 5-HT2B (415 nM), which has been implicated in the development of myofibroblast proliferation.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> While any cardiac issue is of concern, the selectivity for <b>7nn</b> against the other targets in the PDSP panel and the modest affinity for the 5-HT2B receptor bode well that this off-target activity could be minimized with further compound development. The receptor profiling of other compounds will be conducted following additional characterization of select lead compounds for further development.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Profile of representative analogue <b>7nn</b> against a panel of 50 CNS-relevant targets. Values are <i>K</i><sub>i</sub> determinations (nM) of radioligand binding in a displacement assay. Primary screen assays are single-point experiments to determine percent radioligand displacement at 10 μM of test compound. The threshold for hit or miss in the primary screen was 50% displacement.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> In Vitro Combination Profiling with Different Classes of Anti-HCV Drugs</h3><div class="NLM_p">We investigated the combination of <b>7</b><b>ii</b> with different classes of anti-HCV drugs. The HCV-Luc and the ATPlite assays were performed in the presence of various concentrations of <b>7</b><b>ii</b> in combination with various concentrations of each drug (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). A synergistic effect was indicated if the combination led to an HCV inhibitory effect greater than that of either of them alone in a concentration-dependent manner without a toxic effect on cell viability. Log volumes of synergy or antagonism were generated according to the Bliss independence model using the MacSynergy II program.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The results were also analyzed with the CalcuSyn program,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> in which the combination indices were calculated from combination of <b>7ii</b> and the tested drug at or near their EC<sub>50</sub> values when tested alone. The hit compound <b>7a</b> was tested following the same protocol. We found that <b>7ii</b> was synergistic with ribavirin, sofosbuvir, telaprevir, daclatasvir, cyclosporin A, and boceprevir without significant cytotoxicity, supporting its potential for use in combination therapy with these drugs. Similar synergistic effects were observed with compound <b>7a</b>, except for nearly additive effect in combination with sofosbuvir. The observed synergistic effects suggest that the aryloxazole analogues likely operate via a distinct mode of action from that of any one of these drugs. The mechanism of action of ribavirin is mediated through host antiviral response. Telaprevir and boceprevir are NS3/4A protease inhibitors and daclatasvir inhibits HCV NS5A. Cyclosporin A targets virus RNA replication. Together, this collection of HCV drugs covers the known modes of action for currently available therapeutics; thus this chemotype, is attractive for further development.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Synergistic Activity of <b>7a</b> and <b>7ii</b> with Selected HCV Therapeutics</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">program</th><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center">ribavirin</th><th class="colsep0 rowsep0" align="center">sofosbuvir</th><th class="colsep0 rowsep0" align="center">telaprevir</th><th class="colsep0 rowsep0" align="center">daclatasvir</th><th class="colsep0 rowsep0" align="center">cyclosporin A</th><th class="colsep0 rowsep0" align="center">boceprevir</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left"><b>7a</b></td><td class="colsep0 rowsep0" rowspan="2" align="left">CalcuSyn</td><td class="colsep0 rowsep0" align="left">CI value<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.770 ± 0.189</td><td class="colsep0 rowsep0" align="left">1.13 ± 0.12</td><td class="colsep0 rowsep0" align="left">0.775 ± 0.108</td><td class="colsep0 rowsep0" align="left">0.510 ± 0.082</td><td class="colsep0 rowsep0" align="left">0.625 ± 0.136</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">synergy volume<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">±</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MacSynergy</td><td class="colsep0 rowsep0" align="left">synergy volume<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left"><b>7ii</b></td><td class="colsep0 rowsep0" rowspan="2" align="left">CalcuSyn</td><td class="colsep0 rowsep0" align="left">CI value<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.745 ± 0.084</td><td class="colsep0 rowsep0" align="left">0.759 ± 0.076</td><td class="colsep0 rowsep0" align="left">0.740 ± 0.102</td><td class="colsep0 rowsep0" align="left">0.831 ± 0.133</td><td class="colsep0 rowsep0" align="left">0.867 ± 0.115</td><td class="colsep0 rowsep0" align="left">0.782 ± 0.104</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">synergy volume<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">++</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MacSynergy</td><td class="colsep0 rowsep0" align="left">synergy volume<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">++</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Values are mean ± SEM of combination indices (CI) obtained from combinations of the tested drug with <b>7a</b> or <b>7ii</b> at or near their EC<sub>50</sub> values when tested alone (<i>n</i> ≥ 6).</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">The level of synergy is defined as the following: ± means nearly additive (0.9 ≤ CI < 1.1), + means minor synergy (0.8 ≤ CI < 0.9), ++ means moderate synergy (0.7 ≤ CI < 0.8), and +++ means strong synergy (CI < 0.7).</p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">The levels of synergy are defined as the following: ± means nearly additive (0 ≤ log volume <2), + means minor synergy (2 ≤ log volume <5), ++ means moderate synergy (5 ≤ log volume <9), and +++ means strong synergy (log volume >9).</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> In Vivo Pharmacokinetic Studies in Mice at Multiple Doses</h3><div class="NLM_p">We evaluated the in vivo pharmacokinetics and tissue distribution of <b>7ii</b> in a mouse model after a single dose of 10 and 1 mg/kg through the intraperitoneal (i.p.) route of administration (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). Excellent liver distribution was observed at both doses, as shown in the liver/plasma AUC<sub>0–24h</sub> ratio of 41 and 224, respectively. Slow clearance in plasma was observed at both doses, leading to half-lives longer than 24 h. When dosed at 1 mg/kg, the liver concentration of <b>7ii</b> throughout 24 h post-administration (3.33–9.82 μM) was more than 100-fold of its in vitro EC<sub>50</sub> values (0.013 μM). In light of the above results, we further evaluated the pharmacokinetics properties of <b>7ii</b> at 0.1 mg/kg and elongated the study time to 168 h (7 days). <b>7ii</b> retained excellent liver distribution at this dose during the first 24 h post-administration and during the total 168 h study (liver/plasma AUC<sub>0–24h</sub> and AUC<sub>0–168h</sub> = 399 and 448, respectively). A reasonably long half-life was observed in liver (<i>t</i><sub>1/2</sub> = 77 h) as well as in plasma (<i>t</i><sub>1/2</sub> = 26 h). At a dose as low as 0.1 mg/kg, the concentration of <b>7ii</b> in liver reached 0.39 μM in the first 5 min and remained at least 7-fold above its EC<sub>50</sub> values (0.013 μM) throughout the 168 h. Alanine aminotransaminase (ALT) level in the mouse serum was monitored to detect any potential hepatotoxicity effect (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Regardless of the dose, the ALT levels were around or below 80 U/L at most time points with a few exceptions of higher ALT at random time points from individual mice. These elevations are unlikely due to the effect of the compound because similar random elevations were noted in mice treated with vehicle only (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Moreover, there was not an obvious correlation between the ALT levels and the liver concentration of compound <b>7ii</b>. Overall, the representative analogue <b>7ii</b> exhibited high liver distribution and long half-life without obvious hepatotoxicity, indicated by ALT level in the mouse model.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pharmacokinetics studies of compound <b>7ii</b> in the mouse model. (A) Mean liver (green triangle) and plasma (red dot) concentration–time profiles of compound <b>7ii</b> after administration of a single i.p. dose at 10, 1, and 0.1 mg/kg at indicated time points. Compound concentration of <b>7ii</b> was measured by UPLC-MS/MS methods and is shown in means ± SEM (<i>n</i> = 3 per time point). (B) Pharmacokinetic parameters of compound <b>7ii</b>. AUC<sub>last</sub> = AUC<sub>0–24h</sub> or AUC<sub>0–168h</sub> depending on the sample collection interval. N.D., not determined. (C) ALT levels of the mouse serum samples collected during the pharmacokinetics study. Results for each mouse are shown with scatter plots, and error bars show means ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> In Vitro Antiviral Specificity Profiling against a Panel of 13 Viruses</h3><div class="NLM_p last">To assess whether this chemotype exhibits nonspecific antiviral effects against viruses other than HCV, we carried out an antiviral screen with representative lead compound <b>7nn</b> against 13 viruses utilizing the nonclinical and preclinical services program offered by the National Institute of Allergy and Infectious Diseases (<a href="http://www.niaid.nih.gov/labsandresources/resources/dmid/invitro/Pages/invitro.aspx" class="extLink">https://www.niaid.nih.gov/research/vitro-assessment-antimicrobial-activity-resources-niaid</a>). The 13 types of viruses are hepatitis B virus, HCV replicon, herpes simplex virus-1, human cytomegalovirus, vaccinia virus, dengue virus, influenza A (H1N1) virus, respiratory syncytial virus, SARS coronavirus, poliovirus 3, Rift Valley fever virus, Tacaribe virus, and Venezuelan equine encephalitis virus. As shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_001.pdf" class="ext-link">Supporting Information</a>, compound <b>7nn</b> had little or no activity (selective index ≤10 and/or EC<sub>50</sub> > 2 μM) against all the above viruses. These results suggest that this series of compounds is selectively active against HCV infection.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52361" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52361" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We presented the development of a new class of HCV inhibitors. The SAR study generated a number of potent analogues with low cytotoxicity and a promising preliminary PK profile in mice, the species potentially used for the efficacy evaluation. The compound class appears to act via a mode of action distinct from that of HCV inhibitors currently approved for HCV therapy. Such HCV inhibitors with a novel mechanism targeting entry could offer a lower probability of developing resistant virus strains during treatment as well as provide an additional weapon against nonresponsive cases.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The attractiveness of HCV entry as an anti-HCV target is evidenced by multiple recent efforts in developing HCV entry inhibitors.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18-20)</a> To attain an all oral, pangenotypic HCV treatment with the shortest possible course of treatment, it would be of benefit to target multiple viral or host targets simultaneously. Studies toward determining the molecular target and validating the efficacy in vivo are underway and will be reported in due course.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84520" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Synthesis and Analysis Experimental Details</h3><div class="NLM_p last">All reagents were used as received from the following suppliers: Alfa Aesar, Ark Pharm, Aldrich, and Fisher Scientific. Acetonitrile and THF were purified using the Innovative Technology PureSolv solvent purification system. The <sup>1</sup>H and <sup>13</sup>C spectra were recorded on a Bruker Avance 400 or 500 MHz spectrometer. Chemical shifts are reported in parts per million and were referenced to residual proton solvent signals. <sup>13</sup>C multiplicities were determined with the aid of an APT pulse sequence, differentiating the signals for methyl (CH<sub>3</sub>) and methyne (CH) carbons as “d” from methylene (CH<sub>2</sub>) and quaternary (C) carbons as “u”. The infrared (IR) spectra were acquired as thin films using a universal ATR sampling accessory on a Thermo Scientific Nicolet iS5 FT-IR spectrometer, and the absorption frequencies are reported in cm<sup>–1</sup>. Microwave syntheses were conducted in a Biotage Initiator constant temperature microwave synthesizer. Flash column chromatography separations were performed using the Teledyne Isco CombiFlash <i>R</i><sub>F</sub> using RediSep <i>R</i><sub>F</sub> silica gel columns. TLC was performed on Analtech UNIPLATE silica gel GHLF plates (gypsum inorganic hard layer with fluorescence). TLC plates were developed using iodine vapor. Automated preparative RP HPLC purification was performed using an Agilent 1200 mass-directed fractionation system (Prep Pump G1361 with gradient extension, makeup pump G1311A, pH modification pump G1311A, HTS PAL autosampler, UV-DAD detection G1315D, fraction collector G1364B, and Agilent 6120 quadrapole spectrometer G6120A). The preparative chromatography conditions included a Waters X-Bridge C<sub>18</sub> column (19 × 150 mm, 5 μm, with 19 × 10 mm guard column), elution with a water and acetonitrile gradient, which increases 20% in acetonitrile content over 4 min at a flow rate of 20 mL/min (modified to pH 9.8 through addition of NH<sub>4</sub>OH by auxiliary pump), and sample dilution in DMSO. The preparative gradient, triggering thresholds, and UV wavelength were selected according to the analytical RP HPLC analysis of each crude sample. The analytical method used an Agilent 1200 RRLC system with UV detection (Agilent 1200 DAD SL) and mass detection (Agilent 6224 TOF). The analytical method conditions included a Waters Aquity BEH C<sub>18</sub> column (2.1 × 50 mm, 1.7 μm) and elution with a linear gradient of 5% acetonitrile in pH 9.8 buffered aqueous ammonium formate to 100% acetonitrile at 0.4 mL/min flow rate. Compound purity was measured on the basis of peak integration (area under the curve) from UV–vis absorbance (at 214 nm), and compound identity was determined on the basis of mass analysis. Compounds used for assays or biological studies have HPLC purity >95% with the exception of <b>7yy</b> (purity = 94.2%). The analytical HPLC system used is a dedicated instrument for assessing compound purity and routinely detects impurities as low as 0.1% that elute within the detection window. Any compounds with a measured purity of 100% were thus conservatively assigned a purity of >99.8%. Any compounds purified by reverse-phase, preparative HPLC utilized the same solvent gradient and column material as the analytical conditions to minimize the possibility of impurities that were not detected in the analytical method. All final compounds were inspected for functional groups known to contribute PAINS liabilities, and none were found.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General Procedure A-1: Coupling of Carboxylic Acid Fragment <b>5</b> or <b>12</b> and Amine Fragment <b>6</b></h3><div class="NLM_p last">To a mixture of oxazolecarboxylic acid <b>5</b> or thiazolecarboxylic acid <b>12</b>, amine <b>6</b> (1.0–2.0 equiv), and HOBt (1.0 equiv) in MeCN (1.5 to 4 mL, ca. 0.1 M) was added diisopropyl carbodiimide (2.0 equiv). The microwave vial was capped and irradiated at 100 °C for 10 min. After cooling to rt, the solvent was removed, and the residue purified by silica gel chromatography (eluents 0–20% MeOH + 0–2% NH<sub>4</sub>OH<sub>(aq)</sub> in CH<sub>2</sub>Cl<sub>2</sub>) to afford the coupled product.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> General Procedure A-2: Coupling of Oxazolecarboxylic Acid <b>5</b> and Amine Fragment <b>6</b></h3><div class="NLM_p last">To a mixture of oxazolecarboxylic acid <b>5</b>, amine <b>6</b> (1.0–2.0 equiv), and HOBt (1.0 equiv) in MeCN (1.5 to 4 mL, ca. 0.1 M) was added diisopropyl carbodiimide (2.0 equiv). The microwave vial was capped and irradiated at 100 °C for 10 min. After cooling to rt, the solvent was removed, and the residue was purified by C-18 functionalized silica chromatography (eluents 5–100% MeCN in deionized water with 0.5% NH<sub>4</sub>OH<sub>(aq)</sub>) to afford the coupled product <b>7</b>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedure A-3: Coupling of Oxazolecarboxylic Acid <b>5</b> or <b>17</b> and Amine Fragment <b>6</b></h3><div class="NLM_p last">To a mixture of oxazolecarboxylic acid <b>5</b> or <b>17</b>, amine <b>6</b> (1.0–2.0 equiv), DMAP (0.1 equiv), and HOBt (1.0 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (1.5 to 4 mL, ca. 0.05 M) was added diisopropyl carbodiimide (5.0 equiv). The reaction was stirred at rt for 16 h; the solvent was removed, and the residue was purified by silica gel chromatography (eluents 0–20% MeOH + 0–2% NH<sub>4</sub>OH<sub>(aq)</sub> in CH<sub>2</sub>Cl<sub>2</sub>) to afford the coupled product <b>7</b>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Procedure A-4: Coupling of Oxazolecarboxylic Acid <b>5</b> and Amine Fragment <b>6</b></h3><div class="NLM_p last">To a mixture of oxazolecarboxylic acid <b>5</b>, amine <b>6</b> (1.0–2.0 equiv), DMAP (0.1 equiv), and HOBt (1.0 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (1.5 to 4 mL, ca. 0.05 M) was added diisopropyl carbodiimide (5.0 equiv). The reaction was stirred at rt for 16 h; the solvent was removed, and the residue was purified by automated preparative RP HPLC purification as described in the general <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> to afford the coupled product <b>7</b>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Synthesis of Aryloxazole Final Analogues</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7a</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (41 mg, 0.13 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (22 mg, 0.13 mmol) were reacted according to general procedure A-1 to afford the product as a white solid (46 mg, 0.098 mmol, 75% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.53 (q, <i>J</i> = 12.3 Hz, 1H), 0.86 (d, <i>J</i> = 6.6 Hz, 6H), 1.39 (t, <i>J</i> = 11.0 Hz, 2H), 1.58–1.85 (complex, 9H), 1.97–2.14 (m, 3H), 2.39 (s, 3H), 2.42 (t, <i>J</i> = 6.2 Hz, 2H), 2.65 (s, 3H), 2.85–2.89 (m, 2H), 3.01–3.05 (m, 2H), 3.33 (q, <i>J</i> = 6.1 Hz, 2H), 3.48 (s, 2H), 7.22–7.32 (m, 3H), 7.91–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 21.8, 31.4, 43.5, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.9, 29.1, 40.0, 42.1, 53.1, 53.9, 58.3, 61.8, 126.9, 132.4, 137.0, 145.5, 159.8, 174.8; IR 1541, 1646, 2950 cm<sup>–1</sup>; HRMS calcd for C<sub>28</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 467.3381; found 467.3373; HPLC purity: >99.8%.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (±)-<i>N</i>-(3-(<i>trans</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7b</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (64 mg, 0.20 mmol) and 3-(<i>trans</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (42 mg, 0.24 mmol) were reacted according to general procedure A-2 to afford the product as a light yellow solid (81 mg, 0.17 mmol, 85% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.96 (d, <i>J</i> = 6.8 Hz, 6H), 1.30 (t, <i>J</i> = 5.5 Hz, 2H), 1.56–1.71 (m, 2H), 1.76–1.92 (complex, 6H), 1.99–2.11 (complex, 5H), 2.30–2.36 (m, 4H), 2.38 (s, 3H), 2.65 (s, 3H), 3.01–3.06 (m, 2H), 3.19–3.27 (m, 1H), 3.38–3.45 (m, 1H), 3.47 (s, 2H), 7.13 (t, <i>J</i> = 4.1 Hz, 1H), 7.21–7.29 (m, 2H), 7.91–7.93 (m, 1H); <sup>13</sup>C NMR ((101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.3, 21.8, 27.4, 43.6, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.8, 28.9, 29.1, 38.9, 39.8, 53.1, 53.9, 58.4, 61.4, 126.9, 132.3, 137.0, 145.6, 159.7, 174.8; IR 1546, 1645, 1710, 2930 cm<sup>–1</sup>; HRMS calcd for C<sub>28</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 467.3386; found 467.3376; HPLC purity: 98.5%.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-(3-(<i>cis</i>-3,5-Diethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7c</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.08 mmol) and 3-(<i>cis</i>-3,5-diethylpiperidin-1-yl)propan-1-amine (17 mg, 0.09 mmol) were reacted according to general procedure A-3 to afford the product as a light yellow, viscous oil (14 mg, 0.028 mmol, 35% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.91 (t, <i>J</i> = 7.4 Hz, 6H), 1.18–1.29 (m, 4H), 1.41–1.48 (m, 4H), 1.66–1.72 (m, 2H),1.79–1.89 (complex, 5H), 2.00–2.09 (m, 2H), 2.12 (dt, <i>J</i> = 3.4, 11.2 Hz, 2H), 2.41 (s, 3H), 2.44 (t, <i>J</i> = 6.2 Hz, 2H), 2.67 (s, 3H), 2.96 (d, <i>J</i> = 6.8 Hz, 2H), 3.06 (d, <i>J</i> = 11.7 Hz, 2H), 3.36 (q, <i>J</i> = 6.2 Hz, 2H), 3.51 (s, 2H), 7.24–7.34 (m, 3H), 7.93–7.96 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 11.5, 21.8, 38.1, 43.5, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.9, 27.4, 29.1, 37.4, 40.0, 53.1, 53.9, 58.5, 60.4, 126.9, 132.3, 137.0, 145.6, 159.8, 174.9; HRMS calcd for C<sub>30</sub>H<sub>47</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 495.3694; found 495.3693; HPLC purity: >99.8%.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)-<i>N</i>-(3-(<i>cis</i>-3,3,5,5-tetramethylpiperidin-1-yl)propyl)piperidine-4-carboxamide <b>7d</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.08 mmol) and 3-(3,3,5,5-tetramethylpiperidin-1-yl)propan-1-amine (17 mg, 0.09 mmol) were reacted according to general procedure A-3 to afford the product as a light yellow, viscous oil (23 mg, 0.046 mmol, 58% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.03 (s, 12H), 1.21 (s, 2H), 1.74–1.81 (m, 2H), 1.90–1.95 (complex, 4H), 2.17–2.27 (m, 3H), 2.41 (s, 3H), 2.50–2.57 (m, 2H), 2.63 (s, 4H), 2.66 (s, 3H), 3.21–3.27 (m, 2H), 3.34 (q, <i>J</i> = 6.2 Hz, 2H), 3.69 (s, 2H), 7.24–7.34 (m, 3H), 7.91–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 21.9, 29.5, 41.0, 125.9, 128.6, 129.7, 131.5; u (C, CH<sub>2</sub>): 24.9, 27.9, 31.4, 38.2, 50.1, 52.1, 52.8, 56.8, 66.1, 126.4, 129.6, 137.1, 147.3, 160.0, 175.1; IR 1557, 1670, 2957 cm<sup>–1</sup>; HRMS calcd for C<sub>30</sub>H<sub>47</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 495.3694; found 495.3692; HPLC purity: >99.8%.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-(3-(4-Isopropylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7e</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (65 mg, 0.21 mmol) and 3-(4-isopropylpiperidin-1-yl)propan-1-amine (46 mg, 0.25 mmol) were reacted according to general procedure A-2 to afford the product as a colorless, viscous oil (55 mg, 0.11 mmol, 55% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.83 (d, <i>J</i> = 6.8 Hz, 6H), 1.08–0.99 (m, 1H), 1.33–1.15 (m, 2H), 1.40 (dq, <i>J</i> = 13.3, 6.4 Hz, 1H), 1.67–1.61 (m, 4H), 1.91–1.73 (m, 6H), 2.14–1.96 (m, 3H), 2.38 (s, 3H), 2.45–2.39 (m, 2H), 2.65 (s, 3H), 3.08–2.92 (m, 4H), 3.34–3.30 (m, 2H), 3.47 (s, 2H), 7.33–7.17 (m, 3H), 7.59 (t, <i>J</i> = 4.6 Hz, 1H), 7.92 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 21.8, 32.4, 42.2, 43.4, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.7, 29.1, 29.4, 40.2, 53.1, 53.8, 54.5, 58.5, 126.8, 132.3, 137.0, 145.6, 159.7, 174.8; IR 1541, 1641, 2938 cm<sup>–1</sup>; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3543; found 481.3537; HPLC purity: >99.8%.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(3-(4,4-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7f</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (45 mg, 0.14 mmol) and 3-(4,4-dimethylpiperidin-1-yl)propan-1-amine (24 mg, 0.14 mmol) were reacted according to general procedure A-2 to afford the product as a light yellow, viscous oil (47 mg, 0.10 mmol, 70% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.90 (s, 6H), 1.23–1.28 (m, 2H), 1.39 (t, <i>J</i> = 5.6 Hz, 4H), 1.61–1.68 (m, 2H), 1.75–1.90 (complex, 4H), 2.00–2.13 (complex, 3H), 2.38 (s, 3H), 2.39–2.46 (m, 2H), 2.45 (t, <i>J</i> = 6.2 Hz, 2H), 2.64 (s, 3H), 3.05 (d, <i>J</i> = 11.6 Hz, 2H), 3.30–3.35 (m, 2H), 3.48 (s, 2H), 7.21–7.31 (m, 3H), 7.56 (br s, 1H), 7.90–7.93 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 21.8, 43.4, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.7, 28.5, 29.1, 38.9, 40.1, 50.3, 53.1, 53.8, 58.4, 126.9, 132.3, 137.0, 145.6, 159.7, 174.8; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>28</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 467.3381; found 467.3389; HPLC purity: >99.8%.</div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-(3-(4-Methylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7g</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (51 mg, 0.16 mmol) and 3-(4-methylpiperidin-1-yl)propan-1-amine (25 mg, 0.16 mmol) were reacted according to general procedure A-4 to afford the product as a colorless, viscous oil (16 mg, 0.034 mmol, 21% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.90 (d, <i>J</i> = 6.4 Hz, 3H), 1.18 (dq, <i>J</i> = 3.2, 12.2 Hz, 2H), 1.31–1.43 (m, 1H), 1.62–1.68 (complex, 4H), 1.72–1.92 (complex, 6H), 1.99–2.13 (complex, 3H), 2.38 (s, 3H), 2.42 (t, <i>J</i> = 6.2 Hz, 2H), 2.65 (s, 3H), 2.91 (d, <i>J</i> = 11.6 Hz, 2H), 3.04 (d, <i>J</i> = 11.6 Hz, 2H), 3.33 (q, <i>J</i> = 5.5 Hz, 2H), 3.48 (s, 2H), 7.21–7.31 (m, 3H), 7.49 (br s, 1H), 7.90–7.93 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 21.8, 22.0, 30.7, 43.4, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.8, 29.1, 34.5, 40.0, 53.1, 53.8, 54.1, 58.4, 126.9, 132.3, 137.0, 145.6, 159.8, 174.9; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>27</sub>H<sub>41</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 453.3224; found 453.3236; HPLC purity: >99.8%.</div></div><div id="sec4_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-(3-(Piperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7h</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (51 mg, 0.16 mmol) and 3-(piperidin-1-yl)propan-1-amine (23 mg, 0.16 mmol) were reacted according to general procedure A-4 to afford the product as a colorless, viscous oil (19 mg, 0.044 mmol, 27% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.41–1.49 (m, 2H), 1.55–1.68 (complex, 6H), 1.73–1.88 (complex, 5H), 2.00–2.14 (complex, 3H), 2.39 (s, 3H), 2.39–2.42 (complex, 5H), 2.65 (s, 3H), 3.04 (d, <i>J</i> = 11.6 Hz, 2H), 3.33 (q, <i>J</i> = 6.0 Hz, 2H), 3.48 (s, 2H), 7.22–7.31 (m, 3H), 7.40 (br s, 1H), 7.90–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 21.8, 43.5, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.3, 24.7, 26.1, 29.1, 40.0, 53.1, 53.9, 54.7, 58.8, 126.9, 132.4, 137.0, 138.7, 145.6, 174.9; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>26</sub>H<sub>39</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 439.3068; found 439.3083; HPLC purity: >99.8%.</div></div><div id="sec4_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-(3-(<i>cis</i>-2,6-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7i</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (78 mg, 0.25 mmol) and 3-(<i>cis</i>-2,6-dimethylpiperidin-1-yl)propan-1-amine (51 mg, 0.30 mmol) were reacted according to general procedure A-2 to afford the product as a white solid (81 mg, 0.17 mmol, 70% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.53 (q, <i>J</i> = 12.3 Hz, 1H), 0.86 (d, <i>J</i> = 6.6 Hz, 6H), 1.39 (t, <i>J</i> = 11.0 Hz, 2H), 1.58–1.85 (complex, 9H), 1.97–2.14 (m, 3H), 2.39 (s, 3H), 2.42 (t, <i>J</i> = 6.2 Hz, 2H), 2.65 (s, 3H), 2.85–2.89 (m, 2H), 3.01–3.05 (m, 2H), 3.33 (q, <i>J</i> = 6.1 Hz, 2H), 3.48 (s, 2H), 7.22–7.32 (m, 3H), 7.91–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 21.8, 31.4, 43.5, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.9, 29.1, 40.0, 42.1, 53.1, 53.9, 58.3, 61.8, 126.9, 132.4, 137.0, 145.5, 159.8, 174.8; HRMS calcd for C<sub>28</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 467.3386; found 467.3377; HPLC purity: 98.3%.</div></div><div id="sec4_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (±)-<i>N</i>-(3-(3-Methylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7j</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (70 mg, 0.22 mmol) and 3-(3-methylpiperidin-1-yl)propan-1-amine (42 mg, 0.67 mmol) were reacted according to general procedure A-2 to afford the product as a white solid (81 mg, 0.18 mmol, 80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.89 (d, <i>J</i> = 6.5 Hz, 3H), 1.50–1.88 (complex, 13H), 2.01–2.15 (m, 3H), 2.40 (s, 3H), 2.43 (t, <i>J</i> = 6.2 Hz, 2H), 2.66 (s, 3H), 2.84–2.94 (m, 2H), 3.05 (d, <i>J</i> = 11.6 Hz, 2H), 3.35 (q, <i>J</i> = 6.0 Hz, 2H), 3.50 (s, 2H), 7.23–7.33 (m, 3H), 7.37 (br s, 1H), 7.92–7.95 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 19.7, 21.8, 31.3, 43.4, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.7, 25.6, 29.1, 32.9, 39.9, 53.1, 53.9, 54.1, 58.4, 62.2, 126.9, 132.4, 137.0, 145.5, 159.7, 174.9; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>27</sub>H<sub>41</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 453.3224; found 453.3227; HPLC purity: >99.8%.</div></div><div id="sec4_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (±)-<i>N</i>-(3-(3-Isopropylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7k</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.080 mmol) and 3-(3-isopropylpiperidin-1-yl)propan-1-amine (16 mg, 0.087 mmol) were reacted according to general procedure A-3 to afford the product as a light yellow, viscous oil (33 mg, 0.068 mmol, 85% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.88 (dd, <i>J</i> = 2.5, 6.8 Hz, 6H), 0.95 (dq, <i>J</i> = 3.7, 11.9 Hz, 1H), 1.24–1.34 (m, 2H), 1.40–1.56 (m, 2H), 1.60–1.86 (complex, 9H), 2.00–2.14 (m, 3H), 2.40 (s, 3H), 2.43 (t, <i>J</i> = 6.2 Hz, 2H), 2.66 (s, 3H), 2.88–2.94 (m, 2H), 3.05 (d, <i>J</i> = 11.6 Hz, 2H), 3.31–3.37 (m, 2H), 3.49 (s, 2H), 7.23–7.32 (m, 3H), 7.41 (br s, 1H), 7.92–7.95 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.8, 20.2, 21.8, 31.1, 42.8, 43.5, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.8, 25.8, 27.7, 29.0, 39.9, 53.1, 53.8, 54.4, 58.4, 58.6, 126.9, 132.3, 137.0, 145.6, 159.7, 174.9; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3543; found 481.3539; HPLC purity: 99.3%.</div></div><div id="sec4_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (±)-<i>N</i>-(3-(3-Isobutylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7l</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (67 mg, 0.21 mmol) and 3-(3-isobutylpiperidin-1-yl)propan-1-amine (42 mg, 0.21 mmol) were reacted according to general procedure A-4 to afford the product as a colorless, viscous oil (22 mg, 0.044 mmol, 21% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.88 (dd, <i>J</i> = 2.5, 6.8 Hz, 6H), 0.95 (dq, <i>J</i> = 3.7, 11.9 Hz, 1H), 1.24–1.34 (m, 2H), 1.40–1.56 (m, 2H), 1.60–1.86 (complex, 9H), 2.00–2.14 (m, 3H), 2.40 (s, 3H), 2.43(t, <i>J</i> = 6.2 Hz, 2H), 2.66 (s, 3H), 2.88–2.94 (m, 2H), 3.05 (d, <i>J</i> = 11.6 Hz, 2H), 3.31–3.37 (m, 2H), 3.49 (s, 2H), 7.23–7.32 (m, 3H), 7.41 (br s, 1H), 7.92–7.95 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.8, 20.2, 21.8, 31.1, 42.8, 43.5, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.8, 25.8, 27.7, 29.0, 39.9, 53.1, 53.8, 54.4, 58.4, 58.6, 126.9, 132.3, 137.0, 145.6, 159.7, 174.9; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>30</sub>H<sub>47</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 495.3694; found 495.3695; HPLC purity: >99.8%.</div></div><div id="sec4_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-(3-(3-Azabicyclo[3.3.1]nonan-3-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7m</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (62 mg, 0.20 mmol) and 3-(3-azabicyclo[3.3.1]nonan-3-yl)propan-1-amine (55 mg, 0.30 mmol) were reacted according to general procedure A-4 to afford the product as a colorless, viscous oil (21 mg, 0.044 mmol, 22% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.46–1.55 (m, 3H), 1.60–1.86 (complex, 11H), 2.02–2.15 (complex, 5H), 2.22 (t, <i>J</i> = 6.4 Hz, 2H), 2.31–2.44 (m, 2H), 2.38 (s, 3H), 2.64 (s, 3H), 2.90 (d, <i>J</i> = 10.7 Hz, 2H), 3.05 (d, <i>J</i> = 11.8 Hz, 2H), 3.31 (q, <i>J</i> = 5.6 Hz, 2H), 3.48 (s, 2H), 6.06 (br s, 1H), 7.22–7.31 (m, 3H), 7.91–7.93 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 21.8, 29.5, 43.4, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 22.6, 25.8, 29.0, 31.4, 34.1, 38.4, 53.0, 53.8, 57.6, 60.1, 126.9, 132.2, 137.0, 145.6, 159.8, 175.0; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>29</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 479.3381; found 479.3379; HPLC purity: 99.8%.</div></div><div id="sec4_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)ethyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7n</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (50 mg, 0.16 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)ethan-1-amine (30 mg, 0.19 mmol) were reacted according to general procedure A-4 to afford the product as a white solid (59 mg, 0.13 mmol, 82% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.53 (q, <i>J</i> = 11.9 Hz, 1H), 0.82–0.87 (m, 1H), 0.85 (d, <i>J</i> = 6.5 Hz, 6H), 1.46 (t, <i>J</i> = 11.0 Hz, 2H), 1.58–1.88 (complex, 6H), 2.06–2.15 (m, 3H), 2.38 (s, 3H), 2.42 (t, <i>J</i> = 5.9 Hz, 2H), 2.65 (s, 3H), 2.72–2.82 (m, 2H), 3.01–3.05 (m, 2H), 3.32 (q, <i>J</i> = 5.5 Hz, 2H), 3.47 (s, 2H), 6.33 (br s, 1H), 7.22–7.32 (m, 3H), 7.91–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.5, 21.8, 31.1, 43.3, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 28.9, 35.9, 42.1, 53.0, 53.7, 56.6, 61.2, 126.9, 132.2, 137.0, 145.6, 159.8, 175.0; HRMS calcd for C<sub>27</sub>H<sub>41</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 453.3224; found 453.3221; HPLC purity: 96.6%.</div></div><div id="sec4_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)butyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7o</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (51 mg, 0.16 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)butan-1-amine (36 mg, 0.20 mmol) were reacted according to general procedure A-4 to afford the product as a white solid (24 mg, 0.050 mmol, 31% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.52 (q, <i>J</i> = 11.5 Hz, 1H), 0.85 (d, <i>J</i> = 6.5 Hz, 6H), 1.38 (t, <i>J</i> = 11.1 Hz, 2H), 1.49–1.56 (complex, 4H), 1.60–1.72 (m, 3H), 1.75–1.85 (m, 3H), 2.01–2.12 (complex, 4H), 2.29 (t, <i>J</i> = 6.9 Hz, 2H), 2.38 (s, 3H), 2.65 (s, 3H), 2.78–2.83 (m, 2H), 3.04 (d, <i>J</i> = 11.8 Hz, 2H), 3.21–3.26 (m, 2H), 3.47 (s, 2H), 6.22 (br s, 1H), 7.21–7.32 (m, 3H), 7.89–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.7, 21.8, 31.1, 43.5, 125.8, 128.7, 129.5, 131.4; u (C, CH<sub>2</sub>): 24.5, 27.5, 29.0, 39.1, 42.2, 53.0, 53.7, 58.2, 61.6, 126.8, 132.2, 137.0, 145.7, 159.8, 174.9; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3537; found 481.3537; HPLC purity: >99.8%.</div></div><div id="sec4_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(2-(Dipropylamino)ethyl)-1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7p</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (50 mg, 0.16 mmol) and <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-dipropylethane-1,2-diamine (25 mg, 0.18 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (20 mg, 0.046 mmol, 29% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.86–0.95 (complex, 6H), 1.44–1.71 (complex, 7H), 1.95 (dq, <i>J</i> = 3.0, 12.4 Hz, 2H), 2.09–2.18 (m, 4H), 2.41 (s, 3H), 2.57–2.63 (m, 2H), 2.67 (s, 3H), 2.77 (q, <i>J</i> = 7.0 Hz, 2H), 3.03–3.10 (m, 2H), 3.02–3.12 (m, 2H), 3.38–3.46 (m, 2H), 3.53 (s, 2H), 7.24–7.33 (m, 3H), 7.93–7.97 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 11.2, 11.7, 21.8, 39.0, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 21.0, 22.9, 23.2, 29.0, 46.2, 47.7, 48.7, 50.0, 51.8, 53.0, 53.9, 126.9, 132.3, 137.0, 145.6, 159.7, 175.4; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>26</sub>H<sub>41</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 441.3230; found 441.3226; HPLC purity: >99.8%.</div></div><div id="sec4_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)-<i>N</i>-(3-morpholinopropyl)piperidine-4-carboxamide <b>7q</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (47 mg, 0.15 mmol) and 3-morpholinopropan-1-amine (43 mg, 0.30 mmol) were reacted according to general procedure A-4 to afford the product as a viscous, light yellow oil (26 mg, 0.060 mmol, 40% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.86–0.95 (complex, 6H), 1.64–1.70 (m, 2H), 1.72–1.88 (complex, 4H),1.97–2.14 (m, 3H), 2.39 (s, 3H), 2.42–2.50 (complex, 6H), 2.65 (s, 3H), 3.05 (d, <i>J</i> = 11.6 Hz, 2H), 3.34 (q, <i>J</i> = 5.5 Hz, 2H), 3.48 (s, 2H), 3.71 (t, <i>J</i> = 4.6 Hz, 4H), 6.80 (br s, 1H), 7.22–7.32 (m, 3H), 7.90–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 21.8, 43.5, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 24.9, 29.1, 39.4, 53.1, 53.78, 53.83, 58.1, 67.0, 126.9, 132.3, 137.0, 145.6, 159.8, 174.9; IR 1547, 1642, 2812, 2940 cm<sup>–1</sup>; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>25</sub>H<sub>37</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> 441.2860; found 441.2879; HPLC purity: >99.8%.</div></div><div id="sec4_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidin-4-yl)(4-(1-methylpiperidin-4-yl)piperazin-1-yl)methanone <b>7r</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (50 mg, 0.16 mmol) and 1-(1-methylpiperidin-4-yl)piperazine (29 mg, 0.16 mmol) were reacted according to general procedure A-4 to afford the product as an off-white solid (36 mg, 0.075 mmol, 47% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.57 (dq, <i>J</i> = 3.4, 12.0 Hz, 2H), 1.71 (ABq, Δδ<sub>AB</sub> = 0.08, <i>J</i> = 12.2 Hz, 2H), 1.85–1.96 (complex, 4H), 2.13 (dt, <i>J</i> = 2.0, 11.6 Hz, 2H), 2.21–2.29 (m, 1H), 2.26 (s, 3H), 2.39 (s, 3H), 2.40–2.47 (m, 1H), 2.48–2.56 (m, 4H), 2.65 (s, 3H), 2.90 (d, <i>J</i> = 11.8 Hz, 2H), 3.05 (d, <i>J</i> = 11.6 Hz, 2H), 3.45–3.51 (m, 2H), 3.51 (s, 2H), 3.58–3.64 (m, 2H), 7.22–7.32 (m, 3H), 7.91–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 21.9, 38.5, 46.1, 61.6, 125.8, 128.7, 129.4, 131.4; u (C, CH<sub>2</sub>): 28.1, 28.7, 42.0, 45.8, 49.1, 49.4, 52.9, 53.8, 55.4, 126.9, 132.2, 137.0, 145.6, 159.7, 173.3; IR 1446, 1625, 2807, 2938 cm<sup>–1</sup>; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>28</sub>H<sub>42</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 480.3333; found 480.3349; HPLC purity: >99.8%.</div></div><div id="sec4_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (4-(2-(Dimethylamino)ethyl)piperazin-1-yl)(1-((5-methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidin-4-yl)methanone <b>7s</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (55 mg, 0.18 mmol) and <i>N</i>,<i>N</i>-dimethyl-2-(piperazin-1-yl)ethanamine (28 mg, 0.18 mmol) were reacted according to general procedure A-4 to afford the product as an off-white solid (30 mg, 0.067 mmol, 38% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.64–1.71 (m, 2H), 1.90 (dq, <i>J</i> = 2.2, 11.7 Hz, 2H), 2.13 (dt, <i>J</i> = 2.1, 11.6 Hz, 2H), 2.25 (s, 6H), 2.38 (s, 3H), 2.40–2.50 (complex, 5H), 2.59–2.62 (m, 4H), 2.65 (s, 3H), 3.05 (d, <i>J</i> = 11.6 Hz, 2H), 3.47–3.52 (m, 2H), 3.51 (s, 2H), 3.60–3.65 (m, 2H), 7.22–7.31 (m, 3H), 7.91–7.94 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 21.8, 38.5, 41.0, 45.9, 125.8, 128.6, 129.4, 131.4; u (C, CH<sub>2</sub>): 28.7, 41.5, 45.3, 52.9, 53.3, 53.8, 54.1, 56.6, 56.8, 126.9, 132.2, 137.0, 145.6, 159.7, 173.4; IR 1445, 1622, 2816, 2944 cm<sup>–1</sup>; HRMS (<i>m</i>/<i>z</i>): calcd for C<sub>26</sub>H<sub>40</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> 454.3177; found 454.3192; HPLC purity: 99.3%.</div></div><div id="sec4_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>m</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7t</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>m</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (30 mg, 0.095 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (18 mg, 0.10 mmol) were reacted according to general procedure A-3 to afford the product as a white solid (29 mg, 0.062 mmol, 65% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.54 (q, <i>J</i> = 11.9 Hz, 1H), 0.87 (d, <i>J</i> = 6.5 Hz, 6H), 1.41 (t, <i>J</i> = 11.1 Hz, 2H), 1.57–1.88 (complex, 9H), 1.97–2.11 (m, 3H), 2.38 (s, 3H), 2.40 (s, 3H), 2.42 (t, <i>J</i> = 6.1 Hz, 2H), 2.85–2.89 (m, 2H), 2.96–3.05 (m, 2H), 3.33 (q, <i>J</i> = 6.0 Hz, 2H), 3.45 (s, 2H), 7.20–7.23 (m, 1H), 7.28–7.33 (m, 2H), 7.79 (d, <i>J</i> = 7.7 Hz, 1H), 7.86 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 21.3, 31.3, 43.4, 123.1, 126.6, 128.5, 130.6; u (C, CH<sub>2</sub>): 24.9, 29.0, 39.8, 42.1, 53.1, 53.9, 58.1, 61.7, 127.6, 132.6, 138.3, 145.9, 159.5, 174.8; HRMS calcd for C<sub>28</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 467.3381; found 467.3388; HPLC purity: 97.2%.</div></div><div id="sec4_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>p</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7u</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>p</i>-tolyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (28 mg, 0.089 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (17 mg, 0.098 mmol) were reacted according to general procedure A-3 to afford the product as a white solid (28 mg, 0.059 mmol, 66% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.54 (q, <i>J</i> = 11.9 Hz, 1H), 0.87 (d, <i>J</i> = 6.5 Hz, 6H), 1.41 (t, <i>J</i> = 11.1 Hz, 2H), 1.57–1.88 (complex, 9H), 1.97–2.11 (m, 3H), 2.38 (s, 3H), 2.39 (s, 3H), 2.41 (t, <i>J</i> = 6.1 Hz, 2H), 2.85–2.90 (m, 2H), 2.97–3.02 (m, 2H), 3.33 (q, <i>J</i> = 6.1 Hz, 2H), 3.44 (s, 2H), 7.23 (d, <i>J</i> = 8.0 Hz, 2H), 7.31 (br s, 1H), 7.90 (d, <i>J</i> = 8.2 Hz, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 21.5, 31.3, 43.5, 126.0, 129.3; u (C, CH<sub>2</sub>): 24.9, 29.0, 39.9, 42.1, 53.2, 54.0, 58.2, 61.8, 125.1, 132.5, 139.9, 145.6, 159.6, 174.8; HRMS calcd for C<sub>28</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 467.3381; found 467.3389; HPLC purity: 99.0%.</div></div><div id="sec4_6_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-ethylphenyl)- 5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7v</b></h4><div class="NLM_p last">1-((2-(2-Ethylphenyl)- 5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (31 mg, 0.094 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (18 mg, 0.10 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, colorless oil (22 mg, 0.045 mmol, 48% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 11.9 Hz, 1H), 0.87 (d, <i>J</i> = 6.5 Hz, 6H), 1.22 (t, <i>J</i> = 7.5 Hz, 2H), 1.44 (t, <i>J</i> = 11.0 Hz, 2H), 1.59–1.90 (complex, 9H), 1.97–2.18 (m, 3H), 2.40 (s, 3H), 2.44 (t, <i>J</i> = 6.2 Hz, 2H), 2.87–2.92 (m, 2H), 3.01–3.14 (complex, 4H), 3.34 (q, <i>J</i> = 6.4 Hz, 2H), 3.50 (s, 2H), 7.21–7.39 (complex, 4H), 7.89 (dd, <i>J</i> = 1.4, 7.8 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 15.5, 19.6, 31.2, 43.4, 125.8, 129.1, 129.7, 129.8; u (C, CH<sub>2</sub>): 24.8, 27.4, 29.1, 39.8, 42.0, 53.0, 53.7, 58.1, 61.7, 126.4, 132.3, 143.3, 145.5, 159.5, 174.9; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3537; found 481.3543; HPLC purity: 98.7%.</div></div><div id="sec4_6_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2,6-dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7w</b></h4><div class="NLM_p last">1-((2-(2,6-Dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (52 mg, 0.16 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (27 mg, 0.16 mmol) were reacted according to general procedure A-4 to afford the product as a white solid (74 mg, 0.15 mmol, 97% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.52 (q, <i>J</i> = 11.6 Hz, 1H), 0.83 (d, <i>J</i> = 6.4 Hz, 6H), 1.42 (t, <i>J</i> = 11.1 Hz, 2H), 1.60–1.84 (complex, 9H), 1.94–2.02 (m, 1H), 2.08 (dt, <i>J</i> = 2.8, 11.5 Hz, 2H), 2.19 (s, 6H), 2.34 (s, 3H), 2.42 (t, <i>J</i> = 6.1 Hz, 2H), 2.83–2.90 (m, 2H), 2.96–3.02 (m, 2H), 3.30 (q, <i>J</i> = 6.1 Hz, 2H), 3.48 (s, 2H), 7.04 (d, <i>J</i> = 7.6 Hz, 2H), 7.18 (t, <i>J</i> = 7.5 Hz, 1H), 7.31 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 19.5, 20.2, 31.1, 43.3, 127.4, 129.5; u (C, CH<sub>2</sub>): 24.7, 29.0, 39.6, 41.9, 52.9, 53.5, 58.0, 61.5, 128.5, 131.3, 138.3, 145.7, 158.7, 174.9; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3537; found 481.3532; HPLC purity: 98.1%.</div></div><div id="sec4_6_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(3,4-dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7x</b></h4><div class="NLM_p last">1-((2-(3,4-Dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.076 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (16 mg, 0.091 mmol) were reacted according to general procedure A-4 to afford the product as a white solid (34 mg, 0.071 mmol, 93% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.54 (q, <i>J</i> = 11.2 Hz, 1H), 0.86 (d, <i>J</i> = 6.6 Hz, 6H), 1.41 (t, <i>J</i> = 11.1 Hz, 2H), 1.56–1.84 (complex, 9H), 1.99–2.08 (m, 3H), 2.29 (s, 3H), 2.30 (s, 3H), 2.37 (s, 3H), 2.41 (t, <i>J</i> = 6.2 Hz, 2H), 2.84–2.90 (m, 2H), 3.00 (d, <i>J</i> = 11.2 Hz, 2H), 3.32 (q, <i>J</i> = 6.1 Hz, 2H), 3.43 (s, 2H), 7.17 (d, <i>J</i> = 7.9 Hz, 1H), 7.71 (dd, <i>J</i> = 1.7, 7.8 Hz, 1H), 7.80 (s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 19.7, 19.8, 31.3, 43.4, 123.5, 127.2, 129.9; u (C, CH<sub>2</sub>): 24.9, 28.9, 39.8, 42.0, 53.1, 53.9, 58.1, 61.7, 125.3, 132.3, 136.9, 138.7, 145.6, 159.8, 174.9; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3537; found 481.3532; HPLC purity: 95.4%.</div></div><div id="sec4_6_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2,4-dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7y</b></h4><div class="NLM_p last">1-((2-(2,4-Dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.076 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (14 mg, 0.084 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (23 mg, 0.047 mmol, 62% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.53 (q, <i>J</i> = 12.0 Hz, 1H), 0.85 (d, <i>J</i> = 6.5 Hz, 6H), 1.41 (t, <i>J</i> = 11.1 Hz, 2H), 1.59–1.84 (complex, 9H), 1.96–2.11 (m, 3H), 2.32 (s, 3H), 2.36 (s, 3H), 2.41 (t, <i>J</i> = 6.2 Hz, 2H), 2.60 (s, 3H), 2.84–2.91 (m, 2H), 3.02 (d, <i>J</i> = 11.7 Hz, 2H), 3.32 (q, <i>J</i> = 6.5 Hz, 2H), 3.46 (s, 2H), 7.03–7.06 (m, 2H), 7.31 (br s, 1H), 7.80 (d, <i>J</i> = 7.7 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 21.3, 21.7, 31.3, 43.5, 126.5, 128.7, 132.2; u (C, CH<sub>2</sub>): 24.8, 29.0, 39.8, 42.0, 53.1, 53.9, 58.2, 61.7, 124.2, 131.9, 136.8, 139.4, 145.2, 159.9, 174.9; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3537; found 481.3534; HPLC purity: 99.2%.</div></div><div id="sec4_6_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(3,5-dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7z</b></h4><div class="NLM_p last">1-((2-(3,5-Dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.076 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (14 mg, 0.084 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, colorless oil (19 mg, 0.040 mmol, 52% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.54 (q, <i>J</i> = 11.9 Hz, 1H), 0.85 (d, <i>J</i> = 6.5 Hz, 6H), 1.42 (t, <i>J</i> = 11.1 Hz, 2H), 1.60–1.84 (complex, 9H), 1.97–2.08 (m, 3H), 2.34 (s, 6H), 2.37 (s, 3H), 2.42 (t, <i>J</i> = 6.1 Hz, 2H), 2.85–2.90 (m, 2H), 2.99 (d, <i>J</i> = 11.6 Hz, 2H), 3.32 (q, <i>J</i> = 6.3 Hz, 2H), 3.43 (s, 2H), 7.03 (s, 1H), 7.29 (br s, 1H), 7.73 (s, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 21.2, 31.2, 43.4, 123.8, 131.5; u (C, CH<sub>2</sub>): 24.8, 29.0, 39.8, 42.0, 53.2, 54.0, 58.1, 61.7, 127.5, 132.5, 138.2, 145.8, 159.7, 174.9; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3537; found 481.3545; HPLC purity: 99.8%.</div></div><div id="sec4_6_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2,5-dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7aa</b></h4><div class="NLM_p last">1-((2-(2,5-Dimethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.076 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (14 mg, 0.084 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (8 mg, 0.017 mmol, 22% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 11.4 Hz, 1H), 0.87 (d, <i>J</i> = 6.5 Hz, 6H), 1.46 (t, <i>J</i> = 11.0 Hz, 2H), 1.64–1.87 (complex, 9H), 1.98–2.13 (m, 3H), 2.34 (s, 3H), 2.37 (s, 3H), 2.46 (t, <i>J</i> = 6.0 Hz, 2H), 2.59 (s, 3H), 2.87–2.94 (m, 2H), 3.03 (d, <i>J</i> = 11.6 Hz, 2H), 3.33 (q, <i>J</i> = 6.3 Hz, 2H), 3.47 (s, 2H), 7.07–7.15 (m, 2H), 7.30 (br s, 1H), 7.75 (s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.3, 20.8, 21.3, 30.3, 43.1, 129.2, 130.3, 131.4; u (C, CH<sub>2</sub>): 24.4, 28.8, 38.5, 41.2, 52.9, 53.7, 56.8, 60.6, 126.5, 131.9, 133.9, 135.3, 145.7, 160.1, 175.3; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 481.3537; found 481.3533; HPLC purity: 98.7%.</div></div><div id="sec4_6_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-mesityl-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7bb</b></h4><div class="NLM_p last">1-((2-Mesityl-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.073 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (14 mg, 0.080 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (19 mg, 0.039 mmol, 53% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.54 (q, <i>J</i> = 11.8 Hz, 1H), 0.86 (d, <i>J</i> = 6.5 Hz, 6H), 1.43 (t, <i>J</i> = 11.1 Hz, 2H), 1.62–1.87 (complex, 9H), 1.97–2.04 (m, 1H), 2.10 (dt, <i>J</i> = 2.6, 11.4 Hz, 2H), 2.19 (s, 6H), 2.28 (s, 3H), 2.35 (s, 3H), 2.44 (t, <i>J</i> = 6.2 Hz, 2H), 2.86–2.93 (m, 2H), 3.01 (d, <i>J</i> = 11.6 Hz, 2H), 3.32 (q, <i>J</i> = 6.4 Hz, 2H), 3.50 (s, 2H), 6.88 (s, 2H), 7.33 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 19.6, 20.2, 21.2, 31.1, 43.4, 128.3; u (C, CH<sub>2</sub>): 24.7, 29.1, 39.7, 42.0, 52.9, 53.6, 58.0, 61.6, 125.7, 131.3, 138.2, 139.3, 145.7, 158.9, 174.9; HRMS calcd for C<sub>30</sub>H<sub>47</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 495.3694; found 495.3666; HPLC purity: 96.2%.</div></div><div id="sec4_6_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2,3,4,5,6-pentamethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7cc</b></h4><div class="NLM_p last">1-((2-(2,3,4,5,6-Pentamethylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.067 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (13 mg, 0.074 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (24 mg, 0.045 mmol, 67% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.54 (q, <i>J</i> = 12.0 Hz, 1H), 0.85 (d, <i>J</i> = 6.5 Hz, 6H), 1.41 (t, <i>J</i> = 11.0 Hz, 2H), 1.58–1.87 (complex, 9H), 1.94–2.02 (m, 1H), 2.01 (s, 6H), 2.10 (dt, <i>J</i> = 2.6, 11.4 Hz, 2H), 2.19 (s, 6H), 2.24 (s, 3H), 2.35 (s, 3H), 2.42 (t, <i>J</i> = 6.1 Hz, 2H), 2.85–2.93 (m, 2H), 3.02 (d, <i>J</i> = 11.6 Hz, 2H), 3.32 (q, <i>J</i> = 5.9 Hz, 2H), 3.52 (s, 2H), 7.33 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 16.3, 16.9, 17.9, 19.6, 31.2, 43.4; u (C, CH<sub>2</sub>): 24.8, 29.1, 39.9, 42.0, 52.8, 53.5, 58.2, 61.7, 126.9, 130.9, 132.5, 133.5, 136.7, 145.5, 160.4, 174.9; HRMS calcd for C<sub>32</sub>H<sub>51</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 523.4007; found 523.3984; HPLC purity: >99.8%.</div></div><div id="sec4_6_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-chlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7dd</b></h4><div class="NLM_p last">1-((2-(2-Chlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (33 mg, 0.098 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (33 mg, 0.20 mmol) were reacted according to general procedure A-3 to afford the product as a white solid (7 mg, 0.014 mmol, 14% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 12.0 Hz, 1H), 0.89 (d, <i>J</i> = 6.5 Hz, 6H), 1.41 (t, <i>J</i> = 11.0 Hz, 2H), 1.57–1.90 (complex, 9H), 1.98–2.17 (m, 3H), 2.44 (s, 3H), 2.46 (t, <i>J</i> = 6.2 Hz, 2H), 2.86–2.90 (m, 2H), 3.03 (d, <i>J</i> = 11.8 Hz, 2H), 3.34 (q, <i>J</i> = 6.2 Hz, 2H), 3.52 (s, 2H), 7.30–7.40 (m, 2H), 7.45–7.54 (m, 1H), 7.93–8.02 (m, 1H), 8.23 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 19.6, 31.1, 43.4, 126.7, 130.6, 130.9, 131.0; u (C, CH<sub>2</sub>): 24.8, 29.0, 39.7, 41.9, 53.1, 53.8, 58.0, 61.6, 126.7, 132.2, 132.7, 146.7, 157.3, 174.9; HRMS calcd for C<sub>27</sub>H<sub>40</sub>ClN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 487.2834; found 487.2842; HPLC purity: >99.8%.</div></div><div id="sec4_6_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(3-chlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7ee</b></h4><div class="NLM_p last">1-((2-(3-Chlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (47 mg, 0.14 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (24 mg, 0.14 mmol) were reacted according to general procedure A-4 to afford the product as an off-white solid (52 mg, 0.11 mmol, 76% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.54 (q, <i>J</i> = 12.4 Hz, 1H), 0.86 (d, <i>J</i> = 6.6 Hz, 6H), 1.40 (t, <i>J</i> = 11.1 Hz, 2H), 1.57–1.87 (complex, 9H), 2.00–2.10 (m, 3H), 2.39 (s, 3H), 2.40 (t, <i>J</i> = 6.2 Hz, 2H), 2.84–2.89 (m, 2H), 2.96–3.02 (m, 2H), 3.32 (q, <i>J</i> = 6.1 Hz, 2H), 3.43 (s, 2H), 7.29 (br s, 1H), 7.35–7.38 (m, 2H), 7.87–7.90 (m, 1H), 8.00–8.01 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 31.3, 43.4, 124.1, 126.1, 129.8, 130.0; u (C, CH<sub>2</sub>): 24.9, 28.9, 39.8, 42.1, 53.1, 53.8, 58.0, 61.7, 129.3, 133.0, 134.7, 146.7, 158.1, 174.9; HRMS calcd for C<sub>27</sub>H<sub>40</sub>ClN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 487.2834; found 487.2829; HPLC purity: >99.8%.</div></div><div id="sec4_6_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(4-chlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7ff</b></h4><div class="NLM_p last">1-((2-(4-Chlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (28 mg, 0.083 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (28 mg, 0.17 mmol) were reacted according to general procedure A-3 to afford the product as a white solid (27 mg, 0.055 mmol, 66% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.55 (q, <i>J</i> = 11.9 Hz, 1H), 0.87 (d, <i>J</i> = 6.6 Hz, 6H), 1.41 (t, <i>J</i> = 11.1 Hz, 2H), 1.56–1.88 (complex, 9H), 1.95–2.11 (m, 3H), 2.39 (s, 3H), 2.41 (t, <i>J</i> = 6.1 Hz, 2H), 2.85–2.89 (m, 2H), 3.00 (d, <i>J</i> = 11.5 Hz, 2H), 3.33 (q, <i>J</i> = 6.1 Hz, 2H), 3.44 (s, 2H), 7.31 (br s, 1H), 7.38–7.42 (m, 2H), 7.92–7.97 (m, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 31.4, 43.4, 127.3, 128.9; u (C, CH<sub>2</sub>): 24.9, 28.9, 39.9, 42.1, 53.2, 53.9, 58.2, 61.8, 126.2, 133.0, 135.8, 146.3, 158.5, 174.8; HRMS calcd for C<sub>27</sub>H<sub>40</sub>ClN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 487.2834; found 487.2841; HPLC purity: >99.8%.</div></div><div id="sec4_6_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(4-fluorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7gg</b></h4><div class="NLM_p last">1-((2-(4-Fluorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.079 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (15 mg, 0.086 mmol) were reacted according to general procedure A-3 to afford the product as a white solid (25 mg, 0.054 mmol, 69% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.55 (q, <i>J</i> = 11.9 Hz, 1H), 0.87 (d, <i>J</i> = 6.6 Hz, 6H), 1.41 (t, <i>J</i> = 11.1 Hz, 2H), 1.56–1.88 (complex, 9H), 1.95–2.11 (m, 3H), 2.39 (s, 3H), 2.41 (t, <i>J</i> = 6.1 Hz, 2H), 2.85–2.89 (m, 2H), 3.00 (d, <i>J</i> = 11.5 Hz, 2H), 3.33 (q, <i>J</i> = 6.1 Hz, 2H), 3.44 (s, 2H), 7.31 (br s, 1H), 7.38–7.42 (m, 2H), 7.92–7.97 (m, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.4, 30.8, 43.3, 115.7 (d, <i>J</i> = 22.1 Hz), 128.1 (d, <i>J</i> = 8.5 Hz); u (C, CH<sub>2</sub>): 24.7, 28.8, 39.2, 41.7, 53.1, 53.9, 57.5, 61.2, 124.1 (d, <i>J</i> = 3.2 Hz), 132.6, 146.1, 158.6 (d, <i>J</i> = 0.4 Hz), 163.7 (d, <i>J</i> = 250.9 Hz), 175.0; HRMS calcd for C<sub>27</sub>H<sub>40</sub>FN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 471.3130; found 471.3114; HPLC purity: >99.8%.</div></div><div id="sec4_6_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2,6-dichlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7hh</b></h4><div class="NLM_p last">1-((2-(2,5-Dichlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.068 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (13 mg, 0.074 mmol) were reacted according to general procedure A-4 to afford the product as a viscous, light yellow oil (21 mg, 0.041 mmol, 60% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 11.9 Hz, 1H), 0.88 (d, <i>J</i> = 6.5 Hz, 6H), 1.44 (t, <i>J</i> = 11.0 Hz, 2H), 1.64–1.91 (complex, 9H), 1.97–2.06 (m, 1H), 2.12 (dt, <i>J</i> = 2.6, 11.4 Hz, 2H), 2.42 (s, 3H), 2.45 (t, <i>J</i> = 6.1 Hz, 2H), 2.88–2.92 (m, 2H), 3.00–3.03 (m, 2H), 3.34 (q, <i>J</i> = 6.1 Hz, 2H), 3.56 (s, 2H), 7.30–7.43 (complex, 4H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 31.2, 43.4, 128.0, 131.6; u (C, CH<sub>2</sub>): 24.7, 29.1, 39.8, 42.0, 52.8, 53.5, 58.1, 61.6, 128.3, 132.0, 136.4, 147.1, 154.0, 174.9; HRMS calcd for C<sub>27</sub>H<sub>39</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 521.2445; found 521.2439; HPLC purity: >99.8%.</div></div><div id="sec4_6_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(3,4-dichlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7ii</b></h4><div class="NLM_p last">1-((2-(3,4-Dichlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.068 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (13 mg, 0.074 mmol) were reacted according to general procedure A-4 to afford the product as a viscous, light yellow oil (28 mg, 0.053 mmol, 78% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.71 (q, <i>J</i> = 12.9 Hz, 1H), 0.93 (d, <i>J</i> = 6.1 Hz, 6H), 1.82–2.10 (complex, 11H), 2.15–2.32 (m, 3H), 2.37 (s, 3H), 2.94 (t, <i>J</i> = 7.3 Hz, 2H), 3.05–3.14 (m, 2H), 3.23–3.34 (m, 4H), 3.54 (s, 2H), 7.18 (br s, 1H), 7.48 (d, <i>J</i> = 8.4 Hz, 1H), 7.81 (dd, <i>J</i> = 2.0, 8.4 Hz, 1H), 8.08 (d, <i>J</i> = 2.0 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 18.8, 28.8, 41.0, 125.1, 127.9, 130.8; u (C, CH<sub>2</sub>): 23.9, 28.0, 28.0, 36.4, 40.0, 52.6, 53.1, 58.9, 127.4, 133.1, 134.0, 157.6, 162.5, 162.8, 175.6; IR 1541, 1641, 2922, 2947 cm<sup>–1</sup>; HRMS calcd for C<sub>27</sub>H<sub>39</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 521.2445; found 521.2437; HPLC purity: 99.5%.</div></div><div id="sec4_6_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2,6-difluorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7jj</b></h4><div class="NLM_p last">1-((2-(2,6-Difluorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.074 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (14 mg, 0.082 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (12 mg, 0.024 mmol, 33% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 12.3 Hz, 1H), 0.87 (d, <i>J</i> = 6.5 Hz, 6H), 1.49 (t, <i>J</i> = 10.4 Hz, 2H), 1.67–1.86 (complex, 9H), 1.98–2.12 (m, 3H), 2.40 (s, 3H), 2.48 (t, <i>J</i> = 5.4 Hz, 2H), 2.89–2.96 (m, 2H), 3.00 (d, <i>J</i> = 11.6 Hz, 2H), 3.33 (q, <i>J</i> = 6.4 Hz, 2H), 3.50 (s, 2H), 6.99 (t, <i>J</i> = 8.3 Hz, 2H), 7.29–7.40 (m, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 19.5, 30.9, 43.3, 112.0 (d, <i>J</i> = 25.6 Hz), 131.4 (t, <i>J</i> = 10.5 Hz); u (C, CH<sub>2</sub>): 24.6, 29.0, 39.3, 41.7, 53.1, 53.8, 57.7, 61.3, 107.0 (d, <i>J</i> = 16.1 Hz), 132.9, 147.2, 150.5 (d, <i>J</i> = 3.1 Hz), 160.7 (dd, <i>J</i> = 5.9, 257.3 Hz), 175.0; HRMS calcd for C<sub>27</sub>H<sub>39</sub>F<sub>2</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 489.3036; found 489.3040; HPLC purity: 97.3%.</div></div><div id="sec4_6_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>N</i>-(3-(<i>cis</i>-3,5-Diethylpiperidin-1-yl)propyl)-1-((2-(2,6-difluorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7kk</b></h4><div class="NLM_p last">1-((2-(2,6-Difluorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.074 mmol) and 3-(<i>cis</i>-3,5-diethylpiperidin-1-yl)propan-1-amine (16 mg, 0.082 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (19 mg, 0.036 mmol, 49% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.49 (q, <i>J</i> = 12.4 Hz, 1H), 0.91 (d, <i>J</i> = 7.5 Hz, 6H), 1.19–1.29 (m, 4H), 1.14–1.49 (m, 4H), 1.66–1.89 (complex, 7H), 2.00–2.13 (m, 3H), 2.43 (s, 3H), 2.45 (t, <i>J</i> = 6.0 Hz, 2H), 2.94–2.98 (m, 2H), 2.99–3.06 (m, 2H), 3.35 (q, <i>J</i> = 6.3 Hz, 2H), 3.53 (s, 2H), 7.02 (t, <i>J</i> = 8.4 Hz, 2H), 7.34–7.43 (m, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 11.4, 38.0, 43.5, 112.0 (d, <i>J</i> = 25.6 Hz), 131.4 (t, <i>J</i> = 10.5 Hz); u (C, CH<sub>2</sub>): 24.9, 27.4, 29.0, 37.3, 39.9, 53.1, 53.8, 58.4, 60.4, 106.8 (d, <i>J</i> = 16.3 Hz), 132.9, 147.1, 150.5, 160.7 (d, <i>J</i> = 257.3 Hz), 174.9; HRMS calcd for C<sub>29</sub>H<sub>43</sub>F<sub>2</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 517.3349; found 517.3351; HPLC purity: >99.8%.</div></div><div id="sec4_6_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2,6-dibromophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7ll</b></h4><div class="NLM_p last">1-((2-(2,5-Dibromophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.055 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (10 mg, 0.060 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (22 mg, 0.037 mmol, 67% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 12.2 Hz, 1H), 0.88 (d, <i>J</i> = 6.5 Hz, 6H), 1.43 (t, <i>J</i> = 11.0 Hz, 2H), 1.61–1.90 (complex, 9H), 1.98–2.02 (m, 1H), 2.14 (dt, <i>J</i> = 2.8, 11.5 Hz, 2H), 2.42 (s, 3H), 2.44 (t, <i>J</i> = 6.0 Hz, 2H), 2.87–2.93 (m, 2H), 2.99–3.05 (m, 2H), 3.34 (q, <i>J</i> = 6.3 Hz, 2H), 3.58 (s, 2H), 7.20 (t, <i>J</i> = 8.1 Hz, 1H), 7.37 (br s, 1H), 7.62 (d, <i>J</i> = 8.1 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 31.3, 43.4, 131.6, 132.3; u (C, CH<sub>2</sub>): 24.7, 29.1, 39.9, 42.0, 52.6, 53.3, 58.2, 61.7, 125.3, 131.7, 132.2, 146.8, 156.4, 174.9; HRMS calcd for C<sub>27</sub>H<sub>39</sub>Br<sub>2</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 609.1434; found 609.1430; HPLC purity: >99.8%.</div></div><div id="sec4_6_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-bromo-6-chlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7mm</b></h4><div class="NLM_p last">1-((2-(2-Bromo-6-chlorophenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.060 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (11 mg, 0.066 mmol) were reacted according to general procedure A-3 to afford the product as a white solid (23 mg, 0.041 mmol, 68% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 11.8 Hz, 1H), 0.88 (d, <i>J</i> = 6.5 Hz, 6H), 1.44 (t, <i>J</i> = 11.0 Hz, 2H), 1.61–1.90 (complex, 9H), 1.98–2.04 (m, 1H), 2.13 (dt, <i>J</i> = 2.8, 11.5 Hz, 2H), 2.42 (s, 3H), 2.45 (t, <i>J</i> = 6.1 Hz, 2H), 2.86–2.95 (m, 2H), 2.98–3.05 (m, 2H), 3.34 (q, <i>J</i> = 6.3 Hz, 2H), 3.57 (s, 2H), 7.27 (t, <i>J</i> = 8.1 Hz, 1H), 7.36 (br s, 1H), 7.44 (dd, <i>J</i> = 1.1, 8.1 Hz, 1H), 7.58 (dd, <i>J</i> = 1.1, 8.1 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 31.2, 43.4, 128.5, 131.1, 131.9; u (C, CH<sub>2</sub>): 24.7, 29.1, 39.8, 42.0, 52.7, 53.4, 58.2, 61.7, 125.4, 130.3, 131.8, 136.4, 147.0, 155.2, 174.9; HRMS calcd for C<sub>27</sub>H<sub>39</sub>BrClN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 565.1939; found 565.1939; HPLC purity: >99.8%.</div></div><div id="sec4_6_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-chloro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7nn</b></h4><div class="NLM_p last">1-((2-(2-Chloro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (80 mg, 0.23 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (59 mg, 0.078 mmol) were reacted according to general procedure A-4 to afford the product as a white solid (71 mg, 0.14 mmol, 62% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.53 (q, <i>J</i> = 12.0 Hz, 1H), 0.86 (d, <i>J</i> = 6.5 Hz, 6H), 1.39 (t, <i>J</i> = 11.0 Hz, 2H), 1.53–1.89 (complex, 9H), 1.96–2.04 (m, 1H), 2.11 (dt, <i>J</i> = 2.8, 11.5 Hz, 2H), 2.24 (s, 3H), 2.39 (s, 3H), 2.40 (t, <i>J</i> = 6.0 Hz, 2H), 2.84–2.88 (m, 2H), 2.99–3.04 (m, 2H), 3.32 (q, <i>J</i> = 6.0 Hz, 2H), 3.53 (s, 2H), 7.14–7.17 (m, 1H), 7.23–7.31 (m, 2H), 7.36 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 19.6, 20.2, 31.4, 43.4, 126.9, 128.3, 130.7; u (C, CH<sub>2</sub>): 24.8, 29.1, 40.0, 42.1, 52.8, 53.5, 58.3, 61.8, 128.3, 131.7, 134.8, 140.9, 146.5, 156.2, 174.8; HRMS calcd for C<sub>28</sub>H<sub>42</sub>ClN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 501.2991; found 501.2995; HPLC purity: 96.1%.</div></div><div id="sec4_6_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (±)-1-((2-(2-Chloro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)-<i>N</i>-(3-(3-methylpiperidin-1-yl)propyl)piperidine-4-carboxamide <b>7oo</b></h4><div class="NLM_p last">1-((2-(2-Chloro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.072 mmol) and 3-(3-methylpiperidin-1-yl)propan-1-amine (12 mg, 0.079 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (20 mg, 0.041 mmol, 56% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.80–0.91 (m, 1H), 0.85 (d, <i>J</i> = 6.4 Hz, 3H), 1.46–1.88 (complex, 12H), 1.96–2.04 (m, 1H), 2.09 (dt, <i>J</i> = 2.8, 11.5 Hz, 2H), 2.22 (s, 3H), 2.37 (s, 3H), 2.40 (t, <i>J</i> = 6.2 Hz, 2H), 2.81–2.90 (m, 2H), 2.97–3.03 (m, 2H), 3.30 (q, <i>J</i> = 6.2 Hz, 2H), 3.51 (s, 2H), 7.12–7.15 (m, 1H), 7.22–7.28 (m, 2H), 7.38 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 19.7, 20.2, 31.3, 43.4, 126.9, 128.3, 130.7; u (C, CH<sub>2</sub>): 24.7, 25.6, 29.0, 32.9, 39.9, 52.8, 53.5, 54.1, 58.4, 62.1, 128.3, 131.6, 134.8, 140.9, 146.5, 156.2, 174.9; HRMS calcd for C<sub>27</sub>H<sub>40</sub>ClN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 487.2834; found 487.2839; HPLC purity: >99.8%.</div></div><div id="sec4_6_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 1-((2-(2-Chloro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)-<i>N</i>-(3-(4,4-dimethylpiperidin-1-yl)propyl)piperidine-4-carboxamide <b>7pp</b></h4><div class="NLM_p last">1-((2-(2-Chloro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.072 mmol) and 3-(4,4-dimethylpiperidin-1-yl)propan-1-amine (13 mg, 0.079 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (20 mg, 0.039 mmol, 55% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) 0.92 (s, 6H), 1.40 (t, <i>J</i> = 5.7 Hz, 4H), 1.63–1.69 (m, 2H), 1.73–1.93 (complex, 4H), 1.98–2.07 (m, 1H), 2.12 (dt, <i>J</i> = 2.6, 11.5 Hz, 2H), 2.25 (s, 3H), 2.39 (s, 3H), 2.39–2.44 (m, 4H), 2.46 (t, <i>J</i> = 6.1 Hz, 2H), 3.03 (d, <i>J</i> = 11.7 Hz, 2H), 3.33 (q, <i>J</i> = 5.92 Hz, 2H), 3.53 (s, 2H), 7.12–7.20 (m, 1H), 7.22–7.33 (m, 2H), 7.54 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 20.3, 41.0, 43.4, 126.9, 128.3, 130.7; u (C, CH<sub>2</sub>): 24.7, 28.5, 29.1, 38.9, 40.1, 50.3, 52.9, 53.5, 58.3, 128.3, 131.6, 134.8, 140.9, 146.5, 156.2, 174.9; HRMS calcd for C<sub>28</sub>H<sub>42</sub>ClN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 501.2991; found 501.2985; HPLC purity: >99.8%.</div></div><div id="sec4_6_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 1-((2-(2-Chloro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)-<i>N</i>-(3-(4-isopropylpiperidin-1-yl)propyl)piperidine-4-carboxamide <b>7qq</b></h4><div class="NLM_p last">1-((2-(2-Chloro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.072 mmol) and 3-(4-isopropylpiperidin-1-yl)propan-1-amine (15 mg, 0.079 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (21 mg, 0.041 mmol, 57% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.87 (d, <i>J</i> = 6.8 Hz, 6H), 1.02–1.09 (m, 1H), 1.38–1.49 (m, 1H), 1.61–1.71 (m, 4H), 1.73–1.93 (complex, 6H), 1.99–2.05 (m, 1H), 2.11 (dt, <i>J</i> = 2.6, 11.5 Hz, 2H), 2.25 (s, 3H), 2.39 (s, 3H), 2.42 (t, <i>J</i> = 6.0 Hz, 2H), 3.01 (t, <i>J</i> = 10.5 Hz, 4H), 3.34 (q, <i>J</i> = 6.0 Hz, 2H), 3.53 (s, 2H), 7.13–7.20 (m, 1H), 7.23–7.34 (m, 2H), 7.52 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 19.7, 20.3, 32.4, 42.3, 43.4, 126.9, 128.3, 130.7; u (C, CH<sub>2</sub>): 24.8, 29.1, 29.4, 40.1, 52.9, 53.5, 54.5, 58.4, 128.3, 131.6, 134.8, 140.9, 146.5, 156.2, 174.8; HRMS calcd for C<sub>29</sub>H<sub>44</sub>ClN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 515.3147; found 515.3149; HPLC purity: >99.8%.</div></div><div id="sec4_6_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-chloro-6-(trifluoromethyl)phenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7rr</b></h4><div class="NLM_p last">1-((2-(2-Chloro-6-(trifluoromethyl)phenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (24 mg, 0.061 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (11 mg, 0.067 mmol) were reacted according to general procedure A-4 to afford the product as a viscous, light yellow oil (20 mg, 0.035 mmol, 58% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.55 (q, <i>J</i> = 11.9 Hz, 1H), 0.87 (d, <i>J</i> = 6.6 Hz, 6H), 1.41 (t, <i>J</i> = 11.0 Hz, 2H), 1.62–1.91 (complex, 9H), 1.96–2.01 (m, 1H), 2.12 (dt, <i>J</i> = 2.7, 11.5 Hz, 2H), 2.41 (s, 3H), 2.43 (t, <i>J</i> = 5.9 Hz, 2H), 2.85–2.93 (m, 2H), 2.97–3.02 (m, 2H), 3.35 (q, <i>J</i> = 6.4 Hz, 2H), 3.57 (s, 2H), 7.41 (br s, 1H) 7.54–7.59 (m, 1H), 7.70 (d, <i>J</i> = 8.4 Hz, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 19.6, 31.3, 43.4, 124.6 (q, <i>J</i> = 4.9 Hz), 131.3, 132.9; u (C, CH<sub>2</sub>): 24.7, 29.1, 40.1, 42.1, 52.6, 53.3, 58.4, 61.8, 125.5 (q, <i>J</i> = 286.8 Hz), 131.6, 1325 (q, <i>J</i> = 31.6 Hz), 137.1, 147.7, 153.1, 155.3, 175.4; HRMS calcd for C<sub>29</sub>H<sub>39</sub>ClF<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 555.2708; found 555.2705; HPLC purity: >99.8%.</div></div><div id="sec4_6_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-fluoro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7ss</b></h4><div class="NLM_p last">1-((2-(2-Fluoro-6-methylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.075 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (14 mg, 0.083 mmol) were reacted according to general procedure A-4 to afford the product as a viscous, light yellow oil (22 mg, 0.046 mmol, 61% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 12.3 Hz, 1H), 0.88 (d, <i>J</i> = 6.6 Hz, 6H), 1.41 (t, <i>J</i> = 11.0 Hz, 2H), 1.58–1.91 (complex, 9H), 1.96–2.06 (m, 1H), 2.11 (dt, <i>J</i> = 3.0, 11.4 Hz, 2H), 2.41 (s, 3H), 2.43 (s, 3H), 2.43 (t, <i>J</i> = 6.2 Hz, 2H), 2.86–2.91 (m, 2H), 2.99–3.09 (m, 2H), 3.35 (q, <i>J</i> = 6.0 Hz, 2H), 3.52 (s, 2H), 6.99 (t, <i>J</i> = 9.3 Hz, 1H), 7.07 (d, <i>J</i> = 7.7 Hz, 1H), 7.27–7.33 (m, 1H), 7.35 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 20.4, 31.4, 43.5, 113.2 (d, <i>J</i> = 22.1 Hz), 126.0, 130.8; u (C, CH<sub>2</sub>): 24.8, 29.1, 40.0, 42.1, 53.0, 53.7, 58.3, 61.8, 116.7 (d, <i>J</i> = 13.9 Hz), 132.2, 140.6 (d, <i>J</i> = 1.5 Hz), 146.4, 154.4, 161.1 (d, <i>J</i> = 252.2 Hz) 174.8; HRMS calcd for C<sub>28</sub>H<sub>42</sub>FN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 485.3286; found 485.3278; HPLC purity: >99.8%.</div></div><div id="sec4_6_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-bromo-6-methoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7tt</b></h4><div class="NLM_p last">1-((2-(2-Bromo-6-methoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.061 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (11 mg, 0.067 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (26 mg, 0.046 mmol, 76% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 11.9 Hz, 1H), 0.88 (d, <i>J</i> = 6.5 Hz, 6H), 1.45 (t, <i>J</i> = 11.0 Hz, 2H), 1.65–1.90 (complex, 9H), 1.98–2.05 (m, 1H), 2.13 (dt, <i>J</i> = 2.9, 11.5 Hz, 2H), 2.40 (s, 3H), 2.46 (t, <i>J</i> = 6.1 Hz, 2H), 2.87–2.99 (m, 2H), 3.01–3.05 (m, 2H), 3.34 (q, <i>J</i> = 6.3 Hz, 2H), 3.55 (s, 2H), 3.78 (s, 3H), 6.91 (dd, <i>J</i> = 1.3, 8.1 Hz, 1H), 7.20–7.36 (m, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 31.1, 43.4, 56.2, 109.9, 124.6, 132.1; u (C, CH<sub>2</sub>): 24.8, 29.1, 39.7, 41.9, 52.8, 53.6, 58.0, 61.6, 120.2, 125.3, 131.6, 146.6, 155.2, 159.9, 174.9; HRMS calcd for C<sub>28</sub>H<sub>42</sub>BrN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> 561.2435; found 561.2434; HPLC purity: 98.4%.</div></div><div id="sec4_6_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-chloro-6-methoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7uu</b></h4><div class="NLM_p last">1-((2-(2-Chloro-6-methoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.069 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (13 mg, 0.075 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (23 mg, 0.045 mmol, 66% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.55 (q, <i>J</i> = 11.9 Hz, 1H), 0.87 (d, <i>J</i> = 6.5 Hz, 6H), 1.43 (t, <i>J</i> = 11.1 Hz, 2H), 1.58–1.91 (complex, 9H), 1.97–2.07 (m, 1H), 2.12 (dt, <i>J</i> = 2.8, 11.5 Hz, 2H), 2.40 (s, 3H), 2.43 (t, <i>J</i> = 6.2 Hz, 2H), 2.87–2.91 (m, 2H), 3.01–3.05 (m, 2H), 3.33 (q, <i>J</i> = 6.3 Hz, 2H), 3.53 (s, 2H), 3.80 (s, 3H), 6.87 (dd, <i>J</i> = 0.9, 8.5 Hz, 1H), 7.07 (dd, <i>J</i> = 0.9, 8.1 Hz, 1H), 7.32–7.37 (m, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 31.2, 43.4, 56.2, 109.4, 121.6, 131.7; u (C, CH<sub>2</sub>): 24.8, 29.0, 39.8, 42.0, 52.8, 53.6, 58.1, 61.7, 118.0, 135.9, 146.8, 154.1, 159.8, 175.0; HRMS calcd for C<sub>28</sub>H<sub>42</sub>ClN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> 517.2940; found 517.2939; HPLC purity: 99.6%.</div></div><div id="sec4_6_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-fluoro-6-methoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7vv</b></h4><div class="NLM_p last">1-((2-(2-Fluoro-6-methoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.072 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (13 mg, 0.079 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (21 mg, 0.041 mmol, 58% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 11.9 Hz, 1H), 0.88 (d, <i>J</i> = 6.5 Hz, 6H), 1.44 (t, <i>J</i> = 11.0 Hz, 2H), 1.64–1.91 (complex, 9H), 1.98–2.06 (m, 1H), 2.10 (dt, <i>J</i> = 2.8, 11.4 Hz, 2H), 2.40 (s, 3H), 2.43 (t, <i>J</i> = 6.1 Hz, 2H), 2.86–2.95 (m, 2H), 3.02–3.06 (m, 2H), 3.35 (q, <i>J</i> = 6.3 Hz, 2H), 3.51 (s, 2H), 3.85 (s, 3H), 6.73–6.83 (m, 2H), 7.29–7.42 (m, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 19.6, 31.2, 43.4, 56.4, 106.8, 108.2 (d, <i>J</i> = 22.2 Hz), 131.7 (d, <i>J</i> = 10.8 Hz); u (C, CH<sub>2</sub>): 24.8, 29.0, 39.8, 42.0, 53.1, 53.8, 58.1, 61.7, 132.2, 146.7, 152.1, 159.4 (d, <i>J</i> = 5.8 Hz), 161.7 (d, <i>J</i> = 252.4 Hz), 174.9; HRMS calcd for C<sub>28</sub>H<sub>42</sub>FN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> 501.3235; found 501.3228; HPLC purity: >99.8%.</div></div><div id="sec4_6_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(2-chloro-6-methoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7ww</b></h4><div class="NLM_p last">1-((2-(2-Methoxy-6-methylphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.073 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (14 mg, 0.080 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (25 mg, 0.050 mmol, 69% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.55 (q, <i>J</i> = 11.9 Hz, 1H), 0.87 (d, <i>J</i> = 6.5 Hz, 6H), 1.40 (t, <i>J</i> = 11.0 Hz, 2H), 1.58–1.90 (complex, 9H), 1.95–2.05 (m, 1H), 2.10 (dt, <i>J</i> = 2.8, 11.4 Hz, 2H), 2.22 (s, 3H), 2.38 (s, 3H), 2.42 (t, <i>J</i> = 6.1 Hz, 2H), 2.85–2.90 (m, 2H), 3.02–3.06 (m, 2H), 3.34 (q, <i>J</i> = 5.6 Hz, 2H), 3.52 (s, 2H), 3.77 (s, 3H), 6.79 (d, <i>J</i> = 8.3 Hz, 1H), 6.86 (d, <i>J</i> = 7.7 Hz, 1H), 7.24–7.31 (m, 1H), 7.34 (br s, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.4, 19.6, 31.4, 41.0, 43.5, 55.9, 108.4, 122.4, 130.7; u (C, CH<sub>2</sub>): 24.9, 29.1, 40.0, 42.1, 53.0, 53.7, 58.4, 61.8, 118.1, 131.5, 140.2, 146.0, 156.5, 158.7, 174.8; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> 497.3486; found 497.3483; HPLC purity: >99.8%.</div></div><div id="sec4_6_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2,6-dimethoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7xx</b></h4><div class="NLM_p last">1-((2-(2,6-Dimethoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.069 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (13 mg, 0.076 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (22 mg, 0.042 mmol, 61% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 12.0 Hz, 1H), 0.88 (d, <i>J</i> = 6.5 Hz, 6H), 1.43 (t, <i>J</i> = 11.0 Hz, 2H), 1.60–1.91 (complex, 9H), 1.97–2.06 (m, 1H), 2.10 (dt, <i>J</i> = 2.7, 11.5 Hz, 2H), 2.39 (s, 3H), 2.43 (t, <i>J</i> = 6.1 Hz, 2H), 2.87–2.91 (m, 2H), 3.03–3.07 (m, 2H), 3.34 (q, <i>J</i> = 6.2 Hz, 2H), 3.51 (s, 2H), 3.78 (s, 6H), 6.59 (d, <i>J</i> = 8.4 Hz, 2H), 7.29 (br s, 1H), 7.35 (t, <i>J</i> = 8.4 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 19.6, 31.3, 43.5, 56.1, 103.8, 131.7; u (C, CH<sub>2</sub>): 24.9, 29.1, 39.8, 42.0, 53.1, 53.9, 58.2, 61.7, 107.2, 131.7, 146.3, 154.2, 159.7, 175.0; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup> 513.3435; found 513.3431; HPLC purity: >99.8%.</div></div><div id="sec4_6_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((3,4-dimethoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7yy</b></h4><div class="NLM_p last">1-((2-(3,4-Dimethoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (57 mg, 0.16 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (81 mg, 0.47 mmol) were reacted according to general procedure A-1 to afford the product as a white solid (46 mg, 0.090 mmol, 56% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 12.0 Hz, 1H), 0.89 (d, <i>J</i> = 6.5 Hz, 6H), 1.43 (t, <i>J</i> = 11.1 Hz, 2H), 1.60–1.88 (complex, 9H), 2.01–2.12 (m, 3H), 2.40 (s, 3H), 2.43 (t, <i>J</i> = 6.1 Hz, 2H), 2.86–2.93 (m, 2H), 3.00–3.06 (m, 2H), 3.35 (q, <i>J</i> = 5.8 Hz, 2H), 3.45 (s, 2H), 3.94 (s, 3H), 3.98 (s, 3H), 6.93 (d, <i>J</i> = 8.4 Hz, 1H), 7.37 (br s, 1H), 7.55–7.57 (m, 1H), 7.61 (dd, <i>J</i> = 1.8, 8.4 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.3, 19.5, 31.2, 43.3, 55.9, 56.0, 108.9, 110.8, 119.1; u (C, CH<sub>2</sub>): 24.9, 28.8, 39.7, 42.0, 53.1, 53.9, 58.0, 61.6, 120.7, 132.3, 145.4, 148.9, 150.4, 159.3, 174.8; IR 1502, 1644, 2949 cm<sup>–1</sup>; HRMS calcd for C<sub>29</sub>H<sub>45</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup> 513.3435; found 513.3458; HPLC purity: 94.2%.</div></div><div id="sec4_6_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>N</i>-(4-(Diethylamino)butyl)-1-((2-(3,4-dimethoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxamide <b>7zz</b></h4><div class="NLM_p last">1-((2-(3,4-Dimethoxyphenyl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (30 mg, 0.091 mmol) and <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-diethylbutane-1,4-diamine (16 mg, 0.11 mmol) were reacted according to general procedure A-1 to afford the product as a white solid (26 mg, 0.053 mmol, 59% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.10 (t, <i>J</i> = 7.2 Hz, 6H), 1.51–1.64 (complex, 4H), 1.77–1.85 (m, 3H), 1.99–2.14 (m, 3H), 2.37 (s, 3H), 2.56 (t, <i>J</i> = 7.2 Hz, 2H), 2.67 (q, <i>J</i> = 7.2 Hz, 4H), 2.98–3.04 (m, 2H), 3.26 (q, <i>J</i> = 5.8 Hz, 2H), 3.42 (s, 2H), 3.59–3.65 (m, 1H), 3.93 (s, 3H), 3.96 (s, 3H), 6.33 (br s, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 7.54 (d, <i>J</i> = 1.9 Hz, 1H), 7.58 (dd, <i>J</i> = 2.08.4 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 8.1, 10.4, 10.6, 43.4, 55.9, 109.0, 110.9, 119.2; u (C, CH<sub>2</sub>): 23.7, 27.4, 28.8, 38.7, 46.5, 52.1, 53.1, 53.9, 120.8, 132.3, 145.6, 149.0, 150.5, 159.5, 175.1; HRMS calcd for C<sub>27</sub>H<sub>43</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup> 487.3279; found 487.3286; HPLC purity: 95.7%.</div></div><div id="sec4_6_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(naphthalen-1-yl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7aaa</b></h4><div class="NLM_p last">1-((5-Methyl-2-(naphthalen-1-yl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (28 mg, 0.080 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (15 mg, 0.088 mmol) were reacted according to general procedure A-3 to afford the product as a white solid (21 mg, 0.041 mmol, 51% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.57 (q, <i>J</i> = 11.5 Hz, 1H), 0.88 (d, <i>J</i> = 6.4 Hz, 6H), 1.46 (t, <i>J</i> = 11.0 Hz, 2H), 1.69–1.93 (complex, 9H), 2.01–2.10 (m, 1H), 2.16 (dt, <i>J</i> = 3.0, 11.2 Hz, 2H), 2.46 (t, <i>J</i> = 6.2 Hz, 2H), 2.48 (s, 3H), 2.87–2.97 (m, 2H), 3.09–3.12 (m, 2H), 3.36 (q, <i>J</i> = 6.4 Hz, 2H), 3.59 (s, 2H), 7.34 (br s, 1H), 7.51–7.59 (m, 2H), 7.62–7.66 (m, 1H), 7.89–7.95 (m, 2H), 8.17 (dd, <i>J</i> = 1.2, 7.3 Hz, 1H), 9.24 (d, <i>J</i> = 8.7 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 19.6, 31.2, 43.5, 124.9, 126.1, 126.4, 127.3, 127.4, 128.4, 130.6; u (C, CH<sub>2</sub>): 24.8, 29.1, 39.7, 41.9, 53.1, 54.0, 58.0, 61.6, 124.4, 130.1, 132.8, 133.9, 145.9, 159.2, 174.9; HRMS calcd for C<sub>31</sub>H<sub>43</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 503.3381; found 503.3386; HPLC purity: >99.8%.</div></div><div id="sec4_6_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(2-methylnaphthalen-1-yl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>7bbb</b></h4><div class="NLM_p last">1-((5-Methyl-2-(2-methylnaphthalen-1-yl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.069 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (13 mg, 0.075 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (20 mg, 0.038 mmol, 55% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.56 (q, <i>J</i> = 11.8 Hz, 1H), 0.88 (d, <i>J</i> = 6.5 Hz, 6H), 1.44 (t, <i>J</i> = 11.0 Hz, 2H), 1.60–1.95 (complex, 9H), 2.01–2.10 (m, 1H), 2.17 (dt, <i>J</i> = 2.9, 11.4 Hz, 2H), 2.45 (t, <i>J</i> = 6.2 Hz, 2H), 2.461 (s, 3H), 2.464 (s, 3H), 2.88–2.92 (m, 2H), 3.08–3.13 (m, 2H), 3.36 (q, <i>J</i> = 6.3 Hz, 2H), 3.60 (s, 2H), 7.35 (br s, 1H), 7.40 (d, <i>J</i> = 8.4 Hz, 1H), 7.42–7.51 (m, 2H), 7.72–7.75 (m, 1H), 7.79–7.90 (m, 2H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.5, 19.6, 20.6, 31.3, 43.4, 125.1, 125.3, 126.9, 127.9, 128.4, 129.9; u (C, CH<sub>2</sub>): 24.8, 29.1, 39.8, 42.0, 53.1, 53.7, 58.2, 61.7, 124.7, 131.79, 131.83, 132.8, 137.0, 146.3, 158.1, 174.9; HRMS calcd for C<sub>32</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 517.3537; found 517.3535; HPLC purity: >99.8%.</div></div><div id="sec4_6_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 1-((2-(Anthracen-9-yl)-5-methyloxazol-4-yl)methyl)-<i>N</i>-(3-(cis-3,5-dimethylpiperidin-1-yl)propyl)piperidine-4-carboxamide <b>7ccc</b></h4><div class="NLM_p last">1-((2-(Anthracen-9-yl)-5-methyloxazol-4-yl)methyl)piperidine-4-carboxylic acid (25 mg, 0.062 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (12 mg, 0.069 mmol) were reacted according to general procedure A-3 to afford the product as a viscous, light yellow oil (16 mg, 0.029 mmol, 47% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.57 (q, <i>J</i> = 11.2 Hz, 1H), 0.88 (d, <i>J</i> = 6.4 Hz, 6H), 1.51 (t, <i>J</i> = 11.1 Hz, 2H), 1.69–1.98 (complex, 9H), 2.05–2.16 (m, 1H), 2.23 (dt, <i>J</i> = 3.0, 11.1 Hz, 2H), 2.50 (t, <i>J</i> = 6.2 Hz, 2H), 2.53 (s, 3H), 2.91–3.02 (m, 2H), 3.14–3.19 (m, 2H), 3.36 (q, <i>J</i> = 6.4 Hz, 2H), 3.68 (s, 2H), 7.36 (br s, 1H), 7.46–7.58 (complex, 4H), 7.95–8.03 (m, 2H), 8.03–8.11 (m, 2H), 8.59 (s, 3H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 10.6, 19.5, 30.9, 43.3, 125.4, 125.7, 126.9, 128.5, 129.8; u (C, CH<sub>2</sub>): 24.7, 29.0, 39.3, 41.7, 53.1, 53.9, 57.6, 61.3, 122.2, 131.1, 131.3, 132.3, 147.0, 157.8, 175.0; HRMS calcd for C<sub>35</sub>H<sub>45</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 553.3537; found 553.3533; HPLC purity: >99.8%.</div></div><div id="sec4_6_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((5-methyl-2-(<i>o</i>-tolyl)thiazol-4-yl)methyl)piperidine-4-carboxamide <b>13</b></h4><div class="NLM_p last">1-((5-Methyl-2-(<i>o</i>-tolyl)thiazol-4-yl)methyl)piperidine-4-carboxylic acid (56 mg, 0.17 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (29 mg, 0.17 mmol) were reacted according to general procedure A-1 to afford the product as a white solid (37 mg, 0.077 mmol, 45% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.57 (q, <i>J</i> = 12.3 Hz, 1H), 0.87 (d, <i>J</i> = 6.4 Hz, 6H), 1.51 (t, <i>J</i> = 11.0 Hz, 2H), 1.66–1.85 (complex, 9H), 1.98–2.00 (m, 1H), 2.13 (dt, <i>J</i> = 3.2, 11.1 Hz, 2H), 2.47–2.56 (m, 2H), 2.49 (s, 3H), 2.53 (s, 3H), 2.92–2.99 (m, 2H), 2.99–3.06 (m, 2H), 3.32 (q, <i>J</i> = 6.6 Hz, 2H), 3.68 (s, 2H), 7.20–7.30 (complex, 4H), 7.62 (d, <i>J</i> = 7.5 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 11.4, 19.5, 21.3, 30.9, 43.4, 125.9, 129.0, 129.8, 131.2; u (C, CH<sub>2</sub>): 24.7, 29.0, 39.2, 41.7, 53.0, 56.0, 57.5, 61.2, 131.1, 133.3, 136.3, 148.9, 163.0, 175.1; IR 1653, 2928, 2960 cm<sup>–1</sup>; HRMS calcd for C<sub>28</sub>H<sub>43</sub>N<sub>4</sub>OS [M + H]<sup>+</sup> 483.3152; found 483.3154; HPLC purity: 97.1%.</div></div><div id="sec4_6_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>N</i>-(3-(<i>cis</i>-3,5-Dimethylpiperidin-1-yl)propyl)-1-((2-(4-fluoro-3-methylphenyl)oxazol-4-yl)methyl)piperidine-4-carboxamide <b>18a</b></h4><div class="NLM_p last">1-((2-(4-Fluoro-3-methylphenyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (5.95 g, 18.69 mmol) and 3-(<i>cis</i>-3,5-dimethylpiperidin-1-yl)propan-1-amine (3.82 g, 22.43 mmol) were reacted according to general procedure A-3 to afford the product as a white solid (5.30 g, 11.26 mmol, 60% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.52 (q, <i>J</i> = 12.1 Hz, 1H), 0.84 (d, <i>J</i> = 6.5 Hz, 6H), 1.38 (t, <i>J</i> = 11.1 Hz, 2H), 1.57–1.84 (complex, 9H), 1.96–2.08 (m, 3H), 2.29 (s, 3H), 2.39 (t, <i>J</i> = 6.1 Hz, 2H), 2.82–2.88 (m, 2H), 2.98–3.05 (m, 2H), 3.30 (q, <i>J</i> = 6.0 Hz, 2H), 3.48 (s, 2H), 7.03 (t, <i>J</i> = 8.9 Hz, 1H), 7.31 (br s, 1H), 7.52 (s, 1H), 7.77–7.82 (m, 1H), 7.86–7.90 (m, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 14.4 (d, <i>J</i> = 3.5 Hz), 19.6, 31.3, 43.3, 115.4 (d, <i>J</i> = 23.3 Hz), 125.7 (d, <i>J</i> = 8.7 Hz), 129.8 (d, <i>J</i> = 5.7 Hz), 136.1; u (C, CH<sub>2</sub>): 24.8, 28.9, 39.8, 42.0, 53.2, 54.2, 58.1, 61.7, 123.5 (d, <i>J</i> = 3.6 Hz), 125.5 (d, <i>J</i> = 18.2 Hz), 139.0, 161.0 (d, <i>J</i> = 1.1 Hz), 162.6 (d, <i>J</i> = 250.8 Hz), 174.7; <sup>19</sup>F NMR 376 MHz, CDCl<sub>3</sub>) δ −114.1; IR 1549, 1630, 2944 cm<sup>–1</sup>; HRMS calcd for C<sub>27</sub>H<sub>40</sub>FN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 471.3130; found 471.3137; HPLC purity: 99.3%.</div></div><div id="sec4_6_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 1-((2-(4-Fluoro-3-methylphenyl)oxazol-4-yl)methyl)-<i>N</i>-(3-(4-isopropylpiperidin-1-yl)propyl)piperidine-4-carboxamide <b>18b</b></h4><div class="NLM_p last">1-((2-(4-Fluoro-3-methylphenyl)oxazol-4-yl)methyl)piperidine-4-carboxylic acid (40 mg, 0.13 mmol) and 3-(4-isopropylpiperidin-1-yl)propan-1-amine (46 mg, 0.25 mmol) were reacted according to general procedure A-3 to afford the product as a viscous light yellow oil (14 mg, 0.028 mmol, 22% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.80 (d, <i>J</i> = 6.8 Hz, 6H), 0.96–1.06 (m, 1H), 1.24 (dq, <i>J</i> = 2.9, 12.2 Hz, 2H), 1.57–1.84 (complex, 9H), 1.96–2.08 (m, 3H), 2.29 (s, 3H), 2.39 (t, <i>J</i> = 6.1 Hz, 2H), 1.38 (sextet, <i>J</i> = 6.6 Hz, 1H), 1.59–1.68 (complex, 4H), 1.69–1.87 (complex, 5H), 1.97–2.08 (m, 3H), 2.28 (s, 3H), 2.41 (t, <i>J</i> = 6.0 Hz, 2H), 2.94–3.04 (m, 4H), 3.30 (q, <i>J</i> = 5.7 Hz, 2H), 3.47 (s, 2H), 7.02 (t, <i>J</i> = 8.9 Hz, 1H), 7.51 (s, 1H), 7.55 (br s, 1H), 7.75–7.81 (m, 1H), 7.87 (d, <i>J</i> = 7.1 Hz, 1H); <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>, APT pulse sequence) δ d (CH, CH<sub>3</sub>): 14.4 (d, <i>J</i> = 3.5 Hz), 19.6, 32.3, 42.1, 43.3, 115.4 (d, <i>J</i> = 23.3 Hz), 125.7 (d, <i>J</i> = 8.7 Hz), 129.8 (d, <i>J</i> = 5.8 Hz), 136.1; u (C, CH<sub>2</sub>): 24.5, 29.0, 29.1, 39.9, 53.3, 54.2, 54.4, 58.2, 123.5 (d, <i>J</i> = 3.5 Hz), 125.5 (d, <i>J</i> = 18.2 Hz), 138.9, 161.0 (d, <i>J</i> = 1.0 Hz), 162.5 (d, <i>J</i> = 250.7 Hz), 174.8; <sup>19</sup>F NMR 376 MHz, CDCl<sub>3</sub>) δ −114.1; HRMS calcd for C<sub>28</sub>H<sub>42</sub>FN<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup> 485.3286; found 485.3263; HPLC purity: 96.5%.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Cells and Viruses</h3><div class="NLM_p last">All the cells were maintained in Dulbecco’s modified Eagle’s medium (DMEM) (Life Technologies, Grand Island, NY, United States) with 10% fetal bovine serum (FBS) (Life Technologies) and antibiotics in 5% CO<sub>2</sub> at 37 °C. HCV-Luc was made through insertion of luciferase reporter gene in the HCV JFH-1 strain. Single-round infectious defective HCV particles (HCVsc) and pseudotyped viruses (HCVpp-1a, HCVpp-1b, and VSV-Gpp) were produced as reported before.<a onclick="showRef(event, 'ref7 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref10">(7, 10)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> HCV-Luc Infection and ATPlite Cytotoxicity Assays</h3><div class="NLM_p last">Huh7.5.1 cells seeded in 96-well plates (10<sup>4</sup> cells/well) were cultured overnight. HCV-Luc virus was used to infect the cells in the presence of titration of the compound of interest. Viral inhibition was evaluated using Renilla Luciferase assay system (Promega, Madison, WI, United States) 48 h post treatment. Cytotoxicity of each compound was measured in parallel using ATPlite assay kit (PerkinElmer, Waltham, MA, United States). EC<sub>50</sub> and CC<sub>50</sub> values were calculated by nonlinear regression equation in GraphPad Prism 5.0 software (GraphPad Software Inc., La Jolla, CA, United States).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> HCV Replication Cycle Assays</h3><div class="NLM_p last">In HCV single-cycle infection assay, Huh7.5.1 cells seeded in 96-well plates (10<sup>4</sup> cells/well) were cultured overnight. The cells were inoculated with the infectious HCVsc together with the tested compounds. Luciferase activity of the cells was measured 48 h after the compound treatment. In transient replicon assay, Huh7.5.1 cells seeded in 96-well plates (10<sup>4</sup> cells/well) were cultured overnight. Then, the cells were transiently transfected with the replicon RNA transcript with DMRIE-C for 4 h. After the transfection reagent was removed, the cells were incubated with DMEM culture medium containing 10 μM of each compound for 48 h. Luciferase activity was measured. In HCV subgenomic replicon assay with HCV replicon (GT 2a) cells, cells were plated into 96-well plate (10<sup>4</sup> cells/well) and incubated overnight. The cells were treated with tested compounds. Luciferase activity was measured 48 h after the compound treatment. In HCVpp assays, Huh7.5.1 cells were seeded in 96-well plates (10<sup>4</sup> cells/well) and cultured overnight. Then, the cells were treated with 10 μM of the compounds together with infection of HCVpp GT 1a or VSVpp for 4 h. The cells were then washed and cultured for 48 h followed by a luciferase assay to detect the HCV entry. The results shown are the means of five replicates ± SEM.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Compound Binding Affinity in CNS-Relevant Receptor Panel</h3><div class="NLM_p last">Radioligand binding assays using cloned GPCRs, ion channels, and transporters were performed from transiently transfected or stable cell lines as previously detailed<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> through the resources of the National Institute of Mental Health Psychoactive Drug Screening Program. Detailed protocols (including cell handling, buffer composition, assay conditions, etc.) for all assays are available online (<a href="http://pdspdb.unc.edu/pdspWeb/content/PDSP%20Protocols%20II%202013-03-28.pdf" class="extLink">http://pdspdb.unc.edu/pdspWeb/content/PDSP%20Protocols%20II%202013-03-28.pdf</a>). Initial screening assays were performed using 10 μM (final concentration) of test compound, and the percent inhibition of specific binding by the test compound was determined. When the test compound inhibited >50% of radioligand specific binding, <i>K</i><sub><i>i</i></sub> determinations were performed by measuring the inhibition of radioligand binding by various concentrations of test compound (11 concentrations spanning 6 orders of magnitude). Radioligand binding isotherms were regressed using the One Site competition binding function built into GraphPad Prism 4.0 to estimate compound IC<sub>50</sub> values. Affinity constants (<i>K</i><sub><i>i</i></sub> values) were calculated from IC<sub>50</sub> values using the Cheng–Prusoff approximation.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> In Vitro Combination Test</h3><div class="NLM_p last">HCV-Luc infection and ATPlite assays were carried out in 96-well plates in the presences of the compound of interest titrated in vertical and the known antiviral drug titrated in horizontal.<a onclick="showRef(event, 'ref13 ref22'); return false;" href="javascript:void(0);" class="ref ref13 ref22">(13, 22)</a> The known antiviral drugs include ribavirin (Sigma-Aldrich), sofosbuvir (Advanced Chemblocks), telaprevir (Selleckchem), daclatasvir (Selleckchem), cyclosporin A (Sigma-Aldrich), and boceprevir (ChemScene). Two independent mathematical models, the Bliss independence model and the Loewe additivity model, were used to predict the theoretical additive, synergistic, or antagonistic effects. By the Bliss independence model, log volumes of synergistic or antagonistic effect were calculated with the MacSynergy program. By the Loewe additivity model, combination indices were calculated at or near the EC<sub>50</sub> values of the compound and the antiviral drug when tested alone with CalcuSyn program (Biosoft).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Antiviral Screen</h3><div class="NLM_p last">The antiviral activity of compound of interest was screened against 13 viruses, including hepatitis B virus, HCV replicon, herpes simplex virus-1, human cytomegalovirus, vaccinia virus, dengue virus, influenza A (H1N1) virus, respiratory syncytial virus, SARS coronavirus, poliovirus 3, Rift Valley fever virus, Tacaribe virus, and Venezuelan equine encephalitis virus, using the nonclinical and preclinical services program offered by the National Institute of Allergy and Infectious Diseases (NIAID) (<a href="http://www.niaid.nih.gov/labsandresources/resources/dmid/invitro/Pages/invitro.aspx" class="extLink">https://www.niaid.nih.gov/research/vitro-assessment-antimicrobial-activity-resources-niaid</a>).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> In Vitro Biopharmaceutical and in Vivo Pharmacokinetic Properties</h3><div class="NLM_p last">In vitro biopharmaceutical properties were measured with a fully automated system for sample preparation, sample analysis, and data processing. The microsomal stability of compounds was estimated in rat liver microsomes at 37 °C in the presence of the cofactor NADPH. Concentration of a test article was measured by LC–MS/MS, and half-life (<i>t</i><sub>1/2</sub>) was calculated as described before.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The solubility of compounds was determined in phosphate buffer (pH 7.4) using μSOL Evolution (Pion Inc., Billerica, MA, United States). The permeability of compounds was estimated via passive diffusion using stirring double-sink parallel artificial membrane permeability assay (PAMPA) (Pion Inc.). The in vivo pharmacokinetics properties of compounds of interest were evaluated in male CD-1 mice after single intraperitoneal administration. Adult male CD-1 mice (25–39 g, <i>n</i> = 3/sampling time point) were obtained from Charles River Laboratories (Wilmington, MA). Mice were housed at the centralized animal facilities at the NIH (Bethesda, MD) with a 12 h light-dark cycle. The housing temperature and relative humidity were controlled at 22 °C and 55%, respectively. The animals had free access to water and food. All experimental procedures were approved by the Animal Care and Use Committee (ACUC) of the NIH Division of Veterinary Resources (DVR). A single dose of 0.1, 1, and 10 mg/kg was administered through i.p. route of administration. The concentration of compounds in the plasma and liver samples was measured by the ultraperformance liquid chromatography–mass spectrometry analysis (UPLC−MS/MS).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-5"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00561">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15415" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15415" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00561" class="ext-link">10.1021/acs.jmedchem.7b00561</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Tables showing all synthetic intermediates, synthetic procedures, compound characterization for synthetic intermediates, and NIAID antiviral screen data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (SMILES) for final analogues (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_001.pdf">jm7b00561_si_001.pdf (771.48 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_002.csv">jm7b00561_si_002.csv (6.14 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00561" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35828" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35828" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">T. Jake Liang</span> - <span class="hlFld-Affiliation affiliation">Liver
Diseases Branch, National Institute of Diabetes and Digestive and
Kidney Diseases, National Institutes of
Health, 10 Center Drive, Bethesda, Maryland 20892-1800, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#91fbf0faf4fdd1f3f5f6a0a1bffff8f5f5fabffff8f9bff6fee7"><span class="__cf_email__" data-cfemail="83e9e2e8e6efc3e1e7e4b2b3adedeae7e7e8adedeaebade4ecf5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin J. Frankowski</span> - <span class="hlFld-Affiliation affiliation">University
of Kansas Specialized Chemistry Center, University of Kansas, Lawrence, Kansas 66047, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7173-6352" title="Orcid link">http://orcid.org/0000-0002-7173-6352</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#761d13001f18103603181558131203"><span class="__cf_email__" data-cfemail="503b3526393e3610253e337e353425">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shanshan He</span> - <span class="hlFld-Affiliation affiliation">Liver
Diseases Branch, National Institute of Diabetes and Digestive and
Kidney Diseases, National Institutes of
Health, 10 Center Drive, Bethesda, Maryland 20892-1800, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kelin Li</span> - <span class="hlFld-Affiliation affiliation">University
of Kansas Specialized Chemistry Center, University of Kansas, Lawrence, Kansas 66047, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Billy Lin</span> - <span class="hlFld-Affiliation affiliation">Liver
Diseases Branch, National Institute of Diabetes and Digestive and
Kidney Diseases, National Institutes of
Health, 10 Center Drive, Bethesda, Maryland 20892-1800, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zongyi Hu</span> - <span class="hlFld-Affiliation affiliation">Liver
Diseases Branch, National Institute of Diabetes and Digestive and
Kidney Diseases, National Institutes of
Health, 10 Center Drive, Bethesda, Maryland 20892-1800, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jingbo Xiao</span> - <span class="hlFld-Affiliation affiliation">Division
of Preclinical Innovation, National Center
for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Hu</span> - <span class="hlFld-Affiliation affiliation">Division
of Preclinical Innovation, National Center
for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amy Q. Wang</span> - <span class="hlFld-Affiliation affiliation">Division
of Preclinical Innovation, National Center
for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Xu</span> - <span class="hlFld-Affiliation affiliation">Division
of Preclinical Innovation, National Center
for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marc Ferrer</span> - <span class="hlFld-Affiliation affiliation">Division
of Preclinical Innovation, National Center
for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Noel Southall</span> - <span class="hlFld-Affiliation affiliation">Division
of Preclinical Innovation, National Center
for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Zheng</span> - <span class="hlFld-Affiliation affiliation">Division
of Preclinical Innovation, National Center
for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey Aubé</span> - <span class="hlFld-Affiliation affiliation">University
of Kansas Specialized Chemistry Center, University of Kansas, Lawrence, Kansas 66047, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1049-5767" title="Orcid link">http://orcid.org/0000-0003-1049-5767</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Frank J. Schoenen</span> - <span class="hlFld-Affiliation affiliation">University
of Kansas Specialized Chemistry Center, University of Kansas, Lawrence, Kansas 66047, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Juan J. Marugan</span> - <span class="hlFld-Affiliation affiliation">Division
of Preclinical Innovation, National Center
for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): T.J.L., S.H., X.H., Z.H., J.J.M., N.S., J.X., M.F., W.Z., K.J.F., K.L., and F.J.S. are named as inventors on U.S. provisional patent application (no. 62/011,462; Heterocyclic Compounds and Methods of Use Thereof) and international patent application (no. WO 2015192077A1; Preparation of Heterocyclic Compounds Useful in the Treatment of Diseases) related to this work. Other authors declare no competing interests.<br /></br></div></li></ul></div><div class="ack" id="ACK-d152e6211-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i94">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23780" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23780" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank S. Michael and M. Balcom for assistance in robotic operation in qHTS; P. Shinn, M. Itkin, and D. van Leer for help with compound management; B. Zhang for technical assistance; Benjamin Neuenswander for compound purification and analysis; and Patrick Porubsky for compound management. Receptor binding profile was generously provided by the National Institute of Mental Health’s Psychoactive Drug Screening Program, Contract HHSN-271-2013-00017-C (NIMH PDSP). The NIMH PDSP is directed by Bryan L. Roth MD, Ph.D at the University of North Carolina at Chapel Hill and Project Officer Jamie Driscoll at NIMH, Bethesda, MD, United States. This research was funded by the Intramural Research Program of the National Institute of Diabetes and Digestive and Kidney Diseases, the Intramural Research Program of the National Center for Advancing Translational Sciences, and Molecular Libraries Initiative funding to the University of Kansas Specialized Chemistry Center (U54HG005031).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i95" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i95"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i96" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i96"> Abbreviations Used</h2><tr><td class="NLM_term">HCV</td><td class="NLM_def"><p class="first last">hepatitis C virus</p></td></tr><tr><td class="NLM_term">IFN-α</td><td class="NLM_def"><p class="first last">interferon α</p></td></tr><tr><td class="NLM_term">RBV</td><td class="NLM_def"><p class="first last">ribavirin</p></td></tr><tr><td class="NLM_term">DAAs</td><td class="NLM_def"><p class="first last">direct-acting antivirals</p></td></tr><tr><td class="NLM_term">HTAs</td><td class="NLM_def"><p class="first last">host-targeting agents</p></td></tr><tr><td class="NLM_term">qHTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">HCV-Luc</td><td class="NLM_def"><p class="first last">HCV JFH-1 strain with insertion of the luciferase reporter gene</p></td></tr><tr><td class="NLM_term">EC<sub>50</sub></td><td class="NLM_def"><p class="first last">the concentration of compound that inhibited 50% of virus level of DMSO</p></td></tr><tr><td class="NLM_term">CC<sub>50</sub></td><td class="NLM_def"><p class="first last">the concentration of compound that exhibited 50% of cytotoxicity of DMSO</p></td></tr><tr><td class="NLM_term">HCVsc</td><td class="NLM_def"><p class="first last">single-round infectious defective HCV particles</p></td></tr><tr><td class="NLM_term">HCVpp</td><td class="NLM_def"><p class="first last">HCV pseudoparticle</p></td></tr><tr><td class="NLM_term">RLU</td><td class="NLM_def"><p class="first last">relative luminescence units</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and excretion</p></td></tr><tr><td class="NLM_term">i.p</td><td class="NLM_def"><p class="first last">intraperitoneal</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">LC–MS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">TOF</td><td class="NLM_def"><p class="first last">time-of-flight</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">NCS</td><td class="NLM_def"><p class="first last"><i>N</i>-chlorosuccinimide</p></td></tr><tr><td class="NLM_term">SFC</td><td class="NLM_def"><p class="first last">supercritical fluid chromatography</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">NIAID</td><td class="NLM_def"><p class="first last">National Institute of Allergy and Infectious Diseases</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64015" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 23 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Te, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, D. M.</span><span> </span><span class="NLM_article-title">Epidemiology of hepatitis B and C viruses: a global overview</span> <span class="citation_source-journal">Clin. Liver Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><div class="note"><p class="first last">vii.</p></div><span class="refDoi"> DOI: 10.1016/j.cld.2009.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1016%2Fj.cld.2009.11.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=1-21&author=H.+S.+Teauthor=D.+M.+Jensen&title=Epidemiology+of+hepatitis+B+and+C+viruses%3A+a+global+overview&doi=10.1016%2Fj.cld.2009.11.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cld.2009.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cld.2009.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DTe%26aufirst%3DH.%2BS.%26aulast%3DJensen%26aufirst%3DD.%2BM.%26atitle%3DEpidemiology%2520of%2520hepatitis%2520B%2520and%2520C%2520viruses%253A%2520a%2520global%2520overview%26jtitle%3DClin.%2520Liver%2520Dis.%26date%3D2010%26volume%3D14%26spage%3D1%26epage%3D21%26doi%3D10.1016%2Fj.cld.2009.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Ferenci, P.</span><span> </span><span class="NLM_article-title">Treatment of hepatitis C in difficult-to-treat patients</span> <span class="citation_source-journal">Nat. Rev. Gastroenterol. Hepatol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">–</span> <span class="NLM_lpage">292</span><span class="refDoi"> DOI: 10.1038/nrgastro.2015.53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1038%2Fnrgastro.2015.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=25895822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotVeqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=284-292&author=P.+Ferenci&title=Treatment+of+hepatitis+C+in+difficult-to-treat+patients&doi=10.1038%2Fnrgastro.2015.53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of hepatitis C in difficult-to-treat patients</span></div><div class="casAuthors">Ferenci, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Gastroenterology & Hepatology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">284-292</span>CODEN:
                <span class="NLM_cas:coden">NRGHA9</span>;
        ISSN:<span class="NLM_cas:issn">1759-5045</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interferon-free regimes are now the treatment of choice for patients with chronic hepatitis C; previously patients who were 'difficult-to-treat' using interferon-contg. treatments can now safely be treated with such therapies.  More than 90% of patients infected with HCV genotype 1 or 4, compensated cirrhosis, or who have had liver transplantation, can be cured with the use of sofosbuvir combined with simeprevir, daclatasvir or ledipasvir, or by the combination of paritaprevir with ritonavir, ombitasvir and with or without dasabuvir.  Addn. of ribavirin seems to shorten treatment duration.  However, the safety of these drugs is not fully explored in patients with decompensated cirrhosis (i.e., those with Child-Pugh class C disease), and protease inhibitors should not be used in this group.  The optimal use of interferon-free regimes in patients with renal failure or after kidney transplantation is currently being studied.  However, new and improved drugs are needed to treat patients infected with HCV genotype 3.  Unfortunately, the broad application of new HCV treatments is limited by their high costs.  In this Review, I discuss the treatment of patients with hepatitis C with compensated and decompensated cirrhosis, before and after orthotopic liver transplantation and in patients with impaired kidney function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorG-2DpipDwrVg90H21EOLACvtfcHk0lhq31T-NeFmhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotVeqtbs%253D&md5=52ec6909ccbf900e58393267c212de73</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrgastro.2015.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrgastro.2015.53%26sid%3Dliteratum%253Aachs%26aulast%3DFerenci%26aufirst%3DP.%26atitle%3DTreatment%2520of%2520hepatitis%2520C%2520in%2520difficult-to-treat%2520patients%26jtitle%3DNat.%2520Rev.%2520Gastroenterol.%2520Hepatol.%26date%3D2015%26volume%3D12%26spage%3D284%26epage%3D292%26doi%3D10.1038%2Fnrgastro.2015.53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Liang, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehermann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeff, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoofnagle, J. H.</span><span> </span><span class="NLM_article-title">Pathogenesis, natural history, treatment, and prevention of hepatitis C</span> <span class="citation_source-journal">Ann. Intern. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">296</span><span class="NLM_x">–</span> <span class="NLM_lpage">305</span><span class="refDoi"> DOI: 10.7326/0003-4819-132-4-200002150-00008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.7326%2F0003-4819-132-4-200002150-00008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10681285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtlyku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2000&pages=296-305&author=T.+J.+Liangauthor=B.+Rehermannauthor=L.+B.+Seeffauthor=J.+H.+Hoofnagle&title=Pathogenesis%2C+natural+history%2C+treatment%2C+and+prevention+of+hepatitis+C&doi=10.7326%2F0003-4819-132-4-200002150-00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Pathogenesis, natural history, treatment, and prevention of hepatitis c</span></div><div class="casAuthors">Liang, T. Jake; Rehermann, Barbara; Seeff, Leonard B.; Hoofnagle, Jay H.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Internal Medicine</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">296-305</span>CODEN:
                <span class="NLM_cas:coden">AIMEAS</span>;
        ISSN:<span class="NLM_cas:issn">0003-4819</span>.
    
            (<span class="NLM_cas:orgname">American College of Physicians-American Society of Internal Medicine</span>)
        </div><div class="casAbstract">A review with 77 refs.  Approx. 4 million persons in the United States and probably more than 100 million persons worldwide are infected with hepatitis C virus.  The virus has the unique ability to cause persistent infection in susceptible hosts after parenteral or percutaneous transmission, and its underlying mechanisms are not well understood.  The immunol. correlates of protection and viral clearance and the pathogenesis of liver injury are yet to be defined, but recent studies suggest the importance of cell-mediated immune responses.  Although 70% to 80% of infected persons become chronic carriers, most have relatively mild disease with slow progression.  However, chronic and progressive hepatitis C carries significant morbidity and mortality and is a major cause of cirrhosis, end-stage liver disease, and liver cancer.  Development of an effective hepatitis C virus vaccine is not imminent, but recent advances in technol. and basic knowledge of mol. virol. and immunol. have engendered novel approaches to the fundamental problems encountered in vaccine development.  Current therapy for hepatitis C, although effective in some patients, is problematic and still evolving.  Advances in modern biol. and immunol. promise new therapies for this important disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiCUCgLD1TwrVg90H21EOLACvtfcHk0lhq31T-NeFmhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtlyku7g%253D&md5=5c5d04abdc4611c544258fb84caaae6d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.7326%2F0003-4819-132-4-200002150-00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7326%252F0003-4819-132-4-200002150-00008%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DRehermann%26aufirst%3DB.%26aulast%3DSeeff%26aufirst%3DL.%2BB.%26aulast%3DHoofnagle%26aufirst%3DJ.%2BH.%26atitle%3DPathogenesis%252C%2520natural%2520history%252C%2520treatment%252C%2520and%2520prevention%2520of%2520hepatitis%2520C%26jtitle%3DAnn.%2520Intern.%2520Med.%26date%3D2000%26volume%3D132%26spage%3D296%26epage%3D305%26doi%3D10.7326%2F0003-4819-132-4-200002150-00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Liang, T. J.</span><span> </span><span class="NLM_article-title">Current progress in development of hepatitis C virus vaccines</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">–</span> <span class="NLM_lpage">878</span><span class="refDoi"> DOI: 10.1038/nm.3183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1038%2Fnm.3183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=23836237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKqsrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=869-878&author=T.+J.+Liang&title=Current+progress+in+development+of+hepatitis+C+virus+vaccines&doi=10.1038%2Fnm.3183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Current progress in development of hepatitis C virus vaccines</span></div><div class="casAuthors">Liang, T. Jake</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">869-878</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite major advances in the understanding and treatment of hepatitis C, a preventive vaccine remains elusive.  The marked genetic diversity and multiple mechanisms of persistence of hepatitis C virus, combined with the relatively poor immune response of the infected host against the virus, are major barriers.  The lack of robust and convenient model systems further hampers the effort to develop an effective vaccine.  Advances in our understanding of virus-host interactions and protective immunity in hepatitis C virus infection provide an important roadmap to develop potent and broadly directed vaccine candidates targeting both humoral and cellular immune responses.  Multiple approaches to generating and testing viral immunogens have met with variable success.  Several candidates have advanced to clin. trials based on promising results in chimpanzees.  The ultimate path to a successful preventive vaccine requires comprehensive evaluations of all aspects of protective immunity, innovative application of state-of-the-art vaccine technol. and properly designed vaccine trials that can affirm definitive endpoints of efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqap5_3MnE0MLVg90H21EOLACvtfcHk0liYoSxEk9roPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKqsrbO&md5=c3d2c522cc0fc2c078a92d1fcf463b55</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnm.3183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3183%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26atitle%3DCurrent%2520progress%2520in%2520development%2520of%2520hepatitis%2520C%2520virus%2520vaccines%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D869%26epage%3D878%26doi%3D10.1038%2Fnm.3183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Liang, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghany, M. G.</span><span> </span><span class="NLM_article-title">Current and future therapies for hepatitis C virus infection</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">368</span><span class="NLM_x">, </span> <span class="NLM_fpage">1907</span><span class="NLM_x">–</span> <span class="NLM_lpage">1917</span><span class="refDoi"> DOI: 10.1056/NEJMra1213651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1056%2FNEJMra1213651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=23675659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsl2gs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2013&pages=1907-1917&author=T.+J.+Liangauthor=M.+G.+Ghany&title=Current+and+future+therapies+for+hepatitis+C+virus+infection&doi=10.1056%2FNEJMra1213651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Current and future therapies for hepatitis C virus infection</span></div><div class="casAuthors">Liang, T. Jake; Ghany, Marc G.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1907-1917</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  The recent approval of two direct-acting antiviral agents for hepatitis C viral (HCV) infection that specifically inhibit viral replication has dramatically increased the viral clearance rate.  It is anticipated that in the course of such a screening process, a large no. of persons will be found to be infected with HCV; this article reviews the current therapy for HCV infection and the landscape of drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4zjjmKeKrZrVg90H21EOLACvtfcHk0liYoSxEk9roPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsl2gs78%253D&md5=6be058d0298a4441a180ac076d7293aa</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1213651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1213651%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DGhany%26aufirst%3DM.%2BG.%26atitle%3DCurrent%2520and%2520future%2520therapies%2520for%2520hepatitis%2520C%2520virus%2520infection%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D368%26spage%3D1907%26epage%3D1917%26doi%3D10.1056%2FNEJMra1213651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Callaway, E.</span><span> </span><span class="NLM_article-title">Hepatitis C drugs not reaching poor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">508</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">296</span><span class="refDoi"> DOI: 10.1038/508295a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1038%2F508295a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=295-296&author=E.+Callaway&title=Hepatitis+C+drugs+not+reaching+poor&doi=10.1038%2F508295a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F508295a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F508295a%26sid%3Dliteratum%253Aachs%26aulast%3DCallaway%26aufirst%3DE.%26atitle%3DHepatitis%2520C%2520drugs%2520not%2520reaching%2520poor%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D295%26epage%3D296%26doi%3D10.1038%2F508295a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Hu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaroop, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span> </span><span class="NLM_article-title">Novel cell-based hepatitis C virus infection assay for quantitative high-throughput screening of anti-hepatitis C virus compounds</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">995</span><span class="NLM_x">–</span> <span class="NLM_lpage">1004</span><span class="refDoi"> DOI: 10.1128/AAC.02094-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1128%2FAAC.02094-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=24277038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVSrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=995-1004&author=Z.+Huauthor=K.+H.+Lanauthor=S.+Heauthor=M.+Swaroopauthor=X.+Huauthor=N.+Southallauthor=W.+Zhengauthor=T.+J.+Liang&title=Novel+cell-based+hepatitis+C+virus+infection+assay+for+quantitative+high-throughput+screening+of+anti-hepatitis+C+virus+compounds&doi=10.1128%2FAAC.02094-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Novel cell-based hepatitis C virus infection assay for quantitative high-throughput screening of anti-hepatitis C virus compounds</span></div><div class="casAuthors">Hu, Zongyi; Lan, Keng-Hsin; He, Shanshan; Swaroop, Manju; Hu, Xin; Southall, Noel; Zheng, Wei; Liang, T. Jake</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">995-1004, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Therapy for hepatitis C virus (HCV) infection has advanced with the recent approval of direct-acting antivirals in combination with peginterferon and ribavirin.  New antivirals with novel targets are still needed to further improve the treatment of hepatitis C.  Previously reported screening methods for HCV inhibitors either are limited to a virus-specific function or apply a screening method at a single dose, which usually leads to high false-pos. or -neg. rates.  We developed a quant. high-throughput screening (qHTS) assay platform with a cell-based HCV infection system.  This highly sensitive assay can be miniaturized to a 1,536-well format for screening of large chem. libraries.  All candidates are screened over a 7-concn. dose range to give EC50s (compd. concns. at 50% efficacy) and dose-response curves.  Using this assay format, we screened a library of pharmacol. active compds. (LOPAC).  Based on the profile of dose-dependent curves of HCV inhibition and cytotoxicity, 22 compds. with adequate curves and EC50s of <10 μM were selected for validation.  In two addnl. independent assays, 17 of them demonstrated specific inhibition of HCV infection.  Ten potential candidates with efficacies of >70% and CC50s (compd. concns. at 50% cytotoxicity) of <30 μM from these validated hits were characterized for their target stages in the HCV replication cycle.  In this screen, we identified both known and novel hits with diverse structural and functional features targeting various stages of the HCV replication cycle.  The pilot screen demonstrates that this assay system is highly robust and effective in identifying novel HCV inhibitors and that it can be readily applied to large-scale screening of small-mol. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8MWc_pTB2cbVg90H21EOLACvtfcHk0liUs2UIpXab4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVSrtrg%253D&md5=b0fd6fc3af44e8f211cc13ae7c33a408</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FAAC.02094-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02094-13%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DLan%26aufirst%3DK.%2BH.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSwaroop%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26atitle%3DNovel%2520cell-based%2520hepatitis%2520C%2520virus%2520infection%2520assay%2520for%2520quantitative%2520high-throughput%2520screening%2520of%2520anti-hepatitis%2520C%2520virus%2520compounds%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D995%26epage%3D1004%26doi%3D10.1128%2FAAC.02094-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Hu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yim, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaroop, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanega, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kao, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marugan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span> </span><span class="NLM_article-title">Identification of novel anti-hepatitis C virus agents by a quantitative high throughput screen in a cell-based infection assay</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2015.10.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1016%2Fj.antiviral.2015.10.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=26515788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslyqsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2015&pages=20-29&author=Z.+Huauthor=X.+Huauthor=S.+Heauthor=H.+J.+Yimauthor=J.+Xiaoauthor=M.+Swaroopauthor=C.+Tanegaauthor=Y.+Q.+Zhangauthor=G.+Yiauthor=C.+C.+Kaoauthor=J.+Maruganauthor=M.+Ferrerauthor=W.+Zhengauthor=N.+Southallauthor=T.+J.+Liang&title=Identification+of+novel+anti-hepatitis+C+virus+agents+by+a+quantitative+high+throughput+screen+in+a+cell-based+infection+assay&doi=10.1016%2Fj.antiviral.2015.10.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel anti-hepatitis C virus agents by a quantitative high throughput screen in a cell-based infection assay</span></div><div class="casAuthors">Hu, Zongyi; Hu, Xin; He, Shanshan; Yim, Hyung Joon; Xiao, Jingbo; Swaroop, Manju; Tanega, Cordelle; Zhang, Ya-qin; Yi, Guanghui; Kao, C. Cheng; Marugan, Juan; Ferrer, Marc; Zheng, Wei; Southall, Noel; Liang, T. Jake</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20-29</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) poses a major health threat to the world.  The recent development of direct-acting antivirals (DAAs) against HCV has markedly improved the response rate of HCV and reduced the side effects in comparison to the interferon-based therapy.  Despite this therapeutic advance, there is still a need to develop new inhibitors that target different stages of the HCV life cycle because of various limitations of the current regimens.  In this study, we performed a quant. high throughput screening of the Mol. Libraries Small Mol. Repository (MLSMR) of ∼350,000 chems. for novel HCV inhibitors using our previously developed cell-based HCV infection assay.  Following confirmation and structural clustering anal., we narrowed down to 158 compds. from the initial ∼3000 mols. that showed inhibitory activity for further structural and functional analyses.  We were able to assign the majority of these compds. to specific stage(s) in the HCV life cycle.  Three of them are direct inhibitors of NS3/4A protease.  Most of the compds. appear to act on novel targets in HCV life cycle.  Four compds. with novel structure and excellent drug-like properties, three targeting HCV entry and one targeting HCV assembly/secretion, were advanced for further development as lead hits.  These compds. represent diverse chemotypes that are potential lead compds. for further optimization and may offer promising candidates for the development of novel therapeutics against HCV infection.  In addn., they represent novel mol. probes to explore the complex interactions between HCV and the cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnosKyCpJzeLVg90H21EOLACvtfcHk0liUs2UIpXab4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslyqsbjL&md5=d58abbeb96d510e5246929ce171772b5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2015.10.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2015.10.018%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DYim%26aufirst%3DH.%2BJ.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DSwaroop%26aufirst%3DM.%26aulast%3DTanega%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%2BQ.%26aulast%3DYi%26aufirst%3DG.%26aulast%3DKao%26aufirst%3DC.%2BC.%26aulast%3DMarugan%26aufirst%3DJ.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26atitle%3DIdentification%2520of%2520novel%2520anti-hepatitis%2520C%2520virus%2520agents%2520by%2520a%2520quantitative%2520high%2520throughput%2520screen%2520in%2520a%2520cell-based%2520infection%2520assay%26jtitle%3DAntiviral%2520Res.%26date%3D2015%26volume%3D124%26spage%3D20%26epage%3D29%26doi%3D10.1016%2Fj.antiviral.2015.10.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Goto, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamana, M.</span><span> </span><span class="NLM_article-title">Studies on azole compounds. III. Reactions of oxazole N-oxides with phosphoryl chloride and acetic anhydride</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2050</span><span class="NLM_x">–</span> <span class="NLM_lpage">2057</span><span class="refDoi"> DOI: 10.1248/cpb.19.2050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1248%2Fcpb.19.2050" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1971&pages=2050-2057&author=Y.+Y.+Gotoauthor=M.+Hamana&title=Studies+on+azole+compounds.+III.+Reactions+of+oxazole+N-oxides+with+phosphoryl+chloride+and+acetic+anhydride&doi=10.1248%2Fcpb.19.2050"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1248%2Fcpb.19.2050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.19.2050%26sid%3Dliteratum%253Aachs%26aulast%3DGoto%26aufirst%3DY.%2BY.%26aulast%3DHamana%26aufirst%3DM.%26atitle%3DStudies%2520on%2520azole%2520compounds.%2520III.%2520Reactions%2520of%2520oxazole%2520N-oxides%2520with%2520phosphoryl%2520chloride%2520and%2520acetic%2520anhydride%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1971%26volume%3D19%26spage%3D2050%26epage%3D2057%26doi%3D10.1248%2Fcpb.19.2050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Hsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flint, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logvinoff, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng-Mayer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeating, J. A.</span><span> </span><span class="NLM_article-title">Hepatitis C virus glycoproteins mediate pH-dependent cell entry of pseudotyped retroviral particles</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">7271</span><span class="NLM_x">–</span> <span class="NLM_lpage">7276</span><span class="refDoi"> DOI: 10.1073/pnas.0832180100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1073%2Fpnas.0832180100" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=7271-7276&author=M.+Hsuauthor=J.+Zhangauthor=M.+Flintauthor=C.+Logvinoffauthor=C.+Cheng-Mayerauthor=C.+M.+Riceauthor=J.+A.+McKeating&title=Hepatitis+C+virus+glycoproteins+mediate+pH-dependent+cell+entry+of+pseudotyped+retroviral+particles&doi=10.1073%2Fpnas.0832180100"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0832180100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0832180100%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFlint%26aufirst%3DM.%26aulast%3DLogvinoff%26aufirst%3DC.%26aulast%3DCheng-Mayer%26aufirst%3DC.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DMcKeating%26aufirst%3DJ.%2BA.%26atitle%3DHepatitis%2520C%2520virus%2520glycoproteins%2520mediate%2520pH-dependent%2520cell%2520entry%2520of%2520pseudotyped%2520retroviral%2520particles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D7271%26epage%3D7276%26doi%3D10.1073%2Fpnas.0832180100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Fitzgerald, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burn, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patterson, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corjay, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valentine, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Link, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbaszade, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollis, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Largent, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartig, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollis, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meunier, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robichaud, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. W.</span><span> </span><span class="NLM_article-title">Possible role of valvular serotonin 5-HT(2B) receptors in the cardiopathy associated with fenfluramine</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10617681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhslCltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2000&pages=75-81&author=L.+W.+Fitzgeraldauthor=T.+C.+Burnauthor=B.+S.+Brownauthor=J.+P.+Pattersonauthor=M.+H.+Corjayauthor=P.+A.+Valentineauthor=J.+H.+Sunauthor=J.+R.+Linkauthor=I.+Abbaszadeauthor=J.+M.+Hollisauthor=B.+L.+Largentauthor=P.+R.+Hartigauthor=G.+F.+Hollisauthor=P.+C.+Meunierauthor=A.+J.+Robichaudauthor=D.+W.+Robertson&title=Possible+role+of+valvular+serotonin+5-HT%282B%29+receptors+in+the+cardiopathy+associated+with+fenfluramine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Possible role of valvular serotonin 5-HT2B receptors in the cardiopathy associated with fenfluramine</span></div><div class="casAuthors">Fitzgerald, Lawrence W.; Burn, Timothy C.; Brown, Barry S.; Patterson, John P.; Corjay, Martha H.; Valentine, Patricia A.; Sun, Jung-Hui; Link, John R.; Abbaszade, Ilgar; Hollis, Jeannine M.; Largent, Brian L.; Hartig, Paul R.; Hollis, Gregory F.; Meunier, Paul C.; Robichaud, Albert J.; Robertson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-81</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Dexfenfluramine was approved in the United States for longterm use as an appetite suppressant until it was reported to be assocd. with valvular heart disease.  The valvular changes (myofibroblast proliferation) are histopathol. indistinguishable from those obsd. in carcinoid disease or after long-term exposure to 5-hydroxytryptamine (5-HT)2-preferring ergot drugs (ergotamine, methysergide).  5-HT2 receptor stimulation is known to cause fibroblast mitogenesis, which could contribute to this lesion.  To elucidate the mechanism of "fenphen"-assocd. valvular lesions, we examd. the interaction of fenfluramine and its metabolite norfenfluramine with 5-HT2 receptor subtypes and examd. the expression of these receptors in human and porcine heart valves.  Fenfluramine binds weakly to 5-HT2A, 5-HT2B, and 5-HT2C receptors.  In contrast, norfenfluramine exhibited high affinity for 5-HT2B and 5-HT2C receptors and more moderate affinity for 5-HT2A receptors.  In cells expressing recombinant 5-HT2B receptors, norfenfluramine potently stimulated the hydrolysis of inositol phosphates, increased intracellular Ca2+, and activated the mitogen-activated protein kinase cascade, the latter of which has been linked to mitogenic actions of the 5-HT2B receptor.  The level of 5-HT2B and 5-HT2A receptor transcripts in heart valves was at least 300-fold higher than the levels of 5-HT2C receptor transcript, which were barely detectable.  We propose that preferential stimulation of valvular 5-HT2B receptors by norfenfluramine, ergot drugs, or 5-HT released from carcinoid tumors (with or without accompanying 5-HT2A receptor activation) may contribute to valvular fibroplasia in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmkbFOXGVxcLVg90H21EOLACvtfcHk0liDU-JeOb6ulw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhslCltA%253D%253D&md5=b99b778d0de695d377aa4f0006b6dc47</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFitzgerald%26aufirst%3DL.%2BW.%26aulast%3DBurn%26aufirst%3DT.%2BC.%26aulast%3DBrown%26aufirst%3DB.%2BS.%26aulast%3DPatterson%26aufirst%3DJ.%2BP.%26aulast%3DCorjay%26aufirst%3DM.%2BH.%26aulast%3DValentine%26aufirst%3DP.%2BA.%26aulast%3DSun%26aufirst%3DJ.%2BH.%26aulast%3DLink%26aufirst%3DJ.%2BR.%26aulast%3DAbbaszade%26aufirst%3DI.%26aulast%3DHollis%26aufirst%3DJ.%2BM.%26aulast%3DLargent%26aufirst%3DB.%2BL.%26aulast%3DHartig%26aufirst%3DP.%2BR.%26aulast%3DHollis%26aufirst%3DG.%2BF.%26aulast%3DMeunier%26aufirst%3DP.%2BC.%26aulast%3DRobichaud%26aufirst%3DA.%2BJ.%26aulast%3DRobertson%26aufirst%3DD.%2BW.%26atitle%3DPossible%2520role%2520of%2520valvular%2520serotonin%25205-HT%25282B%2529%2520receptors%2520in%2520the%2520cardiopathy%2520associated%2520with%2520fenfluramine%26jtitle%3DMol.%2520Pharmacol.%26date%3D2000%26volume%3D57%26spage%3D75%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Rothman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savage, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauser, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBride, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Evidence for possible involvement of 5-HT(2B) receptors in the cardiac valvulopathy associated with fenfluramine and other serotonergic medications</span> <span class="citation_source-journal">Circulation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">2836</span><span class="NLM_x">–</span> <span class="NLM_lpage">2841</span><span class="refDoi"> DOI: 10.1161/01.CIR.102.23.2836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1161%2F01.CIR.102.23.2836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=11104741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptVSjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2000&pages=2836-2841&author=R.+B.+Rothmanauthor=M.+H.+Baumannauthor=J.+E.+Savageauthor=L.+Rauserauthor=A.+McBrideauthor=S.+J.+Hufeisenauthor=B.+L.+Roth&title=Evidence+for+possible+involvement+of+5-HT%282B%29+receptors+in+the+cardiac+valvulopathy+associated+with+fenfluramine+and+other+serotonergic+medications&doi=10.1161%2F01.CIR.102.23.2836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for possible involvement of 5-HT2B receptors in the cardiac valvulopathy associated with fenfluramine and other serotonergic medications</span></div><div class="casAuthors">Rothman, Richard B.; Baumann, Michael H.; Savage, Jason E.; Rauser, Laura; McBride, Ace; Hufeisen, Sandra J.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2836-2841</span>CODEN:
                <span class="NLM_cas:coden">CIRCAZ</span>;
        ISSN:<span class="NLM_cas:issn">0009-7322</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Serotonergic medications with various mechanisms of action are used to treat psychiatric disorders and are being investigated as treatments for drug dependence.  The occurrence of fenfluramine-assocd. valvular heart disease (VHD) has raised concerns that other serotonergic medications might also increase the risk of developing VHD.  We hypothesized that fenfluramine or its metabolite norfenfluramine and other medications known to produce VHD have preferentially high affinities for a particular serotonin receptor subtype capable of stimulating mitogenesis.  Medications known or suspected to cause VHD (pos. controls) and medications not assocd. with VHD (neg. controls) were screened for activity at 11 cloned serotonin receptor subtypes by use of ligand-binding methods and functional assays.  The pos. control drugs were (±)-fenfluramine; (+)-fenfluramine; (-)-fenfluramine; its metabolites (±)-norfenfluramine, (+)-norfenfluramine, and (-)-norfenfluramine; ergotamine; and methysergide and its metabolite methylergonovine.  The neg. control drugs were phentermine, fluoxetine, its metabolite norfluoxetine, and trazodone and its active metabolite m-chlorophenylpiperazine.  (±)-, (+)-, And (-)-Norfenfluramine, ergotamine, and methylergonovine all had preferentially high affinities for the cloned human serotonin 5-HT2B receptor and were partial to full agonists at the 5-HT2B receptor.  Our data imply that activation of 5-HT2B receptors is necessary to produce VHD and that serotonergic medications that do not activate 5-HT2B receptors are unlikely to produce VHD.  We suggest that all clin. available medications with serotonergic activity and their active metabolites be screened for agonist activity at 5-HT2B receptors and that clinicians should consider suspending their use of medications with significant activity at 5-HT2B receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6spUXOCLtLrVg90H21EOLACvtfcHk0lgTpcWs7FKlPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptVSjtrs%253D&md5=6b8c6059885dd9b058859089914c61b3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1161%2F01.CIR.102.23.2836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.CIR.102.23.2836%26sid%3Dliteratum%253Aachs%26aulast%3DRothman%26aufirst%3DR.%2BB.%26aulast%3DBaumann%26aufirst%3DM.%2BH.%26aulast%3DSavage%26aufirst%3DJ.%2BE.%26aulast%3DRauser%26aufirst%3DL.%26aulast%3DMcBride%26aufirst%3DA.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DEvidence%2520for%2520possible%2520involvement%2520of%25205-HT%25282B%2529%2520receptors%2520in%2520the%2520cardiac%2520valvulopathy%2520associated%2520with%2520fenfluramine%2520and%2520other%2520serotonergic%2520medications%26jtitle%3DCirculation%26date%3D2000%26volume%3D102%26spage%3D2836%26epage%3D2841%26doi%3D10.1161%2F01.CIR.102.23.2836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Prichard, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipman, C.,  Jr.</span><span> </span><span class="NLM_article-title">A three-dimensional model to analyze drug-drug interactions</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">205</span><span class="refDoi"> DOI: 10.1016/0166-3542(90)90001-N</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1016%2F0166-3542%2890%2990001-N" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1990&pages=181-205&author=M.+N.+Prichardauthor=C.+Shipman&title=A+three-dimensional+model+to+analyze+drug-drug+interactions&doi=10.1016%2F0166-3542%2890%2990001-N"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2F0166-3542%2890%2990001-N&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0166-3542%252890%252990001-N%26sid%3Dliteratum%253Aachs%26aulast%3DPrichard%26aufirst%3DM.%2BN.%26aulast%3DShipman%26aufirst%3DC.%26atitle%3DA%2520three-dimensional%2520model%2520to%2520analyze%2520drug-drug%2520interactions%26jtitle%3DAntiviral%2520Res.%26date%3D1990%26volume%3D14%26spage%3D181%26epage%3D205%26doi%3D10.1016%2F0166-3542%2890%2990001-N" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Bassit, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schinazi, R. F.</span><span> </span><span class="NLM_article-title">Combinations of 2′-C-methylcytidine analogues with interferon-alpha2b and triple combination with ribavirin in the hepatitis C virus replicon system</span> <span class="citation_source-journal">Antiviral Chem. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1177/095632020801900104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1177%2F095632020801900104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=25-31&author=L.+Bassitauthor=J.+Grierauthor=M.+Bennettauthor=R.+F.+Schinazi&title=Combinations+of+2%E2%80%B2-C-methylcytidine+analogues+with+interferon-alpha2b+and+triple+combination+with+ribavirin+in+the+hepatitis+C+virus+replicon+system&doi=10.1177%2F095632020801900104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1177%2F095632020801900104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F095632020801900104%26sid%3Dliteratum%253Aachs%26aulast%3DBassit%26aufirst%3DL.%26aulast%3DGrier%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DM.%26aulast%3DSchinazi%26aufirst%3DR.%2BF.%26atitle%3DCombinations%2520of%25202%25E2%2580%25B2-C-methylcytidine%2520analogues%2520with%2520interferon-alpha2b%2520and%2520triple%2520combination%2520with%2520ribavirin%2520in%2520the%2520hepatitis%2520C%2520virus%2520replicon%2520system%26jtitle%3DAntiviral%2520Chem.%2520Chemother.%26date%3D2008%26volume%3D19%26spage%3D25%26epage%3D31%26doi%3D10.1177%2F095632020801900104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Amacher, D. E.</span><span> </span><span class="NLM_article-title">Serum transaminase elevations as indicators of hepatic injury following the administration of drugs</span> <span class="citation_source-journal">Regul. Toxicol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">119</span><span class="NLM_x">–</span> <span class="NLM_lpage">130</span><span class="refDoi"> DOI: 10.1006/rtph.1998.1201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1006%2Frtph.1998.1201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=9671567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADyaK1cXltlehsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1998&pages=119-130&author=D.+E.+Amacher&title=Serum+transaminase+elevations+as+indicators+of+hepatic+injury+following+the+administration+of+drugs&doi=10.1006%2Frtph.1998.1201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Serum transaminase elevations as indicators of hepatic injury following the administration of drugs</span></div><div class="casAuthors">Amacher, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Regulatory Toxicology and Pharmacology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">119-130</span>CODEN:
                <span class="NLM_cas:coden">RTOPDW</span>;
        ISSN:<span class="NLM_cas:issn">0273-2300</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review with 106 refs.  During the preclin., early clin., late-stage clin., and postmarketing phases of the pharmaceutical discovery and development process, one important aspect of drug safety assessment involves monitoring for possible drug-induced hepatic injury.  Hepatic injuries vary in nature from direct, intrinsic effects that are obsd. in most recipients and more than one species to rare idiosyncratic responses seen only in a few clin. subjects.  Histol. types of injuries vary from hepatocellular to hepatobiliary with multiple cellular effects characteristics of each type.  Of the various clin. lab. markers for hepatic injury, serum transaminases, esp. alanine aminotransferase (I), are the most universally important indicators for studies ranging from early preclin. animal testing to postmarketing patient monitoring.  This review examines the characteristics of hepatic toxicity that result in serum I changes, the differences in the etiol. of hepatic responses which govern when liver injury is most likely to be detected during the 4 phases of the drug discovery and development process, and those modulating factors which affect the utility of I as a dependable marker of hepatic injury in clin. populations.  The paper concludes with a summary of some ancillary methods for early preclin. screening such as in vitro metab. and toxicity assays, gene and protein expression anal., and some strategies for enhancing the probability for the early detection of idiosyncratic hepatotoxic responses which are infrequent but significant factors in the safety assessment process.  (c) 1998 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUQF8TAFS7X7Vg90H21EOLACvtfcHk0lgTpcWs7FKlPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltlehsLY%253D&md5=58a0102d58f378b774e7c0843b6dc1c9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1006%2Frtph.1998.1201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Frtph.1998.1201%26sid%3Dliteratum%253Aachs%26aulast%3DAmacher%26aufirst%3DD.%2BE.%26atitle%3DSerum%2520transaminase%2520elevations%2520as%2520indicators%2520of%2520hepatic%2520injury%2520following%2520the%2520administration%2520of%2520drugs%26jtitle%3DRegul.%2520Toxicol.%2520Pharmacol.%26date%3D1998%26volume%3D27%26spage%3D119%26epage%3D130%26doi%3D10.1006%2Frtph.1998.1201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dulcey, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marugan, J. J.</span><span> </span><span class="NLM_article-title">Discovery, optimization, and characterization of novel chlorcyclizine derivatives for the treatment of hepatitis C virus infection</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">853</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00752</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00752" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFSmtLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=841-853&author=S.+Heauthor=J.+Xiaoauthor=A.+E.+Dulceyauthor=B.+Linauthor=A.+Roltauthor=Z.+Huauthor=X.+Huauthor=A.+Q.+Wangauthor=X.+Xuauthor=N.+Southallauthor=M.+Ferrerauthor=W.+Zhengauthor=T.+J.+Liangauthor=J.+J.+Marugan&title=Discovery%2C+optimization%2C+and+characterization+of+novel+chlorcyclizine+derivatives+for+the+treatment+of+hepatitis+C+virus+infection&doi=10.1021%2Facs.jmedchem.5b00752"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, Optimization, and Characterization of Novel Chlorcyclizine Derivatives for the Treatment of Hepatitis C Virus Infection</span></div><div class="casAuthors">He, Shanshan; Xiao, Jingbo; Dulcey, Andres E.; Lin, Billy; Rolt, Adam; Hu, Zongyi; Hu, Xin; Wang, Amy Q.; Xu, Xin; Southall, Noel; Ferrer, Marc; Zheng, Wei; Liang, T. Jake; Marugan, Juan J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">841-853</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, we reported that chlorcyclizine (CCZ, Rac-2), an over-the-counter antihistamine piperazine drug, possesses in vitro and in vivo activity against hepatitis C virus.  Here, we describe structure-activity relationship (SAR) efforts that resulted in the optimization of novel chlorcyclizine derivs. as anti-HCV agents.  Several compds. exhibited EC50 values below 10 nM against HCV infection, cytotoxicity selectivity indexes above 2000, and showed improved in vivo pharmacokinetic properties.  The optimized mols. can serve as lead preclin. candidates for the treatment of hepatitis C virus infection and as probes to study hepatitis C virus pathogenesis and host-virus interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogrp6Aiq03ybVg90H21EOLACvtfcHk0lhOPCWR6xiN5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFSmtLfK&md5=0c93ba41c1d2780c47abb05aad2b88e7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00752%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DDulcey%26aufirst%3DA.%2BE.%26aulast%3DLin%26aufirst%3DB.%26aulast%3DRolt%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26atitle%3DDiscovery%252C%2520optimization%252C%2520and%2520characterization%2520of%2520novel%2520chlorcyclizine%2520derivatives%2520for%2520the%2520treatment%2520of%2520hepatitis%2520C%2520virus%2520infection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D841%26epage%3D853%26doi%3D10.1021%2Facs.jmedchem.5b00752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Schweitzer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span> </span><span class="NLM_article-title">Border control in hepatitis C virus infection: inhibiting viral entry</span> <span class="citation_source-journal">ACS Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">419</span><span class="refDoi"> DOI: 10.1021/acsinfecdis.5b00060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.5b00060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Citb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=416-419&author=C.+J.+Schweitzerauthor=T.+J.+Liang&title=Border+control+in+hepatitis+C+virus+infection%3A+inhibiting+viral+entry&doi=10.1021%2Facsinfecdis.5b00060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Border Control in Hepatitis C Virus Infection: Inhibiting Viral Entry</span></div><div class="casAuthors">Schweitzer, Cameron J.; Liang, T. Jake</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">416-419</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new era has begun in the treatment of hepatitis C virus (HCV) infection with powerful yet expensive therapies.  New treatments are emerging that target the entry step of HCV and could potentially block reinfection after liver transplant.  These treatments include antibodies, which target the virus or host receptors required by HCV.  Addnl., several new and previously approved small-mol. compds. have been described that target unique aspects of HCV entry.  Overall, the blocking entry represents an attractive strategy that could yield powerful combination therapies to combat HCV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfdrwdsj-3obVg90H21EOLACvtfcHk0lhOPCWR6xiN5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Citb%252FM&md5=e0d42c28affc4910ddee9d58c35c7342</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.5b00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.5b00060%26sid%3Dliteratum%253Aachs%26aulast%3DSchweitzer%26aufirst%3DC.%2BJ.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26atitle%3DBorder%2520control%2520in%2520hepatitis%2520C%2520virus%2520infection%253A%2520inhibiting%2520viral%2520entry%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D416%26epage%3D419%26doi%3D10.1021%2Facsinfecdis.5b00060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qing, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Z.</span><span> </span><span class="NLM_article-title">Discovery of imidazo [1, 2-α][1, 8] naphthyridine derivatives as potential HCV entry inhibitor</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">977</span><span class="NLM_x">–</span> <span class="NLM_lpage">981</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00159</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00159" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=977-981&author=H.+Wangauthor=S.+Wangauthor=L.+Chengauthor=L.+Chenauthor=Y.+Wangauthor=J.+Qingauthor=S.+Huangauthor=Y.+Wangauthor=X.+Leiauthor=Y.+Wuauthor=Z.+Ma&title=Discovery+of+imidazo+%5B1%2C+2-%CE%B1%5D%5B1%2C+8%5D+naphthyridine+derivatives+as+potential+HCV+entry+inhibitor&doi=10.1021%2Facsmedchemlett.5b00159"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00159%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DQing%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520imidazo%2520%255B1%252C%25202-%25CE%25B1%255D%255B1%252C%25208%255D%2520naphthyridine%2520derivatives%2520as%2520potential%2520HCV%2520entry%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D977%26epage%3D981%26doi%3D10.1021%2Facsmedchemlett.5b00159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Mittapalli, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKelvy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong-Staal, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macdonald, J. E.</span><span> </span><span class="NLM_article-title">Discovery of highly potent small molecule hepatitis C virus entry inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">6852</span><span class="NLM_x">–</span> <span class="NLM_lpage">6855</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1016%2Fj.bmcl.2011.09.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=6852-6855&author=G.+K.+Mittapalliauthor=A.+Jacksonauthor=F.+Zhaoauthor=H.+Leeauthor=S.+Chowauthor=J.+McKelvyauthor=F.+Wong-Staalauthor=J.+E.+Macdonald&title=Discovery+of+highly+potent+small+molecule+hepatitis+C+virus+entry+inhibitors&doi=10.1016%2Fj.bmcl.2011.09.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DMittapalli%26aufirst%3DG.%2BK.%26aulast%3DJackson%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DChow%26aufirst%3DS.%26aulast%3DMcKelvy%26aufirst%3DJ.%26aulast%3DWong-Staal%26aufirst%3DF.%26aulast%3DMacdonald%26aufirst%3DJ.%2BE.%26atitle%3DDiscovery%2520of%2520highly%2520potent%2520small%2520molecule%2520hepatitis%2520C%2520virus%2520entry%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D6852%26epage%3D6855%26doi%3D10.1016%2Fj.bmcl.2011.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Baldick, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wichroski, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsh, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzucco, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pokornowski, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggers, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhai, W.</span><span> </span><span class="NLM_article-title">A novel small molecule inhibitor of hepatitis C virus entry</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1001086</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1001086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1371%2Fjournal.ppat.1001086" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1001086&author=C.+J.+Baldickauthor=M.+J.+Wichroskiauthor=A.+Pendriauthor=A.+W.+Walshauthor=J.+Fangauthor=C.+E.+Mazzuccoauthor=K.+A.+Pokornowskiauthor=R.+E.+Roseauthor=B.+J.+Eggersauthor=M.+Hsuauthor=W.+Zhai&title=A+novel+small+molecule+inhibitor+of+hepatitis+C+virus+entry&doi=10.1371%2Fjournal.ppat.1001086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1001086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1001086%26sid%3Dliteratum%253Aachs%26aulast%3DBaldick%26aufirst%3DC.%2BJ.%26aulast%3DWichroski%26aufirst%3DM.%2BJ.%26aulast%3DPendri%26aufirst%3DA.%26aulast%3DWalsh%26aufirst%3DA.%2BW.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DMazzucco%26aufirst%3DC.%2BE.%26aulast%3DPokornowski%26aufirst%3DK.%2BA.%26aulast%3DRose%26aufirst%3DR.%2BE.%26aulast%3DEggers%26aufirst%3DB.%2BJ.%26aulast%3DHsu%26aufirst%3DM.%26aulast%3DZhai%26aufirst%3DW.%26atitle%3DA%2520novel%2520small%2520molecule%2520inhibitor%2520of%2520hepatitis%2520C%2520virus%2520entry%26jtitle%3DPLoS%2520Pathog.%26date%3D2010%26volume%3D6%26spage%3De1001086%26doi%3D10.1371%2Fjournal.ppat.1001086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruda, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abecassis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguiz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norval, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeons, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidah, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constam, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetsel, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.+P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0li0pldxFdegmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Lin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yim, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Z.</span><span> </span><span class="NLM_article-title">Evaluation of antiviral drug synergy in an infectious HCV system</span> <span class="citation_source-journal">Antiviral Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">603</span><span class="refDoi"> DOI: 10.3851/IMP3044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.3851%2FIMP3044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=27035622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Olsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=595-603&author=B.+Linauthor=S.+Heauthor=H.+J.+Yimauthor=T.+J.+Liangauthor=Z.+Hu&title=Evaluation+of+antiviral+drug+synergy+in+an+infectious+HCV+system&doi=10.3851%2FIMP3044"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="#"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=&md5=9874b665cc7a056b8e2f928dd3112440</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3851%2FIMP3044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3851%252FIMP3044%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DYim%26aufirst%3DH.%2BJ.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DHu%26aufirst%3DZ.%26atitle%3DEvaluation%2520of%2520antiviral%2520drug%2520synergy%2520in%2520an%2520infectious%2520HCV%2520system%26jtitle%3DAntiviral%2520Ther.%26date%3D2016%26volume%3D21%26spage%3D595%26epage%3D603%26doi%3D10.3851%2FIMP3044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: An examination of in vitro half-life approach and nonspecific binding to microsomes</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1359</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350-1359&author=R.+S.+Obach&title=Prediction+of+human+clearance+of+twenty-nine+drugs+from+hepatic+microsomal+intrinsic+clearance+data%3A+An+examination+of+in+vitro+half-life+approach+and+nonspecific+binding+to+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0ljn-_QYR-llnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520human%2520clearance%2520of%2520twenty-nine%2520drugs%2520from%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520data%253A%2520An%2520examination%2520of%2520in%2520vitro%2520half-life%2520approach%2520and%2520nonspecific%2520binding%2520to%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350%26epage%3D1359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Adam Rolt, Daniel C. Talley, Seung Bum Park, Zongyi Hu, Andrés Dulcey, Christopher Ma, Parker Irvin, Madeleine Leek, Amy Q. Wang, Andrew V. Stachulski, Xin Xu, Noel Southall, Marc Ferrer, T. Jake Liang, <span class="NLM_string-name hlFld-ContribAuthor">Juan J. Marugan</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of a 4-Aminopiperidine Scaffold for Inhibition of Hepatitis C Virus Assembly. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (13)
                                     , 9431-9443. <a href="https://doi.org/10.1021/acs.jmedchem.1c00696" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00696</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00696%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252Ba%252B4-Aminopiperidine%252BScaffold%252Bfor%252BInhibition%252Bof%252BHepatitis%252BC%252BVirus%252BAssembly%26aulast%3DRolt%26aufirst%3DAdam%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D16042021%26date%3D29062021%26volume%3D64%26issue%3D13%26spage%3D9431%26epage%3D9443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wenhan Zhang, Shufeng Liu, Rayelle I. Maiga, Jerry Pelletier, Lauren E. Brown, Tony T. Wang, <span class="NLM_string-name hlFld-ContribAuthor">John A. Porco, Jr.</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Synthesis Enables Structural Reengineering of Aglaroxin C Leading to Inhibition Bias for Hepatitis C Viral Infection. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2019,</strong> <em>141 </em>
                                    (3)
                                     , 1312-1323. <a href="https://doi.org/10.1021/jacs.8b11477" title="DOI URL">https://doi.org/10.1021/jacs.8b11477</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.8b11477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.8b11477%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DChemical%252BSynthesis%252BEnables%252BStructural%252BReengineering%252Bof%252BAglaroxin%252BC%252BLeading%252Bto%252BInhibition%252BBias%252Bfor%252BHepatitis%252BC%252BViral%252BInfection%26aulast%3DZhang%26aufirst%3DWenhan%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D24102018%26date%3D11012019%26date%3D27122018%26volume%3D141%26issue%3D3%26spage%3D1312%26epage%3D1323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher D.  Ma</span>, <span class="hlFld-ContribAuthor ">Michio  Imamura</span>, <span class="hlFld-ContribAuthor ">Daniel C.  Talley</span>, <span class="hlFld-ContribAuthor ">Adam  Rolt</span>, <span class="hlFld-ContribAuthor ">Xin  Xu</span>, <span class="hlFld-ContribAuthor ">Amy Q.  Wang</span>, <span class="hlFld-ContribAuthor ">Derek  Le</span>, <span class="hlFld-ContribAuthor ">Takuro  Uchida</span>, <span class="hlFld-ContribAuthor ">Mitsutaka  Osawa</span>, <span class="hlFld-ContribAuthor ">Yuji  Teraoka</span>, <span class="hlFld-ContribAuthor ">Kelin  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Hu</span>, <span class="hlFld-ContribAuthor ">Seung Bum  Park</span>, <span class="hlFld-ContribAuthor ">Nishanth  Chalasani</span>, <span class="hlFld-ContribAuthor ">Parker H.  Irvin</span>, <span class="hlFld-ContribAuthor ">Andres E.  Dulcey</span>, <span class="hlFld-ContribAuthor ">Noel  Southall</span>, <span class="hlFld-ContribAuthor ">Juan J.  Marugan</span>, <span class="hlFld-ContribAuthor ">Zongyi  Hu</span>, <span class="hlFld-ContribAuthor ">Kazuaki  Chayama</span>, <span class="hlFld-ContribAuthor ">Kevin J.  Frankowski</span>, <span class="hlFld-ContribAuthor ">Tsanyang Jake  Liang</span>. </span><span class="cited-content_cbyCitation_article-title">Fluoxazolevir inhibits hepatitis C virus infection in humanized chimeric mice by blocking viral membrane fusion. </span><span class="cited-content_cbyCitation_journal-name">Nature Microbiology</span><span> <strong>2020,</strong> <em>5 </em>
                                    (12)
                                     , 1532-1541. <a href="https://doi.org/10.1038/s41564-020-0781-2" title="DOI URL">https://doi.org/10.1038/s41564-020-0781-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41564-020-0781-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41564-020-0781-2%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Microbiology%26atitle%3DFluoxazolevir%252Binhibits%252Bhepatitis%252BC%252Bvirus%252Binfection%252Bin%252Bhumanized%252Bchimeric%252Bmice%252Bby%252Bblocking%252Bviral%252Bmembrane%252Bfusion%26aulast%3DMa%26aufirst%3DChristopher%2BD.%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D12%26spage%3D1532%26epage%3D1541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jules  Muhire</span>, <span class="hlFld-ContribAuthor ">Hong Lin  Zhai</span>, <span class="hlFld-ContribAuthor ">Shao Hua  Lu</span>, <span class="hlFld-ContribAuthor ">Sha Sha  Li</span>, <span class="hlFld-ContribAuthor ">Bo  Yin</span>, <span class="hlFld-ContribAuthor ">Jia Ying  Mi</span>. </span><span class="cited-content_cbyCitation_article-title">The activity prediction of indole inhibitors against HCV NS5B polymerase. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2020,</strong> <em>95 </em>
                                    (2)
                                     , 240-247. <a href="https://doi.org/10.1111/cbdd.13637" title="DOI URL">https://doi.org/10.1111/cbdd.13637</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13637%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DThe%252Bactivity%252Bprediction%252Bof%252Bindole%252Binhibitors%252Bagainst%252BHCV%252BNS5B%252Bpolymerase%26aulast%3DMuhire%26aufirst%3DJules%26date%3D2020%26date%3D2019%26volume%3D95%26issue%3D2%26spage%3D240%26epage%3D247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Route to Aryloxazole Analogues <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) HCl<sub>(g)</sub> (4 M in dioxane), AcOH, 0 °C to rt; (b) POCl<sub>3</sub>, DCE, reflux; (c) KOH, EtOH, isonipecotic acid, rt; (d) HOBt hydrate, DMAP, diisopropylcarbodiimide, DCM, rt or HOBt hydrate, diisopropylcarbodiimide, MeCN, μW irradiation, 100 °C, 10 min.</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route to Thiazole Analogue <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <i>i</i>PrOH, μW irradiation, 120 °C, 1 h, 65% yield; (b) NCS, MeCN, 60 °C, 2 h, 82% yield; (c) KOH, EtOH, isonipecotic acid, rt, 78% yield; (d) HOBt hydrate, diisopropylcarbodiimide, MeCN, μW irradiation, 100 °C, 10 min, 45% yield.</p></p></figure><figure data-id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0008.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Representative Synthetic Route to Desmethyloxazole Analogues; Synthesis of <b>18a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) toluene, 140 °C, 5 h, 57% yield; (b) KOH, EtOH, isonipecotic acid, rt, 93% yield; (d) HOBt hydrate, DMAP, diisopropylcarbodiimide, DCM, rt, 60% yield.</p></p></figure><figure data-id="sch4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0009.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. General Synthetic Route to Diamine Fragments <b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) acrylonitrile, formamide, water; (b) Raney nickel, H<sub>2(g)</sub>, 200 psi, MeOH, NH<sub>3(MeOH)</sub>.</p></p></figure><figure data-id="fig1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Summary of structural modifications explored in optimizing the aryloxazole hit and structure of the selected lead analogue <b>18a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Titration curves for anti-HCV activity (EC<sub>50</sub>, triangle in blue) and cytotoxicity (CC<sub>50</sub>, circle in red) of selected leads. Cell-culture adapted HCV (HCVcc) harboring a luciferase reporter gene (HCV-Luc, genotype 2a) was used to infect Huh7.5.1 cells in the presence of increasing concentrations of test compound. Viral infection and replication were measured by luciferase signal 48 h after treatment. Cytotoxicity was evaluated in parallel with the ATP-based cell viability assay (ATPlite). The results are mean from three replicates ± SEM. The EC<sub>50</sub> and CC<sub>50</sub> values were calculated with GraphPad Prism 5.0 software using nonlinear regression. Curves and values for each compound were from single representative experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Profile of representative analogue <b>7nn</b> against a panel of 50 CNS-relevant targets. Values are <i>K</i><sub>i</sub> determinations (nM) of radioligand binding in a displacement assay. Primary screen assays are single-point experiments to determine percent radioligand displacement at 10 μM of test compound. The threshold for hit or miss in the primary screen was 50% displacement.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/medium/jm-2017-00561n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pharmacokinetics studies of compound <b>7ii</b> in the mouse model. (A) Mean liver (green triangle) and plasma (red dot) concentration–time profiles of compound <b>7ii</b> after administration of a single i.p. dose at 10, 1, and 0.1 mg/kg at indicated time points. Compound concentration of <b>7ii</b> was measured by UPLC-MS/MS methods and is shown in means ± SEM (<i>n</i> = 3 per time point). (B) Pharmacokinetic parameters of compound <b>7ii</b>. AUC<sub>last</sub> = AUC<sub>0–24h</sub> or AUC<sub>0–168h</sub> depending on the sample collection interval. N.D., not determined. (C) ALT levels of the mouse serum samples collected during the pharmacokinetics study. Results for each mouse are shown with scatter plots, and error bars show means ± SEM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00561/20170721/images/large/jm-2017-00561n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00561&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62818" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62818" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 23 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Te, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, D. M.</span><span> </span><span class="NLM_article-title">Epidemiology of hepatitis B and C viruses: a global overview</span> <span class="citation_source-journal">Clin. Liver Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><div class="note"><p class="first last">vii.</p></div><span class="refDoi"> DOI: 10.1016/j.cld.2009.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1016%2Fj.cld.2009.11.009" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=1-21&author=H.+S.+Teauthor=D.+M.+Jensen&title=Epidemiology+of+hepatitis+B+and+C+viruses%3A+a+global+overview&doi=10.1016%2Fj.cld.2009.11.009"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cld.2009.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cld.2009.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DTe%26aufirst%3DH.%2BS.%26aulast%3DJensen%26aufirst%3DD.%2BM.%26atitle%3DEpidemiology%2520of%2520hepatitis%2520B%2520and%2520C%2520viruses%253A%2520a%2520global%2520overview%26jtitle%3DClin.%2520Liver%2520Dis.%26date%3D2010%26volume%3D14%26spage%3D1%26epage%3D21%26doi%3D10.1016%2Fj.cld.2009.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Ferenci, P.</span><span> </span><span class="NLM_article-title">Treatment of hepatitis C in difficult-to-treat patients</span> <span class="citation_source-journal">Nat. Rev. Gastroenterol. Hepatol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">–</span> <span class="NLM_lpage">292</span><span class="refDoi"> DOI: 10.1038/nrgastro.2015.53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1038%2Fnrgastro.2015.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=25895822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotVeqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=284-292&author=P.+Ferenci&title=Treatment+of+hepatitis+C+in+difficult-to-treat+patients&doi=10.1038%2Fnrgastro.2015.53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of hepatitis C in difficult-to-treat patients</span></div><div class="casAuthors">Ferenci, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Gastroenterology & Hepatology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">284-292</span>CODEN:
                <span class="NLM_cas:coden">NRGHA9</span>;
        ISSN:<span class="NLM_cas:issn">1759-5045</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interferon-free regimes are now the treatment of choice for patients with chronic hepatitis C; previously patients who were 'difficult-to-treat' using interferon-contg. treatments can now safely be treated with such therapies.  More than 90% of patients infected with HCV genotype 1 or 4, compensated cirrhosis, or who have had liver transplantation, can be cured with the use of sofosbuvir combined with simeprevir, daclatasvir or ledipasvir, or by the combination of paritaprevir with ritonavir, ombitasvir and with or without dasabuvir.  Addn. of ribavirin seems to shorten treatment duration.  However, the safety of these drugs is not fully explored in patients with decompensated cirrhosis (i.e., those with Child-Pugh class C disease), and protease inhibitors should not be used in this group.  The optimal use of interferon-free regimes in patients with renal failure or after kidney transplantation is currently being studied.  However, new and improved drugs are needed to treat patients infected with HCV genotype 3.  Unfortunately, the broad application of new HCV treatments is limited by their high costs.  In this Review, I discuss the treatment of patients with hepatitis C with compensated and decompensated cirrhosis, before and after orthotopic liver transplantation and in patients with impaired kidney function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorG-2DpipDwrVg90H21EOLACvtfcHk0lhJqn5d8urYhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotVeqtbs%253D&md5=52ec6909ccbf900e58393267c212de73</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrgastro.2015.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrgastro.2015.53%26sid%3Dliteratum%253Aachs%26aulast%3DFerenci%26aufirst%3DP.%26atitle%3DTreatment%2520of%2520hepatitis%2520C%2520in%2520difficult-to-treat%2520patients%26jtitle%3DNat.%2520Rev.%2520Gastroenterol.%2520Hepatol.%26date%3D2015%26volume%3D12%26spage%3D284%26epage%3D292%26doi%3D10.1038%2Fnrgastro.2015.53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Liang, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehermann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seeff, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoofnagle, J. H.</span><span> </span><span class="NLM_article-title">Pathogenesis, natural history, treatment, and prevention of hepatitis C</span> <span class="citation_source-journal">Ann. Intern. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">296</span><span class="NLM_x">–</span> <span class="NLM_lpage">305</span><span class="refDoi"> DOI: 10.7326/0003-4819-132-4-200002150-00008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.7326%2F0003-4819-132-4-200002150-00008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10681285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtlyku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2000&pages=296-305&author=T.+J.+Liangauthor=B.+Rehermannauthor=L.+B.+Seeffauthor=J.+H.+Hoofnagle&title=Pathogenesis%2C+natural+history%2C+treatment%2C+and+prevention+of+hepatitis+C&doi=10.7326%2F0003-4819-132-4-200002150-00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Pathogenesis, natural history, treatment, and prevention of hepatitis c</span></div><div class="casAuthors">Liang, T. Jake; Rehermann, Barbara; Seeff, Leonard B.; Hoofnagle, Jay H.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Internal Medicine</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">296-305</span>CODEN:
                <span class="NLM_cas:coden">AIMEAS</span>;
        ISSN:<span class="NLM_cas:issn">0003-4819</span>.
    
            (<span class="NLM_cas:orgname">American College of Physicians-American Society of Internal Medicine</span>)
        </div><div class="casAbstract">A review with 77 refs.  Approx. 4 million persons in the United States and probably more than 100 million persons worldwide are infected with hepatitis C virus.  The virus has the unique ability to cause persistent infection in susceptible hosts after parenteral or percutaneous transmission, and its underlying mechanisms are not well understood.  The immunol. correlates of protection and viral clearance and the pathogenesis of liver injury are yet to be defined, but recent studies suggest the importance of cell-mediated immune responses.  Although 70% to 80% of infected persons become chronic carriers, most have relatively mild disease with slow progression.  However, chronic and progressive hepatitis C carries significant morbidity and mortality and is a major cause of cirrhosis, end-stage liver disease, and liver cancer.  Development of an effective hepatitis C virus vaccine is not imminent, but recent advances in technol. and basic knowledge of mol. virol. and immunol. have engendered novel approaches to the fundamental problems encountered in vaccine development.  Current therapy for hepatitis C, although effective in some patients, is problematic and still evolving.  Advances in modern biol. and immunol. promise new therapies for this important disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiCUCgLD1TwrVg90H21EOLACvtfcHk0lh58FoA8Qs8mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtlyku7g%253D&md5=5c5d04abdc4611c544258fb84caaae6d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.7326%2F0003-4819-132-4-200002150-00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7326%252F0003-4819-132-4-200002150-00008%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DRehermann%26aufirst%3DB.%26aulast%3DSeeff%26aufirst%3DL.%2BB.%26aulast%3DHoofnagle%26aufirst%3DJ.%2BH.%26atitle%3DPathogenesis%252C%2520natural%2520history%252C%2520treatment%252C%2520and%2520prevention%2520of%2520hepatitis%2520C%26jtitle%3DAnn.%2520Intern.%2520Med.%26date%3D2000%26volume%3D132%26spage%3D296%26epage%3D305%26doi%3D10.7326%2F0003-4819-132-4-200002150-00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Liang, T. J.</span><span> </span><span class="NLM_article-title">Current progress in development of hepatitis C virus vaccines</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">869</span><span class="NLM_x">–</span> <span class="NLM_lpage">878</span><span class="refDoi"> DOI: 10.1038/nm.3183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1038%2Fnm.3183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=23836237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKqsrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=869-878&author=T.+J.+Liang&title=Current+progress+in+development+of+hepatitis+C+virus+vaccines&doi=10.1038%2Fnm.3183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Current progress in development of hepatitis C virus vaccines</span></div><div class="casAuthors">Liang, T. Jake</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">869-878</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite major advances in the understanding and treatment of hepatitis C, a preventive vaccine remains elusive.  The marked genetic diversity and multiple mechanisms of persistence of hepatitis C virus, combined with the relatively poor immune response of the infected host against the virus, are major barriers.  The lack of robust and convenient model systems further hampers the effort to develop an effective vaccine.  Advances in our understanding of virus-host interactions and protective immunity in hepatitis C virus infection provide an important roadmap to develop potent and broadly directed vaccine candidates targeting both humoral and cellular immune responses.  Multiple approaches to generating and testing viral immunogens have met with variable success.  Several candidates have advanced to clin. trials based on promising results in chimpanzees.  The ultimate path to a successful preventive vaccine requires comprehensive evaluations of all aspects of protective immunity, innovative application of state-of-the-art vaccine technol. and properly designed vaccine trials that can affirm definitive endpoints of efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqap5_3MnE0MLVg90H21EOLACvtfcHk0lh58FoA8Qs8mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKqsrbO&md5=c3d2c522cc0fc2c078a92d1fcf463b55</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnm.3183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3183%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26atitle%3DCurrent%2520progress%2520in%2520development%2520of%2520hepatitis%2520C%2520virus%2520vaccines%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D869%26epage%3D878%26doi%3D10.1038%2Fnm.3183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Liang, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghany, M. G.</span><span> </span><span class="NLM_article-title">Current and future therapies for hepatitis C virus infection</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">368</span><span class="NLM_x">, </span> <span class="NLM_fpage">1907</span><span class="NLM_x">–</span> <span class="NLM_lpage">1917</span><span class="refDoi"> DOI: 10.1056/NEJMra1213651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1056%2FNEJMra1213651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=23675659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsl2gs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2013&pages=1907-1917&author=T.+J.+Liangauthor=M.+G.+Ghany&title=Current+and+future+therapies+for+hepatitis+C+virus+infection&doi=10.1056%2FNEJMra1213651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Current and future therapies for hepatitis C virus infection</span></div><div class="casAuthors">Liang, T. Jake; Ghany, Marc G.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1907-1917</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  The recent approval of two direct-acting antiviral agents for hepatitis C viral (HCV) infection that specifically inhibit viral replication has dramatically increased the viral clearance rate.  It is anticipated that in the course of such a screening process, a large no. of persons will be found to be infected with HCV; this article reviews the current therapy for HCV infection and the landscape of drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4zjjmKeKrZrVg90H21EOLACvtfcHk0lh58FoA8Qs8mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsl2gs78%253D&md5=6be058d0298a4441a180ac076d7293aa</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1213651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1213651%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DGhany%26aufirst%3DM.%2BG.%26atitle%3DCurrent%2520and%2520future%2520therapies%2520for%2520hepatitis%2520C%2520virus%2520infection%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2013%26volume%3D368%26spage%3D1907%26epage%3D1917%26doi%3D10.1056%2FNEJMra1213651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Callaway, E.</span><span> </span><span class="NLM_article-title">Hepatitis C drugs not reaching poor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">508</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">296</span><span class="refDoi"> DOI: 10.1038/508295a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1038%2F508295a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=295-296&author=E.+Callaway&title=Hepatitis+C+drugs+not+reaching+poor&doi=10.1038%2F508295a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F508295a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F508295a%26sid%3Dliteratum%253Aachs%26aulast%3DCallaway%26aufirst%3DE.%26atitle%3DHepatitis%2520C%2520drugs%2520not%2520reaching%2520poor%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D295%26epage%3D296%26doi%3D10.1038%2F508295a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Hu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaroop, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span> </span><span class="NLM_article-title">Novel cell-based hepatitis C virus infection assay for quantitative high-throughput screening of anti-hepatitis C virus compounds</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">995</span><span class="NLM_x">–</span> <span class="NLM_lpage">1004</span><span class="refDoi"> DOI: 10.1128/AAC.02094-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1128%2FAAC.02094-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=24277038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVSrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=995-1004&author=Z.+Huauthor=K.+H.+Lanauthor=S.+Heauthor=M.+Swaroopauthor=X.+Huauthor=N.+Southallauthor=W.+Zhengauthor=T.+J.+Liang&title=Novel+cell-based+hepatitis+C+virus+infection+assay+for+quantitative+high-throughput+screening+of+anti-hepatitis+C+virus+compounds&doi=10.1128%2FAAC.02094-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Novel cell-based hepatitis C virus infection assay for quantitative high-throughput screening of anti-hepatitis C virus compounds</span></div><div class="casAuthors">Hu, Zongyi; Lan, Keng-Hsin; He, Shanshan; Swaroop, Manju; Hu, Xin; Southall, Noel; Zheng, Wei; Liang, T. Jake</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">995-1004, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Therapy for hepatitis C virus (HCV) infection has advanced with the recent approval of direct-acting antivirals in combination with peginterferon and ribavirin.  New antivirals with novel targets are still needed to further improve the treatment of hepatitis C.  Previously reported screening methods for HCV inhibitors either are limited to a virus-specific function or apply a screening method at a single dose, which usually leads to high false-pos. or -neg. rates.  We developed a quant. high-throughput screening (qHTS) assay platform with a cell-based HCV infection system.  This highly sensitive assay can be miniaturized to a 1,536-well format for screening of large chem. libraries.  All candidates are screened over a 7-concn. dose range to give EC50s (compd. concns. at 50% efficacy) and dose-response curves.  Using this assay format, we screened a library of pharmacol. active compds. (LOPAC).  Based on the profile of dose-dependent curves of HCV inhibition and cytotoxicity, 22 compds. with adequate curves and EC50s of <10 μM were selected for validation.  In two addnl. independent assays, 17 of them demonstrated specific inhibition of HCV infection.  Ten potential candidates with efficacies of >70% and CC50s (compd. concns. at 50% cytotoxicity) of <30 μM from these validated hits were characterized for their target stages in the HCV replication cycle.  In this screen, we identified both known and novel hits with diverse structural and functional features targeting various stages of the HCV replication cycle.  The pilot screen demonstrates that this assay system is highly robust and effective in identifying novel HCV inhibitors and that it can be readily applied to large-scale screening of small-mol. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8MWc_pTB2cbVg90H21EOLACvtfcHk0ljkOhasNpwfnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVSrtrg%253D&md5=b0fd6fc3af44e8f211cc13ae7c33a408</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FAAC.02094-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02094-13%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DLan%26aufirst%3DK.%2BH.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSwaroop%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26atitle%3DNovel%2520cell-based%2520hepatitis%2520C%2520virus%2520infection%2520assay%2520for%2520quantitative%2520high-throughput%2520screening%2520of%2520anti-hepatitis%2520C%2520virus%2520compounds%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D995%26epage%3D1004%26doi%3D10.1128%2FAAC.02094-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Hu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yim, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaroop, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanega, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kao, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marugan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span> </span><span class="NLM_article-title">Identification of novel anti-hepatitis C virus agents by a quantitative high throughput screen in a cell-based infection assay</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2015.10.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1016%2Fj.antiviral.2015.10.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=26515788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslyqsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2015&pages=20-29&author=Z.+Huauthor=X.+Huauthor=S.+Heauthor=H.+J.+Yimauthor=J.+Xiaoauthor=M.+Swaroopauthor=C.+Tanegaauthor=Y.+Q.+Zhangauthor=G.+Yiauthor=C.+C.+Kaoauthor=J.+Maruganauthor=M.+Ferrerauthor=W.+Zhengauthor=N.+Southallauthor=T.+J.+Liang&title=Identification+of+novel+anti-hepatitis+C+virus+agents+by+a+quantitative+high+throughput+screen+in+a+cell-based+infection+assay&doi=10.1016%2Fj.antiviral.2015.10.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel anti-hepatitis C virus agents by a quantitative high throughput screen in a cell-based infection assay</span></div><div class="casAuthors">Hu, Zongyi; Hu, Xin; He, Shanshan; Yim, Hyung Joon; Xiao, Jingbo; Swaroop, Manju; Tanega, Cordelle; Zhang, Ya-qin; Yi, Guanghui; Kao, C. Cheng; Marugan, Juan; Ferrer, Marc; Zheng, Wei; Southall, Noel; Liang, T. Jake</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20-29</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) poses a major health threat to the world.  The recent development of direct-acting antivirals (DAAs) against HCV has markedly improved the response rate of HCV and reduced the side effects in comparison to the interferon-based therapy.  Despite this therapeutic advance, there is still a need to develop new inhibitors that target different stages of the HCV life cycle because of various limitations of the current regimens.  In this study, we performed a quant. high throughput screening of the Mol. Libraries Small Mol. Repository (MLSMR) of ∼350,000 chems. for novel HCV inhibitors using our previously developed cell-based HCV infection assay.  Following confirmation and structural clustering anal., we narrowed down to 158 compds. from the initial ∼3000 mols. that showed inhibitory activity for further structural and functional analyses.  We were able to assign the majority of these compds. to specific stage(s) in the HCV life cycle.  Three of them are direct inhibitors of NS3/4A protease.  Most of the compds. appear to act on novel targets in HCV life cycle.  Four compds. with novel structure and excellent drug-like properties, three targeting HCV entry and one targeting HCV assembly/secretion, were advanced for further development as lead hits.  These compds. represent diverse chemotypes that are potential lead compds. for further optimization and may offer promising candidates for the development of novel therapeutics against HCV infection.  In addn., they represent novel mol. probes to explore the complex interactions between HCV and the cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnosKyCpJzeLVg90H21EOLACvtfcHk0ljkOhasNpwfnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslyqsbjL&md5=d58abbeb96d510e5246929ce171772b5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2015.10.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2015.10.018%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DYim%26aufirst%3DH.%2BJ.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DSwaroop%26aufirst%3DM.%26aulast%3DTanega%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%2BQ.%26aulast%3DYi%26aufirst%3DG.%26aulast%3DKao%26aufirst%3DC.%2BC.%26aulast%3DMarugan%26aufirst%3DJ.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26atitle%3DIdentification%2520of%2520novel%2520anti-hepatitis%2520C%2520virus%2520agents%2520by%2520a%2520quantitative%2520high%2520throughput%2520screen%2520in%2520a%2520cell-based%2520infection%2520assay%26jtitle%3DAntiviral%2520Res.%26date%3D2015%26volume%3D124%26spage%3D20%26epage%3D29%26doi%3D10.1016%2Fj.antiviral.2015.10.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Goto, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamana, M.</span><span> </span><span class="NLM_article-title">Studies on azole compounds. III. Reactions of oxazole N-oxides with phosphoryl chloride and acetic anhydride</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1971</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2050</span><span class="NLM_x">–</span> <span class="NLM_lpage">2057</span><span class="refDoi"> DOI: 10.1248/cpb.19.2050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1248%2Fcpb.19.2050" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1971&pages=2050-2057&author=Y.+Y.+Gotoauthor=M.+Hamana&title=Studies+on+azole+compounds.+III.+Reactions+of+oxazole+N-oxides+with+phosphoryl+chloride+and+acetic+anhydride&doi=10.1248%2Fcpb.19.2050"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1248%2Fcpb.19.2050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.19.2050%26sid%3Dliteratum%253Aachs%26aulast%3DGoto%26aufirst%3DY.%2BY.%26aulast%3DHamana%26aufirst%3DM.%26atitle%3DStudies%2520on%2520azole%2520compounds.%2520III.%2520Reactions%2520of%2520oxazole%2520N-oxides%2520with%2520phosphoryl%2520chloride%2520and%2520acetic%2520anhydride%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1971%26volume%3D19%26spage%3D2050%26epage%3D2057%26doi%3D10.1248%2Fcpb.19.2050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Hsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flint, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logvinoff, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng-Mayer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeating, J. A.</span><span> </span><span class="NLM_article-title">Hepatitis C virus glycoproteins mediate pH-dependent cell entry of pseudotyped retroviral particles</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">7271</span><span class="NLM_x">–</span> <span class="NLM_lpage">7276</span><span class="refDoi"> DOI: 10.1073/pnas.0832180100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1073%2Fpnas.0832180100" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=7271-7276&author=M.+Hsuauthor=J.+Zhangauthor=M.+Flintauthor=C.+Logvinoffauthor=C.+Cheng-Mayerauthor=C.+M.+Riceauthor=J.+A.+McKeating&title=Hepatitis+C+virus+glycoproteins+mediate+pH-dependent+cell+entry+of+pseudotyped+retroviral+particles&doi=10.1073%2Fpnas.0832180100"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0832180100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0832180100%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFlint%26aufirst%3DM.%26aulast%3DLogvinoff%26aufirst%3DC.%26aulast%3DCheng-Mayer%26aufirst%3DC.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DMcKeating%26aufirst%3DJ.%2BA.%26atitle%3DHepatitis%2520C%2520virus%2520glycoproteins%2520mediate%2520pH-dependent%2520cell%2520entry%2520of%2520pseudotyped%2520retroviral%2520particles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D7271%26epage%3D7276%26doi%3D10.1073%2Fpnas.0832180100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Fitzgerald, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burn, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patterson, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corjay, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valentine, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Link, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbaszade, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollis, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Largent, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartig, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollis, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meunier, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robichaud, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, D. W.</span><span> </span><span class="NLM_article-title">Possible role of valvular serotonin 5-HT(2B) receptors in the cardiopathy associated with fenfluramine</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10617681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhslCltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2000&pages=75-81&author=L.+W.+Fitzgeraldauthor=T.+C.+Burnauthor=B.+S.+Brownauthor=J.+P.+Pattersonauthor=M.+H.+Corjayauthor=P.+A.+Valentineauthor=J.+H.+Sunauthor=J.+R.+Linkauthor=I.+Abbaszadeauthor=J.+M.+Hollisauthor=B.+L.+Largentauthor=P.+R.+Hartigauthor=G.+F.+Hollisauthor=P.+C.+Meunierauthor=A.+J.+Robichaudauthor=D.+W.+Robertson&title=Possible+role+of+valvular+serotonin+5-HT%282B%29+receptors+in+the+cardiopathy+associated+with+fenfluramine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Possible role of valvular serotonin 5-HT2B receptors in the cardiopathy associated with fenfluramine</span></div><div class="casAuthors">Fitzgerald, Lawrence W.; Burn, Timothy C.; Brown, Barry S.; Patterson, John P.; Corjay, Martha H.; Valentine, Patricia A.; Sun, Jung-Hui; Link, John R.; Abbaszade, Ilgar; Hollis, Jeannine M.; Largent, Brian L.; Hartig, Paul R.; Hollis, Gregory F.; Meunier, Paul C.; Robichaud, Albert J.; Robertson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-81</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Dexfenfluramine was approved in the United States for longterm use as an appetite suppressant until it was reported to be assocd. with valvular heart disease.  The valvular changes (myofibroblast proliferation) are histopathol. indistinguishable from those obsd. in carcinoid disease or after long-term exposure to 5-hydroxytryptamine (5-HT)2-preferring ergot drugs (ergotamine, methysergide).  5-HT2 receptor stimulation is known to cause fibroblast mitogenesis, which could contribute to this lesion.  To elucidate the mechanism of "fenphen"-assocd. valvular lesions, we examd. the interaction of fenfluramine and its metabolite norfenfluramine with 5-HT2 receptor subtypes and examd. the expression of these receptors in human and porcine heart valves.  Fenfluramine binds weakly to 5-HT2A, 5-HT2B, and 5-HT2C receptors.  In contrast, norfenfluramine exhibited high affinity for 5-HT2B and 5-HT2C receptors and more moderate affinity for 5-HT2A receptors.  In cells expressing recombinant 5-HT2B receptors, norfenfluramine potently stimulated the hydrolysis of inositol phosphates, increased intracellular Ca2+, and activated the mitogen-activated protein kinase cascade, the latter of which has been linked to mitogenic actions of the 5-HT2B receptor.  The level of 5-HT2B and 5-HT2A receptor transcripts in heart valves was at least 300-fold higher than the levels of 5-HT2C receptor transcript, which were barely detectable.  We propose that preferential stimulation of valvular 5-HT2B receptors by norfenfluramine, ergot drugs, or 5-HT released from carcinoid tumors (with or without accompanying 5-HT2A receptor activation) may contribute to valvular fibroplasia in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmkbFOXGVxcLVg90H21EOLACvtfcHk0lgNSAKSf2eZnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhslCltA%253D%253D&md5=b99b778d0de695d377aa4f0006b6dc47</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFitzgerald%26aufirst%3DL.%2BW.%26aulast%3DBurn%26aufirst%3DT.%2BC.%26aulast%3DBrown%26aufirst%3DB.%2BS.%26aulast%3DPatterson%26aufirst%3DJ.%2BP.%26aulast%3DCorjay%26aufirst%3DM.%2BH.%26aulast%3DValentine%26aufirst%3DP.%2BA.%26aulast%3DSun%26aufirst%3DJ.%2BH.%26aulast%3DLink%26aufirst%3DJ.%2BR.%26aulast%3DAbbaszade%26aufirst%3DI.%26aulast%3DHollis%26aufirst%3DJ.%2BM.%26aulast%3DLargent%26aufirst%3DB.%2BL.%26aulast%3DHartig%26aufirst%3DP.%2BR.%26aulast%3DHollis%26aufirst%3DG.%2BF.%26aulast%3DMeunier%26aufirst%3DP.%2BC.%26aulast%3DRobichaud%26aufirst%3DA.%2BJ.%26aulast%3DRobertson%26aufirst%3DD.%2BW.%26atitle%3DPossible%2520role%2520of%2520valvular%2520serotonin%25205-HT%25282B%2529%2520receptors%2520in%2520the%2520cardiopathy%2520associated%2520with%2520fenfluramine%26jtitle%3DMol.%2520Pharmacol.%26date%3D2000%26volume%3D57%26spage%3D75%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Rothman, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savage, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauser, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBride, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Evidence for possible involvement of 5-HT(2B) receptors in the cardiac valvulopathy associated with fenfluramine and other serotonergic medications</span> <span class="citation_source-journal">Circulation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">2836</span><span class="NLM_x">–</span> <span class="NLM_lpage">2841</span><span class="refDoi"> DOI: 10.1161/01.CIR.102.23.2836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1161%2F01.CIR.102.23.2836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=11104741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptVSjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2000&pages=2836-2841&author=R.+B.+Rothmanauthor=M.+H.+Baumannauthor=J.+E.+Savageauthor=L.+Rauserauthor=A.+McBrideauthor=S.+J.+Hufeisenauthor=B.+L.+Roth&title=Evidence+for+possible+involvement+of+5-HT%282B%29+receptors+in+the+cardiac+valvulopathy+associated+with+fenfluramine+and+other+serotonergic+medications&doi=10.1161%2F01.CIR.102.23.2836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for possible involvement of 5-HT2B receptors in the cardiac valvulopathy associated with fenfluramine and other serotonergic medications</span></div><div class="casAuthors">Rothman, Richard B.; Baumann, Michael H.; Savage, Jason E.; Rauser, Laura; McBride, Ace; Hufeisen, Sandra J.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Circulation</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2836-2841</span>CODEN:
                <span class="NLM_cas:coden">CIRCAZ</span>;
        ISSN:<span class="NLM_cas:issn">0009-7322</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Serotonergic medications with various mechanisms of action are used to treat psychiatric disorders and are being investigated as treatments for drug dependence.  The occurrence of fenfluramine-assocd. valvular heart disease (VHD) has raised concerns that other serotonergic medications might also increase the risk of developing VHD.  We hypothesized that fenfluramine or its metabolite norfenfluramine and other medications known to produce VHD have preferentially high affinities for a particular serotonin receptor subtype capable of stimulating mitogenesis.  Medications known or suspected to cause VHD (pos. controls) and medications not assocd. with VHD (neg. controls) were screened for activity at 11 cloned serotonin receptor subtypes by use of ligand-binding methods and functional assays.  The pos. control drugs were (±)-fenfluramine; (+)-fenfluramine; (-)-fenfluramine; its metabolites (±)-norfenfluramine, (+)-norfenfluramine, and (-)-norfenfluramine; ergotamine; and methysergide and its metabolite methylergonovine.  The neg. control drugs were phentermine, fluoxetine, its metabolite norfluoxetine, and trazodone and its active metabolite m-chlorophenylpiperazine.  (±)-, (+)-, And (-)-Norfenfluramine, ergotamine, and methylergonovine all had preferentially high affinities for the cloned human serotonin 5-HT2B receptor and were partial to full agonists at the 5-HT2B receptor.  Our data imply that activation of 5-HT2B receptors is necessary to produce VHD and that serotonergic medications that do not activate 5-HT2B receptors are unlikely to produce VHD.  We suggest that all clin. available medications with serotonergic activity and their active metabolites be screened for agonist activity at 5-HT2B receptors and that clinicians should consider suspending their use of medications with significant activity at 5-HT2B receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6spUXOCLtLrVg90H21EOLACvtfcHk0lgNSAKSf2eZnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptVSjtrs%253D&md5=6b8c6059885dd9b058859089914c61b3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1161%2F01.CIR.102.23.2836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252F01.CIR.102.23.2836%26sid%3Dliteratum%253Aachs%26aulast%3DRothman%26aufirst%3DR.%2BB.%26aulast%3DBaumann%26aufirst%3DM.%2BH.%26aulast%3DSavage%26aufirst%3DJ.%2BE.%26aulast%3DRauser%26aufirst%3DL.%26aulast%3DMcBride%26aufirst%3DA.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DEvidence%2520for%2520possible%2520involvement%2520of%25205-HT%25282B%2529%2520receptors%2520in%2520the%2520cardiac%2520valvulopathy%2520associated%2520with%2520fenfluramine%2520and%2520other%2520serotonergic%2520medications%26jtitle%3DCirculation%26date%3D2000%26volume%3D102%26spage%3D2836%26epage%3D2841%26doi%3D10.1161%2F01.CIR.102.23.2836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Prichard, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipman, C.,  Jr.</span><span> </span><span class="NLM_article-title">A three-dimensional model to analyze drug-drug interactions</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">205</span><span class="refDoi"> DOI: 10.1016/0166-3542(90)90001-N</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1016%2F0166-3542%2890%2990001-N" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1990&pages=181-205&author=M.+N.+Prichardauthor=C.+Shipman&title=A+three-dimensional+model+to+analyze+drug-drug+interactions&doi=10.1016%2F0166-3542%2890%2990001-N"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2F0166-3542%2890%2990001-N&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0166-3542%252890%252990001-N%26sid%3Dliteratum%253Aachs%26aulast%3DPrichard%26aufirst%3DM.%2BN.%26aulast%3DShipman%26aufirst%3DC.%26atitle%3DA%2520three-dimensional%2520model%2520to%2520analyze%2520drug-drug%2520interactions%26jtitle%3DAntiviral%2520Res.%26date%3D1990%26volume%3D14%26spage%3D181%26epage%3D205%26doi%3D10.1016%2F0166-3542%2890%2990001-N" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Bassit, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schinazi, R. F.</span><span> </span><span class="NLM_article-title">Combinations of 2′-C-methylcytidine analogues with interferon-alpha2b and triple combination with ribavirin in the hepatitis C virus replicon system</span> <span class="citation_source-journal">Antiviral Chem. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1177/095632020801900104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1177%2F095632020801900104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=25-31&author=L.+Bassitauthor=J.+Grierauthor=M.+Bennettauthor=R.+F.+Schinazi&title=Combinations+of+2%E2%80%B2-C-methylcytidine+analogues+with+interferon-alpha2b+and+triple+combination+with+ribavirin+in+the+hepatitis+C+virus+replicon+system&doi=10.1177%2F095632020801900104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1177%2F095632020801900104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F095632020801900104%26sid%3Dliteratum%253Aachs%26aulast%3DBassit%26aufirst%3DL.%26aulast%3DGrier%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DM.%26aulast%3DSchinazi%26aufirst%3DR.%2BF.%26atitle%3DCombinations%2520of%25202%25E2%2580%25B2-C-methylcytidine%2520analogues%2520with%2520interferon-alpha2b%2520and%2520triple%2520combination%2520with%2520ribavirin%2520in%2520the%2520hepatitis%2520C%2520virus%2520replicon%2520system%26jtitle%3DAntiviral%2520Chem.%2520Chemother.%26date%3D2008%26volume%3D19%26spage%3D25%26epage%3D31%26doi%3D10.1177%2F095632020801900104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Amacher, D. E.</span><span> </span><span class="NLM_article-title">Serum transaminase elevations as indicators of hepatic injury following the administration of drugs</span> <span class="citation_source-journal">Regul. Toxicol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">119</span><span class="NLM_x">–</span> <span class="NLM_lpage">130</span><span class="refDoi"> DOI: 10.1006/rtph.1998.1201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1006%2Frtph.1998.1201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=9671567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADyaK1cXltlehsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1998&pages=119-130&author=D.+E.+Amacher&title=Serum+transaminase+elevations+as+indicators+of+hepatic+injury+following+the+administration+of+drugs&doi=10.1006%2Frtph.1998.1201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Serum transaminase elevations as indicators of hepatic injury following the administration of drugs</span></div><div class="casAuthors">Amacher, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Regulatory Toxicology and Pharmacology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">119-130</span>CODEN:
                <span class="NLM_cas:coden">RTOPDW</span>;
        ISSN:<span class="NLM_cas:issn">0273-2300</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review with 106 refs.  During the preclin., early clin., late-stage clin., and postmarketing phases of the pharmaceutical discovery and development process, one important aspect of drug safety assessment involves monitoring for possible drug-induced hepatic injury.  Hepatic injuries vary in nature from direct, intrinsic effects that are obsd. in most recipients and more than one species to rare idiosyncratic responses seen only in a few clin. subjects.  Histol. types of injuries vary from hepatocellular to hepatobiliary with multiple cellular effects characteristics of each type.  Of the various clin. lab. markers for hepatic injury, serum transaminases, esp. alanine aminotransferase (I), are the most universally important indicators for studies ranging from early preclin. animal testing to postmarketing patient monitoring.  This review examines the characteristics of hepatic toxicity that result in serum I changes, the differences in the etiol. of hepatic responses which govern when liver injury is most likely to be detected during the 4 phases of the drug discovery and development process, and those modulating factors which affect the utility of I as a dependable marker of hepatic injury in clin. populations.  The paper concludes with a summary of some ancillary methods for early preclin. screening such as in vitro metab. and toxicity assays, gene and protein expression anal., and some strategies for enhancing the probability for the early detection of idiosyncratic hepatotoxic responses which are infrequent but significant factors in the safety assessment process.  (c) 1998 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUQF8TAFS7X7Vg90H21EOLACvtfcHk0lgkKybvXYMFvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltlehsLY%253D&md5=58a0102d58f378b774e7c0843b6dc1c9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1006%2Frtph.1998.1201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Frtph.1998.1201%26sid%3Dliteratum%253Aachs%26aulast%3DAmacher%26aufirst%3DD.%2BE.%26atitle%3DSerum%2520transaminase%2520elevations%2520as%2520indicators%2520of%2520hepatic%2520injury%2520following%2520the%2520administration%2520of%2520drugs%26jtitle%3DRegul.%2520Toxicol.%2520Pharmacol.%26date%3D1998%26volume%3D27%26spage%3D119%26epage%3D130%26doi%3D10.1006%2Frtph.1998.1201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dulcey, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marugan, J. J.</span><span> </span><span class="NLM_article-title">Discovery, optimization, and characterization of novel chlorcyclizine derivatives for the treatment of hepatitis C virus infection</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">853</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00752</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00752" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFSmtLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=841-853&author=S.+Heauthor=J.+Xiaoauthor=A.+E.+Dulceyauthor=B.+Linauthor=A.+Roltauthor=Z.+Huauthor=X.+Huauthor=A.+Q.+Wangauthor=X.+Xuauthor=N.+Southallauthor=M.+Ferrerauthor=W.+Zhengauthor=T.+J.+Liangauthor=J.+J.+Marugan&title=Discovery%2C+optimization%2C+and+characterization+of+novel+chlorcyclizine+derivatives+for+the+treatment+of+hepatitis+C+virus+infection&doi=10.1021%2Facs.jmedchem.5b00752"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, Optimization, and Characterization of Novel Chlorcyclizine Derivatives for the Treatment of Hepatitis C Virus Infection</span></div><div class="casAuthors">He, Shanshan; Xiao, Jingbo; Dulcey, Andres E.; Lin, Billy; Rolt, Adam; Hu, Zongyi; Hu, Xin; Wang, Amy Q.; Xu, Xin; Southall, Noel; Ferrer, Marc; Zheng, Wei; Liang, T. Jake; Marugan, Juan J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">841-853</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, we reported that chlorcyclizine (CCZ, Rac-2), an over-the-counter antihistamine piperazine drug, possesses in vitro and in vivo activity against hepatitis C virus.  Here, we describe structure-activity relationship (SAR) efforts that resulted in the optimization of novel chlorcyclizine derivs. as anti-HCV agents.  Several compds. exhibited EC50 values below 10 nM against HCV infection, cytotoxicity selectivity indexes above 2000, and showed improved in vivo pharmacokinetic properties.  The optimized mols. can serve as lead preclin. candidates for the treatment of hepatitis C virus infection and as probes to study hepatitis C virus pathogenesis and host-virus interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogrp6Aiq03ybVg90H21EOLACvtfcHk0lgkKybvXYMFvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFSmtLfK&md5=0c93ba41c1d2780c47abb05aad2b88e7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00752%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DDulcey%26aufirst%3DA.%2BE.%26aulast%3DLin%26aufirst%3DB.%26aulast%3DRolt%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26atitle%3DDiscovery%252C%2520optimization%252C%2520and%2520characterization%2520of%2520novel%2520chlorcyclizine%2520derivatives%2520for%2520the%2520treatment%2520of%2520hepatitis%2520C%2520virus%2520infection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D841%26epage%3D853%26doi%3D10.1021%2Facs.jmedchem.5b00752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Schweitzer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span> </span><span class="NLM_article-title">Border control in hepatitis C virus infection: inhibiting viral entry</span> <span class="citation_source-journal">ACS Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">419</span><span class="refDoi"> DOI: 10.1021/acsinfecdis.5b00060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.5b00060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Citb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=416-419&author=C.+J.+Schweitzerauthor=T.+J.+Liang&title=Border+control+in+hepatitis+C+virus+infection%3A+inhibiting+viral+entry&doi=10.1021%2Facsinfecdis.5b00060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Border Control in Hepatitis C Virus Infection: Inhibiting Viral Entry</span></div><div class="casAuthors">Schweitzer, Cameron J.; Liang, T. Jake</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">416-419</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new era has begun in the treatment of hepatitis C virus (HCV) infection with powerful yet expensive therapies.  New treatments are emerging that target the entry step of HCV and could potentially block reinfection after liver transplant.  These treatments include antibodies, which target the virus or host receptors required by HCV.  Addnl., several new and previously approved small-mol. compds. have been described that target unique aspects of HCV entry.  Overall, the blocking entry represents an attractive strategy that could yield powerful combination therapies to combat HCV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfdrwdsj-3obVg90H21EOLACvtfcHk0lgkKybvXYMFvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Citb%252FM&md5=e0d42c28affc4910ddee9d58c35c7342</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.5b00060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.5b00060%26sid%3Dliteratum%253Aachs%26aulast%3DSchweitzer%26aufirst%3DC.%2BJ.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26atitle%3DBorder%2520control%2520in%2520hepatitis%2520C%2520virus%2520infection%253A%2520inhibiting%2520viral%2520entry%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D416%26epage%3D419%26doi%3D10.1021%2Facsinfecdis.5b00060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qing, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Z.</span><span> </span><span class="NLM_article-title">Discovery of imidazo [1, 2-α][1, 8] naphthyridine derivatives as potential HCV entry inhibitor</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">977</span><span class="NLM_x">–</span> <span class="NLM_lpage">981</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00159</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00159" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=977-981&author=H.+Wangauthor=S.+Wangauthor=L.+Chengauthor=L.+Chenauthor=Y.+Wangauthor=J.+Qingauthor=S.+Huangauthor=Y.+Wangauthor=X.+Leiauthor=Y.+Wuauthor=Z.+Ma&title=Discovery+of+imidazo+%5B1%2C+2-%CE%B1%5D%5B1%2C+8%5D+naphthyridine+derivatives+as+potential+HCV+entry+inhibitor&doi=10.1021%2Facsmedchemlett.5b00159"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00159%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DQing%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520imidazo%2520%255B1%252C%25202-%25CE%25B1%255D%255B1%252C%25208%255D%2520naphthyridine%2520derivatives%2520as%2520potential%2520HCV%2520entry%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D977%26epage%3D981%26doi%3D10.1021%2Facsmedchemlett.5b00159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Mittapalli, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKelvy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong-Staal, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macdonald, J. E.</span><span> </span><span class="NLM_article-title">Discovery of highly potent small molecule hepatitis C virus entry inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">6852</span><span class="NLM_x">–</span> <span class="NLM_lpage">6855</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2011.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1016%2Fj.bmcl.2011.09.019" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=6852-6855&author=G.+K.+Mittapalliauthor=A.+Jacksonauthor=F.+Zhaoauthor=H.+Leeauthor=S.+Chowauthor=J.+McKelvyauthor=F.+Wong-Staalauthor=J.+E.+Macdonald&title=Discovery+of+highly+potent+small+molecule+hepatitis+C+virus+entry+inhibitors&doi=10.1016%2Fj.bmcl.2011.09.019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DMittapalli%26aufirst%3DG.%2BK.%26aulast%3DJackson%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DChow%26aufirst%3DS.%26aulast%3DMcKelvy%26aufirst%3DJ.%26aulast%3DWong-Staal%26aufirst%3DF.%26aulast%3DMacdonald%26aufirst%3DJ.%2BE.%26atitle%3DDiscovery%2520of%2520highly%2520potent%2520small%2520molecule%2520hepatitis%2520C%2520virus%2520entry%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D6852%26epage%3D6855%26doi%3D10.1016%2Fj.bmcl.2011.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Baldick, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wichroski, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pendri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsh, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzucco, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pokornowski, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggers, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhai, W.</span><span> </span><span class="NLM_article-title">A novel small molecule inhibitor of hepatitis C virus entry</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1001086</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1001086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1371%2Fjournal.ppat.1001086" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1001086&author=C.+J.+Baldickauthor=M.+J.+Wichroskiauthor=A.+Pendriauthor=A.+W.+Walshauthor=J.+Fangauthor=C.+E.+Mazzuccoauthor=K.+A.+Pokornowskiauthor=R.+E.+Roseauthor=B.+J.+Eggersauthor=M.+Hsuauthor=W.+Zhai&title=A+novel+small+molecule+inhibitor+of+hepatitis+C+virus+entry&doi=10.1371%2Fjournal.ppat.1001086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1001086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1001086%26sid%3Dliteratum%253Aachs%26aulast%3DBaldick%26aufirst%3DC.%2BJ.%26aulast%3DWichroski%26aufirst%3DM.%2BJ.%26aulast%3DPendri%26aufirst%3DA.%26aulast%3DWalsh%26aufirst%3DA.%2BW.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DMazzucco%26aufirst%3DC.%2BE.%26aulast%3DPokornowski%26aufirst%3DK.%2BA.%26aulast%3DRose%26aufirst%3DR.%2BE.%26aulast%3DEggers%26aufirst%3DB.%2BJ.%26aulast%3DHsu%26aufirst%3DM.%26aulast%3DZhai%26aufirst%3DW.%26atitle%3DA%2520novel%2520small%2520molecule%2520inhibitor%2520of%2520hepatitis%2520C%2520virus%2520entry%26jtitle%3DPLoS%2520Pathog.%26date%3D2010%26volume%3D6%26spage%3De1001086%26doi%3D10.1371%2Fjournal.ppat.1001086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Besnard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruda, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abecassis, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguiz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norval, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sassano, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeons, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stojanovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prat, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seidah, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Constam, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bickerton, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetsel, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A. L.</span><span> </span><span class="NLM_article-title">Automated design of ligands to polypharmacological profiles</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">215</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.+P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+design+of+ligands+to+polypharmacological+profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lgz4gsA808e_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520design%2520of%2520ligands%2520to%2520polypharmacological%2520profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Lin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yim, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Z.</span><span> </span><span class="NLM_article-title">Evaluation of antiviral drug synergy in an infectious HCV system</span> <span class="citation_source-journal">Antiviral Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">603</span><span class="refDoi"> DOI: 10.3851/IMP3044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10.3851%2FIMP3044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=27035622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Olsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=595-603&author=B.+Linauthor=S.+Heauthor=H.+J.+Yimauthor=T.+J.+Liangauthor=Z.+Hu&title=Evaluation+of+antiviral+drug+synergy+in+an+infectious+HCV+system&doi=10.3851%2FIMP3044"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="#"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=&md5=9874b665cc7a056b8e2f928dd3112440</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3851%2FIMP3044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3851%252FIMP3044%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DYim%26aufirst%3DH.%2BJ.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DHu%26aufirst%3DZ.%26atitle%3DEvaluation%2520of%2520antiviral%2520drug%2520synergy%2520in%2520an%2520infectious%2520HCV%2520system%26jtitle%3DAntiviral%2520Ther.%26date%3D2016%26volume%3D21%26spage%3D595%26epage%3D603%26doi%3D10.3851%2FIMP3044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: An examination of in vitro half-life approach and nonspecific binding to microsomes</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1359</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350-1359&author=R.+S.+Obach&title=Prediction+of+human+clearance+of+twenty-nine+drugs+from+hepatic+microsomal+intrinsic+clearance+data%3A+An+examination+of+in+vitro+half-life+approach+and+nonspecific+binding+to+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0ljldmNnexf33A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520human%2520clearance%2520of%2520twenty-nine%2520drugs%2520from%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520data%253A%2520An%2520examination%2520of%2520in%2520vitro%2520half-life%2520approach%2520and%2520nonspecific%2520binding%2520to%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350%26epage%3D1359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-5"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00561">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_85384"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00561">10.1021/acs.jmedchem.7b00561</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Tables showing all synthetic intermediates, synthetic procedures, compound characterization for synthetic intermediates, and NIAID antiviral screen data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (SMILES) for final analogues (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_001.pdf">jm7b00561_si_001.pdf (771.48 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00561/suppl_file/jm7b00561_si_002.csv">jm7b00561_si_002.csv (6.14 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00561&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-14%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00561%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00561" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799beb66c4e25a5","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
