
<html lang="en"     class="pb-page"  data-request-id="616d21f5-4ee3-4881-9064-6a776521ce3b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2013.56.issue-12;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm4002988;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Interfacial Cavity Filling To Optimize CD4–Mimetic Miniprotein Interactions with HIV-1 Surface Glycoprotein" /></meta><meta name="dc.Creator" content="Laurence  Morellato-Castillo" /></meta><meta name="dc.Creator" content="Priyamvada  Acharya" /></meta><meta name="dc.Creator" content="Olivier  Combes" /></meta><meta name="dc.Creator" content="Johan  Michiels" /></meta><meta name="dc.Creator" content="Anne  Descours" /></meta><meta name="dc.Creator" content="Oscar H. P.  Ramos" /></meta><meta name="dc.Creator" content="Yongping  Yang" /></meta><meta name="dc.Creator" content="Guido  Vanham" /></meta><meta name="dc.Creator" content="Kevin K.  Ariën" /></meta><meta name="dc.Creator" content="Peter D.  Kwong" /></meta><meta name="dc.Creator" content="Loïc  Martin" /></meta><meta name="dc.Creator" content="Pascal  Kessler" /></meta><meta name="dc.Description" content="Ligand affinities can be optimized by interfacial cavity filling. A hollow (Phe43 cavity) between HIV-1 surface glycoprotein (gp120) and cluster of differentiation 4 (CD4) receptor extends beyond r..." /></meta><meta name="Description" content="Ligand affinities can be optimized by interfacial cavity filling. A hollow (Phe43 cavity) between HIV-1 surface glycoprotein (gp120) and cluster of differentiation 4 (CD4) receptor extends beyond r..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 11, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm4002988" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm4002988" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm4002988" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm4002988" /></link>
        
    
    

<title>Interfacial Cavity Filling To Optimize CD4–Mimetic Miniprotein Interactions with HIV-1 Surface Glycoprotein | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm4002988" /></meta><meta property="og:title" content="Interfacial Cavity Filling To Optimize CD4–Mimetic Miniprotein Interactions with HIV-1 Surface Glycoprotein" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0011.jpeg" /></meta><meta property="og:description" content="Ligand affinities can be optimized by interfacial cavity filling. A hollow (Phe43 cavity) between HIV-1 surface glycoprotein (gp120) and cluster of differentiation 4 (CD4) receptor extends beyond residue phenylalanine 43 of CD4 and cannot be fully accessed by natural amino acids. To increase HIV-1 gp120 affinity for a family of CD4-mimetic miniproteins (miniCD4s), we targeted the gp120 Phe43 cavity with 11 non-natural phenylalanine derivatives, introduced into a miniCD4 named M48 (1). The best derivative, named M48U12 (13), bound HIV-1 YU2 gp120 with 8 pM affinity and showed potent HIV-1 neutralization. It contained a methylcyclohexyl derivative of 4-aminophenylalanine, and its cocrystal structure with gp120 revealed the cyclohexane ring buried within the gp120 hydrophobic core but able to assume multiple orientations in the binding pocket, and the aniline nitrogen potentially providing a focus for further improvement. Altogether, the results provide a framework for filling the interfacial Phe43 cavity to enhance miniCD4 affinity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm4002988"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm4002988">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm4002988&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm4002988&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm4002988&amp;href=/doi/10.1021/jm4002988" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 5033-5047</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm4003457" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm400367n" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Interfacial Cavity Filling To Optimize CD4–Mimetic Miniprotein Interactions with HIV-1 Surface Glycoprotein</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Laurence++Morellato-Castillo">Laurence Morellato-Castillo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Priyamvada++Acharya">Priyamvada Acharya</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Olivier++Combes">Olivier Combes</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Johan++Michiels">Johan Michiels</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anne++Descours">Anne Descours</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oscar+H.+P.++Ramos">Oscar H. P. Ramos</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yongping++Yang">Yongping Yang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Guido++Vanham">Guido Vanham</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kevin+K.++Ari%C3%ABn">Kevin K. Ariën</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+D.++Kwong">Peter D. Kwong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lo%C3%AFc++Martin">Loïc Martin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Pascal++Kessler">Pascal Kessler</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Service d’Ingénierie Moléculaire des Protéines, iBiTecS, CEA, F-91191 Gif-sur-Yvette, France</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">§</span> <span class="aff-text">Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">‡</span> <span class="aff-text">Virology Unit, Department of Biomedical Sciences, Institute of Tropical Medicine—Antwerp, Nationalestraat 155, Antwerp 2000, Belgium</span></div><div class="corresp-info"><strong>*</strong>(P.K.) Phone: (33)-169084074. Fax: (33)-169089071. E-mail: <a href="/cdn-cgi/l/email-protection#fe8e9f8d9d9f92d0959b8d8d929b8cbe9d9b9fd0988c"><span class="__cf_email__" data-cfemail="5929382a3a383577323c2a2a353c2b193a3c38773f2b">[email protected]</span></a>. (L.M.) Phone: (33)-169087133. Fax: (33)-169089071. E-mail: <a href="/cdn-cgi/l/email-protection#94f8fbfdf7baf9f5e6e0fdfad4f7f1f5baf2e6"><span class="__cf_email__" data-cfemail="b9d5d6d0da97d4d8cbcdd0d7f9dadcd897dfcb">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm4002988&amp;href=/doi/10.1021%2Fjm4002988" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 5033–5047</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 27, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 March 2013</li><li><span class="item_label"><b>Published</b> online</span>11 June 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 June 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm4002988" title="DOI URL">https://doi.org/10.1021/jm4002988</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5033%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLaurence%2BMorellato-Castillo%252C%2BPriyamvada%2BAcharya%252C%2BOlivier%2BCombes%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D12%26contentID%3Djm4002988%26title%3DInterfacial%2BCavity%2BFilling%2BTo%2BOptimize%2BCD4%25E2%2580%2593Mimetic%2BMiniprotein%2BInteractions%2Bwith%2BHIV-1%2BSurface%2BGlycoprotein%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5047%26publicationDate%3DJune%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm4002988"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1150</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">17</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm4002988" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Interfacial Cavity Filling To Optimize CD4–Mimetic Miniprotein Interactions with HIV-1 Surface Glycoprotein&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Laurence&quot;,&quot;last_name&quot;:&quot;Morellato-Castillo&quot;},{&quot;first_name&quot;:&quot;Priyamvada&quot;,&quot;last_name&quot;:&quot;Acharya&quot;},{&quot;first_name&quot;:&quot;Olivier&quot;,&quot;last_name&quot;:&quot;Combes&quot;},{&quot;first_name&quot;:&quot;Johan&quot;,&quot;last_name&quot;:&quot;Michiels&quot;},{&quot;first_name&quot;:&quot;Anne&quot;,&quot;last_name&quot;:&quot;Descours&quot;},{&quot;first_name&quot;:&quot;Oscar&quot;,&quot;last_name&quot;:&quot;H. P. Ramos&quot;},{&quot;first_name&quot;:&quot;Yongping&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Guido&quot;,&quot;last_name&quot;:&quot;Vanham&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;K. Ariën&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;D. Kwong&quot;},{&quot;first_name&quot;:&quot;Loïc&quot;,&quot;last_name&quot;:&quot;Martin&quot;},{&quot;first_name&quot;:&quot;Pascal&quot;,&quot;last_name&quot;:&quot;Kessler&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;5033-5047&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm4002988&quot;},&quot;abstract&quot;:&quot;Ligand affinities can be optimized by interfacial cavity filling. A hollow (Phe43 cavity) between HIV-1 surface glycoprotein (gp120) and cluster of differentiation 4 (CD4) receptor extends beyond residue phenylalanine 43 of CD4 and cannot be fully accessed by natural amino acids. To increase HIV-1 gp120 affinity for a family of CD4-mimetic miniproteins (miniCD4s), we targeted the gp120 Phe43 cavity with 11 non-natural phenylalanine derivatives, introduced into a miniCD4 named M48 (1). The best derivative, named M48U12 (13), bound HIV-1 YU2 gp120 with 8 pM affinity and showed potent HIV-1 neutralization. It contained a methylcyclohexyl derivative of 4-aminophenylalanine, and its cocrystal structure with gp120 revealed the cyclohexane ring buried within the gp120 hydrophobic core but able to assume multiple orientations in the binding pocket, and the aniline nitrogen potentially providing a focus for further improvement. Altogether, the results provide a framework for filling the interfacial Phe43 cavity to enh&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm4002988&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm4002988" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm4002988&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm4002988" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm4002988&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm4002988" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm4002988&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm4002988&amp;href=/doi/10.1021/jm4002988" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm4002988" /></input><a href="/doi/pdf/10.1021/jm4002988" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm4002988&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm4002988%26sid%3Dliteratum%253Aachs%26pmid%3D23710622%26genre%3Darticle%26aulast%3DMorellato-Castillo%26date%3D2013%26atitle%3DInterfacial%2BCavity%2BFilling%2BTo%2BOptimize%2BCD4%25E2%2580%2593Mimetic%2BMiniprotein%2BInteractions%2Bwith%2BHIV-1%2BSurface%2BGlycoprotein%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D12%26spage%3D5033%26epage%3D5047%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292035" title="Viruses">Viruses</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292541" title="Cavities">Cavities</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jmcmar.2013.56.issue-12/production/jmcmar.2013.56.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Ligand affinities can be optimized by interfacial cavity filling. A hollow (Phe43 cavity) between HIV-1 surface glycoprotein (gp120) and cluster of differentiation 4 (CD4) receptor extends beyond residue phenylalanine 43 of CD4 and cannot be fully accessed by natural amino acids. To increase HIV-1 gp120 affinity for a family of CD4-mimetic miniproteins (miniCD4s), we targeted the gp120 Phe43 cavity with 11 non-natural phenylalanine derivatives, introduced into a miniCD4 named M48 (<b>1</b>). The best derivative, named M48U12 (<b>13</b>), bound HIV-1 YU2 gp120 with 8 pM affinity and showed potent HIV-1 neutralization. It contained a methylcyclohexyl derivative of 4-aminophenylalanine, and its cocrystal structure with gp120 revealed the cyclohexane ring buried within the gp120 hydrophobic core but able to assume multiple orientations in the binding pocket, and the aniline nitrogen potentially providing a focus for further improvement. Altogether, the results provide a framework for filling the interfacial Phe43 cavity to enhance miniCD4 affinity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01456" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01456" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cell entry of the human immunodeficiency virus type 1 (HIV-1) is a multistep process initiated by the binding of the HIV-1 surface glycoprotein (gp120) on the viral spike to the cluster of differentiation 4 (CD4) receptor on appropriate target cells. This association stabilizes a conformation of gp120 that enables it to bind to a chemokine coreceptor, CCR5 or CXCR4.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Coreceptor binding in turn activates the fusogenic properties of the noncovalently associated transmembrane component (gp41) of the viral spike, and subsequent spike-conformational rearrangements and virus–host membrane fusion lead to entry of the HIV-1 genetic material into the cell.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Each of these steps represents a potential target for therapeutic intervention.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8">(5-8)</a></div><div class="NLM_p">For over a decade, we have been developing peptide CD4 mimics, called miniCD4s, which are based on the transplantation of the gp120-binding surface of CD4 into the context of various scorpion-toxin scaffolds.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> These disulfide-rich peptides belong to the knottin family, which has been widely utilized as scaffolds for developing stable peptide ligands through loop grafting of bioactive peptides (see ref <a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">10</a> and references therein). The 31-residue scyllatoxin, from the scorpion <i>Leiurus quinquestriatus hebraeus</i>, was selected based on the similarity of the gp120-binding portion of CD4 with the scyllatoxin structure, which consists of a β-hairpin linked to a short helix. The scyllatoxin β-hairpin between residues 18–29<sub>scyllatoxin</sub> superimposes with a root-mean-square deviation (RMSD) of 1.1 Å with CD4 residues 36–47<sub>CD4</sub>, which are involved in binding gp120<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> (to aid in clarity, residues are identified by a subscript of the parent macromolecule). Moreover, the scyllatoxin fold, stabilized by three disulfide bridges (<a class="ref internalNav" href="#notes-2" aria-label="Figure S1, Supporting Information">Figure S1, Supporting Information</a>),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> is highly permissive to mutation,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> stable to acidic pH and high temperatures,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> relatively resistant toward proteolytic degradation,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and poorly immunogenic.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Optimization of the scyllatoxin peptide structure led to miniCD4s of only 27 residues and affinities for gp120 comparable to that of CD4 itself.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> These miniproteins stabilize the CD4-bound conformation of gp120 as observed by X-ray crystallography<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> as well as by immunogenic elicitation: thus, cross-linking of a miniCD4 to HIV-1 gp120 or HIV-1 gp140 (gp120 coupled to the extracellular part of gp41) enhances its binding to the CCR5 receptor, and injection of this cross-linked complex into rabbits significantly increases the production of CD4-induced antibodies that recognize the site of CCR5 binding on gp120.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> MiniCD4s are also able to neutralize HIV entry in vitro<a onclick="showRef(event, 'ref15 ref16 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref19">(15, 16, 19)</a> and in vivo, as shown recently in a macaque vaginal simian–human immunodeficiency virus (SHIV) challenge.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p last">An interfacial cavity, called the Phe43 cavity, extends from the tip of the gp120-interactive β-hairpin of CD4 into the heart of HIV-1 gp120. This mostly hydrophobic cavity is capped by the phenyl side chain of residue 43<sub>CD4</sub>. The miniCD4, named M48 (<b>1</b>),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> recognizes the entrance of the Phe43 cavity in a manner similar to that of CD4.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> To enhance the interaction between gp120 and miniCD4, we synthesized peptide CD4 mimics, in which the side chain of residue 23<sub>miniCD4</sub>, the miniCD4 residue positioned at the mouth of the Phe43 cavity, was replaced by a number of unnatural phenylalanine derivatives. The antiviral activity of four such peptides (<b>2</b>–<b>4</b> and <b>14</b>) was determined previously,<a onclick="showRef(event, 'ref15 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref19">(15, 19)</a> with <b>2</b>, which contains a methylcyclohexyl derivative of tyrosine at position 23, being the most potent. In a related study, we determine the structural basis for recognition of <b>2</b> by HIV-1 gp120 and show that optimal filling of the Phe43 cavity is a mechanism for enhancing affinity and neutralization potency of a CD4-binding site ligand.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> To gain a fuller understanding of how interactions with the Phe43 cavity might contribute to HIV-1 gp120 affinity, we synthesized miniCD4 derivatives of <b>1</b>, each with a unique 23<sub>miniCD4</sub> extension. This paper describes the synthesis of 11 novel unnatural amino acids, which were used to replace residue 23<sub>M48</sub>. Their incorporation, along with three commercially available unnatural amino acids, into <b>1</b>, allowed us to compare the relative affinity for 15 miniCD4 (including <b>1</b>) ligands to HIV-1, and we did so with two gp120s, one belonging to clade B and the other to clade C. The antiviral activity of the six tightest binding derivatives of <b>1</b> was assessed against two clade B lab strains, two clade C primary isolates, and two transmitter/founder (T/F) viruses, one belonging to clade B and the other to clade C. The tightest binder and most potent neutralizer M48U12 (<b>13</b>) was analyzed for its gp120-binding affinity by surface plasmon resonance (SPR) and crystallized in complex with HIV-1 gp120 from the clade B YU2 isolate. The cocrystal structure at 1.9 Å resolution provides atomic-level details for <b>13</b>, which displays both extraordinary affinity and potent neutralization. Together, these results show how appropriately filling of an interfacial cavity through a combination of rational design, chemical synthesis, and functional/structural feedback allows for potent near-pan neutralization of HIV-1 by miniCD4 mimetics.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80814" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80814" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Peptide Synthesis</h3><div class="NLM_p last">MiniCD4s were obtained by standard solid-phase peptide synthesis using Fmoc chemistry and by introducing various commercial or homemade amino acid derivatives at position 23<sub>miniCD4</sub>. After deprotection, the miniCD4s were folded by adding the crude peptide mixtures to Tris buffer (pH 8) containing the glutathione redox couple. Addition of 2 M guanidinium hydrochloride in the folding buffer was required to increase the solubility of the various linear peptides. MiniCD4s were purified by reverse phase HPLC. Two series of miniCD4s were synthesized. The first one was obtained on the M33<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (<b>16</b>) family (TpaNLHFCQLRCKSLGLLGKCAGSXCACV-NH<sub>2</sub>, where Tpa stands for thiopropionyl, and X for Phe, Trp, or unnatural amino acid) with commercially available amino acids. The second was built with three commercially available and eleven homemade unnatural amino acids on the more recently developed M48 (<b>1</b>) family (TpaNLHFCQLRCKSLGLLGRCA-DPro-TXCACV-NH<sub>2</sub>, where X stands for Phe or unnatural amino acid). We thus obtained a panel of twenty-four miniCD4s, some of which have been partially described previously.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Competitive Binding Assays on a M33 (<b>16</b>) Series</h3><div class="NLM_p">Relative apparent gp120-binding affinities of the various miniCD4s, modified at position 23 were determined by competitive ELISA, using soluble CD4 (sCD4) as competitor. The miniCD4s were tested on clade B gp120SF162 and on clade C gp120CN54. A preliminary study performed on the M33 (<b>16</b>) family, which contains commercially available natural or unnatural amino acids at position 23 showed systematically higher potency with various miniCD4s for the clade B virus compared to the clade C virus (<a class="ref internalNav" href="#notes-2" aria-label="Table S1, Supporting Information">Table S1, Supporting Information</a>; IC<sub>50</sub>CN54/IC<sub>50</sub>SF162 > 10). Most of the miniCD4s showed inhibition constants in the nanomolar range for clade B gp120SF162, but apart from the biphenyl- (<b>16</b>) or phenylalanine (<b>17</b>) derivatives, they all have micromolar IC<sub>50</sub>s for clade C gp120CN54. While the biphenyl derivative marginally improves the apparent affinity over the phenyl derivative, neither tryptophan nor the other six unnatural amino acids tested showed an improvement in affinity (see ratios <i>A</i> and <i>B</i> in <a class="ref internalNav" href="#notes-2" aria-label="Table S1">Table S1</a>). The modifications, whether extending the linker between the α carbon and the phenyl moiety (<b>18</b> and <b>19</b>) or using bicyclic residues (<b>20</b> and <b>21</b>), or introducing a methoxy (<b>23</b>), a hydrophobic methyl (<b>22</b>), or <i>tert-</i>butyl (<b>24</b>) group at the para position of the phenylalanine, all led to a loss in gp120-binding affinity of the miniCD4s. While both tryptophan and 2-naphthylalanine were deleterious, the latter was less deleterious, because <b>20</b> showed low nanomolar affinity for the SF162 surface protein. The most deleterious effect was observed on <b>24</b> and <b>18</b>, where the <i>tert-</i>butyl or the phenyl moiety, respectively, probably clash with the narrowest part of the pocket (this narrowing can be seen in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). Lengthening the linker between the α carbon and the aromatic ring, as was performed for <b>19</b>, enhanced the affinity compared to <b>18</b>, suggesting that the aromatic ring probably lies in the widest and deepest part of the cavity. The presence of a <i>p</i>-methyl group (<b>22</b>) on the phenylalanine resulted in a small loss of affinity for SF162 gp120, which could be reversed by replacing the <i>p</i>-methyl by a <i>p</i>-phenyl moiety (<b>16</b>), which again probably increases hydrophobic contacts with the deepest part of the pocket,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> while the indole entity in <b>21</b> appears to create steric hindrance at the inlet of the cavity.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0001.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HIV-1 YU2 gp120 binding to sCD4 and CD4 mimetic miniproteins. (A) gp120 binding to the CD4-mimetic miniproteins and to sCD4. gp120 is shown in cartoon representation. CD4 (shown as a transparent yellow surface) binds at the interface of three gp120 domains: inner domain (pale cyan), outer domain (pale green), and bridging sheet (light pink). Phe43<sub>CD4</sub> (yellow stick representation) inserts into a gp120 cavity, known as the Phe43 cavity, formed at the intersection of these domains. The miniCD4s are shown in red cartoon representation with the side chain of position 23 shown as sticks. Residue 23<sub>miniCD4</sub> inserts into the gp120 Phe43 cavity; (B) 90° rotated view of A. For clarity, the CD4 surface is not shown in this figure. (C) Zoomed-in cross-section of gp120 surface showing the Phe43 cavity with the bound inserts: Phe43 for CD4 (yellow, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RZK">1RZK</a>), Phe23 for <b>1</b> (wheat, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I60">2I60</a>), biphenylalanine 23 for <b>14</b> (slate, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I5Y">2I5Y</a>), Phe23 for <b>17</b> (brown, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YYM">1YYM</a>), biphenylalanine 23 for <b>16</b> (marine, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YYL">1YYL</a>). (D) 180° rotated view of C. For clarity, gp120 surface from only one complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I60">2I60</a>) is shown in C and D.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">While these results provided useful structure–activity relationships for ligand occupation of the gp120 Phe43 cavity, no improvements in affinity of <b>16</b> could be achieved using commercially available amino acids. We, therefore, decided to synthesize novel, unnatural amino acids for introduction into the more recently developed M48 (<b>1</b>) sequence.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">Novel unnatural amino acids suitable for Fmoc-SPPS were obtained using a synthetic route described previously,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> producing a series of ethers derived from the tyrosine amino acid, using the Mitsunobu reaction in solid-phase synthesis (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>). The tyrosine (<b>1</b>′) phenol and amino groups were first protected by an allyloxycarbonyl group. The obtained compound (<b>2</b>′) was immobilized onto a solid support (Wang resin) via its carboxylic moiety. Loading was estimated by cleavage and weight analysis of small samples of resin. <sup>1</sup>H NMR of crude aliquots confirmed the reaction progress. An average 87% loading was obtained for the bis-allyloxycarbonyl-protected compound (<b>3</b>′), which was close to the 91% previously obtained by Morley.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Piperidine treatment led to selective removal of the carbonate group under mild conditions, thereby avoiding detachment of the template from the solid support and deprotection of the allylcarbamate. With a 95% reaction yield, estimated by weighing the resin, an average loading of 1.03 mmol/g was obtained for the resin harboring the monoprotected allyloxycarbonyl derivative (<b>4</b>′). This selective removal of the phenolic protecting group generated an intermediate suitable for the synthesis of an ether library by using the Mitsunobu reaction. Commercially available alcohols were used to prepare resins <b>5</b>′–<b>12</b>′. Monoprotected diols <b>44</b>′–<b>46</b>′, used for the synthesis of resins <b>10</b>′–<b>12</b>′, were prepared by one protecting step from commercially available compounds (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>). To obtain resin <b>13</b>′, the <i>N</i>-Cbz-<span class="smallcaps smallerCapital">l</span>-proline (<b>47</b>′) carboxylic acid had first to be transformed into an alcohol function. This was performed by formation of a mixed anhydride with isobutyl chloroformate, followed by a reduction with sodium borohydride, yielding <i>N</i>-Cbz-(<i>S</i>)-(+)-2-pyrrolidinemethanol <b>48</b>′ (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>). The Mitsunobu reaction was carried out in a 1/1 mixture of DCM/THF with PPh<sub>3</sub>, diisopropyl azodicarboxylate (DIAD), and the various alcohols, in a double coupling strategy, as proton NMR showed that some phenol remained after the first coupling. Loading yield of the resins <b>5</b>′–<b>13</b>′ obtained after the Mitsunobu reaction as well as the yields for the next two steps was not determined. Instead, a global yield for these three steps was calculated at the end of the synthesis. The next step was the allyloxycarbonyl moiety removal from the nitrogen atom to reveal the primary amine, by activation with Pd(PPh<sub>3</sub>)<sub>4</sub> in mild conditions where other functional groups remained inert. Then, the resulting amino-resins <b>14</b>′–<b>22</b>′ were protected by a Fmoc moiety to give Fmoc-amino-acid resins <b>23</b>′–<b>31</b>′. The last step, i.e., cleavage from the solid support, was carried out under acidic conditions in a mixture of TFA/DCM. After extraction and filtration, the final Fmoc-protected amino acid compounds <b>32</b>′ to <b>40</b>′ were obtained with overall yields ranging between 26% and 63% for the last three steps.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0004.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Unnatural Alkoxy Derivatives of Phenylalanine Protected for Fmoc-Peptide Synthesis<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) ClCOOCH<sub>2</sub>CHCH<sub>2</sub>, 4 N NaOH; (b) Wang resin, diisopropylcarbodiimide, DMAP, NMP; (c) 20% piperidine, DMF; (d) ROH, PPh<sub>3</sub>, DIAD, DCM/THF; (e) Pd(PPh<sub>3</sub>)<sub>4</sub>, Ar, THF/DMSO/HCl/morpholine; (f) Fmoc-Cl, <i>i</i>-Pr<sub>2</sub>NEt, DCM; (g) TFA/DCM.</p></p></figure><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0005.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Monoprotection of Diol Derivative<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) BnBr, NaH, dry THF.</p></p></figure><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0006.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Alcohol <b>48</b>′<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) 4-Methylmorpholine, (CH<sub>3</sub>)<sub>2</sub>CHCH<sub>2</sub>OCOCl, DME; (b) NaBH<sub>4</sub>, DME/H<sub>2</sub>O.</p></p></figure><div class="NLM_p">Two Fmoc-protected amino acids, <b>54</b>′ and <b>57</b>′, were obtained by solution synthesis. For <b>54</b>′ (Scheme <a class="ref internalNav" href="#sch4" aria-label="4">4</a>), one of the two alcohol functions of 2,2′-thiodiethanol (<b>49</b>′) was first protected with a <i>tert-</i>butyldimethylsilyl moiety. The derivative that was obtained (<b>50</b>′) was coupled to Cbz-Tyr(OH)-OBn using the Mitsunobu reaction. The carboxylic and amine functions were then deprotected by catalytic reduction on Pd black, just before a standard Fmoc protection of the amine moiety. Finally, the sulfoxide (<b>54</b>′) was obtained in a 33% overall yield by oxidation with a <i>tert</i>-butyl hydroperoxide and thiourea dioxide mixture. The last unnatural amino acid that we developed was a methylcyclohexyl derivative of Fmoc-<i>p</i>-aminophenylalanine (<b>57</b>′). It is an analogue of <b>32</b>′, with a nitrogen atom leading to a phenylalkylamine replacing the oxygen atom of the phenyl ether moiety. It was synthesized by reductive amination on Fmoc-<i>p</i>-amino-<span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>55</b>′) using sodium triacetoxyborohydride in CH<sub>3</sub>CN/AcOH with cyclohexanecarboxaldehyde (Scheme <a class="ref internalNav" href="#sch5" aria-label="5">5</a>). The secondary amine that was obtained was then protected with the <i>tert</i>-butyloxycarbonyl moiety compatible with Fmoc peptide chemistry. The overall yield of <b>57</b>′ was 73%. Fmoc-protected amino acid <b>59</b>′ was obtained by reaction of the commercially available amino acid <b>58</b>′ with Fmoc-chloroformate (Scheme <a class="ref internalNav" href="#sch6" aria-label="6">6</a>).</div><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0007.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the Sulfoxide Fmoc-Protected Amino Acid <b>54</b>′<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) TBSCl, NaH, dry THF; (b) Z-Tyr-OBn, PPh<sub>3</sub>, DEAD, DMF/toluene; (c) Pd black, H<sub>2</sub>, MeOH; (d) Fmoc-<i>O</i>-succinimide, NaHCO<sub>3</sub>, DMF, CH<sub>3</sub>CN, H<sub>2</sub>O; (e) TBHP, thiourea dioxide, DCM.</p></p></figure><figure id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0008.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Fmoc-Protected Amino Acid <b>57</b>′<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) C<sub>6</sub>H<sub>11</sub>CHO, NaBH(OAc)<sub>3</sub>, CH<sub>3</sub>CN/AcOH; (b) (Boc)<sub>2</sub>O, <i>i</i>-Pr<sub>2</sub>NEt, THF.</p></p></figure><figure id="sch6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0009.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Fmoc-Protected Amino Acid <b>59</b>′<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) Fmoc-Cl, Na<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Competitive Binding Assays on an M48 (<b>1</b>) Series</h3><div class="NLM_p">Competitive ELISA assay showed the global superiority of the M48 (<b>1</b>) series over the M33 (<b>16</b>) series (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> compared to <a class="ref internalNav" href="#notes-2" aria-label="Table S1">Table S1</a>). Eight derivatives shared subnanomolar and five low nanomolar apparent affinities for the clade B SF162 strain. Moreover, two derivatives had subnanomolar and five low nanomolar affinities for the clade C strain. Four derivatives (<b>1</b>, <b>11</b>, <b>12</b>, <b>14</b>) were made with commercially available amino acids, and as was observed for the M33 (<b>16</b>) series, no improvement in binding affinity to gp120, over that of <b>1</b>, could be achieved for <b>11</b>, <b>12</b>, or <b>14</b> (previously called M47<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>). Moreover, the results obtained with <b>12</b> confirmed the data obtained for <b>24</b>, as these <i>p-tert</i>-butylphenylalanine derivatives share the worst IC<sub>50</sub> values for both gp120s. However, the affinity loss, on changing the phenylalanine residue for a <i>p-tert</i>-butylphenylalanine, is much higher in the M48 (<b>1</b>) family versus the M33 (<b>16</b>) family (26-fold higher for gp120SF162 and about 13-fold higher for gp120CN54). Interestingly, replacing the <i>p-tert</i>-butyl group by a slightly less bulky isopropyl group (<b>11</b>) enhanced the apparent affinity on SF162 by a factor of 7, while no difference was observed on the C strain gp120, suggesting differences in the Phe43 cavity shape between these two gp120 proteins. Notably, this phenomenon is also present for <b>2</b>, <b>4</b>, <b>11</b>, <b>13</b>, and <b>15</b>, with their <i>B</i>/<i>A</i> ratio (impact of the miniCD4 amino acid modification on position 23 on the affinity for gp120 on clade C compared to clade B; see Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) deviating significantly from 1. While for the M33 (<b>16</b>) series, the biphenyl moiety marginally increased the apparent binding affinity to gp120 compared to the phenyl group (<a class="ref internalNav" href="#notes-2" aria-label="Table S1">Table S1</a>; <i>A</i> = 0.9, <i>B</i> = 0.6), the effect was opposite in the M48 (<b>1</b>) family (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>; <i>A</i> = 2.6, <i>B</i> = 5.3), with the biphenyl substitution resulting in loss in gp120 binding affinity. X-ray crystallographic studies have already been published for the complexes between the clade B YU2 gp120 and <b>16</b>, <b>17</b> (previously called F23),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a><b>1</b> (previously called [Phe23M47]), and <b>14</b> (previously called M47).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> These data are consistent with the overall small differences between these representative miniCD4s observed in our binding assays, especially on the clade B strain, as the maximum root-mean-square deviation of gp120 is within 0.5 Å for the biphenyl group and 0.3 Å for the phenyl moiety, when comparing the two families (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). Of the miniCD4s built with the eleven novel unnatural amino acids synthesized in our lab, six showed enhanced affinity for gp120SF162, compared to <b>1</b>, and four also showed an affinity increase for the clade C gp120CN54. Interestingly, extending the linker between the phenyl ether and the cyclohexyl moiety one carbon atom at a time, going from <b>2</b> to <b>5</b> and then to <b>6</b>, demonstrated that the shorter the linker, the better the affinity for the HIV-1 surface protein. A small loss of affinity was observed at each step, and the effect, once again, was greater on the clade C strain. Exchanging the cyclohexyl moiety in <b>5</b> for a phenyl group gave opposite results on the two strains, with an increase in affinity for gp120SF162 but slightly decreased affinity for gp120CN54. On the other hand, exchanging the cyclohexyl moiety for a less bulky cyclopentyl group (<b>3</b>) resulted in enhanced affinity for both strains. The difference between <b>2</b> and <b>10</b> is the replacement of the cyclohexyl group for a pyrrolidine, which again engendered a higher loss in affinity for clade C CN54 than for clade B SF162 gp120 surface proteins. Filling the Phe43 cavity with linear flexible ethers gave miniCD4s with high affinity for gp120, notably <b>8</b> and <b>9</b>, while a shorter linker between the alcohol function and the phenyl group, as for <b>7</b>, led to a small loss of affinity. Oxidation of the thioether in <b>9</b> to the racemic sulfoxide in <b>15</b> was dramatically deleterious for the interaction between the miniCD4 and gp120CN54 and dropped the affinity by a factor of over 300-fold, whereas affinity for the SF162 strain was reduced by a factor of 29. This large difference in affinities between the two gp120 proteins due to this small chemical modification in the miniCD4 was unexpected (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>; <i>B</i>/<i>A</i> = 28). The two best miniCD4s identified in this study were <b>13</b> and the previously published miniprotein <b>2</b>,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> both methylcyclohexyl derivatives of either an aniline (<b>13</b>) or a phenol (<b>2</b>) entity. The NHCH<sub>2</sub> or OCH<sub>2</sub> linkers appear to have the right length to place the cyclohexyl moiety in the largest part of the Phe43 cavity. The nonplanar shape of the cyclohexane ring possibly allows better hydrophobic contacts with the deepest part of the pocket than that of the planar phenyl group in <b>14</b>. Interestingly, these two miniCD4s are the only ones that share almost the same subnanomolar affinities for both gp120s (see Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, IC<sub>50</sub>CN54/IC<sub>50</sub>SF162 ≈ 1), suggesting that the methylcyclohexyl moiety contacts the regions of the cavity that are conserved between the two gp120s.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. ELISA IC<sub>50</sub> Values, Obtained by Competition against sCD4 on gp120SF162 and gp120CN54, of the MiniCD4s Belonging to the M48 Series</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0010.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Extrapolated values (maximum concentration tested was 10 mM).</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">Previously called CD4M47<sup>15</sup> or M47.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></p></div><div class="footnote" id="t1fn3"><sup>Table c</sup><p class="last">Mentioned in ref 19.</p></div><div class="footnote" id="t1fn4"><sup>Table d</sup><p class="last">New compounds. IC<sub>50</sub> values are presented as the mean ± SEM values from three independent experiments.</p></div></div><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Antiviral Activity and Cytotoxicity</h3><div class="NLM_p">In addition to competitive binding assays, the antiviral activity of those CD4-mimetics with highest binding affinities was determined against two subtype B and two subtype C viruses. In a previous paper<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> we described the antiviral activity of <b>2</b>, <b>3</b>, and <b>4</b> in pseudoviruses by using GHOST target cells that express CD4 and either CCR5 or CXCR4 HIV coreceptor. Nevertheless, we tested these molecules again in this study using a TZM-bl and replication-competent HIV and showed that the results are similar in both assays (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), particularly for Bal, IIIB, and VI829 viruses. Minor differences were however observed for VI1358, as the EC<sub>50</sub>s are about 1 order of magnitude smaller. Nanomolar EC<sub>50</sub>s were found for all six M48 miniCD4s tested on lab strain clade B viruses (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Overall, the highest activity was associated with miniproteins <b>2</b>, <b>3</b>, and <b>13</b> against subtype B HIV-1. Of note, all tested M48 miniCD4s were more active against the CXCR4-tropic strain IIIB, having low nanomolar or picomolar EC<sub>50</sub>s. While <b>4</b>, <b>8</b>, and <b>9</b> lost potency against subtype C viruses, <b>2</b>, <b>3</b>, and <b>13</b> could still inhibit the entry of the HIV-1 subtype C virus. These miniCD4s were also tested against two transmitted/founder viruses (T/F, subtypes B and C) (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Interestingly, while <b>4</b>, <b>8</b>, and <b>9</b> were highly active against the subtype B lab strains, their antiviral activity was dramatically reduced against the T/F subtype B strain. Although the EC<sub>50</sub> of <b>2</b>, <b>3</b>, and <b>13</b> against the T/F virus was higher than the EC<sub>50</sub> values against other viruses, submicromolar activity was retained. The most notable difference between <b>2</b> and <b>13</b> was against the T/F clade C strain, where <b>13</b> with an EC<sub>50</sub> value of 32 nM was 8.5-fold more active than <b>2</b>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Antiviral Activity of CD4 Miniproteins against HIV-1 Lab Strains (Bal, IIIB), Primary Isolates (VI829, VI1358), and Transmitted/Founder Viruses (pREJO.c/2864, p246F10) and Cytotoxicity of the MiniCD4s</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="center" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="6" align="center">EC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">miniCD4</th><th class="colsep0 rowsep0" align="center">Bal (B, R5)</th><th class="colsep0 rowsep0" align="center">IIIB (B, X4)</th><th class="colsep0 rowsep0" align="center">pREJO.c/2864 (T/F, B, R5)</th><th class="colsep0 rowsep0" align="center">VI829 (C, R5)</th><th class="colsep0 rowsep0" align="center">VI1358 (C, R5)</th><th class="colsep0 rowsep0" align="center">p246F10 (T/F, C, R5)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">1 (2)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.1 (0.2)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">17 (11)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">130 (5)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">272</td><td class="colsep0 rowsep0" align="left">>50000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">3 (4)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.3 (0.5)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">68</td><td class="colsep0 rowsep0" align="left">103 (98)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">826 (42)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">898</td><td class="colsep0 rowsep0" align="left">>15000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">22 (67)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">1.2 (3)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000 (951)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>1000 (364)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>15000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">682</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>50000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">48</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>50000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left">9.8</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">>40000</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">In parentheses: results obtained with GHOST cells from a previous paper.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></p></div></div></div><div class="NLM_p last">Cytotoxicity of the same series of miniCD4s was evaluated using the water-soluble tetrazolium-1 (WST-1) cell proliferation assay on TZM-bl cells. With minimum CC<sub>50</sub> values of 15 μM, no significant cytotoxicity was associated with the six CD4 miniproteins tested (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Surface Plasmon Resonance (SPR) Analysis</h3><div class="NLM_p">Results from the competitive binding and virological assays described above establish knottin <b>13</b> as the most potent miniCD4 synthesized to date, surpassing the in vitro biological properties of the closely related miniprotein <b>2</b>, which has demonstrated efficacy as a microbicide in macaque challenge models.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> To determine the affinity of interaction of <b>13</b> with HIV-1 gp120, we performed SPR analysis with three Tier 2 HIV-1 viruses namely, clade B isolates YU2 and TRO.11, and clade C isolate ZM135. We compared <b>13</b> affinity for HIV-1 gp120 with that of the closely related miniCD4 <b>2</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, and <a class="ref internalNav" href="#notes-2" aria-label="Figure S2, Supporting Information">Figure S2, Supporting Information</a>). In a recently published manuscript<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> we reported an affinity for peptide <b>2</b> of 15.4 ± 0.8 pM for the clade B YU2 isolate. In this study we performed the SPR assays with extended dissociation time (30 min) to obtain more accurate estimates of affinity of these tight binding complexes. We obtained an affinity of 18.1 ± 0.1 pM for <b>2</b> binding to YU2 gp120, comparable to the value obtained in the related study.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> CD4-mimetic <b>13</b> showed an affinity of 8.4 ± 0.2 pM for YU2 gp120, with the increase in affinity primarily a result of an enhanced association rate. For TRO.11, the other clade B isolate tested, a ∼14-fold enhancement in affinity for <b>13</b> relative to <b>2</b>, derived from both an increased on-rate as well as a reduced off-rate of binding, the latter suggesting the formation of a more stable complex with <b>13</b>. At 6.20 × 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup>, the on-rate of <b>13</b> binding to TRO.11 gp120 is ∼3-fold enhanced compared to that of <b>2</b>, and at 1.10 × 10<sup>–4</sup> s<sup>–1</sup>, its off-rate is ∼5-fold slower than that of <b>2</b>. For the clade C ZM135 isolate, 16 ± 1 pM binding affinity of <b>13</b> to gp120 is a ∼3-fold enhancement over <b>2</b>, with the enhancement coming primarily as a result of a 3-fold increase in the on-rate of binding.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Binding Parameters of MiniCD4s <b>2</b> and <b>13</b> to Full-Length gp120 from HIV-1 Clade B Isolates YU2 and TRO.11, and Clade C Isolate ZM135</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">on-rate (M<sup>-1</sup> s<sup>-1</sup>)</th><th class="colsep0 rowsep0" align="center">off-rate (s<sup>-1</sup>)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>D</sub> (pM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center">YU2 (clade B)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">3.82 × 10<sup>6</sup> ± 2.0 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">6.87 × 10<sup>–5</sup> ± 2.0 × 10<sup>–6</sup></td><td class="colsep0 rowsep0" align="left">18.1 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">2.70 × 10<sup>7</sup> ± 2.0 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">2.25 × 10<sup>–4</sup> ± 1.0 × 10<sup>–6</sup></td><td class="colsep0 rowsep0" align="left">8.4 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center">TRO.11 (clade B)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">2.04 × 10<sup>6</sup> ± 2.0 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">5.26 × 10<sup>–4</sup> ± 4.0 × 10<sup>–6</sup></td><td class="colsep0 rowsep0" align="left">257 ± 11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">6.20 × 10<sup>6</sup> ± 3.0 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">1.10 × 10<sup>–4</sup> ± 5.0 × 10<sup>–6</sup></td><td class="colsep0 rowsep0" align="left">17.7 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center">ZM135 (clade C)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">2.34 × 10<sup>6</sup> ± 1.0 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">1.12 × 10<sup>–4</sup> ± 4.0 × 10<sup>–6</sup></td><td class="colsep0 rowsep0" align="left">47 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">6.84 × 10<sup>6</sup> ± 3.0 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">1.08 × 10<sup>–4</sup> ± 5.0 × 10<sup>–5</sup></td><td class="colsep0 rowsep0" align="left">16 ± 1</td></tr></tbody></table></div></div><div class="NLM_p last">The SPR results show that, for all three HIV-1 strains tested, peptide <b>13</b> binds gp120 with greater affinity than <b>2</b>, although the degree of enhancement was isolate dependent. Such isolate dependence is consistent with observations of isolate-specific differences observed with the relative binding affinities obtained by CD4 competition ELISA. With all three isolates tested, an increase in rate of association of <b>13</b> with gp120 was observed, and with the clade B TRO.11 gp120 we also observed a ∼5-fold reduction in the off-rate.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Cocrystal Structure of <b>13</b> in Complex with HIV-1 gp120</h3><div class="NLM_p">To obtain an atomic-level understanding of the extraordinary affinity and neutralization breadth of <b>13</b>, we crystallized miniCD4 <b>13</b> with an extended core version of gp120 from the HIV-1 clade B YU2 strain.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Crystals diffracted to 1.9 Å, and the structure was solved by molecular replacement and refined to an <i>R</i><sub>free</sub>/<i>R</i><sub>work</sub> of 22.4%/16.8% (<a class="ref internalNav" href="#notes-2" aria-label="Table S4, Supporting Information">Table S4, Supporting Information</a>).</div><div class="NLM_p">Overall, and as expected, the conformation of gp120 bound to <b>13</b> resembled the conformation of gp120 bound to CD4 or other previously structurally characterized miniCD4s<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> with an RMSD variation ranging from 0.3 to 0.8 Å. Miniprotein <b>13</b> bound to the CD4 binding site of gp120 (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A), with the non-natural amino acid at position 23 inserting its side chain into the gp120 Phe43 cavity (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B) at the intersection of three gp120 domains: the inner domain, the outer domain, and the bridging sheet. In a related study, we structurally characterized gp120 binding of the closely related miniCD4 <b>2</b>, which contains a methylcyclohexyl derivative of tyrosine at position 23.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Miniprotein <b>13</b> contains a methylcyclohexyl derivative of 4-aminophenylalanine at position 23. Superposing the gp120 molecules in the <b>2</b>–gp120 and <b>13</b>–gp120 complexes revealed close overall similarity in the environment around the Phe43 cavity in the two complexes, including the near-identical placement of the water molecules in the conserved gp120 solvent channel (<a class="ref internalNav" href="#notes-2" aria-label="Figure S3, Supporting Information">Figure S3, Supporting Information</a>). The orientation of the inserted side chain of <b>2</b> and <b>13</b> into the Phe43 cavity revealed subtle differences, however, in the two complexes.</div><figure id="fig2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0002.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. MiniCD4 <b>13</b> binds at the CD4-binding surface and extends into the gp120 Phe43 cavity. (A) gp120 is shown in gray surface representation with the CD4 binding footprint colored yellow. Knottin <b>13</b> is depicted in cartoon representation with the miniprotein colored red except residue 23, which is colored by element type with the nitrogen atoms colored blue and the carbon atoms of one conformation colored orange and the other magenta. (B) A cross-section of gp120 showing the side chain of residue 23 of peptide <b>13</b> penetrating the gp120 Phe43 cavity, packing coefficients, and surface complementarity for each alternate conformation. Packing coefficient (%) = (volume of ligand/volume of binding pocket) × 100. The colored dots span the region of the ligand used for each calculation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The overall orientation of the side chain in the Phe43 cavity is unambiguously defined by the electron density in the miniprotein-<b>13</b>-bound cavity (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A–C and <a class="ref internalNav" href="#notes-2" aria-label="Figure S3">Figure S3</a>), with the anilino group at position 23 appropriately positioned for interacting with the conserved gp120 solvent channel by hydrogen bonding with the proximal water molecule (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>D and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>E). While the overall definition of the insert in the Phe43 cavity is clear for <b>13</b>, its cyclohexane ring displayed asymmetric electron density, indicative of local ligand disorder in the bound state (Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A–C, <a class="ref internalNav" href="#notes-2" aria-label="Figure S3 and Figure S4, Supporting Information">Figure S3 and Figure S4, Supporting Information</a>). Interestingly, the electron density indicates mobility or disorder at similar positions as was observed in different crystal forms of gp120-bound <b>2</b> (<a class="ref internalNav" href="#notes-2" aria-label="Figure S3">Figure S3</a>), suggesting a precise and conserved recognition mode in the Phe43 cavity that allows these Phe43 cavity inserts to retain conformational mobility. The monosubstituted cyclohexane ring was modeled in the chair conformation with the sole substitution that connects the ring to the rest of the miniprotein placed in an equatorial orientation, thereby defining the cyclohexane conformation. The electron density in the cavity appeared to be well modeled by rotations about the flexible linker that connects the cyclohexane ring to the phenyl ring of <b>13</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>F).</div><figure id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0003.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. MiniCD4 <b>13</b> binds at the intersection of gp120 inner domain, outer domain, and bridging sheet. (A–C) Zoomed-in views of <b>13</b> bound to the gp120 Phe43 cavity. gp120 is shown in cartoon representation and colored by domain. Inner domain, outer domain, and bridging sheet are colored cyan, green, and pink, respectively. MiniCD4 <b>13</b> is similarly represented as in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>. A gray mesh surrounding the residue depicts a 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map at 2.0σ defining the position of residue 23. In A–C, the cartoon is made transparent to highlight the placement of the residue 23 side chain in the electron density. (D) MiniCD4 <b>13</b> interacting with a conserved gp120 solvent channel. The red spheres are water molecules located in the solvent channel that starts from inside the Phe43 cavity, traverses under the bridging sheet, and emerges at the CD4-induced coreceptor binding site of gp120. (E) LIGPLOT diagram illustrating the environment around <b>13</b> in the gp120 Phe43 cavity in two-dimensional representation and showing the <b>13</b> hydrogen bonding with the conserved gp120 solvent channel. Only one side chain conformation is shown for clarity. (F) 2D structure of the side chain of <b>13</b> with arrows showing the bonds that rotate and result in multiple conformations bound to the gp120 Phe43 cavity. Riding hydrogen atoms used during refinement are shown in white.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The cyclohexane ring was packed against a hydrophobic patch in the Phe43 cavity composed of atoms from residues Val255, Trp112, Phe382, and Ile424. The observation of residual ligand entropy in the gp120 Phe43 cavity was reminiscent of synthetic host–guest systems, where an enthalpic–entropic balance was found to be critical to achieve optimal binding.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> For complexes in confined apolar cavities, it has been surmised that optimal stability is attained when the guest (or ligand) occupies 55 ± 9% of the cavity volume.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> This rule, first observed in synthetic host–guest systems, was later found to also apply to enzymatic systems.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> To determine whether this rule also applies to miniCD4 <b>13</b> binding to the predominantly apolar gp120 Phe43 cavity, we quantified the packing efficiency of its insert into the Phe43 cavity by calculating the packing coefficient, which is the fraction of the pocket volume occupied by the ligand. We also calculated shape complementarity, which related the goodness of fit between protein and ligand surfaces. We observed that the insert moiety occupies 80% of the available volume in the Phe43 cavity (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B, and <a class="ref internalNav" href="#notes-2" aria-label="Figure S5, Supporting Information">Figure S5, Supporting Information</a>). The cyclohexane ring, on the other hand, only fills 66% of the volume of the cavity to which it binds. The observed packing coefficient for the cyclohexane ring of 66% is just above the conjectured optimum of 55 ± 9%. Taken together, these calculations show that, while the overall occupation of the Phe43 cavity by the insert of <b>13</b> is enthalpy driven and suggestive of tight interactions of the ligand with the protein, local regions of looser packing allow retention of entropy in the bound state. In addition, results from packing coefficient calculations appear to be consistent with shape complementarity analysis: the cyclohexane ring shows lower shape complementarity compared to the overall insert moiety, where tight packing results in high shape complementarity.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02732" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02732" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Although crystal structures of full-length gp120 or trimeric HIV envelope protein gp160 are not yet available, the three-dimensional structures of HIV gp120 core variants have been determined in the CD4-bound conformation<a onclick="showRef(event, 'ref29 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32 ref33">(29-33)</a> as well as in the unliganded state.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> CD4 binding stabilizes a cavity of roughly 150 Å<sup>3</sup>, which penetrates about 14 Å into the gp120 hydrophobic core at the intersection of three gp120 domains. The Phe43<sub>CD4</sub> side chain caps the entrance of this pocket, hence the Phe43 cavity name. However, this pocket is induced not only by the CD4 receptor but also by CD4-binding-site antibodies<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and by synthetic entry inhibitors<a onclick="showRef(event, 'ref24 ref35'); return false;" href="javascript:void(0);" class="ref ref24 ref35">(24, 35)</a> including miniaturized CD4-mimetics,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> called miniCD4s. To increase the affinity of the miniCD4s for the gp120 surface protein and to improve their ability to neutralize HIV-1, we targeted the gp120 Phe43 cavity with various unnatural amino acid side chains. While the effect of modifying position 23 on the miniCD4 was often not identical on the two HIV-1 isolates tested, SF162 and CN54 (see ratio <i>B</i>/<i>A</i> in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref internalNav" href="#notes-2" aria-label="S1">S1</a>), the overall trends were similar. For most derivatives, the variation of affinity compared to knottins <b>17</b>, in the M33 series, or <b>1</b>, in the M48 series, is more pronounced on the CN54 strain than on SF162. In both strains, all the cavity-lining residues are identical, except for Arg426 (HXBc2 numbering) from strain SF162 and Met426 from strain CN54 (<a class="ref internalNav" href="#notes-2" aria-label="Figure S6, Supporting Information">Figure S6, Supporting Information</a>). With the nonconserved residue 426, the Phe43 cavity was lined by its invariant main chain component, with the nonconserved side chain facing away from the cavity. The differences in recognition of SF162 and CN54 thus likely relate to isolate-specific differences in Phe43 cavity shape, due to either the variation in residue 426 or the effect of residue variation outside the Phe43 cavity.</div><div class="NLM_p">Because commercially available amino acids proved to be of limited utility for improving the activity of the miniCD4s, we developed unnatural residues designed to fill the gp120 Phe43 cavity. Of the miniCD4s synthesized and tested thus far, knottin <b>13</b> showed the highest affinity for gp120s tested as well as the most potent antiviral activity on laboratory-adapted strains, primary isolates, and transmitter/founder viruses.</div><div class="NLM_p">A cavity-filling analysis was previously performed on the CD4 mutant, called D1D2F43C,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> in which various chemical moieties were attached to the free cysteine at residue 43<sub>CD4</sub> (analogous to residue 23<sub>miniCD4</sub>). One of the best derivatives of D1D2F43C contained a cyclohexyl group at a distance from residue 23<sub>miniCD4</sub> Cα similar to that of <b>2</b> and <b>13</b>. Introduction of the cyclohexyl group to D1D2F43C increased affinity ∼30-fold, to a level similar to the unmodified D1D2 (sCD4) case.</div><div class="NLM_p">In the current study, introduction of a cyclohexyl moiety enhanced the relative affinity of <b>13</b> by a factor of six for gp120CN54 and two for gp120SF162 compared to <b>1</b>, resulting in a CD4-binding-site ligand with subnanomolar apparent affinity, and about 1 order of magnitude better IC<sub>50</sub> of sCD4 on the SF162 receptor (4 nM, personal communication). SPR analyses on gp120 from three different strains from clades B and C show an increased affinity of <b>13</b> for gp120 compared to <b>2</b>. The SPR results from this study and from a related study<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> together show that the affinity of <b>13</b> for YU2 gp120 surpasses that of <b>1</b>, <b>2</b>, <b>14</b>, and sCD4. The in vitro antiviral activity of <b>13</b> improved more than 100-fold versus <b>1</b>, and between 5- to 25-fold versus sCD4,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> on the laboratory adapted and primary isolates tested.</div><div class="NLM_p last">The crystal structure of <b>13</b> bound to YU2 gp120 shows that the introduction of the aniline nitrogen results in a hydrogen bond interaction with the gp120 solvent channel. Conjugation of this nitrogen may provide a means to improve affinity even further, by utilizing interaction with a conserved gp120 solvent channel, which runs adjacent and roughly perpendicular to the Phe43 cavity. Our results further suggest that binding of the insert moiety of <b>13</b> to the Phe43 cavity is primarily enthalpy driven while at the same time torsional movements about the flexible linker bonds allow different orientations of the cyclohexane ring to be accommodated in the wider, innermost recess of the cavity, thereby preserving conformational entropy of the ligand in its bound state. It remains to be tested whether modifications of the cyclohexane ring via structure-guided substitution will result in better ring packing and higher affinity. While peptides <b>8</b> and <b>9</b> showed IC<sub>50</sub> values similar to miniCD4 <b>13</b> in the gp120SF162 (clade B) assay, their antiviral activities were not comparable to that of <b>13</b> even for clade B isolates. The differences in their pharmacological activity may result from the differences in their cavity-filling properties, described by X-ray crystallographic analyses in this study and in a related one.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> All these inserts displayed flexibility and hydrophobic interactions, but the inserts of <b>2</b><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and <b>13</b> (this study) showed better shape complementarity with the Phe43 cavity than the insert of <b>8</b><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and probably also of peptide <b>9</b>. Taken together our results show that a subtle change in the chemistry of the Phe43 cavity insert can result in improved chemical complementarity and substantially improved biological activity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55521" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55521" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The miniCD4s built on the scyllatoxin scaffold are stable to acidic pH and high temperatures, are relatively resistant toward proteolytic degradation, are poorly immunogenic, and are not cytotoxic. They stabilize the CD4-bound conformation of gp120, and one miniCD4 representative (<b>2</b>) was also shown to neutralize HIV entry in vitro and in vivo. Cost is a key consideration for the potential use of such compounds as microbicides especially in developing countries. One way to achieve low cost is to enhance the potency of the miniCD4s, thereby reducing the amount of drug that needs to be administered. In this paper, we showed that the synthesis of novel unnatural phenylalanine derivatives allowed us to design the most effective miniCD4 ever developed. It is able to block the viral activity of T/F viruses in the nanomolar range not only for clade B but also for clade C, being therefore more potent than the previously described miniprotein <b>2</b>. From the perspective of synthesis of future miniCD4 variants, <b>13</b> has a distinct advantage over <b>2</b>. Replacement of the phenol moiety by an aniline group, i.e., the presence of a secondary amine in place of ether, allows further modification in the direction of the gp120 solvent channel. Such modifications may lead to development of miniCD4s with even greater potency and therefore lower cost for potential use as a microbicide. It will also allow exploration of the gp120 Phe43 cavity, a very highly conserved but relatively poorly understood feature of the HIV-1 envelope structure.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78117" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78117" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Synthesis Procedures</h3><div class="NLM_p last">All reagents and solvents used in the synthesis and purification steps were purchased from Sigma-Aldrich, Fluka, Novabiochem, SDS, or Lancaster and were of the highest purity available. THF was distilled from sodium/benzophenone immediately prior to use. Wang resin (<i>p</i>-benzyloxybenzyl alcohol, polymer-bound resin), used for amino acid synthesis, was purchased from Aldrich (100–200 mesh, loading level 1.82 mmol OH/g). Fmoc-PAL-PEG-PS, used for solid-phase peptide synthesis, was purchased from Applied Biosystems (100–200 mesh, loading level 0.2 mmol/g). Standard Fmoc-protected amino acids were obtained from Novabiochem. Unnatural amino acid Fmoc-<span class="smallcaps smallerCapital">l</span>-4-<i>tert</i>-butyl-Phe was purchased from PepTech Corp., Fmoc-<span class="smallcaps smallerCapital">l</span>-Phe(4-<i>i</i>-Pr)-OH from Carbone Scientific, Fmoc-<span class="smallcaps smallerCapital">l</span>-2-naphthylalanine-OH from Senn Chemicals, Fmoc-<span class="smallcaps smallerCapital">l</span>-styrylalanine from PolyPeptide, Fmoc-<span class="smallcaps smallerCapital">l</span>-Phe(4-Ph)-OH and Fmoc-homoPhe-OH from Bachem, and Fmoc-<span class="smallcaps smallerCapital">l</span>-Tyr(Me)-OH and Fmoc-<span class="smallcaps smallerCapital">l</span>-Phe(4-Me)-OH from Advanced ChemTech. Flash chromatography was carried out with Merck silica gel Si 60 (40–63 μm) as the stationary phase. Melting points of the crystallized compounds were determined on a Kofler melting point apparatus and are uncorrected. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on Bruker AVANCE 250 and 400 NMR spectrometers. The molecular weights of the amino acids and peptides were determined by electrospray mass spectrometry (ES/MS), performed on a Quattro micro mass spectrometer (Micromass, Altricham, U.K.). The purity of Fmoc-amino acids, denoted in parentheses, was evaluated by analytical-HPLC, at 214 nm, on a thermo HPLC apparatus (<b>32</b>′–<b>40</b>′, <b>56</b>′, <b>57</b>′, and <b>59</b>′) or a Waters 600E HPLC (<b>54</b>′) with a Chromolith SpeedROD RP-18e column (VWR, 50 × 4.6 mm, 2 μm) with a linear gradient from 10% to 90% solvent B over 6 min (0.1% TFA in water (solvent A), 90% CH<sub>3</sub>CN with 0.09% TFA in water (solvent B)) at 3 mL/min constant flow rate (these conditions will be noted HPLC-A). The purity of the various tested peptides (see Tables S2 and S3 in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>) was checked on a Waters 600E HPLC, at 214 nm, by reverse phase chromatography at a constant 1 mL/min flow rate, on an analytical Discovery BIO Wide Pore C18, 5 μM (15 cm × 4.6 mm) column with a gradient going from 10% to 50% B in 30 min. Even if we could not always reach a purity ≥95% for the newly synthesized Fmoc-amino acids, all the tested compounds, i.e., the various miniCD4s, have a purity >95%. Retention times (<i>t</i><sub>R</sub>) are reported in minutes.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i>,<i>O</i>-Bis-allyloxycarbonyl-<span class="smallcaps smallerCapital">l</span>-tyrosine (<b>2</b>′)</h3><div class="NLM_p last">Tyrosine (<b>1</b>′) (5.44 g, 30 mmol) was dissolved in 15 mL of 4 N sodium hydroxide (2 equiv). The solution was cooled in an ice-bath and treated with 7 mL (2.2 equiv) of allyl chloroformate and an additional 15 mL of 4 N sodium hydroxide, each added in five equal portions, with vigorous shaking for a few minutes after each addition. After the last addition, the mixture was shaken vigorously for 1.5 h, letting it warming up to rt. The water layer was extracted twice with ether and then acidified to pH 4–5 with concentrated hydrochloric acid. After being cooling overnight, the crystals were filtered, washed with cold water, and dissolved in DCM. The organic layer was washed with H<sub>2</sub>O and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated to yield 9.35 g (89%) of <b>2</b>′ as a white solid: <i>R</i><sub>f</sub> (DCM/MeOH/AcOH: 90/8/2) = 0.75; mp 106–107 °C (105–106 °C);<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a><sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 3.14 (dt, <i>J</i> = 13.9, 5.2 Hz, 2H), 4.56 (d, <i>J</i> = 5.4 Hz, 2H), 4.73 (d, <i>J</i> = 5.8 Hz, 2H), 4.72–4.75 (m, 1H), 5.21 (d, <i>J</i> = 11.1 Hz, 1H), 5.34 (d, <i>J</i> = 10.4 Hz, 1H), 5.35 (d, <i>J</i> = 18.8 Hz, 1H), 5.43 (d, <i>J</i> = 18.6 Hz, 1H), 5.92–6.26 (m, 2H), 6.36 (br s, 1H), 7.12 (d, <i>J</i> = 8.6 Hz, 2H), 7.20 (d, <i>J</i> = 8.6 Hz, 2H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 37.1, 55.1, 66, 69.2, 118, 119.6, 121.1, 130.4, 131, 132.4, 134.1, 150, 153.6, 156, 175.6.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>N</i>,<i>O</i>-Bis-allyloxycarbonyl-<span class="smallcaps smallerCapital">l</span>-tyrosine Immobilized on Wang Resin (<b>3</b>′)</h3><div class="NLM_p">Wang resin (8.5 g; 1.82 mmol OH/g) was swelled in NMP (50 mL) for 15 min in a reactor with external shaking. The resin was filtered and resuspended in NMP (50 mL). A 7.27 mL amount of diisopropylcarbodiimide (3 equiv) and 189 mg of DMAP (0.1 equiv) were added, and the mixture was stirred for 15 min. Compound <b>2</b>′ (16.2 g, 3 equiv) in solution in NMP (50 mL) was added, and shaking was continued for 48 h at rt. The resin was filtered, rinsed with DCM (3 × 100 mL), and finally dried in vacuo to afford 12.95 g of a 0.99 mmol/g (87% theoretical) resin-bound <i>N</i>,<i>O</i>-bis-allyloxycarbonyl-<span class="smallcaps smallerCapital">l</span>-tyrosine <b>3</b>′.</div><div class="NLM_p last">Estimated resin loading by cleavage: A small weighed sample of resin (between 50 and 100 mg) was cleaved by 5 mL of TFA/DCM: 3/7 during 1 h. The mixture was filtered, the resin rinsed with DCM (2 × 3 mL), and the filtrate evaporated. The oil was mixed with a few milliliters of water, frozen, and lyophilized. The obtained crude compound was weighed and analyzed by <sup>1</sup>H NMR. Loading was estimated by comparison with the theoretical obtained quantity.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>N</i>-Allyloxycarbonyl-<span class="smallcaps smallerCapital">l</span>-tyrosine Immobilized on Wang Resin (<b>4</b>′)</h3><div class="NLM_p last">Twelve grams (11.92 mmol) of bis-protected tyrosine resin <b>3</b>′ (0.99 mmol/g) was treated with a solution (150 mL) of 20% piperidine in DMF (v/v) for 24 h at rt. The resin was filtered, rinsed with DCM (3 × 100 mL), and finally dried in vacuo to afford 11.05 g of a 1.03 mmol/g (95% theoretical) resin-bound <i>N</i>-allyloxycarbonyl-<span class="smallcaps smallerCapital">l</span>-tyrosine <b>4</b>′.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>N</i>-Allyloxycarbonyl-<i>O</i>-(c-alkyl or c-aryl-alkyl chain)-<span class="smallcaps smallerCapital">l</span>-tyrosine Immobilized on Wang Resin (<b>5</b>′ to <b>13</b>′)</h3><div class="NLM_p">General procedure G1: PPh<sub>3</sub> and the corresponding alcohol (4–8 equiv depending on the nature of the alcohol; see below) were added to resin <b>4</b>′ (4.5 mmol, 4.37 g), suspended in an anhydrous 22 mL 1:1 (v/v) DCM/THF mixture. After 15 min, DIAD (4–8 equiv depending on the nature of the alcohol, see below), in solution in a few milliliters of DCM, was added portionwise, with shaking for a few minutes after each addition. The reaction continued for 24 h, at rt. The resin was filtered and rinsed with DCM (3 × 100 mL). This procedure was repeated a second time. After 24 h shaking, the resin was filtered and rinsed with DMF (2 × 100 mL), DCM (1 × 100 mL), MeOH (1 × 100 mL), and DCM (2 × 100 mL). A small weighed sample of resin was cleaved under the same conditions as described above. Crude compound was used for <sup>1</sup>H NMR analysis to check the disappearance of the phenol moiety. Loading was not estimated.</div><div class="NLM_p last"><i>N</i>-Allyloxycarbonyl-<i>O</i>-(cyclohexylmethyl)-<span class="smallcaps smallerCapital">l</span>-tyrosine immobilized on Wang resin (<b>5</b>′) was synthesized according to the procedure G1 using 3.9 mL of cyclohexylmethanol (7 equiv, 31.5 mmol) with 7 equiv of PPh<sub>3</sub> and DIAD. For <b>6</b>′ to <b>13</b>′, see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>O</i>-(c-Alkyl or c-aryl-alkyl chain)-<span class="smallcaps smallerCapital">l</span>-tyrosine Immobilized on Wang Resin (<b>14</b>′ to <b>22</b>′)</h3><div class="NLM_p last">General procedure G2: Under argon atmosphere, 2.6 g of tetrakis(triphenylphosphine)palladium (0.5 equiv, 2.25 mmol) was added to the resin <b>5</b>′–<b>13</b>′ (loadings were estimated at 4.5 mmol) in suspension in 150 mL of THF/DMSO/0.5 M HCl/morpholine: 20/20/10/1 ratio. The reactor was protected against light, as Pd(Ph<sub>3</sub>)<sub>4</sub> is light sensitive, and stirred for 24 h. The resin was filtered and rinsed with DMF (2 × 100 mL), a 0.1 M solution of diethyldithiocarbamic acid in DMF (5 × 100 mL), DMF (2 × 100 mL), and DCM (3 × 100 mL). A 100% yield was estimated for whichever of the resins (<b>5</b>′–<b>13</b>′) was used. Washings with 0.1 M solution of diethyldithiocarbamic acid in DMF were essential to remove all traces of catalyst from the resin. <i>O</i>-(Cyclohexylmethyl)-<span class="smallcaps smallerCapital">l</span>-tyrosine immobilized on Wang resin (<b>14</b>′) was synthesized according to procedure G2 using resin <b>5</b>′. For <b>15</b>′ to <b>22</b>′, see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>-(9-Fluorenylmethoxycarbonyl)-<i>O</i>-(c-alkyl or c-aryl-alkyl chain)-<span class="smallcaps smallerCapital">l</span>-tyrosine Immobilized on Wang Resin (<b>23</b>′ to <b>31</b>′)</h3><div class="NLM_p last">General procedure G3: To the resin <b>14</b>′-<b>22</b>′ (loadings not calculated, estimated at 4.5 mmol), in suspension in DCM (90 mL), was added 3.14 mL of diisopropylethylamine (4 equiv, 18 mmol). The mixture was stirred for 15 min. A 3.5 g amount of Fmoc-Cl (3 equiv, 13.5 mmol), in solution in 30 mL of DCM, was added to this suspension and the reaction continued for 24 h at rt. The resin was filtered, rinsed with DCM (3 × 100 mL), and dried in vacuo. <i>N</i>-(9-Fluorenylmethoxycarbonyl)-<i>O</i>-(cyclohexylmethyl)-<span class="smallcaps smallerCapital">l</span>-tyrosine immobilized on Wang resin (<b>23</b>′) was synthesized according to procedure G3 using resin <b>14</b>′. For <b>24</b>′ to <b>31</b>′, see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>-(9-Fluorenylmethoxycarbonyl)-<i>O</i>-(c-alkyl or c-aryl-alkyl chain)-<span class="smallcaps smallerCapital">l</span>-tyrosine (<b>32</b>′ to <b>40</b>′)</h3><div class="NLM_p last">General procedure G4 for resin cleavage: The resin (<b>23</b>′–<b>31</b>′) was treated with 100 mL of TFA/DCM: 3/7 for 1 h at rt. The mixture was filtered, and the obtained organic layer was washed with H<sub>2</sub>O (2 × 50 mL), dried over MgSO<sub>4</sub>, and evaporated. The crude residue was applied to silica gel flash chromatography with DCM and DCM/MeOH: 97/3 (flash chromatography condition 1: FC1) or DCM and DCM/AcOH: 99/1 (flash chromatography condition 2: FC2) as eluents to give compound <b>32</b>′ to <b>40</b>′ with yields between 26% and 63%.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>N</i>-(9-Fluorenylmethoxycarbonyl)-<i>O</i>-(cyclohexylmethyl)-<span class="smallcaps smallerCapital">l</span>-tyrosine (<b>32</b>′)</h3><div class="NLM_p last">Procedure G4 was applied to <b>23</b>′, and the crude residue was purified according to FC2 to yield 1.32 g of <b>32</b>′ (59%) as a white solid: <i>R</i><sub>f</sub> (DCM/AcOH: 99/1) = 0.47; mp 136–137 °C; <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 0.96–1.10 and 1.14–1.37 (2m, 6H), 1.63–1.91 (m, 5H), 3.11 (ddt, <i>J</i> = 14.0, 16.6, 5.8 Hz, 2H), 3.70 (d, <i>J</i> = 6.2 Hz, 2H), 4.21 (t, <i>J</i> = 7.1 Hz, 1H), 4.37 (dd, <i>J</i> = 9.8, 6.6 Hz, 1H), 4.45 (dd, <i>J</i> = 10.5, 7.1 Hz, 1H), 4.67 (dt, <i>J</i> = 6.0, 5.9 Hz, 1H), 5.17 (d, <i>J</i> = 8.1 Hz, 1H), 6.81 (d, <i>J</i> = 8.1 Hz, 2H), 7.04 (d, <i>J</i> = 8.1 Hz, 2H), 7.31 (t, <i>J</i> = 7.4 Hz, 2H), 7.41 (t, <i>J</i> = 7.4 Hz, 2H), 7.55 (t, <i>J</i> = 6.2 Hz, 2H), 7.76 (d, <i>J</i> = 7.3 Hz, 2H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 25.79, 26.50, 30.30, 36.38, 37.66, 47.10, 54.57, 66.99, 73.43, 114.63, 119.98, 125.03, 127.05, 127.72, 130.32, 141.26, 143.76, 155.72, 158.51, 175.78; ES/MS for C<sub>31</sub>H<sub>33</sub>NO<sub>5</sub> (negative ionization): Molecular wt calcd: 499.2, found: 499.1; <i>t</i><sub>R</sub> (HPLC-A): 5.11 (94%). For compounds <b>33</b>′ to <b>40</b>′, see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 4-Benzyloxy-1-butanol (<b>44</b>′)</h3><div class="NLM_p last">To sodium hydride (4.45 g, 111.2 mmol, 60% in mineral oil) in suspension in dry THF (200 mL), under argon, cooled to 0 °C, was slowly added 50 g of 1,4-butanediol (<b>41</b>′) (5 equiv, 555 mmol). The reaction was stirred for 1 h. Benzyl bromide (13 mL, 111.2 mmol), in 200 mL of dry THF, was added dropwise, and the reaction mixture was stirred an additional 5 h. After addition of 300 mL of H<sub>2</sub>O, the solution was extracted twice with Et<sub>2</sub>O. The organic layer was washed with water and brine and dried over MgSO<sub>4</sub>, and the solvent was evaporated under reduced pressure. After purification by column chromatography on silica gel (eluted with DCM and DCM/MeOH: 97/3), 13.2 g (66%) of <b>44</b>′ was obtained as a slightly yellow oil: <i>R</i><sub>f</sub> (DCM/MeOH: 97/3) = 0.39; <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 1.57–1.71 (m, 4H), 3.48 (t, <i>J</i> = 6.1 Hz, 2H), 3.53 (s, 1H), 3.56 (t, <i>J</i> = 6.3 Hz, 2H), 4.49 (s, 2H), 7.24–7.30 (m, 1H), 7.31–7.36 (m, 4H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 26.33, 29.58, 62.10, 70.24, 72.86, 127.67, 128.35, 138.19. For compounds <b>45</b>′ and <b>46</b>′, see <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>N</i>-Cbz-(<i>S</i>)-(+)-2-Pyrrolidinemethanol (<b>48</b>′)</h3><div class="NLM_p last">To 9 g of (−)-Cbz-<span class="smallcaps smallerCapital">l</span>-proline (<b>47</b>′) (36.1 mmol) in DME (55 mL) was added 4-methylmorpholine (1.1 equiv, 39.7 mmol). The reaction mixture was cooled to −15 °C, and isobutyl chloroformate (1.3 equiv, 46.9 mmol) was added slowly. After 20 min of stirring at this temperature, the white precipitate was quickly filtered off and washed with DME. To the obtained solution, cooled again to −15 °C, was added sodium borohydride (1.5 equiv, 54.2 mmol), in 23 mL of water, and the reaction mixture stirred during 1 h, allowing it to warm to rt. After addition of 65 mL of water and evaporation of DME under reduced pressure, the solution was extracted twice with AcOEt. The organic layer was washed with 1 N aqueous KHSO<sub>4</sub> solution, 1 N aqueous NaHCO<sub>3</sub> solution, and finally brine. It was dried over MgSO<sub>4</sub>, and the solvent was evaporated under reduced pressure. A 7.4 g (87%) amount of <b>48</b>′ was obtained as a colorless thick oil: <i>R</i><sub>f</sub> (DCM/MeOH: 95/5) = 0.55; <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 1.52–1.93 (m, 4H), 3.32 (t, <i>J</i> = 6.8 Hz, 1H), 3.42 (t, <i>J</i> = 6.8 Hz, 1H), 3.55 (d, <i>J</i> = 5.4 Hz, 2H), 3.86–3.96 (m, 1H), 5.05 (s, 2H), 7.20–7.29 (m, 5H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 24.53, 29.01, 47.82, 61.10, 67.00, 67.72, 128.43, 128.97, 129.04, 137.06, 157.48.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 2-((2-((<i>tert</i>-Butyldimethylsilyl)oxy)ethyl)thio)ethanol (<b>50</b>′)</h3><div class="NLM_p last">A 5.8 mL (57.6 mmol) amount of 2,2′-thiodiethanol (<b>49</b>′) was dissolved in dry THF containing 3.2 g (63.4 mmol) of NaH (60% in oil) and stirred for 1.5 h under inert atmosphere. The solution was cooled to 0 °C, and 8.6 g (57.6 mmol) of <i>tert</i>-butyldimethylsilyl chloride was added slowly. The mixture was stirred overnight at rt. A 10% K<sub>2</sub>CO<sub>3</sub> solution was then added, THF was evaporated, and diethyl ether was added and extracted with water and brine. The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under reduced pressure. Purification by column chromatography on silica gel (eluted with hexane/AcOEt: 8/2) afforded 8.1 g of <b>50</b>′, as an almost colorless oil (59%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 0.86 (s, 6H), 0.91 (s, 9H), 2.68 (t, <i>J</i> = 6.5 Hz, 2H), 2.76 (t, <i>J</i> = 6.0 Hz, 2H), 2.88 (br s, 1H), 3.74 (t, <i>J</i> = 6.0 Hz, 2H), 3.79 (t, <i>J</i> = 6.75 Hz, 2H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ −5.3, 18.3, 25.9, 34.2, 35.8, 60.9, 63.4.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Benzyl 2-(((Benzyloxy)carbonyl)amino)-3-(4-(2-((2-((<i>tert</i>-butyldimethylsilyl)oxy)ethyl)thio)ethoxy)phenyl)propanoate (<b>51</b>′)</h3><div class="NLM_p last">A 4.1 g (17.4 mmol) amount of <b>50</b>′ was dissolved in toluene and cooled to 0 °C. A 8.8 g (1.2 equiv) amount of Z-Tyr-OBn dissolved in 10 mL of dry DMF was added together with 5.7 g (1.2 equiv) of PPh<sub>3</sub>. After dissolution, DEAD (1.2 equiv) was added dropwise. The reaction was then stirred overnight at rt. Sodium phosphate buffer (0.5 M, pH 7) was added to the mixture, which was then extracted with AcOEt. The organic phase was washed with a saturated solution of NH<sub>4</sub>Cl, water, and brine and dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under reduced pressure. Purification by column chromatography on silica gel (eluted with hexane/AcOEt: 9/1 to 85/15) afforded 10.45 g of <b>51</b>′ as a colorless oil (96%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 0.08 (s, 6H), 0.9 (s, 9H), 2.74 (t, <i>J</i> = 6.75 Hz, 2H), 2.92 (t, <i>J</i> = 6.75 Hz, 2H), 3.03 (d, <i>J</i> = 5.75 Hz, 2H), 3.81 (t, <i>J</i> = 7.0 Hz, 2H), 4.06 (t, <i>J</i> = 7.0 Hz, 2H), 4.65 (m, 1H), 5.11 (m, 4H), 5.27 (d, <i>J</i> = 8.25 Hz, 1H), 6.71 (d, <i>J</i> = 8.5 Hz, 2H), 6.89 (d, <i>J</i> = 8.5 Hz, 2H), 7.30 (m, 10H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ −5.2, 18.3, 25.9, 31.4, 34.9, 37.2, 54.9, 63.4, 66.9, 67.2, 67.7, 114.6, 127.7, 128.1, 128.2, 128.4, 128.5, 130.3, 135.1, 136.2, 155.6, 157.6, 171.4.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 2-Amino-3-(4-(2-((2-((<i>tert</i>-butyldimethylsilyl)oxy)ethyl)thio)ethoxy)phenyl)propanoic Acid (<b>52</b>′)</h3><div class="NLM_p last">A 10.45 g (16.8 mmol) amount of <b>51</b>′ was dissolved in 20 mL of MeOH and reduced overnight with palladium black under 3.4 bar of H<sub>2</sub>. The slurry was filtered. The catalyst was washed five times with MeOH, and the filtrate was evaporated, yielding 6.62 g of compound <b>52</b>′ (quantitative). ES/MS for C<sub>19</sub>H<sub>33</sub>NO<sub>2</sub>SSi (positive ionization): Molecular wt calcd: 399.1, found: 399.1</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(4-(2-((2-((<i>tert</i>-butyldimethylsilyl)oxy)ethyl)thio)ethoxy)phenyl)propanoic Acid (<b>53</b>′)</h3><div class="NLM_p last">Compound <b>52</b>′, dissolved in 50 mL of DMF, was added to 5.9 g (17.5 mmol) of Fmoc <i>N</i>-hydroxysuccinimide ester in a mixture of 750 mL of CH<sub>3</sub>CN, 370 mL of H<sub>2</sub>O, and 5.3 g of NaHCO<sub>3</sub> and stirred overnight at rt. The solution was concentrated and extracted with AcOEt. The organic layer was washed with 10% citric acid, a saturated solution of NH<sub>4</sub>Cl, water, and brine and dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under reduced pressure. Purification by column chromatography on silica gel (eluted with DCM/AcOH: 99.5/0.5 to DCM/MeOH/AcOH: 95/5/0.5) afforded 8 g of <b>53</b>′ as a colorless oil (78%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 0.09 (s, 6H), 0.9 (s, 9H), 2.74 (t, <i>J</i> = 6.75 Hz, 2H), 2.92 (t, <i>J</i> = 6.75 Hz, 2H), 3.09 (m, 2H), 3.81 (t, <i>J</i> = 6.75 Hz, 2H), 4.08 (t, <i>J</i> = 6.75 Hz, 2H), 4.19 (t, <i>J</i> = 6.85 Hz, 1H), 4.41 (m, 2H), 4.65 (m, 1H), 5.28 (t, <i>J</i> = 10.5 Hz, 1H), 6.80 (d, <i>J</i> = 8.5 Hz, 2H), 7.05 (d, <i>J</i> = 8.5 Hz, 2H), 7.16–7.43 (m, 4H), 7.53–7.58 (m, 2H), 7.76 (d, <i>J</i> = 7.5 Hz, 2H), 8.94 (br s, 1H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ −5.2, 18.4, 26.0, 31.5, 34.9, 36.9, 47.2, 54.8, 63.5, 67.1, 67.8, 114.8, 120.1, 125.2, 127.2, 127.8, 128.2, 130.5, 141.4, 143.8, 155.9, 157.8, 176.2. ES/MS for C<sub>34</sub>H<sub>43</sub>NO<sub>6</sub>SSi (positive ionization): Molecular wt calcd: 622.2, found: 622.2</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(4-(2-((2-((<i>tert</i>-butyldimethylsilyl)oxy)ethylsulfinyl)ethoxy)phenyl)propanoic Acid (<b>54</b>′)</h3><div class="NLM_p last">A 1.24 g (2 mmol) amount of <b>53</b>′ was dissolved in 6 mL of dry DCM and treated with 534 mL (1.5 equiv) of <i>tert</i>-butyl hydroperoxide (5.5 M in decane) and 10 mg (6% in mol) of thiourea dioxide for 3 days. The mixture was diluted with DCM, washed with water and brine, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Purification by column chromatography on silica gel (eluted with DCM/MeOH/AcOH: 95/5/0.1) afforded 1.89 g of <b>54</b>′ as a white foam (74%): mp not measurable, behaves like a wax; <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 0.1 (s, 6H), 0.9 (s, 9H), 3.06–3.24 (m, 6H), 4.00–4.62 (m, 8H), 5.55 (d, <i>J</i> = 7.75 Hz, 1H), 6.78 (d, <i>J</i> = 8.25 Hz, 2H), 7.05 (d, <i>J</i> = 8.25 Hz, 2H), 7.25–7.41 (m, 4H), 7.54–7.59 (m, 2H), 7.75 (d, <i>J</i> = 7.5 Hz, 2H), 11.19 (br s, 1H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ −5.5, 18.2, 25.8, 36.9, 47.1, 51.43, 54.7, 55.2, 56.1, 60.3, 66.8, 114.5, 119.9, 125.1, 127.0, 127.7, 128.0, 130.6, 141.2, 143.8, 155.7, 156.9, 173.7. ES/MS for C<sub>34</sub>H<sub>43</sub>NO<sub>7</sub>SSi (positive ionization): Molecular wt calcd: 637.3, found: 637.3; <i>t</i><sub>R</sub> (HPLC-A on a Waters 600E HPLC): 5.23 (94%).</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i>-(9-Fluorenylmethoxycarbonyl)-<i>p</i>-(<i>N</i>-cyclohexylmethylamino)-<span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>56</b>′)</h3><div class="NLM_p last">Cyclohexanecarboxaldehyde (900 mL, 1.5 equiv, 7.45 mmol) was added to a suspension of Fmoc-<i>p</i>-amino-<span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>55</b>′) (2 g, 4.98 mmol) in 50 mL of CH<sub>3</sub>CN/AcOH (1% v/v). The reaction mixture was stirred for 2 h at rt. Solid NaBH(OAc)<sub>3</sub> (2 equiv, 9.94 mmol) was then added portionwise, and stirring was continued overnight at rt. The mixture was poured into 250 mL of a 0.1 N aqueous HCl solution, and the solution was extracted twice with AcOEt. The organic layer was washed twice with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under reduced pressure. After purification by column chromatography on silica gel (eluted with DCM/AcOH: 100/0.5 to DCM/MeOH/AcOH: 99/1/0.5), 1.9 g (77%) of <b>56</b>′ were obtained as a white solid: <i>R</i><sub>f</sub> (DCM/MeOH/AcOH: 99/1/0.5) = 0.18; mp 95 °C; <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 0.86–0.98 (m, 2H), 1.02–1.34 (m, 5H), 1.42–1.83 (m, 6H), 2.88 (d, <i>J</i> = 6.6 Hz, 2H), 3.07 (d, <i>J</i> = 5.4 Hz, 2H), 4.21 (t, <i>J</i> = 6.9 Hz, 1H), 4.35 (dd, <i>J</i> = 10.3, 6.8 Hz, 1H), 4.43 (dd, <i>J</i> = 10.3, 7.3 Hz, 1H), 4.63 (dt, <i>J</i> = 7.9, 5.3 Hz, 1H), 5.28 (d, <i>J</i> = 8.4 Hz, 1H), 6.60 (d, <i>J</i> = 8.4 Hz, 2H), 6.97 (d, <i>J</i> = 8.3 Hz, 2H), 7.31 (t, <i>J</i> = 7.4 Hz, 2H), 7.40 (t, <i>J</i> = 7.2 Hz, 2H), 7.57 (d, <i>J</i> = 7.1 Hz, 2H), 7.76 (d, <i>J</i> = 7.4 Hz, 2H); <sup>13</sup>C NMR (acetone-<i>d</i><sub>6</sub>) δ 26.4, 27.0, 37.0, 37.8, 47.6, 50.7, 56.8, 66.6, 112.5, 120.5, 125.1, 126.1, 127.7, 128.2, 130.4, 141.6, 144.7, 148.7, 156.6, 174.3; ES/MS for C<sub>31</sub>H<sub>34</sub>N<sub>2</sub>O<sub>4</sub> (positive ionization): Molecular wt calcd: 498.3, found: 498.3; <i>t</i><sub>R</sub> (HPLC-A): 3.25 (98.5%).</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-(9-Fluorenylmethoxycarbonyl)-<i>p</i>-(<i>N</i>-cyclohexylmethyl-<i>N</i>′-<i>tert</i>-butyloxycarbonylamino)-<span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>57</b>′)</h3><div class="NLM_p last">To 1.07 g (2.15 mmol) of <b>56</b>′, dissolved in 45 mL of THF, were added 750 mL of diisopropylethylamine (2 equiv) and 1.64 g (3.5 equiv) of di-<i>tert</i>-butyl dicarbonate. The reaction mixture was stirred at 45–50 °C for 3 days. The mixture was concentrated in vacuo. After addition of 100 mL of AcOEt, the organic layer was washed with 10% citric acid and brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure. After purification by column chromatography on silica gel (eluted with DCM/MeOH/AcOH: 97/3/1.5), 1.22 g (94.8%) of <b>57</b>′ was obtained as a pale yellow foam: <i>R</i><sub>f</sub> (DCM/MeOH/AcOH: 97/3/1.5) = 0.45; mp not measurable, behaves like a wax; <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 0.85–0.90 (m, 2H), 1.04–1.14 (m, 3H), 1.45 (s, 9H), 1.53–1.70 (m, 6H), 3.15 (d, <i>J</i> = 4.5 Hz, 2H), 3.44 (d, <i>J</i> = 7.2 Hz, 2H), 4.19 (t, <i>J</i> = 7.0 Hz, 1H), 4.27–4.44 (m, 3H), 4.71 (m, 1H), 5.64 (br s, 1H), 6.98–7.19 (m, 4H), 7.29 (dt, <i>J</i> = 7.2, 1.2 Hz, 2H), 7.39 (t, <i>J</i> = 7.5 Hz, 2H), 7.57 (t, <i>J</i> = 6.5 Hz, 2H), 7.75 (d, <i>J</i> = 7.2 Hz, 2H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 25.8, 26.4, 28.3, 30.6, 36.7, 37.1, 47.1, 54. 3, 56.1, 67.1, 80.36, 119.9, 125.2, 127.1, 127.4, 127.7, 130.0, 133.8, 141.3, 143.7, 155.3, 173.6; ES/MS for C<sub>36</sub>H<sub>42</sub>N<sub>2</sub>O<sub>6</sub> (positive ionization): Molecular wt calcd: 598.3, found: 598.4; <i>t</i><sub>R</sub> (HPLC-A): 5.29 (96%).</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <i>N</i>-(9-Fluorenylmethoxycarbonyl)-<i>p</i>-(isopropyl)-<span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>59</b>′)</h3><div class="NLM_p last">A 350 mg (1.69 mmol) amount of <i>p</i>-isopropyl-<span class="smallcaps smallerCapital">l</span>-phenylalanine (<b>58</b>′) was dissolved in 6.8 mL (4 equiv, 6.76 mmol) of 1 N Na<sub>2</sub>CO<sub>3</sub> aqueous solution and 5 mL of dioxane. Half of the solvent volume was removed under reduced pressure. Then the residual solvent was diluted with 7.5 mL of water (pH was controlled at 9–10) and 5 mL of dioxane and cooled with an ice bath. A 547 mg (1.25 equiv, 2.11 mmol) amount of Fmoc-chloroformate, dissolved in 5 mL of dioxane, was added slowly. The reaction mixture was stirred 2 h at 0 °C and overnight at rt (additional 2 mL of 1 N Na<sub>2</sub>CO<sub>3</sub> aqueous solution was added to keep pH around 9). After addition of 40 mL of H<sub>2</sub>O, the solution was acidified with a 2 N HCl aqueous solution until the product precipitated. The aqueous layer was quickly extracted twice with Et<sub>2</sub>O. The organic layer was washed twice with H<sub>2</sub>O to eliminate all traces of HCl and dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under reduced pressure. The crude residue was applied to silica gel flash chromatography (eluted with DCM and DCM/MeOH: 97/3) to yield 0.34 g of <b>59</b>′ (46%) as a white powder: <i>R</i><sub>f</sub> (DCM/MeOH: 95/5) = 0.33; mp 113 °C; <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 1.21 and 1.24 (2s, 6H), 2.88 (sept, <i>J</i> = 6.9 Hz, 1H), 3.15 (ddt, <i>J</i> = 14.5, 10.6, 5.1 Hz, 2H), 4.21 (t, <i>J</i> = 6.8 Hz, 1H), 4.36 (dd, <i>J</i> = 10.3, 7.2 Hz, 1H), 4.45 (dd, <i>J</i> = 10.6, 7.1 Hz, 1H), 4.67–4.72 (m, 1H), 5.21 (d, <i>J</i> = 8.1 Hz, 1H), 7.07 (d, <i>J</i> = 8.1 Hz, 2H), 7.16 (d, <i>J</i> = 7.4 Hz, 2H), 7.30 (t, <i>J</i> = 7.4 Hz, 2H), 7.41 (t, <i>J</i> = 7.4 Hz, 2H), 7.54–7.55 (m, 2H), 7.77 (d, <i>J</i> = 7.4 Hz, 2H); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 23.92, 23.95, 33.70, 37.22, 47.11, 54.50, 67.08, 119.98, 119.99, 126.76, 127.05, 127.73, 129.28, 132.63, 141.29, 143.67, 155.77, 175.58; ES/MS for C<sub>27</sub>H<sub>27</sub>NO<sub>4</sub> (negative ionization): Molecular wt calcd: 429.1, found: 429.1; <i>t</i><sub>R</sub> (HPLC-A): 4.25 (100%).</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Peptides Synthesis, Refolding, and Purification</h3><div class="NLM_p last">The sequences of the various synthesized peptides were TpaNLHFCQLRCKSLGLLGKCAGSXCACV-NH<sub>2</sub> for the M33 (<b>16</b>) series and TpaNLHFCQLRCKSLGLLGRCA-DPro-TXCACV-NH<sub>2</sub> for the M48 (<b>1</b>) series. Tpa stands for thiopropionyl and X for phenylalanine or a phenylalanine derivative. Peptides were synthesized on an ABI-433A (Applied Biosystems) automated peptide synthesizer, using the stepwise solid-phase method and standard Fmoc chemistry. Synthesis was performed on a 0.1 mmol scale with 10 equiv of Fmoc-protected amino acids, 20% piperidine in NMP for Fmoc-deprotection, DCC and Cl-HOBt for activation, and acetic anhydride for capping. For the M48 (<b>1</b>) series, an N-terminal thiopropionyl group (Tpa) was introduced in its disulfide form. The miniprotein was cleaved from the resin with simultaneous removal of side-chain protecting groups by treatment with reagent K′ [TFA/H<sub>2</sub>O/phenol/thioanisole/1,2-ethanedithiol/triisopropylsilane: 81.5/5/5/5/2.5/1 (all v/v)] or a mixture of TFA/H<sub>2</sub>O/triisopropylsilane: 9.5/0.25/0.25, for 2.5 h at rt. The resin was then filtered off and the fully deprotected peptide was precipitated in methyl <i>tert</i>-butyl ether at 4 °C. After centrifugation and washing with methyl <i>tert</i>-butyl ether, peptides were dissolved in acetic acid (20% v/v) and freeze-dried. Refolding was performed on the crude reduced peptide dissolved in 0.1 mg/mL in Tris/HCl buffer (0.1 M, pH 8.0), containing 2 M guanidinium chloride, 5.0 mM glutathione, and 0.5 mM oxidized glutathione. Glutathione was added 15 min before oxidized glutathione to deprotect the Tpa moiety, when present, before refolding. The refolded peptides were purified by reverse phase chromatography on a Discovery BIO Wide Pore C18, 5 μM (25 cm × 10 mm), using H<sub>2</sub>O/5%CH<sub>3</sub>CN/0.1% TFA as solvent A and CH<sub>3</sub>CN/10%H<sub>2</sub>O/0.09%TFA as solvent B. A classical gradient was 10% to 50% B over 90 min. Peptides were lyophilized and dissolved in water. Their concentration was determined by total hydrolysis under highly acidic conditions. Molecular characteristics and purity of the various miniCD4s are described in the <a class="ref internalNav" href="#notes-2" aria-label="Supporting Information">Supporting Information</a> (Tables S2 and S3).</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Competitive Binding Assays</h3><div class="NLM_p last">Competition binding assays in ELISA were performed in 96-well plates (Maxisorb; Nunc). Competition assay on gp120SF162: 50 ng/well antibody D7324 (Aalto Bio Reagents, Dublin, Ireland) was coated overnight at 4 °C. Wells were then saturated with phosphate-buffered saline containing 3% bovine serum albumin buffer and washed three times, and 15 ng/well gp120SF162 was added, followed by addition of 1.25 ng of sCD4 (Progenics) and different concentrations of soluble competitors. After one night at 4 °C, we successively added anti-CD4 monoclonal antibody L120.3 (Centralized Facility for AIDS Reagents, NIBSC, UK), goat antimouse peroxidase-conjugated antibody (Jackson ImmunoResearch, West Grove, PA), and the 3,3′,5,5′-tetramethylbenzidine substrate (Sigma). After acidification, optical density was measured at 450 nm and expressed as the mean of experiments performed in duplicate. The conditions used for the competition assay on gp120CN54 were the same as for gp120SF162, except that (i) D7324 was replaced by 500 ng of concanavalin A (Sigma-Aldrich), (ii) 5 ng/well of gp120CN54 was added, and (iii) 1 ng of sCD4 was used.</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Antiviral Activity Assays</h3><div class="NLM_p last">The antiviral activity of peptides <b>2</b>–<b>4</b>, <b>8</b>, <b>9</b>, and <b>13</b> was determined by preincubating 10<sup>4</sup> TZM-bl cells/well in a 96-well plate for 30 min at 37 °C and 5% CO<sub>2</sub> with or without a serial dilution of compound. Next, 200 TCID<sub>50</sub> of Bal (subtype B, CCR5), IIIB (subtype B, CXCR4), VI829 (subtype C, CCR5), VI1358 (subtype C, CCR5), pREJO.c/2864 (transmitted/founder, subtype B, CCR5), or p246F10 (transmitted/founder, subtype C, CCR5) viruses were added to each well, and cultures were incubated for 48 h before luciferase activity was quantified. Each compound was tested in triplicate in a single experiment. Antiviral activity was expressed as the percentage of viral inhibition compared to the untreated control and plotted against the compound concentration. Next, nonlinear regression analysis was used to calculate the 50% effective concentration (EC<sub>50</sub>).</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Cellular Toxicity Assays</h3><div class="NLM_p last">Cytotoxicity was determined using the water-soluble tetrazolium-1 (WST-1) cell proliferation assay, which is based on the cleavage of the tetrazolium salt WST-1 to a formazan dye by cellular dehydrogenases. Because this bioreduction is dependent on the glycolytic production of NAD(P)H in viable cells, the amount of formazan dye formed is correlated directly to the number of viable cells in a culture. Quantification is performed by measuring absorbance at 450 nm in a multiwell plate reader. An amount of 10<sup>4</sup> cells was plated per well in a 96-well plate, and a serial dilution of compound was added. Forty eight hours later, cell proliferation reagent was added and cell viability was measured compared to untreated control cultures. Cell viability was plotted against the compound concentration, and nonlinear regression analysis was performed to evaluate the 50% cytotoxic concentration (CC<sub>50</sub>).</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> SPR Assays</h3><div class="NLM_p last">Experiments were carried out on a Biacore 3000 or Biacore T200 instrument (GE Healthcare). gp120 was covalently coupled to a CM5 chip at 1000–2000 RU, and a blank surface with no antigen was created under identical coupling conditions for use as a reference. MiniCD4s <b>2</b> and <b>13</b> were serially diluted 2-fold, into 10 mM HEPES pH 7.4, 150 mM NaCl, 3 mM EDTA, and 0.05% polysorbate 20 (HBS-EP) and injected over the immobilized gp120 and reference cells at 50 μL/min. The data were processed with SCRUBBER-2 and double referenced by subtraction of the blank surface and a blank injection (no analyte). Binding curves were globally fit to a 1:1 binding model.</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Complex Preparation, Crystallization, Data Collection, Structure Determination, Refinement, and Analyses</h3><div class="NLM_p">HIV-1 clade B YU2 gp120 core<sub>e</sub> was expressed, purified, and deglycosylated as previously described.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The <b>13</b>–gp120 complex was formed by mixing deglycosylated YU2 gp120 and the CD4-mimetic miniprotein (1:2 molar ratio) at room temperature and purified by size exclusion chromatography (Hiload 26/60 Superdex S200 prep grade, GE Healthcare) with buffer containing 0.35 M NaCl, 2.5 mM Tris pH 7.5, 0.02% NaN<sub>3</sub>. Fractions with gp120–miniprotein complexes were concentrated to ∼10 mg/mL, flash frozen with liquid nitrogen before storing at −80 °C, and used for crystallization experiments.</div><div class="NLM_p">Vapor-diffusion hanging drops were set up by mixing 0.5 μL of protein with an equal volume of reservoir solution composed of 9.0% PEG 4000, 14.0% 2-propanol, and 100 mM sodium citrate, pH 5.6.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Droplets were allowed to equilibrate at 20 °C, and diffraction-quality crystals were obtained in 1–5 days.</div><div class="NLM_p">Diffraction data were collected under cryogenic conditions using 15% (2<i>R</i>,3<i>R</i>)-butanediol as cryoprotectant. X-ray diffraction data were collected at ID-22 beamline (SER-CAT) at the Advanced Photon Source, Argonne National Laboratory, with 1.0000 Å radiation, processed, and reduced with HKL2000.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div><div class="NLM_p">The crystal structure of the <b>13</b>–gp120 complex was solved by molecular replacement using Phaser<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> in the CCP4 Program Suite.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The structure of YU2 gp120 bound to CD4-mimetic miniprotein <b>2</b><sup>21</sup> was used as a search model (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JZZ">4JZZ</a>). Refinement was carried out with PHENIX.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Starting with torsion-angle-simulated annealing with slow cooling, iterative manual model building was carried out on COOT<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> with maps generated from combinations of standard positional, individual B-factor, TLS refinement algorithms, and noncrystallographic symmetry (NCS) restraints. Ordered solvents were added during each macro cycle. Throughout the refinement processes, a cross validation (<i>R</i><sub>free</sub>) test set consisting of 5% of the data was used, and hydrogens were included as a riding model. Structure validations were performed periodically during the model building/refinement process with MolProbity<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and pdb-care.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> X-ray crystallographic data and refinement statistics are summarized in <a class="ref internalNav" href="#notes-2" aria-label="Table S4">Table S4</a>. For reporting resolution, the data were scaled to a highest resolution of 1.89 Å. For refinement, all data collected to a highest resolution of 1.79 Å were used.</div><div class="NLM_p last">Superpositions were performed within the CCP4 program suite. Figures were made using Pymol. LIGPLOT<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> was used to generate the two-dimensional representation of <b>13</b> environment within the gp120 Phe43 cavity. Pocket-Finder (<a href="http://www.modelling.leeds.ac.uk/pocketfinder/" class="extLink">http://www.modelling.leeds.ac.uk/pocketfinder/</a>), the web interface based on the pocket detection algorithm LIGSITE,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> was used to define the volume of the Phe43 cavity and the region of the Phe43 cavity that binds to the cyclohexane ring of <b>13</b>. The volume of ligands was calculated within the ChemBio3D Ultra 12.0 module within ChemBioOffice 2010 (<a href="http://www.cambridgesoft.com/Ensemble_for_Biology/ChemBio3D" class="extLink">http://www.cambridgesoft.com/Ensemble_for_Biology/ChemBio3D</a>).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47"><a href="/doi/suppl/10.1021/jm4002988">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14284" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14284" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">MiniCD4 characterization data, gp120 sequence alignment, competition ELISA data, SPR profiles, crystallographic statistics, and supplemental figures. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm4002988/suppl_file/jm4002988_si_001.pdf">jm4002988_si_001.pdf (1.91 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Structure factors and coordinates for <b>13</b>–HIV-1–gp120 complex structure have been deposited in the Protein Data Bank with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KA2">4KA2</a>.</p><div class="testing" data-doi="10.1021/jm4002988" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01556" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01556" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Loïc Martin</span> - <span class="hlFld-Affiliation affiliation">Service d’Ingénierie Moléculaire des Protéines,
iBiTecS, CEA, F-91191 Gif-sur-Yvette, France</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5f2f3e2c3c3e3371343a2c2c333a2d1f3c3a3e71392d7f3330363c71323e2d2b36311f3c3a3e71392d"><span class="__cf_email__" data-cfemail="2c5c4d5f4f4d400247495f5f40495e6c4f494d024a5e">[email protected]</span> <span class="__cf_email__" data-cfemail="bdd1d2d4de93d0dccfc9d4d3fdded8dc93dbcf">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pascal Kessler</span> - <span class="hlFld-Affiliation affiliation">Service d’Ingénierie Moléculaire des Protéines,
iBiTecS, CEA, F-91191 Gif-sur-Yvette, France</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#82f2e3f1e1e3eeace9e7f1f1eee7f0c2e1e7e3ace4f0a2eeedebe1acefe3f0f6ebecc2e1e7e3ace4f0"><span class="__cf_email__" data-cfemail="0373627060626f2d686670706f6671436066622d6571">[email protected]</span> <span class="__cf_email__" data-cfemail="deb2b1b7bdf0b3bfacaab7b09ebdbbbff0b8ac">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurence Morellato-Castillo</span> - <span class="hlFld-Affiliation affiliation">Service d’Ingénierie Moléculaire des Protéines,
iBiTecS, CEA, F-91191 Gif-sur-Yvette, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Priyamvada Acharya</span> - <span class="hlFld-Affiliation affiliation">Vaccine Research Center, National
Institute of Allergy and Infectious Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olivier Combes</span> - <span class="hlFld-Affiliation affiliation">Service d’Ingénierie Moléculaire des Protéines,
iBiTecS, CEA, F-91191 Gif-sur-Yvette, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johan Michiels</span> - <span class="hlFld-Affiliation affiliation">Virology Unit, Department
of Biomedical Sciences, Institute of Tropical Medicine—Antwerp, Nationalestraat 155, Antwerp 2000, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne Descours</span> - <span class="hlFld-Affiliation affiliation">Service d’Ingénierie Moléculaire des Protéines,
iBiTecS, CEA, F-91191 Gif-sur-Yvette, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oscar H. P. Ramos</span> - <span class="hlFld-Affiliation affiliation">Service d’Ingénierie Moléculaire des Protéines,
iBiTecS, CEA, F-91191 Gif-sur-Yvette, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongping Yang</span> - <span class="hlFld-Affiliation affiliation">Vaccine Research Center, National
Institute of Allergy and Infectious Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guido Vanham</span> - <span class="hlFld-Affiliation affiliation">Virology Unit, Department
of Biomedical Sciences, Institute of Tropical Medicine—Antwerp, Nationalestraat 155, Antwerp 2000, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin K. Ariën</span> - <span class="hlFld-Affiliation affiliation">Virology Unit, Department
of Biomedical Sciences, Institute of Tropical Medicine—Antwerp, Nationalestraat 155, Antwerp 2000, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter D. Kwong</span> - <span class="hlFld-Affiliation affiliation">Vaccine Research Center, National
Institute of Allergy and Infectious Diseases, National
Institutes of Health, Bethesda, Maryland 20892, United
States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>L.M.-C., A.D., and P.K. synthesized the molecules. ELISA experiments were run by O.C., and J.M.. K.K.A., and G.V. obtained the neutralization data. P.A., Y.Y., P.K., L.M., and P.D.K. were involved with the structure analysis. O.H.P.R., L.M., K.K.A., P.A., P.D.K., and P.K. wrote the manuscript. L.M.-C. and P.A. contributed equally to this work. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3683-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22423" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22423" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the sixth European Union Framework Program Microbicide Project (EMPRO), by the Intramural AIDS Targeted Anti-Retroviral Program (IATAP), NIH, by the Intramural Research Program of the Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health, and Seventh Framework Program (FP7/2007-2013) under grant agreement no. 242135 (CHAARM). L120.3 Ab was provided by the Centralised Facility for AIDS Reagents supported by EU program EVA/MRC (contract QLKZ-CT-1999-00609 and GP828102) and the UK Medical Research Council. We also thank Novartis Vaccine for providing gp120SF162. Synchrotron: Use of sector 22 (Southeast Region Collaborative Access team) at the Advanced Photon Source was supported by the U.S. Department of Energy, Basic Energy Sciences, Office of Science, under Contract W-31-109-Eng-38.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i51" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i51"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i52" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i52"> Abbreviations Used</h2><tr><td class="NLM_term">gp120</td><td class="NLM_def"><p class="first last">HIV-1 surface protein</p></td></tr><tr><td class="NLM_term">miniCD4</td><td class="NLM_def"><p class="first last">CD4-mimetic miniprotein</p></td></tr><tr><td class="NLM_term">CD4</td><td class="NLM_def"><p class="first last">cluster of differentiation 4</p></td></tr><tr><td class="NLM_term">DIAD</td><td class="NLM_def"><p class="first last">diisopropyl azodicarboxylate</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i53">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28358" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28358" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Wu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerard, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parolin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruffing, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borsetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardoso, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desjardin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span> </span><span class="NLM_article-title">CD4-induced interaction of primary HIV-1 gp120 glycoproteins with the chemokine receptor CCR-5</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">179</span><span class="NLM_x">–</span> <span class="NLM_lpage">183</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=179-183&author=L.+Wuauthor=N.+P.+Gerardauthor=R.+Wyattauthor=H.+Choeauthor=C.+Parolinauthor=N.+Ruffingauthor=A.+Borsettiauthor=A.+A.+Cardosoauthor=E.+Desjardinauthor=W.+Newmanauthor=C.+Gerardauthor=J.+Sodroski&title=CD4-induced+interaction+of+primary+HIV-1+gp120+glycoproteins+with+the+chemokine+receptor+CCR-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DL.%26aulast%3DGerard%26aufirst%3DN.%2BP.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DChoe%26aufirst%3DH.%26aulast%3DParolin%26aufirst%3DC.%26aulast%3DRuffing%26aufirst%3DN.%26aulast%3DBorsetti%26aufirst%3DA.%26aulast%3DCardoso%26aufirst%3DA.%2BA.%26aulast%3DDesjardin%26aufirst%3DE.%26aulast%3DNewman%26aufirst%3DW.%26aulast%3DGerard%26aufirst%3DC.%26aulast%3DSodroski%26aufirst%3DJ.%26atitle%3DCD4-induced%2520interaction%2520of%2520primary%2520HIV-1%2520gp120%2520glycoproteins%2520with%2520the%2520chemokine%2520receptor%2520CCR-5%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D179%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Trkola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dragic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arthos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binley, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allaway, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng-Mayer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddon, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. P.</span><span> </span><span class="NLM_article-title">CD4-dependent, antibody-sensitive interactions between HIV-1 and its co-receptor CCR-5</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">184</span><span class="NLM_x">–</span> <span class="NLM_lpage">187</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=184-187&author=A.+Trkolaauthor=T.+Dragicauthor=J.+Arthosauthor=J.+M.+Binleyauthor=W.+C.+Olsonauthor=G.+P.+Allawayauthor=C.+Cheng-Mayerauthor=J.+Robinsonauthor=P.+J.+Maddonauthor=J.+P.+Moore&title=CD4-dependent%2C+antibody-sensitive+interactions+between+HIV-1+and+its+co-receptor+CCR-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrkola%26aufirst%3DA.%26aulast%3DDragic%26aufirst%3DT.%26aulast%3DArthos%26aufirst%3DJ.%26aulast%3DBinley%26aufirst%3DJ.%2BM.%26aulast%3DOlson%26aufirst%3DW.%2BC.%26aulast%3DAllaway%26aufirst%3DG.%2BP.%26aulast%3DCheng-Mayer%26aufirst%3DC.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DMaddon%26aufirst%3DP.%2BJ.%26aulast%3DMoore%26aufirst%3DJ.%2BP.%26atitle%3DCD4-dependent%252C%2520antibody-sensitive%2520interactions%2520between%2520HIV-1%2520and%2520its%2520co-receptor%2520CCR-5%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D184%26epage%3D187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broder, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, E. A.</span><span> </span><span class="NLM_article-title">HIV-1 entry cofactor: functional cDNA cloning of a seven-transmembrane, G protein-coupled receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">872</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1996&pages=872-877&author=Y.+Fengauthor=C.+C.+Broderauthor=P.+E.+Kennedyauthor=E.+A.+Berger&title=HIV-1+entry+cofactor%3A+functional+cDNA+cloning+of+a+seven-transmembrane%2C+G+protein-coupled+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DBroder%26aufirst%3DC.%2BC.%26aulast%3DKennedy%26aufirst%3DP.%2BE.%26aulast%3DBerger%26aufirst%3DE.%2BA.%26atitle%3DHIV-1%2520entry%2520cofactor%253A%2520functional%2520cDNA%2520cloning%2520of%2520a%2520seven-transmembrane%252C%2520G%2520protein-coupled%2520receptor%26jtitle%3DScience%26date%3D1996%26volume%3D272%26spage%3D872%26epage%3D877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span> </span><span class="NLM_article-title">The HIV-1 envelope glycoproteins: Fusogens, antigens, and immunogens</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">1884</span><span class="NLM_x">–</span> <span class="NLM_lpage">1888</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=1998&pages=1884-1888&author=R.+Wyattauthor=J.+Sodroski&title=The+HIV-1+envelope+glycoproteins%3A+Fusogens%2C+antigens%2C+and+immunogens"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DSodroski%26aufirst%3DJ.%26atitle%3DThe%2520HIV-1%2520envelope%2520glycoproteins%253A%2520Fusogens%252C%2520antigens%252C%2520and%2520immunogens%26jtitle%3DScience%26date%3D1998%26volume%3D280%26spage%3D1884%26epage%3D1888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Blair, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meanwell, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, O. B.</span><span> </span><span class="NLM_article-title">HIV-1 entry - an expanding portal for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=183-194&author=W.+S.+Blairauthor=P.+F.+Linauthor=N.+A.+Meanwellauthor=O.+B.+Wallace&title=HIV-1+entry+-+an+expanding+portal+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DW.%2BS.%26aulast%3DLin%26aufirst%3DP.%2BF.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26aulast%3DWallace%26aufirst%3DO.%2BB.%26atitle%3DHIV-1%2520entry%2520-%2520an%2520expanding%2520portal%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2000%26volume%3D5%26spage%3D183%26epage%3D194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Moore, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doms, R. W.</span><span> </span><span class="NLM_article-title">The entry of entry inhibitors: A fusion of science and medicine</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">10598</span><span class="NLM_x">–</span> <span class="NLM_lpage">10602</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=10598-10602&author=J.+P.+Mooreauthor=R.+W.+Doms&title=The+entry+of+entry+inhibitors%3A+A+fusion+of+science+and+medicine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DJ.%2BP.%26aulast%3DDoms%26aufirst%3DR.%2BW.%26atitle%3DThe%2520entry%2520of%2520entry%2520inhibitors%253A%2520A%2520fusion%2520of%2520science%2520and%2520medicine%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2003%26volume%3D100%26spage%3D10598%26epage%3D10602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Vermeire, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schols, D.</span><span> </span><span class="NLM_article-title">Anti-HIV agents targeting the interaction of gp120 with the cellular CD4 receptor</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1199</span><span class="NLM_x">–</span> <span class="NLM_lpage">1212</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=1199-1212&author=K.+Vermeireauthor=D.+Schols&title=Anti-HIV+agents+targeting+the+interaction+of+gp120+with+the+cellular+CD4+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVermeire%26aufirst%3DK.%26aulast%3DSchols%26aufirst%3DD.%26atitle%3DAnti-HIV%2520agents%2520targeting%2520the%2520interaction%2520of%2520gp120%2520with%2520the%2520cellular%2520CD4%2520receptor%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2005%26volume%3D14%26spage%3D1199%26epage%3D1212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Ryser, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fluckiger, R.</span><span> </span><span class="NLM_article-title">Progress in targeting HIV-1 entry</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1085</span><span class="NLM_x">–</span> <span class="NLM_lpage">1094</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=1085-1094&author=H.+J.+Ryserauthor=R.+Fluckiger&title=Progress+in+targeting+HIV-1+entry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRyser%26aufirst%3DH.%2BJ.%26aulast%3DFluckiger%26aufirst%3DR.%26atitle%3DProgress%2520in%2520targeting%2520HIV-1%2520entry%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D1085%26epage%3D1094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Vita, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drakopoulou, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vizzavona, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rochette, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ménez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumestand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ylisastigui, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benjouad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gluckman, J. C.</span><span> </span><span class="NLM_article-title">Rational engineering of a miniprotein that reproduces the core of the CD4 site interacting with HIV-1 envelope glycoprotein</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">13091</span><span class="NLM_x">–</span> <span class="NLM_lpage">13096</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=13091-13096&author=C.+Vitaauthor=E.+Drakopoulouauthor=J.+Vizzavonaauthor=S.+Rochetteauthor=L.+Martinauthor=A.+M%C3%A9nezauthor=C.+Roumestandauthor=Y.+S.+Yangauthor=L.+Ylisastiguiauthor=A.+Benjouadauthor=J.+C.+Gluckman&title=Rational+engineering+of+a+miniprotein+that+reproduces+the+core+of+the+CD4+site+interacting+with+HIV-1+envelope+glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVita%26aufirst%3DC.%26aulast%3DDrakopoulou%26aufirst%3DE.%26aulast%3DVizzavona%26aufirst%3DJ.%26aulast%3DRochette%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DRoumestand%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DY.%2BS.%26aulast%3DYlisastigui%26aufirst%3DL.%26aulast%3DBenjouad%26aufirst%3DA.%26aulast%3DGluckman%26aufirst%3DJ.%2BC.%26atitle%3DRational%2520engineering%2520of%2520a%2520miniprotein%2520that%2520reproduces%2520the%2520core%2520of%2520the%2520CD4%2520site%2520interacting%2520with%2520HIV-1%2520envelope%2520glycoprotein%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D13091%26epage%3D13096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Getz, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daugherty, P. S.</span><span> </span><span class="NLM_article-title">Protease-resistant peptide ligands from a knottin scaffold library</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">837</span><span class="NLM_x">–</span> <span class="NLM_lpage">844</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb200039s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsFWhsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=837-844&author=J.+A.+Getzauthor=J.+J.+Riceauthor=P.+S.+Daugherty&title=Protease-resistant+peptide+ligands+from+a+knottin+scaffold+library"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protease-Resistant Peptide Ligands from a Knottin Scaffold Library</span></div><div class="casAuthors">Getz, Jennifer A.; Rice, Jeffrey J.; Daugherty, Patrick S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">837-844</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptides within the knottin family have been shown to possess inherent stability, making them attractive scaffolds for the development of therapeutic and diagnostic agents.  Given its remarkable stability to proteases, the cyclic peptide kalata B1 was employed as a scaffold to create a large knottin library displayed on the surface of E. coli.  A library exceeding 109 variants was constructed by randomizing seven amino acids within a loop of the kalata B1 scaffold and screened using fluorescence-activated cell sorting to identify peptide ligands specific for the active site of human thrombin.  Refolded thrombin binders exhibited high nanomolar affinities in soln. and slow dissocn. rates and were able to inhibit thrombin's enzymic activity.  Importantly, 80% of a knottin-based thrombin inhibitor remained intact after a 2 h incubation both with trypsin and with chymotrypsin, demonstrating that modifying the kalata B1 sequence did not compromise its stability properties.  In addn., the knottin variant mediated 20-fold enhanced affinity for thrombin, when compared to the same seven residue binding epitope constrained by a single disulfide bond.  Our results indicate that peptide libraries derived from the kalata B1 scaffold can yield high-affinity protein ligands that retain the remarkable protease resistance assocd. with the parent scaffold.  More generally, this strategy may prove useful in the development of stable peptide ligands suitable for in vivo applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqckK3Dh6_BV7Vg90H21EOLACvtfcHk0ljBH7_svUoytw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsFWhsb8%253D&md5=cf0a71b43b8fc192e17f05734938bc9a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fcb200039s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb200039s%26sid%3Dliteratum%253Aachs%26aulast%3DGetz%26aufirst%3DJ.%2BA.%26aulast%3DRice%26aufirst%3DJ.%2BJ.%26aulast%3DDaugherty%26aufirst%3DP.%2BS.%26atitle%3DProtease-resistant%2520peptide%2520ligands%2520from%2520a%2520knottin%2520scaffold%2520library%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D837%26epage%3D844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Martins, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tartar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazdunski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borremans, F. A. M.</span><span> </span><span class="NLM_article-title">Solution conformation of leiurotoxin-I (Scyllatoxin) by H-1 nuclear magnetic-resonance - Resonance assignment and secondary structure</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">260</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=260&publication_year=1990&pages=249-253&author=J.+C.+Martinsauthor=W.+G.+Zhangauthor=A.+Tartarauthor=M.+Lazdunskiauthor=F.+A.+M.+Borremans&title=Solution+conformation+of+leiurotoxin-I+%28Scyllatoxin%29+by+H-1+nuclear+magnetic-resonance+-+Resonance+assignment+and+secondary+structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartins%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DW.%2BG.%26aulast%3DTartar%26aufirst%3DA.%26aulast%3DLazdunski%26aufirst%3DM.%26aulast%3DBorremans%26aufirst%3DF.%2BA.%2BM.%26atitle%3DSolution%2520conformation%2520of%2520leiurotoxin-I%2520%2528Scyllatoxin%2529%2520by%2520H-1%2520nuclear%2520magnetic-resonance%2520-%2520Resonance%2520assignment%2520and%2520secondary%2520structure%26jtitle%3DFEBS%2520Lett.%26date%3D1990%26volume%3D260%26spage%3D249%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Drakopoulou, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinn-Justin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guenneugues, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilquin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ménez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span> </span><span class="NLM_article-title">Changing the structural context of a functional beta-hairpin. Synthesis and characterization of a chimera containing the curaremimetic loop of a snake toxin in the scorpion alpha/beta scaffold</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">271</span><span class="NLM_x">, </span> <span class="NLM_fpage">11979</span><span class="NLM_x">–</span> <span class="NLM_lpage">11987</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=11979-11987&author=E.+Drakopoulouauthor=S.+Zinn-Justinauthor=M.+Guenneuguesauthor=B.+Gilquinauthor=A.+M%C3%A9nezauthor=C.+Vita&title=Changing+the+structural+context+of+a+functional+beta-hairpin.+Synthesis+and+characterization+of+a+chimera+containing+the+curaremimetic+loop+of+a+snake+toxin+in+the+scorpion+alpha%2Fbeta+scaffold"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDrakopoulou%26aufirst%3DE.%26aulast%3DZinn-Justin%26aufirst%3DS.%26aulast%3DGuenneugues%26aufirst%3DM.%26aulast%3DGilquin%26aufirst%3DB.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DVita%26aufirst%3DC.%26atitle%3DChanging%2520the%2520structural%2520context%2520of%2520a%2520functional%2520beta-hairpin.%2520Synthesis%2520and%2520characterization%2520of%2520a%2520chimera%2520containing%2520the%2520curaremimetic%2520loop%2520of%2520a%2520snake%2520toxin%2520in%2520the%2520scorpion%2520alpha%252Fbeta%2520scaffold%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D11979%26epage%3D11987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stricher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misse, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sironi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pugniere, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barthe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prado-Gotor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freulon, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magne, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumestand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ménez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lusso, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veas, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span> </span><span class="NLM_article-title">Rational design of a CD4 mimic that inhibits HIV-1 entry and exposes cryptic neutralization epitopes</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2003&pages=71-76&author=L.+Martinauthor=F.+Stricherauthor=D.+Misseauthor=F.+Sironiauthor=M.+Pugniereauthor=P.+Bartheauthor=R.+Prado-Gotorauthor=I.+Freulonauthor=X.+Magneauthor=C.+Roumestandauthor=A.+M%C3%A9nezauthor=P.+Lussoauthor=F.+Veasauthor=C.+Vita&title=Rational+design+of+a+CD4+mimic+that+inhibits+HIV-1+entry+and+exposes+cryptic+neutralization+epitopes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DStricher%26aufirst%3DF.%26aulast%3DMisse%26aufirst%3DD.%26aulast%3DSironi%26aufirst%3DF.%26aulast%3DPugniere%26aufirst%3DM.%26aulast%3DBarthe%26aufirst%3DP.%26aulast%3DPrado-Gotor%26aufirst%3DR.%26aulast%3DFreulon%26aufirst%3DI.%26aulast%3DMagne%26aufirst%3DX.%26aulast%3DRoumestand%26aufirst%3DC.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DLusso%26aufirst%3DP.%26aulast%3DVeas%26aufirst%3DF.%26aulast%3DVita%26aufirst%3DC.%26atitle%3DRational%2520design%2520of%2520a%2520CD4%2520mimic%2520that%2520inhibits%2520HIV-1%2520entry%2520and%2520exposes%2520cryptic%2520neutralization%2520epitopes%26jtitle%3DNat.%2520Biotechnol.%26date%3D2003%26volume%3D21%26spage%3D71%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyd, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joly, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ménez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulmer, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, I. K.</span><span> </span><span class="NLM_article-title">Development of covalent HIV-1 gp120-CD4 mimic complexes for vaccine application</span> <span class="citation_source-journal">Antiviral Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">P09d10</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=P09d10&issue=Suppl+2&author=L.+Martinauthor=B.+Heydauthor=Y.+Sunauthor=G.+Martinauthor=E.+Kanauthor=P.+Jolyauthor=P.+Kesslerauthor=A.+M%C3%A9nezauthor=J.+B.+Ulmerauthor=C.+Vitaauthor=B.+W.+Barnettauthor=I.+K.+Srivastava&title=Development+of+covalent+HIV-1+gp120-CD4+mimic+complexes+for+vaccine+application"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DHeyd%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DMartin%26aufirst%3DG.%26aulast%3DKan%26aufirst%3DE.%26aulast%3DJoly%26aufirst%3DP.%26aulast%3DKessler%26aufirst%3DP.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DUlmer%26aufirst%3DJ.%2BB.%26aulast%3DVita%26aufirst%3DC.%26aulast%3DBarnett%26aufirst%3DB.%2BW.%26aulast%3DSrivastava%26aufirst%3DI.%2BK.%26atitle%3DDevelopment%2520of%2520covalent%2520HIV-1%2520gp120-CD4%2520mimic%2520complexes%2520for%2520vaccine%2520application%26jtitle%3DAntiviral%2520Ther.%26date%3D2006%26volume%3D11%26issue%3DSuppl%25202%26spage%3DP09d10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Stricher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Descours, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquesnoy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combes, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decker, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lusso, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span> </span><span class="NLM_article-title">Combinatorial optimization of a CD4-mimetic miniprotein and cocrystal structures with HIV-1 gp120 envelope glycoprotein</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">382</span><span class="NLM_x">, </span> <span class="NLM_fpage">510</span><span class="NLM_x">–</span> <span class="NLM_lpage">524</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2008&pages=510-524&author=F.+Stricherauthor=C.+C.+Huangauthor=A.+Descoursauthor=S.+Duquesnoyauthor=O.+Combesauthor=J.+M.+Deckerauthor=Y.+D.+Kwonauthor=P.+Lussoauthor=G.+M.+Shawauthor=C.+Vitaauthor=P.+D.+Kwongauthor=L.+Martin&title=Combinatorial+optimization+of+a+CD4-mimetic+miniprotein+and+cocrystal+structures+with+HIV-1+gp120+envelope+glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStricher%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DDescours%26aufirst%3DA.%26aulast%3DDuquesnoy%26aufirst%3DS.%26aulast%3DCombes%26aufirst%3DO.%26aulast%3DDecker%26aufirst%3DJ.%2BM.%26aulast%3DKwon%26aufirst%3DY.%2BD.%26aulast%3DLusso%26aufirst%3DP.%26aulast%3DShaw%26aufirst%3DG.%2BM.%26aulast%3DVita%26aufirst%3DC.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DMartin%26aufirst%3DL.%26atitle%3DCombinatorial%2520optimization%2520of%2520a%2520CD4-mimetic%2520miniprotein%2520and%2520cocrystal%2520structures%2520with%2520HIV-1%2520gp120%2520envelope%2520glycoprotein%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D382%26spage%3D510%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stricher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decker, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barthe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumestand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Scorpion-toxin mimics of CD4 in complex with human immunodeficiency virus gp120 crystal structures, molecular mimicry, and neutralization breadth</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">768</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=755-768&author=C.+C.+Huangauthor=F.+Stricherauthor=L.+Martinauthor=J.+M.+Deckerauthor=S.+Majeedauthor=P.+Bartheauthor=W.+A.+Hendricksonauthor=J.+Robinsonauthor=C.+Roumestandauthor=J.+Sodroskiauthor=R.+Wyattauthor=G.+M.+Shawauthor=C.+Vitaauthor=P.+D.+Kwong&title=Scorpion-toxin+mimics+of+CD4+in+complex+with+human+immunodeficiency+virus+gp120+crystal+structures%2C+molecular+mimicry%2C+and+neutralization+breadth"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DStricher%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DDecker%26aufirst%3DJ.%2BM.%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DBarthe%26aufirst%3DP.%26aulast%3DHendrickson%26aufirst%3DW.%2BA.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DRoumestand%26aufirst%3DC.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DShaw%26aufirst%3DG.%2BM.%26aulast%3DVita%26aufirst%3DC.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DScorpion-toxin%2520mimics%2520of%2520CD4%2520in%2520complex%2520with%2520human%2520immunodeficiency%2520virus%2520gp120%2520crystal%2520structures%252C%2520molecular%2520mimicry%252C%2520and%2520neutralization%2520breadth%26jtitle%3DStructure%26date%3D2005%26volume%3D13%26spage%3D755%26epage%3D768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Martin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combes, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyd, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geonnotti, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montefiori, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abache, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulmer, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madaoui, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guerois, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span> </span><span class="NLM_article-title">Stabilization of HIV-1 Envelope in the CD4-bound conformation through specific cross-linking of a CD4 mimetic</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">21706</span><span class="NLM_x">–</span> <span class="NLM_lpage">21716</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=21706-21716&author=G.+Martinauthor=B.+Burkeauthor=R.+Thaiauthor=A.+K.+Deyauthor=O.+Combesauthor=B.+Heydauthor=A.+R.+Geonnottiauthor=D.+C.+Montefioriauthor=E.+Kanauthor=Y.+Lianauthor=Y.+D.+Sunauthor=T.+Abacheauthor=J.+B.+Ulmerauthor=H.+Madaouiauthor=R.+Gueroisauthor=S.+W.+Barnettauthor=I.+K.+Srivastavaauthor=P.+Kesslerauthor=L.+Martin&title=Stabilization+of+HIV-1+Envelope+in+the+CD4-bound+conformation+through+specific+cross-linking+of+a+CD4+mimetic"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%26aulast%3DBurke%26aufirst%3DB.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DDey%26aufirst%3DA.%2BK.%26aulast%3DCombes%26aufirst%3DO.%26aulast%3DHeyd%26aufirst%3DB.%26aulast%3DGeonnotti%26aufirst%3DA.%2BR.%26aulast%3DMontefiori%26aufirst%3DD.%2BC.%26aulast%3DKan%26aufirst%3DE.%26aulast%3DLian%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%2BD.%26aulast%3DAbache%26aufirst%3DT.%26aulast%3DUlmer%26aufirst%3DJ.%2BB.%26aulast%3DMadaoui%26aufirst%3DH.%26aulast%3DGuerois%26aufirst%3DR.%26aulast%3DBarnett%26aufirst%3DS.%2BW.%26aulast%3DSrivastava%26aufirst%3DI.%2BK.%26aulast%3DKessler%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DL.%26atitle%3DStabilization%2520of%2520HIV-1%2520Envelope%2520in%2520the%2520CD4-bound%2520conformation%2520through%2520specific%2520cross-linking%2520of%2520a%2520CD4%2520mimetic%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D21706%26epage%3D21716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Dey, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirokman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nandi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartog, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geonnotti, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montefiori, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carfi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, S. W.</span><span> </span><span class="NLM_article-title">Elicitation of neutralizing antibodies directed against CD4-induced epitope(s) using a CD4 mimetic cross-linked to a HIV-1 envelope glycoprotein</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e30233</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e30233&author=A.+K.+Deyauthor=B.+Burkeauthor=Y.+D.+Sunauthor=K.+Sirokmanauthor=A.+Nandiauthor=K.+Hartogauthor=Y.+Lianauthor=A.+R.+Geonnottiauthor=D.+Montefioriauthor=M.+Frantiauthor=G.+Martinauthor=A.+Carfiauthor=P.+Kesslerauthor=L.+Martinauthor=I.+K.+Srivastavaauthor=S.+W.+Barnett&title=Elicitation+of+neutralizing+antibodies+directed+against+CD4-induced+epitope%28s%29+using+a+CD4+mimetic+cross-linked+to+a+HIV-1+envelope+glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDey%26aufirst%3DA.%2BK.%26aulast%3DBurke%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DY.%2BD.%26aulast%3DSirokman%26aufirst%3DK.%26aulast%3DNandi%26aufirst%3DA.%26aulast%3DHartog%26aufirst%3DK.%26aulast%3DLian%26aufirst%3DY.%26aulast%3DGeonnotti%26aufirst%3DA.%2BR.%26aulast%3DMontefiori%26aufirst%3DD.%26aulast%3DFranti%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DG.%26aulast%3DCarfi%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DSrivastava%26aufirst%3DI.%2BK.%26aulast%3DBarnett%26aufirst%3DS.%2BW.%26atitle%3DElicitation%2520of%2520neutralizing%2520antibodies%2520directed%2520against%2520CD4-induced%2520epitope%2528s%2529%2520using%2520a%2520CD4%2520mimetic%2520cross-linked%2520to%2520a%2520HIV-1%2520envelope%2520glycoprotein%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De30233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Van Herrewege, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morellato, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Descours, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aerts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michiels, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyndrickx, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanham, G.</span><span> </span><span class="NLM_article-title">CD4 mimetic miniproteins: potent anti-HIV compounds with promising activity as microbicides</span> <span class="citation_source-journal">J. Antimicrob. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1093%2Fjac%2Fdkn042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=18270220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjs1Ojsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2008&pages=818-826&author=Y.+Van+Herrewegeauthor=L.+Morellatoauthor=A.+Descoursauthor=L.+Aertsauthor=J.+Michielsauthor=L.+Heyndrickxauthor=L.+Martinauthor=G.+Vanham&title=CD4+mimetic+miniproteins%3A+potent+anti-HIV+compounds+with+promising+activity+as+microbicides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">CD4 mimetic miniproteins: potent anti-HIV compounds with promising activity as microbicides</span></div><div class="casAuthors">Van Herrewege, Yven; Morellato, Laurence; Descours, Anne; Aerts, Laetitia; Michiels, Jo; Heyndrickx, Leo; Martin, Loic; Vanham, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">818-826</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives: The antiviral activity of CD4 miniproteins was evaluated as potential HIV microbicides, using relevant in vitro models.  Methods: Compds. were tested in a single-cycle HIV-1 pseudovirus assay and against replication competent HIV-1 in cocultures of monocyte-derived dendritic cells (MO-DC) and CD4+ T cells.  Cytotoxic activity was evaluated in an MTT assay.  Results: Monomeric miniproteins (M47 and M48) showed 50% effective concn. (EC50) values of 79-105 nM against a subtype B, CCR5 coreceptor-using Ba-L pseudovirus.  Higher activity was found for the dimeric miniproteins M48D30, M48D50 and M48D100 (EC50 between 15 and 30 nM), in contrast to the tetrameric miniproteins M48T30, M48T50 and M48T100 (EC50 between 107 and 377 nM).  The hetero-bivalent miniprotein M48-Hep and miniproteins that targeted the Phe-43 cavity on gp120 (M48-U1, M48-U2 and M48-U3) were highly active, with EC50 values as low as 2 nM for M48-U1.  All miniproteins showed high activity against CCR5 or CXCR4 coreceptor-using subtype B and CRF-01.perp.A/E pseudoviruses.  Many early M48-based compds. were much less active against subtype C pseudoviruses, whereas M48-U compds. that targeted the Phe-43 cavity were very active against all pseudoviruses, including subtype C.  In MO-DC/CD4+ T cell cocultures with replication-competent HIV-1 Ba-L, EC50 values ranged between 13 and 1719 nM depending on the miniprotein, with M48-U1, M48-U2 and M48-U3 again being the most potent.  Importantly, the latter compds. completely prevented viral replication by treating the cultures from 2 h before until 24 h after infection, at nontoxic concns. of 66-6564 nM.  Conclusions: These novel CD4 miniproteins might constitute a promising class of HIV microbicides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsNtvf_idCdrVg90H21EOLACvtfcHk0lipQfrSuXv--g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjs1Ojsrg%253D&md5=6bc1668700a84fee7b137398c91d68a8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkn042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkn042%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BHerrewege%26aufirst%3DY.%26aulast%3DMorellato%26aufirst%3DL.%26aulast%3DDescours%26aufirst%3DA.%26aulast%3DAerts%26aufirst%3DL.%26aulast%3DMichiels%26aufirst%3DJ.%26aulast%3DHeyndrickx%26aufirst%3DL.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DVanham%26aufirst%3DG.%26atitle%3DCD4%2520mimetic%2520miniproteins%253A%2520potent%2520anti-HIV%2520compounds%2520with%2520promising%2520activity%2520as%2520microbicides%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2008%26volume%3D61%26spage%3D818%26epage%3D826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Dereuddre-Bosquet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morellato-Castillo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brouwers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustijns, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouchemal, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponchel, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, O. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrera, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stefanidou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shattock, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyndrickx, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span> </span><span class="NLM_article-title">MiniCD4 microbicide prevents HIV infection of human mucosal explants and vaginal transmission of SHIV(162P3) in Cynomolgus macaques</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003071</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=e1003071&author=N.+Dereuddre-Bosquetauthor=L.+Morellato-Castilloauthor=J.+Brouwersauthor=P.+Augustijnsauthor=K.+Bouchemalauthor=G.+Ponchelauthor=O.+H.+Ramosauthor=C.+Herreraauthor=M.+Stefanidouauthor=R.+Shattockauthor=L.+Heyndrickxauthor=G.+Vanhamauthor=P.+Kesslerauthor=R.+Le+Grandauthor=L.+Martin&title=MiniCD4+microbicide+prevents+HIV+infection+of+human+mucosal+explants+and+vaginal+transmission+of+SHIV%28162P3%29+in+Cynomolgus+macaques"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDereuddre-Bosquet%26aufirst%3DN.%26aulast%3DMorellato-Castillo%26aufirst%3DL.%26aulast%3DBrouwers%26aufirst%3DJ.%26aulast%3DAugustijns%26aufirst%3DP.%26aulast%3DBouchemal%26aufirst%3DK.%26aulast%3DPonchel%26aufirst%3DG.%26aulast%3DRamos%26aufirst%3DO.%2BH.%26aulast%3DHerrera%26aufirst%3DC.%26aulast%3DStefanidou%26aufirst%3DM.%26aulast%3DShattock%26aufirst%3DR.%26aulast%3DHeyndrickx%26aufirst%3DL.%26aulast%3DVanham%26aufirst%3DG.%26aulast%3DKessler%26aufirst%3DP.%26aulast%3DLe%2BGrand%26aufirst%3DR.%26aulast%3DMartin%26aufirst%3DL.%26atitle%3DMiniCD4%2520microbicide%2520prevents%2520HIV%2520infection%2520of%2520human%2520mucosal%2520explants%2520and%2520vaginal%2520transmission%2520of%2520SHIV%2528162P3%2529%2520in%2520Cynomolgus%2520macaques%26jtitle%3DPLoS%2520Pathog.%26date%3D2012%26volume%3D8%26spage%3De1003071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Acharya, P.; Luongo, T.; Louder, M. K.; McKee, K.; Yang, Y.; Kwon, Y. D.; Mascola, J. R.; Kessler, P.; Martin, L.; Kwong, P. D.</span><span> </span><span class="NLM_article-title">Structural basis for highly effective HIV-1 neutralization by CD4-mimetic miniproteins revealed by 1.5 Å co-crystal structure of gp120 and M48U1</span>,  <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1016/j.str.2013.04.015</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1016%2Fj.str.2013.04.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=P.+Acharya&author=T.+Luongo&author=M.+K.+Louder&author=K.+McKee&author=Y.+Yang&author=Y.+D.+Kwon&author=J.+R.+Mascola&author=P.+Kessler&author=L.+Martin&author=P.+D.+Kwong&title=Structural+basis+for+highly+effective+HIV-1+neutralization+by+CD4-mimetic+miniproteins+revealed+by+1.5+%C3%85+co-crystal+structure+of+gp120+and+M48U1&doi=10.1016%2Fj.str.2013.04.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DAcharya%26aufirst%3DP.%26atitle%3DStructural%2520basis%2520for%2520highly%2520effective%2520HIV-1%2520neutralization%2520by%2520CD4-mimetic%2520miniproteins%2520revealed%2520by%25201.5%2520%25C3%2585%2520co-crystal%2520structure%2520of%2520gp120%2520and%2520M48U1%26jtitle%3DStructure%26date%3D2013%26doi%3D10.1016%2Fj.str.2013.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Morley, A. D.</span><span> </span><span class="NLM_article-title">Optimisation and synthesis of libraries derived from phenolic amino acid scaffolds</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">7405</span><span class="NLM_x">–</span> <span class="NLM_lpage">7408</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2000&pages=7405-7408&author=A.+D.+Morley&title=Optimisation+and+synthesis+of+libraries+derived+from+phenolic+amino+acid+scaffolds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorley%26aufirst%3DA.%2BD.%26atitle%3DOptimisation%2520and%2520synthesis%2520of%2520libraries%2520derived%2520from%2520phenolic%2520amino%2520acid%2520scaffolds%26jtitle%3DTetrahedron%2520Lett.%26date%3D2000%26volume%3D41%26spage%3D7405%26epage%3D7408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Morley, A. D.</span><span> </span><span class="NLM_article-title">Allyloxycarbonyl - a useful protecting group for phenolic amino acids and applications on solid support</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">7401</span><span class="NLM_x">–</span> <span class="NLM_lpage">7404</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2000&pages=7401-7404&author=A.+D.+Morley&title=Allyloxycarbonyl+-+a+useful+protecting+group+for+phenolic+amino+acids+and+applications+on+solid+support"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorley%26aufirst%3DA.%2BD.%26atitle%3DAllyloxycarbonyl%2520-%2520a%2520useful%2520protecting%2520group%2520for%2520phenolic%2520amino%2520acids%2520and%2520applications%2520on%2520solid%2520support%26jtitle%3DTetrahedron%2520Lett.%26date%3D2000%26volume%3D41%26spage%3D7401%26epage%3D7404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Kwon, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finzi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dogo-Isonagie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vicic, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debnath, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bewley, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascola, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Unliganded HIV-1 gp120 core structures assume the CD4-bound conformation with regulation by quaternary interactions and variable loops</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5663</span><span class="NLM_x">–</span> <span class="NLM_lpage">5668</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1073%2Fpnas.1112391109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=22451932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVOisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5663-5668&author=Y.+D.+Kwonauthor=A.+Finziauthor=X.+Wuauthor=C.+Dogo-Isonagieauthor=L.+K.+Leeauthor=L.+R.+Mooreauthor=S.+D.+Schmidtauthor=J.+Stuckeyauthor=Y.+Yangauthor=T.+Zhouauthor=J.+Zhuauthor=D.+A.+Vicicauthor=A.+K.+Debnathauthor=L.+Shapiroauthor=C.+A.+Bewleyauthor=J.+R.+Mascolaauthor=J.+G.+Sodroskiauthor=P.+D.+Kwong&title=Unliganded+HIV-1+gp120+core+structures+assume+the+CD4-bound+conformation+with+regulation+by+quaternary+interactions+and+variable+loops"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Unliganded HIV-1 gp120 core structures assume the CD4-bound conformation with regulation by quaternary interactions and variable loops</span></div><div class="casAuthors">Kwon, Young Do; Finzi, Andres; Wu, Xueling; Dogo-Isonagie, Cajetan; Lee, Lawrence K.; Moore, Lucas R.; Schmidt, Stephen D.; Stuckey, Jonathan; Yang, Yongping; Zhou, Tongqing; Zhu, Jiang; Vicic, David A.; Debnath, Asim K.; Shapiro, Lawrence; Bewley, Carole A.; Mascola, John R.; Sodroski, Joseph G.; Kwong, Peter D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5663-5668, S5663/1-S5663/33</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The HIV-1 envelope (Env) spike (gp1203/gp413) undergoes considerable structural rearrangements to mediate virus entry into cells and to evade the host immune response.  Engagement of CD4, the primary human receptor, fixes a particular conformation and primes Env for entry.  The CD4-bound state, however, is prone to spontaneous inactivation and susceptible to antibody neutralization.  How does unliganded HIV-1 maintain CD4-binding capacity and regulate transitions to the CD4-bound state.  To define this mechanistically, we detd. crystal structures of unliganded core gp12O from HIV-1 clades B, C, and E.  Notably, all of these unliganded HIV-1 structures resembled the CD4-bound state.  Conformational fixation with ligand selection and thermodn. anal. of full-length and core gp12O interactions revealed that the tendency of HIV-1 gp12O to adopt the CD4-bound conformation was restrained by the V1 N2- and V3-variable loops.  In parallel, we detd. the structure of core gp120 in complex with the small mol., NBD-556, which specifically recognizes the CD4-bound conformation of gp120.  Neutralization by NBD-556 indicated that Env spikes on primary isolates rarely assume the CD4-bound conformation spontaneously, although they could do so when quaternary restraints were loosened.  Together, the results suggest that the CD4-bound conformation represents a "ground state" for the gp12O core, with variable loop and quaternary interactions restraining unliganded gp12O from "snapping" into this conformation.  A mechanism of control involving deformations in unliganded structure from a functionally crit. state (e.g., the CD4-bound state) provides advantages in terms of HIV-1 Env structural diversity and resistance to antibodies and inhibitors, while maintaining elements essential for entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS3b-Y8pziGrVg90H21EOLACvtfcHk0lipQfrSuXv--g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVOisLc%253D&md5=2ad1a0459b9ede5cd0d3db0116fd7a0b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1112391109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1112391109%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DY.%2BD.%26aulast%3DFinzi%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DDogo-Isonagie%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DL.%2BK.%26aulast%3DMoore%26aufirst%3DL.%2BR.%26aulast%3DSchmidt%26aufirst%3DS.%2BD.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DVicic%26aufirst%3DD.%2BA.%26aulast%3DDebnath%26aufirst%3DA.%2BK.%26aulast%3DShapiro%26aufirst%3DL.%26aulast%3DBewley%26aufirst%3DC.%2BA.%26aulast%3DMascola%26aufirst%3DJ.%2BR.%26aulast%3DSodroski%26aufirst%3DJ.%2BG.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DUnliganded%2520HIV-1%2520gp120%2520core%2520structures%2520assume%2520the%2520CD4-bound%2520conformation%2520with%2520regulation%2520by%2520quaternary%2520interactions%2520and%2520variable%2520loops%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D5663%26epage%3D5668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Cromwell, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bystrom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eftink, M. R.</span><span> </span><span class="NLM_article-title">Cyclodextrin adamantanecarboxylate inclusion complexes - Studies of the variation in cavity size</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">326</span><span class="NLM_x">–</span> <span class="NLM_lpage">332</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100248a029" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1985&pages=326-332&author=W.+C.+Cromwellauthor=K.+Bystromauthor=M.+R.+Eftink&title=Cyclodextrin+adamantanecarboxylate+inclusion+complexes+-+Studies+of+the+variation+in+cavity+size"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fj100248a029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100248a029%26sid%3Dliteratum%253Aachs%26aulast%3DCromwell%26aufirst%3DW.%2BC.%26aulast%3DBystrom%26aufirst%3DK.%26aulast%3DEftink%26aufirst%3DM.%2BR.%26atitle%3DCyclodextrin%2520adamantanecarboxylate%2520inclusion%2520complexes%2520-%2520Studies%2520of%2520the%2520variation%2520in%2520cavity%2520size%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1985%26volume%3D89%26spage%3D326%26epage%3D332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Mecozzi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebek, J.</span><span> </span><span class="NLM_article-title">The 55% solution: A formula for molecular recognition in the liquid state</span> <span class="citation_source-journal">Chem.—Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1016</span><span class="NLM_x">–</span> <span class="NLM_lpage">1022</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1002%2F%28SICI%291521-3765%2819980615%294%3A6%3C1016%3A%3AAID-CHEM1016%3E3.0.CO%3B2-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADyaK1cXktV2gtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1998&pages=1016-1022&author=S.+Mecozziauthor=J.+Rebek&title=The+55%25+solution%3A+A+formula+for+molecular+recognition+in+the+liquid+state"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The 55% solution: a formula for molecular recognition in the liquid state</span></div><div class="casAuthors">Mecozzi, Sandro; Rebek, Julius, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1016-1022</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Evidence is presented that mol. recognition through encapsulation processes is largely detd. by the vols. of the guest and host.  Binding of mols. of suitable dimensions in the internal cavity of a mol. receptor in soln. can be expected when the packing coeff., the ratio of the guest vol. to the host vol., is in the range of 0.55 ± 0.09.  Larger packing coeffs., up to 0.70, can be reached if the complex is stabilized by strong intermol. forces such as H bonds.  These considerations also apply to situations in which >1 mol. is encapsulated.  Org. liqs. are generally characterized by the same packing factors as encapsulation complexes, and the short-range structure of liqs. and the complexes resulting from encapsulation are 2 aspects of the same phenomenon.  Similar vol. considerations are expected to apply to the binding of substrates in biol. receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDT6U43hO-XLVg90H21EOLACvtfcHk0lg-ynC8AnBYEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXktV2gtLs%253D&md5=c10f319493dc1ce518b3967eacb1dbf3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3765%2819980615%294%3A6%3C1016%3A%3AAID-CHEM1016%3E3.0.CO%3B2-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3765%252819980615%25294%253A6%253C1016%253A%253AAID-CHEM1016%253E3.0.CO%253B2-B%26sid%3Dliteratum%253Aachs%26aulast%3DMecozzi%26aufirst%3DS.%26aulast%3DRebek%26aufirst%3DJ.%26atitle%3DThe%252055%2525%2520solution%253A%2520A%2520formula%2520for%2520molecular%2520recognition%2520in%2520the%2520liquid%2520state%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D1998%26volume%3D4%26spage%3D1016%26epage%3D1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Zürcher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Structure-based drug design: exploring the proper filling of apolar pockets at enzyme active sites</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">4345</span><span class="NLM_x">–</span> <span class="NLM_lpage">4361</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo800527n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=4345-4361&author=M.+Z%C3%BCrcherauthor=F.+Diederich&title=Structure-based+drug+design%3A+exploring+the+proper+filling+of+apolar+pockets+at+enzyme+active+sites"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjo800527n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo800527n%26sid%3Dliteratum%253Aachs%26aulast%3DZ%25C3%25BCrcher%26aufirst%3DM.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DStructure-based%2520drug%2520design%253A%2520exploring%2520the%2520proper%2520filling%2520of%2520apolar%2520pockets%2520at%2520enzyme%2520active%2520sites%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26spage%3D4345%26epage%3D4361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Kawasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chufan, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafont, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hidaka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiso, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mario Amzel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freire, E.</span><span> </span><span class="NLM_article-title">How much binding affinity can be gained by filling a cavity?</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">143</span><span class="NLM_x">–</span> <span class="NLM_lpage">151</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1111%2Fj.1747-0285.2009.00921.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=20028396" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVSgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=143-151&author=Y.+Kawasakiauthor=E.+E.+Chufanauthor=V.+Lafontauthor=K.+Hidakaauthor=Y.+Kisoauthor=L.+Mario+Amzelauthor=E.+Freire&title=How+much+binding+affinity+can+be+gained+by+filling+a+cavity%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">How much binding affinity can be gained by filling a cavity?</span></div><div class="casAuthors">Kawasaki, Yuko; Chufan, Eduardo E.; Lafont, Virginie; Hidaka, Koushi; Kiso, Yoshiaki; Amzel, L. Mario; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-151</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Binding affinity optimization is crit. during drug development.  Here, we evaluate the thermodn. consequences of filling a binding cavity with functionalities of increasing van der Waals radii (-H, -F, -Cl, and CH3) that improve the geometric fit without participating in hydrogen bonding or other specific interactions.  We observe a binding affinity increase of two orders of magnitude.  There appears to be three phases in the process.  The first phase is assocd. with the formation of stable van der Waals interactions.  This phase is characterized by a gain in binding enthalpy and a loss in binding entropy, attributed to a loss of conformational degrees of freedom.  For the specific case presented in this article, the enthalpy gain amts. to -1.5 kcal/mol while the entropic losses amt. to +0.9 kcal/mol resulting in a net 3.5-fold affinity gain.  The second phase is characterized by simultaneous enthalpic and entropic gains.  This phase improves the binding affinity 25-fold.  The third phase represents the collapse of the trend and is triggered by the introduction of chem. functionalities larger than the binding cavity itself [CH(CH3)2].  It is characterized by large enthalpy and affinity losses.  The thermodn. signatures assocd. with each phase provide guidelines for lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJLOlqtVTY2rVg90H21EOLACvtfcHk0lg-ynC8AnBYEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVSgsg%253D%253D&md5=cf92f069de6def681083f12554ec9bad</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2009.00921.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2009.00921.x%26sid%3Dliteratum%253Aachs%26aulast%3DKawasaki%26aufirst%3DY.%26aulast%3DChufan%26aufirst%3DE.%2BE.%26aulast%3DLafont%26aufirst%3DV.%26aulast%3DHidaka%26aufirst%3DK.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DMario%2BAmzel%26aufirst%3DL.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DHow%2520much%2520binding%2520affinity%2520can%2520be%2520gained%2520by%2520filling%2520a%2520cavity%253F%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2010%26volume%3D75%26spage%3D143%26epage%3D151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, W. A.</span><span> </span><span class="NLM_article-title">Structure of an HIV gp120 envelope glycoprotein in complex with the CD4 receptor and a neutralizing human antibody</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">393</span><span class="NLM_x">, </span> <span class="NLM_fpage">648</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1038%2F31405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=9641677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADyaK1cXktVynsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=393&publication_year=1998&pages=648-659&author=P.+D.+Kwongauthor=R.+Wyattauthor=J.+Robinsonauthor=R.+W.+Sweetauthor=J.+Sodroskiauthor=W.+A.+Hendrickson&title=Structure+of+an+HIV+gp120+envelope+glycoprotein+in+complex+with+the+CD4+receptor+and+a+neutralizing+human+antibody"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of an HIV gp120 envelope glycoprotein in complex with the CD4 receptor and a neutralizing human antibody</span></div><div class="casAuthors">Kwong, Peter D.; Wyatt, Richard; Robinson, James; Sweet, Raymond W.; Sodroski, Joseph; Hendrickson, Wayne A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">393</span>
        (<span class="NLM_cas:issue">6686</span>),
    <span class="NLM_cas:pages">648-659</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The entry of human immunodeficiency virus (HIV) into cells requires the sequential interaction of the viral exterior envelope glycoprotein, gp120, with the CD4 glycoprotein and chemokine receptor on the cell surface.  These interactions initiate a fusion of the viral and cellular membranes.  Although gp120 can elicit virus-neutralizing antibodies, HIV eludes the immune system.  The authors have solved the x-ray crystal structure at 2.5 Å resoln. of an HIV-1 gp120 core complexed with a two-domain fragment of human CD4 and an antigen-binding fragment of a neutralizing antibody that blocks chemokine-receptor binding.  The structure reveals a cavity-laden CD4-gp120 interface, a conserved binding site for the chemokine receptor, evidence for a conformational change upon CD4 binding, the nature of a CD4-induced antibody epitope, and specific mechanisms for immune evasion.  The results provide a framework for understanding the complex biol. of HIV entry into cells and should guide efforts to intervene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5NOtfNxdMJ7Vg90H21EOLACvtfcHk0liMMjhz_DUYug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXktVynsrs%253D&md5=5ce7f5923ab2e9c8c37f940ac71429ed</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2F31405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F31405%26sid%3Dliteratum%253Aachs%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DSweet%26aufirst%3DR.%2BW.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DHendrickson%26aufirst%3DW.%2BA.%26atitle%3DStructure%2520of%2520an%2520HIV%2520gp120%2520envelope%2520glycoprotein%2520in%2520complex%2520with%2520the%2520CD4%2520receptor%2520and%2520a%2520neutralizing%2520human%2520antibody%26jtitle%3DNature%26date%3D1998%26volume%3D393%26spage%3D648%26epage%3D659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, W. A.</span><span> </span><span class="NLM_article-title">Structures of HIV-1 gp120 envelope glycoproteins from laboratory-adapted and primary isolates</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1329</span><span class="NLM_x">–</span> <span class="NLM_lpage">1339</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2000&pages=1329-1339&author=P.+D.+Kwongauthor=R.+Wyattauthor=S.+Majeedauthor=J.+Robinsonauthor=R.+W.+Sweetauthor=J.+Sodroskiauthor=W.+A.+Hendrickson&title=Structures+of+HIV-1+gp120+envelope+glycoproteins+from+laboratory-adapted+and+primary+isolates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DSweet%26aufirst%3DR.%2BW.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DHendrickson%26aufirst%3DW.%2BA.%26atitle%3DStructures%2520of%2520HIV-1%2520gp120%2520envelope%2520glycoproteins%2520from%2520laboratory-adapted%2520and%2520primary%2520isolates%26jtitle%3DStructure%26date%3D2000%26volume%3D8%26spage%3D1329%26epage%3D1339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montabana, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanfield, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimitrov, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korber, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Structure of a V3-containing HIV-1 gp120 core</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">310</span><span class="NLM_x">, </span> <span class="NLM_fpage">1025</span><span class="NLM_x">–</span> <span class="NLM_lpage">1028</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=310&publication_year=2005&pages=1025-1028&author=C.+C.+Huangauthor=M.+Tangauthor=M.+Y.+Zhangauthor=S.+Majeedauthor=E.+Montabanaauthor=R.+L.+Stanfieldauthor=D.+S.+Dimitrovauthor=B.+Korberauthor=J.+Sodroskiauthor=I.+A.+Wilsonauthor=R.+Wyattauthor=P.+D.+Kwong&title=Structure+of+a+V3-containing+HIV-1+gp120+core"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DM.%2BY.%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DMontabana%26aufirst%3DE.%26aulast%3DStanfield%26aufirst%3DR.%2BL.%26aulast%3DDimitrov%26aufirst%3DD.%2BS.%26aulast%3DKorber%26aufirst%3DB.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DStructure%2520of%2520a%2520V3-containing%2520HIV-1%2520gp120%2520core%26jtitle%3DScience%26date%3D2005%26volume%3D310%26spage%3D1025%26epage%3D1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Pancera, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ban, Y. E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schief, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Structure of HIV-1 gp120 with gp41-interactive region reveals layered envelope architecture and basis of conformational mobility</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">1166</span><span class="NLM_x">–</span> <span class="NLM_lpage">1171</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=1166-1171&author=M.+Panceraauthor=S.+Majeedauthor=Y.+E.+A.+Banauthor=L.+Chenauthor=C.+C.+Huangauthor=L.+Kongauthor=Y.+D.+Kwonauthor=J.+Stuckeyauthor=T.+Q.+Zhouauthor=J.+E.+Robinsonauthor=W.+R.+Schiefauthor=J.+Sodroskiauthor=R.+Wyattauthor=P.+D.+Kwong&title=Structure+of+HIV-1+gp120+with+gp41-interactive+region+reveals+layered+envelope+architecture+and+basis+of+conformational+mobility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPancera%26aufirst%3DM.%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DBan%26aufirst%3DY.%2BE.%2BA.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DKong%26aufirst%3DL.%26aulast%3DKwon%26aufirst%3DY.%2BD.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DT.%2BQ.%26aulast%3DRobinson%26aufirst%3DJ.%2BE.%26aulast%3DSchief%26aufirst%3DW.%2BR.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DStructure%2520of%2520HIV-1%2520gp120%2520with%2520gp41-interactive%2520region%2520reveals%2520layered%2520envelope%2520architecture%2520and%2520basis%2520of%2520conformational%2520mobility%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D1166%26epage%3D1171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Diskin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcovecchio, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjorkman, P. J.</span><span> </span><span class="NLM_article-title">Structure of a clade C HIV-1 gp120 bound to CD4 and CD4-induced antibody reveals anti-CD4 polyreactivity</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">613</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1038%2Fnsmb.1796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=20357769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVGjtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=608-613&author=R.+Diskinauthor=P.+M.+Marcovecchioauthor=P.+J.+Bjorkman&title=Structure+of+a+clade+C+HIV-1+gp120+bound+to+CD4+and+CD4-induced+antibody+reveals+anti-CD4+polyreactivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a clade C HIV-1 gp120 bound to CD4 and CD4-induced antibody reveals anti-CD4 polyreactivity</span></div><div class="casAuthors">Diskin, Ron; Marcovecchio, Paola M.; Bjorkman, Pamela J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">608-613</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Strategies to combat HIV-1 require structural knowledge of envelope proteins from viruses in HIV-1 clade C, the most rapidly spreading subtype in the world.  We present a crystal structure contg. a clade C gp120 envelope.  The structure, a complex between gp120, the host receptor CD4 and the CD4-induced antibody 21c, reveals that the 21c epitope involves contacts with gp120, a nonself antigen, and with CD4, an autoantigen.  Binding studies using wild-type and mutant CD4 show that 21c Fab binds CD4 in the absence of gp120, and that binding of 21c to clade C and HIV-2 gp120s requires the crystallog. obsd. 21c-CD4 interaction.  Addnl. binding data suggest a role for the gp120 V1V2 loop in creating a high-affinity, but slow-forming, epitope for 21c after CD4 binds.  These results contribute to a mol. understanding of CD4-induced antibodies and provide the first visualization to our knowledge of a potentially autoreactive antibody Fab complexed with both self and nonself antigens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouhXh7BjzYVLVg90H21EOLACvtfcHk0liMMjhz_DUYug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVGjtLk%253D&md5=fa3447c62732cab7bef67312f69836b9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1796%26sid%3Dliteratum%253Aachs%26aulast%3DDiskin%26aufirst%3DR.%26aulast%3DMarcovecchio%26aufirst%3DP.%2BM.%26aulast%3DBjorkman%26aufirst%3DP.%2BJ.%26atitle%3DStructure%2520of%2520a%2520clade%2520C%2520HIV-1%2520gp120%2520bound%2520to%2520CD4%2520and%2520CD4-induced%2520antibody%2520reveals%2520anti-CD4%2520polyreactivity%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D608%26epage%3D613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Wu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiev, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longo, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Dell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perfetto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonsignori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crump, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapiga, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sam, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koff, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doria-Rose, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connors, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullikin, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nabel, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roederer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascola, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Progra, N. C. S.</span><span> </span><span class="NLM_article-title">Focused evolution of HIV-1 neutralizing antibodies revealed by structures and deep sequencing</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">333</span><span class="NLM_x">, </span> <span class="NLM_fpage">1593</span><span class="NLM_x">–</span> <span class="NLM_lpage">1602</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=333&publication_year=2011&pages=1593-1602&author=X.+L.+Wuauthor=T.+Q.+Zhouauthor=J.+Zhuauthor=B.+S.+Zhangauthor=I.+Georgievauthor=C.+Wangauthor=X.+J.+Chenauthor=N.+S.+Longoauthor=M.+Louderauthor=K.+McKeeauthor=S.+O%E2%80%99Dellauthor=S.+Perfettoauthor=S.+D.+Schmidtauthor=W.+Shiauthor=L.+Wuauthor=Y.+P.+Yangauthor=Z.+Y.+Yangauthor=Z.+J.+Yangauthor=Z.+H.+Zhangauthor=M.+Bonsignoriauthor=J.+A.+Crumpauthor=S.+H.+Kapigaauthor=N.+E.+Samauthor=B.+F.+Haynesauthor=M.+Simekauthor=D.+R.+Burtonauthor=W.+C.+Koffauthor=N.+A.+Doria-Roseauthor=M.+Connorsauthor=J.+C.+Mullikinauthor=G.+J.+Nabelauthor=M.+Roedererauthor=L.+Shapiroauthor=P.+D.+Kwongauthor=J.+R.+Mascolaauthor=N.+C.+S.+Progra&title=Focused+evolution+of+HIV-1+neutralizing+antibodies+revealed+by+structures+and+deep+sequencing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DX.%2BL.%26aulast%3DZhou%26aufirst%3DT.%2BQ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%2BS.%26aulast%3DGeorgiev%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DX.%2BJ.%26aulast%3DLongo%26aufirst%3DN.%2BS.%26aulast%3DLouder%26aufirst%3DM.%26aulast%3DMcKee%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Dell%26aufirst%3DS.%26aulast%3DPerfetto%26aufirst%3DS.%26aulast%3DSchmidt%26aufirst%3DS.%2BD.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DY.%2BP.%26aulast%3DYang%26aufirst%3DZ.%2BY.%26aulast%3DYang%26aufirst%3DZ.%2BJ.%26aulast%3DZhang%26aufirst%3DZ.%2BH.%26aulast%3DBonsignori%26aufirst%3DM.%26aulast%3DCrump%26aufirst%3DJ.%2BA.%26aulast%3DKapiga%26aufirst%3DS.%2BH.%26aulast%3DSam%26aufirst%3DN.%2BE.%26aulast%3DHaynes%26aufirst%3DB.%2BF.%26aulast%3DSimek%26aufirst%3DM.%26aulast%3DBurton%26aufirst%3DD.%2BR.%26aulast%3DKoff%26aufirst%3DW.%2BC.%26aulast%3DDoria-Rose%26aufirst%3DN.%2BA.%26aulast%3DConnors%26aufirst%3DM.%26aulast%3DMullikin%26aufirst%3DJ.%2BC.%26aulast%3DNabel%26aufirst%3DG.%2BJ.%26aulast%3DRoederer%26aufirst%3DM.%26aulast%3DShapiro%26aufirst%3DL.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DMascola%26aufirst%3DJ.%2BR.%26aulast%3DProgra%26aufirst%3DN.%2BC.%2BS.%26atitle%3DFocused%2520evolution%2520of%2520HIV-1%2520neutralizing%2520antibodies%2520revealed%2520by%2520structures%2520and%2520deep%2520sequencing%26jtitle%3DScience%26date%3D2011%26volume%3D333%26spage%3D1593%26epage%3D1602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">LaLonde, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courter, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soeta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Princiotto, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freire, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascola, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madani, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. B.</span><span> </span><span class="NLM_article-title">Structure-based design, synthesis, and characterization of dual hotspot small-molecule HIV-1 entry inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4382</span><span class="NLM_x">–</span> <span class="NLM_lpage">4396</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300265j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4382-4396&author=J.+M.+LaLondeauthor=Y.+D.+Kwonauthor=D.+M.+Jonesauthor=A.+W.+Sunauthor=J.+R.+Courterauthor=T.+Soetaauthor=T.+Kobayashiauthor=A.+M.+Princiottoauthor=X.+L.+Wuauthor=A.+Schonauthor=E.+Freireauthor=P.+D.+Kwongauthor=J.+R.+Mascolaauthor=J.+Sodroskiauthor=N.+Madaniauthor=A.+B.+Smith&title=Structure-based+design%2C+synthesis%2C+and+characterization+of+dual+hotspot+small-molecule+HIV-1+entry+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm300265j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300265j%26sid%3Dliteratum%253Aachs%26aulast%3DLaLonde%26aufirst%3DJ.%2BM.%26aulast%3DKwon%26aufirst%3DY.%2BD.%26aulast%3DJones%26aufirst%3DD.%2BM.%26aulast%3DSun%26aufirst%3DA.%2BW.%26aulast%3DCourter%26aufirst%3DJ.%2BR.%26aulast%3DSoeta%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DPrinciotto%26aufirst%3DA.%2BM.%26aulast%3DWu%26aufirst%3DX.%2BL.%26aulast%3DSchon%26aufirst%3DA.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DMascola%26aufirst%3DJ.%2BR.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DMadani%26aufirst%3DN.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26atitle%3DStructure-based%2520design%252C%2520synthesis%252C%2520and%2520characterization%2520of%2520dual%2520hotspot%2520small-molecule%2520HIV-1%2520entry%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4382%26epage%3D4396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Xie, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savinov, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. B.</span><span> </span><span class="NLM_article-title">3rd; Hendrickson, W. A. Structure–activity relationships in the binding of chemically derivatized CD4 to gp120 from human immunodeficiency virus</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">4898</span><span class="NLM_x">–</span> <span class="NLM_lpage">4908</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm070564e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=4898-4908&author=H.+Xieauthor=D.+Ngauthor=S.+N.+Savinovauthor=B.+Deyauthor=P.+D.+Kwongauthor=R.+Wyattauthor=A.+B.+Smith&title=3rd%3B+Hendrickson%2C+W.+A.+Structure%E2%80%93activity+relationships+in+the+binding+of+chemically+derivatized+CD4+to+gp120+from+human+immunodeficiency+virus"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm070564e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070564e%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DH.%26aulast%3DNg%26aufirst%3DD.%26aulast%3DSavinov%26aufirst%3DS.%2BN.%26aulast%3DDey%26aufirst%3DB.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26atitle%3D3rd%253B%2520Hendrickson%252C%2520W.%2520A.%2520Structure%25E2%2580%2593activity%2520relationships%2520in%2520the%2520binding%2520of%2520chemically%2520derivatized%2520CD4%2520to%2520gp120%2520from%2520human%2520immunodeficiency%2520virus%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D4898%26epage%3D4908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Stevens, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, R.</span><span> </span><span class="NLM_article-title">Amino Acid Derivatives 0.1. Carboallyloxy derivatives of alpha-amino acids</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1950</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">–</span> <span class="NLM_lpage">727</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja01158a020" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1950&pages=725-727&author=C.+M.+Stevensauthor=R.+Watanabe&title=Amino+Acid+Derivatives+0.1.+Carboallyloxy+derivatives+of+alpha-amino+acids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja01158a020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja01158a020%26sid%3Dliteratum%253Aachs%26aulast%3DStevens%26aufirst%3DC.%2BM.%26aulast%3DWatanabe%26aufirst%3DR.%26atitle%3DAmino%2520Acid%2520Derivatives%25200.1.%2520Carboallyloxy%2520derivatives%2520of%2520alpha-amino%2520acids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1950%26volume%3D72%26spage%3D725%26epage%3D727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ofek, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belachew, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Enhancing protein crystallization through precipitant synergy</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1061</span><span class="NLM_x">–</span> <span class="NLM_lpage">1070</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=1061-1070&author=S.+Majeedauthor=G.+Ofekauthor=A.+Belachewauthor=C.+C.+Huangauthor=T.+Zhouauthor=P.+D.+Kwong&title=Enhancing+protein+crystallization+through+precipitant+synergy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DOfek%26aufirst%3DG.%26aulast%3DBelachew%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DEnhancing%2520protein%2520crystallization%2520through%2520precipitant%2520synergy%26jtitle%3DStructure%26date%3D2003%26volume%3D11%26spage%3D1061%26epage%3D1070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Method Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0ljJ0NkZVXTMcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethod%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Mccoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+Mccoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljJ0NkZVXTMcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">McKinnon, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nyanga, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chege, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izulla, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huibner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelmon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cicala, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anzala, A. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arthos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaul, R.</span><span> </span><span class="NLM_article-title">Characterization of a human cervical CD4(+) T cell subset coexpressing multiple markers of HIV susceptibility</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">187</span><span class="NLM_x">, </span> <span class="NLM_fpage">6032</span><span class="NLM_x">–</span> <span class="NLM_lpage">6042</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2011&pages=6032-6042&author=L.+R.+McKinnonauthor=B.+Nyangaauthor=D.+Chegeauthor=P.+Izullaauthor=M.+Kimaniauthor=S.+Huibnerauthor=L.+Gelmonauthor=K.+E.+Blockauthor=C.+Cicalaauthor=A.+O.+Anzalaauthor=J.+Arthosauthor=J.+Kimaniauthor=R.+Kaul&title=Characterization+of+a+human+cervical+CD4%28%2B%29+T+cell+subset+coexpressing+multiple+markers+of+HIV+susceptibility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcKinnon%26aufirst%3DL.%2BR.%26aulast%3DNyanga%26aufirst%3DB.%26aulast%3DChege%26aufirst%3DD.%26aulast%3DIzulla%26aufirst%3DP.%26aulast%3DKimani%26aufirst%3DM.%26aulast%3DHuibner%26aufirst%3DS.%26aulast%3DGelmon%26aufirst%3DL.%26aulast%3DBlock%26aufirst%3DK.%2BE.%26aulast%3DCicala%26aufirst%3DC.%26aulast%3DAnzala%26aufirst%3DA.%2BO.%26aulast%3DArthos%26aufirst%3DJ.%26aulast%3DKimani%26aufirst%3DJ.%26aulast%3DKaul%26aufirst%3DR.%26atitle%3DCharacterization%2520of%2520a%2520human%2520cervical%2520CD4%2528%252B%2529%2520T%2520cell%2520subset%2520coexpressing%2520multiple%2520markers%2520of%2520HIV%2520susceptibility%26jtitle%3DJ.%2520Immunol.%26date%3D2011%26volume%3D187%26spage%3D6032%26epage%3D6042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhxwfGVz87jxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhxwfGVz87jxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leaver-Fay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snoeyink, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">W375</span><span class="NLM_x">–</span> <span class="NLM_lpage">W383</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1093%2Fnar%2Fgkm216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=17452350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W375-W383&author=I.+W.+Davisauthor=A.+Leaver-Fayauthor=V.+B.+Chenauthor=J.+N.+Blockauthor=G.+J.+Kapralauthor=X.+Wangauthor=L.+W.+Murrayauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+contacts+and+structure+validation+for+proteins+and+nucleic+acids"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span></div><div class="casAuthors">Davis Ian W; Leaver-Fay Andrew; Chen Vincent B; Block Jeremy N; Kapral Gary J; Wang Xueyi; Murray Laura W; Arendall W Bryan 3rd; Snoeyink Jack; Richardson Jane S; Richardson David C</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W375-83</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MolProbity is a general-purpose web server offering quality validation for 3D structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact analysis of any steric problems within the molecules as well as updated dihedral-angle diagnostics, and it can calculate and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addition and full optimization of all hydrogen atoms, both polar and nonpolar.  New analysis functions have been added for RNA, for interfaces, and for NMR ensembles.  Additionally, both the web site and major component programs have been rewritten to improve speed, convenience, clarity and integration with other resources.  MolProbity results are reported in multiple forms: as overall numeric scores, as lists or charts of local problems, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KiNG viewer.  This service is available free to all users at http://molprobity.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuyhuetoNS59bHMBt0mMIXfW6udTcc2eY77Ss7oRKeG7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D&md5=726ca1e7e162ab2a4a05048f493731fb</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm216%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DLeaver-Fay%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DBlock%26aufirst%3DJ.%2BN.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520contacts%2520and%2520structure%2520validation%2520for%2520proteins%2520and%2520nucleic%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW375%26epage%3DW383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Lutteke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von der Lieth, C.</span><span> </span><span class="NLM_article-title">W. pdb-care (PDB CArbohydrate REsidue check): a program to support annotation of complex carbohydrate structures in PDB files</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1186%2F1471-2105-5-69" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=15180909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A280%3ADC%252BD2czitlKntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=69&author=T.+Luttekeauthor=C.+von+der+Lieth&title=W.+pdb-care+%28PDB+CArbohydrate+REsidue+check%29%3A+a+program+to+support+annotation+of+complex+carbohydrate+structures+in+PDB+files"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">pdb-care (PDB carbohydrate residue check): a program to support annotation of complex carbohydrate structures in PDB files</span></div><div class="casAuthors">Lutteke Thomas; von der Lieth Claus-W</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">69</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Carbohydrates are involved in a variety of fundamental biological processes and pathological situations.  They therefore have a large pharmaceutical and diagnostic potential.  Knowledge of the 3D structure of glycans is a prerequisite for a complete understanding of their biological functions.  The largest source of biomolecular 3D structures is the Protein Data Bank.  However, about 30% of all 1663 PDB entries (version September 2003) containing carbohydrates comprise errors in glycan description.  Unfortunately, no software is currently available which aligns the 3D information with the reported assignments.  It is the aim of this work to fill this gap.  RESULTS:  The pdb-care program http://www.glycosciences.de/tools/pdb-care/ is able to identify and assign carbohydrate structures using only atom types and their 3D atom coordinates given in PDB-files.  Looking up a translation table where systematic names and the respective PDB residue codes are listed, both assignments are compared and inconsistencies are reported.  Additionally, the reliability of reported and calculated connectivities for molecules listed within the HETATOM records is checked and unusual values are reported.  CONCLUSION:  Frequent use of pdb-care will help to improve the quality of carbohydrate data contained in the PDB.  Automatic assignment of carbohydrate structures contained in PDB entries will enable the cross-linking of glycobiology resources with genomic and proteomic data collections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTpJZU-N5JJxnT4ugvCDgnZfW6udTcc2eZ3oxIo03TyAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czitlKntQ%253D%253D&md5=1bb57895df47d192c04f8e3866a4f648</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-5-69&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-5-69%26sid%3Dliteratum%253Aachs%26aulast%3DLutteke%26aufirst%3DT.%26aulast%3Dvon%2Bder%2BLieth%26aufirst%3DC.%26atitle%3DW.%2520pdb-care%2520%2528PDB%2520CArbohydrate%2520REsidue%2520check%2529%253A%2520a%2520program%2520to%2520support%2520annotation%2520of%2520complex%2520carbohydrate%2520structures%2520in%2520PDB%2520files%26jtitle%3DBMC%2520Bioinf.%26date%3D2004%26volume%3D5%26spage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Wallace, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1093%2Fprotein%2F8.2.127" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1996&pages=127-134&author=A.+C.+Wallaceauthor=R.+A.+Laskowskiauthor=J.+M.+Thornton&title=LIGPLOT%3A+a+program+to+generate+schematic+diagrams+of+protein-ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.2.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.2.127%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DA.%2BC.%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DLIGPLOT%253A%2520a%2520program%2520to%2520generate%2520schematic%2520diagrams%2520of%2520protein-ligand%2520interactions%26jtitle%3DProtein%2520Eng.%26date%3D1996%26volume%3D8%26spage%3D127%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnickel, G.</span><span> </span><span class="NLM_article-title">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span> <span class="citation_source-journal">J. Mol. Graph. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span><div class="note"><p class="first last">389.</p></div></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1016%2FS1093-3263%2898%2900002-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=9704298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADyaK1cXltFyktb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=359-363&author=M.+Hendlichauthor=F.+Rippmannauthor=G.+Barnickel&title=LIGSITE%3A+automatic+and+efficient+detection+of+potential+small+molecule-binding+sites+in+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span></div><div class="casAuthors">Hendlich, Manfred; Rippmann, Friedrich; Barnickel, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">359-363</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">LIGSITE is a new program for the automatic and time-efficient detection of pockets on the surface of proteins that may act as binding sites for small mol. ligands.  Pockets are identified with a series of simple operations on a cubic grid.  Using a set of receptor-ligand complexes we show that LIGSITE is able to identify the binding sites of small mol. ligands with high precision.  The main advantage of LIGSITE is its speed.  Typical search times are in the range of 5 to 20 s for medium-sized proteins.  LIGSITE is therefore well suited for identification of pockets in large sets of proteins (e.g., protein families) for comparative studies.  For graphical display LIGSITE produces VRML representations of the protein-ligand complex and the binding site for display with a VRML viewer such as WebSpace from SGI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLm2ioggbs97Vg90H21EOLACvtfcHk0ljrtJnRcqLlEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltFyktb0%253D&md5=27ccb5fdb0c07d5658f21c5570c8422b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2898%2900002-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252898%252900002-3%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DBarnickel%26aufirst%3DG.%26atitle%3DLIGSITE%253A%2520automatic%2520and%2520efficient%2520detection%2520of%2520potential%2520small%2520molecule-binding%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Graph.%2520Model.%26date%3D1997%26volume%3D15%26spage%3D359%26epage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 17 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adel A.  Rashad</span>, <span class="hlFld-ContribAuthor ">Ramalingam Venkat  Kalyana Sundaram</span>, <span class="hlFld-ContribAuthor ">Rachna  Aneja</span>, <span class="hlFld-ContribAuthor ">Caitlin  Duffy</span>, and <span class="hlFld-ContribAuthor ">Irwin  Chaiken</span>  . </span><span class="cited-content_cbyCitation_article-title">Macrocyclic Envelope Glycoprotein Antagonists that Irreversibly Inactivate HIV-1 before Host Cell Encounter. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (18)
                                     , 7603-7608. <a href="https://doi.org/10.1021/acs.jmedchem.5b00935" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00935</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b00935%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMacrocyclic%252BEnvelope%252BGlycoprotein%252BAntagonists%252Bthat%252BIrreversibly%252BInactivate%252BHIV-1%252Bbefore%252BHost%252BCell%252BEncounter%26aulast%3DRashad%26aufirst%3DAdel%2BA.%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D17062015%26date%3D11092015%26date%3D24092015%26date%3D02092015%26volume%3D58%26issue%3D18%26spage%3D7603%26epage%3D7608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiyong  Park</span>, <span class="hlFld-ContribAuthor ">Hung V.  Pham</span>, <span class="hlFld-ContribAuthor ">Kristian  Mogensen</span>, <span class="hlFld-ContribAuthor ">Theis Ivan  Solling</span>, <span class="hlFld-ContribAuthor ">Martin  Vad Bennetzen</span>, and <span class="hlFld-ContribAuthor ">K. N.  Houk</span>  . </span><span class="cited-content_cbyCitation_article-title">Hydrocarbon Binding by Proteins: Structures of Protein Binding Sites for ≥C10 Linear Alkanes or Long-Chain Alkyl and Alkenyl Groups. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2015,</strong> <em>80 </em>
                                    (2)
                                     , 997-1005. <a href="https://doi.org/10.1021/jo502488e" title="DOI URL">https://doi.org/10.1021/jo502488e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jo502488e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjo502488e%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DHydrocarbon%252BBinding%252Bby%252BProteins%25253A%252BStructures%252Bof%252BProtein%252BBinding%252BSites%252Bfor%252B%2525E2%252589%2525A5C10%252BLinear%252BAlkanes%252Bor%252BLong-Chain%252BAlkyl%252Band%252BAlkenyl%252BGroups%26aulast%3DPark%26aufirst%3DJiyong%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D01112014%26date%3D02012015%26date%3D16012015%26date%3D19122014%26volume%3D80%26issue%3D2%26spage%3D997%26epage%3D1005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tianshu  Xiao</span>, <span class="hlFld-ContribAuthor ">Yongfei  Cai</span>, <span class="hlFld-ContribAuthor ">Bing  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">HIV-1 Entry and Membrane Fusion Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2021,</strong> <em>13 </em>
                                    (5)
                                     , 735. <a href="https://doi.org/10.3390/v13050735" title="DOI URL">https://doi.org/10.3390/v13050735</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v13050735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv13050735%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DHIV-1%252BEntry%252Band%252BMembrane%252BFusion%252BInhibitors%26aulast%3DXiao%26aufirst%3DTianshu%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D5%26spage%3D735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ji  Yu</span>, <span class="hlFld-ContribAuthor ">Zhongxia  Zhou</span>, <span class="hlFld-ContribAuthor ">Jacob  Kongsted</span>, <span class="hlFld-ContribAuthor ">Yuning  Song</span>, <span class="hlFld-ContribAuthor ">Christophe  Pannecouque</span>, <span class="hlFld-ContribAuthor ">Erik  De Clercq</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Vasanthanathan  Poongavanam</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design opportunities presented by solvent‐exposed regions of target proteins. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2019,</strong> <em>39 </em>
                                    (6)
                                     , 2194-2238. <a href="https://doi.org/10.1002/med.21581" title="DOI URL">https://doi.org/10.1002/med.21581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21581%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DMolecular%252Bdesign%252Bopportunities%252Bpresented%252Bby%252Bsolvent%2525E2%252580%252590exposed%252Bregions%252Bof%252Btarget%252Bproteins%26aulast%3DJiang%26aufirst%3DXiangyi%26date%3D2019%26date%3D2019%26volume%3D39%26issue%3D6%26spage%3D2194%26epage%3D2238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Khomson  Suttisintong</span>, <span class="hlFld-ContribAuthor ">Narongpol  Kaewchangwat</span>, <span class="hlFld-ContribAuthor ">Eknarin  Thanayupong</span>, <span class="hlFld-ContribAuthor ">Chakkrapan  Nerungsi</span>, <span class="hlFld-ContribAuthor ">Onsiri  Srikun</span>, <span class="hlFld-ContribAuthor ">Pornpan  Pungpo</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Progress in the Development of HIV-1 Entry Inhibitors: From Small Molecules to Potent Anti-HIV Agents. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>19 </em>
                                    (18)
                                     , 1599-1620. <a href="https://doi.org/10.2174/1568026619666190712204050" title="DOI URL">https://doi.org/10.2174/1568026619666190712204050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026619666190712204050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026619666190712204050%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BProgress%252Bin%252Bthe%252BDevelopment%252Bof%252BHIV-1%252BEntry%252BInhibitors%25253A%252BFrom%252BSmall%252BMolecules%252Bto%252BPotent%252BAnti-HIV%252BAgents%26aulast%3DSuttisintong%26aufirst%3DKhomson%26date%3D2019%26volume%3D19%26issue%3D18%26spage%3D1599%26epage%3D1620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pu</span>, <span class="hlFld-ContribAuthor ">Wang</span>, <span class="hlFld-ContribAuthor ">Xu</span>, <span class="hlFld-ContribAuthor ">Lu</span>, <span class="hlFld-ContribAuthor ">Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Protein- and Peptide-Based HIV Entry Inhibitors Targeting gp120 or gp41. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2019,</strong> <em>11 </em>
                                    (8)
                                     , 705. <a href="https://doi.org/10.3390/v11080705" title="DOI URL">https://doi.org/10.3390/v11080705</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v11080705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv11080705%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DDevelopment%252Bof%252BProtein-%252Band%252BPeptide-Based%252BHIV%252BEntry%252BInhibitors%252BTargeting%252Bgp120%252Bor%252Bgp41%26aulast%3DPu%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D8%26spage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander M.  Andrianov</span>, <span class="hlFld-ContribAuthor ">Grigory I.  Nikolaev</span>, <span class="hlFld-ContribAuthor ">Yuri V.  Kornoushenko</span>, <span class="hlFld-ContribAuthor ">Wei  Xu</span>, <span class="hlFld-ContribAuthor ">Shibo  Jiang</span>, <span class="hlFld-ContribAuthor ">Alexander V.  Tuzikov</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Identification of Novel Aromatic Compounds as Potential HIV-1 Entry Inhibitors Mimicking Cellular Receptor CD4. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2019,</strong> <em>11 </em>
                                    (8)
                                     , 746. <a href="https://doi.org/10.3390/v11080746" title="DOI URL">https://doi.org/10.3390/v11080746</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v11080746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv11080746%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DIn%252BSilico%252BIdentification%252Bof%252BNovel%252BAromatic%252BCompounds%252Bas%252BPotential%252BHIV-1%252BEntry%252BInhibitors%252BMimicking%252BCellular%252BReceptor%252BCD4%26aulast%3DAndrianov%26aufirst%3DAlexander%2BM.%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D8%26spage%3D746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Philip O.  Adero</span>, <span class="hlFld-ContribAuthor ">Dean R.  Jarois</span>, <span class="hlFld-ContribAuthor ">David  Crich</span>. </span><span class="cited-content_cbyCitation_article-title">Hydrogenolytic cleavage of naphthylmethyl ethers in the presence of sulfides. </span><span class="cited-content_cbyCitation_journal-name">Carbohydrate Research</span><span> <strong>2017,</strong> <em>449 </em>, 11-16. <a href="https://doi.org/10.1016/j.carres.2017.06.011" title="DOI URL">https://doi.org/10.1016/j.carres.2017.06.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.carres.2017.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.carres.2017.06.011%26sid%3Dliteratum%253Aachs%26jtitle%3DCarbohydrate%2520Research%26atitle%3DHydrogenolytic%252Bcleavage%252Bof%252Bnaphthylmethyl%252Bethers%252Bin%252Bthe%252Bpresence%252Bof%252Bsulfides%26aulast%3DAdero%26aufirst%3DPhilip%2BO.%26date%3D2017%26volume%3D449%26spage%3D11%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander M.  Andrianov</span>, <span class="hlFld-ContribAuthor ">Ivan A.  Kashyn</span>, <span class="hlFld-ContribAuthor ">Alexander V.  Tuzikov</span>. </span><span class="cited-content_cbyCitation_article-title">Computational identification of novel entry inhibitor scaffolds mimicking primary receptor CD4 of HIV-1 gp120. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2017,</strong> <em>23 </em>
                                    (1)
                                     <a href="https://doi.org/10.1007/s00894-016-3189-4" title="DOI URL">https://doi.org/10.1007/s00894-016-3189-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-016-3189-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-016-3189-4%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DComputational%252Bidentification%252Bof%252Bnovel%252Bentry%252Binhibitor%252Bscaffolds%252Bmimicking%252Bprimary%252Breceptor%252BCD4%252Bof%252BHIV-1%252Bgp120%26aulast%3DAndrianov%26aufirst%3DAlexander%2BM.%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kevin K.  Ariën</span>, <span class="hlFld-ContribAuthor ">Françoise  Baleux</span>, <span class="hlFld-ContribAuthor ">Delphine  Desjardins</span>, <span class="hlFld-ContribAuthor ">Françoise  Porrot</span>, <span class="hlFld-ContribAuthor ">Yves-Marie  Coïc</span>, <span class="hlFld-ContribAuthor ">Johan  Michiels</span>, <span class="hlFld-ContribAuthor ">Kawthar  Bouchemal</span>, <span class="hlFld-ContribAuthor ">David  Bonnaffé</span>, <span class="hlFld-ContribAuthor ">Timothée  Bruel</span>, <span class="hlFld-ContribAuthor ">Olivier  Schwartz</span>, <span class="hlFld-ContribAuthor ">Roger  Le Grand</span>, <span class="hlFld-ContribAuthor ">Guido  Vanham</span>, <span class="hlFld-ContribAuthor ">Nathalie  Dereuddre-Bosquet</span>, <span class="hlFld-ContribAuthor ">Hugues  Lortat-Jacob</span>. </span><span class="cited-content_cbyCitation_article-title">CD4-mimetic sulfopeptide conjugates display sub-nanomolar anti-HIV-1 activity and protect macaques against a SHIV162P3 vaginal challenge. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2016,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/srep34829" title="DOI URL">https://doi.org/10.1038/srep34829</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/srep34829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsrep34829%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DCD4-mimetic%252Bsulfopeptide%252Bconjugates%252Bdisplay%252Bsub-nanomolar%252Banti-HIV-1%252Bactivity%252Band%252Bprotect%252Bmacaques%252Bagainst%252Ba%252BSHIV162P3%252Bvaginal%252Bchallenge%26aulast%3DAri%25C3%25ABn%26aufirst%3DKevin%2BK.%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Curreli</span>, <span class="hlFld-ContribAuthor ">Kashfia  Haque</span>, <span class="hlFld-ContribAuthor ">Lihua  Xie</span>, <span class="hlFld-ContribAuthor ">Qian  Qiu</span>, <span class="hlFld-ContribAuthor ">Jinfeng  Xu</span>, <span class="hlFld-ContribAuthor ">Weizhong  Yong</span>, <span class="hlFld-ContribAuthor ">Xiaohe  Tong</span>, <span class="hlFld-ContribAuthor ">Asim K.  Debnath</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, antiviral activity and resistance of a novel small molecule HIV-1 entry inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2015,</strong> <em>23 </em>
                                    (24)
                                     , 7618-7628. <a href="https://doi.org/10.1016/j.bmc.2015.11.006" title="DOI URL">https://doi.org/10.1016/j.bmc.2015.11.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2015.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2015.11.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bantiviral%252Bactivity%252Band%252Bresistance%252Bof%252Ba%252Bnovel%252Bsmall%252Bmolecule%252BHIV-1%252Bentry%252Binhibitor%26aulast%3DCurreli%26aufirst%3DFrancesca%26date%3D2015%26volume%3D23%26issue%3D24%26spage%3D7618%26epage%3D7628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Reda  Rawi</span>, <span class="hlFld-ContribAuthor ">Khalid  Kunji</span>, <span class="hlFld-ContribAuthor ">Abdelali  Haoudi</span>, <span class="hlFld-ContribAuthor ">Halima  Bensmail</span>, . </span><span class="cited-content_cbyCitation_article-title">Coevolution Analysis of HIV-1 Envelope Glycoprotein Complex. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2015,</strong> <em>10 </em>
                                    (11)
                                     , e0143245. <a href="https://doi.org/10.1371/journal.pone.0143245" title="DOI URL">https://doi.org/10.1371/journal.pone.0143245</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0143245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0143245%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DCoevolution%252BAnalysis%252Bof%252BHIV-1%252BEnvelope%252BGlycoprotein%252BComplex%26aulast%3DRawi%26aufirst%3DReda%26date%3D2015%26date%3D2015%26volume%3D10%26issue%3D11%26spage%3De0143245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Priyamvada  Acharya</span>, <span class="hlFld-ContribAuthor ">Sabrina  Lusvarghi</span>, <span class="hlFld-ContribAuthor ">Carole A  Bewley</span>, <span class="hlFld-ContribAuthor ">Peter D  Kwong</span>. </span><span class="cited-content_cbyCitation_article-title">HIV-1 gp120 as a therapeutic target: navigating a moving labyrinth. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2015,</strong> <em>19 </em>
                                    (6)
                                     , 765-783. <a href="https://doi.org/10.1517/14728222.2015.1010513" title="DOI URL">https://doi.org/10.1517/14728222.2015.1010513</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/14728222.2015.1010513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F14728222.2015.1010513%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DHIV-1%252Bgp120%252Bas%252Ba%252Btherapeutic%252Btarget%25253A%252Bnavigating%252Ba%252Bmoving%252Blabyrinth%26aulast%3DAcharya%26aufirst%3DPriyamvada%26date%3D2015%26date%3D2015%26volume%3D19%26issue%3D6%26spage%3D765%26epage%3D783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiří  Černý</span>, <span class="hlFld-ContribAuthor ">Lada  Biedermannová</span>, <span class="hlFld-ContribAuthor ">Pavel  Mikulecký</span>, <span class="hlFld-ContribAuthor ">Jiří  Zahradník</span>, <span class="hlFld-ContribAuthor ">Tatsiana  Charnavets</span>, <span class="hlFld-ContribAuthor ">Peter  Šebo</span>, <span class="hlFld-ContribAuthor ">Bohdan  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Redesigning Protein Cavities as a Strategy for Increasing Affinity in Protein-Protein Interaction: Interferon-
              
                γ
              
              Receptor 1 as a Model. </span><span class="cited-content_cbyCitation_journal-name">BioMed Research International</span><span> <strong>2015,</strong> <em>2015 </em>, 1-12. <a href="https://doi.org/10.1155/2015/716945" title="DOI URL">https://doi.org/10.1155/2015/716945</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2015/716945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2015%2F716945%26sid%3Dliteratum%253Aachs%26jtitle%3DBioMed%2520Research%2520International%26atitle%3DRedesigning%252BProtein%252BCavities%252Bas%252Ba%252BStrategy%252Bfor%252BIncreasing%252BAffinity%252Bin%252BProtein-Protein%252BInteraction%25253A%252BInterferon-%252B%2525CE%2525B3%252BReceptor%252B1%252Bas%252Ba%252BModel%26aulast%3D%25C4%258Cern%25C3%25BD%26aufirst%3DJi%25C5%2599%25C3%25AD%26date%3D2015%26volume%3D2015%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Young Do  Kwon</span>, <span class="hlFld-ContribAuthor ">Judith M.  LaLonde</span>, <span class="hlFld-ContribAuthor ">Yongping  Yang</span>, <span class="hlFld-ContribAuthor ">Mark A.  Elban</span>, <span class="hlFld-ContribAuthor ">Akihiro  Sugawara</span>, <span class="hlFld-ContribAuthor ">Joel R.  Courter</span>, <span class="hlFld-ContribAuthor ">David M.  Jones</span>, <span class="hlFld-ContribAuthor ">Amos B.  Smith</span>, <span class="hlFld-ContribAuthor ">Asim K.  Debnath</span>, <span class="hlFld-ContribAuthor ">Peter D.  Kwong</span>, . </span><span class="cited-content_cbyCitation_article-title">Crystal Structures of HIV-1 gp120 Envelope Glycoprotein in Complex with NBD Analogues That Target the CD4-Binding Site. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (1)
                                     , e85940. <a href="https://doi.org/10.1371/journal.pone.0085940" title="DOI URL">https://doi.org/10.1371/journal.pone.0085940</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0085940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0085940%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DCrystal%252BStructures%252Bof%252BHIV-1%252Bgp120%252BEnvelope%252BGlycoprotein%252Bin%252BComplex%252Bwith%252BNBD%252BAnalogues%252BThat%252BTarget%252Bthe%252BCD4-Binding%252BSite%26aulast%3DKwon%26aufirst%3DYoung%2BDo%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D1%26spage%3De85940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José  das Neves</span>. </span><span class="cited-content_cbyCitation_article-title">Vaginal Delivery of Biopharmaceuticals. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 261-280. <a href="https://doi.org/10.1007/978-1-4614-9524-6_10" title="DOI URL">https://doi.org/10.1007/978-1-4614-9524-6_10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4614-9524-6_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4614-9524-6_10%26sid%3Dliteratum%253Aachs%26atitle%3DVaginal%252BDelivery%252Bof%252BBiopharmaceuticals%26aulast%3Ddas%2BNeves%26aufirst%3DJos%25C3%25A9%26date%3D2014%26date%3D2014%26spage%3D261%26epage%3D280%26pub%3DSpringer%2520US%26atitle%3DMucosal%252BDelivery%252Bof%252BBiopharmaceuticals%26aulast%3Ddas%2BNeves%26aufirst%3DJos%25C3%25A9%26date%3D2014%26volume%3D82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ivelin S.  Georgiev</span>, <span class="hlFld-ContribAuthor ">M.  Gordon Joyce</span>, <span class="hlFld-ContribAuthor ">Tongqing  Zhou</span>, <span class="hlFld-ContribAuthor ">Peter D.  Kwong</span>. </span><span class="cited-content_cbyCitation_article-title">Elicitation of HIV-1-neutralizing antibodies against the CD4-binding site. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in HIV and AIDS</span><span> <strong>2013,</strong> <em>8 </em>
                                    (5)
                                     , 382-392. <a href="https://doi.org/10.1097/COH.0b013e328363a90e" title="DOI URL">https://doi.org/10.1097/COH.0b013e328363a90e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1097/COH.0b013e328363a90e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1097%2FCOH.0b013e328363a90e%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520HIV%2520and%2520AIDS%26atitle%3DElicitation%252Bof%252BHIV-1-neutralizing%252Bantibodies%252Bagainst%252Bthe%252BCD4-binding%252Bsite%26aulast%3DGeorgiev%26aufirst%3DIvelin%2BS.%26date%3D2013%26volume%3D8%26issue%3D5%26spage%3D382%26epage%3D392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0001.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HIV-1 YU2 gp120 binding to sCD4 and CD4 mimetic miniproteins. (A) gp120 binding to the CD4-mimetic miniproteins and to sCD4. gp120 is shown in cartoon representation. CD4 (shown as a transparent yellow surface) binds at the interface of three gp120 domains: inner domain (pale cyan), outer domain (pale green), and bridging sheet (light pink). Phe43<sub>CD4</sub> (yellow stick representation) inserts into a gp120 cavity, known as the Phe43 cavity, formed at the intersection of these domains. The miniCD4s are shown in red cartoon representation with the side chain of position 23 shown as sticks. Residue 23<sub>miniCD4</sub> inserts into the gp120 Phe43 cavity; (B) 90° rotated view of A. For clarity, the CD4 surface is not shown in this figure. (C) Zoomed-in cross-section of gp120 surface showing the Phe43 cavity with the bound inserts: Phe43 for CD4 (yellow, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RZK">1RZK</a>), Phe23 for <b>1</b> (wheat, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I60">2I60</a>), biphenylalanine 23 for <b>14</b> (slate, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I5Y">2I5Y</a>), Phe23 for <b>17</b> (brown, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YYM">1YYM</a>), biphenylalanine 23 for <b>16</b> (marine, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YYL">1YYL</a>). (D) 180° rotated view of C. For clarity, gp120 surface from only one complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I60">2I60</a>) is shown in C and D.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0004.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Unnatural Alkoxy Derivatives of Phenylalanine Protected for Fmoc-Peptide Synthesis<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) ClCOOCH<sub>2</sub>CHCH<sub>2</sub>, 4 N NaOH; (b) Wang resin, diisopropylcarbodiimide, DMAP, NMP; (c) 20% piperidine, DMF; (d) ROH, PPh<sub>3</sub>, DIAD, DCM/THF; (e) Pd(PPh<sub>3</sub>)<sub>4</sub>, Ar, THF/DMSO/HCl/morpholine; (f) Fmoc-Cl, <i>i</i>-Pr<sub>2</sub>NEt, DCM; (g) TFA/DCM.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0005.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Monoprotection of Diol Derivative<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) BnBr, NaH, dry THF.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0006.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Alcohol <b>48</b>′<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) 4-Methylmorpholine, (CH<sub>3</sub>)<sub>2</sub>CHCH<sub>2</sub>OCOCl, DME; (b) NaBH<sub>4</sub>, DME/H<sub>2</sub>O.</p></p></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0007.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the Sulfoxide Fmoc-Protected Amino Acid <b>54</b>′<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) TBSCl, NaH, dry THF; (b) Z-Tyr-OBn, PPh<sub>3</sub>, DEAD, DMF/toluene; (c) Pd black, H<sub>2</sub>, MeOH; (d) Fmoc-<i>O</i>-succinimide, NaHCO<sub>3</sub>, DMF, CH<sub>3</sub>CN, H<sub>2</sub>O; (e) TBHP, thiourea dioxide, DCM.</p></p></figure><figure data-id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0008.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Fmoc-Protected Amino Acid <b>57</b>′<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) C<sub>6</sub>H<sub>11</sub>CHO, NaBH(OAc)<sub>3</sub>, CH<sub>3</sub>CN/AcOH; (b) (Boc)<sub>2</sub>O, <i>i</i>-Pr<sub>2</sub>NEt, THF.</p></p></figure><figure data-id="sch6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0009.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Fmoc-Protected Amino Acid <b>59</b>′<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents: (a) Fmoc-Cl, Na<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O.</p></p></figure><figure data-id="fig2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0002.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. MiniCD4 <b>13</b> binds at the CD4-binding surface and extends into the gp120 Phe43 cavity. (A) gp120 is shown in gray surface representation with the CD4 binding footprint colored yellow. Knottin <b>13</b> is depicted in cartoon representation with the miniprotein colored red except residue 23, which is colored by element type with the nitrogen atoms colored blue and the carbon atoms of one conformation colored orange and the other magenta. (B) A cross-section of gp120 showing the side chain of residue 23 of peptide <b>13</b> penetrating the gp120 Phe43 cavity, packing coefficients, and surface complementarity for each alternate conformation. Packing coefficient (%) = (volume of ligand/volume of binding pocket) × 100. The colored dots span the region of the ligand used for each calculation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/medium/jm-2013-002988_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0003.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. MiniCD4 <b>13</b> binds at the intersection of gp120 inner domain, outer domain, and bridging sheet. (A–C) Zoomed-in views of <b>13</b> bound to the gp120 Phe43 cavity. gp120 is shown in cartoon representation and colored by domain. Inner domain, outer domain, and bridging sheet are colored cyan, green, and pink, respectively. MiniCD4 <b>13</b> is similarly represented as in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>. A gray mesh surrounding the residue depicts a 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map at 2.0σ defining the position of residue 23. In A–C, the cartoon is made transparent to highlight the placement of the residue 23 side chain in the electron density. (D) MiniCD4 <b>13</b> interacting with a conserved gp120 solvent channel. The red spheres are water molecules located in the solvent channel that starts from inside the Phe43 cavity, traverses under the bridging sheet, and emerges at the CD4-induced coreceptor binding site of gp120. (E) LIGPLOT diagram illustrating the environment around <b>13</b> in the gp120 Phe43 cavity in two-dimensional representation and showing the <b>13</b> hydrogen bonding with the conserved gp120 solvent channel. Only one side chain conformation is shown for clarity. (F) 2D structure of the side chain of <b>13</b> with arrows showing the bonds that rotate and result in multiple conformations bound to the gp120 Phe43 cavity. Riding hydrogen atoms used during refinement are shown in white.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-12/jm4002988/production/images/large/jm-2013-002988_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4002988&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i53">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76867" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76867" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Wu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerard, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parolin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruffing, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borsetti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardoso, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desjardin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerard, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span> </span><span class="NLM_article-title">CD4-induced interaction of primary HIV-1 gp120 glycoproteins with the chemokine receptor CCR-5</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">179</span><span class="NLM_x">–</span> <span class="NLM_lpage">183</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=179-183&author=L.+Wuauthor=N.+P.+Gerardauthor=R.+Wyattauthor=H.+Choeauthor=C.+Parolinauthor=N.+Ruffingauthor=A.+Borsettiauthor=A.+A.+Cardosoauthor=E.+Desjardinauthor=W.+Newmanauthor=C.+Gerardauthor=J.+Sodroski&title=CD4-induced+interaction+of+primary+HIV-1+gp120+glycoproteins+with+the+chemokine+receptor+CCR-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DL.%26aulast%3DGerard%26aufirst%3DN.%2BP.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DChoe%26aufirst%3DH.%26aulast%3DParolin%26aufirst%3DC.%26aulast%3DRuffing%26aufirst%3DN.%26aulast%3DBorsetti%26aufirst%3DA.%26aulast%3DCardoso%26aufirst%3DA.%2BA.%26aulast%3DDesjardin%26aufirst%3DE.%26aulast%3DNewman%26aufirst%3DW.%26aulast%3DGerard%26aufirst%3DC.%26aulast%3DSodroski%26aufirst%3DJ.%26atitle%3DCD4-induced%2520interaction%2520of%2520primary%2520HIV-1%2520gp120%2520glycoproteins%2520with%2520the%2520chemokine%2520receptor%2520CCR-5%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D179%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Trkola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dragic, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arthos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binley, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allaway, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng-Mayer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddon, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. P.</span><span> </span><span class="NLM_article-title">CD4-dependent, antibody-sensitive interactions between HIV-1 and its co-receptor CCR-5</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">184</span><span class="NLM_x">–</span> <span class="NLM_lpage">187</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=184-187&author=A.+Trkolaauthor=T.+Dragicauthor=J.+Arthosauthor=J.+M.+Binleyauthor=W.+C.+Olsonauthor=G.+P.+Allawayauthor=C.+Cheng-Mayerauthor=J.+Robinsonauthor=P.+J.+Maddonauthor=J.+P.+Moore&title=CD4-dependent%2C+antibody-sensitive+interactions+between+HIV-1+and+its+co-receptor+CCR-5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrkola%26aufirst%3DA.%26aulast%3DDragic%26aufirst%3DT.%26aulast%3DArthos%26aufirst%3DJ.%26aulast%3DBinley%26aufirst%3DJ.%2BM.%26aulast%3DOlson%26aufirst%3DW.%2BC.%26aulast%3DAllaway%26aufirst%3DG.%2BP.%26aulast%3DCheng-Mayer%26aufirst%3DC.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DMaddon%26aufirst%3DP.%2BJ.%26aulast%3DMoore%26aufirst%3DJ.%2BP.%26atitle%3DCD4-dependent%252C%2520antibody-sensitive%2520interactions%2520between%2520HIV-1%2520and%2520its%2520co-receptor%2520CCR-5%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D184%26epage%3D187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broder, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, E. A.</span><span> </span><span class="NLM_article-title">HIV-1 entry cofactor: functional cDNA cloning of a seven-transmembrane, G protein-coupled receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">872</span><span class="NLM_x">–</span> <span class="NLM_lpage">877</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1996&pages=872-877&author=Y.+Fengauthor=C.+C.+Broderauthor=P.+E.+Kennedyauthor=E.+A.+Berger&title=HIV-1+entry+cofactor%3A+functional+cDNA+cloning+of+a+seven-transmembrane%2C+G+protein-coupled+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DBroder%26aufirst%3DC.%2BC.%26aulast%3DKennedy%26aufirst%3DP.%2BE.%26aulast%3DBerger%26aufirst%3DE.%2BA.%26atitle%3DHIV-1%2520entry%2520cofactor%253A%2520functional%2520cDNA%2520cloning%2520of%2520a%2520seven-transmembrane%252C%2520G%2520protein-coupled%2520receptor%26jtitle%3DScience%26date%3D1996%26volume%3D272%26spage%3D872%26epage%3D877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span> </span><span class="NLM_article-title">The HIV-1 envelope glycoproteins: Fusogens, antigens, and immunogens</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">1884</span><span class="NLM_x">–</span> <span class="NLM_lpage">1888</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=1998&pages=1884-1888&author=R.+Wyattauthor=J.+Sodroski&title=The+HIV-1+envelope+glycoproteins%3A+Fusogens%2C+antigens%2C+and+immunogens"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DSodroski%26aufirst%3DJ.%26atitle%3DThe%2520HIV-1%2520envelope%2520glycoproteins%253A%2520Fusogens%252C%2520antigens%252C%2520and%2520immunogens%26jtitle%3DScience%26date%3D1998%26volume%3D280%26spage%3D1884%26epage%3D1888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Blair, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, P. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meanwell, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, O. B.</span><span> </span><span class="NLM_article-title">HIV-1 entry - an expanding portal for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=183-194&author=W.+S.+Blairauthor=P.+F.+Linauthor=N.+A.+Meanwellauthor=O.+B.+Wallace&title=HIV-1+entry+-+an+expanding+portal+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DW.%2BS.%26aulast%3DLin%26aufirst%3DP.%2BF.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26aulast%3DWallace%26aufirst%3DO.%2BB.%26atitle%3DHIV-1%2520entry%2520-%2520an%2520expanding%2520portal%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2000%26volume%3D5%26spage%3D183%26epage%3D194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Moore, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doms, R. W.</span><span> </span><span class="NLM_article-title">The entry of entry inhibitors: A fusion of science and medicine</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">10598</span><span class="NLM_x">–</span> <span class="NLM_lpage">10602</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=10598-10602&author=J.+P.+Mooreauthor=R.+W.+Doms&title=The+entry+of+entry+inhibitors%3A+A+fusion+of+science+and+medicine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DJ.%2BP.%26aulast%3DDoms%26aufirst%3DR.%2BW.%26atitle%3DThe%2520entry%2520of%2520entry%2520inhibitors%253A%2520A%2520fusion%2520of%2520science%2520and%2520medicine%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2003%26volume%3D100%26spage%3D10598%26epage%3D10602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Vermeire, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schols, D.</span><span> </span><span class="NLM_article-title">Anti-HIV agents targeting the interaction of gp120 with the cellular CD4 receptor</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1199</span><span class="NLM_x">–</span> <span class="NLM_lpage">1212</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=1199-1212&author=K.+Vermeireauthor=D.+Schols&title=Anti-HIV+agents+targeting+the+interaction+of+gp120+with+the+cellular+CD4+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVermeire%26aufirst%3DK.%26aulast%3DSchols%26aufirst%3DD.%26atitle%3DAnti-HIV%2520agents%2520targeting%2520the%2520interaction%2520of%2520gp120%2520with%2520the%2520cellular%2520CD4%2520receptor%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2005%26volume%3D14%26spage%3D1199%26epage%3D1212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Ryser, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fluckiger, R.</span><span> </span><span class="NLM_article-title">Progress in targeting HIV-1 entry</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1085</span><span class="NLM_x">–</span> <span class="NLM_lpage">1094</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=1085-1094&author=H.+J.+Ryserauthor=R.+Fluckiger&title=Progress+in+targeting+HIV-1+entry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRyser%26aufirst%3DH.%2BJ.%26aulast%3DFluckiger%26aufirst%3DR.%26atitle%3DProgress%2520in%2520targeting%2520HIV-1%2520entry%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D1085%26epage%3D1094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Vita, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drakopoulou, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vizzavona, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rochette, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ménez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumestand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ylisastigui, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benjouad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gluckman, J. C.</span><span> </span><span class="NLM_article-title">Rational engineering of a miniprotein that reproduces the core of the CD4 site interacting with HIV-1 envelope glycoprotein</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">13091</span><span class="NLM_x">–</span> <span class="NLM_lpage">13096</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=13091-13096&author=C.+Vitaauthor=E.+Drakopoulouauthor=J.+Vizzavonaauthor=S.+Rochetteauthor=L.+Martinauthor=A.+M%C3%A9nezauthor=C.+Roumestandauthor=Y.+S.+Yangauthor=L.+Ylisastiguiauthor=A.+Benjouadauthor=J.+C.+Gluckman&title=Rational+engineering+of+a+miniprotein+that+reproduces+the+core+of+the+CD4+site+interacting+with+HIV-1+envelope+glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVita%26aufirst%3DC.%26aulast%3DDrakopoulou%26aufirst%3DE.%26aulast%3DVizzavona%26aufirst%3DJ.%26aulast%3DRochette%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DRoumestand%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DY.%2BS.%26aulast%3DYlisastigui%26aufirst%3DL.%26aulast%3DBenjouad%26aufirst%3DA.%26aulast%3DGluckman%26aufirst%3DJ.%2BC.%26atitle%3DRational%2520engineering%2520of%2520a%2520miniprotein%2520that%2520reproduces%2520the%2520core%2520of%2520the%2520CD4%2520site%2520interacting%2520with%2520HIV-1%2520envelope%2520glycoprotein%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D13091%26epage%3D13096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Getz, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daugherty, P. S.</span><span> </span><span class="NLM_article-title">Protease-resistant peptide ligands from a knottin scaffold library</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">837</span><span class="NLM_x">–</span> <span class="NLM_lpage">844</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb200039s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsFWhsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=837-844&author=J.+A.+Getzauthor=J.+J.+Riceauthor=P.+S.+Daugherty&title=Protease-resistant+peptide+ligands+from+a+knottin+scaffold+library"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protease-Resistant Peptide Ligands from a Knottin Scaffold Library</span></div><div class="casAuthors">Getz, Jennifer A.; Rice, Jeffrey J.; Daugherty, Patrick S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">837-844</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptides within the knottin family have been shown to possess inherent stability, making them attractive scaffolds for the development of therapeutic and diagnostic agents.  Given its remarkable stability to proteases, the cyclic peptide kalata B1 was employed as a scaffold to create a large knottin library displayed on the surface of E. coli.  A library exceeding 109 variants was constructed by randomizing seven amino acids within a loop of the kalata B1 scaffold and screened using fluorescence-activated cell sorting to identify peptide ligands specific for the active site of human thrombin.  Refolded thrombin binders exhibited high nanomolar affinities in soln. and slow dissocn. rates and were able to inhibit thrombin's enzymic activity.  Importantly, 80% of a knottin-based thrombin inhibitor remained intact after a 2 h incubation both with trypsin and with chymotrypsin, demonstrating that modifying the kalata B1 sequence did not compromise its stability properties.  In addn., the knottin variant mediated 20-fold enhanced affinity for thrombin, when compared to the same seven residue binding epitope constrained by a single disulfide bond.  Our results indicate that peptide libraries derived from the kalata B1 scaffold can yield high-affinity protein ligands that retain the remarkable protease resistance assocd. with the parent scaffold.  More generally, this strategy may prove useful in the development of stable peptide ligands suitable for in vivo applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqckK3Dh6_BV7Vg90H21EOLACvtfcHk0li_yxcwVjepwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsFWhsb8%253D&md5=cf0a71b43b8fc192e17f05734938bc9a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fcb200039s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb200039s%26sid%3Dliteratum%253Aachs%26aulast%3DGetz%26aufirst%3DJ.%2BA.%26aulast%3DRice%26aufirst%3DJ.%2BJ.%26aulast%3DDaugherty%26aufirst%3DP.%2BS.%26atitle%3DProtease-resistant%2520peptide%2520ligands%2520from%2520a%2520knottin%2520scaffold%2520library%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D837%26epage%3D844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Martins, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tartar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazdunski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borremans, F. A. M.</span><span> </span><span class="NLM_article-title">Solution conformation of leiurotoxin-I (Scyllatoxin) by H-1 nuclear magnetic-resonance - Resonance assignment and secondary structure</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">260</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=260&publication_year=1990&pages=249-253&author=J.+C.+Martinsauthor=W.+G.+Zhangauthor=A.+Tartarauthor=M.+Lazdunskiauthor=F.+A.+M.+Borremans&title=Solution+conformation+of+leiurotoxin-I+%28Scyllatoxin%29+by+H-1+nuclear+magnetic-resonance+-+Resonance+assignment+and+secondary+structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartins%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DW.%2BG.%26aulast%3DTartar%26aufirst%3DA.%26aulast%3DLazdunski%26aufirst%3DM.%26aulast%3DBorremans%26aufirst%3DF.%2BA.%2BM.%26atitle%3DSolution%2520conformation%2520of%2520leiurotoxin-I%2520%2528Scyllatoxin%2529%2520by%2520H-1%2520nuclear%2520magnetic-resonance%2520-%2520Resonance%2520assignment%2520and%2520secondary%2520structure%26jtitle%3DFEBS%2520Lett.%26date%3D1990%26volume%3D260%26spage%3D249%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Drakopoulou, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinn-Justin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guenneugues, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilquin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ménez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span> </span><span class="NLM_article-title">Changing the structural context of a functional beta-hairpin. Synthesis and characterization of a chimera containing the curaremimetic loop of a snake toxin in the scorpion alpha/beta scaffold</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">271</span><span class="NLM_x">, </span> <span class="NLM_fpage">11979</span><span class="NLM_x">–</span> <span class="NLM_lpage">11987</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=11979-11987&author=E.+Drakopoulouauthor=S.+Zinn-Justinauthor=M.+Guenneuguesauthor=B.+Gilquinauthor=A.+M%C3%A9nezauthor=C.+Vita&title=Changing+the+structural+context+of+a+functional+beta-hairpin.+Synthesis+and+characterization+of+a+chimera+containing+the+curaremimetic+loop+of+a+snake+toxin+in+the+scorpion+alpha%2Fbeta+scaffold"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDrakopoulou%26aufirst%3DE.%26aulast%3DZinn-Justin%26aufirst%3DS.%26aulast%3DGuenneugues%26aufirst%3DM.%26aulast%3DGilquin%26aufirst%3DB.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DVita%26aufirst%3DC.%26atitle%3DChanging%2520the%2520structural%2520context%2520of%2520a%2520functional%2520beta-hairpin.%2520Synthesis%2520and%2520characterization%2520of%2520a%2520chimera%2520containing%2520the%2520curaremimetic%2520loop%2520of%2520a%2520snake%2520toxin%2520in%2520the%2520scorpion%2520alpha%252Fbeta%2520scaffold%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D11979%26epage%3D11987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stricher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Misse, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sironi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pugniere, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barthe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prado-Gotor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freulon, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magne, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumestand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ménez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lusso, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veas, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span> </span><span class="NLM_article-title">Rational design of a CD4 mimic that inhibits HIV-1 entry and exposes cryptic neutralization epitopes</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2003&pages=71-76&author=L.+Martinauthor=F.+Stricherauthor=D.+Misseauthor=F.+Sironiauthor=M.+Pugniereauthor=P.+Bartheauthor=R.+Prado-Gotorauthor=I.+Freulonauthor=X.+Magneauthor=C.+Roumestandauthor=A.+M%C3%A9nezauthor=P.+Lussoauthor=F.+Veasauthor=C.+Vita&title=Rational+design+of+a+CD4+mimic+that+inhibits+HIV-1+entry+and+exposes+cryptic+neutralization+epitopes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DStricher%26aufirst%3DF.%26aulast%3DMisse%26aufirst%3DD.%26aulast%3DSironi%26aufirst%3DF.%26aulast%3DPugniere%26aufirst%3DM.%26aulast%3DBarthe%26aufirst%3DP.%26aulast%3DPrado-Gotor%26aufirst%3DR.%26aulast%3DFreulon%26aufirst%3DI.%26aulast%3DMagne%26aufirst%3DX.%26aulast%3DRoumestand%26aufirst%3DC.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DLusso%26aufirst%3DP.%26aulast%3DVeas%26aufirst%3DF.%26aulast%3DVita%26aufirst%3DC.%26atitle%3DRational%2520design%2520of%2520a%2520CD4%2520mimic%2520that%2520inhibits%2520HIV-1%2520entry%2520and%2520exposes%2520cryptic%2520neutralization%2520epitopes%26jtitle%3DNat.%2520Biotechnol.%26date%3D2003%26volume%3D21%26spage%3D71%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyd, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joly, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ménez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulmer, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, I. K.</span><span> </span><span class="NLM_article-title">Development of covalent HIV-1 gp120-CD4 mimic complexes for vaccine application</span> <span class="citation_source-journal">Antiviral Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">P09d10</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=P09d10&issue=Suppl+2&author=L.+Martinauthor=B.+Heydauthor=Y.+Sunauthor=G.+Martinauthor=E.+Kanauthor=P.+Jolyauthor=P.+Kesslerauthor=A.+M%C3%A9nezauthor=J.+B.+Ulmerauthor=C.+Vitaauthor=B.+W.+Barnettauthor=I.+K.+Srivastava&title=Development+of+covalent+HIV-1+gp120-CD4+mimic+complexes+for+vaccine+application"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DHeyd%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DMartin%26aufirst%3DG.%26aulast%3DKan%26aufirst%3DE.%26aulast%3DJoly%26aufirst%3DP.%26aulast%3DKessler%26aufirst%3DP.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DUlmer%26aufirst%3DJ.%2BB.%26aulast%3DVita%26aufirst%3DC.%26aulast%3DBarnett%26aufirst%3DB.%2BW.%26aulast%3DSrivastava%26aufirst%3DI.%2BK.%26atitle%3DDevelopment%2520of%2520covalent%2520HIV-1%2520gp120-CD4%2520mimic%2520complexes%2520for%2520vaccine%2520application%26jtitle%3DAntiviral%2520Ther.%26date%3D2006%26volume%3D11%26issue%3DSuppl%25202%26spage%3DP09d10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Stricher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Descours, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquesnoy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combes, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decker, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lusso, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span> </span><span class="NLM_article-title">Combinatorial optimization of a CD4-mimetic miniprotein and cocrystal structures with HIV-1 gp120 envelope glycoprotein</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">382</span><span class="NLM_x">, </span> <span class="NLM_fpage">510</span><span class="NLM_x">–</span> <span class="NLM_lpage">524</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2008&pages=510-524&author=F.+Stricherauthor=C.+C.+Huangauthor=A.+Descoursauthor=S.+Duquesnoyauthor=O.+Combesauthor=J.+M.+Deckerauthor=Y.+D.+Kwonauthor=P.+Lussoauthor=G.+M.+Shawauthor=C.+Vitaauthor=P.+D.+Kwongauthor=L.+Martin&title=Combinatorial+optimization+of+a+CD4-mimetic+miniprotein+and+cocrystal+structures+with+HIV-1+gp120+envelope+glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStricher%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DDescours%26aufirst%3DA.%26aulast%3DDuquesnoy%26aufirst%3DS.%26aulast%3DCombes%26aufirst%3DO.%26aulast%3DDecker%26aufirst%3DJ.%2BM.%26aulast%3DKwon%26aufirst%3DY.%2BD.%26aulast%3DLusso%26aufirst%3DP.%26aulast%3DShaw%26aufirst%3DG.%2BM.%26aulast%3DVita%26aufirst%3DC.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DMartin%26aufirst%3DL.%26atitle%3DCombinatorial%2520optimization%2520of%2520a%2520CD4-mimetic%2520miniprotein%2520and%2520cocrystal%2520structures%2520with%2520HIV-1%2520gp120%2520envelope%2520glycoprotein%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D382%26spage%3D510%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stricher, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decker, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barthe, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumestand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vita, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Scorpion-toxin mimics of CD4 in complex with human immunodeficiency virus gp120 crystal structures, molecular mimicry, and neutralization breadth</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">768</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=755-768&author=C.+C.+Huangauthor=F.+Stricherauthor=L.+Martinauthor=J.+M.+Deckerauthor=S.+Majeedauthor=P.+Bartheauthor=W.+A.+Hendricksonauthor=J.+Robinsonauthor=C.+Roumestandauthor=J.+Sodroskiauthor=R.+Wyattauthor=G.+M.+Shawauthor=C.+Vitaauthor=P.+D.+Kwong&title=Scorpion-toxin+mimics+of+CD4+in+complex+with+human+immunodeficiency+virus+gp120+crystal+structures%2C+molecular+mimicry%2C+and+neutralization+breadth"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DStricher%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DDecker%26aufirst%3DJ.%2BM.%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DBarthe%26aufirst%3DP.%26aulast%3DHendrickson%26aufirst%3DW.%2BA.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DRoumestand%26aufirst%3DC.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DShaw%26aufirst%3DG.%2BM.%26aulast%3DVita%26aufirst%3DC.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DScorpion-toxin%2520mimics%2520of%2520CD4%2520in%2520complex%2520with%2520human%2520immunodeficiency%2520virus%2520gp120%2520crystal%2520structures%252C%2520molecular%2520mimicry%252C%2520and%2520neutralization%2520breadth%26jtitle%3DStructure%26date%3D2005%26volume%3D13%26spage%3D755%26epage%3D768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Martin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combes, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyd, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geonnotti, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montefiori, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abache, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulmer, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madaoui, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guerois, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span> </span><span class="NLM_article-title">Stabilization of HIV-1 Envelope in the CD4-bound conformation through specific cross-linking of a CD4 mimetic</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">21706</span><span class="NLM_x">–</span> <span class="NLM_lpage">21716</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=21706-21716&author=G.+Martinauthor=B.+Burkeauthor=R.+Thaiauthor=A.+K.+Deyauthor=O.+Combesauthor=B.+Heydauthor=A.+R.+Geonnottiauthor=D.+C.+Montefioriauthor=E.+Kanauthor=Y.+Lianauthor=Y.+D.+Sunauthor=T.+Abacheauthor=J.+B.+Ulmerauthor=H.+Madaouiauthor=R.+Gueroisauthor=S.+W.+Barnettauthor=I.+K.+Srivastavaauthor=P.+Kesslerauthor=L.+Martin&title=Stabilization+of+HIV-1+Envelope+in+the+CD4-bound+conformation+through+specific+cross-linking+of+a+CD4+mimetic"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%26aulast%3DBurke%26aufirst%3DB.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DDey%26aufirst%3DA.%2BK.%26aulast%3DCombes%26aufirst%3DO.%26aulast%3DHeyd%26aufirst%3DB.%26aulast%3DGeonnotti%26aufirst%3DA.%2BR.%26aulast%3DMontefiori%26aufirst%3DD.%2BC.%26aulast%3DKan%26aufirst%3DE.%26aulast%3DLian%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%2BD.%26aulast%3DAbache%26aufirst%3DT.%26aulast%3DUlmer%26aufirst%3DJ.%2BB.%26aulast%3DMadaoui%26aufirst%3DH.%26aulast%3DGuerois%26aufirst%3DR.%26aulast%3DBarnett%26aufirst%3DS.%2BW.%26aulast%3DSrivastava%26aufirst%3DI.%2BK.%26aulast%3DKessler%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DL.%26atitle%3DStabilization%2520of%2520HIV-1%2520Envelope%2520in%2520the%2520CD4-bound%2520conformation%2520through%2520specific%2520cross-linking%2520of%2520a%2520CD4%2520mimetic%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D21706%26epage%3D21716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Dey, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirokman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nandi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartog, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geonnotti, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montefiori, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carfi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srivastava, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, S. W.</span><span> </span><span class="NLM_article-title">Elicitation of neutralizing antibodies directed against CD4-induced epitope(s) using a CD4 mimetic cross-linked to a HIV-1 envelope glycoprotein</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e30233</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e30233&author=A.+K.+Deyauthor=B.+Burkeauthor=Y.+D.+Sunauthor=K.+Sirokmanauthor=A.+Nandiauthor=K.+Hartogauthor=Y.+Lianauthor=A.+R.+Geonnottiauthor=D.+Montefioriauthor=M.+Frantiauthor=G.+Martinauthor=A.+Carfiauthor=P.+Kesslerauthor=L.+Martinauthor=I.+K.+Srivastavaauthor=S.+W.+Barnett&title=Elicitation+of+neutralizing+antibodies+directed+against+CD4-induced+epitope%28s%29+using+a+CD4+mimetic+cross-linked+to+a+HIV-1+envelope+glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDey%26aufirst%3DA.%2BK.%26aulast%3DBurke%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DY.%2BD.%26aulast%3DSirokman%26aufirst%3DK.%26aulast%3DNandi%26aufirst%3DA.%26aulast%3DHartog%26aufirst%3DK.%26aulast%3DLian%26aufirst%3DY.%26aulast%3DGeonnotti%26aufirst%3DA.%2BR.%26aulast%3DMontefiori%26aufirst%3DD.%26aulast%3DFranti%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DG.%26aulast%3DCarfi%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DSrivastava%26aufirst%3DI.%2BK.%26aulast%3DBarnett%26aufirst%3DS.%2BW.%26atitle%3DElicitation%2520of%2520neutralizing%2520antibodies%2520directed%2520against%2520CD4-induced%2520epitope%2528s%2529%2520using%2520a%2520CD4%2520mimetic%2520cross-linked%2520to%2520a%2520HIV-1%2520envelope%2520glycoprotein%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De30233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Van Herrewege, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morellato, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Descours, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aerts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michiels, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyndrickx, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanham, G.</span><span> </span><span class="NLM_article-title">CD4 mimetic miniproteins: potent anti-HIV compounds with promising activity as microbicides</span> <span class="citation_source-journal">J. Antimicrob. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1093%2Fjac%2Fdkn042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=18270220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjs1Ojsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2008&pages=818-826&author=Y.+Van+Herrewegeauthor=L.+Morellatoauthor=A.+Descoursauthor=L.+Aertsauthor=J.+Michielsauthor=L.+Heyndrickxauthor=L.+Martinauthor=G.+Vanham&title=CD4+mimetic+miniproteins%3A+potent+anti-HIV+compounds+with+promising+activity+as+microbicides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">CD4 mimetic miniproteins: potent anti-HIV compounds with promising activity as microbicides</span></div><div class="casAuthors">Van Herrewege, Yven; Morellato, Laurence; Descours, Anne; Aerts, Laetitia; Michiels, Jo; Heyndrickx, Leo; Martin, Loic; Vanham, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">818-826</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives: The antiviral activity of CD4 miniproteins was evaluated as potential HIV microbicides, using relevant in vitro models.  Methods: Compds. were tested in a single-cycle HIV-1 pseudovirus assay and against replication competent HIV-1 in cocultures of monocyte-derived dendritic cells (MO-DC) and CD4+ T cells.  Cytotoxic activity was evaluated in an MTT assay.  Results: Monomeric miniproteins (M47 and M48) showed 50% effective concn. (EC50) values of 79-105 nM against a subtype B, CCR5 coreceptor-using Ba-L pseudovirus.  Higher activity was found for the dimeric miniproteins M48D30, M48D50 and M48D100 (EC50 between 15 and 30 nM), in contrast to the tetrameric miniproteins M48T30, M48T50 and M48T100 (EC50 between 107 and 377 nM).  The hetero-bivalent miniprotein M48-Hep and miniproteins that targeted the Phe-43 cavity on gp120 (M48-U1, M48-U2 and M48-U3) were highly active, with EC50 values as low as 2 nM for M48-U1.  All miniproteins showed high activity against CCR5 or CXCR4 coreceptor-using subtype B and CRF-01.perp.A/E pseudoviruses.  Many early M48-based compds. were much less active against subtype C pseudoviruses, whereas M48-U compds. that targeted the Phe-43 cavity were very active against all pseudoviruses, including subtype C.  In MO-DC/CD4+ T cell cocultures with replication-competent HIV-1 Ba-L, EC50 values ranged between 13 and 1719 nM depending on the miniprotein, with M48-U1, M48-U2 and M48-U3 again being the most potent.  Importantly, the latter compds. completely prevented viral replication by treating the cultures from 2 h before until 24 h after infection, at nontoxic concns. of 66-6564 nM.  Conclusions: These novel CD4 miniproteins might constitute a promising class of HIV microbicides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsNtvf_idCdrVg90H21EOLACvtfcHk0liHZvGcpKbVGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjs1Ojsrg%253D&md5=6bc1668700a84fee7b137398c91d68a8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkn042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkn042%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BHerrewege%26aufirst%3DY.%26aulast%3DMorellato%26aufirst%3DL.%26aulast%3DDescours%26aufirst%3DA.%26aulast%3DAerts%26aufirst%3DL.%26aulast%3DMichiels%26aufirst%3DJ.%26aulast%3DHeyndrickx%26aufirst%3DL.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DVanham%26aufirst%3DG.%26atitle%3DCD4%2520mimetic%2520miniproteins%253A%2520potent%2520anti-HIV%2520compounds%2520with%2520promising%2520activity%2520as%2520microbicides%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2008%26volume%3D61%26spage%3D818%26epage%3D826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Dereuddre-Bosquet, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morellato-Castillo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brouwers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Augustijns, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouchemal, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponchel, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, O. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrera, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stefanidou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shattock, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heyndrickx, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Grand, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L.</span><span> </span><span class="NLM_article-title">MiniCD4 microbicide prevents HIV infection of human mucosal explants and vaginal transmission of SHIV(162P3) in Cynomolgus macaques</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003071</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=e1003071&author=N.+Dereuddre-Bosquetauthor=L.+Morellato-Castilloauthor=J.+Brouwersauthor=P.+Augustijnsauthor=K.+Bouchemalauthor=G.+Ponchelauthor=O.+H.+Ramosauthor=C.+Herreraauthor=M.+Stefanidouauthor=R.+Shattockauthor=L.+Heyndrickxauthor=G.+Vanhamauthor=P.+Kesslerauthor=R.+Le+Grandauthor=L.+Martin&title=MiniCD4+microbicide+prevents+HIV+infection+of+human+mucosal+explants+and+vaginal+transmission+of+SHIV%28162P3%29+in+Cynomolgus+macaques"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDereuddre-Bosquet%26aufirst%3DN.%26aulast%3DMorellato-Castillo%26aufirst%3DL.%26aulast%3DBrouwers%26aufirst%3DJ.%26aulast%3DAugustijns%26aufirst%3DP.%26aulast%3DBouchemal%26aufirst%3DK.%26aulast%3DPonchel%26aufirst%3DG.%26aulast%3DRamos%26aufirst%3DO.%2BH.%26aulast%3DHerrera%26aufirst%3DC.%26aulast%3DStefanidou%26aufirst%3DM.%26aulast%3DShattock%26aufirst%3DR.%26aulast%3DHeyndrickx%26aufirst%3DL.%26aulast%3DVanham%26aufirst%3DG.%26aulast%3DKessler%26aufirst%3DP.%26aulast%3DLe%2BGrand%26aufirst%3DR.%26aulast%3DMartin%26aufirst%3DL.%26atitle%3DMiniCD4%2520microbicide%2520prevents%2520HIV%2520infection%2520of%2520human%2520mucosal%2520explants%2520and%2520vaginal%2520transmission%2520of%2520SHIV%2528162P3%2529%2520in%2520Cynomolgus%2520macaques%26jtitle%3DPLoS%2520Pathog.%26date%3D2012%26volume%3D8%26spage%3De1003071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Acharya, P.; Luongo, T.; Louder, M. K.; McKee, K.; Yang, Y.; Kwon, Y. D.; Mascola, J. R.; Kessler, P.; Martin, L.; Kwong, P. D.</span><span> </span><span class="NLM_article-title">Structural basis for highly effective HIV-1 neutralization by CD4-mimetic miniproteins revealed by 1.5 Å co-crystal structure of gp120 and M48U1</span>,  <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1016/j.str.2013.04.015</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1016%2Fj.str.2013.04.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=P.+Acharya&author=T.+Luongo&author=M.+K.+Louder&author=K.+McKee&author=Y.+Yang&author=Y.+D.+Kwon&author=J.+R.+Mascola&author=P.+Kessler&author=L.+Martin&author=P.+D.+Kwong&title=Structural+basis+for+highly+effective+HIV-1+neutralization+by+CD4-mimetic+miniproteins+revealed+by+1.5+%C3%85+co-crystal+structure+of+gp120+and+M48U1&doi=10.1016%2Fj.str.2013.04.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DAcharya%26aufirst%3DP.%26atitle%3DStructural%2520basis%2520for%2520highly%2520effective%2520HIV-1%2520neutralization%2520by%2520CD4-mimetic%2520miniproteins%2520revealed%2520by%25201.5%2520%25C3%2585%2520co-crystal%2520structure%2520of%2520gp120%2520and%2520M48U1%26jtitle%3DStructure%26date%3D2013%26doi%3D10.1016%2Fj.str.2013.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Morley, A. D.</span><span> </span><span class="NLM_article-title">Optimisation and synthesis of libraries derived from phenolic amino acid scaffolds</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">7405</span><span class="NLM_x">–</span> <span class="NLM_lpage">7408</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2000&pages=7405-7408&author=A.+D.+Morley&title=Optimisation+and+synthesis+of+libraries+derived+from+phenolic+amino+acid+scaffolds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorley%26aufirst%3DA.%2BD.%26atitle%3DOptimisation%2520and%2520synthesis%2520of%2520libraries%2520derived%2520from%2520phenolic%2520amino%2520acid%2520scaffolds%26jtitle%3DTetrahedron%2520Lett.%26date%3D2000%26volume%3D41%26spage%3D7405%26epage%3D7408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Morley, A. D.</span><span> </span><span class="NLM_article-title">Allyloxycarbonyl - a useful protecting group for phenolic amino acids and applications on solid support</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">7401</span><span class="NLM_x">–</span> <span class="NLM_lpage">7404</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2000&pages=7401-7404&author=A.+D.+Morley&title=Allyloxycarbonyl+-+a+useful+protecting+group+for+phenolic+amino+acids+and+applications+on+solid+support"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorley%26aufirst%3DA.%2BD.%26atitle%3DAllyloxycarbonyl%2520-%2520a%2520useful%2520protecting%2520group%2520for%2520phenolic%2520amino%2520acids%2520and%2520applications%2520on%2520solid%2520support%26jtitle%3DTetrahedron%2520Lett.%26date%3D2000%26volume%3D41%26spage%3D7401%26epage%3D7404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Kwon, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finzi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dogo-Isonagie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vicic, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debnath, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bewley, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascola, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Unliganded HIV-1 gp120 core structures assume the CD4-bound conformation with regulation by quaternary interactions and variable loops</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5663</span><span class="NLM_x">–</span> <span class="NLM_lpage">5668</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1073%2Fpnas.1112391109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=22451932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVOisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5663-5668&author=Y.+D.+Kwonauthor=A.+Finziauthor=X.+Wuauthor=C.+Dogo-Isonagieauthor=L.+K.+Leeauthor=L.+R.+Mooreauthor=S.+D.+Schmidtauthor=J.+Stuckeyauthor=Y.+Yangauthor=T.+Zhouauthor=J.+Zhuauthor=D.+A.+Vicicauthor=A.+K.+Debnathauthor=L.+Shapiroauthor=C.+A.+Bewleyauthor=J.+R.+Mascolaauthor=J.+G.+Sodroskiauthor=P.+D.+Kwong&title=Unliganded+HIV-1+gp120+core+structures+assume+the+CD4-bound+conformation+with+regulation+by+quaternary+interactions+and+variable+loops"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Unliganded HIV-1 gp120 core structures assume the CD4-bound conformation with regulation by quaternary interactions and variable loops</span></div><div class="casAuthors">Kwon, Young Do; Finzi, Andres; Wu, Xueling; Dogo-Isonagie, Cajetan; Lee, Lawrence K.; Moore, Lucas R.; Schmidt, Stephen D.; Stuckey, Jonathan; Yang, Yongping; Zhou, Tongqing; Zhu, Jiang; Vicic, David A.; Debnath, Asim K.; Shapiro, Lawrence; Bewley, Carole A.; Mascola, John R.; Sodroski, Joseph G.; Kwong, Peter D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5663-5668, S5663/1-S5663/33</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The HIV-1 envelope (Env) spike (gp1203/gp413) undergoes considerable structural rearrangements to mediate virus entry into cells and to evade the host immune response.  Engagement of CD4, the primary human receptor, fixes a particular conformation and primes Env for entry.  The CD4-bound state, however, is prone to spontaneous inactivation and susceptible to antibody neutralization.  How does unliganded HIV-1 maintain CD4-binding capacity and regulate transitions to the CD4-bound state.  To define this mechanistically, we detd. crystal structures of unliganded core gp12O from HIV-1 clades B, C, and E.  Notably, all of these unliganded HIV-1 structures resembled the CD4-bound state.  Conformational fixation with ligand selection and thermodn. anal. of full-length and core gp12O interactions revealed that the tendency of HIV-1 gp12O to adopt the CD4-bound conformation was restrained by the V1 N2- and V3-variable loops.  In parallel, we detd. the structure of core gp120 in complex with the small mol., NBD-556, which specifically recognizes the CD4-bound conformation of gp120.  Neutralization by NBD-556 indicated that Env spikes on primary isolates rarely assume the CD4-bound conformation spontaneously, although they could do so when quaternary restraints were loosened.  Together, the results suggest that the CD4-bound conformation represents a "ground state" for the gp12O core, with variable loop and quaternary interactions restraining unliganded gp12O from "snapping" into this conformation.  A mechanism of control involving deformations in unliganded structure from a functionally crit. state (e.g., the CD4-bound state) provides advantages in terms of HIV-1 Env structural diversity and resistance to antibodies and inhibitors, while maintaining elements essential for entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS3b-Y8pziGrVg90H21EOLACvtfcHk0lik4D-d-tQ0GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVOisLc%253D&md5=2ad1a0459b9ede5cd0d3db0116fd7a0b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1112391109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1112391109%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DY.%2BD.%26aulast%3DFinzi%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DDogo-Isonagie%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DL.%2BK.%26aulast%3DMoore%26aufirst%3DL.%2BR.%26aulast%3DSchmidt%26aufirst%3DS.%2BD.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DVicic%26aufirst%3DD.%2BA.%26aulast%3DDebnath%26aufirst%3DA.%2BK.%26aulast%3DShapiro%26aufirst%3DL.%26aulast%3DBewley%26aufirst%3DC.%2BA.%26aulast%3DMascola%26aufirst%3DJ.%2BR.%26aulast%3DSodroski%26aufirst%3DJ.%2BG.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DUnliganded%2520HIV-1%2520gp120%2520core%2520structures%2520assume%2520the%2520CD4-bound%2520conformation%2520with%2520regulation%2520by%2520quaternary%2520interactions%2520and%2520variable%2520loops%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D5663%26epage%3D5668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Cromwell, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bystrom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eftink, M. R.</span><span> </span><span class="NLM_article-title">Cyclodextrin adamantanecarboxylate inclusion complexes - Studies of the variation in cavity size</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">89</span><span class="NLM_x">, </span> <span class="NLM_fpage">326</span><span class="NLM_x">–</span> <span class="NLM_lpage">332</span></span><div class="citationLinks">[<a href="/doi/10.1021/j100248a029" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1985&pages=326-332&author=W.+C.+Cromwellauthor=K.+Bystromauthor=M.+R.+Eftink&title=Cyclodextrin+adamantanecarboxylate+inclusion+complexes+-+Studies+of+the+variation+in+cavity+size"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fj100248a029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100248a029%26sid%3Dliteratum%253Aachs%26aulast%3DCromwell%26aufirst%3DW.%2BC.%26aulast%3DBystrom%26aufirst%3DK.%26aulast%3DEftink%26aufirst%3DM.%2BR.%26atitle%3DCyclodextrin%2520adamantanecarboxylate%2520inclusion%2520complexes%2520-%2520Studies%2520of%2520the%2520variation%2520in%2520cavity%2520size%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1985%26volume%3D89%26spage%3D326%26epage%3D332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Mecozzi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebek, J.</span><span> </span><span class="NLM_article-title">The 55% solution: A formula for molecular recognition in the liquid state</span> <span class="citation_source-journal">Chem.—Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1016</span><span class="NLM_x">–</span> <span class="NLM_lpage">1022</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1002%2F%28SICI%291521-3765%2819980615%294%3A6%3C1016%3A%3AAID-CHEM1016%3E3.0.CO%3B2-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADyaK1cXktV2gtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1998&pages=1016-1022&author=S.+Mecozziauthor=J.+Rebek&title=The+55%25+solution%3A+A+formula+for+molecular+recognition+in+the+liquid+state"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The 55% solution: a formula for molecular recognition in the liquid state</span></div><div class="casAuthors">Mecozzi, Sandro; Rebek, Julius, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1016-1022</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Evidence is presented that mol. recognition through encapsulation processes is largely detd. by the vols. of the guest and host.  Binding of mols. of suitable dimensions in the internal cavity of a mol. receptor in soln. can be expected when the packing coeff., the ratio of the guest vol. to the host vol., is in the range of 0.55 ± 0.09.  Larger packing coeffs., up to 0.70, can be reached if the complex is stabilized by strong intermol. forces such as H bonds.  These considerations also apply to situations in which >1 mol. is encapsulated.  Org. liqs. are generally characterized by the same packing factors as encapsulation complexes, and the short-range structure of liqs. and the complexes resulting from encapsulation are 2 aspects of the same phenomenon.  Similar vol. considerations are expected to apply to the binding of substrates in biol. receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDT6U43hO-XLVg90H21EOLACvtfcHk0lik4D-d-tQ0GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXktV2gtLs%253D&md5=c10f319493dc1ce518b3967eacb1dbf3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3765%2819980615%294%3A6%3C1016%3A%3AAID-CHEM1016%3E3.0.CO%3B2-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3765%252819980615%25294%253A6%253C1016%253A%253AAID-CHEM1016%253E3.0.CO%253B2-B%26sid%3Dliteratum%253Aachs%26aulast%3DMecozzi%26aufirst%3DS.%26aulast%3DRebek%26aufirst%3DJ.%26atitle%3DThe%252055%2525%2520solution%253A%2520A%2520formula%2520for%2520molecular%2520recognition%2520in%2520the%2520liquid%2520state%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D1998%26volume%3D4%26spage%3D1016%26epage%3D1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Zürcher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Structure-based drug design: exploring the proper filling of apolar pockets at enzyme active sites</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">4345</span><span class="NLM_x">–</span> <span class="NLM_lpage">4361</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo800527n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=4345-4361&author=M.+Z%C3%BCrcherauthor=F.+Diederich&title=Structure-based+drug+design%3A+exploring+the+proper+filling+of+apolar+pockets+at+enzyme+active+sites"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjo800527n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo800527n%26sid%3Dliteratum%253Aachs%26aulast%3DZ%25C3%25BCrcher%26aufirst%3DM.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DStructure-based%2520drug%2520design%253A%2520exploring%2520the%2520proper%2520filling%2520of%2520apolar%2520pockets%2520at%2520enzyme%2520active%2520sites%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26spage%3D4345%26epage%3D4361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Kawasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chufan, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafont, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hidaka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiso, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mario Amzel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freire, E.</span><span> </span><span class="NLM_article-title">How much binding affinity can be gained by filling a cavity?</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">143</span><span class="NLM_x">–</span> <span class="NLM_lpage">151</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1111%2Fj.1747-0285.2009.00921.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=20028396" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVSgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2010&pages=143-151&author=Y.+Kawasakiauthor=E.+E.+Chufanauthor=V.+Lafontauthor=K.+Hidakaauthor=Y.+Kisoauthor=L.+Mario+Amzelauthor=E.+Freire&title=How+much+binding+affinity+can+be+gained+by+filling+a+cavity%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">How much binding affinity can be gained by filling a cavity?</span></div><div class="casAuthors">Kawasaki, Yuko; Chufan, Eduardo E.; Lafont, Virginie; Hidaka, Koushi; Kiso, Yoshiaki; Amzel, L. Mario; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-151</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Binding affinity optimization is crit. during drug development.  Here, we evaluate the thermodn. consequences of filling a binding cavity with functionalities of increasing van der Waals radii (-H, -F, -Cl, and CH3) that improve the geometric fit without participating in hydrogen bonding or other specific interactions.  We observe a binding affinity increase of two orders of magnitude.  There appears to be three phases in the process.  The first phase is assocd. with the formation of stable van der Waals interactions.  This phase is characterized by a gain in binding enthalpy and a loss in binding entropy, attributed to a loss of conformational degrees of freedom.  For the specific case presented in this article, the enthalpy gain amts. to -1.5 kcal/mol while the entropic losses amt. to +0.9 kcal/mol resulting in a net 3.5-fold affinity gain.  The second phase is characterized by simultaneous enthalpic and entropic gains.  This phase improves the binding affinity 25-fold.  The third phase represents the collapse of the trend and is triggered by the introduction of chem. functionalities larger than the binding cavity itself [CH(CH3)2].  It is characterized by large enthalpy and affinity losses.  The thermodn. signatures assocd. with each phase provide guidelines for lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJLOlqtVTY2rVg90H21EOLACvtfcHk0lhufScvRfK6aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVSgsg%253D%253D&md5=cf92f069de6def681083f12554ec9bad</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2009.00921.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2009.00921.x%26sid%3Dliteratum%253Aachs%26aulast%3DKawasaki%26aufirst%3DY.%26aulast%3DChufan%26aufirst%3DE.%2BE.%26aulast%3DLafont%26aufirst%3DV.%26aulast%3DHidaka%26aufirst%3DK.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DMario%2BAmzel%26aufirst%3DL.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DHow%2520much%2520binding%2520affinity%2520can%2520be%2520gained%2520by%2520filling%2520a%2520cavity%253F%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2010%26volume%3D75%26spage%3D143%26epage%3D151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, W. A.</span><span> </span><span class="NLM_article-title">Structure of an HIV gp120 envelope glycoprotein in complex with the CD4 receptor and a neutralizing human antibody</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">393</span><span class="NLM_x">, </span> <span class="NLM_fpage">648</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1038%2F31405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=9641677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADyaK1cXktVynsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=393&publication_year=1998&pages=648-659&author=P.+D.+Kwongauthor=R.+Wyattauthor=J.+Robinsonauthor=R.+W.+Sweetauthor=J.+Sodroskiauthor=W.+A.+Hendrickson&title=Structure+of+an+HIV+gp120+envelope+glycoprotein+in+complex+with+the+CD4+receptor+and+a+neutralizing+human+antibody"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of an HIV gp120 envelope glycoprotein in complex with the CD4 receptor and a neutralizing human antibody</span></div><div class="casAuthors">Kwong, Peter D.; Wyatt, Richard; Robinson, James; Sweet, Raymond W.; Sodroski, Joseph; Hendrickson, Wayne A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">393</span>
        (<span class="NLM_cas:issue">6686</span>),
    <span class="NLM_cas:pages">648-659</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The entry of human immunodeficiency virus (HIV) into cells requires the sequential interaction of the viral exterior envelope glycoprotein, gp120, with the CD4 glycoprotein and chemokine receptor on the cell surface.  These interactions initiate a fusion of the viral and cellular membranes.  Although gp120 can elicit virus-neutralizing antibodies, HIV eludes the immune system.  The authors have solved the x-ray crystal structure at 2.5 Å resoln. of an HIV-1 gp120 core complexed with a two-domain fragment of human CD4 and an antigen-binding fragment of a neutralizing antibody that blocks chemokine-receptor binding.  The structure reveals a cavity-laden CD4-gp120 interface, a conserved binding site for the chemokine receptor, evidence for a conformational change upon CD4 binding, the nature of a CD4-induced antibody epitope, and specific mechanisms for immune evasion.  The results provide a framework for understanding the complex biol. of HIV entry into cells and should guide efforts to intervene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5NOtfNxdMJ7Vg90H21EOLACvtfcHk0lhufScvRfK6aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXktVynsrs%253D&md5=5ce7f5923ab2e9c8c37f940ac71429ed</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2F31405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F31405%26sid%3Dliteratum%253Aachs%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DSweet%26aufirst%3DR.%2BW.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DHendrickson%26aufirst%3DW.%2BA.%26atitle%3DStructure%2520of%2520an%2520HIV%2520gp120%2520envelope%2520glycoprotein%2520in%2520complex%2520with%2520the%2520CD4%2520receptor%2520and%2520a%2520neutralizing%2520human%2520antibody%26jtitle%3DNature%26date%3D1998%26volume%3D393%26spage%3D648%26epage%3D659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, W. A.</span><span> </span><span class="NLM_article-title">Structures of HIV-1 gp120 envelope glycoproteins from laboratory-adapted and primary isolates</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1329</span><span class="NLM_x">–</span> <span class="NLM_lpage">1339</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2000&pages=1329-1339&author=P.+D.+Kwongauthor=R.+Wyattauthor=S.+Majeedauthor=J.+Robinsonauthor=R.+W.+Sweetauthor=J.+Sodroskiauthor=W.+A.+Hendrickson&title=Structures+of+HIV-1+gp120+envelope+glycoproteins+from+laboratory-adapted+and+primary+isolates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DSweet%26aufirst%3DR.%2BW.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DHendrickson%26aufirst%3DW.%2BA.%26atitle%3DStructures%2520of%2520HIV-1%2520gp120%2520envelope%2520glycoproteins%2520from%2520laboratory-adapted%2520and%2520primary%2520isolates%26jtitle%3DStructure%26date%3D2000%26volume%3D8%26spage%3D1329%26epage%3D1339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montabana, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanfield, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimitrov, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korber, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Structure of a V3-containing HIV-1 gp120 core</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">310</span><span class="NLM_x">, </span> <span class="NLM_fpage">1025</span><span class="NLM_x">–</span> <span class="NLM_lpage">1028</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=310&publication_year=2005&pages=1025-1028&author=C.+C.+Huangauthor=M.+Tangauthor=M.+Y.+Zhangauthor=S.+Majeedauthor=E.+Montabanaauthor=R.+L.+Stanfieldauthor=D.+S.+Dimitrovauthor=B.+Korberauthor=J.+Sodroskiauthor=I.+A.+Wilsonauthor=R.+Wyattauthor=P.+D.+Kwong&title=Structure+of+a+V3-containing+HIV-1+gp120+core"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DM.%2BY.%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DMontabana%26aufirst%3DE.%26aulast%3DStanfield%26aufirst%3DR.%2BL.%26aulast%3DDimitrov%26aufirst%3DD.%2BS.%26aulast%3DKorber%26aufirst%3DB.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DStructure%2520of%2520a%2520V3-containing%2520HIV-1%2520gp120%2520core%26jtitle%3DScience%26date%3D2005%26volume%3D310%26spage%3D1025%26epage%3D1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Pancera, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ban, Y. E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schief, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Structure of HIV-1 gp120 with gp41-interactive region reveals layered envelope architecture and basis of conformational mobility</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">1166</span><span class="NLM_x">–</span> <span class="NLM_lpage">1171</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=1166-1171&author=M.+Panceraauthor=S.+Majeedauthor=Y.+E.+A.+Banauthor=L.+Chenauthor=C.+C.+Huangauthor=L.+Kongauthor=Y.+D.+Kwonauthor=J.+Stuckeyauthor=T.+Q.+Zhouauthor=J.+E.+Robinsonauthor=W.+R.+Schiefauthor=J.+Sodroskiauthor=R.+Wyattauthor=P.+D.+Kwong&title=Structure+of+HIV-1+gp120+with+gp41-interactive+region+reveals+layered+envelope+architecture+and+basis+of+conformational+mobility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPancera%26aufirst%3DM.%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DBan%26aufirst%3DY.%2BE.%2BA.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DKong%26aufirst%3DL.%26aulast%3DKwon%26aufirst%3DY.%2BD.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DT.%2BQ.%26aulast%3DRobinson%26aufirst%3DJ.%2BE.%26aulast%3DSchief%26aufirst%3DW.%2BR.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DStructure%2520of%2520HIV-1%2520gp120%2520with%2520gp41-interactive%2520region%2520reveals%2520layered%2520envelope%2520architecture%2520and%2520basis%2520of%2520conformational%2520mobility%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D1166%26epage%3D1171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Diskin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcovecchio, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjorkman, P. J.</span><span> </span><span class="NLM_article-title">Structure of a clade C HIV-1 gp120 bound to CD4 and CD4-induced antibody reveals anti-CD4 polyreactivity</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">608</span><span class="NLM_x">–</span> <span class="NLM_lpage">613</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1038%2Fnsmb.1796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=20357769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVGjtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=608-613&author=R.+Diskinauthor=P.+M.+Marcovecchioauthor=P.+J.+Bjorkman&title=Structure+of+a+clade+C+HIV-1+gp120+bound+to+CD4+and+CD4-induced+antibody+reveals+anti-CD4+polyreactivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a clade C HIV-1 gp120 bound to CD4 and CD4-induced antibody reveals anti-CD4 polyreactivity</span></div><div class="casAuthors">Diskin, Ron; Marcovecchio, Paola M.; Bjorkman, Pamela J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">608-613</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Strategies to combat HIV-1 require structural knowledge of envelope proteins from viruses in HIV-1 clade C, the most rapidly spreading subtype in the world.  We present a crystal structure contg. a clade C gp120 envelope.  The structure, a complex between gp120, the host receptor CD4 and the CD4-induced antibody 21c, reveals that the 21c epitope involves contacts with gp120, a nonself antigen, and with CD4, an autoantigen.  Binding studies using wild-type and mutant CD4 show that 21c Fab binds CD4 in the absence of gp120, and that binding of 21c to clade C and HIV-2 gp120s requires the crystallog. obsd. 21c-CD4 interaction.  Addnl. binding data suggest a role for the gp120 V1V2 loop in creating a high-affinity, but slow-forming, epitope for 21c after CD4 binds.  These results contribute to a mol. understanding of CD4-induced antibodies and provide the first visualization to our knowledge of a potentially autoreactive antibody Fab complexed with both self and nonself antigens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouhXh7BjzYVLVg90H21EOLACvtfcHk0lgzQnLVE2NjSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVGjtLk%253D&md5=fa3447c62732cab7bef67312f69836b9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1796%26sid%3Dliteratum%253Aachs%26aulast%3DDiskin%26aufirst%3DR.%26aulast%3DMarcovecchio%26aufirst%3DP.%2BM.%26aulast%3DBjorkman%26aufirst%3DP.%2BJ.%26atitle%3DStructure%2520of%2520a%2520clade%2520C%2520HIV-1%2520gp120%2520bound%2520to%2520CD4%2520and%2520CD4-induced%2520antibody%2520reveals%2520anti-CD4%2520polyreactivity%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D608%26epage%3D613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Wu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiev, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longo, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Louder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Dell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perfetto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonsignori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crump, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapiga, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sam, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burton, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koff, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doria-Rose, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connors, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullikin, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nabel, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roederer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascola, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Progra, N. C. S.</span><span> </span><span class="NLM_article-title">Focused evolution of HIV-1 neutralizing antibodies revealed by structures and deep sequencing</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">333</span><span class="NLM_x">, </span> <span class="NLM_fpage">1593</span><span class="NLM_x">–</span> <span class="NLM_lpage">1602</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=333&publication_year=2011&pages=1593-1602&author=X.+L.+Wuauthor=T.+Q.+Zhouauthor=J.+Zhuauthor=B.+S.+Zhangauthor=I.+Georgievauthor=C.+Wangauthor=X.+J.+Chenauthor=N.+S.+Longoauthor=M.+Louderauthor=K.+McKeeauthor=S.+O%E2%80%99Dellauthor=S.+Perfettoauthor=S.+D.+Schmidtauthor=W.+Shiauthor=L.+Wuauthor=Y.+P.+Yangauthor=Z.+Y.+Yangauthor=Z.+J.+Yangauthor=Z.+H.+Zhangauthor=M.+Bonsignoriauthor=J.+A.+Crumpauthor=S.+H.+Kapigaauthor=N.+E.+Samauthor=B.+F.+Haynesauthor=M.+Simekauthor=D.+R.+Burtonauthor=W.+C.+Koffauthor=N.+A.+Doria-Roseauthor=M.+Connorsauthor=J.+C.+Mullikinauthor=G.+J.+Nabelauthor=M.+Roedererauthor=L.+Shapiroauthor=P.+D.+Kwongauthor=J.+R.+Mascolaauthor=N.+C.+S.+Progra&title=Focused+evolution+of+HIV-1+neutralizing+antibodies+revealed+by+structures+and+deep+sequencing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DX.%2BL.%26aulast%3DZhou%26aufirst%3DT.%2BQ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%2BS.%26aulast%3DGeorgiev%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DX.%2BJ.%26aulast%3DLongo%26aufirst%3DN.%2BS.%26aulast%3DLouder%26aufirst%3DM.%26aulast%3DMcKee%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Dell%26aufirst%3DS.%26aulast%3DPerfetto%26aufirst%3DS.%26aulast%3DSchmidt%26aufirst%3DS.%2BD.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DY.%2BP.%26aulast%3DYang%26aufirst%3DZ.%2BY.%26aulast%3DYang%26aufirst%3DZ.%2BJ.%26aulast%3DZhang%26aufirst%3DZ.%2BH.%26aulast%3DBonsignori%26aufirst%3DM.%26aulast%3DCrump%26aufirst%3DJ.%2BA.%26aulast%3DKapiga%26aufirst%3DS.%2BH.%26aulast%3DSam%26aufirst%3DN.%2BE.%26aulast%3DHaynes%26aufirst%3DB.%2BF.%26aulast%3DSimek%26aufirst%3DM.%26aulast%3DBurton%26aufirst%3DD.%2BR.%26aulast%3DKoff%26aufirst%3DW.%2BC.%26aulast%3DDoria-Rose%26aufirst%3DN.%2BA.%26aulast%3DConnors%26aufirst%3DM.%26aulast%3DMullikin%26aufirst%3DJ.%2BC.%26aulast%3DNabel%26aufirst%3DG.%2BJ.%26aulast%3DRoederer%26aufirst%3DM.%26aulast%3DShapiro%26aufirst%3DL.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DMascola%26aufirst%3DJ.%2BR.%26aulast%3DProgra%26aufirst%3DN.%2BC.%2BS.%26atitle%3DFocused%2520evolution%2520of%2520HIV-1%2520neutralizing%2520antibodies%2520revealed%2520by%2520structures%2520and%2520deep%2520sequencing%26jtitle%3DScience%26date%3D2011%26volume%3D333%26spage%3D1593%26epage%3D1602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">LaLonde, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courter, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soeta, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Princiotto, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freire, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascola, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sodroski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madani, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. B.</span><span> </span><span class="NLM_article-title">Structure-based design, synthesis, and characterization of dual hotspot small-molecule HIV-1 entry inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4382</span><span class="NLM_x">–</span> <span class="NLM_lpage">4396</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300265j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4382-4396&author=J.+M.+LaLondeauthor=Y.+D.+Kwonauthor=D.+M.+Jonesauthor=A.+W.+Sunauthor=J.+R.+Courterauthor=T.+Soetaauthor=T.+Kobayashiauthor=A.+M.+Princiottoauthor=X.+L.+Wuauthor=A.+Schonauthor=E.+Freireauthor=P.+D.+Kwongauthor=J.+R.+Mascolaauthor=J.+Sodroskiauthor=N.+Madaniauthor=A.+B.+Smith&title=Structure-based+design%2C+synthesis%2C+and+characterization+of+dual+hotspot+small-molecule+HIV-1+entry+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm300265j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300265j%26sid%3Dliteratum%253Aachs%26aulast%3DLaLonde%26aufirst%3DJ.%2BM.%26aulast%3DKwon%26aufirst%3DY.%2BD.%26aulast%3DJones%26aufirst%3DD.%2BM.%26aulast%3DSun%26aufirst%3DA.%2BW.%26aulast%3DCourter%26aufirst%3DJ.%2BR.%26aulast%3DSoeta%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DPrinciotto%26aufirst%3DA.%2BM.%26aulast%3DWu%26aufirst%3DX.%2BL.%26aulast%3DSchon%26aufirst%3DA.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DMascola%26aufirst%3DJ.%2BR.%26aulast%3DSodroski%26aufirst%3DJ.%26aulast%3DMadani%26aufirst%3DN.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26atitle%3DStructure-based%2520design%252C%2520synthesis%252C%2520and%2520characterization%2520of%2520dual%2520hotspot%2520small-molecule%2520HIV-1%2520entry%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4382%26epage%3D4396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Xie, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savinov, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. B.</span><span> </span><span class="NLM_article-title">3rd; Hendrickson, W. A. Structure–activity relationships in the binding of chemically derivatized CD4 to gp120 from human immunodeficiency virus</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">4898</span><span class="NLM_x">–</span> <span class="NLM_lpage">4908</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm070564e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=4898-4908&author=H.+Xieauthor=D.+Ngauthor=S.+N.+Savinovauthor=B.+Deyauthor=P.+D.+Kwongauthor=R.+Wyattauthor=A.+B.+Smith&title=3rd%3B+Hendrickson%2C+W.+A.+Structure%E2%80%93activity+relationships+in+the+binding+of+chemically+derivatized+CD4+to+gp120+from+human+immunodeficiency+virus"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm070564e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070564e%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DH.%26aulast%3DNg%26aufirst%3DD.%26aulast%3DSavinov%26aufirst%3DS.%2BN.%26aulast%3DDey%26aufirst%3DB.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26aulast%3DWyatt%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DA.%2BB.%26atitle%3D3rd%253B%2520Hendrickson%252C%2520W.%2520A.%2520Structure%25E2%2580%2593activity%2520relationships%2520in%2520the%2520binding%2520of%2520chemically%2520derivatized%2520CD4%2520to%2520gp120%2520from%2520human%2520immunodeficiency%2520virus%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D4898%26epage%3D4908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Stevens, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, R.</span><span> </span><span class="NLM_article-title">Amino Acid Derivatives 0.1. Carboallyloxy derivatives of alpha-amino acids</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1950</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">725</span><span class="NLM_x">–</span> <span class="NLM_lpage">727</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja01158a020" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1950&pages=725-727&author=C.+M.+Stevensauthor=R.+Watanabe&title=Amino+Acid+Derivatives+0.1.+Carboallyloxy+derivatives+of+alpha-amino+acids"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fja01158a020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja01158a020%26sid%3Dliteratum%253Aachs%26aulast%3DStevens%26aufirst%3DC.%2BM.%26aulast%3DWatanabe%26aufirst%3DR.%26atitle%3DAmino%2520Acid%2520Derivatives%25200.1.%2520Carboallyloxy%2520derivatives%2520of%2520alpha-amino%2520acids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1950%26volume%3D72%26spage%3D725%26epage%3D727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Majeed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ofek, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belachew, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwong, P. D.</span><span> </span><span class="NLM_article-title">Enhancing protein crystallization through precipitant synergy</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1061</span><span class="NLM_x">–</span> <span class="NLM_lpage">1070</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=1061-1070&author=S.+Majeedauthor=G.+Ofekauthor=A.+Belachewauthor=C.+C.+Huangauthor=T.+Zhouauthor=P.+D.+Kwong&title=Enhancing+protein+crystallization+through+precipitant+synergy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMajeed%26aufirst%3DS.%26aulast%3DOfek%26aufirst%3DG.%26aulast%3DBelachew%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DKwong%26aufirst%3DP.%2BD.%26atitle%3DEnhancing%2520protein%2520crystallization%2520through%2520precipitant%2520synergy%26jtitle%3DStructure%26date%3D2003%26volume%3D11%26spage%3D1061%26epage%3D1070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Method Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0liNFmiXbm2y8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethod%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Mccoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+Mccoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liNFmiXbm2y8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">McKinnon, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nyanga, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chege, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izulla, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huibner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gelmon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cicala, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anzala, A. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arthos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimani, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaul, R.</span><span> </span><span class="NLM_article-title">Characterization of a human cervical CD4(+) T cell subset coexpressing multiple markers of HIV susceptibility</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">187</span><span class="NLM_x">, </span> <span class="NLM_fpage">6032</span><span class="NLM_x">–</span> <span class="NLM_lpage">6042</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2011&pages=6032-6042&author=L.+R.+McKinnonauthor=B.+Nyangaauthor=D.+Chegeauthor=P.+Izullaauthor=M.+Kimaniauthor=S.+Huibnerauthor=L.+Gelmonauthor=K.+E.+Blockauthor=C.+Cicalaauthor=A.+O.+Anzalaauthor=J.+Arthosauthor=J.+Kimaniauthor=R.+Kaul&title=Characterization+of+a+human+cervical+CD4%28%2B%29+T+cell+subset+coexpressing+multiple+markers+of+HIV+susceptibility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcKinnon%26aufirst%3DL.%2BR.%26aulast%3DNyanga%26aufirst%3DB.%26aulast%3DChege%26aufirst%3DD.%26aulast%3DIzulla%26aufirst%3DP.%26aulast%3DKimani%26aufirst%3DM.%26aulast%3DHuibner%26aufirst%3DS.%26aulast%3DGelmon%26aufirst%3DL.%26aulast%3DBlock%26aufirst%3DK.%2BE.%26aulast%3DCicala%26aufirst%3DC.%26aulast%3DAnzala%26aufirst%3DA.%2BO.%26aulast%3DArthos%26aufirst%3DJ.%26aulast%3DKimani%26aufirst%3DJ.%26aulast%3DKaul%26aufirst%3DR.%26atitle%3DCharacterization%2520of%2520a%2520human%2520cervical%2520CD4%2528%252B%2529%2520T%2520cell%2520subset%2520coexpressing%2520multiple%2520markers%2520of%2520HIV%2520susceptibility%26jtitle%3DJ.%2520Immunol.%26date%3D2011%26volume%3D187%26spage%3D6032%26epage%3D6042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljWd1S3oyXlQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljWd1S3oyXlQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leaver-Fay, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Block, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snoeyink, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">W375</span><span class="NLM_x">–</span> <span class="NLM_lpage">W383</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1093%2Fnar%2Fgkm216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=17452350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W375-W383&author=I.+W.+Davisauthor=A.+Leaver-Fayauthor=V.+B.+Chenauthor=J.+N.+Blockauthor=G.+J.+Kapralauthor=X.+Wangauthor=L.+W.+Murrayauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+contacts+and+structure+validation+for+proteins+and+nucleic+acids"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span></div><div class="casAuthors">Davis Ian W; Leaver-Fay Andrew; Chen Vincent B; Block Jeremy N; Kapral Gary J; Wang Xueyi; Murray Laura W; Arendall W Bryan 3rd; Snoeyink Jack; Richardson Jane S; Richardson David C</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W375-83</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MolProbity is a general-purpose web server offering quality validation for 3D structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact analysis of any steric problems within the molecules as well as updated dihedral-angle diagnostics, and it can calculate and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addition and full optimization of all hydrogen atoms, both polar and nonpolar.  New analysis functions have been added for RNA, for interfaces, and for NMR ensembles.  Additionally, both the web site and major component programs have been rewritten to improve speed, convenience, clarity and integration with other resources.  MolProbity results are reported in multiple forms: as overall numeric scores, as lists or charts of local problems, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KiNG viewer.  This service is available free to all users at http://molprobity.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuyhuetoNS59bHMBt0mMIXfW6udTcc2ebEw8UFHYJ4Wrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D&md5=726ca1e7e162ab2a4a05048f493731fb</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm216%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DLeaver-Fay%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DBlock%26aufirst%3DJ.%2BN.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520contacts%2520and%2520structure%2520validation%2520for%2520proteins%2520and%2520nucleic%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW375%26epage%3DW383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Lutteke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von der Lieth, C.</span><span> </span><span class="NLM_article-title">W. pdb-care (PDB CArbohydrate REsidue check): a program to support annotation of complex carbohydrate structures in PDB files</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">69</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1186%2F1471-2105-5-69" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=15180909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A280%3ADC%252BD2czitlKntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=69&author=T.+Luttekeauthor=C.+von+der+Lieth&title=W.+pdb-care+%28PDB+CArbohydrate+REsidue+check%29%3A+a+program+to+support+annotation+of+complex+carbohydrate+structures+in+PDB+files"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">pdb-care (PDB carbohydrate residue check): a program to support annotation of complex carbohydrate structures in PDB files</span></div><div class="casAuthors">Lutteke Thomas; von der Lieth Claus-W</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">69</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Carbohydrates are involved in a variety of fundamental biological processes and pathological situations.  They therefore have a large pharmaceutical and diagnostic potential.  Knowledge of the 3D structure of glycans is a prerequisite for a complete understanding of their biological functions.  The largest source of biomolecular 3D structures is the Protein Data Bank.  However, about 30% of all 1663 PDB entries (version September 2003) containing carbohydrates comprise errors in glycan description.  Unfortunately, no software is currently available which aligns the 3D information with the reported assignments.  It is the aim of this work to fill this gap.  RESULTS:  The pdb-care program http://www.glycosciences.de/tools/pdb-care/ is able to identify and assign carbohydrate structures using only atom types and their 3D atom coordinates given in PDB-files.  Looking up a translation table where systematic names and the respective PDB residue codes are listed, both assignments are compared and inconsistencies are reported.  Additionally, the reliability of reported and calculated connectivities for molecules listed within the HETATOM records is checked and unusual values are reported.  CONCLUSION:  Frequent use of pdb-care will help to improve the quality of carbohydrate data contained in the PDB.  Automatic assignment of carbohydrate structures contained in PDB entries will enable the cross-linking of glycobiology resources with genomic and proteomic data collections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTpJZU-N5JJxnT4ugvCDgnZfW6udTcc2ebEw8UFHYJ4Wrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czitlKntQ%253D%253D&md5=1bb57895df47d192c04f8e3866a4f648</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-5-69&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-5-69%26sid%3Dliteratum%253Aachs%26aulast%3DLutteke%26aufirst%3DT.%26aulast%3Dvon%2Bder%2BLieth%26aufirst%3DC.%26atitle%3DW.%2520pdb-care%2520%2528PDB%2520CArbohydrate%2520REsidue%2520check%2529%253A%2520a%2520program%2520to%2520support%2520annotation%2520of%2520complex%2520carbohydrate%2520structures%2520in%2520PDB%2520files%26jtitle%3DBMC%2520Bioinf.%26date%3D2004%26volume%3D5%26spage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Wallace, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions</span> <span class="citation_source-journal">Protein Eng.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1093%2Fprotein%2F8.2.127" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1996&pages=127-134&author=A.+C.+Wallaceauthor=R.+A.+Laskowskiauthor=J.+M.+Thornton&title=LIGPLOT%3A+a+program+to+generate+schematic+diagrams+of+protein-ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F8.2.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F8.2.127%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DA.%2BC.%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DLIGPLOT%253A%2520a%2520program%2520to%2520generate%2520schematic%2520diagrams%2520of%2520protein-ligand%2520interactions%26jtitle%3DProtein%2520Eng.%26date%3D1996%26volume%3D8%26spage%3D127%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Hendlich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rippmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnickel, G.</span><span> </span><span class="NLM_article-title">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span> <span class="citation_source-journal">J. Mol. Graph. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span><div class="note"><p class="first last">389.</p></div></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm4002988&amp;key=10.1016%2FS1093-3263%2898%2900002-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm4002988&amp;key=9704298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm4002988&amp;key=1%3ACAS%3A528%3ADyaK1cXltFyktb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1997&pages=359-363&author=M.+Hendlichauthor=F.+Rippmannauthor=G.+Barnickel&title=LIGSITE%3A+automatic+and+efficient+detection+of+potential+small+molecule-binding+sites+in+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">LIGSITE: automatic and efficient detection of potential small molecule-binding sites in proteins</span></div><div class="casAuthors">Hendlich, Manfred; Rippmann, Friedrich; Barnickel, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">359-363</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">LIGSITE is a new program for the automatic and time-efficient detection of pockets on the surface of proteins that may act as binding sites for small mol. ligands.  Pockets are identified with a series of simple operations on a cubic grid.  Using a set of receptor-ligand complexes we show that LIGSITE is able to identify the binding sites of small mol. ligands with high precision.  The main advantage of LIGSITE is its speed.  Typical search times are in the range of 5 to 20 s for medium-sized proteins.  LIGSITE is therefore well suited for identification of pockets in large sets of proteins (e.g., protein families) for comparative studies.  For graphical display LIGSITE produces VRML representations of the protein-ligand complex and the binding site for display with a VRML viewer such as WebSpace from SGI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLm2ioggbs97Vg90H21EOLACvtfcHk0lifEMxmkG1zVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltFyktb0%253D&md5=27ccb5fdb0c07d5658f21c5570c8422b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS1093-3263%2898%2900002-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1093-3263%252898%252900002-3%26sid%3Dliteratum%253Aachs%26aulast%3DHendlich%26aufirst%3DM.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DBarnickel%26aufirst%3DG.%26atitle%3DLIGSITE%253A%2520automatic%2520and%2520efficient%2520detection%2520of%2520potential%2520small%2520molecule-binding%2520sites%2520in%2520proteins%26jtitle%3DJ.%2520Mol.%2520Graph.%2520Model.%26date%3D1997%26volume%3D15%26spage%3D359%26epage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RZK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1RZK','PDB','1RZK'); return false;">PDB: 1RZK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I60" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I60','PDB','2I60'); return false;">PDB: 2I60</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I5Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I5Y','PDB','2I5Y'); return false;">PDB: 2I5Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YYM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YYM','PDB','1YYM'); return false;">PDB: 1YYM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YYL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YYL','PDB','1YYL'); return false;">PDB: 1YYL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JZZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JZZ','PDB','4JZZ'); return false;">PDB: 4JZZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KA2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KA2','PDB','4KA2'); return false;">PDB: 4KA2</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i47"><a href="/doi/suppl/10.1021/jm4002988">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_80854"></div></div></div></div></div><hr /></hr><p class="last">MiniCD4 characterization data, gp120 sequence alignment, competition ELISA data, SPR profiles, crystallographic statistics, and supplemental figures. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm4002988/suppl_file/jm4002988_si_001.pdf">jm4002988_si_001.pdf (1.91 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Structure factors and coordinates for <b>13</b>–HIV-1–gp120 complex structure have been deposited in the Protein Data Bank with PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KA2">4KA2</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm4002988&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-12%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm4002988%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm4002988" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67999bbd4df92254","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
