
<html lang="en"     class="pb-page"  data-request-id="30a40177-9442-4173-809f-63f8f781d2fa"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-13;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c00479"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="EEDi-5285: An Exceptionally Potent, Efficacious, and Orally Active Small-Molecule Inhibitor of Embryonic Ectoderm Development" /></meta><meta name="dc.Creator" content="Rohan Kalyan  Rej" /></meta><meta name="dc.Creator" content="Changwei  Wang" /></meta><meta name="dc.Creator" content="Jianfeng  Lu" /></meta><meta name="dc.Creator" content="Mi  Wang" /></meta><meta name="dc.Creator" content="Elyse  Petrunak" /></meta><meta name="dc.Creator" content="Kaitlin P.  Zawacki" /></meta><meta name="dc.Creator" content="Donna  McEachern" /></meta><meta name="dc.Creator" content="Ester  Fernandez-Salas" /></meta><meta name="dc.Creator" content="Chao-Yie  Yang" /></meta><meta name="dc.Creator" content="Lu  Wang" /></meta><meta name="dc.Creator" content="Ruiting  Li" /></meta><meta name="dc.Creator" content="Krishnapriya  Chinnaswamy" /></meta><meta name="dc.Creator" content="Bo  Wen" /></meta><meta name="dc.Creator" content="Duxin  Sun" /></meta><meta name="dc.Creator" content="Jeanne  Stuckey" /></meta><meta name="dc.Creator" content="Yunlong  Zhou" /></meta><meta name="dc.Creator" content="Jianyong  Chen" /></meta><meta name="dc.Creator" content="Guozhi  Tang" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Description" content="Inhibition of embryonic ectoderm development (EED) is a new cancer therapeutic strategy. Herein, we report our discovery of EEDi-5285 as an exceptionally potent, efficacious, and orally active EED ..." /></meta><meta name="Description" content="Inhibition of embryonic ectoderm development (EED) is a new cancer therapeutic strategy. Herein, we report our discovery of EEDi-5285 as an exceptionally potent, efficacious, and orally active EED ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 24, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00479" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00479" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00479" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00479" /></link>
        
    
    

<title>EEDi-5285: An Exceptionally Potent, Efficacious, and Orally Active Small-Molecule Inhibitor of Embryonic Ectoderm Development | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00479" /></meta><meta property="og:title" content="EEDi-5285: An Exceptionally Potent, Efficacious, and Orally Active Small-Molecule Inhibitor of Embryonic Ectoderm Development" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0016.jpeg" /></meta><meta property="og:description" content="Inhibition of embryonic ectoderm development (EED) is a new cancer therapeutic strategy. Herein, we report our discovery of EEDi-5285 as an exceptionally potent, efficacious, and orally active EED inhibitor. EEDi-5285 binds to the EED protein with an IC50 value of 0.2 nM and inhibits cell growth with IC50 values of 20 pM and 0.5 nM in the Pfeiffer and KARPAS422 lymphoma cell lines, respectively, carrying an EZH2 mutation. EEDi-5285 is approximately 100 times more potent than EED226 in binding to EED and &gt;300 times more potent than EED226 in inhibition of cell growth in the KARPAS422 cell line. EEDi-5285 has excellent pharmacokinetics and achieves complete and durable tumor regression in the KARPAS422 xenograft model in mice with oral administration. The cocrystal structure of EEDi-5285 in a complex with EED defines the precise structural basis for their high binding affinity. EEDi-5285 is the most potent and efficacious EED inhibitor reported to date." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00479"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00479">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00479&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00479&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00479&amp;href=/doi/10.1021/acs.jmedchem.0c00479" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 7252-7267</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00475" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00483" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">EEDi-5285: An Exceptionally Potent, Efficacious, and Orally Active Small-Molecule Inhibitor of Embryonic Ectoderm Development</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Rohan Kalyan Rej</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rohan Kalyan Rej</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rohan+Kalyan++Rej">Rohan Kalyan Rej</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Changwei Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Changwei Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Changwei++Wang">Changwei Wang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jianfeng Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianfeng Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianfeng++Lu">Jianfeng Lu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Mi Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mi Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mi++Wang">Mi Wang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Elyse Petrunak</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elyse Petrunak</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elyse++Petrunak">Elyse Petrunak</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kaitlin P. Zawacki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaitlin P. Zawacki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaitlin+P.++Zawacki">Kaitlin P. Zawacki</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Donna McEachern</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Donna McEachern</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Donna++McEachern">Donna McEachern</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ester Fernandez-Salas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ester Fernandez-Salas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ester++Fernandez-Salas">Ester Fernandez-Salas</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Chao-Yie Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chao-Yie Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chao-Yie++Yang">Chao-Yie Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5445-0109" title="Orcid link">http://orcid.org/0000-0002-5445-0109</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lu Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lu Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lu++Wang">Lu Wang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ruiting Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ruiting Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ruiting++Li">Ruiting Li</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Krishnapriya Chinnaswamy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Krishnapriya Chinnaswamy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Krishnapriya++Chinnaswamy">Krishnapriya Chinnaswamy</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Bo Wen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bo Wen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bo++Wen">Bo Wen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Duxin Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Duxin Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Duxin++Sun">Duxin Sun</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6406-2126" title="Orcid link">http://orcid.org/0000-0002-6406-2126</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jeanne Stuckey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeanne Stuckey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeanne++Stuckey">Jeanne Stuckey</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yunlong Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yunlong Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Ascentage Pharma Group, Suzhou, Jiangsu 215216, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yunlong++Zhou">Yunlong Zhou</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jianyong Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianyong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Ascentage Pharma Group, Suzhou, Jiangsu 215216, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianyong++Chen">Jianyong Chen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Guozhi Tang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guozhi Tang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Ascentage Pharma Group, Suzhou, Jiangsu 215216, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guozhi++Tang">Guozhi Tang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7131-4678" title="Orcid link">http://orcid.org/0000-0002-7131-4678</a></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Shaomeng Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaomeng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Rogel Cancer Center, Departments of Internal Medicine and Pharmacology, Medical School, and Department of Medicinal Chemistry, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#4e3d262f21232b20290e3b23272d26602b2a3b"><span class="__cf_email__" data-cfemail="04776c656b69616a634471696d676c2a616071">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00479&amp;href=/doi/10.1021%2Facs.jmedchem.0c00479" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 7252â7267</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 24, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 March 2020</li><li><span class="item_label"><b>Published</b> online</span>24 June 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 July 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00479" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00479</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7252%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRohan%2BKalyan%2BRej%252C%2BChangwei%2BWang%252C%2BJianfeng%2BLu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D13%26contentID%3Dacs.jmedchem.0c00479%26title%3DEEDi-5285%253A%2BAn%2BExceptionally%2BPotent%252C%2BEfficacious%252C%2Band%2BOrally%2BActive%2BSmall-Molecule%2BInhibitor%2Bof%2BEmbryonic%2BEctoderm%2BDevelopment%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7267%26publicationDate%3DJuly%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00479"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1599</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00479" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;EEDi-5285: An Exceptionally Potent, Efficacious, and Orally Active Small-Molecule Inhibitor of Embryonic Ectoderm Development&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Rohan&quot;,&quot;last_name&quot;:&quot;Kalyan Rej&quot;},{&quot;first_name&quot;:&quot;Changwei&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Jianfeng&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Mi&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Elyse&quot;,&quot;last_name&quot;:&quot;Petrunak&quot;},{&quot;first_name&quot;:&quot;Kaitlin&quot;,&quot;last_name&quot;:&quot;P. Zawacki&quot;},{&quot;first_name&quot;:&quot;Donna&quot;,&quot;last_name&quot;:&quot;McEachern&quot;},{&quot;first_name&quot;:&quot;Ester&quot;,&quot;last_name&quot;:&quot;Fernandez-Salas&quot;},{&quot;first_name&quot;:&quot;Chao-Yie&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Lu&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Ruiting&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Krishnapriya&quot;,&quot;last_name&quot;:&quot;Chinnaswamy&quot;},{&quot;first_name&quot;:&quot;Bo&quot;,&quot;last_name&quot;:&quot;Wen&quot;},{&quot;first_name&quot;:&quot;Duxin&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Jeanne&quot;,&quot;last_name&quot;:&quot;Stuckey&quot;},{&quot;first_name&quot;:&quot;Yunlong&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Jianyong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Guozhi&quot;,&quot;last_name&quot;:&quot;Tang&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;7252-7267&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00479&quot;},&quot;abstract&quot;:&quot;Inhibition of embryonic ectoderm development (EED) is a new cancer therapeutic strategy. Herein, we report our discovery of EEDi-5285 as an exceptionally potent, efficacious, and orally active EED inhibitor. EEDi-5285 binds to the EED protein with an IC50 value of 0.2 nM and inhibits cell growth with IC50 values of 20 pM and 0.5 nM in the Pfeiffer and KARPAS422 lymphoma cell lines, respectively, carrying an EZH2 mutation. EEDi-5285 is approximately 100 times more potent than EED226 in binding to EED and &gt;300 times more potent than EED226 in inhibition of cell growth in the KARPAS422 cell line. EEDi-5285 has excellent pharmacokinetics and achieves complete and durable tumor regression in the KARPAS422 xenograft model in mice with oral administration. The cocrystal structure of EEDi-5285 in a complex with EED defines the precise structural basis for their high binding affinity. EEDi-5285 is the most potent and efficacious EED inhibitor reported to date.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00479&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00479" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00479&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00479" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00479&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00479" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00479&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00479&amp;href=/doi/10.1021/acs.jmedchem.0c00479" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00479" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00479" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00479%26sid%3Dliteratum%253Aachs%26pmid%3D32580550%26genre%3Darticle%26aulast%3DRej%26date%3D2020%26atitle%3DEEDi-5285%253A%2BAn%2BExceptionally%2BPotent%252C%2BEfficacious%252C%2Band%2BOrally%2BActive%2BSmall-Molecule%2BInhibitor%2Bof%2BEmbryonic%2BEctoderm%2BDevelopment%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D13%26spage%3D7252%26epage%3D7267%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292317" title="Ethyl groups">Ethyl groups</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/jmcmar.2020.63.issue-13/20200709/jmcmar.2020.63.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inhibition of embryonic ectoderm development (EED) is a new cancer therapeutic strategy. Herein, we report our discovery of EEDi-5285 as an exceptionally potent, efficacious, and orally active EED inhibitor. EEDi-5285 binds to the EED protein with an IC<sub>50</sub> value of 0.2 nM and inhibits cell growth with IC<sub>50</sub> values of 20 pM and 0.5 nM in the Pfeiffer and KARPAS422 lymphoma cell lines, respectively, carrying an EZH2 mutation. EEDi-5285 is approximately 100 times more potent than EED226 in binding to EED and >300 times more potent than EED226 in inhibition of cell growth in the KARPAS422 cell line. EEDi-5285 has excellent pharmacokinetics and achieves complete and durable tumor regression in the KARPAS422 xenograft model in mice with oral administration. The cocrystal structure of EEDi-5285 in a complex with EED defines the precise structural basis for their high binding affinity. EEDi-5285 is the most potent and efficacious EED inhibitor reported to date.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03516" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03516" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The polycomb group (PcG) is a family of proteins responsible for cellular differentiation during development via transcriptional repression.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> In mammals, PcG proteins are part of two major transcriptionally repressive complexes known as polycomb-repressive complexes 1 and 2 (PRC1 and PRC2).<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> The core PRC2 complex is composed of three proteins: the catalytic subunit EZH2 (an enhancer of zeste homolog 2), EED (embryonic ectoderm development), and SUZ12 (suppressor of zeste 12). PRC2 acts as an epigenetic modulator of transcription and plays a major role in a variety of biological processes, such as stem-cell maintenance and DNA repair.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">EZH2 is one of the most extensively studied histone methyltransferases, which catalyzes methylation of histone 3 lysine 27 (H3K27). In order to be catalytically active, EZH2 requires, at a minimum, two other PRC2 components, EED and SUZ12. EED enhances the enzymatic activity of PRC2 by binding with trimethylated H3K27 (H3K27me3), and SUZ12 interacts with other subunits within the PRC2 complex and contributes to the stability of the complex.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> By binding directly with H3K27me3, EED localizes the PRC2 complex to the chromatin substrate and allosterically activates the methyltransferase activity.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> The absence of either SUZ12 or EED completely abrogates the PRC2 function.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">Because the dysregulation of the PRC2 activity has been firmly linked to several forms of human cancers, major efforts have been made to develop small-molecule inhibitors targeting the PRC2 activity. One approach is to develop small-molecule inhibitors that directly bind to and inhibit EZH2, and this has yielded several EZH2 inhibitors such as compounds <b>1â3</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which are currently in clinical trials.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14">(11â14)</a> Tazemetostat (<b>2</b>) was approved in January 2020 by the US FDA for the treatment of advanced epithelioid sarcoma as a monotherapy.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative small-molecule inhibitors of EZH2 and EED that target the PRC2 activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although inhibitors of EZH2 have shown promising clinical activity, preclinical data suggest that resistance can be acquired through secondary mutations in EZH2.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Consequently, targeting the allosteric site within the regulatory EED subunit of PRC2 offers a novel therapeutic strategy to target the PRC2 activity.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Scientists from Novartis<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and AbbVie<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> first reported small-molecule inhibitors of the PRC2 activity, which function by targeting the histone-binding pocket in EED. Preclinical studies have shown that EED inhibitors such as EED226 (<b>4</b>), A-395 (<b>5</b>), and BR-001 (<b>6</b>) achieve strong antitumor activity in an EZH2 mutant lymphoma model as well as in EZH2 inhibitor-resistant models.<a onclick="showRef(event, 'ref8 ref9 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref16 ref17 ref18">(8,9,16â18)</a> To date, MAK683, a small-molecular EED inhibitor developed by Novartis, is the only EED inhibitor that has advanced into Phase1/2 clinical trials for advanced malignancies (<a href="https://clinicaltrials.gov/ct2/-show/NCT02900651" class="extLink">https://clinicaltrials.gov/ct2/-show/NCT02900651</a>). The chemical structure and the preclinical and clinical data for MAK683 have not been disclosed.</div><div class="NLM_p last">Herein, we report our structure-guided discovery of small-molecule inhibitors of EED. Our efforts have yielded the discovery of EEDi-5285 as an exceptionally potent, efficacious, and orally active small-molecule inhibitor of EED, which achieves complete and long-lasting tumor regression in mice. EEDi-5285 is the most potent and efficacious EED inhibitor reported to date.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34338" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34338" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The starting point for our design is based upon the cocrystal structure of EED226 (<b>4</b>) in a complex with EED (PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GSA">5GSA</a>).<a onclick="showRef(event, 'ref8 ref16'); return false;" href="javascript:void(0);" class="ref ref8 ref16">(8,16)</a> The cocrystal structure shows that the furan unit in EED226 inserts deeply into an aromatic âcageâ formed by four aromatic residues. The electron-deficient bicyclic [1,2,4]triazolo[4,3-<i>c</i>]pyrimidine core of EED226 forms ÏâÏ stacking interactions with the electron-rich Tyr148 and Tyr365 residues of EED. The electron-rich furan of EED226 has cationâÏ interactions with the guanidinium group of Arg367 of EED and an edge-to-face interaction with Tyr365 of EED. EED226 is also stabilized by two hydrogen bonds with EED: the amino group linking the bicyclic core and the furan group form a hydrogen bond with the side chain carbonyl oxygen of Asn194, and N2 of the triazolopyrimidine core forms a hydrogen bond with the side chain of Lys211. The methylsulfonylphenyl group is mostly exposed to solvents but its methyl group interacts with Pro95 and its phenyl group enjoys an edge-to-face interaction with the side chain of Phe97.</div><div class="NLM_p">For our design of EED inhibitors, we used compound <b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> a previously reported, moderately potent EED inhibitor as the starting point for our optimization efforts based on the following considerations: (a) <b>7</b> has a good binding affinity to EED with IC<sub>50</sub> = 115 nM in our assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>); (b) <b>7</b> is cell-permeable and shows an inhibitory activity in KARPAS422 lymphoma cell lines with IC<sub>50</sub> = 2.6 Î¼M (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>); (c) the published cocrystal structure of EED226 complexed with EED provides a foundation for the structure-guided optimization shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure-guided design and optimization of new EED inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Investigation of the Effect of an EWG on the Imidazo[1,5-<i>c</i>]pyrimidine Core in Compound <b>7</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0010.gif" alt="" id="gr10" /></img><div></div></div><div class="NLM_p">We sought to improve the binding affinity of compound <b>7</b> to EED. Compared to the electron-deficient bicyclic [1,2,4]triazolo[4,3-<i>c</i>]pyridine core in EED226, the imidazo[1,5-<i>c</i>]pyrimidine core in compound <b>7</b> is more electron-rich. We hypothesized that the binding affinity of compound <b>7</b> to EED may be improved by installation of an electron-withdrawing group onto its imidazo[1,5-<i>c</i>]pyrimidine core to enhance its ÏâÏ stacking interactions with the electron-rich Tyr148 and Tyr365 residues of EED. Accordingly, we synthesized compounds <b>8</b> and <b>9</b> and tested their binding affinity to EED and their cell growth inhibitory activity in the KARPAS422 cell line (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). While <b>9</b> with a methylamide group has a binding affinity similar to that of <b>7</b>, compound <b>8</b> containing an ethyl ester group is 2 times more potent than <b>7</b> in binding to EED and in inhibition of cell growth in KARPAS422 cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In our assays, EED226 has an IC<sub>50</sub> value of 17.6 nM in its binding to EED and an IC<sub>50</sub> value of 0.18 Î¼M in inhibition of cell growth in the KARPAS422 cell line. We employed compound <b>8</b> as the new lead compound for further optimization.</div><div class="NLM_p">Because the furan group in compound <b>8</b> is metabolically liable,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> we next synthesized and tested a number of compounds by replacing the furan group with other aromatic rings, with the results summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a><named-content content-type="anchor" rid="tbl3" type="simple"></named-content>. Replacement of the furan with a phenyl group yielded compound <b>10</b>, which is 2â3 times less potent than compound <b>8</b> in both binding and cellular assays. To enhance the cationâÏ interactions with the guanidinium group of Arg367 of EED, an electron-rich 2-methoxy group was introduced onto the phenyl group of <b>10</b>, generating compound <b>11</b>. Compound <b>11</b> has an IC<sub>50</sub> of 86 nM in binding to EED and an IC<sub>50</sub> value of 1.9 Î¼M in cell growth inhibition in KARPAS422 cells and thus is slightly more potent than <b>10</b>. Installation of a fluorine substituent at the C6 in the 2-methoxyphenyl ring of <b>11</b> yielded <b>12</b>, which binds to EED with an IC<sub>50</sub> value of 50 nM and achieves an IC<sub>50</sub> value of 0.3 Î¼M in inhibition of cell growth in KARPAS422 cells. Hence, although <b>12</b> is only slightly more potent than <b>11</b> in binding to EED, it is 6 times more potent than <b>11</b> in cell growth inhibition in KARPAS422 cells. Moving the methoxy group in compound <b>12</b> from C2 to C3 or C4 yielded compounds <b>13</b> and <b>14</b>, respectively. Compound <b>13</b> binds to EED with 138 nM and has an IC<sub>50</sub> of 2.7 Î¼M in KARPAS422 cells. Thus, <b>13</b> is 8 times less potent than <b>12</b> in cell growth inhibition in KARPAS422 cells. Compound <b>14</b> is slightly less potent than <b>13</b> in both binding and cellular assays.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Investigation of SAR by Replacing the Metabolically Labile Furan Ring</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0011.gif" alt="" id="GRAPHIC-d7e723-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0012.gif" alt="" id="gr11" /></img><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Further Optimization of the EWG Using 5-Fluoro-2,3-dihydrobenzofuran for the Deep Pocket</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0013.gif" alt="" id="gr12" /></img><div></div></div><div class="NLM_p">To further explore the deep binding pocket, we next synthesized a number of compounds containing an electron-rich bicyclic group to replace the phenyl group in <b>10</b>. Compound <b>15</b>, with a benzo[<i>d</i>][1,3]dioxole moiety, binds to EED with an IC<sub>50</sub> value of 40 nM and achieves an IC<sub>50</sub> of 0.3 Î¼M in inhibition of KARPAS422 cell growth. Compounds <b>16</b> and <b>17</b> were prepared in an attempt to learn the preferred position of the oxygen atom. Compound <b>17</b> is 7 times more potent than <b>16</b> in binding to EED and 30 times more potent than <b>16</b> in inhibition of cell growth in the KARPAS422 cell line.</div><div class="NLM_p">As introduction of a 6-fluoro substituent on the phenyl ring in <b>11</b> enhances the potency in both binding to EED and in cell growth inhibition, we installed a fluorine substituent at the same position in <b>17</b>, resulting in <b>18</b>. Compound <b>18</b> binds to EED with an IC<sub>50</sub> value of 18 nM and potently inhibits KARPAS422 cell growth with an IC<sub>50</sub> value of 12 nM. Hence, the 5-fluoro-2,3-dihydrobenzofuran group employed (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) in <b>18</b>, which was previously disclosed in a Novartis patent application,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> is the best among a number of groups investigated in our study.</div><div class="NLM_p">Based upon the potent binding affinity of <b>18</b> to EED and its excellent cell growth inhibition in the KARPAS422 cell line, we performed further modifications of its ethyl ester electron-withdrawing group (EWG). Replacement of the ester group in <b>18</b> with a methylamide yielded compound <b>19</b>, which binds to EED with an IC<sub>50</sub> value of 23 nM and inhibits KARPAS422 cell growth with an IC<sub>50</sub> value of 60 nM. Replacing the ethyl ester with CF<sub>3</sub>, a methylsulfone or a dimethylphosphine oxide yielded compounds <b>20</b>, <b>21</b>, and <b>22</b>, respectively. Compound <b>20</b> binds to EED with an IC<sub>50</sub> value of 29 nM and inhibits KARPAS422 cell growth with an IC<sub>50</sub> value of 120 nM. Compound <b>21</b>, containing a methylsulfone, achieves the best potency among this series of compounds, with IC<sub>50</sub> = 0.7 nM to EED and IC<sub>50</sub> = 2.2 nM in inhibition of KARPAS422 cell growth. Compound <b>22</b> achieves an IC<sub>50</sub> value of 3.6 nM in binding to EED and an IC<sub>50</sub> value of 8.1 nM in cell growth inhibition in the KARPAS422 cell line (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Table <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><div class="NLM_p">In the cocrystal structure of EED226 complexed with EED, the phenyl group has interactions with Phe97 of EED and the methylsulfonyl group is largely exposed to a solvent. We next performed modifications of the corresponding phenyl group in compound <b>21</b>, with the results summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. StructureâActivity Relationship and Optimization of the Tail Part, R<sub>2</sub> at C8</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0014.gif" alt="" id="GRAPHIC-d7e877-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0015.gif" alt="" id="gr13" /></img><div></div></div><div class="NLM_p">We first installed a 4-methylsulfonyl group in <b>21</b>, same as that in EED226 and obtained <b>23</b>, which binds to EED with an IC<sub>50</sub> value of 0.4 nM and inhibits KARPAS422 cell growth with IC<sub>50</sub> = 1.4 nM. Installation of a 4-fluoro substituent on the phenyl group of <b>21</b> resulted in <b>24</b>, which is slightly more potent than <b>21</b> in binding to EED (IC<sub>50</sub> = 0.4 nM vs 0.7 nM) but is 2â3 times less potent than <b>21</b> in inhibition of KARPAS422 cell growth.</div><div class="NLM_p">To improve the solubility of the compounds, we made a series of compounds in which the phenyl group in <b>21</b> was replaced with a pyridine. Compound <b>26</b> with a 2-methyl-3-pyridine group is half as potent as <b>21</b> in both binding to EED and inhibition of KARPAS422 cell growth. Compound <b>27</b> with a 4-methyl-3-pyridine group is slightly more potent than <b>21</b> in both binding to EED and inhibition of KARPAS422 cell growth. Compound <b>28</b> with a 4-cyclopropyl-3-pyridine substituent is 3â4 times more potent than <b>21</b> in both binding to EED and inhibition of KARPAS422 cell growth.</div><div class="NLM_p">Encouraged by the results from compound <b>28</b>, we synthesized additional analogues with different substituents at the 4-position of the 3-pyridine group. Compound <b>29</b> with a difluoromethyl substituent binds to EED with IC<sub>50</sub> = 0.2 nM, and is thus 5 times more potent than <b>21</b>, but its potency in inhibition of KARPAS422 cell growth is similar to that of <b>21</b>. Compound <b>30</b> with a trifluoromethyl group is slightly more potent than <b>21</b> in binding to EED and in inhibition of KARPAS422 cell growth. Compound <b>32</b> with 2,4-dimethyl-3-pyridine has a potency similar to that of <b>21</b> in both binding to EED and KARPAS422 cell growth inhibition. Compound <b>33</b> containing 2-methyl-4-trifluoromethyl-3-pyridine is slightly more potent than <b>21</b> in both binding to EED and cell growth inhibition of KARPAS422 cells.</div><div class="NLM_p">We synthesized <b>34</b> by replacing the phenyl group in <b>21</b> with a 1-methyl-1<i>H</i>-pyrazole and found that these two compounds have a similar potency in binding to EED and in inhibition of KARPAS422 cell growth.</div><div class="NLM_p">Because the 4-methylsulfonylphenyl group in EED226 has mainly hydrophobic contacts with EED, we synthesized a number of analogues by replacing the phenyl group in <b>21</b> with a nonaromatic, six-membered ring. Compound <b>35</b> with 4-tetrahydro-2<i>H</i>-pyran in place of this phenyl group binds to EED with IC<sub>50</sub> = 12 nM, and is 17 times less potent than <b>21</b>. Consistent with its weaker binding affinity to EED, <b>35</b> is 24 times less potent than <b>21</b> in inhibition of KARPAS422 cell growth. Replacing the oxygen in the six-membered ring in <b>35</b> with sulfonyl, ethylamino, methylamide, methylsulfonylamide, or cyclopropylsulfonylamide generated compounds <b>36â40</b>, respectively. All of these compounds bind to EED with a weaker affinity than compound <b>21</b> and also show a weaker activity than <b>21</b> in inhibition of KARPAS422 cell growth.</div><div class="NLM_p">In addition to the KARPAS422 cell line, the Pfeiffer lymphoma cell line, which carries an EZH2 mutation, is also very responsive to EED inhibitors.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> We tested a number of representative compounds, which show potent cell growth inhibition in the KARPAS422 cell line, for their cell growth-inhibitory activity in the Pfeiffer cell line, obtaining the results summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><div class="NLM_p">Compounds with potent cell growth inhibition in the KARPAS422 cell line are generally more potent in the Pfeiffer cell line and a number of these compounds achieve picomolar potencies. For example, compounds <b>23</b>, <b>25</b>, <b>27</b>, <b>28</b>, <b>29</b>, and <b>30</b> achieve IC<sub>50</sub> values of 20â30 pM in the Pfeiffer cell line. Thus, our modifications have produced a number of highly potent EED inhibitors, which bind to EED with subnanomolar affinities and have low nanomolar to subnanomolar IC<sub>50</sub> values in inhibition of KARPAS422 cell growth and low picomolar IC<sub>50</sub> values in inhibition of Pfeiffer cell growth.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Determination of Cocrystal Structures of EEDi-5285 and EEDi-1056 in a Complex with EED Protein</h3><div class="NLM_p">To define their precise binding modes and to understand the structural basis for their high affinity binding, we determined high-resolution cocrystal structures for EEDi-5285 (<b>28</b>) and EEDi-1056 (<b>32</b>) in a complex with EED (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7F">6W7F</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7G">6W7G</a>, respectively).</div><div class="NLM_p">For both EEDi-5285 and EEDi-1056, the 5-fluoro-2,3- dihydrobenzofuran group spatially fills the deep pocket of EED excluding all water molecules and engages in cationâÏ interactions with the guanidinium group of Arg367 of EED. This hand-in-glove style fit creates more van der Waals interactions with EED than furan group of EED226, which relies on transient waters for stabilization within the pocket. Although similar cationâÏ interactions are observed in the cocrystal structure of EED226 in a complex with EED (PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GSA">5GSA</a>), the guanidinium group of Arg367 is not directly underneath the electron-rich furan group. In comparison, the guanidinium group of Arg367 is now directly underneath the 5-fluoro-2,3-dihydrobenzofuran group, suggesting stronger cationâÏ interactions than those observed in the EED226 cocrystal structure. In addition, the side chain of Tyr365 is directly above the benzo group of the 5-fluoro-2,3-dihydrobenzofuran group and enjoys more extensive interactions between them than those observed in the EED226 cocrystal structure. Similar to those observed in the EED226 cocrystal structure, there are ÏâÏ interactions between the 1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidine core in both inhibitors and the two electron-rich Tyr148 and Tyr365 residues of EED, with the methylsulfonyl group exposed to the solvent environment. As compared to those observed in the EED226 cocrystal structure, there are also two similar hydrogen bonds formed between EED with either EEDi-5285 or EEDi-1056: a hydrogen bond between the amino group linking the 1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidine core and the (5-fluoro-2,3-dihydrobenzofuran-4-yl)methyl group with the side chain carbonyl oxygen of Asn194 and another hydrogen bond between the N2 atom of the 1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidine core and the side chain of Lys211. The 2-cyclopropylpyridinyl group in EEDi-5285 or 2,6-dimethylpyridinyl group in EEDi-1056 captures very similar interactions as the methylsulfonylphenyl group in EED226. These two cocrystal structures for EEDi-5285 and EEDi-1056 provide a solid structural basis for their very high-affinity affinities with EED (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structures of EED bound to inhibitors. Key interactions of compound (a) EEDi-5285 with EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7F">6W7F</a>), (b) EEDi-1056 with EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7G">6W7G</a>) and (c) EED226 with EED (PDB accession code 5GSA) for comparison. Dashed lines represent hydrogen bonds. Water molecules are depicted as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Interestingly, in both cocrystal structures, we also observed a secondary binding mode with an inhibitor molecule-bound EED opposite the primary, histone-binding pocket. In this secondary binding model, the 5-fluoro-2,3-dihydrobenzofuran group in both inhibitors fits into a relatively shallow pocket, with the rest of the molecule sitting on a large, surface pocket, suggesting low affinity interactions.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Pharmacokinetic Studies of EED Inhibitors in Mice</h3><div class="NLM_p">We evaluated 12 potent EED inhibitors (compounds <b>23â34</b>) in mice with three time-points to provide an initial assessment of their oral bioavailability obtaining the data summarized in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Oral Exposure of EED Inhibitors in Plasma with Oral Administration in Mice<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">plasma drug concentration (ng/mL), PO (25Â mg/kg)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">1Â h</th><th class="colsep0 rowsep0" align="center">3Â h</th><th class="colsep0 rowsep0" align="center">6Â h</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char=".">598</td><td class="colsep0 rowsep0" align="left">171</td><td class="colsep0 rowsep0" align="left">109</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="left"><10</td><td class="colsep0 rowsep0" align="left"><10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="char" char=".">591</td><td class="colsep0 rowsep0" align="left">279</td><td class="colsep0 rowsep0" align="left">98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="char" char=".">488</td><td class="colsep0 rowsep0" align="left">194</td><td class="colsep0 rowsep0" align="left">91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="char" char=".">340</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b> (EEDi-5285)</td><td class="colsep0 rowsep0" align="char" char=".">1096</td><td class="colsep0 rowsep0" align="left">608</td><td class="colsep0 rowsep0" align="left">452</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="char" char=".">242</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left"><10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="char" char=".">468</td><td class="colsep0 rowsep0" align="left">189</td><td class="colsep0 rowsep0" align="left">139</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="char" char=".">351</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b> (EEDi-1056)</td><td class="colsep0 rowsep0" align="char" char=".">2477</td><td class="colsep0 rowsep0" align="left">754</td><td class="colsep0 rowsep0" align="left">485</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="char" char=".">896</td><td class="colsep0 rowsep0" align="left">794</td><td class="colsep0 rowsep0" align="left">415</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="char" char=".">468</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">56</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Each compound was administered orally at 25 mg/kg. Plasma samples were collected at 1, 3, and 6 h with two mice at each time point and analyzed by LCâMS/MS. Mean values of drug concentrations are presented.</p></div></div></div><div class="NLM_p">The exposure data showed that a number of these EED inhibitors, such as <b>23</b>, <b>26</b>, <b>28</b>, <b>30</b>, and <b>32</b>, achieve good plasma concentrations, with <b>28</b> and <b>32</b> being the best.</div><div class="NLM_p">Based on the initial oral exposure data, we selected compounds <b>28</b> and <b>32</b> for pharmacokinetic studies in mice and obtained the data summarized in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. Compound <b>28</b> achieves a <i>C</i><sub>max</sub> of 1.8 Î¼M and an AUC of 6.0 h*Î¼g/ml with 10 mg/kg oral administration and has an oral bioavailability (<i>F</i>) of 75%. Compound <b>32</b> achieves a <i>C</i><sub>max</sub> of 1.3 Î¼M and an AUC of 2.6 h* Î¼g/ml with 10 mg/kg oral administration and has an oral bioavailability of 69%. Both compounds have a moderate volume of distribution of 1.4â1.6 L/kg and a terminal <i>T</i><sub>1/2</sub> of approximately 2 h. The PK data thus show that both <b>28</b> and <b>32</b> have an excellent PK profile in mice.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Pharmacokinetic Parameters of <b>28</b> (EEDi-5285) and <b>32</b> (EEDi-1056) in Mice<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">route/dose</th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center"><i>C</i><sub>max</sub> (Î¼g/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0â<i>t</i></sub> (hÂ·Î¼g/mL)</th><th class="colsep0 rowsep0" align="center" char=".">Cl (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center"><i>F</i>Â (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b> (EEDi-5285)</td><td class="colsep0 rowsep0" align="left">IV/2Â mg/kg</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">PO/10Â mg/kg</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="left">75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b> (EEDi-1056)</td><td class="colsep0 rowsep0" align="left">IV/2Â mg/kg</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">PO/10Â mg/kg</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="left">69</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last"><i>C</i><sub>max</sub>, maximum drug concentration; AUC, area-under-the-curve; Cl = plasma clearance, <i>V</i><sub>ss</sub> = volume of distribution, <i>T</i><sub>1/2</sub> = terminal half-life, <i>F</i> = oral bioavailability.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Pharmacodynamic Studies of EED Inhibitors in KARPAS422 Tumor Tissue</h3><div class="NLM_p">We next evaluated compounds EEDi-5285 and EEDi-1056 for their pharmacodynamics (PD) effect in KARPAS422 tumor tissue in mice. Our PD data showed that a single 100 mg/kg oral administration of EEDi-5285 or EEDi-1056 effectively reduces the level of H3K27me3 at 24 h (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pharmacodynamic analysis of EEDi-5285 and EEDi-1056 in KARPAS422 xenograft tumors. SCID mice bearing KARPAS422 tumors were treated with a single oral administration of EEDi-5285 or EEDi-1056 at 100 mg/kg. Mice were euthanized at the 24 h time-point and tumor tissues were collected. Histone proteins were extracted from the treated cells with the EpiQuik Total Histone Extraction Kit, and the level of H3K27me3 was examined by Western blotting analysis. Total histone H3 was used as the loading control. Three mice were used for each compound with each mouse bearing one tumor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Antitumor Activity of EEDi-5285 and EEDi-1056 in the KARPAS422 Xenograft Model in Mice</h3><div class="NLM_p">We evaluated EEDi-5285 and EEDi-1056 for their antitumor activity in the KARPAS422 xenograft model in mice (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Both compounds were administered 50 and 100 mg/kg via oral gavage, daily for 28 days. Both EEDi-5285 and EEDi-1056 achieved complete tumor regression at both 50 and 100 mg/kg during the treatment period (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Significantly, no regrowth of tumors was observed after treatment had been stopped for 72 days. Both the compounds caused minimal weight loss during the entire experiment (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). This antitumor efficacy experiment demonstrated that both EEDi-5285 and EEDi-1056 are highly efficacious and capable of achieving complete and long-lasting tumor regression in the KARPAS422 xenograft model in mice with oral administration at well-tolerated dose schedules. As both EEDi-5285 and EEDi-1056 achieved complete tumor regression at 50 mg/kg, the lowest dose for them to achieve complete tumor regression in the KARPAS xenograft tumor model was not determined.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Antitumor activity of EEDi-5285 and EEDi-1056 in a KARPAS422 xenograft model in SCID mice. Each group had seven mice and each mouse had one tumor. (B) Changes in animal body weights.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51825" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51825" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compounds <b>8â19</b> were prepared, as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, compounds <b>7</b> and <b>20â34</b> were prepared, as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, and compounds <b>35â40</b> were prepared, as shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Commercially available 5-bromo-4-chloro-2-(methylthio)pyrimidine (compound <b>41</b>) was converted to the corresponding amine (<b>43</b>) by treatment with ethyl 2-(diphenylmethyleneamino)acetate, followed by acid-catalyzed hydrolysis. The amine (<b>43</b>) upon formylation followed by cyclization using POCl<sub>3</sub> afforded the bicyclic compound <b>45</b>, which was converted to <b>46</b> by selective displacement of the SâMe group with furan-2-ylmethanamine at the 5-position.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Using the Suzuki reaction conditions, compound <b>46</b> was converted to the phenyl analog (compound <b>8</b>). Target compounds <b>10â18</b> were made using this protocol with different amines, while compound <b>18</b> was obtained using 2-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>â²,<i>N</i>â²-tetramethyluronium hexafluoro-phosphate (HATU) as the coupling reagent. The details of the synthesis of the compounds in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf" class="ext-link">Supporting Information</a>.</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>8â19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl 2-(diphenylmethyleneamino)acetate, NaH, rt, 2 h; (b) 3 N HCl in THF, rt, 1 h, 70%; (c) HCO<sub>2</sub>H, Ac<sub>2</sub>O, rt, 2 h; (d) POCl<sub>3</sub>, dioxane, reflux, 4 h, 70%; (e) (i) mCPBA, DCM, (ii) furan-2-ylmethanamine, rt, 3 h, 55%; (f) phenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxaneâH<sub>2</sub>O, 90 Â°C overnight, 90%; (g) Li(OH)<sub>2</sub>, THFâH<sub>2</sub>O, 90%; (h) MeNH<sub>2</sub>Â·HCl, DIPEA, HATU, 90%</p></p></figure><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>7</b> and <b>20â34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CH<sub>3</sub>COCl, MeOH, 79%; (b) DIBAL-H, 56%; (c) DIAD, PPh<sub>3</sub>, phthalidomide, 70%; (d) NH<sub>2</sub>Â·NH<sub>2</sub>. H<sub>2</sub>O, EtOH, 80%; (e) HCO<sub>2</sub>H, Ac<sub>2</sub>O, 60%; (f) POCl<sub>3</sub>, dioxane, 47.0%; (g) (i) mCPBA, DCM, (ii) furan-2-ylmethanamine, rt, 3 h, 52%; (h) phenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxaneâH<sub>2</sub>O, 90 Â°C overnight, 88%; (i) (5-fluoro-2,3-dihydrobenzofuran-4-yl)methanamine, 40 Â°C, 24 h, 60%; (j) phenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxaneâH<sub>2</sub>O, 90 Â°C overnight, 88%; (k) NIS, DMF, 0 Â°C, 70%; (l) copper (I) iodide, methyl 2,2-difluoro-2-(fluorosulfonyl)acetate, PdCl<sub>2</sub>(dppf)Cl<sub>2</sub>, 42%; (m) MeSO<sub>2</sub>Na, CuI, DMSO, 50%; (n) dimethylphosphine oxide, Pd<sub>2</sub>(dba)<sub>3</sub>, xantphos, Et<sub>3</sub>N, dioxane, 40%.</p></p></figure><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>35â40</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxaneâH<sub>2</sub>O, 90 Â°C overnight, 88%; (b) H<sub>2</sub>, Pd/C, MeOH; (c) NIS, DMF, 0 Â°C, 70%; (d) MeSO<sub>2</sub>Na, CuI, DMSO, 50%; (e) TFA, DCM; (f) DCM, Et<sub>3</sub>N, alkyl chloride; (g) DCM/AcOH (1:1), alkyl aldehyde or ketone, NaBH(OAc)<sub>3</sub>.</p></p></figure><div class="NLM_p">In <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, the bicyclic compound <b>54</b> was made from the corresponding amine <b>52</b> using formylation followed by cyclization using POCl<sub>3</sub>. Upon Cu-mediated trifluoromethylation<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and sulfonation,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> intermediate <b>58</b> afforded the corresponding compounds <b>20</b> and <b>21</b>, respectively. Intermediate <b>58</b> upon Pd-catalyzed P-arylation<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> with dialkyl phosphine oxide formed compound <b>22</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)<named-content content-type="anchor" rid="sch4" type="simple"></named-content>.</div><figure id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of (5-Fluoro-2,3-dihydrobenzofuran-4-yl)methanamine<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2-bromo-1,1-diethoxyethane, K<sub>2</sub>CO<sub>3</sub>, DMF, 110 Â°C; (b) PPA, toluene, 90 Â°C; (c) Zn(CN)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF; (d) LAH, THF, 50 Â°C; (e) H<sub>2</sub>, Pd/C, 50 Â°C.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32354" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32354" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Starting from a previously reported EED inhibitor (<b>7</b>), we performed structure-guided design and optimization to identify potent and efficacious new EED inhibitors. Our efforts have yielded exceptionally potent EED inhibitors. A number of these new EED inhibitors bind to EED with subnanomolar IC<sub>50</sub> values and inhibit cell growth with subnanomolar to low nanomolar IC<sub>50</sub> values in the KARPAS422 cell line and low picomolar IC<sub>50</sub> values in the Pfeiffer cell line. Among these, EEDi-5285 and EEDi-1056 show excellent PK profiles in mice. Both EEDi-5285 and EEDi-1056 are capable of achieving complete and long-lasting tumor regression in the KARPAS422 xenograft model in mice with oral administration and at well-tolerated dose schedules. Determination of cocrystal structures of EEDi-5285 and EEDi-1056 complexed with EED defines their precise structural basis for their high binding affinity.</div><div class="NLM_p last">Previous studies have reported EED226, A-395, and BR-001 as three of the potent and promising EED inhibitors.<a onclick="showRef(event, 'ref8 ref9 ref18'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref18">(8,9,18)</a> In direct comparison, EEDi-5285 is approximately 100 times more potent than EED226 in binding to EED (0.2 nM vs 17.6 nM) and is >300 times more potent than EED226 in inhibition of cell growth in the KARPAS422 cell line (0.5 nM vs 182 nM). While EED226 was capable of achieving complete KARPAS422 cell tumor regression at 300 mg/kg with twice daily administration via oral gavage,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> EEDi-5285 achieved complete and long-lasting tumor regression with daily administration via oral gavage at 50 mg/kg in the same model. A-395 was shown to have an IC<sub>50</sub> value of 69 nM in the Pfeiffer cell line with a 10-day treatment time and was able to inhibit tumor growth by 84% at 300 mg/kg with twice a week subcutaneous injections for 5 weeks.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> BR-001 was shown to be 2â3 times more potent than EED226 in cells and capable of inhibiting tumor growth by 85% at 100 mg/kg with twice daily administration.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Therefore, EEDi-5285 is the most potent and efficacious EED inhibitor reported to date and warrants extensive preclinical studies.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11770" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11770" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Information</h3><div class="NLM_p">Unless otherwise stated, all commercial reagents were used as supplied without further purification, and all reactions were performed under a nitrogen atmosphere in dry solvents under anhydrous conditions. NMR spectra were obtained with a Bruker 400 Ascend spectrometer at a <sup>1</sup>H frequency of 400 MHz and a <sup>13</sup>C frequency of 100 MHz. Chemical shifts (Î´) are reported in parts per million (ppm) relative to an internal standard. The final products were purified on a preparative high-performance liquid chromatography (HPLC) column (Waters 2545, Quaternary Gradient Module) with a SunFire Prep C18 OBD 5 Î¼m 50 Ã 100 mm<sup>2</sup> reverse phase column. The mobile phase was a gradient of solvent A (H<sub>2</sub>O with 0.1% TFA) and solvent B (CH<sub>3</sub>CN with 0.1% of TFA) at a flow rate of 60 mL/min and 1%/min increase of solvent B. All final compounds have purity â¥95% as determined by Waters ACQUITY ultraperformance liquid chromatography (UPLC) using a reverse phase column (SunFire, C18, 5 Î¼m, 4.6 Ã 150 mm<sup>2</sup>) and a solvent gradient of A (H<sub>2</sub>O with 0.1% of TFA) and solvent B (CH<sub>3</sub>CN with 0.1% of TFA). Electrospray ionization (ESI) mass spectrometry (MS) analysis was performed on a Thermo Scientific LCQ Fleet mass spectrometer.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Ethyl 2-(5-Bromo-2-(methylthio)pyrimidin-4-yl)-2-((diphenylmethylene)amino)acetate (<b>42</b>)</h4><div class="NLM_p last">A solution of ethyl 2-(diphenylmethyleneamino)acetate (18.4 g, 69 mmol) in DMSO (50 mL) was added dropwise at 0 Â°C to a suspension of 60% NaH (5.0 g, 125.5 mmol) in anhydrous DMSO (70 mL). The reaction mixture turned orange immediately. After 5 min, 5-bromo-4-chloro-2-(methylthio)pyrimidine (<b>41</b>, 15 g, 62.7 mmol) in 50 mL of DMSO was added dropwise. The mixture was then stirred at rt for 2 h. Then, the reaction mixture was quenched by careful addition of aq NH<sub>4</sub>Cl solution. The mixture was extracted with EtOAc, washed with brine, dried, and concentrated and used as obtained in the next step. LCâMS: [M + H]<sup>+</sup>, 470.01.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Ethyl 2-Amino-2-(5-bromo-2-(methylthio)pyrimidin-4-yl)acetate (<b>43</b>)</h4><div class="NLM_p last">3 N HCl in H<sub>2</sub>O (10 mL) was added at 0 Â°C to a solution of crude compound <b>42</b> (5.0 g, 10.6 mmol) in THF (50 mL). The mixture was stirred at rt for 1 h and the reaction mixture was then concentrated followed by adjustment to pH 8â9 with aq Na<sub>2</sub>CO<sub>3</sub> solution. The mixture was extracted with DCM and washed with brine. Concentration under reduced pressure followed by purification by flash chromatography (0â100% EtOAc/hexane) gave the desired compound <b>43</b> (2.26 g) in 70% overall yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.58 (s, 1H), 4.99 (s, 1H), 4.26â4.16 (m, 2H), 2.53 (s, 3H), 1.26 (t, 3H). LCâMS: [M + H]<sup>+</sup>, 305.95.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Ethyl 2-(5-Bromo-2-(methylthio)pyrimidin-4-yl)-2-formamidoacetate (<b>44</b>)</h4><div class="NLM_p last">A mixture of HCO<sub>2</sub>H (4 mL) and Ac<sub>2</sub>O (4 mL) was heated at 50 Â°C for 1 h. The reaction mixture was cooled to rt and added to a solution of compound <b>43</b> (2.0 g, 6.55 mmol) in DCM (20 mL). The mixture was stirred at rt for 2 h. After completion of the reaction, the mixture was concentrated. The mixture was extracted with DCM (2 Ã 50 mL) and washed successively with H<sub>2</sub>O (20 mL) and brine (10 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford the crude title compound <b>44</b> as an oil, which was used in the next steps without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.63 (s, 1H), 8.32 (s, 1H), 7.02 (br s, 1H), 6.20 (d, 1H), 4.31â4.11 (m, 2H), 2.54 (s, 3H), 1.27 (t, 3H). LCâMS: [M + H]<sup>+</sup>, 334.05.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Ethyl 8-Bromo-5-(methylthio)imidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>45</b>)</h4><div class="NLM_p last">POCl<sub>3</sub> (1.5 mL) was added dropwise to a solution of compound <b>44</b> (2.0 g, crude) in dioxane (20 mL). The reaction mixture was heated under reflux for 4 h. The mixture was cooled to rt and concentrated. Ice/H<sub>2</sub>O (50 mL) was added, and the mixture was adjusted to pH 8 with satd. aq NaHCO<sub>3</sub>. The mixture was extracted with DCM (2 Ã 50 mL), washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and filtered. The filtrate was concentrated, and the residue was purified by silica gel column chromatography (eluting with 50â100% EtOAc/hexane) to afford the title compound <b>45</b> as a white solid (1.42 g, 4.59 mmol) in 70% overall yield over two steps. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 8.67 (s, 1H), 7.99 (s, 1H), 4.33 (q, 2H), 2.76 (s, 3H), 1.34 (t, 3H). LCâMS: [M + H]<sup>+</sup>, 315.70.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Ethyl 8-Bromo-5-((furan-2-ylmethyl)amino)imidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>46</b>)</h4><div class="NLM_p last"><i>m</i>-CPBA (464 mg, 2.7 mmol, â¤77%, 1.5 equiv) was added at 0 Â°C to a solution of compound <b>45</b> (567 mg, 1.8 mmol, 1.0 equiv) in DCM (18 mL). After 45 min, Et<sub>3</sub>N (1 mL, 7.6 mmol, 4 equiv) was added at 0 Â°C and stirred for 2 min, followed by addition of furan-2-ylmethanamine (175 mg, 1.8 mmol). The reaction mixture was then stirred at rt for 3 h. Then, the reaction mixture was concentrated and the residue was purified by silica gel column chromatography (eluting with 50â100% EtOAc/hexane) to afford the title compound <b>46</b> (361 mg, 0.99 mmol) in 55% yield. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.58 (s, 1H), 7.70 (d, <i>J</i> = 2.1 Hz, 1H), 7.47 (dd, <i>J</i> = 1.8, 1.0 Hz, 1H), 6.39 (dt, <i>J</i> = 3.2, 1.1 Hz, 2H), 4.78 (t, <i>J</i> = 0.9 Hz, 2H), 4.46â4.36 (m, 2H), 1.43â1.35 (m, 3H). LCâMS: [M + H]<sup>+</sup>, 365.01.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Ethyl 5-((Furan-2-ylmethyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>8</b>)</h4><div class="NLM_p last">Potassium carbonate (227 mg, 1.64 mmol), phenylboronic acid (168 mg, 0.82 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (63 mg, 0.055 mmol) were added to a solution of compound <b>46</b> (200 mg, 0.55 mmol) in a mixed solvent (dioxane/H<sub>2</sub>O = 10:2.5 mL). The resulting mixture was stirred under N<sub>2</sub> at 90 Â°C overnight. The mixture was then cooled to rt, and the solvent was removed in vacuo. The residue was purified with silica gel chromatography (eluted with 0â10% MeOH/DCM) to afford title compound <b>8</b> (159 mg, 0.34 mmol) in 80% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.81 (s, 1H), 7.49 (d, <i>J</i> = 1.0 Hz, 1H), 7.41 (d, <i>J</i> = 6.7 Hz, 3H), 7.34 (dq, <i>J</i> = 3.3, 1.7 Hz, 3H), 6.35 (d, <i>J</i> = 3.2 Hz, 1H), 6.30 (dt, <i>J</i> = 3.1, 1.4 Hz, 1H), 4.87 (s, 2H), 3.87 (q, <i>J</i> = 7.1 Hz, 2H), 0.88 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 363.01.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 5-((Furan-2-ylmethyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylic Acid (<b>47</b>)</h4><div class="NLM_p last">A mixture of <b>8</b> (40 mg, 0.11 mmol, 1 equiv) and LiOH (26 mg, 1.10 mmol, 10 equiv) in THF (4 mL) and H<sub>2</sub>O (2.0 mL) was heated at 70 Â°C overnight. 3 N aq HCl was added dropwise at 0 Â°C to pH 2â3. The mixture was concentrated, and the residue was purified by HPLC to afford the title compound <b>47</b> (33 mg, 0.099 mmol) in 90% yield. LCâMS: [M + H]<sup>+</sup>, 335.10.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-(Furan-2-ylmethyl)-1-(((methylamino)oxy)carbonyl)-8-phenylimidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>9</b>)</h4><div class="NLM_p last">Methylamine hydrochloride (4 mg, 0.058 mmol) and diisopropylethylamine (50 Î¼L, 0.29 mmol) were added to a solution of compound <b>47</b> (10 mg, 0.029 mmol) in DMF (1 mL). The reaction mixture was stirred at rt for 10 min and then HATU (11 mg, 0.029 mmol) was added. The reaction mixture was allowed to warm to rt and stirred at rt overnight. The mixture was concentrated, and the residue was purified by HPLC to afford the title compound <b>9</b>(9 mg, 0.026 mmol) in 90% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 9.22 (s, 1H), 7.55â7.43 (m, 3H), 7.38â7.30 (m, 3H), 6.40 (s, 1H), 6.36 (s, 1H), 6.31 (br s, 1H), 4.86 (s, 2H), 2.54 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 348.13.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Ethyl 5-(Benzylamino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>10</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.80 (d, <i>J</i> = 5.9 Hz, 1H), 8.78 (s, 1H), 7.49â7.22 (m, 11H), 4.80 (d, <i>J</i> = 5.5 Hz, 2H), 3.68 (q, <i>J</i> = 7.1 Hz, 2H), 0.74 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 373.16.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Ethyl 5-((2-Methoxybenzyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>11</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.71 (s, 1H), 7.46â7.33 (m, 7H), 7.33â7.26 (m, 1H), 7.03 (dd, <i>J</i> = 8.3, 1.1 Hz, 1H), 6.94 (td, <i>J</i> = 7.5, 1.1 Hz, 1H), 4.84 (s, 2H), 3.91 (s, 3H), 3.80 (q, <i>J</i> = 7.2 Hz, 2H), 0.86 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 403.16.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Ethyl 5-((2-Fluoro-6-methoxybenzyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>12</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.70 (s, 1H), 7.48â7.31 (m, 7H), 6.90 (dt, <i>J</i> = 8.4, 0.9 Hz, 1H), 6.80 (ddd, <i>J</i> = 9.3, 8.4, 0.9 Hz, 1H), 4.88 (s, 2H), 3.91 (s, 3H), 3.79 (q, <i>J</i> = 7.1 Hz, 2H), 0.86 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 421.16.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Ethyl 5-((2-Fluoro-5-methoxybenzyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>13</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.70 (s, 1H), 7.50â7.29 (m, 6H), 7.15â6.98 (m, 2H), 6.91â6.77 (m, 1H), 4.86 (s, 2H), 3.85â3.77 (m, 2H), 3.76 (s, 3H), 0.86 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 421.15.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Ethyl 5-((2-Fluoro-4-methoxybenzyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>14</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.78 (s, 1H), 8.69 (s, 1H), 7.51â7.25 (m, 7H), 6.86 (dd, <i>J</i> = 12.2, 2.5 Hz, 1H), 6.77 (dd, <i>J</i> = 8.5, 2.5 Hz, 1H), 4.74 (s, 2H), 3.85â3.77 (m, 2H), 3.76 (s, 3H), 0.74 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 421.15.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Ethyl 5-((Benzo[<i>d</i>][1,3]dioxol-4-ylmethyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>15</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.70 (s, 1H), 7.47â7.31 (m, 6H), 6.95 (ddt, <i>J</i> = 7.7, 1.4, 0.7 Hz, 1H), 6.88â6.75 (m, 2H), 5.99 (s, 2H), 4.84 (s, 2H), 3.80 (q, <i>J</i> = 7.1 Hz, 2H), 0.86 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 417.14.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Ethyl 5-(((2,3-Dihydrobenzofuran-7-yl)methyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>16</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.80 (s, 1H), 8.66 (t, <i>J</i> = 5.5 Hz, 1H), 7.44â7.26 (m, 6H), 7.15 (t, <i>J</i> = 8.5 Hz, 2H), 6.80 (t, <i>J</i> = 7.5 Hz, 1H), 4.70 (d, <i>J</i> = 5.4 Hz, 2H), 4.59 (t, <i>J</i> = 8.8 Hz, 2H), 3.68 (q, <i>J</i> = 7.1 Hz, 2H), 3.21 (t, <i>J</i> = 8.7 Hz, 2H), 0.74 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 415.17.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Ethyl 5-(((2,3-Dihydrobenzofuran-4-yl)methyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>17</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.80 (s, 1H), 8.70 (t, <i>J</i> = 5.5 Hz, 1H), 7.48â7.25 (m, 6H), 7.07 (t, <i>J</i> = 7.8 Hz, 1H), 6.87 (d, <i>J</i> = 7.7 Hz, 1H), 6.69 (d, <i>J</i> = 7.8 Hz, 1H), 4.72 (d, <i>J</i> = 5.3 Hz, 2H), 4.55 (t, <i>J</i> = 8.7 Hz, 2H), 3.68 (q, <i>J</i> = 7.1 Hz, 2H), 3.25 (t, <i>J</i> = 8.7 Hz, 2H), 0.74 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 415.17.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Ethyl 5-(((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxylate (<b>18</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.70 (s, 1H), 7.50â7.25 (m, 6H), 6.91â6.83 (m, 1H), 6.67 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.84 (d, <i>J</i> = 1.1 Hz, 2H), 4.59 (t, <i>J</i> = 8.7 Hz, 2H), 3.80 (q, <i>J</i> = 7.2 Hz, 2H), 3.44â3.35 (m, 2H), 0.86 (t, <i>J</i> = 7.1 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 433.15.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 5-(((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)amino)-<i>N</i>-methyl-8-phenylimidazo[1,5-<i>c</i>]pyrimidine-1-carboxamide (<b>19</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.64 (s, 1H), 7.50â7.29 (m, 6H), 6.92â6.81 (m, 1H), 6.67 (dd, <i>J</i> = 8.7, 3.9 Hz, 1H), 4.83 (d, <i>J</i> = 1.2 Hz, 2H), 4.59 (t, <i>J</i> = 8.7 Hz, 2H), 3.38 (t, <i>J</i> = 8.7 Hz, 2H), 2.51 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 418.16.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Methyl 5-Bromo-2-(methylthio)pyrimidine-4-carboxylate (<b>49</b>)</h4><div class="NLM_p last">Acetyl chloride (3.1 mL, 43.8 mmol) was added dropwise to MeOH (50 mL) at 0â5 Â°C. The resulting mixture was stirred at this temperature for 5 min; then, 5-bromo-2-(methylthio)pyrimidine-4-carboxylic acid (5.5 g, 22.2 mmol) was added. The reaction mixture was heated to reflux for 1 h and then cooled to rt. The reaction mixture was poured into satd. aq. NaHCO<sub>3</sub> solution (100 mL). The mixture was extracted with DCM (3 Ã 100 mL), washed with H<sub>2</sub>O (50 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated, and the residue was recrystallized from petroleum ether to give the title compound <b>49</b> as a yellow solid (4.6 g, 34.6 mmol) in 79% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.71 (s, 1H), 4.0 (s, 3H), 2.59 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 263.10.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (5-Bromo-2-(methylthio)pyrimidin-4-yl)methanol (<b>50</b>)</h4><div class="NLM_p last">DIBAL-H (4.6 mL, 4.60 mmol, 1 M in cyclohexane) was added dropwise to a solution of methyl 5-bromo-2-(methylthio)pyrimidine-4-carboxylate (<b>49</b>, 600 mg, 2.28 mmol) in DCM (15 mL) at â60 Â°C. The reaction mixture was maintained at â60 to â15 Â°C for 30 min and then was allowed to rise to rt and stirred for another 12 h. The reaction mixture was cooled to 0 Â°C and was quenched with satd. aq NH<sub>4</sub>Cl (50 mL). The resulting mixture was extracted with DCM (3 Ã 100 mL), washed with brine (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by column chromatography (eluting with 0â50% EtOAc/hexane) to afford the title compound <b>50</b> as a yellow solid (300 mg) in 56% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.52 (s, 1H), 4.72 (d, <i>J</i> = 4.8 Hz, 2H), 3.85 (t, <i>J</i> = 4.9 Hz, 1H), 2.61 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 235.02.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 2-((5-Bromo-2-(methylthio)pyrimidin-4-yl)methyl)isoindoline-1,3-dione (<b>51</b>)</h4><div class="NLM_p last">A solution in THF (10 mL) of (5-bromo-2-(methylthio)pyrimidin-4-yl)methanol (<b>50</b>, 1.69 g, 7.22 mmol), phthalimide (1.27 g, 8.66 mmol), and triphenylphosphine (2.19 g, 10.84 mmol) was mixed with DIAD (1.88 g, 10.84 mmol) under cooling with ice and stirred at rt overnight. After completion of the reaction, the mixture was diluted with EtOAc, and the organic layer was washed with saturated brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated under reduced pressure. The residue was purified by column chromatography (eluting with 0â50% EtOAc/hexane) to afford the desired product <b>51</b> (479 mg) in 70% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.51 (s, 1H), 7.95 (dd, <i>J</i> = 5.4, 3.1 Hz, 2H), 7.85â7.76 (m, 2H), 5.02 (s, 2H), 2.05 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 363.12.</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (5-Bromo-2-(methylthio)pyrimidin-4-yl)methanamine (<b>52</b>)</h4><div class="NLM_p last">Compound <b>51</b> (910 mg, 2.51 mmol) in EtOH (10 mL) was stirred with NH<sub>2</sub>NH<sub>2</sub>Â·H<sub>2</sub>O (0.157 mL, 5.02 mmol) at rt for 4 h. After completion of the reaction, the solid was filtered and washed with EtOH, and the filtrate was evaporated under reduced pressure. It was purified by reverse phase chromatography (eluting with 0â20% CH<sub>3</sub>CN/H<sub>2</sub>O) to afford the title compound <b>52</b> as a colorless liquid in 80% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.49 (s, 1H), 4.02 (s, 2H), 2.59 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 236.10.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i>-((5-Bromo-2-(methylthio)pyrimidin-4-yl)methyl)formamide (<b>53</b>)</h4><div class="NLM_p last">A mixture of HCO<sub>2</sub>H (4 mL) and Ac<sub>2</sub>O (4 mL) was heated at 50 Â°C for 1 h. The reaction mixture was cooled to rt and added to a solution of compound <b>52</b> (1.52 g, 6.55 mmol) in DCM (20 mL). The mixture was stirred at rt for 2 h. After completion of the reaction, the mixture was concentrated, then extracted with DCM (2 Ã 50 mL) and washed successively with H<sub>2</sub>O (20 mL) and brine (10 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford the crude title compound <b>53</b> as an oil, which was used in the next steps without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.56 (s, 1H), 8.40 (q, <i>J</i> = 1.2 Hz, 1H), 6.99 (s, 1H), 4.63 (dd, <i>J</i> = 4.6, 1.0 Hz, 2H), 2.59 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 262.12.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 8-Bromo-5-(methylthio)imidazo[1,5-<i>c</i>]pyrimidine (<b>54</b>)</h4><div class="NLM_p last">POCl<sub>3</sub> (0.43 mL, 4.60 mmol) was added dropwise to a solution of compound <b>53</b> (800 mg, 3.06 mmol) in dioxane (30 mL). The reaction mixture was heated under reflux for 2 h. IceâH<sub>2</sub>O (50 mL) was added, and the mixture was adjusted to pH 8 with satd. aq NaHCO<sub>3</sub>. The mixture was extracted with DCM (4 Ã 50 mL), washed with brine (30 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and filtered. The filtrate was concentrated, and the residue was purified by column chromatography (eluting with 0â50% EtOAc/hexane) to afford the title compound <b>54</b> as a yellow solid (350 mg, 47.0%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.20 (s, 1H), 7.55 (d, <i>J</i> = 2.0 Hz, 2H), 2.77 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 244.01.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 8-Bromo-<i>N</i>-(furan-2-ylmethyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>55</b>)</h4><div class="NLM_p last"><i>m</i>-CPBA (464 mg, 2.7 mmol, â¤77%, 1.5 equiv) was added at 0 Â°C to a solution of compound <b>54</b> (439 mg, 1.8 mmol, 1.0 equiv) in DCM (18 mL). After 45 min, Et<sub>3</sub>N (1 mL, 7.6 mmol, 4 equiv) was added at 0 Â°C and stirred for 2 min, followed by addition of furan-2-ylmethanamine (175 mg, 1.8 mmol). The reaction mixture was then stirred at rt for 3 h. Then, the reaction mixture was concentrated and the residue was purified by silica gel column chromatography (eluting with 50â100% EtOAc/hexane) to afford the title compound <b>55</b> (274 mg, 0.93 mmol) in 52% yield. LCâMS: [M + H]<sup>+</sup>, 294.12.</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-(Furan-2-ylmethyl)-8-phenylimidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>7</b>)</h4><div class="NLM_p last">Potassium carbonate (227 mg, 1.64 mmol), phenylboronic acid (168 mg, 0.82 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (63 mg, 0.055 mmol) were added to a solution of compound <b>55</b> (160 mg, 0.55 mmol) in a mixed solvent (dioxane/H<sub>2</sub>O = 10:2.5 mL). The resulting mixture was stirred under N<sub>2</sub> at 90 Â°C overnight. The mixture was then cooled to rt, and the solvent was removed in vacuo. The residue was purified with silica gel chromatography (eluting with 0â10% MeOH/DCM) to afford title compound <b>7</b> (159 mg, 0.34 mmol) in 80% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 10.13 (s, 1H), 7.66 (s, 1H), 7.61 (d, <i>J</i> = 1.0 Hz, 1H), 7.57â7.51 (m, 4H), 7.51â7.45 (m, 1H), 7.37 (dd, <i>J</i> = 1.8, 0.8 Hz, 1H), 6.41 (dd, <i>J</i> = 3.2, 0.8 Hz, 1H), 6.34 (dd, <i>J</i> = 3.2, 1.8 Hz, 1H), 4.89 (s, 2H). LCâMS: [M + H]<sup>+</sup>, 291.11.</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 8-Bromo-<i>N</i>-((5-fluoro-2,3-dihydrobenzofuran-4-yl)methyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>56</b>)</h4><div class="NLM_p last">A mixture of compound <b>54</b> (1.0 g, 4.1 mmol) and (5-fluoro-2,3-dihydrobenzofuran-4-yl)methanamine (1.41 g, 8.2 mmol) was heated at 40 Â°C and stirred for 24 h. After cooling to rt, the crude mixture was purified by reverse phase chromatography (eluting with 0â70% CH<sub>3</sub>CN/H<sub>2</sub>O) to afford the title compound <b>56</b> as a yellow solid in 60% yield. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.58 (s, 1H), 7.38 (s, 1H), 7.32 (s, 1H), 6.91â6.80 (m, 1H), 6.65 (dd, <i>J</i> = 8.7, 3.9 Hz, 1H), 4.74 (s, 2H), 4.62â4.52 (m, 2H), 3.37 (s, 2H). LCâMS: [M + H]<sup>+</sup>, 363.01.</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-phenylimidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>57</b>)</h4><div class="NLM_p last">Potassium carbonate (227 mg, 1.64 mmol), phenylboronic acid (168 mg, 0.82 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (63 mg, 0.055 mmol) were added to a solution of compound <b>56</b> (200 mg, 0.55 mmol) in a mixed solvent (dioxane/H<sub>2</sub>O = 10:2.5 mL). The resulting mixture was stirred under N<sub>2</sub> at 90 Â°C overnight. The mixture was then cooled to rt, and the solvent was removed in vacuo. The residue was purified with silica gel chromatography (eluting with 0â10% MeOH/DCM) to afford title compound <b>57</b> (174 mg, 0.34 mmol) in 88% yield. LCâMS: [M + H]<sup>+</sup>, 361.13.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-iodo-8-phenylimidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>58</b>)</h4><div class="NLM_p last">NIS (278 mg, 1.24 mmol) was added to a solution of compound <b>57</b> (500 mg, 1.38 mmol) in DMF (10 mL) at 0 Â°C and stirred at rt for 15 min. The mixture was extracted with DCM (4 Ã 50 mL), washed with brine (30 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), and filtered. The filtrate was concentrated, and the residue was purified by column chromatography (eluting with 20â50% EtOAc/hexane) to afford the title compound <b>58</b> as a yellow solid (610 mg, 1.26 mmol, 70%). LCâMS: [M + H]<sup>+</sup>, 487.03.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-phenyl-1-(trifluoromethyl)-imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>20</b>)</h4><div class="NLM_p last">A solution of compound <b>58</b> (50 mg, 0.1 mmol) in DMF (5 mL) was added to a mixture of cuprous iodide (190 mg, 1.0 mmol), methyl 2,2-difluoro-2-(fluorosulfonyl)acetate (192 mg, 1.0 mmol), and PdCl<sub>2</sub>(dppf)Cl<sub>2</sub> (7 mg, 0.01 mmol). The reaction was stirred at 90 Â°C overnight, then cooled down to rt, and quenched by being poured into H<sub>2</sub>O. The mixture was filtered and the filtrate was extracted with Et<sub>2</sub>O. The ethereal extract was concentrated, and the residue was purified by HPLC to afford the title compound <b>20</b> (16 mg, 0.04 mmol) in 42% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.85 (s, 1H), 8.61 (s, 1H), 7.44â7.31 (m, 6H), 6.94 (t, <i>J</i> = 8.8 Hz, 1H), 6.71 (dd, <i>J</i> = 8.4, 4.0 Hz, 1H), 4.73 (d, 2H), 4.54 (t, <i>J</i> = 8.8 Hz, 2H), 3.31 (t, <i>J</i> = 8.8 Hz, 2H). LCâMS: [M + H]<sup>+</sup>, 429.12.</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)-8-phenylimidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>21</b>)</h4><div class="NLM_p last">A mixture of compound <b>58</b> (50 mg, 0.1 mmol), MeSO<sub>2</sub>Na (30 mg. 0.3 mmol), and CuI (57 mg. 0.3 mmol) in DMSO (2 mL) was bubbled with N<sub>2</sub> for 5 min, the tube was sealed, and then heated in a microwave reactor at 120 Â°C for 20 min and then at 100 Â°C for 3 h. The mixture was concentrated, and the residue was purified by HPLC to afford the title compound <b>21</b> (21 mg, 0.05 mmol) in 50% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.60 (s, 1H), 7.55â7.35 (m, 6H), 6.77 (dd, <i>J</i> = 10.1, 8.7 Hz, 1H), 6.61 (dd, <i>J</i> = 8.7, 3.9 Hz, 1H), 4.82 (s, 2H), 4.61 (t, <i>J</i> = 8.7 Hz, 2H), 3.40 (t, <i>J</i> = 8.7 Hz, 2H), 2.86 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 439.12.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (5-(((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)amino)-8-phenylimidazo[1,5-<i>c</i>]pyrimidin-1-yl)dimethylphosphine Oxide (<b>22</b>)</h4><div class="NLM_p last">Dimethylphosphine oxide (22 mg, 0.3 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (9 mg, 0.01 mmol), xantphos (6 mg, 0.01 mmol), and Et<sub>3</sub>N (0.2 mL) were added to a solution of compound <b>58</b> (50 mg, 0.1 mmol) in dioxane (2 mL). The mixture was purged with argon and heated at 100 Â°C overnight. The mixture was concentrated, and the residue was purified by HPLC to afford the title compound <b>22</b> (17 mg, 0.04 mmol) in 40% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.85 (s, 1H), 8.52 (s, 1H), 7.58â7.56 (m, 2H), 7.42â7.41 (m, 2H), 7.29 (s, 1H), 6.94 (t, <i>J</i> = 9.2 Hz, 1H), 6.70 (dd, <i>J</i> = 8.4, 4.0 Hz, 1H), 4.73 (d, <i>J</i> = 4.0 Hz, 2H), 4.54 (t, <i>J</i> = 8.8 Hz, 2H), 3.31 (t, <i>J</i> = 8.8 Hz, 2H), 1.21 (s, 1H), 1.18 (s, 1H). LCâMS: [M + H]<sup>+</sup>, 437.14.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)-8-(4(methyl-sulfonyl)phenyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>23</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.12 (s, 1H), 8.03 (d, <i>J</i> = 8.4 Hz, 2H), 7.68 (d, <i>J</i> = 8.4 Hz, 2H), 7.49 (s, 1H), 6.95â6.83 (m, 1H), 6.72 (dd, <i>J</i> = 8.7, 4.0 Hz, 1H), 5.68 (s, 1H), 4.88 (d, <i>J</i> = 4.9 Hz, 2H), 4.66 (t, <i>J</i> = 8.7 Hz, 2H), 3.45 (t, <i>J</i> = 8.7 Hz, 2H), 3.16 (s, 3H), 3.14 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 517.09.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-(4-fluorophenyl)-1(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>24</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.68 (s, 1H), 7.52â7.31 (m, 3H), 7.18â7.04 (m, 2H), 6.92â6.81 (m, 1H), 6.66 (dd, <i>J</i> = 8.6, 3.8 Hz, 1H), 4.84 (d, <i>J</i> = 1.0 Hz, 2H), 4.59 (t, <i>J</i> = 8.7 Hz, 2H), 3.39 (t, <i>J</i> = 8.7 Hz, 2H), 3.00 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 457.10.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-(2-methyl-4-(methylsulfonyl)phenyl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>25</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.92 (s, 1H), 8.73 (s, 1H), 7.80 (d, <i>J</i> = 1.9 Hz, 1H), 7.73 (dd, <i>J</i> = 7.9, 2.0 Hz, 1H), 7.45 (d, <i>J</i> = 7.9 Hz, 1H), 7.36 (s, 1H), 6.96 (dd, <i>J</i> = 10.3, 8.6 Hz, 1H), 6.72 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.75 (q, <i>J</i> = 14.4 Hz, 2H), 4.56 (t, <i>J</i> = 8.7 Hz, 2H), 3.41â3.29 (m, 2H), 3.23 (s, 3H), 2.99 (s, 3H), 2.20 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 531.10.</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-(2-methylpyridin-3-yl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>26</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.78 (d, <i>J</i> = 1.5 Hz, 1H), 8.73 (dd, <i>J</i> = 5.9, 1.8 Hz, 1H), 8.44 (dt, <i>J</i> = 7.9, 1.5 Hz, 1H), 7.95 (dd, <i>J</i> = 7.8, 6.0 Hz, 1H), 7.59 (d, <i>J</i> = 0.9 Hz, 1H), 6.88 (td, <i>J</i> = 9.5, 9.1, 1.9 Hz, 1H), 6.67 (dd, <i>J</i> = 8.6, 4.0 Hz, 1H), 4.88 (d, <i>J</i> = 4.9 Hz, 2H), 4.60 (td, <i>J</i> = 8.7, 1.5 Hz, 2H), 3.43 (t, <i>J</i> = 8.7 Hz, 2H), 3.11â3.00 (m, 3H), 2.63 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 454.12.</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-(6-methylpyridin-3-yl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>27</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.87â8.80 (m, 1H), 8.76 (d, <i>J</i> = 0.9 Hz, 1H), 8.51 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.95 (d, <i>J</i> = 8.3 Hz, 1H), 7.62 (d, <i>J</i> = 0.8 Hz, 1H), 6.91â6.82 (m, 1H), 6.66 (dt, <i>J</i> = 9.0, 3.4 Hz, 1H), 4.85 (s, 2H), 4.60 (td, <i>J</i> = 8.7, 1.4 Hz, 2H), 3.41 (t, <i>J</i> = 8.7 Hz, 2H), 3.18 (s, 3H), 2.86 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 454.12.</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 8-(6-Cyclopropylpyridin-3-yl)-<i>N</i>-((5-fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>28</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.92 (s, 1H), 8.86 (s, 1H), 8.58 (s, 1H), 8.01 (s, 1H), 7.48 (d, <i>J</i> = 11.0 Hz, 2H), 6.96 (dd, <i>J</i> = 10.3, 8.7 Hz, 1H), 6.72 (dd, <i>J</i> = 8.7, 3.9 Hz, 1H), 4.75 (d, <i>J</i> = 4.6 Hz, 2H), 4.56 (t, <i>J</i> = 8.7 Hz, 2H), 3.32 (t, <i>J</i> = 8.7 Hz, 2H), 3.11 (s, 3H), 2.26 (dd, <i>J</i> = 9.1, 4.6 Hz, 1H), 1.25â1.11 (m, 2H), 1.07 (t, <i>J</i> = 3.6 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 159.7, 156.9, 156.2, 154.5, 143.5, 141.9, 130.4, 130.3, 129.7, 126.8, 121.9, 121.8, 121.3, 114.5, 114.3, 108.9, 108.8, 72.0, 43.9, 37.9, 37.8, 29.0, 15.6, 11.5. LCâMS: [M + H]<sup>+</sup>, 480.14.</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 8-(6-(Difluoromethyl)pyridin-3-yl)-<i>N</i>-((5-fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>29</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.74 (s, 1H), 8.73â8.67 (m, 1H), 8.06 (d, <i>J</i> = 2.2 Hz, 1H), 7.77 (d, <i>J</i> = 8.0 Hz, 1H), 7.54 (s, 1H), 6.96â6.77 (m, 2H), 6.73â6.61 (m, 1H), 4.86 (d, <i>J</i> = 1.1 Hz, 2H), 4.60 (t, <i>J</i> = 8.7 Hz, 2H), 3.41 (t, <i>J</i> = 8.7 Hz, 2H), 3.11 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 490.10.</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)-8-(6-(trifluoromethyl)pyridin-3-yl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>30</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.74 (d, <i>J</i> = 2.1 Hz, 1H), 8.39 (s, 1H), 8.05â7.91 (m, 1H), 7.78 (dd, <i>J</i> = 8.1, 0.7 Hz, 1H), 7.49 (s, 1H), 6.89â6.72 (m, 1H), 6.65 (dd, <i>J</i> = 8.7, 3.9 Hz, 1H), 6.46 (s, 1H), 4.85 (s, 2H), 4.63 (t, <i>J</i> = 8.7 Hz, 2H), 3.41 (t, <i>J</i> = 8.7 Hz, 2H), 3.08 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 508.09.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)-8-(6-(methylsulfonyl)pyridin-3-yl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>31</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.79 (dd, <i>J</i> = 2.0, 1.0 Hz, 1H), 8.75 (s, 1H), 8.23â8.01 (m, 2H), 7.57 (s, 1H), 6.93â6.80 (m, 1H), 6.67 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.84 (d, <i>J</i> = 1.0 Hz, 2H), 4.60 (t, <i>J</i> = 8.7 Hz, 2H), 3.41 (t, <i>J</i> = 8.8 Hz, 2H), 3.28 (s, 3H), 3.14 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 518.08.</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 8-(2,6-Dimethylpyridin-3-yl)-<i>N</i>-((5-fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>32</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.54 (s, 1H), 8.05 (d, <i>J</i> = 8.0 Hz, 1H), 7.53 (d, <i>J</i> = 8.0 Hz, 1H), 7.33 (s, 1H), 6.65 (t, <i>J</i> = 9.2 Hz, 1H), 6.44 (dd, <i>J</i> = 8.6, 3.6 Hz, 1H), 4.63 (t, <i>J</i> = 6.4 Hz, 2H), 4.37 (t, <i>J</i> = 8.8 Hz, 1H), 3.20 (t, <i>J</i> = 8.8 Hz, 2H), 2.85 (s, 3H), 2.60 (s, 3H), 2.36 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 156.9, 156.1, 154.5, 154.2, 152.2, 147.2, 143.9, 142.1, 133.3, 130.1, 129.8, 127.3, 124.7, 121.8, 121.6, 114.5, 114.2, 108.8, 108.3, 72.0, 43.8, 37.9, 29.0, 19.4, 18.7. LCâMS: [M + H]<sup>+</sup>, 468.14.</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-(2-methyl-6-(trifluoromethyl)pyridin-3-yl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>33</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.93 (s, 1H), 8.79 (t, <i>J</i> = 5.0 Hz, 1H), 7.84 (d, <i>J</i> = 7.8 Hz, 1H), 7.72 (d, <i>J</i> = 7.9 Hz, 1H), 7.46 (s, 1H), 6.97 (dd, <i>J</i> = 10.3, 8.6 Hz, 1H), 6.73 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.84â4.66 (m, 2H), 4.57 (t, <i>J</i> = 8.7 Hz, 2H), 3.40â3.29 (m, 2H), 3.02 (s, 3H), 2.35 (s, 3H). LCâMS: [M + H]<sup>+</sup>, 522.11.</div></div><div id="sec5_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-(1-methyl-1H-pyrazol-4-yl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>34</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.63 (s, 1H), 7.76 (s, 1H), 7.59 (s, 1H), 7.44 (s, 1H), 6.85 (t, <i>J</i> = 9.6 Hz, 1H), 6.64 (dd, <i>J</i> = 8.4, 3.6 Hz, 1H), 4.81 (s, 2H), 4.57 (t, <i>J</i> = 8.4 Hz, 2H), 3.94 (s, 3H), 3.36 (t, <i>J</i> = 8.8 Hz, 2H), 3.00 (s, 3H), 2.66 (s, 2H). LCâMS: [M + H]<sup>+</sup>, 443.12.</div></div><div id="sec5_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-(tetrahydro-2H-pyran-4-yl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>59</b>)</h4><div class="NLM_p last">3,6-Dihydro-2<i>H</i>-pyran-4-boronic acid pinacol ester (290 mg, 1.38 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (80 mg, 0.069 mmol), and Na<sub>2</sub>CO<sub>3</sub> (285 mg) were added to a solution (dioxane/H<sub>2</sub>O = 10:2.5 mL) of compound <b>56</b> (250 mg, 0.69 mmol). The resulting mixture was stirred under N<sub>2</sub> at 90 Â°C overnight. The mixture was then cooled to rt, and the solvent was removed in vacuo and purified by column chromatography (eluting with 0â5% MeOH/DCM) to obtain the title compound <b>59</b> as a white solid in 80% yield. LCâMS: [M + H]<sup>+</sup>, 367.14.</div></div><div id="sec5_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-8-(tetrahydro-2<i>H</i>-pyran-4-yl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>60</b>)</h4><div class="NLM_p last">Pd/C (100 mg, 10% wt) was added to a solution of compound <b>59</b> (1 g, 2.70 mmol) in MeOH (25 mL). The reaction mixture was degassed with H<sub>2</sub> and stirred under an H<sub>2</sub> atmosphere for 6 h at rt. The mixture was then filtered through Celite and washed with MeOH. Concentration under reduced pressure yielded the desired product. LCâMS: [M + H]<sup>+</sup>, 369.17.</div></div><div id="sec5_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-iodo-8-(tetrahydro-2<i>H</i>-pyran-4-yl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>61</b>)</h4><div class="NLM_p last">NIS (82 mg, 0.4 mmol) was added at 0 Â°C to a solution of compound <b>60</b> (160 mg, 0.43 mmol) in DMF (4 mL) and stirred at rt for 15 min. The mixture was extracted with DCM (4 Ã 50 mL), washed with brine (30 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), and filtered. The filtrate was concentrated, and the residue was purified by column chromatography (eluting with 20â50% EtOAc/hexane) to afford the title compound <b>61</b> in 60% yield (127 mg, 0.25 mmol, 70%). LCâMS: [M + H]<sup>+</sup>, 495.04.</div></div><div id="sec5_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)-8-(tetrahydro-2<i>H</i>-pyran-4-yl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>35</b>)</h4><div class="NLM_p last">A mixture of compound <b>61</b> (50 mg, 0.1 mmol), MeSO<sub>2</sub>Na (30 mg. 0.3 mmol), and CuI (57 mg. 0.3 mmol) in DMSO (2 mL) was bubbled with N<sub>2</sub> for 5 min and the sealed tube was then heated in a microwave reactor at 120 Â°C for 20 min and then at 100 Â°C for 3 h. The mixture was concentrated, and the residue was purified by HPLC to afford the title compound <b>35</b> (21 mg, 0.05 mmol) in 50% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.80 (s, 1H), 8.44 (t, <i>J</i> = 5.1 Hz, 1H), 7.49 (s, 1H), 6.94 (dd, <i>J</i> = 10.3, 8.7 Hz, 1H), 6.70 (dd, <i>J</i> = 8.7, 3.9 Hz, 1H), 4.68 (d, <i>J</i> = 4.4 Hz, 2H), 4.54 (t, <i>J</i> = 8.7 Hz, 2H), 3.95 (dd, <i>J</i> = 11.1, 3.8 Hz, 2H), 3.66 (ddd, <i>J</i> = 11.7, 8.4, 3.3 Hz, 1H), 3.40 (d, <i>J</i> = 1.9 Hz, 1H), 3.36 (s, 3H), 3.29 (t, <i>J</i> = 8.7 Hz, 2H), 1.81 (d, <i>J</i> = 13.6 Hz, 2H), 1.67 (qd, <i>J</i> = 12.3, 4.2 Hz, 2H). LCâMS: [M + H]<sup>+</sup>, 447.14.</div></div><div id="sec5_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 4-(5-(((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)amino)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-8-yl)tetrahydro-2<i>H</i>-thiopyran 1,1-Dioxide (<b>36</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.83 (s, 1H), 8.51 (t, <i>J</i> = 5.1 Hz, 1H), 7.54 (d, <i>J</i> = 0.7 Hz, 1H), 6.93 (dd, <i>J</i> = 10.3, 8.7 Hz, 1H), 6.70 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.67 (d, <i>J</i> = 4.3 Hz, 2H), 4.55 (t, <i>J</i> = 8.7 Hz, 2H), 3.72â3.59 (m, 1H), 3.38 (s, 3H), 3.31 (t, <i>J</i> = 8.7 Hz, 2H), 3.27â3.06 (m, 4H), 2.31â2.19 (m, 2H), 2.19â2.04 (m, 2H). LCâMS: [M + H]<sup>+</sup>, 495.10.</div></div><div id="sec5_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>N</i>-((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)-8-(1-(methylsulfonyl)piperidin-4-yl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>37</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.81 (s, 1H), 8.46 (s, 1H), 7.54 (d, <i>J</i> = 0.7 Hz, 1H), 6.94 (dd, <i>J</i> = 10.3, 8.7 Hz, 1H), 6.70 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.68 (d, <i>J</i> = 3.9 Hz, 2H), 4.54 (t, <i>J</i> = 8.7 Hz, 2H), 3.73â3.65 (m, 2H), 3.59â3.45 (m, 1H), 3.37 (s, 3H), 3.29 (t, <i>J</i> = 8.7 Hz, 2H), 2.91 (s, 3H), 2.76 (td, <i>J</i> = 12.1, 2.3 Hz, 2H), 2.00 (d, <i>J</i> = 12.6 Hz, 2H), 1.69 (qd, <i>J</i> = 12.4, 3.9 Hz, 2H). LCâMS: [M + H]<sup>+</sup>, 524.13.</div></div><div id="sec5_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 8-(1-Ethylpiperidin-4-yl)-<i>N</i>-((5-fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>38</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.85 (s, 1H), 8.55 (br s, 1H), 7.45 (d, <i>J</i> = 0.7 Hz, 1H), 6.94 (dd, <i>J</i> = 10.3, 8.7 Hz, 1H), 6.70 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.69 (d, <i>J</i> = 2.8 Hz, 2H), 4.54 (t, <i>J</i> = 8.7 Hz, 2H), 3.62 (dd, <i>J</i> = 13.8, 10.7 Hz, 3H), 3.38 (s, 3H), 3.29 (t, <i>J</i> = 8.7 Hz, 2H), 3.26â3.10 (m, 2H), 2.94 (q, <i>J</i> = 11.2 Hz, 2H), 2.16 (d, <i>J</i> = 14.0 Hz, 2H), 1.87 (dd, <i>J</i> = 14.6, 11.2 Hz, 2H), 1.25 (t, <i>J</i> = 7.3 Hz, 3H). LCâMS: [M + H]<sup>+</sup>, 474.18.</div></div><div id="sec5_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 1-(4-(5-(((5-Fluoro-2,3-dihydrobenzofuran-4-yl)methyl)amino)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-8-yl)piperidin-1-yl)ethan-1-one (<b>39</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.81 (s, 1H), 8.46 (s, 1H), 7.46 (d, <i>J</i> = 0.7 Hz, 1H), 6.93 (dd, <i>J</i> = 10.3, 8.6 Hz, 1H), 6.69 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.67 (s, 2H), 4.54 (t, <i>J</i> = 8.7 Hz, 3H), 3.93 (d, <i>J</i> = 13.6 Hz, 1H), 3.71â3.57 (m, 1H), 3.37 (s, 3H), 3.29 (t, <i>J</i> = 8.7 Hz, 2H), 3.15â2.97 (m, 1H), 2.50â2.49 (m, 1H), 2.02 (s, 3H), 1.99â1.81 (m, 2H), 1.51 (dqd, <i>J</i> = 41.9, 12.4, 4.0 Hz, 2H). LCâMS: [M + H]<sup>+</sup>, 488.16.</div></div><div id="sec5_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 8-(1-(Cyclopropylsulfonyl)piperidin-4-yl)-<i>N</i>-((5-fluoro-2,3-dihydrobenzofuran-4-yl)methyl)-1-(methylsulfonyl)imidazo[1,5-<i>c</i>]pyrimidin-5-amine (<b>40</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.81 (s, 1H), 8.47 (s, 1H), 7.53 (d, <i>J</i> = 0.6 Hz, 1H), 6.93 (dd, <i>J</i> = 10.3, 8.7 Hz, 1H), 6.69 (dd, <i>J</i> = 8.6, 3.9 Hz, 1H), 4.68 (s, 2H), 4.54 (t, <i>J</i> = 8.7 Hz, 2H), 3.78â3.69 (m, 2H), 3.59â3.48 (m, 1H), 3.32 (s, 3H), 3.29 (t, <i>J</i> = 8.7 Hz, 2H), 2.88 (td, <i>J</i> = 12.2, 2.3 Hz, 2H), 2.71â2.60 (m, 1H), 2.00 (dd, <i>J</i> = 12.4, 3.1 Hz, 2H), 1.69 (qd, <i>J</i> = 12.5, 3.9 Hz, 2H), 1.06â0.89 (m, 4H). LCâMS: [M + H]<sup>+</sup>, 550.15.</div></div><div id="sec5_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 2-Bromo-4-(2,2-diethoxyethoxy)-1-fluorobenzene (<b>66</b>)</h4><div class="NLM_p last">K<sub>2</sub>CO<sub>3</sub> (109 g, 0.78 mol, 3 equiv) was added in one portion to a solution of 3-bromo-4-fluoro phenol (<b>65</b>, 50 g, 0.26 mol, 1 equiv) and 2-bromo-1,1-diethoxyethane (67 g, 0.34 mol, 1.3 equiv) in 250 Ml of DMF. The suspension was heated at 110 Â°C and stirred overnight under N<sub>2</sub>. After cooling to rt, the reaction was diluted with H<sub>2</sub>O and extracted with EtOAc (2 Ã 500 mL). The combined organic phase was washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The residue was purified on silica gel (0â10% EtOAc/hexane) to give the title compound <b>66</b> as a yellow oil (60.12 g, 196 mmol, 75% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 7.13 (d, 1H), 7.04 (dd, 1H), 6.84 (dd, 1H), 4.82 (t, 1H), 3.97 (d, 2H), 3.78 (q, 2H), 3.65 (q, 2H), 1.27 (t, 6H). LCâMS: [M + H]<sup>+</sup>, 307.02.</div></div><div id="sec5_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 4-Bromo-5-fluorobenzofuran (<b>67</b>) and 6-Bromo-5-fluorobenzofuran (<b>68</b>)</h4><div class="NLM_p last">To a solution of PPA (132.4 g, 0.39 mol) and toluene (300 mL) was added compound <b>66</b> (100 g, 0.35 mol) for over 30 min at 100 Â°C. The reaction mixture was heated at 100 Â°C for 4 h. After cooling to rt, 400 mL of ice H<sub>2</sub>O was added and the mixture was extracted with hexane twice. The combined organic phase was washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The residue was purified on silica gel (0â10% EtOAc/hexane) to give the title compound <b>67</b> and <b>68</b> as a isomeric mixture in 45% overall yield. LCâMS: [M + H]<sup>+</sup>, 214.94.</div></div><div id="sec5_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 5-Fluorobenzofuran-4-carbonitrile (<b>69</b>) and 5-Fluorobenzofuran-6-carbonitrile (<b>70</b>)</h4><div class="NLM_p last">Pd(PPh<sub>3</sub>)<sub>4</sub> (16.2 g, 14 mmol) was added to a solution of <b>67</b> and <b>68</b> (31 g, 0.144 mol) and Zn(CN)<sub>2</sub> (25.3 g, 0.216 mol) in 100 mL of DMF. The reaction mixture was degassed with N<sub>2</sub> and stirred under N<sub>2</sub> atmosphere for 24 h at 100 Â°C. After cooling to rt, H<sub>2</sub>O was added and the mixture was extracted with EtOAc (2 Ã 100 mL). The combined organic phase was washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The residue was purified on silica gel (0â20% EtOAc/hexane) to separate the desired isomer <b>69</b> as a white solid. The structure was confirmed using NMR. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): Î´ 8.10 (dd, 1H), 7.89 (dd, 1H), 7.30 (dd, 1H), 7.07 (d, 1H). LCâMS: [M + H]<sup>+</sup>, 162.02.</div></div><div id="sec5_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (5-Fluorobenzofuran-4-yl)methanamine (<b>71</b>)</h4><div class="NLM_p last">The desired isomer <b>69</b> (2.3 g, 14.55 mmol) in 10 mL of THF was treated with a 1 M solution of LAH in THF (36 mL) at 0 Â°C. Then, the temperature was increased to 50 Â°C and the reaction mixture was stirred overnight. After cooling to rt, the reaction was slowly quenched with satd. Na<sub>2</sub>SO<sub>4</sub> at 0 Â°C. It was then filtered and washed several times with EtOAc. Purification by flash chromatography (0â10% MeOH/DCM containing 1% triethylamine) gave the desired compound <b>71</b> (1.63 g, 10.1 mmol) in 70% yield. LCâMS: [M + H]<sup>+</sup>, 166.02.</div></div><div id="sec5_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> (5-Fluoro-2,3-dihydrobenzofuran-4-yl)methanamine (<b>72</b>)</h4><div class="NLM_p last">Pd/C (100 mg, 10% wt) was added to a solution of compound <b>71</b> (1 g, 6.02 mmol) in MeOH (50 mL). The reaction mixture was degassed with H<sub>2</sub> and stirred under an H<sub>2</sub> atmosphere for 6 h at 40 Â°C. The mixture was then filtered through Celite and washed with MeOH. Concentration under reduced pressure followed by purification by flash chromatography (0â10% MeOH/DCM containing 1% triethylamine) gave the desired compound <b>72</b> (859 mg, 5.11 mmol) in 85% yield. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): 6.81 (dd, 1H), 6.59 (dd, 1H), 4.56 (t, 2H), 3.77 (s, 2H), 3.27 (t, 2H). LCâMS: [M + H]<sup>+</sup>, 168.07.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> EED-H3K27me3 Peptide Competition Binding Assay by AlphaScreen</h3><div class="NLM_p last">To assess the potency of the compounds in the EED-H3K27me3 competitive binding assay, compounds were serially diluted in DMSO to obtain a total of 12 concentrations. Then, 2.5 Î¼L of a solution of compounds at each concentration were transferred into 384-well PerkinElmer OptiPlate-384 white plates. The solutions (5 Î¼L) containing 20 nM EED (1-441)-His protein in the buffer (25 mM HEPES, pH 8, 0.02% Tween-20, 0.5% BSA) was added to the wells and then incubated with the compound for 15 min. The solutions (2.5 Î¼L) containing 20 nM biotin-H3K27me3 (19â33) peptide in the buffer (25 mM HEPES, pH 8, 0.02% Tween-20, 0.5% BSA) were added to the wells and incubated with the compound for 30 min. An AlphaScreen detection beads mix was prepared immediately before use by mixing nickel chelate acceptor beads and streptavidin donor beads in a 1:1 ratio (PerkinElmer, product no. 6760619C/M/R) into the buffer described above. Then, 10 Î¼L of the detection beads mix was added to the plate, which was then incubated in dark at rt for 1 h. The final concentration of donor and acceptor beads was 10 Î¼g/mL in each case. Plates were read on a CLARIOStar plate reader (BMG Labtech) using the AlphaScreen setting adapted for optimal signal detection with a 615 nm filter, after sample excitation at 680 nm. The emission signal at 615 nm was used to quantify the inhibition of the compounds. AlphaScreen signals were normalized based on the reading coming from the positive (maximum signal control) and negative controls (minimum signal control) to give the percentage of activity remaining. The data were then fit to a doseâresponse equation to obtain the IC<sub>50</sub> values.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Cell Growth Inhibition</h3><div class="NLM_p last">The human B cell lymphoma cell KARPAS422 and Pfeiffer cell lines were purchased from the American Type Culture Collection (ATCC) and were cultured using standard cell culture conditions in RPMI-1640 (Invitrogen, cat #11875) supplemented with 10% FBS (Invitrogen, cat #10099-141) in a humidified incubator at 37 Â°C, 5% CO<sub>2</sub>. To assess the effect of EED inhibitors on cell growth, the cells were seeded in 96-well cell culture plates at a density of 2000â3000 cells/well in 200 Î¼L of culture medium and treated with serially diluted compounds for 7 days at 37 Â°C in an atmosphere of 5% CO<sub>2</sub>. Cell growth was evaluated by a lactate dehydrogenase-based WST-8 assay (Dojindo Molecular Technologies) using a Tecan Infinite M1000 multimode microplate reader (Tecan, Morrisville, NC). The WST-8 reagent was added to the plate, incubated for 1â4 h, and read at 450 nm. The readings were normalized to the DMSO-treated cells, and the IC<sub>50</sub> was calculated by nonlinear regression analysis using GraphPad Prism 6 software.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> PD and Efficacy Studies in Mice</h3><div class="NLM_p">Animal experiments were performed under an approved animal protocol (Protocol ID: PRO00007499, PI: Shaomeng Wang) by the Institutional Animal Care & committee of the University of Michigan. Xenograft tumors were established by injecting 1 Ã 10<sup>7</sup> KARPAS422 human B cell lymphoma cells in 50% Matrigel subcutaneously on the dorsal side of severe combined immune-deficient (SCID) mice, obtained from Charles River, with one tumor per mouse. When tumors reached â¼100 mm<sup>3</sup>, the mice were randomly assigned to treatment and vehicle control groups. The animals were monitored daily for any signs of toxicity and weighed 2â3 times per week during the treatment period and at least weekly after the treatment ended. Tumor size was measured utilizing electronic calipers 2â3 times per week during the treatment period and at least weekly after the treatment ended. Tumor volume was calculated as <i>V</i> = <i>L</i> Ã <i>W</i><sup>2</sup>/2, where <i>L</i> is the length and <i>W</i> is the width of the tumor. EED inhibitors were formulated as a suspension in PEG 200 and administered orally by gavage at indicated doses. When applicable, results are presented as mean Â± SEM. Graphing and statistical analysis were performed using GraphPad Prism 7.00 (GraphPad Software).</div><div class="NLM_p last">For pharmacodynamic analysis, resected control and treated KARPAS422 xenograft tumor tissues were ground into powder in liquid nitrogen and lysed in CST lysis buffer with halt proteinase inhibitors. Twenty micrograms of whole tumor clarified lysates were separated on 4â20 or 4â12% Novex gels. Histone proteins were extracted from the tumor tissue with the EpiQuik Total Histone Extraction Kit, purchased from EpiGentek (#OP-0006-100, Farmingdale, NY), and the level of H3K27me3 was examined by Western blotting analysis. Total histone H3 was used as the loading control. Three mice were used for each compound with each mouse bearing one tumor. The Histone H3 (#4499) and Tri-Methyl-Histone H3(Lys27) (#9733) antibodies were purchased from Cell Signaling Technology, Inc. (Danvers, MA).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Expression and Purification of the EED Protein</h3><div class="NLM_p">The gene for human EED (residues 77â441 with M370T mutation) was cloned into the pMCSG7 vector and expressed with an N-terminal 6xHis Tag with a TEV cleavage site (His-TEV-EED). <i>Escherichia coli</i> Rosetta 2 (DE3) cells were transformed with the aforementioned vector and grown in Terrific Broth supplemented with 100 Î¼g/mL ampicillin at 37 Â°C. Once cells reached an OD<sub>600</sub> of 1.0, expression was induced by addition of 0.4 mM IPTG. The cells were incubated at 20 Â°C overnight, harvested by centrifugation at 6700<i>g</i>, and stored at â80 Â°C until further purification.</div><div class="NLM_p last">The protein was purified by resuspending a thawed pellet from 1 L expression media in 40 mL lysis buffer containing 25 mM Tris pH 7.5, 200 mM NaCl, 0.1% mercaptoethanol, 10 Î¼g/mL aprotinin, and 1 Î¼g/mL leupeptin. Cells were lysed by sonication and insoluble materials cleared from the lysate by centrifugation at 34,000<i>g</i>. The resulting supernatant was incubated with 5 mL of Ni-NTA resin at 4 Â°C for 1 h, then washed with buffer containing 25 mM Tris pH 7.5, 200 mM NaCl, and 10 mM imidazole. His-TEV-EED was eluted from Ni-NTA resin with 25 mL of buffer comprising 25 mM Tris pH 7.5, 200 mM NaCl, and 300 mM imidazole. The eluted protein was treated with TEV protease to remove the affinity tag and then dialyzed against 1 L of buffer containing 25 mM Tris pH 7.5, 150 mM NaCl, and 0.1% mercaptoethanol overnight. EED was further purified by size exclusion chromatography on a HiLoad 16/60 Superdex 200 column (GE Healthcare) pre-equilibrated with 25 mM Tris pH 7.5, 200 mM NaCl, and 1 mM DTT. The purified protein was dialyzed into 20 mM Tris pH 8.7, 150 mM NaCl, 1 mM TCEP, and concentrated to 4 mg/mL for crystallization.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Crystallization and Structure Determination of EED Protein in a Complex with EED Inhibitors</h3><div class="NLM_p">Human EED was crystallized by sitting drop vapor diffusion at 20 Â°C. Drops producing crystals contained concentrated protein (4 mg/mL in 20 mM Tris pH 8.7, 150 mM NaCl, 1 mM TCEP) mixed with an equal volume of well solution containing 100 mM Tris pH 8.5, 4.2 M sodium formate, 18% glycerol, and 10 mM tris(2-carboxyethyl)phosphine. The crystals were transferred into a soak solution consisting of well solution supplemented with 1 mM EEDi-5285 or 5 mM EEDi-1056 and incubated at 20 Â°C for 18â20 h and then flash-frozen in liquid nitrogen.</div><div class="NLM_p">Diffraction data were collected on the Advanced Photon Source LS-CAT beamlines 21-ID-D or 21-ID-F (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf" class="ext-link">Supporting Information</a>, Table S1), processed with HKL2000,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and solved by molecular replacement in Molrep<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> using a published structure of EED as a search model (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QXV">2QXV</a>, chain A).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Iterative model building and refinement were performed using COOT<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and BUSTER,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> respectively. Ligand restraints were generated using GRADE.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p last">The structures of EED with inhibitors EEDi-5285 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7F">6W7F</a>) and EEDi-1056 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7G">6W7G</a>) were solved in <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> to 2.2 and 1.85 Ã, respectively, with one chain of protein comprising the asymmetric unit. The overall structure of the EEDi-5285 and EEDi-1056 proteins was highly similar, with an rmsd of 0.3 Ã based on SSM superpositioning in COOT.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In addition to the ligand density observed in the methyllysine binding site for each inhibitor (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf" class="ext-link">Figure S1A,B</a>), electron density corresponding to each inhibitor was observed in a secondary site on the surface of the protein in their respective structures at varying degrees of occupancy (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf" class="ext-link">Figure S2A,B</a>). This secondary site is lined by hydrophobic residues L205, M271, and I274, with interactions occurring between the Y277 and the [1,2,4]-triazolopyridine core of each inhibitor as well as between H199 and the dihydrobenzofuran moiety (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf" class="ext-link">Figure S3</a>). Both inhibitors also participate in hydrogen bonding with D202 and interact with residues E249 and the backbone carbonyl of K275 through H<sub>2</sub>O networks. This site is unoccupied in all available cocrystal structures of EED bound to both small molecule inhibitors as well as partner proteins making up the polycomb complex, and whether or not it modulates the function has yet to be determined.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i85"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00479" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84202" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84202" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00479?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00479</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of representative compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Crystallography data collection and refinement statistics; difference in electron density contoured to 3Ï; and molecular string files for all the final target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf">jm0c00479_si_001.pdf (1.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_002.csv">jm0c00479_si_002.csv (3.21 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates of cocrystal structures for compound <b>28</b> (EEDi-5285) in a complex with EED and for compound <b>32</b> (EEDi-1056) in a complex with EED have been deposited into Protein Data Bank with accession codes of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7F">6W7F</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7G">6W7G</a>, respectively. Authors will release the atomic coordinates upon the publication of this article.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00479" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38166" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38166" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, Departments of Internal Medicine and Pharmacology,
Medical School, and Department of Medicinal Chemistry, College of
Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1d6e757c727078737a5d6870747e7533787968"><span class="__cf_email__" data-cfemail="deadb6bfb1b3bbb0b99eabb3b7bdb6f0bbbaab">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rohan Kalyan Rej</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Changwei Wang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianfeng Lu</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mi Wang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elyse Petrunak</span> - <span class="hlFld-Affiliation affiliation">Life Sciences
Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaitlin P. Zawacki</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Donna McEachern</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ester Fernandez-Salas</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao-Yie Yang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span>Present Address:
                        The University of Tennessee Health Science Center, Pharmaceutical Sciences, 881 MADISON AVENUE, MEMPHIS TN 381632198</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5445-0109" title="Orcid link">http://orcid.org/0000-0002-5445-0109</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lu Wang</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruiting Li</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Krishnapriya Chinnaswamy</span> - <span class="hlFld-Affiliation affiliation">Life Sciences
Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bo Wen</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duxin Sun</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6406-2126" title="Orcid link">http://orcid.org/0000-0002-6406-2126</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeanne Stuckey</span> - <span class="hlFld-Affiliation affiliation">Rogel
Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Life Sciences
Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yunlong Zhou</span> - <span class="hlFld-Affiliation affiliation">Ascentage
Pharma Group, Suzhou, Jiangsu 215216, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianyong Chen</span> - <span class="hlFld-Affiliation affiliation">Ascentage
Pharma Group, Suzhou, Jiangsu 215216, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guozhi Tang</span> - <span class="hlFld-Affiliation affiliation">Ascentage
Pharma Group, Suzhou, Jiangsu 215216, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7131-4678" title="Orcid link">http://orcid.org/0000-0002-7131-4678</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.K.R. and C.W. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The University of Michigan has filed a patent application on these EED inhibitors, which has been licensed by Ascentage Pharma Group. S. Wang, R. Rej, C. Wang, M. Wang, J. Lu, C.-Y. Yang, E. Fernandez-Salas, and J. Stuckey are co-inventors on the patent application. The University of Michigan has received a research contract from Ascentage. S.W. is a co-founder of Ascentage, owns shares in Ascentage and is a paid consultant to Ascentage. The University of Michigan also owns shares in Ascentage.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i89">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported in part by a research contract from Ascentage Pharma Group (to S.W.) and the Rogel Cancer Center Support Grant from National Cancer Institute, NIH (P30 CA046592). Use of the Advanced Photon Source, an Office of Science User Facility operated for the U.S. Department of Energy (DOE) Office of Science by Argonne National Laboratory, was supported by the U.S. DOE under contract no. DE-AC02-06CH11357. Use of the LS-CAT Sector 21 was supported by the Michigan Economic Development Corporation and the Michigan Technology Tri-Corridor (Grant 085P1000817).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">PRC</td><td class="NLM_def"><p class="first last">polycomb repressive complex</p></td></tr><tr><td class="NLM_term">EED</td><td class="NLM_def"><p class="first last">embryonic ectoderm development</p></td></tr><tr><td class="NLM_term">H3K27</td><td class="NLM_def"><p class="first last">histone 3 lysine 27</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine hydrolase</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationship</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">PPA</td><td class="NLM_def"><p class="first last">polyphosphoric acid</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultraperformance liquid chromatography</p></td></tr><tr><td class="NLM_term">SCID mice</td><td class="NLM_def"><p class="first last">severe combined immunodeficient mice</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">volume of plasma cleared of the drug per unit time</p></td></tr><tr><td class="NLM_term">TGI</td><td class="NLM_def"><p class="first last">tumor growth inhibition</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">steady-state volume of distribution</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i91">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51086" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51086" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 29 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. B.</span></span> <span> </span><span class="NLM_article-title">A gene complex controlling segmentation in Drosophila</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">565</span>â <span class="NLM_lpage">570</span>, <span class="refDoi">Â DOI: 10.1038/276565a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2F276565a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=103000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A280%3ADyaE1M%252FntlWgsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1978&pages=565-570&author=E.+B.+Lewis&title=A+gene+complex+controlling+segmentation+in+Drosophila&doi=10.1038%2F276565a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A gene complex controlling segmentation in Drosophila</span></div><div class="casAuthors">Lewis E B</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">5688</span>),
    <span class="NLM_cas:pages">565-70</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The bithorax gene complex in Drosophila contains a minimum of eight genes that seem to code for substances controlling levels of thoracic and abdominal development.  The state of repression of at least four of these genes is controlled by cis-regulatory elements and a separate locus (Polycomb) seems to code for a repressor of the complex.  The wild-type and mutant segmentation patterns are consistent with an antero-posterior gradient in repressor concentration along the embryo and a proximo-distal gradient along the chromosome in the affinities for repressor of each gene's cis-regulatory element.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkFLr2ycH_oSsZ1x6zOs0vfW6udTcc2ebX7HjR0qJ_drntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE1M%252FntlWgsQ%253D%253D&md5=984f6a3650a54711616639bbdc5fa131</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F276565a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F276565a0%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DE.%2BB.%26atitle%3DA%2520gene%2520complex%2520controlling%2520segmentation%2520in%2520Drosophila%26jtitle%3DNature%26date%3D1978%26volume%3D276%26spage%3D565%26epage%3D570%26doi%3D10.1038%2F276565a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Margueron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">The polycomb complex PRC2 and its mark in life</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>469</i></span>,  <span class="NLM_fpage">343</span>â <span class="NLM_lpage">349</span>, <span class="refDoi">Â DOI: 10.1038/nature09784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnature09784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=21248841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=343-349&author=R.+Margueronauthor=D.+Reinberg&title=The+polycomb+complex+PRC2+and+its+mark+in+life&doi=10.1038%2Fnature09784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Polycomb complex PRC2 and its mark in life</span></div><div class="casAuthors">Margueron, Raphael; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7330</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group proteins maintain the gene-expression pattern of different cells that is set during early development by regulating chromatin structure.  In mammals, 2 main Polycomb group complexes exist: Polycomb repressive complex 1 (PRC1) and 2 (PRC2).  PRC1 compacts chromatin and catalyzes the monoubiquitination of histone H2A.  PRC2 also contributes to chromatin compaction, and catalyzes the methylation of histone H3 at Lys-27.  PRC2 is involved in various biol. processes, including differentiation, maintaining cell identity and proliferation, and stem-cell plasticity.  Recent studies of PRC2 have expanded the perspectives on its function and regulation, and uncovered a role for non-coding RNA in the recruitment of PRC2 to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fYmuBqnMA7Vg90H21EOLACvtfcHk0ljyQZDEJPAT3Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D&md5=5f15e91cc6ff765f649842041f333841</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature09784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09784%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DThe%2520polycomb%2520complex%2520PRC2%2520and%2520its%2520mark%2520in%2520life%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D343%26epage%3D349%26doi%3D10.1038%2Fnature09784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingston, R. E.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of polycomb gene silencing: knowns and unknowns</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">697</span>â <span class="NLM_lpage">708</span>, <span class="refDoi">Â DOI: 10.1038/nrm2763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnrm2763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=19738629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtV2qtL%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=697-708&author=J.+A.+Simonauthor=R.+E.+Kingston&title=Mechanisms+of+polycomb+gene+silencing%3A+knowns+and+unknowns&doi=10.1038%2Fnrm2763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Polycomb gene silencing: Knowns and unknowns</span></div><div class="casAuthors">Simon, Jeffrey A.; Kingston, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">697-708</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb proteins form chromatin-modifying complexes that implement transcriptional silencing in higher eukaryotes.  Hundreds of genes are silenced by Polycomb proteins, including dozens of genes that encode crucial developmental regulators in organisms ranging from plants to humans.  Two main families of complexes, called Polycomb repressive complex 1 (PRC1) and PRC2, are targeted to repressed regions.  Recent studies have advanced our understanding of these complexes, including their potential mechanisms of gene silencing, the roles of chromatin modifications, their means of delivery to target genes and the functional distinctions among variant complexes.  Emerging concepts include the existence of a Polycomb barrier to transcription elongation and the involvement of non-coding RNAs in the targeting of Polycomb complexes.  These findings have an impact on the epigenetic programming of gene expression in many biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAhDlHpccs6bVg90H21EOLACvtfcHk0ljyQZDEJPAT3Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtV2qtL%252FO&md5=416b665e14eb322565ceff685d30a647</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm2763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2763%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DJ.%2BA.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26atitle%3DMechanisms%2520of%2520polycomb%2520gene%2520silencing%253A%2520knowns%2520and%2520unknowns%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2009%26volume%3D10%26spage%3D697%26epage%3D708%26doi%3D10.1038%2Fnrm2763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingston, R. E.</span></span> <span> </span><span class="NLM_article-title">The core of the polycomb repressive complex is compositionally and functionally conserved in flies and humans</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">6070</span>â <span class="NLM_lpage">6078</span>, <span class="refDoi">Â DOI: 10.1128/mcb.22.17.6070-6078.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1128%2FMCB.22.17.6070-6078.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=12167701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmt1entrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=6070-6078&author=S.+S.+Levineauthor=A.+Weissauthor=H.+Erdjument-Bromageauthor=Z.+Shaoauthor=P.+Tempstauthor=R.+E.+Kingston&title=The+core+of+the+polycomb+repressive+complex+is+compositionally+and+functionally+conserved+in+flies+and+humans&doi=10.1128%2Fmcb.22.17.6070-6078.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The core of the Polycomb repressive complex is compositionally and functionally conserved in flies and humans</span></div><div class="casAuthors">Levine, Stuart S.; Weiss, Alona; Erdjument-Bromage, Hediye; Shao, Zhaohui; Tempst, Paul; Kingston, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6070-6078</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Polycomb group (PcG) genes are required to maintain homeotic genes in a silenced state during development in Drosophila and mammals and are thought to form several distinct silencing complexes that maintain homeotic gene repression during development.  Mutations in the PcG genes result in developmental defects and have been implicated in human cancer.  Although some PcG protein domains are conserved between flies and humans, substantial regions of several PcG proteins are divergent and humans contain multiple versions of each PcG gene.  To det. the effects of these changes on the compn. and function of a PcG complex, we have purified a human Polycomb repressive complex from HeLa cells (hPRC-H) that contains homologues of PcG proteins found in Drosophila embryonic PRC1 (dPRC1).  HPRC-H was found to have fewer components than dPRC1, retaining the PcG core proteins of dPRC1 but lacking most non-PcG proteins.  Prepns. of hPRC-H contained either two or three different homologues of most of the core PcG proteins, including a new Ph homolog we have named HPH3.  Despite differences in compn., dPRC1 and hPRC-H have similar functions: hPRC-H is able to efficiently block remodeling of nucleosomal arrays through a mechanism that does not block the ability of nucleases to access and cleave the arrays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRfUn5IP2kuLVg90H21EOLACvtfcHk0ljyQZDEJPAT3Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmt1entrs%253D&md5=72265e53cc9036f9c97ede6338032027</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FMCB.22.17.6070-6078.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.22.17.6070-6078.2002%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DS.%2BS.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26atitle%3DThe%2520core%2520of%2520the%2520polycomb%2520repressive%2520complex%2520is%2520compositionally%2520and%2520functionally%2520conserved%2520in%2520flies%2520and%2520humans%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2002%26volume%3D22%26spage%3D6070%26epage%3D6078%26doi%3D10.1128%2Fmcb.22.17.6070-6078.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dimou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dincman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evanno, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemmill, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drabkin, H. A.</span></span> <span> </span><span class="NLM_article-title">Epigenetics during EMT in lung cancer: EZH2 as a potential therapeutic target</span>. <i>Cancer Treat. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">40</span>â <span class="NLM_lpage">48</span>, <span class="refDoi">Â DOI: 10.1016/j.ctarc.2017.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1016%2Fj.ctarc.2017.06.003" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=40-48&author=A.+Dimouauthor=T.+Dincmanauthor=E.+Evannoauthor=R.+M.+Gemmillauthor=J.+Rocheauthor=H.+A.+Drabkin&title=Epigenetics+during+EMT+in+lung+cancer%3A+EZH2+as+a+potential+therapeutic+target&doi=10.1016%2Fj.ctarc.2017.06.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ctarc.2017.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctarc.2017.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DDimou%26aufirst%3DA.%26aulast%3DDincman%26aufirst%3DT.%26aulast%3DEvanno%26aufirst%3DE.%26aulast%3DGemmill%26aufirst%3DR.%2BM.%26aulast%3DRoche%26aufirst%3DJ.%26aulast%3DDrabkin%26aufirst%3DH.%2BA.%26atitle%3DEpigenetics%2520during%2520EMT%2520in%2520lung%2520cancer%253A%2520EZH2%2520as%2520a%2520potential%2520therapeutic%2520target%26jtitle%3DCancer%2520Treat.%2520Res.%2520Commun.%26date%3D2017%26volume%3D12%26spage%3D40%26epage%3D48%26doi%3D10.1016%2Fj.ctarc.2017.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Role of histone H3 lysine 27 methylation in polycomb-group silencing</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1039</span>â <span class="NLM_lpage">1043</span>, <span class="refDoi">Â DOI: 10.1126/science.1076997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1126%2Fscience.1076997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=12351676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD38Xot12rs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1039-1043&author=R.+Caoauthor=L.+Wangauthor=H.+Wangauthor=L.+Xiaauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=R.+S.+Jonesauthor=Y.+Zhang&title=Role+of+histone+H3+lysine+27+methylation+in+polycomb-group+silencing&doi=10.1126%2Fscience.1076997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Histone H3 Lysine 27 Methylation in Polycomb-Group Silencing</span></div><div class="casAuthors">Cao, Ru; Wang, Liangjun; Wang, Hengbin; Xia, Li; Erdjument-Bromage, Hediye; Tempst, Paul; Jones, Richard S.; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5595</span>),
    <span class="NLM_cas:pages">1039-1043</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Polycomb group (PcG) proteins play important roles in maintaining the silent state of HOX genes.  Recent studies have implicated histone methylation in long-term gene silencing.  However, a connection between PcG-mediated gene silencing and histone methylation has not been established.  Here the authors report the purifn. and characterization of an EED-EZH2 complex, the human counterpart of the Drosophila ESC-E(Z) complex.  They demonstrate that the complex specifically methylates nucleosomal histone H3 at lysine 27 (H3-K27).  Using chromatin immunopptn. assays, the authors show that H3-K27 methylation colocalizes with, and is dependent on, E(Z) binding at an Ultrabithorax medÂ° (Ubx) Polycomb response element (PRE), and that this methylation correlates with Ubx repression.  Methylation on H3-K27 facilitates binding of Polycomb (PC), a component of the PRC1 complex, to histone H3 amino-terminal tail.  Thus, these studies establish a link between histone methylation and PcG-mediated gene silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnqusXKZEbLVg90H21EOLACvtfcHk0lgo_8Pq34RhMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xot12rs7g%253D&md5=8f24c05ec935c9110b2ff85b65911396</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.1076997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1076997%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DR.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DRole%2520of%2520histone%2520H3%2520lysine%252027%2520methylation%2520in%2520polycomb-group%2520silencing%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1039%26epage%3D1043%26doi%3D10.1126%2Fscience.1076997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmichev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishioka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">Histone methyltransferase activity associated with a human multiprotein complex containing the enhancer of zeste protein</span>. <i>Gene Dev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2893</span>â <span class="NLM_lpage">2905</span>, <span class="refDoi">Â DOI: 10.1101/gad.1035902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1101%2Fgad.1035902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=12435631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD38XovVejsLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=2893-2905&author=A.+Kuzmichevauthor=K.+Nishiokaauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=D.+Reinberg&title=Histone+methyltransferase+activity+associated+with+a+human+multiprotein+complex+containing+the+enhancer+of+zeste+protein&doi=10.1101%2Fgad.1035902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferase activity associated with a human multiprotein complex containing the Enhancer of Zeste protein</span></div><div class="casAuthors">Kuzmichev, Andrei; Nishioka, Kenichi; Erdjument-Bromage, Hediye; Tempst, Paul; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2893-2905</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Enhancer of Zeste [E(z)] is a Polycomb-group transcriptional repressor and one of the founding members of the family of SET domain-contg. proteins.  Several SET-domain proteins possess intrinsic histone methyltransferase (HMT) activity.  However, recombinant E(z) protein was found to be inactive in a HMT assay.  Here we report the isolation of a multiprotein E(z) complex that contains extra sex combs, suppressor of zeste-12 [Su(z)12], and the histone binding proteins RbAp46/RbAp48.  This complex, which we termed Polycomb repressive complex (PRC) 2, possesses HMT activity with specificity for Lys 9 (K9) and Lys 27 (K27) of histone H3.  The HMT activity of PRC2 is dependent on an intact SET domain in the E(z) protein.  We hypothesize that transcriptional repression by the E(z) protein involves methylation-dependent recruitment of PRC1.  The presence of Su(z)12, a strong suppressor of position effect variegation, in PRC2 suggests that PRC2 may play a widespread role in heterochromatin-mediated silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxD09IiaeCvLVg90H21EOLACvtfcHk0lgo_8Pq34RhMg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XovVejsLY%253D&md5=b61767bc9b30b403f5fc7bb202281da1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fgad.1035902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1035902%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmichev%26aufirst%3DA.%26aulast%3DNishioka%26aufirst%3DK.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DHistone%2520methyltransferase%2520activity%2520associated%2520with%2520a%2520human%2520multiprotein%2520complex%2520containing%2520the%2520enhancer%2520of%2520zeste%2520protein%26jtitle%3DGene%2520Dev.%26date%3D2002%26volume%3D16%26spage%3D2893%26epage%3D2905%26doi%3D10.1101%2Fgad.1035902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span> <span> </span><span class="NLM_article-title">An allosteric PRC2 inhibitor targeting the H3K27me3 binding pocket of EED</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">381</span>â <span class="NLM_lpage">388</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnchembio.2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=28135235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvV2hsLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=381-388&author=W.+Qiauthor=K.+Zhaoauthor=J.+Guauthor=Y.+Huangauthor=Y.+Wangauthor=H.+Zhangauthor=M.+Zhangauthor=J.+Zhangauthor=Z.+Yuauthor=L.+Liauthor=L.+Tengauthor=S.+Chuaiauthor=C.+Zhangauthor=M.+Zhaoauthor=H.+Chanauthor=Z.+Chenauthor=D.+Fangauthor=Q.+Feiauthor=L.+Fengauthor=L.+Fengauthor=Y.+Gaoauthor=H.+Geauthor=X.+Geauthor=G.+Liauthor=A.+Lingelauthor=Y.+Linauthor=Y.+Liuauthor=F.+Luoauthor=M.+Shiauthor=L.+Wangauthor=Z.+Wangauthor=Y.+Yuauthor=J.+Zengauthor=C.+Zengauthor=L.+Zhangauthor=Q.+Zhangauthor=S.+Zhouauthor=C.+Oyangauthor=P.+Atadjaauthor=E.+Li&title=An+allosteric+PRC2+inhibitor+targeting+the+H3K27me3+binding+pocket+of+EED&doi=10.1038%2Fnchembio.2304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric PRC2 inhibitor targeting the H3K27me3 binding pocket of EED</span></div><div class="casAuthors">Qi, Wei; Zhao, Kehao; Gu, Justin; Huang, Ying; Wang, Youzhen; Zhang, Hailong; Zhang, Man; Zhang, Jeff; Yu, Zhengtian; Li, Ling; Teng, Lin; Chuai, Shannon; Zhang, Chao; Zhao, Mengxi; Chan, Ho Man; Chen, Zijun; Fang, Douglas; Fei, Qi; Feng, Leying; Feng, Lijian; Gao, Yuan; Ge, Hui; Ge, Xinjian; Li, Guobin; Lingel, Andreas; Lin, Ying; Liu, Yueqin; Luo, Fangjun; Shi, Minlong; Wang, Long; Wang, Zhaofu; Yu, Yanyan; Zeng, Jue; Zeng, Chenhui; Zhang, Lijun; Zhang, Qiong; Zhou, Shaolian; Oyang, Counde; Atadja, Peter; Li, En</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">381-388</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) consists of three core subunits, EZH2, EED and SUZ12, and plays pivotal roles in transcriptional regulation.  The catalytic subunit EZH2 methylates histone H3 lysine 27 (H3K27), and its activity is further enhanced by the binding of EED to trimethylated H3K27 (H3K27me3).  Small-mol. inhibitors that compete with the cofactor S-adenosylmethionine (SAM) have been reported.  Here we report the discovery of EED226, a potent and selective PRC2 inhibitor that directly binds to the H3K27me3 binding pocket of EED.  EED226 induces a conformational change upon binding EED, leading to loss of PRC2 activity.  EED226 shows similar activity to SAM-competitive inhibitors in blocking H3K27 methylation of PRC2 target genes and inducing regression of human lymphoma xenograft tumors.  Interestingly, EED226 also effectively inhibits PRC2 contg. a mutant EZH2 protein resistant to SAM-competitive inhibitors.  Together, we show that EED226 inhibits PRC2 activity via an allosteric mechanism and offers an opportunity for treatment of PRC2-dependent cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjpXExCMohI7Vg90H21EOLACvtfcHk0ljxpAutYtdD0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvV2hsLc%253D&md5=62d10723685450ca7ffd599a61134d42</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2304%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DTeng%26aufirst%3DL.%26aulast%3DChuai%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DFang%26aufirst%3DD.%26aulast%3DFei%26aufirst%3DQ.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DLingel%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DF.%26aulast%3DShi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DOyang%26aufirst%3DC.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DAn%2520allosteric%2520PRC2%2520inhibitor%2520targeting%2520the%2520H3K27me3%2520binding%2520pocket%2520of%2520EED%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D381%26epage%3D388%26doi%3D10.1038%2Fnchembio.2304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvaraju, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comess, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">The, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lima-Fernandes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinge, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algire, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osterling, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasinski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span> <span> </span><span class="NLM_article-title">The EED protein-protein interaction inhibitor A-395 inactivates the PRC2 complex</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">389</span>â <span class="NLM_lpage">395</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnchembio.2306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=28135237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvV2hsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=389-395&author=Y.+Heauthor=S.+Selvarajuauthor=M.+L.+Curtinauthor=C.+G.+Jakobauthor=H.+Zhuauthor=K.+M.+Comessauthor=B.+Shawauthor=J.+Theauthor=E.+Lima-Fernandesauthor=M.+M.+Szewczykauthor=D.+Chengauthor=K.+L.+Klingeauthor=H.-Q.+Liauthor=M.+Pliushchevauthor=M.+A.+Algireauthor=D.+Maagauthor=J.+Guoauthor=J.+Dietrichauthor=S.+C.+Panchalauthor=A.+M.+Petrosauthor=R.+F.+Sweisauthor=M.+Torrentauthor=L.+J.+Bigelowauthor=G.+Senisterraauthor=F.+Liauthor=S.+Kennedyauthor=Q.+Wuauthor=D.+J.+Osterlingauthor=D.+J.+Lindleyauthor=W.+Gaoauthor=S.+Galasinskiauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=F.+G.+Buchananauthor=C.+H.+Arrowsmithauthor=G.+G.+Chiangauthor=C.+Sunauthor=W.+N.+Pappano&title=The+EED+protein-protein+interaction+inhibitor+A-395+inactivates+the+PRC2+complex&doi=10.1038%2Fnchembio.2306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The EED protein-protein interaction inhibitor A-395 inactivates the PRC2 complex</span></div><div class="casAuthors">He, Yupeng; Selvaraju, Sujatha; Curtin, Michael L.; Jakob, Clarissa G.; Zhu, Haizhong; Comess, Kenneth M.; Shaw, Bailin; The, Juliana; Lima-Fernandes, Evelyne; Szewczyk, Magdalena M.; Cheng, Dong; Klinge, Kelly L.; Li, Huan-Qiu; Pliushchev, Marina; Algire, Mikkel A.; Maag, David; Guo, Jun; Dietrich, Justin; Panchal, Sanjay C.; Petros, Andrew M.; Sweis, Ramzi F.; Torrent, Maricel; Bigelow, Lance J.; Senisterra, Guillermo; Li, Fengling; Kennedy, Steven; Wu, Qin; Osterling, Donald J.; Lindley, David J.; Gao, Wenqing; Galasinski, Scott; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Buchanan, Fritz G.; Arrowsmith, Cheryl H.; Chiang, Gary G.; Sun, Chaohong; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">389-395</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) is a regulator of epigenetic states required for development and homeostasis.  PRC2 trimethylates histone H3 at lysine 27 (H3K27me3), which leads to gene silencing, and is dysregulated in many cancers.  The embryonic ectoderm development (EED) protein is an essential subunit of PRC2 that has both a scaffolding function and an H3K27me3-binding function.  Here we report the identification of A-395, a potent antagonist of the H3K27me3 binding functions of EED.  Structural studies demonstrate that A-395 binds to EED in the H3K27me3-binding pocket, thereby preventing allosteric activation of the catalytic activity of PRC2.  Phenotypic effects obsd. in vitro and in vivo are similar to those of known PRC2 enzymic inhibitors; however, A-395 retains potent activity against cell lines resistant to the catalytic inhibitors.  A-395 represents a first-in-class antagonist of PRC2 protein-protein interactions (PPI) for use as a chem. probe to investigate the roles of EED-contg. protein complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX5rHeR_Z5mLVg90H21EOLACvtfcHk0lgrjBbOmQzH0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvV2hsbs%253D&md5=d41eee098ed123f8c7f50babe93d7692</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2306%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DSelvaraju%26aufirst%3DS.%26aulast%3DCurtin%26aufirst%3DM.%2BL.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DThe%26aufirst%3DJ.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DKlinge%26aufirst%3DK.%2BL.%26aulast%3DLi%26aufirst%3DH.-Q.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DAlgire%26aufirst%3DM.%2BA.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DDietrich%26aufirst%3DJ.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DBigelow%26aufirst%3DL.%2BJ.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DOsterling%26aufirst%3DD.%2BJ.%26aulast%3DLindley%26aufirst%3DD.%2BJ.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DGalasinski%26aufirst%3DS.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DThe%2520EED%2520protein-protein%2520interaction%2520inhibitor%2520A-395%2520inactivates%2520the%2520PRC2%2520complex%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D389%26epage%3D395%26doi%3D10.1038%2Fnchembio.2306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Croce, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Transcriptional regulation by Polycomb group proteins</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1147</span>â <span class="NLM_lpage">1155</span>, <span class="refDoi">Â DOI: 10.1038/nsmb.2669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnsmb.2669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=24096405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOiurrP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1147-1155&author=L.+Di%0ACroceauthor=K.+Helin&title=Transcriptional+regulation+by+Polycomb+group+proteins&doi=10.1038%2Fnsmb.2669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation by Polycomb group proteins</span></div><div class="casAuthors">Di Croce, Luciano; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1147-1155</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group (PcG) proteins are epigenetic regulators of transcription that have key roles in stem-cell identity, differentiation and disease.  Mechanistically, they function within multiprotein complexes, called Polycomb repressive complexes (PRCs), which modify histones (and other proteins) and silence target genes.  The dynamics of PRC1 and PRC2 components has been the focus of recent research.  Here we discuss our current knowledge of the PRC complexes, how they are targeted to chromatin and how the high diversity of the PcG proteins allows these complexes to influence cell identity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrArNyrbNU897Vg90H21EOLACvtfcHk0lgrjBbOmQzH0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOiurrP&md5=004c2682edbda681e9f6de84ba87e639</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2669%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BCroce%26aufirst%3DL.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DTranscriptional%2520regulation%2520by%2520Polycomb%2520group%2520proteins%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1147%26epage%3D1155%26doi%3D10.1038%2Fnsmb.2669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganji, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iii, A. D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaFrance, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duquenne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creasy, C. L.</span></span> <span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">108</span>â <span class="NLM_lpage">112</span>, <span class="refDoi">Â DOI: 10.1038/nature11606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnature11606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=23051747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+D.+P.+Iiiauthor=E.+Diazauthor=L.+V.+LaFranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations&doi=10.1038%2Fnature11606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span></div><div class="casAuthors">McCabe, Michael T.; Ott, Heidi M.; Ganji, Gopinath; Korenchuk, Susan; Thompson, Christine; Van Aller, Glenn S.; Liu, Yan; Graves, Alan P.; Della Pietra, Anthony , III; Diaz, Elsie; LaFrance, Louis V.; Mellinger, Mark; Duquenne, Celine; Tian, Xinrong; Kruger, Ryan G.; McHugh, Charles F.; Brandt, Martin; Miller, William H.; Dhanak, Dashyant; Verma, Sharad K.; Tummino, Peter J.; Creasy, Caretha L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7427</span>),
    <span class="NLM_cas:pages">108-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In eukaryotes, post-translational modification of histones is crit. for regulation of chromatin structure and gene expression.  EZH2 is the catalytic subunit of the polycomb repressive complex 2 (PRC2) and is involved in repressing gene expression through methylation of histone H3 on lysine 27 (H3K27).  EZH2 overexpression is implicated in tumorigenesis and correlates with poor prognosis in several tumor types.  Addnl., somatic heterozygous mutations of Y641 and A677 residues within the catalytic SET domain of EZH2 occur in diffuse large B-cell lymphoma (DLBCL) and follicular lymphoma.  The Y641 residue is the most frequently mutated residue, with up to 22% of germinal center B-cell DLBCL and follicular lymphoma harbouring mutations at this site.  These lymphomas have increased H3K27 tri-methylation (H3K27me3) owing to altered substrate preferences of the mutant enzymes.  However, it is unknown whether specific, direct inhibition of EZH2 methyltransferase activity will be effective in treating EZH2 mutant lymphomas.  Here we demonstrate that GSK126, a potent, highly selective, S-adenosyl-methionine-competitive, small-mol. inhibitor of EZH2 methyltransferase activity, decreases global H3K27me3 levels and reactivates silenced PRC2 target genes.  GSK126 effectively inhibits the proliferation of EZH2 mutant DLBCL cell lines and markedly inhibits the growth of EZH2 mutant DLBCL xenografts in mice.  Together, these data demonstrate that pharmacol. inhibition of EZH2 activity may provide a promising treatment for EZH2 mutant lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxw5cLmW3Lc7Vg90H21EOLACvtfcHk0lhe1tGyyTtESQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL&md5=a1dff99556de004c48aae88bc45dffab</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature11606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11606%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DIii%26aufirst%3DA.%2BD.%2BP.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112%26doi%3D10.1038%2Fnature11606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacNevin, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rougie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">An orally bioavailable chemical probe of the Lysine Methyltransferases EZH2 and EZH1</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1324</span>â <span class="NLM_lpage">1334</span>, <span class="refDoi">Â DOI: 10.1021/cb400133j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400133j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1324-1334&author=K.+D.+Konzeauthor=A.+Maauthor=F.+Liauthor=D.+Barsyte-Lovejoyauthor=T.+Partonauthor=C.+J.+MacNevinauthor=F.+Liuauthor=C.+Gaoauthor=X.-P.+Huangauthor=E.+Kuznetsovaauthor=M.+Rougieauthor=A.+Jiangauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=L.+I.+Jamesauthor=B.+L.+Rothauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=K.+M.+Hahnauthor=G.+G.+Wangauthor=M.+Vedadiauthor=J.+Jin&title=An+orally+bioavailable+chemical+probe+of+the+Lysine+Methyltransferases+EZH2+and+EZH1&doi=10.1021%2Fcb400133j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1</span></div><div class="casAuthors">Konze, Kyle D.; Ma, Anqi; Li, Fengling; Barsyte-Lovejoy, Dalia; Parton, Trevor; MacNevin, Christopher J.; Liu, Feng; Gao, Cen; Huang, Xi-Ping; Kuznetsova, Ekaterina; Rougie, Marie; Jiang, Alice; Pattenden, Samantha G.; Norris, Jacqueline L.; James, Lindsey I.; Roth, Bryan L.; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Hahn, Klaus M.; Wang, Gang Greg; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1324-1334</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 or EZH1 is the catalytic subunit of the polycomb repressive complex 2 that catalyzes methylation of histone H3 lysine 27 (H3K27).  The trimethylation of H3K27 (H3K27me3) is a transcriptionally repressive post-translational modification.  Overexpression of EZH2 and hypertrimethylation of H3K27 have been implicated in a no. of cancers.  Several selective inhibitors of EZH2 have been reported recently.  Herein we disclose UNC1999, the first orally bioavailable inhibitor that has high in vitro potency for wild-type and mutant EZH2 as well as EZH1, a closely related H3K27 methyltransferase that shares 96% sequence identity with EZH2 in their resp. catalytic domains.  UNC1999 was highly selective for EZH2 and EZH1 over a broad range of epigenetic and non-epigenetic targets, competitive with the cofactor SAM and non-competitive with the peptide substrate.  This inhibitor potently reduced H3K27me3 levels in cells and selectively killed diffused large B cell lymphoma cell lines harboring the EZH2Y641N mutant.  Importantly, UNC1999 was orally bioavailable in mice, making this inhibitor a valuable tool for investigating the role of EZH2 and EZH1 in chronic animal studies.  We also designed and synthesized UNC2400, a close analog of UNC1999 with potency >1,000-fold lower than that of UNC1999 as a neg. control for cell-based studies.  Finally, we created a biotin-tagged UNC1999 (UNC2399), which enriched EZH2 in pull-down studies, and a UNC1999-dye conjugate (UNC2239) for co-localization studies with EZH2 in live cells.  Taken together, these compds. represent a set of useful tools for the biomedical community to investigate the role of EZH2 and EZH1 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzhUO0iAe8BbVg90H21EOLACvtfcHk0lgMUokK8yGHqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D&md5=311d77f5f0cc0b9e2610d8d002a6709c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fcb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400133j%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DParton%26aufirst%3DT.%26aulast%3DMacNevin%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DRougie%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DHahn%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DAn%2520orally%2520bioavailable%2520chemical%2520probe%2520of%2520the%2520Lysine%2520Methyltransferases%2520EZH2%2520and%2520EZH1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1324%26epage%3D1334%26doi%3D10.1021%2Fcb400133j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warholic, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span> <span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">7922</span>â <span class="NLM_lpage">7927</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1303800110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2&doi=10.1073%2Fpnas.1303800110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0lgMUokK8yGHqg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927%26doi%3D10.1073%2Fpnas.1303800110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warholic, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadowaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uesugi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uenaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of EZH2 by EPZ-6438 leads to potent antitumor activity in EZH2-mutant non-hodgkin lymphoma</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">842</span>â <span class="NLM_lpage">854</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.mct-13-0773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1158%2F1535-7163.MCT-13-0773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=24563539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=842-854&author=S.+K.+Knutsonauthor=S.+Kawanoauthor=Y.+Minoshimaauthor=N.+M.+Warholicauthor=K.-C.+Huangauthor=Y.+Xiaoauthor=T.+Kadowakiauthor=M.+Uesugiauthor=G.+Kuznetsovauthor=N.+Kumarauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+Porter-Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=T.+Uenakaauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=A.+Yokoiauthor=H.+Keilhack&title=Selective+inhibition+of+EZH2+by+EPZ-6438+leads+to+potent+antitumor+activity+in+EZH2-mutant+non-hodgkin+lymphoma&doi=10.1158%2F1535-7163.mct-13-0773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of EZH2 by EPZ-6438 Leads to Potent Antitumor Activity in EZH2-Mutant Non-Hodgkin Lymphoma</span></div><div class="casAuthors">Knutson, Sarah K.; Kawano, Satoshi; Minoshima, Yukinori; Warholic, Natalie M.; Huang, Kuan-Chun; Xiao, Yonghong; Kadowaki, Tadashi; Uesugi, Mai; Kuznetsov, Galina; Kumar, Namita; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Waters, Nigel J.; Smith, Jesse J.; Porter-Scott, Margaret; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Uenaka, Toshimitsu; Pollock, Roy M.; Kuntz, Kevin W.; Yokoi, Akira; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">842-854</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Mutations within the catalytic domain of the histone methyltransferase EZH2 have been identified in subsets of patients with non-Hodgkin lymphoma (NHL).  These genetic alterations are hypothesized to confer an oncogenic dependency on EZH2 enzymic activity in these cancers.  We have previously reported the discovery of EPZ005678 and EPZ-6438, potent and selective S-adenosyl-methionine-competitive small mol. inhibitors of EZH2.  Although both compds. are similar with respect to their mechanism of action and selectivity, EPZ-6438 possesses superior potency and drug-like properties, including good oral bioavailability in animals.  Here, we characterize the activity of EPZ-6438 in preclin. models of NHL.  EPZ-6438 selectively inhibits intracellular lysine 27 of histone H3 (H3K27) methylation in a concn.- and time-dependent manner in both EZH2 wild-type and mutant lymphoma cells.  Inhibition of H3K27 trimethylation (H3K27Me3) leads to selective cell killing of human lymphoma cell lines bearing EZH2 catalytic domain point mutations.  Treatment of EZH2-mutant NHL xenograft-bearing mice with EPZ-6438 causes dose-dependent tumor growth inhibition, including complete and sustained tumor regressions with correlative diminution of H3K27Me3 levels in tumors and selected normal tissues.  Mice dosed orally with EPZ-6438 for 28 days remained tumor free for up to 63 days after stopping compd. treatment in two EZH2-mutant xenograft models.  These data confirm the dependency of EZH2-mutant NHL on EZH2 activity and portend the utility of EPZ-6438 as a potential treatment for these genetically defined cancers.  Mol Cancer Ther; 13(4); 842-54. Â©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq2o1v5F3GzLVg90H21EOLACvtfcHk0litJDmJ79My_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurw%253D&md5=89354e9453dcd7430978d1a6c5ee5846</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0773%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DKawano%26aufirst%3DS.%26aulast%3DMinoshima%26aufirst%3DY.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DHuang%26aufirst%3DK.-C.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DUesugi%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DN.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DUenaka%26aufirst%3DT.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DYokoi%26aufirst%3DA.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DSelective%2520inhibition%2520of%2520EZH2%2520by%2520EPZ-6438%2520leads%2520to%2520potent%2520antitumor%2520activity%2520in%2520EZH2-mutant%2520non-hodgkin%2520lymphoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D842%26epage%3D854%26doi%3D10.1158%2F1535-7163.mct-13-0773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisserier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wajapeyee, N.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of resistance to EZH2 inhibitors in diffuse large B-cell lymphomas</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">2125</span>â <span class="NLM_lpage">2137</span>, <span class="refDoi">Â DOI: 10.1182/blood-2017-08-804344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1182%2Fblood-2017-08-804344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=29572378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFenurrO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2018&pages=2125-2137&author=M.+Bisserierauthor=N.+Wajapeyee&title=Mechanisms+of+resistance+to+EZH2+inhibitors+in+diffuse+large+B-cell+lymphomas&doi=10.1182%2Fblood-2017-08-804344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of resistance to EZH2 inhibitors in diffuse large B-cell lymphomas</span></div><div class="casAuthors">Bisserier, Malik; Wajapeyee, Narendra</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2125-2137</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Resistance to targeted therapies has become increasingly prevalent.  We noted that resistance to different targeted therapies occurs by largely common mechanisms.  In this study, we used this information for identifying the mechanisms of resistance to enhancer of zeste homolog 2 (EZH2) inhibitors in diffuse large B-cell lymphoma (DLBCL) harboring EZH2 mutations.  We discovered that EZH2 inhibitor-resistant DLBCL cells showed activation of the insulin-like growth factor 1 receptor (IGF-1R), MEK, and phosphoinositide-3-kinase (PI3K) pathways.  Constitutive activation of IGF-1R, MEK, or PI3K pathways was sufficient to confer resistance to EZH2 inhibitors in DLBCL.  The activation of the PI3K/AKT andMAPK pathways decreased TNFSF10 and BAD expression through a FOXO3-dependent mechanism, which was required for the antitumor effects of EZH2i GSK126.  We also identified multiple acquired mutations in EZH2 inhibitor-resistant DLBCL cell lines.  These mutations independently conferred resistance to EZH2 inhibitors.  Mechanistically, cellular thermal shift assays revealed that the acquired EZH2 mutations that confer resistance to EZH2 inhibitors prevent EZH2 inhibitor binding to the EZH2 mutants.  Notably, EZH2 inhibitor GSK126- and EPZ-6438-resistant DLBCL cells remained sensitive to the EZH2 inhibitor UNC1999 and embryonic ectoderm development protein inhibitor EED226, which provides an opportunity to treat DLBCLs that are resistant to these drugs.  Collectively, our results underpin the importance for developing a unified approach for forestalling drug resistance by prospectively considering lessons learned from the use of different targeted therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8bPfbf3K8qrVg90H21EOLACvtfcHk0litJDmJ79My_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFenurrO&md5=e1e626bc16271b40be809c7587279cff</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1182%2Fblood-2017-08-804344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2017-08-804344%26sid%3Dliteratum%253Aachs%26aulast%3DBisserier%26aufirst%3DM.%26aulast%3DWajapeyee%26aufirst%3DN.%26atitle%3DMechanisms%2520of%2520resistance%2520to%2520EZH2%2520inhibitors%2520in%2520diffuse%2520large%2520B-cell%2520lymphomas%26jtitle%3DBlood%26date%3D2018%26volume%3D131%26spage%3D2125%26epage%3D2137%26doi%3D10.1182%2Fblood-2017-08-804344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lingel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D. E.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of EED binders allosterically inhibiting the epigenetic polycomb repressive complex 2 (PRC2) methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">415</span>â <span class="NLM_lpage">427</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01473</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01473" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFaiurrF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=415-427&author=A.+Lingelauthor=M.+Sendzikauthor=Y.+Huangauthor=M.+D.+Shultzauthor=J.+Cantwellauthor=M.+P.+Dillonauthor=X.+Fuauthor=J.+Fullerauthor=T.+Gabrielauthor=J.+Guauthor=X.+Jiangauthor=L.+Liauthor=F.+Liangauthor=M.+McKennaauthor=W.+Qiauthor=W.+Raoauthor=X.+Shengauthor=W.+Shuauthor=J.+Suttonauthor=B.+Taftauthor=L.+Wangauthor=J.+Zengauthor=H.+Zhangauthor=M.+Zhangauthor=K.+Zhaoauthor=M.+Lindvallauthor=D.+E.+Bussiere&title=Structure-guided+design+of+EED+binders+allosterically+inhibiting+the+epigenetic+polycomb+repressive+complex+2+%28PRC2%29+methyltransferase&doi=10.1021%2Facs.jmedchem.6b01473"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of EED Binders Allosterically Inhibiting the Epigenetic Polycomb Repressive Complex 2 (PRC2) Methyltransferase</span></div><div class="casAuthors">Lingel, Andreas; Sendzik, Martin; Huang, Ying; Shultz, Michael D.; Cantwell, John; Dillon, Michael P.; Fu, Xingnian; Fuller, John; Gabriel, Tobias; Gu, Justin; Jiang, Xiangqing; Li, Ling; Liang, Fang; McKenna, Maureen; Qi, Wei; Rao, Weijun; Sheng, Xijun; Shu, Wei; Sutton, James; Taft, Benjamin; Wang, Long; Zeng, Jue; Zhang, Hailong; Zhang, Maya; Zhao, Kehao; Lindvall, Mika; Bussiere, Dirksen E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">415-427</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRC2 is a multisubunit methyltransferase involved in epigenetic regulation of early embryonic development and cell growth.  The catalytic subunit EZH2 methylates primarily lysine 27 of histone H3, leading to chromatin compaction and repression of tumor suppressor genes.  Inhibiting this activity by small mols. targeting EZH2 was shown to result in antitumor efficacy.  Here, the authors describe for the first time the optimization of a chem. series representing a new class of PRC2 inhibitors which acts allosterically via the trimethyllysine pocket of the noncatalytic EED subunit.  Deconstruction of a larger and complex screening hit to a simple fragment-sized mol. followed by structure-guided regrowth and careful property modulation were employed to yield compds. which achieve sub-micromolar inhibition in functional assays and cellular activity.  The resulting mols. can serve as a simplified entry point for lead optimization and can be utilized to study this new mechanism of PRC2 inhibition and the assocd. biol. in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRo2rkeTDLibVg90H21EOLACvtfcHk0litJDmJ79My_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFaiurrF&md5=3aec108471439f4ac99728a57b4eaf8e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01473%26sid%3Dliteratum%253Aachs%26aulast%3DLingel%26aufirst%3DA.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DDillon%26aufirst%3DM.%2BP.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DFuller%26aufirst%3DJ.%26aulast%3DGabriel%26aufirst%3DT.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DF.%26aulast%3DMcKenna%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DX.%26aulast%3DShu%26aufirst%3DW.%26aulast%3DSutton%26aufirst%3DJ.%26aulast%3DTaft%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DBussiere%26aufirst%3DD.%2BE.%26atitle%3DStructure-guided%2520design%2520of%2520EED%2520binders%2520allosterically%2520inhibiting%2520the%2520epigenetic%2520polycomb%2520repressive%2520complex%25202%2520%2528PRC2%2529%2520methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D415%26epage%3D427%26doi%3D10.1021%2Facs.jmedchem.6b01473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarris, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvaraju, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algire, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span> <span> </span><span class="NLM_article-title">SAR of amino pyrrolidines as potent and novel protein-protein interaction inhibitors of the PRC2 complex through EED binding</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1576</span>â <span class="NLM_lpage">1583</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2017.02.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1016%2Fj.bmcl.2017.02.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=28254486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Oju7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1576-1583&author=M.+L.+Curtinauthor=M.+A.+Pliushchevauthor=H.-Q.+Liauthor=M.+Torrentauthor=J.+D.+Dietrichauthor=C.+G.+Jakobauthor=H.+Zhuauthor=H.+Zhaoauthor=Y.+Wangauthor=Z.+Jiauthor=R.+F.+Clarkauthor=K.+A.+Sarrisauthor=S.+Selvarajuauthor=B.+Shawauthor=M.+A.+Algireauthor=Y.+Heauthor=P.+L.+Richardsonauthor=R.+F.+Sweisauthor=C.+Sunauthor=G.+G.+Chiangauthor=M.+R.+Michaelides&title=SAR+of+amino+pyrrolidines+as+potent+and+novel+protein-protein+interaction+inhibitors+of+the+PRC2+complex+through+EED+binding&doi=10.1016%2Fj.bmcl.2017.02.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">SAR of amino pyrrolidines as potent and novel protein-protein interaction inhibitors of the PRC2 complex through EED binding</span></div><div class="casAuthors">Curtin, Michael L.; Pliushchev, Marina A.; Li, Huan-Qiu; Torrent, Maricel; Dietrich, Justin D.; Jakob, Clarissa G.; Zhu, Haizhong; Zhao, Hongyu; Wang, Ying; Ji, Zhiqin; Clark, Richard F.; Sarris, Kathy A.; Selvaraju, Sujatha; Shaw, Bailin; Algire, Mikkel A.; He, Yupeng; Richardson, Paul L.; Sweis, Ramzi F.; Sun, Chaohong; Chiang, Gary G.; Michaelides, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1576-1583</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Herein we disclose SAR studies of a series of dimethylamino pyrrolidines which we recently reported as novel inhibitors of the PRC2 complex through disruption of EED/H3K27me3 binding.  Modification of the indole and benzyl moieties of screening hit 1 provided analogs with substantially improved binding and cellular activities.  This work culminated in the identification of compd. 2, our nanomolar proof-of-concept (PoC) inhibitor which provided on-target tumor growth inhibition in a mouse xenograft model.  X-ray crystal structures of several inhibitors bound in the EED active-site are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkdonJy6cBnbVg90H21EOLACvtfcHk0lgRIbFn9UQ_Zw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Oju7s%253D&md5=3396f3f4c2d3944237cfde004ac4204b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.02.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.02.030%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DM.%2BL.%26aulast%3DPliushchev%26aufirst%3DM.%2BA.%26aulast%3DLi%26aufirst%3DH.-Q.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DDietrich%26aufirst%3DJ.%2BD.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DZ.%26aulast%3DClark%26aufirst%3DR.%2BF.%26aulast%3DSarris%26aufirst%3DK.%2BA.%26aulast%3DSelvaraju%26aufirst%3DS.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DAlgire%26aufirst%3DM.%2BA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26atitle%3DSAR%2520of%2520amino%2520pyrrolidines%2520as%2520potent%2520and%2520novel%2520protein-protein%2520interaction%2520inhibitors%2520of%2520the%2520PRC2%2520complex%2520through%2520EED%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D1576%26epage%3D1583%26doi%3D10.1016%2Fj.bmcl.2017.02.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, B.</span></span> <span> </span><span class="NLM_article-title">An allosteric PRC2 inhibitor targeting EED suppresses tumor progression by modulating the immune response</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">5587</span>â <span class="NLM_lpage">5596</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.can-19-0428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1158%2F0008-5472.CAN-19-0428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=31395608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1KitL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=5587-5596&author=H.+Dongauthor=S.+Liuauthor=X.+Zhangauthor=S.+Chenauthor=L.+Kangauthor=Y.+Chenauthor=S.+Maauthor=X.+Fuauthor=Y.+Liuauthor=H.+Zhangauthor=B.+Zou&title=An+allosteric+PRC2+inhibitor+targeting+EED+suppresses+tumor+progression+by+modulating+the+immune+response&doi=10.1158%2F0008-5472.can-19-0428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric PRC2 inhibitor targeting EED suppresses tumor progression by modulating the immune response</span></div><div class="casAuthors">Dong, Hongping; Liu, Shaojun; Zhang, Xuejie; Chen, Sheng; Kang, Lijing; Chen, Yanni; Ma, Shichao; Fu, Xianlei; Liu, Yanchao; Zhang, Hailong; Zou, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5587-5596</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant activity of polycomb repressive complex 2 (PRC2) is involved in a wide range of human cancer progression.  The WD40 repeat-contg. protein EED is a core component of PRC2 and enhances PRC2 activity through interaction with H3K27me3.  In this study, we report the discovery of a class of pyrimidone compds., represented by BR-001 as potent allosteric inhibitors of PRC2.  X-ray co-crystallog. showed that BR-001 directly binds EED in the H3K27me3-binding pocket.  BR-001 displayed antitumor potency in vitro and in vivo.  In Karpas422 and Pfeiffer xenograft mouse models, twice daily oral dosing with BR-001 resulted in robust anti-tumor activity.  BR-001 was also efficacious in syngeneic CT26 colon tumor-bearing mice; oral dosing of 30 mg/kg of BR-001 led to 59.3% tumor growth suppression and increased frequency of effector CD8+ T-cell infiltrates in tumors.  Pharmacodynamic anal. revealed that CXCL10 was highly upregulated, suggesting that CXCL10 triggers the trafficking of CD8+ T cells toward tumor sites.  Our results demonstrate for the first time that inhibition of EED modulates the tumor immune microenvironment to induce regression of colon tumors and therefore has the potential to be used in combination with immune-oncol. therapy.  Significance: BR-001, a potent inhibitor of the EED subunit of the PRC2 complex, suppresses tumor progression by modulating the tumor microenvironment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriYcNHbku1ZbVg90H21EOLACvtfcHk0liTxl-kLdugMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1KitL8%253D&md5=51331eac62193271e40ea4b9246a348c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-19-0428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-19-0428%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZou%26aufirst%3DB.%26atitle%3DAn%2520allosteric%2520PRC2%2520inhibitor%2520targeting%2520EED%2520suppresses%2520tumor%2520progression%2520by%2520modulating%2520the%2520immune%2520response%26jtitle%3DCancer%2520Res.%26date%3D2019%26volume%3D79%26spage%3D5587%26epage%3D5596%26doi%3D10.1158%2F0008-5472.can-19-0428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, L. A.</span></span> <span> </span><span class="NLM_article-title">Reactive metabolites in the biotransformation of molecules containing a furan ring</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">6</span>â <span class="NLM_lpage">25</span>, <span class="refDoi">Â DOI: 10.1021/tx3003824</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx3003824" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=6-25&author=L.+A.+Peterson&title=Reactive+metabolites+in+the+biotransformation+of+molecules+containing+a+furan+ring&doi=10.1021%2Ftx3003824"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Reactive Metabolites in the Biotransformation of Molecules Containing a Furan Ring</span></div><div class="casAuthors">Peterson, Lisa A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6-25</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many xenobiotics contg. a furan ring are toxic and/or carcinogenic.  The harmful effects of these compds. require furan ring oxidn.  This reaction generates an electrophilic intermediate.  Depending on the furan ring substituents, the intermediate is either an epoxide or a cis-enedione with more ring substitution favoring epoxide formation.  Either intermediate reacts with cellular nucleophiles such as protein or DNA to trigger toxicities.  The reactivity of the metabolite dets. which cellular nucleophiles are targeted.  The toxicity of a particular furan is also influenced by the presence of competing metabolic pathways or efficient detoxification routes.  GSH plays an important role in modulating the harmful effects of this class of compd. by reacting with the reactive metabolite.  However, this may not represent a detoxification step in all cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp4Dr5stLUSrVg90H21EOLACvtfcHk0lgGTEzGIoocag"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN&md5=fe3a280d0b7a99a436ec619abc1e1fa0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Ftx3003824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx3003824%26sid%3Dliteratum%253Aachs%26aulast%3DPeterson%26aufirst%3DL.%2BA.%26atitle%3DReactive%2520metabolites%2520in%2520the%2520biotransformation%2520of%2520molecules%2520containing%2520a%2520furan%2520ring%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2013%26volume%3D26%26spage%3D6%26epage%3D25%26doi%3D10.1021%2Ftx3003824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span>; <span class="NLM_string-name">Fu, X.</span>; <span class="NLM_string-name">Gu, X. J. J.</span>; <span class="NLM_string-name">Huang, Y.</span>; <span class="NLM_string-name">Li, L.</span>; <span class="NLM_string-name">Mi, Y.</span>; <span class="NLM_string-name">Qi, W.</span>; <span class="NLM_string-name">Sendzik, M.</span>; <span class="NLM_string-name">Sun, Y.</span>; <span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Yu, Z.</span>; <span class="NLM_string-name">Zhang, H.</span>; <span class="NLM_string-name">Zhang, J. Y.</span>; <span class="NLM_string-name">Zhang, M.</span>; <span class="NLM_string-name">Zhang, Q.</span>; <span class="NLM_string-name">Zhao, K.</span></span> <span> </span><span class="NLM_article-title">Triazolopyridine Compounds and Uses Thereof</span>. International Patent Application PCT/IB2017/053335, WO<span class="NLM_patent">2017221092A1</span>, <span class="NLM_year">2017</span>, <span class="NLM_publisher-name">Novartis AG</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=H.+M.+Chan&author=X.+Fu&author=X.+J.+J.+Gu&author=Y.+Huang&author=L.+Li&author=Y.+Mi&author=W.+Qi&author=M.+Sendzik&author=Y.+Sun&author=L.+Wang&author=Z.+Yu&author=H.+Zhang&author=J.+Y.+Zhang&author=M.+Zhang&author=Q.+Zhang&author=K.+Zhao&title=Triazolopyridine+Compounds+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DH.%2BM.%26atitle%3DTriazolopyridine%2520Compounds%2520and%2520Uses%2520Thereof%26date%3D2017%26pub%3DNovartis%2520AG" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eusterwiemann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolbier, W. R.</span></span> <span> </span><span class="NLM_article-title">Methyl 2,2-difluoro-2-(fluorosulfonyl)acetate, a difluorocarbene reagent with reactivity comparable to that of trimethylsilyl 2,2-difluoro-2-(fluorosulfonyl)acetate (TFDA)</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">5461</span>â <span class="NLM_lpage">5464</span>, <span class="refDoi">Â DOI: 10.1021/jo300876z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo300876z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1yiur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2012&pages=5461-5464&author=S.+Eusterwiemannauthor=H.+Martinezauthor=W.+R.+Dolbier&title=Methyl+2%2C2-difluoro-2-%28fluorosulfonyl%29acetate%2C+a+difluorocarbene+reagent+with+reactivity+comparable+to+that+of+trimethylsilyl+2%2C2-difluoro-2-%28fluorosulfonyl%29acetate+%28TFDA%29&doi=10.1021%2Fjo300876z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Methyl 2,2-Difluoro-2-(fluorosulfonyl)acetate, a Difluorocarbene Reagent with Reactivity Comparable to That of Trimethylsilyl 2,2-Difluoro-2-(fluorosulfonyl)acetate (TFDA)</span></div><div class="casAuthors">Eusterwiemann, Steffen; Martinez, Henry; Dolbier, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5461-5464</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Under specific high concn., high temp. conditions, Me 2,2-difluoro-2-(fluorosulfonyl)acetate (MDFA) has been found to act as a very efficient source of difluorocarbene, exhibiting carbene reactivity characteristics comparable to those exhibited by trimethylsilyl 2,2-difluoro-2-(fluorosulfonyl)acetate (TFDA).  For example, in reaction with highly unreactive Bu acrylate and using only 2 equiv of MDFA, a yield of 76% of difluorocyclopropane product was obtained after 2 days.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqig5Ak3X8SdrVg90H21EOLACvtfcHk0lijdsdpucdDDQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1yiur8%253D&md5=ba86878d06f3037f7bde41bd1999c613</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjo300876z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo300876z%26sid%3Dliteratum%253Aachs%26aulast%3DEusterwiemann%26aufirst%3DS.%26aulast%3DMartinez%26aufirst%3DH.%26aulast%3DDolbier%26aufirst%3DW.%2BR.%26atitle%3DMethyl%25202%252C2-difluoro-2-%2528fluorosulfonyl%2529acetate%252C%2520a%2520difluorocarbene%2520reagent%2520with%2520reactivity%2520comparable%2520to%2520that%2520of%2520trimethylsilyl%25202%252C2-difluoro-2-%2528fluorosulfonyl%2529acetate%2520%2528TFDA%2529%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2012%26volume%3D77%26spage%3D5461%26epage%3D5464%26doi%3D10.1021%2Fjo300876z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">An efficient copper catalyst for the formation of sulfones from sulfinic acid salts and aryl iodides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">4423</span>â <span class="NLM_lpage">4425</span>, <span class="refDoi">Â DOI: 10.1021/ol0269190</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0269190" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFGjt7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=4423-4425&author=J.+M.+Baskinauthor=Z.+Wang&title=An+efficient+copper+catalyst+for+the+formation+of+sulfones+from+sulfinic+acid+salts+and+aryl+iodides&doi=10.1021%2Fol0269190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An Efficient Copper Catalyst for the Formation of Sulfones from Sulfinic Acid Salts and Aryl Iodides</span></div><div class="casAuthors">Baskin, Jeremy M.; Wang, Zhaoyin</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4423-4425</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel copper-catalyzed method for the coupling of sulfinic acid salts and aryl iodides is described.  A variety of Me and diaryl sulfones have been formed in excellent yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoibEB4_JcsSLVg90H21EOLACvtfcHk0lijdsdpucdDDQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFGjt7Y%253D&md5=ed4d7aa2b442c75197e8ff887f27bcff</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fol0269190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0269190%26sid%3Dliteratum%253Aachs%26aulast%3DBaskin%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DAn%2520efficient%2520copper%2520catalyst%2520for%2520the%2520formation%2520of%2520sulfones%2520from%2520sulfinic%2520acid%2520salts%2520and%2520aryl%2520iodides%26jtitle%3DOrg.%2520Lett.%26date%3D2002%26volume%3D4%26spage%3D4423%26epage%3D4425%26doi%3D10.1021%2Fol0269190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwight, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toms, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parillon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohemmad, Q. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of Brigatinib (AP26113), a phosphine oxide-containing, potent, orally active inhibitor of anaplastic lymphoma kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4948</span>â <span class="NLM_lpage">4964</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b00306</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00306" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlCgur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4948-4964&author=W.-S.+Huangauthor=S.+Liuauthor=D.+Zouauthor=M.+Thomasauthor=Y.+Wangauthor=T.+Zhouauthor=J.+Romeroauthor=A.+Kohlmannauthor=F.+Liauthor=J.+Qiauthor=L.+Caiauthor=T.+A.+Dwightauthor=Y.+Xuauthor=R.+Xuauthor=R.+Doddauthor=A.+Tomsauthor=L.+Parillonauthor=X.+Luauthor=R.+Anjumauthor=S.+Zhangauthor=F.+Wangauthor=J.+Keatsauthor=S.+D.+Wardwellauthor=Y.+Ningauthor=Q.+Xuauthor=L.+E.+Moranauthor=Q.+K.+Mohemmadauthor=H.+G.+Jangauthor=T.+Clacksonauthor=N.+I.+Narasimhanauthor=V.+M.+Riveraauthor=X.+Zhuauthor=D.+Dalgarnoauthor=W.+C.+Shakespeare&title=Discovery+of+Brigatinib+%28AP26113%29%2C+a+phosphine+oxide-containing%2C+potent%2C+orally+active+inhibitor+of+anaplastic+lymphoma+kinase&doi=10.1021%2Facs.jmedchem.6b00306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Brigatinib (AP26113), a Phosphine Oxide-Containing, Potent, Orally Active Inhibitor of Anaplastic Lymphoma Kinase</span></div><div class="casAuthors">Huang, Wei-Sheng; Liu, Shuangying; Zou, Dong; Thomas, Mathew; Wang, Yihan; Zhou, Tianjun; Romero, Jan; Kohlmann, Anna; Li, Feng; Qi, Jiwei; Cai, Lisi; Dwight, Timothy A.; Xu, Yongjin; Xu, Rongsong; Dodd, Rory; Toms, Angela; Parillon, Lois; Lu, Xiaohui; Anjum, Rana; Zhang, Sen; Wang, Frank; Keats, Jeffrey; Wardwell, Scott D.; Ning, Yaoyu; Xu, Qihong; Moran, Lauren E.; Mohemmad, Qurish K.; Jang, Hyun Gyung; Clackson, Tim; Narasimhan, Narayana I.; Rivera, Victor M.; Zhu, Xiaotian; Dalgarno, David; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4948-4964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of echinoderm microtubule-assocd. protein-like 4 (EML4)-anaplastic lymphoma kinase pos. (ALK+) non-small-cell lung cancer (NSCLC), secondary mutations within the ALK kinase domain have emerged as a major resistance mechanism to both first- and second-generation ALK inhibitors.  This report describes the design and synthesis of a series of 2,4-diarylaminopyrimidine-based potent and selective ALK inhibitors culminating in identification of the investigational clin. candidate brigatinib.  A unique structural feature of brigatinib is a phosphine oxide, an overlooked but novel hydrogen-bond acceptor that drives potency and selectivity in addn. to favorable ADME properties.  Brigatinib displayed low nanomolar IC50s against native ALK and all tested clin. relevant ALK mutants in both enzyme-based biochem. and cell-based viability assays and demonstrated efficacy in multiple ALK+ xenografts in mice, including Karpas-299 (anaplastic large-cell lymphomas [ALCL]) and H3122 (NSCLC).  Brigatinib represents the most clin. advanced phosphine oxide-contg. drug candidate to date and is currently being evaluated in a global phase 2 registration trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_iqprSA4QbVg90H21EOLACvtfcHk0lgLSBFk5Ybh2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlCgur8%253D&md5=4cb93ff587579c97fda8aff0c7d43317</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00306%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DRomero%26aufirst%3DJ.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DDwight%26aufirst%3DT.%2BA.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DDodd%26aufirst%3DR.%26aulast%3DToms%26aufirst%3DA.%26aulast%3DParillon%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DWardwell%26aufirst%3DS.%2BD.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMoran%26aufirst%3DL.%2BE.%26aulast%3DMohemmad%26aufirst%3DQ.%2BK.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DNarasimhan%26aufirst%3DN.%2BI.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDalgarno%26aufirst%3DD.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26atitle%3DDiscovery%2520of%2520Brigatinib%2520%2528AP26113%2529%252C%2520a%2520phosphine%2520oxide-containing%252C%2520potent%252C%2520orally%2520active%2520inhibitor%2520of%2520anaplastic%2520lymphoma%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4948%26epage%3D4964%26doi%3D10.1021%2Facs.jmedchem.6b00306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1016/s0076-6879(97)76066-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2Fs0076-6879%2897%2976066-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lgLSBFk5Ybh2w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2Fs0076-6879%2897%2976066-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>â <span class="NLM_lpage">1025</span>, <span class="refDoi">Â DOI: 10.1107/s0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2Fs0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0liGt3QSww8THQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2Fs0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis of EZH2 recognition by EED</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1306</span>â <span class="NLM_lpage">1315</span>, <span class="refDoi">Â DOI: 10.1016/j.str.2007.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1016%2Fj.str.2007.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=17937919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKhsLzE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=1306-1315&author=Z.+Hanauthor=X.+Xingauthor=M.+Huauthor=Y.+Zhangauthor=P.+Liuauthor=J.+Chai&title=Structural+basis+of+EZH2+recognition+by+EED&doi=10.1016%2Fj.str.2007.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of EZH2 Recognition by EED</span></div><div class="casAuthors">Han, Zhifu; Xing, Xinmiao; Hu, Min; Zhang, Yin; Liu, Peiyuan; Chai, Jijie</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1306-1315</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The WD-repeat domain is a highly conserved recognition module in eukaryotes involved in diverse cellular processes.  It is still not well understood how the bottom of a WD-repeat domain recognizes its binding partners.  The WD-repeat-contg. protein EED is one component of the PRC2 complex that possesses histone methyltransferase activity required for gene repression.  Here we report the crystal structure of EED in complex with a 30 residue peptide from EZH2.  The structure reveals that the peptide binds to the bottom of the WD-repeat domain of EED.  The structural determinants of EZH2-EED interaction are present not only in EZH2 and EZH1 but also in its Drosophila homolog E(Z), suggesting that the recognition of ESC by E(Z) in Drosophila employs similar structural motifs.  Structure-based mutagenesis identified crit. residues from both EED and EZH2 for their interaction.  The structure presented here may provide a template for understanding of how WD-repeat proteins recognize their interacting proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8t4Clbu4NV7Vg90H21EOLACvtfcHk0liGt3QSww8THQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKhsLzE&md5=1b963045c45d252eff1a36336b85a8c7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2007.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2007.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DXing%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DChai%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520of%2520EZH2%2520recognition%2520by%2520EED%26jtitle%3DStructure%26date%3D2007%26volume%3D15%26spage%3D1306%26epage%3D1315%26doi%3D10.1016%2Fj.str.2007.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>â <span class="NLM_lpage">501</span>, <span class="refDoi">Â DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhjtsHuUlkwJQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, 2.10.3; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+S.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Womack, T. O.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Bricogne, G.</span></span> <i>GRADE</i>, 1.2.13; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=O.+S.+Smart&author=T.+O.+Womack&author=A.+Sharff&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=C.+Vonrhein&author=G.+Bricogne&title=GRADE"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26jtitle%3DGRADE%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Daohai Du, Dandan Xu, Licheng Zhu, Giulia Stazi, Clemens Zwergel, Yanli Liu, Zhongyuan Luo, Yuanqing Li, Yuanyuan Zhang, Kongkai Zhu, Yiluan Ding, Jingqiu Liu, Shijie Fan, Kaiyan Zhao, Naixia Zhang, Xiangqian Kong, Hualiang Jiang, Kaixian Chen, Kehao Zhao, Sergio Valente, Jinrong Min, Wenhu Duan, <span class="NLM_string-name hlFld-ContribAuthor">Cheng Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Development of Small-Molecule PRC2 Inhibitors Targeting EZH2âEED Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8194-8207. <a href="https://doi.org/10.1021/acs.jmedchem.0c02261" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02261</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02261%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Guided%252BDevelopment%252Bof%252BSmall-Molecule%252BPRC2%252BInhibitors%252BTargeting%252BEZH2%2525E2%252580%252593EED%252BInteraction%26aulast%3DDu%26aufirst%3DDaohai%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D30122020%26date%3D02062021%26volume%3D64%26issue%3D12%26spage%3D8194%26epage%3D8207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">M. Cynthia Martin, Guihua Zeng, Jindan Yu, <span class="NLM_string-name hlFld-ContribAuthor">Gary E. Schiltz</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Approaches for Targeting the Polycomb Repressive Complex 2 (PRC2) in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15344-15370. <a href="https://doi.org/10.1021/acs.jmedchem.0c01344" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01344</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01344%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall%252BMolecule%252BApproaches%252Bfor%252BTargeting%252Bthe%252BPolycomb%252BRepressive%252BComplex%252B2%252B%252528PRC2%252529%252Bin%252BCancer%26aulast%3DMartin%26aufirst%3DM.%2BCynthia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31072020%26date%3D07122020%26volume%3D63%26issue%3D24%26spage%3D15344%26epage%3D15370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative small-molecule inhibitors of EZH2 and EED that target the PRC2 activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure-guided design and optimization of new EED inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structures of EED bound to inhibitors. Key interactions of compound (a) EEDi-5285 with EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7F">6W7F</a>), (b) EEDi-1056 with EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7G">6W7G</a>) and (c) EED226 with EED (PDB accession code 5GSA) for comparison. Dashed lines represent hydrogen bonds. Water molecules are depicted as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pharmacodynamic analysis of EEDi-5285 and EEDi-1056 in KARPAS422 xenograft tumors. SCID mice bearing KARPAS422 tumors were treated with a single oral administration of EEDi-5285 or EEDi-1056 at 100 mg/kg. Mice were euthanized at the 24 h time-point and tumor tissues were collected. Histone proteins were extracted from the treated cells with the EpiQuik Total Histone Extraction Kit, and the level of H3K27me3 was examined by Western blotting analysis. Total histone H3 was used as the loading control. Three mice were used for each compound with each mouse bearing one tumor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Antitumor activity of EEDi-5285 and EEDi-1056 in a KARPAS422 xenograft model in SCID mice. Each group had seven mice and each mouse had one tumor. (B) Changes in animal body weights.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>8â19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl 2-(diphenylmethyleneamino)acetate, NaH, rt, 2 h; (b) 3 N HCl in THF, rt, 1 h, 70%; (c) HCO<sub>2</sub>H, Ac<sub>2</sub>O, rt, 2 h; (d) POCl<sub>3</sub>, dioxane, reflux, 4 h, 70%; (e) (i) mCPBA, DCM, (ii) furan-2-ylmethanamine, rt, 3 h, 55%; (f) phenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxaneâH<sub>2</sub>O, 90 Â°C overnight, 90%; (g) Li(OH)<sub>2</sub>, THFâH<sub>2</sub>O, 90%; (h) MeNH<sub>2</sub>Â·HCl, DIPEA, HATU, 90%</p></p></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>7</b> and <b>20â34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) CH<sub>3</sub>COCl, MeOH, 79%; (b) DIBAL-H, 56%; (c) DIAD, PPh<sub>3</sub>, phthalidomide, 70%; (d) NH<sub>2</sub>Â·NH<sub>2</sub>. H<sub>2</sub>O, EtOH, 80%; (e) HCO<sub>2</sub>H, Ac<sub>2</sub>O, 60%; (f) POCl<sub>3</sub>, dioxane, 47.0%; (g) (i) mCPBA, DCM, (ii) furan-2-ylmethanamine, rt, 3 h, 52%; (h) phenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxaneâH<sub>2</sub>O, 90 Â°C overnight, 88%; (i) (5-fluoro-2,3-dihydrobenzofuran-4-yl)methanamine, 40 Â°C, 24 h, 60%; (j) phenylboronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxaneâH<sub>2</sub>O, 90 Â°C overnight, 88%; (k) NIS, DMF, 0 Â°C, 70%; (l) copper (I) iodide, methyl 2,2-difluoro-2-(fluorosulfonyl)acetate, PdCl<sub>2</sub>(dppf)Cl<sub>2</sub>, 42%; (m) MeSO<sub>2</sub>Na, CuI, DMSO, 50%; (n) dimethylphosphine oxide, Pd<sub>2</sub>(dba)<sub>3</sub>, xantphos, Et<sub>3</sub>N, dioxane, 40%.</p></p></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>35â40</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, dioxaneâH<sub>2</sub>O, 90 Â°C overnight, 88%; (b) H<sub>2</sub>, Pd/C, MeOH; (c) NIS, DMF, 0 Â°C, 70%; (d) MeSO<sub>2</sub>Na, CuI, DMSO, 50%; (e) TFA, DCM; (f) DCM, Et<sub>3</sub>N, alkyl chloride; (g) DCM/AcOH (1:1), alkyl aldehyde or ketone, NaBH(OAc)<sub>3</sub>.</p></p></figure><figure data-id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/medium/jm0c00479_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of (5-Fluoro-2,3-dihydrobenzofuran-4-yl)methanamine<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00479/20200701/images/large/jm0c00479_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00479&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2-bromo-1,1-diethoxyethane, K<sub>2</sub>CO<sub>3</sub>, DMF, 110 Â°C; (b) PPA, toluene, 90 Â°C; (c) Zn(CN)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DMF; (d) LAH, THF, 50 Â°C; (e) H<sub>2</sub>, Pd/C, 50 Â°C.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i91">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56265" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56265" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 29 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. B.</span></span> <span> </span><span class="NLM_article-title">A gene complex controlling segmentation in Drosophila</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">565</span>â <span class="NLM_lpage">570</span>, <span class="refDoi">Â DOI: 10.1038/276565a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2F276565a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=103000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A280%3ADyaE1M%252FntlWgsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1978&pages=565-570&author=E.+B.+Lewis&title=A+gene+complex+controlling+segmentation+in+Drosophila&doi=10.1038%2F276565a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A gene complex controlling segmentation in Drosophila</span></div><div class="casAuthors">Lewis E B</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">5688</span>),
    <span class="NLM_cas:pages">565-70</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The bithorax gene complex in Drosophila contains a minimum of eight genes that seem to code for substances controlling levels of thoracic and abdominal development.  The state of repression of at least four of these genes is controlled by cis-regulatory elements and a separate locus (Polycomb) seems to code for a repressor of the complex.  The wild-type and mutant segmentation patterns are consistent with an antero-posterior gradient in repressor concentration along the embryo and a proximo-distal gradient along the chromosome in the affinities for repressor of each gene's cis-regulatory element.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkFLr2ycH_oSsZ1x6zOs0vfW6udTcc2eZpvKtTDuNgtrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE1M%252FntlWgsQ%253D%253D&md5=984f6a3650a54711616639bbdc5fa131</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F276565a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F276565a0%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DE.%2BB.%26atitle%3DA%2520gene%2520complex%2520controlling%2520segmentation%2520in%2520Drosophila%26jtitle%3DNature%26date%3D1978%26volume%3D276%26spage%3D565%26epage%3D570%26doi%3D10.1038%2F276565a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Margueron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">The polycomb complex PRC2 and its mark in life</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>469</i></span>,  <span class="NLM_fpage">343</span>â <span class="NLM_lpage">349</span>, <span class="refDoi">Â DOI: 10.1038/nature09784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnature09784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=21248841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=343-349&author=R.+Margueronauthor=D.+Reinberg&title=The+polycomb+complex+PRC2+and+its+mark+in+life&doi=10.1038%2Fnature09784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Polycomb complex PRC2 and its mark in life</span></div><div class="casAuthors">Margueron, Raphael; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7330</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group proteins maintain the gene-expression pattern of different cells that is set during early development by regulating chromatin structure.  In mammals, 2 main Polycomb group complexes exist: Polycomb repressive complex 1 (PRC1) and 2 (PRC2).  PRC1 compacts chromatin and catalyzes the monoubiquitination of histone H2A.  PRC2 also contributes to chromatin compaction, and catalyzes the methylation of histone H3 at Lys-27.  PRC2 is involved in various biol. processes, including differentiation, maintaining cell identity and proliferation, and stem-cell plasticity.  Recent studies of PRC2 have expanded the perspectives on its function and regulation, and uncovered a role for non-coding RNA in the recruitment of PRC2 to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fYmuBqnMA7Vg90H21EOLACvtfcHk0ljk_oE1Wnr08Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D&md5=5f15e91cc6ff765f649842041f333841</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature09784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09784%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DThe%2520polycomb%2520complex%2520PRC2%2520and%2520its%2520mark%2520in%2520life%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D343%26epage%3D349%26doi%3D10.1038%2Fnature09784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingston, R. E.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of polycomb gene silencing: knowns and unknowns</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">697</span>â <span class="NLM_lpage">708</span>, <span class="refDoi">Â DOI: 10.1038/nrm2763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnrm2763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=19738629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtV2qtL%252FO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=697-708&author=J.+A.+Simonauthor=R.+E.+Kingston&title=Mechanisms+of+polycomb+gene+silencing%3A+knowns+and+unknowns&doi=10.1038%2Fnrm2763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Polycomb gene silencing: Knowns and unknowns</span></div><div class="casAuthors">Simon, Jeffrey A.; Kingston, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">697-708</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb proteins form chromatin-modifying complexes that implement transcriptional silencing in higher eukaryotes.  Hundreds of genes are silenced by Polycomb proteins, including dozens of genes that encode crucial developmental regulators in organisms ranging from plants to humans.  Two main families of complexes, called Polycomb repressive complex 1 (PRC1) and PRC2, are targeted to repressed regions.  Recent studies have advanced our understanding of these complexes, including their potential mechanisms of gene silencing, the roles of chromatin modifications, their means of delivery to target genes and the functional distinctions among variant complexes.  Emerging concepts include the existence of a Polycomb barrier to transcription elongation and the involvement of non-coding RNAs in the targeting of Polycomb complexes.  These findings have an impact on the epigenetic programming of gene expression in many biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAhDlHpccs6bVg90H21EOLACvtfcHk0ljk_oE1Wnr08Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtV2qtL%252FO&md5=416b665e14eb322565ceff685d30a647</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm2763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2763%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DJ.%2BA.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26atitle%3DMechanisms%2520of%2520polycomb%2520gene%2520silencing%253A%2520knowns%2520and%2520unknowns%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2009%26volume%3D10%26spage%3D697%26epage%3D708%26doi%3D10.1038%2Fnrm2763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levine, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kingston, R. E.</span></span> <span> </span><span class="NLM_article-title">The core of the polycomb repressive complex is compositionally and functionally conserved in flies and humans</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">6070</span>â <span class="NLM_lpage">6078</span>, <span class="refDoi">Â DOI: 10.1128/mcb.22.17.6070-6078.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1128%2FMCB.22.17.6070-6078.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=12167701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmt1entrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=6070-6078&author=S.+S.+Levineauthor=A.+Weissauthor=H.+Erdjument-Bromageauthor=Z.+Shaoauthor=P.+Tempstauthor=R.+E.+Kingston&title=The+core+of+the+polycomb+repressive+complex+is+compositionally+and+functionally+conserved+in+flies+and+humans&doi=10.1128%2Fmcb.22.17.6070-6078.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The core of the Polycomb repressive complex is compositionally and functionally conserved in flies and humans</span></div><div class="casAuthors">Levine, Stuart S.; Weiss, Alona; Erdjument-Bromage, Hediye; Shao, Zhaohui; Tempst, Paul; Kingston, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6070-6078</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Polycomb group (PcG) genes are required to maintain homeotic genes in a silenced state during development in Drosophila and mammals and are thought to form several distinct silencing complexes that maintain homeotic gene repression during development.  Mutations in the PcG genes result in developmental defects and have been implicated in human cancer.  Although some PcG protein domains are conserved between flies and humans, substantial regions of several PcG proteins are divergent and humans contain multiple versions of each PcG gene.  To det. the effects of these changes on the compn. and function of a PcG complex, we have purified a human Polycomb repressive complex from HeLa cells (hPRC-H) that contains homologues of PcG proteins found in Drosophila embryonic PRC1 (dPRC1).  HPRC-H was found to have fewer components than dPRC1, retaining the PcG core proteins of dPRC1 but lacking most non-PcG proteins.  Prepns. of hPRC-H contained either two or three different homologues of most of the core PcG proteins, including a new Ph homolog we have named HPH3.  Despite differences in compn., dPRC1 and hPRC-H have similar functions: hPRC-H is able to efficiently block remodeling of nucleosomal arrays through a mechanism that does not block the ability of nucleases to access and cleave the arrays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRfUn5IP2kuLVg90H21EOLACvtfcHk0ljk_oE1Wnr08Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmt1entrs%253D&md5=72265e53cc9036f9c97ede6338032027</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FMCB.22.17.6070-6078.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.22.17.6070-6078.2002%26sid%3Dliteratum%253Aachs%26aulast%3DLevine%26aufirst%3DS.%2BS.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26atitle%3DThe%2520core%2520of%2520the%2520polycomb%2520repressive%2520complex%2520is%2520compositionally%2520and%2520functionally%2520conserved%2520in%2520flies%2520and%2520humans%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2002%26volume%3D22%26spage%3D6070%26epage%3D6078%26doi%3D10.1128%2Fmcb.22.17.6070-6078.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dimou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dincman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evanno, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemmill, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drabkin, H. A.</span></span> <span> </span><span class="NLM_article-title">Epigenetics during EMT in lung cancer: EZH2 as a potential therapeutic target</span>. <i>Cancer Treat. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">40</span>â <span class="NLM_lpage">48</span>, <span class="refDoi">Â DOI: 10.1016/j.ctarc.2017.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1016%2Fj.ctarc.2017.06.003" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=40-48&author=A.+Dimouauthor=T.+Dincmanauthor=E.+Evannoauthor=R.+M.+Gemmillauthor=J.+Rocheauthor=H.+A.+Drabkin&title=Epigenetics+during+EMT+in+lung+cancer%3A+EZH2+as+a+potential+therapeutic+target&doi=10.1016%2Fj.ctarc.2017.06.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ctarc.2017.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ctarc.2017.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DDimou%26aufirst%3DA.%26aulast%3DDincman%26aufirst%3DT.%26aulast%3DEvanno%26aufirst%3DE.%26aulast%3DGemmill%26aufirst%3DR.%2BM.%26aulast%3DRoche%26aufirst%3DJ.%26aulast%3DDrabkin%26aufirst%3DH.%2BA.%26atitle%3DEpigenetics%2520during%2520EMT%2520in%2520lung%2520cancer%253A%2520EZH2%2520as%2520a%2520potential%2520therapeutic%2520target%26jtitle%3DCancer%2520Treat.%2520Res.%2520Commun.%26date%3D2017%26volume%3D12%26spage%3D40%26epage%3D48%26doi%3D10.1016%2Fj.ctarc.2017.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Role of histone H3 lysine 27 methylation in polycomb-group silencing</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1039</span>â <span class="NLM_lpage">1043</span>, <span class="refDoi">Â DOI: 10.1126/science.1076997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1126%2Fscience.1076997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=12351676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD38Xot12rs7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1039-1043&author=R.+Caoauthor=L.+Wangauthor=H.+Wangauthor=L.+Xiaauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=R.+S.+Jonesauthor=Y.+Zhang&title=Role+of+histone+H3+lysine+27+methylation+in+polycomb-group+silencing&doi=10.1126%2Fscience.1076997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Histone H3 Lysine 27 Methylation in Polycomb-Group Silencing</span></div><div class="casAuthors">Cao, Ru; Wang, Liangjun; Wang, Hengbin; Xia, Li; Erdjument-Bromage, Hediye; Tempst, Paul; Jones, Richard S.; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5595</span>),
    <span class="NLM_cas:pages">1039-1043</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Polycomb group (PcG) proteins play important roles in maintaining the silent state of HOX genes.  Recent studies have implicated histone methylation in long-term gene silencing.  However, a connection between PcG-mediated gene silencing and histone methylation has not been established.  Here the authors report the purifn. and characterization of an EED-EZH2 complex, the human counterpart of the Drosophila ESC-E(Z) complex.  They demonstrate that the complex specifically methylates nucleosomal histone H3 at lysine 27 (H3-K27).  Using chromatin immunopptn. assays, the authors show that H3-K27 methylation colocalizes with, and is dependent on, E(Z) binding at an Ultrabithorax medÂ° (Ubx) Polycomb response element (PRE), and that this methylation correlates with Ubx repression.  Methylation on H3-K27 facilitates binding of Polycomb (PC), a component of the PRC1 complex, to histone H3 amino-terminal tail.  Thus, these studies establish a link between histone methylation and PcG-mediated gene silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEnqusXKZEbLVg90H21EOLACvtfcHk0ljW0DI6AKXnFg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xot12rs7g%253D&md5=8f24c05ec935c9110b2ff85b65911396</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.1076997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1076997%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DR.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DRole%2520of%2520histone%2520H3%2520lysine%252027%2520methylation%2520in%2520polycomb-group%2520silencing%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1039%26epage%3D1043%26doi%3D10.1126%2Fscience.1076997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmichev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishioka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">Histone methyltransferase activity associated with a human multiprotein complex containing the enhancer of zeste protein</span>. <i>Gene Dev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2893</span>â <span class="NLM_lpage">2905</span>, <span class="refDoi">Â DOI: 10.1101/gad.1035902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1101%2Fgad.1035902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=12435631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD38XovVejsLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=2893-2905&author=A.+Kuzmichevauthor=K.+Nishiokaauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=D.+Reinberg&title=Histone+methyltransferase+activity+associated+with+a+human+multiprotein+complex+containing+the+enhancer+of+zeste+protein&doi=10.1101%2Fgad.1035902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferase activity associated with a human multiprotein complex containing the Enhancer of Zeste protein</span></div><div class="casAuthors">Kuzmichev, Andrei; Nishioka, Kenichi; Erdjument-Bromage, Hediye; Tempst, Paul; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2893-2905</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Enhancer of Zeste [E(z)] is a Polycomb-group transcriptional repressor and one of the founding members of the family of SET domain-contg. proteins.  Several SET-domain proteins possess intrinsic histone methyltransferase (HMT) activity.  However, recombinant E(z) protein was found to be inactive in a HMT assay.  Here we report the isolation of a multiprotein E(z) complex that contains extra sex combs, suppressor of zeste-12 [Su(z)12], and the histone binding proteins RbAp46/RbAp48.  This complex, which we termed Polycomb repressive complex (PRC) 2, possesses HMT activity with specificity for Lys 9 (K9) and Lys 27 (K27) of histone H3.  The HMT activity of PRC2 is dependent on an intact SET domain in the E(z) protein.  We hypothesize that transcriptional repression by the E(z) protein involves methylation-dependent recruitment of PRC1.  The presence of Su(z)12, a strong suppressor of position effect variegation, in PRC2 suggests that PRC2 may play a widespread role in heterochromatin-mediated silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxD09IiaeCvLVg90H21EOLACvtfcHk0ljW0DI6AKXnFg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XovVejsLY%253D&md5=b61767bc9b30b403f5fc7bb202281da1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fgad.1035902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1035902%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmichev%26aufirst%3DA.%26aulast%3DNishioka%26aufirst%3DK.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DHistone%2520methyltransferase%2520activity%2520associated%2520with%2520a%2520human%2520multiprotein%2520complex%2520containing%2520the%2520enhancer%2520of%2520zeste%2520protein%26jtitle%3DGene%2520Dev.%26date%3D2002%26volume%3D16%26spage%3D2893%26epage%3D2905%26doi%3D10.1101%2Fgad.1035902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span> <span> </span><span class="NLM_article-title">An allosteric PRC2 inhibitor targeting the H3K27me3 binding pocket of EED</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">381</span>â <span class="NLM_lpage">388</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnchembio.2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=28135235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvV2hsLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=381-388&author=W.+Qiauthor=K.+Zhaoauthor=J.+Guauthor=Y.+Huangauthor=Y.+Wangauthor=H.+Zhangauthor=M.+Zhangauthor=J.+Zhangauthor=Z.+Yuauthor=L.+Liauthor=L.+Tengauthor=S.+Chuaiauthor=C.+Zhangauthor=M.+Zhaoauthor=H.+Chanauthor=Z.+Chenauthor=D.+Fangauthor=Q.+Feiauthor=L.+Fengauthor=L.+Fengauthor=Y.+Gaoauthor=H.+Geauthor=X.+Geauthor=G.+Liauthor=A.+Lingelauthor=Y.+Linauthor=Y.+Liuauthor=F.+Luoauthor=M.+Shiauthor=L.+Wangauthor=Z.+Wangauthor=Y.+Yuauthor=J.+Zengauthor=C.+Zengauthor=L.+Zhangauthor=Q.+Zhangauthor=S.+Zhouauthor=C.+Oyangauthor=P.+Atadjaauthor=E.+Li&title=An+allosteric+PRC2+inhibitor+targeting+the+H3K27me3+binding+pocket+of+EED&doi=10.1038%2Fnchembio.2304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric PRC2 inhibitor targeting the H3K27me3 binding pocket of EED</span></div><div class="casAuthors">Qi, Wei; Zhao, Kehao; Gu, Justin; Huang, Ying; Wang, Youzhen; Zhang, Hailong; Zhang, Man; Zhang, Jeff; Yu, Zhengtian; Li, Ling; Teng, Lin; Chuai, Shannon; Zhang, Chao; Zhao, Mengxi; Chan, Ho Man; Chen, Zijun; Fang, Douglas; Fei, Qi; Feng, Leying; Feng, Lijian; Gao, Yuan; Ge, Hui; Ge, Xinjian; Li, Guobin; Lingel, Andreas; Lin, Ying; Liu, Yueqin; Luo, Fangjun; Shi, Minlong; Wang, Long; Wang, Zhaofu; Yu, Yanyan; Zeng, Jue; Zeng, Chenhui; Zhang, Lijun; Zhang, Qiong; Zhou, Shaolian; Oyang, Counde; Atadja, Peter; Li, En</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">381-388</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) consists of three core subunits, EZH2, EED and SUZ12, and plays pivotal roles in transcriptional regulation.  The catalytic subunit EZH2 methylates histone H3 lysine 27 (H3K27), and its activity is further enhanced by the binding of EED to trimethylated H3K27 (H3K27me3).  Small-mol. inhibitors that compete with the cofactor S-adenosylmethionine (SAM) have been reported.  Here we report the discovery of EED226, a potent and selective PRC2 inhibitor that directly binds to the H3K27me3 binding pocket of EED.  EED226 induces a conformational change upon binding EED, leading to loss of PRC2 activity.  EED226 shows similar activity to SAM-competitive inhibitors in blocking H3K27 methylation of PRC2 target genes and inducing regression of human lymphoma xenograft tumors.  Interestingly, EED226 also effectively inhibits PRC2 contg. a mutant EZH2 protein resistant to SAM-competitive inhibitors.  Together, we show that EED226 inhibits PRC2 activity via an allosteric mechanism and offers an opportunity for treatment of PRC2-dependent cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjpXExCMohI7Vg90H21EOLACvtfcHk0lhnK5Bak6dPlA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvV2hsLc%253D&md5=62d10723685450ca7ffd599a61134d42</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2304%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DTeng%26aufirst%3DL.%26aulast%3DChuai%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DFang%26aufirst%3DD.%26aulast%3DFei%26aufirst%3DQ.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DGe%26aufirst%3DH.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DLingel%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DF.%26aulast%3DShi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DOyang%26aufirst%3DC.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DAn%2520allosteric%2520PRC2%2520inhibitor%2520targeting%2520the%2520H3K27me3%2520binding%2520pocket%2520of%2520EED%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D381%26epage%3D388%26doi%3D10.1038%2Fnchembio.2304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvaraju, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comess, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">The, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lima-Fernandes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinge, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algire, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osterling, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindley, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasinski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span> <span> </span><span class="NLM_article-title">The EED protein-protein interaction inhibitor A-395 inactivates the PRC2 complex</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">389</span>â <span class="NLM_lpage">395</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnchembio.2306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=28135237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvV2hsbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=389-395&author=Y.+Heauthor=S.+Selvarajuauthor=M.+L.+Curtinauthor=C.+G.+Jakobauthor=H.+Zhuauthor=K.+M.+Comessauthor=B.+Shawauthor=J.+Theauthor=E.+Lima-Fernandesauthor=M.+M.+Szewczykauthor=D.+Chengauthor=K.+L.+Klingeauthor=H.-Q.+Liauthor=M.+Pliushchevauthor=M.+A.+Algireauthor=D.+Maagauthor=J.+Guoauthor=J.+Dietrichauthor=S.+C.+Panchalauthor=A.+M.+Petrosauthor=R.+F.+Sweisauthor=M.+Torrentauthor=L.+J.+Bigelowauthor=G.+Senisterraauthor=F.+Liauthor=S.+Kennedyauthor=Q.+Wuauthor=D.+J.+Osterlingauthor=D.+J.+Lindleyauthor=W.+Gaoauthor=S.+Galasinskiauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=F.+G.+Buchananauthor=C.+H.+Arrowsmithauthor=G.+G.+Chiangauthor=C.+Sunauthor=W.+N.+Pappano&title=The+EED+protein-protein+interaction+inhibitor+A-395+inactivates+the+PRC2+complex&doi=10.1038%2Fnchembio.2306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The EED protein-protein interaction inhibitor A-395 inactivates the PRC2 complex</span></div><div class="casAuthors">He, Yupeng; Selvaraju, Sujatha; Curtin, Michael L.; Jakob, Clarissa G.; Zhu, Haizhong; Comess, Kenneth M.; Shaw, Bailin; The, Juliana; Lima-Fernandes, Evelyne; Szewczyk, Magdalena M.; Cheng, Dong; Klinge, Kelly L.; Li, Huan-Qiu; Pliushchev, Marina; Algire, Mikkel A.; Maag, David; Guo, Jun; Dietrich, Justin; Panchal, Sanjay C.; Petros, Andrew M.; Sweis, Ramzi F.; Torrent, Maricel; Bigelow, Lance J.; Senisterra, Guillermo; Li, Fengling; Kennedy, Steven; Wu, Qin; Osterling, Donald J.; Lindley, David J.; Gao, Wenqing; Galasinski, Scott; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Buchanan, Fritz G.; Arrowsmith, Cheryl H.; Chiang, Gary G.; Sun, Chaohong; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">389-395</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) is a regulator of epigenetic states required for development and homeostasis.  PRC2 trimethylates histone H3 at lysine 27 (H3K27me3), which leads to gene silencing, and is dysregulated in many cancers.  The embryonic ectoderm development (EED) protein is an essential subunit of PRC2 that has both a scaffolding function and an H3K27me3-binding function.  Here we report the identification of A-395, a potent antagonist of the H3K27me3 binding functions of EED.  Structural studies demonstrate that A-395 binds to EED in the H3K27me3-binding pocket, thereby preventing allosteric activation of the catalytic activity of PRC2.  Phenotypic effects obsd. in vitro and in vivo are similar to those of known PRC2 enzymic inhibitors; however, A-395 retains potent activity against cell lines resistant to the catalytic inhibitors.  A-395 represents a first-in-class antagonist of PRC2 protein-protein interactions (PPI) for use as a chem. probe to investigate the roles of EED-contg. protein complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX5rHeR_Z5mLVg90H21EOLACvtfcHk0lhnK5Bak6dPlA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvV2hsbs%253D&md5=d41eee098ed123f8c7f50babe93d7692</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2306%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DSelvaraju%26aufirst%3DS.%26aulast%3DCurtin%26aufirst%3DM.%2BL.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DThe%26aufirst%3DJ.%26aulast%3DLima-Fernandes%26aufirst%3DE.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DKlinge%26aufirst%3DK.%2BL.%26aulast%3DLi%26aufirst%3DH.-Q.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DAlgire%26aufirst%3DM.%2BA.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DDietrich%26aufirst%3DJ.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DBigelow%26aufirst%3DL.%2BJ.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DOsterling%26aufirst%3DD.%2BJ.%26aulast%3DLindley%26aufirst%3DD.%2BJ.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DGalasinski%26aufirst%3DS.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DThe%2520EED%2520protein-protein%2520interaction%2520inhibitor%2520A-395%2520inactivates%2520the%2520PRC2%2520complex%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D389%26epage%3D395%26doi%3D10.1038%2Fnchembio.2306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Croce, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Transcriptional regulation by Polycomb group proteins</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1147</span>â <span class="NLM_lpage">1155</span>, <span class="refDoi">Â DOI: 10.1038/nsmb.2669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnsmb.2669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=24096405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOiurrP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1147-1155&author=L.+Di%0ACroceauthor=K.+Helin&title=Transcriptional+regulation+by+Polycomb+group+proteins&doi=10.1038%2Fnsmb.2669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation by Polycomb group proteins</span></div><div class="casAuthors">Di Croce, Luciano; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1147-1155</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group (PcG) proteins are epigenetic regulators of transcription that have key roles in stem-cell identity, differentiation and disease.  Mechanistically, they function within multiprotein complexes, called Polycomb repressive complexes (PRCs), which modify histones (and other proteins) and silence target genes.  The dynamics of PRC1 and PRC2 components has been the focus of recent research.  Here we discuss our current knowledge of the PRC complexes, how they are targeted to chromatin and how the high diversity of the PcG proteins allows these complexes to influence cell identity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrArNyrbNU897Vg90H21EOLACvtfcHk0lgR_Gduqnv78w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOiurrP&md5=004c2682edbda681e9f6de84ba87e639</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2669%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BCroce%26aufirst%3DL.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DTranscriptional%2520regulation%2520by%2520Polycomb%2520group%2520proteins%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1147%26epage%3D1155%26doi%3D10.1038%2Fnsmb.2669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganji, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iii, A. D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaFrance, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duquenne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creasy, C. L.</span></span> <span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">108</span>â <span class="NLM_lpage">112</span>, <span class="refDoi">Â DOI: 10.1038/nature11606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1038%2Fnature11606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=23051747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+D.+P.+Iiiauthor=E.+Diazauthor=L.+V.+LaFranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations&doi=10.1038%2Fnature11606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span></div><div class="casAuthors">McCabe, Michael T.; Ott, Heidi M.; Ganji, Gopinath; Korenchuk, Susan; Thompson, Christine; Van Aller, Glenn S.; Liu, Yan; Graves, Alan P.; Della Pietra, Anthony , III; Diaz, Elsie; LaFrance, Louis V.; Mellinger, Mark; Duquenne, Celine; Tian, Xinrong; Kruger, Ryan G.; McHugh, Charles F.; Brandt, Martin; Miller, William H.; Dhanak, Dashyant; Verma, Sharad K.; Tummino, Peter J.; Creasy, Caretha L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7427</span>),
    <span class="NLM_cas:pages">108-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In eukaryotes, post-translational modification of histones is crit. for regulation of chromatin structure and gene expression.  EZH2 is the catalytic subunit of the polycomb repressive complex 2 (PRC2) and is involved in repressing gene expression through methylation of histone H3 on lysine 27 (H3K27).  EZH2 overexpression is implicated in tumorigenesis and correlates with poor prognosis in several tumor types.  Addnl., somatic heterozygous mutations of Y641 and A677 residues within the catalytic SET domain of EZH2 occur in diffuse large B-cell lymphoma (DLBCL) and follicular lymphoma.  The Y641 residue is the most frequently mutated residue, with up to 22% of germinal center B-cell DLBCL and follicular lymphoma harbouring mutations at this site.  These lymphomas have increased H3K27 tri-methylation (H3K27me3) owing to altered substrate preferences of the mutant enzymes.  However, it is unknown whether specific, direct inhibition of EZH2 methyltransferase activity will be effective in treating EZH2 mutant lymphomas.  Here we demonstrate that GSK126, a potent, highly selective, S-adenosyl-methionine-competitive, small-mol. inhibitor of EZH2 methyltransferase activity, decreases global H3K27me3 levels and reactivates silenced PRC2 target genes.  GSK126 effectively inhibits the proliferation of EZH2 mutant DLBCL cell lines and markedly inhibits the growth of EZH2 mutant DLBCL xenografts in mice.  Together, these data demonstrate that pharmacol. inhibition of EZH2 activity may provide a promising treatment for EZH2 mutant lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxw5cLmW3Lc7Vg90H21EOLACvtfcHk0lgR_Gduqnv78w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL&md5=a1dff99556de004c48aae88bc45dffab</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature11606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11606%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DIii%26aufirst%3DA.%2BD.%2BP.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112%26doi%3D10.1038%2Fnature11606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konze, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacNevin, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rougie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">An orally bioavailable chemical probe of the Lysine Methyltransferases EZH2 and EZH1</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1324</span>â <span class="NLM_lpage">1334</span>, <span class="refDoi">Â DOI: 10.1021/cb400133j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400133j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1324-1334&author=K.+D.+Konzeauthor=A.+Maauthor=F.+Liauthor=D.+Barsyte-Lovejoyauthor=T.+Partonauthor=C.+J.+MacNevinauthor=F.+Liuauthor=C.+Gaoauthor=X.-P.+Huangauthor=E.+Kuznetsovaauthor=M.+Rougieauthor=A.+Jiangauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=L.+I.+Jamesauthor=B.+L.+Rothauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=K.+M.+Hahnauthor=G.+G.+Wangauthor=M.+Vedadiauthor=J.+Jin&title=An+orally+bioavailable+chemical+probe+of+the+Lysine+Methyltransferases+EZH2+and+EZH1&doi=10.1021%2Fcb400133j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1</span></div><div class="casAuthors">Konze, Kyle D.; Ma, Anqi; Li, Fengling; Barsyte-Lovejoy, Dalia; Parton, Trevor; MacNevin, Christopher J.; Liu, Feng; Gao, Cen; Huang, Xi-Ping; Kuznetsova, Ekaterina; Rougie, Marie; Jiang, Alice; Pattenden, Samantha G.; Norris, Jacqueline L.; James, Lindsey I.; Roth, Bryan L.; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Hahn, Klaus M.; Wang, Gang Greg; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1324-1334</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 or EZH1 is the catalytic subunit of the polycomb repressive complex 2 that catalyzes methylation of histone H3 lysine 27 (H3K27).  The trimethylation of H3K27 (H3K27me3) is a transcriptionally repressive post-translational modification.  Overexpression of EZH2 and hypertrimethylation of H3K27 have been implicated in a no. of cancers.  Several selective inhibitors of EZH2 have been reported recently.  Herein we disclose UNC1999, the first orally bioavailable inhibitor that has high in vitro potency for wild-type and mutant EZH2 as well as EZH1, a closely related H3K27 methyltransferase that shares 96% sequence identity with EZH2 in their resp. catalytic domains.  UNC1999 was highly selective for EZH2 and EZH1 over a broad range of epigenetic and non-epigenetic targets, competitive with the cofactor SAM and non-competitive with the peptide substrate.  This inhibitor potently reduced H3K27me3 levels in cells and selectively killed diffused large B cell lymphoma cell lines harboring the EZH2Y641N mutant.  Importantly, UNC1999 was orally bioavailable in mice, making this inhibitor a valuable tool for investigating the role of EZH2 and EZH1 in chronic animal studies.  We also designed and synthesized UNC2400, a close analog of UNC1999 with potency >1,000-fold lower than that of UNC1999 as a neg. control for cell-based studies.  Finally, we created a biotin-tagged UNC1999 (UNC2399), which enriched EZH2 in pull-down studies, and a UNC1999-dye conjugate (UNC2239) for co-localization studies with EZH2 in live cells.  Taken together, these compds. represent a set of useful tools for the biomedical community to investigate the role of EZH2 and EZH1 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzhUO0iAe8BbVg90H21EOLACvtfcHk0lhFwbFCOHtQHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D&md5=311d77f5f0cc0b9e2610d8d002a6709c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fcb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400133j%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DParton%26aufirst%3DT.%26aulast%3DMacNevin%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DRougie%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DHahn%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DAn%2520orally%2520bioavailable%2520chemical%2520probe%2520of%2520the%2520Lysine%2520Methyltransferases%2520EZH2%2520and%2520EZH1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1324%26epage%3D1334%26doi%3D10.1021%2Fcb400133j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warholic, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span> <span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">7922</span>â <span class="NLM_lpage">7927</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1303800110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2&doi=10.1073%2Fpnas.1303800110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0lhFwbFCOHtQHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927%26doi%3D10.1073%2Fpnas.1303800110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minoshima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warholic, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadowaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uesugi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uenaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of EZH2 by EPZ-6438 leads to potent antitumor activity in EZH2-mutant non-hodgkin lymphoma</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">842</span>â <span class="NLM_lpage">854</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.mct-13-0773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1158%2F1535-7163.MCT-13-0773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=24563539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=842-854&author=S.+K.+Knutsonauthor=S.+Kawanoauthor=Y.+Minoshimaauthor=N.+M.+Warholicauthor=K.-C.+Huangauthor=Y.+Xiaoauthor=T.+Kadowakiauthor=M.+Uesugiauthor=G.+Kuznetsovauthor=N.+Kumarauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+Porter-Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=T.+Uenakaauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=A.+Yokoiauthor=H.+Keilhack&title=Selective+inhibition+of+EZH2+by+EPZ-6438+leads+to+potent+antitumor+activity+in+EZH2-mutant+non-hodgkin+lymphoma&doi=10.1158%2F1535-7163.mct-13-0773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of EZH2 by EPZ-6438 Leads to Potent Antitumor Activity in EZH2-Mutant Non-Hodgkin Lymphoma</span></div><div class="casAuthors">Knutson, Sarah K.; Kawano, Satoshi; Minoshima, Yukinori; Warholic, Natalie M.; Huang, Kuan-Chun; Xiao, Yonghong; Kadowaki, Tadashi; Uesugi, Mai; Kuznetsov, Galina; Kumar, Namita; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Waters, Nigel J.; Smith, Jesse J.; Porter-Scott, Margaret; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Uenaka, Toshimitsu; Pollock, Roy M.; Kuntz, Kevin W.; Yokoi, Akira; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">842-854</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Mutations within the catalytic domain of the histone methyltransferase EZH2 have been identified in subsets of patients with non-Hodgkin lymphoma (NHL).  These genetic alterations are hypothesized to confer an oncogenic dependency on EZH2 enzymic activity in these cancers.  We have previously reported the discovery of EPZ005678 and EPZ-6438, potent and selective S-adenosyl-methionine-competitive small mol. inhibitors of EZH2.  Although both compds. are similar with respect to their mechanism of action and selectivity, EPZ-6438 possesses superior potency and drug-like properties, including good oral bioavailability in animals.  Here, we characterize the activity of EPZ-6438 in preclin. models of NHL.  EPZ-6438 selectively inhibits intracellular lysine 27 of histone H3 (H3K27) methylation in a concn.- and time-dependent manner in both EZH2 wild-type and mutant lymphoma cells.  Inhibition of H3K27 trimethylation (H3K27Me3) leads to selective cell killing of human lymphoma cell lines bearing EZH2 catalytic domain point mutations.  Treatment of EZH2-mutant NHL xenograft-bearing mice with EPZ-6438 causes dose-dependent tumor growth inhibition, including complete and sustained tumor regressions with correlative diminution of H3K27Me3 levels in tumors and selected normal tissues.  Mice dosed orally with EPZ-6438 for 28 days remained tumor free for up to 63 days after stopping compd. treatment in two EZH2-mutant xenograft models.  These data confirm the dependency of EZH2-mutant NHL on EZH2 activity and portend the utility of EPZ-6438 as a potential treatment for these genetically defined cancers.  Mol Cancer Ther; 13(4); 842-54. Â©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq2o1v5F3GzLVg90H21EOLACvtfcHk0lhHnrZrQpHQfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurw%253D&md5=89354e9453dcd7430978d1a6c5ee5846</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0773%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DKawano%26aufirst%3DS.%26aulast%3DMinoshima%26aufirst%3DY.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DHuang%26aufirst%3DK.-C.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DUesugi%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DN.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DUenaka%26aufirst%3DT.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DYokoi%26aufirst%3DA.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DSelective%2520inhibition%2520of%2520EZH2%2520by%2520EPZ-6438%2520leads%2520to%2520potent%2520antitumor%2520activity%2520in%2520EZH2-mutant%2520non-hodgkin%2520lymphoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D842%26epage%3D854%26doi%3D10.1158%2F1535-7163.mct-13-0773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisserier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wajapeyee, N.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of resistance to EZH2 inhibitors in diffuse large B-cell lymphomas</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">2125</span>â <span class="NLM_lpage">2137</span>, <span class="refDoi">Â DOI: 10.1182/blood-2017-08-804344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1182%2Fblood-2017-08-804344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=29572378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFenurrO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2018&pages=2125-2137&author=M.+Bisserierauthor=N.+Wajapeyee&title=Mechanisms+of+resistance+to+EZH2+inhibitors+in+diffuse+large+B-cell+lymphomas&doi=10.1182%2Fblood-2017-08-804344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of resistance to EZH2 inhibitors in diffuse large B-cell lymphomas</span></div><div class="casAuthors">Bisserier, Malik; Wajapeyee, Narendra</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2125-2137</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Resistance to targeted therapies has become increasingly prevalent.  We noted that resistance to different targeted therapies occurs by largely common mechanisms.  In this study, we used this information for identifying the mechanisms of resistance to enhancer of zeste homolog 2 (EZH2) inhibitors in diffuse large B-cell lymphoma (DLBCL) harboring EZH2 mutations.  We discovered that EZH2 inhibitor-resistant DLBCL cells showed activation of the insulin-like growth factor 1 receptor (IGF-1R), MEK, and phosphoinositide-3-kinase (PI3K) pathways.  Constitutive activation of IGF-1R, MEK, or PI3K pathways was sufficient to confer resistance to EZH2 inhibitors in DLBCL.  The activation of the PI3K/AKT andMAPK pathways decreased TNFSF10 and BAD expression through a FOXO3-dependent mechanism, which was required for the antitumor effects of EZH2i GSK126.  We also identified multiple acquired mutations in EZH2 inhibitor-resistant DLBCL cell lines.  These mutations independently conferred resistance to EZH2 inhibitors.  Mechanistically, cellular thermal shift assays revealed that the acquired EZH2 mutations that confer resistance to EZH2 inhibitors prevent EZH2 inhibitor binding to the EZH2 mutants.  Notably, EZH2 inhibitor GSK126- and EPZ-6438-resistant DLBCL cells remained sensitive to the EZH2 inhibitor UNC1999 and embryonic ectoderm development protein inhibitor EED226, which provides an opportunity to treat DLBCLs that are resistant to these drugs.  Collectively, our results underpin the importance for developing a unified approach for forestalling drug resistance by prospectively considering lessons learned from the use of different targeted therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8bPfbf3K8qrVg90H21EOLACvtfcHk0lhHnrZrQpHQfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFenurrO&md5=e1e626bc16271b40be809c7587279cff</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1182%2Fblood-2017-08-804344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2017-08-804344%26sid%3Dliteratum%253Aachs%26aulast%3DBisserier%26aufirst%3DM.%26aulast%3DWajapeyee%26aufirst%3DN.%26atitle%3DMechanisms%2520of%2520resistance%2520to%2520EZH2%2520inhibitors%2520in%2520diffuse%2520large%2520B-cell%2520lymphomas%26jtitle%3DBlood%26date%3D2018%26volume%3D131%26spage%3D2125%26epage%3D2137%26doi%3D10.1182%2Fblood-2017-08-804344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lingel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D. E.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of EED binders allosterically inhibiting the epigenetic polycomb repressive complex 2 (PRC2) methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">415</span>â <span class="NLM_lpage">427</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01473</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01473" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFaiurrF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=415-427&author=A.+Lingelauthor=M.+Sendzikauthor=Y.+Huangauthor=M.+D.+Shultzauthor=J.+Cantwellauthor=M.+P.+Dillonauthor=X.+Fuauthor=J.+Fullerauthor=T.+Gabrielauthor=J.+Guauthor=X.+Jiangauthor=L.+Liauthor=F.+Liangauthor=M.+McKennaauthor=W.+Qiauthor=W.+Raoauthor=X.+Shengauthor=W.+Shuauthor=J.+Suttonauthor=B.+Taftauthor=L.+Wangauthor=J.+Zengauthor=H.+Zhangauthor=M.+Zhangauthor=K.+Zhaoauthor=M.+Lindvallauthor=D.+E.+Bussiere&title=Structure-guided+design+of+EED+binders+allosterically+inhibiting+the+epigenetic+polycomb+repressive+complex+2+%28PRC2%29+methyltransferase&doi=10.1021%2Facs.jmedchem.6b01473"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of EED Binders Allosterically Inhibiting the Epigenetic Polycomb Repressive Complex 2 (PRC2) Methyltransferase</span></div><div class="casAuthors">Lingel, Andreas; Sendzik, Martin; Huang, Ying; Shultz, Michael D.; Cantwell, John; Dillon, Michael P.; Fu, Xingnian; Fuller, John; Gabriel, Tobias; Gu, Justin; Jiang, Xiangqing; Li, Ling; Liang, Fang; McKenna, Maureen; Qi, Wei; Rao, Weijun; Sheng, Xijun; Shu, Wei; Sutton, James; Taft, Benjamin; Wang, Long; Zeng, Jue; Zhang, Hailong; Zhang, Maya; Zhao, Kehao; Lindvall, Mika; Bussiere, Dirksen E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">415-427</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRC2 is a multisubunit methyltransferase involved in epigenetic regulation of early embryonic development and cell growth.  The catalytic subunit EZH2 methylates primarily lysine 27 of histone H3, leading to chromatin compaction and repression of tumor suppressor genes.  Inhibiting this activity by small mols. targeting EZH2 was shown to result in antitumor efficacy.  Here, the authors describe for the first time the optimization of a chem. series representing a new class of PRC2 inhibitors which acts allosterically via the trimethyllysine pocket of the noncatalytic EED subunit.  Deconstruction of a larger and complex screening hit to a simple fragment-sized mol. followed by structure-guided regrowth and careful property modulation were employed to yield compds. which achieve sub-micromolar inhibition in functional assays and cellular activity.  The resulting mols. can serve as a simplified entry point for lead optimization and can be utilized to study this new mechanism of PRC2 inhibition and the assocd. biol. in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRo2rkeTDLibVg90H21EOLACvtfcHk0lgqnjlrrDqE5Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFaiurrF&md5=3aec108471439f4ac99728a57b4eaf8e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01473%26sid%3Dliteratum%253Aachs%26aulast%3DLingel%26aufirst%3DA.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DDillon%26aufirst%3DM.%2BP.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DFuller%26aufirst%3DJ.%26aulast%3DGabriel%26aufirst%3DT.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DF.%26aulast%3DMcKenna%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DX.%26aulast%3DShu%26aufirst%3DW.%26aulast%3DSutton%26aufirst%3DJ.%26aulast%3DTaft%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DBussiere%26aufirst%3DD.%2BE.%26atitle%3DStructure-guided%2520design%2520of%2520EED%2520binders%2520allosterically%2520inhibiting%2520the%2520epigenetic%2520polycomb%2520repressive%2520complex%25202%2520%2528PRC2%2529%2520methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D415%26epage%3D427%26doi%3D10.1021%2Facs.jmedchem.6b01473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarris, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvaraju, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algire, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span> <span> </span><span class="NLM_article-title">SAR of amino pyrrolidines as potent and novel protein-protein interaction inhibitors of the PRC2 complex through EED binding</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1576</span>â <span class="NLM_lpage">1583</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2017.02.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1016%2Fj.bmcl.2017.02.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=28254486" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Oju7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1576-1583&author=M.+L.+Curtinauthor=M.+A.+Pliushchevauthor=H.-Q.+Liauthor=M.+Torrentauthor=J.+D.+Dietrichauthor=C.+G.+Jakobauthor=H.+Zhuauthor=H.+Zhaoauthor=Y.+Wangauthor=Z.+Jiauthor=R.+F.+Clarkauthor=K.+A.+Sarrisauthor=S.+Selvarajuauthor=B.+Shawauthor=M.+A.+Algireauthor=Y.+Heauthor=P.+L.+Richardsonauthor=R.+F.+Sweisauthor=C.+Sunauthor=G.+G.+Chiangauthor=M.+R.+Michaelides&title=SAR+of+amino+pyrrolidines+as+potent+and+novel+protein-protein+interaction+inhibitors+of+the+PRC2+complex+through+EED+binding&doi=10.1016%2Fj.bmcl.2017.02.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">SAR of amino pyrrolidines as potent and novel protein-protein interaction inhibitors of the PRC2 complex through EED binding</span></div><div class="casAuthors">Curtin, Michael L.; Pliushchev, Marina A.; Li, Huan-Qiu; Torrent, Maricel; Dietrich, Justin D.; Jakob, Clarissa G.; Zhu, Haizhong; Zhao, Hongyu; Wang, Ying; Ji, Zhiqin; Clark, Richard F.; Sarris, Kathy A.; Selvaraju, Sujatha; Shaw, Bailin; Algire, Mikkel A.; He, Yupeng; Richardson, Paul L.; Sweis, Ramzi F.; Sun, Chaohong; Chiang, Gary G.; Michaelides, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1576-1583</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Herein we disclose SAR studies of a series of dimethylamino pyrrolidines which we recently reported as novel inhibitors of the PRC2 complex through disruption of EED/H3K27me3 binding.  Modification of the indole and benzyl moieties of screening hit 1 provided analogs with substantially improved binding and cellular activities.  This work culminated in the identification of compd. 2, our nanomolar proof-of-concept (PoC) inhibitor which provided on-target tumor growth inhibition in a mouse xenograft model.  X-ray crystal structures of several inhibitors bound in the EED active-site are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkdonJy6cBnbVg90H21EOLACvtfcHk0lgqnjlrrDqE5Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Oju7s%253D&md5=3396f3f4c2d3944237cfde004ac4204b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.02.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.02.030%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DM.%2BL.%26aulast%3DPliushchev%26aufirst%3DM.%2BA.%26aulast%3DLi%26aufirst%3DH.-Q.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DDietrich%26aufirst%3DJ.%2BD.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DZ.%26aulast%3DClark%26aufirst%3DR.%2BF.%26aulast%3DSarris%26aufirst%3DK.%2BA.%26aulast%3DSelvaraju%26aufirst%3DS.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DAlgire%26aufirst%3DM.%2BA.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26atitle%3DSAR%2520of%2520amino%2520pyrrolidines%2520as%2520potent%2520and%2520novel%2520protein-protein%2520interaction%2520inhibitors%2520of%2520the%2520PRC2%2520complex%2520through%2520EED%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D1576%26epage%3D1583%26doi%3D10.1016%2Fj.bmcl.2017.02.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, B.</span></span> <span> </span><span class="NLM_article-title">An allosteric PRC2 inhibitor targeting EED suppresses tumor progression by modulating the immune response</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">5587</span>â <span class="NLM_lpage">5596</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.can-19-0428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1158%2F0008-5472.CAN-19-0428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=31395608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1KitL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=5587-5596&author=H.+Dongauthor=S.+Liuauthor=X.+Zhangauthor=S.+Chenauthor=L.+Kangauthor=Y.+Chenauthor=S.+Maauthor=X.+Fuauthor=Y.+Liuauthor=H.+Zhangauthor=B.+Zou&title=An+allosteric+PRC2+inhibitor+targeting+EED+suppresses+tumor+progression+by+modulating+the+immune+response&doi=10.1158%2F0008-5472.can-19-0428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric PRC2 inhibitor targeting EED suppresses tumor progression by modulating the immune response</span></div><div class="casAuthors">Dong, Hongping; Liu, Shaojun; Zhang, Xuejie; Chen, Sheng; Kang, Lijing; Chen, Yanni; Ma, Shichao; Fu, Xianlei; Liu, Yanchao; Zhang, Hailong; Zou, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5587-5596</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant activity of polycomb repressive complex 2 (PRC2) is involved in a wide range of human cancer progression.  The WD40 repeat-contg. protein EED is a core component of PRC2 and enhances PRC2 activity through interaction with H3K27me3.  In this study, we report the discovery of a class of pyrimidone compds., represented by BR-001 as potent allosteric inhibitors of PRC2.  X-ray co-crystallog. showed that BR-001 directly binds EED in the H3K27me3-binding pocket.  BR-001 displayed antitumor potency in vitro and in vivo.  In Karpas422 and Pfeiffer xenograft mouse models, twice daily oral dosing with BR-001 resulted in robust anti-tumor activity.  BR-001 was also efficacious in syngeneic CT26 colon tumor-bearing mice; oral dosing of 30 mg/kg of BR-001 led to 59.3% tumor growth suppression and increased frequency of effector CD8+ T-cell infiltrates in tumors.  Pharmacodynamic anal. revealed that CXCL10 was highly upregulated, suggesting that CXCL10 triggers the trafficking of CD8+ T cells toward tumor sites.  Our results demonstrate for the first time that inhibition of EED modulates the tumor immune microenvironment to induce regression of colon tumors and therefore has the potential to be used in combination with immune-oncol. therapy.  Significance: BR-001, a potent inhibitor of the EED subunit of the PRC2 complex, suppresses tumor progression by modulating the tumor microenvironment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriYcNHbku1ZbVg90H21EOLACvtfcHk0lgqnjlrrDqE5Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1KitL8%253D&md5=51331eac62193271e40ea4b9246a348c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-19-0428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-19-0428%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DKang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZou%26aufirst%3DB.%26atitle%3DAn%2520allosteric%2520PRC2%2520inhibitor%2520targeting%2520EED%2520suppresses%2520tumor%2520progression%2520by%2520modulating%2520the%2520immune%2520response%26jtitle%3DCancer%2520Res.%26date%3D2019%26volume%3D79%26spage%3D5587%26epage%3D5596%26doi%3D10.1158%2F0008-5472.can-19-0428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, L. A.</span></span> <span> </span><span class="NLM_article-title">Reactive metabolites in the biotransformation of molecules containing a furan ring</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">6</span>â <span class="NLM_lpage">25</span>, <span class="refDoi">Â DOI: 10.1021/tx3003824</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx3003824" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=6-25&author=L.+A.+Peterson&title=Reactive+metabolites+in+the+biotransformation+of+molecules+containing+a+furan+ring&doi=10.1021%2Ftx3003824"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Reactive Metabolites in the Biotransformation of Molecules Containing a Furan Ring</span></div><div class="casAuthors">Peterson, Lisa A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6-25</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many xenobiotics contg. a furan ring are toxic and/or carcinogenic.  The harmful effects of these compds. require furan ring oxidn.  This reaction generates an electrophilic intermediate.  Depending on the furan ring substituents, the intermediate is either an epoxide or a cis-enedione with more ring substitution favoring epoxide formation.  Either intermediate reacts with cellular nucleophiles such as protein or DNA to trigger toxicities.  The reactivity of the metabolite dets. which cellular nucleophiles are targeted.  The toxicity of a particular furan is also influenced by the presence of competing metabolic pathways or efficient detoxification routes.  GSH plays an important role in modulating the harmful effects of this class of compd. by reacting with the reactive metabolite.  However, this may not represent a detoxification step in all cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp4Dr5stLUSrVg90H21EOLACvtfcHk0lhOrIxp353S5w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN&md5=fe3a280d0b7a99a436ec619abc1e1fa0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Ftx3003824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx3003824%26sid%3Dliteratum%253Aachs%26aulast%3DPeterson%26aufirst%3DL.%2BA.%26atitle%3DReactive%2520metabolites%2520in%2520the%2520biotransformation%2520of%2520molecules%2520containing%2520a%2520furan%2520ring%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2013%26volume%3D26%26spage%3D6%26epage%3D25%26doi%3D10.1021%2Ftx3003824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span>; <span class="NLM_string-name">Fu, X.</span>; <span class="NLM_string-name">Gu, X. J. J.</span>; <span class="NLM_string-name">Huang, Y.</span>; <span class="NLM_string-name">Li, L.</span>; <span class="NLM_string-name">Mi, Y.</span>; <span class="NLM_string-name">Qi, W.</span>; <span class="NLM_string-name">Sendzik, M.</span>; <span class="NLM_string-name">Sun, Y.</span>; <span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Yu, Z.</span>; <span class="NLM_string-name">Zhang, H.</span>; <span class="NLM_string-name">Zhang, J. Y.</span>; <span class="NLM_string-name">Zhang, M.</span>; <span class="NLM_string-name">Zhang, Q.</span>; <span class="NLM_string-name">Zhao, K.</span></span> <span> </span><span class="NLM_article-title">Triazolopyridine Compounds and Uses Thereof</span>. International Patent Application PCT/IB2017/053335, WO<span class="NLM_patent">2017221092A1</span>, <span class="NLM_year">2017</span>, <span class="NLM_publisher-name">Novartis AG</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=H.+M.+Chan&author=X.+Fu&author=X.+J.+J.+Gu&author=Y.+Huang&author=L.+Li&author=Y.+Mi&author=W.+Qi&author=M.+Sendzik&author=Y.+Sun&author=L.+Wang&author=Z.+Yu&author=H.+Zhang&author=J.+Y.+Zhang&author=M.+Zhang&author=Q.+Zhang&author=K.+Zhao&title=Triazolopyridine+Compounds+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DH.%2BM.%26atitle%3DTriazolopyridine%2520Compounds%2520and%2520Uses%2520Thereof%26date%3D2017%26pub%3DNovartis%2520AG" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eusterwiemann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolbier, W. R.</span></span> <span> </span><span class="NLM_article-title">Methyl 2,2-difluoro-2-(fluorosulfonyl)acetate, a difluorocarbene reagent with reactivity comparable to that of trimethylsilyl 2,2-difluoro-2-(fluorosulfonyl)acetate (TFDA)</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">5461</span>â <span class="NLM_lpage">5464</span>, <span class="refDoi">Â DOI: 10.1021/jo300876z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo300876z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1yiur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2012&pages=5461-5464&author=S.+Eusterwiemannauthor=H.+Martinezauthor=W.+R.+Dolbier&title=Methyl+2%2C2-difluoro-2-%28fluorosulfonyl%29acetate%2C+a+difluorocarbene+reagent+with+reactivity+comparable+to+that+of+trimethylsilyl+2%2C2-difluoro-2-%28fluorosulfonyl%29acetate+%28TFDA%29&doi=10.1021%2Fjo300876z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Methyl 2,2-Difluoro-2-(fluorosulfonyl)acetate, a Difluorocarbene Reagent with Reactivity Comparable to That of Trimethylsilyl 2,2-Difluoro-2-(fluorosulfonyl)acetate (TFDA)</span></div><div class="casAuthors">Eusterwiemann, Steffen; Martinez, Henry; Dolbier, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5461-5464</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Under specific high concn., high temp. conditions, Me 2,2-difluoro-2-(fluorosulfonyl)acetate (MDFA) has been found to act as a very efficient source of difluorocarbene, exhibiting carbene reactivity characteristics comparable to those exhibited by trimethylsilyl 2,2-difluoro-2-(fluorosulfonyl)acetate (TFDA).  For example, in reaction with highly unreactive Bu acrylate and using only 2 equiv of MDFA, a yield of 76% of difluorocyclopropane product was obtained after 2 days.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqig5Ak3X8SdrVg90H21EOLACvtfcHk0lhCKIfFY3YqBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1yiur8%253D&md5=ba86878d06f3037f7bde41bd1999c613</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjo300876z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo300876z%26sid%3Dliteratum%253Aachs%26aulast%3DEusterwiemann%26aufirst%3DS.%26aulast%3DMartinez%26aufirst%3DH.%26aulast%3DDolbier%26aufirst%3DW.%2BR.%26atitle%3DMethyl%25202%252C2-difluoro-2-%2528fluorosulfonyl%2529acetate%252C%2520a%2520difluorocarbene%2520reagent%2520with%2520reactivity%2520comparable%2520to%2520that%2520of%2520trimethylsilyl%25202%252C2-difluoro-2-%2528fluorosulfonyl%2529acetate%2520%2528TFDA%2529%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2012%26volume%3D77%26spage%3D5461%26epage%3D5464%26doi%3D10.1021%2Fjo300876z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span> <span> </span><span class="NLM_article-title">An efficient copper catalyst for the formation of sulfones from sulfinic acid salts and aryl iodides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">4423</span>â <span class="NLM_lpage">4425</span>, <span class="refDoi">Â DOI: 10.1021/ol0269190</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0269190" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFGjt7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=4423-4425&author=J.+M.+Baskinauthor=Z.+Wang&title=An+efficient+copper+catalyst+for+the+formation+of+sulfones+from+sulfinic+acid+salts+and+aryl+iodides&doi=10.1021%2Fol0269190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An Efficient Copper Catalyst for the Formation of Sulfones from Sulfinic Acid Salts and Aryl Iodides</span></div><div class="casAuthors">Baskin, Jeremy M.; Wang, Zhaoyin</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4423-4425</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel copper-catalyzed method for the coupling of sulfinic acid salts and aryl iodides is described.  A variety of Me and diaryl sulfones have been formed in excellent yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoibEB4_JcsSLVg90H21EOLACvtfcHk0lhCKIfFY3YqBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFGjt7Y%253D&md5=ed4d7aa2b442c75197e8ff887f27bcff</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fol0269190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0269190%26sid%3Dliteratum%253Aachs%26aulast%3DBaskin%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DAn%2520efficient%2520copper%2520catalyst%2520for%2520the%2520formation%2520of%2520sulfones%2520from%2520sulfinic%2520acid%2520salts%2520and%2520aryl%2520iodides%26jtitle%3DOrg.%2520Lett.%26date%3D2002%26volume%3D4%26spage%3D4423%26epage%3D4425%26doi%3D10.1021%2Fol0269190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwight, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toms, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parillon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohemmad, Q. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of Brigatinib (AP26113), a phosphine oxide-containing, potent, orally active inhibitor of anaplastic lymphoma kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4948</span>â <span class="NLM_lpage">4964</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b00306</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00306" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlCgur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4948-4964&author=W.-S.+Huangauthor=S.+Liuauthor=D.+Zouauthor=M.+Thomasauthor=Y.+Wangauthor=T.+Zhouauthor=J.+Romeroauthor=A.+Kohlmannauthor=F.+Liauthor=J.+Qiauthor=L.+Caiauthor=T.+A.+Dwightauthor=Y.+Xuauthor=R.+Xuauthor=R.+Doddauthor=A.+Tomsauthor=L.+Parillonauthor=X.+Luauthor=R.+Anjumauthor=S.+Zhangauthor=F.+Wangauthor=J.+Keatsauthor=S.+D.+Wardwellauthor=Y.+Ningauthor=Q.+Xuauthor=L.+E.+Moranauthor=Q.+K.+Mohemmadauthor=H.+G.+Jangauthor=T.+Clacksonauthor=N.+I.+Narasimhanauthor=V.+M.+Riveraauthor=X.+Zhuauthor=D.+Dalgarnoauthor=W.+C.+Shakespeare&title=Discovery+of+Brigatinib+%28AP26113%29%2C+a+phosphine+oxide-containing%2C+potent%2C+orally+active+inhibitor+of+anaplastic+lymphoma+kinase&doi=10.1021%2Facs.jmedchem.6b00306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Brigatinib (AP26113), a Phosphine Oxide-Containing, Potent, Orally Active Inhibitor of Anaplastic Lymphoma Kinase</span></div><div class="casAuthors">Huang, Wei-Sheng; Liu, Shuangying; Zou, Dong; Thomas, Mathew; Wang, Yihan; Zhou, Tianjun; Romero, Jan; Kohlmann, Anna; Li, Feng; Qi, Jiwei; Cai, Lisi; Dwight, Timothy A.; Xu, Yongjin; Xu, Rongsong; Dodd, Rory; Toms, Angela; Parillon, Lois; Lu, Xiaohui; Anjum, Rana; Zhang, Sen; Wang, Frank; Keats, Jeffrey; Wardwell, Scott D.; Ning, Yaoyu; Xu, Qihong; Moran, Lauren E.; Mohemmad, Qurish K.; Jang, Hyun Gyung; Clackson, Tim; Narasimhan, Narayana I.; Rivera, Victor M.; Zhu, Xiaotian; Dalgarno, David; Shakespeare, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4948-4964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the treatment of echinoderm microtubule-assocd. protein-like 4 (EML4)-anaplastic lymphoma kinase pos. (ALK+) non-small-cell lung cancer (NSCLC), secondary mutations within the ALK kinase domain have emerged as a major resistance mechanism to both first- and second-generation ALK inhibitors.  This report describes the design and synthesis of a series of 2,4-diarylaminopyrimidine-based potent and selective ALK inhibitors culminating in identification of the investigational clin. candidate brigatinib.  A unique structural feature of brigatinib is a phosphine oxide, an overlooked but novel hydrogen-bond acceptor that drives potency and selectivity in addn. to favorable ADME properties.  Brigatinib displayed low nanomolar IC50s against native ALK and all tested clin. relevant ALK mutants in both enzyme-based biochem. and cell-based viability assays and demonstrated efficacy in multiple ALK+ xenografts in mice, including Karpas-299 (anaplastic large-cell lymphomas [ALCL]) and H3122 (NSCLC).  Brigatinib represents the most clin. advanced phosphine oxide-contg. drug candidate to date and is currently being evaluated in a global phase 2 registration trial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_iqprSA4QbVg90H21EOLACvtfcHk0lhDDhWoqiqPvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlCgur8%253D&md5=4cb93ff587579c97fda8aff0c7d43317</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00306%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DRomero%26aufirst%3DJ.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DL.%26aulast%3DDwight%26aufirst%3DT.%2BA.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DDodd%26aufirst%3DR.%26aulast%3DToms%26aufirst%3DA.%26aulast%3DParillon%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DAnjum%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DKeats%26aufirst%3DJ.%26aulast%3DWardwell%26aufirst%3DS.%2BD.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMoran%26aufirst%3DL.%2BE.%26aulast%3DMohemmad%26aufirst%3DQ.%2BK.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DClackson%26aufirst%3DT.%26aulast%3DNarasimhan%26aufirst%3DN.%2BI.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDalgarno%26aufirst%3DD.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26atitle%3DDiscovery%2520of%2520Brigatinib%2520%2528AP26113%2529%252C%2520a%2520phosphine%2520oxide-containing%252C%2520potent%252C%2520orally%2520active%2520inhibitor%2520of%2520anaplastic%2520lymphoma%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4948%26epage%3D4964%26doi%3D10.1021%2Facs.jmedchem.6b00306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>â <span class="NLM_lpage">326</span>, <span class="refDoi">Â DOI: 10.1016/s0076-6879(97)76066-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2Fs0076-6879%2897%2976066-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (Ï2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Ã unit cell to rotavirus of 700 Ã diam. crystd. in 700 Ã 1000 Ã 1400 Ã cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhDDhWoqiqPvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2Fs0076-6879%2897%2976066-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1022</span>â <span class="NLM_lpage">1025</span>, <span class="refDoi">Â DOI: 10.1107/s0021889897006766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement&doi=10.1107%2Fs0021889897006766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgwExqrRlN8hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025%26doi%3D10.1107%2Fs0021889897006766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span> <span> </span><span class="NLM_article-title">Structural basis of EZH2 recognition by EED</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1306</span>â <span class="NLM_lpage">1315</span>, <span class="refDoi">Â DOI: 10.1016/j.str.2007.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1016%2Fj.str.2007.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=17937919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKhsLzE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2007&pages=1306-1315&author=Z.+Hanauthor=X.+Xingauthor=M.+Huauthor=Y.+Zhangauthor=P.+Liuauthor=J.+Chai&title=Structural+basis+of+EZH2+recognition+by+EED&doi=10.1016%2Fj.str.2007.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of EZH2 Recognition by EED</span></div><div class="casAuthors">Han, Zhifu; Xing, Xinmiao; Hu, Min; Zhang, Yin; Liu, Peiyuan; Chai, Jijie</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1306-1315</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The WD-repeat domain is a highly conserved recognition module in eukaryotes involved in diverse cellular processes.  It is still not well understood how the bottom of a WD-repeat domain recognizes its binding partners.  The WD-repeat-contg. protein EED is one component of the PRC2 complex that possesses histone methyltransferase activity required for gene repression.  Here we report the crystal structure of EED in complex with a 30 residue peptide from EZH2.  The structure reveals that the peptide binds to the bottom of the WD-repeat domain of EED.  The structural determinants of EZH2-EED interaction are present not only in EZH2 and EZH1 but also in its Drosophila homolog E(Z), suggesting that the recognition of ESC by E(Z) in Drosophila employs similar structural motifs.  Structure-based mutagenesis identified crit. residues from both EED and EZH2 for their interaction.  The structure presented here may provide a template for understanding of how WD-repeat proteins recognize their interacting proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8t4Clbu4NV7Vg90H21EOLACvtfcHk0lgwExqrRlN8hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKhsLzE&md5=1b963045c45d252eff1a36336b85a8c7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2007.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2007.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DXing%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DChai%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520of%2520EZH2%2520recognition%2520by%2520EED%26jtitle%3DStructure%26date%3D2007%26volume%3D15%26spage%3D1306%26epage%3D1315%26doi%3D10.1016%2Fj.str.2007.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>â <span class="NLM_lpage">501</span>, <span class="refDoi">Â DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lirFpBS_tWayQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, 2.10.3; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+S.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O. S.</span>; <span class="NLM_string-name">Womack, T. O.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Bricogne, G.</span></span> <i>GRADE</i>, 1.2.13; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=O.+S.+Smart&author=T.+O.+Womack&author=A.+Sharff&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=C.+Vonrhein&author=G.+Bricogne&title=GRADE"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSmart%26aufirst%3DO.%2BS.%26jtitle%3DGRADE%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GSA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GSA','PDB','5GSA'); return false;">PDB: 5GSA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7F','PDB','6W7F'); return false;">PDB: 6W7F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7G','PDB','6W7G'); return false;">PDB: 6W7G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QXV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QXV','PDB','2QXV'); return false;">PDB: 2QXV</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i85"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00479">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_53024"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00479?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00479</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of representative compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Crystallography data collection and refinement statistics; difference in electron density contoured to 3Ï; and molecular string files for all the final target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_001.pdf">jm0c00479_si_001.pdf (1.34 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00479/suppl_file/jm0c00479_si_002.csv">jm0c00479_si_002.csv (3.21 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates of cocrystal structures for compound <b>28</b> (EEDi-5285) in a complex with EED and for compound <b>32</b> (EEDi-1056) in a complex with EED have been deposited into Protein Data Bank with accession codes of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7F">6W7F</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W7G">6W7G</a>, respectively. Authors will release the atomic coordinates upon the publication of this article.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00479&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-13%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00479" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00479" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a3fb0cc93c6a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
