
<html lang="en"     class="pb-page"  data-request-id="46711f08-ccd5-4874-8881-9496f40efde1"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-5;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.5b01293;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Definitive Metabolite Identification Coupled with Automated Ligand Identification System (ALIS) Technology: A Novel Approach to Uncover Structure–Activity Relationships and Guide Drug Design in a Factor IXa Inhibitor Program" /></meta><meta name="dc.Creator" content="Ting  Zhang" /></meta><meta name="dc.Creator" content="Yong  Liu" /></meta><meta name="dc.Creator" content="Xianshu  Yang" /></meta><meta name="dc.Creator" content="Gary E.  Martin" /></meta><meta name="dc.Creator" content="Huifang  Yao" /></meta><meta name="dc.Creator" content="Jackie  Shang" /></meta><meta name="dc.Creator" content="Randal M.  Bugianesi" /></meta><meta name="dc.Creator" content="Kenneth P.  Ellsworth" /></meta><meta name="dc.Creator" content="Lisa M.  Sonatore" /></meta><meta name="dc.Creator" content="Peter  Nizner" /></meta><meta name="dc.Creator" content="Edward C.  Sherer" /></meta><meta name="dc.Creator" content="Susan E.  Hill" /></meta><meta name="dc.Creator" content="Ian W.  Knemeyer" /></meta><meta name="dc.Creator" content="Wayne M.  Geissler" /></meta><meta name="dc.Creator" content="Peter J.  Dandliker" /></meta><meta name="dc.Creator" content="Roy  Helmy" /></meta><meta name="dc.Creator" content="Harold B.  Wood" /></meta><meta name="dc.Description" content="A potent and selective Factor IXa (FIXa) inhibitor was subjected to a series of liver microsomal incubations, which generated a number of metabolites. Using automated ligand identification system-a..." /></meta><meta name="Description" content="A potent and selective Factor IXa (FIXa) inhibitor was subjected to a series of liver microsomal incubations, which generated a number of metabolites. Using automated ligand identification system-a..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 22, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01293" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01293" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01293" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01293" /></link>
        
    
    

<title>Definitive Metabolite Identification Coupled with Automated Ligand Identification System (ALIS) Technology: A Novel Approach to Uncover Structure–Activity Relationships and Guide Drug Design in a Factor IXa Inhibitor Program | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01293" /></meta><meta property="og:title" content="Definitive Metabolite Identification Coupled with Automated Ligand Identification System (ALIS) Technology: A Novel Approach to Uncover Structure–Activity Relationships and Guide Drug Design in a Factor IXa Inhibitor Program" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0012.jpeg" /></meta><meta property="og:description" content="A potent and selective Factor IXa (FIXa) inhibitor was subjected to a series of liver microsomal incubations, which generated a number of metabolites. Using automated ligand identification system-affinity selection (ALIS-AS) methodology, metabolites in the incubation mixture were prioritized by their binding affinities to the FIXa protein. Microgram quantities of the metabolites of interest were then isolated through microisolation analytical capabilities, and structurally characterized using MicroCryoProbe heteronuclear 2D NMR techniques. The isolated metabolites recovered from the NMR experiments were then submitted directly to an in vitro FIXa enzymatic assay. The order of the metabolites’ binding affinity to the Factor IXa protein from the ALIS assay was completely consistent with the enzymatic assay results. This work showcases an innovative and efficient approach to uncover structure–activity relationships (SARs) and guide drug design via microisolation–structural characterization and ALIS capabilities." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01293"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01293">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01293&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01293&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01293&amp;href=/doi/10.1021/acs.jmedchem.5b01293" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 1818-1829</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01080" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01311" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Definitive Metabolite Identification Coupled with Automated Ligand Identification System (ALIS) Technology: A Novel Approach to Uncover Structure–Activity Relationships and Guide Drug Design in a Factor IXa Inhibitor Program</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ting++Zhang">Ting Zhang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Liu">Yong Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xianshu++Yang">Xianshu Yang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gary+E.++Martin">Gary E. Martin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Huifang++Yao">Huifang Yao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jackie++Shang">Jackie Shang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Randal+M.++Bugianesi">Randal M. Bugianesi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kenneth+P.++Ellsworth">Kenneth P. Ellsworth</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lisa+M.++Sonatore">Lisa M. Sonatore</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Nizner">Peter Nizner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Edward+C.++Sherer">Edward C. Sherer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>¶</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Susan+E.++Hill">Susan E. Hill</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ian+W.++Knemeyer">Ian W. Knemeyer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wayne+M.++Geissler">Wayne M. Geissler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Dandliker">Peter J. Dandliker</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Roy++Helmy">Roy Helmy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Harold+B.++Wood">Harold B. Wood</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1a"><span class="aff-text"><sup>†</sup>Department of Medicinal Chemistry, <sup>‡</sup>Department of Process and Analytical Chemistry, and <sup>¶</sup>Department of Chemistry Modeling and Informatics, Merck Research Laboratories, Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span></div><div class="aff-info" id="aff3a"><span class="aff-text"><sup>#</sup>Department of Pharmacology and <sup>⊥</sup>Department of Pharmacokinetics, Pharmacodynamics and Drug Metabolism, Merck Research Laboratories, Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span></div><div class="aff-info" id="aff4a"><span class="aff-text"><sup>∥</sup>Department of Pharmacokinetics, Pharmacodynamics and Drug Metabolism, and <sup>§</sup>Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#75011c1b122a0f1d141b124635181007161e5b161a18"><span class="__cf_email__" data-cfemail="790d10171e26031118171e4a39141c0b1a12571a1614">[email protected]</span></a>. Phone: +1-732-594-3031.</div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4c3523222b132025397e0c21293e2f27622f2321"><span class="__cf_email__" data-cfemail="aed7c1c0c9f1c2c7db9ceec3cbdccdc580cdc1c3">[email protected]</span></a>. Phone: +1-732-594-9040.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01293&amp;href=/doi/10.1021%2Facs.jmedchem.5b01293" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 1818–1829</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 12, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>20 August 2015</li><li><span class="item_label"><b>Published</b> online</span>22 February 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 March 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01293" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01293</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1818%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DTing%2BZhang%252C%2BYong%2BLiu%252C%2BXianshu%2BYang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D5%26contentID%3Dacs.jmedchem.5b01293%26title%3DDefinitive%2BMetabolite%2BIdentification%2BCoupled%2Bwith%2BAutomated%2BLigand%2BIdentification%2BSystem%2B%2528ALIS%2529%2BTechnology%253A%2BA%2BNovel%2BApproach%2Bto%2BUncover%2BStructure%25E2%2580%2593Activity%2BRelationships%2Band%2BGuide%2BDrug%2BDesign%2Bin%2Ba%2BFactor%2BIXa%2BInhibitor%2BProgram%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1829%26publicationDate%3DMarch%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01293"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2286</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01293" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Definitive Metabolite Identification Coupled with Automated Ligand Identification System (ALIS) Technology: A Novel Approach to Uncover Structure–Activity Relationships and Guide Drug Design in a Factor IXa Inhibitor Program&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ting&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xianshu&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Gary&quot;,&quot;last_name&quot;:&quot;E. Martin&quot;},{&quot;first_name&quot;:&quot;Huifang&quot;,&quot;last_name&quot;:&quot;Yao&quot;},{&quot;first_name&quot;:&quot;Jackie&quot;,&quot;last_name&quot;:&quot;Shang&quot;},{&quot;first_name&quot;:&quot;Randal&quot;,&quot;last_name&quot;:&quot;M. Bugianesi&quot;},{&quot;first_name&quot;:&quot;Kenneth&quot;,&quot;last_name&quot;:&quot;P. Ellsworth&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;M. Sonatore&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Nizner&quot;},{&quot;first_name&quot;:&quot;Edward&quot;,&quot;last_name&quot;:&quot;C. Sherer&quot;},{&quot;first_name&quot;:&quot;Susan&quot;,&quot;last_name&quot;:&quot;E. Hill&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;W. Knemeyer&quot;},{&quot;first_name&quot;:&quot;Wayne&quot;,&quot;last_name&quot;:&quot;M. Geissler&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Dandliker&quot;},{&quot;first_name&quot;:&quot;Roy&quot;,&quot;last_name&quot;:&quot;Helmy&quot;},{&quot;first_name&quot;:&quot;Harold&quot;,&quot;last_name&quot;:&quot;B. Wood&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;1818-1829&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01293&quot;},&quot;abstract&quot;:&quot;A potent and selective Factor IXa (FIXa) inhibitor was subjected to a series of liver microsomal incubations, which generated a number of metabolites. Using automated ligand identification system-affinity selection (ALIS-AS) methodology, metabolites in the incubation mixture were prioritized by their binding affinities to the FIXa protein. Microgram quantities of the metabolites of interest were then isolated through microisolation analytical capabilities, and structurally characterized using MicroCryoProbe heteronuclear 2D NMR techniques. The isolated metabolites recovered from the NMR experiments were then submitted directly to an in vitro FIXa enzymatic assay. The order of the metabolites’ binding affinity to the Factor IXa protein from the ALIS assay was completely consistent with the enzymatic assay results. This work showcases an innovative and efficient approach to uncover structure–activity relationships (SARs) and guide drug design via microisolation–structural characterization and ALIS capabilities.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01293&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01293" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01293&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01293" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01293&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01293" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01293&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01293&amp;href=/doi/10.1021/acs.jmedchem.5b01293" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01293" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01293" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01293%26sid%3Dliteratum%253Aachs%26pmid%3D26871940%26genre%3Darticle%26aulast%3DZhang%26date%3D2016%26atitle%3DDefinitive%2BMetabolite%2BIdentification%2BCoupled%2Bwith%2BAutomated%2BLigand%2BIdentification%2BSystem%2B%2528ALIS%2529%2BTechnology%253A%2BA%2BNovel%2BApproach%2Bto%2BUncover%2BStructure%25E2%2580%2593Activity%2BRelationships%2Band%2BGuide%2BDrug%2BDesign%2Bin%2Ba%2BFactor%2BIXa%2BInhibitor%2BProgram%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D5%26spage%3D1818%26epage%3D1829%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291118" title="Isolation">Isolation</a>,</li><li><a href="/action/doSearch?ConceptID=292143" title="Metabolism">Metabolism</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/jmcmar.2016.59.issue-5/20160310-01/jmcmar.2016.59.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_null" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0012.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A potent and selective Factor IXa (FIXa) inhibitor was subjected to a series of liver microsomal incubations, which generated a number of metabolites. Using automated ligand identification system-affinity selection (ALIS-AS) methodology, metabolites in the incubation mixture were prioritized by their binding affinities to the FIXa protein. Microgram quantities of the metabolites of interest were then isolated through microisolation analytical capabilities, and structurally characterized using MicroCryoProbe heteronuclear 2D NMR techniques. The isolated metabolites recovered from the NMR experiments were then submitted directly to an <i>in vitro</i> FIXa enzymatic assay. The order of the metabolites’ binding affinity to the Factor IXa protein from the ALIS assay was completely consistent with the enzymatic assay results. This work showcases an innovative and efficient approach to uncover structure–activity relationships (SARs) and guide drug design via microisolation–structural characterization and ALIS capabilities.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01312" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01312" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Characterization of drug metabolites provides discovery scientists with information on the sites of metabolism of a structural lead series.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1-5)</a> Such knowledge is especially relevant when the pharmacokinetic (PK) profile of the molecule is suboptimal. Medicinal chemists could benefit from such information in future analog designs intended to improve PK properties by blocking those sites of metabolism.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In addition, metabolites sometimes are more potent and can have even better drug profiles than the parent molecule itself. Hence, a comprehensive understanding of the location(s) in a molecule where metabolism occurs could lead to new drug design and expanded intellectual property space.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a></div><div class="NLM_p">In drug discovery research, metabolites are typically generated <i>in vitro</i> from hepatocytes and/or microsomes, or they may be isolated <i>in vivo</i> from bodily fluids such as urine and/or blood samples from human and preclinical species. Traditionally, metabolite characterization has relied heavily on liquid chromatography–mass spectrometry (LC-MS) techniques for sensitivity reasons. With the advancement of high resolution (HR)-MS/MS capabilities, investigators can define the metabolically modified region(s) of the molecule and propose <i>likely</i> metabolite structures based on the molecular weight change and MS/MS fragmentation pattern.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16">(9-16)</a></div><div class="NLM_p">Definitive structural assignment of a particular metabolite requires a sample of the metabolite that is amenable to NMR analysis. Generally, a suitable metabolite sample can be prepared from large scale incubation, followed by isolation and NMR characterization. After identifying the structure of such metabolite, if the metabolite is of interest, medicinal chemists may then need to design a synthetic sequence to produce larger quantities of the material for additional <i>in vitro</i> and <i>in vivo</i> studies.<a onclick="showRef(event, 'ref6 ref17'); return false;" href="javascript:void(0);" class="ref ref6 ref17">(6, 17)</a> Conversely, if a given metabolite is not of interest, then the cycle begins again for another metabolite.<a onclick="showRef(event, 'ref6 ref17'); return false;" href="javascript:void(0);" class="ref ref6 ref17">(6, 17)</a> The overall process is not targeted and consequently requires a significant and continuing resource commitment. Because of these limitations and the low abundance of the minor metabolites in an incubation mixture, typically only a selected number of metabolites, usually the major ones, are isolated, fully characterized, and synthesized for further studies. However, there remains the distinct possibility that one or more of the minor metabolites could represent missed and new structure–activity relationships (SARs) for better drug design.</div><div class="NLM_p">The relatively recent development of 1.7 mm MicroCryoProbe NMR capabilities has significantly reduced the quantity of a metabolite that must be isolated for NMR characterization.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18-20)</a> This enhanced analytical capability will open new avenues for the characterization and investigation of minor metabolites that may embody potentially new SAR.</div><div class="NLM_p">Recently, the automated ligand identification system (ALIS), an affinity selection–mass spectrometry based (AS-MS) platform has been used to identify hits from mixture-based combinatorial libraries.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25 ref26">(21-26)</a> The ALIS methodology coupled with the microisolation and structure elucidation capabilities can be applied to investigate metabolism incubation mixtures, rank order the binding affinity of metabolites, and identify metabolites that warrant structural characterization and synthetic elaboration. Ultimately, these novel analytical capabilities provide tools that can help guide medicinal chemists to synthesize only the desired compound(s) and improve the discovery time cycle for novel therapeutic leads.</div><div class="NLM_p">Advancements in the area of <i>in vitro</i> pharmacologic testing now allow biologic potency measurements to be undertaken with low microgram (μg) quantities of samples. Typical metabolite microisolation techniques afford quantities of metabolites that, when prepared for NMR characterization using 1.7 mm MicroCryoProbe techniques, allow direct utilization of the NMR sample in the biological assay following structural characterization. The metabolite in the NMR sample can be quantitated by either NMR or HPLC-UV/MS techniques.</div><div class="NLM_p last">We report herein an example showcasing the power of combining the aforementioned technologies in an antithrombotic drug discovery program,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> which enabled the quick identification of the metabolites of interest and uncovered new SAR for improving potency. After microsomal incubation of parent compound, ALIS-AS technology was used to guide and prioritize definitive metabolite isolation and identification, which ultimately led to the discovery of an active minor metabolite with much higher binding affinity than the parent compound. Following isolation on the microgram scale and NMR structural characterization, the intrinsic activity of the μg quantity of the metabolite was confirmed in an enzymatic activity assay.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42555" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42555" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compound <b>7</b> was synthesized via the synthetic route shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Starting from a heterocyclic halide <b>1</b>, Pd-catalyzed α-arylation with compound <b>2</b> led to nitrile <b>3</b>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Compound <b>3</b> was then reduced using literature methods to give primary amine <b>4</b>.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In parallel, carboxylic acid <b>5</b> was converted to the acid chloride <b>6</b>, which was then coupled with amine <b>4</b> to give the final compound as a racemate. Finally, chiral resolution afforded FIXa inhibitor <b>7</b>. Titrant compound <b>8</b> was synthesized using similar methods.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Pd(OAc)<sub>2</sub>, 2,8,9-triisobutyl-2,5,8,9-tetraaza-1-phosphabicyclo[3.3.3]undecane, NaHMDS in THF, dioxane, 100 °C, 3 h; (ii) NaBH<sub>4</sub>, NiCl<sub>2</sub>, EtOH, THF, 25 °C, overnight; (iii) oxalyl chloride, cat. DMF, DCM, 0–25 °C, 2 h; (iv) EtOH, THF, 25 °C, 1 h; (v) Chiral separation on IC column.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23688" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23688" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Previously, our group reported a discovery research effort focused on the development of a novel class of antithrombotic Factor IXa (FIXa) inhibitors.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> A high-throughput screening campaign was conducted and a benzimidazole hit was identified. The discovery chemistry team developed the benzimidazole hit into a highly potent and selective FIXa inhibitor, <b>7</b>, which exhibited a robust response in a pharmacodynamic (PD) assay and demonstrated the characteristics of a true FIXa inhibitor.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> As shown in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, <b>7</b> has good human Factor IXa (hFIXa) potency and is highly selective against off-target human Factor Xa (hFXa). However, when subjected to rat pharmacokinetics (PK) studies, high <i>in vivo</i> clearance of <b>7</b> was observed (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Moreover, high <i>in vitro</i> intrinsic clearance (CLint) values were also observed in human and rat liver microsomal and hepatocyte incubations.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Potency and Rat PK Data of <b>7</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0009.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">hFIXa <i>K</i><sub>i</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">hFXa <i>K</i><sub>i</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> (nM)</th><th class="colsep0 rowsep0" align="center">ratio<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">F%<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">AUC<sub>N</sub> (po)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> μM·h/mpk</th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>1/2</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> h</th><th class="colsep0 rowsep0" align="center">CL<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> mL/min/kg</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9.11</td><td class="colsep0 rowsep0" align="left">2798</td><td class="colsep0 rowsep0" align="left">308</td><td class="colsep0 rowsep0" align="left">11.1</td><td class="colsep0 rowsep0" align="left">0.0970</td><td class="colsep0 rowsep0" align="left">1.24</td><td class="colsep0 rowsep0" align="left">36.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>N</i><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> = 3</td><td class="colsep0 rowsep0" align="left"><i>N</i> = 3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>N</i> = 2</td><td class="colsep0 rowsep0" align="left"><i>N</i> = 2</td><td class="colsep0 rowsep0" align="left"><i>N</i> = 2</td><td class="colsep0 rowsep0" align="left"><i>N</i> = 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SD<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a> = 1.5</td><td class="colsep0 rowsep0" align="left">SD = 737</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SD = 1.2</td><td class="colsep0 rowsep0" align="left">SD = 0.010</td><td class="colsep0 rowsep0" align="left">SD = 0.30</td><td class="colsep0 rowsep0" align="left">SD = 2.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Mean values.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last"><i>N</i>: number of times data acquired.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">SD: standard deviation.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">AUC<sub>N</sub>: AUC normalized by dose.</p></div></div></div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Extracted ion chromatograms (XIC) showing metabolites arising from human liver microsomal (HLM) and rat liver microsomal (RLM) incubations of <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To better understand the high clearance of <b>7</b>, the compound was exposed to human and rat microsomal incubations and the results are shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. Extensive oxidation of <b>7</b> was observed when incubated with both human and rat microsomes. Results from LCMS-MS analysis partially narrowed down the site of oxidation for each metabolite (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). MS/MS spectra and proposed fragmenation patterns are included in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Proposed structures for major oxidative metabolites with chromatographic retention times from the chromatograms shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While the original goal of the metabolite identification study was to improve PK, we were intrigued by the extensive oxidative metabolism observed in multiple regions of the molecule. Each metabolite represents a potential direction for lead optimization. To utilize the results of the microsomal incubation effectively, we would need to know the exact structure of each metabolite. However, pursuing the isolation and characterization of all of the metabolites is extremely resource intensive and time-consuming. Therefore, methodology that can help prioritize the metabolites for isolation and structural characterization is highly desirable. This need prompted us to explore ALIS-AS experiments.</div><div class="NLM_p">Affinity-based competition experiments (ACE) are based on equilibrium affinity selection, whereby a fixed concentration of a mixture of compounds and serially increasing concentrations of a known competitor ligand (titrant) are incubated together with the target protein. The amount of each compound bound to the target protein is then analyzed by ALIS-AS. The MS signals of the ligands and the competitor from such experiments reflect the equilibrium concentrations of each protein–ligand complex, thereby providing information about the equilibrium dissociation constant (<i>K</i><sub>d</sub>) of each component in a complex mixture. As the titrant concentration is increased, MS signals for FIXa-bound metabolites are correspondingly reduced in a sigmoidal response. A curve-fitting algorithm is used to calculate the concentration of titrant at which metabolite MS signal is reduced by 50% compared to a control sample without titrant (ACE<sub>50</sub>). The ability of the titrant to competitively displace the target-bound ligands reveals their binding affinity to the same site of protein and allows relative binding affinity ranking of the ligands.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26">(22-26)</a> In an ACE<sub>50</sub> curve, a higher value indicates a ligand with higher affinity for the protein reflected by a higher titrant concentration required to displace the compound of interest from the binding site.</div><div class="NLM_p">ALIS has demonstrated its unique capability in screening label-free mixture-based libraries and finding hits that were subsequently developed into lead compounds in various classes of protein targets.<a onclick="showRef(event, 'ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44">(32-44)</a> The strength of ALIS in measuring binding affinity directly from a mixture of compounds can be brought into the workflow of metabolite identification. Instead of isolating and characterizing every metabolite and profiling them individually, one can use the FIXa protein and a crude metabolite mixture from rat liver microsomes to rank order binding affinities via ALIS screening. The binding affinity of the metabolite provides an indication of whether a metabolite is potentially active. Consequently, the ALIS screening methodology was utilized to prioritize metabolites of <b>7</b> with highest affinity for FIXa for isolation and subsequent structural characterization.</div><div class="NLM_p">FIXa is a structurally well-defined enzyme target.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Its enzymatic activities have been characterized in several different assays.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> In ALIS screening, a truncated human FIXa protein (415 aa, <i>E. coli</i> derived) was used.</div><div class="NLM_p">Compound <b>8</b>, bearing fluoro (F) substitution on the benzimidazole moiety, is a close analog of <b>7</b>. The hFIXa <i>K</i><sub>i</sub> of <b>8</b> is 10.6 nM (<i>N</i> = 3, SD = 5.6), which is in the same range as that measured for <b>7</b> (<i>K</i><sub>i</sub> = 9.11 nM). Therefore, <b>8</b> was selected as the titrant in the ALIS competition assay. Based on their structures, <b>7</b>, <b>8</b>, and the metabolites of <b>7</b> were assumed to bind to the same site of the FIXa protein. The binding affinity of <b>8</b> can be utilized to assess the binding affinity of the metabolites of <b>7</b> for the FIXa protein in ALIS competition experiments, allowing the metabolites to be rank-ordered.</div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0001.gif" alt="" id="GRAPHIC-d364e885-autogenerated" /></img><div class="NLM_p">The characterization of the binding mode and binding affinity of metabolites of <b>7</b> with FIXa protein was analyzed using ACE described briefly above. In these experiments, a constant concentration of the RLM metabolite mixture derived from <b>7</b> and serially increasing concentrations of the titrant, <b>8</b>, were incubated with the FIXa protein. Protein–ligand complexes were separated from the unbound compounds by SEC. The protein-bound complex of <b>7</b>, its metabolites, and the titrant, <b>8</b>, were then dissociated from the protein using reverse-phase chromatography and finally quantitated by mass spectrometry. The ACE result of <b>7</b> and its metabolites in the presence of the titrant was consistent with the assumption that they share a common binding site of the FIXa protein.</div><div class="NLM_p">As shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>, <b>7</b> and its metabolites were displaced from the binding site of FIXa protein by the titrant. M538a has a higher ACE<sub>50</sub> value than <b>7</b>, indicating it has greater binding affinity in the RLM metabolite mixture derived from <b>7</b>, and therefore requires higher concentration of the titrant, <b>8</b>, for displacement from the binding site. Conversely, M538b had a lower ACE<sub>50</sub> value than <b>7</b>, indicating that M538b has a weaker binding affinity. Finally, M538c and M538d have even weaker binding affinities than M538b. Therefore, the ALIS binding affinity ranking from strongest to weakest is M538a ><b>7</b> > M538b > M538c, M538d. We expected this to be consistent with their respective inhibitory activity in the FIXa enzymatic assay. The rank-ordered binding affinity provides the needed guidance for the prioritization of definitive metabolite isolation and identification work.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. ALIS screen using a crude RLM incubation metabolite mixture. ALIS binding affinity ranking was M538a > <b>7</b> > M538b > M538c, M538d. At each concentration, two replicates were performed and data averaged.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Due to its high binding affinity to FIXa protein, M538a was considered to be a high priority metabolite for isolation and structural characterization. Due to its low abundance, which could pose isolation and characterization challenges using conventional techniques, M538a was subjected to the aforementioned microisolation methodology and MicroCryoProbe NMR characterization. In addition, M538b was selected for definitive structural elucidation due to its close binding affinity compared to compound <b>7</b>. Finally, as a negative control to validate whether ALIS binding affinity is aligned with FIXa enzymatic assay potency, the structural elucidation of one of the weaker binding metabolites, M538c, was also undertaken. It should be noted that M538b and M538c were also selected for PK reasons. In a previous human hepatocyte incubation experiment (not discussed in this report), M538b was shown to be the major metabolite, whereas in the present human liver microsomal (HLM) incubation, M538c was shown to be the major metabolite (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). These two metabolites represent <b>7</b>’s major oxidative metabolism outcome in human <i>in vitro</i> systems.</div><div class="NLM_p">As noted above, incubation of <b>7</b> with liver microsomes resulted in the formation of metabolites M538a, M538b, and M538c; M538a was a minor metabolite relative to M538b and M538c. In order to obtain sufficient quantities of M538a to allow NMR characterization, incubation protocols were optimized. It was found that incubations of <b>7</b> at 100 μM concentration with dog liver microsomes gave the best conversion of <b>7</b> to M538a (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). The same protocol also produced acceptable amounts of M538b and M538c. Even though dog microsomal incubation provided the highest yield of M538a, the absolute conversion was only 0.14% and only about 25 μg of M538a was formed in the large scale incubation mixture. Isolation of μg quantities of metabolites from a complex <i>in vitro</i>/<i>in vivo</i> matrix poses a significant analytical challenge and a systematic approach for microisolation must be implemented. First, due to the limited amounts of metabolites present in the matrix, an analytical scale prep-HPLC system equipped with an analytical HPLC column will need to be employed. Second, analytical control for purity evaluation and concentration measurement of different aliquots should be in place in every step of the isolation protocol such as workup, fraction pooling, and solvent removal to ensure best recovery possible. Next, the buffer used in the isolation (which in most cases is acidic) can cause decomposition of metabolites during the solvent removal step. A probe stability study must be conducted to determine the proper drying procedure to ensure the integrity of metabolites and minimize degradation. This microisolation approach described herein was applied in our study. The final step in the isolation sequence involved dissolving the isolated metabolite in a suitable deuterated solvent for the NMR work and evaporating the sample to dryness a final time.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Dog liver microsomal incubation of <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">By applying the microisolation protocol, microgram quantities of M538a, M538b, and M538c were successfully isolated and the site of oxidation was partially defined by LC-MS. The structures were then unequivocally established by heteronuclear 2D NMR methods (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Isolation and the Characterized Structures of Metabolites of <b>7</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0010.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0011.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table *</sup><p class="last">Tentative stereochemistry based on the known stereochemistry of <b>7</b>.</p></div></div><div></div></div><div class="NLM_p">Structural characterization of low microgram quantities of drug metabolites by NMR has long been a challenging undertaking due to the inherent insensitivity of NMR spectroscopy as compared, for example, to mass spectrometry.<a onclick="showRef(event, 'ref5 ref47'); return false;" href="javascript:void(0);" class="ref ref5 ref47">(5, 47)</a> There has been a continual advancement in NMR probe design over the past several decades.<a onclick="showRef(event, 'ref18 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref18 ref48 ref49">(18, 48, 49)</a> The biggest advance insofar as using NMR to characterize drug metabolites quite likely was the development of a 1.7 mm MicroCryoProbe in 2008–2009, the performance of which has been rigorously benchmarked for small samples.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> When 1.7 mm cryoprobe technology is employed in tandem with a pure shift HSQC (PS-HSQC)<a onclick="showRef(event, 'ref50 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref50 ref51 ref52">(50-52)</a> NMR spectrum, samples of a few micrograms of an isolated drug metabolite can be successfully characterized in very reasonable time.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The sensitivity of this approach can be further enhanced by intentionally folding the PS-HSQC spectrum in the F<sub>1</sub> dimension to either acquire the data in a shorter time or with a commensurate increase in s/n for the same expenditure of spectrometer time.</div><div class="NLM_p">Structural characterization is greatly facilitated by the concerted utilization of accurate mass MS/MS and the acquisition of pure shift HSQC data. The MS/MS data can be utilized to localize the site of oxidative metabolism as denoted by the bracketed segments of the structures shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. While one-dimensional proton spectra may be severely complicated by overlapping extraneous peaks from the chromatographic isolation, e.g., column bleed, contaminating species eluting from the column from previous chromatographic isolations, separating resonances in the second frequency domain of a pure shift HSQC spectrum sidesteps many of the inherent problems of metabolite characterization when only proton homonuclear methods are employed. Fortuitously, <sup>13</sup>C shifts are quite empirical in nature, which allows the <sup>13</sup>C shift perturbation of protonated carbons flanking the site of oxidation to be calculated. Using this approach, the site of hydroxylation can be unequivocally pinpointed. When necessary, pure shift HSQC data can be augmented by long-range heteronuclear shift correlation data, e.g., LR-HSQMBC,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and IDR-(Inverted Direct Response)-HSQC-TOCSY<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> data when there is sufficient sample available.</div><div class="NLM_p">The sample of M538a was the smallest of the three metabolites with ∼12 μg available for analysis by NMR.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> From a PS-HSQC “fingerprint”, it was obvious that one of the protonated carbons of the fluorophenyl moiety was absent from the spectrum of the metabolite.</div><div class="NLM_p">Both fluoro and hydroxyl substituents shift the carbon <i>ortho</i> to the substituent on a phenyl ring upfield. Hence, as shown for the potential 30-hydroxyl metabolite, the C29 resonance between the fluoro and hydroxyl groups would be expected to resonate considerably upfield near 100 ppm. Since none of the protonated aromatic carbons in the PS-HSQC spectrum of M538a resonated upfield of 115.9 ppm, 30-hyroxylation was ruled out. Hydroxylation at the 31-position (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>) would require all three protonated carbons of the fluorophenyl moiety to resonate in the range of 114–117 ppm, which also allowed this possibility to be ruled out. Hence, it was necessary to differentiate between 27- and 29-hydroxylation. The structures were differentiated on the basis of an IDR-HSQC-TOCSY spectrum<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> acquired using a 20 ms mixing time. 27-Hydroxylation would lead to three contiguous protonated carbons whereas 29-hydroxylation would only require two adjacent protonated carbons. The latter case was confirmed with the IDR-HSQC-TOCSY data, establishing the structure of M538a as the 29-hydroxy, which was also consistent with the chemical shifts of the protonated aromatic carbon resonances in the PS-HSQC spectrum.</div><div class="NLM_p">Similarly, the more abundant metabolites, M538b and M538c, were characterized via PS-HSQC and LR-HSQMBC<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> (detailed discussions in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div><div class="NLM_p">Last, the three isolated and structurally characterized μg-quantity metabolites were recovered from NMR experiments and submitted to the FIXa enzymatic assay at a modified low-concentration titration range in order to accurately assess the intrinsic potency of each metabolite. As shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, the FIXa enzymatic potency rank-ordering of M538a, <b>7</b>, M538b, and M538c aligns remarkably well with ALIS FIXa binding affinity ranking shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> This observation further validated the premise that using the resource-sparing ALIS-AS alone on a metabolite mixture can directly rank-order metabolite potency against the target protein. The ALIS-AS assay results can then be used to prioritize further work on the isolation, structural characterization, and synthesis of potentially active metabolites, as well as additional SAR study follow-up on the metabolite(s) of interest. This approach circumvents tedious isolation and structural characterization of every metabolite and instead directs the discovery chemistry effort more effectively. In this particular case, for example, the more potent metabolite, M538a, became a new opportunity for lead optimization.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. FIXa enzymatic assay titration of <b>7</b> and its metabolites. *IC<sub>50</sub> values represent single determinations performed with two replicates per concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75726" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75726" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we have showcased a novel and efficient approach of coupling definitive metabolite isolation/identification and ALIS technology as a means of identifying new SARs in an antithrombotic drug discovery research program. The ALIS-AS technology enabled preliminary affinity ranking using a microsomal incubation mixture without the need of isolating each metabolite. The affinity ranking results were then used to guide and prioritize definitive metabolite isolation and subsequent structural characterization. The isolation of pure metabolites in microgram quantities was made possible by microisolation capabilities, and the structure elucidation of microgram-scale metabolites was enabled by MicroCyroProbe NMR. Last, the intrinsic potencies of the microgram quantities of metabolites were confirmed in a FIXa enzymatic assay and proven to be consistent with their ALIS binding affinities toward the FIXa protein. These results demonstrate that ALIS-AS binding affinity ranking can be used to predict the intrinsic potencies of metabolites in an incubation mixture without isolation and characterization as prerequisites. With the intrinsic potency ranking from an ACE experiment as an initial guide, a more focused effort in definitive metabolite identification can then ensue, leading to quick identification of region(s) on the parent molecule as a lead optimization focus. This approach avoids the traditional tedious isolation, characterization, resynthesis, and <i>in vitro</i> potency assessment for each metabolite in order to identify metabolite(s) of interest. Moreover, as demonstrated herein, ALIS can also reveal highly potent metabolites, some of which could be missed by traditional approaches if such metabolites are present at very low abundance. It is in those highly potent but otherwise overlooked metabolites where potentially new structure–activity relationships that can lead to better drug design may be uncovered. Ultimately, we have demonstrated for the first time an end-to-end workflow using microgram quantities of materials, from microsomal incubation/isolation, and NMR structure elucidation, to <i>in vitro</i> enzymatic assay. This practice eliminates the lengthy and expensive synthetic steps in making pure metabolites for biological assay, which has considerable potential to speed up the process of drug discovery.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88113" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88113" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Chemistry</h3><div class="NLM_p">All reactions were carried out employing standard chemical techniques in ambient atmosphere unless otherwise noted. Solvents used for extraction, washing, and chromatography were HPLC grade. Analytical HPLC was performed on a Waters Acquity Ultra Performance LC with UV detection at 230 and 254 nm along with ELSD and ESI-MS, with all final compounds showing >95% purity and a parent mass ion consistent with the desired structure. (HPLC conditions: Column: Supelco Ascentis Express C18, 2.7 μM, 3.0 × 100 mm. Sovents: A = water/0.05% TFA; B = Acetonitrile/0.05% TFA. Gradient: 10–99% B in 4 min. Flow: 1.0 mL/min, 2 μL injection. UV: 200–400 nm. MS Conditions: Scan Range: 170–900 amu. ESI positive. Run time 5 min). All routine NMR spectra were recorded on a 500 MHz Varian INOVA 500 instrument unless noted otherwise. <sup>1</sup>H chemical shifts are reported as δ values in ppm relative to the residual solvent peak (CD<sub>3</sub>OD = 3.30, acetone-<i>d</i><sub>6</sub> = 2.04). Automated flash column chromatography was performed on a Teledyne ISCO CombiFlash Rf system.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> 2,6-Dichloro-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzoic acid (<b>5</b>)</h4><div id="sec5_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i11" class="anchor-spacer"></div><h5 class="article-section__title" id="_i11"> Step A: Methyl 2,4,6-trichlorobenzoate (<b>10</b>)</h5><div class="NLM_p last">Into a 5000 mL 3-necked round-bottom flask was placed 2,4,6-trichlorobenzoic acid <b>9</b> (200 g, 887 mmol), <i>N</i>,<i>N</i>-dimethylformamide (2000 mL), potassium carbonate (368 g, 2.66 mol), and CH<sub>3</sub>I (379 g, 2.67 mol). The resulting solution was stirred for 12 h at 25 °C. It was then diluted with 3000 mL of water and extracted with 3 × 1500 mL of ethyl acetate. The organic layers were combined, washed with 1 × 2000 mL of brine, dried over anhydrous sodium sulfate, and concentrated under vacuum. This resulted in 192 g (86%) of methyl 2,4,6-trichlorobenzoate <b>10</b> as a light yellow oil.</div></div><div id="sec5_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i12" class="anchor-spacer"></div><h5 class="article-section__title" id="_i12"> Step B: Methyl 2,6-dichloro-4-((diphenylmethylene)amino)benzoate (<b>11</b>)</h5><div class="NLM_p last">Into a 5000 mL 3-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen was placed methyl 2,4,6-trichlorobenzoate <b>10</b> (192 g, 802 mmol), toluene (2000 mL), Cs<sub>2</sub>CO<sub>3</sub> (789 g), Xantphos (46.7 g, 80.7 mmol), diphenylmethanimine (173 g, 956 mmol), and Pd(OAc)<sub>2</sub> (9.1 g, 40.5 mmol). The resulting solution was heated to reflux for 12 h in an oil bath. The reaction mixture was cooled to 25 °C with a water/ice bath. The solid was filtered out and washed with toluene. The combined filtrate was concentrated under vacuum. The residue was applied onto a silica gel column and eluted with ethyl acetate/petroleum ether (1:20). This resulted in 208.1 g (64%) of methyl 2,6-dichloro-4-[(diphenylmethylidene)amino]benzoate <b>11</b> as a colorless oil.</div></div><div id="sec5_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i13" class="anchor-spacer"></div><h5 class="article-section__title" id="_i13"> Step C: Methyl 4-amino-2,6-dichlorobenzoate (<b>12</b>)</h5><div class="NLM_p last">Into a 3000 mL 3-necked round-bottom flask was placed methyl 2,6-dichloro-4-[(diphenylmethylidene)amino]benzoate <b>11</b> (208 g, 542 mmol), tetrahydrofuran (1250 mL), and 2 N hydrogen chloride (504 mL). The resulting solution was stirred for 3 h at 25 °C. The pH value of the solution was adjusted to 9 with aq. sodium bicarbonate and extracted with 3 × 1000 mL of ethyl acetate. The organic layers were combined, dried over anhydrous sodium sulfate, and concentrated under vacuum. The residue was applied onto a silica gel column and eluted with ethyl acetate/petroleum ether (1:20). This resulted in 87 g (69%) of methyl 4-amino-2,6-dichlorobenzoate <b>12</b> as a white solid.</div></div><div id="sec5_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i14" class="anchor-spacer"></div><h5 class="article-section__title" id="_i14"> Step D: Methyl 2,6-dichloro-4-hydrazinylbenzoate hydrochloride (<b>13</b>)</h5><div class="NLM_p last">Into a 2000 mL 3-necked round-bottom flask was placed methyl 4-amino-2,6-dichlorobenzoate <b>12</b> (87 g, 395 mmol), and HCl (37%, 870 mL). This was followed by the addition of a solution of NaNO<sub>2</sub> (27.3 g, 396 mmol) in water (30 mL) dropwise with stirring at 0 °C over 15 min. To this was added a solution of SnCl<sub>2</sub> (210 g, 1.11 mol) in HCl (37%, 200 mL) dropwise with stirring at 0 °C over 30 min. The resulting solution was stirred for 2 h at 0 °C in a water/ice bath. The solid was filtered out and washed with 100 mL of DCM. The combined filtrate was concentrated under vacuum. This resulted in 85.5 g (76%) of methyl 2,6-dichloro-4-hydrazinylbenzoate hydrochloride <b>13</b> as a white solid.</div></div><div id="sec5_1_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i15" class="anchor-spacer"></div><h5 class="article-section__title" id="_i15"> Step E: Methyl 2,6-dichloro-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzoate (<b>14</b>)</h5><div class="NLM_p last">Into a 2000 mL 3-necked round-bottom flask purged and maintained with an inert atmosphere of nitrogen was placed ethylethanecarboximidate hydrochloride (39.0 g, 315 mmol) and dichloromethane (1000 mL). This was followed by the addition of triethylamine (38.2 g, 378 mmol) dropwise with stirring at −40 °C over 30 min. The resulting solution was stirred for 4 h at 0 °C in a water/ice bath. To this was added methyl 2,6-dichloro-4-hydrazinylbenzoate hydrochloride <b>13</b> (85.5 g, 315 mmol), pyridine (99.69 g, 1.26 mol), and (diethoxymethoxy)ethane (934 g, 6.30 mol). The resulting solution was concentrated under vacuum. The residual solution was stirred for 16 h while the temperature was maintained at 80 °C in an oil bath. The reaction mixture was cooled to 25 °C with a water/ice bath and then poured into 1000 mL of EtOAc/water (1:1). The aqueous phase was extracted with 3 × 1000 mL of ethyl acetate. The organic layers were combined, dried over anhydrous sodium sulfate and concentrated under vacuum. The crude product (80 g) was purified with Flash-Prep-HPLC with the following conditions: column, C18-SG; mobile phase, AcCN/aq. NH<sub>4</sub>HCO<sub>3</sub> (0.1%) = 40/60 increasing to AcCN/aq. NH<sub>4</sub>HCO<sub>3</sub> (0.1%) = 55/45 within 25 min; detector, UV 254 nm. This resulted in 35.7 g (38%) of methyl 2,6-dichloro-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzoate <b>14</b> as a white solid.</div></div><div id="sec5_1_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i16" class="anchor-spacer"></div><h5 class="article-section__title" id="_i16"> Step F: 2,6-Dichloro-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzoic acid (<b>5</b>)</h5><div class="NLM_p last">Into a 500 mL 3-necked round-bottom flask was placed methyl 2,6-dichloro-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzoate <b>14</b> (14.7 g, 51.4 mmol), MeOH/H<sub>2</sub>O/THF (1:1:4, 240 mL), and LiOH·H<sub>2</sub>O (10.8 g, 257 mmol). The resulting solution was stirred for 12 h at 60 °C in an oil bath. The reaction mixture was cooled to 25 °C, then concentrated under vacuum. The residual solution was diluted with 100 mL of water, then washed with 200 mL of ethyl acetate. The aqueous layer was adjusted to pH 3 with HCl (2 M). The solid was collected by filtration and dried. This resulted in 9.8 g (67%) of 2,6-dichloro-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzoic acid <b>5</b> as a white solid. LC-MS: <i>m</i>/<i>z</i> = 272 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (Bruker 400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm): δ 14.29 (s, 1H), 9.31 (s, 1H), 8.06 (s, 2H), 2.38 (s, 3H).</div></div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (<i>R</i>)-2,6-dichloro-<i>N</i>-(2-(3-Fluorophenyl)-2-(2-methyl-1<i>H</i>-benzo[d]imidazol-5-yl)ethyl)-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzamide (<b>7</b>)</h4><div id="sec5_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> Step A: (±)-2-(3-Fluorophenyl)-2-(2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)acetonitrile (<b>3</b>)</h5><div class="NLM_p last">The reactor was charged with palladium(II) acetate (0.160 g, 0.711 mmol), 2,8,9-triisobutyl-2,5,8,9-tetraaza-1-phosphabicyclo[3.3.3]undecane (0.507 mL, 1.42 mmol), and 5-bromo-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole <b>1</b> (1.5 g, 7.11 mmol). The vessel was evacuated and refilled with N<sub>2</sub> for three cycles and degassed dioxane (14.2 mL) was added, followed by NaHMDS (28.4 mL, 28.4 mmol). The reaction was stirred at 25 °C for 20 min before 2-(3-fluorophenyl)acetonitrile <b>2</b> (73.3 mg, 0.543 mmol) was added. The reaction was then capped and heated at 100 °C for 3 h. More NaHMDS (6 mL) was added and the reaction was stirred for another 1 h. Additional NaHMDS (4 mL) was added and the reaction was stirred for another 1 h before it was cooled to room temperature, and quenched with water. The aqueous layer was extracted with EtOAc (3 × 300 mL), washed with brine (300 mL), dried over MgSO<sub>4</sub>, filtered, and the filtrate was concentrated <i>in vacuo</i>. The crude material was purified by chromatography on silica gel (eluent: 0–100% A in B. A: 10% MeOH in DCM; B: DCM.) to give 1.62 g (±)-2-(3-Fluorophenyl)-2-(2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)acetonitrile <b>3</b> as a tan goo. LC-MS: <i>m</i>/<i>z</i> = 266.0 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> Step B: 2,6-Dichloro-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzoyl chloride (<b>6</b>)</h5><div class="NLM_p last">At 0 °C, in a preweighed flask containing carboxylic acid <b>5</b> (2.28 g, 8.38 mmol) in 3 mL DCM, was added oxalyl chloride (1.50 mL, 16.8 mmol), followed by one drop of DMF. After 0.5 h, the reaction was warmed to 25 °C and stirred for 2 h. The reaction mixture was concentrated and the excess oxalyl chloride was azeotropically removed with toluene. LC-MS: <i>m</i>/<i>z</i> = 289.9 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Step C: (±)-2-(3-Fluorophenyl)-2-(2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)ethanamine (<b>4</b>)</h5><div class="NLM_p last">Under N<sub>2</sub>, to the step A product <b>3</b> (1.62 g, 6.11 mmol), anhydrous nickel(II) chloride (0.791 g, 6.11 mmol) in anhydrous ethanol (15.3 mL) and THF (15.3 mL) was added sodium borohydride (1.39 g, 36.6 mmol). The reaction was stirred at 25 °C overnight. LC-MS: <i>m</i>/<i>z</i> = 270.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Step D: (<i>R</i>)-2,6-Dichloro-<i>N</i>-(2-(3-fluorophenyl)-2-(2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)ethyl)-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzamide (<b>7</b>)</h5><div class="NLM_p">The reaction mixture from step C was added to the step B product <b>6</b> (2.66 g, 9.16 mmol). The vial containing the step C reaction was rinsed with a total of 5 mL THF that was also added to the step B product <b>6</b>. The mixture was stirred at 25 °C for 1 h. The reaction was then quenched with a saturated NaHCO<sub>3</sub> aqueous solution (200 mL), diluted with 300 mL EtOAc, agitated, and filtered. The filtrate was collected, and the phases separated. The aqueous phase was extracted with EtOAc (3 × 300 mL). The combined aqueous layers were washed with brine (300 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The crude material was purified by chromatography on silica gel (eluent: 0–100% A in B. A: 10% MeOH in DCM; B: DCM.) to yield 0.88 g racemic product. LC-MS: <i>m</i>/<i>z</i> = 523.0 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.97 (s, 1 H); 7.79 (s, 2 H); 7.46 (s, 1 H); 7.39 (t, <i>J</i> = 8.3 Hz, 1 H); 7.33–7.25 (m, 1 H); 7.22–7.09 (m, 3 H); 6.91 (td, <i>J</i> = 8.5, 2.4 Hz, 1 H); 4.55 (t, <i>J</i> = 8.1 Hz, 1 H); 4.16–4.00 (m, 2 H); 2.53 (s, 3 H); 2.39 (s, 3 H).</div><div class="NLM_p last">Chiral Separation. IC column (2 × 15 cm), 35% ethanol (0.1% DEA)/CO<sub>2</sub>, 50 mL/min, 100 bar, 220 nm, injection volume: 1 mL; 9 mg/mL in methanol. Fast-eluting enantiomer A, 378 mg; slow-eluting enantiomer B, 379 mg. The title compound <b>7</b> is enantiomer B. (The assignment of stereochemistry was based on known X-ray cocrystallization data between FIXa enzyme and analogs of <b>7</b>.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a>).</div></div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (<i>S</i>)-2,6-Dichloro-<i>N</i>-(2-(7-fluoro-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-2-(3-fluorophenyl)ethyl)-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzamide (<b>8</b>)</h4><div id="sec5_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Step A: 5-Bromo-7-fluoro-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>16</b>)</h5><div class="NLM_p last">A solution of 5-bromo-3-fluorobenzene-1,2-diamine <b>15</b> (1 g, 4.88 mmol), acetic acid (2.93 g, 48.8 mmol) in PPA (polyphosphoric acid) (4.88 mL) was heated to 135–140 °C for 4 h under N<sub>2</sub>. The reaction was then cooled to 25 °C and transferred into 25 mL of ice cold water. Charcoal (about 1 g) was added and the mixture was filtered over a Celite bed, which was rinsed with water. The filtrate was basified with aq. ammonia solution until pH = 10–12. The basified reaction mixture was extracted with ethyl acetate (3 × 100 mL), and the combined ethyl acetate layers were dried over MgSO<sub>4</sub> and concentrated to a yellow solid residue. To the residue was added hexanes (5 mL) and the mixture was filtered to reveal 0.93 g 5-bromo-7-fluoro-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole <b>16</b> as a white solid. LC-MS: <i>m</i>/<i>z</i> = 228.9 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Step B: 2-(7-Fluoro-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-2-(3-fluorophenyl)acetonitrile (<b>17</b>)</h5><div class="NLM_p last">The reactor was charged with palladium(II) acetate (0.037 g, 0.166 mmol), 2,8,9-triisobutyl-2,5,8,9-tetraaza-1-phosphabicyclo[3.3.3]undecane (114 mg, 0.332 mmol), and 5-bromo-7-fluoro-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole <b>16</b> (380 mg, 1.66 mmol). The vessel was evacuated and refilled with N<sub>2</sub> for 3 cycles and degassed dioxane (3.32 mL) was added, followed by NaHMDS (6.64 mL, 6.64 mmol). The reaction was stirred at 25 °C for 20 min before 2-(3-fluorophenyl)acetonitrile (269 mg, 1.99 mmol) was added. The reaction was then capped and heated at 100 °C for 6 h. The reaction was cooled to room temperature, and quenched with water. The aqueous layer was extracted with EtOAc (3 × 100 mL), washed with brine (100 mL), dried over MgSO<sub>4</sub>, filtered, and the filtrate was concentrated <i>in vacuo</i>. The crude material was passed through a silica short column using eluent 0–100% A in B (A: 10% MeOH in DCM; B: DCM) to yield a crude product, which was used directly in the next step. LC-MS: <i>m</i>/<i>z</i> = 284.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> Step C: 2-(7-Fluoro-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-2-(3-fluorophenyl)ethan-1-amine (<b>18</b>)</h5><div class="NLM_p last">Under N<sub>2</sub>, to product <b>17</b> (180 mg, 0.635 mmol), anhydrous nickel(II) chloride (82 mg, 0.635 mmol) in anhydrous ethanol (1.6 mL) and THF (1.6 mL) was added sodium borohydride (144 mg, 3.81 mmol). The reaction was stirred at 25 °C overnight to produce <b>18</b>.</div></div><div id="sec5_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> Step D: (<i>S</i>)-2,6-Dichloro-<i>N</i>-(2-(7-fluoro-2-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-2-(3-fluorophenyl)ethyl)-4-(3-methyl-1<i>H</i>-1,2,4-triazol-1-yl)benzamide (<b>8</b>)</h5><div class="NLM_p last">The reaction mixture containing <b>18</b> from above was then transferred to a vial containing compound <b>6</b> (119 mg, 0.318 mmol). The vial containing the reaction mixture from above was rinsed with a total of 2 mL THF and then added to the vial containing compound <b>6</b>. The mixture was stirred at 25 °C for 1 h. The reaction was then quenched with saturated NaHCO<sub>3</sub> aqueous solution, diluted with 10 mL EtOAc, agitated, and filtered. The filtrate was collected, and phases were separated. The aqueous phase was extracted with EtOAc (3 × 10 mL). The combined aqueous layers were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The crude material was purified by chromatography on silica gel (eluent: 0–100% A in B. A: 10% MeOH in DCM; B: DCM) to yield 90 mg racemic product containing impurity. This crude product was subjected to chiral resolution and purity upgrade: IC column (2 × 15 cm), 45% 2:1 ACN:isopropanol(0.1% DEA)/CO<sub>2</sub>, 60 mL/min, 100 bar, 220 nm, inj vol: 0.5 mL; 8 mg/mL 1:2 DCM:methanol. Faster eluent, enantiomer A, 25 mg; Slower eluent, enantiomer B, 22 mg. The title compound <b>8</b> is enantiomer B. LC-MS: <i>m</i>/<i>z</i> = 541.1 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.8 (brs, 1 H), 9.02 (s, 1 H); 8.07 (s, 1 H); 7.85 (s, 2 H); 7.37 (t, <i>J</i> = 5.5 Hz, 2 H); 7.31–7.24 (m, 2 H); 6.99 (m, 2 H); 4.62 (t, <i>J</i> = 7.9 Hz, 1 H); 4.25–4.12 (m, 2 H); 2.55 (s, 3 H); 2.38 (s, 3 H).</div></div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> FIXa/FXa Enzymatic Assay Protocols</h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Materials</h4><div class="NLM_p last">Buffer components PEG-8000, HEPES, and CaCl<sub>2</sub>·H<sub>2</sub>O were obtained from Hampton Research. Sodium chloride and <i>n</i>-Acetyl-KPR-AFC (AFC = 7-amido-4-trifluoro-methylcoumarin) were obtained from Sigma. CH<sub>3</sub>SO<sub>2</sub>-<span class="smallcaps smallerCapital">d</span>-cyclohexylGly-Gly-Arg-AFC·AcOH), was obtained from CPC Scientific Custom Peptide. Human FXa was purchased from Sekisui Diagnostics. Recombinant human Factor IXa enzyme was purchased from Haemtech Technologies and further purified via a ResS column.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Methods</h4><div class="NLM_p">Test compounds were titrated with DMSO as the diluent in a 10-point dose response in a separate step followed by a 60-fold dilution into the reaction. Assay buffer contained 50 mM HEPES, pH 7.4, 5 mM CaCl<sub>2</sub>·2H<sub>2</sub>O, 150 mM NaCl, 0.1% PEG-8000. Assay pipetting steps to 384 well microplates were performed with a Hamilton Microlab Star robotic platform.</div><div class="NLM_p">Enzyme activity was measured by recording the increase in fluorescence signal after cleavage of a labeled tripeptide and subsequent release of the free fluorophore AFC as follows.</div><div class="NLM_p">Human Factor IXa protein was incubated with test compound for 30 min followed by the addition of peptide substrate CH<sub>3</sub>SO<sub>2</sub>-D-CHG-Gly-Arg-AFC·AcOH, resulting in a 60-fold dilution of compound and final concentration of 2 nM FIXa and 250 μM substrate. After incubation at ambient temperature for 1 h, the fluorescence was measured on an Envision Multilabel reader with 405 nm excitation and 510 nm emission filters.</div><div class="NLM_p">FXa activity was determined using the same methodology with final concentrations of 15 nM FXa and 100 μM n-Acetyl-KPR-AFC.</div><div class="NLM_p last">Inhibitory activity (IC<sub>50</sub>) of test compounds was determined from a dose titration using a four parameter logistic fit, and subsequently converted to <i>K</i><sub>i</sub> using the Cheng-Prusoff equation (<i>K</i><sub>i</sub> = IC<sub>50</sub>/((1 + (<i>S</i>/<i>K</i><sub>M</sub>))) for competitive inhibition.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Selectivity of the compounds for Factor IXa is represented as a ratio of <i>K</i><sub>i</sub> values.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Rat PK Protocols</h3><div class="NLM_p last">PK studies were conducted in fasted male Hans-Wistar rats (Charles River Laboratories, Inc.), administering a dose of 0.5 mg/kg bolus intravenously (IV) and 2 mg/kg orally (PO).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Metabolite Incubation, Isolation, and Analysis</h3><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Materials</h4><div class="NLM_p last">Reduced nicotinamide adenine dinucleotide phosphate (NADPH), acetonitrile, labetalol, water, methanol, and formic acid were purchased from Sigma-Aldrich Co. (St. Louis, MO), and were the highest grade available. All other reagents and solvents were obtained from Thermo Fisher Scientific Inc. (Waltham, MA). Rat liver microsomes (male sprague dawley with cat#452501), dog liver microsomes (male beagle cat#452601), and human liver microsomes (cat#452117) were purchased from BD Gentest (San Jose, CA).</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Incubations</h4><div class="NLM_p last">Dog liver microsomes (1 mg protein/mL) were suspended in phosphate buffer (100 mM, pH 7.4) containing MgCl<sub>2</sub> (6 mM). Substrate was dissolved in DMSO and was added to a final concentration of 100 μM, such that the concentration of organic solvent in the incubation mixture did not exceed 0.2%. The reactions were initiated at 37 °C by the addition of 1 mM NADPH and terminated at 4 h by the addition of two volumes of acetonitrile. The precipitated proteins were removed by centrifugation and the resulting supernatant was subjected to Genevac dry-down and reconstitution for microscale isolation/purification.</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> LC-MS/MS</h4><div class="NLM_p">Liquid chromatography separations were performed on a Waters Acquity UPLC system (Waters, Milford, MA) equipped with a photodiode array detector monitored in wide band from 190 to 400 nm. The mobile phase program consisted of initial linear gradient from 5% B to 50% B in 10 min, then to 95% B in 3 min, and held for 4 min with post time for 4 min (A = 0.1% formic acid in water; B = 0.1% formic acid in acetonitrile).</div><div class="NLM_p last">The column used was a Phenomenex Kinex C18 (50 × 2.1 mm, 2.6 μm) operated at a flow rate of 0.6 mL/min at 40 °C. Waters QTOF Premier mass spectrometer (Waters, Milford, MA) was operated in the positive ion, V-mode at a mass resolution of 9000 (fwhm) at <i>m</i>/<i>z</i> 556. For the Premier, the source temperature was set at 110 °C, the capillary voltage was set at 3.0 kV, the sample cone voltage was set at 30 V, and the desolvation temperature was set at 250 °C and MCP plate was optimized at 1900 V. Nitrogen was used for desolvation and cone and gas was set at 800 and 20 L/h, respectively. The collision gas was argon at cell pressure of 12 psi. Scans were at 0.2 s duration with a 0.02 s interscan delay. For MS/MS measurements, the collision energy was set at 30 V. For MS<sup>E</sup> experiments, a collision energy ramp from 15 to 45 V was used. The data acquired in a centroid mode were collected from 50 to 1250 amu and MS/MS data collected from 50 to 700 amu. The lock mass was leucine enkephalin (<i>m</i>/<i>z</i> 556.2771) and was infused at 10 μL/min at a concentration of 500 pg/μL.</div></div><div id="sec5_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Metabolite Isolation</h4><div id="sec5_4_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Instrumentation</h5><div class="NLM_p last">An Agilent 1260 analytical scale preparative HPLC system (Agilent Technologies, Palo Alto, CA, USA) equipped with an auto sampler (G1367E), a binary pump (G1312B), a column oven (G1316C), and a diode array detector (G1315D) were employed for the isolation of metabolites from microsomal incubation mixture. The system was operated using the Chemstation software (version 5.5.2).</div></div><div id="sec5_4_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Procedure</h5><div class="NLM_p last">A RPLC method was developed to isolate metabolites from the microsomal incubation of compound <b>7</b>. The conditions are as follows: column, XBridge C18 column (4.6 x150 mm, 3.5 μm; Waters, Milford, MA); flow rate 1.5 mL/min; column temperature, 25 °C; mobile phase, A, 0.1% formic acid in water(v/v), B, 0.1% formic acid in acetonitrile; gradient, A/B, 95/5, V/V, to A/B, 50/50, V/V in 13 min, then to A/B, 5/95 in 1 min and hold for 1 min; post time 6 min; detection, UV at 210 nm; injection volume, 40 μL. Fractions were collected at desired time intervals during the run. The collected fractions were pooled and subjected to solvent removal using a Genevac system (HT-4XHCl) (Genevac Inc., Stone Ridge, NY, USA). See <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for preparative HPLC chromatogram.</div></div></div><div id="sec5_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Metabolite Amount and Purity Assessment</h4><div class="NLM_p last">The amount and purity of isolated metabolites were determined by liquid chromatography using a Waters Acquity UPLC system (Waters, Milford, MA) equipped with a photodiode array detector monitored in wide band from 210 to 400 nm. The mobile phase program consisted of initial linear gradient from 5% B to 40% B in 10 min, then to 95% B in 3 min, and held for 4 min with post time for 4 min (A = 0.1% formic acid in water; B = 0.1% formic acid in acetonitrile). The column used was a Phenomenex Kinex C18 (50 × 2.1 mm, 2.6 μm) operated at a flow rate of 0.6 mL/min at temperature of 40 °C. Purity of each metabolite is assessed by UV at 218 nm (Figure S3 in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). The concentration of isolated metabolite is based on its UV response comparing to compound <b>7</b> as standard with concentration at 0.052 mg/mL and detection wavelength at 218 nm. The amount of metabolite is then estimated by its concentration times the volume.</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Metabolite NMR Data Acquisition</h3><div class="NLM_p">All metabolite NMR data were acquired using a Bruker AVANCE III, three channel 600 MHz NMR spectrometer equipped with a 1.7 mm TXI MicroCryoProbe. Following chromatographic isolation, metabolite samples were finally dried down from 1 to 2 mL of either acetonitrile<i>-d</i><sub>3</sub> or DMSO-<i>d</i><sub>6</sub> (Cambridge Isotope Laboratories, Andover, MA) in a 2 mL conical bottom glass vial that had been pre-rinsed with deuterated solvent and then dried. The metabolite sample was dissolved in 35 μL DMSO-<i>d</i><sub>6</sub> (99.96% D) and then transferred to a 1.7 mm NMR tube (Bruker BioSpin, Billerica, MA) using a 24G flexible Teflon needle (Wilmad Glass Co., Vineland, NJ) attached to a 100 μL Hamilton gastight syringe (Hamilton Company, Reno, NV).</div><div class="NLM_p last">A typical metabolite data acquisition protocol employed the acquisition of a proton reference spectrum with water suppression; the data were acquired with the transmitter centered on the water resonance at ∼3.34 ppm. Proton chemical shifts were referenced to the center line of the residual proton DMSO multiplet at 2.50 ppm. A proton spectral width of 16 ppm digitized with 64K data points was typically employed with the number of transients accumulated dependent on the amount of metabolite available for analysis. Pure shift HSQC (PS-HSQC) spectra were acquired using the pulse sequence previously described.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Data were acquired with water suppression and multiplicity-editing; data were phased so that CH/CH<sub>3</sub> resonances exhibited positive phase while CH<sub>2</sub> resonances were inverted. When the focus of the PS-HSQC experiment was aliphatic correlations, the <sup>1</sup><i>J</i><sub>CH</sub> coupling constant-based delays were optimized for 145. For experiments focused on aromatic correlations, the delay for <sup>1</sup><i>J</i><sub>CH</sub> couplings was optimized for 165 Hz. The loop counter for application of the BIRD pulses during acquisition was set to <i>l</i>0 = 8 for all experiments performed. Long-range heteronuclear correlations were observed using the recently developed LR-HSQMBC experiment<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> (detailed structure elucidation discussions in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> ALIS Experiment</h3><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Materials</h4><div class="NLM_p last">Buffer reagents and β-lactoglobulin were purchased from Sigma-Aldrich (St. Luis, MO) and used as received. HPLC solvents were purchased from Fisher Scientific (Fair Lawn, NJ). Factor IXa protein (415 amino acids) was prepared according to the literature by the SPRI Protein Science Group and was purified to apparent homogeneity as assessed by SDS-PAGE, SEC, LC-MS, and <i>N</i>-terminal sequence.</div></div><div id="sec5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Sample Preparation</h4><div id="sec5_6_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> Metabolite Solution Preparation</h5><div class="NLM_p last">500 μL of RLM microsomal incubation solution was mixed with 500 μL of acetonitrile to precipitate proteins. After centrifugation at 14 000 rpm for 10 min, the supernatant was dried using a speed vac. The residue was resuspended in 75 μL of Tris buffer (50 mM Tris, pH 8.0, 300 mM NaCl, 0.1 mM DTT) to yield an aqueous metabolite mixture.</div></div><div id="sec5_6_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> Metabolite Sample for Affinity Competition Assay</h5><div class="NLM_p last">A mixture containing metabolites and FIXa protein was prepared by combining 6 μL of metabolite solution described above with 34 μL of 23.5 μM of FIXa protein. A 1 μL DMSO aliquot of a serially diluted stock solution of titrant <b>8</b> (10 000, 5000, 2500, 1250, 624, 312, 156, 78, 38.8, 19.6, 10, 5, and 2.4 μM) was dissolved in 19 μL of above Tris buffer. The resulting solutions were mixed by repeated pipetting and clarified by centrifugation at 10 000 <i>g</i> for 10 min. The FIXa/metabolite solution (6 μL) and titration solutions (6 μL) were combined to yield a series of 12 μL experimental samples containing 10 μM FIXa protein, metabolites, and varying concentration of the titrant <b>8</b> (250, 125, 62.5, 31.25, 15.6, 7.8, 3.9, 1.95, 0.97, 0.49, 0.25, 0.12, and 0.06 μM). These prepared samples were incubated at room temperature for 30 min, and then chilled to 4 °C prior to ALIS-AS analysis.</div></div></div><div id="sec5_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> ALIS-AS Data Acquisition</h4><div id="sec5_6_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> Size-Exclusion Chromatography (SEC) Separation</h5><div class="NLM_p last">The ALIS-AS hardware configuration used in this study has been described in previous publications.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26">(22-26)</a> Briefly, SEC was performed at 4 °C using 700 mM of ammonium acetate (NH<sub>4</sub>OAc) buffer of pH 7.5. SEC columns were prepared in-house by proprietary methods. An Agilent (Palo Alto, CA) isocratic pump (G1310A) fitted with an Agilent online degasser (G1322A) was used for eluant delivery at 300 μL/min. Using this configuration, the SEC retention times were reproducible to better than ±2 s for a ∼20 s chromatography run. With reproducible SEC chromatography, the slight variations in retention time from sample-to-sample were not a significant source of error in this method. Sample injection volume was 4.0 μL/injection; each sample was injected twice.</div></div><div id="sec5_6_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> RHPLC Conditions</h5><div class="NLM_p last">The eluant from the SEC column was passed through a UV detector (Agilent G1314A using a G1313 micro flow cell) where the band containing the protein–ligand complexes was identified by its native UV absorbance at 230 nm. After a pause to allow the band to leave the first detector and enter a valving arrangement, the protein–ligand complex peak was automatically transferred to an RPC column (Higgins Targa-C18, Higgins Analytical Inc., Mountain View, CA). Ligands were dissociated from the complex, trapped, and desalted at the head of the RPLC column with 12% acetonitrile in water (0.1% formic acid) for 3 min. The ligands were eluted into the mass spectrometer using a gradient of 12–27% acetonitrile (0.1% formic acid) in water (0.1% formic acid) over 22 min using an Agilent capillary binary pump (G1376A) for eluant delivery at 20 μL/min. To promote dissociation of ligands from the protein–ligand complex, the RPLC column was maintained at 40 °C using an Agilent G1316A column compartment.</div></div><div id="sec5_6_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> MS Conditions</h5><div class="NLM_p last">In this study, MS analysis was performed using a Waters LCT Premier high-resolution time-of-flight mass spectrometer (Milford, MA) with positive mode ionization occurring from a standard nebulized ESI source with the capillary set at 2.7 kV, a desolvation temperature of 250 °C, a source temperature of 100 °C, and 30 V “cone”.</div></div><div id="sec5_6_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> Raw Data Collection</h5><div class="NLM_p">Based on the metabolic profile of <b>7</b>, several ions are monitored such as (M), (M+16), (M+32), along with titrant (<b>8</b>) ions. The positional isomers of some ions were well separated. The Waters Masslynx program was used to calculate the peak areas. The areas underlying the extracted ion chromatograms (XICs) for the singly protonated, doubly protonated, and monosodiated ([M + H]<sup>+</sup>, [M+2H]<sup>2+</sup>, [M + Na]<sup>+</sup>) species were summed.</div><div class="NLM_p last">As binding control experiments, the RLM metabolite mixture from <b>7</b> was incubated with 10 μM of β-lactoglobulin. The sample was analyzed with the same procedure as above. No β-lactoglobulin-bound RLM metabolites of <b>7</b> were detected, which indicated that RLM metabolites of <b>7</b> specifically bind with FIXa protein.</div></div></div><div id="sec5_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Data Analysis</h4><div class="NLM_p last">For each AS-MS experiment representing a single data point in an Affinity Competition Experiment 50% titration (ACE<sub>50</sub>), the peak area of the ions of interest for each data point was normalized for the entire titration curve by dividing each ion’s raw response by its highest response in the titration experiment (typically a data point where the titrant concentration was lowest). The titration data were then fitted to a variable slope sigmoidal dose–response using GraphPad Prism (version 6.0, GraphPad Software, La Jolla, CA, <a href="http://www.graphpad.com" class="extLink">www.graphpad.com</a>) with a maximum normalized value of 1.0. The titrant concentration at which the fit curve passed through 0.5 represents the ACE<sub>50</sub> value. Because the concentration of competitor necessary to reduce the ligand response to one-half its value in the absence of competitor defines the affinity ranking, normalized responses are a more readily interpretable way to represent the data than raw responses. No calibration curves are necessary to determine the affinities of the mixture components.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01293">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01554" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01554" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01293" class="ext-link">10.1021/acs.jmedchem.5b01293</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Experimental procedures and additional results and discussions (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01293/suppl_file/jm5b01293_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Data table (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01293/suppl_file/jm5b01293_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01293/suppl_file/jm5b01293_si_001.pdf">jm5b01293_si_001.pdf (944.26 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01293/suppl_file/jm5b01293_si_002.csv">jm5b01293_si_002.csv (1.5 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01293" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31643" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31643" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ting Zhang</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#06726f6861597c6e67686135466b6374656d2865696b"><span class="__cf_email__" data-cfemail="ed9984838ab297858c838adead80889f8e86c38e8280">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Liu</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#cdb4a2a3aa92a1a4b8ff8da0a8bfaea6e3aea2a0"><span class="__cf_email__" data-cfemail="097066676e5665607c3b49646c7b6a62276a6664">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xianshu Yang</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gary E. Martin</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huifang Yao</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jackie Shang</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Randal M. Bugianesi</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenneth P. Ellsworth</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa M. Sonatore</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Nizner</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edward C. Sherer</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susan E. Hill</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian W. Knemeyer</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wayne M. Geissler</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Dandliker</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roy Helmy</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harold B. Wood</span> - <span class="hlFld-Affiliation affiliation">†Department of Medicinal
Chemistry, ‡Department
of Process and Analytical Chemistry, and ¶Department of Chemistry Modeling and
Informatics, Merck Research Laboratories,
Merck & Co., Inc., PO Box 2000, Rahway, New Jersey 07065, United States</span>; 
    <span class="hlFld-Affiliation affiliation">#Department of Pharmacology and ⊥Department of Pharmacokinetics,
Pharmacodynamics
and Drug Metabolism, Merck Research Laboratories,
Merck & Co., Inc., 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="hlFld-Affiliation affiliation">∥Department of Pharmacokinetics, Pharmacodynamics
and Drug Metabolism, and §Department of Pharmacology, Merck Research Laboratories, Merck & Co., Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d364e2062-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77476" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77476" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to acknowledge Merck Discovery Process Chemistry group for providing synthetic intermediates, Merck Department of Pharmacokinetics for preliminary PK studies, and Drs. Kevin Bateman and Paul O’ Shea for their encouragment and invaluable discussions.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i53" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i53"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i54" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i54"> Abbreviations</h2><tr><td class="NLM_term">AS-ALIS</td><td class="NLM_def"><p class="first last">affinity selection–mass spectrometry-based automated ligand identification system</p></td></tr><tr><td class="NLM_term">ACE</td><td class="NLM_def"><p class="first last">affinity-based competition experiments</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">ELSD</td><td class="NLM_def"><p class="first last">evaporative light scattering detectordynamic light scattering</p></td></tr><tr><td class="NLM_term">ESI-MS</td><td class="NLM_def"><p class="first last">electrospray ionization mass spectrometer</p></td></tr><tr><td class="NLM_term">F%</td><td class="NLM_def"><p class="first last">bioavailability</p></td></tr><tr><td class="NLM_term">FIXa</td><td class="NLM_def"><p class="first last">Factor IXa</p></td></tr><tr><td class="NLM_term">h</td><td class="NLM_def"><p class="first last">hour(s)</p></td></tr><tr><td class="NLM_term">hFIXa</td><td class="NLM_def"><p class="first last">human Factor IXa</p></td></tr><tr><td class="NLM_term">FXa</td><td class="NLM_def"><p class="first last">Factor Xa</p></td></tr><tr><td class="NLM_term">hFXa</td><td class="NLM_def"><p class="first last">human Factor Xa</p></td></tr><tr><td class="NLM_term">IDR</td><td class="NLM_def"><p class="first last">inverted direct response</p></td></tr><tr><td class="NLM_term">HSQC</td><td class="NLM_def"><p class="first last">heteronuclear single quantum coherence</p></td></tr><tr><td class="NLM_term">TOCSY</td><td class="NLM_def"><p class="first last">total correlation spectroscopy</p></td></tr><tr><td class="NLM_term">HLM</td><td class="NLM_def"><p class="first last">human liver microsome</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">HR-MS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">LC-MS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">LR-HSQMBC</td><td class="NLM_def"><p class="first last">long-range heteronuclear single quantum multiple bond correlation</p></td></tr><tr><td class="NLM_term">min</td><td class="NLM_def"><p class="first last">minute(s)</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">phramacodynamics</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PS-HSQC</td><td class="NLM_def"><p class="first last">pure shift heteronuclear single quantum coherence</p></td></tr><tr><td class="NLM_term">RLM</td><td class="NLM_def"><p class="first last">rat liver microsome</p></td></tr><tr><td class="NLM_term">RPLC</td><td class="NLM_def"><p class="first last">reverse-phase liquid chromatography</p></td></tr><tr><td class="NLM_term">s</td><td class="NLM_def"><p class="first last">second(s)</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">SDS-PAGE</td><td class="NLM_def"><p class="first last">sodium dodecyl sulfate polyacrylamide gel electrophoresis</p></td></tr><tr><td class="NLM_term">SEC</td><td class="NLM_def"><p class="first last">size-exclusion chromatography</p></td></tr><tr><td class="NLM_term">s/n</td><td class="NLM_def"><p class="first last">signal-to-noise</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>1/2</sub></td><td class="NLM_def"><p class="first last">half-life</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i55">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 56 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Pelkonen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turpeinen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uusitalo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rautio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raunio, H.</span><span> </span><span class="NLM_article-title">Prediction of drug metabolism and interactions on the basis of in vitro investigations</span> <span class="citation_source-journal">Basic Clin. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">175</span><span class="refDoi"> DOI: 10.1111/j.1742-7843.2005.pto960305.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1111%2Fj.1742-7843.2005.pto960305.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=15733211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisFSmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2005&pages=167-175&author=O.+Pelkonenauthor=M.+Turpeinenauthor=J.+Uusitaloauthor=A.+Rautioauthor=H.+Raunio&title=Prediction+of+drug+metabolism+and+interactions+on+the+basis+of+in+vitro+investigations&doi=10.1111%2Fj.1742-7843.2005.pto960305.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of drug metabolism and interactions on the basis of in vitro investigations</span></div><div class="casAuthors">Pelkonen, Olavi; Turpeinen, Miia; Uusitalo, Jouko; Rautio, Arja; Raunio, Hannu</div><div class="citationInfo"><span class="NLM_cas:title">Basic & Clinical Pharmacology & Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">167-175</span>CODEN:
                <span class="NLM_cas:coden">BCPTBO</span>;
        ISSN:<span class="NLM_cas:issn">1742-7835</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Drug metab. profoundly affects drug action, because almost all drugs are metabolized in the body and thus their concns. and elimination rates are dependent on metabolic activity.  Drug metab. contributes substantially to interindividual differences in drug response and is also often involved in drug interactions, resulting in either therapeutic failure or adverse effects.  Knowledge about the metab. of a new chem. entity and its affinity to drug-metabolizing enzymes helps in the drug development process by providing important information for the selection of a lead compd. from among a no. of substances pharmacol. equally effective in their therapeutic response.  In drug development protocols, metab. characteristics should be assessed very early during the development process.  This was made possible by the advances made esp. in anal. capabilities and in in vitro technologies that are employed to predict in vivo metabolite profile, pharmacokinetic parameters and drug-drug interaction potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9DeBGTnGQWbVg90H21EOLACvtfcHk0lgS3EA7WMyRBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisFSmu70%253D&md5=41ac999c09cf85269e92c9c353a24e9b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-7843.2005.pto960305.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-7843.2005.pto960305.x%26sid%3Dliteratum%253Aachs%26aulast%3DPelkonen%26aufirst%3DO.%26aulast%3DTurpeinen%26aufirst%3DM.%26aulast%3DUusitalo%26aufirst%3DJ.%26aulast%3DRautio%26aufirst%3DA.%26aulast%3DRaunio%26aufirst%3DH.%26atitle%3DPrediction%2520of%2520drug%2520metabolism%2520and%2520interactions%2520on%2520the%2520basis%2520of%2520in%2520vitro%2520investigations%26jtitle%3DBasic%2520Clin.%2520Pharmacol.%2520Toxicol.%26date%3D2005%26volume%3D96%26spage%3D167%26epage%3D175%26doi%3D10.1111%2Fj.1742-7843.2005.pto960305.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Plant, N.</span><span> </span><span class="NLM_article-title">Strategies for using <i>in vitro</i> screens in drug metabolism</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">328</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1016/S1359-6446(03)03019-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2FS1359-6446%2803%2903019-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=328-336&author=N.+Plant&title=Strategies+for+using+in+vitro+screens+in+drug+metabolism&doi=10.1016%2FS1359-6446%2803%2903019-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2903019-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252903019-8%26sid%3Dliteratum%253Aachs%26aulast%3DPlant%26aufirst%3DN.%26atitle%3DStrategies%2520for%2520using%2520in%2520vitro%2520screens%2520in%2520drug%2520metabolism%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D328%26epage%3D336%26doi%3D10.1016%2FS1359-6446%2803%2903019-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Brandon, E. F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raap, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meijerman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beijnen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schellens, J. H. M.</span><span> </span><span class="NLM_article-title">An update on in vitro test methods in human hepatic drug biotransformation research: pros and cons</span> <span class="citation_source-journal">Toxicol. Appl. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">189</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1016/S0041-008X(03)00128-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2FS0041-008X%2803%2900128-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=12791308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1yjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2003&pages=233-246&author=E.+F.+A.+Brandonauthor=C.+D.+Raapauthor=I.+Meijermanauthor=J.+H.+Beijnenauthor=J.+H.+M.+Schellens&title=An+update+on+in+vitro+test+methods+in+human+hepatic+drug+biotransformation+research%3A+pros+and+cons&doi=10.1016%2FS0041-008X%2803%2900128-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">An update on in vitro test methods in human hepatic drug biotransformation research: pros and cons</span></div><div class="casAuthors">Brandon, Esther F. A.; Raap, Christiaan D.; Meijerman, Irma; Beijnen, Jos H.; Schellens, Jan H. M.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">233-246</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">A review.  The liver is the predominant organ in which biotransformation of foreign compds. takes place, although other organs may also be involved.  Ideally, an in vitro model for drug biotransformation by the liver should accurately resemble that in vivo.  Several in vitro human liver models have been developed in the past few decades, including supersomes, microsomes, cytosol, S9 fraction, cell lines, transgenic cell lines, primary hepatocytes, liver slices, and perfused liver.  A general advantage of these models is a reduced complexity of the system being studied.  On the other hand, there are several more or less serious specific drawbacks for each model, which prevents their widespread use and acceptance as an alternative for in vivo screening.  This review describes the practical aspects of selected in vitro human liver models, with comparisons between the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPmQxy6k-Zc7Vg90H21EOLACvtfcHk0ljNuxBZBr7NHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1yjsbg%253D&md5=8b2ee7b791ef35cc4e95fa754bda5cf4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0041-008X%2803%2900128-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0041-008X%252803%252900128-5%26sid%3Dliteratum%253Aachs%26aulast%3DBrandon%26aufirst%3DE.%2BF.%2BA.%26aulast%3DRaap%26aufirst%3DC.%2BD.%26aulast%3DMeijerman%26aufirst%3DI.%26aulast%3DBeijnen%26aufirst%3DJ.%2BH.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%2BM.%26atitle%3DAn%2520update%2520on%2520in%2520vitro%2520test%2520methods%2520in%2520human%2520hepatic%2520drug%2520biotransformation%2520research%253A%2520pros%2520and%2520cons%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2003%26volume%3D189%26spage%3D233%26epage%3D246%26doi%3D10.1016%2FS0041-008X%2803%2900128-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Baillie, T. A.</span><span> </span><span class="NLM_article-title">Metabolism and toxicity of drugs. Two decades of progress in industrial drug metabolism</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span><span class="refDoi"> DOI: 10.1021/tx7002273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx7002273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=129-137&author=T.+A.+Baillie&title=Metabolism+and+toxicity+of+drugs.+Two+decades+of+progress+in+industrial+drug+metabolism&doi=10.1021%2Ftx7002273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism and Toxicity of Drugs. Two Decades of Progress in Industrial Drug Metabolism</span></div><div class="casAuthors">Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-137</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The science of drug metab. and pharmacokinetics (DMPK) has developed significantly over the past 20 years, and its functional role in today's pharmaceutical industry has matured to the point where DMPK has become an indispensable discipline in support of drug discovery and development.  While contributions to the lead optimization phase of discovery efforts have been particularly noteworthy in helping to select only the best drug candidates for entry into development, it should be recognized that the scope of DMPK spans the continuum of discovery through clin. evaluation and even into the post-marketing phase; as such, the breadth of DMPK's involvement is almost unique in contemporary pharmaceutical research.  This perspective summarizes notable advances in the field, many of which have been made possible by technol. developments in areas such as mol. biol., genetics, and bioanal. chem., and highlights the crit. nature of key partnerships between Drug Metab., Medicinal Chem., and Safety Assessment groups in attempting to advance drug candidates with a low potential for causing adverse events in humans.  Finally, some speculative predictions are made of the future role of DMPK in pharmaceutical research, where current advances in our mechanistic understanding of the mol. processes that control the absorption, disposition, metab., elimination, and toxicity of drugs and their biotransformation products will combine to further enhance the impact of DMPK in drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUrTeLhTDYbVg90H21EOLACvtfcHk0ljNuxBZBr7NHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrzF&md5=e8f659251bf82cb0b1602855e725c390</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Ftx7002273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx7002273%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DMetabolism%2520and%2520toxicity%2520of%2520drugs.%2520Two%2520decades%2520of%2520progress%2520in%2520industrial%2520drug%2520metabolism%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D129%26epage%3D137%26doi%3D10.1021%2Ftx7002273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Obach, S. R.; Dalvie, D. K.; Walker, G. S.</span><span> </span><span class="NLM_article-title">Identification of drug metabolites</span>. In  <span class="citation_source-book">Encyclopedia of Drug Metabolism and Interactions</span>, <span class="NLM_edition">1</span>st ed.; <span class="NLM_contrib-group">Lyubimov, A.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470921920.edm052" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=1-55&author=S.+R.+Obach&author=D.+K.+Dalvie&author=G.+S.+Walkerauthor=A.+Lyubimov&title=Encyclopedia+of+Drug+Metabolism+and+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2F9780470921920.edm052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470921920.edm052%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DS.%2BR.%26atitle%3DIdentification%2520of%2520drug%2520metabolites%26btitle%3DEncyclopedia%2520of%2520Drug%2520Metabolism%2520and%2520Interactions%26aulast%3DLyubimov%26aufirst%3DA.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012%26spage%3D1%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Stepan, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascitti, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaumont, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalgutkar, A. S.</span><span> </span><span class="NLM_article-title">Metabolism-guided drug design</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">631</span><span class="refDoi"> DOI: 10.1039/c2md20317k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1039%2Fc2md20317k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkslWmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=631&author=A.+F.+Stepanauthor=V.+Mascittiauthor=K.+Beaumontauthor=A.+S.+Kalgutkar&title=Metabolism-guided+drug+design&doi=10.1039%2Fc2md20317k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism-guided drug design</span></div><div class="casAuthors">Stepan, Antonia F.; Mascitti, Vincent; Beaumont, Kevin; Kalgutkar, Amit S.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">631-652</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Preclin. drug metab. studies play a key role in the lead identification and optimization process in drug discovery.  Characterization of the metabolic pathways of new chem. entities is an integral part of drug discovery not only in optimizing clearance properties but also in eliminating potential safety concerns assocd. with the formation of protein and/or DNA-reactive metabolites.  Metab. studies in early discovery have been used to identify metabolic soft spots leading to high metabolic instability, and also in the characterization of active metabolites.  Availability of such information has aided in the rational design of compds. with increased resistance to metab. and overall improvements in oral pharmacokinetics and dose size.  Mechanistic drug metab. studies have proven particularly invaluable in mitigating reactive metabolite risks, which can lead to mutagenicity, time-dependent inactivation of cytochrome P 450 enzymes and/or idiosyncratic adverse drug reactions.  Characterization of stable conjugates derived from bioactivation of small mol. drug candidates provides indirect information on the structure of the reactive metabolite species, thereby providing insight into the bioactivation mechanism and hence a rationale on which to base subsequent chem. intervention strategies.  This review will showcase case studies of metab.-guided drug design using literature and inhouse examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcMOKNkXIUL7Vg90H21EOLACvtfcHk0ljNuxBZBr7NHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkslWmur0%253D&md5=61c667aa6f5c24de3fe110f384f323bd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1039%2Fc2md20317k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2md20317k%26sid%3Dliteratum%253Aachs%26aulast%3DStepan%26aufirst%3DA.%2BF.%26aulast%3DMascitti%26aufirst%3DV.%26aulast%3DBeaumont%26aufirst%3DK.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26atitle%3DMetabolism-guided%2520drug%2520design%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D631%26doi%3D10.1039%2Fc2md20317k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Fura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, Y.-Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roongta, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphrey, W. G.</span><span> </span><span class="NLM_article-title">Discovering drugs through biological transformation: role of pharmacologically active metabolites in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4340</span><span class="NLM_x">–</span> <span class="NLM_lpage">4351</span><span class="refDoi"> DOI: 10.1021/jm040066v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040066v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4340-4351&author=A.+Furaauthor=Y.-Z.+Shuauthor=M.+Zhuauthor=R.+L.+Hansonauthor=V.+Roongtaauthor=W.+G.+Humphrey&title=Discovering+drugs+through+biological+transformation%3A+role+of+pharmacologically+active+metabolites+in+drug+discovery&doi=10.1021%2Fjm040066v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm040066v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040066v%26sid%3Dliteratum%253Aachs%26aulast%3DFura%26aufirst%3DA.%26aulast%3DShu%26aufirst%3DY.-Z.%26aulast%3DZhu%26aufirst%3DM.%26aulast%3DHanson%26aufirst%3DR.%2BL.%26aulast%3DRoongta%26aufirst%3DV.%26aulast%3DHumphrey%26aufirst%3DW.%2BG.%26atitle%3DDiscovering%2520drugs%2520through%2520biological%2520transformation%253A%2520role%2520of%2520pharmacologically%2520active%2520metabolites%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4340%26epage%3D4351%26doi%3D10.1021%2Fjm040066v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Nassar, A. F.</span><span> </span><span class="NLM_article-title">Improving drug design: considerations for the structural modification process</span>. In  <span class="citation_source-book">Biotransformation and Metabolite Elucidation of Xenobiotics</span>; <span class="NLM_contrib-group">Nassar, A. F.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470890387.ch4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=163-216&author=A.+F.+Nassarauthor=A.+F.+Nassar&title=Biotransformation+and+Metabolite+Elucidation+of+Xenobiotics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2F9780470890387.ch4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470890387.ch4%26sid%3Dliteratum%253Aachs%26aulast%3DNassar%26aufirst%3DA.%2BF.%26atitle%3DImproving%2520drug%2520design%253A%2520considerations%2520for%2520the%2520structural%2520modification%2520process%26btitle%3DBiotransformation%2520and%2520Metabolite%2520Elucidation%2520of%2520Xenobiotics%26aulast%3DNassar%26aufirst%3DA.%2BF.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2010%26spage%3D163%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Clarke, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rindgen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korfmacher, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, K. A.</span><span> </span><span class="NLM_article-title">Systematic LC/MS metabolite identification in drug discovery</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">430A</span><span class="NLM_x">–</span> <span class="NLM_lpage">439A</span><span class="refDoi"> DOI: 10.1021/ac012480y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac012480y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=430A-439A&author=N.+J.+Clarkeauthor=D.+Rindgenauthor=W.+A.+Korfmacherauthor=K.+A.+Cox&title=Systematic+LC%2FMS+metabolite+identification+in+drug+discovery&doi=10.1021%2Fac012480y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fac012480y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac012480y%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DN.%2BJ.%26aulast%3DRindgen%26aufirst%3DD.%26aulast%3DKorfmacher%26aufirst%3DW.%2BA.%26aulast%3DCox%26aufirst%3DK.%2BA.%26atitle%3DSystematic%2520LC%252FMS%2520metabolite%2520identification%2520in%2520drug%2520discovery%26jtitle%3DAnal.%2520Chem.%26date%3D2001%26volume%3D73%26spage%3D430A%26epage%3D439A%26doi%3D10.1021%2Fac012480y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Kamel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prakash, C.</span><span> </span><span class="NLM_article-title">High performance liquid chromatography/atmospheric pressure ionization /tandem mass spectrometry (HPLC/API/MS/MS) in drug metabolism and toxicology studies</span> <span class="citation_source-journal">Curr. Drug Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">837</span><span class="NLM_x">–</span> <span class="NLM_lpage">852</span><span class="refDoi"> DOI: 10.2174/138920006779010593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.2174%2F138920006779010593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=17168686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFynu7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=837-852&author=A.+Kamelauthor=C.+Prakash&title=High+performance+liquid+chromatography%2Fatmospheric+pressure+ionization+%2Ftandem+mass+spectrometry+%28HPLC%2FAPI%2FMS%2FMS%29+in+drug+metabolism+and+toxicology+studies&doi=10.2174%2F138920006779010593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">High performance liquid chromatography/atmospheric pressure ionization/tandem mass spectrometry (HPLC/API/MS/MS) in drug metabolism and toxicology</span></div><div class="casAuthors">Kamel, Amin; Prakash, Chandra</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">837-852</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Studies of the metabolic fate of drugs and other xenobiotics in living systems may be divided into three broad areas: (1) elucidation of biotransformation pathways through identification of circulatory and excretory metabolites (qual. studies); (2) detn. of pharmacokinetics of the parent drug and/or its primary metabolites (quant. studies); and (3) identification of chem.-reactive metabolites, which play a key role as mediators of drug-induced toxicities (mechanistic studies).  Mass spectrometry has been regarded as one of the most important anal. tools in studies of drug metab., pharmacokinetics and biochem. toxicol.  With the com. introduction of new ionization methods such as those based on atm. pressure ionization (API) techniques and the combination of liq. chromatog.-mass spectrometry (LC-MS), it has now become a truly indispensable technique in pharmaceutical research Triple stage quadrupole and ion trap mass spectrometers are presently used for this purpose, because of their sensitivity and selectivity.  API-TOF mass spectrometry has also been very attractive due to its enhanced full-scan sensitivity, scan speed, improved resoln. and ability to measure the accurate masses for protonated mols. and fragment ions.  This review aims to survey the utility of mass spectrometry in drug metab. and toxicol. and to highlight novel applications and future trends in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8gg_jzxAq8rVg90H21EOLACvtfcHk0lg5dlQWkxrYCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFynu7fF&md5=c8955c8b5576387dc37e6b1f4b71795f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F138920006779010593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920006779010593%26sid%3Dliteratum%253Aachs%26aulast%3DKamel%26aufirst%3DA.%26aulast%3DPrakash%26aufirst%3DC.%26atitle%3DHigh%2520performance%2520liquid%2520chromatography%252Fatmospheric%2520pressure%2520ionization%2520%252Ftandem%2520mass%2520spectrometry%2520%2528HPLC%252FAPI%252FMS%252FMS%2529%2520in%2520drug%2520metabolism%2520and%2520toxicology%2520studies%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2006%26volume%3D7%26spage%3D837%26epage%3D852%26doi%3D10.2174%2F138920006779010593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Castro-Perez, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plumb, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granger, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beattie, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joncour, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, A.</span><span> </span><span class="NLM_article-title">Increasing throughput and information content for in vitro drug metabolism experiments using ultra-performance liquid chromatography coupled to a quadrupole time-offlight mass spectrometer</span> <span class="citation_source-journal">Rapid Commun. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">843</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1002/rcm.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2Frcm.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=15723446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitl2ktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=843-848&author=J.+M.+Castro-Perezauthor=R.+Plumbauthor=J.+H.+Grangerauthor=I.+Beattieauthor=K.+Joncourauthor=A.+Wright&title=Increasing+throughput+and+information+content+for+in+vitro+drug+metabolism+experiments+using+ultra-performance+liquid+chromatography+coupled+to+a+quadrupole+time-offlight+mass+spectrometer&doi=10.1002%2Frcm.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing throughput and information content for in vitro drug metabolism experiments using ultra-performance liquid chromatography coupled to a quadrupole time-of-flight mass spectrometer</span></div><div class="casAuthors">Castro-Perez, Jose; Plumb, Robert; Granger, Jennifer H.; Beattie, Iain; Joncour, Karine; Wright, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Rapid Communications in Mass Spectrometry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">843-848</span>CODEN:
                <span class="NLM_cas:coden">RCMSEF</span>;
        ISSN:<span class="NLM_cas:issn">0951-4198</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The field of drug metab. has been revolutionized by liq. chromatog./mass spectrometry (LC/MS) applications with new technologies such as triple quadrupoles, ion traps and time-of-flight (ToF) instrumentation.  Over the years, these developments have often relied on the improvements to the mass spectrometer hardware and software, which has allowed users to benefit from lower levels of detection and ease-of-use.  One area in which the development pace has been slower is in high-performance liq. chromatog. (HPLC).  In the case of metabolite identification, where there are many challenges due to the complex nature of the biol. matrixes and the diversity of the metabolites produced, there is a need to obtain the most accurate data possible.  Reactive or toxic metabolites need to be detected and identified as early as possible in the drug discovery process, in order to reduce the very costly attrition of compds. in late-phase development.  High-resoln., exact mass measurement plays a very important role in metabolite identification because it allows the elimination of false positives and the detn. of non-trivial metabolites in a much faster throughput environment than any other std. current methodol. available to this field.  By improving the chromatog. resoln., increased peak capacity can be achieved with a redn. in the no. of co-eluting species leading to superior sepns.  The overall enhancement in the chromatog. resoln. and peak capacity is transferred into a net redn. in ion suppression leading to an improvement in the MS sensitivity.  To investigate this, a no. of in vitro samples were analyzed using an ultra-performance liq. chromatog. (UPLC) system, with columns packed with porous 1.7 μm particles, coupled to a hybrid quadrupole time-of-flight (ToF) mass spectrometer.  This technique showed very clear examples for fundamental gains in sensitivity, chromatog. resoln. and speed of anal., which are all important factors for the demands of today's HTS in discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogSBhg6yK0NrVg90H21EOLACvtfcHk0lg5dlQWkxrYCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitl2ktro%253D&md5=6531ea8e926eb1e25cc3ff538f067915</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Frcm.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frcm.1859%26sid%3Dliteratum%253Aachs%26aulast%3DCastro-Perez%26aufirst%3DJ.%2BM.%26aulast%3DPlumb%26aufirst%3DR.%26aulast%3DGranger%26aufirst%3DJ.%2BH.%26aulast%3DBeattie%26aufirst%3DI.%26aulast%3DJoncour%26aufirst%3DK.%26aulast%3DWright%26aufirst%3DA.%26atitle%3DIncreasing%2520throughput%2520and%2520information%2520content%2520for%2520in%2520vitro%2520drug%2520metabolism%2520experiments%2520using%2520ultra-performance%2520liquid%2520chromatography%2520coupled%2520to%2520a%2520quadrupole%2520time-offlight%2520mass%2520spectrometer%26jtitle%3DRapid%2520Commun.%2520Mass%2520Spectrom.%26date%3D2005%26volume%3D19%26spage%3D843%26epage%3D848%26doi%3D10.1002%2Frcm.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Dunn, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broadhurst, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kell, D.</span><span> </span><span class="NLM_article-title">Metabolic profiling of serum using ultra performance liquid chromatography and the LTQ-Orbitrap mass spectrometry system</span> <span class="citation_source-journal">J. Chromatogr. B: Anal. Technol. Biomed. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">871</span><span class="NLM_x">, </span> <span class="NLM_fpage">288</span><span class="NLM_x">–</span> <span class="NLM_lpage">298</span><span class="refDoi"> DOI: 10.1016/j.jchromb.2008.03.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.jchromb.2008.03.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=18420470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps12it78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=871&publication_year=2008&pages=288-298&author=W.+B.+Dunnauthor=D.+Broadhurstauthor=M.+Brownauthor=M.+Bakerauthor=C.+Redmanauthor=L.+Kennyauthor=D.+Kell&title=Metabolic+profiling+of+serum+using+ultra+performance+liquid+chromatography+and+the+LTQ-Orbitrap+mass+spectrometry+system&doi=10.1016%2Fj.jchromb.2008.03.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic profiling of serum using Ultra Performance Liquid Chromatography and the LTQ-Orbitrap mass spectrometry system</span></div><div class="casAuthors">Dunn, Warwick B.; Broadhurst, David; Brown, Marie; Baker, Philip N.; Redman, Christopher W. G.; Kenny, Louise C.; Kell, Douglas B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chromatography B: Analytical Technologies in the Biomedical and Life Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">871</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">288-298</span>CODEN:
                <span class="NLM_cas:coden">JCBAAI</span>;
        ISSN:<span class="NLM_cas:issn">1570-0232</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Advances in anal. instrumentation can provide significant advantages to the vol. and quality of biol. knowledge acquired in metabolomic investigations.  The interfacing of sub-2 μm liq. chromatog. (UPLC ACQUITY) and LTQ-Orbitrap mass spectrometry systems provides many theor. advantages.  The applicability of the interfaced systems was investigated using a simple 11-component metabolite mix and a complex mammalian biofluid, serum.  Metabolites were detected in the metabolite mix with signals that were linear with their concn. over 2.5-3.5 orders of magnitude, with correlation coeffs. greater than 0.993 and limits of detection less than 1 μmol L-1.  Reproducibility of retention time (RSD < 3%) and chromatog. peak area (RSD < 15%) and a high mass accuracy (<2 ppm) were obsd. for 14 QC serum samples interspersed with other serum samples, analyzed over a period of 40 h.  The evaluation of a single deconvolution software package (XCMS) was performed and showed that two parameters (snthresh and bw) provided significant changes to the no. of peaks detected and the peak area reproducibility for the dataset used.  The data were used to indicate possible biomarkers of pre-eclampsia and showed both the instruments and XCMS to be applicable to the reproducible and valid detection of disease biomarkers present in serum.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIk9ttSrvmMrVg90H21EOLACvtfcHk0lhX-16ku8ulyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps12it78%253D&md5=21274b1752fadea5afe87641c880f145</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.jchromb.2008.03.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jchromb.2008.03.021%26sid%3Dliteratum%253Aachs%26aulast%3DDunn%26aufirst%3DW.%2BB.%26aulast%3DBroadhurst%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DBaker%26aufirst%3DM.%26aulast%3DRedman%26aufirst%3DC.%26aulast%3DKenny%26aufirst%3DL.%26aulast%3DKell%26aufirst%3DD.%26atitle%3DMetabolic%2520profiling%2520of%2520serum%2520using%2520ultra%2520performance%2520liquid%2520chromatography%2520and%2520the%2520LTQ-Orbitrap%2520mass%2520spectrometry%2520system%26jtitle%3DJ.%2520Chromatogr.%2520B%253A%2520Anal.%2520Technol.%2520Biomed.%2520Life%2520Sci.%26date%3D2008%26volume%3D871%26spage%3D288%26epage%3D298%26doi%3D10.1016%2Fj.jchromb.2008.03.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Bowen, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Northen, T. R.</span><span> </span><span class="NLM_article-title">Dealing with the unknown: metabolomics and metabolite atlases</span> <span class="citation_source-journal">J. Am. Soc. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1471</span><span class="NLM_x">–</span> <span class="NLM_lpage">1476</span><span class="refDoi"> DOI: 10.1016/j.jasms.2010.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.jasms.2010.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=20452782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKiur3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=1471-1476&author=B.+P.+Bowenauthor=T.+R.+Northen&title=Dealing+with+the+unknown%3A+metabolomics+and+metabolite+atlases&doi=10.1016%2Fj.jasms.2010.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Dealing with the Unknown: Metabolomics and Metabolite Atlases</span></div><div class="casAuthors">Bowen, Benjamin P.; Northen, Trent R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1471-1476</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Metabolomics is the comprehensive profiling of the small mol. compn. of a biol. sample.  Since metabolites are often the indirect products of gene expression, this approach is being used to provide new insights into a variety of biol. systems (clin., bioenergy, etc.).  A grand challenge for metabolomics is the complexity of the data, which often include many exptl. artifacts.  This is compounded by the tremendous chem. diversity of metabolites.  Identification of each uncharacterized metabolite is in many ways its own puzzle (compared with proteomics, which is based on predictable fragmentation patterns of polypeptides).  Therefore, effective data redn./prioritization strategies are crit. for this rapidly developing field.  Here the authors review liq. chromatog. electrospray ionization mass spectrometry (LC/MS)-based metabolomics, methods for feature finding/prioritization, approaches for identifying unknown metabolites, and construction of method specific Metabolite Atlases'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWvYdYDCQuzrVg90H21EOLACvtfcHk0lhX-16ku8ulyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKiur3N&md5=d1451b9bc1b1f911950d1c95b67262d5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.jasms.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jasms.2010.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DBowen%26aufirst%3DB.%2BP.%26aulast%3DNorthen%26aufirst%3DT.%2BR.%26atitle%3DDealing%2520with%2520the%2520unknown%253A%2520metabolomics%2520and%2520metabolite%2520atlases%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D2010%26volume%3D21%26spage%3D1471%26epage%3D1476%26doi%3D10.1016%2Fj.jasms.2010.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Kind, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiehn, O.</span><span> </span><span class="NLM_article-title">Advances in structure elucidation of small molecules using mass spectrometry</span> <span class="citation_source-journal">Bioanal. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1007/s12566-010-0015-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1007%2Fs12566-010-0015-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=21289855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A280%3ADC%252BC2sritlCitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=23-60&author=T.+Kindauthor=O.+Fiehn&title=Advances+in+structure+elucidation+of+small+molecules+using+mass+spectrometry&doi=10.1007%2Fs12566-010-0015-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in structure elucidation of small molecules using mass spectrometry</span></div><div class="casAuthors">Kind Tobias; Fiehn Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Bioanalytical reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1-4</span>),
    <span class="NLM_cas:pages">23-60</span>
        ISSN:<span class="NLM_cas:issn">1867-2086</span>.
    </div><div class="casAbstract">The structural elucidation of small molecules using mass spectrometry plays an important role in modern life sciences and bioanalytical approaches.  This review covers different soft and hard ionization techniques and figures of merit for modern mass spectrometers, such as mass resolving power, mass accuracy, isotopic abundance accuracy, accurate mass multiple-stage MS(n) capability, as well as hybrid mass spectrometric and orthogonal chromatographic approaches.  The latter part discusses mass spectral data handling strategies, which includes background and noise subtraction, adduct formation and detection, charge state determination, accurate mass measurements, elemental composition determinations, and complex data-dependent setups with ion maps and ion trees.  The importance of mass spectral library search algorithms for tandem mass spectra and multiple-stage MS(n) mass spectra as well as mass spectral tree libraries that combine multiple-stage mass spectra are outlined.  The successive chapter discusses mass spectral fragmentation pathways, biotransformation reactions and drug metabolism studies, the mass spectral simulation and generation of in silico mass spectra, expert systems for mass spectral interpretation, and the use of computational chemistry to explain gas-phase phenomena.  A single chapter discusses data handling for hyphenated approaches including mass spectral deconvolution for clean mass spectra, cheminformatics approaches and structure retention relationships, and retention index predictions for gas and liquid chromatography.  The last section reviews the current state of electronic data sharing of mass spectra and discusses the importance of software development for the advancement of structure elucidation of small molecules.  ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (doi:10.1007/s12566-010-0015-9) contains supplementary material, which is available to authorized users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUfAK_fKPrOuWhZTslLrfffW6udTcc2eZeI4gSu0KKBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sritlCitA%253D%253D&md5=91d949d5cc62a9874a98b52359342329</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs12566-010-0015-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12566-010-0015-9%26sid%3Dliteratum%253Aachs%26aulast%3DKind%26aufirst%3DT.%26aulast%3DFiehn%26aufirst%3DO.%26atitle%3DAdvances%2520in%2520structure%2520elucidation%2520of%2520small%2520molecules%2520using%2520mass%2520spectrometry%26jtitle%3DBioanal.%2520Rev.%26date%3D2010%26volume%3D2%26spage%3D23%26epage%3D60%26doi%3D10.1007%2Fs12566-010-0015-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Mortishire-Smith, R. J.; Zhang, H.; Bateman, K. P.</span><span> </span><span class="NLM_article-title">High-resolution mass spectrometry and drug metabolite identification</span>. In  <span class="citation_source-book">Mass Spectrometry in Drug Metabolism and Disposition</span>; <span class="NLM_contrib-group">Lee, M. S.; Zhu, M.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">407</span><span class="NLM_x">–</span> <span class="NLM_lpage">448</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470929278.ch13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=407-448&author=R.+J.+Mortishire-Smith&author=H.+Zhang&author=K.+P.+Batemanauthor=M.+S.+Lee&author=M.+Zhu&title=Mass+Spectrometry+in+Drug+Metabolism+and+Disposition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2F9780470929278.ch13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470929278.ch13%26sid%3Dliteratum%253Aachs%26aulast%3DMortishire-Smith%26aufirst%3DR.%2BJ.%26atitle%3DHigh-resolution%2520mass%2520spectrometry%2520and%2520drug%2520metabolite%2520identification%26btitle%3DMass%2520Spectrometry%2520in%2520Drug%2520Metabolism%2520and%2520Disposition%26aulast%3DLee%26aufirst%3DM.%2BS.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2011%26spage%3D407%26epage%3D448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Penner, N.; Zgoda-Pols, J.; Prakash, C.</span><span> </span><span class="NLM_article-title">Metabolite identification using mass spectrometry in drug development</span>. In  <span class="citation_source-book">Mass Spectrometry for Drug Discovery and Drug Development</span>; <span class="NLM_contrib-group">Korfmacher, W. A.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9781118516157.ch5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=115-147&author=N.+Penner&author=J.+Zgoda-Pols&author=C.+Prakashauthor=W.+A.+Korfmacher&title=Mass+Spectrometry+for+Drug+Discovery+and+Drug+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2F9781118516157.ch5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118516157.ch5%26sid%3Dliteratum%253Aachs%26aulast%3DPenner%26aufirst%3DN.%26atitle%3DMetabolite%2520identification%2520using%2520mass%2520spectrometry%2520in%2520drug%2520development%26btitle%3DMass%2520Spectrometry%2520for%2520Drug%2520Discovery%2520and%2520Drug%2520Development%26aulast%3DKorfmacher%26aufirst%3DW.%2BA.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2013%26spage%3D115%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Cusack, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koolman, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, U. E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peltier, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piel, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasudevan, A.</span><span> </span><span class="NLM_article-title">Emerging technologies for metabolite generation and structural diversification</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">5471</span><span class="NLM_x">–</span> <span class="NLM_lpage">5483</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2013.08.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=5471-5483&author=K.+P.+Cusackauthor=H.+F.+Koolmanauthor=U.+E.+W.+Langeauthor=H.+M.+Peltierauthor=I.+Pielauthor=A.+Vasudevan&title=Emerging+technologies+for+metabolite+generation+and+structural+diversification&doi=10.1016%2Fj.bmcl.2013.08.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DCusack%26aufirst%3DK.%2BP.%26aulast%3DKoolman%26aufirst%3DH.%2BF.%26aulast%3DLange%26aufirst%3DU.%2BE.%2BW.%26aulast%3DPeltier%26aufirst%3DH.%2BM.%26aulast%3DPiel%26aufirst%3DI.%26aulast%3DVasudevan%26aufirst%3DA.%26atitle%3DEmerging%2520technologies%2520for%2520metabolite%2520generation%2520and%2520structural%2520diversification%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D5471%26epage%3D5483%26doi%3D10.1016%2Fj.bmcl.2013.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">Small sample cryoprobe NMR applications</span>,  <span class="citation_source-book">eMagRes.</span>; <span class="NLM_contrib-group">Harris, R. K.; Wasylishen, R. A.</span>, Eds.; <span class="NLM_publisher-name">Wiley</span>, <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470034590.emrstm1300" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=G.+E.+Martinauthor=R.+K.+Harris&author=R.+A.+Wasylishen&title=eMagRes."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2F9780470034590.emrstm1300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470034590.emrstm1300%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DSmall%2520sample%2520cryoprobe%2520NMR%2520applications%26btitle%3DeMagRes.%26aulast%3DHarris%26aufirst%3DR.%2BK.%26pub%3DWiley%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Hilton, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">An investigation of experimental limits of small sample heteronuclear 2D NMR</span> <span class="citation_source-journal">J. Nat. Prod.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">1465</span><span class="NLM_x">–</span> <span class="NLM_lpage">1469</span><span class="refDoi"> DOI: 10.1021/np100481m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np100481m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2010&pages=1465-1469&author=B.+D.+Hiltonauthor=G.+E.+Martin&title=An+investigation+of+experimental+limits+of+small+sample+heteronuclear+2D+NMR&doi=10.1021%2Fnp100481m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fnp100481m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp100481m%26sid%3Dliteratum%253Aachs%26aulast%3DHilton%26aufirst%3DB.%2BD.%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DAn%2520investigation%2520of%2520experimental%2520limits%2520of%2520small%2520sample%2520heteronuclear%25202D%2520NMR%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2010%26volume%3D73%26spage%3D1465%26epage%3D1469%26doi%3D10.1021%2Fnp100481m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marques, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bermel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">Using pure shift HSQC to characterize microgram samples of drug metabolites</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5450</span><span class="NLM_x">–</span> <span class="NLM_lpage">5453</span><span class="refDoi"> DOI: 10.1016/j.tetlet.2014.06.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.tetlet.2014.06.067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=5450-5453&author=Y.+Liuauthor=M.+D.+Greenauthor=R.+Marquesauthor=T.+Pereiraauthor=R.+Helmyauthor=R.+T.+Williamsonauthor=W.+Bermelauthor=G.+E.+Martin&title=Using+pure+shift+HSQC+to+characterize+microgram+samples+of+drug+metabolites&doi=10.1016%2Fj.tetlet.2014.06.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2014.06.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2014.06.067%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGreen%26aufirst%3DM.%2BD.%26aulast%3DMarques%26aufirst%3DR.%26aulast%3DPereira%26aufirst%3DT.%26aulast%3DHelmy%26aufirst%3DR.%26aulast%3DWilliamson%26aufirst%3DR.%2BT.%26aulast%3DBermel%26aufirst%3DW.%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DUsing%2520pure%2520shift%2520HSQC%2520to%2520characterize%2520microgram%2520samples%2520of%2520drug%2520metabolites%26jtitle%3DTetrahedron%2520Lett.%26date%3D2014%26volume%3D55%26spage%3D5450%26epage%3D5453%26doi%3D10.1016%2Fj.tetlet.2014.06.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Annis, A.; Chuang, C.-C.; Nazef, N.</span><span> </span><span class="NLM_article-title">ALIS: An affinity selection-mass spectrometry system for the discovery and characterization of protein-ligand interactions</span>. In  <span class="citation_source-book">Mass Spectrometry in Medicinal Chemistry: Applications in Drug Discovery</span>; <span class="NLM_contrib-group">Wanner, K.; Höfner, G.</span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH Verlag GmbH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">36</span>, Chapter 3.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9783527610907.ch3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=A.+Annis&author=C.-C.+Chuang&author=N.+Nazefauthor=K.+Wanner&author=G.+H%C3%B6fner&title=Mass+Spectrometry+in+Medicinal+Chemistry%3A+Applications+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2F9783527610907.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527610907.ch3%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DA.%26atitle%3DALIS%253A%2520An%2520affinity%2520selection-mass%2520spectrometry%2520system%2520for%2520the%2520discovery%2520and%2520characterization%2520of%2520protein-ligand%2520interactions%26btitle%3DMass%2520Spectrometry%2520in%2520Medicinal%2520Chemistry%253A%2520Applications%2520in%2520Drug%2520Discovery%26aulast%3DWanner%26aufirst%3DK.%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26date%3D2007%26volume%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nickbarg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebell, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehurst, C. E.</span><span> </span><span class="NLM_article-title">Affinity selection-mass spectrometry screening techniques for small molecule drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">518</span><span class="NLM_x">–</span> <span class="NLM_lpage">526</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2007.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.cbpa.2007.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=17931956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=518-526&author=D.+A.+Annisauthor=E.+Nickbargauthor=X.+Yangauthor=M.+R.+Ziebellauthor=C.+E.+Whitehurst&title=Affinity+selection-mass+spectrometry+screening+techniques+for+small+molecule+drug+discovery&doi=10.1016%2Fj.cbpa.2007.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Affinity selection-mass spectrometry screening techniques for small molecule drug discovery</span></div><div class="casAuthors">Annis, D. Allen; Nickbarg, Elliot; Yang, Xianshu; Ziebell, Michael R.; Whitehurst, Charles E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">518-526</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Affinity selection-mass spectrometry (AS-MS) techniques assess the binding of candidate mols. to immobilized or sol. receptors, and these methods are gaining acceptance in high throughput screening labs. as valuable complements to traditional drug discovery technologies.  A diversity of receptor types have been evaluated by AS-MS, including those that are difficult to screen using traditional biochem. approaches.  AS-MS techniques that couple liq. chromatog.-MS with size-based sepn. methods, such as ultrafiltration, gel permeation, or size-exclusion chromatog., are particularly amenable to the demands of MS-based screening and have demonstrated the greatest success across a broad range of drug targets.  MS measurements of receptor function have many of the same advantages as AS-MS screening and are increasingly used for drug discovery as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBsk6pA3Gl9LVg90H21EOLACvtfcHk0lg8xW0HVajhrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsbrE&md5=8c98d65db2058eff028a0b879458bf5a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2007.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2007.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DNickbarg%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZiebell%26aufirst%3DM.%2BR.%26aulast%3DWhitehurst%26aufirst%3DC.%2BE.%26atitle%3DAffinity%2520selection-mass%2520spectrometry%2520screening%2520techniques%2520for%2520small%2520molecule%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26spage%3D518%26epage%3D526%26doi%3D10.1016%2Fj.cbpa.2007.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischmann, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duca, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span> </span><span class="NLM_article-title">Discovery of a novel series of CHK1 kinase inhibitors with a distinctive hinge binding mode</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1021/ml200249h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml200249h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC38XpslGntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=123-128&author=X.+Huangauthor=C.+C.+Chengauthor=T.+O.+Fischmannauthor=J.+S.+Ducaauthor=X.+Yangauthor=M.+Richardsauthor=G.+W.+Shipps&title=Discovery+of+a+novel+series+of+CHK1+kinase+inhibitors+with+a+distinctive+hinge+binding+mode&doi=10.1021%2Fml200249h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Series of CHK1 Kinase Inhibitors with a Distinctive Hinge Binding Mode</span></div><div class="casAuthors">Huang, Xiaohua; Cheng, Cliff C.; Fischmann, Thierry O.; Duca, Jose S.; Yang, Xianshu; Richards, Matthew; Shipps, Gerald W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">123-128</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of CHK1 inhibitors with a distinctive hinge binding mode, exemplified by 2-aryl-N-(2-(piperazin-1-yl)phenyl)thiazole-4-carboxamide, was discovered through high-throughput screening using the affinity selection-mass spectrometry (AS-MS)-based Automated Ligand Identification System (ALIS) platform.  Structure-based ligand design and optimization led to significant improvements in potency to the single digit nanomolar range and hundred-fold selectivity against CDK2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphbnP8TZLFsrVg90H21EOLACvtfcHk0lhBOHgCktJ4AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpslGntQ%253D%253D&md5=0033fbe5837cc4c6287c4a5428665247</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fml200249h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml200249h%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520series%2520of%2520CHK1%2520kinase%2520inhibitors%2520with%2520a%2520distinctive%2520hinge%2520binding%2520mode%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D123%26epage%3D128%26doi%3D10.1021%2Fml200249h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Whitehurst, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taremi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojcik, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strizki, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nickbarg, E.</span><span> </span><span class="NLM_article-title">Application of affinity selection-mass spectrometry assays to purification and affinity-based screening of the chemokine receptor CXCR4</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">478</span><span class="refDoi"> DOI: 10.2174/138620712800563945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.2174%2F138620712800563945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=22651846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFalt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2012&pages=473-478&author=C.+E.+Whitehurstauthor=Z.+Yaoauthor=D.+Murphyauthor=M.+Zhangauthor=S.+Taremiauthor=L.+Wojcikauthor=J.+M.+Strizkiauthor=J.+D.+Brackenauthor=C.+C.+Chengauthor=X.+Yangauthor=G.+W.+Shippsauthor=M.+Ziebellauthor=E.+Nickbarg&title=Application+of+affinity+selection-mass+spectrometry+assays+to+purification+and+affinity-based+screening+of+the+chemokine+receptor+CXCR4&doi=10.2174%2F138620712800563945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Application of affinity selection-mass spectrometry assays to purification and affinity-based screening of the chemokine receptor CXCR4</span></div><div class="casAuthors">Whitehurst, Charles E.; Yao, Zhiping; Murphy, Denise; Zhang, Mingxuan; Taremi, Shane; Wojcik, Lisa; Strizki, Julie M.; Bracken, Jack D.; Cheng, Cliff C.; Yang, Xianshu; Shipps, Gerald W., Jr.; Ziebell, Michael; Nickbarg, Elliott</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">473-485</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Affinity selection-mass spectrometry (AS-MS) is a sensitive technol. for identifying small mols. that bind to target proteins, and assays enabled by AS-MS can be used to delineate relative binding affinities of ligands for proteins.  "Indirect" AS-MS assays employ size-exclusion techniques to sep. target-ligand complexes from unbound ligands, and target-assocd. ligands are then specifically detected by liq. chromatog. mass spectrometry.  We report how indirect AS-MS binding assays with known ref. control compds. were used as guideposts for development of an optimized purifn. method for CXCR4, a G-protein coupled chemokine receptor, for which we sought novel antagonists.  The CXCR4 purifn. method that was developed was amenable to scale-up and enabled the screening of purified recombinant human CXCR4 against a large combinatorial library of small mols. by high throughput indirect AS-MS.  The screen resulted in the discovery of new ligands that competed off binding of ref. compds. to CXCR4 in AS-MS binding assays and that antagonized SDF1α-triggered responses and CXCR4-mediated HIV1 viral uptake in cell-based assays.  This report provides a methodol. paradigm whereby indirect AS-MS-based ligand binding assays may be used to guide optimal integral membrane protein purifn. methods that enable downstream affinity selection-based applications such as high throughput AS-MS screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnKOiSohXGXLVg90H21EOLACvtfcHk0lhBOHgCktJ4AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFalt7o%253D&md5=cba5688a5bad20d0cfca1119e8b95fcb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.2174%2F138620712800563945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620712800563945%26sid%3Dliteratum%253Aachs%26aulast%3DWhitehurst%26aufirst%3DC.%2BE.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DMurphy%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DTaremi%26aufirst%3DS.%26aulast%3DWojcik%26aufirst%3DL.%26aulast%3DStrizki%26aufirst%3DJ.%2BM.%26aulast%3DBracken%26aufirst%3DJ.%2BD.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DZiebell%26aufirst%3DM.%26aulast%3DNickbarg%26aufirst%3DE.%26atitle%3DApplication%2520of%2520affinity%2520selection-mass%2520spectrometry%2520assays%2520to%2520purification%2520and%2520affinity-based%2520screening%2520of%2520the%2520chemokine%2520receptor%2520CXCR4%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2012%26volume%3D15%26spage%3D473%26epage%3D478%26doi%3D10.2174%2F138620712800563945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovici-Muller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqui, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gowen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Windsor, W. T.</span><span> </span><span class="NLM_article-title">Method for quantitative protein-ligand affinity measurements in compound mixtures</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">4538</span><span class="NLM_x">–</span> <span class="NLM_lpage">4542</span><span class="refDoi"> DOI: 10.1021/ac0702701</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac0702701" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1WhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2007&pages=4538-4542&author=D.+A.+Annisauthor=G.+W.+Shippsauthor=Y.+Dengauthor=J.+Popovici-Mullerauthor=M.+A.+Siddiquiauthor=P.+J.+Curranauthor=M.+Gowenauthor=W.+T.+Windsor&title=Method+for+quantitative+protein-ligand+affinity+measurements+in+compound+mixtures&doi=10.1021%2Fac0702701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Method for quantitative protein-ligand affinity measurements in compound mixtures</span></div><div class="casAuthors">Annis, D. Allen; Shipps, Gerald W., Jr.; Deng, Yongqi; Popovici-Mueller, Janeta; Siddiqui, M. Arshad; Curran, Patrick J.; Gowen, Matthew; Windsor, William T.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4538-4542</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This manuscript describes an affinity selection-mass spectrometry (AS-MS) method for quant. protein-ligand binding affinity (Kd) measurements in large compd. libraries.  The ability of a titrant ligand to displace a target-bound library member-as measured by MS-reveals the affinity ranking of the mixt. component relative to "internal affinity calibrants", compds. of known affinity for the target.  This technique does not require that the precise concn. of each ligand is known; therefore, unpurified products of mixt.-based combinatorial synthesis may be used for affinity optimization and developing structure-activity relationships.  The method is demonstrated for a series of ligands to the important oncol. target CDK2 that were discovered by AS-MS screening of combinatorial libraries against the basal form of the protein.  AS-MS displacement curves for select hits were acquired over a range of compd. concns., confirming that binding affinity measurement results are concn.-insensitive.  These hits were evaluated in pools of purified compds. to verify the method's applicability to hit triage in large chem. libraries.  The method was further tested using unpurified, mixt.-based combinatorial libraries of >1000 compds., yielding results that mirror those obtained from mixts. of purified compds.  The technique was then used to identify optimized CDK2 ligands from compd. mixts., quant. measure their affinities, and establish structure-activity relationships among these drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopGVdmMNSdmbVg90H21EOLACvtfcHk0lhBOHgCktJ4AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1WhtLk%253D&md5=638ab38a5bc9674a4ccdb1977c721b87</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fac0702701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac0702701%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DPopovici-Muller%26aufirst%3DJ.%26aulast%3DSiddiqui%26aufirst%3DM.%2BA.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DGowen%26aufirst%3DM.%26aulast%3DWindsor%26aufirst%3DW.%2BT.%26atitle%3DMethod%2520for%2520quantitative%2520protein-ligand%2520affinity%2520measurements%2520in%2520compound%2520mixtures%26jtitle%3DAnal.%2520Chem.%26date%3D2007%26volume%3D79%26spage%3D4538%26epage%3D4542%26doi%3D10.1021%2Fac0702701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Deng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">English, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dayananth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruza, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirschmeier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Windsor, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samater, A. A.</span><span> </span><span class="NLM_article-title">Discovery of novel, dual mechanism ERK inhibitors by affinity selection screening of an inactive kinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8817</span><span class="NLM_x">–</span> <span class="NLM_lpage">8826</span><span class="refDoi"> DOI: 10.1021/jm500847m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500847m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCht77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8817-8826&author=D.+Dengauthor=G.+W.+Shippsauthor=A.+Cooperauthor=J.+M.+Englishauthor=D.+A.+Annisauthor=D.+Carrauthor=Y.+Nanauthor=T.+Wangauthor=H.+Y.+Zhuauthor=C.+C.+Chengauthor=P.+Dayananthauthor=A.+W.+Hruzaauthor=J.+Xiaoauthor=W.+Jinauthor=P.+Kirschmeierauthor=W.+T.+Windsorauthor=A.+A.+Samater&title=Discovery+of+novel%2C+dual+mechanism+ERK+inhibitors+by+affinity+selection+screening+of+an+inactive+kinase&doi=10.1021%2Fjm500847m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel, Dual Mechanism ERK Inhibitors by Affinity Selection Screening of an Inactive Kinase</span></div><div class="casAuthors">Deng, Yongqi; Shipps, Gerald W.; Cooper, Alan; English, Jessie M.; Annis, D. Allen; Carr, Donna; Nan, Yang; Wang, Tong; Zhu, Hugh Y.; Chuang, Cheng-Chi; Dayananth, Priya; Hruza, Alan W.; Xiao, Li; Jin, Weihong; Kirschmeier, Paul; Windsor, William T.; Samatar, Ahmed A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8817-8826</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An affinity-based mass spectrometry screening technol. was used to identify novel binders to both nonphosphorylated and phosphorylated ERK2.  Screening of inactive ERK2 identified a pyrrolidine analog 1 that bound to both nonphosphorylated and phosphorylated ERK2 and inhibited ERK2 kinase activity.  Chem. optimization identified compd. 4 as a novel, potent, and highly selective ERK1,2 inhibitor which not only demonstrated inhibition of phosphorylation of ERK substrate p90RSK but also demonstrated inhibition of ERK1,2 phosphorylation on the activation loop.  X-ray cocrystallog. revealed that upon binding of compd. 4 to ERK2, Tyr34 undergoes a rotation (flip) along with a shift in the poly-Gly rich loop to create a new binding pocket into which 4 can bind.  This new binding mode represents a novel mechanism by which high affinity ATP-competitive compds. may achieve excellent kinase selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2kaW1Z2DFXbVg90H21EOLACvtfcHk0lg3uUAOUpj3vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCht77P&md5=10f1922c9c65371564ed0ad1f05cc845</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm500847m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500847m%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DD.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DCooper%26aufirst%3DA.%26aulast%3DEnglish%26aufirst%3DJ.%2BM.%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DCarr%26aufirst%3DD.%26aulast%3DNan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DH.%2BY.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DDayananth%26aufirst%3DP.%26aulast%3DHruza%26aufirst%3DA.%2BW.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DW.%26aulast%3DKirschmeier%26aufirst%3DP.%26aulast%3DWindsor%26aufirst%3DW.%2BT.%26aulast%3DSamater%26aufirst%3DA.%2BA.%26atitle%3DDiscovery%2520of%2520novel%252C%2520dual%2520mechanism%2520ERK%2520inhibitors%2520by%2520affinity%2520selection%2520screening%2520of%2520an%2520inactive%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8817%26epage%3D8826%26doi%3D10.1021%2Fjm500847m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Parker, D. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharipour, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewnani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherer, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichert, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geissler, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonatore, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellsworth, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balkovec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenlee, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, H. B.</span><span> </span><span class="NLM_article-title">Rapid development of two factor IXa inhibitors from hit to lead</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2321</span><span class="NLM_x">–</span> <span class="NLM_lpage">2325</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2015.04.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2321-2325&author=D.+L.+Parkerauthor=S.+Walshauthor=B.+Liauthor=E.+Kimauthor=A.+Sharipourauthor=C.+Smithauthor=Y.-H.+Chenauthor=R.+Bergerauthor=B.+Harperauthor=T.+Zhangauthor=M.+Parkauthor=M.+Shuauthor=J.+Wuauthor=J.+Xuauthor=S.+Dewnaniauthor=E.+C.+Shererauthor=A.+Hruzaauthor=P.+Reichertauthor=W.+Geisslerauthor=L.+Sonatoreauthor=K.+Ellsworthauthor=J.+Balkovecauthor=W.+Greenleeauthor=H.+B.+Wood&title=Rapid+development+of+two+factor+IXa+inhibitors+from+hit+to+lead&doi=10.1016%2Fj.bmcl.2015.04.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DD.%2BL.%26aulast%3DWalsh%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DE.%26aulast%3DSharipour%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.-H.%26aulast%3DBerger%26aufirst%3DR.%26aulast%3DHarper%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DM.%26aulast%3DShu%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDewnani%26aufirst%3DS.%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DReichert%26aufirst%3DP.%26aulast%3DGeissler%26aufirst%3DW.%26aulast%3DSonatore%26aufirst%3DL.%26aulast%3DEllsworth%26aufirst%3DK.%26aulast%3DBalkovec%26aufirst%3DJ.%26aulast%3DGreenlee%26aufirst%3DW.%26aulast%3DWood%26aufirst%3DH.%2BB.%26atitle%3DRapid%2520development%2520of%2520two%2520factor%2520IXa%2520inhibitors%2520from%2520hit%2520to%2520lead%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2321%26epage%3D2325%26doi%3D10.1016%2Fj.bmcl.2015.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andre, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellsworth, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geissler, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, D. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichert, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherer, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonatore, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tschirret-Guth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nolting, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orr, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campeau, L.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Araki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurada, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, H. B.</span><span> </span><span class="NLM_article-title">Development of a novel class of potent and selective FIXa inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">4945</span><span class="NLM_x">–</span> <span class="NLM_lpage">4949</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.04.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2015.04.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=25978966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFygs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=4945-4949&author=T.+Zhangauthor=P.+Andreauthor=T.+J.+Batemanauthor=Y.-H.+Chenauthor=K.+Desaiauthor=K.+Ellsworthauthor=W.+M.+Geisslerauthor=L.+Guoauthor=A.+Hruzaauthor=T.+Jianauthor=D.+Mengauthor=D.+L.+Parkerauthor=X.+Qianauthor=P.+Reichertauthor=E.+C.+Shererauthor=M.+Shuauthor=C.+J.+Smithauthor=L.+M.+Sonatoreauthor=R.+Tschirret-Guthauthor=A.+F.+Noltingauthor=R.+Orrauthor=L.-C.+Campeauauthor=K.+Arakiauthor=T.+Nishimuraauthor=I.+Sakuradaauthor=H.+B.+Wood&title=Development+of+a+novel+class+of+potent+and+selective+FIXa+inhibitors&doi=10.1016%2Fj.bmcl.2015.04.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a novel class of potent and selective FIXa inhibitors</span></div><div class="casAuthors">Zhang, Ting; Andre, Patrick; Bateman, Thomas J.; Chen, Yi-Heng; Desai, Kunal; Ellsworth, Kenneth; Geissler, Wayne M.; Guo, Liangqin; Hruza, Alan; Jian, Tianying; Meng, Dongfang; Parker, Dann L., Jr.; Qian, Xiaoxia; Reichert, Paul; Sherer, Edward C.; Shu, Min; Smith, Cameron J.; Sonatore, Lisa M.; Tschirret-Guth, Richard; Nolting, Andrew F.; Orr, Robert; Campeau, Louis-Charles; Araki, Kazuto; Nishimura, Teruyuki; Sakurada, Isao; Wood, Harold B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4945-4949</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Using structure based drug design (SBDD), a novel class of potent coagulation Factor IXa (FIXa) inhibitors was designed and synthesized.  High selectivity over FXa inhibition was achieved.  Selected compds. demonstrated oral bioavailability in rat IV/PO pharmacokinetic (PK) studies.  Finally, the pharmacodynamics (PD) of this class of mols. was evaluated in Thrombin Generation Assay (TGA) in Corn Trypsin Inhibitor (CTI) citrated human plasma and demonstrated characteristics of a FIXa inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW3NBfPoV5zrVg90H21EOLACvtfcHk0lg3uUAOUpj3vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFygs78%253D&md5=d4944b0798611a811a5ee7d4d7282b5f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.057%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DAndre%26aufirst%3DP.%26aulast%3DBateman%26aufirst%3DT.%2BJ.%26aulast%3DChen%26aufirst%3DY.-H.%26aulast%3DDesai%26aufirst%3DK.%26aulast%3DEllsworth%26aufirst%3DK.%26aulast%3DGeissler%26aufirst%3DW.%2BM.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DJian%26aufirst%3DT.%26aulast%3DMeng%26aufirst%3DD.%26aulast%3DParker%26aufirst%3DD.%2BL.%26aulast%3DQian%26aufirst%3DX.%26aulast%3DReichert%26aufirst%3DP.%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DShu%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DC.%2BJ.%26aulast%3DSonatore%26aufirst%3DL.%2BM.%26aulast%3DTschirret-Guth%26aufirst%3DR.%26aulast%3DNolting%26aufirst%3DA.%2BF.%26aulast%3DOrr%26aufirst%3DR.%26aulast%3DCampeau%26aufirst%3DL.-C.%26aulast%3DAraki%26aufirst%3DK.%26aulast%3DNishimura%26aufirst%3DT.%26aulast%3DSakurada%26aufirst%3DI.%26aulast%3DWood%26aufirst%3DH.%2BB.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520class%2520of%2520potent%2520and%2520selective%2520FIXa%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D4945%26epage%3D4949%26doi%3D10.1016%2Fj.bmcl.2015.04.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">You, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verkade, J. G.</span><span> </span><span class="NLM_article-title">P(<i>i</i>-BuNCH<sub>2</sub>CH<sub>2</sub>)<sub>3</sub>N: An efficient ligand for the direct α-arylation of nitriles with aryl bromides</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">8003</span><span class="NLM_x">–</span> <span class="NLM_lpage">8007</span><span class="refDoi"> DOI: 10.1021/jo034779h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo034779h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnsFyrtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=8003-8007&author=J.+Youauthor=J.+G.+Verkade&title=P%28i-BuNCH2CH2%293N%3A+An+efficient+ligand+for+the+direct+%CE%B1-arylation+of+nitriles+with+aryl+bromides&doi=10.1021%2Fjo034779h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">P(i-BuNCH2CH2)3N: An Efficient Ligand for the Direct α-Arylation of Nitriles with Aryl Bromides</span></div><div class="casAuthors">You, Jingsong; Verkade, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8003-8007</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new catalyst system for the synthesis of α-aryl-substituted nitriles is reported.  The bicyclic triaminophosphine P(i-BuNCH2CH2)3N (I) serves as an efficient and versatile ligand for the palladium-catalyzed direct α-arylation of nitriles with aryl bromides.  Using ligand I, Et cyanoacetate and primary as well as secondary nitriles are efficiently coupled with a wide variety of aryl bromides possessing electron-rich, electron-poor, electron-neutral, and sterically hindered groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGgTlIYPn6nrVg90H21EOLACvtfcHk0ljLE2QY_4mfyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnsFyrtbg%253D&md5=2fd29d6ebbab1eb2130447aa7ce7eaf7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjo034779h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo034779h%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DVerkade%26aufirst%3DJ.%2BG.%26atitle%3DP%2528i-BuNCH2CH2%25293N%253A%2520An%2520efficient%2520ligand%2520for%2520the%2520direct%2520%25CE%25B1-arylation%2520of%2520nitriles%2520with%2520aryl%2520bromides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2003%26volume%3D68%26spage%3D8003%26epage%3D8007%26doi%3D10.1021%2Fjo034779h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Khurana, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kukreja, G.</span><span> </span><span class="NLM_article-title">Reduction of nitriles to primary amines with nickel boride at ambient temperature</span> <span class="citation_source-journal">Synth. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">1265</span><span class="NLM_x">–</span> <span class="NLM_lpage">1269</span><span class="refDoi"> DOI: 10.1081/SCC-120003619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1081%2FSCC-120003619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvV2msb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2002&pages=1265-1269&author=J.+M.+Khuranaauthor=G.+Kukreja&title=Reduction+of+nitriles+to+primary+amines+with+nickel+boride+at+ambient+temperature&doi=10.1081%2FSCC-120003619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid reduction of nitriles to primary amines with nickel boride at ambient temperature</span></div><div class="casAuthors">Khurana, Jitender M.; Kukreja, Gagan</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1265-1269</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">Redn. of a variety of nitriles to their corresponding primary amines can be achieved with Ni boride generated in situ in dry EtOH at ambient temp.  The redns. are very rapid and chemoselective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppr9uclHIZ-bVg90H21EOLACvtfcHk0ljLE2QY_4mfyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvV2msb8%253D&md5=cf018f6e95e8a32c66e8fbccb6372c52</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1081%2FSCC-120003619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1081%252FSCC-120003619%26sid%3Dliteratum%253Aachs%26aulast%3DKhurana%26aufirst%3DJ.%2BM.%26aulast%3DKukreja%26aufirst%3DG.%26atitle%3DReduction%2520of%2520nitriles%2520to%2520primary%2520amines%2520with%2520nickel%2520boride%2520at%2520ambient%2520temperature%26jtitle%3DSynth.%2520Commun.%26date%3D2002%26volume%3D32%26spage%3D1265%26epage%3D1269%26doi%3D10.1081%2FSCC-120003619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="note"><p class="first last"><i>In vitro</i> clearance data of <b>7</b>: human CLint (liver microsomes) = 29.8 mL/min/kg; human CLint (hepatocyte) = 21.0 mL/min/kg; rat CLint (liver microsomes) = 116 mL/min/kg; rat CLint (hepatocyte) = 101 mL/min/kg.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Athanasopoulos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felsch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalghatgi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orminati, J.-P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosner, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaddupathy, G. R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyler, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vilenchik, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wintner, E. A.</span><span> </span><span class="NLM_article-title">An affinity selection–mass spectrometry method for the identification of small molecule ligands from self-encoded combinatorial libraries: Discovery of a novel antagonist of <i>E. coli</i> dihydrofolate reductase</span> <span class="citation_source-journal">Int. J. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">238</span><span class="NLM_x">, </span> <span class="NLM_fpage">77</span><span class="NLM_x">–</span> <span class="NLM_lpage">83</span><span class="refDoi"> DOI: 10.1016/j.ijms.2003.11.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.ijms.2003.11.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpslyisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=238&publication_year=2004&pages=77-83&author=D.+A.+Annisauthor=J.+Athanasopoulosauthor=P.+J.+Curranauthor=J.+S.+Felschauthor=K.+Kalghatgiauthor=W.+H.+Leeauthor=H.+M.+Nashauthor=J.-P.+A.+Orminatiauthor=K.+E.+Rosnerauthor=G.+W.+Shippsauthor=G.+R.+A.+Thaddupathyauthor=A.+N.+Tylerauthor=L.+Vilenchikauthor=C.+R.+Wagnerauthor=E.+A.+Wintner&title=An+affinity+selection%E2%80%93mass+spectrometry+method+for+the+identification+of+small+molecule+ligands+from+self-encoded+combinatorial+libraries%3A+Discovery+of+a+novel+antagonist+of+E.+coli+dihydrofolate+reductase&doi=10.1016%2Fj.ijms.2003.11.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">An affinity selection-mass spectrometry method for the identification of small molecule ligands from self-encoded combinatorial libraries. Discovery of a novel antagonist of E. coli dihydrofolate reductase</span></div><div class="casAuthors">Annis, D. Allen; Athanasopoulos, John; Curran, Patrick J.; Felsch, Jason S.; Kalghatgi, Krishna; Lee, William H.; Nash, Huw M.; Orminati, Jean-Paul A.; Rosner, Kristin E.; Shipps, Gerald W.; Thaddupathy, G. R. A.; Tyler, Andrew N.; Vilenchik, Lev; Wagner, Carston R.; Wintner, Edward A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Mass Spectrometry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">238</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">77-83</span>CODEN:
                <span class="NLM_cas:coden">IMSPF8</span>;
        ISSN:<span class="NLM_cas:issn">1387-3806</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The NeoGenesis Automated Ligand Identification System (ALIS), an affinity selection-mass spectrometry (AS-MS) process consisting of a rapid size-exclusion chromatog. stage integrated with reverse-phase chromatog., electrospray mass spectrometry, and novel data searching algorithms, was used to screen mass-encoded, 2500-member combinatorial libraries, leading to the discovery of a novel, bioactive ligand for the anti-infective target Escherichia coli dihydrofolate reductase (DHFR).  Synthesis of the mass-encoded, ligand-contg. library, discussion of the deconvolution process for verifying the structure of the ligand through independent synthesis and screening in a small mixt. (sub-library) format, and ALIS-MS/MS techniques to assign its regioisomeric connectivity are presented.  ALIS-based competition expts. between the newly discovered ligand and other, known DHFR ligands, and biol. activity assessments with stereo- and regioisomers of the hit compd. confirm its DHFR-specific biol. activity.  The method described requires no foreknowledge of the structure or biochem. of the protein target, consumes less than 1 μg protein to screen >2500 compds. in a single expt., and enables screening of >250,000 compds. per system per day.  These advantages highlight the potential of the ALIS method for drug discovery against genomic targets with unknown biol. function, as well as validated targets for which traditional discovery efforts have failed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF9t0__7HggLVg90H21EOLACvtfcHk0ljLE2QY_4mfyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpslyisLk%253D&md5=2f0e7e5c2f76f28ef6ac697e76d24092</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ijms.2003.11.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijms.2003.11.022%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DAthanasopoulos%26aufirst%3DJ.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DFelsch%26aufirst%3DJ.%2BS.%26aulast%3DKalghatgi%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DNash%26aufirst%3DH.%2BM.%26aulast%3DOrminati%26aufirst%3DJ.-P.%2BA.%26aulast%3DRosner%26aufirst%3DK.%2BE.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DThaddupathy%26aufirst%3DG.%2BR.%2BA.%26aulast%3DTyler%26aufirst%3DA.%2BN.%26aulast%3DVilenchik%26aufirst%3DL.%26aulast%3DWagner%26aufirst%3DC.%2BR.%26aulast%3DWintner%26aufirst%3DE.%2BA.%26atitle%3DAn%2520affinity%2520selection%25E2%2580%2593mass%2520spectrometry%2520method%2520for%2520the%2520identification%2520of%2520small%2520molecule%2520ligands%2520from%2520self-encoded%2520combinatorial%2520libraries%253A%2520Discovery%2520of%2520a%2520novel%2520antagonist%2520of%2520E.%2520coli%2520dihydrofolate%2520reductase%26jtitle%3DInt.%2520J.%2520Mass%2520Spectrom.%26date%3D2004%26volume%3D238%26spage%3D77%26epage%3D83%26doi%3D10.1016%2Fj.ijms.2003.11.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawahata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soucy, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soriano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, T.</span><span> </span><span class="NLM_article-title">Discovery and optimization of antibacterial AccC inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">6507</span><span class="NLM_x">–</span> <span class="NLM_lpage">6514</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.10.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2009.10.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=19875284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGisLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6507-6514&author=C.+C.+Chengauthor=G.+W.+Shippsauthor=Z.+Yangauthor=B.+Sunauthor=N.+Kawahataauthor=K.+A.+Soucyauthor=A.+Sorianoauthor=P.+Orthauthor=L.+Xiaoauthor=P.+Mannauthor=T.+Black&title=Discovery+and+optimization+of+antibacterial+AccC+inhibitors&doi=10.1016%2Fj.bmcl.2009.10.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of antibacterial AccC inhibitors</span></div><div class="casAuthors">Cheng, Cliff C.; Shipps, Gerald W.; Yang, Zhiwei; Sun, Binyuan; Kawahata, Noriyuki; Soucy, Kyle A.; Soriano, Aileen; Orth, Peter; Xiao, Li; Mann, Paul; Black, Todd</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6507-6514</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Benzimidazolecarboxamides and imidazopyridinecarboxamides such as I (R first defined; R1 = H; X = N) and I (R second defined; R1 = O2N; X = CH) are prepd. by soln. and solid-phase methods as inhibitors of biotin carboxylase (AccC) for potential use as antibacterial agents.  For example, I (R first defined; R1 = H; X = N) and I (R second defined; R1 = O2N; X = CH) are potent and selective inhibitors of bacterial AccC with IC50 values of 20 nM and 10 nM, resp., and MIC values of 0.8 μg/mL against a sensitized strain of Escherichia coli (HS294 E. coli).  The structures of two benzimidazolecarboxamides bound to biotin carboxylase are detd. by X-ray crystallog.; the structure of an imidazopyridinecarboxamide bound to the active site of biotin carboxylase is detd. by computer-aided modeling calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgL8pY9SdU07Vg90H21EOLACvtfcHk0lgTnVUUszCdnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGisLvJ&md5=e811f3f617b9125bd852a4d65ff6e6c2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.10.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.10.057%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DKawahata%26aufirst%3DN.%26aulast%3DSoucy%26aufirst%3DK.%2BA.%26aulast%3DSoriano%26aufirst%3DA.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DMann%26aufirst%3DP.%26aulast%3DBlack%26aufirst%3DT.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520antibacterial%2520AccC%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6507%26epage%3D6514%26doi%3D10.1016%2Fj.bmcl.2009.10.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyss, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soucy, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.-k.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrawal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.-C.</span><span> </span><span class="NLM_article-title">Pyridine carboxamides: potent palm site inhibitors of HCV NS5B polymerase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">466</span><span class="NLM_x">–</span> <span class="NLM_lpage">471</span><span class="refDoi"> DOI: 10.1021/ml100128h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100128h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=466-471&author=C.+C.+Chengauthor=X.+Huangauthor=G.+W.+Shippsauthor=Y.-S.+Wangauthor=D.+F.+Wyssauthor=K.+A.+Soucyauthor=C.-k.+Jiangauthor=S.+Agrawalauthor=E.+Ferrariauthor=Z.+Heauthor=H.-C.+Huang&title=Pyridine+carboxamides%3A+potent+palm+site+inhibitors+of+HCV+NS5B+polymerase&doi=10.1021%2Fml100128h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fml100128h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100128h%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DWang%26aufirst%3DY.-S.%26aulast%3DWyss%26aufirst%3DD.%2BF.%26aulast%3DSoucy%26aufirst%3DK.%2BA.%26aulast%3DJiang%26aufirst%3DC.-k.%26aulast%3DAgrawal%26aufirst%3DS.%26aulast%3DFerrari%26aufirst%3DE.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DH.-C.%26atitle%3DPyridine%2520carboxamides%253A%2520potent%2520palm%2520site%2520inhibitors%2520of%2520HCV%2520NS5B%2520polymerase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D466%26epage%3D471%26doi%3D10.1021%2Fml100128h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawahata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesburg, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duca, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandouveres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.-k.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrawal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.-C.</span><span> </span><span class="NLM_article-title">Inhibitors of hepatitis C virus polymerase: Synthesis and characterization of novel 2-oxy-6-fluoro-<i>N</i>-((S)-1-hydroxy-3-phenylpropan-2-yl)-benzamides</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2119</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.02.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2010.02.054" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=2119-2124&author=C.+C.+Chengauthor=G.+W.+Shippsauthor=Z.+Yangauthor=N.+Kawahataauthor=C.+A.+Lesburgauthor=J.+S.+Ducaauthor=J.+Bandouveresauthor=J.+D.+Brackenauthor=C.-k.+Jiangauthor=S.+Agrawalauthor=E.+Ferrariauthor=H.-C.+Huang&title=Inhibitors+of+hepatitis+C+virus+polymerase%3A+Synthesis+and+characterization+of+novel+2-oxy-6-fluoro-N-%28%28S%29-1-hydroxy-3-phenylpropan-2-yl%29-benzamides&doi=10.1016%2Fj.bmcl.2010.02.054"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.02.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.02.054%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DKawahata%26aufirst%3DN.%26aulast%3DLesburg%26aufirst%3DC.%2BA.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26aulast%3DBandouveres%26aufirst%3DJ.%26aulast%3DBracken%26aufirst%3DJ.%2BD.%26aulast%3DJiang%26aufirst%3DC.-k.%26aulast%3DAgrawal%26aufirst%3DS.%26aulast%3DFerrari%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DH.-C.%26atitle%3DInhibitors%2520of%2520hepatitis%2520C%2520virus%2520polymerase%253A%2520Synthesis%2520and%2520characterization%2520of%2520novel%25202-oxy-6-fluoro-N-%2528%2528S%2529-1-hydroxy-3-phenylpropan-2-yl%2529-benzamides%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D2119%26epage%3D2124%26doi%3D10.1016%2Fj.bmcl.2010.02.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovici-Muller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosner, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Girijavallabhan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butkiewicz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cable, M.</span><span> </span><span class="NLM_article-title">Aminothiazole inhibitors of HCV RNA polymerase</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2004.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2004.10.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=115-119&author=G.+W.+Shippsauthor=Y.+Dengauthor=T.+Wangauthor=J.+Popovici-Mullerauthor=P.+J.+Curranauthor=K.+E.+Rosnerauthor=A.+B.+Cooperauthor=V.+Girijavallabhanauthor=N.+Butkiewiczauthor=M.+Cable&title=Aminothiazole+inhibitors+of+HCV+RNA+polymerase&doi=10.1016%2Fj.bmcl.2004.10.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2004.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2004.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DPopovici-Muller%26aufirst%3DJ.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DRosner%26aufirst%3DK.%2BE.%26aulast%3DCooper%26aufirst%3DA.%2BB.%26aulast%3DGirijavallabhan%26aufirst%3DV.%26aulast%3DButkiewicz%26aufirst%3DN.%26aulast%3DCable%26aufirst%3DM.%26atitle%3DAminothiazole%2520inhibitors%2520of%2520HCV%2520RNA%2520polymerase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D115%26epage%3D119%26doi%3D10.1016%2Fj.bmcl.2004.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovici-Müller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqui, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gowen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Windsor, W. T.</span><span> </span><span class="NLM_article-title">Method for quantitative protein-ligand affinity measurements in compound mixtures</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">4538</span><span class="NLM_x">–</span> <span class="NLM_lpage">4542</span><span class="refDoi"> DOI: 10.1021/ac0702701</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac0702701" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1WhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2007&pages=4538-4542&author=D.+A.+Annisauthor=G.+W.+Shippsauthor=Y.+Dengauthor=J.+Popovici-M%C3%BCllerauthor=M.+A.+Siddiquiauthor=P.+J.+Curranauthor=M.+Gowenauthor=W.+T.+Windsor&title=Method+for+quantitative+protein-ligand+affinity+measurements+in+compound+mixtures&doi=10.1021%2Fac0702701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Method for quantitative protein-ligand affinity measurements in compound mixtures</span></div><div class="casAuthors">Annis, D. Allen; Shipps, Gerald W., Jr.; Deng, Yongqi; Popovici-Mueller, Janeta; Siddiqui, M. Arshad; Curran, Patrick J.; Gowen, Matthew; Windsor, William T.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4538-4542</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This manuscript describes an affinity selection-mass spectrometry (AS-MS) method for quant. protein-ligand binding affinity (Kd) measurements in large compd. libraries.  The ability of a titrant ligand to displace a target-bound library member-as measured by MS-reveals the affinity ranking of the mixt. component relative to "internal affinity calibrants", compds. of known affinity for the target.  This technique does not require that the precise concn. of each ligand is known; therefore, unpurified products of mixt.-based combinatorial synthesis may be used for affinity optimization and developing structure-activity relationships.  The method is demonstrated for a series of ligands to the important oncol. target CDK2 that were discovered by AS-MS screening of combinatorial libraries against the basal form of the protein.  AS-MS displacement curves for select hits were acquired over a range of compd. concns., confirming that binding affinity measurement results are concn.-insensitive.  These hits were evaluated in pools of purified compds. to verify the method's applicability to hit triage in large chem. libraries.  The method was further tested using unpurified, mixt.-based combinatorial libraries of >1000 compds., yielding results that mirror those obtained from mixts. of purified compds.  The technique was then used to identify optimized CDK2 ligands from compd. mixts., quant. measure their affinities, and establish structure-activity relationships among these drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopGVdmMNSdmbVg90H21EOLACvtfcHk0ljGswnmVj6xdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1WhtLk%253D&md5=638ab38a5bc9674a4ccdb1977c721b87</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fac0702701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac0702701%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DPopovici-M%25C3%25BCller%26aufirst%3DJ.%26aulast%3DSiddiqui%26aufirst%3DM.%2BA.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DGowen%26aufirst%3DM.%26aulast%3DWindsor%26aufirst%3DW.%2BT.%26atitle%3DMethod%2520for%2520quantitative%2520protein-ligand%2520affinity%2520measurements%2520in%2520compound%2520mixtures%26jtitle%3DAnal.%2520Chem.%26date%3D2007%26volume%3D79%26spage%3D4538%26epage%3D4542%26doi%3D10.1021%2Fac0702701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Lahue, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seghezzi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herbst, R.</span><span> </span><span class="NLM_article-title">Substituted benzimidazoles: a novel chemotype for small molecule hksp inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3405</span><span class="NLM_x">–</span> <span class="NLM_lpage">3409</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.05.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2009.05.040" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=3405-3409&author=B.+R.+Lahueauthor=Y.+Maauthor=G.+W.+Shippsauthor=W.+Seghezziauthor=R.+Herbst&title=Substituted+benzimidazoles%3A+a+novel+chemotype+for+small+molecule+hksp+inhibitors&doi=10.1016%2Fj.bmcl.2009.05.040"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.05.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.05.040%26sid%3Dliteratum%253Aachs%26aulast%3DLahue%26aufirst%3DB.%2BR.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DSeghezzi%26aufirst%3DW.%26aulast%3DHerbst%26aufirst%3DR.%26atitle%3DSubstituted%2520benzimidazoles%253A%2520a%2520novel%2520chemotype%2520for%2520small%2520molecule%2520hksp%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D3405%26epage%3D3409%26doi%3D10.1016%2Fj.bmcl.2009.05.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Lan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, C.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rojas-Triana, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bober, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hedrick, J. A.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of FABP4/5 ameliorate dyslipidemia but not insulin resistance in mice with diet-induced obesity</span> <span class="citation_source-journal">J. Lipid Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">646</span><span class="NLM_x">–</span> <span class="NLM_lpage">656</span><span class="refDoi"> DOI: 10.1194/jlr.M012757</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1194%2Fjlr.M012757" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=21296956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2011&pages=646-656&author=H.+Lanauthor=C.+C.+Chengauthor=T.+J.+Kowalskiauthor=L.+Pangauthor=L.+Shanauthor=C.-C.+Chuangauthor=J.+Jacksonauthor=A.+Rojas-Trianaauthor=L.+Boberauthor=L.+Liuauthor=J.+Voigtauthor=P.+Orthauthor=X.+Yangauthor=G.+W.+Shippsauthor=J.+A.+Hedrick&title=Small-molecule+inhibitors+of+FABP4%2F5+ameliorate+dyslipidemia+but+not+insulin+resistance+in+mice+with+diet-induced+obesity&doi=10.1194%2Fjlr.M012757"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of FABP4/5 ameliorate dyslipidemia but not insulin resistance in mice with diet-induced obesity</span></div><div class="casAuthors">Lan, Hong; Cheng, Cliff C.; Kowalski, Timothy J.; Pang, Ling; Shan, Lixin; Chuang, Cheng-Chi; Jackson, James; Rojas-Triana, Alberto; Bober, Loretta; Liu, Li; Voigt, Johannes; Orth, Peter; Yang, Xianshu; Shipps, Gerald W., Jr.; Hedrick, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">646-656</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology, Inc.</span>)
        </div><div class="casAbstract">Fatty acid binding protein-4 (FABP4) and FABP5 are two closely related FA binding proteins expressed primarily in adipose tissue and/or macrophages.  The small-mol. FABP4 inhibitor BMS309403 was previously reported to improve insulin sensitivity in leptin-deficient Lepob/Lepob (ob/ob) mice.  However, this compd. was not extensively characterized in the more physiol. relevant animal model of mice with diet-induced obesity (DIO).  Here, we report the discovery and characterization of a novel series of FABP4/5 dual inhibitors represented by Compds. 1-3.  Compared with BMS309403, the compds. had significant in vitro potency toward both FABP4 and FABP5.  In cell-based assays, Compds. 2 and 3 were more potent than BMS309403 to inhibit lipolysis in 3T3-L1 adipocytes and in primary human adipocytes.  They also inhibited MCP-1 release from THP-1 macrophages as well as from primary human macrophages.  When chronically administered to DIO mice, BMS309403 and Compd. 3 reduced plasma triglyceride and free FA levels.  Compd. 3 reduced plasma free FAs at a lower dose level than BMS309403.  However, no significant change was obsd. in insulin, glucose, or glucose tolerance.  Our results indicate that the FABP4/5 inhibitors ameliorate dyslipidemia but not insulin resistance in DIO mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyZVUoUyaoRLVg90H21EOLACvtfcHk0ljGswnmVj6xdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOlt74%253D&md5=3e1d382cd198d4904a8d9b3974cf195d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M012757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M012757%26sid%3Dliteratum%253Aachs%26aulast%3DLan%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DKowalski%26aufirst%3DT.%2BJ.%26aulast%3DPang%26aufirst%3DL.%26aulast%3DShan%26aufirst%3DL.%26aulast%3DChuang%26aufirst%3DC.-C.%26aulast%3DJackson%26aufirst%3DJ.%26aulast%3DRojas-Triana%26aufirst%3DA.%26aulast%3DBober%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DVoigt%26aufirst%3DJ.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DHedrick%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520FABP4%252F5%2520ameliorate%2520dyslipidemia%2520but%2520not%2520insulin%2520resistance%2520in%2520mice%2520with%2520diet-induced%2520obesity%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2011%26volume%3D52%26spage%3D646%26epage%3D656%26doi%3D10.1194%2Fjlr.M012757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, C.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCarter, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nazef, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurzeja, R. J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span> </span><span class="NLM_article-title">Inhibitors of the lipid phosphatase SHIP2 discovered by high throughput affinity selection-mass spectrometry screening of combinatorial libraries</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">771</span><span class="refDoi"> DOI: 10.2174/138620709789104870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.2174%2F138620709789104870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=19531013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlKntbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2009&pages=760-771&author=D.+A.+Annisauthor=C.+C.+Chengauthor=C.-C.+Chuangauthor=J.+D.+McCarterauthor=H.+M.+Nashauthor=N.+Nazefauthor=T.+Roweauthor=R.+J.+M.+Kurzejaauthor=G.+W.+Shipps&title=Inhibitors+of+the+lipid+phosphatase+SHIP2+discovered+by+high+throughput+affinity+selection-mass+spectrometry+screening+of+combinatorial+libraries&doi=10.2174%2F138620709789104870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of the lipid phosphatase SHIP2 discovered by high throughput affinity selection-mass spectrometry screening of combinatorial libraries</span></div><div class="casAuthors">Annis, D. Allen; Cheng, Cliff C.; Chuang, Cheng-Chi; McCarter, John D.; Nash, Huw M.; Nazef, Naim; Rowe, Todd; Kurzeja, Robert J. M.; Shipps, Gerald W., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">760-771</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This manuscript describes the discovery and characterization of inhibitors of the lipid phosphatase SHIP2, an important target for the treatment of Type 2 diabetes, using the Automated Ligand Identification System.  ALIS is an affinity selection-mass spectrometry platform for label-free, high throughput screening of mixt.-based combinatorial libraries.  We detail the mass-encoded synthesis of a library that yielded NGD-61338, a pyrazole-based SHIP2 inhibitor.  Quant. ALIS affinity measurements and inhibition of SHIP2 enzymic activity indicate that this compd. has micromolar binding affinity and inhibitory activity for this target.  This inhibitor, which does not contain a phosphatase "warhead," binds the active site of SHIP2 as detd. by ALIS-based competition expts. with the enzyme's natural substrate, phosphatidylinositol 3,4,5-triphosphate (PIP3).  Structure-activity relationships for NGD-61338 and two other ligand classes discovered by ALIS screening were explored using a combination of combinatorial library synthesis and ALIS-enabled affinity ranking in compd. mixts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon_FH7j0n5r7Vg90H21EOLACvtfcHk0lhIIOPrCTI0Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlKntbnM&md5=f603df2bb224c7f45e8e6a22359ad3d6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.2174%2F138620709789104870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620709789104870%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DChuang%26aufirst%3DC.-C.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DNash%26aufirst%3DH.%2BM.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DKurzeja%26aufirst%3DR.%2BJ.%2BM.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26atitle%3DInhibitors%2520of%2520the%2520lipid%2520phosphatase%2520SHIP2%2520discovered%2520by%2520high%2520throughput%2520affinity%2520selection-mass%2520spectrometry%2520screening%2520of%2520combinatorial%2520libraries%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2009%26volume%3D12%26spage%3D760%26epage%3D771%26doi%3D10.2174%2F138620709789104870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spacciapoli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyss, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Achab, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soucy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montavon, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehurst, C. E.</span><span> </span><span class="NLM_article-title">Discovery and hitto-lead optimization of non-ATP competitive MK2 (MAPKAPK2) inhibitors</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">632</span><span class="NLM_x">–</span> <span class="NLM_lpage">637</span><span class="refDoi"> DOI: 10.1021/ml200113y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml200113y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=632-637&author=X.+Huangauthor=G.+W.+Shippsauthor=C.+C.+Chengauthor=P.+Spacciapoliauthor=X.+Zhangauthor=M.+A.+McCoyauthor=D.+F.+Wyssauthor=X.+Yangauthor=A.+Achabauthor=K.+Soucyauthor=D.+K.+Montavonauthor=D.+M.+Murphyauthor=C.+E.+Whitehurst&title=Discovery+and+hitto-lead+optimization+of+non-ATP+competitive+MK2+%28MAPKAPK2%29+inhibitors&doi=10.1021%2Fml200113y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fml200113y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml200113y%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DSpacciapoli%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMcCoy%26aufirst%3DM.%2BA.%26aulast%3DWyss%26aufirst%3DD.%2BF.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DAchab%26aufirst%3DA.%26aulast%3DSoucy%26aufirst%3DK.%26aulast%3DMontavon%26aufirst%3DD.%2BK.%26aulast%3DMurphy%26aufirst%3DD.%2BM.%26aulast%3DWhitehurst%26aufirst%3DC.%2BE.%26atitle%3DDiscovery%2520and%2520hitto-lead%2520optimization%2520of%2520non-ATP%2520competitive%2520MK2%2520%2528MAPKAPK2%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D632%26epage%3D637%26doi%3D10.1021%2Fml200113y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Rosner, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belanger, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Girijavallabhan, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavey, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovici-Muller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ting, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozlowski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madison, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKittrick, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piwinski, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shih, N.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqui, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strickland, C. O.</span><span> </span><span class="NLM_article-title">The discovery of novel tartrate-based TNF-α converting enzyme (TACE) inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1189</span><span class="NLM_x">–</span> <span class="NLM_lpage">1193</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2009.12.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1189-1193&author=K.+E.+Rosnerauthor=Z.+Guoauthor=P.+Orthauthor=G.+W.+Shippsauthor=D.+B.+Belangerauthor=T.+Y.+Chanauthor=P.+J.+Curranauthor=C.+Daiauthor=Y.+Dengauthor=V.+M.+Girijavallabhanauthor=L.+Hongauthor=B.+J.+Laveyauthor=J.+F.+Leeauthor=D.+Liauthor=Z.+Liuauthor=J.+Popovici-Mullerauthor=P.+C.+Tingauthor=H.+Vaccaroauthor=L.+Wangauthor=T.+Wangauthor=W.+Yuauthor=G.+Zhouauthor=X.+Niuauthor=J.+Sunauthor=J.+A.+Kozlowskiauthor=D.+J.+Lundellauthor=V.+Madisonauthor=B.+McKittrickauthor=J.+J.+Piwinskiauthor=N.-Y.+Shihauthor=M.+A.+Siddiquiauthor=C.+O.+Strickland&title=The+discovery+of+novel+tartrate-based+TNF-%CE%B1+converting+enzyme+%28TACE%29+inhibitors&doi=10.1016%2Fj.bmcl.2009.12.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DRosner%26aufirst%3DK.%2BE.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DBelanger%26aufirst%3DD.%2BB.%26aulast%3DChan%26aufirst%3DT.%2BY.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DDai%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DGirijavallabhan%26aufirst%3DV.%2BM.%26aulast%3DHong%26aufirst%3DL.%26aulast%3DLavey%26aufirst%3DB.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BF.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DPopovici-Muller%26aufirst%3DJ.%26aulast%3DTing%26aufirst%3DP.%2BC.%26aulast%3DVaccaro%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DG.%26aulast%3DNiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DKozlowski%26aufirst%3DJ.%2BA.%26aulast%3DLundell%26aufirst%3DD.%2BJ.%26aulast%3DMadison%26aufirst%3DV.%26aulast%3DMcKittrick%26aufirst%3DB.%26aulast%3DPiwinski%26aufirst%3DJ.%2BJ.%26aulast%3DShih%26aufirst%3DN.-Y.%26aulast%3DSiddiqui%26aufirst%3DM.%2BA.%26aulast%3DStrickland%26aufirst%3DC.%2BO.%26atitle%3DThe%2520discovery%2520of%2520novel%2520tartrate-based%2520TNF-%25CE%25B1%2520converting%2520enzyme%2520%2528TACE%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D1189%26epage%3D1193%26doi%3D10.1016%2Fj.bmcl.2009.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Whitehurst, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nazef, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spacciapoli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebell, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felsch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, H. M.</span><span> </span><span class="NLM_article-title">Discovery and characterization of orthosteric and allosteric muscarinic M2 acetylcholine receptor ligands by affinity selection–mass spectrometry</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">194</span><span class="NLM_x">–</span> <span class="NLM_lpage">207</span><span class="refDoi"> DOI: 10.1177/1087057105284340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1177%2F1087057105284340" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=194-207&author=C.+E.+Whitehurstauthor=N.+Nazefauthor=D.+A.+Annisauthor=Y.+Houauthor=D.+M.+Murphyauthor=P.+Spacciapoliauthor=Z.+Yaoauthor=M.+R.+Ziebellauthor=C.+C.+Chengauthor=G.+W.+Shippsauthor=J.+S.+Felschauthor=D.+Lauauthor=H.+M.+Nash&title=Discovery+and+characterization+of+orthosteric+and+allosteric+muscarinic+M2+acetylcholine+receptor+ligands+by+affinity+selection%E2%80%93mass+spectrometry&doi=10.1177%2F1087057105284340"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1177%2F1087057105284340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057105284340%26sid%3Dliteratum%253Aachs%26aulast%3DWhitehurst%26aufirst%3DC.%2BE.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DMurphy%26aufirst%3DD.%2BM.%26aulast%3DSpacciapoli%26aufirst%3DP.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DZiebell%26aufirst%3DM.%2BR.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DFelsch%26aufirst%3DJ.%2BS.%26aulast%3DLau%26aufirst%3DD.%26aulast%3DNash%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520orthosteric%2520and%2520allosteric%2520muscarinic%2520M2%2520acetylcholine%2520receptor%2520ligands%2520by%2520affinity%2520selection%25E2%2580%2593mass%2520spectrometry%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2005%26volume%3D11%26spage%3D194%26epage%3D207%26doi%3D10.1177%2F1087057105284340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nazef, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, C.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, H. M.</span><span> </span><span class="NLM_article-title">A general technique to rank protein-ligand binding affinities and determine allosteric versus direct binding site competition in compound mixtures</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">15495</span><span class="NLM_x">–</span> <span class="NLM_lpage">15503</span><span class="refDoi"> DOI: 10.1021/ja048365x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja048365x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptlOisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=15495-15503&author=D.+A.+Annisauthor=N.+Nazefauthor=C.-C.+Chuangauthor=M.+P.+Scottauthor=H.+M.+Nash&title=A+general+technique+to+rank+protein-ligand+binding+affinities+and+determine+allosteric+versus+direct+binding+site+competition+in+compound+mixtures&doi=10.1021%2Fja048365x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A general technique to rank protein-ligand binding affinities and determine allosteric versus direct binding site competition in compound mixtures</span></div><div class="casAuthors">Annis, D. Allen; Nazef, Naim; Chuang, Cheng-Chi; Scott, Margaret Porter; Nash, Huw M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">15495-15503</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To realize the full potential of combinatorial chem.-based drug discovery, generic and efficient tools must be developed that apply the strengths of diversity-oriented chem. synthesis to the identification and optimization of lead compds. for disease-assocd. protein targets.  We report an affinity selection-mass spectrometry (AS-MS) method for protein-ligand affinity ranking and the classification of ligands by binding site.  The method incorporates the following steps: (1) an affinity selection stage, where protein-binding compds. are selected from pools of ligands in the presence of varying concns. of a competitor ligand, (2) a first chromatog. stage to sep. unbound ligands from protein-ligand complexes, and (3) a second chromatog. stage to dissoc. the ligands from the complexes for identification and quantification by MS.  The ability of the competitor ligand to displace a target-bound library member, as measured by MS, reveals the binding site classification and affinity ranking of the mixt. components.  The technique requires no radiolabel incorporation or direct biochem. assay, no modification or immobilization of the compds. or target protein, and all reaction components, including any buffers or cofactors required for protein stability, are free in soln.  We demonstrate the method for several compds. of wide structural variety against representatives of the most important protein classes in contemporary drug discovery, including novel ATP-competitive and allosteric inhibitors of the Akt-1 (PKB) and Zap-70 kinases, and previously undisclosed antagonists of the M2 muscarinic acetylcholine receptor, a G-protein coupled receptor (GPCR).  The theor. basis of the technique is analyzed math., allowing quant. estn. of binding affinities and, in the case of allosteric interaction, abs. detn. of binding cooperativity.  The method is readily applicable to high-throughput screening hit triage, combinatorial library-based affinity optimization, and developing structure-activity relationships among multiple ligands to a given receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpREa-teJ7tC7Vg90H21EOLACvtfcHk0li97Pa4oxyQZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptlOisrk%253D&md5=bee110f0d9f40e75b5e22d120238a742</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja048365x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja048365x%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DChuang%26aufirst%3DC.-C.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DNash%26aufirst%3DH.%2BM.%26atitle%3DA%2520general%2520technique%2520to%2520rank%2520protein-ligand%2520binding%2520affinities%2520and%2520determine%2520allosteric%2520versus%2520direct%2520binding%2520site%2520competition%2520in%2520compound%2520mixtures%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D15495%26epage%3D15503%26doi%3D10.1021%2Fja048365x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Brandstetter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lollar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bode, W.</span><span> </span><span class="NLM_article-title">X-ray structure of clotting factor IXa: active site and module structure related to Xase activity and hemophilia B</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">9796</span><span class="NLM_x">–</span> <span class="NLM_lpage">9800</span><span class="refDoi"> DOI: 10.1073/pnas.92.21.9796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1073%2Fpnas.92.21.9796" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=9796-9800&author=H.+Brandstetterauthor=M.+Bauerauthor=R.+Huberauthor=P.+Lollarauthor=W.+Bode&title=X-ray+structure+of+clotting+factor+IXa%3A+active+site+and+module+structure+related+to+Xase+activity+and+hemophilia+B&doi=10.1073%2Fpnas.92.21.9796"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.21.9796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.21.9796%26sid%3Dliteratum%253Aachs%26aulast%3DBrandstetter%26aufirst%3DH.%26aulast%3DBauer%26aufirst%3DM.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DLollar%26aufirst%3DP.%26aulast%3DBode%26aufirst%3DW.%26atitle%3DX-ray%2520structure%2520of%2520clotting%2520factor%2520IXa%253A%2520active%2520site%2520and%2520module%2520structure%2520related%2520to%2520Xase%2520activity%2520and%2520hemophilia%2520B%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1995%26volume%3D92%26spage%3D9796%26epage%3D9800%26doi%3D10.1073%2Fpnas.92.21.9796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Sturzebecher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopetzki, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bode, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopfner, K.-P.</span><span> </span><span class="NLM_article-title">Dramatic enhancement of the catalytic activity of coagulation factor IXa by alcohols</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">412</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">300</span><span class="refDoi"> DOI: 10.1016/S0014-5793(97)00811-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2FS0014-5793%2897%2900811-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=9256238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADyaK2sXks12ltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=412&publication_year=1997&pages=295-300&author=J.+Sturzebecherauthor=E.+Kopetzkiauthor=W.+Bodeauthor=K.-P.+Hopfner&title=Dramatic+enhancement+of+the+catalytic+activity+of+coagulation+factor+IXa+by+alcohols&doi=10.1016%2FS0014-5793%2897%2900811-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Dramatic enhancement of the catalytic activity of coagulation factor IXa by alcohols</span></div><div class="casAuthors">Sturzebecher, Jorg; Kopetzki, Erhard; Bode, Wolfram; Hopfner, Karl-Peter</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">412</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">295-300</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The coagulation factor IXa (FIXa) exhibits a very weak proteolytic activity towards natural or synthetic substrates.  Upon complex formation with its cofactor FVIIIa and Ca2+-mediated binding to phospholipid membranes, FIXa becomes a very potent activator of FX.  The presence of FVIIIa has no effect on the cleavage of peptide substrates by FIXa, however.  Several alcs. dramatically enhance the catalytic activity of human FIXa towards synthetic substrates.  Substrates with the tripeptidyl moiety R-D-Xxx-Gly-Arg are esp. susceptible to the enhanced FIXa catalysis.  Maximal increase up to 20-fold was measured in the presence of ethylene glycol.  Alcs. may modify the conformation of FIXa rendering the active-site cleft more easily accessible to tripeptide substrates with a hydrophobic residue in the P3-position.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWImWN_8rRNLVg90H21EOLACvtfcHk0li97Pa4oxyQZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXks12ltLg%253D&md5=a25070c173abc67d51fc20982fa97eb1</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2897%2900811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252897%252900811-9%26sid%3Dliteratum%253Aachs%26aulast%3DSturzebecher%26aufirst%3DJ.%26aulast%3DKopetzki%26aufirst%3DE.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DHopfner%26aufirst%3DK.-P.%26atitle%3DDramatic%2520enhancement%2520of%2520the%2520catalytic%2520activity%2520of%2520coagulation%2520factor%2520IXa%2520by%2520alcohols%26jtitle%3DFEBS%2520Lett.%26date%3D1997%26volume%3D412%26spage%3D295%26epage%3D300%26doi%3D10.1016%2FS0014-5793%2897%2900811-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Riley, M. D.; Huang, S.; Caceres-Cortes, J.; Johnson, K.; Shu, Y.-Z.</span><span> </span><span class="NLM_article-title">Nuclear magnetic resonance spectroscopy for the study of drug metabolism</span>. In  <span class="citation_source-book">Encyclopedia of Drug Metabolism and Interactions</span>, <span class="NLM_contrib-group">Lyubimov, A. V.</span>, Ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">3</span>, pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470921920.edm102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=1-29&author=M.+D.+Riley&author=S.+Huang&author=J.+Caceres-Cortes&author=K.+Johnson&author=Y.-Z.+Shuauthor=A.+V.+Lyubimov&title=Encyclopedia+of+Drug+Metabolism+and+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2F9780470921920.edm102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470921920.edm102%26sid%3Dliteratum%253Aachs%26aulast%3DRiley%26aufirst%3DM.%2BD.%26atitle%3DNuclear%2520magnetic%2520resonance%2520spectroscopy%2520for%2520the%2520study%2520of%2520drug%2520metabolism%26btitle%3DEncyclopedia%2520of%2520Drug%2520Metabolism%2520and%2520Interactions%26aulast%3DLyubimov%26aufirst%3DA.%2BV.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012%26volume%3D3%26spage%3D1%26epage%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">Small volume high sensitivity NMR probes</span>. In  <span class="citation_source-book">Annu. Rep. NMR Spectrosc.</span>, <span class="NLM_contrib-group">Webb, G. A.</span>, Ed.; <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">56</span>, pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2FS0066-4103%2805%2956001-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=1-99&author=G.+E.+Martinauthor=G.+A.+Webb&title=Annu.+Rep.+NMR+Spectrosc."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2FS0066-4103%2805%2956001-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0066-4103%252805%252956001-0%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DSmall%2520volume%2520high%2520sensitivity%2520NMR%2520probes%26btitle%3DAnnu.%2520Rep.%2520NMR%2520Spectrosc.%26aulast%3DWebb%26aufirst%3DG.%2BA.%26pub%3DElsevier%26date%3D2005%26volume%3D56%26spage%3D1%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">Application of cryogenic NMR probe technology for the identification of low level impurities in pharmaceuticals</span>. In  <span class="citation_source-book">Handbook of Modern Magnetic Resonance</span>, <span class="NLM_contrib-group">Webb, G. A.</span>, Ed.; <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">II</span>, pp  <span class="NLM_fpage">1187</span><span class="NLM_x">–</span> <span class="NLM_lpage">1194</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1007%2F1-4020-3910-7_129" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=1187-1194&author=G.+E.+Martinauthor=G.+A.+Webb&title=Handbook+of+Modern+Magnetic+Resonance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1007%2F1-4020-3910-7_129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F1-4020-3910-7_129%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DApplication%2520of%2520cryogenic%2520NMR%2520probe%2520technology%2520for%2520the%2520identification%2520of%2520low%2520level%2520impurities%2520in%2520pharmaceuticals%26btitle%3DHandbook%2520of%2520Modern%2520Magnetic%2520Resonance%26aulast%3DWebb%26aufirst%3DG.%2BA.%26pub%3DElsevier%26date%3D2006%26volume%3DII%26spage%3D1187%26epage%3D1194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Adams, R. W.</span><span> </span><span class="NLM_article-title">Pure shift NMR spectroscopy</span> <span class="citation_source-journal">eMagRes.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">310</span><span class="refDoi"> DOI: 10.1002/9780470034590.emrstm1362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470034590.emrstm1362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVGlurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=295-310&author=R.+W.+Adams&title=Pure+shift+NMR+spectroscopy&doi=10.1002%2F9780470034590.emrstm1362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Pure shift NMR spectroscopy</span></div><div class="casAuthors">Adams, Ralph W.</div><div class="citationInfo"><span class="NLM_cas:title">eMagRes</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-310</span>CODEN:
                <span class="NLM_cas:coden">EMAGCE</span>;
        ISSN:<span class="NLM_cas:issn">2055-6101</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Pure shift NMR spectroscopy, or broadband homonuclear decoupled NMR spectroscopy, produces NMR spectra that contain only chem. shift information.  In a pure shift NMR spectrum the effects of J-coupling are hidden so that all multiplets are collapsed to singlets.  Pure shift methods can deliver a significant improvement in resoln. of NMR signals and greatly reduce signal overlap.  Pure shift NMR can be achieved using direct or indirect acquisition; for direct acquisition, real-time- or interferogram-based FID collection can be used.  The range of pure shift methods includes those that refocus the effects of J-coupling (BIRD, Zangger-Sterk, band selective decoupling, and anti z-COSY); manipulate the evolution of chem. shift within a const. time delay to keep the J-evolution in an indirect acquisition const.; or ext. a 1-D spectrum from a 2-D J dataset.  The methods and their mechanisms in weakly coupled systems are described using the product operator formalism.  Examples are provided for a range of applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMUf_z3uFhILVg90H21EOLACvtfcHk0ljznhVqvHNZAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVGlurw%253D&md5=39bb6a2d92a36a7fe8d56301a073484b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2F9780470034590.emrstm1362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470034590.emrstm1362%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DR.%2BW.%26atitle%3DPure%2520shift%2520NMR%2520spectroscopy%26jtitle%3DeMagRes.%26date%3D2014%26volume%3D3%26spage%3D295%26epage%3D310%26doi%3D10.1002%2F9780470034590.emrstm1362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Zangger, K.</span><span> </span><span class="NLM_article-title">Pure shift NMR</span> <span class="citation_source-journal">Prog. Nucl. Magn. Reson. Spectrosc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">86–87</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1016/j.pnmrs.2015.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.pnmrs.2015.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=25919196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtValt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86%E2%80%9387&publication_year=2015&pages=1-20&author=K.+Zangger&title=Pure+shift+NMR&doi=10.1016%2Fj.pnmrs.2015.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Pure shift NMR</span></div><div class="casAuthors">Zangger, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Nuclear Magnetic Resonance Spectroscopy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">86-87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-20</span>CODEN:
                <span class="NLM_cas:coden">PNMRAT</span>;
        ISSN:<span class="NLM_cas:issn">0079-6565</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although scalar-coupling provides important structural information, the resulting signal splittings significantly reduce the resoln. of NMR spectra.  Limited resoln. is a particular problem in proton NMR expts., resulting in part from the limited proton chem. shift range (∼10 ppm) but even more from the splittings due to scalar coupling to nearby protons.  "Pure shift" NMR spectroscopy (also known as broadband homonuclear decoupling) has been developed for disentangling overlapped proton NMR spectra.  The resulting spectra are considerably simplified as they consist of single lines, reminiscent of proton-decoupled C-13 spectra at natural abundance, with no multiplet structure.  The different approaches to obtaining pure shift spectra are reviewed here and several applications presented.  Pure shift spectra are esp. useful for highly overlapped proton spectra, as found for example in reaction mixts., natural products and biomacromols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokhOn2iyQOQbVg90H21EOLACvtfcHk0ljznhVqvHNZAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtValt7c%253D&md5=6457afd7f42e3cc78062b5c666a9f36a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.pnmrs.2015.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pnmrs.2015.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DZangger%26aufirst%3DK.%26atitle%3DPure%2520shift%2520NMR%26jtitle%3DProg.%2520Nucl.%2520Magn.%2520Reson.%2520Spectrosc.%26date%3D2015%26volume%3D86%25E2%2580%259387%26spage%3D1%26epage%3D20%26doi%3D10.1016%2Fj.pnmrs.2015.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Castañar, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parella, T.</span><span> </span><span class="NLM_article-title">Broadband <sup>1</sup>H homodecoupled NMR experiments: Recent developments, methods and applications</span> <span class="citation_source-journal">Magn. Reson. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">399</span><span class="NLM_x">–</span> <span class="NLM_lpage">426</span><span class="refDoi"> DOI: 10.1002/mrc.4238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2Fmrc.4238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=25899911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslyjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2015&pages=399-426&author=L.+Casta%C3%B1arauthor=T.+Parella&title=Broadband+1H+homodecoupled+NMR+experiments%3A+Recent+developments%2C+methods+and+applications&doi=10.1002%2Fmrc.4238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Broadband 1H homodecoupled NMR experiments: recent developments, methods and applications</span></div><div class="casAuthors">Castanar, Laura; Parella, Teodor</div><div class="citationInfo"><span class="NLM_cas:title">Magnetic Resonance in Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">399-426</span>CODEN:
                <span class="NLM_cas:coden">MRCHEG</span>;
        ISSN:<span class="NLM_cas:issn">0749-1581</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years, a great interest in the development of new broadband 1H homonuclear decoupled techniques providing simplified JHH multiplet patterns has emerged again in the field of small mol. NMR.  The resulting highly resolved 1H NMR spectra display resonances as collapsed singlets, therefore minimizing signal overlap and expediting spectral anal.  This review aims at presenting the most recent advances in pure shift NMR spectroscopy, with a particular emphasis to the Zangger-Sterk expt.  A detailed discussion about the most relevant practical aspects in terms of pulse sequence design, selectivity, sensitivity, spectral resoln. and performance is provided.  Finally, the implementation of the different reported strategies into traditional 1-dimensional and 2-dimensional NMR expts. is described while several practical applications are also reviewed.  Copyright © 2015 John Wiley and Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNiaB_DZi5kLVg90H21EOLACvtfcHk0ljznhVqvHNZAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslyjtbY%253D&md5=e5a6e7da0609e6ad63d8a5b875748ced</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fmrc.4238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmrc.4238%26sid%3Dliteratum%253Aachs%26aulast%3DCasta%25C3%25B1ar%26aufirst%3DL.%26aulast%3DParella%26aufirst%3DT.%26atitle%3DBroadband%25201H%2520homodecoupled%2520NMR%2520experiments%253A%2520Recent%2520developments%252C%2520methods%2520and%2520applications%26jtitle%3DMagn.%2520Reson.%2520Chem.%26date%3D2015%26volume%3D53%26spage%3D399%26epage%3D426%26doi%3D10.1002%2Fmrc.4238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Williamson, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buevich, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parella, T.</span><span> </span><span class="NLM_article-title">LR-HSQMBC: A sensitive NMR technique to probe very long-range heteronuclear coupling pathways</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">3887</span><span class="NLM_x">–</span> <span class="NLM_lpage">3894</span><span class="refDoi"> DOI: 10.1021/jo500333u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo500333u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsl2ksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2014&pages=3887-3894&author=R.+T.+Williamsonauthor=A.+V.+Buevichauthor=G.+E.+Martinauthor=T.+Parella&title=LR-HSQMBC%3A+A+sensitive+NMR+technique+to+probe+very+long-range+heteronuclear+coupling+pathways&doi=10.1021%2Fjo500333u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">LR-HSQMBC: A Sensitive NMR Technique To Probe Very Long-Range Heteronuclear Coupling Pathways</span></div><div class="casAuthors">Williamson, R. Thomas; Buevich, Alexei V.; Martin, Gary E.; Parella, Teodor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3887-3894</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HMBC is one of the most often used and vital NMR expts. for the structure elucidation of org. and inorg. mols.  We have developed a new, high sensitivity NMR pulse sequence that overcomes the typical 2,3JCH limitation of HMBC by extending the visualization of long-range correlation data to 4-, 5-, and even 6-bond long-range nJCH heteronuclear couplings.  This technique should prove to be an effective expt. to complement HMBC for probing the structure of proton-deficient mols.  The LR-HSQMBC NMR expt. can, in effect, extend the range of HMBC to provide data similar to that afforded by 1,n-ADEQUATE even in sample-limited situations.  This is accomplished by optimizing responses for very small nJCH couplings as opposed to relying on the markedly less sensitive detection of long-range coupled 13C-13C homonuclear pairs at natural abundance.  DFT calcns. were employed to det. whether the very long-range correlations obsd. for cervinomycin A2 were reasonable on the basis of the calcd. long-range couplings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZEup7cBKL97Vg90H21EOLACvtfcHk0ljznhVqvHNZAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsl2ksrY%253D&md5=3945cadbae7cebed0057c9980c7a3052</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjo500333u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo500333u%26sid%3Dliteratum%253Aachs%26aulast%3DWilliamson%26aufirst%3DR.%2BT.%26aulast%3DBuevich%26aufirst%3DA.%2BV.%26aulast%3DMartin%26aufirst%3DG.%2BE.%26aulast%3DParella%26aufirst%3DT.%26atitle%3DLR-HSQMBC%253A%2520A%2520sensitive%2520NMR%2520technique%2520to%2520probe%2520very%2520long-range%2520heteronuclear%2520coupling%2520pathways%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2014%26volume%3D79%26spage%3D3887%26epage%3D3894%26doi%3D10.1021%2Fjo500333u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Martin, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzer, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouch, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castle, R. N.</span><span> </span><span class="NLM_article-title">Inverted and suppressed direct response HMQC-TOCSY spectra -- a convenient method of spectral editing</span> <span class="citation_source-journal">J. Heterocycl. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">582</span><span class="refDoi"> DOI: 10.1002/jhet.5570290251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2Fjhet.5570290251" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1992&pages=577-582&author=G.+E.+Martinauthor=T.+D.+Spitzerauthor=R.+C.+Crouchauthor=J.-K.+Luoauthor=R.+N.+Castle&title=Inverted+and+suppressed+direct+response+HMQC-TOCSY+spectra+%2D%2D+a+convenient+method+of+spectral+editing&doi=10.1002%2Fjhet.5570290251"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570290251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570290251%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%2BE.%26aulast%3DSpitzer%26aufirst%3DT.%2BD.%26aulast%3DCrouch%26aufirst%3DR.%2BC.%26aulast%3DLuo%26aufirst%3DJ.-K.%26aulast%3DCastle%26aufirst%3DR.%2BN.%26atitle%3DInverted%2520and%2520suppressed%2520direct%2520response%2520HMQC-TOCSY%2520spectra%2520--%2520a%2520convenient%2520method%2520of%2520spectral%2520editing%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D1992%26volume%3D29%26spage%3D577%26epage%3D582%26doi%3D10.1002%2Fjhet.5570290251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="note"><p class="first last">Structure based rationale for binding affinity changes observed for the metabololites was provided by computational modeling based off a similar FIXa cocrystal structure. See <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><div class="note"><p class="first last">For Cheng-Prusoff equation for competitive inhibition, see:</p></div><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (<i>K<sub>I</sub></i>) and the concentration of inhibitor which causes 50% inhibition (<i>I<sub>50</sub></i>) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099&author=Y.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28KI%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0liwIWUGCOhJ7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528KI%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':[],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':[],'ref56':['cit56']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonia F.  Stepan</span>, <span class="hlFld-ContribAuthor ">Tuan P.  Tran</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Helal</span>, <span class="hlFld-ContribAuthor ">Maria S.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheng  Chang</span>, <span class="hlFld-ContribAuthor ">Rebecca E.  O’Connor</span>, <span class="hlFld-ContribAuthor ">Michael  De Vivo</span>, <span class="hlFld-ContribAuthor ">Shawn D.  Doran</span>, <span class="hlFld-ContribAuthor ">Ethan L.  Fisher</span>, <span class="hlFld-ContribAuthor ">Stephen  Jenkinson</span>, <span class="hlFld-ContribAuthor ">David  Karanian</span>, <span class="hlFld-ContribAuthor ">Bethany L.  Kormos</span>, <span class="hlFld-ContribAuthor ">Raman  Sharma</span>, <span class="hlFld-ContribAuthor ">Gregory S.  Walker</span>, <span class="hlFld-ContribAuthor ">Ann S.  Wright</span>, <span class="hlFld-ContribAuthor ">Edward X.  Yang</span>, <span class="hlFld-ContribAuthor ">Michael A.  Brodney</span>, <span class="hlFld-ContribAuthor ">Travis T.  Wager</span>, <span class="hlFld-ContribAuthor ">Patrick R.  Verhoest</span>, and <span class="hlFld-ContribAuthor ">R. Scott  Obach</span>  . </span><span class="cited-content_cbyCitation_article-title">Late-Stage Microsomal Oxidation Reduces Drug–Drug Interaction and Identifies Phosphodiesterase 2A Inhibitor PF-06815189. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (2)
                                     , 68-72. <a href="https://doi.org/10.1021/acsmedchemlett.7b00343" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.7b00343</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.7b00343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.7b00343%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DLate-Stage%252BMicrosomal%252BOxidation%252BReduces%252BDrug%2525E2%252580%252593Drug%252BInteraction%252Band%252BIdentifies%252BPhosphodiesterase%252B2A%252BInhibitor%252BPF-06815189%26aulast%3DStepan%26aufirst%3DAntonia%2BF.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D19082017%26date%3D03012018%26date%3D30012018%26date%3D08022018%26date%3D04012018%26volume%3D9%26issue%3D2%26spage%3D68%26epage%3D72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. G.  Finn</span> (<span class="NLM_role">Editor-in-Chief</span>). </span><span class="cited-content_cbyCitation_article-title">Technical Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2017,</strong> <em>19 </em>
                                    (5)
                                     , 277-278. <a href="https://doi.org/10.1021/acscombsci.7b00053" title="DOI URL">https://doi.org/10.1021/acscombsci.7b00053</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.7b00053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.7b00053%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DTechnical%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DFinn%26aufirst%3DM.%2BG.%26date%3D2017%26date%3D2017%26date%3D30032017%26date%3D04042017%26date%3D08052017%26volume%3D19%26issue%3D5%26spage%3D277%26epage%3D278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stevan W.  Djuric</span>  , <span class="hlFld-ContribAuthor ">Nicholas A.  Meanwell</span>  . </span><span class="cited-content_cbyCitation_article-title">Journal of Medicinal Chemistry, Technological Advances: Highlights 2015–2016. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (1)
                                     , 1-3. <a href="https://doi.org/10.1021/acs.jmedchem.6b01600" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01600</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01600%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DJournal%252Bof%252BMedicinal%252BChemistry%25252C%252BTechnological%252BAdvances%25253A%252BHighlights%252B2015%2525E2%252580%2525932016%26aulast%3DDjuric%26aufirst%3DStevan%2BW.%26date%3D2017%26date%3D2016%26date%3D01112016%26date%3D08112016%26date%3D12012017%26volume%3D60%26issue%3D1%26spage%3D1%26epage%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dehua  Yang</span>, <span class="hlFld-ContribAuthor ">Qingtong  Zhou</span>, <span class="hlFld-ContribAuthor ">Viktorija  Labroska</span>, <span class="hlFld-ContribAuthor ">Shanshan  Qin</span>, <span class="hlFld-ContribAuthor ">Sanaz  Darbalaei</span>, <span class="hlFld-ContribAuthor ">Yiran  Wu</span>, <span class="hlFld-ContribAuthor ">Elita  Yuliantie</span>, <span class="hlFld-ContribAuthor ">Linshan  Xie</span>, <span class="hlFld-ContribAuthor ">Houchao  Tao</span>, <span class="hlFld-ContribAuthor ">Jianjun  Cheng</span>, <span class="hlFld-ContribAuthor ">Qing  Liu</span>, <span class="hlFld-ContribAuthor ">Suwen  Zhao</span>, <span class="hlFld-ContribAuthor ">Wenqing  Shui</span>, <span class="hlFld-ContribAuthor ">Yi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ming-Wei  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">G protein-coupled receptors: structure- and function-based drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2021,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-020-00435-w" title="DOI URL">https://doi.org/10.1038/s41392-020-00435-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-020-00435-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-020-00435-w%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DG%252Bprotein-coupled%252Breceptors%25253A%252Bstructure-%252Band%252Bfunction-based%252Bdrug%252Bdiscovery%26aulast%3DYang%26aufirst%3DDehua%26date%3D2021%26date%3D2021%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Josep  Saurí</span>, <span class="hlFld-ContribAuthor ">Mikhail  Reibarkh</span>, <span class="hlFld-ContribAuthor ">R. Thomas  Williamson</span>, <span class="hlFld-ContribAuthor ">Gary E.  Martin</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in
              NMR
              ‐based Characterization of Drug Substances, Impurities, Degradants, and Drug Metabolites. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-72. <a href="https://doi.org/10.1002/0471266949.bmc302" title="DOI URL">https://doi.org/10.1002/0471266949.bmc302</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc302%26sid%3Dliteratum%253Aachs%26atitle%3DAdvances%252Bin%252BNMR%252B%2525E2%252580%252590based%252BCharacterization%252Bof%252BDrug%252BSubstances%25252C%252BImpurities%25252C%252BDegradants%25252C%252Band%252BDrug%252BMetabolites%26aulast%3DSaur%25C3%25AD%26aufirst%3DJosep%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D72%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi  Tao</span>, <span class="hlFld-ContribAuthor ">Jizhong  Yan</span>, <span class="hlFld-ContribAuthor ">Baochang  Cai</span>. </span><span class="cited-content_cbyCitation_article-title">LABEL‐FREE BIO‐AFFINITY MASS SPECTROMETRY FOR SCREENING AND LOCATING BIOACTIVE MOLECULES. </span><span class="cited-content_cbyCitation_journal-name">Mass Spectrometry Reviews</span><span> <strong>2021,</strong> <em>40 </em>
                                    (1)
                                     , 53-71. <a href="https://doi.org/10.1002/mas.21613" title="DOI URL">https://doi.org/10.1002/mas.21613</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/mas.21613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmas.21613%26sid%3Dliteratum%253Aachs%26jtitle%3DMass%2520Spectrometry%2520Reviews%26atitle%3DLABEL%2525E2%252580%252590FREE%252BBIO%2525E2%252580%252590AFFINITY%252BMASS%252BSPECTROMETRY%252BFOR%252BSCREENING%252BAND%252BLOCATING%252BBIOACTIVE%252BMOLECULES%26aulast%3DTao%26aufirst%3DYi%26date%3D2021%26date%3D2019%26volume%3D40%26issue%3D1%26spage%3D53%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renaud  Prudent</span>, <span class="hlFld-ContribAuthor ">D. Allen  Annis</span>, <span class="hlFld-ContribAuthor ">Peter J.  Dandliker</span>, <span class="hlFld-ContribAuthor ">Jean-Yves  Ortholand</span>, <span class="hlFld-ContribAuthor ">Didier  Roche</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring new targets and chemical space with affinity selection-mass spectrometry. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Chemistry</span><span> <strong>2021,</strong> <em>5 </em>
                                    (1)
                                     , 62-71. <a href="https://doi.org/10.1038/s41570-020-00229-2" title="DOI URL">https://doi.org/10.1038/s41570-020-00229-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41570-020-00229-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41570-020-00229-2%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Chemistry%26atitle%3DExploring%252Bnew%252Btargets%252Band%252Bchemical%252Bspace%252Bwith%252Baffinity%252Bselection-mass%252Bspectrometry%26aulast%3DPrudent%26aufirst%3DRenaud%26date%3D2021%26date%3D2020%26volume%3D5%26issue%3D1%26spage%3D62%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rhys  Salter</span>, <span class="hlFld-ContribAuthor ">Douglas C.  Beshore</span>, <span class="hlFld-ContribAuthor ">Steven L.  Colletti</span>, <span class="hlFld-ContribAuthor ">Liam  Evans</span>, <span class="hlFld-ContribAuthor ">Yong  Gong</span>, <span class="hlFld-ContribAuthor ">Roy  Helmy</span>, <span class="hlFld-ContribAuthor ">Yong  Liu</span>, <span class="hlFld-ContribAuthor ">Cheri M.  Maciolek</span>, <span class="hlFld-ContribAuthor ">Gary  Martin</span>, <span class="hlFld-ContribAuthor ">Natasa  Pajkovic</span>, <span class="hlFld-ContribAuthor ">Richard  Phipps</span>, <span class="hlFld-ContribAuthor ">James  Small</span>, <span class="hlFld-ContribAuthor ">Jonathan  Steele</span>, <span class="hlFld-ContribAuthor ">Ronald  de Vries</span>, <span class="hlFld-ContribAuthor ">Headley  Williams</span>, <span class="hlFld-ContribAuthor ">Iain J.  Martin</span>. </span><span class="cited-content_cbyCitation_article-title">Microbial biotransformation – an important tool for the study of drug metabolism. </span><span class="cited-content_cbyCitation_journal-name">Xenobiotica</span><span> <strong>2019,</strong> <em>49 </em>
                                    (8)
                                     , 877-886. <a href="https://doi.org/10.1080/00498254.2018.1512018" title="DOI URL">https://doi.org/10.1080/00498254.2018.1512018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/00498254.2018.1512018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F00498254.2018.1512018%26sid%3Dliteratum%253Aachs%26jtitle%3DXenobiotica%26atitle%3DMicrobial%252Bbiotransformation%252B%2525E2%252580%252593%252Ban%252Bimportant%252Btool%252Bfor%252Bthe%252Bstudy%252Bof%252Bdrug%252Bmetabolism%26aulast%3DSalter%26aufirst%3DRhys%26date%3D2019%26date%3D2018%26volume%3D49%26issue%3D8%26spage%3D877%26epage%3D886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuzhen  Niu</span>, <span class="hlFld-ContribAuthor ">Xiaojun  Yao</span>, <span class="hlFld-ContribAuthor ">Hongfang  Ji</span>. </span><span class="cited-content_cbyCitation_article-title">Importance of protein flexibility in ranking ERK2 Type I
              1/2
              inhibitor affinities: a computational study. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2019,</strong> <em>9 </em>
                                    (22)
                                     , 12441-12454. <a href="https://doi.org/10.1039/C9RA01657K" title="DOI URL">https://doi.org/10.1039/C9RA01657K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA01657K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA01657K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DImportance%252Bof%252Bprotein%252Bflexibility%252Bin%252Branking%252BERK2%252BType%252BI%252B1%25252F2%252Binhibitor%252Baffinities%25253A%252Ba%252Bcomputational%252Bstudy%26aulast%3DNiu%26aufirst%3DYuzhen%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D22%26spage%3D12441%26epage%3D12454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. S.  Kabankin</span>, <span class="hlFld-ContribAuthor ">E. I.  Sinauridze</span>, <span class="hlFld-ContribAuthor ">E. N.  Lipets</span>, <span class="hlFld-ContribAuthor ">F. I.  Ataullakhanov</span>. </span><span class="cited-content_cbyCitation_article-title">Computer Design of Low-Molecular-Weight Inhibitors of Coagulation Factors. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry (Moscow)</span><span> <strong>2019,</strong> <em>84 </em>
                                    (2)
                                     , 119-136. <a href="https://doi.org/10.1134/S0006297919020032" title="DOI URL">https://doi.org/10.1134/S0006297919020032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S0006297919020032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS0006297919020032%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%2520%2528Moscow%2529%26atitle%3DComputer%252BDesign%252Bof%252BLow-Molecular-Weight%252BInhibitors%252Bof%252BCoagulation%252BFactors%26aulast%3DKabankin%26aufirst%3DA.%2BS.%26date%3D2019%26date%3D2019%26volume%3D84%26issue%3D2%26spage%3D119%26epage%3D136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael A.  Brodney</span>, <span class="hlFld-ContribAuthor ">Raman  Sharma</span>, <span class="hlFld-ContribAuthor ">John T.  Lazzaro</span>, <span class="hlFld-ContribAuthor ">Gregory S.  Walker</span>, <span class="hlFld-ContribAuthor ">R.  Scott Obach</span>. </span><span class="cited-content_cbyCitation_article-title">Harnessing biosynthesis and quantitative NMR for late stage functionalization of lead molecules: Application to the M1 positive allosteric modulator (PAM) program. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (11)
                                     , 2068-2073. <a href="https://doi.org/10.1016/j.bmcl.2018.04.054" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.04.054</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.04.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.04.054%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DHarnessing%252Bbiosynthesis%252Band%252Bquantitative%252BNMR%252Bfor%252Blate%252Bstage%252Bfunctionalization%252Bof%252Blead%252Bmolecules%25253A%252BApplication%252Bto%252Bthe%252BM1%252Bpositive%252Ballosteric%252Bmodulator%252B%252528PAM%252529%252Bprogram%26aulast%3DBrodney%26aufirst%3DMichael%2BA.%26date%3D2018%26volume%3D28%26issue%3D11%26spage%3D2068%26epage%3D2073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julien  Genovino</span>, <span class="hlFld-ContribAuthor ">Dalibor  Sames</span>, <span class="hlFld-ContribAuthor ">Lawrence G.  Hamann</span>, <span class="hlFld-ContribAuthor ">B. Barry  Touré</span>. </span><span class="cited-content_cbyCitation_article-title">Die Erschließung von Wirkstoffmetaboliten durch übergangsmetallkatalysierte C-H-Oxidation: die Leber als Inspiration für die Synthese. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2016,</strong> <em>128 </em>
                                    (46)
                                     , 14430-14451. <a href="https://doi.org/10.1002/ange.201602644" title="DOI URL">https://doi.org/10.1002/ange.201602644</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201602644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201602644%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DDie%252BErschlie%2525C3%25259Fung%252Bvon%252BWirkstoffmetaboliten%252Bdurch%252B%2525C3%2525BCbergangsmetallkatalysierte%252BC-H-Oxidation%25253A%252Bdie%252BLeber%252Bals%252BInspiration%252Bf%2525C3%2525BCr%252Bdie%252BSynthese%26aulast%3DGenovino%26aufirst%3DJulien%26date%3D2016%26date%3D2016%26volume%3D128%26issue%3D46%26spage%3D14430%26epage%3D14451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julien  Genovino</span>, <span class="hlFld-ContribAuthor ">Dalibor  Sames</span>, <span class="hlFld-ContribAuthor ">Lawrence G.  Hamann</span>, <span class="hlFld-ContribAuthor ">B. Barry  Touré</span>. </span><span class="cited-content_cbyCitation_article-title">Accessing Drug Metabolites via Transition-Metal Catalyzed C−H Oxidation: The Liver as Synthetic Inspiration. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2016,</strong> <em>55 </em>
                                    (46)
                                     , 14218-14238. <a href="https://doi.org/10.1002/anie.201602644" title="DOI URL">https://doi.org/10.1002/anie.201602644</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201602644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201602644%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DAccessing%252BDrug%252BMetabolites%252Bvia%252BTransition-Metal%252BCatalyzed%252BC%2525E2%252588%252592H%252BOxidation%25253A%252BThe%252BLiver%252Bas%252BSynthetic%252BInspiration%26aulast%3DGenovino%26aufirst%3DJulien%26date%3D2016%26date%3D2016%26volume%3D55%26issue%3D46%26spage%3D14218%26epage%3D14238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_null" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0012.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Pd(OAc)<sub>2</sub>, 2,8,9-triisobutyl-2,5,8,9-tetraaza-1-phosphabicyclo[3.3.3]undecane, NaHMDS in THF, dioxane, 100 °C, 3 h; (ii) NaBH<sub>4</sub>, NiCl<sub>2</sub>, EtOH, THF, 25 °C, overnight; (iii) oxalyl chloride, cat. DMF, DCM, 0–25 °C, 2 h; (iv) EtOH, THF, 25 °C, 1 h; (v) Chiral separation on IC column.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Extracted ion chromatograms (XIC) showing metabolites arising from human liver microsomal (HLM) and rat liver microsomal (RLM) incubations of <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Proposed structures for major oxidative metabolites with chromatographic retention times from the chromatograms shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. ALIS screen using a crude RLM incubation metabolite mixture. ALIS binding affinity ranking was M538a > <b>7</b> > M538b > M538c, M538d. At each concentration, two replicates were performed and data averaged.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Dog liver microsomal incubation of <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/medium/jm-2015-01293y_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. FIXa enzymatic assay titration of <b>7</b> and its metabolites. *IC<sub>50</sub> values represent single determinations performed with two replicates per concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-5/acs.jmedchem.5b01293/20160310/images/large/jm-2015-01293y_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01293&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i55">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16406" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16406" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 56 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Pelkonen, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turpeinen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uusitalo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rautio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raunio, H.</span><span> </span><span class="NLM_article-title">Prediction of drug metabolism and interactions on the basis of in vitro investigations</span> <span class="citation_source-journal">Basic Clin. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">175</span><span class="refDoi"> DOI: 10.1111/j.1742-7843.2005.pto960305.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1111%2Fj.1742-7843.2005.pto960305.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=15733211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisFSmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2005&pages=167-175&author=O.+Pelkonenauthor=M.+Turpeinenauthor=J.+Uusitaloauthor=A.+Rautioauthor=H.+Raunio&title=Prediction+of+drug+metabolism+and+interactions+on+the+basis+of+in+vitro+investigations&doi=10.1111%2Fj.1742-7843.2005.pto960305.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of drug metabolism and interactions on the basis of in vitro investigations</span></div><div class="casAuthors">Pelkonen, Olavi; Turpeinen, Miia; Uusitalo, Jouko; Rautio, Arja; Raunio, Hannu</div><div class="citationInfo"><span class="NLM_cas:title">Basic & Clinical Pharmacology & Toxicology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">167-175</span>CODEN:
                <span class="NLM_cas:coden">BCPTBO</span>;
        ISSN:<span class="NLM_cas:issn">1742-7835</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Drug metab. profoundly affects drug action, because almost all drugs are metabolized in the body and thus their concns. and elimination rates are dependent on metabolic activity.  Drug metab. contributes substantially to interindividual differences in drug response and is also often involved in drug interactions, resulting in either therapeutic failure or adverse effects.  Knowledge about the metab. of a new chem. entity and its affinity to drug-metabolizing enzymes helps in the drug development process by providing important information for the selection of a lead compd. from among a no. of substances pharmacol. equally effective in their therapeutic response.  In drug development protocols, metab. characteristics should be assessed very early during the development process.  This was made possible by the advances made esp. in anal. capabilities and in in vitro technologies that are employed to predict in vivo metabolite profile, pharmacokinetic parameters and drug-drug interaction potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9DeBGTnGQWbVg90H21EOLACvtfcHk0lgqYdzhiH40EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisFSmu70%253D&md5=41ac999c09cf85269e92c9c353a24e9b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-7843.2005.pto960305.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-7843.2005.pto960305.x%26sid%3Dliteratum%253Aachs%26aulast%3DPelkonen%26aufirst%3DO.%26aulast%3DTurpeinen%26aufirst%3DM.%26aulast%3DUusitalo%26aufirst%3DJ.%26aulast%3DRautio%26aufirst%3DA.%26aulast%3DRaunio%26aufirst%3DH.%26atitle%3DPrediction%2520of%2520drug%2520metabolism%2520and%2520interactions%2520on%2520the%2520basis%2520of%2520in%2520vitro%2520investigations%26jtitle%3DBasic%2520Clin.%2520Pharmacol.%2520Toxicol.%26date%3D2005%26volume%3D96%26spage%3D167%26epage%3D175%26doi%3D10.1111%2Fj.1742-7843.2005.pto960305.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Plant, N.</span><span> </span><span class="NLM_article-title">Strategies for using <i>in vitro</i> screens in drug metabolism</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">328</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span><span class="refDoi"> DOI: 10.1016/S1359-6446(03)03019-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2FS1359-6446%2803%2903019-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=328-336&author=N.+Plant&title=Strategies+for+using+in+vitro+screens+in+drug+metabolism&doi=10.1016%2FS1359-6446%2803%2903019-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2903019-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252903019-8%26sid%3Dliteratum%253Aachs%26aulast%3DPlant%26aufirst%3DN.%26atitle%3DStrategies%2520for%2520using%2520in%2520vitro%2520screens%2520in%2520drug%2520metabolism%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D328%26epage%3D336%26doi%3D10.1016%2FS1359-6446%2803%2903019-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Brandon, E. F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raap, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meijerman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beijnen, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schellens, J. H. M.</span><span> </span><span class="NLM_article-title">An update on in vitro test methods in human hepatic drug biotransformation research: pros and cons</span> <span class="citation_source-journal">Toxicol. Appl. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">189</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.1016/S0041-008X(03)00128-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2FS0041-008X%2803%2900128-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=12791308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1yjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2003&pages=233-246&author=E.+F.+A.+Brandonauthor=C.+D.+Raapauthor=I.+Meijermanauthor=J.+H.+Beijnenauthor=J.+H.+M.+Schellens&title=An+update+on+in+vitro+test+methods+in+human+hepatic+drug+biotransformation+research%3A+pros+and+cons&doi=10.1016%2FS0041-008X%2803%2900128-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">An update on in vitro test methods in human hepatic drug biotransformation research: pros and cons</span></div><div class="casAuthors">Brandon, Esther F. A.; Raap, Christiaan D.; Meijerman, Irma; Beijnen, Jos H.; Schellens, Jan H. M.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology and Applied Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">233-246</span>CODEN:
                <span class="NLM_cas:coden">TXAPA9</span>;
        ISSN:<span class="NLM_cas:issn">0041-008X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">A review.  The liver is the predominant organ in which biotransformation of foreign compds. takes place, although other organs may also be involved.  Ideally, an in vitro model for drug biotransformation by the liver should accurately resemble that in vivo.  Several in vitro human liver models have been developed in the past few decades, including supersomes, microsomes, cytosol, S9 fraction, cell lines, transgenic cell lines, primary hepatocytes, liver slices, and perfused liver.  A general advantage of these models is a reduced complexity of the system being studied.  On the other hand, there are several more or less serious specific drawbacks for each model, which prevents their widespread use and acceptance as an alternative for in vivo screening.  This review describes the practical aspects of selected in vitro human liver models, with comparisons between the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPmQxy6k-Zc7Vg90H21EOLACvtfcHk0lgqYdzhiH40EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1yjsbg%253D&md5=8b2ee7b791ef35cc4e95fa754bda5cf4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0041-008X%2803%2900128-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0041-008X%252803%252900128-5%26sid%3Dliteratum%253Aachs%26aulast%3DBrandon%26aufirst%3DE.%2BF.%2BA.%26aulast%3DRaap%26aufirst%3DC.%2BD.%26aulast%3DMeijerman%26aufirst%3DI.%26aulast%3DBeijnen%26aufirst%3DJ.%2BH.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%2BM.%26atitle%3DAn%2520update%2520on%2520in%2520vitro%2520test%2520methods%2520in%2520human%2520hepatic%2520drug%2520biotransformation%2520research%253A%2520pros%2520and%2520cons%26jtitle%3DToxicol.%2520Appl.%2520Pharmacol.%26date%3D2003%26volume%3D189%26spage%3D233%26epage%3D246%26doi%3D10.1016%2FS0041-008X%2803%2900128-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Baillie, T. A.</span><span> </span><span class="NLM_article-title">Metabolism and toxicity of drugs. Two decades of progress in industrial drug metabolism</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span><span class="refDoi"> DOI: 10.1021/tx7002273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx7002273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=129-137&author=T.+A.+Baillie&title=Metabolism+and+toxicity+of+drugs.+Two+decades+of+progress+in+industrial+drug+metabolism&doi=10.1021%2Ftx7002273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism and Toxicity of Drugs. Two Decades of Progress in Industrial Drug Metabolism</span></div><div class="casAuthors">Baillie, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-137</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The science of drug metab. and pharmacokinetics (DMPK) has developed significantly over the past 20 years, and its functional role in today's pharmaceutical industry has matured to the point where DMPK has become an indispensable discipline in support of drug discovery and development.  While contributions to the lead optimization phase of discovery efforts have been particularly noteworthy in helping to select only the best drug candidates for entry into development, it should be recognized that the scope of DMPK spans the continuum of discovery through clin. evaluation and even into the post-marketing phase; as such, the breadth of DMPK's involvement is almost unique in contemporary pharmaceutical research.  This perspective summarizes notable advances in the field, many of which have been made possible by technol. developments in areas such as mol. biol., genetics, and bioanal. chem., and highlights the crit. nature of key partnerships between Drug Metab., Medicinal Chem., and Safety Assessment groups in attempting to advance drug candidates with a low potential for causing adverse events in humans.  Finally, some speculative predictions are made of the future role of DMPK in pharmaceutical research, where current advances in our mechanistic understanding of the mol. processes that control the absorption, disposition, metab., elimination, and toxicity of drugs and their biotransformation products will combine to further enhance the impact of DMPK in drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUrTeLhTDYbVg90H21EOLACvtfcHk0lhKwTQ7dmZHZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlygsrzF&md5=e8f659251bf82cb0b1602855e725c390</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Ftx7002273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx7002273%26sid%3Dliteratum%253Aachs%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26atitle%3DMetabolism%2520and%2520toxicity%2520of%2520drugs.%2520Two%2520decades%2520of%2520progress%2520in%2520industrial%2520drug%2520metabolism%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D129%26epage%3D137%26doi%3D10.1021%2Ftx7002273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Obach, S. R.; Dalvie, D. K.; Walker, G. S.</span><span> </span><span class="NLM_article-title">Identification of drug metabolites</span>. In  <span class="citation_source-book">Encyclopedia of Drug Metabolism and Interactions</span>, <span class="NLM_edition">1</span>st ed.; <span class="NLM_contrib-group">Lyubimov, A.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470921920.edm052" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=1-55&author=S.+R.+Obach&author=D.+K.+Dalvie&author=G.+S.+Walkerauthor=A.+Lyubimov&title=Encyclopedia+of+Drug+Metabolism+and+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2F9780470921920.edm052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470921920.edm052%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DS.%2BR.%26atitle%3DIdentification%2520of%2520drug%2520metabolites%26btitle%3DEncyclopedia%2520of%2520Drug%2520Metabolism%2520and%2520Interactions%26aulast%3DLyubimov%26aufirst%3DA.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012%26spage%3D1%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Stepan, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascitti, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaumont, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalgutkar, A. S.</span><span> </span><span class="NLM_article-title">Metabolism-guided drug design</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">631</span><span class="refDoi"> DOI: 10.1039/c2md20317k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1039%2Fc2md20317k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkslWmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=631&author=A.+F.+Stepanauthor=V.+Mascittiauthor=K.+Beaumontauthor=A.+S.+Kalgutkar&title=Metabolism-guided+drug+design&doi=10.1039%2Fc2md20317k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism-guided drug design</span></div><div class="casAuthors">Stepan, Antonia F.; Mascitti, Vincent; Beaumont, Kevin; Kalgutkar, Amit S.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">631-652</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Preclin. drug metab. studies play a key role in the lead identification and optimization process in drug discovery.  Characterization of the metabolic pathways of new chem. entities is an integral part of drug discovery not only in optimizing clearance properties but also in eliminating potential safety concerns assocd. with the formation of protein and/or DNA-reactive metabolites.  Metab. studies in early discovery have been used to identify metabolic soft spots leading to high metabolic instability, and also in the characterization of active metabolites.  Availability of such information has aided in the rational design of compds. with increased resistance to metab. and overall improvements in oral pharmacokinetics and dose size.  Mechanistic drug metab. studies have proven particularly invaluable in mitigating reactive metabolite risks, which can lead to mutagenicity, time-dependent inactivation of cytochrome P 450 enzymes and/or idiosyncratic adverse drug reactions.  Characterization of stable conjugates derived from bioactivation of small mol. drug candidates provides indirect information on the structure of the reactive metabolite species, thereby providing insight into the bioactivation mechanism and hence a rationale on which to base subsequent chem. intervention strategies.  This review will showcase case studies of metab.-guided drug design using literature and inhouse examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcMOKNkXIUL7Vg90H21EOLACvtfcHk0lhKwTQ7dmZHZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkslWmur0%253D&md5=61c667aa6f5c24de3fe110f384f323bd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1039%2Fc2md20317k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2md20317k%26sid%3Dliteratum%253Aachs%26aulast%3DStepan%26aufirst%3DA.%2BF.%26aulast%3DMascitti%26aufirst%3DV.%26aulast%3DBeaumont%26aufirst%3DK.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26atitle%3DMetabolism-guided%2520drug%2520design%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D631%26doi%3D10.1039%2Fc2md20317k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Fura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, Y.-Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roongta, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphrey, W. G.</span><span> </span><span class="NLM_article-title">Discovering drugs through biological transformation: role of pharmacologically active metabolites in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">4340</span><span class="NLM_x">–</span> <span class="NLM_lpage">4351</span><span class="refDoi"> DOI: 10.1021/jm040066v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040066v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4340-4351&author=A.+Furaauthor=Y.-Z.+Shuauthor=M.+Zhuauthor=R.+L.+Hansonauthor=V.+Roongtaauthor=W.+G.+Humphrey&title=Discovering+drugs+through+biological+transformation%3A+role+of+pharmacologically+active+metabolites+in+drug+discovery&doi=10.1021%2Fjm040066v"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm040066v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040066v%26sid%3Dliteratum%253Aachs%26aulast%3DFura%26aufirst%3DA.%26aulast%3DShu%26aufirst%3DY.-Z.%26aulast%3DZhu%26aufirst%3DM.%26aulast%3DHanson%26aufirst%3DR.%2BL.%26aulast%3DRoongta%26aufirst%3DV.%26aulast%3DHumphrey%26aufirst%3DW.%2BG.%26atitle%3DDiscovering%2520drugs%2520through%2520biological%2520transformation%253A%2520role%2520of%2520pharmacologically%2520active%2520metabolites%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D4340%26epage%3D4351%26doi%3D10.1021%2Fjm040066v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Nassar, A. F.</span><span> </span><span class="NLM_article-title">Improving drug design: considerations for the structural modification process</span>. In  <span class="citation_source-book">Biotransformation and Metabolite Elucidation of Xenobiotics</span>; <span class="NLM_contrib-group">Nassar, A. F.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470890387.ch4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=163-216&author=A.+F.+Nassarauthor=A.+F.+Nassar&title=Biotransformation+and+Metabolite+Elucidation+of+Xenobiotics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2F9780470890387.ch4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470890387.ch4%26sid%3Dliteratum%253Aachs%26aulast%3DNassar%26aufirst%3DA.%2BF.%26atitle%3DImproving%2520drug%2520design%253A%2520considerations%2520for%2520the%2520structural%2520modification%2520process%26btitle%3DBiotransformation%2520and%2520Metabolite%2520Elucidation%2520of%2520Xenobiotics%26aulast%3DNassar%26aufirst%3DA.%2BF.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2010%26spage%3D163%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Clarke, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rindgen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korfmacher, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, K. A.</span><span> </span><span class="NLM_article-title">Systematic LC/MS metabolite identification in drug discovery</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">430A</span><span class="NLM_x">–</span> <span class="NLM_lpage">439A</span><span class="refDoi"> DOI: 10.1021/ac012480y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac012480y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=430A-439A&author=N.+J.+Clarkeauthor=D.+Rindgenauthor=W.+A.+Korfmacherauthor=K.+A.+Cox&title=Systematic+LC%2FMS+metabolite+identification+in+drug+discovery&doi=10.1021%2Fac012480y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fac012480y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac012480y%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DN.%2BJ.%26aulast%3DRindgen%26aufirst%3DD.%26aulast%3DKorfmacher%26aufirst%3DW.%2BA.%26aulast%3DCox%26aufirst%3DK.%2BA.%26atitle%3DSystematic%2520LC%252FMS%2520metabolite%2520identification%2520in%2520drug%2520discovery%26jtitle%3DAnal.%2520Chem.%26date%3D2001%26volume%3D73%26spage%3D430A%26epage%3D439A%26doi%3D10.1021%2Fac012480y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Kamel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prakash, C.</span><span> </span><span class="NLM_article-title">High performance liquid chromatography/atmospheric pressure ionization /tandem mass spectrometry (HPLC/API/MS/MS) in drug metabolism and toxicology studies</span> <span class="citation_source-journal">Curr. Drug Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">837</span><span class="NLM_x">–</span> <span class="NLM_lpage">852</span><span class="refDoi"> DOI: 10.2174/138920006779010593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.2174%2F138920006779010593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=17168686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFynu7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=837-852&author=A.+Kamelauthor=C.+Prakash&title=High+performance+liquid+chromatography%2Fatmospheric+pressure+ionization+%2Ftandem+mass+spectrometry+%28HPLC%2FAPI%2FMS%2FMS%29+in+drug+metabolism+and+toxicology+studies&doi=10.2174%2F138920006779010593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">High performance liquid chromatography/atmospheric pressure ionization/tandem mass spectrometry (HPLC/API/MS/MS) in drug metabolism and toxicology</span></div><div class="casAuthors">Kamel, Amin; Prakash, Chandra</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">837-852</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Studies of the metabolic fate of drugs and other xenobiotics in living systems may be divided into three broad areas: (1) elucidation of biotransformation pathways through identification of circulatory and excretory metabolites (qual. studies); (2) detn. of pharmacokinetics of the parent drug and/or its primary metabolites (quant. studies); and (3) identification of chem.-reactive metabolites, which play a key role as mediators of drug-induced toxicities (mechanistic studies).  Mass spectrometry has been regarded as one of the most important anal. tools in studies of drug metab., pharmacokinetics and biochem. toxicol.  With the com. introduction of new ionization methods such as those based on atm. pressure ionization (API) techniques and the combination of liq. chromatog.-mass spectrometry (LC-MS), it has now become a truly indispensable technique in pharmaceutical research Triple stage quadrupole and ion trap mass spectrometers are presently used for this purpose, because of their sensitivity and selectivity.  API-TOF mass spectrometry has also been very attractive due to its enhanced full-scan sensitivity, scan speed, improved resoln. and ability to measure the accurate masses for protonated mols. and fragment ions.  This review aims to survey the utility of mass spectrometry in drug metab. and toxicol. and to highlight novel applications and future trends in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8gg_jzxAq8rVg90H21EOLACvtfcHk0liUydawVjqKVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFynu7fF&md5=c8955c8b5576387dc37e6b1f4b71795f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F138920006779010593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920006779010593%26sid%3Dliteratum%253Aachs%26aulast%3DKamel%26aufirst%3DA.%26aulast%3DPrakash%26aufirst%3DC.%26atitle%3DHigh%2520performance%2520liquid%2520chromatography%252Fatmospheric%2520pressure%2520ionization%2520%252Ftandem%2520mass%2520spectrometry%2520%2528HPLC%252FAPI%252FMS%252FMS%2529%2520in%2520drug%2520metabolism%2520and%2520toxicology%2520studies%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2006%26volume%3D7%26spage%3D837%26epage%3D852%26doi%3D10.2174%2F138920006779010593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Castro-Perez, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plumb, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granger, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beattie, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joncour, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, A.</span><span> </span><span class="NLM_article-title">Increasing throughput and information content for in vitro drug metabolism experiments using ultra-performance liquid chromatography coupled to a quadrupole time-offlight mass spectrometer</span> <span class="citation_source-journal">Rapid Commun. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">843</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1002/rcm.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2Frcm.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=15723446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitl2ktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=843-848&author=J.+M.+Castro-Perezauthor=R.+Plumbauthor=J.+H.+Grangerauthor=I.+Beattieauthor=K.+Joncourauthor=A.+Wright&title=Increasing+throughput+and+information+content+for+in+vitro+drug+metabolism+experiments+using+ultra-performance+liquid+chromatography+coupled+to+a+quadrupole+time-offlight+mass+spectrometer&doi=10.1002%2Frcm.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing throughput and information content for in vitro drug metabolism experiments using ultra-performance liquid chromatography coupled to a quadrupole time-of-flight mass spectrometer</span></div><div class="casAuthors">Castro-Perez, Jose; Plumb, Robert; Granger, Jennifer H.; Beattie, Iain; Joncour, Karine; Wright, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Rapid Communications in Mass Spectrometry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">843-848</span>CODEN:
                <span class="NLM_cas:coden">RCMSEF</span>;
        ISSN:<span class="NLM_cas:issn">0951-4198</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">The field of drug metab. has been revolutionized by liq. chromatog./mass spectrometry (LC/MS) applications with new technologies such as triple quadrupoles, ion traps and time-of-flight (ToF) instrumentation.  Over the years, these developments have often relied on the improvements to the mass spectrometer hardware and software, which has allowed users to benefit from lower levels of detection and ease-of-use.  One area in which the development pace has been slower is in high-performance liq. chromatog. (HPLC).  In the case of metabolite identification, where there are many challenges due to the complex nature of the biol. matrixes and the diversity of the metabolites produced, there is a need to obtain the most accurate data possible.  Reactive or toxic metabolites need to be detected and identified as early as possible in the drug discovery process, in order to reduce the very costly attrition of compds. in late-phase development.  High-resoln., exact mass measurement plays a very important role in metabolite identification because it allows the elimination of false positives and the detn. of non-trivial metabolites in a much faster throughput environment than any other std. current methodol. available to this field.  By improving the chromatog. resoln., increased peak capacity can be achieved with a redn. in the no. of co-eluting species leading to superior sepns.  The overall enhancement in the chromatog. resoln. and peak capacity is transferred into a net redn. in ion suppression leading to an improvement in the MS sensitivity.  To investigate this, a no. of in vitro samples were analyzed using an ultra-performance liq. chromatog. (UPLC) system, with columns packed with porous 1.7 μm particles, coupled to a hybrid quadrupole time-of-flight (ToF) mass spectrometer.  This technique showed very clear examples for fundamental gains in sensitivity, chromatog. resoln. and speed of anal., which are all important factors for the demands of today's HTS in discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogSBhg6yK0NrVg90H21EOLACvtfcHk0liUydawVjqKVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitl2ktro%253D&md5=6531ea8e926eb1e25cc3ff538f067915</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Frcm.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frcm.1859%26sid%3Dliteratum%253Aachs%26aulast%3DCastro-Perez%26aufirst%3DJ.%2BM.%26aulast%3DPlumb%26aufirst%3DR.%26aulast%3DGranger%26aufirst%3DJ.%2BH.%26aulast%3DBeattie%26aufirst%3DI.%26aulast%3DJoncour%26aufirst%3DK.%26aulast%3DWright%26aufirst%3DA.%26atitle%3DIncreasing%2520throughput%2520and%2520information%2520content%2520for%2520in%2520vitro%2520drug%2520metabolism%2520experiments%2520using%2520ultra-performance%2520liquid%2520chromatography%2520coupled%2520to%2520a%2520quadrupole%2520time-offlight%2520mass%2520spectrometer%26jtitle%3DRapid%2520Commun.%2520Mass%2520Spectrom.%26date%3D2005%26volume%3D19%26spage%3D843%26epage%3D848%26doi%3D10.1002%2Frcm.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Dunn, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broadhurst, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kell, D.</span><span> </span><span class="NLM_article-title">Metabolic profiling of serum using ultra performance liquid chromatography and the LTQ-Orbitrap mass spectrometry system</span> <span class="citation_source-journal">J. Chromatogr. B: Anal. Technol. Biomed. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">871</span><span class="NLM_x">, </span> <span class="NLM_fpage">288</span><span class="NLM_x">–</span> <span class="NLM_lpage">298</span><span class="refDoi"> DOI: 10.1016/j.jchromb.2008.03.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.jchromb.2008.03.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=18420470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps12it78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=871&publication_year=2008&pages=288-298&author=W.+B.+Dunnauthor=D.+Broadhurstauthor=M.+Brownauthor=M.+Bakerauthor=C.+Redmanauthor=L.+Kennyauthor=D.+Kell&title=Metabolic+profiling+of+serum+using+ultra+performance+liquid+chromatography+and+the+LTQ-Orbitrap+mass+spectrometry+system&doi=10.1016%2Fj.jchromb.2008.03.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic profiling of serum using Ultra Performance Liquid Chromatography and the LTQ-Orbitrap mass spectrometry system</span></div><div class="casAuthors">Dunn, Warwick B.; Broadhurst, David; Brown, Marie; Baker, Philip N.; Redman, Christopher W. G.; Kenny, Louise C.; Kell, Douglas B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chromatography B: Analytical Technologies in the Biomedical and Life Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">871</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">288-298</span>CODEN:
                <span class="NLM_cas:coden">JCBAAI</span>;
        ISSN:<span class="NLM_cas:issn">1570-0232</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Advances in anal. instrumentation can provide significant advantages to the vol. and quality of biol. knowledge acquired in metabolomic investigations.  The interfacing of sub-2 μm liq. chromatog. (UPLC ACQUITY) and LTQ-Orbitrap mass spectrometry systems provides many theor. advantages.  The applicability of the interfaced systems was investigated using a simple 11-component metabolite mix and a complex mammalian biofluid, serum.  Metabolites were detected in the metabolite mix with signals that were linear with their concn. over 2.5-3.5 orders of magnitude, with correlation coeffs. greater than 0.993 and limits of detection less than 1 μmol L-1.  Reproducibility of retention time (RSD < 3%) and chromatog. peak area (RSD < 15%) and a high mass accuracy (<2 ppm) were obsd. for 14 QC serum samples interspersed with other serum samples, analyzed over a period of 40 h.  The evaluation of a single deconvolution software package (XCMS) was performed and showed that two parameters (snthresh and bw) provided significant changes to the no. of peaks detected and the peak area reproducibility for the dataset used.  The data were used to indicate possible biomarkers of pre-eclampsia and showed both the instruments and XCMS to be applicable to the reproducible and valid detection of disease biomarkers present in serum.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIk9ttSrvmMrVg90H21EOLACvtfcHk0liUydawVjqKVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps12it78%253D&md5=21274b1752fadea5afe87641c880f145</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.jchromb.2008.03.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jchromb.2008.03.021%26sid%3Dliteratum%253Aachs%26aulast%3DDunn%26aufirst%3DW.%2BB.%26aulast%3DBroadhurst%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DBaker%26aufirst%3DM.%26aulast%3DRedman%26aufirst%3DC.%26aulast%3DKenny%26aufirst%3DL.%26aulast%3DKell%26aufirst%3DD.%26atitle%3DMetabolic%2520profiling%2520of%2520serum%2520using%2520ultra%2520performance%2520liquid%2520chromatography%2520and%2520the%2520LTQ-Orbitrap%2520mass%2520spectrometry%2520system%26jtitle%3DJ.%2520Chromatogr.%2520B%253A%2520Anal.%2520Technol.%2520Biomed.%2520Life%2520Sci.%26date%3D2008%26volume%3D871%26spage%3D288%26epage%3D298%26doi%3D10.1016%2Fj.jchromb.2008.03.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Bowen, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Northen, T. R.</span><span> </span><span class="NLM_article-title">Dealing with the unknown: metabolomics and metabolite atlases</span> <span class="citation_source-journal">J. Am. Soc. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1471</span><span class="NLM_x">–</span> <span class="NLM_lpage">1476</span><span class="refDoi"> DOI: 10.1016/j.jasms.2010.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.jasms.2010.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=20452782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKiur3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=1471-1476&author=B.+P.+Bowenauthor=T.+R.+Northen&title=Dealing+with+the+unknown%3A+metabolomics+and+metabolite+atlases&doi=10.1016%2Fj.jasms.2010.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Dealing with the Unknown: Metabolomics and Metabolite Atlases</span></div><div class="casAuthors">Bowen, Benjamin P.; Northen, Trent R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1471-1476</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Metabolomics is the comprehensive profiling of the small mol. compn. of a biol. sample.  Since metabolites are often the indirect products of gene expression, this approach is being used to provide new insights into a variety of biol. systems (clin., bioenergy, etc.).  A grand challenge for metabolomics is the complexity of the data, which often include many exptl. artifacts.  This is compounded by the tremendous chem. diversity of metabolites.  Identification of each uncharacterized metabolite is in many ways its own puzzle (compared with proteomics, which is based on predictable fragmentation patterns of polypeptides).  Therefore, effective data redn./prioritization strategies are crit. for this rapidly developing field.  Here the authors review liq. chromatog. electrospray ionization mass spectrometry (LC/MS)-based metabolomics, methods for feature finding/prioritization, approaches for identifying unknown metabolites, and construction of method specific Metabolite Atlases'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWvYdYDCQuzrVg90H21EOLACvtfcHk0lhPCfNbm_3B6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKiur3N&md5=d1451b9bc1b1f911950d1c95b67262d5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.jasms.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jasms.2010.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DBowen%26aufirst%3DB.%2BP.%26aulast%3DNorthen%26aufirst%3DT.%2BR.%26atitle%3DDealing%2520with%2520the%2520unknown%253A%2520metabolomics%2520and%2520metabolite%2520atlases%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D2010%26volume%3D21%26spage%3D1471%26epage%3D1476%26doi%3D10.1016%2Fj.jasms.2010.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Kind, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiehn, O.</span><span> </span><span class="NLM_article-title">Advances in structure elucidation of small molecules using mass spectrometry</span> <span class="citation_source-journal">Bioanal. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1007/s12566-010-0015-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1007%2Fs12566-010-0015-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=21289855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A280%3ADC%252BC2sritlCitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=23-60&author=T.+Kindauthor=O.+Fiehn&title=Advances+in+structure+elucidation+of+small+molecules+using+mass+spectrometry&doi=10.1007%2Fs12566-010-0015-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in structure elucidation of small molecules using mass spectrometry</span></div><div class="casAuthors">Kind Tobias; Fiehn Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Bioanalytical reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1-4</span>),
    <span class="NLM_cas:pages">23-60</span>
        ISSN:<span class="NLM_cas:issn">1867-2086</span>.
    </div><div class="casAbstract">The structural elucidation of small molecules using mass spectrometry plays an important role in modern life sciences and bioanalytical approaches.  This review covers different soft and hard ionization techniques and figures of merit for modern mass spectrometers, such as mass resolving power, mass accuracy, isotopic abundance accuracy, accurate mass multiple-stage MS(n) capability, as well as hybrid mass spectrometric and orthogonal chromatographic approaches.  The latter part discusses mass spectral data handling strategies, which includes background and noise subtraction, adduct formation and detection, charge state determination, accurate mass measurements, elemental composition determinations, and complex data-dependent setups with ion maps and ion trees.  The importance of mass spectral library search algorithms for tandem mass spectra and multiple-stage MS(n) mass spectra as well as mass spectral tree libraries that combine multiple-stage mass spectra are outlined.  The successive chapter discusses mass spectral fragmentation pathways, biotransformation reactions and drug metabolism studies, the mass spectral simulation and generation of in silico mass spectra, expert systems for mass spectral interpretation, and the use of computational chemistry to explain gas-phase phenomena.  A single chapter discusses data handling for hyphenated approaches including mass spectral deconvolution for clean mass spectra, cheminformatics approaches and structure retention relationships, and retention index predictions for gas and liquid chromatography.  The last section reviews the current state of electronic data sharing of mass spectra and discusses the importance of software development for the advancement of structure elucidation of small molecules.  ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (doi:10.1007/s12566-010-0015-9) contains supplementary material, which is available to authorized users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTUfAK_fKPrOuWhZTslLrfffW6udTcc2eblqIHVk_fnSLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sritlCitA%253D%253D&md5=91d949d5cc62a9874a98b52359342329</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs12566-010-0015-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12566-010-0015-9%26sid%3Dliteratum%253Aachs%26aulast%3DKind%26aufirst%3DT.%26aulast%3DFiehn%26aufirst%3DO.%26atitle%3DAdvances%2520in%2520structure%2520elucidation%2520of%2520small%2520molecules%2520using%2520mass%2520spectrometry%26jtitle%3DBioanal.%2520Rev.%26date%3D2010%26volume%3D2%26spage%3D23%26epage%3D60%26doi%3D10.1007%2Fs12566-010-0015-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Mortishire-Smith, R. J.; Zhang, H.; Bateman, K. P.</span><span> </span><span class="NLM_article-title">High-resolution mass spectrometry and drug metabolite identification</span>. In  <span class="citation_source-book">Mass Spectrometry in Drug Metabolism and Disposition</span>; <span class="NLM_contrib-group">Lee, M. S.; Zhu, M.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2011</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">407</span><span class="NLM_x">–</span> <span class="NLM_lpage">448</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470929278.ch13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=407-448&author=R.+J.+Mortishire-Smith&author=H.+Zhang&author=K.+P.+Batemanauthor=M.+S.+Lee&author=M.+Zhu&title=Mass+Spectrometry+in+Drug+Metabolism+and+Disposition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2F9780470929278.ch13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470929278.ch13%26sid%3Dliteratum%253Aachs%26aulast%3DMortishire-Smith%26aufirst%3DR.%2BJ.%26atitle%3DHigh-resolution%2520mass%2520spectrometry%2520and%2520drug%2520metabolite%2520identification%26btitle%3DMass%2520Spectrometry%2520in%2520Drug%2520Metabolism%2520and%2520Disposition%26aulast%3DLee%26aufirst%3DM.%2BS.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2011%26spage%3D407%26epage%3D448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Penner, N.; Zgoda-Pols, J.; Prakash, C.</span><span> </span><span class="NLM_article-title">Metabolite identification using mass spectrometry in drug development</span>. In  <span class="citation_source-book">Mass Spectrometry for Drug Discovery and Drug Development</span>; <span class="NLM_contrib-group">Korfmacher, W. A.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9781118516157.ch5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=115-147&author=N.+Penner&author=J.+Zgoda-Pols&author=C.+Prakashauthor=W.+A.+Korfmacher&title=Mass+Spectrometry+for+Drug+Discovery+and+Drug+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2F9781118516157.ch5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118516157.ch5%26sid%3Dliteratum%253Aachs%26aulast%3DPenner%26aufirst%3DN.%26atitle%3DMetabolite%2520identification%2520using%2520mass%2520spectrometry%2520in%2520drug%2520development%26btitle%3DMass%2520Spectrometry%2520for%2520Drug%2520Discovery%2520and%2520Drug%2520Development%26aulast%3DKorfmacher%26aufirst%3DW.%2BA.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2013%26spage%3D115%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Cusack, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koolman, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, U. E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peltier, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piel, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasudevan, A.</span><span> </span><span class="NLM_article-title">Emerging technologies for metabolite generation and structural diversification</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">5471</span><span class="NLM_x">–</span> <span class="NLM_lpage">5483</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2013.08.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=5471-5483&author=K.+P.+Cusackauthor=H.+F.+Koolmanauthor=U.+E.+W.+Langeauthor=H.+M.+Peltierauthor=I.+Pielauthor=A.+Vasudevan&title=Emerging+technologies+for+metabolite+generation+and+structural+diversification&doi=10.1016%2Fj.bmcl.2013.08.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DCusack%26aufirst%3DK.%2BP.%26aulast%3DKoolman%26aufirst%3DH.%2BF.%26aulast%3DLange%26aufirst%3DU.%2BE.%2BW.%26aulast%3DPeltier%26aufirst%3DH.%2BM.%26aulast%3DPiel%26aufirst%3DI.%26aulast%3DVasudevan%26aufirst%3DA.%26atitle%3DEmerging%2520technologies%2520for%2520metabolite%2520generation%2520and%2520structural%2520diversification%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D5471%26epage%3D5483%26doi%3D10.1016%2Fj.bmcl.2013.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">Small sample cryoprobe NMR applications</span>,  <span class="citation_source-book">eMagRes.</span>; <span class="NLM_contrib-group">Harris, R. K.; Wasylishen, R. A.</span>, Eds.; <span class="NLM_publisher-name">Wiley</span>, <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470034590.emrstm1300" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=G.+E.+Martinauthor=R.+K.+Harris&author=R.+A.+Wasylishen&title=eMagRes."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2F9780470034590.emrstm1300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470034590.emrstm1300%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DSmall%2520sample%2520cryoprobe%2520NMR%2520applications%26btitle%3DeMagRes.%26aulast%3DHarris%26aufirst%3DR.%2BK.%26pub%3DWiley%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Hilton, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">An investigation of experimental limits of small sample heteronuclear 2D NMR</span> <span class="citation_source-journal">J. Nat. Prod.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">1465</span><span class="NLM_x">–</span> <span class="NLM_lpage">1469</span><span class="refDoi"> DOI: 10.1021/np100481m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np100481m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2010&pages=1465-1469&author=B.+D.+Hiltonauthor=G.+E.+Martin&title=An+investigation+of+experimental+limits+of+small+sample+heteronuclear+2D+NMR&doi=10.1021%2Fnp100481m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fnp100481m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp100481m%26sid%3Dliteratum%253Aachs%26aulast%3DHilton%26aufirst%3DB.%2BD.%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DAn%2520investigation%2520of%2520experimental%2520limits%2520of%2520small%2520sample%2520heteronuclear%25202D%2520NMR%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2010%26volume%3D73%26spage%3D1465%26epage%3D1469%26doi%3D10.1021%2Fnp100481m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marques, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bermel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">Using pure shift HSQC to characterize microgram samples of drug metabolites</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5450</span><span class="NLM_x">–</span> <span class="NLM_lpage">5453</span><span class="refDoi"> DOI: 10.1016/j.tetlet.2014.06.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.tetlet.2014.06.067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=5450-5453&author=Y.+Liuauthor=M.+D.+Greenauthor=R.+Marquesauthor=T.+Pereiraauthor=R.+Helmyauthor=R.+T.+Williamsonauthor=W.+Bermelauthor=G.+E.+Martin&title=Using+pure+shift+HSQC+to+characterize+microgram+samples+of+drug+metabolites&doi=10.1016%2Fj.tetlet.2014.06.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2014.06.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2014.06.067%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGreen%26aufirst%3DM.%2BD.%26aulast%3DMarques%26aufirst%3DR.%26aulast%3DPereira%26aufirst%3DT.%26aulast%3DHelmy%26aufirst%3DR.%26aulast%3DWilliamson%26aufirst%3DR.%2BT.%26aulast%3DBermel%26aufirst%3DW.%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DUsing%2520pure%2520shift%2520HSQC%2520to%2520characterize%2520microgram%2520samples%2520of%2520drug%2520metabolites%26jtitle%3DTetrahedron%2520Lett.%26date%3D2014%26volume%3D55%26spage%3D5450%26epage%3D5453%26doi%3D10.1016%2Fj.tetlet.2014.06.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Annis, A.; Chuang, C.-C.; Nazef, N.</span><span> </span><span class="NLM_article-title">ALIS: An affinity selection-mass spectrometry system for the discovery and characterization of protein-ligand interactions</span>. In  <span class="citation_source-book">Mass Spectrometry in Medicinal Chemistry: Applications in Drug Discovery</span>; <span class="NLM_contrib-group">Wanner, K.; Höfner, G.</span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH Verlag GmbH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">36</span>, Chapter 3.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9783527610907.ch3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=A.+Annis&author=C.-C.+Chuang&author=N.+Nazefauthor=K.+Wanner&author=G.+H%C3%B6fner&title=Mass+Spectrometry+in+Medicinal+Chemistry%3A+Applications+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2F9783527610907.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527610907.ch3%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DA.%26atitle%3DALIS%253A%2520An%2520affinity%2520selection-mass%2520spectrometry%2520system%2520for%2520the%2520discovery%2520and%2520characterization%2520of%2520protein-ligand%2520interactions%26btitle%3DMass%2520Spectrometry%2520in%2520Medicinal%2520Chemistry%253A%2520Applications%2520in%2520Drug%2520Discovery%26aulast%3DWanner%26aufirst%3DK.%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26date%3D2007%26volume%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nickbarg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebell, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehurst, C. E.</span><span> </span><span class="NLM_article-title">Affinity selection-mass spectrometry screening techniques for small molecule drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">518</span><span class="NLM_x">–</span> <span class="NLM_lpage">526</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2007.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.cbpa.2007.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=17931956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=518-526&author=D.+A.+Annisauthor=E.+Nickbargauthor=X.+Yangauthor=M.+R.+Ziebellauthor=C.+E.+Whitehurst&title=Affinity+selection-mass+spectrometry+screening+techniques+for+small+molecule+drug+discovery&doi=10.1016%2Fj.cbpa.2007.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Affinity selection-mass spectrometry screening techniques for small molecule drug discovery</span></div><div class="casAuthors">Annis, D. Allen; Nickbarg, Elliot; Yang, Xianshu; Ziebell, Michael R.; Whitehurst, Charles E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">518-526</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Affinity selection-mass spectrometry (AS-MS) techniques assess the binding of candidate mols. to immobilized or sol. receptors, and these methods are gaining acceptance in high throughput screening labs. as valuable complements to traditional drug discovery technologies.  A diversity of receptor types have been evaluated by AS-MS, including those that are difficult to screen using traditional biochem. approaches.  AS-MS techniques that couple liq. chromatog.-MS with size-based sepn. methods, such as ultrafiltration, gel permeation, or size-exclusion chromatog., are particularly amenable to the demands of MS-based screening and have demonstrated the greatest success across a broad range of drug targets.  MS measurements of receptor function have many of the same advantages as AS-MS screening and are increasingly used for drug discovery as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBsk6pA3Gl9LVg90H21EOLACvtfcHk0lhfc9YK_ibTtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsbrE&md5=8c98d65db2058eff028a0b879458bf5a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2007.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2007.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DNickbarg%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZiebell%26aufirst%3DM.%2BR.%26aulast%3DWhitehurst%26aufirst%3DC.%2BE.%26atitle%3DAffinity%2520selection-mass%2520spectrometry%2520screening%2520techniques%2520for%2520small%2520molecule%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26spage%3D518%26epage%3D526%26doi%3D10.1016%2Fj.cbpa.2007.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischmann, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duca, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span> </span><span class="NLM_article-title">Discovery of a novel series of CHK1 kinase inhibitors with a distinctive hinge binding mode</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1021/ml200249h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml200249h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC38XpslGntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=123-128&author=X.+Huangauthor=C.+C.+Chengauthor=T.+O.+Fischmannauthor=J.+S.+Ducaauthor=X.+Yangauthor=M.+Richardsauthor=G.+W.+Shipps&title=Discovery+of+a+novel+series+of+CHK1+kinase+inhibitors+with+a+distinctive+hinge+binding+mode&doi=10.1021%2Fml200249h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Series of CHK1 Kinase Inhibitors with a Distinctive Hinge Binding Mode</span></div><div class="casAuthors">Huang, Xiaohua; Cheng, Cliff C.; Fischmann, Thierry O.; Duca, Jose S.; Yang, Xianshu; Richards, Matthew; Shipps, Gerald W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">123-128</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of CHK1 inhibitors with a distinctive hinge binding mode, exemplified by 2-aryl-N-(2-(piperazin-1-yl)phenyl)thiazole-4-carboxamide, was discovered through high-throughput screening using the affinity selection-mass spectrometry (AS-MS)-based Automated Ligand Identification System (ALIS) platform.  Structure-based ligand design and optimization led to significant improvements in potency to the single digit nanomolar range and hundred-fold selectivity against CDK2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphbnP8TZLFsrVg90H21EOLACvtfcHk0lhfc9YK_ibTtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpslGntQ%253D%253D&md5=0033fbe5837cc4c6287c4a5428665247</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fml200249h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml200249h%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520series%2520of%2520CHK1%2520kinase%2520inhibitors%2520with%2520a%2520distinctive%2520hinge%2520binding%2520mode%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D123%26epage%3D128%26doi%3D10.1021%2Fml200249h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Whitehurst, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taremi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojcik, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strizki, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nickbarg, E.</span><span> </span><span class="NLM_article-title">Application of affinity selection-mass spectrometry assays to purification and affinity-based screening of the chemokine receptor CXCR4</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">478</span><span class="refDoi"> DOI: 10.2174/138620712800563945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.2174%2F138620712800563945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=22651846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFalt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2012&pages=473-478&author=C.+E.+Whitehurstauthor=Z.+Yaoauthor=D.+Murphyauthor=M.+Zhangauthor=S.+Taremiauthor=L.+Wojcikauthor=J.+M.+Strizkiauthor=J.+D.+Brackenauthor=C.+C.+Chengauthor=X.+Yangauthor=G.+W.+Shippsauthor=M.+Ziebellauthor=E.+Nickbarg&title=Application+of+affinity+selection-mass+spectrometry+assays+to+purification+and+affinity-based+screening+of+the+chemokine+receptor+CXCR4&doi=10.2174%2F138620712800563945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Application of affinity selection-mass spectrometry assays to purification and affinity-based screening of the chemokine receptor CXCR4</span></div><div class="casAuthors">Whitehurst, Charles E.; Yao, Zhiping; Murphy, Denise; Zhang, Mingxuan; Taremi, Shane; Wojcik, Lisa; Strizki, Julie M.; Bracken, Jack D.; Cheng, Cliff C.; Yang, Xianshu; Shipps, Gerald W., Jr.; Ziebell, Michael; Nickbarg, Elliott</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">473-485</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Affinity selection-mass spectrometry (AS-MS) is a sensitive technol. for identifying small mols. that bind to target proteins, and assays enabled by AS-MS can be used to delineate relative binding affinities of ligands for proteins.  "Indirect" AS-MS assays employ size-exclusion techniques to sep. target-ligand complexes from unbound ligands, and target-assocd. ligands are then specifically detected by liq. chromatog. mass spectrometry.  We report how indirect AS-MS binding assays with known ref. control compds. were used as guideposts for development of an optimized purifn. method for CXCR4, a G-protein coupled chemokine receptor, for which we sought novel antagonists.  The CXCR4 purifn. method that was developed was amenable to scale-up and enabled the screening of purified recombinant human CXCR4 against a large combinatorial library of small mols. by high throughput indirect AS-MS.  The screen resulted in the discovery of new ligands that competed off binding of ref. compds. to CXCR4 in AS-MS binding assays and that antagonized SDF1α-triggered responses and CXCR4-mediated HIV1 viral uptake in cell-based assays.  This report provides a methodol. paradigm whereby indirect AS-MS-based ligand binding assays may be used to guide optimal integral membrane protein purifn. methods that enable downstream affinity selection-based applications such as high throughput AS-MS screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnKOiSohXGXLVg90H21EOLACvtfcHk0liZ1Ajg_UrmNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFalt7o%253D&md5=cba5688a5bad20d0cfca1119e8b95fcb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.2174%2F138620712800563945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620712800563945%26sid%3Dliteratum%253Aachs%26aulast%3DWhitehurst%26aufirst%3DC.%2BE.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DMurphy%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DTaremi%26aufirst%3DS.%26aulast%3DWojcik%26aufirst%3DL.%26aulast%3DStrizki%26aufirst%3DJ.%2BM.%26aulast%3DBracken%26aufirst%3DJ.%2BD.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DZiebell%26aufirst%3DM.%26aulast%3DNickbarg%26aufirst%3DE.%26atitle%3DApplication%2520of%2520affinity%2520selection-mass%2520spectrometry%2520assays%2520to%2520purification%2520and%2520affinity-based%2520screening%2520of%2520the%2520chemokine%2520receptor%2520CXCR4%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2012%26volume%3D15%26spage%3D473%26epage%3D478%26doi%3D10.2174%2F138620712800563945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovici-Muller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqui, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gowen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Windsor, W. T.</span><span> </span><span class="NLM_article-title">Method for quantitative protein-ligand affinity measurements in compound mixtures</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">4538</span><span class="NLM_x">–</span> <span class="NLM_lpage">4542</span><span class="refDoi"> DOI: 10.1021/ac0702701</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac0702701" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1WhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2007&pages=4538-4542&author=D.+A.+Annisauthor=G.+W.+Shippsauthor=Y.+Dengauthor=J.+Popovici-Mullerauthor=M.+A.+Siddiquiauthor=P.+J.+Curranauthor=M.+Gowenauthor=W.+T.+Windsor&title=Method+for+quantitative+protein-ligand+affinity+measurements+in+compound+mixtures&doi=10.1021%2Fac0702701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Method for quantitative protein-ligand affinity measurements in compound mixtures</span></div><div class="casAuthors">Annis, D. Allen; Shipps, Gerald W., Jr.; Deng, Yongqi; Popovici-Mueller, Janeta; Siddiqui, M. Arshad; Curran, Patrick J.; Gowen, Matthew; Windsor, William T.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4538-4542</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This manuscript describes an affinity selection-mass spectrometry (AS-MS) method for quant. protein-ligand binding affinity (Kd) measurements in large compd. libraries.  The ability of a titrant ligand to displace a target-bound library member-as measured by MS-reveals the affinity ranking of the mixt. component relative to "internal affinity calibrants", compds. of known affinity for the target.  This technique does not require that the precise concn. of each ligand is known; therefore, unpurified products of mixt.-based combinatorial synthesis may be used for affinity optimization and developing structure-activity relationships.  The method is demonstrated for a series of ligands to the important oncol. target CDK2 that were discovered by AS-MS screening of combinatorial libraries against the basal form of the protein.  AS-MS displacement curves for select hits were acquired over a range of compd. concns., confirming that binding affinity measurement results are concn.-insensitive.  These hits were evaluated in pools of purified compds. to verify the method's applicability to hit triage in large chem. libraries.  The method was further tested using unpurified, mixt.-based combinatorial libraries of >1000 compds., yielding results that mirror those obtained from mixts. of purified compds.  The technique was then used to identify optimized CDK2 ligands from compd. mixts., quant. measure their affinities, and establish structure-activity relationships among these drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopGVdmMNSdmbVg90H21EOLACvtfcHk0liZ1Ajg_UrmNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1WhtLk%253D&md5=638ab38a5bc9674a4ccdb1977c721b87</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fac0702701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac0702701%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DPopovici-Muller%26aufirst%3DJ.%26aulast%3DSiddiqui%26aufirst%3DM.%2BA.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DGowen%26aufirst%3DM.%26aulast%3DWindsor%26aufirst%3DW.%2BT.%26atitle%3DMethod%2520for%2520quantitative%2520protein-ligand%2520affinity%2520measurements%2520in%2520compound%2520mixtures%26jtitle%3DAnal.%2520Chem.%26date%3D2007%26volume%3D79%26spage%3D4538%26epage%3D4542%26doi%3D10.1021%2Fac0702701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Deng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">English, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dayananth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruza, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirschmeier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Windsor, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samater, A. A.</span><span> </span><span class="NLM_article-title">Discovery of novel, dual mechanism ERK inhibitors by affinity selection screening of an inactive kinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8817</span><span class="NLM_x">–</span> <span class="NLM_lpage">8826</span><span class="refDoi"> DOI: 10.1021/jm500847m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500847m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCht77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8817-8826&author=D.+Dengauthor=G.+W.+Shippsauthor=A.+Cooperauthor=J.+M.+Englishauthor=D.+A.+Annisauthor=D.+Carrauthor=Y.+Nanauthor=T.+Wangauthor=H.+Y.+Zhuauthor=C.+C.+Chengauthor=P.+Dayananthauthor=A.+W.+Hruzaauthor=J.+Xiaoauthor=W.+Jinauthor=P.+Kirschmeierauthor=W.+T.+Windsorauthor=A.+A.+Samater&title=Discovery+of+novel%2C+dual+mechanism+ERK+inhibitors+by+affinity+selection+screening+of+an+inactive+kinase&doi=10.1021%2Fjm500847m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel, Dual Mechanism ERK Inhibitors by Affinity Selection Screening of an Inactive Kinase</span></div><div class="casAuthors">Deng, Yongqi; Shipps, Gerald W.; Cooper, Alan; English, Jessie M.; Annis, D. Allen; Carr, Donna; Nan, Yang; Wang, Tong; Zhu, Hugh Y.; Chuang, Cheng-Chi; Dayananth, Priya; Hruza, Alan W.; Xiao, Li; Jin, Weihong; Kirschmeier, Paul; Windsor, William T.; Samatar, Ahmed A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8817-8826</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An affinity-based mass spectrometry screening technol. was used to identify novel binders to both nonphosphorylated and phosphorylated ERK2.  Screening of inactive ERK2 identified a pyrrolidine analog 1 that bound to both nonphosphorylated and phosphorylated ERK2 and inhibited ERK2 kinase activity.  Chem. optimization identified compd. 4 as a novel, potent, and highly selective ERK1,2 inhibitor which not only demonstrated inhibition of phosphorylation of ERK substrate p90RSK but also demonstrated inhibition of ERK1,2 phosphorylation on the activation loop.  X-ray cocrystallog. revealed that upon binding of compd. 4 to ERK2, Tyr34 undergoes a rotation (flip) along with a shift in the poly-Gly rich loop to create a new binding pocket into which 4 can bind.  This new binding mode represents a novel mechanism by which high affinity ATP-competitive compds. may achieve excellent kinase selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2kaW1Z2DFXbVg90H21EOLACvtfcHk0liZ1Ajg_UrmNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCht77P&md5=10f1922c9c65371564ed0ad1f05cc845</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm500847m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500847m%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DD.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DCooper%26aufirst%3DA.%26aulast%3DEnglish%26aufirst%3DJ.%2BM.%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DCarr%26aufirst%3DD.%26aulast%3DNan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DH.%2BY.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DDayananth%26aufirst%3DP.%26aulast%3DHruza%26aufirst%3DA.%2BW.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DW.%26aulast%3DKirschmeier%26aufirst%3DP.%26aulast%3DWindsor%26aufirst%3DW.%2BT.%26aulast%3DSamater%26aufirst%3DA.%2BA.%26atitle%3DDiscovery%2520of%2520novel%252C%2520dual%2520mechanism%2520ERK%2520inhibitors%2520by%2520affinity%2520selection%2520screening%2520of%2520an%2520inactive%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8817%26epage%3D8826%26doi%3D10.1021%2Fjm500847m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Parker, D. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharipour, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewnani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherer, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichert, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geissler, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonatore, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellsworth, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balkovec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenlee, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, H. B.</span><span> </span><span class="NLM_article-title">Rapid development of two factor IXa inhibitors from hit to lead</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2321</span><span class="NLM_x">–</span> <span class="NLM_lpage">2325</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2015.04.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=2321-2325&author=D.+L.+Parkerauthor=S.+Walshauthor=B.+Liauthor=E.+Kimauthor=A.+Sharipourauthor=C.+Smithauthor=Y.-H.+Chenauthor=R.+Bergerauthor=B.+Harperauthor=T.+Zhangauthor=M.+Parkauthor=M.+Shuauthor=J.+Wuauthor=J.+Xuauthor=S.+Dewnaniauthor=E.+C.+Shererauthor=A.+Hruzaauthor=P.+Reichertauthor=W.+Geisslerauthor=L.+Sonatoreauthor=K.+Ellsworthauthor=J.+Balkovecauthor=W.+Greenleeauthor=H.+B.+Wood&title=Rapid+development+of+two+factor+IXa+inhibitors+from+hit+to+lead&doi=10.1016%2Fj.bmcl.2015.04.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DD.%2BL.%26aulast%3DWalsh%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DE.%26aulast%3DSharipour%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.-H.%26aulast%3DBerger%26aufirst%3DR.%26aulast%3DHarper%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DM.%26aulast%3DShu%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDewnani%26aufirst%3DS.%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DReichert%26aufirst%3DP.%26aulast%3DGeissler%26aufirst%3DW.%26aulast%3DSonatore%26aufirst%3DL.%26aulast%3DEllsworth%26aufirst%3DK.%26aulast%3DBalkovec%26aufirst%3DJ.%26aulast%3DGreenlee%26aufirst%3DW.%26aulast%3DWood%26aufirst%3DH.%2BB.%26atitle%3DRapid%2520development%2520of%2520two%2520factor%2520IXa%2520inhibitors%2520from%2520hit%2520to%2520lead%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D2321%26epage%3D2325%26doi%3D10.1016%2Fj.bmcl.2015.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andre, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellsworth, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geissler, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hruza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, D. L.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichert, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherer, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonatore, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tschirret-Guth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nolting, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orr, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campeau, L.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Araki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishimura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurada, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, H. B.</span><span> </span><span class="NLM_article-title">Development of a novel class of potent and selective FIXa inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">4945</span><span class="NLM_x">–</span> <span class="NLM_lpage">4949</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2015.04.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2015.04.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=25978966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFygs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=4945-4949&author=T.+Zhangauthor=P.+Andreauthor=T.+J.+Batemanauthor=Y.-H.+Chenauthor=K.+Desaiauthor=K.+Ellsworthauthor=W.+M.+Geisslerauthor=L.+Guoauthor=A.+Hruzaauthor=T.+Jianauthor=D.+Mengauthor=D.+L.+Parkerauthor=X.+Qianauthor=P.+Reichertauthor=E.+C.+Shererauthor=M.+Shuauthor=C.+J.+Smithauthor=L.+M.+Sonatoreauthor=R.+Tschirret-Guthauthor=A.+F.+Noltingauthor=R.+Orrauthor=L.-C.+Campeauauthor=K.+Arakiauthor=T.+Nishimuraauthor=I.+Sakuradaauthor=H.+B.+Wood&title=Development+of+a+novel+class+of+potent+and+selective+FIXa+inhibitors&doi=10.1016%2Fj.bmcl.2015.04.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a novel class of potent and selective FIXa inhibitors</span></div><div class="casAuthors">Zhang, Ting; Andre, Patrick; Bateman, Thomas J.; Chen, Yi-Heng; Desai, Kunal; Ellsworth, Kenneth; Geissler, Wayne M.; Guo, Liangqin; Hruza, Alan; Jian, Tianying; Meng, Dongfang; Parker, Dann L., Jr.; Qian, Xiaoxia; Reichert, Paul; Sherer, Edward C.; Shu, Min; Smith, Cameron J.; Sonatore, Lisa M.; Tschirret-Guth, Richard; Nolting, Andrew F.; Orr, Robert; Campeau, Louis-Charles; Araki, Kazuto; Nishimura, Teruyuki; Sakurada, Isao; Wood, Harold B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4945-4949</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Using structure based drug design (SBDD), a novel class of potent coagulation Factor IXa (FIXa) inhibitors was designed and synthesized.  High selectivity over FXa inhibition was achieved.  Selected compds. demonstrated oral bioavailability in rat IV/PO pharmacokinetic (PK) studies.  Finally, the pharmacodynamics (PD) of this class of mols. was evaluated in Thrombin Generation Assay (TGA) in Corn Trypsin Inhibitor (CTI) citrated human plasma and demonstrated characteristics of a FIXa inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW3NBfPoV5zrVg90H21EOLACvtfcHk0liYPJVGJQ-izA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFygs78%253D&md5=d4944b0798611a811a5ee7d4d7282b5f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.057%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DAndre%26aufirst%3DP.%26aulast%3DBateman%26aufirst%3DT.%2BJ.%26aulast%3DChen%26aufirst%3DY.-H.%26aulast%3DDesai%26aufirst%3DK.%26aulast%3DEllsworth%26aufirst%3DK.%26aulast%3DGeissler%26aufirst%3DW.%2BM.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DJian%26aufirst%3DT.%26aulast%3DMeng%26aufirst%3DD.%26aulast%3DParker%26aufirst%3DD.%2BL.%26aulast%3DQian%26aufirst%3DX.%26aulast%3DReichert%26aufirst%3DP.%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DShu%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DC.%2BJ.%26aulast%3DSonatore%26aufirst%3DL.%2BM.%26aulast%3DTschirret-Guth%26aufirst%3DR.%26aulast%3DNolting%26aufirst%3DA.%2BF.%26aulast%3DOrr%26aufirst%3DR.%26aulast%3DCampeau%26aufirst%3DL.-C.%26aulast%3DAraki%26aufirst%3DK.%26aulast%3DNishimura%26aufirst%3DT.%26aulast%3DSakurada%26aufirst%3DI.%26aulast%3DWood%26aufirst%3DH.%2BB.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520class%2520of%2520potent%2520and%2520selective%2520FIXa%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D4945%26epage%3D4949%26doi%3D10.1016%2Fj.bmcl.2015.04.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">You, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verkade, J. G.</span><span> </span><span class="NLM_article-title">P(<i>i</i>-BuNCH<sub>2</sub>CH<sub>2</sub>)<sub>3</sub>N: An efficient ligand for the direct α-arylation of nitriles with aryl bromides</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">8003</span><span class="NLM_x">–</span> <span class="NLM_lpage">8007</span><span class="refDoi"> DOI: 10.1021/jo034779h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo034779h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnsFyrtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=8003-8007&author=J.+Youauthor=J.+G.+Verkade&title=P%28i-BuNCH2CH2%293N%3A+An+efficient+ligand+for+the+direct+%CE%B1-arylation+of+nitriles+with+aryl+bromides&doi=10.1021%2Fjo034779h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">P(i-BuNCH2CH2)3N: An Efficient Ligand for the Direct α-Arylation of Nitriles with Aryl Bromides</span></div><div class="casAuthors">You, Jingsong; Verkade, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8003-8007</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new catalyst system for the synthesis of α-aryl-substituted nitriles is reported.  The bicyclic triaminophosphine P(i-BuNCH2CH2)3N (I) serves as an efficient and versatile ligand for the palladium-catalyzed direct α-arylation of nitriles with aryl bromides.  Using ligand I, Et cyanoacetate and primary as well as secondary nitriles are efficiently coupled with a wide variety of aryl bromides possessing electron-rich, electron-poor, electron-neutral, and sterically hindered groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGgTlIYPn6nrVg90H21EOLACvtfcHk0liYPJVGJQ-izA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnsFyrtbg%253D&md5=2fd29d6ebbab1eb2130447aa7ce7eaf7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjo034779h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo034779h%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DVerkade%26aufirst%3DJ.%2BG.%26atitle%3DP%2528i-BuNCH2CH2%25293N%253A%2520An%2520efficient%2520ligand%2520for%2520the%2520direct%2520%25CE%25B1-arylation%2520of%2520nitriles%2520with%2520aryl%2520bromides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2003%26volume%3D68%26spage%3D8003%26epage%3D8007%26doi%3D10.1021%2Fjo034779h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Khurana, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kukreja, G.</span><span> </span><span class="NLM_article-title">Reduction of nitriles to primary amines with nickel boride at ambient temperature</span> <span class="citation_source-journal">Synth. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">1265</span><span class="NLM_x">–</span> <span class="NLM_lpage">1269</span><span class="refDoi"> DOI: 10.1081/SCC-120003619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1081%2FSCC-120003619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvV2msb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2002&pages=1265-1269&author=J.+M.+Khuranaauthor=G.+Kukreja&title=Reduction+of+nitriles+to+primary+amines+with+nickel+boride+at+ambient+temperature&doi=10.1081%2FSCC-120003619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid reduction of nitriles to primary amines with nickel boride at ambient temperature</span></div><div class="casAuthors">Khurana, Jitender M.; Kukreja, Gagan</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1265-1269</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">Redn. of a variety of nitriles to their corresponding primary amines can be achieved with Ni boride generated in situ in dry EtOH at ambient temp.  The redns. are very rapid and chemoselective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppr9uclHIZ-bVg90H21EOLACvtfcHk0liE8O9dnEwVRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvV2msb8%253D&md5=cf018f6e95e8a32c66e8fbccb6372c52</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1081%2FSCC-120003619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1081%252FSCC-120003619%26sid%3Dliteratum%253Aachs%26aulast%3DKhurana%26aufirst%3DJ.%2BM.%26aulast%3DKukreja%26aufirst%3DG.%26atitle%3DReduction%2520of%2520nitriles%2520to%2520primary%2520amines%2520with%2520nickel%2520boride%2520at%2520ambient%2520temperature%26jtitle%3DSynth.%2520Commun.%26date%3D2002%26volume%3D32%26spage%3D1265%26epage%3D1269%26doi%3D10.1081%2FSCC-120003619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="note"><p class="first last"><i>In vitro</i> clearance data of <b>7</b>: human CLint (liver microsomes) = 29.8 mL/min/kg; human CLint (hepatocyte) = 21.0 mL/min/kg; rat CLint (liver microsomes) = 116 mL/min/kg; rat CLint (hepatocyte) = 101 mL/min/kg.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Athanasopoulos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felsch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalghatgi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orminati, J.-P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosner, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaddupathy, G. R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyler, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vilenchik, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wintner, E. A.</span><span> </span><span class="NLM_article-title">An affinity selection–mass spectrometry method for the identification of small molecule ligands from self-encoded combinatorial libraries: Discovery of a novel antagonist of <i>E. coli</i> dihydrofolate reductase</span> <span class="citation_source-journal">Int. J. Mass Spectrom.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">238</span><span class="NLM_x">, </span> <span class="NLM_fpage">77</span><span class="NLM_x">–</span> <span class="NLM_lpage">83</span><span class="refDoi"> DOI: 10.1016/j.ijms.2003.11.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.ijms.2003.11.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpslyisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=238&publication_year=2004&pages=77-83&author=D.+A.+Annisauthor=J.+Athanasopoulosauthor=P.+J.+Curranauthor=J.+S.+Felschauthor=K.+Kalghatgiauthor=W.+H.+Leeauthor=H.+M.+Nashauthor=J.-P.+A.+Orminatiauthor=K.+E.+Rosnerauthor=G.+W.+Shippsauthor=G.+R.+A.+Thaddupathyauthor=A.+N.+Tylerauthor=L.+Vilenchikauthor=C.+R.+Wagnerauthor=E.+A.+Wintner&title=An+affinity+selection%E2%80%93mass+spectrometry+method+for+the+identification+of+small+molecule+ligands+from+self-encoded+combinatorial+libraries%3A+Discovery+of+a+novel+antagonist+of+E.+coli+dihydrofolate+reductase&doi=10.1016%2Fj.ijms.2003.11.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">An affinity selection-mass spectrometry method for the identification of small molecule ligands from self-encoded combinatorial libraries. Discovery of a novel antagonist of E. coli dihydrofolate reductase</span></div><div class="casAuthors">Annis, D. Allen; Athanasopoulos, John; Curran, Patrick J.; Felsch, Jason S.; Kalghatgi, Krishna; Lee, William H.; Nash, Huw M.; Orminati, Jean-Paul A.; Rosner, Kristin E.; Shipps, Gerald W.; Thaddupathy, G. R. A.; Tyler, Andrew N.; Vilenchik, Lev; Wagner, Carston R.; Wintner, Edward A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Mass Spectrometry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">238</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">77-83</span>CODEN:
                <span class="NLM_cas:coden">IMSPF8</span>;
        ISSN:<span class="NLM_cas:issn">1387-3806</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The NeoGenesis Automated Ligand Identification System (ALIS), an affinity selection-mass spectrometry (AS-MS) process consisting of a rapid size-exclusion chromatog. stage integrated with reverse-phase chromatog., electrospray mass spectrometry, and novel data searching algorithms, was used to screen mass-encoded, 2500-member combinatorial libraries, leading to the discovery of a novel, bioactive ligand for the anti-infective target Escherichia coli dihydrofolate reductase (DHFR).  Synthesis of the mass-encoded, ligand-contg. library, discussion of the deconvolution process for verifying the structure of the ligand through independent synthesis and screening in a small mixt. (sub-library) format, and ALIS-MS/MS techniques to assign its regioisomeric connectivity are presented.  ALIS-based competition expts. between the newly discovered ligand and other, known DHFR ligands, and biol. activity assessments with stereo- and regioisomers of the hit compd. confirm its DHFR-specific biol. activity.  The method described requires no foreknowledge of the structure or biochem. of the protein target, consumes less than 1 μg protein to screen >2500 compds. in a single expt., and enables screening of >250,000 compds. per system per day.  These advantages highlight the potential of the ALIS method for drug discovery against genomic targets with unknown biol. function, as well as validated targets for which traditional discovery efforts have failed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF9t0__7HggLVg90H21EOLACvtfcHk0liE8O9dnEwVRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpslyisLk%253D&md5=2f0e7e5c2f76f28ef6ac697e76d24092</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ijms.2003.11.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijms.2003.11.022%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DAthanasopoulos%26aufirst%3DJ.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DFelsch%26aufirst%3DJ.%2BS.%26aulast%3DKalghatgi%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DNash%26aufirst%3DH.%2BM.%26aulast%3DOrminati%26aufirst%3DJ.-P.%2BA.%26aulast%3DRosner%26aufirst%3DK.%2BE.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DThaddupathy%26aufirst%3DG.%2BR.%2BA.%26aulast%3DTyler%26aufirst%3DA.%2BN.%26aulast%3DVilenchik%26aufirst%3DL.%26aulast%3DWagner%26aufirst%3DC.%2BR.%26aulast%3DWintner%26aufirst%3DE.%2BA.%26atitle%3DAn%2520affinity%2520selection%25E2%2580%2593mass%2520spectrometry%2520method%2520for%2520the%2520identification%2520of%2520small%2520molecule%2520ligands%2520from%2520self-encoded%2520combinatorial%2520libraries%253A%2520Discovery%2520of%2520a%2520novel%2520antagonist%2520of%2520E.%2520coli%2520dihydrofolate%2520reductase%26jtitle%3DInt.%2520J.%2520Mass%2520Spectrom.%26date%3D2004%26volume%3D238%26spage%3D77%26epage%3D83%26doi%3D10.1016%2Fj.ijms.2003.11.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawahata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soucy, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soriano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, T.</span><span> </span><span class="NLM_article-title">Discovery and optimization of antibacterial AccC inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">6507</span><span class="NLM_x">–</span> <span class="NLM_lpage">6514</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.10.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2009.10.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=19875284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGisLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6507-6514&author=C.+C.+Chengauthor=G.+W.+Shippsauthor=Z.+Yangauthor=B.+Sunauthor=N.+Kawahataauthor=K.+A.+Soucyauthor=A.+Sorianoauthor=P.+Orthauthor=L.+Xiaoauthor=P.+Mannauthor=T.+Black&title=Discovery+and+optimization+of+antibacterial+AccC+inhibitors&doi=10.1016%2Fj.bmcl.2009.10.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of antibacterial AccC inhibitors</span></div><div class="casAuthors">Cheng, Cliff C.; Shipps, Gerald W.; Yang, Zhiwei; Sun, Binyuan; Kawahata, Noriyuki; Soucy, Kyle A.; Soriano, Aileen; Orth, Peter; Xiao, Li; Mann, Paul; Black, Todd</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6507-6514</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Benzimidazolecarboxamides and imidazopyridinecarboxamides such as I (R first defined; R1 = H; X = N) and I (R second defined; R1 = O2N; X = CH) are prepd. by soln. and solid-phase methods as inhibitors of biotin carboxylase (AccC) for potential use as antibacterial agents.  For example, I (R first defined; R1 = H; X = N) and I (R second defined; R1 = O2N; X = CH) are potent and selective inhibitors of bacterial AccC with IC50 values of 20 nM and 10 nM, resp., and MIC values of 0.8 μg/mL against a sensitized strain of Escherichia coli (HS294 E. coli).  The structures of two benzimidazolecarboxamides bound to biotin carboxylase are detd. by X-ray crystallog.; the structure of an imidazopyridinecarboxamide bound to the active site of biotin carboxylase is detd. by computer-aided modeling calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgL8pY9SdU07Vg90H21EOLACvtfcHk0libeXHIqci4cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGisLvJ&md5=e811f3f617b9125bd852a4d65ff6e6c2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.10.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.10.057%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DKawahata%26aufirst%3DN.%26aulast%3DSoucy%26aufirst%3DK.%2BA.%26aulast%3DSoriano%26aufirst%3DA.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DMann%26aufirst%3DP.%26aulast%3DBlack%26aufirst%3DT.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520antibacterial%2520AccC%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6507%26epage%3D6514%26doi%3D10.1016%2Fj.bmcl.2009.10.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyss, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soucy, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.-k.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrawal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.-C.</span><span> </span><span class="NLM_article-title">Pyridine carboxamides: potent palm site inhibitors of HCV NS5B polymerase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">466</span><span class="NLM_x">–</span> <span class="NLM_lpage">471</span><span class="refDoi"> DOI: 10.1021/ml100128h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100128h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=466-471&author=C.+C.+Chengauthor=X.+Huangauthor=G.+W.+Shippsauthor=Y.-S.+Wangauthor=D.+F.+Wyssauthor=K.+A.+Soucyauthor=C.-k.+Jiangauthor=S.+Agrawalauthor=E.+Ferrariauthor=Z.+Heauthor=H.-C.+Huang&title=Pyridine+carboxamides%3A+potent+palm+site+inhibitors+of+HCV+NS5B+polymerase&doi=10.1021%2Fml100128h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fml100128h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100128h%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DWang%26aufirst%3DY.-S.%26aulast%3DWyss%26aufirst%3DD.%2BF.%26aulast%3DSoucy%26aufirst%3DK.%2BA.%26aulast%3DJiang%26aufirst%3DC.-k.%26aulast%3DAgrawal%26aufirst%3DS.%26aulast%3DFerrari%26aufirst%3DE.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DH.-C.%26atitle%3DPyridine%2520carboxamides%253A%2520potent%2520palm%2520site%2520inhibitors%2520of%2520HCV%2520NS5B%2520polymerase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D466%26epage%3D471%26doi%3D10.1021%2Fml100128h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawahata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesburg, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duca, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandouveres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.-k.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrawal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.-C.</span><span> </span><span class="NLM_article-title">Inhibitors of hepatitis C virus polymerase: Synthesis and characterization of novel 2-oxy-6-fluoro-<i>N</i>-((S)-1-hydroxy-3-phenylpropan-2-yl)-benzamides</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2119</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.02.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2010.02.054" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=2119-2124&author=C.+C.+Chengauthor=G.+W.+Shippsauthor=Z.+Yangauthor=N.+Kawahataauthor=C.+A.+Lesburgauthor=J.+S.+Ducaauthor=J.+Bandouveresauthor=J.+D.+Brackenauthor=C.-k.+Jiangauthor=S.+Agrawalauthor=E.+Ferrariauthor=H.-C.+Huang&title=Inhibitors+of+hepatitis+C+virus+polymerase%3A+Synthesis+and+characterization+of+novel+2-oxy-6-fluoro-N-%28%28S%29-1-hydroxy-3-phenylpropan-2-yl%29-benzamides&doi=10.1016%2Fj.bmcl.2010.02.054"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.02.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.02.054%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DKawahata%26aufirst%3DN.%26aulast%3DLesburg%26aufirst%3DC.%2BA.%26aulast%3DDuca%26aufirst%3DJ.%2BS.%26aulast%3DBandouveres%26aufirst%3DJ.%26aulast%3DBracken%26aufirst%3DJ.%2BD.%26aulast%3DJiang%26aufirst%3DC.-k.%26aulast%3DAgrawal%26aufirst%3DS.%26aulast%3DFerrari%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DH.-C.%26atitle%3DInhibitors%2520of%2520hepatitis%2520C%2520virus%2520polymerase%253A%2520Synthesis%2520and%2520characterization%2520of%2520novel%25202-oxy-6-fluoro-N-%2528%2528S%2529-1-hydroxy-3-phenylpropan-2-yl%2529-benzamides%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D2119%26epage%3D2124%26doi%3D10.1016%2Fj.bmcl.2010.02.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovici-Muller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosner, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Girijavallabhan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butkiewicz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cable, M.</span><span> </span><span class="NLM_article-title">Aminothiazole inhibitors of HCV RNA polymerase</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2004.10.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2004.10.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=115-119&author=G.+W.+Shippsauthor=Y.+Dengauthor=T.+Wangauthor=J.+Popovici-Mullerauthor=P.+J.+Curranauthor=K.+E.+Rosnerauthor=A.+B.+Cooperauthor=V.+Girijavallabhanauthor=N.+Butkiewiczauthor=M.+Cable&title=Aminothiazole+inhibitors+of+HCV+RNA+polymerase&doi=10.1016%2Fj.bmcl.2004.10.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2004.10.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2004.10.024%26sid%3Dliteratum%253Aachs%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DPopovici-Muller%26aufirst%3DJ.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DRosner%26aufirst%3DK.%2BE.%26aulast%3DCooper%26aufirst%3DA.%2BB.%26aulast%3DGirijavallabhan%26aufirst%3DV.%26aulast%3DButkiewicz%26aufirst%3DN.%26aulast%3DCable%26aufirst%3DM.%26atitle%3DAminothiazole%2520inhibitors%2520of%2520HCV%2520RNA%2520polymerase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D115%26epage%3D119%26doi%3D10.1016%2Fj.bmcl.2004.10.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovici-Müller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqui, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gowen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Windsor, W. T.</span><span> </span><span class="NLM_article-title">Method for quantitative protein-ligand affinity measurements in compound mixtures</span> <span class="citation_source-journal">Anal. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">4538</span><span class="NLM_x">–</span> <span class="NLM_lpage">4542</span><span class="refDoi"> DOI: 10.1021/ac0702701</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac0702701" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1WhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2007&pages=4538-4542&author=D.+A.+Annisauthor=G.+W.+Shippsauthor=Y.+Dengauthor=J.+Popovici-M%C3%BCllerauthor=M.+A.+Siddiquiauthor=P.+J.+Curranauthor=M.+Gowenauthor=W.+T.+Windsor&title=Method+for+quantitative+protein-ligand+affinity+measurements+in+compound+mixtures&doi=10.1021%2Fac0702701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Method for quantitative protein-ligand affinity measurements in compound mixtures</span></div><div class="casAuthors">Annis, D. Allen; Shipps, Gerald W., Jr.; Deng, Yongqi; Popovici-Mueller, Janeta; Siddiqui, M. Arshad; Curran, Patrick J.; Gowen, Matthew; Windsor, William T.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4538-4542</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This manuscript describes an affinity selection-mass spectrometry (AS-MS) method for quant. protein-ligand binding affinity (Kd) measurements in large compd. libraries.  The ability of a titrant ligand to displace a target-bound library member-as measured by MS-reveals the affinity ranking of the mixt. component relative to "internal affinity calibrants", compds. of known affinity for the target.  This technique does not require that the precise concn. of each ligand is known; therefore, unpurified products of mixt.-based combinatorial synthesis may be used for affinity optimization and developing structure-activity relationships.  The method is demonstrated for a series of ligands to the important oncol. target CDK2 that were discovered by AS-MS screening of combinatorial libraries against the basal form of the protein.  AS-MS displacement curves for select hits were acquired over a range of compd. concns., confirming that binding affinity measurement results are concn.-insensitive.  These hits were evaluated in pools of purified compds. to verify the method's applicability to hit triage in large chem. libraries.  The method was further tested using unpurified, mixt.-based combinatorial libraries of >1000 compds., yielding results that mirror those obtained from mixts. of purified compds.  The technique was then used to identify optimized CDK2 ligands from compd. mixts., quant. measure their affinities, and establish structure-activity relationships among these drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopGVdmMNSdmbVg90H21EOLACvtfcHk0lgU2ijLiyoHYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1WhtLk%253D&md5=638ab38a5bc9674a4ccdb1977c721b87</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fac0702701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac0702701%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DPopovici-M%25C3%25BCller%26aufirst%3DJ.%26aulast%3DSiddiqui%26aufirst%3DM.%2BA.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DGowen%26aufirst%3DM.%26aulast%3DWindsor%26aufirst%3DW.%2BT.%26atitle%3DMethod%2520for%2520quantitative%2520protein-ligand%2520affinity%2520measurements%2520in%2520compound%2520mixtures%26jtitle%3DAnal.%2520Chem.%26date%3D2007%26volume%3D79%26spage%3D4538%26epage%3D4542%26doi%3D10.1021%2Fac0702701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Lahue, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seghezzi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herbst, R.</span><span> </span><span class="NLM_article-title">Substituted benzimidazoles: a novel chemotype for small molecule hksp inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3405</span><span class="NLM_x">–</span> <span class="NLM_lpage">3409</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.05.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2009.05.040" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=3405-3409&author=B.+R.+Lahueauthor=Y.+Maauthor=G.+W.+Shippsauthor=W.+Seghezziauthor=R.+Herbst&title=Substituted+benzimidazoles%3A+a+novel+chemotype+for+small+molecule+hksp+inhibitors&doi=10.1016%2Fj.bmcl.2009.05.040"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.05.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.05.040%26sid%3Dliteratum%253Aachs%26aulast%3DLahue%26aufirst%3DB.%2BR.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DSeghezzi%26aufirst%3DW.%26aulast%3DHerbst%26aufirst%3DR.%26atitle%3DSubstituted%2520benzimidazoles%253A%2520a%2520novel%2520chemotype%2520for%2520small%2520molecule%2520hksp%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D3405%26epage%3D3409%26doi%3D10.1016%2Fj.bmcl.2009.05.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Lan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowalski, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, C.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rojas-Triana, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bober, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hedrick, J. A.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of FABP4/5 ameliorate dyslipidemia but not insulin resistance in mice with diet-induced obesity</span> <span class="citation_source-journal">J. Lipid Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">646</span><span class="NLM_x">–</span> <span class="NLM_lpage">656</span><span class="refDoi"> DOI: 10.1194/jlr.M012757</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1194%2Fjlr.M012757" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=21296956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2011&pages=646-656&author=H.+Lanauthor=C.+C.+Chengauthor=T.+J.+Kowalskiauthor=L.+Pangauthor=L.+Shanauthor=C.-C.+Chuangauthor=J.+Jacksonauthor=A.+Rojas-Trianaauthor=L.+Boberauthor=L.+Liuauthor=J.+Voigtauthor=P.+Orthauthor=X.+Yangauthor=G.+W.+Shippsauthor=J.+A.+Hedrick&title=Small-molecule+inhibitors+of+FABP4%2F5+ameliorate+dyslipidemia+but+not+insulin+resistance+in+mice+with+diet-induced+obesity&doi=10.1194%2Fjlr.M012757"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of FABP4/5 ameliorate dyslipidemia but not insulin resistance in mice with diet-induced obesity</span></div><div class="casAuthors">Lan, Hong; Cheng, Cliff C.; Kowalski, Timothy J.; Pang, Ling; Shan, Lixin; Chuang, Cheng-Chi; Jackson, James; Rojas-Triana, Alberto; Bober, Loretta; Liu, Li; Voigt, Johannes; Orth, Peter; Yang, Xianshu; Shipps, Gerald W., Jr.; Hedrick, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Lipid Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">646-656</span>CODEN:
                <span class="NLM_cas:coden">JLPRAW</span>;
        ISSN:<span class="NLM_cas:issn">0022-2275</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology, Inc.</span>)
        </div><div class="casAbstract">Fatty acid binding protein-4 (FABP4) and FABP5 are two closely related FA binding proteins expressed primarily in adipose tissue and/or macrophages.  The small-mol. FABP4 inhibitor BMS309403 was previously reported to improve insulin sensitivity in leptin-deficient Lepob/Lepob (ob/ob) mice.  However, this compd. was not extensively characterized in the more physiol. relevant animal model of mice with diet-induced obesity (DIO).  Here, we report the discovery and characterization of a novel series of FABP4/5 dual inhibitors represented by Compds. 1-3.  Compared with BMS309403, the compds. had significant in vitro potency toward both FABP4 and FABP5.  In cell-based assays, Compds. 2 and 3 were more potent than BMS309403 to inhibit lipolysis in 3T3-L1 adipocytes and in primary human adipocytes.  They also inhibited MCP-1 release from THP-1 macrophages as well as from primary human macrophages.  When chronically administered to DIO mice, BMS309403 and Compd. 3 reduced plasma triglyceride and free FA levels.  Compd. 3 reduced plasma free FAs at a lower dose level than BMS309403.  However, no significant change was obsd. in insulin, glucose, or glucose tolerance.  Our results indicate that the FABP4/5 inhibitors ameliorate dyslipidemia but not insulin resistance in DIO mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyZVUoUyaoRLVg90H21EOLACvtfcHk0lgU2ijLiyoHYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOlt74%253D&md5=3e1d382cd198d4904a8d9b3974cf195d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M012757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M012757%26sid%3Dliteratum%253Aachs%26aulast%3DLan%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DKowalski%26aufirst%3DT.%2BJ.%26aulast%3DPang%26aufirst%3DL.%26aulast%3DShan%26aufirst%3DL.%26aulast%3DChuang%26aufirst%3DC.-C.%26aulast%3DJackson%26aufirst%3DJ.%26aulast%3DRojas-Triana%26aufirst%3DA.%26aulast%3DBober%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DVoigt%26aufirst%3DJ.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DHedrick%26aufirst%3DJ.%2BA.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520FABP4%252F5%2520ameliorate%2520dyslipidemia%2520but%2520not%2520insulin%2520resistance%2520in%2520mice%2520with%2520diet-induced%2520obesity%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2011%26volume%3D52%26spage%3D646%26epage%3D656%26doi%3D10.1194%2Fjlr.M012757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, C.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCarter, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nazef, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurzeja, R. J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span> </span><span class="NLM_article-title">Inhibitors of the lipid phosphatase SHIP2 discovered by high throughput affinity selection-mass spectrometry screening of combinatorial libraries</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">771</span><span class="refDoi"> DOI: 10.2174/138620709789104870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.2174%2F138620709789104870" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=19531013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlKntbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2009&pages=760-771&author=D.+A.+Annisauthor=C.+C.+Chengauthor=C.-C.+Chuangauthor=J.+D.+McCarterauthor=H.+M.+Nashauthor=N.+Nazefauthor=T.+Roweauthor=R.+J.+M.+Kurzejaauthor=G.+W.+Shipps&title=Inhibitors+of+the+lipid+phosphatase+SHIP2+discovered+by+high+throughput+affinity+selection-mass+spectrometry+screening+of+combinatorial+libraries&doi=10.2174%2F138620709789104870"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of the lipid phosphatase SHIP2 discovered by high throughput affinity selection-mass spectrometry screening of combinatorial libraries</span></div><div class="casAuthors">Annis, D. Allen; Cheng, Cliff C.; Chuang, Cheng-Chi; McCarter, John D.; Nash, Huw M.; Nazef, Naim; Rowe, Todd; Kurzeja, Robert J. M.; Shipps, Gerald W., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">760-771</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This manuscript describes the discovery and characterization of inhibitors of the lipid phosphatase SHIP2, an important target for the treatment of Type 2 diabetes, using the Automated Ligand Identification System.  ALIS is an affinity selection-mass spectrometry platform for label-free, high throughput screening of mixt.-based combinatorial libraries.  We detail the mass-encoded synthesis of a library that yielded NGD-61338, a pyrazole-based SHIP2 inhibitor.  Quant. ALIS affinity measurements and inhibition of SHIP2 enzymic activity indicate that this compd. has micromolar binding affinity and inhibitory activity for this target.  This inhibitor, which does not contain a phosphatase "warhead," binds the active site of SHIP2 as detd. by ALIS-based competition expts. with the enzyme's natural substrate, phosphatidylinositol 3,4,5-triphosphate (PIP3).  Structure-activity relationships for NGD-61338 and two other ligand classes discovered by ALIS screening were explored using a combination of combinatorial library synthesis and ALIS-enabled affinity ranking in compd. mixts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon_FH7j0n5r7Vg90H21EOLACvtfcHk0lgmYEoD4xZkwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlKntbnM&md5=f603df2bb224c7f45e8e6a22359ad3d6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.2174%2F138620709789104870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620709789104870%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DChuang%26aufirst%3DC.-C.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DNash%26aufirst%3DH.%2BM.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DKurzeja%26aufirst%3DR.%2BJ.%2BM.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26atitle%3DInhibitors%2520of%2520the%2520lipid%2520phosphatase%2520SHIP2%2520discovered%2520by%2520high%2520throughput%2520affinity%2520selection-mass%2520spectrometry%2520screening%2520of%2520combinatorial%2520libraries%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2009%26volume%3D12%26spage%3D760%26epage%3D771%26doi%3D10.2174%2F138620709789104870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spacciapoli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyss, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Achab, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soucy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montavon, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehurst, C. E.</span><span> </span><span class="NLM_article-title">Discovery and hitto-lead optimization of non-ATP competitive MK2 (MAPKAPK2) inhibitors</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">632</span><span class="NLM_x">–</span> <span class="NLM_lpage">637</span><span class="refDoi"> DOI: 10.1021/ml200113y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml200113y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=632-637&author=X.+Huangauthor=G.+W.+Shippsauthor=C.+C.+Chengauthor=P.+Spacciapoliauthor=X.+Zhangauthor=M.+A.+McCoyauthor=D.+F.+Wyssauthor=X.+Yangauthor=A.+Achabauthor=K.+Soucyauthor=D.+K.+Montavonauthor=D.+M.+Murphyauthor=C.+E.+Whitehurst&title=Discovery+and+hitto-lead+optimization+of+non-ATP+competitive+MK2+%28MAPKAPK2%29+inhibitors&doi=10.1021%2Fml200113y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fml200113y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml200113y%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DSpacciapoli%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMcCoy%26aufirst%3DM.%2BA.%26aulast%3DWyss%26aufirst%3DD.%2BF.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DAchab%26aufirst%3DA.%26aulast%3DSoucy%26aufirst%3DK.%26aulast%3DMontavon%26aufirst%3DD.%2BK.%26aulast%3DMurphy%26aufirst%3DD.%2BM.%26aulast%3DWhitehurst%26aufirst%3DC.%2BE.%26atitle%3DDiscovery%2520and%2520hitto-lead%2520optimization%2520of%2520non-ATP%2520competitive%2520MK2%2520%2528MAPKAPK2%2529%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D2%26spage%3D632%26epage%3D637%26doi%3D10.1021%2Fml200113y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Rosner, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orth, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belanger, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curran, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Girijavallabhan, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavey, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovici-Muller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ting, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaccaro, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozlowski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madison, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKittrick, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piwinski, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shih, N.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqui, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strickland, C. O.</span><span> </span><span class="NLM_article-title">The discovery of novel tartrate-based TNF-α converting enzyme (TACE) inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1189</span><span class="NLM_x">–</span> <span class="NLM_lpage">1193</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.bmcl.2009.12.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1189-1193&author=K.+E.+Rosnerauthor=Z.+Guoauthor=P.+Orthauthor=G.+W.+Shippsauthor=D.+B.+Belangerauthor=T.+Y.+Chanauthor=P.+J.+Curranauthor=C.+Daiauthor=Y.+Dengauthor=V.+M.+Girijavallabhanauthor=L.+Hongauthor=B.+J.+Laveyauthor=J.+F.+Leeauthor=D.+Liauthor=Z.+Liuauthor=J.+Popovici-Mullerauthor=P.+C.+Tingauthor=H.+Vaccaroauthor=L.+Wangauthor=T.+Wangauthor=W.+Yuauthor=G.+Zhouauthor=X.+Niuauthor=J.+Sunauthor=J.+A.+Kozlowskiauthor=D.+J.+Lundellauthor=V.+Madisonauthor=B.+McKittrickauthor=J.+J.+Piwinskiauthor=N.-Y.+Shihauthor=M.+A.+Siddiquiauthor=C.+O.+Strickland&title=The+discovery+of+novel+tartrate-based+TNF-%CE%B1+converting+enzyme+%28TACE%29+inhibitors&doi=10.1016%2Fj.bmcl.2009.12.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DRosner%26aufirst%3DK.%2BE.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DBelanger%26aufirst%3DD.%2BB.%26aulast%3DChan%26aufirst%3DT.%2BY.%26aulast%3DCurran%26aufirst%3DP.%2BJ.%26aulast%3DDai%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DGirijavallabhan%26aufirst%3DV.%2BM.%26aulast%3DHong%26aufirst%3DL.%26aulast%3DLavey%26aufirst%3DB.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BF.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DPopovici-Muller%26aufirst%3DJ.%26aulast%3DTing%26aufirst%3DP.%2BC.%26aulast%3DVaccaro%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DG.%26aulast%3DNiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DKozlowski%26aufirst%3DJ.%2BA.%26aulast%3DLundell%26aufirst%3DD.%2BJ.%26aulast%3DMadison%26aufirst%3DV.%26aulast%3DMcKittrick%26aufirst%3DB.%26aulast%3DPiwinski%26aufirst%3DJ.%2BJ.%26aulast%3DShih%26aufirst%3DN.-Y.%26aulast%3DSiddiqui%26aufirst%3DM.%2BA.%26aulast%3DStrickland%26aufirst%3DC.%2BO.%26atitle%3DThe%2520discovery%2520of%2520novel%2520tartrate-based%2520TNF-%25CE%25B1%2520converting%2520enzyme%2520%2528TACE%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D1189%26epage%3D1193%26doi%3D10.1016%2Fj.bmcl.2009.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Whitehurst, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nazef, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spacciapoli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebell, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipps, G. W.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felsch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, H. M.</span><span> </span><span class="NLM_article-title">Discovery and characterization of orthosteric and allosteric muscarinic M2 acetylcholine receptor ligands by affinity selection–mass spectrometry</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">194</span><span class="NLM_x">–</span> <span class="NLM_lpage">207</span><span class="refDoi"> DOI: 10.1177/1087057105284340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1177%2F1087057105284340" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=194-207&author=C.+E.+Whitehurstauthor=N.+Nazefauthor=D.+A.+Annisauthor=Y.+Houauthor=D.+M.+Murphyauthor=P.+Spacciapoliauthor=Z.+Yaoauthor=M.+R.+Ziebellauthor=C.+C.+Chengauthor=G.+W.+Shippsauthor=J.+S.+Felschauthor=D.+Lauauthor=H.+M.+Nash&title=Discovery+and+characterization+of+orthosteric+and+allosteric+muscarinic+M2+acetylcholine+receptor+ligands+by+affinity+selection%E2%80%93mass+spectrometry&doi=10.1177%2F1087057105284340"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1177%2F1087057105284340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057105284340%26sid%3Dliteratum%253Aachs%26aulast%3DWhitehurst%26aufirst%3DC.%2BE.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DMurphy%26aufirst%3DD.%2BM.%26aulast%3DSpacciapoli%26aufirst%3DP.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DZiebell%26aufirst%3DM.%2BR.%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DFelsch%26aufirst%3DJ.%2BS.%26aulast%3DLau%26aufirst%3DD.%26aulast%3DNash%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520orthosteric%2520and%2520allosteric%2520muscarinic%2520M2%2520acetylcholine%2520receptor%2520ligands%2520by%2520affinity%2520selection%25E2%2580%2593mass%2520spectrometry%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2005%26volume%3D11%26spage%3D194%26epage%3D207%26doi%3D10.1177%2F1087057105284340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Annis, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nazef, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuang, C.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nash, H. M.</span><span> </span><span class="NLM_article-title">A general technique to rank protein-ligand binding affinities and determine allosteric versus direct binding site competition in compound mixtures</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">15495</span><span class="NLM_x">–</span> <span class="NLM_lpage">15503</span><span class="refDoi"> DOI: 10.1021/ja048365x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja048365x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptlOisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=15495-15503&author=D.+A.+Annisauthor=N.+Nazefauthor=C.-C.+Chuangauthor=M.+P.+Scottauthor=H.+M.+Nash&title=A+general+technique+to+rank+protein-ligand+binding+affinities+and+determine+allosteric+versus+direct+binding+site+competition+in+compound+mixtures&doi=10.1021%2Fja048365x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A general technique to rank protein-ligand binding affinities and determine allosteric versus direct binding site competition in compound mixtures</span></div><div class="casAuthors">Annis, D. Allen; Nazef, Naim; Chuang, Cheng-Chi; Scott, Margaret Porter; Nash, Huw M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">15495-15503</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To realize the full potential of combinatorial chem.-based drug discovery, generic and efficient tools must be developed that apply the strengths of diversity-oriented chem. synthesis to the identification and optimization of lead compds. for disease-assocd. protein targets.  We report an affinity selection-mass spectrometry (AS-MS) method for protein-ligand affinity ranking and the classification of ligands by binding site.  The method incorporates the following steps: (1) an affinity selection stage, where protein-binding compds. are selected from pools of ligands in the presence of varying concns. of a competitor ligand, (2) a first chromatog. stage to sep. unbound ligands from protein-ligand complexes, and (3) a second chromatog. stage to dissoc. the ligands from the complexes for identification and quantification by MS.  The ability of the competitor ligand to displace a target-bound library member, as measured by MS, reveals the binding site classification and affinity ranking of the mixt. components.  The technique requires no radiolabel incorporation or direct biochem. assay, no modification or immobilization of the compds. or target protein, and all reaction components, including any buffers or cofactors required for protein stability, are free in soln.  We demonstrate the method for several compds. of wide structural variety against representatives of the most important protein classes in contemporary drug discovery, including novel ATP-competitive and allosteric inhibitors of the Akt-1 (PKB) and Zap-70 kinases, and previously undisclosed antagonists of the M2 muscarinic acetylcholine receptor, a G-protein coupled receptor (GPCR).  The theor. basis of the technique is analyzed math., allowing quant. estn. of binding affinities and, in the case of allosteric interaction, abs. detn. of binding cooperativity.  The method is readily applicable to high-throughput screening hit triage, combinatorial library-based affinity optimization, and developing structure-activity relationships among multiple ligands to a given receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpREa-teJ7tC7Vg90H21EOLACvtfcHk0ljyDIPS4oF0jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptlOisrk%253D&md5=bee110f0d9f40e75b5e22d120238a742</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja048365x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja048365x%26sid%3Dliteratum%253Aachs%26aulast%3DAnnis%26aufirst%3DD.%2BA.%26aulast%3DNazef%26aufirst%3DN.%26aulast%3DChuang%26aufirst%3DC.-C.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DNash%26aufirst%3DH.%2BM.%26atitle%3DA%2520general%2520technique%2520to%2520rank%2520protein-ligand%2520binding%2520affinities%2520and%2520determine%2520allosteric%2520versus%2520direct%2520binding%2520site%2520competition%2520in%2520compound%2520mixtures%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D15495%26epage%3D15503%26doi%3D10.1021%2Fja048365x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Brandstetter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lollar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bode, W.</span><span> </span><span class="NLM_article-title">X-ray structure of clotting factor IXa: active site and module structure related to Xase activity and hemophilia B</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">9796</span><span class="NLM_x">–</span> <span class="NLM_lpage">9800</span><span class="refDoi"> DOI: 10.1073/pnas.92.21.9796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1073%2Fpnas.92.21.9796" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=9796-9800&author=H.+Brandstetterauthor=M.+Bauerauthor=R.+Huberauthor=P.+Lollarauthor=W.+Bode&title=X-ray+structure+of+clotting+factor+IXa%3A+active+site+and+module+structure+related+to+Xase+activity+and+hemophilia+B&doi=10.1073%2Fpnas.92.21.9796"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.21.9796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.21.9796%26sid%3Dliteratum%253Aachs%26aulast%3DBrandstetter%26aufirst%3DH.%26aulast%3DBauer%26aufirst%3DM.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DLollar%26aufirst%3DP.%26aulast%3DBode%26aufirst%3DW.%26atitle%3DX-ray%2520structure%2520of%2520clotting%2520factor%2520IXa%253A%2520active%2520site%2520and%2520module%2520structure%2520related%2520to%2520Xase%2520activity%2520and%2520hemophilia%2520B%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1995%26volume%3D92%26spage%3D9796%26epage%3D9800%26doi%3D10.1073%2Fpnas.92.21.9796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Sturzebecher, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopetzki, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bode, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopfner, K.-P.</span><span> </span><span class="NLM_article-title">Dramatic enhancement of the catalytic activity of coagulation factor IXa by alcohols</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">412</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">300</span><span class="refDoi"> DOI: 10.1016/S0014-5793(97)00811-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2FS0014-5793%2897%2900811-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=9256238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADyaK2sXks12ltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=412&publication_year=1997&pages=295-300&author=J.+Sturzebecherauthor=E.+Kopetzkiauthor=W.+Bodeauthor=K.-P.+Hopfner&title=Dramatic+enhancement+of+the+catalytic+activity+of+coagulation+factor+IXa+by+alcohols&doi=10.1016%2FS0014-5793%2897%2900811-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Dramatic enhancement of the catalytic activity of coagulation factor IXa by alcohols</span></div><div class="casAuthors">Sturzebecher, Jorg; Kopetzki, Erhard; Bode, Wolfram; Hopfner, Karl-Peter</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">412</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">295-300</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The coagulation factor IXa (FIXa) exhibits a very weak proteolytic activity towards natural or synthetic substrates.  Upon complex formation with its cofactor FVIIIa and Ca2+-mediated binding to phospholipid membranes, FIXa becomes a very potent activator of FX.  The presence of FVIIIa has no effect on the cleavage of peptide substrates by FIXa, however.  Several alcs. dramatically enhance the catalytic activity of human FIXa towards synthetic substrates.  Substrates with the tripeptidyl moiety R-D-Xxx-Gly-Arg are esp. susceptible to the enhanced FIXa catalysis.  Maximal increase up to 20-fold was measured in the presence of ethylene glycol.  Alcs. may modify the conformation of FIXa rendering the active-site cleft more easily accessible to tripeptide substrates with a hydrophobic residue in the P3-position.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWImWN_8rRNLVg90H21EOLACvtfcHk0ljyDIPS4oF0jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXks12ltLg%253D&md5=a25070c173abc67d51fc20982fa97eb1</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2897%2900811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252897%252900811-9%26sid%3Dliteratum%253Aachs%26aulast%3DSturzebecher%26aufirst%3DJ.%26aulast%3DKopetzki%26aufirst%3DE.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DHopfner%26aufirst%3DK.-P.%26atitle%3DDramatic%2520enhancement%2520of%2520the%2520catalytic%2520activity%2520of%2520coagulation%2520factor%2520IXa%2520by%2520alcohols%26jtitle%3DFEBS%2520Lett.%26date%3D1997%26volume%3D412%26spage%3D295%26epage%3D300%26doi%3D10.1016%2FS0014-5793%2897%2900811-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Riley, M. D.; Huang, S.; Caceres-Cortes, J.; Johnson, K.; Shu, Y.-Z.</span><span> </span><span class="NLM_article-title">Nuclear magnetic resonance spectroscopy for the study of drug metabolism</span>. In  <span class="citation_source-book">Encyclopedia of Drug Metabolism and Interactions</span>, <span class="NLM_contrib-group">Lyubimov, A. V.</span>, Ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">3</span>, pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470921920.edm102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=1-29&author=M.+D.+Riley&author=S.+Huang&author=J.+Caceres-Cortes&author=K.+Johnson&author=Y.-Z.+Shuauthor=A.+V.+Lyubimov&title=Encyclopedia+of+Drug+Metabolism+and+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2F9780470921920.edm102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470921920.edm102%26sid%3Dliteratum%253Aachs%26aulast%3DRiley%26aufirst%3DM.%2BD.%26atitle%3DNuclear%2520magnetic%2520resonance%2520spectroscopy%2520for%2520the%2520study%2520of%2520drug%2520metabolism%26btitle%3DEncyclopedia%2520of%2520Drug%2520Metabolism%2520and%2520Interactions%26aulast%3DLyubimov%26aufirst%3DA.%2BV.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012%26volume%3D3%26spage%3D1%26epage%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">Small volume high sensitivity NMR probes</span>. In  <span class="citation_source-book">Annu. Rep. NMR Spectrosc.</span>, <span class="NLM_contrib-group">Webb, G. A.</span>, Ed.; <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">56</span>, pp  <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2FS0066-4103%2805%2956001-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=1-99&author=G.+E.+Martinauthor=G.+A.+Webb&title=Annu.+Rep.+NMR+Spectrosc."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2FS0066-4103%2805%2956001-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0066-4103%252805%252956001-0%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DSmall%2520volume%2520high%2520sensitivity%2520NMR%2520probes%26btitle%3DAnnu.%2520Rep.%2520NMR%2520Spectrosc.%26aulast%3DWebb%26aufirst%3DG.%2BA.%26pub%3DElsevier%26date%3D2005%26volume%3D56%26spage%3D1%26epage%3D99" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Martin, G. E.</span><span> </span><span class="NLM_article-title">Application of cryogenic NMR probe technology for the identification of low level impurities in pharmaceuticals</span>. In  <span class="citation_source-book">Handbook of Modern Magnetic Resonance</span>, <span class="NLM_contrib-group">Webb, G. A.</span>, Ed.; <span class="NLM_publisher-name">Elsevier</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">II</span>, pp  <span class="NLM_fpage">1187</span><span class="NLM_x">–</span> <span class="NLM_lpage">1194</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1007%2F1-4020-3910-7_129" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=1187-1194&author=G.+E.+Martinauthor=G.+A.+Webb&title=Handbook+of+Modern+Magnetic+Resonance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1007%2F1-4020-3910-7_129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F1-4020-3910-7_129%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%2BE.%26atitle%3DApplication%2520of%2520cryogenic%2520NMR%2520probe%2520technology%2520for%2520the%2520identification%2520of%2520low%2520level%2520impurities%2520in%2520pharmaceuticals%26btitle%3DHandbook%2520of%2520Modern%2520Magnetic%2520Resonance%26aulast%3DWebb%26aufirst%3DG.%2BA.%26pub%3DElsevier%26date%3D2006%26volume%3DII%26spage%3D1187%26epage%3D1194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Adams, R. W.</span><span> </span><span class="NLM_article-title">Pure shift NMR spectroscopy</span> <span class="citation_source-journal">eMagRes.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">310</span><span class="refDoi"> DOI: 10.1002/9780470034590.emrstm1362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2F9780470034590.emrstm1362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVGlurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=295-310&author=R.+W.+Adams&title=Pure+shift+NMR+spectroscopy&doi=10.1002%2F9780470034590.emrstm1362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Pure shift NMR spectroscopy</span></div><div class="casAuthors">Adams, Ralph W.</div><div class="citationInfo"><span class="NLM_cas:title">eMagRes</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-310</span>CODEN:
                <span class="NLM_cas:coden">EMAGCE</span>;
        ISSN:<span class="NLM_cas:issn">2055-6101</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Pure shift NMR spectroscopy, or broadband homonuclear decoupled NMR spectroscopy, produces NMR spectra that contain only chem. shift information.  In a pure shift NMR spectrum the effects of J-coupling are hidden so that all multiplets are collapsed to singlets.  Pure shift methods can deliver a significant improvement in resoln. of NMR signals and greatly reduce signal overlap.  Pure shift NMR can be achieved using direct or indirect acquisition; for direct acquisition, real-time- or interferogram-based FID collection can be used.  The range of pure shift methods includes those that refocus the effects of J-coupling (BIRD, Zangger-Sterk, band selective decoupling, and anti z-COSY); manipulate the evolution of chem. shift within a const. time delay to keep the J-evolution in an indirect acquisition const.; or ext. a 1-D spectrum from a 2-D J dataset.  The methods and their mechanisms in weakly coupled systems are described using the product operator formalism.  Examples are provided for a range of applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMUf_z3uFhILVg90H21EOLACvtfcHk0liiu1IAnT1Ffg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVGlurw%253D&md5=39bb6a2d92a36a7fe8d56301a073484b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2F9780470034590.emrstm1362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470034590.emrstm1362%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DR.%2BW.%26atitle%3DPure%2520shift%2520NMR%2520spectroscopy%26jtitle%3DeMagRes.%26date%3D2014%26volume%3D3%26spage%3D295%26epage%3D310%26doi%3D10.1002%2F9780470034590.emrstm1362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Zangger, K.</span><span> </span><span class="NLM_article-title">Pure shift NMR</span> <span class="citation_source-journal">Prog. Nucl. Magn. Reson. Spectrosc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">86–87</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1016/j.pnmrs.2015.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2Fj.pnmrs.2015.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=25919196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtValt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86%E2%80%9387&publication_year=2015&pages=1-20&author=K.+Zangger&title=Pure+shift+NMR&doi=10.1016%2Fj.pnmrs.2015.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Pure shift NMR</span></div><div class="casAuthors">Zangger, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Nuclear Magnetic Resonance Spectroscopy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">86-87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-20</span>CODEN:
                <span class="NLM_cas:coden">PNMRAT</span>;
        ISSN:<span class="NLM_cas:issn">0079-6565</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although scalar-coupling provides important structural information, the resulting signal splittings significantly reduce the resoln. of NMR spectra.  Limited resoln. is a particular problem in proton NMR expts., resulting in part from the limited proton chem. shift range (∼10 ppm) but even more from the splittings due to scalar coupling to nearby protons.  "Pure shift" NMR spectroscopy (also known as broadband homonuclear decoupling) has been developed for disentangling overlapped proton NMR spectra.  The resulting spectra are considerably simplified as they consist of single lines, reminiscent of proton-decoupled C-13 spectra at natural abundance, with no multiplet structure.  The different approaches to obtaining pure shift spectra are reviewed here and several applications presented.  Pure shift spectra are esp. useful for highly overlapped proton spectra, as found for example in reaction mixts., natural products and biomacromols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokhOn2iyQOQbVg90H21EOLACvtfcHk0liiu1IAnT1Ffg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtValt7c%253D&md5=6457afd7f42e3cc78062b5c666a9f36a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.pnmrs.2015.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pnmrs.2015.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DZangger%26aufirst%3DK.%26atitle%3DPure%2520shift%2520NMR%26jtitle%3DProg.%2520Nucl.%2520Magn.%2520Reson.%2520Spectrosc.%26date%3D2015%26volume%3D86%25E2%2580%259387%26spage%3D1%26epage%3D20%26doi%3D10.1016%2Fj.pnmrs.2015.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Castañar, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parella, T.</span><span> </span><span class="NLM_article-title">Broadband <sup>1</sup>H homodecoupled NMR experiments: Recent developments, methods and applications</span> <span class="citation_source-journal">Magn. Reson. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">399</span><span class="NLM_x">–</span> <span class="NLM_lpage">426</span><span class="refDoi"> DOI: 10.1002/mrc.4238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2Fmrc.4238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=25899911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslyjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2015&pages=399-426&author=L.+Casta%C3%B1arauthor=T.+Parella&title=Broadband+1H+homodecoupled+NMR+experiments%3A+Recent+developments%2C+methods+and+applications&doi=10.1002%2Fmrc.4238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Broadband 1H homodecoupled NMR experiments: recent developments, methods and applications</span></div><div class="casAuthors">Castanar, Laura; Parella, Teodor</div><div class="citationInfo"><span class="NLM_cas:title">Magnetic Resonance in Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">399-426</span>CODEN:
                <span class="NLM_cas:coden">MRCHEG</span>;
        ISSN:<span class="NLM_cas:issn">0749-1581</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years, a great interest in the development of new broadband 1H homonuclear decoupled techniques providing simplified JHH multiplet patterns has emerged again in the field of small mol. NMR.  The resulting highly resolved 1H NMR spectra display resonances as collapsed singlets, therefore minimizing signal overlap and expediting spectral anal.  This review aims at presenting the most recent advances in pure shift NMR spectroscopy, with a particular emphasis to the Zangger-Sterk expt.  A detailed discussion about the most relevant practical aspects in terms of pulse sequence design, selectivity, sensitivity, spectral resoln. and performance is provided.  Finally, the implementation of the different reported strategies into traditional 1-dimensional and 2-dimensional NMR expts. is described while several practical applications are also reviewed.  Copyright © 2015 John Wiley and Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNiaB_DZi5kLVg90H21EOLACvtfcHk0liiu1IAnT1Ffg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslyjtbY%253D&md5=e5a6e7da0609e6ad63d8a5b875748ced</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fmrc.4238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmrc.4238%26sid%3Dliteratum%253Aachs%26aulast%3DCasta%25C3%25B1ar%26aufirst%3DL.%26aulast%3DParella%26aufirst%3DT.%26atitle%3DBroadband%25201H%2520homodecoupled%2520NMR%2520experiments%253A%2520Recent%2520developments%252C%2520methods%2520and%2520applications%26jtitle%3DMagn.%2520Reson.%2520Chem.%26date%3D2015%26volume%3D53%26spage%3D399%26epage%3D426%26doi%3D10.1002%2Fmrc.4238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Williamson, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buevich, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parella, T.</span><span> </span><span class="NLM_article-title">LR-HSQMBC: A sensitive NMR technique to probe very long-range heteronuclear coupling pathways</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">3887</span><span class="NLM_x">–</span> <span class="NLM_lpage">3894</span><span class="refDoi"> DOI: 10.1021/jo500333u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo500333u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsl2ksrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2014&pages=3887-3894&author=R.+T.+Williamsonauthor=A.+V.+Buevichauthor=G.+E.+Martinauthor=T.+Parella&title=LR-HSQMBC%3A+A+sensitive+NMR+technique+to+probe+very+long-range+heteronuclear+coupling+pathways&doi=10.1021%2Fjo500333u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">LR-HSQMBC: A Sensitive NMR Technique To Probe Very Long-Range Heteronuclear Coupling Pathways</span></div><div class="casAuthors">Williamson, R. Thomas; Buevich, Alexei V.; Martin, Gary E.; Parella, Teodor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3887-3894</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HMBC is one of the most often used and vital NMR expts. for the structure elucidation of org. and inorg. mols.  We have developed a new, high sensitivity NMR pulse sequence that overcomes the typical 2,3JCH limitation of HMBC by extending the visualization of long-range correlation data to 4-, 5-, and even 6-bond long-range nJCH heteronuclear couplings.  This technique should prove to be an effective expt. to complement HMBC for probing the structure of proton-deficient mols.  The LR-HSQMBC NMR expt. can, in effect, extend the range of HMBC to provide data similar to that afforded by 1,n-ADEQUATE even in sample-limited situations.  This is accomplished by optimizing responses for very small nJCH couplings as opposed to relying on the markedly less sensitive detection of long-range coupled 13C-13C homonuclear pairs at natural abundance.  DFT calcns. were employed to det. whether the very long-range correlations obsd. for cervinomycin A2 were reasonable on the basis of the calcd. long-range couplings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZEup7cBKL97Vg90H21EOLACvtfcHk0liiu1IAnT1Ffg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsl2ksrY%253D&md5=3945cadbae7cebed0057c9980c7a3052</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjo500333u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo500333u%26sid%3Dliteratum%253Aachs%26aulast%3DWilliamson%26aufirst%3DR.%2BT.%26aulast%3DBuevich%26aufirst%3DA.%2BV.%26aulast%3DMartin%26aufirst%3DG.%2BE.%26aulast%3DParella%26aufirst%3DT.%26atitle%3DLR-HSQMBC%253A%2520A%2520sensitive%2520NMR%2520technique%2520to%2520probe%2520very%2520long-range%2520heteronuclear%2520coupling%2520pathways%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2014%26volume%3D79%26spage%3D3887%26epage%3D3894%26doi%3D10.1021%2Fjo500333u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Martin, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzer, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouch, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castle, R. N.</span><span> </span><span class="NLM_article-title">Inverted and suppressed direct response HMQC-TOCSY spectra -- a convenient method of spectral editing</span> <span class="citation_source-journal">J. Heterocycl. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">582</span><span class="refDoi"> DOI: 10.1002/jhet.5570290251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1002%2Fjhet.5570290251" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1992&pages=577-582&author=G.+E.+Martinauthor=T.+D.+Spitzerauthor=R.+C.+Crouchauthor=J.-K.+Luoauthor=R.+N.+Castle&title=Inverted+and+suppressed+direct+response+HMQC-TOCSY+spectra+%2D%2D+a+convenient+method+of+spectral+editing&doi=10.1002%2Fjhet.5570290251"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570290251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570290251%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DG.%2BE.%26aulast%3DSpitzer%26aufirst%3DT.%2BD.%26aulast%3DCrouch%26aufirst%3DR.%2BC.%26aulast%3DLuo%26aufirst%3DJ.-K.%26aulast%3DCastle%26aufirst%3DR.%2BN.%26atitle%3DInverted%2520and%2520suppressed%2520direct%2520response%2520HMQC-TOCSY%2520spectra%2520--%2520a%2520convenient%2520method%2520of%2520spectral%2520editing%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D1992%26volume%3D29%26spage%3D577%26epage%3D582%26doi%3D10.1002%2Fjhet.5570290251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="note"><p class="first last">Structure based rationale for binding affinity changes observed for the metabololites was provided by computational modeling based off a similar FIXa cocrystal structure. See <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><div class="note"><p class="first last">For Cheng-Prusoff equation for competitive inhibition, see:</p></div><span class="NLM_contrib-group">Cheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (<i>K<sub>I</sub></i>) and the concentration of inhibitor which causes 50% inhibition (<i>I<sub>50</sub></i>) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099&author=Y.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28KI%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lgA0og7n4foHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528KI%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i51"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01293">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_80416"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01293">10.1021/acs.jmedchem.5b01293</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Experimental procedures and additional results and discussions (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01293/suppl_file/jm5b01293_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Data table (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01293/suppl_file/jm5b01293_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01293/suppl_file/jm5b01293_si_001.pdf">jm5b01293_si_001.pdf (944.26 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01293/suppl_file/jm5b01293_si_002.csv">jm5b01293_si_002.csv (1.5 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01293&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01293%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01293" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993e48fdf93c5e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
