
<html lang="en"     class="pb-page"  data-request-id="97047bf0-672c-4042-b840-b867c1486f13"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01670;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-2;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening" /></meta><meta name="dc.Creator" content="Christopher R.  Wellaway" /></meta><meta name="dc.Creator" content="Dominique  Amans" /></meta><meta name="dc.Creator" content="Paul  Bamborough" /></meta><meta name="dc.Creator" content="Heather  Barnett" /></meta><meta name="dc.Creator" content="Rino A.  Bit" /></meta><meta name="dc.Creator" content="Jack A.  Brown" /></meta><meta name="dc.Creator" content="Neil R.  Carlson" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Anthony W. J.  Cooper" /></meta><meta name="dc.Creator" content="Peter D.  Craggs" /></meta><meta name="dc.Creator" content="Robert P.  Davis" /></meta><meta name="dc.Creator" content="Tony W.  Dean" /></meta><meta name="dc.Creator" content="John P.  Evans" /></meta><meta name="dc.Creator" content="Laurie  Gordon" /></meta><meta name="dc.Creator" content="Isobel L.  Harada" /></meta><meta name="dc.Creator" content="David J.  Hirst" /></meta><meta name="dc.Creator" content="Philip G.  Humphreys" /></meta><meta name="dc.Creator" content="Katherine L.  Jones" /></meta><meta name="dc.Creator" content="Antonia J.  Lewis" /></meta><meta name="dc.Creator" content="Matthew J.  Lindon" /></meta><meta name="dc.Creator" content="Dave  Lugo" /></meta><meta name="dc.Creator" content="Mahnoor  Mahmood" /></meta><meta name="dc.Creator" content="Scott  McCleary" /></meta><meta name="dc.Creator" content="Patricia  Medeiros" /></meta><meta name="dc.Creator" content="Darren J.  Mitchell" /></meta><meta name="dc.Creator" content="Michael  O’Sullivan" /></meta><meta name="dc.Creator" content="Armelle  Le Gall" /></meta><meta name="dc.Creator" content="Vipulkumar K.  Patel" /></meta><meta name="dc.Creator" content="Chris  Patten" /></meta><meta name="dc.Creator" content="Darren L.  Poole" /></meta><meta name="dc.Creator" content="Rishi R.  Shah" /></meta><meta name="dc.Creator" content="Jane E.  Smith" /></meta><meta name="dc.Creator" content="Kayleigh A. J.  Stafford" /></meta><meta name="dc.Creator" content="Pamela J.  Thomas" /></meta><meta name="dc.Creator" content="Mythily  Vimal" /></meta><meta name="dc.Creator" content="Ian D.  Wall" /></meta><meta name="dc.Creator" content="Robert J.  Watson" /></meta><meta name="dc.Creator" content="Natalie  Wellaway" /></meta><meta name="dc.Creator" content="Gang  Yao" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Description" content="The bromodomain and extraterminal (BET) family of bromodomain-containing proteins are important regulators of the epigenome through their ability to recognize N-acetyl lysine (KAc) post-translation..." /></meta><meta name="Description" content="The bromodomain and extraterminal (BET) family of bromodomain-containing proteins are important regulators of the epigenome through their ability to recognize N-acetyl lysine (KAc) post-translation..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 6, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01670" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01670" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01670" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01670" /></link>
        
    
    

<title>Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01670" /></meta><meta property="og:title" content="Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0027.jpeg" /></meta><meta property="og:description" content="The bromodomain and extraterminal (BET) family of bromodomain-containing proteins are important regulators of the epigenome through their ability to recognize N-acetyl lysine (KAc) post-translational modifications on histone tails. These interactions have been implicated in various disease states and, consequently, disruption of BET–KAc binding has emerged as an attractive therapeutic strategy with a number of small molecule inhibitors now under investigation in the clinic. However, until the utility of these advanced candidates is fully assessed by these trials, there remains scope for the discovery of inhibitors from new chemotypes with alternative physicochemical, pharmacokinetic, and pharmacodynamic profiles. Herein, we describe the discovery of a candidate-quality dimethylpyridone benzimidazole compound which originated from the hybridization of a dimethylphenol benzimidazole series, identified using encoded library technology, with an N-methyl pyridone series identified through fragment screening. Optimization via structure- and property-based design led to I-BET469, which possesses favorable oral pharmacokinetic properties, displays activity in vivo, and is projected to have a low human efficacious dose." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01670"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01670">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01670&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01670&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01670&amp;href=/doi/10.1021/acs.jmedchem.9b01670" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 714-746</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01626" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01692" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Christopher R. Wellaway</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher R. Wellaway</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#513239233822253e213934237f297f26343d3d302630281136223a7f323e3c"><span class="__cf_email__" data-cfemail="7e1d160c170d0a110e161b0c500650091b12121f091f073e190d15501d1113">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+R.++Wellaway">Christopher R. Wellaway</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0077-3452" title="Orcid link">http://orcid.org/0000-0003-0077-3452</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dominique Amans</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dominique Amans</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dominique++Amans">Dominique Amans</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul Bamborough</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Bamborough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Bamborough">Paul Bamborough</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heather Barnett</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heather Barnett</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heather++Barnett">Heather Barnett</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rino A. Bit</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rino A. Bit</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rino+A.++Bit">Rino A. Bit</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jack A. Brown</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jack A. Brown</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jack+A.++Brown">Jack A. Brown</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Neil R. Carlson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Neil R. Carlson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, 200 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Neil+R.++Carlson">Neil R. Carlson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chun-wa Chung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun-wa Chung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anthony W. J. Cooper</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anthony W. J. Cooper</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anthony+W.+J.++Cooper">Anthony W. J. Cooper</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter D. Craggs</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter D. Craggs</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+D.++Craggs">Peter D. Craggs</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert P. Davis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert P. Davis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+P.++Davis">Robert P. Davis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tony W. Dean</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tony W. Dean</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tony+W.++Dean">Tony W. Dean</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John P. Evans</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John P. Evans</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+P.++Evans">John P. Evans</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laurie Gordon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laurie Gordon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laurie++Gordon">Laurie Gordon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Isobel L. Harada</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Isobel L. Harada</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Isobel+L.++Harada">Isobel L. Harada</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David J. Hirst</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David J. Hirst</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Hirst">David J. Hirst</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9969-2104" title="Orcid link">http://orcid.org/0000-0001-9969-2104</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philip G. Humphreys</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philip G. Humphreys</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philip+G.++Humphreys">Philip G. Humphreys</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8614-7155" title="Orcid link">http://orcid.org/0000-0002-8614-7155</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katherine L. Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katherine L. Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katherine+L.++Jones">Katherine L. Jones</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Antonia J. Lewis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antonia J. Lewis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antonia+J.++Lewis">Antonia J. Lewis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew J. Lindon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew J. Lindon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+J.++Lindon">Matthew J. Lindon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dave Lugo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dave Lugo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dave++Lugo">Dave Lugo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mahnoor Mahmood</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mahnoor Mahmood</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mahnoor++Mahmood">Mahnoor Mahmood</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott McCleary</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott McCleary</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott++McCleary">Scott McCleary</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Patricia Medeiros</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Patricia Medeiros</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, 200 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Patricia++Medeiros">Patricia Medeiros</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darren J. Mitchell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darren J. Mitchell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darren+J.++Mitchell">Darren J. Mitchell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael O’Sullivan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael O’Sullivan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++O%E2%80%99Sullivan">Michael O’Sullivan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Armelle Le Gall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Armelle Le Gall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Armelle++Le+Gall">Armelle Le Gall</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vipulkumar K. Patel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vipulkumar K. Patel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vipulkumar+K.++Patel">Vipulkumar K. Patel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chris Patten</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chris Patten</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chris++Patten">Chris Patten</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darren L. Poole</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darren L. Poole</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darren+L.++Poole">Darren L. Poole</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rishi R. Shah</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rishi R. Shah</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rishi+R.++Shah">Rishi R. Shah</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5018-1730" title="Orcid link">http://orcid.org/0000-0002-5018-1730</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jane E. Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jane E. Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jane+E.++Smith">Jane E. Smith</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kayleigh A. J. Stafford</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kayleigh A. J. Stafford</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kayleigh+A.+J.++Stafford">Kayleigh A. J. Stafford</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pamela J. Thomas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pamela J. Thomas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pamela+J.++Thomas">Pamela J. Thomas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mythily Vimal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mythily Vimal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mythily++Vimal">Mythily Vimal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ian D. Wall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ian D. Wall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ian+D.++Wall">Ian D. Wall</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert J. Watson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Watson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Watson">Robert J. Watson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Natalie Wellaway</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Natalie Wellaway</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Natalie++Wellaway">Natalie Wellaway</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gang Yao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gang Yao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, 200 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gang++Yao">Gang Yao</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Rab K. Prinjha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rab K. Prinjha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01670&amp;href=/doi/10.1021%2Facs.jmedchem.9b01670" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 714–746</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 6, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 October 2019</li><li><span class="item_label"><b>Published</b> online</span>6 January 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 January 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01670" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01670</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D714%26pageCount%3D33%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChristopher%2BR.%2BWellaway%252C%2BDominique%2BAmans%252C%2BPaul%2BBamborough%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D2%26contentID%3Dacs.jmedchem.9b01670%26title%3DDiscovery%2Bof%2Ba%2BBromodomain%2Band%2BExtraterminal%2BInhibitor%2Bwith%2Ba%2BLow%2BPredicted%2BHuman%2BDose%2Bthrough%2BSynergistic%2BUse%2Bof%2BEncoded%2BLibrary%2BTechnology%2Band%2BFragment%2BScreening%26numPages%3D33%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D746%26publicationDate%3DJanuary%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01670"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2562</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01670" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;R. Wellaway&quot;},{&quot;first_name&quot;:&quot;Dominique&quot;,&quot;last_name&quot;:&quot;Amans&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Bamborough&quot;},{&quot;first_name&quot;:&quot;Heather&quot;,&quot;last_name&quot;:&quot;Barnett&quot;},{&quot;first_name&quot;:&quot;Rino&quot;,&quot;last_name&quot;:&quot;A. Bit&quot;},{&quot;first_name&quot;:&quot;Jack&quot;,&quot;last_name&quot;:&quot;A. Brown&quot;},{&quot;first_name&quot;:&quot;Neil&quot;,&quot;last_name&quot;:&quot;R. Carlson&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;W. J. Cooper&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;D. Craggs&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;P. Davis&quot;},{&quot;first_name&quot;:&quot;Tony&quot;,&quot;last_name&quot;:&quot;W. Dean&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;P. Evans&quot;},{&quot;first_name&quot;:&quot;Laurie&quot;,&quot;last_name&quot;:&quot;Gordon&quot;},{&quot;first_name&quot;:&quot;Isobel&quot;,&quot;last_name&quot;:&quot;L. Harada&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Hirst&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;G. Humphreys&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;L. Jones&quot;},{&quot;first_name&quot;:&quot;Antonia&quot;,&quot;last_name&quot;:&quot;J. Lewis&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;J. Lindon&quot;},{&quot;first_name&quot;:&quot;Dave&quot;,&quot;last_name&quot;:&quot;Lugo&quot;},{&quot;first_name&quot;:&quot;Mahnoor&quot;,&quot;last_name&quot;:&quot;Mahmood&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;McCleary&quot;},{&quot;first_name&quot;:&quot;Patricia&quot;,&quot;last_name&quot;:&quot;Medeiros&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;J. Mitchell&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;O’Sullivan&quot;},{&quot;first_name&quot;:&quot;Armelle&quot;,&quot;last_name&quot;:&quot;Le Gall&quot;},{&quot;first_name&quot;:&quot;Vipulkumar&quot;,&quot;last_name&quot;:&quot;K. Patel&quot;},{&quot;first_name&quot;:&quot;Chris&quot;,&quot;last_name&quot;:&quot;Patten&quot;},{&quot;first_name&quot;:&quot;Darren&quot;,&quot;last_name&quot;:&quot;L. Poole&quot;},{&quot;first_name&quot;:&quot;Rishi&quot;,&quot;last_name&quot;:&quot;R. Shah&quot;},{&quot;first_name&quot;:&quot;Jane&quot;,&quot;last_name&quot;:&quot;E. Smith&quot;},{&quot;first_name&quot;:&quot;Kayleigh&quot;,&quot;last_name&quot;:&quot;A. J. Stafford&quot;},{&quot;first_name&quot;:&quot;Pamela&quot;,&quot;last_name&quot;:&quot;J. Thomas&quot;},{&quot;first_name&quot;:&quot;Mythily&quot;,&quot;last_name&quot;:&quot;Vimal&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;D. Wall&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Watson&quot;},{&quot;first_name&quot;:&quot;Natalie&quot;,&quot;last_name&quot;:&quot;Wellaway&quot;},{&quot;first_name&quot;:&quot;Gang&quot;,&quot;last_name&quot;:&quot;Yao&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;06&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;714-746&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01670&quot;},&quot;abstract&quot;:&quot;The bromodomain and extraterminal (BET) family of bromodomain-containing proteins are important regulators of the epigenome through their ability to recognize N-acetyl lysine (KAc) post-translational modifications on histone tails. These interactions have been implicated in various disease states and, consequently, disruption of BET–KAc binding has emerged as an attractive therapeutic strategy with a number of small molecule inhibitors now under investigation in the clinic. However, until the utility of these advanced candidates is fully assessed by these trials, there remains scope for the discovery of inhibitors from new chemotypes with alternative physicochemical, pharmacokinetic, and pharmacodynamic profiles. Herein, we describe the discovery of a candidate-quality dimethylpyridone benzimidazole compound which originated from the hybridization of a dimethylphenol benzimidazole series, identified using encoded library technology, with an N-methyl pyridone series identified through fragment screening. Optim&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01670&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01670" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01670&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01670" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01670&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01670" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01670&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01670&amp;href=/doi/10.1021/acs.jmedchem.9b01670" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01670" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01670" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01670%26sid%3Dliteratum%253Aachs%26pmid%3D31904959%26genre%3Darticle%26aulast%3DWellaway%26date%3D2020%26atitle%3DDiscovery%2Bof%2Ba%2BBromodomain%2Band%2BExtraterminal%2BInhibitor%2Bwith%2Ba%2BLow%2BPredicted%2BHuman%2BDose%2Bthrough%2BSynergistic%2BUse%2Bof%2BEncoded%2BLibrary%2BTechnology%2Band%2BFragment%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D2%26spage%3D714%26epage%3D746%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291864" title="Fluorescence resonance energy transfer">Fluorescence resonance energy transfer</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/2" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/jmcmar.2020.63.issue-2/20200123/jmcmar.2020.63.issue-2.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The bromodomain and extraterminal (BET) family of bromodomain-containing proteins are important regulators of the epigenome through their ability to recognize <i>N</i>-acetyl lysine (KAc) post-translational modifications on histone tails. These interactions have been implicated in various disease states and, consequently, disruption of BET–KAc binding has emerged as an attractive therapeutic strategy with a number of small molecule inhibitors now under investigation in the clinic. However, until the utility of these advanced candidates is fully assessed by these trials, there remains scope for the discovery of inhibitors from new chemotypes with alternative physicochemical, pharmacokinetic, and pharmacodynamic profiles. Herein, we describe the discovery of a candidate-quality dimethylpyridone benzimidazole compound which originated from the hybridization of a dimethylphenol benzimidazole series, identified using encoded library technology, with an <i>N</i>-methyl pyridone series identified through fragment screening. Optimization via structure- and property-based design led to I-BET469, which possesses favorable oral pharmacokinetic properties, displays activity in vivo, and is projected to have a low human efficacious dose.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40262" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40262" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein transcription is a tightly regulated process and is controlled, in part, by proteins which “read” post-translational modifications (PTMs) on flexible histone tails.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> One important class of the reader module, known as bromodomains, recognizes <i>N</i>-acetyl lysine (KAc) PTMs, and dysregulation of their function can lead to pathological conditions.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Of the 46 bromodomain-containing proteins (BCPs) identified,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> the bromodomain and extraterminal domain (BET) family (BRD2, BRD3, BRD4, and BRDT) have been the most extensively studied, with links established to a diverse range of disease states such as cancer, inflammation, and viral infection.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Because of these associations, abrogation of BET–KAc interactions is a burgeoning field of research with an increasing number of small molecule inhibitors being described, several of which are currently under clinical investigation.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Most of these inhibitors bind to each of the four BET isoforms and their tandem bromodomain modules with similar affinity, that is, pan-inhibition, but reports of molecules selective for either the first [(<i>N</i>-terminal) bromodomain, BD1] or second [(C-terminal) bromodomain, BD2] set of bromodomains have emerged,<a onclick="showRef(event, 'ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9">(6−9)</a> as well as isoform-selective inhibitors<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and compounds selective for one out of the eight BET bromodomains.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Furthermore, the BET family has been targeted using contemporary approaches such as bivalent inhibition of the tandem bromodomain modules,<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> degradation via the proteasome using proteolysis targeting chimera technology,<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19">(15−19)</a> and through covalent inhibition.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p">A key goal in the development of BET inhibitors toward medicines has been to establish activity in preclinical species, and success in this area was demonstrated early in the field. For example, a seminal study with I-BET762 <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) demonstrated its ability to protect mice against death in models of endotoxic shock and sepsis,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and it was subsequently shown to inhibit ear swelling in a rat model of delayed-type hypersensitivity.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The related BET inhibitor, JQ1 <b>2</b>, was first shown to reduce the tumor size and promote survival in mouse xenograft models of nuclear protein in testis midline carcinoma,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and has since been evaluated in a range of animal models including male contraception,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> bleomycin-induced lung fibrosis,<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> and cardiac hypotrophy.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Another molecule first published at the start of the last decade, RVX-208 <b>3</b>, has been shown to increase apolipoprotein A-1 and high-density lipoprotein cholesterol in monkeys<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and reduce atherosclerosis in hyperlipidemic apolipoprotein E-deficient mice.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The early generation BET inhibitor, I-BET151 <b>4</b>, promoted survival in mouse models of leukemia<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and, similarly to I-BET762 <b>1</b>, suppressed cytokine production and protected against death in acute inflammatory mouse models.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> Data from in vivo studies with newer inhibitors are emerging,<a onclick="showRef(event, 'ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44 ref45 ref46">(33−46)</a> but such reports often disclose response data without the corresponding drug concentrations, which prevents full understanding of pharmacokinetic/pharmacodynamic (PKPD) effects and can hinder modeling predictions for humans. Furthermore, some inhibitors have been reported to display solubility-limited absorption following oral dosing<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and short half-lives<a onclick="showRef(event, 'ref22 ref23 ref39'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref39">(22,23,39)</a> and are at risk of not engaging the mechanism in humans if poor systemic exposure translates. For these reasons, there remains scope for the development of BET inhibitors with sufficient exposure in multiple preclinical test species to provide confidence that the mechanism can be engaged in humans. Furthermore, inhibitors need to possess both favorable pharmacokinetic parameters and high potency for the target to achieve a low predicted human efficacious dose and minimize the risk of idiosyncratic toxicity. Accordingly, we herein report the development of a potent and soluble dimethylpyridone benzimidazole BET inhibitor exhibiting PK profiles which, after single species scaling based on the allometric principles, correspond to low predicted human doses of <30 mg once every day (QD). Hybridization of a dimethylphenol benzimidazole series identified using encoded library technology (ELT) with a methyl pyridone series discovered by fragment screening resulted in an improved dimethylpyridone benzimidazole scaffold. Subsequent optimization of off-target bromodomain selectivity was carried out alongside frequent PKPD modeling to deliver an inhibitor which fully engaged the BET mechanism and demonstrated immunomodulatory effects in vivo.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Domain architecture of BET highlighting the eight bromodomains. (b) Early generation BET inhibitors with in vivo capability.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17717" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17717" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our overall goal was to discover an orally bioavailable BET inhibitor that was structurally distinct from our existing clinical candidates.<a onclick="showRef(event, 'ref22 ref47'); return false;" href="javascript:void(0);" class="ref ref22 ref47">(22,47)</a> We set approximate potency criteria of <50 nM in a BET biochemical fluorescence resonance energy transfer (FRET) assay and <100 nM in an LPS-stimulated human whole blood (WB) assay measuring inhibition of the proinflammatory cytokines monocyte chemoattractant protein 1 (MCP-1) and interleukin 6 (IL-6), a phenotype of BET inhibition.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> We also required 100-fold selectivity over other BCPs and over representative targets from other target classes. To minimize the risk of compound-based attrition, we aimed for a property forecast index (PFI [ChromLog<i>D</i><sub>7.4</sub> + number of aromatic rings])<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> <6, solubility in FaSSIF >100 μg/mL, and a predicted human daily dose <100 mg. This was in accordance with our oral candidate quality criteria,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and these parameters guided the optimization process.</div><div class="NLM_p">To identify a novel BET inhibitor chemotype, we embarked on a screening campaign using ELT which has proven to be an important lead generation platform for various therapeutic targets.<a onclick="showRef(event, 'ref50 ref51 ref52 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref50 ref51 ref52 ref53 ref54 ref55">(50−55)</a> A 6His-BRD4 (1–477) tandem bromodomain construct was screened against a panel of DNA-encoded small-molecule libraries using affinity-based selection technology, and a hit series was identified from a three-cycle benzimidazole core library (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). This specific library was constructed using a split-and-pool strategy with three cycles of building blocks (BBs) to provide a total warhead diversity of 117 million compounds as previously reported.<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53−55)</a> Nucleophilic aromatic substitution of DNA-appended 4-fluoro-3-nitrobenzoate with 65 monoprotected diamines at cycle 1 was followed by nitro reduction to afford 65 benzenediamines (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). These diamines were condensed with 922 aldehydes at cycle 2 to form benzimidazole ring systems. After amine deprotection, the library was reacted with 1960 amine capping groups (carboxylic acids, isocyanates, sulfonyl chlorides, and heterocyclic halides) to afford the final library. DNA tags were added after each cycle of reaction to unambiguously encode each warhead in the library. Interpretation of the affinity selection from this library was viewed in a cubic scatter plot (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b), where each axis corresponds to a BB cycle. Examination of these data revealed enrichment of a specific BB1 (piperidin-4-ylmethanamine, <b>5</b>) and BB2 (4-hydroxy-3,5-dimethylbenzaldehyde, <b>6</b>) combination which corresponded to 1,2,5-substituted benzimidazole <b>7</b>. The cycle 3 amine capping groups were predominately carboxylic acids of diverse structure, and the appearance of a highly populated (light blue) line along the x-axis indicated no preference for any particular carboxylic acid, suggesting that BB3 contributed little to the binding.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Scheme of the hit DNA-encoded benzimidazole library where BB1, BB2, and BB3 refer to cycle 1, 2, and 3 BBs, respectively. (b) Result of the BRD4 screen of the benzimidazole library. Compounds on the highlighted horizontal line (light blue) in the cubic scatter plot shared the common BB1–BB2 combination (<b>5</b> and <b>6</b>, respectively) with BB3 as a variable component resulting in the generic hit structure <b>7</b>. The sphere size is proportional to the DNA tag copy number. Library members with less than 5 copies were removed to simplify visualization.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having identified a hit series, we prepared several 1,2,5-substituted benzimidazoles without DNA tags as exemplars of the hit BB1–BB2 combination <b>7</b> in a similar manner to that outlined in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The synthetic sequence commenced with reaction of 4-fluoro-<i>N</i>-methyl-3-nitrobenzamide with <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (<b>5</b>) leading to the nucleophilic aromatic substitution product which was used without purification in the formation of the benzimidazole ring. Construction of the heterocycle was carried out in one pot, whereby the <i>ortho</i>-nitroaniline was reacted with 4-hydroxy-3,5-dimethylbenzaldehyde in the presence of sodium dithionite in a nitro reduction–aromatization sequence according to the method of Yang et al.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Subsequent <i>N</i>-Boc removal provided amine <b>8</b> which was acylated to provide the target amides <b>9</b> and <b>10</b>. The activities of these compounds were measured in a historical fluorescence polarization (FP) assay format and/or a routine FRET assay format using a mutant BRD4 construct which determined activity at BD1 and was used as a surrogate for activities at all BET bromodomains (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Lipophilicities were measured using a chromatographic method at pH = 7.4 (ChromLog<i>D</i><sub>7.4</sub>),<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and the corresponding PFI values were calculated.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Dimethylphenol Benzimidazole Analogues <b>8–10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (<b>5</b>), DIPEA, EtOH, 80 °C; (ii) 4-hydroxy-3,5-dimethylbenzaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (4:1), 80 °C; (iii) TFA, DCM, rt; (b) PhCO<sub>2</sub>H, HATU, DIPEA, DMF, rt; (c) (i) Ac<sub>2</sub>O, pyr, THF/MeCN; (ii) 2 M LiOH, THF/MeOH (4:1), rt.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Data for Dimethylphenol Benzimidazole N1 Analogues <b>8–10</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0018.gif" alt="" id="GRAPHIC-d7e807-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">BRD4 pIC<sub>50</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">FP</th><th class="colsep0 rowsep0" align="center">FRET</th><th class="colsep0 rowsep0" align="center">ChromLog<i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center">PFI</th><th class="colsep0 rowsep0" align="center">LE<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">LLE<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>8</b></td><td class="colsep0 rowsep0" align="center">H</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">5.7</td><td class="colsep0 rowsep0" align="center">0.8</td><td class="colsep0 rowsep0" align="center">3.8</td><td class="colsep0 rowsep0" align="center">—/0.27</td><td class="colsep0 rowsep0" align="center">—/2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>9</b></td><td class="colsep0 rowsep0" align="center">PhC(O)</td><td class="colsep0 rowsep0" align="center">6.9</td><td class="colsep0 rowsep0" align="center">6.7</td><td class="colsep0 rowsep0" align="center">3.2</td><td class="colsep0 rowsep0" align="center">7.2</td><td class="colsep0 rowsep0" align="center">0.26/0.25</td><td class="colsep0 rowsep0" align="center">2.5/2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>10</b></td><td class="colsep0 rowsep0" align="center">CH<sub>3</sub>(O)</td><td class="colsep0 rowsep0" align="center">6.6</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">2.0</td><td class="colsep0 rowsep0" align="center">5.0</td><td class="colsep0 rowsep0" align="center">0.28/—</td><td class="colsep0 rowsep0" align="center">4.0/—</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Expressed as the mean from at least two test occasions. NT = Not tested.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Calculated from FP/FRET pIC<sub>50</sub> values.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Defined as BRD4 pIC<sub>50</sub> – clog <i>P</i>.</p></div></div></div><div class="NLM_p">Piperidine <b>8</b> was weakly active at BRD4 but potency was increased by capping as the benzamide <b>9</b> and acetamide <b>10</b>, indicating a basic center was not optimal for binding to the target at this position in the molecule. While acetamide <b>10</b> was the slightly less potent amide, it was less lipophilic than benzamide <b>9</b> leading to a more desirable PFI and higher LLE. Therefore, exploration of alternative benzimidazole N1 substituents was generally conducted with acetamides in place. First, however, we were interested in assessing the importance of the benzimidazole 5-position amide toward binding affinity, given its role as the linking functionality to DNA during ELT hit identification. Accordingly, we prepared benzimidazole <b>15a</b> lacking the 5-amide following <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and, after biochemical testing, found only a slight lowering in activity versus its amide-substituted congener <b>10</b> (BRD4 FP pIC<sub>50</sub> = 6.3 and 6.6, respectively). Therefore, further exploration of SAR at the benzimidazole N1-position was conducted without the 5-amide in place (compounds <b>15b–g</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>); BRD4 FRET assay results are shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Dimethylphenol Benzimidazole N1 Analogues <b>15a–g</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) RNH<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80–110 °C; (b) H<sub>2</sub>, Pd/C, DMF, 30 °C; (c) 4-hydroxy-3,5-dimethylbenzaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMF, 100 °C; (d) 4 M HCl in MeOH, rt−50 °C; (e) AcOH or 2-hydroxyacetic acid, BOP, NEt<sub>3</sub>, DCM, rt.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. BRD4 Activities and PFI Values of Dimethylphenol Benzimidazole N1 Analogues <b>15a–g</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0019.gif" alt="" id="GRAPHIC-d7e1077-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0020.gif" alt="" id="gr18" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Expressed as the mean from at least three test occasions.</p></div></div><div></div></div><div class="NLM_p">BRD4 activity for 4-piperidine acetamide <b>15b</b> was threefold less than that for 4-methylene piperidine acetamide <b>15a</b> and highlighted the importance of a linking methylene between the pendant heterocycle and the benzimidazole core. The size of the heterocycle also affected activity, underscored by the lower potencies for azetidine <b>15c</b> and racemic pyrrolidine <b>15d</b> compared to piperidine <b>15a</b>. Transposition of the amide in piperidine acetamide <b>15a</b> to an exocyclic position gave the cyclohexyl acetamides <b>15e</b> and <b>15f</b>, and activity was maintained with the latter <i>cis</i>-isomer. Finally, it was demonstrated that the acetamide <b>15a</b> could be substituted with a hydroxyl group to give the more polar hydroxy acetamide <b>15g</b> without lowering BRD4 activity.</div><div class="NLM_p">During these investigations, we pursued efforts to obtain an X-ray crystal structure of acetamide <b>10</b>. Gratifyingly, cocrystallization with BRD4 BD1 resulted in a 1.5 Å structure and revealed the dimethylphenol moiety positioned in the KAc binding site (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of acetamide <b>10</b> (carbon = green) in BRD4 BD1 (carbon = gray, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>). Water molecules at the base of the binding pocket are shown as red spheres, and selected hydrogen bonds are shown as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The 2,6-dimethylphenol group replicated the interactions of KAc<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and bound in the same manner as other 2,6-dimethylphenol-containing BET ligands.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Specifically, the hydroxyl accepted a hydrogen bond from Asn140 and donated a hydrogen bond via water to Tyr97, while an ortho-methyl substituent occupied a lipophilic pocket adjacent to Phe83. The phenyl ring of the benzimidazole bicyclic system was positioned between Leu92 and Trp81 (a region known as the ZA channel) and formed an edge-to-face interaction with the latter residue. The benzimidazole N1 position provided an appropriate vector for the 4-methylene piperidine to bind to the lipophilic region flanking the Trp81-Pro82-Phe83 triad, commonly referred to as the WPF shelf, and occupying this part of the binding site has generally been associated with increased potency.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The relatively small Ile146 residue in BRD4 BD1, located near the start of the C-helix, permitted access to the WPF shelf but larger residues at this position in other BCPs preclude access, and thus, binding to the WPF shelf in BRD4 BD1 was expected to impart selectivity over certain BCPs.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The acetamide was directed out of the WPF shelf toward the solvent, and this was consistent with SARs demonstrating that alterations to the amide did not profoundly affect potency. The methylamide substituent at the 5-position of the benzimidazole was directed out of the ZA channel toward the solvent, and this positioning was consistent with its role as the linker to DNA during hit identification. A final observation was that the benzimidazole N3 atom was sufficiently close (2.7 Å) to accept a hydrogen bond from an adjacent water molecule, which in turn formed contacts to other water molecules and Pro82 and was therefore deemed an important interaction.</div><div class="NLM_p">With the binding mode and opportunity for structure-based design established, we were keen to investigate this series further and identify any issues requiring optimization. Our primary concern at this early stage was the reliance on the 2,6-dimethylphenol moiety as the KAc mimetic. Given the general requirement for BET inhibitors to possess a KAc mimetic,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> the 2,6-dimethylphenol was deemed integral to achieve high potency but represented a potential liability because of the propensity of phenols to be rapidly metabolized by phase II enzymes. Therefore, dimethylphenol <b>10</b> was assessed in a mouse PK experiment where clearance from the blood (346 mL/min/kg) was found to be greater than liver blood flow, and a short half-life (0.37 h) was observed after iv dosing (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>). Based on these results, we commenced investigations to replace the 2,6-dimethylphenol KAc mimetic.</div><div class="NLM_p">To find suitable alternative KAc-binders and integrate additional information for optimal lead generation,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> we turned to our extensive collection of bromodomain fragment data. We have previously described our first screen of a fragment library against the BET family bromodomains, which generated multiple chemically diverse low-molecular-weight starting points that bind in the KAc site.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> These are ligand-efficient starting points, and we have shown that they can be optimized for BET bromodomains using structure-based design to build in the extra pharmacophoric components needed for high-affinity binding.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> In addition, we have found the pool of multiple bromodomain/fragment crystal structures to be a valuable source of ideas for isosteric replacements and hybridization during optimization of leads from other sources including high-throughput screening and ELT.</div><div class="NLM_p">One of the fragments emerging from this work was the low-affinity ligand-efficient <i>N</i>-methyl pyridone-containing compound <b>16</b> (BRD4 FRET pIC<sub>50</sub> = 4.9, LE = 0.42) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). A 1.9 Å X-ray crystal structure of this fragment in BRD2 BD1 showed the pyridone group binding in the KAc-site, with its carbonyl group forming a direct hydrogen bond interaction with Asn156 and through the water network to Tyr113 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). The methoxyphenyl group partially occupied the ZA channel without completely filling it.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Identification of pyridones as KAc-mimetics from a fragment-based screen.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> (b) X-ray crystal structure of <i>N</i>-methyl pyridone fragment <b>16</b> (carbon = orange) in BRD2 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ1">6TQ1</a>). (c) X-ray crystal structure of <i>N</i>-methyl pyridone imidazole <b>17</b> (carbon = cyan) in BRD2 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ2">6TQ2</a>). (d) Superposition of X-ray crystal structures of <i>N</i>-methyl pyridone imidazole <b>17</b> (carbon = cyan) in BRD2 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ2">6TQ2</a>) and 2,6-dimethylphenol <b>10</b> (carbon = green) in BRD4 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A substructure search for related <i>N</i>-methyl pyridone analogues in the GSK compound collection identified the imidazole <b>17</b> (BRD4 FRET pIC<sub>50</sub> = 5.2, LE = 0.36). A 2.3 Å X-ray crystal structure of this compound was also obtained in BRD2 BD1 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c) and confirmed that the interactions of the <i>N</i>-methylpyridone are the same as those made by fragment <b>16</b>. The imidazole pointed toward the ZA channel with one of its nitrogens forming a hydrogen bond to water. Although the fluorophenyl ring did not occupy the WPF shelf, this substituent position on the imidazole offered a vector to access the shelf and potentially gain potency.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d shows the superimposed crystal structures of pyridone <b>17</b> in BRD2 BD1 and the phenol <b>10</b> in BRD4 BD1. Unusually, in bromodomain-inhibitor complexes, the phenolic OH group of <b>10</b> probably acts as a hydrogen bond donor toward the W1 water. This contrasts with the carbonyl of pyridone <b>17</b> which can only act as a more typical hydrogen bond acceptor to the W1 water. To accommodate this difference would require a rotation of the W1 water and a change of some of its interactions with the conserved tyrosine or the other conserved water molecules (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), but it is not possible to see the details from the electron density. We speculate that the requirement for an uncommon water orientation to bind the phenols may explain their lower ligand efficiency relative to the pyridone. Regardless of this, the close overlay between the crystal structures of these series suggested that we should attempt a hybridization approach. It is worth noting that other pyridone-containing BET inhibitors have been reported since this work was carried out.<a onclick="showRef(event, 'ref62 ref63 ref64 ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref62 ref63 ref64 ref65 ref66">(62−66)</a></div><div class="NLM_p">Preparation of hydrid 1,3-dimethylpyridone <b>19a</b> and close analogues was carried out according to <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, and biological results are shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Pleasingly, BRD4 activity for 1,3-dimethylpyridone <b>19a</b> was higher than that for 2,6-dimethylphenol <b>15g</b>, and lipophilicity was lowered by ∼10-fold which resulted in a higher LLE. The requirement for two methyl substituents was established by testing mono C3-methyl <b>19b</b> and mono N1-methyl <b>19c</b> and both lowered activity relative to dimethyl <b>19a</b> but to different extents. Removal of both methyl groups to give NH pyridone <b>19d</b> resulted in the loss of measurable activity.</div><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Benzimidazole 2-Position Analogues 19a–d<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: for compounds <b>18a</b>, <b>18b</b>, and <b>18d</b>: (a) (i) RCHO, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMF, 100 °C; (ii) HCl in MeOH, rt; for compound <b>18c</b>: (b) RCO<sub>2</sub>H, PPA, 180 °C; (c) 2-hydroxyacetic acid, EDC·HCl, HOBt, DIPEA, DCM, rt.</p></p></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. BRD4 Activities and PFI Values of Benzimidazole 2-Position Analogues <b>19a–d</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0021.gif" alt="" id="GRAPHIC-d7e1366-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0022.gif" alt="" id="gr19" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Expressed as the mean from at least two test occasions.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">A pIC<sub>50</sub> value of <4.8 was determined on one test occasion out of four and was excluded from the reported mean value.</p></div></div><div></div></div><div class="NLM_p">Notwithstanding the encouraging biochemical and physiochemical data for 1,3-dimethylpyridone <b>19a</b>, the BRD4 activity and MCP-1 WB potency (pIC<sub>50</sub> = 5.8) were still below our target levels. We therefore sought to replace the benzimidazole N1-substituent with a more potent group. SARs established during off-DNA ELT chemistry (data not shown) indicated that lipophilic substituents were preferred at this position, and this was consistent with binding to the nonpolar residues which comprise the WPF shelf (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The reduction in lipophilicity afforded by replacing the dimethylphenol KAc mimetic with the dimethylpyridone allowed investigation of a substituent lacking a polar amide group while maintaining the overall physicochemical properties in a desirable space. This led us to prepare 4-methylene tetrahydropyran (THP) <b>20a</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) which exhibited <100 nM activity at BRD4 and possessed favorable physicochemical properties (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). These characteristics contributed to effective suppression of the proinflammatory cytokines IL-6 and MCP-1 in LPS-stimulated human WB and high solubility in FaSSIF (determined using a crystalline solid form).</div><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Dimethylpyridone Benzimidazole N1 Analogues <b>20a–i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) amine, DIPEA, DMSO or NMP or 2-MeTHF, microwave, 110–125 °C; (b) (i) 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (2:1), 100 °C; (ii) for compound <b>20e</b>: TFA, DCM, rt; (c) (i) H<sub>2</sub>, Pd/C, EtOH, rt; (ii) Oxone, DMF/H<sub>2</sub>O (30:1), rt.</p></p></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Structure of 4-Methylene THP <b>20a</b> and Initial Profile</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0023.gif" alt="" id="GRAPHIC-d7e1453-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 FRET pIC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">7.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WB pIC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> IL-6/MCP-1</td><td class="colsep0 rowsep0" align="center">6.7/6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BAZ2A FRET pIC<sub>50</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChromLog<i>D</i><sub>7.4</sub>/PFI</td><td class="colsep0 rowsep0" align="center">2.7/5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LE/LLE</td><td class="colsep0 rowsep0" align="center">0.39/4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FaSSIF (μg/mL)</td><td class="colsep0 rowsep0" align="center">>1000</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Expressed as the mean from at least four test occasions.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Expressed as the mean from two test occasions.</p></div></div></div><div class="NLM_p">Having identified an attractive lead compound, we carried out more comprehensive profiling to identify and understand any issues to guide the medicinal chemistry strategy. Accordingly, we screened 4-methylene THP <b>20a</b> against a panel of eight representative BCPs (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>) and found that it bound to bromodomain adjacent to zinc finger domain 2A (BAZ2A) with notable affinity (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). BAZ2A has been identified as the large subunit of the nucleolar remodeling complex and plays a role in chromosome stability, and it is known to silence some rRNA genes and can delay proliferative growth when knocked down in cells.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> It was unknown how this profile could translate to humans, so we set the goal to overcome this off-target activity during lead optimization. We also conducted PK profiling on 4-methylene THP <b>20a</b> to benchmark the pyridone benzimidazole series. Initially, compound stability in microsome and hepatocyte preparations was assessed and highlighted differences between species with rats displaying higher turnover than both humans and dogs (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Comparison of these data with observed in vivo clearance following iv administration demonstrated a good correlation. The high clearance observed in rats required improvement to ensure adequate systemic exposure following oral administration for preliminary toxicology studies.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. In Vitro and in Vivo PK Data of 4-Methylene THP <b>20a</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">IVC (mL/min/g)</th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char=".">Mics</th><th class="colsep0 rowsep0" align="center" char=".">Heps</th><th class="colsep0 rowsep0" align="center" char="."><i>f</i><sub>ub</sub></th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>b</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>u</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>dss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub>, iv (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th><th class="colsep0 rowsep0" align="center" char=".">human predicted QD dose (mg)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">0.55</td><td class="colsep0 rowsep0" align="char" char=".">82</td><td class="colsep0 rowsep0" align="char" char=".">149</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">>2000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="."><0.37</td><td class="colsep0 rowsep0" align="char" char="."><1.3</td><td class="colsep0 rowsep0" align="char" char=".">0.47</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char=".">1400</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">0.83</td><td class="colsep0 rowsep0" align="char" char="."><0.87</td><td class="colsep0 rowsep0" align="char" char=".">0.40</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">iv dose (1.0 mg/kg): 1 h infusion in DMSO (2%, v/v) and Kleptose HPB (10%, w/v) in saline (0.9% w/v).</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">iv dose (0.5 mg/kg): 1 h infusion in DMSO (2%, v/v) and Kleptose HPB (10%, w/v) in saline (0.9% w/v).</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">po dose (1.0 mg/kg): a suspension of solid of unknown form in 1% (w/v) methylcellulose (400 cps) (aq).</p></div></div></div><div class="NLM_p">At this stage, the human efficacious dose of 4-methylene THP <b>20a</b> was predicted using the single species scaling approach<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68,69)</a> (correcting for differences in the fraction unbound) in conjunction with the MCP-1 WB activity. A target coverage of IC<sub>50</sub> over a 24 h dosing period was selected at this stage of our investigations for sufficient engagement of the mechanism, and the predicted once-daily human doses based on this were >2 and 1.4 g from rats and dogs, respectively (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a> for details and assumptions). These predicted high human doses were a result of the moderate WB MCP-1 activity combined with a predicted short half-life. With an aim to have a dose <100 mg, these contributing factors became a focus for our future optimization efforts.</div><div class="NLM_p">During profiling lead benzimidazole <b>20a</b>, we obtained a 1.8 Å resolution X-ray crystal structure in BRD4 BD1, and this revealed an identical binding mode to that of hit benzimidazole <b>10</b> with direct overlay of the 1,3-dimethylpyridone and 2,6-dimethylphenol moieties (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). This structure also provided a basis to interrogate the observed binding to BAZ2A through superposition of an apo structure of BAZ2A<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). This showed several differences most notably in the C-helices where Met149 in BRD4 BD1 is replaced by Ala1882 in BAZ2A. The consequence of this difference is the collapse of the WPF motif (consisting of Trp1816-Pro1817-Phe1818) and a closed WPF-shelf in BAZ2A, whereas this region is fully open and accessible in the BET family. It was clear that this superposition did not provide an explanation for the observed BAZ2A activity for lead compound <b>20a</b> as the THP moiety would sterically clash with Trp1816 and preclude binding to this BCP. Another effect of the collapsed WPF motif in BAZ2A was the enlargement of the ZA channel relative to that in BRD4 BD1, and this was increased further by deletions in the ZA loop indicating that Leu1826 was unable to form a narrow junction equivalent to that caused by Leu92 in BRD4 BD1. We speculated that the enlarged ZA channel in BAZ2A permitted distortion of the dimethylphenol-benzimidazole dihedral angle from twisted (42°) in BRD4 BD1 to a more coplanar conformation in BAZ2A and that branching off the methylene group attached to the benzimidazole N1 position would restrict this conformational freedom, thereby increasing selectivity over BAZ2A.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Superposition of X-ray crystal structures of 1,3-dimethylpyridone <b>20a</b> (carbon = cyan, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY">6TPY</a>) and 2,6-dimethylphenol <b>10</b> in BRD4 BD1 (carbon = green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>). A surface was applied to the protein of PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>. (b) View at the base of the binding pocket from superposition of 1,3-dimethylpyridone <b>20a</b> (carbon, water molecules [spheres], and selected hydrogen bonds [dashed lines] = cyan PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY">6TPY</a>) and 2,6-dimethylphenol <b>10</b> (carbon, water molecules [spheres], and selected hydrogen bonds [dashed lines] = green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>) X-ray crystal structures in BRD4 BD1. A surface was applied to the protein of PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>. (c) Superposition of the X-ray crystal structure of 1,3-dimethylpyridone <b>20a</b> in BRD4 BD1 (carbon = cyan, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY">6TPY</a>) with an apo structure of BAZ2A<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> (carbon = orange, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LZ2">4LZ2</a>). A wireframe surface has been applied to Trp1816 of BAZ2A.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Using this structural insight, we investigated branching on the methylene group and other modifications to the 4-methylene THP and counter-screened for their BAZ2A activities (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). An additional consideration during this exploration was to limit significant increases in lipophilicity.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. BRD4 and BAZ2A Activities and ChromLog<i>D</i><sub>7.4</sub> Values of Benzimidazole N1 Analogues <b>20a–i</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0024.gif" alt="" id="GRAPHIC-d7e1878-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0025.gif" alt="" id="gr20" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Expressed as the mean from at least two test occasions.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Calculated as BRD4 FRET pIC<sub>50</sub> – BAZ2A pIC<sub>50</sub>.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">pIC<sub>50</sub> values of <4.0 were determined on two test occasions out of four and were excluded from the reported mean value.</p></div></div><div></div></div><div class="NLM_p">Incorporation of a methyl branch on the methylene gave the enantiomeric pair <b>20b</b> and <b>20b′</b> which was obtained upon chiral chromatographic separation of the racemic mixture. In terms of BRD4 FRET activity, a clear preference for stereoisomer <b>20b′</b> was observed, but BAZ2A activity was not significantly affected when compared to unbranched methylene <b>20a</b>. However, further extension of the branching substituent did reduce BAZ2A activity with ethyl enantiomers <b>20c</b> and <b>20c′</b> exhibiting preferential binding to BRD4 by >125- and 40-fold, respectively. Strikingly, the 3-methylene THP enantiomers <b>20d</b> and <b>20d′</b> reduced BAZ2A activity by several orders of magnitude compared to 4-methylene THP <b>20a</b> with the latter enantiomer displaying particularly high BRD4 activity resulting in ∼3000-fold selectivity. Other simple modifications to the 4-methylene THP group, such as moving the ether oxygen to an exocyclic position to give 4-methylene cyclohexylalcohol <b>20e</b> and homologating the methylene to give ethylene <b>20f</b>, lowered BAZ2A activity while maintaining BRD4 potency. The spirocyclic oxetane <b>20g</b>, a motif used as a replacement for 6-membered cyclic systems,<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(71,72)</a> also increased selectivity over BAZ2A and, additionally, lowered microsomal clearance (rat and human = 0.99 and <0.40 mL/min/g, respectively) while remaining liponeutral compared to THP <b>20a</b>. Acyclic substituents were also explored with racemic propyl methoxy <b>20h</b> demonstrating >100-fold selectivity over BAZ2A but only moderate BRD4 activity. Achiral dimethoxypropyl <b>20i</b> maintained a high level of BRD4 activity and was inactive at BAZ2A. Taken together, these data revealed the 4-methylene THP group to be a privileged motif for binding to BAZ2A and only minor modifications were required to dampen activity.</div><div class="NLM_p">We further profiled dimethoxypropyl <b>20i</b> and found that WB potencies (pIC<sub>50</sub> = 6.6 and 7.0 for IL-6 and MCP-1 analytes, respectively) were commensurate with high BRD4 FRET activity. In addition, high solubility was determined in FaSSIF media (>1000 μg/mL) from a crystalline solid form. We also assessed the compound for stability in microsomes and hepatocytes and found that it was more stable in both rat and human preparations versus 4-methylene THP <b>20a</b> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). This improvement was noteworthy, given the slight increase in lipophilicity incurred upon replacement of the 4-methylene THP with the dimethoxypropyl and, therefore, represented an alternative strategy for improving the stability of this cyclic ether compared to reducing the ring size.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> A subsequent in vivo rat PK study revealed moderate clearance and a longer half-life relative to 4-methylene THP <b>20a</b>. The clearance, and unbound clearance, of dimethoxypropyl <b>20i</b> in dogs was slightly lower than that observed with 4-methylene THP <b>20a</b>, but a lower <i>V</i><sub>dss</sub> was determined resulting in a shorter <i>T</i><sub>1/2</sub>. These PK data, combined with the WB MCP-1 activity, were used to calculate the predicted once-daily human doses of 1.6 g and 470 mg from rat and dog data, respectively. Although the predicted doses were lower than those for 4-methylene THP <b>20a</b>, a further lowering of the projected dose was required to meet our target criterion of <100 mg.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. In Vitro and in Vivo PK Data of Dimethoxypropyl <b>20i</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">IVC (mL/min/g)</th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center">Mics</th><th class="colsep0 rowsep0" align="center">Heps</th><th class="colsep0 rowsep0" align="center" char="."><i>f</i><sub>ub</sub></th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>b</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>u</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>dss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub>, iv (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th><th class="colsep0 rowsep0" align="center" char=".">human predicted QD dose (mg)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left"><0.46</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="char" char=".">0.59</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="char" char=".">1600</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog<a class="ref internalNav" href="#t7fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t7fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left"><0.37</td><td class="colsep0 rowsep0" align="left"><1.3</td><td class="colsep0 rowsep0" align="char" char=".">0.57</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="char" char=".">470</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="left"><0.40</td><td class="colsep0 rowsep0" align="left"><0.87</td><td class="colsep0 rowsep0" align="char" char=".">0.37</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">iv dose (1.5 mg/kg): 1 h infusion in DMSO (2%, v/v) and Kleptose HPB (10%, w/v) in saline (0.9% w/v).</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">po dose (3 mg/kg): a suspension of crystalline solid in 1% (w/v) methylcellulose (400 cps) (aq).</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">iv dose (0.5 mg/kg): 1 h infusion in DMSO (2%, v/v) and Kleptose HPB (10%, w/v) in saline (0.9% w/v).</p></div><div class="footnote" id="t7fn4"><sup><sup>d</sup></sup><p class="last">po dose (1.0 mg/kg): a suspension of crystalline solid in 1% (w/v) methylcellulose (400 cps) (aq).</p></div></div></div><div class="NLM_p">In parallel with investigations at the benzimidazole N1-position, we pursued explorations at the 5- and 6-positions aiming to lower the predicted human dose versus lead compound <b>20a</b> based on a dual approach. First, an increase in potency was predicted by substituting these positions based on the crystal structure of 4-methylene THP <b>20a</b> in BRD4 BD1 (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). Here, the phenyl ring of the benzimidazole formed an edge-to-face interaction with Trp81, and we therefore hypothesized that increasing electron density in the ring would improve this interaction and lead to increased activity; it was also noted that the interaction to water through the benzimidazole N3 nitrogen could be modified through such electronic effects. Second, reports of unsubstituted benzimidazole rings being metabolically vulnerable<a onclick="showRef(event, 'ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref74 ref75">(74,75)</a> led us to block the 5- and 6-positions with appropriate substituents to lower the clearance and increase systemic exposure and the half-life.</div><div class="NLM_p">We sought to test the first hypothesis by preparing analogues where the 5- and 6-positions were substituted with both electron-donating and electron-withdrawing groups. Synthesis of these compounds was carried out using the established S<sub>N</sub>Ar-cyclization sequence, which, in the case of the dimethylamine target compounds <b>22c</b> and <b>22d</b>, required subsequent C–N cross-coupling of the Br-substituted benzimidazoles <b>22a</b> and <b>22b</b>, respectively (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Dimethylpyridone Benzimidazole 5- and 6-Position Analogues <b>22a–p</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: for compounds <b>21a</b> and <b>21b</b>: (a) 4-substituted 1-fluoro-2-nitrobenzene or 4-substituted 4-bromo-2-fluoro-1-nitrobenzene, (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine, DIPEA, 2-MeTHF, 80 °C; for compounds <b>22a</b> and <b>22b</b>: (b) 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (2.5:1), 90 or 80 °C; (c) HN(CH<sub>3</sub>)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub> (2 mol %), JohnPhos (13 mol %), NaO<sup><i>t</i></sup>Bu, toluene or 1,4-dioxane, microwave, 90 °C; for compounds <b>22e–22p</b>: (d) (i) substituted 1-fluoro-2-nitrobenzene, (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine, DIPEA, 2-MeTHF, microwave, 110 °C; (ii) 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (2:1), 100 °C.</p></p></figure><div class="NLM_p">BRD4 FRET data were obtained for these compounds, and the activities were plotted against sigma para values<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). A general correlation for increased electron density with increased potency was observed for both the 5- and 6-positions (<i>r</i><sup>2</sup> values of 0.52 and 0.48, respectively) which supported our hypothesis of improving the benzimidazole-Trp81 edge-to-face and/or the N3-water interaction(s).</div><figure id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Plots of BRD4 FRET mean activities vs sigma para values for substituents at the 5-position (left) and 6-position (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As outlined previously, we were also interested to examine the effects upon metabolic stability by incorporating these substituents on the benzimidazole ring. Toward this end, potent dimethylamino 5-substituted benzimidazole <b>22c</b> was assessed in microsome preparations and levels of turnover were below the lower limit of quantification in all species (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>). This result was noteworthy, given the similar lipophilicities for 5-dimethylamine <b>22c</b> and unsubstituted benzimidazole <b>20a</b> (ChromLog<i>D</i><sub>7.4</sub> = 2.9 and 2.7, respectively), and corroborated our strategy of blocking potential sites of metabolism (see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> for microsome clearance data).</div><div class="NLM_p">Based on the encouraging activities for the dimethylamines <b>22c</b> and <b>22d</b> and the low microsome clearance for the former 5-substituted congener, we prepared further amines at both the 5- and 6-positions to determine the preferred site of substitution for activity (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>).</div><figure id="sch6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Dimethylpyridone Benzimidazole 5- and 6-Position Amine Analogues <b>23a–23t</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) NHRR′, Pd<sub>2</sub>(dba)<sub>3</sub> (2 mol %), DavePhos (5 mol %), NaO<sup><i>t</i></sup>Bu, toluene or 1,4-dioxane, 80 °C; for compounds <b>23m</b> and <b>23n</b>: (ii) 4 M HCl in 1,4-dioxane, rt.</p></p></figure><div class="NLM_p">We considered all matched molecular pairs (MMPs) of <i>N</i>-substituents at both the 5- and 6-positions and found that the 5-position was preferred for activity in most cases (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). This observation also held true for the WB activity where the 5-position substituents were more potent in 10 out of the 11 MMPs. Furthermore, most 5-substituents increased BRD4 FRET and MCP-1 WB activity relative to 5-dimethylamine <b>22c</b>. The 5-morpholine <b>23e</b> was particularly potent in both the biochemical and MCP-1 WB assays (pIC<sub>50</sub> = 8.0 and 7.9, respectively) and warranted further profiling. As expected, this 4-methylene THP-containing compound exhibited BAZ2A activity (pIC<sub>50</sub> = 6.7) corresponding to only 20-fold selectivity for BRD4. Nonetheless, we sought to confirm the earlier observation that substitution at the 5-position improved metabolic stability compared to unsubstituted benzimidazole <b>20a</b>. Similar to 5-dimethylamine <b>22c</b>, no detectable turnover of 5-morpholine <b>23e</b> was measured in human microsomes (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>). These latter data, combined with the low measured lipophilicity (ChromLog<i>D</i><sub>7.4</sub> = 2.3) and high FaSSIF solubility (>1000 μg/mL, from amorphous solid), demonstrated the morpholine to be a favorable substituent at the 5-position.</div><figure id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Plots of MMP activities for 5-position (blue) and 6-position (green) substituents. BRD4 FRET data (left) and MCP-1 WB data (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having made improvements at both the benzimidazole N1- and 5-positions independently, we sought to combine the favored substituents into one molecule predicting that it would display high BRD4 and low BAZ2A activity, while possessing favorable PK and physicochemical properties for testing the mechanism in vivo. Gratifyingly, our hypothesis was correct and the profile of the resulting compound <b>24</b> (I-BET469) (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>) is now discussed.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Profile of Dimethylpyridone Benzimidazole <b>24</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0026.gif" alt="" id="GRAPHIC-d7e2491-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 FRET pIC<sub>50</sub><a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">7.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WB pIC<sub>50</sub><a class="ref internalNav" href="#t8fn1" aria-label="a">a</a> IL-6/MCP-1</td><td class="colsep0 rowsep0" align="center">7.7/7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Cell BRET p<i>K</i><sub>d</sub><a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">8.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BAZ2A FRET pIC<sub>50</sub><a class="ref internalNav" href="#t8fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChromLog<i>D</i><sub>7.4</sub>/PFI</td><td class="colsep0 rowsep0" align="center">2.6/5.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LE/LLE</td><td class="colsep0 rowsep0" align="center">0.35/6.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">Expressed as the mean from at least four test occasions.</p></div><div class="footnote" id="t8fn2"><sup><sup>b</sup></sup><p class="last">pIC<sub>50</sub> values of <4.0 were determined on four test occasions out of 10 and were excluded from the reported mean value.</p></div></div></div><div class="NLM_p">First, dimethylpyridone benzimidazole <b>24</b> displayed high BRD4 FRET and WB activities which exceeded our target levels. Activity at BRD4 in a cellular environment was also assessed for this compound using NanoBRET technology.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> Here, HEK293 cells expressing a nanoluciferase-BRD4 fusion protein were incubated with dimethylpyridone benzimidazole <b>24</b> and a bromosporine tracer. Displacement of the tracer was monitored over 2 h resulting in a cellular p<i>K</i><sub>d</sub> of 8.3, a value greater than that determined for I-BET151 (p<i>K</i><sub>d</sub> = 7.8). The kinetics of binding were also measured and revealed a marginally slower <i>k</i><sub>off</sub> for dimethylpyridone benzimidazole <b>24</b> relative to I-BET151 (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>). With regard to BCP selectivity, benzimidazole <b>24</b> was >5000-fold selective over BAZ2A because of the inclusion of the N1-dimethoxypropyl substituent. Screening in a panel of BCPs revealed the overall high selectivity for the BET family (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>); <i>K</i><sub>d</sub> values of 37 and 49 nM were measured for CREBBP and EP300, respectively but were ≥25-fold higher than that measured for BRD4 BD1 and did not preclude further investigation of the compound.</div><div class="NLM_p">A 1.3 Å resolution X-ray crystal structure of dimethylpyridone benzimidazole <b>24</b> in BRD4 BD1 was obtained (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) and showed the ligand bound in a comparable manner to the previous 4-methylene THP benzimidazole <b>20a</b>. Clear differences with dimethoxypropyl benzimidazole <b>24</b>, however, were that the branched N1-substituent simultaneously contacted the WPF-shelf and residues of the ZA loop (Leu92 and Leu94) on the opposing side of the binding pocket, and the morpholine 5-substituent was directed out of the ZA channel toward the solvent.</div><figure id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. X-ray crystal structure of dimethylpyridone benzimidazole <b>24</b> in BRD4 BD1 (carbon = magenta, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPZ">6TPZ</a>). Water molecules at the base of the binding pocket are shown as red spheres, and selected hydrogen bonds are shown as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The PFI value of dimethylpyridone benzimidazole <b>24</b> (5.6) was within our desired criteria and therefore presented a low risk toward compound-related attrition. At this stage, we were keen to determine FaSSIF solubility from a crystalline form to better understand its pharmaceutical properties and potential for development. We also required further quantities of the compound for in vivo studies, so synthesis of the crystalline material was carried out on a >220 mmol scale (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>).</div><figure id="sch7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Dimethylpyridone Benzimidazole <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1,3-dimethoxypropan-2-amine, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 97%; (b) 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (4:1), 90 °C, 56%; (c) morpholine, Pd<sub>2</sub>(dba)<sub>3</sub> (2 mol %), DavePhos (5 mol %), NaO<sup><i>t</i></sup>Bu, 2-MeTHF, 80 °C, 59%.</p></p></figure><div class="NLM_p">Nucleophilic substitution of 4-bromo-1-fluoro-2-nitrobenzene with commercially available 1,3-dimethoxypropan-2-amine afforded <i>ortho</i>-nitroaniline <b>25</b> and was followed by ring formation to give benzimidazole <b>26</b>. Palladium-catalyzed C–N cross-coupling of bromobenzimidazole <b>26</b> with morpholine followed by recrystallization from EtOAc delivered 50 g of the amination product <b>24</b>. The solid state was confirmed as crystalline via X-ray powder diffraction (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>) and subsequent solubility measurements using this material revealed solubility >1000 μg/mL in FaSSIF and >820 μg/mL in other physiological media and aqueous buffers (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>). With no detectable turnover in rat, dog, and human microsomal and hepatocyte preparations, the crystalline material was dosed in a crossover rat PK study at 1, 3, and 30 mg/kg (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>). Following oral administration, the dose-normalized AUC<sub>0–∞</sub> values for each dose group were equivalent (53 and 52 min·kg/L for the 3 and 30 mg groups, respectively), indicating a linear dose-exposure relationship and because of the low clearance and high oral absorption, oral bioavailabilities were high (ca. 94% following administration at 3 and 30 mg/kg). Additionally, PK parameters were determined in the dog and demonstrated low clearance and high oral bioavailability. Single species scaling of the rat and dog PK parameters predicted human doses within our target of <100 mg (27 and 8 mg, respectively) for 24 h coverage of the WB MCP-1 IC<sub>50</sub>. With the low predicted human dose indicating low developability risk and wider biological profiling demonstrating minimal off-target activity (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>), we progressed dimethylpyridone benzimidazole <b>24</b> into further studies.</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. In Vitro and in Vivo PK Data of Dimethylpyridone Benzimidazole <b>24</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">IVC (mL/min/g)</th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char=".">Mics</th><th class="colsep0 rowsep0" align="center" char=".">Heps</th><th class="colsep0 rowsep0" align="center" char="."><i>f</i><sub>ub</sub></th><th class="colsep0 rowsep0" align="center" char=".">po dose (mg/kg)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0–∞</sub> (ng·h/mL)</th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>b</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>u</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>dss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub>, iv (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th><th class="colsep0 rowsep0" align="center" char=".">human predicted QD dose (mg)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse</td><td class="colsep0 rowsep0" align="char" char="."><0.48</td><td class="colsep0 rowsep0" align="char" char="."><1.0</td><td class="colsep0 rowsep0" align="char" char=".">0.72</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">3618</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="."><0.46</td><td class="colsep0 rowsep0" align="char" char="."><0.80</td><td class="colsep0 rowsep0" align="char" char=".">0.69</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">2614</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">30.0</td><td class="colsep0 rowsep0" align="char" char=".">26973</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">93</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog<a class="ref internalNav" href="#t9fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t9fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char="."><0.37</td><td class="colsep0 rowsep0" align="char" char="."><1.3</td><td class="colsep0 rowsep0" align="char" char=".">0.50</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">12503</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="char" char=".">101</td><td class="colsep0 rowsep0" align="char" char=".">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char="."><0.40</td><td class="colsep0 rowsep0" align="char" char="."><0.87</td><td class="colsep0 rowsep0" align="char" char=".">0.72</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">iv dose (1.0 mg/kg): 1 h infusion in DMSO (2%, v/v) and Kleptose HPB (10%, w/v) in saline (0.9% w/v).</p></div><div class="footnote" id="t9fn2"><sup><sup>b</sup></sup><p class="last">po dose: a suspension of crystalline solid in 1% (w/v) methylcellulose (400 cps) (aq).</p></div><div class="footnote" id="t9fn3"><sup><sup>c</sup></sup><p class="last">iv dose (0.5 mg/kg): 1 h infusion in DMSO (2%, v/v) and Kleptose HPB (10%, w/v) in saline (0.9% w/v).</p></div><div class="footnote" id="t9fn4"><sup><sup>d</sup></sup><p class="last">po dose: a suspension of solid of unknown form in 1% (w/v) methylcellulose (400 cps) (aq).</p></div></div></div><div class="NLM_p">Having established potent binding and functional activity in vitro, we next wanted to demonstrate engagement of the BET mechanism in vivo. An investigation by Tarakhovsky et al. was the first to establish the link between BET inhibition and inflammation.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> These researchers treated LPS-stimulated bone marrow-derived macrophages with I-BET762 and observed suppressed expression of proinflammatory genes including IL-6. Epigenetic profiling provided further insight into the mechanism with the promoter of IL-6 displaying reduced BET recruitment in the presence of I-BET762. Furthermore, experiments in vivo demonstrated that inhibition of BET in a mouse model of endotoxic shock attenuated release of IL-6. We therefore sought to use IL-6 as a biomarker in LPS-challenged mice to establish the suitability of dimethylpyridone benzimidazole <b>24</b> to engage the BET mechanism in vivo. After confirming that adequate exposure would be achieved following oral dosing to the male CD-1 mouse (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>), dimethylpyridone benzimidazole <b>24</b> was administered orally at three different concentrations (1, 3, and 10 mg/kg) 30 min prior to LPS challenge and samples were taken at regular intervals for PKPD analysis over the following 5 h. Immunoassay of mouse IL-6 revealed suppression of cytokine levels in line with increasing unbound exposure over the duration of the study (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).</div><figure id="fig9" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. IL-6 (red) and unbound dimethylpyridone benzimidazole <b>24</b> (blue) AUC concentrations in the acute LPS model of inflammation. Statistical differences in %inhibition of IL-6 AUC levels were determined as <i>p</i> <0.05 (*) and <i>p</i> <0.001 (***).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having confirmed BET-mediated effects in an acute in vivo model, we next investigated dimethylpyridone benzimidazole <b>24</b> in a chronic immune-mediated model more relevant to human immune disease. The ability of BET inhibitors to interfere with the immune response through dampened T cell and B cell activation has previously been established.<a onclick="showRef(event, 'ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref78 ref79">(78,79)</a> We therefore utilized a T cell-dependent immunization model wherein male CD-1 mice were challenged with the 2,4,6-trinitrophenyl keyhole limpet hemocyanin (TNP-KLH) antigen, and IgG1 levels were measured following QD or once every other day (QOD) oral administration of dimethylpyridone benzimidazole <b>24</b> for 14 days. Greater inhibition of IgG1 levels relative to the vehicle on day 14 was achieved in the 3 mg/kg QD versus the 0.3 mg/kg QD dose group (20 and 65%, respectively) which was in accordance with the higher average free drug concentrations (<i>C</i><sub>av</sub>) (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). Additionally, similar suppression of IgG1 levels compared to the vehicle was observed with the 3 mg/kg QOD regimen versus QD (75 and 65%, respectively). The manifestation of auto-antigen presentation to T cells, and consequent auto-antibody production, fundamentally underlies auto-immune disease. Therefore, the demonstration that BET inhibition can attenuate antibody production in T cell-dependent immunization may allow such an approach to be used for clinical benefit in these conditions.</div><figure id="fig10" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Mean %inhibition of IgG1 (relative to the vehicle) at day 14 vs free average drug concentration. Statistical differences in % inhibition of IgG1 were determined as <i>p</i> < 0.01 (**).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Taken together, these experiments demonstrate that dimethylpyridone benzimidazole <b>24</b> possesses suitable properties to engage the BET mechanism in vivo and elicit potent immunomodulatory effects. Moreover, the potency and oral PK properties of this molecule provide the flexibility to model different dosing regimens and levels of mechanism engagement to refine human dose predictions for different disease indications. Overall, the data presented here make this BET inhibitor a potential candidate for assessment in humans.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36744" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36744" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have described the discovery of a hit 3,5-dimethylphenol benzimidazole series using ELT for an oral BET inhibitor program. An X-ray crystal structure revealed the dimethylphenol moiety to be the KAc mimetic. Hybridization of this with an <i>N</i>-methylpyridone fragment-derived series led to increased potency and lowered lipophilicity. A compound containing this KAc mimetic was selected as a lead because of its high potency, encouraging physicochemical properties, and excellent PK profile in dogs, but it exhibited off-target BCP activity and high clearance in the rat leading to a high predicted human dose. As part of lead optimization efforts, structural information guided an exploration at the benzimidazole N1-position and minor modifications to the 4-methylene THP substituent were sufficient to abolish BAZ2A activity. In parallel with these investigations, substituents were included at the 5- and 6-positions and electron-donating groups were found to increase activity while also improving metabolic stability. Combination of the optimized N1-dimethoxypropyl and 5-morpholine groups resulted in dimethylpyridone benzimidazole <b>24</b>, a compound possessing high BRD4 and WB activity, and favorable physicochemical properties. This 3,5-dimethylpyridone benzimidazole is devoid of BAZ2A activity and possesses PK properties leading to predicted human doses of <30 mg and thereby minimizes risk associated with compound-related attrition. Finally, engagement of the BET mechanism and immunomodulatory effects were demonstrated in vivo confirming dimethylpyridone benzimidazole <b>24</b> as a candidate-quality molecule.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56138" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56138" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Chemistry Methods and Characterization for Compounds <b>8–26</b></h3><div class="NLM_p">All commercial chemicals and solvents used were of reagent grade and used without further purification. Hydrophobic frit cartridges by ISOLUTE, which contain a frit selectively permeable to organic solutions, were used for separation of organic phases from aqueous phases under gravity. ISOLUTE aminopropyl cartridges were used for scavenging SPE protocols. ISOLUTE C18 cartridges were used for SPE workup of palladium-catalyzed amination reactions. Column chromatography was carried out either using manual or automated flash chromatography systems including a Biotage SP4 using SNAP silica cartridges or a CombiFlash using RediSep silica cartridges. Preparative high-performance liquid chromatography (HPLC) and mass-directed autopreparative HPLC (MDAP) purification were conducted using a Waters ZQ MS using alternate-scan positive and negative electrospray using either of the following two LC methods: method A: Either a SunFire C18 column (100 mm × 19 mm, 5 μm packing diameter) at a 20 mL/min flow rate or a SunFire C18 column (150 mm × 30 mm, 5 μm packing diameter) at a 40 mL/min flow rate. Gradient elution was carried out at ambient temperature, with the mobile phases as (A) water containing 0.1% (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. The UV detection was a summed signal from wavelengths of 210 to 350 nm. Method B: either an Xbridge C18 column (100 mm × 19 mm, 5 μm packing diameter) at a 20 mL/min flow rate or an Xbridge C18 column (150 mm × 30 mm, 5 μm packing diameter) at a 40 mL/min flow rate. Gradient elution was carried out at ambient temperature, with the mobile phases as (A) 10 mM ammonium bicarbonate in water solution, adjusted to pH 10 with 0.88 ammonia solution, and (B) acetonitrile. The UV detection was a summed signal from wavelengths of 210 to 350 nm. Microwave irradiation was carried out using either an Anton Paar Multiwave Pro or a Biotage I60 system. Melting points were measured using a Buchi M-565 automatic melting point apparatus. IR spectra were obtained on a PerkinElmer Spectrum 2 spectrometer. <sup>1</sup>H NMR spectra were recorded on either a Bruker AV-400 spectrometer at 400 MHz or a Bruker AV-600 spectrometer at 600 MHz or Varian VNMR400 at 400 MHz. <sup>13</sup>C NMR spectra were recorded on a Bruker AV-600 spectrometer at 150 MHz. NMR spectra were acquired at 293 K, and chemical shifts (δ) are reported in parts per million (ppm) relative to CHCl<sub>3</sub> (δ 7.26), dimethyl sulfoxide (DMSO; δ 2.50), and MeOH (δ 3.31). Coupling constants (<i>J</i>) are expressed in hertz (Hz) to the nearest 0.5 Hz. HRMS were recorded on a Micromass Q-Tof Ultima hybrid quadrupole time-of-flight mass spectrometer after liquid chromatography separation using an Agilent 1100 Liquid Chromatograph equipped with a Phenomenex Luna C18 reversed phase column (100 mm × 2.1 mm, 3 μm packing diameter). Elution was carried out at 35 °C with the mobile phases as (A) 0.1% formic acid in water and (B) 0.1% formic acid in acetonitrile using a gradient starting at 5% B, increasing linearly to 100% B over 6 min, and remaining at 100% B for 2.5 min. The flow rate was 0.5 mL/min. The purity of all biologically tested compounds was ≥95% as determined by LC–MS UV traces except for compound <b>19b</b> (93% purity). The specific LC–MS UV method used for purity determination is noted for each compound and is from any of the following: method A: Waters ZQ Acquity UPLC system equipped with an CSH C18 column (50 mm × 2.1 mm, 1.7 μm packing diameter). Elution was carried out at 40 °C with the mobile phases as (A) 10 mM ammonium bicarbonate in water solution, adjusted to pH 10 with 0.88 ammonia solution, and (B) acetonitrile using a gradient starting at 3% B, increasing linearly to 95% B over 1.5 min, and remaining at 95% B for 0.4 min. The flow rate was 1 mL/min. The UV detection was a summed signal from wavelengths of 210–350 nm. MS analysis was carried out using alternate-scan positive and negative electrospray. Method B: Kinetex C8 column (30 mm × 2.1 mm, 1.7 μm packing diameter). Elution was carried out at rt with the mobile phases as (A) 0.1% formic acid in water and (B) 0.1% formic acid in acetonitrile using a gradient starting at 5% B, increasing linearly to 95% B over 2.75 min, and remaining at 95% B for 0.25 min. The flow rate was 0.65 mL/min. The UV detection was a summed signal from wavelengths of 210–400 nm. MS analysis was carried out using alternate-scan positive and negative electrospray. Method C: Shimadzu 2010 Mass Spectrometer equipped with an Xtimate C18 column (30 mm × 2.1 mm, 3 μm packing diameter). Elution was carried out at 50 °C with the mobile phases as (A) 0.04% trifluoroacetic acid (TFA) in water and (B) 0.02% TFA in acetonitrile using a gradient starting at 0% B, increasing linearly to 60% B over 0.90 min, and remaining at 60% B for 0.60 min. The flow rate was 1.2 mL/min. The UV detection was performed at 220 nm. MS analysis was carried out using positive electrospray. Method D: as method C but using a gradient starting at 10% B, increasing linearly to 80% B over 0.90 min, and remaining at 80% B for 0.60 min. Method E: as method C but using a gradient starting at 0% B, increasing linearly to 30% B over 0.90 min, and remaining at 30% B for 0.60 min. Method F: the Waters ZQ Acquity UPLC system equipped with an CSH C18 column (50 mm × 2.1 mm, 1.7 μm packing diameter). Elution was carried out at 40 °C with the mobile phases as (A) 0.1% formic acid in water and (B) 0.1% formic acid in acetonitrile using a gradient starting at 3% B, increasing linearly to 95% B over 1.5 min, and remaining at 95% B for 0.4 min. The flow rate was 1 mL/min. The UV detection was a summed signal from wavelengths of 210 to 350 nm. MS analysis was carried out using alternate-scan positive and negative electrospray.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 2-(4-Hydroxy-3,5-dimethylphenyl)-<i>N</i>-methyl-1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide Trifluoroacetic Acid Salt (<b>8</b>)</h4><div class="NLM_p last">To a solution of 4-fluoro-<i>N</i>-methyl-3-nitrobenzamide (19.8 mg, 0.10 mmol) in EtOH (2 mL) were added <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (21.4 mg, 0.10 mmol) and diisopropylethylamine (DIPEA; 52 μL, 0.30 μmol). The reaction mixture was heated at 80 °C overnight, cooled to rt, and concentrated to dryness. The resulting crude was dissolved in EtOH (2 mL), and 4-hydroxy-3,5-dimethylbenzaldehyde (30.0 mg, 0.20 mmol) was added followed by Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (104 mg, 0.60 mmol) in water (0.5 mL). The reaction mixture was stirred in a sealed tube at 80 °C for 18 h, cooled to rt, and concentrated to a small volume. The residue was diluted with EtOAc (2 mL), and the organic layer was washed with saturated aqueous NaHCO<sub>3</sub> (2 mL). The layers were separated, and the aqueous layer was extracted with EtOAc (2 × 2 mL). The organics were combined, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to dryness to give crude <i>tert</i>-butyl 4-((2-(4-hydroxy-3,5-dimethylphenyl)-5-(methylcarbamoyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate. This was treated with 50% TFA in dichloromethane (DCM; 1 mL) for 3 h at rt and then concentrated to dryness. The crude was purified directly by preparative HPLC (SunFire C8 OBD Prep Column [30 mm × 50 mm, 5 μm], 40 mL/min, water/MeCN containing 0.1% TFA, UV detection at 220 nm) to give the title compound (20 mg, 0.05 mmol, 38% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.10 (br s, 1 H), 8.55 (br s, 1 H), 8.19 (s, 1 H), 8.03 (br s, 1 H), 7.89 (m, 2 H), 7.44 (s, 2 H), 4.40 (d, 2 H, <i>J</i> = 7.0 Hz), 3.17–3.10 (m, 2 H), 2.82 (d, 3 H, <i>J</i> = 4.5 Hz), 2.75–2.65 (m, 2 H), 2.27 (s, 6 H), 2.06 (br s, 1 H), 1.54–1.45 (m, 2 H), 1.20–1.05 (m, 2 H); LC–MS (method B) <i>m</i>/<i>z</i>: 393 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.78 min; 95% purity.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 1-((1-Benzoylpiperidin-4-yl)methyl)-2-(4-hydroxy-3,5-dimethylphenyl)-<i>N</i>-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide Trifluoroacetic Acid Salt (<b>9</b>)</h4><div class="NLM_p last">To a solution of benzoic acid (12.2 mg, 0.10 mmol) in dimethylformamide (DMF; 0.5 mL) were added 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (HATU; 38.0 mg, 0.10 mmol) and DIPEA (52 μL, 0.30 mmol). The reaction mixture was stirred at rt for 5 min and added to a solution of 2-(4-hydroxy-3,5-dimethylphenyl)-<i>N</i>-methyl-1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide trifluoroacetic acid salt <b>8</b> (20 mg, 0.38 mmol) in DMF (0.5 mL). The reaction mixture was stirred at rt overnight and diluted with EtOAc (2 mL). The organic layer was washed with saturated aqueous NaHCO<sub>3</sub> (2 mL), and the layers were separated. The aqueous layer was extracted with EtOAc (2 × 2 mL), and the organics were combined, dried over Na<sub>2</sub>SO<sub>4</sub>, and filtered. The filtrate was concentrated to dryness, and the crude material was purified by preparative HPLC (SunFire C8 OBD Prep Column [30 mm × 50 mm, 5 μm], 40 mL/min, water/MeCN containing 0.1% TFA, UV detection at 220 nm) to give the title compound (13 mg, 0.03 mmol, 53% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.14 (br s, 1 H), 8.60–8.50 (m, 1 H), 8.16 (s, 1 H), 8.01–7.89 (m, 2 H), 7.46 (s, 2 H), 7.42–7.36 (m, 3 H), 7.31–7.25 (m, 2 H), 4.41 (m, 2 H), 4.37–4.21 (m, 2 H), 3.53–3.32 (m, 2 H), 2.8 (d, 3 H, <i>J</i> = 4.5 Hz), 2.27 (s, 6 H), 2.06–1.93 (m, 1 H), 1.42–1.30 (m, 2 H), 1.11–0.96 (m, 2 H); LC–MS (method B) <i>m</i>/<i>z</i>: 497 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.30 min; 95% purity.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 1-((1-Acetylpiperidin-4-yl)methyl)-2-(4-hydroxy-3,5-dimethylphenyl)-<i>N</i>-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide Trifluoroacetic Acid Salt (<b>10</b>)</h4><div class="NLM_p last">To a solution of 2-(4-hydroxy-3,5-dimethylphenyl)-<i>N</i>-methyl-1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide trifluoroacetic acid salt <b>8</b> (20 mg, 0.38 mmol) in a mixture of tetrahydrofuran (THF; 0.5 mL) and MeCN (0.5 mL) were added pyridine (40 μL, 0.50 mmol) and acetic anhydride (28 μL, 0.30 mmol) at 0 °C. The reaction mixture was stirred at rt overnight and concentrated to dryness. The crude was partitioned between EtOAc (1 mL) and aqueous NaHCO<sub>3</sub> (1 mL), and the layers were separated. The aqueous layer was extracted with EtOAc (2 × 1 mL), the organics combined, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The material was dissolved in a mixture of THF (2 mL)/MeOH (0.5 mL) and treated with 2 M LiOH (0.15 mL) for 3 h. The reaction mixture was concentrated to dryness, and the crude material was purified by preparative HPLC (SunFire C8 OBD Prep Column [30 mm × 50 mm, 5 μm], 40 mL/min, water/MeCN containing 0.1% TFA, UV detection at 220 nm) to give the title compound (16 mg, 0.29 mmol, 78% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.15 (br s, 1 H), 8.61–8.56 (m, 1 H), 8.18 (s, 1 H), 7.97 (d, 1 H, <i>J</i> = 8.0 Hz), 7.93 (d, 1 H, <i>J</i> = 8.0 Hz), 7.47 (s, 2 H), 4.45–4.35 (m, 2 H), 4.22–4.13 (m, 1 H), 3.70–3.60 (m, 1H), 2.82 (d, 3 H, <i>J</i> = 4.5 Hz), 2.39–2.25 (m, 2 H), 2.27 (s, 6 H), 2.00–1.92 (m, 1 H), 1.45–1.35 (m, 1 H), 1.32–1.22 (m, 1 H), 1.10–0.95 (m, 1 H), 0.89–0.75 (m, 1 H); LC–MS (method F) <i>m</i>/<i>z</i>: 435 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.52 min; 100% purity.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>tert</i>-Butyl 4-(((2-Nitrophenyl)amino)methyl)piperidine-1-carboxylate (<b>11a</b>)</h4><div class="NLM_p last">A mixture of 1-fluoro-2-nitrobenzene (40 g, 283 mmol), <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (72.9 g, 340 mmol), and K<sub>2</sub>CO<sub>3</sub> (78 g, 567 mmol) in MeCN (1500 mL) was stirred at 80 °C overnight. The reaction mixture was washed with 10% aqueous citric acid, saturated aqueous NaHCO<sub>3</sub>, and brine (×3). The organic layer was dried with Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the crude material. The residue was purified by column chromatography on silica gel eluting with petroleum ether/EtOAc (10/1 to 3:1, v/v) to give the title compound as a crude yellow solid (117 g, 337 mmol). LC–MS (method C) <i>m</i>/<i>z</i>: 280 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.49 min; 97% purity.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>tert</i>-Butyl 4-((2-Nitrophenyl)amino)piperidine-1-carboxylate (<b>11b</b>)</h4><div class="NLM_p last">A mixture of 1-fluoro-2-nitrobenzene (5 g, 35.4 mmol), K<sub>2</sub>CO<sub>3</sub> (9.79 g, 70.9 mmol) and <i>tert</i>-butyl 4-aminopiperidine-1-carboxylate (8.52 g, 42.5 mmol) in MeCN (354 mL) was stirred at 110 °C overnight. To the reaction mixture was added EtOAc and washed with 10% aqueous citric acid, saturated aqueous NaHCO<sub>3</sub>, and brine (×3). The organic was dried with Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the crude material. The residue was purified by column chromatography on silica gel eluting with petroleum ether/EtOAc (10:1 to 3:1) to give the title compound as a yellow solid (11.5 g, 34.2 mmol, 97% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 266 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>], 344 [(M + Na)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.47 min; 96% purity.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>tert</i>-Butyl 3-(((2-Nitrophenyl)amino)methyl)azetidine-1-carboxylate (<b>11c</b>)</h4><div class="NLM_p last">To a solution of 1-fluoro-2-nitrobenzene (5 g, 35.4 mmol) in MeCN (200 mL) were added <i>tert</i>-butyl 3-(aminomethyl)azetidine-1-carboxylate (7.92 g, 42.5 mmol) and K<sub>2</sub>CO<sub>3</sub> (9.79 g, 70.9 mmol). The reaction mixture was stirred at 80 °C overnight. The mixture was filtered, and the filtrate was concentrated in vacuo to give a residue which was diluted with EtOAc (200 mL). The solution was washed with brine (3 × 50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by flash column chromatography eluting with 100% EtOAc in petroleum ether to give the title compound as a bright yellow solid (11.04 g, 34.8 mmol, 98% yield). LC–MS (method D) <i>m</i>/<i>z</i>: 252 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>], 330 [(M + Na)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.10 min; 97% purity.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>tert</i>-Butyl 3-(((2-Nitrophenyl)amino)methyl)pyrrolidine-1-carboxylate (<b>11d</b>)</h4><div class="NLM_p last">A mixture of 1-fluoro-2-nitrobenzene (3 g, 21.2 mmol), K<sub>2</sub>CO<sub>3</sub> (5.88 g, 42.5 mmol), and <i>tert</i>-butyl 3-(aminomethyl)pyrrolidine-1-carboxylate (5.11 g, 25.5 mmol) in MeCN (213 mL) was stirred at 110 °C overnight. To the reaction mixture was added EtOAc, and this was washed with 10% aqueous citric acid, saturated aqueous NaHCO<sub>3</sub>, and brine (× 3). The organic layer was dried with Na<sub>2</sub>SO<sub>4</sub> and concentrated to give the crude material. The residue was purified by column chromatography on silica gel eluting with petroleum ether/EtOAc (10:1 to 3:1) to give the title compound as a yellow solid (6.8 g, 21.0 mmol, 99% yield). LC–MS (method D) <i>m</i>/<i>z</i>: 266 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>], 344 [(M + Na)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.17 min; 99% purity.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>tert</i>-Butyl (<i>trans</i>-4-(((2-Nitrophenyl)amino)methyl)cyclohexyl)carbamate (<b>11e</b>)</h4><div class="NLM_p last">To 1-fluoro-2-nitrobenzene (2.6 g, 18.4 mmol) in MeCN (113 mL) were added <i>tert</i>-butyl (<i>trans</i>-4-(aminomethyl)cyclohexyl)carbamate (5.05 g, 22.1 mmol) and K<sub>2</sub>CO<sub>3</sub> (2.55 g, 18.4 mmol). The reaction mixture was stirred at 100 °C. After cooling to rt, the reaction mixture was filtered on a pad of Celite. The solvent was washed twice with saturated aqueous NaHCO<sub>3</sub> and water and then dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo to give the crude product as a yellow gum. This was purified by column chromatography on silica gel eluting with petroleum ether/EtOAc (5:1 to 4:1) to give the title compound as a yellow gum (5.93 g, 17.0 mmol, 92% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 294 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.49 min; 100% purity.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>tert</i>-Butyl (<i>cis</i>-4-(((2-Nitrophenyl)amino)methyl)cyclohexyl)carbamate (<b>11f</b>)</h4><div class="NLM_p last">A mixture of 1-fluoro-2-nitrobenzene (1.54 g, 10.9 mmol), <i>tert</i>-butyl (<i>cis</i>-4-(aminomethyl)cyclohexyl)carbamate (2.99 g, 13.1 mmol), and K<sub>2</sub>CO<sub>3</sub> (3.02 g, 21.8 mmol) in MeCN (72.8 mL) was stirred at 100 °C overnight. The reaction mixture was filtered and evaporated in vacuo to dryness and purified by column chromatography on silica gel eluting with petroleum ether/EtOAc (5:1 to 1:1) to give the title compound as a yellow solid (3.8 g, 10.9 mmol, 100% yield). LC–MS (method D) <i>m</i>/<i>z</i>: 294 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.22 min; 100% purity.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>tert</i>-Butyl 4-(((2-Aminophenyl)amino)methyl)piperidine-1-carboxylate (<b>12a</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 4-(((2-nitrophenyl)amino)methyl)piperidine-1-carboxylate <b>11a</b> (50 g, 149 mmol) and Pd/C (1 g, 0.940 mmol) in DMF (300 mL) was stirred at 30 °C overnight under 1 atm of hydrogen. The reaction mixture was filtered through a pad of Celite and was washed with EtOAc (3 × 45 mL) and brine. The combined organics was concentrated to give the title compound as a dark brown solid (43 g, 140 mmol, 94% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 206 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.13 min; 99% purity.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>tert</i>-Butyl 4-((2-Aminophenyl)amino)piperidine-1-carboxylate (<b>12b</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 4-((2-nitrophenyl)amino)piperidine-1-carboxylate <b>11b</b> (11.5 g, 35.8 mmol) and palladium (1.15 g, 1.08 mmol) in DMF (300 mL) was stirred at 30 °C overnight under 1 atm of hydrogen. The reaction mixture was filtered through a pad of Celite and was washed with EtOAc (3 × 45 mL) and brine. The combined organics was concentrated to give the title compound as a dark brown solid (10.4 g, 33.7 mmol, 94% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 236 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>], 292 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.05 min; 94% purity.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>tert</i>-Butyl 3-(((2-Aminophenyl)amino)methyl)azetidine-1-carboxylate (<b>12c</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 3-(((2-nitrophenyl)amino)methyl)azetidine-1-carboxylate <b>11c</b> (11.04 g, 34.8 mmol) in DMF (300 mL) was treated with Pd/C (1.0 g, 9.40 mmol). The reaction mixture was hydrogenated under 30 psi pressure at 30 °C overnight. The mixture was filtered through a pad of Celite. The solution was diluted with EtOAc (500 mL), washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give the title compound as a yellow solid (8.89 g, 31.7 mmol, 91% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 178 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.83 min; 89% purity.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>tert</i>-Butyl 3-(((2-Aminophenyl)amino)methyl)pyrrolidine-1-carboxylate (<b>12d</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 3-(((2-nitrophenyl)amino)methyl)pyrrolidine-1-carboxylate <b>11d</b> (6.8 g, 21.2 mmol) and palladium (0.72 g, 0.68 mmol) in DMF (300 mL) was stirred at 30 °C overnight under 1 atm of hydrogen. The reaction mixture was filtered through a pad of Celite and was washed with EtOAc (3 × 45 mL) and brine. The combined organic extracts were concentrated to give the title compound as a dark brown solid (6.3 g, 20.9 mmol, 99% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 192 [(M – <sup><i>t</i></sup>Bu)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.06 min; 97% purity.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>tert</i>-Butyl (<i>cis</i>-4-(((2-Nitrophenyl)amino)methyl)cyclohexyl)carbamate (<b>12f</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl (<i>cis</i>-4-(((2-nitrophenyl)amino)methyl)cyclohexyl)carbamate <b>11f</b> (3.8 g, 10.9 mmol) was hydrogenated utilizing Pd/C (0.231 g, 2.18 mmol) under 30 psi pressure in DMF (150 mL) overnight. The reaction mixture was filtered and evaporated in vacuo to dryness to give the title compound (3.32 g, 10.2 mmol, 93% yield). LC–MS (method D) <i>m</i>/<i>z</i>: 320 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.89 min; 91% purity.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>tert</i>-Butyl 4-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate (<b>13a</b>)</h4><div class="NLM_p last">To a mixture of <i>tert</i>-butyl 4-(((2-aminophenyl)amino)methyl)piperidine-1-carboxylate <b>12a</b> (423 mg, 1.385 mmol) and 4-hydroxy-3,5-dimethylbenzaldehyde (208 mg, 1.39 mmol) in DMF (3 mL) was added Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (342 mg, 1.80 mmol). The reaction mixture was stirred at 100 °C overnight. The mixture was cooled and filtered. The mixture was diluted with EtOAc (30 mL), washed with brine (3 × 10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified on silica gel eluting with 0–3% MeOH in DCM to give the title compound as a yellow solid (469 mg, 1.07 mmol, 77% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 436 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.97 min; 99% purity.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>tert</i>-Butyl 4-(2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)piperidine-1-carboxylate (<b>13b</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 4-((2-aminophenyl)amino)piperidine-1-carboxylate <b>12b</b> (10.4 g, 35.7 mmol), 4-hydroxy-3,5-dimethylbenzaldehyde (5.36 g, 35.7 mmol), and Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (8.82 g, 46.4 mmol) in DMF (357 mL) was stirred at 100 °C under nitrogen overnight. To the mixture was added EtOAc (50 mL) which was then washed with brine (×3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>. The crude was purified by column chromatography on silica gel eluting with petroleum ether/EtOAc (20:1 to 3:1) to give the title compound as a pale solid (10.6 g, 24.8 mmol, 70% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 422 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.17 min; 99% purity.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>tert</i>-Butyl 3-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)azetidine-1-carboxylate (<b>13c</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 3-(((2-aminophenyl)amino)methyl)azetidine-1-carboxylate <b>12c</b> (8.89 g, 32.1 mmol) and 4-hydroxy-3,5-dimethylbenzaldehyde (4.81 g, 32.1 mmol) in DMF (80 mL) was added Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (7.92 g, 41.7 mmol). The reaction mixture was stirred at 100 °C overnight. The mixture was filtered, and the solid was washed with DCM (300 mL). The filtrate was washed with water (3 × 100 mL) and brine (3 × 50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give a residue which was purified on silica gel eluting with 1–2% MeOH in DCM to give the title compound as a yellow solid (5.27 g, 12.3 mmol, 38% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 408 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.10 min; 95% purity.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>tert</i>-Butyl 3-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)pyrrolidine-1-carboxylate (<b>13d</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 3-(((2-aminophenyl)amino)methyl)pyrrolidine-1-carboxylate <b>12d</b> (6.3 g, 21.6 mmol), 4-hydroxy-3,5-dimethylbenzaldehyde (3.08 g, 20.5 mmol), and Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (5.34 g, 28.1 mmol) in DMF (216 mL) was stirred at 100 °C under nitrogen overnight. The mixture was added to EtOAc (50 mL), and this was washed with brine (×3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>. The crude was purified by column chromatography on silica gel eluting with petroleum ether/EtOAc (20:1 to 3:1) to give the title compound as a pale solid (6.6 g, 15.3 mmol, 71% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 422 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.15 min; 97% purity.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>tert</i>-Butyl (<i>trans</i>-4-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-1-yl)methyl)cyclohexyl)carbamate (<b>13e</b>)</h4><div class="NLM_p last">Step b: <i>tert</i>-Butyl (<i>trans</i>-4-(((2-nitrophenyl)amino)methyl)cyclohexyl)carbamate <b>11e</b> (5.93 g, 17.0 mmol) was hydrogenated utilizing Pd/C (0.59 g, 5.54 mmol) under 30 psi pressure in DMF (150 mL) at 30 °C overnight. After cooling to rt, the reaction mixture was filtered and washed with EtOAc (25 mL). The solution was concentrated in vacuo to give the crude <i>tert</i>-butyl (<i>trans</i>-4-(((2-aminophenyl)amino)methyl)cyclohexyl)carbamate <b>12e</b> which was used for the next step directly without purification. Step c: to crude <i>tert</i>-butyl (<i>trans</i>-4-(((2-aminophenyl)amino)methyl)cyclohexyl)carbamate <b>12e</b> (5.3 g, 16.6 mmol) in DMF (150 mL) were added 4-hydroxy-3,5-dimethylbenzaldehyde (2.49 g, 16.6 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (3.78 g, 19.9 mmol). The reaction mixture was stirred under nitrogen at 100 °C overnight. Water was added to the reaction mixture, and the aqueous layer was extracted with DCM (2 × 100 mL). The combined organic layers were washed with brine and water and then dried over Na<sub>2</sub>SO<sub>4</sub>. The crude product was chromatographed on silica gel eluting with petroleum ether/EtOAc (3:1, 2:1, 1:1) to give the title compound as a pale white solid (7 g, 15.4 mmol, 93% overall yield). LC–MS (method C) <i>m</i>/<i>z</i>: 450 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.18 min; 100% purity.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>tert</i>-Butyl (<i>cis</i>-4-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)cyclohexyl)carbamate (<b>13f</b>)</h4><div class="NLM_p last">A mixture of <i>tert</i>-butyl (<i>cis</i>-4-(((2-nitrophenyl)amino)methyl)cyclohexyl)carbamate <b>12f</b> (3.48 g, 10.9 mmol), 4-hydroxy-3,5-dimethylbenzaldehyde (1.72 g, 11.4 mmol), and Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (2.69 g, 14.1 mmol) in DMF (150 mL) was stirred under nitrogen at 100 °C overnight. The reaction mixture was treated with brine, and the organic layer was washed with brine (×2) and then dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo to give the crude which was purified by column chromatography on silica gel eluting with petroleum ether/EtOAc (3:1 to 1:1) to give the title compound as a white solid (4.1 g, 9.1 mmol, 83% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 450 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.19 min; 99% purity.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 2,6-Dimethyl-4-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)phenol Dihydrochloride (<b>14a</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 4-((2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate <b>13a</b> (200 mg, 0.46 mmol) in DCM (2 mL) was charged with 4 M HCl in MeOH (2 mL, 8.00 mmol). The reaction was stirred at 50 °C for 1 h. The solvent was removed under high vacuum to give the title compound as a yellow solid (186 mg, 0.46 mmol, 99% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 336 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.82 min; 100% purity.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2,6-Dimethyl-4-(1-(piperidin-4-yl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)phenol Dihydrochloride (<b>14b</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 4-(2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)piperidine-1-carboxylate <b>13b</b> (3.6 g, 8.54 mmol) and 4 M HCl in MeOH (80 mL, 320 mmol) was stirred at rt for 1 h. The solvent was removed, and the residue was washed with EtOAc (3 × 5 mL) and evaporated to give the title compound as a yellow solid (3 g, 7.61 mmol, 89% yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.89–9.71 (m, 1 H), 9.41 (br s, 1 H), 9.03 (d, <i>J</i> = 9.0 Hz, 1 H), 8.63–8.55 (m, 1 H), 7.88–7.79 (m, 1 H), 7.63–7.53 (m, 2 H), 7.40 (s, 2 H), 4.94–4.78 (m, 1 H), 3.12 (q, <i>J</i> = 11.0 Hz, 2 H), 2.93 (q, <i>J</i> = 11.0 Hz, 2 H), 2.30 (s, 6 H), 2.21 (d, <i>J</i> = 11.0 Hz, 2 H), two piperidine proton signals obscured by DMSO; LC–MS (method C) <i>m</i>/<i>z</i>: 322 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.80 min; 100% purity.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-(1-(Azetidin-3-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-2,6-dimethylphenol Dihydrochloride (<b>14c</b>)</h4><div class="NLM_p last">Batch 1: a solution of <i>tert</i>-butyl 3-((2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)azetidine-1-carboxylate <b>13c</b> (1.0 g, 2.454 mmol) in 4 M HCl in MeOH (20 mL, 80 mmol) was stirred at 45 °C for 1 h. The solvent was removed in vacuo to give the crude product. Batch 2: a solution of <i>tert</i>-butyl 3-((2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)azetidine-1-carboxylate <b>13c</b> (5.27 g, 12.93 mmol) in 4 M HCl in MeOH (100 mL, 400 mmol) was stirred at 45 °C for 1 h. The solvent was removed in vacuo to give the crude product. Both batches were combined and washed with 5% MeOH in DCM to afford the title compound as a pink solid (5.1 g, 12.2 mmol, 79% yield). LC–MS (method E) <i>m</i>/<i>z</i>: 308 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.81 min; 91% purity.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 2,6-Dimethyl-4-(1-(pyrrolidin-3-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)phenol Dihydrochloride (<b>14d</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl 3-((2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)pyrrolidine-1-carboxylate <b>13d</b> (1 g, 2.37 mmol) and 4 M HCl in MeOH (20 mL, 80 mmol) was stirred at rt for 1 h. The solvent was removed, and the residue was washed with EtOAc (3 × 5 mL) and evaporated to give the title compound as a brown solid (0.87 g, 2.21 mmol, 93% yield). <sup>1</sup>H NMR (300 MHz, MeOH-<i>d</i><sub>4</sub>): δ 8.16–8.09 (m, 1 H), 7.88–7.81 (m, 1 H), 7.75–7.64 (m, 2 H), 7.57 (s, 2 H), 4.85 (d, <i>J</i> = 6.5 Hz, 2 H), 3.43–3.34 (m, 1 H), 3.28–3.20 (m, 1 H), 3.20–3.08 (m, 1 H), 3.00–2.81 (m, 2 H), 2.37 (s, 6 H), 2.00 (d, <i>J</i> = 4.5 Hz, 1 H), 1.68–1.51 (m, 1 H); LC–MS (method C) <i>m</i>/<i>z</i>: 322 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.79 min; 100% purity.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-(1-((<i>trans</i>-4-Aminocyclohexyl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-2,6-dimethylphenol Dihydrochloride (<b>14e</b>)</h4><div class="NLM_p last">A suspension of <i>tert</i>-butyl (<i>trans</i>-4-((2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)cyclohexyl)carbamate <b>13e</b> (1 g, 2.22 mmol) in 4 M HCl in MeOH (20 mL, 80 mmol) and MeOH (20 mL) was stirred at rt for 3 h. The reaction mixture was evaporated in vacuo to give the title compound as a yellow gum (0.7 g, 1.66 mmol, 75% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 8.05–8.00 (m, 1 H), 7.85–7.80 (m, 1 H), 7.71–7.66 (m, 2 H), 7.52 (s, 2 H), 4.54 (d, <i>J</i> = 7.5 Hz, 2 H), 3.04–2.94 (m, 1 H), 2.36 (s, 6 H), 1.96 (d, <i>J</i> = 13.0 Hz, 2 H), 1.93–1.86 (m, 1 H), 1.65 (d, <i>J</i> = 13.0 Hz, 2 H), 1.26 (q, <i>J</i> = 13.0 Hz, 2 H), 1.06 (q, <i>J</i> = 13.0 Hz, 2 H); LC–MS (method C) <i>m</i>/<i>z</i>: 350 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.84 min; 99% purity.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 4-(1-((<i>cis</i>-4-Aminocyclohexyl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-2,6-dimethylphenol Dihydrochloride (<b>14f</b>)</h4><div class="NLM_p last">A mixture of <i>tert</i>-butyl (<i>cis</i>-4-((2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)cyclohexyl)carbamate <b>13f</b> (1 g, 2.22 mmol) and 4 M HCl in MeOH (20 mL, 80 mmol) in MeOH (10 mL) was stirred at rt overnight. The reaction mixture was evaporated in vacuo to give the title compound as a pale green solid (0.73 g, 1.71 mmol, 77% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 8.09–8.00 (m, 1 H), 7.86–7.80 (m, 1 H), 7.72–7.63 (m, 2 H), 7.55 (s, 2 H), 4.64 (d, <i>J</i> = 7.5 Hz, 2 H), 3.29–3.26 (m, 1 H), 2.37 (s, 6 H), 2.15–2.04 (m, 1 H), 1.74–1.58 (m, 4 H), 1.56–1.45 (m, 2 H), 1.43–1.30 (m, 2 H); LC–MS (method C) <i>m</i>/<i>z</i>: 350 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.83 min; 99% purity.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 1-(4-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidin-1-yl)ethanone Hydrochloride (<b>15a</b>)</h4><div class="NLM_p last">To a suspension of 2,6-dimethyl-4-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)phenol dihydrochloride <b>14a</b> (30 mg, 0.073 mmol) in DCM (0.5 mL) was added NEt<sub>3</sub> (0.041 mL, 0.29 mmol), and then, the mixture was stirred at rt for 5 min. To the solution were added AcOH (5.3 mg, 0.09 mmol) and (benzotriazol-1-yloxy)tris(dimethylamino)phosphonium hexafluorophosphate (BOP, 48.7 mg, 0.11 mmol), and then, the mixture was stirred at 30 °C for 3 h. The mixture was concentrated in vacuo to give a residue which was purified by preparative HPLC to give the title compound as a colorless gum (26 mg, 0.06 mmol, 43% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 8.03–8.00 (m, 1 H), 7.83–7.80 (m, 1 H), 7.71–7.65 (m, 2 H), 7.52 (s, 2 H), 4.57 (dd, <i>J</i> = 7.0, 3.0 Hz, 2 H), 4.41 (d, <i>J</i> = 13.5 Hz, 1 H), 3.84 (d, <i>J</i> = 14.0 Hz, 1 H), 3.00–2.93 (m, 1 H), 2.48–2.42 (m, 1 H), 2.36 (s, 6 H), 2.20–2.14 (m, 1 H), 2.02 (s, 3 H), 1.61 (d, <i>J</i> = 13.5 Hz, 1 H), 1.50 (d, <i>J</i> = 13.0 Hz, 1 H), 1.22–1.11 (m, 1 H), 1.06–0.96 (m, 1 H); LC–MS (method A) <i>m</i>/<i>z</i>: 378 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.88 min; 98% purity.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 1-(4-(2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)piperidin-1-yl)ethanone (<b>15b</b>)</h4><div class="NLM_p last">A mixture of 2,6-dimethyl-4-(1-(piperidin-4-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)phenol dihydrochloride <b>14b</b> (79 mg, 200 μmol), NEt<sub>3</sub> (24.29 mg, 240 μmol), BOP (133 mg, 300 μmol), and AcOH (48.0 mg, 800 μmol) in DCM (2 mL) was stirred at rt overnight. The reaction mixture was purified by preparative HPLC to give the title compound as a pale white gum (14 mg, 0.37 mmol, 19% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 7.74–7.66 (m, 2 H), 7.32–7.30 (m, 2 H), 7.25 (s, 2 H), 4.12 (d, <i>J</i> = 14.0 Hz, 1 H), 3.24–3.15 (m, 2 H), 2.70–2.42 (m, 3 H), 2.32 (s, 6 H), 2.98 (s, 3 H), 2.01 (br s, 2 H), 1.33–1.30 (m, 1 H); LC–MS (method A) <i>m</i>/<i>z</i>: 364 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.83 min; 100% purity.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 1-(3-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)azetidin-1-yl)ethanone (<b>15c</b>)</h4><div class="NLM_p last">A mixture of 4-(1-(azetidin-3-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-2,6-dimethylphenol <b>14c</b> (61.5 mg, 0.20 mmol), AcOH (14.4 mg, 0.24 mmol), BOP (133 mg, 0.30 mmol), and NEt<sub>3</sub> (81 mg, 0.80 mmol) in DCM (2 mL) was stirred at 25 °C for 20 h. The reaction mixture was filtered and evaporated in vacuo to dryness and purified by preparative HPLC to give the title compound as a yellow gum (30 mg, 0.09 mmol, 43% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 7.69–7.66 (m, 2 H), 7.40–7.32 (m, 4 H), 4.70 (d, <i>J</i> = 7.5 Hz, 2 H), 4.11 (t, <i>J</i> = 8.5 Hz, 1 H), 3.88 (t, <i>J</i> = 9.0 Hz, 1 H), 3.73 (dd, <i>J</i> = 9.0, 5.5 Hz, 1 H), 3.54 (dd, <i>J</i> = 10.0, 5.5 Hz, 1 H), 3.08–3.01 (m, 1 H), 2.34 (s, 6 H), 1.73 (s, 3 H); LC–MS (method A) <i>m</i>/<i>z</i>: 350 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.78 min; 98% purity.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 1-(3-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)pyrrolidin-1-yl)ethanone (<b>15d</b>)</h4><div class="NLM_p last">A mixture of 2,6-dimethyl-4-(1-(pyrrolidin-3-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)phenol dihydrochloride <b>14d</b> (80 mg, 0.20 mmol), AcOH (14.6 mg, 0.24 mmol), BOP (135 mg, 0.30 mmol), and NEt<sub>3</sub> (82 mg, 0.81 mmol) in DCM (2 mL) was stirred at rt under nitrogen overnight. The reaction mixture was evaporated in vacuo, and the crude was purified by preparative HPLC to give the title compound as a pale white gum (46 mg, 0.12 mmol, 61% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 7.70–7.61 (m, 2 H), 7.38–7.29 (m, 4 H), 4.47 (dd, <i>J</i> = 12.0, 8.0 Hz, 2 H), 3.30–3.24 (m, 3 H), 3.02–2.98 (m, 1 H), 2.78–2.67 (m, 1 H), 2.33 (s, 6 H), 1.91–1.83 (m, 4 H), 1.63–1.61 (m, 1 H); LC–MS (method A) <i>m</i>/<i>z</i>: 364 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.81 min; 100% purity.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-(<i>trans</i>-4-((2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)cyclohexyl)acetamide (<b>15e</b>)</h4><div class="NLM_p last">A mixture of 4-(1-((<i>trans</i>-4-aminocyclohexyl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-2,6-dimethylphenol dihydrochloride <b>14e</b> (80 mg, 0.19 mmol), AcOH (13.7 mg, 0.23 mmol), BOP (126 mg, 0.28 mmol), and NEt<sub>3</sub> (77 mg, 0.76 mmol) in DCM (2 mL) was stirred at rt under nitrogen overnight. The reaction mixture was evaporated in vacuo to give the crude, which was purified by preparative HPLC to give the title compound as a pale white gum (28 mg, 0.070 mmol, 37% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 7.67–7.65 (m, 1 H), 7.60–7.58 (m, 1 H), 7.36–7.28 (m, 4 H), 4.28 (d, <i>J</i> = 7.5 Hz, 2 H), 3.51–3.43 (m, 1 H), 2.89 (d, <i>J</i> = 2.0 Hz, 1 H), 2.33 (s, 6 H), 1.86 (s, 3 H), 1.81–1.73 (m, 3 H), 1.47–1.43 (m, 2 H), 1.08–0.90 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 392 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.55 min; 100% purity.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-(<i>cis</i>-4-((2-(4-Hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)cyclohexyl)acetamide (<b>15f</b>)</h4><div class="NLM_p last">A mixture of 4-(1-((<i>cis</i>-4-aminocyclohexyl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-2,6-dimethylphenol dihydrochloride <b>14f</b> (69.9 mg, 0.20 mmol), AcOH (14.4 mg, 0.24 mmol), BOP (133 mg, 0.30 mmol), and NEt<sub>3</sub> (81 mg, 0.80 mmol) in DCM (1 mL) was stirred at rt over the weekend. The reaction mixture was evaporated in vacuo to give the crude product which was purified by preparative HPLC to give the title compound (19 mg, 0.47 mmol, 24% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 7.68–7.65 (m, 1 H), 7.60 (d, <i>J</i> = 7.0 Hz, 1 H), 7.36–7.29 (m, 4 H), 4.34 (d, <i>J</i> = 7.5 Hz, 2 H), 3.80 (br s, 1 H), 2.33 (s, 6 H), 1.97–1.91 (m, 4 H), 1.50–1.30 (m, 6 H), 1.22–1.18 (m, 2 H); LCMS (method F) <i>m</i>/<i>z</i>: 392 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.55 min; 100% purity.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 2-Hydroxy-1-(4-((2-(4-hydroxy-3,5-dimethylphenyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidin-1-yl)ethanone (<b>15g</b>)</h4><div class="NLM_p last">A mixture of 2,6-dimethyl-4-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)phenol hydrochloride <b>14a</b> (60 mg, 0.16 mmol), 2-hydroxyacetic acid (18.4 mg, 0.24 mmol), BOP (107 mg, 0.24 mmol), and NEt<sub>3</sub> (0.09 mL, 0.65 mmol) in DCM (1 mL) under nitrogen was stirred at rt overnight. The crude was purified by preparative thin-layer chromatography (TLC) (DCM/MeOH 20:1) to give the title compound as a pale white gum (24 mg, 0.06 mmol, 36% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 7.68–7.60 (m, 2 H), 7.36–7.27 (m, 4 H), 4.37–4.31 (m, 3 H), 4.13 (d, <i>J</i> = 3.5 Hz, 2 H), 3.63–3.57 (m, 1 H), 2.88–2.78 (m, 1 H), 2.55–2.46 (m, 1 H), 2.32 (s, 6 H), 2.13–2.01 (m, 1H), 1.45–1.39 (m, 2 H), 1.13–0.92 (m, 2 H); LC–MS (method C) <i>m</i>/<i>z</i>: 394 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.94 min; 97% purity.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 1,3-Dimethyl-5-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one Dihydrochloride (<b>18a</b>)</h4><div class="NLM_p last">Step a (i): a solution of <i>tert</i>-butyl 4-(((2-aminophenyl)amino)methyl)piperidine-1-carboxylate <b>12a</b> (2.1 g, 6.9 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (1.14 g, 7.6 mmol), and Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (1.70 g, 8.9 mmol) in DMF (50 mL) was stirred at 100 °C under nitrogen for 6 h. EtOAc (30 mL) was added to the mixture which was washed with brine (×3). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo to give <i>tert</i>-butyl 4-((2-(1,5-dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate as a brown oil (2.86 g, 6.4 mmol, 92% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 437 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.15 min; 97% purity. Step a (ii): a mixture of <i>tert</i>-butyl 4-((2-(1,5-dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate (2.8 g, 6.4 mmol) and 4 M HCl in MeOH (15 mL, 60.0 mmol) in MeOH (15 mL) was stirred at 25 °C for 2 h. The solvent was removed in vacuo to give the title compound as a pale solid (2.44 g, 5.9 mmol, 93% yield). LC–MS (method E) <i>m</i>/<i>z</i>: 337 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.84 min; 100% purity.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 3-Methyl-5-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one Dihydrochloride (<b>18b</b>)</h4><div class="NLM_p last">Step a (i): to a solution of <i>tert</i>-butyl 4-(((2-aminophenyl)amino)methyl)piperidine-1-carboxylate <b>12a</b> (1.03 g, 3.4 mmol) and 5-methyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (640 mg, 3.36 mmol) in DMF (10 mL) was added Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (830 mg, 4.37 mmol). The reaction mixture was stirred at 100 °C overnight. The mixture was filtered, and the solid was washed with EtOAc (30 mL). The filtrate was washed with water (3 × 10 mL) and brine (3 × 10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give the residue which was purified on silica gel eluting with 1–4% MeOH in DCM to give <i>tert</i>-butyl 4-((2-(5-methyl-6-oxo-1,6-dihydropyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate as a brown solid (1.17 g, 2.1 mmol, 63% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 423 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.13 min; 76% purity. Step a (ii): a solution of <i>tert</i>-butyl 4-((2-(5-methyl-6-oxo-1,6-dihydropyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate (1.17 g, 2.1 mmol) in 4 M HCl in MeOH (30 mL, 120 mmol) was stirred at rt for 2 h. The solvent was removed in vacuo to give the title compound as a gray solid (1.04 g, 2.1 mmol, 100% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 323 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.52 min; 80% purity.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 1-Methyl-5-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>18c</b>)</h4><div class="NLM_p last">Step b: a solution of <i>tert</i>-butyl 4-(((2-aminophenyl)amino)methyl)piperidine-1-carboxylate <b>12a</b> (1.8 g, 5.9 mmol) and 1-methyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (0.99 g, 6.48 mmol) in polyphosphoric acid (PPA) (15 g) was stirred at 180 °C under a nitrogen atmosphere for 2 h. The reaction was quenched by saturated aqueous NaHCO<sub>3</sub> and purified by preparative HPLC to give the title compound as a pale solid (1.2 g, 1.9 mmol). LC–MS (method E) <i>m</i>/<i>z</i>: 323 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.74 min; 59% purity.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 5-(1-(Piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one Dihydrochloride (<b>18d</b>)</h4><div class="NLM_p last">Step a (i): a mixture of <i>tert</i>-butyl 4-(((2-aminophenyl)amino)methyl)piperidine-1-carboxylate <b>12a</b> (1.24 g, 4.06 mmol), 6-oxo-1,6-dihydropyridine-3-carbaldehyde (500 mg, 4.06 mmol), and Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> (1.16 g, 6.1 mmol) in DMF (20 mL) was stirred at 100 °C overnight. The reaction mixture was treated with water and extracted with DCM. The organic layer was washed with saturated aqueous NaHCO<sub>3</sub> solution (×2) and water and then dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo to give <i>tert</i>-butyl 4-((2-(6-oxo-1,6-dihydropyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate as a pale yellow solid (1.44 g, 3.5 mmol, 86% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 409 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.06 min; 99% purity. Step a (ii): a mixture of <i>tert</i>-butyl 4-((2-(6-oxo-1,6-dihydropyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-1-yl)methyl)piperidine-1-carboxylate (1.44 g, 3.53 mmol) and 6 M HCl in MeOH (30 mL) was stirred at rt overnight. The reaction mixture was evaporated in vacuo to give the title compound as a white solid (1.3 g, 3.4 mmol, 96% yield). LC–MS (method C) <i>m</i>/<i>z</i>: 309 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.28 min; 99% purity.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 5-(1-((1-(2-Hydroxyacetyl)piperidin-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>19a</b>)</h4><div class="NLM_p last">A mixture of 1,3-dimethyl-5-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one dihydrochloride <b>18a</b> (2.44 g, 6.0 mmol), 2-hydroxyacetic acid (0.97 g, 12.8 mmol), DIPEA (3.31 g, 25.6 mmol), 1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-1-ol (HOBt, 1.08 g, 7.0 mmol), and <i>N</i><sup>1</sup>-((ethylimino)methylene)-<i>N</i><sup>3</sup>,<i>N</i><sup>3</sup>-dimethylpropane-1,3-diamine (EDC)·HCl (1.35 g, 7.0 mmol) in DCM (60 mL) was stirred at 25 °C overnight. The reaction mixture was purified by column chromatography on silica gel eluting with DCM/MeOH (200:1 to 50:1). This was further purified by preparative HPLC to give the title compound as a pale solid (240 mg, 0.58 mmol, 10% yield). <sup>1</sup>H NMR (300 MHz, MeOH-<i>d</i><sub>4</sub>): δ 8.03 (d, <i>J</i> = 2.0 Hz, 1 H), 7.76–7.71 (m, 1 H), 7.67 (d, <i>J</i> = 7.5 Hz, 1 H), 7.63 (d, <i>J</i> = 7.5 Hz, 1 H), 7.39–7.28 (m, 2 H), 4.40 (d, <i>J</i> = 12.5 Hz, 1 H), 4.30 (d, <i>J</i> = 7.5 Hz, 2 H), 4.23–4.07 (m, 2 H), 3.68 (s, 3 H), 3.63 (d, <i>J</i> = 12.5 Hz, 1 H), 2.85 (t, <i>J</i> = 12.5 Hz, 1 H), 2.51 (t, <i>J</i> = 12.5 Hz, 1 H), 2.22 (s, 3 H), 2.18–2.04 (m, 1 H), 1.45 (d, <i>J</i> = 12.5 Hz, 2 H), 1.25–0.99 (m, 2 H); LC–MS (method C) <i>m</i>/<i>z</i>: 395 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.87 min; 95% purity.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 5-(1-((1-(2-Hydroxyacetyl)piperidin-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-3-methylpyridin-2(1<i>H</i>)-one (<b>19b</b>)</h4><div class="NLM_p last">A mixture of 3-methyl-5-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one <b>18b</b> (600 mg, 1.49 mmol) in DCM (10 mL) was treated with DIPEA (962 mg, 7.44 mmol), 2-hydroxyacetic acid (226 mg, 2.98 mmol), HOBt (274 mg, 1.79 mmol), and EDC·HCl (571 mg, 2.98 mmol) and then stirred at rt under a nitrogen atmosphere overnight. The mixture was concentrated in vacuo to give the residue which was purified by preparative TLC (MeOH/DCM = 1:15) to give the impure material. This was repurified by preparative HPLC and dried under high vacuum to give the title compound as a white solid (126 mg, 0.31 mmol, 21% yield). <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>): δ 8.14 (d, <i>J</i> = 2.0 Hz, 1 H), 8.09–8.02 (m, 1 H), 7.88–7.81 (m, 2 H), 7.74–7.66 (m, 2 H), 4.52 (d, <i>J</i> = 5.5 Hz, 2 H), 4.45 (d, <i>J</i> = 12.5 Hz, 1 H), 4.26–4.13 (m, 2 H), 3.69 (d, <i>J</i> = 12.5 Hz, 1 H), 2.93 (t, <i>J</i> = 12.5 Hz, 1 H), 2.57 (t, <i>J</i> = 12.5 Hz, 1 H), 2.28–2.23 (m, 1 H), 2.22 (s, 3 H), 1.61 (t, <i>J</i> = 15.0 Hz, 2 H), 1.34–1.10 (m, 2 H); LC–MS (method C) <i>m</i>/<i>z</i>: 381 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.84 min; 93% purity.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 5-(1-((1-(2-Hydroxyacetyl)piperidin-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1-methylpyridin-2(1<i>H</i>)-one (<b>19c</b>)</h4><div class="NLM_p last">A mixture of 1-methyl-5-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one <b>18c</b> (500 mg, 1.55 mmol), 2-hydroxyacetic acid (236 mg, 3.10 mmol), DIPEA (10.5 mL, 7.8 mmol), HOBt (285 mg, 1.86 mmol), and EDC·HCl (595 mg, 3.10 mmol) in DCM (10 mL) was stirred at rt overnight. The reaction mixture was filtered, evaporated in vacuo to dryness, and purified by preparative TLC (DCM/MeOH = 20:1) to give the title compound (62 mg, 0.16 mmol, 10% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.27 (d, <i>J</i> = 2.5 Hz, 1 H), 7.84 (dd, <i>J</i> = 9.5, 2.5 Hz, 1 H), 7.68 (d, <i>J</i> = 7.5 Hz, 1 H), 7.64 (d, <i>J</i> = 7.5 Hz, 1 H), 7.30–7.20 (m, 2 H), 6.54 (d, <i>J</i> = 9.5 Hz, 1 H), 4.38 (t, <i>J</i> = 5.0 Hz, 1 H), 4.32–4.18 (m, 3 H), 4.07–3.93 (m, 2 H), 3.63–3.51 (m, 4 H), 2.78 (t, <i>J</i> = 12.5 Hz, 1 H), 2.44 (t, <i>J</i> = 12.5 Hz, 1 H), 2.06–1.93 (m, 1 H), 1.34 (t, <i>J</i> = 12.0 Hz, 2 H), 1.11 (q, <i>J</i> = 12.0 Hz, 1 H), 0.97 (q, <i>J</i> = 12.0 Hz, 1 H); LC–MS (method F) <i>m</i>/<i>z</i>: 381 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.42 min; 100% purity.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 5-(1-((1-(2-Hydroxyacetyl)piperidin-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>19d</b>)</h4><div class="NLM_p last">A mixture of 5-(1-(piperidin-4-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one dihydrochloride <b>18d</b> (500 mg, 1.31 mmol), DIPEA (847 mg, 6.56 mmol), HOBt (241 mg, 1.57 mmol), EDC·HCl (503 mg, 2.62 mmol), and 2-hydroxyacetic acid (199 mg, 2.62 mmol) in DCM (10 mL) was stirred at rt overnight. The reaction mixture was purified by preparative HPLC and then purified twice further by preparative TLC (DCM/MeOH = 20:1) to give the title compound as a yellow solid (56 mg, 0.15 mmol, 12% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.90–7.82 (m, 2 H), 7.66 (d, <i>J</i> = 7.5 Hz, 1 H), 7.63 (d, <i>J</i> = 7.5 Hz, 1 H), 7.29–7.19 (m, 2 H), 6.49 (d, <i>J</i> = 9.5 Hz, 1 H), 4.39 (t, <i>J</i> = 5.0 Hz, 1 H), 4.31–4.14 (m, 3 H), 4.08–3.92 (m, 2 H), 3.58 (d, <i>J</i> = 12.5 Hz, 1 H), 2.79 (t, <i>J</i> = 12.5 Hz, 1 H), 2.44 (t, <i>J</i> = 12.5 Hz, 1 H), 2.06–1.92 (m, 1 H), 1.36 (t, <i>J</i> = 12.5 Hz, 2 H), 1.11 (q, <i>J</i> = 12.0 Hz, 1 H), 0.97 (q, <i>J</i> = 12.0 Hz, 1 H), N<i>H</i> not resolved; LC–MS (method F) <i>m</i>/<i>z</i>: 367 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.40 min; 97% purity.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1,3-Dimethyl-5-(1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>20a</b>)</h4><div class="NLM_p last">Step a: batch 1: (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine (0.26 mL, 2.13 mmol) was added to a solution of 1-fluoro-2-nitrobenzene (0.11 mL, 1.06 mmol) and DIPEA (0.56 mL, 3.19 mmol) in NMP (1 mL) at rt in a microwave vial. The vial was sealed and heated in a microwave at 120 °C for 30 min. Once cooled, the reaction mixture was diluted with EtOAc (5 mL) and the organic layer was extracted with water (4 × 5 mL). The aqueous phase was then back extracted with EtOAc (4 × 5 mL). The combined organic layers were dried over a hydrophobic frit before being concentrated in vacuo to give an orange oil (∼70 mg). Batch 2: (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine (3.5 mL, 28.3 mmol) was added to a solution of 1-fluoro-2-nitrobenzene (1.5 mL, 14.1 mmol) and DIPEA (5.0 mL, 28.3 mmol) in NMP (10 mL) at rt in a microwave vial. The vial was sealed and heated in a microwave at 120 °C for 30 min. Once cooled the reaction mixture was diluted with EtOAc (50 mL) and the organic layer extracted with water (4 × 50 mL). The aqueous phase was then back extracted with EtOAc (4 × 50 mL). The combined organic layers were dried over a hydrophobic frit before being concentrated in vacuo to give an orange oil. Both batches were combined, loaded in DCM, and purified with a CombiFlash RF+ 40 g silica cartridge using a gradient of 0–15% EtOAc in cyclohexane over 10 column volumes and then held at 15% for further 10 column volumes. The relevant fractions were combined and concentrated in vacuo to give 2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)aniline as an orange oil (2.3 g, 9.6 mmol, 63% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.18 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 8.21–8.12 (m, 1 H), 7.45 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 6.85 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 6.66 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 4.03 (dd, <i>J</i> = 11.5, 4.0 Hz, 2 H), 3.43 (ddd, <i>J</i> = 12.0, 11.5, 2.0 Hz, 2 H), 3.22 (dd, <i>J</i> = 6.5, 5.5 Hz, 2 H), 2.02–1.90 (m, 1 H), 1.76 (ddd, <i>J</i> = 12.0, 4.0, 2.0 Hz, 2 H), 1.49–1.38 (m, 2 H); LC–MS (method A) <i>m</i>/<i>z</i>: 237 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.07 min; 96% purity. Step b (i): batch 1: to a mixture of 2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)aniline (1.5 g, 6.4 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (0.96 g, 6.35 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (2.0 g, 11.4 mmol) in EtOH (2 mL) and water (1 mL) were added in a sealed tube and stirred at 100 °C for 16 h. The reaction mixture was diluted with water (100 mL) and extracted with EtOAc (3 × 100 mL). The organic layer was washed with brine (250 mL), separated, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The crude material was purified by column chromatography using silica gel 100–200 mesh, eluting with EtOAc and then 2% MeOH in DCM. The collected fractions were concentrated under reduced pressure to afford batch 1 (560 mg). Batch 2: to a mixture of 2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)aniline (1.5 g, 6.4 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (0.96 g, 6.4 mmol), and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (2.0 g, 11.4 mmol) in EtOH (2 mL) and water (1 mL) were added in a sealed tube and stirred at 100 °C for 16 h. The reaction mixture was diluted with water (100 mL) and extracted with EtOAc (3 × 100 mL). The organic layer was washed with brine (250 mL), separated, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The crude material was purified by column chromatography using silica gel 100–200 mesh, eluting with EtOAc and then 5% MeOH in DCM. The collected fractions were concentrated under reduced pressure. The material was purified by column chromatography using silica gel 230–400 mesh, eluting with 2% MeOH in DCM. The collected fractions were concentrated under reduced pressure. The material was washed with Et<sub>2</sub>O (2 × 10 mL), decanted, and dried under reduced pressure to afford batch 2 (900 mg). Both batches were combined and purified by flash chromatography (column: Grace; silica 40 mm; 24 g cartridge). Pure fractions were collected and concentrated under reduced pressure to afford the title compound as a white solid (1.2 g, 3.6 mmol, 28% combined yield). mp 158 °C; IR (solid) ν (cm<sup>–1</sup>): 2918, 2840, 1654, 1619, 1459, 1091; <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.13 (s, 1 H), 7.74 (s, 1 H), 7.68 (d, <i>J</i> = 7.5 Hz, 1 H), 7.62 (d, <i>J</i> = 7.5 Hz, 1 H), 7.26 (t, <i>J</i> = 7.5 Hz, 1 H), 7.21 (t, <i>J</i> = 7.5 Hz, 1 H), 4.28 (d, <i>J</i> = 7.0 Hz, 2 H), 3.72 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.56 (s, 3 H), 3.11 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 2.11 (s, 3 H), 1.96 (ddd, <i>J</i> = 11.0, 7.0, 4.0 Hz, 1 H), 1.24–1.12 (m, 4 H); <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.3, 150.9, 142.8, 138.8, 137.2, 136.5, 128.3, 122.7, 122.3, 119.3, 111.6, 108.8, 66.8 (2C), 49.8, 37.9 (2C), 35.6, 30.4, 17.4; LC–MS (method F) <i>m</i>/<i>z</i>: 338 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.52 min; 100% purity. HRMS: [(M + H)<sup>+</sup>] calcd for C<sub>20</sub>H<sub>24</sub>N<sub>3</sub>O<sub>2</sub>, 338.1869; found, 338.1868.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 1,3-Dimethyl-5-(1-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)pyridin-2(1<i>H</i>)-one (<b>20b</b> and <b>20b′</b>)</h4><div class="NLM_p last">Step a: 1-fluoro-2-nitrobenzene (0.11 mL, 1.04 mmol), 1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethanamine hydrochloride (250 mg, 1.51 mmol), and DIPEA (0.8 mL, 4.58 mmol) were suspended in DMSO (1 mL) within a sealed vessel and heated in a microwave for 20 min at 120 °C. The reaction mixture was allowed to cool to rt. The reaction vessel was sealed and heated within a microwave for a further 20 min at 120 °C. The reaction mixture was allowed to cool to rt. The solvent was removed under a stream of nitrogen, and the residue was taken up into EtOAc (10 mL), diluted with water (10 mL), and extracted with EtOAc (3 × 10 mL). The combined organic extracts were dried over a hydrophobic frit, and the filtrate was purified by normal phase column chromatography on an 80 g silica column and eluted with 0–70% cyclohexane in EtOAc over 10 column volumes. The desired fractions were combined, and the solvent was removed in vacuo to give a yellow oil which was dried in a vacuum oven overnight to yield crude racemic 2-nitro-<i>N</i>-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)aniline (378 mg, 1.51 mmol). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.18 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 8.12 (d, <i>J</i> = 8.0 Hz, 1 H), 7.42 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 6.86 (d, <i>J</i> = 8.5 Hz, 1 H), 6.62 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 4.07–4.00 (m, 2 H), 3.63–3.52 (m, 1 H), 3.44–3.35 (m, 2 H), 1.83–1.73 (m, 2 H), 1.70–1.63 (m, 1 H), 1.55–1.39 (m, 2 H), 1.27 (d, <i>J</i> = 6.5 Hz, 3 H); LC–MS (method A) <i>m</i>/<i>z</i>: 251 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.17 min; 100% purity. Step c (i): racemic 2-nitro-<i>N</i>-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)aniline (142 mg, 0.567 mmol) was dissolved in EtOH (10 mL) and added to 5% Pd/C (15 mg, 0.007 mmol). The reaction mixture was stirred at rt under an atmosphere of hydrogen for 6 h. The reaction mixture was filtered over Celite and washed with EtOH (30 mL). The solvent was removed in vacuo to give racemic <i>N</i><sup>1</sup>-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)benzene-1,2-diamine as a purple gum (125 mg, 0.57 mmol, 100% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 6.53 (dd, <i>J</i> = 7.5, 1.5 Hz, 1 H), 6.47 (td, <i>J</i> = 7.5, 1.5 Hz, 1 H), 6.42 (dd, <i>J</i> = 7.5, 1.5 Hz, 1 H), 6.36 (td, <i>J</i> = 7.5, 1.5 Hz, 1 H), 4.46 (s, 2 H), 3.99 (d, <i>J</i> = 8.5 Hz, 1 H), 3.91–3.84 (m, 2 H), 3.29–3.19 (m, 3 H), 1.74 (d, <i>J</i> = 13.0 Hz, 1 H), 1.71–1.61 (m, 1 H), 1.58 (d, <i>J</i> = 13.0 Hz, 1 H), 1.31 (td, <i>J</i> = 13.0, 4.5 Hz, 1 H), 1.25 (td, <i>J</i> = 13.0, 4.5 Hz, 1 H), 1.06 (d, <i>J</i> = 6.5 Hz, 3 H); LC–MS (method A) <i>m</i>/<i>z</i>: 221 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.93 min; 91% purity. Step c (ii): batch 1: racemic <i>N</i><sup>1</sup>-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)benzene-1,2-diamine (60 mg, 0.27 mmol) and 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (45 mg, 0.30 mmol) were dissolved in DMF (1.5 mL) and water (0.05 mL). Potassium peroxymonosulfate (109 mg, 0.18 mmol) was added, and the reaction mixture was evacuated and purged with nitrogen (×3) and left to stir for 18 h at rt under an atmosphere of nitrogen. The reaction mixture was diluted with DCM (30 mL) and washed with water (30 mL). The aqueous phase was extracted with DCM (3 × 30 mL), and the organic layers were combined and dried over a hydrophobic frit. The solvent was removed in vacuo, and the residue was dissolved in 1:1 DMSO/MeOH (0.6 mL) and purified by MDAP (method B). The solvent was evaporated in vacuo to give batch 1 racemic 1,3-dimethyl-5-(1-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one as a brown solid (66 mg). Batch 2: racemic <i>N</i><sup>1</sup>-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)benzene-1,2-diamine (65 mg, 0.30 mmol) and 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (45 mg, 0.30 mmol) were dissolved in DMF (1.5 mL) and water (0.05 mL). Potassium peroxymonosulfate (118 mg, 0.19 mmol) was added, and the reaction mixture was stirred for 5 h at rt under an atmosphere of nitrogen. The reaction mixture was diluted with water (30 mL). The aqueous phase was extracted with DCM (4 × 30 mL), and the organic layers were combined and dried over a hydrophobic frit. The solvent was removed in vacuo, and the residue was dissolved in 1:1 DMSO/MeOH (0.6 mL) and purified by MDAP (method B). The solvent was evaporated in vacuo to give batch 2 racemic 1,3-dimethyl-5-(1-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)pyridin-2(1<i>H</i>)-one as a brown solid (56 mg). The two batches (batch 1, 36 mg and batch 2, 50 mg) were combined and dissolved in EtOH (2 mL). Two injections were made onto a Chiralpak AD-H column (250 mm × 30 mm). An isocratic system of 30% EtOH in heptane with a flow rate of 25 mL/min was used at rt. The UV detection was performed at 215 nm. The appropriate fractions were combined and evaporated in vacuo to give the title compounds. Isomer 1 <b>20b</b> (28 mg, 0.08 mmol, 14% combined yield). >99% ee determined by HPLC analysis on a Chiralpak AD-H (250 mm × 4.6 mm) column. Peak elution at <i>R</i><sub>t</sub>: 8.88 min. An isocratic system of 30% EtOH in heptane with a flow rate of 1 mL/min was used at rt. The UV detection was performed at 215 nm; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.78 (dd, <i>J</i> = 7.0, 2.0 Hz, 1 H), 7.66 (d, <i>J</i> = 2.0 Hz, 1 H), 7.57 (dd, <i>J</i> = 7.0, 2.0 Hz, 1 H), 7.36–7.33 (m, 1 H), 7.31 (td, <i>J</i> = 7.0, 2.0 Hz, 1 H), 7.27 (td, <i>J</i> = 7.0, 2.0 Hz, 1 H), 4.15 (dq, <i>J</i> = 10.5, 7.0 Hz, 1 H), 4.03 (dd, <i>J</i> = 11.5, 4.0 Hz, 1 H), 3.75 (dd, <i>J</i> = 11.5, 4.0 Hz, 1 H), 3.66 (s, 3 H), 3.39 (td, <i>J</i> = 11.5, 2.0 Hz, 1 H), 3.13 (td, <i>J</i> = 11.5, 2.0 Hz, 1 H), 2.41 (tdt, <i>J</i> = 11.5, 10.5, 4.0 Hz, 1 H), 2.25 (s, 3 H), 1.85 (ddd, <i>J</i> = 13.0, 4.0, 2.0 Hz, 1 H), 1.75 (d, <i>J</i> = 7.0 Hz, 3 H), 1.35 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 1 H), 0.98 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 1 H), 0.70 (ddd, <i>J</i> = 13.0, 4.0, 2.0 Hz, 1 H); LC–MS (method A) <i>m</i>/<i>z</i>: 352 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.87 min; 100% purity. Isomer 2 <b>20b′</b> (28 mg, 0.08 mmol, 14% combined yield). 99% ee determined by HPLC analysis on a Chiralpak AD-H (250 mm × 4.6 mm) column. Peak elution at <i>R</i><sub>t</sub>: 10.83 min. An isocratic system of 30% EtOH in heptane with a flow rate of 1 mL/min was used at rt. The UV detection was performed at 215 nm; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.78 (dd, <i>J</i> = 7.0, 2.0 Hz, 1 H), 7.66 (d, <i>J</i> = 2.0 Hz, 1 H), 7.57 (dd, <i>J</i> = 7.0, 2.0 Hz, 1 H), 7.36–7.33 (m, 1 H), 7.31 (td, <i>J</i> = 7.0, 2.0 Hz, 1 H), 7.28 (td, <i>J</i> = 7.0, 2.0 Hz, 2 H), 4.15 (dq, <i>J</i> = 10.5, 7.0 Hz, 1 H), 4.03 (dd, <i>J</i> = 11.5, 4.0 Hz, 1 H), 3.75 (dd, <i>J</i> = 11.5, 4.0 Hz, 1 H), 3.66 (s, 3 H), 3.39 (td, <i>J</i> = 11.5, 2.0 Hz, 1 H), 3.13 (td, <i>J</i> = 11.5, 2.0 Hz, 1 H), 2.41 (tdt, <i>J</i> = 11.5, 10.5, 4.0 Hz, 1 H), 2.25 (s, 3 H), 1.85 (ddd, <i>J</i> = 13.0, 4.0, 2.0 Hz, 1 H), 1.75 (d, <i>J</i> = 7.0 Hz, 3 H), 1.35 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 2 H), 0.98 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 1 H), 0.70 (ddd, <i>J</i> = 13.0, 4.0, 2.0 Hz, 1 H); LC–MS (method A) <i>m</i>/<i>z</i>: 352 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.87 min; 100% purity.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 1,3-Dimethyl-5-(1-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)propyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)pyridin-2(1<i>H</i>)-one (<b>20c</b> and <b>20c′</b>)</h4><div class="NLM_p last">Step a: 1-fluoro-2-nitrobenzene (0.11 mL, 1.04 mmol), DIPEA (0.22 mL, 1.26 mmol), and 1-(tetrahydro-2<i>H</i>-pyran-4-yl)propan-1-amine (179 mg, 1.25 mmol) were suspended in NMP (1 mL) within a sealed vial and heated in a microwave at 125 °C for 1 h. The mixture was allowed to reach rt and then diluted with water (40 mL) and extracted with EtOAc (5 × 40 mL). The organic layers were combined and dried over a hydrophobic frit, and the solvent was removed in vacuo. The residue was taken up into DCM (5 mL) and purified by normal phase column chromatography on a 40 g silica column and eluting with 0–100% EtOAc in cyclohexane over 10 column volumes. The desired fractions were combined, and the solvent was removed in vacuo to give 2-nitro-<i>N</i>-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)propyl)aniline as an orange gum (265 mg, 1.00 mmol, 96% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.18 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 8.12 (d, <i>J</i> = 8.5 Hz, 1 H), 7.40 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 6.88 (d, <i>J</i> = 8.5 Hz, 1 H), 6.61 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 4.01 (dd, <i>J</i> = 11.5, 4.5 Hz, 2 H), 3.50–3.43 (m, 1 H), 3.42–3.34 (m, 2 H), 1.88–1.74 (m, 2 H), 1.74–1.67 (m, 1 H), 1.65–1.58 (m, 1 H), 1.56–1.40 (m, 3 H), 0.96 (t, <i>J</i> = 7.5 Hz, 3 H); LC–MS (method A) <i>m</i>/<i>z</i>: 263 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.24 min; 99% purity. Step b (i): 2-Nitro-<i>N</i>-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)propyl)aniline (265 mg, 1.00 mmol), Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (524 mg, 3.01 mmol), and 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (212 mg, 1.40 mmol) were suspended in EtOH (1.5 mL) and water (0.75 mL) within a sealed vessel and heated in a microwave at 100 °C for 65 min. The reaction mixture was allowed to cool to rt. The reaction mixture was diluted with saturated aqueous NaHCO<sub>3</sub> (30 mL). The aqueous phase was extracted with DCM (4 × 30 mL), and the organic extracts were combined and dried over a hydrophobic frit. The solvent was removed in vacuo, and the residue was taken up into DCM (5 mL) and purified by normal phase column chromatography on a 40 g silica cartridge eluting with 0–100% EtOAc in cyclohexane over 10 column volumes followed by 100% EtOAc over 10 column volumes and MeOH over 10 column volumes. The desired fractions were combined, and the solvent was removed in vacuo to give racemic 1,3-dimethyl-5-(1-(1-(tetrahydro-2<i>H</i>-pyran-4-yl)propyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (330 mg). Racemate (324 mg) was dissolved in EtOH (3 mL). Three injections were made onto a Chiralcel OD-H column (250 mm × 30 mm). An isocratic system of 10% EtOH in heptane with a flow rate of 30 mL/min was used at rt. The UV detection was performed at 215 nm. The appropriate fractions were combined and evaporated in vacuo. Mixed fractions were reprocessed using the same conditions to give the title compounds. Isomer 1 <b>20c</b> (170 mg, 0.47 mmol, 47% yield). >99% ee determined by HPLC analysis on a Chiralcel OD-H (250 mm × 4.6 mm) column. Peak elution at <i>R</i><sub>t</sub>: 26.02 min. An isocratic system of 10% EtOH in heptane with a flow rate of 1 mL/min was used at rt. The UV detection was performed at 215 nm; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.79 (dd, <i>J</i> = 7.5, 1.5 Hz, 1 H), 7.61 (d, <i>J</i> = 1.5 Hz, 1 H), 7.56 (d, <i>J</i> = 7.5 Hz, 1 H), 7.43 (s, 1 H), 7.31 (td, <i>J</i> = 7.5, 1.5 Hz, 1 H), 7.29–7.25 (m, 1 H), 4.05 (dd, <i>J</i> = 11.5, 4.0 Hz, 1 H), 3.97 (td, <i>J</i> = 11.0, 4.0 Hz, 1 H), 3.82 (dd, <i>J</i> = 11.5, 4.0 Hz, 1 H), 3.65 (s, 3 H), 3.41 (td, <i>J</i> = 11.5, 2.0 Hz, 1 H), 3.19 (td, <i>J</i> = 11.5, 2.0 Hz, 1 H), 2.48–2.34 (m, 1 H), 2.25 (s, 3 H), 2.29–2.16 (m, 1 H), 2.13–2.02 (m, 1 H), 1.87 (ddd, <i>J</i> = 13.0, 4.0, 2.0 Hz, 1 H), 1.39 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 1 H), 1.14 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 1 H), 0.93–0.86 (m, 1 H), 0.77 (t, <i>J</i> = 7.5 Hz, 3 H); LC–MS (method F) <i>m</i>/<i>z</i>: 366 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.66 min; 100% purity. Isomer 2 <b>20c′</b> (151 mg, 0.41 mmol, 41% yield). 99% ee determined by HPLC analysis on a Chiralcel OD-H (250 mm × 4.6 mm) column. Peak elution at <i>R</i><sub>t</sub>: 30.83 min. An isocratic system of 10% EtOH in heptane with a flow rate of 1 mL/min was used at rt. The UV detection was performed at 215 nm; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.79 (dd, <i>J</i> = 7.5, 1.5 Hz, 1 H), 7.61 (d, <i>J</i> = 1.5 Hz, 1 H), 7.56 (d, <i>J</i> = 7.5 Hz, 1 H), 7.43 (s, 1 H), 7.29–7.25 (m, 1 H), 7.31 (td, <i>J</i> = 7.5, 1.5 Hz, 1 H), 4.05 (dd, <i>J</i> = 11.5, 4.0 Hz, 1 H), 3.97 (td, <i>J</i> = 11.0, 4.0 Hz, 1 H), 3.82 (dd, <i>J</i> = 11.5, 4.0 Hz, 1 H), 3.65 (s, 3 H), 3.41 (td, <i>J</i> = 11.5, 2.0 Hz, 1 H), 3.19 (td, <i>J</i> = 11.5, 2.0 Hz, 1 H), 2.48–2.34 (m, 1 H), 2.25 (s, 3 H), 2.29–2.16 (m, 1 H), 2.13–2.02 (m, 1 H), 1.87 (ddd, <i>J</i> = 13.0, 4.0, 2.0 Hz, 1 H), 1.39 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 1 H), 1.14 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 1 H), 0.93–0.86 (m, 1 H), 0.77 (t, <i>J</i> = 7.5 Hz, 3 H); LC–MS (method F) <i>m</i>/<i>z</i>: 366 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.66 min; 100% purity.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 1,3-Dimethyl-5-(1-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>20d</b> and <b>20d′</b>)</h4><div class="NLM_p last">Step a: 1-fluoro-2-nitrobenzene (0.11 mL, 1.042 mmol), DIPEA (0.40 mL, 2.292 mmol), and (tetrahydro-2<i>H</i>-pyran-3-yl)methanamine (144 mg, 1.25 mmol) were suspended in NMP (1 mL) within a sealed vial and heated in a microwave at 125 °C for 1 h. The reaction mixture was allowed to reach rt and then diluted with water (40 mL) and extracted with EtOAc (5 × 40 mL). The organic layers were combined and dried over a hydrophobic frit, and the solvent was removed in vacuo. The residue was taken up into DCM (5 mL) and purified by normal phase column chromatography on a 40 g silica column and eluted with 0–100% EtOAc in cyclohexane over 10 column volumes. The desired fractions were combined, the solvent was removed in vacuo, and the residue was dried in a vacuum oven overnight to give 2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)aniline as an orange gum (201 mg, 0.85 mmol, 82% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.19 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 8.15–8.05 (m, 1 H), 7.45 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 6.85 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 6.66 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 3.96 (ddd, <i>J</i> = 11.0, 4.0, 1.5 Hz, 1 H), 3.86 (dt, <i>J</i> = 11.0, 4.0 Hz, 1 H), 3.51 (ddd, <i>J</i> = 11.0, 9.0, 4.0 Hz, 1 H), 3.34 (dd, <i>J</i> = 11.0, 9.0 Hz, 1 H), 3.31–3.17 (m, 2 H), 2.10–1.95 (m, 2 H), 1.76–1.59 (m, 2 H), 1.48–1.37 (m, 1 H); LC–MS (method A): no mass ion observed; <i>R</i><sub>t</sub>: 1.12 min; 98% purity. Step b (i): 2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)aniline (201 mg, 0.85 mmol), Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (444 mg, 2.55 mmol), and 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (180 mg, 1.19 mmol) were suspended in EtOH (1.5 mL) and water (0.75 mL) within a sealed vessel and heated in a microwave at 100 °C for 65 min. The reaction mixture was allowed to cool to rt. The reaction mixture was diluted with water (30 mL). The aqueous phase was extracted with DCM (4 × 30 mL), and the organic layers were combined and dried over a hydrophobic frit. The solvent was removed in vacuo, and the residue was dissolved in 1:1 DMSO/MeOH (0.6 mL) and purified by MDAP (method B). The solvent was evaporated in vacuo to give racemic 1,3-dimethyl-5-(1-((tetrahydro-2<i>H</i>-pyran-3-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (148 mg). Racemate (144 mg) was dissolved in EtOH (2 mL). Two injections were made onto a Chiralpak AD-H column (250 mm × 30 mm). An isocratic system of 40% EtOH (containing 0.2% isopropylamine) in heptane (containing 0.2% isopropylamine) with a flow rate of 30 mL/min was used at rt. The UV detection was performed at 215 nm. The appropriate fractions were combined and evaporated in vacuo to give the title compounds. Isomer 1 <b>20d</b> (68 mg, 0.20 mmol, 24% yield). >99% ee determined by HPLC analysis on a Chiralpak AD-H (250 mm × 4.6 mm) column. Peak elution at <i>R</i><sub>t</sub>: 11.23 min. An isocratic system of 50% EtOH (containing 0.2% isopropylamine) in heptane with a flow rate of 1 mL/min was used at rt. The UV detection was performed at 215 nm; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.81 (d, <i>J</i> = 2.0 Hz, 1 H), 7.79–7.76 (m, 1 H), 7.58–7.56 (m, 1 H), 7.42–7.37 (m, 1 H), 7.34–7.29 (m, 2 H), 4.28 (dd, <i>J</i> = 14.5, 8.5 Hz, 1 H), 4.13 (dd, <i>J</i> = 14.5, 7.0 Hz, 1 H), 3.76–3.69 (m, 1 H), 3.66 (s, 3 H), 3.62–3.48 (m, 2 H), 3.25 (dd, <i>J</i> = 11.5, 7.5 Hz, 1 H), 2.26 (s, 3 H), 2.25–2.18 (m, 1 H), 1.76–1.67 (m, 1 H), 1.67–1.59 (m, 1 H), 1.55–1.45 (m, 1 H), 1.37–1.27 (m, 1 H); LC–MS (method A) <i>m</i>/<i>z</i>: 338 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.85 min; 100% purity. Isomer 2 <b>20d′</b> (74 mg, 0.22 mmol, 26% yield). >99% ee determined by HPLC analysis on a Chiralpak AD-H (250 mm × 4.6 mm) column. Peak elution at <i>R</i><sub>t</sub>: 17.57 min. An isocratic system of 50% EtOH (containing 0.2% isopropylamine) in heptane with a flow rate of 1 mL/min was used at rt. The UV detection was performed at 215 nm; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.81 (d, <i>J</i> = 2.0 Hz, 1 H), 7.79–7.76 (m, 1 H), 7.58–7.56 (m, 1 H), 7.42–7.37 (m, 1 H), 7.34–7.28 (m, 2 H), 4.28 (dd, <i>J</i> = 14.5, 8.5 Hz, 1 H), 4.13 (dd, <i>J</i> = 14.5, 7.0 Hz, 1 H), 3.76–3.69 (m, 1 H), 3.66 (s, 3 H), 3.62–3.48 (m, 2 H), 3.25 (dd, <i>J</i> = 11.5, 7.5 Hz, 1 H), 2.26 (s, 3 H), 2.25–2.18 (m, 1 H), 1.76–1.67 (m, 1 H), 1.67–1.59 (m, 1 H), 1.55–1.45 (m, 1 H), 1.37–1.27 (m, 1 H); LC–MS (method A) <i>m</i>/<i>z</i>: 338 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.85 min; 100% purity.</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 5-(1-((<i>trans</i>-4-Hydroxycyclohexyl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>20e</b>)</h4><div class="NLM_p last">Step a: to a solution of 1-fluoro-2-nitrobenzene (100 mg, 0.71 mmol) in 2-methyl tetrahydrofuran (2-MeTHF, 1 mL) were added (<i>trans</i>-4-(<i>tert</i>-butoxy)cyclohexyl)methanamine (158 mg, 0.85 mmol) and DIPEA (0.37 mL, 2.13 mmol). The reaction vessel was sealed and heated in a microwave at 110 °C for 20 min. The reaction was cooled and then heated to 110 °C for 20 min. The solvent was removed under a stream of nitrogen. Step b (i): to the dried material were added 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (118 mg, 0.78 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (370 mg, 2.13 mmol) as a solution in EtOH (1.0 mL), followed by water (0.5 mL). The sealed vessel was heated at 100 °C for 6 h. The solvent was removed under a stream of nitrogen. The reaction mixture was partitioned between saturated aqueous NaHCO<sub>3</sub> (3 mL) and DCM (2 × 3 mL). The DCM was removed under a stream of nitrogen, and the residue was purified by MDAP (method B). Step b (ii): to the dried material were added DCM (0.2 mL) and TFA (0.2 mL), and the solution was left for 2 h. The solvent was removed under a stream of nitrogen, and the residue was purified by MDAP (method B) to give the title compound (6 mg, 0.02 mmol, 2% overall yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.11 (d, <i>J</i> = 2.5 Hz, 1 H), 7.74–7.71 (m, 1 H), 7.62 (dd, <i>J</i> = 7.5, 1.5 Hz, 1 H), 7.61 (dd, <i>J</i> = 7.0, 1.5 Hz, 1 H), 7.25 (ddd, <i>J</i> = 7.5, 7.0, 1.5 Hz, 1 H), 7.21 (ddd, <i>J</i> = 7.5, 7.0, 1.5 Hz, 1 H), 4.39 (d, <i>J</i> = 4.5 Hz, 1 H), 4.19 (d, <i>J</i> = 7.5 Hz, 2 H), 3.56 (s, 3 H), 3.29–3.21 (m, 1 H), 2.10 (s, 3 H), 1.75–1.58 (m, 3 H), 1.38–1.28 (m, 2 H), 1.00–0.85 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 352 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.47 min; 96% purity.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 1,3-Dimethyl-5-(1-(2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)pyridin-2(1<i>H</i>)-one (<b>20f</b>)</h4><div class="NLM_p last">Step a: to a solution of 1-fluoro-2-nitrobenzene (50 mg, 0.35 mmol) in 2-MeTHF (1 mL) were added 2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethanamine (55 mg, 0.43 mmol) and DIPEA (0.19 mL, 1.06 mmol). The reaction vessel was sealed and heated in a microwave at 110 °C for 20 min. After cooling the reaction, further 2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethanamine (55 mg, 0.43 mmol) was added and the vial was capped and heated to 120 °C for 20 min. The solvent was removed under a stream of nitrogen. Step b (i): to the dried material were added 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (58.9 mg, 0.39 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (185 mg, 1.06 mmol) as a solution in EtOH (1.0 mL), followed by water (0.5 mL). The sealed vessel was heated to 100 °C for 18 h. The solvent was removed under a stream of nitrogen. The reaction mixture was partitioned between saturated aqueous NaHCO<sub>3</sub> (3 mL) and DCM (2 × 3 mL). The DCM was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was evaporated under a stream of nitrogen to give the title compound (12 mg, 0.03 mmol, 9% overall yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.11 (d, <i>J</i> = 2.5 Hz, 1 H), 7.73–7.71 (m, 1 H), 7.64–7.59 (m, 2 H), 7.27 (t, <i>J</i> = 7.5 Hz, 1 H), 7.22 (t, <i>J</i> = 7.5 Hz, 1 H), 4.32 (t, <i>J</i> = 7.5 Hz, 2 H), 3.77 (dd, <i>J</i> = 11.5, 3.0 Hz, 2 H), 3.56 (s, 3 H), 3.17 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 2.11 (s, 3 H), 1.65 (q, <i>J</i> = 7.5 Hz, 2 H), 1.47 (dd, <i>J</i> = 11.5, 2.0 Hz, 2 H), 1.44–1.36 (m, 1 H), 1.16 (qd, <i>J</i> = 11.5, 4.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 352 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.55 min; 100% purity.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 5-(1-(2-Oxaspiro[3.3]heptan-6-ylmethyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>20g</b>)</h4><div class="NLM_p last">Step a: to a solution of 1-fluoro-2-nitrobenzene (100 mg, 0.71 mmol) in 2-MeTHF (1 mL) were added 2-oxaspiro[3.3]heptan-6-ylmethanamine (108 mg, 0.85 mmol) and DIPEA (0.37 mL, 2.13 mmol). The reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. The solvent was removed under a stream of nitrogen. The residue was purified by MDAP (method B). Step b (i): to the dried material were added 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (118 mg, 0.78 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (370 mg, 2.13 mmol) as a solution in EtOH (1.0 mL), followed by water (0.5 mL). The sealed vessel was heated to 100 °C for 4 h. The solvent was removed under a stream of nitrogen. The reaction mixture was partitioned between saturated aqueous NaHCO<sub>3</sub> (1.5 mL) and DCM (2 × 3 mL). The DCM was removed under a stream of nitrogen. The residue was dissolved in DMSO (1 mL) and purified by MDAP (method B) to give the title compound (10 mg, 0.03 mmol, 4% overall yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.11 (s, 1 H), 7.72 (s, 1 H), 7.63 (d, <i>J</i> = 7.5 Hz, 1 H), 7.60 (d, <i>J</i> = 7.5 Hz, 1 H), 7.25 (t, <i>J</i> = 7.5 Hz, 1 H), 7.21 (t, <i>J</i> = 7.5 Hz, 1 H), 4.44 (s, 2 H), 4.34 (d, <i>J</i> = 7.5 Hz, 2 H), 4.32 (s, 2 H), 3.56 (s, 3 H), 2.41 (tt, <i>J</i> = 7.5 Hz, 1 H), 2.11 (s, 3 H), 2.09 (dd, <i>J</i> = 12.0, 7.5 Hz, 2 H), 1.81 (dd, <i>J</i> = 12.0, 7.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 350 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.56 min; 99% purity.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 5-(1-(1-Methoxypropan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>20h</b>)</h4><div class="NLM_p last">Step a: to a solution of 1-fluoro-2-nitrobenzene (100 mg, 0.71 mmol) in 2-MeTHF (1 mL) were added 1-methoxypropan-2-amine (1.42 mmol, 126 mg) and DIPEA (0.37 mL, 2.13 mmol). The reaction vessel was sealed and heated in a microwave at 110 °C for 20 min. The reaction was cooled and then heated to 120 °C for 20 min. The solvent was removed under a stream of nitrogen. Step b (i): to the dried material were added 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (118 mg, 0.78 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (370 mg, 2.13 mmol) as a solution in EtOH (1.0 mL), followed by water (0.5 mL). The sealed vessel was heated to 100 °C for 18 h. The solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B) to give the title compound (66 mg, 0.21 mmol, 27% overall yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.02 (d, <i>J</i> = 2.5 Hz, 1 H), 7.79–7.74 (m, 1 H), 7.65–7.60 (m, 2 H), 7.24–7.19 (m, 2 H), 4.84–4.73 (m, 1 H), 4.02 (t, <i>J</i> = 10.0 Hz, 1 H), 3.68 (dd, <i>J</i> = 10.0, 4.5 Hz, 1 H), 3.55 (s, 3 H), 3.15 (s, 3 H), 2.10 (s, 3 H), 1.54 (d, <i>J</i> = 7.0 Hz, 3 H); LC–MS (method F) <i>m</i>/<i>z</i>: 312 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.53 min; 99% purity.</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 5-(1-(1,3-Dimethoxypropan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>20i</b>)</h4><div class="NLM_p last">Step a: a mixture of 1-fluoro-2-nitrobenzene (0.15 mL, 1.42 mmol), DIPEA (0.37 mL, 2.12 mmol), and 1,3-dimethoxypropan-2-amine (340 mg, 2.85 mmol) in a microwave vial was diluted with anhydrous NMP (2 mL). The vial was sealed and heated in a microwave at 125 °C for 30 min. The reaction mixture was diluted with water (5 mL) and extracted with EtOAc (3 × 5 mL). The organic extracts were combined, washed with brine (15 mL), and dried through a hydrophobic frit. The solvent was removed in vacuo, and the residue was purified by MDAP (method B). The appropriate fractions were combined, and the solvent was removed by rotary evaporation. The remaining mixture was diluted with water (3 mL) and extracted with EtOAc (3 × 3 mL). The organic extracts were combined and dried through a hydrophobic frit to give <i>N</i>-(1,3-dimethoxypropan-2-yl)-2-nitroaniline as an orange oil (275 mg, 1.15 mmol, 80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.34 (d, <i>J</i> = 7.0 Hz, 1 H), 8.18 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 7.43 (ddd, <i>J</i> = 8.5, 7.0, 1.5 Hz, 1 H), 6.96 (dd, <i>J</i> = 8.5, 1.0 Hz, 1 H), 6.65 (ddd, <i>J</i> = 8.5, 7.0, 1.0 Hz, 1 H), 3.94–3.84 (m, 1 H), 3.65–3.53 (m, 4 H), 3.42 (s, 6 H); LC–MS (method F) <i>m</i>/<i>z</i>: 241 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.07 min; 100% purity. Step b (i): to a mixture of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (185 mg, 1.22 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (580 mg, 3.33 mmol) in a microwave vessel was added a solution of <i>N</i>-(1,3-dimethoxypropan-2-yl)-2-nitroaniline (267 mg, 1.11 mmol) in EtOH (2 mL) followed by water (1 mL). The vial was sealed and heated in a microwave at 100 °C for 65 min. The reaction mixture was diluted with water (3 mL) and extracted with EtOAc (2 × 5 mL). The organic extracts were combined and passed through a hydrophobic frit, and the solvent was removed under vacuum. The residue was purified by MDAP (method B). The appropriate fractions were combined, and the solvent was removed by rotary evaporation to give the title compound as a white solid (129 mg, 0.38 mmol, 34% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.05 (d, <i>J</i> = 2.5 Hz, 1 H), 7.82–7.77 (m, 1 H), 7.70–7.67 (m, 1 H), 7.65–7.60 (m, 1 H), 7.25–7.17 (m, 2 H), 4.84 (tt, <i>J</i> = 9.0, 4.5 Hz, 1 H), 4.02 (dd, <i>J</i> = 10.5, 9.0 Hz, 2 H), 3.77 (dd, <i>J</i> = 10.5, 4.5 Hz, 2 H), 3.54 (s, 3 H), 3.16 (s, 6 H), 2.09 (s, 3 H); LC–MS (method F) <i>m</i>/<i>z</i>: 342 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.60 min; 100% purity.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 4-Bromo-2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)aniline (<b>21a</b>)</h4><div class="NLM_p last">4-Bromo-1-fluoro-2-nitrobenzene (5 g, 22.7 mmol) was taken up in 2-MeTHF (50 mL). DIPEA (6.0 mL, 34.1 mmol) and (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine (3.93 g, 34.1 mmol) were added, and the reaction mixture was heated to 80 °C. The reaction was cooled and diluted with EtOAc (100 mL) and then washed with water (100 mL) and brine (100 mL). The organic layer was dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to yield the title compound as an orange solid (7.15 g, 21.6 mmol, 95% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.33 (d, <i>J</i> = 2.5 Hz, 1 H), 8.13 (t, <i>J</i> = 5.5 Hz, 1 H), 7.51 (dd, <i>J</i> = 9.0, 2.5 Hz, 1 H), 6.77 (d, <i>J</i> = 9.0 Hz, 1 H), 4.03 (dd, <i>J</i> = 11.5, 4.5 Hz, 2 H), 3.42 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 3.21 (dd, <i>J</i> = 6.5, 5.5 Hz, 2 H), 1.95 (ttt, <i>J</i> = 11.5, 6.5, 3.5 Hz, 1 H), 1.75 (ddd, <i>J</i> = 13.0, 3.5, 2.0 Hz, 2 H), 1.43 (dtd, <i>J</i> = 13.0, 11.5, 4.5 Hz, 2 H); LC–MS (method A) <i>m</i>/<i>z</i>: not observed [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.25 min; 90% purity.</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 5-Bromo-2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)aniline (<b>21b</b>)</h4><div class="NLM_p last">4-Bromo-2-fluoro-1-nitrobenzene (10 g, 44.7 mmol) was dissolved in 2-MeTHF (100 mL), and (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine (5.47 mL, 44.7 mmol) and DIPEA (11.7 mL, 67.1 mmol) were added. The reaction mixture was heated at 80 °C for 2 h and then added to water (200 mL) and extracted with EtOAc (2 × 200 mL). The combined organics were washed with water (200 mL) and brine (200 mL), dried, and evaporated in vacuo to give the title compound as a yellow solid (12.2 g, 38.7 mmol, 87% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.14 (t, <i>J</i> = 5.5 Hz, 1 H), 8.04 (d, <i>J</i> = 9.0 Hz, 1 H), 7.02 (d, <i>J</i> = 2.0 Hz, 1 H), 6.77 (dd, <i>J</i> = 9.0, 2.0 Hz, 1 H), 4.04 (dd, <i>J</i> = 11.0, 4.5 Hz, 2 H), 3.44 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 3.20 (dd, <i>J</i> = 6.5, 5.5 Hz, 2 H), 1.96 (ttt, <i>J</i> = 11.5, 6.5, 3.5 Hz, 1 H), 1.76 (ddd, <i>J</i> = 13.0, 3.5, 2.0 Hz, 2 H), 1.44 (dtd, <i>J</i> = 13.0, 11.5, 4.5 Hz, 2 H). LC–MS (method F) <i>m</i>/<i>z</i>: 315, 317 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.25 min; 100% purity.</div></div><div id="sec4_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 5-(5-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22a</b>)</h4><div class="NLM_p last">4-Bromo-2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)aniline <b>21a</b> (15.6 g, 49.5 mmol) and 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (7.48 g, 49.5 mmol) were combined in EtOH (375 mL) and water (150 mL). Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (25.9 g, 148 mmol) was added and the mixture heated to 90 °C. The reaction was cooled and concentrated to remove the EtOH. The residue was partitioned between 10:1 DCM/<sup><i>i</i></sup>PrOH (500 mL) and water (400 mL). The aqueous layer was re-extracted with 10:1 DCM/<sup><i>i</i></sup>PrOH (500 mL), and the combined organics were dried with Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to yield an orange oil. The crude product was applied to a 340 g silica cartridge in the minimum amount of DCM and eluted with 5% (3:1 EtOAc/EtOH) in cyclohexane for 2 column volumes, then 5–50% (3:1 EtOAc/EtOH) over 10 column volumes, and then held at 50% for 15 column volumes. The appropriate fractions were concentrated in vacuo to give the title compound as white foam (9.21 g, 21.0 mmol, 43% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.87 (d, <i>J</i> = 1.5 Hz, 1 H), 7.73 (d, <i>J</i> = 2.5 Hz, 1 H), 7.48–7.45 (m, 1 H), 7.39 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 7.24 (d, <i>J</i> = 8.5 Hz, 1 H), 4.11 (d, <i>J</i> = 7.5 Hz, 2 H), 3.88 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.64 (s, 3 H), 3.22 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 2.23 (s, 3 H), 2.11–1.95 (m, 1 H), 1.35–1.18 (m, 4 H); LC–MS (method A) <i>m</i>/<i>z</i>: 416, 418 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.96 min; 100% purity.</div></div><div id="sec4_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 5-(6-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22b</b>)</h4><div class="NLM_p last">5-Bromo-2-nitro-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)aniline <b>21b</b> (12.2 g, 38.7 mmol) and 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (6.4 g, 42.6 mmol) were combined in EtOH (200 mL) and water (100 mL), and the mixture was heated to 80 °C; then, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (20.2 g, 116 mmol) was added, and the resulting suspension was heated for 18 h at 80 °C. The mixture was evaporated in vacuo to approximately half its original volume and then extracted with EtOAc (3 × 100 mL), and the combined organics were dried and evaporated in vacuo to give colorless foam. This was dissolved in DCM (30 mL) and loaded onto a 340 g silica column and then eluted with 0–25% EtOH in EtOAc, and product-containing fractions evaporated in vacuo to give the title compound as a pale yellow solid (10.2 g, 24.5 mmol, 63% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.73 (d, <i>J</i> = 2.5 Hz, 1 H), 7.62 (d, <i>J</i> = 8.5 Hz, 1 H), 7.54 (d, <i>J</i> = 1.5 Hz, 1 H), 7.49–7.46 (m, 1 H), 7.41 (dd, <i>J</i> = 8.5, 1.5 Hz, 1 H), 4.11 (d, <i>J</i> = 7.5 Hz, 2 H), 3.91 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.66 (s, 3 H), 3.27 (td, <i>J</i> = 11.5, 2.5 Hz, 2 H), 2.25 (s, 3 H), 2.15–2.04 (m, 1 H), 1.38–1.21 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 416, 418 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.85 min; 100% purity.</div></div><div id="sec4_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 5-(5-(Dimethylamino)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22c</b>)</h4><div class="NLM_p last">A mixture of 5-(5-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (100 mg, 0.24 mmol), 2 M dimethylamine in THF (0.6 mL, 1.20 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (9 mg, 0.01 mmol), JohnPhos (9 mg, 0.03 mmol), and NaO<sup><i>t</i></sup>Bu (35 mg, 0.36 mmol) in toluene (2 mL) in a sealed microwave vial was purged with nitrogen and heated in a microwave at 90 °C for 20 min. The reaction mixture was filtered over Celite, and the pad was washed with EtOAc (20 mL). The solvent was removed in vacuo, and the residue was dissolved in 1:1 DMSO/MeOH (2 mL) and purified by MDAP (method A). The solvent was evaporated in vacuo, and the residue was dissolved in MeOH (0.5 mL). The solution was applied to a 0.5 g MeOH-preconditioned aminopropyl cartridge which was then washed with MeOH (3 mL). The wash was evaporated under a stream of nitrogen, and the residue was dried in a high-vacuum oven to give the title compound as an off-white solid (48 mg, 0.13 mmol, 53% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.72 (d, <i>J</i> = 2.5 Hz, 1 H), 7.52–7.48 (m, 1 H), 7.25 (d, <i>J</i> = 9.0 Hz, 1 H), 7.11 (d, <i>J</i> = 2.5 Hz, 1 H), 6.92 (dd, <i>J</i> = 9.0, 2.5 Hz, 1 H), 4.08 (d, <i>J</i> = 7.5 Hz, 2 H), 3.90 (dd, <i>J</i> = 11.5, 4.0 Hz, 2 H), 3.65 (s, 3 H), 3.24 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 2.99 (s, 6 H), 2.25 (s, 3 H), 2.09 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.36 (ddd, <i>J</i> = 13.0, 3.5, 2.0 Hz, 2 H), 1.27 (dtd, <i>J</i> = 13.0, 11.5, 4.0 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 381 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.48 min; 100% purity.</div></div><div id="sec4_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 5-(6-(Dimethylamino)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22d</b>)</h4><div class="NLM_p">5-(6-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (50 mg, 0.12 mmol), JohnPhos (4.6 mg, 0.015 mmol), 2 M dimethylamine in THF (0.10 mL, 0.20 mmol), NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (4.5 mg, 0.05 mmol) were suspended in 1,4-dioxane (1 mL). The reaction vessel was sealed and heated in a microwave at 90 °C for 20 min. The reaction mixture was allowed to cool to rt, and JohnPhos (4.6 mg, 0.015 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (4.5 mg, 0.005 mmol), and NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol) were added to the reaction vessel. The reaction vessel was sealed and heated in a microwave at 90 °C for 20 min. The reaction was allowed to cool to rt, and the solution was loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [3 mL]). The column was flushed with MeCN (3 mL), and the filtrate was evaporated under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The appropriate fractions were evaporated, and the sample was dissolved in DMSO (1 mL) and purified by MDAP (method A) to give the title compound (8.9 mg, 0.02 mmol, 19% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.05 (d, <i>J</i> = 2.5 Hz, 1 H), 7.72–7.68 (m, 1 H), 7.41 (d, <i>J</i> = 9.0 Hz, 1 H), 6.83 (d, <i>J</i> = 2.5 Hz, 1 H), 6.76 (dd, <i>J</i> = 9.0, 2.5 Hz, 1 H), 4.20 (d, <i>J</i> = 7.5 Hz, 2 H), 3.72 (dd, <i>J</i> = 11.5, 4.0 Hz, 2 H), 3.55 (s, 3 H), 3.11 (td, <i>J</i> = 11.5, 2.5 Hz, 2 H), 2.95 (s, 6 H), 2.10 (s, 3 H), 1.94 (ddd, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.28–1.10 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 381 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.50 min; 99% purity.</div><div id="sec4_1_56_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> General S<sub>N</sub>Ar-Cyclization Method</h5><div class="NLM_p last">To a solution of the appropriately substituted 1-fluoro-2-nitrobenzene (83 mg, 0.52 mmol) in 2-MeTHF (0.5 mL) were added (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine (50 mg, 0.43 mmol) and DIPEA (0.23 mL, 1.30 mmol). The reaction vessel was sealed and heated in a microwave at 110 °C for 20 min. The solvent was removed under a stream of nitrogen. To the dried material were added 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (72 mg, 0.48 mmol) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (227 mg, 1.30 mmol) as a solution in EtOH (1.0 mL), followed by water (0.5 mL). The sealed vessels were heated at 100 °C for 18 h. The solvent was removed under a stream of nitrogen. The reaction mixture was partitioned between saturated aqueous NaHCO<sub>3</sub> (3 mL) and DCM (2 × 3 mL). The DCM extracts were evaporated under a stream of nitrogen. The residue was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was removed under a stream of nitrogen to give the compounds <b>22e–22p</b>.</div></div></div><div id="sec4_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 5-(5-Methoxy-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22e</b>)</h4><div class="NLM_p last">7 mg, 0.02 mmol, 4% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.09 (s, 1 H), 7.72 (s, 1 H), 7.56 (d, <i>J</i> = 8.5 Hz, 1 H), 7.13 (d, <i>J</i> = 2.0 Hz, 1 H), 6.88 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.23 (d, <i>J</i> = 7.5 Hz, 2 H), 3.79 (s, 3 H), 3.71 (d, <i>J</i> = 11.5 Hz, 2 H), 3.55 (s, 3 H), 3.10 (t, <i>J</i> = 11.5 Hz, 2 H), 2.10 (s, 3 H), 1.94 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.20 (d, <i>J</i> = 11.5 Hz, 2 H), 1.13 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H). LC–MS (method F) <i>m</i>/<i>z</i>: 368 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.51 min; 96% purity.</div></div><div id="sec4_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 5-(6-Methoxy-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22f</b>)</h4><div class="NLM_p last">8 mg, 0.02 mmol, 4% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.08 (s, 1 H), 7.71 (s, 1 H), 7.48 (d, <i>J</i> = 8.5 Hz, 1 H), 7.22 (s, 1 H), 6.83 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.24 (d, <i>J</i> = 7.5 Hz, 2 H), 3.83 (s, 3 H), 3.71 (d, <i>J</i> = 11.5 Hz, 2 H), 3.55 (s, 3 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.10 (s, 3 H), 1.93 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.20 (d, <i>J</i> = 11.5 Hz, 2 H), 1.19–1.11 (m, 2 H). LC–MS (method F) <i>m</i>/<i>z</i>: 368 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.53 min; 96% purity.</div></div><div id="sec4_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 1,3-Dimethyl-5-(5-methyl-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>22g</b>)</h4><div class="NLM_p last">11 mg, 0.03 mmol, 7% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.11 (s, 1 H), 7.72 (s, 1 H), 7.55 (d, <i>J</i> = 8.0 Hz, 1 H), 7.40 (s, 1 H), 7.08 (d, <i>J</i> = 8.0 Hz, 1 H), 4.24 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.55 (s, 3 H), 3.10 (t, <i>J</i> = 11.5 Hz, 2 H), 2.41 (s, 3 H), 2.10 (s, 3 H), 1.94 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.19 (dd, <i>J</i> = 11.5 Hz, 2 H), 1.13 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H). LC–MS (method F) <i>m</i>/<i>z</i>: 352 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.54 min; 100% purity.</div></div><div id="sec4_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 1,3-Dimethyl-5-(6-methyl-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>22h</b>)</h4><div class="NLM_p last">11 mg, 0.03 mmol, 6% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.10 (s, 1 H), 7.72 (s, 1 H), 7.48 (d, <i>J</i> = 8.5 Hz, 1 H), 7.47 (s, 1 H), 7.03 (d, <i>J</i> = 8.5 Hz, 1 H), 4.23 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.55 (s, 3 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.46 (s, 3 H), 2.10 (s, 3 H), 1.95 (ttt, <i>J</i> = 11.5, 7.5, 3.5, Hz, 1 H), 1.21 (dd, <i>J</i> = 11.5 Hz, 2 H), 1.14 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H). LC–MS (method F) <i>m</i>/<i>z</i>: 352 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.55 min; 100% purity.</div></div><div id="sec4_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 5-(5-Fluoro-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22i</b>)</h4><div class="NLM_p last">15 mg, 0.04 mmol, 9% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.14 (s, 1 H), 7.74 (s, 1 H), 7.72 (dd, <i>J</i> = 8.5, 4.5 Hz, 1 H), 7.41 (dd, <i>J</i> = 9.5, 2.0 Hz, 1 H), 7.13 (ddd, <i>J</i> = 9.5, 8.5, 2.0 Hz, 1 H), 4.29 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.56 (s, 3 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.10 (s, 3 H), 1.95 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.20 (dd, <i>J</i> = 11.5, Hz, 2 H), 1.15 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H). LC–MS (method F) <i>m</i>/<i>z</i>: 356 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.65 min; 99% purity.</div></div><div id="sec4_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 5-(5-Chloro-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22j</b>)</h4><div class="NLM_p last">41 mg, 0.11 mmol, 23% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.15 (s, 1 H), 7.77–7.71 (m, 2 H), 7.67 (s, 1 H), 7.29 (d, <i>J</i> = 8.5 Hz, 1 H), 4.29 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (d, <i>J</i> = 11.5 Hz, 2 H), 3.56 (s, 3 H), 3.10 (t, <i>J</i> = 11.5 Hz, 2 H), 2.10 (s, 3 H), 1.98–1.89 (m, 1 H), 1.22–1.09 (m, 4 H). LC–MS (method F) <i>m</i>/<i>z</i>: 372, 374 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.79 min; 99% purity.</div></div><div id="sec4_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 5-(6-Chloro-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>22k</b>)</h4><div class="NLM_p last">19 mg, 0.05 mmol, 11% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.15 (s, 1 H), 7.89 (s, 1 H), 7.74 (s, 1 H), 7.62 (d, <i>J</i> = 8.5 Hz, 1 H), 7.23 (d, <i>J</i> = 8.5 Hz, 1 H), 4.29 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.55 (s, 3 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.10 (s, 3 H), 1.93 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.20 (dd, <i>J</i> = 11.5 Hz, 2 H), 1.14 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H). LC–MS (method F) <i>m</i>/<i>z</i>: 372, 374 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.78 min; 99% purity.</div></div><div id="sec4_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 2-(1,5-Dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide (<b>22l</b>)</h4><div class="NLM_p last">30 mg, 0.08 mmol, 16% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.18 (s, 1 H), 8.16 (s, 1 H), 7.98 (br s, 1 H), 7.83 (d, <i>J</i> = 8.5 Hz, 1 H), 7.75 (s, 1 H), 7.73 (d, <i>J</i> = 8.5 Hz, 1 H), 7.26 (br s, 1 H), 4.30 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.56 (s, 3 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.11 (s, 3 H), 1.95 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.23–1.12 (m, 4 H). LC–MS (method F) <i>m</i>/<i>z</i>: 381 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.49 min; 100% purity.</div></div><div id="sec4_1_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 2-(1,5-Dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-6-carboxamide (<b>22m</b>)</h4><div class="NLM_p last">1 mg, 0.003 mmol, 1% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.22 (s, 1 H), 8.18 (s, 1 H), 8.00 (br s, 1 H), 7.79 (d, <i>J</i> = 8.5 Hz, 1 H), 7.77 (s, 1 H), 7.63 (d, <i>J</i> = 8.5 Hz, 1 H), 7.35 (br s, 1 H), 4.32 (d, <i>J</i> = 7.5 Hz, 2 H), 3.72 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.56 (s, 3 H), 3.12 (t, <i>J</i> = 11.5 Hz, 2 H), 2.11 (s, 3 H), 2.00 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.24–1.13 (m, 4 H). LC–MS (method F) <i>m</i>/<i>z</i>: 381 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.47 min; 100% purity.</div></div><div id="sec4_1_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 2-(1,5-Dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carbonitrile (<b>22n</b>)</h4><div class="NLM_p last">54 mg, 0.15 mmol, 31% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.21 (s, 1 H), 8.16 (s, 1 H), 7.94 (d, <i>J</i> = 8.5 Hz, 1 H), 7.77 (s, 1 H), 7.67 (d, <i>J</i> = 8.5 Hz, 1 H), 4.36 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.56 (s, 3 H), 3.10 (td, <i>J</i> = 11.5, 3.5 Hz, 2 H), 2.11 (s, 3 H), 1.93 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.22–1.12 (m, 4 H). LC–MS (method F) <i>m</i>/<i>z</i>: 363 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.74 min; 97% purity.</div></div><div id="sec4_1_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 2-(1,5-Dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-6-carbonitrile (<b>22o</b>)</h4><div class="NLM_p last">36 mg, 0.10 mmol, 20% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.40 (s, 1 H), 8.24–8.22 (m, 1 H), 7.79 (s, 1 H), 7.77 (d, <i>J</i> = 8.5 Hz, 1 H), 7.60 (d, <i>J</i> = 8.5 Hz, 1 H), 4.37 (d, <i>J</i> = 7.5 Hz, 2 H), 3.72 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.56 (s, 3 H), 3.12 (td, <i>J</i> = 11.5, 3.5 Hz, 2 H), 2.11 (s, 3 H), 1.97 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.22–1.13 (m, 4 H). LC–MS (method F) <i>m</i>/<i>z</i>: 363 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.74 min; 95% purity.</div></div><div id="sec4_1_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 1,3-Dimethyl-5-(5-(methylsulfonyl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>22p</b>)</h4><div class="NLM_p last">11 mg, 0.03 mmol, 6% yield. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.20 (d, <i>J</i> = 2.5 Hz, 1 H), 8.16–8.15 (m, 1 H), 7.99 (d, <i>J</i> = 8.5 Hz, 1 H), 7.81 (d, <i>J</i> = 8.5 Hz, 1 H), 7.77 (s, 1 H), 4.37 (d, <i>J</i> = 7.5 Hz, 2 H), 3.72 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.57 (s, 3 H), 3.24 (s, 3 H), 3.11 (td, <i>J</i> = 11.5, 3.5 Hz, 2 H), 2.11 (s, 3 H), 1.95 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.24–1.13 (m, 4 H). LC–MS (method F) <i>m</i>/<i>z</i>: 416 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.64 min; 99% purity.</div></div><div id="sec4_1_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 5-(5-(Cyclobutylamino)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>23a</b>)</h4><div class="NLM_p last">A suspension of 5-(5-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (50 mg, 0.12 mmol), NaO<sup><i>t</i></sup>Bu (34.6 mg, 0.36 mmol), BrettPhos (6.5 mg, 0.01 mmol), and BrettPhos Pd G1 precatalyst (9.5 mg, 0.01 mmol) in 1,4-dioxane (1 mL) was prepared in a vial containing cyclobutylamine hydrochloride (15 mg, 0.14 mmol). A stirrer bar was added, and the vial was sealed, sonicated, and heated in a microwave at 110 °C for 30 min. The reaction was allowed to cool and loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN). The cartridge was flushed with MeCN (2 mL) and MeOH (2 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was removed using a plate dryer to give the title compound (5 mg, 0.01 mmol, 9% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.04 (s, 1 H), 7.69 (s, 1 H), 7.34 (d, <i>J</i> = 8.5 Hz, 1 H), 6.61–6.57 (m, 2 H), 5.57 (d, <i>J</i> = 7.0 Hz, 1 H), 4.14 (d, <i>J</i> = 7.5 Hz, 2 H), 3.88–3.79 (m, 1 H), 3.71 (d, <i>J</i> = 11.5 Hz, 2 H), 3.54 (s, 3 H), 3.10 (t, <i>J</i> = 11.5 Hz, 2 H), 2.38–2.31 (m, 2 H), 2.09 (s, 3 H), 1.97–1.88 (m, 1 H), 1.86–1.77 (m, 2 H), 1.77–1.69 (m, 2 H), 1.20 (d, <i>J</i> = 11.5 Hz, 2 H), 1.11 (q, <i>J</i> = 11.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 407 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.61 min; 95% purity.</div></div><div id="sec4_1_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 5-(6-(Cyclobutylamino)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>23b</b>)</h4><div class="NLM_p last">A suspension of 5-(6-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (50 mg, 0.12 mmol), NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol), BrettPhos (6.5 mg, 0.01 mmol), and BrettPhos Pd G1 precatalyst (9.6 mg, 0.012 mmol), in 1,4-dioxane (1 mL), was prepared in a vial containing cyclobutylamine hydrochloride (12.9 mg, 0.12 mmol). A stirrer bar was added, and the vial was sealed and heated in a microwave at 120 °C for 30 min. The reaction was allowed to cool and loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN). The cartridge was flushed with MeCN (2 mL) and MeOH (2 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was removed using a plate dryer to give the title compound (11 mg, 0.03 mmol, 20% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.00 (d, <i>J</i> = 2.5 Hz, 1 H), 7.69–7.66 (m, 1 H), 7.29 (d, <i>J</i> = 8.5 Hz, 1 H), 6.54–6.50 (m, 2 H), 5.77 (d, <i>J</i> = 7.0 Hz, 1 H), 4.12 (d, <i>J</i> = 7.5 Hz, 2 H), 3.98–3.87 (m, 1 H), 3.72 (d, <i>J</i> = 11.5 Hz, 2 H), 3.57–3.52 (m, 3 H), 3.12 (td, <i>J</i> = 11.5, 3.0 Hz, 2 H), 2.44–2.34 (m, 2 H), 2.09 (s, 3 H), 1.93 (d, <i>J</i> = 11.0 Hz, 1 H), 1.87–1.69 (m, 4 H), 1.28–1.09 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 407 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.61 min; 97% purity.</div></div><div id="sec4_1_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 1,3-Dimethyl-5-(5-(pyrrolidin-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23c</b>)</h4><div class="NLM_p last">5-(5-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (100 mg, 0.240 mmol), pyrrolidine (0.02 mL, 0.24 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (9 mg, 0.01 mmol), JohnPhos (9 mg, 0.03 mmol), and NaO<sup><i>t</i></sup>Bu (23 mg, 0.24 mmol) were suspended in toluene (2 mL) within a sealed microwave vial. The reaction was heated in a microwave at 90 °C for 10 min. The reaction mixture was filtered over a Celite pad and washed with EtOAc (20 mL). The solvent was removed in vacuo, and the residue was dissolved in 1:1 DMSO/MeOH (0.6 mL) and purified by MDAP (method B). The solvent was evaporated in vacuo to give the title compound (29 mg, 0.07 mmol, 30% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.72 (d, <i>J</i> = 2.5 Hz, 1 H), 7.52–7.49 (m, 1 H), 7.23 (d, <i>J</i> = 8.5 Hz, 1 H), 6.90 (d, <i>J</i> = 2.0 Hz, 1 H), 6.70 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.08 (d, <i>J</i> = 7.5 Hz, 2 H), 3.89 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.65 (s, 3 H), 3.38–3.32 (m, 4 H), 3.24 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 2.25 (s, 3 H), 2.16–2.03 (m, 5 H), 1.40–1.20 (m, 4 H); LC–MS (method A) <i>m</i>/<i>z</i>: 407 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.98 min; 97% purity.</div></div><div id="sec4_1_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 1,3-Dimethyl-5-(6-(pyrrolidin-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23d</b>)</h4><div class="NLM_p last">5-(6-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (100 mg, 0.24 mmol), pyrrolidine (0.02 mL, 0.24 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (8.8 mg, 0.01 mmol), JohnPhos (8.6 mg, 0.03 mmol), and NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol) were dissolved in toluene (2 mL). The reaction was heated in a microwave at 90 °C for 10 min. The reaction was heated in a microwave at 90 °C for a further 20 min. The cooled reaction mixture was filtered through Celite washing with EtOAc. The filtrate was concentrated, and the crude material was purified by loading onto a 12 g silica column and eluting using a graduating solvent system of 0–30% EtOH in EtOAc. The appropriate fractions were combined, concentrated, and purified by MDAP (method A). Concentration of the collected fractions gave the title compound as a tan solid (22 mg, 0.05 mmol, 23% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.00 (s, 1 H), 7.63 (d, <i>J</i> = 8.5 Hz, 1 H), 7.43 (s, 1 H), 6.73 (d, <i>J</i> = 8.5 Hz, 1 H), 6.34 (s, 1 H), 4.09 (d, <i>J</i> = 7.5 Hz, 2 H), 3.92 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.67 (s, 3 H), 3.41–3.34 (m, 4 H), 3.26 (t, <i>J</i> = 11.5 Hz, 2 H), 2.25 (s, 3 H), 2.18–2.05 (m, 5 H), 1.42–1.22 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 407 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.64 min; 100% purity.</div></div><div id="sec4_1_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 1,3-Dimethyl-5-(5-morpholino-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23e</b>)</h4><div class="NLM_p last">5-(5-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (100 mg, 0.24 mmol), morpholine (0.03 mL, 0.34 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (9 mg, 0.01 mmol), JohnPhos (9 mg, 0.03 mmol), and NaO<sup><i>t</i></sup>Bu (23 mg, 0.24 mmol) were suspended in toluene (2 mL) within a sealed microwave vial. The reaction mixture was heated in a microwave at 90 °C for 10 min. The reaction mixture was cooled to rt, filtered over a Celite pad, and washed with EtOAc (20 mL). The solvent was removed in vacuo, and the residue was dissolved in 1:1 DMSO/MeOH (0.6 mL) and purified by MDAP (method B). The solvent was evaporated in vacuo to give the title compound (56 mg, 0.13 mmol, 55% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.73 (d, <i>J</i> = 2.5 Hz, 1 H), 7.51–7.48 (m, 1 H), 7.29 (d, <i>J</i> = 9.0 Hz, 1 H), 7.27 (d, <i>J</i> = 2.5 Hz, 1 H), 7.04 (dd, <i>J</i> = 9.0, 2.5 Hz, 1 H), 4.10 (d, <i>J</i> = 7.5 Hz, 2 H), 3.94–3.87 (m, 6 H), 3.66 (s, 3 H), 3.25 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 3.20–3.17 (m, 4 H), 2.25 (s, 3 H), 2.09 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.39–1.22 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 423 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.55 min; 100% purity.</div></div><div id="sec4_1_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 1,3-Dimethyl-5-(6-morpholino-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23f</b>)</h4><div class="NLM_p last">5-(6-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (100 mg, 0.24 mmol), morpholine (0.03 mL, 0.34 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (9 mg, 0.01 mmol), JohnPhos (9 mg, 0.03 mmol), and NaO<sup><i>t</i></sup>Bu (23 mg, 0.24 mmol) were suspended in toluene (2 mL) within a sealed microwave vial. The reaction mixture was heated in a microwave at 90 °C for 10 min. The reaction mixture was allowed to cool to rt, filtered over Celite, and washed with EtOAc (50 mL). The solvent was removed in vacuo, and the sample was dissolved in 1:1 DMSO/MeOH (0.6 mL) and purified by MDAP (method B). The solvent was evaporated in vacuo to give the title compound (29 mg, 0.07 mmol, 29% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.70 (d, <i>J</i> = 2.5 Hz, 1 H), 7.65 (d, <i>J</i> = 8.5 Hz, 1 H), 7.49–7.46 (m, 1 H), 7.02 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 6.80 (d, <i>J</i> = 2.0 Hz, 1 H), 4.09 (d, <i>J</i> = 7.5 Hz, 2 H), 3.96–3.88 (m, 6 H), 3.65 (s, 3 H), 3.30–3.20 (m, 6 H), 2.25 (s, 3 H), 2.07 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.40–1.23 (m, 4 H); LC–MS (method A) <i>m</i>/<i>z</i>: 423 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.76 min; 100% purity.</div></div><div id="sec4_1_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 2-((2-(1,5-Dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)-<i>N</i>,<i>N</i>-dimethylacetamide (<b>23g</b>)</h4><div class="NLM_p last">5-(5-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (50 mg, 0.12 mmol), NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol), BrettPhos (8.3 mg, 0.02 mmol), and BrettPhos Pd G1 precatalyst (4.8 mg, 0.006 mmol) were added to 2-amino-<i>N</i>,<i>N</i>-dimethylacetamide acetate (19 mg, 0.12 mmol) and suspended in 1,4-dioxane (1.5 mL). DIPEA (0.02 mL) was added to the mixture, and the reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. After cooling the reaction to rt, the reaction mixture was loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [3 mL]). The cartridge was flushed with MeCN (3 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was dried under a stream of nitrogen to give the title compound (25 mg, 0.06 mmol, 43% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.05 (s, 1 H), 7.69 (s, 1 H), 7.37 (d, <i>J</i> = 8.5 Hz, 1 H), 6.77–6.73 (m, 2 H), 5.30 (t, <i>J</i> = 5.0 Hz, 1 H), 4.16 (d, <i>J</i> = 7.5 Hz, 2 H), 3.89 (d, <i>J</i> = 5.0 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.54 (s, 3 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 3.05 (s, 3 H), 2.89 (s, 3 H), 2.09 (s, 3 H), 1.93 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.20 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 1.12 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 438 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.48 min; 95% purity.</div></div><div id="sec4_1_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 2-((2-(1,5-Dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-6-yl)amino)-<i>N</i>,<i>N</i>-dimethylacetamide (<b>23h</b>)</h4><div class="NLM_p last">5-(6-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (50 mg, 0.12 mmol), BrettPhos (8.3 mg, 0.02 mmol), BrettPhos Pd G1 precatalyst (4.8 mg, 0.006 mmol), and NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol) were added to 2-amino-<i>N</i>,<i>N</i>-dimethylacetamide acetate (19 mg, 0.12 mmol) suspended in 1,4-dioxane (1.5 mL). The reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. After cooling the reaction to rt, BrettPhos (8.3 mg, 0.02 mmol) and BrettPhos Pd G1 precatalyst (4.8 mg, 0.006 mmol) were added. The reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. After cooling the reaction to rt, the reaction mixture was loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [3 mL]). The cartridge was flushed with MeCN (3 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was evaporated under a stream of nitrogen to give the title compound (12 mg, 0.03 mmol, 21% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.03 (s, 1 H), 7.69 (s, 1 H), 7.32 (d, <i>J</i> = 8.5 Hz, 1 H), 6.74 (s, 1 H), 6.68 (d, <i>J</i> = 8.5 Hz, 1 H), 5.47 (t, <i>J</i> = 5.0 Hz, 1 H), 4.14 (d, <i>J</i> = 7.5 Hz, 2 H), 3.94 (d, <i>J</i> = 5.0 Hz, 2 H), 3.73 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.54 (s, 3 H), 3.12 (t, <i>J</i> = 11.5 Hz, 2 H), 3.07 (s, 3 H), 2.89 (s, 3 H), 2.09 (s, 3 H), 1.94 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.23 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 1.16 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 438 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.48 min; 100% purity.</div></div><div id="sec4_1_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 1,3-Dimethyl-5-(5-(4-methyl-1,4-diazepan-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)pyridin-2(1<i>H</i>)-one (<b>23i</b>)</h4><div class="NLM_p last">5-(5-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (50 mg, 0.12 mmol), NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol), and RuPhos Pd G2 precatalyst (2.4 mg, 0.003 mmol) were added to 1-methyl-1,4-diazepane (14 mg, 0.12 mmol) in 1,4-dioxane. The reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. After cooling the reaction to rt, the RuPhos Pd G2 precatalyst (2.4 mg, 0.003 mmol) was added to the reaction mixture. The reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. After cooling the reaction to rt, the reaction mixture was loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [3 mL]). The cartridge was flushed with MeCN (3 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was evaporated under a stream of nitrogen to give the title compound (13 mg, 0.03 mmol, 22% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.05 (s, 1 H), 7.70 (s, 1 H), 7.42 (d, <i>J</i> = 8.5 Hz, 1 H), 6.80 (d, <i>J</i> = 2.0 Hz, 1 H), 6.76 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.16 (d, <i>J</i> = 7.5 Hz, 2 H), 3.72 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.55 (s, 3 H), 3.54–3.51 (m, 2 H), 3.45 (t, <i>J</i> = 6.0 Hz, 2 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.66–2.62 (m, 2 H), 2.48–2.45 (m, 2 H), 2.26 (s, 3 H), 2.09 (s, 3 H), 1.99–1.89 (m, 3 H), 1.22 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 1.13 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 450 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.35 min; 95% purity.</div></div><div id="sec4_1_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 1,3-Dimethyl-5-(6-(4-methyl-1,4-diazepan-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23j</b>)</h4><div class="NLM_p last">1-Methyl-1,4-diazepane (14 mg, 0.120 mmol) was dissolved in 1,4-dioxane (1.5 mL) and added to 5-(6-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (50 mg, 0.12 mmol), JohnPhos (4.6 mg, 0.02 mmol), NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (4.5 mg, 0.005 mmol). The reaction vessel was sealed and heated in a microwave at 110 °C for 20 min. After cooling the reaction to rt, the reaction mixture was loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [3 mL]). The cartridge was flushed with MeCN (3 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was evaporated under a stream of nitrogen to give the title compound (26 mg, 0.06 mmol, 43% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.04 (s, 1 H), 7.69 (s, 1 H), 7.39 (d, <i>J</i> = 8.5 Hz, 1 H), 6.78 (d, <i>J</i> = 2.0 Hz, 1 H), 6.72 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.19 (d, <i>J</i> = 7.5 Hz, 2 H), 3.72 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.66–3.62 (m, 2 H), 3.54 (s, 3 H), 3.51 (t, <i>J</i> = 6.0 Hz, 2 H), 3.10 (t, <i>J</i> = 11.5 Hz, 2 H), 2.89–2.81 (m, 2 H), 2.71–2.65 (m, 2 H), 2.42 (s, 3 H), 2.09 (s, 3 H), 2.05–1.98 (m, 2 H), 1.91 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.21 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 1.16 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 450 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.35 min; 98% purity.</div></div><div id="sec4_1_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 1-(2-(1,5-Dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-<i>N</i>-methylpiperidine-4-carboxamide (<b>23k</b>)</h4><div class="NLM_p last">A stock solution of 5-(5-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (300 mg, 0.12 mmol) was prepared in 1,4-dioxane (3 mL), and 0.5 mL aliquoted into a vial containing <i>N</i>-methylpiperidine-4-carboxamide (20 mg, 0.14 mmol). To the vial was added 1,4-dioxane (0.5 mL) along with NaO<sup><i>t</i></sup>Bu (34.6 mg, 0.36 mmol) and RuPhos Pd G2 precatalyst (18.7 mg, 0.024 mmol). A stirrer bar was added, and the vial was sonicated and capped before heating in a microwave at 110 °C for 30 min. The reaction mixture was allowed to cool and then loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN). The cartridge was flushed with MeCN (2 mL) and MeOH (2 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was removed using a plate dryer to give the title compound (15 mg, 0.03 mmol, 24% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.07 (d, <i>J</i> = 2.5 Hz, 1 H), 7.74 (q, <i>J</i> = 4.5 Hz, 1 H), 7.71–7.69 (m, 1 H), 7.50 (d, <i>J</i> = 8.5 Hz, 1 H), 7.07 (d, <i>J</i> = 2.0 Hz, 1 H), 7.01 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.20 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.61 (d, <i>J</i> = 12.0 Hz, 2 H), 3.55 (s, 3 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.63 (td, <i>J</i> = 12.0, 3.5 Hz, 2 H), 2.58 (d, <i>J</i> = 4.5 Hz, 3 H), 2.21 (tt, <i>J</i> = 11.5, 4.0 Hz, 1 H), 2.10 (s, 3 H), 1.93 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.81–1.68 (m, 4 H), 1.21 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 1.13 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 478 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.47 min; 100% purity.</div></div><div id="sec4_1_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> 1-(2-(1,5-Dimethyl-6-oxo-1,6-dihydropyridin-3-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-6-yl)-<i>N</i>-methylpiperidine-4-carboxamide (<b>23l</b>)</h4><div class="NLM_p last">To a vial containing <i>N</i>-methylpiperidine-4-carboxamide (20 mg, 0.14 mmol) was added 1.0 mL of a stock solution of 5-(6-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (150 mg, 0.12 mmol) in 1,4-dioxane (3 mL). Pd<sub>2</sub>(dba)<sub>3</sub> (11 mg, 0.01 mmol), NaO<sup><i>t</i></sup>Bu (34.6 mg, 0.36 mmol), JohnPhos (7.2 mg, 0.02 mmol), and a stirrer bar were added to the vial, which was sealed, sonicated, and heated in a microwave at 90 °C for 20 min. The reaction mixture was allowed to cool and then loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN). The cartridge was flushed with MeCN (2 mL) and MeOH (2 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was removed using a plate dryer to give the title compound (18 mg, 0.04 mmol, 28% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.06 (d, <i>J</i> = 2.5 Hz, 1 H), 7.75–7.68 (m, 2 H), 7.42 (d, <i>J</i> = 8.5 Hz, 1 H), 7.10 (d, <i>J</i> = 2.0 Hz, 1 H), 6.93 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.22 (d, <i>J</i> = 7.5 Hz, 2 H), 3.75–3.68 (m, 4 H), 3.55 (s, 3 H), 3.11 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 2.69 (td, <i>J</i> = 11.5, 3.5 Hz, 3 H), 2.59 (d, <i>J</i> = 4.5 Hz, 4 H), 2.24 (tt, <i>J</i> = 11.5, 4.0 Hz, 1 H), 2.10 (s, 3 H), 1.98–1.85 (m, 1 H), 1.85–1.67 (m, 4 H), 1.26–1.10 (m, 5 H); LC–MS (method F) <i>m</i>/<i>z</i>: 478 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.44 min; 100% purity.</div></div><div id="sec4_1_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> 1,3-Dimethyl-5-(5-(4-(methylamino)piperidin-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23m</b>)</h4><div class="NLM_p last">5-(5-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (50 mg, 0.12 mmol), NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.240 mmol), and RuPhos Pd G2 precatalyst (2.4 mg, 0.003 mmol) were added to <i>tert</i>-butyl methyl(piperidin-4-yl)carbamate (26 mg, 0.12 mmol) in 1,4-dioxane (1.5 mL). The reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. After cooling the reaction to rt, the RuPhos Pd G2 precatalyst (2.4 mg, 0.003 mmol) was added. The reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. After cooling the reaction to rt, the reaction mixture was loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [3 mL]). The cartridge was flushed with MeCN (3 mL), and the solvent was removed under a stream of nitrogen. HCl (4 M) in dioxane (0.5 mL) was added to the sample, and the reaction was left overnight. The solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was dried under a stream of nitrogen to give the title compound (12 mg, 0.03 mmol, 20% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.07 (s, 1 H), 7.70 (s, 1 H), 7.49 (d, <i>J</i> = 9.0 Hz, 1 H), 7.07 (d, <i>J</i> = 2.0 Hz, 1 H), 7.00 (dd, <i>J</i> = 9.0, 2.0 Hz, 1 H), 4.20 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.57–3.51 (m, 5 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.70 (t, <i>J</i> = 12.0 Hz, 2 H), 2.47–2.40 (m, 1 H), 2.32 (s, 3 H), 2.09 (s, 3 H), 1.98–1.88 (m, 3 H), 1.44–1.36 (m, 2 H), 1.21 (d, <i>J</i> = 11.5 Hz, 2 H), 1.13 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H); LC–MS (method A) <i>m</i>/<i>z</i>: 450 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.74 min; 96% purity.</div></div><div id="sec4_1_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> 1,3-Dimethyl-5-(6-(4-(methylamino)piperidin-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23n</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl methyl(piperidin-4-yl)carbamate (26 mg, 0.12 mmol) was dissolved in 1,4-dioxane (1.5 mL) and added to 5-(6-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (50 mg, 0.12 mmol), JohnPhos (4.6 mg, 0.02 mmol), NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (4.5 mg, 0.005 mmol). The reaction vessel was sealed and heated in a microwave at 110 °C for 30 min. After cooling the reaction to rt, the reaction mixture was loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [3 mL]). The cartridge was flushed with MeCN (3 mL), and the solvent was removed under a stream of nitrogen. HCl (4 M) in dioxane (0.5 mL) was added to the sample, and the reaction was left for 1 h. The solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was dried under a stream of nitrogen to give the title compound (13 mg, 0.03 mmol, 21% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.05 (s, 1 H), 7.70 (s, 1 H), 7.41 (d, <i>J</i> = 9.0 Hz, 1 H), 7.08 (s, 1 H), 6.92 (d, <i>J</i> = 9.0 Hz, 1 H), 4.21 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.61 (d, <i>J</i> = 12.0 Hz, 2 H), 3.54 (s, 3 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 2.76 (t, <i>J</i> = 12.0 Hz, 2 H), 2.44–2.38 (m, 1 H), 2.30 (s, 3 H), 2.09 (s, 3 H), 1.95–1.88 (m, 3 H), 1.43–1.34 (m, 2 H), 1.21 (d, <i>J</i> = 11.5 Hz, 2 H), 1.15 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 450 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.35 min; 98% purity.</div></div><div id="sec4_1_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> 1,3-Dimethyl-5-(5-(3-(methylsulfonyl)pyrrolidin-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23o</b>)</h4><div class="NLM_p last">A stock solution of 5-(5-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (250 mg, 0.12 mmol) was prepared in 1,4-dioxane (4 mL), and an aliquot (0.8 mL) was added to a vial containing 3-(methylsulfonyl)pyrrolidine (21 mg, 0.14 mmol). To the vial was added further 1,4-dioxane (0.4 mL) along with the RuPhos Pd G2 precatalyst (18.7 mg, 0.02 mmol) and NaO<sup><i>t</i></sup>Bu (34.6 mg, 0.36 mmol). A stirrer bar was added, and the vial was sonicated and capped before heating in a microwave at 110 °C for 30 min. The reaction mixture was allowed to cool and then loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN). The cartridge was flushed with MeCN (2 mL) and MeOH (2 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was removed using a plate dryer to give the title compound (21 mg, 0.04 mmol, 32% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.08 (s, 1 H), 7.71 (s, 1 H), 7.51 (d, <i>J</i> = 8.5 Hz, 1 H), 6.79 (s, 1 H), 6.73 (d, <i>J</i> = 8.5 Hz, 1 H), 4.20 (d, <i>J</i> = 7.5 Hz, 2 H), 4.10 (quin, <i>J</i> = 7.0 Hz, 1 H), 3.71 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.65–3.60 (m, 2 H), 3.55 (s, 3 H), 3.47–3.40 (m, 2 H), 3.10 (t, <i>J</i> = 11.5 Hz, 2 H), 3.06 (s, 3 H), 2.42–2.37 (m, 1 H), 2.10 (s, 3 H), 1.98–1.89 (m, 1 H), 1.19 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 1.13 (qd, <i>J</i> = 11.5, 3.5 Hz, 2 H); LC–MS (method F) <i>m</i>/<i>z</i>: 485 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.52 min; 100% purity.</div></div><div id="sec4_1_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> 1,3-Dimethyl-5-(6-(3-(methylsulfonyl)pyrrolidin-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)pyridin-2(1<i>H</i>)-one (<b>23p</b>)</h4><div class="NLM_p last">A stock solution of 5-(6-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (500 mg, 0.12 mmol) was prepared in 1,4-dioxane (5 mL), and an aliquot (0.5 mL) was added to a vial containing 3-(methylsulfonyl)pyrrolidine (18 mg, 0.12 mmol). To the vial was added further 1,4-dioxane (0.5 mL) along with JohnPhos (7.2 mg, 0.02 mmol), NaO<sup><i>t</i></sup>Bu (34.6 mg, 0.36 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (11.0 mg, 0.01 mmol). A stirrer bar was added, and the vial was sonicated and capped before heating in a microwave at 90 °C for 20 min. The reaction mixture was allowed to cool and then loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN). The cartridge was flushed with MeCN (2 mL) and MeOH (2 mL), and the solvent was removed under a stream of nitrogen. The sample was purified by MDAP (method B). The solvent was removed using a plate dryer to give the title compound (9 mg, 0.02 mmol, 14% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.06 (s, 1 H), 7.71 (s, 1 H), 7.44 (d, <i>J</i> = 8.5 Hz, 1 H), 6.77 (s, 1 H), 6.64 (d, <i>J</i> = 8.5 Hz, 1 H), 4.22 (d, <i>J</i> = 7.5 Hz, 2 H), 4.14 (quin, <i>J</i> = 7.0 Hz, 1 H), 3.72 (d, <i>J</i> = 11.5 Hz, 2 H), 3.68–3.62 (m, 2 H), 3.55 (s, 3 H), 3.53–3.47 (m, 2 H), 3.11 (t, <i>J</i> = 11.5 Hz, 2 H), 3.08 (s, 3 H), 2.44–2.37 (m, 2 H), 2.09 (s, 3 H), 1.98–1.88 (m, 1 H), 1.27–1.10 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 485 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.49 min; 99% purity.</div></div><div id="sec4_1_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> 5-(5-(1,1-Dioxidothiomorpholino)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>23q</b>)</h4><div class="NLM_p last">5-(5-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (100 mg, 0.24 mmol), thiomorpholine 1,1-dioxide (39 mg, 0.29 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (9 mg, 0.01 mmol), JohnPhos (9 mg, 0.03 mmol), and NaO<sup><i>t</i></sup>Bu (23 mg, 0.24 mmol) were suspended in toluene (2 mL) within a sealed microwave vial. The reaction mixture was heated in a microwave at 90 °C for 10 min. The reaction mixture was cooled to rt, filtered over Celite, and washed with EtOAc (50 mL). The solvent was removed in vacuo, and the sample was dissolved in 1:1 DMSO/MeOH (0.6 mL) and purified by MDAP (method B). The solvent was evaporated in vacuo to give the title compound (5 mg, 0.01 mmol, 4% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.73 (d, <i>J</i> = 2.5 Hz, 1 H), 7.50–7.47 (m, 1 H), 7.33–7.29 (m, 2 H), 7.01 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.11 (d, <i>J</i> = 7.5 Hz, 2 H), 3.91 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.85–3.79 (m, 4 H), 3.66 (s, 3 H), 3.26 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 3.22–3.17 (m, 4 H), 2.25 (s, 3 H), 2.08 (ttt, <i>J</i> = 11.5, 7.5, 3.5 Hz, 1 H), 1.39–1.23 (m, 4 H); LC–MS (method A) <i>m</i>/<i>z</i>: 471 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.73 min; 100% purity.</div></div><div id="sec4_1_86" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> 5-(6-(1,1-Dioxidothiomorpholino)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>23r</b>)</h4><div class="NLM_p last">5-(6-Bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (100 mg, 0.24 mmol), thiomorpholine 1,1-dioxide (32.5 mg, 0.24 mmol), NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol), JohnPhos (9.2 mg, 0.03 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (9.0 mg, 0.01 mmol) were suspended in 1,4-dioxane (2 mL). The reaction vessel was sealed and heated in a microwave at 90 °C for 20 min. After cooling the reaction to rt, further NaO<sup><i>t</i></sup>Bu (23.1 mg, 0.24 mmol), JohnPhos (9.2 mg, 0.03 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (9.0 mg, 0.01 mmol) were added to the reaction vessel. The reaction vessel was sealed and heated in a microwave at 90 °C for 20 min. After cooling the reaction to rt, the reaction mixture was loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [3 mL]). The cartridge was flushed with MeCN (3 mL), and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was dried under a stream of nitrogen to give the title compound (16 mg, 0.03 mmol, 14% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.07 (d, <i>J</i> = 2.5 Hz, 1 H), 7.72–7.70 (m, 1 H), 7.48 (d, <i>J</i> = 8.5 Hz, 1 H), 7.26 (d, <i>J</i> = 2.0 Hz, 1 H), 6.99 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.24 (d, <i>J</i> = 7.5 Hz, 2 H), 3.81–3.75 (m, 4 H), 3.72 (dd, <i>J</i> = 11.5, 3.5 Hz, 2 H), 3.55 (s, 3 H), 3.23–3.18 (m, 4 H), 3.12 (td, <i>J</i> = 11.5, 2.0 Hz, 2 H), 2.10 (s, 3 H), 1.99–1.86 (m, 1 H), 1.27–1.10 (m, 3 H); LC–MS (method F) <i>m</i>/<i>z</i>: 471 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.52 min; 98% purity.</div></div><div id="sec4_1_87" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> 5-(5-(3-Hydroxypyrrolidin-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>23s</b>)</h4><div class="NLM_p last">Into a vial were added 5-(5-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22a</b> (62 mg, 0.15 mmol), pyrrolidin-3-ol (20 mg, 0.23 mmol), RuPhos Pd G1 precatalyst (12 mg, 0.02 mmol), RuPhos (7.0 mg, 0.02 mmol), and LiHMDS (1 M in THF, 0.36 mL, 0.36 mmol) in 2-MeTHF (0.6 mL). A stirrer bar was added to the vial which was capped and stirred at 65 °C for 4 h. Pyrrolidin-3-ol (20 mg, 0.23 mmol), RuPhos Pd G1 precatalyst (12 mg, 0.02 mmol), RuPhos (7.0 mg, 0.02 mmol), and LiHMDS (1 M in THF, 0.36 mL, 0.36 mmol) were added, and the reaction was heated at 65 °C for 1 h. The sample was purified by MDAP (method B). The solvent was dried under a stream of nitrogen to give the title compound (4.9 mg, 0.01 mmol, 7% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.06 (d, <i>J</i> = 2.5 Hz, 1 H), 7.72–7.69 (m, 1 H), 7.45 (d, <i>J</i> = 8.5 Hz, 1 H), 6.63 (d, <i>J</i> = 2.0 Hz, 1 H), 6.60 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.92–4.89 (m, 1 H), 4.45–4.39 (m, 1 H), 4.18 (d, <i>J</i> = 7.5 Hz, 2 H), 3.71 (d, <i>J</i> = 11.5 Hz, 2 H), 3.55 (s, 3 H), 3.46 (dd, <i>J</i> = 10.0, 5.0 Hz, 1 H), 3.40–3.31 (m, 2 H), 3.41–3.05 (m, 3 H), 2.12–2.03 (m, 4 H), 2.00–1.85 (m, 2 H), 1.26–1.07 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 423 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.51 min; 100% purity.</div></div><div id="sec4_1_88" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> 5-(6-(3-Hydroxypyrrolidin-1-yl)-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>23t</b>)</h4><div class="NLM_p last">In a vial, a suspension of 5-(6-bromo-1-((tetrahydro-2<i>H</i>-pyran-4-yl)methyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>22b</b> (50 mg, 0.12 mmol), pyrrolidin-3-ol (17 mg, 0.20 mmol), NaO<sup><i>t</i></sup>Bu (25 mg, 0.26 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (11 mg, 0.01 mmol), and JohnPhos (8 mg, 0.03 mmol) in 1,4-dioxane (1 mL) was prepared. The vial was sealed and heated in a microwave at 90 °C for 20 min. The reaction mixture was allowed to cool and then loaded onto a 1 g C18 SPE cartridge (preconditioned with MeCN [2 mL]). The cartridge was flushed with MeCN (2 mL) and MeOH (2 mL). The washes were combined, and the solvent was removed under a stream of nitrogen. The sample was dissolved in DMSO (1 mL) and purified by MDAP (method B). The solvent was removed using a plate dryer to give the title compound (23 mg, 0.06 mmol, 46% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.04 (d, <i>J</i> = 2.5 Hz, 1 H), 7.71–7.69 (m, 1 H), 7.39 (d, <i>J</i> = 8.5 Hz, 1 H), 6.57 (d, <i>J</i> = 2.0 Hz, 1 H), 6.53 (dd, <i>J</i> = 8.5, 2.0 Hz, 1 H), 4.94 (d, <i>J</i> = 5.0 Hz, 1 H), 4.47–4.41 (m, 1 H), 4.18 (d, <i>J</i> = 7.5 Hz, 2 H), 3.72 (d, <i>J</i> = 11.5 Hz, 2 H), 3.54 (s, 3 H), 3.49 (dd, <i>J</i> = 10.0, 5.0 Hz, 1 H), 3.44–3.31 (m, 2 H), 3.19–3.07 (m, 3 H), 2.14–2.04 (m, 4 H), 2.00–1.88 (m, 2 H), 1.26–1.09 (m, 4 H); LC–MS (method F) <i>m</i>/<i>z</i>: 423 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.48 min; 100% purity.</div></div><div id="sec4_1_89" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> 5-(1-(1,3-Dimethoxypropan-2-yl)-5-morpholino-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>24</b>)</h4><div class="NLM_p">5-(5-Bromo-1-(1,3-dimethoxypropan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>26</b> (29.5 g, 70.2 mmol), DavePhos (1.381 g, 3.51 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (1.285 g, 1.404 mmol), morpholine (12.23 mL, 140 mmol), and 2-MeTHF (150 mL) were combined in a round bottom flask under nitrogen, and then NaO<sup><i>t</i></sup>Bu (2 M in THF, 105 mL, 211 mmol) was added. The mixture was heated at 80 °C for 2 h. The reaction mixture was diluted with brine (200 mL) and extracted with EtOAc (2 × 200 mL). The combined organics were dried and evaporated in vacuo to give a brown gum, which was triturated with EtOAc (200 mL) and then diluted with ether (100 mL), giving a fine, beige precipitate. This was collected by filtration and washed with ether to give the title compound (batch 1, 21.2 g).</div><div class="NLM_p">The filtrate was evaporated in vacuo to give a brown gum. This was dissolved in DCM (30 mL) and loaded onto a 340 g silica column. Elution was carried out using a gradient of 0–10% MeOH in DCM, and product-containing fractions were evaporated in vacuo. The resulting solid was triturated with ether (100 mL) and collected by filtration to give the title compound (batch 2, 6.5 g).</div><div class="NLM_p">5-(5-Bromo-1-(1,3-dimethoxypropan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>26</b> (33.2 g, 79.0 mmol), DavePhos (1.554 g, 3.95 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (1.447 g, 1.580 mmol), and 2-MeTHF (150 mL) were combined in a round bottom flask under nitrogen; then, NaO<sup><i>t</i></sup>Bu (2 M in THF, 118 mL, 237 mmol) was added, and the mixture was heated at 80 °C for 2 h. The reaction mixture was diluted with brine (200 mL) and extracted with EtOAc (2 × 200 mL). The combined organics were dried and evaporated in vacuo to give a brown gum, which was triturated with EtOAc (200 mL) and then diluted with ether (100 mL), giving a fine, beige precipitate. This was collected by filtration and washed with ether to give the title compound (batch 3, 31.4 g).</div><div class="NLM_p last">5-(5-Bromo-1-(1,3-dimethoxypropan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one <b>26</b> (20 g, 47.6 mmol), DavePhos (0.936 g, 2.379 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (0.871 g, 0.952 mmol), morpholine (8.29 mL, 95 mmol), and 2-MeTHF (150 mL) were combined in a round bottom flask under nitrogen; then, NaO<sup><i>t</i></sup>Bu (2 M in THF, 71.4 mL, 143 mmol) was added, and the mixture was heated at 80 °C for 2 h. The reaction mixture was diluted with brine (200 mL) and extracted with EtOAc (2 × 300 mL). The combined organics were dried and evaporated in vacuo to give a brown gum. This was combined with batches 1, 2, and 3 and dissolved in DCM (200 mL). The solution was loaded onto a 750 g silica column and eluted with EtOAc (5 column volumes) and then a gradient of 0–25% EtOH in EtOAc (20 column volumes). Product-containing fractions were evaporated in vacuo to give a pale yellow solid. This was dissolved in DCM (500 mL), SiliCycle thiourea silica resin (40 g) was added, and then the mixture was stirred at rt for 1 h. The suspension was filtered, and the filtrate evaporated in vacuo. The resulting foam was triturated with ether (300 mL), and the resulting solid was collected by filtration to give a pale yellow solid (60 g). This was suspended in EtOAc (400 mL) and heated to reflux with stirring for 1 h. The mixture was allowed to cool over 2 h and then further cooled using an ice bath and stirred for 1 h. The suspension was filtered, and the solid was washed with ether (200 mL) to give the title compound as an almost colorless solid (49.8 g, 117 mmol, 59% total yield). mp 174 °C; IR (solid) ν (cm<sup>–1</sup>): 2908, 1653, 1611, 1189, 1105, 968; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.00 (d, <i>J</i> = 2.5 Hz, 1 H), 7.66–7.64 (m, 1 H), 7.63 (d, <i>J</i> = 9.0 Hz, 1 H), 7.10 (d, <i>J</i> = 2.5 Hz, 1 H), 6.96 (dd, <i>J</i> = 9.0, 2.5 Hz, 1 H), 4.77 (tt, <i>J</i> = 9.0, 4.5 Hz, 1 H), 3.97 (dd, <i>J</i> = 10.5, 9.0 Hz, 2 H), 3.76–3.78 (m, 4 H), 3.75 (dd, <i>J</i> = 10.5, 4.5 Hz, 2 H), 3.53 (s, 3 H), 3.16 (s, 6 H), 3.10–3.07 (m, 4 H), 2.08 (s, 3 H); <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.6, 151.5, 147.4, 144.0, 138.8, 137.3, 128.0, 127.4, 113.4, 112.4, 108.1, 105.1, 69.8, 66.2, 58.3, 56.7, 50.3, 37.4, 16.9; LC–MS (method A) <i>m/z</i>: 427 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 0.81 min; 100% purity; HRMS: [(M + H)<sup>+</sup>] calcd for C<sub>23</sub>H<sub>31</sub>N<sub>4</sub>O<sub>4</sub>, 427.2340; found, 427.2340.</div></div><div id="sec4_1_90" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> 4-Bromo-<i>N-</i>(1,3-dimethoxypropan-2-yl)-2-nitroaniline (<b>25</b>)</h4><div class="NLM_p last">4-Bromo-1-fluoro-2-nitrobenzene (50 g, 227 mmol) and 1,3-dimethoxypropan-2-amine (32.5 g, 273 mmol) were dissolved in acetonitrile (300 mL), and K<sub>2</sub>CO<sub>3</sub> (47.1 g, 341 mmol) was added. The mixture was stirred at 80 °C for 6 h, then the mixture was allowed to cool and stood over the weekend at rt. The mixture was diluted with water (500 mL) and extracted with EtOAc (2 × 500 mL). The organic layer was washed with water (300 mL) and brine (300 mL), dried, and evaporated in vacuo to give the title compound as an orange solid (70 g, 220 mmol, 97% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.35 (d, <i>J</i> = 8.0 Hz, 1 H), 8.31 (d, <i>J</i> = 2.5 Hz, 1 H), 7.47 (dd, <i>J</i> = 9.0, 2.5 Hz, 1 H), 6.89 (d, <i>J</i> = 9.0 Hz, 1 H), 3.90–3.81 (m, 1 H), 3.62–3.53 (m, 4 H), 3.41 (s, 6 H); LC–MS (method A) <i>m/z</i>: 319, 321 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.25 min; 99% purity.</div></div><div id="sec4_1_91" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> 5-(5-Bromo-1-(1,3-dimethoxypropan-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)-1,3-dimethylpyridin-2(1<i>H</i>)-one (<b>26</b>)</h4><div class="NLM_p">4-Bromo-<i>N</i>-(1,3-dimethoxypropan-2-yl)-2-nitroaniline <b>25</b> (69 g, 216 mmol) was dissolved in EtOH (400 mL) with heating, and on cooling, the starting material crystallized out. 1,5-Dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (35.9 g, 238 mmol) was added to the suspension, followed by water (200 mL) and Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub> (94 g, 540 mmol), and the mixture was heated at 90 °C for 18 h. The mixture was evaporated to approximately half its original volume, then diluted with water (200 mL), and extracted with DCM (2 × 300 mL). The combined organics were washed with brine and then dried and evaporated in vacuo. The resulting yellow solid was triturated with EtOAc (200 mL), and the solid was collected by filtration. The solid was washed with ether (200 mL) and dried under vacuum to give the title compound (batch 1, 31 g, 34% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.90 (d, <i>J</i> = 2.0 Hz, 1 H), 7.87 (d, <i>J</i> = 2.0 Hz, 1 H), 7.68–7.66 (m, 1 H), 7.44 (d, <i>J</i> = 9.0 Hz, 1 H), 7.35 (dd, <i>J</i> = 9.0, 2.0 Hz, 1 H), 4.84 (tt, <i>J</i> = 8.0, 5.0 Hz, 1 H), 3.94 (dd, <i>J</i> = 10.0, 8.0 Hz, 2 H), 3.82 (dd, <i>J</i> = 10.0, 5.0 Hz, 2 H), 3.62 (s, 3 H), 3.28 (s, 6 H), 2.22 (s, 3 H); LC–MS (method A) <i>m/z</i>: 420, 422 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.00 min; 98% purity.</div><div class="NLM_p last">The filtrate was evaporated in vacuo, and the residue was dissolved in DCM (100 mL). The solution was loaded onto a 750 g silica column, and elution was carried out using a gradient of 0–10% MeOH in DCM. Product-containing fractions were evaporated in vacuo to give a yellow solid. This was dissolved in hot EtOAc (150 mL), allowed to cool, and left to stand overnight. The resulting solid was collected by filtration; the product was washed with ether (2 × 100 mL), stirring the product/ether mixture to ensure good trituration, and the product was then dried to give the title compound (batch 2, 20 g, 47.6 mmol, 22% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.06 (d, <i>J</i> = 2.0 Hz, 1 H), 7.82 (d, <i>J</i> = 2.0 Hz, 1 H), 7.80 (d, <i>J</i> = 9.0 Hz, 1 H), 7.68–7.66 (m, 1 H), 7.35 (dd, <i>J</i> = 9.0, 2.0 Hz, 1 H), 4.86 (tt, <i>J</i> = 9.0, 4.5 Hz, 1 H), 4.00 (dd, <i>J</i> = 10.5, 9.0 Hz, 2 H), 3.76 (dd, <i>J</i> = 10.5, 4.5 Hz, 2 H), 3.16 (s, 6 H), 2.09 (s, 3 H); LC–MS (method A) <i>m/z</i>: 420, 422 [(M + H)<sup>+</sup>]; <i>R</i><sub>t</sub>: 1.00 min; 100% purity.</div></div><div id="sec4_1_92" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> 4-((<i>Z</i>)-3-(6-{[5-(2-[(<i>S</i>)-6-(4-Chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl]acetamido)pentyl]amino}-6-oxohexyl)-2-{(2<i>E</i>,4<i>E</i>)-5-[3,3-dimethyl-5-sulfo-1-(4-sulfobutyl)-3<i>H</i>-indol-1-ium-2-yl]penta-2,4-dien-1-ylidene}-3-methyl-5-sulfoindolin-1-yl)butane-1-sulfonate (<b>27</b>)</h4><div class="NLM_p last">To a solution of <i>N</i>-(5-aminopentyl)-2-[(4<i>S</i>)-6-(4-chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-4-yl]acetamide<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> (1.7 mg, 3.53 μmol) in DMF (40 μL) was added a solution of Alexa Fluor 647-ONSu (2.16 mg, 1.966 μmol) in DMF (100 μL). The mixture was basified with DIPEA (1 μL, 5.73 μmol) and agitated overnight on a vortex mixer. The reaction mixture was evaporated to dryness. The solid was dissolved in a mixture of acetonitrile/water/acetic acid (5:4:1, <1 mL), filtered, and applied to a Phenomenex Jupiter C18 preparative column. Gradient elution was carried out at a flow rate of 10 mL/min using 0.1% TFA in water (mobile phase A) and 0.1% TFA in acetonitrile/water (9:1) (mobile phase B) 0–10 min 5% mobile phase B, 10–100 min 35% mobile phase B, 100–115 min 100% mobile phase B. The appropriate fractions were combined, evaporated to dryness, and triturated with dry ether, and the blue solid was dried overnight at <0.2 mbar to give the title compound (1.54 mg). HPLC analysis was conducted on a Spherisorb ODS2 column. Gradient elution was carried out using 0.1% TFA in water (mobile phase A) and 0.1% TFA in acetonitrile/water (9:1) (mobile phase B) 0–60 min 1–35% mobile phase B. Estimated purity >95%. Mass analysis was recorded on a Waters micromass ZQ mass spectrometer operating in the positive electrospray ionization mode giving ES<sup>+</sup> = 661.8 corresponding to [(M – C<sub>2</sub>H<sub>5</sub>/2) + H]<sup>+</sup>.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> BRD4 Protein Expression and Purification</h3><div class="NLM_p last">Recombinant human bromodomains BRD4 (1–477) (Y390A) (BD2 mutation to monitor binding to BD1) were expressed in <i>Escherichia coli</i> cells using a pET15b vector with a 6-His tag at the <i>N</i>-terminal. The His-tagged bromodomain pellet was resuspended in 50 mM 4-(2-hydroxyethyl)piperazine-1-ethanesulfonic acid (HEPES) (pH 7.5), 300 mM NaCl, 10 mM imidazole, and 1 μL/mL protease inhibitor cocktail and extracted from the <i>E. coli</i> cells using sonication. Purification was carried out using a nickel sepharose high-performance column, and the proteins were washed and then eluted with a linear gradient of 0–500 mM imidazole with buffer 50 mM HEPES (pH 7.5), 150 mM NaCl, and 500 mM imidazole, over 20 column volumes. Final purification was completed by using a Superdex 200 prep grade size exclusion column. Purified protein was stored at −80 °C in 20 mM HEPES (pH 7.5) and 100 mM NaCl. The protein identity was confirmed by peptide mass fingerprinting, and the predicted molecular weight was confirmed by mass spectrometry.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i117" class="anchor-spacer"></div><h3 class="article-section__title" id="_i117"> BRD4 TR-FRET Assay</h3><div class="NLM_p last">All assay components were diluted in a buffer composition of 50 mM HEPES (pH 7.4), 150 mM NaCl, 5% glycerol, 1 mM DTT, and 1 mM 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate. The final concentration of bromodomain protein was 10 nM. An Alexa Fluor 647 derivative of I-BET762 (<b>27</b>) was used as the competing ligand and was used at a concentration equal to the <i>K</i><sub>d</sub>. These components were premixed and 5 μL of this reaction mixture was added to wells containing 50 nL of various concentrations of the test compound or DMSO vehicle (0.5% DMSO final) in 384-well black low volume microtiter plates and incubated in the dark for 30 min at rt. The bromodomain protein/fluorescent ligand interaction was detected using TR-FRET following 5 μL addition of a 1.5 nM europium chelate-labeled anti-6His antibody in assay buffer. Time-resolved fluorescence (TRF) was then detected on a TRF laser equipped with a PerkinElmer EnVision multimode plate reader (excitation = 337 nm; emission 1 = 615 nm; emission 2 = 665 nm; dual wavelength bias dichroic = 400, 630 nm). The TR-FRET ratio was calculated using the following equation: ratio = ((acceptor fluorescence at 665 nm)/(donor fluorescence at 615 nm)) × 1000. TR-FRET ratio data were normalized to a mean of 16 replicates per microtiter plate of both 10 μM I-BET151 <b>4</b> and 1% DMSO controls, and IC<sub>50</sub> values were determined for each of the compounds tested by fitting the fluorescence ratio data to a four-parameter model: <i>y</i> = <i>a</i> + ((<i>b</i> – <i>a</i>)/(1 + (10<sup><i>x</i></sup>/10<sup><i>c</i></sup>)<sup><i>d</i></sup>)) where “<i>a</i>” is the minimum, “<i>b</i>” is the Hill slope, “<i>c</i>” is the IC<sub>50</sub>, and “<i>d</i>” is the maximum.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i118" class="anchor-spacer"></div><h3 class="article-section__title" id="_i118"> Cocrystallization and Structure Determination</h3><div class="NLM_p last">BRD4 BD1-dimethylpyridone benzimidazole <b>24</b> complex was cocrystallized with at least 3:1 excess of the compound over protein at 9.0 mg/mL in 120 + 120 nL sitting drops using a 96-well MRC plate. Crystals were grown with a well solution of 0.1 M PCTP pH 8.0, 25% v/v PEG1500 at 20 °C and cryoprotected using the well solution with 20% ethylene glycol prior to flash freezing in liquid nitrogen. Data from a single crystal were collected at 100 K on I03 at the Diamond Light Source at Oxford and processed to 1.30 Å using X-ray detector software (XDS)<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> and AIMLESS.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> A molecular replacement solution was determined using phaser and a previously determined in-house structure as a starting model. The <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> cell (α = β = γ = 90°, <i>a</i> = 39.493 Å, <i>b</i> = 49.533 Å, <i>c</i> = 58.2133 Å) has a single molecule in the ASU. Manual model building was performed using COOT<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> and refined using REFMAC<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> within the CCP4 software suite. There was clear difference density for the ligand in the acetylated lysine binding site, allowing the ligand to be unambiguously modeled. Statistics for the data collection and refined co-ordinates are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> Measurement of LPS-Induced MCP-1 Production from Human WB</h3><div class="NLM_p last">Blood was collected in a tube containing sodium heparin (10 units/mL). 96-well compound plates containing 1 μL test sample in 100% DMSO were prepared (two replicates to account for donor variability). WB (130 μL) was dispensed into each well and incubated for 30 min at 37 °C under a 5% CO<sub>2</sub> atmosphere. LPS (10 μL, <i>Salmonella typhosa</i>) made up in phosphate-buffered saline (PBS; 200 ng/mL final assay concentration) was added to each well of the compound plates. After further incubation for 18–24 h at 37 °C under a 5% CO<sub>2</sub> atmosphere, 140 μL of PBS was added to each well. The plates were sealed and centrifuged for 10 min at 1800 rpm at rt. The cell supernatant (20 μL) was placed in a 96-well meso scale discovery (MSD) plate precoated with a human MCP-1 capture antibody. The plates were sealed and placed on a shaker at 600 rpm for 1.5 h at rt. Anti-human MCP-1 antibody (20 μL) labeled with sulfo-TAG was added to each well of the MSD plate (stock 50× was diluted 1:50 with diluent 100, final assay concentration was 1 μg/mL). The plates were then resealed and shaken for 1.5 h at rt before washing with PBS Tween 0.05% (×3). 2× MSD Read Buffer T (150 μL, stock 4× MSD Read Buffer T was diluted 50:50 with deionized water) was then added to each well and the plates read on the MSD Sector Imager 6000. Concentration response curves for each compound were generated from the data and an IC<sub>50</sub> value was calculated. Human biological samples were sourced ethically and their research use was in accord with the terms of the informed consents under an Institutional Review Board (IRB)/Ethics Committee (EC)-approved protocol.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i120" class="anchor-spacer"></div><h3 class="article-section__title" id="_i120"> Measurement of LPS Induced IL-6 Production from Human WB</h3><div class="NLM_p last">Blood was collected in a tube containing sodium heparin (10 units of heparin/mL of blood). 96-well compound plates containing 1 μL of the test sample in 100% DMSO were prepared (two replicates to account for donor variability). WB (130 μL) was dispensed into each well of the 96-well compound plates and incubated for 30 min at 37 °C under a 5% CO<sub>2</sub> atmosphere. LPS (10 μL, <i>S. typhosa</i>) made up in PBS with 1% bovine serum albumin (200 ng/mL final assay concentration) was added to each well of the compound plates. The plates were then placed in the humidified primary cell incubator for 18–24 h at 37 °C under a 5% CO<sub>2</sub> atmosphere. PBS (140 μL) was added to all wells of the compound plates containing blood, and the plates were sealed and shaken on a plate shaker at 600 rpm for 2 min. The plates were then centrifuged for 10 min at 2500 rpm at rt. The cell supernatant (100 μL) was removed using a Bomec NX robot. MSD plates precoated with the human IL-6 capture antibody were blocked with MSD diluent 2 for 30 min on a plate shaker (600 rpm at rt). The supernatants were diluted 1:40 in PBS, and 25 μL of which was added to 96-well MSD IL-6 plates. The plates were sealed and placed on a shaker at 600 rpm for 1.5 h at rt. Anti-human IL-6 antibody (20 μL) labeled with sulfo-TAG was added to each well of the MSD plate (stock 50× was diluted 1:50 with diluent 3, final assay concentration is 1 μg/mL). The plates were then resealed and shaken for 1 h at rt before washing with PBS Tween 0.05% (×3). 2× MSD Read Buffer T (150 μL, stock 4× MSD Read Buffer T was diluted 50:50 with deionized water) was then added to each well, and the plates were read on the MSD Sector Imager 6000. Concentration response curves for each compound were generated from the data, and an IC<sub>50</sub> value was calculated. Human biological samples were sourced ethically, and their research use was in accord with the terms of the informed consents under an IRB/EC-approved protocol.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i121" class="anchor-spacer"></div><h3 class="article-section__title" id="_i121"> Mouse LPS Model</h3><div class="NLM_p last">Male CD1 mice (Charles River Laboratories, 5 per group) received an iv challenge of LPS (100 μg/kg, L3192 <i>E. coli</i> 0127:B8) 0.5 h after oral administration of dimethylpyridone benzimidazole <b>24</b> (in 1% (w/v) methylcellulose, aq 400). Serial blood samples were collected via tail vein up to 3 h or via cardiac puncture at 5 h (terminal sample) post LPS administration, and the serum harvested from the blood samples was frozen at −80 °C. On the day of analysis, the serum was thawed to rt and diluted 1:50 with assay diluent and levels of IL-6 were measured on MSD plates according to the manufacturer’s guidelines. Plates were read on an MSD Sector Imager 6000. The mean IL-6 AUC and mean unbound blood AUC values were generated using WinNonlin Phoenix version 6.3. All values statistically analyzed are represented as mean ± SD and are compared to the corresponding vehicle-treated group. Levels of significance were calculated by analysis of variance (ANOVA) followed by Dunnett’s multiple comparison <i>t</i>-test using GraphPad Prism version 5.04 (GraphPad Software, San Diego, CA). Statistical differences were determined as <i>p</i> < 0.05 (*), <i>p</i> < 0.001 (***) as compared to the vehicle-treated group. Animal studies were ethically reviewed and carried out in accordance with Animals (Scientific Procedures) Act 1986 and the GSK Policy on the Care, Welfare and Treatment of Laboratory Animals.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122"> T Cell-Dependent Immunization Model</h3><div class="NLM_p last">Male CD1 mice (Charles River Laboratories, 8 per group) received a single oral administration of dimethylpyridone benzimidazole <b>24</b> (in 1% (w/v) methylcellulose, aq 400) either QD or QOD over a 14 day dosing period. On day 1 of the study, each mouse received a single bolus ip administration of TNP-KLH (100 μg/kg) 0.5 h after oral administration of the compound. Serial blood samples were collected at 0.5 h post oral compound administration via tail vein on days 1, 4, 7, 9, and 11 or via cardiac puncture (terminal sample) on day 14, and the serum harvested from the blood samples was frozen at −80 °C. On the day of analysis, the serum was thawed to rt and levels of IgG1 were measured using a TNP ELISA and the plates read on a SpectraMax 190 spectrophotometer. <i>C</i><sub>av</sub> values were generated using WinNonlin Phoenix version 6.3. All values statistically analyzed are represented as mean ± SD and are compared to the corresponding vehicle-treated group. Levels of significance were calculated by ANOVA followed by Dunnett’s multiple comparison <i>t</i>-test using GraphPad Prism version 5.04 (GraphPad Software, San Diego, CA). Statistical differences were determined as <i>p</i> < 0.01 (**) as compared to the vehicle-treated group. Animal studies were ethically reviewed and carried out in accordance with Animals (Scientific Procedures) Act 1986 and the GSK Policy on the Care, Welfare and Treatment of Laboratory Animals.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i123"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01670" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10583" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10583" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01670?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01670</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Affinity screening of the encoded library method, BRD4 FP assay method, representative compound spectra, full assay data table, representative BCP selectivity data for dimethylpyridone benzimidazole <b>20a</b>, chromLog<i>D</i><sub>7.4</sub> method, solubility methods and data, X-ray powder diffraction (XRPD) method and data, mouse in vivo pharmacokinetic data for 2,6-dimethylphenol benzimidazole <b>10a</b>, dose prediction calculations, in vitro clearance data for 5-dimethylamine <b>22c</b> and 5-morpholine <b>23e</b>, bromodomain binding assay panel data, NanoBRET method and data, X-ray crystallization methods and data, and developability and cross screening panel data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data for final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf">jm9b01670_si_001.pdf (1.11 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_002.csv">jm9b01670_si_002.csv (5.89 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Cocrystal structures of BRD4 BD1 in the complex with compounds <b>10</b>, <b>20a</b>, and <b>24</b> and cocrystal structures of BRD2 BD1 in the complex with compounds <b>16</b> and <b>17</b> are deposited under the accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY">6TPY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPZ">6TPZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ1">6TQ1</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ2">6TQ2</a>, in the Protein Data Bank, respectively. Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01670" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10435" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10435" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher R. Wellaway</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0077-3452" title="Orcid link">http://orcid.org/0000-0003-0077-3452</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d0b3b8a2b9a3a4bfa0b8b5a2fea8fea7b5bcbcb1a7b1a990b7a3bbfeb3bfbd"><span class="__cf_email__" data-cfemail="93f0fbe1fae0e7fce3fbf6e1bdebbde4f6fffff2e4f2ead3f4e0f8bdf0fcfe">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominique Amans</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Bamborough</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9479-2894" title="Orcid link">http://orcid.org/0000-0001-9479-2894</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heather Barnett</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rino A. Bit</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jack A. Brown</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Neil R. Carlson</span> - <span class="hlFld-Affiliation affiliation">GSK, 200
Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony W. J. Cooper</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter D. Craggs</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert P. Davis</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tony W. Dean</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John P. Evans</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laurie Gordon</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Isobel L. Harada</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Hirst</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9969-2104" title="Orcid link">http://orcid.org/0000-0001-9969-2104</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip G. Humphreys</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8614-7155" title="Orcid link">http://orcid.org/0000-0002-8614-7155</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine L. Jones</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antonia J. Lewis</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew J. Lindon</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span>Present Address:
                        Research and Early Development, Respiratory, Inflammation and Autoimmunity (RIA) BioPharmaceuticals R&D, AstraZeneca, SE-431 83 Mölndal, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dave Lugo</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mahnoor Mahmood</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott McCleary</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patricia Medeiros</span> - <span class="hlFld-Affiliation affiliation">GSK, 200
Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren J. Mitchell</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael O’Sullivan</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Armelle Le Gall</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vipulkumar K. Patel</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chris Patten</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darren L. Poole</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rishi R. Shah</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5018-1730" title="Orcid link">http://orcid.org/0000-0002-5018-1730</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jane E. Smith</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kayleigh A. J. Stafford</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pamela J. Thomas</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mythily Vimal</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian D. Wall</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Watson</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natalie Wellaway</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gang Yao</span> - <span class="hlFld-Affiliation affiliation">GSK, 200
Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">GSK, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, U.K.</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i127">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72181" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72181" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful to Graham Inglis, Emmanuel Demont, and Cesar Molina for review and proof-reading of the manuscript. We thank Kate Dennis, Alan Ferrie, Iain Reid, Yang Zhuli and colleagues at WuXi, and GVK for supporting laboratory work.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BAZ2A</td><td class="NLM_def"><p class="first last">bromodomain adjacent to zinc finger domain 2A</p></td></tr><tr><td class="NLM_term">BCP</td><td class="NLM_def"><p class="first last">bromodomain-containing proteins</p></td></tr><tr><td class="NLM_term">BD1</td><td class="NLM_def"><p class="first last">first (<i>N</i>-terminal) bromodomain</p></td></tr><tr><td class="NLM_term">BD2</td><td class="NLM_def"><p class="first last">second (C-terminal) bromodomain</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extraterminal</p></td></tr><tr><td class="NLM_term">BOP</td><td class="NLM_def"><p class="first last">(benzotriazol-1-yloxy)tris(dimethylamino)phosphonium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">BRET</td><td class="NLM_def"><p class="first last">bioluminescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">CHAPS</td><td class="NLM_def"><p class="first last">3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate</p></td></tr><tr><td class="NLM_term">dba</td><td class="NLM_def"><p class="first last">dibenzylideneacetone</p></td></tr><tr><td class="NLM_term">DEL</td><td class="NLM_def"><p class="first last">DNA-encoded small-molecule library</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i><sup>1</sup>-((ethylimino)methylene)-<i>N</i><sup>3</sup>,<i>N</i><sup>3</sup>-dimethylpropane-1,3-diamine</p></td></tr><tr><td class="NLM_term">ELT</td><td class="NLM_def"><p class="first last">encoded library technology</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl) piperazine-1-ethanesulfonic acid</p></td></tr><tr><td class="NLM_term">Heps</td><td class="NLM_def"><p class="first last">hepatocytes</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-1-ol</p></td></tr><tr><td class="NLM_term">HPB</td><td class="NLM_def"><p class="first last">hydroxypropyl betacyclodextrin</p></td></tr><tr><td class="NLM_term">IL-6</td><td class="NLM_def"><p class="first last">interleukin 6</p></td></tr><tr><td class="NLM_term">IVC</td><td class="NLM_def"><p class="first last">in vitro clearance</p></td></tr><tr><td class="NLM_term">KAc</td><td class="NLM_def"><p class="first last"><i>N</i>-acetyl lysine</p></td></tr><tr><td class="NLM_term">MCP-1</td><td class="NLM_def"><p class="first last">monocyte chemoattractant protein 1</p></td></tr><tr><td class="NLM_term">MDAP</td><td class="NLM_def"><p class="first last">mass-directed autopreparative high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">2-MeTHF</td><td class="NLM_def"><p class="first last">2-methyl tetrahydrofuran</p></td></tr><tr><td class="NLM_term">Mics</td><td class="NLM_def"><p class="first last">microsomes</p></td></tr><tr><td class="NLM_term">MMP</td><td class="NLM_def"><p class="first last">matched molecular pair</p></td></tr><tr><td class="NLM_term">NMC</td><td class="NLM_def"><p class="first last">nuclear protein in testis midline carcinoma</p></td></tr><tr><td class="NLM_term">PFI</td><td class="NLM_def"><p class="first last">property forecast index</p></td></tr><tr><td class="NLM_term">PKPD</td><td class="NLM_def"><p class="first last">pharmacokinetic/pharmacodynamic</p></td></tr><tr><td class="NLM_term">PPA</td><td class="NLM_def"><p class="first last">polyphosphoric acid</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">PTM</td><td class="NLM_def"><p class="first last">post-translational modification</p></td></tr><tr><td class="NLM_term">TNP-KLH</td><td class="NLM_def"><p class="first last">trinitrophenyl-keyhole limpet hemocyanin</p></td></tr><tr><td class="NLM_term">TRF</td><td class="NLM_def"><p class="first last">time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">WB</td><td class="NLM_def"><p class="first last">whole blood</p></td></tr><tr><td class="NLM_term">XRPD</td><td class="NLM_def"><p class="first last">X-ray powder diffraction</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i129">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33741" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33741" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 83 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taverna, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span> <span> </span><span class="NLM_article-title">How chromatin-binding modules interpret histone modifications: lessons from professional pocket pickers</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1040</span>, <span class="refDoi"> DOI: 10.1038/nsmb1338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnsmb1338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=17984965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Kjs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1025-1040&author=S.+D.+Tavernaauthor=H.+Liauthor=A.+J.+Ruthenburgauthor=C.+D.+Allisauthor=D.+J.+Patel&title=How+chromatin-binding+modules+interpret+histone+modifications%3A+lessons+from+professional+pocket+pickers&doi=10.1038%2Fnsmb1338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">How chromatin-binding modules interpret histone modifications: lessons from professional pocket pickers</span></div><div class="casAuthors">Taverna, Sean D.; Li, Haitao; Ruthenburg, Alexander J.; Allis, C. David; Patel, Dinshaw J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1025-1040</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histones comprise the major protein component of chromatin, the scaffold in which the eukaryotic genome is packaged, and are subject to many types of post-translational modifications (PTMs), esp. on their flexible tails.  These modifications may constitute a 'histone code' and could be used to manage epigenetic information that helps extend the genetic message beyond DNA sequences.  This proposed code, read in part by histone PTM-binding 'effector' modules and their assocd. complexes, has been predicted to define unique functional states of chromatin and/or regulate various chromatin-templated processes.  A wealth of structural and functional data show how chromatin effector modules target their cognate covalent histone modifications.  Here, the authors summarize key features in the mol. recognition of histone PTMs by a diverse family of 'reader pockets' (including bromodomains, Royal superfamily modules, PHD fingers, WD40 protein repeats, 14-3-3 proteins, and BRCT domains), highlighting specific readout mechanisms for individual marks, common themes, and insights into the downstream functional consequences of the interactions.  Changes in these interactions may have far-reaching implications for human biol. and disease, notably cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroaoP5OiDUOLVg90H21EOLACvtfcHk0lhaSQkw9aRFGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Kjs7vE&md5=4ff842f1ef5416b932a89c751707defd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1338%26sid%3Dliteratum%253Aachs%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26atitle%3DHow%2520chromatin-binding%2520modules%2520interpret%2520histone%2520modifications%253A%2520lessons%2520from%2520professional%2520pocket%2520pickers%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1025%26epage%3D1040%26doi%3D10.1038%2Fnsmb1338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lhLrGApkqhyTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lhLrGApkqhyTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Small-molecule BET inhibitors in clinical and preclinical development and their therapeutic potential</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">776</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.2174/1568026615666150302110135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.2174%2F1568026615666150302110135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=25732788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXks1ynurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=776-794&author=L.+Yuauthor=Z.+Wangauthor=Z.+Zhangauthor=X.+Renauthor=X.+Luauthor=K.+Ding&title=Small-molecule+BET+inhibitors+in+clinical+and+preclinical+development+and+their+therapeutic+potential&doi=10.2174%2F1568026615666150302110135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule BET Inhibitors in Clinical and Preclinical Development and Their Therapeutic Potential</span></div><div class="casAuthors">Yu, Lei; Wang, Zhen; Zhang, Zhang; Ren, Xiaomei; Lu, Xiaoyun; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">776-794</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Lysine acetylation is a pivotal mechanism in chromatin processes and the regulation of gene transcription.  The acetylated lysine residues of histones are exclusively recognized by bromodomains (BRDs) known as epigenetic reader.  Proteins contg. BRDs undergo a post-translational modification (PTM) with development of cellular signaling and disease biol.  The bromo and extra-terminal (BET) proteins are the second subfamily, which play important roles in cellular proliferation, cell cycle progression and chromatin compaction.  Recently, a variety of small mols. have been reported to interact with the BET family proteins and accelerate the validation of BET proteins as druggable targets for treatment of cancers, inflammation and related diseases.  In this review, we will summarize the small-mol. inhibitors in clin. and preclin. studies of the BET family bromodomains and their medicinal implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk52FOUPlX8bVg90H21EOLACvtfcHk0lhLrGApkqhyTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXks1ynurs%253D&md5=802bf7b832d46c2b18ef6ac77eda1445</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F1568026615666150302110135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026615666150302110135%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DSmall-molecule%2520BET%2520inhibitors%2520in%2520clinical%2520and%2520preclinical%2520development%2520and%2520their%2520therapeutic%2520potential%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2015%26volume%3D15%26spage%3D776%26epage%3D794%26doi%3D10.2174%2F1568026615666150302110135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4 (BRD4)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4+%28BRD4%29&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lgSyPhwc5agHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujikawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e83190</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0083190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1371%2Fjournal.pone.0083190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=24391744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=K.+G.+McLureauthor=E.+M.+Gesnerauthor=L.+Tsujikawaauthor=O.+A.+Kharenkoauthor=S.+Attwellauthor=E.+Campeauauthor=S.+Wasiakauthor=A.+Steinauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=N.+C.+W.+Wongauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=P.+R.+Young&title=RVX-208%2C+an+inducer+of+ApoA-I+in+humans%2C+is+a+BET+bromodomain+antagonist&doi=10.1371%2Fjournal.pone.0083190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span></div><div class="casAuthors">McLure, Kevin G.; Gesner, Emily M.; Tsujikawa, Laura; Kharenko, Olesya A.; Attwell, Sarah; Campeau, Eric; Wasiak, Sylwia; Stein, Adam; White, Andre; Fontano, Eric; Suto, Robert K.; Wong, Norman C. W.; Wagner, Gregory S.; Hansen, Henrik C.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e83190/1-e83190/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increased synthesis of Apolipoprotein A-I (ApoA-I) and HDL is believed to provide a new approach to treating atherosclerosis through the stimulation of reverse cholesterol transport.  RVX-208 increases the prodn. of ApoA-I in hepatocytes in vitro and in vivo in monkeys and humans, which results in increased HDL-C, but the mol. target was not previously reported.  Using binding assays and X-ray crystallog., we now show that RVX-208 selectively binds to bromodomains of the BET (Bromodomain and Extra Terminal) family, competing for a site bound by the endogenous ligand, acetylated lysine and that this accounts for its pharmacol. activity. siRNA expts. further suggest that induction of ApoA-I mRNA is mediated by BET family member BRD4.  These data indicate that RVX-208 increases ApoA-I prodn. through an epigenetic mechanism and suggests that BET inhibition may be a promising new approach to the treatment of atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHPlAY0x6NLVg90H21EOLACvtfcHk0lgSyPhwc5agHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D&md5=519ef7ed6d06f97ef8a1359832c11d92</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083190%26sid%3Dliteratum%253Aachs%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DWasiak%26aufirst%3DS.%26aulast%3DStein%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DRVX-208%252C%2520an%2520inducer%2520of%2520ApoA-I%2520in%2520humans%252C%2520is%2520a%2520BET%2520bromodomain%2520antagonist%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0083190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lgSyPhwc5agHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gacias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matikainen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casaccia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Selective chemical modulation of gene transcription favors oligodendrocyte lineage progression</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.chembiol.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=24954007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=841-854&author=M.+Gaciasauthor=G.+Gerona-Navarroauthor=A.+N.+Plotnikovauthor=G.+Zhangauthor=L.+Zengauthor=J.+Kaurauthor=G.+Moyauthor=E.+Rusinovaauthor=Y.+Rodriguezauthor=B.+Matikainenauthor=A.+Vincekauthor=J.+Joshuaauthor=P.+Casacciaauthor=M.-M.+Zhou&title=Selective+chemical+modulation+of+gene+transcription+favors+oligodendrocyte+lineage+progression&doi=10.1016%2Fj.chembiol.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span></div><div class="casAuthors">Gacias, Mar; Gerona-Navarro, Guillermo; Plotnikov, Alexander N.; Zhang, Guangtao; Zeng, Lei; Kaur, Jasbir; Moy, Gregory; Rusinova, Elena; Rodriguez, Yoel; Matikainen, Bridget; Vincek, Adam; Joshua, Jennifer; Casaccia, Patrizia; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">841-854</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lysine acetylation regulates gene expression through modulating protein-protein interactions in chromatin.  Chem. inhibition of acetyl-lysine binding bromodomains of the major chromatin regulators BET (bromodomain and extraterminal domain) proteins has been shown to effectively block cell proliferation in cancer and inflammation.  However, whether selective inhibition of individual BET bromodomains has distinctive functional consequences remains only partially understood.  In this study, we show that selective chem. inhibition of the first bromodomain of BET proteins using our small-mol. inhibitor, Olinone, accelerated the progression of mouse primary oligodendrocyte progenitors toward differentiation, whereas inhibition of both bromodomains of BET proteins hindered differentiation.  This effect was target specific, as it was not detected in cells treated with inactive analogs and independent of any effect on proliferation.  Therefore, selective chem. modulation of individual bromodomains, rather than use of broad-based inhibitors, may enhance regenerative strategies in disorders characterized by myelin loss such as aging and neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1fCvOR18yE7Vg90H21EOLACvtfcHk0lgKgoxR4SfqyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK&md5=6b377663a8d0d07441ab4f8a5d8191b3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DGacias%26aufirst%3DM.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DKaur%26aufirst%3DJ.%26aulast%3DMoy%26aufirst%3DG.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DY.%26aulast%3DMatikainen%26aufirst%3DB.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DCasaccia%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DSelective%2520chemical%2520modulation%2520of%2520gene%2520transcription%2520favors%2520oligodendrocyte%2520lineage%2520progression%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D841%26epage%3D854%26doi%3D10.1016%2Fj.chembiol.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>– <span class="NLM_lpage">4334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0lgKgoxR4SfqyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1655</span>– <span class="NLM_lpage">1680</span>, <span class="refDoi"> DOI: 10.4155/fmc-2016-0059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.4155%2Ffmc-2016-0059" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27193077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKhsr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1655-1680&author=C.+Galdeanoauthor=A.+Ciulli&title=Selectivity+on-target+of+bromodomain+chemical+probes+by+structure-guided+medicinal+chemistry+and+chemical+biology&doi=10.4155%2Ffmc-2016-0059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology</span></div><div class="casAuthors">Galdeano, Carles; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1655-1680</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Targeting epigenetic proteins is a rapidly growing area for medicinal chem. and drug discovery.  Recent years have seen an explosion of interest in developing small mols. binding to bromodomains, the readers of acetyl-lysine modifications.  A plethora of co-crystal structures has motivated focused fragment-based design and optimization programs within both industry and academia.  These efforts have yielded several compds. entering the clinic, and many more are increasingly being used as chem. probes to interrogate bromodomain biol.  High selectivity of chem. probes is necessary to ensure biol. activity is due to an on-target effect.  Here, we review the state-of-the-art of bromodomain-targeting compds., focusing on the structural basis for their on-target selectivity or lack thereof.  We also highlight chem. biol. approaches to enhance on-target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1bg8K3kh-RLVg90H21EOLACvtfcHk0lgKgoxR4SfqyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKhsr3K&md5=a4c02dd517e8582755a918eabd7bfeea</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0059%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelectivity%2520on-target%2520of%2520bromodomain%2520chemical%2520probes%2520by%2520structure-guided%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2016%26volume%3D8%26spage%3D1655%26epage%3D1680%26doi%3D10.4155%2Ffmc-2016-0059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voitovich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derviaux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rebuffet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinquet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemot, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunel, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collette, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morelli, X.</span></span> <span> </span><span class="NLM_article-title">Exploring selective inhibition of the first bromodomain of the human bromodomain and extra-terminal domain (BET) proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1634</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1634-1641&author=B.+Rauxauthor=Y.+Voitovichauthor=C.+Derviauxauthor=A.+Lugariauthor=E.+Rebuffetauthor=S.+Milhasauthor=S.+Prietauthor=T.+Rouxauthor=E.+Trinquetauthor=J.-C.+Guillemotauthor=S.+Knappauthor=J.-M.+Brunelauthor=A.+Y.+Fedorovauthor=Y.+Colletteauthor=P.+Rocheauthor=S.+Betziauthor=S.+Combesauthor=X.+Morelli&title=Exploring+selective+inhibition+of+the+first+bromodomain+of+the+human+bromodomain+and+extra-terminal+domain+%28BET%29+proteins&doi=10.1021%2Facs.jmedchem.5b01708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-terminal Domain (BET) Proteins</span></div><div class="casAuthors">Raux, Brigitt; Voitovich, Yuliia; Derviaux, Carine; Lugari, Adrien; Rebuffet, Etienne; Milhas, Sabine; Priet, Stephane; Roux, Thomas; Trinquet, Eric; Guillemot, Jean-Claude; Knapp, Stefan; Brunel, Jean-Michel; Fedorov, Alexey Yu.; Collette, Yves; Roche, Philippe; Betzi, Stephane; Combes, Sebastien; Morelli, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1634-1641</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A midthroughput screening follow-up program targeting the first bromodomain of the human BRD4 protein, BRD4(BD1), identified an acetylated-mimic xanthine deriv. inhibitor.  This compd. binds with an affinity in the low micromolar range yet exerts suitable unexpected selectivity in vitro against the other members of the bromodomain and extra-terminal domain (BET) family.  A structure-based program pinpointed a role of the ZA loop, paving the way for the development of potent and selective BET-BRDi probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouQdXvF3r4zLVg90H21EOLACvtfcHk0lj6FQZTle2U5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D&md5=fb9cfc20c76f65fcf409370b60aecc0a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01708%26sid%3Dliteratum%253Aachs%26aulast%3DRaux%26aufirst%3DB.%26aulast%3DVoitovich%26aufirst%3DY.%26aulast%3DDerviaux%26aufirst%3DC.%26aulast%3DLugari%26aufirst%3DA.%26aulast%3DRebuffet%26aufirst%3DE.%26aulast%3DMilhas%26aufirst%3DS.%26aulast%3DPriet%26aufirst%3DS.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DGuillemot%26aufirst%3DJ.-C.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrunel%26aufirst%3DJ.-M.%26aulast%3DFedorov%26aufirst%3DA.%2BY.%26aulast%3DCollette%26aufirst%3DY.%26aulast%3DRoche%26aufirst%3DP.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DCombes%26aufirst%3DS.%26aulast%3DMorelli%26aufirst%3DX.%26atitle%3DExploring%2520selective%2520inhibition%2520of%2520the%2520first%2520bromodomain%2520of%2520the%2520human%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1634%26epage%3D1641%26doi%3D10.1021%2Facs.jmedchem.5b01708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deangelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of bivalent BET inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1096</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnchembio.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27775715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1089-1096&author=M.+Tanakaauthor=J.+M.+Robertsauthor=H.-S.+Seoauthor=A.+Souzaauthor=J.+Paulkauthor=T.+G.+Scottauthor=S.+L.+Deangeloauthor=S.+Dhe-paganonauthor=J.+E.+Bradner&title=Design+and+characterization+of+bivalent+BET+inhibitors&doi=10.1038%2Fnchembio.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of bivalent BET inhibitors</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Seo, Hyuk-Soo; Souza, Amanda; Paulk, Joshiawa; Scott, Thomas G.; DeAngelo, Stephen L.; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1089-1096</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cellular signaling is often propagated by multivalent interactions.  Multivalency creates avidity, allowing stable biophys. recognition.  Multivalency is an attractive strategy for achieving potent binding to protein targets, as the affinity of bivalent ligands is often greater than the sum of monovalent affinities.  The bromodomain and extraterminal domain (BET) family of transcriptional coactivators features tandem bromodomains through which BET proteins bind acetylated histones and transcription factors.  All reported antagonists of the BET protein BRD4 bind in a monovalent fashion.  Here the authors describe, to the knowledge for the first time, a bivalent BET bromodomain inhibitor-MT1-which has unprecedented potency.  Biophys. and biochem. studies suggest MT1 is an intramol. bivalent BRD4 binder that is more than 100-fold more potent, in cellular assays, than the corresponding monovalent antagonist, JQ1.  MT1 significantly (P < 0.05) delayed leukemia progression in mice, as compared to JQ1.  These data qualify a powerful chem. probe for BET bromodomains and a rationale for further development of multidomain inhibitors of epigenetic reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQwc6X0VYYbVg90H21EOLACvtfcHk0lj6FQZTle2U5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurrM&md5=7319203dac409d5dee3c580b49fcf185</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2209%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DDeangelo%26aufirst%3DS.%2BL.%26aulast%3DDhe-paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520bivalent%2520BET%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D1089%26epage%3D1096%26doi%3D10.1038%2Fnchembio.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bobby, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flavell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jowitt, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikhney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlister, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svergun, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span> <span> </span><span class="NLM_article-title">Potent and selective bivalent inhibitors of BET bromodomains</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1104</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnchembio.2210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27775716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1097-1104&author=M.+J.+Waringauthor=H.+Chenauthor=A.+A.+Rabowauthor=G.+Walkerauthor=R.+Bobbyauthor=S.+Boikoauthor=R.+H.+Bradburyauthor=R.+Callisauthor=E.+Clarkauthor=I.+Daleauthor=D.+L.+Danielsauthor=A.+Dulakauthor=L.+Flavellauthor=G.+Holdgateauthor=T.+A.+Jowittauthor=A.+Kikhneyauthor=M.+McAlisterauthor=J.+M%C3%A9ndezauthor=D.+Oggauthor=J.+Patelauthor=P.+Petterutiauthor=G.+R.+Robbauthor=M.+B.+Robersauthor=S.+Saifauthor=N.+Strattonauthor=D.+I.+Svergunauthor=W.+Wangauthor=D.+Whittakerauthor=D.+M.+Wilsonauthor=Y.+Yao&title=Potent+and+selective+bivalent+inhibitors+of+BET+bromodomains&doi=10.1038%2Fnchembio.2210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective bivalent inhibitors of BET bromodomains</span></div><div class="casAuthors">Waring, Michael J.; Chen, Huawei; Rabow, Alfred A.; Walker, Graeme; Bobby, Romel; Boiko, Scott; Bradbury, Rob H.; Callis, Rowena; Clark, Edwin; Dale, Ian; Daniels, Danette L.; Dulak, Austin; Flavell, Liz; Holdgate, Geoff; Jowitt, Thomas A.; Kikhney, Alexey; McAlister, Mark; Mendez, Jacqui; Ogg, Derek; Patel, Joe; Petteruti, Philip; Robb, Graeme R.; Robers, Matthew B.; Saif, Sakina; Stratton, Natalie; Svergun, Dmitri I.; Wang, Wenxian; Whittaker, David; Wilson, David M.; Yao, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1097-1104</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extraterminal (BET) family, in particular bromodomain-contg. protein 4 (BRD4), are of great interest as biol. targets.  BET proteins contain two sep. bromodomains, and existing inhibitors bind to them monovalently.  Here the authors describe the discovery and characterization of probe compd. biBET, capable of engaging both bromodomains simultaneously in a bivalent, in cis binding mode.  The evidence provided here was obtained in a variety of biophys. and cellular expts.  The bivalent binding results in very high cellular potency for BRD4 binding and pharmacol. responses such as disruption of BRD4-mediator complex subunit 1 foci with an EC50 of 100 pM.  These compds. will be of considerable utility as BET/BRD4 chem. probes.  This work illustrates a novel concept in ligand design-simultaneous targeting of two sep. domains with a drug-like small mol.-providing precedent for a potentially more effective paradigm for developing ligands for other multidomain proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq-VaDebQI0rVg90H21EOLACvtfcHk0lj6FQZTle2U5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurzE&md5=fabaccbac93338af32292ad8907ade0c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2210%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DBobby%26aufirst%3DR.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DCallis%26aufirst%3DR.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DDale%26aufirst%3DI.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DFlavell%26aufirst%3DL.%26aulast%3DHoldgate%26aufirst%3DG.%26aulast%3DJowitt%26aufirst%3DT.%2BA.%26aulast%3DKikhney%26aufirst%3DA.%26aulast%3DMcAlister%26aufirst%3DM.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DOgg%26aufirst%3DD.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DSaif%26aufirst%3DS.%26aulast%3DStratton%26aufirst%3DN.%26aulast%3DSvergun%26aufirst%3DD.%2BI.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DYao%26aufirst%3DY.%26atitle%3DPotent%2520and%2520selective%2520bivalent%2520inhibitors%2520of%2520BET%2520bromodomains%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D1097%26epage%3D1104%26doi%3D10.1038%2Fnchembio.2210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castriotta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahdesmaki, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2563</span>– <span class="NLM_lpage">2574</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-16-0141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1158%2F1535-7163.mct-16-0141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27573426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2563-2574&author=G.+W.+Rhyasenauthor=M.+M.+Hattersleyauthor=Y.+Yaoauthor=A.+Dulakauthor=W.+Wangauthor=P.+Petterutiauthor=I.+L.+Daleauthor=S.+Boikoauthor=T.+Cheungauthor=J.+Zhangauthor=S.+Wenauthor=L.+Castriottaauthor=D.+Lawsonauthor=M.+Collinsauthor=L.+Baoauthor=M.+J.+Ahdesmakiauthor=G.+Walkerauthor=G.+O%E2%80%99Connorauthor=T.+C.+Yehauthor=A.+A.+Rabowauthor=J.+R.+Dryauthor=C.+Reimerauthor=P.+Lyneauthor=G.+B.+Millsauthor=S.+E.+Fawellauthor=M.+J.+Waringauthor=M.+Zindaauthor=E.+Clarkauthor=H.+Chen&title=AZD5153%3A+a+novel+bivalent+BET+bromodomain+inhibitor+highly+active+against+hematologic+malignancies&doi=10.1158%2F1535-7163.mct-16-0141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span></div><div class="casAuthors">Rhyasen, Garrett W.; Hattersley, Maureen M.; Yao, Yi; Dulak, Austin; Wang, Wenxian; Petteruti, Philip; Dale, Ian L.; Boiko, Scott; Cheung, Tony; Zhang, Jingwen; Wen, Shenghua; Castriotta, Lillian; Lawson, Deborah; Collins, Michael; Bao, Larry; Ahdesmaki, Miika J.; Walker, Graeme; O'Connor, Greg; Yeh, Tammie C.; Rabow, Alfred A.; Dry, Jonathan R.; Reimer, Corinne; Lyne, Paul; Mills, Gordon B.; Fawell, Stephen E.; Waring, Michael J.; Zinda, Michael; Clark, Edwin; Chen, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2574</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD4 regulates gene expression via recruitment of transcriptional regulatory complexes to acetylated chromatin.  Pharmacol. targeting of BRD4 bromodomains by small mol. inhibitors has proven to be an effective means to disrupt aberrant transcriptional programs crit. for tumor growth and/or survival.  Herein, we report AZD5153, a potent, selective, and orally available BET/BRD4 bromodomain inhibitor possessing a bivalent binding mode.  Unlike previously described monovalent inhibitors, AZD5153 ligates two bromodomains in BRD4 simultaneously.  The enhanced avidity afforded through bivalent binding translates into increased cellular and antitumor activity in preclin. hematol. tumor models.  In vivo administration of AZD5153 led to tumor stasis or regression in multiple xenograft models of acute myeloid leukemia, multiple myeloma, and diffuse large B-cell lymphoma.  The relationship between AZD5153 exposure and efficacy suggests that prolonged BRD4 target coverage is a primary efficacy driver.  AZD5153 treatment markedly affects transcriptional programs of MYC, E2F, and mTOR.  Of note, mTOR pathway modulation is assocd. with cell line sensitivity to AZD5153.  Transcriptional modulation of MYC and HEXIM1 was confirmed in AZD5153-treated human whole blood, thus supporting their use as clin. pharmacodynamic biomarkers.  This study establishes AZD5153 as a highly potent, orally available BET/BRD4 inhibitor and provides a rationale for clin. development in hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiKYOFhro1bVg90H21EOLACvtfcHk0li9D7TmHLDCsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D&md5=8d0ac899d081705aaa12d49fd90165a3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-16-0141%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DDale%26aufirst%3DI.%2BL.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCastriotta%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DBao%26aufirst%3DL.%26aulast%3DAhdesmaki%26aufirst%3DM.%2BJ.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DLyne%26aufirst%3DP.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DAZD5153%253A%2520a%2520novel%2520bivalent%2520BET%2520bromodomain%2520inhibitor%2520highly%2520active%2520against%2520hematologic%2520malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2563%26epage%3D2574%26doi%3D10.1158%2F1535-7163.mct-16-0141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0li9D7TmHLDCsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lhEfD1jniQt3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lhEfD1jniQt3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.-Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lhEfD1jniQt3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Impact of target warhead and linkage vector on inducing protein degradation: comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=504-513&author=K.-H.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=Impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Target Warhead and Linkage Vector on Inducing Protein Degradation: Comparison of Bromodomain and Extra-Terminal (BET) Degraders Derived from Triazolodiazepine (JQ1) and Tetrahydroquinoline (I-BET726) BET Inhibitor Scaffolds</span></div><div class="casAuthors">Chan, Kwok-Ho; Zengerle, Michael; Testa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">504-513</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of proteolysis-targeting chimeras (PROTACs) is a powerful small-mol. approach for inducing protein degrdn.  PROTACs conjugate a target warhead to an E3 ubiquitin ligase ligand via a linker.  Here the authors examd. the impact of derivatizing two different BET bromodomain inhibitors, triazolodiazepine JQ1 and the more potent tetrahydroquinoline I-BET726, via distinct exit vectors, using different polyethylene glycol linkers to VHL ligand VH032.  Triazolodiazepine PROTACs exhibited pos. cooperativities of ternary complex formation and were more potent degraders than tetrahydroquinoline compds., which showed neg. cooperativities instead.  Marked dependency on linker length was obsd. for BET-degrading and cMyc-driven antiproliferative activities in acute myeloid leukemia cell lines.  This work exemplifies as a cautionary tale how a more potent inhibitor does not necessarily generate more potent PROTACs and underscores the key roles played by the conjugation.  The provided insights and framework for structure-activity relationships of bivalent degraders are anticipated to have wide future applicability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvEL41TefwTbVg90H21EOLACvtfcHk0lg9IfRLIFCTrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D&md5=c3bfd9377d8e7805ba962cd0180431af</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DImpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D504%26epage%3D513%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calosing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitchen, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Horst, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of novel covalent bromodomain and extra-terminal domain (BET) inhibitors targeting a methionine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8202</span>– <span class="NLM_lpage">8211</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8202-8211&author=O.+A.+Kharenkoauthor=R.+G.+Patelauthor=S.+D.+Brownauthor=C.+Calosingauthor=A.+Whiteauthor=D.+Lakshminarasimhanauthor=R.+K.+Sutoauthor=B.+C.+Duffyauthor=D.+B.+Kitchenauthor=K.+G.+McLureauthor=H.+C.+Hansenauthor=E.+H.+van+der+Horstauthor=P.+R.+Young&title=Design+and+characterization+of+novel+covalent+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+targeting+a+methionine&doi=10.1021%2Facs.jmedchem.8b00666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Novel Covalent Bromodomain and Extra-Terminal Domain (BET) Inhibitors Targeting a Methionine</span></div><div class="casAuthors">Kharenko, Olesya A.; Patel, Reena G.; Brown, S. David; Calosing, Cyrus; White, Andre; Lakshminarasimhan, Damodharan; Suto, Robert K.; Duffy, Bryan C.; Kitchen, Douglas B.; McLure, Kevin G.; Hansen, Henrik C.; van der Horst, Edward H.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8202-8211</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET proteins are key epigenetic regulators that regulate transcription through binding to acetylated lysine (AcLys) residues of histones and transcription factors through bromodomains (BDs).  The disruption of this interaction with small mol. bromodomain inhibitors is a promising approach to treat various diseases including cancer, autoimmune and cardiovascular diseases.  Covalent inhibitors can potentially offer a more durable target inhibition leading to improved in vivo pharmacol.  Here the authors describe the design of covalent inhibitors of BRD4(BD1) that target a methionine in the binding pocket by attaching an epoxide warhead to a suitably oriented noncovalent inhibitor.  Using thermal denaturation, MALDI-TOF mass spectrometry, and an x-ray crystal structure, the authors demonstrate that these inhibitors selectively form a covalent bond with Met 149 in BRD4(BD1) but not other bromodomains and provide durable transcriptional and antiproliferative activity in cell based assays.  Covalent targeting of methionine offers a novel approach to drug discovery for BET proteins and other targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHccGRxf5ShrVg90H21EOLACvtfcHk0lg9IfRLIFCTrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO&md5=1c3ae393a5f8a8c12ffb16e6f3798118</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00666%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DBrown%26aufirst%3DS.%2BD.%26aulast%3DCalosing%26aufirst%3DC.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DDuffy%26aufirst%3DB.%2BC.%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%2BH.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520novel%2520covalent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520targeting%2520a%2520methionine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8202%26epage%3D8211%26doi%3D10.1021%2Facs.jmedchem.8b00666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lgAYZ5ExV16FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lgAYZ5ExV16FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgAYZ5ExV16FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matzuk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agno, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibition of BRDT for male contraception</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.06.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.cell.2012.06.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22901802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=673-684&author=M.+M.+Matzukauthor=M.+R.+McKeownauthor=P.+Filippakopoulosauthor=Q.+Liauthor=L.+Maauthor=J.+E.+Agnoauthor=M.+E.+Lemieuxauthor=S.+Picaudauthor=R.+N.+Yuauthor=J.+Qiauthor=S.+Knappauthor=J.+E.+Bradner&title=Small-molecule+inhibition+of+BRDT+for+male+contraception&doi=10.1016%2Fj.cell.2012.06.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibition of BRDT for Male Contraception</span></div><div class="casAuthors">Matzuk, Martin M.; McKeown, Michael R.; Filippakopoulos, Panagis; Li, Qinglei; Ma, Lang; Agno, Julio E.; Lemieux, Madeleine E.; Picaud, Sarah; Yu, Richard N.; Qi, Jun; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">673-684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A pharmacol. approach to male contraception remains a longstanding challenge in medicine.  Toward this objective, the authors explored the spermatogenic effects of a selective small-mol. inhibitor (JQ1) of the bromodomain and extraterminal (BET) subfamily of epigenetic reader proteins.  Here, the authors report potent inhibition of the testis-specific member BRDT, which is essential for chromatin remodeling during spermatogenesis.  Biochem. and crystallog. studies confirm that occupancy of the BRDT acetyl-lysine binding pocket by JQ1 prevents recognition of acetylated histone H4.  Treatment of mice with JQ1 reduced seminiferous tubule area, testis size, and spermatozoa no. and motility without affecting hormone levels.  Although JQ1-treated males mate normally, inhibitory effects of JQ1 evident at the spermatocyte and round spermatid stages cause a complete and reversible contraceptive effect.  These data establish a new contraceptive that can cross the blood:testis boundary and inhibit bromodomain activity during spermatogenesis, providing a lead compd. targeting the male germ cell for contraception.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxVWUgXTC77Vg90H21EOLACvtfcHk0lgvbLzHOuTLmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO&md5=ccfa1b443c156612714e4b7f88590041</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.045%26sid%3Dliteratum%253Aachs%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DAgno%26aufirst%3DJ.%2BE.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%2BN.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520BRDT%2520for%2520male%2520contraception%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D673%26epage%3D684%26doi%3D10.1016%2Fj.cell.2012.06.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deguzman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apparsundaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMartino, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogaboam, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denton, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budd, D. C.</span></span> <span> </span><span class="NLM_article-title">Assessment of Brd4 inhibition in idiopathic pulmonary fibrosis lung fibroblasts and in vivo models of lung fibrosis</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">470</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1016/j.ajpath.2013.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.ajpath.2013.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23759512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFynur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2013&pages=470-479&author=X.+Tangauthor=R.+Pengauthor=J.+E.+Phillipsauthor=J.+Deguzmanauthor=Y.+Renauthor=S.+Apparsundaramauthor=Q.+Luoauthor=C.+M.+Bauerauthor=M.+E.+Fuentesauthor=J.+A.+DeMartinoauthor=G.+Tyagiauthor=R.+Garridoauthor=C.+M.+Hogaboamauthor=C.+P.+Dentonauthor=A.+M.+Holmesauthor=C.+Kitsonauthor=C.+S.+Stevensonauthor=D.+C.+Budd&title=Assessment+of+Brd4+inhibition+in+idiopathic+pulmonary+fibrosis+lung+fibroblasts+and+in+vivo+models+of+lung+fibrosis&doi=10.1016%2Fj.ajpath.2013.04.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Brd4 Inhibition in Idiopathic Pulmonary Fibrosis Lung Fibroblasts and in Vivo Models of Lung Fibrosis</span></div><div class="casAuthors">Tang, Xiaoyan; Peng, Ruoqi; Phillips, Jonathan E.; Deguzman, Jeremy; Ren, Yonglin; Apparsundaram, Subramanium; Luo, Qi; Bauer, Carla M.; Fuentes, Maria E.; DeMartino, Julie A.; Tyagi, Gaurav; Garrido, Rosario; Hogaboam, Cory M.; Denton, Christopher P.; Holmes, Alan M.; Kitson, Christopher; Stevenson, Christopher S.; Budd, David C.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">470-479</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Idiopathic pulmonary fibrosis (IPF) is a chronic lung disease of high unmet medical need.  Although bromodomain (Brd) and extra terminal domain isoforms have recently been implicated in mediating inflammatory and oncol. indications, their roles in lung fibrosis have not been comprehensively assessed.  We investigated the role of Brd on the profibrotic responses of lung fibroblasts (LFs) in patients with rapidly progressing IPF and a mouse bleomycin model of lung fibrosis.  The enhanced migration, proliferation, and IL-6 release obsd. in LFs from patients with rapidly progressing IPF are attenuated by pharmacol. inhibition of Brd4.  These changes are accompanied by enhanced histone H4 lysine5 acetylation and assocn. of Brd4 with genes involved in the profibrotic responses in IPF LFs as demonstrated using chromatin immunopptn. and quant. PCR.  Oral administration of 200 mg/kg per day Brd4 inhibitor JQ1 in a therapeutic dosing regimen substantially attenuated lung fibrosis induced by bleomycin in C57BL/6 mice.  In conclusion, this study shows that the Brd4 inhibitor JQ1, administered in a therapeutic dosage, is capable of inhibiting the profibrotic effects of IPF LFs and attenuates bleomycin-induced lung fibrosis in mice.  These results suggest that Brd4 inhibitors may represent a novel therapy for the treatment of rapidly progressing IPF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeyA6Tfi6PObVg90H21EOLACvtfcHk0lgvbLzHOuTLmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFynur7I&md5=8a7b094630ce44df6cd1b2b9de6cc6ec</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2013.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2013.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DJ.%2BE.%26aulast%3DDeguzman%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DApparsundaram%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DBauer%26aufirst%3DC.%2BM.%26aulast%3DFuentes%26aufirst%3DM.%2BE.%26aulast%3DDeMartino%26aufirst%3DJ.%2BA.%26aulast%3DTyagi%26aufirst%3DG.%26aulast%3DGarrido%26aufirst%3DR.%26aulast%3DHogaboam%26aufirst%3DC.%2BM.%26aulast%3DDenton%26aufirst%3DC.%2BP.%26aulast%3DHolmes%26aufirst%3DA.%2BM.%26aulast%3DKitson%26aufirst%3DC.%26aulast%3DStevenson%26aufirst%3DC.%2BS.%26aulast%3DBudd%26aufirst%3DD.%2BC.%26atitle%3DAssessment%2520of%2520Brd4%2520inhibition%2520in%2520idiopathic%2520pulmonary%2520fibrosis%2520lung%2520fibroblasts%2520and%2520in%2520vivo%2520models%2520of%2520lung%2520fibrosis%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2013%26volume%3D183%26spage%3D470%26epage%3D479%26doi%3D10.1016%2Fj.ajpath.2013.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apparsundaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deguzman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demartino, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budd, D. C.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain proteins mediate downstream signaling events following growth factor stimulation in human lung fibroblasts and are involved in bleomycin-induced pulmonary fibrosis</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1124/mol.112.081661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1124%2Fmol.112.081661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23115324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFKitLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2013&pages=283-293&author=X.+Tangauthor=R.+Pengauthor=Y.+Renauthor=S.+Apparsundaramauthor=J.+Deguzmanauthor=C.+M.+Bauerauthor=A.+F.+Hoffmanauthor=S.+Hamiltonauthor=Z.+Liangauthor=H.+Zengauthor=M.+E.+Fuentesauthor=J.+A.+Demartinoauthor=C.+Kitsonauthor=C.+S.+Stevensonauthor=D.+C.+Budd&title=BET+bromodomain+proteins+mediate+downstream+signaling+events+following+growth+factor+stimulation+in+human+lung+fibroblasts+and+are+involved+in+bleomycin-induced+pulmonary+fibrosis&doi=10.1124%2Fmol.112.081661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain proteins mediate downstream signaling events following growth factor stimulation in human lung fibroblasts and are involved in bleomycin-induced pulmonary fibrosis</span></div><div class="casAuthors">Tang, Xiaoyan; Peng, Ruoqi; Ren, Yonglin; Apparsundaram, Subramanium; Deguzman, Jeremy; Bauer, Carla M.; Hoffman, Ann F.; Hamilton, Shannon; Liang, Zhenmin; Zeng, Hang; Fuentes, Maria E.; DeMartino, Julie A.; Kitson, Christopher; Stevenson, Christopher S.; Budd, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">283-293</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Epigenetic alterations, such as histone acetylation, regulate the signaling outcomes and phenotypic responses of fibroblasts after growth factor stimulation.  The bromodomain and extra-terminal domain-contg. proteins (Brd) bind to acetylated histone residues, resulting in recruitment of components of the transcriptional machinery and subsequent gene transcription.  Given the central importance of fibroblasts in tissue fibrosis, this study sought to det. the role of Brd proteins in human lung fibroblasts (LFs) after growth factor stimulation and in the murine bleomycin model of lung fibrosis.  Using small interfering RNA against human Brd2 and Brd4 and pharmacol. Brd inhibitors, this study found that Brd2 and Brd4 are essential in mediating the phenotypic responses of LFs downstream of multiple growth factor pathways.  Growth factor stimulation of LFs causes increased histone acetylation, assocn. of Brd4 with growth factor-responsive genes, and enhanced transcription of these genes that could be attenuated with pharmacol. Brd inhibitors.  Of note, lung fibrosis induced after intratracheal bleomycin challenge in mice could be prevented by pretreatment of animals with pharmacol. inhibitors of Brd proteins.  This study is the first demonstration of a role for Brd2 and Brd4 proteins in mediating the responses of LFs after growth factor stimulation and in driving the induction of lung fibrosis in mice in response to bleomycin challenge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLn5H_z8-iULVg90H21EOLACvtfcHk0lgeB88EiswOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFKitLc%253D&md5=59f634a394e3a0f09b72e07a1eb0ccda</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.081661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.081661%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DR.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DApparsundaram%26aufirst%3DS.%26aulast%3DDeguzman%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DC.%2BM.%26aulast%3DHoffman%26aufirst%3DA.%2BF.%26aulast%3DHamilton%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DFuentes%26aufirst%3DM.%2BE.%26aulast%3DDemartino%26aufirst%3DJ.%2BA.%26aulast%3DKitson%26aufirst%3DC.%26aulast%3DStevenson%26aufirst%3DC.%2BS.%26aulast%3DBudd%26aufirst%3DD.%2BC.%26atitle%3DBET%2520bromodomain%2520proteins%2520mediate%2520downstream%2520signaling%2520events%2520following%2520growth%2520factor%2520stimulation%2520in%2520human%2520lung%2520fibroblasts%2520and%2520are%2520involved%2520in%2520bleomycin-induced%2520pulmonary%2520fibrosis%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D83%26spage%3D283%26epage%3D293%26doi%3D10.1124%2Fmol.112.081661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spiltoir, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavasin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demos-Davies, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinsey, T. A.</span></span> <span> </span><span class="NLM_article-title">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span>. <i>J. Mol. Cell. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1016/j.yjmcc.2013.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.yjmcc.2013.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23939492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2013&pages=175-179&author=J.+I.+Spiltoirauthor=M.+S.+Strattonauthor=M.+A.+Cavasinauthor=K.+Demos-Daviesauthor=B.+G.+Reidauthor=J.+Qiauthor=J.+E.+Bradnerauthor=T.+A.+McKinsey&title=BET+acetyl-lysine+binding+proteins+control+pathological+cardiac+hypertrophy&doi=10.1016%2Fj.yjmcc.2013.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span></div><div class="casAuthors">Spiltoir, Jessica I.; Stratton, Matthew S.; Cavasin, Maria A.; Demos-Davies, Kim; Reid, Brian G.; Qi, Jun; Bradner, James E.; McKinsey, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">175-179</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cardiac hypertrophy is an independent predictor of adverse outcomes in patients with heart failure, and thus represents an attractive target for novel therapeutic intervention.  JQ1, a small mol. inhibitor of bromodomain and extraterminal (BET) acetyl-lysine reader proteins, was identified in a high throughput screen designed to discover novel small mol. regulators of cardiomyocyte hypertrophy.  JQ1 dose-dependently blocked agonist-dependent hypertrophy of cultured neonatal rat ventricular myocytes (NRVMs) and reversed the prototypical gene program assocd. with pathol. cardiac hypertrophy.  JQ1 also blocked left ventricular hypertrophy (LVH) and improved cardiac function in adult mice subjected to transverse aortic constriction (TAC).  The BET family consists of BRD2, BRD3, BRD4 and BRDT.  BRD4 protein expression was increased during cardiac hypertrophy, and hypertrophic stimuli promoted recruitment of BRD4 to the transcriptional start site (TSS) of the gene encoding atrial natriuretic factor (ANF).  Binding of BRD4 to the ANF TSS was assocd. with increased phosphorylation of local RNA polymerase II.  These findings define a novel function for BET proteins as signal-responsive regulators of cardiac hypertrophy, and suggest that small mol. inhibitors of these epigenetic reader proteins have potential as therapeutics for heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDUzCSA2tNtrVg90H21EOLACvtfcHk0lgeB88EiswOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF&md5=ffe6d80a94c27e5d89b38baf33af365c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2013.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2013.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSpiltoir%26aufirst%3DJ.%2BI.%26aulast%3DStratton%26aufirst%3DM.%2BS.%26aulast%3DCavasin%26aufirst%3DM.%2BA.%26aulast%3DDemos-Davies%26aufirst%3DK.%26aulast%3DReid%26aufirst%3DB.%2BG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26atitle%3DBET%2520acetyl-lysine%2520binding%2520proteins%2520control%2520pathological%2520cardiac%2520hypertrophy%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2013%26volume%3D63%26spage%3D175%26epage%3D179%26doi%3D10.1016%2Fj.yjmcc.2013.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafiane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiacchia, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimbou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genest, J.</span></span> <span> </span><span class="NLM_article-title">RVX-208: a small molecule that increases apolipoprotein A-I and high-density lipoprotein cholesterol in vitro and in vivo</span>. <i>J. Am. Coll. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2580</span>– <span class="NLM_lpage">2589</span>, <span class="refDoi"> DOI: 10.1016/j.jacc.2010.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.jacc.2010.02.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=20513599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptV2msr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2010&pages=2580-2589&author=D.+Baileyauthor=R.+Jahagirdarauthor=A.+Gordonauthor=A.+Hafianeauthor=S.+Campbellauthor=S.+Chaturauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=F.+S.+Chiacchiaauthor=J.+Johanssonauthor=L.+Krimbouauthor=N.+C.+W.+Wongauthor=J.+Genest&title=RVX-208%3A+a+small+molecule+that+increases+apolipoprotein+A-I+and+high-density+lipoprotein+cholesterol+in+vitro+and+in+vivo&doi=10.1016%2Fj.jacc.2010.02.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208. A small molecule that increases apolipoprotein A-I and high-density lipoprotein cholesterol in vitro and in vivo</span></div><div class="casAuthors">Bailey, Dana; Jahagirdar, Ravi; Gordon, Allan; Hafiane, Anouar; Campbell, Steven; Chatur, Safia; Wagner, Gregory S.; Hansen, Henrik C.; Chiacchia, Fabrizio S.; Johansson, Jan; Krimbou, Larbi; Wong, Norman C. W.; Genest, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American College of Cardiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2580-2589</span>CODEN:
                <span class="NLM_cas:coden">JACCDI</span>;
        ISSN:<span class="NLM_cas:issn">0735-1097</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Objectives: The aim of this study was to det. whether a novel small mol. RVX-208 affects apolipoprotein (apo)A-I and high-d. lipoprotein cholesterol (HDL-C) levels in vitro and in vivo.  Background: Increased apoA-I and HDL-C levels are potential therapeutic targets for reducing atherosclerotic disease.  Methods: HepG2 cells were treated with 0 to 60 μmol/l RVX-208 followed by assays for apoA-I and HDL-C prodn.  For in vivo studies, African green monkeys (AGMs) received 15 to 60 mg/kg/day RVX-208, and the serum was analyzed for lipoprotein levels, HDL-subparticle distribution, cholesterol efflux, and activity of lipid-modifying enzymes.  A phase I clin. trial was conducted in healthy volunteers (given 1 to 20 mg/kg/day of RVX-208) to assess safety, tolerability, and pharmacokinetics.  Results: The RVX-208 induced apoA-I mRNA and protein synthesis in HepG2 cells, leading to in- creased levels of pre-β-migrating and α-lipoprotein particles contg. apoA-I (LpA-I) in spent media.  Similarly, in AGMs, RVX-208 treatment for 63 days increased serum apoA-I and HDL-C levels (60% and 97%, resp.).  In addn., the levels of pre-β1-LpA-I and α1-LpA-I HDL-subparticles were increased as well as ATP binding cassette AI, ATP binding cassette G1, and scavenger receptor class B type I-dependent cholesterol efflux.  These changes were not mediated by cholesteryl-ester-transfer protein.  Treatment of humans for 1 wk with oral RVX-208 increased apoA-I, pre-β-HDL, and HDL functionality.  Conclusions: RVX-208 increases apoA-I and HDL-C in vitro and in vivo.  In AGMs, RVX-208 raises serum pre-β1-LpA-I and α-LpA-I levels and enhances cholesterol efflux.  Data in humans point to beneficial features of RVX-208 that might be useful for treating atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEGExcnx6drLVg90H21EOLACvtfcHk0lgeB88EiswOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptV2msr0%253D&md5=aab9ac26e73704f1cd569e2ff47d0c51</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.jacc.2010.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jacc.2010.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DD.%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DA.%26aulast%3DHafiane%26aufirst%3DA.%26aulast%3DCampbell%26aufirst%3DS.%26aulast%3DChatur%26aufirst%3DS.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DChiacchia%26aufirst%3DF.%2BS.%26aulast%3DJohansson%26aufirst%3DJ.%26aulast%3DKrimbou%26aufirst%3DL.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DGenest%26aufirst%3DJ.%26atitle%3DRVX-208%253A%2520a%2520small%2520molecule%2520that%2520increases%2520apolipoprotein%2520A-I%2520and%2520high-density%2520lipoprotein%2520cholesterol%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DJ.%2520Am.%2520Coll.%2520Cardiol.%26date%3D2010%26volume%3D55%26spage%3D2580%26epage%3D2589%26doi%3D10.1016%2Fj.jacc.2010.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azhar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, R. A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, J.</span></span> <span> </span><span class="NLM_article-title">A novel BET bromodomain inhibitor, RVX-208, shows reduction of atherosclerosis in hyperlipidemic ApoE deficient mice</span>. <i>Atherosclerosis</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>236</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.atherosclerosis.2014.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.atherosclerosis.2014.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=25016363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Grt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=236&publication_year=2014&pages=91-100&author=R.+Jahagirdarauthor=H.+Zhangauthor=S.+Azharauthor=J.+Tobinauthor=S.+Attwellauthor=R.+Yuauthor=J.+Wuauthor=K.+G.+McLureauthor=H.+C.+Hansenauthor=G.+S.+Wagnerauthor=P.+R.+Youngauthor=R.+A.+K.+Srivastavaauthor=N.+C.+W.+Wongauthor=J.+Johansson&title=A+novel+BET+bromodomain+inhibitor%2C+RVX-208%2C+shows+reduction+of+atherosclerosis+in+hyperlipidemic+ApoE+deficient+mice&doi=10.1016%2Fj.atherosclerosis.2014.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A novel BET bromodomain inhibitor, RVX-208, shows reduction of atherosclerosis in hyperlipidemic ApoE deficient mice</span></div><div class="casAuthors">Jahagirdar, Ravi; Zhang, Haiyan; Azhar, Salman; Tobin, Jennifer; Attwell, Sarah; Yu, Raymond; Wu, Jin; McLure, Kevin G.; Hansen, Henrik C.; Wagner, Gregory S.; Young, Peter R.; Srivastava, Rai Ajit K.; Wong, Norman C. W.; Johansson, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Atherosclerosis (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">236</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-100</span>CODEN:
                <span class="NLM_cas:coden">ATHSBL</span>;
        ISSN:<span class="NLM_cas:issn">0021-9150</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Despite the benefit of statins in reducing cardiovascular risk, a sizable proportion of patients still remain at risk.  Since HDL reduces CVD risk through a process that involves formation of pre-beta particles that facilitates the removal of cholesterol from the lipid-laden macrophages in the arteries, inducing pre-beta particles, may reduce the risk of CVD.  A novel BET bromodomain antagonist, RVX-208, was reported to raise apoA-I and increase preβ-HDL particles in non-human primates and humans.  In the present study, we investigated the effect of RVX-208 on aortic lesion formation in hyperlipidemic apoE-/- mice.  Oral treatments of apoE-/- mice with 150 mg/kg b.i.d RVX-208 for 12 wk significantly reduced aortic lesion formation, accompanied by 2-fold increases in the levels of circulating HDL-C, and ∼50% decreases in LDL-C, although no significant changes in plasma apoA-I were obsd.  Circulating adhesion mols. as well as cytokines also showed significant redn.  Haptoglobin, a proinflammatory protein, known to bind with HDL/apoA-I, decreased >2.5-fold in the RVX-208 treated group.  With a therapeutic dosing regimen in which mice were fed Western diet for 10 wk to develop lesions followed by switching to a low fat diet and concurrent treatment with RVX-208 for 14 wk, RVX-208 similarly reduced lesion formation by 39% in the whole aorta without significant changes in the plasma lipid parameters.  RVX-208 significantly reduced the proinflammatory cytokines IP-10, MIP1 and MDC.  These results show that the antiatherogenic activity of BET inhibitor, RVX-208, occurs via a combination of lipid changes and anti-inflammatory activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXAgzwwmvsdrVg90H21EOLACvtfcHk0lhpE8nSefpXIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Grt77P&md5=135bbfa7ce02b3684be45e9c39f9e092</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.atherosclerosis.2014.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.atherosclerosis.2014.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DAzhar%26aufirst%3DS.%26aulast%3DTobin%26aufirst%3DJ.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DSrivastava%26aufirst%3DR.%2BA.%2BK.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DJohansson%26aufirst%3DJ.%26atitle%3DA%2520novel%2520BET%2520bromodomain%2520inhibitor%252C%2520RVX-208%252C%2520shows%2520reduction%2520of%2520atherosclerosis%2520in%2520hyperlipidemic%2520ApoE%2520deficient%2520mice%26jtitle%3DAtherosclerosis%26date%3D2014%26volume%3D236%26spage%3D91%26epage%3D100%26doi%3D10.1016%2Fj.atherosclerosis.2014.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huthmacher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudgin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doehner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.-I.+Chanauthor=S.+C.+Robsonauthor=C.-w.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+P.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lhpE8nSefpXIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.-I.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delannée, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tellier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudit, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">From ApoA1 upregulation to BET family bromodomain inhibition: discovery of I-BET151</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2963</span>– <span class="NLM_lpage">2967</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.01.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2012.01.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22386529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1elsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2963-2967&author=O.+Mirguetauthor=Y.+Lamotteauthor=F.+Doncheauthor=J.+Toumauthor=F.+Gellibertauthor=A.+Bouillotauthor=R.+Gosminiauthor=V.-L.+Nguyenauthor=D.+Delann%C3%A9eauthor=J.+Sealauthor=F.+Blandelauthor=A.-B.+Boullayauthor=E.+Boursierauthor=S.+Martinauthor=J.-M.+Brusqauthor=G.+Krysaauthor=A.+Riouauthor=R.+Tellierauthor=A.+Costazauthor=P.+Huetauthor=Y.+Duditauthor=L.+Trottetauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=From+ApoA1+upregulation+to+BET+family+bromodomain+inhibition%3A+discovery+of+I-BET151&doi=10.1016%2Fj.bmcl.2012.01.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">From ApoA1 upregulation to BET family bromodomain inhibition: Discovery of I-BET151</span></div><div class="casAuthors">Mirguet, Olivier; Lamotte, Yann; Donche, Frederic; Toum, Jerome; Gellibert, Francoise; Bouillot, Anne; Gosmini, Romain; Nguyen, Van-Loc; Delannee, Delphine; Seal, Jonathan; Blandel, Florence; Boullay, Anne-Benedicte; Boursier, Eric; Martin, Sandrine; Brusq, Jean-Marie; Krysa, Gael; Riou, Alizon; Tellier, Remi; Costaz, Agnes; Huet, Pascal; Dudit, Yann; Trottet, Lionel; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2963-2967</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery, synthesis, and biol. evaluation of a novel series of 7-isoxazoloquinolines is described.  Several analogs are shown to increase ApoA1 expression within the nanomolar range in the human hepatic cell line HepG2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIu6f80tbgE7Vg90H21EOLACvtfcHk0lhhYs0ygWQlCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1elsbY%253D&md5=48c9a8dc4b70bea4a86807f7a1cec5f1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.01.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.01.125%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.-L.%26aulast%3DDelann%25C3%25A9e%26aufirst%3DD.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DBlandel%26aufirst%3DF.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DBoursier%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DRiou%26aufirst%3DA.%26aulast%3DTellier%26aufirst%3DR.%26aulast%3DCostaz%26aufirst%3DA.%26aulast%3DHuet%26aufirst%3DP.%26aulast%3DDudit%26aufirst%3DY.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DFrom%2520ApoA1%2520upregulation%2520to%2520BET%2520family%2520bromodomain%2520inhibition%253A%2520discovery%2520of%2520I-BET151%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2963%26epage%3D2967%26doi%3D10.1016%2Fj.bmcl.2012.01.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel series of BET family bromodomain inhibitors: binding mode and profile of I-BET151 (GSK1210151A)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2968</span>– <span class="NLM_lpage">2972</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.02.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2012.02.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22437115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2968-2972&author=J.+Sealauthor=Y.+Lamotteauthor=F.+Doncheauthor=A.+Bouillotauthor=O.+Mirguetauthor=F.+Gellibertauthor=E.+Nicodemeauthor=G.+Krysaauthor=J.+Kirilovskyauthor=S.+Beinkeauthor=S.+McClearyauthor=I.+Riojaauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=L.+Gordonauthor=T.+Lewisauthor=A.+L.+Walkerauthor=L.+Cutlerauthor=D.+Lugoauthor=D.+M.+Wilsonauthor=J.+Witheringtonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Identification+of+a+novel+series+of+BET+family+bromodomain+inhibitors%3A+binding+mode+and+profile+of+I-BET151+%28GSK1210151A%29&doi=10.1016%2Fj.bmcl.2012.02.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span></div><div class="casAuthors">Seal, Jonathan; Lamotte, Yann; Donche, Frederic; Bouillot, Anne; Mirguet, Olivier; Gellibert, Francoise; Nicodeme, Edwige; Krysa, Gael; Kirilovsky, Jorge; Beinke, Soren; McCleary, Scott; Rioja, Inma; Bamborough, Paul; Chung, Chun-Wa; Gordon, Laurie; Lewis, Toni; Walker, Ann L.; Cutler, Leanne; Lugo, David; Wilson, David M.; Witherington, Jason; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2968-2972</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of quinoline isoxazole BET family bromodomain inhibitors are discussed.  Crystallog. is used to illustrate binding modes and rationalize their SAR.  One member, I-BET151 (GSK1210151A), shows good oral bioavailability in both the rat and minipig as well as demonstrating efficient suppression of bacterial induced inflammation and sepsis in a murine in vivo endotoxemia model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJKtxBCr-pRbVg90H21EOLACvtfcHk0lhhYs0ygWQlCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D&md5=a8037c89bcb93948d663208bd1ebfd55</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.041%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DIdentification%2520of%2520a%2520novel%2520series%2520of%2520BET%2520family%2520bromodomain%2520inhibitors%253A%2520binding%2520mode%2520and%2520profile%2520of%2520I-BET151%2520%2528GSK1210151A%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2968%26epage%3D2972%26doi%3D10.1016%2Fj.bmcl.2012.02.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bish, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nocka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primiano, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralph, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciammetta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span> <span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra C-terminal bromodomain inhibition through optimization of a fragment-derived hit</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9831</span>– <span class="NLM_lpage">9837</span>, <span class="refDoi"> DOI: 10.1021/jm3010515</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+C-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit&doi=10.1021%2Fjm3010515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0lhhYs0ygWQlCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520C-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837%26doi%3D10.1021%2Fjm3010515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wyce, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganji, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smitheman, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy-Wilson, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span> <span> </span><span class="NLM_article-title">BET inhibition silences expression of MYCN and BCL2 and induces cytotoxicity in neuroblastoma tumor models</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e72967</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0072967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1371%2Fjournal.pone.0072967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=24009722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2iu7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=A.+Wyceauthor=G.+Ganjiauthor=K.+N.+Smithemanauthor=C.-w.+Chungauthor=S.+Korenchukauthor=Y.+Baiauthor=O.+Barbashauthor=B.+Leauthor=P.+D.+Craggsauthor=M.+T.+McCabeauthor=K.+M.+Kennedy-Wilsonauthor=L.+V.+Sanchezauthor=R.+L.+Gosminiauthor=N.+Parrauthor=C.+F.+McHughauthor=D.+Dhanakauthor=R.+K.+Prinjhaauthor=K.+R.+Augerauthor=P.+J.+Tummino&title=BET+inhibition+silences+expression+of+MYCN+and+BCL2+and+induces+cytotoxicity+in+neuroblastoma+tumor+models&doi=10.1371%2Fjournal.pone.0072967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibition silences expression of MYCN and BCL2 and induces cytotoxicity in neuroblastoma tumor models</span></div><div class="casAuthors">Wyce, Anastasia; Ganji, Gopinath; Smitheman, Kimberly N.; Chung, Chun-Wa; Korenchuk, Susan; Bai, Yuchen; Barbash, Olena; Le, Bao Chau; Craggs, Peter D.; McCabe, Michael T.; Kennedy-Wilson, Karen M.; Sanchez, Lydia V.; Gosmini, Romain L.; Parr, Nigel; McHugh, Charles F.; Dhanak, Dashyant; Prinjha, Rab K.; Auger, Kurt R.; Tummino, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e72967</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">BET family proteins are epigenetic regulators known to control expression of genes involved in cell growth and oncogenesis.  Selective inhibitors of BET proteins exhibit potent anti-proliferative activity in a no. of hematol. cancer models, in part through suppression of the MYC oncogene and downstream Myc-driven pathways.  However, little is currently known about the activity of BET inhibitors in solid tumor models, and whether down-regulation of MYC family genes contributes to sensitivity.  Here we provide evidence for potent BET inhibitor activity in neuroblastoma, a pediatric solid tumor assocd. with a high frequency of MYCN amplifications.  We treated a panel of neuroblastoma cell lines with a novel small mol. inhibitor of BET proteins, GSK1324726A (I-BET726), and obsd. potent growth inhibition and cytotoxicity in most cell lines irresp. of MYCN copy no. or expression level.  Gene expression analyses in neuroblastoma cell lines suggest a role of BET inhibition in apoptosis, signaling, and N-Myc-driven pathways, including the direct suppression of BCL2 and MYCN.  Reversal of MYCN or BCL2 suppression reduces the potency of I-BET726-induced cytotoxicity in a cell line-specific manner; however, neither factor fully accounts for I-BET726 sensitivity.  Oral administration of I-BET726 to mouse xenograft models of human neuroblastoma results in tumor growth inhibition and down-regulation MYCN and BCL2 expression, suggesting a potential role for these genes in tumor growth.  Taken together, our data highlight the potential of BET inhibitors as novel therapeutics for neuroblastoma, and suggest that sensitivity is driven by pleiotropic effects on cell growth and apoptotic pathways in a context-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecB0_0elFVbVg90H21EOLACvtfcHk0liJzOZdVGs2Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2iu7fI&md5=d39ce25a50de58f18a278a9f67935bbc</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0072967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0072967%26sid%3Dliteratum%253Aachs%26aulast%3DWyce%26aufirst%3DA.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DKennedy-Wilson%26aufirst%3DK.%2BM.%26aulast%3DSanchez%26aufirst%3DL.%2BV.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26atitle%3DBET%2520inhibition%2520silences%2520expression%2520of%2520MYCN%2520and%2520BCL2%2520and%2520induces%2520cytotoxicity%2520in%2520neuroblastoma%2520tumor%2520models%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0072967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou-Eymard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8111</span>– <span class="NLM_lpage">8131</span>, <span class="refDoi"> DOI: 10.1021/jm5010539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5010539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8111-8131&author=R.+Gosminiauthor=V.+L.+Nguyenauthor=J.+Toumauthor=C.+Simonauthor=J.-M.+G.+Brusqauthor=G.+Krysaauthor=O.+Mirguetauthor=A.+M.+Riou-Eymardauthor=E.+V.+Boursierauthor=L.+Trottetauthor=P.+Bamboroughauthor=H.+Clarkauthor=C.-w.+Chungauthor=L.+Cutlerauthor=E.+H.+Demontauthor=R.+Kaurauthor=A.+J.+Lewisauthor=M.+B.+Schillingauthor=P.+E.+Sodenauthor=S.+Taylorauthor=A.+L.+Walkerauthor=M.+D.+Walkerauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=The+Discovery+of+I-BET726+%28GSK1324726A%29%2C+a+Potent+Tetrahydroquinoline+ApoA1+Up-Regulator+and+Selective+BET+Bromodomain+Inhibitor&doi=10.1021%2Fjm5010539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span></div><div class="casAuthors">Gosmini, Romain; Nguyen, Van Loc; Toum, Jerome; Simon, Christophe; Brusq, Jean-Marie G.; Krysa, Gael; Mirguet, Olivier; Riou-Eymard, Alizon M.; Boursier, Eric V.; Trottet, Lionel; Bamborough, Paul; Clark, Hugh; Chung, Chun-wa; Cutler, Leanne; Demont, Emmanuel H.; Kaur, Rejbinder; Lewis, Antonia J.; Schilling, Mark B.; Soden, Peter E.; Taylor, Simon; Walker, Ann L.; Walker, Matthew D.; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8111-8131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through their function as epigenetic readers of the histone code, the BET family of bromodomain-contg. proteins regulate expression of multiple genes of therapeutic relevance, including those involved in tumor cell growth and inflammation.  BET bromodomain inhibitors have profound antiproliferative and anti-inflammatory effects which translate into efficacy in oncol. and inflammation models, and the first compds. have now progressed into clin. trials.  The exciting biol. of the BETs has led to great interest in the discovery of novel inhibitor classes.  Here we describe the identification of a novel tetrahydroquinoline series through up-regulation of apolipoprotein A1 and the optimization into potent compds. active in murine models of septic shock and neuroblastoma.  At the mol. level, these effects are produced by inhibition of BET bromodomains.  X-ray crystallog. reveals the interactions explaining the structure-activity relationships of binding.  The resulting lead mol., I-BET726, represents a new, potent, and selective class of tetrahydroquinoline-based BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5OdMn_TjoFLVg90H21EOLACvtfcHk0lhqaex2qbVuNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J&md5=b1c9c9cae73c41252cf73908d561bd9b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm5010539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5010539%26sid%3Dliteratum%253Aachs%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRiou-Eymard%26aufirst%3DA.%2BM.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DSchilling%26aufirst%3DM.%2BB.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DWalker%26aufirst%3DM.%2BD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DThe%2520Discovery%2520of%2520I-BET726%2520%2528GSK1324726A%2529%252C%2520a%2520Potent%2520Tetrahydroquinoline%2520ApoA1%2520Up-Regulator%2520and%2520Selective%2520BET%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8111%26epage%3D8131%26doi%3D10.1021%2Fjm5010539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowlati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagey, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, S. B.</span></span> <span> </span><span class="NLM_article-title">Abstract A49: Clinically efficacy of the BET bromodomain inhibitor TEN-010 in an open-label substudy with patients with documented NUT-midline carcinoma (NMC)</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">A49</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.TARG-15-A49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1158%2F1535-7163.TARG-15-A49" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=A49&author=G.+I.+Shapiroauthor=A.+Dowlatiauthor=P.+M.+LoRussoauthor=J.+P.+Ederauthor=A.+Andersonauthor=K.+T.+Doauthor=M.+H.+Kageyauthor=C.+Sirardauthor=J.+E.+Bradnerauthor=S.+B.+Landau&title=Abstract+A49%3A+Clinically+efficacy+of+the+BET+bromodomain+inhibitor+TEN-010+in+an+open-label+substudy+with+patients+with+documented+NUT-midline+carcinoma+%28NMC%29&doi=10.1158%2F1535-7163.TARG-15-A49"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-A49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-A49%26sid%3Dliteratum%253Aachs%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DDowlati%26aufirst%3DA.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DAnderson%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DK.%2BT.%26aulast%3DKagey%26aufirst%3DM.%2BH.%26aulast%3DSirard%26aufirst%3DC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLandau%26aufirst%3DS.%2BB.%26atitle%3DAbstract%2520A49%253A%2520Clinically%2520efficacy%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520TEN-010%2520in%2520an%2520open-label%2520substudy%2520with%2520patients%2520with%2520documented%2520NUT-midline%2520carcinoma%2520%2528NMC%2529%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DA49%26doi%3D10.1158%2F1535-7163.TARG-15-A49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panaroni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N.</span></span> <span> </span><span class="NLM_article-title">Preclinical activity of CPI-0610, a novel small-molecule bromodomain and extra-terminal protein inhibitor in the therapy of multiple myeloma</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1760</span>– <span class="NLM_lpage">1769</span>, <span class="refDoi"> DOI: 10.1038/leu.2016.355</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fleu.2016.355" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27890933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=1760-1769&author=K.+T.+Siuauthor=J.+Ramachandranauthor=A.+J.+Yeeauthor=H.+Edaauthor=L.+Santoauthor=C.+Panaroniauthor=J.+A.+Mertzauthor=R.+J.+Simsauthor=M.+R.+Cooperauthor=N.+Raje&title=Preclinical+activity+of+CPI-0610%2C+a+novel+small-molecule+bromodomain+and+extra-terminal+protein+inhibitor+in+the+therapy+of+multiple+myeloma&doi=10.1038%2Fleu.2016.355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical activity of CPI-0610, a novel small-molecule bromodomain and extra-terminal protein inhibitor in the therapy of multiple myeloma</span></div><div class="casAuthors">Siu, K. T.; Ramachandran, J.; Yee, A. J.; Eda, H.; Santo, L.; Panaroni, C.; Mertz, J. A.; Sims, R. J., III; Cooper, M. R.; Raje, N.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1760-1769</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inhibition of the bromodomain and extra-terminal (BET) proteins is a promising therapeutic strategy for various hematol. cancers.  Previous studies suggest that BET inhibitors constrain tumor cell proliferation and survival mainly through the suppression of MYC transcription and activity.  However, suppression of the transcription of addnl. genes also contributes to the antitumor activity of BET inhibitors but is less well understood.  Here we examd. the therapeutic potential of CPI-0610, a potent BET inhibitor currently undergoing phase I clin. testing, in multiple myeloma (MM).  CPI-0610 displays potent cytotoxicity against MM cell lines and patient-derived MM cells through G1 cell cycle arrest and caspase-dependent apoptosis.  CPI-0610-mediated BET inhibition overcomes the protective effects conferred by cytokines and bone marrow stromal cells.  We also confirmed the in vivo efficacy of CPI-0610 in a MM xenograft mouse model.  Our study found IKZF1 and IRF4 to be among the primary targets of CPI-0610, along with MYC.  Given that immunomodulatory drugs (IMiDs) stabilize cereblon and facilitate Ikaros degrdn. in MM cells, we combined it with CPI-0610.  Combination studies of CPI-0610 with IMiDs show in vitro synergism, in part due to concomitant suppression of IKZF1, IRF4 and MYC, providing a rationale for clin. testing of this drug combination in MM patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod3GErt1kMIbVg90H21EOLACvtfcHk0lj59Wc-qqmraA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kq&md5=d0a386df69df69d187e10d4e784b986a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fleu.2016.355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2016.355%26sid%3Dliteratum%253Aachs%26aulast%3DSiu%26aufirst%3DK.%2BT.%26aulast%3DRamachandran%26aufirst%3DJ.%26aulast%3DYee%26aufirst%3DA.%2BJ.%26aulast%3DEda%26aufirst%3DH.%26aulast%3DSanto%26aufirst%3DL.%26aulast%3DPanaroni%26aufirst%3DC.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DRaje%26aufirst%3DN.%26atitle%3DPreclinical%2520activity%2520of%2520CPI-0610%252C%2520a%2520novel%2520small-molecule%2520bromodomain%2520and%2520extra-terminal%2520protein%2520inhibitor%2520in%2520the%2520therapy%2520of%2520multiple%2520myeloma%26jtitle%3DLeukemia%26date%3D2017%26volume%3D31%26spage%3D1760%26epage%3D1769%26doi%3D10.1038%2Fleu.2016.355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span> <span> </span><span class="NLM_article-title">Identification of a benzoisoxazoloazepine inhibitor (CPI-0610) of the bromodomain and extra-terminal (BET) family as a candidate for human clinical trials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1330</span>– <span class="NLM_lpage">1339</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1330-1339&author=B.+K.+Albrechtauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=A.+C%C3%B4t%C3%A9author=Y.+Leblancauthor=C.+G.+Nasveschukauthor=S.+Bellonauthor=L.+Bergeronauthor=R.+Campbellauthor=N.+Cantoneauthor=M.+R.+Cooperauthor=R.+T.+Cummingsauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Normantauthor=M.+O%E2%80%99Mearaauthor=E.+Pardoauthor=F.+Poyauthor=P.+Sandyauthor=J.+Supkoauthor=R.+J.+Simsauthor=J.-C.+Harmangeauthor=A.+M.+Taylorauthor=J.+E.+Audia&title=Identification+of+a+benzoisoxazoloazepine+inhibitor+%28CPI-0610%29+of+the+bromodomain+and+extra-terminal+%28BET%29+family+as+a+candidate+for+human+clinical+trials&doi=10.1021%2Facs.jmedchem.5b01882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span></div><div class="casAuthors">Albrecht, Brian K.; Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Cote, Alexandre; Leblanc, Yves; Nasveschuk, Christopher G.; Bellon, Steve; Bergeron, Louise; Campbell, Robert; Cantone, Nico; Cooper, Michael R.; Cummings, Richard T.; Jayaram, Hariharan; Joshi, Shivangi; Mertz, Jennifer A.; Neiss, Adrianne; Normant, Emmanuel; O'Meara, Michael; Pardo, Eneida; Poy, Florence; Sandy, Peter; Supko, Jeffrey; Sims, Robert J.; Harmange, Jean-Christophe; Taylor, Alexander M.; Audia, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1330-1339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, inhibition of the interaction between the bromodomain and extra-terminal domain (BET) family of chromatin adaptors and acetyl-lysine residues on chromatin has emerged as a promising approach to regulate the expression of important disease-relevant genes, including MYC, BCL-2, and NF-κB.  Here we describe the identification and characterization of a potent and selective benzoisoxazoloazepine BET bromodomain inhibitor that attenuates BET-dependent gene expression in vivo, demonstrates antitumor efficacy in an MV-4-11 mouse xenograft model, and is currently undergoing human clin. trials for hematol. malignancies (CPI-0610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp40qjnYl4EZLVg90H21EOLACvtfcHk0lj59Wc-qqmraA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D&md5=5e1c5aab26aa58844a9f6670a7bbd78b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01882%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSupko%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26atitle%3DIdentification%2520of%2520a%2520benzoisoxazoloazepine%2520inhibitor%2520%2528CPI-0610%2529%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520family%2520as%2520a%2520candidate%2520for%2520human%2520clinical%2520trials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1330%26epage%3D1339%26doi%3D10.1021%2Facs.jmedchem.5b01882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span> <span> </span><span class="NLM_article-title">Development of methyl isoxazoleazepines as inhibitors of BET</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1842</span>– <span class="NLM_lpage">1848</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.03.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2015.03.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=25851940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVKrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1842-1848&author=M.+C.+Hewittauthor=Y.+Leblancauthor=V.+S.+Gehlingauthor=R.+G.+Vaswaniauthor=A.+C%C3%B4t%C3%A9author=C.+G.+Nasveschukauthor=A.+M.+Taylorauthor=J.-C.+Harmangeauthor=J.+E.+Audiaauthor=E.+Pardoauthor=R.+Cummingsauthor=S.+Joshiauthor=P.+Sandyauthor=J.+A.+Mertzauthor=R.+J.+Simsauthor=L.+Bergeronauthor=B.+M.+Bryantauthor=S.+Bellonauthor=F.+Poyauthor=H.+Jayaramauthor=Y.+Tangauthor=B.+K.+Albrecht&title=Development+of+methyl+isoxazoleazepines+as+inhibitors+of+BET&doi=10.1016%2Fj.bmcl.2015.03.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Development of methyl isoxazoleazepines as inhibitors of BET</span></div><div class="casAuthors">Hewitt, Michael C.; Leblanc, Yves; Gehling, Victor S.; Vaswani, Rishi G.; Cote, Alexandre; Nasveschuk, Christopher G.; Taylor, Alexander M.; Harmange, Jean-Christophe; Audia, James E.; Pardo, Eneida; Cummings, Rich; Joshi, Shivangi; Sandy, Peter; Mertz, Jennifer A.; Sims, Robert J.; Bergeron, Louise; Bryant, Barbara M.; Bellon, Steve; Poy, Florence; Jayaran, Hariharan; Tang, Yong; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1842-1848</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this report we detail the evolution of our previously reported thiophene isoxazole BET inhibitor chemotype exemplified by CPI-3 to a novel bromodomain selective chemotype (the Me isoxazoleazepine chemotype) exemplified by carboxamide 23.  The Me isoxazoleazepine chemotype provides potent inhibition of the bromodomains of the BET family, excellent in vivo PK across species, low unbound clearance, and target engagement in a MYC PK-PD model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrdQuVhIoU1rVg90H21EOLACvtfcHk0lgdg2Uappwqiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVKrsL8%253D&md5=fb9a78528287f82ccf004e751e491ee4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.03.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.03.045%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DDevelopment%2520of%2520methyl%2520isoxazoleazepines%2520as%2520inhibitors%2520of%2520BET%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D1842%26epage%3D1848%26doi%3D10.1016%2Fj.bmcl.2015.03.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setser, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzotriazolo[4,3-d][1,4]diazepines as orally active inhibitors of BET bromodomains</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1021/ml500411h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500411h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlt1Wls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=145-150&author=A.+M.+Taylorauthor=R.+G.+Vaswaniauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=Y.+Leblancauthor=J.+E.+Audiaauthor=S.+Bellonauthor=R.+T.+Cummingsauthor=A.+C%C3%B4t%C3%A9author=J.-C.+Harmangeauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+M.+Loraauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Pardoauthor=C.+G.+Nasveschukauthor=F.+Poyauthor=P.+Sandyauthor=J.+W.+Setserauthor=R.+J.+Simsauthor=Y.+Tangauthor=B.+K.+Albrecht&title=Discovery+of+benzotriazolo%5B4%2C3-d%5D%5B1%2C4%5Ddiazepines+as+orally+active+inhibitors+of+BET+bromodomains&doi=10.1021%2Fml500411h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzotriazolo[4,3-d][1,4]diazepines as Orally Active Inhibitors of BET Bromodomains</span></div><div class="casAuthors">Taylor, Alexander M.; Vaswani, Rishi G.; Gehling, Victor S.; Hewitt, Michael C.; Leblanc, Yves; Audia, James E.; Bellon, Steve; Cummings, Richard T.; Cote, Alexandre; Harmange, Jean-Christophe; Jayaram, Hari; Joshi, Shivangi; Lora, Jose M.; Mertz, Jennifer A.; Neiss, Adrianne; Pardo, Eneida; Nasveschuk, Christopher G.; Poy, Florence; Sandy, Peter; Setser, Jeremy W.; Sims, Robert J.; Tang, Yong; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-150</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of the bromodomains of the BET family, of which BRD4 is a member, has been shown to decrease myc and interleukin (IL) 6 in vivo, markers that are of therapeutic relevance to cancer and inflammatory disease, resp.  Herein the authors report substituted benzo[b]isoxazolo[4,5-d]azepines and benzotriazolo[4,3-d][1,4]diazepines as fragment-derived novel inhibitors of the bromodomain of BRD4.  Compds. from these series were potent and selective in cells, and subsequent optimization of microsomal stability yielded representatives that demonstrated dose- and time-dependent redn. of plasma IL-6 in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDybfuij7kebVg90H21EOLACvtfcHk0lgdg2Uappwqiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlt1Wls7k%253D&md5=38759c818814869d6c951a035bad7617</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fml500411h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500411h%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSetser%26aufirst%3DJ.%2BW.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DDiscovery%2520of%2520benzotriazolo%255B4%252C3-d%255D%255B1%252C4%255Ddiazepines%2520as%2520orally%2520active%2520inhibitors%2520of%2520BET%2520bromodomains%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D145%26epage%3D150%26doi%3D10.1021%2Fml500411h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankaranarayanan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yellapantula, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bangalore Srinivasamurthy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birudukota, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span> <span> </span><span class="NLM_article-title">Discovery, design, and optimization of isoxazole azepine BET inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">835</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1021/ml4001485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4001485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFanu7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=835-840&author=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=Y.+Leblancauthor=A.+C%C3%B4t%C3%A9author=C.+G.+Nasveschukauthor=A.+M.+Taylorauthor=J.-C.+Harmangeauthor=J.+E.+Audiaauthor=E.+Pardoauthor=S.+Joshiauthor=P.+Sandyauthor=J.+A.+Mertzauthor=R.+J.+Simsauthor=L.+Bergeronauthor=B.+M.+Bryantauthor=S.+Bellonauthor=F.+Poyauthor=H.+Jayaramauthor=R.+Sankaranarayananauthor=S.+Yellapantulaauthor=N.+Bangalore+Srinivasamurthyauthor=S.+Birudukotaauthor=B.+K.+Albrecht&title=Discovery%2C+design%2C+and+optimization+of+isoxazole+azepine+BET+inhibitors&doi=10.1021%2Fml4001485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, Design, and Optimization of Isoxazole Azepine BET Inhibitors</span></div><div class="casAuthors">Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Leblanc, Yves; Cote, Alexandre; Nasveschuk, Christopher G.; Taylor, Alexander M.; Harmange, Jean-Christophe; Audia, James E.; Pardo, Eneida; Joshi, Shivangi; Sandy, Peter; Mertz, Jennifer A.; Sims, Robert J.; Bergeron, Louise; Bryant, Barbara M.; Bellon, Steve; Poy, Florence; Jayaram, Hariharan; Sankaranarayanan, Ravichandran; Yellapantula, Sreegouri; Bangalore Srinivasamurthy, Nandana; Birudukota, Swarnakumari; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">835-840</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of a novel series of small mol. BET inhibitors is described.  Using crystallog. binding modes of an amino-isoxazole fragment and known BET inhibitors, a structure-based drug design effort lead to a novel isoxazole azepine scaffold, e.g. I.  This scaffold showed good potency in biochem. and cellular assays and oral activity in an in vivo model of BET inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCIGHOq7LlgrVg90H21EOLACvtfcHk0lgdg2Uappwqiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFanu7jI&md5=7a3ab8ec50ba88e865c5a6524cf02c48</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fml4001485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4001485%26sid%3Dliteratum%253Aachs%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSankaranarayanan%26aufirst%3DR.%26aulast%3DYellapantula%26aufirst%3DS.%26aulast%3DBangalore%2BSrinivasamurthy%26aufirst%3DN.%26aulast%3DBirudukota%26aufirst%3DS.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DDiscovery%252C%2520design%252C%2520and%2520optimization%2520of%2520isoxazole%2520azepine%2520BET%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D835%26epage%3D840%26doi%3D10.1021%2Fml4001485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thieblemont, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffoux, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarotto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leleu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Roca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odore, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roumier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourdel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesnel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michallet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preudhomme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palumbo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombret, H.</span></span> <span> </span><span class="NLM_article-title">Abstract CT231: BET-bromodomain inhibitor OTX015 shows clinically meaningful activity at nontoxic doses: interim results of an ongoing phase I trial in hematologic malignancies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">CT231</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2014-CT231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1158%2F1538-7445.AM2014-CT231" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=CT231&author=P.+E.+Heraitauthor=C.+Berthonauthor=C.+Thieblemontauthor=E.+Raffouxauthor=V.+Magarottoauthor=A.+Stathisauthor=X.+Thomasauthor=X.+Leleuauthor=C.+Gomez-Rocaauthor=E.+Odoreauthor=C.+Roumierauthor=F.+Bourdelauthor=B.+Quesnelauthor=E.+Zuccaauthor=M.+Michalletauthor=C.+Recherauthor=E.+Cvitkovicauthor=K.+Rezaiauthor=C.+Preudhommeauthor=T.+Faconauthor=A.+Palumboauthor=H.+Dombret&title=Abstract+CT231%3A+BET-bromodomain+inhibitor+OTX015+shows+clinically+meaningful+activity+at+nontoxic+doses%3A+interim+results+of+an+ongoing+phase+I+trial+in+hematologic+malignancies&doi=10.1158%2F1538-7445.AM2014-CT231"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2014-CT231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2014-CT231%26sid%3Dliteratum%253Aachs%26aulast%3DHerait%26aufirst%3DP.%2BE.%26aulast%3DBerthon%26aufirst%3DC.%26aulast%3DThieblemont%26aufirst%3DC.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DMagarotto%26aufirst%3DV.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DX.%26aulast%3DLeleu%26aufirst%3DX.%26aulast%3DGomez-Roca%26aufirst%3DC.%26aulast%3DOdore%26aufirst%3DE.%26aulast%3DRoumier%26aufirst%3DC.%26aulast%3DBourdel%26aufirst%3DF.%26aulast%3DQuesnel%26aufirst%3DB.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DMichallet%26aufirst%3DM.%26aulast%3DRecher%26aufirst%3DC.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DPreudhomme%26aufirst%3DC.%26aulast%3DFacon%26aufirst%3DT.%26aulast%3DPalumbo%26aufirst%3DA.%26aulast%3DDombret%26aufirst%3DH.%26atitle%3DAbstract%2520CT231%253A%2520BET-bromodomain%2520inhibitor%2520OTX015%2520shows%2520clinically%2520meaningful%2520activity%2520at%2520nontoxic%2520doses%253A%2520interim%2520results%2520of%2520an%2520ongoing%2520phase%2520I%2520trial%2520in%2520hematologic%2520malignancies%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3DCT231%26doi%3D10.1158%2F1538-7445.AM2014-CT231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Roca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouge, T. d. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uro-Coste, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Braud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelosi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span> <span> </span><span class="NLM_article-title">Clinical response of carcinomas harboring the BRD4-NUT oncoprotein to the targeted bromodomain inhibitor OTX015/MK-8628</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">492</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-15-1335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1158%2F2159-8290.cd-15-1335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=26976114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlOqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=492-500&author=A.+Stathisauthor=E.+Zuccaauthor=M.+Bekraddaauthor=C.+Gomez-Rocaauthor=J.-P.+Delordauthor=T.+d.+L.+M.+Rougeauthor=E.+Uro-Costeauthor=F.+de+Braudauthor=G.+Pelosiauthor=C.+A.+French&title=Clinical+response+of+carcinomas+harboring+the+BRD4-NUT+oncoprotein+to+the+targeted+bromodomain+inhibitor+OTX015%2FMK-8628&doi=10.1158%2F2159-8290.cd-15-1335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Response of Carcinomas Harboring the BRD4-NUT Oncoprotein to the Targeted Bromodomain Inhibitor OTX015/MK-8628</span></div><div class="casAuthors">Stathis, Anastasios; Zucca, Emanuele; Bekradda, Mohamed; Gomez-Roca, Carlos; Delord, Jean-Pierre; de La Motte Rouge, Thibault; Uro-Coste, Emmanuelle; de Braud, Filippo; Pelosi, Giuseppe; French, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">492-500</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The antineoplastic, prodifferentiative effects of bromodomain and extra-terminal (BET) bromodomain (BRD) inhibitors were initially discovered in NUT midline carcinoma (NMC), an aggressive subtype of squamous cancer driven by the BRD4-NUT fusion oncoprotein.  BRD4-NUT blocks differentiation and maintains tumor growth through a potent chromatin-modifying mechanism.  OTX015/MK-8628, a novel oral BET inhibitor, targets BRD2/3/4/T with preclin. activity in NMC and several other tumor types and is currently in clin. development.  Antitumor activity was evaluated in four patients with advanced-stage NMC with confirmed BRD4-NUT fusions who were treated with 80 mg OTX015/MK-8628 once daily in a compassionate-use context.  Two patients responded rapidly with tumor regression and symptomatic relief, and a third had meaningful disease stabilization with a minor metabolic response.  The main side effects were mild to moderate gastrointestinal toxicity and fatigue, and reversible grade 3 thrombocytopenia.  This is the first proof-of-concept evidence of clin. activity of a BRD inhibitor in targeting BRD4-NUT.  Significance: We present the first clin. proof-of-concept that targeting BRD4-NUT with a BET inhibitor results in impressive and rapid antitumor activity in NMC.  It offers strong potential for future clin. application in this rare patient population as either a single agent or in combination with other agents.  Cancer Discov; 6(5); 492-500. ©2016 AACR.  This article is highlighted in the In This Issue feature, p.  461.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGqXhyj-x5pLVg90H21EOLACvtfcHk0lh7Igpg3VLo6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlOqsrs%253D&md5=73f6d61463bc6db72178d70771544787</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.cd-15-1335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.cd-15-1335%26sid%3Dliteratum%253Aachs%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DGomez-Roca%26aufirst%3DC.%26aulast%3DDelord%26aufirst%3DJ.-P.%26aulast%3DRouge%26aufirst%3DT.%2Bd.%2BL.%2BM.%26aulast%3DUro-Coste%26aufirst%3DE.%26aulast%3Dde%2BBraud%26aufirst%3DF.%26aulast%3DPelosi%26aufirst%3DG.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26atitle%3DClinical%2520response%2520of%2520carcinomas%2520harboring%2520the%2520BRD4-NUT%2520oncoprotein%2520to%2520the%2520targeted%2520bromodomain%2520inhibitor%2520OTX015%252FMK-8628%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26spage%3D492%26epage%3D500%26doi%3D10.1158%2F2159-8290.cd-15-1335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odore, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitor OTX015 (MK-8628) combined with targeted agents shows strong in vivo antitumor activity in lymphoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">58142</span>– <span class="NLM_lpage">58147</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.10983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.18632%2Foncotarget.10983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27494885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A280%3ADC%252BC2s3otVelsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=58142-58147&author=E.+Gaudioauthor=C.+Tarantelliauthor=M.+Ponzoniauthor=E.+Odoreauthor=K.+Rezaiauthor=E.+Bernasconiauthor=L.+Cascioneauthor=A.+Rinaldiauthor=A.+Stathisauthor=E.+Riveiroauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=Bromodomain+inhibitor+OTX015+%28MK-8628%29+combined+with+targeted+agents+shows+strong+in+vivo+antitumor+activity+in+lymphoma&doi=10.18632%2Foncotarget.10983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibitor OTX015 (MK-8628) combined with targeted agents shows strong in vivo antitumor activity in lymphoma</span></div><div class="casAuthors">Gaudio Eugenio; Tarantelli Chiara; Bernasconi Elena; Cascione Luciano; Rinaldi Andrea; Bertoni Francesco; Ponzoni Maurilio; Odore Elodie; Rezai Keyvan; Cascione Luciano; Stathis Anastasios; Zucca Emanuele; Bertoni Francesco; Riveiro Eugenia; Cvitkovic Esteban</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">58142-58147</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The bromodomain inhibitor OTX015 (MK-8628) has shown anti-lymphoma activity as a single agent in both the preclinical and clinical settings, as well as in vitro synergism with several anticancer agents.  Here, we report in vivo data for OTX015 in combination with the histone deacetylase inhibitor vorinostat, the Bruton's tyrosine kinase inhibitor ibrutinib, the anti-CD20 monoclonal antibody rituximab, and the mTOR inhibitor everolimus in a diffuse large B cell lymphoma model.  The antitumor effect of OTX015-containing combinations in SU-DHL-2 xenografts in mice was much stronger than the activity of the corresponding single agents with almost complete tumor eradication for all four combinations.  Pharmacokinetic analyses showed similar OTX015 levels in plasma and tumor samples of approximately 1.5 μM, which is equivalent to the concentration showing strong in vitro activity.  For all four combinations, mean terminal levels of the bromodomain inhibitor differed from those in mice exposed to single agent OTX015, indicating a need for thorough pharmacokinetic investigations in phase I combination studies.  In conclusion, our results provide a strong rationale to explore OTX015-containing combinations in the clinical lymphoma setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRjLbpvJwtLh1UIs6wFK5xJfW6udTcc2ebab6KAU1da7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3otVelsA%253D%253D&md5=2180a408b42b0d96851a1ff52f0754ef</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.10983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.10983%26sid%3Dliteratum%253Aachs%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DOdore%26aufirst%3DE.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DRiveiro%26aufirst%3DE.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DBromodomain%2520inhibitor%2520OTX015%2520%2528MK-8628%2529%2520combined%2520with%2520targeted%2520agents%2520shows%2520strong%2520in%2520vivo%2520antitumor%2520activity%2520in%2520lymphoma%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D58142%26epage%3D58147%26doi%3D10.18632%2Foncotarget.10983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Astorgues-Xerri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odore, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erba, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltrame, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frapolli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Incalci, M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain inhibitor OTX015 (MK-8628) exerts anti-tumor activity in triple-negative breast cancer models as single agent and in combination with everolimus</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">7598</span>– <span class="NLM_lpage">7613</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.18632%2Foncotarget.13814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27935867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252Fis1Cntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=7598-7613&author=R.+V%C3%A1zquezauthor=M.+E.+Riveiroauthor=L.+Astorgues-Xerriauthor=E.+Odoreauthor=K.+Rezaiauthor=E.+Erbaauthor=N.+Paniniauthor=A.+Rinaldiauthor=I.+Kweeauthor=L.+Beltrameauthor=M.+Bekraddaauthor=E.+Cvitkovicauthor=F.+Bertoniauthor=R.+Frapolliauthor=M.+D%E2%80%99Incalci&title=The+bromodomain+inhibitor+OTX015+%28MK-8628%29+exerts+anti-tumor+activity+in+triple-negative+breast+cancer+models+as+single+agent+and+in+combination+with+everolimus&doi=10.18632%2Foncotarget.13814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain inhibitor OTX015 (MK-8628) exerts anti-tumor activity in triple-negative breast cancer models as single agent and in combination with everolimus</span></div><div class="casAuthors">Vazquez Ramiro; Erba Eugenio; Panini Nicolo; Beltrame Luca; Frapolli Roberta; D'Incalci Maurizio; Riveiro Maria E; Astorgues-Xerri Lucile; Odore Elodie; Bekradda Mohamed; Cvitkovic Esteban; Odore Elodie; Rezai Keyvan; Rinaldi Andrea; Kwee Ivo; Bertoni Francesco; Kwee Ivo; Kwee Ivo; Cvitkovic Esteban; Bertoni Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">7598-7613</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Triple-negative breast cancer (TNBC) is an aggressive and heterogeneous subgroup of breast tumors clinically defined by the lack of estrogen, progesterone and HER2 receptors, limiting the use of the targeted therapies employed in other breast malignancies.  Recent evidence indicates that c-MYC is a key driver of TNBC.  The BET-bromodomain inhibitor OTX015 (MK-8628) has potent antiproliferative activity accompanied by c-MYC down-regulation in several tumor types, and has demonstrated synergism with the mTOR inhibitor everolimus in different models.  The aim of this study was to evaluate the anti-tumor activity of OTX015 as single agent and in combination with everolimus in TNBC models.  OTX015 was assayed in three human TNBC-derived cell lines, HCC1937, MDA-MB-231 and MDA-MB-468, all showing antiproliferative activity after 72 h (GI50 = 75-650 nM).  This was accompanied by cell cycle arrest and decreased expression of cancer stem cells markers.  However, c-MYC protein and mRNA levels were only down-regulated in MDA-MB-468 cells.  Gene set enrichment analysis showed up-regulation of genes involved in epigenetic control of transcription, chromatin and the cell cycle, and down-regulation of stemness-related genes.  In vitro, combination with everolimus was additive in HCC1937 and MDA-MB-231 cells, but antagonistic in MDA-MB-468 cells.  In MDA-MB-231 murine xenografts, tumor mass was significantly (p < 0.05) reduced by OTX015 with respect to vehicle-treated animals (best T/C = 40.7%).  Although everolimus alone was not active, the combination was more effective than OTX015 alone (best T/C = 20.7%).  This work supports current clinical trials with OTX015 in TNBC (NCT02259114).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpC5NvWs-BQRJ5E29YNGvQfW6udTcc2eaystWuuLR4Rrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252Fis1Cntw%253D%253D&md5=ac5f6998cb398ab9dbb9c9493880a5c7</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13814%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez%26aufirst%3DR.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DAstorgues-Xerri%26aufirst%3DL.%26aulast%3DOdore%26aufirst%3DE.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DErba%26aufirst%3DE.%26aulast%3DPanini%26aufirst%3DN.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBeltrame%26aufirst%3DL.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26aulast%3DFrapolli%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Incalci%26aufirst%3DM.%26atitle%3DThe%2520bromodomain%2520inhibitor%2520OTX015%2520%2528MK-8628%2529%2520exerts%2520anti-tumor%2520activity%2520in%2520triple-negative%2520breast%2520cancer%2520models%2520as%2520single%2520agent%2520and%2520in%2520combination%2520with%2520everolimus%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D7598%26epage%3D7613%26doi%3D10.18632%2Foncotarget.13814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of <i>N</i>-(4-(2,4-difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a potent and orally available bromodomain and extraterminal domain (BET) family bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>– <span class="NLM_lpage">8384</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-difluorophenoxy%29-3-%286-methyl-7-oxo-6%2C7-dihydro-1H-pyrrolo%5B2%2C3-c%5Dpyridin-4-yl%29phenyl%29ethanesulfonamide+%28ABBV-075%2FMivebresib%29%2C+a+potent+and+orally+available+bromodomain+and+extraterminal+domain+%28BET%29+family+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0ljaF4z7a30OXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-difluorophenoxy%2529-3-%25286-methyl-7-oxo-6%252C7-dihydro-1H-pyrrolo%255B2%252C3-c%255Dpyridin-4-yl%2529phenyl%2529ethanesulfonamide%2520%2528ABBV-075%252FMivebresib%2529%252C%2520a%2520potent%2520and%2520orally%2520available%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span> </span><span class="NLM_article-title">Dose escalation
study of GSK2820151 in subjects with advanced or
recurrent solid tumors</span>; <span class="NLM_publisher-name">Clinical trials</span> <a href="https://clinicaltrials.gov/ct2/show/NCT02630251" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02630251</a> (accessed Nov 30, 2015).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Dose+escalation%0Astudy+of+GSK2820151+in+subjects+with+advanced+or%0Arecurrent+solid+tumors%3B+Clinical+trials+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02630251+%28accessed+Nov+30%2C+2015%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DDose%2520escalation%250Astudy%2520of%2520GSK2820151%2520in%2520subjects%2520with%2520advanced%2520or%250Arecurrent%2520solid%2520tumors%26pub%3DClinical%2520trials" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+physical+in+drug+discovery+II%3A+the+impact+of+chromatographic+hydrophobicity+measurements+and+aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0lhm-3B-X4W1kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%2520II%253A%2520the%2520impact%2520of%2520chromatographic%2520hydrophobicity%2520measurements%2520and%2520aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palovich, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. J.</span></span> <span> </span><span class="NLM_article-title">Quality guidelines for oral drug candidates: dose, solubility and lipophilicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1719</span>– <span class="NLM_lpage">1727</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.drudis.2016.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27423371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1aisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1719-1727&author=M.+K.+Baylissauthor=J.+Butlerauthor=P.+L.+Feldmanauthor=D.+V.+S.+Greenauthor=P.+D.+Leesonauthor=M.+R.+Palovichauthor=A.+J.+Taylor&title=Quality+guidelines+for+oral+drug+candidates%3A+dose%2C+solubility+and+lipophilicity&doi=10.1016%2Fj.drudis.2016.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Quality guidelines for oral drug candidates: dose, solubility and lipophilicity</span></div><div class="casAuthors">Bayliss, Martin K.; Butler, James; Feldman, Paul L.; Green, Darren V. S.; Leeson, Paul D.; Palovich, Michael R.; Taylor, Anthony J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1719-1727</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In an attempt to seek increased understanding of compd. attributes that influence successful drug pipeline progression, GlaxoSmithKline's portfolio of oral candidates was compared with ref. sets of marketed oral drugs.  The approach differs from other attrition studies by explicitly focusing on choosing 'the right compd.' by applying relevant, exptl. derived properties.  The anal. led to four proposed compd. quality categories, created by combining specific criteria for three measures: dose, soly. and the property forecast index, a composite measure of lipophilicity using chromatog. detd. Log D and aromaticity.  The 'three properties' provide benchmarked guidelines for project teams to use when seeking and selecting clin. candidates, because they reflect the property distribution of marketed oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRMpjV3EZ_4rVg90H21EOLACvtfcHk0lhm-3B-X4W1kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1aisb%252FK&md5=d9d145b1ee399b6ad54cdc55fff43c48</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DBayliss%26aufirst%3DM.%2BK.%26aulast%3DButler%26aufirst%3DJ.%26aulast%3DFeldman%26aufirst%3DP.%2BL.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DPalovich%26aufirst%3DM.%2BR.%26aulast%3DTaylor%26aufirst%3DA.%2BJ.%26atitle%3DQuality%2520guidelines%2520for%2520oral%2520drug%2520candidates%253A%2520dose%252C%2520solubility%2520and%2520lipophilicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1719%26epage%3D1727%26doi%3D10.1016%2Fj.drudis.2016.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franzini, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded chemical libraries: advancing beyond conventional small-molecule libraries</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1021/ar400284t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar400284t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltF2ls7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2014&pages=1247-1255&author=R.+M.+Franziniauthor=D.+Neriauthor=J.+Scheuermann&title=DNA-encoded+chemical+libraries%3A+advancing+beyond+conventional+small-molecule+libraries&doi=10.1021%2Far400284t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Chemical Libraries: Advancing beyond Conventional Small-Molecule Libraries</span></div><div class="casAuthors">Franzini, Raphael M.; Neri, Dario; Scheuermann, Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1255</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA-encoded chem. libraries (DECLs) represent a promising tool in drug discovery.  DECL technol. allows the synthesis and screening of chem. libraries of unprecedented size at moderate costs.  In analogy to phage-display technol., where large antibody libraries are displayed on the surface of filamentous phage and are genetically encoded in the phage genome, DECLs feature the display of individual small org. chem. moieties on DNA fragments serving as amplifiable identification barcodes.  The DNA-tag facilitates the synthesis and allows the simultaneous screening of very large sets of compds. (up to billions of mols.), because the hit compds. can easily be identified and quantified by PCR-amplification of the DNA-barcode followed by high-throughput DNA sequencing.  Several approaches have been used to generate DECLs, differing both in the methods used for library encoding and for the combinatorial assembly of chem. moieties.  For example, DECLs can be used for fragment-based drug discovery, displaying a single mol. on DNA or two chem. moieties at the extremities of complementary DNA strands.  DECLs can vary substantially in the chem. structures and the library size.  While ultralarge libraries contg. billions of compds. have been reported contg. four or more sets of building blocks, also smaller libraries have been shown to be efficient for ligand discovery.  In general, it has been found that the overall library size is a poor predictor for library performance and that the no. and diversity of the building blocks are rather important indicators.  Smaller libraries consisting of two to three sets of building blocks better fulfill the criteria of drug-likeness and often have higher quality.  In this Account, we present advances in the DECL field from proof-of-principle studies to practical applications for drug discovery, both in industry and in academia.  DECL technol. can yield specific binders to a variety of target proteins and is likely to become a std. tool for pharmaceutical hit discovery, lead expansion, and Chem. Biol. research.  The introduction of new methodologies for library encoding and for compd. synthesis in the presence of DNA is an exciting research field and will crucially contribute to the performance and the propagation of the technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcaCmssNgPLLVg90H21EOLACvtfcHk0lhm-3B-X4W1kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltF2ls7g%253D&md5=c37bd1625b668e869fcc26d5ef0a017a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Far400284t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far400284t%26sid%3Dliteratum%253Aachs%26aulast%3DFranzini%26aufirst%3DR.%2BM.%26aulast%3DNeri%26aufirst%3DD.%26aulast%3DScheuermann%26aufirst%3DJ.%26atitle%3DDNA-encoded%2520chemical%2520libraries%253A%2520advancing%2520beyond%2520conventional%2520small-molecule%2520libraries%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2014%26volume%3D47%26spage%3D1247%26epage%3D1255%26doi%3D10.1021%2Far400284t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleiner, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. R.</span></span> <span> </span><span class="NLM_article-title">Small-molecule discovery from DNA-encoded chemical libraries</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">5707</span>– <span class="NLM_lpage">5717</span>, <span class="refDoi"> DOI: 10.1039/c1cs15076f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1039%2Fc1cs15076f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=21674077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGrsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=5707-5717&author=R.+E.+Kleinerauthor=C.+E.+Dumelinauthor=D.+R.+Liu&title=Small-molecule+discovery+from+DNA-encoded+chemical+libraries&doi=10.1039%2Fc1cs15076f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule discovery from DNA-encoded chemical libraries</span></div><div class="casAuthors">Kleiner, Ralph E.; Dumelin, Christoph E.; Liu, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5707-5717</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Researchers seeking to improve the efficiency and cost effectiveness of the bioactive small-mol. discovery process have recently embraced selection-based approaches, which in principle offer much higher throughput and simpler infrastructure requirements compared with traditional small-mol. screening methods.  Since selection methods benefit greatly from an information-encoding mol. that can be readily amplified and decoded, several academic and industrial groups have turned to DNA as the basis for library encoding and, in some cases, library synthesis.  The resulting DNA-encoded synthetic small-mol. libraries, integrated with the high sensitivity of PCR and the recent development of ultra high-throughput DNA sequencing technol., can be evaluated very rapidly for binding or bond formation with a target of interest while consuming minimal quantities of material and requiring only modest investments of time and equipment.  In this tutorial review we describe the development of two classes of approaches for encoding chem. structures and reactivity with DNA: DNA-recorded library synthesis, in which encoding and library synthesis take place sep., and DNA-directed library synthesis, in which DNA both encodes and templates library synthesis.  We also describe in vitro selection methods used to evaluate DNA-encoded libraries and summarize successful applications of these approaches to the discovery of bioactive small mols. and novel chem. reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRlmzulmrgLLVg90H21EOLACvtfcHk0lhW2z0eEbN1gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGrsb3F&md5=b39426ff390f49581a22cb82215fc242</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1039%2Fc1cs15076f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cs15076f%26sid%3Dliteratum%253Aachs%26aulast%3DKleiner%26aufirst%3DR.%2BE.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DLiu%26aufirst%3DD.%2BR.%26atitle%3DSmall-molecule%2520discovery%2520from%2520DNA-encoded%2520chemical%2520libraries%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D5707%26epage%3D5717%26doi%3D10.1039%2Fc1cs15076f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjamin, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creaser, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzen, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gefter, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavarana, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vloten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1038/nchembio.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+V.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kelleyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries&doi=10.1038%2Fnchembio.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lhW2z0eEbN1gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelley%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3Dvan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654%26doi%3D10.1038%2Fnchembio.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundersingh, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ancellin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedard, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beljean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delorey, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grady, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herry, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushansky, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">Discovery, SAR, and x-ray binding mode study of BCATm inhibitors from a novel DNA-encoded library</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">924</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00179</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00179" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=919-924&author=H.+Dengauthor=J.+Zhouauthor=F.+S.+Sundersinghauthor=J.+Summerfieldauthor=D.+Somersauthor=J.+A.+Messerauthor=A.+L.+Satzauthor=N.+Ancellinauthor=C.+C.+Arico-Muendelauthor=K.+L.+Bedardauthor=A.+Beljeanauthor=S.+L.+Belyanskayaauthor=R.+Binghamauthor=S.+E.+Smithauthor=E.+Boursierauthor=P.+Carterauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.-w.+Chungauthor=C.+P.+Davieauthor=J.+L.+Deloreyauthor=Y.+Dingauthor=G.+J.+Franklinauthor=L.+C.+Gradyauthor=K.+Herryauthor=C.+Hobbsauthor=C.+S.+Kollmannauthor=B.+A.+Morganauthor=L.+J.+Kaushanskyauthor=Q.+Zhou&title=Discovery%2C+SAR%2C+and+x-ray+binding+mode+study+of+BCATm+inhibitors+from+a+novel+DNA-encoded+library&doi=10.1021%2Facsmedchemlett.5b00179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, SAR, and X-ray Binding Mode Study of BCATm Inhibitors from a Novel DNA-Encoded Library</span></div><div class="casAuthors">Deng, Hongfeng; Zhou, Jingye; Sundersingh, Flora S.; Summerfield, Jennifer; Somers, Don; Messer, Jeffrey A.; Satz, Alexander L.; Ancellin, Nicolas; Arico-Muendel, Christopher C.; Bedard, Katie L.; Beljean, Arthur; Belyanskaya, Svetlana L.; Bingham, Ryan; Smith, Sarah E.; Boursier, Eric; Carter, Paul; Centrella, Paolo A.; Clark, Matthew A.; Chung, Chun-wa; Davie, Christopher P.; Delorey, Jennifer L.; Ding, Yun; Franklin, G. Joseph; Grady, LaShadric C.; Herry, Kenny; Hobbs, Clare; Kollmann, Christopher S.; Morgan, Barry A.; Kaushansky, Laura J.; Zhou, Quan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">919-924</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As a potential target for obesity, human BCATm was screened against more than 14 billion DNA encoded compds. of distinct scaffolds followed by off-DNA synthesis and activity confirmation.  As a consequence, several series of BCATm inhibitors were discovered.  One representative compd. I from a novel compd. library synthesized via on-DNA Suzuki-Miyaura cross-coupling showed BCATm inhibitory activity with IC50 = 2.0 μM.  A protein crystal structure of I revealed that it binds to BCATm within the catalytic site adjacent to the PLP cofactor.  The identification of this novel inhibitor series plus the establishment of a BCATm protein structure provided a good starting point for future structure-based discovery of BCATm inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxzUuENOvoCrVg90H21EOLACvtfcHk0lj55zN4v0pFhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lt77L&md5=85fb50b6ee9c5b4ed3d1afd0bc9face6</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00179%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DSundersingh%26aufirst%3DF.%2BS.%26aulast%3DSummerfield%26aufirst%3DJ.%26aulast%3DSomers%26aufirst%3DD.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DAncellin%26aufirst%3DN.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBedard%26aufirst%3DK.%2BL.%26aulast%3DBeljean%26aufirst%3DA.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBingham%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DS.%2BE.%26aulast%3DBoursier%26aufirst%3DE.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDelorey%26aufirst%3DJ.%2BL.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DGrady%26aufirst%3DL.%2BC.%26aulast%3DHerry%26aufirst%3DK.%26aulast%3DHobbs%26aufirst%3DC.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26aulast%3DKaushansky%26aufirst%3DL.%2BJ.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DDiscovery%252C%2520SAR%252C%2520and%2520x-ray%2520binding%2520mode%2520study%2520of%2520BCATm%2520inhibitors%2520from%2520a%2520novel%2520DNA-encoded%2520library%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D919%26epage%3D924%26doi%3D10.1021%2Facsmedchemlett.5b00179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundersingh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beljean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blazensky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herry, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphries, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ancellin, N.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of potent, selective, and in vivo efficacious 2-aryl benzimidazole BCATm inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00389</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Kru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=379-384&author=H.+Dengauthor=J.+Zhouauthor=F.+Sundersinghauthor=J.+A.+Messerauthor=D.+O.+Somersauthor=M.+Ajakaneauthor=C.+C.+Arico-Muendelauthor=A.+Beljeanauthor=S.+L.+Belyanskayaauthor=R.+Binghamauthor=E.+Blazenskyauthor=A.-B.+Boullayauthor=E.+Boursierauthor=J.+Chaiauthor=P.+Carterauthor=C.-W.+Chungauthor=A.+Dauganauthor=Y.+Dingauthor=K.+Herryauthor=C.+Hobbsauthor=E.+Humphriesauthor=C.+Kollmannauthor=V.+L.+Nguyenauthor=E.+Nicodemeauthor=S.+E.+Smithauthor=N.+Dodicauthor=N.+Ancellin&title=Discovery+and+optimization+of+potent%2C+selective%2C+and+in+vivo+efficacious+2-aryl+benzimidazole+BCATm+inhibitors&doi=10.1021%2Facsmedchemlett.5b00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Potent, Selective, and in Vivo Efficacious 2-Aryl Benzimidazole BCATm Inhibitors</span></div><div class="casAuthors">Deng, Hongfeng; Zhou, Jingye; Sundersingh, Flora; Messer, Jeffrey A.; Somers, Donald O.; Ajakane, Myriam; Arico-Muendel, Christopher C.; Beljean, Arthur; Belyanskaya, Svetlana L.; Bingham, Ryan; Blazensky, Emily; Boullay, Anne-Benedicte; Boursier, Eric; Chai, Jing; Carter, Paul; Chung, Chun-Wa; Daugan, Alain; Ding, Yun; Herry, Kenny; Hobbs, Clare; Humphries, Eric; Kollmann, Christopher; Nguyen, Van Loc; Nicodeme, Edwige; Smith, Sarah E.; Dodic, Nerina; Ancellin, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-384</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To identify BCATm inhibitors suitable for in vivo study, Encoded Library Technol. (ELT) was used to affinity screen a 117 million member benzimidazole based DNA encoded library, which identified an inhibitor series with both biochem. and cellular activities.  Subsequent SAR studies led to the discovery of a highly potent and selective compd. I with much improved PK properties.  X-ray structure revealed that I binds to the active site of BACTm in a unique mode via multiple H-bond and van der Waals interactions.  After oral administration, I raised mouse blood levels of all three branched chain amino acids as a consequence of BCATm inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXwx-YRCyoe7Vg90H21EOLACvtfcHk0lj55zN4v0pFhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Kru7g%253D&md5=3d6116d0a992d0b3d051098b40eb179e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00389%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DSundersingh%26aufirst%3DF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DSomers%26aufirst%3DD.%2BO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBeljean%26aufirst%3DA.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBingham%26aufirst%3DR.%26aulast%3DBlazensky%26aufirst%3DE.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DBoursier%26aufirst%3DE.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DHerry%26aufirst%3DK.%26aulast%3DHobbs%26aufirst%3DC.%26aulast%3DHumphries%26aufirst%3DE.%26aulast%3DKollmann%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DSmith%26aufirst%3DS.%2BE.%26aulast%3DDodic%26aufirst%3DN.%26aulast%3DAncellin%26aufirst%3DN.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520potent%252C%2520selective%252C%2520and%2520in%2520vivo%2520efficacious%25202-aryl%2520benzimidazole%2520BCATm%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D379%26epage%3D384%26doi%3D10.1021%2Facsmedchemlett.5b00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gondo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLorey, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Development and synthesis of DNA-encoded benzimidazole library</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.8b00009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.8b00009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFSgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=251-255&author=Y.+Dingauthor=J.+Chaiauthor=P.+A.+Centrellaauthor=C.+Gondoauthor=J.+L.+DeLoreyauthor=M.+A.+Clark&title=Development+and+synthesis+of+DNA-encoded+benzimidazole+library&doi=10.1021%2Facscombsci.8b00009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Synthesis of DNA-Encoded Benzimidazole Library</span></div><div class="casAuthors">Ding, Yun; Chai, Jing; Centrella, Paolo A.; Gondo, Chenaimwoyo; De Lorey, Jennifer L.; Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">251-255</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Encoded library technol. (ELT) is an effective approach to the discovery of novel small-mol. ligands for biol. targets.  A key factor for the success of the technol. is the chem. diversity of the libraries.  Here we report the development of DNA-conjugated benzimidazoles.  Using 4-fluoro-3-nitrobenzoic acid as a key synthon, we synthesized a 320 million-member DNA-encoded benzimidazole library using Fmoc-protected amino acids, amines and aldehydes as diversity elements.  Affinity selection of the library led to the discovery of a novel, potent and specific antagonist of the NK3 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj58UnFYV0b7Vg90H21EOLACvtfcHk0lid0hCYFhRGjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFSgt7Y%253D&md5=2a8db7621b99bf9390c446dc0fc56780</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.8b00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.8b00009%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DY.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DGondo%26aufirst%3DC.%26aulast%3DDeLorey%26aufirst%3DJ.%2BL.%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DDevelopment%2520and%2520synthesis%2520of%2520DNA-encoded%2520benzimidazole%2520library%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2018%26volume%3D20%26spage%3D251%26epage%3D255%26doi%3D10.1021%2Facscombsci.8b00009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldino, C.</span></span> <span> </span><span class="NLM_article-title">A versatile method for the synthesis of benzimidazoles from o-nitroanilines and aldehydes in one step via a reductive cyclization</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1055/s-2004-834926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1055%2Fs-2004-834926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvFSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=47-56&author=D.+Yangauthor=D.+Fokasauthor=J.+Liauthor=L.+Yuauthor=C.+Baldino&title=A+versatile+method+for+the+synthesis+of+benzimidazoles+from+o-nitroanilines+and+aldehydes+in+one+step+via+a+reductive+cyclization&doi=10.1055%2Fs-2004-834926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A versatile method for the synthesis of benzimidazoles from o-nitroanilines and aldehydes in one step via a reductive cyclization</span></div><div class="casAuthors">Yang, Donglai; Fokas, Demosthenes; Li, Jingzhou; Yu, Libing; Baldino, Carmen M.</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-56</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">0039-7881</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A highly efficient and versatile method for the synthesis of benzimidazoles, e.g., I, was achieved in one step by the Na2S2O4 redn. of o-nitroanilines in the presence of aldehydes.  Heating a soln. of o-nitroaniline and an aldehyde, in the presence of aq. or solid Na2S2O4, provided facile access to a series of 2-substituted N-H benzimidazoles contg. a wide range of functional groups not always compatible with the existing synthetic methods.  This methodol. has also been applied to the regioselective synthesis of N-alkyl and N-aryl benzimidazoles by the cyclization of the corresponding N-substituted nitroanilines, resp.  In addn., the method was applied successfully to the synthesis of other imidazole contg. heterocyclic ring systems such as 1H-imidazo[4,5-b]pyridines and 1H-imidazo[4,5-f]quinoline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo731FJUu4mCLVg90H21EOLACvtfcHk0lid0hCYFhRGjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvFSquw%253D%253D&md5=9459f8a5a5235627a318997d8af5b3ec</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1055%2Fs-2004-834926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-2004-834926%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DD.%26aulast%3DFokas%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DBaldino%26aufirst%3DC.%26atitle%3DA%2520versatile%2520method%2520for%2520the%2520synthesis%2520of%2520benzimidazoles%2520from%2520o-nitroanilines%2520and%2520aldehydes%2520in%2520one%2520step%2520via%2520a%2520reductive%2520cyclization%26jtitle%3DSynthesis%26date%3D2005%26spage%3D47%26epage%3D56%26doi%3D10.1055%2Fs-2004-834926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of potent diazobenzene inhibitors for the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9251</span>– <span class="NLM_lpage">9264</span>, <span class="refDoi"> DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&author=G.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.-M.+Zhou&title=Structure-guided+design+of+potent+diazobenzene+inhibitors+for+the+BET+bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0lid0hCYFhRGjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure-guided%2520design%2520of%2520potent%2520diazobenzene%2520inhibitors%2520for%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3838</span>, <span class="refDoi"> DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.-w.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.-M.+G.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=A.-B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+characterization+of+small+molecule+inhibitors+of+the+BET+family+bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0liD4hcRZoRJ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520BET%2520family%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holenz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoy, P.</span></span> <span> </span><span class="NLM_article-title">Advances in lead generation</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2018.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=30606700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlsFOr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=517-524&author=J.+Holenzauthor=P.+Stoy&title=Advances+in+lead+generation&doi=10.1016%2Fj.bmcl.2018.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in Lead Generation</span></div><div class="casAuthors">Holenz, Joerg; Stoy, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">517-524</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lead Generation represents a crit. drug discovery phase where chem. starting points and their resp. mechanism of action, quality, and potential liabilities are largely predefined.  Recent advances such as DNA-encoded libraries or fragment-, chem. biol.-, and virtual screening-based approaches are today as common as traditional High Throughput Screening.  Innovations in characterizing lead quality have allowed more informed decision-making by discovery teams.  The key challenge today is to individually tailor the right mix of methods for each project to facilitate data integration with the purpose of creating multiple high-quality lead series, ultimately translating to reduced chem.-related pipeline attrition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa_L2ocMTSq7Vg90H21EOLACvtfcHk0liD4hcRZoRJ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlsFOr&md5=25444e43f6918fcc346e7d5fcb75c7d3</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DHolenz%26aufirst%3DJ.%26aulast%3DStoy%26aufirst%3DP.%26atitle%3DAdvances%2520in%2520lead%2520generation%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D517%26epage%3D524%26doi%3D10.1016%2Fj.bmcl.2018.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 1: inhibitor binding modes and implications for lead discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">576</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1021/jm201320w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201320w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=576-586&author=C.-w.+Chungauthor=A.+W.+Deanauthor=J.+M.+Woolvenauthor=P.+Bamborough&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+1%3A+inhibitor+binding+modes+and+implications+for+lead+discovery&doi=10.1021%2Fjm201320w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span></div><div class="casAuthors">Chung, Chun-wa; Dean, Anthony W.; Woolven, James M.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are key epigenetic regulators of gene transcription and readers of the histone code.  However, the therapeutic benefits of modulating this target class are largely unexplored due to the lack of suitable chem. probes.  This article describes the generation of lead mols. for the BET bromodomains through screening a fragment set chosen using structural insights and computational approaches.  Anal. of 40 BRD2/fragment x-ray complexes highlights both shared and disparate interaction features that may be exploited for affinity and selectivity.  Six representative crystal structures are then exemplified in detail.  Two of the fragments are completely new bromodomain chemotypes, and three have never before been crystd. in a bromodomain, so our results significantly extend the limited public knowledge-base of crystallog. small mol./bromodomain interactions.  Certain fragments (including paracetamol) bind in a consistent mode to different bromodomains such as CREBBP, suggesting their potential to act as generic bromodomain templates.  An important implication is that the bromodomains are not only a phylogenetic family but also a system in which chem. and structural knowledge of one bromodomain gives insights transferable to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonNb7GnZnS_rVg90H21EOLACvtfcHk0liD4hcRZoRJ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN&md5=5b7d7239d0de57266d94beef1ea080ae</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm201320w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201320w%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDean%26aufirst%3DA.%2BW.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25201%253A%2520inhibitor%2520binding%2520modes%2520and%2520implications%2520for%2520lead%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D576%26epage%3D586%26doi%3D10.1021%2Fjm201320w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodacre, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodrow, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 2: optimization of phenylisoxazole sulfonamides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">587</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1021/jm201283q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201283q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=587-596&author=P.+Bamboroughauthor=H.+Dialloauthor=J.+D.+Goodacreauthor=L.+Gordonauthor=A.+Lewisauthor=J.+T.+Sealauthor=D.+M.+Wilsonauthor=M.+D.+Woodrowauthor=C.-w.+Chung&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+2%3A+optimization+of+phenylisoxazole+sulfonamides&doi=10.1021%2Fjm201283q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 2: Optimization of Phenylisoxazole Sulfonamides</span></div><div class="casAuthors">Bamborough, Paul; Diallo, Hawa; Goodacre, Jonathan D.; Gordon, Laurie; Lewis, Antonia; Seal, Jonathan T.; Wilson, David M.; Woodrow, Michael D.; Chung, Chun-wa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">587-596</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic reader modules that regulate gene transcription through their recognition of acetyl-lysine modified histone tails.  Inhibitors of this protein-protein interaction have the potential to modulate multiple diseases as demonstrated by the profound anti-inflammatory and antiproliferative effects of a recently disclosed class of BET compds.  While these compds. were discovered using phenotypic assays, here we present a highly efficient alternative approach to find new chem. templates, exploiting the abundant structural knowledge that exists for this target class.  A Ph di-Me isoxazole chemotype resulting from a focused fragment screen has been rapidly optimized through structure-based design, leading to a sulfonamide series showing anti-inflammatory activity in cellular assays.  This proof-of-principle expt. demonstrates the tractability of the BET family and bromodomain target class to fragment-based hit discovery and structure-based lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr6lsic9qjJrVg90H21EOLACvtfcHk0lhQa2tuGft7CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF&md5=e73b16b3fddf88b43b037264b000b276</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm201283q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201283q%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DGoodacre%26aufirst%3DJ.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWoodrow%26aufirst%3DM.%2BD.%26aulast%3DChung%26aufirst%3DC.-w.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25202%253A%2520optimization%2520of%2520phenylisoxazole%2520sulfonamides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D587%26epage%3D596%26doi%3D10.1021%2Fjm201283q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bogdan, A.</span>; <span class="NLM_string-name">Kati, W. M.</span>; <span class="NLM_string-name">McDaniel, K. F.</span>; <span class="NLM_string-name">Park, C. H.</span>; <span class="NLM_string-name">Sheppard, G. S.</span>; <span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyridinyl Containing Compounds as Bromodomain Inhibitors for Therapy</span>. U.S. Patent <span class="NLM_patent">20150158873A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=A.+Bogdan&author=W.+M.+Kati&author=K.+F.+McDaniel&author=C.+H.+Park&author=G.+S.+Sheppard&author=L.+Wang&title=Preparation+of+Pyridinyl+Containing+Compounds+as+Bromodomain+Inhibitors+for+Therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBogdan%26aufirst%3DA.%26atitle%3DPreparation%2520of%2520Pyridinyl%2520Containing%2520Compounds%2520as%2520Bromodomain%2520Inhibitors%2520for%2520Therapy%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, H.</span>; <span class="NLM_string-name">Martin, L.</span>; <span class="NLM_string-name">Smethurst, C.</span></span> <span> </span><span class="NLM_article-title">Preparation of Substituted Pyridinones as Anticancer Agents</span>. <span class="NLM_patent">WO2015022332A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=H.+Engelhardt&author=L.+Martin&author=C.+Smethurst&title=Preparation+of+Substituted+Pyridinones+as+Anticancer+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEngelhardt%26aufirst%3DH.%26atitle%3DPreparation%2520of%2520Substituted%2520Pyridinones%2520as%2520Anticancer%2520Agents%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Frey, R. R.</span>; <span class="NLM_string-name">Hansen, T. M.</span>; <span class="NLM_string-name">Liu, D.</span>; <span class="NLM_string-name">McClellan, W. J.</span>; <span class="NLM_string-name">McDaniel, K. F.</span>; <span class="NLM_string-name">Pratt, J. K.</span>; <span class="NLM_string-name">Wada, C. K.</span></span> <span> </span><span class="NLM_article-title">Preparation of Multicyclic Compounds as Bromodomain Inhibitors</span>. <span class="NLM_patent">WO2015085925A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=L.+Wang&author=R.+R.+Frey&author=T.+M.+Hansen&author=D.+Liu&author=W.+J.+McClellan&author=K.+F.+McDaniel&author=J.+K.+Pratt&author=C.+K.+Wada&title=Preparation+of+Multicyclic+Compounds+as+Bromodomain+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26atitle%3DPreparation%2520of%2520Multicyclic%2520Compounds%2520as%2520Bromodomain%2520Inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, H.</span></span> <span> </span><span class="NLM_article-title">Benzimidazole Derivatives as BRD4 Inhibitors and their Preparation and Use for the Treatment of Cancer</span>. <span class="NLM_patent">WO2015169962A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=H.+Engelhardt&title=Benzimidazole+Derivatives+as+BRD4+Inhibitors+and+their+Preparation+and+Use+for+the+Treatment+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEngelhardt%26aufirst%3DH.%26atitle%3DBenzimidazole%2520Derivatives%2520as%2520BRD4%2520Inhibitors%2520and%2520their%2520Preparation%2520and%2520Use%2520for%2520the%2520Treatment%2520of%2520Cancer%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holms, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClellan, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Fragment-based, structure-enabled discovery of novel pyridones and pyridone macrocycles as potent bromodomain and extra-terminal domain (BET) family bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3828</span>– <span class="NLM_lpage">3850</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00017</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00017" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsVygu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3828-3850&author=L.+Wangauthor=J.+K.+Prattauthor=T.+Soltwedelauthor=G.+S.+Sheppardauthor=S.+D.+Fidanzeauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=R.+A.+Manteiauthor=J.+H.+Holmsauthor=W.+J.+McClellanauthor=M.+D.+Wendtauthor=C.+Wadaauthor=R.+Freyauthor=T.+M.+Hansenauthor=R.+Hubbardauthor=C.+H.+Parkauthor=L.+Liauthor=T.+J.+Magocauthor=D.+H.+Albertauthor=X.+Linauthor=S.+E.+Warderauthor=P.+Kovarauthor=X.+Huangauthor=D.+Wilcoxauthor=R.+Wangauthor=G.+Rajaramanauthor=A.+M.+Petrosauthor=C.+W.+Hutchinsauthor=S.+C.+Panchalauthor=C.+Sunauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Fragment-based%2C+structure-enabled+discovery+of+novel+pyridones+and+pyridone+macrocycles+as+potent+bromodomain+and+extra-terminal+domain+%28BET%29+family+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.7b00017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors</span></div><div class="casAuthors">Wang, Le; Pratt, John K.; Soltwedel, Todd; Sheppard, George S.; Fidanze, Steven D.; Liu, Dachun; Hasvold, Lisa A.; Mantei, Robert A.; Holms, James H.; McClellan, William J.; Wendt, Michael D.; Wada, Carol; Frey, Robin; Hansen, T. Matthew; Hubbard, Robert; Park, Chang H.; Li, Leiming; Magoc, Terrance J.; Albert, Daniel H.; Lin, Xiaoyu; Warder, Scott E.; Kovar, Peter; Huang, Xiaoli; Wilcox, Denise; Wang, Rongqi; Rajaraman, Ganesh; Petros, Andrew M.; Hutchins, Charles W.; Panchal, Sanjay C.; Sun, Chaohong; Elmore, Steven W.; Shen, Yu; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3828-3850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the BET family of bromodomain contg. proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines.  A 2-dimensional NMR fragment screen for binders to the bromodomains of BRD4 identified a Ph pyridazinone fragment with a weak binding affinity (I, Ki = 160 μM).  SAR investigation of fragment I, aided by X-ray structure-based design, enabled the synthesis of potent pyridone and macrocyclic pyridone inhibitors exhibiting single digit nanomolar potency in both biochem. and cell based assays, e.g. II.  Advanced analogs in these series exhibited high oral exposures in rodent PK studies and demonstrated significant tumor growth inhibition efficacy in mouse flank xenograft models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDaiYdyL4LtrVg90H21EOLACvtfcHk0lhQa2tuGft7CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsVygu7k%253D&md5=91750e660bafec08b8bcbe6bcd5e818f</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00017%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DHolms%26aufirst%3DJ.%2BH.%26aulast%3DMcClellan%26aufirst%3DW.%2BJ.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DWada%26aufirst%3DC.%26aulast%3DFrey%26aufirst%3DR.%26aulast%3DHansen%26aufirst%3DT.%2BM.%26aulast%3DHubbard%26aufirst%3DR.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DFragment-based%252C%2520structure-enabled%2520discovery%2520of%2520novel%2520pyridones%2520and%2520pyridone%2520macrocycles%2520as%2520potent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3828%26epage%3D3850%26doi%3D10.1021%2Facs.jmedchem.7b00017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postepska-Igielska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich-Bode, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukamp, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grummt, I.</span></span> <span> </span><span class="NLM_article-title">The chromatin remodelling complex NoRC safeguards genome stability by heterochromatin formation at telomeres and centromeres</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">704</span>– <span class="NLM_lpage">710</span>, <span class="refDoi"> DOI: 10.1038/embor.2013.87</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fembor.2013.87" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23797874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvFeqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=704-710&author=A.+Postepska-Igielskaauthor=D.+Krunicauthor=N.+Schmittauthor=K.+M.+Greulich-Bodeauthor=P.+Boukampauthor=I.+Grummt&title=The+chromatin+remodelling+complex+NoRC+safeguards+genome+stability+by+heterochromatin+formation+at+telomeres+and+centromeres&doi=10.1038%2Fembor.2013.87"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">The chromatin remodelling complex NoRC safeguards genome stability by heterochromatin formation at telomeres and centromeres</span></div><div class="casAuthors">Postepska-Igielska, Anna; Krunic, Damir; Schmitt, Nina; Greulich-Bode, Karin M.; Boukamp, Petra; Grummt, Ingrid</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">704-710</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Constitutive heterochromatin is crucial for the integrity of chromosomes and genomic stability.  Here, we show that the chromatin remodelling complex NoRC, known to silence a fraction of rRNA genes, also establishes a repressive heterochromatic structure at centromeres and telomeres, preserving the structural integrity of these repetitive loci.  Knockdown of NoRC leads to relaxation of centromeric and telomeric heterochromatin, abnormalities in mitotic spindle assembly, impaired chromosome segregation and enhanced chromosomal instability.  The results demonstrate that NoRC safeguards genomic stability by coordinating enzymic activities that establish features of repressive chromatin at centromeric and telomeric regions, and this heterochromatic structure is required for sustaining genomic integrity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAXnCQyfI2xLVg90H21EOLACvtfcHk0libs9c71lr4og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvFeqtbo%253D&md5=2dfa275fa4fc7c02eadfddc90982fedc</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fembor.2013.87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2013.87%26sid%3Dliteratum%253Aachs%26aulast%3DPostepska-Igielska%26aufirst%3DA.%26aulast%3DKrunic%26aufirst%3DD.%26aulast%3DSchmitt%26aufirst%3DN.%26aulast%3DGreulich-Bode%26aufirst%3DK.%2BM.%26aulast%3DBoukamp%26aufirst%3DP.%26aulast%3DGrummt%26aufirst%3DI.%26atitle%3DThe%2520chromatin%2520remodelling%2520complex%2520NoRC%2520safeguards%2520genome%2520stability%2520by%2520heterochromatin%2520formation%2520at%2520telomeres%2520and%2520centromeres%26jtitle%3DEMBO%2520Rep.%26date%3D2013%26volume%3D14%26spage%3D704%26epage%3D710%26doi%3D10.1038%2Fembor.2013.87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosea, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, R. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakar, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, K.</span></span> <span> </span><span class="NLM_article-title">Prediction of human pharmacokinetics from preclinical information: comparative accuracy of quantitative prediction approaches</span>. <i>Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1177/0091270009333209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1177%2F0091270009333209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsFWmu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=513-533&author=N.+A.+Hoseaauthor=W.+T.+Collardauthor=S.+Coleauthor=T.+S.+Maurerauthor=R.+X.+Fangauthor=H.+Jonesauthor=S.+M.+Kakarauthor=Y.+Nakaiauthor=B.+J.+Smithauthor=R.+Websterauthor=K.+Beaumont&title=Prediction+of+human+pharmacokinetics+from+preclinical+information%3A+comparative+accuracy+of+quantitative+prediction+approaches&doi=10.1177%2F0091270009333209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human pharmacokinetics from preclinical information: comparative accuracy of quantitative prediction approaches</span></div><div class="casAuthors">Hosea, Natilie A.; Collard, Wendy T.; Cole, Susan; Maurer, Tristan S.; Fang, Rick X.; Jones, Hannah; Kakar, Shefali M.; Nakai, Yasuhiro; Smith, Bill J.; Webster, Rob; Beaumont, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">513-533</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Quant. prediction of human pharmacokinetics is crit. in assessing the viability of drug candidates and in detg. first-in-human dosing.  Numerous prediction methodologies, incorporating both in vitro and preclin. in vivo data, have been developed in recent years, each with advantages and disadvantages.  However, the lack of a comprehensive data set, both preclin. and clin., has limited efforts to evaluate the optimal strategy (or strategies) that results in quant. predictions of human pharmacokinetics.  To address this issue, the authors conducted a retrospective anal. using 50 proprietary compds. for which in vitro, preclin. pharmacokinetic data and oral single-dose human pharmacokinetic data were available.  Five predictive strategies, involving either allometry or use of unbound intrinsic clearance from microsomes or hepatocytes, were then compared for their ability to predict human oral clearance, half-life through predictions of systemic clearance, vol. of distribution, and bioavailability.  Use of a single-species scaling approach with rat, dog, or monkey was as accurate as or more accurate than using multiple-species allometry.  For those compds. cleared almost exclusively by P 450-mediated pathways, scaling from human liver microsomes was as predictive as single-species scaling of clearance based on data from rat, dog, or monkey.  These data suggest that use of predictive methods involving either single-species in vivo data or in vitro human liver microsomes can quant. predict human in vivo pharmacokinetics and suggest the possibility of streamlining the predictive methodol. through use of a single species or use only of human in vitro microsomal prepns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg67HcwS4O4LVg90H21EOLACvtfcHk0libs9c71lr4og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsFWmu74%253D&md5=a0941d53fe43392f8fb1562ee546705e</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1177%2F0091270009333209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0091270009333209%26sid%3Dliteratum%253Aachs%26aulast%3DHosea%26aufirst%3DN.%2BA.%26aulast%3DCollard%26aufirst%3DW.%2BT.%26aulast%3DCole%26aufirst%3DS.%26aulast%3DMaurer%26aufirst%3DT.%2BS.%26aulast%3DFang%26aufirst%3DR.%2BX.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKakar%26aufirst%3DS.%2BM.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DB.%2BJ.%26aulast%3DWebster%26aufirst%3DR.%26aulast%3DBeaumont%26aufirst%3DK.%26atitle%3DPrediction%2520of%2520human%2520pharmacokinetics%2520from%2520preclinical%2520information%253A%2520comparative%2520accuracy%2520of%2520quantitative%2520prediction%2520approaches%26jtitle%3DClin.%2520Pharmacol.%26date%3D2009%26volume%3D49%26spage%3D513%26epage%3D533%26doi%3D10.1177%2F0091270009333209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahmood, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balian, J. D.</span></span> <span> </span><span class="NLM_article-title">Interspecies scaling: predicting clearance of drugs in humans. Three different approaches</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">887</span>– <span class="NLM_lpage">895</span>, <span class="refDoi"> DOI: 10.3109/00498259609052491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.3109%2F00498259609052491" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=8893036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaK28XmtVyntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1996&pages=887-895&author=I.+Mahmoodauthor=J.+D.+Balian&title=Interspecies+scaling%3A+predicting+clearance+of+drugs+in+humans.+Three+different+approaches&doi=10.3109%2F00498259609052491"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Interspecies scaling: predicting clearance of drugs in humans. Three different approaches</span></div><div class="casAuthors">Mahmood, I.; Balian, J. D.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">887-895</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis</span>)
        </div><div class="casAbstract">The interspecies scaling approach to predict clearance in humans from animal data was tested for a wide variety of drugs.  Three different methods were utilized to generate plots to scale-up the clearance values: (i) method I, clearance vs. body wt. (simple allometric equation); (ii) method II, product of clearance and max. life-span potential; (iii) method III, product of clearance and brain wt. vs. body wt.,.  The circumstances under which the three methods I predicts clearance reasonably well.  If the exponent lies between 0·55 to 0·77, then method I predicts clearance reasonably well.  If the exponent lies between 0·71 to 1·0, clearance can be predicted reasonably well by method II.  If the exponent is > 1·0, clearance can be predicted using method III.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps3KGdN89O0LVg90H21EOLACvtfcHk0libs9c71lr4og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtVyntrs%253D&md5=c7ed947e99fefabf08d3eaca7d2a08c5</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.3109%2F00498259609052491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498259609052491%26sid%3Dliteratum%253Aachs%26aulast%3DMahmood%26aufirst%3DI.%26aulast%3DBalian%26aufirst%3DJ.%2BD.%26atitle%3DInterspecies%2520scaling%253A%2520predicting%2520clearance%2520of%2520drugs%2520in%2520humans.%2520Three%2520different%2520approaches%26jtitle%3DXenobiotica%26date%3D1996%26volume%3D26%26spage%3D887%26epage%3D895%26doi%3D10.3109%2F00498259609052491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overvoorde, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svergun, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of histone tail recognition by human TIP5 PHD finger and bromodomain of the chromatin remodeling complex NoRC</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.str.2014.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.str.2014.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=25533489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvFWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=80-92&author=C.+Tallantauthor=E.+Valentiniauthor=O.+Fedorovauthor=L.+Overvoordeauthor=F.+M.+Fergusonauthor=P.+Filippakopoulosauthor=D.+I.+Svergunauthor=S.+Knappauthor=A.+Ciulli&title=Molecular+basis+of+histone+tail+recognition+by+human+TIP5+PHD+finger+and+bromodomain+of+the+chromatin+remodeling+complex+NoRC&doi=10.1016%2Fj.str.2014.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis of Histone Tail Recognition by Human TIP5 PHD Finger and Bromodomain of the Chromatin Remodeling Complex NoRC</span></div><div class="casAuthors">Tallant, Cynthia; Valentini, Erica; Fedorov, Oleg; Overvoorde, Lois; Ferguson, Fleur M.; Filippakopoulos, Panagis; Svergun, Dmitri I.; Knapp, Stefan; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-92</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Binding of the chromatin remodeling complex NoRC to RNA complementary to the rDNA promoter mediates transcriptional repression.  TIP5, the largest subunit of NoRC, is involved in recruitment to rDNA by interactions with promoter-bound TTF-I, pRNA, and acetylation of H4K16.  TIP5 domains that recognize posttranslational modifications on histones are essential for recruitment of NoRC to chromatin, but how these reader modules recognize site-specific histone tails has remained elusive.  Here, we report crystal structures of PHD zinc finger and bromodomains from human TIP5 and BAZ2B in free form and bound to H3 and/or H4 histones.  PHD finger functions as an independent structural module in recognizing unmodified H3 histone tails, and the bromodomain prefers H3 and H4 acetylation marks followed by a key basic residue, KacXXR.  Further low-resoln. analyses of PHD-bromodomain modules provide mol. insights into their trans histone tail recognition, required for nucleosome recruitment and transcriptional repression of the NoRC complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOrAPnFDwkzbVg90H21EOLACvtfcHk0lgWBPgyNLyxeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvFWrug%253D%253D&md5=2f891613f59dc2c584db33f475461e18</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DValentini%26aufirst%3DE.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DOvervoorde%26aufirst%3DL.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DSvergun%26aufirst%3DD.%2BI.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520basis%2520of%2520histone%2520tail%2520recognition%2520by%2520human%2520TIP5%2520PHD%2520finger%2520and%2520bromodomain%2520of%2520the%2520chromatin%2520remodeling%2520complex%2520NoRC%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D80%26epage%3D92%26doi%3D10.1016%2Fj.str.2014.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wuitschik, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers-Evans, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Märki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrilla, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carreira, E. M.</span></span> <span> </span><span class="NLM_article-title">Spirocyclic oxetanes: synthesis and properties</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">4512</span>– <span class="NLM_lpage">4515</span>, <span class="refDoi"> DOI: 10.1002/anie.200800450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1002%2Fanie.200800450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1yrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=4512-4515&author=G.+Wuitschikauthor=M.+Rogers-Evansauthor=A.+Bucklauthor=M.+Bernasconiauthor=M.+M%C3%A4rkiauthor=T.+Godelauthor=H.+Fischerauthor=B.+Wagnerauthor=I.+Parrillaauthor=F.+Schulerauthor=J.+Schneiderauthor=A.+Alkerauthor=W.+B.+Schweizerauthor=K.+M%C3%BCllerauthor=E.+M.+Carreira&title=Spirocyclic+oxetanes%3A+synthesis+and+properties&doi=10.1002%2Fanie.200800450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Spirocyclic oxetanes: synthesis and properties</span></div><div class="casAuthors">Wuitschik, Georg; Rogers-Evans, Mark; Buckl, Andreas; Bernasconi, Maurizio; Marki, Moritz; Godel, Thierry; Fischer, Holger; Wagner, Bjorn; Parrilla, Isabelle; Schuler, Franz; Schneider, Josef; Alker, Andre; Schweizer, W. Bernd; Muller, Klaus; Carreira, Erick M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4512-4515</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Spirocyclic oxetanes are described as analogs of morpholine and also as topol. siblings of their carbonyl counterparts.  They are particularly promising in terms of both their physicochem. properties and the ease with which they can be grafted onto mol. structures.  The data collected highlight oxetanes as both the hydrophilic sister of a gem-di-Me unit and the carbonyl group's lipophilic brother.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpjICUKXgpt7Vg90H21EOLACvtfcHk0lgWBPgyNLyxeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1yrurs%253D&md5=9522eb5cfb870b872d8f9c1f06ae2946</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2Fanie.200800450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200800450%26sid%3Dliteratum%253Aachs%26aulast%3DWuitschik%26aufirst%3DG.%26aulast%3DRogers-Evans%26aufirst%3DM.%26aulast%3DBuckl%26aufirst%3DA.%26aulast%3DBernasconi%26aufirst%3DM.%26aulast%3DM%25C3%25A4rki%26aufirst%3DM.%26aulast%3DGodel%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DH.%26aulast%3DWagner%26aufirst%3DB.%26aulast%3DParrilla%26aufirst%3DI.%26aulast%3DSchuler%26aufirst%3DF.%26aulast%3DSchneider%26aufirst%3DJ.%26aulast%3DAlker%26aufirst%3DA.%26aulast%3DSchweizer%26aufirst%3DW.%2BB.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26aulast%3DCarreira%26aufirst%3DE.%2BM.%26atitle%3DSpirocyclic%2520oxetanes%253A%2520synthesis%2520and%2520properties%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D4512%26epage%3D4515%26doi%3D10.1002%2Fanie.200800450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagdanoff, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madiera, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, D.</span></span> <span> </span><span class="NLM_article-title">Tetrahydropyrrolo-diazepenones as inhibitors of ERK2 kinase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3788</span>– <span class="NLM_lpage">3792</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.07.091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2015.07.091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=26259804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KrtLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=3788-3792&author=J.+T.+Bagdanoffauthor=R.+Jainauthor=W.+Hanauthor=S.+Zhuauthor=A.-M.+Madieraauthor=P.+S.+Leeauthor=X.+Maauthor=D.+Poon&title=Tetrahydropyrrolo-diazepenones+as+inhibitors+of+ERK2+kinase&doi=10.1016%2Fj.bmcl.2015.07.091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Tetrahydropyrrolo-diazepenones as inhibitors of ERK2 kinase</span></div><div class="casAuthors">Bagdanoff, Jeffrey T.; Jain, Rama; Han, Wooseok; Zhu, Shejin; Madiera, Ann-Marie; Lee, Patrick S.; Ma, Xiaolei; Poon, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3788-3792</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of structure based drug design hypotheses and focused screening efforts led to the identification of tetrahydropyrrolo-diazepenones with striking potency against ERK2 kinase.  The role of fluorination in mitigating microsomal clearance was systematically explored.  Ultimately, it was found that fluorination of a cyclopentanol substructure provided significant improvement in both potency and human metabolic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHXEHIlXy0QLVg90H21EOLACvtfcHk0lgWBPgyNLyxeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KrtLnF&md5=d94c52debb8203a3b9e655319ea32754</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.07.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.07.091%26sid%3Dliteratum%253Aachs%26aulast%3DBagdanoff%26aufirst%3DJ.%2BT.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DHan%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DMadiera%26aufirst%3DA.-M.%26aulast%3DLee%26aufirst%3DP.%2BS.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DPoon%26aufirst%3DD.%26atitle%3DTetrahydropyrrolo-diazepenones%2520as%2520inhibitors%2520of%2520ERK2%2520kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D3788%26epage%3D3792%26doi%3D10.1016%2Fj.bmcl.2015.07.091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stepan, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefer, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M.</span></span> <span> </span><span class="NLM_article-title">Evaluating the differences in cycloalkyl ether metabolism using the design parameter “lipophilic metabolism efficiency” (LipMetE) and a matched molecular pairs analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6985</span>– <span class="NLM_lpage">6990</span>, <span class="refDoi"> DOI: 10.1021/jm4008642</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4008642" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFKgurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6985-6990&author=A.+F.+Stepanauthor=G.+W.+Kauffmanauthor=C.+E.+Keeferauthor=P.+R.+Verhoestauthor=M.+Edwards&title=Evaluating+the+differences+in+cycloalkyl+ether+metabolism+using+the+design+parameter+%E2%80%9Clipophilic+metabolism+efficiency%E2%80%9D+%28LipMetE%29+and+a+matched+molecular+pairs+analysis&doi=10.1021%2Fjm4008642"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the Differences in Cycloalkyl Ether Metabolism Using the Design Parameter "Lipophilic Metabolism Efficiency" (LipMetE) and a Matched Molecular Pairs Analysis</span></div><div class="casAuthors">Stepan, Antonia F.; Kauffman, Gregory W.; Keefer, Christopher E.; Verhoest, Patrick R.; Edwards, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6985-6990</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have obsd. previously that modification of ring size and substitution pattern may be used as a strategy to mitigate the metabolic instability of cycloalkyl ethers.  In this article, we introduce a medicinal chem. design parameter named "lipophilic metab. efficiency" (LipMetE) that indicates that these changes in metabolic stability can be largely ascribed to changes in lipophilicity.  Our matched mol. pair anal. also indicates that this finding is a general phenomenon, widely obsd. across different chemotypes.  It is our hope that both the LipMetE design parameter and the results from our pairwise anal. will be useful tools for medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpiFJ_fPxurbVg90H21EOLACvtfcHk0liOzX5G_Dic_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFKgurvJ&md5=596673d35831b979b8daf5a4ccb8a218</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fjm4008642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4008642%26sid%3Dliteratum%253Aachs%26aulast%3DStepan%26aufirst%3DA.%2BF.%26aulast%3DKauffman%26aufirst%3DG.%2BW.%26aulast%3DKeefer%26aufirst%3DC.%2BE.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DEdwards%26aufirst%3DM.%26atitle%3DEvaluating%2520the%2520differences%2520in%2520cycloalkyl%2520ether%2520metabolism%2520using%2520the%2520design%2520parameter%2520%25E2%2580%259Clipophilic%2520metabolism%2520efficiency%25E2%2580%259D%2520%2528LipMetE%2529%2520and%2520a%2520matched%2520molecular%2520pairs%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6985%26epage%3D6990%26doi%3D10.1021%2Fjm4008642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottschall, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theodorides, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span> <span> </span><span class="NLM_article-title">The metabolism of benzimidazole anthelmintics</span>. <i>Parasitol. Today</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">124</span>, <span class="refDoi"> DOI: 10.1016/0169-4758(90)90228-v</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2F0169-4758%2890%2990228-v" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=15463313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaK3cXksFWru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1990&pages=115-124&author=D.+W.+Gottschallauthor=V.+J.+Theodoridesauthor=R.+Wang&title=The+metabolism+of+benzimidazole+anthelmintics&doi=10.1016%2F0169-4758%2890%2990228-v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">The metabolism of benzimidazole anthelmintics</span></div><div class="casAuthors">Gottschall, David W.; Theodorides, Vassilios J.; Wang, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Parasitology Today</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">115-24</span>CODEN:
                <span class="NLM_cas:coden">PATOE2</span>;
        ISSN:<span class="NLM_cas:issn">0169-4758</span>.
    </div><div class="casAbstract">A review with 26 refs. discussing the metab. of benzimidazole carbamates in lab. animals and humans and the problems of poor water soly. and absorption of these anthelmintics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG4gum4VKifrVg90H21EOLACvtfcHk0liOzX5G_Dic_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXksFWru7g%253D&md5=f5ffb20d83d4040275fa38121db7f6dc</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2F0169-4758%2890%2990228-v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0169-4758%252890%252990228-v%26sid%3Dliteratum%253Aachs%26aulast%3DGottschall%26aufirst%3DD.%2BW.%26aulast%3DTheodorides%26aufirst%3DV.%2BJ.%26aulast%3DWang%26aufirst%3DR.%26atitle%3DThe%2520metabolism%2520of%2520benzimidazole%2520anthelmintics%26jtitle%3DParasitol.%2520Today%26date%3D1990%26volume%3D6%26spage%3D115%26epage%3D124%26doi%3D10.1016%2F0169-4758%2890%2990228-v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiltshire, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heeps, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betty, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angus, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worth, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welker, H. A.</span></span> <span> </span><span class="NLM_article-title">Metabolism of the calcium antagonist, mibefradil (POSICORTM, Ro 40-5967). Part III. Comparative pharmacokinetics of mibefradil and its major metabolites in rat, marmoset, cynomolgus monkey and man</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">557</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1080/004982597240343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1080%2F004982597240343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=9211656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaK2sXkt1Cjt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1997&pages=557-571&author=H.+R.+Wiltshireauthor=B.+M.+Suttonauthor=G.+Heepsauthor=A.+M.+Bettyauthor=D.+W.+Angusauthor=S.+R.+Harrisauthor=E.+Worthauthor=H.+A.+Welker&title=Metabolism+of+the+calcium+antagonist%2C+mibefradil+%28POSICORTM%2C+Ro+40-5967%29.+Part+III.+Comparative+pharmacokinetics+of+mibefradil+and+its+major+metabolites+in+rat%2C+marmoset%2C+cynomolgus+monkey+and+man&doi=10.1080%2F004982597240343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of the calcium antagonist, mibefradil (POSICOR, Ro 40-5967). Part III. Comparative pharmacokinetics of mibefradil and its major metabolites in rat, marmoset, cynomolgus monkey and man</span></div><div class="casAuthors">Wiltshire, H. R.; Sutton, B. M.; Heeps, G.; Betty, A. M.; Angus, D. W.; Harris, S. R.; Worth, E.; Welker, H. A.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">557-571</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis</span>)
        </div><div class="casAbstract">1. The metab. of mibefradil has been examd. in rat, marmoset, cynomolgus monkey and man after single and multiple oral administration.  2. Metabolites typically represent between 50 and 80% of the circulating drug-related material after single oral doses of mibefradil to man, rat and marmoset.  They arise by a combination of enzymic processes: cytochrome P 450-mediated oxidn. at satd. and unsatd. carbon atoms, cytochrome P 450-catalyzed dealkylation and hydrolysis of the ester side-chain.  3. Plasma levels of mibefradil in the cynomolgus monkey are extremely low as a result of very efficient first-pass hydrolysis of its side-chain to give the corresponding alc.  Steady-state concns. of this metabolite are comparable with those of the parent drug in man and marmoset, but are relatively low in rat plasma.  4. Hydroxylation at the benzylic carbon of the tetrahydronaphthyl ring leads to further important metabolites in primates, whereas the products of O- and N-demethylation are found in small amts. in all four species.  5. Ests. of the exposure of the various species to the principal metabolites indicate that the choice of the rat, marmoset and cynomolgus monkey for the toxicol. assessment of mibefradil was appropriate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrikNYXcqCgYLVg90H21EOLACvtfcHk0liOzX5G_Dic_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkt1Cjt7s%253D&md5=e2ce7f237b43d9a2f1ec51155be0c9a0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1080%2F004982597240343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F004982597240343%26sid%3Dliteratum%253Aachs%26aulast%3DWiltshire%26aufirst%3DH.%2BR.%26aulast%3DSutton%26aufirst%3DB.%2BM.%26aulast%3DHeeps%26aufirst%3DG.%26aulast%3DBetty%26aufirst%3DA.%2BM.%26aulast%3DAngus%26aufirst%3DD.%2BW.%26aulast%3DHarris%26aufirst%3DS.%2BR.%26aulast%3DWorth%26aufirst%3DE.%26aulast%3DWelker%26aufirst%3DH.%2BA.%26atitle%3DMetabolism%2520of%2520the%2520calcium%2520antagonist%252C%2520mibefradil%2520%2528POSICORTM%252C%2520Ro%252040-5967%2529.%2520Part%2520III.%2520Comparative%2520pharmacokinetics%2520of%2520mibefradil%2520and%2520its%2520major%2520metabolites%2520in%2520rat%252C%2520marmoset%252C%2520cynomolgus%2520monkey%2520and%2520man%26jtitle%3DXenobiotica%26date%3D1997%26volume%3D27%26spage%3D557%26epage%3D571%26doi%3D10.1080%2F004982597240343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craig, P. N.</span></span> <span> </span><span class="NLM_article-title">Interdependence between physical parameters and selection of substituent groups for correlation studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">680</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1021/jm00290a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00290a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaE3MXkvFKqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1971&pages=680-684&author=P.+N.+Craig&title=Interdependence+between+physical+parameters+and+selection+of+substituent+groups+for+correlation+studies&doi=10.1021%2Fjm00290a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Interdependence between physical parameters and selection of substituent groups for correlation studies</span></div><div class="casAuthors">Craig, Paul N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">680-4</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The phys. properties, σ, π, and ER values, are not linearly related with biol. activities, although the interdependence of ER and σ2 values is confirmed.  The aliphatic π values are correlated with the Taft's aliphatic Es values.  Significant correlations are also found for π and mol. vol., and for refractivity and mol. vol. values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRh8piBnGLELVg90H21EOLACvtfcHk0ljqeVOoDDo6hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXkvFKqu7w%253D&md5=05557e45ee7c24a71d7785b20ddc4160</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fjm00290a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00290a004%26sid%3Dliteratum%253Aachs%26aulast%3DCraig%26aufirst%3DP.%2BN.%26atitle%3DInterdependence%2520between%2520physical%2520parameters%2520and%2520selection%2520of%2520substituent%2520groups%2520for%2520correlation%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1971%26volume%3D14%26spage%3D680%26epage%3D684%26doi%3D10.1021%2Fjm00290a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niles, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Target engagement and drug residence time can be observed in living cells with BRET</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">10091</span>, <span class="refDoi"> DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=26631872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=10091&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.-Q.+Chengauthor=K.+V.+Wood&title=Target+engagement+and+drug+residence+time+can+be+observed+in+living+cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement and drug residence time can be observed in living cells with BRET</span></div><div class="casAuthors">Robers, Matthew B.; Dart, Melanie L.; Woodroofe, Carolyn C.; Zimprich, Chad A.; Kirkland, Thomas A.; Machleidt, Thomas; Kupcho, Kevin R.; Levin, Sergiy; Hartnett, James R.; Zimmerman, Kristopher; Niles, Andrew L.; Ohana, Rachel Friedman; Daniels, Danette L.; Slater, Michael; Wood, Monika G.; Cong, Mei; Cheng, Yi-Qiang; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10091</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The therapeutic action of drugs is predicated on their phys. engagement with cellular targets.  Here we describe a broadly applicable method using bioluminescence resonance energy transfer (BRET) to reveal the binding characteristics of a drug with selected targets within intact cells.  Cell-permeable fluorescent tracers are used in a competitive binding format to quantify drug engagement with the target proteins fused to Nanoluc luciferase.  The approach enabled us to profile isoenzyme-specific engagement and binding kinetics for a panel of histone deacetylase (HDAC) inhibitors.  Our anal. was directed particularly to the clin. approved prodrug FK228 (Istodax/Romidepsin) because of its unique and largely unexplained mechanism of sustained intracellular action.  Anal. of the binding kinetics by BRET revealed remarkably long intracellular residence times for FK228 at HDAC1, explaining the protracted intracellular behavior of this prodrug.  Our results demonstrate a novel application of BRET for assessing target engagement within the complex milieu of the intracellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFFk0LNiL_LVg90H21EOLACvtfcHk0ljqeVOoDDo6hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO&md5=723cc5dd7d229b1115da4c2c9d75c913</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.-Q.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520engagement%2520and%2520drug%2520residence%2520time%2520can%2520be%2520observed%2520in%2520living%2520cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D10091%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14532</span>– <span class="NLM_lpage">14537</span>, <span class="refDoi"> DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0liQpmFOWHgbXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0liQpmFOWHgbXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liQpmFOWHgbXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/s0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2Fs0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0limRpfWojYhmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2Fs0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0limRpfWojYhmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0limRpfWojYhmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Francesco Rianjongdee, Stephen J. Atkinson, Chun-wa Chung, Paola Grandi, James R. J. Gray, Laura J. Kaushansky, Patricia Medeiros, Cassie Messenger, Alex Phillipou, Alex Preston, Rab K. Prinjha, Inmaculada Rioja, Alexander L. Satz, Simon Taylor, Ian D. Wall, Robert J. Watson, Gang Yao, <span class="NLM_string-name hlFld-ContribAuthor">Emmanuel H. Demont</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00412%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BHighly%252BSelective%252BBET%252BBD2%252BInhibitor%252Bfrom%252Ba%252BDNA-Encoded%252BLibrary%252BTechnology%252BScreening%252BHit%26aulast%3DRianjongdee%26aufirst%3DFrancesco%26date%3D2021%26date%3D2021%26date%3D08032021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher A. Reiher, David P. Schuman, Nicholas Simmons, <span class="NLM_string-name hlFld-ContribAuthor">Scott E. Wolkenberg</span>. </span><span class="cited-content_cbyCitation_article-title">Trends in Hit-to-Lead Optimization Following DNA-Encoded Library Screens. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (3)
                                     , 343-350. <a href="https://doi.org/10.1021/acsmedchemlett.0c00615" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00615</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00615%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DTrends%252Bin%252BHit-to-Lead%252BOptimization%252BFollowing%252BDNA-Encoded%252BLibrary%252BScreens%26aulast%3DReiher%26aufirst%3DChristopher%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D22112020%26date%3D28012021%26date%3D11022021%26volume%3D12%26issue%3D3%26spage%3D343%26epage%3D350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Josephine Yuen, Jing Chai, <span class="NLM_string-name hlFld-ContribAuthor">Yun Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Condensation of DNA-Conjugated Imines with Homophthalic Anhydride for the Synthesis of Isoquinolones on DNA. </span><span class="cited-content_cbyCitation_journal-name">Bioconjugate Chemistry</span><span> <strong>2020,</strong> <em>31 </em>
                                    (12)
                                     , 2712-2718. <a href="https://doi.org/10.1021/acs.bioconjchem.0c00508" title="DOI URL">https://doi.org/10.1021/acs.bioconjchem.0c00508</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.bioconjchem.0c00508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.bioconjchem.0c00508%26sid%3Dliteratum%253Aachs%26jtitle%3DBioconjugate%2520Chemistry%26atitle%3DCondensation%252Bof%252BDNA-Conjugated%252BImines%252Bwith%252BHomophthalic%252BAnhydride%252Bfor%252Bthe%252BSynthesis%252Bof%252BIsoquinolones%252Bon%252BDNA%26aulast%3DYuen%26aufirst%3DJosephine%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D13092020%26date%3D29102020%26date%3D12112020%26volume%3D31%26issue%3D12%26spage%3D2712%26epage%3D2718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher R. Wellaway, Paul Bamborough, Sharon G. Bernard, Chun-wa Chung, Peter D. Craggs, Leanne Cutler, Emmanuel H. Demont, John P. Evans, Laurie Gordon, Bhumika Karamshi, Antonia J. Lewis, Matthew J. Lindon, Darren J. Mitchell, Inmaculada Rioja, Peter E. Soden, Simon Taylor, Robert J. Watson, Rob Willis, James M. Woolven, Beata S. Wyspiańska, William J. Kerr, <span class="NLM_string-name hlFld-ContribAuthor">Rab K. Prinjha</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9020-9044. <a href="https://doi.org/10.1021/acs.jmedchem.0c00566" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00566</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252Ba%252BBromodomain%252Band%252BExtraterminal%252BDomain%252B%252528BET%252529%252BInhibitor%252BSelective%252Bfor%252Bthe%252BN-Terminal%252BBromodomains%252BThat%252BRetains%252Ban%252BAnti-inflammatory%252Band%252BAntiproliferative%252BPhenotype%26aulast%3DWellaway%26aufirst%3DChristopher%2BR.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06042020%26date%3D03082020%26volume%3D63%26issue%3D17%26spage%3D9020%26epage%3D9044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kunal  Nepali</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Recent developments in epigenetic cancer therapeutics: clinical advancement and emerging trends. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomedical Science</span><span> <strong>2021,</strong> <em>28 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12929-021-00721-x" title="DOI URL">https://doi.org/10.1186/s12929-021-00721-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12929-021-00721-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12929-021-00721-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomedical%2520Science%26atitle%3DRecent%252Bdevelopments%252Bin%252Bepigenetic%252Bcancer%252Btherapeutics%25253A%252Bclinical%252Badvancement%252Band%252Bemerging%252Btrends%26aulast%3DNepali%26aufirst%3DKunal%26date%3D2021%26date%3D2021%26volume%3D28%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yun  Ding</span>, <span class="hlFld-ContribAuthor ">Svetlana  Belyanskaya</span>, <span class="hlFld-ContribAuthor ">Jennifer L.  DeLorey</span>, <span class="hlFld-ContribAuthor ">Jeffrey A.  Messer</span>, <span class="hlFld-ContribAuthor ">G.  Joseph Franklin</span>, <span class="hlFld-ContribAuthor ">Paolo A.  Centrella</span>, <span class="hlFld-ContribAuthor ">Barry A.  Morgan</span>, <span class="hlFld-ContribAuthor ">Matthew A.  Clark</span>, <span class="hlFld-ContribAuthor ">Steven R.  Skinner</span>, <span class="hlFld-ContribAuthor ">Jason W.  Dodson</span>, <span class="hlFld-ContribAuthor ">Peng  Li</span>, <span class="hlFld-ContribAuthor ">Joseph P.  Marino</span>, <span class="hlFld-ContribAuthor ">David I.  Israel</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of soluble epoxide hydrolase inhibitors through DNA-encoded library technology (ELT). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>41 </em>, 116216. <a href="https://doi.org/10.1016/j.bmc.2021.116216" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116216</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116216%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bsoluble%252Bepoxide%252Bhydrolase%252Binhibitors%252Bthrough%252BDNA-encoded%252Blibrary%252Btechnology%252B%252528ELT%252529%26aulast%3DDing%26aufirst%3DYun%26date%3D2021%26volume%3D41%26spage%3D116216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Zhou</span>, <span class="hlFld-ContribAuthor ">Wenyin  Shen</span>, <span class="hlFld-ContribAuthor ">Jianzhao  Peng</span>, <span class="hlFld-ContribAuthor ">Yuqing  Deng</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of isoform/domain-selective fragments from the selection of DNA-encoded dynamic library. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>10 </em>, 116328. <a href="https://doi.org/10.1016/j.bmc.2021.116328" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116328</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116328%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bisoform%25252Fdomain-selective%252Bfragments%252Bfrom%252Bthe%252Bselection%252Bof%252BDNA-encoded%252Bdynamic%252Blibrary%26aulast%3DZhou%26aufirst%3DYu%26date%3D2021%26volume%3D10%26spage%3D116328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Verena B. K.  Kunig</span>, <span class="hlFld-ContribAuthor ">Marco  Potowski</span>, <span class="hlFld-ContribAuthor ">Mateja  Klika Škopić</span>, <span class="hlFld-ContribAuthor ">Andreas  Brunschweiger</span>. </span><span class="cited-content_cbyCitation_article-title">Scanning Protein Surfaces with DNA‐Encoded Libraries. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1048-1062. <a href="https://doi.org/10.1002/cmdc.202000869" title="DOI URL">https://doi.org/10.1002/cmdc.202000869</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000869%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DScanning%252BProtein%252BSurfaces%252Bwith%252BDNA%2525E2%252580%252590Encoded%252BLibraries%26aulast%3DKunig%26aufirst%3DVerena%2BB.%2BK.%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D7%26spage%3D1048%26epage%3D1062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yun  Zhang</span>, <span class="hlFld-ContribAuthor ">Andrew  Pike</span>. </span><span class="cited-content_cbyCitation_article-title">Pyridones in drug discovery: Recent advances. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>38 </em>, 127849. <a href="https://doi.org/10.1016/j.bmcl.2021.127849" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.127849</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.127849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.127849%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPyridones%252Bin%252Bdrug%252Bdiscovery%25253A%252BRecent%252Badvances%26aulast%3DZhang%26aufirst%3DYun%26date%3D2021%26volume%3D38%26spage%3D127849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Shi</span>, <span class="hlFld-ContribAuthor ">Yan-ran  Wu</span>, <span class="hlFld-ContribAuthor ">Jian-qiang  Yu</span>, <span class="hlFld-ContribAuthor ">Wan-nian  Zhang</span>, <span class="hlFld-ContribAuthor ">Chun-lin  Zhuang</span>. </span><span class="cited-content_cbyCitation_article-title">DNA-encoded libraries (DELs): a review of on-DNA chemistries and their output. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (4)
                                     , 2359-2376. <a href="https://doi.org/10.1039/D0RA09889B" title="DOI URL">https://doi.org/10.1039/D0RA09889B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA09889B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA09889B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DDNA-encoded%252Blibraries%252B%252528DELs%252529%25253A%252Ba%252Breview%252Bof%252Bon-DNA%252Bchemistries%252Band%252Btheir%252Boutput%26aulast%3DShi%26aufirst%3DYing%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D4%26spage%3D2359%26epage%3D2376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ellen E.  Guest</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Pickett</span>, <span class="hlFld-ContribAuthor ">Jonathan D.  Hirst</span>. </span><span class="cited-content_cbyCitation_article-title">Structural variation of protein–ligand complexes of the first bromodomain of BRD4. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>1 </em><a href="https://doi.org/10.1039/D1OB00658D" title="DOI URL">https://doi.org/10.1039/D1OB00658D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1OB00658D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1OB00658D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DStructural%252Bvariation%252Bof%252Bprotein%2525E2%252580%252593ligand%252Bcomplexes%252Bof%252Bthe%252Bfirst%252Bbromodomain%252Bof%252BBRD4%26aulast%3DGuest%26aufirst%3DEllen%2BE.%26date%3D2021%26date%3D2021%26volume%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mike  Lemke</span>, <span class="hlFld-ContribAuthor ">Hannah  Ravenscroft</span>, <span class="hlFld-ContribAuthor ">Nicole J.  Rueb</span>, <span class="hlFld-ContribAuthor ">Dmitri  Kireev</span>, <span class="hlFld-ContribAuthor ">Dana  Ferraris</span>, <span class="hlFld-ContribAuthor ">Raphael M.  Franzini</span>. </span><span class="cited-content_cbyCitation_article-title">Integrating DNA-encoded chemical libraries with virtual combinatorial library screening: Optimizing a PARP10 inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (19)
                                     , 127464. <a href="https://doi.org/10.1016/j.bmcl.2020.127464" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127464</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127464%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIntegrating%252BDNA-encoded%252Bchemical%252Blibraries%252Bwith%252Bvirtual%252Bcombinatorial%252Blibrary%252Bscreening%25253A%252BOptimizing%252Ba%252BPARP10%252Binhibitor%26aulast%3DLemke%26aufirst%3DMike%26date%3D2020%26volume%3D30%26issue%3D19%26spage%3D127464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando D.  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Current advances on the development of BET inhibitors: insights from computational methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 127-180. <a href="https://doi.org/10.1016/bs.apcsb.2020.06.002" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.06.002%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252Badvances%252Bon%252Bthe%252Bdevelopment%252Bof%252BBET%252Binhibitors%25253A%252Binsights%252Bfrom%252Bcomputational%252Bmethods%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DFernando%2BD.%26date%3D2020%26spage%3D127%26epage%3D180%26pub%3DElsevier%26date%3D2020%26volume%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Domain architecture of BET highlighting the eight bromodomains. (b) Early generation BET inhibitors with in vivo capability.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Scheme of the hit DNA-encoded benzimidazole library where BB1, BB2, and BB3 refer to cycle 1, 2, and 3 BBs, respectively. (b) Result of the BRD4 screen of the benzimidazole library. Compounds on the highlighted horizontal line (light blue) in the cubic scatter plot shared the common BB1–BB2 combination (<b>5</b> and <b>6</b>, respectively) with BB3 as a variable component resulting in the generic hit structure <b>7</b>. The sphere size is proportional to the DNA tag copy number. Library members with less than 5 copies were removed to simplify visualization.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Dimethylphenol Benzimidazole Analogues <b>8–10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (<b>5</b>), DIPEA, EtOH, 80 °C; (ii) 4-hydroxy-3,5-dimethylbenzaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (4:1), 80 °C; (iii) TFA, DCM, rt; (b) PhCO<sub>2</sub>H, HATU, DIPEA, DMF, rt; (c) (i) Ac<sub>2</sub>O, pyr, THF/MeCN; (ii) 2 M LiOH, THF/MeOH (4:1), rt.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Dimethylphenol Benzimidazole N1 Analogues <b>15a–g</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) RNH<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80–110 °C; (b) H<sub>2</sub>, Pd/C, DMF, 30 °C; (c) 4-hydroxy-3,5-dimethylbenzaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMF, 100 °C; (d) 4 M HCl in MeOH, rt−50 °C; (e) AcOH or 2-hydroxyacetic acid, BOP, NEt<sub>3</sub>, DCM, rt.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of acetamide <b>10</b> (carbon = green) in BRD4 BD1 (carbon = gray, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>). Water molecules at the base of the binding pocket are shown as red spheres, and selected hydrogen bonds are shown as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Identification of pyridones as KAc-mimetics from a fragment-based screen.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> (b) X-ray crystal structure of <i>N</i>-methyl pyridone fragment <b>16</b> (carbon = orange) in BRD2 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ1">6TQ1</a>). (c) X-ray crystal structure of <i>N</i>-methyl pyridone imidazole <b>17</b> (carbon = cyan) in BRD2 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ2">6TQ2</a>). (d) Superposition of X-ray crystal structures of <i>N</i>-methyl pyridone imidazole <b>17</b> (carbon = cyan) in BRD2 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ2">6TQ2</a>) and 2,6-dimethylphenol <b>10</b> (carbon = green) in BRD4 BD1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Benzimidazole 2-Position Analogues 19a–d<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: for compounds <b>18a</b>, <b>18b</b>, and <b>18d</b>: (a) (i) RCHO, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMF, 100 °C; (ii) HCl in MeOH, rt; for compound <b>18c</b>: (b) RCO<sub>2</sub>H, PPA, 180 °C; (c) 2-hydroxyacetic acid, EDC·HCl, HOBt, DIPEA, DCM, rt.</p></p></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Dimethylpyridone Benzimidazole N1 Analogues <b>20a–i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) amine, DIPEA, DMSO or NMP or 2-MeTHF, microwave, 110–125 °C; (b) (i) 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (2:1), 100 °C; (ii) for compound <b>20e</b>: TFA, DCM, rt; (c) (i) H<sub>2</sub>, Pd/C, EtOH, rt; (ii) Oxone, DMF/H<sub>2</sub>O (30:1), rt.</p></p></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Superposition of X-ray crystal structures of 1,3-dimethylpyridone <b>20a</b> (carbon = cyan, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY">6TPY</a>) and 2,6-dimethylphenol <b>10</b> in BRD4 BD1 (carbon = green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>). A surface was applied to the protein of PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>. (b) View at the base of the binding pocket from superposition of 1,3-dimethylpyridone <b>20a</b> (carbon, water molecules [spheres], and selected hydrogen bonds [dashed lines] = cyan PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY">6TPY</a>) and 2,6-dimethylphenol <b>10</b> (carbon, water molecules [spheres], and selected hydrogen bonds [dashed lines] = green, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>) X-ray crystal structures in BRD4 BD1. A surface was applied to the protein of PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>. (c) Superposition of the X-ray crystal structure of 1,3-dimethylpyridone <b>20a</b> in BRD4 BD1 (carbon = cyan, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY">6TPY</a>) with an apo structure of BAZ2A<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> (carbon = orange, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LZ2">4LZ2</a>). A wireframe surface has been applied to Trp1816 of BAZ2A.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Dimethylpyridone Benzimidazole 5- and 6-Position Analogues <b>22a–p</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: for compounds <b>21a</b> and <b>21b</b>: (a) 4-substituted 1-fluoro-2-nitrobenzene or 4-substituted 4-bromo-2-fluoro-1-nitrobenzene, (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine, DIPEA, 2-MeTHF, 80 °C; for compounds <b>22a</b> and <b>22b</b>: (b) 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (2.5:1), 90 or 80 °C; (c) HN(CH<sub>3</sub>)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub> (2 mol %), JohnPhos (13 mol %), NaO<sup><i>t</i></sup>Bu, toluene or 1,4-dioxane, microwave, 90 °C; for compounds <b>22e–22p</b>: (d) (i) substituted 1-fluoro-2-nitrobenzene, (tetrahydro-2<i>H</i>-pyran-4-yl)methanamine, DIPEA, 2-MeTHF, microwave, 110 °C; (ii) 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (2:1), 100 °C.</p></p></figure><figure data-id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Plots of BRD4 FRET mean activities vs sigma para values for substituents at the 5-position (left) and 6-position (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Dimethylpyridone Benzimidazole 5- and 6-Position Amine Analogues <b>23a–23t</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) NHRR′, Pd<sub>2</sub>(dba)<sub>3</sub> (2 mol %), DavePhos (5 mol %), NaO<sup><i>t</i></sup>Bu, toluene or 1,4-dioxane, 80 °C; for compounds <b>23m</b> and <b>23n</b>: (ii) 4 M HCl in 1,4-dioxane, rt.</p></p></figure><figure data-id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Plots of MMP activities for 5-position (blue) and 6-position (green) substituents. BRD4 FRET data (left) and MCP-1 WB data (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. X-ray crystal structure of dimethylpyridone benzimidazole <b>24</b> in BRD4 BD1 (carbon = magenta, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPZ">6TPZ</a>). Water molecules at the base of the binding pocket are shown as red spheres, and selected hydrogen bonds are shown as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Dimethylpyridone Benzimidazole <b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1,3-dimethoxypropan-2-amine, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 97%; (b) 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>, EtOH/H<sub>2</sub>O (4:1), 90 °C, 56%; (c) morpholine, Pd<sub>2</sub>(dba)<sub>3</sub> (2 mol %), DavePhos (5 mol %), NaO<sup><i>t</i></sup>Bu, 2-MeTHF, 80 °C, 59%.</p></p></figure><figure data-id="fig9" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. IL-6 (red) and unbound dimethylpyridone benzimidazole <b>24</b> (blue) AUC concentrations in the acute LPS model of inflammation. Statistical differences in %inhibition of IL-6 AUC levels were determined as <i>p</i> <0.05 (*) and <i>p</i> <0.001 (***).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/medium/jm9b01670_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Mean %inhibition of IgG1 (relative to the vehicle) at day 14 vs free average drug concentration. Statistical differences in % inhibition of IgG1 were determined as <i>p</i> < 0.01 (**).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-2/acs.jmedchem.9b01670/20200115/images/large/jm9b01670_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01670&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i129">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04870" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04870" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 83 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taverna, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruthenburg, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span> <span> </span><span class="NLM_article-title">How chromatin-binding modules interpret histone modifications: lessons from professional pocket pickers</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1040</span>, <span class="refDoi"> DOI: 10.1038/nsmb1338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnsmb1338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=17984965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Kjs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1025-1040&author=S.+D.+Tavernaauthor=H.+Liauthor=A.+J.+Ruthenburgauthor=C.+D.+Allisauthor=D.+J.+Patel&title=How+chromatin-binding+modules+interpret+histone+modifications%3A+lessons+from+professional+pocket+pickers&doi=10.1038%2Fnsmb1338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">How chromatin-binding modules interpret histone modifications: lessons from professional pocket pickers</span></div><div class="casAuthors">Taverna, Sean D.; Li, Haitao; Ruthenburg, Alexander J.; Allis, C. David; Patel, Dinshaw J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1025-1040</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histones comprise the major protein component of chromatin, the scaffold in which the eukaryotic genome is packaged, and are subject to many types of post-translational modifications (PTMs), esp. on their flexible tails.  These modifications may constitute a 'histone code' and could be used to manage epigenetic information that helps extend the genetic message beyond DNA sequences.  This proposed code, read in part by histone PTM-binding 'effector' modules and their assocd. complexes, has been predicted to define unique functional states of chromatin and/or regulate various chromatin-templated processes.  A wealth of structural and functional data show how chromatin effector modules target their cognate covalent histone modifications.  Here, the authors summarize key features in the mol. recognition of histone PTMs by a diverse family of 'reader pockets' (including bromodomains, Royal superfamily modules, PHD fingers, WD40 protein repeats, 14-3-3 proteins, and BRCT domains), highlighting specific readout mechanisms for individual marks, common themes, and insights into the downstream functional consequences of the interactions.  Changes in these interactions may have far-reaching implications for human biol. and disease, notably cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroaoP5OiDUOLVg90H21EOLACvtfcHk0lg6xmz6YmtZ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Kjs7vE&md5=4ff842f1ef5416b932a89c751707defd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1338%26sid%3Dliteratum%253Aachs%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26atitle%3DHow%2520chromatin-binding%2520modules%2520interpret%2520histone%2520modifications%253A%2520lessons%2520from%2520professional%2520pocket%2520pickers%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D1025%26epage%3D1040%26doi%3D10.1038%2Fnsmb1338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4t%C3%A9author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lg6xmz6YmtZ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lg6xmz6YmtZ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Small-molecule BET inhibitors in clinical and preclinical development and their therapeutic potential</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">776</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.2174/1568026615666150302110135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.2174%2F1568026615666150302110135" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=25732788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXks1ynurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=776-794&author=L.+Yuauthor=Z.+Wangauthor=Z.+Zhangauthor=X.+Renauthor=X.+Luauthor=K.+Ding&title=Small-molecule+BET+inhibitors+in+clinical+and+preclinical+development+and+their+therapeutic+potential&doi=10.2174%2F1568026615666150302110135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule BET Inhibitors in Clinical and Preclinical Development and Their Therapeutic Potential</span></div><div class="casAuthors">Yu, Lei; Wang, Zhen; Zhang, Zhang; Ren, Xiaomei; Lu, Xiaoyun; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">776-794</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Lysine acetylation is a pivotal mechanism in chromatin processes and the regulation of gene transcription.  The acetylated lysine residues of histones are exclusively recognized by bromodomains (BRDs) known as epigenetic reader.  Proteins contg. BRDs undergo a post-translational modification (PTM) with development of cellular signaling and disease biol.  The bromo and extra-terminal (BET) proteins are the second subfamily, which play important roles in cellular proliferation, cell cycle progression and chromatin compaction.  Recently, a variety of small mols. have been reported to interact with the BET family proteins and accelerate the validation of BET proteins as druggable targets for treatment of cancers, inflammation and related diseases.  In this review, we will summarize the small-mol. inhibitors in clin. and preclin. studies of the BET family bromodomains and their medicinal implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk52FOUPlX8bVg90H21EOLACvtfcHk0lhXS3rPQ50Xvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXks1ynurs%253D&md5=802bf7b832d46c2b18ef6ac77eda1445</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F1568026615666150302110135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026615666150302110135%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DSmall-molecule%2520BET%2520inhibitors%2520in%2520clinical%2520and%2520preclinical%2520development%2520and%2520their%2520therapeutic%2520potential%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2015%26volume%3D15%26spage%3D776%26epage%3D794%26doi%3D10.2174%2F1568026615666150302110135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4 (BRD4)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4+%28BRD4%29&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lhXS3rPQ50Xvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%2520%2528BRD4%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujikawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e83190</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0083190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1371%2Fjournal.pone.0083190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=24391744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=K.+G.+McLureauthor=E.+M.+Gesnerauthor=L.+Tsujikawaauthor=O.+A.+Kharenkoauthor=S.+Attwellauthor=E.+Campeauauthor=S.+Wasiakauthor=A.+Steinauthor=A.+Whiteauthor=E.+Fontanoauthor=R.+K.+Sutoauthor=N.+C.+W.+Wongauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=P.+R.+Young&title=RVX-208%2C+an+inducer+of+ApoA-I+in+humans%2C+is+a+BET+bromodomain+antagonist&doi=10.1371%2Fjournal.pone.0083190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inducer of ApoA-I in humans, is a BET bromodomain antagonist</span></div><div class="casAuthors">McLure, Kevin G.; Gesner, Emily M.; Tsujikawa, Laura; Kharenko, Olesya A.; Attwell, Sarah; Campeau, Eric; Wasiak, Sylwia; Stein, Adam; White, Andre; Fontano, Eric; Suto, Robert K.; Wong, Norman C. W.; Wagner, Gregory S.; Hansen, Henrik C.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e83190/1-e83190/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increased synthesis of Apolipoprotein A-I (ApoA-I) and HDL is believed to provide a new approach to treating atherosclerosis through the stimulation of reverse cholesterol transport.  RVX-208 increases the prodn. of ApoA-I in hepatocytes in vitro and in vivo in monkeys and humans, which results in increased HDL-C, but the mol. target was not previously reported.  Using binding assays and X-ray crystallog., we now show that RVX-208 selectively binds to bromodomains of the BET (Bromodomain and Extra Terminal) family, competing for a site bound by the endogenous ligand, acetylated lysine and that this accounts for its pharmacol. activity. siRNA expts. further suggest that induction of ApoA-I mRNA is mediated by BET family member BRD4.  These data indicate that RVX-208 increases ApoA-I prodn. through an epigenetic mechanism and suggests that BET inhibition may be a promising new approach to the treatment of atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLHPlAY0x6NLVg90H21EOLACvtfcHk0lhXS3rPQ50Xvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXis1Wju78%253D&md5=519ef7ed6d06f97ef8a1359832c11d92</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0083190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0083190%26sid%3Dliteratum%253Aachs%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DGesner%26aufirst%3DE.%2BM.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DWasiak%26aufirst%3DS.%26aulast%3DStein%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DFontano%26aufirst%3DE.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DRVX-208%252C%2520an%2520inducer%2520of%2520ApoA-I%2520in%2520humans%252C%2520is%2520a%2520BET%2520bromodomain%2520antagonist%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0083190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0liMGnDyz0yNgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gacias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matikainen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casaccia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Selective chemical modulation of gene transcription favors oligodendrocyte lineage progression</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.chembiol.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=24954007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=841-854&author=M.+Gaciasauthor=G.+Gerona-Navarroauthor=A.+N.+Plotnikovauthor=G.+Zhangauthor=L.+Zengauthor=J.+Kaurauthor=G.+Moyauthor=E.+Rusinovaauthor=Y.+Rodriguezauthor=B.+Matikainenauthor=A.+Vincekauthor=J.+Joshuaauthor=P.+Casacciaauthor=M.-M.+Zhou&title=Selective+chemical+modulation+of+gene+transcription+favors+oligodendrocyte+lineage+progression&doi=10.1016%2Fj.chembiol.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Chemical Modulation of Gene Transcription Favors Oligodendrocyte Lineage Progression</span></div><div class="casAuthors">Gacias, Mar; Gerona-Navarro, Guillermo; Plotnikov, Alexander N.; Zhang, Guangtao; Zeng, Lei; Kaur, Jasbir; Moy, Gregory; Rusinova, Elena; Rodriguez, Yoel; Matikainen, Bridget; Vincek, Adam; Joshua, Jennifer; Casaccia, Patrizia; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">841-854</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lysine acetylation regulates gene expression through modulating protein-protein interactions in chromatin.  Chem. inhibition of acetyl-lysine binding bromodomains of the major chromatin regulators BET (bromodomain and extraterminal domain) proteins has been shown to effectively block cell proliferation in cancer and inflammation.  However, whether selective inhibition of individual BET bromodomains has distinctive functional consequences remains only partially understood.  In this study, we show that selective chem. inhibition of the first bromodomain of BET proteins using our small-mol. inhibitor, Olinone, accelerated the progression of mouse primary oligodendrocyte progenitors toward differentiation, whereas inhibition of both bromodomains of BET proteins hindered differentiation.  This effect was target specific, as it was not detected in cells treated with inactive analogs and independent of any effect on proliferation.  Therefore, selective chem. modulation of individual bromodomains, rather than use of broad-based inhibitors, may enhance regenerative strategies in disorders characterized by myelin loss such as aging and neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1fCvOR18yE7Vg90H21EOLACvtfcHk0liMGnDyz0yNgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtLrK&md5=6b377663a8d0d07441ab4f8a5d8191b3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DGacias%26aufirst%3DM.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DKaur%26aufirst%3DJ.%26aulast%3DMoy%26aufirst%3DG.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DRodriguez%26aufirst%3DY.%26aulast%3DMatikainen%26aufirst%3DB.%26aulast%3DVincek%26aufirst%3DA.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DCasaccia%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DSelective%2520chemical%2520modulation%2520of%2520gene%2520transcription%2520favors%2520oligodendrocyte%2520lineage%2520progression%26jtitle%3DChem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D841%26epage%3D854%26doi%3D10.1016%2Fj.chembiol.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>– <span class="NLM_lpage">4334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0lgw_dW_sjhCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1655</span>– <span class="NLM_lpage">1680</span>, <span class="refDoi"> DOI: 10.4155/fmc-2016-0059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.4155%2Ffmc-2016-0059" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27193077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKhsr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1655-1680&author=C.+Galdeanoauthor=A.+Ciulli&title=Selectivity+on-target+of+bromodomain+chemical+probes+by+structure-guided+medicinal+chemistry+and+chemical+biology&doi=10.4155%2Ffmc-2016-0059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology</span></div><div class="casAuthors">Galdeano, Carles; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1655-1680</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Targeting epigenetic proteins is a rapidly growing area for medicinal chem. and drug discovery.  Recent years have seen an explosion of interest in developing small mols. binding to bromodomains, the readers of acetyl-lysine modifications.  A plethora of co-crystal structures has motivated focused fragment-based design and optimization programs within both industry and academia.  These efforts have yielded several compds. entering the clinic, and many more are increasingly being used as chem. probes to interrogate bromodomain biol.  High selectivity of chem. probes is necessary to ensure biol. activity is due to an on-target effect.  Here, we review the state-of-the-art of bromodomain-targeting compds., focusing on the structural basis for their on-target selectivity or lack thereof.  We also highlight chem. biol. approaches to enhance on-target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1bg8K3kh-RLVg90H21EOLACvtfcHk0lgw_dW_sjhCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKhsr3K&md5=a4c02dd517e8582755a918eabd7bfeea</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0059%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelectivity%2520on-target%2520of%2520bromodomain%2520chemical%2520probes%2520by%2520structure-guided%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2016%26volume%3D8%26spage%3D1655%26epage%3D1680%26doi%3D10.4155%2Ffmc-2016-0059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voitovich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derviaux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rebuffet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinquet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemot, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunel, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collette, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morelli, X.</span></span> <span> </span><span class="NLM_article-title">Exploring selective inhibition of the first bromodomain of the human bromodomain and extra-terminal domain (BET) proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1634</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1634-1641&author=B.+Rauxauthor=Y.+Voitovichauthor=C.+Derviauxauthor=A.+Lugariauthor=E.+Rebuffetauthor=S.+Milhasauthor=S.+Prietauthor=T.+Rouxauthor=E.+Trinquetauthor=J.-C.+Guillemotauthor=S.+Knappauthor=J.-M.+Brunelauthor=A.+Y.+Fedorovauthor=Y.+Colletteauthor=P.+Rocheauthor=S.+Betziauthor=S.+Combesauthor=X.+Morelli&title=Exploring+selective+inhibition+of+the+first+bromodomain+of+the+human+bromodomain+and+extra-terminal+domain+%28BET%29+proteins&doi=10.1021%2Facs.jmedchem.5b01708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-terminal Domain (BET) Proteins</span></div><div class="casAuthors">Raux, Brigitt; Voitovich, Yuliia; Derviaux, Carine; Lugari, Adrien; Rebuffet, Etienne; Milhas, Sabine; Priet, Stephane; Roux, Thomas; Trinquet, Eric; Guillemot, Jean-Claude; Knapp, Stefan; Brunel, Jean-Michel; Fedorov, Alexey Yu.; Collette, Yves; Roche, Philippe; Betzi, Stephane; Combes, Sebastien; Morelli, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1634-1641</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A midthroughput screening follow-up program targeting the first bromodomain of the human BRD4 protein, BRD4(BD1), identified an acetylated-mimic xanthine deriv. inhibitor.  This compd. binds with an affinity in the low micromolar range yet exerts suitable unexpected selectivity in vitro against the other members of the bromodomain and extra-terminal domain (BET) family.  A structure-based program pinpointed a role of the ZA loop, paving the way for the development of potent and selective BET-BRDi probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouQdXvF3r4zLVg90H21EOLACvtfcHk0lgw_dW_sjhCfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D&md5=fb9cfc20c76f65fcf409370b60aecc0a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01708%26sid%3Dliteratum%253Aachs%26aulast%3DRaux%26aufirst%3DB.%26aulast%3DVoitovich%26aufirst%3DY.%26aulast%3DDerviaux%26aufirst%3DC.%26aulast%3DLugari%26aufirst%3DA.%26aulast%3DRebuffet%26aufirst%3DE.%26aulast%3DMilhas%26aufirst%3DS.%26aulast%3DPriet%26aufirst%3DS.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DGuillemot%26aufirst%3DJ.-C.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrunel%26aufirst%3DJ.-M.%26aulast%3DFedorov%26aufirst%3DA.%2BY.%26aulast%3DCollette%26aufirst%3DY.%26aulast%3DRoche%26aufirst%3DP.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DCombes%26aufirst%3DS.%26aulast%3DMorelli%26aufirst%3DX.%26atitle%3DExploring%2520selective%2520inhibition%2520of%2520the%2520first%2520bromodomain%2520of%2520the%2520human%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1634%26epage%3D1641%26doi%3D10.1021%2Facs.jmedchem.5b01708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deangelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of bivalent BET inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1096</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnchembio.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27775715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1089-1096&author=M.+Tanakaauthor=J.+M.+Robertsauthor=H.-S.+Seoauthor=A.+Souzaauthor=J.+Paulkauthor=T.+G.+Scottauthor=S.+L.+Deangeloauthor=S.+Dhe-paganonauthor=J.+E.+Bradner&title=Design+and+characterization+of+bivalent+BET+inhibitors&doi=10.1038%2Fnchembio.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of bivalent BET inhibitors</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Seo, Hyuk-Soo; Souza, Amanda; Paulk, Joshiawa; Scott, Thomas G.; DeAngelo, Stephen L.; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1089-1096</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cellular signaling is often propagated by multivalent interactions.  Multivalency creates avidity, allowing stable biophys. recognition.  Multivalency is an attractive strategy for achieving potent binding to protein targets, as the affinity of bivalent ligands is often greater than the sum of monovalent affinities.  The bromodomain and extraterminal domain (BET) family of transcriptional coactivators features tandem bromodomains through which BET proteins bind acetylated histones and transcription factors.  All reported antagonists of the BET protein BRD4 bind in a monovalent fashion.  Here the authors describe, to the knowledge for the first time, a bivalent BET bromodomain inhibitor-MT1-which has unprecedented potency.  Biophys. and biochem. studies suggest MT1 is an intramol. bivalent BRD4 binder that is more than 100-fold more potent, in cellular assays, than the corresponding monovalent antagonist, JQ1.  MT1 significantly (P < 0.05) delayed leukemia progression in mice, as compared to JQ1.  These data qualify a powerful chem. probe for BET bromodomains and a rationale for further development of multidomain inhibitors of epigenetic reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQwc6X0VYYbVg90H21EOLACvtfcHk0ljYUVFd9dJe6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurrM&md5=7319203dac409d5dee3c580b49fcf185</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2209%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DDeangelo%26aufirst%3DS.%2BL.%26aulast%3DDhe-paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520bivalent%2520BET%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D1089%26epage%3D1096%26doi%3D10.1038%2Fnchembio.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bobby, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flavell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jowitt, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikhney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlister, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Méndez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saif, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svergun, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span> <span> </span><span class="NLM_article-title">Potent and selective bivalent inhibitors of BET bromodomains</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1104</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnchembio.2210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27775716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1097-1104&author=M.+J.+Waringauthor=H.+Chenauthor=A.+A.+Rabowauthor=G.+Walkerauthor=R.+Bobbyauthor=S.+Boikoauthor=R.+H.+Bradburyauthor=R.+Callisauthor=E.+Clarkauthor=I.+Daleauthor=D.+L.+Danielsauthor=A.+Dulakauthor=L.+Flavellauthor=G.+Holdgateauthor=T.+A.+Jowittauthor=A.+Kikhneyauthor=M.+McAlisterauthor=J.+M%C3%A9ndezauthor=D.+Oggauthor=J.+Patelauthor=P.+Petterutiauthor=G.+R.+Robbauthor=M.+B.+Robersauthor=S.+Saifauthor=N.+Strattonauthor=D.+I.+Svergunauthor=W.+Wangauthor=D.+Whittakerauthor=D.+M.+Wilsonauthor=Y.+Yao&title=Potent+and+selective+bivalent+inhibitors+of+BET+bromodomains&doi=10.1038%2Fnchembio.2210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective bivalent inhibitors of BET bromodomains</span></div><div class="casAuthors">Waring, Michael J.; Chen, Huawei; Rabow, Alfred A.; Walker, Graeme; Bobby, Romel; Boiko, Scott; Bradbury, Rob H.; Callis, Rowena; Clark, Edwin; Dale, Ian; Daniels, Danette L.; Dulak, Austin; Flavell, Liz; Holdgate, Geoff; Jowitt, Thomas A.; Kikhney, Alexey; McAlister, Mark; Mendez, Jacqui; Ogg, Derek; Patel, Joe; Petteruti, Philip; Robb, Graeme R.; Robers, Matthew B.; Saif, Sakina; Stratton, Natalie; Svergun, Dmitri I.; Wang, Wenxian; Whittaker, David; Wilson, David M.; Yao, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1097-1104</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extraterminal (BET) family, in particular bromodomain-contg. protein 4 (BRD4), are of great interest as biol. targets.  BET proteins contain two sep. bromodomains, and existing inhibitors bind to them monovalently.  Here the authors describe the discovery and characterization of probe compd. biBET, capable of engaging both bromodomains simultaneously in a bivalent, in cis binding mode.  The evidence provided here was obtained in a variety of biophys. and cellular expts.  The bivalent binding results in very high cellular potency for BRD4 binding and pharmacol. responses such as disruption of BRD4-mediator complex subunit 1 foci with an EC50 of 100 pM.  These compds. will be of considerable utility as BET/BRD4 chem. probes.  This work illustrates a novel concept in ligand design-simultaneous targeting of two sep. domains with a drug-like small mol.-providing precedent for a potentially more effective paradigm for developing ligands for other multidomain proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq-VaDebQI0rVg90H21EOLACvtfcHk0ljYUVFd9dJe6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurzE&md5=fabaccbac93338af32292ad8907ade0c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2210%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DBobby%26aufirst%3DR.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DCallis%26aufirst%3DR.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DDale%26aufirst%3DI.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DFlavell%26aufirst%3DL.%26aulast%3DHoldgate%26aufirst%3DG.%26aulast%3DJowitt%26aufirst%3DT.%2BA.%26aulast%3DKikhney%26aufirst%3DA.%26aulast%3DMcAlister%26aufirst%3DM.%26aulast%3DM%25C3%25A9ndez%26aufirst%3DJ.%26aulast%3DOgg%26aufirst%3DD.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DSaif%26aufirst%3DS.%26aulast%3DStratton%26aufirst%3DN.%26aulast%3DSvergun%26aufirst%3DD.%2BI.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DYao%26aufirst%3DY.%26atitle%3DPotent%2520and%2520selective%2520bivalent%2520inhibitors%2520of%2520BET%2520bromodomains%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D1097%26epage%3D1104%26doi%3D10.1038%2Fnchembio.2210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castriotta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahdesmaki, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2563</span>– <span class="NLM_lpage">2574</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-16-0141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1158%2F1535-7163.mct-16-0141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27573426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2563-2574&author=G.+W.+Rhyasenauthor=M.+M.+Hattersleyauthor=Y.+Yaoauthor=A.+Dulakauthor=W.+Wangauthor=P.+Petterutiauthor=I.+L.+Daleauthor=S.+Boikoauthor=T.+Cheungauthor=J.+Zhangauthor=S.+Wenauthor=L.+Castriottaauthor=D.+Lawsonauthor=M.+Collinsauthor=L.+Baoauthor=M.+J.+Ahdesmakiauthor=G.+Walkerauthor=G.+O%E2%80%99Connorauthor=T.+C.+Yehauthor=A.+A.+Rabowauthor=J.+R.+Dryauthor=C.+Reimerauthor=P.+Lyneauthor=G.+B.+Millsauthor=S.+E.+Fawellauthor=M.+J.+Waringauthor=M.+Zindaauthor=E.+Clarkauthor=H.+Chen&title=AZD5153%3A+a+novel+bivalent+BET+bromodomain+inhibitor+highly+active+against+hematologic+malignancies&doi=10.1158%2F1535-7163.mct-16-0141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span></div><div class="casAuthors">Rhyasen, Garrett W.; Hattersley, Maureen M.; Yao, Yi; Dulak, Austin; Wang, Wenxian; Petteruti, Philip; Dale, Ian L.; Boiko, Scott; Cheung, Tony; Zhang, Jingwen; Wen, Shenghua; Castriotta, Lillian; Lawson, Deborah; Collins, Michael; Bao, Larry; Ahdesmaki, Miika J.; Walker, Graeme; O'Connor, Greg; Yeh, Tammie C.; Rabow, Alfred A.; Dry, Jonathan R.; Reimer, Corinne; Lyne, Paul; Mills, Gordon B.; Fawell, Stephen E.; Waring, Michael J.; Zinda, Michael; Clark, Edwin; Chen, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2574</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD4 regulates gene expression via recruitment of transcriptional regulatory complexes to acetylated chromatin.  Pharmacol. targeting of BRD4 bromodomains by small mol. inhibitors has proven to be an effective means to disrupt aberrant transcriptional programs crit. for tumor growth and/or survival.  Herein, we report AZD5153, a potent, selective, and orally available BET/BRD4 bromodomain inhibitor possessing a bivalent binding mode.  Unlike previously described monovalent inhibitors, AZD5153 ligates two bromodomains in BRD4 simultaneously.  The enhanced avidity afforded through bivalent binding translates into increased cellular and antitumor activity in preclin. hematol. tumor models.  In vivo administration of AZD5153 led to tumor stasis or regression in multiple xenograft models of acute myeloid leukemia, multiple myeloma, and diffuse large B-cell lymphoma.  The relationship between AZD5153 exposure and efficacy suggests that prolonged BRD4 target coverage is a primary efficacy driver.  AZD5153 treatment markedly affects transcriptional programs of MYC, E2F, and mTOR.  Of note, mTOR pathway modulation is assocd. with cell line sensitivity to AZD5153.  Transcriptional modulation of MYC and HEXIM1 was confirmed in AZD5153-treated human whole blood, thus supporting their use as clin. pharmacodynamic biomarkers.  This study establishes AZD5153 as a highly potent, orally available BET/BRD4 inhibitor and provides a rationale for clin. development in hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiKYOFhro1bVg90H21EOLACvtfcHk0ljYUVFd9dJe6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D&md5=8d0ac899d081705aaa12d49fd90165a3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-16-0141%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DDale%26aufirst%3DI.%2BL.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCastriotta%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DBao%26aufirst%3DL.%26aulast%3DAhdesmaki%26aufirst%3DM.%2BJ.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DLyne%26aufirst%3DP.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DAZD5153%253A%2520a%2520novel%2520bivalent%2520BET%2520bromodomain%2520inhibitor%2520highly%2520active%2520against%2520hematologic%2520malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2563%26epage%3D2574%26doi%3D10.1158%2F1535-7163.mct-16-0141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0ljOOUD4A0Ogfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0ljOOUD4A0Ogfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lgm7POpEIrsYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.-Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lgm7POpEIrsYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Impact of target warhead and linkage vector on inducing protein degradation: comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=504-513&author=K.-H.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=Impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Target Warhead and Linkage Vector on Inducing Protein Degradation: Comparison of Bromodomain and Extra-Terminal (BET) Degraders Derived from Triazolodiazepine (JQ1) and Tetrahydroquinoline (I-BET726) BET Inhibitor Scaffolds</span></div><div class="casAuthors">Chan, Kwok-Ho; Zengerle, Michael; Testa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">504-513</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of proteolysis-targeting chimeras (PROTACs) is a powerful small-mol. approach for inducing protein degrdn.  PROTACs conjugate a target warhead to an E3 ubiquitin ligase ligand via a linker.  Here the authors examd. the impact of derivatizing two different BET bromodomain inhibitors, triazolodiazepine JQ1 and the more potent tetrahydroquinoline I-BET726, via distinct exit vectors, using different polyethylene glycol linkers to VHL ligand VH032.  Triazolodiazepine PROTACs exhibited pos. cooperativities of ternary complex formation and were more potent degraders than tetrahydroquinoline compds., which showed neg. cooperativities instead.  Marked dependency on linker length was obsd. for BET-degrading and cMyc-driven antiproliferative activities in acute myeloid leukemia cell lines.  This work exemplifies as a cautionary tale how a more potent inhibitor does not necessarily generate more potent PROTACs and underscores the key roles played by the conjugation.  The provided insights and framework for structure-activity relationships of bivalent degraders are anticipated to have wide future applicability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvEL41TefwTbVg90H21EOLACvtfcHk0lgm7POpEIrsYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D&md5=c3bfd9377d8e7805ba962cd0180431af</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DImpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D504%26epage%3D513%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calosing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshminarasimhan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suto, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitchen, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Horst, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span> <span> </span><span class="NLM_article-title">Design and characterization of novel covalent bromodomain and extra-terminal domain (BET) inhibitors targeting a methionine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8202</span>– <span class="NLM_lpage">8211</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8202-8211&author=O.+A.+Kharenkoauthor=R.+G.+Patelauthor=S.+D.+Brownauthor=C.+Calosingauthor=A.+Whiteauthor=D.+Lakshminarasimhanauthor=R.+K.+Sutoauthor=B.+C.+Duffyauthor=D.+B.+Kitchenauthor=K.+G.+McLureauthor=H.+C.+Hansenauthor=E.+H.+van+der+Horstauthor=P.+R.+Young&title=Design+and+characterization+of+novel+covalent+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+targeting+a+methionine&doi=10.1021%2Facs.jmedchem.8b00666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Novel Covalent Bromodomain and Extra-Terminal Domain (BET) Inhibitors Targeting a Methionine</span></div><div class="casAuthors">Kharenko, Olesya A.; Patel, Reena G.; Brown, S. David; Calosing, Cyrus; White, Andre; Lakshminarasimhan, Damodharan; Suto, Robert K.; Duffy, Bryan C.; Kitchen, Douglas B.; McLure, Kevin G.; Hansen, Henrik C.; van der Horst, Edward H.; Young, Peter R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8202-8211</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET proteins are key epigenetic regulators that regulate transcription through binding to acetylated lysine (AcLys) residues of histones and transcription factors through bromodomains (BDs).  The disruption of this interaction with small mol. bromodomain inhibitors is a promising approach to treat various diseases including cancer, autoimmune and cardiovascular diseases.  Covalent inhibitors can potentially offer a more durable target inhibition leading to improved in vivo pharmacol.  Here the authors describe the design of covalent inhibitors of BRD4(BD1) that target a methionine in the binding pocket by attaching an epoxide warhead to a suitably oriented noncovalent inhibitor.  Using thermal denaturation, MALDI-TOF mass spectrometry, and an x-ray crystal structure, the authors demonstrate that these inhibitors selectively form a covalent bond with Met 149 in BRD4(BD1) but not other bromodomains and provide durable transcriptional and antiproliferative activity in cell based assays.  Covalent targeting of methionine offers a novel approach to drug discovery for BET proteins and other targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHccGRxf5ShrVg90H21EOLACvtfcHk0lilqdetjpgsYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ensLjO&md5=1c3ae393a5f8a8c12ffb16e6f3798118</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00666%26sid%3Dliteratum%253Aachs%26aulast%3DKharenko%26aufirst%3DO.%2BA.%26aulast%3DPatel%26aufirst%3DR.%2BG.%26aulast%3DBrown%26aufirst%3DS.%2BD.%26aulast%3DCalosing%26aufirst%3DC.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DLakshminarasimhan%26aufirst%3DD.%26aulast%3DSuto%26aufirst%3DR.%2BK.%26aulast%3DDuffy%26aufirst%3DB.%2BC.%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%2BH.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26atitle%3DDesign%2520and%2520characterization%2520of%2520novel%2520covalent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520targeting%2520a%2520methionine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8202%26epage%3D8211%26doi%3D10.1021%2Facs.jmedchem.8b00666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lilqdetjpgsYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lilqdetjpgsYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lj9NjUVi6BVjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matzuk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agno, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibition of BRDT for male contraception</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.06.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.cell.2012.06.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22901802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=673-684&author=M.+M.+Matzukauthor=M.+R.+McKeownauthor=P.+Filippakopoulosauthor=Q.+Liauthor=L.+Maauthor=J.+E.+Agnoauthor=M.+E.+Lemieuxauthor=S.+Picaudauthor=R.+N.+Yuauthor=J.+Qiauthor=S.+Knappauthor=J.+E.+Bradner&title=Small-molecule+inhibition+of+BRDT+for+male+contraception&doi=10.1016%2Fj.cell.2012.06.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibition of BRDT for Male Contraception</span></div><div class="casAuthors">Matzuk, Martin M.; McKeown, Michael R.; Filippakopoulos, Panagis; Li, Qinglei; Ma, Lang; Agno, Julio E.; Lemieux, Madeleine E.; Picaud, Sarah; Yu, Richard N.; Qi, Jun; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">673-684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A pharmacol. approach to male contraception remains a longstanding challenge in medicine.  Toward this objective, the authors explored the spermatogenic effects of a selective small-mol. inhibitor (JQ1) of the bromodomain and extraterminal (BET) subfamily of epigenetic reader proteins.  Here, the authors report potent inhibition of the testis-specific member BRDT, which is essential for chromatin remodeling during spermatogenesis.  Biochem. and crystallog. studies confirm that occupancy of the BRDT acetyl-lysine binding pocket by JQ1 prevents recognition of acetylated histone H4.  Treatment of mice with JQ1 reduced seminiferous tubule area, testis size, and spermatozoa no. and motility without affecting hormone levels.  Although JQ1-treated males mate normally, inhibitory effects of JQ1 evident at the spermatocyte and round spermatid stages cause a complete and reversible contraceptive effect.  These data establish a new contraceptive that can cross the blood:testis boundary and inhibit bromodomain activity during spermatogenesis, providing a lead compd. targeting the male germ cell for contraception.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxVWUgXTC77Vg90H21EOLACvtfcHk0lj9NjUVi6BVjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO&md5=ccfa1b443c156612714e4b7f88590041</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.045%26sid%3Dliteratum%253Aachs%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DAgno%26aufirst%3DJ.%2BE.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%2BN.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520BRDT%2520for%2520male%2520contraception%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D673%26epage%3D684%26doi%3D10.1016%2Fj.cell.2012.06.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deguzman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apparsundaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMartino, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogaboam, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denton, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budd, D. C.</span></span> <span> </span><span class="NLM_article-title">Assessment of Brd4 inhibition in idiopathic pulmonary fibrosis lung fibroblasts and in vivo models of lung fibrosis</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">470</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1016/j.ajpath.2013.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.ajpath.2013.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23759512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFynur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2013&pages=470-479&author=X.+Tangauthor=R.+Pengauthor=J.+E.+Phillipsauthor=J.+Deguzmanauthor=Y.+Renauthor=S.+Apparsundaramauthor=Q.+Luoauthor=C.+M.+Bauerauthor=M.+E.+Fuentesauthor=J.+A.+DeMartinoauthor=G.+Tyagiauthor=R.+Garridoauthor=C.+M.+Hogaboamauthor=C.+P.+Dentonauthor=A.+M.+Holmesauthor=C.+Kitsonauthor=C.+S.+Stevensonauthor=D.+C.+Budd&title=Assessment+of+Brd4+inhibition+in+idiopathic+pulmonary+fibrosis+lung+fibroblasts+and+in+vivo+models+of+lung+fibrosis&doi=10.1016%2Fj.ajpath.2013.04.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Brd4 Inhibition in Idiopathic Pulmonary Fibrosis Lung Fibroblasts and in Vivo Models of Lung Fibrosis</span></div><div class="casAuthors">Tang, Xiaoyan; Peng, Ruoqi; Phillips, Jonathan E.; Deguzman, Jeremy; Ren, Yonglin; Apparsundaram, Subramanium; Luo, Qi; Bauer, Carla M.; Fuentes, Maria E.; DeMartino, Julie A.; Tyagi, Gaurav; Garrido, Rosario; Hogaboam, Cory M.; Denton, Christopher P.; Holmes, Alan M.; Kitson, Christopher; Stevenson, Christopher S.; Budd, David C.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">470-479</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Idiopathic pulmonary fibrosis (IPF) is a chronic lung disease of high unmet medical need.  Although bromodomain (Brd) and extra terminal domain isoforms have recently been implicated in mediating inflammatory and oncol. indications, their roles in lung fibrosis have not been comprehensively assessed.  We investigated the role of Brd on the profibrotic responses of lung fibroblasts (LFs) in patients with rapidly progressing IPF and a mouse bleomycin model of lung fibrosis.  The enhanced migration, proliferation, and IL-6 release obsd. in LFs from patients with rapidly progressing IPF are attenuated by pharmacol. inhibition of Brd4.  These changes are accompanied by enhanced histone H4 lysine5 acetylation and assocn. of Brd4 with genes involved in the profibrotic responses in IPF LFs as demonstrated using chromatin immunopptn. and quant. PCR.  Oral administration of 200 mg/kg per day Brd4 inhibitor JQ1 in a therapeutic dosing regimen substantially attenuated lung fibrosis induced by bleomycin in C57BL/6 mice.  In conclusion, this study shows that the Brd4 inhibitor JQ1, administered in a therapeutic dosage, is capable of inhibiting the profibrotic effects of IPF LFs and attenuates bleomycin-induced lung fibrosis in mice.  These results suggest that Brd4 inhibitors may represent a novel therapy for the treatment of rapidly progressing IPF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeyA6Tfi6PObVg90H21EOLACvtfcHk0lg-acPXgoL1tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFynur7I&md5=8a7b094630ce44df6cd1b2b9de6cc6ec</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2013.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2013.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DJ.%2BE.%26aulast%3DDeguzman%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DApparsundaram%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DQ.%26aulast%3DBauer%26aufirst%3DC.%2BM.%26aulast%3DFuentes%26aufirst%3DM.%2BE.%26aulast%3DDeMartino%26aufirst%3DJ.%2BA.%26aulast%3DTyagi%26aufirst%3DG.%26aulast%3DGarrido%26aufirst%3DR.%26aulast%3DHogaboam%26aufirst%3DC.%2BM.%26aulast%3DDenton%26aufirst%3DC.%2BP.%26aulast%3DHolmes%26aufirst%3DA.%2BM.%26aulast%3DKitson%26aufirst%3DC.%26aulast%3DStevenson%26aufirst%3DC.%2BS.%26aulast%3DBudd%26aufirst%3DD.%2BC.%26atitle%3DAssessment%2520of%2520Brd4%2520inhibition%2520in%2520idiopathic%2520pulmonary%2520fibrosis%2520lung%2520fibroblasts%2520and%2520in%2520vivo%2520models%2520of%2520lung%2520fibrosis%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2013%26volume%3D183%26spage%3D470%26epage%3D479%26doi%3D10.1016%2Fj.ajpath.2013.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apparsundaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deguzman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demartino, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budd, D. C.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain proteins mediate downstream signaling events following growth factor stimulation in human lung fibroblasts and are involved in bleomycin-induced pulmonary fibrosis</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1124/mol.112.081661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1124%2Fmol.112.081661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23115324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFKitLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2013&pages=283-293&author=X.+Tangauthor=R.+Pengauthor=Y.+Renauthor=S.+Apparsundaramauthor=J.+Deguzmanauthor=C.+M.+Bauerauthor=A.+F.+Hoffmanauthor=S.+Hamiltonauthor=Z.+Liangauthor=H.+Zengauthor=M.+E.+Fuentesauthor=J.+A.+Demartinoauthor=C.+Kitsonauthor=C.+S.+Stevensonauthor=D.+C.+Budd&title=BET+bromodomain+proteins+mediate+downstream+signaling+events+following+growth+factor+stimulation+in+human+lung+fibroblasts+and+are+involved+in+bleomycin-induced+pulmonary+fibrosis&doi=10.1124%2Fmol.112.081661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain proteins mediate downstream signaling events following growth factor stimulation in human lung fibroblasts and are involved in bleomycin-induced pulmonary fibrosis</span></div><div class="casAuthors">Tang, Xiaoyan; Peng, Ruoqi; Ren, Yonglin; Apparsundaram, Subramanium; Deguzman, Jeremy; Bauer, Carla M.; Hoffman, Ann F.; Hamilton, Shannon; Liang, Zhenmin; Zeng, Hang; Fuentes, Maria E.; DeMartino, Julie A.; Kitson, Christopher; Stevenson, Christopher S.; Budd, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">283-293</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Epigenetic alterations, such as histone acetylation, regulate the signaling outcomes and phenotypic responses of fibroblasts after growth factor stimulation.  The bromodomain and extra-terminal domain-contg. proteins (Brd) bind to acetylated histone residues, resulting in recruitment of components of the transcriptional machinery and subsequent gene transcription.  Given the central importance of fibroblasts in tissue fibrosis, this study sought to det. the role of Brd proteins in human lung fibroblasts (LFs) after growth factor stimulation and in the murine bleomycin model of lung fibrosis.  Using small interfering RNA against human Brd2 and Brd4 and pharmacol. Brd inhibitors, this study found that Brd2 and Brd4 are essential in mediating the phenotypic responses of LFs downstream of multiple growth factor pathways.  Growth factor stimulation of LFs causes increased histone acetylation, assocn. of Brd4 with growth factor-responsive genes, and enhanced transcription of these genes that could be attenuated with pharmacol. Brd inhibitors.  Of note, lung fibrosis induced after intratracheal bleomycin challenge in mice could be prevented by pretreatment of animals with pharmacol. inhibitors of Brd proteins.  This study is the first demonstration of a role for Brd2 and Brd4 proteins in mediating the responses of LFs after growth factor stimulation and in driving the induction of lung fibrosis in mice in response to bleomycin challenge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLn5H_z8-iULVg90H21EOLACvtfcHk0lg-acPXgoL1tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFKitLc%253D&md5=59f634a394e3a0f09b72e07a1eb0ccda</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.081661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.081661%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DR.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DApparsundaram%26aufirst%3DS.%26aulast%3DDeguzman%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DC.%2BM.%26aulast%3DHoffman%26aufirst%3DA.%2BF.%26aulast%3DHamilton%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DFuentes%26aufirst%3DM.%2BE.%26aulast%3DDemartino%26aufirst%3DJ.%2BA.%26aulast%3DKitson%26aufirst%3DC.%26aulast%3DStevenson%26aufirst%3DC.%2BS.%26aulast%3DBudd%26aufirst%3DD.%2BC.%26atitle%3DBET%2520bromodomain%2520proteins%2520mediate%2520downstream%2520signaling%2520events%2520following%2520growth%2520factor%2520stimulation%2520in%2520human%2520lung%2520fibroblasts%2520and%2520are%2520involved%2520in%2520bleomycin-induced%2520pulmonary%2520fibrosis%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D83%26spage%3D283%26epage%3D293%26doi%3D10.1124%2Fmol.112.081661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spiltoir, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavasin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demos-Davies, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinsey, T. A.</span></span> <span> </span><span class="NLM_article-title">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span>. <i>J. Mol. Cell. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1016/j.yjmcc.2013.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.yjmcc.2013.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23939492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2013&pages=175-179&author=J.+I.+Spiltoirauthor=M.+S.+Strattonauthor=M.+A.+Cavasinauthor=K.+Demos-Daviesauthor=B.+G.+Reidauthor=J.+Qiauthor=J.+E.+Bradnerauthor=T.+A.+McKinsey&title=BET+acetyl-lysine+binding+proteins+control+pathological+cardiac+hypertrophy&doi=10.1016%2Fj.yjmcc.2013.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span></div><div class="casAuthors">Spiltoir, Jessica I.; Stratton, Matthew S.; Cavasin, Maria A.; Demos-Davies, Kim; Reid, Brian G.; Qi, Jun; Bradner, James E.; McKinsey, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">175-179</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cardiac hypertrophy is an independent predictor of adverse outcomes in patients with heart failure, and thus represents an attractive target for novel therapeutic intervention.  JQ1, a small mol. inhibitor of bromodomain and extraterminal (BET) acetyl-lysine reader proteins, was identified in a high throughput screen designed to discover novel small mol. regulators of cardiomyocyte hypertrophy.  JQ1 dose-dependently blocked agonist-dependent hypertrophy of cultured neonatal rat ventricular myocytes (NRVMs) and reversed the prototypical gene program assocd. with pathol. cardiac hypertrophy.  JQ1 also blocked left ventricular hypertrophy (LVH) and improved cardiac function in adult mice subjected to transverse aortic constriction (TAC).  The BET family consists of BRD2, BRD3, BRD4 and BRDT.  BRD4 protein expression was increased during cardiac hypertrophy, and hypertrophic stimuli promoted recruitment of BRD4 to the transcriptional start site (TSS) of the gene encoding atrial natriuretic factor (ANF).  Binding of BRD4 to the ANF TSS was assocd. with increased phosphorylation of local RNA polymerase II.  These findings define a novel function for BET proteins as signal-responsive regulators of cardiac hypertrophy, and suggest that small mol. inhibitors of these epigenetic reader proteins have potential as therapeutics for heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDUzCSA2tNtrVg90H21EOLACvtfcHk0lg-acPXgoL1tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF&md5=ffe6d80a94c27e5d89b38baf33af365c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2013.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2013.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSpiltoir%26aufirst%3DJ.%2BI.%26aulast%3DStratton%26aufirst%3DM.%2BS.%26aulast%3DCavasin%26aufirst%3DM.%2BA.%26aulast%3DDemos-Davies%26aufirst%3DK.%26aulast%3DReid%26aufirst%3DB.%2BG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26atitle%3DBET%2520acetyl-lysine%2520binding%2520proteins%2520control%2520pathological%2520cardiac%2520hypertrophy%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2013%26volume%3D63%26spage%3D175%26epage%3D179%26doi%3D10.1016%2Fj.yjmcc.2013.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafiane, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiacchia, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimbou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genest, J.</span></span> <span> </span><span class="NLM_article-title">RVX-208: a small molecule that increases apolipoprotein A-I and high-density lipoprotein cholesterol in vitro and in vivo</span>. <i>J. Am. Coll. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2580</span>– <span class="NLM_lpage">2589</span>, <span class="refDoi"> DOI: 10.1016/j.jacc.2010.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.jacc.2010.02.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=20513599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptV2msr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2010&pages=2580-2589&author=D.+Baileyauthor=R.+Jahagirdarauthor=A.+Gordonauthor=A.+Hafianeauthor=S.+Campbellauthor=S.+Chaturauthor=G.+S.+Wagnerauthor=H.+C.+Hansenauthor=F.+S.+Chiacchiaauthor=J.+Johanssonauthor=L.+Krimbouauthor=N.+C.+W.+Wongauthor=J.+Genest&title=RVX-208%3A+a+small+molecule+that+increases+apolipoprotein+A-I+and+high-density+lipoprotein+cholesterol+in+vitro+and+in+vivo&doi=10.1016%2Fj.jacc.2010.02.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208. A small molecule that increases apolipoprotein A-I and high-density lipoprotein cholesterol in vitro and in vivo</span></div><div class="casAuthors">Bailey, Dana; Jahagirdar, Ravi; Gordon, Allan; Hafiane, Anouar; Campbell, Steven; Chatur, Safia; Wagner, Gregory S.; Hansen, Henrik C.; Chiacchia, Fabrizio S.; Johansson, Jan; Krimbou, Larbi; Wong, Norman C. W.; Genest, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American College of Cardiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2580-2589</span>CODEN:
                <span class="NLM_cas:coden">JACCDI</span>;
        ISSN:<span class="NLM_cas:issn">0735-1097</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Objectives: The aim of this study was to det. whether a novel small mol. RVX-208 affects apolipoprotein (apo)A-I and high-d. lipoprotein cholesterol (HDL-C) levels in vitro and in vivo.  Background: Increased apoA-I and HDL-C levels are potential therapeutic targets for reducing atherosclerotic disease.  Methods: HepG2 cells were treated with 0 to 60 μmol/l RVX-208 followed by assays for apoA-I and HDL-C prodn.  For in vivo studies, African green monkeys (AGMs) received 15 to 60 mg/kg/day RVX-208, and the serum was analyzed for lipoprotein levels, HDL-subparticle distribution, cholesterol efflux, and activity of lipid-modifying enzymes.  A phase I clin. trial was conducted in healthy volunteers (given 1 to 20 mg/kg/day of RVX-208) to assess safety, tolerability, and pharmacokinetics.  Results: The RVX-208 induced apoA-I mRNA and protein synthesis in HepG2 cells, leading to in- creased levels of pre-β-migrating and α-lipoprotein particles contg. apoA-I (LpA-I) in spent media.  Similarly, in AGMs, RVX-208 treatment for 63 days increased serum apoA-I and HDL-C levels (60% and 97%, resp.).  In addn., the levels of pre-β1-LpA-I and α1-LpA-I HDL-subparticles were increased as well as ATP binding cassette AI, ATP binding cassette G1, and scavenger receptor class B type I-dependent cholesterol efflux.  These changes were not mediated by cholesteryl-ester-transfer protein.  Treatment of humans for 1 wk with oral RVX-208 increased apoA-I, pre-β-HDL, and HDL functionality.  Conclusions: RVX-208 increases apoA-I and HDL-C in vitro and in vivo.  In AGMs, RVX-208 raises serum pre-β1-LpA-I and α-LpA-I levels and enhances cholesterol efflux.  Data in humans point to beneficial features of RVX-208 that might be useful for treating atherosclerosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEGExcnx6drLVg90H21EOLACvtfcHk0lj0EzrPUqpaxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptV2msr0%253D&md5=aab9ac26e73704f1cd569e2ff47d0c51</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.jacc.2010.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jacc.2010.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DD.%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DA.%26aulast%3DHafiane%26aufirst%3DA.%26aulast%3DCampbell%26aufirst%3DS.%26aulast%3DChatur%26aufirst%3DS.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DChiacchia%26aufirst%3DF.%2BS.%26aulast%3DJohansson%26aufirst%3DJ.%26aulast%3DKrimbou%26aufirst%3DL.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DGenest%26aufirst%3DJ.%26atitle%3DRVX-208%253A%2520a%2520small%2520molecule%2520that%2520increases%2520apolipoprotein%2520A-I%2520and%2520high-density%2520lipoprotein%2520cholesterol%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DJ.%2520Am.%2520Coll.%2520Cardiol.%26date%3D2010%26volume%3D55%26spage%3D2580%26epage%3D2589%26doi%3D10.1016%2Fj.jacc.2010.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azhar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLure, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, R. A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, J.</span></span> <span> </span><span class="NLM_article-title">A novel BET bromodomain inhibitor, RVX-208, shows reduction of atherosclerosis in hyperlipidemic ApoE deficient mice</span>. <i>Atherosclerosis</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>236</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.atherosclerosis.2014.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.atherosclerosis.2014.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=25016363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Grt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=236&publication_year=2014&pages=91-100&author=R.+Jahagirdarauthor=H.+Zhangauthor=S.+Azharauthor=J.+Tobinauthor=S.+Attwellauthor=R.+Yuauthor=J.+Wuauthor=K.+G.+McLureauthor=H.+C.+Hansenauthor=G.+S.+Wagnerauthor=P.+R.+Youngauthor=R.+A.+K.+Srivastavaauthor=N.+C.+W.+Wongauthor=J.+Johansson&title=A+novel+BET+bromodomain+inhibitor%2C+RVX-208%2C+shows+reduction+of+atherosclerosis+in+hyperlipidemic+ApoE+deficient+mice&doi=10.1016%2Fj.atherosclerosis.2014.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A novel BET bromodomain inhibitor, RVX-208, shows reduction of atherosclerosis in hyperlipidemic ApoE deficient mice</span></div><div class="casAuthors">Jahagirdar, Ravi; Zhang, Haiyan; Azhar, Salman; Tobin, Jennifer; Attwell, Sarah; Yu, Raymond; Wu, Jin; McLure, Kevin G.; Hansen, Henrik C.; Wagner, Gregory S.; Young, Peter R.; Srivastava, Rai Ajit K.; Wong, Norman C. W.; Johansson, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Atherosclerosis (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">236</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-100</span>CODEN:
                <span class="NLM_cas:coden">ATHSBL</span>;
        ISSN:<span class="NLM_cas:issn">0021-9150</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Despite the benefit of statins in reducing cardiovascular risk, a sizable proportion of patients still remain at risk.  Since HDL reduces CVD risk through a process that involves formation of pre-beta particles that facilitates the removal of cholesterol from the lipid-laden macrophages in the arteries, inducing pre-beta particles, may reduce the risk of CVD.  A novel BET bromodomain antagonist, RVX-208, was reported to raise apoA-I and increase preβ-HDL particles in non-human primates and humans.  In the present study, we investigated the effect of RVX-208 on aortic lesion formation in hyperlipidemic apoE-/- mice.  Oral treatments of apoE-/- mice with 150 mg/kg b.i.d RVX-208 for 12 wk significantly reduced aortic lesion formation, accompanied by 2-fold increases in the levels of circulating HDL-C, and ∼50% decreases in LDL-C, although no significant changes in plasma apoA-I were obsd.  Circulating adhesion mols. as well as cytokines also showed significant redn.  Haptoglobin, a proinflammatory protein, known to bind with HDL/apoA-I, decreased >2.5-fold in the RVX-208 treated group.  With a therapeutic dosing regimen in which mice were fed Western diet for 10 wk to develop lesions followed by switching to a low fat diet and concurrent treatment with RVX-208 for 14 wk, RVX-208 similarly reduced lesion formation by 39% in the whole aorta without significant changes in the plasma lipid parameters.  RVX-208 significantly reduced the proinflammatory cytokines IP-10, MIP1 and MDC.  These results show that the antiatherogenic activity of BET inhibitor, RVX-208, occurs via a combination of lipid changes and anti-inflammatory activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXAgzwwmvsdrVg90H21EOLACvtfcHk0lj0EzrPUqpaxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Grt77P&md5=135bbfa7ce02b3684be45e9c39f9e092</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.atherosclerosis.2014.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.atherosclerosis.2014.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DAzhar%26aufirst%3DS.%26aulast%3DTobin%26aufirst%3DJ.%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DMcLure%26aufirst%3DK.%2BG.%26aulast%3DHansen%26aufirst%3DH.%2BC.%26aulast%3DWagner%26aufirst%3DG.%2BS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DSrivastava%26aufirst%3DR.%2BA.%2BK.%26aulast%3DWong%26aufirst%3DN.%2BC.%2BW.%26aulast%3DJohansson%26aufirst%3DJ.%26atitle%3DA%2520novel%2520BET%2520bromodomain%2520inhibitor%252C%2520RVX-208%252C%2520shows%2520reduction%2520of%2520atherosclerosis%2520in%2520hyperlipidemic%2520ApoE%2520deficient%2520mice%26jtitle%3DAtherosclerosis%26date%3D2014%26volume%3D236%26spage%3D91%26epage%3D100%26doi%3D10.1016%2Fj.atherosclerosis.2014.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huthmacher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudgin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doehner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.-I.+Chanauthor=S.+C.+Robsonauthor=C.-w.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+P.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lgti2gWdfbf9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.-I.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delannée, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tellier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudit, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">From ApoA1 upregulation to BET family bromodomain inhibition: discovery of I-BET151</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2963</span>– <span class="NLM_lpage">2967</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.01.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2012.01.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22386529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1elsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2963-2967&author=O.+Mirguetauthor=Y.+Lamotteauthor=F.+Doncheauthor=J.+Toumauthor=F.+Gellibertauthor=A.+Bouillotauthor=R.+Gosminiauthor=V.-L.+Nguyenauthor=D.+Delann%C3%A9eauthor=J.+Sealauthor=F.+Blandelauthor=A.-B.+Boullayauthor=E.+Boursierauthor=S.+Martinauthor=J.-M.+Brusqauthor=G.+Krysaauthor=A.+Riouauthor=R.+Tellierauthor=A.+Costazauthor=P.+Huetauthor=Y.+Duditauthor=L.+Trottetauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=From+ApoA1+upregulation+to+BET+family+bromodomain+inhibition%3A+discovery+of+I-BET151&doi=10.1016%2Fj.bmcl.2012.01.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">From ApoA1 upregulation to BET family bromodomain inhibition: Discovery of I-BET151</span></div><div class="casAuthors">Mirguet, Olivier; Lamotte, Yann; Donche, Frederic; Toum, Jerome; Gellibert, Francoise; Bouillot, Anne; Gosmini, Romain; Nguyen, Van-Loc; Delannee, Delphine; Seal, Jonathan; Blandel, Florence; Boullay, Anne-Benedicte; Boursier, Eric; Martin, Sandrine; Brusq, Jean-Marie; Krysa, Gael; Riou, Alizon; Tellier, Remi; Costaz, Agnes; Huet, Pascal; Dudit, Yann; Trottet, Lionel; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2963-2967</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery, synthesis, and biol. evaluation of a novel series of 7-isoxazoloquinolines is described.  Several analogs are shown to increase ApoA1 expression within the nanomolar range in the human hepatic cell line HepG2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIu6f80tbgE7Vg90H21EOLACvtfcHk0lgti2gWdfbf9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1elsbY%253D&md5=48c9a8dc4b70bea4a86807f7a1cec5f1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.01.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.01.125%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.-L.%26aulast%3DDelann%25C3%25A9e%26aufirst%3DD.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DBlandel%26aufirst%3DF.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DBoursier%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DRiou%26aufirst%3DA.%26aulast%3DTellier%26aufirst%3DR.%26aulast%3DCostaz%26aufirst%3DA.%26aulast%3DHuet%26aufirst%3DP.%26aulast%3DDudit%26aufirst%3DY.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DFrom%2520ApoA1%2520upregulation%2520to%2520BET%2520family%2520bromodomain%2520inhibition%253A%2520discovery%2520of%2520I-BET151%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2963%26epage%3D2967%26doi%3D10.1016%2Fj.bmcl.2012.01.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel series of BET family bromodomain inhibitors: binding mode and profile of I-BET151 (GSK1210151A)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2968</span>– <span class="NLM_lpage">2972</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.02.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2012.02.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22437115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2968-2972&author=J.+Sealauthor=Y.+Lamotteauthor=F.+Doncheauthor=A.+Bouillotauthor=O.+Mirguetauthor=F.+Gellibertauthor=E.+Nicodemeauthor=G.+Krysaauthor=J.+Kirilovskyauthor=S.+Beinkeauthor=S.+McClearyauthor=I.+Riojaauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=L.+Gordonauthor=T.+Lewisauthor=A.+L.+Walkerauthor=L.+Cutlerauthor=D.+Lugoauthor=D.+M.+Wilsonauthor=J.+Witheringtonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Identification+of+a+novel+series+of+BET+family+bromodomain+inhibitors%3A+binding+mode+and+profile+of+I-BET151+%28GSK1210151A%29&doi=10.1016%2Fj.bmcl.2012.02.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span></div><div class="casAuthors">Seal, Jonathan; Lamotte, Yann; Donche, Frederic; Bouillot, Anne; Mirguet, Olivier; Gellibert, Francoise; Nicodeme, Edwige; Krysa, Gael; Kirilovsky, Jorge; Beinke, Soren; McCleary, Scott; Rioja, Inma; Bamborough, Paul; Chung, Chun-Wa; Gordon, Laurie; Lewis, Toni; Walker, Ann L.; Cutler, Leanne; Lugo, David; Wilson, David M.; Witherington, Jason; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2968-2972</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of quinoline isoxazole BET family bromodomain inhibitors are discussed.  Crystallog. is used to illustrate binding modes and rationalize their SAR.  One member, I-BET151 (GSK1210151A), shows good oral bioavailability in both the rat and minipig as well as demonstrating efficient suppression of bacterial induced inflammation and sepsis in a murine in vivo endotoxemia model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJKtxBCr-pRbVg90H21EOLACvtfcHk0lgti2gWdfbf9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D&md5=a8037c89bcb93948d663208bd1ebfd55</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.041%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DIdentification%2520of%2520a%2520novel%2520series%2520of%2520BET%2520family%2520bromodomain%2520inhibitors%253A%2520binding%2520mode%2520and%2520profile%2520of%2520I-BET151%2520%2528GSK1210151A%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2968%26epage%3D2972%26doi%3D10.1016%2Fj.bmcl.2012.02.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bish, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nocka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primiano, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralph, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciammetta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzupek, J. D.</span></span> <span> </span><span class="NLM_article-title">Identification of a chemical probe for bromo and extra C-terminal bromodomain inhibition through optimization of a fragment-derived hit</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9831</span>– <span class="NLM_lpage">9837</span>, <span class="refDoi"> DOI: 10.1021/jm3010515</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3010515" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9831-9837&author=P.+V.+Fishauthor=P.+Filippakopoulosauthor=G.+Bishauthor=P.+E.+Brennanauthor=M.+E.+Bunnageauthor=A.+S.+Cookauthor=O.+Federovauthor=B.+S.+Gerstenbergerauthor=H.+Jonesauthor=S.+Knappauthor=B.+Marsdenauthor=K.+Nockaauthor=D.+R.+Owenauthor=M.+Philpottauthor=S.+Picaudauthor=M.+J.+Primianoauthor=M.+J.+Ralphauthor=N.+Sciammettaauthor=J.+D.+Trzupek&title=Identification+of+a+chemical+probe+for+bromo+and+extra+C-terminal+bromodomain+inhibition+through+optimization+of+a+fragment-derived+hit&doi=10.1021%2Fjm3010515"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Chemical Probe for Bromo and Extra C-Terminal Bromodomain Inhibition through Optimization of a Fragment-Derived Hit</span></div><div class="casAuthors">Fish, Paul V.; Filippakopoulos, Panagis; Bish, Gerwyn; Brennan, Paul E.; Bunnage, Mark E.; Cook, Andrew S.; Federov, Oleg; Gerstenberger, Brian S.; Jones, Hannah; Knapp, Stefan; Marsden, Brian; Nocka, Karl; Owen, Dafydd R.; Philpott, Martin; Picaud, Sarah; Primiano, Michael J.; Ralph, Michael J.; Sciammetta, Nunzio; Trzupek, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9831-9837</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The posttranslational modification of chromatin through acetylation at selected histone lysine residues is governed by histone acetyltransferases (HATs) and histone deacetylases (HDACs).  The significance of this subset of the epigenetic code is interrogated and interpreted by an acetyllysine-specific protein-protein interaction with bromodomain reader modules.  Selective inhibition of the bromo and extra C-terminal domain (BET) family of bromodomains with a small mol. is feasible, and this may represent an opportunity for disease intervention through the recently disclosed antiproliferative and anti-inflammatory properties of such inhibitors.  Herein, we describe the discovery and structure-activity relationship (SAR) of a novel, small-mol. chem. probe for BET family inhibition that was identified through the application of structure-based fragment assessment and optimization techniques.  This has yielded a potent, selective compd. with cell-based activity (PFI-1) that may further add to the understanding of BET family function within the bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphzpWnuXoLdrVg90H21EOLACvtfcHk0ljMpKo3QVgr1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGmtLnE&md5=ba864f3916bea5f793dd0b26732784f0</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm3010515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3010515%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBish%26aufirst%3DG.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DFederov%26aufirst%3DO.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPrimiano%26aufirst%3DM.%2BJ.%26aulast%3DRalph%26aufirst%3DM.%2BJ.%26aulast%3DSciammetta%26aufirst%3DN.%26aulast%3DTrzupek%26aufirst%3DJ.%2BD.%26atitle%3DIdentification%2520of%2520a%2520chemical%2520probe%2520for%2520bromo%2520and%2520extra%2520C-terminal%2520bromodomain%2520inhibition%2520through%2520optimization%2520of%2520a%2520fragment-derived%2520hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9831%26epage%3D9837%26doi%3D10.1021%2Fjm3010515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wyce, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganji, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smitheman, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy-Wilson, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span> <span> </span><span class="NLM_article-title">BET inhibition silences expression of MYCN and BCL2 and induces cytotoxicity in neuroblastoma tumor models</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e72967</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0072967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1371%2Fjournal.pone.0072967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=24009722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2iu7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=A.+Wyceauthor=G.+Ganjiauthor=K.+N.+Smithemanauthor=C.-w.+Chungauthor=S.+Korenchukauthor=Y.+Baiauthor=O.+Barbashauthor=B.+Leauthor=P.+D.+Craggsauthor=M.+T.+McCabeauthor=K.+M.+Kennedy-Wilsonauthor=L.+V.+Sanchezauthor=R.+L.+Gosminiauthor=N.+Parrauthor=C.+F.+McHughauthor=D.+Dhanakauthor=R.+K.+Prinjhaauthor=K.+R.+Augerauthor=P.+J.+Tummino&title=BET+inhibition+silences+expression+of+MYCN+and+BCL2+and+induces+cytotoxicity+in+neuroblastoma+tumor+models&doi=10.1371%2Fjournal.pone.0072967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibition silences expression of MYCN and BCL2 and induces cytotoxicity in neuroblastoma tumor models</span></div><div class="casAuthors">Wyce, Anastasia; Ganji, Gopinath; Smitheman, Kimberly N.; Chung, Chun-Wa; Korenchuk, Susan; Bai, Yuchen; Barbash, Olena; Le, Bao Chau; Craggs, Peter D.; McCabe, Michael T.; Kennedy-Wilson, Karen M.; Sanchez, Lydia V.; Gosmini, Romain L.; Parr, Nigel; McHugh, Charles F.; Dhanak, Dashyant; Prinjha, Rab K.; Auger, Kurt R.; Tummino, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e72967</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">BET family proteins are epigenetic regulators known to control expression of genes involved in cell growth and oncogenesis.  Selective inhibitors of BET proteins exhibit potent anti-proliferative activity in a no. of hematol. cancer models, in part through suppression of the MYC oncogene and downstream Myc-driven pathways.  However, little is currently known about the activity of BET inhibitors in solid tumor models, and whether down-regulation of MYC family genes contributes to sensitivity.  Here we provide evidence for potent BET inhibitor activity in neuroblastoma, a pediatric solid tumor assocd. with a high frequency of MYCN amplifications.  We treated a panel of neuroblastoma cell lines with a novel small mol. inhibitor of BET proteins, GSK1324726A (I-BET726), and obsd. potent growth inhibition and cytotoxicity in most cell lines irresp. of MYCN copy no. or expression level.  Gene expression analyses in neuroblastoma cell lines suggest a role of BET inhibition in apoptosis, signaling, and N-Myc-driven pathways, including the direct suppression of BCL2 and MYCN.  Reversal of MYCN or BCL2 suppression reduces the potency of I-BET726-induced cytotoxicity in a cell line-specific manner; however, neither factor fully accounts for I-BET726 sensitivity.  Oral administration of I-BET726 to mouse xenograft models of human neuroblastoma results in tumor growth inhibition and down-regulation MYCN and BCL2 expression, suggesting a potential role for these genes in tumor growth.  Taken together, our data highlight the potential of BET inhibitors as novel therapeutics for neuroblastoma, and suggest that sensitivity is driven by pleiotropic effects on cell growth and apoptotic pathways in a context-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecB0_0elFVbVg90H21EOLACvtfcHk0liw12Cz2A5drQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2iu7fI&md5=d39ce25a50de58f18a278a9f67935bbc</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0072967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0072967%26sid%3Dliteratum%253Aachs%26aulast%3DWyce%26aufirst%3DA.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DKennedy-Wilson%26aufirst%3DK.%2BM.%26aulast%3DSanchez%26aufirst%3DL.%2BV.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26atitle%3DBET%2520inhibition%2520silences%2520expression%2520of%2520MYCN%2520and%2520BCL2%2520and%2520induces%2520cytotoxicity%2520in%2520neuroblastoma%2520tumor%2520models%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0072967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou-Eymard, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8111</span>– <span class="NLM_lpage">8131</span>, <span class="refDoi"> DOI: 10.1021/jm5010539</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5010539" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8111-8131&author=R.+Gosminiauthor=V.+L.+Nguyenauthor=J.+Toumauthor=C.+Simonauthor=J.-M.+G.+Brusqauthor=G.+Krysaauthor=O.+Mirguetauthor=A.+M.+Riou-Eymardauthor=E.+V.+Boursierauthor=L.+Trottetauthor=P.+Bamboroughauthor=H.+Clarkauthor=C.-w.+Chungauthor=L.+Cutlerauthor=E.+H.+Demontauthor=R.+Kaurauthor=A.+J.+Lewisauthor=M.+B.+Schillingauthor=P.+E.+Sodenauthor=S.+Taylorauthor=A.+L.+Walkerauthor=M.+D.+Walkerauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=The+Discovery+of+I-BET726+%28GSK1324726A%29%2C+a+Potent+Tetrahydroquinoline+ApoA1+Up-Regulator+and+Selective+BET+Bromodomain+Inhibitor&doi=10.1021%2Fjm5010539"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of I-BET726 (GSK1324726A), a Potent Tetrahydroquinoline ApoA1 Up-Regulator and Selective BET Bromodomain Inhibitor</span></div><div class="casAuthors">Gosmini, Romain; Nguyen, Van Loc; Toum, Jerome; Simon, Christophe; Brusq, Jean-Marie G.; Krysa, Gael; Mirguet, Olivier; Riou-Eymard, Alizon M.; Boursier, Eric V.; Trottet, Lionel; Bamborough, Paul; Clark, Hugh; Chung, Chun-wa; Cutler, Leanne; Demont, Emmanuel H.; Kaur, Rejbinder; Lewis, Antonia J.; Schilling, Mark B.; Soden, Peter E.; Taylor, Simon; Walker, Ann L.; Walker, Matthew D.; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8111-8131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Through their function as epigenetic readers of the histone code, the BET family of bromodomain-contg. proteins regulate expression of multiple genes of therapeutic relevance, including those involved in tumor cell growth and inflammation.  BET bromodomain inhibitors have profound antiproliferative and anti-inflammatory effects which translate into efficacy in oncol. and inflammation models, and the first compds. have now progressed into clin. trials.  The exciting biol. of the BETs has led to great interest in the discovery of novel inhibitor classes.  Here we describe the identification of a novel tetrahydroquinoline series through up-regulation of apolipoprotein A1 and the optimization into potent compds. active in murine models of septic shock and neuroblastoma.  At the mol. level, these effects are produced by inhibition of BET bromodomains.  X-ray crystallog. reveals the interactions explaining the structure-activity relationships of binding.  The resulting lead mol., I-BET726, represents a new, potent, and selective class of tetrahydroquinoline-based BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5OdMn_TjoFLVg90H21EOLACvtfcHk0liw12Cz2A5drQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wlur7J&md5=b1c9c9cae73c41252cf73908d561bd9b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm5010539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5010539%26sid%3Dliteratum%253Aachs%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DSimon%26aufirst%3DC.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DRiou-Eymard%26aufirst%3DA.%2BM.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DClark%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DSchilling%26aufirst%3DM.%2BB.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DWalker%26aufirst%3DM.%2BD.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DThe%2520Discovery%2520of%2520I-BET726%2520%2528GSK1324726A%2529%252C%2520a%2520Potent%2520Tetrahydroquinoline%2520ApoA1%2520Up-Regulator%2520and%2520Selective%2520BET%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8111%26epage%3D8131%26doi%3D10.1021%2Fjm5010539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowlati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagey, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, S. B.</span></span> <span> </span><span class="NLM_article-title">Abstract A49: Clinically efficacy of the BET bromodomain inhibitor TEN-010 in an open-label substudy with patients with documented NUT-midline carcinoma (NMC)</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">A49</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.TARG-15-A49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1158%2F1535-7163.TARG-15-A49" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=A49&author=G.+I.+Shapiroauthor=A.+Dowlatiauthor=P.+M.+LoRussoauthor=J.+P.+Ederauthor=A.+Andersonauthor=K.+T.+Doauthor=M.+H.+Kageyauthor=C.+Sirardauthor=J.+E.+Bradnerauthor=S.+B.+Landau&title=Abstract+A49%3A+Clinically+efficacy+of+the+BET+bromodomain+inhibitor+TEN-010+in+an+open-label+substudy+with+patients+with+documented+NUT-midline+carcinoma+%28NMC%29&doi=10.1158%2F1535-7163.TARG-15-A49"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-A49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-A49%26sid%3Dliteratum%253Aachs%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DDowlati%26aufirst%3DA.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DAnderson%26aufirst%3DA.%26aulast%3DDo%26aufirst%3DK.%2BT.%26aulast%3DKagey%26aufirst%3DM.%2BH.%26aulast%3DSirard%26aufirst%3DC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLandau%26aufirst%3DS.%2BB.%26atitle%3DAbstract%2520A49%253A%2520Clinically%2520efficacy%2520of%2520the%2520BET%2520bromodomain%2520inhibitor%2520TEN-010%2520in%2520an%2520open-label%2520substudy%2520with%2520patients%2520with%2520documented%2520NUT-midline%2520carcinoma%2520%2528NMC%2529%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DA49%26doi%3D10.1158%2F1535-7163.TARG-15-A49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panaroni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N.</span></span> <span> </span><span class="NLM_article-title">Preclinical activity of CPI-0610, a novel small-molecule bromodomain and extra-terminal protein inhibitor in the therapy of multiple myeloma</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1760</span>– <span class="NLM_lpage">1769</span>, <span class="refDoi"> DOI: 10.1038/leu.2016.355</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fleu.2016.355" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27890933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Kq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=1760-1769&author=K.+T.+Siuauthor=J.+Ramachandranauthor=A.+J.+Yeeauthor=H.+Edaauthor=L.+Santoauthor=C.+Panaroniauthor=J.+A.+Mertzauthor=R.+J.+Simsauthor=M.+R.+Cooperauthor=N.+Raje&title=Preclinical+activity+of+CPI-0610%2C+a+novel+small-molecule+bromodomain+and+extra-terminal+protein+inhibitor+in+the+therapy+of+multiple+myeloma&doi=10.1038%2Fleu.2016.355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical activity of CPI-0610, a novel small-molecule bromodomain and extra-terminal protein inhibitor in the therapy of multiple myeloma</span></div><div class="casAuthors">Siu, K. T.; Ramachandran, J.; Yee, A. J.; Eda, H.; Santo, L.; Panaroni, C.; Mertz, J. A.; Sims, R. J., III; Cooper, M. R.; Raje, N.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1760-1769</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inhibition of the bromodomain and extra-terminal (BET) proteins is a promising therapeutic strategy for various hematol. cancers.  Previous studies suggest that BET inhibitors constrain tumor cell proliferation and survival mainly through the suppression of MYC transcription and activity.  However, suppression of the transcription of addnl. genes also contributes to the antitumor activity of BET inhibitors but is less well understood.  Here we examd. the therapeutic potential of CPI-0610, a potent BET inhibitor currently undergoing phase I clin. testing, in multiple myeloma (MM).  CPI-0610 displays potent cytotoxicity against MM cell lines and patient-derived MM cells through G1 cell cycle arrest and caspase-dependent apoptosis.  CPI-0610-mediated BET inhibition overcomes the protective effects conferred by cytokines and bone marrow stromal cells.  We also confirmed the in vivo efficacy of CPI-0610 in a MM xenograft mouse model.  Our study found IKZF1 and IRF4 to be among the primary targets of CPI-0610, along with MYC.  Given that immunomodulatory drugs (IMiDs) stabilize cereblon and facilitate Ikaros degrdn. in MM cells, we combined it with CPI-0610.  Combination studies of CPI-0610 with IMiDs show in vitro synergism, in part due to concomitant suppression of IKZF1, IRF4 and MYC, providing a rationale for clin. testing of this drug combination in MM patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod3GErt1kMIbVg90H21EOLACvtfcHk0lher-0zkLVzJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Kq&md5=d0a386df69df69d187e10d4e784b986a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fleu.2016.355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2016.355%26sid%3Dliteratum%253Aachs%26aulast%3DSiu%26aufirst%3DK.%2BT.%26aulast%3DRamachandran%26aufirst%3DJ.%26aulast%3DYee%26aufirst%3DA.%2BJ.%26aulast%3DEda%26aufirst%3DH.%26aulast%3DSanto%26aufirst%3DL.%26aulast%3DPanaroni%26aufirst%3DC.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DRaje%26aufirst%3DN.%26atitle%3DPreclinical%2520activity%2520of%2520CPI-0610%252C%2520a%2520novel%2520small-molecule%2520bromodomain%2520and%2520extra-terminal%2520protein%2520inhibitor%2520in%2520the%2520therapy%2520of%2520multiple%2520myeloma%26jtitle%3DLeukemia%26date%3D2017%26volume%3D31%26spage%3D1760%26epage%3D1769%26doi%3D10.1038%2Fleu.2016.355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span> <span> </span><span class="NLM_article-title">Identification of a benzoisoxazoloazepine inhibitor (CPI-0610) of the bromodomain and extra-terminal (BET) family as a candidate for human clinical trials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1330</span>– <span class="NLM_lpage">1339</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1330-1339&author=B.+K.+Albrechtauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=A.+C%C3%B4t%C3%A9author=Y.+Leblancauthor=C.+G.+Nasveschukauthor=S.+Bellonauthor=L.+Bergeronauthor=R.+Campbellauthor=N.+Cantoneauthor=M.+R.+Cooperauthor=R.+T.+Cummingsauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Normantauthor=M.+O%E2%80%99Mearaauthor=E.+Pardoauthor=F.+Poyauthor=P.+Sandyauthor=J.+Supkoauthor=R.+J.+Simsauthor=J.-C.+Harmangeauthor=A.+M.+Taylorauthor=J.+E.+Audia&title=Identification+of+a+benzoisoxazoloazepine+inhibitor+%28CPI-0610%29+of+the+bromodomain+and+extra-terminal+%28BET%29+family+as+a+candidate+for+human+clinical+trials&doi=10.1021%2Facs.jmedchem.5b01882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span></div><div class="casAuthors">Albrecht, Brian K.; Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Cote, Alexandre; Leblanc, Yves; Nasveschuk, Christopher G.; Bellon, Steve; Bergeron, Louise; Campbell, Robert; Cantone, Nico; Cooper, Michael R.; Cummings, Richard T.; Jayaram, Hariharan; Joshi, Shivangi; Mertz, Jennifer A.; Neiss, Adrianne; Normant, Emmanuel; O'Meara, Michael; Pardo, Eneida; Poy, Florence; Sandy, Peter; Supko, Jeffrey; Sims, Robert J.; Harmange, Jean-Christophe; Taylor, Alexander M.; Audia, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1330-1339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, inhibition of the interaction between the bromodomain and extra-terminal domain (BET) family of chromatin adaptors and acetyl-lysine residues on chromatin has emerged as a promising approach to regulate the expression of important disease-relevant genes, including MYC, BCL-2, and NF-κB.  Here we describe the identification and characterization of a potent and selective benzoisoxazoloazepine BET bromodomain inhibitor that attenuates BET-dependent gene expression in vivo, demonstrates antitumor efficacy in an MV-4-11 mouse xenograft model, and is currently undergoing human clin. trials for hematol. malignancies (CPI-0610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp40qjnYl4EZLVg90H21EOLACvtfcHk0lher-0zkLVzJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D&md5=5e1c5aab26aa58844a9f6670a7bbd78b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01882%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSupko%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26atitle%3DIdentification%2520of%2520a%2520benzoisoxazoloazepine%2520inhibitor%2520%2528CPI-0610%2529%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520family%2520as%2520a%2520candidate%2520for%2520human%2520clinical%2520trials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1330%26epage%3D1339%26doi%3D10.1021%2Facs.jmedchem.5b01882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span> <span> </span><span class="NLM_article-title">Development of methyl isoxazoleazepines as inhibitors of BET</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1842</span>– <span class="NLM_lpage">1848</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.03.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2015.03.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=25851940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVKrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1842-1848&author=M.+C.+Hewittauthor=Y.+Leblancauthor=V.+S.+Gehlingauthor=R.+G.+Vaswaniauthor=A.+C%C3%B4t%C3%A9author=C.+G.+Nasveschukauthor=A.+M.+Taylorauthor=J.-C.+Harmangeauthor=J.+E.+Audiaauthor=E.+Pardoauthor=R.+Cummingsauthor=S.+Joshiauthor=P.+Sandyauthor=J.+A.+Mertzauthor=R.+J.+Simsauthor=L.+Bergeronauthor=B.+M.+Bryantauthor=S.+Bellonauthor=F.+Poyauthor=H.+Jayaramauthor=Y.+Tangauthor=B.+K.+Albrecht&title=Development+of+methyl+isoxazoleazepines+as+inhibitors+of+BET&doi=10.1016%2Fj.bmcl.2015.03.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Development of methyl isoxazoleazepines as inhibitors of BET</span></div><div class="casAuthors">Hewitt, Michael C.; Leblanc, Yves; Gehling, Victor S.; Vaswani, Rishi G.; Cote, Alexandre; Nasveschuk, Christopher G.; Taylor, Alexander M.; Harmange, Jean-Christophe; Audia, James E.; Pardo, Eneida; Cummings, Rich; Joshi, Shivangi; Sandy, Peter; Mertz, Jennifer A.; Sims, Robert J.; Bergeron, Louise; Bryant, Barbara M.; Bellon, Steve; Poy, Florence; Jayaran, Hariharan; Tang, Yong; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1842-1848</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this report we detail the evolution of our previously reported thiophene isoxazole BET inhibitor chemotype exemplified by CPI-3 to a novel bromodomain selective chemotype (the Me isoxazoleazepine chemotype) exemplified by carboxamide 23.  The Me isoxazoleazepine chemotype provides potent inhibition of the bromodomains of the BET family, excellent in vivo PK across species, low unbound clearance, and target engagement in a MYC PK-PD model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrdQuVhIoU1rVg90H21EOLACvtfcHk0ljFoM_oWT0kzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVKrsL8%253D&md5=fb9a78528287f82ccf004e751e491ee4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.03.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.03.045%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DDevelopment%2520of%2520methyl%2520isoxazoleazepines%2520as%2520inhibitors%2520of%2520BET%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D1842%26epage%3D1848%26doi%3D10.1016%2Fj.bmcl.2015.03.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setser, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzotriazolo[4,3-d][1,4]diazepines as orally active inhibitors of BET bromodomains</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1021/ml500411h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500411h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlt1Wls7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=145-150&author=A.+M.+Taylorauthor=R.+G.+Vaswaniauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=Y.+Leblancauthor=J.+E.+Audiaauthor=S.+Bellonauthor=R.+T.+Cummingsauthor=A.+C%C3%B4t%C3%A9author=J.-C.+Harmangeauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+M.+Loraauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Pardoauthor=C.+G.+Nasveschukauthor=F.+Poyauthor=P.+Sandyauthor=J.+W.+Setserauthor=R.+J.+Simsauthor=Y.+Tangauthor=B.+K.+Albrecht&title=Discovery+of+benzotriazolo%5B4%2C3-d%5D%5B1%2C4%5Ddiazepines+as+orally+active+inhibitors+of+BET+bromodomains&doi=10.1021%2Fml500411h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzotriazolo[4,3-d][1,4]diazepines as Orally Active Inhibitors of BET Bromodomains</span></div><div class="casAuthors">Taylor, Alexander M.; Vaswani, Rishi G.; Gehling, Victor S.; Hewitt, Michael C.; Leblanc, Yves; Audia, James E.; Bellon, Steve; Cummings, Richard T.; Cote, Alexandre; Harmange, Jean-Christophe; Jayaram, Hari; Joshi, Shivangi; Lora, Jose M.; Mertz, Jennifer A.; Neiss, Adrianne; Pardo, Eneida; Nasveschuk, Christopher G.; Poy, Florence; Sandy, Peter; Setser, Jeremy W.; Sims, Robert J.; Tang, Yong; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-150</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of the bromodomains of the BET family, of which BRD4 is a member, has been shown to decrease myc and interleukin (IL) 6 in vivo, markers that are of therapeutic relevance to cancer and inflammatory disease, resp.  Herein the authors report substituted benzo[b]isoxazolo[4,5-d]azepines and benzotriazolo[4,3-d][1,4]diazepines as fragment-derived novel inhibitors of the bromodomain of BRD4.  Compds. from these series were potent and selective in cells, and subsequent optimization of microsomal stability yielded representatives that demonstrated dose- and time-dependent redn. of plasma IL-6 in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDybfuij7kebVg90H21EOLACvtfcHk0ljFoM_oWT0kzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlt1Wls7k%253D&md5=38759c818814869d6c951a035bad7617</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fml500411h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500411h%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSetser%26aufirst%3DJ.%2BW.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DDiscovery%2520of%2520benzotriazolo%255B4%252C3-d%255D%255B1%252C4%255Ddiazepines%2520as%2520orally%2520active%2520inhibitors%2520of%2520BET%2520bromodomains%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D145%26epage%3D150%26doi%3D10.1021%2Fml500411h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankaranarayanan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yellapantula, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bangalore Srinivasamurthy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birudukota, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span> <span> </span><span class="NLM_article-title">Discovery, design, and optimization of isoxazole azepine BET inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">835</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1021/ml4001485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4001485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFanu7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=835-840&author=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=Y.+Leblancauthor=A.+C%C3%B4t%C3%A9author=C.+G.+Nasveschukauthor=A.+M.+Taylorauthor=J.-C.+Harmangeauthor=J.+E.+Audiaauthor=E.+Pardoauthor=S.+Joshiauthor=P.+Sandyauthor=J.+A.+Mertzauthor=R.+J.+Simsauthor=L.+Bergeronauthor=B.+M.+Bryantauthor=S.+Bellonauthor=F.+Poyauthor=H.+Jayaramauthor=R.+Sankaranarayananauthor=S.+Yellapantulaauthor=N.+Bangalore+Srinivasamurthyauthor=S.+Birudukotaauthor=B.+K.+Albrecht&title=Discovery%2C+design%2C+and+optimization+of+isoxazole+azepine+BET+inhibitors&doi=10.1021%2Fml4001485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, Design, and Optimization of Isoxazole Azepine BET Inhibitors</span></div><div class="casAuthors">Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Leblanc, Yves; Cote, Alexandre; Nasveschuk, Christopher G.; Taylor, Alexander M.; Harmange, Jean-Christophe; Audia, James E.; Pardo, Eneida; Joshi, Shivangi; Sandy, Peter; Mertz, Jennifer A.; Sims, Robert J.; Bergeron, Louise; Bryant, Barbara M.; Bellon, Steve; Poy, Florence; Jayaram, Hariharan; Sankaranarayanan, Ravichandran; Yellapantula, Sreegouri; Bangalore Srinivasamurthy, Nandana; Birudukota, Swarnakumari; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">835-840</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of a novel series of small mol. BET inhibitors is described.  Using crystallog. binding modes of an amino-isoxazole fragment and known BET inhibitors, a structure-based drug design effort lead to a novel isoxazole azepine scaffold, e.g. I.  This scaffold showed good potency in biochem. and cellular assays and oral activity in an in vivo model of BET inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCIGHOq7LlgrVg90H21EOLACvtfcHk0lgpGIojo912sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFanu7jI&md5=7a3ab8ec50ba88e865c5a6524cf02c48</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fml4001485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4001485%26sid%3Dliteratum%253Aachs%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSankaranarayanan%26aufirst%3DR.%26aulast%3DYellapantula%26aufirst%3DS.%26aulast%3DBangalore%2BSrinivasamurthy%26aufirst%3DN.%26aulast%3DBirudukota%26aufirst%3DS.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DDiscovery%252C%2520design%252C%2520and%2520optimization%2520of%2520isoxazole%2520azepine%2520BET%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D835%26epage%3D840%26doi%3D10.1021%2Fml4001485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thieblemont, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffoux, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarotto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leleu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Roca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odore, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roumier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourdel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesnel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michallet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preudhomme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palumbo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombret, H.</span></span> <span> </span><span class="NLM_article-title">Abstract CT231: BET-bromodomain inhibitor OTX015 shows clinically meaningful activity at nontoxic doses: interim results of an ongoing phase I trial in hematologic malignancies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">CT231</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2014-CT231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1158%2F1538-7445.AM2014-CT231" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=CT231&author=P.+E.+Heraitauthor=C.+Berthonauthor=C.+Thieblemontauthor=E.+Raffouxauthor=V.+Magarottoauthor=A.+Stathisauthor=X.+Thomasauthor=X.+Leleuauthor=C.+Gomez-Rocaauthor=E.+Odoreauthor=C.+Roumierauthor=F.+Bourdelauthor=B.+Quesnelauthor=E.+Zuccaauthor=M.+Michalletauthor=C.+Recherauthor=E.+Cvitkovicauthor=K.+Rezaiauthor=C.+Preudhommeauthor=T.+Faconauthor=A.+Palumboauthor=H.+Dombret&title=Abstract+CT231%3A+BET-bromodomain+inhibitor+OTX015+shows+clinically+meaningful+activity+at+nontoxic+doses%3A+interim+results+of+an+ongoing+phase+I+trial+in+hematologic+malignancies&doi=10.1158%2F1538-7445.AM2014-CT231"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2014-CT231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2014-CT231%26sid%3Dliteratum%253Aachs%26aulast%3DHerait%26aufirst%3DP.%2BE.%26aulast%3DBerthon%26aufirst%3DC.%26aulast%3DThieblemont%26aufirst%3DC.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DMagarotto%26aufirst%3DV.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DX.%26aulast%3DLeleu%26aufirst%3DX.%26aulast%3DGomez-Roca%26aufirst%3DC.%26aulast%3DOdore%26aufirst%3DE.%26aulast%3DRoumier%26aufirst%3DC.%26aulast%3DBourdel%26aufirst%3DF.%26aulast%3DQuesnel%26aufirst%3DB.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DMichallet%26aufirst%3DM.%26aulast%3DRecher%26aufirst%3DC.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DPreudhomme%26aufirst%3DC.%26aulast%3DFacon%26aufirst%3DT.%26aulast%3DPalumbo%26aufirst%3DA.%26aulast%3DDombret%26aufirst%3DH.%26atitle%3DAbstract%2520CT231%253A%2520BET-bromodomain%2520inhibitor%2520OTX015%2520shows%2520clinically%2520meaningful%2520activity%2520at%2520nontoxic%2520doses%253A%2520interim%2520results%2520of%2520an%2520ongoing%2520phase%2520I%2520trial%2520in%2520hematologic%2520malignancies%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3DCT231%26doi%3D10.1158%2F1538-7445.AM2014-CT231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Roca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouge, T. d. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uro-Coste, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Braud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelosi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span> <span> </span><span class="NLM_article-title">Clinical response of carcinomas harboring the BRD4-NUT oncoprotein to the targeted bromodomain inhibitor OTX015/MK-8628</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">492</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-15-1335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1158%2F2159-8290.cd-15-1335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=26976114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlOqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=492-500&author=A.+Stathisauthor=E.+Zuccaauthor=M.+Bekraddaauthor=C.+Gomez-Rocaauthor=J.-P.+Delordauthor=T.+d.+L.+M.+Rougeauthor=E.+Uro-Costeauthor=F.+de+Braudauthor=G.+Pelosiauthor=C.+A.+French&title=Clinical+response+of+carcinomas+harboring+the+BRD4-NUT+oncoprotein+to+the+targeted+bromodomain+inhibitor+OTX015%2FMK-8628&doi=10.1158%2F2159-8290.cd-15-1335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Response of Carcinomas Harboring the BRD4-NUT Oncoprotein to the Targeted Bromodomain Inhibitor OTX015/MK-8628</span></div><div class="casAuthors">Stathis, Anastasios; Zucca, Emanuele; Bekradda, Mohamed; Gomez-Roca, Carlos; Delord, Jean-Pierre; de La Motte Rouge, Thibault; Uro-Coste, Emmanuelle; de Braud, Filippo; Pelosi, Giuseppe; French, Christopher A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">492-500</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The antineoplastic, prodifferentiative effects of bromodomain and extra-terminal (BET) bromodomain (BRD) inhibitors were initially discovered in NUT midline carcinoma (NMC), an aggressive subtype of squamous cancer driven by the BRD4-NUT fusion oncoprotein.  BRD4-NUT blocks differentiation and maintains tumor growth through a potent chromatin-modifying mechanism.  OTX015/MK-8628, a novel oral BET inhibitor, targets BRD2/3/4/T with preclin. activity in NMC and several other tumor types and is currently in clin. development.  Antitumor activity was evaluated in four patients with advanced-stage NMC with confirmed BRD4-NUT fusions who were treated with 80 mg OTX015/MK-8628 once daily in a compassionate-use context.  Two patients responded rapidly with tumor regression and symptomatic relief, and a third had meaningful disease stabilization with a minor metabolic response.  The main side effects were mild to moderate gastrointestinal toxicity and fatigue, and reversible grade 3 thrombocytopenia.  This is the first proof-of-concept evidence of clin. activity of a BRD inhibitor in targeting BRD4-NUT.  Significance: We present the first clin. proof-of-concept that targeting BRD4-NUT with a BET inhibitor results in impressive and rapid antitumor activity in NMC.  It offers strong potential for future clin. application in this rare patient population as either a single agent or in combination with other agents.  Cancer Discov; 6(5); 492-500. ©2016 AACR.  This article is highlighted in the In This Issue feature, p.  461.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGqXhyj-x5pLVg90H21EOLACvtfcHk0lgpGIojo912sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntlOqsrs%253D&md5=73f6d61463bc6db72178d70771544787</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.cd-15-1335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.cd-15-1335%26sid%3Dliteratum%253Aachs%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DGomez-Roca%26aufirst%3DC.%26aulast%3DDelord%26aufirst%3DJ.-P.%26aulast%3DRouge%26aufirst%3DT.%2Bd.%2BL.%2BM.%26aulast%3DUro-Coste%26aufirst%3DE.%26aulast%3Dde%2BBraud%26aufirst%3DF.%26aulast%3DPelosi%26aufirst%3DG.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26atitle%3DClinical%2520response%2520of%2520carcinomas%2520harboring%2520the%2520BRD4-NUT%2520oncoprotein%2520to%2520the%2520targeted%2520bromodomain%2520inhibitor%2520OTX015%252FMK-8628%26jtitle%3DCancer%2520Discovery%26date%3D2016%26volume%3D6%26spage%3D492%26epage%3D500%26doi%3D10.1158%2F2159-8290.cd-15-1335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odore, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitor OTX015 (MK-8628) combined with targeted agents shows strong in vivo antitumor activity in lymphoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">58142</span>– <span class="NLM_lpage">58147</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.10983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.18632%2Foncotarget.10983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27494885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A280%3ADC%252BC2s3otVelsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=58142-58147&author=E.+Gaudioauthor=C.+Tarantelliauthor=M.+Ponzoniauthor=E.+Odoreauthor=K.+Rezaiauthor=E.+Bernasconiauthor=L.+Cascioneauthor=A.+Rinaldiauthor=A.+Stathisauthor=E.+Riveiroauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=Bromodomain+inhibitor+OTX015+%28MK-8628%29+combined+with+targeted+agents+shows+strong+in+vivo+antitumor+activity+in+lymphoma&doi=10.18632%2Foncotarget.10983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibitor OTX015 (MK-8628) combined with targeted agents shows strong in vivo antitumor activity in lymphoma</span></div><div class="casAuthors">Gaudio Eugenio; Tarantelli Chiara; Bernasconi Elena; Cascione Luciano; Rinaldi Andrea; Bertoni Francesco; Ponzoni Maurilio; Odore Elodie; Rezai Keyvan; Cascione Luciano; Stathis Anastasios; Zucca Emanuele; Bertoni Francesco; Riveiro Eugenia; Cvitkovic Esteban</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">58142-58147</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The bromodomain inhibitor OTX015 (MK-8628) has shown anti-lymphoma activity as a single agent in both the preclinical and clinical settings, as well as in vitro synergism with several anticancer agents.  Here, we report in vivo data for OTX015 in combination with the histone deacetylase inhibitor vorinostat, the Bruton's tyrosine kinase inhibitor ibrutinib, the anti-CD20 monoclonal antibody rituximab, and the mTOR inhibitor everolimus in a diffuse large B cell lymphoma model.  The antitumor effect of OTX015-containing combinations in SU-DHL-2 xenografts in mice was much stronger than the activity of the corresponding single agents with almost complete tumor eradication for all four combinations.  Pharmacokinetic analyses showed similar OTX015 levels in plasma and tumor samples of approximately 1.5 μM, which is equivalent to the concentration showing strong in vitro activity.  For all four combinations, mean terminal levels of the bromodomain inhibitor differed from those in mice exposed to single agent OTX015, indicating a need for thorough pharmacokinetic investigations in phase I combination studies.  In conclusion, our results provide a strong rationale to explore OTX015-containing combinations in the clinical lymphoma setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRjLbpvJwtLh1UIs6wFK5xJfW6udTcc2eZxh_KrlXkK5bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3otVelsA%253D%253D&md5=2180a408b42b0d96851a1ff52f0754ef</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.10983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.10983%26sid%3Dliteratum%253Aachs%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DOdore%26aufirst%3DE.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DRiveiro%26aufirst%3DE.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DBromodomain%2520inhibitor%2520OTX015%2520%2528MK-8628%2529%2520combined%2520with%2520targeted%2520agents%2520shows%2520strong%2520in%2520vivo%2520antitumor%2520activity%2520in%2520lymphoma%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D58142%26epage%3D58147%26doi%3D10.18632%2Foncotarget.10983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Astorgues-Xerri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odore, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erba, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltrame, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frapolli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Incalci, M.</span></span> <span> </span><span class="NLM_article-title">The bromodomain inhibitor OTX015 (MK-8628) exerts anti-tumor activity in triple-negative breast cancer models as single agent and in combination with everolimus</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">7598</span>– <span class="NLM_lpage">7613</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.18632%2Foncotarget.13814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27935867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252Fis1Cntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=7598-7613&author=R.+V%C3%A1zquezauthor=M.+E.+Riveiroauthor=L.+Astorgues-Xerriauthor=E.+Odoreauthor=K.+Rezaiauthor=E.+Erbaauthor=N.+Paniniauthor=A.+Rinaldiauthor=I.+Kweeauthor=L.+Beltrameauthor=M.+Bekraddaauthor=E.+Cvitkovicauthor=F.+Bertoniauthor=R.+Frapolliauthor=M.+D%E2%80%99Incalci&title=The+bromodomain+inhibitor+OTX015+%28MK-8628%29+exerts+anti-tumor+activity+in+triple-negative+breast+cancer+models+as+single+agent+and+in+combination+with+everolimus&doi=10.18632%2Foncotarget.13814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain inhibitor OTX015 (MK-8628) exerts anti-tumor activity in triple-negative breast cancer models as single agent and in combination with everolimus</span></div><div class="casAuthors">Vazquez Ramiro; Erba Eugenio; Panini Nicolo; Beltrame Luca; Frapolli Roberta; D'Incalci Maurizio; Riveiro Maria E; Astorgues-Xerri Lucile; Odore Elodie; Bekradda Mohamed; Cvitkovic Esteban; Odore Elodie; Rezai Keyvan; Rinaldi Andrea; Kwee Ivo; Bertoni Francesco; Kwee Ivo; Kwee Ivo; Cvitkovic Esteban; Bertoni Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">7598-7613</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Triple-negative breast cancer (TNBC) is an aggressive and heterogeneous subgroup of breast tumors clinically defined by the lack of estrogen, progesterone and HER2 receptors, limiting the use of the targeted therapies employed in other breast malignancies.  Recent evidence indicates that c-MYC is a key driver of TNBC.  The BET-bromodomain inhibitor OTX015 (MK-8628) has potent antiproliferative activity accompanied by c-MYC down-regulation in several tumor types, and has demonstrated synergism with the mTOR inhibitor everolimus in different models.  The aim of this study was to evaluate the anti-tumor activity of OTX015 as single agent and in combination with everolimus in TNBC models.  OTX015 was assayed in three human TNBC-derived cell lines, HCC1937, MDA-MB-231 and MDA-MB-468, all showing antiproliferative activity after 72 h (GI50 = 75-650 nM).  This was accompanied by cell cycle arrest and decreased expression of cancer stem cells markers.  However, c-MYC protein and mRNA levels were only down-regulated in MDA-MB-468 cells.  Gene set enrichment analysis showed up-regulation of genes involved in epigenetic control of transcription, chromatin and the cell cycle, and down-regulation of stemness-related genes.  In vitro, combination with everolimus was additive in HCC1937 and MDA-MB-231 cells, but antagonistic in MDA-MB-468 cells.  In MDA-MB-231 murine xenografts, tumor mass was significantly (p < 0.05) reduced by OTX015 with respect to vehicle-treated animals (best T/C = 40.7%).  Although everolimus alone was not active, the combination was more effective than OTX015 alone (best T/C = 20.7%).  This work supports current clinical trials with OTX015 in TNBC (NCT02259114).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpC5NvWs-BQRJ5E29YNGvQfW6udTcc2eZxh_KrlXkK5bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252Fis1Cntw%253D%253D&md5=ac5f6998cb398ab9dbb9c9493880a5c7</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13814%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25A1zquez%26aufirst%3DR.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DAstorgues-Xerri%26aufirst%3DL.%26aulast%3DOdore%26aufirst%3DE.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DErba%26aufirst%3DE.%26aulast%3DPanini%26aufirst%3DN.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBeltrame%26aufirst%3DL.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26aulast%3DFrapolli%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Incalci%26aufirst%3DM.%26atitle%3DThe%2520bromodomain%2520inhibitor%2520OTX015%2520%2528MK-8628%2529%2520exerts%2520anti-tumor%2520activity%2520in%2520triple-negative%2520breast%2520cancer%2520models%2520as%2520single%2520agent%2520and%2520in%2520combination%2520with%2520everolimus%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D7598%26epage%3D7613%26doi%3D10.18632%2Foncotarget.13814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of <i>N</i>-(4-(2,4-difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a potent and orally available bromodomain and extraterminal domain (BET) family bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>– <span class="NLM_lpage">8384</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-difluorophenoxy%29-3-%286-methyl-7-oxo-6%2C7-dihydro-1H-pyrrolo%5B2%2C3-c%5Dpyridin-4-yl%29phenyl%29ethanesulfonamide+%28ABBV-075%2FMivebresib%29%2C+a+potent+and+orally+available+bromodomain+and+extraterminal+domain+%28BET%29+family+bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0lg_zMVWuynekg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-difluorophenoxy%2529-3-%25286-methyl-7-oxo-6%252C7-dihydro-1H-pyrrolo%255B2%252C3-c%255Dpyridin-4-yl%2529phenyl%2529ethanesulfonamide%2520%2528ABBV-075%252FMivebresib%2529%252C%2520a%2520potent%2520and%2520orally%2520available%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span> </span><span class="NLM_article-title">Dose escalation
study of GSK2820151 in subjects with advanced or
recurrent solid tumors</span>; <span class="NLM_publisher-name">Clinical trials</span> <a href="https://clinicaltrials.gov/ct2/show/NCT02630251" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02630251</a> (accessed Nov 30, 2015).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Dose+escalation%0Astudy+of+GSK2820151+in+subjects+with+advanced+or%0Arecurrent+solid+tumors%3B+Clinical+trials+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02630251+%28accessed+Nov+30%2C+2015%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DDose%2520escalation%250Astudy%2520of%2520GSK2820151%2520in%2520subjects%2520with%2520advanced%2520or%250Arecurrent%2520solid%2520tumors%26pub%3DClinical%2520trials" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+physical+in+drug+discovery+II%3A+the+impact+of+chromatographic+hydrophobicity+measurements+and+aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0lg_zMVWuynekg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%2520II%253A%2520the%2520impact%2520of%2520chromatographic%2520hydrophobicity%2520measurements%2520and%2520aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palovich, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. J.</span></span> <span> </span><span class="NLM_article-title">Quality guidelines for oral drug candidates: dose, solubility and lipophilicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1719</span>– <span class="NLM_lpage">1727</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.drudis.2016.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27423371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1aisb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1719-1727&author=M.+K.+Baylissauthor=J.+Butlerauthor=P.+L.+Feldmanauthor=D.+V.+S.+Greenauthor=P.+D.+Leesonauthor=M.+R.+Palovichauthor=A.+J.+Taylor&title=Quality+guidelines+for+oral+drug+candidates%3A+dose%2C+solubility+and+lipophilicity&doi=10.1016%2Fj.drudis.2016.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Quality guidelines for oral drug candidates: dose, solubility and lipophilicity</span></div><div class="casAuthors">Bayliss, Martin K.; Butler, James; Feldman, Paul L.; Green, Darren V. S.; Leeson, Paul D.; Palovich, Michael R.; Taylor, Anthony J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1719-1727</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In an attempt to seek increased understanding of compd. attributes that influence successful drug pipeline progression, GlaxoSmithKline's portfolio of oral candidates was compared with ref. sets of marketed oral drugs.  The approach differs from other attrition studies by explicitly focusing on choosing 'the right compd.' by applying relevant, exptl. derived properties.  The anal. led to four proposed compd. quality categories, created by combining specific criteria for three measures: dose, soly. and the property forecast index, a composite measure of lipophilicity using chromatog. detd. Log D and aromaticity.  The 'three properties' provide benchmarked guidelines for project teams to use when seeking and selecting clin. candidates, because they reflect the property distribution of marketed oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRMpjV3EZ_4rVg90H21EOLACvtfcHk0lg_zMVWuynekg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1aisb%252FK&md5=d9d145b1ee399b6ad54cdc55fff43c48</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DBayliss%26aufirst%3DM.%2BK.%26aulast%3DButler%26aufirst%3DJ.%26aulast%3DFeldman%26aufirst%3DP.%2BL.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DPalovich%26aufirst%3DM.%2BR.%26aulast%3DTaylor%26aufirst%3DA.%2BJ.%26atitle%3DQuality%2520guidelines%2520for%2520oral%2520drug%2520candidates%253A%2520dose%252C%2520solubility%2520and%2520lipophilicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1719%26epage%3D1727%26doi%3D10.1016%2Fj.drudis.2016.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franzini, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded chemical libraries: advancing beyond conventional small-molecule libraries</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1255</span>, <span class="refDoi"> DOI: 10.1021/ar400284t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar400284t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltF2ls7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2014&pages=1247-1255&author=R.+M.+Franziniauthor=D.+Neriauthor=J.+Scheuermann&title=DNA-encoded+chemical+libraries%3A+advancing+beyond+conventional+small-molecule+libraries&doi=10.1021%2Far400284t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Chemical Libraries: Advancing beyond Conventional Small-Molecule Libraries</span></div><div class="casAuthors">Franzini, Raphael M.; Neri, Dario; Scheuermann, Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1255</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA-encoded chem. libraries (DECLs) represent a promising tool in drug discovery.  DECL technol. allows the synthesis and screening of chem. libraries of unprecedented size at moderate costs.  In analogy to phage-display technol., where large antibody libraries are displayed on the surface of filamentous phage and are genetically encoded in the phage genome, DECLs feature the display of individual small org. chem. moieties on DNA fragments serving as amplifiable identification barcodes.  The DNA-tag facilitates the synthesis and allows the simultaneous screening of very large sets of compds. (up to billions of mols.), because the hit compds. can easily be identified and quantified by PCR-amplification of the DNA-barcode followed by high-throughput DNA sequencing.  Several approaches have been used to generate DECLs, differing both in the methods used for library encoding and for the combinatorial assembly of chem. moieties.  For example, DECLs can be used for fragment-based drug discovery, displaying a single mol. on DNA or two chem. moieties at the extremities of complementary DNA strands.  DECLs can vary substantially in the chem. structures and the library size.  While ultralarge libraries contg. billions of compds. have been reported contg. four or more sets of building blocks, also smaller libraries have been shown to be efficient for ligand discovery.  In general, it has been found that the overall library size is a poor predictor for library performance and that the no. and diversity of the building blocks are rather important indicators.  Smaller libraries consisting of two to three sets of building blocks better fulfill the criteria of drug-likeness and often have higher quality.  In this Account, we present advances in the DECL field from proof-of-principle studies to practical applications for drug discovery, both in industry and in academia.  DECL technol. can yield specific binders to a variety of target proteins and is likely to become a std. tool for pharmaceutical hit discovery, lead expansion, and Chem. Biol. research.  The introduction of new methodologies for library encoding and for compd. synthesis in the presence of DNA is an exciting research field and will crucially contribute to the performance and the propagation of the technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcaCmssNgPLLVg90H21EOLACvtfcHk0lgcqJnnjWoq3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltF2ls7g%253D&md5=c37bd1625b668e869fcc26d5ef0a017a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Far400284t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far400284t%26sid%3Dliteratum%253Aachs%26aulast%3DFranzini%26aufirst%3DR.%2BM.%26aulast%3DNeri%26aufirst%3DD.%26aulast%3DScheuermann%26aufirst%3DJ.%26atitle%3DDNA-encoded%2520chemical%2520libraries%253A%2520advancing%2520beyond%2520conventional%2520small-molecule%2520libraries%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2014%26volume%3D47%26spage%3D1247%26epage%3D1255%26doi%3D10.1021%2Far400284t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleiner, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. R.</span></span> <span> </span><span class="NLM_article-title">Small-molecule discovery from DNA-encoded chemical libraries</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">5707</span>– <span class="NLM_lpage">5717</span>, <span class="refDoi"> DOI: 10.1039/c1cs15076f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1039%2Fc1cs15076f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=21674077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGrsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=5707-5717&author=R.+E.+Kleinerauthor=C.+E.+Dumelinauthor=D.+R.+Liu&title=Small-molecule+discovery+from+DNA-encoded+chemical+libraries&doi=10.1039%2Fc1cs15076f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule discovery from DNA-encoded chemical libraries</span></div><div class="casAuthors">Kleiner, Ralph E.; Dumelin, Christoph E.; Liu, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5707-5717</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Researchers seeking to improve the efficiency and cost effectiveness of the bioactive small-mol. discovery process have recently embraced selection-based approaches, which in principle offer much higher throughput and simpler infrastructure requirements compared with traditional small-mol. screening methods.  Since selection methods benefit greatly from an information-encoding mol. that can be readily amplified and decoded, several academic and industrial groups have turned to DNA as the basis for library encoding and, in some cases, library synthesis.  The resulting DNA-encoded synthetic small-mol. libraries, integrated with the high sensitivity of PCR and the recent development of ultra high-throughput DNA sequencing technol., can be evaluated very rapidly for binding or bond formation with a target of interest while consuming minimal quantities of material and requiring only modest investments of time and equipment.  In this tutorial review we describe the development of two classes of approaches for encoding chem. structures and reactivity with DNA: DNA-recorded library synthesis, in which encoding and library synthesis take place sep., and DNA-directed library synthesis, in which DNA both encodes and templates library synthesis.  We also describe in vitro selection methods used to evaluate DNA-encoded libraries and summarize successful applications of these approaches to the discovery of bioactive small mols. and novel chem. reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRlmzulmrgLLVg90H21EOLACvtfcHk0lgcqJnnjWoq3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGrsb3F&md5=b39426ff390f49581a22cb82215fc242</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1039%2Fc1cs15076f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cs15076f%26sid%3Dliteratum%253Aachs%26aulast%3DKleiner%26aufirst%3DR.%2BE.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DLiu%26aufirst%3DD.%2BR.%26atitle%3DSmall-molecule%2520discovery%2520from%2520DNA-encoded%2520chemical%2520libraries%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D5707%26epage%3D5717%26doi%3D10.1039%2Fc1cs15076f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjamin, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creaser, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzen, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gefter, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavarana, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vloten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1038/nchembio.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+V.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kelleyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries&doi=10.1038%2Fnchembio.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lhZ_FyjD0Anbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelley%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3Dvan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654%26doi%3D10.1038%2Fnchembio.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundersingh, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ancellin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedard, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beljean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delorey, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grady, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herry, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushansky, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">Discovery, SAR, and x-ray binding mode study of BCATm inhibitors from a novel DNA-encoded library</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">924</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00179</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00179" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lt77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=919-924&author=H.+Dengauthor=J.+Zhouauthor=F.+S.+Sundersinghauthor=J.+Summerfieldauthor=D.+Somersauthor=J.+A.+Messerauthor=A.+L.+Satzauthor=N.+Ancellinauthor=C.+C.+Arico-Muendelauthor=K.+L.+Bedardauthor=A.+Beljeanauthor=S.+L.+Belyanskayaauthor=R.+Binghamauthor=S.+E.+Smithauthor=E.+Boursierauthor=P.+Carterauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.-w.+Chungauthor=C.+P.+Davieauthor=J.+L.+Deloreyauthor=Y.+Dingauthor=G.+J.+Franklinauthor=L.+C.+Gradyauthor=K.+Herryauthor=C.+Hobbsauthor=C.+S.+Kollmannauthor=B.+A.+Morganauthor=L.+J.+Kaushanskyauthor=Q.+Zhou&title=Discovery%2C+SAR%2C+and+x-ray+binding+mode+study+of+BCATm+inhibitors+from+a+novel+DNA-encoded+library&doi=10.1021%2Facsmedchemlett.5b00179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, SAR, and X-ray Binding Mode Study of BCATm Inhibitors from a Novel DNA-Encoded Library</span></div><div class="casAuthors">Deng, Hongfeng; Zhou, Jingye; Sundersingh, Flora S.; Summerfield, Jennifer; Somers, Don; Messer, Jeffrey A.; Satz, Alexander L.; Ancellin, Nicolas; Arico-Muendel, Christopher C.; Bedard, Katie L.; Beljean, Arthur; Belyanskaya, Svetlana L.; Bingham, Ryan; Smith, Sarah E.; Boursier, Eric; Carter, Paul; Centrella, Paolo A.; Clark, Matthew A.; Chung, Chun-wa; Davie, Christopher P.; Delorey, Jennifer L.; Ding, Yun; Franklin, G. Joseph; Grady, LaShadric C.; Herry, Kenny; Hobbs, Clare; Kollmann, Christopher S.; Morgan, Barry A.; Kaushansky, Laura J.; Zhou, Quan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">919-924</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As a potential target for obesity, human BCATm was screened against more than 14 billion DNA encoded compds. of distinct scaffolds followed by off-DNA synthesis and activity confirmation.  As a consequence, several series of BCATm inhibitors were discovered.  One representative compd. I from a novel compd. library synthesized via on-DNA Suzuki-Miyaura cross-coupling showed BCATm inhibitory activity with IC50 = 2.0 μM.  A protein crystal structure of I revealed that it binds to BCATm within the catalytic site adjacent to the PLP cofactor.  The identification of this novel inhibitor series plus the establishment of a BCATm protein structure provided a good starting point for future structure-based discovery of BCATm inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxzUuENOvoCrVg90H21EOLACvtfcHk0lhZ_FyjD0Anbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lt77L&md5=85fb50b6ee9c5b4ed3d1afd0bc9face6</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00179%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DSundersingh%26aufirst%3DF.%2BS.%26aulast%3DSummerfield%26aufirst%3DJ.%26aulast%3DSomers%26aufirst%3DD.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DAncellin%26aufirst%3DN.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBedard%26aufirst%3DK.%2BL.%26aulast%3DBeljean%26aufirst%3DA.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBingham%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DS.%2BE.%26aulast%3DBoursier%26aufirst%3DE.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDelorey%26aufirst%3DJ.%2BL.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DGrady%26aufirst%3DL.%2BC.%26aulast%3DHerry%26aufirst%3DK.%26aulast%3DHobbs%26aufirst%3DC.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26aulast%3DKaushansky%26aufirst%3DL.%2BJ.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DDiscovery%252C%2520SAR%252C%2520and%2520x-ray%2520binding%2520mode%2520study%2520of%2520BCATm%2520inhibitors%2520from%2520a%2520novel%2520DNA-encoded%2520library%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D919%26epage%3D924%26doi%3D10.1021%2Facsmedchemlett.5b00179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundersingh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beljean, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blazensky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herry, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphries, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ancellin, N.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of potent, selective, and in vivo efficacious 2-aryl benzimidazole BCATm inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00389</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Kru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=379-384&author=H.+Dengauthor=J.+Zhouauthor=F.+Sundersinghauthor=J.+A.+Messerauthor=D.+O.+Somersauthor=M.+Ajakaneauthor=C.+C.+Arico-Muendelauthor=A.+Beljeanauthor=S.+L.+Belyanskayaauthor=R.+Binghamauthor=E.+Blazenskyauthor=A.-B.+Boullayauthor=E.+Boursierauthor=J.+Chaiauthor=P.+Carterauthor=C.-W.+Chungauthor=A.+Dauganauthor=Y.+Dingauthor=K.+Herryauthor=C.+Hobbsauthor=E.+Humphriesauthor=C.+Kollmannauthor=V.+L.+Nguyenauthor=E.+Nicodemeauthor=S.+E.+Smithauthor=N.+Dodicauthor=N.+Ancellin&title=Discovery+and+optimization+of+potent%2C+selective%2C+and+in+vivo+efficacious+2-aryl+benzimidazole+BCATm+inhibitors&doi=10.1021%2Facsmedchemlett.5b00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Potent, Selective, and in Vivo Efficacious 2-Aryl Benzimidazole BCATm Inhibitors</span></div><div class="casAuthors">Deng, Hongfeng; Zhou, Jingye; Sundersingh, Flora; Messer, Jeffrey A.; Somers, Donald O.; Ajakane, Myriam; Arico-Muendel, Christopher C.; Beljean, Arthur; Belyanskaya, Svetlana L.; Bingham, Ryan; Blazensky, Emily; Boullay, Anne-Benedicte; Boursier, Eric; Chai, Jing; Carter, Paul; Chung, Chun-Wa; Daugan, Alain; Ding, Yun; Herry, Kenny; Hobbs, Clare; Humphries, Eric; Kollmann, Christopher; Nguyen, Van Loc; Nicodeme, Edwige; Smith, Sarah E.; Dodic, Nerina; Ancellin, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-384</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To identify BCATm inhibitors suitable for in vivo study, Encoded Library Technol. (ELT) was used to affinity screen a 117 million member benzimidazole based DNA encoded library, which identified an inhibitor series with both biochem. and cellular activities.  Subsequent SAR studies led to the discovery of a highly potent and selective compd. I with much improved PK properties.  X-ray structure revealed that I binds to the active site of BACTm in a unique mode via multiple H-bond and van der Waals interactions.  After oral administration, I raised mouse blood levels of all three branched chain amino acids as a consequence of BCATm inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXwx-YRCyoe7Vg90H21EOLACvtfcHk0ljyH9NJ7scOuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Kru7g%253D&md5=3d6116d0a992d0b3d051098b40eb179e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00389%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DSundersingh%26aufirst%3DF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DSomers%26aufirst%3DD.%2BO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBeljean%26aufirst%3DA.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBingham%26aufirst%3DR.%26aulast%3DBlazensky%26aufirst%3DE.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DBoursier%26aufirst%3DE.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DHerry%26aufirst%3DK.%26aulast%3DHobbs%26aufirst%3DC.%26aulast%3DHumphries%26aufirst%3DE.%26aulast%3DKollmann%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DV.%2BL.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DSmith%26aufirst%3DS.%2BE.%26aulast%3DDodic%26aufirst%3DN.%26aulast%3DAncellin%26aufirst%3DN.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520potent%252C%2520selective%252C%2520and%2520in%2520vivo%2520efficacious%25202-aryl%2520benzimidazole%2520BCATm%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D379%26epage%3D384%26doi%3D10.1021%2Facsmedchemlett.5b00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gondo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLorey, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Development and synthesis of DNA-encoded benzimidazole library</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.8b00009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.8b00009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFSgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=251-255&author=Y.+Dingauthor=J.+Chaiauthor=P.+A.+Centrellaauthor=C.+Gondoauthor=J.+L.+DeLoreyauthor=M.+A.+Clark&title=Development+and+synthesis+of+DNA-encoded+benzimidazole+library&doi=10.1021%2Facscombsci.8b00009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Synthesis of DNA-Encoded Benzimidazole Library</span></div><div class="casAuthors">Ding, Yun; Chai, Jing; Centrella, Paolo A.; Gondo, Chenaimwoyo; De Lorey, Jennifer L.; Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">251-255</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Encoded library technol. (ELT) is an effective approach to the discovery of novel small-mol. ligands for biol. targets.  A key factor for the success of the technol. is the chem. diversity of the libraries.  Here we report the development of DNA-conjugated benzimidazoles.  Using 4-fluoro-3-nitrobenzoic acid as a key synthon, we synthesized a 320 million-member DNA-encoded benzimidazole library using Fmoc-protected amino acids, amines and aldehydes as diversity elements.  Affinity selection of the library led to the discovery of a novel, potent and specific antagonist of the NK3 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj58UnFYV0b7Vg90H21EOLACvtfcHk0ljyH9NJ7scOuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFSgt7Y%253D&md5=2a8db7621b99bf9390c446dc0fc56780</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.8b00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.8b00009%26sid%3Dliteratum%253Aachs%26aulast%3DDing%26aufirst%3DY.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DGondo%26aufirst%3DC.%26aulast%3DDeLorey%26aufirst%3DJ.%2BL.%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DDevelopment%2520and%2520synthesis%2520of%2520DNA-encoded%2520benzimidazole%2520library%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2018%26volume%3D20%26spage%3D251%26epage%3D255%26doi%3D10.1021%2Facscombsci.8b00009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldino, C.</span></span> <span> </span><span class="NLM_article-title">A versatile method for the synthesis of benzimidazoles from o-nitroanilines and aldehydes in one step via a reductive cyclization</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1055/s-2004-834926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1055%2Fs-2004-834926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvFSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=47-56&author=D.+Yangauthor=D.+Fokasauthor=J.+Liauthor=L.+Yuauthor=C.+Baldino&title=A+versatile+method+for+the+synthesis+of+benzimidazoles+from+o-nitroanilines+and+aldehydes+in+one+step+via+a+reductive+cyclization&doi=10.1055%2Fs-2004-834926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A versatile method for the synthesis of benzimidazoles from o-nitroanilines and aldehydes in one step via a reductive cyclization</span></div><div class="casAuthors">Yang, Donglai; Fokas, Demosthenes; Li, Jingzhou; Yu, Libing; Baldino, Carmen M.</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-56</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">0039-7881</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A highly efficient and versatile method for the synthesis of benzimidazoles, e.g., I, was achieved in one step by the Na2S2O4 redn. of o-nitroanilines in the presence of aldehydes.  Heating a soln. of o-nitroaniline and an aldehyde, in the presence of aq. or solid Na2S2O4, provided facile access to a series of 2-substituted N-H benzimidazoles contg. a wide range of functional groups not always compatible with the existing synthetic methods.  This methodol. has also been applied to the regioselective synthesis of N-alkyl and N-aryl benzimidazoles by the cyclization of the corresponding N-substituted nitroanilines, resp.  In addn., the method was applied successfully to the synthesis of other imidazole contg. heterocyclic ring systems such as 1H-imidazo[4,5-b]pyridines and 1H-imidazo[4,5-f]quinoline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo731FJUu4mCLVg90H21EOLACvtfcHk0ljyH9NJ7scOuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvFSquw%253D%253D&md5=9459f8a5a5235627a318997d8af5b3ec</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1055%2Fs-2004-834926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-2004-834926%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DD.%26aulast%3DFokas%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DBaldino%26aufirst%3DC.%26atitle%3DA%2520versatile%2520method%2520for%2520the%2520synthesis%2520of%2520benzimidazoles%2520from%2520o-nitroanilines%2520and%2520aldehydes%2520in%2520one%2520step%2520via%2520a%2520reductive%2520cyclization%26jtitle%3DSynthesis%26date%3D2005%26spage%3D47%26epage%3D56%26doi%3D10.1055%2Fs-2004-834926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshua, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of potent diazobenzene inhibitors for the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9251</span>– <span class="NLM_lpage">9264</span>, <span class="refDoi"> DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&author=G.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.-M.+Zhou&title=Structure-guided+design+of+potent+diazobenzene+inhibitors+for+the+BET+bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0lhMrbvb_lAP0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure-guided%2520design%2520of%2520potent%2520diazobenzene%2520inhibitors%2520for%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3838</span>, <span class="refDoi"> DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.-w.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.-M.+G.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=A.-B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+characterization+of+small+molecule+inhibitors+of+the+BET+family+bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lhMrbvb_lAP0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.-M.%2BG.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.-B.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520BET%2520family%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holenz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoy, P.</span></span> <span> </span><span class="NLM_article-title">Advances in lead generation</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2018.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=30606700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlsFOr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=517-524&author=J.+Holenzauthor=P.+Stoy&title=Advances+in+lead+generation&doi=10.1016%2Fj.bmcl.2018.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in Lead Generation</span></div><div class="casAuthors">Holenz, Joerg; Stoy, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">517-524</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lead Generation represents a crit. drug discovery phase where chem. starting points and their resp. mechanism of action, quality, and potential liabilities are largely predefined.  Recent advances such as DNA-encoded libraries or fragment-, chem. biol.-, and virtual screening-based approaches are today as common as traditional High Throughput Screening.  Innovations in characterizing lead quality have allowed more informed decision-making by discovery teams.  The key challenge today is to individually tailor the right mix of methods for each project to facilitate data integration with the purpose of creating multiple high-quality lead series, ultimately translating to reduced chem.-related pipeline attrition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa_L2ocMTSq7Vg90H21EOLACvtfcHk0lhMrbvb_lAP0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlsFOr&md5=25444e43f6918fcc346e7d5fcb75c7d3</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DHolenz%26aufirst%3DJ.%26aulast%3DStoy%26aufirst%3DP.%26atitle%3DAdvances%2520in%2520lead%2520generation%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D517%26epage%3D524%26doi%3D10.1016%2Fj.bmcl.2018.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 1: inhibitor binding modes and implications for lead discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">576</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1021/jm201320w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201320w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=576-586&author=C.-w.+Chungauthor=A.+W.+Deanauthor=J.+M.+Woolvenauthor=P.+Bamborough&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+1%3A+inhibitor+binding+modes+and+implications+for+lead+discovery&doi=10.1021%2Fjm201320w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span></div><div class="casAuthors">Chung, Chun-wa; Dean, Anthony W.; Woolven, James M.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are key epigenetic regulators of gene transcription and readers of the histone code.  However, the therapeutic benefits of modulating this target class are largely unexplored due to the lack of suitable chem. probes.  This article describes the generation of lead mols. for the BET bromodomains through screening a fragment set chosen using structural insights and computational approaches.  Anal. of 40 BRD2/fragment x-ray complexes highlights both shared and disparate interaction features that may be exploited for affinity and selectivity.  Six representative crystal structures are then exemplified in detail.  Two of the fragments are completely new bromodomain chemotypes, and three have never before been crystd. in a bromodomain, so our results significantly extend the limited public knowledge-base of crystallog. small mol./bromodomain interactions.  Certain fragments (including paracetamol) bind in a consistent mode to different bromodomains such as CREBBP, suggesting their potential to act as generic bromodomain templates.  An important implication is that the bromodomains are not only a phylogenetic family but also a system in which chem. and structural knowledge of one bromodomain gives insights transferable to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonNb7GnZnS_rVg90H21EOLACvtfcHk0ljoLr9Jb2lEiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN&md5=5b7d7239d0de57266d94beef1ea080ae</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm201320w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201320w%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDean%26aufirst%3DA.%2BW.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25201%253A%2520inhibitor%2520binding%2520modes%2520and%2520implications%2520for%2520lead%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D576%26epage%3D586%26doi%3D10.1021%2Fjm201320w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodacre, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodrow, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of bromodomain inhibitors part 2: optimization of phenylisoxazole sulfonamides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">587</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1021/jm201283q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201283q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=587-596&author=P.+Bamboroughauthor=H.+Dialloauthor=J.+D.+Goodacreauthor=L.+Gordonauthor=A.+Lewisauthor=J.+T.+Sealauthor=D.+M.+Wilsonauthor=M.+D.+Woodrowauthor=C.-w.+Chung&title=Fragment-based+discovery+of+bromodomain+inhibitors+part+2%3A+optimization+of+phenylisoxazole+sulfonamides&doi=10.1021%2Fjm201283q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 2: Optimization of Phenylisoxazole Sulfonamides</span></div><div class="casAuthors">Bamborough, Paul; Diallo, Hawa; Goodacre, Jonathan D.; Gordon, Laurie; Lewis, Antonia; Seal, Jonathan T.; Wilson, David M.; Woodrow, Michael D.; Chung, Chun-wa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">587-596</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are epigenetic reader modules that regulate gene transcription through their recognition of acetyl-lysine modified histone tails.  Inhibitors of this protein-protein interaction have the potential to modulate multiple diseases as demonstrated by the profound anti-inflammatory and antiproliferative effects of a recently disclosed class of BET compds.  While these compds. were discovered using phenotypic assays, here we present a highly efficient alternative approach to find new chem. templates, exploiting the abundant structural knowledge that exists for this target class.  A Ph di-Me isoxazole chemotype resulting from a focused fragment screen has been rapidly optimized through structure-based design, leading to a sulfonamide series showing anti-inflammatory activity in cellular assays.  This proof-of-principle expt. demonstrates the tractability of the BET family and bromodomain target class to fragment-based hit discovery and structure-based lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr6lsic9qjJrVg90H21EOLACvtfcHk0ljoLr9Jb2lEiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrfF&md5=e73b16b3fddf88b43b037264b000b276</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm201283q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201283q%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DGoodacre%26aufirst%3DJ.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWoodrow%26aufirst%3DM.%2BD.%26aulast%3DChung%26aufirst%3DC.-w.%26atitle%3DFragment-based%2520discovery%2520of%2520bromodomain%2520inhibitors%2520part%25202%253A%2520optimization%2520of%2520phenylisoxazole%2520sulfonamides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D587%26epage%3D596%26doi%3D10.1021%2Fjm201283q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bogdan, A.</span>; <span class="NLM_string-name">Kati, W. M.</span>; <span class="NLM_string-name">McDaniel, K. F.</span>; <span class="NLM_string-name">Park, C. H.</span>; <span class="NLM_string-name">Sheppard, G. S.</span>; <span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Preparation of Pyridinyl Containing Compounds as Bromodomain Inhibitors for Therapy</span>. U.S. Patent <span class="NLM_patent">20150158873A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=A.+Bogdan&author=W.+M.+Kati&author=K.+F.+McDaniel&author=C.+H.+Park&author=G.+S.+Sheppard&author=L.+Wang&title=Preparation+of+Pyridinyl+Containing+Compounds+as+Bromodomain+Inhibitors+for+Therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBogdan%26aufirst%3DA.%26atitle%3DPreparation%2520of%2520Pyridinyl%2520Containing%2520Compounds%2520as%2520Bromodomain%2520Inhibitors%2520for%2520Therapy%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, H.</span>; <span class="NLM_string-name">Martin, L.</span>; <span class="NLM_string-name">Smethurst, C.</span></span> <span> </span><span class="NLM_article-title">Preparation of Substituted Pyridinones as Anticancer Agents</span>. <span class="NLM_patent">WO2015022332A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=H.+Engelhardt&author=L.+Martin&author=C.+Smethurst&title=Preparation+of+Substituted+Pyridinones+as+Anticancer+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEngelhardt%26aufirst%3DH.%26atitle%3DPreparation%2520of%2520Substituted%2520Pyridinones%2520as%2520Anticancer%2520Agents%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Frey, R. R.</span>; <span class="NLM_string-name">Hansen, T. M.</span>; <span class="NLM_string-name">Liu, D.</span>; <span class="NLM_string-name">McClellan, W. J.</span>; <span class="NLM_string-name">McDaniel, K. F.</span>; <span class="NLM_string-name">Pratt, J. K.</span>; <span class="NLM_string-name">Wada, C. K.</span></span> <span> </span><span class="NLM_article-title">Preparation of Multicyclic Compounds as Bromodomain Inhibitors</span>. <span class="NLM_patent">WO2015085925A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=L.+Wang&author=R.+R.+Frey&author=T.+M.+Hansen&author=D.+Liu&author=W.+J.+McClellan&author=K.+F.+McDaniel&author=J.+K.+Pratt&author=C.+K.+Wada&title=Preparation+of+Multicyclic+Compounds+as+Bromodomain+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26atitle%3DPreparation%2520of%2520Multicyclic%2520Compounds%2520as%2520Bromodomain%2520Inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, H.</span></span> <span> </span><span class="NLM_article-title">Benzimidazole Derivatives as BRD4 Inhibitors and their Preparation and Use for the Treatment of Cancer</span>. <span class="NLM_patent">WO2015169962A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=H.+Engelhardt&title=Benzimidazole+Derivatives+as+BRD4+Inhibitors+and+their+Preparation+and+Use+for+the+Treatment+of+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEngelhardt%26aufirst%3DH.%26atitle%3DBenzimidazole%2520Derivatives%2520as%2520BRD4%2520Inhibitors%2520and%2520their%2520Preparation%2520and%2520Use%2520for%2520the%2520Treatment%2520of%2520Cancer%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holms, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClellan, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Fragment-based, structure-enabled discovery of novel pyridones and pyridone macrocycles as potent bromodomain and extra-terminal domain (BET) family bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3828</span>– <span class="NLM_lpage">3850</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00017</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00017" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsVygu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3828-3850&author=L.+Wangauthor=J.+K.+Prattauthor=T.+Soltwedelauthor=G.+S.+Sheppardauthor=S.+D.+Fidanzeauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=R.+A.+Manteiauthor=J.+H.+Holmsauthor=W.+J.+McClellanauthor=M.+D.+Wendtauthor=C.+Wadaauthor=R.+Freyauthor=T.+M.+Hansenauthor=R.+Hubbardauthor=C.+H.+Parkauthor=L.+Liauthor=T.+J.+Magocauthor=D.+H.+Albertauthor=X.+Linauthor=S.+E.+Warderauthor=P.+Kovarauthor=X.+Huangauthor=D.+Wilcoxauthor=R.+Wangauthor=G.+Rajaramanauthor=A.+M.+Petrosauthor=C.+W.+Hutchinsauthor=S.+C.+Panchalauthor=C.+Sunauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Fragment-based%2C+structure-enabled+discovery+of+novel+pyridones+and+pyridone+macrocycles+as+potent+bromodomain+and+extra-terminal+domain+%28BET%29+family+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.7b00017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based, Structure-Enabled Discovery of Novel Pyridones and Pyridone Macrocycles as Potent Bromodomain and Extra-Terminal Domain (BET) Family Bromodomain Inhibitors</span></div><div class="casAuthors">Wang, Le; Pratt, John K.; Soltwedel, Todd; Sheppard, George S.; Fidanze, Steven D.; Liu, Dachun; Hasvold, Lisa A.; Mantei, Robert A.; Holms, James H.; McClellan, William J.; Wendt, Michael D.; Wada, Carol; Frey, Robin; Hansen, T. Matthew; Hubbard, Robert; Park, Chang H.; Li, Leiming; Magoc, Terrance J.; Albert, Daniel H.; Lin, Xiaoyu; Warder, Scott E.; Kovar, Peter; Huang, Xiaoli; Wilcox, Denise; Wang, Rongqi; Rajaraman, Ganesh; Petros, Andrew M.; Hutchins, Charles W.; Panchal, Sanjay C.; Sun, Chaohong; Elmore, Steven W.; Shen, Yu; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3828-3850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the BET family of bromodomain contg. proteins have been identified as potential targets for blocking proliferation in a variety of cancer cell lines.  A 2-dimensional NMR fragment screen for binders to the bromodomains of BRD4 identified a Ph pyridazinone fragment with a weak binding affinity (I, Ki = 160 μM).  SAR investigation of fragment I, aided by X-ray structure-based design, enabled the synthesis of potent pyridone and macrocyclic pyridone inhibitors exhibiting single digit nanomolar potency in both biochem. and cell based assays, e.g. II.  Advanced analogs in these series exhibited high oral exposures in rodent PK studies and demonstrated significant tumor growth inhibition efficacy in mouse flank xenograft models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDaiYdyL4LtrVg90H21EOLACvtfcHk0lgZBcDU-G76Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsVygu7k%253D&md5=91750e660bafec08b8bcbe6bcd5e818f</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00017%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DHolms%26aufirst%3DJ.%2BH.%26aulast%3DMcClellan%26aufirst%3DW.%2BJ.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DWada%26aufirst%3DC.%26aulast%3DFrey%26aufirst%3DR.%26aulast%3DHansen%26aufirst%3DT.%2BM.%26aulast%3DHubbard%26aufirst%3DR.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DFragment-based%252C%2520structure-enabled%2520discovery%2520of%2520novel%2520pyridones%2520and%2520pyridone%2520macrocycles%2520as%2520potent%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520family%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3828%26epage%3D3850%26doi%3D10.1021%2Facs.jmedchem.7b00017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Postepska-Igielska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krunic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich-Bode, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukamp, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grummt, I.</span></span> <span> </span><span class="NLM_article-title">The chromatin remodelling complex NoRC safeguards genome stability by heterochromatin formation at telomeres and centromeres</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">704</span>– <span class="NLM_lpage">710</span>, <span class="refDoi"> DOI: 10.1038/embor.2013.87</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fembor.2013.87" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23797874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvFeqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=704-710&author=A.+Postepska-Igielskaauthor=D.+Krunicauthor=N.+Schmittauthor=K.+M.+Greulich-Bodeauthor=P.+Boukampauthor=I.+Grummt&title=The+chromatin+remodelling+complex+NoRC+safeguards+genome+stability+by+heterochromatin+formation+at+telomeres+and+centromeres&doi=10.1038%2Fembor.2013.87"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">The chromatin remodelling complex NoRC safeguards genome stability by heterochromatin formation at telomeres and centromeres</span></div><div class="casAuthors">Postepska-Igielska, Anna; Krunic, Damir; Schmitt, Nina; Greulich-Bode, Karin M.; Boukamp, Petra; Grummt, Ingrid</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">704-710</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Constitutive heterochromatin is crucial for the integrity of chromosomes and genomic stability.  Here, we show that the chromatin remodelling complex NoRC, known to silence a fraction of rRNA genes, also establishes a repressive heterochromatic structure at centromeres and telomeres, preserving the structural integrity of these repetitive loci.  Knockdown of NoRC leads to relaxation of centromeric and telomeric heterochromatin, abnormalities in mitotic spindle assembly, impaired chromosome segregation and enhanced chromosomal instability.  The results demonstrate that NoRC safeguards genomic stability by coordinating enzymic activities that establish features of repressive chromatin at centromeric and telomeric regions, and this heterochromatic structure is required for sustaining genomic integrity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAXnCQyfI2xLVg90H21EOLACvtfcHk0lgZBcDU-G76Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvFeqtbo%253D&md5=2dfa275fa4fc7c02eadfddc90982fedc</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fembor.2013.87&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2013.87%26sid%3Dliteratum%253Aachs%26aulast%3DPostepska-Igielska%26aufirst%3DA.%26aulast%3DKrunic%26aufirst%3DD.%26aulast%3DSchmitt%26aufirst%3DN.%26aulast%3DGreulich-Bode%26aufirst%3DK.%2BM.%26aulast%3DBoukamp%26aufirst%3DP.%26aulast%3DGrummt%26aufirst%3DI.%26atitle%3DThe%2520chromatin%2520remodelling%2520complex%2520NoRC%2520safeguards%2520genome%2520stability%2520by%2520heterochromatin%2520formation%2520at%2520telomeres%2520and%2520centromeres%26jtitle%3DEMBO%2520Rep.%26date%3D2013%26volume%3D14%26spage%3D704%26epage%3D710%26doi%3D10.1038%2Fembor.2013.87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosea, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, R. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakar, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, K.</span></span> <span> </span><span class="NLM_article-title">Prediction of human pharmacokinetics from preclinical information: comparative accuracy of quantitative prediction approaches</span>. <i>Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1177/0091270009333209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1177%2F0091270009333209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlsFWmu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=513-533&author=N.+A.+Hoseaauthor=W.+T.+Collardauthor=S.+Coleauthor=T.+S.+Maurerauthor=R.+X.+Fangauthor=H.+Jonesauthor=S.+M.+Kakarauthor=Y.+Nakaiauthor=B.+J.+Smithauthor=R.+Websterauthor=K.+Beaumont&title=Prediction+of+human+pharmacokinetics+from+preclinical+information%3A+comparative+accuracy+of+quantitative+prediction+approaches&doi=10.1177%2F0091270009333209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human pharmacokinetics from preclinical information: comparative accuracy of quantitative prediction approaches</span></div><div class="casAuthors">Hosea, Natilie A.; Collard, Wendy T.; Cole, Susan; Maurer, Tristan S.; Fang, Rick X.; Jones, Hannah; Kakar, Shefali M.; Nakai, Yasuhiro; Smith, Bill J.; Webster, Rob; Beaumont, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">513-533</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Quant. prediction of human pharmacokinetics is crit. in assessing the viability of drug candidates and in detg. first-in-human dosing.  Numerous prediction methodologies, incorporating both in vitro and preclin. in vivo data, have been developed in recent years, each with advantages and disadvantages.  However, the lack of a comprehensive data set, both preclin. and clin., has limited efforts to evaluate the optimal strategy (or strategies) that results in quant. predictions of human pharmacokinetics.  To address this issue, the authors conducted a retrospective anal. using 50 proprietary compds. for which in vitro, preclin. pharmacokinetic data and oral single-dose human pharmacokinetic data were available.  Five predictive strategies, involving either allometry or use of unbound intrinsic clearance from microsomes or hepatocytes, were then compared for their ability to predict human oral clearance, half-life through predictions of systemic clearance, vol. of distribution, and bioavailability.  Use of a single-species scaling approach with rat, dog, or monkey was as accurate as or more accurate than using multiple-species allometry.  For those compds. cleared almost exclusively by P 450-mediated pathways, scaling from human liver microsomes was as predictive as single-species scaling of clearance based on data from rat, dog, or monkey.  These data suggest that use of predictive methods involving either single-species in vivo data or in vitro human liver microsomes can quant. predict human in vivo pharmacokinetics and suggest the possibility of streamlining the predictive methodol. through use of a single species or use only of human in vitro microsomal prepns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg67HcwS4O4LVg90H21EOLACvtfcHk0lhmiqcHzSK-gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlsFWmu74%253D&md5=a0941d53fe43392f8fb1562ee546705e</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1177%2F0091270009333209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0091270009333209%26sid%3Dliteratum%253Aachs%26aulast%3DHosea%26aufirst%3DN.%2BA.%26aulast%3DCollard%26aufirst%3DW.%2BT.%26aulast%3DCole%26aufirst%3DS.%26aulast%3DMaurer%26aufirst%3DT.%2BS.%26aulast%3DFang%26aufirst%3DR.%2BX.%26aulast%3DJones%26aufirst%3DH.%26aulast%3DKakar%26aufirst%3DS.%2BM.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DB.%2BJ.%26aulast%3DWebster%26aufirst%3DR.%26aulast%3DBeaumont%26aufirst%3DK.%26atitle%3DPrediction%2520of%2520human%2520pharmacokinetics%2520from%2520preclinical%2520information%253A%2520comparative%2520accuracy%2520of%2520quantitative%2520prediction%2520approaches%26jtitle%3DClin.%2520Pharmacol.%26date%3D2009%26volume%3D49%26spage%3D513%26epage%3D533%26doi%3D10.1177%2F0091270009333209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahmood, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balian, J. D.</span></span> <span> </span><span class="NLM_article-title">Interspecies scaling: predicting clearance of drugs in humans. Three different approaches</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">887</span>– <span class="NLM_lpage">895</span>, <span class="refDoi"> DOI: 10.3109/00498259609052491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.3109%2F00498259609052491" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=8893036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaK28XmtVyntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1996&pages=887-895&author=I.+Mahmoodauthor=J.+D.+Balian&title=Interspecies+scaling%3A+predicting+clearance+of+drugs+in+humans.+Three+different+approaches&doi=10.3109%2F00498259609052491"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Interspecies scaling: predicting clearance of drugs in humans. Three different approaches</span></div><div class="casAuthors">Mahmood, I.; Balian, J. D.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">887-895</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis</span>)
        </div><div class="casAbstract">The interspecies scaling approach to predict clearance in humans from animal data was tested for a wide variety of drugs.  Three different methods were utilized to generate plots to scale-up the clearance values: (i) method I, clearance vs. body wt. (simple allometric equation); (ii) method II, product of clearance and max. life-span potential; (iii) method III, product of clearance and brain wt. vs. body wt.,.  The circumstances under which the three methods I predicts clearance reasonably well.  If the exponent lies between 0·55 to 0·77, then method I predicts clearance reasonably well.  If the exponent lies between 0·71 to 1·0, clearance can be predicted reasonably well by method II.  If the exponent is > 1·0, clearance can be predicted using method III.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps3KGdN89O0LVg90H21EOLACvtfcHk0lhmiqcHzSK-gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtVyntrs%253D&md5=c7ed947e99fefabf08d3eaca7d2a08c5</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.3109%2F00498259609052491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498259609052491%26sid%3Dliteratum%253Aachs%26aulast%3DMahmood%26aufirst%3DI.%26aulast%3DBalian%26aufirst%3DJ.%2BD.%26atitle%3DInterspecies%2520scaling%253A%2520predicting%2520clearance%2520of%2520drugs%2520in%2520humans.%2520Three%2520different%2520approaches%26jtitle%3DXenobiotica%26date%3D1996%26volume%3D26%26spage%3D887%26epage%3D895%26doi%3D10.3109%2F00498259609052491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overvoorde, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svergun, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of histone tail recognition by human TIP5 PHD finger and bromodomain of the chromatin remodeling complex NoRC</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.str.2014.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.str.2014.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=25533489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvFWrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=80-92&author=C.+Tallantauthor=E.+Valentiniauthor=O.+Fedorovauthor=L.+Overvoordeauthor=F.+M.+Fergusonauthor=P.+Filippakopoulosauthor=D.+I.+Svergunauthor=S.+Knappauthor=A.+Ciulli&title=Molecular+basis+of+histone+tail+recognition+by+human+TIP5+PHD+finger+and+bromodomain+of+the+chromatin+remodeling+complex+NoRC&doi=10.1016%2Fj.str.2014.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis of Histone Tail Recognition by Human TIP5 PHD Finger and Bromodomain of the Chromatin Remodeling Complex NoRC</span></div><div class="casAuthors">Tallant, Cynthia; Valentini, Erica; Fedorov, Oleg; Overvoorde, Lois; Ferguson, Fleur M.; Filippakopoulos, Panagis; Svergun, Dmitri I.; Knapp, Stefan; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-92</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Binding of the chromatin remodeling complex NoRC to RNA complementary to the rDNA promoter mediates transcriptional repression.  TIP5, the largest subunit of NoRC, is involved in recruitment to rDNA by interactions with promoter-bound TTF-I, pRNA, and acetylation of H4K16.  TIP5 domains that recognize posttranslational modifications on histones are essential for recruitment of NoRC to chromatin, but how these reader modules recognize site-specific histone tails has remained elusive.  Here, we report crystal structures of PHD zinc finger and bromodomains from human TIP5 and BAZ2B in free form and bound to H3 and/or H4 histones.  PHD finger functions as an independent structural module in recognizing unmodified H3 histone tails, and the bromodomain prefers H3 and H4 acetylation marks followed by a key basic residue, KacXXR.  Further low-resoln. analyses of PHD-bromodomain modules provide mol. insights into their trans histone tail recognition, required for nucleosome recruitment and transcriptional repression of the NoRC complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOrAPnFDwkzbVg90H21EOLACvtfcHk0lhmiqcHzSK-gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvFWrug%253D%253D&md5=2f891613f59dc2c584db33f475461e18</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DValentini%26aufirst%3DE.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DOvervoorde%26aufirst%3DL.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DSvergun%26aufirst%3DD.%2BI.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520basis%2520of%2520histone%2520tail%2520recognition%2520by%2520human%2520TIP5%2520PHD%2520finger%2520and%2520bromodomain%2520of%2520the%2520chromatin%2520remodeling%2520complex%2520NoRC%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D80%26epage%3D92%26doi%3D10.1016%2Fj.str.2014.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wuitschik, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers-Evans, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Märki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrilla, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carreira, E. M.</span></span> <span> </span><span class="NLM_article-title">Spirocyclic oxetanes: synthesis and properties</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">4512</span>– <span class="NLM_lpage">4515</span>, <span class="refDoi"> DOI: 10.1002/anie.200800450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1002%2Fanie.200800450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD1cXns1yrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=4512-4515&author=G.+Wuitschikauthor=M.+Rogers-Evansauthor=A.+Bucklauthor=M.+Bernasconiauthor=M.+M%C3%A4rkiauthor=T.+Godelauthor=H.+Fischerauthor=B.+Wagnerauthor=I.+Parrillaauthor=F.+Schulerauthor=J.+Schneiderauthor=A.+Alkerauthor=W.+B.+Schweizerauthor=K.+M%C3%BCllerauthor=E.+M.+Carreira&title=Spirocyclic+oxetanes%3A+synthesis+and+properties&doi=10.1002%2Fanie.200800450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Spirocyclic oxetanes: synthesis and properties</span></div><div class="casAuthors">Wuitschik, Georg; Rogers-Evans, Mark; Buckl, Andreas; Bernasconi, Maurizio; Marki, Moritz; Godel, Thierry; Fischer, Holger; Wagner, Bjorn; Parrilla, Isabelle; Schuler, Franz; Schneider, Josef; Alker, Andre; Schweizer, W. Bernd; Muller, Klaus; Carreira, Erick M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4512-4515</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Spirocyclic oxetanes are described as analogs of morpholine and also as topol. siblings of their carbonyl counterparts.  They are particularly promising in terms of both their physicochem. properties and the ease with which they can be grafted onto mol. structures.  The data collected highlight oxetanes as both the hydrophilic sister of a gem-di-Me unit and the carbonyl group's lipophilic brother.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpjICUKXgpt7Vg90H21EOLACvtfcHk0ljsYsaHaUf93A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXns1yrurs%253D&md5=9522eb5cfb870b872d8f9c1f06ae2946</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2Fanie.200800450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200800450%26sid%3Dliteratum%253Aachs%26aulast%3DWuitschik%26aufirst%3DG.%26aulast%3DRogers-Evans%26aufirst%3DM.%26aulast%3DBuckl%26aufirst%3DA.%26aulast%3DBernasconi%26aufirst%3DM.%26aulast%3DM%25C3%25A4rki%26aufirst%3DM.%26aulast%3DGodel%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DH.%26aulast%3DWagner%26aufirst%3DB.%26aulast%3DParrilla%26aufirst%3DI.%26aulast%3DSchuler%26aufirst%3DF.%26aulast%3DSchneider%26aufirst%3DJ.%26aulast%3DAlker%26aufirst%3DA.%26aulast%3DSchweizer%26aufirst%3DW.%2BB.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26aulast%3DCarreira%26aufirst%3DE.%2BM.%26atitle%3DSpirocyclic%2520oxetanes%253A%2520synthesis%2520and%2520properties%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D4512%26epage%3D4515%26doi%3D10.1002%2Fanie.200800450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagdanoff, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madiera, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, D.</span></span> <span> </span><span class="NLM_article-title">Tetrahydropyrrolo-diazepenones as inhibitors of ERK2 kinase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3788</span>– <span class="NLM_lpage">3792</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.07.091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.bmcl.2015.07.091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=26259804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KrtLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=3788-3792&author=J.+T.+Bagdanoffauthor=R.+Jainauthor=W.+Hanauthor=S.+Zhuauthor=A.-M.+Madieraauthor=P.+S.+Leeauthor=X.+Maauthor=D.+Poon&title=Tetrahydropyrrolo-diazepenones+as+inhibitors+of+ERK2+kinase&doi=10.1016%2Fj.bmcl.2015.07.091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Tetrahydropyrrolo-diazepenones as inhibitors of ERK2 kinase</span></div><div class="casAuthors">Bagdanoff, Jeffrey T.; Jain, Rama; Han, Wooseok; Zhu, Shejin; Madiera, Ann-Marie; Lee, Patrick S.; Ma, Xiaolei; Poon, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3788-3792</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of structure based drug design hypotheses and focused screening efforts led to the identification of tetrahydropyrrolo-diazepenones with striking potency against ERK2 kinase.  The role of fluorination in mitigating microsomal clearance was systematically explored.  Ultimately, it was found that fluorination of a cyclopentanol substructure provided significant improvement in both potency and human metabolic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHXEHIlXy0QLVg90H21EOLACvtfcHk0ljsYsaHaUf93A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KrtLnF&md5=d94c52debb8203a3b9e655319ea32754</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.07.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.07.091%26sid%3Dliteratum%253Aachs%26aulast%3DBagdanoff%26aufirst%3DJ.%2BT.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DHan%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DMadiera%26aufirst%3DA.-M.%26aulast%3DLee%26aufirst%3DP.%2BS.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DPoon%26aufirst%3DD.%26atitle%3DTetrahydropyrrolo-diazepenones%2520as%2520inhibitors%2520of%2520ERK2%2520kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D3788%26epage%3D3792%26doi%3D10.1016%2Fj.bmcl.2015.07.091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stepan, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefer, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M.</span></span> <span> </span><span class="NLM_article-title">Evaluating the differences in cycloalkyl ether metabolism using the design parameter “lipophilic metabolism efficiency” (LipMetE) and a matched molecular pairs analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6985</span>– <span class="NLM_lpage">6990</span>, <span class="refDoi"> DOI: 10.1021/jm4008642</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4008642" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFKgurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6985-6990&author=A.+F.+Stepanauthor=G.+W.+Kauffmanauthor=C.+E.+Keeferauthor=P.+R.+Verhoestauthor=M.+Edwards&title=Evaluating+the+differences+in+cycloalkyl+ether+metabolism+using+the+design+parameter+%E2%80%9Clipophilic+metabolism+efficiency%E2%80%9D+%28LipMetE%29+and+a+matched+molecular+pairs+analysis&doi=10.1021%2Fjm4008642"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the Differences in Cycloalkyl Ether Metabolism Using the Design Parameter "Lipophilic Metabolism Efficiency" (LipMetE) and a Matched Molecular Pairs Analysis</span></div><div class="casAuthors">Stepan, Antonia F.; Kauffman, Gregory W.; Keefer, Christopher E.; Verhoest, Patrick R.; Edwards, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6985-6990</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have obsd. previously that modification of ring size and substitution pattern may be used as a strategy to mitigate the metabolic instability of cycloalkyl ethers.  In this article, we introduce a medicinal chem. design parameter named "lipophilic metab. efficiency" (LipMetE) that indicates that these changes in metabolic stability can be largely ascribed to changes in lipophilicity.  Our matched mol. pair anal. also indicates that this finding is a general phenomenon, widely obsd. across different chemotypes.  It is our hope that both the LipMetE design parameter and the results from our pairwise anal. will be useful tools for medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpiFJ_fPxurbVg90H21EOLACvtfcHk0ljsYsaHaUf93A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFKgurvJ&md5=596673d35831b979b8daf5a4ccb8a218</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fjm4008642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4008642%26sid%3Dliteratum%253Aachs%26aulast%3DStepan%26aufirst%3DA.%2BF.%26aulast%3DKauffman%26aufirst%3DG.%2BW.%26aulast%3DKeefer%26aufirst%3DC.%2BE.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DEdwards%26aufirst%3DM.%26atitle%3DEvaluating%2520the%2520differences%2520in%2520cycloalkyl%2520ether%2520metabolism%2520using%2520the%2520design%2520parameter%2520%25E2%2580%259Clipophilic%2520metabolism%2520efficiency%25E2%2580%259D%2520%2528LipMetE%2529%2520and%2520a%2520matched%2520molecular%2520pairs%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6985%26epage%3D6990%26doi%3D10.1021%2Fjm4008642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottschall, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theodorides, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span> <span> </span><span class="NLM_article-title">The metabolism of benzimidazole anthelmintics</span>. <i>Parasitol. Today</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">124</span>, <span class="refDoi"> DOI: 10.1016/0169-4758(90)90228-v</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2F0169-4758%2890%2990228-v" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=15463313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaK3cXksFWru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1990&pages=115-124&author=D.+W.+Gottschallauthor=V.+J.+Theodoridesauthor=R.+Wang&title=The+metabolism+of+benzimidazole+anthelmintics&doi=10.1016%2F0169-4758%2890%2990228-v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">The metabolism of benzimidazole anthelmintics</span></div><div class="casAuthors">Gottschall, David W.; Theodorides, Vassilios J.; Wang, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Parasitology Today</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">115-24</span>CODEN:
                <span class="NLM_cas:coden">PATOE2</span>;
        ISSN:<span class="NLM_cas:issn">0169-4758</span>.
    </div><div class="casAbstract">A review with 26 refs. discussing the metab. of benzimidazole carbamates in lab. animals and humans and the problems of poor water soly. and absorption of these anthelmintics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG4gum4VKifrVg90H21EOLACvtfcHk0lj_riDq08MGPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXksFWru7g%253D&md5=f5ffb20d83d4040275fa38121db7f6dc</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2F0169-4758%2890%2990228-v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0169-4758%252890%252990228-v%26sid%3Dliteratum%253Aachs%26aulast%3DGottschall%26aufirst%3DD.%2BW.%26aulast%3DTheodorides%26aufirst%3DV.%2BJ.%26aulast%3DWang%26aufirst%3DR.%26atitle%3DThe%2520metabolism%2520of%2520benzimidazole%2520anthelmintics%26jtitle%3DParasitol.%2520Today%26date%3D1990%26volume%3D6%26spage%3D115%26epage%3D124%26doi%3D10.1016%2F0169-4758%2890%2990228-v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiltshire, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heeps, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betty, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angus, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worth, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welker, H. A.</span></span> <span> </span><span class="NLM_article-title">Metabolism of the calcium antagonist, mibefradil (POSICORTM, Ro 40-5967). Part III. Comparative pharmacokinetics of mibefradil and its major metabolites in rat, marmoset, cynomolgus monkey and man</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">557</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1080/004982597240343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1080%2F004982597240343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=9211656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaK2sXkt1Cjt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1997&pages=557-571&author=H.+R.+Wiltshireauthor=B.+M.+Suttonauthor=G.+Heepsauthor=A.+M.+Bettyauthor=D.+W.+Angusauthor=S.+R.+Harrisauthor=E.+Worthauthor=H.+A.+Welker&title=Metabolism+of+the+calcium+antagonist%2C+mibefradil+%28POSICORTM%2C+Ro+40-5967%29.+Part+III.+Comparative+pharmacokinetics+of+mibefradil+and+its+major+metabolites+in+rat%2C+marmoset%2C+cynomolgus+monkey+and+man&doi=10.1080%2F004982597240343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of the calcium antagonist, mibefradil (POSICOR, Ro 40-5967). Part III. Comparative pharmacokinetics of mibefradil and its major metabolites in rat, marmoset, cynomolgus monkey and man</span></div><div class="casAuthors">Wiltshire, H. R.; Sutton, B. M.; Heeps, G.; Betty, A. M.; Angus, D. W.; Harris, S. R.; Worth, E.; Welker, H. A.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">557-571</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis</span>)
        </div><div class="casAbstract">1. The metab. of mibefradil has been examd. in rat, marmoset, cynomolgus monkey and man after single and multiple oral administration.  2. Metabolites typically represent between 50 and 80% of the circulating drug-related material after single oral doses of mibefradil to man, rat and marmoset.  They arise by a combination of enzymic processes: cytochrome P 450-mediated oxidn. at satd. and unsatd. carbon atoms, cytochrome P 450-catalyzed dealkylation and hydrolysis of the ester side-chain.  3. Plasma levels of mibefradil in the cynomolgus monkey are extremely low as a result of very efficient first-pass hydrolysis of its side-chain to give the corresponding alc.  Steady-state concns. of this metabolite are comparable with those of the parent drug in man and marmoset, but are relatively low in rat plasma.  4. Hydroxylation at the benzylic carbon of the tetrahydronaphthyl ring leads to further important metabolites in primates, whereas the products of O- and N-demethylation are found in small amts. in all four species.  5. Ests. of the exposure of the various species to the principal metabolites indicate that the choice of the rat, marmoset and cynomolgus monkey for the toxicol. assessment of mibefradil was appropriate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrikNYXcqCgYLVg90H21EOLACvtfcHk0lj_riDq08MGPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkt1Cjt7s%253D&md5=e2ce7f237b43d9a2f1ec51155be0c9a0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1080%2F004982597240343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F004982597240343%26sid%3Dliteratum%253Aachs%26aulast%3DWiltshire%26aufirst%3DH.%2BR.%26aulast%3DSutton%26aufirst%3DB.%2BM.%26aulast%3DHeeps%26aufirst%3DG.%26aulast%3DBetty%26aufirst%3DA.%2BM.%26aulast%3DAngus%26aufirst%3DD.%2BW.%26aulast%3DHarris%26aufirst%3DS.%2BR.%26aulast%3DWorth%26aufirst%3DE.%26aulast%3DWelker%26aufirst%3DH.%2BA.%26atitle%3DMetabolism%2520of%2520the%2520calcium%2520antagonist%252C%2520mibefradil%2520%2528POSICORTM%252C%2520Ro%252040-5967%2529.%2520Part%2520III.%2520Comparative%2520pharmacokinetics%2520of%2520mibefradil%2520and%2520its%2520major%2520metabolites%2520in%2520rat%252C%2520marmoset%252C%2520cynomolgus%2520monkey%2520and%2520man%26jtitle%3DXenobiotica%26date%3D1997%26volume%3D27%26spage%3D557%26epage%3D571%26doi%3D10.1080%2F004982597240343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craig, P. N.</span></span> <span> </span><span class="NLM_article-title">Interdependence between physical parameters and selection of substituent groups for correlation studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">680</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1021/jm00290a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00290a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaE3MXkvFKqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1971&pages=680-684&author=P.+N.+Craig&title=Interdependence+between+physical+parameters+and+selection+of+substituent+groups+for+correlation+studies&doi=10.1021%2Fjm00290a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Interdependence between physical parameters and selection of substituent groups for correlation studies</span></div><div class="casAuthors">Craig, Paul N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">680-4</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The phys. properties, σ, π, and ER values, are not linearly related with biol. activities, although the interdependence of ER and σ2 values is confirmed.  The aliphatic π values are correlated with the Taft's aliphatic Es values.  Significant correlations are also found for π and mol. vol., and for refractivity and mol. vol. values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRh8piBnGLELVg90H21EOLACvtfcHk0lj_riDq08MGPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXkvFKqu7w%253D&md5=05557e45ee7c24a71d7785b20ddc4160</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fjm00290a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00290a004%26sid%3Dliteratum%253Aachs%26aulast%3DCraig%26aufirst%3DP.%2BN.%26atitle%3DInterdependence%2520between%2520physical%2520parameters%2520and%2520selection%2520of%2520substituent%2520groups%2520for%2520correlation%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1971%26volume%3D14%26spage%3D680%26epage%3D684%26doi%3D10.1021%2Fjm00290a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dart, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupcho, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niles, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Target engagement and drug residence time can be observed in living cells with BRET</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">10091</span>, <span class="refDoi"> DOI: 10.1038/ncomms10091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1038%2Fncomms10091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=26631872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=10091&author=M.+B.+Robersauthor=M.+L.+Dartauthor=C.+C.+Woodroofeauthor=C.+A.+Zimprichauthor=T.+A.+Kirklandauthor=T.+Machleidtauthor=K.+R.+Kupchoauthor=S.+Levinauthor=J.+R.+Hartnettauthor=K.+Zimmermanauthor=A.+L.+Nilesauthor=R.+F.+Ohanaauthor=D.+L.+Danielsauthor=M.+Slaterauthor=M.+G.+Woodauthor=M.+Congauthor=Y.-Q.+Chengauthor=K.+V.+Wood&title=Target+engagement+and+drug+residence+time+can+be+observed+in+living+cells+with+BRET&doi=10.1038%2Fncomms10091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Target engagement and drug residence time can be observed in living cells with BRET</span></div><div class="casAuthors">Robers, Matthew B.; Dart, Melanie L.; Woodroofe, Carolyn C.; Zimprich, Chad A.; Kirkland, Thomas A.; Machleidt, Thomas; Kupcho, Kevin R.; Levin, Sergiy; Hartnett, James R.; Zimmerman, Kristopher; Niles, Andrew L.; Ohana, Rachel Friedman; Daniels, Danette L.; Slater, Michael; Wood, Monika G.; Cong, Mei; Cheng, Yi-Qiang; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10091</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The therapeutic action of drugs is predicated on their phys. engagement with cellular targets.  Here we describe a broadly applicable method using bioluminescence resonance energy transfer (BRET) to reveal the binding characteristics of a drug with selected targets within intact cells.  Cell-permeable fluorescent tracers are used in a competitive binding format to quantify drug engagement with the target proteins fused to Nanoluc luciferase.  The approach enabled us to profile isoenzyme-specific engagement and binding kinetics for a panel of histone deacetylase (HDAC) inhibitors.  Our anal. was directed particularly to the clin. approved prodrug FK228 (Istodax/Romidepsin) because of its unique and largely unexplained mechanism of sustained intracellular action.  Anal. of the binding kinetics by BRET revealed remarkably long intracellular residence times for FK228 at HDAC1, explaining the protracted intracellular behavior of this prodrug.  Our results demonstrate a novel application of BRET for assessing target engagement within the complex milieu of the intracellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFFk0LNiL_LVg90H21EOLACvtfcHk0liYaAn6i6OxrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGmtrnO&md5=723cc5dd7d229b1115da4c2c9d75c913</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fncomms10091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10091%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DDart%26aufirst%3DM.%2BL.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKupcho%26aufirst%3DK.%2BR.%26aulast%3DLevin%26aufirst%3DS.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DNiles%26aufirst%3DA.%2BL.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DM.%2BG.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DY.-Q.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DTarget%2520engagement%2520and%2520drug%2520residence%2520time%2520can%2520be%2520observed%2520in%2520living%2520cells%2520with%2520BRET%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D10091%26doi%3D10.1038%2Fncomms10091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14532</span>– <span class="NLM_lpage">14537</span>, <span class="refDoi"> DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0liYaAn6i6OxrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0liLZ6k7ZYhUog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liLZ6k7ZYhUog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/s0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2Fs0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0liLZ6k7ZYhUog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2Fs0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liLZ6k7ZYhUog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgSHTjz7YpQtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX','PDB','6TPX'); return false;">PDB: 6TPX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ1','PDB','6TQ1'); return false;">PDB: 6TQ1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ2','PDB','6TQ2'); return false;">PDB: 6TQ2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY','PDB','6TPY'); return false;">PDB: 6TPY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LZ2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LZ2','PDB','4LZ2'); return false;">PDB: 4LZ2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPZ','PDB','6TPZ'); return false;">PDB: 6TPZ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i123"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01670">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10627"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01670?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01670</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Affinity screening of the encoded library method, BRD4 FP assay method, representative compound spectra, full assay data table, representative BCP selectivity data for dimethylpyridone benzimidazole <b>20a</b>, chromLog<i>D</i><sub>7.4</sub> method, solubility methods and data, X-ray powder diffraction (XRPD) method and data, mouse in vivo pharmacokinetic data for 2,6-dimethylphenol benzimidazole <b>10a</b>, dose prediction calculations, in vitro clearance data for 5-dimethylamine <b>22c</b> and 5-morpholine <b>23e</b>, bromodomain binding assay panel data, NanoBRET method and data, X-ray crystallization methods and data, and developability and cross screening panel data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data for final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_001.pdf">jm9b01670_si_001.pdf (1.11 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01670/suppl_file/jm9b01670_si_002.csv">jm9b01670_si_002.csv (5.89 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Cocrystal structures of BRD4 BD1 in the complex with compounds <b>10</b>, <b>20a</b>, and <b>24</b> and cocrystal structures of BRD2 BD1 in the complex with compounds <b>16</b> and <b>17</b> are deposited under the accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPX">6TPX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPY">6TPY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TPZ">6TPZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ1">6TQ1</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TQ2">6TQ2</a>, in the Protein Data Bank, respectively. Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01670%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01670" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994b23ffe53d48","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
