
<html lang="en"     class="pb-page"  data-request-id="ada99031-13bb-4ec6-8d29-9295b301da3c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2017.60.issue-13;article:article:10.1021/acs.jmedchem.7b00461;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Allosteric Inhibitors Targeting the Spliceosomal RNA Helicase Brr2" /></meta><meta name="dc.Creator" content="Misa  Iwatani-Yoshihara" /></meta><meta name="dc.Creator" content="Masahiro  Ito" /></meta><meta name="dc.Creator" content="Michael G.  Klein" /></meta><meta name="dc.Creator" content="Takeshi  Yamamoto" /></meta><meta name="dc.Creator" content="Kazuko  Yonemori" /></meta><meta name="dc.Creator" content="Toshio  Tanaka" /></meta><meta name="dc.Creator" content="Masanori  Miwa" /></meta><meta name="dc.Creator" content="Daisuke  Morishita" /></meta><meta name="dc.Creator" content="Satoshi  Endo" /></meta><meta name="dc.Creator" content="Richard  Tjhen" /></meta><meta name="dc.Creator" content="Ling  Qin" /></meta><meta name="dc.Creator" content="Atsushi  Nakanishi" /></meta><meta name="dc.Creator" content="Hironobu  Maezaki" /></meta><meta name="dc.Creator" content="Tomohiro  Kawamoto" /></meta><meta name="dc.Description" content="Brr2 is an RNA helicase belonging to the Ski2-like subfamily and an essential component of spliceosome. Brr2 catalyzes an ATP-dependent unwinding of the U4/U6 RNA duplex, which is a critical step f..." /></meta><meta name="Description" content="Brr2 is an RNA helicase belonging to the Ski2-like subfamily and an essential component of spliceosome. Brr2 catalyzes an ATP-dependent unwinding of the U4/U6 RNA duplex, which is a critical step f..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 20, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00461" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00461" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00461" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00461" /></link>
        
    
    

<title>Discovery of Allosteric Inhibitors Targeting the Spliceosomal RNA Helicase Brr2 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00461" /></meta><meta property="og:title" content="Discovery of Allosteric Inhibitors Targeting the Spliceosomal RNA Helicase Brr2" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0012.jpeg" /></meta><meta property="og:description" content="Brr2 is an RNA helicase belonging to the Ski2-like subfamily and an essential component of spliceosome. Brr2 catalyzes an ATP-dependent unwinding of the U4/U6 RNA duplex, which is a critical step for spliceosomal activation. An HTS campaign using an RNA-dependent ATPase assay and initial SAR study identified two different Brr2 inhibitors, 3 and 12. Cocrystal structures revealed 3 binds to an unexpected allosteric site between the C-terminal and the N-terminal helicase cassettes, while 12 binds an RNA-binding site inside the N-terminal cassette. Selectivity profiling indicated the allosteric inhibitor 3 is more Brr2-selective than the RNA site binder 12. Chemical optimization of 3 using SBDD culminated in the discovery of the potent and selective Brr2 inhibitor 9 with helicase inhibitory activity. Our findings demonstrate an effective strategy to explore selective inhibitors for helicases, and 9 could be a promising starting point for exploring molecular probes to elucidate biological functions and the therapeutic relevance of Brr2." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00461"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00461">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00461&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00461&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00461&amp;href=/doi/10.1021/acs.jmedchem.7b00461" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 5759-5771</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00428" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00457" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Allosteric Inhibitors Targeting the Spliceosomal RNA Helicase Brr2</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Misa++Iwatani-Yoshihara">Misa Iwatani-Yoshihara</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="author-xref-symbol "><sup>#</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-8083-3560" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masahiro++Ito">Masahiro Ito</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span><span class="author-xref-symbol "><sup>#</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-5965-535X" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+G.++Klein">Michael G. Klein</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Takeshi++Yamamoto">Takeshi Yamamoto</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kazuko++Yonemori">Kazuko Yonemori</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Toshio++Tanaka">Toshio Tanaka</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masanori++Miwa">Masanori Miwa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daisuke++Morishita">Daisuke Morishita</a></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Satoshi++Endo">Satoshi Endo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Tjhen">Richard Tjhen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ling++Qin">Ling Qin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Atsushi++Nakanishi">Atsushi Nakanishi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hironobu++Maezaki">Hironobu Maezaki</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tomohiro++Kawamoto">Tomohiro Kawamoto</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1.0"><span class="aff-text"><sup>â </sup>Biomolecular Research Laboratories, <sup>â¡</sup>Medicinal Chemistry Research Laboratories, <sup>Â§</sup>Oncology Drug Discovery Unit, and <sup>â¥</sup>Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">â¥</span> <span class="aff-text">Department of Structural Biology, Takeda California Inc., 10410 Science Center Drive, San Diego, California 92121, United States</span></div><div class="corresp-info"><strong>*</strong>M.I.-Y.: phone, +81 466 32 2809; e-mail, <a href="/cdn-cgi/l/email-protection#fa9793899bd4938d9b8e9b9493ba8e9b919f9e9bd4999597"><span class="__cf_email__" data-cfemail="b3dedac0d29ddac4d2c7d2dddaf3c7d2d8d6d7d29dd0dcde">[emailÂ protected]</span></a></div><div class="corresp-info"><strong>*</strong>M.I.: phone, +81 466 32 1196; e-mail, <a href="/cdn-cgi/l/email-protection#335e5240525b5a415c1d5a475c734752585657521d505c5e"><span class="__cf_email__" data-cfemail="8ee3effdefe6e7fce1a0e7fae1cefaefe5ebeaefa0ede1e3">[emailÂ protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00461&amp;href=/doi/10.1021%2Facs.jmedchem.7b00461" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 5759â5771</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 6, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 March 2017</li><li><span class="item_label"><b>Published</b> online</span>20 June 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 July 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00461" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00461</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5759%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMisa%2BIwatani-Yoshihara%252C%2BMasahiro%2BIto%252C%2BMichael%2BG.%2BKlein%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D13%26contentID%3Dacs.jmedchem.7b00461%26title%3DDiscovery%2Bof%2BAllosteric%2BInhibitors%2BTargeting%2Bthe%2BSpliceosomal%2BRNA%2BHelicase%2BBrr2%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5771%26publicationDate%3DJuly%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00461"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2227</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">17</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00461" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Allosteric Inhibitors Targeting the Spliceosomal RNA Helicase Brr2&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Misa&quot;,&quot;last_name&quot;:&quot;Iwatani-Yoshihara&quot;},{&quot;first_name&quot;:&quot;Masahiro&quot;,&quot;last_name&quot;:&quot;Ito&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;G. Klein&quot;},{&quot;first_name&quot;:&quot;Takeshi&quot;,&quot;last_name&quot;:&quot;Yamamoto&quot;},{&quot;first_name&quot;:&quot;Kazuko&quot;,&quot;last_name&quot;:&quot;Yonemori&quot;},{&quot;first_name&quot;:&quot;Toshio&quot;,&quot;last_name&quot;:&quot;Tanaka&quot;},{&quot;first_name&quot;:&quot;Masanori&quot;,&quot;last_name&quot;:&quot;Miwa&quot;},{&quot;first_name&quot;:&quot;Daisuke&quot;,&quot;last_name&quot;:&quot;Morishita&quot;},{&quot;first_name&quot;:&quot;Satoshi&quot;,&quot;last_name&quot;:&quot;Endo&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Tjhen&quot;},{&quot;first_name&quot;:&quot;Ling&quot;,&quot;last_name&quot;:&quot;Qin&quot;},{&quot;first_name&quot;:&quot;Atsushi&quot;,&quot;last_name&quot;:&quot;Nakanishi&quot;},{&quot;first_name&quot;:&quot;Hironobu&quot;,&quot;last_name&quot;:&quot;Maezaki&quot;},{&quot;first_name&quot;:&quot;Tomohiro&quot;,&quot;last_name&quot;:&quot;Kawamoto&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;5759-5771&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00461&quot;},&quot;abstract&quot;:&quot;Brr2 is an RNA helicase belonging to the Ski2-like subfamily and an essential component of spliceosome. Brr2 catalyzes an ATP-dependent unwinding of the U4/U6 RNA duplex, which is a critical step for spliceosomal activation. An HTS campaign using an RNA-dependent ATPase assay and initial SAR study identified two different Brr2 inhibitors, 3 and 12. Cocrystal structures revealed 3 binds to an unexpected allosteric site between the C-terminal and the N-terminal helicase cassettes, while 12 binds an RNA-binding site inside the N-terminal cassette. Selectivity profiling indicated the allosteric inhibitor 3 is more Brr2-selective than the RNA site binder 12. Chemical optimization of 3 using SBDD culminated in the discovery of the potent and selective Brr2 inhibitor 9 with helicase inhibitory activity. Our findings demonstrate an effective strategy to explore selective inhibitors for helicases, and 9 could be a promising starting point for exploring molecular probes to elucidate biological functions and the therape&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00461&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00461" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00461&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00461" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00461&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00461" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00461&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00461&amp;href=/doi/10.1021/acs.jmedchem.7b00461" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00461" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00461" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00461%26sid%3Dliteratum%253Aachs%26pmid%3D28586220%26genre%3Darticle%26aulast%3DIwatani-Yoshihara%26date%3D2017%26atitle%3DDiscovery%2Bof%2BAllosteric%2BInhibitors%2BTargeting%2Bthe%2BSpliceosomal%2BRNA%2BHelicase%2BBrr2%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D13%26spage%3D5759%26epage%3D5771%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/jmcmar.2017.60.issue-13/20170713/jmcmar.2017.60.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Brr2 is an RNA helicase belonging to the Ski2-like subfamily and an essential component of spliceosome. Brr2 catalyzes an ATP-dependent unwinding of the U4/U6 RNA duplex, which is a critical step for spliceosomal activation. An HTS campaign using an RNA-dependent ATPase assay and initial SAR study identified two different Brr2 inhibitors, <b>3</b> and <b>12</b>. Cocrystal structures revealed <b>3</b> binds to an unexpected allosteric site between the C-terminal and the N-terminal helicase cassettes, while <b>12</b> binds an RNA-binding site inside the N-terminal cassette. Selectivity profiling indicated the allosteric inhibitor <b>3</b> is more Brr2-selective than the RNA site binder <b>12</b>. Chemical optimization of <b>3</b> using SBDD culminated in the discovery of the potent and selective Brr2 inhibitor <b>9</b> with helicase inhibitory activity. Our findings demonstrate an effective strategy to explore selective inhibitors for helicases, and <b>9</b> could be a promising starting point for exploring molecular probes to elucidate biological functions and the therapeutic relevance of Brr2.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In eukaryotic cells, protein-coding transcripts are produced as pre-mRNAs that contain introns. Splicing of pre-mRNAs is an essential step where introns are removed, and exons are joined together to form mature mRNA, which is a prerequisite step for protein translation. Splicing is catalyzed by a dynamic multimegadalton ribonucleoprotein (RNP) complex, called the spliceosome.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> The spliceosome consists of five small nuclear RNPs (snRNPs), U1, U2, U4, U5, and U6. Brr2 is an essential component of U5 and belongs to the Ski2-like RNA helicase family.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a> Brr2 disrupts the base-pairing of the U4/U6 small molecular RNA duplex, which is the major driving force for catalytic activation of the spliceosome. Since Brr2 remains tightly associated with the spliceosome even after the splicing catalysis step, it is considered to play critical roles through the splicing event.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">Brr2 consists of two multidomain helicase cassettes. Both cassettesâ domains are composed of dual RecA-like domains, a winged helix (WH) domain, and a Sec63 homology unit composed of a helical bundle (HB) domain, a helixâloopâhelix (HLH) domain, and an immunoglobulin-like (IG) domain.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Several reports have demonstrated that only the N-terminal cassette possesses ATPase activity and RNA unwinding activity.<a onclick="showRef(event, 'ref5 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref7">(5, 7)</a> In contrast, the C-terminal cassette is reportedly inactive<a onclick="showRef(event, 'ref5 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref8 ref9">(5, 7-9)</a> and functions as a platform mediating proteinâprotein interaction.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Mutations in the <i>BRR2</i> gene are implicated in autosomal-dominant retinitis pigmentosa, a group of progressive retinal degenerative disorders. The pathogenic mutations c.3260C>T (p.S1087L) and c.3269G>T (p.R1090L) impair the fidelity of gene expression accompanied by reduction of the ATPase and U4/U6 unwinding activity,<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11-13)</a> indicating that the biological function of Brr2 is critically connected to the mechanism of disease.</div><div class="NLM_p last">In spite of its biological importance, molecular probes targeting Brr2 have not been reported to date. Selective small molecular Brr2 inhibitors will provide a new and attractive option for molecular biologists addressing splicing machinery. However, identification of specific inhibitors for helicases has been quite challenging because screening using helicase assays is generally low-throughput and yields many false positives such as nucleic acid binders.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Here, we present the discovery of highly specific Brr2 inhibitors using RNA-dependent ATPase assay, X-ray cocrystal structure analysis, and structure based chemical optimization.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07620" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07620" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Outline of the Discovery Process of Brr2 Inhibitors</h3><div class="NLM_p">The discovery process of selective Brr2 inhibitors is summarized in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. At first, a high-throughput screening of in-house library yielded two structurally distinct chemical series, 4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione derivative <b>1</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and 5-imino-7-oxo-5<i>H</i>-[1,3,4]thiadiazolo[3,2-<i>a</i>]pyrimidin derivative <b>11</b>. Next, the validity of these chemical series was further investigated using SAR study, X-ray cocrystal structure analysis, and selectivity assays. Compound <b>3</b>, an analog of compound <b>1</b>, was ascertained to bind an allosteric site of Brr2 by X-ray crystal structure analysis. Meanwhile, compound <b>12</b>, an analog of compound <b>11</b>, turned out to be a RNA site binder. Since Brr2 selectivity of the allosteric inhibitor <b>3</b> is superior to that of the RNA site inhibitor <b>12</b>, we selected <b>3</b> as a lead compound for further chemical optimization. Then, structural simplification of compound <b>3</b> using SBDD led to the identification of the more potent compound <b>6</b>. Further, structure based optimization of <b>6</b> culminated in the discovery of compound <b>9</b> with double digit nanomolar Brr2 ATPase inhibitory activity, excellent Brr2 selectivity, good aqueous solubility, and optimal Brr2 helicase inhibitory activity. Further details of each process are described in the following sections.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic flowchart summarizing the process of the discovery of Brr2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> HTS, Hit Validation, and Preliminary SAR</h3><div class="NLM_p">First screening of Brr2 inhibitors was conducted using full-length recombinant Brr2 protein and RNA-dependent ATPase activity assay, which was developed based on ADP-Glo assay. Then, a phosphate sensor assay, which detects inorganic phosphate, was used for the secondary screening. Compounds meeting target criteria of the first and secondary screening (see details in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Table S1 in Supporting Information</a>) were subjected to a dose-dependent ATPase assay, and their IC<sub>50</sub> values were determined. After that, the identified single- or double-digit micromolar inhibitors were characterized using a substrate competition assay and a thermal shift assay to exclude nonspecific binders. Following this process, several chemical series, including compound <b>1</b> and compound <b>11</b>, were identified (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Compounds <b>1</b> and <b>11</b> inhibited the ATPase activity of Brr2 with IC<sub>50</sub> values of 36 Î¼M and 4.2 Î¼M, respectively. Additionally, <b>1</b> and <b>11</b> induced thermal shifts (Î<i>T</i><sub>m</sub>) of 2.0 and 2.3 Â°C, respectively. In order to elucidate the key functional groups for inhibitory activity of <b>1</b> and <b>11</b>, preliminary SAR was investigated. As shown in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, replacement of the trifluoroethyl group of <b>1</b> with a methyl group was not tolerated (<b>2</b>), whereas that with a larger benzyl group (<b>3</b>) showed increased activity. Methylation at the R<sub>2</sub> position in compound <b>3</b> resulted in reduced activity (<b>4</b>), while demethylation at the R<sub>3</sub> group (<b>5</b>) was tolerated for activity. Meanwhile, SAR study of <b>11</b> revealed that the carboxyl group was essential for activity (<b>11</b> vs <b>13</b>), and the distal propyl chain was replaceable with a cyclohexyl ring (<b>12</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. In Vitro Profile of Compounds <b>1</b> and <b>11</b><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0008.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">Brr2 ATPase IC<sub>50</sub>Â (Î¼M)<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">thermal shift Î<i>T</i><sub>m</sub>Â (Â°C), 100Â Î¼M/33Â Î¼M</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">36 (23â55)</td><td class="colsep0 rowsep0" align="left">2.0/1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">4.2 (2.6â6.8)</td><td class="colsep0 rowsep0" align="left">2.3/1.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">The denaturation rates are plotted as indicated by green for DMSO-treated control, blue for 33.3 Î¼M, and red for 100 Î¼M of test compounds, respectively.</p></div><div class="footnote" id="t1fn1"><sup>b</sup><p class="last"><i>n</i> = 2, 95% confidence intervals shown in parentheses.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of Compounds <b>1</b>â<b>5</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0009.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">R<sub>3</sub></th><th class="colsep0 rowsep0" align="center">Brr2 ATPase IC<sub>50</sub>Â (Î¼M)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b> (hit)</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">36 (23â55)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">Bn</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">5.3 (2.9â9.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">Bn</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">Bn</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">1.7 (0.96â3.2)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last"><i>n</i> = 2, 95% confidence intervals shown in parentheses.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Crystal Structural Analysis for Brr2 Inhibitors and Selectivity Study</h3><div class="NLM_p">The X-ray cocrystal structures of compound <b>3</b> and <b>12</b> in complex with Brr2 were determined as shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A. For crystallization trials, we used the truncated human Brr2 lacking the N-terminal extension and some C-terminal residues (395â2129 residues). Focused views of the binding modes of <b>3</b> and <b>12</b> are shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B. Notably, the Brr2 cocrystal structure revealed that compound <b>3</b> binds to a newly identified allosteric site located in the protein interface formed between the second RecA domain of the C-terminal helicase cassette and the Ig domain from the N-terminal helicase cassette (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C and <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Overall cocrystal structure of Brr2 with compound <b>3</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ">5URJ</a>) and <b>12</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URM">5URM</a>). (Upper) Ribbon plot of Brr2. A green ligand represents compound <b>3</b>, a magenta ligand represents compound <b>12</b>, and yellow ligands represent nucleotides from a published structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F93">4F93</a>). N-terminal cassette: RecA-1, orange; RecA-2, yellow; WH, gray; HB, blue; HLH, pink; IG, green. C-terminal cassette: RecA-1, pale red; RecA-2, pale yellow; WH, gray; HB, cyan; HLH, pale pink; IG, yellow green. (Lower) Schematic representation of Brr2 in domain borders. (B) Binding site of <b>3</b>* (left) and <b>12</b> (right) in Brr2. (C) Binding mode of the left side of <b>3</b>. (D) Binding mode of right side of <b>3</b>. *The binding isomer is estimated to be 2<i>R</i>-form.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There are four key features of the cocrystal structure of <b>3</b> with Brr2: (1) Compound <b>3</b> binds in a U-shaped conformation and is surrounded by numerous hydrophobic residues including five aromatic side chains (i.e., Phe1254, Phe1255, Tyr1682, Phe1713, and Phe1717). (2) The carbonyl of the pyridone ring in <b>3</b> forms a hydrogen-bonding interaction with the main chain NH of Ile1681 at a distance of 2.7 Ã. (3) The benzyl moiety at the R<sub>1</sub> in <b>3</b> occupies a large hydrophobic cavity formed by side chains of Ile1193, Phe1255, Phe1713, Phe1717, Leu1722, and Ile1681. (4) The distal 3-fluoro-pyridin-2-yl moiety occupies a small cavity formed by Thr1197, Pro1257, Asp1712, Phe1713, Lys1716, and Phe1717. The fact that compound <b>3</b> resides in the pocket formed between C-terminal helicase cassette and the N-terminal cassette suggests that compound <b>3</b> may affect N- and C-terminal cassette interactions<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> of Brr2. However, apparent positional changes of the amino acid residues, essential for intercassette interactions,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> were not observed in the absence or presence of ligand <b>3</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Figure S1</a>). Thus, the inhibitory mechanism of <b>3</b> remains unclear, and further investigation using other techniques (e.g., site mutation or NMR) is necessary to elucidate the exact inhibitory mechanism of compound <b>3</b>. In contrast, the SAR in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> can be clearly explained by the cocrystal structure. Specifically, the decreased activity of compound <b>2</b> (R<sub>1</sub> = methyl), when compared to <b>1</b> and <b>3</b>, is accounted for by insufficient bulkiness and hydrophobicity of the R<sub>1</sub> group at the hydrophobic cavity. Meanwhile, a large unidentified blob of electron density was found between the urea carbonyl and residues Thr1666 and Asp1678. On the basis of their relative position to the cyclic urea of <b>3</b> and the SAR (<b>3</b> vs <b>4</b>), it is reasonable to assume that Thr1666 or Asp1678 could make a hydrogen bond to the NH group of <b>3</b> via water-mediated interactions, whereas the putative water molecule was undefined in the crystal, presumably due to limitations in the resolution of the diffraction data. Meanwhile, the methyl group (R<sub>3</sub>) of <b>3</b> directs toward the side chain of Asp1678 and has no positive interaction with Brr2 protein. The absence of interaction is consistent with the SAR between <b>3</b> and <b>5</b>. The role of the dihydropyran ring of <b>3</b> was not clearly defined by the cocrystal structure. We speculated that the ring would not be important for binding and was just working as a linker between the phenyl ring and the 3-fluoropyridin-2-yl moiety.</div><div class="NLM_p">On the contrary, as shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B, compound <b>12</b> sits inside an RNA binding site, leading to reduction of RNA-stimulated ATPase activity. The RNA binding site is formed by the dual RecA domains and the HB/HLH domains of the N-terminal helicase cassette and runs through the cassette as a large open-ended channel. Notably, a pyrimidine nitrogen atom in compound <b>12</b> forms a hydrogen bond with the backbone NH of residue Arg545, and an oxygen atom from the carboxylic acid moiety of compound <b>12</b> interacts with the side chain of Thr589. Especially, the latter observation is consistent with the carboxy group being essential for its activity. (Further details for the binding site of compound <b>12</b> are described in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Supporting Information, Figure S2</a>.) Thus, direct binding of <b>3</b> and <b>12</b> against Brr2 and their binding modes were ascertained. Then, <b>3</b> and <b>12</b> were subjected to selectivity profiling against other helicases. As a result, it turned out that allosteric inhibitor <b>3</b> is more selective than RNA-site binder <b>12</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Hence, we selected compound <b>3</b> as a lead compound for further optimization.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Selectivity of <b>3</b> and <b>12</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub>Â (Î¼M)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">Brr2 ATPase</th><th class="colsep0 rowsep0" align="center">eIF4A1 ATPase</th><th class="colsep0 rowsep0" align="center">eIF4A3 ATPase</th><th class="colsep0 rowsep0" align="center">DHX29 ATPase</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">5.3 (2.9â9.4)</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">NT<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">5.3 (4.0â7.1)</td><td class="colsep0 rowsep0" align="left">57 (48â69)</td><td class="colsep0 rowsep0" align="left">32 (24â41)</td><td class="colsep0 rowsep0" align="left">58 (41â80)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last"><i>n</i> = 2 or 4, 95% confidence intervals shown in parentheses.</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">NT = not tested.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Structure-Based Drug Design of 4,6-Dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione Derivatives</h3><div class="NLM_p">With insights from crystallographic data of <b>3</b>, a structurally simplified a ring-opened derivative <b>6</b> was designed and synthesized (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). As we expected, compound <b>6</b> was equipotent to <b>5</b> and showed submicromolar Brr2 inhibitory activity. The cocrystal structure of <b>6</b> with Brr2 revealed that the dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione ring, the benzyl moiety on the pyridone ring, and the phenyl group substituted on the pyrimidinone ring are well overlapped with those in the cocrystal structure of <b>3</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). In contrast, the distal phenyl ring of <b>6</b> oriented perpendicularly to the 6-fluoropyridin-2-yl group of <b>3</b>. Considering the conformationally constrained structure of <b>3</b>, it seemed that the phenyl group of <b>6</b> can reside in a more favorable position.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of Compounds <b>6</b>â<b>10</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0010.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0011.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last"><i>n</i> = 2, 95% confidence intervals shown in parentheses.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">Parallel artificial membrane permeability assay.</p></div><div class="footnote" id="t4fn3"><sup>Table c</sup><p class="last">Distribution coefficient.</p></div></div><div></div></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray cocrystal structure of inhibitors bound to Brr2. (A) Overlay of X-ray crystal structure of <b>3</b> in Brr2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ">5URJ</a>, depicted in green) and <b>6</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URK">5URK</a>, depicted in yellow). (B) Polar amino acid residues around benzyl group of <b>6</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To improve the activity and solubility, further modifications of <b>6</b> were examined. Compounds with a polar functional group were designed to make an additional hydrogen bonding with an amino acid residue and reduce the hydrophobicity. As suitable amino acids (Thr1197, Ser1196, and Lys1716) are located around the 3-position of the distal phenyl ring in compound <b>6</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B), the phenyl group was replaced by a hydrogen-bond-accepting heterocyclic group, such as pyridyl (<b>7</b> and <b>8</b>) and thiazolyl (<b>9</b>) groups as shown in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>. Compounds <b>8</b> and <b>9</b> bearing a nitrogen atom oriented toward the similar direction (meta-direction) showed enhanced activity, whereas 2-pyridyl derivative <b>7</b> did not show increased activity. As shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, the docking model of compound <b>9</b> suggested the formation of a hydrogen bond between the nitrogen atom of the thiazole ring and Thr1197,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> which should be geometrically inaccessible by 2-pyridiyl derivative <b>7</b>. Compounds <b>8</b> and <b>9</b> not only exhibited the most potent activity among this chemical series but showed better physicochemical properties (logÂ <i>D</i> value, solubility, and permeability). As can be expected by the cocrystal structure of compound <b>6</b>, <i>o</i>-substituted derivative <b>10</b> showed deteriorated activity. On the basis of the structural similarity of <b>10</b> to the potent compounds <b>6</b>, <b>8</b>, and <b>9</b>, we adopted <b>10</b> as a negative control compound.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking model of compound <b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Characterization of Compound <b>9</b></h3><div class="NLM_p">In accordance with the cocrystal structural data of a near neighbor analogue, optimized compound <b>9</b> demonstrated noncompetitive Brr2 inhibition against either ATP or RNA (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Figure S3</a>). Selectivity study showed that compound <b>9</b> did not inhibit single-cassette RNA helicases such as DEAD box-family RNA helicase (eIF4A1 and eIF4A3) and DEAH box-family RNA helicase (DHX29), as shown in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>. In addition, compound <b>8</b> also did not affect other helicases, which indicates the high selectivity of this chemical series but still remains to be seen if they are Brr2-specific. We next investigated the effects of <b>9</b> on in vitro ATP-dependent RNA unwinding activity of Brr2 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). Its activity was measured using fluorescent-labeled RNA with the single- or double-stranded form separated by electrophoresis. We set the assay conditions within the initial velocity phase, which allowed us to estimate the accurate IC<sub>50</sub> values (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Figure S4</a>). Compound <b>9</b> inhibited helicase activity in dose-dependent manner with an IC<sub>50</sub> value of 1.3 Î¼M. On the contrary, its negative control, compound <b>10</b> did not inhibit the activity even at 100 Î¼M, demonstrating a positive correlation between ATPase- and helicase-activity. Thus, compound <b>9</b> could be an excellent starting point leading to molecular probes for Brr2. Furthermore, since there are several amino acids (e.g., Tyr 1682) not directly involved in the ligandâprotein interactions, further enhancement of inhibitory activity could be possible by forming additional hydrogen bonding with these residues.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Selectivity of <b>8</b> and <b>9</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub>Â (Î¼M)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">Brr2 ATPase</th><th class="colsep0 rowsep0" align="center">eIF4A1 ATPase</th><th class="colsep0 rowsep0" align="center">eIF4A3 ATPase</th><th class="colsep0 rowsep0" align="center">DHX29 ATPase</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">0.18 (0.15â0.22)</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">NT<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">0.079 (0.069â0.091)</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last"><i>n</i> = 2 or 4, 95% confidence intervals shown in parentheses.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">NT = not tested.</p></div></div></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. RNA helicase activity of Brr2 monitored using fluorescent-labeled RNA. Single- or double-stranded RNAs were separated by electrophoresis and detected as fluorescent signals. Dose-dependent inhibition as indicated concentration of Brr2 inhibitor <b>9</b> is represented (lanes 4â8). Negative control <b>10</b> was also evaluated at 100 Î¼M (lane 9). The enzymatic reaction without compounds was termed as 100% of control (lane 3), and that without enzyme was determined as 0% control (lane 10). Double- and single-stranded RNAs and total reactions without ATP are lanes 1, 2, and 11, respectively. The intensity of each band was measured, and inhibition rate was calculated. The IC<sub>50</sub> value against helicase activity was estimated by plotting of each inhibition rate, which is described in the right graph.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemistry</h3><div class="NLM_p">The synthesis of compounds <b>2</b>â<b>5</b> is illustrated in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Reported intermediates <b>14a</b><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and <b>14b</b><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> were subjected to a nucleophilic azidation reaction, followed by hydrolysis of the ester group to afford azido acid <b>15a</b>,<b>b</b>. The carboxyl group in <b>15a</b>,<b>b</b> was converted to Weinreb amide, and subsequent reduction using LiAlH<sub>4</sub> provided amino aldehyde <b>16a</b>,<b>b</b>. Reductive coupling reaction of <b>16a</b>,<b>b</b> and <b>17a</b>,<b>b</b>,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> followed by treatment with 1,1â²-carbonyldiimidazole (CDI), furnished compounds <b>2</b>, <b>3</b>, and <b>5</b>. Finally, methylation of the nitrogen atom of cyclic urea in <b>3</b> gave compound <b>4</b>.</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>2</b>â<b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) NaN<sub>3</sub>, DMF, 55 Â°Câ60 Â°C; (ii) LiOHÂ·H<sub>2</sub>O, EtOH (or EtOH and THF), H<sub>2</sub>O, rt, 71%â95% for 2 steps; (b) (i) MeNH(OMe)Â·HCl, T<sub>3</sub>P, DIPEA, DMF, 0 Â°C to rt; (ii) LiAlH<sub>4</sub>, THF, 0 Â°C to rt, 24% for 2 steps; (c) (i) MeNH(OMe)Â·HCl, EDCÂ·HCl, HOBt, DIPEA, DMF, rt; (ii) LiAlH<sub>4</sub>, THF, 0 Â°C, 58% for 2 steps; (d) (i) <b>17a</b> or <b>17b</b>, Ti(<i>i</i>-PrO)<sub>4</sub>, THF, 50 Â°C, 16 h; (ii) NaBH(OAc)<sub>3</sub>, 50 Â°C; (iii) CDI, DBU, THF, rt, 6%â27% for 3 steps; (e) NaH, MeI, DMF, 0 Â°C to rt, 73%.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>6</b>â<b>10</b> is depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Intermediate <b>16b</b> was coupled with commercially available amine <b>18a</b> and <b>18b</b>, followed by treatment with CDI to afford compounds <b>6</b> and <b>19</b>. Removal of <i>p</i>-methoxybenzyl group in <b>19</b> using TFA and subsequent alkylation gave compounds <b>7</b>, <b>8</b>, and <b>9</b>. Reductive amination of <b>16b</b> with <b>18c</b> using tin chloride and polymethylhydrosiloxane (PMHS) followed by treatment with CDI gave compounds <b>10</b>.</div><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>6</b>â<b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) <b>18a</b> or <b>18b</b>, Ti(<i>i</i>-PrO)<sub>4</sub>, THF, 50 Â°C, 16 h; (ii) NaBH(OAc)<sub>3</sub>, 50 Â°C; (iii) CDI, DBU, THF, rt, 34%â67% for 3 steps; (b) TFA, anisole, rt, quant; (c) <b>21a</b> or <b>21b</b>, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 13%â36%; (d) <b>21c</b>, K<sub>2</sub>CO<sub>3</sub>, KI, NMP, rt, 27%; (d) (i) <b>18c</b>, PMHS, SnCl<sub>2</sub>Â·MeOH, rt, 31%; (ii) CDI, DBU, THF, rt, 81%.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44436" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44436" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">By using RNA-dependent ATPase activity assays, crystal structure analysis, and SBDD, we successfully discovered a Brr2 specific inhibitor <b>9</b> with optimal helicase inhibitory activity. Our study demonstrated that the newly identified allosteric site in a proteinâprotein interface of Brr2 was a druggable pocket that led to highly selective and potent small-molecule inhibitors of Brr2. Since compound <b>9</b> is endowed with not only potency and selectivity but also excellent physicochemical properties, it could be a promising starting point leading to molecular probes for elucidating biological function or therapeutic relevance of Brr2. Moreover, we recently reported another successful example using ATPase assay for a helicase inhibitor project;<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> our studies demonstrate the effectiveness and robustness of ATPase activity-based screening for exploring helicase inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45378" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45378" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Chemistry Procedure</h3><div class="NLM_p">Proton nuclear magnetic resonance (<sup>1</sup>H NMR) spectra were recorded on Bruker AVANCE-300 (300 MHz), Bruker AVANCE-400 (400 MHz), and Bruker AVANCE-600 (600 MHz), and carbon nuclear magnetic resonance (<sup>13</sup>C NMR) spectra were recorded on Bruker AVANCE-300 (75 MHz), Bruker AVANCE-400 (101 MHz), and Bruker AVANCE-600 (151 MHz) in CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub> solution. Chemical shifts are given in parts per million (ppm) with tetramethylsilane as an internal standard. Abbreviations are used as follows: s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, dd = doublet of doublets, dt = doublet of triplets, brs = broad singlet. Coupling constants (<i>J</i> values) are given in hertz (Hz). Low-resolution mass spectra (MS) were acquired using an Agilent LC/MS system (Agilent1200SL/Agilent6130MS, Agilent1200SL/Agilent1956MS, or Agilent1200SL/Agilent6110MS) or Shimadzu UFLC/MS (Prominence UFLC high pressure gradient system/LCMS-2020) operating in an electron spray ionization mode (ESI+). The column used was an L-column 2 ODS (3.0 mm Ã 50 mm i.d., 3 Î¼m, CERI) with a temperature of 40 Â°C and a flow rate of 1.2 or 1.5 mL/min. Condition 1: mobile phases A and B under an acidic condition were 0.05% TFA in water and 0.05% TFA in MeCN, respectively. The ratio of mobile phase B was increased linearly from 5% to 90% over 0.9 min, 90% over the next 1.1 min. Condition 2: mobile phases A and B under a neutral condition were a mixture of 5 mmol/L AcONH<sub>4</sub> and MeCN (9/1, v/v) and a mixture of 5 mmol/L AcONH<sub>4</sub> and MeCN (1/9, v/v), respectively. The ratio of mobile phase B was increased linearly from 5% to 90% over 0.9 min, 90% over the next 1.1 min. The purities of all synthesized compounds tested in biological systems were assessed as being >95% using elemental analysis. Elemental analyses were carried out by Sumika Chemical Analysis Service or Toray Research Center and were within 0.4% of the theoretical values.</div><div class="NLM_p">Reaction progress was determined by thin layer chromatography (TLC) analysis on Merck Kieselgel 60 F254 plates or Fuji Silysia NH plates. Chromatographic purification was carried out on silica gel columns (Merck Kieselgel 60, 70â230 mesh or 230â400 mesh, Merck; Chromatorex NH-DM 1020, 100â200 mesh, Fuji Silysia Chemical; inject column and Universal column, YAMAZEN, <a href="http://yamazenusa.com/products/columns/" class="extLink">http://yamazenusa.com/products/columns/</a>; or Purif-Pack Si or NH, Shoko Scientific, <a href="http://shoko-sc.co.jp/english2/" class="extLink">http://shoko-sc.co.jp/english2/</a>). Preparative TLC was carried out on Merck Kieselgel 60 PLC plates. Preparative HPLC was acquired using a Gilson preparative HPLC system (conditions 1 and 2) or MassLynx UV prep system (condition 3) with UV detector (220 and 254 nm). Condition 1: Mobile phases A and B under a basic condition were 0.05% aqueous ammonia solution and MeCN, respectively. Condition 2: Mobile phases A and B under an acidic condition were 0.225% formic acid in water and MeCN, respectively. Condition 3: Mobile phases A and B under an acidic condition were 0.1% TFA in water and 0.1% TFA in MeCN, respectively. The ratio of mobile phase B was increased linearly between 7 and 12 min. The column used was a Gemini C18 (25 mm Ã 150 mm i.d., 10 Î¼m or 150 mm Ã 25 mm i.d., 5 Î¼m, Phenomenex) for condition 1, a Synergi C18 ([30 mm Ã 150 mm i.d., 4 Î¼m] or [25 Ã 150 mm i.d., 10 Î¼m], Phenomenex) for condition 2, an L-column 2 ODS (20 mm Ã 150 mm i.d., 5 Î¼m, CERI) for condition 3, and a flow rate of 25 mL/min (conditions 1 and 2) or 20 mL/min (condition 3). All commercially available solvents and reagents were used without further purification. Yields were not optimized.</div><div class="NLM_p last">Compound <b>1</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> was used from the in-house library (>95% purity was confirmed by element analysis). Compounds <b>11</b>â<b>13</b> were purchased from ChemDiv.</div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> (Â±)-3-(2-(3-Fluoropyridin-2-yl)-6-methyl-3,4-dihydro-2<i>H</i>-chromen-7-yl)-6-methyl-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>2</b>)</h3><div class="NLM_p">To a mixture of <b>16a</b> (120 mg, 0.789 mmol) and <b>17a</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (224 mg, 0.868 mmol) in THF (10 mL) was added Ti(<i>i</i>-PrO)<sub>4</sub> (0.47 mL, 1.6 mmol) at room temperature under N<sub>2</sub>. The reaction mixture was heated to 50 Â°C and stirred for 16 h. To the reaction mixture was added NaBH(OAc)<sub>3</sub> (669 mg, 3.15 mmol), and the reaction mixture was stirred at 50 Â°C for 48 h. The reaction mixture was quenched with saturated aqueous NaHCO<sub>3</sub>, and the mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10/1). The organic layer was separated, washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative TLC on silica gel (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10/1) to afford (Â±)-4-amino-5-(((2-(3-fluoropyridin-2-yl)-6-methyl-3,4-dihydro-2<i>H</i>-chromen-7-yl)amino)methyl)-1-methylpyridin-2(1<i>H</i>)-one (70 mg, 23%) as a yellow solid. MS <i>m</i>/<i>z</i> 395 (M + H)<sup>+</sup>.</div><div class="NLM_p last">A mixture of (Â±)-4-amino-5-(((2-(3-fluoropyridin-2-yl)-6-methyl-3,4-dihydro-2<i>H</i>-chromen-7-yl)amino)methyl)-1-methylpyridin-2(1<i>H</i>)-one (70 mg, 0.18 mmol), CDI (95.0 mg, 0.586 mmol), and DBU (88 Î¼L, 0.59 mmol) in THF (10 mL) was stirred at room temperature for 16 h under N<sub>2</sub>. Then the reaction mixture was allowed to warm to room temperature and stirred for 20 h. To the reaction mixture was added CDI (86.3 mg, 0.532 mmol) and the reaction mixture was stirred at room temperature for 20 h. The reaction mixture was diluted with water (20 mL), and the mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10/1). The organic layer was separated, washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative HPLC (condition 1, 0.05% aqueous ammonia solution/MeCN = 70/30 to 51/49) to afford <b>2</b> (44 mg, 59%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 2.04 (s, 3H), 2.13â2.24 (m, 1H), 2.32â2.36 (m, 1H), 2.76â2.86 (m, 1H), 2.88â3.02 (m, 1H), 3.32 (s, 3H), 4.23 (dd, <i>J</i> = 13.6, 4.8 Hz, 1H), 4.60 (d, <i>J</i> = 13.6 Hz, 1H), 5.42 (d, <i>J</i> = 10.0 Hz, 1H), 5.68 (s, 1H), 6.76 (s, 1H), 7.02 (s, 1H), 7.48â7.62 (m, 2H), 7.76â7.84 (m, 1H), 8.48 (d, <i>J</i> = 4.4 Hz, 1H), 9.82 (brs, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 90 Â°C) Î´ 15.9, 23.2, 24.8, 35.4, 47.3, 73.0 (d, <i>J</i><sub>CâF</sub> = 1.8 Hz, 1C), 95.7, 100.7, 114.4, 120.9, 123.5 (d, <i>J</i><sub>CâF</sub> = 19.1 Hz, 1C), 124.9 (d, <i>J</i><sub>CâF</sub> = 4.4 Hz, 1C) 126.5, 130.6, 134.8, 139.4, 144.7 (d, <i>J</i><sub>CâF</sub> = 5.1 Hz, 1C), 146.2 (d, <i>J</i><sub>CâF</sub> = 12.1 Hz, 1C), 147.8, 151.0, 152.6, 156.9 (d, <i>J</i><sub>CâF</sub> = 258.6 Hz, 1C), 161.5. MS <i>m</i>/<i>z</i> 421 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>23</sub>H<sub>21</sub>FN<sub>4</sub>O<sub>3</sub>Â·0.5H<sub>2</sub>O: C, 64.33; H, 5.16; N, 13.05. Found: C, 64.49; H, 5.18; N, 13.05.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> (Â±)-6-Benzyl-3-(2-(3-fluoropyridin-2-yl)-6-methyl-3,4-dihydro-2<i>H</i>-chromen-7-yl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>3</b>)</h3><div class="NLM_p">To a mixture of 4-amino-1-benzyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde <b>16b</b> (30 mg, 0.13 mmol) and <b>17a</b> (34.0 mg, 0.13 mmol) in THF (3 mL) was added Ti(<i>i</i>-PrO)<sub>4</sub> (59 Î¼L, 0.20 mmol) at room temperature under N<sub>2</sub>. The reaction mixture was heated to 50 Â°C and stirred for 16 h. To the reaction mixture was added NaBH(OAc)<sub>3</sub> (111 mg, 0.52 mmol) and the reaction mixture was stirred at 50 Â°C for 18 h. The reaction mixture was concentrated, and the residue was purified by preparative TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 20/1) to afford (Â±)-4-amino-1-benzyl-5-(((2-(3-fluoropyridin-2-yl)-6-methyl-3,4-dihydro-2<i>H</i>-chromen-7-yl)amino)methyl)pyridin-2(1<i>H</i>)-one (10 mg, 16%) as a yellow solid. MS <i>m</i>/<i>z</i> 471 (M + H)<sup>+</sup>.</div><div class="NLM_p last">A mixture of (Â±)-4-amino-1-benzyl-5-(((2-(3-fluoropyridin-2-yl)-6-methyl-3,4-dihydro-2<i>H</i>-chromen-7-yl)amino)methyl)pyridin-2(1<i>H</i>)-one (30 mg, 0.064 mmol), CDI (52 mg, 0.32 mmol), and DBU (48 Î¼L, 0.32 mmol) in THF (3 mL) was stirred at room temperature for 18 h. The reaction mixture was concentrated. The residue was purified by preparative HPLC (condition 2, 0.225% aqueous formic acid solution/MeCN = 55/45 to 34/66) to afford <b>3</b> (12 mg, 38%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 2.03 (s, 3H), 2.11â2.22 (m, 1H), 2.27â2.34 (m, 1H), 2.75â2.85 (m, 1H), 2.87â3.04 (m, 1H), 4.20â4.31 (m, 1H), 4.53â4.65 (m, 1H), 4.91â5.09 (m, 2H), 5.36â5.46 (m, 1H), 5.71 (s, 1H), 6.75 (s, 1H), 7.00 (s, 1H), 7.23â7.29 (m, 3H), 7.30â7.36 (m, 2H), 7.48â7.56 (m, 1H), 7.65 (s, 1H), 7.74â7.85 (m, 1H), 8.47 (d, <i>J</i> = 4.4 Hz, 1H), 9.87 (s, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 90 Â°C) Î´ 15.9, 23.2, 24.7, 47.2, 50.0, 72.9 (d, <i>J</i><sub>CâF</sub> = 1.8 Hz, 1C), 95.9, 101.3, 114.3, 120.8, 123.4 (d, <i>J</i><sub>CâF</sub> = 19.4 Hz, 1C), 124.8 (d, <i>J</i><sub>CâF</sub> = 4.4 Hz, 1C), 126.5, 126.8, 127.1(2C), 128.0 (2C), 130.5, 134.0, 137.3, 139.2, 144.6 (d, <i>J</i><sub>CâF</sub> = 5.1 Hz, 1C), 146.1 (d, <i>J</i><sub>CâF</sub> = 12.5 Hz, 1C), 147.8, 150.8, 152.5, 156.8 (d, <i>J</i><sub>CâF</sub> = 258.6 Hz, 1C), 161.1. MS <i>m</i>/<i>z</i> 497 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>29</sub>H<sub>25</sub>FN<sub>4</sub>O<sub>3</sub>Â·2.5H<sub>2</sub>O: C, 64.32; H, 5.58; N, 10.35. Found: C, 64.34; H, 5.23; N, 10.36.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> (Â±)-6-Benzyl-3-(2-(3-fluoropyridin-2-yl)-6-methyl-3,4-dihydro-2<i>H</i>-chromen-7-yl)-1-methyl-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>4</b>)</h3><div class="NLM_p last">To a solution of <b>3</b> (60 mg, 0.12 mmol) in DMF (2 mL) was added NaH (60%, 10 mg, 0.25 mmol) at 0 Â°C under N<sub>2</sub>. The reaction mixture was stirred at 0 Â°C for 0.5 h. To the reaction mixture was added iodomethane (34.3 mg, 0.24 mmol) at 0 Â°C. The reaction mixture was allowed to warm to room temperature and stirred for 2 h. The reaction mixture was quenched with water at 0 Â°C, and the mixture was extracted with EtOAc. The organic layer was separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative HPLC (condition 2, 0.225% aqueous formic acid solution/MeCN = 55/45 to 25/75) to afford <b>4</b> (45 mg, 73%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 2.00 (s, 3H), 2.11â2.22 (m, 1H), 2.28â2.35 (m, 1H), 2.73â2.86 (m, 1H), 2.88â3.03 (m, 1H), 3.15 (s, 3H), 4.15â4.33 (m, 1H), 4.50â4.65 (m, 1H), 4.90â5.17 (m, 2H), 5.41 (d, <i>J</i> = 10.0 Hz, 1H), 5.86 (s, 1H), 6.74 (s, 1H), 7.00 (s, 1H), 7.23â7.38 (m, 5H), 7.49â7.56 (m, 1H), 7.65 (s, 1H), 7.76â7.84 (m, 1H), 8.46 (d, <i>J</i> = 4.0 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 90 Â°C) Î´ 15.9, 23.1, 24.7, 29.6, 46.6, 50.0, 72.9 (d, <i>J</i><sub>CâF</sub> = 2.2 Hz, 1C), 97.0, 102.5, 114.0, 120.8, 123.4 (d, <i>J</i><sub>CâF</sub> = 19.1 Hz, 1C), 124.9 (d, <i>J</i><sub>CâF</sub> = 4.0 Hz, 1C), 126.2, 126.9, 127.1 (2C), 128.0 (2C), 130.5, 133.0, 137.1, 139.8, 144.6 (d, <i>J</i><sub>CâF</sub> = 5.1 Hz, 1C), 146.1 (d, <i>J</i><sub>CâF</sub> = 12.1 Hz, 1C), 148.9, 151.4, 152.5, 156.8 (d, <i>J</i><sub>CâF</sub> = 258.6 Hz, 1C), 161.2. MS <i>m</i>/<i>z</i> 511 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>30</sub>H<sub>27</sub>FN<sub>4</sub>O<sub>3</sub>Â·1.8H<sub>2</sub>O: C, 63.36; H, 5.68; N, 10.32. Found: C, 63.49; H, 5.55; N, 10.44.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> (Â±)-6-Benzyl-3-(2-(3-fluoropyridin-2-yl)-3,4-dihydro-2<i>H</i>-chromen-7-yl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>5</b>)</h3><div class="NLM_p">To a mixture of <b>16b</b> (30 mg, 0.13 mmol) and <b>17b</b><a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (32 mg, 0.13 mmol) in THF (3 mL) was added Ti(<i>i</i>-PrO)<sub>4</sub> (59 Î¼L, 0.20 mmol) at room temperature under N<sub>2</sub>. The reaction mixture was heated to 50 Â°C and stirred for 16 h. To the reaction mixture was added NaBH(OAc)<sub>3</sub> (111 mg, 0.52 mmol), and the reaction mixture was stirred at 50 Â°C for 18 h. The reaction mixture was concentrated, and the residue was purified by preparative TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 20/1) to afford (Â±)-4-amino-1-benzyl-5-(((2-(3-fluoropyridin-2-yl)-3,4-dihydro-2<i>H</i>-chromen-7-yl)amino)methyl)pyridin-2(1<i>H</i>)-one (30 mg, 50%) as a yellow solid. MS <i>m</i>/<i>z</i> 457 (M + H)<sup>+</sup>.</div><div class="NLM_p last">A mixture of (Â±)-4-amino-1-benzyl-5-(((2-(3-fluoropyridin-2-yl)-3,4-dihydro-2<i>H</i>-chromen-7-yl)amino)methyl)pyridin-2(1<i>H</i>)-one (140 mg, 0.307 mmol), CDI (249 mg, 1.53 mmol), and DBU (0.23 mL, 1.54 mmol) in THF (5 mL) was stirred at room temperature for 18 h. The reaction mixture was concentrated, and the residue was purified by preparative HPLC (condition 2, 0.225% aqueous formic acid solution/MeCN = 65/35 to 35/65) to afford <b>5</b> (80 mg, 54%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 2.13â2.23 (m, 1H), 2.27â2.36 (m, 1H), 2.75â2.87 (m, 1H), 2.90â3.04 (m, 1H), 4.55 (s, 2H), 5.00 (s, 2H), 5.44 (dd, <i>J</i> = 10.0, 2.0 Hz, 1H), 5.71 (s, 1H), 6.74â6.78 (m, 1H), 6.84 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.11 (d, <i>J</i> = 8.4 Hz, 1H), 7.23â7.37 (m, 5H), 7.48â7.56 (m, 1H), 7.67 (s, 1H), 7.76â7.85 (m, 1H), 8.43â8.51 (m, 1H), 9.94 (s, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 23.5, 24.8, 46.8, 50.3, 72.9 (d, <i>J</i><sub>CâF</sub> = 2.2 Hz, 1C) 96.1, 101.8, 113.0, 117.3, 119.5, 124.0 (d, <i>J</i><sub>CâF</sub> = 18.8 Hz, 1C) 125.5 (d, <i>J</i><sub>CâF</sub> = 4.4 Hz, 1C) 127.3, 127.4 (2C), 128.4 (2 C), 129.2, 134.6, 137.6, 140.9, 145.2 (d, <i>J</i><sub>CâF</sub> = 5.0 Hz, 1C) 146.1 (d, <i>J</i><sub>CâF</sub> = 12.2 Hz, 1C), 147.8, 151.8, 154.1, 157.1 (d, <i>J</i><sub>CâF</sub> = 258.2 Hz, 1C), 161.34. MS <i>m</i>/<i>z</i> 483 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>28</sub>H<sub>23</sub>FN<sub>4</sub>O<sub>3</sub>Â·2.6H<sub>2</sub>O: C, 63.53; H, 5.37; N, 10.58. Found: C, 63.85; H, 5.05; N, 10.64.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 6-Benzyl-3-(3-(benzyloxy)phenyl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>6</b>)</h3><div class="NLM_p">To a mixture of <b>18a</b> (30.0 mg, 0.151 mmol) and <b>16b</b> (34.4 mg, 0.151 mmol) in THF (3 mL) was added Ti(<i>i</i>-PrO)<sub>4</sub> (68 Î¼L, 0.23 mmol) at room temperature under N<sub>2</sub>. The reaction mixture was heated to 50 Â°C and stirred for 16 h. To the reaction mixture was added NaBH(OAc)<sub>3</sub> (128 mg, 0.603 mmol), and the reaction mixture was stirred at 50 Â°C for 18 h. The reaction mixture was concentrated. The residue was purified by preparative TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 20/1) to afford 4-amino-1-benzyl-5-(((3-(benzyloxy)phenyl)amino)methyl)pyridin-2(1<i>H</i>)-one (40.0 mg, 65%) as a yellow solid. MS <i>m</i>/<i>z</i> 421 (M + H)<sup>+</sup>.</div><div class="NLM_p last">A mixture of 4-amino-1-benzyl-5-(((3-(benzyloxy)phenyl)amino)methyl)pyridin-2(1<i>H</i>)-one (160 mg, 0.39 mmol), CDI (315 mg, 1.94 mmol), and DBU (0.290 mL, 1.94 mmol) in THF (10 mL) was stirred at room temperature for 18 h. The reaction mixture was concentrated, and the residue was purified by preparative HPLC (condition 2, 0.225% aqueous formic acid solution/MeCN = 50/50 to 29/71) to afford <b>6</b> (90 mg, 53%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 4.59 (s, 2H), 5.01 (s, 2H), 5.09 (s, 2H), 5.75 (s, 1H), 6.87â6.96 (m, 2H), 7.00â7.05 (m, 1H), 7.23â7.30 (m, 4H), 7.31â7.36 (m, 3H), 7.37â7.42 (m, 2H), 7.43â7.48 (m, 2H), 7.68 (s, 1H), 10.00 (s, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 46.8, 50.3, 69.3, 96.2, 101.8, 111.9, 112.1, 117.6, 127.3, 127.5 (2C), 127.6 (2C), 127.8, 128.3 (2C), 128.4 (2C), 129.2, 134.6, 136.8, 137.6, 143.2, 147.7, 151.8, 158.4, 161.4. MS <i>m</i>/<i>z</i> 438 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>23</sub>N<sub>3</sub>O<sub>3</sub>Â·1.9H<sub>2</sub>O: C, 68.75; H,5.73; N, 8.91. Found: C, 68.84; H, 5.60; N, 9.09.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 6-Benzyl-3-(3-(pyridin-2-ylmethoxy)phenyl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>7</b>)</h3><div class="NLM_p last">A mixture of <b>20</b> (200 mg, 0.576 mmol), 2-(bromomethyl)pyridine hydrobromide (<b>21a</b>, 160 mg, 0.638 mmol), and K<sub>2</sub>CO<sub>3</sub> (318 mg, 2.30 mmol) in DMF (5 mL) was stirred at room temperature for 15 h. The reaction mixture was diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10/1). The organic layer was separated, washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 12/1) and subsequently by preparative HPLC (condition 1, 0.05% aqueous ammonia solution/MeCN = 75/25 to 45/55) to afford <b>7</b> (91 mg, 36%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 4.59 (s, 2H), 5.01 (s, 2H), 5.17 (s, 2H), 5.75 (s, 1H), 6.90 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 6.94 (dd, <i>J</i> = 8.0, 1.2 Hz, 1H), 7.05â7.06 (m, 1H), 7.25â7.36 (m, 7H), 7.52 (d, <i>J</i> = 7.6 Hz, 1H), 7.68 (s, 1H), 7.82â7.86 (m, 1H), 8.55â8.60 (m, 1H), 10.00 (brs, 1H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 47.3, 50.8, 70.9, 96.8, 102.3, 112.4, 112.7, 118.3, 122.2, 123.5, 127.9, 128.0 (2C) 129.0 (2C) 129.8, 135.2, 137.5, 138.2, 143.7, 148.3, 149.6, 152.4, 157.0, 158.8, 161.9. MS <i>m</i>/<i>z</i> 439 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>26</sub>H<sub>22</sub>N<sub>4</sub>O<sub>3</sub>Â·1.0H<sub>2</sub>O: C, 68.41; H, 5.30; N, 12.27. Found: C, 68.50; H, 5.41; N, 12.31.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 6-Benzyl-3-(3-(pyridin-3-ylmethoxy)phenyl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>8</b>)</h3><div class="NLM_p last">A mixture of <b>20</b> (200 mg, 0.576 mmol), <b>21b</b> (160 mg, 0.638 mmol), and K<sub>2</sub>CO<sub>3</sub> (318 mg, 2.30 mmol) in DMF (2 mL) was stirred at room temperature for 14 h under N<sub>2</sub>. The reaction mixture was poured into water and extracted with CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10/1). The organic layer was separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 20/1) and subsequently by preparative HPLC (condition 1, 0.05% aqueous ammonia solution/MeCN = 68/32 to 40/60) to afford <b>8</b> (33 mg, 13%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 4.59 (s, 2H), 5.01 (s, 2H), 5.14 (s, 2H), 5.75 (s, 1H), 6.87â6.98 (m, 2H), 7.01â7.08 (m, 1H), 7.25â7.35 (m, 6H), 7.38â7.48 (m, 1H), 7.68 (s, 1H), 7.82â7.91 (m, 1H), 8.50â8.59 (m, 1H), 8.64â8.71 (m, 1H), 9.99 (brs, 1H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 47.4, 50.8, 67.5, 96.8, 102.3, 112.5, 112.7, 118.4, 124.1, 127.9, 128.0 (2C), 129.0 (2C), 129.8, 133.0, 135.2, 136.2, 138.2, 143.8, 148.3, 149.6, 149.7, 152.4, 158.7, 161.9. MS <i>m</i>/<i>z</i> 439 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>26</sub>H<sub>22</sub>N<sub>4</sub>O<sub>3</sub>Â·1.5H<sub>2</sub>O: C, 67.08; H, 5.41; N, 12.04. Found: C, 67.22; H, 5.30; N, 12.08.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 6-Benzyl-3-(3-(1,3-thiazol-5-ylmethoxy)phenyl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>9</b>)</h3><div class="NLM_p last">To a solution of <b>20</b> (82 mg, 0.24 mmol) and <b>21c</b> (60 mg, 0.35 mmol) in NMP (3 mL) were added K<sub>2</sub>CO<sub>3</sub> (147 mg, 1.06 mmol) and KI (5.9 mg, 0.036 mmol) at room temperature. The mixture was stirred at room temperature overnight, diluted with EtOAc and 5% aqueous citric acid, and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by silica gel column chromatography (EtOAc/MeOH = 100/0 to 88/12) and subsequently by preparative HPLC (condition 3, 0.1% TFA in water/0.1% TFA in MeCN = 80/20 to 0/100) to give <b>9</b> (28 mg, 27%) as a white solid. <sup>1</sup>H NMR Î´ (300 MHz, DMSO-<i>d</i><sub>6</sub>) 4.59 (s, 2H), 5.01 (s, 2H), 5.39 (s, 2H), 5.75 (s, 1H), 6.88â6.98 (m, 2H), 7.04 (s, 1H), 7.23â7.38 (m, 6H), 7.68 (s, 1H), 8.01 (s, 1H), 9.12 (s, 1H), 10.00 (s, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 46.8, 50.3, 61.7, 96.2, 101.8, 112.1, 112.3, 118.1, 127.3, 127.5 (2C), 128.4 (2C), 129.2, 133.8, 134.7, 137.6, 142.9, 143.2, 147.7, 151.8, 155.2, 157.6, 161.3. MS <i>m</i>/<i>z</i> 445 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>24</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>SÂ·0.5H<sub>2</sub>O: C, 63.56; H, 4.67; N, 12.35. Found: C, 63.84; H, 4.89; N, 12.52.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 6-Benzyl-3-(2-(benzyloxy)phenyl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>10</b>)</h3><div class="NLM_p">To a solution of 4-amino-1-benzyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde <b>16b</b> (100 mg, 0.438 mmol), 2-(benzyloxy)aniline (0.085 mL, 0.481 mmol), and SnCl<sub>2</sub>Â·2H<sub>2</sub>O (20 mg, 0.089 mmol) in MeOH (10 mL) was added polymethylhydrosiloxane (81 Î¼L) at room temperature. The mixture was stirred at room temperature for 3 d, diluted with EtOAc, passed through a short column (Na<sub>2</sub>SO<sub>4</sub> and silica gel, EtOAc), and concentrated. The residue was purified by silica gel column chromatography (EtOAc/MeOH = 100/0 to 88/12) to give 4-amino-1-benzyl-5-(((2-(benzyloxy)phenyl)amino)methyl)pyridin-2(1<i>H</i>)-one (55 mg, 31%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 4.03 (d, <i>J</i> = 6.7 Hz, 2H), 4.89 (s, 2H), 5.10 (s, 2H), 5.32 (s, 1H), 5.38 (t, <i>J</i> = 5.8 Hz, 1H), 6.11 (s, 2H), 6.45â6.54 (m, 1H),6.60 (s, 1H), 6.65â6.73 (m, 1H), 6.86 (d, <i>J</i> = 8.1 Hz, 1H), 7.13â7.20 (m, 2H), 7.22â7.40 (m, 6H), 7.47 (d, <i>J</i> = 9.9 Hz, 3H). MS <i>m</i>/<i>z</i> 412 (M + H)<sup>+</sup>.</div><div class="NLM_p last">To 4-amino-1-benzyl-5-(((2-(benzyloxy)phenyl)amino)methyl)pyridin-2(1<i>H</i>)-one (52 mg, 0.13 mmol) and CDI (102 mg, 0.63 mmol) in THF (10 mL) was added DBU (95 Î¼L, 0.63 mmol) at room temperature. The mixture was stirred at room temperature for 4 h, diluted with EtOAc, acidified with 5% citric acid, and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residual solid was suspended in EtOAc, collected by filtration, washed with EtOAc, and dried to give <b>10</b> (45 mg, 81%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 4.45 (2H, s), 5.00 (2H, s), 5.14 (2H, s), 5.73 (1H, s), 6.98 (1H, t, <i>J</i> = 7.5 Hz), 7.16 (1H, d, <i>J</i> = 8.3 Hz), 7.21â7.41 (12H, m),7.64 (1H, s), 9.91 (1H, s). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ ppm 47.2, 50.3, 69.4, 96.2, 101.7, 113.7, 120.8, 126.9 (2C)127.3, 127.4 (2C), 127.6, 128.2 (2C), 128.4 (2C), 128.5, 129.0, 130.7, 134.6, 136.9, 137.6, 148.2, 151.7, 153.9, 161.3. MS <i>m</i>/<i>z</i> 438 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>23</sub>N<sub>3</sub>O<sub>3</sub>Â·0.2H<sub>2</sub>O: C, 73.52; H, 5.35; N, 9.53. Found: C, 73.64; H, 5.28; N, 9.53.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 4-Azido-1-methyl-6-oxo-1,6-dihydropyridine-3-carboxylic Acid (<b>15a</b>)</h3><div class="NLM_p last">A mixture of <b>14a</b><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (3.25 g, 15.1 mmol) and sodium azide (1.96 g, 30.1 mmol) in DMF (32 mL) was heated to 55 Â°C and stirred for 5 d. The reaction mixture was diluted with saturated aqueous NaHCO<sub>3</sub> and saturated aqueous Na<sub>2</sub>CO<sub>3</sub>, and the mixture was extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated to afford ethyl 4-azido-1-methyl-6-oxo-1,6-dihydropyridine-3-carboxylate (6.60 g) as a crude product. The crude (6.60 g) and lithium hydroxide monohydrate (2.18 g, 52.0 mmol) in EtOH (30 mL) and water (30 mL) was stirred at room temperature for 4 h. The reaction mixture was acidified with conc hydrochloric acid to pH 2. The resultant precipitate solid was collected by filtration to afford <b>15a</b> (2.08 g, 71% for 2 steps) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 3.46 (s, 3H), 6.16 (s, 1H), 8.45 (s, 1H). 1H was not observed. MS <i>m</i>/<i>z</i> 195 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 4-Azido-1-benzyl-6-oxo-1,6-dihydropyridine-3-carboxylic Acid (<b>15b</b>)</h3><div class="NLM_p last">To a solution of <b>14b</b><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (10.5 g, 36.0 mmol) in DMF (100 mL) was added sodium azide (4.75 g, 73.1 mmol) at room temperature. The mixture was stirred at 60 Â°C for 2 d, cooled to room temperature, diluted with water, and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated to give ethyl 4-azido-1-benzyl-6-oxo-1,6-dihydropyridine-3-carboxylate (10.7 g) as a crude product. To a solution of the crude (10.7 g) in THF (50 mL) and EtOH (50 mL) was added 1 M NaOH (43 mL, 43.0 mmol) at room temperature. The mixture was stirred at room temperature for 1 h, and THF was removed by evaporation. The residual solution was cooled to 0 Â°C and acidified with 1 M HCl to pH 2. The resultant precipitated solid was collected by filtration, washed with water, and dried to give <b>15b</b> (9.20 g, 95% for 2 steps) as a yellow solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 5.18 (s, 2H), 6.24 (s, 1H), 7.25â7.41 (m, 5H), 8.58 (s, 1H), 12.92 (brs, 1H). MS <i>m</i>/<i>z</i> 243 (M â N<sub>2</sub> + H)<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 4-Amino-1-methyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (<b>16a</b>)</h3><div class="NLM_p last">To a mixture of <b>15a</b> (700 mg, 3.61 mmol), <i>N</i>,<i>O</i>-dimethylhydroxylamine hydrochloride (700 mg, 7.21 mmol) and DIPEA (3.77 mL, 21.6 mmol) in DMF (5 mL) was added T<sub>3</sub>P (50% in ethyl acetate, 4.28 mL, 7.21 mmol) dropwise at 0 Â°C. The reaction mixture was warmed to room temperature and stirred for 16 h. The reaction mixture was diluted with water, and the mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10/1). The organic layer was separated, washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford 4-azido-<i>N</i>-methoxy-<i>N</i>,1-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (800 mg) as a crude product. To a solution of the crude (800 mg) in THF (20 mL) was added LiAlH<sub>4</sub> (384 mg, 10.1 mmol) portionwise at 0 Â°C under N<sub>2</sub>. The reaction mixture was warmed to room temperature and stirred for 2 h. The reaction mixture was poured into saturated aqueous NH<sub>4</sub>Cl at 0 Â°C, and the mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10/1). The organic layer was separated, washed with brine (30 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative TLC on silica gel (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10/1) to afford <b>16a</b> (130 mg, 24% for 2 steps) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 3.38 (s, 3H), 5.29 (s, 1H), 7.05 (brs, 2H), 8.35 (s, 1H), 9.44 (s, 1H). MS <i>m</i>/<i>z</i> 153 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 4-Amino-1-benzyl-6-oxo-1,6-dihydropyridine-3-carbaldehyde (<b>16b</b>)</h3><div class="NLM_p">To a solution of <b>15b</b> (9.19 g, 34.0 mmol), <i>N</i>,<i>O</i>-dimethylhydroxylamine hydrochloride (4.98 g, 51.0 mmol), HOBt H<sub>2</sub>O (7.81 g, 51.0 mmol), and EDC HCl (9.78 g, 51.0 mmol) in DMF (120 mL) was added DIPEA (26.7 mL, 153 mmol) at room temperature. The mixture was stirred at room temperature for 3 days. The reaction mixture was concentrated to one-fourth volume, diluted with water, and extracted with EtOAc. The organic layer was separated, washed with 5% aqueous citric acid, water, 5% aqueous NaHCO<sub>3</sub>, water, and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by silica gel column chromatography (hexane/EtOAc = 50/50 to 0/100) to give 4-azido-1-benzyl-<i>N</i>-methoxy-<i>N</i>-methyl-6-oxo-1,6-dihydropyridine-3-carboxamide (9.00 g, 84%) as a yellow solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 3.19 (s, 3H), 3.52 (s, 3H), 5.09 (s, 2H), 6.29 (s, 1H), 7.23â7.41 (m, 5H), 8.10 (s, 1H). MS <i>m</i>/<i>z</i> 314 (M + H)<sup>+</sup>.</div><div class="NLM_p last">To a suspension of LiAlH<sub>4</sub> (807 mg, 21.3 mmol) in THF (30 mL) was added dropwise a solution of 4-azido-1-benzyl-<i>N</i>-methoxy-<i>N</i>-methyl-6-oxo-1,6-dihydropyridine-3-carboxamide (3.33 g, 10.6 mmol) in THF (30 mL) at 0 Â°C. The mixture was stirred at 0 Â°C for 0.5 h and quenched with EtOH in THF at 0 Â°C. To the mixture was added EtOAc and saturated aqueous potassium sodium (+)-tartarate at 0 Â°C. The mixture was filtered through a pad of Celite, and the filtrate was extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, passed through a short column (silica gel, EtOAc), and concentrated to give <b>16b</b> (1.64 g, 68%) as a yellow solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 5.06 (s, 2H), 5.32 (s, 1H), 7.10 (s, 2H), 7.24â7.38 (m, 5H), 8.49 (1H, s), 9.47 (1H, s). MS <i>m</i>/<i>z</i> 229 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 6-Benzyl-3-(3-((4-methoxybenzyl)oxy)phenyl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>19</b>)</h3><div class="NLM_p">To a mixture of <b>18b</b> (510 mg, 2.22 mmol) and <b>16b</b> (460 mg, 2.01 mmol) in THF (10 mL) was added Ti(<i>i</i>-PrO)<sub>4</sub> (0.89 mL, 3.0 mmol) at room temperature under N<sub>2</sub>. The reaction mixture was heated to 50 Â°C and stirred for 16 h. To the reaction mixture was added NaBH(OAc)<sub>3</sub> (1.70 g, 8.02 mmol), and the reaction mixture was stirred at 50 Â°C for 18 h. The reaction mixture was concentrated, and the residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 100/1 to 20/1) to afford 4-amino-1-benzyl-5-(((3-((4-methoxybenzyl)oxy)phenyl)amino)methyl)pyridin-2(1<i>H</i>)-one (700 mg, 79%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 3.75 (s, 3H), 3.89 (d, <i>J</i> = 5.6 Hz, 1H), 4.06â4.08 (m, 1H), 4.86â4.92 (m, 4H), 5.35â5.43 (m, 1H), 5.88â5.93 (m, 2H), 6.17â6.33 (m, 3H), 6.90â6.96 (m, 3H), 7.04â7.08 (m, 1H), 7.16â7.28 (m, 5H), 7.30â7.34 (m, 2H), 7.38â7.46 (m, 1H). MS <i>m</i>/<i>z</i> 442 (M + H)<sup>+</sup>.</div><div class="NLM_p last">To a 4-amino-1-benzyl-5-(((3-((4-methoxybenzyl)oxy)phenyl)amino)methyl)pyridin-2(1<i>H</i>)-one (4.64 g, 10.5 mmol) and CDI (8.52 g, 52.6 mmol) in THF (150 mL) was added DBU (7.92 mL, 52.6 mmol) at room temperature. The mixture was stirred at room temperature overnight. The reaction mixture was diluted with EtOAc, acidified with 5% citric acid and extracted with EtOAc. The organic layer was separated, washed with water, brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by silica gel column chromatography (EtOAc/MeOH = 100/0 to 88/12) to give <b>19</b> (4.20 g, 85%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 3.75 (s, 3H), 4.59 (s, 2H), 5.00 (brs, 2H), 5.01 (brs, 2H), 5.74 (s, 1H), 6.85â6.98 (m, 4H), 7.00 (s, 1H), 7.24â7.40 (m, 8H),7.68 (s, 1H), 9.97 (s, 1H). MS <i>m</i>/<i>z</i> 468 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 6-Benzyl-3-(3-hydroxyphenyl)-4,6-dihydropyrido[4,3-<i>d</i>]pyrimidine-2,7(1<i>H</i>,3<i>H</i>)-dione (<b>20</b>)</h3><div class="NLM_p last">To a mixture of <b>19</b> (300 mg, 0.642 mmol) and anisole (697 Î¼L, 6.42 mmol) was added TFA (3.0 mL, 39 mmol) at room temperature. The mixture was stirred room temperature for 1 h, diluted with EtOAc and basified with 5% aqueous NaHCO<sub>3</sub>, and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residual solid was suspended in hexane/EtOAc (4/1), collected by filtration, washed with hexane/EtOAc (4/1), and dried to give <b>20</b> (223 mg, quant) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 4.55 (s, 2H), 5.00 (s, 2H), 5.73 (s, 1H), 6.63 (d, <i>J</i> = 8.0 Hz, 1H), 6.70â6.77 (m, 2H), 7.15 (t, <i>J</i> = 8.3 Hz, 1H), 7.22â7.40 (m, 5H), 7.68 (s, 1H), 9.51 (s, 1H), 9.93 (s, 1H). MS <i>m</i>/<i>z</i> 348 (M + H)<sup>+</sup>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Evaluation of Solubility</h3><div class="NLM_p last">Small volumes of compound solution dissolved in DMSO were added to the aqueous buffer solution (pH 6.8). After incubation, precipitates were separated by filtration. The solubility was determined by UV absorbance of each filtrate.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Parallel Artificial Membrane Permeability Assay (PAMPA)</h3><div class="NLM_p last">The donor wells were filled with 200 Î¼L of PRISMA HT buffer (pH 7.4, pION Inc.) containing 10 Î¼mol L<sup>â1</sup> test compound. The filter on the bottom of each acceptor well was coated with 4 Î¼L of a GIT-0 lipid solution (pION Inc.) and filled with 200 Î¼L of acceptor sink buffer (pION Inc.). The acceptor filter plate was put on the donor plate and incubated for 3 h at room temperature. After the incubation, the amount of test compound in both the donor and acceptor wells was measured by LC/MS/MS.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Evaluation of logÂ <i>D</i></h3><div class="NLM_p last">The logÂ <i>D</i><sub>7.4</sub>, which is a partition coefficient between 1-octanol and aqueous buffer pH 7.4, of the compounds was measured using the chromatographic procedure whose condition was developed based on a published method.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Preparation of Enzymes for Biochemical Assays</h3><div class="NLM_p last">For ATPase assay, the human recombinant full-length Brr2 and full-length DHX29 were expressed in Sf-9 insect cells as fusion proteins with His and His-FLAG tag at the N-terminus, respectively, using the BaculoDirect C-term baculovirus expression system (Thermo Fisher Scientific Inc., Waltham, MA, USA). His-Brr2 and His-FLAG-DHX29 were purified by Ni-NTA superflow affinity column and Superdex 200 gel-filtration column. The human recombinant proteins, full-length eIF4A3, MLN51 (residues 137â283), full-length eIF4A1, full-length eIF4B, and eIF4G (residues 712â1451) were expressed in <i>Escherichia coli</i> BL21(DE3) as fusion proteins with 6Ã His- SUMO or His tag followed by a tobacco etch virus (TEV) protease cleavage site at the N terminus and purification by Ni-NTA superflow affinity column (QIAGEN, Venlo, The Netherlands) and Superdex 200 gel-filtration column (GE Healthcare, Chicago, IL, USA). The His-SUMO or His tags were cleaved with SUMO protease or TEV protease. Protein concentrations were determined using a BCA protein assay kit (Thermo Fisher Scientific Inc.) with bovine serum albumin as a standard.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> RNA-Dependent ATPase Assay</h3><div class="NLM_p last">The RNA-dependent ATPase assay was performed using the ADP-Glo assay system (Promega Corp., Madison, WI, USA) for primary screening and a phosphate sensor (Thermo Fisher Scientific Inc.) for secondary screening. Single-stranded RNA poly(U) was purchased from MP Biomedicals, LLC (Solon, OH, USA). The assay buffer comprised 20 mM Tris-HCl (pH 7.5), 2.5 mM MgCl<sub>2</sub>, 100 mM KCl, 1 mM dithiothreitol (DTT), and 0.01% (v/v) Tween 20. As for ADP-Glo assay, after the addition of 20 Î¼M ATP and 2.5 Î¼g mL<sup>â1</sup> poly(U) and test compounds, the ATPase reactions were started by the addition of 6.25 nM Brr2. They were incubated at room temperature for 30 min, and then the enzymatic reactions were terminated by ADP-Glo reagent. Following addition of ADP-Glo detection reagent, luminescent signals were measured using an EnVision 2102 multilabel plate reader (PerkinElmer, Waltham, MA, USA). For an assay using a phosphate sensor, 2 nM Brr2 was mixed with test compounds and substrates, and enzymatic reaction was terminated by addition of EDTA after 120 min. Following addition of 1 Î¼M phosphate sensor, fluorescent signals (excitation 430 nm/emission 450 nm) were detected using EnVision 2102 multilabel plate reader (PerkinElmer). We defined the luminescent or fluorescent signals of the reaction without enzyme as 100% inhibitory activity and those of the complete reaction mixture as 0% inhibitory activity. Curve fittings and calculations of IC<sub>50</sub> values were performed using the program XLfit version 5 (ID Business Solutions Ltd., Guildford, Surrey, U.K.). For evaluating selectivity, we also conducted ATPase assay for eIF4A1, eIF4A3, and DHX29. To enhance ATPase activity for eIF4A, the equivalent molar concentrations of MLN51 for 150 nM eIF4A3 or eIF4B and eIF4G for 100 nM eIF4A1 or eIF4A2 were added. Regarding the ATPase assays for DHX29, the optimal concentrations were 6.3 nM. Concentrations of ATP or RNA were set at the <i>K</i><sub>m</sub> value of each substrate for each enzyme as follows: 35 Î¼M ATP and 1.5 Î¼g mL<sup>â1</sup> poly(U) for eIF4A1 and eIF4A3; 30 Î¼M ATP and 1.8 Î¼g mL<sup>â1</sup> poly(U) for DHX29. Detection of luminescent signals or estimation of IC<sub>50</sub> values was performed as described above.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Thermal Shift Assay</h3><div class="NLM_p last">A mixture of 500 nM Brr2 and 500-diluted SYPRO Orange (Thermo Fisher Scientific Inc.) in assay buffer containing 20 mM Tris-HCl (pH 7.5), 2.5 mM MgCl<sub>2</sub>, 100 mM KCl, and 1 mM DTT was mixed with test compounds in 384-well plates. Fluorescent signals were measured by ABI 7900HT FAST real-time PCR system (Thermo Fisher Scientific Inc.). The temperature was held for 1 min per degree from 40 to 70 Â°C. The maximum fluorescent signal within detected ones at various temperatures is defined as 100% denaturation rate, and the minimum signal is defined as 0% denaturation rate. <i>T</i><sub>m</sub> values were calculated using the program Graphfit Prism version 5.03 (GraphPad Software, Inc., San Diego, CA, USA).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Substrate-Competition Assay</h3><div class="NLM_p last">ATP or RNA competition assays for Brr2 were performed using the ADP-Glo assay system. The enzymatic reactions were performed under identical conditions to those used for RNA-dependent ATPase assay except for substrate concentrations. ATP or RNA concentrations were set at the <i>K</i><sub>m</sub> values (25 Î¼M or 2.5 Î¼g mL<sup>â1</sup>, respectively), and 500, 200, or 2.5 Î¼M ATP and 50 Î¼g mL<sup>â1</sup> or 0.25 Î¼g mL<sup>â1</sup> poly(U) were used to examine ATP or RNA competitive inhibition. The enzymatic reactions were performed at room temperature for 30 min except for 45 min at 25 Î¼M ATP/0.25 Î¼g mL<sup>â1</sup> poly(U). Curve fittings and calculations of IC<sub>50</sub> values were performed, as previously described in the section of RNA-dependent ATPase assay.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> RNA Unwinding Assay</h3><div class="NLM_p last">RNA unwinding assays were performed in assay buffer containing 40 mM Tris-HCl (pH 7.5), 2 mM MgCl<sub>2</sub>, 30 mM NaCl, 1 mM DTT, and 0.01% (v/v) Tween 20. Duplex RNAs were generated with fluorescence-labeled 14-mer (5â²-[FITC]-UUCCCCUGCAUAAC-3â²) and 36-mer (5â²-GUUAUGCAGGGGAACCAACGCAUAUCAGUGAGGAUU-3â²) oligo RNAs purchased from Integrated DNA Technologies, Inc. (Coralville, IA, USA). The RNAs in assay buffer (30 nM) were annealed by heating at 95 Â°C for 5 min and cooling to 4 Â°C at 0.1 Â°C/s. Helicase activity was initiated with 30 nM duplex RNAs, 200 nM Brr2, and 20 Î¼M ATP at 37 Â°C for 10 min. Unlabeled 14-mer oligo RNA (1.5 Î¼M; 5â²-UUCCCCUGCAUAAC-3â²) was included to capture any free 36-mer oligo RNAs. Reactions were terminated with 40 mM EDTA, 3.2% (w/v) SDS, and 40% (v/v) glycerol. The reaction solutions were electrophoresed with 15% polyacrylamide gel (ATTO Corp.) at 10.5 mV for 45 min under native running buffer (25 mM Tris and 192 mM glycine). The fluorescence-labeled RNAs were visualized using a Typhoon 9400 (GE Healthcare). The intensity of each band was measured using ImageQuant TL (GE Healthcare). The intensity of single-stranded RNAs as products after enzymatic reaction was normalized with the total of the intensity of single- and double-stranded RNAs, and background was subtracted. We defined the fluorescent signals of the reaction without enzyme as 100% inhibitory activity and those of the complete reaction mixture as 0% inhibitory activity. Curve fittings and calculations of IC<sub>50</sub> values were performed using the program Graphfit Prism version 5.03 (GraphPad Software, Inc.).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Crystallography</h3><div class="NLM_p last">The generation of recombinant human Brr2 protein is detailed in the <a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Supporting Information</a>. Prior to crystallization, the purified Brr2 was diluted to 10 mg mL<sup>â1</sup> using delivery buffer (20 mM Tris, pH 7.6, 200 mM NaCl, 1 mM TCEP), and complexes were generated by addition of stock DMSO solutions of ligands to the protein solution at a final concentration of 1 mM (the final DMSO concentrations were 10% for Brr2-compound <b>12</b> and 1% for Brr2-compound <b>3</b> and Brr2-compound <b>6</b>). After addition of compounds, the mixture was placed on ice for 30 min to promote complex formation. Crystallization of Brr2 was performed using the hanging drop vapor diffusion method using a published protocol.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Briefly, the crystallization was performed with a reservoir solution composed of 0.1 M sodium citrate (pH 5.0â5.3) and 1.2â1.6 M sodium malonate (pH 7.0) (Hampton Research, Aliso Viejo, CA, USA). Crystal trials were set up and stored at room temperature, with crystals appearing overnight and reaching maximum size after 1â2 weeks. Crystals were harvested directly from the drops (with the high concentration of sodium malonate functioning as a cryoprotectant) and flash-cooled into ALS-style pucks submerged in liquid nitrogen. Diffraction data were collected at Advanced Light Source beamline 5.0.2 (compound <b>3</b>) and 5.0.3 (compound <b>6</b>) (Lawrence Berkeleyâs National Laboratory, Berkeley, CA) and at the Diamond Light Source beamline I04 (Didcot, U.K.) (compound <b>12</b>). Diffraction data were processed with XDS<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and Aimless.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The structures were solved by molecular replacement with Phaser<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> using the coordinates of human Brr2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F91">4F91</a>) as a search model. The graphics program COOT<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> was used for model building and refinement was performed with REFMAC5<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and PHENIX.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Phaser and REFMAC5 are distributed as part of CCP4.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Structure validation was performed using Molprobity.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> X-ray diffraction data collection and refinement statistics are reported in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Table S3</a>, and omit density and ligplots corresponding to the ligands are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">Figures S5âS7</a>. The coordinates of the three cocomplex structures were deposited in the PDB under the accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ">5URJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URK">5URK</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URM">5URM</a> corresponding to complexes with compounds <b>3</b>, <b>6</b>, and <b>12</b>, respectively.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i43"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00461">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62287" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62287" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00461" class="ext-link">10.1021/acs.jmedchem.7b00461</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Scheme to explore Brr2 inhibitors, SAR of compounds <b>11</b>â<b>13</b>, protein expression and purification for crystallization, comparison of the crystal structure of Brr2 with or without compound <b>3</b>, crystal structure of Brr2 in complex with compound <b>12</b>, substrate-competitive assay for compound <b>9</b>, time course of RNA helicase activity of Brr2, X-ray data collection and refinement statistics, omit electron density of Brr2 inhibitors and the ligand interactions, and helicase inhibitory activity of compound <b>12</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all the compounds reported in this study (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Coordinate file of docking model for compound <b>9</b> bound to Brr2 (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf">jm7b00461_si_001.pdf (1.36 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_002.csv">jm7b00461_si_002.csv (1.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_003.pdb">jm7b00461_si_003.pdb (2.14 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are the following: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ">5URJ</a> (compound <b>3</b>); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URK">5URK</a> (compound <b>6</b>); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URM">5URM</a> (compound <b>12</b>). Authors will release the atomatic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b00461" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61651" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61651" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Misa Iwatani-Yoshihara</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8083-3560" title="Orcid link">http://orcid.org/0000-0001-8083-3560</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a3cecad0c28dcad4c2d7c2cdcae3d7c2c8c6c7c28dc0ccce"><span class="__cf_email__" data-cfemail="a7caced4c689ced0c6d3c6c9cee7d3c6ccc2c3c689c4c8ca">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masahiro Ito</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5965-535X" title="Orcid link">http://orcid.org/0000-0001-5965-535X</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bad7dbc9dbd2d3c8d594d3ced5facedbd1dfdedb94d9d5d7"><span class="__cf_email__" data-cfemail="8fe2eefceee7e6fde0a1e6fbe0cffbeee4eaebeea1ece0e2">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael G. Klein</span> - <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takeshi Yamamoto</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kazuko Yonemori</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Toshio Tanaka</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masanori Miwa</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daisuke Morishita</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Satoshi Endo</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Tjhen</span> - <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling Qin</span> - <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Atsushi Nakanishi</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hironobu Maezaki</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tomohiro Kawamoto</span> - <span class="hlFld-Affiliation affiliation">â Biomolecular Research Laboratories, â¡Medicinal Chemistry Research Laboratories, Â§Oncology Drug Discovery
Unit, and â¥Shonan Incubation Laboratories, Pharmaceutical Research Division, Takeda Pharmaceutical Company Ltd., 26-1, Muraoka-Higashi 2-chome, Fujisawa, Kanagawa 251-8555, Japan</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Structural Biology, Takeda California
Inc., 10410 Science Center
Drive, San Diego, California 92121, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.I.-Y. and M.I. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d241e3619-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36475" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36475" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Hiromichi Kimura for starting this project, Shoichi Okubo for preparing plasmids, Takashi Ito for preparing proteins, Tsutomu Henta and Takashi Santou for conducting the high-throughput screening, and Motomi Oonishi for compound evaluation. We thank Yusuke Ohba, Jun Fujimoto, Masaki Ogino, and Shuji Kitamura for providing us with synthetic intermediates and for advice on synthetic protocols. We also thank Yasumi Kumagai and Motoo Iida for their assistance with NMR spectroscopic experiments, Haruyuki Nishida for supervising of structure determination, Kouichi Iwanaga and Izumi Nomura for conducting high-throughput synthesis, and Michiko Tawada, Ryosuke Tokunoh, Keiji Kamiyama, Akinori Toita, and Nobuo Cho for their cooperation in helping us execute this work.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i47" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i47"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i48" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i48"> Abbreviations Used</h2><tr><td class="NLM_term">Brr2</td><td class="NLM_def"><p class="first last">bad response to refrigeration 2</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationship</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high throughput screening</p></td></tr><tr><td class="NLM_term">SBDD</td><td class="NLM_def"><p class="first last">structure based drug design</p></td></tr><tr><td class="NLM_term">DHX29</td><td class="NLM_def"><p class="first last">DEAH box protein 29</p></td></tr><tr><td class="NLM_term">eIF4A1</td><td class="NLM_def"><p class="first last">eukaryotic initiation factor 4A-I</p></td></tr><tr><td class="NLM_term">eIF4A3</td><td class="NLM_def"><p class="first last">eukaryotic initiation factor 4A-III</p></td></tr><tr><td class="NLM_term">SUMO</td><td class="NLM_def"><p class="first last">small ubiquitin-like modifier</p></td></tr><tr><td class="NLM_term">TEV</td><td class="NLM_def"><p class="first last">tobacco etch virus</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13477" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13477" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 30 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Wahl, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Will, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LÃ¼hrmann, R.</span><span> </span><span class="NLM_article-title">The spliceosome: design principles of a dynamic RNP machine</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">â</span> <span class="NLM_lpage">718</span><span class="refDoi">Â DOI: 10.1016/j.cell.2009.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.cell.2009.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=19239890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvFGksbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=701-718&author=M.+C.+Wahlauthor=C.+L.+Willauthor=R.+L%C3%BChrmann&title=The+spliceosome%3A+design+principles+of+a+dynamic+RNP+machine&doi=10.1016%2Fj.cell.2009.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The spliceosome: Design principles of a dynamic RNP machine</span></div><div class="casAuthors">Wahl, Markus C.; Will, Cindy L.; Luehrmann, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">701-718</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ribonucleoproteins (RNPs) mediate key cellular functions such as gene expression and its regulation.  Whereas most RNP enzymes are stable in compn. and harbor preformed active sites, the spliceosome, which removes noncoding introns from precursor mRNAs (pre-mRNAs), follows fundamentally different strategies.  In order to provide both accuracy to the recognition of reactive splice sites in the pre-mRNA and flexibility to the choice of splice sites during alternative splicing, the spliceosome exhibits exceptional compositional and structural dynamics that are exploited during substrate-dependent complex assembly, catalytic activation, and active site remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphwXkeaEfBWLVg90H21EOLACvtfcHk0ljP0G3MVF9fIA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvFGksbk%253D&md5=589aa1782ba069c55bff0b6bb7762b84</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DWahl%26aufirst%3DM.%2BC.%26aulast%3DWill%26aufirst%3DC.%2BL.%26aulast%3DL%25C3%25BChrmann%26aufirst%3DR.%26atitle%3DThe%2520spliceosome%253A%2520design%2520principles%2520of%2520a%2520dynamic%2520RNP%2520machine%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D701%26epage%3D718%26doi%3D10.1016%2Fj.cell.2009.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Staley, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, C.</span><span> </span><span class="NLM_article-title">Mechanical devices of the spliceosome: motors, clocks, springs, and things</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">â</span> <span class="NLM_lpage">326</span><span class="refDoi">Â DOI: 10.1016/S0092-8674(00)80925-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2FS0092-8674%2800%2980925-3" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1998&pages=315-326&author=J.+P.+Staleyauthor=C.+Guthrie&title=Mechanical+devices+of+the+spliceosome%3A+motors%2C+clocks%2C+springs%2C+and+things&doi=10.1016%2FS0092-8674%2800%2980925-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980925-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980925-3%26sid%3Dliteratum%253Aachs%26aulast%3DStaley%26aufirst%3DJ.%2BP.%26aulast%3DGuthrie%26aufirst%3DC.%26atitle%3DMechanical%2520devices%2520of%2520the%2520spliceosome%253A%2520motors%252C%2520clocks%252C%2520springs%252C%2520and%2520things%26jtitle%3DCell%26date%3D1998%26volume%3D92%26spage%3D315%26epage%3D326%26doi%3D10.1016%2FS0092-8674%2800%2980925-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Laggerbauer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Achsel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LÃ¼hrmann, R.</span><span> </span><span class="NLM_article-title">The human U5-200kD DEXH-box protein unwinds U4/U6 RNA duplices in vitro</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">4188</span><span class="NLM_x">â</span> <span class="NLM_lpage">4192</span><span class="refDoi">Â DOI: 10.1073/pnas.95.8.4188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1073%2Fpnas.95.8.4188" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=4188-4192&author=B.+Laggerbauerauthor=T.+Achselauthor=R.+L%C3%BChrmann&title=The+human+U5-200kD+DEXH-box+protein+unwinds+U4%2FU6+RNA+duplices+in+vitro&doi=10.1073%2Fpnas.95.8.4188"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.8.4188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.8.4188%26sid%3Dliteratum%253Aachs%26aulast%3DLaggerbauer%26aufirst%3DB.%26aulast%3DAchsel%26aufirst%3DT.%26aulast%3DL%25C3%25BChrmann%26aufirst%3DR.%26atitle%3DThe%2520human%2520U5-200kD%2520DEXH-box%2520protein%2520unwinds%2520U4%252FU6%2520RNA%2520duplices%2520in%2520vitro%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D4188%26epage%3D4192%26doi%3D10.1073%2Fpnas.95.8.4188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Raghunathan, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, C.</span><span> </span><span class="NLM_article-title">RNA unwinding in U4/U6 snRNPs requires ATP hydrolysis and the DEIH-box splicing factor Brr2</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">â</span> <span class="NLM_lpage">855</span><span class="refDoi">Â DOI: 10.1016/S0960-9822(07)00345-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2FS0960-9822%2807%2900345-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=9705931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADyaK1cXks1Ghtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1998&pages=847-855&author=P.+L.+Raghunathanauthor=C.+Guthrie&title=RNA+unwinding+in+U4%2FU6+snRNPs+requires+ATP+hydrolysis+and+the+DEIH-box+splicing+factor+Brr2&doi=10.1016%2FS0960-9822%2807%2900345-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">RNA unwinding in U4/U6 snRNPs requires ATP hydrolysis and the DEIH-box splicing factor Brr2</span></div><div class="casAuthors">Raghunathan, Pratima L.; Guthrie, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">847-853</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Ltd.</span>)
        </div><div class="casAbstract">The dynamic rearrangements of RNA structure which occur during pre-mRNA splicing are thought to be mediated by members of the DExD/H-box family of RNA-dependent ATPases.  Although three DExD/H-box splicing factors have recently been shown to unwind synthetic RNA duplexes in purified systems, in no case has the natural biol. substrate been identified.  A duplex RNA target of particular interest is the extensive base-pairing interaction between U4 and U6 small nuclear RNAs.  Because these helixes must be disrupted to activate the spliceosome for catalysis, this rearrangement is believed to be tightly regulated in vivo.  The authors have immunopurified Brr2, a DEIH-box ATPase, in a native complex contg. U1, U2, U5 and duplex U4/U6 small nuclear ribonucleoprotein particles (snRNPs).  Addn. of hydrolyzable ATP to this complex results in the disruption of U4/U6 base-pairing, and the release of free U4 and U6 snRNPs.  A mutation in the helicase-like domain of Brr2 (brr2-1) prevents these RNA rearrangements.  Notably, U4/U6 dissocn. and release occur in the absence of exogenously added pre-mRNA.  Disruption of U4/U6 base-pairing in native snRNPs requires ATP hydrolysis and Brr2.  This is the first assignment of a DExD/H-box splicing factor to a specific biol. unwinding event.  The unwinding function of Brr2 can be antagonized by the annealing activity of Prp24.  The authors propose the existence of a dynamic cycle, uncoupled from splicing, that interconverts free and base-paired U4/U6 snRNPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVAGASfwtpRLVg90H21EOLACvtfcHk0liZc0Gf5zgAkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXks1Ghtrg%253D&md5=e32b31baad4ee735032cf10a22bd8c34</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2807%2900345-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252807%252900345-4%26sid%3Dliteratum%253Aachs%26aulast%3DRaghunathan%26aufirst%3DP.%2BL.%26aulast%3DGuthrie%26aufirst%3DC.%26atitle%3DRNA%2520unwinding%2520in%2520U4%252FU6%2520snRNPs%2520requires%2520ATP%2520hydrolysis%2520and%2520the%2520DEIH-box%2520splicing%2520factor%2520Brr2%26jtitle%3DCurr.%2520Biol.%26date%3D1998%26volume%3D8%26spage%3D847%26epage%3D855%26doi%3D10.1016%2FS0960-9822%2807%2900345-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Kim, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, J. J.</span><span> </span><span class="NLM_article-title">The first ATPase domain of the yeast 246-kDa protein is required for in vivo unwinding of the U4/U6 duplex</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">959</span><span class="NLM_x">â</span> <span class="NLM_lpage">971</span><span class="refDoi">Â DOI: 10.1017/S135583829999012X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1017%2FS135583829999012X" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=959-971&author=D.+H.+Kimauthor=J.+J.+Rossi&title=The+first+ATPase+domain+of+the+yeast+246-kDa+protein+is+required+for+in+vivo+unwinding+of+the+U4%2FU6+duplex&doi=10.1017%2FS135583829999012X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1017%2FS135583829999012X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS135583829999012X%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%2BH.%26aulast%3DRossi%26aufirst%3DJ.%2BJ.%26atitle%3DThe%2520first%2520ATPase%2520domain%2520of%2520the%2520yeast%2520246-kDa%2520protein%2520is%2520required%2520for%2520in%2520vivo%2520unwinding%2520of%2520the%2520U4%252FU6%2520duplex%26jtitle%3DRNA%26date%3D1999%26volume%3D5%26spage%3D959%26epage%3D971%26doi%3D10.1017%2FS135583829999012X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Small, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leggett, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winans, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staley, J. P.</span><span> </span><span class="NLM_article-title">The EF-G-like GTPase Snu114p regulates spliceosome dynamics mediated by Brr2p, a DExD/H box ATPase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">â</span> <span class="NLM_lpage">399</span><span class="refDoi">Â DOI: 10.1016/j.molcel.2006.05.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.molcel.2006.05.043" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=389-399&author=E.+C.+Smallauthor=S.+R.+Leggettauthor=A.+A.+Winansauthor=J.+P.+Staley&title=The+EF-G-like+GTPase+Snu114p+regulates+spliceosome+dynamics+mediated+by+Brr2p%2C+a+DExD%2FH+box+ATPase&doi=10.1016%2Fj.molcel.2006.05.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.05.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.05.043%26sid%3Dliteratum%253Aachs%26aulast%3DSmall%26aufirst%3DE.%2BC.%26aulast%3DLeggett%26aufirst%3DS.%2BR.%26aulast%3DWinans%26aufirst%3DA.%2BA.%26aulast%3DStaley%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520EF-G-like%2520GTPase%2520Snu114p%2520regulates%2520spliceosome%2520dynamics%2520mediated%2520by%2520Brr2p%252C%2520a%2520DExD%252FH%2520box%2520ATPase%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D389%26epage%3D399%26doi%3D10.1016%2Fj.molcel.2006.05.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Santos, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jovin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pena, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luhrmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, M. C.</span><span> </span><span class="NLM_article-title">Structural basis for functional cooperation between tandem helicase cassettes in Brr2-mediated remodeling of the spliceosome</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">17418</span><span class="NLM_x">â</span> <span class="NLM_lpage">17423</span><span class="refDoi">Â DOI: 10.1073/pnas.1208098109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1073%2Fpnas.1208098109" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17418-17423&author=K.+F.+Santosauthor=S.+M.+Jovinauthor=G.+Weberauthor=V.+Penaauthor=R.+Luhrmannauthor=M.+C.+Wahl&title=Structural+basis+for+functional+cooperation+between+tandem+helicase+cassettes+in+Brr2-mediated+remodeling+of+the+spliceosome&doi=10.1073%2Fpnas.1208098109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1208098109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1208098109%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DK.%2BF.%26aulast%3DJovin%26aufirst%3DS.%2BM.%26aulast%3DWeber%26aufirst%3DG.%26aulast%3DPena%26aufirst%3DV.%26aulast%3DLuhrmann%26aufirst%3DR.%26aulast%3DWahl%26aufirst%3DM.%2BC.%26atitle%3DStructural%2520basis%2520for%2520functional%2520cooperation%2520between%2520tandem%2520helicase%2520cassettes%2520in%2520Brr2-mediated%2520remodeling%2520of%2520the%2520spliceosome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17418%26epage%3D17423%26doi%3D10.1073%2Fpnas.1208098109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Pena, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jovin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabrizio, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlowski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bujnicki, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luhrmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, M. C.</span><span> </span><span class="NLM_article-title">Common design principles in the spliceosomal RNA helicase Brr2 and in the Hel308 DNA helicase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">454</span><span class="NLM_x">â</span> <span class="NLM_lpage">466</span><span class="refDoi">Â DOI: 10.1016/j.molcel.2009.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.molcel.2009.08.006" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2009&pages=454-466&author=V.+Penaauthor=S.+M.+Jovinauthor=P.+Fabrizioauthor=J.+Orlowskiauthor=J.+M.+Bujnickiauthor=R.+Luhrmannauthor=M.+C.+Wahl&title=Common+design+principles+in+the+spliceosomal+RNA+helicase+Brr2+and+in+the+Hel308+DNA+helicase&doi=10.1016%2Fj.molcel.2009.08.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DPena%26aufirst%3DV.%26aulast%3DJovin%26aufirst%3DS.%2BM.%26aulast%3DFabrizio%26aufirst%3DP.%26aulast%3DOrlowski%26aufirst%3DJ.%26aulast%3DBujnicki%26aufirst%3DJ.%2BM.%26aulast%3DLuhrmann%26aufirst%3DR.%26aulast%3DWahl%26aufirst%3DM.%2BC.%26atitle%3DCommon%2520design%2520principles%2520in%2520the%2520spliceosomal%2520RNA%2520helicase%2520Brr2%2520and%2520in%2520the%2520Hel308%2520DNA%2520helicase%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D35%26spage%3D454%26epage%3D466%26doi%3D10.1016%2Fj.molcel.2009.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maeder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bud, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanks, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nix, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pleiss, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, R.</span><span> </span><span class="NLM_article-title">Structural evidence for consecutive Hel308-like modules in the spliceosomal ATPase Brr2</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">731</span><span class="NLM_x">â</span> <span class="NLM_lpage">739</span><span class="refDoi">Â DOI: 10.1038/nsmb.1625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1038%2Fnsmb.1625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=19525970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1ShsL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=731-739&author=L.+Zhangauthor=T.+Xuauthor=C.+Maederauthor=L.+O.+Budauthor=J.+Shanksauthor=J.+Nixauthor=C.+Guthrieauthor=J.+A.+Pleissauthor=R.+Zhao&title=Structural+evidence+for+consecutive+Hel308-like+modules+in+the+spliceosomal+ATPase+Brr2&doi=10.1038%2Fnsmb.1625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural evidence for consecutive Hel308-like modules in the spliceosomal ATPase Brr2</span></div><div class="casAuthors">Zhang, Lingdi; Xu, Tao; Maeder, Corina; Bud, Laura-Oana; Shanks, James; Nix, Jay; Guthrie, Christine; Pleiss, Jeffrey A.; Zhao, Rui</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">731-739</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Brr2 is a DExD/H-box helicase responsible for U4/U6 unwinding during spliceosomal activation.  Brr2 contains two helicase-like domains, each of which is followed by a Sec63 domain with unknown function.  We detd. the crystal structure of the second Sec63 domain, which unexpectedly resembles domains 4 and 5 of DNA helicase Hel308.  This, together with sequence similarities between Brr2's helicase-like domains and domains 1-3 of Hel308, led us to hypothesize that Brr2 contains two consecutive Hel308-like modules (Hel308-I and Hel308-II).  Our structural model and mutagenesis data suggest that Brr2 shares a similar helicase mechanism with Hel308.  We demonstrate that Hel308-II interacts with Prp8 and Snu114 in vitro and in vivo.  We further find that the C-terminal region of Prp8 (Prp8-CTR) facilitates the binding of the Brr2-Prp8-CTR complex to U4/U6.  Our results have important implications for the mechanism and regulation of Brr2's activity in splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojfFiQIh2fJbVg90H21EOLACvtfcHk0lg6pxvUcs2_kA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1ShsL0%253D&md5=f9a5ba148222e024af16edccea66f78e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1625%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DMaeder%26aufirst%3DC.%26aulast%3DBud%26aufirst%3DL.%2BO.%26aulast%3DShanks%26aufirst%3DJ.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DGuthrie%26aufirst%3DC.%26aulast%3DPleiss%26aufirst%3DJ.%2BA.%26aulast%3DZhao%26aufirst%3DR.%26atitle%3DStructural%2520evidence%2520for%2520consecutive%2520Hel308-like%2520modules%2520in%2520the%2520spliceosomal%2520ATPase%2520Brr2%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D731%26epage%3D739%26doi%3D10.1038%2Fnsmb.1625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">van Nues, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beggs, J. D.</span><span> </span><span class="NLM_article-title">Functional contacts with a range of splicing proteins suggest a central role for Brr2p in the dynamic control of the order of events in spliceosomes of Saccharomyces cerevisiae</span> <span class="citation_source-journal">Genetics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">1451</span><span class="NLM_x">â</span> <span class="NLM_lpage">1467</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2001&pages=1451-1467&author=R.+W.+van+Nuesauthor=J.+D.+Beggs&title=Functional+contacts+with+a+range+of+splicing+proteins+suggest+a+central+role+for+Brr2p+in+the+dynamic+control+of+the+order+of+events+in+spliceosomes+of+Saccharomyces+cerevisiae"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BNues%26aufirst%3DR.%2BW.%26aulast%3DBeggs%26aufirst%3DJ.%2BD.%26atitle%3DFunctional%2520contacts%2520with%2520a%2520range%2520of%2520splicing%2520proteins%2520suggest%2520a%2520central%2520role%2520for%2520Brr2p%2520in%2520the%2520dynamic%2520control%2520of%2520the%2520order%2520of%2520events%2520in%2520spliceosomes%2520of%2520Saccharomyces%2520cerevisiae%26jtitle%3DGenetics%26date%3D2001%26volume%3D157%26spage%3D1451%26epage%3D1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">CvaÄkovÃ¡, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MatÄjÅ¯, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">StanÄk, D.</span><span> </span><span class="NLM_article-title">Retinitis pigmentosa mutations of SNRNP200 enhance cryptic splice-site recognition</span> <span class="citation_source-journal">Hum. Mutat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">308</span><span class="NLM_x">â</span> <span class="NLM_lpage">317</span><span class="refDoi">Â DOI: 10.1002/humu.22481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1002%2Fhumu.22481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=24302620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC2cXislems70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=308-317&author=Z.+Cva%C4%8Dkov%C3%A1author=D.+Mat%C4%9Bj%C5%AFauthor=D.+Stan%C4%9Bk&title=Retinitis+pigmentosa+mutations+of+SNRNP200+enhance+cryptic+splice-site+recognition&doi=10.1002%2Fhumu.22481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Retinitis Pigmentosa Mutations of SNRNP200 Enhance Cryptic Splice-Site Recognition</span></div><div class="casAuthors">Cvackova, Zuzana; Mateju, Daniel; Stanek, David</div><div class="citationInfo"><span class="NLM_cas:title">Human Mutation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">308-317</span>CODEN:
                <span class="NLM_cas:coden">HUMUE3</span>;
        ISSN:<span class="NLM_cas:issn">1059-7794</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Mutations in SNRP200 gene cause autosomal-dominant retinal disorder retinitis pigmentosa (RP).  The protein product of SNRNP200 is BRR2, a DExD/H box RNA helicase crucial for pre-mRNA splicing.  In this study, we prepd. p.S1087L and p.R1090L mutations of human BRR2 using bacterial artificial chromosome recombineering and stably expressed them in human cell culture.  Mutations in BRR2 did not compromise snRNP assembly and both mutants were incorporated into the spliceosome just as the wild-type (wt) protein.  Surprisingly, cells expressing RP mutants exhibited increased splicing efficiency of the LDHA gene.  Next, we found that depletion of endogenous BRR2 enhanced usage of a Î²-globin cryptic splice site while splicing at the correct splice site was inhibited.  Proper splicing of optimal and cryptic splice sites was restored in cells expressing BRR2-wt but not in cells expressing RP mutants.  Taken together, our data suggest that BRR2 is an important factor in 5'-splice-site recognition and that the RP-linked mutations c.3260C>T (p.S1087L) and c.3269G>T (p.R1090L) affect this BRR2 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrm8j7wARgaOLVg90H21EOLACvtfcHk0lg6pxvUcs2_kA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXislems70%253D&md5=a14881b54c35dfae55511abf31a02550</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fhumu.22481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhumu.22481%26sid%3Dliteratum%253Aachs%26aulast%3DCva%25C4%258Dkov%25C3%25A1%26aufirst%3DZ.%26aulast%3DMat%25C4%259Bj%25C5%25AF%26aufirst%3DD.%26aulast%3DStan%25C4%259Bk%26aufirst%3DD.%26atitle%3DRetinitis%2520pigmentosa%2520mutations%2520of%2520SNRNP200%2520enhance%2520cryptic%2520splice-site%2520recognition%26jtitle%3DHum.%2520Mutat.%26date%3D2014%26volume%3D35%26spage%3D308%26epage%3D317%26doi%3D10.1002%2Fhumu.22481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Zhao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellur, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grassi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowne, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daiger, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staley, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, C.</span><span> </span><span class="NLM_article-title">Autosomal-dominant retinitis pigmentosa caused by a mutation in SNRNP200, a gene required for unwinding of U4/U6 snRNAs</span> <span class="citation_source-journal">Am. J. Hum. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">617</span><span class="NLM_x">â</span> <span class="NLM_lpage">627</span><span class="refDoi">Â DOI: 10.1016/j.ajhg.2009.09.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.ajhg.2009.09.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=19878916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFWruw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2009&pages=617-627&author=C.+Zhaoauthor=D.+L.+Bellurauthor=S.+Luauthor=F.+Zhaoauthor=M.+A.+Grassiauthor=S.+J.+Bowneauthor=L.+S.+Sullivanauthor=S.+P.+Daigerauthor=L.+J.+Chenauthor=C.+P.+Pangauthor=K.+Zhaoauthor=J.+P.+Staleyauthor=C.+Larsson&title=Autosomal-dominant+retinitis+pigmentosa+caused+by+a+mutation+in+SNRNP200%2C+a+gene+required+for+unwinding+of+U4%2FU6+snRNAs&doi=10.1016%2Fj.ajhg.2009.09.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Autosomal-dominant retinitis pigmentosa caused by a mutation in SNRNP200, a gene required for unwinding of U4/U6 snRNAs</span></div><div class="casAuthors">Zhao, Chen; Bellur, Deepti L.; Lu, Shasha; Zhao, Feng; Grassi, Michael A.; Bowne, Sara J.; Sullivan, Lori S.; Daiger, Stephen P.; Chen, Li Jia; Pang, Chi Pui; Zhao, Kanxing; Staley, Jonathan P.; Larsson, Catharina</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">617-627</span>CODEN:
                <span class="NLM_cas:coden">AJHGAG</span>;
        ISSN:<span class="NLM_cas:issn">0002-9297</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mutations in genes assocd. with the U4/U6-U5 small nuclear ribonucleoprotein (snRNP) complex of the spliceosome are implicated in autosomal-dominant retinitis pigmentosa (adRP), a group of progressive retinal degenerative disorders leading to visual impairment, loss of visual field, and even blindness.  We recently assigned a locus (RP33) for adRP to 2cen-q12.1, a region that harbors the SNRNP200 gene encoding hBrr2, another U4/U6-U5 snRNP component that is required for unwinding of U4/U6 snRNAs during spliceosome activation and for disassembly of the spliceosome.  Here, we report the identification of a missense mutation, c.3260C>T (p.S1087L), in exon 25 of the SNRNP200 gene in an RP33-linked family.  The c.3260C>T substitution showed complete cosegregation with the retinitis pigmentosa (RP) phenotype over four generations, but was absent in a panel of 400 controls.  The p.S1087L mutation and p.R1090L, another adRP-assocd. allele, reside in the "ratchet" helix of the first of two Sec63 domains implicated in the directionality and processivity of nucleic acid unwinding.  Indeed, marked defects in U4/U6 unwinding, but not U4/U6-U5 snRNP assembly, were obsd. in budding yeast for the analogous mutations (N1104L and R1107L) of the corresponding Brr2p residues.  The linkage of hBrr2 to adRP suggests that the mechanism of pathogenesis for splicing-factor-related RP may fundamentally derive from a defect in hBrr2-dependent RNA unwinding and a consequent defect in spliceosome activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX7wOyj5QzPrVg90H21EOLACvtfcHk0lg6pxvUcs2_kA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFWruw%253D%253D&md5=273111d5d7d4bc6fb087e31b9100c92a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ajhg.2009.09.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajhg.2009.09.020%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DBellur%26aufirst%3DD.%2BL.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DGrassi%26aufirst%3DM.%2BA.%26aulast%3DBowne%26aufirst%3DS.%2BJ.%26aulast%3DSullivan%26aufirst%3DL.%2BS.%26aulast%3DDaiger%26aufirst%3DS.%2BP.%26aulast%3DChen%26aufirst%3DL.%2BJ.%26aulast%3DPang%26aufirst%3DC.%2BP.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DStaley%26aufirst%3DJ.%2BP.%26aulast%3DLarsson%26aufirst%3DC.%26atitle%3DAutosomal-dominant%2520retinitis%2520pigmentosa%2520caused%2520by%2520a%2520mutation%2520in%2520SNRNP200%252C%2520a%2520gene%2520required%2520for%2520unwinding%2520of%2520U4%252FU6%2520snRNAs%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2009%26volume%3D85%26spage%3D617%26epage%3D627%26doi%3D10.1016%2Fj.ajhg.2009.09.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Ledoux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, C.</span><span> </span><span class="NLM_article-title">Retinitis Pigmentosa mutations in Bad Response to Refrigeration 2 (Brr2) impair ATPase and helicase activity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x">, </span> <span class="NLM_fpage">11954</span><span class="NLM_x">â</span> <span class="NLM_lpage">11965</span><span class="refDoi">Â DOI: 10.1074/jbc.M115.710848</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1074%2Fjbc.M115.710848" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=11954-11965&author=S.+Ledouxauthor=C.+Guthrie&title=Retinitis+Pigmentosa+mutations+in+Bad+Response+to+Refrigeration+2+%28Brr2%29+impair+ATPase+and+helicase+activity&doi=10.1074%2Fjbc.M115.710848"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.710848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.710848%26sid%3Dliteratum%253Aachs%26aulast%3DLedoux%26aufirst%3DS.%26aulast%3DGuthrie%26aufirst%3DC.%26atitle%3DRetinitis%2520Pigmentosa%2520mutations%2520in%2520Bad%2520Response%2520to%2520Refrigeration%25202%2520%2528Brr2%2529%2520impair%2520ATPase%2520and%2520helicase%2520activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D11954%26epage%3D11965%26doi%3D10.1074%2Fjbc.M115.710848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Shadrick, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ndjomou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frick, D. N.</span><span> </span><span class="NLM_article-title">Discovering new medicines targeting helicases: challenges and recent progress</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">761</span><span class="NLM_x">â</span> <span class="NLM_lpage">781</span><span class="refDoi">Â DOI: 10.1177/1087057113482586</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1177%2F1087057113482586" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=23536547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWitr7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=761-781&author=W.+R.+Shadrickauthor=J.+Ndjomouauthor=R.+Kolliauthor=S.+Mukherjeeauthor=A.+M.+Hansonauthor=D.+N.+Frick&title=Discovering+new+medicines+targeting+helicases%3A+challenges+and+recent+progress&doi=10.1177%2F1087057113482586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering new medicines targeting helicases: challenges and recent progress</span></div><div class="casAuthors">Shadrick, William R.; Ndjomou, Jean; Kolli, Rajesh; Mukherjee, Sourav; Hanson, Alicia M.; Frick, David N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">761-781, 21</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Helicases are ubiquitous motor proteins that sep. and/or rearrange nucleic acid duplexes in reactions fueled by ATP (ATP) hydrolysis.  Helicases encoded by bacteria, viruses, and human cells are widely studied targets for new antiviral, antibiotic, and anticancer drugs.  This review summarizes the biochem. of frequently targeted helicases.  These proteins include viral enzymes from herpes simplex virus, papillomaviruses, polyomaviruses, coronaviruses, the hepatitis C virus, and various flaviviruses.  Bacterial targets examd. include DnaB-like and RecBCD-like helicases.  The human DEAD-box protein DDX3 is the cellular antiviral target discussed, and cellular anticancer drug targets discussed are the human RecQ-like helicases and eIF4A.  We also review assays used for helicase inhibitor discovery and the most promising and common helicase inhibitor chemotypes, such as nucleotide analogs, polyphenyls, metal ion chelators, flavones, polycyclic arom. polymers, coumarins, and various DNA binding pharmacophores.  Also discussed are common complications encountered while searching for potent helicase inhibitors and possible solns. for these problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruzz0kEpdpZrVg90H21EOLACvtfcHk0ljvL5g-teTidQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWitr7J&md5=b0fb1a502a37f3eed08bdc090db8a40c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1177%2F1087057113482586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113482586%26sid%3Dliteratum%253Aachs%26aulast%3DShadrick%26aufirst%3DW.%2BR.%26aulast%3DNdjomou%26aufirst%3DJ.%26aulast%3DKolli%26aufirst%3DR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DHanson%26aufirst%3DA.%2BM.%26aulast%3DFrick%26aufirst%3DD.%2BN.%26atitle%3DDiscovering%2520new%2520medicines%2520targeting%2520helicases%253A%2520challenges%2520and%2520recent%2520progress%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D761%26epage%3D781%26doi%3D10.1177%2F1087057113482586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Ogino, M.; Ikeda, Z.; Fujimoto, J.; Ohba, Y.; Ishii, N.; Fujimoto, T.; Oda, T.; Taya, N.; Yamashita, T.; Matsunaga, N.</span><span> </span><span class="NLM_article-title">Heterocyclic compound</span>. PCT Int. Appl. WO2014030743 A1, February 27,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=M.+Ogino&author=Z.+Ikeda&author=J.+Fujimoto&author=Y.+Ohba&author=N.+Ishii&author=T.+Fujimoto&author=T.+Oda&author=N.+Taya&author=T.+Yamashita&author=N.+Matsunaga&title=Heterocyclic+compound"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOgino%26aufirst%3DM.%26atitle%3DHeterocyclic%2520compound%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="note"><p class="first last">Since helicase RNA binding sites are less similar to each other, one might expect RNA site binder would have higher selectivity. However, it was not correct at least in our case. The reason for the lower selectivity of <b>12</b> is still unclear, and further investigation such as cocrystal structure analysis with other helicases is necessary.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="note"><p class="first last">We could not rule out the possibility of the formation of hydrogen bonding interaction with Lys2500 in place of Thr1197 so far.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Wallace, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyssikatos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarski, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingston, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otten, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hingorani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woessner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, K.</span><span> </span><span class="NLM_article-title">Potent and selective mitogen-activated protein kinase kinase (MEK) 1,2 inhibitors. 1. 4-(4-bromo-2-fluorophenylamino)-1- methylpyridin-2(1H)-ones</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">441</span><span class="NLM_x">â</span> <span class="NLM_lpage">444</span><span class="refDoi">Â DOI: 10.1021/jm050834y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050834y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=441-444&author=E.+M.+Wallaceauthor=J.+Lyssikatosauthor=J.+F.+Blakeauthor=J.+Seoauthor=H.+W.+Yangauthor=T.+C.+Yehauthor=M.+Perrierauthor=H.+Jarskiauthor=V.+Marshauthor=G.+Pochauthor=M.+G.+Livingstonauthor=J.+Ottenauthor=G.+Hingoraniauthor=R.+Woessnerauthor=P.+Leeauthor=J.+Winklerauthor=K.+Koch&title=Potent+and+selective+mitogen-activated+protein+kinase+kinase+%28MEK%29+1%2C2+inhibitors.+1.+4-%284-bromo-2-fluorophenylamino%29-1-+methylpyridin-2%281H%29-ones&doi=10.1021%2Fjm050834y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm050834y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050834y%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DE.%2BM.%26aulast%3DLyssikatos%26aufirst%3DJ.%26aulast%3DBlake%26aufirst%3DJ.%2BF.%26aulast%3DSeo%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%2BW.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DPerrier%26aufirst%3DM.%26aulast%3DJarski%26aufirst%3DH.%26aulast%3DMarsh%26aufirst%3DV.%26aulast%3DPoch%26aufirst%3DG.%26aulast%3DLivingston%26aufirst%3DM.%2BG.%26aulast%3DOtten%26aufirst%3DJ.%26aulast%3DHingorani%26aufirst%3DG.%26aulast%3DWoessner%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DWinkler%26aufirst%3DJ.%26aulast%3DKoch%26aufirst%3DK.%26atitle%3DPotent%2520and%2520selective%2520mitogen-activated%2520protein%2520kinase%2520kinase%2520%2528MEK%2529%25201%252C2%2520inhibitors.%25201.%25204-%25284-bromo-2-fluorophenylamino%2529-1-%2520methylpyridin-2%25281H%2529-ones%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D441%26epage%3D444%26doi%3D10.1021%2Fjm050834y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Kai, H.; Endoh, T.; Jikihara, S.; Asahi, K.; Horiguchi, T.</span><span> </span><span class="NLM_article-title">Novel heterocyclic derivatives and pharmaceutical composition containing same</span>. PCT Int. Appl. WO2012020742 A1, February 16,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=H.+Kai&author=T.+Endoh&author=S.+Jikihara&author=K.+Asahi&author=T.+Horiguchi&title=Novel+heterocyclic+derivatives+and+pharmaceutical+composition+containing+same"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKai%26aufirst%3DH.%26atitle%3DNovel%2520heterocyclic%2520derivatives%2520and%2520pharmaceutical%2520composition%2520containing%2520same%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Ito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwatani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sogabe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aparicio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakanishi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imaeda, Y.</span><span> </span><span class="NLM_article-title">Discovery of selective ATP-competitive eIF4A3 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2200</span><span class="NLM_x">â</span> <span class="NLM_lpage">2209</span><span class="refDoi">Â DOI: 10.1016/j.bmc.2017.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.bmc.2017.02.035" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2200-2209&author=M.+Itoauthor=M.+Iwataniauthor=Y.+Kamadaauthor=S.+Sogabeauthor=S.+Nakaoauthor=T.+Tanakaauthor=T.+Kawamotoauthor=S.+Aparicioauthor=A.+Nakanishiauthor=Y.+Imaeda&title=Discovery+of+selective+ATP-competitive+eIF4A3+inhibitors&doi=10.1016%2Fj.bmc.2017.02.035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DM.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DNakao%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DKawamoto%26aufirst%3DT.%26aulast%3DAparicio%26aufirst%3DS.%26aulast%3DNakanishi%26aufirst%3DA.%26aulast%3DImaeda%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520selective%2520ATP-competitive%2520eIF4A3%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2200%26epage%3D2209%26doi%3D10.1016%2Fj.bmc.2017.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Nakashima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeda, Y.</span><span> </span><span class="NLM_article-title">Impact of physicochemical profiling for rational approach on drug discovery</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">1228</span><span class="NLM_x">â</span> <span class="NLM_lpage">1238</span><span class="refDoi">Â DOI: 10.1248/cpb.c13-00436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1248%2Fcpb.c13-00436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=24292785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2013&pages=1228-1238&author=S.+Nakashimaauthor=K.+Yamamotoauthor=Y.+Araiauthor=Y.+Ikeda&title=Impact+of+physicochemical+profiling+for+rational+approach+on+drug+discovery&doi=10.1248%2Fcpb.c13-00436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of physicochemical profiling for rational approach on drug discovery</span></div><div class="casAuthors">Nakashima, Shoko; Yamamoto, Katsuhiko; Arai, Yuta; Ikeda, Yukihiro</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1228-1238</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Solid-state characterization plays a vital role in lead optimization and candidate selection with the appropriate physicochem. properties for proper oral dosage formulation.  Aq. soly. is an important parameter in the successful development of oral dosage formulation since poor aq. soly. limits absorption.  In this study, we summarized an efficient approach using a small amt. of sample for solid-state characterization, including thermodn. soly., which is defined as physicochem. profiling.  By using the physicochem. profiling results of 75 anti-cancer drugs and clin. candidates, we examd. the relationship between thermodn. soly. and mol. structural parameters and assessed the effects of thermodn. soly. on pharmacokinetic profile for rational sol. drug design.  The Log DpH 7.4, arom. ring count, and hydrogen bond count were good indicators for predicting sparingly sol. compds. that increase the lattice energy because of Ï-Ï stacking and hydrogen bonds, resulting in lowered thermodn. soly.  The level of thermodn. soly. in simulated intestinal fluid (pH 6.8) in the presence and absence of bile acid, which is required for minimal acceptable bioavailability (>30%), was 1 Î¼g/mL and 10 Î¼g/mL, resp.  Physicochem. profiling, which includes thermodn. soly. considering its solid-state properties, contributes to rational lead optimization and efficient candidate selection for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBXTGzqqcJSbVg90H21EOLACvtfcHk0lj2MEdya748Uw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D&md5=c1291e838e8509c2fe2a1edfaf509ed4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c13-00436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c13-00436%26sid%3Dliteratum%253Aachs%26aulast%3DNakashima%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DArai%26aufirst%3DY.%26aulast%3DIkeda%26aufirst%3DY.%26atitle%3DImpact%2520of%2520physicochemical%2520profiling%2520for%2520rational%2520approach%2520on%2520drug%2520discovery%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2013%26volume%3D61%26spage%3D1228%26epage%3D1238%26doi%3D10.1248%2Fcpb.c13-00436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Masumoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeyasu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oizumi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span> </span><span class="NLM_article-title">Studies of novel 1,4-dihydropyridine Ca antagonist CS-905. I. Measurement of partition coefficient (log P) by high performance liquid chromatography (HPLC)</span> <span class="citation_source-journal">Yakugaku Zasshi</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">â</span> <span class="NLM_lpage">220</span><span class="refDoi">Â DOI: 10.1248/yakushi1947.115.3_213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1248%2Fyakushi1947.115.3_213" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1995&pages=213-220&author=K.+Masumotoauthor=A.+Takeyasuauthor=K.+Oizumiauthor=T.+Kobayashi&title=Studies+of+novel+1%2C4-dihydropyridine+Ca+antagonist+CS-905.+I.+Measurement+of+partition+coefficient+%28log+P%29+by+high+performance+liquid+chromatography+%28HPLC%29&doi=10.1248%2Fyakushi1947.115.3_213"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1248%2Fyakushi1947.115.3_213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fyakushi1947.115.3_213%26sid%3Dliteratum%253Aachs%26aulast%3DMasumoto%26aufirst%3DK.%26aulast%3DTakeyasu%26aufirst%3DA.%26aulast%3DOizumi%26aufirst%3DK.%26aulast%3DKobayashi%26aufirst%3DT.%26atitle%3DStudies%2520of%2520novel%25201%252C4-dihydropyridine%2520Ca%2520antagonist%2520CS-905.%2520I.%2520Measurement%2520of%2520partition%2520coefficient%2520%2528log%2520P%2529%2520by%2520high%2520performance%2520liquid%2520chromatography%2520%2528HPLC%2529%26jtitle%3DYakugaku%2520Zasshi%26date%3D1995%26volume%3D115%26spage%3D213%26epage%3D220%26doi%3D10.1248%2Fyakushi1947.115.3_213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span><span class="refDoi">Â DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lj2MEdya748Uw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">â</span> <span class="NLM_lpage">1214</span><span class="refDoi">Â DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lhzl5xe2CMlDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">â</span> <span class="NLM_lpage">674</span><span class="refDoi">Â DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ÎF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhzl5xe2CMlDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">â</span> <span class="NLM_lpage">2132</span><span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhzl5xe2CMlDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">â</span> <span class="NLM_lpage">255</span><span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhzl5xe2CMlDg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">â</span> <span class="NLM_lpage">221</span><span class="refDoi">Â DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgzm8BGLcN6Ew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography. Collaborative Computational Project Number 4</span>.  <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">â</span> <span class="NLM_lpage">763</span>, DOI: <span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+CCP4+suite%3A+programs+for+protein+crystallography.+Collaborative+Computational+Project+Number+4.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+1994%2C+50%2C+760%E2%80%93763%2C+DOI%3A+10.1107%2FS0907444994003112."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography.%2520Collaborative%2520Computational%2520Project%2520Number%25204%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">â</span> <span class="NLM_lpage">21</span><span class="refDoi">Â DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lgzm8BGLcN6Ew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':[],'ref17':[],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 17 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert E. Boer, Zachary R. Torrey, <span class="NLM_string-name hlFld-ContribAuthor">John S. Schneekloth, Jr.</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Modulation of Pre-mRNA Splicing in Mammalian Systems. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (4)
                                     , 808-818. <a href="https://doi.org/10.1021/acschembio.0c00001" title="DOI URL">https://doi.org/10.1021/acschembio.0c00001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00001%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DChemical%252BModulation%252Bof%252BPre-mRNA%252BSplicing%252Bin%252BMammalian%252BSystems%26aulast%3DBoer%26aufirst%3DRobert%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03012020%26date%3D19032020%26date%3D07042020%26date%3D19032020%26volume%3D15%26issue%3D4%26spage%3D808%26epage%3D818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Beatrice  Eymin</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the spliceosome machinery: A new therapeutic axis in cancer?. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2021,</strong> <em>189 </em>, 114039. <a href="https://doi.org/10.1016/j.bcp.2020.114039" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.114039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.114039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.114039%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DTargeting%252Bthe%252Bspliceosome%252Bmachinery%25253A%252BA%252Bnew%252Btherapeutic%252Baxis%252Bin%252Bcancer%25253F%26aulast%3DEymin%26aufirst%3DBeatrice%26date%3D2021%26volume%3D189%26spage%3D114039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">I Gede Putu  Supadmanaba</span>, <span class="hlFld-ContribAuthor ">Giulia  Mantini</span>, <span class="hlFld-ContribAuthor ">Ornella  Randazzo</span>, <span class="hlFld-ContribAuthor ">Mjriam  Capula</span>, <span class="hlFld-ContribAuthor ">Ittai B.  Muller</span>, <span class="hlFld-ContribAuthor ">Stella  Cascioferro</span>, <span class="hlFld-ContribAuthor ">Patrizia  Diana</span>, <span class="hlFld-ContribAuthor ">Godefridus J.  Peters</span>, <span class="hlFld-ContribAuthor ">Elisa  Giovannetti</span>. </span><span class="cited-content_cbyCitation_article-title">Interrelationship between miRNA and splicing factors in pancreatic ductal adenocarcinoma. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2021,</strong> <em>111 </em>, 1-24. <a href="https://doi.org/10.1080/15592294.2021.1916697" title="DOI URL">https://doi.org/10.1080/15592294.2021.1916697</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2021.1916697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2021.1916697%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DInterrelationship%252Bbetween%252BmiRNA%252Band%252Bsplicing%252Bfactors%252Bin%252Bpancreatic%252Bductal%252Badenocarcinoma%26aulast%3DSupadmanaba%26aufirst%3DI%2BGede%2BPutu%26date%3D2021%26date%3D2021%26volume%3D111%26spage%3D1%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maxime  Blijlevens</span>, <span class="hlFld-ContribAuthor ">Jing  Li</span>, <span class="hlFld-ContribAuthor ">Victor W.  van Beusechem</span>. </span><span class="cited-content_cbyCitation_article-title">Biology of the mRNA Splicing Machinery and Its Dysregulation in Cancer Providing Therapeutic Opportunities. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (10)
                                     , 5110. <a href="https://doi.org/10.3390/ijms22105110" title="DOI URL">https://doi.org/10.3390/ijms22105110</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22105110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22105110%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DBiology%252Bof%252Bthe%252BmRNA%252BSplicing%252BMachinery%252Band%252BIts%252BDysregulation%252Bin%252BCancer%252BProviding%252BTherapeutic%252BOpportunities%26aulast%3DBlijlevens%26aufirst%3DMaxime%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D10%26spage%3D5110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yosser Zina  Abdelkrim</span>, <span class="hlFld-ContribAuthor ">Josette  Banroques</span>, <span class="hlFld-ContribAuthor ">N.  Kyle Tanner</span>. </span><span class="cited-content_cbyCitation_article-title">Known Inhibitors of RNA Helicases and Their Therapeutic Potential. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 35-52. <a href="https://doi.org/10.1007/978-1-0716-0935-4_3" title="DOI URL">https://doi.org/10.1007/978-1-0716-0935-4_3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-0716-0935-4_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-0716-0935-4_3%26sid%3Dliteratum%253Aachs%26atitle%3DKnown%252BInhibitors%252Bof%252BRNA%252BHelicases%252Band%252BTheir%252BTherapeutic%252BPotential%26aulast%3DAbdelkrim%26aufirst%3DYosser%2BZina%26date%3D2021%26date%3D2020%26spage%3D35%26epage%3D52%26pub%3DSpringer%2520US%26atitle%3DRNA%252BRemodeling%252BProteins%26aulast%3DBoudvillain%26aufirst%3DMarc%26date%3D2021%26volume%3D2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonietta Rosella  Farina</span>, <span class="hlFld-ContribAuthor ">Lucia  Cappabianca</span>, <span class="hlFld-ContribAuthor ">Michela  Sebastiano</span>, <span class="hlFld-ContribAuthor ">Veronica  Zelli</span>, <span class="hlFld-ContribAuthor ">Stefano  Guadagni</span>, <span class="hlFld-ContribAuthor ">Andrew Reay  Mackay</span>. </span><span class="cited-content_cbyCitation_article-title">Hypoxia-induced alternative splicing: the 11th Hallmark of Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Experimental & Clinical Cancer Research</span><span> <strong>2020,</strong> <em>39 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13046-020-01616-9" title="DOI URL">https://doi.org/10.1186/s13046-020-01616-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13046-020-01616-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13046-020-01616-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Experimental%2520%2526%2520Clinical%2520Cancer%2520Research%26atitle%3DHypoxia-induced%252Balternative%252Bsplicing%25253A%252Bthe%252B11th%252BHallmark%252Bof%252BCancer%26aulast%3DFarina%26aufirst%3DAntonietta%2BRosella%26date%3D2020%26date%3D2020%26volume%3D39%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sophie C.  Bonnal</span>, <span class="hlFld-ContribAuthor ">Irene  LÃ³pez-Oreja</span>, <span class="hlFld-ContribAuthor ">Juan  ValcÃ¡rcel</span>. </span><span class="cited-content_cbyCitation_article-title">Roles and mechanisms of alternative splicing in cancer â implications for care. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Clinical Oncology</span><span> <strong>2020,</strong> <em>17 </em>
                                    (8)
                                     , 457-474. <a href="https://doi.org/10.1038/s41571-020-0350-x" title="DOI URL">https://doi.org/10.1038/s41571-020-0350-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41571-020-0350-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41571-020-0350-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Clinical%2520Oncology%26atitle%3DRoles%252Band%252Bmechanisms%252Bof%252Balternative%252Bsplicing%252Bin%252Bcancer%252B%2525E2%252580%252594%252Bimplications%252Bfor%252Bcare%26aulast%3DBonnal%26aufirst%3DSophie%2BC.%26date%3D2020%26date%3D2020%26volume%3D17%26issue%3D8%26spage%3D457%26epage%3D474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marc  SuÃ±Ã©-Pou</span>, <span class="hlFld-ContribAuthor ">MarÃ­a J.  Limeres</span>, <span class="hlFld-ContribAuthor ">Cristina  Moreno-Castro</span>, <span class="hlFld-ContribAuthor ">Cristina  HernÃ¡ndez-Munain</span>, <span class="hlFld-ContribAuthor ">Josep M.  SuÃ±Ã©-Negre</span>, <span class="hlFld-ContribAuthor ">MarÃ­a L.  Cuestas</span>, <span class="hlFld-ContribAuthor ">Carlos  SuÃ±Ã©</span>. </span><span class="cited-content_cbyCitation_article-title">Innovative Therapeutic and Delivery Approaches Using Nanotechnology to Correct Splicing Defects Underlying Disease. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fgene.2020.00731" title="DOI URL">https://doi.org/10.3389/fgene.2020.00731</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2020.00731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2020.00731%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DInnovative%252BTherapeutic%252Band%252BDelivery%252BApproaches%252BUsing%252BNanotechnology%252Bto%252BCorrect%252BSplicing%252BDefects%252BUnderlying%252BDisease%26aulast%3DSu%25C3%25B1%25C3%25A9-Pou%26aufirst%3DMarc%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongbo  Wang</span>, <span class="hlFld-ContribAuthor ">Yufang  Bao</span>, <span class="hlFld-ContribAuthor ">Sirui  Zhang</span>, <span class="hlFld-ContribAuthor ">Zefeng  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Splicing dysregulation in cancer: from mechanistic understanding to a new class of therapeutic targets. </span><span class="cited-content_cbyCitation_journal-name">Science China Life Sciences</span><span> <strong>2020,</strong> <em>63 </em>
                                    (4)
                                     , 469-484. <a href="https://doi.org/10.1007/s11427-019-1605-0" title="DOI URL">https://doi.org/10.1007/s11427-019-1605-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11427-019-1605-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11427-019-1605-0%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520China%2520Life%2520Sciences%26atitle%3DSplicing%252Bdysregulation%252Bin%252Bcancer%25253A%252Bfrom%252Bmechanistic%252Bunderstanding%252Bto%252Ba%252Bnew%252Bclass%252Bof%252Btherapeutic%252Btargets%26aulast%3DWang%26aufirst%3DYongbo%26date%3D2020%26date%3D2020%26volume%3D63%26issue%3D4%26spage%3D469%26epage%3D484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eva  Absmeier</span>, <span class="hlFld-ContribAuthor ">Karine F.  Santos</span>, <span class="hlFld-ContribAuthor ">Markus C.  Wahl</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Mechanism Underlying Inhibition of Intrinsic ATPase Activity in a Ski2-like RNA Helicase. </span><span class="cited-content_cbyCitation_journal-name">Structure</span><span> <strong>2020,</strong> <em>28 </em>
                                    (2)
                                     , 236-243.e3. <a href="https://doi.org/10.1016/j.str.2019.11.014" title="DOI URL">https://doi.org/10.1016/j.str.2019.11.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.str.2019.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.str.2019.11.014%26sid%3Dliteratum%253Aachs%26jtitle%3DStructure%26atitle%3DMolecular%252BMechanism%252BUnderlying%252BInhibition%252Bof%252BIntrinsic%252BATPase%252BActivity%252Bin%252Ba%252BSki2-like%252BRNA%252BHelicase%26aulast%3DAbsmeier%26aufirst%3DEva%26date%3D2020%26volume%3D28%26issue%3D2%26spage%3D236%26epage%3D243.e3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony B  DeNicola</span>, <span class="hlFld-ContribAuthor ">Yi  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic approaches to treat human spliceosomal diseases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Biotechnology</span><span> <strong>2019,</strong> <em>60 </em>, 72-81. <a href="https://doi.org/10.1016/j.copbio.2019.01.003" title="DOI URL">https://doi.org/10.1016/j.copbio.2019.01.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.copbio.2019.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.copbio.2019.01.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Biotechnology%26atitle%3DTherapeutic%252Bapproaches%252Bto%252Btreat%252Bhuman%252Bspliceosomal%252Bdiseases%26aulast%3DDeNicola%26aufirst%3DAnthony%2BB%26date%3D2019%26volume%3D60%26spage%3D72%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shakur  Mohibi</span>, <span class="hlFld-ContribAuthor ">Xinbin  Chen</span>, <span class="hlFld-ContribAuthor ">Jin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer theâRBPâeuticsâRNA-binding proteins as therapeutic targets for cancer. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2019,</strong> <em>203 </em>, 107390. <a href="https://doi.org/10.1016/j.pharmthera.2019.07.001" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2019.07.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2019.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2019.07.001%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DCancer%252Bthe%2525E2%252580%252598RBP%2525E2%252580%252599eutics%2525E2%252580%252593RNA-binding%252Bproteins%252Bas%252Btherapeutic%252Btargets%252Bfor%252Bcancer%26aulast%3DMohibi%26aufirst%3DShakur%26date%3D2019%26volume%3D203%26spage%3D107390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krister J  Barkovich</span>, <span class="hlFld-ContribAuthor ">Megan K  Moore</span>, <span class="hlFld-ContribAuthor ">Qi  Hu</span>, <span class="hlFld-ContribAuthor ">Kevan M  Shokat</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical genetic inhibition of DEAD-box proteins using covalent complementarity. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2018,</strong> <em>46 </em>
                                    (17)
                                     , 8689-8699. <a href="https://doi.org/10.1093/nar/gky706" title="DOI URL">https://doi.org/10.1093/nar/gky706</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gky706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgky706%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DChemical%252Bgenetic%252Binhibition%252Bof%252BDEAD-box%252Bproteins%252Busing%252Bcovalent%252Bcomplementarity%26aulast%3DBarkovich%26aufirst%3DKrister%2BJ%26date%3D2018%26date%3D2018%26volume%3D46%26issue%3D17%26spage%3D8689%26epage%3D8699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anant A  Agrawal</span>, <span class="hlFld-ContribAuthor ">Lihua  Yu</span>, <span class="hlFld-ContribAuthor ">Peter G  Smith</span>, <span class="hlFld-ContribAuthor ">Silvia  Buonamici</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting splicing abnormalities in cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Genetics & Development</span><span> <strong>2018,</strong> <em>48 </em>, 67-74. <a href="https://doi.org/10.1016/j.gde.2017.10.010" title="DOI URL">https://doi.org/10.1016/j.gde.2017.10.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.gde.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.gde.2017.10.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Genetics%2520%2526%2520Development%26atitle%3DTargeting%252Bsplicing%252Babnormalities%252Bin%252Bcancer%26aulast%3DAgrawal%26aufirst%3DAnant%2BA%26date%3D2018%26volume%3D48%26spage%3D67%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jung-Chun  Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic Applications of Targeted Alternative Splicing to Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2018,</strong> <em>19 </em>
                                    (1)
                                     , 75. <a href="https://doi.org/10.3390/ijms19010075" title="DOI URL">https://doi.org/10.3390/ijms19010075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms19010075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms19010075%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DTherapeutic%252BApplications%252Bof%252BTargeted%252BAlternative%252BSplicing%252Bto%252BCancer%252BTreatment%26aulast%3DLin%26aufirst%3DJung-Chun%26date%3D2018%26date%3D2017%26volume%3D19%26issue%3D1%26spage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masahiro  Ito</span>, <span class="hlFld-ContribAuthor ">Misa  Iwatani</span>, <span class="hlFld-ContribAuthor ">Takeshi  Yamamoto</span>, <span class="hlFld-ContribAuthor ">Toshio  Tanaka</span>, <span class="hlFld-ContribAuthor ">Tomohiro  Kawamoto</span>, <span class="hlFld-ContribAuthor ">Daisuke  Morishita</span>, <span class="hlFld-ContribAuthor ">Atsushi  Nakanishi</span>, <span class="hlFld-ContribAuthor ">Hironobu  Maezaki</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of spiro[indole-3,2â²-pyrrolidin]-2(1H)-one based inhibitors targeting Brr2, a core component of the U5 snRNP. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (17)
                                     , 4753-4767. <a href="https://doi.org/10.1016/j.bmc.2017.07.017" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.07.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.07.017%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bspiro%25255Bindole-3%25252C2%2525E2%252580%2525B2-pyrrolidin%25255D-2%2525281H%252529-one%252Bbased%252Binhibitors%252Btargeting%252BBrr2%25252C%252Ba%252Bcore%252Bcomponent%252Bof%252Bthe%252BU5%252BsnRNP%26aulast%3DIto%26aufirst%3DMasahiro%26date%3D2017%26volume%3D25%26issue%3D17%26spage%3D4753%26epage%3D4767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic flowchart summarizing the process of the discovery of Brr2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Overall cocrystal structure of Brr2 with compound <b>3</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ">5URJ</a>) and <b>12</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URM">5URM</a>). (Upper) Ribbon plot of Brr2. A green ligand represents compound <b>3</b>, a magenta ligand represents compound <b>12</b>, and yellow ligands represent nucleotides from a published structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F93">4F93</a>). N-terminal cassette: RecA-1, orange; RecA-2, yellow; WH, gray; HB, blue; HLH, pink; IG, green. C-terminal cassette: RecA-1, pale red; RecA-2, pale yellow; WH, gray; HB, cyan; HLH, pale pink; IG, yellow green. (Lower) Schematic representation of Brr2 in domain borders. (B) Binding site of <b>3</b>* (left) and <b>12</b> (right) in Brr2. (C) Binding mode of the left side of <b>3</b>. (D) Binding mode of right side of <b>3</b>. *The binding isomer is estimated to be 2<i>R</i>-form.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray cocrystal structure of inhibitors bound to Brr2. (A) Overlay of X-ray crystal structure of <b>3</b> in Brr2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ">5URJ</a>, depicted in green) and <b>6</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URK">5URK</a>, depicted in yellow). (B) Polar amino acid residues around benzyl group of <b>6</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking model of compound <b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. RNA helicase activity of Brr2 monitored using fluorescent-labeled RNA. Single- or double-stranded RNAs were separated by electrophoresis and detected as fluorescent signals. Dose-dependent inhibition as indicated concentration of Brr2 inhibitor <b>9</b> is represented (lanes 4â8). Negative control <b>10</b> was also evaluated at 100 Î¼M (lane 9). The enzymatic reaction without compounds was termed as 100% of control (lane 3), and that without enzyme was determined as 0% control (lane 10). Double- and single-stranded RNAs and total reactions without ATP are lanes 1, 2, and 11, respectively. The intensity of each band was measured, and inhibition rate was calculated. The IC<sub>50</sub> value against helicase activity was estimated by plotting of each inhibition rate, which is described in the right graph.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>2</b>â<b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) NaN<sub>3</sub>, DMF, 55 Â°Câ60 Â°C; (ii) LiOHÂ·H<sub>2</sub>O, EtOH (or EtOH and THF), H<sub>2</sub>O, rt, 71%â95% for 2 steps; (b) (i) MeNH(OMe)Â·HCl, T<sub>3</sub>P, DIPEA, DMF, 0 Â°C to rt; (ii) LiAlH<sub>4</sub>, THF, 0 Â°C to rt, 24% for 2 steps; (c) (i) MeNH(OMe)Â·HCl, EDCÂ·HCl, HOBt, DIPEA, DMF, rt; (ii) LiAlH<sub>4</sub>, THF, 0 Â°C, 58% for 2 steps; (d) (i) <b>17a</b> or <b>17b</b>, Ti(<i>i</i>-PrO)<sub>4</sub>, THF, 50 Â°C, 16 h; (ii) NaBH(OAc)<sub>3</sub>, 50 Â°C; (iii) CDI, DBU, THF, rt, 6%â27% for 3 steps; (e) NaH, MeI, DMF, 0 Â°C to rt, 73%.</p></p></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/medium/jm-2017-00461f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>6</b>â<b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-13/acs.jmedchem.7b00461/20170707/images/large/jm-2017-00461f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00461&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) <b>18a</b> or <b>18b</b>, Ti(<i>i</i>-PrO)<sub>4</sub>, THF, 50 Â°C, 16 h; (ii) NaBH(OAc)<sub>3</sub>, 50 Â°C; (iii) CDI, DBU, THF, rt, 34%â67% for 3 steps; (b) TFA, anisole, rt, quant; (c) <b>21a</b> or <b>21b</b>, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 13%â36%; (d) <b>21c</b>, K<sub>2</sub>CO<sub>3</sub>, KI, NMP, rt, 27%; (d) (i) <b>18c</b>, PMHS, SnCl<sub>2</sub>Â·MeOH, rt, 31%; (ii) CDI, DBU, THF, rt, 81%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i49">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51842" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51842" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 30 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Wahl, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Will, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LÃ¼hrmann, R.</span><span> </span><span class="NLM_article-title">The spliceosome: design principles of a dynamic RNP machine</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">â</span> <span class="NLM_lpage">718</span><span class="refDoi">Â DOI: 10.1016/j.cell.2009.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.cell.2009.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=19239890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvFGksbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=701-718&author=M.+C.+Wahlauthor=C.+L.+Willauthor=R.+L%C3%BChrmann&title=The+spliceosome%3A+design+principles+of+a+dynamic+RNP+machine&doi=10.1016%2Fj.cell.2009.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The spliceosome: Design principles of a dynamic RNP machine</span></div><div class="casAuthors">Wahl, Markus C.; Will, Cindy L.; Luehrmann, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">701-718</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ribonucleoproteins (RNPs) mediate key cellular functions such as gene expression and its regulation.  Whereas most RNP enzymes are stable in compn. and harbor preformed active sites, the spliceosome, which removes noncoding introns from precursor mRNAs (pre-mRNAs), follows fundamentally different strategies.  In order to provide both accuracy to the recognition of reactive splice sites in the pre-mRNA and flexibility to the choice of splice sites during alternative splicing, the spliceosome exhibits exceptional compositional and structural dynamics that are exploited during substrate-dependent complex assembly, catalytic activation, and active site remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphwXkeaEfBWLVg90H21EOLACvtfcHk0ljA598_OfQU4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvFGksbk%253D&md5=589aa1782ba069c55bff0b6bb7762b84</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DWahl%26aufirst%3DM.%2BC.%26aulast%3DWill%26aufirst%3DC.%2BL.%26aulast%3DL%25C3%25BChrmann%26aufirst%3DR.%26atitle%3DThe%2520spliceosome%253A%2520design%2520principles%2520of%2520a%2520dynamic%2520RNP%2520machine%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D701%26epage%3D718%26doi%3D10.1016%2Fj.cell.2009.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Staley, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, C.</span><span> </span><span class="NLM_article-title">Mechanical devices of the spliceosome: motors, clocks, springs, and things</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">â</span> <span class="NLM_lpage">326</span><span class="refDoi">Â DOI: 10.1016/S0092-8674(00)80925-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2FS0092-8674%2800%2980925-3" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1998&pages=315-326&author=J.+P.+Staleyauthor=C.+Guthrie&title=Mechanical+devices+of+the+spliceosome%3A+motors%2C+clocks%2C+springs%2C+and+things&doi=10.1016%2FS0092-8674%2800%2980925-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980925-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980925-3%26sid%3Dliteratum%253Aachs%26aulast%3DStaley%26aufirst%3DJ.%2BP.%26aulast%3DGuthrie%26aufirst%3DC.%26atitle%3DMechanical%2520devices%2520of%2520the%2520spliceosome%253A%2520motors%252C%2520clocks%252C%2520springs%252C%2520and%2520things%26jtitle%3DCell%26date%3D1998%26volume%3D92%26spage%3D315%26epage%3D326%26doi%3D10.1016%2FS0092-8674%2800%2980925-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Laggerbauer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Achsel, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LÃ¼hrmann, R.</span><span> </span><span class="NLM_article-title">The human U5-200kD DEXH-box protein unwinds U4/U6 RNA duplices in vitro</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">4188</span><span class="NLM_x">â</span> <span class="NLM_lpage">4192</span><span class="refDoi">Â DOI: 10.1073/pnas.95.8.4188</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1073%2Fpnas.95.8.4188" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=4188-4192&author=B.+Laggerbauerauthor=T.+Achselauthor=R.+L%C3%BChrmann&title=The+human+U5-200kD+DEXH-box+protein+unwinds+U4%2FU6+RNA+duplices+in+vitro&doi=10.1073%2Fpnas.95.8.4188"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.8.4188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.8.4188%26sid%3Dliteratum%253Aachs%26aulast%3DLaggerbauer%26aufirst%3DB.%26aulast%3DAchsel%26aufirst%3DT.%26aulast%3DL%25C3%25BChrmann%26aufirst%3DR.%26atitle%3DThe%2520human%2520U5-200kD%2520DEXH-box%2520protein%2520unwinds%2520U4%252FU6%2520RNA%2520duplices%2520in%2520vitro%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D4188%26epage%3D4192%26doi%3D10.1073%2Fpnas.95.8.4188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Raghunathan, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, C.</span><span> </span><span class="NLM_article-title">RNA unwinding in U4/U6 snRNPs requires ATP hydrolysis and the DEIH-box splicing factor Brr2</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">â</span> <span class="NLM_lpage">855</span><span class="refDoi">Â DOI: 10.1016/S0960-9822(07)00345-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2FS0960-9822%2807%2900345-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=9705931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADyaK1cXks1Ghtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1998&pages=847-855&author=P.+L.+Raghunathanauthor=C.+Guthrie&title=RNA+unwinding+in+U4%2FU6+snRNPs+requires+ATP+hydrolysis+and+the+DEIH-box+splicing+factor+Brr2&doi=10.1016%2FS0960-9822%2807%2900345-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">RNA unwinding in U4/U6 snRNPs requires ATP hydrolysis and the DEIH-box splicing factor Brr2</span></div><div class="casAuthors">Raghunathan, Pratima L.; Guthrie, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">847-853</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Ltd.</span>)
        </div><div class="casAbstract">The dynamic rearrangements of RNA structure which occur during pre-mRNA splicing are thought to be mediated by members of the DExD/H-box family of RNA-dependent ATPases.  Although three DExD/H-box splicing factors have recently been shown to unwind synthetic RNA duplexes in purified systems, in no case has the natural biol. substrate been identified.  A duplex RNA target of particular interest is the extensive base-pairing interaction between U4 and U6 small nuclear RNAs.  Because these helixes must be disrupted to activate the spliceosome for catalysis, this rearrangement is believed to be tightly regulated in vivo.  The authors have immunopurified Brr2, a DEIH-box ATPase, in a native complex contg. U1, U2, U5 and duplex U4/U6 small nuclear ribonucleoprotein particles (snRNPs).  Addn. of hydrolyzable ATP to this complex results in the disruption of U4/U6 base-pairing, and the release of free U4 and U6 snRNPs.  A mutation in the helicase-like domain of Brr2 (brr2-1) prevents these RNA rearrangements.  Notably, U4/U6 dissocn. and release occur in the absence of exogenously added pre-mRNA.  Disruption of U4/U6 base-pairing in native snRNPs requires ATP hydrolysis and Brr2.  This is the first assignment of a DExD/H-box splicing factor to a specific biol. unwinding event.  The unwinding function of Brr2 can be antagonized by the annealing activity of Prp24.  The authors propose the existence of a dynamic cycle, uncoupled from splicing, that interconverts free and base-paired U4/U6 snRNPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVAGASfwtpRLVg90H21EOLACvtfcHk0ljA598_OfQU4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXks1Ghtrg%253D&md5=e32b31baad4ee735032cf10a22bd8c34</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2807%2900345-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252807%252900345-4%26sid%3Dliteratum%253Aachs%26aulast%3DRaghunathan%26aufirst%3DP.%2BL.%26aulast%3DGuthrie%26aufirst%3DC.%26atitle%3DRNA%2520unwinding%2520in%2520U4%252FU6%2520snRNPs%2520requires%2520ATP%2520hydrolysis%2520and%2520the%2520DEIH-box%2520splicing%2520factor%2520Brr2%26jtitle%3DCurr.%2520Biol.%26date%3D1998%26volume%3D8%26spage%3D847%26epage%3D855%26doi%3D10.1016%2FS0960-9822%2807%2900345-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Kim, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, J. J.</span><span> </span><span class="NLM_article-title">The first ATPase domain of the yeast 246-kDa protein is required for in vivo unwinding of the U4/U6 duplex</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">959</span><span class="NLM_x">â</span> <span class="NLM_lpage">971</span><span class="refDoi">Â DOI: 10.1017/S135583829999012X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1017%2FS135583829999012X" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=959-971&author=D.+H.+Kimauthor=J.+J.+Rossi&title=The+first+ATPase+domain+of+the+yeast+246-kDa+protein+is+required+for+in+vivo+unwinding+of+the+U4%2FU6+duplex&doi=10.1017%2FS135583829999012X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1017%2FS135583829999012X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS135583829999012X%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%2BH.%26aulast%3DRossi%26aufirst%3DJ.%2BJ.%26atitle%3DThe%2520first%2520ATPase%2520domain%2520of%2520the%2520yeast%2520246-kDa%2520protein%2520is%2520required%2520for%2520in%2520vivo%2520unwinding%2520of%2520the%2520U4%252FU6%2520duplex%26jtitle%3DRNA%26date%3D1999%26volume%3D5%26spage%3D959%26epage%3D971%26doi%3D10.1017%2FS135583829999012X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Small, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leggett, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winans, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staley, J. P.</span><span> </span><span class="NLM_article-title">The EF-G-like GTPase Snu114p regulates spliceosome dynamics mediated by Brr2p, a DExD/H box ATPase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">â</span> <span class="NLM_lpage">399</span><span class="refDoi">Â DOI: 10.1016/j.molcel.2006.05.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.molcel.2006.05.043" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=389-399&author=E.+C.+Smallauthor=S.+R.+Leggettauthor=A.+A.+Winansauthor=J.+P.+Staley&title=The+EF-G-like+GTPase+Snu114p+regulates+spliceosome+dynamics+mediated+by+Brr2p%2C+a+DExD%2FH+box+ATPase&doi=10.1016%2Fj.molcel.2006.05.043"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.05.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.05.043%26sid%3Dliteratum%253Aachs%26aulast%3DSmall%26aufirst%3DE.%2BC.%26aulast%3DLeggett%26aufirst%3DS.%2BR.%26aulast%3DWinans%26aufirst%3DA.%2BA.%26aulast%3DStaley%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520EF-G-like%2520GTPase%2520Snu114p%2520regulates%2520spliceosome%2520dynamics%2520mediated%2520by%2520Brr2p%252C%2520a%2520DExD%252FH%2520box%2520ATPase%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D389%26epage%3D399%26doi%3D10.1016%2Fj.molcel.2006.05.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Santos, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jovin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pena, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luhrmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, M. C.</span><span> </span><span class="NLM_article-title">Structural basis for functional cooperation between tandem helicase cassettes in Brr2-mediated remodeling of the spliceosome</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">17418</span><span class="NLM_x">â</span> <span class="NLM_lpage">17423</span><span class="refDoi">Â DOI: 10.1073/pnas.1208098109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1073%2Fpnas.1208098109" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17418-17423&author=K.+F.+Santosauthor=S.+M.+Jovinauthor=G.+Weberauthor=V.+Penaauthor=R.+Luhrmannauthor=M.+C.+Wahl&title=Structural+basis+for+functional+cooperation+between+tandem+helicase+cassettes+in+Brr2-mediated+remodeling+of+the+spliceosome&doi=10.1073%2Fpnas.1208098109"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1208098109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1208098109%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DK.%2BF.%26aulast%3DJovin%26aufirst%3DS.%2BM.%26aulast%3DWeber%26aufirst%3DG.%26aulast%3DPena%26aufirst%3DV.%26aulast%3DLuhrmann%26aufirst%3DR.%26aulast%3DWahl%26aufirst%3DM.%2BC.%26atitle%3DStructural%2520basis%2520for%2520functional%2520cooperation%2520between%2520tandem%2520helicase%2520cassettes%2520in%2520Brr2-mediated%2520remodeling%2520of%2520the%2520spliceosome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17418%26epage%3D17423%26doi%3D10.1073%2Fpnas.1208098109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Pena, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jovin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabrizio, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlowski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bujnicki, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luhrmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, M. C.</span><span> </span><span class="NLM_article-title">Common design principles in the spliceosomal RNA helicase Brr2 and in the Hel308 DNA helicase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">454</span><span class="NLM_x">â</span> <span class="NLM_lpage">466</span><span class="refDoi">Â DOI: 10.1016/j.molcel.2009.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.molcel.2009.08.006" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2009&pages=454-466&author=V.+Penaauthor=S.+M.+Jovinauthor=P.+Fabrizioauthor=J.+Orlowskiauthor=J.+M.+Bujnickiauthor=R.+Luhrmannauthor=M.+C.+Wahl&title=Common+design+principles+in+the+spliceosomal+RNA+helicase+Brr2+and+in+the+Hel308+DNA+helicase&doi=10.1016%2Fj.molcel.2009.08.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DPena%26aufirst%3DV.%26aulast%3DJovin%26aufirst%3DS.%2BM.%26aulast%3DFabrizio%26aufirst%3DP.%26aulast%3DOrlowski%26aufirst%3DJ.%26aulast%3DBujnicki%26aufirst%3DJ.%2BM.%26aulast%3DLuhrmann%26aufirst%3DR.%26aulast%3DWahl%26aufirst%3DM.%2BC.%26atitle%3DCommon%2520design%2520principles%2520in%2520the%2520spliceosomal%2520RNA%2520helicase%2520Brr2%2520and%2520in%2520the%2520Hel308%2520DNA%2520helicase%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D35%26spage%3D454%26epage%3D466%26doi%3D10.1016%2Fj.molcel.2009.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maeder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bud, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanks, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nix, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pleiss, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, R.</span><span> </span><span class="NLM_article-title">Structural evidence for consecutive Hel308-like modules in the spliceosomal ATPase Brr2</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">731</span><span class="NLM_x">â</span> <span class="NLM_lpage">739</span><span class="refDoi">Â DOI: 10.1038/nsmb.1625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1038%2Fnsmb.1625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=19525970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1ShsL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=731-739&author=L.+Zhangauthor=T.+Xuauthor=C.+Maederauthor=L.+O.+Budauthor=J.+Shanksauthor=J.+Nixauthor=C.+Guthrieauthor=J.+A.+Pleissauthor=R.+Zhao&title=Structural+evidence+for+consecutive+Hel308-like+modules+in+the+spliceosomal+ATPase+Brr2&doi=10.1038%2Fnsmb.1625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural evidence for consecutive Hel308-like modules in the spliceosomal ATPase Brr2</span></div><div class="casAuthors">Zhang, Lingdi; Xu, Tao; Maeder, Corina; Bud, Laura-Oana; Shanks, James; Nix, Jay; Guthrie, Christine; Pleiss, Jeffrey A.; Zhao, Rui</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">731-739</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Brr2 is a DExD/H-box helicase responsible for U4/U6 unwinding during spliceosomal activation.  Brr2 contains two helicase-like domains, each of which is followed by a Sec63 domain with unknown function.  We detd. the crystal structure of the second Sec63 domain, which unexpectedly resembles domains 4 and 5 of DNA helicase Hel308.  This, together with sequence similarities between Brr2's helicase-like domains and domains 1-3 of Hel308, led us to hypothesize that Brr2 contains two consecutive Hel308-like modules (Hel308-I and Hel308-II).  Our structural model and mutagenesis data suggest that Brr2 shares a similar helicase mechanism with Hel308.  We demonstrate that Hel308-II interacts with Prp8 and Snu114 in vitro and in vivo.  We further find that the C-terminal region of Prp8 (Prp8-CTR) facilitates the binding of the Brr2-Prp8-CTR complex to U4/U6.  Our results have important implications for the mechanism and regulation of Brr2's activity in splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojfFiQIh2fJbVg90H21EOLACvtfcHk0liCKQkky0JF0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1ShsL0%253D&md5=f9a5ba148222e024af16edccea66f78e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1625%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DMaeder%26aufirst%3DC.%26aulast%3DBud%26aufirst%3DL.%2BO.%26aulast%3DShanks%26aufirst%3DJ.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DGuthrie%26aufirst%3DC.%26aulast%3DPleiss%26aufirst%3DJ.%2BA.%26aulast%3DZhao%26aufirst%3DR.%26atitle%3DStructural%2520evidence%2520for%2520consecutive%2520Hel308-like%2520modules%2520in%2520the%2520spliceosomal%2520ATPase%2520Brr2%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D731%26epage%3D739%26doi%3D10.1038%2Fnsmb.1625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">van Nues, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beggs, J. D.</span><span> </span><span class="NLM_article-title">Functional contacts with a range of splicing proteins suggest a central role for Brr2p in the dynamic control of the order of events in spliceosomes of Saccharomyces cerevisiae</span> <span class="citation_source-journal">Genetics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">1451</span><span class="NLM_x">â</span> <span class="NLM_lpage">1467</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2001&pages=1451-1467&author=R.+W.+van+Nuesauthor=J.+D.+Beggs&title=Functional+contacts+with+a+range+of+splicing+proteins+suggest+a+central+role+for+Brr2p+in+the+dynamic+control+of+the+order+of+events+in+spliceosomes+of+Saccharomyces+cerevisiae"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BNues%26aufirst%3DR.%2BW.%26aulast%3DBeggs%26aufirst%3DJ.%2BD.%26atitle%3DFunctional%2520contacts%2520with%2520a%2520range%2520of%2520splicing%2520proteins%2520suggest%2520a%2520central%2520role%2520for%2520Brr2p%2520in%2520the%2520dynamic%2520control%2520of%2520the%2520order%2520of%2520events%2520in%2520spliceosomes%2520of%2520Saccharomyces%2520cerevisiae%26jtitle%3DGenetics%26date%3D2001%26volume%3D157%26spage%3D1451%26epage%3D1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">CvaÄkovÃ¡, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MatÄjÅ¯, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">StanÄk, D.</span><span> </span><span class="NLM_article-title">Retinitis pigmentosa mutations of SNRNP200 enhance cryptic splice-site recognition</span> <span class="citation_source-journal">Hum. Mutat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">308</span><span class="NLM_x">â</span> <span class="NLM_lpage">317</span><span class="refDoi">Â DOI: 10.1002/humu.22481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1002%2Fhumu.22481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=24302620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC2cXislems70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=308-317&author=Z.+Cva%C4%8Dkov%C3%A1author=D.+Mat%C4%9Bj%C5%AFauthor=D.+Stan%C4%9Bk&title=Retinitis+pigmentosa+mutations+of+SNRNP200+enhance+cryptic+splice-site+recognition&doi=10.1002%2Fhumu.22481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Retinitis Pigmentosa Mutations of SNRNP200 Enhance Cryptic Splice-Site Recognition</span></div><div class="casAuthors">Cvackova, Zuzana; Mateju, Daniel; Stanek, David</div><div class="citationInfo"><span class="NLM_cas:title">Human Mutation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">308-317</span>CODEN:
                <span class="NLM_cas:coden">HUMUE3</span>;
        ISSN:<span class="NLM_cas:issn">1059-7794</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Mutations in SNRP200 gene cause autosomal-dominant retinal disorder retinitis pigmentosa (RP).  The protein product of SNRNP200 is BRR2, a DExD/H box RNA helicase crucial for pre-mRNA splicing.  In this study, we prepd. p.S1087L and p.R1090L mutations of human BRR2 using bacterial artificial chromosome recombineering and stably expressed them in human cell culture.  Mutations in BRR2 did not compromise snRNP assembly and both mutants were incorporated into the spliceosome just as the wild-type (wt) protein.  Surprisingly, cells expressing RP mutants exhibited increased splicing efficiency of the LDHA gene.  Next, we found that depletion of endogenous BRR2 enhanced usage of a Î²-globin cryptic splice site while splicing at the correct splice site was inhibited.  Proper splicing of optimal and cryptic splice sites was restored in cells expressing BRR2-wt but not in cells expressing RP mutants.  Taken together, our data suggest that BRR2 is an important factor in 5'-splice-site recognition and that the RP-linked mutations c.3260C>T (p.S1087L) and c.3269G>T (p.R1090L) affect this BRR2 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrm8j7wARgaOLVg90H21EOLACvtfcHk0lj_HBsm3KuNHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXislems70%253D&md5=a14881b54c35dfae55511abf31a02550</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fhumu.22481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhumu.22481%26sid%3Dliteratum%253Aachs%26aulast%3DCva%25C4%258Dkov%25C3%25A1%26aufirst%3DZ.%26aulast%3DMat%25C4%259Bj%25C5%25AF%26aufirst%3DD.%26aulast%3DStan%25C4%259Bk%26aufirst%3DD.%26atitle%3DRetinitis%2520pigmentosa%2520mutations%2520of%2520SNRNP200%2520enhance%2520cryptic%2520splice-site%2520recognition%26jtitle%3DHum.%2520Mutat.%26date%3D2014%26volume%3D35%26spage%3D308%26epage%3D317%26doi%3D10.1002%2Fhumu.22481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Zhao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellur, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grassi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowne, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daiger, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pang, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staley, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, C.</span><span> </span><span class="NLM_article-title">Autosomal-dominant retinitis pigmentosa caused by a mutation in SNRNP200, a gene required for unwinding of U4/U6 snRNAs</span> <span class="citation_source-journal">Am. J. Hum. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">85</span><span class="NLM_x">, </span> <span class="NLM_fpage">617</span><span class="NLM_x">â</span> <span class="NLM_lpage">627</span><span class="refDoi">Â DOI: 10.1016/j.ajhg.2009.09.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.ajhg.2009.09.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=19878916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFWruw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2009&pages=617-627&author=C.+Zhaoauthor=D.+L.+Bellurauthor=S.+Luauthor=F.+Zhaoauthor=M.+A.+Grassiauthor=S.+J.+Bowneauthor=L.+S.+Sullivanauthor=S.+P.+Daigerauthor=L.+J.+Chenauthor=C.+P.+Pangauthor=K.+Zhaoauthor=J.+P.+Staleyauthor=C.+Larsson&title=Autosomal-dominant+retinitis+pigmentosa+caused+by+a+mutation+in+SNRNP200%2C+a+gene+required+for+unwinding+of+U4%2FU6+snRNAs&doi=10.1016%2Fj.ajhg.2009.09.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Autosomal-dominant retinitis pigmentosa caused by a mutation in SNRNP200, a gene required for unwinding of U4/U6 snRNAs</span></div><div class="casAuthors">Zhao, Chen; Bellur, Deepti L.; Lu, Shasha; Zhao, Feng; Grassi, Michael A.; Bowne, Sara J.; Sullivan, Lori S.; Daiger, Stephen P.; Chen, Li Jia; Pang, Chi Pui; Zhao, Kanxing; Staley, Jonathan P.; Larsson, Catharina</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">617-627</span>CODEN:
                <span class="NLM_cas:coden">AJHGAG</span>;
        ISSN:<span class="NLM_cas:issn">0002-9297</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mutations in genes assocd. with the U4/U6-U5 small nuclear ribonucleoprotein (snRNP) complex of the spliceosome are implicated in autosomal-dominant retinitis pigmentosa (adRP), a group of progressive retinal degenerative disorders leading to visual impairment, loss of visual field, and even blindness.  We recently assigned a locus (RP33) for adRP to 2cen-q12.1, a region that harbors the SNRNP200 gene encoding hBrr2, another U4/U6-U5 snRNP component that is required for unwinding of U4/U6 snRNAs during spliceosome activation and for disassembly of the spliceosome.  Here, we report the identification of a missense mutation, c.3260C>T (p.S1087L), in exon 25 of the SNRNP200 gene in an RP33-linked family.  The c.3260C>T substitution showed complete cosegregation with the retinitis pigmentosa (RP) phenotype over four generations, but was absent in a panel of 400 controls.  The p.S1087L mutation and p.R1090L, another adRP-assocd. allele, reside in the "ratchet" helix of the first of two Sec63 domains implicated in the directionality and processivity of nucleic acid unwinding.  Indeed, marked defects in U4/U6 unwinding, but not U4/U6-U5 snRNP assembly, were obsd. in budding yeast for the analogous mutations (N1104L and R1107L) of the corresponding Brr2p residues.  The linkage of hBrr2 to adRP suggests that the mechanism of pathogenesis for splicing-factor-related RP may fundamentally derive from a defect in hBrr2-dependent RNA unwinding and a consequent defect in spliceosome activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX7wOyj5QzPrVg90H21EOLACvtfcHk0lj_HBsm3KuNHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFWruw%253D%253D&md5=273111d5d7d4bc6fb087e31b9100c92a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ajhg.2009.09.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajhg.2009.09.020%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DBellur%26aufirst%3DD.%2BL.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DGrassi%26aufirst%3DM.%2BA.%26aulast%3DBowne%26aufirst%3DS.%2BJ.%26aulast%3DSullivan%26aufirst%3DL.%2BS.%26aulast%3DDaiger%26aufirst%3DS.%2BP.%26aulast%3DChen%26aufirst%3DL.%2BJ.%26aulast%3DPang%26aufirst%3DC.%2BP.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DStaley%26aufirst%3DJ.%2BP.%26aulast%3DLarsson%26aufirst%3DC.%26atitle%3DAutosomal-dominant%2520retinitis%2520pigmentosa%2520caused%2520by%2520a%2520mutation%2520in%2520SNRNP200%252C%2520a%2520gene%2520required%2520for%2520unwinding%2520of%2520U4%252FU6%2520snRNAs%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2009%26volume%3D85%26spage%3D617%26epage%3D627%26doi%3D10.1016%2Fj.ajhg.2009.09.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Ledoux, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guthrie, C.</span><span> </span><span class="NLM_article-title">Retinitis Pigmentosa mutations in Bad Response to Refrigeration 2 (Brr2) impair ATPase and helicase activity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x">, </span> <span class="NLM_fpage">11954</span><span class="NLM_x">â</span> <span class="NLM_lpage">11965</span><span class="refDoi">Â DOI: 10.1074/jbc.M115.710848</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1074%2Fjbc.M115.710848" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=11954-11965&author=S.+Ledouxauthor=C.+Guthrie&title=Retinitis+Pigmentosa+mutations+in+Bad+Response+to+Refrigeration+2+%28Brr2%29+impair+ATPase+and+helicase+activity&doi=10.1074%2Fjbc.M115.710848"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.710848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.710848%26sid%3Dliteratum%253Aachs%26aulast%3DLedoux%26aufirst%3DS.%26aulast%3DGuthrie%26aufirst%3DC.%26atitle%3DRetinitis%2520Pigmentosa%2520mutations%2520in%2520Bad%2520Response%2520to%2520Refrigeration%25202%2520%2528Brr2%2529%2520impair%2520ATPase%2520and%2520helicase%2520activity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D11954%26epage%3D11965%26doi%3D10.1074%2Fjbc.M115.710848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Shadrick, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ndjomou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frick, D. N.</span><span> </span><span class="NLM_article-title">Discovering new medicines targeting helicases: challenges and recent progress</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">761</span><span class="NLM_x">â</span> <span class="NLM_lpage">781</span><span class="refDoi">Â DOI: 10.1177/1087057113482586</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1177%2F1087057113482586" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=23536547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWitr7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=761-781&author=W.+R.+Shadrickauthor=J.+Ndjomouauthor=R.+Kolliauthor=S.+Mukherjeeauthor=A.+M.+Hansonauthor=D.+N.+Frick&title=Discovering+new+medicines+targeting+helicases%3A+challenges+and+recent+progress&doi=10.1177%2F1087057113482586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering new medicines targeting helicases: challenges and recent progress</span></div><div class="casAuthors">Shadrick, William R.; Ndjomou, Jean; Kolli, Rajesh; Mukherjee, Sourav; Hanson, Alicia M.; Frick, David N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">761-781, 21</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Helicases are ubiquitous motor proteins that sep. and/or rearrange nucleic acid duplexes in reactions fueled by ATP (ATP) hydrolysis.  Helicases encoded by bacteria, viruses, and human cells are widely studied targets for new antiviral, antibiotic, and anticancer drugs.  This review summarizes the biochem. of frequently targeted helicases.  These proteins include viral enzymes from herpes simplex virus, papillomaviruses, polyomaviruses, coronaviruses, the hepatitis C virus, and various flaviviruses.  Bacterial targets examd. include DnaB-like and RecBCD-like helicases.  The human DEAD-box protein DDX3 is the cellular antiviral target discussed, and cellular anticancer drug targets discussed are the human RecQ-like helicases and eIF4A.  We also review assays used for helicase inhibitor discovery and the most promising and common helicase inhibitor chemotypes, such as nucleotide analogs, polyphenyls, metal ion chelators, flavones, polycyclic arom. polymers, coumarins, and various DNA binding pharmacophores.  Also discussed are common complications encountered while searching for potent helicase inhibitors and possible solns. for these problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruzz0kEpdpZrVg90H21EOLACvtfcHk0lj_HBsm3KuNHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWitr7J&md5=b0fb1a502a37f3eed08bdc090db8a40c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1177%2F1087057113482586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113482586%26sid%3Dliteratum%253Aachs%26aulast%3DShadrick%26aufirst%3DW.%2BR.%26aulast%3DNdjomou%26aufirst%3DJ.%26aulast%3DKolli%26aufirst%3DR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DHanson%26aufirst%3DA.%2BM.%26aulast%3DFrick%26aufirst%3DD.%2BN.%26atitle%3DDiscovering%2520new%2520medicines%2520targeting%2520helicases%253A%2520challenges%2520and%2520recent%2520progress%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D761%26epage%3D781%26doi%3D10.1177%2F1087057113482586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Ogino, M.; Ikeda, Z.; Fujimoto, J.; Ohba, Y.; Ishii, N.; Fujimoto, T.; Oda, T.; Taya, N.; Yamashita, T.; Matsunaga, N.</span><span> </span><span class="NLM_article-title">Heterocyclic compound</span>. PCT Int. Appl. WO2014030743 A1, February 27,<span class="NLM_x"> </span><span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=M.+Ogino&author=Z.+Ikeda&author=J.+Fujimoto&author=Y.+Ohba&author=N.+Ishii&author=T.+Fujimoto&author=T.+Oda&author=N.+Taya&author=T.+Yamashita&author=N.+Matsunaga&title=Heterocyclic+compound"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOgino%26aufirst%3DM.%26atitle%3DHeterocyclic%2520compound%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="note"><p class="first last">Since helicase RNA binding sites are less similar to each other, one might expect RNA site binder would have higher selectivity. However, it was not correct at least in our case. The reason for the lower selectivity of <b>12</b> is still unclear, and further investigation such as cocrystal structure analysis with other helicases is necessary.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="note"><p class="first last">We could not rule out the possibility of the formation of hydrogen bonding interaction with Lys2500 in place of Thr1197 so far.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Wallace, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyssikatos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarski, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingston, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otten, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hingorani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woessner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winkler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, K.</span><span> </span><span class="NLM_article-title">Potent and selective mitogen-activated protein kinase kinase (MEK) 1,2 inhibitors. 1. 4-(4-bromo-2-fluorophenylamino)-1- methylpyridin-2(1H)-ones</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">441</span><span class="NLM_x">â</span> <span class="NLM_lpage">444</span><span class="refDoi">Â DOI: 10.1021/jm050834y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050834y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=441-444&author=E.+M.+Wallaceauthor=J.+Lyssikatosauthor=J.+F.+Blakeauthor=J.+Seoauthor=H.+W.+Yangauthor=T.+C.+Yehauthor=M.+Perrierauthor=H.+Jarskiauthor=V.+Marshauthor=G.+Pochauthor=M.+G.+Livingstonauthor=J.+Ottenauthor=G.+Hingoraniauthor=R.+Woessnerauthor=P.+Leeauthor=J.+Winklerauthor=K.+Koch&title=Potent+and+selective+mitogen-activated+protein+kinase+kinase+%28MEK%29+1%2C2+inhibitors.+1.+4-%284-bromo-2-fluorophenylamino%29-1-+methylpyridin-2%281H%29-ones&doi=10.1021%2Fjm050834y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm050834y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050834y%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DE.%2BM.%26aulast%3DLyssikatos%26aufirst%3DJ.%26aulast%3DBlake%26aufirst%3DJ.%2BF.%26aulast%3DSeo%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%2BW.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DPerrier%26aufirst%3DM.%26aulast%3DJarski%26aufirst%3DH.%26aulast%3DMarsh%26aufirst%3DV.%26aulast%3DPoch%26aufirst%3DG.%26aulast%3DLivingston%26aufirst%3DM.%2BG.%26aulast%3DOtten%26aufirst%3DJ.%26aulast%3DHingorani%26aufirst%3DG.%26aulast%3DWoessner%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DWinkler%26aufirst%3DJ.%26aulast%3DKoch%26aufirst%3DK.%26atitle%3DPotent%2520and%2520selective%2520mitogen-activated%2520protein%2520kinase%2520kinase%2520%2528MEK%2529%25201%252C2%2520inhibitors.%25201.%25204-%25284-bromo-2-fluorophenylamino%2529-1-%2520methylpyridin-2%25281H%2529-ones%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D441%26epage%3D444%26doi%3D10.1021%2Fjm050834y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Kai, H.; Endoh, T.; Jikihara, S.; Asahi, K.; Horiguchi, T.</span><span> </span><span class="NLM_article-title">Novel heterocyclic derivatives and pharmaceutical composition containing same</span>. PCT Int. Appl. WO2012020742 A1, February 16,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=H.+Kai&author=T.+Endoh&author=S.+Jikihara&author=K.+Asahi&author=T.+Horiguchi&title=Novel+heterocyclic+derivatives+and+pharmaceutical+composition+containing+same"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKai%26aufirst%3DH.%26atitle%3DNovel%2520heterocyclic%2520derivatives%2520and%2520pharmaceutical%2520composition%2520containing%2520same%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Ito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwatani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sogabe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aparicio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakanishi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imaeda, Y.</span><span> </span><span class="NLM_article-title">Discovery of selective ATP-competitive eIF4A3 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2200</span><span class="NLM_x">â</span> <span class="NLM_lpage">2209</span><span class="refDoi">Â DOI: 10.1016/j.bmc.2017.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1016%2Fj.bmc.2017.02.035" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2200-2209&author=M.+Itoauthor=M.+Iwataniauthor=Y.+Kamadaauthor=S.+Sogabeauthor=S.+Nakaoauthor=T.+Tanakaauthor=T.+Kawamotoauthor=S.+Aparicioauthor=A.+Nakanishiauthor=Y.+Imaeda&title=Discovery+of+selective+ATP-competitive+eIF4A3+inhibitors&doi=10.1016%2Fj.bmc.2017.02.035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DM.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DNakao%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DKawamoto%26aufirst%3DT.%26aulast%3DAparicio%26aufirst%3DS.%26aulast%3DNakanishi%26aufirst%3DA.%26aulast%3DImaeda%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520selective%2520ATP-competitive%2520eIF4A3%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2200%26epage%3D2209%26doi%3D10.1016%2Fj.bmc.2017.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Nakashima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeda, Y.</span><span> </span><span class="NLM_article-title">Impact of physicochemical profiling for rational approach on drug discovery</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">1228</span><span class="NLM_x">â</span> <span class="NLM_lpage">1238</span><span class="refDoi">Â DOI: 10.1248/cpb.c13-00436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1248%2Fcpb.c13-00436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=24292785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2013&pages=1228-1238&author=S.+Nakashimaauthor=K.+Yamamotoauthor=Y.+Araiauthor=Y.+Ikeda&title=Impact+of+physicochemical+profiling+for+rational+approach+on+drug+discovery&doi=10.1248%2Fcpb.c13-00436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of physicochemical profiling for rational approach on drug discovery</span></div><div class="casAuthors">Nakashima, Shoko; Yamamoto, Katsuhiko; Arai, Yuta; Ikeda, Yukihiro</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1228-1238</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Solid-state characterization plays a vital role in lead optimization and candidate selection with the appropriate physicochem. properties for proper oral dosage formulation.  Aq. soly. is an important parameter in the successful development of oral dosage formulation since poor aq. soly. limits absorption.  In this study, we summarized an efficient approach using a small amt. of sample for solid-state characterization, including thermodn. soly., which is defined as physicochem. profiling.  By using the physicochem. profiling results of 75 anti-cancer drugs and clin. candidates, we examd. the relationship between thermodn. soly. and mol. structural parameters and assessed the effects of thermodn. soly. on pharmacokinetic profile for rational sol. drug design.  The Log DpH 7.4, arom. ring count, and hydrogen bond count were good indicators for predicting sparingly sol. compds. that increase the lattice energy because of Ï-Ï stacking and hydrogen bonds, resulting in lowered thermodn. soly.  The level of thermodn. soly. in simulated intestinal fluid (pH 6.8) in the presence and absence of bile acid, which is required for minimal acceptable bioavailability (>30%), was 1 Î¼g/mL and 10 Î¼g/mL, resp.  Physicochem. profiling, which includes thermodn. soly. considering its solid-state properties, contributes to rational lead optimization and efficient candidate selection for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBXTGzqqcJSbVg90H21EOLACvtfcHk0lhVYXy7cPZF6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D&md5=c1291e838e8509c2fe2a1edfaf509ed4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c13-00436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c13-00436%26sid%3Dliteratum%253Aachs%26aulast%3DNakashima%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DArai%26aufirst%3DY.%26aulast%3DIkeda%26aufirst%3DY.%26atitle%3DImpact%2520of%2520physicochemical%2520profiling%2520for%2520rational%2520approach%2520on%2520drug%2520discovery%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2013%26volume%3D61%26spage%3D1228%26epage%3D1238%26doi%3D10.1248%2Fcpb.c13-00436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Masumoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeyasu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oizumi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span> </span><span class="NLM_article-title">Studies of novel 1,4-dihydropyridine Ca antagonist CS-905. I. Measurement of partition coefficient (log P) by high performance liquid chromatography (HPLC)</span> <span class="citation_source-journal">Yakugaku Zasshi</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">â</span> <span class="NLM_lpage">220</span><span class="refDoi">Â DOI: 10.1248/yakushi1947.115.3_213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1248%2Fyakushi1947.115.3_213" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1995&pages=213-220&author=K.+Masumotoauthor=A.+Takeyasuauthor=K.+Oizumiauthor=T.+Kobayashi&title=Studies+of+novel+1%2C4-dihydropyridine+Ca+antagonist+CS-905.+I.+Measurement+of+partition+coefficient+%28log+P%29+by+high+performance+liquid+chromatography+%28HPLC%29&doi=10.1248%2Fyakushi1947.115.3_213"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1248%2Fyakushi1947.115.3_213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fyakushi1947.115.3_213%26sid%3Dliteratum%253Aachs%26aulast%3DMasumoto%26aufirst%3DK.%26aulast%3DTakeyasu%26aufirst%3DA.%26aulast%3DOizumi%26aufirst%3DK.%26aulast%3DKobayashi%26aufirst%3DT.%26atitle%3DStudies%2520of%2520novel%25201%252C4-dihydropyridine%2520Ca%2520antagonist%2520CS-905.%2520I.%2520Measurement%2520of%2520partition%2520coefficient%2520%2528log%2520P%2529%2520by%2520high%2520performance%2520liquid%2520chromatography%2520%2528HPLC%2529%26jtitle%3DYakugaku%2520Zasshi%26date%3D1995%26volume%3D115%26spage%3D213%26epage%3D220%26doi%3D10.1248%2Fyakushi1947.115.3_213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span><span class="refDoi">Â DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljRNnqF5ZuXJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">â</span> <span class="NLM_lpage">1214</span><span class="refDoi">Â DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0ljRNnqF5ZuXJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">â</span> <span class="NLM_lpage">674</span><span class="refDoi">Â DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ÎF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljRNnqF5ZuXJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">â</span> <span class="NLM_lpage">2132</span><span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhkpCFSMKpwIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">â</span> <span class="NLM_lpage">255</span><span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhkpCFSMKpwIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">â</span> <span class="NLM_lpage">221</span><span class="refDoi">Â DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhkpCFSMKpwIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography. Collaborative Computational Project Number 4</span>.  <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">â</span> <span class="NLM_lpage">763</span>, DOI: <span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+CCP4+suite%3A+programs+for+protein+crystallography.+Collaborative+Computational+Project+Number+4.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+1994%2C+50%2C+760%E2%80%93763%2C+DOI%3A+10.1107%2FS0907444994003112."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography.%2520Collaborative%2520Computational%2520Project%2520Number%25204%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">â</span> <span class="NLM_lpage">21</span><span class="refDoi">Â DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljuPnqKpxp0nw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ','PDB','5URJ'); return false;">PDB: 5URJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URM','PDB','5URM'); return false;">PDB: 5URM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F93" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F93','PDB','4F93'); return false;">PDB: 4F93</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URK','PDB','5URK'); return false;">PDB: 5URK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F91" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4F91','PDB','4F91'); return false;">PDB: 4F91</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i43"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00461">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_20288"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00461">10.1021/acs.jmedchem.7b00461</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Scheme to explore Brr2 inhibitors, SAR of compounds <b>11</b>â<b>13</b>, protein expression and purification for crystallization, comparison of the crystal structure of Brr2 with or without compound <b>3</b>, crystal structure of Brr2 in complex with compound <b>12</b>, substrate-competitive assay for compound <b>9</b>, time course of RNA helicase activity of Brr2, X-ray data collection and refinement statistics, omit electron density of Brr2 inhibitors and the ligand interactions, and helicase inhibitory activity of compound <b>12</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all the compounds reported in this study (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_002.csv">CSV</a>)</p></li><li><p class="inline">Coordinate file of docking model for compound <b>9</b> bound to Brr2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_001.pdf">jm7b00461_si_001.pdf (1.36 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_002.csv">jm7b00461_si_002.csv (1.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00461/suppl_file/jm7b00461_si_003.pdb">jm7b00461_si_003.pdb (2.14 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes are the following: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URJ">5URJ</a> (compound <b>3</b>); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URK">5URK</a> (compound <b>6</b>); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5URM">5URM</a> (compound <b>12</b>). Authors will release the atomatic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00461&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-13%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00461%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00461" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67998d7d3f703d82","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
