
<html lang="en"     class="pb-page"  data-request-id="0736b8d2-f88b-4f71-bf60-77f7772899b6"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-9;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b02164;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery, Structure–Activity Relationship, and Biological Activity of Histone-Competitive Inhibitors of Histone Acetyltransferases P300/CBP" /></meta><meta name="dc.Creator" content="Fangrui  Wu" /></meta><meta name="dc.Creator" content="Yuanda  Hua" /></meta><meta name="dc.Creator" content="Salma  Kaochar" /></meta><meta name="dc.Creator" content="Shenyou  Nie" /></meta><meta name="dc.Creator" content="Yi-Lun  Lin" /></meta><meta name="dc.Creator" content="Yuan  Yao" /></meta><meta name="dc.Creator" content="Jingyu  Wu" /></meta><meta name="dc.Creator" content="Xiaowei  Wu" /></meta><meta name="dc.Creator" content="Xiaoyong  Fu" /></meta><meta name="dc.Creator" content="Rachel  Schiff" /></meta><meta name="dc.Creator" content="Christel M.  Davis" /></meta><meta name="dc.Creator" content="Matthew  Robertson" /></meta><meta name="dc.Creator" content="Erik A.  Ehli" /></meta><meta name="dc.Creator" content="Cristian  Coarfa" /></meta><meta name="dc.Creator" content="Nicholas  Mitsiades" /></meta><meta name="dc.Creator" content="Yongcheng  Song" /></meta><meta name="dc.Description" content="Histone acetyltransferase (HAT) p300 and its paralog CBP acetylate histone lysine side chains and play critical roles in regulating gene transcription. The HAT domain of p300/CBP is a potential dru..." /></meta><meta name="Description" content="Histone acetyltransferase (HAT) p300 and its paralog CBP acetylate histone lysine side chains and play critical roles in regulating gene transcription. The HAT domain of p300/CBP is a potential dru..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 21, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b02164" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02164" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b02164" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02164" /></link>
        
    
    

<title>Discovery, Structure–Activity Relationship, and Biological Activity of Histone-Competitive Inhibitors of Histone Acetyltransferases P300/CBP | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b02164" /></meta><meta property="og:title" content="Discovery, Structure–Activity Relationship, and Biological Activity of Histone-Competitive Inhibitors of Histone Acetyltransferases P300/CBP" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0009.jpeg" /></meta><meta property="og:description" content="Histone acetyltransferase (HAT) p300 and its paralog CBP acetylate histone lysine side chains and play critical roles in regulating gene transcription. The HAT domain of p300/CBP is a potential drug target for cancer. Through compound screening and medicinal chemistry, novel inhibitors of p300/CBP HAT with their IC50 values as low as 620 nM were discovered. The most potent inhibitor is competitive against histone substrates and exhibits a high selectivity for p300/CBP. It inhibited cellular acetylation and had strong activity with EC50 of 1–3 μM against proliferation of several tumor cell lines. Gene expression profiling in estrogen receptor (ER)-positive breast cancer MCF-7 cells showed that inhibitor treatment recapitulated siRNA-mediated p300 knockdown, inhibited ER-mediated gene transcription, and suppressed expression of numerous cancer-related gene signatures. These results demonstrate that the inhibitor is not only a useful probe for biological studies of p300/CBP HAT but also a pharmacological lead for further drug development targeting cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b02164"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02164">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b02164&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b02164&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b02164&amp;href=/doi/10.1021/acs.jmedchem.9b02164" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4716-4731</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02161" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00003" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery, Structure–Activity Relationship, and Biological Activity of Histone-Competitive Inhibitors of Histone Acetyltransferases P300/CBP</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Fangrui Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fangrui Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology & Chemical Biology, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fangrui++Wu">Fangrui Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuanda Hua</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuanda Hua</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology & Chemical Biology, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuanda++Hua">Yuanda Hua</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Salma Kaochar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Salma Kaochar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine,  Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Salma++Kaochar">Salma Kaochar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shenyou Nie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shenyou Nie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology & Chemical Biology, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shenyou++Nie">Shenyou Nie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yi-Lun Lin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi-Lun Lin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology & Chemical Biology, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi-Lun++Lin">Yi-Lun Lin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuan Yao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuan Yao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology & Chemical Biology, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuan++Yao">Yuan Yao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jingyu Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jingyu Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology & Chemical Biology, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jingyu++Wu">Jingyu Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaowei Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaowei Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology & Chemical Biology, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaowei++Wu">Xiaowei Wu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7248-9149" title="Orcid link">http://orcid.org/0000-0001-7248-9149</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaoyong Fu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoyong Fu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular & Cellular Biology,  Lester and Sue Smith Breast Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyong++Fu">Xiaoyong Fu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rachel Schiff</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rachel Schiff</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine,  Department of Molecular & Cellular Biology,  Lester and Sue Smith Breast Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rachel++Schiff">Rachel Schiff</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christel M. Davis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christel M. Davis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Avera Institute for Human Genetics, Sioux Falls, South Dakota 57108, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christel+M.++Davis">Christel M. Davis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew Robertson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew Robertson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew++Robertson">Matthew Robertson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Erik A. Ehli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Erik A. Ehli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Avera Institute for Human Genetics, Sioux Falls, South Dakota 57108, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Erik+A.++Ehli">Erik A. Ehli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cristian Coarfa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cristian Coarfa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cristian++Coarfa">Cristian Coarfa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas Mitsiades</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas Mitsiades</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicine,  Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas++Mitsiades">Nicholas Mitsiades</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Yongcheng Song</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yongcheng Song</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology & Chemical Biology,  Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: 713-798-7415. Email: <a href="/cdn-cgi/l/email-protection#85fcf6eaebe2c5e7e6e8abe0e1f0"><span class="__cf_email__" data-cfemail="1d646e72737a5d7f7e7033787968">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yongcheng++Song">Yongcheng Song</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2611-2476" title="Orcid link">http://orcid.org/0000-0003-2611-2476</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02164&amp;href=/doi/10.1021%2Facs.jmedchem.9b02164" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4716–4731</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 21, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 December 2019</li><li><span class="item_label"><b>Published</b> online</span>21 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b02164" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02164</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4716%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFangrui%2BWu%252C%2BYuanda%2BHua%252C%2BSalma%2BKaochar%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D9%26contentID%3Dacs.jmedchem.9b02164%26title%3DDiscovery%252C%2BStructure%25E2%2580%2593Activity%2BRelationship%252C%2Band%2BBiological%2BActivity%2Bof%2BHistone-Competitive%2BInhibitors%2Bof%2BHistone%2BAcetyltransferases%2BP300%252FCBP%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4731%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b02164"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1365</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b02164" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery, Structure–Activity Relationship, and Biological Activity of Histone-Competitive Inhibitors of Histone Acetyltransferases P300/CBP&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Fangrui&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Yuanda&quot;,&quot;last_name&quot;:&quot;Hua&quot;},{&quot;first_name&quot;:&quot;Salma&quot;,&quot;last_name&quot;:&quot;Kaochar&quot;},{&quot;first_name&quot;:&quot;Shenyou&quot;,&quot;last_name&quot;:&quot;Nie&quot;},{&quot;first_name&quot;:&quot;Yi-Lun&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Yuan&quot;,&quot;last_name&quot;:&quot;Yao&quot;},{&quot;first_name&quot;:&quot;Jingyu&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Xiaowei&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Xiaoyong&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Rachel&quot;,&quot;last_name&quot;:&quot;Schiff&quot;},{&quot;first_name&quot;:&quot;Christel&quot;,&quot;last_name&quot;:&quot;M. Davis&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;Robertson&quot;},{&quot;first_name&quot;:&quot;Erik&quot;,&quot;last_name&quot;:&quot;A. Ehli&quot;},{&quot;first_name&quot;:&quot;Cristian&quot;,&quot;last_name&quot;:&quot;Coarfa&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;Mitsiades&quot;},{&quot;first_name&quot;:&quot;Yongcheng&quot;,&quot;last_name&quot;:&quot;Song&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;4716-4731&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b02164&quot;},&quot;abstract&quot;:&quot;Histone acetyltransferase (HAT) p300 and its paralog CBP acetylate histone lysine side chains and play critical roles in regulating gene transcription. The HAT domain of p300/CBP is a potential drug target for cancer. Through compound screening and medicinal chemistry, novel inhibitors of p300/CBP HAT with their IC50 values as low as 620 nM were discovered. The most potent inhibitor is competitive against histone substrates and exhibits a high selectivity for p300/CBP. It inhibited cellular acetylation and had strong activity with EC50 of 1–3 μM against proliferation of several tumor cell lines. Gene expression profiling in estrogen receptor (ER)-positive breast cancer MCF-7 cells showed that inhibitor treatment recapitulated siRNA-mediated p300 knockdown, inhibited ER-mediated gene transcription, and suppressed expression of numerous cancer-related gene signatures. These results demonstrate that the inhibitor is not only a useful probe for biological studies of p300/CBP HAT but also a pharmacological lead fo&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02164&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02164" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02164&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02164" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02164&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02164" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02164&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02164&amp;href=/doi/10.1021/acs.jmedchem.9b02164" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b02164" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b02164" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b02164%26sid%3Dliteratum%253Aachs%26pmid%3D32314924%26genre%3Darticle%26aulast%3DWu%26date%3D2020%26atitle%3DDiscovery%252C%2BStructure%25E2%2580%2593Activity%2BRelationship%252C%2Band%2BBiological%2BActivity%2Bof%2BHistone-Competitive%2BInhibitors%2Bof%2BHistone%2BAcetyltransferases%2BP300%252FCBP%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D9%26spage%3D4716%26epage%3D4731%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/jmcmar.2020.63.issue-9/20200514/jmcmar.2020.63.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Histone acetyltransferase (HAT) p300 and its paralog CBP acetylate histone lysine side chains and play critical roles in regulating gene transcription. The HAT domain of p300/CBP is a potential drug target for cancer. Through compound screening and medicinal chemistry, novel inhibitors of p300/CBP HAT with their IC<sub>50</sub> values as low as 620 nM were discovered. The most potent inhibitor is competitive against histone substrates and exhibits a high selectivity for p300/CBP. It inhibited cellular acetylation and had strong activity with EC<sub>50</sub> of 1–3 μM against proliferation of several tumor cell lines. Gene expression profiling in estrogen receptor (ER)-positive breast cancer MCF-7 cells showed that inhibitor treatment recapitulated siRNA-mediated p300 knockdown, inhibited ER-mediated gene transcription, and suppressed expression of numerous cancer-related gene signatures. These results demonstrate that the inhibitor is not only a useful probe for biological studies of p300/CBP HAT but also a pharmacological lead for further drug development targeting cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45153" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45153" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Histone acetylation is one of the most important post-translational modifications. Acetylation of the side chain amino group of a histone lysine residue neutralizes its positive charge (under physiological pH) and renders a more open DNA conformation to facilitate binding of transcription factors as well as other proteins for gene transcription, DNA replication, and repair.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> In addition, acetylated lysine can be recognized by a number of proteins (such as bromodomain-containing proteins) and serve as an anchor point in chromatin to form a transcription complex that regulates gene expression.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a></div><div class="NLM_p">Human p300 (E1A binding protein p300) and its paralog CBP [CREB (cAMP-response element binding protein) binding protein] are large proteins with ∼2400 amino acids,<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a> containing multiple structured domains including cysteine-histidine rich 1 (CH1), CREB-binding KIX, bromodomain, histone acetyltransferase (HAT), CH3, and steroid receptor coactivator interaction (SID) domains. These structured domains, which share a high homology between p300 and CBP, are connected with less conserved intrinsically disordered regions (IDR). The CH1, KIX, CH3, and SID domains as well as IDR of p300/CBP are transactivation domains, which interact with a number of transcription factors (e.g., CREB, p53, and HIF-1) and transcription coactivators (e.g., steroid receptor coactivators).<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> Biologically, p300/CBP not only acetylates a lysine residue of histone (e.g., histone H3 lysine 27, or H3K27) and certain transcription factors (e.g., p53 and c-Myc)<a onclick="showRef(event, 'ref7 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref11 ref12">(7,11,12)</a> but also serves as a hub protein to link and stabilize a transcription complex.</div><div class="NLM_p">Previous studies have shown that HAT activity of p300/CBP is essential for many transcription factor-mediated gene transcription programs. For example, estrogen receptor (ER) or androgen receptor (AR) mediated gene transcription pathway is of importance in female or male development as well as in breast or prostate cancer, respectively. HAT activity of p300 has been found to be required for ER- or AR-mediated gene expression.<a onclick="showRef(event, 'ref8 ref10 ref13'); return false;" href="javascript:void(0);" class="ref ref8 ref10 ref13">(8,10,13)</a> In addition, HAT of p300/CBP is a potential drug target for cancer therapy. Although there have been debates on whether p300/CBP alone is a tumor suppressor or an oncogene, convincing evidence has shown that p300/CBP HAT is essential for breast and prostate cancer as well as other cancers with p300 overexpression or harboring a p300/CBP fusion oncogene.<a onclick="showRef(event, 'ref7 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref7 ref14 ref15">(7,14,15)</a></div><div class="NLM_p">To date, several chemotypes of p300/CBP HAT inhibitors have been reported (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> However, except for recently disclosed compound A-485,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> other compounds are not drug-like probes for cellular or in vivo studies. Lys-CoA and its analogs are not cell-permeable.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> Compounds C646,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> L002,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and Cpd-2c<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> contain a “PAINS” (pan-assay interfering compound)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> or related structure and are unfavorable for drug development or cell biology. A-485 is a potent inhibitor of p300/CBP HAT, which is competitive against the enzyme cofactor acetyl coenzyme A (Ac-CoA).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> It showed strong activity against proliferation of several cancer cells. Nevertheless, given the low success rate in drug discovery, more chemotypes of p300 HAT inhibitors are needed. Here, we report discovery, structure–activity relationships (SAR),and biochemical and biological activities of a novel series of inhibitors of p300/CBP HAT that are competitive against the histone substrate.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. P300/CBP HAT catalyzed reaction and inhibitors with their IC<sub>50</sub> values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Inhibitor Discovery</h3><div class="NLM_p">A biochemical assay to determine the activity and inhibition of recombinant HAT domain of human p300 was developed, using the substrate histone H3 (1–21) peptide and the <sup>3</sup>H-labeled cofactor Ac-CoA. We screened our proprietary compound library containing ∼300 compounds, which were synthesized for SAR studies of lysine specific demethylase 1 (LSD1)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> whose substrate is methylated histone H3 lysine 4 (H3K4). The rationale is that since the peptide sequence of H3K4 (ARTKQ) is similar to those of p300 HAT’s substrate H3K27 (AARKS), there is a higher likelihood to find an inhibitor from the LSD1 targeting compounds. The compound screen led to the discovery of thiophene-2-carbamide compound <b>1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) that inhibited p300 HAT with an IC<sub>50</sub> (concentration at which the enzyme activity is inhibited by 50%) value of 8.6 μM.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure and Inhibitory Activity of Compounds <b>1</b>–<b>32</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0008.gif" alt="" id="GRAPHIC-d7e427-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center">R<sup>5</sup></th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM) (% inhibition at 10 μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">piperidin-4-ylmethyl</td><td class="colsep0 rowsep0" align="left">8.6 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">4-(furan-3-yl)-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">1.6 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">4-methoxybiphenyl</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">2.8 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">4-(furan-3-yl)-Ph</td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">7.4 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">4-aminomethyl-Ph</td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">10.0 ± 1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">4-(piperidin-1-ylmethyl)-Ph</td><td class="colsep0 rowsep0" align="left">4-(furan-3-yl)-Ph</td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">1.6 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">4-(piperazin-1-ylmethyl)-Ph</td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (12%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">furan-3-yl</td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (36%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">pyridin-3-yl</td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">-CONH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (31%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">4-methoxybiphenyl</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">7.4 ± 1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">4-(furan-3-yl)-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">1.7 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">0.62 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>O-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (0%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">piperidin-4-yl</td><td class="colsep0 rowsep0" align="left">5.0 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">2-(piperidin-4-yl)ethyl</td><td class="colsep0 rowsep0" align="left">2.2 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">3-aminopropyl</td><td class="colsep0 rowsep0" align="left">3.0 ± 1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">6-aminohexyl</td><td class="colsep0 rowsep0" align="left">1.4 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>-</td><td class="colsep0 rowsep0" align="left">piperazin-1-yl</td><td class="colsep0 rowsep0" align="left">>10 (5%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>-</td><td class="colsep0 rowsep0" align="left">morpholin-4-yl</td><td class="colsep0 rowsep0" align="left">>10 (20%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>N(Me)-</td><td class="colsep0 rowsep0" align="left">piperidin-4-ylmethyl</td><td class="colsep0 rowsep0" align="left">4.4 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>N(CHO)-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">7.0 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">4-<i>i</i>-Pr-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">5.4 ± 1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">4-Bu-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (10%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">4-(furan-3-yl)-Ph</td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">5.8 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">4-(furan-3-yl)-Ph</td><td class="colsep0 rowsep0" align="left">4-(piperidin-1-ylmethyl)-Ph</td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">4.6 ± 1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">4-aminomethyl-Ph</td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">2.0 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">4-aminomethyl-Ph</td><td class="colsep0 rowsep0" align="left">4-<i>t</i>-Bu-Ph</td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (35%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">4-aminomethyl-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (0%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">4-hydroxymethyl-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (22%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">furan-3-yl</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (23%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">3,4-dimethoxy-Ph</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (0%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">3-biphenyl</td><td class="colsep0 rowsep0" align="left">= R<sup>4</sup></td><td class="colsep0 rowsep0" align="left">-CH<sub>2</sub>NH-</td><td class="colsep0 rowsep0" align="left">same as above</td><td class="colsep0 rowsep0" align="left">>10 (18%)</td></tr></tbody></table></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis</h3><div class="NLM_p">Given its seemingly drug-like structure, more derivatives of compound <b>1</b> were synthesized and tested for their activities against p300 HAT. General methods for synthesis of compounds <b>1</b>–<b>40</b> for SAR studies are shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Methods for Thiophene-Containing Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide, 1-hydroxybenzotriazole, 4-(aminomethyl)-1-BOC-piperidine, CH<sub>2</sub>Cl<sub>2</sub>, 96% yield; (ii) Ar<sup>1</sup>-B(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 80 °C, 80–90% yield; (iii) Ar<sup>2</sup>-B(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 100 °C, 80–90% yield; (iv) HCl in 1,4-dioxane, >90% yield; (v) NaBH<sub>4</sub>, MeOH; (vi) cyanuric chloride, DMF, 81% yield for the 2 steps; (vii) 4-(aminomethyl)-1-BOC-piperidine or other amine analogs, K<sub>2</sub>CO<sub>3</sub>, DMF, 60–85% yield; (viii) <i>N</i>-bromosuccinimide, CH<sub>2</sub>Cl<sub>2</sub>; (ix) NaClO<sub>2</sub>, NaH<sub>2</sub>PO<sub>4</sub>, <i>tert</i>-butanol-H<sub>2</sub>O, 2-methyl-2-butene; (x) 2,3-diaminopropanoic acid, NaOH, MeOH, reflux, 55% yield; (xi) ClCO<sub>2</sub>Me, (<i>i</i>Pr)<sub>2</sub>NEt, THF, 95% yield.</p></p></figure><div class="NLM_p">Synthesis of thiophene-2-carbamide compounds <b>1</b>–<b>9</b> was started from a reaction between 4,5-dibromothiophene-2-carboxylic acid (<b>41</b>) and 1-<i>tert</i>-butyloxycarbonyl (BOC) protected 4-(aminomethyl)piperidine, together with amide bond forming reagents 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDC) and 1-hydroxybenzotriazole. The 5- and 4-Br groups of the thiophene-2-carbamide compounds (<b>42</b>) can undergo two successive Suzuki coupling reactions with different aryl boronic acids with a high selectivity:<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> 5-Br was reacted first at a lower temperature of 80 °C, followed by 4-Br at 100 °C, to give compound <b>43</b>. Removal of the BOC protecting group using HCl produced the target compounds <b>1</b>–<b>9</b>. Synthesis of 2-aminomethylthiophene series of compounds <b>10</b>–<b>12</b> and <b>14</b>–<b>32</b> was started from 4,5-dibromothiophene-2-carbaldehyde (<b>44</b>, X = S), in which the 2-CHO group was reduced to an alcohol, followed by conversion to a -CH<sub>2</sub>Cl group by treatment with cyanuric chloride in DMF. The -Cl of the product <b>45</b> was next substituted with 4-(aminomethyl)-1-BOC-piperidine or other amine analogs. The resulting 2-substituted 4,5-dibromothiophene compound <b>46</b> was subjected to two selective Suzuki coupling reactions to produce, upon deprotection of BOC, the target 2-aminomethylthiophene series of compounds.</div><div class="NLM_p last">Furan-containing inhibitors <b>34</b> and <b>35</b> were prepared similarly from 4,5-dibromofuran-2-carbaldehyde (<b>44</b>, X = O). All thiazole compounds were synthesized from 4-bromothiazole-2-carbaldehyde (<b>48</b>), which was treated with <i>N</i>-bromosuccinimide to produce di-Br compound <b>49</b>. The aldehyde group of <b>48</b> and <b>49</b> was reduced and converted to a -CH<sub>2</sub>Cl group in compound <b>50</b>, which was substituted with 1-BOC-4-(aminomethyl)piperidine followed by Suzuki coupling reactions to give, after deprotection, compounds <b>37</b> and <b>38</b>. Oxidation of <b>48</b> (or <b>49</b>) produced thiazole-2-carboxylic acid <b>51</b>, which was reacted with 1-BOC-4-(aminomethyl)piperidine followed by Suzuki coupling reactions to afford thiazole compound <b>39</b> (or <b>36</b>). For the synthesis of the pyrazine compound <b>40</b>, di-4-bromobenzil (<b>52</b>) was refluxed with 2,3-diaminopropanoic acid in methanol to give 5,6-di(4-bromophenyl)pyrazine-2-carboxylic acid (<b>53</b>), whose -CO<sub>2</sub>H group was acylated, reduced (with NaBH<sub>4</sub>) to -CH<sub>2</sub>OH, and converted to -CH<sub>2</sub>Cl (<b>54</b>). A substitution reaction with 4-(aminomethyl)-1-BOC-piperidine followed by Suzuki coupling reactions produced compound <b>55</b>, which was treated with HCl to give compound <b>40</b>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Activity Relationship</h3><div class="NLM_p">The structures and inhibitory activities of compounds <b>1</b>–<b>32</b> are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf" class="ext-link">Supporting Information Charts S1–S3</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf" class="ext-link">Figure S1</a>. Compound <b>1</b>, having 4,5-disubstituted <i>tert</i>-butyl groups as well as a piperidine-containing amide group, had a moderate IC<sub>50</sub> of 8.6 μM. Compound <b>2</b> with 4,5-disubstituted furan-3-yl groups was found to have significantly enhanced inhibitory activity with an IC<sub>50</sub> of 1.6 μM, while compound <b>3</b> with two 4-methoxybiphenyl substituents showed slightly reduced inhibitory activity (IC<sub>50</sub> = 2.8 μM). Compound <b>4</b> bearing a mixed <i>tert</i>-butyl and furanyl group also exhibited a compromised activity (IC<sub>50</sub> = 7.4 μM). Compound <b>5</b> with a polar 4-aminomethylphenyl substituent rendered little activity change (IC<sub>50</sub> = 10 μM). However, compound <b>6</b> with a piperidine-containing 4-substituent, which is less polar as compared to the polar primary amine in <b>5</b>, showed an improved activity (IC<sub>50</sub> = 1.6 μM). Compound <b>7</b> having a piperazine group was found to be almost inactive at 10 μM, showing an additional amino group (as compared to <b>6</b>) is likely disfavored. Compounds <b>8</b> and <b>9</b> with no substituents in their 4-aromatic rings were found to have modest activity, inhibiting p300-HAT by ∼30% at 10 μM, suggesting a substituent (e.g., furan-3-yl in <b>4</b>) at this position is favorable.</div><div class="NLM_p">Next, activity optimization of the 2-carboxamide side chain was performed for compounds <b>10</b>–<b>21</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf" class="ext-link">Chart S2</a>). The first attempt was to replace the amide side chain (e.g., in <b>1</b> and <b>2</b>) with a piperidin-4-yl-methylaminomethyl group in compounds <b>10</b>–<b>12</b>. While removal of the carbonyl group reduces the activity of compound <b>10</b> (IC<sub>50</sub> = 7.4 μM) by ∼50% (as compared to <b>3</b>), compound <b>11</b> with an IC<sub>50</sub> of 1.7 μM exhibited a comparable activity to that of <b>2</b>. Of interest is that compound <b>12</b> showed a potent activity against p300-HAT with an IC<sub>50</sub> of 620 nM, which is ∼13× more active than its amide analog <b>1</b>. Changing the -NH- linkage (in <b>10</b>–<b>12</b>) to an -O- in compound <b>13</b> resulted in a complete loss of inhibitory activity, suggesting the -NH- (which is protonated at physiological pH) could be a key hydrogen bond donor. Compounds <b>14</b> and <b>15</b>, whose 2-substituents are one -CH<sub>2</sub>- shorter and longer than that of <b>12</b>, respectively, were found to possess ∼8- and 3-fold activity reduction (IC<sub>50</sub> = 5.0 and 2.2 μM). Compounds <b>16</b> and <b>17</b>, which carry a linear ω-aminopropyl and ω-aminohexyl side chain, respectively, are also strong inhibitors with IC<sub>50</sub> values of 3.0 and 1.4 μM. Loss of activity was found for compounds <b>18</b> and <b>19</b> bearing a 2-substituent of piperazin-1-ylmethyl and morpholin-4-ylmethyl group, showing shorter cyclic 2-substituents are disfavored. Moreover, methylation and formylation of the middle -NH- group (in compound <b>12</b>) produced compounds <b>20</b> and <b>21</b>, respectively, which were found to have less inhibitory activities with IC<sub>50</sub> values of 4.4 and 7.0 μM. These results suggest an -NH- or -CONH- linkage, which might be a hydrogen bond donor, is of importance to the inhibition.</div><div class="NLM_p">Compounds <b>22</b>–<b>32</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf" class="ext-link">Chart S3</a>) were synthesized to evaluate effects of varying 4- and/or 5-substituents of the most potent inhibitor <b>12</b>. Replacing the <i>tert</i>-butyl with isopropyl groups in <b>22</b> (IC<sub>50</sub> = 5.4 μM) resulted in ∼8× activity reduction, suggesting a bulkier group is favorable for the 4- and 5-positions. Loss of activity for compound <b>23</b> with two <i>n</i>-butyl groups at these positions showed that long alkyl substituents are disfavored. Changing the 4-substituent to a furan-3-yl group in compound <b>24</b> (IC<sub>50</sub> = 5.8 μM) as well as to a piperidin-1-ylmethyl group in <b>25</b> (IC<sub>50</sub> = 4.6 μM) resulted in reduced activities, as compared to compounds <b>12</b> and <b>11</b>. Compound <b>26</b> with a polar -CH<sub>2</sub>NH<sub>2</sub> group at the 5-position (of the thiophene core) is still a strong inhibitor with an IC<sub>50</sub> of 2.0 μM, although it exhibited ∼3-fold activity reduction as compared to <b>12</b>. However, moving the -CH<sub>2</sub>NH<sub>2</sub> group to the 4-position in compound <b>27</b> suffered a major activity loss. Compound <b>28</b> with the amine group at both positions is inactive. Similarly, compound <b>29</b> with two -CH<sub>2</sub>OH groups is also a weak inhibitor. Activities for compounds <b>26</b>–<b>29</b> suggest polar groups are not favorable at the 4- or 5-position. In addition, very low activity for compound <b>30</b> suggests a smaller aromatic ring at 4- and 5-positions is unfavored. Lack of inhibition for compounds <b>31</b> and <b>32</b> with <i>meta</i>-substitutions at the 4- and 5-positions indicated that <i>para</i>-substitution (e.g., those in compound <b>10</b>) is more favorable for p300 HAT inhibition. Compound <b>33</b> (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) was synthesized with a -Br substituent at the 3-position of the thiophene core. It turned out to be still a strong inhibitor with an IC<sub>50</sub> of 1.8 μM, while 3-Br reduces the inhibitory activity by ∼3-fold (as compared to <b>12</b>).</div><figure id="cht1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0001.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Structures and Inhibitory Activity of Compounds <b>33</b>–<b>40</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Finally, compounds <b>34</b>–<b>40</b> (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) were synthesized to study SARs for the central core. Furan analogs <b>34</b> and <b>35</b> with IC<sub>50</sub> values of 4.9 and 3.0 μM, respectively, were found to be significantly less active than thiophene compounds <b>12</b> and <b>11</b>, suggesting a furan core is less favored. While the thiazole compound <b>36</b> (IC<sub>50</sub> = 2.4 μM) exhibited an improved activity as compared to its thiophene analog <b>1</b>, the thiazole compound <b>37</b> (IC<sub>50</sub> = 8.6 μM) is more than 13× less active than its thiophene analog <b>12</b>. Loss of activity for compounds <b>38</b> and <b>39</b> shows that the 5-substituent is of importance to the inhibition of p300 HAT. Moreover, compound <b>40</b> with a six-membered pyrazine core was found to be significantly less active than its thiophene analog <b>11</b>. These results suggest thiophene is the most favorable aromatic core structure for this series of inhibitors.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Compound <b>12</b> Is Competitive against Histone</h3><div class="NLM_p">We next performed steady-state enzyme kinetics studies to investigate the mode of inhibition for the most potent inhibitor <b>12</b>. Inhibitory activities of compound <b>12</b> were determined with varying concentrations of the cofactor Ac-CoA and the substrate histone. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A (left), IC<sub>50</sub> values of <b>12</b> did not change with increasing concentrations of Ac-CoA (from 0.5–8 μM, or (0.07–1.2) × <i>K</i><sub>m</sub><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>), indicating compound <b>12</b> is likely a noncompetitive inhibitor against the enzyme cofactor. Its inhibitory activities were found to be reduced in a linear manner (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) when increasing concentrations of the substrate histone peptide (2–100 μM) were used in the assay, suggesting compound <b>12</b> is a competitive inhibitor against the substrate histone.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>12</b> is a competitive inhibitor of p300 HAT against the substrate histone. (A) Plots of IC<sub>50</sub> values of <b>12</b> versus increasing concentrations of (left) Ac-CoA (0.5–8 μM, or (0.07–1.2) × <i>K</i><sub>m</sub>) and (right) histone H3 suggest the inhibitor is noncompetitive against Ac-CoA and competitive against histone. (B) Alpha assay results show compound <b>12</b> can dose-dependently disrupt the binding between p300 HAT and histone H4. But such binding was not affected by inactive compound <b>28</b> as well as A-485 which is an inhibitor competitive against Ac-CoA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Next, we used a reported Alpha (amplified luminescent proximity homogeneous assay) assay<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> to confirm this finding. Upon excitation by a laser beam (680 nm), histone H4(1–21) peptide coated donor beads generate singlet oxygen radicals, which can travel for a very short distance (<200 nm) in the solution and cannot reach the free acceptor beads. The binding between the histone peptide and p300 HAT brings the donor and acceptor beads together, which allows the radicals to activate the p300 HAT coated acceptor beads and produce luminescence at 570 nm. Use of histone H4 is because the assay with histone H3 did not produce a luminescence signal, presumably due to a weaker binding between p300 HAT and histone H3. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, compound <b>12</b> can disrupt the binding between the protein and histone H4 and reduce the Alpha-signal in a dose-dependent manner (IC<sub>50</sub> = 13.5 μM). This is in contrast to Ac-CoA-competitive inhibitor A-485,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> which did not affect the binding between p300 HAT and histone H4. In addition, such binding was not interfered by inactive compound <b>28</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Increasing concentrations of compound <b>12</b> did not significantly affect the Alpha-signals using a biotinylated His6 peptide that can directly link the donor and acceptor beads (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf" class="ext-link">Figure S2</a>), showing <b>12</b> does not interfere with the singlet oxygen radicals or His6 tag. Taken together, these results indicate the inhibitor <b>12</b> is competitive against the substrate histone, which are of interest in that due to a different mode of inhibition, histone-competitive inhibitor <b>12</b> might have a different cellular activity or selectivity, as compared to Ac-CoA-competitive inhibitors A-485 and C646.<a onclick="showRef(event, 'ref16 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref20">(16,20)</a></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Docking Studies</h3><div class="NLM_p">We next performed molecular modeling to find possible binding structures of the most potent inhibitor <b>12</b> in p300 HAT. The program Glide in the Schrödinger small-molecule drug discovery software suite was used for the docking studies, with the crystal structure of human p300 HAT in complex with Ac-CoA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PZS">4PZS</a>)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> as a template.</div><div class="NLM_p">Since compound <b>12</b> is a competitive inhibitor against the substrate histone, Ac-CoA was treated as an integral part of the protein. Upon protein structure preparation as well as docking grid generation, compound <b>12</b> was docked into the protein-Ac-CoA complex, and the results are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. The inhibitor can be favorably docked into the protein, as the 10 distinct docking conformations of compound <b>12</b> with the lowest energies exhibit similar binding features (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). One of the <i>tert</i>-butyl groups of these conformations is predicted to insert into and have favorable hydrophobic interactions with the so-called “lysine channel”, through which the histone lysine side chain inserts into p300 HAT, attacks the carbonyl of Ac-CoA, and gets acetylated. Residues Trp1436, Cys1438, Tyr1446, and Ser1396 form the lysine channel of p300 HAT (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, the <i>tert</i>-butyl group is predicted to be ∼3.5 Å from these residues with favorable van de Waals interactions. The central thiophene core and the two <i>para</i>-<i>tert</i>-butylphenyl groups of these structures have extensive hydrophobic interactions with either upper or lower surface of the protein (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A/B). In addition, the positively charged, two amino groups of many of these docking structures form hydrogen bonds and have favorable electrostatic interactions with Asp1628 and Glu1505, as exemplified in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B. These favorable binding features are consistent with the strong inhibitory activity of compound <b>12</b>.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking results for compound <b>12</b>: (A) 10 docking conformations of <b>12</b> (with C atoms in green) with lowest energies in the crystal structure of p300-HAT (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PZS">4PZS</a>, shown as an electrostatic surface) in complex with Ac-CoA (as a tube model with C atoms in brown); (B) the lowest energy docking conformation of <b>12</b> (as a ball-and-stick model) with surrounding residues and Ac-CoA in p300-HAT (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PZS">4PZS</a>). Hydrogen bonds are shown as yellow dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Moreover, the docking results might be used to rationalize some of the observed SARs. For example, H-bond/electrostatic interactions between -NH- (positively charged at physiological pH) and Asp1628 are predicted to contribute significantly to the binding of compound <b>12</b>. This could explain the loss of activity for compound <b>13</b> with an -O- linkage, which does not favorably interact with the (negatively charged) Asp1628 side chain. Since an amide -CONH- mimics the positively charged -NH- in <b>12</b>, many of the amide compounds (e.g., compounds <b>2</b> and <b>6</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) are potent inhibitors. Docking results also suggest favorable interactions of <b>12</b> with the lysine channel are important for tight binding. This could explain weak or no activities for compounds <b>30</b>–<b>32</b>, which do not have a <i>para</i>-substituent or have a <i>meta</i>-substituent which might not favorably interact with the lysine channel residues of the protein.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Enzyme Selectivity</h3><div class="NLM_p">Humans have three classes of histone/protein lysine acetyltransferases with distinct conserved motifs and structures, including p300/CBP, Gcn5-related <i>N</i>-acetyltransferase (GNAT) and MYST (MOZ, Ybf2, Sas2, and Tip60) family of HATs. Compound <b>12</b> was tested for its activities against selected HATs from these three classes of HATs. CBP is a homolog of p300. The HAT domains of CBP and p300 exhibit 87% identity in sequence. PCAF (p300/CBP associating factor) is a member of the GNAT family, while Myst3 belongs to the MYST family of HATs. P300/CBP HATs have a distinct sequence and three-dimensional structure from the GNAT and MYST HAT proteins. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, compound <b>12</b> was found to be also a strong inhibitor of human CBP HAT with an IC<sub>50</sub> of 1.2 μM, comparable to its IC<sub>50</sub> against p300 HAT. However, compound <b>12</b> did not significantly inhibit the activity of PCAF and Myst3 even at 50 μM, showing a high selectivity for p300/CBP HAT. In addition, since the initial hit compound <b>1</b> was for SAR studies targeting LSD1, both compounds <b>1</b> and <b>12</b> did not significantly inhibit activity of LSD1 at 50 μM.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitory Activities of Compound <b>12</b> against HATs</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P300-HAT</td><td class="colsep0 rowsep0" align="left">0.62 ± 0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CBP-HAT</td><td class="colsep0 rowsep0" align="left">1.2 ± 0.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PCAF</td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Myst3</td><td class="colsep0 rowsep0" align="left">>50</td></tr></tbody></table></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Inhibition of Cellular Histone Acetylation</h3><div class="NLM_p">The ability of compound <b>12</b> to inhibit cellular p300/CBP HAT was evaluated. Also included in the experiment are an inactive compound <b>28</b> and a known inhibitor A-485. Kasumi-1 leukemia cells were treated with increasing concentrations of these compounds for 12 h. Histone was extracted and subjected to electrophoresis separation and Western blot staining to determine levels of acetylation at various lysine residues. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, compound <b>12</b> was found to significantly inhibit cellular acetylation of histone H3K9, H3K18, and H3K27 at 5 and 10 μM in a dose dependent manner. H3K27 seems to be the most sensitive cellular substrate of p300 HAT. Inactive compound <b>28</b> was found to have no significant activity against acetylation of these histone residues. As compared to A-485 (enzyme IC<sub>50</sub> of 60 nM <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>), compound <b>12</b> is less active against acetylation of H3K27 and K18, which is consistent with the enzyme activities of these compounds. However, compound <b>12</b> seems to exhibit more potent cellular activity against acetylation of H3K9. These differences might be due to different mode of inhibition (competitive against histone vs Ac-CoA) or cell permeability for the two compounds.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cellular activity of compounds <b>12</b>, <b>28</b>, and A-485 against acetylation of H3K9, H3K18, and H3K27.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Inhibition of Tumor Cell Proliferation</h3><div class="NLM_p">Activity of compound <b>12</b> was tested against proliferation of several tumor cell lines in which p300/CBP HAT is of importance. First, ER-mediated transcription is critical to ER+ breast cancer. It is an ordered, stepwise assembly of ER and coactivator proteins for gene expression.<a onclick="showRef(event, 'ref10 ref27'); return false;" href="javascript:void(0);" class="ref ref10 ref27">(10,27)</a> Upon binding with an estrogen molecule in cytoplasm, ER undergoes conformational changes, forms a homodimer, and is translocated into the nucleus, where ER binds to estrogen response element, a short segment of DNA within a gene promoter. Next, a steroid receptor coactivator (SRC) protein binds to ER, followed by recruitment of p300 through its SRC-binding SID domain.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> P300 HAT activity is required for the ER-mediated gene expression because a mutant p300 without the enzyme function failed to activate it.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Given the critical role of p300-mediated histone acetylation in the ER signaling pathway, compound <b>12</b> was assessed for its activity against the ER+ breast cancer cell line MCF-7. In addition, ER antagonist tamoxifen is widely used in the clinic to treat ER+ breast cancer by suppressing ER mediated gene transcription. However, many patients eventually develop resistance to tamoxifen and die of the cancer. In our previous work,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> we have developed MCF-7 derivative cells that were made resistant to tamoxifen. Activity of compound <b>12</b> was also tested against tamoxifen-resistant MCF-7 to find whether inhibition of HAT can affect growth of these cells. As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, compound <b>12</b> exhibited strong antiproliferative activity against parent MCF-7 cells (EC<sub>50</sub> = 2.8 μM) as well as tamoxifen-resistant cells (EC<sub>50</sub> = 3.4 μM). These results show that inhibition of p300-mediated histone acetylation can inhibit growth of MCF-7 breast cancer cells and such activity is independent upon the status of tamoxifen resistance, suggesting the p300 HAT inhibitor could have potential clinical applications in breast cancer therapy.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Antiproliferative Activity EC<sub>50</sub> (μM) of Compounds <b>12</b>, <b>28</b>, <b>31</b>, and A-485</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>12</b></th><th class="colsep0 rowsep0" align="center"><b>28</b></th><th class="colsep0 rowsep0" align="center"><b>31</b></th><th class="colsep0 rowsep0" align="center">A-485</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCF-7</td><td class="colsep0 rowsep0" align="left">2.8 ± 0.1</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCF-7 (Tam-R)</td><td class="colsep0 rowsep0" align="left">3.4 ± 0.1</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PANC-1</td><td class="colsep0 rowsep0" align="left">1.0 ± 0.2</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-PANC-28</td><td class="colsep0 rowsep0" align="left">2.8 ± 0.4</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">>30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Kasumi-1</td><td class="colsep0 rowsep0" align="left">2.6 ± 0.6</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">18.7 ± 0.1</td><td class="colsep0 rowsep0" align="left">0.33 ± 0.01</td></tr></tbody></table></div></div><div class="NLM_p">Moreover, p300 HAT activity has been reported to play important roles in pancreatic cancer<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and acute myeloid leukemia caused by oncogene RUNX1-ETO.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Activity of compound <b>12</b> was evaluated against two pancreatic cancer cells PANC-1 and MDA-PANC-28 and RUNX1-ETO leukemia cell line Kasumi-1. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, compound <b>12</b> showed potent activities against proliferation of the two pancreatic cancer cells with EC<sub>50</sub> values of 1.0 and 2.8 μM. It also inhibited growth of Kasumi-1 cells with an EC<sub>50</sub> of 2.6 μM.</div><div class="NLM_p last">Inactive compounds <b>28</b> and <b>31</b> exhibited no inhibitory activities against proliferation of these cancer cells, supporting activity of compound <b>12</b> is related to inhibition of p300 HAT. Surprisingly, although A-485 exhibited more potent activity (EC<sub>50</sub> = 0.33 μM) against Kasumi-1 leukemia cells, it had no significant activity against proliferation of breast and pancreatic cancer cells (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). These differences might be due to different modes of inhibition (competitive against histone vs Ac-CoA) or cell permeability for compound <b>12</b> and A-485.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Gene Expression Profiling</h3><div class="NLM_p">RNA sequencing was used to investigate how treatment with p300 HAT inhibitor <b>12</b> affects gene expression in ER+ breast cancer MCF-7 cells. Upon starvation (i.e., culturing in charcoal treated FBS without hydrophobic hormones including estrogens) for 7 days, MCF-7 cells were supplemented with estradiol (10 nM), followed by treatment with compound <b>12</b> for 2 days. Total RNA from three groups of MCF-7 cells, including starved, control (with estradiol), and treated (with estradiol and compound <b>12</b>) groups, was purified, prepared, and sequenced. RNA sequencing data were mapped onto the human genome, and gene expression profiles were determined and normalized. Genes with significantly changed expression levels were determined using a two-sided parametric <i>t</i>-test with the <i>p</i> value of <0.05. Gene set enrichment analysis (GSEA)<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> was used to analyze significant gene expression changes between these three groups of MCF-7 cells.</div><div class="NLM_p">First, to determine whether the observed cellular activities of compound <b>12</b> are due to inhibition of p300 HAT, we used GSEA to compare the transcriptional profile of the <b>12</b>-treated MCF-7 cells with a publicly available p300 transcriptional signature (GSE31873), which was derived from siRNA-mediated p300 knockdown in C4-2B prostate cancer cells.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, there is a strong concordance between the two transcriptional signatures: Genes suppressed by compound <b>12</b> were strongly enriched among those suppressed by p300 knockdown, with normalized enrichment score (NES) of −3.32 and <i>p</i> value of <0.001. This result demonstrates that treatment with the p300 HAT inhibitor mimics the transcriptional footprint of p300 depletion and supports that p300 HAT is the cellular target of compound <b>12</b>.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Gene set enrichment analysis (GSEA) results showing significant gene expression changes (<i>p</i> < 0.001) in ER+ MCF-7 cells by treatment with p300 HAT inhibitor <b>12</b>, as compared to the starved (without estradiol) or control (with estradiol) group. (A) Inhibitor treatment caused significant downregulation of genes that were suppressed by siRNA-mediated p300 knockdown (GSE31873). (B–D) Results showing compound <b>12</b> counteracted estradiol in MCF-7 cells. Treatment with <b>12</b> significantly (B) suppressed expression of genes that were upregulated by estradiol and (C) induced expression of genes that were downregulated by estradiol. (D, E) GSEA heatmaps of normalized enrichment scores (NES) for publicly available gene signatures (from the MSigDB database). (D) Inhibitor <b>12</b> caused significant downregulation of cancer-related gene sets that were induced by estradiol. (E) Inhibitor <b>12</b> suppressed expression of gene sets associated with breast cancer as well as the poor prognosis, progression, invasion, and relapse of breast cancer.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, activity of compound <b>12</b> in the ER signaling pathway was analyzed. As compared to starved cells, supplementation with estradiol potently caused upregulation as well as downregulation of several publicly available ER-related gene signatures in the control group of MCF-7 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf" class="ext-link">Figure S3</a>). Importantly, treatment with compound <b>12</b> counteracted such estrogen-induced gene transcription: The gene set upregulated by estradiol was strongly downregulated upon treatment with <b>12</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, NES = −9.19 and <i>p</i> < 0.001), and the genes suppressed by estradiol were significantly upregulated by compound <b>12</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C, NES = 7.41 and <i>p</i> < 0.001). These results are consistent with previous studies showing p300 HAT is essential for ER-mediated gene expression and demonstrate that pharmacological inhibition of p300 HAT can offset the effects of estrogen in gene regulation.</div><div class="NLM_p last">Moreover, we investigated gene expression changes caused by compound <b>12</b> in the context of cancer biology and therapy. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D, global gene expression profiling analysis revealed that inhibition of p300 HAT by compound <b>12</b> counteracted estradiol and caused significant suppression of a number of estrogen-induced, cancer-related gene sets, including E2F and c-Myc gene signatures as well as those involved in cell proliferation, cell cycle, mitosis, DNA replication, DNA repair, and self-renewal (stemness) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf" class="ext-link">Figure S4</a>). Specifically for breast cancer, treatment with compound <b>12</b> downregulated a number of gene signatures that have previously been shown to be upregulated in multiple breast cancer patient data sets (as compared to normal breast tissues), associated with poor prognosis, cancer progression, invasion, and relapse (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). Collectively, these GSEA results show the importance of p300 HAT in breast and other types of cancer, as well as the perspective for pharmacological inhibition of p300 HAT (by e.g., compound <b>12</b>) in breast cancer therapy.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17202" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17202" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Histone acetylation by the HAT domain of p300/CBP has been found to play critical roles in many nuclear receptor-regulated signaling pathways, such as ER, AR, and peroxisome proliferator-activated receptors (PPARs).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Dysfunction of these gene transcription programs has been found in a number of diseases such as cancer and obesity. Moreover, p300/CBP has been found to directly contribute to oncogenesis (e.g., in RUNX1-ETO leukemia) or be part of a fusion oncogene due to chromosome translocation.<a onclick="showRef(event, 'ref7 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref7 ref14 ref15">(7,14,15)</a> Therefore, drug-like inhibitors of p300/CBP HAT are needed, which could be useful chemical probes and potential therapeutics for these indications. In addition, p300 and CBP are large proteins (∼2400 amino acids) with multiple other domains playing important physiological roles. They also share a high degree of homology with many duplicate functions. Given these two points, cell-permeable, small-molecule inhibitors of the HAT domain of p300/CBP are particularly useful for studying biological functions of p300/CBP HAT because observed activities by genetic knockdown/knockout of p300 or CBP might not be relevant.</div><div class="NLM_p">Compound screening followed by medicinal chemistry studies have led to the discovery of trisubstituted thiophene compound <b>12</b> that is a novel inhibitor with an IC<sub>50</sub> of 620 nM against p300 HAT and 1.2 μM against CBP HAT. It did not significantly inhibit two other major human HATs PCAF and Myst3 at 50 μM, showing a high selectivity. Biochemical studies showed that compound <b>12</b> is competitive against the substrate histone and noncompetitive against the cofactor Ac-CoA, showing a distinct mode of inhibition from previous studied inhibitors C646 and A-485. Docking studies of <b>12</b> into the histone binding pocket of the p300 HAT structure provided possible binding models of the inhibitor in the protein. In addition, compound <b>12</b> was found to be cell-permeable and inhibit cellular acetylation at several histone lysine residues, with H3K27 being the most sensitive substrate. With EC<sub>50</sub> of ∼3 μM, compound <b>12</b> strongly inhibited proliferation of ER+ breast cancer cell line MCF-7, regardless of its sensitivity or resistance to the commonly used estrogen antagonist tamoxifen, suggesting the potential therapeutic application of this compound in the context of endocrine resistance. Compound <b>12</b> also exhibited strong activity against growth of several pancreatic cancer cells and RUNX1-ETO leukemia cell Kasumi-1, in which p300 HAT is known to be important.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a></div><div class="NLM_p">Global gene transcription profiling was performed to investigate the cellular target of compound <b>12</b> as well as the mechanism for its observed biological activities in MCF-7 cells. First, the overall transcriptional changes upon treatment with compound <b>12</b> recapitulated those caused by siRNA-mediated p300 knockdown. This result, together with its inhibitory activity in cellular histone lysine acetylation, support that p300 HAT is the cellular target of compound <b>12</b>. Second, inhibition of p300 HAT activity by compound <b>12</b> significantly offset estrogen-induced gene expression, showing p300 HAT is essential for the ER activity. Moreover, treatment with compound <b>12</b> strongly downregulated a number of (general) cancer-related gene signatures, as well as gene sets that have been identified in the clinic for the progression, invasion, relapse, and poor prognosis of breast cancer. These results, together with the compound’s potent antiproliferative activities against several types of cancer cells, suggest that pharmacological inhibition of p300 HAT is a useful therapeutic approach to cancer treatment.</div><div class="NLM_p last">In conclusion, our results demonstrate that compound <b>12</b> is not only a useful small-molecule probe for biological studies of p300/CBP HAT but also a novel pharmacological lead for further drug development targeting breast and other types of cancer.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88783" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88783" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All chemicals for synthesis were purchased from Alfa Aesar (Ward Hill, MA) or Aldrich (Milwaukee, WI). The identity of the synthesized compounds was characterized by <sup>1</sup>H and <sup>13</sup>C NMR on a Varian (Palo Alto, CA) 400-MR spectrometer and mass spectrometer (Shimadzu LCMS-2020). The identity of the potent inhibitors was confirmed with high resolution mass spectra (HRMS) using an Agilent 6550 iFunnel quadrupole-time-of-flight (Q-TOF) mass spectrometer with electrospray ionization (ESI). The purities of the final compounds were determined to be >95% with a Shimadzu Prominence HPLC using a Zorbax C18 (or C8) column (4.6 mm × 250 mm) monitored by UV at 254 nm.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Method A (Reactions i–iv)</h3><div class="NLM_p">To a mixture of 4,5-dibromothiophene-2-carboxylic acid <b>41</b> (2.86 g, 10 mmol), 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (1.71 g, 11 mmol), 1-hydroxybenzotriazole (1.48 g, 11 mmol) in dichloromethane (20 mL), 1-BOC-4-(aminomethyl)piperidine (2.57 g, 12 mmol), and triethylamine (2.8 mL, 20 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) were added slowly at 0 °C. The reaction mixture was stirred at room temperature for 1 h and quenched by adding saturated NaHCO<sub>3</sub> (50 mL). The product was extracted with dichloromethane (3 × 50 mL) and the combined organic layers were washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>. Upon removal of the solvent, the product was purified by column chromatography (silica gel, <i>n</i>-hexanes/ethyl acetate from 10:1 to 2:1) to give the amide <b>42</b> as a white solid (4.63 g, 96% yield).</div><div class="NLM_p">A mixture of <b>42</b> (0.4 mmol), Ar<sup>1</sup>-boronic acid or its pinacol ester (0.42 mmol), tetrakis(triphenylphosphine)palladium (13.9 mg), and sodium carbonate (84.8 mg, 0.8 mmol) in 1,4-dioxane/H<sub>2</sub>O (6 mL, 5:1) was heated to 80 °C for 12 h. The reaction was then quenched with brine (10 mL). The product was extracted with diethyl ether (3 × 20 mL), and the combined organic layers were washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>. Upon removal of the solvent, the product was purified by column chromatography (silica gel, <i>n</i>-hexanes/ethyl acetate from 5:1 to 1:2) to give 5-Ar<sup>1</sup> substituted thiophene product as a white solid (60–90% yield). The product thus obtained (0.2 mmol) was added into a solution of Ar<sup>2</sup>-boronic acid or its pinacol ester (0.21 mmol), tetrakis(triphenylphosphine)palladium (7 mg), and sodium carbonate (42.4 mg, 0.4 mmol) in 1,4-dioxane/H<sub>2</sub>O (5/1 mL). The mixture was heated to 100 °C for 12 h. Similar workup and purification gave 4,5-disubstituted thiophene compound <b>43</b> as a white solid (60–90% yield).</div><div class="NLM_p last">To a solution of <b>43</b> (0.1 mmol) in dichloromethane (2 mL) at 0 °C, HCl (0.1–0.2 mL, 4 N in 1,4-dioxane) was added slowly and then stirred at room temperature for 12 h. Upon removal of the solvent carefully, the residual oil was treated with anhydrous diethyl ether and vacuum-dried to give the target compound (white powder) as a hydrochloric salt (90–100% yield).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Method B (Reactions v–vii)</h3><div class="NLM_p">To a solution of 4,5-dibromothiophene-2-carboxaldehyde <b>44</b> (5.40 g, 20 mmol) in MeOH (40 mL), NaBH<sub>4</sub> (0.79 g, 21 mmol) was added slowly 0 °C. The reaction mixture was stirred at room temperature for 1 h and quenched with water (50 mL). The product was extracted with diethyl ether (3 × 100 mL), and the combined organic layers were washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>. Upon removal of the solvent carefully, the residual oil was dried and used in the next step without purification. It was dissolved in DMF (20 mL), and the solution was cooled to 0 °C. Cyanuric chloride (3.69 g, 20 mmol) was added slowly at 0 °C and stirred at room temperature for 10 h. The reaction was quenched with saturated NaHCO<sub>3</sub> (50 mL). The product was extracted with diethyl ether (3 × 100 mL), and the combined organic layers were washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>. Removal of the solvents <i>in vacuo</i> afforded compound <b>45</b> as a colorless oil, which is used without purification. To a solution of an amine (10 mmol) and potassium carbonate (1.38 g, 10 mmol) in DMF (10 mL), <b>45</b> (1.45 g, 5 mmol) in DMF (10 mL) was added slowly at 0 °C, and the mixture was stirred at room temperature for 10 h. Upon quenching with saturated NaHCO<sub>3</sub> (50 mL), the product was extracted with diethyl ether (3 × 50 mL) and the combined organic layers were washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>. A column chromatography (silica gel, hexanes/ethyl acetate from 40:1 to 2:1) for the residue oil gave compound <b>46</b> as a pale-yellow oil (60–85% yield for the three steps).</div><div class="NLM_p last">Suzuki coupling reactions of compound <b>46</b> as well as deprotection of the BOC group were performed as described above, to produce the target compounds.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General Method C (Reactions viii and ix)</h3><div class="NLM_p">4-Bromothiazole-2-carbaldehyde <b>48</b> (1.92 g, 10 mmol) and <i>N</i>-bromosuccinimide (NBS, 1.78 g, 10 mmol) were dissolved in dichloromethane (20 mL) and stirred at room temperature for 12 h. Upon removal of the solvent carefully, the residue was purified by column chromatography (silica gel, hexanes/ethyl acetate from 40:1 to 10:1) to give 4,5-dibromothiazole-2-carbaldehyde <b>49</b> as a white solid (433 mg, 16% yield). Conversion of aldehydes <b>48</b> and <b>49</b> to compounds <b>38</b> and <b>37</b> followed the reactions described above.</div><div class="NLM_p last">4,5-Dibromothiazole-2-carbaldehyde <b>49</b> (406 mg, 1.5 mmol) or <b>48</b>, sodium chlorite (190 mg, 2.1 mmol), and 2-methylbut-2-ene (1.6 mL, 15 mmol) were dissolved by <sup><i>t</i></sup>BuOH/H<sub>2</sub>O (5/1 mL). Sodium dihydrogen phosphate (900 mg in 5 mL H<sub>2</sub>O) was added slowly into the reaction mixture. The mixture was stirred at rt for 12 h. The reaction was then quenched with brine (10 mL). The product was extracted with diethyl ether (3 × 10 mL), and the combined organic layers were washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>. The volatiles were removed <i>in vacuo</i> to afford acid <b>51</b> as a crude oil, which was used directly in the next step without further purification.</div></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 4,5-Bis(4-(<i>tert</i>-butyl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)thiophene-2-carboxamide Hydrochloride (<b>1</b>)</h3><div class="NLM_p last"><b>1</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.69 (br, 3H), 7.92 (s, 1H), 7.38–7.35 (m, 4H), 7.26–7.21 (m, 4H), 3.26–3.23 (m, 2H), 3.17 (br, 2H), 2.85–2.79 (m, 2H), 1.82–1.79 (m, 3H), 1.42–1.36 (m, 2H), and 1.27 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 161.1, 150.8, 149.8, 141.5, 137.6 (2), 132.7, 131.4, 130.4, 128.4, 128.2, 125.6, 125.3, 43.8, 42.8, 34.4, 34.3, 33.7, 31.1, 31.0, and 26.22. MS (ESI) calcd for (C<sub>31</sub>H<sub>41</sub>N<sub>2</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 489.7, found 489.6.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 4,5-Bis(4-(furan-3-yl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)thiophene-2-carboxamide Hydrochloride (<b>2</b>)</h3><div class="NLM_p last"><b>2</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.89 (br, 1H), 8.76 (br, 1H), 8.58 (br, 1H), 8.21 (s, 2H), 8.00 (s, 1H), 7.74 (s, 2H), 7.61–7.59 (m, 4H), 7.31–7.28 (m, 4H), 6.97 (s, 2H), 3.27–3.23 (m, 2H), 3.19 (br, 2H), 2.86–2.78 (m, 2H), 1.85–1.82 (m, 3H), and 1.42–1.36 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 161.0, 147.1, 144.5, 141.6, 139.9, 139.6, 138.6, 138.0, 137.8, 133.9, 132.08, 131.96, 131.65, 131.54, 131.0, 130.8, 129.2, 129.0, 125.9, 125.7, 108.6, 43.8, 42.8, 33.8, and 26.2. MS (ESI) calcd for (C<sub>31</sub>H<sub>29</sub>N<sub>2</sub>O<sub>3</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 509.6, found 509.5.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 4,5-Bis(4′-methoxy-[1,1′-biphenyl]-4-yl)-<i>N</i>-(piperidin-4-ylmethyl)thiophene-2-carboxamide Hydrochloride (<b>3</b>)</h3><div class="NLM_p last"><b>3</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.91 (br, 1H), 8.79 (br, 1H), 8.62 (br, 1H), 8.02 (s, 1H), 7.70–7.56 (m, 8H), 7.38–7.35 (m, 4H), 7.02–6.99 (m, 4H), 3.78 (s, 6H), 3.27–3.23 (m, 2H), 3.19 (br, 2H), 2.87–2.79 (m, 2H), 1.85–1.82 (m, 3H), and 1.42–1.36 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 161.1, 159.2, 159.0, 141.6, 139.6, 138.6, 138.1, 137.7, 133.8, 131.64, 131.57, 131.46, 131.38, 129.3, 129.2, 127.72, 127.66, 126.5, 126.2, 114.5, 114.4, 55.2, 43.8, 42.9, 33.8, 28.1, and 26.3. MS (ESI) calcd for (C<sub>37</sub>H<sub>37</sub>N<sub>2</sub>O<sub>3</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 589.8, found 589.9.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 5-(4-(<i>tert</i>-Butyl)phenyl)-4-(4-(furan-3-yl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)thiophene-2-carboxamide Hydrochloride (<b>4</b>)</h3><div class="NLM_p last"><b>4</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.90 (br, 1H), 8.76 (br, 1H), 8.61 (br, 1H), 8.21 (s, 1H), 7.97 (s, 1H), 7.92 (s, 1H), 7.74 (s, 1H), 7.59 (s, 1H), 7.36–7.23 (m, 6H), 6.97 (s, 2H), 3.27–3.23 (m, 2H), 3.18 (br, 2H), 2.85–2.79 (m, 2H), 1.83–1.80 (m, 3H), 1.40–1.36 (m, 2H), and 1.26 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 161.1, 151.0, 144.4, 142.0, 139.6, 137.8, 137.5, 134.1, 131.0, 129.0, 128.44, 128.40, 128.3, 125.74, 125.70, 125.4, 108.6, 43.8, 42.9, 34.4, 33.8, 31.1, and 26.3. MS (ESI) calcd for (C<sub>31</sub>H<sub>35</sub>N<sub>2</sub>O<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 499.7, found 499.5.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 4-(4-(Aminomethyl)phenyl)-5-(4-(<i>tert</i>-butyl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)thiophene-2-carboxamide Hydrochloride (<b>5</b>)</h3><div class="NLM_p last"><b>5</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 7.76 (s, 1H), 7.25 (d, <i>J</i> = 7.6 Hz, 2H), 7.14 (d, <i>J</i> = 7.6 Hz, 2H), 6.97 (d, <i>J</i> = 7.3 Hz, 2H), 6.90 (d, <i>J</i> = 7.3 Hz, 2H), 4.03 (s, 2H), 3.45 (d, <i>J</i> = 11.7 Hz, 2H), 3.29–3.27 (m, 2H), 2.95 (t, <i>J</i> = 12.6 Hz, 2H), 1.98–1.94 (m, 3H), 1.52–1.42 (m, 2H), and 0.97 (s, 9H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O): 163.4, 151.3, 143.8, 137.3, 136.1, 134.3, 131.6, 129.9, 129.4, 129.1, 128.5, 128.1, 125.4, 44.3, 43.6, 42.6, 33.9, 33.5, 30.5, and 26.1. MS (ESI) calcd for (C<sub>28</sub>H<sub>36</sub>N<sub>3</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 462.7, found 462.6.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 5-(4-(Furan-3-yl)phenyl)-4-(4-(piperidin-1-ylmethyl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)thiophene-2-carboxamide Hydrochloride (<b>6</b>)</h3><div class="NLM_p last"><b>6</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.54 (br, 1H), 8.94 (br, 1H), 8.84 (br, 1H), 8.64 (br, 1H), 8.22 (s, 1H), 8.03 (s, 1H), 7.75 (s, 1H), 7.60 (d, <i>J</i> = 7.8 Hz, 4H), 7.36 (d, <i>J</i> = 7.8 Hz, 2H), 7.28 (d, <i>J</i> = 7.8 Hz, 2H), 6.97 (s, 1H), 4.24 (s, 4H), 3.25–3.24 (m, 4H), 3.19–3.16 (m, 2H), 2.86–2.78 (m, 4H), 1.83–1.78 (m, 6H), 1.70–1.66 (m, 1H), and 1.48–1.34 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 161.0, 144.6, 142.2, 140.0, 138.2, 137.3, 136.6, 132.1, 131.8, 131.6, 131.5, 129.3, 129.0, 126.0, 125.2, 115.5, 108.6, 58.6, 51.7, 43.9, 42.9, 33.7, 26.3, 22.2, and 21.5. MS (ESI) calcd for (C<sub>33</sub>H<sub>38</sub>N<sub>3</sub>O<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 540.7, found 540.6.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 5-(4-(<i>tert</i>-Butyl)phenyl)-4-(4-(piperazin-1-ylmethyl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)thiophene-2-carboxamide Hydrochloride (<b>7</b>)</h3><div class="NLM_p last"><b>7</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.83 (s, 1H), 7.58 (d, <i>J</i> = 8.1 Hz, 2H), 7.42 (d, <i>J</i> = 8.1 Hz, 2H), 7.37 (d, <i>J</i> = 8.4 Hz, 2H), 7.23 (d, <i>J</i> = 8.4 Hz, 2H), 4.44 (s, 2H), 3.61 (br, 4H), 3.56 (br, 4H), 3.43 (d, <i>J</i> = 8.1 Hz, 2H), 3.35 (d, <i>J</i> = 6.0 Hz, 2H), 3.00 (t, <i>J</i> = 12.2 Hz, 2H), 2.03–1.98 (m, 3H), 1.55–1.45 (m, 2H), and 1.31 (s, 9H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD): 164.3, 153.1, 145.8, 139.2, 138.5, 138.1, 134.2, 132.8, 132.3, 131.6, 130.9, 130.0, 126.9, 61.4, 58.7, 45.4, 45.0, 42.2, 35.6, 31.6, 28.5, and 27.8. MS (ESI) calcd for (C<sub>32</sub>H<sub>43</sub>N<sub>4</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 531.8, found 531.7.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 5-(4-(<i>tert</i>-Butyl)phenyl)-4-(furan-3-yl)-<i>N</i>-(piperidin-4-ylmethyl)thiophene-2-carboxamide Hydrochloride (<b>8</b>)</h3><div class="NLM_p last"><b>8</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.90 (br, 1H), 8.73 (br, 1H), 8.59 (br, 1H), 7.97 (s, 1H), 7.65 (s, 1H), 7.44–7.34 (m, 5H), 6.29 (s, 1H), 3.27–3.24 (m, 2H), 3.17 (br, 2H), 2.84–2.80 (m, 2H), 1.83–1.78 (m, 3H), 1.40–1.37 (m, 2H), and 1.29 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 161.4, 151.7, 144.1, 140.4, 138.2, 130.7, 130.5, 129.2, 129.1, 126.3, 126.0, 120.8, 110.6, 44.2, 43.2, 34.9, 34.1, 31.4, and 26.6. MS (ESI) calcd for (C<sub>25</sub>H<sub>31</sub>N<sub>2</sub>O<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 423.6, found 423.5.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 5-(4-(<i>tert</i>-Butyl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)-4-(pyridin-3-yl)thiophene-2-carboxamide Hydrochloride (<b>9</b>)</h3><div class="NLM_p last"><b>9</b> was prepared from 4,5-dibromothiophene-2-carboxylic acid, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.00 (br, 1H), 8.92 (br, 1H), 8.74 (br, 1H), 8.70 (br, 1H), 8.17 (br, 2H), 7.85 (s, 1H), 7.61–7.55 (m, 2H), 7.40–7.26 (m, 4H), 3.27–3.18 (m, 4H), 2.83–2.80 (m, 2H), 1.84–1.79 (m, 3H), 1.42–1.39 (m, 2H), and 1.26 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 161.2, 152.1, 145.2, 143.8, 143.3, 142.8, 139.2, 134.1, 132.2, 131.1, 129.5, 129.1, 126.6, 126.5, 44.3, 43.2, 34.9, 34.1, 31.4, and 26.6. MS (ESI) calcd for (C<sub>26</sub>H<sub>32</sub>N<sub>3</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 434.6, found 434.4.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 1-(4,5-Bis(4′-methoxy-[1,1′-biphenyl]-4-yl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>10</b>)</h3><div class="NLM_p last"><b>10</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.46 (br, 1H), 9.39 (br, 1H), 8.90 (br, 1H), 8.72 (br, 1H), 7.69 (s, 1H), 7.63–7.54 (m, 8H), 7.34 (d, <i>J</i> = 8.2 Hz, 4H), 7.01 (d, <i>J</i> = 8.2 Hz, 4H), 4.41 (s, 2H), 3.78 (s, 6H), 3.29–3.26 (m, 2H), 2.94 (br, 2H), 2.87–2.79 (m, 2H), 2.08 (br, 1H), 1.98–1.92 (m, 2H), and 1.49–1.39 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 159.2, 159.1, 139.4, 139.2, 138.6, 137.2, 134.2, 133.9, 131.7, 131.4, 129.4, 129.2, 127.76, 127.70, 126.9, 126.6, 126.4, 126.0, 114.54. 114.50, 55.3, 50.7, 44.7, 42.5, 30.60, 30.55, and 26.1. MS (ESI) calcd for (C<sub>37</sub>H<sub>39</sub>N<sub>2</sub>O<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 575.8, found 575.7.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 1-(4,5-Bis(4-(furan-3-yl)phenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>11</b>)</h3><div class="NLM_p last"><b>11</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (yellow powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.54 (br, 2H), 9.03 (br, 1H), 8.88 (br, 1H), 7.46 (s, 1H), 7.16–7.12 (m, 8H), 4.35 (s, 2H), 3.26–3.23 (m, 2H), 2.92–2.88 (m, 2H), 2.84–2.79 (m, 2H), 2.54 (t, J = 7.4 Hz, 4H), 2.09 (br, 1H), 1.97–1.93 (m, 2H), 1.52 (p, <i>J</i> = 7.4 Hz, 4H), 1.46–1.39 (m, 2H), 1.32–1.23 (m, 4H), and 0.87 (t, <i>J</i> = 7.4 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 144.5, 144.4, 139.8, 139.6, 139.1, 137.2, 134.04, 133.97, 131.79, 131.69, 131.62, 130.9, 129.2, 129.0, 126.0, 125.8, 125.3, 125.2, 108.58, 108.56, 50.6, 44.6, 42.4, 30.6, and 26.1. MS (ESI) calcd for (C<sub>31</sub>H<sub>31</sub>N<sub>2</sub>O<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 495.7, found 495.6.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 1-(4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>12</b>)</h3><div class="NLM_p last"><b>12</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.56 (br, 2H), 9.00 (br, 1H), 8.84 (br, 1H), 7.48 (s, 1H), 7.36 (d, <i>J</i> = 7.2 Hz, 4H), 7.20 (d, <i>J</i> = 7.2 Hz, 4H), 4.35 (s, 2H), 3.27–3.24 (m, 2H), 2.89 (br, 2H), 2.86–2.82 (m, 2H), 2.05 (br, 1H), 1.97–1.94 (m, 2H), 1.45–1.40 (m, 2H), and 1.27 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.5, 149.6, 139.0, 137.0, 134.3, 132.8, 131.4, 130.6, 128.4, 128.2, 125.7, 125.4, 50.6, 44.6, 42.3, 34.42, 34.34, 31.13, 31.05, 30.6, and 26.1. MS (ESI) calcd for (C<sub>31</sub>H<sub>43</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 475.8, found 475.7; HRMS (ESI<sup>+</sup>) calcd for C<sub>31</sub>H<sub>42</sub>N<sub>2</sub>S [M + H]<sup>+</sup> 475.3147, found 475.3145.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 4-(((4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)methoxy)methyl)piperidine Hydrochloride (<b>13</b>)</h3><div class="NLM_p last">Starting from 4,5-dibromothiophene-2-carboxaldehyde, reactions v and vi gave 2-chloromethyl-4,5-dibromothiophene, which reacted with sodium salt of 1-BOC-4-hydroxymethylpiperidine, to give 4,5-dibromothiophen-2-ylmethyl 1-BOC-piperidin-4-yl-methyl ether. Reactions ii and iii (general method B) produced compound <b>13</b> as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.72 (br, 1H), 8.38 (br, 1H), 7.36 (d, <i>J</i> = 8.1 Hz, 2H), 7.33 (d, <i>J</i> = 8.1 Hz, 2H), 7.48 (s, 1H), 7.21 (d, <i>J</i> = 8.1 Hz, 2H), 7.17 (d, <i>J</i> = 8.1 Hz, 2H), 5.05 (s, 2H), 3.27–3.23 (m, 4H), 2.86–2.77 (m, 2H), 1.78–1.75 (m, 2H), 1.62 (br, 1H), and 1.26 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.4, 149.6, 138.8, 138.6, 136.7, 132.8, 131.7, 130.7, 128.4, 128.2, 125.6, 125.3, 65.0, 42.9, 40.8, 35.9, 34.4, 34.3, 31.10, 31.02, and 25.2. MS (ESI) calcd for (C<sub>31</sub>H<sub>42</sub>NOS)<sup>+</sup> [M + H]<sup>+</sup> 476.7, found 476.6.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>N</i>-((4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)methyl)piperidin-4-amine Hydrochloride (<b>14</b>)</h3><div class="NLM_p last"><b>14</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.85 (br, 2H), 9.24 (br, 1H), 8.99 (br, 1H), 7.50 (s, 1H), 7.36 (d, <i>J</i> = 7.2 Hz, 4H), 7.20 (d, <i>J</i> = 7.2 Hz, 4H), 4.40 (s, 2H), 2.95 (br, 3H), 2.30 (br, 2H), 1.95 (m, 2H), and 1.27 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.6, 149.7, 139.1, 137.1, 134.3, 132.8, 131.2, 130.6, 128.4, 128.2, 125.7, 125.4, 51.4, 41.4, 34.42, 34.34, 31.14, 31.06, 28.9, and 24.9. MS (ESI) calcd for (C<sub>30</sub>H<sub>41</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 461.7, found 461.6.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i>-((4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)methyl)-2-(piperidin-4-yl)ethanamine Hydrochloride (<b>15</b>)</h3><div class="NLM_p last"><b>15</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.55 (br, 2H), 8.99 (br, 1H), 8.81 (br, 1H), 7.47 (s, 1H), 7.35 (br, 4H), 7.20 (br, 4H), 4.35 (s, 2H), 3.23 (br, 2H), 2.98 (br, 2H), 2.80 (br, 2H), 1.78 (br, 1H), 1.65 (br, 2H), 1.35 (br, 2H), and 1.26 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.5, 149.6, 138.9, 137.0, 134.1, 132.7, 131.4, 130.5, 128.4, 128.2, 125.7, 125.4, 44.0, 43.8, 43.0, 34.39, 34.31, 31.3, 31.1, 31.0, 30.6, and 28.0. MS (ESI) calcd for (C<sub>32</sub>H<sub>45</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 489.8, found 489.7.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i><sup>1</sup>-((4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)methyl)propane-1,3-diamine Hydrochloride (<b>16</b>)</h3><div class="NLM_p last"><b>16</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.70 (br, 2H), 8.24 (br, 3H), 7.34 (s, 1H), 7.17–7.07 (m, 8H), 7.24 (d, <i>J</i> = 7.6 Hz, 4H), 4.22 (s, 2H), 3.29–3.20 (m, 2H), 2.47 (br, 2H), 1.34–1.32 (m, 2H), and 1.19 (s, 18H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 150.6, 149.8, 140.8, 137.7, 135.0, 132.9, 130.8, 129.0, 128.8, 128.7, 125.5, 125.4, 45.5, 44.0, 37.8, 34.62, 34.57, 31.44, 31.36, and 29.8. MS (ESI) calcd for (C<sub>28</sub>H<sub>39</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 435.7, found 435.6.</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i><sup>1</sup>-((4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)methyl)hexane-1,6-diamine Hydrochloride (<b>17</b>)</h3><div class="NLM_p last"><b>17</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.34 (br, 2H), 7.87 (br, 3H), 7.44 (s, 1H), 7.38 (d, <i>J</i> = 8.2 Hz, 2H), 7.36 (d, <i>J</i> = 8.2 Hz, 2H), 7.21 (d, <i>J</i> = 8.2 Hz, 2H), 7.19 (d, <i>J</i> = 8.2 Hz, 2H), 4.35 (s, 2H), 2.94 (br, 2H), 2.76 (p, <i>J</i> = 7.4 Hz, 2H), 1.66 (p, <i>J</i> = 7.4 Hz, 2H), 1.54 (p, <i>J</i> = 7.4 Hz, 2H), 1.35–1.33 (m, 4H), and 1.27 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.6, 149.7, 139.0, 137.0, 134.0, 132.7, 131.4, 130.5, 128.3, 128.2, 125.6, 125.4, 46.7, 46.1, 44.1, 34.4, 34.3, 31.1, 31.0, 26.7, 25.5, 25.3, and 25.2. MS (ESI) calcd for (C<sub>31</sub>H<sub>45</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 477.8, found 477.6.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 1-((4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)methyl)piperazine Hydrochloride (<b>18</b>)</h3><div class="NLM_p last"><b>18</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.61 (br, 1H), 9.50 (br, 2H), 7.44 (s, 1H), 7.38–7.34 (m, 4H), 7.22–7.19 (m, 4H), 4.48 (s, 2H), 3.33 (s, 4H), 2.50 (s, 4H), and 1.27 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.6, 149.7, 140.0, 138.6, 137.1, 132.6, 131.7, 130.4, 128.4, 128.3, 125.7, 125.4, 52.6, 47.1, 40.8, 34.4, 34.3, 31.1, and 31.0. MS (ESI) calcd for (C<sub>29</sub>H<sub>39</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 447.7, found 447.6.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4-((4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)methyl)morpholine Hydrochloride (<b>19</b>)</h3><div class="NLM_p last"><b>19</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 13.34 (br, 1H), 7.45 (s, 1H), 7.30 (d, <i>J</i> = 7.6 Hz, 4H), 7.24 (d, <i>J</i> = 7.6 Hz, 4H), 4.46 (s, 2H), 4.30 (br, 2H), 4.01 (br, 2H), 3.46 (br, 2H), 3.08 (br, 2H), and 1.32 (s, 18H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 151.3, 150.4, 142.2, 138.4, 136.6, 132.5, 130.3, 128.8, 128.6, 125.6, 125.5, 125.3, 63.8, 54.8, 50.6, 34.72, 34.64, 31.39, and 31.32. MS (ESI) calcd for (C<sub>29</sub>H<sub>38</sub>NOS)<sup>+</sup> [M + H]<sup>+</sup> 448.7, found 448.6.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 1-(4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)-<i>N</i>-methyl-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>20</b>)</h3><div class="NLM_p last"><b>20</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.58 (br, 1H), 9.02 (br, 1H), 8.85 (br, 1H), 7.53 (s, 1H), 7.38–7.35 (m, 4H), 7.24–7.20 (m, 4H), 4.55 (s, 2H), 3.27–3.24 (m, 2H), 3.05–2.99 (m, 2H), 2.90–2.84 (m, 2H), 2.77 (s, 3H), 2.17–2.10 (m, 2H), 1.98–1.95 (m, 1H), 1.46–1.39 (m, 2H), and 1.27 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.7, 149.7, 140.4, 137.2, 136.4, 132.6, 130.4, 128.4, 128.3, 128.2, 125.7, 125.4, 58.5, 52.6, 42.42, 42.37, 34.42, 34.33, 31.14, 31.05, 28.9, and 26.4. MS (ESI) calcd for (C<sub>32</sub>H<sub>45</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 489.8, found 489.6.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>-((4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)methyl)-<i>N</i>-(piperidin-4-ylmethyl)formamide Hydrochloride (<b>21</b>)</h3><div class="NLM_p last"><b>21</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.34 (br, 1H), 9.01 (br, 1H), 7.29–7.25 (m, 4H), 7.20–7.17 (m, 4H), 6.99 (s, 1H), 4.65 (s, 2H), 3.31–3.27 (m, 2H), 3.14–3.11 (m, 2H), 2.93–2.89 (m, 2H), 1.83–1.76 (m, 3H), 1.62–1.59 (m, 2H), and 1.29 (s, 18H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 162.9, 154.8, 150.1, 138.7, 137.0, 136.5, 131.2, 130.8, 130.3, 128.7, 128.6, 125.5, 125.4, 52.7, 47.6, 47.5, 41.1, 34.6, 31.44, 31.36, 29.8, and 28.5. MS (ESI) calcd for (C<sub>32</sub>H<sub>43</sub>N<sub>2</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 503.8, found 503.6.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 1-(4,5-Bis(4-isopropylphenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>22</b>)</h3><div class="NLM_p last"><b>22</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.67 (br, 2H), 9.13 (br, 1H), 9.03 (br, 1H), 7.50 (s, 1H), 7.20–7.15 (m, 8H), 4.34 (s, 2H), 3.26–3.23 (m, 2H), 2.90–2.83 (m, 4H), 2.75 (sep, <i>J</i> = 7.0 Hz, 2H), 2.10 (br, 1H), 1.99–1.94 (m, 2H), 1.47–1.40 (m, 2H), and 1.18 (d, <i>J</i> = 7.0 Hz, 12H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 148.2, 147.3, 139.0, 137.1, 134.2, 133.1, 131.3, 130.9, 128.6, 128.5, 126.8, 126.5, 50.5, 44.5, 42.3, 33.1, 30.6, 28.1, 26.1, 23.8, and 23.7. MS (ESI) calcd for (C<sub>29</sub>H<sub>39</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 447.7, found 447.6.</div></div><div id="sec4_3_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 1-(4,5-Bis(4-butylphenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>23</b>)</h3><div class="NLM_p last"><b>23</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.54 (br, 2H), 9.03 (br, 1H), 8.88 (br, 1H), 7.46 (s, 1H), 7.16–7.12 (m, 8H), 4.35 (s, 2H), 3.26–3.23 (m, 2H), 2.90 (br, 2H), 2.84–2.79 (m, 2H), 2.54 (t, <i>J</i> = 7.4 Hz, 4H), 2.09 (br, 1H), 1.97–1.93 (m, 2H), 1.52 (p, <i>J</i> = 7.4 Hz, 4H), 1.46–1.39 (m, 2H), 1.32–1.23 (m, 4H), and 0.87 (t, <i>J</i> = 7.4 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 142.5, 141.5, 139.3, 137.3, 134.2, 133.0, 131.2, 130.8, 128.86, 128.77, 128.62, 128.56, 50.7, 44.7, 42.5, 34.6, 33.09, 33.00, 30.6, 26.1, 24.8, 21.9 (2), and 13.9 (2). MS (ESI) calcd for (C<sub>31</sub>H<sub>43</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 475.8, found 475.7.</div></div><div id="sec4_3_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 1-(5-(4-(<i>tert</i>-Butyl)phenyl)-4-(4-(furan-3-yl)phenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>24</b>)</h3><div class="NLM_p last"><b>24</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.51 (br, 2H), 8.95 (br, 1H), 8.79 (br, 1H), 8.08 (s, 1H), 7.89 (d, <i>J</i> = 8.1 Hz, 1H), 7.61–7.58 (m, 1H), 7.52 (s, 1H), 7.39–7.36 (m, 2H), 7.32 (d, <i>J</i> = 8.1 Hz, 1H), 7.24–7.19 (m, 4H), 6.97 (s, 1H), 4.38 (s, 2H), 3.28–3.25 (m, 2H), 2.93–2.88 (m, 2H), 2.86–2.82 (m, 2H), 2.08 (br, 1H), 1.98–1.95 (m, 2H), 1.47–1.42 (m, 2H), and 1.26 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 146.5, 144.5, 139.6, 136.7, 134.1, 133.9, 131.6, 131.4, 129.0, 128.8, 128.4, 128.3, 127.4, 125.8, 125.7, 108.5, 50.6, 44.6, 42.4, 34.4, 31.0, 30.5, and 26.1. MS (ESI) calcd for (C<sub>31</sub>H<sub>37</sub>N<sub>2</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 485.7, found 485.6.</div></div><div id="sec4_3_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 1-(4-(4-(Furan-3-yl)phenyl)-5-(4-(piperidin-1-ylmethyl)phenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>25</b>)</h3><div class="NLM_p last"><b>25</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.50 (br, 1H), 9.60 (br, 2H), 8.91 (br, 1H), 8.78 (br, 1H), 8.21 (s, 1H), 7.75 (s, 1H), 7.64–7.54 (m, 5H), 7.35 (d, <i>J</i> = 8.1 Hz, 2H), 7.24 (d, <i>J</i> = 8.1 Hz, 2H), 6.97 (s, 1H), 4.40 (s, 2H), 4.24 (s, 2H), 3.28–3.24 (m, 4H), 2.94–2.90 (m, 2H), 2.86–2.81 (m, 4H), 2.09 (br, 1H), 1.98–1.95 (m, 2H), 1.77 (br, 4H), 1.71–1.67 (m, 2H), and 1.48–1.39 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 144.5, 139.6, 138.4, 137.6, 134.4, 134.1, 133.7, 132.1, 132.0, 131.5, 131.4, 129.0, 128.8, 128.7, 125.8, 108.6, 51.6, 50.6, 44.5, 42.3, 30.7, 30.5, 26.1, 22.1, and 21.4. MS (ESI) calcd for (C<sub>33</sub>H<sub>40</sub>N<sub>3</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 526.8, found 526.6.</div></div><div id="sec4_3_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 1-(5-(4-(Aminomethyl)phenyl)-4-(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>26</b>)</h3><div class="NLM_p last"><b>26</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 7.63 (s, 1H), 7.45 (d, <i>J</i> = 9.2 Hz, 2H), 7.42–7.39 (m, 4H), 7.28 (d, <i>J</i> = 9.2 Hz, 2H), 4.54 (s, 2H), 4.18 (s, 2H), 3.50–3.46 (m, 2H), 3.14 (br, 2H), 3.03 (t, <i>J</i> = 14.4 Hz, 2H), 2.15 (br, 1H), 2.07–2.04 (m, 2H), 1.57–1.48 (m, 2H), and 1.29 (s, 9H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O): 144.2, 139.8, 138.3, 137.4, 134.0, 132.4, 130.2, 129.8, 129.1, 128.7, 125.6, 108.8, 50.7, 44.4, 43.1, 42.6, 30.7, 30.3, 25.8, and 22.1. MS (ESI) calcd for (C<sub>28</sub>H<sub>38</sub>N<sub>3</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 448.7, found 448.8.</div></div><div id="sec4_3_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 1-(4-(4-(Aminomethyl)phenyl)-5-(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>27</b>)</h3><div class="NLM_p last"><b>27</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.73 (br, 1H), 9.12 (br, 1H), 9.00 (br, 1H), 8.50 (br, 2H), 8.28 (br, 2H), 7.55 (s, 1H), 7.47 (d, <i>J</i> = 7.9 Hz, 2H), 7.37 (d, <i>J</i> = 7.9 Hz, 2H), 7.28 (d, <i>J</i> = 7.9 Hz, 2H), 7.19 (d, <i>J</i> = 7.9 Hz, 2H), 4.35 (s, 2H), 4.00 (s, 2H), 3.24 (d, <i>J</i> = 12.7 Hz, 2H), 2.89–2.83 (m, 4H), 2.10 (br, 1H), 1.97 (d, <i>J</i> = 12.3 Hz, 2H), 1.50–1.42 (m, 2H), and 1.27 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.7, 139.6, 136.6, 135.7, 134.2, 133.0, 130.5, 129.2, 128.7, 128.4, 125.7, 115.3, 50.5, 44.4, 42.3, 41.8, 34.4, 31.0, 30.5, and 26.1. MS (ESI) calcd for (C<sub>28</sub>H<sub>38</sub>N<sub>3</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 448.7, found 448.8.</div></div><div id="sec4_3_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> ((5-(((Piperidin-4-ylmethyl)amino)methyl)thiophene-2,3-diyl)bis(4,1-phenylene))dimethanamine Hydrochloride (<b>28</b>)</h3><div class="NLM_p last"><b>28</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) δ 7.43 (s, 1H), 7.40 (br, 8H), 4.57 (s, 2H), 4.19 (s, 4H), 3.50 (d, <i>J</i> = 12.8 Hz, 2H), 3.15 (d, <i>J</i> = 6.8 Hz, 2H), 3.05 (t, <i>J</i> = 12.0 Hz, 2H), 2.18 (s, 1H), 2.08 (d, <i>J</i> = 14.0 Hz, 2H), and 1.59–1.50 (m, 2H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O) 140.6, 137.9, 136.1, 133.9, 133.8, 132.6, 131.8, 130.5, 129.9, 129.6, 129.1, 129.0, 50.8, 45.4, 43.1, 42.64, 42.57, 30.7, and 25.8. MS (ESI) calcd for (C<sub>25</sub>H<sub>33</sub>N<sub>4</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 421.6, found 421.5.</div></div><div id="sec4_3_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> ((5-(((Piperidin-4-ylmethyl)amino)methyl)thiophene-2,3-diyl)bis(4,1-phenylene))dimethanol Hydrochloride (<b>29</b>)</h3><div class="NLM_p last"><b>29</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.42 (br, 2H), 8.91 (br, 1H), 8.74 (br, 1H), 7.47 (s, 1H), 7.28 (d, <i>J</i> = 7.1 Hz, 2H), 7.26 (d, <i>J</i> = 7.1 Hz, 2H), 7.19 (d, <i>J</i> = 7.1 Hz, 2H), 7.17 (d, <i>J</i> = 7.1 Hz, 2H), 4.47 (s, 4H), 4.37 (s, 2H), 3.27–3.24 (m, 2H), 2.92 (br, 2H), 2.89–2.82 (m, 2H), 2.07 (br, 1H), 1.97–1.93 (m, 2H), and 1.48–1.39 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 142.6, 141.7, 139.5, 137.4, 134.2, 134.0, 131.8, 131.4, 128.7, 128.4, 127.0, 126.8, 62.7, 62.6, 50.8, 44.8, 42.5, 30.6, and 26.1. MS (ESI) calcd for (C<sub>25</sub>H<sub>31</sub>N<sub>2</sub>O<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 423.6, found 423.5.</div></div><div id="sec4_3_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 1-(4,5-Di(furan-3-yl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>30</b>)</h3><div class="NLM_p last"><b>30</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.61 (br, 2H), 9.09 (br, 1H), 8.96 (br, 1H), 7.88 (s, 1H), 7.78 (s, 2H), 7.71 (s, 1H), 7.48 (s, 1H), 6.52 (s, 1H), 6.48 (s, 1H), 4.32 (s, 2H), 3.27–3.24 (m, 2H), 2.87–2.82 (m, 4H), 2.08 (br, 1H), 1.97–1.94 (m, 2H), and 1.45–1.40 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 144.2, 143.7, 140.9, 140.2, 133.1, 131.1, 129.8, 129.2, 120.1, 118.1, 111.1, 110.3, 50.4, 44.4, 42.3, 30.5, and 26.0. MS (ESI) calcd for (C<sub>19</sub>H<sub>23</sub>N<sub>2</sub>O<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 343.5, found 343.4.</div></div><div id="sec4_3_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 1-(4,5-Bis(3,4-dimethoxyphenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>31</b>)</h3><div class="NLM_p last"><b>31</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.40 (br, 2H), 8.94 (br, 1H), 8.77 (br, 1H), 7.47 (s, 1H), 7.17–7.14 (m, 1H), 7.01–6.91 (m, 2H), 6.84 (d, <i>J</i> = 8.4 Hz, 1H), 6.78 (s, 1H), 6.74 (s, 1H), 4.34 (s, 2H), 3.75 (s, 3H), 3.73 (s, 3H), 3.56 (s, 3H), 3.54 (s, 3H), 3.28–3.25 (m, 2H), 2.92 (br, 2H), 2.88–2.82 (m, 2H), 2.07 (br, 1H), 1.98–1.91 (m, 2H), and 1.48–1.38 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 148.8, 148.51, 148.48, 148.1, 136.9, 131.6, 131.5, 128.9, 128.8, 125.9, 121.4, 121.0, 112.54, 112.52, 112.0, 111.9, 55.69, 55.62, 55.59, 55.36, 50.7, 44.8, 42.4, 30.6, and 26.1. MS (ESI) calcd for (C<sub>27</sub>H<sub>35</sub>N<sub>2</sub>O<sub>4</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 483.6, found 483.5.</div></div><div id="sec4_3_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 1-(4,5-Di([1,1′-biphenyl]-3-yl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>32</b>)</h3><div class="NLM_p last"><b>32</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.50 (br, 2H), 8.88 (br, 1H), 8.68 (br, 1H), 7.65 (s, 2H), 7.52–7.31 (m, 17H), 4.42 (s, 2H), 3.29–3.26 (m, 2H), 2.95 (br, 2H), 2.90–2.82 (m, 2H), 2.08 (br, 1H), 1.98–1.95 (m, 2H), and 1.48–1.39 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 140.7, 140.5, 139.8, 139.5, 139.4, 137.6, 136.0, 133.9, 129.7, 129.4, 128.95 (2), 128.90 (2), 127.9, 127.7, 127.6 (2), 127.24, 127.17, 126.6 (2), 126.5, 125.8, 50.5, 44.5, 42.3, 30.5, and 26.1. MS (ESI) calcd for (C<sub>35</sub>H<sub>35</sub>N<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 515.7, found 515.5.</div></div><div id="sec4_3_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 1-(3-Bromo-4,5-bis(4-(<i>tert</i>-butyl)phenyl)thiophen-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>33</b>)</h3><div class="NLM_p last"><b>33</b> was prepared from 4,5-dibromothiophene-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.65 (br, 1H), 9.00 (br, 1H), 8.84 (br, 1H), 8.10 (br, 1H), 7.44 (d, <i>J</i> = 8.2 Hz, 2H), 7.31 (d, <i>J</i> = 8.2 Hz, 2H), 7.14 (d, <i>J</i> = 8.2 Hz, 2H), 7.11 (d, <i>J</i> = 8.2 Hz, 2H), 4.41 (s, 2H), 3.28–3.25 (m, 2H), 2.97 (br, 2H), 2.89–2.82 (m, 2H), 2.09 (br, 1H), 1.99–1.96 (m, 2H), 1.49–1.43 (m, 2H), 1.30 (s, 9H), and 1.22 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 151.1, 150.4, 141.2, 137.0, 132.0, 129.95, 129.84, 128.0, 126.6, 125.6, 125.3, 118.0, 50.9, 44.7, 43.0, 42.3, 34.44, 34.40, 31.1, 30.9, and 26.0. MS (ESI) calcd for (C<sub>31</sub>H<sub>42</sub>BrN<sub>2</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 554.7, found 554.6.</div></div><div id="sec4_3_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 1-(4,5-Bis(4-(<i>tert</i>-butyl)phenyl)furan-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>34</b>)</h3><div class="NLM_p last"><b>34</b> was prepared from 4,5-dibromofuran-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.54 (br, 2H), 8.97 (br, 1H), 8.78 (br, 1H), 7.46–7.33 (m, 8H), 6.88 (s, 1H), 4.28 (s, 2H), 3.28–3.25 (m, 2H), 2.94 (br, 2H), 2.85–2.83 (m, 2H), 2.06 (br, 1H), 1.97–1.94 (br, 2H), 1.48–1.42 (m, 2H), 1.30 (s, 9H), and 1.27 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 150.9, 150.0, 148.4, 144.8, 130.2, 127.8, 127.5, 125.8, 125.7, 125.5, 122.2, 116.2, 50.7, 42.8, 42.4, 34.49, 34.39, 31.15, 31.02, 30.5 and 26.0. MS (ESI) calcd for (C<sub>31</sub>H<sub>43</sub>N<sub>2</sub>O)<sup>+</sup> [M + H]<sup>+</sup> 459.7, found 459.6.</div></div><div id="sec4_3_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 1-(4,5-Bis(4-(furan-3-yl)phenyl)furan-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>35</b>)</h3><div class="NLM_p last"><b>35</b> was prepared from 4,5-dibromofuran-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (yellow powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.57 (br, 2H), 8.96 (br, 1H), 8.78 (br, 1H), 8.24–8.21 (m, 1H), 7.78–7.74 (m, 2H), 7.70–7.58 (m, 5H), 7.55 (d, <i>J</i> = 8.0 Hz, 2H), 7.41 (d, <i>J</i> = 8.0 Hz, 2H), 6.99 (s, 1H), 6.96 (d, <i>J</i> = 8.0 Hz, 2H), 4.31 (s, 2H), 3.27–3.23 (m, 2H), 2.95 (br, 2H), 2.89–2.83 (m, 2H), 2.08 (br, 1H), 1.98–1.92 (m, 2H), and 1.49–1.38 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 148.5, 145.2, 144.6, 139.9, 139.7, 131.6, 131.5, 128.9, 128.8, 128.7, 126.6, 126.1, 125.8, 125.4, 124.3, 122.6, 116.1, 114.7, 108.67, 108.61, 50.7, 42.9, 42.4, 30.6, and 26.1. MS (ESI) calcd for (C<sub>31</sub>H<sub>31</sub>N<sub>2</sub>O<sub>3</sub>)<sup>+</sup> [M + H]<sup>+</sup> 479.6, found 479.5.</div></div><div id="sec4_3_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 4,5-Bis(4-(<i>tert</i>-butyl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)thiazole-2-carboxamide Hydrochloride (<b>36</b>)</h3><div class="NLM_p last"><b>36</b> was prepared from 4-bromothiozole-2-carboxaldehyde, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.96 (br, 1H), 8.85 (br, 1H), 8.53 (br, 1H), 7.48–7.42 (m, 4H), 7.38–7.33 (m, 4H), 3.26–3.20 (m, 4H), 2.82 (br, 2H), 1.87 (br, 1H), 1.82–1.76 (m, 2H), 1.42–1.36 (m, 2H), and 1.28 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 161.0, 152.1, 151.3, 150.4, 137.8, 137.5, 131.7, 129.4, 128.9, 128.3, 126.4, 125.6, 44.2, 43.2, 34.9, 34.8, 34.1, 31.5, 31.4, and 26.6. MS (ESI) calcd for (C<sub>30</sub>H<sub>40</sub>N<sub>3</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 490.7, found 490.6.</div></div><div id="sec4_3_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 1-(4,5-Bis(4-(<i>tert</i>-butyl)phenyl)thiazol-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>37</b>)</h3><div class="NLM_p last"><b>37</b> was prepared from 4-bromothiozole-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.83 (br, 2H), 9.13 (br, 1H), 8.98 (br, 1H), 7.44–7.31 (m, 8H), 4.57 (s, 2H), 3.27–3.24 (m, 2H), 3.03 (br, 2H), 2.85 (br, 2H), 2.10 (br, 1H), 1.98–1.95 (m, 2H), 1.55–1.50 (m, 2H), and 1.27 (s, 18H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 151.4, 150.6, 148.5, 134.2, 131.7, 131.3, 129.0, 128.2, 128.0, 126.0, 125.2, 51.3, 46.8, 42.3, 34.52, 34.42, 31.07, 31.02, 30.6, and 28.1. MS (ESI) calcd for (C<sub>30</sub>H<sub>42</sub>N<sub>3</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 476.7, found 476.5.</div></div><div id="sec4_3_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 1-(4-(4-(<i>tert</i>-Butyl)phenyl)thiazol-2-yl)-<i>N</i>-(piperidin-4-ylmethyl)methanamine Hydrochloride (<b>38</b>)</h3><div class="NLM_p last"><b>38</b> was prepared from 4-bromothiozole-2-carboxaldehyde, following the general method B, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.77 (br, 2H), 9.01 (br, 2H), 8.01 (s, 1H), 7.86 (d, <i>J</i> = 8.4 Hz, 2H), 7.42 (d, <i>J</i> = 8.4 Hz, 2H), 4.55 (s, 2H), 3.27–3.24 (m, 2H), 3.03 (br, 2H), 2.85 (br, 2H), 2.10 (br, 1H), 1.98–1.95 (m, 2H), 1.47–1.39 (m, 2H), and 1.25 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 160.0, 154.3, 150.9, 131.0, 125.9, 125.6, 116.0, 51.2, 46.8, 42.3, 34.4, 31.1, 30.6, and 26.0. MS (ESI) calcd for (C<sub>20</sub>H<sub>30</sub>N<sub>3</sub>S)<sup>+</sup> [M + H]<sup>+</sup> 344.5, found 344.6.</div></div><div id="sec4_3_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 4-(4-(<i>tert</i>-Butyl)phenyl)-<i>N</i>-(piperidin-4-ylmethyl)thiazole-2-carboxamide Hydrochloride (<b>39</b>)</h3><div class="NLM_p last"><b>39</b> was prepared from 4-bromothiozole-2-carboxaldehyde, following the general method A, as a hydrochloric acid salt (white powder). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.99 (br, 1H), 8.90 (br, 1H), 8.68 (br, 1H), 8.34 (s, 1H), 7.98 (d, <i>J</i> = 8.2 Hz, 2H), 7.49 (d, <i>J</i> = 8.2 Hz, 2H), 3.27–3.19 (m, 4H), 2.88–2.79 (m, 2H), 1.91 (br, 1H), 1.83–1.79 (m, 2H), 1.67–1.63 (m, 2H), and 1.31 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 163.5, 159.4, 153.9, 151.2, 130.8, 126.1, 125.5, 118.8, 43.8, 42.9, 34.4, 31.1, 28.1, and 26.3. MS (ESI) calcd for (C<sub>20</sub>H<sub>28</sub>N<sub>3</sub>OS)<sup>+</sup> [M + H]<sup>+</sup> 358.5, found 358.4.</div></div><div id="sec4_3_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> <i>N</i>-((5,6-Bis(4-(furan-3-yl)phenyl)pyrazin-2-yl)methyl)piperidin-4-amine Hydrochloride (<b>40</b>)</h3><div class="NLM_p last">A mixture of 2,3-diaminopropanoic acid hydrochloride (600.4 mg, 4.26 mmol), 1,2-bis(4-bromophenyl)ethane-1,2-dione <b>52</b> (1.745 g, 4.26 mmol), and NaOH (677.2 mg, 16.93 mmol) in methanol (25 mL) was refluxed for 2 h. The mixture was cooled to 25 °C and air was bubbled through the solution for 2 days, after which the pH was adjusted to 2 by HCl. Upon removal of the solvents, the product was extracted with diethyl ether (3 × 50 mL) and the combined organic phases were dried, filtered, and evaporated under reduced pressure. The crude product <b>53</b> was dissolved in THF (22 mL) followed by addition of methyl chloroformate (0.22 mL, 2.8 mmol) and diisopropylethylamine (0.6 mL, 3.5 mmol). The reaction mixture was stirred at room temperature for 5 h. Methanol (1.8 mL) was then added followed by NaBH<sub>4</sub> (537.2 mg, 14.2 mmol) in small portions at 0 °C. Stirring was continued at 0 °C for 1 h. The product was extracted with ether and purified with flash chromatography (silica gel, hexane/ethyl acetate = 5/1) to give the corresponding alcohol (509 mg, 52% for two steps), which was converted to compound <b>40</b> as a pale yellow powder using the reactions vi, vii, ii, and iv described in general methods A and B. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.94 (br, 2H), 8.72 (br, 1H), 8.69 (s, 1H), 8.24 (br, 2H), 7.92 (d, <i>J</i> = 7.1 Hz, 2H), 7.75 (s, 2H), 7.62 (d, <i>J</i> = 7.7 Hz, 4H), 7.45 (d, <i>J</i> = 4.5 Hz, 2H), 7.43 (d, <i>J</i> = 4.5 Hz, 2H), 6.99 (s, 2H), 5.26 (s, 2H), 3.21 (d, <i>J</i> = 10.2 Hz, 2H), 2.94 (br, 2H), 2.79–2.75 (m, 2H), 1.76 (d, <i>J</i> = 14.2 Hz, 2H), 1.69 (br, 1H), and 1.33–1.26 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): 156.0, 150.48, 150.44, 149.5, 144.5, 140.4, 140.02, 139.99, 136.63, 136.57, 132.3, 130.1, 130.0, 129.7, 128.0, 126.4, 125.3, 125.2, 108.58, 108.56, 53.4, 45.3, 42.7, 33.9, and 26.0. MS (ESI) calcd for (C<sub>31</sub>H<sub>31</sub>N<sub>4</sub>O<sub>2</sub>)<sup>+</sup> [M + H]<sup>+</sup> 491.6, found 491.5.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Inhibition of p300-HAT and Other HATs</h3><div class="NLM_p">The HAT domain (1195–1673) of human p300 was cloned, inserted into pGEX-KG vector, and the DNA sequence was verified by sequencing. The p300-HAT expression plasmid was transformed into <i>E. coli</i> BL21-CodonPlus strain (Agilent) and cultured at 37 °C in LB medium containing ampicillin (50 μg/mL) and chloramphenicol (34 μg/mL). After the optical density of the bacterial culture reached ∼0.9 at 600 nm, p300 HAT expression was induced by adding 300 μM isopropylthiogalactoside (IPTG) at 18 °C for 48 h. Cells were collected, lysed, and centrifuged for 20 min at 20 000 rpm. The supernatant was subjected to column chromatography with glutathione sepharose resins. The recombinant GST-p300-HAT fusion protein was obtained in ∼90% purity (SDS–PAGE) by elution with 10 mM glutathione solution, which was further purified using a Superdex 200 gel filtration column chromatography. CBP, PCAF, and Myst3 HATs were obtained using similar methods.</div><div class="NLM_p last">To determine inhibitory activity, a compound with concentrations ranging from 100 nM to 10 μM was incubated with p300-HAT (10 nM) in 20 μL of 50 mM phosphate buffer (pH = 7.0) containing 0.01% Brij-35 for 10 min at 25 °C. Histone H3 peptide (ARTKQTARKSTGGKAPRKQLA) (20 μM) and acetyl-CoA (1 μM <sup>3</sup>H-Ac-CoA and 19 μM Ac-CoA) were added to initiate the reaction. After 30 min at 25 °C, the reaction was stopped by adding 6 N formic acid (5 μL). 20 μL of reaction mixture was then transferred to a small piece of P81 filter paper (Whatman) that binds histone H3 peptide. The filter paper was washed 3× with 50 mM NaHCO<sub>3</sub>, dried, and transferred into a scintillation vial containing 2 mL of scintillation cocktail. Radioactivity on the filter paper was measured with a Beckman LS-6500 scintillation counter. IC<sub>50</sub> values were obtained by using a standard sigmoidal dose–response curve fitting in Prism (version 5.0, GraphPad Software, Inc., La Jolla, CA). IC<sub>50</sub> values were the mean values from at least three experiments.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Alpha Assay</h3><div class="NLM_p last">We followed a published method<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> to investigate whether compound <b>2</b> can disrupt the binding of p300 HAT and histone H4 peptide. In brief, the assay was performed in a 384-well plate using His6-tagged P300 HAT (125 nM) coated nickel chelate acceptor beads (5 μL, PerkinElmer), biotinylated H4 peptide [SGRGKGGKGLGKGGAKRHRKVLRGG-K(biotin)-NH<sub>2</sub>] (30 nM) coated streptavidin donor beads (5 μL, PerkinElmer), and varying concentrations of <b>12</b> in a PBS buffer with 0.5% BSA (final volume of 25 μL). Upon incubation for 1 h, the Alpha signal was determined (laser excitations at 680 nm and reading at 570 nm) using a Tecan Spark microplate reader. The IC<sub>50</sub> values were determined using the sigmoidal dose–response fitting in the program of Prism 5.0 (GraphPad). The biotinylated His6 peptide (PerkinElmer no. 6760302) was used as a reference to eliminate the possible interference of compounds on the signal generated by singlet oxygen transfer.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Molecular Modeling</h3><div class="NLM_p last">Docking studies were performed with Schrödinger small-molecule drug discovery software suite (Schrödinger, LLC, New York, NY, 2017), using our previous published methods.<a onclick="showRef(event, 'ref23 ref35'); return false;" href="javascript:void(0);" class="ref ref23 ref35">(23,35)</a> The crystal structure of p300-HAT in complex with Ac-CoA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PZS">4PZS</a>) was prepared using the module “Protein Preparation Wizard” in Maestro with the default protein parameters. Hydrogen atoms were added, and water molecules were extracted. Ac-CoA was included in the protein structure for docking. Hydrogen bonds were optimized, the partial charges were assigned, and the protein structure was energy-minimized using OPLS-2005 force field. A receptor grid was generated using the program Glide without any constraints. Compound <b>12</b> was constructed, energy-minimized using OPLS-2005 force field in Maestro, and then docked into the prepared protein structure using Glide (docking parameters: standard-precision and dock flexibly).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Western Blot</h3><div class="NLM_p last">10<sup>6</sup> Kasumi-1 cells/well were incubated with compound <b>12</b> at 0, 5, and 10 μM for 12 h. Histone proteins were extracted using EpiQuik total histone extraction kit (Epigentek), according to the manufacturer’s protocol. Equal amounts of histones (2 μg) were separated on SDS–PAGE and transferred to PVDF membranes. The blots were probed with primary antibodies against H3K9Ac, H3K18Ac, H3K27Ac, and H3 (Cell Signaling), followed by anti-rabbit IgG (Thermo Scientific) secondary antibodies.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Cell Growth Inhibition</h3><div class="NLM_p last">The antiproliferation assays for nonbreast cancer cells were performed using our previous method.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> In brief, for Kasumi-1 cells, 10<sup>6</sup> cells per well were added into 96-well plates and cultured with increasing concentrations of a compound in RPMI-1640 medium supplemented with 20% fetal bovine serum and penicillin (100 U/mL) and streptomycin (100 μg/mL) at 37 °C in a 5% CO<sub>2</sub> atmosphere with 100% humidity. For solid tumor cells, 10<sup>5</sup> cells per well were added into 96-well plates and cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum and penicillin (100 U/mL) and streptomycin (100 μg/mL) overnight. Upon addition of increasing concentrations of a compound, cells were incubated for 5 days. Cell viability was assessed by using an XTT assay kit (Roche) for leukemia cells or an MTT assay (Sigma) for attachment cells. For the breast cancer MCF-7 cells, culture and inhibition assays were performed according to our previous methods.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> Compound EC<sub>50</sub> values were calculated from dose–response curves using Prism 5.0.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> RNA Extraction</h3><div class="NLM_p last">Upon starvation (culturing in phenol red-free DMEM medium with 10% charcoal treated FBS to remove hydrophobic hormones such as estrogens) for 7 days, MCF-7 cells were treated with the p300 inhibitor (6 μM) or DMSO control, in the presence or absence of (10 nM) estradiol (E2) for 48 h. Total RNA was extracted using RNeasy Plus Mini Kit (Qiagen), according to the manufacturer’s instructions.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Library Preparation</h3><div class="NLM_p last">The RNA integrity for each sample was assessed with a RNA 6000 Nano chip on a 2100 BioAnalyzer (Agilent; Santa Clara, CA). The average RNA integrity score for the sample set was 6.48. Sequencing libraries were prepared using the TruSeq Stranded Total RNA Library Prep Kit (Illumina, Inc.; San Diego, CA). Concisely, rRNA (rRNA) was depleted from total RNA where the remaining (ribo-depleted RNA) was purified, fragmented for 7.5 min, and primed for cDNA synthesis. Blunt-ended cDNA was generated after first and second strand synthesis. Adenylation of the 3′ blunt-ends was followed by adapter ligation prior to the enrichment of the cDNA fragments. Final library quality control was carried out by measuring the fragment size on a DNA1000 chip on a 2100 BioAnalyzer (Agilent; Santa Clara, CA). The average library yielded an insert bp size of 258. The concentration of each library was determined by quantitative PCR (qPCR) by the KAPA Library Quantification Kit for Next Generation Sequencing (KAPA Biosystems; Woburn, MA).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Sequencing</h3><div class="NLM_p last">Libraries were normalized to 50 nmol/L in 10 mM Tris-Cl, pH 8.5, with 0.1% Tween 20 and then pooled evenly. The 50 nmol/L library pool was diluted to 2 nmol/L with 10 mM Tris-Cl, pH 8.5, with 0.1% Tween 20. The pooled libraries were denatured with 0.05 N NaOH and diluted to 20 pmol/L. Cluster generation of the denatured libraries was performed according to the manufacturer’s specifications (Illumina, Inc.; San Diego, CA) utilizing the HiSeq Rapid PE Cluster Kit v2 chemistry and flow cell. Libraries were clustered appropriately with a 1% PhiX spike-in. Sequencing-by-synthesis (SBS) was performed on a HiSeq 2500 utilizing v2 chemistry with paired-end 101 bp reads and a 6 bp index read culminating in an average output of 20 million paired-end reads (or 40 million total reads) per sample. Sequence read data were processed and converted to FASTQ format by Illumina BaseSpace analysis software (v2.0.13).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Bioinformatics Analysis</h3><div class="NLM_p last">RNAseq data files were trimmed using Trim Galore!<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> Then the data were mapped using Hisat2 onto the human reference genome build UCSC hg38/GRCh38 and sorted using samtools.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> Gene expression was assessed, and fragments per kilobase of transcript per million (FPKM) values were determined using Stringtie.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Gene expression profiles were normalized using the quantile normalization method. Significantly changed genes were determined using a two-sided parametric <i>t</i> test, with significance assessed at <i>p</i> < 0.05; only genes for which at least one sample exceeded 1 FPKM in expression were considered. Enriched pathways were inferred using the gene set enrichment (GSEA) method and the molecular signature database (MSigDB) pathway collection.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72"><a href="/doi/suppl/10.1021/acs.jmedchem.9b02164" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22437" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22437" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b02164?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02164</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Charts S1–S3, Figures S1–S4, representative HPLC traces, and <sup>1</sup>H and <sup>13</sup>C NMR spectra for compound <b>12</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">PDB files for docking models (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_003.zip" class="ext-link">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf">jm9b02164_si_001.pdf (1.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_002.csv">jm9b02164_si_002.csv (3.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_003.zip">jm9b02164_si_003.zip (1.12 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b02164" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03401" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03401" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongcheng Song</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Chemical Biology, 
    Dan L. Duncan Cancer
Center, 
    , , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2611-2476" title="Orcid link">http://orcid.org/0000-0003-2611-2476</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9ce5eff3f2fbdcfefff1b2f9f8e9"><span class="__cf_email__" data-cfemail="a3dad0cccdc4e3c1c0ce8dc6c7d6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fangrui Wu</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Chemical Biology, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuanda Hua</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Chemical Biology, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Salma Kaochar</span> - <span class="hlFld-Affiliation affiliation">Department of Medicine, 
    Dan L. Duncan Cancer
Center, 
    , , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shenyou Nie</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Chemical Biology, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi-Lun Lin</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Chemical Biology, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuan Yao</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Chemical Biology, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jingyu Wu</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Chemical Biology, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaowei Wu</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology & Chemical Biology, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7248-9149" title="Orcid link">http://orcid.org/0000-0001-7248-9149</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyong Fu</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular & Cellular Biology, 
    Lester and Sue Smith Breast Center, 
    , , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachel Schiff</span> - <span class="hlFld-Affiliation affiliation">Department of Medicine, 
    Department of Molecular & Cellular Biology, 
    Lester and Sue Smith Breast Center, 
    , , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christel M. Davis</span> - <span class="hlFld-Affiliation affiliation">Avera Institute
for Human Genetics, Sioux Falls, South Dakota 57108, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew Robertson</span> - <span class="hlFld-Affiliation affiliation">Dan L. Duncan Cancer
Center, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Erik A. Ehli</span> - <span class="hlFld-Affiliation affiliation">Avera Institute
for Human Genetics, Sioux Falls, South Dakota 57108, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cristian Coarfa</span> - <span class="hlFld-Affiliation affiliation">Dan L. Duncan Cancer
Center, , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas Mitsiades</span> - <span class="hlFld-Affiliation affiliation">Department of Medicine, 
    Dan L. Duncan Cancer
Center, 
    , , , , and , Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.W. and Y.H. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82516" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82516" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Cancer Prevention and Research Institute of Texas (CPRIT) Grants RP150129 and RP180177 to Y.S. It was also partly supported by the Breast Cancer Research Foundation Grants 16-142, 17-143, and 18-145 (to R.S.), Susan G. Komen for the Cure Foundation Promise Grant PG12221410 (to R.S.), Department of Defense (Grant W81XWH-14-1-0326 to X.F.), the NCI SPORE Grants P50CA58183 and P50CA186784 (to R.S. and N.M.), the NCI grant U54CA233223 (to S.K. and N.M.), and the NIEHS grants 1P30ES030285 and 1P42ES0327725 (C.C.). The gene expression profiling was supported by the CPRIT Grant RP170005, the Prostate Cancer Foundation Grant 18YOUN09 (to S.K.), and NCI SPORE Grant P50 CA186784 (to N.M.). The authors also acknowledge the assistance of the Dan L Duncan Cancer Center Core Resources (supported by the NCI Cancer Center Support Grant P30CA125123).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">Ac-CoA</td><td class="NLM_def"><p class="first last">acetyl coenzyme A</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">CBP</td><td class="NLM_def"><p class="first last">CREB (cAMP-response element binding protein) binding protein</p></td></tr><tr><td class="NLM_term">CH1</td><td class="NLM_def"><p class="first last">cysteine-histidine rich 1</p></td></tr><tr><td class="NLM_term">CoA</td><td class="NLM_def"><p class="first last">coenzyme A</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">estrogen receptor</p></td></tr><tr><td class="NLM_term">GNAT</td><td class="NLM_def"><p class="first last">Gcn5-related <i>N</i>-acetyltransferase</p></td></tr><tr><td class="NLM_term">GSEA</td><td class="NLM_def"><p class="first last">gene set enrichment analysis</p></td></tr><tr><td class="NLM_term">H3K4</td><td class="NLM_def"><p class="first last">histone H3 lysine 4</p></td></tr><tr><td class="NLM_term">H3K9</td><td class="NLM_def"><p class="first last">histone H3 lysine 9</p></td></tr><tr><td class="NLM_term">H3K27</td><td class="NLM_def"><p class="first last">histone H3 lysine 27</p></td></tr><tr><td class="NLM_term">LSD1</td><td class="NLM_def"><p class="first last">lysine specific demethylase 1</p></td></tr><tr><td class="NLM_term">MYST</td><td class="NLM_def"><p class="first last">MOZ, Ybf2, Sas2, and Tip60p300</p></td></tr><tr><td class="NLM_term">NES</td><td class="NLM_def"><p class="first last">normalized enrichment score</p></td></tr><tr><td class="NLM_term">E1A</td><td class="NLM_def"><p class="first last">binding protein p300</p></td></tr><tr><td class="NLM_term">PCAF</td><td class="NLM_def"><p class="first last">p300/CBP associating factor</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SID</td><td class="NLM_def"><p class="first last">steroid receptor coactivator interaction</p></td></tr><tr><td class="NLM_term">SRC</td><td class="NLM_def"><p class="first last">steroid receptor coactivator</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03463" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span> <span> </span><span class="NLM_article-title">The epigenomics of cancer</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">683</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.01.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.cell.2007.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=17320506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=683-692&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+epigenomics+of+cancer&doi=10.1016%2Fj.cell.2007.01.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenomics of cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-692</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Aberrant gene function and altered patterns of gene expression are key features of cancer.  Growing evidence shows that acquired epigenetic abnormalities participate with genetic alterations to cause this dysregulation.  Here, we review recent advances in understanding how epigenetic alterations participate in the earliest stages of neoplasia, including stem/precursor cell contributions, and discuss the growing implications of these advances for strategies to control cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBpHNAkcSPbVg90H21EOLACvtfcHk0lg6LMvTwvO31A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D&md5=78eac8ecc093f1aed0acbfd578f34717</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520epigenomics%2520of%2520cancer%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D683%26epage%3D692%26doi%3D10.1016%2Fj.cell.2007.01.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Chromatin modifications and their function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lg6LMvTwvO31A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1038/nrc3256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer&doi=10.1038%2Fnrc3256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lg6LMvTwvO31A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477%26doi%3D10.1038%2Fnrc3256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yik, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">Recruitment of P-TEFb for stimulation of transcriptional elongation by the bromodomain protein Brd4</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.06.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.molcel.2005.06.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=16109377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=535-545&author=Z.+Yangauthor=J.+H.+Yikauthor=R.+Chenauthor=N.+Heauthor=M.+K.+Jangauthor=K.+Ozatoauthor=Q.+Zhou&title=Recruitment+of+P-TEFb+for+stimulation+of+transcriptional+elongation+by+the+bromodomain+protein+Brd4&doi=10.1016%2Fj.molcel.2005.06.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Recruitment of P-TEFb for stimulation of transcriptional elongation by the bromodomain protein Brd4</span></div><div class="casAuthors">Yang, Zhiyuan; Yik, Jasper H. N.; Chen, Ruichuan; He, Nanhai; Jang, Moon Kyoo; Ozato, Keiko; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">535-545</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cyclinT1/Cdk9 heterodimer that constitutes core P-TEFb is generally presumed to be the transcriptionally active form for stimulating RNA polymerase II elongation.  About half of cellular P-TEFb also exists in an inactive complex with the 7SK snRNA and the HEXIM1 protein.  Here, we show that the remaining half assocs. with the bromodomain protein Brd4.  In stress-induced cells, the 7SK/HEXIM1-bound P-TEFb is quant. converted into the Brd4-assocd. form.  The assocn. with Brd4 is necessary to form the transcriptionally active P-TEFb, recruits P-TEFb to a promoter, and enables P-TEFb to contact the Mediator complex, a potential target for the Brd4-mediated recruitment.  Although generally required for transcription, the P-TEFb-recruitment function of Brd4 can be substituted by that of HIV-1 Tat, which recruits P-TEFb directly for activated HIV-1 transcription.  Brd4, HEXIM1, and 7SK are all implicated in regulating cell growth, which may result from their dynamic control of the general transcription factor P-TEFb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptFELcZRBnvLVg90H21EOLACvtfcHk0liypD-lzjzP1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur8%253D&md5=50325264e1eb27789eb46999ca3f799d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.06.029%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DYik%26aufirst%3DJ.%2BH.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DRecruitment%2520of%2520P-TEFb%2520for%2520stimulation%2520of%2520transcriptional%2520elongation%2520by%2520the%2520bromodomain%2520protein%2520Brd4%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D535%26epage%3D545%26doi%3D10.1016%2Fj.molcel.2005.06.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiltz, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizzen, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span> <span> </span><span class="NLM_article-title">Overlapping but distinct patterns of histone acetylation by the human coactivators p300 and PCAF within nucleosomal substrates</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">1189</span>– <span class="NLM_lpage">1192</span>, <span class="refDoi"> DOI: 10.1074/jbc.274.3.1189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1074%2Fjbc.274.3.1189" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=9880483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADyaK1MXpsFKktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=1189-1192&author=R.+L.+Schiltzauthor=C.+A.+Mizzenauthor=A.+Vassilevauthor=R.+G.+Cookauthor=C.+D.+Allisauthor=Y.+Nakatani&title=Overlapping+but+distinct+patterns+of+histone+acetylation+by+the+human+coactivators+p300+and+PCAF+within+nucleosomal+substrates&doi=10.1074%2Fjbc.274.3.1189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Overlapping but distinct patterns of histone acetylation by the human coactivators p300 and PCAF within nucleosomal substrates</span></div><div class="casAuthors">Schiltz, R. Louis; Mizzen, Craig A.; Vassilev, Alex; Cook, Richard G.; Allis, C. David; Nakatani, Yoshihiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1189-1192</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A no. of transcriptional coactivators possess intrinsic histone acetylase activity, providing a direct link between hyperacetylated chromatin and transcriptional activation.  We have detd. the core histone residues acetylated in vitro by recombinant p300 and PCAF within mononucleosomes.  P300 specifically acetylates all sites of histones H2A and H2B known to be acetylated in bulk chromatin in vivo but preferentially acetylates lysines 14 and 18 of histone H3 and lysines 5 and 8 of histone H4.  PCAF primarily acetylates lysine 14 of H3 but also less efficiently acetylates lysine 8 of H4.  PCAF in its native form, which is present in a stable multimeric protein complex lacking p300/CBP, primarily acetylates H3 to a monoacetylated form, suggesting that PCAF-assocd. polypeptides do not alter the substrate specificity.  These distinct patterns of acetylation by the p300 and PCAF may contribute to their differential roles in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMMQsOC8Vdl7Vg90H21EOLACvtfcHk0liypD-lzjzP1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXpsFKktw%253D%253D&md5=b7d50ee9c8640dcabd7baf63473e69fc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.3.1189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.3.1189%26sid%3Dliteratum%253Aachs%26aulast%3DSchiltz%26aufirst%3DR.%2BL.%26aulast%3DMizzen%26aufirst%3DC.%2BA.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DCook%26aufirst%3DR.%2BG.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DNakatani%26aufirst%3DY.%26atitle%3DOverlapping%2520but%2520distinct%2520patterns%2520of%2520histone%2520acetylation%2520by%2520the%2520human%2520coactivators%2520p300%2520and%2520PCAF%2520within%2520nucleosomal%2520substrates%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D1189%26epage%3D1192%26doi%3D10.1074%2Fjbc.274.3.1189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dancy, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Protein lysine acetylation by p300/CBP</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">2419</span>– <span class="NLM_lpage">2452</span>, <span class="refDoi"> DOI: 10.1021/cr500452k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500452k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=2419-2452&author=B.+M.+Dancyauthor=P.+A.+Cole&title=Protein+lysine+acetylation+by+p300%2FCBP&doi=10.1021%2Fcr500452k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Acetylation by p300/CBP</span></div><div class="casAuthors">Dancy, Beverley M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2419-2452</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The human protein lysine acetyltransferase encoded by the paralogous p300 and CBP genes is involved in regulating the expression of genes.  In this capacity, p300/CBP controls several basic cellular processes such as proliferation and homeostasis, and plays a role in a variety of human diseases, particularly solid tumors.  This review summarizes information related to the current understanding of p300/CBP, including the novel technologies that have co-evolved with p300/CBP research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYgTImMiuo7rVg90H21EOLACvtfcHk0liypD-lzjzP1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D&md5=40a46cf68990f4a7d11a338c54081628</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fcr500452k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500452k%26sid%3Dliteratum%253Aachs%26aulast%3DDancy%26aufirst%3DB.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DProtein%2520lysine%2520acetylation%2520by%2520p300%252FCBP%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D2419%26epage%3D2452%26doi%3D10.1021%2Fcr500452k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikura, M.</span></span> <span> </span><span class="NLM_article-title">Transcriptional/epigenetic regulator CBP/p300 in tumorigenesis: structural and functional versatility in target recognition</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">3989</span>– <span class="NLM_lpage">4008</span>, <span class="refDoi"> DOI: 10.1007/s00018-012-1254-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1007%2Fs00018-012-1254-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=23307074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Wmt7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2013&pages=3989-4008&author=F.+Wangauthor=C.+B.+Marshallauthor=M.+Ikura&title=Transcriptional%2Fepigenetic+regulator+CBP%2Fp300+in+tumorigenesis%3A+structural+and+functional+versatility+in+target+recognition&doi=10.1007%2Fs00018-012-1254-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional/epigenetic regulator CBP/p300 in tumorigenesis: structural and functional versatility in target recognition</span></div><div class="casAuthors">Wang, Feng; Marshall, Christopher B.; Ikura, Mitsuhiko</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3989-4008</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">A review.  In eukaryotic cells, gene transcription is regulated by sequence-specific DNA-binding transcription factors that recognize promoter and enhancer elements near the transcriptional start site.  Some coactivators promote transcription by connecting transcription factors to the basal transcriptional machinery.  The highly conserved coactivators CREB-binding protein (CBP) and its paralog, E1A-binding protein (p300), each have four sep. transactivation domains (TADs) that interact with the TADs of a no. of DNA-binding transcription activators as well as general transcription factors (GTFs), thus mediating recruitment of basal transcription machinery to the promoter.  Most promoters comprise multiple activator-binding sites, and many activators contain tandem TADs, thus multivalent interactions may stabilize CBP/p300 at the promoter, and intrinsically disordered regions in CBP/p300 and many activators may confer adaptability to these multivalent complexes.  CBP/p300 contains a catalytic histone acetyltransferase (HAT) domain, which remodels chromatin to 'relax' its superstructure and enables transcription of proximal genes.  The HAT activity of CBP/p300 also acetylates some transcription factors (e.g., p53), hence modulating the function of key transcriptional regulators.  Through these numerous interactions, CBP/p300 has been implicated in complex physiol. and pathol. processes, and, in response to different signals, can drive cells towards proliferation or apoptosis.  Dysregulation of the transcriptional and epigenetic functions of CBP/p300 is assocd. with leukemia and other types of cancer, thus it has been recognized as a potential anti-cancer drug target.  In this review, we focus on recent exciting findings in the structural mechanisms of CBP/p300 involving multivalent and dynamic interactions with binding partners, which may pave new avenues for anti-cancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCCkxACa47q7Vg90H21EOLACvtfcHk0lj8zg_G5BXHzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Wmt7rJ&md5=76333cf8a4e24cab255412d2655cdd72</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00018-012-1254-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-012-1254-4%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMarshall%26aufirst%3DC.%2BB.%26aulast%3DIkura%26aufirst%3DM.%26atitle%3DTranscriptional%252Fepigenetic%2520regulator%2520CBP%252Fp300%2520in%2520tumorigenesis%253A%2520structural%2520and%2520functional%2520versatility%2520in%2520target%2520recognition%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2013%26volume%3D70%26spage%3D3989%26epage%3D4008%26doi%3D10.1007%2Fs00018-012-1254-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Malley, B. W.</span></span> <span> </span><span class="NLM_article-title">Steroid receptor coactivators 1, 2, and 3: critical regulators of nuclear receptor activity and steroid receptor modulator (SRM)-based cancer therapy</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2011.04.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.mce.2011.04.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=21664237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFegsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2012&pages=430-439&author=A.+B.+Johnsonauthor=B.+W.+O%E2%80%99Malley&title=Steroid+receptor+coactivators+1%2C+2%2C+and+3%3A+critical+regulators+of+nuclear+receptor+activity+and+steroid+receptor+modulator+%28SRM%29-based+cancer+therapy&doi=10.1016%2Fj.mce.2011.04.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Steroid receptor coactivators 1, 2, and 3: Critical regulators of nuclear receptor activity and steroid receptor modulator (SRM)-based cancer therapy</span></div><div class="casAuthors">Johnson, Amber B.; O'Malley, Bert W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">430-439</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Coactivators are a diverse group of non-DNA binding proteins that induce structural changes in agonist-bound nuclear receptors (NRs) that are essential for NR-mediated transcriptional activation.  Once bound, coactivators function to bridge enhancer binding proteins to the general transcription machinery, as well as to recruit secondary coactivators that modify promoter and enhancer chromatin in a manner permissive for transcriptional activation.  In the following review article, we focus on one of the most in-depth studied families of coactivators, the steroid receptor coactivators (SRC) 1, 2, and 3.  SRCs are widely implicated in NR-mediated diseases, esp. in cancers, with the majority of studies focused on their roles in breast cancer.  We highlight the relevant literature supporting the oncogenic activity of SRCs and their future as diagnostic and prognostic indicators.  With much interest in the development of selective receptor modulators (SRMs), we focus on how these coactivators regulate the interactions between SRMs and their resp. NRs; and, importantly, the influence that coactivators have on the functional output of SRMs.  Furthermore, we speculate that coactivator-specific inhibitors could provide powerful, all-encompassing treatments that target multiple modes of oncogenic regulation in cancers resistant to typical anti-endocrine treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2-LVSF6p5yLVg90H21EOLACvtfcHk0lj8zg_G5BXHzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFegsbzM&md5=07d8c01d3f3d11ff0e4bde556fbab664</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2011.04.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2011.04.021%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DA.%2BB.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DB.%2BW.%26atitle%3DSteroid%2520receptor%2520coactivators%25201%252C%25202%252C%2520and%25203%253A%2520critical%2520regulators%2520of%2520nuclear%2520receptor%2520activity%2520and%2520steroid%2520receptor%2520modulator%2520%2528SRM%2529-based%2520cancer%2520therapy%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2012%26volume%3D348%26spage%3D430%26epage%3D439%26doi%3D10.1016%2Fj.mce.2011.04.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dilworth, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambon, P.</span></span> <span> </span><span class="NLM_article-title">Nuclear receptors coordinate the activities of chromatin remodeling complexes and coactivators to facilitate initiation of transcription</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3047</span>– <span class="NLM_lpage">3054</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fsj.onc.1204329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=11420720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1Glurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=3047-3054&author=F.+J.+Dilworthauthor=P.+Chambon&title=Nuclear+receptors+coordinate+the+activities+of+chromatin+remodeling+complexes+and+coactivators+to+facilitate+initiation+of+transcription&doi=10.1038%2Fsj.onc.1204329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear receptors coordinate the activities of chromatin remodeling complexes and coactivators to facilitate initiation of transcription</span></div><div class="casAuthors">Dilworth, F. Jeffrey; Chambon, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3047-3054</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent advances in the field of in vitro chromatin assembly have led to in vitro transcription systems which reproduce in the test tube, in vivo characteristics of ligand-dependent transcriptional activation by nuclear receptors.  Dissection of these systems has begun to provide us with information concerning the underlying mol. mechanisms.  Through recruitment of coactivator proteins, nuclear receptors act first to remodel chromatin within the promoter region and then to recruit the transcriptional machinery to the promoter region in order to initiate transcription.  Here we present a possible sequential mechanism for ligand-dependent transcriptional activation by nuclear receptors and discuss the in vitro and in vivo data that support this model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqisxFLqc8kKbVg90H21EOLACvtfcHk0lj8zg_G5BXHzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1Glurw%253D&md5=b99fe17405e01dd20f906c7169e2f842</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204329%26sid%3Dliteratum%253Aachs%26aulast%3DDilworth%26aufirst%3DF.%2BJ.%26aulast%3DChambon%26aufirst%3DP.%26atitle%3DNuclear%2520receptors%2520coordinate%2520the%2520activities%2520of%2520chromatin%2520remodeling%2520complexes%2520and%2520coactivators%2520to%2520facilitate%2520initiation%2520of%2520transcription%26jtitle%3DOncogene%26date%3D2001%26volume%3D20%26spage%3D3047%26epage%3D3054%26doi%3D10.1038%2Fsj.onc.1204329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manning, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadonaga, J. T.</span></span> <span> </span><span class="NLM_article-title">Biochemical analysis of distinct activation functions in p300 that enhance transcription initiation with chromatin templates</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">8123</span>– <span class="NLM_lpage">8135</span>, <span class="refDoi"> DOI: 10.1128/MCB.19.12.8123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1128%2FMCB.19.12.8123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10567538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ygt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1999&pages=8123-8135&author=W.+L.+Krausauthor=E.+T.+Manningauthor=J.+T.+Kadonaga&title=Biochemical+analysis+of+distinct+activation+functions+in+p300+that+enhance+transcription+initiation+with+chromatin+templates&doi=10.1128%2FMCB.19.12.8123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical analysis of distinct activation functions in p300 that enhance transcription initiation with chromatin templates</span></div><div class="casAuthors">Kraus, W. Lee; Manning, E. Tory; Kadonaga, James T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8123-8135</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">To investigate the mechanisms of transcriptional enhancement by the p300 coactivator, we analyzed wild-type and mutant versions of p300 with a chromatin transcription system in vitro.  Estrogen receptor, NF-κB p65 plus Sp1, and Gal4-VP16 were used as different sequence-specific activators.  The CH3 domain (or E1A-binding region) was found to be essential for the function of each of the activators tested.  The bromodomain was also obsd. to be generally important for p300 coactivator activity, though to a lesser extent than the CH3 domain/E1A-binding region.  The acetyltransferase activity and the C-terminal region (contg. the steroid receptor coactivator/p160-binding region and the glutamine-rich region) were each found to be important for activation by estrogen receptor but not for that by Gal4-VP16.  The N-terminal region of p300, which had been previously found to interact with nuclear hormone receptors, was not seen to be required for any of the activators, including estrogen receptor.  Single-round transcription expts. revealed that the functionally important subregions of p300 contribute to its ability to promote the assembly of transcription initiation complexes.  In addn., the acetyltransferase activity of p300 was obsd. to be distinct from the broadly essential activation function of the CH3 domain/E1A-binding region.  These results indicate that specific regions of p300 possess distinct activation functions that are differentially required to enhance the assembly of transcription initiation complexes.  Interestingly, with the estrogen receptor, four distinct regions of p300 each have an essential role in the transcription activation process.  These data exemplify a situation in which a network of multiple activation functions is required to achieve gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcZAOnfpDfObVg90H21EOLACvtfcHk0lhCbl_K5kn0Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXns1ygt74%253D&md5=b13149bf90d7f7d302a833df34ed14f6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.19.12.8123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.19.12.8123%26sid%3Dliteratum%253Aachs%26aulast%3DKraus%26aufirst%3DW.%2BL.%26aulast%3DManning%26aufirst%3DE.%2BT.%26aulast%3DKadonaga%26aufirst%3DJ.%2BT.%26atitle%3DBiochemical%2520analysis%2520of%2520distinct%2520activation%2520functions%2520in%2520p300%2520that%2520enhance%2520transcription%2520initiation%2520with%2520chromatin%2520templates%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1999%26volume%3D19%26spage%3D8123%26epage%3D8135%26doi%3D10.1128%2FMCB.19.12.8123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">Synergistic activation of transcription by CBP and p53</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">819</span>– <span class="NLM_lpage">823</span>, <span class="refDoi"> DOI: 10.1038/42972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2F42972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=9194564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADyaK2sXktVyjsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=1997&pages=819-823&author=W.+Guauthor=X.+L.+Shiauthor=R.+G.+Roeder&title=Synergistic+activation+of+transcription+by+CBP+and+p53&doi=10.1038%2F42972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic activation of transcription by CBP and p53</span></div><div class="casAuthors">Gu, Wei; Shi, Xiao-Lu; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">6635</span>),
    <span class="NLM_cas:pages">819-823</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The tumor suppressor p53 is a transcriptional regulator whose ability to inhibit cell growth is dependent upon its transactivation function.  Here the authors demonstrate that the transcription factor CBP, which is also implicated in cell proliferation and differentiation, act as a p53 coactivator and potentiates its transcriptional activity.  The amino-terminal activation domain of p53 interacts with the carboxy-terminal portion of the CBP protein both in vitro and in vivo.  In transfected SaoS-2 cells, CBP potentiates activation of the mdm-2 gene by p53 and, reciprocally, p53 potentiates activation of a Gal4-responsive target gene by a Gal4(1-147)-CBP(1678-2441) fusion protein.  A double point mutation that destroys the transactivation function of p53 also abolishes its binding to CBP and its synergistic function with CBP.  The ability of p53 to interact phys. and functionally with a coactivator (CBP) that has histone acetyltransferase activity and with components (TAFs) of the general transcription machinery indicates that it may have different functions in a multistep activation pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvql5Yg3ZCM7Vg90H21EOLACvtfcHk0lhCbl_K5kn0Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktVyjsbw%253D&md5=716c92c46feff7d9b01bafa62da3e61e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2F42972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F42972%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%2BL.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DSynergistic%2520activation%2520of%2520transcription%2520by%2520CBP%2520and%2520p53%26jtitle%3DNature%26date%3D1997%26volume%3D387%26spage%3D819%26epage%3D823%26doi%3D10.1038%2F42972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vervoorts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscher-Firzlaff, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rottmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lilischkis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsemann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dohmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscher, B.</span></span> <span> </span><span class="NLM_article-title">Stimulation of c-MYC transcriptional activity and acetylation by recruitment of the cofactor CBP</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">490</span>, <span class="refDoi"> DOI: 10.1038/sj.embor.embor821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fsj.embor.embor821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=12776737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtlaks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=484-490&author=J.+Vervoortsauthor=J.+M.+Luscher-Firzlaffauthor=S.+Rottmannauthor=R.+Lilischkisauthor=G.+Walsemannauthor=K.+Dohmannauthor=M.+Austenauthor=B.+Luscher&title=Stimulation+of+c-MYC+transcriptional+activity+and+acetylation+by+recruitment+of+the+cofactor+CBP&doi=10.1038%2Fsj.embor.embor821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Stimulation of c-MYC transcriptional activity and acetylation by recruitment of the cofactor CBP</span></div><div class="casAuthors">Vervoorts, Joerg; Luescher-Firzlaff, Juliane M.; Rottmann, Sabine; Lilischkis, Richard; Walsemann, Gesa; Dohmann, Karen; Austen, Matthias; Luescher, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">484-490</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The c-MYC oncoprotein regulates various aspects of cell behavior by modulating gene expression.  Here, we report the identification of the cAMP-response-element-binding protein (CBP) as a novel c-MYC binding partner.  The two proteins interact both in vitro and in cells, and CBP binds to the carboxy-terminal region of c-MYC.  Importantly, CBP, as well as p300, is assocd. with E-box-contg. promoter regions of genes that are regulated by c-MYC.  Furthermore, c-MYC and CBP/p300 function synergistically in the activation of reporter-gene constructs.  Thus, CBP and p300 function as pos. cofactors for c-MYC.  In addn., c-MYC is acetylated in cells.  This modification does not require MYC box II, suggesting that it is independent of TRRAP complexes.  Instead, CBP acetylates c-MYC in vitro, and co-expression of CBP with c-MYC stimulates in vivo acetylation.  Functionally, this results in a decrease in ubiquitination and stabilization of c-MYC proteins.  Thus, CBP and p300 are novel functional binding partners of c-MYC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLtwFRV0LMVrVg90H21EOLACvtfcHk0lhCbl_K5kn0Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtlaks7c%253D&md5=79df9e56a829f6b1e453ec4543ae82b7</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.embor.embor821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.embor.embor821%26sid%3Dliteratum%253Aachs%26aulast%3DVervoorts%26aufirst%3DJ.%26aulast%3DLuscher-Firzlaff%26aufirst%3DJ.%2BM.%26aulast%3DRottmann%26aufirst%3DS.%26aulast%3DLilischkis%26aufirst%3DR.%26aulast%3DWalsemann%26aufirst%3DG.%26aulast%3DDohmann%26aufirst%3DK.%26aulast%3DAusten%26aufirst%3DM.%26aulast%3DLuscher%26aufirst%3DB.%26atitle%3DStimulation%2520of%2520c-MYC%2520transcriptional%2520activity%2520and%2520acetylation%2520by%2520recruitment%2520of%2520the%2520cofactor%2520CBP%26jtitle%3DEMBO%2520Rep.%26date%3D2003%26volume%3D4%26spage%3D484%26epage%3D490%26doi%3D10.1038%2Fsj.embor.embor821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kushner, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coetzee, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stallcup, M. R.</span></span> <span> </span><span class="NLM_article-title">Multiple signal input and output domains of the 160-kilodalton nuclear receptor coactivator proteins</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6164</span>– <span class="NLM_lpage">6173</span>, <span class="refDoi"> DOI: 10.1128/MCB.19.9.6164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1128%2FMCB.19.9.6164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10454563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADyaK1MXlslWjsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1999&pages=6164-6173&author=H.+Maauthor=H.+Hongauthor=S.+M.+Huangauthor=R.+A.+Irvineauthor=P.+Webbauthor=P.+J.+Kushnerauthor=G.+A.+Coetzeeauthor=M.+R.+Stallcup&title=Multiple+signal+input+and+output+domains+of+the+160-kilodalton+nuclear+receptor+coactivator+proteins&doi=10.1128%2FMCB.19.9.6164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple signal input and output domains of the 160-kilodalton nuclear receptor coactivator proteins</span></div><div class="casAuthors">Ma, Han; Hong, Heng; Huang, Shih-Ming; Irvine, Ryan A.; Webb, Paul; Kushner, Peter J.; Coetzee, Gerhard A.; Stallcup, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">6164-6173</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Members of the 160-kDa nuclear receptor coactivator family (p160 coactivators) bind to the conserved AF-2 activation function found in the hormone binding domains of nuclear receptors (NR) and are potent transcriptional coactivators for NRs.  Here we report that the C-terminal region of p160 coactivators glucocorticoid receptor interacting protein 1 (GRIP1), steroid receptor coactivator 1 (SRC-1a), and SRC-1e binds the N-terminal AF-1 activation function of the androgen receptor (AR), and p160 coactivators can thereby enhance transcriptional activation by AR.  While they all interact efficiently with AR AF-1, these same coactivators have vastly different binding strengths with and coactivator effects on AR AF-2.  P160 activation domain AD1, which binds secondary coactivators CREB binding protein (CBP) and p300, was previously implicated as the principal domain for transmitting the activating signal to the transcription machinery.  We identified a new highly conserved motif in the AD1 region which is important for CBP/p300 binding.  Deletion of AD1 only partially reduced p160 coactivator function, due to signaling through AD2, another activation domain located at the C-terminal end of p160 coactivators.  C-terminal coactivator fragments lacking AD1 but contg. AD2 and the AR AF-1 binding site served as efficient coactivators for full-length AR and AR AF-1.  The two signal input domains (one that binds NR AF-2 domains and one that binds AF-1 domains of some but not all NRs) and the two signal output domains (AD1 and AD2) of p160 coactivators played different relative roles for two different NRs: AR and thyroid hormone receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVz-XWR7pJGLVg90H21EOLACvtfcHk0lgSdQ8QNPR41g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlslWjsLc%253D&md5=40252087d311cf794e805c5cc91e5172</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.19.9.6164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.19.9.6164%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DIrvine%26aufirst%3DR.%2BA.%26aulast%3DWebb%26aufirst%3DP.%26aulast%3DKushner%26aufirst%3DP.%2BJ.%26aulast%3DCoetzee%26aufirst%3DG.%2BA.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26atitle%3DMultiple%2520signal%2520input%2520and%2520output%2520domains%2520of%2520the%2520160-kilodalton%2520nuclear%2520receptor%2520coactivator%2520proteins%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1999%26volume%3D19%26spage%3D6164%26epage%3D6173%26doi%3D10.1128%2FMCB.19.9.6164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N.</span></span> <span> </span><span class="NLM_article-title">Chromatin-related properties of CBP fused to MLL generate a myelodysplastic-like syndrome that evolves into myeloid leukemia</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4655</span>– <span class="NLM_lpage">4664</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.17.4655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1093%2Femboj%2F19.17.4655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10970858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmslGku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=4655-4664&author=C.+Lavauauthor=C.+Duauthor=M.+Thirmanauthor=N.+Zeleznik-Le&title=Chromatin-related+properties+of+CBP+fused+to+MLL+generate+a+myelodysplastic-like+syndrome+that+evolves+into+myeloid+leukemia&doi=10.1093%2Femboj%2F19.17.4655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin-related properties of CBP fused to MLL generate a myelodysplastic-like syndrome that evolves into myeloid leukemia</span></div><div class="casAuthors">Lavau, Catherine; Du, Changchun; Thirman, Michael; Zeleznik-Le, Nancy</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4655-4664</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">As a result of the recurring translocation t(11;16) (q23;p13.3), MLL (mixed-lineage leukemia) is fused in frame to CBP (CREB binding protein).  This translocation has been documented almost exclusively in cases of acute leukemia or myelodysplasia secondary to therapy with drugs that target DNA topoisomerase II.  The minimal chimeric protein that is produced fuses MLL to the bromodomain, histone acetyltransferase (HAT) domain, EIA-binding domain and steroid-receptor coactivator binding domains of CBP.  We show that transplantation of bone marrow retrovirally transduced with MLL-CBP induces myeloid leukemias in mice that are preceded by a long preleukemic phase similar to the myelodysplastic syndrome (MDS) seen in many t(11;16) patients but unusual for other MLL translocations.  Structure-function anal. demonstrated that fusion of both the bromodomain and HAT domain of CBP to the amino portion of MLL is required for full in vitro transformation and is sufficient to induce the leukemic phenotype in vivo.  This suggests that the leukemic effect of MLL-CBP results from the fusion of the chromatin assocn. and modifying activities of CBP with the DNA binding activities of MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKd2exsNd2TbVg90H21EOLACvtfcHk0lgSdQ8QNPR41g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmslGku7k%253D&md5=6b48fda474ab9795cb043ebe55962fd2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.17.4655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.17.4655%26sid%3Dliteratum%253Aachs%26aulast%3DLavau%26aufirst%3DC.%26aulast%3DDu%26aufirst%3DC.%26aulast%3DThirman%26aufirst%3DM.%26aulast%3DZeleznik-Le%26aufirst%3DN.%26atitle%3DChromatin-related%2520properties%2520of%2520CBP%2520fused%2520to%2520MLL%2520generate%2520a%2520myelodysplastic-like%2520syndrome%2520that%2520evolves%2520into%2520myeloid%2520leukemia%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D4655%26epage%3D4664%26doi%3D10.1093%2Femboj%2F19.17.4655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gervais, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helias, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eischen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippert, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tigaud, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penther, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugneret, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speleman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talmant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanDen Akker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luquet, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen-Khac, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maarek, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sainty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flandrin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birnbaum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mozziconacci, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessard, M.</span></span> <span> </span><span class="NLM_article-title">Acute myeloid leukaemia with 8p11 (MYST3) rearrangement: an integrated cytologic, cytogenetic and molecular study by the groupe francophone de cytogenetique hematologique</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1567</span>– <span class="NLM_lpage">1575</span>, <span class="refDoi"> DOI: 10.1038/leu.2008.128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fleu.2008.128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=18528428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpslSntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1567-1575&author=C.+Gervaisauthor=A.+Muratiauthor=C.+Heliasauthor=S.+Struskiauthor=A.+Eischenauthor=E.+Lippertauthor=I.+Tigaudauthor=D.+Pentherauthor=C.+Bastardauthor=F.+Mugneretauthor=B.+Poppeauthor=F.+Spelemanauthor=P.+Talmantauthor=J.+VanDen+Akkerauthor=L.+Barangerauthor=C.+Barinauthor=I.+Luquetauthor=N.+Nadalauthor=F.+Nguyen-Khacauthor=O.+Maarekauthor=C.+Herensauthor=D.+Saintyauthor=G.+Flandrinauthor=D.+Birnbaumauthor=M.+J.+Mozziconacciauthor=M.+Lessard&title=Acute+myeloid+leukaemia+with+8p11+%28MYST3%29+rearrangement%3A+an+integrated+cytologic%2C+cytogenetic+and+molecular+study+by+the+groupe+francophone+de+cytogenetique+hematologique&doi=10.1038%2Fleu.2008.128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Acute myeloid leukaemia with 8p11 (MYST3) rearrangement: an integrated cytologic, cytogenetic and molecular study by the groupe francophone de cytogenetique hematologique</span></div><div class="casAuthors">Gervais, C.; Murati, A.; Helias, C.; Struski, S.; Eischen, A.; Lippert, E.; Tigaud, I.; Penther, D.; Bastard, C.; Mugneret, F.; Poppe, B.; Speleman, F.; Talmant, P.; Van Den Akker, J.; Baranger, L.; Barin, C.; Luquet, I.; Nadal, N.; Nguyen-Khac, F.; Maarek, O.; Herens, C.; Sainty, D.; Flandrin, G.; Birnbaum, D.; Mozziconacci, M.-J.; Lessard, M.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1567-1575</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thirty cases of acute myeloid leukemia (AML) with MYST histone acetyltransferase 3 (MYST3) rearrangement were collected in a retrospective study from 14 centers in France and Belgium.  The mean age at diagnosis was 59.4 years and 67% of the patients were females.  Most cases (77%) were secondary to solid cancer (57%), haematol. malignancy (35%) or both (8%), and appeared 25 mo after the primary disease.  Clin., cutaneous localization and disseminated intravascular coagulation were present in 30 and 40% of the cases, resp.  AMLs were myelomonocytic (7%) or monocytic (93%), with erythrophagocytosis (75%) and cytoplasmic vacuoles (75%).  Immunophenotype showed no particularity compared with monocytic leukemia without MYST3 abnormality.  Twenty-eight cases carried t(8;16)(p11;p13) with MYST3-CREBBP fusion, one case carried a variant t(8;22)(p11;q13) and one case carried a t(8;19)(p11;q13).  Type I (MYST3 exon 16-CREBBP exon 3) was the most frequent MYST3-CREBBP fusion transcript (65%).  MYST3 rearrangement was assocd. with a poor prognosis, as 50% of patients deceased during the first 10 mo.  All those particular clin., cytol., cytogenetic, mol. and prognostic characteristics of AML with MYST3 rearrangement may have allowed an individualization into the World Health Organization classification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0XL7JYU4FOLVg90H21EOLACvtfcHk0lgSdQ8QNPR41g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpslSntbg%253D&md5=7d4824ac2008c2fd6e3d3685b4b98da2</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fleu.2008.128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2008.128%26sid%3Dliteratum%253Aachs%26aulast%3DGervais%26aufirst%3DC.%26aulast%3DMurati%26aufirst%3DA.%26aulast%3DHelias%26aufirst%3DC.%26aulast%3DStruski%26aufirst%3DS.%26aulast%3DEischen%26aufirst%3DA.%26aulast%3DLippert%26aufirst%3DE.%26aulast%3DTigaud%26aufirst%3DI.%26aulast%3DPenther%26aufirst%3DD.%26aulast%3DBastard%26aufirst%3DC.%26aulast%3DMugneret%26aufirst%3DF.%26aulast%3DPoppe%26aufirst%3DB.%26aulast%3DSpeleman%26aufirst%3DF.%26aulast%3DTalmant%26aufirst%3DP.%26aulast%3DVanDen%2BAkker%26aufirst%3DJ.%26aulast%3DBaranger%26aufirst%3DL.%26aulast%3DBarin%26aufirst%3DC.%26aulast%3DLuquet%26aufirst%3DI.%26aulast%3DNadal%26aufirst%3DN.%26aulast%3DNguyen-Khac%26aufirst%3DF.%26aulast%3DMaarek%26aufirst%3DO.%26aulast%3DHerens%26aufirst%3DC.%26aulast%3DSainty%26aufirst%3DD.%26aulast%3DFlandrin%26aufirst%3DG.%26aulast%3DBirnbaum%26aufirst%3DD.%26aulast%3DMozziconacci%26aufirst%3DM.%2BJ.%26aulast%3DLessard%26aufirst%3DM.%26atitle%3DAcute%2520myeloid%2520leukaemia%2520with%25208p11%2520%2528MYST3%2529%2520rearrangement%253A%2520an%2520integrated%2520cytologic%252C%2520cytogenetic%2520and%2520molecular%2520study%2520by%2520the%2520groupe%2520francophone%2520de%2520cytogenetique%2520hematologique%26jtitle%3DLeukemia%26date%3D2008%26volume%3D22%26spage%3D1567%26epage%3D1575%26doi%3D10.1038%2Fleu.2008.128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lasko, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Digiammarino, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaves, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algire, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karukurichi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluge, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patane, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Drie, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElligott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesicki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, K. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of a selective catalytic p300/CBP inhibitor that targets lineage-specific tumours</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>550</i></span>,  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nature24028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fnature24028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=28953875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=128-132&author=L.+M.+Laskoauthor=C.+G.+Jakobauthor=R.+P.+Edaljiauthor=W.+Qiuauthor=D.+Montgomeryauthor=E.+L.+Digiammarinoauthor=T.+M.+Hansenauthor=R.+M.+Risiauthor=R.+Freyauthor=V.+Manavesauthor=B.+Shawauthor=M.+Algireauthor=P.+Hesslerauthor=L.+T.+Lamauthor=T.+Uzielauthor=E.+Faivreauthor=D.+Fergusonauthor=F.+G.+Buchananauthor=R.+L.+Martinauthor=M.+Torrentauthor=G.+G.+Chiangauthor=K.+Karukurichiauthor=J.+W.+Langstonauthor=B.+T.+Weinertauthor=C.+Choudharyauthor=P.+de+Vriesauthor=A.+F.+Klugeauthor=M.+A.+Pataneauthor=J.+H.+Van+Drieauthor=C.+Wangauthor=D.+McElligottauthor=E.+Kesickiauthor=R.+Marmorsteinauthor=C.+Sunauthor=P.+A.+Coleauthor=S.+H.+Rosenbergauthor=M.+R.+Michaelidesauthor=A.+Laiauthor=K.+D.+Bromberg&title=Discovery+of+a+selective+catalytic+p300%2FCBP+inhibitor+that+targets+lineage-specific+tumours&doi=10.1038%2Fnature24028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective catalytic p300/CBP inhibitor that targets lineage-specific tumours</span></div><div class="casAuthors">Lasko, Loren M.; Jakob, Clarissa G.; Edalji, Rohinton P.; Qiu, Wei; Montgomery, Debra; Digiammarino, Enrico L.; Hansen, T. Matt; Risi, Roberto M.; Frey, Robin; Manaves, Vlasios; Shaw, Bailin; Algire, Mikkel; Hessler, Paul; Lam, Lloyd T.; Uziel, Tamar; Faivre, Emily; Ferguson, Debra; Buchanan, Fritz G.; Martin, Ruth L.; Torrent, Maricel; Chiang, Gary G.; Karukurichi, Kannan; Langston, J. William; Weinert, Brian T.; Choudhary, Chunaram; de Vries, Peter; Van Drie, John H.; McElligott, David; Kesicki, Ed; Marmorstein, Ronen; Sun, Chaohong; Cole, Philip A.; Rosenberg, Saul H.; Michaelides, Michael R.; Lai, Albert; Bromberg, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">550</span>
        (<span class="NLM_cas:issue">7674</span>),
    <span class="NLM_cas:pages">128-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The dynamic and reversible acetylation of proteins, catalyzed by histone acetyltransferases (HATs) and histone deacetylases (HDACs), is a major epigenetic regulatory mechanism of gene transcription and is assocd. with multiple diseases.  Histone deacetylase inhibitors are currently approved to treat certain cancers, but progress on the development of drug-like histone actyltransferase inhibitors has lagged behind.  The histone acetyltransferase paralogues p300 and CREB-binding protein (CBP) are key transcriptional co-activators that are essential for a multitude of cellular processes, and have also been implicated in human pathol. conditions (including cancer).  Current inhibitors of the p300 and CBP histone acetyltransferase domains, including natural products, bi-substrate analogs and the widely used small mol. C646, lack potency or selectivity.  Here, we describe A-485, a potent, selective and drug-like catalytic inhibitor of p300 and CBP.  We present a high resoln. (1.95 Å) co-crystal structure of a small mol. bound to the catalytic active site of p300 and demonstrate that A-485 competes with acetyl CoA (acetyl-CoA).  A-485 selectively inhibited proliferation in lineage-specific tumor types, including several haematol. malignancies and androgen receptor-pos. prostate cancer.  A-485 inhibited the androgen receptor transcriptional program in both androgen-sensitive and castration-resistant prostate cancer and inhibited tumor growth in a castration-resistant xenograft model.  These results demonstrate the feasibility of using small mol. inhibitors to selectively target the catalytic activity of histone acetyltransferases, which may provide effective treatments for transcriptional activator-driven malignancies and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqfy_rMa47C7Vg90H21EOLACvtfcHk0lgL02MLfKvWtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsLzM&md5=050103c08a6abe2e0035573085fff6be</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature24028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24028%26sid%3Dliteratum%253Aachs%26aulast%3DLasko%26aufirst%3DL.%2BM.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DMontgomery%26aufirst%3DD.%26aulast%3DDigiammarino%26aufirst%3DE.%2BL.%26aulast%3DHansen%26aufirst%3DT.%2BM.%26aulast%3DRisi%26aufirst%3DR.%2BM.%26aulast%3DFrey%26aufirst%3DR.%26aulast%3DManaves%26aufirst%3DV.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DAlgire%26aufirst%3DM.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DMartin%26aufirst%3DR.%2BL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DKarukurichi%26aufirst%3DK.%26aulast%3DLangston%26aufirst%3DJ.%2BW.%26aulast%3DWeinert%26aufirst%3DB.%2BT.%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3Dde%2BVries%26aufirst%3DP.%26aulast%3DKluge%26aufirst%3DA.%2BF.%26aulast%3DPatane%26aufirst%3DM.%2BA.%26aulast%3DVan%2BDrie%26aufirst%3DJ.%2BH.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DMcElligott%26aufirst%3DD.%26aulast%3DKesicki%26aufirst%3DE.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBromberg%26aufirst%3DK.%2BD.%26atitle%3DDiscovery%2520of%2520a%2520selective%2520catalytic%2520p300%252FCBP%2520inhibitor%2520that%2520targets%2520lineage-specific%2520tumours%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D128%26epage%3D132%26doi%3D10.1038%2Fnature24028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soccio, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ait-Si-Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolffe, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">HATs off: selective synthetic inhibitors of the histone acetyltransferases p300 and PCAF</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(00)80452-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2FS1097-2765%2800%2980452-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10882143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisVWrs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=589-595&author=O.+D.+Lauauthor=T.+K.+Kunduauthor=R.+E.+Soccioauthor=S.+Ait-Si-Aliauthor=E.+M.+Khalilauthor=A.+Vassilevauthor=A.+P.+Wolffeauthor=Y.+Nakataniauthor=R.+G.+Roederauthor=P.+A.+Cole&title=HATs+off%3A+selective+synthetic+inhibitors+of+the+histone+acetyltransferases+p300+and+PCAF&doi=10.1016%2FS1097-2765%2800%2980452-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">HATs off: selective synthetic inhibitors of the histone acetyltransferases p300 and PCAF</span></div><div class="casAuthors">Lau, Ontario D.; Kundu, Tapas K.; Soccio, Raymond E.; Ait-Si-Ali, Slimane; Khalil, Ehab M.; Vassilev, Alex; Wolffe, Alan P.; Nakatani, Yoshihiro; Roeder, Robert G.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-595</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone acetyltransferases (HATs) play important roles in the regulation of gene expression.  In this report, we describe the design, synthesis, and application of peptide CoA conjugates as selective HAT inhibitors for the transcriptional coactivators p300 and PCAF.  Two inhibitors (Lys-CoA for p300 and H3-CoA-20 for PCAF) were found to be potent (IC50 ≈ 0.5 μM) and selective (∼200-fold) in blocking p300 and PCAF HAT activities.  These inhibitors were used to probe enzymic and transcriptional features of HAT function in several assay systems.  These compds. should be broadly useful as biol. tools for evaluating the roles of HATs in transcriptional studies and may serve as lead agents for the development of novel antineoplastic therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPttQ2l59vKLVg90H21EOLACvtfcHk0lgL02MLfKvWtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisVWrs7c%253D&md5=dd8242124fbb980225bdded4fd7fbd62</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2800%2980452-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252800%252980452-9%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DO.%2BD.%26aulast%3DKundu%26aufirst%3DT.%2BK.%26aulast%3DSoccio%26aufirst%3DR.%2BE.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26aulast%3DKhalil%26aufirst%3DE.%2BM.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DWolffe%26aufirst%3DA.%2BP.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DHATs%2520off%253A%2520selective%2520synthetic%2520inhibitors%2520of%2520the%2520histone%2520acetyltransferases%2520p300%2520and%2520PCAF%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D5%26spage%3D589%26epage%3D595%26doi%3D10.1016%2FS1097-2765%2800%2980452-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwie, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soupaya, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maturano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blonski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lherbet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudoin-Dehoux, C.</span></span> <span> </span><span class="NLM_article-title">New potent bisubstrate inhibitors of histone acetyltransferase p300: design, synthesis and biological evaluation</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2010.01056.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1111%2Fj.1747-0285.2010.01056.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=21118378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252FitVylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=86-92&author=F.+H.+Kwieauthor=M.+Brietauthor=D.+Soupayaauthor=P.+Hoffmannauthor=M.+Maturanoauthor=F.+Rodriguezauthor=C.+Blonskiauthor=C.+Lherbetauthor=C.+Baudoin-Dehoux&title=New+potent+bisubstrate+inhibitors+of+histone+acetyltransferase+p300%3A+design%2C+synthesis+and+biological+evaluation&doi=10.1111%2Fj.1747-0285.2010.01056.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">New potent bisubstrate inhibitors of histone acetyltransferase p300: design, synthesis and biological evaluation</span></div><div class="casAuthors">Kwie Franciane Ho A; Briet Martine; Soupaya David; Hoffmann Pascal; Maturano Marie; Rodriguez Frederic; Blonski Casimir; Lherbet Christian; Baudoin-Dehoux Cecile</div><div class="citationInfo"><span class="NLM_cas:title">Chemical biology & drug design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bisubstrate-type compound Lys-CoA has been shown to inhibit the p300 histone acetyl transferase activity efficiently and may constitute a lead compound for a novel class of anticancer therapeutics.  Based on this strategy, we synthesized a series of CoA derivatives and evaluated these molecules for their activity as p300 histone acetyltransferases inhibitor.  The best activity was obtained with compound 3 bearing a C-5 spacing linker that connects the CoA moiety to a tert-butyloxycarbonyl (Boc) group.  Based on docking simulations, this inhibitor exhibits favorable interactions with two binding areas, namely pockets P1 and P2, within the active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJhwGTLVwV9kBuFdjgVNKSfW6udTcc2eYOMnmMF3tSs7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252FitVylsg%253D%253D&md5=6f4ed0fcaab0fcf054994954e5d7ed3a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2010.01056.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2010.01056.x%26sid%3Dliteratum%253Aachs%26aulast%3DKwie%26aufirst%3DF.%2BH.%26aulast%3DBriet%26aufirst%3DM.%26aulast%3DSoupaya%26aufirst%3DD.%26aulast%3DHoffmann%26aufirst%3DP.%26aulast%3DMaturano%26aufirst%3DM.%26aulast%3DRodriguez%26aufirst%3DF.%26aulast%3DBlonski%26aufirst%3DC.%26aulast%3DLherbet%26aufirst%3DC.%26aulast%3DBaudoin-Dehoux%26aufirst%3DC.%26atitle%3DNew%2520potent%2520bisubstrate%2520inhibitors%2520of%2520histone%2520acetyltransferase%2520p300%253A%2520design%252C%2520synthesis%2520and%2520biological%2520evaluation%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2011%26volume%3D77%26spage%3D86%26epage%3D92%26doi%3D10.1111%2Fj.1747-0285.2010.01056.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazzalin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liszczak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larocca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldanha, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahadevan, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alani, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Virtual ligand screening of the p300/CBP histone acetyltransferase: identification of a selective small molecule inhibitor</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">482</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.chembiol.2010.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=20534345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslCjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=471-482&author=E.+M.+Bowersauthor=G.+Yanauthor=C.+Mukherjeeauthor=A.+Orryauthor=L.+Wangauthor=M.+A.+Holbertauthor=N.+T.+Crumpauthor=C.+A.+Hazzalinauthor=G.+Liszczakauthor=H.+Yuanauthor=C.+Laroccaauthor=S.+A.+Saldanhaauthor=R.+Abagyanauthor=Y.+Sunauthor=D.+J.+Meyersauthor=R.+Marmorsteinauthor=L.+C.+Mahadevanauthor=R.+M.+Alaniauthor=P.+A.+Cole&title=Virtual+ligand+screening+of+the+p300%2FCBP+histone+acetyltransferase%3A+identification+of+a+selective+small+molecule+inhibitor&doi=10.1016%2Fj.chembiol.2010.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Ligand Screening of the p300/CBP Histone Acetyltransferase: Identification of a Selective Small Molecule Inhibitor</span></div><div class="casAuthors">Bowers, Erin M.; Yan, Gai; Mukherjee, Chandrani; Orry, Andrew; Wang, Ling; Holbert, Marc A.; Crump, Nicholas T.; Hazzalin, Catherine A.; Liszczak, Glen; Yuan, Hua; Larocca, Cecilia; Saldanha, S. Adrian; Abagyan, Ruben; Sun, Yan; Meyers, David J.; Marmorstein, Ronen; Mahadevan, Louis C.; Alani, Rhoda M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-482</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The histone acetyltransferase (HAT) p300/CBP is a transcriptional coactivator implicated in many gene regulatory pathways and protein acetylation events.  Although p300 inhibitors have been reported, a potent, selective, and readily available active-site-directed small mol. inhibitor is not yet known.  Here the authors use a structure-based, in silico screening approach to identify a com. available pyrazolone-contg. small mol. p300 HAT inhibitor, C646.  C646 is a competitive p300 inhibitor with a Ki of 400 nM and is selective vs. other acetyltransferases.  Studies on site-directed p300 HAT mutants and synthetic modifications of C646 confirm the importance of predicted interactions in conferring potency.  Inhibition of histone acetylation and cell growth by C646 in cells validate its utility as a pharmacol. probe and suggest that p300/CBP HAT is a worthy anticancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7_p7jSyRB-bVg90H21EOLACvtfcHk0ljBN0HSs0qxZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslCjsbg%253D&md5=551fc005ecd3142c76e2b8e33d51c7e1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DE.%2BM.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DMukherjee%26aufirst%3DC.%26aulast%3DOrry%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHolbert%26aufirst%3DM.%2BA.%26aulast%3DCrump%26aufirst%3DN.%2BT.%26aulast%3DHazzalin%26aufirst%3DC.%2BA.%26aulast%3DLiszczak%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DLarocca%26aufirst%3DC.%26aulast%3DSaldanha%26aufirst%3DS.%2BA.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DMeyers%26aufirst%3DD.%2BJ.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DMahadevan%26aufirst%3DL.%2BC.%26aulast%3DAlani%26aufirst%3DR.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DVirtual%2520ligand%2520screening%2520of%2520the%2520p300%252FCBP%2520histone%2520acetyltransferase%253A%2520identification%2520of%2520a%2520selective%2520small%2520molecule%2520inhibitor%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D471%26epage%3D482%26doi%3D10.1016%2Fj.chembiol.2010.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinello, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahn, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldanha, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planck, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geary, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roush, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, D.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of acetyltransferase p300 identified by high-throughput screening are potent anticancer agents</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-12-0930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1158%2F1535-7163.MCT-12-0930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=23625935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=610-620&author=H.+Yangauthor=C.+E.+Pinelloauthor=J.+Luoauthor=D.+Liauthor=Y.+Wangauthor=L.+Y.+Zhaoauthor=S.+C.+Jahnauthor=S.+A.+Saldanhaauthor=P.+Chaseauthor=J.+Planckauthor=K.+R.+Gearyauthor=H.+Maauthor=B.+K.+Lawauthor=W.+R.+Roushauthor=P.+Hodderauthor=D.+Liao&title=Small-molecule+inhibitors+of+acetyltransferase+p300+identified+by+high-throughput+screening+are+potent+anticancer+agents&doi=10.1158%2F1535-7163.MCT-12-0930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Acetyltransferase p300 Identified by High-Throughput Screening Are Potent Anticancer Agents</span></div><div class="casAuthors">Yang, Heng; Pinello, Christie E.; Luo, Jian; Li, Dawei; Wang, Yunfei; Zhao, Lisa Y.; Jahn, Stephan C.; Saldanha, Sanjay Adrian; Planck, Jamie; Geary, Kyla R.; Ma, Haiching; Law, Brian K.; Roush, William R.; Hodder, Peter; Liao, Daiqing</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">610-620</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Acetyltransferase p300 (KAT3B) plays key roles in signaling cascades that support cancer cell survival and sustained proliferation.  Thus, p300 represents a potential anticancer therapeutic target.  To discover novel anticancer agents that target p300, we conducted a high-throughput screening campaign.  A library of 622,079 compds. was assayed for cytotoxicity to the triple-neg. breast cancer (TNBC) cell line MDA-MB-231 but not to the human mammary epithelial cells.  The resulting compds. were tested in a biochem. assay for inhibiting the enzymic activity of p300.  One compd. (L002, NSC764414) displayed an IC50 of 1.98 μmol/L against p300 in vitro, inhibited acetylation of histones and p53, and suppressed STAT3 activation in cell-based assays.  L002 could be docked to the active site of the p300 catalytic domain.  Biochem. tests of a series of related compds. revealed functional groups that may impact inhibitory potency of L002 against p300.  Interestingly, these analogs showed inhibitory activities against the cellular paralog of p300 (CBP), p300/CBP-assocd. factor, and GCN5, but not to other acetyltransferases (KAT5, KAT6B, and KAT7), histone deacetylases, and histone methyltransferases.  Among the NCI-60 panel of cancer cell lines, leukemia and lymphoma cell lines were extremely sensitive to L002, whereas it is toxic to only a limited no. of cell lines derived from solid tumors.  Notably, breast cancer cell lines, esp. those derived from TNBC, were highly susceptible to L002.  In vivo, it potently suppressed tumor growth and histone acetylation of MDA-MB-468 xenografts.  Thus, these new acetyltransferase inhibitors are potential anticancer therapeutics.  Mol Cancer Ther; 12(5); 610-20. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9-EbQlof5DrVg90H21EOLACvtfcHk0lgw4QWrnRfzBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSisrY%253D&md5=1a874cea55a65eb911134c6f3f7bebcf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-12-0930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-12-0930%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DH.%26aulast%3DPinello%26aufirst%3DC.%2BE.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DL.%2BY.%26aulast%3DJahn%26aufirst%3DS.%2BC.%26aulast%3DSaldanha%26aufirst%3DS.%2BA.%26aulast%3DChase%26aufirst%3DP.%26aulast%3DPlanck%26aufirst%3DJ.%26aulast%3DGeary%26aufirst%3DK.%2BR.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DLaw%26aufirst%3DB.%2BK.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DLiao%26aufirst%3DD.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520acetyltransferase%2520p300%2520identified%2520by%2520high-throughput%2520screening%2520are%2520potent%2520anticancer%2520agents%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2013%26volume%3D12%26spage%3D610%26epage%3D620%26doi%3D10.1158%2F1535-7163.MCT-12-0930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Santo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artico, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miele, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cereseto, A.</span></span> <span> </span><span class="NLM_article-title">Cinnamoyl compounds as simple molecules that inhibit p300 histone acetyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1973</span>– <span class="NLM_lpage">1977</span>, <span class="refDoi"> DOI: 10.1021/jm060943s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060943s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1agu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1973-1977&author=R.+Costiauthor=R.+Di+Santoauthor=M.+Articoauthor=G.+Mieleauthor=P.+Valentiniauthor=E.+Novellinoauthor=A.+Cereseto&title=Cinnamoyl+compounds+as+simple+molecules+that+inhibit+p300+histone+acetyltransferase&doi=10.1021%2Fjm060943s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Cinnamoyl Compounds as Simple Molecules that Inhibit p300 Histone Acetyltransferase</span></div><div class="casAuthors">Costi, Roberta; Di Santo, Roberto; Artico, Marino; Miele, Gaetano; Valentini, Paola; Novellino, Ettore; Cereseto, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1973-1977</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cinnamoyl compds. 1a-c and 2a-d were designed, synthesized, and in vitro tested as p300 inhibitors.  At different degrees, all tested compds. were proven to inactivate p300, particularly, deriv. 2c (I) was the most active inhibitor, also showing high specificity for p300 as compared to other histone acetyltransferases.  Most notably, 2c showed anti-acetylase activity in mammalian cells.  These compds. represent a new class of synthetic inhibitors of p300, characterized by simple chem. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoYbzB2HLUhrVg90H21EOLACvtfcHk0lgw4QWrnRfzBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1agu78%253D&md5=04ed97d4a972182d8b62107760528db0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm060943s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060943s%26sid%3Dliteratum%253Aachs%26aulast%3DCosti%26aufirst%3DR.%26aulast%3DDi%2BSanto%26aufirst%3DR.%26aulast%3DArtico%26aufirst%3DM.%26aulast%3DMiele%26aufirst%3DG.%26aulast%3DValentini%26aufirst%3DP.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DCereseto%26aufirst%3DA.%26atitle%3DCinnamoyl%2520compounds%2520as%2520simple%2520molecules%2520that%2520inhibit%2520p300%2520histone%2520acetyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1973%26epage%3D1977%26doi%3D10.1021%2Fjm060943s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span> <span> </span><span class="NLM_article-title">Chemistry: Chemical con artists foil drug discovery</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1038/513481a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2F513481a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=25254460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1WjsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=481-483&author=J.+Baellauthor=M.+A.+Walters&title=Chemistry%3A+Chemical+con+artists+foil+drug+discovery&doi=10.1038%2F513481a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry: Chemical con artists foil drug discovery</span></div><div class="casAuthors">Baell, Jonathan; Walters, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">7519</span>),
    <span class="NLM_cas:pages">481-483</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Naivety about promiscuous, assay-duping mols. is polluting the literature and wasting resources, warn Jonathan Baell and Michael A. Walters.  Academic drug discoverers must be more vigilant.  Mols. that show the strongest activity in screening might not be the best starting points for drugs.  PAINS hits should almost always be ignored.  Even trained medicinal chemists have to be careful until they become experienced in screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFYQ0B4VVoLVg90H21EOLACvtfcHk0lgw4QWrnRfzBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1WjsL7M&md5=f5788d81d006460c688e4b78e6200503</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2F513481a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F513481a%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DChemistry%253A%2520Chemical%2520con%2520artists%2520foil%2520drug%2520discovery%26jtitle%3DNature%26date%3D2014%26volume%3D513%26spage%3D481%26epage%3D483%26doi%3D10.1038%2F513481a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">3-(Piperidin-4-ylmethoxy)pyridine containing cmpounds are potent inhibitors of lysine specific demethylase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=253-263&author=F.+Wuauthor=C.+Zhouauthor=Y.+Yaoauthor=L.+Weiauthor=Z.+Fengauthor=L.+Dengauthor=Y.+Song&title=3-%28Piperidin-4-ylmethoxy%29pyridine+containing+cmpounds+are+potent+inhibitors+of+lysine+specific+demethylase+1&doi=10.1021%2Facs.jmedchem.5b01361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span></div><div class="casAuthors">Wu, Fangrui; Zhou, Chao; Yao, Yuan; Wei, Liping; Feng, Zizhen; Deng, Lisheng; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-263</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation.  Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4).  LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression.  We report the design, synthesis, and structure-activity relationships of 3-(piperidin-4-ylmethoxy)pyridine contg. compds. as potent LSD1 inhibitors with Ki values as low as 29 nM.  These compds. exhibited high selectivity (>160×) against related monoamine oxidase A and B.  Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode.  The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC50 values as low as 280 nM, while they had negligible effects on normal cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKAqQdCeodaLVg90H21EOLACvtfcHk0lj33ndxOXEFvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N&md5=2c74e06ed3c755826d25917d02cb845f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01361%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DY.%26atitle%3D3-%2528Piperidin-4-ylmethoxy%2529pyridine%2520containing%2520cmpounds%2520are%2520potent%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D253%26epage%3D263%26doi%3D10.1021%2Facs.jmedchem.5b01361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furst, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglesias, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germain, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gronemeyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Lera, A. R.</span></span> <span> </span><span class="NLM_article-title">Insights into the mechanism of the site-selective sequential palladium-catalyzed cross-coupling reactions of dibromothiophenes/dibromothiazoles and arylboronic acids. Synthesis of PPARbeta/delta agonists</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">4514</span>– <span class="NLM_lpage">4525</span>, <span class="refDoi"> DOI: 10.1039/B612235C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1039%2FB612235C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=17268648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1OhsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2006&pages=4514-4525&author=R.+Pereiraauthor=A.+Furstauthor=B.+Iglesiasauthor=P.+Germainauthor=H.+Gronemeyerauthor=A.+R.+de+Lera&title=Insights+into+the+mechanism+of+the+site-selective+sequential+palladium-catalyzed+cross-coupling+reactions+of+dibromothiophenes%2Fdibromothiazoles+and+arylboronic+acids.+Synthesis+of+PPARbeta%2Fdelta+agonists&doi=10.1039%2FB612235C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the mechanism of the site-selective sequential palladium-catalyzed cross-coupling reactions of dibromothiophenes/dibromothiazoles and arylboronic acids. Synthesis of PPARβ/δ agonists</span></div><div class="casAuthors">Pereira, Raquel; Furst, Audrey; Iglesias, Beatriz; Germain, Pierre; Gronemeyer, Hinrich; de Lera, Angel R.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4514-4525</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A reactivity study, aided by NMR spectroscopy, allowed a mechanistic rationale to be postulated for the palladium-catalyzed regioselective coupling of arylboronic acid (and arylstannane where feasible) at the position next to the sulfur atom in functionalized dibromothiophenes and dibromothiazoles.  The anal. of the NMR spectra (using 19F from the boronic acid CF3 group and 31P from the phosphine of the catalyst as probes) of the entire reaction starting from the dibromoheterocycles allowed the qual. proposal that the transmetalation is the rate-limiting step for both sequential substitution processes.  The extremely facile oxidative addn. at the C-Br bond next to the sulfur atom of the heterocycle instead dets. the positional selectivity.  An addnl. Stille reaction then replaced the second halogen, providing the trisubstituted heterocyclic scaffolds of PPAR ligands, which displayed PPARβ/δ agonist activity, as revealed by reporter assays in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa-yAhaXeJV7Vg90H21EOLACvtfcHk0lj33ndxOXEFvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1OhsbbM&md5=b09ee42d4b0ef033817c255e6a27f7f2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FB612235C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB612235C%26sid%3Dliteratum%253Aachs%26aulast%3DPereira%26aufirst%3DR.%26aulast%3DFurst%26aufirst%3DA.%26aulast%3DIglesias%26aufirst%3DB.%26aulast%3DGermain%26aufirst%3DP.%26aulast%3DGronemeyer%26aufirst%3DH.%26aulast%3Dde%2BLera%26aufirst%3DA.%2BR.%26atitle%3DInsights%2520into%2520the%2520mechanism%2520of%2520the%2520site-selective%2520sequential%2520palladium-catalyzed%2520cross-coupling%2520reactions%2520of%2520dibromothiophenes%252Fdibromothiazoles%2520and%2520arylboronic%2520acids.%2520Synthesis%2520of%2520PPARbeta%252Fdelta%2520agonists%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2006%26volume%3D4%26spage%3D4514%26epage%3D4525%26doi%3D10.1039%2FB612235C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">The structural basis of protein acetylation by the p300/CBP transcriptional coactivator</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>451</i></span>,  <span class="NLM_fpage">846</span>– <span class="NLM_lpage">850</span>, <span class="refDoi"> DOI: 10.1038/nature06546</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fnature06546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=18273021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvVyms74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2008&pages=846-850&author=X.+Liuauthor=L.+Wangauthor=K.+Zhaoauthor=P.+R.+Thompsonauthor=Y.+Hwangauthor=R.+Marmorsteinauthor=P.+A.+Cole&title=The+structural+basis+of+protein+acetylation+by+the+p300%2FCBP+transcriptional+coactivator&doi=10.1038%2Fnature06546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis of protein acetylation by the p300/CBP transcriptional coactivator</span></div><div class="casAuthors">Liu, Xin; Wang, Ling; Zhao, Kehao; Thompson, Paul R.; Hwang, Yousang; Marmorstein, Ronen; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">7180</span>),
    <span class="NLM_cas:pages">846-850</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The transcriptional coactivator p300/CBP (CREBBP) is a histone acetyltransferase (HAT) that regulates gene expression by acetylating histones and other transcription factors.  Dysregulation of p300/CBP HAT activity contributes to various diseases including cancer.  Sequence alignments, enzymol. expts. and inhibitor studies on p300/CBP have led to contradictory results about its catalytic mechanism and its structural relation to the Gcn5/PCAF and MYST HATs.  Here we describe a high-resoln. x-ray crystal structure of a semi-synthetic heterodimeric p300 HAT domain in complex with a bi-substrate inhibitor, Lys-CoA.  This structure shows that p300/CBP is a distant cousin of other structurally characterized HATs, but reveals several novel features that explain the broad substrate specificity and preference for nearby basic residues.  Based on this structure and accompanying biochem. data, we propose that p300/CBP uses an unusual 'hit-and-run' (Theorell-Chance) catalytic mechanism that is distinct from other characterized HATs.  Several disease-assocd. mutations can also be readily accounted for by the p300 HAT structure.  These studies pave the way for new epigenetic therapies involving modulation of p300/CBP HAT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeYOui0tFND7Vg90H21EOLACvtfcHk0lj33ndxOXEFvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvVyms74%253D&md5=c393f167458f8532f2d3b06e96f7eb00</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature06546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06546%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DHwang%26aufirst%3DY.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DThe%2520structural%2520basis%2520of%2520protein%2520acetylation%2520by%2520the%2520p300%252FCBP%2520transcriptional%2520coactivator%26jtitle%3DNature%26date%3D2008%26volume%3D451%26spage%3D846%26epage%3D850%26doi%3D10.1038%2Fnature06546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maksimoska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura-Pena, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structure of the p300 histone acetyltransferase bound to acetyl-coenzyme A and its analogues</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3415</span>– <span class="NLM_lpage">3422</span>, <span class="refDoi"> DOI: 10.1021/bi500380f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi500380f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvVSgsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=3415-3422&author=J.+Maksimoskaauthor=D.+Segura-Penaauthor=P.+A.+Coleauthor=R.+Marmorstein&title=Structure+of+the+p300+histone+acetyltransferase+bound+to+acetyl-coenzyme+A+and+its+analogues&doi=10.1021%2Fbi500380f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the p300 histone acetyltransferase bound to acetyl-coenzyme A and its analogues</span></div><div class="casAuthors">Maksimoska, Jasna; Segura-Pena, Dario; Cole, Philip A.; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3415-3422</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The p300 and CBP transcriptional coactivator paralogs (p300/CBP) regulate a variety of different cellular pathways, in part, by acetylating histones and >70 non-histone protein substrates.  Mutation, chromosomal translocation, or other aberrant activities of p300/CBP are linked to many different diseases, including cancer.  Because of its pleiotropic biol. roles and connection to disease, it is important to understand the mechanism of acetyl transfer by p300/CBP, in part so that inhibitors can be more rationally developed.  Toward this goal, a structure of p300 bound to a Lys-CoA bisubstrate histone acetyltransferase inhibitor was previously elucidated, and the enzyme's catalytic mechanism has been investigated.  Nonetheless, many questions underlying p300/CBP structure and mechanism remain.  Here, the authors report a structural characterization of different reaction states in the p300 activity cycle.  The authors present the structures of p300 in complex with an acetyl-CoA substrate, a CoA product, and an acetonyl-CoA inhibitor.  A comparison of these structures with the previously reported p300/Lys-CoA complex demonstrated that the conformation of the enzyme active site depends on the interaction of the enzyme with the cofactor, and is not apparently influenced by protein substrate Lys residue binding.  The p300/CoA crystals also contained 2 poly(ethylene glycol) moieties bound proximal to the cofactor-binding site, implicating the path of protein substrate assocn.  The structure of the p300/acetonyl-CoA complex explained the inhibitory and tight binding properties of acetonyl-CoA toward p300.  Together, these studies provide new insights into the mol. basis of acetylation by p300 and have implications for the rational development of new small-mol. p300 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAvoxOYiIJWbVg90H21EOLACvtfcHk0lj33ndxOXEFvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvVSgsrk%253D&md5=d4b60cc12bcc790a92c91d6b9b2d3c23</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fbi500380f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi500380f%26sid%3Dliteratum%253Aachs%26aulast%3DMaksimoska%26aufirst%3DJ.%26aulast%3DSegura-Pena%26aufirst%3DD.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructure%2520of%2520the%2520p300%2520histone%2520acetyltransferase%2520bound%2520to%2520acetyl-coenzyme%2520A%2520and%2520its%2520analogues%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D3415%26epage%3D3422%26doi%3D10.1021%2Fbi500380f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demarest, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez-Yamout, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyson, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, P. E.</span></span> <span> </span><span class="NLM_article-title">Mutual synergistic folding in recruitment of CBP/p300 by p160 nuclear receptor coactivators</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>415</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1038/415549a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2F415549a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=11823864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD38Xht1Ghtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=415&publication_year=2002&pages=549-553&author=S.+J.+Demarestauthor=M.+Martinez-Yamoutauthor=J.+Chungauthor=H.+Chenauthor=W.+Xuauthor=H.+J.+Dysonauthor=R.+M.+Evansauthor=P.+E.+Wright&title=Mutual+synergistic+folding+in+recruitment+of+CBP%2Fp300+by+p160+nuclear+receptor+coactivators&doi=10.1038%2F415549a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Mutual synergistic folding in recruitment of CBP/p200 by p160 nuclear receptor coactivators</span></div><div class="casAuthors">Demarest, Stephen J.; Martinez-Yamout, Maria; Chung, John; Chen, Hongwu; Xu, Wel; Dyson, H. Jane; Evans, Ronald M.; Wright, Peter E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">415</span>
        (<span class="NLM_cas:issue">6871</span>),
    <span class="NLM_cas:pages">549-553</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nuclear hormone receptors are ligand-activated transcription factors that regulate the expression of genes that are essential for development, reprodn. and homeostasis.  The hormone response is mediated through recruitment of p160 receptor coactivators and the general transcriptional coactivator CBP/p300, which function synergistically to activate transcription.  These coactivators exhibit intrinsic histone acetyltransferase activity, function in the remodelling of chromatin, and facilitate the recruitment of RNA polymerase II and the basal transcription machinery.  The activities of the P160 coactivators are dependent on CBP.  Both coactivators are essential for proper cell-cycle control, differentiation and apoptosis, and are implicated in cancer and other diseases.  To elucidate the mol. basis of assembling the multiprotein activation complex, we undertook a structural and thermodn. anal. of the interaction domains of CBP and the activator for thyroid hormone and retinoid receptors.  Here we show that although the isolated domains are intrinsically disordered, they combine with high affinity to form a cooperatively folded helical heterodimer.  Our study uncovers a unique mechanism, called 'synergistic folding', through which p160 coactivators recruit CBP/p300 to allow transmission of the hormonal signal to the transcriptional machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd2arByyIp47Vg90H21EOLACvtfcHk0liOe6FOS43KeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xht1Ghtb0%253D&md5=dce1ccdf207ec7e7729271b902d718e8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2F415549a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F415549a%26sid%3Dliteratum%253Aachs%26aulast%3DDemarest%26aufirst%3DS.%2BJ.%26aulast%3DMartinez-Yamout%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DDyson%26aufirst%3DH.%2BJ.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26aulast%3DWright%26aufirst%3DP.%2BE.%26atitle%3DMutual%2520synergistic%2520folding%2520in%2520recruitment%2520of%2520CBP%252Fp300%2520by%2520p160%2520nuclear%2520receptor%2520coactivators%26jtitle%3DNature%26date%3D2002%26volume%3D415%26spage%3D549%26epage%3D553%26doi%3D10.1038%2F415549a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeselsohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollingsworth, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heiser, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grasso, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellman, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, O. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsimelzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trivedi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimawi, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Terrada, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilsenbeck, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">FOXA1 overexpression mediates endocrine resistance by altering the ER transcriptome and IL-8 expression in ER-positive breast cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">E6600</span>– <span class="NLM_lpage">E6609</span>, <span class="refDoi"> DOI: 10.1073/pnas.1612835113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1073%2Fpnas.1612835113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=27791031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1aksb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=E6600-E6609&author=X.+Fuauthor=R.+Jeselsohnauthor=R.+Pereiraauthor=E.+F.+Hollingsworthauthor=C.+J.+Creightonauthor=F.+Liauthor=M.+Sheaauthor=A.+Nardoneauthor=C.+De+Angelisauthor=L.+M.+Heiserauthor=P.+Anurauthor=N.+Wangauthor=C.+S.+Grassoauthor=P.+T.+Spellmanauthor=O.+L.+Griffithauthor=A.+Tsimelzonauthor=C.+Gutierrezauthor=S.+Huangauthor=D.+P.+Edwardsauthor=M.+V.+Trivediauthor=M.+F.+Rimawiauthor=D.+Lopez-Terradaauthor=S.+G.+Hilsenbeckauthor=J.+W.+Grayauthor=M.+Brownauthor=C.+K.+Osborneauthor=R.+Schiff&title=FOXA1+overexpression+mediates+endocrine+resistance+by+altering+the+ER+transcriptome+and+IL-8+expression+in+ER-positive+breast+cancer&doi=10.1073%2Fpnas.1612835113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">FOXA1 overexpression mediates endocrine resistance by altering the ER transcriptome and IL-8 expression in ER-positive breast cancer</span></div><div class="casAuthors">Fu, Xiaoyong; Jeselsohn, Rinath; Pereira, Resel; Hollingsworth, Emporia F.; Creighton, Chad J.; Li, Fugen; Shea, Martin; Nardone, Agostina; De Angelis, Carmine; Heiser, Laura M.; Anur, Pavana; Wang, Nicholas; Grasso, Catherine S.; Spellman, Paul T.; Griffith, Obi L.; Tsimelzon, Anna; Gutierrez, Carolina; Huang, Shixia; Edwards, Dean P.; Trivedi, Meghana V.; Rimawi, Mothaffar F.; Lopez-Terrada, Dolores; Hilsenbeck, Susan G.; Gray, Joe W.; Brown, Myles; Osborne, C. Kent; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">E6600-E6609</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Forkhead box protein A1 (FOXA1) is a pioneer factor of estrogen receptor α (ER)-chromatin binding and function, yet its aberration in endocrine-resistant (Endo-R) breast cancer is unknown.  Here, we report preclin. evidence for a role of FOXA1 in Endo-R breast cancer as well as evidence for its clin. significance.  FOXA1 is gene-amplified and/or overexpressed in Endo-R derivs. of several breast cancer cell line models.  Induced FOXA1 triggers oncogenic gene signatures and proteomic profiles highly assocd. with endocrine resistance.  Integrated omics data reveal IL8 as one of the most perturbed genes regulated by FOXA1 and ER transcriptional reprogramming in Endo-R cells.  IL-8 knockdown inhibits tamoxifen-resistant cell growth and invasion and partially attenuates the effect of overexpressed FOXA1.  Our study highlights a role of FOXA1 via IL-8 signaling as a potential therapeutic target in FOXA1-overexpressing ER-pos. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrB2OAHVe9NM7Vg90H21EOLACvtfcHk0liOe6FOS43KeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1aksb3I&md5=0f64adff4b747bd30d54aa42df97e1d4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1612835113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1612835113%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DX.%26aulast%3DJeselsohn%26aufirst%3DR.%26aulast%3DPereira%26aufirst%3DR.%26aulast%3DHollingsworth%26aufirst%3DE.%2BF.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DShea%26aufirst%3DM.%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DHeiser%26aufirst%3DL.%2BM.%26aulast%3DAnur%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DGrasso%26aufirst%3DC.%2BS.%26aulast%3DSpellman%26aufirst%3DP.%2BT.%26aulast%3DGriffith%26aufirst%3DO.%2BL.%26aulast%3DTsimelzon%26aufirst%3DA.%26aulast%3DGutierrez%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DEdwards%26aufirst%3DD.%2BP.%26aulast%3DTrivedi%26aufirst%3DM.%2BV.%26aulast%3DRimawi%26aufirst%3DM.%2BF.%26aulast%3DLopez-Terrada%26aufirst%3DD.%26aulast%3DHilsenbeck%26aufirst%3DS.%2BG.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DFOXA1%2520overexpression%2520mediates%2520endocrine%2520resistance%2520by%2520altering%2520the%2520ER%2520transcriptome%2520and%2520IL-8%2520expression%2520in%2520ER-positive%2520breast%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3DE6600%26epage%3DE6609%26doi%3D10.1073%2Fpnas.1612835113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimawi, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of the dual EGFR/HER2 inhibitor AZD8931 in circumventing endocrine resistance</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1007/s10549-014-2878-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1007%2Fs10549-014-2878-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=24554387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtFSltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2014&pages=263-272&author=G.+Morrisonauthor=X.+Fuauthor=M.+Sheaauthor=S.+Nandaauthor=M.+Giulianoauthor=T.+Wangauthor=T.+Klinowskaauthor=C.+K.+Osborneauthor=M.+F.+Rimawiauthor=R.+Schiff&title=Therapeutic+potential+of+the+dual+EGFR%2FHER2+inhibitor+AZD8931+in+circumventing+endocrine+resistance&doi=10.1007%2Fs10549-014-2878-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic potential of the dual EGFR/HER2 inhibitor AZD8931 in circumventing endocrine resistance</span></div><div class="casAuthors">Morrison, Gladys; Fu, Xiaoyong; Shea, Martin; Nanda, Sarmistha; Giuliano, Mario; Wang, Tao; Klinowska, Teresa; Osborne, C. Kent; Rimawi, Mothaffar F.; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-272</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Modest up-regulation of either HER-ligands or receptors has been implicated in acquired endocrine resistance.  AZD8931, a dual tyrosine kinase inhibitor (TKI) of epithelial growth factor receptor (EGFR)/HER2, has been shown to more effectively block ligand-dependent HER signaling than the HER TKIs lapatinib or gefitinib.  We therefore examd. the effect of AZD8931 in ER-pos./HER2-neg. breast cancer cells with acquired resistance to tamoxifen, where there is ligand up-regulation assocd. with HER pathway activation.  RNA-seq ligand profiling and levels of HER receptors and signaling by western blotting were conducted in ER+ MCF7 and T47D parental cells and their Tam-resistant derivs. (TamRes).  In vitro cell growth and apoptosis and HER ligand-stimulated signaling were measured in response to endocrine and HER TKIs.  For studies in vivo, transplantable MCF7/TamRes xenografts were treated with tamoxifen or fulvestrant, either alone or in combination with AZD8931.  AZD8931 only minimally enhanced endocrine sensitivity in MCF7 parental cells, but showed a greater effect in the T47D parental model.  AZD8931 combined with either tamoxifen or fulvestrant inhibited cell growth more than lapatinib in T47D TamRes cells, and was also significantly, though modestly, more potent in MCF7 TamRes cells.  In both TamRes models, AZD8931 significantly inhibited cell proliferation and induced apoptosis.  Under ligand-stimulated conditions, AZD8931 more potently inhibited HER signaling than lapatinib or gefitinib.  AZD8931 also significantly delayed the growth of MCF7 TamRes xenografts in the presence of tamoxifen or fulvestrant.  The strongest inhibition was achieved with a fulvestrant and AZD8931 combination, though no tumor regression was obsd.  This study provides evidence that AZD8931 has greater inhibitory efficacy in tamoxifen-resistant settings than in an endocrine therapy naive setting.  The absence of tumor regression, however, suggests that addnl. escape pathways contribute to resistant growth and will need to be targeted to fully circumvent tamoxifen resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwFAxc7BoAB7Vg90H21EOLACvtfcHk0liOe6FOS43KeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtFSltLc%253D&md5=92d9536d6c1bd9363239eb2ad873a630</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs10549-014-2878-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-014-2878-x%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DG.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DShea%26aufirst%3DM.%26aulast%3DNanda%26aufirst%3DS.%26aulast%3DGiuliano%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DRimawi%26aufirst%3DM.%2BF.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DTherapeutic%2520potential%2520of%2520the%2520dual%2520EGFR%252FHER2%2520inhibitor%2520AZD8931%2520in%2520circumventing%2520endocrine%2520resistance%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2014%26volume%3D144%26spage%3D263%26epage%3D272%26doi%3D10.1007%2Fs10549-014-2878-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, H.</span></span> <span> </span><span class="NLM_article-title">P300 inhibition enhances gemcitabine-induced apoptosis of pancreatic cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">51301</span>– <span class="NLM_lpage">51310</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.10117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.18632%2Foncotarget.10117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=27322077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A280%3ADC%252BC2s7hs1Ciug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=51301-51310&author=H.+Onoauthor=M.+D.+Bassonauthor=H.+Ito&title=P300+inhibition+enhances+gemcitabine-induced+apoptosis+of+pancreatic+cancer&doi=10.18632%2Foncotarget.10117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">P300 inhibition enhances gemcitabine-induced apoptosis of pancreatic cancer</span></div><div class="casAuthors">Ono Hiroaki; Ito Hiromichi; Basson Marc D</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">51301-51310</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The transcriptional cofactor p300 has histone acetyltransferase activity (HAT) and has been reported to participate in chromatin remodeling and DNA repair.  We hypothesized that targeting p300 can enhance the cytotoxicity of gemcitabine, which induces pancreatic cancer cell apoptosis by damaging DNA.  Expression of p300 was confirmed in pancreatic cancer cell lines and human pancreatic adenocarcinoma tissues by western blotting and immunohistochemistry.  When pancreatic cancer cells were treated with gemcitabine, p300 was recruited to chromatin within 24 hours, indicating the role in response to DNA damage.  When p300 was gene-silenced with siRNA, histone acetylation was substantially reduced and pancreatic cancer cells were sensitized to gemcitabine.  The selective p300 HAT inhibitor C646 similarly decreased histone acetylation, increased gemcitabine-induced apoptosis and thus enhanced the cytotoxicity of gemcitabine on pancreatic cancer cells.  These findings indicate that p300 contributes to chemo-resistance of pancreatic cancer against gemcitabine and suggest that p300 and its HAT activity may be a potential therapeutic target to improve outcomes in patients with pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQHSrWrD2HP4Xl8icjszYl0fW6udTcc2eax3cKP-mi1zrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7hs1Ciug%253D%253D&md5=467ff424e8cfcf8d35c89230de540d14</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.10117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.10117%26sid%3Dliteratum%253Aachs%26aulast%3DOno%26aufirst%3DH.%26aulast%3DBasson%26aufirst%3DM.%2BD.%26aulast%3DIto%26aufirst%3DH.%26atitle%3DP300%2520inhibition%2520enhances%2520gemcitabine-induced%2520apoptosis%2520of%2520pancreatic%2520cancer%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D51301%26epage%3D51310%26doi%3D10.18632%2Foncotarget.10117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gural, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perna, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatlen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deblasio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menendez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voza, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimer, S. D.</span></span> <span> </span><span class="NLM_article-title">The leukemogenicity of AML1-ETO is dependent on site-specific lysine acetylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>333</i></span>,  <span class="NLM_fpage">765</span>– <span class="NLM_lpage">769</span>, <span class="refDoi"> DOI: 10.1126/science.1201662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1126%2Fscience.1201662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=21764752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1SgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=333&publication_year=2011&pages=765-769&author=L.+Wangauthor=A.+Guralauthor=X.+J.+Sunauthor=X.+Zhaoauthor=F.+Pernaauthor=G.+Huangauthor=M.+A.+Hatlenauthor=L.+Vuauthor=F.+Liuauthor=H.+Xuauthor=T.+Asaiauthor=H.+Xuauthor=T.+Deblasioauthor=S.+Menendezauthor=F.+Vozaauthor=Y.+Jiangauthor=P.+A.+Coleauthor=J.+Zhangauthor=A.+Melnickauthor=R.+G.+Roederauthor=S.+D.+Nimer&title=The+leukemogenicity+of+AML1-ETO+is+dependent+on+site-specific+lysine+acetylation&doi=10.1126%2Fscience.1201662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The leukemogenicity of AML1-ETO is dependent on site-specific lysine acetylation</span></div><div class="casAuthors">Wang, Lan; Gural, Alexander; Sun, Xiao-Jian; Zhao, Xinyang; Perna, Fabiana; Huang, Gang; Hatlen, Megan A.; Vu, Ly; Liu, Fan; Xu, Haiming; Asai, Takashi; Xu, Hao; Deblasio, Tony; Menendez, Silvia; Voza, Francesca; Jiang, Yanwen; Cole, Philip A.; Zhang, Jinsong; Melnick, Ari; Roeder, Robert G.; Nimer, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">333</span>
        (<span class="NLM_cas:issue">6043</span>),
    <span class="NLM_cas:pages">765-769</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The chromosomal translocations found in acute myelogenous leukemia (AML) generate oncogenic fusion transcription factors with aberrant transcriptional regulatory properties.  Although therapeutic targeting of most leukemia fusion proteins remains elusive, the post-translational modifications that control their function could be targetable.  We found that AML1-ETO, the fusion protein generated by the t(8;21) translocation, is acetylated by the transcriptional coactivator p300 in leukemia cells isolated from t(8;21) AML patients, and that this acetylation is essential for its self-renewal-promoting effects in human cord blood CD34+ cells and its leukemogenicity in mouse models.  Inhibition of p300 abrogates the acetylation of AML1-ETO and impairs its ability to promote leukemic transformation.  Thus, lysine acetyltransferases represent a potential therapeutic target in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsv3PTdm2bWbVg90H21EOLACvtfcHk0lijmPEiVLtCgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1SgtLc%253D&md5=b77a02c2653eef8faff013aef7356c5a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.1201662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1201662%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DGural%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DX.%2BJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DPerna%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DHatlen%26aufirst%3DM.%2BA.%26aulast%3DVu%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DAsai%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DDeblasio%26aufirst%3DT.%26aulast%3DMenendez%26aufirst%3DS.%26aulast%3DVoza%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26atitle%3DThe%2520leukemogenicity%2520of%2520AML1-ETO%2520is%2520dependent%2520on%2520site-specific%2520lysine%2520acetylation%26jtitle%3DScience%26date%3D2011%26volume%3D333%26spage%3D765%26epage%3D769%26doi%3D10.1126%2Fscience.1201662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mootha, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulovich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomeroy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span> <span> </span><span class="NLM_article-title">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">15545</span>– <span class="NLM_lpage">15550</span>, <span class="refDoi"> DOI: 10.1073/pnas.0506580102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1073%2Fpnas.0506580102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=16199517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ShtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&author=A.+Subramanianauthor=P.+Tamayoauthor=V.+K.+Moothaauthor=S.+Mukherjeeauthor=B.+L.+Ebertauthor=M.+A.+Gilletteauthor=A.+Paulovichauthor=S.+L.+Pomeroyauthor=T.+R.+Golubauthor=E.+S.+Landerauthor=J.+P.+Mesirov&title=Gene+set+enrichment+analysis%3A+A+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles&doi=10.1073%2Fpnas.0506580102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span></div><div class="casAuthors">Subramanian, Aravind; Tamayo, Pablo; Mootha, Vamsi K.; Mukherjee, Sayan; Ebert, Benjamin L.; Gillette, Michael A.; Paulovich, Amanda; Pomeroy, Scott L.; Golub, Todd R.; Lander, Eric S.; Mesirov, Jill P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15545-15550</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Although genomewide RNA expression anal. has become a routine tool in biomedical research, extg. biol. insight from such information remains a major challenge.  Here, we describe a powerful anal. method called Gene Set Enrichment Anal. (GSEA) for interpreting gene expression data.  The method derives its power by focusing on gene sets, i.e., groups of genes that share common biol. function, chromosomal location, or regulation.  We demonstrate how GSEA yields insights into several cancer-related data sets, including leukemia and lung cancer.  Notably, where single-gene anal. finds little similarity between two independent studies of patients survival in lung cancer, GSEA reveals many biol. pathways in common.  The GSEA method is embodied in a freely available software package, together with an initial database of 1,325 biol. defined gene sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH2_E6gWd0yrVg90H21EOLACvtfcHk0lgp6ROx5s5aXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ShtrnO&md5=ca2eb221010f20379199e6442c65fc2e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0506580102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0506580102%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DTamayo%26aufirst%3DP.%26aulast%3DMootha%26aufirst%3DV.%2BK.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DGillette%26aufirst%3DM.%2BA.%26aulast%3DPaulovich%26aufirst%3DA.%26aulast%3DPomeroy%26aufirst%3DS.%2BL.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26atitle%3DGene%2520set%2520enrichment%2520analysis%253A%2520A%2520knowledge-based%2520approach%2520for%2520interpreting%2520genome-wide%2520expression%2520profiles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D15545%26epage%3D15550%26doi%3D10.1073%2Fpnas.0506580102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ianculescu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegmund, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stallcup, M. R.</span></span> <span> </span><span class="NLM_article-title">Selective roles for cAMP response element-binding protein binding protein and p300 protein as coregulators for androgen-regulated gene expression in advanced prostate cancer cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">4000</span>– <span class="NLM_lpage">4013</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.300194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1074%2Fjbc.M111.300194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=22174411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=4000-4013&author=I.+Ianculescuauthor=D.+Y.+Wuauthor=K.+D.+Siegmundauthor=M.+R.+Stallcup&title=Selective+roles+for+cAMP+response+element-binding+protein+binding+protein+and+p300+protein+as+coregulators+for+androgen-regulated+gene+expression+in+advanced+prostate+cancer+cells&doi=10.1074%2Fjbc.M111.300194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Roles for cAMP Response Element-binding Protein Binding Protein and p300 Protein as Coregulators for Androgen-regulated Gene Expression in Advanced Prostate Cancer Cells</span></div><div class="casAuthors">Ianculescu, Irina; Wu, Dai-Ying; Siegmund, Kimberly D.; Stallcup, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4000-4013</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein acetyltransferases p300 and cAMP response element-binding protein binding protein (CBP) are homologous, ubiquitously expressed proteins that interact with hundreds of proteins involved in transcriptional regulation and are involved globally as transcriptional coregulators.  Although these two proteins acetylate and interact with overlapping sets of proteins, we found that p300 and CBP contribute to androgen-induced regulation of distinct sets of genes in C4-2B prostate cancer cells, a model of advanced prostate cancer.  CBP cannot compensate for the loss of p300 to support androgen-induced expression of many genes, such as TMPRSS2 and PSA.  Global gene expression anal. indicated that 47% of androgen-regulated genes are p300-dependent in these cells, whereas, surprisingly, only 0.3% of them are CBP-dependent.  Chromatin immunopptn. anal. after depletion of cellular p300 indicated that p300 is required for androgen-induced acetylation of histones H3 and H4, methylation of histone H3 at Lys-4, and recruitment of TATA box binding protein (TBP) and RNA polymerase II, but not recruitment of the androgen receptor, on the TMPRSS2 gene in response to androgen.  Thus, p300 is the dominant coregulator of the CBP/p300 pair for androgen-regulated gene expression in C4-2B cells. p300 is required at an early stage of chromatin remodeling and transcription complex assembly after binding of androgen receptor to the gene but before many crit. histone modifications occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7uJfPL3B4h7Vg90H21EOLACvtfcHk0lgp6ROx5s5aXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSjtLg%253D&md5=095ab7582da1b7602e2ab898caa5b094</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.300194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.300194%26sid%3Dliteratum%253Aachs%26aulast%3DIanculescu%26aufirst%3DI.%26aulast%3DWu%26aufirst%3DD.%2BY.%26aulast%3DSiegmund%26aufirst%3DK.%2BD.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26atitle%3DSelective%2520roles%2520for%2520cAMP%2520response%2520element-binding%2520protein%2520binding%2520protein%2520and%2520p300%2520protein%2520as%2520coregulators%2520for%2520androgen-regulated%2520gene%2520expression%2520in%2520advanced%2520prostate%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D4000%26epage%3D4013%26doi%3D10.1074%2Fjbc.M111.300194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasper, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindle, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dent, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span> <span> </span><span class="NLM_article-title">Distinct roles of GCN5/PCAF-mediated H3K9ac and CBP/p300-mediated H3K18/27ac in nuclear receptor transactivation</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/emboj.2010.318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Femboj.2010.318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=21131905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFSrsr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=249-262&author=Q.+Jinauthor=L.+R.+Yuauthor=L.+Wangauthor=Z.+Zhangauthor=L.+H.+Kasperauthor=J.+E.+Leeauthor=C.+Wangauthor=P.+K.+Brindleauthor=S.+Y.+Dentauthor=K.+Ge&title=Distinct+roles+of+GCN5%2FPCAF-mediated+H3K9ac+and+CBP%2Fp300-mediated+H3K18%2F27ac+in+nuclear+receptor+transactivation&doi=10.1038%2Femboj.2010.318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct roles of GCN5/PCAF-mediated H3K9ac and CBP/p300-mediated H3K18/27ac in nuclear receptor transactivation</span></div><div class="casAuthors">Jin, Qihuang; Yu, Li-Rong; Wang, Lifeng; Zhang, Zhijing; Kasper, Lawryn H.; Lee, Ji-Eun; Wang, Chaochen; Brindle, Paul K.; Dent, Sharon Y. R.; Ge, Kai</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-262</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone acetyltransferases (HATs) GCN5 and PCAF (GCN5/PCAF) and CBP and p300 (CBP/p300) are transcription co-activators.  However, how these two distinct families of HATs regulate gene activation remains unclear.  Here, we show deletion of GCN5/PCAF in cells specifically and dramatically reduces acetylation on histone H3K9 (H3K9ac) while deletion of CBP/p300 specifically and dramatically reduces acetylations on H3K18 and H3K27 (H3K18/27ac).  A ligand (GW501516) for nuclear receptor (NR) PPARδ induces sequential enrichment of H3K18/27ac, RNA polymerase II (Pol II) and H3K9ac on PPARδ target gene Angptl4 promoter, which correlates with a robust Angptl4 expression.  Inhibiting transcription elongation blocks ligand-induced H3K9ac, but not H3K18/27ac, on the Angptl4 promoter.  Finally, we show GCN5/PCAF and GCN5/PCAF-mediated H3K9ac correlate with, but are surprisingly dispensable for, NR target gene activation.  In contrast, CBP/p300 and their HAT activities are essential for ligand-induced Pol II recruitment on, and activation of, NR target genes.  These results highlight the substrate and site specificities of HATs in cells, demonstrate the distinct roles of GCN5/PCAF- and CBP/p300-mediated histone acetylations in gene activation, and suggest an important role of CBP/p300-mediated H3K18/27ac in NR-dependent transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIm-siX9LrK7Vg90H21EOLACvtfcHk0lgp6ROx5s5aXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFSrsr3E&md5=bdb41206845730e0875b8a73e70b0030</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.318%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DYu%26aufirst%3DL.%2BR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKasper%26aufirst%3DL.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DBrindle%26aufirst%3DP.%2BK.%26aulast%3DDent%26aufirst%3DS.%2BY.%26aulast%3DGe%26aufirst%3DK.%26atitle%3DDistinct%2520roles%2520of%2520GCN5%252FPCAF-mediated%2520H3K9ac%2520and%2520CBP%252Fp300-mediated%2520H3K18%252F27ac%2520in%2520nuclear%2520receptor%2520transactivation%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D249%26epage%3D262%26doi%3D10.1038%2Femboj.2010.318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anglin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, B. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Crystallographic investigation and selective inhibition of mutant isocitrate dehydrogenase</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1021/ml400036z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400036z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVWrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=542-546&author=B.+Zhengauthor=Y.+Yaoauthor=Z.+Liuauthor=L.+Dengauthor=J.+L.+Anglinauthor=H.+Jiangauthor=B.+V.+Prasadauthor=Y.+Song&title=Crystallographic+investigation+and+selective+inhibition+of+mutant+isocitrate+dehydrogenase&doi=10.1021%2Fml400036z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic Investigation and Selective Inhibition of Mutant Isocitrate Dehydrogenase</span></div><div class="casAuthors">Zheng, Baisong; Yao, Yuan; Liu, Zhen; Deng, Lisheng; Anglin, Justin L.; Jiang, Hong; Prasad, B. V. Venkataram; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">542-546</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mutations in isocitrate dehydrogenase (IDH), a key enzyme in the tricarboxylic acid cycle, have recently been found in ∼75% glioma and ∼20% acute myeloid leukemia.  Different from the wild-type enzyme, mutant IDH1 catalyzes the redn. of α-ketoglutaric acid to D-2-hydroxyglutaric acid.  Strong evidence has shown mutant IDH1 represents a novel target for this type of cancer.  We found two 1-hydroxypyridin-2-one compds. that are potent inhibitors of R132H and R132C IDH1 mutants with Ki values as low as 120 nM.  These compds. exhibit >60-fold selectivity against wild-type IDH1 and can inhibit the prodn. of D-2-hydroxyglutaric acid in IDH1 mutated cells, representing novel chem. probes for cancer biol. studies.  We also report the first inhibitor-bound crystal structures of IDH1(R132H), showing these inhibitors have H-bond, electrostatic, and hydrophobic interactions with the mutant enzyme.  Comparison with the substrate-bound IDH1 structures revealed the structural basis for the high enzyme selectivity of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH0vzVLGCPZLVg90H21EOLACvtfcHk0lg_OLMDKBbL_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVWrtLg%253D&md5=c34c67949fe816bdf151525a578535f3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fml400036z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400036z%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DCrystallographic%2520investigation%2520and%2520selective%2520inhibition%2520of%2520mutant%2520isocitrate%2520dehydrogenase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D542%26epage%3D546%26doi%3D10.1021%2Fml400036z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogiso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Inhibition of cancer-associated mutant isocitrate dehydrogenases by 2-thiohydantoin compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6899</span>– <span class="NLM_lpage">6908</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlequ7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6899-6908&author=F.+Wuauthor=H.+Jiangauthor=B.+Zhengauthor=M.+Kogisoauthor=Y.+Yaoauthor=C.+Zhouauthor=X.+N.+Liauthor=Y.+Song&title=Inhibition+of+cancer-associated+mutant+isocitrate+dehydrogenases+by+2-thiohydantoin+compounds&doi=10.1021%2Facs.jmedchem.5b00684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Cancer-Associated Mutant Isocitrate Dehydrogenases by 2-Thiohydantoin Compounds</span></div><div class="casAuthors">Wu, Fangrui; Jiang, Hong; Zheng, Baisong; Kogiso, Mari; Yao, Yuan; Zhou, Chao; Li, Xiao-Nan; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6899-6908</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Somatic mutations of isocitrate dehydrogenase 1 (IDH1) at R132 are frequently found in certain cancers such as glioma.  With losing the activity of wild-type IDH1, the R132H and R132C mutant proteins can reduce α-ketoglutaric acid (α-KG) to D-2-hydroxyglutaric acid (D2HG).  The resulting high concn. of D2HG inhibits many α-KG-dependent dioxygenases, including histone demethylases, to cause broad histone hypermethylation.  These aberrant epigenetic changes are responsible for the initiation of these cancers.  The authors report the synthesis, structure-activity relationships, enzyme kinetics, and binding thermodn. of a novel series of 2-thiohydantoin and related compds. I [R1 = H, Me, Ph, etc.; R2 = 3,4-(HO)2C6H3, pyridin-3-yl, 2-pyridinone-5-yl, etc.], among which several compds. are potent inhibitors of mutant IDH1 with Ki as low as 420 nM.  X-ray crystal structures of IDH1(R132H) in complex with two inhibitors are reported, showing their inhibitor-protein interactions.  These compds. can decrease the cellular concn. of D2HG, reduce the levels of histone methylation, and suppress the proliferation of stem-like cancer cells in BT142 glioma with IDH1 R132H mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHlhXpk4PcnbVg90H21EOLACvtfcHk0lg_OLMDKBbL_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlequ7nI&md5=57dd47bcd29b65244b33e564910419de</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00684%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DKogiso%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DX.%2BN.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520cancer-associated%2520mutant%2520isocitrate%2520dehydrogenases%2520by%25202-thiohydantoin%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6899%26epage%3D6908%26doi%3D10.1021%2Facs.jmedchem.5b00684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of LSD1 for the treatment of MLL-rearranged leukemia</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">24</span>, <span class="refDoi"> DOI: 10.1186/s13045-016-0252-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1186%2Fs13045-016-0252-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=26970896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1eitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=24&author=Z.+Fengauthor=Y.+Yaoauthor=C.+Zhouauthor=F.+Chenauthor=F.+Wuauthor=L.+Weiauthor=W.+Liuauthor=S.+Dongauthor=M.+Redellauthor=Q.+Moauthor=Y.+Song&title=Pharmacological+inhibition+of+LSD1+for+the+treatment+of+MLL-rearranged+leukemia&doi=10.1186%2Fs13045-016-0252-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of LSD1 for the treatment of MLL-rearranged leukemia</span></div><div class="casAuthors">Feng, Zizhen; Yao, Yuan; Zhou, Chao; Chen, Fengju; Wu, Fangrui; Wei, Liping; Liu, Wei; Dong, Shuo; Redell, Michele; Mo, Qianxing; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hematology & Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24/1-24/13</span>CODEN:
                <span class="NLM_cas:coden">JHOOAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-8722</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Mixed lineage leukemia (MLL) gene translocations are found in ∼75 % infant and 10 % adult acute leukemia, showing a poor prognosis.  Lysine-specific demethylase 1 (LSD1) has recently been implicated to be a drug target for this subtype of leukemia.  More studies using potent LSD1 inhibitors against MLL-rearranged leukemia are needed.  Methods: LSD1 inhibitors were examd. for their biochem. and biol. activities against LSD1 and MLL-rearranged leukemia as well as other cancer cells.  Results: Potent LSD1 inhibitors with biochem. IC50 values of 9.8-77 nM were found to strongly inhibit proliferation of MLL-rearranged leukemia cells with EC50 of 10-320 nM, while these compds. are generally non-cytotoxic to several other tumor cells.  LSD1 inhibition increased histone H3 lysine 4 (H3K4) methylation, downregulated expression of several leukemia-relevant genes, induced apoptosis and differentiation, and inhibited self-renewal of stem-like leukemia cells.  Moreover, LSD1 inhibitors worked synergistically with inhibition of DOT1L, a histone H3 lysine 79 (H3K79) methyltransferase, against MLL-rearranged leukemia.  The most potent LSD1 inhibitor showed significant in vivo activity in a systemic mouse model of MLL-rearranged leukemia without overt toxicities.  Mechanistically, LSD1 inhibitors caused significant upregulation of several pathways that promote hematopoietic differentiation and apoptosis.  Conclusions: LSD1 is a drug target for MLL-rearranged leukemia, and LSD1 inhibitors are potential therapeutics for the malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUF2TPVvRgIbVg90H21EOLACvtfcHk0lg_OLMDKBbL_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1eitr8%253D&md5=3ffec94f558982ca319649ebc8708286</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2Fs13045-016-0252-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-016-0252-7%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DRedell%26aufirst%3DM.%26aulast%3DMo%26aufirst%3DQ.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DPharmacological%2520inhibition%2520of%2520LSD1%2520for%2520the%2520treatment%2520of%2520MLL-rearranged%2520leukemia%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2016%26volume%3D9%26spage%3D24%26doi%3D10.1186%2Fs13045-016-0252-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Inhibition of histone H3K79 methylation selectively inhibits proliferation, self-renewal and metastatic potential of breast cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10665</span>– <span class="NLM_lpage">10677</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2496</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.18632%2Foncotarget.2496" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=25359765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A280%3ADC%252BC2M3kslylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=10665-10677&author=L.+Zhangauthor=L.+Dengauthor=F.+Chenauthor=Y.+Yaoauthor=B.+Wuauthor=L.+Weiauthor=Q.+Moauthor=Y.+Song&title=Inhibition+of+histone+H3K79+methylation+selectively+inhibits+proliferation%2C+self-renewal+and+metastatic+potential+of+breast+cancer&doi=10.18632%2Foncotarget.2496"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of histone H3K79 methylation selectively inhibits proliferation, self-renewal and metastatic potential of breast cancer</span></div><div class="casAuthors">Zhang Li; Deng Lisheng; Yao Yuan; Wu Bulan; Wei Liping; Chen Fengju; Mo Qianxing; Song Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">10665-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone lysine methylation regulates gene expression and cancer initiation.  Bioinformatics analysis suggested that DOT1L, a histone H3-lysine79 (H3K79) methyltransferase, plays a potentially important role in breast cancer.  DOT1L inhibition selectively inhibited proliferation, self-renewal, metastatic potential of breast cancer cells and induced cell differentiation.  In addition, inhibitors of S-adenosylhomocysteine hydrolase (SAHH), such as neplanocin and 3-deazaneplanocin, also inhibited both H3K79 methylation and proliferation of breast cancer cells in vitro and in vivo.  The activity of SAHH inhibitors was previously attributed to inhibition of H3K27 methyltransferase EZH2.  However, inhibition of EZH2 by a specific inhibitor did not contribute to cell death.  SAHH inhibitors had only weak activity against H3K27 methylation and their activity is therefore mainly due to DOT1L/H3K79 methylation inhibition.  Overall, we showed that DOT1L is a potential drug target for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcToXKVEBdgQcLHHuJhbyicZfW6udTcc2eZs1YgGMxs-DLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3kslylsg%253D%253D&md5=c495f4521165f0fb79399b80d8181955</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2496&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2496%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DMo%26aufirst%3DQ.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520histone%2520H3K79%2520methylation%2520selectively%2520inhibits%2520proliferation%252C%2520self-renewal%2520and%2520metastatic%2520potential%2520of%2520breast%2520cancer%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D10665%26epage%3D10677%26doi%3D10.18632%2Foncotarget.2496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M.</span></span> <span> </span><span class="NLM_article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <i>EMBnet.journal</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.14806/ej.17.1.200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.14806%2Fej.17.1.200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=3&author=M.+Martin&title=Cutadapt+removes+adapter+sequences+from+high-throughput+sequencing+reads&doi=10.14806%2Fej.17.1.200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.14806%2Fej.17.1.200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14806%252Fej.17.1.200%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%26atitle%3DCutadapt%2520removes%2520adapter%2520sequences%2520from%2520high-throughput%2520sequencing%2520reads%26jtitle%3DEMBnet.journal%26date%3D2011%26volume%3D17%26spage%3D3%26doi%3D10.14806%2Fej.17.1.200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S.</span></span> FastQC. A quality control tool for high throughput sequence data. <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="extLink">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a> (accessed March 27, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Andrews%2C+S.+FastQC.+A+quality+control+tool+for+high+throughput+sequence+data.+http%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F+%28accessed+March+27%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DS.%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pertea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pertea, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leek, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzberg, S. L.</span></span> <span> </span><span class="NLM_article-title">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1650</span>– <span class="NLM_lpage">1667</span>, <span class="refDoi"> DOI: 10.1038/nprot.2016.095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fnprot.2016.095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=27560171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCgsbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1650-1667&author=M.+Perteaauthor=D.+Kimauthor=G.+M.+Perteaauthor=J.+T.+Leekauthor=S.+L.+Salzberg&title=Transcript-level+expression+analysis+of+RNA-seq+experiments+with+HISAT%2C+StringTie+and+Ballgown&doi=10.1038%2Fnprot.2016.095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown</span></div><div class="casAuthors">Pertea, Mihaela; Kim, Daehwan; Pertea, Geo M.; Leek, Jeffrey T.; Salzberg, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1650-1667</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput sequencing of mRNA (RNA-seq) has become the std. method for measuring and comparing the levels of gene expression in a wide variety of species and conditions.  RNA-seq expts. generate very large, complex data sets that demand fast, accurate and flexible software to reduce the raw read data to comprehensible results.  HISAT (hierarchical indexing for spliced alignment of transcripts), StringTie and Ballgown are free, open-source software tools for comprehensive anal. of RNA-seq expts.  Together, they allow scientists to align reads to a genome, assemble transcripts including novel splice variants, compute the abundance of these transcripts in each sample and compare expts. to identify differentially expressed genes and transcripts.  This protocol describes all the steps necessary to process a large set of raw sequencing reads and create lists of gene transcripts, expression levels, and differentially expressed genes and transcripts.  The protocol's execution time depends on the computing resources, but it typically takes under 45 min of computer time.  HISAT, StringTie and Ballgown are available from http://ccb.jhu.edu/software.shtml.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnBTDxxNF_yrVg90H21EOLACvtfcHk0lg48jR00FY9Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCgsbzI&md5=17da34fc59b6d109b96a5e0068d61555</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.095%26sid%3Dliteratum%253Aachs%26aulast%3DPertea%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DPertea%26aufirst%3DG.%2BM.%26aulast%3DLeek%26aufirst%3DJ.%2BT.%26aulast%3DSalzberg%26aufirst%3DS.%2BL.%26atitle%3DTranscript-level%2520expression%2520analysis%2520of%2520RNA-seq%2520experiments%2520with%2520HISAT%252C%2520StringTie%2520and%2520Ballgown%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1650%26epage%3D1667%26doi%3D10.1038%2Fnprot.2016.095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handsaker, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wysoker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marth, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecasis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, R.</span></span> <span> </span><span class="NLM_article-title">The sequence alignment/map format and SAMtools</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2078</span>– <span class="NLM_lpage">2079</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btp352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1093%2Fbioinformatics%2Fbtp352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=19505943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpslertr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2078-2079&author=H.+Liauthor=B.+Handsakerauthor=A.+Wysokerauthor=T.+Fennellauthor=J.+Ruanauthor=N.+Homerauthor=G.+Marthauthor=G.+Abecasisauthor=R.+Durbin&title=The+sequence+alignment%2Fmap+format+and+SAMtools&doi=10.1093%2Fbioinformatics%2Fbtp352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Sequence Alignment/Map format and SAMtools</span></div><div class="casAuthors">Li, Heng; Handsaker, Bob; Wysoker, Alec; Fennell, Tim; Ruan, Jue; Homer, Nils; Marth, Gabor; Abecasis, Goncalo; Durbin, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2078-2079</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: The Sequence Alignment/Map (SAM) format is a generic alignment format for storing read alignments against ref. sequences, supporting short and long reads (up to 128 Mbp) produced by different sequencing platforms.  It is flexible in style, compact in size, efficient in random access and is the format in which alignments from the 1000 Genomes Project are released.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments.  Availability: http://samtools.sourceforge.net Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="24564064454a4341560a45470a514f0a">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-IkvoI5O3jLVg90H21EOLACvtfcHk0lg48jR00FY9Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpslertr8%253D&md5=1ab7714968487a35cce7f81b751a0b1a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp352%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DHandsaker%26aufirst%3DB.%26aulast%3DWysoker%26aufirst%3DA.%26aulast%3DFennell%26aufirst%3DT.%26aulast%3DRuan%26aufirst%3DJ.%26aulast%3DHomer%26aufirst%3DN.%26aulast%3DMarth%26aufirst%3DG.%26aulast%3DAbecasis%26aufirst%3DG.%26aulast%3DDurbin%26aufirst%3DR.%26atitle%3DThe%2520sequence%2520alignment%252Fmap%2520format%2520and%2520SAMtools%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3D2078%26epage%3D2079%26doi%3D10.1093%2Fbioinformatics%2Fbtp352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Anthony Mastracchio, Chunqiu Lai, Enrico Digiammarino, Damien B. Ready, Loren M. Lasko, Kenneth D. Bromberg, William J. McClellan, Debra Montgomery, Vlasios Manaves, Bailin Shaw, Mikkel Algire, Melanie J. Patterson, Chaohong C. Sun, Saul Rosenberg, Albert Lai, <span class="NLM_string-name hlFld-ContribAuthor">Michael R. Michaelides</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a Potent and Selective Covalent p300/CBP Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (5)
                                     , 726-731. <a href="https://doi.org/10.1021/acsmedchemlett.0c00654" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00654</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00654%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Ba%252BPotent%252Band%252BSelective%252BCovalent%252Bp300%25252FCBP%252BInhibitor%26aulast%3DMastracchio%26aufirst%3DAnthony%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16122020%26date%3D02042021%26date%3D05042021%26volume%3D12%26issue%3D5%26spage%3D726%26epage%3D731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aaron R.  Waddell</span>, <span class="hlFld-ContribAuthor ">Haojie  Huang</span>, <span class="hlFld-ContribAuthor ">Daiqing  Liao</span>. </span><span class="cited-content_cbyCitation_article-title">CBP/p300: Critical Co-Activators for Nuclear Steroid Hormone Receptors and Emerging Therapeutic Targets in Prostate and Breast Cancers. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (12)
                                     , 2872. <a href="https://doi.org/10.3390/cancers13122872" title="DOI URL">https://doi.org/10.3390/cancers13122872</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13122872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13122872%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DCBP%25252Fp300%25253A%252BCritical%252BCo-Activators%252Bfor%252BNuclear%252BSteroid%252BHormone%252BReceptors%252Band%252BEmerging%252BTherapeutic%252BTargets%252Bin%252BProstate%252Band%252BBreast%252BCancers%26aulast%3DWaddell%26aufirst%3DAaron%2BR.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D12%26spage%3D2872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anca M  Farcas</span>, <span class="hlFld-ContribAuthor ">Sankari  Nagarajan</span>, <span class="hlFld-ContribAuthor ">Sabina  Cosulich</span>, <span class="hlFld-ContribAuthor ">Jason S  Carroll</span>. </span><span class="cited-content_cbyCitation_article-title">Genome-Wide Estrogen Receptor Activity in Breast Cancer. </span><span class="cited-content_cbyCitation_journal-name">Endocrinology</span><span> <strong>2021,</strong> <em>162 </em>
                                    (2)
                                     <a href="https://doi.org/10.1210/endocr/bqaa224" title="DOI URL">https://doi.org/10.1210/endocr/bqaa224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1210/endocr/bqaa224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1210%2Fendocr%2Fbqaa224%26sid%3Dliteratum%253Aachs%26jtitle%3DEndocrinology%26atitle%3DGenome-Wide%252BEstrogen%252BReceptor%252BActivity%252Bin%252BBreast%252BCancer%26aulast%3DFarcas%26aufirst%3DAnca%2BM%26date%3D2021%26date%3D2020%26volume%3D162%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhang-Xu  He</span>, <span class="hlFld-ContribAuthor ">Bing-Fei  Wei</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Yun-Peng  Gong</span>, <span class="hlFld-ContribAuthor ">Li-Ying  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Current development of CBP/p300 inhibitors in the last decade. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112861. <a href="https://doi.org/10.1016/j.ejmech.2020.112861" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112861</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112861%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCurrent%252Bdevelopment%252Bof%252BCBP%25252Fp300%252Binhibitors%252Bin%252Bthe%252Blast%252Bdecade%26aulast%3DHe%26aufirst%3DZhang-Xu%26date%3D2021%26volume%3D209%26spage%3D112861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Hong</span>, <span class="hlFld-ContribAuthor ">Jun-Ming  Yue</span>, <span class="hlFld-ContribAuthor ">Wen-Jing  Zhang</span>, <span class="hlFld-ContribAuthor ">Bing-Mei  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Mechanisms of Angiogenesis in the Ischemic Heart Diseases with Acupuncture Treatment. </span><span class="cited-content_cbyCitation_journal-name">Medical Acupuncture</span><span> <strong>2020,</strong> <em>32 </em>
                                    (6)
                                     , 381-384. <a href="https://doi.org/10.1089/acu.2020.1487" title="DOI URL">https://doi.org/10.1089/acu.2020.1487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/acu.2020.1487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Facu.2020.1487%26sid%3Dliteratum%253Aachs%26jtitle%3DMedical%2520Acupuncture%26atitle%3DEpigenetic%252BMechanisms%252Bof%252BAngiogenesis%252Bin%252Bthe%252BIschemic%252BHeart%252BDiseases%252Bwith%252BAcupuncture%252BTreatment%26aulast%3DHong%26aufirst%3DHao%26date%3D2020%26volume%3D32%26issue%3D6%26spage%3D381%26epage%3D384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Hong</span>, <span class="hlFld-ContribAuthor ">Xin-yue  Jing</span>, <span class="hlFld-ContribAuthor ">Shu-bao  Liu</span>, <span class="hlFld-ContribAuthor ">Yong-jun  Peng</span>, <span class="hlFld-ContribAuthor ">Bing-mei  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Landscape Changes Due to Acupuncture Treatment: From Clinical to Basic Research. </span><span class="cited-content_cbyCitation_journal-name">Chinese Journal of Integrative Medicine</span><span> <strong>2020,</strong> <em>26 </em>
                                    (8)
                                     , 633-640. <a href="https://doi.org/10.1007/s11655-020-2852-x" title="DOI URL">https://doi.org/10.1007/s11655-020-2852-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11655-020-2852-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11655-020-2852-x%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Journal%2520of%2520Integrative%2520Medicine%26atitle%3DEpigenetic%252BLandscape%252BChanges%252BDue%252Bto%252BAcupuncture%252BTreatment%25253A%252BFrom%252BClinical%252Bto%252BBasic%252BResearch%26aulast%3DHong%26aufirst%3DHao%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D8%26spage%3D633%26epage%3D640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. P300/CBP HAT catalyzed reaction and inhibitors with their IC<sub>50</sub> values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Methods for Thiophene-Containing Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide, 1-hydroxybenzotriazole, 4-(aminomethyl)-1-BOC-piperidine, CH<sub>2</sub>Cl<sub>2</sub>, 96% yield; (ii) Ar<sup>1</sup>-B(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 80 °C, 80–90% yield; (iii) Ar<sup>2</sup>-B(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 100 °C, 80–90% yield; (iv) HCl in 1,4-dioxane, >90% yield; (v) NaBH<sub>4</sub>, MeOH; (vi) cyanuric chloride, DMF, 81% yield for the 2 steps; (vii) 4-(aminomethyl)-1-BOC-piperidine or other amine analogs, K<sub>2</sub>CO<sub>3</sub>, DMF, 60–85% yield; (viii) <i>N</i>-bromosuccinimide, CH<sub>2</sub>Cl<sub>2</sub>; (ix) NaClO<sub>2</sub>, NaH<sub>2</sub>PO<sub>4</sub>, <i>tert</i>-butanol-H<sub>2</sub>O, 2-methyl-2-butene; (x) 2,3-diaminopropanoic acid, NaOH, MeOH, reflux, 55% yield; (xi) ClCO<sub>2</sub>Me, (<i>i</i>Pr)<sub>2</sub>NEt, THF, 95% yield.</p></p></figure><figure data-id="cht1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0001.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Structures and Inhibitory Activity of Compounds <b>33</b>–<b>40</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>12</b> is a competitive inhibitor of p300 HAT against the substrate histone. (A) Plots of IC<sub>50</sub> values of <b>12</b> versus increasing concentrations of (left) Ac-CoA (0.5–8 μM, or (0.07–1.2) × <i>K</i><sub>m</sub>) and (right) histone H3 suggest the inhibitor is noncompetitive against Ac-CoA and competitive against histone. (B) Alpha assay results show compound <b>12</b> can dose-dependently disrupt the binding between p300 HAT and histone H4. But such binding was not affected by inactive compound <b>28</b> as well as A-485 which is an inhibitor competitive against Ac-CoA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking results for compound <b>12</b>: (A) 10 docking conformations of <b>12</b> (with C atoms in green) with lowest energies in the crystal structure of p300-HAT (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PZS">4PZS</a>, shown as an electrostatic surface) in complex with Ac-CoA (as a tube model with C atoms in brown); (B) the lowest energy docking conformation of <b>12</b> (as a ball-and-stick model) with surrounding residues and Ac-CoA in p300-HAT (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PZS">4PZS</a>). Hydrogen bonds are shown as yellow dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cellular activity of compounds <b>12</b>, <b>28</b>, and A-485 against acetylation of H3K9, H3K18, and H3K27.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/medium/jm9b02164_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Gene set enrichment analysis (GSEA) results showing significant gene expression changes (<i>p</i> < 0.001) in ER+ MCF-7 cells by treatment with p300 HAT inhibitor <b>12</b>, as compared to the starved (without estradiol) or control (with estradiol) group. (A) Inhibitor treatment caused significant downregulation of genes that were suppressed by siRNA-mediated p300 knockdown (GSE31873). (B–D) Results showing compound <b>12</b> counteracted estradiol in MCF-7 cells. Treatment with <b>12</b> significantly (B) suppressed expression of genes that were upregulated by estradiol and (C) induced expression of genes that were downregulated by estradiol. (D, E) GSEA heatmaps of normalized enrichment scores (NES) for publicly available gene signatures (from the MSigDB database). (D) Inhibitor <b>12</b> caused significant downregulation of cancer-related gene sets that were induced by estradiol. (E) Inhibitor <b>12</b> suppressed expression of gene sets associated with breast cancer as well as the poor prognosis, progression, invasion, and relapse of breast cancer.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b02164/20200507/images/large/jm9b02164_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02164&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77215" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77215" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span> <span> </span><span class="NLM_article-title">The epigenomics of cancer</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">683</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.01.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.cell.2007.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=17320506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=683-692&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+epigenomics+of+cancer&doi=10.1016%2Fj.cell.2007.01.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenomics of cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-692</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Aberrant gene function and altered patterns of gene expression are key features of cancer.  Growing evidence shows that acquired epigenetic abnormalities participate with genetic alterations to cause this dysregulation.  Here, we review recent advances in understanding how epigenetic alterations participate in the earliest stages of neoplasia, including stem/precursor cell contributions, and discuss the growing implications of these advances for strategies to control cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBpHNAkcSPbVg90H21EOLACvtfcHk0ljdpFMFi8COgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D&md5=78eac8ecc093f1aed0acbfd578f34717</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520epigenomics%2520of%2520cancer%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D683%26epage%3D692%26doi%3D10.1016%2Fj.cell.2007.01.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Chromatin modifications and their function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lhwRG-gjAUgRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1038/nrc3256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer&doi=10.1038%2Fnrc3256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lhwRG-gjAUgRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477%26doi%3D10.1038%2Fnrc3256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yik, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span> <span> </span><span class="NLM_article-title">Recruitment of P-TEFb for stimulation of transcriptional elongation by the bromodomain protein Brd4</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.06.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.molcel.2005.06.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=16109377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=535-545&author=Z.+Yangauthor=J.+H.+Yikauthor=R.+Chenauthor=N.+Heauthor=M.+K.+Jangauthor=K.+Ozatoauthor=Q.+Zhou&title=Recruitment+of+P-TEFb+for+stimulation+of+transcriptional+elongation+by+the+bromodomain+protein+Brd4&doi=10.1016%2Fj.molcel.2005.06.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Recruitment of P-TEFb for stimulation of transcriptional elongation by the bromodomain protein Brd4</span></div><div class="casAuthors">Yang, Zhiyuan; Yik, Jasper H. N.; Chen, Ruichuan; He, Nanhai; Jang, Moon Kyoo; Ozato, Keiko; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">535-545</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cyclinT1/Cdk9 heterodimer that constitutes core P-TEFb is generally presumed to be the transcriptionally active form for stimulating RNA polymerase II elongation.  About half of cellular P-TEFb also exists in an inactive complex with the 7SK snRNA and the HEXIM1 protein.  Here, we show that the remaining half assocs. with the bromodomain protein Brd4.  In stress-induced cells, the 7SK/HEXIM1-bound P-TEFb is quant. converted into the Brd4-assocd. form.  The assocn. with Brd4 is necessary to form the transcriptionally active P-TEFb, recruits P-TEFb to a promoter, and enables P-TEFb to contact the Mediator complex, a potential target for the Brd4-mediated recruitment.  Although generally required for transcription, the P-TEFb-recruitment function of Brd4 can be substituted by that of HIV-1 Tat, which recruits P-TEFb directly for activated HIV-1 transcription.  Brd4, HEXIM1, and 7SK are all implicated in regulating cell growth, which may result from their dynamic control of the general transcription factor P-TEFb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptFELcZRBnvLVg90H21EOLACvtfcHk0lhwRG-gjAUgRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvF2lur8%253D&md5=50325264e1eb27789eb46999ca3f799d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.06.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.06.029%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DYik%26aufirst%3DJ.%2BH.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DHe%26aufirst%3DN.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DRecruitment%2520of%2520P-TEFb%2520for%2520stimulation%2520of%2520transcriptional%2520elongation%2520by%2520the%2520bromodomain%2520protein%2520Brd4%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D535%26epage%3D545%26doi%3D10.1016%2Fj.molcel.2005.06.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiltz, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizzen, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span> <span> </span><span class="NLM_article-title">Overlapping but distinct patterns of histone acetylation by the human coactivators p300 and PCAF within nucleosomal substrates</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">1189</span>– <span class="NLM_lpage">1192</span>, <span class="refDoi"> DOI: 10.1074/jbc.274.3.1189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1074%2Fjbc.274.3.1189" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=9880483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADyaK1MXpsFKktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=1189-1192&author=R.+L.+Schiltzauthor=C.+A.+Mizzenauthor=A.+Vassilevauthor=R.+G.+Cookauthor=C.+D.+Allisauthor=Y.+Nakatani&title=Overlapping+but+distinct+patterns+of+histone+acetylation+by+the+human+coactivators+p300+and+PCAF+within+nucleosomal+substrates&doi=10.1074%2Fjbc.274.3.1189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Overlapping but distinct patterns of histone acetylation by the human coactivators p300 and PCAF within nucleosomal substrates</span></div><div class="casAuthors">Schiltz, R. Louis; Mizzen, Craig A.; Vassilev, Alex; Cook, Richard G.; Allis, C. David; Nakatani, Yoshihiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1189-1192</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A no. of transcriptional coactivators possess intrinsic histone acetylase activity, providing a direct link between hyperacetylated chromatin and transcriptional activation.  We have detd. the core histone residues acetylated in vitro by recombinant p300 and PCAF within mononucleosomes.  P300 specifically acetylates all sites of histones H2A and H2B known to be acetylated in bulk chromatin in vivo but preferentially acetylates lysines 14 and 18 of histone H3 and lysines 5 and 8 of histone H4.  PCAF primarily acetylates lysine 14 of H3 but also less efficiently acetylates lysine 8 of H4.  PCAF in its native form, which is present in a stable multimeric protein complex lacking p300/CBP, primarily acetylates H3 to a monoacetylated form, suggesting that PCAF-assocd. polypeptides do not alter the substrate specificity.  These distinct patterns of acetylation by the p300 and PCAF may contribute to their differential roles in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMMQsOC8Vdl7Vg90H21EOLACvtfcHk0lh42qyBChhBiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXpsFKktw%253D%253D&md5=b7d50ee9c8640dcabd7baf63473e69fc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.3.1189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.3.1189%26sid%3Dliteratum%253Aachs%26aulast%3DSchiltz%26aufirst%3DR.%2BL.%26aulast%3DMizzen%26aufirst%3DC.%2BA.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DCook%26aufirst%3DR.%2BG.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DNakatani%26aufirst%3DY.%26atitle%3DOverlapping%2520but%2520distinct%2520patterns%2520of%2520histone%2520acetylation%2520by%2520the%2520human%2520coactivators%2520p300%2520and%2520PCAF%2520within%2520nucleosomal%2520substrates%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D1189%26epage%3D1192%26doi%3D10.1074%2Fjbc.274.3.1189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dancy, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Protein lysine acetylation by p300/CBP</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">2419</span>– <span class="NLM_lpage">2452</span>, <span class="refDoi"> DOI: 10.1021/cr500452k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500452k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=2419-2452&author=B.+M.+Dancyauthor=P.+A.+Cole&title=Protein+lysine+acetylation+by+p300%2FCBP&doi=10.1021%2Fcr500452k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Acetylation by p300/CBP</span></div><div class="casAuthors">Dancy, Beverley M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2419-2452</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The human protein lysine acetyltransferase encoded by the paralogous p300 and CBP genes is involved in regulating the expression of genes.  In this capacity, p300/CBP controls several basic cellular processes such as proliferation and homeostasis, and plays a role in a variety of human diseases, particularly solid tumors.  This review summarizes information related to the current understanding of p300/CBP, including the novel technologies that have co-evolved with p300/CBP research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYgTImMiuo7rVg90H21EOLACvtfcHk0lh42qyBChhBiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D&md5=40a46cf68990f4a7d11a338c54081628</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fcr500452k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500452k%26sid%3Dliteratum%253Aachs%26aulast%3DDancy%26aufirst%3DB.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DProtein%2520lysine%2520acetylation%2520by%2520p300%252FCBP%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D2419%26epage%3D2452%26doi%3D10.1021%2Fcr500452k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikura, M.</span></span> <span> </span><span class="NLM_article-title">Transcriptional/epigenetic regulator CBP/p300 in tumorigenesis: structural and functional versatility in target recognition</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">3989</span>– <span class="NLM_lpage">4008</span>, <span class="refDoi"> DOI: 10.1007/s00018-012-1254-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1007%2Fs00018-012-1254-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=23307074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Wmt7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2013&pages=3989-4008&author=F.+Wangauthor=C.+B.+Marshallauthor=M.+Ikura&title=Transcriptional%2Fepigenetic+regulator+CBP%2Fp300+in+tumorigenesis%3A+structural+and+functional+versatility+in+target+recognition&doi=10.1007%2Fs00018-012-1254-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional/epigenetic regulator CBP/p300 in tumorigenesis: structural and functional versatility in target recognition</span></div><div class="casAuthors">Wang, Feng; Marshall, Christopher B.; Ikura, Mitsuhiko</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3989-4008</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">A review.  In eukaryotic cells, gene transcription is regulated by sequence-specific DNA-binding transcription factors that recognize promoter and enhancer elements near the transcriptional start site.  Some coactivators promote transcription by connecting transcription factors to the basal transcriptional machinery.  The highly conserved coactivators CREB-binding protein (CBP) and its paralog, E1A-binding protein (p300), each have four sep. transactivation domains (TADs) that interact with the TADs of a no. of DNA-binding transcription activators as well as general transcription factors (GTFs), thus mediating recruitment of basal transcription machinery to the promoter.  Most promoters comprise multiple activator-binding sites, and many activators contain tandem TADs, thus multivalent interactions may stabilize CBP/p300 at the promoter, and intrinsically disordered regions in CBP/p300 and many activators may confer adaptability to these multivalent complexes.  CBP/p300 contains a catalytic histone acetyltransferase (HAT) domain, which remodels chromatin to 'relax' its superstructure and enables transcription of proximal genes.  The HAT activity of CBP/p300 also acetylates some transcription factors (e.g., p53), hence modulating the function of key transcriptional regulators.  Through these numerous interactions, CBP/p300 has been implicated in complex physiol. and pathol. processes, and, in response to different signals, can drive cells towards proliferation or apoptosis.  Dysregulation of the transcriptional and epigenetic functions of CBP/p300 is assocd. with leukemia and other types of cancer, thus it has been recognized as a potential anti-cancer drug target.  In this review, we focus on recent exciting findings in the structural mechanisms of CBP/p300 involving multivalent and dynamic interactions with binding partners, which may pave new avenues for anti-cancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCCkxACa47q7Vg90H21EOLACvtfcHk0lh42qyBChhBiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Wmt7rJ&md5=76333cf8a4e24cab255412d2655cdd72</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00018-012-1254-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-012-1254-4%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMarshall%26aufirst%3DC.%2BB.%26aulast%3DIkura%26aufirst%3DM.%26atitle%3DTranscriptional%252Fepigenetic%2520regulator%2520CBP%252Fp300%2520in%2520tumorigenesis%253A%2520structural%2520and%2520functional%2520versatility%2520in%2520target%2520recognition%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2013%26volume%3D70%26spage%3D3989%26epage%3D4008%26doi%3D10.1007%2Fs00018-012-1254-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Malley, B. W.</span></span> <span> </span><span class="NLM_article-title">Steroid receptor coactivators 1, 2, and 3: critical regulators of nuclear receptor activity and steroid receptor modulator (SRM)-based cancer therapy</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2011.04.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.mce.2011.04.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=21664237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFegsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2012&pages=430-439&author=A.+B.+Johnsonauthor=B.+W.+O%E2%80%99Malley&title=Steroid+receptor+coactivators+1%2C+2%2C+and+3%3A+critical+regulators+of+nuclear+receptor+activity+and+steroid+receptor+modulator+%28SRM%29-based+cancer+therapy&doi=10.1016%2Fj.mce.2011.04.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Steroid receptor coactivators 1, 2, and 3: Critical regulators of nuclear receptor activity and steroid receptor modulator (SRM)-based cancer therapy</span></div><div class="casAuthors">Johnson, Amber B.; O'Malley, Bert W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">430-439</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Coactivators are a diverse group of non-DNA binding proteins that induce structural changes in agonist-bound nuclear receptors (NRs) that are essential for NR-mediated transcriptional activation.  Once bound, coactivators function to bridge enhancer binding proteins to the general transcription machinery, as well as to recruit secondary coactivators that modify promoter and enhancer chromatin in a manner permissive for transcriptional activation.  In the following review article, we focus on one of the most in-depth studied families of coactivators, the steroid receptor coactivators (SRC) 1, 2, and 3.  SRCs are widely implicated in NR-mediated diseases, esp. in cancers, with the majority of studies focused on their roles in breast cancer.  We highlight the relevant literature supporting the oncogenic activity of SRCs and their future as diagnostic and prognostic indicators.  With much interest in the development of selective receptor modulators (SRMs), we focus on how these coactivators regulate the interactions between SRMs and their resp. NRs; and, importantly, the influence that coactivators have on the functional output of SRMs.  Furthermore, we speculate that coactivator-specific inhibitors could provide powerful, all-encompassing treatments that target multiple modes of oncogenic regulation in cancers resistant to typical anti-endocrine treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2-LVSF6p5yLVg90H21EOLACvtfcHk0ljnJB0Srpm-Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFegsbzM&md5=07d8c01d3f3d11ff0e4bde556fbab664</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2011.04.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2011.04.021%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DA.%2BB.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DB.%2BW.%26atitle%3DSteroid%2520receptor%2520coactivators%25201%252C%25202%252C%2520and%25203%253A%2520critical%2520regulators%2520of%2520nuclear%2520receptor%2520activity%2520and%2520steroid%2520receptor%2520modulator%2520%2528SRM%2529-based%2520cancer%2520therapy%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2012%26volume%3D348%26spage%3D430%26epage%3D439%26doi%3D10.1016%2Fj.mce.2011.04.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dilworth, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambon, P.</span></span> <span> </span><span class="NLM_article-title">Nuclear receptors coordinate the activities of chromatin remodeling complexes and coactivators to facilitate initiation of transcription</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3047</span>– <span class="NLM_lpage">3054</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1204329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fsj.onc.1204329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=11420720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD3MXks1Glurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2001&pages=3047-3054&author=F.+J.+Dilworthauthor=P.+Chambon&title=Nuclear+receptors+coordinate+the+activities+of+chromatin+remodeling+complexes+and+coactivators+to+facilitate+initiation+of+transcription&doi=10.1038%2Fsj.onc.1204329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear receptors coordinate the activities of chromatin remodeling complexes and coactivators to facilitate initiation of transcription</span></div><div class="casAuthors">Dilworth, F. Jeffrey; Chambon, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3047-3054</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent advances in the field of in vitro chromatin assembly have led to in vitro transcription systems which reproduce in the test tube, in vivo characteristics of ligand-dependent transcriptional activation by nuclear receptors.  Dissection of these systems has begun to provide us with information concerning the underlying mol. mechanisms.  Through recruitment of coactivator proteins, nuclear receptors act first to remodel chromatin within the promoter region and then to recruit the transcriptional machinery to the promoter region in order to initiate transcription.  Here we present a possible sequential mechanism for ligand-dependent transcriptional activation by nuclear receptors and discuss the in vitro and in vivo data that support this model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqisxFLqc8kKbVg90H21EOLACvtfcHk0ljnJB0Srpm-Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXks1Glurw%253D&md5=b99fe17405e01dd20f906c7169e2f842</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1204329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1204329%26sid%3Dliteratum%253Aachs%26aulast%3DDilworth%26aufirst%3DF.%2BJ.%26aulast%3DChambon%26aufirst%3DP.%26atitle%3DNuclear%2520receptors%2520coordinate%2520the%2520activities%2520of%2520chromatin%2520remodeling%2520complexes%2520and%2520coactivators%2520to%2520facilitate%2520initiation%2520of%2520transcription%26jtitle%3DOncogene%26date%3D2001%26volume%3D20%26spage%3D3047%26epage%3D3054%26doi%3D10.1038%2Fsj.onc.1204329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manning, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadonaga, J. T.</span></span> <span> </span><span class="NLM_article-title">Biochemical analysis of distinct activation functions in p300 that enhance transcription initiation with chromatin templates</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">8123</span>– <span class="NLM_lpage">8135</span>, <span class="refDoi"> DOI: 10.1128/MCB.19.12.8123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1128%2FMCB.19.12.8123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10567538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ygt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1999&pages=8123-8135&author=W.+L.+Krausauthor=E.+T.+Manningauthor=J.+T.+Kadonaga&title=Biochemical+analysis+of+distinct+activation+functions+in+p300+that+enhance+transcription+initiation+with+chromatin+templates&doi=10.1128%2FMCB.19.12.8123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical analysis of distinct activation functions in p300 that enhance transcription initiation with chromatin templates</span></div><div class="casAuthors">Kraus, W. Lee; Manning, E. Tory; Kadonaga, James T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8123-8135</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">To investigate the mechanisms of transcriptional enhancement by the p300 coactivator, we analyzed wild-type and mutant versions of p300 with a chromatin transcription system in vitro.  Estrogen receptor, NF-κB p65 plus Sp1, and Gal4-VP16 were used as different sequence-specific activators.  The CH3 domain (or E1A-binding region) was found to be essential for the function of each of the activators tested.  The bromodomain was also obsd. to be generally important for p300 coactivator activity, though to a lesser extent than the CH3 domain/E1A-binding region.  The acetyltransferase activity and the C-terminal region (contg. the steroid receptor coactivator/p160-binding region and the glutamine-rich region) were each found to be important for activation by estrogen receptor but not for that by Gal4-VP16.  The N-terminal region of p300, which had been previously found to interact with nuclear hormone receptors, was not seen to be required for any of the activators, including estrogen receptor.  Single-round transcription expts. revealed that the functionally important subregions of p300 contribute to its ability to promote the assembly of transcription initiation complexes.  In addn., the acetyltransferase activity of p300 was obsd. to be distinct from the broadly essential activation function of the CH3 domain/E1A-binding region.  These results indicate that specific regions of p300 possess distinct activation functions that are differentially required to enhance the assembly of transcription initiation complexes.  Interestingly, with the estrogen receptor, four distinct regions of p300 each have an essential role in the transcription activation process.  These data exemplify a situation in which a network of multiple activation functions is required to achieve gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcZAOnfpDfObVg90H21EOLACvtfcHk0ljnJB0Srpm-Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXns1ygt74%253D&md5=b13149bf90d7f7d302a833df34ed14f6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.19.12.8123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.19.12.8123%26sid%3Dliteratum%253Aachs%26aulast%3DKraus%26aufirst%3DW.%2BL.%26aulast%3DManning%26aufirst%3DE.%2BT.%26aulast%3DKadonaga%26aufirst%3DJ.%2BT.%26atitle%3DBiochemical%2520analysis%2520of%2520distinct%2520activation%2520functions%2520in%2520p300%2520that%2520enhance%2520transcription%2520initiation%2520with%2520chromatin%2520templates%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1999%26volume%3D19%26spage%3D8123%26epage%3D8135%26doi%3D10.1128%2FMCB.19.12.8123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span> <span> </span><span class="NLM_article-title">Synergistic activation of transcription by CBP and p53</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">819</span>– <span class="NLM_lpage">823</span>, <span class="refDoi"> DOI: 10.1038/42972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2F42972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=9194564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADyaK2sXktVyjsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=1997&pages=819-823&author=W.+Guauthor=X.+L.+Shiauthor=R.+G.+Roeder&title=Synergistic+activation+of+transcription+by+CBP+and+p53&doi=10.1038%2F42972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic activation of transcription by CBP and p53</span></div><div class="casAuthors">Gu, Wei; Shi, Xiao-Lu; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">6635</span>),
    <span class="NLM_cas:pages">819-823</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The tumor suppressor p53 is a transcriptional regulator whose ability to inhibit cell growth is dependent upon its transactivation function.  Here the authors demonstrate that the transcription factor CBP, which is also implicated in cell proliferation and differentiation, act as a p53 coactivator and potentiates its transcriptional activity.  The amino-terminal activation domain of p53 interacts with the carboxy-terminal portion of the CBP protein both in vitro and in vivo.  In transfected SaoS-2 cells, CBP potentiates activation of the mdm-2 gene by p53 and, reciprocally, p53 potentiates activation of a Gal4-responsive target gene by a Gal4(1-147)-CBP(1678-2441) fusion protein.  A double point mutation that destroys the transactivation function of p53 also abolishes its binding to CBP and its synergistic function with CBP.  The ability of p53 to interact phys. and functionally with a coactivator (CBP) that has histone acetyltransferase activity and with components (TAFs) of the general transcription machinery indicates that it may have different functions in a multistep activation pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvql5Yg3ZCM7Vg90H21EOLACvtfcHk0ljnJB0Srpm-Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktVyjsbw%253D&md5=716c92c46feff7d9b01bafa62da3e61e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2F42972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F42972%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%2BL.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DSynergistic%2520activation%2520of%2520transcription%2520by%2520CBP%2520and%2520p53%26jtitle%3DNature%26date%3D1997%26volume%3D387%26spage%3D819%26epage%3D823%26doi%3D10.1038%2F42972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vervoorts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscher-Firzlaff, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rottmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lilischkis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsemann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dohmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscher, B.</span></span> <span> </span><span class="NLM_article-title">Stimulation of c-MYC transcriptional activity and acetylation by recruitment of the cofactor CBP</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">490</span>, <span class="refDoi"> DOI: 10.1038/sj.embor.embor821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fsj.embor.embor821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=12776737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtlaks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=484-490&author=J.+Vervoortsauthor=J.+M.+Luscher-Firzlaffauthor=S.+Rottmannauthor=R.+Lilischkisauthor=G.+Walsemannauthor=K.+Dohmannauthor=M.+Austenauthor=B.+Luscher&title=Stimulation+of+c-MYC+transcriptional+activity+and+acetylation+by+recruitment+of+the+cofactor+CBP&doi=10.1038%2Fsj.embor.embor821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Stimulation of c-MYC transcriptional activity and acetylation by recruitment of the cofactor CBP</span></div><div class="casAuthors">Vervoorts, Joerg; Luescher-Firzlaff, Juliane M.; Rottmann, Sabine; Lilischkis, Richard; Walsemann, Gesa; Dohmann, Karen; Austen, Matthias; Luescher, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">484-490</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The c-MYC oncoprotein regulates various aspects of cell behavior by modulating gene expression.  Here, we report the identification of the cAMP-response-element-binding protein (CBP) as a novel c-MYC binding partner.  The two proteins interact both in vitro and in cells, and CBP binds to the carboxy-terminal region of c-MYC.  Importantly, CBP, as well as p300, is assocd. with E-box-contg. promoter regions of genes that are regulated by c-MYC.  Furthermore, c-MYC and CBP/p300 function synergistically in the activation of reporter-gene constructs.  Thus, CBP and p300 function as pos. cofactors for c-MYC.  In addn., c-MYC is acetylated in cells.  This modification does not require MYC box II, suggesting that it is independent of TRRAP complexes.  Instead, CBP acetylates c-MYC in vitro, and co-expression of CBP with c-MYC stimulates in vivo acetylation.  Functionally, this results in a decrease in ubiquitination and stabilization of c-MYC proteins.  Thus, CBP and p300 are novel functional binding partners of c-MYC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLtwFRV0LMVrVg90H21EOLACvtfcHk0ljAbefzdL-Anw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtlaks7c%253D&md5=79df9e56a829f6b1e453ec4543ae82b7</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsj.embor.embor821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.embor.embor821%26sid%3Dliteratum%253Aachs%26aulast%3DVervoorts%26aufirst%3DJ.%26aulast%3DLuscher-Firzlaff%26aufirst%3DJ.%2BM.%26aulast%3DRottmann%26aufirst%3DS.%26aulast%3DLilischkis%26aufirst%3DR.%26aulast%3DWalsemann%26aufirst%3DG.%26aulast%3DDohmann%26aufirst%3DK.%26aulast%3DAusten%26aufirst%3DM.%26aulast%3DLuscher%26aufirst%3DB.%26atitle%3DStimulation%2520of%2520c-MYC%2520transcriptional%2520activity%2520and%2520acetylation%2520by%2520recruitment%2520of%2520the%2520cofactor%2520CBP%26jtitle%3DEMBO%2520Rep.%26date%3D2003%26volume%3D4%26spage%3D484%26epage%3D490%26doi%3D10.1038%2Fsj.embor.embor821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irvine, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kushner, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coetzee, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stallcup, M. R.</span></span> <span> </span><span class="NLM_article-title">Multiple signal input and output domains of the 160-kilodalton nuclear receptor coactivator proteins</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6164</span>– <span class="NLM_lpage">6173</span>, <span class="refDoi"> DOI: 10.1128/MCB.19.9.6164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1128%2FMCB.19.9.6164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10454563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADyaK1MXlslWjsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1999&pages=6164-6173&author=H.+Maauthor=H.+Hongauthor=S.+M.+Huangauthor=R.+A.+Irvineauthor=P.+Webbauthor=P.+J.+Kushnerauthor=G.+A.+Coetzeeauthor=M.+R.+Stallcup&title=Multiple+signal+input+and+output+domains+of+the+160-kilodalton+nuclear+receptor+coactivator+proteins&doi=10.1128%2FMCB.19.9.6164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple signal input and output domains of the 160-kilodalton nuclear receptor coactivator proteins</span></div><div class="casAuthors">Ma, Han; Hong, Heng; Huang, Shih-Ming; Irvine, Ryan A.; Webb, Paul; Kushner, Peter J.; Coetzee, Gerhard A.; Stallcup, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">6164-6173</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Members of the 160-kDa nuclear receptor coactivator family (p160 coactivators) bind to the conserved AF-2 activation function found in the hormone binding domains of nuclear receptors (NR) and are potent transcriptional coactivators for NRs.  Here we report that the C-terminal region of p160 coactivators glucocorticoid receptor interacting protein 1 (GRIP1), steroid receptor coactivator 1 (SRC-1a), and SRC-1e binds the N-terminal AF-1 activation function of the androgen receptor (AR), and p160 coactivators can thereby enhance transcriptional activation by AR.  While they all interact efficiently with AR AF-1, these same coactivators have vastly different binding strengths with and coactivator effects on AR AF-2.  P160 activation domain AD1, which binds secondary coactivators CREB binding protein (CBP) and p300, was previously implicated as the principal domain for transmitting the activating signal to the transcription machinery.  We identified a new highly conserved motif in the AD1 region which is important for CBP/p300 binding.  Deletion of AD1 only partially reduced p160 coactivator function, due to signaling through AD2, another activation domain located at the C-terminal end of p160 coactivators.  C-terminal coactivator fragments lacking AD1 but contg. AD2 and the AR AF-1 binding site served as efficient coactivators for full-length AR and AR AF-1.  The two signal input domains (one that binds NR AF-2 domains and one that binds AF-1 domains of some but not all NRs) and the two signal output domains (AD1 and AD2) of p160 coactivators played different relative roles for two different NRs: AR and thyroid hormone receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVz-XWR7pJGLVg90H21EOLACvtfcHk0ljAbefzdL-Anw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlslWjsLc%253D&md5=40252087d311cf794e805c5cc91e5172</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.19.9.6164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.19.9.6164%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DIrvine%26aufirst%3DR.%2BA.%26aulast%3DWebb%26aufirst%3DP.%26aulast%3DKushner%26aufirst%3DP.%2BJ.%26aulast%3DCoetzee%26aufirst%3DG.%2BA.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26atitle%3DMultiple%2520signal%2520input%2520and%2520output%2520domains%2520of%2520the%2520160-kilodalton%2520nuclear%2520receptor%2520coactivator%2520proteins%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1999%26volume%3D19%26spage%3D6164%26epage%3D6173%26doi%3D10.1128%2FMCB.19.9.6164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N.</span></span> <span> </span><span class="NLM_article-title">Chromatin-related properties of CBP fused to MLL generate a myelodysplastic-like syndrome that evolves into myeloid leukemia</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4655</span>– <span class="NLM_lpage">4664</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.17.4655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1093%2Femboj%2F19.17.4655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10970858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmslGku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=4655-4664&author=C.+Lavauauthor=C.+Duauthor=M.+Thirmanauthor=N.+Zeleznik-Le&title=Chromatin-related+properties+of+CBP+fused+to+MLL+generate+a+myelodysplastic-like+syndrome+that+evolves+into+myeloid+leukemia&doi=10.1093%2Femboj%2F19.17.4655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin-related properties of CBP fused to MLL generate a myelodysplastic-like syndrome that evolves into myeloid leukemia</span></div><div class="casAuthors">Lavau, Catherine; Du, Changchun; Thirman, Michael; Zeleznik-Le, Nancy</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4655-4664</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">As a result of the recurring translocation t(11;16) (q23;p13.3), MLL (mixed-lineage leukemia) is fused in frame to CBP (CREB binding protein).  This translocation has been documented almost exclusively in cases of acute leukemia or myelodysplasia secondary to therapy with drugs that target DNA topoisomerase II.  The minimal chimeric protein that is produced fuses MLL to the bromodomain, histone acetyltransferase (HAT) domain, EIA-binding domain and steroid-receptor coactivator binding domains of CBP.  We show that transplantation of bone marrow retrovirally transduced with MLL-CBP induces myeloid leukemias in mice that are preceded by a long preleukemic phase similar to the myelodysplastic syndrome (MDS) seen in many t(11;16) patients but unusual for other MLL translocations.  Structure-function anal. demonstrated that fusion of both the bromodomain and HAT domain of CBP to the amino portion of MLL is required for full in vitro transformation and is sufficient to induce the leukemic phenotype in vivo.  This suggests that the leukemic effect of MLL-CBP results from the fusion of the chromatin assocn. and modifying activities of CBP with the DNA binding activities of MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKd2exsNd2TbVg90H21EOLACvtfcHk0ljAbefzdL-Anw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmslGku7k%253D&md5=6b48fda474ab9795cb043ebe55962fd2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.17.4655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.17.4655%26sid%3Dliteratum%253Aachs%26aulast%3DLavau%26aufirst%3DC.%26aulast%3DDu%26aufirst%3DC.%26aulast%3DThirman%26aufirst%3DM.%26aulast%3DZeleznik-Le%26aufirst%3DN.%26atitle%3DChromatin-related%2520properties%2520of%2520CBP%2520fused%2520to%2520MLL%2520generate%2520a%2520myelodysplastic-like%2520syndrome%2520that%2520evolves%2520into%2520myeloid%2520leukemia%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D4655%26epage%3D4664%26doi%3D10.1093%2Femboj%2F19.17.4655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gervais, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helias, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eischen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippert, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tigaud, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penther, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugneret, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speleman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talmant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanDen Akker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luquet, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen-Khac, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maarek, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sainty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flandrin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birnbaum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mozziconacci, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessard, M.</span></span> <span> </span><span class="NLM_article-title">Acute myeloid leukaemia with 8p11 (MYST3) rearrangement: an integrated cytologic, cytogenetic and molecular study by the groupe francophone de cytogenetique hematologique</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1567</span>– <span class="NLM_lpage">1575</span>, <span class="refDoi"> DOI: 10.1038/leu.2008.128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fleu.2008.128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=18528428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpslSntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1567-1575&author=C.+Gervaisauthor=A.+Muratiauthor=C.+Heliasauthor=S.+Struskiauthor=A.+Eischenauthor=E.+Lippertauthor=I.+Tigaudauthor=D.+Pentherauthor=C.+Bastardauthor=F.+Mugneretauthor=B.+Poppeauthor=F.+Spelemanauthor=P.+Talmantauthor=J.+VanDen+Akkerauthor=L.+Barangerauthor=C.+Barinauthor=I.+Luquetauthor=N.+Nadalauthor=F.+Nguyen-Khacauthor=O.+Maarekauthor=C.+Herensauthor=D.+Saintyauthor=G.+Flandrinauthor=D.+Birnbaumauthor=M.+J.+Mozziconacciauthor=M.+Lessard&title=Acute+myeloid+leukaemia+with+8p11+%28MYST3%29+rearrangement%3A+an+integrated+cytologic%2C+cytogenetic+and+molecular+study+by+the+groupe+francophone+de+cytogenetique+hematologique&doi=10.1038%2Fleu.2008.128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Acute myeloid leukaemia with 8p11 (MYST3) rearrangement: an integrated cytologic, cytogenetic and molecular study by the groupe francophone de cytogenetique hematologique</span></div><div class="casAuthors">Gervais, C.; Murati, A.; Helias, C.; Struski, S.; Eischen, A.; Lippert, E.; Tigaud, I.; Penther, D.; Bastard, C.; Mugneret, F.; Poppe, B.; Speleman, F.; Talmant, P.; Van Den Akker, J.; Baranger, L.; Barin, C.; Luquet, I.; Nadal, N.; Nguyen-Khac, F.; Maarek, O.; Herens, C.; Sainty, D.; Flandrin, G.; Birnbaum, D.; Mozziconacci, M.-J.; Lessard, M.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1567-1575</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thirty cases of acute myeloid leukemia (AML) with MYST histone acetyltransferase 3 (MYST3) rearrangement were collected in a retrospective study from 14 centers in France and Belgium.  The mean age at diagnosis was 59.4 years and 67% of the patients were females.  Most cases (77%) were secondary to solid cancer (57%), haematol. malignancy (35%) or both (8%), and appeared 25 mo after the primary disease.  Clin., cutaneous localization and disseminated intravascular coagulation were present in 30 and 40% of the cases, resp.  AMLs were myelomonocytic (7%) or monocytic (93%), with erythrophagocytosis (75%) and cytoplasmic vacuoles (75%).  Immunophenotype showed no particularity compared with monocytic leukemia without MYST3 abnormality.  Twenty-eight cases carried t(8;16)(p11;p13) with MYST3-CREBBP fusion, one case carried a variant t(8;22)(p11;q13) and one case carried a t(8;19)(p11;q13).  Type I (MYST3 exon 16-CREBBP exon 3) was the most frequent MYST3-CREBBP fusion transcript (65%).  MYST3 rearrangement was assocd. with a poor prognosis, as 50% of patients deceased during the first 10 mo.  All those particular clin., cytol., cytogenetic, mol. and prognostic characteristics of AML with MYST3 rearrangement may have allowed an individualization into the World Health Organization classification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0XL7JYU4FOLVg90H21EOLACvtfcHk0lgugtkCie6IAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpslSntbg%253D&md5=7d4824ac2008c2fd6e3d3685b4b98da2</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fleu.2008.128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2008.128%26sid%3Dliteratum%253Aachs%26aulast%3DGervais%26aufirst%3DC.%26aulast%3DMurati%26aufirst%3DA.%26aulast%3DHelias%26aufirst%3DC.%26aulast%3DStruski%26aufirst%3DS.%26aulast%3DEischen%26aufirst%3DA.%26aulast%3DLippert%26aufirst%3DE.%26aulast%3DTigaud%26aufirst%3DI.%26aulast%3DPenther%26aufirst%3DD.%26aulast%3DBastard%26aufirst%3DC.%26aulast%3DMugneret%26aufirst%3DF.%26aulast%3DPoppe%26aufirst%3DB.%26aulast%3DSpeleman%26aufirst%3DF.%26aulast%3DTalmant%26aufirst%3DP.%26aulast%3DVanDen%2BAkker%26aufirst%3DJ.%26aulast%3DBaranger%26aufirst%3DL.%26aulast%3DBarin%26aufirst%3DC.%26aulast%3DLuquet%26aufirst%3DI.%26aulast%3DNadal%26aufirst%3DN.%26aulast%3DNguyen-Khac%26aufirst%3DF.%26aulast%3DMaarek%26aufirst%3DO.%26aulast%3DHerens%26aufirst%3DC.%26aulast%3DSainty%26aufirst%3DD.%26aulast%3DFlandrin%26aufirst%3DG.%26aulast%3DBirnbaum%26aufirst%3DD.%26aulast%3DMozziconacci%26aufirst%3DM.%2BJ.%26aulast%3DLessard%26aufirst%3DM.%26atitle%3DAcute%2520myeloid%2520leukaemia%2520with%25208p11%2520%2528MYST3%2529%2520rearrangement%253A%2520an%2520integrated%2520cytologic%252C%2520cytogenetic%2520and%2520molecular%2520study%2520by%2520the%2520groupe%2520francophone%2520de%2520cytogenetique%2520hematologique%26jtitle%3DLeukemia%26date%3D2008%26volume%3D22%26spage%3D1567%26epage%3D1575%26doi%3D10.1038%2Fleu.2008.128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lasko, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Digiammarino, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaves, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algire, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karukurichi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluge, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patane, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Drie, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElligott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesicki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bromberg, K. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of a selective catalytic p300/CBP inhibitor that targets lineage-specific tumours</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>550</i></span>,  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nature24028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fnature24028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=28953875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=128-132&author=L.+M.+Laskoauthor=C.+G.+Jakobauthor=R.+P.+Edaljiauthor=W.+Qiuauthor=D.+Montgomeryauthor=E.+L.+Digiammarinoauthor=T.+M.+Hansenauthor=R.+M.+Risiauthor=R.+Freyauthor=V.+Manavesauthor=B.+Shawauthor=M.+Algireauthor=P.+Hesslerauthor=L.+T.+Lamauthor=T.+Uzielauthor=E.+Faivreauthor=D.+Fergusonauthor=F.+G.+Buchananauthor=R.+L.+Martinauthor=M.+Torrentauthor=G.+G.+Chiangauthor=K.+Karukurichiauthor=J.+W.+Langstonauthor=B.+T.+Weinertauthor=C.+Choudharyauthor=P.+de+Vriesauthor=A.+F.+Klugeauthor=M.+A.+Pataneauthor=J.+H.+Van+Drieauthor=C.+Wangauthor=D.+McElligottauthor=E.+Kesickiauthor=R.+Marmorsteinauthor=C.+Sunauthor=P.+A.+Coleauthor=S.+H.+Rosenbergauthor=M.+R.+Michaelidesauthor=A.+Laiauthor=K.+D.+Bromberg&title=Discovery+of+a+selective+catalytic+p300%2FCBP+inhibitor+that+targets+lineage-specific+tumours&doi=10.1038%2Fnature24028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective catalytic p300/CBP inhibitor that targets lineage-specific tumours</span></div><div class="casAuthors">Lasko, Loren M.; Jakob, Clarissa G.; Edalji, Rohinton P.; Qiu, Wei; Montgomery, Debra; Digiammarino, Enrico L.; Hansen, T. Matt; Risi, Roberto M.; Frey, Robin; Manaves, Vlasios; Shaw, Bailin; Algire, Mikkel; Hessler, Paul; Lam, Lloyd T.; Uziel, Tamar; Faivre, Emily; Ferguson, Debra; Buchanan, Fritz G.; Martin, Ruth L.; Torrent, Maricel; Chiang, Gary G.; Karukurichi, Kannan; Langston, J. William; Weinert, Brian T.; Choudhary, Chunaram; de Vries, Peter; Van Drie, John H.; McElligott, David; Kesicki, Ed; Marmorstein, Ronen; Sun, Chaohong; Cole, Philip A.; Rosenberg, Saul H.; Michaelides, Michael R.; Lai, Albert; Bromberg, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">550</span>
        (<span class="NLM_cas:issue">7674</span>),
    <span class="NLM_cas:pages">128-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The dynamic and reversible acetylation of proteins, catalyzed by histone acetyltransferases (HATs) and histone deacetylases (HDACs), is a major epigenetic regulatory mechanism of gene transcription and is assocd. with multiple diseases.  Histone deacetylase inhibitors are currently approved to treat certain cancers, but progress on the development of drug-like histone actyltransferase inhibitors has lagged behind.  The histone acetyltransferase paralogues p300 and CREB-binding protein (CBP) are key transcriptional co-activators that are essential for a multitude of cellular processes, and have also been implicated in human pathol. conditions (including cancer).  Current inhibitors of the p300 and CBP histone acetyltransferase domains, including natural products, bi-substrate analogs and the widely used small mol. C646, lack potency or selectivity.  Here, we describe A-485, a potent, selective and drug-like catalytic inhibitor of p300 and CBP.  We present a high resoln. (1.95 Å) co-crystal structure of a small mol. bound to the catalytic active site of p300 and demonstrate that A-485 competes with acetyl CoA (acetyl-CoA).  A-485 selectively inhibited proliferation in lineage-specific tumor types, including several haematol. malignancies and androgen receptor-pos. prostate cancer.  A-485 inhibited the androgen receptor transcriptional program in both androgen-sensitive and castration-resistant prostate cancer and inhibited tumor growth in a castration-resistant xenograft model.  These results demonstrate the feasibility of using small mol. inhibitors to selectively target the catalytic activity of histone acetyltransferases, which may provide effective treatments for transcriptional activator-driven malignancies and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqfy_rMa47C7Vg90H21EOLACvtfcHk0lgugtkCie6IAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsLzM&md5=050103c08a6abe2e0035573085fff6be</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature24028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24028%26sid%3Dliteratum%253Aachs%26aulast%3DLasko%26aufirst%3DL.%2BM.%26aulast%3DJakob%26aufirst%3DC.%2BG.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DMontgomery%26aufirst%3DD.%26aulast%3DDigiammarino%26aufirst%3DE.%2BL.%26aulast%3DHansen%26aufirst%3DT.%2BM.%26aulast%3DRisi%26aufirst%3DR.%2BM.%26aulast%3DFrey%26aufirst%3DR.%26aulast%3DManaves%26aufirst%3DV.%26aulast%3DShaw%26aufirst%3DB.%26aulast%3DAlgire%26aufirst%3DM.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DMartin%26aufirst%3DR.%2BL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DKarukurichi%26aufirst%3DK.%26aulast%3DLangston%26aufirst%3DJ.%2BW.%26aulast%3DWeinert%26aufirst%3DB.%2BT.%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3Dde%2BVries%26aufirst%3DP.%26aulast%3DKluge%26aufirst%3DA.%2BF.%26aulast%3DPatane%26aufirst%3DM.%2BA.%26aulast%3DVan%2BDrie%26aufirst%3DJ.%2BH.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DMcElligott%26aufirst%3DD.%26aulast%3DKesicki%26aufirst%3DE.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBromberg%26aufirst%3DK.%2BD.%26atitle%3DDiscovery%2520of%2520a%2520selective%2520catalytic%2520p300%252FCBP%2520inhibitor%2520that%2520targets%2520lineage-specific%2520tumours%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D128%26epage%3D132%26doi%3D10.1038%2Fnature24028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soccio, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ait-Si-Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolffe, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakatani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">HATs off: selective synthetic inhibitors of the histone acetyltransferases p300 and PCAF</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(00)80452-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2FS1097-2765%2800%2980452-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10882143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisVWrs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=589-595&author=O.+D.+Lauauthor=T.+K.+Kunduauthor=R.+E.+Soccioauthor=S.+Ait-Si-Aliauthor=E.+M.+Khalilauthor=A.+Vassilevauthor=A.+P.+Wolffeauthor=Y.+Nakataniauthor=R.+G.+Roederauthor=P.+A.+Cole&title=HATs+off%3A+selective+synthetic+inhibitors+of+the+histone+acetyltransferases+p300+and+PCAF&doi=10.1016%2FS1097-2765%2800%2980452-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">HATs off: selective synthetic inhibitors of the histone acetyltransferases p300 and PCAF</span></div><div class="casAuthors">Lau, Ontario D.; Kundu, Tapas K.; Soccio, Raymond E.; Ait-Si-Ali, Slimane; Khalil, Ehab M.; Vassilev, Alex; Wolffe, Alan P.; Nakatani, Yoshihiro; Roeder, Robert G.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-595</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone acetyltransferases (HATs) play important roles in the regulation of gene expression.  In this report, we describe the design, synthesis, and application of peptide CoA conjugates as selective HAT inhibitors for the transcriptional coactivators p300 and PCAF.  Two inhibitors (Lys-CoA for p300 and H3-CoA-20 for PCAF) were found to be potent (IC50 ≈ 0.5 μM) and selective (∼200-fold) in blocking p300 and PCAF HAT activities.  These inhibitors were used to probe enzymic and transcriptional features of HAT function in several assay systems.  These compds. should be broadly useful as biol. tools for evaluating the roles of HATs in transcriptional studies and may serve as lead agents for the development of novel antineoplastic therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPttQ2l59vKLVg90H21EOLACvtfcHk0ljmUCeco42M5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisVWrs7c%253D&md5=dd8242124fbb980225bdded4fd7fbd62</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2800%2980452-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252800%252980452-9%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DO.%2BD.%26aulast%3DKundu%26aufirst%3DT.%2BK.%26aulast%3DSoccio%26aufirst%3DR.%2BE.%26aulast%3DAit-Si-Ali%26aufirst%3DS.%26aulast%3DKhalil%26aufirst%3DE.%2BM.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DWolffe%26aufirst%3DA.%2BP.%26aulast%3DNakatani%26aufirst%3DY.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DHATs%2520off%253A%2520selective%2520synthetic%2520inhibitors%2520of%2520the%2520histone%2520acetyltransferases%2520p300%2520and%2520PCAF%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D5%26spage%3D589%26epage%3D595%26doi%3D10.1016%2FS1097-2765%2800%2980452-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwie, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soupaya, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maturano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blonski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lherbet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudoin-Dehoux, C.</span></span> <span> </span><span class="NLM_article-title">New potent bisubstrate inhibitors of histone acetyltransferase p300: design, synthesis and biological evaluation</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2010.01056.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1111%2Fj.1747-0285.2010.01056.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=21118378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252FitVylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=86-92&author=F.+H.+Kwieauthor=M.+Brietauthor=D.+Soupayaauthor=P.+Hoffmannauthor=M.+Maturanoauthor=F.+Rodriguezauthor=C.+Blonskiauthor=C.+Lherbetauthor=C.+Baudoin-Dehoux&title=New+potent+bisubstrate+inhibitors+of+histone+acetyltransferase+p300%3A+design%2C+synthesis+and+biological+evaluation&doi=10.1111%2Fj.1747-0285.2010.01056.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">New potent bisubstrate inhibitors of histone acetyltransferase p300: design, synthesis and biological evaluation</span></div><div class="casAuthors">Kwie Franciane Ho A; Briet Martine; Soupaya David; Hoffmann Pascal; Maturano Marie; Rodriguez Frederic; Blonski Casimir; Lherbet Christian; Baudoin-Dehoux Cecile</div><div class="citationInfo"><span class="NLM_cas:title">Chemical biology & drug design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bisubstrate-type compound Lys-CoA has been shown to inhibit the p300 histone acetyl transferase activity efficiently and may constitute a lead compound for a novel class of anticancer therapeutics.  Based on this strategy, we synthesized a series of CoA derivatives and evaluated these molecules for their activity as p300 histone acetyltransferases inhibitor.  The best activity was obtained with compound 3 bearing a C-5 spacing linker that connects the CoA moiety to a tert-butyloxycarbonyl (Boc) group.  Based on docking simulations, this inhibitor exhibits favorable interactions with two binding areas, namely pockets P1 and P2, within the active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJhwGTLVwV9kBuFdjgVNKSfW6udTcc2eaduxhv62DH5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252FitVylsg%253D%253D&md5=6f4ed0fcaab0fcf054994954e5d7ed3a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2010.01056.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2010.01056.x%26sid%3Dliteratum%253Aachs%26aulast%3DKwie%26aufirst%3DF.%2BH.%26aulast%3DBriet%26aufirst%3DM.%26aulast%3DSoupaya%26aufirst%3DD.%26aulast%3DHoffmann%26aufirst%3DP.%26aulast%3DMaturano%26aufirst%3DM.%26aulast%3DRodriguez%26aufirst%3DF.%26aulast%3DBlonski%26aufirst%3DC.%26aulast%3DLherbet%26aufirst%3DC.%26aulast%3DBaudoin-Dehoux%26aufirst%3DC.%26atitle%3DNew%2520potent%2520bisubstrate%2520inhibitors%2520of%2520histone%2520acetyltransferase%2520p300%253A%2520design%252C%2520synthesis%2520and%2520biological%2520evaluation%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2011%26volume%3D77%26spage%3D86%26epage%3D92%26doi%3D10.1111%2Fj.1747-0285.2010.01056.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holbert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazzalin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liszczak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larocca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldanha, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahadevan, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alani, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Virtual ligand screening of the p300/CBP histone acetyltransferase: identification of a selective small molecule inhibitor</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">482</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1016%2Fj.chembiol.2010.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=20534345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslCjsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=471-482&author=E.+M.+Bowersauthor=G.+Yanauthor=C.+Mukherjeeauthor=A.+Orryauthor=L.+Wangauthor=M.+A.+Holbertauthor=N.+T.+Crumpauthor=C.+A.+Hazzalinauthor=G.+Liszczakauthor=H.+Yuanauthor=C.+Laroccaauthor=S.+A.+Saldanhaauthor=R.+Abagyanauthor=Y.+Sunauthor=D.+J.+Meyersauthor=R.+Marmorsteinauthor=L.+C.+Mahadevanauthor=R.+M.+Alaniauthor=P.+A.+Cole&title=Virtual+ligand+screening+of+the+p300%2FCBP+histone+acetyltransferase%3A+identification+of+a+selective+small+molecule+inhibitor&doi=10.1016%2Fj.chembiol.2010.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Ligand Screening of the p300/CBP Histone Acetyltransferase: Identification of a Selective Small Molecule Inhibitor</span></div><div class="casAuthors">Bowers, Erin M.; Yan, Gai; Mukherjee, Chandrani; Orry, Andrew; Wang, Ling; Holbert, Marc A.; Crump, Nicholas T.; Hazzalin, Catherine A.; Liszczak, Glen; Yuan, Hua; Larocca, Cecilia; Saldanha, S. Adrian; Abagyan, Ruben; Sun, Yan; Meyers, David J.; Marmorstein, Ronen; Mahadevan, Louis C.; Alani, Rhoda M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-482</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The histone acetyltransferase (HAT) p300/CBP is a transcriptional coactivator implicated in many gene regulatory pathways and protein acetylation events.  Although p300 inhibitors have been reported, a potent, selective, and readily available active-site-directed small mol. inhibitor is not yet known.  Here the authors use a structure-based, in silico screening approach to identify a com. available pyrazolone-contg. small mol. p300 HAT inhibitor, C646.  C646 is a competitive p300 inhibitor with a Ki of 400 nM and is selective vs. other acetyltransferases.  Studies on site-directed p300 HAT mutants and synthetic modifications of C646 confirm the importance of predicted interactions in conferring potency.  Inhibition of histone acetylation and cell growth by C646 in cells validate its utility as a pharmacol. probe and suggest that p300/CBP HAT is a worthy anticancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7_p7jSyRB-bVg90H21EOLACvtfcHk0liDRbsNFsC5mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmslCjsbg%253D&md5=551fc005ecd3142c76e2b8e33d51c7e1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DE.%2BM.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DMukherjee%26aufirst%3DC.%26aulast%3DOrry%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHolbert%26aufirst%3DM.%2BA.%26aulast%3DCrump%26aufirst%3DN.%2BT.%26aulast%3DHazzalin%26aufirst%3DC.%2BA.%26aulast%3DLiszczak%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DLarocca%26aufirst%3DC.%26aulast%3DSaldanha%26aufirst%3DS.%2BA.%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DMeyers%26aufirst%3DD.%2BJ.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DMahadevan%26aufirst%3DL.%2BC.%26aulast%3DAlani%26aufirst%3DR.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DVirtual%2520ligand%2520screening%2520of%2520the%2520p300%252FCBP%2520histone%2520acetyltransferase%253A%2520identification%2520of%2520a%2520selective%2520small%2520molecule%2520inhibitor%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D471%26epage%3D482%26doi%3D10.1016%2Fj.chembiol.2010.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinello, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahn, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldanha, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planck, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geary, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roush, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, D.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of acetyltransferase p300 identified by high-throughput screening are potent anticancer agents</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-12-0930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1158%2F1535-7163.MCT-12-0930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=23625935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntlSisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=610-620&author=H.+Yangauthor=C.+E.+Pinelloauthor=J.+Luoauthor=D.+Liauthor=Y.+Wangauthor=L.+Y.+Zhaoauthor=S.+C.+Jahnauthor=S.+A.+Saldanhaauthor=P.+Chaseauthor=J.+Planckauthor=K.+R.+Gearyauthor=H.+Maauthor=B.+K.+Lawauthor=W.+R.+Roushauthor=P.+Hodderauthor=D.+Liao&title=Small-molecule+inhibitors+of+acetyltransferase+p300+identified+by+high-throughput+screening+are+potent+anticancer+agents&doi=10.1158%2F1535-7163.MCT-12-0930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of Acetyltransferase p300 Identified by High-Throughput Screening Are Potent Anticancer Agents</span></div><div class="casAuthors">Yang, Heng; Pinello, Christie E.; Luo, Jian; Li, Dawei; Wang, Yunfei; Zhao, Lisa Y.; Jahn, Stephan C.; Saldanha, Sanjay Adrian; Planck, Jamie; Geary, Kyla R.; Ma, Haiching; Law, Brian K.; Roush, William R.; Hodder, Peter; Liao, Daiqing</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">610-620</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Acetyltransferase p300 (KAT3B) plays key roles in signaling cascades that support cancer cell survival and sustained proliferation.  Thus, p300 represents a potential anticancer therapeutic target.  To discover novel anticancer agents that target p300, we conducted a high-throughput screening campaign.  A library of 622,079 compds. was assayed for cytotoxicity to the triple-neg. breast cancer (TNBC) cell line MDA-MB-231 but not to the human mammary epithelial cells.  The resulting compds. were tested in a biochem. assay for inhibiting the enzymic activity of p300.  One compd. (L002, NSC764414) displayed an IC50 of 1.98 μmol/L against p300 in vitro, inhibited acetylation of histones and p53, and suppressed STAT3 activation in cell-based assays.  L002 could be docked to the active site of the p300 catalytic domain.  Biochem. tests of a series of related compds. revealed functional groups that may impact inhibitory potency of L002 against p300.  Interestingly, these analogs showed inhibitory activities against the cellular paralog of p300 (CBP), p300/CBP-assocd. factor, and GCN5, but not to other acetyltransferases (KAT5, KAT6B, and KAT7), histone deacetylases, and histone methyltransferases.  Among the NCI-60 panel of cancer cell lines, leukemia and lymphoma cell lines were extremely sensitive to L002, whereas it is toxic to only a limited no. of cell lines derived from solid tumors.  Notably, breast cancer cell lines, esp. those derived from TNBC, were highly susceptible to L002.  In vivo, it potently suppressed tumor growth and histone acetylation of MDA-MB-468 xenografts.  Thus, these new acetyltransferase inhibitors are potential anticancer therapeutics.  Mol Cancer Ther; 12(5); 610-20. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9-EbQlof5DrVg90H21EOLACvtfcHk0liDRbsNFsC5mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntlSisrY%253D&md5=1a874cea55a65eb911134c6f3f7bebcf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-12-0930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-12-0930%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DH.%26aulast%3DPinello%26aufirst%3DC.%2BE.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DL.%2BY.%26aulast%3DJahn%26aufirst%3DS.%2BC.%26aulast%3DSaldanha%26aufirst%3DS.%2BA.%26aulast%3DChase%26aufirst%3DP.%26aulast%3DPlanck%26aufirst%3DJ.%26aulast%3DGeary%26aufirst%3DK.%2BR.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DLaw%26aufirst%3DB.%2BK.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DLiao%26aufirst%3DD.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520acetyltransferase%2520p300%2520identified%2520by%2520high-throughput%2520screening%2520are%2520potent%2520anticancer%2520agents%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2013%26volume%3D12%26spage%3D610%26epage%3D620%26doi%3D10.1158%2F1535-7163.MCT-12-0930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Santo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artico, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miele, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cereseto, A.</span></span> <span> </span><span class="NLM_article-title">Cinnamoyl compounds as simple molecules that inhibit p300 histone acetyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1973</span>– <span class="NLM_lpage">1977</span>, <span class="refDoi"> DOI: 10.1021/jm060943s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060943s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1agu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1973-1977&author=R.+Costiauthor=R.+Di+Santoauthor=M.+Articoauthor=G.+Mieleauthor=P.+Valentiniauthor=E.+Novellinoauthor=A.+Cereseto&title=Cinnamoyl+compounds+as+simple+molecules+that+inhibit+p300+histone+acetyltransferase&doi=10.1021%2Fjm060943s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Cinnamoyl Compounds as Simple Molecules that Inhibit p300 Histone Acetyltransferase</span></div><div class="casAuthors">Costi, Roberta; Di Santo, Roberto; Artico, Marino; Miele, Gaetano; Valentini, Paola; Novellino, Ettore; Cereseto, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1973-1977</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cinnamoyl compds. 1a-c and 2a-d were designed, synthesized, and in vitro tested as p300 inhibitors.  At different degrees, all tested compds. were proven to inactivate p300, particularly, deriv. 2c (I) was the most active inhibitor, also showing high specificity for p300 as compared to other histone acetyltransferases.  Most notably, 2c showed anti-acetylase activity in mammalian cells.  These compds. represent a new class of synthetic inhibitors of p300, characterized by simple chem. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoYbzB2HLUhrVg90H21EOLACvtfcHk0li22jFFsETkwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1agu78%253D&md5=04ed97d4a972182d8b62107760528db0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm060943s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060943s%26sid%3Dliteratum%253Aachs%26aulast%3DCosti%26aufirst%3DR.%26aulast%3DDi%2BSanto%26aufirst%3DR.%26aulast%3DArtico%26aufirst%3DM.%26aulast%3DMiele%26aufirst%3DG.%26aulast%3DValentini%26aufirst%3DP.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DCereseto%26aufirst%3DA.%26atitle%3DCinnamoyl%2520compounds%2520as%2520simple%2520molecules%2520that%2520inhibit%2520p300%2520histone%2520acetyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1973%26epage%3D1977%26doi%3D10.1021%2Fjm060943s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A.</span></span> <span> </span><span class="NLM_article-title">Chemistry: Chemical con artists foil drug discovery</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1038/513481a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2F513481a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=25254460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1WjsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=481-483&author=J.+Baellauthor=M.+A.+Walters&title=Chemistry%3A+Chemical+con+artists+foil+drug+discovery&doi=10.1038%2F513481a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry: Chemical con artists foil drug discovery</span></div><div class="casAuthors">Baell, Jonathan; Walters, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">7519</span>),
    <span class="NLM_cas:pages">481-483</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Naivety about promiscuous, assay-duping mols. is polluting the literature and wasting resources, warn Jonathan Baell and Michael A. Walters.  Academic drug discoverers must be more vigilant.  Mols. that show the strongest activity in screening might not be the best starting points for drugs.  PAINS hits should almost always be ignored.  Even trained medicinal chemists have to be careful until they become experienced in screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFYQ0B4VVoLVg90H21EOLACvtfcHk0li22jFFsETkwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1WjsL7M&md5=f5788d81d006460c688e4b78e6200503</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2F513481a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F513481a%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DChemistry%253A%2520Chemical%2520con%2520artists%2520foil%2520drug%2520discovery%26jtitle%3DNature%26date%3D2014%26volume%3D513%26spage%3D481%26epage%3D483%26doi%3D10.1038%2F513481a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">3-(Piperidin-4-ylmethoxy)pyridine containing cmpounds are potent inhibitors of lysine specific demethylase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=253-263&author=F.+Wuauthor=C.+Zhouauthor=Y.+Yaoauthor=L.+Weiauthor=Z.+Fengauthor=L.+Dengauthor=Y.+Song&title=3-%28Piperidin-4-ylmethoxy%29pyridine+containing+cmpounds+are+potent+inhibitors+of+lysine+specific+demethylase+1&doi=10.1021%2Facs.jmedchem.5b01361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span></div><div class="casAuthors">Wu, Fangrui; Zhou, Chao; Yao, Yuan; Wei, Liping; Feng, Zizhen; Deng, Lisheng; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-263</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation.  Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4).  LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression.  We report the design, synthesis, and structure-activity relationships of 3-(piperidin-4-ylmethoxy)pyridine contg. compds. as potent LSD1 inhibitors with Ki values as low as 29 nM.  These compds. exhibited high selectivity (>160×) against related monoamine oxidase A and B.  Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode.  The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC50 values as low as 280 nM, while they had negligible effects on normal cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKAqQdCeodaLVg90H21EOLACvtfcHk0ljkORy8iddNvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N&md5=2c74e06ed3c755826d25917d02cb845f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01361%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DY.%26atitle%3D3-%2528Piperidin-4-ylmethoxy%2529pyridine%2520containing%2520cmpounds%2520are%2520potent%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D253%26epage%3D263%26doi%3D10.1021%2Facs.jmedchem.5b01361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furst, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglesias, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germain, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gronemeyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Lera, A. R.</span></span> <span> </span><span class="NLM_article-title">Insights into the mechanism of the site-selective sequential palladium-catalyzed cross-coupling reactions of dibromothiophenes/dibromothiazoles and arylboronic acids. Synthesis of PPARbeta/delta agonists</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">4514</span>– <span class="NLM_lpage">4525</span>, <span class="refDoi"> DOI: 10.1039/B612235C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1039%2FB612235C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=17268648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1OhsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2006&pages=4514-4525&author=R.+Pereiraauthor=A.+Furstauthor=B.+Iglesiasauthor=P.+Germainauthor=H.+Gronemeyerauthor=A.+R.+de+Lera&title=Insights+into+the+mechanism+of+the+site-selective+sequential+palladium-catalyzed+cross-coupling+reactions+of+dibromothiophenes%2Fdibromothiazoles+and+arylboronic+acids.+Synthesis+of+PPARbeta%2Fdelta+agonists&doi=10.1039%2FB612235C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the mechanism of the site-selective sequential palladium-catalyzed cross-coupling reactions of dibromothiophenes/dibromothiazoles and arylboronic acids. Synthesis of PPARβ/δ agonists</span></div><div class="casAuthors">Pereira, Raquel; Furst, Audrey; Iglesias, Beatriz; Germain, Pierre; Gronemeyer, Hinrich; de Lera, Angel R.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4514-4525</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A reactivity study, aided by NMR spectroscopy, allowed a mechanistic rationale to be postulated for the palladium-catalyzed regioselective coupling of arylboronic acid (and arylstannane where feasible) at the position next to the sulfur atom in functionalized dibromothiophenes and dibromothiazoles.  The anal. of the NMR spectra (using 19F from the boronic acid CF3 group and 31P from the phosphine of the catalyst as probes) of the entire reaction starting from the dibromoheterocycles allowed the qual. proposal that the transmetalation is the rate-limiting step for both sequential substitution processes.  The extremely facile oxidative addn. at the C-Br bond next to the sulfur atom of the heterocycle instead dets. the positional selectivity.  An addnl. Stille reaction then replaced the second halogen, providing the trisubstituted heterocyclic scaffolds of PPAR ligands, which displayed PPARβ/δ agonist activity, as revealed by reporter assays in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa-yAhaXeJV7Vg90H21EOLACvtfcHk0ljkORy8iddNvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1OhsbbM&md5=b09ee42d4b0ef033817c255e6a27f7f2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FB612235C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB612235C%26sid%3Dliteratum%253Aachs%26aulast%3DPereira%26aufirst%3DR.%26aulast%3DFurst%26aufirst%3DA.%26aulast%3DIglesias%26aufirst%3DB.%26aulast%3DGermain%26aufirst%3DP.%26aulast%3DGronemeyer%26aufirst%3DH.%26aulast%3Dde%2BLera%26aufirst%3DA.%2BR.%26atitle%3DInsights%2520into%2520the%2520mechanism%2520of%2520the%2520site-selective%2520sequential%2520palladium-catalyzed%2520cross-coupling%2520reactions%2520of%2520dibromothiophenes%252Fdibromothiazoles%2520and%2520arylboronic%2520acids.%2520Synthesis%2520of%2520PPARbeta%252Fdelta%2520agonists%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2006%26volume%3D4%26spage%3D4514%26epage%3D4525%26doi%3D10.1039%2FB612235C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">The structural basis of protein acetylation by the p300/CBP transcriptional coactivator</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>451</i></span>,  <span class="NLM_fpage">846</span>– <span class="NLM_lpage">850</span>, <span class="refDoi"> DOI: 10.1038/nature06546</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fnature06546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=18273021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvVyms74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2008&pages=846-850&author=X.+Liuauthor=L.+Wangauthor=K.+Zhaoauthor=P.+R.+Thompsonauthor=Y.+Hwangauthor=R.+Marmorsteinauthor=P.+A.+Cole&title=The+structural+basis+of+protein+acetylation+by+the+p300%2FCBP+transcriptional+coactivator&doi=10.1038%2Fnature06546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis of protein acetylation by the p300/CBP transcriptional coactivator</span></div><div class="casAuthors">Liu, Xin; Wang, Ling; Zhao, Kehao; Thompson, Paul R.; Hwang, Yousang; Marmorstein, Ronen; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">7180</span>),
    <span class="NLM_cas:pages">846-850</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The transcriptional coactivator p300/CBP (CREBBP) is a histone acetyltransferase (HAT) that regulates gene expression by acetylating histones and other transcription factors.  Dysregulation of p300/CBP HAT activity contributes to various diseases including cancer.  Sequence alignments, enzymol. expts. and inhibitor studies on p300/CBP have led to contradictory results about its catalytic mechanism and its structural relation to the Gcn5/PCAF and MYST HATs.  Here we describe a high-resoln. x-ray crystal structure of a semi-synthetic heterodimeric p300 HAT domain in complex with a bi-substrate inhibitor, Lys-CoA.  This structure shows that p300/CBP is a distant cousin of other structurally characterized HATs, but reveals several novel features that explain the broad substrate specificity and preference for nearby basic residues.  Based on this structure and accompanying biochem. data, we propose that p300/CBP uses an unusual 'hit-and-run' (Theorell-Chance) catalytic mechanism that is distinct from other characterized HATs.  Several disease-assocd. mutations can also be readily accounted for by the p300 HAT structure.  These studies pave the way for new epigenetic therapies involving modulation of p300/CBP HAT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeYOui0tFND7Vg90H21EOLACvtfcHk0ljl8nMHxUQazg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvVyms74%253D&md5=c393f167458f8532f2d3b06e96f7eb00</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature06546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06546%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DHwang%26aufirst%3DY.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DThe%2520structural%2520basis%2520of%2520protein%2520acetylation%2520by%2520the%2520p300%252FCBP%2520transcriptional%2520coactivator%26jtitle%3DNature%26date%3D2008%26volume%3D451%26spage%3D846%26epage%3D850%26doi%3D10.1038%2Fnature06546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maksimoska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura-Pena, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structure of the p300 histone acetyltransferase bound to acetyl-coenzyme A and its analogues</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3415</span>– <span class="NLM_lpage">3422</span>, <span class="refDoi"> DOI: 10.1021/bi500380f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi500380f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvVSgsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=3415-3422&author=J.+Maksimoskaauthor=D.+Segura-Penaauthor=P.+A.+Coleauthor=R.+Marmorstein&title=Structure+of+the+p300+histone+acetyltransferase+bound+to+acetyl-coenzyme+A+and+its+analogues&doi=10.1021%2Fbi500380f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the p300 histone acetyltransferase bound to acetyl-coenzyme A and its analogues</span></div><div class="casAuthors">Maksimoska, Jasna; Segura-Pena, Dario; Cole, Philip A.; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3415-3422</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The p300 and CBP transcriptional coactivator paralogs (p300/CBP) regulate a variety of different cellular pathways, in part, by acetylating histones and >70 non-histone protein substrates.  Mutation, chromosomal translocation, or other aberrant activities of p300/CBP are linked to many different diseases, including cancer.  Because of its pleiotropic biol. roles and connection to disease, it is important to understand the mechanism of acetyl transfer by p300/CBP, in part so that inhibitors can be more rationally developed.  Toward this goal, a structure of p300 bound to a Lys-CoA bisubstrate histone acetyltransferase inhibitor was previously elucidated, and the enzyme's catalytic mechanism has been investigated.  Nonetheless, many questions underlying p300/CBP structure and mechanism remain.  Here, the authors report a structural characterization of different reaction states in the p300 activity cycle.  The authors present the structures of p300 in complex with an acetyl-CoA substrate, a CoA product, and an acetonyl-CoA inhibitor.  A comparison of these structures with the previously reported p300/Lys-CoA complex demonstrated that the conformation of the enzyme active site depends on the interaction of the enzyme with the cofactor, and is not apparently influenced by protein substrate Lys residue binding.  The p300/CoA crystals also contained 2 poly(ethylene glycol) moieties bound proximal to the cofactor-binding site, implicating the path of protein substrate assocn.  The structure of the p300/acetonyl-CoA complex explained the inhibitory and tight binding properties of acetonyl-CoA toward p300.  Together, these studies provide new insights into the mol. basis of acetylation by p300 and have implications for the rational development of new small-mol. p300 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAvoxOYiIJWbVg90H21EOLACvtfcHk0ljl8nMHxUQazg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvVSgsrk%253D&md5=d4b60cc12bcc790a92c91d6b9b2d3c23</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fbi500380f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi500380f%26sid%3Dliteratum%253Aachs%26aulast%3DMaksimoska%26aufirst%3DJ.%26aulast%3DSegura-Pena%26aufirst%3DD.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructure%2520of%2520the%2520p300%2520histone%2520acetyltransferase%2520bound%2520to%2520acetyl-coenzyme%2520A%2520and%2520its%2520analogues%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D3415%26epage%3D3422%26doi%3D10.1021%2Fbi500380f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demarest, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez-Yamout, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyson, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, P. E.</span></span> <span> </span><span class="NLM_article-title">Mutual synergistic folding in recruitment of CBP/p300 by p160 nuclear receptor coactivators</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>415</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1038/415549a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2F415549a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=11823864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD38Xht1Ghtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=415&publication_year=2002&pages=549-553&author=S.+J.+Demarestauthor=M.+Martinez-Yamoutauthor=J.+Chungauthor=H.+Chenauthor=W.+Xuauthor=H.+J.+Dysonauthor=R.+M.+Evansauthor=P.+E.+Wright&title=Mutual+synergistic+folding+in+recruitment+of+CBP%2Fp300+by+p160+nuclear+receptor+coactivators&doi=10.1038%2F415549a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Mutual synergistic folding in recruitment of CBP/p200 by p160 nuclear receptor coactivators</span></div><div class="casAuthors">Demarest, Stephen J.; Martinez-Yamout, Maria; Chung, John; Chen, Hongwu; Xu, Wel; Dyson, H. Jane; Evans, Ronald M.; Wright, Peter E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">415</span>
        (<span class="NLM_cas:issue">6871</span>),
    <span class="NLM_cas:pages">549-553</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nuclear hormone receptors are ligand-activated transcription factors that regulate the expression of genes that are essential for development, reprodn. and homeostasis.  The hormone response is mediated through recruitment of p160 receptor coactivators and the general transcriptional coactivator CBP/p300, which function synergistically to activate transcription.  These coactivators exhibit intrinsic histone acetyltransferase activity, function in the remodelling of chromatin, and facilitate the recruitment of RNA polymerase II and the basal transcription machinery.  The activities of the P160 coactivators are dependent on CBP.  Both coactivators are essential for proper cell-cycle control, differentiation and apoptosis, and are implicated in cancer and other diseases.  To elucidate the mol. basis of assembling the multiprotein activation complex, we undertook a structural and thermodn. anal. of the interaction domains of CBP and the activator for thyroid hormone and retinoid receptors.  Here we show that although the isolated domains are intrinsically disordered, they combine with high affinity to form a cooperatively folded helical heterodimer.  Our study uncovers a unique mechanism, called 'synergistic folding', through which p160 coactivators recruit CBP/p300 to allow transmission of the hormonal signal to the transcriptional machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd2arByyIp47Vg90H21EOLACvtfcHk0ljOyDZay9VUCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xht1Ghtb0%253D&md5=dce1ccdf207ec7e7729271b902d718e8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2F415549a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F415549a%26sid%3Dliteratum%253Aachs%26aulast%3DDemarest%26aufirst%3DS.%2BJ.%26aulast%3DMartinez-Yamout%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DDyson%26aufirst%3DH.%2BJ.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26aulast%3DWright%26aufirst%3DP.%2BE.%26atitle%3DMutual%2520synergistic%2520folding%2520in%2520recruitment%2520of%2520CBP%252Fp300%2520by%2520p160%2520nuclear%2520receptor%2520coactivators%26jtitle%3DNature%26date%3D2002%26volume%3D415%26spage%3D549%26epage%3D553%26doi%3D10.1038%2F415549a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeselsohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollingsworth, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heiser, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grasso, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellman, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, O. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsimelzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trivedi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimawi, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Terrada, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilsenbeck, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">FOXA1 overexpression mediates endocrine resistance by altering the ER transcriptome and IL-8 expression in ER-positive breast cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">E6600</span>– <span class="NLM_lpage">E6609</span>, <span class="refDoi"> DOI: 10.1073/pnas.1612835113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1073%2Fpnas.1612835113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=27791031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1aksb3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=E6600-E6609&author=X.+Fuauthor=R.+Jeselsohnauthor=R.+Pereiraauthor=E.+F.+Hollingsworthauthor=C.+J.+Creightonauthor=F.+Liauthor=M.+Sheaauthor=A.+Nardoneauthor=C.+De+Angelisauthor=L.+M.+Heiserauthor=P.+Anurauthor=N.+Wangauthor=C.+S.+Grassoauthor=P.+T.+Spellmanauthor=O.+L.+Griffithauthor=A.+Tsimelzonauthor=C.+Gutierrezauthor=S.+Huangauthor=D.+P.+Edwardsauthor=M.+V.+Trivediauthor=M.+F.+Rimawiauthor=D.+Lopez-Terradaauthor=S.+G.+Hilsenbeckauthor=J.+W.+Grayauthor=M.+Brownauthor=C.+K.+Osborneauthor=R.+Schiff&title=FOXA1+overexpression+mediates+endocrine+resistance+by+altering+the+ER+transcriptome+and+IL-8+expression+in+ER-positive+breast+cancer&doi=10.1073%2Fpnas.1612835113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">FOXA1 overexpression mediates endocrine resistance by altering the ER transcriptome and IL-8 expression in ER-positive breast cancer</span></div><div class="casAuthors">Fu, Xiaoyong; Jeselsohn, Rinath; Pereira, Resel; Hollingsworth, Emporia F.; Creighton, Chad J.; Li, Fugen; Shea, Martin; Nardone, Agostina; De Angelis, Carmine; Heiser, Laura M.; Anur, Pavana; Wang, Nicholas; Grasso, Catherine S.; Spellman, Paul T.; Griffith, Obi L.; Tsimelzon, Anna; Gutierrez, Carolina; Huang, Shixia; Edwards, Dean P.; Trivedi, Meghana V.; Rimawi, Mothaffar F.; Lopez-Terrada, Dolores; Hilsenbeck, Susan G.; Gray, Joe W.; Brown, Myles; Osborne, C. Kent; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">E6600-E6609</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Forkhead box protein A1 (FOXA1) is a pioneer factor of estrogen receptor α (ER)-chromatin binding and function, yet its aberration in endocrine-resistant (Endo-R) breast cancer is unknown.  Here, we report preclin. evidence for a role of FOXA1 in Endo-R breast cancer as well as evidence for its clin. significance.  FOXA1 is gene-amplified and/or overexpressed in Endo-R derivs. of several breast cancer cell line models.  Induced FOXA1 triggers oncogenic gene signatures and proteomic profiles highly assocd. with endocrine resistance.  Integrated omics data reveal IL8 as one of the most perturbed genes regulated by FOXA1 and ER transcriptional reprogramming in Endo-R cells.  IL-8 knockdown inhibits tamoxifen-resistant cell growth and invasion and partially attenuates the effect of overexpressed FOXA1.  Our study highlights a role of FOXA1 via IL-8 signaling as a potential therapeutic target in FOXA1-overexpressing ER-pos. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrB2OAHVe9NM7Vg90H21EOLACvtfcHk0lgOSL84Bz8Kww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1aksb3I&md5=0f64adff4b747bd30d54aa42df97e1d4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1612835113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1612835113%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DX.%26aulast%3DJeselsohn%26aufirst%3DR.%26aulast%3DPereira%26aufirst%3DR.%26aulast%3DHollingsworth%26aufirst%3DE.%2BF.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DShea%26aufirst%3DM.%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DHeiser%26aufirst%3DL.%2BM.%26aulast%3DAnur%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DGrasso%26aufirst%3DC.%2BS.%26aulast%3DSpellman%26aufirst%3DP.%2BT.%26aulast%3DGriffith%26aufirst%3DO.%2BL.%26aulast%3DTsimelzon%26aufirst%3DA.%26aulast%3DGutierrez%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DEdwards%26aufirst%3DD.%2BP.%26aulast%3DTrivedi%26aufirst%3DM.%2BV.%26aulast%3DRimawi%26aufirst%3DM.%2BF.%26aulast%3DLopez-Terrada%26aufirst%3DD.%26aulast%3DHilsenbeck%26aufirst%3DS.%2BG.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DFOXA1%2520overexpression%2520mediates%2520endocrine%2520resistance%2520by%2520altering%2520the%2520ER%2520transcriptome%2520and%2520IL-8%2520expression%2520in%2520ER-positive%2520breast%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3DE6600%26epage%3DE6609%26doi%3D10.1073%2Fpnas.1612835113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimawi, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of the dual EGFR/HER2 inhibitor AZD8931 in circumventing endocrine resistance</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1007/s10549-014-2878-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1007%2Fs10549-014-2878-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=24554387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtFSltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2014&pages=263-272&author=G.+Morrisonauthor=X.+Fuauthor=M.+Sheaauthor=S.+Nandaauthor=M.+Giulianoauthor=T.+Wangauthor=T.+Klinowskaauthor=C.+K.+Osborneauthor=M.+F.+Rimawiauthor=R.+Schiff&title=Therapeutic+potential+of+the+dual+EGFR%2FHER2+inhibitor+AZD8931+in+circumventing+endocrine+resistance&doi=10.1007%2Fs10549-014-2878-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic potential of the dual EGFR/HER2 inhibitor AZD8931 in circumventing endocrine resistance</span></div><div class="casAuthors">Morrison, Gladys; Fu, Xiaoyong; Shea, Martin; Nanda, Sarmistha; Giuliano, Mario; Wang, Tao; Klinowska, Teresa; Osborne, C. Kent; Rimawi, Mothaffar F.; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-272</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Modest up-regulation of either HER-ligands or receptors has been implicated in acquired endocrine resistance.  AZD8931, a dual tyrosine kinase inhibitor (TKI) of epithelial growth factor receptor (EGFR)/HER2, has been shown to more effectively block ligand-dependent HER signaling than the HER TKIs lapatinib or gefitinib.  We therefore examd. the effect of AZD8931 in ER-pos./HER2-neg. breast cancer cells with acquired resistance to tamoxifen, where there is ligand up-regulation assocd. with HER pathway activation.  RNA-seq ligand profiling and levels of HER receptors and signaling by western blotting were conducted in ER+ MCF7 and T47D parental cells and their Tam-resistant derivs. (TamRes).  In vitro cell growth and apoptosis and HER ligand-stimulated signaling were measured in response to endocrine and HER TKIs.  For studies in vivo, transplantable MCF7/TamRes xenografts were treated with tamoxifen or fulvestrant, either alone or in combination with AZD8931.  AZD8931 only minimally enhanced endocrine sensitivity in MCF7 parental cells, but showed a greater effect in the T47D parental model.  AZD8931 combined with either tamoxifen or fulvestrant inhibited cell growth more than lapatinib in T47D TamRes cells, and was also significantly, though modestly, more potent in MCF7 TamRes cells.  In both TamRes models, AZD8931 significantly inhibited cell proliferation and induced apoptosis.  Under ligand-stimulated conditions, AZD8931 more potently inhibited HER signaling than lapatinib or gefitinib.  AZD8931 also significantly delayed the growth of MCF7 TamRes xenografts in the presence of tamoxifen or fulvestrant.  The strongest inhibition was achieved with a fulvestrant and AZD8931 combination, though no tumor regression was obsd.  This study provides evidence that AZD8931 has greater inhibitory efficacy in tamoxifen-resistant settings than in an endocrine therapy naive setting.  The absence of tumor regression, however, suggests that addnl. escape pathways contribute to resistant growth and will need to be targeted to fully circumvent tamoxifen resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwFAxc7BoAB7Vg90H21EOLACvtfcHk0lhPX1M1pK_JMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtFSltLc%253D&md5=92d9536d6c1bd9363239eb2ad873a630</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs10549-014-2878-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-014-2878-x%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DG.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DShea%26aufirst%3DM.%26aulast%3DNanda%26aufirst%3DS.%26aulast%3DGiuliano%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DRimawi%26aufirst%3DM.%2BF.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DTherapeutic%2520potential%2520of%2520the%2520dual%2520EGFR%252FHER2%2520inhibitor%2520AZD8931%2520in%2520circumventing%2520endocrine%2520resistance%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2014%26volume%3D144%26spage%3D263%26epage%3D272%26doi%3D10.1007%2Fs10549-014-2878-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, H.</span></span> <span> </span><span class="NLM_article-title">P300 inhibition enhances gemcitabine-induced apoptosis of pancreatic cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">51301</span>– <span class="NLM_lpage">51310</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.10117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.18632%2Foncotarget.10117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=27322077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A280%3ADC%252BC2s7hs1Ciug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=51301-51310&author=H.+Onoauthor=M.+D.+Bassonauthor=H.+Ito&title=P300+inhibition+enhances+gemcitabine-induced+apoptosis+of+pancreatic+cancer&doi=10.18632%2Foncotarget.10117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">P300 inhibition enhances gemcitabine-induced apoptosis of pancreatic cancer</span></div><div class="casAuthors">Ono Hiroaki; Ito Hiromichi; Basson Marc D</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">51301-51310</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The transcriptional cofactor p300 has histone acetyltransferase activity (HAT) and has been reported to participate in chromatin remodeling and DNA repair.  We hypothesized that targeting p300 can enhance the cytotoxicity of gemcitabine, which induces pancreatic cancer cell apoptosis by damaging DNA.  Expression of p300 was confirmed in pancreatic cancer cell lines and human pancreatic adenocarcinoma tissues by western blotting and immunohistochemistry.  When pancreatic cancer cells were treated with gemcitabine, p300 was recruited to chromatin within 24 hours, indicating the role in response to DNA damage.  When p300 was gene-silenced with siRNA, histone acetylation was substantially reduced and pancreatic cancer cells were sensitized to gemcitabine.  The selective p300 HAT inhibitor C646 similarly decreased histone acetylation, increased gemcitabine-induced apoptosis and thus enhanced the cytotoxicity of gemcitabine on pancreatic cancer cells.  These findings indicate that p300 contributes to chemo-resistance of pancreatic cancer against gemcitabine and suggest that p300 and its HAT activity may be a potential therapeutic target to improve outcomes in patients with pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQHSrWrD2HP4Xl8icjszYl0fW6udTcc2eaKGfHPSUJDybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7hs1Ciug%253D%253D&md5=467ff424e8cfcf8d35c89230de540d14</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.10117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.10117%26sid%3Dliteratum%253Aachs%26aulast%3DOno%26aufirst%3DH.%26aulast%3DBasson%26aufirst%3DM.%2BD.%26aulast%3DIto%26aufirst%3DH.%26atitle%3DP300%2520inhibition%2520enhances%2520gemcitabine-induced%2520apoptosis%2520of%2520pancreatic%2520cancer%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D51301%26epage%3D51310%26doi%3D10.18632%2Foncotarget.10117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gural, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perna, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatlen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deblasio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menendez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voza, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimer, S. D.</span></span> <span> </span><span class="NLM_article-title">The leukemogenicity of AML1-ETO is dependent on site-specific lysine acetylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>333</i></span>,  <span class="NLM_fpage">765</span>– <span class="NLM_lpage">769</span>, <span class="refDoi"> DOI: 10.1126/science.1201662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1126%2Fscience.1201662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=21764752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1SgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=333&publication_year=2011&pages=765-769&author=L.+Wangauthor=A.+Guralauthor=X.+J.+Sunauthor=X.+Zhaoauthor=F.+Pernaauthor=G.+Huangauthor=M.+A.+Hatlenauthor=L.+Vuauthor=F.+Liuauthor=H.+Xuauthor=T.+Asaiauthor=H.+Xuauthor=T.+Deblasioauthor=S.+Menendezauthor=F.+Vozaauthor=Y.+Jiangauthor=P.+A.+Coleauthor=J.+Zhangauthor=A.+Melnickauthor=R.+G.+Roederauthor=S.+D.+Nimer&title=The+leukemogenicity+of+AML1-ETO+is+dependent+on+site-specific+lysine+acetylation&doi=10.1126%2Fscience.1201662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The leukemogenicity of AML1-ETO is dependent on site-specific lysine acetylation</span></div><div class="casAuthors">Wang, Lan; Gural, Alexander; Sun, Xiao-Jian; Zhao, Xinyang; Perna, Fabiana; Huang, Gang; Hatlen, Megan A.; Vu, Ly; Liu, Fan; Xu, Haiming; Asai, Takashi; Xu, Hao; Deblasio, Tony; Menendez, Silvia; Voza, Francesca; Jiang, Yanwen; Cole, Philip A.; Zhang, Jinsong; Melnick, Ari; Roeder, Robert G.; Nimer, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">333</span>
        (<span class="NLM_cas:issue">6043</span>),
    <span class="NLM_cas:pages">765-769</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The chromosomal translocations found in acute myelogenous leukemia (AML) generate oncogenic fusion transcription factors with aberrant transcriptional regulatory properties.  Although therapeutic targeting of most leukemia fusion proteins remains elusive, the post-translational modifications that control their function could be targetable.  We found that AML1-ETO, the fusion protein generated by the t(8;21) translocation, is acetylated by the transcriptional coactivator p300 in leukemia cells isolated from t(8;21) AML patients, and that this acetylation is essential for its self-renewal-promoting effects in human cord blood CD34+ cells and its leukemogenicity in mouse models.  Inhibition of p300 abrogates the acetylation of AML1-ETO and impairs its ability to promote leukemic transformation.  Thus, lysine acetyltransferases represent a potential therapeutic target in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsv3PTdm2bWbVg90H21EOLACvtfcHk0lgm4HF_SUgrwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1SgtLc%253D&md5=b77a02c2653eef8faff013aef7356c5a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.1201662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1201662%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DGural%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DX.%2BJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DPerna%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DHatlen%26aufirst%3DM.%2BA.%26aulast%3DVu%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DAsai%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DDeblasio%26aufirst%3DT.%26aulast%3DMenendez%26aufirst%3DS.%26aulast%3DVoza%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26atitle%3DThe%2520leukemogenicity%2520of%2520AML1-ETO%2520is%2520dependent%2520on%2520site-specific%2520lysine%2520acetylation%26jtitle%3DScience%26date%3D2011%26volume%3D333%26spage%3D765%26epage%3D769%26doi%3D10.1126%2Fscience.1201662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mootha, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulovich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomeroy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span> <span> </span><span class="NLM_article-title">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">15545</span>– <span class="NLM_lpage">15550</span>, <span class="refDoi"> DOI: 10.1073/pnas.0506580102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1073%2Fpnas.0506580102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=16199517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ShtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&author=A.+Subramanianauthor=P.+Tamayoauthor=V.+K.+Moothaauthor=S.+Mukherjeeauthor=B.+L.+Ebertauthor=M.+A.+Gilletteauthor=A.+Paulovichauthor=S.+L.+Pomeroyauthor=T.+R.+Golubauthor=E.+S.+Landerauthor=J.+P.+Mesirov&title=Gene+set+enrichment+analysis%3A+A+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles&doi=10.1073%2Fpnas.0506580102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span></div><div class="casAuthors">Subramanian, Aravind; Tamayo, Pablo; Mootha, Vamsi K.; Mukherjee, Sayan; Ebert, Benjamin L.; Gillette, Michael A.; Paulovich, Amanda; Pomeroy, Scott L.; Golub, Todd R.; Lander, Eric S.; Mesirov, Jill P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15545-15550</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Although genomewide RNA expression anal. has become a routine tool in biomedical research, extg. biol. insight from such information remains a major challenge.  Here, we describe a powerful anal. method called Gene Set Enrichment Anal. (GSEA) for interpreting gene expression data.  The method derives its power by focusing on gene sets, i.e., groups of genes that share common biol. function, chromosomal location, or regulation.  We demonstrate how GSEA yields insights into several cancer-related data sets, including leukemia and lung cancer.  Notably, where single-gene anal. finds little similarity between two independent studies of patients survival in lung cancer, GSEA reveals many biol. pathways in common.  The GSEA method is embodied in a freely available software package, together with an initial database of 1,325 biol. defined gene sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH2_E6gWd0yrVg90H21EOLACvtfcHk0lhzNfjDlK1-lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ShtrnO&md5=ca2eb221010f20379199e6442c65fc2e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0506580102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0506580102%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DTamayo%26aufirst%3DP.%26aulast%3DMootha%26aufirst%3DV.%2BK.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DGillette%26aufirst%3DM.%2BA.%26aulast%3DPaulovich%26aufirst%3DA.%26aulast%3DPomeroy%26aufirst%3DS.%2BL.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26atitle%3DGene%2520set%2520enrichment%2520analysis%253A%2520A%2520knowledge-based%2520approach%2520for%2520interpreting%2520genome-wide%2520expression%2520profiles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D15545%26epage%3D15550%26doi%3D10.1073%2Fpnas.0506580102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ianculescu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegmund, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stallcup, M. R.</span></span> <span> </span><span class="NLM_article-title">Selective roles for cAMP response element-binding protein binding protein and p300 protein as coregulators for androgen-regulated gene expression in advanced prostate cancer cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">4000</span>– <span class="NLM_lpage">4013</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.300194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1074%2Fjbc.M111.300194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=22174411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=4000-4013&author=I.+Ianculescuauthor=D.+Y.+Wuauthor=K.+D.+Siegmundauthor=M.+R.+Stallcup&title=Selective+roles+for+cAMP+response+element-binding+protein+binding+protein+and+p300+protein+as+coregulators+for+androgen-regulated+gene+expression+in+advanced+prostate+cancer+cells&doi=10.1074%2Fjbc.M111.300194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Roles for cAMP Response Element-binding Protein Binding Protein and p300 Protein as Coregulators for Androgen-regulated Gene Expression in Advanced Prostate Cancer Cells</span></div><div class="casAuthors">Ianculescu, Irina; Wu, Dai-Ying; Siegmund, Kimberly D.; Stallcup, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4000-4013</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein acetyltransferases p300 and cAMP response element-binding protein binding protein (CBP) are homologous, ubiquitously expressed proteins that interact with hundreds of proteins involved in transcriptional regulation and are involved globally as transcriptional coregulators.  Although these two proteins acetylate and interact with overlapping sets of proteins, we found that p300 and CBP contribute to androgen-induced regulation of distinct sets of genes in C4-2B prostate cancer cells, a model of advanced prostate cancer.  CBP cannot compensate for the loss of p300 to support androgen-induced expression of many genes, such as TMPRSS2 and PSA.  Global gene expression anal. indicated that 47% of androgen-regulated genes are p300-dependent in these cells, whereas, surprisingly, only 0.3% of them are CBP-dependent.  Chromatin immunopptn. anal. after depletion of cellular p300 indicated that p300 is required for androgen-induced acetylation of histones H3 and H4, methylation of histone H3 at Lys-4, and recruitment of TATA box binding protein (TBP) and RNA polymerase II, but not recruitment of the androgen receptor, on the TMPRSS2 gene in response to androgen.  Thus, p300 is the dominant coregulator of the CBP/p300 pair for androgen-regulated gene expression in C4-2B cells. p300 is required at an early stage of chromatin remodeling and transcription complex assembly after binding of androgen receptor to the gene but before many crit. histone modifications occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7uJfPL3B4h7Vg90H21EOLACvtfcHk0lhzNfjDlK1-lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSjtLg%253D&md5=095ab7582da1b7602e2ab898caa5b094</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.300194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.300194%26sid%3Dliteratum%253Aachs%26aulast%3DIanculescu%26aufirst%3DI.%26aulast%3DWu%26aufirst%3DD.%2BY.%26aulast%3DSiegmund%26aufirst%3DK.%2BD.%26aulast%3DStallcup%26aufirst%3DM.%2BR.%26atitle%3DSelective%2520roles%2520for%2520cAMP%2520response%2520element-binding%2520protein%2520binding%2520protein%2520and%2520p300%2520protein%2520as%2520coregulators%2520for%2520androgen-regulated%2520gene%2520expression%2520in%2520advanced%2520prostate%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D4000%26epage%3D4013%26doi%3D10.1074%2Fjbc.M111.300194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasper, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindle, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dent, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span> <span> </span><span class="NLM_article-title">Distinct roles of GCN5/PCAF-mediated H3K9ac and CBP/p300-mediated H3K18/27ac in nuclear receptor transactivation</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/emboj.2010.318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Femboj.2010.318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=21131905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFSrsr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=249-262&author=Q.+Jinauthor=L.+R.+Yuauthor=L.+Wangauthor=Z.+Zhangauthor=L.+H.+Kasperauthor=J.+E.+Leeauthor=C.+Wangauthor=P.+K.+Brindleauthor=S.+Y.+Dentauthor=K.+Ge&title=Distinct+roles+of+GCN5%2FPCAF-mediated+H3K9ac+and+CBP%2Fp300-mediated+H3K18%2F27ac+in+nuclear+receptor+transactivation&doi=10.1038%2Femboj.2010.318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct roles of GCN5/PCAF-mediated H3K9ac and CBP/p300-mediated H3K18/27ac in nuclear receptor transactivation</span></div><div class="casAuthors">Jin, Qihuang; Yu, Li-Rong; Wang, Lifeng; Zhang, Zhijing; Kasper, Lawryn H.; Lee, Ji-Eun; Wang, Chaochen; Brindle, Paul K.; Dent, Sharon Y. R.; Ge, Kai</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-262</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone acetyltransferases (HATs) GCN5 and PCAF (GCN5/PCAF) and CBP and p300 (CBP/p300) are transcription co-activators.  However, how these two distinct families of HATs regulate gene activation remains unclear.  Here, we show deletion of GCN5/PCAF in cells specifically and dramatically reduces acetylation on histone H3K9 (H3K9ac) while deletion of CBP/p300 specifically and dramatically reduces acetylations on H3K18 and H3K27 (H3K18/27ac).  A ligand (GW501516) for nuclear receptor (NR) PPARδ induces sequential enrichment of H3K18/27ac, RNA polymerase II (Pol II) and H3K9ac on PPARδ target gene Angptl4 promoter, which correlates with a robust Angptl4 expression.  Inhibiting transcription elongation blocks ligand-induced H3K9ac, but not H3K18/27ac, on the Angptl4 promoter.  Finally, we show GCN5/PCAF and GCN5/PCAF-mediated H3K9ac correlate with, but are surprisingly dispensable for, NR target gene activation.  In contrast, CBP/p300 and their HAT activities are essential for ligand-induced Pol II recruitment on, and activation of, NR target genes.  These results highlight the substrate and site specificities of HATs in cells, demonstrate the distinct roles of GCN5/PCAF- and CBP/p300-mediated histone acetylations in gene activation, and suggest an important role of CBP/p300-mediated H3K18/27ac in NR-dependent transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIm-siX9LrK7Vg90H21EOLACvtfcHk0lgaQTgNvs8D1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFSrsr3E&md5=bdb41206845730e0875b8a73e70b0030</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.318%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DYu%26aufirst%3DL.%2BR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKasper%26aufirst%3DL.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BE.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DBrindle%26aufirst%3DP.%2BK.%26aulast%3DDent%26aufirst%3DS.%2BY.%26aulast%3DGe%26aufirst%3DK.%26atitle%3DDistinct%2520roles%2520of%2520GCN5%252FPCAF-mediated%2520H3K9ac%2520and%2520CBP%252Fp300-mediated%2520H3K18%252F27ac%2520in%2520nuclear%2520receptor%2520transactivation%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D249%26epage%3D262%26doi%3D10.1038%2Femboj.2010.318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anglin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, B. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Crystallographic investigation and selective inhibition of mutant isocitrate dehydrogenase</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">546</span>, <span class="refDoi"> DOI: 10.1021/ml400036z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400036z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVWrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=542-546&author=B.+Zhengauthor=Y.+Yaoauthor=Z.+Liuauthor=L.+Dengauthor=J.+L.+Anglinauthor=H.+Jiangauthor=B.+V.+Prasadauthor=Y.+Song&title=Crystallographic+investigation+and+selective+inhibition+of+mutant+isocitrate+dehydrogenase&doi=10.1021%2Fml400036z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic Investigation and Selective Inhibition of Mutant Isocitrate Dehydrogenase</span></div><div class="casAuthors">Zheng, Baisong; Yao, Yuan; Liu, Zhen; Deng, Lisheng; Anglin, Justin L.; Jiang, Hong; Prasad, B. V. Venkataram; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">542-546</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mutations in isocitrate dehydrogenase (IDH), a key enzyme in the tricarboxylic acid cycle, have recently been found in ∼75% glioma and ∼20% acute myeloid leukemia.  Different from the wild-type enzyme, mutant IDH1 catalyzes the redn. of α-ketoglutaric acid to D-2-hydroxyglutaric acid.  Strong evidence has shown mutant IDH1 represents a novel target for this type of cancer.  We found two 1-hydroxypyridin-2-one compds. that are potent inhibitors of R132H and R132C IDH1 mutants with Ki values as low as 120 nM.  These compds. exhibit >60-fold selectivity against wild-type IDH1 and can inhibit the prodn. of D-2-hydroxyglutaric acid in IDH1 mutated cells, representing novel chem. probes for cancer biol. studies.  We also report the first inhibitor-bound crystal structures of IDH1(R132H), showing these inhibitors have H-bond, electrostatic, and hydrophobic interactions with the mutant enzyme.  Comparison with the substrate-bound IDH1 structures revealed the structural basis for the high enzyme selectivity of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH0vzVLGCPZLVg90H21EOLACvtfcHk0lgaQTgNvs8D1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVWrtLg%253D&md5=c34c67949fe816bdf151525a578535f3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fml400036z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400036z%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DCrystallographic%2520investigation%2520and%2520selective%2520inhibition%2520of%2520mutant%2520isocitrate%2520dehydrogenase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D542%26epage%3D546%26doi%3D10.1021%2Fml400036z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogiso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Inhibition of cancer-associated mutant isocitrate dehydrogenases by 2-thiohydantoin compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6899</span>– <span class="NLM_lpage">6908</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlequ7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6899-6908&author=F.+Wuauthor=H.+Jiangauthor=B.+Zhengauthor=M.+Kogisoauthor=Y.+Yaoauthor=C.+Zhouauthor=X.+N.+Liauthor=Y.+Song&title=Inhibition+of+cancer-associated+mutant+isocitrate+dehydrogenases+by+2-thiohydantoin+compounds&doi=10.1021%2Facs.jmedchem.5b00684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Cancer-Associated Mutant Isocitrate Dehydrogenases by 2-Thiohydantoin Compounds</span></div><div class="casAuthors">Wu, Fangrui; Jiang, Hong; Zheng, Baisong; Kogiso, Mari; Yao, Yuan; Zhou, Chao; Li, Xiao-Nan; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6899-6908</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Somatic mutations of isocitrate dehydrogenase 1 (IDH1) at R132 are frequently found in certain cancers such as glioma.  With losing the activity of wild-type IDH1, the R132H and R132C mutant proteins can reduce α-ketoglutaric acid (α-KG) to D-2-hydroxyglutaric acid (D2HG).  The resulting high concn. of D2HG inhibits many α-KG-dependent dioxygenases, including histone demethylases, to cause broad histone hypermethylation.  These aberrant epigenetic changes are responsible for the initiation of these cancers.  The authors report the synthesis, structure-activity relationships, enzyme kinetics, and binding thermodn. of a novel series of 2-thiohydantoin and related compds. I [R1 = H, Me, Ph, etc.; R2 = 3,4-(HO)2C6H3, pyridin-3-yl, 2-pyridinone-5-yl, etc.], among which several compds. are potent inhibitors of mutant IDH1 with Ki as low as 420 nM.  X-ray crystal structures of IDH1(R132H) in complex with two inhibitors are reported, showing their inhibitor-protein interactions.  These compds. can decrease the cellular concn. of D2HG, reduce the levels of histone methylation, and suppress the proliferation of stem-like cancer cells in BT142 glioma with IDH1 R132H mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHlhXpk4PcnbVg90H21EOLACvtfcHk0lhqfWk0oCS17Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlequ7nI&md5=57dd47bcd29b65244b33e564910419de</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00684%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DKogiso%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DX.%2BN.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520cancer-associated%2520mutant%2520isocitrate%2520dehydrogenases%2520by%25202-thiohydantoin%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6899%26epage%3D6908%26doi%3D10.1021%2Facs.jmedchem.5b00684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of LSD1 for the treatment of MLL-rearranged leukemia</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">24</span>, <span class="refDoi"> DOI: 10.1186/s13045-016-0252-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1186%2Fs13045-016-0252-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=26970896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1eitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=24&author=Z.+Fengauthor=Y.+Yaoauthor=C.+Zhouauthor=F.+Chenauthor=F.+Wuauthor=L.+Weiauthor=W.+Liuauthor=S.+Dongauthor=M.+Redellauthor=Q.+Moauthor=Y.+Song&title=Pharmacological+inhibition+of+LSD1+for+the+treatment+of+MLL-rearranged+leukemia&doi=10.1186%2Fs13045-016-0252-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of LSD1 for the treatment of MLL-rearranged leukemia</span></div><div class="casAuthors">Feng, Zizhen; Yao, Yuan; Zhou, Chao; Chen, Fengju; Wu, Fangrui; Wei, Liping; Liu, Wei; Dong, Shuo; Redell, Michele; Mo, Qianxing; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hematology & Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24/1-24/13</span>CODEN:
                <span class="NLM_cas:coden">JHOOAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-8722</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Mixed lineage leukemia (MLL) gene translocations are found in ∼75 % infant and 10 % adult acute leukemia, showing a poor prognosis.  Lysine-specific demethylase 1 (LSD1) has recently been implicated to be a drug target for this subtype of leukemia.  More studies using potent LSD1 inhibitors against MLL-rearranged leukemia are needed.  Methods: LSD1 inhibitors were examd. for their biochem. and biol. activities against LSD1 and MLL-rearranged leukemia as well as other cancer cells.  Results: Potent LSD1 inhibitors with biochem. IC50 values of 9.8-77 nM were found to strongly inhibit proliferation of MLL-rearranged leukemia cells with EC50 of 10-320 nM, while these compds. are generally non-cytotoxic to several other tumor cells.  LSD1 inhibition increased histone H3 lysine 4 (H3K4) methylation, downregulated expression of several leukemia-relevant genes, induced apoptosis and differentiation, and inhibited self-renewal of stem-like leukemia cells.  Moreover, LSD1 inhibitors worked synergistically with inhibition of DOT1L, a histone H3 lysine 79 (H3K79) methyltransferase, against MLL-rearranged leukemia.  The most potent LSD1 inhibitor showed significant in vivo activity in a systemic mouse model of MLL-rearranged leukemia without overt toxicities.  Mechanistically, LSD1 inhibitors caused significant upregulation of several pathways that promote hematopoietic differentiation and apoptosis.  Conclusions: LSD1 is a drug target for MLL-rearranged leukemia, and LSD1 inhibitors are potential therapeutics for the malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUF2TPVvRgIbVg90H21EOLACvtfcHk0lhqfWk0oCS17Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1eitr8%253D&md5=3ffec94f558982ca319649ebc8708286</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2Fs13045-016-0252-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-016-0252-7%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DRedell%26aufirst%3DM.%26aulast%3DMo%26aufirst%3DQ.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DPharmacological%2520inhibition%2520of%2520LSD1%2520for%2520the%2520treatment%2520of%2520MLL-rearranged%2520leukemia%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2016%26volume%3D9%26spage%3D24%26doi%3D10.1186%2Fs13045-016-0252-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Inhibition of histone H3K79 methylation selectively inhibits proliferation, self-renewal and metastatic potential of breast cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10665</span>– <span class="NLM_lpage">10677</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2496</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.18632%2Foncotarget.2496" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=25359765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A280%3ADC%252BC2M3kslylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=10665-10677&author=L.+Zhangauthor=L.+Dengauthor=F.+Chenauthor=Y.+Yaoauthor=B.+Wuauthor=L.+Weiauthor=Q.+Moauthor=Y.+Song&title=Inhibition+of+histone+H3K79+methylation+selectively+inhibits+proliferation%2C+self-renewal+and+metastatic+potential+of+breast+cancer&doi=10.18632%2Foncotarget.2496"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of histone H3K79 methylation selectively inhibits proliferation, self-renewal and metastatic potential of breast cancer</span></div><div class="casAuthors">Zhang Li; Deng Lisheng; Yao Yuan; Wu Bulan; Wei Liping; Chen Fengju; Mo Qianxing; Song Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">10665-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone lysine methylation regulates gene expression and cancer initiation.  Bioinformatics analysis suggested that DOT1L, a histone H3-lysine79 (H3K79) methyltransferase, plays a potentially important role in breast cancer.  DOT1L inhibition selectively inhibited proliferation, self-renewal, metastatic potential of breast cancer cells and induced cell differentiation.  In addition, inhibitors of S-adenosylhomocysteine hydrolase (SAHH), such as neplanocin and 3-deazaneplanocin, also inhibited both H3K79 methylation and proliferation of breast cancer cells in vitro and in vivo.  The activity of SAHH inhibitors was previously attributed to inhibition of H3K27 methyltransferase EZH2.  However, inhibition of EZH2 by a specific inhibitor did not contribute to cell death.  SAHH inhibitors had only weak activity against H3K27 methylation and their activity is therefore mainly due to DOT1L/H3K79 methylation inhibition.  Overall, we showed that DOT1L is a potential drug target for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcToXKVEBdgQcLHHuJhbyicZfW6udTcc2easiRYNLRVQ1Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3kslylsg%253D%253D&md5=c495f4521165f0fb79399b80d8181955</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2496&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2496%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DMo%26aufirst%3DQ.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520histone%2520H3K79%2520methylation%2520selectively%2520inhibits%2520proliferation%252C%2520self-renewal%2520and%2520metastatic%2520potential%2520of%2520breast%2520cancer%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D10665%26epage%3D10677%26doi%3D10.18632%2Foncotarget.2496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M.</span></span> <span> </span><span class="NLM_article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. <i>EMBnet.journal</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.14806/ej.17.1.200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.14806%2Fej.17.1.200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=3&author=M.+Martin&title=Cutadapt+removes+adapter+sequences+from+high-throughput+sequencing+reads&doi=10.14806%2Fej.17.1.200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.14806%2Fej.17.1.200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14806%252Fej.17.1.200%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%26atitle%3DCutadapt%2520removes%2520adapter%2520sequences%2520from%2520high-throughput%2520sequencing%2520reads%26jtitle%3DEMBnet.journal%26date%3D2011%26volume%3D17%26spage%3D3%26doi%3D10.14806%2Fej.17.1.200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S.</span></span> FastQC. A quality control tool for high throughput sequence data. <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="extLink">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a> (accessed March 27, <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Andrews%2C+S.+FastQC.+A+quality+control+tool+for+high+throughput+sequence+data.+http%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F+%28accessed+March+27%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DS.%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pertea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pertea, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leek, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzberg, S. L.</span></span> <span> </span><span class="NLM_article-title">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1650</span>– <span class="NLM_lpage">1667</span>, <span class="refDoi"> DOI: 10.1038/nprot.2016.095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1038%2Fnprot.2016.095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=27560171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCgsbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1650-1667&author=M.+Perteaauthor=D.+Kimauthor=G.+M.+Perteaauthor=J.+T.+Leekauthor=S.+L.+Salzberg&title=Transcript-level+expression+analysis+of+RNA-seq+experiments+with+HISAT%2C+StringTie+and+Ballgown&doi=10.1038%2Fnprot.2016.095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown</span></div><div class="casAuthors">Pertea, Mihaela; Kim, Daehwan; Pertea, Geo M.; Leek, Jeffrey T.; Salzberg, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1650-1667</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput sequencing of mRNA (RNA-seq) has become the std. method for measuring and comparing the levels of gene expression in a wide variety of species and conditions.  RNA-seq expts. generate very large, complex data sets that demand fast, accurate and flexible software to reduce the raw read data to comprehensible results.  HISAT (hierarchical indexing for spliced alignment of transcripts), StringTie and Ballgown are free, open-source software tools for comprehensive anal. of RNA-seq expts.  Together, they allow scientists to align reads to a genome, assemble transcripts including novel splice variants, compute the abundance of these transcripts in each sample and compare expts. to identify differentially expressed genes and transcripts.  This protocol describes all the steps necessary to process a large set of raw sequencing reads and create lists of gene transcripts, expression levels, and differentially expressed genes and transcripts.  The protocol's execution time depends on the computing resources, but it typically takes under 45 min of computer time.  HISAT, StringTie and Ballgown are available from http://ccb.jhu.edu/software.shtml.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnBTDxxNF_yrVg90H21EOLACvtfcHk0lgw-sgxNzgn0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCgsbzI&md5=17da34fc59b6d109b96a5e0068d61555</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.095%26sid%3Dliteratum%253Aachs%26aulast%3DPertea%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DPertea%26aufirst%3DG.%2BM.%26aulast%3DLeek%26aufirst%3DJ.%2BT.%26aulast%3DSalzberg%26aufirst%3DS.%2BL.%26atitle%3DTranscript-level%2520expression%2520analysis%2520of%2520RNA-seq%2520experiments%2520with%2520HISAT%252C%2520StringTie%2520and%2520Ballgown%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1650%26epage%3D1667%26doi%3D10.1038%2Fnprot.2016.095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handsaker, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wysoker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fennell, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marth, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecasis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, R.</span></span> <span> </span><span class="NLM_article-title">The sequence alignment/map format and SAMtools</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2078</span>– <span class="NLM_lpage">2079</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btp352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=10.1093%2Fbioinformatics%2Fbtp352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=19505943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpslertr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2078-2079&author=H.+Liauthor=B.+Handsakerauthor=A.+Wysokerauthor=T.+Fennellauthor=J.+Ruanauthor=N.+Homerauthor=G.+Marthauthor=G.+Abecasisauthor=R.+Durbin&title=The+sequence+alignment%2Fmap+format+and+SAMtools&doi=10.1093%2Fbioinformatics%2Fbtp352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Sequence Alignment/Map format and SAMtools</span></div><div class="casAuthors">Li, Heng; Handsaker, Bob; Wysoker, Alec; Fennell, Tim; Ruan, Jue; Homer, Nils; Marth, Gabor; Abecasis, Goncalo; Durbin, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2078-2079</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: The Sequence Alignment/Map (SAM) format is a generic alignment format for storing read alignments against ref. sequences, supporting short and long reads (up to 128 Mbp) produced by different sequencing platforms.  It is flexible in style, compact in size, efficient in random access and is the format in which alignments from the 1000 Genomes Project are released.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments.  Availability: http://samtools.sourceforge.net Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="65170125040b0200174b04064b100e4b">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-IkvoI5O3jLVg90H21EOLACvtfcHk0liD3rbg0asb0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpslertr8%253D&md5=1ab7714968487a35cce7f81b751a0b1a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp352%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DHandsaker%26aufirst%3DB.%26aulast%3DWysoker%26aufirst%3DA.%26aulast%3DFennell%26aufirst%3DT.%26aulast%3DRuan%26aufirst%3DJ.%26aulast%3DHomer%26aufirst%3DN.%26aulast%3DMarth%26aufirst%3DG.%26aulast%3DAbecasis%26aufirst%3DG.%26aulast%3DDurbin%26aufirst%3DR.%26atitle%3DThe%2520sequence%2520alignment%252Fmap%2520format%2520and%2520SAMtools%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3D2078%26epage%3D2079%26doi%3D10.1093%2Fbioinformatics%2Fbtp352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PZS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PZS','PDB','4PZS'); return false;">PDB: 4PZS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i72"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02164">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_24212"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b02164?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02164</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Charts S1–S3, Figures S1–S4, representative HPLC traces, and <sup>1</sup>H and <sup>13</sup>C NMR spectra for compound <b>12</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_002.csv">CSV</a>)</p></li><li><p class="inline">PDB files for docking models (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_003.zip">ZIP</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_001.pdf">jm9b02164_si_001.pdf (1.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_002.csv">jm9b02164_si_002.csv (3.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02164/suppl_file/jm9b02164_si_003.zip">jm9b02164_si_003.zip (1.12 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b02164&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-9%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b02164%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b02164" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799aaa988953c46","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
