
<html lang="en"     class="pb-page"  data-request-id="fb08e936-d8c9-4660-b01f-9c76053f05ef"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.1c00127;issue:issue:10.1021/jmcmar.2021.64.issue-8;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Bispecific Estrogen Receptor α Degraders Incorporating Novel Binders Identified Using DNA-Encoded Chemical Library Screening" /></meta><meta name="dc.Creator" content="Jeremy S.  Disch" /></meta><meta name="dc.Creator" content="Jennifer M.  Duffy" /></meta><meta name="dc.Creator" content="Esther C. Y.  Lee" /></meta><meta name="dc.Creator" content="Diana  Gikunju" /></meta><meta name="dc.Creator" content="Betty  Chan" /></meta><meta name="dc.Creator" content="Benjamin  Levin" /></meta><meta name="dc.Creator" content="Michael I.  Monteiro" /></meta><meta name="dc.Creator" content="Sarah A.  Talcott" /></meta><meta name="dc.Creator" content="Anthony C.  Lau" /></meta><meta name="dc.Creator" content="Fei  Zhou" /></meta><meta name="dc.Creator" content="Anton  Kozhushnyan" /></meta><meta name="dc.Creator" content="Neil E.  Westlund" /></meta><meta name="dc.Creator" content="Patrick B.  Mullins" /></meta><meta name="dc.Creator" content="Yan  Yu" /></meta><meta name="dc.Creator" content="Moritz  von Rechenberg" /></meta><meta name="dc.Creator" content="Junyi  Zhang" /></meta><meta name="dc.Creator" content="Yelena A.  Arnautova" /></meta><meta name="dc.Creator" content="Yanbin  Liu" /></meta><meta name="dc.Creator" content="Ying  Zhang" /></meta><meta name="dc.Creator" content="Andrew J.  McRiner" /></meta><meta name="dc.Creator" content="Anthony D.  Keefe" /></meta><meta name="dc.Creator" content="Anna  Kohlmann" /></meta><meta name="dc.Creator" content="Matthew A.  Clark" /></meta><meta name="dc.Creator" content="John W.  Cuozzo" /></meta><meta name="dc.Creator" content="Christelle  Huguet" /></meta><meta name="dc.Creator" content="Shilpi  Arora" /></meta><meta name="dc.Description" content="Bispecific degraders (PROTACs) of ERα are expected to be advantageous over current inhibitors of ERα signaling (aromatase inhibitors/SERMs/SERDs) used to treat ER+ breast cancer. Information from D..." /></meta><meta name="Description" content="Bispecific degraders (PROTACs) of ERα are expected to be advantageous over current inhibitors of ERα signaling (aromatase inhibitors/SERMs/SERDs) used to treat ER+ breast cancer. Information from D..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 12, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00127" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00127" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00127" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00127" /></link>
        
    
    

<title>Bispecific Estrogen Receptor α Degraders Incorporating Novel Binders Identified Using DNA-Encoded Chemical Library Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00127" /></meta><meta property="og:title" content="Bispecific Estrogen Receptor α Degraders Incorporating Novel Binders Identified Using DNA-Encoded Chemical Library Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0011.jpeg" /></meta><meta property="og:description" content="Bispecific degraders (PROTACs) of ERα are expected to be advantageous over current inhibitors of ERα signaling (aromatase inhibitors/SERMs/SERDs) used to treat ER+ breast cancer. Information from DNA-encoded chemical library (DECL) screening provides a method to identify novel PROTAC binding features as the linker positioning, and binding elements are determined directly from the screen. After screening ∼120 billion DNA-encoded molecules with ERα WT and 3 gain-of-function (GOF) mutants, with and without estradiol to identify features that enrich ERα competitively, the off-DNA synthesized small molecule exemplar 7 exhibited nanomolar ERα binding, antagonism, and degradation. Click chemistry synthesis  on an alkyne E3 ligase engagers panel and an azide variant of 7 rapidly generated bispecific nanomolar degraders of ERα, with PROTACs 18 and 21 inhibiting ER+ MCF7 tumor growth in a mouse xenograft model of breast cancer. This study validates this approach toward identifying novel bispecific degrader leads from DECL screening with minimal optimization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00127"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00127">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00127&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00127&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00127&amp;href=/doi/10.1021/acs.jmedchem.1c00127" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 5049-5066</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00124" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00128" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Bispecific Estrogen Receptor α Degraders Incorporating Novel Binders Identified Using DNA-Encoded Chemical Library Screening</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jeremy S. Disch</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeremy S. Disch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#96fcf2ffe5f5feafa4d6f1fbf7fffab8f5f9fb"><span class="__cf_email__" data-cfemail="5339373a20303b6a6113343e323a3f7d303c3e">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeremy+S.++Disch">Jeremy S. Disch</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0357-1672" title="Orcid link">http://orcid.org/0000-0002-0357-1672</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer M. Duffy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer M. Duffy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+M.++Duffy">Jennifer M. Duffy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Esther C. Y. Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Esther C. Y. Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Esther+C.+Y.++Lee">Esther C. Y. Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Diana Gikunju</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Diana Gikunju</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Diana++Gikunju">Diana Gikunju</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Betty Chan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Betty Chan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Betty++Chan">Betty Chan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Benjamin Levin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin Levin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin++Levin">Benjamin Levin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael I. Monteiro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael I. Monteiro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+I.++Monteiro">Michael I. Monteiro</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sarah A. Talcott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sarah A. Talcott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sarah+A.++Talcott">Sarah A. Talcott</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anthony C. Lau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anthony C. Lau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anthony+C.++Lau">Anthony C. Lau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fei Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fei Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fei++Zhou">Fei Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anton Kozhushnyan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anton Kozhushnyan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anton++Kozhushnyan">Anton Kozhushnyan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Neil E. Westlund</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Neil E. Westlund</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Neil+E.++Westlund">Neil E. Westlund</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Patrick B. Mullins</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Patrick B. Mullins</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Patrick+B.++Mullins">Patrick B. Mullins</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yan Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Yu">Yan Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Moritz von Rechenberg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Moritz von Rechenberg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Moritz++von+Rechenberg">Moritz von Rechenberg</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Junyi Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junyi Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junyi++Zhang">Junyi Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yelena A. Arnautova</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yelena A. Arnautova</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yelena+A.++Arnautova">Yelena A. Arnautova</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanbin Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanbin Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanbin++Liu">Yanbin Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ying Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Zhang">Ying Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew J. McRiner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew J. McRiner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+J.++McRiner">Andrew J. McRiner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anthony D. Keefe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anthony D. Keefe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anthony+D.++Keefe">Anthony D. Keefe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anna Kohlmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anna Kohlmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anna++Kohlmann">Anna Kohlmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew A. Clark</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew A. Clark</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+A.++Clark">Matthew A. Clark</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John W. Cuozzo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John W. Cuozzo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+W.++Cuozzo">John W. Cuozzo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6229-0395" title="Orcid link">http://orcid.org/0000-0002-6229-0395</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christelle Huguet</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christelle Huguet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christelle++Huguet">Christelle Huguet</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shilpi Arora</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shilpi Arora</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c8bbbba7a6a1a9baa7baa988afa5a9a1a4e6aba7a5"><span class="__cf_email__" data-cfemail="fe8d8d9190979f8c918c9fbe99939f9792d09d9193">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shilpi++Arora">Shilpi Arora</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00127&amp;href=/doi/10.1021%2Facs.jmedchem.1c00127" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 5049–5066</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 12, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 January 2021</li><li><span class="item_label"><b>Published</b> online</span>12 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00127" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00127</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5049%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJeremy%2BS.%2BDisch%252C%2BJennifer%2BM.%2BDuffy%252C%2BEsther%2BC.%2BY.%2BLee%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D8%26contentID%3Dacs.jmedchem.1c00127%26title%3DBispecific%2BEstrogen%2BReceptor%2B%25CE%25B1%2BDegraders%2BIncorporating%2BNovel%2BBinders%2BIdentified%2BUsing%2BDNA-Encoded%2BChemical%2BLibrary%2BScreening%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5066%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00127"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2302</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00127" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Bispecific Estrogen Receptor α Degraders Incorporating Novel Binders Identified Using DNA-Encoded Chemical Library Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jeremy&quot;,&quot;last_name&quot;:&quot;S. Disch&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;M. Duffy&quot;},{&quot;first_name&quot;:&quot;Esther&quot;,&quot;last_name&quot;:&quot;C. Y. Lee&quot;},{&quot;first_name&quot;:&quot;Diana&quot;,&quot;last_name&quot;:&quot;Gikunju&quot;},{&quot;first_name&quot;:&quot;Betty&quot;,&quot;last_name&quot;:&quot;Chan&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;Levin&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;I. Monteiro&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;A. Talcott&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;C. Lau&quot;},{&quot;first_name&quot;:&quot;Fei&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Anton&quot;,&quot;last_name&quot;:&quot;Kozhushnyan&quot;},{&quot;first_name&quot;:&quot;Neil&quot;,&quot;last_name&quot;:&quot;E. Westlund&quot;},{&quot;first_name&quot;:&quot;Patrick&quot;,&quot;last_name&quot;:&quot;B. Mullins&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Moritz&quot;,&quot;last_name&quot;:&quot;von Rechenberg&quot;},{&quot;first_name&quot;:&quot;Junyi&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yelena&quot;,&quot;last_name&quot;:&quot;A. Arnautova&quot;},{&quot;first_name&quot;:&quot;Yanbin&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;J. McRiner&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;D. Keefe&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;Kohlmann&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;A. Clark&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;W. Cuozzo&quot;},{&quot;first_name&quot;:&quot;Christelle&quot;,&quot;last_name&quot;:&quot;Huguet&quot;},{&quot;first_name&quot;:&quot;Shilpi&quot;,&quot;last_name&quot;:&quot;Arora&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;5049-5066&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00127&quot;},&quot;abstract&quot;:&quot;Bispecific degraders (PROTACs) of ERα are expected to be advantageous over current inhibitors of ERα signaling (aromatase inhibitors/SERMs/SERDs) used to treat ER+ breast cancer. Information from DNA-encoded chemical library (DECL) screening provides a method to identify novel PROTAC binding features as the linker positioning, and binding elements are determined directly from the screen. After screening ∼120 billion DNA-encoded molecules with ERα WT and 3 gain-of-function (GOF) mutants, with and without estradiol to identify features that enrich ERα competitively, the off-DNA synthesized small molecule exemplar 7 exhibited nanomolar ERα binding, antagonism, and degradation. Click chemistry synthesis on an alkyne E3 ligase engagers panel and an azide variant of 7 rapidly generated bispecific nanomolar degraders of ERα, with PROTACs 18 and 21 inhibiting ER+ MCF7 tumor growth in a mouse xenograft model of breast cancer. This study validates this approach toward identifying novel bispecific degrader leads from DE&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00127&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00127" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00127&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00127" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00127&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00127" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00127&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00127&amp;href=/doi/10.1021/acs.jmedchem.1c00127" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00127" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00127" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00127%26sid%3Dliteratum%253Aachs%26pmid%3D33844532%26genre%3Darticle%26aulast%3DDisch%26date%3D2021%26atitle%3DBispecific%2BEstrogen%2BReceptor%2B%25CE%25B1%2BDegraders%2BIncorporating%2BNovel%2BBinders%2BIdentified%2BUsing%2BDNA-Encoded%2BChemical%2BLibrary%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D8%26spage%3D5049%26epage%3D5066%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292038" title="Azides">Azides</a>,</li><li><a href="/action/doSearch?ConceptID=290658" title="Amides">Amides</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/jmcmar.2021.64.issue-8/20210422/jmcmar.2021.64.issue-8.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Bispecific degraders (PROTACs) of ERα are expected to be advantageous over current inhibitors of ERα signaling (aromatase inhibitors/SERMs/SERDs) used to treat ER+ breast cancer. Information from DNA-encoded chemical library (DECL) screening provides a method to identify novel PROTAC binding features as the linker positioning, and binding elements are determined directly from the screen. After screening ∼120 billion DNA-encoded molecules with ERα WT and 3 gain-of-function (GOF) mutants, with and without estradiol to identify features that enrich ERα competitively, the off-DNA synthesized small molecule exemplar <b>7</b> exhibited nanomolar ERα binding, antagonism, and degradation. Click chemistry synthesis  on an alkyne E3 ligase engagers panel and an azide variant of <b>7</b> rapidly generated bispecific nanomolar degraders of ERα, with PROTACs <b>18</b> and <b>21</b> inhibiting ER+ MCF7 tumor growth in a mouse xenograft model of breast cancer. This study validates this approach toward identifying novel bispecific degrader leads from DECL screening with minimal optimization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15851" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15851" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Breast cancer is the most common cancer in women, with an annual worldwide incidence of  ∼2 million  and accounting for 15% of all cancer-related deaths in women.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Estrogen receptor (ER) is a ligand-dependent nuclear hormone receptor, which, upon activation, acts as a transcription factor in normal breast cells. ERα, one of the two main subtypes of estrogen receptor, is over-expressed in ∼70% of all breast cancers. ERα signaling is responsible for cancer cell proliferation, survival, and metastasis in breast cancer.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Approved endocrine therapies include aromatase inhibitors (AIs) such as letrozole, selective ER modulators (SERMs) such as tamoxifen, and selective ER degrader (SERDs) such as Fulvestrant. These therapies have allowed for effective management of ER+ breast cancer.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Fulvestrant’s approval as a first line therapy for metastatic ER+ breast cancer suggests that degradation of estrogen receptor is a valid and useful strategy for treating breast cancer.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Poor pharmacokinetic properties of Fulvestrant have inspired the development of second generation SERDs, both steroidal and non-steroidal, that are currently in clinical trials or under development. These include compounds in Phase 2/3 clinical trials, Elacestrant (Radius), RG6171 (Roche), SAR439839 (Sanofi), and AZD9833 (AstraZeneca), and others in Phase 1 from Eli Lilly, Novartis, G1 Therapeutics, and Pfizer.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7">(4−7)</a> For currently approved therapeutics, despite initial efficacy, the development of acquired resistance limits their utility. Gain-of-function (GOF) mutations in ERα are enriched in one-third of endocrine therapy-resistant recurrences, leading to ligand-independent activation of the ERα pathway,<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> thereby conferring partial resistance to existing classes of endocrine therapies. These acquired mutations are associated with aggressive disease biology and reduced overall survival.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The most common among these GOF mutations result in altering the amino acids Y537 and D538, generating a constitutively active ER.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> These GOF mutants are resistant to estrogen deprivation and are less responsive to tamoxifen or Fulvestrant. Thus, there is a need to develop a next generation of ERα-targeted therapeutics that can address aberrant activities of both wild-type and GOF mutants of ERα.</div><div class="NLM_p">Recently, bispecific degraders (PROTACs, proteolysis targeting chimeras) have emerged as a novel therapeutic strategy in drug discovery and development and the first PROTAC compounds have recently entered into clinical trials.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> These bispecific degraders utilize two active domains attached with a linker, allowing for the protein of interest (POI) to be held in close proximity to a targeted E3 ligase, thus targeting the former for degradation through the ubiquitin proteasome system.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> Most ERα-targeting bifunctional degraders are based on well-known binders to ERα, such as raloxifene, tamoxifene, lasofoxifene, or estradiol, among others.<a onclick="showRef(event, 'ref4 ref18 ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref4 ref18 ref19 ref20 ref21 ref22 ref23">(4,18−23)</a> Though an immensely promising technology, the field acknowledges issues with optimization of these molecules, such as linker presentation, which may be a limiting factor in optimal design. However, one of the biggest challenges is identifying  novel POI and ubiquitin ligase binders.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The necessity for a tether may be best approached using screening methodologies that utilize tethered molecules. Thus, an approach like DNA-encoded chemical libraries (DECL) screening may be the best option for identifying such novel ligands.</div><div class="NLM_p">Herein, we report the discovery of a series of novel ERα bispecific degraders (PROTACs) that target both WT and mutant ERα, by utilizing DECL screening to discover novel ERα binders and antagonists, and quickly using this information to develop novel POI engagers for generating bifunctional degraders. DECL platforms have emerged as an attractive approach to affinity screen billions of compounds in a single selection experiment.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This platform  uses split and pool combinatorial chemistry to synthesize very large, thousands to billions, member combinatorial libraries, in which a unique DNA sequence is covalently tethered to each small molecule. The DNA sequence is used to record the library, building block, and reaction information during the course of library synthesis. DECL technology has generated several clinical candidates<a onclick="showRef(event, 'ref26 ref27 ref200'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref200">(2627−28)</a> and, most recently, has been extended to screens in live cells<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(29)</a> and used to train machine learning models.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(30)</a></div><div class="NLM_p">In a typical DECL screening campaign (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a),  DNA-encoded libraries are incubated with a protein, affinity captured, and washed to enrich the library pool. The final selection output is amplified, sequenced, and translated to identify the structures of the tethered small molecules. DECL selections are frequently multiplexed, allowing for comparative evaluation of enrichment to define a mechanism of action. For instance, by screening homologs or including a known competitor, substrate, or cofactor during the selection process, one can focus on a particular binding profile, mutant, or species specificity.<a onclick="showRef(event, 'ref25 ref28'); return false;" href="javascript:void(0);" class="ref ref25 ref28">(25,29)</a> Once the structures are translated from the DNA tag sequences, structures are filtered by removing promiscuous and  matrix binders. After evaluating the enrichment profile across the multiplexed selection conditions, hits are synthesized off-DNA to confirm binding in assays.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic for DECL  screening and PROTAC array platforms. (a) DNA-encoded chemical library (combinatorial synthesized small molecules tethered to DNA tags) is incubated with ERα, affinity captured, washed to remove non-binding library, and then enriched library members are eluted. After repeating an additional round, the DNA tags are sequenced to identify the small molecule binders. (b) Azide variants of the binders are prepared and reacted in a parallel click chemistry library of E3 binder alkynes to obtain an array of PROTACs for ERα.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A key advantage of the DECL platform is that a substantially large collection of novel tethered molecules is being presented to the target in their linked presentation, a key requirement for the development of PROTACs. After identification of a valid binder to a protein of interest, the DECL information can be directly leveraged to produce PROTACs very efficiently by preparing an azide linked variant with the DNA tag replaced by an azide and then cross-reacted with a series of E3 binder alkynes utilizing click chemistry<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(31)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b). Ultimately, this would lead to sets of PROTACs, with novel POI binders identified by large combinatorial space to progress into biochemistry, cellular biology, ADME, and efficacy studies.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85548" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85548" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Affinity-Mediated Selection of DNA-Encoded Chemical Libraries</h3><div class="NLM_p last">To identify novel ERα binders, we conducted a DNA chemical library affinity screen utilizing X-Chem’s ∼120-billion-member library collection. A collection of 44 distinct DNA-encoded chemical libraries were combined and incubated with no target, the ligand binding domains (LBD) of ERα WT 7.7 μM to 24 nM, ERα WT at 7.7 μM with estradiol at 15 μM, and each of three mutant forms of ERα, D538G, S463P, and Y537S at 7.7 μM. Supplementary <a href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf" class="ext-link">Figure S1</a> shows a denaturing PAGE of all the ERα LBD variants used in the affinity-mediated selection with the DNA-encoded chemical libraries. Proteins and associated library members were captured using an IMAC matrix followed by stringent washing and elution at 85 °C. The output of the first round of selection was used as the input for the second round of selection with fresh protein and competitor reagents, and the output of the second round of selection was PCR-amplified and submitted for Illumina sequencing. A total of 528 million single-end reads were generated across all 44 libraries, and all selection conditions averaged 1.2 million reads per library per selection condition.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Selection Output Data Analysis and Hit ID</h3><div class="NLM_p">Individual sequence reads were translated back into the corresponding building block, and library schemes and statistical prevalence data were calculated for all building block combinations in each library across all selection conditions. A large number of strongly enriched building block combinations were observed across all libraries. During the process of deciding which compounds to synthesize off-DNA for hit-confirmation, a range of parameters was considered including the extent of enrichment, the profile of enrichment across the different protein variants, whether competition was observed with estradiol, how the enrichment extent varied across the dilution series, and the extent to which “SAR” was observed for co-enriching building blocks that manifest themselves in structurally related compounds. Enrichment profiles were calculated for each building block combination and the extent to which individual building block combinations co-enriched. Data are presented in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a,b for two individual building block pairs (disynthons) that were resynthesized off-DNA, determined to be active, and then used as the founder members or hits to establish the series. The relative enrichment of the two representative building block pairs can be seen across the selection conditions with each showing significant enrichment at 7.7 and 1.6 μM ERα, competition with estradiol, and also significant enrichment against the Y537S and S463P mutants with lesser enrichment against the D538G mutant. ENRv1 represents a statistical metric of enrichment significance for each disynthon (negative log10 of the asymptotic significance value).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(32)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. DECL selection output and off-DNA activity confirmation of two related members. (a, b) Schematic of two structurally related 2-cycle library members and their enrichment (ENRv1) suggests they bind ERα WT and mutants Y537S, S463P, and D538G. These library members do not bind the matrix (NTC; no target control) and are competed off by the presence of Estradiol (E2), suggesting that these library members bind in the ligand binding site. (c) Analogous “off-DNA” synthesized exemplars, with the linker and DNA truncated to a single methyl, bind in fluorescence polarization binding displacement assay. Assay results are an average of duplicate runs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Hit Confirmation and SAR Follow-Up</h3><div class="NLM_p">To confirm the selection output, direct representative off-DNA compounds were synthesized, and their biochemical and cellular activities were evaluated. Two of the most potent off-DNA representative compounds (<b>1</b> and <b>2</b>) identified in the screen are shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c. The potency of these ERα binders was assessed in an <i>in vitro</i> biochemical FP (fluorescence polarization) assay that was established for ERα WT and three of the most commonly occurring mutants, S463P, Y537S, and D538G. Compounds identified to be active in the biochemical assay were then tested in a dose titration in the ER+ cell line, MCF7, in a proliferation assay in both antagonist and agonist mode.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(33)</a> The preferred phenotype in the cell-based assay is an anti-proliferative effect on ER-dependent cells in antagonist mode (+E2), with no increase in growth of cells  seen in hormone-deprived media (agonist mode).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(33)</a> In addition to the proliferation assay, PROTACs were also tested in an ERα degradation assay in MCF7 cells  (in-cell western assay) to confirm the degradation of ERα in cells. ERα levels were assessed in dose titration upon 24 or 48 h exposure to the respective compounds.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(33)</a></div><div class="NLM_p">The chlorophenol compounds (<b>1</b> and <b>2</b>) were potent binders of WT and mutant (S463P, Y537S, and D538G) ERα LBDs with <b>2</b> having 2- to 5-fold higher potency than <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c), and both had modest cellular agonist activity (0.46–0.55 μM) in MCF7 cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). With the guidance of the DECL screening selection data and molecular docking, we designed, prepared, and evaluated the <i>in vitro</i> small molecule SAR with an emphasis on potency and antagonism. The DNA-encoded library information indicated that the likely solvent accessible exit vector was at the tertiary amine that provides the connection between the encoded compounds and the linker to DNA. However, docking also suggested other poses with the fluorophenyl ring projecting toward solvent, with the phenol also buried deep in the pocket. We sought an approach to build SAR by maintaining potency while modulating the cellular activity from agonism to antagonism by appending a polar amine group off these vectors to interact with Asp351, which is known to introduce antagonism (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Initially replacing the methyl with phenyl at R<sub>1</sub> gave compound <b>3</b>, which maintained binding activity (15 nM); however, this change resulted in producing a more potent cellular agonist (<b>3</b>, GI<sub>50</sub> < 5 nM). Progressively longer alkyl-piperidine chains <b>4–7</b> correlated with improvements in potency, with <b>7</b> having the desired characteristics of notably potent antagonism, no observed cellular agonism, and interestingly, degradation of ERα as a small molecule in an in-cell western assay (DC<sub>50</sub> = 15 nM; <i>D</i><sub>max</sub> ≈ 70%) in starved CSS media conditions. Curiously, the other attachment points appeared to generate a significantly weaker binder (<b>8</b>) or potent agonist behavior (<b>9</b>), indicating that there was sensitivity to the linker location. We decided to further progress compound <b>7</b> forward as a moiety in a bispecific PROTAC format.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Off-DNA Small Molecule SAR<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0008.gif" alt="" id="gr7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0009.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The structure–activity relationships for small molecules without DNA barcodes in ERα binding FP assay, MCF7 agonism and antagonism viability, and ERα degradation (DC<sub>50</sub>). ND = not determined. Results in the table represent an average of duplicate runs.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Development of Potent ERα Binders into PROTACs</h3><div class="NLM_p">To determine if our internal PROTAC strategy was feasible, we started with a known ERα binder, bazedoxifene,<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(34,35)</a> to rapidly identify which E3 ligands could be utilized to degrade ERα protein. To concurrently explore the effect of different E3 targeting ligands and linker lengths for attaching the ERα binding moiety, we utilized an exploration strategy that capitalizes on Huisgen 1,3-dipolar cycloaddition reaction. Commonly referred to as “click chemistry,” the cycloaddition forms a triazole ring by reacting an azide and an alkyne moiety under relatively mild conditions (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). A toolbox of E3 ligase binders with varying linker lengths to alkynes<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(31)</a> enabled the parallel synthesis of PROTACs with the azide of a POI binder in a facile manner. Our toolbox included alkynes linked to cereblon, VHL, and IAP E3 ligase binders with one, two, or three PEG units between the E3 binder and the azide moiety (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. PROTACs Utilizing a Bazedoxifene or Compound <b>7</b> Moeity as the POI Binder<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0010.gif" alt="" id="gr9" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">PROTAC made from:</th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">ERα FP IC<sub>50</sub> (nM)</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">MCF7 GI<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" align="center">MCF7 DC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">ERα binder azide</th><th class="colsep0 rowsep0" align="center">E3 binder alkyne</th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center" char=".">WT</th><th class="colsep0 rowsep0" align="center">S463P</th><th class="colsep0 rowsep0" align="center">Y537S</th><th class="colsep0 rowsep0" align="center">D538G</th><th class="colsep0 rowsep0" align="center" char=".">Ag.</th><th class="colsep0 rowsep0" align="center" char=".">Ant.</th><th class="colsep0 rowsep0" align="center">degradation</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">Baz azide</td><td class="colsep0 rowsep0" align="left">Thalidomide</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">3.2</td><td class="colsep0 rowsep0" align="left">4.7</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">3.25</td><td class="colsep0 rowsep0" align="left">2.5<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">Baz azide</td><td class="colsep0 rowsep0" align="left">VHL amide</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">8.1</td><td class="colsep0 rowsep0" align="left">287</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.247</td><td class="colsep0 rowsep0" align="left">0.020<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">Baz azide</td><td class="colsep0 rowsep0" align="left">VHL amide</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.335</td><td class="colsep0 rowsep0" align="left">0.020<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">Baz azide</td><td class="colsep0 rowsep0" align="left">VHL amide</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">9.3</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.211</td><td class="colsep0 rowsep0" align="left">0.009<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">Baz azide</td><td class="colsep0 rowsep0" align="left">VHL phenoxy</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.034</td><td class="colsep0 rowsep0" align="left">0.027<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">Baz azide</td><td class="colsep0 rowsep0" align="left">IAP phenoxy</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">61</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.051</td><td class="colsep0 rowsep0" align="left">0.174<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">Baz azide</td><td class="colsep0 rowsep0" align="left">IAP amide</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="left">>10<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">7 azide</td><td class="colsep0 rowsep0" align="left">VHL amide</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="left">170</td><td class="colsep0 rowsep0" align="left">332</td><td class="colsep0 rowsep0" align="left">269</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.165</td><td class="colsep0 rowsep0" align="left">0.0042<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">7 azide</td><td class="colsep0 rowsep0" align="left">VHL amide</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">53</td><td class="colsep0 rowsep0" align="left">203</td><td class="colsep0 rowsep0" align="left">419</td><td class="colsep0 rowsep0" align="left">517</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.030</td><td class="colsep0 rowsep0" align="left">0.006<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">7 azide</td><td class="colsep0 rowsep0" align="left">VHL amide</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="left">238</td><td class="colsep0 rowsep0" align="left">534</td><td class="colsep0 rowsep0" align="left">332</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.261</td><td class="colsep0 rowsep0" align="left">0.008<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">7 azide</td><td class="colsep0 rowsep0" align="left">VHL phenoxy</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">198</td><td class="colsep0 rowsep0" align="left">1129</td><td class="colsep0 rowsep0" align="left">3720</td><td class="colsep0 rowsep0" align="left">2205</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.423</td><td class="colsep0 rowsep0" align="left">0.038<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">7 azide</td><td class="colsep0 rowsep0" align="left">VHL phenoxy</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="left">143</td><td class="colsep0 rowsep0" align="left">527</td><td class="colsep0 rowsep0" align="left">329</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.165</td><td class="colsep0 rowsep0" align="left">0.037<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">7 azide</td><td class="colsep0 rowsep0" align="left">VHL phenoxy</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">44</td><td class="colsep0 rowsep0" align="left">134</td><td class="colsep0 rowsep0" align="left">302</td><td class="colsep0 rowsep0" align="left">261</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">0.131</td><td class="colsep0 rowsep0" align="left">0.0053<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">7 azide</td><td class="colsep0 rowsep0" align="left">VHL amide epimer</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="left">122</td><td class="colsep0 rowsep0" align="left">341</td><td class="colsep0 rowsep0" align="left">199</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">1.75</td><td class="colsep0 rowsep0" align="left">1.78<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a>, >10<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a><sup>,</sup><a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Original assay.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">48 h.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">24 h.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">6 h.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">16 h.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last">Individual compounds from the PROTAC array design were prepared utilizing click chemistry to obtain PROTACs <b>10</b>−<b>23</b> with variable ERα and E3 binding elements and variable linker lengths (<i>n</i>). The compounds were assessed in FP binding assays for WT and mutant ERα, MCF7 viability assay in agonist or antagonist mode, and in MCF7 in cell western ERα degradation. ND = not determined. Results in the table represent an average of duplicate runs.</p></div></div></div><div class="NLM_p">From the literature, it is known that the indole portion of bazedoxifene binds to the ligand-binding domain of the estrogen receptor and thus the azepane points toward the solvent.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(34)</a> Therefore, an azide version of bazedoxifene was prepared to react with the corresponding E3 binder alkynes to produce PROTACs based on thalidomide (<b>10</b>); VHL amide (<b>11–13</b>), VHL-phenoxy (<b>14</b>), IAP phenoxy (<b>15</b>), and IAP amide (<b>16</b>). These compounds were tested in a binding assay as well as the previously described cellular degradation and viability assays. All seven compounds were potent in the binding assay (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, IC<sub>50</sub> range 1.6–52 nM). However, only the VHL amide (<b>11–13</b>), VHL phenoxy (<b>14</b>), and IAP phenoxy (<b>15</b>)-based PROTACs had sub-micromolar cellular degradation. These same compounds were also confirmed to be antagonists in the cellular viability assay  with GI<sub>50</sub> values ranging from 34 to 335 nM (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), demonstrating that VHL-based PROTACs with bazedoxifene generate potent degraders (<b>11–14</b>).</div><div class="NLM_p last">Based on the success of utilizing a VHL engager in the proof of concept bazedoxifene series, we decided to focus on the VHL ligands for designing PROTACs with our structurally novel series. Click chemistry on an azide intermediate of <b>7</b>, where an azide replaced the piperidine in compound <b>7</b>, was used to prepare an array with three linker length alkynes of PROTACs based on VHL amide (<b>17</b>–<b>19</b>) and VHL-phenoxy (<b>20</b>–<b>22</b>). The binding affinity of the VHL amide PROTACs <b>17</b>–<b>19</b> seemed  less potent with increasing linker length (2- to 5-fold). However, the cellular degradation potency values (DC<sub>50</sub>) for all three PROTACs were single-digit nM. In the VHL phenoxy series, the binding affinities of <b>21</b> and <b>22</b> were similar (IC<sub>50</sub> = 32 and 44 nM), whereas the shortest linker <b>20</b> was less potent (IC<sub>50</sub> = 198 nM). Unpredictably, the cellular ERα degradation of <b>20</b> and <b>21</b> was the same (38 and 37 nM), and the compound with the longest linker <b>22</b> was more potent and with similar potency to the VHL amides at 5.3 nM (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). These compounds were also tested in the corresponding binding assay with mutants D538G, S463P, and Y537S and showed pan inhibition with binding within 20-fold of WT. All compounds were shown to be antagonists in the cell viability assay (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Understanding the MOA and Kinetics of ERα Inhibition and Degradation in MCF7 Cells</h3><div class="NLM_p">Two of the most potent PROTACs from the two VHL variants (<b>18</b> and <b>21</b>) were utilized to understand the mechanism of action (MOA) of ERα inhibition and degradation (structures shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). Biochemically, both compounds were very potent in the WT and mutant ERα FP binding assays (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In-cell western-based degradation assay in MCF7 cells showed dose-dependent reduction in the levels of ERα at 24 h, with both compounds tested, <b>18</b> (DC<sub>50</sub>: 4.5 nM) and <b>21</b> (DC<sub>50</sub>: 20 nM; <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c and Supplementary <a href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf" class="ext-link">Figure S2</a>). These graphs represent normalized ERα levels. We did not assess <i>D</i><sub>max</sub> as a parameter in this study (due to the use of a PROTAC control rather than an absolute control in the assay), but upon comparing the two compounds, we saw at least 20% more degradation of ERα with <b>21</b> than <b>18</b>, demonstrating a more robust degradation for <b>21</b>. Next, the cellular viability assay in MCF7 cells demonstrated negligible agonist effects, while both compounds demonstrated robust antagonistic activity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). As ERα is a transcription factor, we next tested its immediate downstream effect on the gene expression of its target progesterone receptor (PGR). The results demonstrated a dose-dependent reduction in the levels of PGR upon treatment with <b>18</b> (EC<sub>50</sub> 27 nM) and <b>21</b> (EC<sub>50</sub> 90 nM) at 24 h (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e). The graphs represent normalized PGR levels.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Biochemical, cellular, and mouse PK properties of ERα PROTACs, <b>18</b> and <b>21</b>. (a) Structures of <b>18</b> and <b>21</b>. (b) Biochemical inhibition using purified recombinant ERα WT and mutants D538G, Y537Y, and S463P. (c) Degradation of ERα in MCF7 cells treated for 24 h followed by in-cell western analysis of ERα protein normalized to tubulin. The graphs represent normalized ERα levels. (d) MCF7 viability assay with cells treated for 7 days followed by an end-point viability by CellTiter-Glo. (e) Downstream PGR expression analysis in MCF7 cells treated for 24 h followed by RNA extraction, first strand synthesis, and qPCR analysis for progesterone receptor RNA estimation, normalized to TBP (Tata-binding protein); graphs represent normalized PGR levels. (f) Mean plasma concentration (μM) for 10 mpk (mg/kg) subcutaneous dosing in CD1 mouse (<i>n</i> = 3). Curves shown are from a single run with technical replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to assess the kinetics of PROTAC-mediated ERα degradation, we performed time-course experiments followed by western blotting to determine optimal ERα degradation with PROTACs in both hormone-deprived media (agonist mode) and full-serum media (antagonist mode). MCF7 cells were treated with 1 μM Fulvestrant or <b>18</b> over a time course of 48 h. Results demonstrated degradation of ERα with Fulvestrant 4 h after treatment, but complete degradation with Fulvestrant under these conditions was not observed (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). In contrast, strong, near-complete degradation was seen as early as 2 h with PROTAC-based degrader <b>18</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a) (data for agonist mode not shown but was similar to antagonist mode). Next, to understand the durability of ERα degradation, we performed a wash-out experiment<i>.</i> Upon compound wash-out, we observed a 10-fold loss in degradation efficacy with <b>18</b>, but <b>21</b> demonstrated  a 25-fold reduction  (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). In order to confirm that the observed degradation is VHL-mediated, we performed a competition experiment utilizing the E3 binder, VHL-298 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>e).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(36)</a> Cells co-treated with VHL-298 (5 μM) alongside ERα PROTACs (<b>18</b> or <b>21</b>) observed a DC<sub>50</sub> shift of ∼8-fold, supporting the hypothesis that VHL-engagement is required for proteasome-mediated degradation of ERα (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. ERα PROTACs inhibit and degrade ERα in MCF7 cells. (a) Time course degradation of ERα treated with Fulvestrant (1 μM) or <b>18</b> (1 μM). Cells were lysed and assayed by western blot, and tubulin was used as a normalizer control. (b) In-cell western evaluating the persistence of degradation effect after a 2 h treatment, compound washout and cell collection after another 22 h, compared to full 24 h compound treatment. Continued ERα reduction was observed after the washout. ERα normalized to tubulin control. (c) Proteasomal degradation of ERα is VHL-mediated by observing that co-treating with VHL298 (5 μM, literature <i>K</i><sub>d</sub> = 90 nM) resulted in a higher DC<sub>50</sub> compared to compound alone, 48 h total treatment time followed by collection. (d) Degradation of ERα is significantly impaired in the negative control prolinol epimer <b>23</b> compared to the analogous active prolinol epimer <b>18</b>, showing that degradation is proceeding through a VHL-based mechanism. (e) Structure of VHL298. All traces and DC<sub>50</sub> values are from a single experiment with technical replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To additionally confirm the PROTAC mechanism is working <i>via</i> VHL engagement, we prepared the VHL prolinol epimer <b>23</b> of compound <b>18</b> as a negative control. Although the ERα binding potency for the negative control VHL amide epimer <b>23</b> was 90 nM, the in-cell western degradation result at 48 h showed  markedly less potent degradation (>770-fold) than the corresponding active degrader <b>18</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d). To further explore the ameliorated activity of compound <b>23</b>, we also evaluated the degradation at shorter timepoints (6 and 16 h) to reduce any potential assay interference/toxicity. At these shorter timepoints, compound <b>23</b> exhibited no protein degradation. Therefore, we concluded that, at longer timepoints (48 h), the slight reduction in protein levels was likely due to compound toxicity, and the degradation of ERα of this novel series proceeds through VHL-mediated degradation.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> PROTACs Degrade ERα in Multiple ER-Positive Cell Lines and Reduce Cell Viability</h3><div class="NLM_p">To demonstrate degradation of ERα across multiple cell lines, we tested ERα PROTACs, <b>18</b> and <b>21</b>, in an additional ER-positive cell line, T47D. MCF7 and T47D cell lines were treated with increasing doses of <b>18</b> and <b>21</b> for 24 h. Cell lysates were prepared, and western blotting was performed, with Actin used as a loading control. Similar to observations with the MCF7 cell line, robust ERα degradation was observed in T47D cells and the DC<sub>50</sub> values for the two cell lines were very similar (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a and Supplementary <a href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf" class="ext-link">Figure S3</a>). Given the remarkable potency of the compounds on all the GOF mutants, we would expect those ERα mutant proteins to be degraded in the cells, but due to unavailability of any commercially available cellular models for these mutants, we did not test this.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. ERα PROTACs demonstrate on-target effects in ER+ cell lines and not in ER– cell lines. (a) MCF7 and T47D cell lines were treated with increasing doses of <b>18</b> and <b>21</b> for 24 h showing ERα degradation. Cell lysates were prepared and western blotted for ERα with Actin as a loading control. (B) ER-positive (MCF7, T47D, and ZR-75-1) and ER-negative breast cancer cell lines (MBA-MB231) and immortalized breast cell line (MCF10A) were treated with increasing doses of ERα PROTACs, <b>18</b> and <b>21</b>, in full serum conditions for 7 days. Relative cell numbers were assessed using CellTiter-Glo.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, to test on-target <i>vs</i> off-target effects, we tested various breast cancer cell lines; ER+ (MCF7, T47D, and ZR-75-1), ER-negative (MDA-MB-231), and an immortalized normal breast cell line (MCF10A) in cellular viability assays. Various breast cell lines were treated with increasing doses of ERα PROTACs, <b>18</b> and <b>21</b>, in full serum conditions for 7 days. Relative cell numbers were assessed using CellTiter-Glo.</div><div class="NLM_p last">As expected, the results showed viability defects in all ER+ cell lines with GI<sub>50</sub> values ranging between 13 and 170 nM for <b>18</b> and 34 and 154 nM for <b>21</b>, while no effect on viability was seen in ER-negative breast cancer cell line MBA-MB 231 and the immortalized breast cell line MCF10A (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). These results confirm the on-target effects of this new series of ERα PROTACs.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> <i>In Vitro</i> ADME and PK</h3><div class="NLM_p">Overall, the amide and phenoxy VHL-based compounds <b>17</b>–<b>22</b> were potent degraders of ERα in ER+ cell lines, and we sought to evaluate the series further <i>in vivo</i>. For both classes, we decided to progress the two shortest of the three linker lengths <b>17</b>, <b>18</b>, <b>20</b>, and <b>21</b> in <i>in vitro</i> ADME assays and in mouse intra-venous and subcutaneous PK studies (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). In general, the optimization of ADME/PK of PROTACs can be challenging.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(37)</a> These compounds all have low solubility (<0.2 μM), low permeability (MDCK P<sub>app</sub> A-B < 0.1 cm/s × 10<sup>–6</sup>), and high PPB (>99.6%) (data not shown). These compounds do have moderate <i>in vitro</i> clearance in human and mouse microsomes (HLM CL<sub>int</sub> = 31–59 mL/min/kg; MLM CL<sub>int</sub> 139–249 mL/min/kg). <i>In vivo</i>, the IV clearance of the VHL phenoxy compounds <b>20</b> and <b>21</b> (CL<sub>obs</sub> = 1.2 and 0.9 mL/min/kg) was much lower than the VHL amide <b>17</b> and <b>18</b> (20 and 23 mL/min/kg), and the volume of distribution is much lower for the VHL phenoxy linked PROTACs (<b>20</b>, <b>21</b>: <i>Vd</i><sub>ss</sub> = 0.12 and 0.18 L/kg) compared the VHL amide (<b>17</b>, <b>18</b>: <i>Vd</i><sub>ss</sub> =1.24 and 4.82 L/kg), resulting in higher exposures for VHL-phenoxy PROTACs (<b>20</b>, <b>21</b>: AUC<sub>inf</sub> = 44.5 and 53.9 μM·hr) over VHL amide PROTACs (<b>17</b>, <b>18</b>: AUC<sub>inf</sub> = 9.8 and 14.6 μM·hr), as seen in the exposure curve depicted in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>f for <b>18</b> and <b>21</b>. Minimal exposure was observed when <b>18</b> and <b>21</b> were dosed orally (data not shown), which was anticipated given their low solubility and permeability, factors known to impede oral exposure.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(38)</a> Despite the low permeability, solubility, and high plasma protein binding of <b>18</b> and <b>21</b>, these two PROTACs present with potent degradation activity (though <b>21</b> leads to at least 20% greater <i>D</i><sub>max</sub> in cells), significant plasma exposures, yet differentiated PK profiles when dosed subcutaneously that warranted evaluation of ERα degradation in the <i>in vivo</i> setting.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <i>In Vitro</i> Clearance and <i>In Vivo</i> PK of Select Compounds in Male CD1 Mice<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">microsomal stability</th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">i.v. PK CD1 mouse</th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">s.c. PK CD1 mouse</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center" char=".">HLM CL<sub>int</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">MLM CL<sub>int</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center" char=".">Cl<sub>obs</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char=".">Vd<sub>ss,obs</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>inf</sub> (μM·hr)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">%<i>F</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">199</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">1.24</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">1.50</td><td class="colsep0 rowsep0" align="char" char=".">9.8</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">59</td><td class="colsep0 rowsep0" align="char" char=".">249</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">4.82</td><td class="colsep0 rowsep0" align="char" char=".">8.7</td><td class="colsep0 rowsep0" align="char" char=".">2.13</td><td class="colsep0 rowsep0" align="char" char=".">14.6</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char=".">162</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">0.12</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">2.08</td><td class="colsep0 rowsep0" align="char" char=".">44.5</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">47</td><td class="colsep0 rowsep0" align="char" char=".">139</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">0.18</td><td class="colsep0 rowsep0" align="char" char=".">4.7</td><td class="colsep0 rowsep0" align="char" char=".">3.55</td><td class="colsep0 rowsep0" align="char" char=".">53.9</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">25</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Human (HLM) and mouse (MLM) liver microsome stability. All compounds were formulated in 40% HP-β-CD in water. Cl<sub>obs</sub>, Vd<sub>ss</sub><i>T</i><sub>1/2</sub> determined by 0.3 mg/kg CD1 mouse IV dose cassette dosing, <i>C</i><sub>max</sub>, AUC<sub>inf</sub>, and <i>T</i><sub>1/2</sub> were determined by 10 mpk subcutaneous in CD1 mouse, %<i>F</i> is reported for AUC<sub>inf</sub> (sc/iv).</p></div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> <i>In Vivo</i> Efficacy Studies</h3><div class="NLM_p">Based on reasonable subcutaneous single-dose PK data (10 mpk in CD1 mice) for <b>18</b> and <b>21</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>f) showing exposure over 24 h, we progressed these two lead PROTACs, <b>18</b> and <b>21</b>, into a mouse efficacy study. The MCF7 xenograft model was chosen, and we tested the two compounds alongside Fulvestrant (reference compound) in a 28 day efficacy study (8 mice/arm). Fulvestrant was highly effective in suppressing MCF7 tumor growth and on day 28, and TGI (tumor growth inhibition) of 91.6% was seen, <i>p</i> < 0.001. Subcutaneous administration of 10 mg/kg qd of <b>21</b> also demonstrated significant tumor growth inhibition (TGI) = 84.3%, <i>p</i> < 0.001. <b>18</b> at 10 mg/kg showed marginal effect on tumor volume (TGI = 30.8% <i>p</i> > 0.05), and as such, dosing was discontinued at day 20 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a). No changes in average body weight in mice were recorded, and all compounds appeared to be well tolerated (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b). We collected plasma and tumor samples for end-point PK from the <b>18</b> and <b>21</b> arms, and the exposure in plasma and tumor correlated well with the efficacy seen <i>in vivo</i>, where <b>21</b> showed much better exposure as compared to <b>18</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c,d), though high variability was observed in the levels of <b>21</b> measure in tumor samples at the end of the study. We also collected tumors from a separate set of mice treated with all three compounds for 3 days. Western blot results demonstrate a reduction in ERα levels in all treatment arms, though some mouse-to-mouse variability in the levels is observed (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>e). These results clearly demonstrated that <b>21</b> is an excellent tool compound for inhibition and degradation of ERα both <i>in vitro</i> and <i>in vivo</i>.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <i>In vivo</i> efficacy and PK/PD study demonstrating efficacy in MCF7 xenograft model. (a) Antitumor activity of <b>18</b> and <b>21</b> compared to Fulvestrant in WT Estrogen-dependent MCF7 xenograft model. <b>18</b> and <b>21</b> were administered subcutaneously once daily at 10 mg/kg. Fulvestrant was dosed SC QW at 3 mg/mouse. Data represent the mean tumor volume ± SEM (<i>n</i> = 8). *<i>p</i> < 0.05 versus vehicle control (Student’s <i>t</i>-test). (b) All compounds were well tolerated with no changes in body weight. (c, d) Terminal Plasma PK and tumor PK of <b>18</b> and <b>21</b> at 3 h post-last dose (Day 20; <b>18</b>) and Day 27; <b>21</b>). (e) PD analysis from tumor bearing mice treated with compounds mentioned above for 3 days. Western blot analysis was performed to demonstrate reduction in ERα levels upon treatment with <b>18</b>, <b>21</b>, and Fulvestrant.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Since its introduction in 2001, the proteolysis targeting chimera (PROTAC) concept has been applied to many targets, including ERα.<a onclick="showRef(event, 'ref18 ref19 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref38 ref39">(18,19,39,40)</a> Currently, there is one ERα PROTAC in the clinic<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(41)</a> (ARV-471, structure was recently disclosed during review<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>), which is based on the cereblon binder lenalidomide and the SERM lasofoxifene. Although there are few other PROTACs in the clinic, the drug discovery path for PROTAC-based degraders is still being established, and there are many reviews trying to bring understanding to the community.<a onclick="showRef(event, 'ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43 ref44">(42−45)</a> One of the challenges of discovering suitable PROTACs is finding an appropriate high affinity novel binder with a vector for attaching a linker to an E3 engager.  DECL screening has a distinct advantage as an appropriate screening platform for designing PROTACs. Besides being an affinity (not activity)-based screening approach, the location of the linker attachment point is known <i>a priori</i>; thus, in the absence of structural information, the DNA-encoded chemistry platform affords a data-driven path forward to immediately begin linker optimization and small molecule bait presentation for a given POI. In conjunction with our DNA-encoded chemistry platform, we also established a toolkit to readily produce PROTACs based on well-known E3 ligase binders to enable the facile generation of arrays of PROTACs, where in one synthetic step, the POI binder, E3 binder, linker type and length can be explored.</div><div class="NLM_p">Using the information generated by DNA-encoded chemistry screening, we were able to generate a small set of compounds with nM binding affinity. Attachment of a basic amine at the linker region converted these compounds from potent agonists into potent antagonists and further validated that we were targeting ERα directly.</div><div class="NLM_p">Similar to bazedoxifene-based PROTACs, we also observed robust degradation with our novel ERα ligand with two peptidic ligands for VHL. We also demonstrated that this degradation was VHL-mediated by using a VHL ligand as a competitor and  by also synthesizing the negative control PROTAC incorporating an inactive stereoisomer in the VHL-binding portion (see compound <b>18</b><i>vs</i><b>23</b>). Though the VHL amide and VHL phenoxy were nearly equivalent in their <i>in vitro</i> assay profiles, they behaved quite differently in PK studies, with  markedly different AUC, CL<sub>obs</sub>, and Vd<sub>ss</sub>. We decided to progress a compound from each VHL variant (VHL amide and VHL-phenoxy) into the efficacy study and, aligned with the PK data, we observed, significant shrinkage in tumor volume with the phenoxy-linked PROTAC <b>21</b>.</div><div class="NLM_p">For ERα, both monovalent (Fulvestrant and other second-generation SERDs) as well as PROTAC-based degraders seem to be viable approaches to ERα degradation. Both approaches have their advantages, wherein monovalent degraders are usually smaller and Lipinski rule compliant. However the rules of converting a target binding ligand into a monovalent degrader are not very well understood and the levels of degradation could vary widely. For PROTACs, their modular design allows for relatively simple construction, but the resulting molecules are usually large and need optimization of physiochemical properties. PROTACs may offer advantages of catalytic activity<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(46)</a> and an acceptance of relatively weak POI binders.<a onclick="showRef(event, 'ref41 ref46'); return false;" href="javascript:void(0);" class="ref ref41 ref46">(42,47)</a></div><div class="NLM_p last">Overall, we have demonstrated that by utilizing our DECL platform, we were able to quickly identify very potent novel ERα binders that were efficiently incorporated into VHL-engaging PROTACs which demonstrated nanomolar ERα DC<sub>50</sub> values with activity in ER-positive cells while demonstrating no effects in ER-negative cells. We also tested these compounds in normal immortalized breast cells and observed no off-target effects. The compounds also demonstrated suitable properties for <i>in vivo</i> applications and were efficacious in an ERα-dependent xenograft model. These compounds are valuable tools to further the development of novel ERα-based PROTAC-based therapeutics for patients with breast cancer.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32345" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32345" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Construct Design and Cloning</h3><div class="NLM_p last">The nucleotide sequences of the ligand-binding domains (LBD), containing amino acids 307–554, of ERα WT and the clinically relevant mutations D538G, S463P, and Y537S were synthesized with an N-terminal Avi-Tev-His tandem affinity tag and cloned into a pD454-SR <i>E. coli</i> expression vector by ATUM (Newark, CA, USA).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Protein Expression and Purification</h3><div class="NLM_p last">Proteins were expressed in T7 Express cells for 12 h at 20 °C after induction with IPTG and then purified using standard Ni-NTA affinity chromatography followed by size exclusion chromatography.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Affinity-Mediated Selection of DNA-Encoded Chemical Libraries</h3><div class="NLM_p last">Selections were performed as previously described.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(48,49)</a> Forty-four different DNA-encoded chemical libraries were combined and incubated with ERα WT, D538G, S463P, or Y537S. Separate incubations were set up with no target, ERα WT over a five-fold dilution series down from 7.7 μM to 24 nM, ERα WT at 7.7 μM with estradiol at 15 μM, and each of the D538G, S463P, and Y537S at 7.7 μM. After incubation, the mixtures were captured on His-Select IMAC matrix. After capture, the matrices were washed and then heat-eluted at 85 °C. The second round of selection was performed using the round-one eluate as an input and fresh proteins and estradiol repeating the conditions of the first round. The encoding DNA in the output of the second round of selection was amplified and then sequenced.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> FP Binding Assay</h3><div class="NLM_p last">Inhibition values for ERα compounds were determined using a fluorescent polarization (FP) assay. Compounds were pre-incubated with protein for 15 min, prior to the addition of the fluorescent ligand. The plates were further incubated at room temperature for 1 h and read on Spectramax Paradigm (λex 485 nm, λem 535 nm).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Cell Lines</h3><div class="NLM_p last">Breast lines were obtained from the American Type Culture Collection: MCF7, MDA-MB231, T47D, ZR-75-1, and MCF10A and maintained in media specified by the ATCC.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Proliferation Assay</h3><div class="NLM_p last">Cancer cells were seeded in hormone-deprived media and incubated for 72 h. In agonist mode, test compounds were added to the cells in a 10-point dose series in the same media. For antagonist mode, all wells (except the minimum signal control wells) were treated with 0.1 nM estradiol for 30 min followed by treatment with compounds. Plates were incubated a further 7 days at 37 °C, 5% CO<sub>2</sub>, and then the cellular ATP content was measured using CellTiter-Glo 2.0 (Promega, Madison, WI) on a Tecan M1000 (Mannedorf, Switzerland). Once we began testing the PROTACs, the proliferation assay was run in complete media followed by compound treatment and a 7 day incubation period (results shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Degradation Assay-ICW (Hormone-Deprived Media)</h3><div class="NLM_p last">MCF7 cells were seeded in hormone-deprived media and incubated for 72 h. Test compounds were added to the cells at 10-point dose series in hormone-deprived media. DMSO was dispensed into the maximum-signal control wells, and a compound control (<b>13</b>) was dispensed into the minimum-signal control wells to give a final concentration of 1 μM and were incubated a further 48 h. Cells were fixed with 4% formaldehyde, permeabilized with 0.1% Triton-X 100, and blocked with 3% fish gelatin. Primary antibodies (ERα and α-tubulin) were incubated overnight and secondary antibodies for 1 h before imaging on the Azure Biosystem Sapphire Biomolecular Imager (Dublin, CA).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Degradation Assay (Western Blotting)</h3><div class="NLM_p last">For western blots, 5 × 10<sup>5</sup> MCF7 cells in 2 mL of media per well were seeded into six-well plates 48 h before treatment. Cells were treated for the indicated times and doses, and western blot images were obtained through detection of rabbit anti-ERα (D6R2W, 1:1000, Cell Signaling Technologies, Danvers, MA) and mouse anti-β-actin (1:10000, Sigma) antibodies with goat anti-rabbit IRDye 800CW secondary antibody (1:10000, LI-COR) and donkey anti-mouse IRDye 680RD secondary antibody (1:10000, LI-COR) using an Azure Biosystems Sapphire Biomolecular Imager (Dublin, CA).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Gene Expression Analysis</h3><div class="NLM_p last">MCF7 cells were seeded at 1.5 × 10<sup>4</sup> cells per well in 96-well plates in a volume of 90 μL complete media. Cell plates were incubated for 72 h. Test compounds were added in a 6-point dose series (10 μL). DMSO was dispensed into control wells, and Fulvestrant was dispensed as a reference (100 nM). All compounds were dosed for a final concentration of 0.5% DMSO (v/v). Cells with compound were further incubated for 24 h in humidified chambers at 37 °C, 5% CO<sub>2</sub>. RNA was extracted, cDNA prepared, and qPCRs run using specific TaqMan probes (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf" class="ext-link">Supplemental Methods</a>). Data were analyzed using the QuantStudio Design & Analysis Software v1.4.3 using “Comparative C<sub>T</sub> (ΔΔC<sub>T</sub>)” to analyze and export data to make dose–response curves in GraphPad Prism (San Diego, CA).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(50)</a></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> ADME/PK</h3><div class="NLM_p last">Determination of microsomal stability was performed at Pharmaron (Beijing, China). Test compounds (at 2 μM final concentration) were subjected to pooled human or male mouse liver microsomes, and % of compound remaining at 0, 15, 30, 45, and 60 min was measured and used to calculate an <i>in vitro</i> half-life (in minutes). PK studies were also performed at Pharmaron (Beijing, China) and were carried out according to the guidelines approved by the Institutional Animal Care and Use Committee (IACUC) of Pharmaron. Cassette iv (intravenous) PK was performed in male CD1 mice (3 mice per group) with up to five compounds each dosed at 0.3 mg/kg (dose volume of 2 mL/kg). The concentration of compounds was measured in plasma at nine timepoints (2, 5, 15, and 30 min and 1, 2, 4, 6, and 8 h post-dose), with 0.03 mL of blood collected via the dorsal metatarsal vein at each timepoint. Single dose sc (subcutaneous) PK was performed in male CD1 mice with each compound dosed at 10 mg/kg (dose volume of 10 mL/kg). The concentration of compounds was measured in plasma at nine timepoints (15 and 30 min and 1, 2, 4, 6, 8, 12, and 24 h post-dose), with 0.03 mL of blood collected via the dorsal metatarsal vein at each timepoint.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>In Vivo</i> Experiments</h3><div class="NLM_p last">All the procedures were performed according to the guidelines approved by the Institutional Animal Care and Use Committee (IACUC) of WuXi AppTec, Shanghai, China following the guidance of the Association for Assessment and Accreditation of Laboratory Animal Care (AAALAC). Each mouse was inoculated subcutaneously at the right flank with exponentially growing MCF-7 tumor cells (10 × 10<sup>6</sup>) in 0.2 mL of PBS mixed with Matrigel (50:50) for tumor development. 17β-Estradiol (0.18 mg) pellets (Innovative Research of America) were implanted 2 days before cell inoculation. Sixty-four animals were randomized when the average tumor volume reached 165 mm<sup>3</sup> for the efficacy study. Each group consisted of eight randomly assigned tumor-bearing mice. Mice were dosed with vehicle, <b>18</b>, <b>21</b>, or Fulvestrant until the vehicle volume reached 2000 mm<sup>3</sup> as per IACUC guidelines. Tumor size was measured two times a week in two dimensions using a caliper, and the tumor volume (<i>V</i>) was expressed in mm<sup>3</sup> using the formula: <i>V</i> = 0.5<i>a</i> × <i>b</i><sup>2</sup>, where <i>a</i> and <i>b</i> were the long and short diameters of the tumor, respectively. The mice were weighed every time prior to dosing. Tumor growth inhibition was calculated using the formula TGI (%) = (Vc – Vt)/(Vc −Vo) × 100, where Vc and Vt are the median of control and treated groups at the end of the study, respectively, and Vo at the start. Tumors were collected at the end of the study for PD analysis.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Synthetic Methods</h3><div class="NLM_p last">All solvents and chemicals were used as purchased without further purification. All the reported yields are isolated yields. <sup>1</sup>H NMR spectra were recorded on a Bruker Advance spectrometer at 400 MHz for <sup>1</sup>H NMR. The chemical shift is expressed in ppm relative to tetramethylsilane (TMS) as an internal standard, and CDCl<sub>3</sub>, DMSO-<i>d</i><sub>6</sub>, and CD<sub>3</sub>OD were used as solvents. Multiplicity of peaks is expressed as s (singlet), d (doublet), t (triplet), q (quartet), dd (doublet of doublets), td (triplet of doublets), qd (quartet of doublets), dt (doublet of triplets), and m (multiplet). Compound identity and purity was assessed by LCMS using a Thermo UltiMate 3000 instrument, Phenomenex XB-C18 2.6 μM 2.1 × 30 mm column, mobile phase A: 99.9 acetonitrile/0.1 formic acid; mobile phase B: 99.9 water/0.1 formic acid. Elution was accomplished via a 7.0 min gradient beginning at 90:10 A/B and ending with 100% B with compounds identified by UV, λ = 254 nM, and ESI positive ion mass spectrometry. High-resolution mass spectrometry was acquired on an Agilent 6500 Q-TOF. All purities were >95%, unless specified (see supplemental for tabulation). NMR and MS data are consistent with in silico predicted values. See <a href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf" class="ext-link">Supporting Information</a> for LC-MS traces (<b>1</b>–<b>23</b>) and <sup>1</sup>H and <sup>13</sup>C NMR spectra for <b>18</b> and <b>21</b>.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 2-Chloro-3-(((1-cyclopentyl-2,2,2-trifluoroethyl)(methyl)amino)methyl)phenol (<b>1</b>)</h3><div class="NLM_p last">To a solution of 1-cyclopentyl-2,2,2-trifluoroethan-1-amine hydrochloride (406 mg, 2 mmol) and 2-chloro-3-hydroxybenzaldehyde (312 mg, 2 mmol) in MeOH (15 mL) was added AcOH (3 drops) and NaBH<sub>3</sub>CN (126 mg, 2 mmol). The reaction mixture was stirred at room temperature overnight, and then formaldehyde (2 mL, dissolved in water) was added to the solution, and the reaction mixture was stirred for an additional 2 h. The solvent was removed under reduced pressure to afford the residue, which was purified by prep-HPLC to obtain the title compound (164 mg, 25%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 322.24; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.12 (t, <i>J</i> = 7.6 Hz, 1H), 6.89 (t, <i>J</i> = 8.8 Hz, 2H), 3.89 (s, 2H), 3.33–3.10 (m, 1H), 2.29 (s, 3H), 2.23–2.17 (m, 1H), 1.86–1.24 (m, 8H).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 2-Chloro-3-((((1-(2-fluorophenyl)cyclopentyl)methyl)(methyl)amino)methyl)phenol (<b>2</b>)</h3><div class="NLM_p last">To a solution of (1-(2-fluorophenyl)cyclopentyl)methanamine (386 mg, 2 mmol) and 2-chloro-3-hydroxybenzaldehyde (312 mg, 2 mmol) in MeOH (15 mL) was added AcOH (3 drops) followed by NaBH<sub>3</sub>CN (126 mg, 2 mmol). The mixture was stirred at room temperature overnight, and then formaldehyde (2 mL, dissolved in water) was added to the solution, and the reaction mixture was stirred for an additional 2 h. The solvent was removed under reduced pressure to obtain a residue, which was purified by prep-HPLC to obtain the title compound (329 mg, 47%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 348.34; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.46–7.36 (m, 2H), 7.23–7.17 (m, 3H), 7.09 (d, <i>J</i> = 7.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.2 Hz, 1H), 4.30 (d, <i>J</i> = 44.1 Hz, 2H), 3.54 (d, <i>J</i> = 22.8 Hz, 2H), 2.49 (s, 3H), 2.01–1.91 (m, 4H), 1.57–1.34 (m, 4H).</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 2-Chloro-3-((((1-(2-fluorophenyl)cyclopentyl)methyl)(phenyl)amino)methyl)phenol (<b>3</b>)</h3><div class="NLM_p">A solution of 1-(2-fluorophenyl)cyclopentane-1-carbaldehyde (200 mg, 1 mmol), aniline (93 mg, 1 mmol), NaBH<sub>3</sub>CN (126 mg, 2 mmol), and acetic acid (120 mg, 2 mmol) in methanol (5 mL) was stirred at room temperature overnight. The reaction mixture was concentrated under vacuum, and the residue was purified by column chromatography (20% EtOAc in petroleum ether with 1% TEA) to obtain <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)aniline (210 mg, 80%) as a colorless oil. MS: (ESI) [M + H]<sup>+</sup> 270.</div><div class="NLM_p last">A solution of <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)aniline (210 mg, 0.8 mmol), 2-chloro-3-hydroxybenzaldehyde (125 mg, 0.8 mmol), NaBH<sub>3</sub>CN (100 mg, 1.6 mmol), and acetic acid (100 mg, 1.6 mmol) in methanol (5 mL) was stirred at room temperature overnight. The reaction mixture was concentrated under vacuum and the crude product was purified by prep-HPLC to obtain the title compound (30 mg, 9%) as a white solid. HPLC purity, 94.7%. MS: (ESI) [M + H]<sup>+</sup> 410.33; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.30–7.08 (m, 2H), 7.01 (ddd, <i>J</i> = 12.3, 6.7, 2.0 Hz, 4H), 6.87 (t, <i>J</i> = 7.9 Hz, 1H), 6.73 (dd, <i>J</i> = 8.0, 1.3 Hz, 1H), 6.60–6.49 (m, 3H), 6.42–6.34 (m, 1H), 4.19 (s, 2H), 3.74 (s, 2H), 2.25–2.12 (m, 2H), 2.08–1.96 (m, 2H), 1.93–1.79 (m, 2H), 1.77–1.63 (m, 2H).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 2-Chloro-3-((((1-(2-fluorophenyl)cyclopentyl)methyl)(2-(piperidin-1-yl)ethyl)amino)methyl)phenol (<b>4</b>)</h3><div class="NLM_p">To a solution of (1-(2-fluorophenyl)cyclopentyl)methanamine (300 mg, 1.55 mmol), 1-(2-chloroethyl)piperidine (228 mg,1.55 mmol) in acetonitrile (10 mL) was added potassium carbonate (212 mg,1.55 mmol). Then, the mixture was stirred at 80 °C for 2 h. The solution was concentrated under vacuum, and the residue was purified by silica gel column chromatography (5% MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to obtain <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)-2-(piperidin-1-yl)ethan-1-amine (400 mg, 85%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 305.</div><div class="NLM_p last">To a solution of <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)-2-(piperidin-1-yl)ethan-1-amine (400 mg,1.31 mmol), 2-chloro-3-hydroxybenzaldehyde (204 mg,1.31 mmol) in MeOH (10 mL) was added acetic acid (1 mL). The mixture was stirred at room temperature for 5 h, and then NaBH<sub>3</sub>CN (83 mg, 1.31 mmol) was added. After stirring overnight, the reaction mixture was concentrated under vacuum, and the residue was purified by prep-HPLC to obtain the title compound (30 mg, 5%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 445.33; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ = 7.40 (s, 1H), 7.27 (m, 1H), 7.07(m, 3H), 6.86 (m, 2H), 3.63 (s, 2H), 2.85(m, 8H), 2.52 (m, 1H), 2.01 (s, 2H), 1.87 (m, 2H), 1.45–1.57 (m, 9H)).</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 2-Chloro-3-((((1-(2-fluorophenyl)cyclopentyl)methyl)(3-(piperidin-1-yl)propyl)amino)methyl)phenol (<b>5</b>)</h3><div class="NLM_p">To a solution of (1-(2-fluorophenyl)cyclopentyl)methanamine (300 mg, 1.55 mmol), 1-(3-chloropropyl)piperidine (250 mg, 1.55 mmol) in CH<sub>3</sub>CN (10 mL) was added K<sub>2</sub>CO<sub>3</sub> (212 mg, 1.55 mmol). The mixture was stirred at 80 °C for 2 h and then concentrated under vacuum. The residue was purified by column chromatography (5% MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to obtain <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)-3-(piperidin-1-yl)propan-1-amine (400 mg, 81%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 319.</div><div class="NLM_p last">To a solution of <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)-3-(piperidin-1-yl)propan-1-amine (400 mg, 1.25 mmol), 2-chloro-3-hydroxybenzaldehyde (295 mg,1.25 mmol) in methanol (10 mL) was added AcOH (1 mL). The mixture was stirred at 25 °C for 5 h, and then NaBH<sub>3</sub>CN (79 mg, 1.25 mmol) was added to the solution. The solution was concentrated under vacuum and the residue was purified by prep-HPLC to obtain the title compound (30 mg, 5%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 459.24; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ = 10.07 (s, 1H), 7.29 (m, 2H), 7.06 (m, 3H), 76.89(d, 1H), 6.82 (d, 1H), 3.40 (s, 2H), 3.22(d, 2H), 2.68 (m, 4H), 2.58 (s, 2H), 2.07 (m, 2H), 1.95 (s, 2H)), 1.71 (m, 6H), 1.32–1.50 (m, 6H).</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 2-Chloro-3-((((1-(2-fluorophenyl)cyclopentyl)methyl)(2-(2-(piperidin-1-yl)ethoxy)ethyl)amino)methyl)phenol (<b>6</b>)</h3><div class="NLM_p">To a solution of 2-(2-(piperidin-1-yl)ethoxy)ethan-1-amine (172 mg, 1 mmol) and 1-(2-fluorophenyl)cyclopentane-1-carbaldehyde (192 mg, 1 mmol) in MeOH (20 mL) was added acetic acid (0.2 mL). The reaction mixture was stirred for 3 h, and then NaBH<sub>3</sub>CN (124 mg, 2 mmol) was added, and the reaction mixture was stirred overnight at room temperature. The reaction mixture was concentration under vacuum, and the crude product was purified by column chromatography (5% MeOH in CH<sub>2</sub>Cl<sub>2</sub>) to obtain <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)-2-(2-(piperidin-1-yl)ethoxy)ethan-1-amine (174 mg) as a yellow oil. MS: (ESI) [M + H]<sup>+</sup> 349.</div><div class="NLM_p last">To a solution of <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)-2-(2-(piperidin-1-yl)ethoxy)ethan-1-amine (174 mg, 0.5 mmol) and 2-chloro-3-hydroxybenzaldehyde (78 mg, 0.5 mmol) in MeOH (20 mL) was added titanium(IV) isopropoxide (0.2 mL). The reaction mixture was stirred for 3 h, then NaBH<sub>3</sub>CN (92 mg, 1.5 mmol) was added, and the mixture was stirred overnight at room temperature. The reaction mixture was concentration under vacuum, and the crude product was purified by column chromatography (5% MeOH in CH<sub>2</sub>Cl<sub>2</sub>), and then by prep-HPLC to obtain the title compound (70 mg, 29%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 489.34; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.31 (td, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.21 (ddd, <i>J</i> = 7.2, 4.9, 1.7 Hz, 1H), 7.10–6.96 (m, 3H), 6.92 (d, <i>J</i> = 6.2 Hz, 1H), 6.81 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 3.59–3.53 (m, 4H), 3.48–3.34 (m, 4H), 3.23–3.19 (m, 2H), 3.14–2.87 (m, 2H), 2.85 (s, 2H), 2.41 (q, <i>J</i> = 6.4 Hz, 2H), 2.06 (d, <i>J</i> = 6.2 Hz, 2H), 1.95–1.67 (m, 7H), 1.62–1.42 (m, 5H).</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 2-Chloro-3-((((1-(2-fluorophenyl)cyclopentyl)methyl)(4-(2-(piperidin-1-yl)ethoxy)phenyl)amino)methyl)phenol (<b>7</b>)</h3><div class="NLM_p">To a solution of 1-(2-fluorophenyl)cyclopentane-1-carboxylic acid (1 g, 5 mmol) in methanol (10 mL) was added SOCl<sub>2</sub> (1.2 g, 10 mmol) dropwise, and the solution was stirred overnight and then concentrated under vacuum. The residue was diluted with water (20 mL) and extracted with ethyl acetate (10 mL × 3). The combined organic layers were washed with NaHCO<sub>3</sub> (15 mL × 2) and brine (15 mL) and then dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentrated under vacuum to give methyl 1-(2-fluorophenyl)cyclopentane-1-carboxylate (1 g, crude) as a colorless oil, which was used directly in the next step. MS: (ESI) [M + H]<sup>+</sup> 223.</div><div class="NLM_p">A solution of methyl 1-(2-fluorophenyl)cyclopentane-1-carboxylate (5 mmol) and LiAlH<sub>4</sub> (tetrahydrofuran solution, 2 M, 5 mL) in tetrahydrofuran (10 mL) was stirred for 2 h at room temperature. The reaction was quenched by Na<sub>2</sub>SO<sub>4</sub>·10H<sub>2</sub>O. After filtration, the filtrate was concentrated under vacuum, and the crude product was purified by column chromatography (1:15 methanol/dichloromethane) to obtain (1-(2-fluorophenyl)cyclopentyl)methanol (970 mg, 72%) as a colorless oil. MS: (ESI) [M + H]<sup>+</sup> 195.</div><div class="NLM_p">A solution (1-(2-fluorophenyl)cyclopentyl)methanol (970 mg, 5 mmol), 2-iodoxybenzoic acid (1680 mg, 6 mmol), and DMSO (1 drop) in ethyl acetate (10 mL) was stirred at 80 °C overnight. The reaction mixture was filtered, and the filtrate was concentrated, diluted with water (20 mL), and extracted with ethyl acetate (10 mL × 3). The combined organic layers were washed with brine (15 mL × 3) and dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentrated under vacuum to obtain 1-(2-fluorophenyl)cyclopentane-1-carbaldehyde (1 g, crude) as a colorless oil. MS: (ESI) [M + H]<sup>+</sup> 193.</div><div class="NLM_p">A solution of 1-(2-fluorophenyl)cyclopentane-1-carbaldehyde (200 mg, 1 mmol), 4-(2-(piperidin-1-yl)ethoxy)aniline (220 mg, 1 mmol), NaBH<sub>3</sub>CN (126 mg, 2 mmol), and acetic acid (120 mg, 2 mmol) in methanol (5 mL) was stirred overnight at room temperature. The reaction mixture was concentrated under vacuum, and the crude product was purified by column chromatography (33% EtOAc in petroleum ether with 1% TEA) to obtain <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)-4-(2-(piperidin-1-yl)ethoxy)aniline (300 mg, 75%) as a colorless oil. MS: (ESI) [M + H]<sup>+</sup> 397.</div><div class="NLM_p last">A solution <i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)-4-(2-(piperidin-1-yl)ethoxy)aniline (300 mg, 0.8 mmol), 2-chloro-3-hydroxybenzaldehyde (125 mg, 0.8 mmol), NaBH<sub>3</sub>CN (100 mg, 1.6 mmol), and acetic acid (100 mg, 1.6 mmol) in MeOH (5 mL) was stirred at room temperature overnight. The reaction mixture was concentrated under vacuum, and the crude product was purified by prep-HPLC to obtain the title compound (40 mg, 10%) as a white solid. HPLC purity, 94.9%. MS: (ESI) [M + H]<sup>+</sup> 537.47; <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.26–7.16 (m, 2H), 7.04–6.94 (m, 2H), 6.90 (t, <i>J</i> = 7.9 Hz, 1H), 6.72 (dd, <i>J</i> = 26.9, 20.0 Hz, 5H), 6.49 (d, <i>J</i> = 7.5 Hz, 1H), 4.23 (d, <i>J</i> = 15.5 Hz, 4H), 3.77 (s, 2H), 3.59 (d, <i>J</i> = 12.1 Hz, 2H), 3.52–3.45 (m, 2H), 3.13–2.96 (m, 2H), 2.34–1.90 (m, 7H), 1.90–1.73 (m, 5H), 1.73–1.47 (m, 2H).</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 2-Chloro-3-((methyl((1-(2-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methyl)amino)methyl)phenol (<b>8</b>)</h3><div class="NLM_p">To a solution of (1-(2-methoxyphenyl)cyclopentyl)methanamine (800 mg, 3.9 mmol) and NaHCO<sub>3</sub> (982 mg, 11.7 mmol) in THF (10 mL) and water (10 mL) was added (Boc)<sub>2</sub>O (1.7 g, 7.8 mmol). The reaction mixture was stirred overnight at room temperature and then concentrated in vacuo. The residue was extracted with dichloromethane (10 mL × 2), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated in vacuo, and purified by column chromatography to obtain <i>tert</i>-butyl ((1-(2-methoxyphenyl)cyclopentyl)methyl)carbamate (925 mg, 78%) as an oil. MS: (ESI) [M + Na]<sup>+</sup> 328.1.</div><div class="NLM_p">To a solution of <i>tert</i>-butyl ((1-(2-methoxyphenyl)cyclopentyl)methyl)carbamate (200 mg, 0.67 mmol) in THF (10 mL) at 0 °C was added NaH (47 mg, 1.97 mmol). The mixture was stirred at 0 °C for 30 min, and then MeI (186 mg, 1.310 mmol) was added. The mixture was stirred at room temperature overnight and then quenched with ice water (20 mL) and extracted with ethyl acetate (15 mL × 3). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated, and the residue was purified by column chromatography to obtain <i>tert</i>-butyl ((1-(2-methoxyphenyl)cyclopentyl)methyl)(methyl)carbamate (175 mg, 84%) as an oil. MS: (ESI) [M + Na]<sup>+</sup> 342.1.</div><div class="NLM_p">To a solution of <i>tert</i>-butyl ((1-(2-methoxyphenyl)cyclopentyl)methyl)(methyl)carbamate (175 mg, 0.56 mmol) in dichloromethane (8 mL) was added BBr<sub>3</sub> (278 mg, 1.11 mmol) at 0 °C. Then, the mixture was stirred at room temperature for 3 h. The solution was quenched with 1:9 ice water/methanol (10 mL) and extracted with dichloromethane (15 mL × 3). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated, and the residue was purified by column chromatography to obtain 2-(1-((methylamino)methyl)cyclopentyl)phenol (52 mg, 46%) as an oil. MS: (ESI) [M + H]<sup>+</sup> 206.1.</div><div class="NLM_p">To a solution of 2-(1-((methylamino)methyl)cyclopentyl)phenol (52 mg, 0.253 mmol) and NaHCO<sub>3</sub> (64 mg, 0.759 mmol) in THF (5 mL) and water (5 mL) was added (Boc)<sub>2</sub>O (110 mg, 0.506 mmol). The reaction mixture was stirred overnight at room temperature and concentrated in vacuo, and the residue was acidified by 1 N HCl to pH = 3. The solution was extracted with ethyl acetate (10 mL × 2), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, concentrated in vacuo, and purified by column chromatography to obtain <i>tert</i>-butyl ((1-(2-hydroxyphenyl)cyclopentyl)methyl)(methyl)carbamate (50 mg, 77%) as an oil. MS: (ESI) [M + Na]<sup>+</sup> 328.1.</div><div class="NLM_p">To a solution of <i>tert</i>-butyl ((1-(2-hydroxyphenyl)cyclopentyl)methyl)(methyl)carbamate (47 mg, 0.154 mmol) and K<sub>2</sub>CO<sub>3</sub> (64 mg, 0.462 mmol) in acetonitrile (5 mL) was added 1-(2-chloroethyl)piperidine (34 mg, 0.231 mmol). The mixture was heated to 80 °C for 5 h and concentrated in vacuo, and the residue was purified by column chromatography to obtain <i>tert</i>-butyl methyl((1-(2-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methyl)carbamate (50 mg, 77%) as an oil. MS: (ESI) [M + H]<sup>+</sup> 417.3.</div><div class="NLM_p">A solution of <i>tert</i>-butyl methyl((1-(2-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methyl)carbamate (47 mg, 0.113 mmol) in HCl/dioxane (5 mL) was stirred at for 2 h at room temperature. The solution was concentrated in vacuo to obtain <i>N</i>-methyl-1-(1-(2-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methanamine (50 mg, crude) as an oil. LC-MS (M + H)<sup>+</sup> = 317.2.</div><div class="NLM_p last">A solution of <i>N</i>-methyl-1-(1-(2-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methanamine (47 mg, 0.149 mmol) and 2-chloro-3-hydroxybenzaldehyde (35 mg, 0.224 mmol) in Ti(OPr)<sub>4</sub> (2 mL) was stirred for 4 h at room temperature. NaBH<sub>3</sub>CN (28 mg, 0.447 mmol) was added, and after stirring overnight at room temperature, the reaction was quenched with methanol (20 mL) and water (1 mL). The solid was removed by filtration, and the solution was concentrated in vacuo. The residue was purified by prep-HPLC to obtain the title compound (2.6 mg, 4%) as an off-white solid. MS: (ESI) [M + H]<sup>+</sup> 457.27; <sup>1</sup>H NMR (400 MHz, DMSO) δ 9.97 (s, 1H), 8.14 (s, 1H), 7.21–7.16 (m, 1H), 7.08–7.04 (m, 1H), 6.97 (d, 1H), 6.88–6.76 (m, 3H), 4.19 (t, 2H), 3.38–3.32 (m, 4H), 3.18–3.13 (m, 2H), 2.89–2.80 (m, 3H), 2.75 (s, 2H), 2.12–2.08 (m, 2H), 1.79–1.68 (m, 9H), 1.51–1.48 (m, 5H).</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 2-Chloro-3-((((1-(2-fluoro-4-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methyl)(methyl)amino)methyl)phenol (<b>9</b>)</h3><div class="NLM_p">A solution of 2-(2-fluoro-4-methoxyphenyl)acetonitrile (1650 mg, 10 mmol), 1,4-dibromobutane (3210 mg, 15 mmol), and NaH (360 mg, 15 mmol) in dried DMF (10 mL) was stirred at room temperature for 6 h. After the reaction was completed, water (20 mL) was added 0 °C. The solution was concentrated, water was added, and the solution was extracted with CH<sub>2</sub>Cl<sub>2</sub> (30 mL × 3). The combined organic layers were dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtrated, and concentrated under reduced pressure, and the residue was purified by column chromatography to obtain 1-(2-fluoro-4-methoxyphenyl)cyclopentane-1-carbonitrile (768 mg, 35%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 220.1.</div><div class="NLM_p">A solution of 1-(2-fluoro-4-methoxyphenyl)cyclopentane-1-carbonitrile (760 mg, 3.47 mmol) and LiAlH<sub>4</sub> (132 mg, 3.47 mmol) in dried THF (10 mL) was stirred at 0 °C for 3 h. The reaction was completed by LC-MS, and Na<sub>2</sub>SO<sub>4</sub>·10 H<sub>2</sub>O (1117 mg, 3.47 mmol) was added to the solution at 0 °C. The solution was filtrated and concentrated to obtain (1-(2-fluoro-4-methoxyphenyl)cyclopentyl)methanamine (610 mg, 79%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 224.2.</div><div class="NLM_p">To a solution of (1-(2-fluoro-4-methoxyphenyl)cyclopentyl)methanamine (600 mg, 2.69 mmol) in dry CH<sub>2</sub>Cl<sub>2</sub> (10 mL) at 0 °C was added BBr<sub>3</sub> (2 mL) dropwise, and then the mixture was stirred for 2 h until the product was detected by LC-MS. After the reaction was completed, water (10 mL) was added to the solution at 0 °C, and the solution extracted with CH<sub>2</sub>Cl<sub>2</sub> (30 mL × 3), and the combined organic layer was dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtrated, and then concentrated under vacuum to obtain 4-(1-(aminomethyl)cyclopentyl)-3-fluorophenol (368 mg, 65%) as a white oil. MS: (ESI) [M + H]<sup>+</sup> 210.1.</div><div class="NLM_p">A solution of 4-(1-(aminomethyl)cyclopentyl)-3-fluorophenol (360 mg, 1.72 mmol), Boc<sub>2</sub>O (376 mg, 1.72 mmol), NaHCO<sub>3</sub> (168 mg, 2 mmol) in THF (10 mL), and water (2 mL) was stirred at room temperature for 3 h. The solution was concentrated, diluted with water, and extracted with CH<sub>2</sub>Cl<sub>2</sub> (20 mL × 3), and the combined organic layers were dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtrated, and concentrated under vacuum, and the crude product was purified using column chromatography to obtain <i>tert</i>-butyl ((1-(2-fluoro-4-hydroxyphenyl)cyclopentyl)methyl)carbamate (185 mg, 35%) as a white solid. MS: (ESI) [M + H-<i>t</i>Bu]<sup>+</sup> 254.2.</div><div class="NLM_p">A solution <i>tert</i>-butyl ((1-(2-fluoro-4-hydroxyphenyl)cyclopentyl)methyl)carbamate (170 mg, 0.55 mmol), 1-(2-chloroethyl)piperidine (88 mg, 0.6 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (196 mg, 0.6 mmol) in DMF (10 mL) was stirred at 100 °C for 3 h. The reaction mixture was cooled, concentrated, diluted with water, and extracted with CH<sub>2</sub>Cl<sub>2</sub> (20 mL × 3). The combined organic layers were dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub> and filtrated, and the residue was purified by column chromatography to obtain <i>tert</i>-butyl ((1-(2-fluoro-4-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methyl)carbamate (118 mg, 51%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 421.3.</div><div class="NLM_p">A solution of <i>tert</i>-butyl ((1-(2-fluoro-4-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methyl)carbamate (118 mg, 0.28 mmol) in 1,4-dioxane HCl (4 M, 5 mL) was stirred at room temperature. After 2 h, the solution was concentrated to obtain (1-(2-fluoro-4-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methanamine (85 mg, 95%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 321.2.</div><div class="NLM_p last">A solution of 2-chloro-3-hydroxybenzaldehyde (47 mg, 0.3 mmol), (1-(2-fluoro-4-(2-(piperidin-1-yl)ethoxy)phenyl)cyclopentyl)methanamine (85 mg, 0.27 mmol) in AcOH (0.2 mL), and MeOH (10 mL) was stirred for 2 h at room temperature. Then, NaBH<sub>3</sub>CN (19 mg, 0.3 mmol) was added, and the reaction mixture was stirred overnight. Paraformaldehyde (12 mg, 0.4 mmol) was added, and the mixture was stirred for 2 h. Then, another aliquot of NaBH<sub>3</sub>CN (19 mg, 0.3 mmol) was added and stirring was continued at room temperature for 5 h. After the reaction was completed, the solution was concentrated diluted with water and extracted with CH<sub>2</sub>Cl<sub>2</sub> (20 mL × 3). The combined organic layers were dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtrated, and concentrated under vacuum, and the residue was purified by column chromatography followed by prep-HPLC to obtain the title compound (35 mg, 27%) as a white solid. MS: (ESI) [M + H]<sup>+</sup> 475.27; <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.33–7.29 (m, 1H), 7.13–7.10 (m, 1H), 6.93–6.91 (m, 2H), 6.78–6.74 (m, 2H), 4.35 (s, 2H), 3.80 (s, 2.67H), 3.54–3.50 (m, 2.36H), 3.15 (s, 3H), 2.32 (s, 3H), 2.04–1.90 (m, 8H), 1.71–1.63 (m, 6H).</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 3-(((4-(2-Azidoethoxy)phenyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)methyl)-2-chlorophenol (Intermediate 7-Azide)</h3><div class="NLM_p">To a solution of 2-(4-aminophenoxy)ethan-1-ol (306 mg, 2 mmol) and (Boc)<sub>2</sub>O (436 mg, 2 mmol) in THF (20 mL) was added K<sub>2</sub>CO<sub>3</sub> (828 mg, 6 mmol). The reaction mixture was stirred overnight and then concentrated under vacuum. The residue was diluted with water (30 mL) and extracted with ethyl acetate (30 mL × 3). The combined organic layers were washed with brine (25 mL × 3) and then dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentration under vacuum, and the crude product was purified by column chromatography (5% MeOH in DCM) to obtain <i>tert</i>-butyl (4-(2-hydroxyethoxy)phenyl)carbamate (540 mg, crude) as a yellow oil. MS: (ESI) [M + H]<sup>+</sup> 254.</div><div class="NLM_p">To a solution of <i>tert</i>-butyl (4-(2-hydroxyethoxy)phenyl)carbamate (540 mg, crude) in DCM (20 mL) was added MsCl (0.5 mL) and DIEA (1 mL). The reaction mixture was stirred for 1 h, and then the reaction mixture was washed with brine (25 mL × 3) and dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentration under vacuum to a obtain 2-(4-((<i>tert</i>-butoxycarbonyl)amino)phenoxy)ethyl methanesulfonate (470 mg) as a yellow oil. MS: (ESI) [M + H]<sup>+</sup> 332.</div><div class="NLM_p">To a solution of 2-(4-((<i>tert</i>-butoxycarbonyl)amino)phenoxy)ethyl methanesulfonate (470 mg, crude) in DMF (10 mL) was added NaN<sub>3</sub> (650 mg, 10 mmol). The reaction mixture was stirred at 60 °C overnight. The reaction mixture was diluted with water (50 mL) and extracted with ethyl acetate (30 mL × 3). The combined organic layer was washed with brine (30 mL × 3) and then dried over with anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentration under vacuum to give a crude <i>tert</i>-butyl (4-(2-azidoethoxy)phenyl)carbamate (530 mg, crude) as a yellow oil. MS: (ESI) [M + H]<sup>+</sup> 279.</div><div class="NLM_p">To a solution of <i>tert</i>-butyl (4-(2-azidoethoxy)phenyl)carbamate (200 mg, crude) in DMF (10 mL) was added TFA (1 mL). The reaction mixture was stirred at room temperature for 2 h and then concentrated under vacuum to obtain crude 4-(2-azidoethoxy)aniline (350 mg, crude) as a yellow oil. MS: (ESI) [M + H]<sup>+</sup> 179.</div><div class="NLM_p">To a solution of 4-(2-azidoethoxy)aniline (300 mg, crude) and 1-(2-fluorophenyl)cyclopentane-1-carbaldehyde (192 mg, 1 mmol) in MeOH (20 mL) was added acetic acid (0.2 mL). The reaction mixture was stirred at room temperature for 3 h. Then, NaBH<sub>3</sub>CN (124 mg, 2 mmol) was added and the reaction mixture was stirred overnight. The reaction mixture was concentration under vacuum, and the crude product was purified by column chromatography (5% MeOH in DCM) to obtain 4-(2-azidoethoxy)-<i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)aniline (390 mg) as a yellow oil. MS: (ESI) [M + H]<sup>+</sup> 355.</div><div class="NLM_p last">To a solution of 4-(2-azidoethoxy)-<i>N</i>-((1-(2-fluorophenyl)cyclopentyl)methyl)aniline (350 mg, crude) and 1-(2-fluorophenyl)cyclopentane-1-carbaldehyde (78 mg, 0.5 mmol) in MeOH (20 mL) was added AcOH (0.2 mL), and the reaction mixture was stirred overnight. Then, NaBH<sub>3</sub>CN (124 mg, 2 mmol) was added, and the reaction mixture was stirred overnight at room temperature. The reaction mixture was concentrated and then purified by prep-HPLC to obtain the title intermediate (110 mg) as a colorless oil. MS: (ESI) [M + H]<sup>+</sup> 489. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.30–7.16 (m, 2H), 7.04–6.97 (m, 2H), 6.90 (t, <i>J</i> = 7.9 Hz, 1H), 6.73 (dd, <i>J</i> = 8.0, 1.2 Hz, 1H), 6.70–6.64 (m, 2H), 6.54 (d, <i>J</i> = 9.2 Hz, 2H), 6.47 (d, <i>J</i> = 7.6 Hz, 1H), 4.15 (s, 2H), 4.05–4.00 (m, 2H), 3.69 (s, 2H), 3.52–3.48 (m, 2H), 2.23–2.13 (m, 2H), 2.05–1.95 (m, 2H), 1.88–1.76 (m, 2H), 1.75–1.63 (m, 2H).</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> General Procedure for Huisgen 1,3-Diploar Cycloaddition</h3><div class="NLM_p">Equal volumes of 200 mM THPTA aqueous solution and 100 mM CuSO<sub>4</sub>·5H<sub>2</sub>O aqueous solution were combined, vortexed and let stand for 5 min. Separately, 1 mL of azide solution (25 mM in DMSO) and 1 mL of alkyne solution (25 mM) were added to a reaction vial, vortexed,  and then 75 μL of the premixed THPTA/CuSO<sub>4</sub>·5H<sub>2</sub>O solution was added, followed by  225 μL of sodium ascorbate solution (100 mM in water). The reaction mixture was  vortexed  and then the reaction was monitored  by LCMS. If necessary  the reaction was heated to 50 °C  overnight. After reaction completion, the cooled reaction mixture was loaded directly and purified by a mass triggered Waters LCMS reversed phase purification system using water/acetonitrile/0.1% formic acid as an eluent. The fractions were dried by lyophilization to afford the title compound as a solid.</div><div id="sec4_23_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-(2,6-Dioxopiperidin-3-yl)-4-(2-((1-(2-(4-((5-hydroxy-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-1-yl)methyl)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)isoindoline-1,3-dione (<b>10</b>)</h4><div class="NLM_p last">Following General Procedure using 1-(4-(2-azidoethoxy)benzyl)-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-5-ol (10.4 mg, 0.025 mmol) and 2-(2,6-dioxopiperidin-3-yl)-4-(2-(prop-2-yn-1-yloxy)ethoxy)isoindoline-1,3-dione (8.9 mg, 0.025 mmol) to obtain the title compound (14.5 mg, 75% yield). MS: (ESI) [M + H]<sup>+</sup> 771.42.</div></div><div id="sec4_23_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-((<i>S</i>)-2-(2-((1-(2-(4-((5-hydroxy-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-1-yl)methyl)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)acetamido)-3,3-dimethylbutanoyl)-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>11</b>)</h4><div class="NLM_p last">Following General Procedure using 1-(4-(2-azidoethoxy)benzyl)-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-5-ol (10.4 mg, 0.025 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(1-fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)- 4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)-2-(2-(prop-2-yn-1-yloxy)ethoxy)benzyl)pyrrolidine-2-carboxamide (13.5 mg, 0.025 mmol) to obtain the title compound (21.2 mg, 89% yield). MS: (ESI) [M + H]<sup>+</sup> 955.43.</div></div><div id="sec4_23_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-((<i>S</i>)-2-(2-(2-((1-(2-(4-((5-hydroxy-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-1-yl)methyl)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)acetamido)-3,3-dimethylbutanoyl)-<i>N</i>-((S)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>12</b>)</h4><div class="NLM_p last">Following General Procedure using 1-(4-(2-azidoethoxy)benzyl)-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-5-ol solution (10.4 mg, 0.025 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-3,3-dimethyl-2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)acetamido)butanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide solution (14.6 mg, 0.025 mmol) to afford the title compound (22.2 mg, 89% yield). MS: (ESI) [M + H]<sup>+</sup> 999.38.</div></div><div id="sec4_23_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-12-(<i>tert</i>-Butyl)-1-(1-(2-(4-((5-hydroxy-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-1-yl)methyl)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)-10-oxo-2,5,8-trioxa-11-azatridecan-13-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>13</b>)</h4><div class="NLM_p last">Following General Procedure using 1-(4-(2-azidoethoxy)benzyl)-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-5-ol (10.4 mg, 0.025 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(<i>tert</i>-butyl)-4-oxo-6,9,12-trioxa-3-azapentadec-14-ynoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (15.7 mg, 0.025 mmol) to afford the title compound (22.3 mg, 86%). MS: (ESI) [M + H]<sup>+</sup> 1043.43.</div></div><div id="sec4_23_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(1-Fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(2-(2-((1-(2-(4-((5-hydroxy-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-1-yl)methyl)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)-4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>14</b>)</h4><div class="NLM_p last">Following General Procedure using 1-(4-(2-azidoethoxy)benzyl)-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-5-ol and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(1-fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)-2-(2-(prop-2-yn-1-yloxy)ethoxy)benzyl)pyrrolidine-2-carboxamide to obtain the title compound (55 mg, 71%). MS: (ESI) [M + H]<sup>+</sup> 1029.26.</div></div><div id="sec4_23_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>S</i>)-<i>N</i>-((<i>S</i>)-1-Cyclohexyl-2-((<i>S</i>)-2-(4-(3-(2-((1-(2-(4-((5-hydroxy-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-1-yl)methyl)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)benzoyl)thiazol-2-yl)pyrrolidin-1-yl)-2-oxoethyl)-2-(methylamino)propanamide (<b>15</b>)</h4><div class="NLM_p last">Following General Procedure using 1-(4-(2-azidoethoxy)benzyl)-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-5-ol and (<i>S</i>)-<i>N</i>-((<i>S</i>)-1-cyclohexyl-2-oxo-2-((<i>S</i>)-2-(4-(3-(2-(prop-2-yn-1-yloxy)ethoxy)benzoyl)thiazol-2-yl)pyrrolidin-1-yl)ethyl)-2-(methylamino)propanamide to obtain the title compound as the formic salt. MS: (ESI) [M + H]<sup>+</sup> 995.47.</div></div><div id="sec4_23_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>S</i>)-<i>N</i>-((<i>S</i>)-2-((<i>S</i>)-2-(4-(4-Fluorobenzoyl)thiazol-2-yl)pyrrolidin-1-yl)-1-(1-(2-(2-((1-(2-(4-((5-hydroxy-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-1-yl)methyl)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)acetyl)piperidin-4-yl)-2-oxoethyl)-2-(methylamino)propanamide (<b>16</b>)</h4><div class="NLM_p last">To a solution of 1-(4-(2-azidoethoxy)benzyl)-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-5-ol (22 mg, 0.053 mmol) and (<i>S</i>)-<i>N</i>-((<i>S</i>)-2-((S)-2-(4-(4-fluorobenzoyl)thiazol-2-yl)pyrrolidin-1-yl)-2-oxo-1-(1-(2-(2-(prop-2-yn-1-yloxy)ethoxy)acetyl)piperidin-4-yl)ethyl)-2-(methylamino)propanamide (21 mg, 0.033 mmol) in DMSO (2 mL) was added 0.08 mL of premixed aqueous solution of CuSO<sub>4</sub>·5H<sub>2</sub>O (100 mM, 0.04 mL) and THPTA (200 mM, 0.04 mL), and the mixture was stirred at room temperature for 2 min followed by the addition of sodium ascorbate (100 mM, 0.225 mL). The combined mixture was then shaken at 50 °C for 12 h. The cooled reaction mixture was loaded directly onto the Waters LCMS reversed phase purification system using water/acetonitrile/0.1% FA as an eluent. The fractions were dried by lyophilization to afford the title compound as the formate salt (14.7 mg, 42% yield). MS: (ESI) [M + H]<sup>+</sup> 1056.37.</div></div><div id="sec4_23_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-((1-(2-(4-((2-Chloro-3-hydroxybenzyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>17</b>)</h4><div class="NLM_p last">Following General Procedure using 3-(((4-(2-azidoethoxy)phenyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)methyl)-2-chlorophenol (12.4 mg, 0.025 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-3,3-dimethyl-2-(2-(prop-2-yn-1-yloxy)acetamido)butanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (13.5 mg, 0.025 mmol) to afford the title compound as the formic acid salt (9.6 mg, 35% yield). MS: (ESI) [M + H]<sup>+</sup> 1035.35.</div></div><div id="sec4_23_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-(2-((1-(2-(4-((2-Chloro-3-hydroxybenzyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>18</b>)</h4><div class="NLM_p last">To a solution of 3-(((4-(2-azidoethoxy)phenyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)methyl)-2-chlorophenol(372 mg, 0.75 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-3,3-dimethyl-2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)acetamido)butanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (422 mg, 0.723 mmol) in DMSO (15 mL) was added 0.8 mL of premixed aqueous solution of CuSO<sub>4</sub> ( 100 mM, 0.4 mL) and THPTA (200 mM, 0.4 mL), and the mixture was stirred at room temperature for 2 min followed by the addition of sodium ascorbate (100 mM, 2.25 mL). The combined mixture was then stirred at 25 °C for 12 h. The mixture was diluted with water (30 mL) and extracted with EtOAc (30 mL × 2), and the organic phase was washed with brine (20 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, concentrated to an oil, and purified over silica gel column chromatography eluted with DCM/MeOH (gradient) to obtain the title compound as a white solid (777 mg, 96% yield). MS: (ESI) [M + H]<sup>+</sup> 1079.37; HRMS (ES<sup>+</sup>) for C<sub>57</sub>H<sub>68</sub>ClFN<sub>8</sub>O<sub>8</sub>S (M + H)<sup>+</sup>: calcd 1079.4632; found 1079.4646; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 0.92 (s, 9H), 1.35 (d, <i>J</i> = 7.03 Hz, 3H), 1.57 (br s, 2H), 1.65–1.81 (m, 3H), 1.84–1.96 (m, 2H), 1.97–2.10(m, 3H), 2.45 (s, 3H), 3.55–3.68 (m, 8H), 3.95 (s, 2H), 4.05 (s, 2H), 4.17–4.31 (m, 3H), 4.45 (t, <i>J</i> = 8.28 Hz, 1H), 4.52–4.62 (m, 3H), 4.66 (br t, <i>J</i> = 4.89 Hz, 2H), 4.90 (br t, <i>J</i> = 7.15 Hz, 1H), 5.13 (d, <i>J</i> = 3.51 Hz, 1H), 6.30 (d, <i>J</i> = 7.53 Hz, 1H), 6.42–6.51 (m, 2H), 6.60–6.65 (m, 2H), 6.75 (d, <i>J</i> = 7.28 Hz, 1H), 6.90 (t, <i>J</i> = 7.78 Hz, 1H), 7.03–7.12 (m, 2H), 7.18–7.29 (m, 2H), 7.33–7.46 (m, 5H), 8.12–8.17 (m, 1H), 8.44 (d, <i>J</i> = 7.53 Hz, 1H), 8.98 (s, 1 H), 10.03 (s, 1H).</div></div><div id="sec4_23_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-12-(<i>tert</i>-Butyl)-1-(1-(2-(4-((2-chloro-3-hydroxybenzyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)-10-oxo-2,5,8-trioxa-11-azatridecan-13-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>19</b>)</h4><div class="NLM_p last">Following General Procedure using 3-(((4-(2-azidoethoxy)phenyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)methyl)-2-chlorophenol (12.4 mg, 0.025 mmol) and (2S,4R)-1-((<i>S</i>)-2-(<i>tert</i>-butyl)-4-oxo-6,9,12-trioxa-3-azapentadec-14-ynoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (15.7 mg, 0.025 mmol) to obtain the title compound as a formate salt (1.5 mg, 5%). MS: (ESI) [M + H]<sup>+</sup> 1123.25.</div></div><div id="sec4_23_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (2<i>S</i>,4<i>R</i>)-<i>N</i>-(2-(2-((1-(2-(4-((2-Chloro-3-hydroxybenzyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)-4-(4-methylthiazol-5-yl)benzyl)-1-((<i>S</i>)-2-(1-fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamide (<b>20</b>)</h4><div class="NLM_p last">Following General Procedure using 3-(((4-(2-azidoethoxy)phenyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)methyl)-2-chlorophenol (12.4 mg, 0.025 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(1-fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)-2-(2-(prop-2-yn-1-yloxy)ethoxy)benzyl)pyrrolidine-2-carboxamide (15.4 mg, 0.025 mmol) to obtain the title compound as the formic acid salt (9.4 mg, 29%). MS: (ESI) [M + H]<sup>+</sup> 1109.38.</div></div><div id="sec4_23_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (2<i>S</i>,4<i>R</i>)-<i>N</i>-(2-(2-(2-((1-(2-(4-((2-Chloro-3-hydroxybenzyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)ethoxy)-4-(4-methylthiazol-5-yl)benzyl)-1-((<i>S</i>)-2-(1-fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamide (<b>21</b>)</h4><div class="NLM_p last">To a solution of 3-(((4-(2-azidoethoxy)phenyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)methyl)-2-chlorophenol (421 mg, 0.85 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(1-fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)-2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)ethoxy)benzyl)pyrrolidine-2-carboxamide (519 mg, 0.788 mmol) in DMSO (15 mL) was added 0.8 mL of premixed aqueous solution of CuSO<sub>4</sub> (100 mM, 0.4 mL) and THPTA (200 mM, 0.4 mL), and the mixture was stirred at room temperature for 2 min followed by the addition of sodium ascorbate (100 mM, 2.25 mL). The combined mixture was then stirred at 25 °C for 12 h. The mixture was diluted with water (30 mL), extracted with EtOAc (30 mL x 2), the organic phase was washed with brine (20 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, concentrated to an oil, and purified over silica gel column chromatography eluted with DCM/MeOH (gradient) to obtain the title compound as a white solid (864 mg, 95%). MS: (ESI) [M + H]<sup>+</sup> 1153.41; HRMS (ES<sup>+</sup>) for C<sub>60</sub>H<sub>71</sub>ClF<sub>2</sub>N<sub>8</sub>O<sub>9</sub>S (M + H)<sup>+</sup>: calcd 1153.4800; found 1153.4794; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 0.93 (s, 9H), 1.10–1.28 (m, 2H), 1.29–1.43 (m, 2H), 1.57 (m, 2H), 1.73 (m, 2H), 1.84–1.92 (m, 3H), 1.99–2.12 (m, 3H), 2.43 (s, 3H), 3.56–3.67 (m, 8H), 3.71–3.83 (m, 2H), 4.04 (s, 2H), 4.12–4.23 (m, 5H), 4.25–4.38 (m, 2H), 4.46–4.55 (m, 3H), 4.56–4.70 (m, 3H), 5.17 (d, <i>J</i> = 3.51 Hz, 1H), 6.30 (d, <i>J</i> = 7.53 Hz, 1H), 6.45 (m, <i>J</i> = 9.03 Hz, 2H), 6.61 (m, <i>J</i> = 9.03 Hz, 2H), 6.75 (d, <i>J</i> = 7.03 Hz, 1H), 6.89 (t, <i>J</i> = 7.41 Hz, 1H), 6.96 (d, <i>J</i> = 7.26 Hz, 1H), 7.01–7.11 (m, 3H), 7.18–7.32 (m, 3H), 7.41 (d, <i>J</i> = 8.03 Hz, 1H), 8.09 (s, 1H), 8.49 (t, <i>J</i> = 5.90 Hz, 1H), 8.97 (s, 1H), 10.03 (s, 1H).</div></div><div id="sec4_23_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (2<i>S</i>,4<i>R</i>)-<i>N</i>-(2-(2-(2-(2-((1-(2-(4-((2-Chloro-3-hydroxybenzyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)ethoxy)ethoxy)-4-(4-methylthiazol-5-yl)benzyl)-1-((<i>S</i>)-2-(1-fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamide (<b>22</b>)</h4><div class="NLM_p last">Following General Procedure using 3-(((4-(2-azidoethoxy)phenyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)methyl)-2-chlorophenol (12.4 mg, 0.025 mmol) and (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(1-fluorocyclopropane-1-carboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)-2-(2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)ethoxy)ethoxy)benzyl)pyrrolidine-2-carboxamide (17.6 mg, 0.025 mmol) to obtain the title compound as a formate salt (5.5 mg, 18% yield). MS: (ESI) [M + H]<sup>+</sup> 1197.38.</div></div><div id="sec4_23_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (2<i>S</i>,4<i>S</i>)-1-((<i>S</i>)-2-(2-(2-((1-(2-(4-((2-Chloro-3-hydroxybenzyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)phenoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)ethoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>23</b>)</h4><div class="NLM_p last">Following General Procedure using 3-(((4-(2-azidoethoxy)phenyl)((1-(2-fluorophenyl)cyclopentyl)methyl)amino)methyl)-2-chlorophenol and (2<i>S</i>,4<i>S</i>)-1-((S)-3,3-dimethyl-2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)acetamido)butanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide to afford the title compound as the formic acid salt. MS: (ESI) [M + H]<sup>+</sup> 1079.40.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i54"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00127" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21116" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21116" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00127?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00127</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Denaturing gel of proteins, LC-MS for compounds <b>1–23</b> (parent ion, retention time, purity, and chromatograms), <sup>1</sup>H and <sup>13</sup>C NMR spectra for compounds <b>18</b> and <b>21</b>, supplemental figures demonstrating purification of ERα proteins and degradation of ERα in ER+ breast cancer cells, and supplemental methods giving detailed protocols for protein production, biochemical and cellular assays, and DECL screening (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf">jm1c00127_si_001.pdf (1.63 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_002.csv">jm1c00127_si_002.csv (4.58 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00127" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeremy S. Disch</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: ZebiAI Therapeutics, Waltham, Massachusetts 02453, United States (J.S.D., J.M.D., M.I.M., S.A.T., M.v.R., J.Z., Y.A.A., J.W.C.)</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0357-1672" title="Orcid link">http://orcid.org/0000-0002-0357-1672</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3f555b564c5c57060d7f58525e5653115c5052"><span class="__cf_email__" data-cfemail="fa909e93899992c3c8ba9d979b9396d4999597">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shilpi Arora</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: Exo Therapeutics, Cambridge, Massachusetts 02139, United States (S.A.)</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8dfefee2e3e4ecffe2ffeccdeae0ece4e1a3eee2e0"><span class="__cf_email__" data-cfemail="f182829e9f9890839e8390b1969c90989ddf929e9c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer M. Duffy</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: ZebiAI Therapeutics, Waltham, Massachusetts 02453, United States (J.S.D., J.M.D., M.I.M., S.A.T., M.v.R., J.Z., Y.A.A., J.W.C.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Esther C. Y. Lee</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: AstraZeneca, Waltham, Massachusetts 02451, United States (E.C.Y.L.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Diana Gikunju</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Betty Chan</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: Auron Therapeutics, Wellesley, Massachusetts 02481, United States (B.C., A.J.M.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin Levin</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: Dewpoint Therapeutics Boston, Massachusetts 02210, United States (B.L.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael I. Monteiro</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: ZebiAI Therapeutics, Waltham, Massachusetts 02453, United States (J.S.D., J.M.D., M.I.M., S.A.T., M.v.R., J.Z., Y.A.A., J.W.C.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah A. Talcott</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: ZebiAI Therapeutics, Waltham, Massachusetts 02453, United States (J.S.D., J.M.D., M.I.M., S.A.T., M.v.R., J.Z., Y.A.A., J.W.C.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony C. Lau</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: Broad Institute, Cambridge, Massachusetts 02142, United States (A.C.L.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fei Zhou</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anton Kozhushnyan</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Neil E. Westlund</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrick B. Mullins</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Yu</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Moritz von Rechenberg</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: ZebiAI Therapeutics, Waltham, Massachusetts 02453, United States (J.S.D., J.M.D., M.I.M., S.A.T., M.v.R., J.Z., Y.A.A., J.W.C.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junyi Zhang</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: ZebiAI Therapeutics, Waltham, Massachusetts 02453, United States (J.S.D., J.M.D., M.I.M., S.A.T., M.v.R., J.Z., Y.A.A., J.W.C.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yelena A. Arnautova</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: ZebiAI Therapeutics, Waltham, Massachusetts 02453, United States (J.S.D., J.M.D., M.I.M., S.A.T., M.v.R., J.Z., Y.A.A., J.W.C.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanbin Liu</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: Cyteir Therapeutics, Lexington, Massachusetts 02421, United States (Y.L.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Zhang</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew J. McRiner</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: Auron Therapeutics, Wellesley, Massachusetts 02481, United States (B.C., A.J.M.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony D. Keefe</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna Kohlmann</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: Black Diamond Therapeutics, Cambridge, Massachusetts 02142, United States (A.K.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew A. Clark</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John W. Cuozzo</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: ZebiAI Therapeutics, Waltham, Massachusetts 02453, United States (J.S.D., J.M.D., M.I.M., S.A.T., M.v.R., J.Z., Y.A.A., J.W.C.)</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6229-0395" title="Orcid link">http://orcid.org/0000-0002-6229-0395</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christelle Huguet</span> - <span class="hlFld-Affiliation affiliation">X-Chem Inc., 100 Beaver Street, Waltham, Massachusetts 02453, United States</span>; 
    <span>Present Address:
                        Present address: Ipsen, Cambridge, Massachusetts 02142, United States (C.H.)</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.S.D., E.C.Y.L., J.Z., A.J.M., Y.Z., and M.A.C. contributed to the design of compounds, the establishment of the E3 alkyne toolkit, and evaluation of results; J.M.D. optimized and ran the cell-based viability and degradation assays and other cell-based assays; A.C.L. ran some of the degradation assays; S.A.T. optimized and executed the gene expression analysis. D.G. and B.C. planned and performed the biochemical assays; A.D.K. designed and M.I.M.  performed the selections; B.L. and M.v.R. oversaw protein production and performed protein purifications; J.W.C. and C.H. helped plan the biology and in vivo experiments; A.K. and Y.A.A. performed the molecular docking; compounds were synthesized by F.Z., A.K., N.E.W., P.B.M., and Y.Y.; Y.L. planned and oversaw the synthesis of the small molecules; A.J.M. and E.C.Y.L. oversaw all outsourced ADME and PK work; J.S.D., E.C.Y.L., A.J.M., and S.A. planned the mouse xenograft efficacy study; S.A. planned and conceived all the biology experiments, oversaw the performance of the outsourced mouse xenograft efficacy study, and was the project lead. All authors helped in writing and reviewing of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>The work included in the manuscript was performed as part of internal drug discovery efforts at X-Chem Inc. No grant support was used.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors are current or former employees of X-Chem Inc., which conducted the work.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32238" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32238" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Authors would like to thank Marie-Aude Guié and Scott Lindberg for providing scientific database management and Surayya Sana, Masarah Taktak, and Autumn Jetson for compound management. Some of the small molecule synthesis was performed at BioDuro (China).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">ABBREVIATIONS</td></tr><tr><td class="NLM_term">ERα</td><td class="NLM_def"><p class="first last">estrogen receptor α</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau E3 ubiquitin ligase</p></td></tr><tr><td class="NLM_term">IAP</td><td class="NLM_def"><p class="first last">inhibitor of apoptosis protein</p></td></tr><tr><td class="NLM_term">DECL</td><td class="NLM_def"><p class="first last">DNA-encoded chemical library</p></td></tr><tr><td class="NLM_term">POI</td><td class="NLM_def"><p class="first last">protein of interest</p></td></tr><tr><td class="NLM_term">TGI</td><td class="NLM_def"><p class="first last">tumor growth inhibition</p></td></tr><tr><td class="NLM_term">ENRv1</td><td class="NLM_def"><p class="first last">enrichment metric</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34727" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34727" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soerjomataram, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torre, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span>. <i>CA Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.3322/caac.21492</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.3322%2Fcaac.21492" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30207593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&author=F.+Brayauthor=J.+Ferlayauthor=I.+Soerjomataramauthor=R.+L.+Siegelauthor=L.+A.+Torreauthor=A.+Jemal&title=Global+cancer+statistics+2018%3A+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+countries&doi=10.3322%2Fcaac.21492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span></div><div class="casAuthors">Bray Freddie; Ferlay Jacques; Soerjomataram Isabelle; Siegel Rebecca L; Torre Lindsey A; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">394-424</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This article provides a status report on the global burden of cancer worldwide using the GLOBOCAN 2018 estimates of cancer incidence and mortality produced by the International Agency for Research on Cancer, with a focus on geographic variability across 20 world regions.  There will be an estimated 18.1 million new cancer cases (17.0 million excluding nonmelanoma skin cancer) and 9.6 million cancer deaths (9.5 million excluding nonmelanoma skin cancer) in 2018.  In both sexes combined, lung cancer is the most commonly diagnosed cancer (11.6% of the total cases) and the leading cause of cancer death (18.4% of the total cancer deaths), closely followed by female breast cancer (11.6%), prostate cancer (7.1%), and colorectal cancer (6.1%) for incidence and colorectal cancer (9.2%), stomach cancer (8.2%), and liver cancer (8.2%) for mortality.  Lung cancer is the most frequent cancer and the leading cause of cancer death among males, followed by prostate and colorectal cancer (for incidence) and liver and stomach cancer (for mortality).  Among females, breast cancer is the most commonly diagnosed cancer and the leading cause of cancer death, followed by colorectal and lung cancer (for incidence), and vice versa (for mortality); cervical cancer ranks fourth for both incidence and mortality.  The most frequently diagnosed cancer and the leading cause of cancer death, however, substantially vary across countries and within each country depending on the degree of economic development and associated social and life style factors.  It is noteworthy that high-quality cancer registry data, the basis for planning and implementing evidence-based cancer control programs, are not available in most low- and middle-income countries.  The Global Initiative for Cancer Registry Development is an international partnership that supports better estimation, as well as the collection and use of local data, to prioritize and evaluate national cancer control efforts.  CA: A Cancer Journal for Clinicians 2018;0:1-31. © 2018 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaW5wv8ocuhLxanA7vODO6fW6udTcc2ebgtnTvvOkhp7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D&md5=48529f8f794092779d70a27eec5c9017</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21492%26sid%3Dliteratum%253Aachs%26aulast%3DBray%26aufirst%3DF.%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DSoerjomataram%26aufirst%3DI.%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DTorre%26aufirst%3DL.%2BA.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DGlobal%2520cancer%2520statistics%25202018%253A%2520GLOBOCAN%2520estimates%2520of%2520incidence%2520and%2520mortality%2520worldwide%2520for%252036%2520cancers%2520in%2520185%2520countries%26jtitle%3DCA%2520Cancer%2520J.%2520Clin.%26date%3D2018%26volume%3D68%26spage%3D394%26epage%3D424%26doi%3D10.3322%2Fcaac.21492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siersbaek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span> <span> </span><span class="NLM_article-title">Signaling pathways and steroid receptors modulating estrogen receptor alpha function in breast cancer</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1141</span>– <span class="NLM_lpage">1154</span>, <span class="refDoi"> DOI: 10.1101/gad.316646.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1101%2Fgad.316646.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30181360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVKjtr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=1141-1154&author=R.+Siersbaekauthor=S.+Kumarauthor=J.+S.+Carroll&title=Signaling+pathways+and+steroid+receptors+modulating+estrogen+receptor+alpha+function+in+breast+cancer&doi=10.1101%2Fgad.316646.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Signaling pathways and steroid receptors modulating estrogen receptor α function in breast cancer</span></div><div class="casAuthors">Siersbaek, Rasmus; Kumar, Sanjeev; Carroll, Jason S.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">17-18</span>),
    <span class="NLM_cas:pages">1141-1154</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Estrogen receptor α(ER) is the major driver of ∼75% of breast cancers, and multiple ER targeting drugs are routinely used clin. to treat patients with ER+ breast cancer.  However, many patients relapse on these targeted therapies and ultimately develop metastatic and incurable disease, and understanding the mechanisms leading to drug resistance is consequently of utmost importance.  It is now clear that, in addn. to estrogens, ER function is modulated by other steroid receptors and multiple signaling pathways (e.g., growth factor and cytokine signaling), and many of these pathways affect drug resistance and patient outcome.  Here, we review the mechanisms through which these pathways impact ER function and drug resistance as well as discuss the clin. implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou8_pK06gF4LVg90H21EOLACvtfcHk0ljsE1b-ZsedGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVKjtr3E&md5=d5dda5bf6f5ba7f2b578acc2e73733c4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1101%2Fgad.316646.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.316646.118%26sid%3Dliteratum%253Aachs%26aulast%3DSiersbaek%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26atitle%3DSignaling%2520pathways%2520and%2520steroid%2520receptors%2520modulating%2520estrogen%2520receptor%2520alpha%2520function%2520in%2520breast%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2018%26volume%3D32%26spage%3D1141%26epage%3D1154%26doi%3D10.1101%2Fgad.316646.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waks, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span> <span> </span><span class="NLM_article-title">Breast Cancer Treatment: A Review</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">288</span>– <span class="NLM_lpage">300</span>, <span class="refDoi"> DOI: 10.1001/jama.2018.19323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1001%2Fjama.2018.19323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30667505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvV2qt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2019&pages=288-300&author=A.+G.+Waksauthor=E.+P.+Winer&title=Breast+Cancer+Treatment%3A+A+Review&doi=10.1001%2Fjama.2018.19323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Breast cancer treatment: a review</span></div><div class="casAuthors">Waks, Adrienne G.; Winer, Eric P.</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">288-300</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">1538-3598</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">A review.  Importance Breast cancer will be diagnosed in 12% of women in the United States over the course of their lifetimes and more than 250 000 new cases of breast cancer were diagnosed in the United States in 2017.  This focuses on current approaches and evolving strategies for local and systemic therapy of breast cancer. observations Breast cancer is categorized into 3 major subtypes based on the presence or absence of mol. markers for estrogen or progesterone receptors and human epidermal growth factor 2 (ERBB2; formerly HER2): hormone receptor pos./ERBB2 neg. (70% of patients), ERBB2 pos. (15%-20%), and triple-neg. (tumors lacking all 3 std. mol. markers; 15%).  More than 90% of breast cancers are not metastatic at the time of diagnosis.  For people presenting without metastatic disease, therapeutic goals are tumor eradication and preventing recurrence.  Triple-neg. breast cancer is more likely to recur than the other 2 subtypes, with 85% 5-yr breast cancer-specific survival for stage I triple-neg. tumors vs 94% to 99% for hormone receptor pos. and ERBB2 pos.  Systemic therapy for nonmetastatic breast cancer is detd. by subtype: patients with hormone receptor-pos. tumors receive endocrine therapy, and a minority receive chemotherapy as well; patients with ERBB2-pos. tumors receive ERBB2-targeted antibody or small-mol. inhibitor therapy combined with chemotherapy; and patients with triple-neg. tumors receive chemotherapy alone.  Local therapy for all patients with nonmetastatic breast cancer consists of surgical resection, with consideration of postoperative radiation if lumpectomy is performed.  Increasingly, some systemic therapy is delivered before surgery.  Tailoring postoperative treatment based on preoperative treatment response is under investigation.  Metastatic breast cancer is treated according to subtype, with goals of prolonging life and palliating symptoms.  Median overall survival for metastatic triple-neg. breast cancer is approx. 1 yr vs approx. 5 years for the other 2 subtypes. conclusions and relevance Breast cancer consists of 3 major tumor subtypes categorized according to estrogen or progesterone receptor expression and ERBB2 gene amplification.  The 3 subtypes have distinct risk profiles and treatment strategies.  Optimal therapy for each patient depends on tumor subtype, anat. cancer stage, and patient preferences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriTXwcPIcZ_LVg90H21EOLACvtfcHk0ljsE1b-ZsedGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvV2qt7c%253D&md5=548a8f22dc89eb559f95d779b37f3a9d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1001%2Fjama.2018.19323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2018.19323%26sid%3Dliteratum%253Aachs%26aulast%3DWaks%26aufirst%3DA.%2BG.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26atitle%3DBreast%2520Cancer%2520Treatment%253A%2520A%2520Review%26jtitle%3DJAMA%26date%3D2019%26volume%3D321%26spage%3D288%26epage%3D300%26doi%3D10.1001%2Fjama.2018.19323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span> <span> </span><span class="NLM_article-title">Targeting estrogen receptor alpha for degradation with PROTACs: A promising approach to overcome endocrine resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>206</i></span>,  <span class="NLM_fpage">112689</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.ejmech.2020.112689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32829249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1GjsbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=206&publication_year=2020&pages=112689&author=X.+Linauthor=H.+Xiangauthor=G.+Luo&title=Targeting+estrogen+receptor+alpha+for+degradation+with+PROTACs%3A+A+promising+approach+to+overcome+endocrine+resistance&doi=10.1016%2Fj.ejmech.2020.112689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting estrogen receptor α for degradation with PROTACs: A promising approach to overcome endocrine resistance</span></div><div class="casAuthors">Lin, Xin; Xiang, Hua; Luo, Guoshun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">206</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112689</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Estrogen receptor alfa (ERα) is expressed in approx. 70% of breast cancer (BC) which is the most common malignancy in women worldwide.  To date, the foremost intervention in the treatment of ER pos. (ER+) BC is still the endocrine therapy.  However, resistance to endocrine therapies remains a major hurdle in the long-term management of ER + BC.  Although the mechanisms underlying endocrine resistance are complex, cumulative evidence revealed that ERα still plays a crit. role in driving BC tumor cells to grow in resistance state.  Fulvestrant, a selective estrogen receptor degrader (SERD), has moved to first line therapy for metastatic ER + BC, suggesting that removing ERα would be a useful strategy to overcome endocrine resistance.  Proteolysis-Targeting Chimera (PROTAC) technol., an emerging paradigm for protein degrdn., has the potential to eliminate both wild type and mutant ERα in breast cancer cells.  Excitingly, ARV-471, an ERα-targeted PROTAC developed by Arvinas, has been in phase 1 clin. trials.  In this review, we will summarize recent progress of ER-targeting PROTACs from publications and patents along with their therapeutic opportunities for the treatment of endocrine-resistant BC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL6a9CQdHKErVg90H21EOLACvtfcHk0ljsE1b-ZsedGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1GjsbjK&md5=522029b5f85322bbe26f3e307acd51db</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112689%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DXiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DG.%26atitle%3DTargeting%2520estrogen%2520receptor%2520alpha%2520for%2520degradation%2520with%2520PROTACs%253A%2520A%2520promising%2520approach%2520to%2520overcome%2520endocrine%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D206%26spage%3D112689%26doi%3D10.1016%2Fj.ejmech.2020.112689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shagufta</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, S.</span></span> <span> </span><span class="NLM_article-title">Recent progress in selective estrogen receptor downregulators (SERDs) for the treatment of breast cancer</span>. <i>RSC Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">454</span>, <span class="refDoi"> DOI: 10.1039/C9MD00570F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1039%2FC9MD00570F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktl2qsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=438-454&author=+Shaguftaauthor=I.+Ahmadauthor=S.+Mathewauthor=S.+Rahman&title=Recent+progress+in+selective+estrogen+receptor+downregulators+%28SERDs%29+for+the+treatment+of+breast+cancer&doi=10.1039%2FC9MD00570F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in selective estrogen receptor downregulators (SERDs) for the treatment of breast cancer</span></div><div class="casAuthors">Shagufta; Ahmad, Irshad; Mathew, Shimy; Rahman, Sofia</div><div class="citationInfo"><span class="NLM_cas:title">RSC Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">438-454</span>CODEN:
                <span class="NLM_cas:coden">RMCSEZ</span>;
        ISSN:<span class="NLM_cas:issn">2632-8682</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Selective estrogen receptor downregulators (SERDs) are a novel class of compds. capable of reducing the ERα protein level and blocking ER activity.  Therefore, SERDs are considered as a significant therapeutic approach to treat ER+ breast cancer in both early stage and more advanced drug-resistant cases.  After the FDA approval of a steroidal drug, fulvestrant, as a SERD for the treatment of breast cancer in patients who have progressed on antihormonal agents, several mols. with diverse chem. structures have been rapidly developed, studied and evaluated for selective estrogen receptor downregulation activity.  Here we compile the promising SERDs reported in recent years and discuss the chem. structure and pharmacol. profile of the most potent compd. of the considered series.  Because of the availability of only a limited no. of effective drugs for the treatment of breast cancer, the quest for a potent SERD with respectable activity and bioavailability is still ongoing.  The goal of this article is to make available to the reader an overview of the current progress in SERDs and provide clues for the future discovery and development of novel pharmacol. potent SERDs for the treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSZ-tHcNc5-LVg90H21EOLACvtfcHk0ljsE1b-ZsedGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktl2qsrY%253D&md5=73ad3f6be94f660cf47b13146781aa04</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2FC9MD00570F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00570F%26sid%3Dliteratum%253Aachs%26aulast%3DShagufta%26aulast%3DAhmad%26aufirst%3DI.%26aulast%3DMathew%26aufirst%3DS.%26aulast%3DRahman%26aufirst%3DS.%26atitle%3DRecent%2520progress%2520in%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%2520for%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DRSC%2520Med.%2520Chem.%26date%3D2020%26volume%3D11%26spage%3D438%26epage%3D454%26doi%3D10.1039%2FC9MD00570F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gombos, A.</span></span> <span> </span><span class="NLM_article-title">Selective oestrogen receptor degraders in breast cancer: a review and perspectives</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1097/CCO.0000000000000567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1097%2FCCO.0000000000000567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31335829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1alsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=424-429&author=A.+Gombos&title=Selective+oestrogen+receptor+degraders+in+breast+cancer%3A+a+review+and+perspectives&doi=10.1097%2FCCO.0000000000000567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective oestrogen receptor degraders in breast cancer: a review and perspectives</span></div><div class="casAuthors">Gombos, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">424-429</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: Estrogen receptor-pos. breast cancer accounts for 70% of all breast cancers.  Sequential endocrine treatment in monotherapy or in combination with CDK 4/6 or m-TOR inhibitors is the mainstay of recommended treatment options in the management of metastatic breast cancer even in the presence of visceral metastasis.  There is an emerging need to address endocrine resistance, which despite highly efficacious treatment combinations still can develop.  Recent findings: One of the mechanisms of endocrine resistance is mol. alteration of the estrogen receptor itself, such as ESR1 mutations affecting the ligand-binding domain.  These mutations emerge under the selective pressure of aromatase inhibitors.  The efficacy of selective estrogen receptor degraders (SERDs) might not be affected by the presence of mol. alterations of estrogen receptor.  Fulvestrant is the only SERD used in current clin. practice.  Numerous novel, nonsteroidal orally available SERDs are currently in clin. development.  Efficacious estrogen receptor target engagement and promising clin. activity was shown in early phase clin. trials.  Summary: Therefore, a new class of orally available nonsteroidal SERDs gains high interest in tackling endocrine resistance in estrogen receptor-pos. (ER+) advanced breast cancer.  Clin. efficacy needs to be confirmed in larger patient populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9D3cmdEVhP7Vg90H21EOLACvtfcHk0ljgDMVQmQZJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1alsrbL&md5=45f46bc71806fb9e79e642c0e9307191</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1097%2FCCO.0000000000000567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0000000000000567%26sid%3Dliteratum%253Aachs%26aulast%3DGombos%26aufirst%3DA.%26atitle%3DSelective%2520oestrogen%2520receptor%2520degraders%2520in%2520breast%2520cancer%253A%2520a%2520review%2520and%2520perspectives%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2019%26volume%3D31%26spage%3D424%26epage%3D429%26doi%3D10.1097%2FCCO.0000000000000567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gombos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzoi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awada, A.</span></span> <span> </span><span class="NLM_article-title">Investigational drugs in early stage clinical trials for the treatment of HER2+ breast cancer</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">617</span>– <span class="NLM_lpage">627</span>, <span class="refDoi"> DOI: 10.1080/13543784.2019.1633306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1080%2F13543784.2019.1633306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31230485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1emsL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=617-627&author=A.+Gombosauthor=M.+A.+Franzoiauthor=A.+Awada&title=Investigational+drugs+in+early+stage+clinical+trials+for+the+treatment+of+HER2%2B+breast+cancer&doi=10.1080%2F13543784.2019.1633306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Investigational drugs in early stage clinical trials for the treatment of HER2+ breast cancer</span></div><div class="casAuthors">Gombos, Andrea; Franzoi, Maria Alice; Awada, Ahmad</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">617-627</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : Despite improvements in the management of HER2+ breast cancer, metastatic disease is still fatal.  Usually, these patients receive several lines of chemotherapy assocd. with HER2 targeted treatments.  Most of the trials using innovative approaches are positioning themselves in disease that is resistant to pertuzumab and trastuzumab emtansine (TDM1).: We describe the recent advances in clin. development of anti-HER2 treatments.  To this aim, we used literature search via Pubmed and made an inventory of abstrs. published during the last two years in major oncol. conferences.: Further changes will probably occur during the next decade in the management of metastatic HER2-pos. breast cancer.  This is mainly driven by the fact that the two mainstay drugs (pertuzumab and TDM-1) that confer prolonged survival (56 mo) to these patients are currently being used in the treatment of early-stage disease in a subset of patients.  Thus, there is an urgent need to develop new, innovative approaches in those patients whose disease has become resistant to these highly potent drugs.  Several new antibody-drug conjugates, bispecific antibodies or new generation tyrosine kinase inhibitor (TKIs) hold promise and should be assessed and compared with drugs currently used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0xmhTyI5v4bVg90H21EOLACvtfcHk0ljgDMVQmQZJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1emsL3M&md5=0b1e769bd87571565526d06d91b7e06b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1080%2F13543784.2019.1633306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543784.2019.1633306%26sid%3Dliteratum%253Aachs%26aulast%3DGombos%26aufirst%3DA.%26aulast%3DFranzoi%26aufirst%3DM.%2BA.%26aulast%3DAwada%26aufirst%3DA.%26atitle%3DInvestigational%2520drugs%2520in%2520early%2520stage%2520clinical%2520trials%2520for%2520the%2520treatment%2520of%2520HER2%252B%2520breast%2520cancer%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2019%26volume%3D28%26spage%3D617%26epage%3D627%26doi%3D10.1080%2F13543784.2019.1633306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nayar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapstad, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuoco, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waks, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wander, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persky, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helvie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenblatt-Rosen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, N. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagle, N.</span></span> <span> </span><span class="NLM_article-title">Acquired HER2 mutations in ER(+) metastatic breast cancer confer resistance to estrogen receptor-directed therapies</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">216</span>, <span class="refDoi"> DOI: 10.1038/s41588-018-0287-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fs41588-018-0287-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30531871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhtbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=207-216&author=U.+Nayarauthor=O.+Cohenauthor=C.+Kapstadauthor=M.+S.+Cuocoauthor=A.+G.+Waksauthor=S.+A.+Wanderauthor=C.+Painterauthor=S.+Freemanauthor=N.+S.+Perskyauthor=L.+Mariniauthor=K.+Helvieauthor=N.+Oliverauthor=O.+Rozenblatt-Rosenauthor=C.+X.+Maauthor=A.+Regevauthor=E.+P.+Winerauthor=N.+U.+Linauthor=N.+Wagle&title=Acquired+HER2+mutations+in+ER%28%2B%29+metastatic+breast+cancer+confer+resistance+to+estrogen+receptor-directed+therapies&doi=10.1038%2Fs41588-018-0287-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired HER2 mutations in ER+ metastatic breast cancer confer resistance to estrogen receptor-directed therapies</span></div><div class="casAuthors">Nayar, Utthara; Cohen, Ofir; Kapstad, Christian; Cuoco, Michael S.; Waks, Adrienne G.; Wander, Seth A.; Painter, Corrie; Freeman, Samuel; Persky, Nicole S.; Marini, Lori; Helvie, Karla; Oliver, Nelly; Rozenblatt-Rosen, Orit; Ma, Cynthia X.; Regev, Aviv; Winer, Eric P.; Lin, Nancy U.; Wagle, Nikhil</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">207-216</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Seventy percent of breast cancers express the estrogen receptor (ER), and agents that target the ER are the mainstay of treatment.  However, virtually all people with ER+ breast cancer develop resistance to ER-directed agents in the metastatic setting.  Beyond mutations in the ER itself, which occur in 25-30% of people treated with aromatase inhibitors1-4, knowledge about clin. resistance mechanisms remains incomplete.  We identified activating HER2 mutations in metastatic biopsies from eight patients with ER+ metastatic breast cancer who had developed resistance to aromatase inhibitors, tamoxifen or fulvestrant.  Examn. of treatment-naive primary tumors in five patients showed no evidence of pre-existing mutations in four of five patients, suggesting that these mutations were acquired under the selective pressure of ER-directed therapy.  The HER2 mutations and ER mutations were mutually exclusive, suggesting a distinct mechanism of acquired resistance to ER-directed therapies.  In vitro anal. confirmed that the HER2 mutations conferred estrogen independence as well as-in contrast to ER mutations-resistance to tamoxifen, fulvestrant and the CDK4 and CDK6 inhibitor palbociclib.  Resistance was overcome by combining ER-directed therapy with the irreversible HER2 kinase inhibitor neratinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPW14nLf7qarVg90H21EOLACvtfcHk0ljgDMVQmQZJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhtbbM&md5=8c9f16d4a94653efd684fd43f356cd16</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41588-018-0287-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-018-0287-5%26sid%3Dliteratum%253Aachs%26aulast%3DNayar%26aufirst%3DU.%26aulast%3DCohen%26aufirst%3DO.%26aulast%3DKapstad%26aufirst%3DC.%26aulast%3DCuoco%26aufirst%3DM.%2BS.%26aulast%3DWaks%26aufirst%3DA.%2BG.%26aulast%3DWander%26aufirst%3DS.%2BA.%26aulast%3DPainter%26aufirst%3DC.%26aulast%3DFreeman%26aufirst%3DS.%26aulast%3DPersky%26aufirst%3DN.%2BS.%26aulast%3DMarini%26aufirst%3DL.%26aulast%3DHelvie%26aufirst%3DK.%26aulast%3DOliver%26aufirst%3DN.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DMa%26aufirst%3DC.%2BX.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26aulast%3DLin%26aufirst%3DN.%2BU.%26aulast%3DWagle%26aufirst%3DN.%26atitle%3DAcquired%2520HER2%2520mutations%2520in%2520ER%2528%252B%2529%2520metastatic%2520breast%2520cancer%2520confer%2520resistance%2520to%2520estrogen%2520receptor-directed%2520therapies%26jtitle%3DNat.%2520Genet.%26date%3D2019%26volume%3D51%26spage%3D207%26epage%3D216%26doi%3D10.1038%2Fs41588-018-0287-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J.-A.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptor mutations and functional consequences for breast cancer</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">476</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2015.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.tem.2015.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=26183887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeisLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=467-476&author=C.+Thomasauthor=J.-A.+Gustafsson&title=Estrogen+receptor+mutations+and+functional+consequences+for+breast+cancer&doi=10.1016%2Fj.tem.2015.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen receptor mutations and functional consequences for breast cancer</span></div><div class="casAuthors">Thomas, Christoforos; Gustafsson, Jan-Ake</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">467-476</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A significant no. of estrogen receptor α (ERα)-pos. breast tumors develop resistance to endocrine therapy and recur with metastatic disease.  Several mechanisms of endocrine resistance have been proposed, including genetic alterations that lead to ERs with altered protein sequence.  By altering the conformation of the protein and increasing the interaction with coactivators, point mutations in ESR1, the gene encoding ERα, promote an active form of the receptor in the absence of hormone that assists tumor cells to evade hormonal treatments.  Recent studies have confirmed that ESR1 point mutations frequently occur in metastatic breast tumors that are refractory to endocrine therapy, and suggest the development of novel strategies that may be more effective in controlling ER signaling and benefit patients with recurrent and metastatic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_CLOzUVLBLVg90H21EOLACvtfcHk0lipJY6n0TZCdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeisLfI&md5=d6b1f2fe49c77b991f953e802514a32e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2015.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2015.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DC.%26aulast%3DGustafsson%26aufirst%3DJ.-A.%26atitle%3DEstrogen%2520receptor%2520mutations%2520and%2520functional%2520consequences%2520for%2520breast%2520cancer%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2015%26volume%3D26%26spage%3D467%26epage%3D476%26doi%3D10.1016%2Fj.tem.2015.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alluri, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptor mutations and their role in breast cancer progression</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">494</span>, <span class="refDoi"> DOI: 10.1186/s13058-014-0494-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1186%2Fs13058-014-0494-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=25928204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=494&author=P.+G.+Alluriauthor=C.+Speersauthor=A.+M.+Chinnaiyan&title=Estrogen+receptor+mutations+and+their+role+in+breast+cancer+progression&doi=10.1186%2Fs13058-014-0494-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen receptor mutations and their role in breast cancer progression</span></div><div class="casAuthors">Alluri, Prasanna G.; Speers, Corey; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">494/1-494/8, 8 pp.</span>CODEN:
                <span class="NLM_cas:coden">BRCRFS</span>;
        ISSN:<span class="NLM_cas:issn">1465-542X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Endocrine therapy is the mainstay of treatment in estrogen receptor-pos. breast cancers and significantly reduces disease recurrence and breast cancer-related mortality.  However, acquired resistance to therapy has been noted in nearly one-third of women treated with tamoxifen and other endocrine therapies.  Mutations in the estrogen receptor have long been speculated to play a role in endocrine therapy resistance but have been rarely detected.  However, recent studies utilizing next-generation sequencing on estrogen receptor-pos., metastatic clin. samples have revealed that recurrent ESR1 mutations are far more frequent than previously thought and may play an important role in acquired endocrine therapy resistance.  Here we review recent advances in detection and characterization of ESR1 mutations in advanced, endocrine therapy-resistant breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokrNygDcI-MbVg90H21EOLACvtfcHk0lipJY6n0TZCdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislWntr0%253D&md5=fae44eb9eba40ff0be77e776edc5633b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1186%2Fs13058-014-0494-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13058-014-0494-7%26sid%3Dliteratum%253Aachs%26aulast%3DAlluri%26aufirst%3DP.%2BG.%26aulast%3DSpeers%26aufirst%3DC.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DEstrogen%2520receptor%2520mutations%2520and%2520their%2520role%2520in%2520breast%2520cancer%2520progression%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2014%26volume%3D16%26spage%3D494%26doi%3D10.1186%2Fs13058-014-0494-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merenbakh-Lamin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Baruch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeheskel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvir, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soussan-Gutman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeselsohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yelensky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarid, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, I.</span></span> <span> </span><span class="NLM_article-title">D538G mutation in estrogen receptor-α: A novel mechanism for acquired endocrine resistance in breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6856</span>– <span class="NLM_lpage">6864</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1158%2F0008-5472.CAN-13-1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=24217577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKhurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6856-6864&author=K.+Merenbakh-Laminauthor=N.+Ben-Baruchauthor=A.+Yeheskelauthor=A.+Dvirauthor=L.+Soussan-Gutmanauthor=R.+Jeselsohnauthor=R.+Yelenskyauthor=M.+Brownauthor=V.+A.+Millerauthor=D.+Saridauthor=S.+Rizelauthor=B.+Kleinauthor=T.+Rubinekauthor=I.+Wolf&title=D538G+mutation+in+estrogen+receptor-%CE%B1%3A+A+novel+mechanism+for+acquired+endocrine+resistance+in+breast+cancer&doi=10.1158%2F0008-5472.CAN-13-1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">D538G Mutation in Estrogen Receptor-α: A Novel Mechanism for Acquired Endocrine Resistance in Breast Cancer</span></div><div class="casAuthors">Merenbakh-Lamin, Keren; Ben-Baruch, Noa; Yeheskel, Adva; Dvir, Addie; Soussan-Gutman, Lior; Jeselsohn, Rinath; Yelensky, Roman; Brown, Myles; Miller, Vincent A.; Sarid, David; Rizel, Shulamith; Klein, Baruch; Rubinek, Tami; Wolf, Ido</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6856-6864</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to endocrine therapy occurs in virtually all patients with estrogen receptor a (ERa)-pos. metastatic breast cancer, and is attributed to various mechanisms including loss of ERα expression, altered activity of coregulators, and cross-talk between the ERa and growth factor signaling pathways.  To our knowledge, acquired mutations of the ERα have not been described as mediating endocrine resistance.  Samples of 13 patients with metastatic breast cancer were analyzed for mutations in cancer-related genes.  In five patients who developed resistance to hormonal therapy, a mutation of A to G at position 1,613 of ERα, resulting in a substitution of aspartic acid at position 538 to glycine (D538G), was identified in liver metastases.  Importantly, the mutation was not detected in the primary tumors obtained prior to endocrine treatment.  Structural modeling indicated that D538G substitution leads to a conformational change in the ligand-binding domain, which mimics the conformation of activated ligand-bound receptor and alters binding of tamoxifen.  Indeed, expts. in breast cancer cells indicated constitutive, ligand-independent transcriptional activity of the D538G receptor, and overexpression of it enhanced proliferation and conferred resistance to tamoxifen.  These data indicate a novel mechanism of acquired endocrine resistance in breast cancer.  Further studies are needed to assess the frequency of D538G-ERα among patients with breast cancer and explore ways to inhibit its activity and restore endocrine sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWljQ_odNNUrVg90H21EOLACvtfcHk0lipJY6n0TZCdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKhurfN&md5=6dca8ee6cdb77bfc8a94d02fe5630ea8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1197%26sid%3Dliteratum%253Aachs%26aulast%3DMerenbakh-Lamin%26aufirst%3DK.%26aulast%3DBen-Baruch%26aufirst%3DN.%26aulast%3DYeheskel%26aufirst%3DA.%26aulast%3DDvir%26aufirst%3DA.%26aulast%3DSoussan-Gutman%26aufirst%3DL.%26aulast%3DJeselsohn%26aufirst%3DR.%26aulast%3DYelensky%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DSarid%26aufirst%3DD.%26aulast%3DRizel%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DB.%26aulast%3DRubinek%26aufirst%3DT.%26aulast%3DWolf%26aufirst%3DI.%26atitle%3DD538G%2520mutation%2520in%2520estrogen%2520receptor-%25CE%25B1%253A%2520A%2520novel%2520mechanism%2520for%2520acquired%2520endocrine%2520resistance%2520in%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6856%26epage%3D6864%26doi%3D10.1158%2F0008-5472.CAN-13-1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vats, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonigro, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyana-Sundaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlins, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roychowdhury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pienta, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rae, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Poznak, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chugh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunju, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schott, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Activating ESR1 mutations in hormone-resistant metastatic breast cancer</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1446</span>– <span class="NLM_lpage">1451</span>, <span class="refDoi"> DOI: 10.1038/ng.2823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fng.2823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=24185510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWju7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=1446-1451&author=D.+R.+Robinsonauthor=Y.+M.+Wuauthor=P.+Vatsauthor=F.+Suauthor=R.+J.+Lonigroauthor=X.+Caoauthor=S.+Kalyana-Sundaramauthor=R.+Wangauthor=Y.+Ningauthor=L.+Hodgesauthor=A.+Gurskyauthor=J.+Siddiquiauthor=S.+A.+Tomlinsauthor=S.+Roychowdhuryauthor=K.+J.+Pientaauthor=S.+Y.+Kimauthor=J.+S.+Robertsauthor=J.+M.+Raeauthor=C.+H.+Van+Poznakauthor=D.+F.+Hayesauthor=R.+Chughauthor=L.+P.+Kunjuauthor=M.+Talpazauthor=A.+F.+Schottauthor=A.+M.+Chinnaiyan&title=Activating+ESR1+mutations+in+hormone-resistant+metastatic+breast+cancer&doi=10.1038%2Fng.2823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Activating ESR1 mutations in hormone-resistant metastatic breast cancer</span></div><div class="casAuthors">Robinson, Dan R.; Wu, Yi-Mi; Vats, Pankaj; Su, Fengyun; Lonigro, Robert J.; Cao, Xuhong; Kalyana-Sundaram, Shanker; Wang, Rui; Ning, Yu; Hodges, Lynda; Gursky, Amy; Siddiqui, Javed; Tomlins, Scott A.; Roychowdhury, Sameek; Pienta, Kenneth J.; Kim, Scott Y.; Roberts, J. Scott; Rae, James M.; Van Poznak, Catherine H.; Hayes, Daniel F.; Chugh, Rashmi; Kunju, Lakshmi P.; Talpaz, Moshe; Schott, Anne F.; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1446-1451</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Breast cancer is the most prevalent cancer in women, and over two-thirds of cases express estrogen receptor-α (ER-α, encoded by ESR1).  Through a prospective clin. sequencing program for advanced cancers, we enrolled 11 patients with ER-pos. metastatic breast cancer.  Whole-exome and transcriptome anal. showed that six cases harbored mutations of ESR1 affecting its ligand-binding domain (LBD), all of whom had been treated with anti-estrogens and estrogen deprivation therapies.  A survey of The Cancer Genome Atlas (TCGA) identified four endometrial cancers with similar mutations of ESR1.  The five new LBD-localized ESR1 mutations identified here (encoding p.Leu536Gln, p.Tyr537Ser, p.Tyr537Cys, p.Tyr537Asn and p.Asp538Gly) were shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.  Taken together, these studies suggest that activating mutations in ESR1 are a key mechanism in acquired endocrine resistance in breast cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtQSL6mlhMOrVg90H21EOLACvtfcHk0lj1ao10PFGF8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWju7%252FN&md5=af9e69d6064726af238fe74dad7d2204</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fng.2823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2823%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DD.%2BR.%26aulast%3DWu%26aufirst%3DY.%2BM.%26aulast%3DVats%26aufirst%3DP.%26aulast%3DSu%26aufirst%3DF.%26aulast%3DLonigro%26aufirst%3DR.%2BJ.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DKalyana-Sundaram%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DHodges%26aufirst%3DL.%26aulast%3DGursky%26aufirst%3DA.%26aulast%3DSiddiqui%26aufirst%3DJ.%26aulast%3DTomlins%26aufirst%3DS.%2BA.%26aulast%3DRoychowdhury%26aufirst%3DS.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DRoberts%26aufirst%3DJ.%2BS.%26aulast%3DRae%26aufirst%3DJ.%2BM.%26aulast%3DVan%2BPoznak%26aufirst%3DC.%2BH.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChugh%26aufirst%3DR.%26aulast%3DKunju%26aufirst%3DL.%2BP.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DSchott%26aufirst%3DA.%2BF.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DActivating%2520ESR1%2520mutations%2520in%2520hormone-resistant%2520metastatic%2520breast%2520cancer%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D1446%26epage%3D1451%26doi%3D10.1038%2Fng.2823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razavi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goeppert, A. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Stanchina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uddin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzenellenbogen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandarlapaty, S.</span></span> <span> </span><span class="NLM_article-title">Activating ESR1 Mutations Differentially Affect the Efficacy of ER Antagonists</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1158%2F2159-8290.CD-15-1523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=27986707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Klsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=277-287&author=W.+Toyauthor=H.+Weirauthor=P.+Razaviauthor=M.+Lawsonauthor=A.+U.+Goeppertauthor=A.+M.+Mazzolaauthor=A.+Smithauthor=J.+Wilsonauthor=C.+Morrowauthor=W.+L.+Wongauthor=E.+De+Stanchinaauthor=K.+E.+Carlsonauthor=T.+S.+Martinauthor=S.+Uddinauthor=Z.+Liauthor=S.+Fanningauthor=J.+A.+Katzenellenbogenauthor=G.+Greeneauthor=J.+Baselgaauthor=S.+Chandarlapaty&title=Activating+ESR1+Mutations+Differentially+Affect+the+Efficacy+of+ER+Antagonists&doi=10.1158%2F2159-8290.CD-15-1523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Activating ESR1 Mutations Differentially Affect the Efficacy of ER Antagonists</span></div><div class="casAuthors">Toy, Weiyi; Weir, Hazel; Razavi, Pedram; Lawson, Mandy; Goeppert, Anne U.; Mazzola, Anne Marie; Smith, Aaron; Wilson, Joanne; Morrow, Christopher; Wong, Wai Lin; De Stanchina, Elisa; Carlson, Kathryn E.; Martin, Teresa S.; Uddin, Sharmeen; Li, Zhiqiang; Fanning, Sean; Katzenellenbogen, John A.; Greene, Geoffrey; Baselga, Jose; Chandarlapaty, Sarat</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-287</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Recent studies have identified somatic ESR1 mutations in patients with metastatic breast cancer and found some of them to promote estrogen-independent activation of the receptor.  The degree to which all recurrent mutants can drive estrogen-independent activities and reduced sensitivity to ER antagonists like fulvestrant is not established.  In this report, we characterize the spectrum of ESR1 mutations from more than 900 patients.  ESR1 mutations were detected in 10%, with D538G being the most frequent (36%), followed by Y537S (14%).  Several novel, activating mutations were also detected (e.g., L469V, V422del, and Y537D).  Although many mutations lead to constitutive activity and reduced sensitivity to ER antagonists, only select mutants such as Y537S caused a magnitude of change assocd. with fulvestrant resistance in vivo.  Correspondingly, tumors driven by Y537S, but not D5358G, E380Q, or S463P, were less effectively inhibited by fulvestrant than more potent and bioavailable antagonists, including AZD9496.  These data point to a need for antagonists with optimal pharmacokinetic properties to realize clin. efficacy against certain ESR1 mutants.  Significance: A diversity of activating ESR1 mutations exist, only some of which confer resistance to existing ER antagonists that might be overcome by next-generation inhibitors such as AZD9496.  Cancer Discov; 7(3); 277-87. ©2016 AACR.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZyS-eS6T9ZLVg90H21EOLACvtfcHk0lj1ao10PFGF8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Klsbo%253D&md5=398dd5202db358ba8facab67055895c9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1523%26sid%3Dliteratum%253Aachs%26aulast%3DToy%26aufirst%3DW.%26aulast%3DWeir%26aufirst%3DH.%26aulast%3DRazavi%26aufirst%3DP.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DGoeppert%26aufirst%3DA.%2BU.%26aulast%3DMazzola%26aufirst%3DA.%2BM.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DMorrow%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DW.%2BL.%26aulast%3DDe%2BStanchina%26aufirst%3DE.%26aulast%3DCarlson%26aufirst%3DK.%2BE.%26aulast%3DMartin%26aufirst%3DT.%2BS.%26aulast%3DUddin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DFanning%26aufirst%3DS.%26aulast%3DKatzenellenbogen%26aufirst%3DJ.%2BA.%26aulast%3DGreene%26aufirst%3DG.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DChandarlapaty%26aufirst%3DS.%26atitle%3DActivating%2520ESR1%2520Mutations%2520Differentially%2520Affect%2520the%2520Efficacy%2520of%2520ER%2520Antagonists%26jtitle%3DCancer%2520Discovery%26date%3D2017%26volume%3D7%26spage%3D277%26epage%3D287%26doi%3D10.1158%2F2159-8290.CD-15-1523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pooh, A.</span></span> <span> </span><span class="NLM_article-title">Proof-of-Concept with PROTACs in Prostate Cancer</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1084</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=1084&author=A.+Pooh&title=Proof-of-Concept+with+PROTACs+in+Prostate+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPooh%26aufirst%3DA.%26atitle%3DProof-of-Concept%2520with%2520PROTACs%2520in%2520Prostate%2520Cancer%26jtitle%3DCancer%2520Discovery%26date%3D2020%26volume%3D10%26spage%3D1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC Technology: Opportunities and Challenges</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=237-240&author=H.+Gaoauthor=X.+Sunauthor=Y.+Rao&title=PROTAC+Technology%3A+Opportunities+and+Challenges&doi=10.1021%2Facsmedchemlett.9b00597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC Technology: Opportunities and Challenges</span></div><div class="casAuthors">Gao Hongying; Sun Xiuyun; Rao Yu; Gao Hongying; Sun Xiuyun</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">237-240</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">PROTACs-induced targeted protein degradation has emerged as a novel therapeutic strategy in drug development and attracted the favor of academic institutions, large pharmaceutical enterprises (e.g., AstraZeneca, Bayer, Novartis, Amgen, Pfizer, GlaxoSmithKline, Merck, and Boehringer Ingelheim, etc.), and biotechnology companies.  PROTACs opened a new chapter for novel drug development.  However, any new technology will face many new problems and challenges.  Perspectives on the potential opportunities and challenges of PROTACs will contribute to the research and development of new protein degradation drugs and degrader tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSj8qlWczrRKmx8TEN9K18QfW6udTcc2eYPs7URDsL8Yrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D&md5=68988d6477e83f4ebc26d11e3b8ce5b1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00597%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC%2520Technology%253A%2520Opportunities%2520and%2520Challenges%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D237%26epage%3D240%26doi%3D10.1021%2Facsmedchemlett.9b00597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=64&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2eYPs7URDsL8Yrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D64%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimera (PROTAC) for targeted protein degradation and cancer therapy</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">50</span>, <span class="refDoi"> DOI: 10.1186/s13045-020-00885-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1186%2Fs13045-020-00885-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32404196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB38vmsFGntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2020&pages=50&author=X.+Liauthor=Y.+Song&title=Proteolysis-targeting+chimera+%28PROTAC%29+for+targeted+protein+degradation+and+cancer+therapy&doi=10.1186%2Fs13045-020-00885-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-targeting chimera (PROTAC) for targeted protein degradation and cancer therapy</span></div><div class="casAuthors">Li Xin; Song Yongcheng; Song Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteolysis-targeting chimera (PROTAC) has been developed to be a useful technology for targeted protein degradation.  A bifunctional PROTAC molecule consists of a ligand (mostly small-molecule inhibitor) of the protein of interest (POI) and a covalently linked ligand of an E3 ubiquitin ligase (E3).  Upon binding to the POI, the PROTAC can recruit E3 for POI ubiquitination, which is subjected to proteasome-mediated degradation.  PROTAC complements nucleic acid-based gene knockdown/out technologies for targeted protein reduction and could mimic pharmacological protein inhibition.  To date, PROTACs targeting ~ 50 proteins, many of which are clinically validated drug targets, have been successfully developed with several in clinical trials for cancer therapy.  This article reviews PROTAC-mediated degradation of critical oncoproteins in cancer, particularly those in hematological malignancies.  Chemical structures, cellular and in vivo activities, pharmacokinetics, and pharmacodynamics of these PROTACs are summarized.  In addition, potential advantages, challenges, and perspectives of PROTAC technology in cancer therapy are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxVqxtAax1Ha-9TkgmBqTqfW6udTcc2eYPs7URDsL8Yrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38vmsFGntA%253D%253D&md5=baf560d35ef911d2be3d8b4c6d4e0191</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2Fs13045-020-00885-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-020-00885-3%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DProteolysis-targeting%2520chimera%2520%2528PROTAC%2529%2520for%2520targeted%2520protein%2520degradation%2520and%2520cancer%2520therapy%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2020%26volume%3D13%26spage%3D50%26doi%3D10.1186%2Fs13045-020-00885-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, H.</span></span> <span> </span><span class="NLM_article-title">Development of cell-permeable peptide-based PROTACs targeting estrogen receptor α</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>187</i></span>,  <span class="NLM_fpage">111967</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.ejmech.2019.111967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31865016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVKgtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2020&pages=111967&author=Y.+Daiauthor=N.+Yueauthor=J.+Gongauthor=C.+Liuauthor=Q.+Liauthor=J.+Zhouauthor=W.+Huangauthor=H.+Qian&title=Development+of+cell-permeable+peptide-based+PROTACs+targeting+estrogen+receptor+%CE%B1&doi=10.1016%2Fj.ejmech.2019.111967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of cell-permeable peptide-based PROTACs targeting estrogen receptor α</span></div><div class="casAuthors">Dai, Yuxuan; Yue, Na; Gong, Junni; Liu, Chunxia; Li, Qifei; Zhou, Jiaqi; Huang, Wenlong; Qian, Hai</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">187</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111967</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Proteolysis-targeting chimera (PROTAC) could selectively degrade target protein and may become a promising strategy for treating estrogen receptor α (ERα) pos. breast cancers.  Here, we designed penetrated peptide-based PROTACs by constructing an N-terminal lactam cyclic to improve proteolytic stability and cell penetration.  We used a lactam cyclic peptide as ERα binding ligand, 6-aminocaproic acid as a linker, and a hydroxylated pentapeptide structure for recruiting E3 ligase to obtain heterobifunctional compds.  The resulting optimized compd. I-6 selectively recruited ERα to the E3 ligase complex for promoting the degrdn. of ERα.  Compd. I-6 possessed strong effect on MCF-7 cell toxicity (IC50 ∼9.7μM) and significantly enhanced activities in inducing ERα degrdn.  Meanwhile, I-6 performed much stronger potency in inhibition of tumors growth than tamoxifen.  This work is a successful template to construct PROTACs based on cell-permeable peptides, which could extend the chem. space of PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxg4zaUgSlt7Vg90H21EOLACvtfcHk0lgmN3XluyvOuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVKgtL7F&md5=0bd840b597c93d7f06ef7384277beade</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111967%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DY.%26aulast%3DYue%26aufirst%3DN.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DQian%26aufirst%3DH.%26atitle%3DDevelopment%2520of%2520cell-permeable%2520peptide-based%2520PROTACs%2520targeting%2520estrogen%2520receptor%2520%25CE%25B1%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D187%26spage%3D111967%26doi%3D10.1016%2Fj.ejmech.2019.111967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span> <span> </span><span class="NLM_article-title">Targeting Nuclear Receptors with PROTAC degraders</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>493</i></span>,  <span class="NLM_fpage">110452</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2019.110452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.mce.2019.110452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31125586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFensL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=493&publication_year=2019&pages=110452&author=J.+J.+Flanaganauthor=T.+K.+Neklesa&title=Targeting+Nuclear+Receptors+with+PROTAC+degraders&doi=10.1016%2Fj.mce.2019.110452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Nuclear Receptors with PROTAC degraders</span></div><div class="casAuthors">Flanagan, John J.; Neklesa, Taavi K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">493</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">110452</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Nuclear receptors comprise a class of intracellular transcription factors whose major role is to act as sensors of various stimuli and to convert the external signal into a transcriptional output.  Nuclear receptors (NRs) achieve this by possessing a ligand binding domain, which can bind cell permeable agonists, a DNA-binding domain, which binds the upstream sequences of target genes, and a regulatory domain that recruits the transcriptional machinery.  The ligand binding alters the activation state of the NR, either by activating or inactivating its transcriptional output.  Given the central role of NRs in signal transduction, many currently approved therapeutics modulate the activity of NRs.  Here we discuss how PROTAC degraders afford a novel approach to abrogate the downstream signaling activity of NRs.  We highlight six broad functional reasons why PROTAC degraders are preferable to the classical ligand binding pocket antagonists, with specific examples provided for each category.  Lastly, as Androgen Receptor and Estrogen Receptor PROTAC degraders are being pursued as treatment for prostate cancer and breast cancer, resp., a rationale is provided for the translational utility for the degrdn. of these two NRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0pFL-LkrjELVg90H21EOLACvtfcHk0lhI3BkcqqqHOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFensL%252FN&md5=8988ee44083a37b65282af214a432d1e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2019.110452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2019.110452%26sid%3Dliteratum%253Aachs%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26atitle%3DTargeting%2520Nuclear%2520Receptors%2520with%2520PROTAC%2520degraders%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2019%26volume%3D493%26spage%3D110452%26doi%3D10.1016%2Fj.mce.2019.110452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.+Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+Highly+Potent+Proteolysis+Targeting+Chimera+%28PROTAC%29+Degrader+of+Estrogen+Receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lhI3BkcqqqHOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520Highly%2520Potent%2520Proteolysis%2520Targeting%2520Chimera%2520%2528PROTAC%2529%2520Degrader%2520of%2520Estrogen%2520Receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span> <span> </span><span class="NLM_article-title">Two-Stage Strategy for Development of Proteolysis Targeting Chimeras and its Application for Estrogen Receptor Degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1487</span>– <span class="NLM_lpage">1496</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00140</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFKru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1487-1496&author=B.+L.+Robertsauthor=Z.+X.+Maauthor=A.+Gaoauthor=E.+D.+Leistenauthor=D.+Yinauthor=W.+Xuauthor=W.+Tang&title=Two-Stage+Strategy+for+Development+of+Proteolysis+Targeting+Chimeras+and+its+Application+for+Estrogen+Receptor+Degraders&doi=10.1021%2Facschembio.0c00140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Two-Stage Strategy for Development of Proteolysis Targeting Chimeras and its Application for Estrogen Receptor Degraders</span></div><div class="casAuthors">Roberts, Brett L.; Ma, Zhi-Xiong; Gao, Ang; Leisten, Eric D.; Yin, Dan; Xu, Wei; Tang, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1487-1496</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) have emerged as useful chem. probes and potential therapeutics by taking advantage of the ubiquitin-proteasome system to degrade intracellular disease-assocd. proteins.  PROTACs are heterobifunctional mols. composed of a target protein ligand, E3 ubiquitin ligase ligand, and a linker between them.  The generation of efficient PROTACs requires screening of many parameters, esp. the lengths and types of the linkers.  The authors report the proof-of-concept study using a two-stage strategy to facilitate the development of PROTACs against the estrogen receptor (ER).  In stage one, a library of close to 100 PROTACs was synthesized by simply mixing a library of ERα ligands contg. a hydrazide functional group at different positions with a preassembled library of E3 ligase ligands bearing different types and lengths of linkers with a terminal aldehyde group in a 1:1 ratio.  Cell-based screening occurred without further purifn., because the formation of the acylhydrazone linkage is highly efficient and produces water as the only byproduct.  Compd. A3 was the most potent ER degrader in two ER+ cell lines (DC50= ~ 10 nM, Dmax= ≥ 95%).  Stage two involved transformation to a more stable amide linker to generate a more drug-like mol.  The new compd., AM-A3, showed comparable biol. activity (DC50 = 1.1 nM, Dmax = 98%) and induced potent antiproliferation (IC50= 13.2 nM, Imax= 69%) in MCF-7.  This proof-of -concept study demonstrates that the two-stage strategy can significantly facilitate the development of PROTACs against ER without the tedious process of making large nos. of PROTACs one by one.  It has the potential to be expanded to many other targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLYFrBdI2Sy7Vg90H21EOLACvtfcHk0lj_i7UreWmSpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFKru7w%253D&md5=80e8aeeba613ecff7d98944e7a579225</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00140%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DB.%2BL.%26aulast%3DMa%26aufirst%3DZ.%2BX.%26aulast%3DGao%26aufirst%3DA.%26aulast%3DLeisten%26aufirst%3DE.%2BD.%26aulast%3DYin%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DW.%26atitle%3DTwo-Stage%2520Strategy%2520for%2520Development%2520of%2520Proteolysis%2520Targeting%2520Chimeras%2520and%2520its%2520Application%2520for%2520Estrogen%2520Receptor%2520Degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D1487%26epage%3D1496%26doi%3D10.1021%2Facschembio.0c00140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cyrus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehenkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span> <span> </span><span class="NLM_article-title">Jostling for position: optimizing linker location in the design of estrogen receptor-targeting PROTACs</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">979</span>– <span class="NLM_lpage">985</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201000146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1002%2Fcmdc.201000146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=20512796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVOmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=979-985&author=K.+Cyrusauthor=M.+Wehenkelauthor=E.+Y.+Choiauthor=H.+Leeauthor=H.+Swansonauthor=K.+B.+Kim&title=Jostling+for+position%3A+optimizing+linker+location+in+the+design+of+estrogen+receptor-targeting+PROTACs&doi=10.1002%2Fcmdc.201000146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Jostling for Position: Optimizing Linker Location in the Design of Estrogen Receptor-Targeting PROTACs</span></div><div class="casAuthors">Cyrus, Kedra; Wehenkel, Marie; Choi, Eun-Young; Lee, Hyosung; Swanson, Hollie; Kim, Kyung-Bo</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">979-985</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Estrogen receptor-α (ER) antagonists have been widely used for breast cancer therapy.  Despite initial responsiveness, hormone-sensitive ER-pos. cancer cells eventually develop resistance to ER antagonists.  It has been shown that in most of these resistant tumor cells, the ER is expressed and continues to regulate tumor growth.  Recent studies indicate that tamoxifen initially acts as an antagonist, but later functions as an ER agonist, promoting tumor growth.  This suggests that targeted ER degrdn. may provide an effective therapeutic approach for breast cancers, even those that are resistant to conventional therapies.  With this in mind, we previously demonstrated that proteolysis targeting chimeras (PROTACs) effectively induce degrdn. of the ER as a proof-of-concept expt.  Herein we further refined the PROTAC approach to target the ER for degrdn.  The ER-targeting PROTACs are composed of an estradiol on one end and a hypoxia-inducing factor 1α (HIF-1α)-derived synthetic pentapeptide on the other.  The pentapeptide is recognized by an E3 ubiquitin ligase called the von Hippel Lindau tumor suppressor protein (pVHL), thereby recruiting the ER to this E3 ligase for ubiquitination and degrdn.  Specifically, the pentapeptide is attached at three different locations on estradiol to generate three different PROTAC types.  With the pentapeptide linked through the C7α position of estradiol, the resulting PROTAC shows the most effective ER degrdn. and highest affinity for the estrogen receptor.  This result provides an opportunity to develop a novel type of ER antagonist that may overcome the resistance of breast tumors to conventional drugs such as tamoxifen and fulvestrant (Faslodex).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYQTb73o13F7Vg90H21EOLACvtfcHk0lj_i7UreWmSpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVOmt74%253D&md5=1bd032e4640fe04d72b1a69d746fdada</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000146%26sid%3Dliteratum%253Aachs%26aulast%3DCyrus%26aufirst%3DK.%26aulast%3DWehenkel%26aufirst%3DM.%26aulast%3DChoi%26aufirst%3DE.%2BY.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DSwanson%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DK.%2BB.%26atitle%3DJostling%2520for%2520position%253A%2520optimizing%2520linker%2520location%2520in%2520the%2520design%2520of%2520estrogen%2520receptor-targeting%2520PROTACs%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D979%26epage%3D985%26doi%3D10.1002%2Fcmdc.201000146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, Andrew P.</span>; <span class="NLM_string-name">Flanagin, John</span>; <span class="NLM_string-name">Gough Sheryl, Maxine</span>; <span class="NLM_string-name">Haskell Royal, J.</span>; <span class="NLM_string-name">Moore Marcia, D.</span>; <span class="NLM_string-name">Qian, Yimin</span>; <span class="NLM_string-name">Taylor Ian, C. A.</span>; <span class="NLM_string-name">Wang, Jing</span>; <span class="NLM_string-name">Xin, C.</span></span> <span> </span><span class="NLM_article-title">Methods of treating breast cancer with tetrahydronaphthalene derivatives as Estrogen Receptor Degraders</span>, Patent Application <span class="NLM_patent">WO/2021/041348</span>. March 4, <span class="NLM_year">2021</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=Andrew+P.+Crew&author=Flanagin+John&author=Gough+Sheryl+Maxine&author=J.+Haskell+Royal&author=D.+Moore+Marcia&author=Qian+Yimin&author=C.+A.+Taylor+Ian&author=Wang+Jing&author=C.+Xin&title=Methods+of+treating+breast+cancer+with+tetrahydronaphthalene+derivatives+as+Estrogen+Receptor+Degraders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DAndrew%2BP.%26atitle%3DMethods%2520of%2520treating%2520breast%2520cancer%2520with%2520tetrahydronaphthalene%2520derivatives%2520as%2520Estrogen%2520Receptor%2520Degraders%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Irrizary, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czabaniuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niosi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span> <span> </span><span class="NLM_article-title">Delineating the role of cooperativity in the design of potent PROTACs for BTK</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E7285</span>– <span class="NLM_lpage">E7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.1803662115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1073%2Fpnas.1803662115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30012605" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E7285-E7292&author=A.+Zorbaauthor=C.+Nguyenauthor=Y.+Xuauthor=J.+Starrauthor=K.+Borzilleriauthor=J.+Smithauthor=H.+Zhuauthor=K.+A.+Farleyauthor=W.+Dingauthor=J.+Schiemerauthor=X.+Fengauthor=J.+S.+Changauthor=D.+P.+Uccelloauthor=J.+A.+Youngauthor=C.+N.+Garcia-Irrizaryauthor=L.+Czabaniukauthor=B.+Schuffauthor=R.+Oliverauthor=J.+Montgomeryauthor=M.+M.+Haywardauthor=J.+Coeauthor=J.+Chenauthor=M.+Niosiauthor=S.+Luthraauthor=J.+C.+Shahauthor=A.+El-Kattanauthor=X.+Qiuauthor=G.+M.+Westauthor=M.+C.+Noeauthor=V.+Shanmugasundaramauthor=A.+M.+Gilbertauthor=M.+F.+Brownauthor=M.+F.+Calabrese&title=Delineating+the+role+of+cooperativity+in+the+design+of+potent+PROTACs+for+BTK&doi=10.1073%2Fpnas.1803662115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1803662115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1803662115%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DSchiemer%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DGarcia-Irrizary%26aufirst%3DC.%2BN.%26aulast%3DCzabaniuk%26aufirst%3DL.%26aulast%3DSchuff%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNiosi%26aufirst%3DM.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DJ.%2BC.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26atitle%3DDelineating%2520the%2520role%2520of%2520cooperativity%2520in%2520the%2520design%2520of%2520potent%2520PROTACs%2520for%2520BTK%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3DE7285%26epage%3DE7292%26doi%3D10.1073%2Fpnas.1803662115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjamin, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creaser, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzen, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gefter, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavarana, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vloten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1038/nchembio.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kelleyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries&doi=10.1038%2Fnchembio.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lizLWyFCj2vmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelley%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3Dvan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654%26doi%3D10.1038%2Fnchembio.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grady, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerfield, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded Library Screening Identifies Benzo[b][1,4]oxazepin-4-ones as Highly Potent and Monoselective Receptor Interacting Protein 1 Kinase Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2163</span>– <span class="NLM_lpage">2178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01898</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01898" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Kqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2163-2178&author=P.+A.+Harrisauthor=B.+W.+Kingauthor=D.+Bandyopadhyayauthor=S.+B.+Bergerauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=J.+N.+Fingerauthor=L.+C.+Gradyauthor=S.+J.+Hoffmanauthor=J.+U.+Jeongauthor=J.+Kangauthor=V.+Kasparcovaauthor=A.+S.+Lakdawalaauthor=R.+Lehrauthor=D.+E.+McNultyauthor=R.+Nagillaauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+R.+Rendinaauthor=M.+C.+Schaefferauthor=J.+D.+Summerfieldauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=P.+Wardauthor=A.+Zhangauthor=D.+Zhangauthor=R.+W.+Marquisauthor=J.+Bertinauthor=P.+J.+Gough&title=DNA-Encoded+Library+Screening+Identifies+Benzo%5Bb%5D%5B1%2C4%5Doxazepin-4-ones+as+Highly+Potent+and+Monoselective+Receptor+Interacting+Protein+1+Kinase+Inhibitors&doi=10.1021%2Facs.jmedchem.5b01898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Library Screening Identifies Benzo[b][1,4]oxazepin-4-ones as Highly Potent and Monoselective Receptor Interacting Protein 1 Kinase Inhibitors</span></div><div class="casAuthors">Harris, Philip A.; King, Bryan W.; Bandyopadhyay, Deepak; Berger, Scott B.; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Finger, Joshua N.; Grady, LaShadric C.; Hoffman, Sandra J.; Jeong, Jae U.; Kang, James; Kasparcova, Viera; Lakdawala, Ami S.; Lehr, Ruth; McNulty, Dean E.; Nagilla, Rakesh; Ouellette, Michael T.; Pao, Christina S.; Rendina, Alan R.; Schaeffer, Michelle C.; Summerfield, Jennifer D.; Swift, Barbara A.; Totoritis, Rachel D.; Ward, Paris; Zhang, Aming; Zhang, Daohua; Marquis, Robert W.; Bertin, John; Gough, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2163-2178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent discovery of the role of receptor interacting protein 1 (RIP1) kinase in tumor necrosis factor (TNF)-mediated inflammation has led to its emergence as a highly promising target for the treatment of multiple inflammatory diseases.  The authors screened RIP1 against GSK's DNA-encoded small-mol. libraries and identified a novel highly potent benzoxazepinone inhibitor series.  The authors demonstrate that this template possesses complete monokinase selectivity for RIP1 plus unique species selectivity for primate vs. nonprimate RIP1.  The authors elucidate the conformation of RIP1 bound to this benzoxazepinone inhibitor driving its high kinase selectivity and design specific mutations in murine RIP1 to restore potency to levels similar to primate RIP1.  This series differentiates itself from known RIP1 inhibitors in combining high potency and kinase selectivity with good pharmacokinetic profiles in rodents.  The favorable developability profile of this benzoxazepinone template, as exemplified by compd. 14 (GSK'481), makes it an excellent starting point for further optimization into a RIP1 clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4XvpziWGWg7Vg90H21EOLACvtfcHk0lizLWyFCj2vmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Kqsr8%253D&md5=d6325dcaed97b7c339d1830ca041298c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01898%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DGrady%26aufirst%3DL.%2BC.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DMcNulty%26aufirst%3DD.%2BE.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSummerfield%26aufirst%3DJ.%2BD.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DDNA-Encoded%2520Library%2520Screening%2520Identifies%2520Benzo%255Bb%255D%255B1%252C4%255Doxazepin-4-ones%2520as%2520Highly%2520Potent%2520and%2520Monoselective%2520Receptor%2520Interacting%2520Protein%25201%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2163%26epage%3D2178%26doi%3D10.1021%2Facs.jmedchem.5b01898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazaar, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span> <span> </span><span class="NLM_article-title">Discovering Drugs with DNA-Encoded Library Technology: From Concept to Clinic with an Inhibitor of Soluble Epoxide Hydrolase</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">837</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1002/cbic.201700014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1002%2Fcbic.201700014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=28281333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVWjurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=837-842&author=S.+L.+Belyanskayaauthor=Y.+Dingauthor=J.+F.+Callahanauthor=A.+L.+Lazaarauthor=D.+I.+Israel&title=Discovering+Drugs+with+DNA-Encoded+Library+Technology%3A+From+Concept+to+Clinic+with+an+Inhibitor+of+Soluble+Epoxide+Hydrolase&doi=10.1002%2Fcbic.201700014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering Drugs with DNA-Encoded Library Technology: From Concept to Clinic with an Inhibitor of Soluble Epoxide Hydrolase</span></div><div class="casAuthors">Belyanskaya, Svetlana L.; Ding, Yun; Callahan, James F.; Lazaar, Aili L.; Israel, David I.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">837-842</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">DNA-encoded chem. library technol. was developed with the vision of its becoming a transformational platform for drug discovery.  The hope was that a new paradigm for the discovery of low-mol.-wt. drugs would be enabled by combining the vast mol. diversity achievable with combinatorial chem., the information-encoding attributes of DNA, the power of mol. biol., and a streamlined selection-based discovery process.  Here, we describe the discovery and early clin. development of GSK2256294, an inhibitor of sol. epoxide hydrolase (sEH, EPHX2), by using encoded-library technol. (ELT).  GSK2256294 is an orally bioavailable, potent and selective inhibitor of sEH that has a long half life and produced no serious adverse events in a first-time-in-human clin. study.  To our knowledge, GSK2256294 is the first mol. discovered from this technol. to enter human clin. testing and represents a realization of the vision that DNA-encoded chem. library technol. can efficiently yield mols. with favorable properties that can be readily progressed into high-quality drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7WTPZp2SE0rVg90H21EOLACvtfcHk0lhc0RmxfRzGug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVWjurk%253D&md5=a34e28c8817864356f6ff689e9236940</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201700014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201700014%26sid%3Dliteratum%253Aachs%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DLazaar%26aufirst%3DA.%2BL.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26atitle%3DDiscovering%2520Drugs%2520with%2520DNA-Encoded%2520Library%2520Technology%253A%2520From%2520Concept%2520to%2520Clinic%2520with%2520an%2520Inhibitor%2520of%2520Soluble%2520Epoxide%2520Hydrolase%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D837%26epage%3D842%26doi%3D10.1002%2Fcbic.201700014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref200"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref200'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit200"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnicow, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruebsam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Novel autotaxin inhibitor for the treatment of idiopathic pulmonary fibrosis: a clinical candidate discovered using DNA-encoded chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7840</span>– <span class="NLM_lpage">7856</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00688</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00688" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7840-7856&author=J.+W.+Cuozzoauthor=M.+A.+Clarkauthor=A.+D.+Keefeauthor=A.+Kohlmannauthor=M.+Mulvihillauthor=H.+Niauthor=L.+M.+Renzettiauthor=D.+I.+Resnicowauthor=F.+Ruebsamauthor=E.+A.+Sigelauthor=H.+A.+Thomsonauthor=C.+Wangauthor=Z.+Xieauthor=Y.+Zhang&title=Novel+autotaxin+inhibitor+for+the+treatment+of+idiopathic+pulmonary+fibrosis%3A+a+clinical+candidate+discovered+using+DNA-encoded+chemistry&doi=10.1021%2Facs.jmedchem.0c00688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit200R"><div class="casContent"><span class="casTitleNuber">200</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Autotaxin Inhibitor for the Treatment of Idiopathic Pulmonary Fibrosis: A Clinical Candidate Discovered Using DNA-Encoded Chemistry</span></div><div class="casAuthors">Cuozzo, John W.; Clark, Matthew A.; Keefe, Anthony D.; Kohlmann, Anna; Mulvihill, Mark; Ni, Haihong; Renzetti, Louis M.; Resnicow, Daniel I.; Ruebsam, Frank; Sigel, Eric A.; Thomson, Heather A.; Wang, Ce; Xie, Zhifeng; Zhang, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7840-7856</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The activity of the secreted phosphodiesterase autotaxin produces the inflammatory signaling mol. LPA and has been assocd. with a no. of human diseases including idiopathic pulmonary fibrosis (IPF).  We screened a single DNA-encoded chem. library (DECL) of 225 million compds. and identified a series of potent inhibitors.  Optimization of this series led to the discovery of compd. 1 (X-165)(I), a highly potent, selective, and bioavailable small mol.  Cocrystn. of compd. 1 with human autotaxin demonstrated that it has a novel binding mode occupying both the hydrophobic pocket and a channel near the autotaxin active site.  Compd. 1 inhibited the prodn. of LPA in human and mouse plasma at nanomolar levels and showed efficacy in a mouse model of human lung fibrosis.  After successfully completing IND-enabling studies, compd. 1 was approved by the FDA for a Phase I clin. trial.  These results demonstrate that DECL hits can be readily optimized into clin. candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaofuGyFvOJrVg90H21EOLACvtfcHk0lhc0RmxfRzGug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM&md5=c945f9bb71089c94314a27afce04fb96</span></div><a href="/servlet/linkout?suffix=cit200&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00688%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DMulvihill%26aufirst%3DM.%26aulast%3DNi%26aufirst%3DH.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DResnicow%26aufirst%3DD.%2BI.%26aulast%3DRuebsam%26aufirst%3DF.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DNovel%2520autotaxin%2520inhibitor%2520for%2520the%2520treatment%2520of%2520idiopathic%2520pulmonary%2520fibrosis%253A%2520a%2520clinical%2520candidate%2520discovered%2520using%2520DNA-encoded%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7840%26epage%3D7856%26doi%3D10.1021%2Facs.jmedchem.0c00688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folkesson, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristensen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slok, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakskjaer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madsen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azevedo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micco, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span> <span> </span><span class="NLM_article-title">Screening of DNA-Encoded Small Molecule Libraries inside a Living Cell</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">2751</span>– <span class="NLM_lpage">2756</span>, <span class="refDoi"> DOI: 10.1021/jacs.0c09213</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c09213" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjvVeitro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2021&pages=2751-2756&author=L.+K.+Petersenauthor=A.+B.+Christensenauthor=J.+Andersenauthor=C.+G.+Folkessonauthor=O.+Kristensenauthor=C.+Andersenauthor=A.+Alzuauthor=F.+A.+Slokauthor=P.+Blakskjaerauthor=D.+Madsenauthor=C.+Azevedoauthor=I.+Miccoauthor=N.+J.+V.+Hansen&title=Screening+of+DNA-Encoded+Small+Molecule+Libraries+inside+a+Living+Cell&doi=10.1021%2Fjacs.0c09213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Screening of DNA-Encoded Small Molecule Libraries inside a Living Cell</span></div><div class="casAuthors">Petersen, Lars K.; Christensen, Allan B.; Andersen, Jacob; Folkesson, Charlotta G.; Kristensen, Ole; Andersen, Charlotte; Alzu, Amaya; Sloek, Frank A.; Blakskjaer, Peter; Madsen, Daniel; Azevedo, Carlos; Micco, Iolanda; Hansen, Nils J. V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2751-2756</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded small mol. libraries (DELs) have facilitated the discovery of novel modulators of many different therapeutic protein targets.  We report the first successful screening of a multimillion membered DEL inside a living cell.  We demonstrate a novel method using oocytes from the South African clawed frog Xenopus laevis.  The large size of the oocytes of 1 μL, or 100 000 times bigger than a normal somatic cell, permits simple injection of DELs, thus resolving the fundamental problem of delivering DELs across cell membranes for in vivo screening.  The target protein was expressed in the oocytes fused to a prey protein, to allow specific DNA labeling and hereby discriminate between DEL members binding to the target protein and the endogenous cell proteins.  The 194 million member DEL was screened against three pharmaceutically relevant protein targets, p38α, ACSS2, and DOCK5.  For all three targets multiple chem. clusters were identified.  For p38α, validated hits with single digit nanomolar potencies were obtained.  This work demonstrates a powerful new approach to DEL screening, which eliminates the need for highly purified active target protein and which performs the screening under physiol. relevant conditions and thus is poised to increase the DEL amenable target space and reduce the attrition rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu0SRtLn2gZLVg90H21EOLACvtfcHk0lhc0RmxfRzGug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjvVeitro%253D&md5=6735556674600c60712bb74f171e50f3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c09213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c09213%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DL.%2BK.%26aulast%3DChristensen%26aufirst%3DA.%2BB.%26aulast%3DAndersen%26aufirst%3DJ.%26aulast%3DFolkesson%26aufirst%3DC.%2BG.%26aulast%3DKristensen%26aufirst%3DO.%26aulast%3DAndersen%26aufirst%3DC.%26aulast%3DAlzu%26aufirst%3DA.%26aulast%3DSlok%26aufirst%3DF.%2BA.%26aulast%3DBlakskjaer%26aufirst%3DP.%26aulast%3DMadsen%26aufirst%3DD.%26aulast%3DAzevedo%26aufirst%3DC.%26aulast%3DMicco%26aufirst%3DI.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26atitle%3DScreening%2520of%2520DNA-Encoded%2520Small%2520Molecule%2520Libraries%2520inside%2520a%2520Living%2520Cell%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2021%26volume%3D143%26spage%3D2751%26epage%3D2756%26doi%3D10.1021%2Fjacs.0c09213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCloskey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kearnes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moccia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazzaz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guie, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guilinger, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulhern, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, P.</span></span> <span> </span><span class="NLM_article-title">Machine Learning on DNA-Encoded Libraries: A New Paradigm for Hit Finding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8857</span>– <span class="NLM_lpage">8866</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00452</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00452" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFamtLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8857-8866&author=K.+McCloskeyauthor=E.+A.+Sigelauthor=S.+Kearnesauthor=L.+Xueauthor=X.+Tianauthor=D.+Mocciaauthor=D.+Gikunjuauthor=S.+Bazzazauthor=B.+Chanauthor=M.+A.+Clarkauthor=J.+W.+Cuozzoauthor=M.+A.+Guieauthor=J.+P.+Guilingerauthor=C.+Huguetauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=C.+J.+Mulhernauthor=Y.+Zhangauthor=P.+Riley&title=Machine+Learning+on+DNA-Encoded+Libraries%3A+A+New+Paradigm+for+Hit+Finding&doi=10.1021%2Facs.jmedchem.0c00452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Machine Learning on DNA-Encoded Libraries: A New Paradigm for Hit Finding</span></div><div class="casAuthors">McCloskey, Kevin; Sigel, Eric A.; Kearnes, Steven; Xue, Ling; Tian, Xia; Moccia, Dennis; Gikunju, Diana; Bazzaz, Sana; Chan, Betty; Clark, Matthew A.; Cuozzo, John W.; Guie, Marie-Aude; Guilinger, John P.; Huguet, Christelle; Hupp, Christopher D.; Keefe, Anthony D.; Mulhern, Christopher J.; Zhang, Ying; Riley, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8857-8866</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded small mol. libraries (DELs) have enabled discovery of novel inhibitors for many distinct protein targets of therapeutic value.  We demonstrate a new approach applying machine learning to DEL selection data by identifying active mols. from large libraries of com. and easily synthesizable compds.  We train models using only DEL selection data and apply automated or automatable filters to the predictions.  We perform a large prospective study (~ 2000 compds.) across three diverse protein targets: sEH (a hydrolase), ERα (a nuclear receptor), and c-KIT (a kinase).  The approach is effective, with an overall hit rate of ~ 30% at 30 μM and discovery of potent compds. (IC50 < 10 nM) for every target.  The system makes useful predictions even for mols. dissimilar to the original DEL, and the compds. identified are diverse, predominantly drug-like, and different from known ligands.  This work demonstrates a powerful new approach to hit-finding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1kezutJD3kbVg90H21EOLACvtfcHk0ljWoUaqlCGEUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFamtLjO&md5=1a2472a8c7d6a77ec2689b1e6034e2ec</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00452%26sid%3Dliteratum%253Aachs%26aulast%3DMcCloskey%26aufirst%3DK.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DKearnes%26aufirst%3DS.%26aulast%3DXue%26aufirst%3DL.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMoccia%26aufirst%3DD.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DBazzaz%26aufirst%3DS.%26aulast%3DChan%26aufirst%3DB.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DGuie%26aufirst%3DM.%2BA.%26aulast%3DGuilinger%26aufirst%3DJ.%2BP.%26aulast%3DHuguet%26aufirst%3DC.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DMulhern%26aufirst%3DC.%2BJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DRiley%26aufirst%3DP.%26atitle%3DMachine%2520Learning%2520on%2520DNA-Encoded%2520Libraries%253A%2520A%2520New%2520Paradigm%2520for%2520Hit%2520Finding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8857%26epage%3D8866%26doi%3D10.1021%2Facs.jmedchem.0c00452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wurz, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellamaggiore, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span> <span> </span><span class="NLM_article-title">A ″Click Chemistry Platform″ for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01781</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01781" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Gls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=453-461&author=R.+P.+Wurzauthor=K.+Dellamaggioreauthor=H.+Douauthor=N.+Javierauthor=M.+C.+Loauthor=J.+D.+McCarterauthor=D.+Mohlauthor=C.+Sastriauthor=J.+R.+Lipfordauthor=V.+J.+Cee&title=A+%E2%80%B3Click+Chemistry+Platform%E2%80%B3+for+the+Rapid+Synthesis+of+Bispecific+Molecules+for+Inducing+Protein+Degradation&doi=10.1021%2Facs.jmedchem.6b01781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A "Click Chemistry Platform" for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation</span></div><div class="casAuthors">Wurz, Ryan P.; Dellamaggiore, Ken; Dou, Hannah; Javier, Noelle; Lo, Mei-Chu; McCarter, John D.; Mohl, Dane; Sastri, Christine; Lipford, J. Russell; Cee, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">453-461</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) are bispecific mols. contg. a target protein binder and an ubiquitin ligase binder connected by a linker.  By recruiting an ubiquitin ligase to a target protein, PROTACs promote ubiquitination and proteasomal degrdn. of the target protein.  The generation of effective PROTACs depends on the nature of the protein/ligase ligand pair, linkage site, linker length, and linker compn., all of which have been difficult to address in a systematic way.  Herein, the authors describe a "click chem." approach for the synthesis of PROTACs.  The authors demonstrate the utility of this approach with the bromodomain and extra terminal domain-4 (BRD4) ligand JQ-1 (3) and ligase binders targeting cereblon (CRBN) and Von Hippel-Lindau (VHL) proteins.  An AlphaScreen proximity assay was used to det. the ability of PROTACs to form the ternary ligase-PROTAC-target protein complex and a MSD assay to measure cellular degrdn. of the target protein promoted by PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpooapLJ2Zln7Vg90H21EOLACvtfcHk0ljWoUaqlCGEUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Gls7w%253D&md5=090c569ae1f079ab6504348aabec8909</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01781%26sid%3Dliteratum%253Aachs%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DDellamaggiore%26aufirst%3DK.%26aulast%3DDou%26aufirst%3DH.%26aulast%3DJavier%26aufirst%3DN.%26aulast%3DLo%26aufirst%3DM.%2BC.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DMohl%26aufirst%3DD.%26aulast%3DSastri%26aufirst%3DC.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26atitle%3DA%2520%25E2%2580%25B3Click%2520Chemistry%2520Platform%25E2%2580%25B3%2520for%2520the%2520Rapid%2520Synthesis%2520of%2520Bispecific%2520Molecules%2520for%2520Inducing%2520Protein%2520Degradation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D453%26epage%3D461%26doi%3D10.1021%2Facs.jmedchem.6b01781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faver, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancia, D. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milbank, J. B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matzuk, M. M.</span></span> <span> </span><span class="NLM_article-title">Quantitative Comparison of Enrichment from DNA-Encoded Chemical Library Selections</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.8b00116</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.8b00116" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsV2jtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=75-82&author=J.+C.+Faverauthor=K.+Riehleauthor=D.+R.+Lanciaauthor=J.+B.+J.+Milbankauthor=C.+S.+Kollmannauthor=N.+Simmonsauthor=Z.+Yuauthor=M.+M.+Matzuk&title=Quantitative+Comparison+of+Enrichment+from+DNA-Encoded+Chemical+Library+Selections&doi=10.1021%2Facscombsci.8b00116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative comparison of enrichment from DNA-encoded chemical library selections</span></div><div class="casAuthors">Faver, John C.; Riehle, Kevin; Lancia, David R., Jr.; Milbank, Jared B. J.; Kollmann, Christopher S.; Simmons, Nicholas; Yu, Zhifeng; Matzuk, Martin M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">75-82</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded chem. libraries (DELs) provide a high-throughput and cost-effective route for screening billions of unique mols. for binding affinity for diverse protein targets.  Identifying candidate compds. from these libraries involves affinity selection, DNA sequencing, and measuring enrichment in a sample pool of DNA barcodes.  Successful detection of potent binders is affected by many factors, including selection parameters, chem. yields, library amplification, sequencing depth, sequencing errors, library sizes, and the chosen enrichment metric.  To date, there has not been a clear consensus about how enrichment from DEL selections should be measured or reported.  We propose a normalized z-score enrichment metric using a binomial distribution model that satisfies important criteria that are relevant for anal. of DEL selection data.  The introduced metric is robust with respect to library diversity and sampling and allows for quant. comparisons of enrichment of n-synthons from parallel DEL selections.  These features enable a comparative enrichment anal. strategy that can provide valuable information about hit compds. in early stage drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplYUndBUCVTLVg90H21EOLACvtfcHk0ljWoUaqlCGEUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsV2jtrY%253D&md5=28bd49450ec839438006676ad36f2e36</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.8b00116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.8b00116%26sid%3Dliteratum%253Aachs%26aulast%3DFaver%26aufirst%3DJ.%2BC.%26aulast%3DRiehle%26aufirst%3DK.%26aulast%3DLancia%26aufirst%3DD.%2BR.%26aulast%3DMilbank%26aufirst%3DJ.%2BB.%2BJ.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DSimmons%26aufirst%3DN.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26atitle%3DQuantitative%2520Comparison%2520of%2520Enrichment%2520from%2520DNA-Encoded%2520Chemical%2520Library%2520Selections%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2019%26volume%3D21%26spage%3D75%26epage%3D82%26doi%3D10.1021%2Facscombsci.8b00116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Challinor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span> <span> </span><span class="NLM_article-title">A Screening Assay Cascade to Identify and Characterize Novel Selective Estrogen Receptor Downregulators (SERDs)</span>. <i>J. Biomol. Screen</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">748</span>– <span class="NLM_lpage">759</span>, <span class="refDoi"> DOI: 10.1177/1087057115580298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1177%2F1087057115580298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=25851036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=748-759&author=R.+Callisauthor=A.+Rabowauthor=M.+Tongeauthor=R.+Bradburyauthor=M.+Challinorauthor=K.+Robertsauthor=K.+Jonesauthor=G.+Walker&title=A+Screening+Assay+Cascade+to+Identify+and+Characterize+Novel+Selective+Estrogen+Receptor+Downregulators+%28SERDs%29&doi=10.1177%2F1087057115580298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A screening assay cascade to identify and characterize novel selective estrogen receptor downregulators (SERDs)</span></div><div class="casAuthors">Callis, Rowena; Rabow, Alfred; Tonge, Michael; Bradbury, Robert; Challinor, Mairi; Roberts, Karen; Jones, Karen; Walker, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">748-759</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Here, we describe an approach to identify novel selective estrogen receptor downregulator (SERD) compds. with improved properties such as oral bioavailability and the potential of increased efficacy compared to currently marketed drug treatments.  Previously, methodologies such as Western blotting and transient cell reporter assays have been used to identify and characterize SERD compds., but such approaches can be limited due to low throughput and sensitivity, resp.  We have used an endogenous cell-imaging strategy that has both the throughput and sensitivity to support a large-scale hit-to-lead program to identify novel compds.  A screening cascade with a suite of assays has been developed to characterize compds. that modulate estrogen receptor α (ERα)-mediated signaling or downregulate ERα levels in cells.  Initially, from a focused high-throughput screening, novel ERα binders were identified that could be modified chem. into ERα downregulators.  Following this, cellular assays helped det. the mechanism of action of compds. to distinguish between on-target and off-target compds. and differentiate SERDs, selective estrogen receptor modulator (SERM) compds., and agonist ERα ligands.  Data are shown to exemplify the characterization of ERα-mediated signaling inhibitors using a selection of literature compds. and illustrate how this cascade has been used to drive the chem. design of novel SERD compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqNNWMPPJDNrVg90H21EOLACvtfcHk0ljeSlXoLWSh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsbzE&md5=13c27dbf08f315d469f81b2bcefb4499</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1177%2F1087057115580298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115580298%26sid%3Dliteratum%253Aachs%26aulast%3DCallis%26aufirst%3DR.%26aulast%3DRabow%26aufirst%3DA.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DBradbury%26aufirst%3DR.%26aulast%3DChallinor%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DWalker%26aufirst%3DG.%26atitle%3DA%2520Screening%2520Assay%2520Cascade%2520to%2520Identify%2520and%2520Characterize%2520Novel%2520Selective%2520Estrogen%2520Receptor%2520Downregulators%2520%2528SERDs%2529%26jtitle%3DJ.%2520Biomol.%2520Screen%26date%3D2015%26volume%3D20%26spage%3D748%26epage%3D759%26doi%3D10.1177%2F1087057115580298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komm, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharode, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyttle, C. R.</span></span> <span> </span><span class="NLM_article-title">Bazedoxifene acetate: a selective estrogen receptor modulator with improved selectivity</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">3999</span>– <span class="NLM_lpage">4008</span>, <span class="refDoi"> DOI: 10.1210/en.2005-0030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1210%2Fen.2005-0030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=15961563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsVegsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2005&pages=3999-4008&author=B.+S.+Kommauthor=Y.+P.+Kharodeauthor=P.+V.+Bodineauthor=H.+A.+Harrisauthor=C.+P.+Millerauthor=C.+R.+Lyttle&title=Bazedoxifene+acetate%3A+a+selective+estrogen+receptor+modulator+with+improved+selectivity&doi=10.1210%2Fen.2005-0030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Bazedoxifene acetate: a selective estrogen receptor modulator with improved selectivity</span></div><div class="casAuthors">Komm, Barry S.; Kharode, Yogendra P.; Bodine, Peter V. N.; Harris, Heather A.; Miller, Chris P.; Lyttle, C. Richard</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3999-4008</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">We assessed the preclin. characteristics of a novel, stringently screened selective estrogen receptor modulator, bazedoxifene acetate, including its ability to bind to and activate estrogen receptors and promote increased bone mineral d. and bone strength in rats, and the effects impacting the uterine endometrium, breast cancer cell proliferation, and central nervous system-assocd. vasomotor responses in an animal model.  Bazedoxifene bound to estrogen receptor-α with an IC50 of 26 nM, an affinity similar to that of raloxifene.  Bazedoxifene did not stimulate proliferation of MCF-7 cells but did inhibit 17β-estradiol-induced proliferation with an IC50 of 0.19 nM.  In an immature rat uterine model, bazedoxifene (0.5 and 5.0 mg/kg) was assocd. with less increase in uterine wet wt. than either ethinyl estradiol (10 μg/kg) or raloxifene (0.5 and 5.0 mg/kg).  Histol. anal. revealed that coadministration of bazedoxifene also appeared to reduce raloxifene-stimulated endometrial luminal epithelial cell and myometrial cell hypertrophy.  In ovariectomized rats, bazedoxifene was assocd. with significant increases in bone mineral d. at 6 wk, compared with control, and better compressive strength of bone samples from the L4 vertebrae, compared with samples from ovariectomized animals.  In the morphine-addicted rat model of vasomotor activity, bone-sparing doses of bazedoxifene alone were not assocd. with 17β-estradiol inhibition of increased vasomotor activity.  Bazedoxifene acetate represents a promising new treatment for osteoporosis, with a potential for less uterine and vasomotor effects than selective estrogen receptor modulators currently used in clin. practice.  Controlled clin. trial data will be needed to confirm these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou3Bu1u-p7dbVg90H21EOLACvtfcHk0ljeSlXoLWSh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsVegsb4%253D&md5=667468f62112aede01f2dec0ed95314f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1210%2Fen.2005-0030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2005-0030%26sid%3Dliteratum%253Aachs%26aulast%3DKomm%26aufirst%3DB.%2BS.%26aulast%3DKharode%26aufirst%3DY.%2BP.%26aulast%3DBodine%26aufirst%3DP.%2BV.%26aulast%3DHarris%26aufirst%3DH.%2BA.%26aulast%3DMiller%26aufirst%3DC.%2BP.%26aulast%3DLyttle%26aufirst%3DC.%2BR.%26atitle%3DBazedoxifene%2520acetate%253A%2520a%2520selective%2520estrogen%2520receptor%2520modulator%2520with%2520improved%2520selectivity%26jtitle%3DEndocrinology%26date%3D2005%26volume%3D146%26spage%3D3999%26epage%3D4008%26doi%3D10.1210%2Fen.2005-0030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collini, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharode, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzolf, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unwalla, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberger, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yardley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou-Gharbia, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyttle, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komm, B. S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and preclinical characterization of novel, highly selective indole estrogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1654</span>– <span class="NLM_lpage">1657</span>, <span class="refDoi"> DOI: 10.1021/jm010086m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010086m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFSgt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1654-1657&author=C.+P.+Millerauthor=M.+D.+Colliniauthor=B.+D.+Tranauthor=H.+A.+Harrisauthor=Y.+P.+Kharodeauthor=J.+T.+Marzolfauthor=R.+A.+Moranauthor=R.+A.+Hendersonauthor=R.+H.+Benderauthor=R.+J.+Unwallaauthor=L.+M.+Greenbergerauthor=J.+P.+Yardleyauthor=M.+A.+Abou-Gharbiaauthor=C.+R.+Lyttleauthor=B.+S.+Komm&title=Design%2C+synthesis%2C+and+preclinical+characterization+of+novel%2C+highly+selective+indole+estrogens&doi=10.1021%2Fjm010086m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Preclinical Characterization of Novel, Highly Selective Indole Estrogens</span></div><div class="casAuthors">Miller, Chris P.; Collini, Michael D.; Tran, Bach D.; Harris, Heather A.; Kharode, Yogendra P.; Marzolf, James T.; Moran, Robert A.; Henderson, Ruth A.; Bender, Reinhold H. W.; Unwalla, Rayomond J.; Greenberger, Lee M.; Yardley, John P.; Abou-Gharbia, Magid A.; Lyttle, C. Richard; Komm, Barry S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1654-1657</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The indoles I [n = 1, 2] were prepd. from 5-benzyloxy-2-(4-benzyloxyphenyl)-3-methylindole, 4-EtO2CCH2OC6H4CH2Cl, and piperidine or hexamethyleneimine.  I display attractive selective estrogen profiles.  I [n = 2] is effective in treating post-menopausal osteoporosis and I [n = 1] is an efficient antiestrogen in in vivo MCF-7 xenografts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKcOJFkvSQKLVg90H21EOLACvtfcHk0ljeSlXoLWSh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFSgt7g%253D&md5=1e355073a49a908821acb9497de9ffbd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm010086m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010086m%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DC.%2BP.%26aulast%3DCollini%26aufirst%3DM.%2BD.%26aulast%3DTran%26aufirst%3DB.%2BD.%26aulast%3DHarris%26aufirst%3DH.%2BA.%26aulast%3DKharode%26aufirst%3DY.%2BP.%26aulast%3DMarzolf%26aufirst%3DJ.%2BT.%26aulast%3DMoran%26aufirst%3DR.%2BA.%26aulast%3DHenderson%26aufirst%3DR.%2BA.%26aulast%3DBender%26aufirst%3DR.%2BH.%26aulast%3DUnwalla%26aufirst%3DR.%2BJ.%26aulast%3DGreenberger%26aufirst%3DL.%2BM.%26aulast%3DYardley%26aufirst%3DJ.%2BP.%26aulast%3DAbou-Gharbia%26aufirst%3DM.%2BA.%26aulast%3DLyttle%26aufirst%3DC.%2BR.%26aulast%3DKomm%26aufirst%3DB.%2BS.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520preclinical%2520characterization%2520of%2520novel%252C%2520highly%2520selective%2520indole%2520estrogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1654%26epage%3D1657%26doi%3D10.1021%2Fjm010086m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13312</span>, <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13312&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-%CE%B1+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0lgJwU4lf3-gGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-%25CE%25B1%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13312%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) in ’beyond rule-of-five’ chemical space: Recent progress and future challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+targeting+chimeras+%28PROTACs%29+in+%E2%80%99beyond+rule-of-five%E2%80%99+chemical+space%3A+Recent+progress+and+future+challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0lgJwU4lf3-gGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2599beyond%2520rule-of-five%25E2%2580%2599%2520chemical%2520space%253A%2520Recent%2520progress%2520and%2520future%2520challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cantrill, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaturvedi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrig Schaffland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M. B.</span></span> <span> </span><span class="NLM_article-title">Fundamental aspects of DMPK optimization of targeted protein degraders</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">969</span>– <span class="NLM_lpage">982</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.drudis.2020.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32298797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntFGht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=969-982&author=C.+Cantrillauthor=P.+Chaturvediauthor=C.+Rynnauthor=J.+Petrig+Schafflandauthor=I.+Walterauthor=M.+B.+Wittwer&title=Fundamental+aspects+of+DMPK+optimization+of+targeted+protein+degraders&doi=10.1016%2Fj.drudis.2020.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental aspects of DMPK optimization of targeted protein degraders</span></div><div class="casAuthors">Cantrill, Carina; Chaturvedi, Prasoon; Rynn, Caroline; Petrig Schaffland, Jeannine; Walter, Isabelle; Wittwer, Matthias B.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">969-982</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Targeted protein degraders are an emerging modality.  Their properties fall outside the traditional small-mol. property space and are in the 'beyond rule of 5' space.  Consequently, drug discovery programs focused on developing orally bioavailable degraders are expected to face complex drug metab. and pharmacokinetics (DMPK) challenges compared with traditional small mols.  Nevertheless, little information is available on the DMPK optimization of oral degraders.  Therefore, in this review, we discuss our experience of these DMPK optimization challenges and present methodologies and strategies to overcome the hurdles dealing with this new small-mol. modality, specifically in developing oral degraders to treat cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz2WUoP7TUvbVg90H21EOLACvtfcHk0lgJwU4lf3-gGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntFGht7s%253D&md5=465ad2dab5183e0f229aeec7e57e396a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DCantrill%26aufirst%3DC.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DRynn%26aufirst%3DC.%26aulast%3DPetrig%2BSchaffland%26aufirst%3DJ.%26aulast%3DWalter%26aufirst%3DI.%26aulast%3DWittwer%26aufirst%3DM.%2BB.%26atitle%3DFundamental%2520aspects%2520of%2520DMPK%2520optimization%2520of%2520targeted%2520protein%2520degraders%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26spage%3D969%26epage%3D982%26doi%3D10.1016%2Fj.drudis.2020.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukumo, Y.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation by Chimeric Small Molecules, PROTACs and SNIPERs</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">849</span>, <span class="refDoi"> DOI: 10.3389/fchem.2019.00849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.3389%2Ffchem.2019.00849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31921772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1KktLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=849&author=M.+Naitoauthor=N.+Ohokaauthor=N.+Shibataauthor=Y.+Tsukumo&title=Targeted+Protein+Degradation+by+Chimeric+Small+Molecules%2C+PROTACs+and+SNIPERs&doi=10.3389%2Ffchem.2019.00849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation by chimeric small molecules, PROTACs and SNIPERs</span></div><div class="casAuthors">Naito, Mikihiko; Ohoka, Nobumichi; Shibata, Norihito; Tsukumo, Yoshinori</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">849</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Technologies that induce targeted protein degrdn. by small mols. have been developed recently.  Chimeric small mols. such as Proteolysis Targeting Chimeras (PROTACs) and Specific and Non-genetic IAP-dependent Protein Erasers (SNIPERs), and E3 modulators such as thalidomides, hijack the cellular machinery for ubiquitylation, and the ubiquitylated proteins are subjected to proteasomal degrdn.  This has motivated drug development in industry and academia because "undruggable targets" can now be degraded by targeted protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu8jFxYJ0HebVg90H21EOLACvtfcHk0ljXhkLfs6ETFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1KktLrJ&md5=a58ebd83aba5334f215500bb50915284</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2019.00849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2019.00849%26sid%3Dliteratum%253Aachs%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DTsukumo%26aufirst%3DY.%26atitle%3DTargeted%2520Protein%2520Degradation%2520by%2520Chimeric%2520Small%2520Molecules%252C%2520PROTACs%2520and%2520SNIPERs%26jtitle%3DFront.%2520Chem.%26date%3D2019%26volume%3D7%26spage%3D849%26doi%3D10.3389%2Ffchem.2019.00849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0ljXhkLfs6ETFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">First targeted protein degrader hits the clinic</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00043-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fd41573-019-00043-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=237-239&author=A.+Mullard&title=First+targeted+protein+degrader+hits+the+clinic&doi=10.1038%2Fd41573-019-00043-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00043-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00043-6%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DFirst%2520targeted%2520protein%2520degrader%2520hits%2520the%2520clinic%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D237%26epage%3D239%26doi%3D10.1038%2Fd41573-019-00043-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0liXbrnXBc5Esw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-Induced+Protein+Degradation+in+Drug+Discovery%3A+Breaking+the+Rules+or+Just+Making+New+Ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0liXbrnXBc5Esw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-Induced%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%253A%2520Breaking%2520the%2520Rules%2520or%2520Just%2520Making%2520New%2520Ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott-Stevens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaohua, L.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation in vivo with Proteolysis Targeting Chimeras: Current status and future considerations</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.ddtec.2019.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31200862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=69-80&author=G.+F.+Wattauthor=P.+Scott-Stevensauthor=L.+Gaohua&title=Targeted+protein+degradation+in+vivo+with+Proteolysis+Targeting+Chimeras%3A+Current+status+and+future+considerations&doi=10.1016%2Fj.ddtec.2019.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation in vivo with Proteolysis Targeting Chimeras: Current status and future considerations</span></div><div class="casAuthors">Watt Gillian F; Scott-Stevens Paul; Gaohua Lu</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">69-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are a rapidly expanding new therapeutic modality inducing selective protein degradation and offering the potential of a differentiated pharmacological profile across multiple therapeutic areas.  As the repertoire of protein targets and E3 ligases available for incorporation into PROTACs continues to grow, understanding the drug- and system-dependent parameters for PROTACs will be critical for achieving tissue/cell specific pharmacology.  The review discusses the current knowledge and future direction of in vivo PROTAC study evaluation.  The importance of establishing the quantitative relationship between loss of protein target and biological function in vivo, coupled with building mechanistic PK/PD and ultimately PBPK/PD models, is emphasised with the aim to aid translation from preclinical to clinical space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXobYtC3U5jeOifW6udTcc2ebrhULDTVrQZbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtA%253D%253D&md5=2dd51eb704b03eee904c1b04377f0a8e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWatt%26aufirst%3DG.%2BF.%26aulast%3DScott-Stevens%26aufirst%3DP.%26aulast%3DGaohua%26aufirst%3DL.%26atitle%3DTargeted%2520protein%2520degradation%2520in%2520vivo%2520with%2520Proteolysis%2520Targeting%2520Chimeras%253A%2520Current%2520status%2520and%2520future%2520considerations%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D69%26epage%3D80%26doi%3D10.1016%2Fj.ddtec.2019.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kargbo, R. B.</span></span> <span> </span><span class="NLM_article-title">PROTAC-Mediated Degradation of Estrogen Receptor in the Treatment of Cancer</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1367</span>– <span class="NLM_lpage">1369</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00397</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00397" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslegt77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1367-1369&author=R.+B.+Kargbo&title=PROTAC-Mediated+Degradation+of+Estrogen+Receptor+in+the+Treatment+of+Cancer&doi=10.1021%2Facsmedchemlett.9b00397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-Mediated Degradation of Estrogen Receptor in the Treatment of Cancer</span></div><div class="casAuthors">Kargbo, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1367-1369</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoD-uHJ-PklLVg90H21EOLACvtfcHk0liXbrnXBc5Esw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslegt77F&md5=77eb546538b6dec6ddf1316be5d9b092</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00397%26sid%3Dliteratum%253Aachs%26aulast%3DKargbo%26aufirst%3DR.%2BB.%26atitle%3DPROTAC-Mediated%2520Degradation%2520of%2520Estrogen%2520Receptor%2520in%2520the%2520Treatment%2520of%2520Cancer%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1367%26epage%3D1369%26doi%3D10.1021%2Facsmedchemlett.9b00397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rackham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thawani, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haile, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beal, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossenkamper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dace, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benowitz, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denyer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott-Stevens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span> <span> </span><span class="NLM_article-title">Extended pharmacodynamic responses observed upon PROTAC-mediated degradation of RIPK2</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">140</span>, <span class="refDoi"> DOI: 10.1038/s42003-020-0868-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fs42003-020-0868-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32198438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlslyqt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=140&author=A.+Maresauthor=A.+H.+Miahauthor=I.+E.+D.+Smithauthor=M.+Rackhamauthor=A.+R.+Thawaniauthor=J.+Cryanauthor=P.+A.+Haileauthor=B.+J.+Vottaauthor=A.+M.+Bealauthor=C.+Capriottiauthor=M.+A.+Reillyauthor=D.+T.+Fisherauthor=N.+Zinnauthor=M.+Bantscheffauthor=T.+T.+MacDonaldauthor=A.+Vossenkamperauthor=P.+Daceauthor=I.+Churcherauthor=A.+B.+Benowitzauthor=G.+Wattauthor=J.+Denyerauthor=P.+Scott-Stevensauthor=J.+D.+Harling&title=Extended+pharmacodynamic+responses+observed+upon+PROTAC-mediated+degradation+of+RIPK2&doi=10.1038%2Fs42003-020-0868-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Extended pharmacodynamic responses observed upon PROTAC-mediated degradation of RIPK2</span></div><div class="casAuthors">Mares, Alina; Miah, Afjal H.; Smith, Ian E. D.; Rackham, Mark; Thawani, Aditya R.; Cryan, Jenni; Haile, Pamela A.; Votta, Bartholomew J.; Beal, Allison M.; Capriotti, Carol; Reilly, Michael A.; Fisher, Don T.; Zinn, Nico; Bantscheff, Marcus; MacDonald, Thomas T.; Vossenkamper, Anna; Dace, Phoebe; Churcher, Ian; Benowitz, Andrew B.; Watt, Gillian; Denyer, Jane; Scott-Stevens, Paul; Harling, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Communications Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140</span>CODEN:
                <span class="NLM_cas:coden">CBOIDQ</span>;
        ISSN:<span class="NLM_cas:issn">2399-3642</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteolysis-Targeting Chimeras (PROTACs) are heterobifunctional small-mols. that can promote the rapid and selective proteasome-mediated degrdn. of intracellular proteins through the recruitment of E3 ligase complexes to non-native protein substrates.  The catalytic mechanism of action of PROTACs represents an exciting new modality in drug discovery that offers several potential advantages over traditional small-mol. inhibitors, including the potential to deliver pharmacodynamic (PD) efficacy which extends beyond the detectable pharmacokinetic (PK) presence of the PROTAC, driven by the synthesis rate of the protein.  Herein we report the identification and development of PROTACs that selectively degrade Receptor-Interacting Serine/Threonine Protein Kinase 2 (RIPK2) and demonstrate in vivo degrdn. of endogenous RIPK2 in rats at low doses and extended PD that persists in the absence of detectable compd.  This disconnect between PK and PD, when coupled with low nanomolar potency, offers the potential for low human doses and infrequent dosing regimens with PROTAC medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXhjAOWsJhfLVg90H21EOLACvtfcHk0lhUFNGpxVIjDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlslyqt7c%253D&md5=6a9773698a2ba5e244974a9ec3fc8f3a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs42003-020-0868-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-020-0868-6%26sid%3Dliteratum%253Aachs%26aulast%3DMares%26aufirst%3DA.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DRackham%26aufirst%3DM.%26aulast%3DThawani%26aufirst%3DA.%2BR.%26aulast%3DCryan%26aufirst%3DJ.%26aulast%3DHaile%26aufirst%3DP.%2BA.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3DBeal%26aufirst%3DA.%2BM.%26aulast%3DCapriotti%26aufirst%3DC.%26aulast%3DReilly%26aufirst%3DM.%2BA.%26aulast%3DFisher%26aufirst%3DD.%2BT.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DMacDonald%26aufirst%3DT.%2BT.%26aulast%3DVossenkamper%26aufirst%3DA.%26aulast%3DDace%26aufirst%3DP.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DBenowitz%26aufirst%3DA.%2BB.%26aulast%3DWatt%26aufirst%3DG.%26aulast%3DDenyer%26aufirst%3DJ.%26aulast%3DScott-Stevens%26aufirst%3DP.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26atitle%3DExtended%2520pharmacodynamic%2520responses%2520observed%2520upon%2520PROTAC-mediated%2520degradation%2520of%2520RIPK2%26jtitle%3DCommun.%2520Biol.%26date%3D2020%26volume%3D3%26spage%3D140%26doi%3D10.1038%2Fs42003-020-0868-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">den
Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span> <span> </span><span class="NLM_article-title">Prospecting for molecular glues</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0620-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fs41589-020-0620-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32747810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFaiur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1157-1158&author=W.+den%0ABestenauthor=J.+R.+Lipford&title=Prospecting+for+molecular+glues&doi=10.1038%2Fs41589-020-0620-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Prospecting for molecular glues</span></div><div class="casAuthors">den Besten, Willem; Lipford, J. Russell</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1157-1158</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Two recent studies identified CDK12 inhibitors that bind to CDK12-cyclin K complexes and act as mol. glues to stabilize an interaction with the ubiquitin ligase CUL4-DDB1, leading to cyclin K degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqej0SQYEVKmLVg90H21EOLACvtfcHk0lhUFNGpxVIjDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFaiur7F&md5=00ba8e9c5ec1aaa6e0a8beb0fc8cdb5c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0620-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0620-z%26sid%3Dliteratum%253Aachs%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26atitle%3DProspecting%2520for%2520molecular%2520glues%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D1157%26epage%3D1158%26doi%3D10.1038%2Fs41589-020-0620-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalvan, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Saez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjanacz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&author=A.+E.+Fern%C3%A1ndez-Montalvanauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+Stockigtauthor=A.+Ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+Diaz-Saezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=J.+Hubnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorjanacz&title=Isoform-Selective+ATAD2+Chemical+Probe+with+Novel+Chemical+Structure+and+Unusual+Mode+of+Action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span></div><div class="casAuthors">Fernandez-Montalvan, Amaury E.; Berger, Markus; Kuropka, Benno; Koo, Seong Joo; Badock, Volker; Weiske, Joerg; Puetter, Vera; Holton, Simon J.; Stoeckigt, Detlef; ter Laak, Antonius; Centrella, Paolo A.; Clark, Matthew A.; Dumelin, Christoph E.; Sigel, Eric A.; Soutter, Holly H.; Troast, Dawn M.; Zhang, Ying; Cuozzo, John W.; Keefe, Anthony D.; Roche, Didier; Rodeschini, Vincent; Chaikuad, Apirat; Diaz-Saez, Laura; Bennett, James M.; Fedorov, Oleg; Huber, Kilian V. M.; Huebner, Jan; Weinmann, Hilmar; Hartung, Ingo V.; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2730-2736</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 (ANCCA) is an epigenetic regulator and transcriptional cofactor, whose overexpression has been linked to the progress of various cancer types.  Here, we report a DNA-encoded library screen leading to the discovery of BAY-850, a potent and isoform selective inhibitor that specifically induces ATAD2 bromodomain dimerization and prevents interactions with acetylated histones in vitro, as well as with chromatin in cells.  These features qualify BAY-850 as a chem. probe to explore ATAD2 biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCB3nyG9YvubVg90H21EOLACvtfcHk0lhUFNGpxVIjDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL&md5=1f09000a7d7e60e30540fbbae117acfe</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DStockigt%26aufirst%3DD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DDiaz-Saez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DHubner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorjanacz%26aufirst%3DM.%26atitle%3DIsoform-Selective%2520ATAD2%2520Chemical%2520Probe%2520with%2520Novel%2520Chemical%2520Structure%2520and%2520Unusual%2520Mode%2520of%2520Action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Litovchick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeshian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guié, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Encoded Library Synthesis Using Chemical Ligation and the Discovery of sEH Inhibitors from a 334-Million Member Library</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10916</span>, <span class="refDoi"> DOI: 10.1038/srep10916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fsrep10916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=26061191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyiu7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=10916&author=A.+Litovchickauthor=C.+E.+Dumelinauthor=S.+Habeshianauthor=D.+Gikunjuauthor=M.-A.+Gui%C3%A9author=P.+Centrellaauthor=Y.+Zhangauthor=E.+A.+Sigelauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=M.+A.+Clark&title=Encoded+Library+Synthesis+Using+Chemical+Ligation+and+the+Discovery+of+sEH+Inhibitors+from+a+334-Million+Member+Library&doi=10.1038%2Fsrep10916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded Library Synthesis Using Chemical Ligation and the Discovery of sEH Inhibitors from a 334-Million Member Library</span></div><div class="casAuthors">Litovchick, Alexander; Dumelin, Christoph E.; Habeshian, Sevan; Gikunju, Diana; Guie, Marie-Aude; Centrella, Paolo; Zhang, Ying; Sigel, Eric A.; Cuozzo, John W.; Keefe, Anthony D.; Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10916</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A chem. ligation method for construction of DNA-encoded small-mol. libraries has been developed.  Taking advantage of the ability of the Klenow fragment of DNA polymerase to accept templates with triazole linkages in place of phosphodiesters, we have designed a strategy for chem. ligating oligonucleotide tags using cycloaddn. chem.  We have utilized this strategy in the construction and selection of a small mol. library, and successfully identified inhibitors of the enzyme sol. epoxide hydrolase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGlK1fEcxcbbVg90H21EOLACvtfcHk0lhqhcaN5km2_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyiu7bI&md5=5251dd6ce343cae4639b9cae82dea08f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fsrep10916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep10916%26sid%3Dliteratum%253Aachs%26aulast%3DLitovchick%26aufirst%3DA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DHabeshian%26aufirst%3DS.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DGui%25C3%25A9%26aufirst%3DM.-A.%26aulast%3DCentrella%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DEncoded%2520Library%2520Synthesis%2520Using%2520Chemical%2520Ligation%2520and%2520the%2520Discovery%2520of%2520sEH%2520Inhibitors%2520from%2520a%2520334-Million%2520Member%2520Library%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D10916%26doi%3D10.1038%2Fsrep10916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livak, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmittgen, T. D.</span></span> <span> </span><span class="NLM_article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">402</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1006/meth.2001.1262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1006%2Fmeth.2001.1262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=11846609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtFelt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2001&pages=402-408&author=K.+J.+Livakauthor=T.+D.+Schmittgen&title=Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28-Delta+Delta+C%28T%29%29+Method&doi=10.1006%2Fmeth.2001.1262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of relative gene expression data using real-time quantitative PCR and the 2-ΔΔCT method</span></div><div class="casAuthors">Livak, Kenneth J.; Schmittgen, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">402-408</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The two most commonly used methods to analyze data from real-time, quant. PCR expts. are abs. quantification and relative quantification.  Abs. quantification dets. the input copy no., usually by relating the PCR signal to a std. curve.  Relative quantification relates the PCR signal of the target transcript in a treatment group to that of another sample such as an untreated control.  The 2-ΔΔCT method is a convenient way to analyze the relative changes in gene expression from real-time quant. PCR expts.  The purpose of this report is to present the derivation, assumptions, and applications of the 2-ΔΔCT method.  In addn., we present the derivation and applications of two variations of the 2-ΔΔCT method that may be useful in the anal. of real-time, quant. PCR data.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXNe1s-mDpprVg90H21EOLACvtfcHk0lhqhcaN5km2_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtFelt7s%253D&md5=f849383250ea43f7380ed540db8a64b1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1006%2Fmeth.2001.1262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fmeth.2001.1262%26sid%3Dliteratum%253Aachs%26aulast%3DLivak%26aufirst%3DK.%2BJ.%26aulast%3DSchmittgen%26aufirst%3DT.%2BD.%26atitle%3DAnalysis%2520of%2520relative%2520gene%2520expression%2520data%2520using%2520real-time%2520quantitative%2520PCR%2520and%2520the%25202%2528-Delta%2520Delta%2520C%2528T%2529%2529%2520Method%26jtitle%3DMethods%26date%3D2001%26volume%3D25%26spage%3D402%26epage%3D408%26doi%3D10.1006%2Fmeth.2001.1262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref200':['cit200'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic for DECL  screening and PROTAC array platforms. (a) DNA-encoded chemical library (combinatorial synthesized small molecules tethered to DNA tags) is incubated with ERα, affinity captured, washed to remove non-binding library, and then enriched library members are eluted. After repeating an additional round, the DNA tags are sequenced to identify the small molecule binders. (b) Azide variants of the binders are prepared and reacted in a parallel click chemistry library of E3 binder alkynes to obtain an array of PROTACs for ERα.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. DECL selection output and off-DNA activity confirmation of two related members. (a, b) Schematic of two structurally related 2-cycle library members and their enrichment (ENRv1) suggests they bind ERα WT and mutants Y537S, S463P, and D538G. These library members do not bind the matrix (NTC; no target control) and are competed off by the presence of Estradiol (E2), suggesting that these library members bind in the ligand binding site. (c) Analogous “off-DNA” synthesized exemplars, with the linker and DNA truncated to a single methyl, bind in fluorescence polarization binding displacement assay. Assay results are an average of duplicate runs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Biochemical, cellular, and mouse PK properties of ERα PROTACs, <b>18</b> and <b>21</b>. (a) Structures of <b>18</b> and <b>21</b>. (b) Biochemical inhibition using purified recombinant ERα WT and mutants D538G, Y537Y, and S463P. (c) Degradation of ERα in MCF7 cells treated for 24 h followed by in-cell western analysis of ERα protein normalized to tubulin. The graphs represent normalized ERα levels. (d) MCF7 viability assay with cells treated for 7 days followed by an end-point viability by CellTiter-Glo. (e) Downstream PGR expression analysis in MCF7 cells treated for 24 h followed by RNA extraction, first strand synthesis, and qPCR analysis for progesterone receptor RNA estimation, normalized to TBP (Tata-binding protein); graphs represent normalized PGR levels. (f) Mean plasma concentration (μM) for 10 mpk (mg/kg) subcutaneous dosing in CD1 mouse (<i>n</i> = 3). Curves shown are from a single run with technical replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. ERα PROTACs inhibit and degrade ERα in MCF7 cells. (a) Time course degradation of ERα treated with Fulvestrant (1 μM) or <b>18</b> (1 μM). Cells were lysed and assayed by western blot, and tubulin was used as a normalizer control. (b) In-cell western evaluating the persistence of degradation effect after a 2 h treatment, compound washout and cell collection after another 22 h, compared to full 24 h compound treatment. Continued ERα reduction was observed after the washout. ERα normalized to tubulin control. (c) Proteasomal degradation of ERα is VHL-mediated by observing that co-treating with VHL298 (5 μM, literature <i>K</i><sub>d</sub> = 90 nM) resulted in a higher DC<sub>50</sub> compared to compound alone, 48 h total treatment time followed by collection. (d) Degradation of ERα is significantly impaired in the negative control prolinol epimer <b>23</b> compared to the analogous active prolinol epimer <b>18</b>, showing that degradation is proceeding through a VHL-based mechanism. (e) Structure of VHL298. All traces and DC<sub>50</sub> values are from a single experiment with technical replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. ERα PROTACs demonstrate on-target effects in ER+ cell lines and not in ER– cell lines. (a) MCF7 and T47D cell lines were treated with increasing doses of <b>18</b> and <b>21</b> for 24 h showing ERα degradation. Cell lysates were prepared and western blotted for ERα with Actin as a loading control. (B) ER-positive (MCF7, T47D, and ZR-75-1) and ER-negative breast cancer cell lines (MBA-MB231) and immortalized breast cell line (MCF10A) were treated with increasing doses of ERα PROTACs, <b>18</b> and <b>21</b>, in full serum conditions for 7 days. Relative cell numbers were assessed using CellTiter-Glo.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/medium/jm1c00127_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. <i>In vivo</i> efficacy and PK/PD study demonstrating efficacy in MCF7 xenograft model. (a) Antitumor activity of <b>18</b> and <b>21</b> compared to Fulvestrant in WT Estrogen-dependent MCF7 xenograft model. <b>18</b> and <b>21</b> were administered subcutaneously once daily at 10 mg/kg. Fulvestrant was dosed SC QW at 3 mg/mouse. Data represent the mean tumor volume ± SEM (<i>n</i> = 8). *<i>p</i> < 0.05 versus vehicle control (Student’s <i>t</i>-test). (b) All compounds were well tolerated with no changes in body weight. (c, d) Terminal Plasma PK and tumor PK of <b>18</b> and <b>21</b> at 3 h post-last dose (Day 20; <b>18</b>) and Day 27; <b>21</b>). (e) PD analysis from tumor bearing mice treated with compounds mentioned above for 3 days. Western blot analysis was performed to demonstrate reduction in ERα levels upon treatment with <b>18</b>, <b>21</b>, and Fulvestrant.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.1c00127/20210415/images/large/jm1c00127_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00127&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62865" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62865" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soerjomataram, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torre, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span>. <i>CA Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.3322/caac.21492</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.3322%2Fcaac.21492" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30207593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&author=F.+Brayauthor=J.+Ferlayauthor=I.+Soerjomataramauthor=R.+L.+Siegelauthor=L.+A.+Torreauthor=A.+Jemal&title=Global+cancer+statistics+2018%3A+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+countries&doi=10.3322%2Fcaac.21492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span></div><div class="casAuthors">Bray Freddie; Ferlay Jacques; Soerjomataram Isabelle; Siegel Rebecca L; Torre Lindsey A; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">394-424</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This article provides a status report on the global burden of cancer worldwide using the GLOBOCAN 2018 estimates of cancer incidence and mortality produced by the International Agency for Research on Cancer, with a focus on geographic variability across 20 world regions.  There will be an estimated 18.1 million new cancer cases (17.0 million excluding nonmelanoma skin cancer) and 9.6 million cancer deaths (9.5 million excluding nonmelanoma skin cancer) in 2018.  In both sexes combined, lung cancer is the most commonly diagnosed cancer (11.6% of the total cases) and the leading cause of cancer death (18.4% of the total cancer deaths), closely followed by female breast cancer (11.6%), prostate cancer (7.1%), and colorectal cancer (6.1%) for incidence and colorectal cancer (9.2%), stomach cancer (8.2%), and liver cancer (8.2%) for mortality.  Lung cancer is the most frequent cancer and the leading cause of cancer death among males, followed by prostate and colorectal cancer (for incidence) and liver and stomach cancer (for mortality).  Among females, breast cancer is the most commonly diagnosed cancer and the leading cause of cancer death, followed by colorectal and lung cancer (for incidence), and vice versa (for mortality); cervical cancer ranks fourth for both incidence and mortality.  The most frequently diagnosed cancer and the leading cause of cancer death, however, substantially vary across countries and within each country depending on the degree of economic development and associated social and life style factors.  It is noteworthy that high-quality cancer registry data, the basis for planning and implementing evidence-based cancer control programs, are not available in most low- and middle-income countries.  The Global Initiative for Cancer Registry Development is an international partnership that supports better estimation, as well as the collection and use of local data, to prioritize and evaluate national cancer control efforts.  CA: A Cancer Journal for Clinicians 2018;0:1-31. © 2018 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaW5wv8ocuhLxanA7vODO6fW6udTcc2eYwgQKb-EOUV7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D&md5=48529f8f794092779d70a27eec5c9017</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21492%26sid%3Dliteratum%253Aachs%26aulast%3DBray%26aufirst%3DF.%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DSoerjomataram%26aufirst%3DI.%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DTorre%26aufirst%3DL.%2BA.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DGlobal%2520cancer%2520statistics%25202018%253A%2520GLOBOCAN%2520estimates%2520of%2520incidence%2520and%2520mortality%2520worldwide%2520for%252036%2520cancers%2520in%2520185%2520countries%26jtitle%3DCA%2520Cancer%2520J.%2520Clin.%26date%3D2018%26volume%3D68%26spage%3D394%26epage%3D424%26doi%3D10.3322%2Fcaac.21492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siersbaek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span> <span> </span><span class="NLM_article-title">Signaling pathways and steroid receptors modulating estrogen receptor alpha function in breast cancer</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1141</span>– <span class="NLM_lpage">1154</span>, <span class="refDoi"> DOI: 10.1101/gad.316646.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1101%2Fgad.316646.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30181360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVKjtr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=1141-1154&author=R.+Siersbaekauthor=S.+Kumarauthor=J.+S.+Carroll&title=Signaling+pathways+and+steroid+receptors+modulating+estrogen+receptor+alpha+function+in+breast+cancer&doi=10.1101%2Fgad.316646.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Signaling pathways and steroid receptors modulating estrogen receptor α function in breast cancer</span></div><div class="casAuthors">Siersbaek, Rasmus; Kumar, Sanjeev; Carroll, Jason S.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">17-18</span>),
    <span class="NLM_cas:pages">1141-1154</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Estrogen receptor α(ER) is the major driver of ∼75% of breast cancers, and multiple ER targeting drugs are routinely used clin. to treat patients with ER+ breast cancer.  However, many patients relapse on these targeted therapies and ultimately develop metastatic and incurable disease, and understanding the mechanisms leading to drug resistance is consequently of utmost importance.  It is now clear that, in addn. to estrogens, ER function is modulated by other steroid receptors and multiple signaling pathways (e.g., growth factor and cytokine signaling), and many of these pathways affect drug resistance and patient outcome.  Here, we review the mechanisms through which these pathways impact ER function and drug resistance as well as discuss the clin. implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou8_pK06gF4LVg90H21EOLACvtfcHk0ljGTBtu8Q_UHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVKjtr3E&md5=d5dda5bf6f5ba7f2b578acc2e73733c4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1101%2Fgad.316646.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.316646.118%26sid%3Dliteratum%253Aachs%26aulast%3DSiersbaek%26aufirst%3DR.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26atitle%3DSignaling%2520pathways%2520and%2520steroid%2520receptors%2520modulating%2520estrogen%2520receptor%2520alpha%2520function%2520in%2520breast%2520cancer%26jtitle%3DGenes%2520Dev.%26date%3D2018%26volume%3D32%26spage%3D1141%26epage%3D1154%26doi%3D10.1101%2Fgad.316646.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waks, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span> <span> </span><span class="NLM_article-title">Breast Cancer Treatment: A Review</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">288</span>– <span class="NLM_lpage">300</span>, <span class="refDoi"> DOI: 10.1001/jama.2018.19323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1001%2Fjama.2018.19323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30667505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvV2qt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2019&pages=288-300&author=A.+G.+Waksauthor=E.+P.+Winer&title=Breast+Cancer+Treatment%3A+A+Review&doi=10.1001%2Fjama.2018.19323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Breast cancer treatment: a review</span></div><div class="casAuthors">Waks, Adrienne G.; Winer, Eric P.</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">288-300</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">1538-3598</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">A review.  Importance Breast cancer will be diagnosed in 12% of women in the United States over the course of their lifetimes and more than 250 000 new cases of breast cancer were diagnosed in the United States in 2017.  This focuses on current approaches and evolving strategies for local and systemic therapy of breast cancer. observations Breast cancer is categorized into 3 major subtypes based on the presence or absence of mol. markers for estrogen or progesterone receptors and human epidermal growth factor 2 (ERBB2; formerly HER2): hormone receptor pos./ERBB2 neg. (70% of patients), ERBB2 pos. (15%-20%), and triple-neg. (tumors lacking all 3 std. mol. markers; 15%).  More than 90% of breast cancers are not metastatic at the time of diagnosis.  For people presenting without metastatic disease, therapeutic goals are tumor eradication and preventing recurrence.  Triple-neg. breast cancer is more likely to recur than the other 2 subtypes, with 85% 5-yr breast cancer-specific survival for stage I triple-neg. tumors vs 94% to 99% for hormone receptor pos. and ERBB2 pos.  Systemic therapy for nonmetastatic breast cancer is detd. by subtype: patients with hormone receptor-pos. tumors receive endocrine therapy, and a minority receive chemotherapy as well; patients with ERBB2-pos. tumors receive ERBB2-targeted antibody or small-mol. inhibitor therapy combined with chemotherapy; and patients with triple-neg. tumors receive chemotherapy alone.  Local therapy for all patients with nonmetastatic breast cancer consists of surgical resection, with consideration of postoperative radiation if lumpectomy is performed.  Increasingly, some systemic therapy is delivered before surgery.  Tailoring postoperative treatment based on preoperative treatment response is under investigation.  Metastatic breast cancer is treated according to subtype, with goals of prolonging life and palliating symptoms.  Median overall survival for metastatic triple-neg. breast cancer is approx. 1 yr vs approx. 5 years for the other 2 subtypes. conclusions and relevance Breast cancer consists of 3 major tumor subtypes categorized according to estrogen or progesterone receptor expression and ERBB2 gene amplification.  The 3 subtypes have distinct risk profiles and treatment strategies.  Optimal therapy for each patient depends on tumor subtype, anat. cancer stage, and patient preferences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriTXwcPIcZ_LVg90H21EOLACvtfcHk0ljGTBtu8Q_UHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvV2qt7c%253D&md5=548a8f22dc89eb559f95d779b37f3a9d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1001%2Fjama.2018.19323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2018.19323%26sid%3Dliteratum%253Aachs%26aulast%3DWaks%26aufirst%3DA.%2BG.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26atitle%3DBreast%2520Cancer%2520Treatment%253A%2520A%2520Review%26jtitle%3DJAMA%26date%3D2019%26volume%3D321%26spage%3D288%26epage%3D300%26doi%3D10.1001%2Fjama.2018.19323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span> <span> </span><span class="NLM_article-title">Targeting estrogen receptor alpha for degradation with PROTACs: A promising approach to overcome endocrine resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>206</i></span>,  <span class="NLM_fpage">112689</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.ejmech.2020.112689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32829249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1GjsbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=206&publication_year=2020&pages=112689&author=X.+Linauthor=H.+Xiangauthor=G.+Luo&title=Targeting+estrogen+receptor+alpha+for+degradation+with+PROTACs%3A+A+promising+approach+to+overcome+endocrine+resistance&doi=10.1016%2Fj.ejmech.2020.112689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting estrogen receptor α for degradation with PROTACs: A promising approach to overcome endocrine resistance</span></div><div class="casAuthors">Lin, Xin; Xiang, Hua; Luo, Guoshun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">206</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112689</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Estrogen receptor alfa (ERα) is expressed in approx. 70% of breast cancer (BC) which is the most common malignancy in women worldwide.  To date, the foremost intervention in the treatment of ER pos. (ER+) BC is still the endocrine therapy.  However, resistance to endocrine therapies remains a major hurdle in the long-term management of ER + BC.  Although the mechanisms underlying endocrine resistance are complex, cumulative evidence revealed that ERα still plays a crit. role in driving BC tumor cells to grow in resistance state.  Fulvestrant, a selective estrogen receptor degrader (SERD), has moved to first line therapy for metastatic ER + BC, suggesting that removing ERα would be a useful strategy to overcome endocrine resistance.  Proteolysis-Targeting Chimera (PROTAC) technol., an emerging paradigm for protein degrdn., has the potential to eliminate both wild type and mutant ERα in breast cancer cells.  Excitingly, ARV-471, an ERα-targeted PROTAC developed by Arvinas, has been in phase 1 clin. trials.  In this review, we will summarize recent progress of ER-targeting PROTACs from publications and patents along with their therapeutic opportunities for the treatment of endocrine-resistant BC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL6a9CQdHKErVg90H21EOLACvtfcHk0ljGTBtu8Q_UHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1GjsbjK&md5=522029b5f85322bbe26f3e307acd51db</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112689%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DXiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DG.%26atitle%3DTargeting%2520estrogen%2520receptor%2520alpha%2520for%2520degradation%2520with%2520PROTACs%253A%2520A%2520promising%2520approach%2520to%2520overcome%2520endocrine%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D206%26spage%3D112689%26doi%3D10.1016%2Fj.ejmech.2020.112689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shagufta</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, S.</span></span> <span> </span><span class="NLM_article-title">Recent progress in selective estrogen receptor downregulators (SERDs) for the treatment of breast cancer</span>. <i>RSC Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">454</span>, <span class="refDoi"> DOI: 10.1039/C9MD00570F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1039%2FC9MD00570F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktl2qsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=438-454&author=+Shaguftaauthor=I.+Ahmadauthor=S.+Mathewauthor=S.+Rahman&title=Recent+progress+in+selective+estrogen+receptor+downregulators+%28SERDs%29+for+the+treatment+of+breast+cancer&doi=10.1039%2FC9MD00570F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in selective estrogen receptor downregulators (SERDs) for the treatment of breast cancer</span></div><div class="casAuthors">Shagufta; Ahmad, Irshad; Mathew, Shimy; Rahman, Sofia</div><div class="citationInfo"><span class="NLM_cas:title">RSC Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">438-454</span>CODEN:
                <span class="NLM_cas:coden">RMCSEZ</span>;
        ISSN:<span class="NLM_cas:issn">2632-8682</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Selective estrogen receptor downregulators (SERDs) are a novel class of compds. capable of reducing the ERα protein level and blocking ER activity.  Therefore, SERDs are considered as a significant therapeutic approach to treat ER+ breast cancer in both early stage and more advanced drug-resistant cases.  After the FDA approval of a steroidal drug, fulvestrant, as a SERD for the treatment of breast cancer in patients who have progressed on antihormonal agents, several mols. with diverse chem. structures have been rapidly developed, studied and evaluated for selective estrogen receptor downregulation activity.  Here we compile the promising SERDs reported in recent years and discuss the chem. structure and pharmacol. profile of the most potent compd. of the considered series.  Because of the availability of only a limited no. of effective drugs for the treatment of breast cancer, the quest for a potent SERD with respectable activity and bioavailability is still ongoing.  The goal of this article is to make available to the reader an overview of the current progress in SERDs and provide clues for the future discovery and development of novel pharmacol. potent SERDs for the treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSZ-tHcNc5-LVg90H21EOLACvtfcHk0ljGTBtu8Q_UHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktl2qsrY%253D&md5=73ad3f6be94f660cf47b13146781aa04</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2FC9MD00570F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00570F%26sid%3Dliteratum%253Aachs%26aulast%3DShagufta%26aulast%3DAhmad%26aufirst%3DI.%26aulast%3DMathew%26aufirst%3DS.%26aulast%3DRahman%26aufirst%3DS.%26atitle%3DRecent%2520progress%2520in%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%2520for%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DRSC%2520Med.%2520Chem.%26date%3D2020%26volume%3D11%26spage%3D438%26epage%3D454%26doi%3D10.1039%2FC9MD00570F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gombos, A.</span></span> <span> </span><span class="NLM_article-title">Selective oestrogen receptor degraders in breast cancer: a review and perspectives</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1097/CCO.0000000000000567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1097%2FCCO.0000000000000567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31335829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1alsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=424-429&author=A.+Gombos&title=Selective+oestrogen+receptor+degraders+in+breast+cancer%3A+a+review+and+perspectives&doi=10.1097%2FCCO.0000000000000567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Selective oestrogen receptor degraders in breast cancer: a review and perspectives</span></div><div class="casAuthors">Gombos, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">424-429</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: Estrogen receptor-pos. breast cancer accounts for 70% of all breast cancers.  Sequential endocrine treatment in monotherapy or in combination with CDK 4/6 or m-TOR inhibitors is the mainstay of recommended treatment options in the management of metastatic breast cancer even in the presence of visceral metastasis.  There is an emerging need to address endocrine resistance, which despite highly efficacious treatment combinations still can develop.  Recent findings: One of the mechanisms of endocrine resistance is mol. alteration of the estrogen receptor itself, such as ESR1 mutations affecting the ligand-binding domain.  These mutations emerge under the selective pressure of aromatase inhibitors.  The efficacy of selective estrogen receptor degraders (SERDs) might not be affected by the presence of mol. alterations of estrogen receptor.  Fulvestrant is the only SERD used in current clin. practice.  Numerous novel, nonsteroidal orally available SERDs are currently in clin. development.  Efficacious estrogen receptor target engagement and promising clin. activity was shown in early phase clin. trials.  Summary: Therefore, a new class of orally available nonsteroidal SERDs gains high interest in tackling endocrine resistance in estrogen receptor-pos. (ER+) advanced breast cancer.  Clin. efficacy needs to be confirmed in larger patient populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9D3cmdEVhP7Vg90H21EOLACvtfcHk0lg6s410wtSVpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1alsrbL&md5=45f46bc71806fb9e79e642c0e9307191</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1097%2FCCO.0000000000000567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0000000000000567%26sid%3Dliteratum%253Aachs%26aulast%3DGombos%26aufirst%3DA.%26atitle%3DSelective%2520oestrogen%2520receptor%2520degraders%2520in%2520breast%2520cancer%253A%2520a%2520review%2520and%2520perspectives%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2019%26volume%3D31%26spage%3D424%26epage%3D429%26doi%3D10.1097%2FCCO.0000000000000567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gombos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzoi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awada, A.</span></span> <span> </span><span class="NLM_article-title">Investigational drugs in early stage clinical trials for the treatment of HER2+ breast cancer</span>. <i>Expert Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">617</span>– <span class="NLM_lpage">627</span>, <span class="refDoi"> DOI: 10.1080/13543784.2019.1633306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1080%2F13543784.2019.1633306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31230485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1emsL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=617-627&author=A.+Gombosauthor=M.+A.+Franzoiauthor=A.+Awada&title=Investigational+drugs+in+early+stage+clinical+trials+for+the+treatment+of+HER2%2B+breast+cancer&doi=10.1080%2F13543784.2019.1633306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Investigational drugs in early stage clinical trials for the treatment of HER2+ breast cancer</span></div><div class="casAuthors">Gombos, Andrea; Franzoi, Maria Alice; Awada, Ahmad</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">617-627</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : Despite improvements in the management of HER2+ breast cancer, metastatic disease is still fatal.  Usually, these patients receive several lines of chemotherapy assocd. with HER2 targeted treatments.  Most of the trials using innovative approaches are positioning themselves in disease that is resistant to pertuzumab and trastuzumab emtansine (TDM1).: We describe the recent advances in clin. development of anti-HER2 treatments.  To this aim, we used literature search via Pubmed and made an inventory of abstrs. published during the last two years in major oncol. conferences.: Further changes will probably occur during the next decade in the management of metastatic HER2-pos. breast cancer.  This is mainly driven by the fact that the two mainstay drugs (pertuzumab and TDM-1) that confer prolonged survival (56 mo) to these patients are currently being used in the treatment of early-stage disease in a subset of patients.  Thus, there is an urgent need to develop new, innovative approaches in those patients whose disease has become resistant to these highly potent drugs.  Several new antibody-drug conjugates, bispecific antibodies or new generation tyrosine kinase inhibitor (TKIs) hold promise and should be assessed and compared with drugs currently used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0xmhTyI5v4bVg90H21EOLACvtfcHk0lg6s410wtSVpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1emsL3M&md5=0b1e769bd87571565526d06d91b7e06b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1080%2F13543784.2019.1633306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543784.2019.1633306%26sid%3Dliteratum%253Aachs%26aulast%3DGombos%26aufirst%3DA.%26aulast%3DFranzoi%26aufirst%3DM.%2BA.%26aulast%3DAwada%26aufirst%3DA.%26atitle%3DInvestigational%2520drugs%2520in%2520early%2520stage%2520clinical%2520trials%2520for%2520the%2520treatment%2520of%2520HER2%252B%2520breast%2520cancer%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2019%26volume%3D28%26spage%3D617%26epage%3D627%26doi%3D10.1080%2F13543784.2019.1633306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nayar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapstad, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuoco, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waks, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wander, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persky, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helvie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenblatt-Rosen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, N. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagle, N.</span></span> <span> </span><span class="NLM_article-title">Acquired HER2 mutations in ER(+) metastatic breast cancer confer resistance to estrogen receptor-directed therapies</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">216</span>, <span class="refDoi"> DOI: 10.1038/s41588-018-0287-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fs41588-018-0287-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30531871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhtbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2019&pages=207-216&author=U.+Nayarauthor=O.+Cohenauthor=C.+Kapstadauthor=M.+S.+Cuocoauthor=A.+G.+Waksauthor=S.+A.+Wanderauthor=C.+Painterauthor=S.+Freemanauthor=N.+S.+Perskyauthor=L.+Mariniauthor=K.+Helvieauthor=N.+Oliverauthor=O.+Rozenblatt-Rosenauthor=C.+X.+Maauthor=A.+Regevauthor=E.+P.+Winerauthor=N.+U.+Linauthor=N.+Wagle&title=Acquired+HER2+mutations+in+ER%28%2B%29+metastatic+breast+cancer+confer+resistance+to+estrogen+receptor-directed+therapies&doi=10.1038%2Fs41588-018-0287-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired HER2 mutations in ER+ metastatic breast cancer confer resistance to estrogen receptor-directed therapies</span></div><div class="casAuthors">Nayar, Utthara; Cohen, Ofir; Kapstad, Christian; Cuoco, Michael S.; Waks, Adrienne G.; Wander, Seth A.; Painter, Corrie; Freeman, Samuel; Persky, Nicole S.; Marini, Lori; Helvie, Karla; Oliver, Nelly; Rozenblatt-Rosen, Orit; Ma, Cynthia X.; Regev, Aviv; Winer, Eric P.; Lin, Nancy U.; Wagle, Nikhil</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">207-216</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Seventy percent of breast cancers express the estrogen receptor (ER), and agents that target the ER are the mainstay of treatment.  However, virtually all people with ER+ breast cancer develop resistance to ER-directed agents in the metastatic setting.  Beyond mutations in the ER itself, which occur in 25-30% of people treated with aromatase inhibitors1-4, knowledge about clin. resistance mechanisms remains incomplete.  We identified activating HER2 mutations in metastatic biopsies from eight patients with ER+ metastatic breast cancer who had developed resistance to aromatase inhibitors, tamoxifen or fulvestrant.  Examn. of treatment-naive primary tumors in five patients showed no evidence of pre-existing mutations in four of five patients, suggesting that these mutations were acquired under the selective pressure of ER-directed therapy.  The HER2 mutations and ER mutations were mutually exclusive, suggesting a distinct mechanism of acquired resistance to ER-directed therapies.  In vitro anal. confirmed that the HER2 mutations conferred estrogen independence as well as-in contrast to ER mutations-resistance to tamoxifen, fulvestrant and the CDK4 and CDK6 inhibitor palbociclib.  Resistance was overcome by combining ER-directed therapy with the irreversible HER2 kinase inhibitor neratinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPW14nLf7qarVg90H21EOLACvtfcHk0lg6s410wtSVpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhtbbM&md5=8c9f16d4a94653efd684fd43f356cd16</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41588-018-0287-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-018-0287-5%26sid%3Dliteratum%253Aachs%26aulast%3DNayar%26aufirst%3DU.%26aulast%3DCohen%26aufirst%3DO.%26aulast%3DKapstad%26aufirst%3DC.%26aulast%3DCuoco%26aufirst%3DM.%2BS.%26aulast%3DWaks%26aufirst%3DA.%2BG.%26aulast%3DWander%26aufirst%3DS.%2BA.%26aulast%3DPainter%26aufirst%3DC.%26aulast%3DFreeman%26aufirst%3DS.%26aulast%3DPersky%26aufirst%3DN.%2BS.%26aulast%3DMarini%26aufirst%3DL.%26aulast%3DHelvie%26aufirst%3DK.%26aulast%3DOliver%26aufirst%3DN.%26aulast%3DRozenblatt-Rosen%26aufirst%3DO.%26aulast%3DMa%26aufirst%3DC.%2BX.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26aulast%3DLin%26aufirst%3DN.%2BU.%26aulast%3DWagle%26aufirst%3DN.%26atitle%3DAcquired%2520HER2%2520mutations%2520in%2520ER%2528%252B%2529%2520metastatic%2520breast%2520cancer%2520confer%2520resistance%2520to%2520estrogen%2520receptor-directed%2520therapies%26jtitle%3DNat.%2520Genet.%26date%3D2019%26volume%3D51%26spage%3D207%26epage%3D216%26doi%3D10.1038%2Fs41588-018-0287-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J.-A.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptor mutations and functional consequences for breast cancer</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">476</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2015.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.tem.2015.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=26183887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeisLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2015&pages=467-476&author=C.+Thomasauthor=J.-A.+Gustafsson&title=Estrogen+receptor+mutations+and+functional+consequences+for+breast+cancer&doi=10.1016%2Fj.tem.2015.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen receptor mutations and functional consequences for breast cancer</span></div><div class="casAuthors">Thomas, Christoforos; Gustafsson, Jan-Ake</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">467-476</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A significant no. of estrogen receptor α (ERα)-pos. breast tumors develop resistance to endocrine therapy and recur with metastatic disease.  Several mechanisms of endocrine resistance have been proposed, including genetic alterations that lead to ERs with altered protein sequence.  By altering the conformation of the protein and increasing the interaction with coactivators, point mutations in ESR1, the gene encoding ERα, promote an active form of the receptor in the absence of hormone that assists tumor cells to evade hormonal treatments.  Recent studies have confirmed that ESR1 point mutations frequently occur in metastatic breast tumors that are refractory to endocrine therapy, and suggest the development of novel strategies that may be more effective in controlling ER signaling and benefit patients with recurrent and metastatic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_CLOzUVLBLVg90H21EOLACvtfcHk0lgZj0zXZFHadw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeisLfI&md5=d6b1f2fe49c77b991f953e802514a32e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2015.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2015.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DC.%26aulast%3DGustafsson%26aufirst%3DJ.-A.%26atitle%3DEstrogen%2520receptor%2520mutations%2520and%2520functional%2520consequences%2520for%2520breast%2520cancer%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2015%26volume%3D26%26spage%3D467%26epage%3D476%26doi%3D10.1016%2Fj.tem.2015.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alluri, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptor mutations and their role in breast cancer progression</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">494</span>, <span class="refDoi"> DOI: 10.1186/s13058-014-0494-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1186%2Fs13058-014-0494-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=25928204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=494&author=P.+G.+Alluriauthor=C.+Speersauthor=A.+M.+Chinnaiyan&title=Estrogen+receptor+mutations+and+their+role+in+breast+cancer+progression&doi=10.1186%2Fs13058-014-0494-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen receptor mutations and their role in breast cancer progression</span></div><div class="casAuthors">Alluri, Prasanna G.; Speers, Corey; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">494/1-494/8, 8 pp.</span>CODEN:
                <span class="NLM_cas:coden">BRCRFS</span>;
        ISSN:<span class="NLM_cas:issn">1465-542X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Endocrine therapy is the mainstay of treatment in estrogen receptor-pos. breast cancers and significantly reduces disease recurrence and breast cancer-related mortality.  However, acquired resistance to therapy has been noted in nearly one-third of women treated with tamoxifen and other endocrine therapies.  Mutations in the estrogen receptor have long been speculated to play a role in endocrine therapy resistance but have been rarely detected.  However, recent studies utilizing next-generation sequencing on estrogen receptor-pos., metastatic clin. samples have revealed that recurrent ESR1 mutations are far more frequent than previously thought and may play an important role in acquired endocrine therapy resistance.  Here we review recent advances in detection and characterization of ESR1 mutations in advanced, endocrine therapy-resistant breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokrNygDcI-MbVg90H21EOLACvtfcHk0lgZj0zXZFHadw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislWntr0%253D&md5=fae44eb9eba40ff0be77e776edc5633b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1186%2Fs13058-014-0494-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13058-014-0494-7%26sid%3Dliteratum%253Aachs%26aulast%3DAlluri%26aufirst%3DP.%2BG.%26aulast%3DSpeers%26aufirst%3DC.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DEstrogen%2520receptor%2520mutations%2520and%2520their%2520role%2520in%2520breast%2520cancer%2520progression%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2014%26volume%3D16%26spage%3D494%26doi%3D10.1186%2Fs13058-014-0494-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merenbakh-Lamin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Baruch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeheskel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvir, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soussan-Gutman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeselsohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yelensky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarid, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, I.</span></span> <span> </span><span class="NLM_article-title">D538G mutation in estrogen receptor-α: A novel mechanism for acquired endocrine resistance in breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6856</span>– <span class="NLM_lpage">6864</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1158%2F0008-5472.CAN-13-1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=24217577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKhurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6856-6864&author=K.+Merenbakh-Laminauthor=N.+Ben-Baruchauthor=A.+Yeheskelauthor=A.+Dvirauthor=L.+Soussan-Gutmanauthor=R.+Jeselsohnauthor=R.+Yelenskyauthor=M.+Brownauthor=V.+A.+Millerauthor=D.+Saridauthor=S.+Rizelauthor=B.+Kleinauthor=T.+Rubinekauthor=I.+Wolf&title=D538G+mutation+in+estrogen+receptor-%CE%B1%3A+A+novel+mechanism+for+acquired+endocrine+resistance+in+breast+cancer&doi=10.1158%2F0008-5472.CAN-13-1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">D538G Mutation in Estrogen Receptor-α: A Novel Mechanism for Acquired Endocrine Resistance in Breast Cancer</span></div><div class="casAuthors">Merenbakh-Lamin, Keren; Ben-Baruch, Noa; Yeheskel, Adva; Dvir, Addie; Soussan-Gutman, Lior; Jeselsohn, Rinath; Yelensky, Roman; Brown, Myles; Miller, Vincent A.; Sarid, David; Rizel, Shulamith; Klein, Baruch; Rubinek, Tami; Wolf, Ido</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6856-6864</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to endocrine therapy occurs in virtually all patients with estrogen receptor a (ERa)-pos. metastatic breast cancer, and is attributed to various mechanisms including loss of ERα expression, altered activity of coregulators, and cross-talk between the ERa and growth factor signaling pathways.  To our knowledge, acquired mutations of the ERα have not been described as mediating endocrine resistance.  Samples of 13 patients with metastatic breast cancer were analyzed for mutations in cancer-related genes.  In five patients who developed resistance to hormonal therapy, a mutation of A to G at position 1,613 of ERα, resulting in a substitution of aspartic acid at position 538 to glycine (D538G), was identified in liver metastases.  Importantly, the mutation was not detected in the primary tumors obtained prior to endocrine treatment.  Structural modeling indicated that D538G substitution leads to a conformational change in the ligand-binding domain, which mimics the conformation of activated ligand-bound receptor and alters binding of tamoxifen.  Indeed, expts. in breast cancer cells indicated constitutive, ligand-independent transcriptional activity of the D538G receptor, and overexpression of it enhanced proliferation and conferred resistance to tamoxifen.  These data indicate a novel mechanism of acquired endocrine resistance in breast cancer.  Further studies are needed to assess the frequency of D538G-ERα among patients with breast cancer and explore ways to inhibit its activity and restore endocrine sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWljQ_odNNUrVg90H21EOLACvtfcHk0lgZj0zXZFHadw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKhurfN&md5=6dca8ee6cdb77bfc8a94d02fe5630ea8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1197%26sid%3Dliteratum%253Aachs%26aulast%3DMerenbakh-Lamin%26aufirst%3DK.%26aulast%3DBen-Baruch%26aufirst%3DN.%26aulast%3DYeheskel%26aufirst%3DA.%26aulast%3DDvir%26aufirst%3DA.%26aulast%3DSoussan-Gutman%26aufirst%3DL.%26aulast%3DJeselsohn%26aufirst%3DR.%26aulast%3DYelensky%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DSarid%26aufirst%3DD.%26aulast%3DRizel%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DB.%26aulast%3DRubinek%26aufirst%3DT.%26aulast%3DWolf%26aufirst%3DI.%26atitle%3DD538G%2520mutation%2520in%2520estrogen%2520receptor-%25CE%25B1%253A%2520A%2520novel%2520mechanism%2520for%2520acquired%2520endocrine%2520resistance%2520in%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6856%26epage%3D6864%26doi%3D10.1158%2F0008-5472.CAN-13-1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vats, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonigro, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyana-Sundaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlins, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roychowdhury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pienta, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rae, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Poznak, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chugh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunju, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talpaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schott, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Activating ESR1 mutations in hormone-resistant metastatic breast cancer</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1446</span>– <span class="NLM_lpage">1451</span>, <span class="refDoi"> DOI: 10.1038/ng.2823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fng.2823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=24185510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWju7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=1446-1451&author=D.+R.+Robinsonauthor=Y.+M.+Wuauthor=P.+Vatsauthor=F.+Suauthor=R.+J.+Lonigroauthor=X.+Caoauthor=S.+Kalyana-Sundaramauthor=R.+Wangauthor=Y.+Ningauthor=L.+Hodgesauthor=A.+Gurskyauthor=J.+Siddiquiauthor=S.+A.+Tomlinsauthor=S.+Roychowdhuryauthor=K.+J.+Pientaauthor=S.+Y.+Kimauthor=J.+S.+Robertsauthor=J.+M.+Raeauthor=C.+H.+Van+Poznakauthor=D.+F.+Hayesauthor=R.+Chughauthor=L.+P.+Kunjuauthor=M.+Talpazauthor=A.+F.+Schottauthor=A.+M.+Chinnaiyan&title=Activating+ESR1+mutations+in+hormone-resistant+metastatic+breast+cancer&doi=10.1038%2Fng.2823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Activating ESR1 mutations in hormone-resistant metastatic breast cancer</span></div><div class="casAuthors">Robinson, Dan R.; Wu, Yi-Mi; Vats, Pankaj; Su, Fengyun; Lonigro, Robert J.; Cao, Xuhong; Kalyana-Sundaram, Shanker; Wang, Rui; Ning, Yu; Hodges, Lynda; Gursky, Amy; Siddiqui, Javed; Tomlins, Scott A.; Roychowdhury, Sameek; Pienta, Kenneth J.; Kim, Scott Y.; Roberts, J. Scott; Rae, James M.; Van Poznak, Catherine H.; Hayes, Daniel F.; Chugh, Rashmi; Kunju, Lakshmi P.; Talpaz, Moshe; Schott, Anne F.; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1446-1451</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Breast cancer is the most prevalent cancer in women, and over two-thirds of cases express estrogen receptor-α (ER-α, encoded by ESR1).  Through a prospective clin. sequencing program for advanced cancers, we enrolled 11 patients with ER-pos. metastatic breast cancer.  Whole-exome and transcriptome anal. showed that six cases harbored mutations of ESR1 affecting its ligand-binding domain (LBD), all of whom had been treated with anti-estrogens and estrogen deprivation therapies.  A survey of The Cancer Genome Atlas (TCGA) identified four endometrial cancers with similar mutations of ESR1.  The five new LBD-localized ESR1 mutations identified here (encoding p.Leu536Gln, p.Tyr537Ser, p.Tyr537Cys, p.Tyr537Asn and p.Asp538Gly) were shown to result in constitutive activity and continued responsiveness to anti-estrogen therapies in vitro.  Taken together, these studies suggest that activating mutations in ESR1 are a key mechanism in acquired endocrine resistance in breast cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtQSL6mlhMOrVg90H21EOLACvtfcHk0liBf3-c0dIOTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWju7%252FN&md5=af9e69d6064726af238fe74dad7d2204</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fng.2823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2823%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DD.%2BR.%26aulast%3DWu%26aufirst%3DY.%2BM.%26aulast%3DVats%26aufirst%3DP.%26aulast%3DSu%26aufirst%3DF.%26aulast%3DLonigro%26aufirst%3DR.%2BJ.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DKalyana-Sundaram%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DHodges%26aufirst%3DL.%26aulast%3DGursky%26aufirst%3DA.%26aulast%3DSiddiqui%26aufirst%3DJ.%26aulast%3DTomlins%26aufirst%3DS.%2BA.%26aulast%3DRoychowdhury%26aufirst%3DS.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DRoberts%26aufirst%3DJ.%2BS.%26aulast%3DRae%26aufirst%3DJ.%2BM.%26aulast%3DVan%2BPoznak%26aufirst%3DC.%2BH.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChugh%26aufirst%3DR.%26aulast%3DKunju%26aufirst%3DL.%2BP.%26aulast%3DTalpaz%26aufirst%3DM.%26aulast%3DSchott%26aufirst%3DA.%2BF.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DActivating%2520ESR1%2520mutations%2520in%2520hormone-resistant%2520metastatic%2520breast%2520cancer%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D1446%26epage%3D1451%26doi%3D10.1038%2Fng.2823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razavi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goeppert, A. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Stanchina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uddin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanning, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzenellenbogen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandarlapaty, S.</span></span> <span> </span><span class="NLM_article-title">Activating ESR1 Mutations Differentially Affect the Efficacy of ER Antagonists</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">287</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-15-1523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1158%2F2159-8290.CD-15-1523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=27986707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Klsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=277-287&author=W.+Toyauthor=H.+Weirauthor=P.+Razaviauthor=M.+Lawsonauthor=A.+U.+Goeppertauthor=A.+M.+Mazzolaauthor=A.+Smithauthor=J.+Wilsonauthor=C.+Morrowauthor=W.+L.+Wongauthor=E.+De+Stanchinaauthor=K.+E.+Carlsonauthor=T.+S.+Martinauthor=S.+Uddinauthor=Z.+Liauthor=S.+Fanningauthor=J.+A.+Katzenellenbogenauthor=G.+Greeneauthor=J.+Baselgaauthor=S.+Chandarlapaty&title=Activating+ESR1+Mutations+Differentially+Affect+the+Efficacy+of+ER+Antagonists&doi=10.1158%2F2159-8290.CD-15-1523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Activating ESR1 Mutations Differentially Affect the Efficacy of ER Antagonists</span></div><div class="casAuthors">Toy, Weiyi; Weir, Hazel; Razavi, Pedram; Lawson, Mandy; Goeppert, Anne U.; Mazzola, Anne Marie; Smith, Aaron; Wilson, Joanne; Morrow, Christopher; Wong, Wai Lin; De Stanchina, Elisa; Carlson, Kathryn E.; Martin, Teresa S.; Uddin, Sharmeen; Li, Zhiqiang; Fanning, Sean; Katzenellenbogen, John A.; Greene, Geoffrey; Baselga, Jose; Chandarlapaty, Sarat</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-287</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Recent studies have identified somatic ESR1 mutations in patients with metastatic breast cancer and found some of them to promote estrogen-independent activation of the receptor.  The degree to which all recurrent mutants can drive estrogen-independent activities and reduced sensitivity to ER antagonists like fulvestrant is not established.  In this report, we characterize the spectrum of ESR1 mutations from more than 900 patients.  ESR1 mutations were detected in 10%, with D538G being the most frequent (36%), followed by Y537S (14%).  Several novel, activating mutations were also detected (e.g., L469V, V422del, and Y537D).  Although many mutations lead to constitutive activity and reduced sensitivity to ER antagonists, only select mutants such as Y537S caused a magnitude of change assocd. with fulvestrant resistance in vivo.  Correspondingly, tumors driven by Y537S, but not D5358G, E380Q, or S463P, were less effectively inhibited by fulvestrant than more potent and bioavailable antagonists, including AZD9496.  These data point to a need for antagonists with optimal pharmacokinetic properties to realize clin. efficacy against certain ESR1 mutants.  Significance: A diversity of activating ESR1 mutations exist, only some of which confer resistance to existing ER antagonists that might be overcome by next-generation inhibitors such as AZD9496.  Cancer Discov; 7(3); 277-87. ©2016 AACR.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZyS-eS6T9ZLVg90H21EOLACvtfcHk0lgKcLhEDlMzAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Klsbo%253D&md5=398dd5202db358ba8facab67055895c9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-1523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-1523%26sid%3Dliteratum%253Aachs%26aulast%3DToy%26aufirst%3DW.%26aulast%3DWeir%26aufirst%3DH.%26aulast%3DRazavi%26aufirst%3DP.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DGoeppert%26aufirst%3DA.%2BU.%26aulast%3DMazzola%26aufirst%3DA.%2BM.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DMorrow%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DW.%2BL.%26aulast%3DDe%2BStanchina%26aufirst%3DE.%26aulast%3DCarlson%26aufirst%3DK.%2BE.%26aulast%3DMartin%26aufirst%3DT.%2BS.%26aulast%3DUddin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DFanning%26aufirst%3DS.%26aulast%3DKatzenellenbogen%26aufirst%3DJ.%2BA.%26aulast%3DGreene%26aufirst%3DG.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DChandarlapaty%26aufirst%3DS.%26atitle%3DActivating%2520ESR1%2520Mutations%2520Differentially%2520Affect%2520the%2520Efficacy%2520of%2520ER%2520Antagonists%26jtitle%3DCancer%2520Discovery%26date%3D2017%26volume%3D7%26spage%3D277%26epage%3D287%26doi%3D10.1158%2F2159-8290.CD-15-1523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pooh, A.</span></span> <span> </span><span class="NLM_article-title">Proof-of-Concept with PROTACs in Prostate Cancer</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1084</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=1084&author=A.+Pooh&title=Proof-of-Concept+with+PROTACs+in+Prostate+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPooh%26aufirst%3DA.%26atitle%3DProof-of-Concept%2520with%2520PROTACs%2520in%2520Prostate%2520Cancer%26jtitle%3DCancer%2520Discovery%26date%3D2020%26volume%3D10%26spage%3D1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC Technology: Opportunities and Challenges</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=237-240&author=H.+Gaoauthor=X.+Sunauthor=Y.+Rao&title=PROTAC+Technology%3A+Opportunities+and+Challenges&doi=10.1021%2Facsmedchemlett.9b00597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC Technology: Opportunities and Challenges</span></div><div class="casAuthors">Gao Hongying; Sun Xiuyun; Rao Yu; Gao Hongying; Sun Xiuyun</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">237-240</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">PROTACs-induced targeted protein degradation has emerged as a novel therapeutic strategy in drug development and attracted the favor of academic institutions, large pharmaceutical enterprises (e.g., AstraZeneca, Bayer, Novartis, Amgen, Pfizer, GlaxoSmithKline, Merck, and Boehringer Ingelheim, etc.), and biotechnology companies.  PROTACs opened a new chapter for novel drug development.  However, any new technology will face many new problems and challenges.  Perspectives on the potential opportunities and challenges of PROTACs will contribute to the research and development of new protein degradation drugs and degrader tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSj8qlWczrRKmx8TEN9K18QfW6udTcc2eYbOicvZ5COvbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D&md5=68988d6477e83f4ebc26d11e3b8ce5b1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00597%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC%2520Technology%253A%2520Opportunities%2520and%2520Challenges%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D237%26epage%3D240%26doi%3D10.1021%2Facsmedchemlett.9b00597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=64&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2ebcrR79pZPtHrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D64%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimera (PROTAC) for targeted protein degradation and cancer therapy</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">50</span>, <span class="refDoi"> DOI: 10.1186/s13045-020-00885-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1186%2Fs13045-020-00885-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32404196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB38vmsFGntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2020&pages=50&author=X.+Liauthor=Y.+Song&title=Proteolysis-targeting+chimera+%28PROTAC%29+for+targeted+protein+degradation+and+cancer+therapy&doi=10.1186%2Fs13045-020-00885-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-targeting chimera (PROTAC) for targeted protein degradation and cancer therapy</span></div><div class="casAuthors">Li Xin; Song Yongcheng; Song Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteolysis-targeting chimera (PROTAC) has been developed to be a useful technology for targeted protein degradation.  A bifunctional PROTAC molecule consists of a ligand (mostly small-molecule inhibitor) of the protein of interest (POI) and a covalently linked ligand of an E3 ubiquitin ligase (E3).  Upon binding to the POI, the PROTAC can recruit E3 for POI ubiquitination, which is subjected to proteasome-mediated degradation.  PROTAC complements nucleic acid-based gene knockdown/out technologies for targeted protein reduction and could mimic pharmacological protein inhibition.  To date, PROTACs targeting ~ 50 proteins, many of which are clinically validated drug targets, have been successfully developed with several in clinical trials for cancer therapy.  This article reviews PROTAC-mediated degradation of critical oncoproteins in cancer, particularly those in hematological malignancies.  Chemical structures, cellular and in vivo activities, pharmacokinetics, and pharmacodynamics of these PROTACs are summarized.  In addition, potential advantages, challenges, and perspectives of PROTAC technology in cancer therapy are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxVqxtAax1Ha-9TkgmBqTqfW6udTcc2ebcrR79pZPtHrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38vmsFGntA%253D%253D&md5=baf560d35ef911d2be3d8b4c6d4e0191</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2Fs13045-020-00885-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-020-00885-3%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DProteolysis-targeting%2520chimera%2520%2528PROTAC%2529%2520for%2520targeted%2520protein%2520degradation%2520and%2520cancer%2520therapy%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2020%26volume%3D13%26spage%3D50%26doi%3D10.1186%2Fs13045-020-00885-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, H.</span></span> <span> </span><span class="NLM_article-title">Development of cell-permeable peptide-based PROTACs targeting estrogen receptor α</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>187</i></span>,  <span class="NLM_fpage">111967</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111967</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.ejmech.2019.111967" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31865016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVKgtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2020&pages=111967&author=Y.+Daiauthor=N.+Yueauthor=J.+Gongauthor=C.+Liuauthor=Q.+Liauthor=J.+Zhouauthor=W.+Huangauthor=H.+Qian&title=Development+of+cell-permeable+peptide-based+PROTACs+targeting+estrogen+receptor+%CE%B1&doi=10.1016%2Fj.ejmech.2019.111967"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of cell-permeable peptide-based PROTACs targeting estrogen receptor α</span></div><div class="casAuthors">Dai, Yuxuan; Yue, Na; Gong, Junni; Liu, Chunxia; Li, Qifei; Zhou, Jiaqi; Huang, Wenlong; Qian, Hai</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">187</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111967</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Proteolysis-targeting chimera (PROTAC) could selectively degrade target protein and may become a promising strategy for treating estrogen receptor α (ERα) pos. breast cancers.  Here, we designed penetrated peptide-based PROTACs by constructing an N-terminal lactam cyclic to improve proteolytic stability and cell penetration.  We used a lactam cyclic peptide as ERα binding ligand, 6-aminocaproic acid as a linker, and a hydroxylated pentapeptide structure for recruiting E3 ligase to obtain heterobifunctional compds.  The resulting optimized compd. I-6 selectively recruited ERα to the E3 ligase complex for promoting the degrdn. of ERα.  Compd. I-6 possessed strong effect on MCF-7 cell toxicity (IC50 ∼9.7μM) and significantly enhanced activities in inducing ERα degrdn.  Meanwhile, I-6 performed much stronger potency in inhibition of tumors growth than tamoxifen.  This work is a successful template to construct PROTACs based on cell-permeable peptides, which could extend the chem. space of PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxg4zaUgSlt7Vg90H21EOLACvtfcHk0lgQyIv4uEdd4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVKgtL7F&md5=0bd840b597c93d7f06ef7384277beade</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111967%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DY.%26aulast%3DYue%26aufirst%3DN.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DQian%26aufirst%3DH.%26atitle%3DDevelopment%2520of%2520cell-permeable%2520peptide-based%2520PROTACs%2520targeting%2520estrogen%2520receptor%2520%25CE%25B1%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D187%26spage%3D111967%26doi%3D10.1016%2Fj.ejmech.2019.111967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span> <span> </span><span class="NLM_article-title">Targeting Nuclear Receptors with PROTAC degraders</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>493</i></span>,  <span class="NLM_fpage">110452</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2019.110452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.mce.2019.110452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31125586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFensL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=493&publication_year=2019&pages=110452&author=J.+J.+Flanaganauthor=T.+K.+Neklesa&title=Targeting+Nuclear+Receptors+with+PROTAC+degraders&doi=10.1016%2Fj.mce.2019.110452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Nuclear Receptors with PROTAC degraders</span></div><div class="casAuthors">Flanagan, John J.; Neklesa, Taavi K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">493</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">110452</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  Nuclear receptors comprise a class of intracellular transcription factors whose major role is to act as sensors of various stimuli and to convert the external signal into a transcriptional output.  Nuclear receptors (NRs) achieve this by possessing a ligand binding domain, which can bind cell permeable agonists, a DNA-binding domain, which binds the upstream sequences of target genes, and a regulatory domain that recruits the transcriptional machinery.  The ligand binding alters the activation state of the NR, either by activating or inactivating its transcriptional output.  Given the central role of NRs in signal transduction, many currently approved therapeutics modulate the activity of NRs.  Here we discuss how PROTAC degraders afford a novel approach to abrogate the downstream signaling activity of NRs.  We highlight six broad functional reasons why PROTAC degraders are preferable to the classical ligand binding pocket antagonists, with specific examples provided for each category.  Lastly, as Androgen Receptor and Estrogen Receptor PROTAC degraders are being pursued as treatment for prostate cancer and breast cancer, resp., a rationale is provided for the translational utility for the degrdn. of these two NRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0pFL-LkrjELVg90H21EOLACvtfcHk0lgQyIv4uEdd4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFensL%252FN&md5=8988ee44083a37b65282af214a432d1e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2019.110452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2019.110452%26sid%3Dliteratum%253Aachs%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26atitle%3DTargeting%2520Nuclear%2520Receptors%2520with%2520PROTAC%2520degraders%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2019%26volume%3D493%26spage%3D110452%26doi%3D10.1016%2Fj.mce.2019.110452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.+Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+Highly+Potent+Proteolysis+Targeting+Chimera+%28PROTAC%29+Degrader+of+Estrogen+Receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0ljOr-XB0HHEjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520Highly%2520Potent%2520Proteolysis%2520Targeting%2520Chimera%2520%2528PROTAC%2529%2520Degrader%2520of%2520Estrogen%2520Receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span> <span> </span><span class="NLM_article-title">Two-Stage Strategy for Development of Proteolysis Targeting Chimeras and its Application for Estrogen Receptor Degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1487</span>– <span class="NLM_lpage">1496</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00140</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFKru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1487-1496&author=B.+L.+Robertsauthor=Z.+X.+Maauthor=A.+Gaoauthor=E.+D.+Leistenauthor=D.+Yinauthor=W.+Xuauthor=W.+Tang&title=Two-Stage+Strategy+for+Development+of+Proteolysis+Targeting+Chimeras+and+its+Application+for+Estrogen+Receptor+Degraders&doi=10.1021%2Facschembio.0c00140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Two-Stage Strategy for Development of Proteolysis Targeting Chimeras and its Application for Estrogen Receptor Degraders</span></div><div class="casAuthors">Roberts, Brett L.; Ma, Zhi-Xiong; Gao, Ang; Leisten, Eric D.; Yin, Dan; Xu, Wei; Tang, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1487-1496</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) have emerged as useful chem. probes and potential therapeutics by taking advantage of the ubiquitin-proteasome system to degrade intracellular disease-assocd. proteins.  PROTACs are heterobifunctional mols. composed of a target protein ligand, E3 ubiquitin ligase ligand, and a linker between them.  The generation of efficient PROTACs requires screening of many parameters, esp. the lengths and types of the linkers.  The authors report the proof-of-concept study using a two-stage strategy to facilitate the development of PROTACs against the estrogen receptor (ER).  In stage one, a library of close to 100 PROTACs was synthesized by simply mixing a library of ERα ligands contg. a hydrazide functional group at different positions with a preassembled library of E3 ligase ligands bearing different types and lengths of linkers with a terminal aldehyde group in a 1:1 ratio.  Cell-based screening occurred without further purifn., because the formation of the acylhydrazone linkage is highly efficient and produces water as the only byproduct.  Compd. A3 was the most potent ER degrader in two ER+ cell lines (DC50= ~ 10 nM, Dmax= ≥ 95%).  Stage two involved transformation to a more stable amide linker to generate a more drug-like mol.  The new compd., AM-A3, showed comparable biol. activity (DC50 = 1.1 nM, Dmax = 98%) and induced potent antiproliferation (IC50= 13.2 nM, Imax= 69%) in MCF-7.  This proof-of -concept study demonstrates that the two-stage strategy can significantly facilitate the development of PROTACs against ER without the tedious process of making large nos. of PROTACs one by one.  It has the potential to be expanded to many other targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLYFrBdI2Sy7Vg90H21EOLACvtfcHk0ljOr-XB0HHEjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFKru7w%253D&md5=80e8aeeba613ecff7d98944e7a579225</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00140%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DB.%2BL.%26aulast%3DMa%26aufirst%3DZ.%2BX.%26aulast%3DGao%26aufirst%3DA.%26aulast%3DLeisten%26aufirst%3DE.%2BD.%26aulast%3DYin%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DW.%26atitle%3DTwo-Stage%2520Strategy%2520for%2520Development%2520of%2520Proteolysis%2520Targeting%2520Chimeras%2520and%2520its%2520Application%2520for%2520Estrogen%2520Receptor%2520Degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D1487%26epage%3D1496%26doi%3D10.1021%2Facschembio.0c00140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cyrus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehenkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span> <span> </span><span class="NLM_article-title">Jostling for position: optimizing linker location in the design of estrogen receptor-targeting PROTACs</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">979</span>– <span class="NLM_lpage">985</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201000146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1002%2Fcmdc.201000146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=20512796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVOmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=979-985&author=K.+Cyrusauthor=M.+Wehenkelauthor=E.+Y.+Choiauthor=H.+Leeauthor=H.+Swansonauthor=K.+B.+Kim&title=Jostling+for+position%3A+optimizing+linker+location+in+the+design+of+estrogen+receptor-targeting+PROTACs&doi=10.1002%2Fcmdc.201000146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Jostling for Position: Optimizing Linker Location in the Design of Estrogen Receptor-Targeting PROTACs</span></div><div class="casAuthors">Cyrus, Kedra; Wehenkel, Marie; Choi, Eun-Young; Lee, Hyosung; Swanson, Hollie; Kim, Kyung-Bo</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">979-985</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Estrogen receptor-α (ER) antagonists have been widely used for breast cancer therapy.  Despite initial responsiveness, hormone-sensitive ER-pos. cancer cells eventually develop resistance to ER antagonists.  It has been shown that in most of these resistant tumor cells, the ER is expressed and continues to regulate tumor growth.  Recent studies indicate that tamoxifen initially acts as an antagonist, but later functions as an ER agonist, promoting tumor growth.  This suggests that targeted ER degrdn. may provide an effective therapeutic approach for breast cancers, even those that are resistant to conventional therapies.  With this in mind, we previously demonstrated that proteolysis targeting chimeras (PROTACs) effectively induce degrdn. of the ER as a proof-of-concept expt.  Herein we further refined the PROTAC approach to target the ER for degrdn.  The ER-targeting PROTACs are composed of an estradiol on one end and a hypoxia-inducing factor 1α (HIF-1α)-derived synthetic pentapeptide on the other.  The pentapeptide is recognized by an E3 ubiquitin ligase called the von Hippel Lindau tumor suppressor protein (pVHL), thereby recruiting the ER to this E3 ligase for ubiquitination and degrdn.  Specifically, the pentapeptide is attached at three different locations on estradiol to generate three different PROTAC types.  With the pentapeptide linked through the C7α position of estradiol, the resulting PROTAC shows the most effective ER degrdn. and highest affinity for the estrogen receptor.  This result provides an opportunity to develop a novel type of ER antagonist that may overcome the resistance of breast tumors to conventional drugs such as tamoxifen and fulvestrant (Faslodex).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYQTb73o13F7Vg90H21EOLACvtfcHk0ljOr-XB0HHEjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVOmt74%253D&md5=1bd032e4640fe04d72b1a69d746fdada</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000146%26sid%3Dliteratum%253Aachs%26aulast%3DCyrus%26aufirst%3DK.%26aulast%3DWehenkel%26aufirst%3DM.%26aulast%3DChoi%26aufirst%3DE.%2BY.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DSwanson%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DK.%2BB.%26atitle%3DJostling%2520for%2520position%253A%2520optimizing%2520linker%2520location%2520in%2520the%2520design%2520of%2520estrogen%2520receptor-targeting%2520PROTACs%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D979%26epage%3D985%26doi%3D10.1002%2Fcmdc.201000146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, Andrew P.</span>; <span class="NLM_string-name">Flanagin, John</span>; <span class="NLM_string-name">Gough Sheryl, Maxine</span>; <span class="NLM_string-name">Haskell Royal, J.</span>; <span class="NLM_string-name">Moore Marcia, D.</span>; <span class="NLM_string-name">Qian, Yimin</span>; <span class="NLM_string-name">Taylor Ian, C. A.</span>; <span class="NLM_string-name">Wang, Jing</span>; <span class="NLM_string-name">Xin, C.</span></span> <span> </span><span class="NLM_article-title">Methods of treating breast cancer with tetrahydronaphthalene derivatives as Estrogen Receptor Degraders</span>, Patent Application <span class="NLM_patent">WO/2021/041348</span>. March 4, <span class="NLM_year">2021</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&author=Andrew+P.+Crew&author=Flanagin+John&author=Gough+Sheryl+Maxine&author=J.+Haskell+Royal&author=D.+Moore+Marcia&author=Qian+Yimin&author=C.+A.+Taylor+Ian&author=Wang+Jing&author=C.+Xin&title=Methods+of+treating+breast+cancer+with+tetrahydronaphthalene+derivatives+as+Estrogen+Receptor+Degraders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DAndrew%2BP.%26atitle%3DMethods%2520of%2520treating%2520breast%2520cancer%2520with%2520tetrahydronaphthalene%2520derivatives%2520as%2520Estrogen%2520Receptor%2520Degraders%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Irrizary, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czabaniuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niosi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span> <span> </span><span class="NLM_article-title">Delineating the role of cooperativity in the design of potent PROTACs for BTK</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E7285</span>– <span class="NLM_lpage">E7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.1803662115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1073%2Fpnas.1803662115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=30012605" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E7285-E7292&author=A.+Zorbaauthor=C.+Nguyenauthor=Y.+Xuauthor=J.+Starrauthor=K.+Borzilleriauthor=J.+Smithauthor=H.+Zhuauthor=K.+A.+Farleyauthor=W.+Dingauthor=J.+Schiemerauthor=X.+Fengauthor=J.+S.+Changauthor=D.+P.+Uccelloauthor=J.+A.+Youngauthor=C.+N.+Garcia-Irrizaryauthor=L.+Czabaniukauthor=B.+Schuffauthor=R.+Oliverauthor=J.+Montgomeryauthor=M.+M.+Haywardauthor=J.+Coeauthor=J.+Chenauthor=M.+Niosiauthor=S.+Luthraauthor=J.+C.+Shahauthor=A.+El-Kattanauthor=X.+Qiuauthor=G.+M.+Westauthor=M.+C.+Noeauthor=V.+Shanmugasundaramauthor=A.+M.+Gilbertauthor=M.+F.+Brownauthor=M.+F.+Calabrese&title=Delineating+the+role+of+cooperativity+in+the+design+of+potent+PROTACs+for+BTK&doi=10.1073%2Fpnas.1803662115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1803662115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1803662115%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DSchiemer%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DGarcia-Irrizary%26aufirst%3DC.%2BN.%26aulast%3DCzabaniuk%26aufirst%3DL.%26aulast%3DSchuff%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNiosi%26aufirst%3DM.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DJ.%2BC.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26atitle%3DDelineating%2520the%2520role%2520of%2520cooperativity%2520in%2520the%2520design%2520of%2520potent%2520PROTACs%2520for%2520BTK%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3DE7285%26epage%3DE7292%26doi%3D10.1073%2Fpnas.1803662115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjamin, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creaser, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzen, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gefter, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavarana, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vloten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1038/nchembio.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kelleyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries&doi=10.1038%2Fnchembio.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lhsygl2o2F5lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelley%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3Dvan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654%26doi%3D10.1038%2Fnchembio.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grady, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakdawala, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerfield, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded Library Screening Identifies Benzo[b][1,4]oxazepin-4-ones as Highly Potent and Monoselective Receptor Interacting Protein 1 Kinase Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2163</span>– <span class="NLM_lpage">2178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01898</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01898" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Kqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2163-2178&author=P.+A.+Harrisauthor=B.+W.+Kingauthor=D.+Bandyopadhyayauthor=S.+B.+Bergerauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=J.+N.+Fingerauthor=L.+C.+Gradyauthor=S.+J.+Hoffmanauthor=J.+U.+Jeongauthor=J.+Kangauthor=V.+Kasparcovaauthor=A.+S.+Lakdawalaauthor=R.+Lehrauthor=D.+E.+McNultyauthor=R.+Nagillaauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+R.+Rendinaauthor=M.+C.+Schaefferauthor=J.+D.+Summerfieldauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=P.+Wardauthor=A.+Zhangauthor=D.+Zhangauthor=R.+W.+Marquisauthor=J.+Bertinauthor=P.+J.+Gough&title=DNA-Encoded+Library+Screening+Identifies+Benzo%5Bb%5D%5B1%2C4%5Doxazepin-4-ones+as+Highly+Potent+and+Monoselective+Receptor+Interacting+Protein+1+Kinase+Inhibitors&doi=10.1021%2Facs.jmedchem.5b01898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Library Screening Identifies Benzo[b][1,4]oxazepin-4-ones as Highly Potent and Monoselective Receptor Interacting Protein 1 Kinase Inhibitors</span></div><div class="casAuthors">Harris, Philip A.; King, Bryan W.; Bandyopadhyay, Deepak; Berger, Scott B.; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Finger, Joshua N.; Grady, LaShadric C.; Hoffman, Sandra J.; Jeong, Jae U.; Kang, James; Kasparcova, Viera; Lakdawala, Ami S.; Lehr, Ruth; McNulty, Dean E.; Nagilla, Rakesh; Ouellette, Michael T.; Pao, Christina S.; Rendina, Alan R.; Schaeffer, Michelle C.; Summerfield, Jennifer D.; Swift, Barbara A.; Totoritis, Rachel D.; Ward, Paris; Zhang, Aming; Zhang, Daohua; Marquis, Robert W.; Bertin, John; Gough, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2163-2178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent discovery of the role of receptor interacting protein 1 (RIP1) kinase in tumor necrosis factor (TNF)-mediated inflammation has led to its emergence as a highly promising target for the treatment of multiple inflammatory diseases.  The authors screened RIP1 against GSK's DNA-encoded small-mol. libraries and identified a novel highly potent benzoxazepinone inhibitor series.  The authors demonstrate that this template possesses complete monokinase selectivity for RIP1 plus unique species selectivity for primate vs. nonprimate RIP1.  The authors elucidate the conformation of RIP1 bound to this benzoxazepinone inhibitor driving its high kinase selectivity and design specific mutations in murine RIP1 to restore potency to levels similar to primate RIP1.  This series differentiates itself from known RIP1 inhibitors in combining high potency and kinase selectivity with good pharmacokinetic profiles in rodents.  The favorable developability profile of this benzoxazepinone template, as exemplified by compd. 14 (GSK'481), makes it an excellent starting point for further optimization into a RIP1 clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4XvpziWGWg7Vg90H21EOLACvtfcHk0lgY2MjK9RML3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Kqsr8%253D&md5=d6325dcaed97b7c339d1830ca041298c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01898%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DGrady%26aufirst%3DL.%2BC.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DLakdawala%26aufirst%3DA.%2BS.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DMcNulty%26aufirst%3DD.%2BE.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSummerfield%26aufirst%3DJ.%2BD.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DBertin%26aufirst%3DJ.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26atitle%3DDNA-Encoded%2520Library%2520Screening%2520Identifies%2520Benzo%255Bb%255D%255B1%252C4%255Doxazepin-4-ones%2520as%2520Highly%2520Potent%2520and%2520Monoselective%2520Receptor%2520Interacting%2520Protein%25201%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2163%26epage%3D2178%26doi%3D10.1021%2Facs.jmedchem.5b01898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazaar, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span> <span> </span><span class="NLM_article-title">Discovering Drugs with DNA-Encoded Library Technology: From Concept to Clinic with an Inhibitor of Soluble Epoxide Hydrolase</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">837</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1002/cbic.201700014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1002%2Fcbic.201700014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=28281333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVWjurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=837-842&author=S.+L.+Belyanskayaauthor=Y.+Dingauthor=J.+F.+Callahanauthor=A.+L.+Lazaarauthor=D.+I.+Israel&title=Discovering+Drugs+with+DNA-Encoded+Library+Technology%3A+From+Concept+to+Clinic+with+an+Inhibitor+of+Soluble+Epoxide+Hydrolase&doi=10.1002%2Fcbic.201700014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering Drugs with DNA-Encoded Library Technology: From Concept to Clinic with an Inhibitor of Soluble Epoxide Hydrolase</span></div><div class="casAuthors">Belyanskaya, Svetlana L.; Ding, Yun; Callahan, James F.; Lazaar, Aili L.; Israel, David I.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">837-842</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">DNA-encoded chem. library technol. was developed with the vision of its becoming a transformational platform for drug discovery.  The hope was that a new paradigm for the discovery of low-mol.-wt. drugs would be enabled by combining the vast mol. diversity achievable with combinatorial chem., the information-encoding attributes of DNA, the power of mol. biol., and a streamlined selection-based discovery process.  Here, we describe the discovery and early clin. development of GSK2256294, an inhibitor of sol. epoxide hydrolase (sEH, EPHX2), by using encoded-library technol. (ELT).  GSK2256294 is an orally bioavailable, potent and selective inhibitor of sEH that has a long half life and produced no serious adverse events in a first-time-in-human clin. study.  To our knowledge, GSK2256294 is the first mol. discovered from this technol. to enter human clin. testing and represents a realization of the vision that DNA-encoded chem. library technol. can efficiently yield mols. with favorable properties that can be readily progressed into high-quality drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7WTPZp2SE0rVg90H21EOLACvtfcHk0lgY2MjK9RML3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVWjurk%253D&md5=a34e28c8817864356f6ff689e9236940</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201700014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201700014%26sid%3Dliteratum%253Aachs%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DCallahan%26aufirst%3DJ.%2BF.%26aulast%3DLazaar%26aufirst%3DA.%2BL.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26atitle%3DDiscovering%2520Drugs%2520with%2520DNA-Encoded%2520Library%2520Technology%253A%2520From%2520Concept%2520to%2520Clinic%2520with%2520an%2520Inhibitor%2520of%2520Soluble%2520Epoxide%2520Hydrolase%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D837%26epage%3D842%26doi%3D10.1002%2Fcbic.201700014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref200"><div class="reference"><strong class="refLabel"><a href="#ref200" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit200"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnicow, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruebsam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Novel autotaxin inhibitor for the treatment of idiopathic pulmonary fibrosis: a clinical candidate discovered using DNA-encoded chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7840</span>– <span class="NLM_lpage">7856</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00688</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00688" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7840-7856&author=J.+W.+Cuozzoauthor=M.+A.+Clarkauthor=A.+D.+Keefeauthor=A.+Kohlmannauthor=M.+Mulvihillauthor=H.+Niauthor=L.+M.+Renzettiauthor=D.+I.+Resnicowauthor=F.+Ruebsamauthor=E.+A.+Sigelauthor=H.+A.+Thomsonauthor=C.+Wangauthor=Z.+Xieauthor=Y.+Zhang&title=Novel+autotaxin+inhibitor+for+the+treatment+of+idiopathic+pulmonary+fibrosis%3A+a+clinical+candidate+discovered+using+DNA-encoded+chemistry&doi=10.1021%2Facs.jmedchem.0c00688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit200R"><div class="casContent"><span class="casTitleNuber">200</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Autotaxin Inhibitor for the Treatment of Idiopathic Pulmonary Fibrosis: A Clinical Candidate Discovered Using DNA-Encoded Chemistry</span></div><div class="casAuthors">Cuozzo, John W.; Clark, Matthew A.; Keefe, Anthony D.; Kohlmann, Anna; Mulvihill, Mark; Ni, Haihong; Renzetti, Louis M.; Resnicow, Daniel I.; Ruebsam, Frank; Sigel, Eric A.; Thomson, Heather A.; Wang, Ce; Xie, Zhifeng; Zhang, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7840-7856</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The activity of the secreted phosphodiesterase autotaxin produces the inflammatory signaling mol. LPA and has been assocd. with a no. of human diseases including idiopathic pulmonary fibrosis (IPF).  We screened a single DNA-encoded chem. library (DECL) of 225 million compds. and identified a series of potent inhibitors.  Optimization of this series led to the discovery of compd. 1 (X-165)(I), a highly potent, selective, and bioavailable small mol.  Cocrystn. of compd. 1 with human autotaxin demonstrated that it has a novel binding mode occupying both the hydrophobic pocket and a channel near the autotaxin active site.  Compd. 1 inhibited the prodn. of LPA in human and mouse plasma at nanomolar levels and showed efficacy in a mouse model of human lung fibrosis.  After successfully completing IND-enabling studies, compd. 1 was approved by the FDA for a Phase I clin. trial.  These results demonstrate that DECL hits can be readily optimized into clin. candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaofuGyFvOJrVg90H21EOLACvtfcHk0liOodTxmaIYag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CrtL%252FM&md5=c945f9bb71089c94314a27afce04fb96</span></div><a href="/servlet/linkout?suffix=cit200&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00688%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DKohlmann%26aufirst%3DA.%26aulast%3DMulvihill%26aufirst%3DM.%26aulast%3DNi%26aufirst%3DH.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DResnicow%26aufirst%3DD.%2BI.%26aulast%3DRuebsam%26aufirst%3DF.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DNovel%2520autotaxin%2520inhibitor%2520for%2520the%2520treatment%2520of%2520idiopathic%2520pulmonary%2520fibrosis%253A%2520a%2520clinical%2520candidate%2520discovered%2520using%2520DNA-encoded%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7840%26epage%3D7856%26doi%3D10.1021%2Facs.jmedchem.0c00688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folkesson, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristensen, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slok, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakskjaer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madsen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azevedo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micco, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span> <span> </span><span class="NLM_article-title">Screening of DNA-Encoded Small Molecule Libraries inside a Living Cell</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">2751</span>– <span class="NLM_lpage">2756</span>, <span class="refDoi"> DOI: 10.1021/jacs.0c09213</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c09213" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjvVeitro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2021&pages=2751-2756&author=L.+K.+Petersenauthor=A.+B.+Christensenauthor=J.+Andersenauthor=C.+G.+Folkessonauthor=O.+Kristensenauthor=C.+Andersenauthor=A.+Alzuauthor=F.+A.+Slokauthor=P.+Blakskjaerauthor=D.+Madsenauthor=C.+Azevedoauthor=I.+Miccoauthor=N.+J.+V.+Hansen&title=Screening+of+DNA-Encoded+Small+Molecule+Libraries+inside+a+Living+Cell&doi=10.1021%2Fjacs.0c09213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Screening of DNA-Encoded Small Molecule Libraries inside a Living Cell</span></div><div class="casAuthors">Petersen, Lars K.; Christensen, Allan B.; Andersen, Jacob; Folkesson, Charlotta G.; Kristensen, Ole; Andersen, Charlotte; Alzu, Amaya; Sloek, Frank A.; Blakskjaer, Peter; Madsen, Daniel; Azevedo, Carlos; Micco, Iolanda; Hansen, Nils J. V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2751-2756</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded small mol. libraries (DELs) have facilitated the discovery of novel modulators of many different therapeutic protein targets.  We report the first successful screening of a multimillion membered DEL inside a living cell.  We demonstrate a novel method using oocytes from the South African clawed frog Xenopus laevis.  The large size of the oocytes of 1 μL, or 100 000 times bigger than a normal somatic cell, permits simple injection of DELs, thus resolving the fundamental problem of delivering DELs across cell membranes for in vivo screening.  The target protein was expressed in the oocytes fused to a prey protein, to allow specific DNA labeling and hereby discriminate between DEL members binding to the target protein and the endogenous cell proteins.  The 194 million member DEL was screened against three pharmaceutically relevant protein targets, p38α, ACSS2, and DOCK5.  For all three targets multiple chem. clusters were identified.  For p38α, validated hits with single digit nanomolar potencies were obtained.  This work demonstrates a powerful new approach to DEL screening, which eliminates the need for highly purified active target protein and which performs the screening under physiol. relevant conditions and thus is poised to increase the DEL amenable target space and reduce the attrition rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu0SRtLn2gZLVg90H21EOLACvtfcHk0liOodTxmaIYag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjvVeitro%253D&md5=6735556674600c60712bb74f171e50f3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c09213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c09213%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DL.%2BK.%26aulast%3DChristensen%26aufirst%3DA.%2BB.%26aulast%3DAndersen%26aufirst%3DJ.%26aulast%3DFolkesson%26aufirst%3DC.%2BG.%26aulast%3DKristensen%26aufirst%3DO.%26aulast%3DAndersen%26aufirst%3DC.%26aulast%3DAlzu%26aufirst%3DA.%26aulast%3DSlok%26aufirst%3DF.%2BA.%26aulast%3DBlakskjaer%26aufirst%3DP.%26aulast%3DMadsen%26aufirst%3DD.%26aulast%3DAzevedo%26aufirst%3DC.%26aulast%3DMicco%26aufirst%3DI.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26atitle%3DScreening%2520of%2520DNA-Encoded%2520Small%2520Molecule%2520Libraries%2520inside%2520a%2520Living%2520Cell%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2021%26volume%3D143%26spage%3D2751%26epage%3D2756%26doi%3D10.1021%2Fjacs.0c09213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCloskey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kearnes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moccia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazzaz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guie, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guilinger, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulhern, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, P.</span></span> <span> </span><span class="NLM_article-title">Machine Learning on DNA-Encoded Libraries: A New Paradigm for Hit Finding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8857</span>– <span class="NLM_lpage">8866</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00452</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00452" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFamtLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8857-8866&author=K.+McCloskeyauthor=E.+A.+Sigelauthor=S.+Kearnesauthor=L.+Xueauthor=X.+Tianauthor=D.+Mocciaauthor=D.+Gikunjuauthor=S.+Bazzazauthor=B.+Chanauthor=M.+A.+Clarkauthor=J.+W.+Cuozzoauthor=M.+A.+Guieauthor=J.+P.+Guilingerauthor=C.+Huguetauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=C.+J.+Mulhernauthor=Y.+Zhangauthor=P.+Riley&title=Machine+Learning+on+DNA-Encoded+Libraries%3A+A+New+Paradigm+for+Hit+Finding&doi=10.1021%2Facs.jmedchem.0c00452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Machine Learning on DNA-Encoded Libraries: A New Paradigm for Hit Finding</span></div><div class="casAuthors">McCloskey, Kevin; Sigel, Eric A.; Kearnes, Steven; Xue, Ling; Tian, Xia; Moccia, Dennis; Gikunju, Diana; Bazzaz, Sana; Chan, Betty; Clark, Matthew A.; Cuozzo, John W.; Guie, Marie-Aude; Guilinger, John P.; Huguet, Christelle; Hupp, Christopher D.; Keefe, Anthony D.; Mulhern, Christopher J.; Zhang, Ying; Riley, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8857-8866</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded small mol. libraries (DELs) have enabled discovery of novel inhibitors for many distinct protein targets of therapeutic value.  We demonstrate a new approach applying machine learning to DEL selection data by identifying active mols. from large libraries of com. and easily synthesizable compds.  We train models using only DEL selection data and apply automated or automatable filters to the predictions.  We perform a large prospective study (~ 2000 compds.) across three diverse protein targets: sEH (a hydrolase), ERα (a nuclear receptor), and c-KIT (a kinase).  The approach is effective, with an overall hit rate of ~ 30% at 30 μM and discovery of potent compds. (IC50 < 10 nM) for every target.  The system makes useful predictions even for mols. dissimilar to the original DEL, and the compds. identified are diverse, predominantly drug-like, and different from known ligands.  This work demonstrates a powerful new approach to hit-finding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1kezutJD3kbVg90H21EOLACvtfcHk0liOodTxmaIYag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFamtLjO&md5=1a2472a8c7d6a77ec2689b1e6034e2ec</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00452%26sid%3Dliteratum%253Aachs%26aulast%3DMcCloskey%26aufirst%3DK.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DKearnes%26aufirst%3DS.%26aulast%3DXue%26aufirst%3DL.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMoccia%26aufirst%3DD.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DBazzaz%26aufirst%3DS.%26aulast%3DChan%26aufirst%3DB.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DGuie%26aufirst%3DM.%2BA.%26aulast%3DGuilinger%26aufirst%3DJ.%2BP.%26aulast%3DHuguet%26aufirst%3DC.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DMulhern%26aufirst%3DC.%2BJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DRiley%26aufirst%3DP.%26atitle%3DMachine%2520Learning%2520on%2520DNA-Encoded%2520Libraries%253A%2520A%2520New%2520Paradigm%2520for%2520Hit%2520Finding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8857%26epage%3D8866%26doi%3D10.1021%2Facs.jmedchem.0c00452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wurz, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellamaggiore, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span> <span> </span><span class="NLM_article-title">A ″Click Chemistry Platform″ for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01781</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01781" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Gls7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=453-461&author=R.+P.+Wurzauthor=K.+Dellamaggioreauthor=H.+Douauthor=N.+Javierauthor=M.+C.+Loauthor=J.+D.+McCarterauthor=D.+Mohlauthor=C.+Sastriauthor=J.+R.+Lipfordauthor=V.+J.+Cee&title=A+%E2%80%B3Click+Chemistry+Platform%E2%80%B3+for+the+Rapid+Synthesis+of+Bispecific+Molecules+for+Inducing+Protein+Degradation&doi=10.1021%2Facs.jmedchem.6b01781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A "Click Chemistry Platform" for the Rapid Synthesis of Bispecific Molecules for Inducing Protein Degradation</span></div><div class="casAuthors">Wurz, Ryan P.; Dellamaggiore, Ken; Dou, Hannah; Javier, Noelle; Lo, Mei-Chu; McCarter, John D.; Mohl, Dane; Sastri, Christine; Lipford, J. Russell; Cee, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">453-461</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) are bispecific mols. contg. a target protein binder and an ubiquitin ligase binder connected by a linker.  By recruiting an ubiquitin ligase to a target protein, PROTACs promote ubiquitination and proteasomal degrdn. of the target protein.  The generation of effective PROTACs depends on the nature of the protein/ligase ligand pair, linkage site, linker length, and linker compn., all of which have been difficult to address in a systematic way.  Herein, the authors describe a "click chem." approach for the synthesis of PROTACs.  The authors demonstrate the utility of this approach with the bromodomain and extra terminal domain-4 (BRD4) ligand JQ-1 (3) and ligase binders targeting cereblon (CRBN) and Von Hippel-Lindau (VHL) proteins.  An AlphaScreen proximity assay was used to det. the ability of PROTACs to form the ternary ligase-PROTAC-target protein complex and a MSD assay to measure cellular degrdn. of the target protein promoted by PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpooapLJ2Zln7Vg90H21EOLACvtfcHk0lgw4yhFhBJJsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Gls7w%253D&md5=090c569ae1f079ab6504348aabec8909</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01781%26sid%3Dliteratum%253Aachs%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DDellamaggiore%26aufirst%3DK.%26aulast%3DDou%26aufirst%3DH.%26aulast%3DJavier%26aufirst%3DN.%26aulast%3DLo%26aufirst%3DM.%2BC.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DMohl%26aufirst%3DD.%26aulast%3DSastri%26aufirst%3DC.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26atitle%3DA%2520%25E2%2580%25B3Click%2520Chemistry%2520Platform%25E2%2580%25B3%2520for%2520the%2520Rapid%2520Synthesis%2520of%2520Bispecific%2520Molecules%2520for%2520Inducing%2520Protein%2520Degradation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D453%26epage%3D461%26doi%3D10.1021%2Facs.jmedchem.6b01781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faver, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancia, D. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milbank, J. B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matzuk, M. M.</span></span> <span> </span><span class="NLM_article-title">Quantitative Comparison of Enrichment from DNA-Encoded Chemical Library Selections</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.8b00116</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.8b00116" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsV2jtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=75-82&author=J.+C.+Faverauthor=K.+Riehleauthor=D.+R.+Lanciaauthor=J.+B.+J.+Milbankauthor=C.+S.+Kollmannauthor=N.+Simmonsauthor=Z.+Yuauthor=M.+M.+Matzuk&title=Quantitative+Comparison+of+Enrichment+from+DNA-Encoded+Chemical+Library+Selections&doi=10.1021%2Facscombsci.8b00116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative comparison of enrichment from DNA-encoded chemical library selections</span></div><div class="casAuthors">Faver, John C.; Riehle, Kevin; Lancia, David R., Jr.; Milbank, Jared B. J.; Kollmann, Christopher S.; Simmons, Nicholas; Yu, Zhifeng; Matzuk, Martin M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">75-82</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded chem. libraries (DELs) provide a high-throughput and cost-effective route for screening billions of unique mols. for binding affinity for diverse protein targets.  Identifying candidate compds. from these libraries involves affinity selection, DNA sequencing, and measuring enrichment in a sample pool of DNA barcodes.  Successful detection of potent binders is affected by many factors, including selection parameters, chem. yields, library amplification, sequencing depth, sequencing errors, library sizes, and the chosen enrichment metric.  To date, there has not been a clear consensus about how enrichment from DEL selections should be measured or reported.  We propose a normalized z-score enrichment metric using a binomial distribution model that satisfies important criteria that are relevant for anal. of DEL selection data.  The introduced metric is robust with respect to library diversity and sampling and allows for quant. comparisons of enrichment of n-synthons from parallel DEL selections.  These features enable a comparative enrichment anal. strategy that can provide valuable information about hit compds. in early stage drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplYUndBUCVTLVg90H21EOLACvtfcHk0lgw4yhFhBJJsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsV2jtrY%253D&md5=28bd49450ec839438006676ad36f2e36</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.8b00116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.8b00116%26sid%3Dliteratum%253Aachs%26aulast%3DFaver%26aufirst%3DJ.%2BC.%26aulast%3DRiehle%26aufirst%3DK.%26aulast%3DLancia%26aufirst%3DD.%2BR.%26aulast%3DMilbank%26aufirst%3DJ.%2BB.%2BJ.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DSimmons%26aufirst%3DN.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26atitle%3DQuantitative%2520Comparison%2520of%2520Enrichment%2520from%2520DNA-Encoded%2520Chemical%2520Library%2520Selections%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2019%26volume%3D21%26spage%3D75%26epage%3D82%26doi%3D10.1021%2Facscombsci.8b00116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Challinor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span> <span> </span><span class="NLM_article-title">A Screening Assay Cascade to Identify and Characterize Novel Selective Estrogen Receptor Downregulators (SERDs)</span>. <i>J. Biomol. Screen</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">748</span>– <span class="NLM_lpage">759</span>, <span class="refDoi"> DOI: 10.1177/1087057115580298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1177%2F1087057115580298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=25851036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=748-759&author=R.+Callisauthor=A.+Rabowauthor=M.+Tongeauthor=R.+Bradburyauthor=M.+Challinorauthor=K.+Robertsauthor=K.+Jonesauthor=G.+Walker&title=A+Screening+Assay+Cascade+to+Identify+and+Characterize+Novel+Selective+Estrogen+Receptor+Downregulators+%28SERDs%29&doi=10.1177%2F1087057115580298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A screening assay cascade to identify and characterize novel selective estrogen receptor downregulators (SERDs)</span></div><div class="casAuthors">Callis, Rowena; Rabow, Alfred; Tonge, Michael; Bradbury, Robert; Challinor, Mairi; Roberts, Karen; Jones, Karen; Walker, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">748-759</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Here, we describe an approach to identify novel selective estrogen receptor downregulator (SERD) compds. with improved properties such as oral bioavailability and the potential of increased efficacy compared to currently marketed drug treatments.  Previously, methodologies such as Western blotting and transient cell reporter assays have been used to identify and characterize SERD compds., but such approaches can be limited due to low throughput and sensitivity, resp.  We have used an endogenous cell-imaging strategy that has both the throughput and sensitivity to support a large-scale hit-to-lead program to identify novel compds.  A screening cascade with a suite of assays has been developed to characterize compds. that modulate estrogen receptor α (ERα)-mediated signaling or downregulate ERα levels in cells.  Initially, from a focused high-throughput screening, novel ERα binders were identified that could be modified chem. into ERα downregulators.  Following this, cellular assays helped det. the mechanism of action of compds. to distinguish between on-target and off-target compds. and differentiate SERDs, selective estrogen receptor modulator (SERM) compds., and agonist ERα ligands.  Data are shown to exemplify the characterization of ERα-mediated signaling inhibitors using a selection of literature compds. and illustrate how this cascade has been used to drive the chem. design of novel SERD compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqNNWMPPJDNrVg90H21EOLACvtfcHk0lgw4yhFhBJJsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsbzE&md5=13c27dbf08f315d469f81b2bcefb4499</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1177%2F1087057115580298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115580298%26sid%3Dliteratum%253Aachs%26aulast%3DCallis%26aufirst%3DR.%26aulast%3DRabow%26aufirst%3DA.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DBradbury%26aufirst%3DR.%26aulast%3DChallinor%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DWalker%26aufirst%3DG.%26atitle%3DA%2520Screening%2520Assay%2520Cascade%2520to%2520Identify%2520and%2520Characterize%2520Novel%2520Selective%2520Estrogen%2520Receptor%2520Downregulators%2520%2528SERDs%2529%26jtitle%3DJ.%2520Biomol.%2520Screen%26date%3D2015%26volume%3D20%26spage%3D748%26epage%3D759%26doi%3D10.1177%2F1087057115580298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komm, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharode, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyttle, C. R.</span></span> <span> </span><span class="NLM_article-title">Bazedoxifene acetate: a selective estrogen receptor modulator with improved selectivity</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">3999</span>– <span class="NLM_lpage">4008</span>, <span class="refDoi"> DOI: 10.1210/en.2005-0030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1210%2Fen.2005-0030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=15961563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsVegsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2005&pages=3999-4008&author=B.+S.+Kommauthor=Y.+P.+Kharodeauthor=P.+V.+Bodineauthor=H.+A.+Harrisauthor=C.+P.+Millerauthor=C.+R.+Lyttle&title=Bazedoxifene+acetate%3A+a+selective+estrogen+receptor+modulator+with+improved+selectivity&doi=10.1210%2Fen.2005-0030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Bazedoxifene acetate: a selective estrogen receptor modulator with improved selectivity</span></div><div class="casAuthors">Komm, Barry S.; Kharode, Yogendra P.; Bodine, Peter V. N.; Harris, Heather A.; Miller, Chris P.; Lyttle, C. Richard</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3999-4008</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">We assessed the preclin. characteristics of a novel, stringently screened selective estrogen receptor modulator, bazedoxifene acetate, including its ability to bind to and activate estrogen receptors and promote increased bone mineral d. and bone strength in rats, and the effects impacting the uterine endometrium, breast cancer cell proliferation, and central nervous system-assocd. vasomotor responses in an animal model.  Bazedoxifene bound to estrogen receptor-α with an IC50 of 26 nM, an affinity similar to that of raloxifene.  Bazedoxifene did not stimulate proliferation of MCF-7 cells but did inhibit 17β-estradiol-induced proliferation with an IC50 of 0.19 nM.  In an immature rat uterine model, bazedoxifene (0.5 and 5.0 mg/kg) was assocd. with less increase in uterine wet wt. than either ethinyl estradiol (10 μg/kg) or raloxifene (0.5 and 5.0 mg/kg).  Histol. anal. revealed that coadministration of bazedoxifene also appeared to reduce raloxifene-stimulated endometrial luminal epithelial cell and myometrial cell hypertrophy.  In ovariectomized rats, bazedoxifene was assocd. with significant increases in bone mineral d. at 6 wk, compared with control, and better compressive strength of bone samples from the L4 vertebrae, compared with samples from ovariectomized animals.  In the morphine-addicted rat model of vasomotor activity, bone-sparing doses of bazedoxifene alone were not assocd. with 17β-estradiol inhibition of increased vasomotor activity.  Bazedoxifene acetate represents a promising new treatment for osteoporosis, with a potential for less uterine and vasomotor effects than selective estrogen receptor modulators currently used in clin. practice.  Controlled clin. trial data will be needed to confirm these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou3Bu1u-p7dbVg90H21EOLACvtfcHk0liZtkLm4pcJcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsVegsb4%253D&md5=667468f62112aede01f2dec0ed95314f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1210%2Fen.2005-0030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2005-0030%26sid%3Dliteratum%253Aachs%26aulast%3DKomm%26aufirst%3DB.%2BS.%26aulast%3DKharode%26aufirst%3DY.%2BP.%26aulast%3DBodine%26aufirst%3DP.%2BV.%26aulast%3DHarris%26aufirst%3DH.%2BA.%26aulast%3DMiller%26aufirst%3DC.%2BP.%26aulast%3DLyttle%26aufirst%3DC.%2BR.%26atitle%3DBazedoxifene%2520acetate%253A%2520a%2520selective%2520estrogen%2520receptor%2520modulator%2520with%2520improved%2520selectivity%26jtitle%3DEndocrinology%26date%3D2005%26volume%3D146%26spage%3D3999%26epage%3D4008%26doi%3D10.1210%2Fen.2005-0030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collini, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharode, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzolf, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unwalla, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberger, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yardley, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou-Gharbia, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyttle, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komm, B. S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and preclinical characterization of novel, highly selective indole estrogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1654</span>– <span class="NLM_lpage">1657</span>, <span class="refDoi"> DOI: 10.1021/jm010086m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010086m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFSgt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1654-1657&author=C.+P.+Millerauthor=M.+D.+Colliniauthor=B.+D.+Tranauthor=H.+A.+Harrisauthor=Y.+P.+Kharodeauthor=J.+T.+Marzolfauthor=R.+A.+Moranauthor=R.+A.+Hendersonauthor=R.+H.+Benderauthor=R.+J.+Unwallaauthor=L.+M.+Greenbergerauthor=J.+P.+Yardleyauthor=M.+A.+Abou-Gharbiaauthor=C.+R.+Lyttleauthor=B.+S.+Komm&title=Design%2C+synthesis%2C+and+preclinical+characterization+of+novel%2C+highly+selective+indole+estrogens&doi=10.1021%2Fjm010086m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Preclinical Characterization of Novel, Highly Selective Indole Estrogens</span></div><div class="casAuthors">Miller, Chris P.; Collini, Michael D.; Tran, Bach D.; Harris, Heather A.; Kharode, Yogendra P.; Marzolf, James T.; Moran, Robert A.; Henderson, Ruth A.; Bender, Reinhold H. W.; Unwalla, Rayomond J.; Greenberger, Lee M.; Yardley, John P.; Abou-Gharbia, Magid A.; Lyttle, C. Richard; Komm, Barry S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1654-1657</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The indoles I [n = 1, 2] were prepd. from 5-benzyloxy-2-(4-benzyloxyphenyl)-3-methylindole, 4-EtO2CCH2OC6H4CH2Cl, and piperidine or hexamethyleneimine.  I display attractive selective estrogen profiles.  I [n = 2] is effective in treating post-menopausal osteoporosis and I [n = 1] is an efficient antiestrogen in in vivo MCF-7 xenografts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKcOJFkvSQKLVg90H21EOLACvtfcHk0liZtkLm4pcJcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFSgt7g%253D&md5=1e355073a49a908821acb9497de9ffbd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm010086m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010086m%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DC.%2BP.%26aulast%3DCollini%26aufirst%3DM.%2BD.%26aulast%3DTran%26aufirst%3DB.%2BD.%26aulast%3DHarris%26aufirst%3DH.%2BA.%26aulast%3DKharode%26aufirst%3DY.%2BP.%26aulast%3DMarzolf%26aufirst%3DJ.%2BT.%26aulast%3DMoran%26aufirst%3DR.%2BA.%26aulast%3DHenderson%26aufirst%3DR.%2BA.%26aulast%3DBender%26aufirst%3DR.%2BH.%26aulast%3DUnwalla%26aufirst%3DR.%2BJ.%26aulast%3DGreenberger%26aufirst%3DL.%2BM.%26aulast%3DYardley%26aufirst%3DJ.%2BP.%26aulast%3DAbou-Gharbia%26aufirst%3DM.%2BA.%26aulast%3DLyttle%26aufirst%3DC.%2BR.%26aulast%3DKomm%26aufirst%3DB.%2BS.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520preclinical%2520characterization%2520of%2520novel%252C%2520highly%2520selective%2520indole%2520estrogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1654%26epage%3D1657%26doi%3D10.1021%2Fjm010086m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13312</span>, <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13312&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-%CE%B1+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0lg0uBn6BWccaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-%25CE%25B1%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13312%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) in ’beyond rule-of-five’ chemical space: Recent progress and future challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+targeting+chimeras+%28PROTACs%29+in+%E2%80%99beyond+rule-of-five%E2%80%99+chemical+space%3A+Recent+progress+and+future+challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0lg0uBn6BWccaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2599beyond%2520rule-of-five%25E2%2580%2599%2520chemical%2520space%253A%2520Recent%2520progress%2520and%2520future%2520challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cantrill, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaturvedi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrig Schaffland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M. B.</span></span> <span> </span><span class="NLM_article-title">Fundamental aspects of DMPK optimization of targeted protein degraders</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">969</span>– <span class="NLM_lpage">982</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.drudis.2020.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32298797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntFGht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=969-982&author=C.+Cantrillauthor=P.+Chaturvediauthor=C.+Rynnauthor=J.+Petrig+Schafflandauthor=I.+Walterauthor=M.+B.+Wittwer&title=Fundamental+aspects+of+DMPK+optimization+of+targeted+protein+degraders&doi=10.1016%2Fj.drudis.2020.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental aspects of DMPK optimization of targeted protein degraders</span></div><div class="casAuthors">Cantrill, Carina; Chaturvedi, Prasoon; Rynn, Caroline; Petrig Schaffland, Jeannine; Walter, Isabelle; Wittwer, Matthias B.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">969-982</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Targeted protein degraders are an emerging modality.  Their properties fall outside the traditional small-mol. property space and are in the 'beyond rule of 5' space.  Consequently, drug discovery programs focused on developing orally bioavailable degraders are expected to face complex drug metab. and pharmacokinetics (DMPK) challenges compared with traditional small mols.  Nevertheless, little information is available on the DMPK optimization of oral degraders.  Therefore, in this review, we discuss our experience of these DMPK optimization challenges and present methodologies and strategies to overcome the hurdles dealing with this new small-mol. modality, specifically in developing oral degraders to treat cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz2WUoP7TUvbVg90H21EOLACvtfcHk0lg0uBn6BWccaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntFGht7s%253D&md5=465ad2dab5183e0f229aeec7e57e396a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DCantrill%26aufirst%3DC.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DRynn%26aufirst%3DC.%26aulast%3DPetrig%2BSchaffland%26aufirst%3DJ.%26aulast%3DWalter%26aufirst%3DI.%26aulast%3DWittwer%26aufirst%3DM.%2BB.%26atitle%3DFundamental%2520aspects%2520of%2520DMPK%2520optimization%2520of%2520targeted%2520protein%2520degraders%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26spage%3D969%26epage%3D982%26doi%3D10.1016%2Fj.drudis.2020.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukumo, Y.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation by Chimeric Small Molecules, PROTACs and SNIPERs</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">849</span>, <span class="refDoi"> DOI: 10.3389/fchem.2019.00849</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.3389%2Ffchem.2019.00849" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31921772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1KktLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=849&author=M.+Naitoauthor=N.+Ohokaauthor=N.+Shibataauthor=Y.+Tsukumo&title=Targeted+Protein+Degradation+by+Chimeric+Small+Molecules%2C+PROTACs+and+SNIPERs&doi=10.3389%2Ffchem.2019.00849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation by chimeric small molecules, PROTACs and SNIPERs</span></div><div class="casAuthors">Naito, Mikihiko; Ohoka, Nobumichi; Shibata, Norihito; Tsukumo, Yoshinori</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">849</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Technologies that induce targeted protein degrdn. by small mols. have been developed recently.  Chimeric small mols. such as Proteolysis Targeting Chimeras (PROTACs) and Specific and Non-genetic IAP-dependent Protein Erasers (SNIPERs), and E3 modulators such as thalidomides, hijack the cellular machinery for ubiquitylation, and the ubiquitylated proteins are subjected to proteasomal degrdn.  This has motivated drug development in industry and academia because "undruggable targets" can now be degraded by targeted protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu8jFxYJ0HebVg90H21EOLACvtfcHk0lg0uBn6BWccaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1KktLrJ&md5=a58ebd83aba5334f215500bb50915284</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2019.00849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2019.00849%26sid%3Dliteratum%253Aachs%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DTsukumo%26aufirst%3DY.%26atitle%3DTargeted%2520Protein%2520Degradation%2520by%2520Chimeric%2520Small%2520Molecules%252C%2520PROTACs%2520and%2520SNIPERs%26jtitle%3DFront.%2520Chem.%26date%3D2019%26volume%3D7%26spage%3D849%26doi%3D10.3389%2Ffchem.2019.00849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lipz2nDBLJ-rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">First targeted protein degrader hits the clinic</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">239</span>, <span class="refDoi"> DOI: 10.1038/d41573-019-00043-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fd41573-019-00043-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=237-239&author=A.+Mullard&title=First+targeted+protein+degrader+hits+the+clinic&doi=10.1038%2Fd41573-019-00043-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fd41573-019-00043-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41573-019-00043-6%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DFirst%2520targeted%2520protein%2520degrader%2520hits%2520the%2520clinic%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D237%26epage%3D239%26doi%3D10.1038%2Fd41573-019-00043-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lipz2nDBLJ-rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-Induced+Protein+Degradation+in+Drug+Discovery%3A+Breaking+the+Rules+or+Just+Making+New+Ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lgEguZFsAy66Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-Induced%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%253A%2520Breaking%2520the%2520Rules%2520or%2520Just%2520Making%2520New%2520Ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott-Stevens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaohua, L.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation in vivo with Proteolysis Targeting Chimeras: Current status and future considerations</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1016%2Fj.ddtec.2019.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=31200862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=69-80&author=G.+F.+Wattauthor=P.+Scott-Stevensauthor=L.+Gaohua&title=Targeted+protein+degradation+in+vivo+with+Proteolysis+Targeting+Chimeras%3A+Current+status+and+future+considerations&doi=10.1016%2Fj.ddtec.2019.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation in vivo with Proteolysis Targeting Chimeras: Current status and future considerations</span></div><div class="casAuthors">Watt Gillian F; Scott-Stevens Paul; Gaohua Lu</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">69-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are a rapidly expanding new therapeutic modality inducing selective protein degradation and offering the potential of a differentiated pharmacological profile across multiple therapeutic areas.  As the repertoire of protein targets and E3 ligases available for incorporation into PROTACs continues to grow, understanding the drug- and system-dependent parameters for PROTACs will be critical for achieving tissue/cell specific pharmacology.  The review discusses the current knowledge and future direction of in vivo PROTAC study evaluation.  The importance of establishing the quantitative relationship between loss of protein target and biological function in vivo, coupled with building mechanistic PK/PD and ultimately PBPK/PD models, is emphasised with the aim to aid translation from preclinical to clinical space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXobYtC3U5jeOifW6udTcc2ebp7xB3HSUo1rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrtA%253D%253D&md5=2dd51eb704b03eee904c1b04377f0a8e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWatt%26aufirst%3DG.%2BF.%26aulast%3DScott-Stevens%26aufirst%3DP.%26aulast%3DGaohua%26aufirst%3DL.%26atitle%3DTargeted%2520protein%2520degradation%2520in%2520vivo%2520with%2520Proteolysis%2520Targeting%2520Chimeras%253A%2520Current%2520status%2520and%2520future%2520considerations%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D69%26epage%3D80%26doi%3D10.1016%2Fj.ddtec.2019.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kargbo, R. B.</span></span> <span> </span><span class="NLM_article-title">PROTAC-Mediated Degradation of Estrogen Receptor in the Treatment of Cancer</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1367</span>– <span class="NLM_lpage">1369</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00397</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00397" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslegt77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1367-1369&author=R.+B.+Kargbo&title=PROTAC-Mediated+Degradation+of+Estrogen+Receptor+in+the+Treatment+of+Cancer&doi=10.1021%2Facsmedchemlett.9b00397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-Mediated Degradation of Estrogen Receptor in the Treatment of Cancer</span></div><div class="casAuthors">Kargbo, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1367-1369</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoD-uHJ-PklLVg90H21EOLACvtfcHk0lgEguZFsAy66Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslegt77F&md5=77eb546538b6dec6ddf1316be5d9b092</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00397%26sid%3Dliteratum%253Aachs%26aulast%3DKargbo%26aufirst%3DR.%2BB.%26atitle%3DPROTAC-Mediated%2520Degradation%2520of%2520Estrogen%2520Receptor%2520in%2520the%2520Treatment%2520of%2520Cancer%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1367%26epage%3D1369%26doi%3D10.1021%2Facsmedchemlett.9b00397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rackham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thawani, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haile, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beal, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossenkamper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dace, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benowitz, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denyer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott-Stevens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span> <span> </span><span class="NLM_article-title">Extended pharmacodynamic responses observed upon PROTAC-mediated degradation of RIPK2</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">140</span>, <span class="refDoi"> DOI: 10.1038/s42003-020-0868-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fs42003-020-0868-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32198438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlslyqt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=140&author=A.+Maresauthor=A.+H.+Miahauthor=I.+E.+D.+Smithauthor=M.+Rackhamauthor=A.+R.+Thawaniauthor=J.+Cryanauthor=P.+A.+Haileauthor=B.+J.+Vottaauthor=A.+M.+Bealauthor=C.+Capriottiauthor=M.+A.+Reillyauthor=D.+T.+Fisherauthor=N.+Zinnauthor=M.+Bantscheffauthor=T.+T.+MacDonaldauthor=A.+Vossenkamperauthor=P.+Daceauthor=I.+Churcherauthor=A.+B.+Benowitzauthor=G.+Wattauthor=J.+Denyerauthor=P.+Scott-Stevensauthor=J.+D.+Harling&title=Extended+pharmacodynamic+responses+observed+upon+PROTAC-mediated+degradation+of+RIPK2&doi=10.1038%2Fs42003-020-0868-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Extended pharmacodynamic responses observed upon PROTAC-mediated degradation of RIPK2</span></div><div class="casAuthors">Mares, Alina; Miah, Afjal H.; Smith, Ian E. D.; Rackham, Mark; Thawani, Aditya R.; Cryan, Jenni; Haile, Pamela A.; Votta, Bartholomew J.; Beal, Allison M.; Capriotti, Carol; Reilly, Michael A.; Fisher, Don T.; Zinn, Nico; Bantscheff, Marcus; MacDonald, Thomas T.; Vossenkamper, Anna; Dace, Phoebe; Churcher, Ian; Benowitz, Andrew B.; Watt, Gillian; Denyer, Jane; Scott-Stevens, Paul; Harling, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Communications Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140</span>CODEN:
                <span class="NLM_cas:coden">CBOIDQ</span>;
        ISSN:<span class="NLM_cas:issn">2399-3642</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteolysis-Targeting Chimeras (PROTACs) are heterobifunctional small-mols. that can promote the rapid and selective proteasome-mediated degrdn. of intracellular proteins through the recruitment of E3 ligase complexes to non-native protein substrates.  The catalytic mechanism of action of PROTACs represents an exciting new modality in drug discovery that offers several potential advantages over traditional small-mol. inhibitors, including the potential to deliver pharmacodynamic (PD) efficacy which extends beyond the detectable pharmacokinetic (PK) presence of the PROTAC, driven by the synthesis rate of the protein.  Herein we report the identification and development of PROTACs that selectively degrade Receptor-Interacting Serine/Threonine Protein Kinase 2 (RIPK2) and demonstrate in vivo degrdn. of endogenous RIPK2 in rats at low doses and extended PD that persists in the absence of detectable compd.  This disconnect between PK and PD, when coupled with low nanomolar potency, offers the potential for low human doses and infrequent dosing regimens with PROTAC medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXhjAOWsJhfLVg90H21EOLACvtfcHk0lgoAsWgFMT81Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlslyqt7c%253D&md5=6a9773698a2ba5e244974a9ec3fc8f3a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fs42003-020-0868-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-020-0868-6%26sid%3Dliteratum%253Aachs%26aulast%3DMares%26aufirst%3DA.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DRackham%26aufirst%3DM.%26aulast%3DThawani%26aufirst%3DA.%2BR.%26aulast%3DCryan%26aufirst%3DJ.%26aulast%3DHaile%26aufirst%3DP.%2BA.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3DBeal%26aufirst%3DA.%2BM.%26aulast%3DCapriotti%26aufirst%3DC.%26aulast%3DReilly%26aufirst%3DM.%2BA.%26aulast%3DFisher%26aufirst%3DD.%2BT.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DMacDonald%26aufirst%3DT.%2BT.%26aulast%3DVossenkamper%26aufirst%3DA.%26aulast%3DDace%26aufirst%3DP.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DBenowitz%26aufirst%3DA.%2BB.%26aulast%3DWatt%26aufirst%3DG.%26aulast%3DDenyer%26aufirst%3DJ.%26aulast%3DScott-Stevens%26aufirst%3DP.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26atitle%3DExtended%2520pharmacodynamic%2520responses%2520observed%2520upon%2520PROTAC-mediated%2520degradation%2520of%2520RIPK2%26jtitle%3DCommun.%2520Biol.%26date%3D2020%26volume%3D3%26spage%3D140%26doi%3D10.1038%2Fs42003-020-0868-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">den
Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span> <span> </span><span class="NLM_article-title">Prospecting for molecular glues</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0620-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fs41589-020-0620-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=32747810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFaiur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1157-1158&author=W.+den%0ABestenauthor=J.+R.+Lipford&title=Prospecting+for+molecular+glues&doi=10.1038%2Fs41589-020-0620-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Prospecting for molecular glues</span></div><div class="casAuthors">den Besten, Willem; Lipford, J. Russell</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1157-1158</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Two recent studies identified CDK12 inhibitors that bind to CDK12-cyclin K complexes and act as mol. glues to stabilize an interaction with the ubiquitin ligase CUL4-DDB1, leading to cyclin K degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqej0SQYEVKmLVg90H21EOLACvtfcHk0lgoAsWgFMT81Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFaiur7F&md5=00ba8e9c5ec1aaa6e0a8beb0fc8cdb5c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0620-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0620-z%26sid%3Dliteratum%253Aachs%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26atitle%3DProspecting%2520for%2520molecular%2520glues%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D1157%26epage%3D1158%26doi%3D10.1038%2Fs41589-020-0620-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalvan, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz-Saez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjanacz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&author=A.+E.+Fern%C3%A1ndez-Montalvanauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+Stockigtauthor=A.+Ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+Diaz-Saezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=J.+Hubnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorjanacz&title=Isoform-Selective+ATAD2+Chemical+Probe+with+Novel+Chemical+Structure+and+Unusual+Mode+of+Action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span></div><div class="casAuthors">Fernandez-Montalvan, Amaury E.; Berger, Markus; Kuropka, Benno; Koo, Seong Joo; Badock, Volker; Weiske, Joerg; Puetter, Vera; Holton, Simon J.; Stoeckigt, Detlef; ter Laak, Antonius; Centrella, Paolo A.; Clark, Matthew A.; Dumelin, Christoph E.; Sigel, Eric A.; Soutter, Holly H.; Troast, Dawn M.; Zhang, Ying; Cuozzo, John W.; Keefe, Anthony D.; Roche, Didier; Rodeschini, Vincent; Chaikuad, Apirat; Diaz-Saez, Laura; Bennett, James M.; Fedorov, Oleg; Huber, Kilian V. M.; Huebner, Jan; Weinmann, Hilmar; Hartung, Ingo V.; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2730-2736</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 (ANCCA) is an epigenetic regulator and transcriptional cofactor, whose overexpression has been linked to the progress of various cancer types.  Here, we report a DNA-encoded library screen leading to the discovery of BAY-850, a potent and isoform selective inhibitor that specifically induces ATAD2 bromodomain dimerization and prevents interactions with acetylated histones in vitro, as well as with chromatin in cells.  These features qualify BAY-850 as a chem. probe to explore ATAD2 biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCB3nyG9YvubVg90H21EOLACvtfcHk0lgoAsWgFMT81Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL&md5=1f09000a7d7e60e30540fbbae117acfe</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Montalvan%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DStockigt%26aufirst%3DD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DDiaz-Saez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DHubner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorjanacz%26aufirst%3DM.%26atitle%3DIsoform-Selective%2520ATAD2%2520Chemical%2520Probe%2520with%2520Novel%2520Chemical%2520Structure%2520and%2520Unusual%2520Mode%2520of%2520Action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Litovchick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeshian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guié, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Encoded Library Synthesis Using Chemical Ligation and the Discovery of sEH Inhibitors from a 334-Million Member Library</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10916</span>, <span class="refDoi"> DOI: 10.1038/srep10916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1038%2Fsrep10916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=26061191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyiu7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=10916&author=A.+Litovchickauthor=C.+E.+Dumelinauthor=S.+Habeshianauthor=D.+Gikunjuauthor=M.-A.+Gui%C3%A9author=P.+Centrellaauthor=Y.+Zhangauthor=E.+A.+Sigelauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=M.+A.+Clark&title=Encoded+Library+Synthesis+Using+Chemical+Ligation+and+the+Discovery+of+sEH+Inhibitors+from+a+334-Million+Member+Library&doi=10.1038%2Fsrep10916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded Library Synthesis Using Chemical Ligation and the Discovery of sEH Inhibitors from a 334-Million Member Library</span></div><div class="casAuthors">Litovchick, Alexander; Dumelin, Christoph E.; Habeshian, Sevan; Gikunju, Diana; Guie, Marie-Aude; Centrella, Paolo; Zhang, Ying; Sigel, Eric A.; Cuozzo, John W.; Keefe, Anthony D.; Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10916</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A chem. ligation method for construction of DNA-encoded small-mol. libraries has been developed.  Taking advantage of the ability of the Klenow fragment of DNA polymerase to accept templates with triazole linkages in place of phosphodiesters, we have designed a strategy for chem. ligating oligonucleotide tags using cycloaddn. chem.  We have utilized this strategy in the construction and selection of a small mol. library, and successfully identified inhibitors of the enzyme sol. epoxide hydrolase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGlK1fEcxcbbVg90H21EOLACvtfcHk0lhxA6xilPE4Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyiu7bI&md5=5251dd6ce343cae4639b9cae82dea08f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fsrep10916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep10916%26sid%3Dliteratum%253Aachs%26aulast%3DLitovchick%26aufirst%3DA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DHabeshian%26aufirst%3DS.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DGui%25C3%25A9%26aufirst%3DM.-A.%26aulast%3DCentrella%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DEncoded%2520Library%2520Synthesis%2520Using%2520Chemical%2520Ligation%2520and%2520the%2520Discovery%2520of%2520sEH%2520Inhibitors%2520from%2520a%2520334-Million%2520Member%2520Library%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D10916%26doi%3D10.1038%2Fsrep10916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livak, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmittgen, T. D.</span></span> <span> </span><span class="NLM_article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">402</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1006/meth.2001.1262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=10.1006%2Fmeth.2001.1262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=11846609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtFelt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2001&pages=402-408&author=K.+J.+Livakauthor=T.+D.+Schmittgen&title=Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28-Delta+Delta+C%28T%29%29+Method&doi=10.1006%2Fmeth.2001.1262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of relative gene expression data using real-time quantitative PCR and the 2-ΔΔCT method</span></div><div class="casAuthors">Livak, Kenneth J.; Schmittgen, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">402-408</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The two most commonly used methods to analyze data from real-time, quant. PCR expts. are abs. quantification and relative quantification.  Abs. quantification dets. the input copy no., usually by relating the PCR signal to a std. curve.  Relative quantification relates the PCR signal of the target transcript in a treatment group to that of another sample such as an untreated control.  The 2-ΔΔCT method is a convenient way to analyze the relative changes in gene expression from real-time quant. PCR expts.  The purpose of this report is to present the derivation, assumptions, and applications of the 2-ΔΔCT method.  In addn., we present the derivation and applications of two variations of the 2-ΔΔCT method that may be useful in the anal. of real-time, quant. PCR data.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXNe1s-mDpprVg90H21EOLACvtfcHk0lhxA6xilPE4Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtFelt7s%253D&md5=f849383250ea43f7380ed540db8a64b1</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1006%2Fmeth.2001.1262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fmeth.2001.1262%26sid%3Dliteratum%253Aachs%26aulast%3DLivak%26aufirst%3DK.%2BJ.%26aulast%3DSchmittgen%26aufirst%3DT.%2BD.%26atitle%3DAnalysis%2520of%2520relative%2520gene%2520expression%2520data%2520using%2520real-time%2520quantitative%2520PCR%2520and%2520the%25202%2528-Delta%2520Delta%2520C%2528T%2529%2529%2520Method%26jtitle%3DMethods%26date%3D2001%26volume%3D25%26spage%3D402%26epage%3D408%26doi%3D10.1006%2Fmeth.2001.1262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i54"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00127">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_42245"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00127?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00127</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Denaturing gel of proteins, LC-MS for compounds <b>1–23</b> (parent ion, retention time, purity, and chromatograms), <sup>1</sup>H and <sup>13</sup>C NMR spectra for compounds <b>18</b> and <b>21</b>, supplemental figures demonstrating purification of ERα proteins and degradation of ERα in ER+ breast cancer cells, and supplemental methods giving detailed protocols for protein production, biochemical and cellular assays, and DECL screening (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_001.pdf">jm1c00127_si_001.pdf (1.63 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00127/suppl_file/jm1c00127_si_002.csv">jm1c00127_si_002.csv (4.58 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00127&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00127%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-8%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00127" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679917595bb9d187","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
