
<html lang="en"     class="pb-page"  data-request-id="4fc6e03d-209e-4819-bc31-11920e3b8884"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01351;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-4;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure–Activity Relationship and Binding Mode Analysis" /></meta><meta name="dc.Creator" content="Radka  Houštecká" /></meta><meta name="dc.Creator" content="Martin  Hadzima" /></meta><meta name="dc.Creator" content="Jindřich  Fanfrlík" /></meta><meta name="dc.Creator" content="Jiří  Brynda" /></meta><meta name="dc.Creator" content="Lenka  Pallová" /></meta><meta name="dc.Creator" content="Iva  Hánová" /></meta><meta name="dc.Creator" content="Helena  Mertlíková-Kaiserová" /></meta><meta name="dc.Creator" content="Martin  Lepšík" /></meta><meta name="dc.Creator" content="Martin  Horn" /></meta><meta name="dc.Creator" content="Martin  Smrčina" /></meta><meta name="dc.Creator" content="Pavel  Majer" /></meta><meta name="dc.Creator" content="Michael  Mareš" /></meta><meta name="dc.Description" content="Human cathepsin D (CatD), a pepsin-family aspartic protease, plays an important role in tumor progression and metastasis. Here, we report the development of biomimetic inhibitors of CatD as novel t..." /></meta><meta name="Description" content="Human cathepsin D (CatD), a pepsin-family aspartic protease, plays an important role in tumor progression and metastasis. Here, we report the development of biomimetic inhibitors of CatD as novel t..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 31, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01351" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01351" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01351" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01351" /></link>
        
    
    

<title>Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure–Activity Relationship and Binding Mode Analysis | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01351" /></meta><meta property="og:title" content="Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure–Activity Relationship and Binding Mode Analysis" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0014.jpeg" /></meta><meta property="og:description" content="Human cathepsin D (CatD), a pepsin-family aspartic protease, plays an important role in tumor progression and metastasis. Here, we report the development of biomimetic inhibitors of CatD as novel tools for regulation of this therapeutic target. We designed a macrocyclic scaffold to mimic the spatial conformation of the minimal pseudo-dipeptide binding motif of pepstatin A, a microbial oligopeptide inhibitor, in the CatD active site. A library of more than 30 macrocyclic peptidomimetic inhibitors was employed for scaffold optimization, mapping of subsite interactions, and profiling of inhibitor selectivity. Furthermore, we solved high-resolution crystal structures of three macrocyclic inhibitors with low nanomolar or subnanomolar potency in complex with CatD and determined their binding mode using quantum chemical calculations. The study provides a new structural template and functional profile that can be exploited for design of potential chemotherapeutics that specifically inhibit CatD and related aspartic proteases." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01351"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01351">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01351&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01351&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01351&amp;href=/doi/10.1021/acs.jmedchem.9b01351" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1576-1596</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01409" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01328" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure–Activity Relationship and Binding Mode Analysis</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Radka Houštecká</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Radka Houštecká</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div><div class="loa-info-affiliations-info">First Faculty of Medicine, Charles University, Kateřinská 32, 12108 Praha 2, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Radka++Hou%C5%A1teck%C3%A1">Radka Houštecká</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Hadzima</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Hadzima</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div><div class="loa-info-affiliations-info">Department of Organic Chemistry, Faculty of Science, Charles University, Albertov 6, 12800 Praha 2, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Hadzima">Martin Hadzima</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jindřich Fanfrlík</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jindřich Fanfrlík</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jind%C5%99ich++Fanfrl%C3%ADk">Jindřich Fanfrlík</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiří Brynda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiří Brynda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ji%C5%99%C3%AD++Brynda">Jiří Brynda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lenka Pallová</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lenka Pallová</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lenka++Pallov%C3%A1">Lenka Pallová</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Iva Hánová</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Iva Hánová</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div><div class="loa-info-affiliations-info">Department of Biochemistry, Faculty of Science, Charles University, Albertov 6, 12800 Praha 2, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Iva++H%C3%A1nov%C3%A1">Iva Hánová</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7173-8200" title="Orcid link">http://orcid.org/0000-0002-7173-8200</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Helena Mertlíková-Kaiserová</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helena Mertlíková-Kaiserová</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helena++Mertl%C3%ADkov%C3%A1-Kaiserov%C3%A1">Helena Mertlíková-Kaiserová</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Lepšík</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Lepšík</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Lep%C5%A1%C3%ADk">Martin Lepšík</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Horn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Horn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Horn">Martin Horn</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9110-2018" title="Orcid link">http://orcid.org/0000-0001-9110-2018</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Smrčina</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Smrčina</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Tucson Research Center, Icagen Inc., 2090 E. Innovation Park Drive, Oro Valley, Arizona 85755, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Smr%C4%8Dina">Martin Smrčina</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pavel Majer</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pavel Majer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#472a262d2235073228242f256924263469243d"><span class="__cf_email__" data-cfemail="bad7dbd0dfc8facfd5d9d2d894d9dbc994d9c0">[email protected]</span></a> (P.M.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pavel++Majer">Pavel Majer</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Michael Mareš</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Mareš</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#7e131f0c1b0d3e0b111d161c501d1f0d501d04"><span class="__cf_email__" data-cfemail="7815190a1d0b380d171b101a561b190b561b02">[email protected]</span></a> (M.M.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Mare%C5%A1">Michael Mareš</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0847-5022" title="Orcid link">http://orcid.org/0000-0002-0847-5022</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01351&amp;href=/doi/10.1021%2Facs.jmedchem.9b01351" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1576–1596</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 31, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 August 2019</li><li><span class="item_label"><b>Published</b> online</span>31 January 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 February 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01351" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01351</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1576%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRadka%2BHou%25C5%25A1teck%25C3%25A1%252C%2BMartin%2BHadzima%252C%2BJind%25C5%2599ich%2BFanfrl%25C3%25ADk%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D4%26contentID%3Dacs.jmedchem.9b01351%26title%3DBiomimetic%2BMacrocyclic%2BInhibitors%2Bof%2BHuman%2BCathepsin%2BD%253A%2BStructure%25E2%2580%2593Activity%2BRelationship%2Band%2BBinding%2BMode%2BAnalysis%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1596%26publicationDate%3DFebruary%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01351"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1487</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01351" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Biomimetic Macrocyclic Inhibitors of Human Cathepsin D: Structure–Activity Relationship and Binding Mode Analysis&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Radka&quot;,&quot;last_name&quot;:&quot;Houštecká&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Hadzima&quot;},{&quot;first_name&quot;:&quot;Jindřich&quot;,&quot;last_name&quot;:&quot;Fanfrlík&quot;},{&quot;first_name&quot;:&quot;Jiří&quot;,&quot;last_name&quot;:&quot;Brynda&quot;},{&quot;first_name&quot;:&quot;Lenka&quot;,&quot;last_name&quot;:&quot;Pallová&quot;},{&quot;first_name&quot;:&quot;Iva&quot;,&quot;last_name&quot;:&quot;Hánová&quot;},{&quot;first_name&quot;:&quot;Helena&quot;,&quot;last_name&quot;:&quot;Mertlíková-Kaiserová&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Lepšík&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Horn&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Smrčina&quot;},{&quot;first_name&quot;:&quot;Pavel&quot;,&quot;last_name&quot;:&quot;Majer&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Mareš&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;1576-1596&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01351&quot;},&quot;abstract&quot;:&quot;Human cathepsin D (CatD), a pepsin-family aspartic protease, plays an important role in tumor progression and metastasis. Here, we report the development of biomimetic inhibitors of CatD as novel tools for regulation of this therapeutic target. We designed a macrocyclic scaffold to mimic the spatial conformation of the minimal pseudo-dipeptide binding motif of pepstatin A, a microbial oligopeptide inhibitor, in the CatD active site. A library of more than 30 macrocyclic peptidomimetic inhibitors was employed for scaffold optimization, mapping of subsite interactions, and profiling of inhibitor selectivity. Furthermore, we solved high-resolution crystal structures of three macrocyclic inhibitors with low nanomolar or subnanomolar potency in complex with CatD and determined their binding mode using quantum chemical calculations. The study provides a new structural template and functional profile that can be exploited for design of potential chemotherapeutics that specifically inhibit CatD and related aspartic p&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01351&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01351" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01351&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01351" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01351&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01351" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01351&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01351&amp;href=/doi/10.1021/acs.jmedchem.9b01351" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01351" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01351" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01351%26sid%3Dliteratum%253Aachs%26pmid%3D32003991%26genre%3Darticle%26aulast%3DHou%25C5%25A1teck%25C3%25A1%26date%3D2020%26atitle%3DBiomimetic%2BMacrocyclic%2BInhibitors%2Bof%2BHuman%2BCathepsin%2BD%253A%2BStructure%25E2%2580%2593Activity%2BRelationship%2Band%2BBinding%2BMode%2BAnalysis%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D4%26spage%3D1576%26epage%3D1596%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/jmcmar.2020.63.issue-4/20200227/jmcmar.2020.63.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Human cathepsin D (CatD), a pepsin-family aspartic protease, plays an important role in tumor progression and metastasis. Here, we report the development of biomimetic inhibitors of CatD as novel tools for regulation of this therapeutic target. We designed a macrocyclic scaffold to mimic the spatial conformation of the minimal pseudo-dipeptide binding motif of pepstatin A, a microbial oligopeptide inhibitor, in the CatD active site. A library of more than 30 macrocyclic peptidomimetic inhibitors was employed for scaffold optimization, mapping of subsite interactions, and profiling of inhibitor selectivity. Furthermore, we solved high-resolution crystal structures of three macrocyclic inhibitors with low nanomolar or subnanomolar potency in complex with CatD and determined their binding mode using quantum chemical calculations. The study provides a new structural template and functional profile that can be exploited for design of potential chemotherapeutics that specifically inhibit CatD and related aspartic proteases.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74585" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74585" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cathepsin D (CatD), a ubiquitous aspartic protease, is involved in several important physiological and pathological processes in humans. In addition to its primary role in lysosomal digestion of proteins, CatD performs a variety of other specialized functions, including processing of peptide hormones, activation of enzymatic precursors, antigen presentation, degradation of cytoskeletal proteins, matrix remodeling, and regulation of apoptosis.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> CatD is implicated in numerous pathologies, including several neurodegenerative conditions, osteoarthritis, and acute pancreatitis.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> However, it has mainly been studied in connection with different types of carcinomas. CatD is overexpressed and hypersecreted by breast cancer cells and is associated with tumor development and progression.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> It serves as an independent marker of poor breast cancer prognosis and is correlated with the incidence of clinical metastasis.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Therefore, CatD represents a promising target for the treatment of breast cancer using antibody-based therapy or chemotherapy.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a></div><div class="NLM_p">Various aspartic proteases are targets for drugs that regulate their proteolytic activity. Renin inhibitors are used to treat hypertension and HIV protease inhibitors to treat HIV infection, whereas β-secretase (BACE1) and plasmepsins are being investigated as targets for drugs against Alzheimer’s disease and malaria, respectively.<a onclick="showRef(event, 'ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13">(10−13)</a> Several types of potential small-molecule CatD inhibitors have been designed de novo.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17">(14−17)</a> The most potent natural inhibitor of CatD discovered to date is pepstatin A.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> This linear pseudo-pentapeptide originally isolated from actinobacteria is effective against many aspartic proteases from the pepsin family with inhibition constants as low as 10<sup>–10</sup> M. Its potency is attributable to the nonproteinogenic amino acid statine [(3<i>S</i>,4<i>S</i>)-4-amino-3-hydroxy-6-methylheptanoic acid], the hydroxyl group of which interacts with catalytic aspartate residues and acts as a transition-state analogue. Pepstatin A is widely used as a research tool for enzyme affinity purification, active site titration, imaging, and structural mapping of binding sites.<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> However, its use as a therapeutic is limited due to its large size and unfavorable physicochemical properties including poor solubility, which are reflected in its low bioavailability and rapid clearance.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p last">In this work, we used the spatial conformation that pepstatin A adopts upon binding to the CatD active site to design a macrocyclic mimetic with minimized size and potentiated inhibition. This biomimetic scaffold was subjected to comprehensive structural characterization, including binding mode analysis based on crystallography of CatD–inhibitor complexes and quantum chemical calculations. Furthermore, we performed a detailed structure–activity relationship (SAR) analysis of the scaffold using an extensive library of substituted macrocycles, leading to identification of CatD inhibitors with low nanomolar and subnanomolar potency. In summary, our study yielded a novel class of inhibitors inspired by natural compound but with improved drug-like properties. These new compounds represent a starting point for rational development of potential chemotherapeutics against pathologies associated with human CatD and CatD-like proteases in general.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80450" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80450" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of Macrocyclic Mimetics of the Natural Inhibitor Pepstatin A and SAR Analysis of Human CatD inhibition</h3><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5"> Truncation of the Linear Scaffold of Pepstatin A Decreases Inhibitory Potency</h4><div class="NLM_p">A series of linear truncated fragments of pepstatin A (compound <b>1</b>) was designed to identify portions of the inhibitor that can be removed without loss of potency toward human CatD (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Removal of the C-terminal P3′ statine from pepstatin A (<b>2</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) led to a 2-order-of-magnitude drop in potency (IC<sub>50</sub> of 0.5 vs 40 nM), and further trimming of the Ala residue resulted in an additional 1-order-of-magnitude decrease in potency (<b>3</b>). Truncation of the N-terminal isovaleryl-Val segment in the P4–P3 positions (<b>4</b>) resulted in a dramatic decrease in inhibition to the μM level. Compound <b>5</b>, which retained only the P2 to P2′ residues, showed very weak sub-mM inhibition. These observations indicate that a full-sized natural inhibitor spanning P4 to P3′ is needed to inhibit CatD in the subnanomolar range. Stepwise removal of substituents from both termini around the linear central segment with the P1 statine (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) leads to substantial reductions in inhibitory potency. This is in line with previous SAR data for bovine CatD.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In the next step, we explored the effect of cyclization on the truncated central segment.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic representation of the rational design of the macrocyclic inhibitor scaffold. The scaffold mimics the spatial conformation of pepstatin A, a natural linear oligopeptide inhibitor, in the active site of human CatD (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYB">1LYB</a>). Binding subsites (S) are marked and colored in blue; corresponding inhibitor positions (P) are in brown. The central hydroxyl group of the statine residue that interacts with the catalytic aspartates of CatD is highlighted in yellow. Lines with scissors indicate the region used for macrocycle scaffold design. Pepstatin A structure: Iva-Val-Val-Sta-Ala-Sta, where Iva is isovaleryl and Sta is statine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Effect of Pepstatin A (<b>1</b>) Truncation on Inhibitory Potency toward Human CatD</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0010.gif" alt="" id="gr10" /></img><div></div></div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Scaffold Cyclization Boosts Inhibition and Yields Macrocycles with Low and Subnanomolar Potency</h4><div class="NLM_p">When bound to human CatD, the pepstatin A molecule forms a U-shape with the P2 and P3′ side chains in proximity to each other (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). These side chains form contiguous interactions in the corresponding S2 and S3′ subsites of the enzyme. S2 and S3′ are located in a markedly hydrophobic, spacious area that also includes S1′, the subsite for the P1′ side chain, which is missing in statine-based inhibitors. The absence of a P1′ substituent might be the reason that the P2 isopropyl and P3′ isobutyl bend toward each other to fill the empty space. The proximity of those substituents led us to the idea of connecting them covalently to form a macrocycle (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). We designed a P2–P3′ cycle to span a sufficient portion of the binding cavity and include the backbone of the P1 statine residue as a transition-state isostere. Most of the interaction free energy of pepstatin A with CatD comes from hydrophobic interactions, as demonstrated by the dominant contribution of the entropic term.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Therefore, we assumed that improving the hydrophobic interactions of the P2/P3′ substituents, along with organization into the presumed bound conformation, may compensate for the lost interactions caused by truncation (removal of residues outside of P2 and P3′) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><div class="NLM_p">Our cyclic pseudo-tripeptide scaffold is formed by the P1 statine residue and the neighboring P2′ residue bridged by 12-aminododecanoic acid, which serves as a mimetic of the linked P2 and P3′ residues (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>a). The size of the bridge was based on molecular modeling, which indicated sufficient space in the active site to accommodate it. Compound <b>6</b> with the original pepstatin A substituents, P1 isobutyl and P2′ methyl, exhibited only ∼300 nM potency. The crystal structure of bound pepstatin A revealed that both these side chains only partially fill the subsites. We therefore introduced the bulkier benzyl moiety into P1 (<b>8</b>) and isopropyl into P2′ (<b>7</b>); these compounds were roughly an order of magnitude more potent than <b>6</b>. Simultaneous replacement in <b>9</b> led to further improvement in potency (IC<sub>50</sub> of ∼6 nM). Next, we optimized the size of the macrocycle and prepared a series of inhibitors with 10–14 methylene units (<b>9</b> to <b>13</b>). The IC<sub>50</sub> values varied within 1 order of magnitude, with optimal inhibition by compounds with 11 or 12 methylene units (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>b). We chose the smaller ring size for further optimization of the P1 and P2′ substituents.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Initial Design of Pepstatin A-Mimicking Macrocyclic Inhibitors of Human CatD<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0011.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Compound <b>6</b> contains the original pepstatin A substituents at R<sub>1</sub> and R<sub>2.</sub> Modification of these two positions (a) and optimization of macrocycle size (b) improved the inhibitory potency of the macrocyclic scaffold.</p></div></div><div></div></div><div class="NLM_p">We introduced structural changes at the P1 and P2′ positions (i.e., substituents R<sub>1</sub> and R<sub>2</sub>, respectively) to map the CatD subsite preferences, using <b>9</b> as a standard to which the IC<sub>50</sub> values were compared. In the first series, we optimized the R<sub>1</sub> substituent by evaluating different statine analogues with a fixed valine (R<sub>2</sub> isopropyl) in the P2′ position (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>a). In general, hydrophobic aromatic residues were preferred in the P1 position. Compound <b>14</b> with β-naphthyl exhibited the lowest IC<sub>50</sub> of 0.6 nM. Inhibitors with a single aromatic ring (<b>15</b>, <b>16</b>, <b>17</b>, and <b>9</b>) had slightly higher IC<sub>50</sub> values. The minimal difference between <b>9</b> with benzyl and <b>18</b> with cyclohexyl-methyl suggested that hydrophobic interaction prevails over aromatic stacking with the several aromatic residues in the S1 pocket. Introduction of more hydrophilic residues (<b>20</b>, <b>22</b>, <b>23</b>, and <b>24</b>) led to substantial loss of potency. Hence, <b>22</b> with β-indolyl was 2 orders of magnitude weaker than <b>14</b> with β-naphthyl, and <b>24</b> with a smaller thiazolylmethyl moiety was the weakest compound of the entire series, 2 orders of magnitude weaker than <b>9</b> with benzyl.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Substitution Analysis of Macrocyclic Inhibitors of Human CatD<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0012.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Compounds with variation at the R<sub>1</sub> substituent (a) and the R<sub>2</sub> substituent (b) are ordered according to their potency. The series also includes compounds from <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>a for comparison.</p></div></div><div></div></div><div class="NLM_p">The second series of compounds included a fixed phenylstatine residue in P1 (R<sub>1</sub> benzyl) and modifications at the P2′ position (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>b). The most potent compound <b>25</b> with 3-pyridyl alanine had an IC<sub>50</sub> of approximately 1 nM, five times more potent than <b>9</b> with valine. Comparison of <b>25</b> and <b>28</b> with benzyl (Phe) in P2′ yielded an interesting result: it is the only example in the series in which introduction of a more polar residue increased the potency. In general, medium-size hydrophobic side chains were preferred (<b>26</b>, <b>9</b>, <b>27</b>, and <b>30</b>). Inhibitor <b>31</b> with small hydrophilic threonine was 2 orders of magnitude weaker than <b>9</b>, as was <b>32</b>, likely due to excessive steric volume in the relatively small P2′ pocket.</div><div class="NLM_p">We also assessed various modifications along the macrocycle backbone (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The most active compound in this series—<b>33</b> with a urethane moiety—has the same cycle size (22 ring members) as the “full carbon” analogue <b>11</b> and exhibited similar potency. Replacement of amide with urea in <b>34</b> (also a 22-membered ring) led to a slight reduction in potency, whereas replacement of the polymethylene chain with an ethylene glycol-based linker led to a weak inhibitor (<b>35</b>) with an IC<sub>50</sub> of 850 nM. This further highlights the importance of hydrophobic interactions in the P2/P3′ region. Compound <b>36</b>, which lacks a hydroxyl on the phenylstatine backbone, has a three-order-of-magnitude lower potency than its closest analogue <b>9</b>, which directly demonstrates the essential role of the hydroxyl group interacting with catalytic aspartates.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Selected Backbone Modifications of Macrocyclic Inhibitors of Human CatD</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0013.gif" alt="" id="gr13" /></img><div></div></div></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Macrocycles Have Drug-like Properties</h4><div class="NLM_p">We evaluated the basic physicochemical properties of representative macrocyclic inhibitors and calculated parameters of Lipinski’s “rule of five” for drug-likeness to compare a set of the most potent macrocycles with pepstatin A (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 1). Numbers of hydrogen bond donors and acceptors for macrocycles (4 and 7/8, respectively) meet the “rule of five” criteria (values lower than 5 and 10), while values for pepstatin A are roughly double that of macrocycles and do not meet the criteria. The molecular mass of macrocycles was in the range of 488–557 Da; the favorable values are below 500 Da; however, this parameter has previously been shown to be rather relaxed for macrocycle drugs.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Pepstatin A, with molecular mass of 686 Da, violated the rule. The partition coefficients for all compounds complied with the rule (log <i>P</i> below 5). Additional parameters used in drug design, including the number of rotatable bonds and topological polar surface area, were within the favorable range (below 10 and below 140 Å<sup>2</sup>, respectively) for macrocycles (3 and 108–120 Å<sup>2</sup>) and out of range for pepstatin A (22 and 233 Å<sup>2</sup>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 1).</div><div class="NLM_p last">Furthermore, we selected two representative macrocycles to demonstrate that highly potent inhibitors (with single-digit nanomolar IC<sub>50</sub> values) can be optimized to display desired properties including solubility and cell permeability. Ionizable <b>25</b> showed better solubility than pepstatin A (342 and 197 μg·mL<sup>–1</sup>, respectively), and nonionizable <b>33</b> displayed better permeability in Caco 2 cells, with moderate efficiency compared to the poor efficiency measured for pepstatin A (<i>P</i><sub>app</sub> of 16.8 × 10<sup>–7</sup> and 1.1 × 10<sup>–7</sup> cm·s<sup>–1</sup>, respectively). Notably, these representative macrocycles were noncytotoxic, similar to pepstatin A (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 2). In conclusion, the developed macrocyclic scaffold displays drug-like properties and significantly improves upon the pepstatin A parent molecule in a number of parameters relevant to drug design.</div></div><div id="sec2_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Synthesis of Pepstatin A Fragments and Macrocycles</h4><div class="NLM_p">Linear truncated analogues of pepstatin A (<b>2–5</b>) were prepared by standard solution techniques from commercially available amino acid derivatives. During synthesis, a <i>t</i>-butoxycarbonyl (Boc) group was used for transient protection of the α-amino group, whereas the terminal carboxylate was left unprotected. <i>N</i>-hydroxysuccinimidyl esters were used for coupling reactions; these esters were purchased or generated in situ using <i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyl-<i>O</i>-(<i>N</i>-succinimidyl)uronium tetrafluoroborate (TSTU).</div><div class="NLM_p">Statine analogues were prepared according to a modified version of the procedure described by Jouin et al.,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> except for commercially available analogues. Preparation of the analogues (<b>41a–f</b>) is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The long-chain ω-amino acids were purchased, with the exception of methyl 14-aminotetradecanoate (<b>46</b>) and methyl 15-aminopentadecanoate. Compound <b>46</b> was prepared from Boc-12-aminododecanoic acid (<b>42</b>) as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Methyl 15-aminopentadecanoate was prepared according to the published procedure.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Statine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a): R = 2-naphthyl; (b): R = 4-thiazolyl; (c): R = 4-Me-C<sub>6</sub>H<sub>4</sub>-CH<sub>2</sub>S; (d): R = 3,4-dichlorophenyl; (e): R = 3,4-difluorophenyl; (f): R = 4-fluorophenyl. Reagents and conditions: (a) DCC, DMAP, Meldrum’s acid, DCM; (b) EtOAc, reflux; (c) NaBH<sub>4</sub>, DCM/AcOH/10:1; (d) NaOH, acetone.</p></p></figure><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Methyl 14-Aminotetradecanoate<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DCC, DMAP, Meldrum’s acid, DCM; (b) NaBH<sub>4</sub>, DCM/AcOH/10:1; (c) 110 °C, DMF/H<sub>2</sub>O/10:1; (d) (1) SOCl<sub>2</sub>, MeOH, 0 °C; (2) TFA.</p></p></figure><div class="NLM_p">Macrocyclic inhibitors with an aliphatic P2–P3′ bridge (<b>6–32</b>) were prepared according to <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. First, a Boc-protected amino acid (<b>47</b>) was coupled with an ω-amino fatty acid methyl ester (<b>48</b>) using standard coupling conditions to yield <b>49</b>. After removal of the Boc group with TFA, the free amine was coupled with a corresponding Boc-protected statine analogue to yield <b>50</b>. Methyl ester <b>50</b> was then hydrolyzed using LiOH, and carboxylic acid <b>51</b> was converted to an <i>N</i>-hydroxysuccinimidyl ester with TSTU. The Boc-group was removed with TFA, and the intermediate was cyclized by slow addition into basic solution. Compounds <b>33</b>, <b>34</b>, and <b>35</b> were prepared by a slightly modified procedure; details are provided in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. A different bond was chosen for cyclization of the PEG-containing inhibitor <b>35</b> due to availability of the Boc-protected precursor (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). A similar procedure was used to synthesize the carbamate-containing compound <b>33</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. General Synthesis of Macrocycles with an Aliphatic Chain<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a): R<sup>1</sup> = benzyl, R<sup>2</sup> = isopropyl; (b): R<sup>1</sup> = benzyl, R<sup>2</sup> = 3-pyridylmethyl. Reagents and conditions: (a) HBTU, DIEA, DMF; (b) (1) TFA, DCM; (2) Boc-statine-OH, HBTU, DIEA, DMF; (c) LiOH, THF/H<sub>2</sub>O; (d) (1) TSTU, DIEA, DMF; (2) TFA; (3) DIEA, DMF.</p></p></figure><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Macrocycle with a PEG Chain<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) HBTU, DIEA, DMF; (2) H-Val-OMe; (b) (1) TFA; (2) BocHN-PEG-3-CH<sub>2</sub>CH<sub>2</sub>COOH, HBTU, DIEA, DMF; (c) (1) LiOH, THF/H<sub>2</sub>O; (2) TFA; (3) DPPA, DIEA, DMF.</p></p></figure><figure id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Macrocycle with a Carbamate Moiety<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DIEA, DMF; (b) (1) TFA, DCM; (2) Boc-Statine-OH, TSTU, DIEA, DMF; (c) (1) DSC, DMF; (2) TFA, DCM; (3) DIEA, DMF.</p></p></figure></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Crystal Structures of Three Macrocyclic Inhibitors in Complex with Human CatD</h3><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Macrocyclic Inhibitors Target the Central Part of the CatD Active Site</h4><div class="NLM_p">Native human CatD was crystallized in complexes with macrocyclic inhibitors <b>9</b>, <b>14</b>, and <b>33</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a), which exhibit high inhibitory potency in the low nanomolar and subnanomolar range. The structures were determined by molecular replacement using the structure of uncomplexed CatD (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYA">1LYA</a>) as a model. All complexes crystallized in the orthorhombic <i>C</i>222<sub>1</sub> group with one molecule in the asymmetric unit. The CatD-<b>9</b>, CatD-<b>14</b>, and CatD-<b>33</b> structures were refined to resolutions of 1.55, 1.80, and 1.85 Å, respectively. The CatD structures consist of two chains: the light chain spans residues 1–97 and the heavy chain includes residues 106–347 (mature CatD numbering). One N-terminal residue from the heavy chain and one C-terminal amino acid from both chains could not be located in the electron density map. Electron densities used for modeling of inhibitors were of excellent quality in all structures (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b). The CatD structures did not show any significant differences; root-mean-square deviation for superposition of protein backbones was 0.14 Å for CatD-<b>14</b> versus CatD-<b>33</b>/CatD-<b>9</b> and 0.11 Å for CatD-<b>33</b> versus CatD-<b>9</b>. All structures are <i>N</i>-glycosylated at Asn70 and Asn199; the Asn70 glycan in CatD-<b>9</b> occurs in two conformations.</div><figure id="fig2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structures of human CatD in complex with three macrocyclic inhibitors (<b>9</b>, <b>14</b>, and <b>33</b>, PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QCB">6QCB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBG">6QBG</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBH">6QBH</a>). (<b>a</b>) Top view of the CatD active site with the superimposed macrocyclic inhibitors. The enzyme is shown as semitransparent surface and the inhibitors as sticks (<b>9</b>, magenta; <b>14</b>, green; <b>33</b>, cyan). The active site residues are colored as follows: Tyr flap residues are pale blue, polyproline loop residues are pale orange, and the rest of the residues are light yellow. (b) Chemical structures and electron density maps of macrocyclic inhibitors. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> maps are contoured at 1.2σ. (c) Hydrogen bond networks (dashed lines) formed between CatD and macrocyclic inhibitors are shown in two views rotated by 180°. Inhibitors are colored as in (b); interacting enzyme residues are gray. Heteroatoms have standard color coding. (d) Superposition of macrocyclic inhibitors colored as in (b) and pepstatin A (yellow) in the CatD active site. The enzyme is in semitransparent surface representation, and residues that form hydrophobic interactions with inhibitors are highlighted. The residues interacting with the macrocycles are pale cyan, with pepstatin A are yellow, and with both inhibitors are pale green. (e) Hydrogen bond network (dashed lines) formed between CatD and pepstatin A. Pepstatin A is colored yellow; interacting enzyme residues are gray. Yellow boxes indicate segments of pepstatin A that are absent in the macrocycle scaffold. Residues labeled in brown form pepstatin A-specific hydrogen bonds; macrocycle-specific hydrogen bonding is labeled in (c).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The complexed macrocycles contain valine in the P2′ position and a statine derivative with an aromatic ring (benzyl for <b>9</b> and <b>33</b>, β-naphthyl for <b>14</b>) in the P1 position. They are linked by an aminododecanoic acid bridge, which is extended by an oxygen atom in the CatD-<b>33</b> complex. Positions of the macrocyclic scaffold in all structures overlap; P1 and P2′ are buried deep inside the active site pocket, while the aliphatic bridge faces out (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a). The orientation of the β-naphthyl group in CatD-<b>14</b> differs slightly from that of aromatic rings in the other structures (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c), and there is an additional atomic contact formed between this substituent and the protein backbone (Thr125, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 3). The prolongation of the bridge by one atom in CatD-<b>33</b> resulted in a change in bridge conformation associated with formation of a new hydrophobic interaction with Met307 and loss of a hydrogen bond with Ser80 compared to the other complexes (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 3). In the CatD-<b>14</b> structure, a second inhibitor molecule was found outside the active site and appears to be a crystallographic artifact caused by an excess of inhibitor during crystallization.</div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Small Macrocycles Mimic the Binding Conformation of Pepstatin A and Establish a Similar Number of Interactions</h4><div class="NLM_p">The active site of CatD contains a catalytic dyad of two aspartic residues (Asp33 and Asp231) in a conserved Asp-Thr-Gly motif. The dyad is hydrogen bonded to the hydroxyl group of the phenylstatine/naphthylstatine residue of macrocyclic inhibitors (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c), occupying the position of a water molecule that interacts with the dyad in uncomplexed CatD (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYA">1LYA</a>). This interaction, which mimics a tetrahedral intermediate of the enzymatic reaction, is the key inhibitory determinant in the CatD–macrocycle complexes and is identical to that observed in the CatD–pepstatin A complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYB">1LYB</a>).</div><div class="NLM_p last">The macrocyclic inhibitors occupy the S2 to S3′ subsites of CatD, which are part of the larger binding area of linear pepstatin A spanning the S4 to S4′ subsites. However, both types of inhibitors form comparable numbers of atomic contacts and hydrogen bonds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 3). The three macrocyclic inhibitors share a set of common interactions with the CatD active site residues. These include hydrogen bonds with Asp33, Gly35, His77, Gly79, Ser80, Tyr205, Asp231, and Gly233 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c). Pepstatin A forms a very similar network of hydrogen bonds with a few differences. There are two extra hydrogen bonds between Ser235 and the N-terminal segment of pepstatin A, whereas the hydrogen bond formed between His77 and macrocyclic inhibitors is absent in the pepstatin A complex (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e). In addition, macrocyclic inhibitors form more hydrophobic interactions than pepstatin A due to bulkier groups in the P1 and P2′ positions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d). The benzyl and β-naphthyl aromatic rings in the P1 position of macrocycles form hydrophobic contacts with Phe126 and Phe131, whereas pepstatin A, with a smaller isobutyl group in P1, lacks these interactions. Similarly, the P2′ Val residue of macrocycles forms more contacts than the P2′ Ala of pepstatin A (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 3).</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Computational Analysis of the Binding Mode of Macrocycles</h3><div class="NLM_p">We evaluated the interaction energies of macrocyclic inhibitors <b>9</b>, <b>14</b>, and <b>33</b> bound to CatD subsites by performing quantum chemical calculations on the crystallographic complexes (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). We found that the interaction of the hydroxyl group of the inhibitor phenylstatine/naphthylstatine residue has the most negative Δ<i>G</i>′. The critical role of this interaction is reflected in the high IC<sub>50</sub> measured for macrocycle derivative <b>36</b>, which lacks this hydroxyl (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Furthermore, <b>14</b> had a more negative P1 Δ<i>G</i>′ value than <b>9</b> and <b>33</b>, which might be due to the more hydrophobic β-naphthyl group of <b>14</b> compared to phenyl. The bridge segments (br and br′) in <b>33</b> formed more favorable interactions than those in <b>9</b> and <b>14</b>. None of the segments of <b>9</b> formed considerably stronger interactions compared to the other two inhibitors, which correlates with their IC<sub>50</sub> values.</div><figure id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Quantum chemical calculations of interaction “free” energies of three crystallographically characterized macrocyclic inhibitors in the CatD active site. The 3D view shows fragmentation of an inhibitor into segments (alternating black and orange) and the active site residues of CatD (gray) used for the calculations (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QCB">6QCB</a>). (a) Contributions of individual inhibitor segments. (b) Contributions of the active site residues obtained from a “virtual glycine” scan.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Next, we computed the energy contributions of individual amino acid residues in the CatD active site interacting with the macrocyclic inhibitors in crystallographic complexes (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). These results confirmed the critical contribution of the catalytic Asp33 and Asp231 residues. Additionally, we observed a large contribution from residues Tyr78, Ser80, and Tyr205. To gain deeper insights into the effects of P1 and P2′ variations, we modeled and scored inhibitors <b>8</b>, <b>22</b>, <b>24</b>, <b>25</b>, and <b>31</b> in the CatD active site using the crystallographic complexes as a guide (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Figure 1). These representative inhibitors were selected so that the substituents differed markedly in size and hydrophobicity. Compounds <b>22</b> and <b>24</b>, in which the hydrophobic P1 phenyl group of <b>9</b> is replaced with less hydrophobic heterocycles, had less negative scores due to lower negative interaction energy terms, likely reflecting the missing optimal interaction partners in the S1 subsite. Compounds <b>25</b>, <b>8</b>, and <b>31</b> differ in P2′. The Val of <b>9</b> is replaced with Ala in <b>8</b>, which resulted in a less negative interaction energy term and total score. In <b>31</b> with P2′ Thr, the hydroxyl group forms only one hydrogen bond with the resident water molecule W14. The hydroxyl likely loses some hydrogen bonds upon binding, and the favorable increase in the interaction energy term is thus smaller than the unfavorable change in the desolvation term (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Figure 1). The resulting score was less negative than that of <b>9</b>, which correlates with the IC<sub>50</sub> values observed for these inhibitors. Introduction of a pyridine group in <b>25</b> decreased the IC<sub>50</sub> value. Our in silico model showed that the N atom of pyridine forms a hydrogen bond with the His77 backbone, and the ring forms a stacking interaction with the Val144 side chain. The corresponding favorable change in interaction energy term is larger than the increase in the desolvation term (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Figure 1), resulting in a more negative score.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Inhibition Kinetics of Macrocycles Differ from Those of Pepstatin A</h3><div class="NLM_p last">Compound <b>9</b> was chosen as a model macrocyclic inhibitor for a detailed analysis of inhibition kinetics with human CatD. First, we demonstrated that the inhibitor has a competitive mode of inhibition, consistent with its crystallographic binding mode (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Figure 2a). Second, we monitored the inhibition after adding enzyme over a prolonged period of time to assess the rate at which the system establishes equilibrium. A linear dependency was observed (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Figure 2b), suggesting that the macrocyclic inhibitors do not follow slow-binding kinetics, as has been previously described for the interaction of pepstatin A with pepsin.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Pepstatin A exhibits tight-binding properties with aspartic proteases and is therefore used as a titrant for determination of active enzyme concentration.<a onclick="showRef(event, 'ref18 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref21">(18,21)</a> However, we did not observe tight-binding properties for the macrocyclic inhibitor; its potency was independent of the enzyme concentration in the assay (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Figure 2c). These findings suggest that redesigning pepstatin A into a small macrocyclic mimetic is associated with changes in kinetic inhibition behavior.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Macrocycles Selectively Inhibit CatD and Related Aspartic Proteases</h3><div class="NLM_p">We assessed the selectivity of macrocyclic inhibitors using a panel of model aspartic proteases from the A1 (pepsin) and A2 (retropepsin) families. At 50 nM inhibitor concentration, human and tick CatDs, yeast proteinase A, and porcine pepsin were generally susceptible to inhibition but exhibited variations in selectivity profile (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). This is consistent with the fact that these A1-family enzymes are closely related with very similar active site structure and substrate specificity. Several of the macrocyclic compounds displayed a substantial degree of selectivity. Compounds <b>20</b>, <b>21</b>, and <b>27</b> exhibited a preference for human CatD, whereas <b>7</b>, <b>12</b>, <b>18</b>, <b>22</b>, <b>26</b>, and <b>28–30</b> preferred porcine pepsin. In contrast, macrocyclic inhibitors were not effective at 50 nM concentration against more distantly related aspartic proteases, including human renin and BACE1 from the A1 family and HIV-1 protease from the A2 family. Increasing the inhibitor concentration to 5 μM resulted in only modest inhibitory activity, with pronounced differences in selectivity (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Inhibitory selectivity of macrocyclic compounds against representative aspartic proteases. Inhibition of proteolytic activity was screened in a kinetic activity assay with FRET fluorogenic substrates. Inhibitors were applied at 50 nM concentration (upper panel) for human cathepsin D, tick cathepsin D1 (IrCatD), yeast proteinase A (ProtA), and porcine pepsin (pPEP), and at 5 μM concentration (lower panel) for human β-secretase (BACE1), human renin, and HIV-1 protease (HIV1 PR). Data are expressed as % inhibition relative to the uninhibited control (0%) and displayed as a heat map (blue and green scales). Inhibitors are grouped according to their appearance in data tables.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Furthermore, we assessed the proteolytic stability of the macrocycles. Human CatD, papain, trypsin/chymotrypsin, and thermolysin were used as typical representatives of four major protease classes: aspartic proteases, cysteine proteases, serine proteases, and metalloproteases, respectively. Selected macrocycles (<b>25</b> and <b>33</b>) subjected to exhaustive treatment with these proteases did not undergo significant degradation (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Notably, the non-aspartic proteases papain, trypsin/chymotrypsin, and thermolysin were not inhibited by <b>25</b> and <b>33</b>, which further demonstrates the high selectivity of the macrocycles.</div><div class="NLM_p last">In conclusion, our macrocyclic inhibitor scaffold is proteolytically stable and effectively regulates the activity of human CatD and its close homologues from the A1 family of aspartic proteases. The inhibitory selectivity can be fine-tuned by introducing structural diversity into the P1, P2′, and linker regions.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70214" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70214" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The high affinity of naturally occurring pepstatin A for human CatD makes it an attractive template for biomimetic inhibitor design.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Here, we used the crystal structure of the CatD–pepstatin A complex and previously obtained SAR data as guides for rational design of small peptidomimetic inhibitors.<a onclick="showRef(event, 'ref19 ref30'); return false;" href="javascript:void(0);" class="ref ref19 ref30">(19,30)</a> We aimed to avoid the inherent difficulties connected with use of linear peptides as therapeutics. Linear peptides suffer from poor oral absorption, short duration of action, proteolytic instability, and rapid biliary clearance. In contrast, cyclic peptidomimetics in general have better pharmacokinetic properties and are being increasingly employed in drug development.<a onclick="showRef(event, 'ref25 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref25 ref31 ref32 ref33">(25,31−33)</a></div><div class="NLM_p">First, we evaluated linear fragments of pepstatin A to assess size requirements for the inhibitory motif. Second, we analyzed pepstatin A bound in the active site of human CatD. The inhibitor adopts a U-shaped conformation with several side chains in positions that can be effectively connected by an aliphatic bridge to impose conformational constraint by cyclization. Based on these data, we designed a functional macrocyclic scaffold with minimized size. High-resolution crystal structures of CatD–macrocycle complexes revealed that this scaffold stabilizes the active site conformation seen in pepstatin A and forms a comparable number of atomic contacts and hydrogen bonds as the larger structure of the parent ligand. Quantum chemical calculations of the interaction energies of macrocycles demonstrated the critical role of the statine residue, which is correctly positioned in the scaffold to interact with the enzyme catalytic center. The results of the structural analysis are in line with findings that proteases recognize peptide ligands with extended β-strand conformation and that this extended-like conformation can be induced in short peptides by macrocyclization to improve their affinity.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a></div><div class="NLM_p last">We prepared a library of more than 30 macrocyclic compounds with variable bridge size and various side chains in the P1 and P2′ positions. SAR analysis revealed that the minimized scaffold effectively potentiates inhibition, and we identified human CatD inhibitors with low nanomolar and subnanomolar potency. The inhibitor selectivity can be altered by residue substitutions in macrocycles to preferentially target human CatD or its close homologues from the A1 family of aspartic proteases. Notably, the developed compounds display favorable drug-like properties comparable to those reported for other medicinally relevant macrocycles.<a onclick="showRef(event, 'ref25 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref25 ref31 ref32 ref33">(25,31−33)</a> They meet Lipinski’s “rule of five” criteria and other physicochemical parameters for drug design that pepstatin A generally does not. Furthermore, the macrocycles are proteolytically stable and noncytotoxic, and parameters such as solubility and cell permeability can be controlled by structural substitutions. In conclusion, we present new biomimetic macrocyclics as a powerful template for further optimization and future development of potential chemotherapeutics against pathologies associated with human CatD and CatD-like proteases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43746" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43746" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Materials</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div class="NLM_p first">Mature human CatD was isolated from human placenta as previously described<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> with minor modifications. Briefly, the tissue was homogenized and resuspended in 3% NaCl solution. The pH was adjusted to 3.5, and the suspension stirred for 12 h and centrifuged at 6000<i>g</i> for 1 h. The supernatant was precipitated with 45% saturated ammonium sulfate, centrifuged at 11,000<i>g</i> for 1 h, and precipitated with 65% ammonium sulfate. The pellet was resuspended in water and dialyzed against 1 mM EDTA. The resulting material was separated on a Sephadex G150 column (GE Healthcare) equilibrated with 10 mM sodium phosphate, pH 6.8. Fractions with CatD activity were pooled and dialyzed against equilibrated in 30 mM sodium acetate, pH 3.5, 0.5 M NaCl. CatD was purified by pepstatinyl-Sepharose affinity chromatography. Several wash steps (30 mM sodium acetate, pH 3.5; 30 mM sodium acetate, pH 3.5, 5% Triton X-100; and 30 mM Tris-HCl, pH 7.5, 0.5 M NaCl) were performed, and CatD was eluted with 30 mM Tris-HCl, pH 8.2, 0.5 M NaCl. The elution fraction was buffer-exchanged into 25 mM Tris-HCl, pH 7.0, and concentrated by ultrafiltration to 7.5 mg mL<sup>–1</sup>.</div><div class="NLM_p last">CatD from the hard tick <i>Ixodes ricinus</i> was produced in the bacterial expression, refolded, and chromatographically purified as previously described.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Human BACE1 and human renin were purchased from R&D and Enzo, respectively; yeast proteinase A, porcine pepsin, bovine trypsin, bovine chymotrypsin, papain, and thermolysin were purchased from Sigma-Aldrich. HIV-1 protease and its substrate were kindly donated by Pavlı́na Řezáčová and Tat’ána Majerová, respectively. The substrates for BACE1 and renin were purchased from R&D and Cayman Chemicals, respectively; the substrates for non-aspartic proteases were from Bachem; the CatD substrate was prepared as previously described;<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and pepstatin A was purchased from Sigma-Aldrich.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Procedures</h3><div class="NLM_p">Commercially available high-performance liquid chromatography (HPLC) grade acetonitrile, catalysts, and reagent-grade materials were used for synthesis. TLC was performed on Silica gel 60 F254-coated aluminum sheets (Merck). Flash chromatography was performed on Silica gel 60 (0.040–0.063 mm, Fluka). All chemicals were purchased from Sigma-Aldrich, TCI, Combi-Blocks, or Iris Biotech GmbH and were used without further purification. <sup>1</sup>H and <sup>13</sup>C NMR spectra were measured using Bruker AVANCE III HD 400 MHz, Bruker AVANCE III HD 500 MHz, and Bruker AVANCE III 600 MHz instruments. The internal signal of TMS (δ 0.0, CDCl<sub>3</sub>) or residual signal of CDCl<sub>3</sub> (δ 7.26), DMSO-<i>d</i><sub>6</sub> (δ 2.50, 3.33), or <i>d</i><sub>4</sub>-MeOD (δ 3.31, 4.87) was used for standardization of <sup>1</sup>H NMR spectra. For <sup>13</sup>C NMR spectra (APT experiments), the residual signal of CDCl<sub>3</sub> (δ 77.16), DMSO-<i>d</i><sub>6</sub> (δ 39.52), or <i>d</i><sub>4</sub>-MeOD (δ 49.00) was used. NMR spectra were recorded at room temperature unless noted otherwise. Chemical shifts are given in δ scale; coupling constants (<i>J</i>) are given in Hz. The ESI mass spectra were recorded using a ZQ micromass mass spectrometer (Waters) equipped with an ESCi multimode ion source and controlled by MassLynx software. Low-resolution ESI mass spectra were recorded using a quadrupole orthogonal acceleration time-of-flight tandem mass spectrometer (Q-Tof micro, Waters) and high-resolution ESI mass spectra using a hybrid FT mass spectrometer combining a linear ion trap MS and Orbitrap mass analyzer (LTQ Orbitrap XL, Thermo Fisher Scientific). The conditions were optimized for suitable ionization in the ESI Orbitrap source (sheath gas flow rate of 35 au, aux gas flow rate of 10 au of nitrogen, source voltage of 4.3 kV, capillary voltage of 40 V, capillary temperature of 275 °C, tube lens voltage of 155 V). The samples were dissolved in methanol and applied by direct injection. Inhibitors were purified by preparative scale HPLC on a JASCO PU-975 instrument (flow rate of 10 mL min<sup>–1</sup>) equipped with a UV-975 UV detector and Waters YMC-PACK ODS-AM C18 Prep Column (5 μm, 20 × 250 mm). The purity of compounds was assessed on an analytical JASCO PU-1580 HPLC (flow rate of 1 mL min<sup>–1</sup>, invariable gradient from 2 to 100% acetonitrile in 30 min) with a Watrex C18 Analytical Column (5 μm, 250 × 5 mm). The final inhibitors were of at least 95% purity. Optical rotations were measured using an Autopol IV instrument (Rudolph Research Analytical). All final compounds passed the PAINS filter using false positive remover.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Aqueous solubility of compounds was determined in phosphate-buffered saline, pH 7.4, at 26 °C, and quantified by HPLC-MS using a Waters ACQUITY C18 BEH column (1.7 μm, 2.1 × 100 mm) with a formic acid/acetonitrile elution system and UV detection.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Truncated Pepstatin A Analogues (<b>2–5</b>)</h4><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Isovaleroyl-Val-Val-Sta-Ala-OH (<b>2</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 0.83–1.03 (24H, m), 1.27–1.37 (2H, m), 1.40 (3H, d, <i>J</i> = 7.3), 1.52–1.67 (2H, m), 1.75–1.83 (1H, m), 1.99–2.16 (5H, m), 2.28–2.36 (2H, m), 2.63 (1H, p, <i>J</i> = 1.9), 3.09–3.17 (1H, m), 3.91–4.05 (2H, m), 4.13–4.23 (2H, m), 4.36 (1H, q, <i>J</i> = 7.3), 7.72 (1H, d, <i>J</i> = 9.0), 8.00 (1H, t, <i>J</i> = 8.6). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 17.9, 18.9, 19.1, 19.9, 20.1, 22.2, 22.8 (2C), 23.9, 25.8, 27.5, 31.6, 31.7, 41.1, 41.6, 46.0, 49.6, 52.9, 60.4, 60.6, 71.3, 173.5, 173.6, 173.8, 175.5, 176.5. ESI MS: 551 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>26</sub>H<sub>48</sub>O<sub>7</sub>N<sub>4</sub>Na, 551.3415; found, 551.3413.</div></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Isovaleroyl-Val-Val-Sta-OH (<b>3</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 0.85–1.00 (24H, m), 1.26–1.39 (1H, m), 1.47–1.68 (2H, m), 1.95–2.18 (6H, m), 2.36 (1H, dd, <i>J</i> = 15.9, 8.7), 2.45 (1H, dd, <i>J</i> = 15.9, 4.3), 3.96–4.08 (2H, m), 4.16–4.23 (2H, m), 7.71 (1H, d, <i>J</i> = 9.1), 7.98 (1H, d, <i>J</i> = 8.4), 8.06 (1H, d, <i>J</i> = 8.7). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 18.9, 19.1, 19.9, 20.0, 22.3, 22.8, 23.8, 25.7, 27.5, 31.7, 31.8, 39.8, 41.1, 46.0, 52.5, 60.3, 60.4, 70.6, 173.3, 173.7, 175.3, 175.5. ESI MS: 458 ([M + H]<sup>+</sup>). HR ESI MS: calcd for C<sub>23</sub>H<sub>44</sub>O<sub>6</sub>N<sub>3</sub>, 458.3225; found, 458.3227.</div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Isovaleroyl-Sta-Ala-Sta-OH (<b>4</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 0.83–1.01 (18H, m), 1.26–1.41 (5H, m), 1.51–1.70 (4H, m), 2.06–2.18 (3H, m), 2.28–2.49 (4H, m), 3.97–4.06 (4H, m), 4.31 (1H, q, <i>J</i> = 7.2). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 18.2, 22.4, 22.7, 23.0, 23.7, 23.8, 25.9, 26.1, 27.3, 39.9, 41.1, 41.2, 46.4, 50.9, 52.0, 52.5, 71.0, 71.3, 174.0, 175.3, 175.4, 175.7. ESI MS: 488 ([M + H]<sup>+</sup>). HR ESI MS: calcd for C<sub>24</sub>H<sub>46</sub>O<sub>7</sub>N<sub>3</sub>, 488.3330; found, 488.3331.</div></div><div id="sec4_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Isovaleroyl-Sta-Ala-OH (<b>5</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 0.88–1.03 (12H, m), 1.25–1.38 (1H, m), 1.40 (3H, d, <i>J</i> = 7.3), 1.51–1.68 (2H, m), 2.02–2.17 (3H, m), 2.24–2.42 (2H, m), 3.96 (1H, ddd, <i>J</i> = 7.8, 5.0, 2.6), 4.01–4.11 (1H, m), 4.38 (1H, q, <i>J</i> = 7.3). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 17.6, 22.1, 22.7, 22.9, 23.9, 26.1, 27.4, 41.0, 41.5, 46.4, 49.4, 52.5, 71.6, 173.8, 175.8, 176.0. ESI MS: 329 ([M – H]<sup>−</sup>). HR ESI MS: calcd for C<sub>16</sub>H<sub>29</sub>O<sub>5</sub>N<sub>2</sub>, 329.2082; found, 329.2078.</div></div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Macrocyclic Inhibitors (<b>6–36</b>)</h4><div id="sec4_2_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-7-Hydroxy-8-isobutyl-3-methyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>6</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.80 (3H, d, <i>J</i> = 6.4), 0.86 (3H, d, <i>J</i> = 6.5), 1.10–1.64 (21H, m), 2.06–2.26 (4H, m), 2.76–2.86 (1H, m), 3.18–3.28 (1H, m), 3.72–3.82 (1H, m), 3.88 (1H, q, <i>J</i> = 6.7), 4.05 (1H, p, <i>J</i> = 7.2), 4.79 (1H, d, <i>J</i> = 5.3), 7.44 (1H, d, <i>J</i> = 9.3), 7.93 (1H, d, <i>J</i> = 7.0), 7.97 (1H, t, <i>J</i> = 5.8). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 17.6, 21.7, 23.3, 24.4, 24.9, 26.1, 26.6, 26.7, 27.1, 27.3, 27.8, 34.9, 38.3, 39.8, 48.9, 49.1, 69.5, 170.3, 172.2, 172.5. ESI MS: 448 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>23</sub>H<sub>43</sub>O<sub>4</sub>N<sub>3</sub>Na, 448.31458; found, 448.31401.</div></div><div id="sec4_2_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-7-Hydroxy-8-isobutyl-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>7</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.80 (3H, d, <i>J</i> = 6.5), 0.84 (6H, d, <i>J</i> = 6.8), 0.86 (3H, d, <i>J</i> = 6.5), 1.14–1.57 (21H, m), 1.95–2.03 (1H, m), 2.16–2.21 (2H, m), 2.13 (1H, dd, <i>J</i> = 14.5, 6.1), 2.21 (1H, dd, <i>J</i> = 14.5, 8.0), 2.77–2.85 (1H, m), 3.19–3.28 (1H, m), 3.77 (1H, dddd, <i>J</i> = 10.7, 10.5, 4.1, 1.6), 3.93–3.98 (1H, m), 3.86–3.90 (1H, m), 4.79 (1H, br s), 7.44 (1H, d, <i>J</i> = 9.4), 7.93 (1H, d, <i>J</i> = 6.9), 7.97 (1H, dd, <i>J</i> = 6.4, 5.1). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.0, 19.2, 21.6, 23.3, 24.4, 24.8, 24.9, 26.1, 26.6, 26.7, 27.0, 27.2, 27.8, 29.5, 34.8, 38.3, 39.7, 48.8, 49.1, 69.5, 170.3, 172.1, 172.5. ESI MS: 476 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>25</sub>H<sub>47</sub>O<sub>4</sub>N<sub>3</sub>Na, 476.34588; found, 476.34557.</div></div><div id="sec4_2_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-methyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>8</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): 0.92–1.02 (2H, m), 1.09–1.45 (18H, m), 2.06–2.12 (3H, m), 2.16–2.26 (2H, m), 2.68–2.81 (3H, m), 3.21 (1H, dq, <i>J</i> = 14.0, 7.1), 3.86–3.92 (1H, m), 3.99–4.03 (1H, m), 4.07 (1H, p, <i>J</i> = 7.2), 5.02 (1H, br s), 7.11–7.25 (5H, m), 7.64 (1H, d, <i>J</i> = 9.3), 7.91–7.99 (2H, m). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>): 17.6, 24.8 (2C), 26.0, 26.5, 26.6, 27.0, 27.1, 27.2, 27.6, 30.7, 34.8, 36.9, 38.3, 39.9, 40.4, 48.9, 53.2, 69.4, 125.8, 127.9, 128.9, 139.5, 170.1, 172.2, 172.4. ESI MS: 482 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>26</sub>H<sub>41</sub>O<sub>4</sub>N<sub>3</sub>Na, 482.29893; found, 482.29871.</div></div><div id="sec4_2_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>9</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.84 (6H, d, <i>J</i> = 6.8), 0.97–1.07 (2H, m), 1.09–1.25 (12H, m), 1.26–1.45 (4H, m), 1.92–2.17 (3H, m), 2.18–2.34 (2H, m), 2.64–2.75 (2H, m), 2.80 (1H, dd, <i>J</i> = 13.8, 4.1), 3.29 (1H, dt, <i>J</i> = 13.8, 6.9), 3.51 (1H, s), 3.84–3.91 (1H, m), 3.92–4.02 (2H, m), 7.11–7.25 (5H, m), 7.60 (1H, d, <i>J</i> = 8.3), 7.66 (1H, d, <i>J</i> = 9.2), 7.96 (1H, dd, <i>J</i> = 7.0, 4.4). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.0, 19.3, 24.9, 25.1, 26.3, 26.8, 27.2, 27.3, 27.4, 27.6, 28.0, 29.0, 29.5, 34.8, 36.6, 38.2, 53.8, 58.8, 69.4, 125.8, 128.0, 129.0, 139.6, 170.5, 170.8, 172.5. ESI MS: 510 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>45</sub>O<sub>4</sub>N<sub>3</sub>Na, 510.33023; found, 510.33010.</div></div><div id="sec4_2_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-isopropyl-1,4,9-triazacycloicosane-2,5,10-trione (<b>10</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.82 (3H, d, <i>J</i> = 5.2), 0.84 (3H, d, <i>J</i> = 6.0), 0.92–1.04 (1H, m), 1.04–1.28 (11H, m), 1.29–1.45 (4H, m), 1.89–2.04 (2H, m), 2.12 (1H, dt, <i>J</i> = 13.2, 6.3), 2.18–2.30 (2H, m), 2.58–2.69 (1H, m), 2.76–2.89 (2H, m), 3.22–3.29 (1H, m), 3.91–4.04 (3H, m), 4.99 (1H, d, <i>J</i> = 5.2), 7.12–7.26 (5H, m), 7.64 (1H, d, <i>J</i> = 8.7), 7.68 (1H, d, <i>J</i> = 8.9), 7.86 (1H, dd, <i>J</i> = 7.2, 4.4). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.2, 19.3, 24.6, 26.9, 27.0, 27.3, 27.5, 27.8, 28.0, 29.4, 35.0, 35.6, 37.6, 40.0, 54.0, 58.5, 69.9, 125.8, 128.0, 128.9, 139.7, 170.6, 170.9, 172.2. ESI MS: 496 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>27</sub>H<sub>43</sub>O<sub>4</sub>N<sub>3</sub>Na, 496.31458; found, 496.31396.</div></div><div id="sec4_2_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-isopropyl-1,4,9-triazacyclodocosane-2,5,10-trione (<b>11</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.77–0.88 (6H, m), 1.03–1.52 (20H, m), 1.92–2.03 (2H, m), 2.07–2.16 (1H, m), 2.18–2.30 (2H, m), 2.60–2.70 (1H, m), 2.74–2.88 (2H, m), 3.23–3.33 (1H, m), 3.91–3.99 (2H, m), 4.04 (1H, dd, <i>J</i> = 8.7, 6.3), 7.11–7.29 (5H, m), 7.58 (1H, d, <i>J</i> = 8.7), 7.64 (1H, d, <i>J</i> = 9.0), 7.91 (1H, dd, <i>J</i> = 7.1, 4.5). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.0, 19.3, 24.9, 25.0, 27.0 (2C), 27.2, 27.3, 27.4, 27.6, 28.3, 29.8, 35.2, 36.1, 38.1, 40.0, 53.8, 58.1, 68.9, 125.8, 128.0, 128.9, 139.6, 170.5, 170.9, 172.3. ESI MS: 524 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>29</sub>H<sub>47</sub>O<sub>4</sub>N<sub>3</sub>Na, 524.34588; found, 524.34564.</div></div><div id="sec4_2_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-isopropyl-1,4,9-triazacyclotricosane-2,5,10-trione (<b>12</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 0.94 (3H, d, <i>J</i> = 6.9), 0.95 (3H, d, <i>J</i> = 6.9), 1.11–1.34 (18H, m), 1.40–1.52 (4H, m), 2.08–2.23 (3H, m), 2.37–2.49 (2H, m), 2.78–2.94 (3H, m), 3.41 (1H, dt, <i>J</i> = 14.0, 7.0), 4.03 (1H, ddd, <i>J</i> = 10.0, 4.9, 1.7), 4.11–4.17 (2H, m), 7.13–7.25 (5H, m). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 16.8, 18.3, 25.2, 25.7, 26.8, 27.0, 27.2, 27.4, 27.5, 27.8, 27.9, 28.1, 28.6, 29.9, 35.6, 37.5, 39.0, 40.2, 54.1, 59.4, 69.9, 125.9, 127.9, 128.9, 138.9, 172.4 (2C), 174.6. ESI MS: 538 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>30</sub>H<sub>49</sub>O<sub>4</sub>N<sub>3</sub>Na, 538.36153; found, 538.36136.</div></div><div id="sec4_2_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-isopropyl-1,4,9-triazacyclotetracosane-2,5,10-trione (<b>13</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 0.93 (3H, d, <i>J</i> = 7.1), 0.95 (3H, d, <i>J</i> = 7.1), 1.11–1.35 (20H, m), 1.39–1.49 (4H, m), 2.05–2.24 (3H, m), 2.38–2.52 (2H, m), 2.78–3.00 (3H, m), 3.34–3.36 (1H, m), 4.04–4.09 (1H, m), 4.12–4.21 (2H, m), 7.12–7.25 (5H, m). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 18.0, 19.7, 26.9, 27.4, 27.9, 28.3, 28.5, 28.6, 28.7, 29.0, 29.2, 29.6, 29.7, 30.3, 31.3, 37.1, 38.9, 40.4, 41.5, 55.4, 60.5, 71.2, 127.3, 129.3, 130.3, 140.3, 173.7, 173.8, 176.0. ESI MS: 552 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>31</sub>H<sub>51</sub>O<sub>4</sub>N<sub>3</sub>Na, 552.37718; found, 552.37729.</div></div><div id="sec4_2_2_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-7-Hydroxy-3-isopropyl-8-(naphthalen-2-ylmethyl)-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>14</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.82 (3H, d, <i>J</i> = 6.7), 0.83 (3H, d, <i>J</i> = 6.7), 0.94–1.41 (18H, m), 1.93–2.03 (1H, m), 2.03–2.16 (2H, m), 2.25 (1H, dd, <i>J</i> = 14.0, 6.1), 2.33 (1H, dd, <i>J</i> = 14.0, 7.8), 2.61–2.68 (1H, m), 2.88 (1H, dd, <i>J</i> = 13.6, 10.3), 2.95 (1H, dd, <i>J</i> = 12.6, 4.2), 3.24–3.32 (1H, m), 3.93 (1H, dd, <i>J</i> = 8.0, 6.2), 3.93–3.98 (1H, m), 4.03–4.08 (1H, m), 5.10 (1H, br s), 7.38 (1H, dd, <i>J</i> = 8.4, 1.5), 7.39–7.47 (2H, m), 7.59 (1H, d, <i>J</i> = 8.0), 7.70 (1H, br s), 7.71–7.87 (4H, m), 7.98 (1H, dd, <i>J</i> = 7.0, 4.4). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 17.9, 19.2, 24.6, 24.8, 25.7, 26.4, 26.7, 27.0, 27.2, 27.7, 29.4, 34.8, 37.2, 38.2, 40.2, 53.7, 58.8, 69.8, 125.0, 125.7, 127.1, 127.2, 127.3, 127.4, 127.8, 131.6, 133.0, 137.2, 170.4, 170.7, 172.5. ESI MS: 560 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>32</sub>H<sub>47</sub>O<sub>4</sub>N<sub>3</sub>Na, 560.34588; found, 560.34551.</div></div><div id="sec4_2_2_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-(3,4-Difluorobenzyl)-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>15</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.83 (6H, d, <i>J</i> = 6.8), 0.88–1.48 (18H, m), 1.95–2.05 (1H, m), 2.09–2.19 (2H, m), 2.21 (1H, dd, <i>J</i> = 14.0, 5.8), 2.32 (1H, dd, <i>J</i> = 14.0, 8.3), 2.62–2.70 (1H, m), 2.70–2.79 (2H, m), 3.24–3.32 (1H, m), 3.75–3.83 (1H, m), 3.93 (1H, dd, <i>J</i> = 7.9, 5.9), 3.98–4.03 (1H, m), 5.11 (1H, d, <i>J</i> = 5.2), 7.01–7.06 (1H, m), 7.21–7.30 (2H, m), 7.60 (1H, d, <i>J</i> = 7.9), 7.70 (1H, d, <i>J</i> = 9.4), 7.98 (1H, dd, <i>J</i> = 7.0, 4.4). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 17.9, 19.2, 24.6, 24.8, 25.6, 26.3, 26.5, 27.0, 27.2, 27.7, 29.3, 34.7, 36.3, 38.2, 40.3, 53.5, 58.8, 69.9, 116.8 (d, <i>J</i><sub>CCF</sub> = 17), 117.3 (dd, <i>J</i><sub>CCF</sub> = 135.6, <i>J</i><sub>CCCF</sub> = 16.4), 125.7 (dd, <i>J</i><sub>CCCF</sub> = 6, <i>J</i><sub>CCCCF</sub> = 3.2), 137.4 (dd, <i>J</i><sub>CCCF</sub> = 5.8, <i>J</i><sub>CCCCF</sub> = 3.2), 147.9 (dd, <i>J</i><sub>CF</sub> = 243, <i>J</i><sub>CCF</sub> = 12.6), 148.9 (dd, <i>J</i><sub>CF</sub> = 244.3, <i>J</i><sub>CCF</sub> = 12.6), 170.3, 170.7, 172.6. ESI MS: 546 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>43</sub>O<sub>4</sub>N<sub>3</sub>F<sub>2</sub>Na, 546.31138; found, 546.31104.</div></div><div id="sec4_2_2_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-(3,4-Dichlorobenzyl)-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>16</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.83 (6H, d, <i>J</i> = 6.8), 0.84–1.49 (18H, m), 1.95–2.06 (1H, m), 2.08–2.20 (2H, m), 2.20 (1H, dd, <i>J</i> = 14.0, 5.5), 2.33 (1H, dd, <i>J</i> = 14.0, 8.6), 2.61–2.68 (1H, m), 2.69–2.79 (2H, m), 3.25–3.34 (1H, m), 3.79 (1H, dddd, <i>J</i> = 9.7, 9.7, 4.6, 1.2), 3.92 (1H, dd, <i>J</i> = 7.8, 5.8), 3.99–4.04 (1H, m), 5.12 (1H, br s), 7.20 (1H, dd, <i>J</i> = 8.3, 1.9), 7.46 (1H, d, <i>J</i> = 8.3), 7.47 (1H, d, <i>J</i> = 1.9), 7.61 (1H, d, <i>J</i> = 7.9), 7.72 (1H, d, <i>J</i> = 9.4), 7.98 (1H, dd, <i>J</i> = 7.1, 4.5). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 17.8, 19.2, 24.5, 24.7, 25.6, 26.2, 26.4, 26.9, 27.0, 27.1, 27.6, 29.3, 34.7, 36.4, 38.2, 40.3, 53.4, 58.9, 70.0, 128.6, 129.5, 130.0, 130.5, 131.2, 140.9, 170.3, 170.6, 172.6. ESI MS: 578 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>43</sub>O<sub>4</sub>N<sub>3</sub>Cl<sub>2</sub>Na, 578.25228; found, 578.25196.</div></div><div id="sec4_2_2_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-(4-Fluorobenzyl)-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>17</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.84 (6H, d, <i>J</i> = 6.8), 0.94–1.47 (18H, m), 1.95–2.03 (1H, m), 2.03–2.10 (1H, m), 2.10–2.17 (1H, m), 2.22 (1H, dd, <i>J</i> = 14.0, 6.4), 2.29 (1H, dd, <i>J</i> = 14.0, 7.5), 2.65–2.72 (2H, m), 2.77 (1H, dd, <i>J</i> = 13.7, 4.0), 3.25–3.34 (1H, m), 3.82 (1H, dddd, <i>J</i> = 11.0, 10.9, 3.8, 1.7), 3.94 (1H, dd, <i>J</i> = 8.0, 6.2), 3.96–4.01 (1H, m), 5.05 (1H, br s), 7.00–7.06 (2H, m), 7.20–7.25 (2H, m), 7.60 (1H, d, <i>J</i> = 8.1), 7.67 (1H, d, <i>J</i> = 9.2), 7.97 (1H, dd, <i>J</i> = 7.0, 4.4). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 17.9, 19.2, 24.8, 25.0, 26.0, 26.6, 26.9, 27.1, 27.2, 27.4, 27.8, 29.4, 34.7, 36.0, 38.1, 40.1, 53.8, 58.7, 69.5, 114.6 (d, <i>J</i><sub>CCF</sub> = 20.9), 130.7 (d, <i>J</i><sub>CCCF</sub> = 8), 135.7 (d, <i>J</i><sub>CCCCF</sub> = 3), 160.7 (d, <i>J</i><sub>CF</sub> = 240.9), 170.4, 170.7, 172.5. ESI MS: 528 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>44</sub>O<sub>4</sub>N<sub>3</sub>FNa, 528.32081; found, 528.32039.</div></div><div id="sec4_2_2_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-(Cyclohexylmethyl)-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>18</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.84 (3H, d, <i>J</i> = 6.8), 0.85 (3H, d, <i>J</i> = 6.8), 1.06–1.73 (31H, m), 1.97–2.05 (1H, m), 2.15 (1H, dd, <i>J</i> = 14.2, 6.3), 2.19 (2H, dd, <i>J</i> = 6.3, 6.1), 2.28 (1H, dd, <i>J</i> = 14.2, 7.8), 2.71–2.79 (1H, m), 3.28–3.37 (1H, m), 3.74–3.81 (1H, m), 3.93 (1H, dd, <i>J</i> = 8.0, 5.9), 3.82–3.87 (1H, m), 4.80 (1H, d, <i>J</i> = 5.5), 7.45 (1H, d, <i>J</i> = 9.2), 8.02 (1H, dd, <i>J</i> = 7.1, 4.4). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 17.9, 19.2, 24.9, 25.0, 25.8, 26.0, 26.1, 26.6, 27.0, 27.1, 27.2, 27.4, 27.9, 29.4, 31.9, 33.6, 33.9, 34.8, 38.2, 48.6, 58.7, 69.8, 170.6, 170.7, 172.5. ESI MS: 516 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>52</sub>O<sub>4</sub>N<sub>3</sub>, 494.39523; found, 494.39501.</div></div><div id="sec4_2_2_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> (3<i>S</i>,7<i>S</i>,8<i>R</i>)-8-((Benzylthio)methyl)-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>19</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.85 (3H, d, <i>J</i> = 6.7), 0.86 (3H, d, <i>J</i> = 6.7), 1.15–1.62 (18H, m), 1.94–2.05 (1H, m), 2.13–2.24 (4H, m), 2.48–2.55 (1H, m), 2.59 (1H, dd, <i>J</i> = 13.4, 5.9), 2.78–2.85 (1H, m), 3.27–3.37 (1H, m), 3.69 (2H, s), 3.93–3.97 (1H, m), 3.98 (1H, dd, <i>J</i> = 8.1, 6.6), 4.06–4.11 (1H, m), 4.99 (1H, br s), 7.21–7.26 (1H, m), 7.28–7.31 (4H, m), 7.63 (1H, d, <i>J</i> = 9.2), 7.66 (1H, d, <i>J</i> = 8.2), 7.92 (1H, dd, <i>J</i> = 7.0, 4.5). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.0, 19.2, 24.9, 25.2, 26.6, 27.2, 27.4, 27.5, 27.9, 28.1, 29.5, 31.9, 34.7, 34.8, 38.1, 39.8, 51.0, 58.6, 68.0, 126.7, 128.3, 128.8, 138.5, 170.4, 170.8, 172.7. ESI MS: 556 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>29</sub>H<sub>47</sub>O<sub>4</sub>N<sub>3</sub>NaS, 556.31795; found, 556.31774.</div></div><div id="sec4_2_2_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-7-Hydroxy-8-(4-hydroxybenzyl)-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>20</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.84 (6H, d, <i>J</i> = 6.9), 1.02–1.46 (18H, m), 1.94–2.03 (1H, m), 2.03–2.10 (1H, m), 2.10–2.17 (1H, m), 2.21 (1H, dd, <i>J</i> = 14.1, 6.8), 2.26 (1H, dd, <i>J</i> = 14.1, 6.9), 2.57 (1H, dd, <i>J</i> = 13.9, 10.2), 2.69–2.77 (2H, m), 3.26–3.34 (1H, m), 3.81 (1H, dddd, <i>J</i> = 9.7, 9.5, 4.7, 1.6), 3.92–3.98 (3H, m), 6.59–6.63 (2H, m), 6.95–7.00 (2H, m), 7.58 (1H, br s), 7.60 (1H, d, <i>J</i> = 1.8), 7.96 (1H, dd, <i>J</i> = 7.0, 4.4), 9.05 (1H, br s). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.0, 19.2, 24.8, 25.1, 26.2, 26.9, 27.2, 27.3, 27.4, 27.6, 28.0, 29.5, 34.9, 35.7, 38.1, 40.1, 53.9, 58.7, 69.2, 114.7, 129.5, 129.7, 155.4, 170.5, 170.8, 172.4. ESI MS: 526 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>45</sub>O<sub>5</sub>N<sub>3</sub>Na, 526.32514; found, 526.32546.</div></div><div id="sec4_2_2_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-(4-(Benzyloxy)benzyl)-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>21</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.84 (6H, d, <i>J</i> = 6.8), 0.98–1.48 (18H, m), 1.94–2.02 (1H, m), 2.04–2.17 (2H, m), 2.21 (1H, dd, <i>J</i> = 14.1, 6.6), 2.27 (1H, dd, <i>J</i> = 14.1, 7.2), 2.63 (1H, dd, <i>J</i> = 13.7, 10.3), 2.64–2.75 (2H, m), 3.25–3.32 (1H, m), 3.78–3.84 (1H, m), 3.97 (1H, dd, <i>J</i> = 8.1, 6.2), 3.94–4.00 (1H, m), 5.01 (1H, s), 6.85–6.88 (2H, m), 7.09–7.14 (2H, m), 7.30–7.34 (1H, m), 7.36–7.44 (4H, m), 7.58 (1H, d, <i>J</i> = 8.2), 7.63 (1H, d, <i>J</i> = 9.2), 7.97 (1H, dd, <i>J</i> = 7.0, 4.4). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.0, 19.2, 24.8, 25.0, 26.1, 26.4, 26.7, 27.0, 27.2, 27.3, 27.5, 27.9, 29.4, 34.8, 35.9, 38.1, 53.9, 58.7, 69.1, 69.4, 114.2, 127.5, 127.7, 128.4, 129.9, 131.7, 137.2, 156.6, 170.4, 170.7, 172.5. ESI MS: 616 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>35</sub>H<sub>51</sub>O<sub>5</sub>N<sub>3</sub>Na, 616.37209; found, 616.37168.</div></div><div id="sec4_2_2_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-((1<i>H</i>-Indol-3-yl)methyl)-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>22</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.83 (6H, d, <i>J</i> = 6.8), 1.08–1.55 (18H, m), 1.92–2.01 (1H, m), 2.06 (1H, ddd, <i>J</i> = 13.8, 7.2, 4.7), 2.16 (1H, ddd, <i>J</i> = 13.8, 8.7, 4.8), 2.23 (1H, dd, <i>J</i> = 14.2, 6.2), 2.28 (1H, dd, <i>J</i> = 14.2, 7.4), 2.65–2.72 (1H, m), 2.75 (1H, dd, <i>J</i> = 14.8, 8.9), 2.96 (1H, dd, <i>J</i> = 14.8, 5.4), 3.22–3.30 (1H, m), 3.95 (1H, dd, <i>J</i> = 8.4, 6.7), 4.00–4.05 (1H, m), 5.01 (1H, br s), 6.94 (1H, ddd, <i>J</i> = 9.6, 8.7, 0.9), 7.03 (1H, ddd, <i>J</i> = 9.0, 8.0, 1.0), 7.08 (1H, d, <i>J</i> = 2.2), 7.25–7.35 (1H, m), 7.54 (1H, d, <i>J</i> = 8.0), 7.59 (1H, d, <i>J</i> = 8.6), 7.62 (1H, d, <i>J</i> = 9.0), 7.88 (1H, dd, <i>J</i> = 7.0, 4.6). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.1, 19.2, 24.9, 25.2, 26.3, 26.6, 27.1, 27.4, 27.6, 27.9, 28.0, 29.5, 34.9, 38.0, 39.7, 52.9, 58.6, 68.7, 111.2, 111.7, 118.0, 118.3, 119.8, 120.7, 122.0, 122.9, 127.5, 128.9, 136.2, 170.6, 170.8, 172.4. ESI MS: 549 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>30</sub>H<sub>46</sub>O<sub>4</sub>N<sub>4</sub>Na, 549.34113; found, 549.34082.</div></div><div id="sec4_2_2_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> (3<i>S</i>,7<i>S</i>,8<i>R</i>)-7-Hydroxy-3-isopropyl-8-(mercaptomethyl)-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>23</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz DMSO-<i>d</i><sub>6</sub>): 0.84 (3H, d, <i>J</i> = 7.0), 0.85 (3H, d, <i>J</i> = 7.0), 1.15–1.62 (18H, m), 1.92–2.03 (1H, m), 2.10–2.25 (5H, m), 2.48–2.55 (1H, m), 2.67 (1H, ddd, <i>J</i> = 13.7, 9.1, 5.5), 2.78–2.86 (1H, m), 3.27–3.34 (1H, m), 3.72 (1H, dddd, <i>J</i> = 8.7, 8.6, 5.7, 2.1), 3.98 (1H, dd, <i>J</i> = 7.6, 7.4), 4.07–4.13 (1H, m), 4.99 (1H, br s), 7.60 (1H, d, <i>J</i> = 9.1), 7.62 (1H, d, <i>J</i> = 8.7), 7.88 (1H, dd, <i>J</i> = 6.6, 4.7). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.2, 19.2, 24.7, 25.0, 25.3, 26.8, 27.3, 27.5, 27.6, 27.7, 28.0, 28.2, 29.6, 34.9, 38.1, 39.6, 55.5, 58.5, 67.9, 170.3, 170.8, 172.8. ESI MS: 442 ([M – H]<sup>−</sup>). HR ESI MS: calcd for C<sub>22</sub>H<sub>40</sub>O<sub>4</sub>N<sub>3</sub>S, 442.27450; found, 442.27416.</div></div><div id="sec4_2_2_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-7-Hydroxy-3-isopropyl-8-(thiazol-4-ylmethyl)-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>24</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.84 (3H, d, <i>J</i> = 6.8), 0.85 (3H, d, <i>J</i> = 6.8), 1.03–1.48 (18H, m), 1.96–2.04 (1H, m), 2.04–2.10 (1H, m), 2.10–2.17 (1H, m), 2.23 (1H, dd, <i>J</i> = 14.2, 6.8), 2.29 (1H, dd, <i>J</i> = 14.2, 7.0), 2.73–2.81 (1H, m), 2.92 (1H, dd, <i>J</i> = 14.8, 10.0), 2.99 (1H, dd, <i>J</i> = 14.8, 4.4), 3.27–3.38 (1H, m), 4.09 (1H, dddd, <i>J</i> = 10.0, 9.8, 4.4, 2.0), 3.96 (1H, dd, <i>J</i> = 8.2, 6.2), 3.99 (1H, ddd, <i>J</i> = 6.9, 6.8, 2.0), 5.05 (1H, br s), 7.25 (1H, d, <i>J</i> = 2.0), 7.62 (1H, d, <i>J</i> = 8.2), 7.65 (1H, d, <i>J</i> = 9.5), 7.96 (1H, dd, <i>J</i> = 7.3, 4.4), 8.97 (1H, d, <i>J</i> = 2.0). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.0, 19.2, 24.7, 25.1, 26.2, 26.8, 27.2, 27.3, 27.5, 28.0, 29.5, 32.5, 34.8, 38.1, 39.8, 51.7, 58.7, 69.3, 114.3, 153.1, 154.9, 170.4, 170.7, 172.5. ESI MS: 517 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>25</sub>H<sub>42</sub>O<sub>4</sub>N<sub>4</sub>NaS, 517.28190; found, 517.28161.</div></div><div id="sec4_2_2_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-(pyridin-3-ylmethyl)-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>25</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 0.98–1.07 (2H, m), 1.12–1.32 (14H, m), 1.35–1.53 (4H, m), 2.10–2.22 (2H, m), 2.26–2.33 (2H, m), 2.75–2.89 (3H, m), 3.06–3.14 (1H, m), 3.34–3.41 (2H, m), 3.94–3.99 (1H, m), 4.05 (1H, td, <i>J</i> = 7.1, 1.8), 4.64 (1H, dd, <i>J</i> = 9.3, 5.4), 7.11–7.16 (1H, m), 7.17–7.24 (4H, m), 7.65 (1H, d, <i>J</i> = 9.3), 7.92 (1H, dd, <i>J</i> = 8.1, 5.6), 8.33–8.37 (1H, m), 8.42–8.47 (1H, m), 8.69 (1H, d, <i>J</i> = 5.6), 8.72 (1H, s). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 26.6, 27.9, 28.5, 28.6, 28.8, 29.0, 29.1, 29.2, 35.4, 36.7, 39.0, 40.7, 40.8, 41.7, 55.6, 55.8, 71.5, 127.6, 128.1, 129.6, 130.4, 140.0, 140.5, 142.0, 144.2, 147.9, 172.5, 173.8, 176.2. ESI MS: 559 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>31</sub>H<sub>45</sub>O<sub>4</sub>N<sub>4</sub>Na, 559.32548; found, 559.32554.</div></div><div id="sec4_2_2_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-3-((<i>R</i>)-<i>s</i>-butyl)-7-hydroxy-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>26</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.76–0.84 (6H, m), 0.99–1.47 (20H, m), 1.66–1.76 (1H, m), 1.99–2.17 (2H, m), 2.18–2.28 (2H, m), 2.62–2.73 (2H, m), 2.81 (1H, dd, <i>J</i> = 13.8, 4.3), 3.25–3.31 (1H, m), 3.83–3.93 (1H, m), 3.93–4.02 (2H, m), 5.02 (1H, d, <i>J</i> = 5.3), 7.10–7.26 (5H, m), 7.60 (1H, d, <i>J</i> = 8.2), 7.65 (1H, d, <i>J</i> = 9.1), 7.95 (1H, dd, <i>J</i> = 7.3, 4.3). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 11.4, 15.5, 24.4, 24.8, 25.1, 26.3, 26.8, 27.2, 27.3, 27.4, 27.7, 28.0, 34.8, 35.9, 36.5, 38.1, 40.0, 53.8, 57.7, 69.4, 125.8, 128.0, 128.9, 139.6, 170.4, 170.8, 172.4. ESI MS: 524 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>29</sub>H<sub>47</sub>O<sub>4</sub>N<sub>3</sub>Na, 524.34588; found, 524.34528.</div></div><div id="sec4_2_2_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-3-cyclohexyl-7-hydroxy-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>27</b>)</h5><div class="NLM_p last">NMR spectra for this compound were recorded at 80 °C, due to limited solubility of the compound. <sup>1</sup>H NMR (600 MHz, acetic acid-<i>d</i><sub>4</sub>): 1.00–1.87 (31H, m), 2.13–2.19 (1H, m), 2.22–2.33 (1H, m), 2.50–2.63 (1H, m), 2.85–3.03 (3H, m), 3.44–3.63 (1H, m), 4.12–4.40 (2H, m), 7.16–7.36 (5H, m). <sup>13</sup>C NMR (150 MHz, acetic acid-<i>d</i><sub>4</sub>): 11.4, 15.5, 24.4, 26.5, 26.8, 26.9 (2C), 27.1, 28.6, 28.8, 29.1, 29.2, 29.4, 29.5, 29.7, 30.6, 36.8, 38.6, 40.4, 42.1, 56.1, 60.4, 71.4, 127.6, 129.6, 130.3, 139.6, 171.5, 173.8, 173.9. ESI MS: 550 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>31</sub>H<sub>49</sub>O<sub>4</sub>N<sub>3</sub>Na, 550.36153; found, 550.36104.</div></div><div id="sec4_2_2_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-3,8-Dibenzyl-7-hydroxy-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>28</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.91–1.05 (2H, m), 1.01–1.50 (16H, m), 2.04–2.14 (3H, m), 2.19 (1H, dd, <i>J</i> = 14.4, 7.7), 2.64–2.80 (4H, m), 3.05 (1H, dd, <i>J</i> = 14.0, 4.6), 3.18–3.28 (1H, m), 3.83–3.97 (2H, m), 4.28–4.36 (1H, m), 4.97 (1H, d, <i>J</i> = 5.1), 7.08–7.30 (10H, m), 7.65 (1H, d, <i>J</i> = 9.2), 7.96 (1H, d, <i>J</i> = 8.2), 7.98–8.06 (1H, m). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 24.8, 24.9, 26.1, 26.6, 26.7, 27.0, 27.1, 27.3, 27.7, 34.9, 36.9, 37.0, 38.4, 40.0, 53.3, 54.6, 69.2, 125.8, 126.2, 128.1, 128.9, 129.0, 138.4, 139.5, 170.3, 170.8, 172.5. ESI MS: 558 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>32</sub>H<sub>45</sub>O<sub>4</sub>N<sub>3</sub>Na, 558.33023; found, 558.33006.</div></div><div id="sec4_2_2_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> <i>N</i>-(4-((3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-2,5,10-trioxo-1,4,9-triazacyclohenicosan-3-yl)butyl)acetamide (<b>29</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): 0.92–1.02 (2H, m), 1.04–1.50 (20H, m), 1.58–1.69 (1H, m), 1.77 (3H, s), 2.05–2.14 (2H, m), 2.17–2.26 (2H, m), 2.36 (1H, p, <i>J</i> = 1.8), 2.63 (1H, p, <i>J</i> = 1.8), 2.66–2.80 (3H, m), 2.97 (2H, q, <i>J</i> = 6.6), 3.83–3.90 (1H, m), 3.95–4.02 (2H, m), 5.07 (1H, d, <i>J</i> = 5.1), 7.11–7.24 (5H, m), 7.68 (1H, d, <i>J</i> = 9.3), 7.80 (1H, t, <i>J</i> = 5.6), 7.85 (1H, d, <i>J</i> = 7.5), 7.99 (1H, t, <i>J</i> = 5.9). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): 22.7, 23.1, 24.9, 26.0, 26.6, 26.7, 27.0, 27.1, 27.4, 27.7, 28.9, 31.1, 34.8, 36.9, 38.3, 38.4, 40.0, 53.4, 53.5, 69.5, 125.8, 128.0, 129.0, 139.6, 170.0, 170.5, 171.6, 172.4. ESI MS: 581 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>31</sub>H<sub>50</sub>O<sub>5</sub>N<sub>4</sub>Na, 581.36734; found, 581.36685.</div></div><div id="sec4_2_2_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-phenyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>30</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): 1.03–1.46 (16H, m), 1.99–2.08 (1H, m), 2.10–2.18 (1H, m), 2.29 (2H, dd, <i>J</i> = 6.9, 2.8), 2.36 (1H, p, <i>J</i> = 1.8), 2.63 (1H, p, <i>J</i> = 1.9), 2.68 (1H, dd, <i>J</i> = 13.7, 10.1), 2.71–2.78 (2H, m), 2.81 (1H, dd, <i>J</i> = 13.8, 4.4), 3.88–3.94 (1H, m), 3.95–4.00 (1H, m), 5.18 (1H, br s), 5.22 (1H, d, <i>J</i> = 7.2), 7.12–7.41 (10H, m), 7.72 (1H, d, <i>J</i> = 9.1), 8.37 (1H, d, <i>J</i> = 7.2), 8.41 (1H, dd, <i>J</i> = 7.9, 4.6). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): 24.3, 24.8, 25.1, 26.9, 27.3, 27.4, 27.6, 27.9, 28.0, 34.7, 36.5, 38.3, 40.0, 53.8, 57.8, 69.3, 125.8, 127.6, 128.0, 128.3, 129.0, 138.3, 139.7, 169.9, 170.3, 172.4. ESI MS: 522 ([M + H]<sup>+</sup>). HR ESI MS: calcd for C<sub>31</sub>H<sub>44</sub>O<sub>4</sub>N<sub>3</sub>, 522.33263; found, 522.33248.</div></div><div id="sec4_2_2_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-((<i>R</i>)-1-hydroxyethyl)-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>31</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): 1.00 (3H, d, <i>J</i> = 6.4), 1.04–1.47 (16H, m), 2.10 (2H, t, <i>J</i> = 6.5), 2.23 (1H, dd, <i>J</i> = 14.1, 6.0), 2.30 (1H, dd, <i>J</i> = 14.2, 8.1), 2.34 (1H, p, <i>J</i> = 1.9), 2.62 (1H, p, <i>J</i> = 1.9), 2.67–2.85 (3H, m), 3.07–3.17 (1H, m), 3.82–3.88 (1H, m), 3.94–4.01 (2H, m), 4.02–4.04 (1H, m), 4.76 (1H, d, <i>J</i> = 5.6), 5.06 (1H, d, <i>J</i> = 5.2), 7.08–7.23 (5H, m), 7.48 (1H, d, <i>J</i> = 8.5), 7.69 (1H, d, <i>J</i> = 9.2), 7.88 (1H, t, <i>J</i> = 5.8). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): 20.6, 24.8, 25.0, 25.9, 26.4, 26.5, 26.9, 27.2 (2C), 27.4, 34.9, 37.2, 38.6, 39.9, 53.4, 59.0, 65.8, 69.7, 125.9, 128.0, 129.0, 139.5, 170.3, 170.8, 172.6. ESI MS: 512 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>27</sub>H<sub>43</sub>O<sub>5</sub>N<sub>3</sub>Na, 512.30949; found, 512.30878.</div></div><div id="sec4_2_2_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-3-((<i>R</i>)-1-(benzyloxy)ethyl)-7-hydroxy-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>32</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.88–1.00 (3H, m), 1.06–1.48 (19H, m), 1.40–1.54 (1H, m), 2.13 (2H, t, <i>J</i> = 6.4), 2.24 (1H, dd, <i>J</i> = 14.1, 8.7), 2.38 (1H, dd, <i>J</i> = 14.1, 8.7), 2.64–2.80 (2H, m), 3.19–3.32 (1H, m), 3.82–3.90 (1H, m), 3.99–4.05 (1H, m), 4.09 (1H, qd, <i>J</i> = 6.3, 2.6), 4.16 (1H, dd, <i>J</i> = 8.7, 2.6), 4.39 (1H, d, <i>J</i> = 11.8), 4.50 (1H, d, <i>J</i> = 11.9), 7.09–7.35 (10H, m), 7.56 (1H, d, <i>J</i> = 8.7), 7.73 (1H, d, <i>J</i> = 9.3), 8.13 (1H, t, <i>J</i> = 5.8). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 16.8, 24.6, 24.7, 25.5, 26.0, 26.4, 26.8, 27.0, 27.1, 27.3, 34.9, 37.5, 40.0, 40.6, 53.3, 58.0, 70.1, 70.7, 74.4, 125.8, 127.3, 127.9 (2C), 129.0, 138.6, 139.5, 170.0, 170.9, 172.8. ESI MS: 602 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>34</sub>H<sub>49</sub>O<sub>5</sub>N<sub>3</sub>Na, 602.35644; found, 602.35600.</div></div><div id="sec4_2_2_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> (4<i>S</i>,5<i>S</i>,9<i>S</i>)-4-Benzyl-5-hydroxy-9-isopropyl-1-oxa-3,8,11-triazacyclodocosane-2,7,10-trione (<b>33</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.83 (3H, d, <i>J</i> = 6.5), 0.85 (3H, d, <i>J</i> = 6.5), 1.09–1.53 (16H, m), 1.85–1.99 (1H, m), 2.19–2.30 (2H, m), 2.57 (1H, dd, <i>J</i> = 13.8, 10.6), 2.72–2.90 (3H, m), 3.23–3.26 (2H, m), 3.59–3.76 (2H, m), 3.94–4.01 (2H, m), 4.10 (1H, dt, <i>J</i> = 11.4, 5.9), 6.92 (1H, d, <i>J</i> = 9.1), 7.11–7.29 (5H, m), 7.70 (1H, d, <i>J</i> = 8.5), 7.75 (1H, dd, <i>J</i> = 7.0, 4.5). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.3, 19.3, 25.0, 25.5, 27.3, 27.6, 27.9, 28.0, 28.2, 28.4 (2C), 29.6, 35.1, 38.1, 40.0, 56.5, 58.6, 63.3, 68.9, 125.8, 128.0, 129.0, 139.8, 156.2, 170.6, 171.0. ESI MS: 526 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>45</sub>O<sub>5</sub>N<sub>3</sub>Na, 526.32514; found, 526.32488.</div></div><div id="sec4_2_2_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> (4<i>S</i>,20<i>S</i>,21<i>S</i>)-20-Benzyl-21-hydroxy-4-isopropyl-1,3,6,19-tetraazacyclodocosane-2,5,18-trione (<b>34</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.73–0.87 (6H, m), 1.03–1.59 (19H, m), 1.73–1.93 (2H, m), 2.10–2.20 (1H, m), 2.61–2.84 (4H, m), 3.09–3.16 (1H, m), 3.56–3.62 (1H, m), 3.78 (1H, t, <i>J</i> = 8.1), 3.93 (1H, q, <i>J</i> = 8.3), 5.17 (1H, d, <i>J</i> = 5.8), 6.00–6.04 (1H, m), 6.19 (1H, d, <i>J</i> = 8.9), 7.10–7.35 (5H, m), 7.56–7.74 (2H, m). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.3, 19.5, 25.0, 26.0, 27.8 (2C), 28.0, 28.2, 28.5, 28.8, 28.9, 29.1, 30.0, 35.2, 37.3, 37.9, 40.0, 43.5, 52.9, 59.0, 70.7, 125.9, 128.1, 129.2, 139.4, 158.0, 172.0, 172.2. ESI MS: 525 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>46</sub>O<sub>4</sub>N<sub>4</sub>Na, 525.34113; found, 525.34071.</div></div><div id="sec4_2_2_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> (12<i>S</i>,16<i>S</i>,17<i>S</i>)-17-Benzyl-16-hydroxy-12-isopropyl-1,4,7-trioxa-10,13,18-triazacyclohenicosane-11,14,19-trione (<b>35</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.82 (6H, d, <i>J</i> = 6.9), 2.04–2.14 (1H, m), 2.14–2.46 (5H, m), 2.63–2.74 (1H, m), 2.84 (1H, dd, <i>J</i> = 13.9, 4.9), 2.91–3.02 (1H, m), 3.36–3.48 (10H, m), 3.56–3.63 (3H, m), 3.93–4.01 (2H, m), 4.05 (1H, dd, <i>J</i> = 8.6, 5.7), 7.12–7.18 (1H, m), 7.18–7.28 (4H, m), 7.62 (1H, d, <i>J</i> = 8.7), 7.76 (1H, d, <i>J</i> = 9.2), 7.91 (1H, t, <i>J</i> = 5.3). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 17.6, 19.3, 29.4, 36.2, 36.4, 39.9, 53.3, 58.0, 66.9, 68.4, 68.9, 69.6, 69.7, 70.1, 70.4, 125.8, 128.0, 128.9, 139.5, 170.6, 170.7, 171.1. ESI MS: 516 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>25</sub>H<sub>39</sub>O<sub>7</sub>N<sub>3</sub>Na, 516.26802; found, 516.26782.</div></div><div id="sec4_2_2_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> (3<i>S</i>,8<i>R</i>)-8-Benzyl-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>36</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 0.71–0.88 (6H, m), 0.95–1.53 (18H, m), 1.79–1.91 (1H, m), 1.92–2.04 (2H, m), 2.05–2.20 (3H, m), 2.57–2.77 (3H, m), 2.89–3.09 (1H, m), 3.78–3.90 (2H, m), 7.60 (1H, d, <i>J</i> = 7.9), 7.09–7.28 (5H, m), 7.67 (1H, dd, <i>J</i> = 8.5, 5.6), 7.91–7.97 (1H, m). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 18.0, 19.3, 24.8, 25.1, 26.4, 26.8, 27.1, 27.3, 27.4, 27.8, 28.0, 35.0, 38.0, 39.8, 49.2, 57.8, 58.8, 125.9, 128.0, 129.0, 139.1, 170.8, 172.0, 172.2. ESI MS: 494 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>28</sub>H<sub>45</sub>O<sub>3</sub>N<sub>3</sub>Na, 494.33531; found, 494.33495.</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Precursors of Macrocyclic Inhibitors</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Preparation of Statine Analogues</h4><div id="sec4_3_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> <i>t</i>-Butyl (<i>S</i>)-(1-(2,2-Dimethyl-4,6-dioxo-1,3-dioxan-5-yl)-3-(naphthalen-2-yl)-1-oxopropan-2-yl)carbamate (<b>38a</b>)</h5><div class="NLM_p last">A solution of Boc-3-(2-naphthyl)-<span class="smallcaps smallerCapital">l</span>-alanine (<b>37a</b>) (0.32 g, 1.00 mmol), Meldrum’s acid (0.15 g, 1.05 mmol, 1.05 equiv), and DMAP (0.24 g, 2.00 mmol, 2.0 equiv) was prepared in anhydrous dichloromethane (DCM) (5 mL), and dicyclohexylcarbodiimide (DCC, 0.22 g, 1.05 mmol, 1.05 equiv) was added. The reaction mixture was stirred overnight and filtered, and the solid was washed with DCM (2 × 10 mL). The organic phase was washed with 10% KHSO<sub>4</sub> (2 × 20 mL), water (2 × 20 mL), and brine (20 mL). The solvent was evaporated to yield crude <b>38a</b>, which was used in the next step without further purification.</div></div><div id="sec4_3_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> <i>t</i>-Butyl (<i>S</i>)-3-Hydroxy-2-(naphthalen-2-ylmethyl)-5-oxo-2,5-dihydro-1<i>H</i>-pyrrole-1-carboxylate (<b>39a</b>)</h5><div class="NLM_p last">The crude product <b>38a</b> (0.44 g, 1.00 mmol) was dissolved in EtOAc (10 mL), and the solution was heated to reflux for 30 min. After cooling, the product was extracted into the aqueous phase using 5% NaHCO<sub>3</sub> (3 × 10 mL). The aqueous phase was then acidified using KHSO<sub>4</sub> to pH 2. The product was then extracted using EtOAc (3 × 15 mL). The extract was evaporated to dryness to yield crude <b>39a</b>.</div></div><div id="sec4_3_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> <i>t</i>-Butyl (2<i>S</i>,3<i>S</i>)-3-Hydroxy-2-(naphthalen-2-ylmethyl)-5-oxopyrrolidine-1-carboxylate (<b>40a</b>)</h5><div class="NLM_p last">A solution of crude <b>39a</b> (0.34 g, 1.00 mmol) in a mixture of DCM (10 mL) and AcOH (1 mL) was cooled (0 °C), and NaBH<sub>4</sub> (0.08 g, 2.00 mmol, 2.0 equiv) was added portionwise. The mixture was stirred overnight at RT. It was then poured into cold water, and the organic layer was washed with water, dried with MgSO<sub>4</sub>, and evaporated to dryness. Chromatography of the residue on silica gel (cyclohexane/EtOAc 2:1) afforded purified <b>40a</b> (0.20 g, 60% yield over 3 steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 1.46 (9H, s), 2.10 (1H, br s), 2.40–2.50 (1H, m), 2.55–2.63 (1H, m), 3.29 (2H, d, <i>J</i> = 6.0), 4.43–4.57 (2H, m), 7.40–7.50 (3H, m), 7.73–7.83 (4H, m). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 27.9, 34.2, 40.1, 62.6, 65.6, 83.4, 125.7, 126.2, 127.5, 127.6, 127.9, 128.2 (2C), 132.2, 133.5, 135.3, 149.8, 171.3.</div></div><div id="sec4_3_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> (3<i>S</i>,4<i>S</i>)-4-<i>t</i>-Butoxycarbonylamino-3-hydroxy-5-(2-napthyl)pentanoic Acid (<b>41a</b>)</h5><div class="NLM_p last">A solution of <b>40a</b> (0.20 g, 0.60 mmol) in acetone (5 mL) was treated dropwise with NaOH (1 M, 1.20 mL, 1.20 mmol, 2.0 equiv). After stirring for 30 min at RT, the solution was acidified to pH 3 with 1 M HCl and extracted with EtOAc (3 × 10 mL) to afford <b>41a</b> (0.20 g, 92% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 1.26 (9H, s), 2.23 (1H, dd, <i>J</i> = 15.5, 9.2), 2.41 (1H, dd, <i>J</i> = 15.5, 3.6), 2.76 (1H, dd, <i>J</i> = 13.6, 9.4), 3.00 (1H, dd, <i>J</i> = 13.6, 5.1), 3.74–3.84 (1H, m), 3.90–3.98 (1H, m), 6.56 (1H, d, <i>J</i> = 9.3), 7.38–7.51 (3H, m), 7.71 (1H, s), 7.79–7.88 (3H, m). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 28.1, 36.1, 38.6, 55.7, 68.4, 77.6, 125.2, 125.8, 127.1, 127.2, 127.3, 127.4, 128.6, 131.9, 133.5, 135.3, 155.4, 173.1. EI MS: 359 ([M]<sup>+</sup>). HR EI MS: calcd for C<sub>20</sub>H<sub>25</sub>O<sub>5</sub>N, 359.1732; found, 359.1725. mp 180–181 °C. [α]<sub class="stack">D</sub><sup class="stack">20</sup> (<i>c</i> 2.0, EtOH): −26.0°. EA: calcd for C, 66.83; H, 7.01; N, 3.90. Found C, 66.68; H, 7.08; N, 3.89.</div></div><div id="sec4_3_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> (3<i>S</i>,4<i>S</i>)-4-((<i>t</i>-Butoxycarbonyl)amino)-3-hydroxy-5-(thiazol-4-yl)pentanoic Acid (<b>41b</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 1.32 (9H, s), 2.16 (1H, dd, <i>J</i> = 15.5, 9.3), 2.30 (1H, dd, <i>J</i> = 15.5, 3.2), 2.80 (1H, dd, <i>J</i> = 14.5, 9.1), 3.00 (1H, dd, <i>J</i> = 14.5, 4.8), 3.82–3.90 (2H, m), 6.42 (1H, d, <i>J</i> = 9.2), 7.28 (1H, d, <i>J</i> = 1.9), 9.00 (1H, d, <i>J</i> = 1.9). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 28.2, 32.1, 39.0, 53.8, 68.6, 77.6, 114.5, 153.1, 154.9, 155.3, 174.5. EI MS: 316 ([M]<sup>+</sup>). HR EI MS: calcd for C<sub>13</sub>H<sub>20</sub>O<sub>5</sub>N<sub>2</sub>S, 316.1093; found, 316.1081. mp 145–148 °C. [α]<sub class="stack">D</sub><sup class="stack">20</sup> (<i>c</i> 1.9, EtOH): −17.7°. EA: calcd for C, 49.35; H, 6.37; N, 8.85; S, 10.13. Found C, 48.96; H, 6.32; N, 8.48; S, 9.78.</div></div><div id="sec4_3_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> (3<i>S</i>,4<i>R</i>)-4-((<i>t</i>-Butoxycarbonyl)amino)-3-hydroxy-5-((4-methylbenzyl)thio)pentanoic Acid (<b>41c</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 1.41 (9H, s), 2.18 (1H, dd, <i>J</i> = 15.6, 8.9), 2.27 (3H, s), 2.31–2.41 (2H, m), 2.57 (1H, dd, <i>J</i> = 13.4, 6.0), 3.58–3.74 (3H, m), 3.99–4.07 (1H, m), 6.42 (1H, d, <i>J</i> = 9.3), 7.10 (2H, d, <i>J</i> = 7.7), 7.20 (1H, d, <i>J</i> = 8.0). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 20.7, 28.2, 31.9, 34.6, 38.6, 53.3, 67.7, 77.8, 128.8, 128.9, 135.4, 135.8, 155.6, 172.8. ESI MS: 392 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>18</sub>H<sub>27</sub>O<sub>5</sub>NNaS, 392.15021; found, 392.14989. mp 95–96 °C. [α]<sub class="stack">D</sub><sup class="stack">20</sup> (<i>c</i> 8.8, MeOH): −23.1°. EA: calcd for C, 58.51; H, 7.37; N, 3.79; S, 8.68. Found C, 58.58; H, 7.44; N, 3.78; S, 8.55.</div></div><div id="sec4_3_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> (3<i>S</i>,4<i>S</i>)-4-<i>t</i>-Butoxycarbonylamino-3-hydroxy-5-(3,4-dichlorophenyl)-pentanoic Acid (<b>41d</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 1.29 (9H, s), 2.19 (1H, dd, <i>J</i> = 15.5, 9.1), 2.39 (1H, dd, <i>J</i> = 15.5, 3.6), 2.56 (1H, dd, <i>J</i> = 13.4, 10.1), 2.82 (1H, dd, <i>J</i> = 13.4, 4.2), 3.56–3.72 (1H, m), 3.83–3.96 (1H, m), 5.00 (1H, br s), 6.54 (1H, d, <i>J</i> = 9.4), 7.21 (1H, dd, <i>J</i> = 8.2, 1.9), 7.45–7.55 (2H, m), 11.20 (1H, br s). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 28.0, 35.1, 38.4, 55.6, 68.7, 77.7, 128.5, 129.7, 130.1, 130.5, 131.1, 140.9, 155.5, 173.0. CI MS: 400 ([M + Na]<sup>+</sup>). HR EI MS: calcd for C<sub>16</sub>H<sub>21</sub>O<sub>5</sub>NCl<sub>2</sub>, 377.0795; found, 377.0796. mp 170–172 °C. [α]<sub class="stack">D</sub><sup class="stack">20</sup> (<i>c</i> 1.2, EtOH): −25.0°. EA: calcd for C, 50.81; H, 5.60; N, 3.70; Cl, 18.75. Found C, 50.55; H, 5.60; N, 3.58; Cl, 18.43.</div></div><div id="sec4_3_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> (3<i>S</i>,4<i>S</i>)-4-<i>t</i>-Butoxycarbonylamino-3-hydroxy-5-(3,4-difluorophenyl)pentanoic Acid (<b>41e</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 1.29 (9H, s), 2.18 (1H, dd, <i>J</i> = 15.6, 9.1), 2.38 (1H, dd, <i>J</i> = 15.6, 3.7), 2.57 (1H, dd, <i>J</i> = 13.5, 10.2), 2.80 (1H, dd, <i>J</i> = 13.5, 4.5), 3.54–3.73 (1H, m), 3.81–3.97 (1H, m), 4.90 (1H, br s), 6.50 (1H, d, <i>J</i> = 9.4), 6.99–7.10 (1H, m), 7.17–7.40 (2H, m), 11.87 (1H, br s). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 28.0, 35.2, 38.5, 55.7, 68.6, 77.6, 116.9 (d, <i>J</i><sub>CCCF</sub> = 16.3), 117.4 (dd, <i>J</i><sub>CCF</sub> = 51.3, <i>J</i><sub>CCCF</sub> = 16.6), 125.9 (dd, <i>J</i><sub>CCCF</sub> = 6.2, <i>J</i><sub>CCCCF</sub> = 3.1), 137.4 (dd, <i>J</i><sub>CCCF</sub> = 5.8, <i>J</i><sub>CCCCF</sub> = 3.5), 148.4 (dd, <i>J</i><sub>CF</sub> = 243.6, <i>J</i><sub>CCF</sub> = 52.5), 148.6 (dd, <i>J</i><sub>CF</sub> = 243.8, <i>J</i><sub>CCF</sub> = 52.5), 155.5, 173.0. CI MS: 368 ([M + Na]<sup>+</sup>). HR EI MS: calcd for C<sub>16</sub>H<sub>21</sub>O<sub>5</sub>NF<sub>2</sub>, 345.1315; found, 345.1387. mp 155–157 °C. [α]<sub class="stack">D</sub><sup class="stack">20</sup> (<i>c</i> 1.33, EtOH): −26.4°. EA: calcd for C, 55.65; H, 6.13; N, 4.06; F, 11.00. Found C, 56.09; H, 6.25; N, 4.00; F, 10.79.</div></div><div id="sec4_3_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> (3<i>S</i>,4<i>S</i>)-4-<i>t</i>-Butoxycarbonylamino-3-hydroxy-5-(4-fluorophenyl)pentanoic Acid (<b>41f</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): 1.29 (9H, s), 2.18 (1H, dd, <i>J</i> = 15.5, 9.2), 2.37 (1H, dd, <i>J</i> = 15.5, 3.6), 2.55 (1H, dd, <i>J</i> = 13.6, 9.7), 2.80 (1H, dd, <i>J</i> = 13.6, 4.8), 3.54–3.71 (1H, m), 3.88 (1H, dddd, <i>J</i> = 8.8, 8.7, 3.2, 3.1), 6.48 (1H, d, <i>J</i> = 9.4), 7.01–7.15 (2H, m), 7.18–7.28 (2H, m). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): 28.1, 35.1, 38.5, 55.8, 68.4, 77.6, 114.7 (d, <i>J</i><sub>CCCF</sub> = 20.9), 130.9 (d, <i>J</i><sub>CCCF</sub> = 7.8), 135.7 (d, <i>J</i><sub>CCCCF</sub> = 2.8), 155.5, 160.8 (d, <i>J</i><sub>CF</sub> = 241.4), 173.1. CI MS: 334 ([M + Li]<sup>+</sup>). HR EI MS: calcd for C<sub>16</sub>H<sub>22</sub>O<sub>5</sub>NF, 327.1482; found, 327.1468. mp 162–163 °C. [α]<sub class="stack">D</sub><sup class="stack">20</sup> (<i>c</i> 2.5, EtOH): −31.2°. EA: calcd for C, 58.71; H, 6.77; N, 4.28; F, 5.80. Found C, 58.40; H, 6.71; N, 4.22; F, 5.67.</div></div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Preparation of Aliphatic Aminoesters</h4><div id="sec4_3_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> <i>t</i>-Butyl 12-(2,2-Dimethyl-4,6-dioxo-1,3-dioxan-5-yl)-12-oxododecylcarbamate (<b>43</b>)</h5><div class="NLM_p last">12-(<i>t</i>-Butoxycarbonylamino)dodecanoic acid (<b>42</b>) (1.04 g, 3.29 mmol), dimethylaminopyridine (DMAP, 0.65 g, 5.28 mmol, 1.6 equiv), and Meldrum’s acid (0.53 g, 3.62 mmol, 1.1 equiv) were partially dissolved in anhydrous DCM (30 mL) under an inert atmosphere. The mixture was cooled to 0 °C, and a solution of DCC (0.75 g, 3.62 mmol, 1.1 equiv) in DCM (20 mL) was added slowly to the suspension. The reaction was stirred at 0 °C under an inert atmosphere overnight. Dicyclohexylurea was filtered off and washed with DCM (2 × 10 mL). The solvent was evaporated, and the residue was chromatographed on silica gel (CHCl<sub>3</sub>/acetone, gradient from 10:1 to 5:1) to afford desired derivative <b>43</b> (1.22 g, 84% yield) as a yellowish oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 1.19–1.36 (14H, m), 1.42 (9H, s), 1.50–1.58 (2H, m), 1.61 (3H, s), 1.65–1.70 (2H, m), 1.72 (3H, s), 1.84–1.92 (2H, m), 3.00–3.14 (2H, m), 3.38 (1H, s), 4.54 (1H, s). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 25.0, 25.7, 26.3, 26.9, 28.6, 29.4, 29.6, 29.8, 30.2, 31.1, 33.8, 35.9, 40.8, 42.3, 49.4, 79.2, 104.9, 157.2, 170.7, 198.5. ESI MS: 464 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>23</sub>H<sub>39</sub>O<sub>7</sub>NNa, 464.26187; found, 464.26224.</div></div><div id="sec4_3_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> <i>t</i>-Butyl 12-(2,2-Dimethyl-4,6-dioxo-1,3-dioxan-5-yl)dodecylcarbamate (<b>44</b>)</h5><div class="NLM_p last">The intermediate <b>43</b> (1.22 g, 2.72 mmol) was dissolved in DCM (30 mL) and cooled to −5 °C. Acetic acid (0.96 mL, 16.58 mmol, 6.0 equiv) was added dropwise, and sodium borohydride (0.31 g, 8.15 mmol, 3.0 equiv) was added in three portions. The mixture was stirred at −5 °C overnight and then washed with a saturated solution of NaCl (2 × 20 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and evaporated in vacuo. Column chromatography (CHCl<sub>3</sub>/acetone 25:1) afforded desired derivative <b>44</b> (0.65 g, 56% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 1.22–1.37 (16H, m), 1.38–1.50 (4H, m), 1.43 (9H, s), 1.75 (3H, s), 1.78 (3H, s), 2.05–2.14 (2H, m), 3.09 (2H, t, <i>J</i> = 7.0), 3.49 (1H, dd, <i>J</i> = 6.8, 3.3), 4.48 (1H, s). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 26.5, 26.7, 26.8, 27.0, 28.4, 29.2, 29.3, 29.5 (5C), 30.1, 40.7, 46.1, 79.0, 104.8, 156.0, 165.7. ESI MS: 450 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>23</sub>H<sub>41</sub>O<sub>6</sub>NNa, 450.28261; found, 450.28273.</div></div><div id="sec4_3_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 14-(<i>t</i>-Butoxycarbonylamino)tetradecanoic Acid (<b>45</b>)</h5><div class="NLM_p last">Intermediate <b>44</b> (0.55 g, 1.28 mmol) was heated to 110 °C for 3 h in DMF/H<sub>2</sub>O (10:1, 5 mL). After cooling, solvents were evaporated to obtain <b>45</b> (0.42 g, 94% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 1.30 (18H, s), 1.43 (9H, s), 1.45–1.50 (2H, m), 1.55–1 65 (2H, m), 2.27 (2H, t, <i>J</i> = 7.4), 2.62 (1H, s), 3.01 (2H, t, <i>J</i> = 7.0). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 26.2, 27.9, 28.8, 30.3, 30.4, 30.6, 30.7 (6C), 31.0, 35.1, 41.4, 79.7, 158.6, 177.8. ESI MS: 342 ([M – H]<sup>−</sup>). HR ESI MS: calcd for C<sub>19</sub>H<sub>36</sub>O<sub>4</sub>N, 342.26498; found, 342.26445.</div></div><div id="sec4_3_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> Methyl 14-Aminotetradecanoate (<b>46</b>)</h5><div class="NLM_p last">A solution of <b>45</b> (0.36 g, 1.04 mmol) in MeOH (5 mL) was added slowly to a cooled (0 °C) solution of thionyl chloride (0.31 mL, 4.17 mmol, 4.0 equiv) in MeOH (10 mL). The reaction was stirred at 0 °C for 1 h, and the organic solvent was evaporated. The residue was dissolved in trifluoroacetic acid (TFA, 2 mL) and stirred at room temperature for 15 min. TFA was removed by flow of nitrogen. The residue was partially dissolved in a saturated solution of NaHCO<sub>3</sub> (10 mL) and extracted with EtOAc (3 × 15 mL). The organic phases were combined and dried over MgSO<sub>4</sub>. After filtration, the solvent was evaporated to obtain <b>46</b> (0.17 g, 62% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 1.25–1.39 (18H, m), 1.50–1.64 (4H, m), 2.31 (2H, t, <i>J</i> = 7.4), 3.75 (2H, t, <i>J</i> = 7.4), 3.65 (3H, s). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 26.0, 27.8, 30.2, 30.4, 30.5, 30.6 (2C), 30.7 (3C), 31.4, 34.8, 41.7, 51.9, 176.0. ESI MS: 258 ([M + H]<sup>+</sup>). HR ESI MS: calcd for C<sub>15</sub>H<sub>31</sub>O<sub>2</sub>NH, 258.42276; found, 258.42278.</div></div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> Procedures for Preparation of Macrocyclic Inhibitors with an Aliphatic Chain (<b>6–32</b>, <b>34</b> and <b>36</b>)</h4><div id="sec4_3_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> Methyl (<i>S</i>)-12-(2-((<i>t</i>-Butoxycarbonyl)amino)-3-methylbutanamido) Dodecanoate (<b>49a</b>)</h5><div class="NLM_p last">Boc-<span class="smallcaps smallerCapital">l</span>-Val-OH (<b>47a</b>) (0.84 g, 3.88 mmol) and HBTU (1.48 g, 3.88 mmol, 1.0 equiv) were dissolved in anhydrous DMF (15 mL), and DIEA (1.97 mL, 11.32 mmol, 3.0 equiv) was added. The reaction mixture was stirred at room temperature for 25 min, and a solution of methyl 12-aminododecanoate (<b>48</b>) (0.86 g, 3.23 mmol, 0.8 equiv) in anhydrous DMF (5 mL) was added. The reaction mixture was stirred at room temperature for 6 h under an inert atmosphere. The reaction mixture was diluted with CHCl<sub>3</sub> (30 mL) and washed with saturated NaHCO<sub>3</sub> (2 × 25 mL), 10% KHSO<sub>4</sub> (2 × 25 mL), and brine (35 mL). The mixture was dried over MgSO<sub>4</sub>, and the organic solvent was evaporated in vacuo. Column chromatography on silica gel (CHCl<sub>3</sub>/MeOH 30:1) afforded the desired product <b>49a</b> (1.23 g, 89% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 0.90–0.95 (6H, m), 1.20–1.34 (14H, m), 1.42 (9H, s), 1.45–1.51 (2H, m), 1.56–1.64 (2H, m), 2.03–2.14 (1H, m), 2.29 (2H, t, <i>J</i> = 7.7), 3.14–3.31 (2H, m), 3.66 (3H, s), 3.86 (1H, dd, <i>J</i> = 8.8, 6.6), 5.20 (1H, s), 6.31 (1H, s). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 18.1, 19.4, 25.0, 27.0, 28.4, 29.2, 29.3 (3C), 29.5, 29.6, 29.7, 31.0, 34.2, 39.5, 51.5, 60.2, 156.0, 171.7, 174.4. ESI MS: 451 ([M + Na]<sup>+</sup>).</div></div><div id="sec4_3_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> Methyl (<i>S</i>)-12-(2-((<i>t</i>-Butoxycarbonyl)amino)-3-(pyridin-3-yl)propanamido) Dodecanoate (<b>49b</b>)</h5><div class="NLM_p last">Boc-<span class="smallcaps smallerCapital">l</span>-3Pal-OH (<b>47b</b>) (0.18 g, 0.68 mmol) and HBTU (0.26 g, 0.68 mmol, 1.0 equiv) were dissolved in anhydrous DMF (3 mL), and DIEA (0.34 mL, 1.98 mmol, 3.0 equiv) was added. The reaction mixture was stirred at room temperature for 25 min, and a solution of methyl 12-aminododecanoate (<b>48</b>) (0.15 g, 0.56 mmol, 0.8 equiv) in anhydrous DMF (2 mL) was added. The reaction mixture was stirred at room temperature for 6 h under an inert atmosphere. The reaction mixture was diluted with CHCl<sub>3</sub> (15 mL) and washed with saturated NaHCO<sub>3</sub> (2 × 15 mL), 10% KHSO<sub>4</sub> (2 × 15 mL), and brine (15 mL). The mixture was dried over MgSO<sub>4</sub>, and the organic solvent was evaporated in vacuo. Column chromatography (CHCl<sub>3</sub>/MeOH 25:1) afforded the desired product <b>49b</b> (0.23 g, 85% yield) as an amorphous solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 1.12–1.29 (14H, m), 1.34 (9H, s), 1.35–1.37 (2H, m), 1.58 (2H, p, <i>J</i> = 7.5), 2.27 (2H, t, <i>J</i> = 7.5), 2.91–2.98 (1H, m), 3.05–3.21 (3H, m), 3.63 (3H, s), 4.29–4.36 (1H, m), 5.38 (1H, d, <i>J</i> = 8.6), 6.46 (1H, s), 7.20 (1H, dd <i>J</i> = 7.9, 4.8), 7.54–7.57 (1H, m), 8.41–8.45 (2H, m). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 25.0, 26.9, 28.3, 29.2, 29.3 (2C), 29.5 (2C), 29.6, 29.8, 34.2, 36.1, 39.7, 51.5, 55.5, 80.3, 123.6, 133.1, 137.5, 147.8, 150.2, 155.5, 170.7, 174.4. ESI MS: 478 ([M + H]<sup>+</sup>). HR ESI MS: calcd for C<sub>26</sub>H<sub>44</sub>O<sub>5</sub>N<sub>3</sub>, 478.32755; found, 478.32760.</div></div><div id="sec4_3_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> Methyl (6<i>S</i>,7<i>S</i>,11<i>S</i>)-6-Benzyl-7-hydroxy-2,2-dimethyl-4,9,12-trioxo-11-isopropyl-3-oxa-5,10,13-triazapentacosan-25-oate (<b>50a</b>)</h5><div class="NLM_p last">Compound <b>49a</b> (0.18 g, 0.42 mmol) was dissolved in DCM (2 mL), and TFA (0.2 mL) was added. The reaction mixture was stirred at room temperature for 1 h. The reaction mixture was diluted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL), washed with saturated NaHCO<sub>3</sub> (2 × 10 mL), and dried over MgSO<sub>4</sub>. The organic solvent was evaporated in vacuo to obtain crude amine. Boc-phenylstatine-OH (<b>52</b>) (0.16 g, 0.51 mmol) and HBTU (0.2 g, 0.51 mmol, 1.0 equiv) were dissolved in anhydrous DMF (4 mL), and triethylamine (0.19 mL, 1.36 mmol, 2.5 equiv) was added. The reaction mixture was stirred at room temperature for 30 min, and then a solution of the amine (0.14 g, 0.42 mmol, 0.8 equiv) in anhydrous DMF (2 mL) was added. The reaction mixture was stirred at room temperature for 14 h under an inert atmosphere. The organic solvent was evaporated in vacuo. The residue was chromatographed on silica gel (CHCl<sub>3</sub>/MeOH 10:1) to obtain the desired product (0.16 g, 61% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 0.87–0.94 (6H, m), 1.21–1.33 (14H, m), 1.37 (9H, s), 1.42–1.49 (2H, m), 1.56–1.64 (2H, m), 2.01–2.14 (1H, m), 2.25–2.33 (3H, m), 2.50 (1H, dd, <i>J</i> = 15.0, 9.2), 2.80 (1H, s), 2.89 (2H, d, <i>J</i> = 7.5), 3.09–3.27 (2H, m), 3.65 (3H, s), 3.68–3.74 (1H, m), 3.94–4.01 (1H, m), 4.17 (1H, dd, <i>J</i> = 8.4, 7.0), 5.05 (1H, d, <i>J</i> = 9.7), 6.46 (1H, s), 6.77 (1H, d, <i>J</i> = 8.4), 7.14–7.29 (5H, m). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 18.3, 19.4, 25.1, 27.0, 29.3 (2C), 29.4, 29.5 (2C), 29.6 (2C), 31.0, 34.2, 38.7, 39.8, 40.5, 51.6, 55.6, 59.0, 68.7, 79.7, 126.5, 128.5, 129.4, 138.3, 156.4, 171.0, 172.6, 174.5. ESI MS: 642 ([M + Na]<sup>+</sup>).</div></div><div id="sec4_3_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> Methyl (6<i>S</i>,7<i>S</i>,11<i>S</i>)-6-Benzyl-7-hydroxy-2,2-dimethyl-4,9,12-trioxo-11-(pyridin-3-ylmethyl)-3-oxa-5,10,13-triazapentacosan-25-oate (<b>50b</b>)</h5><div class="NLM_p last">Compound <b>49b</b> (0.18 g, 0.38 mmol) was dissolved in DCM (2 mL), and TFA (0.2 mL) was added. The reaction mixture was stirred at room temperature for 1 h. The reaction mixture was diluted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL), washed with saturated NaHCO<sub>3</sub> (2 × 10 mL), and dried over MgSO<sub>4</sub>. The organic solvent was evaporated in vacuo to obtain crude amine. Boc-phenylstatine-OH (<b>52</b>) (0.14 g, 0.46 mmol) and HBTU (0.17 g, 0.46 mmol, 1.0 equiv) were dissolved in anhydrous DMF (3 mL), and triethylamine (0.17 mL, 1.23 mmol, 2.7 equiv) was added. The reaction mixture was stirred at room temperature for 30 min, and a solution of the amine (0.15 g, 0.38 mmol, 0.8 equiv) in anhydrous DMF (2 mL) was added. The reaction mixture was stirred at room temperature for 14 h under an inert atmosphere. The organic solvent was evaporated in vacuo. The residue was chromatographed on silica gel (CHCl<sub>3</sub>/MeOH 15:1) to obtain the desired product (0.13 g, 51% yield) as an amorphous solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 1.11–1.36 (25H, m), 1.55–1.62 (2H, m), 2.23–2.28 (1H, m), 2.27 (2H, t, <i>J</i> = 7.5), 2.42 (1H, dd, <i>J</i> = 14.9, 9.0), 2.82–2.88 (2H, m), 2.96–3.15 (4H, m), 3.64 (3H, s), 3.67–3.74 (1H, m), 3.92–3.97 (1H, m), 4.77 (1H, dd, <i>J</i> = 7.5, 7.2), 4.84–4.89 (1H, m), 5.11 (1H, d, <i>J</i> = 9.7), 5.79–5.81 (1H, m), 7.03–7.08 (1H, m), 7.11–7.24 (6H, m), 7.51–7.55 (1H, m), 8.36–8.40 (2H, m). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 25.0, 26.9, 28.4, 29.2, 29.3 (2C), 29.5 (2C), 29.6 (2C), 34.2, 36.0, 38.7, 39.7, 40.9, 51.5, 54.0, 55.6, 68.5, 79.6, 123.6, 126.4, 128.5, 129.4, 132.8, 137.6, 138.3, 147.8, 150.2, 156.3, 170.2, 172.1, 174.4. ESI MS: 691 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>37</sub>H<sub>56</sub>O<sub>7</sub>N<sub>4</sub>Na, 691.40412; found, 691.40431.</div></div><div id="sec4_3_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> (6<i>S</i>,7<i>S</i>,11<i>S</i>)-6-Benzyl-7-hydroxy-11-isopropyl-2,2-dimethyl-4,9,12-trioxo-3-oxa-5,10,13-triazapentacosan-25-oic Acid (<b>51a</b>)</h5><div class="NLM_p last">To a solution of methylester <b>50a</b> (0.15 g, 0.24 mmol) in THF (2 mL), a solution of 1 M LiOH (2 mL, 2.0 mmol, 8.3 equiv) was added dropwise. The reaction mixture was stirred at room temperature for 2 h and then acidified with 1 M hydrochloric acid (5 mL). CHCl<sub>3</sub> (10 mL) was added. The layers were separated, and the product was extracted with CHCl<sub>3</sub> (3 × 10 mL). The organic layers were combined and dried over MgSO<sub>4</sub>, and the solvent was removed in vacuo. The residue was chromatographed on silica gel (CHCl<sub>3</sub>/MeOH 20:1) to yield the desired acid <b>51a</b> (0.13 g, 89% yield) as white crystals. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 0.87–0.94 (6H, m), 1.21–1.35 (16H, m), 1.38 (9H, s), 1.41–1.50 (2H, m), 1.57–1.66 (2H, m), 2.04 (1H, s), 2.29–2.36 (3H, m), 2.47–2.57 (1H, m), 2.89 (2H, d, <i>J</i> = 7.4), 3.10–3.30 (2H, m), 3.48 (1H, s), 3.67–3.76 (1H, m), 3.94–4.02 (1H, m), 4.18–4.24 (1H, m), 5.01–5.12 (1H, m), 6.73 (1H, m), 7.16–7.29 (5H, m). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 18.5, 19.4, 24.8, 26.8, 28.5, 28.9, 29.0 (2C), 29.1, 29.2, 29.3, 29.8, 31.0, 34.1, 38.7, 39.8, 40.2, 55.7, 59.2, 68.6, 79.7, 126.5, 128.3, 129.5, 138.3, 156.4, 171.5, 173.1, 175.6, 178.0. ESI MS: 628 ([M + Na]<sup>+</sup>).</div></div><div id="sec4_3_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> (6<i>S</i>,7<i>S</i>,11<i>S</i>)-6-Benzyl-7-hydroxy-2,2-dimethyl-4,9,12-trioxo-11-(pyridin-3-ylmethyl)-3-oxa-5,10,13-triazapentacosan-25-oic Acid (<b>51b</b>)</h5><div class="NLM_p last">To a solution of methylester <b>50b</b> (0.13 g, 0.19 mmol) in THF (2 mL), a solution of 1 M LiOH (2 mL, 2.0 mmol, 10.5 equiv) was added dropwise. The reaction mixture was stirred at room temperature for 2 h and then acidified with 1 M hydrochloric acid (5 mL). CHCl<sub>3</sub> (10 mL) was added. The layers were separated, and the product was extracted with CHCl<sub>3</sub> (3 × 10 mL). The organic layers were combined and dried over MgSO<sub>4</sub>, and the solvent was removed in vacuo. The residue was chromatographed on silica gel (CHCl<sub>3</sub>/MeOH gradient from 15:1 to 5:1) to yield the desired acid <b>51b</b> (0.10 g, 77% yield) as an amorphous solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 1.12–1.35 (25H, m), 1.54–1.62 (2H, m), 2.21–2.27 (1H, m), 2.28 (2H, t, <i>J</i> = 7.5), 2.44 (1H, dd, <i>J</i> = 14.9, 9.0), 2.83–2.88 (2H, m), 2.96–3.14 (4H, m), 3.68–3.75 (1H, m), 3.92–3.98 (1H, m), 4.78 (1H, dd, <i>J</i> = 7.5, 7.3), 4.84–4.88 (1H, m), 5.11 (1H, d, <i>J</i> = 9.7), 5.77–5.81 (1H, m), 7.02–7.09 (1H, m), 7.12–7.24 (6H, m), 7.51–7.54 (1H, m), 8.36–8.41 (2H, m). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 25.0, 26.9, 28.4, 29.2, 29.3 (2C), 29.5 (2C), 29.6 (2C), 34.2, 36.1, 38.7, 39.7, 41.0, 54.0, 55.7, 68.5, 79.6, 123.6, 126.4, 128.5, 129.3, 132.8, 137.6, 138.3, 147.8, 150.2, 156.4, 170.3, 172.1, 172.2. ESI MS: 677 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>36</sub>H<sub>54</sub>O<sub>7</sub>N<sub>4</sub>Na, 677.38847; found, 677.38849.</div></div><div id="sec4_3_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-isopropyl-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>9</b>)</h5><div class="NLM_p last">To a solution of acid <b>51a</b> (92 mg; 0.14 mmol) and TSTU (42 mg, 0.14 mmol, 1.0 equiv) in anhydrous DMF (2 mL), DIEA (26 μL, 0.14 mmol, 1.0 equiv) was added. The reaction mixture was stirred at room temperature for 2 h under an inert atmosphere. The reaction mixture was diluted with CHCl<sub>3</sub> (15 mL), washed with brine (2 × 10 mL), and dried over MgSO<sub>4</sub>. The organic solvent was evaporated in vacuo. The residue was dissolved in TFA (1 mL), and the reaction mixture was stirred at room temperature for 15 min. TFA was removed by flow of nitrogen to obtain crude amine. The crude amine was dissolved in anhydrous DMF (5 mL) and added dropwise overnight to a solution of DIEA (5 mL) in anhydrous DMF (5 mL) under stirring. The solvent was evaporated, and the crude product was purified by preparative HPLC to afford <b>9</b> (29 mg, 41% yield). Spectral data for <b>9</b> are reported in the part characterization of macrocyclic inhibitors.</div></div><div id="sec4_3_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> (3<i>S</i>,7<i>S</i>,8<i>S</i>)-8-Benzyl-7-hydroxy-3-(pyridin-3-ylmethyl)-1,4,9-triazacyclohenicosane-2,5,10-trione (<b>25</b>)</h5><div class="NLM_p last">To a solution of acid <b>51b</b> (90 mg, 0.14 mmol) and TSTU (42 mg, 0.14 mmol, 1.0 equiv) in anhydrous DMF (2 mL), DIEA (24 μL, 0.14 mmol, 1.0 equiv) was added. The reaction mixture was stirred at room temperature for 2 h under an inert atmosphere. The reaction mixture was diluted with CHCl<sub>3</sub> (15 mL), washed with brine (2 × 10 mL), and dried over MgSO<sub>4</sub>. The organic solvent was evaporated in vacuo. The residue was dissolved in TFA (1 mL), and the mixture was stirred at room temperature for 15 min. TFA was removed by flow of nitrogen to obtain crude amine. The crude amine was dissolved in anhydrous DMF (5 mL) and added dropwise overnight to a solution of DIEA (5 mL) in anhydrous DMF (5 mL) under stirring. The solvent was evaporated, and crude product was purified by preparative HPLC to afford <b>25</b> (35 mg, 47% yield). Spectral data for <b>25</b> are reported in the part characterization of macrocyclic inhibitors.</div></div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> Procedures for Preparation of Macrocycle with a PEG Chain (<b>35</b>)</h4><div id="sec4_3_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> (<i>S</i>)-Methyl 2-((3<i>S</i>,4<i>S</i>)-4-(<i>t</i>-Butoxycarbonylamino)-3-hydroxy-5-phenylpentanamido)-3-methylbutanoate (<b>53</b>)</h5><div class="NLM_p last">Boc-phenylstatine-OH (<b>52</b>) (0.19 g, 0.61 mmol) and HBTU (0.23 g, 0.61 mmol, 1.0 equiv) were dissolved in anhydrous DMF (2.5 mL), and DIEA (0.36 mL, 2.05 mmol, 3.4 equiv) was added. The reaction mixture was stirred at room temperature for 30 min, and a solution of H-Val-OMe (90 mg, 0.50 mmol, 0.8 equiv) in anhydrous DMF (2 mL) was added. The reaction mixture was stirred at room temperature for 12 h under an inert atmosphere. The reaction mixture was diluted with CHCl<sub>3</sub> (30 mL) and washed with saturated NaHCO<sub>3</sub> (2 × 20 mL), 10% KHSO<sub>4</sub> (2 × 20 mL), and brine (30 mL). The mixture was dried over MgSO<sub>4</sub> and the organic solvent was evaporated in vacuo. The residue was chromatographed on silica gel (CHCl<sub>3</sub>/MeOH 20:1) to obtain the desired product <b>53</b> (0.16 g, 62% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 0.90 (3H, d, <i>J</i> = 6.9), 0.93 (3H, d, <i>J</i> = 6.9), 1.40 (9H, s), 2.08–2.20 (1H, m), 2.29 (1H, dd, <i>J</i> = 15.2, 3.1), 2.52 (1H, dd, <i>J</i> = 15.2, 9.6), 2.85 (1H, s), 2.91 (2H, d, <i>J</i> = 7.6), 3.72 (3H, s), 3.73–3.81 (1H, m), 3.96 (1H, d, <i>J</i> = 8.1), 4.49 (1H, dd, <i>J</i> = 8.6, 4.9), 4.99 (1H, d, <i>J</i> = 9.4), 6.53 (1H, d, <i>J</i> = 8.4), 7.16–7.31 (5H, m). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 17.9, 19.1, 28.5, 31.2, 38.7, 40.0, 52.4, 55.6, 57.3, 68.2, 79.7, 126.5, 128.6, 129.5, 138.3, 156.4 172.3, 172.9. ESI MS: 445 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>22</sub>H<sub>34</sub>O<sub>6</sub>N<sub>2</sub>Na, 445.23091, found, 445.23092.</div></div><div id="sec4_3_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> (19<i>S</i>,20<i>S</i>,24<i>S</i>)-Methyl 19-Benzyl-20-hydroxy-24-isopropyl-2,2-dimethyl-4,17,22-trioxo-3,8,11,14-tetraoxa-5,18,23-triazapentacosan-25-oate (<b>54</b>)</h5><div class="NLM_p last">Compound <b>53</b> (0.15 g, 0.33 mmol) was dissolved in TFA (1 mL), and the reaction mixture was stirred at room temperature for 15 min. TFA was removed by flow of nitrogen to obtain crude amine. BocHN-PEG-3-CH<sub>2</sub>CH<sub>2</sub>COOH (0.10 g, 0.33 mmol) and HBTU (0.12 g, 0.33 mmol, 1.0 equiv) were dissolved in anhydrous DMF (2.5 mL), and DIEA (0.17 mL, 0.99 mmol, 3.0 equiv) was added. The reaction mixture was stirred at room temperature for 30 min, and a solution of the amine (0.10 g, 0.33 mmol, 1.0 equiv) in anhydrous DMF (2 mL) was added. The reaction mixture was stirred at room temperature for 16 h under an inert atmosphere. Column chromatography on silica gel (CHCl<sub>3</sub>/MeOH 20:1) afforded the desired product <b>54</b> (0.15 g, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 0.92 (3H, d, <i>J</i> = 6.9), 0.94 (3H, d, <i>J</i> = 6.9), 1.43 (9H, s), 2.10–2.21 (1H, m), 2.38 (1H, dd, <i>J</i> = 14.9, 4.7), 2.47–2.51 (2H, m), 2.53 (1H, dd, <i>J</i> = 15.0, 8.9), 2.93–2.98 (4H, m), 3.29 (2H, t, <i>J</i> = 5.1), 3.53 (2H, t, <i>J</i> = 5.2), 3.58–3.65 (8H, m), 3.70 (3H, s), 3.99–4.04 (1H, m), 4.15 (1H, dd, <i>J</i> = 16.1, 8.4), 4.44 (1H, dd, <i>J</i> = 8.4, 5.2), 7.23–7.28 (5H, m). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 18.0, 19.1, 28.6, 30.8, 36.8, 37.8, 40.4, 52.3, 54.8, 57.7, 67.3, 69.0, 70.3, 70.4 (4C), 70.5, 79.6, 126.6, 128.6, 129.4, 138.2, 156.3, 172.3, 172.9, 173.3. ESI MS: 648 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>31</sub>H<sub>51</sub>O<sub>10</sub>N<sub>3</sub>Na, 648.34667; found, 648.34678.</div></div><div id="sec4_3_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> (12<i>S</i>,16<i>S</i>,17<i>S</i>)-17-Benzyl-16-hydroxy-12-isopropyl-1,4,7-trioxa-10,13,18-triazacyclo Henicosane-11,14,19-trione (<b>35</b>)</h5><div class="NLM_p last">To a solution of corresponding methylester <b>54</b> (65 mg, 0.10 mmol) in THF (2 mL), a solution of LiOH (7 mg, 0.15 mmol, 1.5 equiv) in water (2 mL) was added dropwise. The reaction mixture was stirred at room temperature for 2 h, and then the organic solvent was evaporated in vacuo. The residue was dissolved in TFA (2 mL), and the reaction mixture was stirred at room temperature for 15 min. TFA was removed by flow of nitrogen to obtain crude amino acid salt. A solution of DIEA (0.37 mL, 2.00 mmol, 20 equiv) and DPPA (30 mg, 0.11 mmol, 1.1 equiv) in anhydrous DMF (10 mL) was slowly added to a solution of the crude amino acid salt in DMF (5 mL). The reaction mixture was stirred overnight. The solvent was evaporated and crude product was purified by preparative HPLC to afford desired compound <b>35</b> (27 mg, 55% yield). Spectral data for <b>35</b> are reported in the part characterization of macrocyclic inhibitors.</div></div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> Procedures for Preparation of Macrocycle with a Carbamate Moiety (<b>33</b>)</h4><div id="sec4_3_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> <i>t</i>-Butyl (<i>S</i>)-(1-((11-Hydroxyundecyl)amino)-3-methyl-1-oxobutan-2-yl)carbamate (<b>57</b>)</h5><div class="NLM_p last">Boc-<span class="smallcaps smallerCapital">l</span>-Val-OSu (<b>55</b>) (1.00 g, 3.18 mmol) and 11-aminoundecan-1-ol (<b>56</b>) (0.65 g, 3.50 mmol, 1.1 equiv) were dissolved in anhydrous DMF (10 mL), and DIEA (1.19 mL, 6.40 mmol, 2.0 equiv) was added. The reaction mixture was stirred at room temperature overnight. The reaction mixture was diluted with EtOAc (50 mL) and washed with saturated NaHCO<sub>3</sub> (3 × 50 mL), 10% KHSO<sub>4</sub> (3 × 50 mL), and brine (35 mL). The mixture was dried over MgSO<sub>4</sub> and the organic solvent was evaporated in vacuo to yield the desired product <b>57</b> (1.13 g, 93% yield) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): 0.90 (3H, d, <i>J</i> = 6.8), 0.93 (3H, d, <i>J</i> = 6.8), 1.20–1.34 (12H, m), 1.42 (9H, s), 1.44–1.58 (4H, m), 1.78–1.94 (1H, m), 2.00–2.15 (1H, m), 3.13–3.30 (2H, m), 3.61 (2H, t, <i>J</i> = 6.6), 3.82 (1H, dd, <i>J</i> = 9.0, 6.6), 5.16 (1H, d, <i>J</i> = 9.1), 6.21 (1H, t, <i>J</i> = 5.8). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): 18.1, 19.4, 25.8, 26.9, 28.4, 29.3, 29.5 (3C), 29.6, 30.9, 32.9, 39.6, 60.3, 63.0, 79.9, 156.1, 171.7. ESI MS: 409 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>21</sub>H<sub>42</sub>O<sub>4</sub>N<sub>2</sub>Na, 409.30368; found, 409.30348.</div></div><div id="sec4_3_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> <i>t</i>-Butyl ((2<i>S</i>,3<i>S</i>)-3-Hydroxy-5-(((<i>S</i>)-1-((11-hydroxyundecyl)amino)-3-methyl-1-oxobutan-2-yl)amino)-5-oxo-1-phenylpentan-2-yl)carbamate (<b>58</b>)</h5><div class="NLM_p last">Compound <b>57</b> (1.0 g, 2.59 mmol) was dissolved in DCM (5 mL), and TFA (5 mL) was added. The reaction mixture was stirred at room temperature for 1 h. The organic solvent was evaporated in vacuo to obtain crude amine. Boc-phenylstatine-OH (<b>52</b>) (0.72 g, 2.33 mmol) and TSTU (0.70 g, 2.33 mmol, 1.0 equiv) were dissolved in anhydrous DMF (5 mL), and DIEA (1.44 mL, 7.77 mmol, 3.0 equiv) was added. The reaction mixture was stirred at room temperature for 1 h. A solution of the amine in anhydrous DMF (2 mL) was added. The reaction mixture was stirred at room temperature for 2 h. The reaction mixture was then diluted with EtOAc (50 mL) and washed with saturated NaHCO<sub>3</sub> (3 × 50 mL), 10% KHSO<sub>4</sub> (3 × 50 mL), and brine (35 mL). The mixture was dried over MgSO<sub>4</sub> and the organic solvent was evaporated in vacuo. Column chromatography on silica gel (DCM/MeOH 20:1) afforded the desired product <b>58</b> (1.0 g, 75% yield). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 0.94 (3H, d, <i>J</i> = 6.5), 0.96 (3H, d, <i>J</i> = 6.5), 1.21–1.33 (14H, m), 1.35 (9H, s), 1.41–1.57 (5H, m), 2.12 (1H, dq, <i>J</i> = 13.5, 6.8), 2.45 (1H, dd, <i>J</i> = 7.0, 3.1), 2.77 (1H, dd, <i>J</i> = 13.7, 9.5), 2.87 (1H, dd, <i>J</i> = 13.7, 5.4), 3.15 (2H, ddp, <i>J</i> = 20.3, 13.4, 7.1), 3.53 (2H, t, <i>J</i> = 6.7), 3.74 (1H, ddd, <i>J</i> = 9.6, 5.4, 2.1), 4.06 (1H, td, <i>J</i> = 7.0, 2.0), 4.11 (1H, d, <i>J</i> = 6.0), 7.11–7.28 (5H, m). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): 18.2, 19.7, 27.0, 28.0, 28.8, 30.4 (2C), 30.6, 30.7 (2C), 31.4, 33.7, 39.2, 40.4, 41.5, 57.0, 60.7, 63.0, 70.8, 80.1, 127.2, 129.2, 130.4, 140.2, 158.4, 173.6, 174.0. ESI MS: 600 ([M + Na]<sup>+</sup>). HR ESI MS: calcd for C<sub>32</sub>H<sub>55</sub>O<sub>6</sub>N<sub>3</sub>Na, 600.39831; found, 600.39754.</div></div><div id="sec4_3_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> (4<i>S</i>,5<i>S</i>,9<i>S</i>)-4-Benzyl-5-hydroxy-9-isopropyl-1-oxa-3,8,11-triazacyclodocosane-2,7,10-trione (<b>33</b>)</h5><div class="NLM_p last">Compound <b>58</b> (50 mg, 0.09 mmol) and disuccinimidyl carbonate (33 mg, 0.13 mmol, 1.5 equiv) were dissolved in anhydrous DMF (2 mL). The reaction mixture was stirred at room temperature overnight. The reaction mixture was then diluted with EtOAc (25 mL) and washed with saturated NaHCO<sub>3</sub> (3 × 25 mL), 10% KHSO<sub>4</sub> (3 × 25 mL), and brine (20 mL). It was then dried over MgSO<sub>4</sub>, and the organic solvent was evaporated in vacuo. The intermediate was then dissolved in DCM (2 mL), and TFA (2 mL) was added. The reaction mixture was stirred at room temperature for 20 min. The organic solvent was evaporated in vacuo to obtain crude amine. The crude amine was dissolved in anhydrous DMF (2 mL) and added dropwise overnight to a solution of DIEA (1 mL) in anhydrous DMF (5 mL) under stirring. The solvent was evaporated, and the crude product was purified by preparative HPLC to afford <b>33</b> (23 mg, 52% yield). Spectral data for <b>33</b> are reported in the part characterization of macrocyclic inhibitors.</div></div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> Enzymatic Assays</h3><div class="NLM_p">The enzymatic activity of human CatD was measured in a kinetic assay using the FRET substrate Abz-Lys-Pro-Ala-Glu-Phe-Nph-Ala-Leu.<a onclick="showRef(event, 'ref38 ref40'); return false;" href="javascript:void(0);" class="ref ref38 ref40">(38,40)</a> The assay was performed in 96-well format with a total reaction volume of 100 μL at 37 °C. The kinetics of the product release were continuously monitored in an Infinite M1000 microplate reader (Tecan) at excitation and emission wavelengths of 330 and 420 nm, respectively. For inhibition measurements, 1 nM CatD was preincubated for 10 min with 0–200 μM inhibitors in 0.1 M sodium acetate, pH 4.2, 0.1% (w/v) PEG 1500, followed by addition of 25 μM substrate. The final concentration of DMSO in the assay did not exceed 2% (v/v). IC<sub>50</sub> values were determined from residual velocities using dose–response plots; nonlinear regression was fitted using GraFit software (Erithacus). The mode of inhibition was analyzed at varying substrate concentrations, and the initial velocities of product release were interpreted using double reciprocal plots.</div><div class="NLM_p">Screening of macrocyclic inhibitors against a panel of aspartic proteases was performed using a modified human CatD assay in 384-well format with a total reaction volume of 20 μL at 37 °C. Inhibitors were applied at a concentration of 50 nM. The assay conditions for individual proteases were as follows: 1.2 nM tick parasite CatD and 25 μM Abz-Lys-Pro-Ala-Glu-Phe-Nph-Ala-Leu substrate in 0.1 M sodium acetate, pH 4.0; 2.7 nM yeast proteinase A and 25 μM Abz-Lys-Pro-Ala-Glu-Phe-Nph-Ala-Leu substrate in 0.1 M sodium acetate, pH 5.0; 10 nM porcine pepsin and 10 μM Abz-Lys-Pro-Ala-Glu-Phe-Nph-Ala-Leu substrate in 0.1 M sodium acetate, pH 4.0; 30 nM human BACE1 and 25 μM Mca-Ser-Glu-Val-Asn-Leu-Asp-Ala-Glu-Phe-Arg-Lys(Dnp)-Arg-Arg substrate in 0.1 M sodium acetate, pH 4.6; 14 nM HIV-1 protease and 40 μM Abz-Tyr-Ile-Nle-Nph-Gln-Arg substrate in 0.1 M sodium acetate, pH 5.0, 1 mM EDTA; and 20 nM human renin and 10 μM Arg-Glu(EDANS)-Ile-His-Pro-Phe-His-Leu-Val-Ile-His-Thr-Lys(Dabcyl)-Arg substrate in 0.1 M MES, pH 6.0, 0.15 M NaCl.<a onclick="showRef(event, 'ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref41 ref42 ref43">(41−43)</a></div><div class="NLM_p">Screening of macrocyclic inhibitors against a panel of non-aspartic proteases was performed in 98-well format with a total reaction volume of 100 μL at 37 °C. Inhibitors were applied at a concentration of 1 μM. The assay conditions for individual proteases were as follows: 2 nM trypsin and 10 μM Z-Phe-Arg-AMC substrate in 0.1 M Tris-HCl, pH 7.5, 10 mM CaCl<sub>2</sub>; 0.34 nM chymotrypsin and 10 μM Z-Phe-Arg-AMC substrate in 0.1 M Tris-HCl, pH 7.5; 1.2 nM papain and 10 μM Z-Phe-Arg-AMC substrate in 0.1 M sodium acetate, pH 5.5; 4.3 nM thermolysin, and 10 μM Abz-Ala-Gly-Leu-Ala-nitrobenzylamide in 0.1 M Tris-HCl, pH 7.5, 10 mM CaCl<sub>2</sub>.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a></div><div class="NLM_p">All enzymatic activity measurements were performed in triplicate in at least two independent experiments.</div><div class="NLM_p last">Resistance of macrocycles to proteolysis was investigated using a previously described assay.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The reaction mixture, containing 25 μg macrocycle and 1 μg protease (trypsin, chymotrypsin, papain, thermolysin, or human CatD), was incubated in the corresponding protease activity assay buffer in a total volume of 150 μL at 26 °C for 18 h. The reaction was stopped by adding 100 μL acetic acid and 50 μL of 10% TFA. The mixture was analyzed by HPLC, and the obtained profile was compared to that of a control not subjected to protease treatment. Proteolytic degradation was considered insignificant if the change in macrocycle peak area was less than 10% (within the precision limit of the method). The HPLC separation was performed on a Jupiter C18 300A column (5 μm, 250 × 4.6 mm; Phenomenex) equilibrated in 10% (v/v) acetonitrile in 0.1% (v/v) TFA and eluted with a linear gradient of 99% (v/v) acetonitrile in 0.1% (v/v) TFA.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> Cell-Based Assays</h3><div class="NLM_p last">Compound cytotoxicity was evaluated on nontumor human dermal fibroblasts and peripheral blood mononuclear cells. Cells were seeded in 384-well plates at a density of 2,000–50,000 cells per well, and the tested compounds were added at varying concentrations. The cells were incubated at 37 °C, 5% CO<sub>2</sub> for 72 h, after which CellTiter-Glo (Promega) detection reagent was added. The luminescence signal of the compound-treated cells was compared to that of an untreated control set to 100% cell viability. The transport potential of the tested compounds across cell membranes was evaluated with a Caco-2 permeability assay (BD BioCoat HTS Caco-2 assay system, BD Biosciences) as previously described.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Inhibitors were applied at 5 μM concentration to the apical side of the Caco-2 cell monolayer. After 3 h, the concentration of the compounds was determined in both the apical and basolateral compartments using liquid chromatography–mass spectrometry, and apparent permeability (<i>P</i><sub>app</sub>) was calculated.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> Crystal Structure Determination</h3><div class="NLM_p last">Preparation and crystallization of complexes of human CatD with inhibitors were as follows. Purified human CatD (7.5 mg mL<sup>–1</sup>) was incubated for 30 min at room temperature with a twofold molar excess of macrocyclic inhibitor (applied as DMSO solution) in 25 mM Tris-HCl, pH 7.0 (the final DMSO concentration did not exceed 7%). The mixture was centrifuged for 5 min at 16,000<i>g</i>. Crystals were grown at 20 °C by the hanging-drop vapor-diffusion method. Drops were equilibrated over a precipitant solution of 0.1 M Bis-Tris, pH 5.5, containing 25% PEG 3350, and 0.2 M Li<sub>2</sub>SO<sub>4</sub>. The ratio of protein to precipitant solution was 2:1 for CatD-<b>9</b> and CatD-<b>33</b> complexes and 1:1 for CatD-<b>14</b>. The crystals were flash-cooled in liquid nitrogen with the precipitant solution as self-cryoprotectant. Diffraction data were collected at 100 K on MX 14.1 operated by the Joint Berlin MX-Laboratory at the BESSY II electron storage ring in Berlin-Adlershof, Germany. All diffraction data were processed using the XDS suite of programs. The structures of the complexes were solved by molecular replacement with the program MOLREP<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> using uncomplexed CatD (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYA">1LYA</a>) as a search model. Model refinement was carried out using the REFMAC 5.2 program<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> from the CCP4 package,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> with manual adjustments in Coot.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The quality of the final model was validated with MolProbity.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The final refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 4. All figures showing structural representations were prepared with the program PyMOL (Schrodinger LLC). Atomic contacts between inhibitors and protein moiety were generated using CCP4 Contact. Crystal parameters and data collection statistics are given in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">Supporting Information</a> Table 4.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> Interaction Energy Calculations</h3><div class="NLM_p">The crystal structures were examined by the hybrid QM/SQM method. Hydrogen atoms were added at the authentic protonation state, and water molecule W14 was included in the calculations. Noncrystallographic complexes were built in multiple orientations based on CatD-<b>14</b> and CatD-<b>9</b> crystal structures.</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> Structure Preparation</h4><div class="NLM_p last">Hydrogen atoms were added by the Reduce and Leap programs in AMBER 14.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> PROPKA 3.0<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> was used to assign protonation states. Aspartates, glutamates, lysines, arginines, and histidines were charged, except for Glu5, Asp50, Glu180, Asp231, and Asp323. Hydrogen atoms were relaxed by annealing from 600 to 0 K at the MM level in AMBER 14. The FF14SB force field was used for the protein, whereas the GAFF force field was used for the ligand. The cooling runs were 4 ps long with 1 fs step and Berendsen thermostat used.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> QM/SQM Setup</h4><div class="NLM_p last">The QM part comprised residues located within 2.5 Å of inhibitor <b>9</b> in the CatD-<b>9</b> crystal structure. The QM part was treated at the DFT-D3/BLYP/TZVPP level for energy calculations and at the DFT-D3/BLYP/DZVP level for optimization.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The rest of the system was treated at the PM6-D3H4X level.<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a> The environment was described by the COSMO implicit solvent model.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The coupling between QM and SQM was performed with Cuby4,<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> which calls Turbomole 7.0<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and MOPAC<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> for QM and SQM, respectively. Residues farther than 6 Å from inhibitor <b>9</b> in the CatD-<b>9</b> crystal structure were frozen during the optimization.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> Scoring Function</h4><div class="NLM_p last">The score was computed as a sum of gas-phase interaction energy (Δ<i>E</i><sub>int</sub>), the interaction solvation free energy (ΔΔ<i>G</i><sub>solv</sub>), and the change in conformational “free” energy of the ligand (Δ<i>G</i><sub class="stack">conf</sub><sup class="stack">′W</sup>(L)).<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62,63)</a> Δ<i>E</i><sub>int</sub> was computed using the QM/SQM methodology described above. Two methods were used for solvation free energy calculations: PM6/COSMO implemented in MOPAC for the protein and SMD/HF/6-31G*<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> implemented in Gaussian09<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> for the ligand. Δ<i>G</i><sub class="stack">conf</sub><sup class="stack">′W</sup>(L) was computed as the “free” energy change between the ligand structure in the optimized protein–ligand complex and the ligand structure optimized in solution by the same method. The “free” energy was defined as the sum of the gas-phase DFT-D3/TPSS/TZVPP energy with the SMD/HF/6-31G* solvation free energy for the Δ<i>G</i><sub class="stack">conf</sub><sup class="stack">′W</sup>(L) term.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Glycine Scan and Inhibitor Fragmentation</h4><div class="NLM_p last">The energy contributions of individual amino acid residues were calculated as the difference between the interaction “free” energy (Δ<i>G</i>′) with the wild-type amino acid and the Δ<i>G</i>′ with the mutated glycine residue. For the purposes of inhibitor fragmentation, selected C–C bonds of the inhibitors were cut and capped with hydrogens, and Δ<i>G</i>′ between the ligand fragment and the whole protein was computed. The Δ<i>G</i>′ values were determined using the QM/SQM methodology described above (DFT-D3/TPSS/TZVPP for QM and PM6-D3H4X for SQM combined with the COSMO implicit solvent model).</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> Noncrystallographic Complexes</h4><div class="NLM_p last">The complex of CatD with <b>22</b> was built based on the CatD-<b>14</b> crystal structure. Complexes with inhibitors <b>8</b>, <b>24</b>, <b>25</b>, and <b>31</b> were built based on the CatD-<b>9</b> crystal structure. The ligand modifications were built manually using the PyMOL program in multiple orientations. All complexes were optimized using the hybrid QM/SQM methodology, and the most stable complexes were used for scoring (as described above).</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i107"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01351" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30438" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30438" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01351?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01351</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Physicochemical parameters of pepstatin A (<b>1</b>) and the most potent macrocyclic inhibitors with subnanomolar and single-digit nanomolar inhibition of CatD; cytotoxicity analysis of model macrocyclic inhibitors and pepstatin A (<b>1</b>); correlation of computed scores plotted against experimental binding free energy for selected macrocyclic inhibitors; inhibition kinetics for the macrocyclic inhibitor 9 and human CatD; list of atomic contacts between macrocyclic inhibitors and CatD; X-ray data collection and refinement statistics; and analytical RP-HPLC traces for the key compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Protien data bank for CatD-<b>9</b> complex (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Protien data bank for CatD-<b>14</b> complex (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Protien data bank for CatD-<b>33</b> complex (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_005.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf">jm9b01351_si_001.pdf (877.3 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_002.csv">jm9b01351_si_002.csv (2.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_003.pdb">jm9b01351_si_003.pdb (304.7 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_004.pdb">jm9b01351_si_004.pdb (288.4 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_005.pdb">jm9b01351_si_005.pdb (280.57 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structures presented in this article have been deposited in the Protein Data Bank under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QCB">6QCB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBG">6QBG</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBH">6QBH</a> for the CatD-<b>9</b>, CatD-<b>14</b>, and CatD-<b>33</b> complexes, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01351" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18035" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18035" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pavel Majer</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bfd2ded5dacdffcad0dcd7dd91dcdecc91dcc5"><span class="__cf_email__" data-cfemail="91fcf0fbf4e3d1e4fef2f9f3bff2f0e2bff2eb">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Mareš</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0847-5022" title="Orcid link">http://orcid.org/0000-0002-0847-5022</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2845495a4d5b685d474b404a064b495b064b52"><span class="__cf_email__" data-cfemail="503d3122352310253f3338327e3331237e332a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Radka Houštecká</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span>; 
    <span class="hlFld-Affiliation affiliation">First
Faculty of Medicine, Charles University, Kateřinská 32, 12108 Praha 2, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Hadzima</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Organic Chemistry, Faculty of Science, Charles University, Albertov 6, 12800 Praha 2, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jindřich Fanfrlík</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiří Brynda</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lenka Pallová</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Iva Hánová</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Faculty of Science, Charles
University, Albertov
6, 12800 Praha 2, Czech Republic</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7173-8200" title="Orcid link">http://orcid.org/0000-0002-7173-8200</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helena Mertlíková-Kaiserová</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Lepšík</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Horn</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Czech
Academy of Sciences, Flemingovo nám. 2, 16610 Praha 6, Czech Republic</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9110-2018" title="Orcid link">http://orcid.org/0000-0001-9110-2018</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Smrčina</span> - <span class="hlFld-Affiliation affiliation">Tucson
Research Center, Icagen Inc., 2090 E. Innovation Park Drive, Oro Valley, Arizona 85755, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i109">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26135" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26135" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by project ChemBioDrug CZ.02.1.01/0.0/0.0/16_019/0000729 from the European Regional Development Fund (OP RDE), grant 15-18929S from the Czech Science Foundation, project InterBioMed LO1302 from the Ministry of Education of the Czech Republic, grant 802216 from the Charles University Grant Agency, Gilead Sciences and IOCB Research Center, and institutional project RVO 61388963. We would like to thank Radko Souček for amino acid analysis and solubility analysis, and Pavlı́na Řezáčová and Tat’ána Majerová (Institute of Organic Chemistry and Biochemistry, C.A.S., Prague, Czech Republic) for materials for the HIV-1 protease assay. We are also very grateful to Jaroslav Feyereisl and voluntary donors from the Institute for the Care of Mother and Child (Prague, Czech Republic) for placental tissue.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Abz</td><td class="NLM_def"><p class="first last">aminobenzoic acid</p></td></tr><tr><td class="NLM_term">AMC</td><td class="NLM_def"><p class="first last">7-amino-4-methylcoumarin</p></td></tr><tr><td class="NLM_term">CatD</td><td class="NLM_def"><p class="first last">cathepsin D</p></td></tr><tr><td class="NLM_term">Iva</td><td class="NLM_def"><p class="first last">isovaleryl</p></td></tr><tr><td class="NLM_term">Sta</td><td class="NLM_def"><p class="first last">statine</p></td></tr><tr><td class="NLM_term">TSTU</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyl-<i>O</i>-(<i>N</i>-succinimidyl)uronium tetrafluoroborate</p></td></tr><tr><td class="NLM_term">Z</td><td class="NLM_def"><p class="first last"><i>N</i>-carbobenzyloxy</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i111">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85868" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85868" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 65 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fusek, M.</span>; <span class="NLM_string-name">Mareš, M.</span>; <span class="NLM_string-name">Větvička, V.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D</span>. In  <i>Handbook of Proteolytic Enzymes</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlings, N.</span>, <span class="NLM_string-name">Salvesen, G.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Oxford</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">63</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2FB978-0-12-382219-2.00008-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=54-63&author=M.+Fusek&author=M.+Mare%C5%A1&author=V.+V%C4%9Btvi%C4%8Dkaauthor=N.+Rawlings&author=G.+Salvesen&title=Handbook+of+Proteolytic+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-382219-2.00008-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-382219-2.00008-9%26sid%3Dliteratum%253Aachs%26aulast%3DFusek%26aufirst%3DM.%26atitle%3DCathepsin%2520D%26btitle%3DHandbook%2520of%2520Proteolytic%2520Enzymes%26aulast%3DRawlings%26aufirst%3DN.%26pub%3DAcademic%2520Press%26date%3D2013%26spage%3D54%26epage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melone, M. A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isidoro, C.</span></span> <span> </span><span class="NLM_article-title">The Role of Cathepsin D in the Pathogenesis of Human Neurodegenerative Disorders</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">870</span>, <span class="refDoi"> DOI: 10.1002/med.21394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fmed.21394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=27114232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=845-870&author=C.+Vidoniauthor=C.+Folloauthor=M.+Savinoauthor=M.+A.+B.+Meloneauthor=C.+Isidoro&title=The+Role+of+Cathepsin+D+in+the+Pathogenesis+of+Human+Neurodegenerative+Disorders&doi=10.1002%2Fmed.21394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Cathepsin D in the Pathogenesis of Human Neurodegenerative Disorders</span></div><div class="casAuthors">Vidoni, Chiara; Follo, Carlo; Savino, Miriam; Melone, Mariarosa A. B.; Isidoro, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">845-870</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  A review.  In familial neurodegenerative disorders, protein aggregates form continuously because of genetic mutations that drive the synthesis of truncated or unfolded proteins.  The oxidative stress imposed by neurotransmitters and environmental neurotoxins constitutes an addnl. threat to the folding of the proteins and the integrity of organelle membranes in neurons.  Failure in degrading such altered materials compromises the function of neurons and eventually leads to neurodegeneration.  The lysosomal proteolytic enzyme Cathepsin D is the only aspartic-type protease ubiquitously expressed in all the cells of the human body, and it is expressed at high level in the brain.  In general, cathepsin D mediated proteolysis is essential to neuronal cell homeostasis through the degrdn. of unfolded or oxidized protein aggregates delivered to lysosomes via autophagy or endocytosis.  More specifically, many altered neuronal proteins that hallmark neurodegenerative diseases (e.g., the amyloid precursor, α-synuclein, and huntingtin) are physiol. substrates of cathepsin D and would abnormally accumulate if not efficiently degraded by this enzyme.  Furthermore, exptl. evidence indicates that cathepsin D activity is linked to the metab. of cholesterol and of glycosaminoglycans, which accounts for its involvement in neuronal plasticity.  This review focuses on the unique role of cathepsin D mediated proteolysis in the pathogenesis of human neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCbQT1BCqB7rVg90H21EOLACvtfcHk0lgUI08xlaAlBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bE&md5=2e0d3c04b6038f01e77a515dfc329c22</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fmed.21394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21394%26sid%3Dliteratum%253Aachs%26aulast%3DVidoni%26aufirst%3DC.%26aulast%3DFollo%26aufirst%3DC.%26aulast%3DSavino%26aufirst%3DM.%26aulast%3DMelone%26aufirst%3DM.%2BA.%2BB.%26aulast%3DIsidoro%26aufirst%3DC.%26atitle%3DThe%2520Role%2520of%2520Cathepsin%2520D%2520in%2520the%2520Pathogenesis%2520of%2520Human%2520Neurodegenerative%2520Disorders%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D845%26epage%3D870%26doi%3D10.1002%2Fmed.21394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aghdassi, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinheckel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayerle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerch, M. M.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D Regulates Cathepsin B Activation and Disease Severity Predominantly in Inflammatory Cells during Experimental Pancreatitis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1029</span>, <span class="refDoi"> DOI: 10.1074/jbc.m117.814772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1074%2Fjbc.m117.814772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=29229780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=1018-1029&author=A.+A.+Aghdassiauthor=D.+S.+Johnauthor=M.+Sendlerauthor=F.+U.+Weissauthor=T.+Reinheckelauthor=J.+Mayerleauthor=M.+M.+Lerch&title=Cathepsin+D+Regulates+Cathepsin+B+Activation+and+Disease+Severity+Predominantly+in+Inflammatory+Cells+during+Experimental+Pancreatitis&doi=10.1074%2Fjbc.m117.814772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D regulates cathepsin B activation and disease severity predominantly in inflammatory cells during experimental pancreatitis</span></div><div class="casAuthors">Aghdassi, Ali A.; John, Daniel S.; Sendler, Matthias; Weiss, F. Ulrich; Reinheckel, Thomas; Mayerle, Julia; Lerch, Markus M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1018-1029</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Acute pancreatitis is a complex disorder involving both premature intracellular protease activation and inflammatory cell invasion.  An initiating event is the intracellular activation of trypsinogen by cathepsin B (CTSB), which can be induced directly via G protein-coupled receptors on acinar cells or through inflammatory cells.  Here, we studied CTSB regulation by another lysosomal hydrolase, cathepsin D (CTSD), using mice with a complete (CTSD-/-) or pancreas-specific conditional CTSD knockout (KO) (CTSDf/f/p48Cre/+).  We induced acute pancreatitis by repeated caerulein injections and isolated acinar and bone marrow cells for ex vivo studies.  Supramaximal caerulein stimulation induced subcellular redistribution of CTSD from the lysosomal to the zymogen-contg. subcellular compartment of acinar cells and activation of CTSD, CTSB, and trypsinogen.  Of note, the CTSD KO greatly reduced CTSB and trypsinogen activation in acinar cells, and CTSD directly activated CTSB but not trypsinogen in vitro.  During pancreatitis in pancreas-specific CTSDf/f/p48Cre/+ animals, markers of severity were reduced only at 1 h, whereas in the complete KO, this effect also included the late disease phase (8 h), indicating an important effect of extra-acinar CTSD on course of the disease.  CTSD-/- leukocytes exhibited reduced cytokine release after lipopolysaccharide (LPS) stimulation, and CTSD KO also reduced caspase-3 activation and apoptosis in acinar cells stimulated with the intestinal hormone cholecystokinin.  In summary, CTSD is expressed in pancreatic acinar and inflammatory cells, undergoes subcellular redistribution and activation during exptl. pancreatitis, and regulates disease severity by potently activating CTSB.  Its impact is only minimal and transient in the early, acinar cell-dependent phase of pancreatitis and much greater in the later, inflammatory cell-dependent phase of the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5K4-R8ZsIfrVg90H21EOLACvtfcHk0lgUI08xlaAlBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjsrs%253D&md5=a79fd4fd21337c3fc81daa9cbc63db84</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m117.814772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m117.814772%26sid%3Dliteratum%253Aachs%26aulast%3DAghdassi%26aufirst%3DA.%2BA.%26aulast%3DJohn%26aufirst%3DD.%2BS.%26aulast%3DSendler%26aufirst%3DM.%26aulast%3DWeiss%26aufirst%3DF.%2BU.%26aulast%3DReinheckel%26aufirst%3DT.%26aulast%3DMayerle%26aufirst%3DJ.%26aulast%3DLerch%26aufirst%3DM.%2BM.%26atitle%3DCathepsin%2520D%2520Regulates%2520Cathepsin%2520B%2520Activation%2520and%2520Disease%2520Severity%2520Predominantly%2520in%2520Inflammatory%2520Cells%2520during%2520Experimental%2520Pancreatitis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D1018%26epage%3D1029%26doi%3D10.1074%2Fjbc.m117.814772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liaudet-Coopman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaujouin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derocq, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glondu-Lassis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent-Matha, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prébois, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rochefort, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vignon, F.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D: Newly Discovered Functions of a Long-Standing Aspartic Protease in Cancer and Apoptosis</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>237</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2005.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.canlet.2005.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=16046058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD28XltF2qsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=237&publication_year=2006&pages=167-179&author=E.+Liaudet-Coopmanauthor=M.+Beaujouinauthor=D.+Derocqauthor=M.+Garciaauthor=M.+Glondu-Lassisauthor=V.+Laurent-Mathaauthor=C.+Pr%C3%A9boisauthor=H.+Rochefortauthor=F.+Vignon&title=Cathepsin+D%3A+Newly+Discovered+Functions+of+a+Long-Standing+Aspartic+Protease+in+Cancer+and+Apoptosis&doi=10.1016%2Fj.canlet.2005.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D: Newly discovered functions of a long-standing aspartic protease in cancer and apoptosis</span></div><div class="casAuthors">Liaudet-Coopman, Emmanuelle; Beaujouin, Melanie; Derocq, Danielle; Garcia, Marcel; Glondu-Lassis, Murielle; Laurent-Matha, Valerie; Prebois, Christine; Rochefort, Henri; Vignon, Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">237</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-179</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The lysosomal aspartic protease cathepsin D (cath-D) is over-expressed and hyper-secreted by epithelial breast cancer cells.  This protease is an independent marker of poor prognosis in breast cancer being correlated with the incidence of clin. metastasis.  Cath-D over-expression stimulates tumorigenicity and metastasis.  Indeed it plays an essential role in the multiple steps of tumor progression, in stimulating cancer cell proliferation, fibroblast outgrowth and angiogenesis, as well as in inhibiting tumor apoptosis.  A mutated cath-D devoid of catalytic activity still proved mitogenic for cancer, endothelial and fibroblastic cells, suggesting an extra-cellular mode of action of cath-D involving a triggering, either directly or indirectly, of an as yet unidentified cell surface receptor.  Cath-D is also a key mediator of induced-apoptosis and its proteolytic activity has been involved generally in this event.  During apoptosis, mature lysosomal cath-D is translocated to the cytosol.  Since cath-D is one of the lysosomal enzymes which requires a more acidic pH to be proteolytically-active relative to the cysteine lysosomal enzymes, such as cath-B and -L, it is open to question whether cytosolic cath-D might be able to cleave substrate(s) implicated in the apoptotic cascade.  This review summarises our current knowledge on cath-D action in cancer progression and metastasis, as well as its dual function in apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFJdjUCIwta7Vg90H21EOLACvtfcHk0lgUI08xlaAlBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltF2qsbc%253D&md5=eb7be2df7685a6574ca0b161a6c9716d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2005.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2005.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DLiaudet-Coopman%26aufirst%3DE.%26aulast%3DBeaujouin%26aufirst%3DM.%26aulast%3DDerocq%26aufirst%3DD.%26aulast%3DGarcia%26aufirst%3DM.%26aulast%3DGlondu-Lassis%26aufirst%3DM.%26aulast%3DLaurent-Matha%26aufirst%3DV.%26aulast%3DPr%25C3%25A9bois%26aufirst%3DC.%26aulast%3DRochefort%26aufirst%3DH.%26aulast%3DVignon%26aufirst%3DF.%26atitle%3DCathepsin%2520D%253A%2520Newly%2520Discovered%2520Functions%2520of%2520a%2520Long-Standing%2520Aspartic%2520Protease%2520in%2520Cancer%2520and%2520Apoptosis%26jtitle%3DCancer%2520Lett.%26date%3D2006%26volume%3D237%26spage%3D167%26epage%3D179%26doi%3D10.1016%2Fj.canlet.2005.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetvicka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fusek, M.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D—Many Functions of One Aspartic Protease</span>. <i>Crit. Rev. Oncol. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1016/j.critrevonc.2008.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.critrevonc.2008.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=18396408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADC%252BD1crnsVehsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=12-28&author=P.+Benesauthor=V.+Vetvickaauthor=M.+Fusek&title=Cathepsin+D%E2%80%94Many+Functions+of+One+Aspartic+Protease&doi=10.1016%2Fj.critrevonc.2008.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D--many functions of one aspartic protease</span></div><div class="casAuthors">Benes Petr; Vetvicka Vaclav; Fusek Martin</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-28</span>
        ISSN:<span class="NLM_cas:issn">1040-8428</span>.
    </div><div class="casAbstract">For years, it has been held that cathepsin D (CD) is involved in rather non-specific protein degradation in a strongly acidic milieu of lysosomes.  Studies with CD knock-out mice revealed that CD is not necessary for embryonal development, but it is indispensable for postnatal tissue homeostasis.  Mutation that abolishes CD enzymatic activity causes neuronal ceroid lipofuscinosis (NCL) characterized by severe neurodegeneration, developmental regression, visual loss and epilepsy in both animals and humans.  In the last decade, however, an increasing number of studies demonstrated that enzymatic function of CD is not restricted solely to acidic milieu of lysosomes with important consequences in regulation of apoptosis.  In addition to CD enzymatic activity, it has been shown that apoptosis is also regulated by catalytically inactive mutants of CD which suggests that CD interacts with other important molecules and influences cell signaling.  Moreover, procathepsin D (pCD), secreted from cancer cells, acts as a mitogen on both cancer and stromal cells and stimulates their pro-invasive and pro-metastatic properties.  Numerous studies found that pCD/CD level represents an independent prognostic factor in a variety of cancers and is therefore considered to be a potential target of anti-cancer therapy.  Studies dealing with functions of cathepsin D are complicated by the fact that there are several simultaneous forms of CD in a cell-pCD, intermediate enzymatically active CD and mature heavy and light chain CD.  It became evident that these forms may differently regulate the above-mentioned processes.  In this article, we review the possible functions of CD and its various forms in cells and organisms during physiological and pathological conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS9gz1Ed--NTiEJCPKhJPuqfW6udTcc2eYOnU8EBP1OGbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1crnsVehsw%253D%253D&md5=eb34f9d185621ee4b845685ec1efa125</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2008.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2008.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DBenes%26aufirst%3DP.%26aulast%3DVetvicka%26aufirst%3DV.%26aulast%3DFusek%26aufirst%3DM.%26atitle%3DCathepsin%2520D%25E2%2580%2594Many%2520Functions%2520of%2520One%2520Aspartic%2520Protease%26jtitle%3DCrit.%2520Rev.%2520Oncol.%2520Hematol.%26date%3D2008%26volume%3D68%26spage%3D12%26epage%3D28%26doi%3D10.1016%2Fj.critrevonc.2008.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Look, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolt-de
Vries, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer-van Gelder, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Putten, W. L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klijn, J. G. M.</span></span> <span> </span><span class="NLM_article-title">Cathepsin-D in Primary Breast Cancer: Prognostic Evaluation Involving 2810 Patients</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">307</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6690048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1038%2Fsj.bjc.6690048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=9888472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADyaK1M7gtlGntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1999&pages=300-307&author=J.+A.+Foekensauthor=M.+P.+Lookauthor=J.+B.+Bolt-de%0AVriesauthor=M.+E.+Meijer-van+Gelderauthor=W.+L.+J.+van+Puttenauthor=J.+G.+M.+Klijn&title=Cathepsin-D+in+Primary+Breast+Cancer%3A+Prognostic+Evaluation+Involving+2810+Patients&doi=10.1038%2Fsj.bjc.6690048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin-D in primary breast cancer: prognostic evaluation involving 2810 patients</span></div><div class="casAuthors">Foekens J A; Look M P; Bolt-de Vries J; Meijer-van Gelder M E; van Putten W L; Klijn J G</div><div class="citationInfo"><span class="NLM_cas:title">British journal of cancer</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-7</span>
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    </div><div class="casAbstract">There is controversy regarding the prognostic value of cathepsin-D in primary breast cancer.  An increased level of cathepsin-D in tumour extracts has been found to be associated with a poor relapse-free and overall survival.  Studies performed with immunohistochemistry or Western blotting have produced diverse results.  We have analysed 2810 cytosolic extracts obtained from human primary breast tumours for cathepsin-D expression, and have correlated their levels with prognosis.  The median follow-up of the patients still alive was 88 months.  Patients with high cathepsin-D levels had a significantly worse relapse-free and overall survival, also in multivariate analysis (P < 0.0001).  Adjuvant therapy which was associated with an improved prognosis in node-positive patients in univariate analysis, also significantly added to the multivariate models for relapse-free and overall survival.  There were no statistically significant interactions between the levels of cathepsin-D and any of the classical prognostic factors in analysis for relapse-free survival, suggesting that the prognostic value of cathepsin-D is not different in the various subgroups of patients.  Indeed, multivariate analyses in subgroups of node-negative and -positive patients, pre- and post-menopausal patients, and their combinations, showed that tumours with high cathepsin-D values had a significantly poor relapse-free survival, with relative hazard rates ranging from 1.3 to 1.5, compared with tumours with low cathepsin-D levels.  The results presented here on 2810 patients confirm that high cytosolic cathepsin-D values are associated with poor prognosis in human primary breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSvgQsgnUMs2JotVjB3IRIyfW6udTcc2eYOnU8EBP1OGbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1M7gtlGntQ%253D%253D&md5=afd3571f4ccb8e410d7bb119c0a7c1d0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6690048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6690048%26sid%3Dliteratum%253Aachs%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26aulast%3DLook%26aufirst%3DM.%2BP.%26aulast%3DBolt-de%2BVries%26aufirst%3DJ.%2BB.%26aulast%3DMeijer-van%2BGelder%26aufirst%3DM.%2BE.%26aulast%3Dvan%2BPutten%26aufirst%3DW.%2BL.%2BJ.%26aulast%3DKlijn%26aufirst%3DJ.%2BG.%2BM.%26atitle%3DCathepsin-D%2520in%2520Primary%2520Breast%2520Cancer%253A%2520Prognostic%2520Evaluation%2520Involving%25202810%2520Patients%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D1999%26volume%3D79%26spage%3D300%26epage%3D307%26doi%3D10.1038%2Fsj.bjc.6690048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrandina, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scambia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti Panici, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancuso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messori, A.</span></span> <span> </span><span class="NLM_article-title">Relationship between Cathepsin-D Content and Disease-Free Survival in Node-Negative Breast Cancer Patients: A Meta-Analysis</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1038/bjc.1997.442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1038%2Fbjc.1997.442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=9303368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADyaK2svkslKhug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=1997&pages=661-666&author=G.+Ferrandinaauthor=G.+Scambiaauthor=F.+Bardelliauthor=P.+Benedetti+Paniciauthor=S.+Mancusoauthor=A.+Messori&title=Relationship+between+Cathepsin-D+Content+and+Disease-Free+Survival+in+Node-Negative+Breast+Cancer+Patients%3A+A+Meta-Analysis&doi=10.1038%2Fbjc.1997.442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Relationship between cathepsin-D content and disease-free survival in node-negative breast cancer patients: a meta-analysis</span></div><div class="casAuthors">Ferrandina G; Scambia G; Bardelli F; Benedetti Panici P; Mancuso S; Messori A</div><div class="citationInfo"><span class="NLM_cas:title">British journal of cancer</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">661-6</span>
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    </div><div class="casAbstract">Several reports have evaluated the correlation between cathepsin-D and overall survival or disease-free survival in node-negative breast cancer patients.  Because conflicting data have so far been reported, a meta-analysis was conducted to clarify this problem.  Eleven studies were included in our meta-analysis (total of 2690 patients).  A specific meta-analytical methodology for censored data was used, and disease-free survival was the primary end point.  Patients with low cathepsin-D levels had a significantly better disease-free survival than patients with high cathepsin-D values (meta-analytical odds ratio from 0.59 to 0.60 over the interval from 1 to 7 years).  A secondary meta-analysis conducted exclusively on the data from eight studies based on cytosol assay gave substantially similar results.  One limitation of our study is that the cut-off values to define high and low cathepsin-D concentrations were not identical in the various studies included in our meta-analysis (range from 20 to 78 pmol mg(-1) protein), thus introducing a possible bias in the statistical analysis of the data.  However, a simulation based on the well-accepted method of the so-called publication bias showed that more than 100 null studies would be required to lead our results to a statistical level of non-significance.  Considering the results of our meta-analysis, we conclude that the data presently available confirm a statistically significant association between high cathepsin-D values and poor disease-free survival in node-negative breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDXEt60c3tNsjl5L8hwNX0fW6udTcc2eYOnU8EBP1OGbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2svkslKhug%253D%253D&md5=e0726118891dec91f1e4ffdf964f571a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fbjc.1997.442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.1997.442%26sid%3Dliteratum%253Aachs%26aulast%3DFerrandina%26aufirst%3DG.%26aulast%3DScambia%26aufirst%3DG.%26aulast%3DBardelli%26aufirst%3DF.%26aulast%3DBenedetti%2BPanici%26aufirst%3DP.%26aulast%3DMancuso%26aufirst%3DS.%26aulast%3DMessori%26aufirst%3DA.%26atitle%3DRelationship%2520between%2520Cathepsin-D%2520Content%2520and%2520Disease-Free%2520Survival%2520in%2520Node-Negative%2520Breast%2520Cancer%2520Patients%253A%2520A%2520Meta-Analysis%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D1997%26volume%3D76%26spage%3D661%26epage%3D666%26doi%3D10.1038%2Fbjc.1997.442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashraf, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansouri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent-Matha, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guiu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derocq, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, H.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delpech, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarlier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugnière, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landomiel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourquard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achour, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fruitier-Arnaudin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshayes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turtoi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poupon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simony-Lafontaine, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boissière-Michot, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernex, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacot, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">du Manoir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theillet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouget, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro-Teulon, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefoy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pèlegrin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chardès, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martineau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaudet-Coopman, E.</span></span> <span> </span><span class="NLM_article-title">Immunotherapy of Triple-Negative Breast Cancer with Cathepsin D-Targeting Antibodies</span>. <i>J. Immunother. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">29</span>, <span class="refDoi"> DOI: 10.1186/s40425-019-0498-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1186%2Fs40425-019-0498-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=30717773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADC%252BB3cjovVyksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=29&author=Y.+Ashrafauthor=H.+Mansouriauthor=V.+Laurent-Mathaauthor=L.+B.+Alcarazauthor=P.+Rogerauthor=S.+Guiuauthor=D.+Derocqauthor=G.+Robinauthor=H.-A.+Michaudauthor=H.+Delpechauthor=M.+Jarlierauthor=M.+Pugni%C3%A8reauthor=B.+Robertauthor=A.+Puelauthor=L.+Martinauthor=F.+Landomielauthor=T.+Bourquardauthor=O.+Achourauthor=I.+Fruitier-Arnaudinauthor=A.+Pichardauthor=E.+Deshayesauthor=A.+Turtoiauthor=A.+Pouponauthor=J.+Simony-Lafontaineauthor=F.+Boissi%C3%A8re-Michotauthor=N.+Pirotauthor=F.+Bernexauthor=W.+Jacotauthor=S.+du+Manoirauthor=C.+Theilletauthor=J.-P.+Pougetauthor=I.+Navarro-Teulonauthor=N.+Bonnefoyauthor=A.+P%C3%A8legrinauthor=T.+Chard%C3%A8sauthor=P.+Martineauauthor=E.+Liaudet-Coopman&title=Immunotherapy+of+Triple-Negative+Breast+Cancer+with+Cathepsin+D-Targeting+Antibodies&doi=10.1186%2Fs40425-019-0498-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Immunotherapy of triple-negative breast cancer with cathepsin D-targeting antibodies</span></div><div class="casAuthors">Ashraf Yahya; Mansouri Hanane; Laurent-Matha Valerie; Alcaraz Lindsay B; Roger Pascal; Guiu Severine; Derocq Danielle; Robin Gautier; Michaud Henri-Alexandre; Delpech Helene; Pugniere Martine; Robert Bruno; Puel Anthony; Martin Lucie; Pichard Alexandre; Deshayes Emmanuel; Turtoi Andrei; Jacot William; du Manoir Stanislas; Theillet Charles; Pouget Jean-Pierre; Navarro-Teulon Isabelle; Bonnefoy Nathalie; Pelegrin Andre; Chardes Thierry; Martineau Pierre; Liaudet-Coopman Emmanuelle; Roger Pascal; Guiu Severine; Jacot William; Jarlier Marta; Landomiel Flavie; Bourquard Thomas; Poupon Anne; Achour Oussama; Fruitier-Arnaudin Ingrid; Simony-Lafontaine Joelle; Boissiere-Michot Florence; Jacot William; Pirot Nelly; Bernex Florence</div><div class="citationInfo"><span class="NLM_cas:title">Journal for immunotherapy of cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Triple-negative breast cancer (TNBC) treatment is currently restricted to chemotherapy.  Hence, tumor-specific molecular targets and/or alternative therapeutic strategies for TNBC are urgently needed.  Immunotherapy is emerging as an exciting treatment option for TNBC patients.  The aspartic protease cathepsin D (cath-D), a marker of poor prognosis in breast cancer (BC), is overproduced and hypersecreted by human BC cells.  This study explores whether cath-D is a tumor cell-associated extracellular biomarker and a potent target for antibody-based therapy in TNBC.  METHODS:  Cath-D prognostic value and localization was evaluated by transcriptomics, proteomics and immunohistochemistry in TNBC.  First-in-class anti-cath-D human scFv fragments binding to both human and mouse cath-D were generated using phage display and cloned in the human IgG1 λ format (F1 and E2).  Anti-cath-D antibody biodistribution, antitumor efficacy and in vivo underlying mechanisms were investigated in TNBC MDA-MB-231 tumor xenografts in nude mice.  Antitumor effect was further assessed in TNBC patient-derived xenografts (PDXs).  RESULTS:  High CTSD mRNA levels correlated with shorter recurrence-free survival in TNBC, and extracellular cath-D was detected in the tumor microenvironment, but not in matched normal breast stroma.  Anti-cath-D F1 and E2 antibodies accumulated in TNBC MDA-MB-231 tumor xenografts, inhibited tumor growth and improved mice survival without apparent toxicity.  The Fc function of F1, the best antibody candidate, was essential for maximal tumor inhibition in the MDA-MB-231 model.  Mechanistically, F1 antitumor response was triggered through natural killer cell activation via IL-15 upregulation, associated with granzyme B and perforin production, and the release of antitumor IFNγ cytokine.  The F1 antibody also prevented the tumor recruitment of immunosuppressive tumor-associated macrophages M2 and myeloid-derived suppressor cells, a specific effect associated with a less immunosuppressive tumor microenvironment highlighted by TGFβ decrease.  Finally, the antibody F1 inhibited tumor growth of two TNBC PDXs, isolated from patients resistant or not to neo-adjuvant chemotherapy.  CONCLUSION:  Cath-D is a tumor-specific extracellular target in TNBC suitable for antibody-based therapy.  Immunomodulatory antibody-based strategy against cath-D is a promising immunotherapy to treat patients with TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBu63j-Oqd97P3FVUi_gf8fW6udTcc2eZHg7ShqhgWJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjovVyksA%253D%253D&md5=ccc512eb50ec1cf0180fa8c588dbab6a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1186%2Fs40425-019-0498-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40425-019-0498-z%26sid%3Dliteratum%253Aachs%26aulast%3DAshraf%26aufirst%3DY.%26aulast%3DMansouri%26aufirst%3DH.%26aulast%3DLaurent-Matha%26aufirst%3DV.%26aulast%3DAlcaraz%26aufirst%3DL.%2BB.%26aulast%3DRoger%26aufirst%3DP.%26aulast%3DGuiu%26aufirst%3DS.%26aulast%3DDerocq%26aufirst%3DD.%26aulast%3DRobin%26aufirst%3DG.%26aulast%3DMichaud%26aufirst%3DH.-A.%26aulast%3DDelpech%26aufirst%3DH.%26aulast%3DJarlier%26aufirst%3DM.%26aulast%3DPugni%25C3%25A8re%26aufirst%3DM.%26aulast%3DRobert%26aufirst%3DB.%26aulast%3DPuel%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DLandomiel%26aufirst%3DF.%26aulast%3DBourquard%26aufirst%3DT.%26aulast%3DAchour%26aufirst%3DO.%26aulast%3DFruitier-Arnaudin%26aufirst%3DI.%26aulast%3DPichard%26aufirst%3DA.%26aulast%3DDeshayes%26aufirst%3DE.%26aulast%3DTurtoi%26aufirst%3DA.%26aulast%3DPoupon%26aufirst%3DA.%26aulast%3DSimony-Lafontaine%26aufirst%3DJ.%26aulast%3DBoissi%25C3%25A8re-Michot%26aufirst%3DF.%26aulast%3DPirot%26aufirst%3DN.%26aulast%3DBernex%26aufirst%3DF.%26aulast%3DJacot%26aufirst%3DW.%26aulast%3Ddu%2BManoir%26aufirst%3DS.%26aulast%3DTheillet%26aufirst%3DC.%26aulast%3DPouget%26aufirst%3DJ.-P.%26aulast%3DNavarro-Teulon%26aufirst%3DI.%26aulast%3DBonnefoy%26aufirst%3DN.%26aulast%3DP%25C3%25A8legrin%26aufirst%3DA.%26aulast%3DChard%25C3%25A8s%26aufirst%3DT.%26aulast%3DMartineau%26aufirst%3DP.%26aulast%3DLiaudet-Coopman%26aufirst%3DE.%26atitle%3DImmunotherapy%2520of%2520Triple-Negative%2520Breast%2520Cancer%2520with%2520Cathepsin%2520D-Targeting%2520Antibodies%26jtitle%3DJ.%2520Immunother.%2520Cancer%26date%3D2019%26volume%3D7%26spage%3D29%26doi%3D10.1186%2Fs40425-019-0498-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derocq, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prébois, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent-Matha, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattingre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaudet-Coopman, E.</span></span> <span> </span><span class="NLM_article-title">Pathophysiological Functions of Cathepsin D: Targeting Its Catalytic Activity versus Its Protein Binding Activity?</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1635</span>– <span class="NLM_lpage">1643</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2010.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.biochi.2010.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=20493920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2isLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2010&pages=1635-1643&author=O.+Massonauthor=A.-S.+Bachauthor=D.+Derocqauthor=C.+Pr%C3%A9boisauthor=V.+Laurent-Mathaauthor=S.+Pattingreauthor=E.+Liaudet-Coopman&title=Pathophysiological+Functions+of+Cathepsin+D%3A+Targeting+Its+Catalytic+Activity+versus+Its+Protein+Binding+Activity%3F&doi=10.1016%2Fj.biochi.2010.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pathophysiological functions of cathepsin D: Targeting its catalytic activity versus its protein binding activity?</span></div><div class="casAuthors">Masson, Olivier; Bach, Anne-Sophie; Derocq, Danielle; Prebois, Christine; Laurent-Matha, Valerie; Pattingre, Sophie; Liaudet-Coopman, Emmanuelle</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1635-1643</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The lysosomal aspartic protease cathepsin D (cath-D) is overexpressed and hyper-secreted by epithelial breast cancer cells.  This protease is an independent marker of poor prognosis in breast cancer as it is correlated with the incidence of clin. metastasis.  In normal cells, cath-D is localized in intracellular vesicles (lysosomes and endosomes).  In cancer cells, overexpressed cath-D accumulates in cells, where it may affect their degradative capacities, and the pro-enzyme is hyper-secreted in the tumor micro-environment.  In addn., during apoptosis, lysosomal cath-D is released into the cytosol, where it may interact with and/or cleave pro-apoptotic, anti-apoptotic, or nuclear proteins.  Several studies have shown that cath-D affects various different steps in tumor progression and metastasis.  Cath-D stimulates cancer cell growth in an autocrine manner, and also cath-D plays a crucial paracrine role in the tumor micro-environment by stimulating fibroblast outgrowth and tumor angiogenesis.  A mutant D231N-cath-D, which is devoid of catalytic activity, remained mitogenic, indicating an addnl. action of cath-D by protein-protein interaction.  Targeting cath-D in cancer may require the use of inhibitors of its catalytic activity, but also the development of new tools to inhibit its protein binding functions.  Thus, elucidation of the mechanism of action of cath-D is crucial if an appropriate strategy is to be developed to target this protease in cancer.  The discovery of new physiol. substrates of cath-D using proteomic approaches can be expected to generate new crit. targets.  The aim of this review is to describe the roles of the cath-D protease in cancer progression and metastasis, as well as its function in apoptosis, and to discuss how it can be targeted in cancer by inhibiting its proteolytic activity and/or its binding protein activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjTXmyIl5WaLVg90H21EOLACvtfcHk0li05SH8q8D_8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2isLvF&md5=544dfe4c940fc1a18e0cd710962ca6c3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2010.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2010.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DMasson%26aufirst%3DO.%26aulast%3DBach%26aufirst%3DA.-S.%26aulast%3DDerocq%26aufirst%3DD.%26aulast%3DPr%25C3%25A9bois%26aufirst%3DC.%26aulast%3DLaurent-Matha%26aufirst%3DV.%26aulast%3DPattingre%26aufirst%3DS.%26aulast%3DLiaudet-Coopman%26aufirst%3DE.%26atitle%3DPathophysiological%2520Functions%2520of%2520Cathepsin%2520D%253A%2520Targeting%2520Its%2520Catalytic%2520Activity%2520versus%2520Its%2520Protein%2520Binding%2520Activity%253F%26jtitle%3DBiochimie%26date%3D2010%26volume%3D92%26spage%3D1635%26epage%3D1643%26doi%3D10.1016%2Fj.biochi.2010.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maibaum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahuel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, N.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasetti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rüger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göschke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stutz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigollier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cumin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedigian, M. P.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Aliskiren, a Novel Orally Effective Renin Inhibitor</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>308</i></span>,  <span class="NLM_fpage">698</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1016/s0006-291x(03)01451-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fs0006-291x%2803%2901451-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=12927775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVGkt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=308&publication_year=2003&pages=698-705&author=J.+M.+Woodauthor=J.+Maibaumauthor=J.+Rahuelauthor=M.+G.+Gr%C3%BCtterauthor=N.-C.+Cohenauthor=V.+Rasettiauthor=H.+R%C3%BCgerauthor=R.+G%C3%B6schkeauthor=S.+Stutzauthor=W.+Fuhrerauthor=W.+Schillingauthor=P.+Rigollierauthor=Y.+Yamaguchiauthor=F.+Cuminauthor=H.-P.+Baumauthor=C.+R.+Schnellauthor=P.+Heroldauthor=R.+Mahauthor=C.+Jensenauthor=E.+O%E2%80%99Brienauthor=A.+Stantonauthor=M.+P.+Bedigian&title=Structure-Based+Design+of+Aliskiren%2C+a+Novel+Orally+Effective+Renin+Inhibitor&doi=10.1016%2Fs0006-291x%2803%2901451-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of aliskiren, a novel orally effective renin inhibitor</span></div><div class="casAuthors">Wood, Jeanette M.; Maibaum, Juergen; Rahuel, Joseph; Gruetter, Markus G.; Cohen, Nissim-Claude; Rasetti, Vittorio; Rueger, Heinrich; Goeschke, Richard; Stutz, Stefan; Fuhrer, Walter; Schilling, Walter; Rigollier, Pascal; Yamaguchi, Yasuchika; Cumin, Frederic; Baum, Hans-Peter; Schnell, Christian R.; Herold, Peter; Mah, Robert; Jensen, Chris; O'Brien, Eoin; Stanton, Alice; Bedigian, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">308</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">698-705</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">Hypertension is a major risk factor for cardiovascular diseases such as stroke, myocardial infarction, and heart failure, the leading causes of death in the Western world.  Inhibitors of the renin-angiotensin system (RAS) have proven to be successful treatments for hypertension.  As renin specifically catalyzes the rate-limiting step of the RAS, it represents the optimal target for RAS inhibition.  Several peptide-like renin inhibitors have been synthesized previously, but poor pharmacokinetic properties meant that these compds. were not clin. useful.  We employed a combination of mol. modeling and crystallog. structure anal. to design renin inhibitors lacking the extended peptide-like backbone of earlier inhibitors, for improved pharmacokinetic properties.  This led to the discovery of aliskiren, a highly potent and selective inhibitor of human renin in vitro, and in vivo; once-daily oral doses of aliskiren inhibit renin and lower blood pressure in sodium-depleted marmosets and hypertensive human patients.  Aliskiren represents the first in a novel class of renin inhibitors with the potential for treatment of hypertension and related cardiovascular diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV5YmV3cdr0LVg90H21EOLACvtfcHk0lgpFQHO_ekkzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVGkt7k%253D&md5=20b71ca3ccfe2324752cb2bb541d9ba2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fs0006-291x%2803%2901451-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0006-291x%252803%252901451-7%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DJ.%2BM.%26aulast%3DMaibaum%26aufirst%3DJ.%26aulast%3DRahuel%26aufirst%3DJ.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DM.%2BG.%26aulast%3DCohen%26aufirst%3DN.-C.%26aulast%3DRasetti%26aufirst%3DV.%26aulast%3DR%25C3%25BCger%26aufirst%3DH.%26aulast%3DG%25C3%25B6schke%26aufirst%3DR.%26aulast%3DStutz%26aufirst%3DS.%26aulast%3DFuhrer%26aufirst%3DW.%26aulast%3DSchilling%26aufirst%3DW.%26aulast%3DRigollier%26aufirst%3DP.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DCumin%26aufirst%3DF.%26aulast%3DBaum%26aufirst%3DH.-P.%26aulast%3DSchnell%26aufirst%3DC.%2BR.%26aulast%3DHerold%26aufirst%3DP.%26aulast%3DMah%26aufirst%3DR.%26aulast%3DJensen%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DE.%26aulast%3DStanton%26aufirst%3DA.%26aulast%3DBedigian%26aufirst%3DM.%2BP.%26atitle%3DStructure-Based%2520Design%2520of%2520Aliskiren%252C%2520a%2520Novel%2520Orally%2520Effective%2520Renin%2520Inhibitor%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2003%26volume%3D308%26spage%3D698%26epage%3D705%26doi%3D10.1016%2Fs0006-291x%2803%2901451-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prato, G.</span></span> <span> </span><span class="NLM_article-title">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5172</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01697</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01697" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5172-5208&author=A.+K.+Ghoshauthor=H.+L.+Osswaldauthor=G.+Prato&title=Recent+Progress+in+the+Development+of+HIV-1+Protease+Inhibitors+for+the+Treatment+of+HIV%2FAIDS&doi=10.1021%2Facs.jmedchem.5b01697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span></div><div class="casAuthors">Ghosh, Arun K.; Osswald, Heather L.; Prato, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5172-5208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 protease inhibitors continue to play an important role in the treatment of HIV/AIDS, transforming this deadly ailment into a more manageable chronic infection.  Over the years, intensive research has led to a variety of approved protease inhibitors for the treatment of HIV/AIDS.  In this review, we outline current drug design and medicinal chem. efforts toward the development of next-generation protease inhibitors beyond the currently approved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4DisuSRbQXbVg90H21EOLACvtfcHk0ljwKzWRxV8vJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D&md5=e0994f3d84ab627d3ac3f757e8d11725</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01697%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DPrato%26aufirst%3DG.%26atitle%3DRecent%2520Progress%2520in%2520the%2520Development%2520of%2520HIV-1%2520Protease%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520HIV%252FAIDS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5172%26epage%3D5208%26doi%3D10.1021%2Facs.jmedchem.5b01697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ersmark, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuelsson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallberg, A.</span></span> <span> </span><span class="NLM_article-title">Plasmepsins as Potential Targets for New Antimalarial Therapy</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">626</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1002/med.20082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fmed.20082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=16838300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsFGru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=626-666&author=K.+Ersmarkauthor=B.+Samuelssonauthor=A.+Hallberg&title=Plasmepsins+as+Potential+Targets+for+New+Antimalarial+Therapy&doi=10.1002%2Fmed.20082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Plasmepsins as potential targets for new antimalarial therapy</span></div><div class="casAuthors">Ersmark, Karolina; Samuelsson, Bertil; Hallberg, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">626-666</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Malaria is one of the major diseases in the world.  Due to the rapid spread of parasite resistance to available antimalarial drugs there is an urgent need for new antimalarials with novel mechanisms of action.  Several promising targets for drug intervention have been revealed in recent years.  This review addresses the parasitic aspartic proteases termed plasmepsins (Plms) that are involved in the Hb catabolism that occurs during the erythrocytic stage of the malarial parasite life cycle.  Four Plasmodium species are responsible for human malaria; P. vivax, P. ovale, P. malariae, and P. falciparum.  This review focuses on inhibitors of the Hb-degrading plasmepsins of the most lethal species, P. falciparum; Plm I, Plm II, Plm IV, and histo-aspartic protease (HAP).  Previously, Plm II has attracted the most attention.  With the identification and characterization of new plasmepsins and the results from recent plasmepsin knockout studies, it now seems clear that in order to achieve high-antiparasitic activities in P. falciparum-infected erythrocytes it is necessary to inhibit several of the Hb-degrading plasmepsins.  Herein we summarize the structure-activity relationships of the Plm I, II, IV, and HAP inhibitors.  These inhibitors represent all classes which, to the best of our knowledge, have been disclosed in journal articles to date.  The 3D structures of inhibitor/plasmepsin II complexes available in the protein data bank are briefly discussed and compared.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwP4wAeM2fT7Vg90H21EOLACvtfcHk0ljwKzWRxV8vJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsFGru7k%253D&md5=fcc65b50fccd246a31d1915c1d8274f9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fmed.20082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20082%26sid%3Dliteratum%253Aachs%26aulast%3DErsmark%26aufirst%3DK.%26aulast%3DSamuelsson%26aufirst%3DB.%26aulast%3DHallberg%26aufirst%3DA.%26atitle%3DPlasmepsins%2520as%2520Potential%2520Targets%2520for%2520New%2520Antimalarial%2520Therapy%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2006%26volume%3D26%26spage%3D626%26epage%3D666%26doi%3D10.1002%2Fmed.20082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coimbra, J. R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marques, D. F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baptista, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, C. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreira, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinis, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvador, J. A. R.</span></span> <span> </span><span class="NLM_article-title">Highlights in BACE1 Inhibitors for Alzheimer’s Disease Treatment</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">178</span>, <span class="refDoi"> DOI: 10.3389/fchem.2018.00178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.3389%2Ffchem.2018.00178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=29881722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisV2mtr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=178&author=J.+R.+M.+Coimbraauthor=D.+F.+F.+Marquesauthor=S.+J.+Baptistaauthor=C.+M.+F.+Pereiraauthor=P.+I.+Moreiraauthor=T.+C.+P.+Dinisauthor=A.+E.+Santosauthor=J.+A.+R.+Salvador&title=Highlights+in+BACE1+Inhibitors+for+Alzheimer%E2%80%99s+Disease+Treatment&doi=10.3389%2Ffchem.2018.00178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Highlights in bace1 inhibitors for alzheimer's disease treatment</span></div><div class="casAuthors">Coimbra, Judite R. M.; Marques, Daniela F. F.; Baptista, Salete J.; Pereira, Claudia M. F.; Moreira, Paula I.; Dinis, Teresa C. P.; Santos, Armanda E.; Salvador, Jorge A. R.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">178/1-178/10</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease (AD) is a severe neurodegenerative disorder and the most common type of dementia in the elderly.  The clin. symptoms of AD include a progressive loss of memory and impairment of cognitive functions interfering with daily life activities.  The main neuropathol. features consist in extracellular amyloid-β (Aβ) plaque deposition and intracellular Neurofibrillary tangles (NFTs) of hyperphosphorylated Tau.  Understanding the pathophysiol. mechanisms that underlie neurodegeneration in AD is essential for rational design of neuroprotective agents able to prevent disease progression.  According to the "Amyloid Cascade Hypothesis" the crit. mol. event in the pathogenesis of AD is the accumulation of Aβ neurotoxic oligomers.  Since the proteolytic processing of Amyloid Precursor Protein (APP) by β-secretase (beta-site APP cleaving enzyme 1, BACE1) is the rate-limiting step in the prodn. of Aβ, this enzyme is considered a major therapeutic target and BACE1 inhibitors have the potential to be disease-modifying drugs for AD treatment.  Therefore, intensive efforts to discover and develop inhibitors that can reach the brain and effectively inhibit BACE1 have been pursued by several groups worldwide.  The aim of this review is to highlight the progress in the discovery of potent and selective small mol. BACE1 inhibitors over the past decade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyV_176RT6cLVg90H21EOLACvtfcHk0lgF5DXp7ftn9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisV2mtr3N&md5=79f20f1bde2dd9993b8b84ba1f188274</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2018.00178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2018.00178%26sid%3Dliteratum%253Aachs%26aulast%3DCoimbra%26aufirst%3DJ.%2BR.%2BM.%26aulast%3DMarques%26aufirst%3DD.%2BF.%2BF.%26aulast%3DBaptista%26aufirst%3DS.%2BJ.%26aulast%3DPereira%26aufirst%3DC.%2BM.%2BF.%26aulast%3DMoreira%26aufirst%3DP.%2BI.%26aulast%3DDinis%26aufirst%3DT.%2BC.%2BP.%26aulast%3DSantos%26aufirst%3DA.%2BE.%26aulast%3DSalvador%26aufirst%3DJ.%2BA.%2BR.%26atitle%3DHighlights%2520in%2520BACE1%2520Inhibitors%2520for%2520Alzheimer%25E2%2580%2599s%2520Disease%2520Treatment%26jtitle%3DFront.%2520Chem.%26date%3D2018%26volume%3D6%26spage%3D178%26doi%3D10.3389%2Ffchem.2018.00178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kick, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoffrey
Skillman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewing, T. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design and Combinatorial Chemistry Yield Low Nanomolar Inhibitors of Cathepsin D</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">307</span>, <span class="refDoi"> DOI: 10.1016/s1074-5521(97)90073-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fs1074-5521%2897%2990073-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=9195867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1ekt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1997&pages=297-307&author=E.+K.+Kickauthor=D.+C.+Roeauthor=A.+Geoffrey%0ASkillmanauthor=G.+Liuauthor=T.+J.+A.+Ewingauthor=Y.+Sunauthor=I.+D.+Kuntzauthor=J.+A.+Ellman&title=Structure-Based+Design+and+Combinatorial+Chemistry+Yield+Low+Nanomolar+Inhibitors+of+Cathepsin+D&doi=10.1016%2Fs1074-5521%2897%2990073-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design and combinatorial chemistry yield low nanomolar inhibitors of cathepsin D</span></div><div class="casAuthors">Kick, Ellen K.; Roe, Diana C.; Skillman, A. Geoffrey; Liu, Guangcheng; Ewing, Todd, J. A.; Sun, Yaxiong; Kuntz, Irwin D.; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">297-307</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">The identification of potent small mol. ligands to receptors and enzymes is one of the major goals of chem. and biol. research.  Two powerful new tools that can be used in these efforts are combinatorial chem. and structure-based design.  Here we address how to join these methods in a design protocol that produces libraries of compds. that are directed against specific macromol. targets.  The aspartyl class of proteases, which is involved in numerous biol. processes, was chosen to demonstrate this effective procedure.  Using cathepsin D, a prototypical aspartyl protease, a no. of low nanomolar inhibitors were rapidly identified.  Although cathepsin D is implicated in a no. of therapeutically relevant processes, potent nonpeptide inhibitors have not been reported previously.  The libraries, synthesized on solid support, displayed nonpeptide functionality about the (hydroxyethyl)amine isostere.  The (hydroxyethyl)amine isostere, which targets the aspartyl protease class, is a stable mimetic of the tetrahedral intermediate of amide hydrolysis.  Structure-based design, using the crystal structure of cathepsin D complexed with the peptide-based natural product pepstatin, was used to select the building blocks for the library synthesis.  The library yielded a 'hit rate' of 6-7% at 1 μM inhibitor concns., with the most potent compd. having a Ki value of 73 nM.  More potent, nonpeptide inhibitors (Ki = 9-15 nM) of cathepsin D were rapidly identified by synthesizing and screening a small second generation library.  The success of these studies clearly demonstrates the power of coupling the complementary methods of combinatorial chem. and structure-based design.  We anticipate that the general approaches described here will be successful for other members of the aspartyl protease class and for many other enzyme classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5II3prqAvlrVg90H21EOLACvtfcHk0lgF5DXp7ftn9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1ekt7s%253D&md5=0ca170cd545b8d34c1c509b2d0b21c95</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fs1074-5521%2897%2990073-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1074-5521%252897%252990073-9%26sid%3Dliteratum%253Aachs%26aulast%3DKick%26aufirst%3DE.%2BK.%26aulast%3DRoe%26aufirst%3DD.%2BC.%26aulast%3DGeoffrey%2BSkillman%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DEwing%26aufirst%3DT.%2BJ.%2BA.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DStructure-Based%2520Design%2520and%2520Combinatorial%2520Chemistry%2520Yield%2520Low%2520Nanomolar%2520Inhibitors%2520of%2520Cathepsin%2520D%26jtitle%3DChem.%2520Biol.%26date%3D1997%26volume%3D4%26spage%3D297%26epage%3D307%26doi%3D10.1016%2Fs1074-5521%2897%2990073-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxena, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luqman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanuja, S. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negi, A. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Gallic Acid Based Naphthophenone Fatty Acid Amides as Cathepsin D Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4603</span>– <span class="NLM_lpage">4608</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2006.06.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.bmcl.2006.06.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=16797987" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsFOltbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=4603-4608&author=V.+Srivastavaauthor=H.+O.+Saxenaauthor=K.+Shankerauthor=J.+K.+Kumarauthor=S.+Luqmanauthor=M.+M.+Guptaauthor=S.+P.+S.+Khanujaauthor=A.+S.+Negi&title=Synthesis+of+Gallic+Acid+Based+Naphthophenone+Fatty+Acid+Amides+as+Cathepsin+D+Inhibitors&doi=10.1016%2Fj.bmcl.2006.06.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of gallic acid based naphthophenone fatty acid amides as cathepsin D inhibitors</span></div><div class="casAuthors">Srivastava, Vandana; Saxena, Hari Om; Shanker, Karuna; Kumar, J. K.; Luqman, Suaib; Gupta, M. M.; Khanuja, S. P. S.; Negi, Arvind S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4603-4608</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Gallic acid, one of the most abundant plant phenolic acids, has been modified to cathepsin D protease inhibitors.  The strategy of modification was proposed basing on some previously reported structure activity relationship (SAR) studies.  The synthesized naphthophenone fatty acid amide derivs. have been evaluated for in vitro cathepsin D inhibition activity.  Two of them have shown significant inhibition activity with IC50 values of 0.06 (I; R = CH2CO2H) and 0.14 μM (I; R = CH2CH=CHCONHBu) as compared against pepstatin (0.0023 μM), the most potent inhibitor known so far.  The study revealed that such attempts on gallic acid based pharmacophores might result in potent inhibitors of cathepsin D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9cRPClG1Im7Vg90H21EOLACvtfcHk0licGdPR6esuXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsFOltbs%253D&md5=7f20d0b8682f88b6b6bc95ee19f503e0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.06.010%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DV.%26aulast%3DSaxena%26aufirst%3DH.%2BO.%26aulast%3DShanker%26aufirst%3DK.%26aulast%3DKumar%26aufirst%3DJ.%2BK.%26aulast%3DLuqman%26aufirst%3DS.%26aulast%3DGupta%26aufirst%3DM.%2BM.%26aulast%3DKhanuja%26aufirst%3DS.%2BP.%2BS.%26aulast%3DNegi%26aufirst%3DA.%2BS.%26atitle%3DSynthesis%2520of%2520Gallic%2520Acid%2520Based%2520Naphthophenone%2520Fatty%2520Acid%2520Amides%2520as%2520Cathepsin%2520D%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D4603%26epage%3D4608%26doi%3D10.1016%2Fj.bmcl.2006.06.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grädler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsaklakidis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werkmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindemann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maskos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of Non-Peptidic Cathepsin D Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4141</span>– <span class="NLM_lpage">4150</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.07.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.bmcl.2014.07.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=25086681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADC%252BC2cbot12ktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=4141-4150&author=U.+Gr%C3%A4dlerauthor=P.+Czodrowskiauthor=C.+Tsaklakidisauthor=M.+Kleinauthor=D.+Werkmannauthor=S.+Lindemannauthor=K.+Maskosauthor=B.+Leuthner&title=Structure-Based+Optimization+of+Non-Peptidic+Cathepsin+D+Inhibitors&doi=10.1016%2Fj.bmcl.2014.07.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based optimization of non-peptidic Cathepsin D inhibitors</span></div><div class="casAuthors">Gradler Ulrich; Czodrowski Paul; Tsaklakidis Christos; Klein Markus; Werkmann Daniela; Lindemann Sven; Leuthner Birgitta; Maskos Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4141-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We discovered a novel series of non-peptidic acylguanidine inhibitors of Cathepsin D as target for osteoarthritis.  The initial HTS-hits were optimized by structure-based design using CatD X-ray structures resulting in single digit nanomolar potency in the biochemical CatD assay.  However, the most potent analogues showed only micromolar activities in an ex vivo glycosaminoglycan (GAG) release assay in bovine cartilage together with low cellular permeability and suboptimal microsomal stability.  This new scaffold can serve as a starting point for further optimization towards in vivo efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJw2Tu4OKgfXT4ugvCDgnZfW6udTcc2eZuj95N8F__yrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbot12ktw%253D%253D&md5=df3353a5f78787ec5a540fa92194a769</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.07.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.07.054%26sid%3Dliteratum%253Aachs%26aulast%3DGr%25C3%25A4dler%26aufirst%3DU.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DTsaklakidis%26aufirst%3DC.%26aulast%3DKlein%26aufirst%3DM.%26aulast%3DWerkmann%26aufirst%3DD.%26aulast%3DLindemann%26aufirst%3DS.%26aulast%3DMaskos%26aufirst%3DK.%26aulast%3DLeuthner%26aufirst%3DB.%26atitle%3DStructure-Based%2520Optimization%2520of%2520Non-Peptidic%2520Cathepsin%2520D%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D4141%26epage%3D4150%26doi%3D10.1016%2Fj.bmcl.2014.07.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kick, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">General Solid-Phase Synthesis Approach to Prepare Mechanism-Based Aspartyl Protease Inhibitor Libraries. Identification of Potent Cathepsin D Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">9735</span>– <span class="NLM_lpage">9747</span>, <span class="refDoi"> DOI: 10.1021/ja981812m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja981812m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK1cXlslaktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=9735-9747&author=C.+E.+Leeauthor=E.+K.+Kickauthor=J.+A.+Ellman&title=General+Solid-Phase+Synthesis+Approach+to+Prepare+Mechanism-Based+Aspartyl+Protease+Inhibitor+Libraries.+Identification+of+Potent+Cathepsin+D+Inhibitors&doi=10.1021%2Fja981812m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">General solid-phase synthesis approach to prepare mechanism-based aspartyl protease inhibitor libraries. Identification of potent cathepsin D inhibitors</span></div><div class="casAuthors">Lee, Christina E.; Kick, Ellen K.; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">9735-9747</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A general method has been developed for the expedient solid-phase synthesis of aspartyl protease inhibitors that do not incorporate any amino acids, e.g. amides I (R = 3-PhOC6H4, 2-naphthalenyl, 4-PhC6H4, 4-BrC6H4).  The synthesis approach was designed to allow the introduction of diverse functionality at all four variable sites using readily available precursors.  Representative, anal. pure inhibitors were obtained in 45-64% overall yields for the 12-step solid-phase synthesis sequence that includes both stereoselective redn. and carbon-carbon bond forming steps.  A library of 204 spatially sep. compds. was also prepd. and screened against cathepsin D and resulted in the identification of a no. of potent small mol. inhibitors.  The described synthesis approach enables the rapid identification of small mol. inhibitors to diverse aspartyl protease targets.  The general design principles are also applicable to other enzyme classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2pHlBMx4jkrVg90H21EOLACvtfcHk0ljX2f1v5m7tFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlslaktrw%253D&md5=92b5e1523a037d54b77d7e02f0080eda</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja981812m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja981812m%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BE.%26aulast%3DKick%26aufirst%3DE.%2BK.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DGeneral%2520Solid-Phase%2520Synthesis%2520Approach%2520to%2520Prepare%2520Mechanism-Based%2520Aspartyl%2520Protease%2520Inhibitor%2520Libraries.%2520Identification%2520of%2520Potent%2520Cathepsin%2520D%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D9735%26epage%3D9747%26doi%3D10.1021%2Fja981812m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, A. J.</span></span> <span> </span><span class="NLM_article-title">Interaction of Human Cathepsin D with the Inhibitor Pepstatin</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1042/bj1550117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1042%2Fbj1550117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=938470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaE28XksVakurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=1976&pages=117-125&author=C.+G.+Knightauthor=A.+J.+Barrett&title=Interaction+of+Human+Cathepsin+D+with+the+Inhibitor+Pepstatin&doi=10.1042%2Fbj1550117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of human cathepsin D with the inhibitor pepstatin</span></div><div class="casAuthors">Knight, C. Graham; Barrett, Alan J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">117-25</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    </div><div class="casAbstract">The apparent dissocn. const. (KD) for the binding of pepstatin, pepstatin Me ester, and pepstatinyl-glycine to cathepsin D (EC 3.4.23.5) at pH 3.5 was ∼5 × 10-10M.  The KD of pepstatinyl-glycine rose from 5 × 10-10 to 2 × 10-6M on raising the pH from 5.0 to 6.4.  The catalytic activity of cathepsin D fell to ∼0 on going from pH 5.0 to 7.0.  The binding site for substrate and pepstatin may be abolished by a conformational change in cathepsin D.  Cathepsin D tightly bound 1 mole pepstatin per 43,000 g protein, indicating that abs. cocns. of the enzyme can be detd. by titration with pepstatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwv8-RPbtj0rVg90H21EOLACvtfcHk0ljX2f1v5m7tFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XksVakurk%253D&md5=9e2504b37e08e7569a0d2af2f95e85c0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1042%2Fbj1550117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj1550117%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DC.%2BG.%26aulast%3DBarrett%26aufirst%3DA.%2BJ.%26atitle%3DInteraction%2520of%2520Human%2520Cathepsin%2520D%2520with%2520the%2520Inhibitor%2520Pepstatin%26jtitle%3DBiochem.%2520J.%26date%3D1976%26volume%3D155%26spage%3D117%26epage%3D125%26doi%3D10.1042%2Fbj1550117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulnik, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, J. W.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Subsite Specificity Mapping of Human Cathepsin D Using Statine-Based Inhibitors</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1458</span>– <span class="NLM_lpage">1466</span>, <span class="refDoi"> DOI: 10.1002/pro.5560060710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fpro.5560060710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=9232647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK2sXksF2rtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1997&pages=1458-1466&author=P.+Majerauthor=J.+R.+Collinsauthor=S.+V.+Gulnikauthor=J.+W.+Erickson&title=Structure-Based+Subsite+Specificity+Mapping+of+Human+Cathepsin+D+Using+Statine-Based+Inhibitors&doi=10.1002%2Fpro.5560060710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based subsite specificity mapping of human cathepsin D using statine-based inhibitors</span></div><div class="casAuthors">Majer, Pavel; Collins, Jack R.; Gulnik, Sergei V.; Erickson, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1458-1466</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Human cathepsin D is a lysosomal aspartic protease that has been implicated in breast cancer metastasis and Alzheimer's disease.  Based on a crystal structure of a human cathepsin D-pepstatin A complex, a series of statine-contg. inhibitors was designed, synthesized, and tested for inhibitory activity toward the enzyme in vitro.  The compds. were modified systematically at individual positions (P4, P3, P2, P1, and P'2) with the aim of mapping the cathepsin D subsite preferences.  The exptl. obtained SAR data were correlated on the basis of mol. modeling.  Side-chain preferences for the peptidomimetic inhibitors differed from those found previously using peptide substrates (Scarborough PE et al., 1993, Protein Sci 2:264-276).  In addn., the effects of single side-chain modifications were often nonadditive.  Structure-activity relationships, modeling, and thermodn. anal. indicated that entropy plays a major stabilizing role in inhibitor binding to cathepsin D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgKoFaE1fzZ7Vg90H21EOLACvtfcHk0ljX2f1v5m7tFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksF2rtr8%253D&md5=56f9c56fa0bb620fc366abc27aa165f5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560060710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560060710%26sid%3Dliteratum%253Aachs%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DJ.%2BR.%26aulast%3DGulnik%26aufirst%3DS.%2BV.%26aulast%3DErickson%26aufirst%3DJ.%2BW.%26atitle%3DStructure-Based%2520Subsite%2520Specificity%2520Mapping%2520of%2520Human%2520Cathepsin%2520D%2520Using%2520Statine-Based%2520Inhibitors%26jtitle%3DProtein%2520Sci.%26date%3D1997%26volume%3D6%26spage%3D1458%26epage%3D1466%26doi%3D10.1002%2Fpro.5560060710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumerová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Máša, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hradilek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šanda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiniš, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Single- and Double-Headed Chemical Probes for Detection of Active Cathepsin D in a Cancer Cell Proteome</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1538</span>– <span class="NLM_lpage">1541</span>, <span class="refDoi"> DOI: 10.1002/cbic.201000182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fcbic.201000182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=20564284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpt1Clu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=1538-1541&author=M.+Nussbaumerov%C3%A1author=J.+Srpauthor=M.+M%C3%A1%C5%A1aauthor=M.+Hradilekauthor=M.+%C5%A0andaauthor=M.+Reini%C5%A1author=M.+Hornauthor=M.+Mare%C5%A1&title=Single-+and+Double-Headed+Chemical+Probes+for+Detection+of+Active+Cathepsin+D+in+a+Cancer+Cell+Proteome&doi=10.1002%2Fcbic.201000182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Single- and double-headed chemical probes for detection of active cathepsin D in a cancer cell proteome</span></div><div class="casAuthors">Nussbaumerova, Martina; Srp, Jaroslav; Masa, Martin; Hradilek, Martin; Sanda, Miloslav; Reinis, Milan; Horn, Martin; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1538-1541</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The design of specific active site-directed probes for cathepsins D and related aspartic proteases, which enable the effective labeling of these proteases in a complex proteome, is presented.  The applicability of this system to activity-based proteomics was demonstrated for cancer cell-derived cathepsin D, a therapeutic target in several human tumors.  The modular assembly rendered the probe synthesis convenient and widely available.  Due to their versatile fluorescent and biotin tags, the probes can be used as a powerful tool for a broad range of "in-gel" and "on-blot" proteomic applications in profiling medically important cathepsin D-type proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0lXD5RkZKY7Vg90H21EOLACvtfcHk0lhrf2I6cNrArw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpt1Clu7s%253D&md5=9c656dd3fd94005e9e9ead95fb4fa32f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201000182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201000182%26sid%3Dliteratum%253Aachs%26aulast%3DNussbaumerov%25C3%25A1%26aufirst%3DM.%26aulast%3DSrp%26aufirst%3DJ.%26aulast%3DM%25C3%25A1%25C5%25A1a%26aufirst%3DM.%26aulast%3DHradilek%26aufirst%3DM.%26aulast%3D%25C5%25A0anda%26aufirst%3DM.%26aulast%3DReini%25C5%25A1%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DSingle-%2520and%2520Double-Headed%2520Chemical%2520Probes%2520for%2520Detection%2520of%2520Active%2520Cathepsin%2520D%2520in%2520a%2520Cancer%2520Cell%2520Proteome%26jtitle%3DChemBioChem%26date%3D2010%26volume%3D11%26spage%3D1538%26epage%3D1541%26doi%3D10.1002%2Fcbic.201000182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumerová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šanda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovářová, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srba, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franta, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sojka, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopáček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Hemoglobin Digestion in Blood-Feeding Ticks: Mapping a Multipeptidase Pathway by Functional Proteomics</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1053</span>– <span class="NLM_lpage">1063</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2009.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.chembiol.2009.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=19875079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlCnt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=1053-1063&author=M.+Hornauthor=M.+Nussbaumerov%C3%A1author=M.+%C5%A0andaauthor=Z.+Kov%C3%A1%C5%99ov%C3%A1author=J.+Srbaauthor=Z.+Frantaauthor=D.+Sojkaauthor=M.+Bogyoauthor=C.+R.+Caffreyauthor=P.+Kop%C3%A1%C4%8Dekauthor=M.+Mare%C5%A1&title=Hemoglobin+Digestion+in+Blood-Feeding+Ticks%3A+Mapping+a+Multipeptidase+Pathway+by+Functional+Proteomics&doi=10.1016%2Fj.chembiol.2009.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Hemoglobin Digestion in Blood-Feeding Ticks: Mapping a Multipeptidase Pathway by Functional Proteomics</span></div><div class="casAuthors">Horn, Martin; Nussbaumerova, Martina; Sanda, Miloslav; Kovarova, Zuzana; Srba, Jindrich; Franta, Zdenek; Sojka, Daniel; Bogyo, Matthew; Caffrey, Conor R.; Kopacek, Petr; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1053-1063</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Hb digestion is an essential process for blood-feeding parasites.  Using chem. tools, we deconvoluted the intracellular hemoglobinolytic cascade in the tick Ixodes ricinus, a vector of Lyme disease and tick-borne encephalitis.  In tick gut tissue, a network of peptidases was demonstrated through imaging with specific activity-based probes and activity profiling with peptidic substrates and inhibitors.  This peptidase network is induced upon blood feeding and degrades Hb at acidic pH.  Selective inhibitors were applied to dissect the roles of the individual peptidases and to det. the peptidase-specific cleavage map of the Hb mol.  The degrdn. pathway is initiated by endopeptidases of aspartic and cysteine class (cathepsin D supported by cathepsin L and legumain) and is continued by cysteine amino- and carboxy-dipeptidases (cathepsins C and B).  The identified enzymes are potential targets to developing novel anti-tick vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTsuPLZ5dd7rVg90H21EOLACvtfcHk0lhrf2I6cNrArw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlCnt7bF&md5=be0bcaea4706c1f819fc65522afd28f9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DNussbaumerov%25C3%25A1%26aufirst%3DM.%26aulast%3D%25C5%25A0anda%26aufirst%3DM.%26aulast%3DKov%25C3%25A1%25C5%2599ov%25C3%25A1%26aufirst%3DZ.%26aulast%3DSrba%26aufirst%3DJ.%26aulast%3DFranta%26aufirst%3DZ.%26aulast%3DSojka%26aufirst%3DD.%26aulast%3DBogyo%26aufirst%3DM.%26aulast%3DCaffrey%26aufirst%3DC.%2BR.%26aulast%3DKop%25C3%25A1%25C4%258Dek%26aufirst%3DP.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DHemoglobin%2520Digestion%2520in%2520Blood-Feeding%2520Ticks%253A%2520Mapping%2520a%2520Multipeptidase%2520Pathway%2520by%2520Functional%2520Proteomics%26jtitle%3DChem.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D1053%26epage%3D1063%26doi%3D10.1016%2Fj.chembiol.2009.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tumminello, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernacki, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebbia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leto, G.</span></span> <span> </span><span class="NLM_article-title">Pepstatins: Aspartic Proteinase Inhibitors Having Potential Therapeutic Applications</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1002/med.2610130206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fmed.2610130206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=8445958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3sXktF2ltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1993&pages=199-208&author=F.+M.+Tumminelloauthor=R.+J.+Bernackiauthor=N.+Gebbiaauthor=G.+Leto&title=Pepstatins%3A+Aspartic+Proteinase+Inhibitors+Having+Potential+Therapeutic+Applications&doi=10.1002%2Fmed.2610130206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Pepstatins: aspartic proteinase inhibitors having potential therapeutic applications</span></div><div class="casAuthors">Tumminello, F. M.; Bernacki, R. J.; Gebbia, N.; Leto, G.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">199-208</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    </div><div class="casAbstract">A review with 57 refs. of the potential therapeutic applications of pepstatins in diseases such as muscular dystrophy, myocardial ischemia, cancer and microbial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosZlqcTW8lzLVg90H21EOLACvtfcHk0lhrf2I6cNrArw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktF2ltrk%253D&md5=93fc6a7c612cf852ffa6f7401a251f69</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fmed.2610130206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.2610130206%26sid%3Dliteratum%253Aachs%26aulast%3DTumminello%26aufirst%3DF.%2BM.%26aulast%3DBernacki%26aufirst%3DR.%2BJ.%26aulast%3DGebbia%26aufirst%3DN.%26aulast%3DLeto%26aufirst%3DG.%26atitle%3DPepstatins%253A%2520Aspartic%2520Proteinase%2520Inhibitors%2520Having%2520Potential%2520Therapeutic%2520Applications%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1993%26volume%3D13%26spage%3D199%26epage%3D208%26doi%3D10.1002%2Fmed.2610130206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, D. H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Cathepsin D by Substrate Analogues Containing Statine and by Analogues of Pepstatin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2519</span>– <span class="NLM_lpage">2524</span>, <span class="refDoi"> DOI: 10.1021/jm00162a015</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00162a015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaL28XmtV2gtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=2519-2524&author=N.+S.+Agarwalauthor=D.+H.+Rich&title=Inhibition+of+Cathepsin+D+by+Substrate+Analogues+Containing+Statine+and+by+Analogues+of+Pepstatin&doi=10.1021%2Fjm00162a015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of cathepsin D by substrate analogs containing statine and by analogs of pepstatin</span></div><div class="casAuthors">Agarwal, Nirankar S.; Rich, Daniel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2519-24</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Statine-contg. peptides Boc-Phe-X-Ala-Sta-X1-Val-Leu-OMe [I; Boc = Me3CO2C; Sta = (3S,4S)-4-amino-3-hydroxy-6-methylheptanoic acid (statine) residue; X = Ala, Leu, X1 = Phe, null, X = X1 = Phe] were prepd. by conventional soln. methods and they were tested as inhibitors of bovine cathepsin D.  I were derived by replacing a Phe-Phe unit of cathepsin D substrate Boc-Phe-Leu-Ala-Phe-Phe-Val-Leu-OR with Sta or Sta-Phe.  I (X = Leu, X1 = null) was the best inhibitor (ki = 1.1 mM).  Thirty-five known pepstatin analogs were also evaluated as cathepsin D inhibitors.  Substituents in the P4' and P3' positions are important for max. inhibition and the P4 substituent is more important for inhibition of cathepsin D than for other aspartic proteinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLvR9izPjhLrVg90H21EOLACvtfcHk0lgTpbj2gBwzyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XmtV2gtb0%253D&md5=bf8d95f8d1f2dd401fc2bf0f0f2607c8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm00162a015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00162a015%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwal%26aufirst%3DN.%2BS.%26aulast%3DRich%26aufirst%3DD.%2BH.%26atitle%3DInhibition%2520of%2520Cathepsin%2520D%2520by%2520Substrate%2520Analogues%2520Containing%2520Statine%2520and%2520by%2520Analogues%2520of%2520Pepstatin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1986%26volume%3D29%26spage%3D2519%26epage%3D2524%26doi%3D10.1021%2Fjm00162a015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulnik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustchina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suvorov, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, J. W.</span></span> <span> </span><span class="NLM_article-title">Dissection of the pH Dependence of Inhibitor Binding Energetics for an Aspartic Protease: Direct Measurement of the Protonation States of the Catalytic Aspartic Acid Residues</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">16166</span>– <span class="NLM_lpage">16172</span>, <span class="refDoi"> DOI: 10.1021/bi971550l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi971550l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK1cXjs1Ggtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1997&pages=16166-16172&author=D.+Xieauthor=S.+Gulnikauthor=L.+Collinsauthor=E.+Gustchinaauthor=L.+Suvorovauthor=J.+W.+Erickson&title=Dissection+of+the+pH+Dependence+of+Inhibitor+Binding+Energetics+for+an+Aspartic+Protease%3A+Direct+Measurement+of+the+Protonation+States+of+the+Catalytic+Aspartic+Acid+Residues&doi=10.1021%2Fbi971550l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Dissection of the pH Dependence of Inhibitor Binding Energetics for an Aspartic Protease: Direct Measurement of the Protonation States of the Catalytic Aspartic Acid Residues</span></div><div class="casAuthors">Xie, Dong; Gulnik, Sergei; Collins, Laura; Gustchina, Elena; Suvorov, Leonid; Erickson, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">16166-16172</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic activity and inhibitor binding energetics of enzymes are often pH-dependent properties.  Aspartic proteases comprise an important class of enzyme targets for structure-based drug design.  The authors have performed a complete thermodn. study of pepstatin binding to plasmepsin II, an aspartic proteinase found in Plasmodium falciparum, using isothermal titrn. calorimetry and CD.  Thermodn. parameters (ΔG, ΔH, ΔCp, and ΔS) were measured as functions of both pH and temp.  In the pH range from 4.5 to 7.0, pepstatin binding is accompanied by proton transfer between the solvent and the complex.  The authors used thermodn. proton linkage theory to derive both the pH-independent binding energetics for pepstatin and the no. and pKa values of ionizable residues whose pKa values change during ligand binding.  These residues were identified as the two catalytic aspartates, with pKas of 6.5 and 3.0, and His 164, with a pKa of 7.5, based on the three-dimensional structure of the pepstatin-plasmepsin II complex.  At pH 5.0, where the protease has optimum activity, the proton transfer process contributes almost 40% of the total binding free energy change and the total charge of the active-site aspartic acid residues is -1.  These exptl. results provide direct measurement for the protonation states of the catalytic aspartates in the presence of bound ligands.  Comparison of the thermodn. and structural data for pepstatin binding with human cathepsin D, a lysosomal aspartic protease that shares 35% sequence identity with plasmepsin II, suggests that the energetic differences between these two proteins are due to a higher interdomain flexibility in plasmepsin II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC2btgtkp6IrVg90H21EOLACvtfcHk0lgTpbj2gBwzyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjs1Ggtw%253D%253D&md5=a7ddbf32e432004b471f818717da0ff3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fbi971550l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi971550l%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DD.%26aulast%3DGulnik%26aufirst%3DS.%26aulast%3DCollins%26aufirst%3DL.%26aulast%3DGustchina%26aufirst%3DE.%26aulast%3DSuvorov%26aufirst%3DL.%26aulast%3DErickson%26aufirst%3DJ.%2BW.%26atitle%3DDissection%2520of%2520the%2520pH%2520Dependence%2520of%2520Inhibitor%2520Binding%2520Energetics%2520for%2520an%2520Aspartic%2520Protease%253A%2520Direct%2520Measurement%2520of%2520the%2520Protonation%2520States%2520of%2520the%2520Catalytic%2520Aspartic%2520Acid%2520Residues%26jtitle%3DBiochemistry%26date%3D1997%26volume%3D36%26spage%3D16166%26epage%3D16172%26doi%3D10.1021%2Fbi971550l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6853</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6843-6853&author=M.+D.+Cummingsauthor=S.+Sekharan&title=Structure-Based+Macrocycle+Design+in+Small-Molecule+Drug+Discovery+and+Simple+Metrics+To+Identify+Opportunities+for+Macrocyclization+of+Small-Molecule+Ligands&doi=10.1021%2Facs.jmedchem.8b01985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span></div><div class="casAuthors">Cummings, Maxwell D.; Sekharan, Sivakumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6843-6853</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Interest is growing in the use of macrocycles in pharmaceutical discovery.  Macrocylization may provide a gateway to an expanded chem. space for small-mol. drug discovery, and this could be beneficial in prosecuting difficult targets, e.g., protein-protein interactions.  Most, but not all, macrocycle drugs are derived from natural products.  Studies on synthetic drug-like small-mol. macrocycles are limited, and our current understanding of macrocycle drugs is similarly limited.  Following some background discussion, we review several examples of the structure-based design of synthetic macrocycles.  Our opinion is that in conformationally suitable systems macrocycles are an analog class worthy of consideration.  We then summarize an approach for the initial evaluation of mols. as candidates for macrocyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCAQg11Q6Hk7Vg90H21EOLACvtfcHk0lg6yxRouATjjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D&md5=32ac78d837075171d19c4cea694dce3f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DSekharan%26aufirst%3DS.%26atitle%3DStructure-Based%2520Macrocycle%2520Design%2520in%2520Small-Molecule%2520Drug%2520Discovery%2520and%2520Simple%2520Metrics%2520To%2520Identify%2520Opportunities%2520for%2520Macrocyclization%2520of%2520Small-Molecule%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6843%26epage%3D6853%26doi%3D10.1021%2Facs.jmedchem.8b01985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jouin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nisato, D.</span></span> <span> </span><span class="NLM_article-title">Stereospecific Synthesis of N-Protected Statine and Its Analogues via Chiral Tetramic Acid</span>. <i>J. Chem. Soc., Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_fpage">1177</span>– <span class="NLM_lpage">1182</span>, <span class="refDoi"> DOI: 10.1039/p19870001177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1039%2Fp19870001177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaL1cXnsFOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1987&pages=1177-1182&author=P.+Jouinauthor=B.+Castroauthor=D.+Nisato&title=Stereospecific+Synthesis+of+N-Protected+Statine+and+Its+Analogues+via+Chiral+Tetramic+Acid&doi=10.1039%2Fp19870001177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Stereospecific synthesis of N-protected statine and its analogs via chiral tetramic acid</span></div><div class="casAuthors">Jouin, Patrick; Castro, Bertrand; Nisato, Dino</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 1:  Organic and Bio-Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1177-82</span>CODEN:
                <span class="NLM_cas:coden">JCPRB4</span>;
        ISSN:<span class="NLM_cas:issn">0300-922X</span>.
    </div><div class="casAbstract">The condensation of chiral N-protected amino acids with Meldrum's acid in the presence of isopropenyl chloroformate and 4-DMAP has been examd.  The cyclization of the reaction product, by heating in an org. solvent, gave the N-protected tetramic acid derivs., which were reduced to afforded the corresponding threo-4-hydroxypyrrolidin-2-ones I [R = amino acid side chain; R1 = Me3CO2C (BOC), PhCH2O2C].  The regioselective alk. or acid hydrolysis of I led to enantiomeric pure N-Boc-statine (II; R = Me2CHCH2, R1 = Boc) and its analogs in 40-60% yield from the N-protected amino acids.  The corresponding statine Me esters could also be synthesized in high yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZRntmUCA4TrVg90H21EOLACvtfcHk0lg6yxRouATjjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXnsFOqtA%253D%253D&md5=36555e17c0b318f2415b97ff7c67411b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2Fp19870001177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp19870001177%26sid%3Dliteratum%253Aachs%26aulast%3DJouin%26aufirst%3DP.%26aulast%3DCastro%26aufirst%3DB.%26aulast%3DNisato%26aufirst%3DD.%26atitle%3DStereospecific%2520Synthesis%2520of%2520N-Protected%2520Statine%2520and%2520Its%2520Analogues%2520via%2520Chiral%2520Tetramic%2520Acid%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25201%26date%3D1987%26spage%3D1177%26epage%3D1182%26doi%3D10.1039%2Fp19870001177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smrcina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majerová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerassina, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eissenstat, M. A.</span></span> <span> </span><span class="NLM_article-title">Facile Stereoselective Synthesis of γ-Substituted γ-Amino Acids from the Corresponding α-Amino Acids</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">12867</span>– <span class="NLM_lpage">12874</span>, <span class="refDoi"> DOI: 10.1016/s0040-4020(97)00840-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fs0040-4020%2897%2900840-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK2sXmtlGhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=12867-12874&author=M.+Smrcinaauthor=P.+Majerauthor=E.+Majerov%C3%A1author=T.+A.+Guerassinaauthor=M.+A.+Eissenstat&title=Facile+Stereoselective+Synthesis+of+%CE%B3-Substituted+%CE%B3-Amino+Acids+from+the+Corresponding+%CE%B1-Amino+Acids&doi=10.1016%2Fs0040-4020%2897%2900840-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Facile stereoselective synthesis of γ-substituted γ-amino acids from the corresponding α-amino acids</span></div><div class="casAuthors">Smrcina, Martin; Majer, Pavel; Majerova, Eva; Guerassina, Tatiana A.; Eissenstat, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">12867-12874</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A facile stereoselective method for the synthesis of γ-substituted, γ-amino acids I (R = CH2Ph, i-Pr, i-Bu, CH2OCH2Ph, CH2SCH2Ph, CH2CO2CH2Ph; Boc = Me3CO2C) from (L)-BocNHCHRCO2H and Meldrum's acid was developed.  The key step of the procedure is complete redn. of the keto functionality of α-amino acyl Meldrum's acid II by sodium acetoxyborohydride.  The resulting amino alkyl Meldrum's acid III undergoes thermal decarboxylative ring closure to a 5-substituted pyrrolidinone IV which yields the corresponding γ-amino acid I after basic hydrolysis.  The overall yield of the procedure ranges from 40 to 65%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3j_gvg6MY37Vg90H21EOLACvtfcHk0lhM8ZEebOeiaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmtlGhu7s%253D&md5=d5263748e902494efd11fa2bfdf54d43</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fs0040-4020%2897%2900840-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0040-4020%252897%252900840-5%26sid%3Dliteratum%253Aachs%26aulast%3DSmrcina%26aufirst%3DM.%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DMajerov%25C3%25A1%26aufirst%3DE.%26aulast%3DGuerassina%26aufirst%3DT.%2BA.%26aulast%3DEissenstat%26aufirst%3DM.%2BA.%26atitle%3DFacile%2520Stereoselective%2520Synthesis%2520of%2520%25CE%25B3-Substituted%2520%25CE%25B3-Amino%2520Acids%2520from%2520the%2520Corresponding%2520%25CE%25B1-Amino%2520Acids%26jtitle%3DTetrahedron%26date%3D1997%26volume%3D53%26spage%3D12867%26epage%3D12874%26doi%3D10.1016%2Fs0040-4020%2897%2900840-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishikawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inagaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatanaka, M.</span></span> <span> </span><span class="NLM_article-title">Radical Photocyclization Route for Macrocyclic Lactone Ring Expansion and Conversion to Macrocyclic Lactams and Ketones</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">582</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1021/jo3024126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo3024126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVKhurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=582-589&author=K.+Nishikawaauthor=Y.+Yoshimiauthor=K.+Maedaauthor=T.+Moritaauthor=I.+Takahashiauthor=T.+Itouauthor=S.+Inagakiauthor=M.+Hatanaka&title=Radical+Photocyclization+Route+for+Macrocyclic+Lactone+Ring+Expansion+and+Conversion+to+Macrocyclic+Lactams+and+Ketones&doi=10.1021%2Fjo3024126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Radical Photocyclization Route for Macrocyclic Lactone Ring Expansion and Conversion to Macrocyclic Lactams and Ketones</span></div><div class="casAuthors">Nishikawa, Keisuke; Yoshimi, Yasuharu; Maeda, Kousuke; Morita, Toshio; Takahashi, Ichiro; Itou, Tatsuya; Inagaki, Sho; Hatanaka, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">582-589</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrocyclic lactones, lactams, and ketones such as I (X = O, NH, CH2) were prepd. by a sequence using the photochem. decarboxylation and macrocyclization of acryloyloxy-, acryloylamino-, and oxopropenyl-substituted carboxylic acids such as II as the key step.  Hydrolysis of macrocyclic lactones followed by methoxymethylation of the carboxylate yielded methoxymethyl hydroxyalkanoates; acylation of the alc. with acryloyl chloride followed by ester hydrolysis yielded acryloyl carboxylic acids such as II (X = O).  Alternatively, hydrolysis of the lactone and O-methylation of the carboxylate, mesylation of the free alc., substitution with azide, Staudinger redn., acylation, and ester hydrolysis yielded acryloylamino carboxylic acids such as II (X = NH); oxidn. of the alc., addn. of vinylmagnesium bromide, oxidn., and ester hydrolysis yielded terminal oxopropenyl carboxylic acids such as II (X = CH2).  Photocyclization of the carboxylic acids in the presence of phenanthrene and 1,4-dicyanobenzene yielded macrocyclic lactones, lactams, or ketones; the method thus constitutes a two-carbon homologation of macrocyclic lactones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo225gq_nIUaLVg90H21EOLACvtfcHk0lhM8ZEebOeiaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVKhurvJ&md5=c478a67ac475e441812cb57125113bc7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo3024126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo3024126%26sid%3Dliteratum%253Aachs%26aulast%3DNishikawa%26aufirst%3DK.%26aulast%3DYoshimi%26aufirst%3DY.%26aulast%3DMaeda%26aufirst%3DK.%26aulast%3DMorita%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DI.%26aulast%3DItou%26aufirst%3DT.%26aulast%3DInagaki%26aufirst%3DS.%26aulast%3DHatanaka%26aufirst%3DM.%26atitle%3DRadical%2520Photocyclization%2520Route%2520for%2520Macrocyclic%2520Lactone%2520Ring%2520Expansion%2520and%2520Conversion%2520to%2520Macrocyclic%2520Lactams%2520and%2520Ketones%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2013%26volume%3D78%26spage%3D582%26epage%3D589%26doi%3D10.1021%2Fjo3024126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rich, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, E. T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulm, E.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Analogues of the Carboxyl Protease Inhibitor Pepstatin. Effects of Structure on Inhibition of Pepsin and Renin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1021/jm00175a006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00175a006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaL3cXlt1CisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1980&pages=27-33&author=D.+H.+Richauthor=E.+T.+O.+Sunauthor=E.+Ulm&title=Synthesis+of+Analogues+of+the+Carboxyl+Protease+Inhibitor+Pepstatin.+Effects+of+Structure+on+Inhibition+of+Pepsin+and+Renin&doi=10.1021%2Fjm00175a006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of analogs of the carboxyl protease inhibitor pepstatin.  Effect of structure on inhibition of pepsin and renin</span></div><div class="casAuthors">Rich, Daniel H.; Sun, Eric T. O.; Ulm, Edgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-33</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Analogs of carboxyl protease inhibitor, pepstatin  [26305-03-3], were synthesized from optically pure forms of N-(tert-butoxycarbonyl)-4(S)-amino-3(S)-hydroxy-6-methylheptanoic acid (Boc-Sta)  [49642-07-1], and the inhibition of pepsin  [9001-75-6] and renin  [9015-94-5] was detd.  In addn., the new amino acid (3S, 4S)-4-amino-3-hydroxy-5-phenylpentanoic acid  [72155-50-1] was synthesized and the stereochem. of the 3 and 4 positions established.  The tripeptides isovaleryl-L-valyl-(3S,4S)-4-amino-3-hydroxy-6-methylheptanoyl-L-alanineisoamylamide  [72155-64-7] and isovaleryl-L-valyl-4(S)-amino-3(S)-hydroxy-5-phenylpentanoyl-L-alanylisoamylamide  [72155-69-2] were potent inhibitors of pepsin with inhibition consts. of 1 × 10-9 and 0.9 × 10-9M, resp.  Three structural requirements necessary for potent inhibition of pepsin are a 3(S)-hydroxy group in the 3rd residue, a substituent occupying space equiv. to the 1st residue in pepstatin, and a C-terminal group extending from the statine residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvCoE6T-Px4LVg90H21EOLACvtfcHk0lhM8ZEebOeiaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXlt1CisQ%253D%253D&md5=fe29d3aee059f05259964ba2ffee743c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm00175a006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00175a006%26sid%3Dliteratum%253Aachs%26aulast%3DRich%26aufirst%3DD.%2BH.%26aulast%3DSun%26aufirst%3DE.%2BT.%2BO.%26aulast%3DUlm%26aufirst%3DE.%26atitle%3DSynthesis%2520of%2520Analogues%2520of%2520the%2520Carboxyl%2520Protease%2520Inhibitor%2520Pepstatin.%2520Effects%2520of%2520Structure%2520on%2520Inhibition%2520of%2520Pepsin%2520and%2520Renin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1980%26volume%3D23%26spage%3D27%26epage%3D33%26doi%3D10.1021%2Fjm00175a006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulnik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosur, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sowder, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cachau, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, J. W.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Native and Inhibited Forms of Human Cathepsin D: Implications for Lysosomal Targeting and Drug Design</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">6796</span>– <span class="NLM_lpage">6800</span>, <span class="refDoi"> DOI: 10.1073/pnas.90.14.6796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1073%2Fpnas.90.14.6796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=8393577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3sXltlCgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=6796-6800&author=E.+T.+Baldwinauthor=T.+N.+Bhatauthor=S.+Gulnikauthor=M.+V.+Hosurauthor=R.+C.+Sowderauthor=R.+E.+Cachauauthor=J.+Collinsauthor=A.+M.+Silvaauthor=J.+W.+Erickson&title=Crystal+Structures+of+Native+and+Inhibited+Forms+of+Human+Cathepsin+D%3A+Implications+for+Lysosomal+Targeting+and+Drug+Design&doi=10.1073%2Fpnas.90.14.6796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of native and inhibited forms of human cathepsin D: Implications for lysosomal targeting and drug design</span></div><div class="casAuthors">Baldwin, Eric T.; Bhat, T. Narayana; Gulnik, Sergei; Hosur, Madhusoodan V.; Sowder, Raymond C.; Cachau, Raul E.; Collins, Jack; Silva, Abelardo M.; Erickson, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6796-800</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Cathepsin D (EC 3.4.23.5) (I) is a lysosomal protease suspected to play important roles in protein catabolism, antigen processing, degenerative diseases, and breast cancer progression.  The detn. of the crystal structures of I and its complex with pepstatin (II) at 2.5 Å resoln. provided insights into inhibitor binding and lysosomal targeting for this 2-chain, N-glycosylated aspartic protease.  A comparison with the structures of a complex of II bound to rhizopuspepsin and with a human renin-inhibitor complex revealed differences in subsite structures and inhibitor-enzyme interactions that were consistent with affinity differences and structure-activity relations and suggest strategies for fine-tuning the specificity of I inhibitors.  Mutagenesis studies identified a phosphotransferase recognition region that was required for oligosaccharide phosphorylation but was 32 Å distant from the N-domain glycosylation site at Asn-70.  The electron d. for the crystal structure of cathepsin D indicated the presence of an N-linked oligosaccharide that extended from Asn-70 toward Lys-203, which was a key component of the phosphotransferase recognition region, and thus provided a structural explanation for how the phosphotransferase can recognize apparently distant sites on the protein surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbNgboD93gFLVg90H21EOLACvtfcHk0liq8vwRxpbADg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltlCgtrY%253D&md5=e0c671e72699d471fdcdfcaf222eb514</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.14.6796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.14.6796%26sid%3Dliteratum%253Aachs%26aulast%3DBaldwin%26aufirst%3DE.%2BT.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DGulnik%26aufirst%3DS.%26aulast%3DHosur%26aufirst%3DM.%2BV.%26aulast%3DSowder%26aufirst%3DR.%2BC.%26aulast%3DCachau%26aufirst%3DR.%2BE.%26aulast%3DCollins%26aufirst%3DJ.%26aulast%3DSilva%26aufirst%3DA.%2BM.%26aulast%3DErickson%26aufirst%3DJ.%2BW.%26atitle%3DCrystal%2520Structures%2520of%2520Native%2520and%2520Inhibited%2520Forms%2520of%2520Human%2520Cathepsin%2520D%253A%2520Implications%2520for%2520Lysosomal%2520Targeting%2520and%2520Drug%2520Design%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1993%26volume%3D90%26spage%3D6796%26epage%3D6800%26doi%3D10.1073%2Fpnas.90.14.6796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+Drugs+and+Clinical+Candidates%3A+What+Can+Medicinal+Chemists+Learn+from+Their+Properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0liq8vwRxpbADg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520Drugs%2520and%2520Clinical%2520Candidates%253A%2520What%2520Can%2520Medicinal%2520Chemists%2520Learn%2520from%2520Their%2520Properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+Are+Great+Cycles%3A+Applications%2C+Opportunities%2C+and+Challenges+of+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0liq8vwRxpbADg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520Are%2520Great%2520Cycles%253A%2520Applications%252C%2520Opportunities%252C%2520and%2520Challenges%2520of%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in New Drug Discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+New+Drug+Discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0ljHQ7U7LkuBpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520New%2520Drug%2520Discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyndall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D.</span></span> <span> </span><span class="NLM_article-title">Macrocycles Mimic The Extended Peptide Conformation Recognized By Aspartic, Serine, Cysteine and Metallo Proteases</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">893</span>– <span class="NLM_lpage">907</span>, <span class="refDoi"> DOI: 10.2174/0929867013372715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.2174%2F0929867013372715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=11375757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltVeiur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=893-907&author=J.+Tyndallauthor=D.+Fairlie&title=Macrocycles+Mimic+The+Extended+Peptide+Conformation+Recognized+By+Aspartic%2C+Serine%2C+Cysteine+and+Metallo+Proteases&doi=10.2174%2F0929867013372715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles mimic the extended peptide conformation recognized by aspartic, serine, cysteine and metallo proteases</span></div><div class="casAuthors">Tyndall, Joel D. A.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">893-907</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review with refs.  It has been previously demonstrated that aspartic, serine, metallo and cysteine proteases bind to their inhibitors and substrate analogs in a single conformation, the saw-tooth or extended beta-strand.  Consequently a generic approach to the development of protease inhibitors is the use of constraints that conformationally restrict putative inhibitor mols. to an extended form.  In this way the inhibitor is pre-organized for binding to a protease and does not need to rearrange its structure.  One constraining device that has proven to be effective for such pre-organization is macrocyclization.  This article illustrates the general principle that macrocycles, esp. those composed of 3-4 amino acids and usually 13-17 ring atoms, can effectively mimic the extended conformation of short peptide sequences.  Such structure-stabilizing macrocycles are stable to degrdn. by proteases, valuable components of potent protease inhibitors, and in many cases they are also bioavailable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyuoMyDlMrlbVg90H21EOLACvtfcHk0ljHQ7U7LkuBpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltVeiur4%253D&md5=2525f64466090d8f371452ed9f0d676c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F0929867013372715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867013372715%26sid%3Dliteratum%253Aachs%26aulast%3DTyndall%26aufirst%3DJ.%26aulast%3DFairlie%26aufirst%3DD.%26atitle%3DMacrocycles%2520Mimic%2520The%2520Extended%2520Peptide%2520Conformation%2520Recognized%2520By%2520Aspartic%252C%2520Serine%252C%2520Cysteine%2520and%2520Metallo%2520Proteases%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2001%26volume%3D8%26spage%3D893%26epage%3D907%26doi%3D10.2174%2F0929867013372715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madala, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyndall, J. D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nall, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Update 1 of: Proteases Universally Recognize Beta Strands in Their Active Sites</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">PR1</span>– <span class="NLM_lpage">PR31</span>, <span class="refDoi"> DOI: 10.1021/cr900368a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr900368a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADC%252BC3M7pvFWksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2010&pages=PR1-PR31&author=P.+K.+Madalaauthor=J.+D.+A.+Tyndallauthor=T.+Nallauthor=D.+P.+Fairlie&title=Update+1+of%3A+Proteases+Universally+Recognize+Beta+Strands+in+Their+Active+Sites&doi=10.1021%2Fcr900368a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Update 1 of: Proteases universally recognize beta strands in their active sites</span></div><div class="casAuthors">Madala Praveen K; Tyndall Joel D A; Nall Tessa; Fairlie David P</div><div class="citationInfo"><span class="NLM_cas:title">Chemical reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">PR1-31</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvuWw904ECClw2ofyr07jXfW6udTcc2eZhXbtM0pC8CLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M7pvFWksQ%253D%253D&md5=2f82533e0689b1e9e040bb76cd2ebc61</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fcr900368a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr900368a%26sid%3Dliteratum%253Aachs%26aulast%3DMadala%26aufirst%3DP.%2BK.%26aulast%3DTyndall%26aufirst%3DJ.%2BD.%2BA.%26aulast%3DNall%26aufirst%3DT.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DUpdate%25201%2520of%253A%2520Proteases%2520Universally%2520Recognize%2520Beta%2520Strands%2520in%2520Their%2520Active%2520Sites%26jtitle%3DChem.%2520Rev.%26date%3D2010%26volume%3D110%26spage%3DPR1%26epage%3DPR31%26doi%3D10.1021%2Fcr900368a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fusek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudyš, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, P.</span></span> <span> </span><span class="NLM_article-title">Purification and Crystallization of Human Cathepsin D</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>226</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">557</span>, <span class="refDoi"> DOI: 10.1016/0022-2836(92)90968-p</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2F0022-2836%2892%2990968-p" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1640466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK38XlsVGru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=226&publication_year=1992&pages=555-557&author=M.+Fusekauthor=M.+Baudy%C5%A1author=P.+Metcalf&title=Purification+and+Crystallization+of+Human+Cathepsin+D&doi=10.1016%2F0022-2836%2892%2990968-p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and crystallization of human cathepsin D</span></div><div class="casAuthors">Fusek, Martin; Baudys, Miroslav; Metcalf, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">226</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">555-7</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The two-chain form of human cathepsin D was purified from human spleen with a method utilizing an ion-exchange chromatog. step prior to the pepstatin affinity column normally used to purify aspartic proteases.  The protein was crystd. from 21% polyethylene glycol 8000 at pH 4.0 using the hanging drop vapor diffusion method.  Small crystals were used as seeds to grow crystals suitable for x-ray data collection.  The crystals diffract to a resoln. of 3.2 Å and have space group P212121 with unit cell dimensions a = 59.9 Å, b = 99.6 Å, c = 133.6 Å.  There are two mols. in the asym. unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdR1DNXjyRbLVg90H21EOLACvtfcHk0ljvVRbZzWz8DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsVGru7k%253D&md5=b2b6373be614f155767f7e536c5bd790</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2892%2990968-p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252892%252990968-p%26sid%3Dliteratum%253Aachs%26aulast%3DFusek%26aufirst%3DM.%26aulast%3DBaudy%25C5%25A1%26aufirst%3DM.%26aulast%3DMetcalf%26aufirst%3DP.%26atitle%3DPurification%2520and%2520Crystallization%2520of%2520Human%2520Cathepsin%2520D%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1992%26volume%3D226%26spage%3D555%26epage%3D557%26doi%3D10.1016%2F0022-2836%2892%2990968-p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hánová, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houštecká, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sojka, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopáček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marešová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schueler-Furman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Novel Structural Mechanism of Allosteric Regulation of Aspartic Peptidases via an Evolutionarily Conserved Exosite</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">329.e4</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.chembiol.2018.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=29396291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitl2qsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=318-329.e4&author=I.+H%C3%A1nov%C3%A1author=J.+Bryndaauthor=R.+Hou%C5%A1teck%C3%A1author=N.+Alamauthor=D.+Sojkaauthor=P.+Kop%C3%A1%C4%8Dekauthor=L.+Mare%C5%A1ov%C3%A1author=J.+Vondr%C3%A1%C5%A1ekauthor=M.+Hornauthor=O.+Schueler-Furmanauthor=M.+Mare%C5%A1&title=Novel+Structural+Mechanism+of+Allosteric+Regulation+of+Aspartic+Peptidases+via+an+Evolutionarily+Conserved+Exosite&doi=10.1016%2Fj.chembiol.2018.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Structural Mechanism of Allosteric Regulation of Aspartic Peptidases via an Evolutionarily Conserved Exosite</span></div><div class="casAuthors">Hanova, Iva; Brynda, Jiri; Houstecka, Radka; Alam, Nawsad; Sojka, Daniel; Kopacek, Petr; Maresova, Lucie; Vondrasek, Jiri; Horn, Martin; Schueler-Furman, Ora; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">318-329.e4</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Pepsin-family aspartic peptidases are biosynthesized as inactive zymogens in which the propeptide blocks the active site until its proteolytic removal upon enzyme activation.  Here, we describe a novel dual regulatory function for the propeptide using a set of crystal structures of the parasite cathepsin D IrCD1.  In the IrCD1 zymogen, intramol. autoinhibition by the intact propeptide is mediated by an evolutionarily conserved exosite on the enzyme core.  After activation, the mature enzyme employs the same exosite to rebind a small fragment derived from the cleaved propeptide.  This fragment functions as an effective natural inhibitor of mature IrCD1 that operates in a pH-dependent manner through a unique allosteric inhibition mechanism.  The study uncovers the propeptide-binding exosite as a target for the regulation of pepsin-family aspartic peptidases and defines the structural requirements for exosite inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0J167H_OHt7Vg90H21EOLACvtfcHk0ljvVRbZzWz8DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitl2qsLo%253D&md5=926ccddbe7a6681ee9c0fd2e9ec940d0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25A1nov%25C3%25A1%26aufirst%3DI.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DHou%25C5%25A1teck%25C3%25A1%26aufirst%3DR.%26aulast%3DAlam%26aufirst%3DN.%26aulast%3DSojka%26aufirst%3DD.%26aulast%3DKop%25C3%25A1%25C4%258Dek%26aufirst%3DP.%26aulast%3DMare%25C5%25A1ov%25C3%25A1%26aufirst%3DL.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DSchueler-Furman%26aufirst%3DO.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DNovel%2520Structural%2520Mechanism%2520of%2520Allosteric%2520Regulation%2520of%2520Aspartic%2520Peptidases%2520via%2520an%2520Evolutionarily%2520Conserved%2520Exosite%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D318%26epage%3D329.e4%26doi%3D10.1016%2Fj.chembiol.2018.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Máša, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marešová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ježek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D Propeptide: Mechanism and Regulation of Its Interaction with the Catalytic Core</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">15474</span>– <span class="NLM_lpage">15482</span>, <span class="refDoi"> DOI: 10.1021/bi0614986</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0614986" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1yiur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=15474-15482&author=M.+M%C3%A1%C5%A1aauthor=L.+Mare%C5%A1ov%C3%A1author=J.+Vondr%C3%A1%C5%A1ekauthor=M.+Hornauthor=J.+Je%C5%BEekauthor=M.+Mare%C5%A1&title=Cathepsin+D+Propeptide%3A+Mechanism+and+Regulation+of+Its+Interaction+with+the+Catalytic+Core&doi=10.1021%2Fbi0614986"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D Propeptide: Mechanism and Regulation of Its Interaction with the Catalytic Core</span></div><div class="casAuthors">Masa, Martin; Maresova, Lucie; Vondrasek, Jiri; Horn, Martin; Jezek, Jan; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">15474-15482</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Propeptide blocks the active site in the inactive zymogen of cathepsin D and is cleaved off during zymogen activation.  We have designed a set of peptidic fragments derived from the propeptide structure and evaluated their inhibitory potency against mature cathepsin D using a kinetic assay.  Our mapping of the cathepsin D propeptide indicated two domains in the propeptide involved in the inhibitory interaction with the enzyme core: the active site "anchor" domain and the N-terminus of the propeptide.  The latter plays a dominant role in propeptide inhibition (nanomolar Ki), and its high-affinity binding was corroborated by fluorescence polarization measurements.  In addn. to the inhibitory domains of propeptide, a fragment derived from the N-terminus of mature cathepsin D displayed inhibition.  This finding supports its proposed regulatory function.  The interaction mechanisms of the identified inhibitory domains were characterized by detg. their modes of inhibition as well as by spatial modeling of the propeptide in the zymogen mol.  The inhibitory interaction of the N-terminal propeptide domain was abolished in the presence of sulfated polysaccharides, which interact with basic propeptide residues.  The inhibitory potency of the active site anchor domain was affected by the Ala38pVal substitution, a propeptide polymorphism reported to be assocd. with the pathol. of Alzheimer's disease.  We infer that propeptide is a sensitive tethered ligand that allows for complex modulation of cathepsin D zymogen activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfewqajyt-OrVg90H21EOLACvtfcHk0ljvVRbZzWz8DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1yiur3F&md5=4829737bb45ca56d93d189320d8afc08</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fbi0614986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0614986%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25A1%25C5%25A1a%26aufirst%3DM.%26aulast%3DMare%25C5%25A1ov%25C3%25A1%26aufirst%3DL.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DJe%25C5%25BEek%26aufirst%3DJ.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DCathepsin%2520D%2520Propeptide%253A%2520Mechanism%2520and%2520Regulation%2520of%2520Its%2520Interaction%2520with%2520the%2520Catalytic%2520Core%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D15474%26epage%3D15482%26doi%3D10.1021%2Fbi0614986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgDLzdZluKiXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Žebrakovská, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Máša, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vávrová, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Complex Modulation of Peptidolytic Activity of Cathepsin D by Sphingolipids</span>. <i>Biochim. Biophys. Acta, Mol. Cell Biol. Lipids</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>1811</i></span>,  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1104</span>, <span class="refDoi"> DOI: 10.1016/j.bbalip.2011.09.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.bbalip.2011.09.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2nu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1811&publication_year=2011&pages=1097-1104&author=I.+%C5%BDebrakovsk%C3%A1author=M.+M%C3%A1%C5%A1aauthor=J.+Srpauthor=M.+Hornauthor=K.+V%C3%A1vrov%C3%A1author=M.+Mare%C5%A1&title=Complex+Modulation+of+Peptidolytic+Activity+of+Cathepsin+D+by+Sphingolipids&doi=10.1016%2Fj.bbalip.2011.09.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Complex modulation of peptidolytic activity of cathepsin D by sphingolipids</span></div><div class="casAuthors">Zebrakovska, Iva; Masa, Martin; Srp, Jaroslav; Horn, Martin; Vavrova, Katerina; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular and Cell Biology of Lipids</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1811</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1097-1104</span>CODEN:
                <span class="NLM_cas:coden">BBMLFG</span>;
        ISSN:<span class="NLM_cas:issn">1388-1981</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">Cathepsin D is an aspartic peptidase involved in cellular processes including proliferation and apoptosis and implicated in human pathologies such as cancer and neurodegeneration.  Our knowledge about the relationship between proteolysis and bioactive sphingolipids is still very limited.  Here, we describe a complex pattern of modulation of the peptidolytic activity of cathepsin D by sphingolipids.  A panel of sphingolipid derivs. was screened in a FRET-based assay; these mols. demonstrated neg. or pos. modulation of cathepsin D peptidolytic activity, depending on the sphingolipid structure.  Certain sphingosines and ceramides inhibited cathepsin D in the submicromolar range, and structural requirements for this inhibitory effect were evaluated.  The interaction of cathepsin D with sphingolipids was also demonstrated by fluorescence polarization measurements and detd. to follow a competitive inhibition mode.  In contrast, monoester phosphosphingolipids, esp. ceramide-1-phosphate, were identified as activators of cathepsin D peptidolytic activity at submicromolar concns.  Thus, sphingolipids and phosphosphingolipids, known to be antagonistic in their cell-signaling functions, displayed opposite modulation of cathepsin D.  Sphingolipid-based modulators of cathepsin D are potentially involved in the control of cathepsin D-dependent processes and might serve as a scaffold for the development of novel regulators of this therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI_gDsPWriw7Vg90H21EOLACvtfcHk0lgDLzdZluKiXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2nu7fP&md5=c9221adb25355b37952351c0e1837a0f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.bbalip.2011.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbalip.2011.09.005%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25BDebrakovsk%25C3%25A1%26aufirst%3DI.%26aulast%3DM%25C3%25A1%25C5%25A1a%26aufirst%3DM.%26aulast%3DSrp%26aufirst%3DJ.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DV%25C3%25A1vrov%25C3%25A1%26aufirst%3DK.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DComplex%2520Modulation%2520of%2520Peptidolytic%2520Activity%2520of%2520Cathepsin%2520D%2520by%2520Sphingolipids%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Biol.%2520Lipids%26date%3D2011%26volume%3D1811%26spage%3D1097%26epage%3D1104%26doi%3D10.1016%2Fj.bbalip.2011.09.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uhlíková, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichová, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souček, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kräusslich, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span> <span> </span><span class="NLM_article-title">A Modular Approach to HIV-1 Proteinase Inhibitor Design</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>222</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1996.0694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1006%2Fbbrc.1996.0694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=8630071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK28XivVOisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=222&publication_year=1996&pages=38-43&author=T.+Uhl%C3%ADkov%C3%A1author=J.+Konvalinkaauthor=I.+Pichov%C3%A1author=M.+Sou%C4%8Dekauthor=H.-G.+Kr%C3%A4usslichauthor=J.+Vondr%C3%A1%C5%A1ek&title=A+Modular+Approach+to+HIV-1+Proteinase+Inhibitor+Design&doi=10.1006%2Fbbrc.1996.0694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A modular approach to HIV-1 proteinase inhibitor design</span></div><div class="casAuthors">Uhlikova, Tat'ana; Konvalinka, Jan; Pichova, Iva; Soucek, Milan; Krausslich, Hans-Georg; Vondrasek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">222</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-43</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">HIV-1 proteinase represents a promising target for antiviral chemotherapy.  The authors have designed, synthesized, and tested modular inhibitors combining an active-site inhibitor tethered to a structure targeted to the dimerization domain of the enzyme.  At pH 5, the parent active site inhibitor, the equimolar mixt. of active site and dimerization inhibitors, and the best compd. from the authors series of modular inhibitors show the same inhibition activity.  At neutral pH, however, the combination of the dimerization and active-site inhibitors shows a synergistic effect.  Moreover, the modular inhibitor has an IC50 value 5x lower than the parent active site inhibitor and 2x lower than the equimolar mixt. of the two parent inhibitors.  The Lineweaver-Burk plot for modular inhibitors corresponds to a pattern for mixed type inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCOV3RJNyOB7Vg90H21EOLACvtfcHk0lj-e21iMHnnfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XivVOisbY%253D&md5=874ceb808d7294d55ddf093837cc84c8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1996.0694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1996.0694%26sid%3Dliteratum%253Aachs%26aulast%3DUhl%25C3%25ADkov%25C3%25A1%26aufirst%3DT.%26aulast%3DKonvalinka%26aufirst%3DJ.%26aulast%3DPichov%25C3%25A1%26aufirst%3DI.%26aulast%3DSou%25C4%258Dek%26aufirst%3DM.%26aulast%3DKr%25C3%25A4usslich%26aufirst%3DH.-G.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26atitle%3DA%2520Modular%2520Approach%2520to%2520HIV-1%2520Proteinase%2520Inhibitor%2520Design%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1996%26volume%3D222%26spage%3D38%26epage%3D43%26doi%3D10.1006%2Fbbrc.1996.0694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumanchin-Njock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayral, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vizzavona, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farzan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boisbrun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulcrand, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefranc-Jullien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Checler, F.</span></span> <span> </span><span class="NLM_article-title">BACE1- and BACE2-Expressing Human Cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">25859</span>– <span class="NLM_lpage">25866</span>, <span class="refDoi"> DOI: 10.1074/jbc.m302622200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1074%2Fjbc.m302622200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=12736275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlt1ClsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=25859-25866&author=D.+Andrauauthor=C.+Dumanchin-Njockauthor=E.+Ayralauthor=J.+Vizzavonaauthor=M.+Farzanauthor=M.+Boisbrunauthor=P.+Fulcrandauthor=J.-F.+Hernandezauthor=J.+Martinezauthor=S.+Lefranc-Jullienauthor=F.+Checler&title=BACE1-+and+BACE2-Expressing+Human+Cells&doi=10.1074%2Fjbc.m302622200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">BACE1- and BACE2-expressing Human Cells. Characterization of β-Amyloid Precursor Protein-Derived Catabolites, Design of a Novel Fluorimetric Assay, and Identification of New In Vitro Inhibitors</span></div><div class="casAuthors">Andrau, David; Dumanchin-Njock, Cecile; Ayral, Erwan; Vizzavona, Jean; Farzan, Mike; Boisbrun, Michel; Fulcrand, Pierre; Hernandez, Jean-Francois; Martinez, Jean; Lefranc-Jullien, Solveig; Checler, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">25859-25866</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We have set up stably transfected HEK293 cells overexpressing the β-secretases BACE1 and BACE2 either alone or in combination with wild-type β-amyloid precursor protein (βAPP).  The characterization of the βAPP-derived catabolites indicates that cells expressing BACEs produce less genuine Aβ1- 40/42 but higher amts. of secreted sAPPβ and N-terminal-truncated Aβ species.  This was accompanied by a concomitant modulation of the C-terminal counterpart products C89 and C79 for BACE1 and BACE2, resp.  These cells were used to set up a novel BACE assay based on two quenched fluorimetric substrates mimicking the wild-type (JMV2235) and Swedish-mutated (JMV2236) βAPP sequences targeted by BACE activities.  We show that BACEs activities are enhanced by the Swedish mutation and maximal at pH 4.5.  The specificity of this double assay for genuine β-secretase activity was demonstrated by means of cathepsin D, a "false pos." BACE candidate.  Thus, cathepsin D was unable to cleave preferentially the JMV2236-mutated substrate.  The selectivity of the assay was also emphasized by the lack of JMV cleavage triggered by other "secretases" candidates such as ADAM10 (A disintegrin and metalloprotease 10), tumor necrosis α-converting enzyme, and presenilins 1 and 2.  Finally, the assay was used to screen for putative in vitro BACE inhibitors.  We identified a series of statine-derived sequences that dose-dependently inhibited BACE1 and BACE2 activities with IC50 in the micromolar range, some of which displaying selectivity for either BACE1 or BACE2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp47akCuIAZhLVg90H21EOLACvtfcHk0lj-e21iMHnnfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlt1ClsLs%253D&md5=fe070e46df90ab30006d4688c0859fda</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m302622200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m302622200%26sid%3Dliteratum%253Aachs%26aulast%3DAndrau%26aufirst%3DD.%26aulast%3DDumanchin-Njock%26aufirst%3DC.%26aulast%3DAyral%26aufirst%3DE.%26aulast%3DVizzavona%26aufirst%3DJ.%26aulast%3DFarzan%26aufirst%3DM.%26aulast%3DBoisbrun%26aufirst%3DM.%26aulast%3DFulcrand%26aufirst%3DP.%26aulast%3DHernandez%26aufirst%3DJ.-F.%26aulast%3DMartinez%26aufirst%3DJ.%26aulast%3DLefranc-Jullien%26aufirst%3DS.%26aulast%3DChecler%26aufirst%3DF.%26atitle%3DBACE1-%2520and%2520BACE2-Expressing%2520Human%2520Cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D25859%26epage%3D25866%26doi%3D10.1074%2Fjbc.m302622200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holzman, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubbins, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pederson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krafft, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. T.</span></span> <span> </span><span class="NLM_article-title">Recombinant Human Prorenin from CHO Cells: Expression and Purification</span>. <i>J. Protein Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1007/bf01024761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1007%2FBF01024761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1963533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3MXps1Wqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1990&pages=663-672&author=T.+F.+Holzmanauthor=C.+C.+Chungauthor=R.+Edaljiauthor=D.+A.+Eganauthor=E.+J.+Gubbinsauthor=A.+Rueterauthor=G.+Howardauthor=L.+K.+Yangauthor=T.+M.+Pedersonauthor=G.+A.+Krafftauthor=G.+T.+Wang&title=Recombinant+Human+Prorenin+from+CHO+Cells%3A+Expression+and+Purification&doi=10.1007%2Fbf01024761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Recombinant human prorenin from CHO cells:  expression and purification</span></div><div class="casAuthors">Holzman, Thomas F.; Chung, Christine C.; Edalji, Rohinton; Egan, David A.; Gubbins, Earl J.; Rueter, Annemarie; Howard, Gail; Yang, Lana K.; Pederson, Terry M.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Protein Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">663-72</span>CODEN:
                <span class="NLM_cas:coden">JPCHD2</span>;
        ISSN:<span class="NLM_cas:issn">0277-8033</span>.
    </div><div class="casAbstract">The gene for human preprorenin was obtained from total RNA prepd. from primary human chorion cells.  An expression vector was constructed contg. an SV40 early promoter, a human preprorenin cDNA, bovine growth hormone poly-A addn. signal, and a dihydrofolate reductase (dhfr) expression cassette.  This vector was inserted into the DXB 11 Chinese hamster ovary (CHO) cell line.  The recombinant protein was exported by CHO cells into the tissue culture media.  At harvest the prorenin levels ranged from ∼1-5 mg/L.  For protein isolation the cell culture supernatants were processed by filtration, concn., dialysis, and batch extn.  Preparative-scale isolation of prorenin was accomplished using blue-dye chromatog. and size-exclusion chromatog.  The isolated prorenin yielded a single SDS-gel band with Mr ∼ 40,000.  The proprotein was characterized with respect to N-terminal sequence and N-linked sugar compn.  Trypsin-activated renin prepd. from the proprotein was characterized with respect to N-terminal sequence and pH-activity profile.  Enzyme activity was measured with a newly developed fluorogenic peptide substrate contg. the P6-P'3 sequence of human angiotensinogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFuzdaHY7v7Vg90H21EOLACvtfcHk0lhTpDPGnOeU6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXps1Wqtw%253D%253D&md5=6e4eb32ddfb7bcb68b13ef2b25f46f41</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2FBF01024761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01024761%26sid%3Dliteratum%253Aachs%26aulast%3DHolzman%26aufirst%3DT.%2BF.%26aulast%3DChung%26aufirst%3DC.%2BC.%26aulast%3DEdalji%26aufirst%3DR.%26aulast%3DEgan%26aufirst%3DD.%2BA.%26aulast%3DGubbins%26aufirst%3DE.%2BJ.%26aulast%3DRueter%26aufirst%3DA.%26aulast%3DHoward%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DL.%2BK.%26aulast%3DPederson%26aufirst%3DT.%2BM.%26aulast%3DKrafft%26aufirst%3DG.%2BA.%26aulast%3DWang%26aufirst%3DG.%2BT.%26atitle%3DRecombinant%2520Human%2520Prorenin%2520from%2520CHO%2520Cells%253A%2520Expression%2520and%2520Purification%26jtitle%3DJ.%2520Protein%2520Chem.%26date%3D1990%26volume%3D9%26spage%3D663%26epage%3D672%26doi%3D10.1007%2Fbf01024761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salát, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paesen, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáčová, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotsyfakis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovářová, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šanda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majtán, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunclová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horká, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunn, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopáček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopecký, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure and Functional Characterization of an Immunomodulatory Salivary Cystatin from the Soft Tick Ornithodoros moubata</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1042/bj20100280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1042%2Fbj20100280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=20545626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsVejsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2010&pages=103-112&author=J.+Sal%C3%A1tauthor=G.+C.+Paesenauthor=P.+%C5%98ez%C3%A1%C4%8Dov%C3%A1author=M.+Kotsyfakisauthor=Z.+Kov%C3%A1%C5%99ov%C3%A1author=M.+%C5%A0andaauthor=J.+Majt%C3%A1nauthor=L.+Grunclov%C3%A1author=H.+Hork%C3%A1author=J.+F.+Andersenauthor=J.+Bryndaauthor=M.+Hornauthor=M.+A.+Nunnauthor=P.+Kop%C3%A1%C4%8Dekauthor=J.+Kopeck%C3%BDauthor=M.+Mare%C5%A1&title=Crystal+Structure+and+Functional+Characterization+of+an+Immunomodulatory+Salivary+Cystatin+from+the+Soft+Tick+Ornithodoros+moubata&doi=10.1042%2Fbj20100280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure and functional characterization of an immunomodulatory salivary cystatin from the soft tick Ornithodoros moubata</span></div><div class="casAuthors">Salat, Jiri; Paesen, Guido C.; Rezacova, Pavlina; Kotsyfakis, Michalis; Kovarova, Zuzana; Sanda, Miloslav; Majtan, Juraj; Grunclova, Lenka; Horka, Helena; Andersen, John F.; Brynda, Jiri; Horn, Martin; Nunn, Miles A.; Kopacek, Petr; Kopecky, Jan; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-112</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The saliva of blood-feeding parasites is a rich source of peptidase inhibitors that help to overcome the host's defense during host-parasite interactions.  Using proteomic anal., the cystatin OmC2 was demonstrated in the saliva of the soft tick Ornithodoros moubata, an important disease vector transmitting African swine fever virus and the spirochaete Borrelia duttoni.  A structural, biochem., and biol. characterization of this peptidase inhibitor was undertaken in the present study.  Recombinant OmC2 was screened against a panel of physiol. relevant peptidases and was found to be an effective broad-specificity inhibitor of cysteine cathepsins, including endopeptidases (cathepsins L and S) and exopeptidases (cathepsins B, C, and H).  The crystal structure of OmC2 was detd. at a resoln. of 2.45 Å (1 Å = 0.1 nm) and was used to describe the structure-inhibitory activity relationship.  The biol. impact of OmC2 was demonstrated both in vitro and in vivo.  OmC2 affected the function of antigen-presenting mouse dendritic cells (DCs) by reducing the prodn. of the pro-inflammatory cytokines tumor necrosis factor α (TNFα) and interleukin-12 (IL-12), and proliferation of antigen-specific CD4+ T-cells.  This suggests that OmC2 may suppress the host's adaptive immune response.  Immunization of mice with OmC2 significantly suppressed the survival of O. moubata in infestation expts.  We conclude that OmC2 is a promising target for the development of a novel anti-tick vaccine to control O. moubata populations and combat the spread of assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-TcGTTb9FTbVg90H21EOLACvtfcHk0lhTpDPGnOeU6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsVejsr4%253D&md5=e1a613bd43157cbfcb8346efacc894f0</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1042%2Fbj20100280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20100280%26sid%3Dliteratum%253Aachs%26aulast%3DSal%25C3%25A1t%26aufirst%3DJ.%26aulast%3DPaesen%26aufirst%3DG.%2BC.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258Dov%25C3%25A1%26aufirst%3DP.%26aulast%3DKotsyfakis%26aufirst%3DM.%26aulast%3DKov%25C3%25A1%25C5%2599ov%25C3%25A1%26aufirst%3DZ.%26aulast%3D%25C5%25A0anda%26aufirst%3DM.%26aulast%3DMajt%25C3%25A1n%26aufirst%3DJ.%26aulast%3DGrunclov%25C3%25A1%26aufirst%3DL.%26aulast%3DHork%25C3%25A1%26aufirst%3DH.%26aulast%3DAndersen%26aufirst%3DJ.%2BF.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DNunn%26aufirst%3DM.%2BA.%26aulast%3DKop%25C3%25A1%25C4%258Dek%26aufirst%3DP.%26aulast%3DKopeck%25C3%25BD%26aufirst%3DJ.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DCrystal%2520Structure%2520and%2520Functional%2520Characterization%2520of%2520an%2520Immunomodulatory%2520Salivary%2520Cystatin%2520from%2520the%2520Soft%2520Tick%2520Ornithodoros%2520moubata%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D429%26spage%3D103%26epage%3D112%26doi%3D10.1042%2Fbj20100280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumerová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Digestive Proteolysis in the Colorado Potato Beetle, Leptinotarsa decemlineata: Activity-Based Profiling and Imaging of a Multipeptidase Network</span>. <i>Insect Biochem. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.ibmb.2016.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.ibmb.2016.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=27539253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVamsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2016&pages=1-11&author=J.+Srpauthor=M.+Nussbaumerov%C3%A1author=M.+Hornauthor=M.+Mare%C5%A1&title=Digestive+Proteolysis+in+the+Colorado+Potato+Beetle%2C+Leptinotarsa+decemlineata%3A+Activity-Based+Profiling+and+Imaging+of+a+Multipeptidase+Network&doi=10.1016%2Fj.ibmb.2016.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Digestive proteolysis in the Colorado potato beetle, Leptinotarsa decemlineata: Activity-based profiling and imaging of a multipeptidase network</span></div><div class="casAuthors">Srp, Jaroslav; Nussbaumerova, Martina; Horn, Martin; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Insect Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">IBMBES</span>;
        ISSN:<span class="NLM_cas:issn">0965-1748</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Colorado potato beetle (CPB), Leptinotarsa decemlineata, is a major pest of potato plants, and its digestive system is a promising target for development of pest control strategies.  This work focuses on functional proteomic anal. of the digestive proteolytic enzymes expressed in the CPB gut.  We identified a set of peptidases using imaging with specific activity-based probes and activity profiling with selective substrates and inhibitors.  The secreted luminal peptidases were classified as: (i) endopeptidases of cathepsin D, cathepsin L, and trypsin types and (ii) exopeptidases with aminopeptidase (cathepsin H), carboxypeptidase (serine carboxypeptidase, prolyl carboxypeptidase), and carboxydipeptidase (cathepsin B) activities.  The proteolytic arsenal also includes non-luminal peptidases with prolyl oligopeptidase and metalloaminopeptidase activities.  Our results indicate that the CPB gut employs a multienzyme network of peptidases with complementary specificities to efficiently degrade ingested proteins.  This proteolytic system functions in both CPB larvae and adults and is controlled mainly by cysteine and aspartic peptidases and supported by serine and metallopeptidases.  The component enzymes identified here are potential targets for inhibitors with tailored specificities that could be engineered into potato plants to confer resistance to CPB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokT6cNqUFhs7Vg90H21EOLACvtfcHk0lj6B9Hjmyv68g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVamsbjE&md5=a814b7107f414d0f85d9a6dec4fd9160</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ibmb.2016.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ibmb.2016.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DSrp%26aufirst%3DJ.%26aulast%3DNussbaumerov%25C3%25A1%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DDigestive%2520Proteolysis%2520in%2520the%2520Colorado%2520Potato%2520Beetle%252C%2520Leptinotarsa%2520decemlineata%253A%2520Activity-Based%2520Profiling%2520and%2520Imaging%2520of%2520a%2520Multipeptidase%2520Network%26jtitle%3DInsect%2520Biochem.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D78%26spage%3D1%26epage%3D11%26doi%3D10.1016%2Fj.ibmb.2016.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jílková, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáčová, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marešová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fajtová, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKerrow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Activation Route of the Schistosoma mansoni Cathepsin B1 Drug Target: Structural Map with a Glycosaminoglycan Switch</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1786</span>– <span class="NLM_lpage">1798</span>, <span class="refDoi"> DOI: 10.1016/j.str.2014.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.str.2014.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=25456815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=1786-1798&author=A.+J%C3%ADlkov%C3%A1author=M.+Hornauthor=P.+%C5%98ez%C3%A1%C4%8Dov%C3%A1author=L.+Mare%C5%A1ov%C3%A1author=P.+Fajtov%C3%A1author=J.+Bryndaauthor=J.+Vondr%C3%A1%C5%A1ekauthor=J.+H.+McKerrowauthor=C.+R.+Caffreyauthor=M.+Mare%C5%A1&title=Activation+Route+of+the+Schistosoma+mansoni+Cathepsin+B1+Drug+Target%3A+Structural+Map+with+a+Glycosaminoglycan+Switch&doi=10.1016%2Fj.str.2014.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Activation Route of the Schistosoma mansoni Cathepsin B1 Drug Target: Structural Map with a Glycosaminoglycan Switch</span></div><div class="casAuthors">Jilkova, Adela; Horn, Martin; Rezacova, Pavlina; Maresova, Lucie; Fajtova, Pavla; Brynda, Jiri; Vondrasek, Jiri; McKerrow, James H.; Caffrey, Conor R.; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1786-1798</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cathepsin B1 (SmCB1) is a digestive protease of the parasitic blood fluke Schistosoma mansoni and a drug target for the treatment of schistosomiasis, a disease that afflicts over 200 million people.  SmCB1 is synthesized as an inactive zymogen in which the N-terminal propeptide blocks the active site.  We investigated the activation of the zymogen by which the propeptide is proteolytically removed and its regulation by sulfated polysaccharides (SPs).  We detd. crystal structures of three mol. forms of SmCB1 along the activation pathway: the zymogen, an activation intermediate with a partially cleaved propeptide, and the mature enzyme.  We demonstrate that SPs are essential for the autocatalytic activation of SmCB1, as they interact with a specific heparin-binding domain in the propeptide.  An alternative activation route is mediated by an S. mansoni asparaginyl endopeptidase (legumain) which is downregulated by SPs, indicating that SPs act as a mol. switch between both activation mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzrhuTfrvUy7Vg90H21EOLACvtfcHk0lj6B9Hjmyv68g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7bO&md5=bd180bad1ea5ef69a4bd81bb76c80827</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DJ%25C3%25ADlkov%25C3%25A1%26aufirst%3DA.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258Dov%25C3%25A1%26aufirst%3DP.%26aulast%3DMare%25C5%25A1ov%25C3%25A1%26aufirst%3DL.%26aulast%3DFajtov%25C3%25A1%26aufirst%3DP.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DCaffrey%26aufirst%3DC.%2BR.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DActivation%2520Route%2520of%2520the%2520Schistosoma%2520mansoni%2520Cathepsin%2520B1%2520Drug%2520Target%253A%2520Structural%2520Map%2520with%2520a%2520Glycosaminoglycan%2520Switch%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D1786%26epage%3D1798%26doi%3D10.1016%2Fj.str.2014.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Šmídková, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvoráková, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tloust’ová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Česnek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertlíková-Kaiserová, H.</span></span> <span> </span><span class="NLM_article-title">Amidate Prodrugs of 9-[2-(Phosphonomethoxy)Ethyl]Adenine as Inhibitors of Adenylate Cyclase Toxin from Bordetella pertussis</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">671</span>, <span class="refDoi"> DOI: 10.1128/aac.01685-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1128%2Faac.01685-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=24145524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVSru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=664-671&author=M.+%C5%A0m%C3%ADdkov%C3%A1author=A.+Dvor%C3%A1kov%C3%A1author=E.+Tloust%E2%80%99ov%C3%A1author=M.+%C4%8Cesnekauthor=Z.+Janebaauthor=H.+Mertl%C3%ADkov%C3%A1-Kaiserov%C3%A1&title=Amidate+Prodrugs+of+9-%5B2-%28Phosphonomethoxy%29Ethyl%5DAdenine+as+Inhibitors+of+Adenylate+Cyclase+Toxin+from+Bordetella+pertussis&doi=10.1128%2Faac.01685-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Amidate prodrugs of 9-[2-(phosphonomethoxy)ethyl]adenine as inhibitors of adenylate cyclase toxin from Bordetella pertussis</span></div><div class="casAuthors">Smidkova, Marketa; Dvorakova, Alexandra; Tloust'ova, Eva; Cesnek, Michal; Janeba, Zlatko; Mertlikova-Kaiserova, Helena</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">664-671, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Adenylate cyclase toxin (ACT) is the key virulence factor of Bordetella pertussis that facilitates its invasion into the mammalian body. 9-[2-(Phosphonomethoxy)ethyl]adenine diphosphate (PMEApp), the active metabolite of the antiviral drug bis(POM)PMEA (adefovir dipivoxil), has been shown to inhibit ACT.  The objective of this study was to evaluate six novel amidate prodrugs of PMEA, both phenyloxy phosphonamidates and phosphonodiamidates, for their ability to inhibit ACT activity in the J774A.1 macrophage cell line.  The two phenyloxy phosphonamidate prodrugs exhibited greater inhibitory activity (50% inhibitory concn. [IC50] = 22 and 46 nM) than the phosphonodiamidates (IC50 = 84 to 3,960 nM).  The inhibitory activity of the prodrugs correlated with their lipophilicity and the degree of their hydrolysis into free PMEA in J774A.1 cells.  Although the prodrugs did not inhibit ACT as effectively as bis(POM)PMEA (IC50 = 6 nM), they were significantly less cytotoxic.  Moreover, they all reduced apoptotic effects of ACT and prevented an ACT-induced elevation of intracellular [Ca2+]i.  The amidate prodrugs were less susceptible to degrdn. in Caco-2 cells compared to bis(POM)PMEA, while they exerted good transepithelial permeability in this assay.  As a consequence, a large amt. of intact amidate prodrug is expected to be available to target macrophages in vivo.  This feature makes nontoxic amidate prodrugs attractive candidates for further investigation as novel antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplArKpHfZBhrVg90H21EOLACvtfcHk0litRE-UFMRXdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVSru7c%253D&md5=fbd9057b4f326fb45cd131419ad90772</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1128%2Faac.01685-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01685-13%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0m%25C3%25ADdkov%25C3%25A1%26aufirst%3DM.%26aulast%3DDvor%25C3%25A1kov%25C3%25A1%26aufirst%3DA.%26aulast%3DTloust%25E2%2580%2599ov%25C3%25A1%26aufirst%3DE.%26aulast%3D%25C4%258Cesnek%26aufirst%3DM.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DMertl%25C3%25ADkov%25C3%25A1-Kaiserov%25C3%25A1%26aufirst%3DH.%26atitle%3DAmidate%2520Prodrugs%2520of%25209-%255B2-%2528Phosphonomethoxy%2529Ethyl%255DAdenine%2520as%2520Inhibitors%2520of%2520Adenylate%2520Cyclase%2520Toxin%2520from%2520Bordetella%2520pertussis%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D664%26epage%3D671%26doi%3D10.1128%2Faac.01685-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">An Approach to Multi-Copy Search in Molecular Replacement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1622</span>– <span class="NLM_lpage">1624</span>, <span class="refDoi"> DOI: 10.1107/s0907444900013780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2Fs0907444900013780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=11092928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVOksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2000&pages=1622-1624&author=A.+Vaginauthor=A.+Teplyakov&title=An+Approach+to+Multi-Copy+Search+in+Molecular+Replacement&doi=10.1107%2Fs0907444900013780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">An approach to multi-copy search in molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">D56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1622-1624</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">The mol.-replacement method has been extended to a simultaneous search for multiple copies of the macromol. in the unit cell.  The central point of this approach is the construction of a multi-copy search model from the properly oriented monomers using a special translation function.  The multi-copy search method has been implemented in the program MOLREP and successfully tested using exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWCUH_hO_qmLVg90H21EOLACvtfcHk0litRE-UFMRXdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVOksLo%253D&md5=b0143869509b3906e5f6e861d6fa2b32</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2Fs0907444900013780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444900013780%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DAn%2520Approach%2520to%2520Multi-Copy%2520Search%2520in%2520Molecular%2520Replacement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2000%26volume%3D56%26spage%3D1622%26epage%3D1624%26doi%3D10.1107%2Fs0907444900013780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhUymA4do__Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S.</span></span> <span> </span><span class="NLM_article-title">The Ccp4 Suite - Programs for Protein Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=S.+Bailey&title=The+Ccp4+Suite+-+Programs+for+Protein+Crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DS.%26atitle%3DThe%2520Ccp4%2520Suite%2520-%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhUymA4do__Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhvO-pBGF8VWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Babin, V.</span>; <span class="NLM_string-name">Berryman, J. T.</span>; <span class="NLM_string-name">Betz, R. M.</span>; <span class="NLM_string-name">Cai, Q.</span>; <span class="NLM_string-name">Cerutti, D. S.</span>; <span class="NLM_string-name">Cheatham, T. E.</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Duke, R. E.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Goetz, A. W.</span>; <span class="NLM_string-name">Gusarov, S.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Janowski, P.</span>; <span class="NLM_string-name">Kaus, J.</span>; <span class="NLM_string-name">Kolossváry, I.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Madej, B.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Paesani, F.</span>; <span class="NLM_string-name">Roe, D. R.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Salomon-Ferrer, R.</span>; <span class="NLM_string-name">Seabra, G.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Smith, W.</span>; <span class="NLM_string-name">Swails, J.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wolf, R. M.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>Amber 14</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+A.+Case&author=V.+Babin&author=J.+T.+Berryman&author=R.+M.+Betz&author=Q.+Cai&author=D.+S.+Cerutti&author=T.+E.+Cheatham&author=T.+A.+Darden&author=R.+E.+Duke&author=H.+Gohlke&author=A.+W.+Goetz&author=S.+Gusarov&author=N.+Homeyer&author=P.+Janowski&author=J.+Kaus&author=I.+Kolossv%C3%A1ry&author=A.+Kovalenko&author=T.+S.+Lee&author=S.+LeGrand&author=T.+Luchko&author=R.+Luo&author=B.+Madej&author=K.+M.+Merz&author=F.+Paesani&author=D.+R.+Roe&author=A.+Roitberg&author=C.+Sagui&author=R.+Salomon-Ferrer&author=G.+Seabra&author=C.+L.+Simmerling&author=W.+Smith&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=R.+M.+Wolf&author=X.+Wu&author=P.+A.+Kollman&title=Amber+14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAmber%252014%26pub%3DUniversity%2520of%2520California%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, M. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Søndergaard, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span> <span> </span><span class="NLM_article-title">PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa Predictions</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1021/ct100578z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100578z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=525-537&author=M.+H.+M.+Olssonauthor=C.+R.+S%C3%B8ndergaardauthor=M.+Rostkowskiauthor=J.+H.+Jensen&title=PROPKA3%3A+Consistent+Treatment+of+Internal+and+Surface+Residues+in+Empirical+pKa+Predictions&doi=10.1021%2Fct100578z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa Predictions</span></div><div class="casAuthors">Olsson, Mats H. M.; Sondergaard, Chresten R.; Rostkowski, Michal; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">525-537</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have revised the rules and parameters for one of the most commonly used empirical pKa predictors, PROPKA, based on better phys. description of the desolvation and dielec. response for the protein.  The authors have introduced a new and consistent approach to interpolate the description between the previously distinct classifications into internal and surface residues, which otherwise is found to give rise to an erratic and discontinuous behavior.  Since the goal of this study is to lay out the framework and validate the concept, it focuses on Asp and Glu residues where the protein pKa values and structures are assumed to be more reliable.  The new and improved implementation is evaluated and discussed; it is found to agree better with expt. than the previous implementation (in parentheses): rmsd = 0.79 (0.91) for Asp and Glu, 0.75 (0.97) for Tyr, 0.65 (0.72) for Lys, and 1.00 (1.37) for His residues.  The most significant advance, however, is in reducing the no. of outliers and removing unreasonable sensitivity to small structural changes that arise from classifying residues as either internal or surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm6urTZNdjY7Vg90H21EOLACvtfcHk0lhvO-pBGF8VWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aqsA%253D%253D&md5=9b1666b1c56e1129789e62948eb4d001</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fct100578z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100578z%26sid%3Dliteratum%253Aachs%26aulast%3DOlsson%26aufirst%3DM.%2BH.%2BM.%26aulast%3DS%25C3%25B8ndergaard%26aufirst%3DC.%2BR.%26aulast%3DRostkowski%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DPROPKA3%253A%2520Consistent%2520Treatment%2520of%2520Internal%2520and%2520Surface%2520Residues%2520in%2520Empirical%2520pKa%2520Predictions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D525%26epage%3D537%26doi%3D10.1021%2Fct100578z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hostaš, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span> <span> </span><span class="NLM_article-title">Accurate DFT-D3 Calculations in a Small Basis Set</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3575</span>– <span class="NLM_lpage">3585</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.7b00365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.7b00365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFyqsbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=3575-3585&author=J.+Hosta%C5%A1author=J.+%C5%98ez%C3%A1%C4%8D&title=Accurate+DFT-D3+Calculations+in+a+Small+Basis+Set&doi=10.1021%2Facs.jctc.7b00365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate DFT-D3 Calculations in a Small Basis Set</span></div><div class="casAuthors">Hostas, Jiri; Rezac, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3575-3585</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Calcns. of interaction energies of noncovalent interactions in small basis sets are affected by the basis set superposition error and dispersion-cor. DFT-D methods and are thus usually parametrized only for triple-ζ and larger basis sets.  Nevertheless, some smaller basis sets could also perform well.  Among many combinations tested, we obtained excellent results with the DZVP-DFT basis and newly parametrized D3 dispersion correction.  The accuracy of interaction energies and geometries is close to significantly more expensive calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEin_d4_ETFbVg90H21EOLACvtfcHk0ljLTTtjnuxmdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFyqsbjM&md5=29d0d41c5b76ade15db5e13f154baf5f</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.7b00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.7b00365%26sid%3Dliteratum%253Aachs%26aulast%3DHosta%25C5%25A1%26aufirst%3DJ.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26atitle%3DAccurate%2520DFT-D3%2520Calculations%2520in%2520a%2520Small%2520Basis%2520Set%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D3575%26epage%3D3585%26doi%3D10.1021%2Facs.jctc.7b00365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, J. J. P.</span></span> <span> </span><span class="NLM_article-title">Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations and Application to 70 Elements</span>. <i>J. Mol. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1173</span>– <span class="NLM_lpage">1213</span>, <span class="refDoi"> DOI: 10.1007/s00894-007-0233-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1007%2Fs00894-007-0233-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=17828561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlequr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=1173-1213&author=J.+J.+P.+Stewart&title=Optimization+of+Parameters+for+Semiempirical+Methods+V%3A+Modification+of+NDDO+Approximations+and+Application+to+70+Elements&doi=10.1007%2Fs00894-007-0233-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of parameters for semiempirical methods V: modification of NDDO approximations and application to 70 elements</span></div><div class="casAuthors">Stewart, James J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1173-1213</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Several modifications that have been made to the NDDO core-core interaction term and to the method of parameter optimization are described.  These changes have resulted in a more complete parameter optimization, called PM6, which has, in turn, allowed 70 elements to be parameterized.  The av. unsigned error (AUE) between calcd. and ref. heats of formation for 4,492 species was 8.0 kcal mol-1.  For the subset of 1,373 compds. involving only the elements H, C, N, O, F, P, S, Cl, and Br, the PM6 AUE was 4.4 kcal mol-1.  The equivalent AUE for other methods were: RM1: 5.0, B3LYP 6-31G*: 5.2, PM5: 5.7, PM3: 6.3, HF 6-31G*: 7.4, and AM1: 10.0 kcal mol-1.  Several long-standing faults in AM1 and PM3 have been cor. and significant improvements have been made in the prediction of geometries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl-g7DsFjK8LVg90H21EOLACvtfcHk0ljLTTtjnuxmdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlequr7N&md5=7d22928c8e3423f3ca8f2e7c77e6e6c9</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1007%2Fs00894-007-0233-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-007-0233-4%26sid%3Dliteratum%253Aachs%26aulast%3DStewart%26aufirst%3DJ.%2BJ.%2BP.%26atitle%3DOptimization%2520of%2520Parameters%2520for%2520Semiempirical%2520Methods%2520V%253A%2520Modification%2520of%2520NDDO%2520Approximations%2520and%2520Application%2520to%252070%2520Elements%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2007%26volume%3D13%26spage%3D1173%26epage%3D1213%26doi%3D10.1007%2Fs00894-007-0233-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobza, P.</span></span> <span> </span><span class="NLM_article-title">Advanced Corrections of Hydrogen Bonding and Dispersion for Semiempirical Quantum Mechanical Methods</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1021/ct200751e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200751e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Gls77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=141-151&author=J.+%C5%98ez%C3%A1%C4%8Dauthor=P.+Hobza&title=Advanced+Corrections+of+Hydrogen+Bonding+and+Dispersion+for+Semiempirical+Quantum+Mechanical+Methods&doi=10.1021%2Fct200751e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Advanced Corrections of Hydrogen Bonding and Dispersion for Semiempirical Quantum Mechanical Methods</span></div><div class="casAuthors">Rezac, Jan; Hobza, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">141-151</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Semiempirical quantum mech. methods with corrections for noncovalent interactions, namely dispersion and hydrogen bonds, reach an accuracy comparable to much more expensive methods while being applicable to very large systems (up to 10 000 atoms).  These corrections have been successfully applied in computer-assisted drug design, where they significantly improve the correlation with the exptl. data.  Despite these successes, there are still several unresolved issues that limit the applicability of these methods.  We introduce a new generation of both hydrogen-bonding and dispersion corrections that address these problems, make the method more robust, and improve its accuracy.  The hydrogen-bonding correction has been completely redesigned and for the first time can be used for geometry optimization and mol.-dynamics simulations without any limitations, as it and its derivs. have a smooth potential energy surface.  The form of this correction is simpler than its predecessors, while the accuracy has been improved.  For the dispersion correction, we adopt the latest developments in DFT-D, using the D3 formalism by Grimme.  The new corrections have been parametrized on a large set of benchmark data including nonequil. geometries, the S66x8 data set.  As a result, the newly developed D3H4 correction can accurately describe a wider range of interactions.  We have parametrized this correction for the PM6, RM1, OM3, PM3, AM1, and SCC-DFTB methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhPaWNi5-P4bVg90H21EOLACvtfcHk0ljLTTtjnuxmdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Gls77L&md5=1c15239efc1338484dcd0b9d6e70518d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fct200751e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200751e%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26aulast%3DHobza%26aufirst%3DP.%26atitle%3DAdvanced%2520Corrections%2520of%2520Hydrogen%2520Bonding%2520and%2520Dispersion%2520for%2520Semiempirical%2520Quantum%2520Mechanical%2520Methods%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D141%26epage%3D151%26doi%3D10.1021%2Fct200751e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüürmann, G.</span></span> <span> </span><span class="NLM_article-title">COSMO: A New Approach to Dielectric Screening in Solvents with Explicit Expressions for the Screening Energy and Its Gradient</span>. <i>J. Chem. Soc., Perkin Trans. 2</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_fpage">799</span>– <span class="NLM_lpage">805</span>, <span class="refDoi"> DOI: 10.1039/p29930000799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1039%2FP29930000799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3sXkvVSku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&pages=799-805&author=A.+Klamtauthor=G.+Sch%C3%BC%C3%BCrmann&title=COSMO%3A+A+New+Approach+to+Dielectric+Screening+in+Solvents+with+Explicit+Expressions+for+the+Screening+Energy+and+Its+Gradient&doi=10.1039%2Fp29930000799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">COSMO: a new approach to dielectric screening in solvents with explicit expressions for the screening energy and its gradient</span></div><div class="casAuthors">Klamt, A.; Schueuermann, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 2:  Physical Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">799-805</span>CODEN:
                <span class="NLM_cas:coden">JCPKBH</span>;
        ISSN:<span class="NLM_cas:issn">0300-9580</span>.
    </div><div class="casAbstract">Starting from the screening in conductors, an algorithm for the accurate calcn. of dielec. screening effects in solvents is presented, which leads to rather simple explicit expressions for the screening energy and its analytic gradient with respect to the solute coordinates.  Thus geometry optimization of a solute within a realistic dielec. continuum model becomes practicable for the first time.  The algorithm is suited for mol. mechanics as well as for any MO algorithm.  The implementation into MOPAC and some example applications are reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUvfeTHvmEb7Vg90H21EOLACvtfcHk0ljLTTtjnuxmdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvVSku7o%253D&md5=a8e9e41514d60ca7f11212cf5c281424</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1039%2FP29930000799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FP29930000799%26sid%3Dliteratum%253Aachs%26aulast%3DKlamt%26aufirst%3DA.%26aulast%3DSch%25C3%25BC%25C3%25BCrmann%26aufirst%3DG.%26atitle%3DCOSMO%253A%2520A%2520New%2520Approach%2520to%2520Dielectric%2520Screening%2520in%2520Solvents%2520with%2520Explicit%2520Expressions%2520for%2520the%2520Screening%2520Energy%2520and%2520Its%2520Gradient%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25202%26date%3D1993%26spage%3D799%26epage%3D805%26doi%3D10.1039%2Fp29930000799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span> <span> </span><span class="NLM_article-title">Cuby: An Integrative Framework for Computational Chemistry</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1230</span>– <span class="NLM_lpage">1237</span>, <span class="refDoi"> DOI: 10.1002/jcc.24312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fjcc.24312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=26841135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVShs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=1230-1237&author=J.+%C5%98ez%C3%A1%C4%8D&title=Cuby%3A+An+Integrative+Framework+for+Computational+Chemistry&doi=10.1002%2Fjcc.24312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Cuby: An integrative framework for computational chemistry</span></div><div class="casAuthors">Rezac, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1230-1237</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Cuby is a computational chem. framework written in the Ruby programming language.  It provides unified access to a wide range of computational methods by interfacing external software and it implements various protocols that operate on their results.  Using structured input files, elementary calcns. can be combined into complex workflows.  For users, Cuby provides a unified and userfriendly way to automate their work, seamlessly integrating calcns. carried out in different computational chem. programs.  For example, the QM/MM module allows combining methods across the interfaced programs and the builtin mol. dynamics engine makes it possible to run a simulation on the resulting potential.  For programmers, it provides high-level, object-oriented environment that allows rapid development and testing of new methods and computational protocols.  The Cuby framework is available for download at http://cuby4.mol.cz.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCiLHT2DVtXrVg90H21EOLACvtfcHk0lhr8VvptMCUWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVShs74%253D&md5=6ec4c9c15ca341ce5fea73517623680b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fjcc.24312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.24312%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26atitle%3DCuby%253A%2520An%2520Integrative%2520Framework%2520for%2520Computational%2520Chemistry%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2016%26volume%3D37%26spage%3D1230%26epage%3D1237%26doi%3D10.1002%2Fjcc.24312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahlrichs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bär, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kölmel, C.</span></span> <span> </span><span class="NLM_article-title">Electronic Structure Calculations on Workstation Computers: The Program System Turbomole</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1016/0009-2614(89)85118-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2F0009-2614%2889%2985118-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3cXkt1yrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=1989&pages=165-169&author=R.+Ahlrichsauthor=M.+B%C3%A4rauthor=M.+H%C3%A4serauthor=H.+Hornauthor=C.+K%C3%B6lmel&title=Electronic+Structure+Calculations+on+Workstation+Computers%3A+The+Program+System+Turbomole&doi=10.1016%2F0009-2614%2889%2985118-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Electronic structure calculations on workstation computers:  the program system TURBOMOLE</span></div><div class="casAuthors">Ahlrichs, Reinhart; Baer, Michael; Haeser, Marco; Horn, Hans; Koelmel, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">165-9</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    </div><div class="casAbstract">The basic structure of the program system TURBOMOLE for SCF - including first and second anal. derivs. with respect to nuclear coordinates - and MP2 calcns. is briefly described.  The program takes full advantage of all discrete point group symmetries and has only modest - and (partially) adjustable - I/O and background storage requirements.  The performance of TURBOMOLE is documented for demonstrative applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPKLD1c-7-ZbVg90H21EOLACvtfcHk0lhr8VvptMCUWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXkt1yrtg%253D%253D&md5=b6aa32e6226a8e11b511e4d09cc60dc8</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2F0009-2614%2889%2985118-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0009-2614%252889%252985118-8%26sid%3Dliteratum%253Aachs%26aulast%3DAhlrichs%26aufirst%3DR.%26aulast%3DB%25C3%25A4r%26aufirst%3DM.%26aulast%3DH%25C3%25A4ser%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DH.%26aulast%3DK%25C3%25B6lmel%26aufirst%3DC.%26atitle%3DElectronic%2520Structure%2520Calculations%2520on%2520Workstation%2520Computers%253A%2520The%2520Program%2520System%2520Turbomole%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1989%26volume%3D162%26spage%3D165%26epage%3D169%26doi%3D10.1016%2F0009-2614%2889%2985118-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, J. J. P.</span></span> <span> </span><span class="NLM_article-title">Optimization of Parameters for Semiempirical Methods IV: Extension of MNDO, AM1, and PM3 to More Main Group Elements</span>. <i>J. Mol. Model.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1007/s00894-004-0183-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1007%2Fs00894-004-0183-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=14997367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktV2msL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=155-164&author=J.+J.+P.+Stewart&title=Optimization+of+Parameters+for+Semiempirical+Methods+IV%3A+Extension+of+MNDO%2C+AM1%2C+and+PM3+to+More+Main+Group+Elements&doi=10.1007%2Fs00894-004-0183-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of parameters for semiempirical methods. IV: extension of MNDO, AM1, and PM3 to more main group elements</span></div><div class="casAuthors">Stewart, James J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">155-164</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">The NDDO semiempirical methods MNDO, AM1, and PM3 were extended to all the remaining non-radioactive elements of the main group, excluding the noble gases.  Most of the new elements are of Groups I and II.  44 Sets of parameters are presented for the following methods and elements.  MNDO: Na, Mg, K, Ca, Ga, As, Se, Rb, Sr, In, Sb, Te, Cs, Ba, Tl, and Bi; AM1: Li, Be, Na, Mg, K, Ca, Ga, As, Se, Rb, Sr, In, Sn, Sb, Te, Cs, Ba, Tl, Pb, and Bi; PM3: B, Na, K, Ca, Rb, Sr, Cs, and Ba.  Av. errors are presented for heats of formation, mol. geometries, etc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquoCOa8eJqQ7Vg90H21EOLACvtfcHk0lhr8VvptMCUWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktV2msL8%253D&md5=f06f5d6054d81d44cf41c0b919200a45</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs00894-004-0183-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-004-0183-z%26sid%3Dliteratum%253Aachs%26aulast%3DStewart%26aufirst%3DJ.%2BJ.%2BP.%26atitle%3DOptimization%2520of%2520Parameters%2520for%2520Semiempirical%2520Methods%2520IV%253A%2520Extension%2520of%2520MNDO%252C%2520AM1%252C%2520and%2520PM3%2520to%2520More%2520Main%2520Group%2520Elements%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2004%26volume%3D10%26spage%3D155%26epage%3D164%26doi%3D10.1007%2Fs00894-004-0183-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lepšík, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolář, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobza, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanfrlík, J.</span></span> <span> </span><span class="NLM_article-title">The Semiempirical Quantum Mechanical Scoring Function for in Silico Drug Design</span>. <i>Chempluschem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">921</span>– <span class="NLM_lpage">931</span>, <span class="refDoi"> DOI: 10.1002/cplu.201300199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fcplu.201300199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=31986741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWrsLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=921-931&author=M.+Lep%C5%A1%C3%ADkauthor=J.+%C5%98ez%C3%A1%C4%8Dauthor=M.+Kol%C3%A1%C5%99author=A.+Pecinaauthor=P.+Hobzaauthor=J.+Fanfrl%C3%ADk&title=The+Semiempirical+Quantum+Mechanical+Scoring+Function+for+in+Silico+Drug+Design&doi=10.1002%2Fcplu.201300199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The Semiempirical Quantum Mechanical Scoring Function for In Silico Drug Design</span></div><div class="casAuthors">Lepsik, Martin; Rezac, Jan; Kolar, Michal; Pecina, Adam; Hobza, Pavel; Fanfrlik, Jindrich</div><div class="citationInfo"><span class="NLM_cas:title">ChemPlusChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">921-931</span>CODEN:
                <span class="NLM_cas:coden">CHEMM5</span>;
        ISSN:<span class="NLM_cas:issn">2192-6506</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  This Minireview discusses the latest developments in modern quantum mechanics (QM)-based computer-aided drug design, esp. using semiempirical QM (SQM) methods.  It first tackles biochem. and biophys. quantities and the approaches to their measurements.  Protein-ligand affinities are detd. mostly by noncovalent interactions.  The text thus illustrates how these can be accurately treated with SQM methods.  Next, a construction of a modern SQM-based scoring function is presented and its applications listed.  In summary, SQM-based scoring is a promising modern efficient strategy to be exploited in computer-aided drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1tavGvbuP_rVg90H21EOLACvtfcHk0lhWcdQr2HoIjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWrsLrM&md5=9f9a2633e55a52b720a650dfca71c9ff</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fcplu.201300199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcplu.201300199%26sid%3Dliteratum%253Aachs%26aulast%3DLep%25C5%25A1%25C3%25ADk%26aufirst%3DM.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26aulast%3DKol%25C3%25A1%25C5%2599%26aufirst%3DM.%26aulast%3DPecina%26aufirst%3DA.%26aulast%3DHobza%26aufirst%3DP.%26aulast%3DFanfrl%25C3%25ADk%26aufirst%3DJ.%26atitle%3DThe%2520Semiempirical%2520Quantum%2520Mechanical%2520Scoring%2520Function%2520for%2520in%2520Silico%2520Drug%2520Design%26jtitle%3DChempluschem%26date%3D2013%26volume%3D78%26spage%3D921%26epage%3D931%26doi%3D10.1002%2Fcplu.201300199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fanfrlík, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronowska, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Přenosil, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobza, P.</span></span> <span> </span><span class="NLM_article-title">A Reliable Docking/Scoring Scheme Based on the Semiempirical Quantum Mechanical PM6-DH2 Method Accurately Covering Dispersion and H-Bonding: HIV-1 Protease with 22 Ligands</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">12666</span>– <span class="NLM_lpage">12678</span>, <span class="refDoi"> DOI: 10.1021/jp1032965</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp1032965" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGhsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2010&pages=12666-12678&author=J.+Fanfrl%C3%ADkauthor=A.+K.+Bronowskaauthor=J.+%C5%98ez%C3%A1%C4%8Dauthor=O.+P%C5%99enosilauthor=J.+Konvalinkaauthor=P.+Hobza&title=A+Reliable+Docking%2FScoring+Scheme+Based+on+the+Semiempirical+Quantum+Mechanical+PM6-DH2+Method+Accurately+Covering+Dispersion+and+H-Bonding%3A+HIV-1+Protease+with+22+Ligands&doi=10.1021%2Fjp1032965"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">A Reliable Docking/Scoring Scheme Based on the Semiempirical Quantum Mechanical PM6-DH2 Method Accurately Covering Dispersion and H-Bonding: HIV-1 Protease with 22 Ligands</span></div><div class="casAuthors">Fanfrlik, Jindrich; Bronowska, Agnieszka K.; Rezac, Jan; Prenosil, Ondrej; Konvalinka, Jan; Hobza, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">12666-12678</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, we introduce a fast and reliable rescoring scheme for docked complexes based on a semiempirical quantum mech. PM6-DH2 method.  The method utilizes a PM6-based Hamiltonian with corrections for dispersion energy and hydrogen bonds.  The total score is constructed as the sum of the PM6-DH2 interaction enthalpy, the empirical force field (AMBER) interaction entropy, and the sum of the deformation (PM6-DH2, SMD) and the desolvation (SMD) energies of the ligand.  The main advantage of the procedure is the fact that we do not add any empirical parameter for either an individual component of the total score or an individual protein-ligand complex.  This rescoring method is applied to a very challenging system, namely, the HIV-1 protease with a set of ligands.  As opposed to the conventional DOCK procedure, the PM6-DH2 rescoring based on all of the terms distinguishes between binders and nonbinders and provides a reliable correlation of the theor. and exptl. binding free energies.  Such a dramatic improvement, resulting from the PM6-DH2 rescoring of all the complexes, provides a valuable yet inexpensive tool for rational drug discovery and de novo ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3uSeswff_QbVg90H21EOLACvtfcHk0lhWcdQr2HoIjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGhsb%252FF&md5=8bc9a22daa9392461db7469d90653778</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjp1032965&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp1032965%26sid%3Dliteratum%253Aachs%26aulast%3DFanfrl%25C3%25ADk%26aufirst%3DJ.%26aulast%3DBronowska%26aufirst%3DA.%2BK.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26aulast%3DP%25C5%2599enosil%26aufirst%3DO.%26aulast%3DKonvalinka%26aufirst%3DJ.%26aulast%3DHobza%26aufirst%3DP.%26atitle%3DA%2520Reliable%2520Docking%252FScoring%2520Scheme%2520Based%2520on%2520the%2520Semiempirical%2520Quantum%2520Mechanical%2520PM6-DH2%2520Method%2520Accurately%2520Covering%2520Dispersion%2520and%2520H-Bonding%253A%2520HIV-1%2520Protease%2520with%252022%2520Ligands%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26spage%3D12666%26epage%3D12678%26doi%3D10.1021%2Fjp1032965" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marenich, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truhlar, D. G.</span></span> <span> </span><span class="NLM_article-title">Universal Solvation Model Based on Solute Electron Density and on a Continuum Model of the Solvent Defined by the Bulk Dielectric Constant and Atomic Surface Tensions</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">6378</span>– <span class="NLM_lpage">6396</span>, <span class="refDoi"> DOI: 10.1021/jp810292n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp810292n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksV2is74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=6378-6396&author=A.+V.+Marenichauthor=C.+J.+Cramerauthor=D.+G.+Truhlar&title=Universal+Solvation+Model+Based+on+Solute+Electron+Density+and+on+a+Continuum+Model+of+the+Solvent+Defined+by+the+Bulk+Dielectric+Constant+and+Atomic+Surface+Tensions&doi=10.1021%2Fjp810292n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Universal Solvation Model Based on Solute Electron Density and on a Continuum Model of the Solvent Defined by the Bulk Dielectric Constant and Atomic Surface Tensions</span></div><div class="casAuthors">Marenich, Aleksandr V.; Cramer, Christopher J.; Truhlar, Donald G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6378-6396</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a new continuum solvation model based on the quantum mech. charge d. of a solute mol. interacting with a continuum description of the solvent.  The model is called SMD, where the "D" stands for "d." to denote that the full solute electron d. is used without defining partial at. charges.  "Continuum" denotes that the solvent is not represented explicitly but rather as a dielec. medium with surface tension at the solute-solvent boundary.  SMD is a universal solvation model, where "universal" denotes its applicability to any charged or uncharged solute in any solvent or liq. medium for which a few key descriptors are known (in particular, dielec. const., refractive index, bulk surface tension, and acidity and basicity parameters).  The model separates the observable solvation free energy into two main components.  The first component is the bulk electrostatic contribution arising from a self-consistent reaction field treatment that involves the soln. of the nonhomogeneous Poisson equation for electrostatics in terms of the integral-equation-formalism polarizable continuum model (IEF-PCM).  The cavities for the bulk electrostatic calcn. are defined by superpositions of nuclear-centered spheres.  The second component is called the cavity-dispersion-solvent-structure term and is the contribution arising from short-range interactions between the solute and solvent mols. in the first solvation shell.  This contribution is a sum of terms that are proportional (with geometry-dependent proportionality consts. called at. surface tensions) to the solvent-accessible surface areas of the individual atoms of the solute.  The SMD model has been parametrized with a training set of 2821 solvation data including 112 aq. ionic solvation free energies, 220 solvation free energies for 166 ions in acetonitrile, methanol, and DMSO, 2346 solvation free energies for 318 neutral solutes in 91 solvents (90 nonaq. org. solvents and water), and 143 transfer free energies for 93 neutral solutes between water and 15 org. solvents.  The elements present in the solutes are H, C, N, O, F, Si, P, S, Cl, and Br.  The SMD model employs a single set of parameters (intrinsic at. Coulomb radii and at. surface tension coeffs.) optimized over six electronic structure methods: M05-2X/MIDI!6D, M05-2X/6-31G*, M05-2X/6-31+G**, M05-2X/cc-pVTZ, B3LYP/6-31G*, and HF/6-31G*.  Although the SMD model has been parametrized using the IEF-PCM protocol for bulk electrostatics, it may also be employed with other algorithms for solving the nonhomogeneous Poisson equation for continuum solvation calcns. in which the solute is represented by its electron d. in real space.  This includes, for example, the conductor-like screening algorithm.  With the 6-31G* basis set, the SMD model achieves mean unsigned errors of 0.6-1.0 kcal/mol in the solvation free energies of tested neutrals and mean unsigned errors of 4 kcal/mol on av. for ions with either Gaussian03 or GAMESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQrvm7K8c8dLVg90H21EOLACvtfcHk0lhWcdQr2HoIjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksV2is74%253D&md5=54931a64c70d28445ee53876a8b1a4b9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjp810292n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp810292n%26sid%3Dliteratum%253Aachs%26aulast%3DMarenich%26aufirst%3DA.%2BV.%26aulast%3DCramer%26aufirst%3DC.%2BJ.%26aulast%3DTruhlar%26aufirst%3DD.%2BG.%26atitle%3DUniversal%2520Solvation%2520Model%2520Based%2520on%2520Solute%2520Electron%2520Density%2520and%2520on%2520a%2520Continuum%2520Model%2520of%2520the%2520Solvent%2520Defined%2520by%2520the%2520Bulk%2520Dielectric%2520Constant%2520and%2520Atomic%2520Surface%2520Tensions%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D6378%26epage%3D6396%26doi%3D10.1021%2Fjp810292n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Petersson, G. A.</span></span> <i>Gaussian 09</i>, Revision D. 01, 2009; <span class="NLM_publisher-name">Gaussian Inc.</span>: <span class="NLM_publisher-loc">Wallingford CT</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Robin Kryštůfek, Pavel Šácha, Jana Starková, Jiří Brynda, Martin Hradilek, Eva Tloušt’ová, Justyna Grzymska, Wioletta Rut, Michael J. Boucher, Marcin Drąg, Pavel Majer, Miroslav Hájek, Pavlína Řezáčová, Hiten D. Madhani, Charles S. Craik, <span class="NLM_string-name hlFld-ContribAuthor">Jan Konvalinka</span>. </span><span class="cited-content_cbyCitation_article-title">Re-emerging Aspartic Protease Targets: Examining Cryptococcus neoformans Major Aspartyl Peptidase 1 as a Target for Antifungal Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (10)
                                     , 6706-6719. <a href="https://doi.org/10.1021/acs.jmedchem.0c02177" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02177</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02177%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRe-emerging%252BAspartic%252BProtease%252BTargets%25253A%252BExamining%252BCryptococcus%252Bneoformans%252BMajor%252BAspartyl%252BPeptidase%252B1%252Bas%252Ba%252BTarget%252Bfor%252BAntifungal%252BDrug%252BDiscovery%26aulast%3DKry%25C5%25A1t%25C5%25AFfek%26aufirst%3DRobin%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16122020%26date%3D18052021%26volume%3D64%26issue%3D10%26spage%3D6706%26epage%3D6719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Felix  Brosge</span>, <span class="hlFld-ContribAuthor ">Pardeep  Singh</span>, <span class="hlFld-ContribAuthor ">Fredrik  Almqvist</span>, <span class="hlFld-ContribAuthor ">Carsten  Bolm</span>. </span><span class="cited-content_cbyCitation_article-title">Selected applications of Meldrum's acid – a tutorial. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>19 </em>
                                    (23)
                                     , 5014-5027. <a href="https://doi.org/10.1039/D1OB00395J" title="DOI URL">https://doi.org/10.1039/D1OB00395J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1OB00395J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1OB00395J%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DSelected%252Bapplications%252Bof%252BMeldrum%252527s%252Bacid%252B%2525E2%252580%252593%252Ba%252Btutorial%26aulast%3DBrosge%26aufirst%3DFelix%26date%3D2021%26date%3D2021%26volume%3D19%26issue%3D23%26spage%3D5014%26epage%3D5027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olja  Mijanovic</span>, <span class="hlFld-ContribAuthor ">Anastasiia I.  Petushkova</span>, <span class="hlFld-ContribAuthor ">Ana  Brankovic</span>, <span class="hlFld-ContribAuthor ">Boris  Turk</span>, <span class="hlFld-ContribAuthor ">Anna B.  Solovieva</span>, <span class="hlFld-ContribAuthor ">Angelina I.  Nikitkina</span>, <span class="hlFld-ContribAuthor ">Sergey  Bolevich</span>, <span class="hlFld-ContribAuthor ">Peter S.  Timashev</span>, <span class="hlFld-ContribAuthor ">Alessandro  Parodi</span>, <span class="hlFld-ContribAuthor ">Andrey A.  Zamyatnin</span>. </span><span class="cited-content_cbyCitation_article-title">Cathepsin D—Managing the Delicate Balance. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutics</span><span> <strong>2021,</strong> <em>13 </em>
                                    (6)
                                     , 837. <a href="https://doi.org/10.3390/pharmaceutics13060837" title="DOI URL">https://doi.org/10.3390/pharmaceutics13060837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/pharmaceutics13060837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fpharmaceutics13060837%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutics%26atitle%3DCathepsin%252BD%2525E2%252580%252594Managing%252Bthe%252BDelicate%252BBalance%26aulast%3DMijanovic%26aufirst%3DOlja%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D6%26spage%3D837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiří  Dostál</span>, <span class="hlFld-ContribAuthor ">Jiří  Brynda</span>, <span class="hlFld-ContribAuthor ">Lucie  Vaňková</span>, <span class="hlFld-ContribAuthor ">Syeda Rehana  Zia</span>, <span class="hlFld-ContribAuthor ">Iva  Pichová</span>, <span class="hlFld-ContribAuthor ">Olga  Heidingsfeld</span>, <span class="hlFld-ContribAuthor ">Martin  Lepšík</span>. </span><span class="cited-content_cbyCitation_article-title">Structural determinants for subnanomolar inhibition of the secreted aspartic protease Sapp1p from
              Candida parapsilosis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2021,</strong> <em>36 </em>
                                    (1)
                                     , 914-921. <a href="https://doi.org/10.1080/14756366.2021.1906664" title="DOI URL">https://doi.org/10.1080/14756366.2021.1906664</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2021.1906664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2021.1906664%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DStructural%252Bdeterminants%252Bfor%252Bsubnanomolar%252Binhibition%252Bof%252Bthe%252Bsecreted%252Baspartic%252Bprotease%252BSapp1p%252Bfrom%252BCandida%252Bparapsilosis%26aulast%3DDost%25C3%25A1l%26aufirst%3DJi%25C5%2599%25C3%25AD%26date%3D2021%26date%3D2021%26volume%3D36%26issue%3D1%26spage%3D914%26epage%3D921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Pecina</span>, <span class="hlFld-ContribAuthor ">Saltuk M.  Eyrilmez</span>, <span class="hlFld-ContribAuthor ">Cemal  Köprülüoğlu</span>, <span class="hlFld-ContribAuthor ">Vijay Madhav  Miriyala</span>, <span class="hlFld-ContribAuthor ">Martin  Lepšík</span>, <span class="hlFld-ContribAuthor ">Jindřich  Fanfrlík</span>, <span class="hlFld-ContribAuthor ">Jan  Řezáč</span>, <span class="hlFld-ContribAuthor ">Pavel  Hobza</span>. </span><span class="cited-content_cbyCitation_article-title">SQM/COSMO Scoring Function: Reliable Quantum‐Mechanical Tool for Sampling and Ranking in Structure‐Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">ChemPlusChem</span><span> <strong>2020,</strong> <em>85 </em>
                                    (11)
                                     , 2362-2371. <a href="https://doi.org/10.1002/cplu.202000120" title="DOI URL">https://doi.org/10.1002/cplu.202000120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cplu.202000120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcplu.202000120%26sid%3Dliteratum%253Aachs%26jtitle%3DChemPlusChem%26atitle%3DSQM%25252FCOSMO%252BScoring%252BFunction%25253A%252BReliable%252BQuantum%2525E2%252580%252590Mechanical%252BTool%252Bfor%252BSampling%252Band%252BRanking%252Bin%252BStructure%2525E2%252580%252590Based%252BDrug%252BDesign%26aulast%3DPecina%26aufirst%3DAdam%26date%3D2020%26date%3D2020%26volume%3D85%26issue%3D11%26spage%3D2362%26epage%3D2371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valeria  De Pasquale</span>, <span class="hlFld-ContribAuthor ">Anna  Moles</span>, <span class="hlFld-ContribAuthor ">Luigi Michele  Pavone</span>. </span><span class="cited-content_cbyCitation_article-title">Cathepsins in the Pathophysiology of Mucopolysaccharidoses: New Perspectives for Therapy. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2020,</strong> <em>9 </em>
                                    (4)
                                     , 979. <a href="https://doi.org/10.3390/cells9040979" title="DOI URL">https://doi.org/10.3390/cells9040979</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells9040979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells9040979%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DCathepsins%252Bin%252Bthe%252BPathophysiology%252Bof%252BMucopolysaccharidoses%25253A%252BNew%252BPerspectives%252Bfor%252BTherapy%26aulast%3DDe%2BPasquale%26aufirst%3DValeria%26date%3D2020%26date%3D2020%26volume%3D9%26issue%3D4%26spage%3D979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic representation of the rational design of the macrocyclic inhibitor scaffold. The scaffold mimics the spatial conformation of pepstatin A, a natural linear oligopeptide inhibitor, in the active site of human CatD (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYB">1LYB</a>). Binding subsites (S) are marked and colored in blue; corresponding inhibitor positions (P) are in brown. The central hydroxyl group of the statine residue that interacts with the catalytic aspartates of CatD is highlighted in yellow. Lines with scissors indicate the region used for macrocycle scaffold design. Pepstatin A structure: Iva-Val-Val-Sta-Ala-Sta, where Iva is isovaleryl and Sta is statine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Statine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a): R = 2-naphthyl; (b): R = 4-thiazolyl; (c): R = 4-Me-C<sub>6</sub>H<sub>4</sub>-CH<sub>2</sub>S; (d): R = 3,4-dichlorophenyl; (e): R = 3,4-difluorophenyl; (f): R = 4-fluorophenyl. Reagents and conditions: (a) DCC, DMAP, Meldrum’s acid, DCM; (b) EtOAc, reflux; (c) NaBH<sub>4</sub>, DCM/AcOH/10:1; (d) NaOH, acetone.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Methyl 14-Aminotetradecanoate<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DCC, DMAP, Meldrum’s acid, DCM; (b) NaBH<sub>4</sub>, DCM/AcOH/10:1; (c) 110 °C, DMF/H<sub>2</sub>O/10:1; (d) (1) SOCl<sub>2</sub>, MeOH, 0 °C; (2) TFA.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. General Synthesis of Macrocycles with an Aliphatic Chain<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a): R<sup>1</sup> = benzyl, R<sup>2</sup> = isopropyl; (b): R<sup>1</sup> = benzyl, R<sup>2</sup> = 3-pyridylmethyl. Reagents and conditions: (a) HBTU, DIEA, DMF; (b) (1) TFA, DCM; (2) Boc-statine-OH, HBTU, DIEA, DMF; (c) LiOH, THF/H<sub>2</sub>O; (d) (1) TSTU, DIEA, DMF; (2) TFA; (3) DIEA, DMF.</p></p></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Macrocycle with a PEG Chain<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (1) HBTU, DIEA, DMF; (2) H-Val-OMe; (b) (1) TFA; (2) BocHN-PEG-3-CH<sub>2</sub>CH<sub>2</sub>COOH, HBTU, DIEA, DMF; (c) (1) LiOH, THF/H<sub>2</sub>O; (2) TFA; (3) DPPA, DIEA, DMF.</p></p></figure><figure data-id="sch5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Macrocycle with a Carbamate Moiety<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DIEA, DMF; (b) (1) TFA, DCM; (2) Boc-Statine-OH, TSTU, DIEA, DMF; (c) (1) DSC, DMF; (2) TFA, DCM; (3) DIEA, DMF.</p></p></figure><figure data-id="fig2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structures of human CatD in complex with three macrocyclic inhibitors (<b>9</b>, <b>14</b>, and <b>33</b>, PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QCB">6QCB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBG">6QBG</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBH">6QBH</a>). (<b>a</b>) Top view of the CatD active site with the superimposed macrocyclic inhibitors. The enzyme is shown as semitransparent surface and the inhibitors as sticks (<b>9</b>, magenta; <b>14</b>, green; <b>33</b>, cyan). The active site residues are colored as follows: Tyr flap residues are pale blue, polyproline loop residues are pale orange, and the rest of the residues are light yellow. (b) Chemical structures and electron density maps of macrocyclic inhibitors. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> maps are contoured at 1.2σ. (c) Hydrogen bond networks (dashed lines) formed between CatD and macrocyclic inhibitors are shown in two views rotated by 180°. Inhibitors are colored as in (b); interacting enzyme residues are gray. Heteroatoms have standard color coding. (d) Superposition of macrocyclic inhibitors colored as in (b) and pepstatin A (yellow) in the CatD active site. The enzyme is in semitransparent surface representation, and residues that form hydrophobic interactions with inhibitors are highlighted. The residues interacting with the macrocycles are pale cyan, with pepstatin A are yellow, and with both inhibitors are pale green. (e) Hydrogen bond network (dashed lines) formed between CatD and pepstatin A. Pepstatin A is colored yellow; interacting enzyme residues are gray. Yellow boxes indicate segments of pepstatin A that are absent in the macrocycle scaffold. Residues labeled in brown form pepstatin A-specific hydrogen bonds; macrocycle-specific hydrogen bonding is labeled in (c).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Quantum chemical calculations of interaction “free” energies of three crystallographically characterized macrocyclic inhibitors in the CatD active site. The 3D view shows fragmentation of an inhibitor into segments (alternating black and orange) and the active site residues of CatD (gray) used for the calculations (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QCB">6QCB</a>). (a) Contributions of individual inhibitor segments. (b) Contributions of the active site residues obtained from a “virtual glycine” scan.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/medium/jm9b01351_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Inhibitory selectivity of macrocyclic compounds against representative aspartic proteases. Inhibition of proteolytic activity was screened in a kinetic activity assay with FRET fluorogenic substrates. Inhibitors were applied at 50 nM concentration (upper panel) for human cathepsin D, tick cathepsin D1 (IrCatD), yeast proteinase A (ProtA), and porcine pepsin (pPEP), and at 5 μM concentration (lower panel) for human β-secretase (BACE1), human renin, and HIV-1 protease (HIV1 PR). Data are expressed as % inhibition relative to the uninhibited control (0%) and displayed as a heat map (blue and green scales). Inhibitors are grouped according to their appearance in data tables.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-4/acs.jmedchem.9b01351/20200221/images/large/jm9b01351_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01351&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i111">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31471" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31471" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 65 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fusek, M.</span>; <span class="NLM_string-name">Mareš, M.</span>; <span class="NLM_string-name">Větvička, V.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D</span>. In  <i>Handbook of Proteolytic Enzymes</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlings, N.</span>, <span class="NLM_string-name">Salvesen, G.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Oxford</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">63</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2FB978-0-12-382219-2.00008-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=54-63&author=M.+Fusek&author=M.+Mare%C5%A1&author=V.+V%C4%9Btvi%C4%8Dkaauthor=N.+Rawlings&author=G.+Salvesen&title=Handbook+of+Proteolytic+Enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-382219-2.00008-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-382219-2.00008-9%26sid%3Dliteratum%253Aachs%26aulast%3DFusek%26aufirst%3DM.%26atitle%3DCathepsin%2520D%26btitle%3DHandbook%2520of%2520Proteolytic%2520Enzymes%26aulast%3DRawlings%26aufirst%3DN.%26pub%3DAcademic%2520Press%26date%3D2013%26spage%3D54%26epage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melone, M. A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isidoro, C.</span></span> <span> </span><span class="NLM_article-title">The Role of Cathepsin D in the Pathogenesis of Human Neurodegenerative Disorders</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">870</span>, <span class="refDoi"> DOI: 10.1002/med.21394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fmed.21394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=27114232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=845-870&author=C.+Vidoniauthor=C.+Folloauthor=M.+Savinoauthor=M.+A.+B.+Meloneauthor=C.+Isidoro&title=The+Role+of+Cathepsin+D+in+the+Pathogenesis+of+Human+Neurodegenerative+Disorders&doi=10.1002%2Fmed.21394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Cathepsin D in the Pathogenesis of Human Neurodegenerative Disorders</span></div><div class="casAuthors">Vidoni, Chiara; Follo, Carlo; Savino, Miriam; Melone, Mariarosa A. B.; Isidoro, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">845-870</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  A review.  In familial neurodegenerative disorders, protein aggregates form continuously because of genetic mutations that drive the synthesis of truncated or unfolded proteins.  The oxidative stress imposed by neurotransmitters and environmental neurotoxins constitutes an addnl. threat to the folding of the proteins and the integrity of organelle membranes in neurons.  Failure in degrading such altered materials compromises the function of neurons and eventually leads to neurodegeneration.  The lysosomal proteolytic enzyme Cathepsin D is the only aspartic-type protease ubiquitously expressed in all the cells of the human body, and it is expressed at high level in the brain.  In general, cathepsin D mediated proteolysis is essential to neuronal cell homeostasis through the degrdn. of unfolded or oxidized protein aggregates delivered to lysosomes via autophagy or endocytosis.  More specifically, many altered neuronal proteins that hallmark neurodegenerative diseases (e.g., the amyloid precursor, α-synuclein, and huntingtin) are physiol. substrates of cathepsin D and would abnormally accumulate if not efficiently degraded by this enzyme.  Furthermore, exptl. evidence indicates that cathepsin D activity is linked to the metab. of cholesterol and of glycosaminoglycans, which accounts for its involvement in neuronal plasticity.  This review focuses on the unique role of cathepsin D mediated proteolysis in the pathogenesis of human neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCbQT1BCqB7rVg90H21EOLACvtfcHk0lj0p8-EbL36Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSms7bE&md5=2e0d3c04b6038f01e77a515dfc329c22</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fmed.21394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21394%26sid%3Dliteratum%253Aachs%26aulast%3DVidoni%26aufirst%3DC.%26aulast%3DFollo%26aufirst%3DC.%26aulast%3DSavino%26aufirst%3DM.%26aulast%3DMelone%26aufirst%3DM.%2BA.%2BB.%26aulast%3DIsidoro%26aufirst%3DC.%26atitle%3DThe%2520Role%2520of%2520Cathepsin%2520D%2520in%2520the%2520Pathogenesis%2520of%2520Human%2520Neurodegenerative%2520Disorders%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D845%26epage%3D870%26doi%3D10.1002%2Fmed.21394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aghdassi, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinheckel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayerle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerch, M. M.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D Regulates Cathepsin B Activation and Disease Severity Predominantly in Inflammatory Cells during Experimental Pancreatitis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1029</span>, <span class="refDoi"> DOI: 10.1074/jbc.m117.814772</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1074%2Fjbc.m117.814772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=29229780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=1018-1029&author=A.+A.+Aghdassiauthor=D.+S.+Johnauthor=M.+Sendlerauthor=F.+U.+Weissauthor=T.+Reinheckelauthor=J.+Mayerleauthor=M.+M.+Lerch&title=Cathepsin+D+Regulates+Cathepsin+B+Activation+and+Disease+Severity+Predominantly+in+Inflammatory+Cells+during+Experimental+Pancreatitis&doi=10.1074%2Fjbc.m117.814772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D regulates cathepsin B activation and disease severity predominantly in inflammatory cells during experimental pancreatitis</span></div><div class="casAuthors">Aghdassi, Ali A.; John, Daniel S.; Sendler, Matthias; Weiss, F. Ulrich; Reinheckel, Thomas; Mayerle, Julia; Lerch, Markus M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1018-1029</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Acute pancreatitis is a complex disorder involving both premature intracellular protease activation and inflammatory cell invasion.  An initiating event is the intracellular activation of trypsinogen by cathepsin B (CTSB), which can be induced directly via G protein-coupled receptors on acinar cells or through inflammatory cells.  Here, we studied CTSB regulation by another lysosomal hydrolase, cathepsin D (CTSD), using mice with a complete (CTSD-/-) or pancreas-specific conditional CTSD knockout (KO) (CTSDf/f/p48Cre/+).  We induced acute pancreatitis by repeated caerulein injections and isolated acinar and bone marrow cells for ex vivo studies.  Supramaximal caerulein stimulation induced subcellular redistribution of CTSD from the lysosomal to the zymogen-contg. subcellular compartment of acinar cells and activation of CTSD, CTSB, and trypsinogen.  Of note, the CTSD KO greatly reduced CTSB and trypsinogen activation in acinar cells, and CTSD directly activated CTSB but not trypsinogen in vitro.  During pancreatitis in pancreas-specific CTSDf/f/p48Cre/+ animals, markers of severity were reduced only at 1 h, whereas in the complete KO, this effect also included the late disease phase (8 h), indicating an important effect of extra-acinar CTSD on course of the disease.  CTSD-/- leukocytes exhibited reduced cytokine release after lipopolysaccharide (LPS) stimulation, and CTSD KO also reduced caspase-3 activation and apoptosis in acinar cells stimulated with the intestinal hormone cholecystokinin.  In summary, CTSD is expressed in pancreatic acinar and inflammatory cells, undergoes subcellular redistribution and activation during exptl. pancreatitis, and regulates disease severity by potently activating CTSB.  Its impact is only minimal and transient in the early, acinar cell-dependent phase of pancreatitis and much greater in the later, inflammatory cell-dependent phase of the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5K4-R8ZsIfrVg90H21EOLACvtfcHk0lj3LxKuVn2FtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjsrs%253D&md5=a79fd4fd21337c3fc81daa9cbc63db84</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m117.814772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m117.814772%26sid%3Dliteratum%253Aachs%26aulast%3DAghdassi%26aufirst%3DA.%2BA.%26aulast%3DJohn%26aufirst%3DD.%2BS.%26aulast%3DSendler%26aufirst%3DM.%26aulast%3DWeiss%26aufirst%3DF.%2BU.%26aulast%3DReinheckel%26aufirst%3DT.%26aulast%3DMayerle%26aufirst%3DJ.%26aulast%3DLerch%26aufirst%3DM.%2BM.%26atitle%3DCathepsin%2520D%2520Regulates%2520Cathepsin%2520B%2520Activation%2520and%2520Disease%2520Severity%2520Predominantly%2520in%2520Inflammatory%2520Cells%2520during%2520Experimental%2520Pancreatitis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D1018%26epage%3D1029%26doi%3D10.1074%2Fjbc.m117.814772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liaudet-Coopman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaujouin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derocq, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glondu-Lassis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent-Matha, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prébois, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rochefort, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vignon, F.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D: Newly Discovered Functions of a Long-Standing Aspartic Protease in Cancer and Apoptosis</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>237</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2005.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.canlet.2005.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=16046058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD28XltF2qsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=237&publication_year=2006&pages=167-179&author=E.+Liaudet-Coopmanauthor=M.+Beaujouinauthor=D.+Derocqauthor=M.+Garciaauthor=M.+Glondu-Lassisauthor=V.+Laurent-Mathaauthor=C.+Pr%C3%A9boisauthor=H.+Rochefortauthor=F.+Vignon&title=Cathepsin+D%3A+Newly+Discovered+Functions+of+a+Long-Standing+Aspartic+Protease+in+Cancer+and+Apoptosis&doi=10.1016%2Fj.canlet.2005.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D: Newly discovered functions of a long-standing aspartic protease in cancer and apoptosis</span></div><div class="casAuthors">Liaudet-Coopman, Emmanuelle; Beaujouin, Melanie; Derocq, Danielle; Garcia, Marcel; Glondu-Lassis, Murielle; Laurent-Matha, Valerie; Prebois, Christine; Rochefort, Henri; Vignon, Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">237</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-179</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The lysosomal aspartic protease cathepsin D (cath-D) is over-expressed and hyper-secreted by epithelial breast cancer cells.  This protease is an independent marker of poor prognosis in breast cancer being correlated with the incidence of clin. metastasis.  Cath-D over-expression stimulates tumorigenicity and metastasis.  Indeed it plays an essential role in the multiple steps of tumor progression, in stimulating cancer cell proliferation, fibroblast outgrowth and angiogenesis, as well as in inhibiting tumor apoptosis.  A mutated cath-D devoid of catalytic activity still proved mitogenic for cancer, endothelial and fibroblastic cells, suggesting an extra-cellular mode of action of cath-D involving a triggering, either directly or indirectly, of an as yet unidentified cell surface receptor.  Cath-D is also a key mediator of induced-apoptosis and its proteolytic activity has been involved generally in this event.  During apoptosis, mature lysosomal cath-D is translocated to the cytosol.  Since cath-D is one of the lysosomal enzymes which requires a more acidic pH to be proteolytically-active relative to the cysteine lysosomal enzymes, such as cath-B and -L, it is open to question whether cytosolic cath-D might be able to cleave substrate(s) implicated in the apoptotic cascade.  This review summarises our current knowledge on cath-D action in cancer progression and metastasis, as well as its dual function in apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFJdjUCIwta7Vg90H21EOLACvtfcHk0lj3LxKuVn2FtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltF2qsbc%253D&md5=eb7be2df7685a6574ca0b161a6c9716d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2005.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2005.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DLiaudet-Coopman%26aufirst%3DE.%26aulast%3DBeaujouin%26aufirst%3DM.%26aulast%3DDerocq%26aufirst%3DD.%26aulast%3DGarcia%26aufirst%3DM.%26aulast%3DGlondu-Lassis%26aufirst%3DM.%26aulast%3DLaurent-Matha%26aufirst%3DV.%26aulast%3DPr%25C3%25A9bois%26aufirst%3DC.%26aulast%3DRochefort%26aufirst%3DH.%26aulast%3DVignon%26aufirst%3DF.%26atitle%3DCathepsin%2520D%253A%2520Newly%2520Discovered%2520Functions%2520of%2520a%2520Long-Standing%2520Aspartic%2520Protease%2520in%2520Cancer%2520and%2520Apoptosis%26jtitle%3DCancer%2520Lett.%26date%3D2006%26volume%3D237%26spage%3D167%26epage%3D179%26doi%3D10.1016%2Fj.canlet.2005.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetvicka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fusek, M.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D—Many Functions of One Aspartic Protease</span>. <i>Crit. Rev. Oncol. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1016/j.critrevonc.2008.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.critrevonc.2008.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=18396408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADC%252BD1crnsVehsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=12-28&author=P.+Benesauthor=V.+Vetvickaauthor=M.+Fusek&title=Cathepsin+D%E2%80%94Many+Functions+of+One+Aspartic+Protease&doi=10.1016%2Fj.critrevonc.2008.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D--many functions of one aspartic protease</span></div><div class="casAuthors">Benes Petr; Vetvicka Vaclav; Fusek Martin</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-28</span>
        ISSN:<span class="NLM_cas:issn">1040-8428</span>.
    </div><div class="casAbstract">For years, it has been held that cathepsin D (CD) is involved in rather non-specific protein degradation in a strongly acidic milieu of lysosomes.  Studies with CD knock-out mice revealed that CD is not necessary for embryonal development, but it is indispensable for postnatal tissue homeostasis.  Mutation that abolishes CD enzymatic activity causes neuronal ceroid lipofuscinosis (NCL) characterized by severe neurodegeneration, developmental regression, visual loss and epilepsy in both animals and humans.  In the last decade, however, an increasing number of studies demonstrated that enzymatic function of CD is not restricted solely to acidic milieu of lysosomes with important consequences in regulation of apoptosis.  In addition to CD enzymatic activity, it has been shown that apoptosis is also regulated by catalytically inactive mutants of CD which suggests that CD interacts with other important molecules and influences cell signaling.  Moreover, procathepsin D (pCD), secreted from cancer cells, acts as a mitogen on both cancer and stromal cells and stimulates their pro-invasive and pro-metastatic properties.  Numerous studies found that pCD/CD level represents an independent prognostic factor in a variety of cancers and is therefore considered to be a potential target of anti-cancer therapy.  Studies dealing with functions of cathepsin D are complicated by the fact that there are several simultaneous forms of CD in a cell-pCD, intermediate enzymatically active CD and mature heavy and light chain CD.  It became evident that these forms may differently regulate the above-mentioned processes.  In this article, we review the possible functions of CD and its various forms in cells and organisms during physiological and pathological conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS9gz1Ed--NTiEJCPKhJPuqfW6udTcc2eY2gAqwL20Ke7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1crnsVehsw%253D%253D&md5=eb34f9d185621ee4b845685ec1efa125</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2008.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2008.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DBenes%26aufirst%3DP.%26aulast%3DVetvicka%26aufirst%3DV.%26aulast%3DFusek%26aufirst%3DM.%26atitle%3DCathepsin%2520D%25E2%2580%2594Many%2520Functions%2520of%2520One%2520Aspartic%2520Protease%26jtitle%3DCrit.%2520Rev.%2520Oncol.%2520Hematol.%26date%3D2008%26volume%3D68%26spage%3D12%26epage%3D28%26doi%3D10.1016%2Fj.critrevonc.2008.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Look, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolt-de
Vries, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer-van Gelder, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Putten, W. L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klijn, J. G. M.</span></span> <span> </span><span class="NLM_article-title">Cathepsin-D in Primary Breast Cancer: Prognostic Evaluation Involving 2810 Patients</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">307</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6690048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1038%2Fsj.bjc.6690048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=9888472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADyaK1M7gtlGntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1999&pages=300-307&author=J.+A.+Foekensauthor=M.+P.+Lookauthor=J.+B.+Bolt-de%0AVriesauthor=M.+E.+Meijer-van+Gelderauthor=W.+L.+J.+van+Puttenauthor=J.+G.+M.+Klijn&title=Cathepsin-D+in+Primary+Breast+Cancer%3A+Prognostic+Evaluation+Involving+2810+Patients&doi=10.1038%2Fsj.bjc.6690048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin-D in primary breast cancer: prognostic evaluation involving 2810 patients</span></div><div class="casAuthors">Foekens J A; Look M P; Bolt-de Vries J; Meijer-van Gelder M E; van Putten W L; Klijn J G</div><div class="citationInfo"><span class="NLM_cas:title">British journal of cancer</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-7</span>
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    </div><div class="casAbstract">There is controversy regarding the prognostic value of cathepsin-D in primary breast cancer.  An increased level of cathepsin-D in tumour extracts has been found to be associated with a poor relapse-free and overall survival.  Studies performed with immunohistochemistry or Western blotting have produced diverse results.  We have analysed 2810 cytosolic extracts obtained from human primary breast tumours for cathepsin-D expression, and have correlated their levels with prognosis.  The median follow-up of the patients still alive was 88 months.  Patients with high cathepsin-D levels had a significantly worse relapse-free and overall survival, also in multivariate analysis (P < 0.0001).  Adjuvant therapy which was associated with an improved prognosis in node-positive patients in univariate analysis, also significantly added to the multivariate models for relapse-free and overall survival.  There were no statistically significant interactions between the levels of cathepsin-D and any of the classical prognostic factors in analysis for relapse-free survival, suggesting that the prognostic value of cathepsin-D is not different in the various subgroups of patients.  Indeed, multivariate analyses in subgroups of node-negative and -positive patients, pre- and post-menopausal patients, and their combinations, showed that tumours with high cathepsin-D values had a significantly poor relapse-free survival, with relative hazard rates ranging from 1.3 to 1.5, compared with tumours with low cathepsin-D levels.  The results presented here on 2810 patients confirm that high cytosolic cathepsin-D values are associated with poor prognosis in human primary breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSvgQsgnUMs2JotVjB3IRIyfW6udTcc2eZRruTBA-ztw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1M7gtlGntQ%253D%253D&md5=afd3571f4ccb8e410d7bb119c0a7c1d0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6690048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6690048%26sid%3Dliteratum%253Aachs%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26aulast%3DLook%26aufirst%3DM.%2BP.%26aulast%3DBolt-de%2BVries%26aufirst%3DJ.%2BB.%26aulast%3DMeijer-van%2BGelder%26aufirst%3DM.%2BE.%26aulast%3Dvan%2BPutten%26aufirst%3DW.%2BL.%2BJ.%26aulast%3DKlijn%26aufirst%3DJ.%2BG.%2BM.%26atitle%3DCathepsin-D%2520in%2520Primary%2520Breast%2520Cancer%253A%2520Prognostic%2520Evaluation%2520Involving%25202810%2520Patients%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D1999%26volume%3D79%26spage%3D300%26epage%3D307%26doi%3D10.1038%2Fsj.bjc.6690048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferrandina, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scambia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti Panici, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancuso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messori, A.</span></span> <span> </span><span class="NLM_article-title">Relationship between Cathepsin-D Content and Disease-Free Survival in Node-Negative Breast Cancer Patients: A Meta-Analysis</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1038/bjc.1997.442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1038%2Fbjc.1997.442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=9303368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADyaK2svkslKhug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=1997&pages=661-666&author=G.+Ferrandinaauthor=G.+Scambiaauthor=F.+Bardelliauthor=P.+Benedetti+Paniciauthor=S.+Mancusoauthor=A.+Messori&title=Relationship+between+Cathepsin-D+Content+and+Disease-Free+Survival+in+Node-Negative+Breast+Cancer+Patients%3A+A+Meta-Analysis&doi=10.1038%2Fbjc.1997.442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Relationship between cathepsin-D content and disease-free survival in node-negative breast cancer patients: a meta-analysis</span></div><div class="casAuthors">Ferrandina G; Scambia G; Bardelli F; Benedetti Panici P; Mancuso S; Messori A</div><div class="citationInfo"><span class="NLM_cas:title">British journal of cancer</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">661-6</span>
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    </div><div class="casAbstract">Several reports have evaluated the correlation between cathepsin-D and overall survival or disease-free survival in node-negative breast cancer patients.  Because conflicting data have so far been reported, a meta-analysis was conducted to clarify this problem.  Eleven studies were included in our meta-analysis (total of 2690 patients).  A specific meta-analytical methodology for censored data was used, and disease-free survival was the primary end point.  Patients with low cathepsin-D levels had a significantly better disease-free survival than patients with high cathepsin-D values (meta-analytical odds ratio from 0.59 to 0.60 over the interval from 1 to 7 years).  A secondary meta-analysis conducted exclusively on the data from eight studies based on cytosol assay gave substantially similar results.  One limitation of our study is that the cut-off values to define high and low cathepsin-D concentrations were not identical in the various studies included in our meta-analysis (range from 20 to 78 pmol mg(-1) protein), thus introducing a possible bias in the statistical analysis of the data.  However, a simulation based on the well-accepted method of the so-called publication bias showed that more than 100 null studies would be required to lead our results to a statistical level of non-significance.  Considering the results of our meta-analysis, we conclude that the data presently available confirm a statistically significant association between high cathepsin-D values and poor disease-free survival in node-negative breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDXEt60c3tNsjl5L8hwNX0fW6udTcc2eZRruTBA-ztw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2svkslKhug%253D%253D&md5=e0726118891dec91f1e4ffdf964f571a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fbjc.1997.442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.1997.442%26sid%3Dliteratum%253Aachs%26aulast%3DFerrandina%26aufirst%3DG.%26aulast%3DScambia%26aufirst%3DG.%26aulast%3DBardelli%26aufirst%3DF.%26aulast%3DBenedetti%2BPanici%26aufirst%3DP.%26aulast%3DMancuso%26aufirst%3DS.%26aulast%3DMessori%26aufirst%3DA.%26atitle%3DRelationship%2520between%2520Cathepsin-D%2520Content%2520and%2520Disease-Free%2520Survival%2520in%2520Node-Negative%2520Breast%2520Cancer%2520Patients%253A%2520A%2520Meta-Analysis%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D1997%26volume%3D76%26spage%3D661%26epage%3D666%26doi%3D10.1038%2Fbjc.1997.442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashraf, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansouri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent-Matha, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guiu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derocq, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, H.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delpech, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarlier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugnière, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landomiel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourquard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achour, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fruitier-Arnaudin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshayes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turtoi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poupon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simony-Lafontaine, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boissière-Michot, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernex, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacot, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">du Manoir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theillet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouget, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro-Teulon, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefoy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pèlegrin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chardès, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martineau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaudet-Coopman, E.</span></span> <span> </span><span class="NLM_article-title">Immunotherapy of Triple-Negative Breast Cancer with Cathepsin D-Targeting Antibodies</span>. <i>J. Immunother. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">29</span>, <span class="refDoi"> DOI: 10.1186/s40425-019-0498-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1186%2Fs40425-019-0498-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=30717773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADC%252BB3cjovVyksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=29&author=Y.+Ashrafauthor=H.+Mansouriauthor=V.+Laurent-Mathaauthor=L.+B.+Alcarazauthor=P.+Rogerauthor=S.+Guiuauthor=D.+Derocqauthor=G.+Robinauthor=H.-A.+Michaudauthor=H.+Delpechauthor=M.+Jarlierauthor=M.+Pugni%C3%A8reauthor=B.+Robertauthor=A.+Puelauthor=L.+Martinauthor=F.+Landomielauthor=T.+Bourquardauthor=O.+Achourauthor=I.+Fruitier-Arnaudinauthor=A.+Pichardauthor=E.+Deshayesauthor=A.+Turtoiauthor=A.+Pouponauthor=J.+Simony-Lafontaineauthor=F.+Boissi%C3%A8re-Michotauthor=N.+Pirotauthor=F.+Bernexauthor=W.+Jacotauthor=S.+du+Manoirauthor=C.+Theilletauthor=J.-P.+Pougetauthor=I.+Navarro-Teulonauthor=N.+Bonnefoyauthor=A.+P%C3%A8legrinauthor=T.+Chard%C3%A8sauthor=P.+Martineauauthor=E.+Liaudet-Coopman&title=Immunotherapy+of+Triple-Negative+Breast+Cancer+with+Cathepsin+D-Targeting+Antibodies&doi=10.1186%2Fs40425-019-0498-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Immunotherapy of triple-negative breast cancer with cathepsin D-targeting antibodies</span></div><div class="casAuthors">Ashraf Yahya; Mansouri Hanane; Laurent-Matha Valerie; Alcaraz Lindsay B; Roger Pascal; Guiu Severine; Derocq Danielle; Robin Gautier; Michaud Henri-Alexandre; Delpech Helene; Pugniere Martine; Robert Bruno; Puel Anthony; Martin Lucie; Pichard Alexandre; Deshayes Emmanuel; Turtoi Andrei; Jacot William; du Manoir Stanislas; Theillet Charles; Pouget Jean-Pierre; Navarro-Teulon Isabelle; Bonnefoy Nathalie; Pelegrin Andre; Chardes Thierry; Martineau Pierre; Liaudet-Coopman Emmanuelle; Roger Pascal; Guiu Severine; Jacot William; Jarlier Marta; Landomiel Flavie; Bourquard Thomas; Poupon Anne; Achour Oussama; Fruitier-Arnaudin Ingrid; Simony-Lafontaine Joelle; Boissiere-Michot Florence; Jacot William; Pirot Nelly; Bernex Florence</div><div class="citationInfo"><span class="NLM_cas:title">Journal for immunotherapy of cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Triple-negative breast cancer (TNBC) treatment is currently restricted to chemotherapy.  Hence, tumor-specific molecular targets and/or alternative therapeutic strategies for TNBC are urgently needed.  Immunotherapy is emerging as an exciting treatment option for TNBC patients.  The aspartic protease cathepsin D (cath-D), a marker of poor prognosis in breast cancer (BC), is overproduced and hypersecreted by human BC cells.  This study explores whether cath-D is a tumor cell-associated extracellular biomarker and a potent target for antibody-based therapy in TNBC.  METHODS:  Cath-D prognostic value and localization was evaluated by transcriptomics, proteomics and immunohistochemistry in TNBC.  First-in-class anti-cath-D human scFv fragments binding to both human and mouse cath-D were generated using phage display and cloned in the human IgG1 λ format (F1 and E2).  Anti-cath-D antibody biodistribution, antitumor efficacy and in vivo underlying mechanisms were investigated in TNBC MDA-MB-231 tumor xenografts in nude mice.  Antitumor effect was further assessed in TNBC patient-derived xenografts (PDXs).  RESULTS:  High CTSD mRNA levels correlated with shorter recurrence-free survival in TNBC, and extracellular cath-D was detected in the tumor microenvironment, but not in matched normal breast stroma.  Anti-cath-D F1 and E2 antibodies accumulated in TNBC MDA-MB-231 tumor xenografts, inhibited tumor growth and improved mice survival without apparent toxicity.  The Fc function of F1, the best antibody candidate, was essential for maximal tumor inhibition in the MDA-MB-231 model.  Mechanistically, F1 antitumor response was triggered through natural killer cell activation via IL-15 upregulation, associated with granzyme B and perforin production, and the release of antitumor IFNγ cytokine.  The F1 antibody also prevented the tumor recruitment of immunosuppressive tumor-associated macrophages M2 and myeloid-derived suppressor cells, a specific effect associated with a less immunosuppressive tumor microenvironment highlighted by TGFβ decrease.  Finally, the antibody F1 inhibited tumor growth of two TNBC PDXs, isolated from patients resistant or not to neo-adjuvant chemotherapy.  CONCLUSION:  Cath-D is a tumor-specific extracellular target in TNBC suitable for antibody-based therapy.  Immunomodulatory antibody-based strategy against cath-D is a promising immunotherapy to treat patients with TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBu63j-Oqd97P3FVUi_gf8fW6udTcc2eb2cyNWc-sdEbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjovVyksA%253D%253D&md5=ccc512eb50ec1cf0180fa8c588dbab6a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1186%2Fs40425-019-0498-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40425-019-0498-z%26sid%3Dliteratum%253Aachs%26aulast%3DAshraf%26aufirst%3DY.%26aulast%3DMansouri%26aufirst%3DH.%26aulast%3DLaurent-Matha%26aufirst%3DV.%26aulast%3DAlcaraz%26aufirst%3DL.%2BB.%26aulast%3DRoger%26aufirst%3DP.%26aulast%3DGuiu%26aufirst%3DS.%26aulast%3DDerocq%26aufirst%3DD.%26aulast%3DRobin%26aufirst%3DG.%26aulast%3DMichaud%26aufirst%3DH.-A.%26aulast%3DDelpech%26aufirst%3DH.%26aulast%3DJarlier%26aufirst%3DM.%26aulast%3DPugni%25C3%25A8re%26aufirst%3DM.%26aulast%3DRobert%26aufirst%3DB.%26aulast%3DPuel%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DLandomiel%26aufirst%3DF.%26aulast%3DBourquard%26aufirst%3DT.%26aulast%3DAchour%26aufirst%3DO.%26aulast%3DFruitier-Arnaudin%26aufirst%3DI.%26aulast%3DPichard%26aufirst%3DA.%26aulast%3DDeshayes%26aufirst%3DE.%26aulast%3DTurtoi%26aufirst%3DA.%26aulast%3DPoupon%26aufirst%3DA.%26aulast%3DSimony-Lafontaine%26aufirst%3DJ.%26aulast%3DBoissi%25C3%25A8re-Michot%26aufirst%3DF.%26aulast%3DPirot%26aufirst%3DN.%26aulast%3DBernex%26aufirst%3DF.%26aulast%3DJacot%26aufirst%3DW.%26aulast%3Ddu%2BManoir%26aufirst%3DS.%26aulast%3DTheillet%26aufirst%3DC.%26aulast%3DPouget%26aufirst%3DJ.-P.%26aulast%3DNavarro-Teulon%26aufirst%3DI.%26aulast%3DBonnefoy%26aufirst%3DN.%26aulast%3DP%25C3%25A8legrin%26aufirst%3DA.%26aulast%3DChard%25C3%25A8s%26aufirst%3DT.%26aulast%3DMartineau%26aufirst%3DP.%26aulast%3DLiaudet-Coopman%26aufirst%3DE.%26atitle%3DImmunotherapy%2520of%2520Triple-Negative%2520Breast%2520Cancer%2520with%2520Cathepsin%2520D-Targeting%2520Antibodies%26jtitle%3DJ.%2520Immunother.%2520Cancer%26date%3D2019%26volume%3D7%26spage%3D29%26doi%3D10.1186%2Fs40425-019-0498-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bach, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derocq, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prébois, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent-Matha, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattingre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaudet-Coopman, E.</span></span> <span> </span><span class="NLM_article-title">Pathophysiological Functions of Cathepsin D: Targeting Its Catalytic Activity versus Its Protein Binding Activity?</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1635</span>– <span class="NLM_lpage">1643</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2010.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.biochi.2010.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=20493920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2isLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2010&pages=1635-1643&author=O.+Massonauthor=A.-S.+Bachauthor=D.+Derocqauthor=C.+Pr%C3%A9boisauthor=V.+Laurent-Mathaauthor=S.+Pattingreauthor=E.+Liaudet-Coopman&title=Pathophysiological+Functions+of+Cathepsin+D%3A+Targeting+Its+Catalytic+Activity+versus+Its+Protein+Binding+Activity%3F&doi=10.1016%2Fj.biochi.2010.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pathophysiological functions of cathepsin D: Targeting its catalytic activity versus its protein binding activity?</span></div><div class="casAuthors">Masson, Olivier; Bach, Anne-Sophie; Derocq, Danielle; Prebois, Christine; Laurent-Matha, Valerie; Pattingre, Sophie; Liaudet-Coopman, Emmanuelle</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1635-1643</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The lysosomal aspartic protease cathepsin D (cath-D) is overexpressed and hyper-secreted by epithelial breast cancer cells.  This protease is an independent marker of poor prognosis in breast cancer as it is correlated with the incidence of clin. metastasis.  In normal cells, cath-D is localized in intracellular vesicles (lysosomes and endosomes).  In cancer cells, overexpressed cath-D accumulates in cells, where it may affect their degradative capacities, and the pro-enzyme is hyper-secreted in the tumor micro-environment.  In addn., during apoptosis, lysosomal cath-D is released into the cytosol, where it may interact with and/or cleave pro-apoptotic, anti-apoptotic, or nuclear proteins.  Several studies have shown that cath-D affects various different steps in tumor progression and metastasis.  Cath-D stimulates cancer cell growth in an autocrine manner, and also cath-D plays a crucial paracrine role in the tumor micro-environment by stimulating fibroblast outgrowth and tumor angiogenesis.  A mutant D231N-cath-D, which is devoid of catalytic activity, remained mitogenic, indicating an addnl. action of cath-D by protein-protein interaction.  Targeting cath-D in cancer may require the use of inhibitors of its catalytic activity, but also the development of new tools to inhibit its protein binding functions.  Thus, elucidation of the mechanism of action of cath-D is crucial if an appropriate strategy is to be developed to target this protease in cancer.  The discovery of new physiol. substrates of cath-D using proteomic approaches can be expected to generate new crit. targets.  The aim of this review is to describe the roles of the cath-D protease in cancer progression and metastasis, as well as its function in apoptosis, and to discuss how it can be targeted in cancer by inhibiting its proteolytic activity and/or its binding protein activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjTXmyIl5WaLVg90H21EOLACvtfcHk0lhexRh7dJRu3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2isLvF&md5=544dfe4c940fc1a18e0cd710962ca6c3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2010.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2010.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DMasson%26aufirst%3DO.%26aulast%3DBach%26aufirst%3DA.-S.%26aulast%3DDerocq%26aufirst%3DD.%26aulast%3DPr%25C3%25A9bois%26aufirst%3DC.%26aulast%3DLaurent-Matha%26aufirst%3DV.%26aulast%3DPattingre%26aufirst%3DS.%26aulast%3DLiaudet-Coopman%26aufirst%3DE.%26atitle%3DPathophysiological%2520Functions%2520of%2520Cathepsin%2520D%253A%2520Targeting%2520Its%2520Catalytic%2520Activity%2520versus%2520Its%2520Protein%2520Binding%2520Activity%253F%26jtitle%3DBiochimie%26date%3D2010%26volume%3D92%26spage%3D1635%26epage%3D1643%26doi%3D10.1016%2Fj.biochi.2010.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maibaum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahuel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, N.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasetti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rüger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göschke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stutz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schilling, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigollier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cumin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedigian, M. P.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Aliskiren, a Novel Orally Effective Renin Inhibitor</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>308</i></span>,  <span class="NLM_fpage">698</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1016/s0006-291x(03)01451-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fs0006-291x%2803%2901451-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=12927775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVGkt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=308&publication_year=2003&pages=698-705&author=J.+M.+Woodauthor=J.+Maibaumauthor=J.+Rahuelauthor=M.+G.+Gr%C3%BCtterauthor=N.-C.+Cohenauthor=V.+Rasettiauthor=H.+R%C3%BCgerauthor=R.+G%C3%B6schkeauthor=S.+Stutzauthor=W.+Fuhrerauthor=W.+Schillingauthor=P.+Rigollierauthor=Y.+Yamaguchiauthor=F.+Cuminauthor=H.-P.+Baumauthor=C.+R.+Schnellauthor=P.+Heroldauthor=R.+Mahauthor=C.+Jensenauthor=E.+O%E2%80%99Brienauthor=A.+Stantonauthor=M.+P.+Bedigian&title=Structure-Based+Design+of+Aliskiren%2C+a+Novel+Orally+Effective+Renin+Inhibitor&doi=10.1016%2Fs0006-291x%2803%2901451-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of aliskiren, a novel orally effective renin inhibitor</span></div><div class="casAuthors">Wood, Jeanette M.; Maibaum, Juergen; Rahuel, Joseph; Gruetter, Markus G.; Cohen, Nissim-Claude; Rasetti, Vittorio; Rueger, Heinrich; Goeschke, Richard; Stutz, Stefan; Fuhrer, Walter; Schilling, Walter; Rigollier, Pascal; Yamaguchi, Yasuchika; Cumin, Frederic; Baum, Hans-Peter; Schnell, Christian R.; Herold, Peter; Mah, Robert; Jensen, Chris; O'Brien, Eoin; Stanton, Alice; Bedigian, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">308</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">698-705</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">Hypertension is a major risk factor for cardiovascular diseases such as stroke, myocardial infarction, and heart failure, the leading causes of death in the Western world.  Inhibitors of the renin-angiotensin system (RAS) have proven to be successful treatments for hypertension.  As renin specifically catalyzes the rate-limiting step of the RAS, it represents the optimal target for RAS inhibition.  Several peptide-like renin inhibitors have been synthesized previously, but poor pharmacokinetic properties meant that these compds. were not clin. useful.  We employed a combination of mol. modeling and crystallog. structure anal. to design renin inhibitors lacking the extended peptide-like backbone of earlier inhibitors, for improved pharmacokinetic properties.  This led to the discovery of aliskiren, a highly potent and selective inhibitor of human renin in vitro, and in vivo; once-daily oral doses of aliskiren inhibit renin and lower blood pressure in sodium-depleted marmosets and hypertensive human patients.  Aliskiren represents the first in a novel class of renin inhibitors with the potential for treatment of hypertension and related cardiovascular diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV5YmV3cdr0LVg90H21EOLACvtfcHk0ljpH7aaocVfEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVGkt7k%253D&md5=20b71ca3ccfe2324752cb2bb541d9ba2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fs0006-291x%2803%2901451-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0006-291x%252803%252901451-7%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DJ.%2BM.%26aulast%3DMaibaum%26aufirst%3DJ.%26aulast%3DRahuel%26aufirst%3DJ.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DM.%2BG.%26aulast%3DCohen%26aufirst%3DN.-C.%26aulast%3DRasetti%26aufirst%3DV.%26aulast%3DR%25C3%25BCger%26aufirst%3DH.%26aulast%3DG%25C3%25B6schke%26aufirst%3DR.%26aulast%3DStutz%26aufirst%3DS.%26aulast%3DFuhrer%26aufirst%3DW.%26aulast%3DSchilling%26aufirst%3DW.%26aulast%3DRigollier%26aufirst%3DP.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DCumin%26aufirst%3DF.%26aulast%3DBaum%26aufirst%3DH.-P.%26aulast%3DSchnell%26aufirst%3DC.%2BR.%26aulast%3DHerold%26aufirst%3DP.%26aulast%3DMah%26aufirst%3DR.%26aulast%3DJensen%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DE.%26aulast%3DStanton%26aufirst%3DA.%26aulast%3DBedigian%26aufirst%3DM.%2BP.%26atitle%3DStructure-Based%2520Design%2520of%2520Aliskiren%252C%2520a%2520Novel%2520Orally%2520Effective%2520Renin%2520Inhibitor%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2003%26volume%3D308%26spage%3D698%26epage%3D705%26doi%3D10.1016%2Fs0006-291x%2803%2901451-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prato, G.</span></span> <span> </span><span class="NLM_article-title">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5172</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01697</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01697" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5172-5208&author=A.+K.+Ghoshauthor=H.+L.+Osswaldauthor=G.+Prato&title=Recent+Progress+in+the+Development+of+HIV-1+Protease+Inhibitors+for+the+Treatment+of+HIV%2FAIDS&doi=10.1021%2Facs.jmedchem.5b01697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span></div><div class="casAuthors">Ghosh, Arun K.; Osswald, Heather L.; Prato, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5172-5208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 protease inhibitors continue to play an important role in the treatment of HIV/AIDS, transforming this deadly ailment into a more manageable chronic infection.  Over the years, intensive research has led to a variety of approved protease inhibitors for the treatment of HIV/AIDS.  In this review, we outline current drug design and medicinal chem. efforts toward the development of next-generation protease inhibitors beyond the currently approved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4DisuSRbQXbVg90H21EOLACvtfcHk0ljpH7aaocVfEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D&md5=e0994f3d84ab627d3ac3f757e8d11725</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01697%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DPrato%26aufirst%3DG.%26atitle%3DRecent%2520Progress%2520in%2520the%2520Development%2520of%2520HIV-1%2520Protease%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520HIV%252FAIDS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5172%26epage%3D5208%26doi%3D10.1021%2Facs.jmedchem.5b01697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ersmark, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuelsson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallberg, A.</span></span> <span> </span><span class="NLM_article-title">Plasmepsins as Potential Targets for New Antimalarial Therapy</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">626</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.1002/med.20082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fmed.20082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=16838300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsFGru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=626-666&author=K.+Ersmarkauthor=B.+Samuelssonauthor=A.+Hallberg&title=Plasmepsins+as+Potential+Targets+for+New+Antimalarial+Therapy&doi=10.1002%2Fmed.20082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Plasmepsins as potential targets for new antimalarial therapy</span></div><div class="casAuthors">Ersmark, Karolina; Samuelsson, Bertil; Hallberg, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">626-666</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Malaria is one of the major diseases in the world.  Due to the rapid spread of parasite resistance to available antimalarial drugs there is an urgent need for new antimalarials with novel mechanisms of action.  Several promising targets for drug intervention have been revealed in recent years.  This review addresses the parasitic aspartic proteases termed plasmepsins (Plms) that are involved in the Hb catabolism that occurs during the erythrocytic stage of the malarial parasite life cycle.  Four Plasmodium species are responsible for human malaria; P. vivax, P. ovale, P. malariae, and P. falciparum.  This review focuses on inhibitors of the Hb-degrading plasmepsins of the most lethal species, P. falciparum; Plm I, Plm II, Plm IV, and histo-aspartic protease (HAP).  Previously, Plm II has attracted the most attention.  With the identification and characterization of new plasmepsins and the results from recent plasmepsin knockout studies, it now seems clear that in order to achieve high-antiparasitic activities in P. falciparum-infected erythrocytes it is necessary to inhibit several of the Hb-degrading plasmepsins.  Herein we summarize the structure-activity relationships of the Plm I, II, IV, and HAP inhibitors.  These inhibitors represent all classes which, to the best of our knowledge, have been disclosed in journal articles to date.  The 3D structures of inhibitor/plasmepsin II complexes available in the protein data bank are briefly discussed and compared.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwP4wAeM2fT7Vg90H21EOLACvtfcHk0ljpH7aaocVfEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsFGru7k%253D&md5=fcc65b50fccd246a31d1915c1d8274f9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fmed.20082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20082%26sid%3Dliteratum%253Aachs%26aulast%3DErsmark%26aufirst%3DK.%26aulast%3DSamuelsson%26aufirst%3DB.%26aulast%3DHallberg%26aufirst%3DA.%26atitle%3DPlasmepsins%2520as%2520Potential%2520Targets%2520for%2520New%2520Antimalarial%2520Therapy%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2006%26volume%3D26%26spage%3D626%26epage%3D666%26doi%3D10.1002%2Fmed.20082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coimbra, J. R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marques, D. F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baptista, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, C. M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreira, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinis, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvador, J. A. R.</span></span> <span> </span><span class="NLM_article-title">Highlights in BACE1 Inhibitors for Alzheimer’s Disease Treatment</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">178</span>, <span class="refDoi"> DOI: 10.3389/fchem.2018.00178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.3389%2Ffchem.2018.00178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=29881722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisV2mtr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=178&author=J.+R.+M.+Coimbraauthor=D.+F.+F.+Marquesauthor=S.+J.+Baptistaauthor=C.+M.+F.+Pereiraauthor=P.+I.+Moreiraauthor=T.+C.+P.+Dinisauthor=A.+E.+Santosauthor=J.+A.+R.+Salvador&title=Highlights+in+BACE1+Inhibitors+for+Alzheimer%E2%80%99s+Disease+Treatment&doi=10.3389%2Ffchem.2018.00178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Highlights in bace1 inhibitors for alzheimer's disease treatment</span></div><div class="casAuthors">Coimbra, Judite R. M.; Marques, Daniela F. F.; Baptista, Salete J.; Pereira, Claudia M. F.; Moreira, Paula I.; Dinis, Teresa C. P.; Santos, Armanda E.; Salvador, Jorge A. R.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">178/1-178/10</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Alzheimer's disease (AD) is a severe neurodegenerative disorder and the most common type of dementia in the elderly.  The clin. symptoms of AD include a progressive loss of memory and impairment of cognitive functions interfering with daily life activities.  The main neuropathol. features consist in extracellular amyloid-β (Aβ) plaque deposition and intracellular Neurofibrillary tangles (NFTs) of hyperphosphorylated Tau.  Understanding the pathophysiol. mechanisms that underlie neurodegeneration in AD is essential for rational design of neuroprotective agents able to prevent disease progression.  According to the "Amyloid Cascade Hypothesis" the crit. mol. event in the pathogenesis of AD is the accumulation of Aβ neurotoxic oligomers.  Since the proteolytic processing of Amyloid Precursor Protein (APP) by β-secretase (beta-site APP cleaving enzyme 1, BACE1) is the rate-limiting step in the prodn. of Aβ, this enzyme is considered a major therapeutic target and BACE1 inhibitors have the potential to be disease-modifying drugs for AD treatment.  Therefore, intensive efforts to discover and develop inhibitors that can reach the brain and effectively inhibit BACE1 have been pursued by several groups worldwide.  The aim of this review is to highlight the progress in the discovery of potent and selective small mol. BACE1 inhibitors over the past decade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyV_176RT6cLVg90H21EOLACvtfcHk0lgKFo0t8kzyWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisV2mtr3N&md5=79f20f1bde2dd9993b8b84ba1f188274</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2018.00178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2018.00178%26sid%3Dliteratum%253Aachs%26aulast%3DCoimbra%26aufirst%3DJ.%2BR.%2BM.%26aulast%3DMarques%26aufirst%3DD.%2BF.%2BF.%26aulast%3DBaptista%26aufirst%3DS.%2BJ.%26aulast%3DPereira%26aufirst%3DC.%2BM.%2BF.%26aulast%3DMoreira%26aufirst%3DP.%2BI.%26aulast%3DDinis%26aufirst%3DT.%2BC.%2BP.%26aulast%3DSantos%26aufirst%3DA.%2BE.%26aulast%3DSalvador%26aufirst%3DJ.%2BA.%2BR.%26atitle%3DHighlights%2520in%2520BACE1%2520Inhibitors%2520for%2520Alzheimer%25E2%2580%2599s%2520Disease%2520Treatment%26jtitle%3DFront.%2520Chem.%26date%3D2018%26volume%3D6%26spage%3D178%26doi%3D10.3389%2Ffchem.2018.00178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kick, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoffrey
Skillman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewing, T. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design and Combinatorial Chemistry Yield Low Nanomolar Inhibitors of Cathepsin D</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">307</span>, <span class="refDoi"> DOI: 10.1016/s1074-5521(97)90073-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fs1074-5521%2897%2990073-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=9195867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1ekt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1997&pages=297-307&author=E.+K.+Kickauthor=D.+C.+Roeauthor=A.+Geoffrey%0ASkillmanauthor=G.+Liuauthor=T.+J.+A.+Ewingauthor=Y.+Sunauthor=I.+D.+Kuntzauthor=J.+A.+Ellman&title=Structure-Based+Design+and+Combinatorial+Chemistry+Yield+Low+Nanomolar+Inhibitors+of+Cathepsin+D&doi=10.1016%2Fs1074-5521%2897%2990073-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design and combinatorial chemistry yield low nanomolar inhibitors of cathepsin D</span></div><div class="casAuthors">Kick, Ellen K.; Roe, Diana C.; Skillman, A. Geoffrey; Liu, Guangcheng; Ewing, Todd, J. A.; Sun, Yaxiong; Kuntz, Irwin D.; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">297-307</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">The identification of potent small mol. ligands to receptors and enzymes is one of the major goals of chem. and biol. research.  Two powerful new tools that can be used in these efforts are combinatorial chem. and structure-based design.  Here we address how to join these methods in a design protocol that produces libraries of compds. that are directed against specific macromol. targets.  The aspartyl class of proteases, which is involved in numerous biol. processes, was chosen to demonstrate this effective procedure.  Using cathepsin D, a prototypical aspartyl protease, a no. of low nanomolar inhibitors were rapidly identified.  Although cathepsin D is implicated in a no. of therapeutically relevant processes, potent nonpeptide inhibitors have not been reported previously.  The libraries, synthesized on solid support, displayed nonpeptide functionality about the (hydroxyethyl)amine isostere.  The (hydroxyethyl)amine isostere, which targets the aspartyl protease class, is a stable mimetic of the tetrahedral intermediate of amide hydrolysis.  Structure-based design, using the crystal structure of cathepsin D complexed with the peptide-based natural product pepstatin, was used to select the building blocks for the library synthesis.  The library yielded a 'hit rate' of 6-7% at 1 μM inhibitor concns., with the most potent compd. having a Ki value of 73 nM.  More potent, nonpeptide inhibitors (Ki = 9-15 nM) of cathepsin D were rapidly identified by synthesizing and screening a small second generation library.  The success of these studies clearly demonstrates the power of coupling the complementary methods of combinatorial chem. and structure-based design.  We anticipate that the general approaches described here will be successful for other members of the aspartyl protease class and for many other enzyme classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5II3prqAvlrVg90H21EOLACvtfcHk0lgKFo0t8kzyWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1ekt7s%253D&md5=0ca170cd545b8d34c1c509b2d0b21c95</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fs1074-5521%2897%2990073-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1074-5521%252897%252990073-9%26sid%3Dliteratum%253Aachs%26aulast%3DKick%26aufirst%3DE.%2BK.%26aulast%3DRoe%26aufirst%3DD.%2BC.%26aulast%3DGeoffrey%2BSkillman%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DEwing%26aufirst%3DT.%2BJ.%2BA.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DStructure-Based%2520Design%2520and%2520Combinatorial%2520Chemistry%2520Yield%2520Low%2520Nanomolar%2520Inhibitors%2520of%2520Cathepsin%2520D%26jtitle%3DChem.%2520Biol.%26date%3D1997%26volume%3D4%26spage%3D297%26epage%3D307%26doi%3D10.1016%2Fs1074-5521%2897%2990073-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxena, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luqman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanuja, S. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negi, A. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Gallic Acid Based Naphthophenone Fatty Acid Amides as Cathepsin D Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4603</span>– <span class="NLM_lpage">4608</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2006.06.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.bmcl.2006.06.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=16797987" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsFOltbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=4603-4608&author=V.+Srivastavaauthor=H.+O.+Saxenaauthor=K.+Shankerauthor=J.+K.+Kumarauthor=S.+Luqmanauthor=M.+M.+Guptaauthor=S.+P.+S.+Khanujaauthor=A.+S.+Negi&title=Synthesis+of+Gallic+Acid+Based+Naphthophenone+Fatty+Acid+Amides+as+Cathepsin+D+Inhibitors&doi=10.1016%2Fj.bmcl.2006.06.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of gallic acid based naphthophenone fatty acid amides as cathepsin D inhibitors</span></div><div class="casAuthors">Srivastava, Vandana; Saxena, Hari Om; Shanker, Karuna; Kumar, J. K.; Luqman, Suaib; Gupta, M. M.; Khanuja, S. P. S.; Negi, Arvind S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4603-4608</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Gallic acid, one of the most abundant plant phenolic acids, has been modified to cathepsin D protease inhibitors.  The strategy of modification was proposed basing on some previously reported structure activity relationship (SAR) studies.  The synthesized naphthophenone fatty acid amide derivs. have been evaluated for in vitro cathepsin D inhibition activity.  Two of them have shown significant inhibition activity with IC50 values of 0.06 (I; R = CH2CO2H) and 0.14 μM (I; R = CH2CH=CHCONHBu) as compared against pepstatin (0.0023 μM), the most potent inhibitor known so far.  The study revealed that such attempts on gallic acid based pharmacophores might result in potent inhibitors of cathepsin D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9cRPClG1Im7Vg90H21EOLACvtfcHk0lgKFo0t8kzyWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsFOltbs%253D&md5=7f20d0b8682f88b6b6bc95ee19f503e0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.06.010%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DV.%26aulast%3DSaxena%26aufirst%3DH.%2BO.%26aulast%3DShanker%26aufirst%3DK.%26aulast%3DKumar%26aufirst%3DJ.%2BK.%26aulast%3DLuqman%26aufirst%3DS.%26aulast%3DGupta%26aufirst%3DM.%2BM.%26aulast%3DKhanuja%26aufirst%3DS.%2BP.%2BS.%26aulast%3DNegi%26aufirst%3DA.%2BS.%26atitle%3DSynthesis%2520of%2520Gallic%2520Acid%2520Based%2520Naphthophenone%2520Fatty%2520Acid%2520Amides%2520as%2520Cathepsin%2520D%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D4603%26epage%3D4608%26doi%3D10.1016%2Fj.bmcl.2006.06.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grädler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsaklakidis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werkmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindemann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maskos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Optimization of Non-Peptidic Cathepsin D Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4141</span>– <span class="NLM_lpage">4150</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.07.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.bmcl.2014.07.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=25086681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADC%252BC2cbot12ktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=4141-4150&author=U.+Gr%C3%A4dlerauthor=P.+Czodrowskiauthor=C.+Tsaklakidisauthor=M.+Kleinauthor=D.+Werkmannauthor=S.+Lindemannauthor=K.+Maskosauthor=B.+Leuthner&title=Structure-Based+Optimization+of+Non-Peptidic+Cathepsin+D+Inhibitors&doi=10.1016%2Fj.bmcl.2014.07.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based optimization of non-peptidic Cathepsin D inhibitors</span></div><div class="casAuthors">Gradler Ulrich; Czodrowski Paul; Tsaklakidis Christos; Klein Markus; Werkmann Daniela; Lindemann Sven; Leuthner Birgitta; Maskos Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4141-50</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We discovered a novel series of non-peptidic acylguanidine inhibitors of Cathepsin D as target for osteoarthritis.  The initial HTS-hits were optimized by structure-based design using CatD X-ray structures resulting in single digit nanomolar potency in the biochemical CatD assay.  However, the most potent analogues showed only micromolar activities in an ex vivo glycosaminoglycan (GAG) release assay in bovine cartilage together with low cellular permeability and suboptimal microsomal stability.  This new scaffold can serve as a starting point for further optimization towards in vivo efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJw2Tu4OKgfXT4ugvCDgnZfW6udTcc2ea2sGxr8oIYGrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbot12ktw%253D%253D&md5=df3353a5f78787ec5a540fa92194a769</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.07.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.07.054%26sid%3Dliteratum%253Aachs%26aulast%3DGr%25C3%25A4dler%26aufirst%3DU.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DTsaklakidis%26aufirst%3DC.%26aulast%3DKlein%26aufirst%3DM.%26aulast%3DWerkmann%26aufirst%3DD.%26aulast%3DLindemann%26aufirst%3DS.%26aulast%3DMaskos%26aufirst%3DK.%26aulast%3DLeuthner%26aufirst%3DB.%26atitle%3DStructure-Based%2520Optimization%2520of%2520Non-Peptidic%2520Cathepsin%2520D%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D4141%26epage%3D4150%26doi%3D10.1016%2Fj.bmcl.2014.07.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kick, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">General Solid-Phase Synthesis Approach to Prepare Mechanism-Based Aspartyl Protease Inhibitor Libraries. Identification of Potent Cathepsin D Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">9735</span>– <span class="NLM_lpage">9747</span>, <span class="refDoi"> DOI: 10.1021/ja981812m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja981812m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK1cXlslaktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=9735-9747&author=C.+E.+Leeauthor=E.+K.+Kickauthor=J.+A.+Ellman&title=General+Solid-Phase+Synthesis+Approach+to+Prepare+Mechanism-Based+Aspartyl+Protease+Inhibitor+Libraries.+Identification+of+Potent+Cathepsin+D+Inhibitors&doi=10.1021%2Fja981812m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">General solid-phase synthesis approach to prepare mechanism-based aspartyl protease inhibitor libraries. Identification of potent cathepsin D inhibitors</span></div><div class="casAuthors">Lee, Christina E.; Kick, Ellen K.; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">9735-9747</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A general method has been developed for the expedient solid-phase synthesis of aspartyl protease inhibitors that do not incorporate any amino acids, e.g. amides I (R = 3-PhOC6H4, 2-naphthalenyl, 4-PhC6H4, 4-BrC6H4).  The synthesis approach was designed to allow the introduction of diverse functionality at all four variable sites using readily available precursors.  Representative, anal. pure inhibitors were obtained in 45-64% overall yields for the 12-step solid-phase synthesis sequence that includes both stereoselective redn. and carbon-carbon bond forming steps.  A library of 204 spatially sep. compds. was also prepd. and screened against cathepsin D and resulted in the identification of a no. of potent small mol. inhibitors.  The described synthesis approach enables the rapid identification of small mol. inhibitors to diverse aspartyl protease targets.  The general design principles are also applicable to other enzyme classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2pHlBMx4jkrVg90H21EOLACvtfcHk0ljbHrIw2lpbZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlslaktrw%253D&md5=92b5e1523a037d54b77d7e02f0080eda</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja981812m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja981812m%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BE.%26aulast%3DKick%26aufirst%3DE.%2BK.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DGeneral%2520Solid-Phase%2520Synthesis%2520Approach%2520to%2520Prepare%2520Mechanism-Based%2520Aspartyl%2520Protease%2520Inhibitor%2520Libraries.%2520Identification%2520of%2520Potent%2520Cathepsin%2520D%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D9735%26epage%3D9747%26doi%3D10.1021%2Fja981812m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, A. J.</span></span> <span> </span><span class="NLM_article-title">Interaction of Human Cathepsin D with the Inhibitor Pepstatin</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1042/bj1550117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1042%2Fbj1550117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=938470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaE28XksVakurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=1976&pages=117-125&author=C.+G.+Knightauthor=A.+J.+Barrett&title=Interaction+of+Human+Cathepsin+D+with+the+Inhibitor+Pepstatin&doi=10.1042%2Fbj1550117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of human cathepsin D with the inhibitor pepstatin</span></div><div class="casAuthors">Knight, C. Graham; Barrett, Alan J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">117-25</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    </div><div class="casAbstract">The apparent dissocn. const. (KD) for the binding of pepstatin, pepstatin Me ester, and pepstatinyl-glycine to cathepsin D (EC 3.4.23.5) at pH 3.5 was ∼5 × 10-10M.  The KD of pepstatinyl-glycine rose from 5 × 10-10 to 2 × 10-6M on raising the pH from 5.0 to 6.4.  The catalytic activity of cathepsin D fell to ∼0 on going from pH 5.0 to 7.0.  The binding site for substrate and pepstatin may be abolished by a conformational change in cathepsin D.  Cathepsin D tightly bound 1 mole pepstatin per 43,000 g protein, indicating that abs. cocns. of the enzyme can be detd. by titration with pepstatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwv8-RPbtj0rVg90H21EOLACvtfcHk0ljbHrIw2lpbZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XksVakurk%253D&md5=9e2504b37e08e7569a0d2af2f95e85c0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1042%2Fbj1550117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj1550117%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DC.%2BG.%26aulast%3DBarrett%26aufirst%3DA.%2BJ.%26atitle%3DInteraction%2520of%2520Human%2520Cathepsin%2520D%2520with%2520the%2520Inhibitor%2520Pepstatin%26jtitle%3DBiochem.%2520J.%26date%3D1976%26volume%3D155%26spage%3D117%26epage%3D125%26doi%3D10.1042%2Fbj1550117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulnik, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, J. W.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Subsite Specificity Mapping of Human Cathepsin D Using Statine-Based Inhibitors</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1458</span>– <span class="NLM_lpage">1466</span>, <span class="refDoi"> DOI: 10.1002/pro.5560060710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fpro.5560060710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=9232647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK2sXksF2rtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1997&pages=1458-1466&author=P.+Majerauthor=J.+R.+Collinsauthor=S.+V.+Gulnikauthor=J.+W.+Erickson&title=Structure-Based+Subsite+Specificity+Mapping+of+Human+Cathepsin+D+Using+Statine-Based+Inhibitors&doi=10.1002%2Fpro.5560060710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based subsite specificity mapping of human cathepsin D using statine-based inhibitors</span></div><div class="casAuthors">Majer, Pavel; Collins, Jack R.; Gulnik, Sergei V.; Erickson, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1458-1466</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Human cathepsin D is a lysosomal aspartic protease that has been implicated in breast cancer metastasis and Alzheimer's disease.  Based on a crystal structure of a human cathepsin D-pepstatin A complex, a series of statine-contg. inhibitors was designed, synthesized, and tested for inhibitory activity toward the enzyme in vitro.  The compds. were modified systematically at individual positions (P4, P3, P2, P1, and P'2) with the aim of mapping the cathepsin D subsite preferences.  The exptl. obtained SAR data were correlated on the basis of mol. modeling.  Side-chain preferences for the peptidomimetic inhibitors differed from those found previously using peptide substrates (Scarborough PE et al., 1993, Protein Sci 2:264-276).  In addn., the effects of single side-chain modifications were often nonadditive.  Structure-activity relationships, modeling, and thermodn. anal. indicated that entropy plays a major stabilizing role in inhibitor binding to cathepsin D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgKoFaE1fzZ7Vg90H21EOLACvtfcHk0ljbHrIw2lpbZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksF2rtr8%253D&md5=56f9c56fa0bb620fc366abc27aa165f5</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560060710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560060710%26sid%3Dliteratum%253Aachs%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DJ.%2BR.%26aulast%3DGulnik%26aufirst%3DS.%2BV.%26aulast%3DErickson%26aufirst%3DJ.%2BW.%26atitle%3DStructure-Based%2520Subsite%2520Specificity%2520Mapping%2520of%2520Human%2520Cathepsin%2520D%2520Using%2520Statine-Based%2520Inhibitors%26jtitle%3DProtein%2520Sci.%26date%3D1997%26volume%3D6%26spage%3D1458%26epage%3D1466%26doi%3D10.1002%2Fpro.5560060710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumerová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Máša, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hradilek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šanda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiniš, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Single- and Double-Headed Chemical Probes for Detection of Active Cathepsin D in a Cancer Cell Proteome</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1538</span>– <span class="NLM_lpage">1541</span>, <span class="refDoi"> DOI: 10.1002/cbic.201000182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fcbic.201000182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=20564284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpt1Clu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=1538-1541&author=M.+Nussbaumerov%C3%A1author=J.+Srpauthor=M.+M%C3%A1%C5%A1aauthor=M.+Hradilekauthor=M.+%C5%A0andaauthor=M.+Reini%C5%A1author=M.+Hornauthor=M.+Mare%C5%A1&title=Single-+and+Double-Headed+Chemical+Probes+for+Detection+of+Active+Cathepsin+D+in+a+Cancer+Cell+Proteome&doi=10.1002%2Fcbic.201000182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Single- and double-headed chemical probes for detection of active cathepsin D in a cancer cell proteome</span></div><div class="casAuthors">Nussbaumerova, Martina; Srp, Jaroslav; Masa, Martin; Hradilek, Martin; Sanda, Miloslav; Reinis, Milan; Horn, Martin; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1538-1541</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The design of specific active site-directed probes for cathepsins D and related aspartic proteases, which enable the effective labeling of these proteases in a complex proteome, is presented.  The applicability of this system to activity-based proteomics was demonstrated for cancer cell-derived cathepsin D, a therapeutic target in several human tumors.  The modular assembly rendered the probe synthesis convenient and widely available.  Due to their versatile fluorescent and biotin tags, the probes can be used as a powerful tool for a broad range of "in-gel" and "on-blot" proteomic applications in profiling medically important cathepsin D-type proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0lXD5RkZKY7Vg90H21EOLACvtfcHk0ljupqzllKC6Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpt1Clu7s%253D&md5=9c656dd3fd94005e9e9ead95fb4fa32f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201000182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201000182%26sid%3Dliteratum%253Aachs%26aulast%3DNussbaumerov%25C3%25A1%26aufirst%3DM.%26aulast%3DSrp%26aufirst%3DJ.%26aulast%3DM%25C3%25A1%25C5%25A1a%26aufirst%3DM.%26aulast%3DHradilek%26aufirst%3DM.%26aulast%3D%25C5%25A0anda%26aufirst%3DM.%26aulast%3DReini%25C5%25A1%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DSingle-%2520and%2520Double-Headed%2520Chemical%2520Probes%2520for%2520Detection%2520of%2520Active%2520Cathepsin%2520D%2520in%2520a%2520Cancer%2520Cell%2520Proteome%26jtitle%3DChemBioChem%26date%3D2010%26volume%3D11%26spage%3D1538%26epage%3D1541%26doi%3D10.1002%2Fcbic.201000182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumerová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šanda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovářová, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srba, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franta, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sojka, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopáček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Hemoglobin Digestion in Blood-Feeding Ticks: Mapping a Multipeptidase Pathway by Functional Proteomics</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1053</span>– <span class="NLM_lpage">1063</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2009.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.chembiol.2009.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=19875079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlCnt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=1053-1063&author=M.+Hornauthor=M.+Nussbaumerov%C3%A1author=M.+%C5%A0andaauthor=Z.+Kov%C3%A1%C5%99ov%C3%A1author=J.+Srbaauthor=Z.+Frantaauthor=D.+Sojkaauthor=M.+Bogyoauthor=C.+R.+Caffreyauthor=P.+Kop%C3%A1%C4%8Dekauthor=M.+Mare%C5%A1&title=Hemoglobin+Digestion+in+Blood-Feeding+Ticks%3A+Mapping+a+Multipeptidase+Pathway+by+Functional+Proteomics&doi=10.1016%2Fj.chembiol.2009.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Hemoglobin Digestion in Blood-Feeding Ticks: Mapping a Multipeptidase Pathway by Functional Proteomics</span></div><div class="casAuthors">Horn, Martin; Nussbaumerova, Martina; Sanda, Miloslav; Kovarova, Zuzana; Srba, Jindrich; Franta, Zdenek; Sojka, Daniel; Bogyo, Matthew; Caffrey, Conor R.; Kopacek, Petr; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1053-1063</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Hb digestion is an essential process for blood-feeding parasites.  Using chem. tools, we deconvoluted the intracellular hemoglobinolytic cascade in the tick Ixodes ricinus, a vector of Lyme disease and tick-borne encephalitis.  In tick gut tissue, a network of peptidases was demonstrated through imaging with specific activity-based probes and activity profiling with peptidic substrates and inhibitors.  This peptidase network is induced upon blood feeding and degrades Hb at acidic pH.  Selective inhibitors were applied to dissect the roles of the individual peptidases and to det. the peptidase-specific cleavage map of the Hb mol.  The degrdn. pathway is initiated by endopeptidases of aspartic and cysteine class (cathepsin D supported by cathepsin L and legumain) and is continued by cysteine amino- and carboxy-dipeptidases (cathepsins C and B).  The identified enzymes are potential targets to developing novel anti-tick vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTsuPLZ5dd7rVg90H21EOLACvtfcHk0ljupqzllKC6Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlCnt7bF&md5=be0bcaea4706c1f819fc65522afd28f9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2009.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2009.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DNussbaumerov%25C3%25A1%26aufirst%3DM.%26aulast%3D%25C5%25A0anda%26aufirst%3DM.%26aulast%3DKov%25C3%25A1%25C5%2599ov%25C3%25A1%26aufirst%3DZ.%26aulast%3DSrba%26aufirst%3DJ.%26aulast%3DFranta%26aufirst%3DZ.%26aulast%3DSojka%26aufirst%3DD.%26aulast%3DBogyo%26aufirst%3DM.%26aulast%3DCaffrey%26aufirst%3DC.%2BR.%26aulast%3DKop%25C3%25A1%25C4%258Dek%26aufirst%3DP.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DHemoglobin%2520Digestion%2520in%2520Blood-Feeding%2520Ticks%253A%2520Mapping%2520a%2520Multipeptidase%2520Pathway%2520by%2520Functional%2520Proteomics%26jtitle%3DChem.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D1053%26epage%3D1063%26doi%3D10.1016%2Fj.chembiol.2009.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tumminello, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernacki, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebbia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leto, G.</span></span> <span> </span><span class="NLM_article-title">Pepstatins: Aspartic Proteinase Inhibitors Having Potential Therapeutic Applications</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1002/med.2610130206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fmed.2610130206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=8445958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3sXktF2ltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1993&pages=199-208&author=F.+M.+Tumminelloauthor=R.+J.+Bernackiauthor=N.+Gebbiaauthor=G.+Leto&title=Pepstatins%3A+Aspartic+Proteinase+Inhibitors+Having+Potential+Therapeutic+Applications&doi=10.1002%2Fmed.2610130206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Pepstatins: aspartic proteinase inhibitors having potential therapeutic applications</span></div><div class="casAuthors">Tumminello, F. M.; Bernacki, R. J.; Gebbia, N.; Leto, G.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">199-208</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    </div><div class="casAbstract">A review with 57 refs. of the potential therapeutic applications of pepstatins in diseases such as muscular dystrophy, myocardial ischemia, cancer and microbial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosZlqcTW8lzLVg90H21EOLACvtfcHk0ljupqzllKC6Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXktF2ltrk%253D&md5=93fc6a7c612cf852ffa6f7401a251f69</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fmed.2610130206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.2610130206%26sid%3Dliteratum%253Aachs%26aulast%3DTumminello%26aufirst%3DF.%2BM.%26aulast%3DBernacki%26aufirst%3DR.%2BJ.%26aulast%3DGebbia%26aufirst%3DN.%26aulast%3DLeto%26aufirst%3DG.%26atitle%3DPepstatins%253A%2520Aspartic%2520Proteinase%2520Inhibitors%2520Having%2520Potential%2520Therapeutic%2520Applications%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1993%26volume%3D13%26spage%3D199%26epage%3D208%26doi%3D10.1002%2Fmed.2610130206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, D. H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Cathepsin D by Substrate Analogues Containing Statine and by Analogues of Pepstatin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2519</span>– <span class="NLM_lpage">2524</span>, <span class="refDoi"> DOI: 10.1021/jm00162a015</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00162a015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaL28XmtV2gtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=2519-2524&author=N.+S.+Agarwalauthor=D.+H.+Rich&title=Inhibition+of+Cathepsin+D+by+Substrate+Analogues+Containing+Statine+and+by+Analogues+of+Pepstatin&doi=10.1021%2Fjm00162a015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of cathepsin D by substrate analogs containing statine and by analogs of pepstatin</span></div><div class="casAuthors">Agarwal, Nirankar S.; Rich, Daniel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2519-24</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Statine-contg. peptides Boc-Phe-X-Ala-Sta-X1-Val-Leu-OMe [I; Boc = Me3CO2C; Sta = (3S,4S)-4-amino-3-hydroxy-6-methylheptanoic acid (statine) residue; X = Ala, Leu, X1 = Phe, null, X = X1 = Phe] were prepd. by conventional soln. methods and they were tested as inhibitors of bovine cathepsin D.  I were derived by replacing a Phe-Phe unit of cathepsin D substrate Boc-Phe-Leu-Ala-Phe-Phe-Val-Leu-OR with Sta or Sta-Phe.  I (X = Leu, X1 = null) was the best inhibitor (ki = 1.1 mM).  Thirty-five known pepstatin analogs were also evaluated as cathepsin D inhibitors.  Substituents in the P4' and P3' positions are important for max. inhibition and the P4 substituent is more important for inhibition of cathepsin D than for other aspartic proteinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLvR9izPjhLrVg90H21EOLACvtfcHk0linbfY-Xx1BBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XmtV2gtb0%253D&md5=bf8d95f8d1f2dd401fc2bf0f0f2607c8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm00162a015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00162a015%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwal%26aufirst%3DN.%2BS.%26aulast%3DRich%26aufirst%3DD.%2BH.%26atitle%3DInhibition%2520of%2520Cathepsin%2520D%2520by%2520Substrate%2520Analogues%2520Containing%2520Statine%2520and%2520by%2520Analogues%2520of%2520Pepstatin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1986%26volume%3D29%26spage%3D2519%26epage%3D2524%26doi%3D10.1021%2Fjm00162a015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulnik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustchina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suvorov, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, J. W.</span></span> <span> </span><span class="NLM_article-title">Dissection of the pH Dependence of Inhibitor Binding Energetics for an Aspartic Protease: Direct Measurement of the Protonation States of the Catalytic Aspartic Acid Residues</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">16166</span>– <span class="NLM_lpage">16172</span>, <span class="refDoi"> DOI: 10.1021/bi971550l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi971550l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK1cXjs1Ggtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1997&pages=16166-16172&author=D.+Xieauthor=S.+Gulnikauthor=L.+Collinsauthor=E.+Gustchinaauthor=L.+Suvorovauthor=J.+W.+Erickson&title=Dissection+of+the+pH+Dependence+of+Inhibitor+Binding+Energetics+for+an+Aspartic+Protease%3A+Direct+Measurement+of+the+Protonation+States+of+the+Catalytic+Aspartic+Acid+Residues&doi=10.1021%2Fbi971550l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Dissection of the pH Dependence of Inhibitor Binding Energetics for an Aspartic Protease: Direct Measurement of the Protonation States of the Catalytic Aspartic Acid Residues</span></div><div class="casAuthors">Xie, Dong; Gulnik, Sergei; Collins, Laura; Gustchina, Elena; Suvorov, Leonid; Erickson, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">16166-16172</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic activity and inhibitor binding energetics of enzymes are often pH-dependent properties.  Aspartic proteases comprise an important class of enzyme targets for structure-based drug design.  The authors have performed a complete thermodn. study of pepstatin binding to plasmepsin II, an aspartic proteinase found in Plasmodium falciparum, using isothermal titrn. calorimetry and CD.  Thermodn. parameters (ΔG, ΔH, ΔCp, and ΔS) were measured as functions of both pH and temp.  In the pH range from 4.5 to 7.0, pepstatin binding is accompanied by proton transfer between the solvent and the complex.  The authors used thermodn. proton linkage theory to derive both the pH-independent binding energetics for pepstatin and the no. and pKa values of ionizable residues whose pKa values change during ligand binding.  These residues were identified as the two catalytic aspartates, with pKas of 6.5 and 3.0, and His 164, with a pKa of 7.5, based on the three-dimensional structure of the pepstatin-plasmepsin II complex.  At pH 5.0, where the protease has optimum activity, the proton transfer process contributes almost 40% of the total binding free energy change and the total charge of the active-site aspartic acid residues is -1.  These exptl. results provide direct measurement for the protonation states of the catalytic aspartates in the presence of bound ligands.  Comparison of the thermodn. and structural data for pepstatin binding with human cathepsin D, a lysosomal aspartic protease that shares 35% sequence identity with plasmepsin II, suggests that the energetic differences between these two proteins are due to a higher interdomain flexibility in plasmepsin II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC2btgtkp6IrVg90H21EOLACvtfcHk0linbfY-Xx1BBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjs1Ggtw%253D%253D&md5=a7ddbf32e432004b471f818717da0ff3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fbi971550l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi971550l%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DD.%26aulast%3DGulnik%26aufirst%3DS.%26aulast%3DCollins%26aufirst%3DL.%26aulast%3DGustchina%26aufirst%3DE.%26aulast%3DSuvorov%26aufirst%3DL.%26aulast%3DErickson%26aufirst%3DJ.%2BW.%26atitle%3DDissection%2520of%2520the%2520pH%2520Dependence%2520of%2520Inhibitor%2520Binding%2520Energetics%2520for%2520an%2520Aspartic%2520Protease%253A%2520Direct%2520Measurement%2520of%2520the%2520Protonation%2520States%2520of%2520the%2520Catalytic%2520Aspartic%2520Acid%2520Residues%26jtitle%3DBiochemistry%26date%3D1997%26volume%3D36%26spage%3D16166%26epage%3D16172%26doi%3D10.1021%2Fbi971550l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6853</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6843-6853&author=M.+D.+Cummingsauthor=S.+Sekharan&title=Structure-Based+Macrocycle+Design+in+Small-Molecule+Drug+Discovery+and+Simple+Metrics+To+Identify+Opportunities+for+Macrocyclization+of+Small-Molecule+Ligands&doi=10.1021%2Facs.jmedchem.8b01985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span></div><div class="casAuthors">Cummings, Maxwell D.; Sekharan, Sivakumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6843-6853</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Interest is growing in the use of macrocycles in pharmaceutical discovery.  Macrocylization may provide a gateway to an expanded chem. space for small-mol. drug discovery, and this could be beneficial in prosecuting difficult targets, e.g., protein-protein interactions.  Most, but not all, macrocycle drugs are derived from natural products.  Studies on synthetic drug-like small-mol. macrocycles are limited, and our current understanding of macrocycle drugs is similarly limited.  Following some background discussion, we review several examples of the structure-based design of synthetic macrocycles.  Our opinion is that in conformationally suitable systems macrocycles are an analog class worthy of consideration.  We then summarize an approach for the initial evaluation of mols. as candidates for macrocyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCAQg11Q6Hk7Vg90H21EOLACvtfcHk0linbfY-Xx1BBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D&md5=32ac78d837075171d19c4cea694dce3f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DSekharan%26aufirst%3DS.%26atitle%3DStructure-Based%2520Macrocycle%2520Design%2520in%2520Small-Molecule%2520Drug%2520Discovery%2520and%2520Simple%2520Metrics%2520To%2520Identify%2520Opportunities%2520for%2520Macrocyclization%2520of%2520Small-Molecule%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6843%26epage%3D6853%26doi%3D10.1021%2Facs.jmedchem.8b01985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jouin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nisato, D.</span></span> <span> </span><span class="NLM_article-title">Stereospecific Synthesis of N-Protected Statine and Its Analogues via Chiral Tetramic Acid</span>. <i>J. Chem. Soc., Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_fpage">1177</span>– <span class="NLM_lpage">1182</span>, <span class="refDoi"> DOI: 10.1039/p19870001177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1039%2Fp19870001177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaL1cXnsFOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1987&pages=1177-1182&author=P.+Jouinauthor=B.+Castroauthor=D.+Nisato&title=Stereospecific+Synthesis+of+N-Protected+Statine+and+Its+Analogues+via+Chiral+Tetramic+Acid&doi=10.1039%2Fp19870001177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Stereospecific synthesis of N-protected statine and its analogs via chiral tetramic acid</span></div><div class="casAuthors">Jouin, Patrick; Castro, Bertrand; Nisato, Dino</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 1:  Organic and Bio-Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1177-82</span>CODEN:
                <span class="NLM_cas:coden">JCPRB4</span>;
        ISSN:<span class="NLM_cas:issn">0300-922X</span>.
    </div><div class="casAbstract">The condensation of chiral N-protected amino acids with Meldrum's acid in the presence of isopropenyl chloroformate and 4-DMAP has been examd.  The cyclization of the reaction product, by heating in an org. solvent, gave the N-protected tetramic acid derivs., which were reduced to afforded the corresponding threo-4-hydroxypyrrolidin-2-ones I [R = amino acid side chain; R1 = Me3CO2C (BOC), PhCH2O2C].  The regioselective alk. or acid hydrolysis of I led to enantiomeric pure N-Boc-statine (II; R = Me2CHCH2, R1 = Boc) and its analogs in 40-60% yield from the N-protected amino acids.  The corresponding statine Me esters could also be synthesized in high yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZRntmUCA4TrVg90H21EOLACvtfcHk0libY6ZJ78zPGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXnsFOqtA%253D%253D&md5=36555e17c0b318f2415b97ff7c67411b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2Fp19870001177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp19870001177%26sid%3Dliteratum%253Aachs%26aulast%3DJouin%26aufirst%3DP.%26aulast%3DCastro%26aufirst%3DB.%26aulast%3DNisato%26aufirst%3DD.%26atitle%3DStereospecific%2520Synthesis%2520of%2520N-Protected%2520Statine%2520and%2520Its%2520Analogues%2520via%2520Chiral%2520Tetramic%2520Acid%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25201%26date%3D1987%26spage%3D1177%26epage%3D1182%26doi%3D10.1039%2Fp19870001177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smrcina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majerová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerassina, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eissenstat, M. A.</span></span> <span> </span><span class="NLM_article-title">Facile Stereoselective Synthesis of γ-Substituted γ-Amino Acids from the Corresponding α-Amino Acids</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">12867</span>– <span class="NLM_lpage">12874</span>, <span class="refDoi"> DOI: 10.1016/s0040-4020(97)00840-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fs0040-4020%2897%2900840-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK2sXmtlGhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=12867-12874&author=M.+Smrcinaauthor=P.+Majerauthor=E.+Majerov%C3%A1author=T.+A.+Guerassinaauthor=M.+A.+Eissenstat&title=Facile+Stereoselective+Synthesis+of+%CE%B3-Substituted+%CE%B3-Amino+Acids+from+the+Corresponding+%CE%B1-Amino+Acids&doi=10.1016%2Fs0040-4020%2897%2900840-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Facile stereoselective synthesis of γ-substituted γ-amino acids from the corresponding α-amino acids</span></div><div class="casAuthors">Smrcina, Martin; Majer, Pavel; Majerova, Eva; Guerassina, Tatiana A.; Eissenstat, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">12867-12874</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A facile stereoselective method for the synthesis of γ-substituted, γ-amino acids I (R = CH2Ph, i-Pr, i-Bu, CH2OCH2Ph, CH2SCH2Ph, CH2CO2CH2Ph; Boc = Me3CO2C) from (L)-BocNHCHRCO2H and Meldrum's acid was developed.  The key step of the procedure is complete redn. of the keto functionality of α-amino acyl Meldrum's acid II by sodium acetoxyborohydride.  The resulting amino alkyl Meldrum's acid III undergoes thermal decarboxylative ring closure to a 5-substituted pyrrolidinone IV which yields the corresponding γ-amino acid I after basic hydrolysis.  The overall yield of the procedure ranges from 40 to 65%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3j_gvg6MY37Vg90H21EOLACvtfcHk0libY6ZJ78zPGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmtlGhu7s%253D&md5=d5263748e902494efd11fa2bfdf54d43</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fs0040-4020%2897%2900840-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0040-4020%252897%252900840-5%26sid%3Dliteratum%253Aachs%26aulast%3DSmrcina%26aufirst%3DM.%26aulast%3DMajer%26aufirst%3DP.%26aulast%3DMajerov%25C3%25A1%26aufirst%3DE.%26aulast%3DGuerassina%26aufirst%3DT.%2BA.%26aulast%3DEissenstat%26aufirst%3DM.%2BA.%26atitle%3DFacile%2520Stereoselective%2520Synthesis%2520of%2520%25CE%25B3-Substituted%2520%25CE%25B3-Amino%2520Acids%2520from%2520the%2520Corresponding%2520%25CE%25B1-Amino%2520Acids%26jtitle%3DTetrahedron%26date%3D1997%26volume%3D53%26spage%3D12867%26epage%3D12874%26doi%3D10.1016%2Fs0040-4020%2897%2900840-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishikawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inagaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatanaka, M.</span></span> <span> </span><span class="NLM_article-title">Radical Photocyclization Route for Macrocyclic Lactone Ring Expansion and Conversion to Macrocyclic Lactams and Ketones</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">582</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1021/jo3024126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo3024126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVKhurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=582-589&author=K.+Nishikawaauthor=Y.+Yoshimiauthor=K.+Maedaauthor=T.+Moritaauthor=I.+Takahashiauthor=T.+Itouauthor=S.+Inagakiauthor=M.+Hatanaka&title=Radical+Photocyclization+Route+for+Macrocyclic+Lactone+Ring+Expansion+and+Conversion+to+Macrocyclic+Lactams+and+Ketones&doi=10.1021%2Fjo3024126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Radical Photocyclization Route for Macrocyclic Lactone Ring Expansion and Conversion to Macrocyclic Lactams and Ketones</span></div><div class="casAuthors">Nishikawa, Keisuke; Yoshimi, Yasuharu; Maeda, Kousuke; Morita, Toshio; Takahashi, Ichiro; Itou, Tatsuya; Inagaki, Sho; Hatanaka, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">582-589</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrocyclic lactones, lactams, and ketones such as I (X = O, NH, CH2) were prepd. by a sequence using the photochem. decarboxylation and macrocyclization of acryloyloxy-, acryloylamino-, and oxopropenyl-substituted carboxylic acids such as II as the key step.  Hydrolysis of macrocyclic lactones followed by methoxymethylation of the carboxylate yielded methoxymethyl hydroxyalkanoates; acylation of the alc. with acryloyl chloride followed by ester hydrolysis yielded acryloyl carboxylic acids such as II (X = O).  Alternatively, hydrolysis of the lactone and O-methylation of the carboxylate, mesylation of the free alc., substitution with azide, Staudinger redn., acylation, and ester hydrolysis yielded acryloylamino carboxylic acids such as II (X = NH); oxidn. of the alc., addn. of vinylmagnesium bromide, oxidn., and ester hydrolysis yielded terminal oxopropenyl carboxylic acids such as II (X = CH2).  Photocyclization of the carboxylic acids in the presence of phenanthrene and 1,4-dicyanobenzene yielded macrocyclic lactones, lactams, or ketones; the method thus constitutes a two-carbon homologation of macrocyclic lactones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo225gq_nIUaLVg90H21EOLACvtfcHk0libY6ZJ78zPGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVKhurvJ&md5=c478a67ac475e441812cb57125113bc7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo3024126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo3024126%26sid%3Dliteratum%253Aachs%26aulast%3DNishikawa%26aufirst%3DK.%26aulast%3DYoshimi%26aufirst%3DY.%26aulast%3DMaeda%26aufirst%3DK.%26aulast%3DMorita%26aufirst%3DT.%26aulast%3DTakahashi%26aufirst%3DI.%26aulast%3DItou%26aufirst%3DT.%26aulast%3DInagaki%26aufirst%3DS.%26aulast%3DHatanaka%26aufirst%3DM.%26atitle%3DRadical%2520Photocyclization%2520Route%2520for%2520Macrocyclic%2520Lactone%2520Ring%2520Expansion%2520and%2520Conversion%2520to%2520Macrocyclic%2520Lactams%2520and%2520Ketones%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2013%26volume%3D78%26spage%3D582%26epage%3D589%26doi%3D10.1021%2Fjo3024126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rich, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, E. T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulm, E.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Analogues of the Carboxyl Protease Inhibitor Pepstatin. Effects of Structure on Inhibition of Pepsin and Renin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1021/jm00175a006</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00175a006" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaL3cXlt1CisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1980&pages=27-33&author=D.+H.+Richauthor=E.+T.+O.+Sunauthor=E.+Ulm&title=Synthesis+of+Analogues+of+the+Carboxyl+Protease+Inhibitor+Pepstatin.+Effects+of+Structure+on+Inhibition+of+Pepsin+and+Renin&doi=10.1021%2Fjm00175a006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of analogs of the carboxyl protease inhibitor pepstatin.  Effect of structure on inhibition of pepsin and renin</span></div><div class="casAuthors">Rich, Daniel H.; Sun, Eric T. O.; Ulm, Edgar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-33</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Analogs of carboxyl protease inhibitor, pepstatin  [26305-03-3], were synthesized from optically pure forms of N-(tert-butoxycarbonyl)-4(S)-amino-3(S)-hydroxy-6-methylheptanoic acid (Boc-Sta)  [49642-07-1], and the inhibition of pepsin  [9001-75-6] and renin  [9015-94-5] was detd.  In addn., the new amino acid (3S, 4S)-4-amino-3-hydroxy-5-phenylpentanoic acid  [72155-50-1] was synthesized and the stereochem. of the 3 and 4 positions established.  The tripeptides isovaleryl-L-valyl-(3S,4S)-4-amino-3-hydroxy-6-methylheptanoyl-L-alanineisoamylamide  [72155-64-7] and isovaleryl-L-valyl-4(S)-amino-3(S)-hydroxy-5-phenylpentanoyl-L-alanylisoamylamide  [72155-69-2] were potent inhibitors of pepsin with inhibition consts. of 1 × 10-9 and 0.9 × 10-9M, resp.  Three structural requirements necessary for potent inhibition of pepsin are a 3(S)-hydroxy group in the 3rd residue, a substituent occupying space equiv. to the 1st residue in pepstatin, and a C-terminal group extending from the statine residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvCoE6T-Px4LVg90H21EOLACvtfcHk0li3gVbusBOOmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXlt1CisQ%253D%253D&md5=fe29d3aee059f05259964ba2ffee743c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm00175a006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00175a006%26sid%3Dliteratum%253Aachs%26aulast%3DRich%26aufirst%3DD.%2BH.%26aulast%3DSun%26aufirst%3DE.%2BT.%2BO.%26aulast%3DUlm%26aufirst%3DE.%26atitle%3DSynthesis%2520of%2520Analogues%2520of%2520the%2520Carboxyl%2520Protease%2520Inhibitor%2520Pepstatin.%2520Effects%2520of%2520Structure%2520on%2520Inhibition%2520of%2520Pepsin%2520and%2520Renin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1980%26volume%3D23%26spage%3D27%26epage%3D33%26doi%3D10.1021%2Fjm00175a006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulnik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosur, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sowder, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cachau, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, J. W.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Native and Inhibited Forms of Human Cathepsin D: Implications for Lysosomal Targeting and Drug Design</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">6796</span>– <span class="NLM_lpage">6800</span>, <span class="refDoi"> DOI: 10.1073/pnas.90.14.6796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1073%2Fpnas.90.14.6796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=8393577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3sXltlCgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=6796-6800&author=E.+T.+Baldwinauthor=T.+N.+Bhatauthor=S.+Gulnikauthor=M.+V.+Hosurauthor=R.+C.+Sowderauthor=R.+E.+Cachauauthor=J.+Collinsauthor=A.+M.+Silvaauthor=J.+W.+Erickson&title=Crystal+Structures+of+Native+and+Inhibited+Forms+of+Human+Cathepsin+D%3A+Implications+for+Lysosomal+Targeting+and+Drug+Design&doi=10.1073%2Fpnas.90.14.6796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of native and inhibited forms of human cathepsin D: Implications for lysosomal targeting and drug design</span></div><div class="casAuthors">Baldwin, Eric T.; Bhat, T. Narayana; Gulnik, Sergei; Hosur, Madhusoodan V.; Sowder, Raymond C.; Cachau, Raul E.; Collins, Jack; Silva, Abelardo M.; Erickson, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6796-800</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Cathepsin D (EC 3.4.23.5) (I) is a lysosomal protease suspected to play important roles in protein catabolism, antigen processing, degenerative diseases, and breast cancer progression.  The detn. of the crystal structures of I and its complex with pepstatin (II) at 2.5 Å resoln. provided insights into inhibitor binding and lysosomal targeting for this 2-chain, N-glycosylated aspartic protease.  A comparison with the structures of a complex of II bound to rhizopuspepsin and with a human renin-inhibitor complex revealed differences in subsite structures and inhibitor-enzyme interactions that were consistent with affinity differences and structure-activity relations and suggest strategies for fine-tuning the specificity of I inhibitors.  Mutagenesis studies identified a phosphotransferase recognition region that was required for oligosaccharide phosphorylation but was 32 Å distant from the N-domain glycosylation site at Asn-70.  The electron d. for the crystal structure of cathepsin D indicated the presence of an N-linked oligosaccharide that extended from Asn-70 toward Lys-203, which was a key component of the phosphotransferase recognition region, and thus provided a structural explanation for how the phosphotransferase can recognize apparently distant sites on the protein surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbNgboD93gFLVg90H21EOLACvtfcHk0li3gVbusBOOmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltlCgtrY%253D&md5=e0c671e72699d471fdcdfcaf222eb514</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.14.6796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.14.6796%26sid%3Dliteratum%253Aachs%26aulast%3DBaldwin%26aufirst%3DE.%2BT.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DGulnik%26aufirst%3DS.%26aulast%3DHosur%26aufirst%3DM.%2BV.%26aulast%3DSowder%26aufirst%3DR.%2BC.%26aulast%3DCachau%26aufirst%3DR.%2BE.%26aulast%3DCollins%26aufirst%3DJ.%26aulast%3DSilva%26aufirst%3DA.%2BM.%26aulast%3DErickson%26aufirst%3DJ.%2BW.%26atitle%3DCrystal%2520Structures%2520of%2520Native%2520and%2520Inhibited%2520Forms%2520of%2520Human%2520Cathepsin%2520D%253A%2520Implications%2520for%2520Lysosomal%2520Targeting%2520and%2520Drug%2520Design%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1993%26volume%3D90%26spage%3D6796%26epage%3D6800%26doi%3D10.1073%2Fpnas.90.14.6796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+Drugs+and+Clinical+Candidates%3A+What+Can+Medicinal+Chemists+Learn+from+Their+Properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0li3gVbusBOOmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520Drugs%2520and%2520Clinical%2520Candidates%253A%2520What%2520Can%2520Medicinal%2520Chemists%2520Learn%2520from%2520Their%2520Properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+Are+Great+Cycles%3A+Applications%2C+Opportunities%2C+and+Challenges+of+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0li3EcPC-bOeGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520Are%2520Great%2520Cycles%253A%2520Applications%252C%2520Opportunities%252C%2520and%2520Challenges%2520of%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in New Drug Discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+New+Drug+Discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0li3EcPC-bOeGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520New%2520Drug%2520Discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyndall, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D.</span></span> <span> </span><span class="NLM_article-title">Macrocycles Mimic The Extended Peptide Conformation Recognized By Aspartic, Serine, Cysteine and Metallo Proteases</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">893</span>– <span class="NLM_lpage">907</span>, <span class="refDoi"> DOI: 10.2174/0929867013372715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.2174%2F0929867013372715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=11375757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltVeiur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=893-907&author=J.+Tyndallauthor=D.+Fairlie&title=Macrocycles+Mimic+The+Extended+Peptide+Conformation+Recognized+By+Aspartic%2C+Serine%2C+Cysteine+and+Metallo+Proteases&doi=10.2174%2F0929867013372715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles mimic the extended peptide conformation recognized by aspartic, serine, cysteine and metallo proteases</span></div><div class="casAuthors">Tyndall, Joel D. A.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">893-907</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review with refs.  It has been previously demonstrated that aspartic, serine, metallo and cysteine proteases bind to their inhibitors and substrate analogs in a single conformation, the saw-tooth or extended beta-strand.  Consequently a generic approach to the development of protease inhibitors is the use of constraints that conformationally restrict putative inhibitor mols. to an extended form.  In this way the inhibitor is pre-organized for binding to a protease and does not need to rearrange its structure.  One constraining device that has proven to be effective for such pre-organization is macrocyclization.  This article illustrates the general principle that macrocycles, esp. those composed of 3-4 amino acids and usually 13-17 ring atoms, can effectively mimic the extended conformation of short peptide sequences.  Such structure-stabilizing macrocycles are stable to degrdn. by proteases, valuable components of potent protease inhibitors, and in many cases they are also bioavailable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyuoMyDlMrlbVg90H21EOLACvtfcHk0li3EcPC-bOeGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltVeiur4%253D&md5=2525f64466090d8f371452ed9f0d676c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F0929867013372715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867013372715%26sid%3Dliteratum%253Aachs%26aulast%3DTyndall%26aufirst%3DJ.%26aulast%3DFairlie%26aufirst%3DD.%26atitle%3DMacrocycles%2520Mimic%2520The%2520Extended%2520Peptide%2520Conformation%2520Recognized%2520By%2520Aspartic%252C%2520Serine%252C%2520Cysteine%2520and%2520Metallo%2520Proteases%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2001%26volume%3D8%26spage%3D893%26epage%3D907%26doi%3D10.2174%2F0929867013372715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madala, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyndall, J. D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nall, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Update 1 of: Proteases Universally Recognize Beta Strands in Their Active Sites</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">PR1</span>– <span class="NLM_lpage">PR31</span>, <span class="refDoi"> DOI: 10.1021/cr900368a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr900368a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A280%3ADC%252BC3M7pvFWksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2010&pages=PR1-PR31&author=P.+K.+Madalaauthor=J.+D.+A.+Tyndallauthor=T.+Nallauthor=D.+P.+Fairlie&title=Update+1+of%3A+Proteases+Universally+Recognize+Beta+Strands+in+Their+Active+Sites&doi=10.1021%2Fcr900368a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Update 1 of: Proteases universally recognize beta strands in their active sites</span></div><div class="casAuthors">Madala Praveen K; Tyndall Joel D A; Nall Tessa; Fairlie David P</div><div class="citationInfo"><span class="NLM_cas:title">Chemical reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">PR1-31</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvuWw904ECClw2ofyr07jXfW6udTcc2ebtzi7ulq6Pn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M7pvFWksQ%253D%253D&md5=2f82533e0689b1e9e040bb76cd2ebc61</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fcr900368a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr900368a%26sid%3Dliteratum%253Aachs%26aulast%3DMadala%26aufirst%3DP.%2BK.%26aulast%3DTyndall%26aufirst%3DJ.%2BD.%2BA.%26aulast%3DNall%26aufirst%3DT.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DUpdate%25201%2520of%253A%2520Proteases%2520Universally%2520Recognize%2520Beta%2520Strands%2520in%2520Their%2520Active%2520Sites%26jtitle%3DChem.%2520Rev.%26date%3D2010%26volume%3D110%26spage%3DPR1%26epage%3DPR31%26doi%3D10.1021%2Fcr900368a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fusek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudyš, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, P.</span></span> <span> </span><span class="NLM_article-title">Purification and Crystallization of Human Cathepsin D</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>226</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">557</span>, <span class="refDoi"> DOI: 10.1016/0022-2836(92)90968-p</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2F0022-2836%2892%2990968-p" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1640466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK38XlsVGru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=226&publication_year=1992&pages=555-557&author=M.+Fusekauthor=M.+Baudy%C5%A1author=P.+Metcalf&title=Purification+and+Crystallization+of+Human+Cathepsin+D&doi=10.1016%2F0022-2836%2892%2990968-p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and crystallization of human cathepsin D</span></div><div class="casAuthors">Fusek, Martin; Baudys, Miroslav; Metcalf, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">226</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">555-7</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The two-chain form of human cathepsin D was purified from human spleen with a method utilizing an ion-exchange chromatog. step prior to the pepstatin affinity column normally used to purify aspartic proteases.  The protein was crystd. from 21% polyethylene glycol 8000 at pH 4.0 using the hanging drop vapor diffusion method.  Small crystals were used as seeds to grow crystals suitable for x-ray data collection.  The crystals diffract to a resoln. of 3.2 Å and have space group P212121 with unit cell dimensions a = 59.9 Å, b = 99.6 Å, c = 133.6 Å.  There are two mols. in the asym. unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdR1DNXjyRbLVg90H21EOLACvtfcHk0li_YUVJAAXE3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlsVGru7k%253D&md5=b2b6373be614f155767f7e536c5bd790</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2892%2990968-p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252892%252990968-p%26sid%3Dliteratum%253Aachs%26aulast%3DFusek%26aufirst%3DM.%26aulast%3DBaudy%25C5%25A1%26aufirst%3DM.%26aulast%3DMetcalf%26aufirst%3DP.%26atitle%3DPurification%2520and%2520Crystallization%2520of%2520Human%2520Cathepsin%2520D%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1992%26volume%3D226%26spage%3D555%26epage%3D557%26doi%3D10.1016%2F0022-2836%2892%2990968-p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hánová, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houštecká, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sojka, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopáček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marešová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schueler-Furman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Novel Structural Mechanism of Allosteric Regulation of Aspartic Peptidases via an Evolutionarily Conserved Exosite</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">329.e4</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.chembiol.2018.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=29396291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitl2qsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=318-329.e4&author=I.+H%C3%A1nov%C3%A1author=J.+Bryndaauthor=R.+Hou%C5%A1teck%C3%A1author=N.+Alamauthor=D.+Sojkaauthor=P.+Kop%C3%A1%C4%8Dekauthor=L.+Mare%C5%A1ov%C3%A1author=J.+Vondr%C3%A1%C5%A1ekauthor=M.+Hornauthor=O.+Schueler-Furmanauthor=M.+Mare%C5%A1&title=Novel+Structural+Mechanism+of+Allosteric+Regulation+of+Aspartic+Peptidases+via+an+Evolutionarily+Conserved+Exosite&doi=10.1016%2Fj.chembiol.2018.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Structural Mechanism of Allosteric Regulation of Aspartic Peptidases via an Evolutionarily Conserved Exosite</span></div><div class="casAuthors">Hanova, Iva; Brynda, Jiri; Houstecka, Radka; Alam, Nawsad; Sojka, Daniel; Kopacek, Petr; Maresova, Lucie; Vondrasek, Jiri; Horn, Martin; Schueler-Furman, Ora; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">318-329.e4</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Pepsin-family aspartic peptidases are biosynthesized as inactive zymogens in which the propeptide blocks the active site until its proteolytic removal upon enzyme activation.  Here, we describe a novel dual regulatory function for the propeptide using a set of crystal structures of the parasite cathepsin D IrCD1.  In the IrCD1 zymogen, intramol. autoinhibition by the intact propeptide is mediated by an evolutionarily conserved exosite on the enzyme core.  After activation, the mature enzyme employs the same exosite to rebind a small fragment derived from the cleaved propeptide.  This fragment functions as an effective natural inhibitor of mature IrCD1 that operates in a pH-dependent manner through a unique allosteric inhibition mechanism.  The study uncovers the propeptide-binding exosite as a target for the regulation of pepsin-family aspartic peptidases and defines the structural requirements for exosite inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0J167H_OHt7Vg90H21EOLACvtfcHk0li_YUVJAAXE3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitl2qsLo%253D&md5=926ccddbe7a6681ee9c0fd2e9ec940d0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25A1nov%25C3%25A1%26aufirst%3DI.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DHou%25C5%25A1teck%25C3%25A1%26aufirst%3DR.%26aulast%3DAlam%26aufirst%3DN.%26aulast%3DSojka%26aufirst%3DD.%26aulast%3DKop%25C3%25A1%25C4%258Dek%26aufirst%3DP.%26aulast%3DMare%25C5%25A1ov%25C3%25A1%26aufirst%3DL.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DSchueler-Furman%26aufirst%3DO.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DNovel%2520Structural%2520Mechanism%2520of%2520Allosteric%2520Regulation%2520of%2520Aspartic%2520Peptidases%2520via%2520an%2520Evolutionarily%2520Conserved%2520Exosite%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D318%26epage%3D329.e4%26doi%3D10.1016%2Fj.chembiol.2018.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Máša, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marešová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ježek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Cathepsin D Propeptide: Mechanism and Regulation of Its Interaction with the Catalytic Core</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">15474</span>– <span class="NLM_lpage">15482</span>, <span class="refDoi"> DOI: 10.1021/bi0614986</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0614986" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1yiur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=15474-15482&author=M.+M%C3%A1%C5%A1aauthor=L.+Mare%C5%A1ov%C3%A1author=J.+Vondr%C3%A1%C5%A1ekauthor=M.+Hornauthor=J.+Je%C5%BEekauthor=M.+Mare%C5%A1&title=Cathepsin+D+Propeptide%3A+Mechanism+and+Regulation+of+Its+Interaction+with+the+Catalytic+Core&doi=10.1021%2Fbi0614986"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Cathepsin D Propeptide: Mechanism and Regulation of Its Interaction with the Catalytic Core</span></div><div class="casAuthors">Masa, Martin; Maresova, Lucie; Vondrasek, Jiri; Horn, Martin; Jezek, Jan; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">15474-15482</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Propeptide blocks the active site in the inactive zymogen of cathepsin D and is cleaved off during zymogen activation.  We have designed a set of peptidic fragments derived from the propeptide structure and evaluated their inhibitory potency against mature cathepsin D using a kinetic assay.  Our mapping of the cathepsin D propeptide indicated two domains in the propeptide involved in the inhibitory interaction with the enzyme core: the active site "anchor" domain and the N-terminus of the propeptide.  The latter plays a dominant role in propeptide inhibition (nanomolar Ki), and its high-affinity binding was corroborated by fluorescence polarization measurements.  In addn. to the inhibitory domains of propeptide, a fragment derived from the N-terminus of mature cathepsin D displayed inhibition.  This finding supports its proposed regulatory function.  The interaction mechanisms of the identified inhibitory domains were characterized by detg. their modes of inhibition as well as by spatial modeling of the propeptide in the zymogen mol.  The inhibitory interaction of the N-terminal propeptide domain was abolished in the presence of sulfated polysaccharides, which interact with basic propeptide residues.  The inhibitory potency of the active site anchor domain was affected by the Ala38pVal substitution, a propeptide polymorphism reported to be assocd. with the pathol. of Alzheimer's disease.  We infer that propeptide is a sensitive tethered ligand that allows for complex modulation of cathepsin D zymogen activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfewqajyt-OrVg90H21EOLACvtfcHk0ljelD7VelR40g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1yiur3F&md5=4829737bb45ca56d93d189320d8afc08</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fbi0614986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0614986%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25A1%25C5%25A1a%26aufirst%3DM.%26aulast%3DMare%25C5%25A1ov%25C3%25A1%26aufirst%3DL.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DJe%25C5%25BEek%26aufirst%3DJ.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DCathepsin%2520D%2520Propeptide%253A%2520Mechanism%2520and%2520Regulation%2520of%2520Its%2520Interaction%2520with%2520the%2520Catalytic%2520Core%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D15474%26epage%3D15482%26doi%3D10.1021%2Fbi0614986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljelD7VelR40g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Žebrakovská, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Máša, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vávrová, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Complex Modulation of Peptidolytic Activity of Cathepsin D by Sphingolipids</span>. <i>Biochim. Biophys. Acta, Mol. Cell Biol. Lipids</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>1811</i></span>,  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1104</span>, <span class="refDoi"> DOI: 10.1016/j.bbalip.2011.09.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.bbalip.2011.09.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2nu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1811&publication_year=2011&pages=1097-1104&author=I.+%C5%BDebrakovsk%C3%A1author=M.+M%C3%A1%C5%A1aauthor=J.+Srpauthor=M.+Hornauthor=K.+V%C3%A1vrov%C3%A1author=M.+Mare%C5%A1&title=Complex+Modulation+of+Peptidolytic+Activity+of+Cathepsin+D+by+Sphingolipids&doi=10.1016%2Fj.bbalip.2011.09.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Complex modulation of peptidolytic activity of cathepsin D by sphingolipids</span></div><div class="casAuthors">Zebrakovska, Iva; Masa, Martin; Srp, Jaroslav; Horn, Martin; Vavrova, Katerina; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular and Cell Biology of Lipids</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1811</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1097-1104</span>CODEN:
                <span class="NLM_cas:coden">BBMLFG</span>;
        ISSN:<span class="NLM_cas:issn">1388-1981</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">Cathepsin D is an aspartic peptidase involved in cellular processes including proliferation and apoptosis and implicated in human pathologies such as cancer and neurodegeneration.  Our knowledge about the relationship between proteolysis and bioactive sphingolipids is still very limited.  Here, we describe a complex pattern of modulation of the peptidolytic activity of cathepsin D by sphingolipids.  A panel of sphingolipid derivs. was screened in a FRET-based assay; these mols. demonstrated neg. or pos. modulation of cathepsin D peptidolytic activity, depending on the sphingolipid structure.  Certain sphingosines and ceramides inhibited cathepsin D in the submicromolar range, and structural requirements for this inhibitory effect were evaluated.  The interaction of cathepsin D with sphingolipids was also demonstrated by fluorescence polarization measurements and detd. to follow a competitive inhibition mode.  In contrast, monoester phosphosphingolipids, esp. ceramide-1-phosphate, were identified as activators of cathepsin D peptidolytic activity at submicromolar concns.  Thus, sphingolipids and phosphosphingolipids, known to be antagonistic in their cell-signaling functions, displayed opposite modulation of cathepsin D.  Sphingolipid-based modulators of cathepsin D are potentially involved in the control of cathepsin D-dependent processes and might serve as a scaffold for the development of novel regulators of this therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI_gDsPWriw7Vg90H21EOLACvtfcHk0ljelD7VelR40g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2nu7fP&md5=c9221adb25355b37952351c0e1837a0f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.bbalip.2011.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbalip.2011.09.005%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25BDebrakovsk%25C3%25A1%26aufirst%3DI.%26aulast%3DM%25C3%25A1%25C5%25A1a%26aufirst%3DM.%26aulast%3DSrp%26aufirst%3DJ.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DV%25C3%25A1vrov%25C3%25A1%26aufirst%3DK.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DComplex%2520Modulation%2520of%2520Peptidolytic%2520Activity%2520of%2520Cathepsin%2520D%2520by%2520Sphingolipids%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Biol.%2520Lipids%26date%3D2011%26volume%3D1811%26spage%3D1097%26epage%3D1104%26doi%3D10.1016%2Fj.bbalip.2011.09.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uhlíková, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichová, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souček, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kräusslich, H.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span> <span> </span><span class="NLM_article-title">A Modular Approach to HIV-1 Proteinase Inhibitor Design</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>222</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">43</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1996.0694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1006%2Fbbrc.1996.0694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=8630071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK28XivVOisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=222&publication_year=1996&pages=38-43&author=T.+Uhl%C3%ADkov%C3%A1author=J.+Konvalinkaauthor=I.+Pichov%C3%A1author=M.+Sou%C4%8Dekauthor=H.-G.+Kr%C3%A4usslichauthor=J.+Vondr%C3%A1%C5%A1ek&title=A+Modular+Approach+to+HIV-1+Proteinase+Inhibitor+Design&doi=10.1006%2Fbbrc.1996.0694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A modular approach to HIV-1 proteinase inhibitor design</span></div><div class="casAuthors">Uhlikova, Tat'ana; Konvalinka, Jan; Pichova, Iva; Soucek, Milan; Krausslich, Hans-Georg; Vondrasek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">222</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-43</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">HIV-1 proteinase represents a promising target for antiviral chemotherapy.  The authors have designed, synthesized, and tested modular inhibitors combining an active-site inhibitor tethered to a structure targeted to the dimerization domain of the enzyme.  At pH 5, the parent active site inhibitor, the equimolar mixt. of active site and dimerization inhibitors, and the best compd. from the authors series of modular inhibitors show the same inhibition activity.  At neutral pH, however, the combination of the dimerization and active-site inhibitors shows a synergistic effect.  Moreover, the modular inhibitor has an IC50 value 5x lower than the parent active site inhibitor and 2x lower than the equimolar mixt. of the two parent inhibitors.  The Lineweaver-Burk plot for modular inhibitors corresponds to a pattern for mixed type inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCOV3RJNyOB7Vg90H21EOLACvtfcHk0ljelD7VelR40g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XivVOisbY%253D&md5=874ceb808d7294d55ddf093837cc84c8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1996.0694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1996.0694%26sid%3Dliteratum%253Aachs%26aulast%3DUhl%25C3%25ADkov%25C3%25A1%26aufirst%3DT.%26aulast%3DKonvalinka%26aufirst%3DJ.%26aulast%3DPichov%25C3%25A1%26aufirst%3DI.%26aulast%3DSou%25C4%258Dek%26aufirst%3DM.%26aulast%3DKr%25C3%25A4usslich%26aufirst%3DH.-G.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26atitle%3DA%2520Modular%2520Approach%2520to%2520HIV-1%2520Proteinase%2520Inhibitor%2520Design%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1996%26volume%3D222%26spage%3D38%26epage%3D43%26doi%3D10.1006%2Fbbrc.1996.0694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumanchin-Njock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayral, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vizzavona, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farzan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boisbrun, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulcrand, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefranc-Jullien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Checler, F.</span></span> <span> </span><span class="NLM_article-title">BACE1- and BACE2-Expressing Human Cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">25859</span>– <span class="NLM_lpage">25866</span>, <span class="refDoi"> DOI: 10.1074/jbc.m302622200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1074%2Fjbc.m302622200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=12736275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlt1ClsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=25859-25866&author=D.+Andrauauthor=C.+Dumanchin-Njockauthor=E.+Ayralauthor=J.+Vizzavonaauthor=M.+Farzanauthor=M.+Boisbrunauthor=P.+Fulcrandauthor=J.-F.+Hernandezauthor=J.+Martinezauthor=S.+Lefranc-Jullienauthor=F.+Checler&title=BACE1-+and+BACE2-Expressing+Human+Cells&doi=10.1074%2Fjbc.m302622200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">BACE1- and BACE2-expressing Human Cells. Characterization of β-Amyloid Precursor Protein-Derived Catabolites, Design of a Novel Fluorimetric Assay, and Identification of New In Vitro Inhibitors</span></div><div class="casAuthors">Andrau, David; Dumanchin-Njock, Cecile; Ayral, Erwan; Vizzavona, Jean; Farzan, Mike; Boisbrun, Michel; Fulcrand, Pierre; Hernandez, Jean-Francois; Martinez, Jean; Lefranc-Jullien, Solveig; Checler, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">25859-25866</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We have set up stably transfected HEK293 cells overexpressing the β-secretases BACE1 and BACE2 either alone or in combination with wild-type β-amyloid precursor protein (βAPP).  The characterization of the βAPP-derived catabolites indicates that cells expressing BACEs produce less genuine Aβ1- 40/42 but higher amts. of secreted sAPPβ and N-terminal-truncated Aβ species.  This was accompanied by a concomitant modulation of the C-terminal counterpart products C89 and C79 for BACE1 and BACE2, resp.  These cells were used to set up a novel BACE assay based on two quenched fluorimetric substrates mimicking the wild-type (JMV2235) and Swedish-mutated (JMV2236) βAPP sequences targeted by BACE activities.  We show that BACEs activities are enhanced by the Swedish mutation and maximal at pH 4.5.  The specificity of this double assay for genuine β-secretase activity was demonstrated by means of cathepsin D, a "false pos." BACE candidate.  Thus, cathepsin D was unable to cleave preferentially the JMV2236-mutated substrate.  The selectivity of the assay was also emphasized by the lack of JMV cleavage triggered by other "secretases" candidates such as ADAM10 (A disintegrin and metalloprotease 10), tumor necrosis α-converting enzyme, and presenilins 1 and 2.  Finally, the assay was used to screen for putative in vitro BACE inhibitors.  We identified a series of statine-derived sequences that dose-dependently inhibited BACE1 and BACE2 activities with IC50 in the micromolar range, some of which displaying selectivity for either BACE1 or BACE2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp47akCuIAZhLVg90H21EOLACvtfcHk0ljiKjxmOjmC2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlt1ClsLs%253D&md5=fe070e46df90ab30006d4688c0859fda</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m302622200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m302622200%26sid%3Dliteratum%253Aachs%26aulast%3DAndrau%26aufirst%3DD.%26aulast%3DDumanchin-Njock%26aufirst%3DC.%26aulast%3DAyral%26aufirst%3DE.%26aulast%3DVizzavona%26aufirst%3DJ.%26aulast%3DFarzan%26aufirst%3DM.%26aulast%3DBoisbrun%26aufirst%3DM.%26aulast%3DFulcrand%26aufirst%3DP.%26aulast%3DHernandez%26aufirst%3DJ.-F.%26aulast%3DMartinez%26aufirst%3DJ.%26aulast%3DLefranc-Jullien%26aufirst%3DS.%26aulast%3DChecler%26aufirst%3DF.%26atitle%3DBACE1-%2520and%2520BACE2-Expressing%2520Human%2520Cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D25859%26epage%3D25866%26doi%3D10.1074%2Fjbc.m302622200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holzman, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubbins, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pederson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krafft, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G. T.</span></span> <span> </span><span class="NLM_article-title">Recombinant Human Prorenin from CHO Cells: Expression and Purification</span>. <i>J. Protein Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">663</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1007/bf01024761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1007%2FBF01024761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1963533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3MXps1Wqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1990&pages=663-672&author=T.+F.+Holzmanauthor=C.+C.+Chungauthor=R.+Edaljiauthor=D.+A.+Eganauthor=E.+J.+Gubbinsauthor=A.+Rueterauthor=G.+Howardauthor=L.+K.+Yangauthor=T.+M.+Pedersonauthor=G.+A.+Krafftauthor=G.+T.+Wang&title=Recombinant+Human+Prorenin+from+CHO+Cells%3A+Expression+and+Purification&doi=10.1007%2Fbf01024761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Recombinant human prorenin from CHO cells:  expression and purification</span></div><div class="casAuthors">Holzman, Thomas F.; Chung, Christine C.; Edalji, Rohinton; Egan, David A.; Gubbins, Earl J.; Rueter, Annemarie; Howard, Gail; Yang, Lana K.; Pederson, Terry M.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Protein Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">663-72</span>CODEN:
                <span class="NLM_cas:coden">JPCHD2</span>;
        ISSN:<span class="NLM_cas:issn">0277-8033</span>.
    </div><div class="casAbstract">The gene for human preprorenin was obtained from total RNA prepd. from primary human chorion cells.  An expression vector was constructed contg. an SV40 early promoter, a human preprorenin cDNA, bovine growth hormone poly-A addn. signal, and a dihydrofolate reductase (dhfr) expression cassette.  This vector was inserted into the DXB 11 Chinese hamster ovary (CHO) cell line.  The recombinant protein was exported by CHO cells into the tissue culture media.  At harvest the prorenin levels ranged from ∼1-5 mg/L.  For protein isolation the cell culture supernatants were processed by filtration, concn., dialysis, and batch extn.  Preparative-scale isolation of prorenin was accomplished using blue-dye chromatog. and size-exclusion chromatog.  The isolated prorenin yielded a single SDS-gel band with Mr ∼ 40,000.  The proprotein was characterized with respect to N-terminal sequence and N-linked sugar compn.  Trypsin-activated renin prepd. from the proprotein was characterized with respect to N-terminal sequence and pH-activity profile.  Enzyme activity was measured with a newly developed fluorogenic peptide substrate contg. the P6-P'3 sequence of human angiotensinogen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFuzdaHY7v7Vg90H21EOLACvtfcHk0ljiKjxmOjmC2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXps1Wqtw%253D%253D&md5=6e4eb32ddfb7bcb68b13ef2b25f46f41</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2FBF01024761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01024761%26sid%3Dliteratum%253Aachs%26aulast%3DHolzman%26aufirst%3DT.%2BF.%26aulast%3DChung%26aufirst%3DC.%2BC.%26aulast%3DEdalji%26aufirst%3DR.%26aulast%3DEgan%26aufirst%3DD.%2BA.%26aulast%3DGubbins%26aufirst%3DE.%2BJ.%26aulast%3DRueter%26aufirst%3DA.%26aulast%3DHoward%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DL.%2BK.%26aulast%3DPederson%26aufirst%3DT.%2BM.%26aulast%3DKrafft%26aufirst%3DG.%2BA.%26aulast%3DWang%26aufirst%3DG.%2BT.%26atitle%3DRecombinant%2520Human%2520Prorenin%2520from%2520CHO%2520Cells%253A%2520Expression%2520and%2520Purification%26jtitle%3DJ.%2520Protein%2520Chem.%26date%3D1990%26volume%3D9%26spage%3D663%26epage%3D672%26doi%3D10.1007%2Fbf01024761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salát, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paesen, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáčová, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotsyfakis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovářová, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šanda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majtán, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunclová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horká, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunn, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopáček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopecký, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure and Functional Characterization of an Immunomodulatory Salivary Cystatin from the Soft Tick Ornithodoros moubata</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1042/bj20100280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1042%2Fbj20100280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=20545626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsVejsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2010&pages=103-112&author=J.+Sal%C3%A1tauthor=G.+C.+Paesenauthor=P.+%C5%98ez%C3%A1%C4%8Dov%C3%A1author=M.+Kotsyfakisauthor=Z.+Kov%C3%A1%C5%99ov%C3%A1author=M.+%C5%A0andaauthor=J.+Majt%C3%A1nauthor=L.+Grunclov%C3%A1author=H.+Hork%C3%A1author=J.+F.+Andersenauthor=J.+Bryndaauthor=M.+Hornauthor=M.+A.+Nunnauthor=P.+Kop%C3%A1%C4%8Dekauthor=J.+Kopeck%C3%BDauthor=M.+Mare%C5%A1&title=Crystal+Structure+and+Functional+Characterization+of+an+Immunomodulatory+Salivary+Cystatin+from+the+Soft+Tick+Ornithodoros+moubata&doi=10.1042%2Fbj20100280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure and functional characterization of an immunomodulatory salivary cystatin from the soft tick Ornithodoros moubata</span></div><div class="casAuthors">Salat, Jiri; Paesen, Guido C.; Rezacova, Pavlina; Kotsyfakis, Michalis; Kovarova, Zuzana; Sanda, Miloslav; Majtan, Juraj; Grunclova, Lenka; Horka, Helena; Andersen, John F.; Brynda, Jiri; Horn, Martin; Nunn, Miles A.; Kopacek, Petr; Kopecky, Jan; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-112</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The saliva of blood-feeding parasites is a rich source of peptidase inhibitors that help to overcome the host's defense during host-parasite interactions.  Using proteomic anal., the cystatin OmC2 was demonstrated in the saliva of the soft tick Ornithodoros moubata, an important disease vector transmitting African swine fever virus and the spirochaete Borrelia duttoni.  A structural, biochem., and biol. characterization of this peptidase inhibitor was undertaken in the present study.  Recombinant OmC2 was screened against a panel of physiol. relevant peptidases and was found to be an effective broad-specificity inhibitor of cysteine cathepsins, including endopeptidases (cathepsins L and S) and exopeptidases (cathepsins B, C, and H).  The crystal structure of OmC2 was detd. at a resoln. of 2.45 Å (1 Å = 0.1 nm) and was used to describe the structure-inhibitory activity relationship.  The biol. impact of OmC2 was demonstrated both in vitro and in vivo.  OmC2 affected the function of antigen-presenting mouse dendritic cells (DCs) by reducing the prodn. of the pro-inflammatory cytokines tumor necrosis factor α (TNFα) and interleukin-12 (IL-12), and proliferation of antigen-specific CD4+ T-cells.  This suggests that OmC2 may suppress the host's adaptive immune response.  Immunization of mice with OmC2 significantly suppressed the survival of O. moubata in infestation expts.  We conclude that OmC2 is a promising target for the development of a novel anti-tick vaccine to control O. moubata populations and combat the spread of assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-TcGTTb9FTbVg90H21EOLACvtfcHk0lhg3RMMvbXpoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsVejsr4%253D&md5=e1a613bd43157cbfcb8346efacc894f0</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1042%2Fbj20100280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20100280%26sid%3Dliteratum%253Aachs%26aulast%3DSal%25C3%25A1t%26aufirst%3DJ.%26aulast%3DPaesen%26aufirst%3DG.%2BC.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258Dov%25C3%25A1%26aufirst%3DP.%26aulast%3DKotsyfakis%26aufirst%3DM.%26aulast%3DKov%25C3%25A1%25C5%2599ov%25C3%25A1%26aufirst%3DZ.%26aulast%3D%25C5%25A0anda%26aufirst%3DM.%26aulast%3DMajt%25C3%25A1n%26aufirst%3DJ.%26aulast%3DGrunclov%25C3%25A1%26aufirst%3DL.%26aulast%3DHork%25C3%25A1%26aufirst%3DH.%26aulast%3DAndersen%26aufirst%3DJ.%2BF.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DNunn%26aufirst%3DM.%2BA.%26aulast%3DKop%25C3%25A1%25C4%258Dek%26aufirst%3DP.%26aulast%3DKopeck%25C3%25BD%26aufirst%3DJ.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DCrystal%2520Structure%2520and%2520Functional%2520Characterization%2520of%2520an%2520Immunomodulatory%2520Salivary%2520Cystatin%2520from%2520the%2520Soft%2520Tick%2520Ornithodoros%2520moubata%26jtitle%3DBiochem.%2520J.%26date%3D2010%26volume%3D429%26spage%3D103%26epage%3D112%26doi%3D10.1042%2Fbj20100280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumerová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Digestive Proteolysis in the Colorado Potato Beetle, Leptinotarsa decemlineata: Activity-Based Profiling and Imaging of a Multipeptidase Network</span>. <i>Insect Biochem. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.ibmb.2016.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.ibmb.2016.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=27539253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVamsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2016&pages=1-11&author=J.+Srpauthor=M.+Nussbaumerov%C3%A1author=M.+Hornauthor=M.+Mare%C5%A1&title=Digestive+Proteolysis+in+the+Colorado+Potato+Beetle%2C+Leptinotarsa+decemlineata%3A+Activity-Based+Profiling+and+Imaging+of+a+Multipeptidase+Network&doi=10.1016%2Fj.ibmb.2016.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Digestive proteolysis in the Colorado potato beetle, Leptinotarsa decemlineata: Activity-based profiling and imaging of a multipeptidase network</span></div><div class="casAuthors">Srp, Jaroslav; Nussbaumerova, Martina; Horn, Martin; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Insect Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">IBMBES</span>;
        ISSN:<span class="NLM_cas:issn">0965-1748</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Colorado potato beetle (CPB), Leptinotarsa decemlineata, is a major pest of potato plants, and its digestive system is a promising target for development of pest control strategies.  This work focuses on functional proteomic anal. of the digestive proteolytic enzymes expressed in the CPB gut.  We identified a set of peptidases using imaging with specific activity-based probes and activity profiling with selective substrates and inhibitors.  The secreted luminal peptidases were classified as: (i) endopeptidases of cathepsin D, cathepsin L, and trypsin types and (ii) exopeptidases with aminopeptidase (cathepsin H), carboxypeptidase (serine carboxypeptidase, prolyl carboxypeptidase), and carboxydipeptidase (cathepsin B) activities.  The proteolytic arsenal also includes non-luminal peptidases with prolyl oligopeptidase and metalloaminopeptidase activities.  Our results indicate that the CPB gut employs a multienzyme network of peptidases with complementary specificities to efficiently degrade ingested proteins.  This proteolytic system functions in both CPB larvae and adults and is controlled mainly by cysteine and aspartic peptidases and supported by serine and metallopeptidases.  The component enzymes identified here are potential targets for inhibitors with tailored specificities that could be engineered into potato plants to confer resistance to CPB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokT6cNqUFhs7Vg90H21EOLACvtfcHk0lhg3RMMvbXpoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVamsbjE&md5=a814b7107f414d0f85d9a6dec4fd9160</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ibmb.2016.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ibmb.2016.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DSrp%26aufirst%3DJ.%26aulast%3DNussbaumerov%25C3%25A1%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DDigestive%2520Proteolysis%2520in%2520the%2520Colorado%2520Potato%2520Beetle%252C%2520Leptinotarsa%2520decemlineata%253A%2520Activity-Based%2520Profiling%2520and%2520Imaging%2520of%2520a%2520Multipeptidase%2520Network%26jtitle%3DInsect%2520Biochem.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D78%26spage%3D1%26epage%3D11%26doi%3D10.1016%2Fj.ibmb.2016.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jílková, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáčová, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marešová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fajtová, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondrášek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKerrow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mareš, M.</span></span> <span> </span><span class="NLM_article-title">Activation Route of the Schistosoma mansoni Cathepsin B1 Drug Target: Structural Map with a Glycosaminoglycan Switch</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1786</span>– <span class="NLM_lpage">1798</span>, <span class="refDoi"> DOI: 10.1016/j.str.2014.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2Fj.str.2014.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=25456815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=1786-1798&author=A.+J%C3%ADlkov%C3%A1author=M.+Hornauthor=P.+%C5%98ez%C3%A1%C4%8Dov%C3%A1author=L.+Mare%C5%A1ov%C3%A1author=P.+Fajtov%C3%A1author=J.+Bryndaauthor=J.+Vondr%C3%A1%C5%A1ekauthor=J.+H.+McKerrowauthor=C.+R.+Caffreyauthor=M.+Mare%C5%A1&title=Activation+Route+of+the+Schistosoma+mansoni+Cathepsin+B1+Drug+Target%3A+Structural+Map+with+a+Glycosaminoglycan+Switch&doi=10.1016%2Fj.str.2014.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Activation Route of the Schistosoma mansoni Cathepsin B1 Drug Target: Structural Map with a Glycosaminoglycan Switch</span></div><div class="casAuthors">Jilkova, Adela; Horn, Martin; Rezacova, Pavlina; Maresova, Lucie; Fajtova, Pavla; Brynda, Jiri; Vondrasek, Jiri; McKerrow, James H.; Caffrey, Conor R.; Mares, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1786-1798</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cathepsin B1 (SmCB1) is a digestive protease of the parasitic blood fluke Schistosoma mansoni and a drug target for the treatment of schistosomiasis, a disease that afflicts over 200 million people.  SmCB1 is synthesized as an inactive zymogen in which the N-terminal propeptide blocks the active site.  We investigated the activation of the zymogen by which the propeptide is proteolytically removed and its regulation by sulfated polysaccharides (SPs).  We detd. crystal structures of three mol. forms of SmCB1 along the activation pathway: the zymogen, an activation intermediate with a partially cleaved propeptide, and the mature enzyme.  We demonstrate that SPs are essential for the autocatalytic activation of SmCB1, as they interact with a specific heparin-binding domain in the propeptide.  An alternative activation route is mediated by an S. mansoni asparaginyl endopeptidase (legumain) which is downregulated by SPs, indicating that SPs act as a mol. switch between both activation mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzrhuTfrvUy7Vg90H21EOLACvtfcHk0lhg3RMMvbXpoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGit7bO&md5=bd180bad1ea5ef69a4bd81bb76c80827</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DJ%25C3%25ADlkov%25C3%25A1%26aufirst%3DA.%26aulast%3DHorn%26aufirst%3DM.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258Dov%25C3%25A1%26aufirst%3DP.%26aulast%3DMare%25C5%25A1ov%25C3%25A1%26aufirst%3DL.%26aulast%3DFajtov%25C3%25A1%26aufirst%3DP.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DVondr%25C3%25A1%25C5%25A1ek%26aufirst%3DJ.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DCaffrey%26aufirst%3DC.%2BR.%26aulast%3DMare%25C5%25A1%26aufirst%3DM.%26atitle%3DActivation%2520Route%2520of%2520the%2520Schistosoma%2520mansoni%2520Cathepsin%2520B1%2520Drug%2520Target%253A%2520Structural%2520Map%2520with%2520a%2520Glycosaminoglycan%2520Switch%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D1786%26epage%3D1798%26doi%3D10.1016%2Fj.str.2014.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Šmídková, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvoráková, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tloust’ová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Česnek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertlíková-Kaiserová, H.</span></span> <span> </span><span class="NLM_article-title">Amidate Prodrugs of 9-[2-(Phosphonomethoxy)Ethyl]Adenine as Inhibitors of Adenylate Cyclase Toxin from Bordetella pertussis</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">671</span>, <span class="refDoi"> DOI: 10.1128/aac.01685-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1128%2Faac.01685-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=24145524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVSru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=664-671&author=M.+%C5%A0m%C3%ADdkov%C3%A1author=A.+Dvor%C3%A1kov%C3%A1author=E.+Tloust%E2%80%99ov%C3%A1author=M.+%C4%8Cesnekauthor=Z.+Janebaauthor=H.+Mertl%C3%ADkov%C3%A1-Kaiserov%C3%A1&title=Amidate+Prodrugs+of+9-%5B2-%28Phosphonomethoxy%29Ethyl%5DAdenine+as+Inhibitors+of+Adenylate+Cyclase+Toxin+from+Bordetella+pertussis&doi=10.1128%2Faac.01685-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Amidate prodrugs of 9-[2-(phosphonomethoxy)ethyl]adenine as inhibitors of adenylate cyclase toxin from Bordetella pertussis</span></div><div class="casAuthors">Smidkova, Marketa; Dvorakova, Alexandra; Tloust'ova, Eva; Cesnek, Michal; Janeba, Zlatko; Mertlikova-Kaiserova, Helena</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">664-671, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Adenylate cyclase toxin (ACT) is the key virulence factor of Bordetella pertussis that facilitates its invasion into the mammalian body. 9-[2-(Phosphonomethoxy)ethyl]adenine diphosphate (PMEApp), the active metabolite of the antiviral drug bis(POM)PMEA (adefovir dipivoxil), has been shown to inhibit ACT.  The objective of this study was to evaluate six novel amidate prodrugs of PMEA, both phenyloxy phosphonamidates and phosphonodiamidates, for their ability to inhibit ACT activity in the J774A.1 macrophage cell line.  The two phenyloxy phosphonamidate prodrugs exhibited greater inhibitory activity (50% inhibitory concn. [IC50] = 22 and 46 nM) than the phosphonodiamidates (IC50 = 84 to 3,960 nM).  The inhibitory activity of the prodrugs correlated with their lipophilicity and the degree of their hydrolysis into free PMEA in J774A.1 cells.  Although the prodrugs did not inhibit ACT as effectively as bis(POM)PMEA (IC50 = 6 nM), they were significantly less cytotoxic.  Moreover, they all reduced apoptotic effects of ACT and prevented an ACT-induced elevation of intracellular [Ca2+]i.  The amidate prodrugs were less susceptible to degrdn. in Caco-2 cells compared to bis(POM)PMEA, while they exerted good transepithelial permeability in this assay.  As a consequence, a large amt. of intact amidate prodrug is expected to be available to target macrophages in vivo.  This feature makes nontoxic amidate prodrugs attractive candidates for further investigation as novel antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplArKpHfZBhrVg90H21EOLACvtfcHk0lhfhl4nS1kWkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVSru7c%253D&md5=fbd9057b4f326fb45cd131419ad90772</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1128%2Faac.01685-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.01685-13%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0m%25C3%25ADdkov%25C3%25A1%26aufirst%3DM.%26aulast%3DDvor%25C3%25A1kov%25C3%25A1%26aufirst%3DA.%26aulast%3DTloust%25E2%2580%2599ov%25C3%25A1%26aufirst%3DE.%26aulast%3D%25C4%258Cesnek%26aufirst%3DM.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DMertl%25C3%25ADkov%25C3%25A1-Kaiserov%25C3%25A1%26aufirst%3DH.%26atitle%3DAmidate%2520Prodrugs%2520of%25209-%255B2-%2528Phosphonomethoxy%2529Ethyl%255DAdenine%2520as%2520Inhibitors%2520of%2520Adenylate%2520Cyclase%2520Toxin%2520from%2520Bordetella%2520pertussis%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2014%26volume%3D58%26spage%3D664%26epage%3D671%26doi%3D10.1128%2Faac.01685-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">An Approach to Multi-Copy Search in Molecular Replacement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1622</span>– <span class="NLM_lpage">1624</span>, <span class="refDoi"> DOI: 10.1107/s0907444900013780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2Fs0907444900013780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=11092928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVOksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2000&pages=1622-1624&author=A.+Vaginauthor=A.+Teplyakov&title=An+Approach+to+Multi-Copy+Search+in+Molecular+Replacement&doi=10.1107%2Fs0907444900013780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">An approach to multi-copy search in molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">D56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1622-1624</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">The mol.-replacement method has been extended to a simultaneous search for multiple copies of the macromol. in the unit cell.  The central point of this approach is the construction of a multi-copy search model from the properly oriented monomers using a special translation function.  The multi-copy search method has been implemented in the program MOLREP and successfully tested using exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWCUH_hO_qmLVg90H21EOLACvtfcHk0lhfhl4nS1kWkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVOksLo%253D&md5=b0143869509b3906e5f6e861d6fa2b32</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2Fs0907444900013780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444900013780%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DAn%2520Approach%2520to%2520Multi-Copy%2520Search%2520in%2520Molecular%2520Replacement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2000%26volume%3D56%26spage%3D1622%26epage%3D1624%26doi%3D10.1107%2Fs0907444900013780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhfhl4nS1kWkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S.</span></span> <span> </span><span class="NLM_article-title">The Ccp4 Suite - Programs for Protein Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=S.+Bailey&title=The+Ccp4+Suite+-+Programs+for+Protein+Crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DS.%26atitle%3DThe%2520Ccp4%2520Suite%2520-%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhfhl4nS1kWkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljoRuFRkD1NKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Babin, V.</span>; <span class="NLM_string-name">Berryman, J. T.</span>; <span class="NLM_string-name">Betz, R. M.</span>; <span class="NLM_string-name">Cai, Q.</span>; <span class="NLM_string-name">Cerutti, D. S.</span>; <span class="NLM_string-name">Cheatham, T. E.</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Duke, R. E.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Goetz, A. W.</span>; <span class="NLM_string-name">Gusarov, S.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Janowski, P.</span>; <span class="NLM_string-name">Kaus, J.</span>; <span class="NLM_string-name">Kolossváry, I.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Madej, B.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Paesani, F.</span>; <span class="NLM_string-name">Roe, D. R.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Salomon-Ferrer, R.</span>; <span class="NLM_string-name">Seabra, G.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Smith, W.</span>; <span class="NLM_string-name">Swails, J.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wolf, R. M.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>Amber 14</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+A.+Case&author=V.+Babin&author=J.+T.+Berryman&author=R.+M.+Betz&author=Q.+Cai&author=D.+S.+Cerutti&author=T.+E.+Cheatham&author=T.+A.+Darden&author=R.+E.+Duke&author=H.+Gohlke&author=A.+W.+Goetz&author=S.+Gusarov&author=N.+Homeyer&author=P.+Janowski&author=J.+Kaus&author=I.+Kolossv%C3%A1ry&author=A.+Kovalenko&author=T.+S.+Lee&author=S.+LeGrand&author=T.+Luchko&author=R.+Luo&author=B.+Madej&author=K.+M.+Merz&author=F.+Paesani&author=D.+R.+Roe&author=A.+Roitberg&author=C.+Sagui&author=R.+Salomon-Ferrer&author=G.+Seabra&author=C.+L.+Simmerling&author=W.+Smith&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=R.+M.+Wolf&author=X.+Wu&author=P.+A.+Kollman&title=Amber+14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAmber%252014%26pub%3DUniversity%2520of%2520California%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, M. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Søndergaard, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span> <span> </span><span class="NLM_article-title">PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa Predictions</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1021/ct100578z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100578z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=525-537&author=M.+H.+M.+Olssonauthor=C.+R.+S%C3%B8ndergaardauthor=M.+Rostkowskiauthor=J.+H.+Jensen&title=PROPKA3%3A+Consistent+Treatment+of+Internal+and+Surface+Residues+in+Empirical+pKa+Predictions&doi=10.1021%2Fct100578z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa Predictions</span></div><div class="casAuthors">Olsson, Mats H. M.; Sondergaard, Chresten R.; Rostkowski, Michal; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">525-537</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have revised the rules and parameters for one of the most commonly used empirical pKa predictors, PROPKA, based on better phys. description of the desolvation and dielec. response for the protein.  The authors have introduced a new and consistent approach to interpolate the description between the previously distinct classifications into internal and surface residues, which otherwise is found to give rise to an erratic and discontinuous behavior.  Since the goal of this study is to lay out the framework and validate the concept, it focuses on Asp and Glu residues where the protein pKa values and structures are assumed to be more reliable.  The new and improved implementation is evaluated and discussed; it is found to agree better with expt. than the previous implementation (in parentheses): rmsd = 0.79 (0.91) for Asp and Glu, 0.75 (0.97) for Tyr, 0.65 (0.72) for Lys, and 1.00 (1.37) for His residues.  The most significant advance, however, is in reducing the no. of outliers and removing unreasonable sensitivity to small structural changes that arise from classifying residues as either internal or surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm6urTZNdjY7Vg90H21EOLACvtfcHk0ljoRuFRkD1NKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aqsA%253D%253D&md5=9b1666b1c56e1129789e62948eb4d001</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fct100578z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100578z%26sid%3Dliteratum%253Aachs%26aulast%3DOlsson%26aufirst%3DM.%2BH.%2BM.%26aulast%3DS%25C3%25B8ndergaard%26aufirst%3DC.%2BR.%26aulast%3DRostkowski%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DPROPKA3%253A%2520Consistent%2520Treatment%2520of%2520Internal%2520and%2520Surface%2520Residues%2520in%2520Empirical%2520pKa%2520Predictions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D525%26epage%3D537%26doi%3D10.1021%2Fct100578z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hostaš, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span> <span> </span><span class="NLM_article-title">Accurate DFT-D3 Calculations in a Small Basis Set</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3575</span>– <span class="NLM_lpage">3585</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.7b00365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.7b00365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFyqsbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=3575-3585&author=J.+Hosta%C5%A1author=J.+%C5%98ez%C3%A1%C4%8D&title=Accurate+DFT-D3+Calculations+in+a+Small+Basis+Set&doi=10.1021%2Facs.jctc.7b00365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate DFT-D3 Calculations in a Small Basis Set</span></div><div class="casAuthors">Hostas, Jiri; Rezac, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3575-3585</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Calcns. of interaction energies of noncovalent interactions in small basis sets are affected by the basis set superposition error and dispersion-cor. DFT-D methods and are thus usually parametrized only for triple-ζ and larger basis sets.  Nevertheless, some smaller basis sets could also perform well.  Among many combinations tested, we obtained excellent results with the DZVP-DFT basis and newly parametrized D3 dispersion correction.  The accuracy of interaction energies and geometries is close to significantly more expensive calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEin_d4_ETFbVg90H21EOLACvtfcHk0ljUw45XVWd9Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFyqsbjM&md5=29d0d41c5b76ade15db5e13f154baf5f</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.7b00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.7b00365%26sid%3Dliteratum%253Aachs%26aulast%3DHosta%25C5%25A1%26aufirst%3DJ.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26atitle%3DAccurate%2520DFT-D3%2520Calculations%2520in%2520a%2520Small%2520Basis%2520Set%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D3575%26epage%3D3585%26doi%3D10.1021%2Facs.jctc.7b00365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, J. J. P.</span></span> <span> </span><span class="NLM_article-title">Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations and Application to 70 Elements</span>. <i>J. Mol. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1173</span>– <span class="NLM_lpage">1213</span>, <span class="refDoi"> DOI: 10.1007/s00894-007-0233-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1007%2Fs00894-007-0233-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=17828561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlequr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=1173-1213&author=J.+J.+P.+Stewart&title=Optimization+of+Parameters+for+Semiempirical+Methods+V%3A+Modification+of+NDDO+Approximations+and+Application+to+70+Elements&doi=10.1007%2Fs00894-007-0233-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of parameters for semiempirical methods V: modification of NDDO approximations and application to 70 elements</span></div><div class="casAuthors">Stewart, James J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1173-1213</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Several modifications that have been made to the NDDO core-core interaction term and to the method of parameter optimization are described.  These changes have resulted in a more complete parameter optimization, called PM6, which has, in turn, allowed 70 elements to be parameterized.  The av. unsigned error (AUE) between calcd. and ref. heats of formation for 4,492 species was 8.0 kcal mol-1.  For the subset of 1,373 compds. involving only the elements H, C, N, O, F, P, S, Cl, and Br, the PM6 AUE was 4.4 kcal mol-1.  The equivalent AUE for other methods were: RM1: 5.0, B3LYP 6-31G*: 5.2, PM5: 5.7, PM3: 6.3, HF 6-31G*: 7.4, and AM1: 10.0 kcal mol-1.  Several long-standing faults in AM1 and PM3 have been cor. and significant improvements have been made in the prediction of geometries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl-g7DsFjK8LVg90H21EOLACvtfcHk0ljUw45XVWd9Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlequr7N&md5=7d22928c8e3423f3ca8f2e7c77e6e6c9</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1007%2Fs00894-007-0233-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-007-0233-4%26sid%3Dliteratum%253Aachs%26aulast%3DStewart%26aufirst%3DJ.%2BJ.%2BP.%26atitle%3DOptimization%2520of%2520Parameters%2520for%2520Semiempirical%2520Methods%2520V%253A%2520Modification%2520of%2520NDDO%2520Approximations%2520and%2520Application%2520to%252070%2520Elements%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2007%26volume%3D13%26spage%3D1173%26epage%3D1213%26doi%3D10.1007%2Fs00894-007-0233-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobza, P.</span></span> <span> </span><span class="NLM_article-title">Advanced Corrections of Hydrogen Bonding and Dispersion for Semiempirical Quantum Mechanical Methods</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1021/ct200751e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200751e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Gls77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=141-151&author=J.+%C5%98ez%C3%A1%C4%8Dauthor=P.+Hobza&title=Advanced+Corrections+of+Hydrogen+Bonding+and+Dispersion+for+Semiempirical+Quantum+Mechanical+Methods&doi=10.1021%2Fct200751e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Advanced Corrections of Hydrogen Bonding and Dispersion for Semiempirical Quantum Mechanical Methods</span></div><div class="casAuthors">Rezac, Jan; Hobza, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">141-151</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Semiempirical quantum mech. methods with corrections for noncovalent interactions, namely dispersion and hydrogen bonds, reach an accuracy comparable to much more expensive methods while being applicable to very large systems (up to 10 000 atoms).  These corrections have been successfully applied in computer-assisted drug design, where they significantly improve the correlation with the exptl. data.  Despite these successes, there are still several unresolved issues that limit the applicability of these methods.  We introduce a new generation of both hydrogen-bonding and dispersion corrections that address these problems, make the method more robust, and improve its accuracy.  The hydrogen-bonding correction has been completely redesigned and for the first time can be used for geometry optimization and mol.-dynamics simulations without any limitations, as it and its derivs. have a smooth potential energy surface.  The form of this correction is simpler than its predecessors, while the accuracy has been improved.  For the dispersion correction, we adopt the latest developments in DFT-D, using the D3 formalism by Grimme.  The new corrections have been parametrized on a large set of benchmark data including nonequil. geometries, the S66x8 data set.  As a result, the newly developed D3H4 correction can accurately describe a wider range of interactions.  We have parametrized this correction for the PM6, RM1, OM3, PM3, AM1, and SCC-DFTB methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhPaWNi5-P4bVg90H21EOLACvtfcHk0ljUw45XVWd9Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Gls77L&md5=1c15239efc1338484dcd0b9d6e70518d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fct200751e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200751e%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26aulast%3DHobza%26aufirst%3DP.%26atitle%3DAdvanced%2520Corrections%2520of%2520Hydrogen%2520Bonding%2520and%2520Dispersion%2520for%2520Semiempirical%2520Quantum%2520Mechanical%2520Methods%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D141%26epage%3D151%26doi%3D10.1021%2Fct200751e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüürmann, G.</span></span> <span> </span><span class="NLM_article-title">COSMO: A New Approach to Dielectric Screening in Solvents with Explicit Expressions for the Screening Energy and Its Gradient</span>. <i>J. Chem. Soc., Perkin Trans. 2</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_fpage">799</span>– <span class="NLM_lpage">805</span>, <span class="refDoi"> DOI: 10.1039/p29930000799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1039%2FP29930000799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3sXkvVSku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&pages=799-805&author=A.+Klamtauthor=G.+Sch%C3%BC%C3%BCrmann&title=COSMO%3A+A+New+Approach+to+Dielectric+Screening+in+Solvents+with+Explicit+Expressions+for+the+Screening+Energy+and+Its+Gradient&doi=10.1039%2Fp29930000799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">COSMO: a new approach to dielectric screening in solvents with explicit expressions for the screening energy and its gradient</span></div><div class="casAuthors">Klamt, A.; Schueuermann, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Chemical Society, Perkin Transactions 2:  Physical Organic Chemistry (1972-1999)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume"></span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">799-805</span>CODEN:
                <span class="NLM_cas:coden">JCPKBH</span>;
        ISSN:<span class="NLM_cas:issn">0300-9580</span>.
    </div><div class="casAbstract">Starting from the screening in conductors, an algorithm for the accurate calcn. of dielec. screening effects in solvents is presented, which leads to rather simple explicit expressions for the screening energy and its analytic gradient with respect to the solute coordinates.  Thus geometry optimization of a solute within a realistic dielec. continuum model becomes practicable for the first time.  The algorithm is suited for mol. mechanics as well as for any MO algorithm.  The implementation into MOPAC and some example applications are reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUvfeTHvmEb7Vg90H21EOLACvtfcHk0lihNFOvb97DKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXkvVSku7o%253D&md5=a8e9e41514d60ca7f11212cf5c281424</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1039%2FP29930000799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FP29930000799%26sid%3Dliteratum%253Aachs%26aulast%3DKlamt%26aufirst%3DA.%26aulast%3DSch%25C3%25BC%25C3%25BCrmann%26aufirst%3DG.%26atitle%3DCOSMO%253A%2520A%2520New%2520Approach%2520to%2520Dielectric%2520Screening%2520in%2520Solvents%2520with%2520Explicit%2520Expressions%2520for%2520the%2520Screening%2520Energy%2520and%2520Its%2520Gradient%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25202%26date%3D1993%26spage%3D799%26epage%3D805%26doi%3D10.1039%2Fp29930000799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span> <span> </span><span class="NLM_article-title">Cuby: An Integrative Framework for Computational Chemistry</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1230</span>– <span class="NLM_lpage">1237</span>, <span class="refDoi"> DOI: 10.1002/jcc.24312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fjcc.24312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=26841135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVShs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=1230-1237&author=J.+%C5%98ez%C3%A1%C4%8D&title=Cuby%3A+An+Integrative+Framework+for+Computational+Chemistry&doi=10.1002%2Fjcc.24312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Cuby: An integrative framework for computational chemistry</span></div><div class="casAuthors">Rezac, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1230-1237</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Cuby is a computational chem. framework written in the Ruby programming language.  It provides unified access to a wide range of computational methods by interfacing external software and it implements various protocols that operate on their results.  Using structured input files, elementary calcns. can be combined into complex workflows.  For users, Cuby provides a unified and userfriendly way to automate their work, seamlessly integrating calcns. carried out in different computational chem. programs.  For example, the QM/MM module allows combining methods across the interfaced programs and the builtin mol. dynamics engine makes it possible to run a simulation on the resulting potential.  For programmers, it provides high-level, object-oriented environment that allows rapid development and testing of new methods and computational protocols.  The Cuby framework is available for download at http://cuby4.mol.cz.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCiLHT2DVtXrVg90H21EOLACvtfcHk0lihNFOvb97DKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVShs74%253D&md5=6ec4c9c15ca341ce5fea73517623680b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fjcc.24312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.24312%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26atitle%3DCuby%253A%2520An%2520Integrative%2520Framework%2520for%2520Computational%2520Chemistry%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2016%26volume%3D37%26spage%3D1230%26epage%3D1237%26doi%3D10.1002%2Fjcc.24312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahlrichs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bär, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kölmel, C.</span></span> <span> </span><span class="NLM_article-title">Electronic Structure Calculations on Workstation Computers: The Program System Turbomole</span>. <i>Chem. Phys. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1016/0009-2614(89)85118-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1016%2F0009-2614%2889%2985118-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADyaK3cXkt1yrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=1989&pages=165-169&author=R.+Ahlrichsauthor=M.+B%C3%A4rauthor=M.+H%C3%A4serauthor=H.+Hornauthor=C.+K%C3%B6lmel&title=Electronic+Structure+Calculations+on+Workstation+Computers%3A+The+Program+System+Turbomole&doi=10.1016%2F0009-2614%2889%2985118-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Electronic structure calculations on workstation computers:  the program system TURBOMOLE</span></div><div class="casAuthors">Ahlrichs, Reinhart; Baer, Michael; Haeser, Marco; Horn, Hans; Koelmel, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics Letters</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">165-9</span>CODEN:
                <span class="NLM_cas:coden">CHPLBC</span>;
        ISSN:<span class="NLM_cas:issn">0009-2614</span>.
    </div><div class="casAbstract">The basic structure of the program system TURBOMOLE for SCF - including first and second anal. derivs. with respect to nuclear coordinates - and MP2 calcns. is briefly described.  The program takes full advantage of all discrete point group symmetries and has only modest - and (partially) adjustable - I/O and background storage requirements.  The performance of TURBOMOLE is documented for demonstrative applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPKLD1c-7-ZbVg90H21EOLACvtfcHk0lihNFOvb97DKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXkt1yrtg%253D%253D&md5=b6aa32e6226a8e11b511e4d09cc60dc8</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2F0009-2614%2889%2985118-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0009-2614%252889%252985118-8%26sid%3Dliteratum%253Aachs%26aulast%3DAhlrichs%26aufirst%3DR.%26aulast%3DB%25C3%25A4r%26aufirst%3DM.%26aulast%3DH%25C3%25A4ser%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DH.%26aulast%3DK%25C3%25B6lmel%26aufirst%3DC.%26atitle%3DElectronic%2520Structure%2520Calculations%2520on%2520Workstation%2520Computers%253A%2520The%2520Program%2520System%2520Turbomole%26jtitle%3DChem.%2520Phys.%2520Lett.%26date%3D1989%26volume%3D162%26spage%3D165%26epage%3D169%26doi%3D10.1016%2F0009-2614%2889%2985118-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, J. J. P.</span></span> <span> </span><span class="NLM_article-title">Optimization of Parameters for Semiempirical Methods IV: Extension of MNDO, AM1, and PM3 to More Main Group Elements</span>. <i>J. Mol. Model.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1007/s00894-004-0183-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1007%2Fs00894-004-0183-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=14997367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktV2msL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=155-164&author=J.+J.+P.+Stewart&title=Optimization+of+Parameters+for+Semiempirical+Methods+IV%3A+Extension+of+MNDO%2C+AM1%2C+and+PM3+to+More+Main+Group+Elements&doi=10.1007%2Fs00894-004-0183-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of parameters for semiempirical methods. IV: extension of MNDO, AM1, and PM3 to more main group elements</span></div><div class="casAuthors">Stewart, James J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">155-164</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">The NDDO semiempirical methods MNDO, AM1, and PM3 were extended to all the remaining non-radioactive elements of the main group, excluding the noble gases.  Most of the new elements are of Groups I and II.  44 Sets of parameters are presented for the following methods and elements.  MNDO: Na, Mg, K, Ca, Ga, As, Se, Rb, Sr, In, Sb, Te, Cs, Ba, Tl, and Bi; AM1: Li, Be, Na, Mg, K, Ca, Ga, As, Se, Rb, Sr, In, Sn, Sb, Te, Cs, Ba, Tl, Pb, and Bi; PM3: B, Na, K, Ca, Rb, Sr, Cs, and Ba.  Av. errors are presented for heats of formation, mol. geometries, etc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquoCOa8eJqQ7Vg90H21EOLACvtfcHk0lihNFOvb97DKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktV2msL8%253D&md5=f06f5d6054d81d44cf41c0b919200a45</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs00894-004-0183-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-004-0183-z%26sid%3Dliteratum%253Aachs%26aulast%3DStewart%26aufirst%3DJ.%2BJ.%2BP.%26atitle%3DOptimization%2520of%2520Parameters%2520for%2520Semiempirical%2520Methods%2520IV%253A%2520Extension%2520of%2520MNDO%252C%2520AM1%252C%2520and%2520PM3%2520to%2520More%2520Main%2520Group%2520Elements%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2004%26volume%3D10%26spage%3D155%26epage%3D164%26doi%3D10.1007%2Fs00894-004-0183-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lepšík, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolář, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobza, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanfrlík, J.</span></span> <span> </span><span class="NLM_article-title">The Semiempirical Quantum Mechanical Scoring Function for in Silico Drug Design</span>. <i>Chempluschem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">921</span>– <span class="NLM_lpage">931</span>, <span class="refDoi"> DOI: 10.1002/cplu.201300199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=10.1002%2Fcplu.201300199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=31986741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWrsLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=921-931&author=M.+Lep%C5%A1%C3%ADkauthor=J.+%C5%98ez%C3%A1%C4%8Dauthor=M.+Kol%C3%A1%C5%99author=A.+Pecinaauthor=P.+Hobzaauthor=J.+Fanfrl%C3%ADk&title=The+Semiempirical+Quantum+Mechanical+Scoring+Function+for+in+Silico+Drug+Design&doi=10.1002%2Fcplu.201300199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The Semiempirical Quantum Mechanical Scoring Function for In Silico Drug Design</span></div><div class="casAuthors">Lepsik, Martin; Rezac, Jan; Kolar, Michal; Pecina, Adam; Hobza, Pavel; Fanfrlik, Jindrich</div><div class="citationInfo"><span class="NLM_cas:title">ChemPlusChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">921-931</span>CODEN:
                <span class="NLM_cas:coden">CHEMM5</span>;
        ISSN:<span class="NLM_cas:issn">2192-6506</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  This Minireview discusses the latest developments in modern quantum mechanics (QM)-based computer-aided drug design, esp. using semiempirical QM (SQM) methods.  It first tackles biochem. and biophys. quantities and the approaches to their measurements.  Protein-ligand affinities are detd. mostly by noncovalent interactions.  The text thus illustrates how these can be accurately treated with SQM methods.  Next, a construction of a modern SQM-based scoring function is presented and its applications listed.  In summary, SQM-based scoring is a promising modern efficient strategy to be exploited in computer-aided drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1tavGvbuP_rVg90H21EOLACvtfcHk0lg6sDwME0pQ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWrsLrM&md5=9f9a2633e55a52b720a650dfca71c9ff</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fcplu.201300199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcplu.201300199%26sid%3Dliteratum%253Aachs%26aulast%3DLep%25C5%25A1%25C3%25ADk%26aufirst%3DM.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26aulast%3DKol%25C3%25A1%25C5%2599%26aufirst%3DM.%26aulast%3DPecina%26aufirst%3DA.%26aulast%3DHobza%26aufirst%3DP.%26aulast%3DFanfrl%25C3%25ADk%26aufirst%3DJ.%26atitle%3DThe%2520Semiempirical%2520Quantum%2520Mechanical%2520Scoring%2520Function%2520for%2520in%2520Silico%2520Drug%2520Design%26jtitle%3DChempluschem%26date%3D2013%26volume%3D78%26spage%3D921%26epage%3D931%26doi%3D10.1002%2Fcplu.201300199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fanfrlík, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronowska, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáč, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Přenosil, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konvalinka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobza, P.</span></span> <span> </span><span class="NLM_article-title">A Reliable Docking/Scoring Scheme Based on the Semiempirical Quantum Mechanical PM6-DH2 Method Accurately Covering Dispersion and H-Bonding: HIV-1 Protease with 22 Ligands</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">12666</span>– <span class="NLM_lpage">12678</span>, <span class="refDoi"> DOI: 10.1021/jp1032965</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp1032965" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFGhsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2010&pages=12666-12678&author=J.+Fanfrl%C3%ADkauthor=A.+K.+Bronowskaauthor=J.+%C5%98ez%C3%A1%C4%8Dauthor=O.+P%C5%99enosilauthor=J.+Konvalinkaauthor=P.+Hobza&title=A+Reliable+Docking%2FScoring+Scheme+Based+on+the+Semiempirical+Quantum+Mechanical+PM6-DH2+Method+Accurately+Covering+Dispersion+and+H-Bonding%3A+HIV-1+Protease+with+22+Ligands&doi=10.1021%2Fjp1032965"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">A Reliable Docking/Scoring Scheme Based on the Semiempirical Quantum Mechanical PM6-DH2 Method Accurately Covering Dispersion and H-Bonding: HIV-1 Protease with 22 Ligands</span></div><div class="casAuthors">Fanfrlik, Jindrich; Bronowska, Agnieszka K.; Rezac, Jan; Prenosil, Ondrej; Konvalinka, Jan; Hobza, Pavel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">12666-12678</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, we introduce a fast and reliable rescoring scheme for docked complexes based on a semiempirical quantum mech. PM6-DH2 method.  The method utilizes a PM6-based Hamiltonian with corrections for dispersion energy and hydrogen bonds.  The total score is constructed as the sum of the PM6-DH2 interaction enthalpy, the empirical force field (AMBER) interaction entropy, and the sum of the deformation (PM6-DH2, SMD) and the desolvation (SMD) energies of the ligand.  The main advantage of the procedure is the fact that we do not add any empirical parameter for either an individual component of the total score or an individual protein-ligand complex.  This rescoring method is applied to a very challenging system, namely, the HIV-1 protease with a set of ligands.  As opposed to the conventional DOCK procedure, the PM6-DH2 rescoring based on all of the terms distinguishes between binders and nonbinders and provides a reliable correlation of the theor. and exptl. binding free energies.  Such a dramatic improvement, resulting from the PM6-DH2 rescoring of all the complexes, provides a valuable yet inexpensive tool for rational drug discovery and de novo ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3uSeswff_QbVg90H21EOLACvtfcHk0lg6sDwME0pQ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGhsb%252FF&md5=8bc9a22daa9392461db7469d90653778</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjp1032965&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp1032965%26sid%3Dliteratum%253Aachs%26aulast%3DFanfrl%25C3%25ADk%26aufirst%3DJ.%26aulast%3DBronowska%26aufirst%3DA.%2BK.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258D%26aufirst%3DJ.%26aulast%3DP%25C5%2599enosil%26aufirst%3DO.%26aulast%3DKonvalinka%26aufirst%3DJ.%26aulast%3DHobza%26aufirst%3DP.%26atitle%3DA%2520Reliable%2520Docking%252FScoring%2520Scheme%2520Based%2520on%2520the%2520Semiempirical%2520Quantum%2520Mechanical%2520PM6-DH2%2520Method%2520Accurately%2520Covering%2520Dispersion%2520and%2520H-Bonding%253A%2520HIV-1%2520Protease%2520with%252022%2520Ligands%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26spage%3D12666%26epage%3D12678%26doi%3D10.1021%2Fjp1032965" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marenich, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truhlar, D. G.</span></span> <span> </span><span class="NLM_article-title">Universal Solvation Model Based on Solute Electron Density and on a Continuum Model of the Solvent Defined by the Bulk Dielectric Constant and Atomic Surface Tensions</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">6378</span>– <span class="NLM_lpage">6396</span>, <span class="refDoi"> DOI: 10.1021/jp810292n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp810292n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksV2is74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=6378-6396&author=A.+V.+Marenichauthor=C.+J.+Cramerauthor=D.+G.+Truhlar&title=Universal+Solvation+Model+Based+on+Solute+Electron+Density+and+on+a+Continuum+Model+of+the+Solvent+Defined+by+the+Bulk+Dielectric+Constant+and+Atomic+Surface+Tensions&doi=10.1021%2Fjp810292n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Universal Solvation Model Based on Solute Electron Density and on a Continuum Model of the Solvent Defined by the Bulk Dielectric Constant and Atomic Surface Tensions</span></div><div class="casAuthors">Marenich, Aleksandr V.; Cramer, Christopher J.; Truhlar, Donald G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6378-6396</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a new continuum solvation model based on the quantum mech. charge d. of a solute mol. interacting with a continuum description of the solvent.  The model is called SMD, where the "D" stands for "d." to denote that the full solute electron d. is used without defining partial at. charges.  "Continuum" denotes that the solvent is not represented explicitly but rather as a dielec. medium with surface tension at the solute-solvent boundary.  SMD is a universal solvation model, where "universal" denotes its applicability to any charged or uncharged solute in any solvent or liq. medium for which a few key descriptors are known (in particular, dielec. const., refractive index, bulk surface tension, and acidity and basicity parameters).  The model separates the observable solvation free energy into two main components.  The first component is the bulk electrostatic contribution arising from a self-consistent reaction field treatment that involves the soln. of the nonhomogeneous Poisson equation for electrostatics in terms of the integral-equation-formalism polarizable continuum model (IEF-PCM).  The cavities for the bulk electrostatic calcn. are defined by superpositions of nuclear-centered spheres.  The second component is called the cavity-dispersion-solvent-structure term and is the contribution arising from short-range interactions between the solute and solvent mols. in the first solvation shell.  This contribution is a sum of terms that are proportional (with geometry-dependent proportionality consts. called at. surface tensions) to the solvent-accessible surface areas of the individual atoms of the solute.  The SMD model has been parametrized with a training set of 2821 solvation data including 112 aq. ionic solvation free energies, 220 solvation free energies for 166 ions in acetonitrile, methanol, and DMSO, 2346 solvation free energies for 318 neutral solutes in 91 solvents (90 nonaq. org. solvents and water), and 143 transfer free energies for 93 neutral solutes between water and 15 org. solvents.  The elements present in the solutes are H, C, N, O, F, Si, P, S, Cl, and Br.  The SMD model employs a single set of parameters (intrinsic at. Coulomb radii and at. surface tension coeffs.) optimized over six electronic structure methods: M05-2X/MIDI!6D, M05-2X/6-31G*, M05-2X/6-31+G**, M05-2X/cc-pVTZ, B3LYP/6-31G*, and HF/6-31G*.  Although the SMD model has been parametrized using the IEF-PCM protocol for bulk electrostatics, it may also be employed with other algorithms for solving the nonhomogeneous Poisson equation for continuum solvation calcns. in which the solute is represented by its electron d. in real space.  This includes, for example, the conductor-like screening algorithm.  With the 6-31G* basis set, the SMD model achieves mean unsigned errors of 0.6-1.0 kcal/mol in the solvation free energies of tested neutrals and mean unsigned errors of 4 kcal/mol on av. for ions with either Gaussian03 or GAMESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQrvm7K8c8dLVg90H21EOLACvtfcHk0lg6sDwME0pQ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksV2is74%253D&md5=54931a64c70d28445ee53876a8b1a4b9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjp810292n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp810292n%26sid%3Dliteratum%253Aachs%26aulast%3DMarenich%26aufirst%3DA.%2BV.%26aulast%3DCramer%26aufirst%3DC.%2BJ.%26aulast%3DTruhlar%26aufirst%3DD.%2BG.%26atitle%3DUniversal%2520Solvation%2520Model%2520Based%2520on%2520Solute%2520Electron%2520Density%2520and%2520on%2520a%2520Continuum%2520Model%2520of%2520the%2520Solvent%2520Defined%2520by%2520the%2520Bulk%2520Dielectric%2520Constant%2520and%2520Atomic%2520Surface%2520Tensions%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2009%26volume%3D113%26spage%3D6378%26epage%3D6396%26doi%3D10.1021%2Fjp810292n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Petersson, G. A.</span></span> <i>Gaussian 09</i>, Revision D. 01, 2009; <span class="NLM_publisher-name">Gaussian Inc.</span>: <span class="NLM_publisher-loc">Wallingford CT</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYB','PDB','1LYB'); return false;">PDB: 1LYB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LYA','PDB','1LYA'); return false;">PDB: 1LYA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QCB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QCB','PDB','6QCB'); return false;">PDB: 6QCB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBG','PDB','6QBG'); return false;">PDB: 6QBG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBH','PDB','6QBH'); return false;">PDB: 6QBH</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i107"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01351">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_04738"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01351?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01351</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Physicochemical parameters of pepstatin A (<b>1</b>) and the most potent macrocyclic inhibitors with subnanomolar and single-digit nanomolar inhibition of CatD; cytotoxicity analysis of model macrocyclic inhibitors and pepstatin A (<b>1</b>); correlation of computed scores plotted against experimental binding free energy for selected macrocyclic inhibitors; inhibition kinetics for the macrocyclic inhibitor 9 and human CatD; list of atomic contacts between macrocyclic inhibitors and CatD; X-ray data collection and refinement statistics; and analytical RP-HPLC traces for the key compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_002.csv">CSV</a>)</p></li><li><p class="inline">Protien data bank for CatD-<b>9</b> complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Protien data bank for CatD-<b>14</b> complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Protien data bank for CatD-<b>33</b> complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_005.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_001.pdf">jm9b01351_si_001.pdf (877.3 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_002.csv">jm9b01351_si_002.csv (2.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_003.pdb">jm9b01351_si_003.pdb (304.7 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_004.pdb">jm9b01351_si_004.pdb (288.4 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01351/suppl_file/jm9b01351_si_005.pdb">jm9b01351_si_005.pdb (280.57 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structures presented in this article have been deposited in the Protein Data Bank under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QCB">6QCB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBG">6QBG</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QBH">6QBH</a> for the CatD-<b>9</b>, CatD-<b>14</b>, and CatD-<b>33</b> complexes, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01351&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01351%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-4%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01351" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bb7f7dfb3ca0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
