
<html lang="en"     class="pb-page"  data-request-id="1b83d0a5-24bf-43eb-849a-dcfe46ed3f19"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.1c00507;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a First-in-Class Inhibitor of the PRMT5–Substrate Adaptor Interaction" /></meta><meta name="dc.Creator" content="David C.  McKinney" /></meta><meta name="dc.Creator" content="Brian J.  McMillan" /></meta><meta name="dc.Creator" content="Matthew J.  Ranaghan" /></meta><meta name="dc.Creator" content="Jamie A.  Moroco" /></meta><meta name="dc.Creator" content="Merissa  Brousseau" /></meta><meta name="dc.Creator" content="Zachary  Mullin-Bernstein" /></meta><meta name="dc.Creator" content="Meghan  O’Keefe" /></meta><meta name="dc.Creator" content="Patrick  McCarren" /></meta><meta name="dc.Creator" content="Michael F.  Mesleh" /></meta><meta name="dc.Creator" content="Kathleen M.  Mulvaney" /></meta><meta name="dc.Creator" content="Foxy  Robinson" /></meta><meta name="dc.Creator" content="Ritu  Singh" /></meta><meta name="dc.Creator" content="Besnik  Bajrami" /></meta><meta name="dc.Creator" content="Florence F.  Wagner" /></meta><meta name="dc.Creator" content="Robert  Hilgraf" /></meta><meta name="dc.Creator" content="Martin J.  Drysdale" /></meta><meta name="dc.Creator" content="Arthur J.  Campbell" /></meta><meta name="dc.Creator" content="Adam  Skepner" /></meta><meta name="dc.Creator" content="David E.  Timm" /></meta><meta name="dc.Creator" content="Dale  Porter" /></meta><meta name="dc.Creator" content="Virendar K.  Kaushik" /></meta><meta name="dc.Creator" content="William R.  Sellers" /></meta><meta name="dc.Creator" content="Alessandra  Ianari" /></meta><meta name="dc.Description" content="PRMT5 and its substrate adaptor proteins (SAPs), pICln and Riok1, are synthetic lethal dependencies in MTAP-deleted cancer cells. SAPs share a conserved PRMT5 binding motif (PBM) which mediates bin..." /></meta><meta name="Description" content="PRMT5 and its substrate adaptor proteins (SAPs), pICln and Riok1, are synthetic lethal dependencies in MTAP-deleted cancer cells. SAPs share a conserved PRMT5 binding motif (PBM) which mediates bin..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 3, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00507" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00507" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00507" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00507" /></link>
        
    
    

<title>Discovery of a First-in-Class Inhibitor of the PRMT5–Substrate Adaptor Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00507" /></meta><meta property="og:title" content="Discovery of a First-in-Class Inhibitor of the PRMT5–Substrate Adaptor Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0018.jpeg" /></meta><meta property="og:description" content="PRMT5 and its substrate adaptor proteins (SAPs), pICln and Riok1, are synthetic lethal dependencies in MTAP-deleted cancer cells. SAPs share a conserved PRMT5 binding motif (PBM) which mediates binding to a surface of PRMT5 distal to the catalytic site. This interaction is required for methylation of several PRMT5 substrates, including histone and spliceosome complexes. We screened for small molecule inhibitors of the PRMT5–PBM interaction and validated a compound series which binds to the PRMT5–PBM interface and directly inhibits binding of SAPs. Mode of action studies revealed the formation of a covalent bond between a halogenated pyridazinone group and cysteine 278 of PRMT5. Optimization of the starting hit produced a lead compound, BRD0639, which engages the target in cells, disrupts PRMT5–RIOK1 complexes, and reduces substrate methylation. BRD0639 is a first-in-class PBM-competitive inhibitor that can support studies of PBM-dependent PRMT5 activities and the development of novel PRMT5 inhibitors that selectively target these functions." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00507"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00507">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00507&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00507&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00507&amp;href=/doi/10.1021/acs.jmedchem.1c00507" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00682" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a First-in-Class Inhibitor of the PRMT5–Substrate Adaptor Interaction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">David C. McKinney</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David C. McKinney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+C.++McKinney">David C. McKinney</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1787-1841" title="Orcid link">https://orcid.org/0000-0003-1787-1841</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Brian J. McMillan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brian J. McMillan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brian+J.++McMillan">Brian J. McMillan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew J. Ranaghan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew J. Ranaghan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+J.++Ranaghan">Matthew J. Ranaghan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3572-5127" title="Orcid link">https://orcid.org/0000-0003-3572-5127</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jamie A. Moroco</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jamie A. Moroco</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jamie+A.++Moroco">Jamie A. Moroco</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-8250-5923" title="Orcid link">https://orcid.org/0000-0001-8250-5923</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Merissa Brousseau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Merissa Brousseau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Merissa++Brousseau">Merissa Brousseau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zachary Mullin-Bernstein</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zachary Mullin-Bernstein</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Program, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zachary++Mullin-Bernstein">Zachary Mullin-Bernstein</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Meghan O’Keefe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Meghan O’Keefe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Meghan++O%E2%80%99Keefe">Meghan O’Keefe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Patrick McCarren</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Patrick McCarren</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Patrick++McCarren">Patrick McCarren</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael F. Mesleh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael F. Mesleh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+F.++Mesleh">Michael F. Mesleh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kathleen M. Mulvaney</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kathleen M. Mulvaney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Program, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kathleen+M.++Mulvaney">Kathleen M. Mulvaney</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Foxy Robinson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Foxy Robinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Program, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Foxy++Robinson">Foxy Robinson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ritu Singh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ritu Singh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ritu++Singh">Ritu Singh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Besnik Bajrami</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Besnik Bajrami</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Besnik++Bajrami">Besnik Bajrami</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Florence F. Wagner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Florence F. Wagner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Florence+F.++Wagner">Florence F. Wagner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert Hilgraf</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert Hilgraf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert++Hilgraf">Robert Hilgraf</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin J. Drysdale</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin J. Drysdale</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin+J.++Drysdale">Martin J. Drysdale</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Arthur J. Campbell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Arthur J. Campbell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Arthur+J.++Campbell">Arthur J. Campbell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adam Skepner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adam Skepner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adam++Skepner">Adam Skepner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David E. Timm</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David E. Timm</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, University of Utah, 1390 Presidents Circle, Salt Lake City, Utah 84112, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+E.++Timm">David E. Timm</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dale Porter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dale Porter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Program, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dale++Porter">Dale Porter</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Virendar K. Kaushik</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Virendar K. Kaushik</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for the Development of Therapeutics, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Virendar+K.++Kaushik">Virendar K. Kaushik</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William R. Sellers</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William R. Sellers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Program, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div><div class="loa-info-affiliations-info">Department of Medical Oncology, Dana-Farber Cancer Institute, Department of Medicine, Harvard Medical School, 44 Binney Street, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#592e2a3c35353c2b2a193b2b36383d30372a2d302d2c2d3c77362b3e"><span class="__cf_email__" data-cfemail="e19692848d8d849392a183938e8085888f92958895949584cf8e9386">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+R.++Sellers">William R. Sellers</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Alessandra Ianari</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alessandra Ianari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Program, The Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ceafa7afa0afbca78eacbca1afaaa7a0bdbaa7babbbaabe0a1bca9"><span class="__cf_email__" data-cfemail="fc9d959d929d8e95bc9e8e939d9895928f889588898899d2938e9b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alessandra++Ianari">Alessandra Ianari</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-5850-0476" title="Orcid link">https://orcid.org/0000-0002-5850-0476</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00507&amp;href=/doi/10.1021%2Facs.jmedchem.1c00507" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 3, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 March 2021</li><li><span class="item_label"><b>Published</b> online</span>3 August 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00507" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00507</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDavid%2BC.%2BMcKinney%252C%2BBrian%2BJ.%2BMcMillan%252C%2BMatthew%2BJ.%2BRanaghan%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00507%26title%3DDiscovery%2Bof%2Ba%2BFirst-in-Class%2BInhibitor%2Bof%2Bthe%2BPRMT5%25E2%2580%2593Substrate%2BAdaptor%2BInteraction%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D21%26publicationDate%3DAugust%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00507"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">-</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00507" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a First-in-Class Inhibitor of the PRMT5–Substrate Adaptor Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;C. McKinney&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;J. McMillan&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;J. Ranaghan&quot;},{&quot;first_name&quot;:&quot;Jamie&quot;,&quot;last_name&quot;:&quot;A. Moroco&quot;},{&quot;first_name&quot;:&quot;Merissa&quot;,&quot;last_name&quot;:&quot;Brousseau&quot;},{&quot;first_name&quot;:&quot;Zachary&quot;,&quot;last_name&quot;:&quot;Mullin-Bernstein&quot;},{&quot;first_name&quot;:&quot;Meghan&quot;,&quot;last_name&quot;:&quot;O’Keefe&quot;},{&quot;first_name&quot;:&quot;Patrick&quot;,&quot;last_name&quot;:&quot;McCarren&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;F. Mesleh&quot;},{&quot;first_name&quot;:&quot;Kathleen&quot;,&quot;last_name&quot;:&quot;M. Mulvaney&quot;},{&quot;first_name&quot;:&quot;Foxy&quot;,&quot;last_name&quot;:&quot;Robinson&quot;},{&quot;first_name&quot;:&quot;Ritu&quot;,&quot;last_name&quot;:&quot;Singh&quot;},{&quot;first_name&quot;:&quot;Besnik&quot;,&quot;last_name&quot;:&quot;Bajrami&quot;},{&quot;first_name&quot;:&quot;Florence&quot;,&quot;last_name&quot;:&quot;F. Wagner&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;Hilgraf&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;J. Drysdale&quot;},{&quot;first_name&quot;:&quot;Arthur&quot;,&quot;last_name&quot;:&quot;J. Campbell&quot;},{&quot;first_name&quot;:&quot;Adam&quot;,&quot;last_name&quot;:&quot;Skepner&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;E. Timm&quot;},{&quot;first_name&quot;:&quot;Dale&quot;,&quot;last_name&quot;:&quot;Porter&quot;},{&quot;first_name&quot;:&quot;Virendar&quot;,&quot;last_name&quot;:&quot;K. Kaushik&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;R. Sellers&quot;},{&quot;first_name&quot;:&quot;Alessandra&quot;,&quot;last_name&quot;:&quot;Ianari&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00507&quot;},&quot;abstract&quot;:&quot;PRMT5 and its substrate adaptor proteins (SAPs), pICln and Riok1, are synthetic lethal dependencies in MTAP-deleted cancer cells. SAPs share a conserved PRMT5 binding motif (PBM) which mediates binding to a surface of PRMT5 distal to the catalytic site. This interaction is required for methylation of several PRMT5 substrates, including histone and spliceosome complexes. We screened for small molecule inhibitors of the PRMT5–PBM interaction and validated a compound series which binds to the PRMT5–PBM interface and directly inhibits binding of SAPs. Mode of action studies revealed the formation of a covalent bond between a halogenated pyridazinone group and cysteine 278 of PRMT5. Optimization of the starting hit produced a lead compound, BRD0639, which engages the target in cells, disrupts PRMT5–RIOK1 complexes, and reduces substrate methylation. BRD0639 is a first-in-class PBM-competitive inhibitor that can support studies of PBM-dependent PRMT5 activities and the development of novel PRMT5 inhibitors that sel&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00507&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00507" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00507&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00507" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00507&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00507" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00507&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00507&amp;href=/doi/10.1021/acs.jmedchem.1c00507" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00507" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00507" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00507%26sid%3Dliteratum%253Aachs%26pmid%3D34342224%26genre%3Darticle%26aulast%3DMcKinney%26date%3D2021%26atitle%3DDiscovery%2Bof%2Ba%2BFirst-in-Class%2BInhibitor%2Bof%2Bthe%2BPRMT5%25E2%2580%2593Substrate%2BAdaptor%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292161" title="Modification">Modification</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291452" title="Reactivity">Reactivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">PRMT5 and its substrate adaptor proteins (SAPs), pICln and Riok1, are synthetic lethal dependencies in MTAP-deleted cancer cells. SAPs share a conserved PRMT5 binding motif (PBM) which mediates binding to a surface of PRMT5 distal to the catalytic site. This interaction is required for methylation of several PRMT5 substrates, including histone and spliceosome complexes. We screened for small molecule inhibitors of the PRMT5–PBM interaction and validated a compound series which binds to the PRMT5–PBM interface and directly inhibits binding of SAPs. Mode of action studies revealed the formation of a covalent bond between a halogenated pyridazinone group and cysteine 278 of PRMT5. Optimization of the starting hit produced a lead compound, BRD0639, which engages the target in cells, disrupts PRMT5–RIOK1 complexes, and reduces substrate methylation. BRD0639 is a first-in-class PBM-competitive inhibitor that can support studies of PBM-dependent PRMT5 activities and the development of novel PRMT5 inhibitors that selectively target these functions.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span> Special Issue</h4><p class="last">Published as part of the <i>Journal of Medicinal Chemistry</i> special issue <a href="/page/vi/epigenetics" class="ext-link">Epigenetics 2022</a>.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05834" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05834" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The protein arginine methyltransferase PRMT5 plays a central role in the maintenance of cellular homeostasis. By catalyzing symmetric dimethylation of arginines in over 100 nuclear and cytoplasmic proteins, PRMT5 participates in the regulation of gene expression, ribosomal biogenesis, mRNA splicing, protein translation, DNA damage response, and immune functions.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Not surprisingly, PRMT5 is an essential gene, and its complete inactivation is not compatible with survival of most cell lines.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">PRMT5 has also been shown to exert oncogenic functions, and its levels are dysregulated in many solid and hematological malignancies, an event that correlates with advanced disease and poor clinical outcomes.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> As such, PRMT5 is an attractive drug target in oncology, and several potent and selective inhibitors of the catalytic pocket have been developed and are in clinical trials.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, given the essential role of PRMT5 in normal tissue homeostasis, it is likely that PRMT5 inhibition could have a limited therapeutic window.</div><div class="NLM_p">A wider therapeutic index might be achieved by selectively targeting PRMT5 in cancers bearing deletions of the methylthioadenosine phosphorylase gene, MTAP, with which PRMT5 has been shown to be synthetic lethal. MTAP deletion is a relatively common cancer genomic aberration that occurs in pancreatic tumors (26%), glioblastoma (53%), and mesotheliomas (67%).<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> This loss is due to its proximity to the CDKN2A tumor suppressor locus, with which MTAP is frequently codeleted. Loss of MTAP has unknown functional consequences for tumorigenesis, but it leads to accumulation of its substrate methylthioadenosine (MTA), which can act as an endogenous competitor of <i>S</i>-adenosylmethionine (SAM), the methyl donor used by all arginine <i>N</i>-methyltransferases. Notably, MTA appears to selectively inhibit PRMT5, but not other PRMT family members, likely due to key structural features within the PRMT5 SAM binding pocket.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The partial inhibition of PRMT5 sensitizes MTAP-deleted cells to further loss of PRMT5 function by siRNA. Thus, pharmacological inhibition of PRMT5 could be a viable strategy with a suitable therapeutic window for clinical use against MTAP-deleted cancers.<a onclick="showRef(event, 'ref5 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref8">(5,7,8)</a> Unfortunately, available inhibitors of the PRMT5 catalytic pocket act with either a SAM cooperative<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> or SAM/MTA-competitive mode of action<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10−12)</a> and as such do not appear to take advantage of MTAP deletion. MTA-cooperative PRMT5 inhibitors are predicted to better leverage this synthetic lethality. Two such molecules have been recently disclosed with different degrees of selectivity for MTAP-deleted cancer cells in vitro<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and in preclinical studies.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Inhibitors of the upstream enzyme MAT2A, which catalyzes the synthesis of SAM, have also been discovered and optimized to exploit the unbalanced SAM/MTA ratio of MTAP-deleted cancers. Preclinical data support an MTAP-dependent and synergistic antitumor activity when these inhibitors are used in combination with taxanes or gemcitabine.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> However, SAM is a ubiquitous methyl donor, and a reduction of its levels would alter the activity of multiple methyltransferases, potentially posing a toxicity risk to this approach. The efficacy and safety profile of MAT2A inhibitors are currently being evaluated in clinical trials (NCT03435250).</div><div class="NLM_p">In addition to PRMT5 itself, large-scale shRNA screens also identified PRMT5 substrate adaptors pICln and RIOK1, as well as its obligate partner WDR77, as synthetic lethalities in MTAP-null cancers.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> These results suggested alternative routes for therapeutic modulation of PRMT5 activity, such as direct inhibition of the substrate adaptor proteins or inhibition of their interaction with PRMT5. These mechanisms would be distinct from inhibition of the catalytic site and potentially provide synergy with, or a different therapeutic profile than, previously mentioned approaches. We recently elucidated the molecular mechanism mediating the interaction between PRMT5 and its SAPs. Specifically, we identified a conserved seven residue peptide sequence, GQF(D/E)DA(D/E), present in all three known PRMT5 SAPs (pICln, RIOK1, and COPR5), that mediates substrate adaptor binding to PRMT5, which we termed the PRMT5 binding motif (PBM). We solved the structure of a PBM peptide bound to its cognate site on PRMT5, which is distinct and distal from the catalytic domain. We also determined that genetic perturbation of this site results in loss of substrate methylation and a reduction in MTAP-deleted cell growth relative to WT.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Based on these observations, we launched a screening campaign to identify small molecules capable of inhibiting this interaction.</div><div class="NLM_p last">Here we describe the identification and development of the first-in-class PBM competitive covalent compound, BRD0639, a chemical probe with on-target cellular activity that can support the exploration of a new class of PRMT5 inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38423" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38423" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Hit-Finding Activities</h3><div class="NLM_p">To identify inhibitors of the PBM peptide interaction, we utilized several hit-finding approaches to screen lead-like or fragment compound libraries (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a). As a primary strategy, we utilized a competition fluorescence polarization (FP) assay to measure the ability of compounds to displace the interaction between a fluorophore-labeled RIOK1 PBM peptide and the purified PRMT5:WDR77 hetero-octameric complex. In total, we screened more than 900K small molecules: 850K from the diversity and lead-like compound libraries from Charles River Laboratories, 50K from the Chembridge DIVERSet, 14K compounds from the Broad Institute Diversity Oriented Synthesis (DOS) library,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and 1K compounds selected by virtual screening. In parallel, we also carried out an NMR-based fragment screen and an <i>in silico</i> virtual pharmacophore screen (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">SI</a>). These latter approaches did not yield starting points for optimization and were thus deprioritized.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Hit-finding activities. (a) Low chart for inhibitors of the PBM:PRMT5 interaction. (b) Chemical structure of <b>1</b>. (c) Dose-dependent displacement of a fluorescently labeled RIOK1-derived PBM peptide probe by <b>1</b>, as measured by FP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Conversely, initial hits from the FP assay-based screen (with ≥20% displacement) were reassessed in duplicate and counter-screened against an unrelated protein complex FP assay (MCL1-Noxa) to remove false positives (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a,c). The IC<sub>50</sub> values of confirmed hits were determined using the PRMT5 FP assay across an 8-point dose range. These studies led to the identification of only one cluster, based on a <i>N</i>-aryl acetamide substituted dichloropyridazinone, as exemplified by compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) with an FP IC<sub>50</sub> of 12 μM and solubility of 1.2 μM in PBS. Compound <b>1</b> was validated by SPR, NMR, and X-ray crystallography (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. R<sub>1</sub> Modifications</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0010.gif" alt="" id="GRAPHIC-d7e556-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0011.gif" alt="" id="gr11" /></img><div></div></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Mechanism of binding. (a) Compound <b>1</b> (magenta) binds distal to the catalytic site (SAM analog, sinefungin, red). A protomer of PRMT5 (light green) and WDR77 (dark green) are shown as cartoons. The three other protomer units from the hetero-octamer complex are shown in gray (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0P">6V0P</a>). (b) Crystal structure of compound <b>1</b> (magenta) bound to PRMT5 (green). Residues described in the text are labeled. Two alternate conformations of Y286 are shown (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0P">6V0P</a>). (c) Cryo-EM structure of compound <b>6</b> (cyan) bound to PRMT5 (green) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7M05">7M05</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The PRMT5 catalytic site is distal from the PBM groove, and <i>S</i>-adenosyl methionine (SAM) binding does not affect the interaction between PRMT5 and the PBM peptide.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Consistent with this observation, we found that SAM competitive (JNJ-64619178) and SAM co-operative (EPZ015666) inhibitors of the PRMT5 catalytic site failed to inhibit this binding (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Early SAR</h3><div class="NLM_p">Early hit exploration focused on improving solubility and potency of compound <b>1</b>. Analysis of commercially available analogs highlighted the requirement for the pyridazinone portion, and as such, we began exploring the SAR of sulfonamide variations (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The majority of analogs were synthesized in five steps from available reagents (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The azepane could readily be modified, and its solubility greatly enhanced by the introduction of a basic nitrogen (compound <b>2</b>, solubility 98 μM). Smaller rings (compound <b>4</b>, solubility 89 μM), as well as acyclic sulfonamides (compound <b>5</b>, solubility 56 μM), were both more amenable to rapid analog synthesis and retained significant potency (FP IC<sub>50</sub> 10 μM and 14 μM, respectively), opening the path forward for further SAR exploration. While these simple modifications significantly improved compound solubility, we found that pyridine isomers (compounds <b>6</b>–<b>8</b>) also led to an improvement in potency. Indeed, compound <b>6</b> showed a good balance of potency and solubility (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Modification of the aryl ring (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), either by substitution of the methyl group (compounds <b>9</b>–<b>11</b>) or by the addition of methyl groups to the unsubstituted carbons (X, Y, or Z, compounds <b>12</b>–<b>14</b>), was tolerated, albeit with modest reduction in either potency, solubility, or both. Pyridine isomer <b>15</b> was tolerated, if poorly soluble, whereas other isomers <b>16</b> and <b>17</b> were not, and saturated analog <b>18</b> demonstrated no ability to displace the PBM peptide. Modification of the central acetamide (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) afforded significant improvements in potency by the installation of single methyl group α to the carbonyl <b>20</b>, whereas the other enantiomer <b>21</b> and the <i>N</i>-methyl analog <b>19</b> were decidedly less active. These amide modifications all resulted in increased stability in mouse plasma, presumably by blocking amide hydrolysis by carboxylesterase, which is present in high concentrations in mice.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Central Ring Modifications</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0012.gif" alt="" id="GRAPHIC-d7e698-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0013.gif" alt="" id="gr12" /></img><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. R<sub>3</sub> and R<sub>4</sub> Modifications</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0014.gif" alt="" id="GRAPHIC-d7e712-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0015.gif" alt="" id="GRAPHIC-d7e713-autogenerated" /></img><div></div></div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. (A) Syntheses of <b>1</b> and <b>6</b> and (B) Synthesis of Sulfonamides <b>34</b> and <b>36</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Mechanism of Binding</h3><div class="NLM_p">To understand the mode of binding and support compound optimization, we generated a high-resolution (1.9 Å after elliptical truncation of anisotropic data) cocrystal structure of compound <b>1</b> with the PRMT5:WDR77 complex (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a,b, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0P">6V0P</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Table S1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Figure S1b</a>). Only one site of new electron density was observed and overlapped with the known PBM binding site, confirming inhibition by direct competition. The refined electron density (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Figure S1a</a> with 2Fo – Fc) suggested that the 4-position of the pyridazinone ring forms a covalent bond with PRMT5 Cys278 as a result of nucleophilic attack by the Cys278 thiol and subsequent loss of a chlorine, driven by the rearomatization of the pyridazinone. The crystal structure also points to key noncovalent interactions that are likely to drive initial binding and site specificity. The core aniline of the compound forms a π–π stack with Phe243 and Tyr286, in a manner analogous to that formed with Phe230 of the Riok1 peptide<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0N">6V0N</a>). Several hydrogen bonds are also made between the compound and protein, including (1) between the compound sulfonamide and Asn239, (2) between the compound amide and Ser279, and (3) between Gln282 and the free nitrogen on the pyridazinone.</div><div class="NLM_p">Despite the high-resolution nature of the crystal structure, no unambiguous density was observed for the azepane functional group which might be due to its location at a crystal contact and this was left unmodelled (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Figure S1b</a>).</div><div class="NLM_p last">As significant potency improvements were made via substitution of the original azepane group, we sought to understand these changes from a structural perspective. However, none of the improved potency variants examined were amenable to high resolution crystallization likely due to the local crystal contact interface. We therefore solved a cryo-EM structure of the PRMT5:WDR77 hetero-octameric complex bound to the pyridyl ethyl variant, compound <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Figure S1c</a>, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7M05">7M05</a>). This structure was solved to an overall resolution of 2.4 Å, which represents a substantial improvement compared to previously published cryo-EM structures of PRMT5 resolved at 3.4 Å (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UGH">6UGH</a>) and 3.7 Å (EMD-7137).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The structure has excellent side-chain resolution and clear density for compound <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Figure S1c</a>). The pyridyl ethyl position is well-defined in the density, and the other functional groups are in similar poses compared to the X-ray structure of compound <b>1</b>. The pyridyl ethyl side chain is oriented back toward the core of the molecule via rotation of the sulfonamide and flexibility of the ethyl linker. This molecular conformation produces a new 4-ring stack of π–π interactions consisting of (from “front to back”) Tyr286-pyridine-aryl core-Phe243. Notably, this produces a folded-back structure analogous to the distinct conformation of the PBM peptide.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Compared to the crystal structure of compound <b>1</b>, the density for Lys241 is better resolved, and a hydrogen bond between this side chain and the compound sulfonamide can now be modeled.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Compound <b>6</b> Is a Covalent Binder</h3><div class="NLM_p">We began exploring the putative covalent mode of compound binding by performing a time course FP analysis of compound <b>6</b>. Here, IC<sub>50</sub> values were determined at 10 min intervals up to 4 h. The observed IC<sub>50</sub> values decreased more than 10-fold over time, reaching the assay floor of ∼100 nM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). The covalent nature and site specificity of the compound were further verified using a PRMT5 complex with a Cys278 to Ala mutation. In FP competition experiments, this compound series, as exemplified by compound <b>6</b> (blue squares), has little or no activity against the C278A mutant (maroon squares, 0.6 vs >100 μM) in contrast to a competitor PBM peptide, which is comparable against both WT and C278A proteins (blue and red circles IC<sub>50</sub> values 2.0 vs 1.4 μM, respectively) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>6</b> is a covalent binder. (a) Compound <b>6</b> shows time-dependent displacement of a fluorescently labeled PBM peptide probe as measured by FP. (b) FP competition assay with compound <b>6</b> (squares) or pICln 13-mer control peptide (circles) displacing the fluorescently labeled PBM probe from WT PRMT5 (blue) and C278A mutant (green). (c) Deconvoluted mass spectra for WT and C278A PRMT5 ± compound <b>6</b>. (d) SPR competition assay. PRMT5:WDR77 complex was preincubated with compound <b>6</b> or DMSO and then immobilized to the chip surface. Full-length pICln protein was titrated as analyte.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, intact mass measurement of the WT PRMT5:WDR77 complex before and after compound treatment showed an increase for PRMT5 of 460 Da with compound <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c), correlating with the expected mass of the compound, less one molecule of HCl that is lost upon covalent attachment of a single adduct to PRMT5. No change in the mass of WDR77 was observed (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Figure S2</a>). Consistent with the above findings, treatment of PRMT5<sup>C278A</sup> with compound <b>6</b> did not result in a mass shift, indicating that the covalent attachment took place at Cys278. For reference, the PRMT5 and WDR77 proteins contain 12 and 13 total cysteine residues, respectively, indicating selective binding.</div><div class="NLM_p last">Finally, we investigated whether preincubation with compound <b>6</b> was sufficient to block binding, as measured by SPR, between PRMT5 and full-length pICln. To do so, PRMT5 was preincubated with either DMSO or compound <b>6</b> and then immobilized to the chip surface without further compound treatment. Titration of pICln to DMSO-treated PRMT5 produced binding with a <i>K</i><sub>D</sub> of 32 nM. In contrast, a greatly reduced binding response was observed for compound-treated PRMT5 consistent with linear, nonspecific binding (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). Together, these results demonstrate that the compound binds by an irreversible covalent mechanism and reacts with a single cysteine on the PRMT5:WDR77 complex.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Optimization of Reactivity</h3><div class="NLM_p">Concerned that the highly reactive nature of these molecules could negatively impact their selectivity, we explored modifications of the warhead with the goal of decoupling potency from reactivity. The electrophilic reactivity of compounds toward the cysteine residue of glutathione (GSH) was used as a surrogate for intrinsic reactivity.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Not surprisingly, modifications outside of the warhead had negligible effects on reactivity to GSH, but affected FP potency (see acetamide, sulfonamide and phenyl ring substitutions (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). However, with changes at the pyridazinone warhead we observed a strong correlation between GSH reactivity and FP assay potency (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Of the various substitutions tested, the most interesting was the monochloro substitution, exemplified by compound <b>26</b>, which provided an acceptable balance between activity and reactivity (FP IC<sub>50</sub><sub>40 min</sub> 72.9 μM; GSH <i>T</i><sub>1/2</sub> 693 min). From <b>26</b>, the addition of a chiral α-methyl substituent led to <b>27</b> (BRD0639), which, as observed before, significantly improved potency while retaining low reactivity (FP IC<sub>50</sub><sub>40 min</sub> 13.8 μM; GSH <i>T</i><sub>1/2</sub> 916 min).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. X and Y Modifications</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0016.gif" alt="" id="GRAPHIC-d7e954-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0017.gif" alt="" id="GRAPHIC-d7e955-autogenerated" /></img><div></div></div><div class="NLM_p">To quantify the contribution of various substitutions to both initial reversible binding (<i>K</i><sub>I</sub>) and rate of maximum covalent reactivity (<i>K</i><sub>inact</sub>), we developed a mass spectrometry-based <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> assay. The <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> ratio is considered the best time-independent measure of covalent compound potency.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> In this assay, we incubated purified PRMT5:WDR77 protein with varying concentrations of the inhibitors and quenched over a time course by rapidly reducing the pH with the addition of formic acid. Subsequent LC-MS analysis enabled the quantitation of the loss of unmodified PRMT5 represented as percent occupancy as a function of time across a range of concentrations (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). This allowed the generation of <i>K</i><sub>obs</sub> vs concentration curves (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Figure S3</a>) from which we were able to generate <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> values for a number of inhibitors (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0004.jpeg" id="GRAPHIC-d7e1056-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinetic formation of the covalent adducts between 6 and PRMT5. (a) Dose-dependent kinetics of <b>6</b>-PRMT5 as determined by intact mass spectrometry. (b) <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> analysis of covalent modification by <b>26</b> (blue), <b>25</b> (red), <b>20</b> (green), and <b>6</b> (purple) as determined by intact mass spectrometry. (c) Kinetic parameters for select compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This assay revealed a high degree of correlation between <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> values and the FP IC<sub>50</sub><sub>40 min</sub> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c). For example, the highly potent dichloro compound <b>20</b>, shows a >30-fold increase in the overall rate of adduct formation, compared to compound <b>1</b> (<i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> = 14,626 vs 449 (M<sup>–1</sup> s<sup>–1</sup>), respectively). Here, an improvement in both initial binding (<i>K</i><sub>I</sub>) and maximum potential inactivation rate (<i>K</i><sub>inact</sub>) contributed significantly to the increased potency of compound <b>20</b>. A similar correlation is observed between the pair of monochloro compounds <b>27</b> and <b>26</b> (FP IC<sub>50</sub><sub>40 min</sub> 13.8 μM vs 72.9 μM and <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> 244 (M<sup>–1</sup> s<sup>–1</sup>) vs 56 (M<sup>–1</sup> s<sup>–1</sup>), respectively). However, in this case, the improved potency is primarily the result of a higher <i>K</i><sub>inact</sub> (0.715 vs 0.123 M<sup>–1</sup> s<sup>–1</sup>), which compensates for a modest reduction in apparent <i>K</i><sub>I</sub> (49 vs 36 μM). Thus, while we do not have a structural understanding of this activity relationship, we hypothesize that the affinity differences observed might arise as a result of the methyl group favoring the formation of an “encounter complex” prior to the covalent bond formation.</div><div class="NLM_p last">Indeed, while we observed a clear correlation between FP values and <i>K</i><sub>inact</sub>/<i>K</i><sub>i</sub> values, reactivity to GSH was not always correlated. For example, a few compounds with similar reactivity to GSH (i.e., dichloro <b>6</b> and monofluoro <b>25</b>: <i>T</i><sub>1/2</sub> 31 vs 35 min, respectively) showed more than a 40-fold different rate of covalent modification of PRMT5 (<i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> = 2856 vs 69 (M<sup>–1</sup> s<sup>–1</sup>)), which otherwise correlates well with the observed 17-fold difference in FP IC<sub>50</sub><sub>40 min</sub> (2.2 μM vs 35.6 μM, respectively). This result suggests that the nature of the leaving group, chlorine or fluorine, has significant effects on the overall rate of adduct formation that is not fully explained by their differences in intrinsic reactivity. The same is true for other modifications in close proximity to the leaving group. Indeed, this kinetic analysis guided us to the monochloro pyridazinone as a low intrinsic reactivity warhead that retains good potency.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Cellular Activity of 27: On-Target and On-Mechanism Mode of Action</h3><div class="NLM_p">To determine whether <b>27</b> could engage PRMT5 in a cellular context, we treated Expi293 cells overexpressing HA-tagged WDR77 and WT PRMT5 for 6 h and then analyzed anti-HA immunoprecipitates by LC-MS. This approach revealed a dose-dependent formation of PRMT5-adducts (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). An EC<sub>50</sub> of 3 μM was observed; however, only ∼40% of total PRMT5 protein was labeled by <b>27</b>, possibly due to the high rate of protein production in this overexpression system. Similar results were obtained by treating the cells for 24 h (data not shown).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0021.jpeg" id="GRAPHIC-d7e1259-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cell activities of <b>27</b>. (a) PRMT5 adduct formation using <b>27</b>. Expi293 cells were transiently transfected for 48 h with an HA-tagged PRMT5:WDR77 complex before treatment with <b>27</b>. Cultures were harvested at 6 h and PRMT5 complex isolated by HA affinity and analyzed for modification by LC/MS. (b) PRMT5-RIOK1 NanoBiT assay in permeabilized cells. 293T cells stably expressing SmBiT-PRMT5 and LgBiT-RIOK1 proteins. (c) NanoBit assay performed in intact cells after treatment with <b>27</b> or <b>30</b>. (d) Left panel: WB analysis of total symmetric arginine dimethylation levels in MTAP–/– HCT116 cells, in response to <b>27</b> or <b>30</b>. Right panel: WB analysis of total symmetric arginine dimethylation levels in MTAP–/– HCT116 cells overexpressing the PRMT5 ADA mutant which is unable to bind to the PBM peptide, with and without KO of endogenous PRMT5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To test if compounds were able to disrupt the interaction between PRMT5 and an SAP in cells, we used a PRMT5-RIOK1 NanoBiT complementation assay.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Compounds <b>1</b> and <b>27</b> were tested alongside related inactive compounds <b>29</b> and <b>30</b> (BRD2198) in permeabilized cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). Both compounds <b>1</b> and <b>27</b>, but neither of the inactive compounds, disrupt the PRMT5-RIOK1 complex with an IC<sub>50</sub> of 4 μM and 7.5 μM, respectively. Next, we assessed whether <b>27</b> could engage the target in intact, nonpermeabilized cells. To this end, we treated PRMT5-RIOK1 NanoBit expressing cells with either <b>27</b> or <b>30</b> for 40 min and then assessed the stability of the PRMT5-RIOK1 complex (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c). Consistent with what was observed in permeabilized cells, <b>27</b> but not <b>30</b> disrupted the complex (IC<sub>50</sub> 16 μM vs >100 μM). The different IC<sub>50</sub> values observed in permeabilized vs intact cells (7.5 μM vs 16 μM) suggests a defect in cellular permeability. Finally, cellular treatment with <b>27</b> resulted in inhibition of PRMT5 methyltransferase function, as demonstrated by the reduction in symmetric arginine dimethylation levels by WB (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>d). We found that methylation of some, but not all substrates, was inhibited by treatment with <b>27</b>. Notably, albeit modest when compared to the effects of a PRMT5 catalytic inhibitor (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Suppl. Figure 4</a>), this phenotype is very consistent and closely recapitulates the one observed following genetic disruption of the PBM–PBM groove interaction using a PRMT5 PBM mutant, which failed to rescue methylation of some, but not all substrates<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>d; compare lanes 2 and 5, arrows). This finding is consistent with the conclusion that <b>27</b> induced changes in methylation are on-target and on-mechanism.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We report here the discovery and characterization of a first-in-class PRMT5–PBM competitive inhibitor. Our recent characterization of the binding interface between PRMT5 and its SAPs enabled the development of a robust screening system to identify PBM-competitive small molecule inhibitors. Despite considerable efforts in hit identification using three independent screening approaches, we identified only one chemical starting point. As protein–protein interaction sites can be challenging drug targets, it is possible that larger or more diverse libraries, potentially including DNA-encoded libraries, might yield additional scaffolds.</div><div class="NLM_p">Our initial validated hit was a highly reactive covalent binder. Other liabilities of compound <b>1</b> were poor aqueous solubility and high instability in mouse plasma. Our hit-to-lead efforts resolved the solubility and the plasma instability issues and significantly reduced the intrinsic GSH reactivity to levels comparable with or below that for clinically approved acrylamide warheads.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In addition to a robust FP system for HTS and routine potency measurements, we developed a novel LC-MS assay to quantitatively measure <i>K</i><sub>I</sub> and <i>K</i><sub>inact</sub> to understand the effects of compound modifications more fully. This system revealed that key points of compound modification, such as the azepane to ethylpyridine substitution and <i>S</i>-methyl addition, led to significant improvements in reversible binding without otherwise altering the maximum rate of reactivity. Nonetheless, it is clear that the covalent interaction is a necessary component of compound activity, which proved difficult to remove while retaining potency. However, we were able to tune reactivity to acceptable levels via alteration from the initial dichloro-pyridazinone to a monochloro. Importantly, our lead compound <b>27</b> can engage the cellular target and effectively outcompete binding between full-length PRMT5 and RIOK1 proteins with an IC<sub>50</sub> of 7.5 μM and 16 μM in permeabilized and living cells, respectively. Moreover, we demonstrate a pharmacodynamic response to the most proximal marker of PRMT5 activity, the formation of symmetrically dimethylated arginine. This effect appears to be on-target as closely related compounds with the same warhead, but inactive in FP assays, are also inactive in the cellular context. Consistent with an on-target effect, <b>27</b> (BRD0639) reduces SDMA in the same subset of proteins also affected by genetic perturbation of the PBM binding site.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">A PBM-competitive inhibitor may result in a different pharmacodynamic and therapeutic response as compared to that of a catalytic inhibitor. PRMT5 has a key role in regulating multiple essential cellular activities, including transcription, ribosomal biogenesis, and mRNA splicing. Genetic disruption of the PBM binding site, however, leads to selective impairment of its splicing function mediated by the SAP pICln.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> In contrast, catalytic site inhibitors of PRMT5 are pan-substrate and therefore may be more likely to result in nontolerated on-target toxicity.</div><div class="NLM_p last">Moving forward, it would be interesting to explore additional strategies for targeting the PRMT5 methylosome. It is notable that the relevance of PRMT5 to MTAP-deleted cancers was observed by shRNA knockdown, suggesting that partial reduction in total PRMT5 protein concentration would also be sufficient to see selective viability effects. As such, PRMT5 degraders could represent an alternative therapeutic approach to this target. It should be noted that the development of heterobifunctional degraders directed against PRMT5 has been reported recently.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> However, their potency is limited, possibly since conventional PRMT5 catalytic inhibitors were used as the target bait. Here, we believe that the depth of the catalytic pocket might pose a structural challenge in connecting a catalytic PRMT5 inhibitor, via a linker, to an E3-ligase binder. From this perspective, the discovery of a new surface exposed and druggable site on PRMT5, the development of a fully validated suite of assays, and the generation of a tool compound to support screenings for new, noncovalent binders could offer a unique starting point for the development of potent PRMT5 degraders.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Recombinant Protein Expression and Purification</h3><div class="NLM_p last">Codon optimized DNAs for PRMT5 (uniprot: <a href="https://O14744" class="ext-link">O14744</a>) and WDR77 (uniprot: <a href="https://Q9BQA1" class="ext-link">Q9BQA1</a>) were cloned into the pFastBac Dual baculovirus vector (Thermo Fisher) under the control of either the p10 or polyhedrin promoter, respectively. The WDR77 protein was engineered with either an N-terminal His6-Strep tag II followed by the HRV 3C protease site (PRMT5WT:WDR77 construct) or a C-terminal ybbR-sortase and His6 tag (PRMT5C278A:WDR77 construct) to aid purification, resulting in the mass difference observed for intact WDR77 in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">Figure S2</a>. The C278A mutation was introduced to PRMT5 using the QuickChange mutagenesis method (Agilent Technologies). PRMT5 was untagged in all constructs. The PRMT5:WDR77 heterocomplex was coexpressed in Sf9 (<i>Spodoptera frugiperda</i>) cells with a BIIC ratio of 1:10,000. Cultures were harvested after 72 h. Purification began with resuspension of the cell pellet in lysis buffer (25 mM HEPES (pH 7.4), 300 mM NaCl, 1 mM TCEP, 10% glycerol, 0.1% Triton X100, SigmaFast Protease Inhibitor tablets) before breaking with a microfluidizer (Avestin). The lysate was clarified via centrifugation, filtered with a 0.2 μm filter, and passed over a Streptactin XT affinity column (IBA GmbH). The column was washed for 20 CV with wash buffer (25 mM HEPES (pH 7.4), 300 mM NaCl, 1 mM TCEP, 10% glycerol) before eluting bound protein with buffer containing 2.5 mM desthiobiotin. The N-terminal tag was cleaved with the addition of HRV 3C protease (AG Scientific) and incubation overnight at 4 °C. The PRMT5:WDR77 complex was further purified by passing samples over a Superose 6 (10/300) column (GE LifeSciences) using SEC buffer (50 mM HEPES (pH 7.4), 10% glycerol, 1 mM TCEP, 150 mM NaCl). Aliquots were frozen at −80 °C.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Fluorescence Polarization</h3><div class="NLM_p last">The FP competition assay had final concentrations 200 nM PRMT5:WDR77 protomer, (experimentally determined <i>K</i><sub>D</sub> for the interaction),<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> 10 nM peptide probe, 50 mM HEPES pH 7.4, 100 mM NaCl, 0.5 mM TCEP, and 0.01% v/v Tween 20. For HTS, 1536-well black nonbinding surface plates (Corning) were used with a 5 μL assay volume. The assay reagents were added to preplated compounds with a final compound concentration of 20 μM. The plates were incubated at room temperature and the final end point read at 40 min. For SAR data, experiments were performed in triplicate in 384-well black nonbinding surface plates (Corning) at a final assay volume of 20 μL. Data were collected using either a Spectramax Paradigm with Rhodamine FP filter set or a PerkinElmer Envision with Bodipy TMR FP filter set. The peptide probe was a KU560 fluorophore (KU dyes, catalog KU560-R-6)-labeled peptide derived from the RIOK1 PBM sequence: [acetyl]SRVVPGQFDDADSSD[C^KU560][amide]). As a positive control, peptide [acetyl]LMSRVVPGQFDDADSSD[amide] was used at a 20 μM concentration. Fluorescence polarization response was normalized across each plate to 100 (average of DMSO control wells) and 0 (average of unlabeled peptide control).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Nuclear Magnetic Resonance</h3><div class="NLM_p last">Experiments were performed on a 600 MHz Bruker Avance III spectrometer equipped with a 5 mm QCI cryoprobe and a SampleJet for automated sample handling. All experiments were conducted at 280 K. For STD NMR (Mayer, 1999; Begley, 2013), final sample conditions involved 200 μM ligand, 2% DMSO, 1 μM protomer, 25 mM HEPES-<i>d</i>, pH = 7.4, 150 mM NaCl, 1 mM TCEP-<i>d</i>, followed by competition with 20 μM 13-mer peptide (<i>K</i><sub>D</sub> 300 nM). All NMR data were analyzed using Topspin (Bruker). The SPY peptide was synthesized by Thermo-Fisher, sequence: [acetyl]GQF*EDAD[amide], where F* is 3-fluoro phenylalanine; <sup>19</sup>F signal at −113.75 ppm.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Surface Plasmon Resonance</h3><div class="NLM_p last">Biacore experiments were performed as previously described<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> with the exception that PRMT5 protein was preincubated at a concentration of 25 nM with 2 μM compound <b>6</b> or DMSO overnight at 4C prior to immobilization. Labeling was confirmed to be complete by LC-MS. Data were fit in Prism using a one site, total and nonspecific binding model.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Crystallography</h3><div class="NLM_p last">Crystals were produced as previously described<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and then soaked with compound <b>1</b> at an approximate concentration of 500 μM for 72 h before harvesting. Diffraction images were indexed and integrated using XDS and further processed for anisotropy via elliptical truncation using the STARANISO server (Global Phasing) Cambridge, United Kingdom: Global Phasing Ltd.). Refinement was performed in Buster (Global Phasing) and Phenix with manual building/review in Coot. Compound restraints were generated using Glide (Global Phasing).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Cryo-EM</h3><div class="NLM_p last">PRMT5:WDR77 complex at a 5 μM concentration was coincubated with 25 μM compound <b>6</b> and 20 μM JNJ-64619178 overnight at 4C. Covalent modification was confirmed by LC-MS. PRMT5 complex was then purified by size exclusion on a Superose 6 Increase 10/300 GL (GE Healthcare) column in mobile phase buffer 10 mM HEPES pH 7.4, 150 mM NaCl, 10% glycerol, 1 mM TCEP, and 100 nM JNJ-64619178. Protein was concentrated to 10 mg/mL using a Proteus X-spinner with 10 kDa filter (Anatrace) and snap frozen in liquid nitrogen. UltrAuFoil 300 mesh grids were pretreated by glow discharge for 30 s. Immediately prior to application, protein samples were thawed and diluted 7-fold in a buffer containing 10 mM HEPES pH 7.4, 150 mM NaCl, 1 mM TCEP, and 100 nM JNJ-64619178. Using a FEI Vitrobot Mark IV, 3.5 μL of the protein sample was applied to the grid before blotting and plunge-freezing into liquid ethane. Data were acquired on a Titan Krios microscope with Gatan K2 Quantum detector at a nominal magnification of 130,000× and randomized defocus values of −1.4, −1.7, or −2.1 Å. For each image, 40 movie frames were recorded with a total exposure time of 8 s and estimated electron dose of 62.4 e<sup>–</sup>/Å<sup>2</sup>. A total of 2169 movies were collected from one grid using automatic acquisition. All data processing was performed in the cisTEM software suite.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Frames 4–40 from each movie were motion corrected, summed, and then processed by CTF estimation. All images with a CTF fit resolution ≤3.1 Å (1836 images) were retained for single particle analysis. A total of 956,646 particles were picked for 2D classification. Due to particle crowding, only 474,404 particles (22 of 50 classes) were selected for 3D refinement. EMD-7137 was used as a starting volume model with an initial high-resolution limit of 20 Å and a defined <i>D</i><sub>2</sub> symmetry. 3D refinement converged at an estimated resolution of 2.39 Å using half-map analysis with a 0.143 FSC cutoff. Phenix Autosharpen was used prior to rigid body docking of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0P">6V0P</a> crystal structure as a starting model. Manual model building was performed in Coot followed by real space refinement in Phenix.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Stability to GSH</h3><div class="NLM_p last">Ten μL of a compound at a final concentration of 0.1, 1, and 10 μM, or control working solution was diluted in 190 μL of 5 mM GSH in PBS (pH 7.4) and incubated at 37 °C for 0, 15, 30, 60, 120, and 1440 min, in duplicate. As a negative control, incubations without GSH were carried out at two time-points (0 and 1440 min). Ibrutinib and afatinib were used as positive controls and were tested at a final concentration of 10 μM. At each time point, the reaction was terminated by adding 600 μL of cold acetonitrile containing labetalol as the internal standard. The samples were stored at −80 °C until the last incubation time point was completed. Sample plates were defrosted and placed on a shaker for 5 min, followed by centrifugation at 4000 rpm for 20 min. 100 μL of each sample, after centrifugation, was further diluted with water before mass spec analysis. Samples with compound concentration at 0.1, 1, and 10 μM were diluted with 100 μL, 300 μL, and 600 μL water, respectively. Samples were analyzed by LC-MS/MS (Sciex API 4000). The percent of the parent compound remaining at each time point was determined based on peak area ratios at the 0 min time point, and half-life was calculated using the first-order kinetics equation. In addition, the samples were analyzed for formation of the predicted GSH adduct at each time point.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Intact Mass Measurement by LC-MS</h3><div class="NLM_p last">Purified PRMT5:WDR77 was thawed on ice and centrifuged to remove potential aggregates from the freeze/thaw process. The protein was then solvent exchanged into the reaction buffer (10 mM HEPES 7.4, 150 mM NaCl, 1 mM TCEP), using a 40 kDa Zeba desalting column (ThermoFisher), and diluted to 10 μM. For single time point experiments, a mixture of 10 μL of protein, 1 μL of compound <b>6</b> (0.5 mM in DMSO), or DMSO and 90 μL reaction buffer were incubated at RT for 4 h. Intact mass measurement of the PRMT5:WDR77 complex with and without compounds was performed using the BioAccord LC-ToF (composed of an ACQUITY I-Class UPLC and RDa detector with ESI source, Waters Corporation). One μL of each sample was injected onto a C4 column (ACQUITY UPLC Protein BEH, 300 Å, 1.7 μm, 2.1 × 50 mm, Waters Corporation) held at 80 °C. Mobile phases A and B consisted of 0.1% formic acid (MilliporeSigma LiChroPur) in LC-MS grade water or LC-MS grade acetonitrile (JT Baker), respectively, with initial column conditions set to 95% water/5% acetonitrile. Protein was desalted for 1 min before elution with a gradient of 5% to 85% mobile phase B in 2.5 min followed by ionization in positive ionization mode with the cone voltage set to 55 V and desolvation temperature of 550 °C. The instrument scan rate was 5 Hz over 50 to 2000 <i>m</i>/<i>z</i>. PRMT5 and WDR77 coeluted at 2.38 min, and mass spectra were deconvoluted using UNIFI and the MaxEnt1 algorithm.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Covalent Modification of PRMT5 and Kinetic Analysis (<i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub>) by LC-MS</h3><div class="NLM_p last">Purified PRMT5:WDR77 was thawed and treated as described in the previous section. For <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> analyses, PRMT5:WDR77 was diluted to 50 nM and dispensed into 384-well plates (Greiner Bio-One #781280) containing a 12-point dilution series of various compounds to collect 8 time points (0, 2,5, 10, 20, 30, 45, and 60 min) using a 384ST-head Agilent Bravo. Time points were quenched with formic acid (LiChropur, MilliporeSigma, final concentration 0.5%). To quantify the unmodified PRMT5 and WDR77, a multiple-reaction monitoring (MRM) method was developed using a Waters ACQUITY UPLC I-Class PLUS chromatography system connected to a Xevo TQ-XS mass spectrometer by focusing on a single charge state for each protein (PRMT5, <i>m</i>/<i>z</i> 855.0 → <i>m</i>/<i>z</i> 854.90, <i>z</i> = 85; WDR77, <i>m</i>/<i>z</i> 870.80 → <i>m</i>/<i>z</i> 870.75, <i>z</i> = 46). ESI source parameters were set as follows: positive ionization mode, capillary voltage 2.5 kV, cone voltage, 50 V; collision energy, 5 V; cone gas flow, 200 L/h; collision gas flow, 0.16 mL/min; source temperature, 150 °C; desolvation temperature, 350 °C; and desolvation gas flow, 900 L/h. Analytes were separated on an Agilent PLRP-S column (5 μm; 2.1 × 50 mm, 1000 Å Agilent Technologies) at 60 °C. Sample storage temperature was set to 20 °C. Mobile phases and initial conditions were the same as described above. Proteins were eluted using a 1.5 min gradient of 5% to 98% acetonitrile at 0.4 mL/min. MassLynx and TargetLynx software (Waters, version 4.2) were used for sample acquisition and data quantification, respectively. Analytes were quantified by integration of peak areas for each charge state described above. For analysis, the peak areas were normalized to the 0 min time point. Kinetic analysis of the <i>K</i><sub>inact</sub> and <i>K</i><sub>I</sub> values were fit in Graphpad Prism 7.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Target Engagement in Cells</h3><div class="NLM_p last">Expi293 cells were cultured in a 1:1 mixture of Expi293:Freestyle media (Thermofisher). At a cell density of ∼2.5 × 10<sup>6</sup> cells/mL, 250 mL of cells was transfected using 200 μL FectoPRO (Polyplus Transfection) reagent mixed with 100 μg of PRMT5 and 100 μg HA-tagged WDR77 plasmids.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> One day after transfection, 3 mM valproic acid and 0.4% w/v glucose were added to the culture. Two days after transfection, cells were split into multiple flasks with 30 mL cells each and treated with compound or DMSO at a final 0.2% DMSO concentration. Cell sample was collected after 6 h, pelleted, and washed 3 times with ice-cold PBS. Additional aliquots were analyzed for viability and cell number using a Vi-Cell XR Cell Counter (Beckman Coulter), and all samples were determined to have >98% viability based on trypan blue exclusion. Each sample was lysed in buffer 50 mM HEPES pH 7.5, 300 mM NaCl, 1 mM TCEP, 1% v/v Tween-20, and 2 mM reduced GSH. The PRMT5:WDR77 complex was immunoprecipitated by anti-HA agarose resin (Thermofisher) and then eluted by 50 μM 3X-HA peptide (AnaSpec). Each eluate was measured by intact mass LC-MS to determine the percentage of complex with compound adduct.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Nanobit</h3><div class="NLM_p">HEK293T cells stably expressing PRMT5 tagged with an N-terminal SmBiT peptide and RIOK1 tagged with N-terminal LgBiT peptide were described previously.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Cells were cultured at 37 °C in 5% CO<sub>2</sub> in Dulbecco’s modified Eagle’s medium (DMEM, Life Technologies) supplemented with 10% FBS (Life Technologies) and 1% penicillin-streptomycin (Life Technologies). For the assay, cells were centrifuged at 150 rad/s, media removed, and resuspended in Opti-MEM (Life Technologies). Cells were plated at (10 × 10<sup>4</sup> cells/well) in 96-well black wall, clear bottom, tissue culture treated plates (Corning) and treated with the indicated compounds using D300e Digital Printer (Tecan). All wells normalized to highest DMSO concentration (1% v/v). Each experimental condition was tested in triplicate.</div><div id="sec4_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> NanoBiT in Permeabilized Cells</h4><div class="NLM_p last">50 μL of lysis buffer was added to each well (10% glycerol, 50 mM Tris-HCL, 150 mM KCl, 2 mM EDTA, 0.1% NP40), and cells were incubated for 5 min at RT. Nano-Glo Luciferase Assay Substrate (Promega #N1110) was diluted 1:50 in buffer, and 100 μL added to each well and pipetted to mix.</div></div><div id="sec4_11_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> NanoBiT in Intact Cells</h4><div class="NLM_p last">Cells were treated for 40 min, at which point Nano-Glo Live Cell Substrate (Promega, # N2011) was diluted 1:20 in buffer, 25 μL mixture was added to each well, and the plate tapped gently to mix. Luciferase signal was measured using the EnVision plate reader (PerkinElmer). Results shown are representative of three independent experiments.</div></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Western Blot</h3><div class="NLM_p last">HCT116 MTAP–/– were cultured at 37 °C in 5% CO<sub>2</sub> in DMEM (Life Technologies), supplemented with 10% FBS (Life Technologies), and 1% penicillin-streptomycin (Life Technologies). Cells were plated at 1 × 10<sup>6</sup>/well in 6-well tissue culture treated plates (Corning) and treated the next day with the indicated compounds to a final concentration of 25 μM. DMSO and nontreated control wells included. Twelve h after treatment, the media was refreshed and cells retreated. Twelve h later, cells were washed 1× with PBS and lysed on ice for 15 min in 50 μL lysis buffer (1 mL RIPA, 10% glycerol, 1% protease inhibitor, 1% phosphatase inhibitor). Protein concentration was calculated using Pierce BCA Protein Assay Kit, 40 μg protein per sample was run on a 4–12% Bis-Tris gels (NuPAGE, Life Technologies) using MES buffer. Gels were dry transferred to nitrocellulose membranes (iBlot system, Life Technologies). Membranes were blocked using intercept blocking buffer (LI-COR) for 1 h and probed overnight with primary antibodies (Rabbit anti-SDMA, CST13222; mouse antivinculin, Sigma, Rabbit anti-PRMT5 (Abcam Ab31751). Blots were washed 3× with 1% TBST buffer and probed with secondary antibodies 680RD goat anti-Mouse and 800CW goat anti-Rabbit (LI-COR) for 1 h. Membranes were washed 3× with 1% TBST and imaged using LI-COR Odyssey imaging system. Results shown are representative of three independent experiments.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Chemical Synthesis</h3><div id="sec4_13_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> General Experimental Conditions for Small Molecule Synthesis</h4><div class="NLM_p last">All anhydrous solvents and reagent grade solvents for chromatography and starting materials were purchased from either Sigma-Aldrich Chemical Co. or Fisher Scientific. Water was distilled and purified through a Milli-Q water system (Millipore Corp., Bedford, MA). General methods for purification of compounds involved the use of silica cartridges purchased from Grace or Combiflash Purification systems. The reactions were monitored by TLC on precoated Merck 60 F254 silica gel plates and visualized using UV light (254 nm). All compounds were analyzed for purity by UPLC and shown to have ≥95% purity. NMR spectra were obtained on a Bruker NanoBay 400 instrument. Chemical shifts are reported in parts per million (ppm, δ) relative to the residual solvent peak in the corresponding spectra (chloroform δ 7.26, methanol δ 3.31, DMSO δ 3.33) and coupling constants (<i>J</i>) are reported in hertz (Hz) (where s = singlet, bs = broad singlet, d = doublet, dd = double doublet, bd = broad doublet, ddd = double doublet of doublet, t = triplet, tt = triple triplet, q = quartet, m = multiplet) and analyzed using ACD NMR or MestReNova data processing. Mass spectra values are reported as <i>m</i>/<i>z</i>. All reactions were conducted under nitrogen unless otherwise noted. Solvents were removed in vacuo on a rotary evaporator.</div></div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Analytical UPLC Methods</h3><div class="NLM_p">LRMS (LC-MS) Instrument: Agilent 1200\G1956A; Column: Kinetex EVO C18 30*2.1 mm, 5 μm; eluent A: 0.0375% TFA in water (v/v); eluent B: 0.01875% TFA in acetonitrile (v/v); gradient 0–0.8 min 5–95% B, 0.8–1.2 min 95% B, 1.2–1.5 5% B; flow 1.5 mL/min; temperature 50 °C; DAD scan: 200–500 nm.</div><div class="NLM_p">HRMS (LC-MS) was performed on purified compounds, diluted to 0.1 mM in DMSO, injecting 3 μL, and reported data are an average of triplicate runs. Instrument: Agilent 1290 UHPLC; Column: Waters Acquity UPLC BEH C18, 1.7 μm, 2.1 × 50 mm; eluent A: water with 0.1% formic acid, B: acetonitrile with 0.1% formic acid; gradient 0–0.1 min 5% B, 0.1–5.0 min 5–95% B, 5.0–5.5 min 95% B, 5.5–5.6 min 5% B, 5.6–6.0 min 5% B; flow 0.5 mL/min; temperature 45 °C; DAD scan: 200–500 nm, MS System Agilent 6545. Quadrupole time of flight source parameters: gas temp (°C) 350, drying gas (l/min) 10, nebulizer (psi) 20, sheath gas temperature (°C) 400, sheath gas flow (l/min) 12, capillary voltage (V) 3500, nozzle voltage (V) 2000.</div><div id="sec4_14_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Synthesis of <b>1</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>A)</h4><div id="sec4_14_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Ethyl 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)acetate (<b>31</b>)</h5><div class="NLM_p last">To a mixture of 4,5-dichloropyridazin-3(2H)-one (CAS 932-22-9, 30 g, 181.84 mmol) and K<sub>2</sub>CO<sub>3</sub> (50.27 g, 363.69 mmol) in DMF (150 mL) was added ethyl 2-bromoacetate (33.40 g, 200.03 mmol, 22.12 mL) at 25 °C, and the mixture was stirred at 50 °C for 2 h. The reaction mixture was diluted with water (100 mL) and ethyl acetate (300 mL), and the layers separated. The organic layer was washed with saturated aqueous sodium chloride (200 mL), dried over anhydrous sodium sulfate, insoluble materials removed by filtration, and volatiles removed under reduced pressure. The resulting residue was purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether to give the title compound as a white solid (38.4 g, 84% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>8</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>3</sub> 260.0; found 250.9; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 7.82 (s, 1H), 4.90 (s, 2H), 4.28 (q, 2H), 1.30 (t, 3H).</div></div><div id="sec4_14_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)acetic acid (<b>32</b>)</h5><div class="NLM_p last">Ethyl 2-(4,5-dichloro-6-oxopyridazin-1(6H)-yl)acetate (<b>31</b>, 15 g, 59.75 mmol) was suspended in aqueous hydrochloric acid (5% w/v, 300 mL), the mixture was stirred at 105 °C for 90 min, cooled to room temperature, and the resulting solid was isolated by filtration to give the title compound (10 g, 75% yield) as a white solid. LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>6</sub>H<sub>5</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>3</sub>223.0; found 223.0; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 7.85 (s, 1H), 4.95 (s, 2H).</div></div></div><div id="sec4_14_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Synthesis of <b>34</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>B)</h4><div id="sec4_14_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 1-((2-Methyl-5-nitrophenyl)sulfonyl)azepane (<b>33</b>)</h5><div class="NLM_p last">To a solution of 2-methyl-5-nitrobenzene-1-sulfonyl chloride (5 g, 21.22 mmol) and azepane (4.32 g, 31.83 mmol, 4.91 mL) in THF (100 mL) at 0 °C was slowly added TEA (6.44 g, 63.66 mmol, 8.86 mL). The mixture was stirred for 12 h at 25 °C, volatiles removed under reduced pressure, and the residue partitioned between ethyl acetate and water. The layers were separated, and the aqueous phase was extracted twice more with ethyl acetate. The combined organic layers were washed with saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, insoluble materials removed by filtration, and volatiles removed under reduced pressure to provide the title compound as a yellow oil (3.7 g, 58% yield), which was used without further manipulation. LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>13</sub>H<sub>19</sub>N<sub>2</sub>O<sub>4</sub>S 289.1; found 299.0; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 8.60 (d, <i>J</i> = 2.4 Hz, 1H), 8.27 (dd, <i>J</i> = 2.4, 8.4 Hz, 1H), 7.51 (d, <i>J</i> = 8.3 Hz, 1H), 3.47–3.40 (m, 4H), 2.75 (s, 3H), 1.88–1.78 (m, 4H), 1.74–1.65 (m, 4H).</div></div><div id="sec4_14_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 3-(Azepan-1-ylsulfonyl)-4-methylaniline (<b>34</b>)</h5><div class="NLM_p last">A mixture of 1-((2-methyl-5-nitrophenyl)sulfonyl)azepane (<b>33</b>, 3.7 g, 12.40 mmol) and palladium on carbon (100 mg, 10% w/w) in methanol (80 mL) was degassed and purged with hydrogen three times, and the mixture was stirred at 25 °C for 16 h under hydrogen atmosphere (15 psi). The mixture was filtered through Celite, and volatiles removed under reduced pressure to afford the title compound as a yellow oil (3.4 g, 102% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>13</sub>H<sub>21</sub>N<sub>2</sub>O<sub>2</sub>S 269.1; found 269.0.</div></div><div id="sec4_14_2_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> <i>N</i>-(3-(Azepan-1-ylsulfonyl)-4-methylphenyl)-2-(4,5-dichloro-6-oxopyridazin-1(6H)-yl)acetamide (<b>1</b>)</h5><div class="NLM_p last">A mixture of 2-(4,5-dichloro-6-oxopyridazin-1(6H)-yl)acetic acid (<b>32</b>, 997.17 mg, 4.47 mmol) and DMF (27.24 mg, 372.61 μmol, 28.67 μL) and thionyl chloride (664.95 mg, 5.59 mmol) in THF (20 mL) at 0 °C was heated at 60 °C for 30 min. The solution was cooled to 0 °C, treated with 3-(azepan-1-ylsulfonyl)-4-methylaniline (<b>34</b>, 1 g, 3.73 mmol), and TEA (754.10 mg, 7.45 mmol, 1.04 mL) and warmed to 25 °C and stirred for 30 min. The reaction mixture was diluted with saturated aqueous ammonium chloride and extracted twice with ethyl acetate, and combined organic layers were dried over sodium sulfate. Insoluble materials were removed by filtration, volatiles were removed under reduced pressure, and the residue was purified by prep-HPLC (column: Kromasil 250*50 mm*10 μm; mobile phase: [water (0.225% FA)-ACN]; B%: 37ACN%-67ACN%, 26 min, 85% min) to give the title compound as a white solid (1.4 g, 79% yield). LCMS: RT = 0.805 min, [M + H]<sup>+</sup> = 418.2; <sup>1</sup>H NMR (400 MHz, DMSO) δ 10.60 (s, 1H), 8.29 (s, 1H), 8.06 (d, <i>J</i> = 2.3 Hz, 1H), 7.66 (dd, <i>J</i> = 8.3, 2.3 Hz, 1H), 7.38 (d, <i>J</i> = 8.3 Hz, 1H), 4.98 (s, 2H), 3.27 (t, <i>J</i> = 5.8 Hz, 4H), 2.47 (s, 3H), 1.65 (tt, <i>J</i> = 6.4, 3.8 Hz, 4H), 1.56 (dq, <i>J</i> = 5.8, 4.2, 2.7 Hz, 4H); 13C{1H} NMR (101 MHz, DMSO) δ 164.70, 155.89, 138.32, 136.64, 136.54, 136.11, 133.33, 132.85, 131.26, 122.50, 118.77, 55.79, 47.63, 28.91, 26.35, 19.26. HRMS (ESI/Q-TOF) RT 3.35 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>4</sub>O<sub>4</sub>S 473.0817; found 473.0371.</div></div><div id="sec4_14_2_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 2-Methyl-5-nitro-<i>N</i>-[2-(pyridin-2-yl)ethyl]benzene-1-sulfonamide (<b>35</b>)</h5><div class="NLM_p last">Compound <b>35</b> was made analogously to <b>33</b>. LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>14</sub>H<sub>16</sub>N<sub>3</sub>O<sub>4</sub>S 322.36; found 322.0; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.83 (d, <i>J</i> = 2.4 Hz, 1H), 8.48 (ddd, <i>J</i> = 5.0, 1.9, 0.9 Hz, 1H), 8.26 (dd, <i>J</i> = 8.3, 2.5 Hz, 1H), 7.62 (td, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.47 (d, <i>J</i> = 8.3 Hz, 1H), 7.18 (ddd, <i>J</i> = 7.6, 4.9, 1.1 Hz, 1H), 7.10 (d, <i>J</i> = 7.8 Hz, 1H), 6.87 (t, <i>J</i> = 5.8 Hz, 1H), 3.42 (q, <i>J</i> = 5.7 Hz, 2H), 3.01–2.96 (m, 2H), 2.76 (s, 3H).</div></div><div id="sec4_14_2_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 5-Amino-2-methyl-<i>N</i>-[2-(pyridin-2-yl)ethyl]benzene-1-sulfonamide (<b>36</b>)</h5><div class="NLM_p">Compound <b>36</b> was made analogously to <b>34</b>. LRMS (<i>m</i>/<i>z</i>): calcd [M – H]<sup>−</sup> for C<sub>14</sub>H<sub>16</sub>N<sub>3</sub>O<sub>2</sub>S 290.4; found 290.0; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.47 (dt, <i>J</i> = 4.8, 1.4 Hz, 1H), 7.58 (td, <i>J</i> = 7.7, 1.9 Hz, 1H), 7.33 (d, <i>J</i> = 2.5 Hz, 1H), 7.17–7.11 (m, 1H), 7.06 (d, <i>J</i> = 7.8 Hz, 1H), 7.01 (d, <i>J</i> = 8.0 Hz, 1H), 6.72 (dd, <i>J</i> = 8.1, 2.6 Hz, 1H), 6.24 (t, <i>J</i> = 6.0 Hz, 1H), 3.57 (s, 2H), 3.33 (q, <i>J</i> = 6.0 Hz, 2H), 2.92 (t, <i>J</i> = 5.9 Hz, 2H), 2.45 (s, 3H).13C{1H} NMR (101 MHz, CDCl3) δ 159.29, 149.06, 144.64, 138.41, 136.88, 133.47, 126.07, 123.67, 121.89, 118.90, 116.17, 42.13, 36.00, 19.09.</div><div class="NLM_p last">Compounds <b>2</b> and <b>4</b>–<b>14</b> were made analogously to <b>1</b>. Compound <b>3</b> was purchased from Enamine (catalog ID: Z19024493) and used without further purification.</div></div><div id="sec4_14_2_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-((4-methyl-1,4-diazepan-1-yl)sulfonyl)phenyl)acetamide (<b>2</b>)</h5><div class="NLM_p last">LCMS: RT = 0.742 min, [M + H]<sup>+</sup> = 488.1; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.66–10.53 (m, 1H), 8.33–8.25 (m, 1H), 8.12–8.01 (m, 1H), 7.77–7.62 (m, 1H), 7.49–7.29 (m, 1H), 5.12–4.91 (m, 2H), 3.34–3.30 (m, 2H), 2.60–2.52 (m, 6H), 2.49–2.44 (m, 3H), 2.29–2.20 (m, 3H), 1.83–1.75 (m, 2H); HRMS (ESI/Q-TOF) RT = 2.01 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>24</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 488.0926; found 488.0919.</div></div><div id="sec4_14_2_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-(piperidin-1-ylsulfonyl)phenyl)acetamide (<b>3</b>)</h5><div class="NLM_p last">HRMS (ESI/Q-TOF) RT = 3.22 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>21</sub>C<sub>l2</sub>N<sub>4</sub>O<sub>4</sub>S 459.0660; found 459.0205.</div></div><div id="sec4_14_2_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-((4-methylpiperazin-1-yl)sulfonyl)phenyl)acetamide (<b>4</b>)</h5><div class="NLM_p last">LCMS: RT = 0.753 min, [M + H]<sup>+</sup> = 474.0; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.66 (s, 1H), 8.30 (s, 1H), 8.11 (d, <i>J</i> = 2.1 Hz, 1H), 7.69 (br d, <i>J</i> = 2.1 Hz, 1H), 7.42 (d, <i>J</i> = 8.3 Hz, 1H), 4.99 (s, 2H), 3.06 (br s, 4H), 2.53–2.52 (m, 3H), 2.35 (br s, 4H), 2.16 (s, 3H). HRMS (ESI/Q-TOF) RT = 1.97 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 474.0769; found 474.0763.</div></div><div id="sec4_14_2_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(3-(<i>N</i>,<i>N</i>-dimethylsulfamoyl)-4-methylphenyl)acetamide (<b>5</b>)</h5><div class="NLM_p last">LCMS (ESI): RT = 0.877 min, [M + H]<sup>+</sup> = 418.9; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.67–10.62 (m, 1H), 8.30 (s, 1H), 8.09 (d, <i>J</i> = 2.1 Hz, 1H), 7.70 (dd, <i>J</i> = 2.3, 8.3 Hz, 1H), 7.42 (s, 1H), 4.99 (s, 2H), 2.73 (s, 6H), 2.54–2.53 (m, 3H); HRMS (ESI/Q-TOF) RT = 2.72 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>17</sub>C<sub>l2</sub>N<sub>4</sub>O<sub>4</sub>S 419.0347; found 419.0345.</div></div><div id="sec4_14_2_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>6</b>)</h5><div class="NLM_p last">LCMS: RT = 0.732 min, [M + H]<sup>+</sup> = 496.1; <sup>1</sup>H NMR (400 MHz, DMSO) δ 10.58 (s, 1H), 8.40 (ddd, <i>J</i> = 4.8, 1.9, 0.9 Hz, 1H), 8.29 (s, 1H), 8.09 (d, <i>J</i> = 2.3 Hz, 1H), 7.77 (t, <i>J</i> = 5.8 Hz, 1H), 7.69 (dd, <i>J</i> = 8.2, 2.3 Hz, 1H), 7.64 (td, <i>J</i> = 7.7, 1.9 Hz, 1H), 7.31 (d, <i>J</i> = 8.4 Hz, 1H), 7.17 (ddd, <i>J</i> = 7.6, 4.8, 1.2 Hz, 1H), 7.13 (dt, <i>J</i> = 7.8, 1.1 Hz, 1H), 4.97 (s, 2H), 3.19–3.08 (m, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H); <sup>13</sup>C{<sup>1</sup>H} NMR (101 MHz, DMSO) δ 164.61, 158.23, 155.90, 148.97, 138.72, 136.50, 136.45, 136.40, 136.08, 132.99, 132.85, 131.22, 123.20, 122.55, 121.58, 119.29, 55.76, 41.96, 37.28, 19.0;. HRMS (ESI/Q-TOF) RT = 1.95 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 496.0613; found 496.0606.</div></div><div id="sec4_14_2_2_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-(<i>N</i>-(2-(pyridin-3-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>7</b>)</h5><div class="NLM_p last">LCMS: RT = 0.732 min, [M + H]<sup>+</sup> = 496.1; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.62 (s, 1H), 8.38 (br d, <i>J</i> = 3.6 Hz, 1H), 8.32 (s, 1H), 8.29 (s, 1H), 8.12 (d, <i>J</i> = 1.9 Hz, 1H), 7.80 (br t, <i>J</i> = 5.3 Hz, 1H), 7.66 (dd, <i>J</i> = 1.9, 8.2 Hz, 1H), 7.51 (br d, <i>J</i> = 7.8 Hz, 1H), 7.30 (d, <i>J</i> = 8.3 Hz, 1H), 7.24 (dd, <i>J</i> = 4.8, 7.6 Hz, 1H), 4.98 (s, 2H), 3.03 (q, <i>J</i> = 6.5 Hz, 2H), 2.68 (br t, <i>J</i> = 6.9 Hz, 2H), 2.42 (s, 3H); HRMS (ESI/Q-TOF) RT = 1.93 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 496.0613; found 496.0606.</div></div><div id="sec4_14_2_2_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-(<i>N</i>-(2-(pyridin-4-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>8</b>)</h5><div class="NLM_p last">LCMS: RT = 0.752 min, [M + H]<sup>+</sup> = 496.0; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.58 (s, 1H), 8.40 (br d, <i>J</i> = 5.0 Hz, 2H), 8.29 (s, 1H), 8.13 (d, <i>J</i> = 2.1 Hz, 1H), 7.79 (t, <i>J</i> = 5.7 Hz, 1H), 7.65 (dd, <i>J</i> = 2.2, 8.2 Hz, 1H), 7.31 (d, <i>J</i> = 8.3 Hz, 1H), 7.11 (d, <i>J</i> = 5.8 Hz, 2H), 4.98 (s, 2H), 3.08–3.03 (m, 2H), 2.70–2.66 (m, 2H), 2.41 (s, 3H); HRMS (ESI/Q-TOF) RT = 1.91 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 496.0613; found 496.0606.</div></div><div id="sec4_14_2_2_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 2-(4,5-Dichloro-6-oxo-pyridazin-1-yl)-<i>N</i>-[3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]acetamide (<b>9</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.96 (s, 1H), 8.52 (d, <i>J</i> = 5.1 Hz, 1H), 8.14 (s, 1H), 7.95 (dd, <i>J</i> = 8.3, 1.8 Hz, 1H), 7.92 (s, 1H), 7.79 (d, <i>J</i> = 2.3 Hz, 1H), 7.71 (td, <i>J</i> = 7.7, 1.7 Hz, 1H), 7.54 (d, <i>J</i> = 7.7 Hz, 1H), 7.38 (t, <i>J</i> = 8.0 Hz, 1H), 7.29 (s, 1H), 7.21 (d, <i>J</i> = 7.8 Hz, 1H), 5.06 (s, 2H), 3.41 (t, <i>J</i> = 6.2 Hz, 2H), 3.04 (t, <i>J</i> = 6.3 Hz, 2H); HRMS (ESI/Q-TOF) RT = 1.84 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>18</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 482.0456; found 482.0447.</div></div><div id="sec4_14_2_2_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methoxy-3-(<i>N</i>-(2-(pyridin-2-yl)ethyl) sulfamoyl)phenyl)acetamide (<b>10</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.45 (s, 1H), 8.45 (d, <i>J</i> = 3.9 Hz, 1H), 8.29 (s, 1H), 8.00 (d, <i>J</i> = 2.6 Hz, 1H), 7.77 (dd, <i>J</i> = 2.6, 9.0 Hz, 1H), 7.67 (dt, <i>J</i> = 1.8, 7.6 Hz, 1H), 7.31 (t, <i>J</i> = 5.8 Hz, 1H), 7.23–7.19 (m, 1H), 7.18 (d, <i>J</i> = 2.1 Hz, 1H), 7.17–7.14 (m, 1H), 4.95 (s, 2H), 3.82 (s, 3H), 3.17–3.11 (m, 2H), 2.83 (t, <i>J</i> = 7.2 Hz, 2H); HRMS (ESI/Q-TOF) RT = 1.79 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>5</sub>S 512.0562; found 512.0556.</div></div><div id="sec4_14_2_2_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-ethyl-3-(N-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>11</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.60 (s, 1H), 8.42 (br d, <i>J</i> = 4.3 Hz, 1H), 8.29 (s, 1H), 8.09 (d, <i>J</i> = 1.8 Hz, 1H), 7.82 (br t, <i>J</i> = 5.6 Hz, 1H), 7.74 (dd, <i>J</i> = 1.8, 8.3 Hz, 1H), 7.65 (dt, <i>J</i> = 1.5, 7.6 Hz, 1H), 7.38 (d, <i>J</i> = 8.3 Hz, 1H), 7.22–7.11 (m, 2H), 4.98 (s, 2H), 3.20–3.14 (m, 2H), 2.90–2.81 (m, 4H), 1.16 (t, <i>J</i> = 7.5 Hz, 3H); HRMS (ESI/Q-TOF) RT = 2.16 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 510.0769; found 510.0760.</div></div><div id="sec4_14_2_2_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(2,4-dimethyl-5-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>12</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 9.82 (s, 1H), 8.40 (br d, <i>J</i> = 4.0 Hz, 1H), 8.29 (s, 1H), 7.90 (s, 1H), 7.69 (br t, <i>J</i> = 5.6 Hz, 1H), 7.63 (dt, <i>J</i> = 1.5, 7.6 Hz, 1H), 7.22 (s, 1H), 7.20–7.15 (m, 1H), 7.13 (d, <i>J</i> = 7.8 Hz, 1H), 5.03 (s, 2H), 3.12 (q, <i>J</i> = 6.9 Hz, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H), 2.25 (s, 3H); HRMS (ESI/Q-TOF) RT = 1.96 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 510.0769; found 510.0762.</div></div><div id="sec4_14_2_2_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(3,4-dimethyl-5-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>13</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.49 (s, 1H), 8.43–8.36 (m, 1H), 8.29 (s, 1H), 7.97 (d, <i>J</i> = 1.7 Hz, 1H), 7.76 (t, <i>J</i> = 5.7 Hz, 1H), 7.66–7.61 (m, 2H), 7.17 (dd, <i>J</i> = 5.1, 6.9 Hz, 1H), 7.11 (d, <i>J</i> = 7.8 Hz, 1H), 4.97 (s, 2H), 3.17 (q, <i>J</i> = 6.9 Hz, 2H), 2.82 (t, <i>J</i> = 7.2 Hz, 2H), 2.33–2.24 (m, 6H);HRMS (ESI/Q-TOF) RT = 2.08 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 510.0769; found 510.0762.</div></div><div id="sec4_14_2_2_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(2,4-dimethyl-3-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>14</b>)</h5><div class="NLM_p"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 9.85 (s, 1H), 8.39 (br d, <i>J</i> = 4.0 Hz, 1H), 8.29 (s, 1H), 7.70 (br t, <i>J</i> = 5.6 Hz, 1H), 7.61 (dt, <i>J</i> = 1.8, 7.6 Hz, 1H), 7.36 (br d, <i>J</i> = 8.1 Hz, 1H), 7.22–7.16 (m, 1H), 7.16–7.12 (m, 1H), 7.09 (br d, <i>J</i> = 7.7 Hz, 1H), 5.01 (s, 2H), 3.20–3.12 (m, 2H), 2.81 (br t, <i>J</i> = 7.2 Hz, 2H), 2.56 (s, 3H), 2.42 (s, 3H); HRMS (ESI/Q-TOF) RT = 1.89 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 510.0769; found 510.0764; see <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.</div><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Pyridine Analogs (A) <b>15</b>, (B) <b>16</b>, and (C) <b>17</b> and (D) piperidine <b>18</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_14_2_2_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> <i>N</i>-(6-Chloro-5-methylpyridin-2-yl)acetamide (<b>37</b>)</h5><div class="NLM_p last">To a mixture of 6-chloro-5-methylpyridin-2-amine (CAS 442129-37-5, 6 g, 42.08 mmol) and pyridine (4.99 g, 63.12 mmol, 5.09 mL) in DCM (50 mL) was added acetyl chloride (4.29 g, 54.70 mmol, 3.90 mL), and the mixture was stirred at 20 °C for 12 h. The mixture was diluted with water and extracted twice with ethyl acetate. The combined organic layers were washed with saturated aqueous sodium chloride, dried over sodium sulfate, filtered to remove insoluble materials, and volatiles removed to provide the title compound as a yellow solid (8.1 g, assumed quant.). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C8H10ClN2O 185.4; found 185.4; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 8.12–8.02 (m, 1H), 8.02–7.83 (m, 1H), 7.61–7.50 (m, 1H), 2.36–2.32 (m, 3H), 2.20–2.16 (m, 3H).</div></div><div id="sec4_14_2_2_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> <i>N</i>-(6-(Benzylthio)-5-methylpyridin-2-yl)acetamide (<b>38</b>)</h5><div class="NLM_p last">To a solution of <i>N</i>-(6-chloro-5-methylpyridin-2-yl)acetamide (<b>37</b>, 10.1 g, 54.71 mmol) in DMF (100 mL) was added potassium carbonate (30.24 g, 218.82 mmol) and phenylmethanethiol (20.38 g, 164.12 mmol, 19.23 mL) at 20 °C. The mixture was stirred at 60 °C for 12 h, diluted with water, and extracted twice with ethyl acetate. The combined organic layers were washed with water, saturated aqueous sodium chloride, dried over anhydrous sodium sulfate, insoluble materials removed by filtration, volatiles removed under reduced pressure, and the residue was purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether to afford the title compound as a yellow solid (9.33 g, 63% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 7.88–7.77 (m, 1H), 7.77–7.63 (m, 1H), 7.42–7.37 (m, 2H), 7.36–7.29 (m, 3H), 7.28–7.23 (m, 1H), 4.49–4.35 (m, 2H), 2.29–2.21 (m, 3H), 2.21–2.18 (m, 3H); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.33 (s, 1H), 7.73 (br d, <i>J</i> = 8.1 Hz, 1H), 7.49–7.43 (m, 3H), 7.33–7.28 (m, 2H), 7.26–7.21 (m, 1H), 4.50 (s, 2H), 2.13 (s, 3H), 2.11 (s, 3H).</div></div><div id="sec4_14_2_2_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 6-Acetamido-3-methylpyridine-2-sulfonyl chloride (<b>39</b>)</h5><div class="NLM_p last">To a solution of <i>N</i>-(6-(benzylthio)-5-methylpyridin-2-yl)acetamide (<b>38</b>, 9.33 g, 34.26 mmol) in DCM (100 mL) were added water (20 mL) and sulfuryl chloride (32.36 g, 239.79 mmol, 23.97 mL) at 0 °C. The reaction mixture slowly warmed to 20 °C and stirred for 2 h. Water was added, and the mixture was extracted three times with dichloromethane. The combined organic layers were washed with saturated aqueous sodium chloride, dried over sodium sulfate, insoluble materials removed by filtration, and volatiles removed under reduced pressure to afford the title compound as a yellow oil (11.6 g, assumed quant.).</div></div><div id="sec4_14_2_2_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> <i>N</i>-[5-Methyl-6-[2-(2-pyridyl)ethylsulfamoyl]-2-pyridyl]acetamide (<b>40</b>)</h5><div class="NLM_p last">To a solution of 6-acetamido-3-methyl-pyridine-2-sulfonyl chloride (<b>39</b>, 6 g, 24.13 mmol) in THF (60 mL) were added triethylamine (7.32 g, 72.38 mmol, 10.07 mL) and 2-(2-pyridyl)ethanamine (4.42 g, 36.19 mmol, 4.33 mL). The mixture was stirred at 25 °C for 12 h, diluted with water, and extracted twice with ethyl acetate. The combined organic layers were washed with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials were removed by filtration, volatiles removed under reduced pressure, and the residue purified by column chromatography on silica gel, eluting with a gradient of ethyl acetate in petroleum ether to provide the title compound as a yellow solid (4.7 g, 58% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>15</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub>S 335.1; found 335.3.</div></div><div id="sec4_14_2_2_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 6-Amino-3-methyl-<i>N</i>-[2-(2-pyridyl)ethyl]pyridine-2-sulfonamide (<b>41</b>)</h5><div class="NLM_p last">A mixture of <i>N</i>-[5-methyl-6-[2-(2-pyridyl)ethylsulfamoyl]-2-pyridyl]acetamide (<b>40</b>, 4.7 g, 14.06 mmol, 1 equiv) in hydrogen chloride (6 M in water, 50 mL) was stirred at 80 °C for 12 h. Volatiles were removed under reduced pressure to provide the title compound as a white solid (4.6 g, HCl salt, 99% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.87–8.77 (m, 1H), 8.54 (dt, <i>J</i> = 1.5, 7.9 Hz, 1H), 8.11 (br s, 1H), 8.03–7.90 (m, 2H), 7.50 (d, <i>J</i> = 8.4 Hz, 1H), 6.73 (d, <i>J</i> = 8.6 Hz, 1H), 3.61 (q, <i>J</i> = 5.9 Hz, 2H), 3.30 (t, <i>J</i> = 6.5 Hz, 2H), 2.29 (s, 3H).</div></div><div id="sec4_14_2_2_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(5-methyl-6-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)pyridin-2-yl)acetamide (<b>15</b>)</h5><div class="NLM_p last">Prepared in a manner analogous to compound <b>6</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.05 (s, 1H), 8.51 (d, <i>J</i> = 4.2 Hz, 1H), 8.09 (br d, <i>J</i> = 2.2 Hz, 1H), 7.97 (br t, <i>J</i> = 5.7 Hz, 1H), 7.89 (d, <i>J</i> = 8.3 Hz, 1H), 7.74 (dt, <i>J</i> = 1.8, 7.6 Hz, 1H), 7.31–7.23 (m, 2H), 5.16 (br s, 2H), 3.62–3.50 (m, 2H), 2.99 (t, <i>J</i> = 7.3 Hz, 2H), 2.53 (s, 3H); HRMS (ESI/Q-TOF) RT = 2.08 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C19H19Cl2N6O4S 497.0565; found 497.0558.</div></div><div id="sec4_14_2_2_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 3-(Benzylthio)-2-methyl-5-nitropyridine (<b>42</b>)</h5><div class="NLM_p last">A mixture of 3-bromo-2-methyl-5-nitropyridine (2 g, 9.22 mmol) was added to phenylmethanethiol (1.49 g, 11.98 mmol, 1.40 mL), <i>N</i>,<i>N</i>-diethyl isopropylamine (2.38 g, 18.43 mmol, 3.21 mL), tris(dibenzylideneacetone)dipalladium (421.95 mg, 460.79 μmol), and 4,5-bis(diphenylphosphino)-9,9-dimethylxanthene (533.24 mg, 921.57 μmol) in dioxane (30 mL) and was heated at reflux (100 °C) for 12 h. The mixture was cooled to room temperature, diluted with water, and extracted three times with ethyl acetate. The combined organic layers were washed with water, saturated aqueous sodium chloride, and dried over sodium sulfate. The insoluble materials were removed by filtration, volatiles removed under reduced pressure, and the residue purified by column chromatography on silica gel, eluting with a gradient of ethyl acetate in petroleum ether, to afford the title compound as a yellow solid (2.6 g, assumed quant.). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 9.04–8.91 (m, 1H), 8.25–8.04 (m, 1H), 7.33–7.22 (m, 5H), 4.23–4.06 (m, 2H), 2.62–2.53 (m, 3H).</div></div><div id="sec4_14_2_2_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 2-Methyl-5-nitropyridine-3-sulfonyl chloride (<b>43</b>)</h5><div class="NLM_p last">To 3-(benzylthio)-2-methyl-5-nitropyridine (<b>42</b>, 1.0 g, 3.84 mmol) in DCM (20 mL) was added water (4 mL) and sulfuryl chloride (3.63 g, 26.89 mmol, 2.69 mL) at 0 °C. The reaction mixture slowly warmed to 20 °C and stirred for 30 min. The volatiles were removed under reduced pressure, and the residue was suspended in dichloromethane, dried over sodium sulfate. The insoluble materials were removed by filtration, and volatiles removed under reduced pressure to afford the title compound as a yellow oil (1.0 g, assumed quant.), which was used without further manipulation.</div></div><div id="sec4_14_2_2_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 1-Methyl-4-((2-methyl-5-nitropyridin-3-yl)sulfonyl)-1,4-diazepane (<b>44</b>)</h5><div class="NLM_p last">A mixture of 2-methyl-5-nitropyridine-3-sulfonyl chloride (<b>43</b>, 1.0 g, 4.23 mmol), 1-methyl-1,4-diazepane (723.83 mg, 6.34 mmol, 788.49 μL), and triethylamine (1.28 g, 12.68 mmol, 1.76 mL) in THF (20 mL) was stirred for 12 h at 20 °C. Volatiles were removed under reduced pressure, and the residue was purified by prep-HPLC (column: Kromasil 250*50 mm*10 μm; mobile phase: [water(0.225%FA)-ACN]; B%: 5%–30%,15 min) to afford the title compound as a brown solid (430 mg, 32% yield).</div></div><div id="sec4_14_2_2_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 6-Methyl-5-((4-methyl-1,4-diazepan-1-yl)sulfonyl)pyridin-3-amine (<b>45</b>)</h5><div class="NLM_p last">A mixture of 1-methyl-4-((2-methyl-5-nitropyridin-3-yl)sulfonyl)-1,4-diazepane (<b>44</b>, 100 mg, 318.11 μmol) ammonium chloride (170.16 mg, 3.18 mmol) and Zn dust (83.20 mg, 1.27 mmol) in THF (2 mL) was stirred at 66 °C for 12 h. After cooling to room temperature, the insoluble materials were removed by filtration through Celite. Aqueous hydrochloric acid was added to the filtrate, and volatiles removed under reduced pressure to afford the title compound as a yellow solid (75 mg, di-HCl salt, 66% yield).</div></div><div id="sec4_14_2_2_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(6-methyl-5-((4-methyl-1,4-diazepan-1-yl)sulfonyl)pyridin-3-yl)acetamide (<b>16</b>)</h5><div class="NLM_p last">A mixture of 6-methyl-5-((4-methyl-1,4-diazepan-1-yl)sulfonyl)pyridin-3-amine (<b>45</b>, 75 mg, 209.91 μmol, 2 HCl salt), 2-(4,5-dichloro-6-oxo-pyridazin-1-yl)acetic acid (32, 56.17 mg, 251.89 μmol), 2-chloro-1-methyl-pyridin-1-ium iodide (80.44 mg, 314.86 μmol), and <i>N</i>,<i>N</i>-diethyl isopropylamine (135.64 mg, 1.05 mmol, 182.81 μL, 5 equiv) in THF (5 mL) was stirred at 20 °C for 12 h. Volatiles were removed under reduced pressure, and the residue was purified by prep-HPLC (column: Shim-pack C18 150*25*10 μm; mobile phase: [water(0.225%FA)-ACN];B%: 3%–33%,10 min) to afford the title compound as a yellow solid (10 mg, 9% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ = 8.75–8.70 (m, 1H), 8.65–8.60 (m, 1H), 8.14–8.10 (m, 1H), 5.12–5.06 (m, 2H), 3.74–3.67 (m, 2H), 3.58–3.51 (m, 2H), 3.24–3.18 (m, 4H), 2.80–2.77 (m, 3H), 2.76–2.72 (m, 3H), 2.20–2.07 (m, 2H); HRMS (ESI/Q-TOF) RT = 1.74 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>23</sub>C<sub>l2</sub>N<sub>6</sub>O<sub>4</sub>S 489.0878; found 489.0872.</div></div><div id="sec4_14_2_2_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 2-Chloro-5-methyl-pyridine-4-sulfonyl chloride (<b>46</b>)</h5><div class="NLM_p last">2-Chloro-5-methyl-pyridin-4-amine (500 mg, 3.51 mmol), TFA (5 mL), and hydrochloric acid (12 M in water, 2.5 mL) were treated with sodium nitrite (723.40 mg, 10.48 mmol) in water (3.5 mL) and stirred for 1 h at 0 °C, and the insoluble materials were removed by filtration. Sulfur dioxide gas was bubbled through a mixture of copper(I)chloride (34.95 mg, 353.00 μmol) and copper(II)chloride (234.14 mg, 1.74 mmol) in acetic acid (30 mL) for 30 min at 0 °C. The mixtures were combined and stirred at 0 °C for 1.5 h, diluted with dichloromethane, washed twice with ice–water, saturated aqueous sodium hydrogen carbonate, and saturated aqueous sodium chloride, and dried over sodium sulfate. The insoluble materials were removed by filtration, and volatiles were removed under reduced pressure to afford the title compound as a yellow oil (530 mg, 67% yield).</div></div><div id="sec4_14_2_2_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 2-Chloro-<i>N</i>,<i>N</i>,5-trimethyl-pyridine-4-sulfonamide (<b>47</b>)</h5><div class="NLM_p last">A mixture of 2-chloro-5-methyl-pyridine-4-sulfonyl chloride (<b>46</b>, 530 mg, 2.34 mmol) and <i>N</i>-methylmethanamine (2 M in THF, 5 mL) was stirred for 2 h at 15 °C. The mixture was diluted with ethyl acetate, washed with saturated aqueous sodium chloride, and dried over sodium sulfate. The insoluble materials removed by filtration, volatiles removed under reduced pressure, and the residue was purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petreolum ether to afford the title compound as a white solid (300 mg, 55% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 8.41 (s, 1H), 7.69 (s, 1H), 2.90 (s, 6H), 2.58 (s, 3H).</div></div><div id="sec4_14_2_2_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> <i>tert</i>-Butyl <i>N</i>-[4-(Dimethylsulfamoyl)-5-methyl-2-pyridyl]carbamate (<b>48</b>)</h5><div class="NLM_p last">A mixture of 2-chloro-<i>N</i>,<i>N</i>,5-trimethyl-pyridine-4-sulfonamide (<b>47</b>, 250 mg, 1.07 mmol), <i>tert</i>-butyl carbamate (623.91 mg, 5.33 mmol), cesium carbonate (624.70 mg, 1.92 mmol), 4,5-bis(diphenylphosphino)-9,9-dimethylxanthene (73.96 mg, 127.82 μmol), and tris(dibenzylideneacetone)dipalladium (39.02 mg, 42.61 μmol) in dioxane (5 mL) was stirred at 80 °C for 4 h. The mixture was diluted with water and extracted three times with ethyl acetate. The combined organic layers were washed with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials were removed by filtration, volatiles removed under reduced pressure, and the residue was triturated with petroleum ether:ethyl acetate (1:1). The title compound was isolated by filtration as a yellow solid (260 mg, 77% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.14 (s, 1H), 8.36 (s, 1H), 8.15–8.12 (m, 1H), 2.79 (s, 6H), 2.44 (s, 3H), 1.49 (s, 9H).</div></div><div id="sec4_14_2_2_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 2-Amino-<i>N</i>,<i>N</i>,5-trimethyl-pyridine-4-sulfonamide (<b>49</b>)</h5><div class="NLM_p last"><i>tert</i>-Butyl <i>N</i>-[4-(dimethylsulfamoyl)-5-methyl-2-pyridyl]carbamate (<b>48</b>, 240 mg, 760.97 μmol) in hydrochloric acid (2 M in methanol, 5 mL) was stirred at 25 °C for 12 h. Volatiles were removed under reduced pressure to afford the title compound as a yellow solid (300 mg, HCl salt, assumed quant.). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>8</sub>H<sub>14</sub>N<sub>3</sub>O<sub>2</sub>S 216.1; found 216.1.</div></div><div id="sec4_14_2_2_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> 2-(4,5-Dichloro-6-oxo-pyridazin-1-yl)-<i>N</i>-[4-(dimethylsulfamoyl)-5-methyl-2-pyridyl]acetamide (<b>17</b>)</h5><div class="NLM_p last">Compound <b>17</b> was prepared analogously to compound <b>16</b>, using <b>49</b> and purified by prep-HPLC (column: Phenomenex Synergi C18 150*25*10 μm; mobile phase: [water (0.225%FA)-ACN]; B%: 26%–56%,10 min) to afford the title compound as a white solid (45 mg, 38% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.29 (s, 1H), 8.49 (s, 1H), 8.32 (s, 1H), 8.29 (s, 1H), 5.07 (s, 2H), 2.78 (s, 6H), 2.48 (s, 3H); HRMS (ESI/Q-TOF) RT 2.61 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>16</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 420.0300; found 420.0295.</div></div><div id="sec4_14_2_2_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> <i>tert</i>-Butyl 3-(2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)acetamido)piperidine-1-carboxylate (<b>50</b>)</h5><div class="NLM_p last">A mixture of <i>tert</i>-butyl 3-aminopiperidine-1-carboxylate (179.61 mg, 896.80 μmol), 2-(4,5-dichloro-6-oxopyridazin-1(6H)-yl)acetic acid (<b>32</b>, 200 mg, 896.80 μmol), <i>N</i>,<i>N</i>-diethyl isopropylamine (347.72 mg, 2.69 mmol, 468.62 μL), and 1-[bis(dimethylamino)methylene]-1H-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (511.49 mg, 1.35 mmol) in DMF (4 mL) was stirred at 25 °C for 12 h. Insoluble materials were removed by filtration, volatiles were removed under reduced pressure, and the residue was purified by prep-HPLC (column: Phenomenex Luna C18 150 *40 mm 10 μm; mobile phase: [water (0.225%FA)-ACN]; B%: 28%–58%, 8.5 min) to afford the title compound as a white solid (220 mg, 61% yield). LRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> = 305.1; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.23 (s, 1H), 8.19 (br d, <i>J</i> = 7.3 Hz, 1H), 4.74 (d, <i>J</i> = 6.4 Hz, 2H), 3.58 (br d, <i>J</i> = 2.9 Hz, 2H), 3.32 (s, 1H), 2.90 (br d, <i>J</i> = 2.8 Hz, 2H), 1.80 (br s, 2H), 1.68 (br d, <i>J</i> = 8.3 Hz, 2H), 1.39 (s, 9H).</div></div><div id="sec4_14_2_2_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(piperidin-3-yl)acetamide (<b>51</b>)</h5><div class="NLM_p last">To <i>tert</i>-butyl 3-(2-(4,5-dichloro-6-oxopyridazin-1(6H)-yl)acetamido)piperidine-1-carboxylate (<b>50</b>, 120 mg, 296.10 μmol) in DCM (1 mL) was added hydrochloric acid (4 M in methanol, 4 mL). The mixture was stirred at 25 °C for 1 h. Volatiles were removed under reduced pressure to afford the title compound as a white solid (100 mg, HCl salt, 98% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>11</sub>H<sub>15</sub>C<sub>l2</sub>N<sub>4</sub>O<sub>2</sub> 305.1; found 304.8.</div></div><div id="sec4_14_2_2_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(1-(<i>N</i>,<i>N</i>-dimethylsulfamoyl)piperidin-3-yl)acetamide (<b>18</b>)</h5><div class="NLM_p last">A mixture of 2-(4,5-dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(piperidin-3-yl)acetamide (<b>51</b>, 100 mg, 292.72 μmol, HCl salt), <i>N</i>,<i>N</i>-dimethylsulfamoyl chloride (63.05 mg, 439.08 μmol, 47.05 μL), and triethylamine (88.86 mg, 878.17 μmol, 122.23 μL) in THF (2 mL) was stirred at 25 °C for 2 h. Volatiles were removed under reduced pressure, and the residue was purified by prep-TLC on silica gel, eluting with ethyl acetate to afford the title compound as a white solid (30 mg, 25% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.33 (d, <i>J</i> = 7.3 Hz, 1H), 8.31 (s, 1H), 4.81 (d, <i>J</i> = 4.9 Hz, 2H), 3.83–3.66 (m, 1H), 3.56 (dd, <i>J</i> = 3.9, 11.8 Hz, 1H), 3.00–2.91 (m, 1H), 2.80 (s, 6H), 2.77–2.66 (m, 2H), 1.92–1.78 (m, 2H), 1.64–1.51 (m, 1H), 1.49–1.37 (m, 1H); HRMS (ESI/Q-TOF) RT 2.18 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>20</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 412.0613; found 412.0609.</div></div><div id="sec4_14_2_2_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> <i>N</i>-[4-Methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]formamide (<b>52</b>)</h5><div class="NLM_p last">A mixture of 5-amino-2-methyl-<i>N</i>-[2-(2-pyridyl)ethyl]benzenesulfonamide (<b>36</b>, 1 g, 3.43 mmol, 1 equiv) and formic acid (1.90 g, 41.19 mmol, 1.55 mL) in toluene (10 mL) was stirred at 115 C for 2 h. The mixture was cooled to room temperature, diluted with water, and extracted four times with ethyl acetate. The combined organics were washed repeatedly with saturated aqueous sodium chloride and dried over sodium sulfate The insoluble materials were removed by filtration, and volatiles removed under reduced pressure to afford the title compound as a light-yellow oil (1.23 g, assumed quant.). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.39 (s, 1H), 8.43–8.40 (m, 1H), 8.30 (d, <i>J</i> = 1.8 Hz, 1H), 8.12 (d, <i>J</i> = 2.1 Hz, 1H), 7.79 (br t, <i>J</i> = 5.6 Hz, 1H), 7.71 (dd, <i>J</i> = 2.3, 8.1 Hz, 1H), 7.65 (dt, <i>J</i> = 1.8, 7.7 Hz, 1H), 7.30 (d, <i>J</i> = 8.3 Hz, 1H), 7.21–7.13 (m, 2H), 3.22–3.12 (m, 2H), 2.83 (t, <i>J</i> = 7.4 Hz, 2H), 2.44 (s, 3H).</div></div><div id="sec4_14_2_2_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 2-Methyl-5-(methylamino)-<i>N</i>-[2-(2-pyridyl)ethyl]benzenesulfonamide (<b>53</b>)</h5><div class="NLM_p last">To a mixture of <i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]formamide (<b>52</b>, 1 g, 3.13 mmol) in THF (10 mL) was added borane dimethyl sulfide complex (626.22 μL) dropwise at 0 °C over a period of 15 min. The mixture was heated to 60 °C for 12 h. The reaction mixture was diluted with water and extracted 4 times with ethyl acetate. The combined organic phases were washed three times with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials were removed by filtration, volatiles removed under reduced pressure, and the residue purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether to afford the title compound as an off-white solid (565 mg, 59% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C15H19N3O2S 306.1; found 306.0; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.45–8.41 (m, 1H), 8.45–8.41 (m, 1H), 7.66 (dt, <i>J</i> = 1.9, 7.7 Hz, 1H), 7.53 (t, <i>J</i> = 5.8 Hz, 1H), 7.21–7.13 (m, 2H), 7.07–7.03 (m, 2H), 6.62 (dd, <i>J</i> = 2.6, 8.2 Hz, 1H), 5.90 (q, <i>J</i> = 4.9 Hz, 1H), 3.15–3.08 (m, 2H), 2.86–2.80 (m, 2H), 2.66 (d, <i>J</i> = 5.0 Hz, 3H), 2.33 (s, 3H).</div></div><div id="sec4_14_2_2_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 2-(4,5-Dichloro-6-oxo-pyridazin-1-yl)-<i>N</i>-methyl-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]acetamide (<b>19</b>)</h5><div class="NLM_p">Prepared in an analogous manner to <b>6</b> using <b>53</b>. <sup>1</sup>H NMR (400 MHz, METHANOL-<i>d</i><sub>4</sub>) δ = 8.27 (br d, <i>J</i> = 4.2 Hz, 1H), 7.95 (s, 1H), 7.89 (br s, 1H), 7.62 (br t, <i>J</i> = 7.0 Hz, 1H), 7.52 (br d, <i>J</i> = 7.2 Hz, 1H), 7.42 (br d, <i>J</i> = 7.8 Hz, 1H), 7.21–7.10 (m, 2H), 4.64 (s, 2H), 3.31–3.27 (m, 2H), 3.25 (br d, <i>J</i> = 1.7 Hz, 3H), 2.86 (br t, <i>J</i> = 7.0 Hz, 2H), 2.51 (s, 3H); HRMS (ESI/Q-TOF) RT 1.98 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 510.0769; found 510.0763; see <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>.</div><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of (A) <b>19</b>, (B) <b>20</b>, and (C) <b>21</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_14_2_2_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> Methyl (<i>S</i>)-2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)propanoate (<b>54</b>)</h5><div class="NLM_p last">To a solution of 4,5-dichloropyridazin-3(2H)-one (5 g, 30.31 mmol) and methyl (2<i>R</i>)-2-hydroxypropanoate (6.31 g, 60.61 mmol, 5.79 mL) in THF (50 mL) was added triphenylphosphine (15.90 g, 60.61 mmol). The solution was stirred at 20 °C for 10 min, cooled to 0 °C, and diisopropyl azodicarboxylate (12.26 g, 60.61 mmol, 11.79 mL) was added. The reaction mixture was stirred at 20 °C for 2h. Volatiles were removed under reduced pressure, and the residue was purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether to provide the title compound as a white solid (7.6 g, 99% yield). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 7.84 (s, 1H), 5.58 (q, <i>J</i> = 7.3 Hz, 1H), 3.76 (s, 3H), 1.70 (d, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_14_2_2_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> (<i>S</i>)-2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)propanoic acid (<b>55</b>)</h5><div class="NLM_p last">Methyl (<i>S</i>)-2-(4,5-dichloro-6-oxopyridazin-1(6H)-yl)propanoate (<b>54</b>, 7.6 g, 30.27 mmol) was suspended in aqueous hydrogen chloride (5% w/v, 150 mL), and the mixture was stirred at 105 °C for 1.5 h. The reaction was cooled to room temperature, and the resulting solid was isolated by filtration and washed with water to afford the title compound as a white solid (5.7 g, 79% yield).</div></div><div id="sec4_14_2_2_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> (<i>S</i>)-2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)propanamide (<b>20</b>)</h5><div class="NLM_p last">A mixture of (2<i>S</i>)-2-(4,5-dichloro-6-oxo-pyridazin-1-yl)propanoic acid (<b>55</b>, 100 mg, 421.87 μmol), 5-amino-2-methyl-<i>N</i>-[2-(2-pyridyl)ethyl]benzenesulfonamide (<b>36</b>, 122.92 mg, 421.87 μmol), 2,4,6-tripropyl-1,3,5,2λ5,4λ5,6λ5-trioxatriphosphinane 2,4,6-trioxide (295.31 mg, 464.06 μmol, 275.99 μL, 50% w/w in ethyl acetate), and <i>N</i>,<i>N</i>-diethyl isopropylamine (65.43 mg, 506.24 μmol, 88.18 μL) in DMF (1 mL) was stirred at 25 °C for 3 h. Volatiles were removed under reduced pressure, and the residue was purified by prep-HPLC (column: Phenomenex Synergi C18 150*25*10 μm; mobile phase: [water(0.225%FA)-ACN]; B%: 13%–43%, 10 min) to afford the title compound as a white solid (35 mg, 16% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.41 (s, 1H), 8.40 (ddd, <i>J</i> = 4.9, 1.9, 1.0 Hz, 1H), 8.32 (s, 1H), 8.07 (d, <i>J</i> = 2.3 Hz, 1H), 7.76 (t, <i>J</i> = 5.8 Hz, 1H), 7.71 (dd, <i>J</i> = 8.3, 2.3 Hz, 1H), 7.64 (td, <i>J</i> = 7.6, 1.9 Hz, 1H), 7.29 (d, <i>J</i> = 8.4 Hz, 1H), 7.17 (ddd, <i>J</i> = 7.6, 4.8, 1.2 Hz, 1H), 7.13 (dt, <i>J</i> = 7.8, 1.1 Hz, 1H), 5.44 (q, <i>J</i> = 7.0 Hz, 1H), 3.14 (td, <i>J</i> = 7.2, 5.7 Hz, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H), 1.61 (d, <i>J</i> = 7.0 Hz, 3H); 13C{1H} NMR (101 MHz, DMSO) δ 167.93, 158.24, 156.02, 148.97, 138.61, 136.65, 136.40, 136.19, 135.68, 132.87, 132.44, 131.17, 123.21, 122.78, 121.59, 119.52, 59.23, 41.93, 37.28, 19.05, 15.74; SFC (Chiralpac AD-3, 50 × 4.6 mm I.D., 3 μM, mobile phase: gradient elution 5%–40% 2-propanol (0.05% diethyl amine) in CO<sub>2</sub>, flow rate 3 mL/min, PDA detector, column temp 35 °C, back pressure 100 bar) RT = 1.029 min, 96.8% ee; HRMS (ESI/Q-TOF) RT = 2.18 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 510.0769; found 510.0759.</div></div><div id="sec4_14_2_2_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> (<i>R</i>)-2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)propanamide (<b>21</b>)</h5><div class="NLM_p last">Compound <b>21</b> was prepared analogously to <b>20</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.41 (s, 1H), 8.40 (ddd, <i>J</i> = 4.9, 1.9, 1.0 Hz, 1H), 8.32 (s, 1H), 8.07 (d, <i>J</i> = 2.3 Hz, 1H), 7.76 (t, <i>J</i> = 5.8 Hz, 1H), 7.71 (dd, <i>J</i> = 8.2, 2.3 Hz, 1H), 7.64 (td, <i>J</i> = 7.7, 1.9 Hz, 1H), 7.29 (d, <i>J</i> = 8.4 Hz, 1H), 7.17 (ddd, <i>J</i> = 7.6, 4.8, 1.2 Hz, 1H), 7.13 (dd, <i>J</i> = 7.8, 1.2 Hz, 1H), 5.44 (q, <i>J</i> = 7.0 Hz, 1H), 3.13 (dd, <i>J</i> = 7.6, 5.9 Hz, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H), 1.61 (d, <i>J</i> = 7.1 Hz, 3H); 13C{1H} NMR (101 MHz, DMSO) δ 167.93, 158.24, 156.02, 148.97, 138.61, 136.65, 136.40, 136.19, 135.68, 132.87, 132.44, 131.17, 123.21, 122.77, 121.59, 119.52, 59.23, 41.93, 37.28, 19.05, 15.74; SFC (Chiralpac AD-3, 50 × 4.6 mm I.D., 3 μM, mobile phase: gradient elution 5%–40% 2-propanol (0.05% diethylamine) in CO<sub>2</sub>, flow rate 3 mL/min, PDA detector, column temp 35 °C, back pressure 100 bar) RT = 0.642 min, 97.8% ee; HRMS (ESI/Q-TOF) RT = 2.17 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>C<sub>l2</sub>N<sub>5</sub>O<sub>4</sub>S 510.0769; found 510.0759.</div></div><div id="sec4_14_2_2_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> (<i>rac</i>)-Ethyl 2-(4,5-Dichloro-6-oxopyridazin-1(6H)-yl)propanoate (<b>56</b>)</h5><div class="NLM_p last">To a solution of 4,5-dichloropyridazin-3(2H)-one (CAS 932-22-9, 1 g, 6.06 mmol) and potassium carbonate (1.68 g, 12.12 mmol) in DMF (10 mL) was added ethyl 2-bromopropanoate (1.21 g, 6.67 mmol, 868.36 μL) at 25 °C, and the mixture was stirred at 50 °C for 2 h. The mixture was diluted with ethyl acetate and water, layers separated, and the organic phase washed with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials removed by filtration, volatiles removed under reduced pressure, and the residue purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether, to afford the title compound as a white oil (1.14 g, 71% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>9</sub>H<sub>10</sub>C<sub>l2</sub>N<sub>2</sub>O<sub>3</sub> 265.0; found 264.9; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ ppm 1.19 (t, <i>J</i> = 7.13 Hz, 4 H) 1.62 (d, <i>J</i> = 7.25 Hz, 4 H) 1.97 (s, 1 H) 4.15 (q, <i>J</i> = 7.09 Hz, 2 H) 5.47 (q, <i>J</i> = 7.21 Hz, 1 H) 7.77 (s, 1 H).</div></div><div id="sec4_14_2_2_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> Ethyl 2-(4-Chloro-5-methyl-6-oxo-pyridazin-1-yl)propanoate (<b>57</b>)</h5><div class="NLM_p last">To a mixture of ethyl 2-(4,5-dichloro-6-oxo-pyridazin-1-yl)propanoate (<b>56</b>, 14.5 g, 54.70 mmol) and methylboronic acid (3.11 g, 51.96 mmol) in dioxane (150 mL) and H<sub>2</sub>O (15 mL) were added Cs<sub>2</sub>CO<sub>3</sub> (53.46 g, 164.09 mmol) and [1,1′-Bis(diphenylphosphino)ferrocene]dichloropalladium(II) dichloride, complex with dichloromethane (2.00 g, 2.73 mmol). The mixture was stirred at 110 °C for 2 h, diluted with aqueous sodium hydrogen carbonate, and extracted twice with ethyl acetate. The combined organic layers were washed with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials were removed by filtration, volatiles removed under reduced pressure, and the residue purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether to afford the title compound as a white solid (4.12 g, 31% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.10 (s, 1H), 5.40 (q, <i>J</i> = 7.1 Hz, 1H), 4.12 (q, <i>J</i> = 7.0 Hz, 2H), 2.17 (s, 3H), 1.54 (d, <i>J</i> = 7.2 Hz, 3H), 1.16 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_14_2_2_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> 2-(4-Chloro-5-methyl-6-oxopyridazin-1(6H)-yl)propanoic acid (<b>58</b>)</h5><div class="NLM_p last">Compound <b>58</b> was prepared in an analogous manner to <b>55</b>.</div></div><div id="sec4_14_2_2_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> 2-(4-Chloro-5-methyl-6-oxo-pyridazin-1-yl)-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]propanamide (<b>22</b>)</h5><div class="NLM_p last">Compound <b>22</b> was prepared in an analogous manner to <b>6</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.41 (s, 1H), 8.40 (dd, <i>J</i> = 0.9, 4.8 Hz, 1H), 8.13–8.07 (m, 2H), 7.78–7.69 (m, 2H), 7.64 (dt, <i>J</i> = 1.8, 7.6 Hz, 1H), 7.29 (d, <i>J</i> = 8.4 Hz, 1H), 7.20–7.11 (m, 2H), 5.39 (q, <i>J</i> = 7.1 Hz, 1H), 3.18–3.10 (m, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H), 2.17 (s, 3H), 1.59 (d, <i>J</i> = 7.1 Hz, 3H); HRMS (ESI/Q-TOF) RT = 2.14 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>25</sub>ClN<sub>5</sub>O<sub>4</sub>S 490.1315; found 490.1310.</div></div><div id="sec4_14_2_2_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> Ethyl 2-(5-Chloro-4-fluoro-6-oxopyridazin-1(6H)-yl)acetate (<b>59</b>)</h5><div class="NLM_p last">A mixture of ethyl 2-(4,5-dichloro-6-oxopyridazin-1(6H)-yl)acetate (<b>31</b>, 1 g, 3.98 mmol), potassium fluoride (1.16 g, 19.92 mmol, 466.53 μL), and silver sulfate (124.19 mg, 398.30 μmol, 67.49 μL) in DMF (10 mL) was stirred at 110 °C for 12 h. The mixture was diluted with water and extracted twice with ethyl acetate. The combined organic layers were washed with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials removed by filtration, volatiles removed under reduced pressure, and the residue purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether to afford the title compound as a white solid (470 mg, 50% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>8</sub>H<sub>8</sub>ClFN<sub>2</sub>O<sub>3</sub> 235.0; found 235.0; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.85 (s, 1H), 4.89 (s, 2H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 1.27 (t, <i>J</i> = 7.2 Hz, 3H); <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>) δ −110.33; <sup>13</sup>C{<sup>1</sup>H} NMR (101 MHz, chloroform-<i>d</i>) δ 166.72, 158.65 (d, <i>J</i> = 6.4 Hz), 158.18 (d, <i>J</i> = 279.6 Hz), 129.70 (d, <i>J</i> = 28.6 Hz), 119.29 (d, <i>J</i> = 9.3 Hz), 62.16, 53.96, 14.17.</div></div><div id="sec4_14_2_2_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> 2-(5-Chloro-4-fluoro-6-oxopyridazin-1(6H)-yl)acetic acid (<b>60</b>)</h5><div class="NLM_p last">A mixture of ethyl 2-(5-chloro-4-fluoro-6-oxopyridazin-1(6H)-yl)acetate (<b>59</b>, 100 mg, 426.24 μmol) in hydrochloric acid (3 M in water, 10 mL) was stirred at 90 °C for 1 h. The mixture was frozen, and volatiles removed by lyophilization to afford the title compound as a white solid (100 mg, assumed quant.), which was used without further manipulation. LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>6</sub>H<sub>4</sub>ClFN<sub>2</sub>O<sub>3</sub> 207.0; found 207.1.</div></div><div id="sec4_14_2_2_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> 2-(5-Chloro-4-fluoro-6-oxo-pyridazin-1-yl)-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]acetamide (<b>23</b>)</h5><div class="NLM_p">A mixture of 5-amino-2-methyl-<i>N</i>-[2-(2-pyridyl)ethyl]benzenesulfonamide (<b>36</b>, 100 mg, 343.21 μmol), 2-(5-chloro-4-fluoro-6-oxopyridazin-1(6H)-yl)acetic acid (<b>60</b>, 85.07 mg, 411.85 μmol), 2-chloro-1-methyl-pyridin-1-ium iodide (131.53 mg, 514.81 μmol), and <i>N</i>,<i>N</i>-diethyl isopropylamine (88.71 mg, 686.42 μmol, 119.56 μL) in THF (2 mL) was stirred at 25 °C for 2 h. Volatiles were removed under reduced pressure, and the residue was purified by prep-HPLC (column: Shim-pack C18 150*25*10 μm; mobile phase: [water (0.225%FA)-ACN]; B%: 5%–38%, 11 min) to afford the title compound as a white solid (85 mg, 52% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.59 (s, 1H), 8.45–8.39 (m, 2H), 8.10 (d, <i>J</i> = 2.2 Hz, 1H), 7.78 (t, <i>J</i> = 5.7 Hz, 1H), 7.71–7.62 (m, 2H), 7.31 (d, <i>J</i> = 8.4 Hz, 1H), 7.18 (dd, <i>J</i> = 5.3, 7.0 Hz, 1H), 7.14 (d, <i>J</i> = 7.8 Hz, 1H), 5.00 (s, 2H), 3.18–3.13 (m, 2H), 2.82 (t, <i>J</i> = 7.3 Hz, 2H), 2.44 (s, 3H); <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>) δ −111.08; HRMS (ESI/Q-TOF) RT = 1.75 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>ClFN<sub>5</sub>O<sub>4</sub>S 480.0908; found 480.0901; see <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>.</div><figure id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Syntheses of Fluoro Pyridazinones (A) <b>23</b>, (B) <b>24</b>, and (C) <b>25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_14_2_2_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> Ethyl 2-(4-Fluoro-5-methyl-6-oxopyridazin-1(6H)-yl)acetate (<b>61</b>)</h5><div class="NLM_p last">A mixture of ethyl 2-(5-chloro-4-fluoro-6-oxopyridazin-1(6H)-yl)acetate (<b>59</b>, 1 g, 4.26 mmol), methylboronic acid (280.66 mg, 4.69 mmol), cesium carbonate (4.17 g, 12.79 mmol), [1,1′-bis(diphenylphosphino)ferrocene]dichloropalladium(II) dichloride, and complex with dichloromethane (155.94 mg, 213.12 μmol) in dioxane (20 mL) and water (2 mL) was stirred at 110 °C for 6 h. The mixture was cooled to room temperature, diluted with water, and extracted twice with ethyl acetate. The combined organic phases were washed with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials removed by filtration, volatiles removed under reduced pressure, and the residue was purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether to provide the title compound as a pink solid (560 mg, 61% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C9H11FN2O3 215.1; found 215.0; <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ = 7.79 (s, 1H), 4.89 (s, 2H), 4.29–4.23 (m, 2H), 2.15 (d, <i>J</i> = 2.6 Hz, 3H), 1.35–1.29 (m, 3H).</div></div><div id="sec4_14_2_2_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> 2-(4-Fluoro-5-methyl-6-oxopyridazin-1(6H)-yl)acetic acid (<b>62</b>)</h5><div class="NLM_p last">A mixture of ethyl 2-(4-fluoro-5-methyl-6-oxopyridazin-1(6H)-yl)acetate (<b>61</b>, 150 mg, 700.30 μmol) in hydrochloric acid (3 M in water, 15 mL) was stirred at 90 °C for 90 min. The mixture was frozen, and volatiles were removed by lyophilized to afford the title compound (120 mg, 92% yield) as a off-white solid. LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>7</sub>H<sub>8</sub>FN<sub>2</sub>O<sub>3</sub> 187.1; found 186.9;</div></div><div id="sec4_14_2_2_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> 2-(4-Fluoro-5-methyl-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>24</b>)</h5><div class="NLM_p last">Compound <b>24</b> was prepared analogously to <b>23</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.57 (s, 1H), 8.47–8.38 (m, 1H), 8.21 (s, 1H), 8.11 (d, <i>J</i> = 2.1 Hz, 1H), 7.78 (t, <i>J</i> = 5.7 Hz, 1H), 7.72–7.62 (m, 2H), 7.31 (d, <i>J</i> = 8.4 Hz, 1H), 7.19 (dd, <i>J</i> = 5.3, 7.0 Hz, 1H), 7.14 (d, <i>J</i> = 7.7 Hz, 1H), 4.92 (s, 2H), 3.19–3.11 (m, 2H), 2.82 (t, <i>J</i> = 7.3 Hz, 2H), 2.44 (s, 3H), 2.02 (d, <i>J</i> = 2.3 Hz, 3H); HRMS (ESI/Q-TOF) RT = 1.68 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>23</sub>FN<sub>5</sub>O<sub>4</sub>S 460.1455; found 460.1448.</div></div><div id="sec4_14_2_2_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> Ethyl 2-(4-Fluoro-6-oxo-pyridazin-1-yl)acetate (<b>63</b>)</h5><div class="NLM_p last">A mixture of ethyl 2-(5-chloro-4-fluoro-6-oxopyridazin-1(6H)-yl)acetate (<b>59</b>, 300 mg, 1.21 mmol), palladium on carbon (60 mg, 5% w/w), and triethylamine (366.28 mg, 3.62 mmol, 503.82 μL) in EtOAc (6 mL) was stirred under an atmosphere of hydrogen at 25 °C for 12 h. Insoluble materials were removed by filtration through Celite, volatiles removed under reduced pressure, and the residue purified by column chromatography on silica gel eluting with a gradient of ethyl acetate in petroleum ether to afford the title compound as a colorless oil (214 mg, 88% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>8</sub>H<sub>9</sub>FN<sub>2</sub>O<sub>3</sub> 201.1; found 201.0.</div></div><div id="sec4_14_2_2_54" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> 2-(4-Fluoro-6-oxo-pyridazin-1-yl)-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]acetamide (<b>25</b>)</h5><div class="NLM_p last">Compound <b>25</b> was prepared analogously to <b>23</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.56 (s, 1H), 8.41–8.40 (d, 1H), 8.26 (d, 1H), 8.10 (d, 1H), 7.77 (m, 1H), 7.69 (m, 1H), 7.64 (m, 1H), 7.31–7.29 (m, 1H), 7.14–7.12 (m, 1H), 7.02–6.99 (m, 1H), 4.92 (s, 2H), 3.18–3.13 (m, 2H), 2.84–2.80 (m, 2H), 2.44 (s, 3H); HRMS (ESI/Q-TOF) RT = 1.47 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>FN<sub>5</sub>O<sub>4</sub>S 446.1298; found 446.1290.</div></div><div id="sec4_14_2_2_55" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> 2-Chloro-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]acetamide (<b>65</b>)</h5><div class="NLM_p last">A mixture of 5-amino-2-methyl-<i>N</i>-[2-(2-pyridyl)ethyl]benzenesulfonamide (<b>36</b>, 300 mg, 1.03 mmol) and triethylamine (208.38 mg, 2.06 mmol, 286.62 μL) in DCM (5 mL) was treated with 2-chloroacetyl chloride (151.18 mg, 1.34 mmol, 106.46 μL) at 0 °C. The mixture was stirred at 20 °C for 12 h. The mixture was diluted with water and extracted twice with ethyl acetate. The combined organic layers were washed with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials removed by filtration, volatiles removed under reduced pressure, and the residue purified by prep-TLC on silica gel, eluting with dichloromethane:methanol 10:1, to give the title compound as a yellow solid (200 mg, 53% yield). LRMS (<i>m</i>/<i>z</i>): calcd [M + H]<sup>+</sup> for C<sub>16</sub>H<sub>18</sub>ClN<sub>3</sub>O<sub>3</sub>S 368.1; found 368.1; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 10.53 (s, 1H), 8.45–8.37 (m, 1H), 8.09 (d, <i>J</i> = 2.3 Hz, 1H), 7.82–7.72 (m, 2H), 7.65 (dt, <i>J</i> = 1.8, 7.6 Hz, 1H), 7.32 (d, <i>J</i> = 8.4 Hz, 1H), 7.21–7.12 (m, 2H), 4.26 (s, 2H), 3.20–3.14 (m, 2H), 2.83 (t, <i>J</i> = 7.4 Hz, 2H), 2.45 (s, 3H).</div></div><div id="sec4_14_2_2_56" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> 2-(4-Chloro-6-oxopyridazin-1(6H)-yl)-<i>N</i>-(4-methyl-3-(<i>N</i>-(2-(pyridin-2-yl)ethyl)sulfamoyl)phenyl)acetamide (<b>26</b>)</h5><div class="NLM_p last">A mixture of 4-chloro-1H-pyridazin-6-one (35.51 mg, 272.07 μmol), 2-chloro-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]acetamide (<b>65</b>, 100 mg, 247.33 μmol), and potassium carbonate (102.55 mg, 742.00 μmol) in DMF (2 mL) was stirred at 50 °C for 2 h. The mixture was diluted with water and extracted twice with ethyl acetate. The combined organic phases were washed with saturated aqueous sodium chloride and dried over sodium sulfate. The insoluble materials were removed by filtration, volatiles removed under reduced pressure, and the residue was purified by prep-HPLC (column: Phenomenex Synergi C18 150*25*10 μm; mobile phase: [water(0.225%FA)-ACN]; B%: 2%–32%,10 min) to give the title compound as a yellow solid (45 mg, 36% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.56 (s, 1H), 8.40 (ddd, <i>J</i> = 4.9, 1.9, 0.9 Hz, 1H), 8.14 (d, <i>J</i> = 2.4 Hz, 1H), 8.09 (d, <i>J</i> = 2.3 Hz, 1H), 7.77 (t, <i>J</i> = 5.8 Hz, 1H), 7.69 (dd, <i>J</i> = 8.2, 2.3 Hz, 1H), 7.64 (td, <i>J</i> = 7.7, 1.9 Hz, 1H), 7.35 (d, <i>J</i> = 2.4 Hz, 1H), 7.30 (d, <i>J</i> = 8.3 Hz, 1H), 7.17 (ddd, <i>J</i> = 7.6, 4.9, 1.2 Hz, 1H), 7.13 (dt, <i>J</i> = 7.9, 1.1 Hz, 1H), 4.90 (s, 2H), 3.14 (td, <i>J</i> = 7.2, 5.6 Hz, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H); 13C{1H} NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.96, 158.53, 158.24, 148.97, 139.12, 138.70, 136.54, 136.51, 136.41, 132.97, 131.10, 127.07, 123.20, 122.47, 121.59, 119.22, 54.62, 41.96, 37.28, 19.06; HRMS (ESI/Q-TOF) RT = 1.68 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>ClN<sub>5</sub>O<sub>4</sub>S 462.1002; found 462.0994.</div></div></div><div id="sec4_14_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> Synthesis of BRD0639</h4><div id="sec4_14_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> Methyl (<i>S</i>)-2-(4-Chloro-6-oxopyridazin-1(6H)-yl)propanoate (<b>66</b>)</h5><div class="NLM_p last">To a solution of 5-chloro-3(2<i>H</i>)-pyridazinone (CAS 660425-07-0, 300 mg, 2.30 mmol) and methyl (2<i>R</i>)-2-hydroxypropanoate (478.88 mg, 4.60 mmol, 439.34 μL) in THF (3 mL) was added triphenylphosphine (1.21 g, 4.60 mmol) at room temperature. The solution was placed in an ice water bath, and after 10 min, diisopropyl azodicarboxylate (929.46 mg, 4.60 mmol, 893.72 μL) was added. The mixture was removed from the ice water bath and allowed to warm to 20 °C for 2h. Volatiles were removed under reduced pressure, and the residue was purified by column chromatography on silica gel, eluting with a gradient of ethyl acetate in petroleum ether to give the title compound as a white solid (480 mg, 96% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.15 (d, <i>J</i> = 2.4 Hz, 1H), 7.33 (d, <i>J</i> = 2.4 Hz, 1H), 5.43 (q, <i>J</i> = 7.2 Hz, 1H), 3.64 (s, 3H), 1.53 (d, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_14_3_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> (2<i>S</i>)-2-(4-Chloro-6-oxo-pyridazin-1-yl)propanoic acid (<b>67</b>)</h5><div class="NLM_p last">Methyl (2<i>S</i>)-2-(4-chloro-6-oxo-pyridazin-1-yl)propanoate (<b>66</b>, 480 mg, 2.22 mmol) was suspended in aqueous hydrochloric acid (5% w/v, 4 mL), heated to 105 °C for 12 h, and cooled to room temperature, and the volatiles were removed to provide a brown solid. This was suspended in hot toluene, and a small amount of materials were removed by filtration. After cooling to room temperature, the title compound was isolated by filtration as an off white solid (170 mg, 38% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.14 (d, <i>J</i> = 2.3 Hz, 1H), 7.31 (d, <i>J</i> = 2.3 Hz, 1H), 5.34 (q, <i>J</i> = 7.2 Hz, 1H), 1.52 (d, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_14_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> (2<i>S</i>)-2-(4-Chloro-6-oxo-pyridazin-1-yl)-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]propanamide (<b>27</b>) (BRD0639)</h5><div class="NLM_p">To a mixture of (2<i>S</i>)-2-(4-chloro-6-oxo-pyridazin-1-yl)propanoic acid (<b>67</b>, 50.00 mg, 246.80 μmol) and 5-amino-2-methyl-<i>N</i>-[2-(2-pyridyl)ethyl]benzenesulfonamide (<b>36</b>, 71.91 mg, 246.80 μmol) in THF (1 mL) was added DIEA (127.59 mg, 987.19 μmol, 171.95 μL) over 5 min, followed by 2-chloro-1-methyl-pyridin-1-ium iodide (94.58 mg, 370.20 μmol). The reaction mixture was stirred at 20 °C for 1 h, volatiles were removed under reduced pressure, and the residue was purified by prep-HPLC (column: Phenomenex Gemini NX-C18(75*30 mm*3 μm); mobile phase: [water(10 mM NH4HCO3)-ACN]; B%: 18%–48%,8 min) to give the title compound as a white solid (21 mg, 18% yield). <sup>1</sup>H NMR (400 MHz, DMSO) δ 10.42 (s, 1H), 8.43–8.38 (m, 1H), 8.17 (d, <i>J</i> = 2.4 Hz, 1H), 8.08 (d, <i>J</i> = 2.3 Hz, 1H), 7.75 (s, 1H), 7.71 (dd, <i>J</i> = 8.2, 2.3 Hz, 1H), 7.64 (td, <i>J</i> = 7.7, 1.9 Hz, 1H), 7.32 (d, <i>J</i> = 2.4 Hz, 1H), 7.29 (d, <i>J</i> = 8.4 Hz, 1H), 7.17 (ddd, <i>J</i> = 7.7, 4.9, 1.2 Hz, 1H), 7.13 (d, <i>J</i> = 7.7 Hz, 1H), 5.39 (q, <i>J</i> = 7.1 Hz, 1H), 3.14 (t, <i>J</i> = 7.3 Hz, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H), 1.60 (d, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C{<sup>1</sup>H} NMR (101 MHz, DMSO) δ 168.39, 158.64, 158.25, 148.97, 138.88, 138.61, 136.75, 136.40, 136.04, 132.85, 131.05, 126.60, 123.21, 122.67, 121.59, 119.43, 57.99, 41.94, 37.29, 19.05, 15.69; SFC (Chiralpac AS-3, 50 × 4.6 mm I.D., 3 μM, mobile phase: gradient elution 5%–40% methanol (0.05% diethylamine) in CO<sub>2</sub>, flow rate 3 mL/min, PDA detector, column temp 35 °C, back pressure 100 bar) RT 2.037 min, 98.7% ee; HRMS (ESI/Q-TOF) RT = 1.88 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>4</sub>S 476.1159; found 476.1152; see <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>B.</div><figure id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Syntheses of Monochloropyridazinones (A) <b>26</b>, (B) <b>27</b>, and (C) <b>30</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_14_3_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> (2<i>R</i>)-2-(4-Chloro-6-oxo-pyridazin-1-yl)-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]propanamide (<b>28</b>)</h5><div class="NLM_p last">Compound <b>28</b> was prepared analogously to <b>27</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.42 (s, 1H), 8.40 (ddd, <i>J</i> = 4.9, 1.9, 0.9 Hz, 1H), 8.17 (d, <i>J</i> = 2.4 Hz, 1H), 8.08 (d, <i>J</i> = 2.3 Hz, 1H), 7.75 (s, 1H), 7.71 (dd, <i>J</i> = 8.2, 2.3 Hz, 1H), 7.64 (td, <i>J</i> = 7.6, 1.9 Hz, 1H), 7.32 (d, <i>J</i> = 2.4 Hz, 1H), 7.29 (d, <i>J</i> = 8.4 Hz, 1H), 7.17 (ddd, <i>J</i> = 7.6, 4.9, 1.2 Hz, 1H), 7.13 (dt, <i>J</i> = 7.8, 1.1 Hz, 1H), 5.39 (q, <i>J</i> = 7.1 Hz, 1H), 3.14 (t, <i>J</i> = 7.3 Hz, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H), 1.60 (d, <i>J</i> = 7.1 Hz, 3H); 13C{1H} NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.39, 158.64, 158.25, 148.97, 138.88, 138.61, 136.75, 136.40, 136.04, 132.85, 131.05, 126.60, 123.21, 122.67, 121.59, 119.43, 57.99, 41.94, 37.29, 19.05, 15.70; SFC (Chiralpac AS-3, 50 × 4.6 mm I.D., 3 μM, mobile phase: gradient elution 5%–40% methanol (0.05% diethylamine) in CO<sub>2</sub>, flow rate 3 mL/min, PDA detector, column temp 35 °C, back pressure 100 bar) RT 1.700 min, 95.7% ee; HRMS (ESI/Q-TOF) RT = 1.89 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>4</sub>S 476.1159; found 476.1152.</div></div><div id="sec4_14_3_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> (2<i>S</i>)-2-(4-Chloro-6-oxo-pyridazin-1-yl)-<i>N</i>-methyl-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]propanamide (<b>29</b>)</h5><div class="NLM_p last">Compound <b>29</b> was prepared analogously to <b>20</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.33 (br d, <i>J</i> = 4.8 Hz, 1H), 7.96 (d, <i>J</i> = 2.0 Hz, 1H), 7.88 (d, <i>J</i> = 1.6 Hz, 1H), 7.68 (dt, <i>J</i> = 1.9, 7.7 Hz, 1H), 7.52 (dd, <i>J</i> = 1.6, 8.0 Hz, 1H), 7.40 (d, <i>J</i> = 8.0 Hz, 1H), 7.24–7.17 (m, 2H), 6.86 (s, 1H), 5.26 (q, <i>J</i> = 6.8 Hz, 1H), 3.30 (br s, 2H), 3.26 (s, 3H), 2.91 (dt, <i>J</i> = 1.8, 7.2 Hz, 2H), 2.55 (s, 3H), 1.43 (d, <i>J</i> = 7.0 Hz, 3H); SFC (Chiralpac AD-3, 50 × 4.6 mm I.D., 3 μM, mobile phase: gradient elution 5%–40% 2-propanol (0.05% diethylamine) in CO<sub>2</sub>, flow rate 3 mL/min, PDA detector, column temp 35 °C, back pressure 100 bar) RT = 2.072 min, 86.3% ee (minor isomer at 1.895 min); HRMS (ESI/Q-TOF) RT = 1.83 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>25</sub>ClN<sub>5</sub>O<sub>4</sub>S 490.1315; found 490.1308.</div></div><div id="sec4_14_3_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> 2-(5-Chloro-6-oxo-pyridazin-1-yl)-<i>N</i>-[4-methyl-3-[2-(2-pyridyl)ethylsulfamoyl]phenyl]acetamide (<b>30</b>)</h5><div class="NLM_p last">Compound <b>30</b> was prepared analogously to <b>26</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.59 (s, 1H), 8.41 (ddd, <i>J</i> = 4.8, 1.9, 0.9 Hz, 1H), 8.10 (d, <i>J</i> = 2.4 Hz, 1H), 7.95 (d, <i>J</i> = 4.5 Hz, 1H), 7.86 (d, <i>J</i> = 4.5 Hz, 1H), 7.77 (t, <i>J</i> = 5.8 Hz, 1H), 7.69 (dd, <i>J</i> = 8.2, 2.3 Hz, 1H), 7.64 (td, <i>J</i> = 7.6, 1.8 Hz, 1H), 7.30 (d, <i>J</i> = 8.4 Hz, 1H), 7.17 (ddd, <i>J</i> = 7.6, 4.9, 1.2 Hz, 1H), 7.13 (dt, <i>J</i> = 7.8, 1.1 Hz, 1H), 4.97 (s, 2H), 3.15 (td, <i>J</i> = 7.3, 5.7 Hz, 2H), 2.81 (t, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H);13C{1H} NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.87, 158.24, 156.91, 148.97, 138.71, 136.54, 136.40, 136.07, 135.17, 132.98, 131.11, 130.90, 123.20, 122.47, 121.58, 119.21, 55.85, 41.97, 37.28, 19.07; HRMS (ESI/Q-TOF) RT = 1.52 min, <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>ClN<sub>5</sub>O<sub>4</sub>S 462.1002; found 462.0997.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i108"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00507" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44541" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44541" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00507?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00507</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional methods: NMR-based fragment screening; virtual screening; crystallography data collection and refinement statistics; electron density maps; protein mass spectra data; kinetic parameters plotted data; <sup>1</sup>H and <sup>13</sup>C NMR spectra; LCMS chromatograms (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf">jm1c00507_si_001.pdf (1.55 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_002.csv">jm1c00507_si_002.csv (2.77 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00507" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60547" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60547" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William R. Sellers</span> - <span class="hlFld-Affiliation affiliation">Cancer
Program, The Broad Institute of MIT and
Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology, Dana-Farber Cancer Institute, Department of Medicine, Harvard Medical School, 44 Binney Street, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#374044525b5b5245447755455856535e5944435e4342435219584550"><span class="__cf_email__" data-cfemail="1a6d697f76767f68695a7868757b7e7374696e736e6f6e7f3475687d">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessandra Ianari</span> - <span class="hlFld-Affiliation affiliation">Cancer
Program, The Broad Institute of MIT and
Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-5850-0476" title="Orcid link">https://orcid.org/0000-0002-5850-0476</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#107179717e7162795072627f7174797e636479646564753e7f6277"><span class="__cf_email__" data-cfemail="fc9d959d929d8e95bc9e8e939d9895928f889588898899d2938e9b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David C. McKinney</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1787-1841" title="Orcid link">https://orcid.org/0000-0003-1787-1841</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian J. McMillan</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew J. Ranaghan</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3572-5127" title="Orcid link">https://orcid.org/0000-0003-3572-5127</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jamie A. Moroco</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-8250-5923" title="Orcid link">https://orcid.org/0000-0001-8250-5923</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Merissa Brousseau</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zachary Mullin-Bernstein</span> - <span class="hlFld-Affiliation affiliation">Cancer
Program, The Broad Institute of MIT and
Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meghan O’Keefe</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrick McCarren</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael F. Mesleh</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kathleen M. Mulvaney</span> - <span class="hlFld-Affiliation affiliation">Cancer
Program, The Broad Institute of MIT and
Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Foxy Robinson</span> - <span class="hlFld-Affiliation affiliation">Cancer
Program, The Broad Institute of MIT and
Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ritu Singh</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Besnik Bajrami</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florence F. Wagner</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert Hilgraf</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin J. Drysdale</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Arthur J. Campbell</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adam Skepner</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David E. Timm</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, University of Utah, 1390 Presidents Circle, Salt Lake City, Utah 84112, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dale Porter</span> - <span class="hlFld-Affiliation affiliation">Cancer
Program, The Broad Institute of MIT and
Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Virendar K. Kaushik</span> - <span class="hlFld-Affiliation affiliation">Center
for the Development of Therapeutics, The
Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">Atomic coordinates of PRMT5:WDR77 bound to compound 1 (PDB code 6V0P), and compound 6 (PDB code 7M05) have been deposited with the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.<br /></br>The authors declare the following competing financial interest(s): W.R.S. is a Board or SAB member and holds equity in Ideaya Biosciences, Civetta Therapeutics, and Bluebird bio and has consulted for Array, Astex, Epidarex Capital, Ipsen, PearlRiver Therapeutics, Merck Pharmaceuticals, Sanofi, Servier, and Syndax Pharmaceuticals and receives research funding from Deerfield Investments, Pfizer Pharmaceuticals, Merck Pharmaceuticals, Ideaya Biosciences, and Ridgeline Discovery. W.R.S. is a copatent holder on EGFR mutation diagnostic patents. P.M. and B.J.M. are employees of and hold equity in Tango Therapeutics. V.K.K. holds equity in Dogma Therapeutics and is a consultant for Anji Pharmaceuticals. A.I. consults for Ridgeline Discovery.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i110">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was funded as part of an ongoing research collaboration between the Broad Institute and an affiliate of Deerfield Management Company, L.P. and by the NIH grant 1R01CA233626-01A1 to W.R.S.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">COPR5</td><td class="NLM_def"><p class="first last">cooperator of PRMT5</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">GSH</td><td class="NLM_def"><p class="first last">glutathione</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">2-[4-(2-hydroxyethyl)piperazin-1-yl]ethanesulfonic acid</p></td></tr><tr><td class="NLM_term">MCL1</td><td class="NLM_def"><p class="first last">induced myeloid leukemia cell differentiation protein 1</p></td></tr><tr><td class="NLM_term">MTA</td><td class="NLM_def"><p class="first last">5′-<i>S</i>-methyl-5′-thioadenosine</p></td></tr><tr><td class="NLM_term">ND</td><td class="NLM_def"><p class="first last">not determined</p></td></tr><tr><td class="NLM_term">PBM</td><td class="NLM_def"><p class="first last">PRMT5 binding motif</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate buffered saline</p></td></tr><tr><td class="NLM_term">pICln</td><td class="NLM_def"><p class="first last">chloride conductance regulatory protein</p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferases</p></td></tr><tr><td class="NLM_term">PRMT5</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 5</p></td></tr><tr><td class="NLM_term">RIOK1</td><td class="NLM_def"><p class="first last">RIO kinase 1</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SAP</td><td class="NLM_def"><p class="first last">substrate adaptor proteins</p></td></tr><tr><td class="NLM_term">SDMA</td><td class="NLM_def"><p class="first last">symmetrically dimethylated arginine</p></td></tr><tr><td class="NLM_term">SmBiT</td><td class="NLM_def"><p class="first last">SmallBit</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">TCEP</td><td class="NLM_def"><p class="first last">tris(2-carboxyethyl)phosphine</p></td></tr><tr><td class="NLM_term">WDR77</td><td class="NLM_def"><p class="first last">WD repeat-containing protein 77</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i112">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07381" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07381" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 24 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musiani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bok, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massignani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabaglio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ippolito, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuomo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozbek, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorgati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghoshdastider, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonaldi, T.</span></span> <span> </span><span class="NLM_article-title">Proteomics profiling of arginine methylation defines PRMT5 substrate specificity</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">575</span>),  <span class="NLM_fpage">eaat8388</span>, <span class="refDoi"> DOI: 10.1126/scisignal.aat8388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1126%2Fscisignal.aat8388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=30940768" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=eaat8388&issue=575&author=D.+Musianiauthor=J.+Bokauthor=E.+Massignaniauthor=L.+Wuauthor=T.+Tabaglioauthor=M.+R.+Ippolitoauthor=A.+Cuomoauthor=U.+Ozbekauthor=H.+Zorgatiauthor=U.+Ghoshdastiderauthor=R.+C.+Robinsonauthor=E.+Guccioneauthor=T.+Bonaldi&title=Proteomics+profiling+of+arginine+methylation+defines+PRMT5+substrate+specificity&doi=10.1126%2Fscisignal.aat8388"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aat8388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aat8388%26sid%3Dliteratum%253Aachs%26aulast%3DMusiani%26aufirst%3DD.%26aulast%3DBok%26aufirst%3DJ.%26aulast%3DMassignani%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DTabaglio%26aufirst%3DT.%26aulast%3DIppolito%26aufirst%3DM.%2BR.%26aulast%3DCuomo%26aufirst%3DA.%26aulast%3DOzbek%26aufirst%3DU.%26aulast%3DZorgati%26aufirst%3DH.%26aulast%3DGhoshdastider%26aufirst%3DU.%26aulast%3DRobinson%26aufirst%3DR.%2BC.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBonaldi%26aufirst%3DT.%26atitle%3DProteomics%2520profiling%2520of%2520arginine%2520methylation%2520defines%2520PRMT5%2520substrate%2520specificity%26jtitle%3DSci.%2520Signaling%26date%3D2019%26volume%3D12%26issue%3D575%26spage%3Deaat8388%26doi%3D10.1126%2Fscisignal.aat8388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Weck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gampa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sovath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caothien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caushi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortes-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desplat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwoske, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsidjani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schouwey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiedt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Project DRIVE: A Compendium of Cancer Dependencies and Synthetic Lethal Relationships Uncovered by Large-Scale, Deep RNAi Screening</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1016%2Fj.cell.2017.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=28753431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=577-592&issue=3&author=E.+R.+McDonaldauthor=A.+de+Weckauthor=M.+R.+Schlabachauthor=E.+Billyauthor=K.+J.+Mavrakisauthor=G.+R.+Hoffmanauthor=D.+Belurauthor=D.+Castellettiauthor=E.+Friasauthor=K.+Gampaauthor=J.+Goljiauthor=I.+Kaoauthor=L.+Liauthor=P.+Megelauthor=T.+A.+Perkinsauthor=N.+Ramadanauthor=D.+A.+Ruddyauthor=S.+J.+Silverauthor=S.+Sovathauthor=M.+Stumpauthor=O.+Weberauthor=R.+Widmerauthor=J.+Yuauthor=K.+Yuauthor=Y.+Yueauthor=D.+Abramowskiauthor=E.+Ackleyauthor=R.+Barrettauthor=J.+Bergerauthor=J.+L.+Bernardauthor=R.+Billigauthor=S.+M.+Brachmannauthor=F.+Buxtonauthor=R.+Caothienauthor=J.+X.+Caushiauthor=F.+S.+Chungauthor=M.+Cortes-Crosauthor=R.+S.+deBeaumontauthor=C.+Delaunayauthor=A.+Desplatauthor=W.+Duongauthor=D.+A.+Dwoskeauthor=R.+S.+Eldridgeauthor=A.+Farsidjaniauthor=F.+Fengauthor=J.+Fengauthor=D.+Flemmingauthor=W.+Forresterauthor=G.+G.+Galliauthor=Z.+Gaoauthor=F.+Gauterauthor=V.+Gibajaauthor=K.+Haasauthor=M.+Hattenbergerauthor=T.+Hoodauthor=K.+E.+Hurovauthor=Z.+Jaganiauthor=M.+Jenalauthor=J.+A.+Johnsonauthor=M.+D.+Jonesauthor=A.+Kapoorauthor=J.+Kornauthor=J.+Liuauthor=Q.+Liuauthor=S.+Liuauthor=Y.+Liuauthor=A.+T.+Looauthor=K.+J.+Macchiauthor=T.+Martinauthor=G.+McAllisterauthor=A.+Meyerauthor=S.+Molleauthor=R.+A.+Pagliariniauthor=T.+Phadkeauthor=B.+Repkoauthor=T.+Schouweyauthor=F.+Shanahanauthor=Q.+Shenauthor=C.+Stammauthor=C.+Stephanauthor=V.+M.+Stuckeauthor=R.+Tiedtauthor=M.+Varadarajanauthor=K.+Venkatesanauthor=A.+C.+Vitariauthor=M.+Wallrothauthor=J.+Weilerauthor=J.+Zhangauthor=C.+Mickaninauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=A.+Laiauthor=H.+Bitterauthor=E.+Leesauthor=N.+Keenauthor=A.+Kauffmannauthor=F.+Stegmeierauthor=F.+Hofmannauthor=T.+Schmelzleauthor=W.+R.+Sellers&title=Project+DRIVE%3A+A+Compendium+of+Cancer+Dependencies+and+Synthetic+Lethal+Relationships+Uncovered+by+Large-Scale%2C+Deep+RNAi+Screening&doi=10.1016%2Fj.cell.2017.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Project DRIVE: A compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span></div><div class="casAuthors">McDonald, E. Robert, III; de Weck, Antoine; Schlabach, Michael R.; Billy, Eric; Mavrakis, Konstantinos J.; Hoffman, Gregory R.; Belur, Dhiren; Castelletti, Deborah; Frias, Elizabeth; Gampa, Kalyani; Golji, Javad; Kao, Iris; Li, Li; Megel, Philippe; Perkins, Thomas A.; Ramadan, Nadire; Ruddy, David A.; Silver, Serena J.; Sovath, Sosathya; Stump, Mark; Weber, Odile; Widmer, Roland; Yu, Jianjun; Yu, Kristine; Yue, Yingzi; Abramowski, Dorothee; Ackley, Elizabeth; Barrett, Rosemary; Berger, Joel; Bernard, Julie L.; Billig, Rebecca; Brachmann, Saskia M.; Buxton, Frank; Caothien, Roger; Caushi, Justina X.; Chung, Franklin S.; Cortes-Cros, Marta; de Beaumont, Rosalie S.; Delaunay, Clara; Desplat, Aurore; Duong, William; Dwoske, Donald A.; Eldridge, Richard S.; Farsidjani, Ali; Feng, Fei; Feng, Jia Jia; Flemming, Daisy; Forrester, William; Galli, Giorgio G.; Gao, Zhenhai; Gauter, Francois; Gibaja, Veronica; Haas, Kristy; Hattenberger, Marc; Hood, Tami; Hurov, Kristen E.; Jagani, Zainab; Jenal, Mathias; Johnson, Jennifer A.; Jones, Michael D.; Kapoor, Avnish; Korn, Joshua; Liu, Jilin; Liu, Qiumei; Liu, Shumei; Liu, Yue; Loo, Alice T.; Macchi, Kaitlin J.; Martin, Typhaine; McAllister, Gregory; Meyer, Amandine; Molle, Sandra; Pagliarini, Raymond A.; Phadke, Tanushree; Repko, Brian; Schouwey, Tanja; Shanahan, Frances; Shen, Qiong; Stamm, Christelle; Stephan, Christine; Stucke, Volker M.; Tiedt, Ralph; Varadarajan, Malini; Venkatesan, Kavitha; Vitari, Alberto C.; Wallroth, Marco; Weiler, Jan; Zhang, Jing; Mickanin, Craig; Myer, Vic E.; Porter, Jeffery A.; Lai, Albert; Bitter, Hans; Lees, Emma; Keen, Nicholas; Kauffmann, Audrey; Stegmeier, Frank; Hofmann, Francesco; Schmelzle, Tobias; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">577-592.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elucidation of the mutational landscape of human cancer has progressed rapidly and been accompanied by the development of therapeutics targeting mutant oncogenes.  However, a comprehensive mapping of cancer dependencies has lagged behind and the discovery of therapeutic targets for counteracting tumor suppressor gene loss is needed.  To identify vulnerabilities relevant to specific cancer subtypes, we conducted a large-scale RNAi screen in which viability effects of mRNA knockdown were assessed for 7837 genes using an av. of 20 shRNAs per gene in 398 cancer cell lines.  We describe findings of this screen, outlining the classes of cancer dependency genes and their relationships to genetic, expression, and lineage features.  In addn., we describe robust gene-interaction networks recapitulating both protein complexes and functional cooperation among complexes and pathways.  This dataset along with a web portal is provided to the community to assist in the discovery and translation of new therapeutic approaches for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUs33_s2nC7Vg90H21EOLACvtfcHk0lg7U1fBnY7IMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI&md5=41592e684cc8f2cd28117cb3303f6120</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3Dde%2BWeck%26aufirst%3DA.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DBelur%26aufirst%3DD.%26aulast%3DCastelletti%26aufirst%3DD.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DGampa%26aufirst%3DK.%26aulast%3DGolji%26aufirst%3DJ.%26aulast%3DKao%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMegel%26aufirst%3DP.%26aulast%3DPerkins%26aufirst%3DT.%2BA.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DSilver%26aufirst%3DS.%2BJ.%26aulast%3DSovath%26aufirst%3DS.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DWidmer%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DAckley%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DJ.%2BL.%26aulast%3DBillig%26aufirst%3DR.%26aulast%3DBrachmann%26aufirst%3DS.%2BM.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DCaothien%26aufirst%3DR.%26aulast%3DCaushi%26aufirst%3DJ.%2BX.%26aulast%3DChung%26aufirst%3DF.%2BS.%26aulast%3DCortes-Cros%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%2BS.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DDesplat%26aufirst%3DA.%26aulast%3DDuong%26aufirst%3DW.%26aulast%3DDwoske%26aufirst%3DD.%2BA.%26aulast%3DEldridge%26aufirst%3DR.%2BS.%26aulast%3DFarsidjani%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFlemming%26aufirst%3DD.%26aulast%3DForrester%26aufirst%3DW.%26aulast%3DGalli%26aufirst%3DG.%2BG.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGauter%26aufirst%3DF.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DHattenberger%26aufirst%3DM.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DHurov%26aufirst%3DK.%2BE.%26aulast%3DJagani%26aufirst%3DZ.%26aulast%3DJenal%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DKapoor%26aufirst%3DA.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLoo%26aufirst%3DA.%2BT.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DMolle%26aufirst%3DS.%26aulast%3DPagliarini%26aufirst%3DR.%2BA.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DRepko%26aufirst%3DB.%26aulast%3DSchouwey%26aufirst%3DT.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DC.%26aulast%3DStucke%26aufirst%3DV.%2BM.%26aulast%3DTiedt%26aufirst%3DR.%26aulast%3DVaradarajan%26aufirst%3DM.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DWallroth%26aufirst%3DM.%26aulast%3DWeiler%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DProject%2520DRIVE%253A%2520A%2520Compendium%2520of%2520Cancer%2520Dependencies%2520and%2520Synthetic%2520Lethal%2520Relationships%2520Uncovered%2520by%2520Large-Scale%252C%2520Deep%2520RNAi%2520Screening%26jtitle%3DCell%26date%3D2017%26volume%3D170%26issue%3D3%26spage%3D577%26epage%3D592%26doi%3D10.1016%2Fj.cell.2017.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span> <span> </span><span class="NLM_article-title">Role of protein arginine methyltransferase 5 in human cancers</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">108790</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2019.108790</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1016%2Fj.biopha.2019.108790" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=30903920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtV2isb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2019&pages=108790&author=W.+Xiaoauthor=X.+Chenauthor=L.+Liuauthor=Y.+Shuauthor=M.+Zhangauthor=Y.+Zhong&title=Role+of+protein+arginine+methyltransferase+5+in+human+cancers&doi=10.1016%2Fj.biopha.2019.108790"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Role of protein arginine methyltransferase 5 in human cancers</span></div><div class="casAuthors">Xiao, Wendi; Chen, Xiaoqing; Liu, Lisa; Shu, Yuansen; Zhang, Min; Zhong, Yucheng</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108790</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) play important roles in protein methylation.  PRMT5 is the major type II arginine methyltransferase that catalyzes the transfer of two Me groups sym. to the arginine residues of either histone or non-histone proteins.  In recent years, increasing evidence has shown that PRMT5, as an oncogene, plays an indispensable regulatory role in the pathol. progression of several human cancers by promoting the proliferation, invasion, and migration of cancer cells.  PRMT5 is overexpressed in many malignant tumors and plays an important role in the occurrence and development of cancer, which suggests that PRMT5 may become a potential biomarker or therapeutic target of cancer.  This article reviews the biol. function, mechanism, and clin. significance of PRMT5 in tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPOq7QOZDBmbVg90H21EOLACvtfcHk0ljlVlYX3xHtCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtV2isb4%253D&md5=db7dfee60fc048cd7408eb34d3cd1d6a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2019.108790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2019.108790%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DShu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DY.%26atitle%3DRole%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520in%2520human%2520cancers%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2019%26volume%3D114%26spage%3D108790%26doi%3D10.1016%2Fj.biopha.2019.108790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, B.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality as an engine for cancer drug target discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0046-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1038%2Fs41573-019-0046-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=31712683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFegs7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=23-38&issue=1&author=A.+Huangauthor=L.+A.+Garrawayauthor=A.+Ashworthauthor=B.+Weber&title=Synthetic+lethality+as+an+engine+for+cancer+drug+target+discovery&doi=10.1038%2Fs41573-019-0046-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethality as an engine for cancer drug target discovery</span></div><div class="casAuthors">Huang, Alan; Garraway, Levi A.; Ashworth, Alan; Weber, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-38</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The first wave of genetically targeted therapies for cancer focused on drugging gene products that are recurrently mutated in specific cancer types.  However, mutational anal. of tumors has largely been exhausted as a strategy for the identification of new cancer targets that are druggable with conventional approaches.  Furthermore, some known genetic drivers of cancer have not been directly targeted yet owing to their mol. structure (undruggable oncogenes) or because they result in functional loss (tumor suppressor genes).  Functional genomic screening based on the genetic concept of synthetic lethality provides an avenue to discover drug targets in all these areas.  Although synthetic lethality is not a new idea, recent advances, including CRISPR-based gene editing, have made possible systematic screens for synthetic lethal drug targets in human cancers.  Such approaches have broad potential to drive the discovery of the next wave of genetic cancer targets and ultimately the introduction of effective medicines that are still needed for most cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUGZtPqjLRlrVg90H21EOLACvtfcHk0ljlVlYX3xHtCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFegs7jK&md5=227059e12105feecc402ea6ff4f98640</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0046-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0046-z%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DA.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DWeber%26aufirst%3DB.%26atitle%3DSynthetic%2520lethality%2520as%2520an%2520engine%2520for%2520cancer%2520drug%2520target%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2020%26volume%3D19%26issue%3D1%26spage%3D23%26epage%3D38%26doi%3D10.1038%2Fs41573-019-0046-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deWeck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan-Neale, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kipp, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprague, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccone, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span> (<span class="NLM_issue">6278</span>),  <span class="NLM_fpage">1208</span>– <span class="NLM_lpage">1213</span>, <span class="refDoi"> DOI: 10.1126/science.aad5944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1126%2Fscience.aad5944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=26912361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1208-1213&issue=6278&author=K.+J.+Mavrakisauthor=E.+R.+McDonaldauthor=M.+R.+Schlabachauthor=E.+Billyauthor=G.+R.+Hoffmanauthor=A.+deWeckauthor=D.+A.+Ruddyauthor=K.+Venkatesanauthor=J.+Yuauthor=G.+McAllisterauthor=M.+Stumpauthor=R.+deBeaumontauthor=S.+Hoauthor=Y.+Yueauthor=Y.+Liuauthor=Y.+Yan-Nealeauthor=G.+Yangauthor=F.+Linauthor=H.+Yinauthor=H.+Gaoauthor=D.+R.+Kippauthor=S.+Zhaoauthor=J.+T.+McNamaraauthor=E.+R.+Spragueauthor=B.+Zhengauthor=Y.+Linauthor=Y.+S.+Choauthor=J.+Guauthor=K.+Crawfordauthor=D.+Cicconeauthor=A.+C.+Vitariauthor=A.+Laiauthor=V.+Capkaauthor=K.+Hurovauthor=J.+A.+Porterauthor=J.+Tallaricoauthor=C.+Mickaninauthor=E.+Leesauthor=R.+Pagliariniauthor=N.+Keenauthor=T.+Schmelzleauthor=F.+Hofmannauthor=F.+Stegmeierauthor=W.+R.+Sellers&title=Disordered+methionine+metabolism+in+MTAP%2FCDKN2A-deleted+cancers+leads+to+dependence+on+PRMT5&doi=10.1126%2Fscience.aad5944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5</span></div><div class="casAuthors">Mavrakis, Konstantinos J.; McDonald, E. Robert, III; Schlabach, Michael R.; Billy, Eric; Hoffman, Gregory R.; de Weck, Antoine; Ruddy, David A.; Venkatesan, Kavitha; Yu, Jianjun; McAllister, Gregg; Stump, Mark; de Beaumont, Rosalie; Ho, Samuel; Yue, Yingzi; Liu, Yue; Yan-Neale, Yan; Yang, Guizhi; Lin, Fallon; Yin, Hong; Gao, Hui; Kipp, D. Randal; Zhao, Songping; McNamara, Joshua T.; Sprague, Elizabeth R.; Zheng, Bing; Lin, Ying; Cho, Young Shin; Gu, Justin; Crawford, Kenneth; Ciccone, David; Vitari, Alberto C.; Lai, Albert; Capka, Vladimir; Hurov, Kristen; Porter, Jeffery A.; Tallarico, John; Mickanin, Craig; Lees, Emma; Pagliarini, Raymond; Keen, Nicholas; Schmelzle, Tobias; Hofmann, Francesco; Stegmeier, Frank; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1208-1213</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">5-Methylthioadenosine phosphorylase (MTAP) is a key enzyme in the methionine salvage pathway.  The MTAP gene is frequently deleted in human cancers because of its chromosomal proximity to the tumor suppressor gene CDKN2A.  By interrogating data from a large-scale short hairpin RNA-mediated screen across 390 cancer cell line models, we found that the viability of MTAP-deficient cancer cells is impaired by depletion of the protein arginine methyltransferase PRMT5.  MTAP-deleted cells accumulate the metabolite methylthioadenosine (MTA), which we found to inhibit PRMT5 methyltransferase activity.  Deletion of MTAP in MTAP-proficient cells rendered them sensitive to PRMT5 depletion.  Conversely, reconstitution of MTAP in an MTAP-deficient cell line rescued PRMT5 dependence.  Thus, MTA accumulation in MTAP-deleted cancers creates a hypomorphic PRMT5 state that is selectively sensitized toward further PRMT5 inhibition.  Inhibitors of PRMT5 that leverage this dysregulated metabolic state merit further investigation as a potential therapy for MTAP/CDKN2A-deleted tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNsuzYFo1qn7Vg90H21EOLACvtfcHk0ljtPFSqG0y6GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtr0%253D&md5=d77531ea765db9762d6815840d39ee4a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5944%26sid%3Dliteratum%253Aachs%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DdeWeck%26aufirst%3DA.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%26aulast%3DHo%26aufirst%3DS.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYan-Neale%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DKipp%26aufirst%3DD.%2BR.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DMcNamara%26aufirst%3DJ.%2BT.%26aulast%3DSprague%26aufirst%3DE.%2BR.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DCho%26aufirst%3DY.%2BS.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DK.%26aulast%3DCiccone%26aufirst%3DD.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DCapka%26aufirst%3DV.%26aulast%3DHurov%26aufirst%3DK.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DTallarico%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DPagliarini%26aufirst%3DR.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DDisordered%2520methionine%2520metabolism%2520in%2520MTAP%252FCDKN2A-deleted%2520cancers%2520leads%2520to%2520dependence%2520on%2520PRMT5%26jtitle%3DScience%26date%3D2016%26volume%3D351%26issue%3D6278%26spage%3D1208%26epage%3D1213%26doi%3D10.1126%2Fscience.aad5944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krasinskas, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieply, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dacic, S.</span></span> <span> </span><span class="NLM_article-title">CDKN2A and MTAP deletions in peritoneal mesotheliomas are correlated with loss of p16 protein expression and poor survival</span>. <i>Mod. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/modpathol.2009.186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1038%2Fmodpathol.2009.186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=20081810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVCqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2010&pages=531-538&issue=4&author=A.+M.+Krasinskasauthor=D.+L.+Bartlettauthor=K.+Cieplyauthor=S.+Dacic&title=CDKN2A+and+MTAP+deletions+in+peritoneal+mesotheliomas+are+correlated+with+loss+of+p16+protein+expression+and+poor+survival&doi=10.1038%2Fmodpathol.2009.186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">CDKN2A and MTAP deletions in peritoneal mesotheliomas are correlated with loss of p16 protein expression and poor survival</span></div><div class="casAuthors">Krasinskas, Alyssa M.; Bartlett, David L.; Cieply, Kathleen; Dacic, Sanja</div><div class="citationInfo"><span class="NLM_cas:title">Modern Pathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-538</span>CODEN:
                <span class="NLM_cas:coden">MODPEO</span>;
        ISSN:<span class="NLM_cas:issn">0893-3952</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Homozygous deletion of CDKN2A (p16) is one of the most common genetic alterations in pleural mesotheliomas, occurring in up to 74% of cases.  MTAP resides in the same gene cluster of the 9p21 region and is co-deleted in the majority of CDKN2A deleted cases.  This study examines the genetic alterations in peritoneal mesotheliomas, which may have a different pathogenesis than their pleural counterparts.  Twenty-six cases of peritoneal mesotheliomas in a triplicate tissue microarray were studied.  Dual-color fluorescence in situ hybridization was performed with CDKN2A and MTAP locus-specific probes.  Nine of 26 (35%) peritoneal mesotheliomas had homozygous deletion of CDKN2A; MTAP was co-deleted in every case.  All cases with CDKN2A deletions had loss of p16 protein expression; five cases had loss of p16 protein without evidence of CDKN2A deletions.  All patients with CDKN2A deletions were men (P, NS) and were significantly older (mean, 63 years) than the patients with no deletions (mean, 52 years) (P=0.033, t-test).  An assocn. with asbestos exposure could not be proved in this study.  Similar to pleural mesotheliomas, patients with CDKN2A deletions and loss of p16 protein expression had worse overall and disease-specific survival (P=0.010 and 0.006, resp.; Kaplan-Meier log rank).  Detection of CDKN2A-MTAP co-deletion in peritoneal mesotheliomas, coupled with a p16 immunohistochem. stain as an inexpensive screening tool, can help identify those patients who may have an unfavorable outcome after aggressive cytoreductive surgery combined with hyperthermic i.p. chemotherapy and those who may respond to targeted therapy of the MTAP pathway.  Modern Pathol. (2010) 23, 531-538; doi:10.1038/modpathol.2009.186; published online 15 Jan. 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6etmI06rXTbVg90H21EOLACvtfcHk0ljtPFSqG0y6GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVCqsbY%253D&md5=f1c0141447219e09e8e755aa80bc0b97</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fmodpathol.2009.186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmodpathol.2009.186%26sid%3Dliteratum%253Aachs%26aulast%3DKrasinskas%26aufirst%3DA.%2BM.%26aulast%3DBartlett%26aufirst%3DD.%2BL.%26aulast%3DCieply%26aufirst%3DK.%26aulast%3DDacic%26aufirst%3DS.%26atitle%3DCDKN2A%2520and%2520MTAP%2520deletions%2520in%2520peritoneal%2520mesotheliomas%2520are%2520correlated%2520with%2520loss%2520of%2520p16%2520protein%2520expression%2520and%2520poor%2520survival%26jtitle%3DMod.%2520Pathol.%26date%3D2010%26volume%3D23%26issue%3D4%26spage%3D531%26epage%3D538%26doi%3D10.1038%2Fmodpathol.2009.186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielski, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span> <span> </span><span class="NLM_article-title">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span> (<span class="NLM_issue">6278</span>),  <span class="NLM_fpage">1214</span>– <span class="NLM_lpage">1218</span>, <span class="refDoi"> DOI: 10.1126/science.aad5214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1126%2Fscience.aad5214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=26912360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1214-1218&issue=6278&author=G.+V.+Kryukovauthor=F.+H.+Wilsonauthor=J.+R.+Ruthauthor=J.+Paulkauthor=A.+Tsherniakauthor=S.+E.+Marlowauthor=F.+Vazquezauthor=B.+A.+Weirauthor=M.+E.+Fitzgeraldauthor=M.+Tanakaauthor=C.+M.+Bielskiauthor=J.+M.+Scottauthor=C.+Dennisauthor=G.+S.+Cowleyauthor=J.+S.+Boehmauthor=D.+E.+Rootauthor=T.+R.+Golubauthor=C.+B.+Clishauthor=J.+E.+Bradnerauthor=W.+C.+Hahnauthor=L.+A.+Garraway&title=MTAP+deletion+confers+enhanced+dependency+on+the+PRMT5+arginine+methyltransferase+in+cancer+cells&doi=10.1126%2Fscience.aad5214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span></div><div class="casAuthors">Kryukov, Gregory V.; Wilson, Frederick H.; Ruth, Jason R.; Paulk, Joshiawa; Tsherniak, Aviad; Marlow, Sara E.; Vazquez, Francisca; Weir, Barbara A.; Fitzgerald, Mark E.; Tanaka, Minoru; Bielski, Craig M.; Scott, Justin M.; Dennis, Courtney; Cowley, Glenn S.; Boehm, Jesse S.; Root, David E.; Golub, Todd R.; Clish, Clary B.; Bradner, James E.; Hahn, William C.; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1214-1218</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The discovery of cancer dependencies has the potential to inform therapeutic strategies and to identify putative drug targets.  Integrating data from comprehensive genomic profiling of cancer cell lines and from functional characterization of cancer cell dependencies, we discovered that loss of the enzyme methylthioadenosine phosphorylase (MTAP) confers a selective dependence on protein arginine methyltransferase 5 (PRMT5) and its binding partner WDR77.  MTAP is frequently lost due to its proximity to the commonly deleted tumor suppressor gene, CDKN2A.  We obsd. increased intracellular concns. of methylthioadenosine (MTA, the metabolite cleaved by MTAP) in cells harboring MTAP deletions.  Furthermore, MTA specifically inhibited PRMT5 enzymic activity.  Administration of either MTA or a small-mol. PRMT5 inhibitor showed a modest preferential impairment of cell viability for MTAP-null cancer cell lines compared with isogenic MTAP-expressing counterparts.  Together, our findings reveal PRMT5 as a potential vulnerability across multiple cancer lineages augmented by a common "passenger" genomic alteration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94dQPnMgrqbVg90H21EOLACvtfcHk0ljJqzk8ofjf5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D&md5=2b017a8c54972828c9024a9d85ff2e2d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5214%26sid%3Dliteratum%253Aachs%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DWilson%26aufirst%3DF.%2BH.%26aulast%3DRuth%26aufirst%3DJ.%2BR.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DMarlow%26aufirst%3DS.%2BE.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DBielski%26aufirst%3DC.%2BM.%26aulast%3DScott%26aufirst%3DJ.%2BM.%26aulast%3DDennis%26aufirst%3DC.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DMTAP%2520deletion%2520confers%2520enhanced%2520dependency%2520on%2520the%2520PRMT5%2520arginine%2520methyltransferase%2520in%2520cancer%2520cells%26jtitle%3DScience%26date%3D2016%26volume%3D351%26issue%3D6278%26spage%3D1214%26epage%3D1218%26doi%3D10.1126%2Fscience.aad5214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalev, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyer, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguado-Fraile, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lizotte, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuncay, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clasquin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murtie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagaraja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biller, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marjon, K.</span></span> <span> </span><span class="NLM_article-title">MAT2A inhibition blocks the growth of MTAP-deleted cancer cells by reducing PRMT5-dependent mRNA splicing and inducing DNA damage</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">209</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2020.12.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1016%2Fj.ccell.2020.12.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=33450196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhs1OmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=209-224&issue=2&author=P.+Kalevauthor=M.+L.+Hyerauthor=S.+Grossauthor=Z.+Konteatisauthor=C.+C.+Chenauthor=M.+Fletcherauthor=M.+Leinauthor=E.+Aguado-Fraileauthor=V.+Frankauthor=A.+Barnettauthor=E.+Mandleyauthor=J.+Goldfordauthor=Y.+Chenauthor=K.+Sellersauthor=S.+Hayesauthor=K.+Lizotteauthor=P.+Quangauthor=Y.+Tuncayauthor=M.+Clasquinauthor=R.+Petersauthor=J.+Weierauthor=E.+Simoneauthor=J.+Murtieauthor=W.+Liuauthor=R.+Nagarajaauthor=L.+Dangauthor=Z.+Suiauthor=S.+A.+Billerauthor=J.+Travinsauthor=K.+M.+Marksauthor=K.+Marjon&title=MAT2A+inhibition+blocks+the+growth+of+MTAP-deleted+cancer+cells+by+reducing+PRMT5-dependent+mRNA+splicing+and+inducing+DNA+damage&doi=10.1016%2Fj.ccell.2020.12.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">MAT2A inhibition blocks the growth of MTAP-deleted cancer cells by reducing PRMT5-dependent mRNA splicing and inducing DNA damage</span></div><div class="casAuthors">Kalev, Peter; Hyer, Marc L.; Gross, Stefan; Konteatis, Zenon; Chen, Chi-Chao; Fletcher, Mark; Lein, Max; Aguado-Fraile, Elia; Frank, Victoria; Barnett, Amelia; Mandley, Everton; Goldford, Joshua; Chen, Yue; Sellers, Katie; Hayes, Sebastian; Lizotte, Kate; Quang, Phong; Tuncay, Yesim; Clasquin, Michelle; Peters, Rachel; Weier, Jaclyn; Simone, Eric; Murtie, Joshua; Liu, Wei; Nagaraja, Raj; Dang, Lenny; Sui, Zhihua; Biller, Scott A.; Travins, Jeremy; Marks, Kevin M.; Marjon, Katya</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">209-224.e11</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The methylthioadenosine phosphorylase (MTAP) gene is located adjacent to the cyclin-dependent kinase inhibitor 2A (CDKN2A) tumor-suppressor gene and is co-deleted with CDKN2A in approx. 15% of all cancers.  This co-deletion leads to aggressive tumors with poor prognosis that lack effective, molecularly targeted therapies.  The metabolic enzyme methionine adenosyltransferase 2α (MAT2A) was identified as a synthetic lethal target in MTAP-deleted cancers.  We report the characterization of potent MAT2A inhibitors that substantially reduce levels of S-adenosylmethionine (SAM) and demonstrate antiproliferative activity in MTAP-deleted cancer cells and tumors.  Using RNA sequencing and proteomics, we demonstrate that MAT2A inhibition is mechanistically linked to reduced protein arginine methyltransferase 5 (PRMT5) activity and splicing perturbations.  We further show that DNA damage and mitotic defects ensue upon MAT2A inhibition in HCT116 MTAP-/- cells, providing a rationale for combining the MAT2A clin. candidate AG-270 with antimitotic taxanes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0SqOh3MkAALVg90H21EOLACvtfcHk0ljJqzk8ofjf5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhs1OmsL4%253D&md5=d4c0eba5310e7a85d0904ad81331a938</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2020.12.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2020.12.010%26sid%3Dliteratum%253Aachs%26aulast%3DKalev%26aufirst%3DP.%26aulast%3DHyer%26aufirst%3DM.%2BL.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DKonteatis%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DC.%2BC.%26aulast%3DFletcher%26aufirst%3DM.%26aulast%3DLein%26aufirst%3DM.%26aulast%3DAguado-Fraile%26aufirst%3DE.%26aulast%3DFrank%26aufirst%3DV.%26aulast%3DBarnett%26aufirst%3DA.%26aulast%3DMandley%26aufirst%3DE.%26aulast%3DGoldford%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSellers%26aufirst%3DK.%26aulast%3DHayes%26aufirst%3DS.%26aulast%3DLizotte%26aufirst%3DK.%26aulast%3DQuang%26aufirst%3DP.%26aulast%3DTuncay%26aufirst%3DY.%26aulast%3DClasquin%26aufirst%3DM.%26aulast%3DPeters%26aufirst%3DR.%26aulast%3DWeier%26aufirst%3DJ.%26aulast%3DSimone%26aufirst%3DE.%26aulast%3DMurtie%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DNagaraja%26aufirst%3DR.%26aulast%3DDang%26aufirst%3DL.%26aulast%3DSui%26aufirst%3DZ.%26aulast%3DBiller%26aufirst%3DS.%2BA.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26aulast%3DMarjon%26aufirst%3DK.%26atitle%3DMAT2A%2520inhibition%2520blocks%2520the%2520growth%2520of%2520MTAP-deleted%2520cancer%2520cells%2520by%2520reducing%2520PRMT5-dependent%2520mRNA%2520splicing%2520and%2520inducing%2520DNA%2520damage%26jtitle%3DCancer%2520Cell%26date%3D2021%26volume%3D39%26issue%3D2%26spage%3D209%26epage%3D224%26doi%3D10.1016%2Fj.ccell.2020.12.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&issue=6&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0ljJqzk8ofjf5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D6%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">612</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=612-617&issue=7&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+Potent+and+Selective+Inhibitor+of+Arginine+Methyltransferase+5%2C+PRMT5%2C+with+Antitumor+Activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span></div><div class="casAuthors">Bonday, Zahid Q.; Cortez, Guillermo S.; Grogan, Michael J.; Antonysamy, Stephen; Weichert, Ken; Bocchinfuso, Wayne P.; Li, Fengling; Kennedy, Steven; Li, Binghui; Mader, Mary M.; Arrowsmith, Cheryl H.; Brown, Peter J.; Eram, Mohammad S.; Szewczyk, Magdalena M.; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Guccione, Ernesto; Campbell, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">612-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that catalyzes the formation of sym. dimethylarginine in a no. of nuclear and cytoplasmic proteins.  Although the cellular functions of PRMT5 have not been fully unraveled, it has been implicated in a no. of cellular processes like RNA processing, signal transduction and transcriptional regulation.  PRMT5 is ubiquitously expressed in most tissues and its expression has been shown to be elevated in several cancers including breast cancer, gastric cancer, glioblastoma and lymphoma.  Here we describe the identification and characterization of a novel and selective PRMT5 inhibitor with potent in vitro and in vivo activity.  Compd. 1 (also called LLY-283) inhibited PRMT5 enzymic activity in vitro and in cells with IC50 of 22 ± 3 nM and 25 ± 1 nM, resp. while its diastereomer, compd. 2 (also called LLY-284) was much less active.  Compd. 1 also showed antitumor activity in mouse xenografts when dosed orally and can serve as an excellent probe mol. for understanding the biol. function of PRMT5 in normal and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYIq2FC-EskrVg90H21EOLACvtfcHk0lj2brY5qnkNAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D&md5=93517c34cf9f99d76730a538f594ddd7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520Arginine%2520Methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520Antitumor%2520Activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26issue%3D7%26spage%3D612%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinken, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuring, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poggesi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckx, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Heerde, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nys, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanovich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herkert, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhulst, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du Jardin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meerpoel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diels, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viellevoye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepens, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poncelet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linders, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chetty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laquerre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span> <span> </span><span class="NLM_article-title">A novel PRMT5 inhibitor with potent in vitro and in vivo activity in preclinical lung cancer models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">DDT02-04</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2017-DDT02-04</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1158%2F1538-7445.AM2017-DDT02-04" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=DDT02-04&issue=13&author=D.+Brehmerauthor=T.+F.+Wuauthor=G.+Mannensauthor=L.+Bekeauthor=P.+Vinkenauthor=D.+Gaffneyauthor=W.+M.+Sunauthor=V.+Pandeauthor=J.+W.+Thuringauthor=H.+Millarauthor=I.+Poggesiauthor=I.+Somersauthor=A.+Boeckxauthor=M.+Paradeauthor=E.+van+Heerdeauthor=T.+Nysauthor=C.+Yanovichauthor=B.+Herkertauthor=T.+Verhulstauthor=M.+Du+Jardinauthor=L.+Meerpoelauthor=C.+Moyauthor=G.+Dielsauthor=M.+Viellevoyeauthor=W.+Schepensauthor=A.+Ponceletauthor=J.+T.+Lindersauthor=E.+C.+Lawsonauthor=J.+P.+Edwardsauthor=D.+Chettyauthor=S.+Laquerreauthor=M.+V.+Lorenzi&title=A+novel+PRMT5+inhibitor+with+potent+in+vitro+and+in+vivo+activity+in+preclinical+lung+cancer+models&doi=10.1158%2F1538-7445.AM2017-DDT02-04"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-DDT02-04&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-DDT02-04%26sid%3Dliteratum%253Aachs%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DT.%2BF.%26aulast%3DMannens%26aufirst%3DG.%26aulast%3DBeke%26aufirst%3DL.%26aulast%3DVinken%26aufirst%3DP.%26aulast%3DGaffney%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DW.%2BM.%26aulast%3DPande%26aufirst%3DV.%26aulast%3DThuring%26aufirst%3DJ.%2BW.%26aulast%3DMillar%26aufirst%3DH.%26aulast%3DPoggesi%26aufirst%3DI.%26aulast%3DSomers%26aufirst%3DI.%26aulast%3DBoeckx%26aufirst%3DA.%26aulast%3DParade%26aufirst%3DM.%26aulast%3Dvan%2BHeerde%26aufirst%3DE.%26aulast%3DNys%26aufirst%3DT.%26aulast%3DYanovich%26aufirst%3DC.%26aulast%3DHerkert%26aufirst%3DB.%26aulast%3DVerhulst%26aufirst%3DT.%26aulast%3DDu%2BJardin%26aufirst%3DM.%26aulast%3DMeerpoel%26aufirst%3DL.%26aulast%3DMoy%26aufirst%3DC.%26aulast%3DDiels%26aufirst%3DG.%26aulast%3DViellevoye%26aufirst%3DM.%26aulast%3DSchepens%26aufirst%3DW.%26aulast%3DPoncelet%26aufirst%3DA.%26aulast%3DLinders%26aufirst%3DJ.%2BT.%26aulast%3DLawson%26aufirst%3DE.%2BC.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26aulast%3DChetty%26aufirst%3DD.%26aulast%3DLaquerre%26aufirst%3DS.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26atitle%3DA%2520novel%2520PRMT5%2520inhibitor%2520with%2520potent%2520in%2520vitro%2520and%2520in%2520vivo%2520activity%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26issue%3D13%26spage%3DDDT02-04%26doi%3D10.1158%2F1538-7445.AM2017-DDT02-04" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinken, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommers, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuring, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haddish-Berhane, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laquerre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span> <span> </span><span class="NLM_article-title">JNJ-64619178, a selective and pseudo-irreversible PRMT5 inhibitor with potent in vitro and in vivo activity, demonstrated in several lung cancer models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">4859</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2018-4859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1158%2F1538-7445.AM2018-4859" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=4859&issue=13&author=T.+F.+Wuauthor=H.+Millarauthor=D.+Gaffneyauthor=L.+Bekeauthor=G.+Mannensauthor=P.+Vinkenauthor=I.+Sommersauthor=J.+W.+Thuringauthor=W.+M.+Sunauthor=C.+Moyauthor=V.+Pandeauthor=J.+G.+Zhouauthor=N.+Haddish-Berhaneauthor=M.+Salvatiauthor=S.+Laquerreauthor=M.+V.+Lorenziauthor=D.+Brehmer&title=JNJ-64619178%2C+a+selective+and+pseudo-irreversible+PRMT5+inhibitor+with+potent+in+vitro+and+in+vivo+activity%2C+demonstrated+in+several+lung+cancer+models&doi=10.1158%2F1538-7445.AM2018-4859"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2018-4859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2018-4859%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%2BF.%26aulast%3DMillar%26aufirst%3DH.%26aulast%3DGaffney%26aufirst%3DD.%26aulast%3DBeke%26aufirst%3DL.%26aulast%3DMannens%26aufirst%3DG.%26aulast%3DVinken%26aufirst%3DP.%26aulast%3DSommers%26aufirst%3DI.%26aulast%3DThuring%26aufirst%3DJ.%2BW.%26aulast%3DSun%26aufirst%3DW.%2BM.%26aulast%3DMoy%26aufirst%3DC.%26aulast%3DPande%26aufirst%3DV.%26aulast%3DZhou%26aufirst%3DJ.%2BG.%26aulast%3DHaddish-Berhane%26aufirst%3DN.%26aulast%3DSalvati%26aufirst%3DM.%26aulast%3DLaquerre%26aufirst%3DS.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DBrehmer%26aufirst%3DD.%26atitle%3DJNJ-64619178%252C%2520a%2520selective%2520and%2520pseudo-irreversible%2520PRMT5%2520inhibitor%2520with%2520potent%2520in%2520vitro%2520and%2520in%2520vivo%2520activity%252C%2520demonstrated%2520in%2520several%2520lung%2520cancer%2520models%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26issue%3D13%26spage%3D4859%26doi%3D10.1158%2F1538-7445.AM2018-4859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mulvaney, K. M.</span>; <span class="NLM_string-name">Blomquist, C.</span>; <span class="NLM_string-name">Acharya, N.</span>; <span class="NLM_string-name">Li, R.</span>; <span class="NLM_string-name">O’Keefe, M.</span>; <span class="NLM_string-name">Ranaghan, M.</span>; <span class="NLM_string-name">Stokes, M.</span>; <span class="NLM_string-name">Nelson, A. J.</span>; <span class="NLM_string-name">Jain, S. S.</span>; <span class="NLM_string-name">Columbus, J.</span>; <span class="NLM_string-name">Bozal, F. K.</span>; <span class="NLM_string-name">Skepner, A.</span>; <span class="NLM_string-name">Raymond, D.</span>; <span class="NLM_string-name">McKinney, D. C.</span>; <span class="NLM_string-name">Freyzon, Y.</span>; <span class="NLM_string-name">Baidi, Y.</span>; <span class="NLM_string-name">Porter, D.</span>; <span class="NLM_string-name">Ianari, A.</span>; <span class="NLM_string-name">McMillan, B.</span>; <span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for substrate recruitment to the PRMT5 methylosome</span>. <i>bioRxiv</i>, August 22, 2020, ver. 1.<span class="refDoi"> DOI: 10.1101/2020.08.22.256347</span>  (accessed ).</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1101%2F2020.08.22.256347" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mulvaney%2C+K.+M.%3B+Blomquist%2C+C.%3B+Acharya%2C+N.%3B+Li%2C+R.%3B+O%E2%80%99Keefe%2C+M.%3B+Ranaghan%2C+M.%3B+Stokes%2C+M.%3B+Nelson%2C+A.+J.%3B+Jain%2C+S.+S.%3B+Columbus%2C+J.%3B+Bozal%2C+F.+K.%3B+Skepner%2C+A.%3B+Raymond%2C+D.%3B+McKinney%2C+D.+C.%3B+Freyzon%2C+Y.%3B+Baidi%2C+Y.%3B+Porter%2C+D.%3B+Ianari%2C+A.%3B+McMillan%2C+B.%3B+Sellers%2C+W.+R.+Molecular+basis+for+substrate+recruitment+to+the+PRMT5+methylosome.+bioRxiv%2C+August+22%2C+2020%2C+ver.+1.+10.1101%2F2020.08.22.256347+%28accessed+%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1101%2F2020.08.22.256347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26id%3Ddoi%3A10.1101%252F2020.08.22.256347%26sid%3Dliteratum%253Aachs%26aulast%3DMulvaney%26aufirst%3DK.%2BM.%26atitle%3DMolecular%2520basis%2520for%2520substrate%2520recruitment%2520to%2520the%2520PRMT5%2520methylosome%26doi%3D10.1101%2F2020.08.22.256347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marie, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemercier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murillo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akella, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcaurelle, L. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of stereochemically and skeletally diverse fused ring systems from functionalized C-glycosides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>78</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">5160</span>– <span class="NLM_lpage">5171</span>, <span class="refDoi"> DOI: 10.1021/jo4000916</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo4000916" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvFyisr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=5160-5171&issue=11&author=B.+Gerardauthor=M.+D.+T.+Leeauthor=S.+Dandapaniauthor=J.+R.+Duvallauthor=M.+E.+Fitzgeraldauthor=S.+Kesavanauthor=J.+T.+Loweauthor=J.+C.+Marieauthor=B.+A.+Pandyaauthor=B.+C.+Suhauthor=M.+W.+O%E2%80%99Sheaauthor=M.+Dombrowskiauthor=D.+Hamannauthor=B.+Lemercierauthor=T.+Murilloauthor=L.+B.+Akellaauthor=M.+A.+Foleyauthor=L.+A.+Marcaurelle&title=Synthesis+of+stereochemically+and+skeletally+diverse+fused+ring+systems+from+functionalized+C-glycosides&doi=10.1021%2Fjo4000916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Stereochemically and Skeletally Diverse Fused Ring Systems from Functionalized C-Glycosides</span></div><div class="casAuthors">Gerard, Baudouin; Lee, Maurice D., IV; Dandapani, Sivaraman; Duvall, Jeremy R.; Fitzgerald, Mark E.; Kesavan, Sarathy; Lowe, Jason T.; Marie, Jean-Charles; Pandya, Bhaumik A.; Suh, Byung-Chul; O'Shea, Morgan Welzel; Dombrowski, Michael; Hamann, Diane; Lemercier, Berenice; Murillo, Tiffanie; Akella, Lakshmi B.; Foley, Michael A.; Marcaurelle, Lisa A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5160-5171</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A diversity-oriented synthesis (DOS) strategy was developed for the synthesis of stereochem. diverse fused-ring systems contg. a pyran moiety.  Each scaffold contains an amine and Me ester for further diversification via amine capping and amide coupling.  Scaffold diversity was evaluated in comparison to previously prepd. scaffolds by a shape-based principal moments of inertia (PMI) anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUJ_merOEm2LVg90H21EOLACvtfcHk0ljMCj97SVzAHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvFyisr0%253D&md5=c1882e312248cc18f53ab203b881ff89</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjo4000916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo4000916%26sid%3Dliteratum%253Aachs%26aulast%3DGerard%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DM.%2BD.%2BT.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DKesavan%26aufirst%3DS.%26aulast%3DLowe%26aufirst%3DJ.%2BT.%26aulast%3DMarie%26aufirst%3DJ.%2BC.%26aulast%3DPandya%26aufirst%3DB.%2BA.%26aulast%3DSuh%26aufirst%3DB.%2BC.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DM.%2BW.%26aulast%3DDombrowski%26aufirst%3DM.%26aulast%3DHamann%26aufirst%3DD.%26aulast%3DLemercier%26aufirst%3DB.%26aulast%3DMurillo%26aufirst%3DT.%26aulast%3DAkella%26aufirst%3DL.%2BB.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26aulast%3DMarcaurelle%26aufirst%3DL.%2BA.%26atitle%3DSynthesis%2520of%2520stereochemically%2520and%2520skeletally%2520diverse%2520fused%2520ring%2520systems%2520from%2520functionalized%2520C-glycosides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2013%26volume%3D78%26issue%3D11%26spage%3D5160%26epage%3D5171%26doi%3D10.1021%2Fjo4000916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, T.</span></span> <span> </span><span class="NLM_article-title">The emerging role of human esterases</span>. <i>Drug Metab. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">466</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.2133/dmpk.DMPK-12-RV-042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.2133%2Fdmpk.DMPK-12-RV-042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=22813719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVykurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2012&pages=466-477&issue=5&author=T.+Fukamiauthor=T.+Yokoi&title=The+emerging+role+of+human+esterases&doi=10.2133%2Fdmpk.DMPK-12-RV-042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging role of human esterases</span></div><div class="casAuthors">Fukami, Tatsuki; Yokoi, Tsuyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Pharmacokinetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">466-477</span>CODEN:
                <span class="NLM_cas:coden">DMPRB8</span>;
        ISSN:<span class="NLM_cas:issn">1347-4367</span>.
    
            (<span class="NLM_cas:orgname">Japanese Society for the Study of Xenobiotics</span>)
        </div><div class="casAbstract">In this review, novel aspects of the role of esterases, which contribute to the metab. of 10% of therapeutic drugs, are described.  Esterases hydrolyze the compds. that contain ester, amide, and thioester bonds, which cause prodrug activation or detoxification.  Among esterases, carboxylesterases are well known to be involved in the hydrolysis of a variety of drugs.  Addnl., other esterases have recently received attention for their pharmacol. and toxicol. roles.  Arylacetamide deacetylase (AADAC) is involved in the hydrolysis of flutamide, phenacetin, and rifamycins.  AADAC is assocd. with adverse drug reactions because the hydrolytic metabolites of flutamide and phenacetin appear to be assocd. with hepatotoxicity and nephrotoxicity/hematotoxicity, resp.  Paraoxonase and butyrylcholinesterase hydrolyze pirocarpine/simvastatin and succinylcholine/bambuterol, resp.  Although the esterases that hydrolyze the acyl-glucuronides of drugs have largely been unknown, we recently found that α/β hydrolase domain contg. 10 (ABHD10) is responsible for the hydrolysis of mycophenolic acid acyl-glucuronide in human liver.  Because acyl-glucuronides are assocd. with toxicity, ABHD10 might function as a detoxification enzyme.  Thus, various esterases, which include enzymes that have not been known to hydrolyze drugs, are involved in drug metab. with different substrate specificity.  Further esterase studies should be conducted to promote our understanding in clin. pharmacotherapy and drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDhsUJ4NrYQbVg90H21EOLACvtfcHk0liujW40nPK6PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVykurzE&md5=d760b8a6a3217a4ab9716190800012b0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2133%2Fdmpk.DMPK-12-RV-042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2133%252Fdmpk.DMPK-12-RV-042%26sid%3Dliteratum%253Aachs%26aulast%3DFukami%26aufirst%3DT.%26aulast%3DYokoi%26aufirst%3DT.%26atitle%3DThe%2520emerging%2520role%2520of%2520human%2520esterases%26jtitle%3DDrug%2520Metab.%2520Pharmacokinet.%26date%3D2012%26volume%3D27%26issue%3D5%26spage%3D466%26epage%3D477%26doi%3D10.2133%2Fdmpk.DMPK-12-RV-042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedlacek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manoharan, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boopathy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duysen, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockridge, O.</span></span> <span> </span><span class="NLM_article-title">Butyrylcholinesterase, paraoxonase, and albumin esterase, but not carboxylesterase, are present in human plasma</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>70</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1673</span>– <span class="NLM_lpage">1684</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2005.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1016%2Fj.bcp.2005.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=16213467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKhsLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2005&pages=1673-1684&issue=11&author=B.+Liauthor=M.+Sedlacekauthor=I.+Manoharanauthor=R.+Boopathyauthor=E.+G.+Duysenauthor=P.+Massonauthor=O.+Lockridge&title=Butyrylcholinesterase%2C+paraoxonase%2C+and+albumin+esterase%2C+but+not+carboxylesterase%2C+are+present+in+human+plasma&doi=10.1016%2Fj.bcp.2005.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Butyrylcholinesterase, paraoxonase, and albumin esterase, but not carboxylesterase, are present in human plasma</span></div><div class="casAuthors">Li, Bin; Sedlacek, Meghan; Manoharan, Indumathi; Boopathy, Rathnam; Duysen, Ellen G.; Masson, Patrick; Lockridge, Oksana</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1673-1684</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The goal of this work was to identify the esterases in human plasma and to clarify common misconceptions.  The method for identifying esterases was nondenaturing gradient gel electrophoresis stained for esterase activity.  We report that human plasma contains four esterases: butyrylcholinesterase (E.C. 3.1.1.8), paraoxonase (E.C. 3.1.8.1), acetylcholinesterase (E.C. 3.1.1.7), and albumin.  Butyrylcholinesterase (BChE), paraoxonase (PON1), and albumin are in high enough concns. to contribute significantly to ester hydrolysis.  However, only trace amts. of acetylcholinesterase (AChE) are present.  Monomeric AChE is seen in wild-type as well as in silent BChE plasma.  Albumin has esterase activity with alpha- and beta-naphthylacetate as well as with p-nitrophenyl acetate.  Misconception #1 is that human plasma contains carboxylesterase.  We demonstrate that human plasma contains no carboxylesterase (E.C. 3.1.1.1), in contrast to mouse, rat, rabbit, horse, cat, and tiger that have high amts. of plasma carboxylesterase.  Misconception #2 is that lab. animals have BChE but no AChE in their plasma.  We demonstrate that mice, unlike humans, have substantial amts. of sol. AChE as well as BChE in their plasma.  Plasma from AChE and BChE knockout mice allowed identification of AChE and BChE bands without the use of inhibitors.  Human BChE is irreversibly inhibited by diisopropylfluorophosphate, echothiophate, and paraoxon, but mouse BChE spontaneously reactivates.  Since human plasma contains no carboxylesterase, only BChE, PON1, and albumin esterases need to be considered when evaluating hydrolysis of an ester drug in human plasma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobFY6T-ENpA7Vg90H21EOLACvtfcHk0liujW40nPK6PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKhsLzJ&md5=d9d557d16e4e43d82f1c2d391dc3ac89</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2005.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2005.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DSedlacek%26aufirst%3DM.%26aulast%3DManoharan%26aufirst%3DI.%26aulast%3DBoopathy%26aufirst%3DR.%26aulast%3DDuysen%26aufirst%3DE.%2BG.%26aulast%3DMasson%26aufirst%3DP.%26aulast%3DLockridge%26aufirst%3DO.%26atitle%3DButyrylcholinesterase%252C%2520paraoxonase%252C%2520and%2520albumin%2520esterase%252C%2520but%2520not%2520carboxylesterase%252C%2520are%2520present%2520in%2520human%2520plasma%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2005%26volume%3D70%26issue%3D11%26spage%3D1673%26epage%3D1684%26doi%3D10.1016%2Fj.bcp.2005.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Timm, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madsen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, C.</span></span> <span> </span><span class="NLM_article-title">Cryo-electron microscopy structure of a human PRMT5:MEP50 complex</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>), <span class="NLM_elocation-id">e0193205</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0193205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1371%2Fjournal.pone.0193205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=29518110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWms7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&issue=3&author=D.+E.+Timmauthor=V.+Bowmanauthor=R.+Madsenauthor=C.+Rauch&title=Cryo-electron+microscopy+structure+of+a+human+PRMT5%3AMEP50+complex&doi=10.1371%2Fjournal.pone.0193205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-electron microscopy structure of a human PRMT5:MEP50 complex</span></div><div class="casAuthors">Timm, David E.; Bowman, Valorie; Madsen, Russell; Rauch, Charles</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0193205/1-e0193205/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine Me transferase 5 (PRMT5) is a signaling protein and histone modifying enzyme that is important in many cellular processes, including regulation of eukaryotic gene transcription.  Reported here is a 3.7 Å structure of PRMT5, solved in complex with regulatory binding subunit MEP50 (methylosome assocd. protein 50, WDR77, p44), by single particle (SP) cryo-Electron Microscopy (cryo-EM) using micrographs of particles that are visibly crowded and aggregated.  Despite suboptimal micrograph appearance, this cryo-EM structure is in good agreement with previously reported crystal structures of the complex, which revealed a 450 kDa hetero-octameric assembly having internal D2 symmetry.  The catalytic PRMT5 subunits form a core tetramer and the MEP50 subunits are arranged peripherally in complex with the PRMT5 N-terminal domain.  The cryo-EM reconstruction shows good side chain definition and shows a well-resolved peak for a bound dehydrosinefungin inhibitor mol.  These results demonstrate the applicability of cryo-EM in detg. structures of human protein complexes of biomedical significance and suggests cryo-EM could be further utilized to understand PRMT5 interactions with other biol. important binding proteins and ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVrd4Ga-2QJrVg90H21EOLACvtfcHk0liujW40nPK6PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWms7%252FO&md5=e2410a0d8b80ec2f9c37bb96b390b46f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0193205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0193205%26sid%3Dliteratum%253Aachs%26aulast%3DTimm%26aufirst%3DD.%2BE.%26aulast%3DBowman%26aufirst%3DV.%26aulast%3DMadsen%26aufirst%3DR.%26aulast%3DRauch%26aufirst%3DC.%26atitle%3DCryo-electron%2520microscopy%2520structure%2520of%2520a%2520human%2520PRMT5%253AMEP50%2520complex%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26issue%3D3%26doi%3D10.1371%2Fjournal.pone.0193205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, M.</span></span> <span> </span><span class="NLM_article-title">The role of extrahepatic metabolism in the pharmacokinetics of the targeted covalent inhibitors afatinib, ibrutinib, and neratinib</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1124/dmd.114.061424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1124%2Fdmd.114.061424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=25504185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1OmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=375-384&issue=3&author=Y.+Shibataauthor=M.+Chiba&title=The+role+of+extrahepatic+metabolism+in+the+pharmacokinetics+of+the+targeted+covalent+inhibitors+afatinib%2C+ibrutinib%2C+and+neratinib&doi=10.1124%2Fdmd.114.061424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The role of extrahepatic metabolism in the pharmacokinetics of the targeted covalent inhibitors afatinib, ibrutinib, and neratinib</span></div><div class="casAuthors">Shibata, Yoshihiro; Chiba, Masato</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-384, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Despite the fact that much progress has been made recently in the development of targeted covalent inhibitors (TCIs), their pharmacokinetics (PK) have not been well characterized in the light of extrahepatic clearance (CLextH) by glutathione (GSH)/glutathione S-transferase (GST)-dependent conjugation attributable to the unique electrophilic structure (e.g., acrylamide moiety) of TCI compds.  In the present study, CLextH values were examd. in rat, dog, and monkey to predict the contribution of CLextH to the PK of the TCIs afatinib, ibrutinib, and neratinib in humans.  Afatinib and neratinib both underwent extensive conjugation with GSH in buffer and cytosol fractions of liver and kidney, whereas ibrutinib showed much lower reactivity/susceptibility to GSH/GST-dependent conjugation.  The CLextH in each species was calcd. from the difference between obsd. total body clearance and predicted hepatic clearance (CLH) in cryopreserved hepatocytes suspended in 100% serum of the corresponding species.  The power-based simple allometry relating the CLextH for the unbound compd. to animal body wt. was applicable across species for afatinib and neratinib (R2 ≥ 0.9) but not for ibrutinib (R2 = 0.04).  The predicted AUC after oral administration of afatinib and neratinib agreed reasonably closely with reported values in phase I dose-escalation studies.  Comparisons of CLextH and CLH predicted that CLextH largely detd. the PK of afatinib (>90% as a proportion of total body clearance) and neratinib (∼34%) in humans.  The present method can serve as one of the tools for the optimization of PK in humans at the discovery stage for the development of TCI candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquWC6d6e6DU7Vg90H21EOLACvtfcHk0liujW40nPK6PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1OmsrY%253D&md5=eaac67ce7cea5f2432c5144922ba3dd2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1124%2Fdmd.114.061424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.114.061424%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DY.%26aulast%3DChiba%26aufirst%3DM.%26atitle%3DThe%2520role%2520of%2520extrahepatic%2520metabolism%2520in%2520the%2520pharmacokinetics%2520of%2520the%2520targeted%2520covalent%2520inhibitors%2520afatinib%252C%2520ibrutinib%252C%2520and%2520neratinib%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26issue%3D3%26spage%3D375%26epage%3D384%26doi%3D10.1124%2Fdmd.114.061424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelow, J. M.</span></span> <span> </span><span class="NLM_article-title">A Perspective on the Kinetics of Covalent and Irreversible Inhibition</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1177/1087057116671509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1177%2F1087057116671509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=27703080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=3-20&issue=1&author=J.+M.+Strelow&title=A+Perspective+on+the+Kinetics+of+Covalent+and+Irreversible+Inhibition&doi=10.1177%2F1087057116671509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on the kinetics of covalent and irreversible inhibition</span></div><div class="casAuthors">Strelow, John M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-20</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The clin. and com. success of covalent drugs has prompted a renewed and more deliberate pursuit of covalent and irreversible mechanisms within drug discovery.  A covalent mechanism can produce potent inhibition in a biochem., cellular, or in vivo setting.  In many cases, teams choose to focus on the consequences of the covalent event, defined by an IC50 value.  In a biochem. assay, the IC50 may simply reflect the target protein concn. in the assay.  What has received less attention is the importance of the rate of covalent modification, defined by kinact/KI.  The kinact/KI is a rate const. describing the efficiency of covalent bond formation resulting from the potency (KI) of the first reversible binding event and the max. potential rate (kinact) of inactivation.  In this perspective, it is proposed that the kinact/KI should be employed as a crit. parameter to identify covalent inhibitors, interpret structure-activity relationships (SARs), translate activity from biochem. assays to the cell, and more accurately define selectivity.  It is also proposed that a physiol. relevant kinact/KI and an (unbound) AUC generated from a pharmacokinetic profile reflecting direct exposure of the inhibitor to the target protein are two crit. determinants of in vivo covalent occupancy.  A simple equation is presented to define this relationship and improve the interpretation of covalent and irreversible kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgzDNXDESHDbVg90H21EOLACvtfcHk0ljZbBG-IW7Fdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ&md5=bd404e6b26df20b8761cd38f0f3c3541</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1087057116671509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116671509%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%2BM.%26atitle%3DA%2520Perspective%2520on%2520the%2520Kinetics%2520of%2520Covalent%2520and%2520Irreversible%2520Inhibition%26jtitle%3DSLAS%2520Discov%26date%3D2017%26volume%3D22%26issue%3D1%26spage%3D3%26epage%3D20%26doi%3D10.1177%2F1087057116671509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderton, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butterworth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colclough, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorley, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debreczeni, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finlay, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orme, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span> <span> </span><span class="NLM_article-title">Structure- and reactivity-based development of covalent inhibitors of the activating and gatekeeper mutant forms of the epidermal growth factor receptor (EGFR)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">7025</span>– <span class="NLM_lpage">7048</span>, <span class="refDoi"> DOI: 10.1021/jm400822z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400822z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1CltL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7025-7048&issue=17&author=R.+A.+Wardauthor=M.+J.+Andertonauthor=S.+Ashtonauthor=P.+A.+Bethelauthor=M.+Boxauthor=S.+Butterworthauthor=N.+Colcloughauthor=C.+G.+Chorleyauthor=C.+Chuaquiauthor=D.+A.+Crossauthor=L.+A.+Dakinauthor=J.+E.+Debreczeniauthor=C.+Eberleinauthor=M.+R.+Finlayauthor=G.+B.+Hillauthor=M.+Gristauthor=T.+C.+Klinowskaauthor=C.+Laneauthor=S.+Martinauthor=J.+P.+Ormeauthor=P.+Smithauthor=F.+Wangauthor=M.+J.+Waring&title=Structure-+and+reactivity-based+development+of+covalent+inhibitors+of+the+activating+and+gatekeeper+mutant+forms+of+the+epidermal+growth+factor+receptor+%28EGFR%29&doi=10.1021%2Fjm400822z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure- and Reactivity-Based Development of Covalent Inhibitors of the Activating and Gatekeeper Mutant Forms of the Epidermal Growth Factor Receptor (EGFR)</span></div><div class="casAuthors">Ward, Richard A.; Anderton, Mark J.; Ashton, Susan; Bethel, Paul A.; Box, Matthew; Butterworth, Sam; Colclough, Nicola; Chorley, Christopher G.; Chuaqui, Claudio; Cross, Darren A. E.; Dakin, Les A.; Debreczeni, Judit E.; Eberlein, Cath; Finlay, M. Raymond V.; Hill, George B.; Grist, Matthew; Klinowska, Teresa C. M.; Lane, Clare; Martin, Scott; Orme, Jonathon P.; Smith, Peter; Wang, Fengjiang; Waring, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7025-7048</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of small-mol. inhibitors has been developed to target the double mutant form of the epidermal growth factor receptor (EGFR) tyrosine kinase, which is resistant to treatment with gefitinib and erlotinib.  Our reported compds. also show selectivity over wild-type EGFR.  Guided by mol. modeling, this series was evolved to target a cysteine residue in the ATP binding site via covalent bond formation and demonstrates high levels of activity in cellular models of the double mutant form of EGFR.  In addn., these compds. show significant activity against the activating mutations, which gefitinib and erlotinib target and inhibition of which gives rise to their obsd. clin. efficacy.  A glutathione (GSH)-based assay was used to measure thiol reactivity toward the electrophilic functionality of the inhibitor series, enabling both the identification of a suitable reactivity window for their potency and the development of a reactivity quant. structure-property relationship (QSPR) to support design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKi6oNvtQPEbVg90H21EOLACvtfcHk0ljZbBG-IW7Fdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1CltL%252FN&md5=59c86415d71ac2abe5c6d4486e5d2517</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm400822z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400822z%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DAnderton%26aufirst%3DM.%2BJ.%26aulast%3DAshton%26aufirst%3DS.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DBox%26aufirst%3DM.%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DChorley%26aufirst%3DC.%2BG.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDebreczeni%26aufirst%3DJ.%2BE.%26aulast%3DEberlein%26aufirst%3DC.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DHill%26aufirst%3DG.%2BB.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DKlinowska%26aufirst%3DT.%2BC.%26aulast%3DLane%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DStructure-%2520and%2520reactivity-based%2520development%2520of%2520covalent%2520inhibitors%2520of%2520the%2520activating%2520and%2520gatekeeper%2520mutant%2520forms%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D17%26spage%3D7025%26epage%3D7048%26doi%3D10.1021%2Fjm400822z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">9977</span>– <span class="NLM_lpage">9989</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01111</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01111" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFOisLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9977-9989&issue=17&author=Y.+Shenauthor=G.+Gaoauthor=X.+Yuauthor=H.+Kimauthor=L.+Wangauthor=L.+Xieauthor=M.+Schwarzauthor=X.+Chenauthor=E.+Guccioneauthor=J.+Liuauthor=M.+T.+Bedfordauthor=J.+Jin&title=Discovery+of+First-in-Class+Protein+Arginine+Methyltransferase+5+%28PRMT5%29+Degraders&doi=10.1021%2Facs.jmedchem.0c01111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders</span></div><div class="casAuthors">Shen, Yudao; Gao, Guozhen; Yu, Xufen; Kim, Huensuk; Wang, Li; Xie, Ling; Schwarz, Megan; Chen, Xian; Guccione, Ernesto; Liu, Jing; Bedford, Mark T.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9977-9989</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The aberrant expression of protein arginine methyltransferase 5 (PRMT5) has been assocd. with multiple cancers.  Using the proteolysis targeting chimera technol., we discovered a first-in-class PRMT5 degrader 15 (MS4322).  Here, we report the design, synthesis, and characterization of compd. 15 and two structurally similar controls 17 (MS4370) and 21 (MS4369), with impaired binding to the von Hippel-Lindau E3 ligase and PRMT5, resp.  Compd. 15, but not 17 and 21, effectively reduced the PRMT5 protein level in MCF-7 cells.  Our mechanism studies indicate that compd. 15 degraded PRMT5 in an E3 ligase- and proteasome-dependent manner.  Compd. 15 also effectively reduced the PRMT5 protein level and inhibited growth in multiple cancer cell lines.  Moreover, compd. 15 was highly selective for PRMT5 in a global proteomic study and exhibited good plasma exposure in mice.  Collectively, compd. 15 and its two controls 17 and 21 are valuable chem. tools for exploring the PRMT5 functions in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZoL-YbkPwxrVg90H21EOLACvtfcHk0ljZbBG-IW7Fdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOisLrE&md5=5400a4b400d1a420eff5b24f735de462</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01111%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520First-in-Class%2520Protein%2520Arginine%2520Methyltransferase%25205%2520%2528PRMT5%2529%2520Degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D17%26spage%3D9977%26epage%3D9989%26doi%3D10.1021%2Facs.jmedchem.0c01111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grant, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grigorieff, N.</span></span> <span> </span><span class="NLM_article-title">cisTEM, user-friendly software for single-particle image processing</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e35383</span>, <span class="refDoi"> DOI: 10.7554/eLife.35383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.7554%2FeLife.35383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=29513216" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=e35383&author=T.+Grantauthor=A.+Rohouauthor=N.+Grigorieff&title=cisTEM%2C+user-friendly+software+for+single-particle+image+processing&doi=10.7554%2FeLife.35383"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.7554%2FeLife.35383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.35383%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DT.%26aulast%3DRohou%26aufirst%3DA.%26aulast%3DGrigorieff%26aufirst%3DN.%26atitle%3DcisTEM%252C%2520user-friendly%2520software%2520for%2520single-particle%2520image%2520processing%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26spage%3De35383%26doi%3D10.7554%2FeLife.35383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marx, M. A.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery Of MRTX9768, A Synthetic Lethal-Based Inhibitor Designed To Bind The PRMT5·TA Complex And Selectively Target MTAPDEL Tumors</span>. Proceedings from the  <i>AACR Annual Meeting 2021</i>, April 10–15 and May 17–21, 2021; <span class="NLM_publisher-name">American Association for Cancer Research</span>: <span class="NLM_publisher-loc">Philadelphia, PA</span>, <span class="NLM_year">2021</span>; p  <span class="NLM_fpage">LB003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&pages=LB003&author=M.+A.+Marx&title=Fragment-Based+Discovery+Of+MRTX9768%2C+A+Synthetic+Lethal-Based+Inhibitor+Designed+To+Bind+The+PRMT5%C2%B7TA+Complex+And+Selectively+Target+MTAPDEL+Tumors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DMarx%26aufirst%3DM.%2BA.%26atitle%3DFragment-Based%2520Discovery%2520Of%2520MRTX9768%252C%2520A%2520Synthetic%2520Lethal-Based%2520Inhibitor%2520Designed%2520To%2520Bind%2520The%2520PRMT5%25C2%25B7TA%2520Complex%2520And%2520Selectively%2520Target%2520MTAPDEL%2520Tumors%26jtitle%3DAACR%2520Annual%2520Meeting%25202021%26pub%3DAmerican%2520Association%2520for%2520Cancer%2520Research%26date%3D2021%26spage%3DLB003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malik, R.</span>; <span class="NLM_string-name">Park, P. K.</span>; <span class="NLM_string-name">Barbieri, C.</span>; <span class="NLM_string-name">Blat, Y.</span>; <span class="NLM_string-name">Sheriff, S.</span>; <span class="NLM_string-name">Weigelt, C.</span>; <span class="NLM_string-name">Kopcho, L.</span>; <span class="NLM_string-name">Celiktas, M.</span>; <span class="NLM_string-name">Ruzanov, M.</span>; <span class="NLM_string-name">Naglich, J.</span>; <span class="NLM_string-name">Price, J.</span>; <span class="NLM_string-name">Harner, M.</span>; <span class="NLM_string-name">O’Malley, K.</span>; <span class="NLM_string-name">Den, J.</span>; <span class="NLM_string-name">Schmitz, W.</span>; <span class="NLM_string-name">Li, G.</span>; <span class="NLM_string-name">Ruan, Z.</span>; <span class="NLM_string-name">Qin, L.</span>; <span class="NLM_string-name">Duke, G.</span>; <span class="NLM_string-name">Rodrigo, I.</span>; <span class="NLM_string-name">Witmer, M.</span>; <span class="NLM_string-name">Harden, D.</span>; <span class="NLM_string-name">Arey, B.</span>; <span class="NLM_string-name">Soars, M.</span>; <span class="NLM_string-name">Fink, B.</span>; <span class="NLM_string-name">Gavai, A.</span>; <span class="NLM_string-name">Vite, G.</span>; <span class="NLM_string-name">Voliva, C. F.</span></span> <span> </span><span class="NLM_article-title">A novel MTA non-competitive PRMT5 inhibitor</span>. Proceedings from the  <i>AACR Annual Meeting 2021</i>, April 10–15 and May 17–21, 2021; <span class="NLM_publisher-name">American Association for Cancer Research</span>: <span class="NLM_publisher-loc">Philadelphia, PA</span>, <span class="NLM_year">2021</span>;  <span class="NLM_fpage">1140</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1158%2F1538-7445.AM2021-1140" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&pages=1140&author=R.+Malik&author=P.+K.+Park&author=C.+Barbieri&author=Y.+Blat&author=S.+Sheriff&author=C.+Weigelt&author=L.+Kopcho&author=M.+Celiktas&author=M.+Ruzanov&author=J.+Naglich&author=J.+Price&author=M.+Harner&author=K.+O%E2%80%99Malley&author=J.+Den&author=W.+Schmitz&author=G.+Li&author=Z.+Ruan&author=L.+Qin&author=G.+Duke&author=I.+Rodrigo&author=M.+Witmer&author=D.+Harden&author=B.+Arey&author=M.+Soars&author=B.+Fink&author=A.+Gavai&author=G.+Vite&author=C.+F.+Voliva&title=A+novel+MTA+non-competitive+PRMT5+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2021-1140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26id%3Ddoi%3A10.1158%252F1538-7445.AM2021-1140%26sid%3Dliteratum%253Aachs%26aulast%3DMalik%26aufirst%3DR.%26atitle%3DA%2520novel%2520MTA%2520non-competitive%2520PRMT5%2520inhibitor%26jtitle%3DAACR%2520Annual%2520Meeting%25202021%26pub%3DAmerican%2520Association%2520for%2520Cancer%2520Research%26date%3D2021%26spage%3D1140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Hit-finding activities. (a) Low chart for inhibitors of the PBM:PRMT5 interaction. (b) Chemical structure of <b>1</b>. (c) Dose-dependent displacement of a fluorescently labeled RIOK1-derived PBM peptide probe by <b>1</b>, as measured by FP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Mechanism of binding. (a) Compound <b>1</b> (magenta) binds distal to the catalytic site (SAM analog, sinefungin, red). A protomer of PRMT5 (light green) and WDR77 (dark green) are shown as cartoons. The three other protomer units from the hetero-octamer complex are shown in gray (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0P">6V0P</a>). (b) Crystal structure of compound <b>1</b> (magenta) bound to PRMT5 (green). Residues described in the text are labeled. Two alternate conformations of Y286 are shown (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0P">6V0P</a>). (c) Cryo-EM structure of compound <b>6</b> (cyan) bound to PRMT5 (green) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7M05">7M05</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. (A) Syntheses of <b>1</b> and <b>6</b> and (B) Synthesis of Sulfonamides <b>34</b> and <b>36</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>6</b> is a covalent binder. (a) Compound <b>6</b> shows time-dependent displacement of a fluorescently labeled PBM peptide probe as measured by FP. (b) FP competition assay with compound <b>6</b> (squares) or pICln 13-mer control peptide (circles) displacing the fluorescently labeled PBM probe from WT PRMT5 (blue) and C278A mutant (green). (c) Deconvoluted mass spectra for WT and C278A PRMT5 ± compound <b>6</b>. (d) SPR competition assay. PRMT5:WDR77 complex was preincubated with compound <b>6</b> or DMSO and then immobilized to the chip surface. Full-length pICln protein was titrated as analyte.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0004.jpeg" id="rightTab-GRAPHIC-d7e1056-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinetic formation of the covalent adducts between 6 and PRMT5. (a) Dose-dependent kinetics of <b>6</b>-PRMT5 as determined by intact mass spectrometry. (b) <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> analysis of covalent modification by <b>26</b> (blue), <b>25</b> (red), <b>20</b> (green), and <b>6</b> (purple) as determined by intact mass spectrometry. (c) Kinetic parameters for select compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0021.jpeg" id="rightTab-GRAPHIC-d7e1259-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cell activities of <b>27</b>. (a) PRMT5 adduct formation using <b>27</b>. Expi293 cells were transiently transfected for 48 h with an HA-tagged PRMT5:WDR77 complex before treatment with <b>27</b>. Cultures were harvested at 6 h and PRMT5 complex isolated by HA affinity and analyzed for modification by LC/MS. (b) PRMT5-RIOK1 NanoBiT assay in permeabilized cells. 293T cells stably expressing SmBiT-PRMT5 and LgBiT-RIOK1 proteins. (c) NanoBit assay performed in intact cells after treatment with <b>27</b> or <b>30</b>. (d) Left panel: WB analysis of total symmetric arginine dimethylation levels in MTAP–/– HCT116 cells, in response to <b>27</b> or <b>30</b>. Right panel: WB analysis of total symmetric arginine dimethylation levels in MTAP–/– HCT116 cells overexpressing the PRMT5 ADA mutant which is unable to bind to the PBM peptide, with and without KO of endogenous PRMT5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Pyridine Analogs (A) <b>15</b>, (B) <b>16</b>, and (C) <b>17</b> and (D) piperidine <b>18</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of (A) <b>19</b>, (B) <b>20</b>, and (C) <b>21</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Syntheses of Fluoro Pyridazinones (A) <b>23</b>, (B) <b>24</b>, and (C) <b>25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/medium/jm1c00507_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Syntheses of Monochloropyridazinones (A) <b>26</b>, (B) <b>27</b>, and (C) <b>30</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00507/20210803/images/large/jm1c00507_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00507&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i112">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86583" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86583" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 24 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musiani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bok, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massignani, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabaglio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ippolito, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuomo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozbek, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorgati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghoshdastider, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonaldi, T.</span></span> <span> </span><span class="NLM_article-title">Proteomics profiling of arginine methylation defines PRMT5 substrate specificity</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">575</span>),  <span class="NLM_fpage">eaat8388</span>, <span class="refDoi"> DOI: 10.1126/scisignal.aat8388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1126%2Fscisignal.aat8388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=30940768" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=eaat8388&issue=575&author=D.+Musianiauthor=J.+Bokauthor=E.+Massignaniauthor=L.+Wuauthor=T.+Tabaglioauthor=M.+R.+Ippolitoauthor=A.+Cuomoauthor=U.+Ozbekauthor=H.+Zorgatiauthor=U.+Ghoshdastiderauthor=R.+C.+Robinsonauthor=E.+Guccioneauthor=T.+Bonaldi&title=Proteomics+profiling+of+arginine+methylation+defines+PRMT5+substrate+specificity&doi=10.1126%2Fscisignal.aat8388"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.aat8388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.aat8388%26sid%3Dliteratum%253Aachs%26aulast%3DMusiani%26aufirst%3DD.%26aulast%3DBok%26aufirst%3DJ.%26aulast%3DMassignani%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DTabaglio%26aufirst%3DT.%26aulast%3DIppolito%26aufirst%3DM.%2BR.%26aulast%3DCuomo%26aufirst%3DA.%26aulast%3DOzbek%26aufirst%3DU.%26aulast%3DZorgati%26aufirst%3DH.%26aulast%3DGhoshdastider%26aufirst%3DU.%26aulast%3DRobinson%26aufirst%3DR.%2BC.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBonaldi%26aufirst%3DT.%26atitle%3DProteomics%2520profiling%2520of%2520arginine%2520methylation%2520defines%2520PRMT5%2520substrate%2520specificity%26jtitle%3DSci.%2520Signaling%26date%3D2019%26volume%3D12%26issue%3D575%26spage%3Deaat8388%26doi%3D10.1126%2Fscisignal.aat8388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Weck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gampa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sovath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caothien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caushi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortes-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desplat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwoske, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsidjani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schouwey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiedt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Project DRIVE: A Compendium of Cancer Dependencies and Synthetic Lethal Relationships Uncovered by Large-Scale, Deep RNAi Screening</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1016%2Fj.cell.2017.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=28753431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=577-592&issue=3&author=E.+R.+McDonaldauthor=A.+de+Weckauthor=M.+R.+Schlabachauthor=E.+Billyauthor=K.+J.+Mavrakisauthor=G.+R.+Hoffmanauthor=D.+Belurauthor=D.+Castellettiauthor=E.+Friasauthor=K.+Gampaauthor=J.+Goljiauthor=I.+Kaoauthor=L.+Liauthor=P.+Megelauthor=T.+A.+Perkinsauthor=N.+Ramadanauthor=D.+A.+Ruddyauthor=S.+J.+Silverauthor=S.+Sovathauthor=M.+Stumpauthor=O.+Weberauthor=R.+Widmerauthor=J.+Yuauthor=K.+Yuauthor=Y.+Yueauthor=D.+Abramowskiauthor=E.+Ackleyauthor=R.+Barrettauthor=J.+Bergerauthor=J.+L.+Bernardauthor=R.+Billigauthor=S.+M.+Brachmannauthor=F.+Buxtonauthor=R.+Caothienauthor=J.+X.+Caushiauthor=F.+S.+Chungauthor=M.+Cortes-Crosauthor=R.+S.+deBeaumontauthor=C.+Delaunayauthor=A.+Desplatauthor=W.+Duongauthor=D.+A.+Dwoskeauthor=R.+S.+Eldridgeauthor=A.+Farsidjaniauthor=F.+Fengauthor=J.+Fengauthor=D.+Flemmingauthor=W.+Forresterauthor=G.+G.+Galliauthor=Z.+Gaoauthor=F.+Gauterauthor=V.+Gibajaauthor=K.+Haasauthor=M.+Hattenbergerauthor=T.+Hoodauthor=K.+E.+Hurovauthor=Z.+Jaganiauthor=M.+Jenalauthor=J.+A.+Johnsonauthor=M.+D.+Jonesauthor=A.+Kapoorauthor=J.+Kornauthor=J.+Liuauthor=Q.+Liuauthor=S.+Liuauthor=Y.+Liuauthor=A.+T.+Looauthor=K.+J.+Macchiauthor=T.+Martinauthor=G.+McAllisterauthor=A.+Meyerauthor=S.+Molleauthor=R.+A.+Pagliariniauthor=T.+Phadkeauthor=B.+Repkoauthor=T.+Schouweyauthor=F.+Shanahanauthor=Q.+Shenauthor=C.+Stammauthor=C.+Stephanauthor=V.+M.+Stuckeauthor=R.+Tiedtauthor=M.+Varadarajanauthor=K.+Venkatesanauthor=A.+C.+Vitariauthor=M.+Wallrothauthor=J.+Weilerauthor=J.+Zhangauthor=C.+Mickaninauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=A.+Laiauthor=H.+Bitterauthor=E.+Leesauthor=N.+Keenauthor=A.+Kauffmannauthor=F.+Stegmeierauthor=F.+Hofmannauthor=T.+Schmelzleauthor=W.+R.+Sellers&title=Project+DRIVE%3A+A+Compendium+of+Cancer+Dependencies+and+Synthetic+Lethal+Relationships+Uncovered+by+Large-Scale%2C+Deep+RNAi+Screening&doi=10.1016%2Fj.cell.2017.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Project DRIVE: A compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span></div><div class="casAuthors">McDonald, E. Robert, III; de Weck, Antoine; Schlabach, Michael R.; Billy, Eric; Mavrakis, Konstantinos J.; Hoffman, Gregory R.; Belur, Dhiren; Castelletti, Deborah; Frias, Elizabeth; Gampa, Kalyani; Golji, Javad; Kao, Iris; Li, Li; Megel, Philippe; Perkins, Thomas A.; Ramadan, Nadire; Ruddy, David A.; Silver, Serena J.; Sovath, Sosathya; Stump, Mark; Weber, Odile; Widmer, Roland; Yu, Jianjun; Yu, Kristine; Yue, Yingzi; Abramowski, Dorothee; Ackley, Elizabeth; Barrett, Rosemary; Berger, Joel; Bernard, Julie L.; Billig, Rebecca; Brachmann, Saskia M.; Buxton, Frank; Caothien, Roger; Caushi, Justina X.; Chung, Franklin S.; Cortes-Cros, Marta; de Beaumont, Rosalie S.; Delaunay, Clara; Desplat, Aurore; Duong, William; Dwoske, Donald A.; Eldridge, Richard S.; Farsidjani, Ali; Feng, Fei; Feng, Jia Jia; Flemming, Daisy; Forrester, William; Galli, Giorgio G.; Gao, Zhenhai; Gauter, Francois; Gibaja, Veronica; Haas, Kristy; Hattenberger, Marc; Hood, Tami; Hurov, Kristen E.; Jagani, Zainab; Jenal, Mathias; Johnson, Jennifer A.; Jones, Michael D.; Kapoor, Avnish; Korn, Joshua; Liu, Jilin; Liu, Qiumei; Liu, Shumei; Liu, Yue; Loo, Alice T.; Macchi, Kaitlin J.; Martin, Typhaine; McAllister, Gregory; Meyer, Amandine; Molle, Sandra; Pagliarini, Raymond A.; Phadke, Tanushree; Repko, Brian; Schouwey, Tanja; Shanahan, Frances; Shen, Qiong; Stamm, Christelle; Stephan, Christine; Stucke, Volker M.; Tiedt, Ralph; Varadarajan, Malini; Venkatesan, Kavitha; Vitari, Alberto C.; Wallroth, Marco; Weiler, Jan; Zhang, Jing; Mickanin, Craig; Myer, Vic E.; Porter, Jeffery A.; Lai, Albert; Bitter, Hans; Lees, Emma; Keen, Nicholas; Kauffmann, Audrey; Stegmeier, Frank; Hofmann, Francesco; Schmelzle, Tobias; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">577-592.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elucidation of the mutational landscape of human cancer has progressed rapidly and been accompanied by the development of therapeutics targeting mutant oncogenes.  However, a comprehensive mapping of cancer dependencies has lagged behind and the discovery of therapeutic targets for counteracting tumor suppressor gene loss is needed.  To identify vulnerabilities relevant to specific cancer subtypes, we conducted a large-scale RNAi screen in which viability effects of mRNA knockdown were assessed for 7837 genes using an av. of 20 shRNAs per gene in 398 cancer cell lines.  We describe findings of this screen, outlining the classes of cancer dependency genes and their relationships to genetic, expression, and lineage features.  In addn., we describe robust gene-interaction networks recapitulating both protein complexes and functional cooperation among complexes and pathways.  This dataset along with a web portal is provided to the community to assist in the discovery and translation of new therapeutic approaches for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUs33_s2nC7Vg90H21EOLACvtfcHk0liDtGidFTqtZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI&md5=41592e684cc8f2cd28117cb3303f6120</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3Dde%2BWeck%26aufirst%3DA.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DBelur%26aufirst%3DD.%26aulast%3DCastelletti%26aufirst%3DD.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DGampa%26aufirst%3DK.%26aulast%3DGolji%26aufirst%3DJ.%26aulast%3DKao%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMegel%26aufirst%3DP.%26aulast%3DPerkins%26aufirst%3DT.%2BA.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DSilver%26aufirst%3DS.%2BJ.%26aulast%3DSovath%26aufirst%3DS.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DWidmer%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DAckley%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DJ.%2BL.%26aulast%3DBillig%26aufirst%3DR.%26aulast%3DBrachmann%26aufirst%3DS.%2BM.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DCaothien%26aufirst%3DR.%26aulast%3DCaushi%26aufirst%3DJ.%2BX.%26aulast%3DChung%26aufirst%3DF.%2BS.%26aulast%3DCortes-Cros%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%2BS.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DDesplat%26aufirst%3DA.%26aulast%3DDuong%26aufirst%3DW.%26aulast%3DDwoske%26aufirst%3DD.%2BA.%26aulast%3DEldridge%26aufirst%3DR.%2BS.%26aulast%3DFarsidjani%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFlemming%26aufirst%3DD.%26aulast%3DForrester%26aufirst%3DW.%26aulast%3DGalli%26aufirst%3DG.%2BG.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGauter%26aufirst%3DF.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DHattenberger%26aufirst%3DM.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DHurov%26aufirst%3DK.%2BE.%26aulast%3DJagani%26aufirst%3DZ.%26aulast%3DJenal%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DKapoor%26aufirst%3DA.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLoo%26aufirst%3DA.%2BT.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DMolle%26aufirst%3DS.%26aulast%3DPagliarini%26aufirst%3DR.%2BA.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DRepko%26aufirst%3DB.%26aulast%3DSchouwey%26aufirst%3DT.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DC.%26aulast%3DStucke%26aufirst%3DV.%2BM.%26aulast%3DTiedt%26aufirst%3DR.%26aulast%3DVaradarajan%26aufirst%3DM.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DWallroth%26aufirst%3DM.%26aulast%3DWeiler%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DProject%2520DRIVE%253A%2520A%2520Compendium%2520of%2520Cancer%2520Dependencies%2520and%2520Synthetic%2520Lethal%2520Relationships%2520Uncovered%2520by%2520Large-Scale%252C%2520Deep%2520RNAi%2520Screening%26jtitle%3DCell%26date%3D2017%26volume%3D170%26issue%3D3%26spage%3D577%26epage%3D592%26doi%3D10.1016%2Fj.cell.2017.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span> <span> </span><span class="NLM_article-title">Role of protein arginine methyltransferase 5 in human cancers</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">108790</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2019.108790</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1016%2Fj.biopha.2019.108790" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=30903920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtV2isb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2019&pages=108790&author=W.+Xiaoauthor=X.+Chenauthor=L.+Liuauthor=Y.+Shuauthor=M.+Zhangauthor=Y.+Zhong&title=Role+of+protein+arginine+methyltransferase+5+in+human+cancers&doi=10.1016%2Fj.biopha.2019.108790"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Role of protein arginine methyltransferase 5 in human cancers</span></div><div class="casAuthors">Xiao, Wendi; Chen, Xiaoqing; Liu, Lisa; Shu, Yuansen; Zhang, Min; Zhong, Yucheng</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108790</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) play important roles in protein methylation.  PRMT5 is the major type II arginine methyltransferase that catalyzes the transfer of two Me groups sym. to the arginine residues of either histone or non-histone proteins.  In recent years, increasing evidence has shown that PRMT5, as an oncogene, plays an indispensable regulatory role in the pathol. progression of several human cancers by promoting the proliferation, invasion, and migration of cancer cells.  PRMT5 is overexpressed in many malignant tumors and plays an important role in the occurrence and development of cancer, which suggests that PRMT5 may become a potential biomarker or therapeutic target of cancer.  This article reviews the biol. function, mechanism, and clin. significance of PRMT5 in tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPOq7QOZDBmbVg90H21EOLACvtfcHk0lizzBq5rxHtNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtV2isb4%253D&md5=db7dfee60fc048cd7408eb34d3cd1d6a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2019.108790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2019.108790%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DShu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DY.%26atitle%3DRole%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520in%2520human%2520cancers%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2019%26volume%3D114%26spage%3D108790%26doi%3D10.1016%2Fj.biopha.2019.108790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, B.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality as an engine for cancer drug target discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0046-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1038%2Fs41573-019-0046-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=31712683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFegs7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=23-38&issue=1&author=A.+Huangauthor=L.+A.+Garrawayauthor=A.+Ashworthauthor=B.+Weber&title=Synthetic+lethality+as+an+engine+for+cancer+drug+target+discovery&doi=10.1038%2Fs41573-019-0046-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethality as an engine for cancer drug target discovery</span></div><div class="casAuthors">Huang, Alan; Garraway, Levi A.; Ashworth, Alan; Weber, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-38</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The first wave of genetically targeted therapies for cancer focused on drugging gene products that are recurrently mutated in specific cancer types.  However, mutational anal. of tumors has largely been exhausted as a strategy for the identification of new cancer targets that are druggable with conventional approaches.  Furthermore, some known genetic drivers of cancer have not been directly targeted yet owing to their mol. structure (undruggable oncogenes) or because they result in functional loss (tumor suppressor genes).  Functional genomic screening based on the genetic concept of synthetic lethality provides an avenue to discover drug targets in all these areas.  Although synthetic lethality is not a new idea, recent advances, including CRISPR-based gene editing, have made possible systematic screens for synthetic lethal drug targets in human cancers.  Such approaches have broad potential to drive the discovery of the next wave of genetic cancer targets and ultimately the introduction of effective medicines that are still needed for most cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUGZtPqjLRlrVg90H21EOLACvtfcHk0liFXruvJhc_ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFegs7jK&md5=227059e12105feecc402ea6ff4f98640</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0046-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0046-z%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DA.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DWeber%26aufirst%3DB.%26atitle%3DSynthetic%2520lethality%2520as%2520an%2520engine%2520for%2520cancer%2520drug%2520target%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2020%26volume%3D19%26issue%3D1%26spage%3D23%26epage%3D38%26doi%3D10.1038%2Fs41573-019-0046-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deWeck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan-Neale, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kipp, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprague, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccone, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span> (<span class="NLM_issue">6278</span>),  <span class="NLM_fpage">1208</span>– <span class="NLM_lpage">1213</span>, <span class="refDoi"> DOI: 10.1126/science.aad5944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1126%2Fscience.aad5944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=26912361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1208-1213&issue=6278&author=K.+J.+Mavrakisauthor=E.+R.+McDonaldauthor=M.+R.+Schlabachauthor=E.+Billyauthor=G.+R.+Hoffmanauthor=A.+deWeckauthor=D.+A.+Ruddyauthor=K.+Venkatesanauthor=J.+Yuauthor=G.+McAllisterauthor=M.+Stumpauthor=R.+deBeaumontauthor=S.+Hoauthor=Y.+Yueauthor=Y.+Liuauthor=Y.+Yan-Nealeauthor=G.+Yangauthor=F.+Linauthor=H.+Yinauthor=H.+Gaoauthor=D.+R.+Kippauthor=S.+Zhaoauthor=J.+T.+McNamaraauthor=E.+R.+Spragueauthor=B.+Zhengauthor=Y.+Linauthor=Y.+S.+Choauthor=J.+Guauthor=K.+Crawfordauthor=D.+Cicconeauthor=A.+C.+Vitariauthor=A.+Laiauthor=V.+Capkaauthor=K.+Hurovauthor=J.+A.+Porterauthor=J.+Tallaricoauthor=C.+Mickaninauthor=E.+Leesauthor=R.+Pagliariniauthor=N.+Keenauthor=T.+Schmelzleauthor=F.+Hofmannauthor=F.+Stegmeierauthor=W.+R.+Sellers&title=Disordered+methionine+metabolism+in+MTAP%2FCDKN2A-deleted+cancers+leads+to+dependence+on+PRMT5&doi=10.1126%2Fscience.aad5944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5</span></div><div class="casAuthors">Mavrakis, Konstantinos J.; McDonald, E. Robert, III; Schlabach, Michael R.; Billy, Eric; Hoffman, Gregory R.; de Weck, Antoine; Ruddy, David A.; Venkatesan, Kavitha; Yu, Jianjun; McAllister, Gregg; Stump, Mark; de Beaumont, Rosalie; Ho, Samuel; Yue, Yingzi; Liu, Yue; Yan-Neale, Yan; Yang, Guizhi; Lin, Fallon; Yin, Hong; Gao, Hui; Kipp, D. Randal; Zhao, Songping; McNamara, Joshua T.; Sprague, Elizabeth R.; Zheng, Bing; Lin, Ying; Cho, Young Shin; Gu, Justin; Crawford, Kenneth; Ciccone, David; Vitari, Alberto C.; Lai, Albert; Capka, Vladimir; Hurov, Kristen; Porter, Jeffery A.; Tallarico, John; Mickanin, Craig; Lees, Emma; Pagliarini, Raymond; Keen, Nicholas; Schmelzle, Tobias; Hofmann, Francesco; Stegmeier, Frank; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1208-1213</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">5-Methylthioadenosine phosphorylase (MTAP) is a key enzyme in the methionine salvage pathway.  The MTAP gene is frequently deleted in human cancers because of its chromosomal proximity to the tumor suppressor gene CDKN2A.  By interrogating data from a large-scale short hairpin RNA-mediated screen across 390 cancer cell line models, we found that the viability of MTAP-deficient cancer cells is impaired by depletion of the protein arginine methyltransferase PRMT5.  MTAP-deleted cells accumulate the metabolite methylthioadenosine (MTA), which we found to inhibit PRMT5 methyltransferase activity.  Deletion of MTAP in MTAP-proficient cells rendered them sensitive to PRMT5 depletion.  Conversely, reconstitution of MTAP in an MTAP-deficient cell line rescued PRMT5 dependence.  Thus, MTA accumulation in MTAP-deleted cancers creates a hypomorphic PRMT5 state that is selectively sensitized toward further PRMT5 inhibition.  Inhibitors of PRMT5 that leverage this dysregulated metabolic state merit further investigation as a potential therapy for MTAP/CDKN2A-deleted tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNsuzYFo1qn7Vg90H21EOLACvtfcHk0liFXruvJhc_ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtr0%253D&md5=d77531ea765db9762d6815840d39ee4a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5944%26sid%3Dliteratum%253Aachs%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DdeWeck%26aufirst%3DA.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%26aulast%3DHo%26aufirst%3DS.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYan-Neale%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DLin%26aufirst%3DF.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DKipp%26aufirst%3DD.%2BR.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DMcNamara%26aufirst%3DJ.%2BT.%26aulast%3DSprague%26aufirst%3DE.%2BR.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DCho%26aufirst%3DY.%2BS.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DK.%26aulast%3DCiccone%26aufirst%3DD.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DCapka%26aufirst%3DV.%26aulast%3DHurov%26aufirst%3DK.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DTallarico%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DPagliarini%26aufirst%3DR.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DDisordered%2520methionine%2520metabolism%2520in%2520MTAP%252FCDKN2A-deleted%2520cancers%2520leads%2520to%2520dependence%2520on%2520PRMT5%26jtitle%3DScience%26date%3D2016%26volume%3D351%26issue%3D6278%26spage%3D1208%26epage%3D1213%26doi%3D10.1126%2Fscience.aad5944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krasinskas, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieply, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dacic, S.</span></span> <span> </span><span class="NLM_article-title">CDKN2A and MTAP deletions in peritoneal mesotheliomas are correlated with loss of p16 protein expression and poor survival</span>. <i>Mod. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/modpathol.2009.186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1038%2Fmodpathol.2009.186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=20081810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVCqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2010&pages=531-538&issue=4&author=A.+M.+Krasinskasauthor=D.+L.+Bartlettauthor=K.+Cieplyauthor=S.+Dacic&title=CDKN2A+and+MTAP+deletions+in+peritoneal+mesotheliomas+are+correlated+with+loss+of+p16+protein+expression+and+poor+survival&doi=10.1038%2Fmodpathol.2009.186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">CDKN2A and MTAP deletions in peritoneal mesotheliomas are correlated with loss of p16 protein expression and poor survival</span></div><div class="casAuthors">Krasinskas, Alyssa M.; Bartlett, David L.; Cieply, Kathleen; Dacic, Sanja</div><div class="citationInfo"><span class="NLM_cas:title">Modern Pathology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-538</span>CODEN:
                <span class="NLM_cas:coden">MODPEO</span>;
        ISSN:<span class="NLM_cas:issn">0893-3952</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Homozygous deletion of CDKN2A (p16) is one of the most common genetic alterations in pleural mesotheliomas, occurring in up to 74% of cases.  MTAP resides in the same gene cluster of the 9p21 region and is co-deleted in the majority of CDKN2A deleted cases.  This study examines the genetic alterations in peritoneal mesotheliomas, which may have a different pathogenesis than their pleural counterparts.  Twenty-six cases of peritoneal mesotheliomas in a triplicate tissue microarray were studied.  Dual-color fluorescence in situ hybridization was performed with CDKN2A and MTAP locus-specific probes.  Nine of 26 (35%) peritoneal mesotheliomas had homozygous deletion of CDKN2A; MTAP was co-deleted in every case.  All cases with CDKN2A deletions had loss of p16 protein expression; five cases had loss of p16 protein without evidence of CDKN2A deletions.  All patients with CDKN2A deletions were men (P, NS) and were significantly older (mean, 63 years) than the patients with no deletions (mean, 52 years) (P=0.033, t-test).  An assocn. with asbestos exposure could not be proved in this study.  Similar to pleural mesotheliomas, patients with CDKN2A deletions and loss of p16 protein expression had worse overall and disease-specific survival (P=0.010 and 0.006, resp.; Kaplan-Meier log rank).  Detection of CDKN2A-MTAP co-deletion in peritoneal mesotheliomas, coupled with a p16 immunohistochem. stain as an inexpensive screening tool, can help identify those patients who may have an unfavorable outcome after aggressive cytoreductive surgery combined with hyperthermic i.p. chemotherapy and those who may respond to targeted therapy of the MTAP pathway.  Modern Pathol. (2010) 23, 531-538; doi:10.1038/modpathol.2009.186; published online 15 Jan. 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6etmI06rXTbVg90H21EOLACvtfcHk0ljyL09JT8XPmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVCqsbY%253D&md5=f1c0141447219e09e8e755aa80bc0b97</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fmodpathol.2009.186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmodpathol.2009.186%26sid%3Dliteratum%253Aachs%26aulast%3DKrasinskas%26aufirst%3DA.%2BM.%26aulast%3DBartlett%26aufirst%3DD.%2BL.%26aulast%3DCieply%26aufirst%3DK.%26aulast%3DDacic%26aufirst%3DS.%26atitle%3DCDKN2A%2520and%2520MTAP%2520deletions%2520in%2520peritoneal%2520mesotheliomas%2520are%2520correlated%2520with%2520loss%2520of%2520p16%2520protein%2520expression%2520and%2520poor%2520survival%26jtitle%3DMod.%2520Pathol.%26date%3D2010%26volume%3D23%26issue%3D4%26spage%3D531%26epage%3D538%26doi%3D10.1038%2Fmodpathol.2009.186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielski, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span> <span> </span><span class="NLM_article-title">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span> (<span class="NLM_issue">6278</span>),  <span class="NLM_fpage">1214</span>– <span class="NLM_lpage">1218</span>, <span class="refDoi"> DOI: 10.1126/science.aad5214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1126%2Fscience.aad5214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=26912360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1214-1218&issue=6278&author=G.+V.+Kryukovauthor=F.+H.+Wilsonauthor=J.+R.+Ruthauthor=J.+Paulkauthor=A.+Tsherniakauthor=S.+E.+Marlowauthor=F.+Vazquezauthor=B.+A.+Weirauthor=M.+E.+Fitzgeraldauthor=M.+Tanakaauthor=C.+M.+Bielskiauthor=J.+M.+Scottauthor=C.+Dennisauthor=G.+S.+Cowleyauthor=J.+S.+Boehmauthor=D.+E.+Rootauthor=T.+R.+Golubauthor=C.+B.+Clishauthor=J.+E.+Bradnerauthor=W.+C.+Hahnauthor=L.+A.+Garraway&title=MTAP+deletion+confers+enhanced+dependency+on+the+PRMT5+arginine+methyltransferase+in+cancer+cells&doi=10.1126%2Fscience.aad5214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span></div><div class="casAuthors">Kryukov, Gregory V.; Wilson, Frederick H.; Ruth, Jason R.; Paulk, Joshiawa; Tsherniak, Aviad; Marlow, Sara E.; Vazquez, Francisca; Weir, Barbara A.; Fitzgerald, Mark E.; Tanaka, Minoru; Bielski, Craig M.; Scott, Justin M.; Dennis, Courtney; Cowley, Glenn S.; Boehm, Jesse S.; Root, David E.; Golub, Todd R.; Clish, Clary B.; Bradner, James E.; Hahn, William C.; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1214-1218</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The discovery of cancer dependencies has the potential to inform therapeutic strategies and to identify putative drug targets.  Integrating data from comprehensive genomic profiling of cancer cell lines and from functional characterization of cancer cell dependencies, we discovered that loss of the enzyme methylthioadenosine phosphorylase (MTAP) confers a selective dependence on protein arginine methyltransferase 5 (PRMT5) and its binding partner WDR77.  MTAP is frequently lost due to its proximity to the commonly deleted tumor suppressor gene, CDKN2A.  We obsd. increased intracellular concns. of methylthioadenosine (MTA, the metabolite cleaved by MTAP) in cells harboring MTAP deletions.  Furthermore, MTA specifically inhibited PRMT5 enzymic activity.  Administration of either MTA or a small-mol. PRMT5 inhibitor showed a modest preferential impairment of cell viability for MTAP-null cancer cell lines compared with isogenic MTAP-expressing counterparts.  Together, our findings reveal PRMT5 as a potential vulnerability across multiple cancer lineages augmented by a common "passenger" genomic alteration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94dQPnMgrqbVg90H21EOLACvtfcHk0ljyL09JT8XPmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D&md5=2b017a8c54972828c9024a9d85ff2e2d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5214%26sid%3Dliteratum%253Aachs%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DWilson%26aufirst%3DF.%2BH.%26aulast%3DRuth%26aufirst%3DJ.%2BR.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DMarlow%26aufirst%3DS.%2BE.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DBielski%26aufirst%3DC.%2BM.%26aulast%3DScott%26aufirst%3DJ.%2BM.%26aulast%3DDennis%26aufirst%3DC.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DMTAP%2520deletion%2520confers%2520enhanced%2520dependency%2520on%2520the%2520PRMT5%2520arginine%2520methyltransferase%2520in%2520cancer%2520cells%26jtitle%3DScience%26date%3D2016%26volume%3D351%26issue%3D6278%26spage%3D1214%26epage%3D1218%26doi%3D10.1126%2Fscience.aad5214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalev, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyer, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguado-Fraile, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lizotte, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuncay, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clasquin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murtie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagaraja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biller, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marjon, K.</span></span> <span> </span><span class="NLM_article-title">MAT2A inhibition blocks the growth of MTAP-deleted cancer cells by reducing PRMT5-dependent mRNA splicing and inducing DNA damage</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">209</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2020.12.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1016%2Fj.ccell.2020.12.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=33450196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhs1OmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=209-224&issue=2&author=P.+Kalevauthor=M.+L.+Hyerauthor=S.+Grossauthor=Z.+Konteatisauthor=C.+C.+Chenauthor=M.+Fletcherauthor=M.+Leinauthor=E.+Aguado-Fraileauthor=V.+Frankauthor=A.+Barnettauthor=E.+Mandleyauthor=J.+Goldfordauthor=Y.+Chenauthor=K.+Sellersauthor=S.+Hayesauthor=K.+Lizotteauthor=P.+Quangauthor=Y.+Tuncayauthor=M.+Clasquinauthor=R.+Petersauthor=J.+Weierauthor=E.+Simoneauthor=J.+Murtieauthor=W.+Liuauthor=R.+Nagarajaauthor=L.+Dangauthor=Z.+Suiauthor=S.+A.+Billerauthor=J.+Travinsauthor=K.+M.+Marksauthor=K.+Marjon&title=MAT2A+inhibition+blocks+the+growth+of+MTAP-deleted+cancer+cells+by+reducing+PRMT5-dependent+mRNA+splicing+and+inducing+DNA+damage&doi=10.1016%2Fj.ccell.2020.12.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">MAT2A inhibition blocks the growth of MTAP-deleted cancer cells by reducing PRMT5-dependent mRNA splicing and inducing DNA damage</span></div><div class="casAuthors">Kalev, Peter; Hyer, Marc L.; Gross, Stefan; Konteatis, Zenon; Chen, Chi-Chao; Fletcher, Mark; Lein, Max; Aguado-Fraile, Elia; Frank, Victoria; Barnett, Amelia; Mandley, Everton; Goldford, Joshua; Chen, Yue; Sellers, Katie; Hayes, Sebastian; Lizotte, Kate; Quang, Phong; Tuncay, Yesim; Clasquin, Michelle; Peters, Rachel; Weier, Jaclyn; Simone, Eric; Murtie, Joshua; Liu, Wei; Nagaraja, Raj; Dang, Lenny; Sui, Zhihua; Biller, Scott A.; Travins, Jeremy; Marks, Kevin M.; Marjon, Katya</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">209-224.e11</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The methylthioadenosine phosphorylase (MTAP) gene is located adjacent to the cyclin-dependent kinase inhibitor 2A (CDKN2A) tumor-suppressor gene and is co-deleted with CDKN2A in approx. 15% of all cancers.  This co-deletion leads to aggressive tumors with poor prognosis that lack effective, molecularly targeted therapies.  The metabolic enzyme methionine adenosyltransferase 2α (MAT2A) was identified as a synthetic lethal target in MTAP-deleted cancers.  We report the characterization of potent MAT2A inhibitors that substantially reduce levels of S-adenosylmethionine (SAM) and demonstrate antiproliferative activity in MTAP-deleted cancer cells and tumors.  Using RNA sequencing and proteomics, we demonstrate that MAT2A inhibition is mechanistically linked to reduced protein arginine methyltransferase 5 (PRMT5) activity and splicing perturbations.  We further show that DNA damage and mitotic defects ensue upon MAT2A inhibition in HCT116 MTAP-/- cells, providing a rationale for combining the MAT2A clin. candidate AG-270 with antimitotic taxanes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0SqOh3MkAALVg90H21EOLACvtfcHk0ljyL09JT8XPmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhs1OmsL4%253D&md5=d4c0eba5310e7a85d0904ad81331a938</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2020.12.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2020.12.010%26sid%3Dliteratum%253Aachs%26aulast%3DKalev%26aufirst%3DP.%26aulast%3DHyer%26aufirst%3DM.%2BL.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DKonteatis%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DC.%2BC.%26aulast%3DFletcher%26aufirst%3DM.%26aulast%3DLein%26aufirst%3DM.%26aulast%3DAguado-Fraile%26aufirst%3DE.%26aulast%3DFrank%26aufirst%3DV.%26aulast%3DBarnett%26aufirst%3DA.%26aulast%3DMandley%26aufirst%3DE.%26aulast%3DGoldford%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSellers%26aufirst%3DK.%26aulast%3DHayes%26aufirst%3DS.%26aulast%3DLizotte%26aufirst%3DK.%26aulast%3DQuang%26aufirst%3DP.%26aulast%3DTuncay%26aufirst%3DY.%26aulast%3DClasquin%26aufirst%3DM.%26aulast%3DPeters%26aufirst%3DR.%26aulast%3DWeier%26aufirst%3DJ.%26aulast%3DSimone%26aufirst%3DE.%26aulast%3DMurtie%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DNagaraja%26aufirst%3DR.%26aulast%3DDang%26aufirst%3DL.%26aulast%3DSui%26aufirst%3DZ.%26aulast%3DBiller%26aufirst%3DS.%2BA.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26aulast%3DMarjon%26aufirst%3DK.%26atitle%3DMAT2A%2520inhibition%2520blocks%2520the%2520growth%2520of%2520MTAP-deleted%2520cancer%2520cells%2520by%2520reducing%2520PRMT5-dependent%2520mRNA%2520splicing%2520and%2520inducing%2520DNA%2520damage%26jtitle%3DCancer%2520Cell%26date%3D2021%26volume%3D39%26issue%3D2%26spage%3D209%26epage%3D224%26doi%3D10.1016%2Fj.ccell.2020.12.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&issue=6&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lgfOLuNoYNUmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D6%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortez, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonysamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocchinfuso, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span> <span> </span><span class="NLM_article-title">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">612</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=612-617&issue=7&author=Z.+Q.+Bondayauthor=G.+S.+Cortezauthor=M.+J.+Groganauthor=S.+Antonysamyauthor=K.+Weichertauthor=W.+P.+Bocchinfusoauthor=F.+Liauthor=S.+Kennedyauthor=B.+Liauthor=M.+M.+Maderauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=E.+Guccioneauthor=R.+M.+Campbell&title=LLY-283%2C+a+Potent+and+Selective+Inhibitor+of+Arginine+Methyltransferase+5%2C+PRMT5%2C+with+Antitumor+Activity&doi=10.1021%2Facsmedchemlett.8b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity</span></div><div class="casAuthors">Bonday, Zahid Q.; Cortez, Guillermo S.; Grogan, Michael J.; Antonysamy, Stephen; Weichert, Ken; Bocchinfuso, Wayne P.; Li, Fengling; Kennedy, Steven; Li, Binghui; Mader, Mary M.; Arrowsmith, Cheryl H.; Brown, Peter J.; Eram, Mohammad S.; Szewczyk, Magdalena M.; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Guccione, Ernesto; Campbell, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">612-617</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase 5 (PRMT5) is a type II arginine methyltransferase that catalyzes the formation of sym. dimethylarginine in a no. of nuclear and cytoplasmic proteins.  Although the cellular functions of PRMT5 have not been fully unraveled, it has been implicated in a no. of cellular processes like RNA processing, signal transduction and transcriptional regulation.  PRMT5 is ubiquitously expressed in most tissues and its expression has been shown to be elevated in several cancers including breast cancer, gastric cancer, glioblastoma and lymphoma.  Here we describe the identification and characterization of a novel and selective PRMT5 inhibitor with potent in vitro and in vivo activity.  Compd. 1 (also called LLY-283) inhibited PRMT5 enzymic activity in vitro and in cells with IC50 of 22 ± 3 nM and 25 ± 1 nM, resp. while its diastereomer, compd. 2 (also called LLY-284) was much less active.  Compd. 1 also showed antitumor activity in mouse xenografts when dosed orally and can serve as an excellent probe mol. for understanding the biol. function of PRMT5 in normal and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYIq2FC-EskrVg90H21EOLACvtfcHk0lgfOLuNoYNUmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotFarurc%253D&md5=93517c34cf9f99d76730a538f594ddd7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26aulast%3DCortez%26aufirst%3DG.%2BS.%26aulast%3DGrogan%26aufirst%3DM.%2BJ.%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DBocchinfuso%26aufirst%3DW.%2BP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26atitle%3DLLY-283%252C%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520Arginine%2520Methyltransferase%25205%252C%2520PRMT5%252C%2520with%2520Antitumor%2520Activity%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26issue%3D7%26spage%3D612%26epage%3D617%26doi%3D10.1021%2Facsmedchemlett.8b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinken, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuring, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poggesi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somers, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckx, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Heerde, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nys, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanovich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herkert, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhulst, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du Jardin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meerpoel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diels, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viellevoye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepens, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poncelet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linders, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chetty, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laquerre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span> <span> </span><span class="NLM_article-title">A novel PRMT5 inhibitor with potent in vitro and in vivo activity in preclinical lung cancer models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">DDT02-04</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2017-DDT02-04</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1158%2F1538-7445.AM2017-DDT02-04" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=DDT02-04&issue=13&author=D.+Brehmerauthor=T.+F.+Wuauthor=G.+Mannensauthor=L.+Bekeauthor=P.+Vinkenauthor=D.+Gaffneyauthor=W.+M.+Sunauthor=V.+Pandeauthor=J.+W.+Thuringauthor=H.+Millarauthor=I.+Poggesiauthor=I.+Somersauthor=A.+Boeckxauthor=M.+Paradeauthor=E.+van+Heerdeauthor=T.+Nysauthor=C.+Yanovichauthor=B.+Herkertauthor=T.+Verhulstauthor=M.+Du+Jardinauthor=L.+Meerpoelauthor=C.+Moyauthor=G.+Dielsauthor=M.+Viellevoyeauthor=W.+Schepensauthor=A.+Ponceletauthor=J.+T.+Lindersauthor=E.+C.+Lawsonauthor=J.+P.+Edwardsauthor=D.+Chettyauthor=S.+Laquerreauthor=M.+V.+Lorenzi&title=A+novel+PRMT5+inhibitor+with+potent+in+vitro+and+in+vivo+activity+in+preclinical+lung+cancer+models&doi=10.1158%2F1538-7445.AM2017-DDT02-04"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-DDT02-04&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-DDT02-04%26sid%3Dliteratum%253Aachs%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DT.%2BF.%26aulast%3DMannens%26aufirst%3DG.%26aulast%3DBeke%26aufirst%3DL.%26aulast%3DVinken%26aufirst%3DP.%26aulast%3DGaffney%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DW.%2BM.%26aulast%3DPande%26aufirst%3DV.%26aulast%3DThuring%26aufirst%3DJ.%2BW.%26aulast%3DMillar%26aufirst%3DH.%26aulast%3DPoggesi%26aufirst%3DI.%26aulast%3DSomers%26aufirst%3DI.%26aulast%3DBoeckx%26aufirst%3DA.%26aulast%3DParade%26aufirst%3DM.%26aulast%3Dvan%2BHeerde%26aufirst%3DE.%26aulast%3DNys%26aufirst%3DT.%26aulast%3DYanovich%26aufirst%3DC.%26aulast%3DHerkert%26aufirst%3DB.%26aulast%3DVerhulst%26aufirst%3DT.%26aulast%3DDu%2BJardin%26aufirst%3DM.%26aulast%3DMeerpoel%26aufirst%3DL.%26aulast%3DMoy%26aufirst%3DC.%26aulast%3DDiels%26aufirst%3DG.%26aulast%3DViellevoye%26aufirst%3DM.%26aulast%3DSchepens%26aufirst%3DW.%26aulast%3DPoncelet%26aufirst%3DA.%26aulast%3DLinders%26aufirst%3DJ.%2BT.%26aulast%3DLawson%26aufirst%3DE.%2BC.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26aulast%3DChetty%26aufirst%3DD.%26aulast%3DLaquerre%26aufirst%3DS.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26atitle%3DA%2520novel%2520PRMT5%2520inhibitor%2520with%2520potent%2520in%2520vitro%2520and%2520in%2520vivo%2520activity%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26issue%3D13%26spage%3DDDT02-04%26doi%3D10.1158%2F1538-7445.AM2017-DDT02-04" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinken, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommers, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuring, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haddish-Berhane, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laquerre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span> <span> </span><span class="NLM_article-title">JNJ-64619178, a selective and pseudo-irreversible PRMT5 inhibitor with potent in vitro and in vivo activity, demonstrated in several lung cancer models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">4859</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2018-4859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1158%2F1538-7445.AM2018-4859" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=4859&issue=13&author=T.+F.+Wuauthor=H.+Millarauthor=D.+Gaffneyauthor=L.+Bekeauthor=G.+Mannensauthor=P.+Vinkenauthor=I.+Sommersauthor=J.+W.+Thuringauthor=W.+M.+Sunauthor=C.+Moyauthor=V.+Pandeauthor=J.+G.+Zhouauthor=N.+Haddish-Berhaneauthor=M.+Salvatiauthor=S.+Laquerreauthor=M.+V.+Lorenziauthor=D.+Brehmer&title=JNJ-64619178%2C+a+selective+and+pseudo-irreversible+PRMT5+inhibitor+with+potent+in+vitro+and+in+vivo+activity%2C+demonstrated+in+several+lung+cancer+models&doi=10.1158%2F1538-7445.AM2018-4859"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2018-4859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2018-4859%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%2BF.%26aulast%3DMillar%26aufirst%3DH.%26aulast%3DGaffney%26aufirst%3DD.%26aulast%3DBeke%26aufirst%3DL.%26aulast%3DMannens%26aufirst%3DG.%26aulast%3DVinken%26aufirst%3DP.%26aulast%3DSommers%26aufirst%3DI.%26aulast%3DThuring%26aufirst%3DJ.%2BW.%26aulast%3DSun%26aufirst%3DW.%2BM.%26aulast%3DMoy%26aufirst%3DC.%26aulast%3DPande%26aufirst%3DV.%26aulast%3DZhou%26aufirst%3DJ.%2BG.%26aulast%3DHaddish-Berhane%26aufirst%3DN.%26aulast%3DSalvati%26aufirst%3DM.%26aulast%3DLaquerre%26aufirst%3DS.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DBrehmer%26aufirst%3DD.%26atitle%3DJNJ-64619178%252C%2520a%2520selective%2520and%2520pseudo-irreversible%2520PRMT5%2520inhibitor%2520with%2520potent%2520in%2520vitro%2520and%2520in%2520vivo%2520activity%252C%2520demonstrated%2520in%2520several%2520lung%2520cancer%2520models%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26issue%3D13%26spage%3D4859%26doi%3D10.1158%2F1538-7445.AM2018-4859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mulvaney, K. M.</span>; <span class="NLM_string-name">Blomquist, C.</span>; <span class="NLM_string-name">Acharya, N.</span>; <span class="NLM_string-name">Li, R.</span>; <span class="NLM_string-name">O’Keefe, M.</span>; <span class="NLM_string-name">Ranaghan, M.</span>; <span class="NLM_string-name">Stokes, M.</span>; <span class="NLM_string-name">Nelson, A. J.</span>; <span class="NLM_string-name">Jain, S. S.</span>; <span class="NLM_string-name">Columbus, J.</span>; <span class="NLM_string-name">Bozal, F. K.</span>; <span class="NLM_string-name">Skepner, A.</span>; <span class="NLM_string-name">Raymond, D.</span>; <span class="NLM_string-name">McKinney, D. C.</span>; <span class="NLM_string-name">Freyzon, Y.</span>; <span class="NLM_string-name">Baidi, Y.</span>; <span class="NLM_string-name">Porter, D.</span>; <span class="NLM_string-name">Ianari, A.</span>; <span class="NLM_string-name">McMillan, B.</span>; <span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for substrate recruitment to the PRMT5 methylosome</span>. <i>bioRxiv</i>, August 22, 2020, ver. 1.<span class="refDoi"> DOI: 10.1101/2020.08.22.256347</span>  (accessed ).</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1101%2F2020.08.22.256347" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mulvaney%2C+K.+M.%3B+Blomquist%2C+C.%3B+Acharya%2C+N.%3B+Li%2C+R.%3B+O%E2%80%99Keefe%2C+M.%3B+Ranaghan%2C+M.%3B+Stokes%2C+M.%3B+Nelson%2C+A.+J.%3B+Jain%2C+S.+S.%3B+Columbus%2C+J.%3B+Bozal%2C+F.+K.%3B+Skepner%2C+A.%3B+Raymond%2C+D.%3B+McKinney%2C+D.+C.%3B+Freyzon%2C+Y.%3B+Baidi%2C+Y.%3B+Porter%2C+D.%3B+Ianari%2C+A.%3B+McMillan%2C+B.%3B+Sellers%2C+W.+R.+Molecular+basis+for+substrate+recruitment+to+the+PRMT5+methylosome.+bioRxiv%2C+August+22%2C+2020%2C+ver.+1.+10.1101%2F2020.08.22.256347+%28accessed+%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1101%2F2020.08.22.256347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26id%3Ddoi%3A10.1101%252F2020.08.22.256347%26sid%3Dliteratum%253Aachs%26aulast%3DMulvaney%26aufirst%3DK.%2BM.%26atitle%3DMolecular%2520basis%2520for%2520substrate%2520recruitment%2520to%2520the%2520PRMT5%2520methylosome%26doi%3D10.1101%2F2020.08.22.256347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandapani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marie, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemercier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murillo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akella, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcaurelle, L. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of stereochemically and skeletally diverse fused ring systems from functionalized C-glycosides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>78</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">5160</span>– <span class="NLM_lpage">5171</span>, <span class="refDoi"> DOI: 10.1021/jo4000916</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo4000916" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvFyisr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2013&pages=5160-5171&issue=11&author=B.+Gerardauthor=M.+D.+T.+Leeauthor=S.+Dandapaniauthor=J.+R.+Duvallauthor=M.+E.+Fitzgeraldauthor=S.+Kesavanauthor=J.+T.+Loweauthor=J.+C.+Marieauthor=B.+A.+Pandyaauthor=B.+C.+Suhauthor=M.+W.+O%E2%80%99Sheaauthor=M.+Dombrowskiauthor=D.+Hamannauthor=B.+Lemercierauthor=T.+Murilloauthor=L.+B.+Akellaauthor=M.+A.+Foleyauthor=L.+A.+Marcaurelle&title=Synthesis+of+stereochemically+and+skeletally+diverse+fused+ring+systems+from+functionalized+C-glycosides&doi=10.1021%2Fjo4000916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Stereochemically and Skeletally Diverse Fused Ring Systems from Functionalized C-Glycosides</span></div><div class="casAuthors">Gerard, Baudouin; Lee, Maurice D., IV; Dandapani, Sivaraman; Duvall, Jeremy R.; Fitzgerald, Mark E.; Kesavan, Sarathy; Lowe, Jason T.; Marie, Jean-Charles; Pandya, Bhaumik A.; Suh, Byung-Chul; O'Shea, Morgan Welzel; Dombrowski, Michael; Hamann, Diane; Lemercier, Berenice; Murillo, Tiffanie; Akella, Lakshmi B.; Foley, Michael A.; Marcaurelle, Lisa A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5160-5171</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A diversity-oriented synthesis (DOS) strategy was developed for the synthesis of stereochem. diverse fused-ring systems contg. a pyran moiety.  Each scaffold contains an amine and Me ester for further diversification via amine capping and amide coupling.  Scaffold diversity was evaluated in comparison to previously prepd. scaffolds by a shape-based principal moments of inertia (PMI) anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUJ_merOEm2LVg90H21EOLACvtfcHk0lhVKvf-A4mSBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvFyisr0%253D&md5=c1882e312248cc18f53ab203b881ff89</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjo4000916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo4000916%26sid%3Dliteratum%253Aachs%26aulast%3DGerard%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DM.%2BD.%2BT.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DKesavan%26aufirst%3DS.%26aulast%3DLowe%26aufirst%3DJ.%2BT.%26aulast%3DMarie%26aufirst%3DJ.%2BC.%26aulast%3DPandya%26aufirst%3DB.%2BA.%26aulast%3DSuh%26aufirst%3DB.%2BC.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DM.%2BW.%26aulast%3DDombrowski%26aufirst%3DM.%26aulast%3DHamann%26aufirst%3DD.%26aulast%3DLemercier%26aufirst%3DB.%26aulast%3DMurillo%26aufirst%3DT.%26aulast%3DAkella%26aufirst%3DL.%2BB.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26aulast%3DMarcaurelle%26aufirst%3DL.%2BA.%26atitle%3DSynthesis%2520of%2520stereochemically%2520and%2520skeletally%2520diverse%2520fused%2520ring%2520systems%2520from%2520functionalized%2520C-glycosides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2013%26volume%3D78%26issue%3D11%26spage%3D5160%26epage%3D5171%26doi%3D10.1021%2Fjo4000916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, T.</span></span> <span> </span><span class="NLM_article-title">The emerging role of human esterases</span>. <i>Drug Metab. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>27</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">466</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.2133/dmpk.DMPK-12-RV-042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.2133%2Fdmpk.DMPK-12-RV-042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=22813719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVykurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2012&pages=466-477&issue=5&author=T.+Fukamiauthor=T.+Yokoi&title=The+emerging+role+of+human+esterases&doi=10.2133%2Fdmpk.DMPK-12-RV-042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging role of human esterases</span></div><div class="casAuthors">Fukami, Tatsuki; Yokoi, Tsuyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Pharmacokinetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">466-477</span>CODEN:
                <span class="NLM_cas:coden">DMPRB8</span>;
        ISSN:<span class="NLM_cas:issn">1347-4367</span>.
    
            (<span class="NLM_cas:orgname">Japanese Society for the Study of Xenobiotics</span>)
        </div><div class="casAbstract">In this review, novel aspects of the role of esterases, which contribute to the metab. of 10% of therapeutic drugs, are described.  Esterases hydrolyze the compds. that contain ester, amide, and thioester bonds, which cause prodrug activation or detoxification.  Among esterases, carboxylesterases are well known to be involved in the hydrolysis of a variety of drugs.  Addnl., other esterases have recently received attention for their pharmacol. and toxicol. roles.  Arylacetamide deacetylase (AADAC) is involved in the hydrolysis of flutamide, phenacetin, and rifamycins.  AADAC is assocd. with adverse drug reactions because the hydrolytic metabolites of flutamide and phenacetin appear to be assocd. with hepatotoxicity and nephrotoxicity/hematotoxicity, resp.  Paraoxonase and butyrylcholinesterase hydrolyze pirocarpine/simvastatin and succinylcholine/bambuterol, resp.  Although the esterases that hydrolyze the acyl-glucuronides of drugs have largely been unknown, we recently found that α/β hydrolase domain contg. 10 (ABHD10) is responsible for the hydrolysis of mycophenolic acid acyl-glucuronide in human liver.  Because acyl-glucuronides are assocd. with toxicity, ABHD10 might function as a detoxification enzyme.  Thus, various esterases, which include enzymes that have not been known to hydrolyze drugs, are involved in drug metab. with different substrate specificity.  Further esterase studies should be conducted to promote our understanding in clin. pharmacotherapy and drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDhsUJ4NrYQbVg90H21EOLACvtfcHk0lgEshkPnHRBAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVykurzE&md5=d760b8a6a3217a4ab9716190800012b0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2133%2Fdmpk.DMPK-12-RV-042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2133%252Fdmpk.DMPK-12-RV-042%26sid%3Dliteratum%253Aachs%26aulast%3DFukami%26aufirst%3DT.%26aulast%3DYokoi%26aufirst%3DT.%26atitle%3DThe%2520emerging%2520role%2520of%2520human%2520esterases%26jtitle%3DDrug%2520Metab.%2520Pharmacokinet.%26date%3D2012%26volume%3D27%26issue%3D5%26spage%3D466%26epage%3D477%26doi%3D10.2133%2Fdmpk.DMPK-12-RV-042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedlacek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manoharan, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boopathy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duysen, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockridge, O.</span></span> <span> </span><span class="NLM_article-title">Butyrylcholinesterase, paraoxonase, and albumin esterase, but not carboxylesterase, are present in human plasma</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>70</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1673</span>– <span class="NLM_lpage">1684</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2005.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1016%2Fj.bcp.2005.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=16213467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKhsLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2005&pages=1673-1684&issue=11&author=B.+Liauthor=M.+Sedlacekauthor=I.+Manoharanauthor=R.+Boopathyauthor=E.+G.+Duysenauthor=P.+Massonauthor=O.+Lockridge&title=Butyrylcholinesterase%2C+paraoxonase%2C+and+albumin+esterase%2C+but+not+carboxylesterase%2C+are+present+in+human+plasma&doi=10.1016%2Fj.bcp.2005.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Butyrylcholinesterase, paraoxonase, and albumin esterase, but not carboxylesterase, are present in human plasma</span></div><div class="casAuthors">Li, Bin; Sedlacek, Meghan; Manoharan, Indumathi; Boopathy, Rathnam; Duysen, Ellen G.; Masson, Patrick; Lockridge, Oksana</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1673-1684</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The goal of this work was to identify the esterases in human plasma and to clarify common misconceptions.  The method for identifying esterases was nondenaturing gradient gel electrophoresis stained for esterase activity.  We report that human plasma contains four esterases: butyrylcholinesterase (E.C. 3.1.1.8), paraoxonase (E.C. 3.1.8.1), acetylcholinesterase (E.C. 3.1.1.7), and albumin.  Butyrylcholinesterase (BChE), paraoxonase (PON1), and albumin are in high enough concns. to contribute significantly to ester hydrolysis.  However, only trace amts. of acetylcholinesterase (AChE) are present.  Monomeric AChE is seen in wild-type as well as in silent BChE plasma.  Albumin has esterase activity with alpha- and beta-naphthylacetate as well as with p-nitrophenyl acetate.  Misconception #1 is that human plasma contains carboxylesterase.  We demonstrate that human plasma contains no carboxylesterase (E.C. 3.1.1.1), in contrast to mouse, rat, rabbit, horse, cat, and tiger that have high amts. of plasma carboxylesterase.  Misconception #2 is that lab. animals have BChE but no AChE in their plasma.  We demonstrate that mice, unlike humans, have substantial amts. of sol. AChE as well as BChE in their plasma.  Plasma from AChE and BChE knockout mice allowed identification of AChE and BChE bands without the use of inhibitors.  Human BChE is irreversibly inhibited by diisopropylfluorophosphate, echothiophate, and paraoxon, but mouse BChE spontaneously reactivates.  Since human plasma contains no carboxylesterase, only BChE, PON1, and albumin esterases need to be considered when evaluating hydrolysis of an ester drug in human plasma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobFY6T-ENpA7Vg90H21EOLACvtfcHk0lgEshkPnHRBAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKhsLzJ&md5=d9d557d16e4e43d82f1c2d391dc3ac89</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2005.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2005.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DSedlacek%26aufirst%3DM.%26aulast%3DManoharan%26aufirst%3DI.%26aulast%3DBoopathy%26aufirst%3DR.%26aulast%3DDuysen%26aufirst%3DE.%2BG.%26aulast%3DMasson%26aufirst%3DP.%26aulast%3DLockridge%26aufirst%3DO.%26atitle%3DButyrylcholinesterase%252C%2520paraoxonase%252C%2520and%2520albumin%2520esterase%252C%2520but%2520not%2520carboxylesterase%252C%2520are%2520present%2520in%2520human%2520plasma%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2005%26volume%3D70%26issue%3D11%26spage%3D1673%26epage%3D1684%26doi%3D10.1016%2Fj.bcp.2005.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Timm, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madsen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, C.</span></span> <span> </span><span class="NLM_article-title">Cryo-electron microscopy structure of a human PRMT5:MEP50 complex</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">3</span>), <span class="NLM_elocation-id">e0193205</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0193205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1371%2Fjournal.pone.0193205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=29518110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWms7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&issue=3&author=D.+E.+Timmauthor=V.+Bowmanauthor=R.+Madsenauthor=C.+Rauch&title=Cryo-electron+microscopy+structure+of+a+human+PRMT5%3AMEP50+complex&doi=10.1371%2Fjournal.pone.0193205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-electron microscopy structure of a human PRMT5:MEP50 complex</span></div><div class="casAuthors">Timm, David E.; Bowman, Valorie; Madsen, Russell; Rauch, Charles</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0193205/1-e0193205/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein arginine Me transferase 5 (PRMT5) is a signaling protein and histone modifying enzyme that is important in many cellular processes, including regulation of eukaryotic gene transcription.  Reported here is a 3.7 Å structure of PRMT5, solved in complex with regulatory binding subunit MEP50 (methylosome assocd. protein 50, WDR77, p44), by single particle (SP) cryo-Electron Microscopy (cryo-EM) using micrographs of particles that are visibly crowded and aggregated.  Despite suboptimal micrograph appearance, this cryo-EM structure is in good agreement with previously reported crystal structures of the complex, which revealed a 450 kDa hetero-octameric assembly having internal D2 symmetry.  The catalytic PRMT5 subunits form a core tetramer and the MEP50 subunits are arranged peripherally in complex with the PRMT5 N-terminal domain.  The cryo-EM reconstruction shows good side chain definition and shows a well-resolved peak for a bound dehydrosinefungin inhibitor mol.  These results demonstrate the applicability of cryo-EM in detg. structures of human protein complexes of biomedical significance and suggests cryo-EM could be further utilized to understand PRMT5 interactions with other biol. important binding proteins and ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVrd4Ga-2QJrVg90H21EOLACvtfcHk0lgEshkPnHRBAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWms7%252FO&md5=e2410a0d8b80ec2f9c37bb96b390b46f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0193205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0193205%26sid%3Dliteratum%253Aachs%26aulast%3DTimm%26aufirst%3DD.%2BE.%26aulast%3DBowman%26aufirst%3DV.%26aulast%3DMadsen%26aufirst%3DR.%26aulast%3DRauch%26aufirst%3DC.%26atitle%3DCryo-electron%2520microscopy%2520structure%2520of%2520a%2520human%2520PRMT5%253AMEP50%2520complex%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26issue%3D3%26doi%3D10.1371%2Fjournal.pone.0193205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, M.</span></span> <span> </span><span class="NLM_article-title">The role of extrahepatic metabolism in the pharmacokinetics of the targeted covalent inhibitors afatinib, ibrutinib, and neratinib</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1124/dmd.114.061424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1124%2Fdmd.114.061424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=25504185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1OmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=375-384&issue=3&author=Y.+Shibataauthor=M.+Chiba&title=The+role+of+extrahepatic+metabolism+in+the+pharmacokinetics+of+the+targeted+covalent+inhibitors+afatinib%2C+ibrutinib%2C+and+neratinib&doi=10.1124%2Fdmd.114.061424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The role of extrahepatic metabolism in the pharmacokinetics of the targeted covalent inhibitors afatinib, ibrutinib, and neratinib</span></div><div class="casAuthors">Shibata, Yoshihiro; Chiba, Masato</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">375-384, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Despite the fact that much progress has been made recently in the development of targeted covalent inhibitors (TCIs), their pharmacokinetics (PK) have not been well characterized in the light of extrahepatic clearance (CLextH) by glutathione (GSH)/glutathione S-transferase (GST)-dependent conjugation attributable to the unique electrophilic structure (e.g., acrylamide moiety) of TCI compds.  In the present study, CLextH values were examd. in rat, dog, and monkey to predict the contribution of CLextH to the PK of the TCIs afatinib, ibrutinib, and neratinib in humans.  Afatinib and neratinib both underwent extensive conjugation with GSH in buffer and cytosol fractions of liver and kidney, whereas ibrutinib showed much lower reactivity/susceptibility to GSH/GST-dependent conjugation.  The CLextH in each species was calcd. from the difference between obsd. total body clearance and predicted hepatic clearance (CLH) in cryopreserved hepatocytes suspended in 100% serum of the corresponding species.  The power-based simple allometry relating the CLextH for the unbound compd. to animal body wt. was applicable across species for afatinib and neratinib (R2 ≥ 0.9) but not for ibrutinib (R2 = 0.04).  The predicted AUC after oral administration of afatinib and neratinib agreed reasonably closely with reported values in phase I dose-escalation studies.  Comparisons of CLextH and CLH predicted that CLextH largely detd. the PK of afatinib (>90% as a proportion of total body clearance) and neratinib (∼34%) in humans.  The present method can serve as one of the tools for the optimization of PK in humans at the discovery stage for the development of TCI candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquWC6d6e6DU7Vg90H21EOLACvtfcHk0lh1iZMdwb8dUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1OmsrY%253D&md5=eaac67ce7cea5f2432c5144922ba3dd2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1124%2Fdmd.114.061424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.114.061424%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DY.%26aulast%3DChiba%26aufirst%3DM.%26atitle%3DThe%2520role%2520of%2520extrahepatic%2520metabolism%2520in%2520the%2520pharmacokinetics%2520of%2520the%2520targeted%2520covalent%2520inhibitors%2520afatinib%252C%2520ibrutinib%252C%2520and%2520neratinib%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26issue%3D3%26spage%3D375%26epage%3D384%26doi%3D10.1124%2Fdmd.114.061424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelow, J. M.</span></span> <span> </span><span class="NLM_article-title">A Perspective on the Kinetics of Covalent and Irreversible Inhibition</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1177/1087057116671509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1177%2F1087057116671509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=27703080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=3-20&issue=1&author=J.+M.+Strelow&title=A+Perspective+on+the+Kinetics+of+Covalent+and+Irreversible+Inhibition&doi=10.1177%2F1087057116671509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on the kinetics of covalent and irreversible inhibition</span></div><div class="casAuthors">Strelow, John M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-20</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The clin. and com. success of covalent drugs has prompted a renewed and more deliberate pursuit of covalent and irreversible mechanisms within drug discovery.  A covalent mechanism can produce potent inhibition in a biochem., cellular, or in vivo setting.  In many cases, teams choose to focus on the consequences of the covalent event, defined by an IC50 value.  In a biochem. assay, the IC50 may simply reflect the target protein concn. in the assay.  What has received less attention is the importance of the rate of covalent modification, defined by kinact/KI.  The kinact/KI is a rate const. describing the efficiency of covalent bond formation resulting from the potency (KI) of the first reversible binding event and the max. potential rate (kinact) of inactivation.  In this perspective, it is proposed that the kinact/KI should be employed as a crit. parameter to identify covalent inhibitors, interpret structure-activity relationships (SARs), translate activity from biochem. assays to the cell, and more accurately define selectivity.  It is also proposed that a physiol. relevant kinact/KI and an (unbound) AUC generated from a pharmacokinetic profile reflecting direct exposure of the inhibitor to the target protein are two crit. determinants of in vivo covalent occupancy.  A simple equation is presented to define this relationship and improve the interpretation of covalent and irreversible kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgzDNXDESHDbVg90H21EOLACvtfcHk0lh1iZMdwb8dUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSntrbJ&md5=bd404e6b26df20b8761cd38f0f3c3541</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1087057116671509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116671509%26sid%3Dliteratum%253Aachs%26aulast%3DStrelow%26aufirst%3DJ.%2BM.%26atitle%3DA%2520Perspective%2520on%2520the%2520Kinetics%2520of%2520Covalent%2520and%2520Irreversible%2520Inhibition%26jtitle%3DSLAS%2520Discov%26date%3D2017%26volume%3D22%26issue%3D1%26spage%3D3%26epage%3D20%26doi%3D10.1177%2F1087057116671509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderton, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bethel, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butterworth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colclough, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorley, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debreczeni, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberlein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finlay, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orme, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span> <span> </span><span class="NLM_article-title">Structure- and reactivity-based development of covalent inhibitors of the activating and gatekeeper mutant forms of the epidermal growth factor receptor (EGFR)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">7025</span>– <span class="NLM_lpage">7048</span>, <span class="refDoi"> DOI: 10.1021/jm400822z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400822z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1CltL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7025-7048&issue=17&author=R.+A.+Wardauthor=M.+J.+Andertonauthor=S.+Ashtonauthor=P.+A.+Bethelauthor=M.+Boxauthor=S.+Butterworthauthor=N.+Colcloughauthor=C.+G.+Chorleyauthor=C.+Chuaquiauthor=D.+A.+Crossauthor=L.+A.+Dakinauthor=J.+E.+Debreczeniauthor=C.+Eberleinauthor=M.+R.+Finlayauthor=G.+B.+Hillauthor=M.+Gristauthor=T.+C.+Klinowskaauthor=C.+Laneauthor=S.+Martinauthor=J.+P.+Ormeauthor=P.+Smithauthor=F.+Wangauthor=M.+J.+Waring&title=Structure-+and+reactivity-based+development+of+covalent+inhibitors+of+the+activating+and+gatekeeper+mutant+forms+of+the+epidermal+growth+factor+receptor+%28EGFR%29&doi=10.1021%2Fjm400822z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure- and Reactivity-Based Development of Covalent Inhibitors of the Activating and Gatekeeper Mutant Forms of the Epidermal Growth Factor Receptor (EGFR)</span></div><div class="casAuthors">Ward, Richard A.; Anderton, Mark J.; Ashton, Susan; Bethel, Paul A.; Box, Matthew; Butterworth, Sam; Colclough, Nicola; Chorley, Christopher G.; Chuaqui, Claudio; Cross, Darren A. E.; Dakin, Les A.; Debreczeni, Judit E.; Eberlein, Cath; Finlay, M. Raymond V.; Hill, George B.; Grist, Matthew; Klinowska, Teresa C. M.; Lane, Clare; Martin, Scott; Orme, Jonathon P.; Smith, Peter; Wang, Fengjiang; Waring, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7025-7048</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of small-mol. inhibitors has been developed to target the double mutant form of the epidermal growth factor receptor (EGFR) tyrosine kinase, which is resistant to treatment with gefitinib and erlotinib.  Our reported compds. also show selectivity over wild-type EGFR.  Guided by mol. modeling, this series was evolved to target a cysteine residue in the ATP binding site via covalent bond formation and demonstrates high levels of activity in cellular models of the double mutant form of EGFR.  In addn., these compds. show significant activity against the activating mutations, which gefitinib and erlotinib target and inhibition of which gives rise to their obsd. clin. efficacy.  A glutathione (GSH)-based assay was used to measure thiol reactivity toward the electrophilic functionality of the inhibitor series, enabling both the identification of a suitable reactivity window for their potency and the development of a reactivity quant. structure-property relationship (QSPR) to support design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKi6oNvtQPEbVg90H21EOLACvtfcHk0lh1iZMdwb8dUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1CltL%252FN&md5=59c86415d71ac2abe5c6d4486e5d2517</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm400822z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400822z%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DAnderton%26aufirst%3DM.%2BJ.%26aulast%3DAshton%26aufirst%3DS.%26aulast%3DBethel%26aufirst%3DP.%2BA.%26aulast%3DBox%26aufirst%3DM.%26aulast%3DButterworth%26aufirst%3DS.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DChorley%26aufirst%3DC.%2BG.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DDebreczeni%26aufirst%3DJ.%2BE.%26aulast%3DEberlein%26aufirst%3DC.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DHill%26aufirst%3DG.%2BB.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DKlinowska%26aufirst%3DT.%2BC.%26aulast%3DLane%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DStructure-%2520and%2520reactivity-based%2520development%2520of%2520covalent%2520inhibitors%2520of%2520the%2520activating%2520and%2520gatekeeper%2520mutant%2520forms%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D17%26spage%3D7025%26epage%3D7048%26doi%3D10.1021%2Fjm400822z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">9977</span>– <span class="NLM_lpage">9989</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01111</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01111" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFOisLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9977-9989&issue=17&author=Y.+Shenauthor=G.+Gaoauthor=X.+Yuauthor=H.+Kimauthor=L.+Wangauthor=L.+Xieauthor=M.+Schwarzauthor=X.+Chenauthor=E.+Guccioneauthor=J.+Liuauthor=M.+T.+Bedfordauthor=J.+Jin&title=Discovery+of+First-in-Class+Protein+Arginine+Methyltransferase+5+%28PRMT5%29+Degraders&doi=10.1021%2Facs.jmedchem.0c01111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders</span></div><div class="casAuthors">Shen, Yudao; Gao, Guozhen; Yu, Xufen; Kim, Huensuk; Wang, Li; Xie, Ling; Schwarz, Megan; Chen, Xian; Guccione, Ernesto; Liu, Jing; Bedford, Mark T.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9977-9989</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The aberrant expression of protein arginine methyltransferase 5 (PRMT5) has been assocd. with multiple cancers.  Using the proteolysis targeting chimera technol., we discovered a first-in-class PRMT5 degrader 15 (MS4322).  Here, we report the design, synthesis, and characterization of compd. 15 and two structurally similar controls 17 (MS4370) and 21 (MS4369), with impaired binding to the von Hippel-Lindau E3 ligase and PRMT5, resp.  Compd. 15, but not 17 and 21, effectively reduced the PRMT5 protein level in MCF-7 cells.  Our mechanism studies indicate that compd. 15 degraded PRMT5 in an E3 ligase- and proteasome-dependent manner.  Compd. 15 also effectively reduced the PRMT5 protein level and inhibited growth in multiple cancer cell lines.  Moreover, compd. 15 was highly selective for PRMT5 in a global proteomic study and exhibited good plasma exposure in mice.  Collectively, compd. 15 and its two controls 17 and 21 are valuable chem. tools for exploring the PRMT5 functions in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZoL-YbkPwxrVg90H21EOLACvtfcHk0lif4MUdkW2X6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOisLrE&md5=5400a4b400d1a420eff5b24f735de462</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01111%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520First-in-Class%2520Protein%2520Arginine%2520Methyltransferase%25205%2520%2528PRMT5%2529%2520Degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26issue%3D17%26spage%3D9977%26epage%3D9989%26doi%3D10.1021%2Facs.jmedchem.0c01111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grant, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grigorieff, N.</span></span> <span> </span><span class="NLM_article-title">cisTEM, user-friendly software for single-particle image processing</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e35383</span>, <span class="refDoi"> DOI: 10.7554/eLife.35383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.7554%2FeLife.35383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=29513216" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=e35383&author=T.+Grantauthor=A.+Rohouauthor=N.+Grigorieff&title=cisTEM%2C+user-friendly+software+for+single-particle+image+processing&doi=10.7554%2FeLife.35383"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.7554%2FeLife.35383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.35383%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DT.%26aulast%3DRohou%26aufirst%3DA.%26aulast%3DGrigorieff%26aufirst%3DN.%26atitle%3DcisTEM%252C%2520user-friendly%2520software%2520for%2520single-particle%2520image%2520processing%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26spage%3De35383%26doi%3D10.7554%2FeLife.35383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marx, M. A.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery Of MRTX9768, A Synthetic Lethal-Based Inhibitor Designed To Bind The PRMT5·TA Complex And Selectively Target MTAPDEL Tumors</span>. Proceedings from the  <i>AACR Annual Meeting 2021</i>, April 10–15 and May 17–21, 2021; <span class="NLM_publisher-name">American Association for Cancer Research</span>: <span class="NLM_publisher-loc">Philadelphia, PA</span>, <span class="NLM_year">2021</span>; p  <span class="NLM_fpage">LB003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&pages=LB003&author=M.+A.+Marx&title=Fragment-Based+Discovery+Of+MRTX9768%2C+A+Synthetic+Lethal-Based+Inhibitor+Designed+To+Bind+The+PRMT5%C2%B7TA+Complex+And+Selectively+Target+MTAPDEL+Tumors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DMarx%26aufirst%3DM.%2BA.%26atitle%3DFragment-Based%2520Discovery%2520Of%2520MRTX9768%252C%2520A%2520Synthetic%2520Lethal-Based%2520Inhibitor%2520Designed%2520To%2520Bind%2520The%2520PRMT5%25C2%25B7TA%2520Complex%2520And%2520Selectively%2520Target%2520MTAPDEL%2520Tumors%26jtitle%3DAACR%2520Annual%2520Meeting%25202021%26pub%3DAmerican%2520Association%2520for%2520Cancer%2520Research%26date%3D2021%26spage%3DLB003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malik, R.</span>; <span class="NLM_string-name">Park, P. K.</span>; <span class="NLM_string-name">Barbieri, C.</span>; <span class="NLM_string-name">Blat, Y.</span>; <span class="NLM_string-name">Sheriff, S.</span>; <span class="NLM_string-name">Weigelt, C.</span>; <span class="NLM_string-name">Kopcho, L.</span>; <span class="NLM_string-name">Celiktas, M.</span>; <span class="NLM_string-name">Ruzanov, M.</span>; <span class="NLM_string-name">Naglich, J.</span>; <span class="NLM_string-name">Price, J.</span>; <span class="NLM_string-name">Harner, M.</span>; <span class="NLM_string-name">O’Malley, K.</span>; <span class="NLM_string-name">Den, J.</span>; <span class="NLM_string-name">Schmitz, W.</span>; <span class="NLM_string-name">Li, G.</span>; <span class="NLM_string-name">Ruan, Z.</span>; <span class="NLM_string-name">Qin, L.</span>; <span class="NLM_string-name">Duke, G.</span>; <span class="NLM_string-name">Rodrigo, I.</span>; <span class="NLM_string-name">Witmer, M.</span>; <span class="NLM_string-name">Harden, D.</span>; <span class="NLM_string-name">Arey, B.</span>; <span class="NLM_string-name">Soars, M.</span>; <span class="NLM_string-name">Fink, B.</span>; <span class="NLM_string-name">Gavai, A.</span>; <span class="NLM_string-name">Vite, G.</span>; <span class="NLM_string-name">Voliva, C. F.</span></span> <span> </span><span class="NLM_article-title">A novel MTA non-competitive PRMT5 inhibitor</span>. Proceedings from the  <i>AACR Annual Meeting 2021</i>, April 10–15 and May 17–21, 2021; <span class="NLM_publisher-name">American Association for Cancer Research</span>: <span class="NLM_publisher-loc">Philadelphia, PA</span>, <span class="NLM_year">2021</span>;  <span class="NLM_fpage">1140</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;key=10.1158%2F1538-7445.AM2021-1140" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&pages=1140&author=R.+Malik&author=P.+K.+Park&author=C.+Barbieri&author=Y.+Blat&author=S.+Sheriff&author=C.+Weigelt&author=L.+Kopcho&author=M.+Celiktas&author=M.+Ruzanov&author=J.+Naglich&author=J.+Price&author=M.+Harner&author=K.+O%E2%80%99Malley&author=J.+Den&author=W.+Schmitz&author=G.+Li&author=Z.+Ruan&author=L.+Qin&author=G.+Duke&author=I.+Rodrigo&author=M.+Witmer&author=D.+Harden&author=B.+Arey&author=M.+Soars&author=B.+Fink&author=A.+Gavai&author=G.+Vite&author=C.+F.+Voliva&title=A+novel+MTA+non-competitive+PRMT5+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2021-1140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26id%3Ddoi%3A10.1158%252F1538-7445.AM2021-1140%26sid%3Dliteratum%253Aachs%26aulast%3DMalik%26aufirst%3DR.%26atitle%3DA%2520novel%2520MTA%2520non-competitive%2520PRMT5%2520inhibitor%26jtitle%3DAACR%2520Annual%2520Meeting%25202021%26pub%3DAmerican%2520Association%2520for%2520Cancer%2520Research%26date%3D2021%26spage%3D1140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0P','PDB','6V0P'); return false;">PDB: 6V0P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7M05" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7M05','PDB','7M05'); return false;">PDB: 7M05</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6V0N','PDB','6V0N'); return false;">PDB: 6V0N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UGH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6UGH','PDB','6UGH'); return false;">PDB: 6UGH</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i108"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00507">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_06525"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00507?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00507</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional methods: NMR-based fragment screening; virtual screening; crystallography data collection and refinement statistics; electron density maps; protein mass spectra data; kinetic parameters plotted data; <sup>1</sup>H and <sup>13</sup>C NMR spectra; LCMS chromatograms (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_001.pdf">jm1c00507_si_001.pdf (1.55 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00507/suppl_file/jm1c00507_si_002.csv">jm1c00507_si_002.csv (2.77 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00507&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00507%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00507" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679918751b9824ad","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
